<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/3/1113" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/3/1113.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/3/1113.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/3/1113.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/3/1113/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/3/1113.atom" />
<meta name="HW.pisa" content="ggg;10/3/1113" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (Hordeum vulgare L.)" />
<meta name="DC.Identifier" content="10.1534/g3.119.400968" />
<meta name="DC.Date" content="2020-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Bhatta et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Plant breeders regularly evaluate multiple traits across multiple environments, which opens an avenue for using multiple traits in genomic prediction models. We assessed the potential of multi-trait (MT) genomic prediction model through evaluating several strategies of incorporating multiple traits (eight agronomic and malting quality traits) into the prediction models with two cross-validation schemes (CV1, predicting new lines with genotypic information only and CV2, predicting partially phenotyped lines using both genotypic and phenotypic information from correlated traits) in barley. The predictive ability was similar for single (ST-CV1) and multi-trait (MT-CV1) models to predict new lines. However, the predictive ability for agronomic traits was considerably increased when partially phenotyped lines (MT-CV2) were used. The predictive ability for grain yield using the MT-CV2 model with other agronomic traits resulted in 57% and 61% higher predictive ability than ST-CV1 and MT-CV1 models, respectively. Therefore, complex traits such as grain yield are better predicted when correlated traits are used. Similarly, a considerable increase in the predictive ability of malting quality traits was observed when correlated traits were used. The predictive ability for grain protein content using the MT-CV2 model with both agronomic and malting traits resulted in a 76% higher predictive ability than ST-CV1 and MT-CV1 models. Additionally, the higher predictive ability for new environments was obtained for all traits using the MT-CV2 model compared to the MT-CV1 model. This study showed the potential of improving the genomic prediction of complex traits by incorporating the information from multiple traits (cost-friendly and easy to measure traits) collected throughout breeding programs which could assist in speeding up breeding cycles." />
<meta name="DC.Contributor" content="Madhav Bhatta" />
<meta name="DC.Contributor" content="Lucia Gutierrez" />
<meta name="DC.Contributor" content="Lorena Cammarota" />
<meta name="DC.Contributor" content="Fernanda Cardozo" />
<meta name="DC.Contributor" content="Silvia Germán" />
<meta name="DC.Contributor" content="Blanca Gómez-Guerrero" />
<meta name="DC.Contributor" content="María Fernanda Pardo" />
<meta name="DC.Contributor" content="Valeria Lanaro" />
<meta name="DC.Contributor" content="Mercedes Sayas" />
<meta name="DC.Contributor" content="Ariel J. Castro" />
<meta name="article:published_time" content="2020-03-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (Hordeum vulgare L.)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Plant breeders regularly evaluate multiple traits across multiple environments, which opens an avenue for using multiple traits in genomic prediction models. We assessed the potential of multi-trait (MT) genomic prediction model through evaluating several strategies of incorporating multiple traits (eight agronomic and malting quality traits) into the prediction models with two cross-validation schemes (CV1, predicting new lines with genotypic information only and CV2, predicting partially phenotyped lines using both genotypic and phenotypic information from correlated traits) in barley. The predictive ability was similar for single (ST-CV1) and multi-trait (MT-CV1) models to predict new lines. However, the predictive ability for agronomic traits was considerably increased when partially phenotyped lines (MT-CV2) were used. The predictive ability for grain yield using the MT-CV2 model with other agronomic traits resulted in 57% and 61% higher predictive ability than ST-CV1 and MT-CV1 models, respectively. Therefore, complex traits such as grain yield are better predicted when correlated traits are used. Similarly, a considerable increase in the predictive ability of malting quality traits was observed when correlated traits were used. The predictive ability for grain protein content using the MT-CV2 model with both agronomic and malting traits resulted in a 76% higher predictive ability than ST-CV1 and MT-CV1 models. Additionally, the higher predictive ability for new environments was obtained for all traits using the MT-CV2 model compared to the MT-CV1 model. This study showed the potential of improving the genomic prediction of complex traits by incorporating the information from multiple traits (cost-friendly and easy to measure traits) collected throughout breeding programs which could assist in speeding up breeding cycles.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/03/01" />
<meta name="citation_mjid" content="ggg;10/3/1113" />
<meta name="citation_id" content="10/3/1113" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/3/1113" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/3/1113.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/3/1113.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/3/1113.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400968" />
<meta name="citation_pmid" content="31974097" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="1113" />
<meta name="citation_lastpage" content="1124" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Madhav Bhatta" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, 1575 Linden Dr., WI, 53706," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4959-0481" />
<meta name="citation_author" content="Lucia Gutierrez" />
<meta name="citation_author_institution" content="Department of Agronomy, University of Wisconsin-Madison, 1575 Linden Dr., WI, 53706," />
<meta name="citation_author_email" content="gutierrezcha@wisc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2957-3086" />
<meta name="citation_author" content="Lorena Cammarota" />
<meta name="citation_author_institution" content="Department of plant production, Facultad de Agronomía, Universidad de la República, Ruta 3, Km363, Paysandú 60000, Uruguay," />
<meta name="citation_author_institution" content="Maltería Uruguay S.A. Ruta 55, Km26, Ombúes de Lavalle, Uruguay," />
<meta name="citation_author" content="Fernanda Cardozo" />
<meta name="citation_author_institution" content="Maltería Uruguay S.A. Ruta 55, Km26, Ombúes de Lavalle, Uruguay," />
<meta name="citation_author" content="Silvia Germán" />
<meta name="citation_author_institution" content="Instituto Nacional de Investigación Agropuecuaria, Estación Experimental La Estanzuela, Ruta 50, Km11, Colonia, Uruguay," />
<meta name="citation_author" content="Blanca Gómez-Guerrero" />
<meta name="citation_author_institution" content="Latitud, LATU Foundation, Av Italia 6201, Montevideo 11500, Uruguay, and" />
<meta name="citation_author" content="María Fernanda Pardo" />
<meta name="citation_author_institution" content="Maltería Oriental S.A., Camino Abrevadero 5525, Montevideo 12400, Uruguay" />
<meta name="citation_author" content="Valeria Lanaro" />
<meta name="citation_author_institution" content="Latitud, LATU Foundation, Av Italia 6201, Montevideo 11500, Uruguay, and" />
<meta name="citation_author" content="Mercedes Sayas" />
<meta name="citation_author_institution" content="Maltería Oriental S.A., Camino Abrevadero 5525, Montevideo 12400, Uruguay" />
<meta name="citation_author" content="Ariel J. Castro" />
<meta name="citation_author_institution" content="Department of plant production, Facultad de Agronomía, Universidad de la República, Ruta 3, Km363, Paysandú 60000, Uruguay," />
<meta name="citation_reference" content="citation_author=V. Belamkar;citation_author=M.J. Guttieri;citation_author=W. Hussain;citation_author=D. Jarquín;citation_author=I. El-basyoni;citation_year=2018;citation_pmid=29945967;citation_doi=10.1534/g3.118.200415" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=I. M. Berro;citation_author=B. Lado;citation_author=R. S. Nalin;citation_author=M. Quincke;citation_author=L. Gutierrez;citation_title=Training population optimization for genomic selection.;citation_pages=190028;citation_volume=12;citation_year=2019;citation_doi=10.3835/plantgenome2019.04.0028" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Agron.;citation_author=N. O. Bertholdsson;citation_title=Characterization of malting barley cultivars with more or less stable grain protein content under varying environmental conditions.;citation_pages=1-8;citation_volume=10;citation_year=1999;citation_doi=10.1016/S1161-0301(98)00043-4" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Mol. Sci.;citation_author=M. Bhatta;citation_author=P. Baenziger;citation_author=B. Waters;citation_author=R. Poudel;citation_author=V. Belamkar;citation_title=Genome-wide association study reveals novel genomic regions associated with 10 grain minerals in synthetic hexaploid wheat.;citation_pages=3237;citation_volume=19;citation_year=2018;citation_doi=10.3390/ijms19103237" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Mol. Sci.;citation_author=M. Bhatta;citation_author=A. Morgounov;citation_author=V. Belamkar;citation_author=P. Baenziger;citation_title=Genome-wide association study reveals novel genomic regions for grain yield and yield-related traits in drought-stressed synthetic hexaploid wheat.;citation_pages=3011;citation_volume=19;citation_year=2018;citation_doi=10.3390/ijms19103011" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype× environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. P. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods.;citation_pages=26-26;citation_volume=43;citation_year=2011;citation_pmid=21729282;citation_doi=10.1186/1297-9686-43-26" />
<meta name="citation_reference" content="de los Campos, G., and A. Grüneberg, 2016 MTM Package. http://quantgen.github.io/MTM/vignette.html (accessed February 28, 2019)." />
<meta name="citation_reference" content="Covarrubias-Pazaran, G., 2018. Quantitative genetics using the sommer package. https://cran.r-project.org/web/packages/sommer/vignettes/sommer.pdf (accessed February 28, 2019)." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Pérez;citation_author=J. M. Hickey;citation_title=Genomic prediction in maize breeding populations with genotyping-by-sequencing. G3: Genes, Genomes;citation_pages=1903-1926;citation_volume=3;citation_year=2013;citation_doi=10.1534/g3.113.008227" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Pérez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=B. R. Cullis;citation_author=A. B. Smith;citation_author=N. E. Coombes;citation_title=On the design of early generation variety trials with correlated data.;citation_pages=381-393;citation_volume=11;citation_year=2006;citation_doi=10.1198/108571106X154443" />
<meta name="citation_reference" content="citation_journal_title=Food Chem.;citation_author=F. Dai;citation_author=J. Wang;citation_author=S. Zhang;citation_author=Z. Xu;citation_author=G. Zhang;citation_title=Genotypic and environmental variation in phytic acid content and its relation to protein content and malt quality in barley.;citation_pages=606-611;citation_volume=105;citation_year=2007;citation_doi=10.1016/j.foodchem.2007.04.019" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. P. Dos Santos;citation_author=R. C. Vasconcellos;citation_author=L. P. Pires;citation_author=M. Balestre;citation_author=R. G. Von Pinho;citation_title=Inclusion of dominance effects in the multivariate GBLUP model.;citation_pages=e0152045;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0152045" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=J. Duangjit;citation_author=M. Causse;citation_author=C. Sauvage;citation_title=Efficiency of genomic selection for tomato fruit quality.;citation_pages=29;citation_volume=36;citation_year=2016;citation_doi=10.1007/s11032-016-0453-3" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. B. Endelman;citation_author=G. N. Atlin;citation_author=Y. Beyene;citation_author=K. Semagn;citation_author=X. Zhang;citation_title=Optimal design of preliminary yield trials with genome-wide markers.;citation_pages=48-59;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.03.0154" />
<meta name="citation_reference" content="FAOSTAT, 2019 Food and Agriculture Organization of the United Nations. http://www.fao.org/faostat/en/#data (accessed February 28, 2019)." />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=Y. Guo;citation_author=J. He;citation_author=S. Zhao;citation_author=H. Wu;citation_author=X. Zhong;citation_title=Illumina human exome genotyping array clustering and quality control.;citation_pages=2643-2662;citation_volume=9;citation_year=2014;citation_pmid=25321409;citation_doi=10.1038/nprot.2014.174" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=G. Guo;citation_author=F. Zhao;citation_author=Y. Wang;citation_author=Y. Zhang;citation_author=L. Du;citation_title=Comparison of single-trait and multiple-trait genomic prediction models.;citation_pages=30;citation_volume=15;citation_year=2014;citation_pmid=24593261;citation_doi=10.1186/1471-2156-15-30" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=L. Gutiérrez;citation_author=A. Cuesta-Marcos;citation_author=A. J. Castro;citation_author=J. von Zitzewitz;citation_author=M. Schmitt;citation_title=Association mapping of malting quality quantitative trait loci in winter barley: positive signals from small germplasm arrays.;citation_pages=256-272;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.07.0020" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=L. Gutiérrez;citation_author=S. Germán;citation_author=S. Pereyra;citation_author=P. M. Hayes;citation_author=C. A. Pérez;citation_title=Multi-environment multi-QTL association mapping identifies disease resistance QTL in barley germplasm from Latin America.;citation_pages=501-516;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-014-2448-y" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=F. Han;citation_author=I. Romagosa;citation_author=S. E. Ullrich;citation_author=B. L. Jones;citation_author=P. M. Hayes;citation_title=MOLECULAR MARKER-ASSISTED SELECTION FOR MALTING QUALITY TRAITS IN BARLEY;citation_pages=427-437;citation_volume=3;citation_year=1997;citation_doi=10.1023/A:1009608312385" />
<meta name="citation_reference" content="Harrell, F. E., 2019 Package “Hmisc.” https://cran.r-project.org/web/packages/Hmisc/Hmisc.pdf. (accessed February 28, 2019)." />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. J. Hayes;citation_author=J. Panozzo;citation_author=C. K. Walker;citation_author=A. L. Choy;citation_author=S. Kant;citation_title=Accelerating wheat breeding for end-use quality with multi-trait genomic predictions incorporating near infrared and nuclear magnetic resonance-derived phenotypes.;citation_pages=2505-2519;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2972-7" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=N. Heslot;citation_author=D. Akdemir;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.;citation_pages=463-480;citation_volume=127;citation_year=2014;citation_pmid=24264761;citation_doi=10.1007/s00122-013-2231-5" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Perspectives for genomic selection applications and research in plants.;citation_pages=1-12;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.03.0249" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. Igartua;citation_author=M. Edney;citation_author=B. G. Rossnagel;citation_author=D. Spaner;citation_author=W. G. Legge;citation_title=Marker-Based Selection of QTL Affecting Grain and Malt Quality in Two-Row Barley;citation_pages=1426-1433;citation_volume=40;citation_year=2000;citation_issue=5;citation_doi=10.2135/cropsci2000.4051426x" />
<meta name="citation_reference" content="citation_author=Y. Jia;citation_author=J.-L. Jannink;citation_year=2012;citation_pmid=23086217;citation_doi=10.1534/genetics.112.144246" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Jiang;citation_author=Q. Zhang;citation_author=L. Ma;citation_author=J. Li;citation_author=Z. Wang;citation_title=Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.;citation_pages=29-36;citation_volume=115;citation_year=2015;citation_pmid=25873147;citation_doi=10.1038/hdy.2015.9" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Juliana;citation_author=R. P. Singh;citation_author=J. Poland;citation_author=S. Mondal;citation_author=J. Crossa;citation_title=Prospects and challenges of applied genomic selection—a new paradigm in breeding for grain yield in bread wheat.;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2018.03.0017" />
<meta name="citation_reference" content="Kassambara, A., and F. Mundt, 2017 factoextra: Extract and Visualize the Results of Multivariate Data Analyses. https://cran.r-project.org/web/packages/factoextra/factoextra.pdf (accessed February 28, 2019)." />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=B. Lado;citation_author=P. G. Barrios;citation_author=M. Quincke;citation_author=P. Silva;citation_author=L. Gutiérrez;citation_title=Modeling genotype × environment interaction for genomic selection with unbalanced data from a wheat breeding program.;citation_pages=2165-2179;citation_volume=56;citation_year=2016;citation_doi=10.2135/cropsci2015.04.0207" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. Lado;citation_author=D. Vázquez;citation_author=M. Quincke;citation_author=P. Silva;citation_author=I. Aguilar;citation_title=Resource allocation optimization with multi-trait genomic prediction for bread wheat (Triticum aestivum L.) baking quality.;citation_pages=2719-2731;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-018-3186-3" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=A. J. Lorenz;citation_author=K. P. Smith;citation_author=J.-L. Jannink;citation_title=Potential and optimization of genomic selection for fusarium head blight resistance in six-row barley.;citation_pages=1609-1621;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.09.0503" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomics;citation_author=D. N. Lozada;citation_author=A. H. Carter;citation_title=Accuracy of single and multi-trait genomic prediction models for grain yield in US pacific northwest winter wheat. Crop Breeding;citation_pages=190012e;citation_volume=1;citation_year=2019;citation_doi=10.20900/cbgg20190012" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=D. N. Lozada;citation_author=R. E. Mason;citation_author=J. M. Sarinelli;citation_author=G. Brown-Guedira;citation_title=Accuracy of genomic selection for grain yield and agronomic traits in soft red winter wheat.;citation_pages=82;citation_volume=20;citation_year=2019;citation_doi=10.1186/s12863-019-0785-1" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. Michel;citation_author=C. Kummer;citation_author=M. Gallee;citation_author=J. Hellinger;citation_author=C. Ametz;citation_title=Improving the baking quality of bread wheat by genomic selection in early generations.;citation_pages=477-493;citation_volume=131;citation_year=2018;citation_doi=10.1007/s00122-017-2998-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. H. Nielsen;citation_author=A. Jahoor;citation_author=J. D. Jensen;citation_author=J. Orabi;citation_author=F. Cericola;citation_title=Genomic prediction of seed quality traits using advanced barley breeding lines.;citation_pages=e0164494;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0164494" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. R. Resende;citation_author=P. Muñoz;citation_author=M. D. V. Resende;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=Accuracy of genomic selection methods in a standard data set of loblolly pine (Pinus taeda L.).;citation_volume=190;citation_year=2012;citation_pmid=22271763;citation_doi=10.1534/genetics.111.137026" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=J. Rode;citation_author=J. Ahlemeyer;citation_author=W. Friedt;citation_author=F. Ordon;citation_title=Identification of marker-trait associations in the German winter barley breeding gene pool (Hordeum vulgare L.).;citation_pages=831-843;citation_volume=30;citation_year=2012;citation_doi=10.1007/s11032-011-9667-6" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Rutkoski;citation_author=J. Benson;citation_author=Y. Jia;citation_author=G. Brown-Guedira;citation_author=J. L. Jannink;citation_title=Evaluation of genomic prediction methods for fusarium head blight resistance in wheat.;citation_pages=51-61;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.02.0001" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome,;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=S. Bhavani;citation_author=J. Poland;citation_title=Genetic gain from phenotypic and genomic selection for quantitative resistance to stem rust of wheat.;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.10.0074" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. A. Saghai-Maroof;citation_author=K. M. Soliman;citation_author=R. A. Jorgensen;citation_author=R. W. Allard;citation_title=Ribosomal DNA spacer-length polymorphisms in barley: mendelian inheritance, chromosomal location, and population dynamics.;citation_pages=8014-8018;citation_volume=81;citation_year=1984;citation_issue=24;citation_pmid=6096873;citation_doi=10.1073/pnas.81.24.8014" />
<meta name="citation_reference" content="SAS Institute, 2018. 2018 SAS 9.4 Product Documentation. https://support.sas.com/documentation/94/ (accessed February 28, 2019)." />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Schmidt;citation_author=S. Kollers;citation_author=A. Maasberg-Prelle;citation_author=J. Großer;citation_author=B. Schinkel;citation_title=Prediction of malting quality traits in barley based on genome-wide marker data to assess the potential of genomic selection.;citation_pages=203-213;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-015-2639-1" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=D. A. Schmierer;citation_author=N. Kandemir;citation_author=D. A. Kudrna;citation_author=B. L. Jones;citation_author=S. E. Ullrich;citation_title=Molecular marker-assisted selection for enhanced yield in malting barley.;citation_pages=463-473;citation_volume=14;citation_year=2005;citation_doi=10.1007/s11032-005-0903-9" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. W. Schulthess;citation_author=Y. Wang;citation_author=T. Miedaner;citation_author=P. Wilde;citation_author=J. C. Reif;citation_title=Multiple-trait- and selection indices-genomic predictions for grain yield and protein content in rye for feeding purposes.;citation_pages=273-287;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-015-2626-6" />
<meta name="citation_reference" content="citation_journal_title=PLANT GENOME;citation_author=J. Sun;citation_author=J. E. Rutkoski;citation_author=J. A. Poland;citation_author=J. Crossa;citation_author=J.-L. Jannink;citation_pages=1;citation_volume=18;citation_year=2017;citation_doi=10.3835/plantgenome2016.11.0111" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. Thorwarth;citation_author=J. Ahlemeyer;citation_author=A.-M. Bochard;citation_author=K. Krumnacker;citation_author=H. Blümel;citation_title=Genomic prediction ability for yield-related traits in German winter barley elite material.;citation_pages=1669-1683;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2917-1" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=B. P. Ward;citation_author=G. Brown-Guedira;citation_author=P. Tyagi;citation_author=F. L. Kolb;citation_author=D. A. Van Sanford;citation_title=Multienvironment and multitrait genomic selection models in unbalanced early-generation wheat yield trials.;citation_pages=491-507;citation_volume=59;citation_year=2019;citation_doi=10.2135/cropsci2018.03.0189" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=X. Wang;citation_author=L. Li;citation_author=Z. Yang;citation_author=X. Zheng;citation_author=S. Yu;citation_title=Predicting rice hybrid performance using univariate and multivariate GBLUP models based on North Carolina mating design II.;citation_pages=302-310;citation_volume=118;citation_year=2017;citation_doi=10.1038/hdy.2016.87" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=Y. Xu;citation_author=P. Li;citation_author=C. Zou;citation_author=Y. Lu;citation_author=C. Xie;citation_title=Enhancing genetic gain in the era of molecular breeding.;citation_pages=2641-2666;citation_volume=68;citation_year=2017;citation_doi=10.1093/jxb/erx135" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Zhang;citation_author=Q. Song;citation_author=P. B. Cregan;citation_author=G.-L. Jiang;citation_title=Genome-wide association study, genomic prediction and marker-assisted selection for seed weight in soybean (Glycine max).;citation_pages=117-130;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-015-2614-x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (Hordeum vulgare L.)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="twitter:description" content="Plant breeders regularly evaluate multiple traits across multiple environments, which opens an avenue for using multiple traits in genomic prediction models. We assessed the potential of multi-trait (MT) genomic prediction model through evaluating several strategies of incorporating multiple traits (eight agronomic and malting quality traits) into the prediction models with two cross-validation schemes (CV1, predicting new lines with genotypic information only and CV2, predicting partially phenotyped lines using both genotypic and phenotypic information from correlated traits) in barley. The predictive ability was similar for single (ST-CV1) and multi-trait (MT-CV1) models to predict new lines. However, the predictive ability for agronomic traits was considerably increased when partially phenotyped lines (MT-CV2) were used. The predictive ability for grain yield using the MT-CV2 model with other agronomic traits resulted in 57% and 61% higher predictive ability than ST-CV1 and MT-CV1 models, respectively. Therefore, complex traits such as grain yield are better predicted when correlated traits are used. Similarly, a considerable increase in the predictive ability of malting quality traits was observed when correlated traits were used. The predictive ability for grain protein content using the MT-CV2 model with both agronomic and malting traits resulted in a 76% higher predictive ability than ST-CV1 and MT-CV1 models. Additionally, the higher predictive ability for new environments was obtained for all traits using the MT-CV2 model compared to the MT-CV1 model. This study showed the potential of improving the genomic prediction of complex traits by incorporating the information from multiple traits (cost-friendly and easy to measure traits) collected throughout breeding programs which could assist in speeding up breeding cycles." />
<meta name="og-title" property="og:title" content="Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (Hordeum vulgare L.)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/3/1113" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Plant breeders regularly evaluate multiple traits across multiple environments, which opens an avenue for using multiple traits in genomic prediction models. We assessed the potential of multi-trait (MT) genomic prediction model through evaluating several strategies of incorporating multiple traits (eight agronomic and malting quality traits) into the prediction models with two cross-validation schemes (CV1, predicting new lines with genotypic information only and CV2, predicting partially phenotyped lines using both genotypic and phenotypic information from correlated traits) in barley. The predictive ability was similar for single (ST-CV1) and multi-trait (MT-CV1) models to predict new lines. However, the predictive ability for agronomic traits was considerably increased when partially phenotyped lines (MT-CV2) were used. The predictive ability for grain yield using the MT-CV2 model with other agronomic traits resulted in 57% and 61% higher predictive ability than ST-CV1 and MT-CV1 models, respectively. Therefore, complex traits such as grain yield are better predicted when correlated traits are used. Similarly, a considerable increase in the predictive ability of malting quality traits was observed when correlated traits were used. The predictive ability for grain protein content using the MT-CV2 model with both agronomic and malting traits resulted in a 76% higher predictive ability than ST-CV1 and MT-CV1 models. Additionally, the higher predictive ability for new environments was obtained for all traits using the MT-CV2 model compared to the MT-CV1 model. This study showed the potential of improving the genomic prediction of complex traits by incorporating the information from multiple traits (cost-friendly and easy to measure traits) collected throughout breeding programs which could assist in speeding up breeding cycles." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
    <title>Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (Hordeum vulgare L.) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__L27CoAClcEsxY9a29Y0lWctM-B9DY7Wom4B41-wuQsk__hwC62gz5iT0HOH5FHQ1euI6uc2_KNqFWQY8gJMTjS4M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"sgsL0v7bfDdGktvaZ1NXCbXthXtggv6xX_yIwtirz3E","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495864","apath":"\/ggg\/10\/3\/1113.atom","pisa":"ggg;10\/3\/1113","ac":{"\/ggg\/10\/3\/1113.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/3\/1113.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/3\/1113"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/3\/1113"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/3\/1113?rss=1":true,"\/content\/10\/3\/1113":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022K4b\\\\\\\/m9U4\\\\\\\/kaKVpmV5kwS62N92sHLag\\\\\\\/KZdLz8VmR1lCxFKzKJo1dt0tYrxRRLp9KQayMKc0IxRDOHfOgoIoSHvMQ5DU76CPIb2zlIQfUcQoMdHp8j82ieEJQyKGpvYW0vE9LSA39YB4AiV+osTbShsor8Ncb735HvI6jZ2eCuJP+3\\\\\\\/sIduOzhCYmDTlUimHKUSmpi\\\\\\\/wZdw2eMrI\\\\\\\/TCveCocrUU8Mx7XBg8CDliLib4rbhafi29Qlj61uqpNzPFMKjJdOvj3X0z8xoPoVHAlMNom5l1ZCVBCsRA9vAC6X8xpWzKuvAhZkEPxa3RorEu\\\\\\\/AJgUIdeWAxQoSTCi04QJJRktWu13Pcuat\\\\\\\/lsPyDPe0sEcK0H7um2xVOkIhe0C3qq6EpB7noNVQys1BOwGaONfEFGu6qWm3R+TgBixwaige4t4NGnKgxs22fALUkBlohQyB0mhDP2CVY3nU8dCRjDH91WT1gDUm5bwnOLnT3NG4I4P5RSbj3avh81GZZPbZ\\\\\\\/5\\\\\\\/bVea86IxvWN7A+0g+EgiaRr3SjAscDIFD3olOK3N70JlXJ3UYy143b2BbELxy01pZWJF1ueuEyPSg+24pV9bsy9OYsirhO6JUu2b4E3wTwmUP4mMpj4\\\\\\\/KGhfrcexp3PD+ZtarMMLk1WQ45rTsAXlQCCHAVwcTFgNpEWPhCbu7qc6nsgmgbC2if8ysS4Mi9FZy0up7LUi3fBxnFV8ZXkgn56wBxMNM33R82VJ1e8qbj2ueJ\\\\\\\/1C45T9dlykhMf5WRQ7LF7jyr6Ez+I8yjrD8wfgxILsp2De57MPDjJYjhg4WQhr1Bg8kYFa9rwcDnw6BG4J4XU2ok8Jqst5QUzjI\\\\\\\/JnNCSLOCiepqD0AS76BnZWMA5nkkKkVaFlbnoWDNClgXFbo0vNzYI6o9VL4miQhlMtvxphvB0NGP1XInK7sSmJKAvvNSAV0sUUhlrIW5KPYpMnSEs8VAyTPENxdJPHHGonVjzQBXcePX\\\\\\\/Hsm6SdGA1KPpQBIpOJWfPghNJz7d6vc0wmI5saeZ6BDXBx7sDWJNUcBV917+P+ork8n9K+wm1P1oq9856xCHMwoVyqKFnh5vXM10UOUNpn3rLNTL2ejpixLrj1bBJTqwx77CXeHy\\\\\\\/5kIGo3YoQRLHlO\\\\\\\/dLPDCrGPRreVKYkNpwXtv+jgIEkDul4V4k3whcuK6Us6OkPy2W3xjlX0c9BJqGOFeG0TqYdZdLkJcuydlU0ZSABNrKU67YOyyyxvq2tTU6aKn3VQ3mGXP3tTJ8bfU+JCXtl8Je4J\\\u0022,\\\u0022iv\\\u0022:\\\u0022VH4ebm7SnuI2U\\\\\\\/gfMkBB3g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228c5d321c7606c8d0252dac40196c20743d9c43b1181a815c56cf93ad13741960\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Kch6rFrjbOAXCdkyfF98AMVm+3ATy97M0WPABMSN2uzKnQsQIdHATRZIE6MKscuZ64KupTJUaPNICMTrjzMk4mqQ+RzYsVuK1xEoHtklXeXAfsMrvJUVdyX4os\\\\\\\/NqOM7KAlJPLgq2TOGDryC6QQOTevcQxg3Uk3+fjW00QK4FNgYkPV6ZCtokCW6gw61KPwWqN0bP6aJwTnxWdyOssI5JEgzzFv9CCdecfWLw1lpYBGkWBo9VRU9tAyyB0XLr5GaHmObE5gvhSId7s39dVocNwYy8nDuZuLdkmBz8ev6zW3ZjJiDGHrf\\\\\\\/35eUDtALIEwVyBPlIse98I32Soa95nn7Qf7rMzld1vQyLd6pmRP2yy5qCLsUGQnMROR7YN3Wq8yRPzOypEgdmLuhmeB5dKJZgQ1S9XIMxBsE+L5RrerSzX+5Cpd0DfmHhSnZNcxZZ+q6bKWmsRwL4aAXfPYE7RxvpBTVG\\\\\\\/9iWQQxJbNjeRt3VV8eJ0Gs0OdSvj\\\\\\\/xYPHeBcS\\\\\\\/LBM7yYRZFImvS01EZDkFawZyQu3dgmeLwHUaPqv+5Chp46PPbxnisixaaa2Z\\\\\\\/TJxma0RetDbJzt\\\\\\\/Tm\\\\\\\/WE3rJkEEpX3mP6yME2gR152n75pNo4lh\\\\\\\/n1WOUpcP\\\\\\\/pm23gxchKJAHWXmbauSsPRp1icVCXkw6KaPQTQmGUPUCx+I6JVtlxUV\\\\\\\/LInTMSSYw2cxS7QKUD3Y7VlwUdeiNaaB0Ok2iG0e5e6KatYjVHSZsq\\\\\\\/9KoNfAg5WopAzy5Ih\\\\\\\/qOBDa9CNhjQfBEJBLclldPVqSv6XG0ueE1ymLZvJnAfgZv9lpSrrAi3fuZO8jyI6aRJ947DWgLtyJlGYfkkExbK1HWSif6HJepuZJV4DguaahYZE\\\\\\\/u9e2GHn4mmmkxbWgTSDteJkVdi523v1FUZGQOP84bYzH514XXQXJgwjtiEWH9BDDzOko\\\\\\\/wsEwa6t4r4Ai5o5Lpi1393fdO4hENznybSfrhWc5Aq793ZDFBXJ3rw1R9qRmGzBOjDHAuKQk036fz\\\\\\\/WG9meFWhI2u4jvnLZkojIcEChbbfm9tKGpJPLcC0ynDI8eqF3niz2ThAVWkfFxjrERrtsHTUdOnIpIm7xyaBlqXtAIKIWkmNe70xKeJ7hbW7+jtt3n8SODvnhonpY1Lmr+ae3csG594BAe7nTdui\\\\\\\/YCkSRS4v\\\\\\\/4an6lJn\\\\\\\/oeE5KhhPUKIwCDIfP+RHhm7i7NMvjoWO\\\\\\\/lGkok7m5YJQhyaTUbkRhm+Di9h+OxT\\\\\\\/mC0NE6mHpk6\\\\\\\/uQjskZSq4Ac6bSVKJH3Ul9pxmzZuns\\\\\\\/92XbKRe6hlCZypBaU9MRVd6AHeZsezfh6hP3zHcjZGfSku2V7GmwHeXrG3Yy1HE+EdzRguXFpRdEToAvpd77TxO+cL3OR0MWxDLWOoB3KaqqZhQhvJ01BHuvUEw\\\\\\\/aFGFNlo+ezqP5hVwWBcLEG1FZP2uA2tV44BGrCox8hqCLjvjMtkUUDji349NQqh95c\\\\\\\/aSTOGdhPpiYSdvVD7z1O+BnvAKp4+6Joh\\\\\\\/7+F5XRgOwGs7v0FwczFrl4raJBuxE\\\\\\\/JIGKM8yCISbw4xreMEh0P6cpVL7rpH5QO7Dopko5bQ8gfZLCH6Em5CpnrwEDz7qWfqMv\\\\\\\/LdsVmj7DoyDAiX3UO2v14P\\\\\\\/v3Bf2BmwgcWfn61U63anpaKP5oYBf0FjZEFiYZJPoOb2UgRkYUGBd1s2EfAf9JLCoSMKL96TGnBHNwmuIKQiz0FmR21ozSRS4+UlMfpqOPend8X9mpFkPnJnKUkv\\\\\\\/VxC68XZMQkR+kQEhwVgu+\\\\\\\/fNM6OTsnwkScdrEDkCYnPrt6JZ\\\\\\\/g9aPWBqY9StN1+ubHnodvsfL6msBHzMWnWfzsNFbBXGBYWWvwh12lJ\\\\\\\/NGnd6HJFu+sPfhtQCLt58+g2QH2Ot+KzgDVI4vemlFFTt0QRk4wB8nsHTvgWOddDlGTuFf8FkQBjtOnQKkIw1MLxwGIk9boUW2JjPzzI7qKEZMfdoM\\\\\\\/DB+IS5Cgu22KkrMdMKwB+8oRYV1FQt1i4mm7Z0IYRxULEWNJ6Z\\\\\\\/IApRgrdlgTIMJSNiLRU5unNeYa\\\\\\\/SKRpPhhAx6no8UFaOlCyju6tQGbrDFB2oSuKmIwBbGnFmNmK+tCxzFAa1Xp42lKeUGK4W0pv1PRMvvP3xtImVAOQ3diZE2j8Xn9gr\\\\\\\/aTrU0WMWg3UlmnGdzQkpfbATQPkd8Fefw5nOIH6nIFs3UBjkrb57mpW5uVQLwW6c7DHUACwNRxwDlVDXzM5G0yk+RxEaTTS27V7P4OnT+tiLdRO8ZTqrRel0JF+c\\\\\\\/w6xRW2CUDaCljzwjP9zccCaqDZOMQV1msYwrFTRujJuDfXyNcuScqchboKwnv1pQ1VXC2hCUDPad7wQoTbcGBOtk0neH2W4uoLEErRz2dtnW7e6\\\\\\\/mp90ZnAvKYm2ErJ8AfXniUpwCGeG5f\\\\\\\/a7sTxQRTQFJufd5eRhb31eDIW5KET7w0jesUCgywhWUBV8VhqcWlbUvMPSI06cTkvCQgnfPHX0kIGHYTUdcFgh52lADWAFXEr0g0WmQl7Ci3DhbZ0NbT7+RhBu55HbPfwdP0AUPQ8Qi\\\\\\\/L+KpWye7XPPTVkNcCQBKhYXG3c5EVVi+XnhNwBBX5F+XdPPa2mwDLRkvTckaYI88XXOM+QoWJE0Gl1iaweOHiYGAtoihUJk4vcfvztQmtqLTvshKv\\\\\\\/FIVz3\\\\\\\/nZN6S68odbbrVRXK0DTB9xgW2R44bX10Owh5twZ4zo9OCEHXXF7mio3MCO9Yyc1ythUFA0oV\\\\\\\/sBNnSPe9E47z6hijL8GChrXTDbQ2U1wwkSLCdG8HW+BBp+ZoWXAdCg4Wa2idRqnLlkKKKu8K+tclIca8T+zuT3sHjWOSDtojorRBFktP2ocOvFozzTAxW3OSBH6+OGok+\\\\\\\/dUzRueUGbgLLF5QIJsocAO2KtMeDQYwDloWkqosRKESsS3wmf8bXoo2oJm\\\\\\\/Ny1RAKb6ZIuRVv7Qu7g7QM0vveIu9pXXoJdjByPhLU8T6NQ9+qhIH9cMxewsrXxNwcS6R\\\\\\\/N\\\\\\\/B5MoRwfEhHzyNd7M3SN+vtAS0nxHN02QmiV9JEOcOn4cF47syj3qc5pS2hu6keiDJVF\\\\\\\/tq8fkVvnkesGz4hXe8UUFlGV\\\\\\\/OGHfi5Yw+cHgal4nI67E\\\\\\\/nBOUXi4yVFS77dsstRiINsoHYWzj\\\\\\\/MKhxOgLue5exAel6hByE7m0ezkwfreaDu4kXVr6gjrXTABr6stej92BGZpp\\\\\\\/KxljqDIP9ioi662LXJorm3CNleaFl+wEtWF6lm+GK55Us8uDoaZ4U7maqqfiq\\\\\\\/6XTq2KZr+\\\\\\\/gs8nKlGiKp3oArMkTnNf9RkVqHLBfMqTCTDMTtUFU1YXP8wswGNqhz7B70sUqMtC0uiWK1nVY9TPGS4lOH5qlaccKvvqVduBeGLh7yGcSXNPyRw9vAPtlq+YfiavLVkmr0cCkeC3g04aHjYbZ0k5l\\\\\\\/jSIWToGBJ0qYlaAYhN+kSCdFEX1k5z4h6KZ8h7XCXBO4dIfpT4zJnVoDf21CovDlY5vFNqsP\\\\\\\/RxStl9bRFO2mcLF5ZpKdOwys6jQFkPcRGhGBUWVmAgwdF+bIGrQRoqb4hE9hJLg6Q7oP7HJYAH2I6ZXRjWvMrPYm2NsfAhn42cqHvOMvvIPNN0WA7nVTIuTNPQgqmLRcuQXmdRyV334EXUfSgraBHYKlLtilvafyhbR9DiUL6eWDBC4rWMteh20odBASEIlENJfgfcpGWfOT3vM1GB7O\\\\\\\/QRO\\\\\\\/SUdWngCOkpDFmnazTppXqwhNpJvcxSbfU71ZGLeDQufIaGRqqqFOMmAdmIDJ85TekhpLjjzBmD+pc\\\\\\\/zkCBKUoZugJaHFJrao1T4jmMclUdS8cQ97DXsliq3z1jTCnroQPD1tm8wMLLXkmRU5kyE\\\\\\\/vwTuFm8N0Ik3wnCJkiCgQ1\\\\\\\/Xmgzs4+dA4wNUoeyOUHdKRi8foiQdRhreVscHhjAF0sGHns1ENCjYOR2Go8ARLsZtM5R+eqSRBU09Yva93G2iKfv1rrPwNWc6u8vqwdFh2x1FLGs9g7l6XtGoU+BChzBelT4foVV7iFPkvJE+mtIbDj1jqf+LJMbq3u80kZ6N9mTRADMiTYdyu90Jw4n+zlWw1NZYPzuKYhRykEyeCjHx5dP6NaisI0NgBdg2SFg5cNnfZtaYa99W8JJBO60B05JNBRctpFphUxuLfnJl9ZIyEZLvModiPCLKtg\\\\\\\/l0wCUNyexLcHfkuuCH\\\\\\\/pf4tAaA0uGxMMIGr\\\\\\\/XOasuLRw4MBLmsC0nARafACMdMTPzle69KKPfPUj5of3P6PZmPIUvfFtW8qYuUzT+fttbkGL7fawi3G5ottly4AXDV2DSi8XqbPiS9oywfEEsZc9ryTJajeznxXf8cIjciY+pqt1oaoxvWxDxGPLPK1CRCndKSJdJ2Bu0szupQlbyJlhknFbOVqz6sR23NPpj3xonssET1kXIXcULfBDDVJcT3ONbFl+Cv2spCbUZuZyX3XM\\\\\\\/3cyDFbpT15ae2DC4iNqmOGYfnRNKxg9DMPT\\\\\\\/W+vBzjPfzczMG3hqYV6FIohApjsrVqmjYbz6fGeP+hgU34+Y8twt9Zm04+2ArM88yzTVOZZCl7Auet2h0cwk3mowpjlM7k5wr7Y3qB+ZNG5YqScj5p+gAj1typyFJpH440+PQFz9Wj1XyWg3RZInUPc+kJR\\\\\\\/R7qlz51u1zlNwnqOkYixEvmcKFsDxHREkzh3n5ZE3VlDPDkwBHgblN21BWTZ\\\\\\\/HDNk9EyI+e6xdxhFSG7RPEAGNdFjn58DDmC6vuSe16ZpGms+ZKjm2QsnPHAJ4K3uvOhDcbe1KPf96ViE3R8FDWSUS9x3lJvxb7VZsSxKl1jGlmadE9G8tNI4m3\\\\\\\/Em1l7iS+j7EPlkJBmNf1M0u3Y0+ABstZs0RWvkCW7O\\\\\\\/2wWQYiw8xn927r8pgvyYXH+25SMraWSuqAl52BoKQcaUGh3JkPehaaDZyFzZWAANIW0uhel0Gy8l6zeWtbG+ImV6IG\\\\\\\/b1OPgKp+JGiBLeKkhTyCotsM8JOa\\\\\\\/+h4xiDv\\\\\\\/fZTjyIZ877Vqtng32APTJt6eDIdWi3eRudKJtcQfAicI05ArYyjaTWBkSdlitlVBnK9kf4ZakRT7VOlHRJPr0keBDE8hYyIFx7LiqzoOMlN4iRbfEQRr0Or5KHAEp72bS3NobUuBAyKJWnPYt156JVObjNCtrF1Ce2Uhx\\\\\\\/OwcYr\\\\\\\/VRxLNgOrJW35xjBn0NZP8GZWA\\\\\\\/6Q7mAS9M4QZl9PQO\\\\\\\/4BzswcH+pgel6uXxsNUpV56nVUqY3cwYNnBFU9G+v6dTbqsKsj\\\\\\\/4UD65E7ztvAPa\\\\\\\/yV0xoviTaENyR4J7JOvyDu8Z+P\\\\\\\/MvGODFhjTL++z5yGvg3NcLH\\\\\\\/on0WHJYzGb1kxQVv6Qp4gD4VyuwlCJ4bWyRDbmjaSV0P8EnxIPbquSWChQ8Mz8qKBuD0NPppGPnuhh8kzxpsuiHc60o8rGsQpblHV7bXZ8suYNsbGty3AEObgqFyfIHGwj3o4tbGlbM4Rlu2L3KCYXHk5RjhzAU9S4fWuAQZTEFlBE2dELE1fjpzqXuw3t6i6QVnUkqgXyFFk+z5bgv\\\\\\\/OmLXCB5eRW6VFzws9Vsk1rpWKK15evjUlA8eChH01xxTPs1GFXBLD9YSGVi4fLQMmIPbdn2v7Oq9GTKeOsQim+AzyOQ016Rk1iCFCUj7tIymk69jLuRhAc+hiz8hmZbOBvVpUjI\\\\\\\/dFofwHGPiAXITLq3h9OgurJmTSRo+599MldSl3FPig38dZI3flEgxRdJSEBBfuaEZdPqDQqV\\\\\\\/48QUwAL21HttGKo9MwBQdkdYeDrR47O\\\\\\\/6BFc9eo93cLYVD9ghzHlzBoqMv1+Q1KZH7tTFXA5i9iNgARVQgIrJuIRzMGO3xwu9ufFDOCrbmG0\\\\\\\/kY7bp2RFHbEn8\\\\\\\/glBZWhBxI7q+lfACGsNMavQA+1lRLUS4kCjy3YUB2vKohEFaMvrJ4cefcdkYYt3anluAB6OmfvnyYhfgWriJwzK6y9cVmYi0MByOG5Ld2GETFT1rAeK6C4vmLvs0n0mTRhtUtnkUWo8qeQoPuU+4Sd6FIdM6J1SQLaPML89H7WutEwhsUcJ4aIhu5ujVYRNcr5SDJnIMdKMnPNz2MxKrwPkH8RHIq4AJ6FC+2e5gLIvj8CTgv1SU4FdaGfsZIGK2T6t5gOa+uvc39gDcpVEaxWBHPgXqzqNqWUwYBh98XLIPKS1yp30HrcLiqlM20iJHrfbP\\\\\\\/QybnHASlgitaVJKZ9nnsIplHPYLInEXjSqFC1P2DZMqmcDywR3sg4\\\\\\\/aCPtrzD5lCBgANqEr9tr+JnSnRsSd8jmwgpjn705v8JvFDpVw3wcz\\\\\\\/f3aQKfqt5\\\\\\\/f5WLB7gNVWsBUkETYkrnSW4S2Cs5n6CxVaiUw6JnaiFcLNI\\\\\\\/ldkqSLSMQ1DYupd\\\\\\\/fFIYp1bNzBCScOzUKscVBXcrhDnQJfJiv3NsUe8dEtJUdkNdmqNrWldjhiHspJDw6SfTv3AUFjdDjECc5dDUz7jdWqcKwfXqPpxXVo7t64Vq7ibHyT71m8ttLjXAGfp0mofK4wcWY+UHjxNe49pVdk5F9NvUrxoG1JGKn+h1EKLnvn6OpSwGVuujhGizZn6pwC3EoUyYBkeFxQzXU16syGIP4uGk6L8wTuhxT61fQJeRkSwltqovl1gS0eMt8hNIZ2MP30D8RNxZE0YjmZK\\\\\\\/fsdsNRFtYXFUUbpDHUkKcABZUXY0n1q6eXfwnFfwwrZ7sBostR5lUUGr6dLQZCozZu1eMaePIY45nBdhp5SevvMuB2Gt7O\\\\\\\/XTrs6soj99ODh1lkbt9YX5ADlrjk7+nRKYH9TVuhDFUYFBWJ3oA\\\\\\\/K6ITKgm7vANyLN\\\\\\\/1lvXLJYiYoDRqhzrZth23yPxTEvEbQpmPxW2MkCsVv9TB7F0dElEUMBI0wKsitJrVtTf6YEY5Uz+fcwcXBFAjpkSlmmTnLC\\\\\\\/1C2xsIjfpipsR8NgFjdvKob+znzYjnfzMn1QaD9aVRYVByl7nGAbTjGRRELxB\\\\\\\/9y8ovYF9wQwC4mM25hfeabWvXsWHxfvkl4rt8\\\\\\\/ZPWFvDX5DT+\\\\\\\/0dgNPW14cMQ7kF7ieN3Q5PC0euMO1TqaN0ZpY1IdThR1frzIzKHbKqQMntswxqOcyjJx07d6dJx5CM8tXr2AhNUb\\\\\\\/BFot9FYwR+715gmWeBGC1C5Sxm97Njq6FfW2tF6TAEXud4s5UmoT9Va62EpWtXRainGzi0BTE0gDv5DUZ\\\\\\\/0TYVepaDReNveiCZslqgG4Ce\\\\\\\/CZvviWU5hPTfvuSnMROtkl0e1Q9vogFubABaMhsHki6sqmO1EE7iClzqxZXwp+Xa9\\\\\\\/3DoG8KUMS5WOSh1Uz5wIYFTH338F3vIV\\\\\\\/F59pxd1ZfIsea5Ro0AfnWDbpbDawQ2ZaLC\\\\\\\/KjMNt4l7RTzy0fQz9OXu\\\\\\\/Gce5HXvHvC5scX7hTIkLH\\\\\\\/5V\\\\\\\/FV4plIolaiRvGADtQgd0k6ToetTjq23Z\\\\\\\/qI\\\\\\\/+o3Ti4sfX1izwMDC9IUumNzF8wBZHDrAsxYgOXziDC+3EgMemPwJh9XIgKB1GTtOrAbPmRkdqrzUPQO1WGVm+\\\\\\\/j7xRJ\\\\\\\/B0VvXu0hMfrmu7KF8m86S+cjR7UbCMfiFobaeLvT1YQ6sgZu7LeLuNtVNt2keYs+F1hpvuDVrCIwG2MlbdK9iXTI2tc1SPAt2uB7cFyWEfLz+Eqj+D5AFBnbpPOmtim7sXXYp7mQVEZHGzHoGuHO3EYEilC2J45e8pfynZg8wYFOqlW+9eRAvLUy4EACg7wQtCR+CIMbBoLGmwZF8AXJZdd3aPl1E+8GpPx4hPbYVTTep+aKIi9KY\\\\\\\/OqxCHJq2CJ4RopPqgxYZ+ngs55133Knb+CxS\\\\\\\/93Vu+xjSObWM+x+ur3UiJAnqTz+AXzae9EiCPMnFNVQEzul9GpmgmVrdxxvQrrLDc8I2oE9L\\\\\\\/4CrZ\\\\\\\/RDWyj2MsjtwXlfmS89\\\\\\\/OLO2KPNE7x+KnE+PG673CenpgYZrjOmJfFYWaiRBkZvLJUYuUI+woDkmAV+2tqkRyCpwvuqTB2FWzIXfjUATXDz\\\\\\\/FHuMb39t9PmFypw0z3KrUPay6N11FjwN+w7QMV10woyY7cke86+NJAsrgkUhuITGjwKdBn9D3zs\\\\\\\/zwARt3hwHVHqI62hbIaGz6pl6tWvsrj\\\\\\\/EmS9fbSn4SOlMw9lLYO8\\\\\\\/RVF\\\\\\\/VqJh1dPKFroecESljiVaN09KXo49J\\\\\\\/DZZ2fA\\\\\\\/8sVhTnFDSAGn43Rhz5Cp2akH7FpQ92MG83YmlqJWcOku6K6qMhd\\\\\\\/nn82iXZEqEPchVVzMH1pmiwEVFGJunOqbvH2AVWDX1ET+1\\\\\\\/lolJSGZaEqJd\\\\\\\/WqT4t3RUlhLEno25zR3d\\\\\\\/2MfE\\\\\\\/yCHiB82FKM6BHpcpKkZf8GTqDCfHsC+ye8wIqDiKGyIzPfm1JBWD3h3rt1CXKY\\\\\\\/cGvbfM7RqvNcx43nuvhp3j+gQq8Nx4+Ad\\\\\\\/coY7kj5BrAhHrLJ7b4v4WStFg35Dwd7hm\\\\\\\/Da3AF1PHfOVe\\\\\\\/kth81TfPsXKOELxG43UGPTqQ0zxP6InrhWtIXCr\\\\\\\/kL924BysqX3F0hxhL7inAQKKAX03WbrB2jVmHDhV3w3AKmbxizFuq4SZ+ioWHZi9Y3uSMSMN1iaYY+47ZlCV15Kz9zsVi5ozVz0ePUNtFXXRscUel0VyMfsCQFPRa5cZevtBhJYboJyOGE7680hVadBY9fayhdZuT41I0JP7CYaTji02HwBx5Qnkm5HDW2nWt9D4n7clF3yu6YdwCfiqu8erYjVBNItzKWevOE9uI\\\\\\\/mspIazqpNKgygaxtS9B0AQ5Gj7+OOnuNqGXwq37bUfxPhhHewjRItTCA18jgAMiFBluOWejqkssIHtJR3o7+vqDe5XQHsA8NkoNW5+vQUNIf4+YyjmlUiym5cflF\\\\\\\/bweR0WH6qdLbRN\\\\\\\/o03UUr2BJ3YeANQeouQRgWN8+PA1b7s3ySSK1kdXjLBNGL4kwhjgE5tT2YOUzTLYx1ODHcMWp5UV\\\\\\\/lNrixlVqN0IvxSgJCFwbjVRyS0N1gUDmNdretBLdti89BKGN+HoR9HtoxKMA9ydR\\\\\\\/oP836BDzSBFaKOB5xaCysSJ+Hh5L641wmkH99sSZjaB4nIuU2YhpkoF+bpqGCG6z1jveTAGkdq7gKLPE\\\\\\\/D234Y9IGCRkz1dzbGU8SQKK2\\\\\\\/y60lYxQ1OGwat8RKI0bOcwlSOnC79pxGOGczAycctk473XhiDC2+teFlGczdsXFri45Ilnrj83W3cBIla8npS0ADRdtWTwm1CS2e3MQv1H6GDonhoc2RQWk955K45QPqICs2EvSNqqDdv2rVE72rkiN8W7JJN3b3pYQkEjg0fsEQw8gKZZHlAl29bm\\\\\\\/fky\\\\\\\/Bp\\\\\\\/tJphRbH\\\\\\\/++RRMmfLRCH7L7jK1XwFRtSElLsiUsz7i4uBVtTUWJA08Qgq7KNWfOnVTsSFtfp6TFPYt8WuvYjVmbt+QeBgWB5dziGJbRI6PUpSVCAM4gvIFycLJZEEpWkmNppCWIK6H+pUhjidi9f2XNjEcj3ztYH4IHTFsn6WCKtyDtBAIMGCU6Spo\\\\\\\/azbRPvyJBYXBdBqt7KM8sIwB1er1Wew+PNrAHKGHkdlKlMjjEr1LmjTjskPhSQsZ48MhHqwYudX+Tchig6PBRqdI6zOPR1liSiseYeUPN5wDmqmnZiDPAJxG2ItFBmiLhKrsysDcwIrNOVeQLtsl1\\\\\\\/XXlXLWqE2e6wpm5AsQa74YK2vkWrx0zhunFnnyVfjrDGAr0+ayV90b55+rb\\\\\\\/i2AHqM64W\\\\\\\/st\\\\\\\/ViCjERMK+2jLV6zHYXPrTdOWUlQvaSmheZ4CoVukUz\\\\\\\/ZNr2gJLB6mlnc5v5lo90X3HaXJLYif+01IAzrb\\\\\\\/bbFRCqwWLnUP\\\\\\\/u\\\\\\\/xuGx1dTxDSpzAS8TL2RSwQoQuM0L2JIxwI96NMSEOJy00RoLwJA63NO16aaKuhpftFutYWwsjCe9EekYK0oTwMYpHM3IxcT8cBddOTaMVB8TGxRO7EdWS2Vwg44W3dYAYoU+egqOoAFjfi84SOz6YhuJRGOB2CgqhFbYQQXtXuHOagAdtaTg9fZROVpz+V6Tkie\\\\\\\/9tAU0ix2fMXRa493NJ+ZXX\\\\\\\/BxDlHNcPQw5BJbvdahqisB6cgRJGM1eYFacNShGRL1Oowok1G09Cww81Z352M1fJMJVi\\\\\\\/cuMMcPPrGjgnn3UfN\\\\\\\/tNzBBK9ynax7unY5gwM1EAiBtJuetgGH0x+JI++tPCqk4m8kkBbcd0gN9j6CeGmQZsHr3gN8FrupCqurJwFYbdv2eP5in9KcLvc6tkoh3pQpD1b9T3pxwpxkj2O7cfkN0pM4DQlLrMvz10c7piKXkquISINhBCDbB+l+DQx8qs7hJVsxeDgD387gkZo4Ld27Ig23464p071ya3rTAJpk7Wxc\\\\\\\/ZcUPpydw3KD7xjrkbek6fEXdqGxKlJ6QRjdJzOVbMWIas9yZxJa706to7B2l2w6+u31jFuzHLthjvA7aw6ipywhwtNouRgXxbm0cWgTMlqQUZPAoU7HHfoUOOW1wuulMFwYHPAVZQTsXR7yDHyIcFptHPoZPXBDQYMvDJ0bDh+oto1UXQSVKuZmNt929L8xeXOPVOoVJLwrfaPIkCTuTunIdZ+HsHvcPGgFlTT0uAV+\\\\\\\/zC6WauepbcmEMh2XQqHhfaBTrrShnMnzv3F7W34eLdRBCfpyy\\\\\\\/OF7vwqsMdjxiL2cdfD8mAXSm27ZQoni4adao6ViRBt+5YIsee\\\\\\\/GJ5Mrk+\\\\\\\/+sBwfemqFmW4xhV476y148FX1pBT1eo0rNN7tQSWix2TEcuI7rr4Qd2FxoS8lVPQDyvRUfgSaeNwFZqf1b6VjVKjtAQTcVYFPCSqGXoZ5aqw5CAKGDTG5WkFaTaOMd9qLiw33O3HBm9ytTRpKMp4gmWOMONTV2AtbF3GVdD8v2eqBF4meuxY3iHS3XOO9Oi4MSYav55VcWKjiHQpbUdnCemQ5gdjigGPg2LJSxxoT\\\\\\\/KAlz1L9rABqlh9E\\\\\\\/e+KgeSgBm\\\\\\\/E7J\\\\\\\/Oxqz73C2Fs+q5fI2yeP2EZBu7xGV\\\\\\\/\\\\\\\/FiPQNPqJqZChE6nsOI0gDel836QQFlXtRKP5TALD\\\\\\\/E2s9uyh30yn02txL+yRcP1U3JBlTkMVE1CqF4vPIxn0yQm62JG1givlLqbgWZ7BR1BPCw8QwLftrds0bF31zlMVfNQqERN2ep\\\\\\\/Sk\\\\\\\/vdkCJ1nItOjvT2hVR1U3wVAL3cQIZCJ2cjg9WHChXgAQFCHrIGr9H1qfj0XY2U8t+9kS4sjkZK5b2TS\\\\\\\/LlrHbmNS1OCPTQiTZ1QmsvaNoT\\\\\\\/W+9iO+\\\\\\\/q+0d5XnR4EBhxuYe6+H53i6b+wL2On9PVtnG61XBT09NAOHlDkSO6c948arpqkFZfasuboodgT3yzEZzAkgPlEw2G5AFJNZqIfiI7XCWrPlaj916FXXnsGzmvRlUKMqNBewnvPKhWF4IT46PlUfSBfxoWocCfBeaqdIBKazdniKEhkl5rybFBd1hFKNv7RjiwbaX4LCTHmcGRRpEOhVrKckblWFVTaPJR3xCU4RDxySnKzcYFQ\\\\\\\/8Q\\\\\\\/Cm0RU1v0LLOO4BFMOCxABknueXxTCYpCqywSHxw+dvEmKyuFhpFE7MCpqjqMY3oBb+yQoM5ynG4PTPuGLB55XYZ3Hb7NBh3w9IdIhzEv2IiB1yUAZmLD\\\\\\\/wVBPt32IrhPDcBkzbIjwnttNlaMoW+uLlR6+iu5tClVJ+obav4kdRDKwI8Xoqt0KCnr7UPIlLuk22zdgyYPa2sNAfnqvi0Mp1mpXucgSHQkUbzxCgO2TzqyWZN6KMPvNpVJkQfaQaEwb2EnQtHaIX5sDzfrs2UOfFGnp+DYrJQOkOsr1izrNAB+HM5enBgyc6spa3b0Lw1DoMM4r0ljGmjCaAinWx9+ukI53iQzSPxLPA+2xsPA8pIcrsVRX+paEXKBnItN\\\\\\\/b6T8ICNXcEEsjNIB81xysjbK4fMiqyHS1boyO5mbe64wvXuec88wbIn4G+V5TwYa3i+o+jAkNm282iWxD+SyNUyFNupmjVrRgPLQQmgyxgel9ugdsVB3GojlDe2owPOsz+2Kr8LJe0vqVv25Y81YaeBAK1vtNYGF4dq5iie0KTyou\\\\\\\/YUkOu5WXDkT7ZS6umt6DOxZ\\\\\\\/QyEUeTYNiXLnljpO9NEt4xS0+gUyiWUQ4gMNLrtJk9aN1gxCrJjuXvdSYF7iWh3lJaoz3miqcDHvtGOruKfu+lUlxyfZs64JVx9g3PJpCLTSj1kY8UnUNbOKpoIOySXRsyFdtVe9dHT0dMAhH2OGbNEq\\\\\\\/Oq\\\\\\\/rVBxl1ZIjQiyIO+0Xa0\\\\\\\/56PyGAo71Fhna5nkEgXkAl1BZwSHjRbayVwjZUBRTpBxYDNOuxXHAJpAHQHUW\\\\\\\/dt1hHDi4lVHh4asgvavWkeHgmlkW0Ytkmpk4CIWUYqAFh8eUpPmljWe9GhDRBv2rT2XAbkRK6+ywzqvrbHFErpflGdIYbAa0y+lm299IhiHik7PMhf4O\\\\\\\/lckyBn\\\\\\\/QC6\\\\\\\/BKy6PJ7iXfp8t3BVs4QQUJOuqacymJWDo5CD7QS7vc3V5+PDZ8i16Pw4mkay6jOSJZ40eQLPc2E05K2LS0g5q6x4l0Iscv5gzO6ERKzt6pcDgmJUKN1c6ewXbeAfpdBQ\\\\\\\/SA3SODEQOdk4Mruc3Tlfjd7ivaQ66i+GUADNausLX8uQmMAZ8V5fW96xfUO\\\\\\\/R4BPPeBPqxdPLDLeVfEcpxbMYUBCNM1YFVipeUnYSFzI1QDxVRwBk9Fh0UZlYvI+jHlEu1UqsUvZt5w\\\\\\\/Vi6I6OBmC9BRxOHEA67Ze45e2I+9xyZWqzvshPMf6b807s\\\\\\\/B3UOoqvz0NJsUEZHl4PrpUYFnwtzfKgguFenKjL7uFYASm5crQ3wHW9IZT0AP6COQfNYfzYXlMUtDSmiNee3BhUHUzudWac8E0H9aWQEuWDSayTltPKbTdO2KLP8LMJEtpaypYKFV\\\\\\\/3VRqwrBcECtqEBF9gIyk1OZZTakmiI0wm1F8qDNvZ5vmZVCOfnhuEfjZpIsZYIRGRbcVGHvq59YcbRhzYu8rwg0JHqdBWPLr3R3IWj1G6\\\\\\\/JlaiFwDBr2uwvYJXl\\\\\\\/wQyS9M\\\\\\\/D0Sh90XNXfQKGa87Mvems6E2wUaq+F5\\\\\\\/3aJLOrJP3NpBKQ0C7RySfiBJ30kIQHQZLCfCGLmM2zejw\\\\\\\/tB9jvkXvUeK\\\\\\\/TYu\\\\\\\/XnGRmYbB+uddq3TXNeAAXlWDJbPjysri\\\\\\\/CYu0vDwIWL2JmqnFHinvJ6ao1oIobgt73oAOvQFJv3WOZM5IFoAR4bmcbr4VDVmxsMn2mRlamr2kRrYFLkkHDCsyncrL9uGj5bk6oR5xlL+OAy1wIubfARLjoUWKXUfBvAYbLNPrOtJb5uRzzq\\\\\\\/TyPZExtV6EPVlWaP2s9JAiSsOx6+BpSLKcnSTtoiptc4agIjBVMrPbYOKjAAfQTGrhslRRuDGitCGBpFE621r7TMlAhDZxcqVgvbfrm+q7fwhv09TVvOSxBX1z5fZcUDjLtzFIzAfdzjDN0vaFU6o1Ivwkw6GjeK+yTOk\\\\\\\/WKWtKGD5jP5aa7Z91NCWhzXZKl6XTVpuDu9\\\\\\\/siEzo8g0zFcqX+RrH\\\\\\\/cjA88GKXAC0qOs3ACIORFzyT\\\\\\\/Bjf9MmbqQPlKh8LOT7P\\\\\\\/xpESoMWE3GDwiKWeoeEC2PYoxFY4BvKPLUccQl+PUVqOreNFwrJWDTQDfBe99eSWtxegr7+1MF5zDwmS491wiwcIHcG\\\\\\\/PbjQqyEMzr93yJgwQq2+1pibE\\\\\\\/SQUS3xB73KotQoi8jcabaeS++EbB0qGjGJhJFso\\\\\\\/ZaFmun65UgBg5ZQj7ScA9EUGd1xgCHD4llrC9zF\\\\\\\/4Fmq+Ss46eMXM9mTQHvrDC09Nfe7dK2VbT4lyzq7v+VmwZj9Cpk0Ih+szoutkOlE+jZj+HR06qJtcftOPiQAhUzaZEKz6AWAeE\\\\\\\/KVfpyKHJNVG\\\\\\\/bXcBHUPg66efEotp2Yj7gGwl7MflPooBh9ZxcEf+FswMBpP37qeCCzyBww4A04jEBO2T6QIZeZWhEuNreLh2hAstt24R64EOJ6HsagKBLb\\\\\\\/OCoP5mJY6H90+ENQpODEWCTBKsz9yPHkZt20jCx86JGqWUZQ98Hu9JmQfSCHef3\\\\\\\/y6KzCfve8JuhQb1ciBNP8wr81x8MVdqVRVxpntWvLUYuf9fu5n+OmXv9iqW3de5HqxJbwK7daqAnJB2N1D7t0OOW9yJg4XFrd0tp7ftODNyrhw42Sgor\\\\\\\/LeonzXnSjG8OTUUOeL+XdOYmrx0GEJxIcclOm3ZgiPgVtsdC0t\\\\\\\/T9OPS3oHtoEVf9j66QfFMMhMrWiwVbEAq\\\\\\\/39BJnVV8caVSafwEXNvvb+GrqqIkRne5AC4i4VLp3wi8WjEVZQ5H6p5emAxKA6mEVgHNiiNRz3ehV4PK38RcQjK6t1uXc6AOYGvpUH98T4k7vK4hQfBPbW2t+8DDv75gHR47NpN4\\\\\\\/jTzgLcyuQ2xaVufSNyIHfi8nlYC1RPMpAy2DblbY6+tT11V9j+4My5JjGZ+8NjoXz5Tsguhk13lEns+SHHHs0CewL9M0XNyQTK6Doqq5e7grjdWIyakiM4bNkSazMIcowxZ\\\\\\\/57drO7XhBU0A10GCwr+lTjKk9V22ZHwq1ENu9Mm8zIhwT3vcuFLFmhvm1qooxhtVq1lk\\\\\\\/Udyp9Rtz5s5dGcagHDlwOt448eEIwYDP3Ra7wkzk0tCo8TI7PPPSqjmrIRdXfsOIsoQ0QIEm3m8\\\\\\\/XEdU0rPzENrKBJ3jt233sqmpUv85twAvk5XZ\\\\\\\/HMkw52QblQTGEjh2qs1JwSPidGoSU+Wd\\\\\\\/V5U5WNhqQzSeRuEsYEPJ7CfTIOlvZ2OCR0hESSJkWzUMnJ52Th0rGttEnX0qm9K9TZt02VCZrv61OTf3ZdAInTpHPDmOxteP9LwANr3\\\\\\\/29Z6dsSwsI4nxZQfNAihFXID8uKnmn0ZbIfXGBBZ8xI7FJX0xmU93YlvqcvQgqRwf3noyzo8UJ\\\\\\\/JArovuiFpgTEpnlzvLifxovif5cuJI3GvPstHhIcAGBHp5TG+Nfv0CbOmm75D8r2B7L1p2WDq2K3Sewy5io2d8f3sBGrsLuMRJxbdVGKOVJzToOItS+aAcH8\\\\\\\/5mUkeTIac7A6iFR3xVqmAmvKjL4HZ7cIUB4OpD4Q+Zc1ur2FWWtus6ML3rkY7zkbcN3iVxneauaLR5odB+G++9vAPy+vnb\\\\\\\/HV4uQ1DYIqYywMWU2Kda1qxSGglLkC9PqhZ8F0m7SsHVxpDQaeY\\\\\\\/4vp1U+R8UdP44A8n6NFOhoNtDQfjPc2MGWsGBDEcrpaOgS5iXvxHvqzOht+gfbtit9AtjFs5dj4UKRBcAKTLmeiJeh+hfOoUIORrJjbsz5T\\\\\\\/qIbgJYmQVMKotFFaRd4rnTNf10uB7ydu1ub9fSfuSIQYkcEG+IMXJa1\\\\\\\/14PaZ5aVBpgtUemtJmQNTWj91HZhYcDxC35C+i1mrNjMmpIYHbpRYbhuwZor6YNdAy+NWKQXW8WVJHW3uDQKdT40nij7fOZcaJCSWM1\\\\\\\/8nC98q9aMjeQYImYo+mSvBaUOBvVKwqqMwPXEwq4fnGvAb7u8guIr+hgvO0sFFEqfwqG0AzE8BR6nP6\\\\\\\/szRwV+t0m9nxF2IiIPhiPK0Hs0zu1pDATIHgO6ENcIUBMD3y5ffJ1w4m72AFzhwCTYWnJ7tAwmIexh+u3G+yPn4UrSDc1AO6Mu5kyuMg6e0GV0t0X9viIJ8tp8ijKTc\\\\\\\/Kgy6v2FT5n3EYto2vS6AfmoylkABput9HXg0gAFqArWyuBfqGmZTY1UzSPESKjmatnt9S+Yft34o8c69VA3rluXeld2s8Mf9l56BdvBKJnmjmJtCx5U2LMrp6PK8st0BKtDA6W2fP3xaPuUfgT8qUCDaUH6Npjws2w9xqS+z9oZzUEidvdWf4hv7dXTAp+rsldfrf4lbOmrzJJfD20MS9KrNRlYU2EcCalpDnmwwsjTGQQQNN6RG4ohgPyZrTtG6U41u+MKHekaf5pBgGiSTOCblyV2DEHZhNPmQnC2hWbkq2ABG8gYJe1ZwO8lZQw8n1ftW4kV\\\\\\\/LmhmjbIoQmdyi9tpQAShj5ox2upoNcUcZGex1jd4Ic7bE8XpSenWcpvGla9qUgwnUmYh+RMp40zVQLS2LI9yKQSNaiQGz3TZFUYHtNi3WAMVaINHO0FwJK\\\\\\\/SEICC9t\\\\\\\/hOVz1Aj43mC\\\\\\\/wzeFNwyS+QK5G+9Q3TJwDvw+EQdk798r0949gzJcp1mzezaNfzrtdB7any04ldyWa8jaGD\\\\\\\/7DtbE\\\\\\\/r39xGg7\\\\\\\/H1qX7IMUwijoU2wKZZvwzUXB0mH5a1+wJXNQD3rrTACq33JsOlWBfdHqrg5JvRaheBsVB9RFSPLOuVRsVxWsci1M\\\\\\\/psViJOSH55lHRXCTz1xnq6+w9g6G2cHB8NSb45+g8sppm161lxs0DhUfj2WtA6WvIABN259ZULQL0xPWiswkM1\\\\\\\/03xjvREMJmZT2ENx+NkgxyG9+ZNF8VcbgA43DTdVfYwHasKv+d9SNtPAn0VHBL1JAjk10RQAuzKn4\\\\\\\/Lo1Jw9rtSD50eQXOe5qPm9Xqo+8YWF7OIIXEe7Mgu4V4Jbi\\\\\\\/zGhQNj\\\\\\\/p0uOGSa7aX5Q03Pt0c6TreH8bdW21HskzRub7YccxAK4D+xrJKrv5FmYp2j2KlAE8\\\\\\\/OZaEFbnb4znkAp8v1Ts1pzcjuFoJ0Dq8iet88p4YnKQQYmLNFkFX7e69SWhzUyZUk7GgE4kkLwK0prOe4See6PwhKqQPAfP2jGUyBaemwVgs74Zk+HlvgUlad\\\\\\\/wnDx3oNW4IpFvkByRCrRtxIopJsNBjB+pn1lRnzT6QiZJuiXXA\\\\\\\/WoNE9VQ6\\\\\\\/0u41i2r7MyEM+K9dpBydR9cX2xF7I73NsfMtD0eOAATXOiPVgdR7D8IYHfp5Ry4ziGOCpb6ZSw4Qvcc2QEGTD4PaDgaiu+fcBjkIq0HQaMNAR8lGMEdFtxb4USl3E922bNsRJdDGUqKfxJcN5rHP9JdMA9mJ+O8G6WOn6BqAFDjz5tSkfxJ4BS0iBsRkGHJ6gWOZoGOXMJwc0eJXRyrhBHJUDd7rB0+0gPrrsYXWHtxebl4mWoBE1+7df1La3A4XL3qFuIHJKhIYnVVvAyxc7\\\\\\\/GF3t4AyUMKUn0Bznj4ozgiZrGXSaT343dn1\\\\\\\/iy97c\\\\\\\/kd09vvAMxxPCOAGQ0EHFof2EAcaRAeDMzfltf1zwFEkecDPGG2Jk8JJRPzGcYnPPhztNVOgonvr3oblQQr4oH+aNQGmy+BR78Yug5Zy3hxO8LMWPcS8gQVIrVi9+BgnCl7g8gl0+4OnaW7Xf2SFSTDtnJ\\\\\\\/+Q6v0KQ8kYA3mf85weomMhUxBkrq+8RkMkjndSHvJRgi5rVkT+V4ylz4xq5v9UZjAV+sXR\\\\\\\/EEKKQMZh1IsvaA8x+Qs9hnQeVqbX\\\\\\\/M1l6AZ0gNM0N\\\\\\\/sTI6WniTaRvNdYkNlrD4AC2XA0319wuyujJTMuhBi+sVrKcp+zGI+8ZNfw0Z2QPh7Bjuh+vEvwjm3g\\\\\\\/8ZX+o8r3Mm2TguAtU0vHDNNY4BLh+wPIrbSPq04qCaNWhhU8+Xcum+VpSm9UlGivyCuJjwhPkoX8gUZaFHfowIpMpZhEo6rA0TeEhBoa3k4xwQYlIZYxKOE0tWxsoTh3otXAzkh0ahk8p8lZCc95fTeqUQ+UNQSBrt4mAjjx1ipv83\\\\\\\/3QTAC7SPeV0w56Y2kCo7Me3bUOy+gwbxA6SsJF\\\\\\\/VfRQV2HBuYGwyN4YcW4c4l0rDWo1cgFWHozpx+JK+0Vgj3fQG9DmFuqbNk6L15xL3M6LnXqEnqwidjnho8hh4wWeaXG4mCqno+BEHkQOVouG42etmLlFtM30ZgV12zmqczE6jX3xO5NhJ7AYIHJLsuvMGmJwrFVp+DFSHQzux6\\\\\\\/39ZU5GUcG9bdUL4c7zAQfBKXeMvxqStDN5qYxzSMewJPQeUOHx1PBh0dENc\\\\\\\/x+n1WzQFASGA69bqu814l33sKjT+cAviyYQr3nL2Y1UHAcI05Zs\\\\\\\/FYjxABnTSS9v1igrAFjdwXjmreWNA7qoQUagYIDShAR122jdx9ii66OXgnpPEXcQAf2nzz\\\\\\\/mlvblDEiMC99LUou\\\\\\\/Q\\\\\\\/JT2dvXUdyHKSP1pJQUmvpA3dp0mYOf2mHIUDPV\\\\\\\/w9Y\\\\\\\/mx2jgOtyKHXo7sFBdg9smzcTlOwJUyP1kqRoEUWDkahgHp9lcI680jS6umMjaR64vsKnm8MDQI6c1YHzu\\\\\\\/5wFUdUoIPtJx6P4\\\\\\\/4Ri6o9\\\\\\\/oPar1bB3dLCpS581QgoXp5U7dPuztqVU7M4P0L0v5YpUWjn9GcEk4HiAr1KSvtO9C2urQeWq1UgGoE39i+JeF92QC8l0WIMVLUYAskeRfoava15oQ\\\\\\\/wnWLhr2t\\\\\\\/f3GXVy6OV6f7uQcm0b3SvSY\\\\\\\/+sCvjk4zn58X1nCQNk8egCOk5EgPd77qNgDRHrUp2Kx2WnRl2rW8R\\\\\\\/n6sgljLrnPsFldBtuJ8SsEmioH\\\\\\\/BUBauHrmMv4lpPiv\\\\\\\/z5hkgkNeCjRkHhPivQHNTO9nzb9IIMEy\\\\\\\/tDNX5IVR\\\\\\\/hPFlufsVkNYb8+kuDPczgXiNZCzwNRmVlgyAzL\\\\\\\/al2yLPamMjgKQMzR0OYwFjrigLbJgGvl5u480brv\\\\\\\/6gTnHamMDUCSETr8ktWhD\\\\\\\/YlnUdIcqrdnfOJoIyHFqx5kVPqyVMojam5KRCB4mlbLznzl1EZhCd3WlTxnCpWTg4FnTEZsbL\\\\\\\/FLTOratUKnlGDdYRYkLrlMyaGQqpvY92XGKNPZdA\\\\\\\/1BE11\\\\\\\/8M0XeNxUIl1my5T6qKzTJDiJjD+QuJmMlEcB9lpfyGyd7BpD+TZ9Sk37eL1e3eTTLUyBznDoouVip0xgDh2I62D0w85gQitSnM\\\\\\\/RZxL3tZEQQ\\\\\\\/r+gi2BJ4g51xxJI67Xl1P09zcR9+hcxz133tRQiBSXZKux0b5RnA3kXJ1p66fAvA6pmGoS7U8t30tbDuaoQ2KDtOFqDDtU4XH70Cj2ZI\\\\\\\/8jeSjPmj0y+d3ug+8vC22dMkUhe9ywhvsX0LOz316wn08hHuPXlFg3dhTJtA3baupQvh3oSiPtSKUHlvWk\\\\\\\/XJa\\\\\\\/Is+IiE66ZCPkeqgWa2zIChZFzU6BoiX0xPKhFzUs8f\\\\\\\/CEFREVwGcf9ONnVLicSfn\\\\\\\/3aoS5i3iCpvPAuejspML\\\\\\\/2TmhIpLLEimkzfCTGHt\\\\\\\/gPFSHARiHMxHoOhLHdNTT\\\\\\\/rsKX2lZroKrZNCTbJ6FoLzgI8htnxVfqkfmquCsvpNACiDTs8smnUR\\\\\\\/rQRgNHQJc+LjRDqcMxcI2pYlRrf9\\\\\\\/Irqxr2Rhqc1R6CxgSXa\\\\\\\/Wo8Mt9CO6qtwe\\\\\\\/r1y7WF4d9kBMtzYrvTuEuwseI7hc1GjqT7IoX5GbNZWu2RkJWejXosn0v5f8zf45XxmTVG5x7vXVgMa9jiS0ME1bm+nWrA87Ib9f1Xy9nHWkaZgLh2jJzu13mGdE4ph55ok0CgifOrA3FAPAN\\\\\\\/rTGibw\\\\\\\/QRIbr1NF7bWrFhaNqG4zfIZ97Fa5pW3vQx4nunZhwlwjjRMAraDIeAXFZAHQZBj8N6nVvI0LYodgKhSv8pKAlg4Gn3YQdFDfSQl4O5ACVQi4Y4Bhb0EXMLBIKL8rJGa78QuwrHHCYgYnOnfTZW8wwy7vYI+\\\\\\\/9ZhIx0fAVM6WhxiJBgCXOFsd7S4cVhYaezixVa7soJNEHNppMIOdJI2Z71Jyjp4UYzJNFDIqgujB75UMjP\\\\\\\/uHLZAccZIdO4Px7XJNy\\\\\\\/W0r0NU6Vs9lY0gn4F1OlxJ+WElOG54I0fvI9iaXEpqa+mGGD0cTc0VxAWJKEbuOrgsAuMlYCdVFLuUbXO8k5udXdkjLuXwYxxiLYLdX7VnFUrLm8zXAX92i1gzAPCs1iradIDQbYFg5llfE+ZfCoJeVLwPyzd3XKc9se9eIYGFs\\\\\\\/kljhp+ysbyQK7H1xOAc75VExUBW2quMFGJRhmd5LMIzSMintqpGy5aWI\\\\\\\/LMfmhuNnuMqlK280gvXxBSXp70IUg5wy9Nm\\\\\\\/fS1XmoMJS7crJmOTmcO\\\\\\\/PRJ2bFwBz\\\\\\\/SZNpZMlP3EbnHAXi4GAV9Gzq+Xd4UGNdhQ8ZXzK8tS3QH9ssMxU0OVoM8WZtTdXy8FBxveVPx5TOdw65N+Qjc5wNVN\\\\\\\/46iSP9\\\\\\\/jHzIK4xxnn4ynEts7YWqb8dTy+vMa24foYJplD2fopRXX3Eaa\\\\\\\/i5bthGWRtBtschrDi9LAsVyRSz4JQGqxaxsoiGuAVphJQFAY4MJnRk3zzIDjWLYci4GzNgdb5FOmZ8Ea07UJxngZMAExpk2Rv28p247KahbW+fK3c1VMvH\\\\\\\/nnDgcmR\\\\\\\/8g4bIkrjpoSmYFNC3bS5wZ\\\\\\\/sxYUXFFf5QXfs2D2B5wq4TUtDtlXEFbzDNY6dqxjYQLCzyFnnZvJxnVrxq8fpbPHf7EKbEJO43s+GR5QH2lyUVxWn4ncXAVozKrmsrdhNSC7sHo1b4vTrn5y6lCFxz8Pq4h8hkuCuxb+3CqYkMBFqxGmoRWgJAm8KQZiMsVhISiyhH1fsoBC1bjTA3Ye0ZN\\\\\\\/wr4AYcve8Z6klkmkENBSitKnVGpZjpk9zzj34bKctDi2mdIXwXzFGaQubjIgAb+SeFFwZS9+4CsqYSJ4Dm5GeGTsFd\\\\\\\/LFJQhJp2aFGe3vFOgsyNQIrwncbnWeT7VOudsWYe4LR5jbCSqKWx331JvjiIb7t3I\\\\\\\/xLx6MG+JmwMXddHlQTrYTbIuX6wip6ZskBOlAbgVcUKziX1zYZobSFKOJaN47FKbBdUEyjllsfifHvaPuAL+Mp4KWAsfGgvLvDUWf4FAxdcoyr9r+rtZVH2\\\\\\\/2HSVuJvo+Bu043wp8Tir29BpX1YaPoX4SxkYbeWxWvI8pac8s4Z+lSKPfX1Dns1loeAjN0vowJTTmx9vuwaUENIm1XZTzAVryH5aZ4IU2Ke1yEOZF6X2gNvQScqliLumBYSprwGyVhPWhYAA9gP4NwtJOe3HlJtI90RPPE6ZO63fFFaw1hiIlrGvGI+0cJjaFP097CVBiAPzn2L+ZRstzODpGhVuYwVUZ6FC+eb+zjYEPNIYnmQQqpgympD2fOxvKhefKozYFJZma0s4H246K6ln06GBZNTpH+COkhpxEpvJgprLin3aa8TFSH0WnsQAwatyyo3mizGYrYge+kmfwQGKzEDuTdAopsJs8zAELuj6tCI2l0dPqCD7ZLiP3balxpPGsGwCvkTQq5Hl5gUNG8sTFQoXvUumPh9rln0bg2InRgOTA7NCub+wqHeS74J9Ci3r+EpL\\\\\\\/lGk4HpdYCYIMiO1+tJNum84zMuUeX66QydsEAFkWyoY2n5wCZ9Dqqfo626B5mYQy5zj7i9\\\\\\\/LxXQq91byGmS80IW6QyEs338aYPOwwG1Ik68N1mRNkIZ+pJVYzNxutX6pNY5AtjNSgEq3roSkvXSQ6qs7GSy60igKq+nqOj2EiBDL8ccCtnIhW2PZKbf+ayqTVY1GQB2DnPXoRCU3fn8FEUG0m8JsKygxKYRU0eCkNJXb3GLpL6+zcW2OY+mIrypsdV\\\\\\\/fRrC78DI82pIKdmN2lIc1au2Ur3FYpcC2GiiG2GvT2siItpbEjDtLl+3eBp0hkr\\\\\\\/gKs1o4fUiw8V15wRGENPTnRAOZsASA1HuhE3RKJl+oipqIbWiEIwQwEatW+GGIAE56yogDZqKHPdS+P\\\\\\\/4JnPB0Hgw6Hx\\\\\\\/8u+dAz7dvqzIpJATBNIQv1cM6pb4f20KH6ohiOaTqJ3gwAwCPmBWVsaEI5wwzmcBdnKJRTcGgeQc1NFvfKCbZJcO\\\\\\\/Td4+GLghSOxdXXdcqnelAMpWiZ9j+NDZ4b3uNdethQGVN710XuzHnN6s0YNlLoil6IRgjFSTXvzppwjS8hLrmg248\\\\\\\/1caTBGbjU39l0JjOkWPhBWKpYAK6WvOdr48lTy2veLyjhzPirg4KD6RmvwC4v+1i\\\\\\\/0TQ\\\\\\\/iEG43DD28dMa4b+aXTy0R0v2X1uoE7j6DJFhltT28nOBirv8NNRafmVRQ1HbGy2XGrzKH3DOb6wfZDeqRPLXjXEOYSjhoyvuOBX7Q3wJCa3i8ELefMxB1xDmW8NtmnsBwPGDwruIMfi7y3SeOHJ+7DbAnTZECiL1y5aTdatLIz71HjbrITMgy7H0jS8WT756CRiGNO4cASnbyF0RiAlTlTAPgSTNcwHlYoSoexFySDxaZbaorGOVG3djfz0uAQ4dRljrZkmEBtHDtv7tPUIP\\\\\\\/Dk6yOiAUyZMVajC1zvtjw9C0BKPyDAkGbzoOBfruB2IaShVh7oQVCxEa0VLVrkcNo2UT\\\\\\\/C8j\\\\\\\/ZNpsMiS52D\\\\\\\/UJ5DWRXJOf+vrPZqqbOGQTpmOOvaCZOSi7vUXWcEZ1j6Dm4ANsGsfF5FbCZPUSu2okmXaZl0AwAzgn8qVCf\\\\\\\/INkyiOBNQw8FMExt3xaaLWJp3btAwELE7JM5+vAUDoFd8pI1EjOMZm+RHIUYkaeBUqXiYYwaHfDinPjNF0x+g1Qr\\\\\\\/m9BukRyOR98oP8VMraqp\\\\\\\/JfCcBK5ibN1AG4lBclogykGouIOnMnuu422MdJ68R86O4Ib8EOsafAps57bdNChhNfwbu8lv28\\\\\\\/YFKfrwxPhP79rze3yxqY3D7+X2XRh8DSVZhXDjecf7iSY5lo2uNUDXz37AnNG+V3lb8DU\\\\\\\/mkHR49OtNZpXbWlJJrQUbL3d\\\\\\\/745fZ+HlTMjnBZ1i7HZC4S1+YUHYUZLM9kANAm8jo\\\\\\\/b36gkavnkTwG\\\\\\\/j7wnGig1oYNui2jBcGXjShmQvolSBXk0LUrDd9sjhuuvB3tNrdX8F481+4sMJcWTXPfen6jwa1PhikSBC7TDQw5UMtgWct6gV4EhyyVNhwyNSkeRWwByUx4QZ+2A6r96b1Cs4YENHaM6P5+eUK47sxVWB150TGN0LVD+qXjzYIxMvXShWKDEZH5xw37iJnlCRtSRWCDI1QJaPh9mh0jKpJTzNTWWQw7nbYf9m0A4gMwh7VD6dXUt\\\\\\\/0NBHmrAvsyEFB4EmIf6hdR\\\\\\\/o26X\\\\\\\/k2wI22Gjoy69sM+tprng\\\\\\\/oSBvlWwwC6BPhyIQv93HKD8c1mTxa8e6mktL3fEJMnPcpk8EFdN173rtZfaDd0Jti\\\\\\\/8kLPibE0VhEtE2o01CCjvRoivaRcTjOn1cgw8cmc4kDKaZvVS67v+ODYEtHOxEro7vBda3XZfGwP4Y371pokUHZJy73LLYflDRjZlQJxBpOG4ghp7EwSc+Ypm9cgZ5LBUX4NR+BygnYra3IJfEwsWVjI3buLGLaIPgxGz6Nq3USM4Mwc0SWWocnuXry+rJa1+BXRUmqgKZoaV2xNgQt37FP\\\\\\\/ZM+z3iOXlTl02vyskR8IuUDlJGUJPSlf6OwJb6aZsINHe7V39chizS0IFNPv0zwHRfHP3bhV9poq\\\\\\\/eza0Y+1iKV2D2ddk4WkJoqflos3wO4HJ7yGAp8Pwl5KqN9qmEweFdDstw76cvJuEBvqgtBB4OX88TZbspqfLepSOzjq7gpQUfUdTcFXuzDF6AHS4ufTtM524JLmaUm5sxSgsTmN3tY\\\\\\\/++lS1S4W0zo2LnHexA85xVsjlgmxMPPyEJerJR2Y1KAFUtLqLAQuwZ4ZkKGfjDpiRLj4Z4xWvmoHMNDMt74iP43aKSc6LFCd2bQXhKAsUZ8Rp06ab9sR\\\\\\\/V1KrO08O+KmtWodXATDzmUkVaex9jS9ti7PJ88zel10\\\\\\\/QGKJBMla\\\\\\\/BTmQNgHaZnDd\\\\\\\/12hUaBSGXjYkeoghYNWGtrExxxnVlJnsSDtdE28HiBOPDkwrGariIFu3Kejhx8YWoCW4Xa0d9++BtlXCyLgkeZhttP\\\\\\\/WCXFuLgqPHGCvz1Y2guDM5Og0ZZdXBXcliViNaBHMQuyXYaqebJCQ0f2r0ygw1r0DIEcDA8Ta3EFpQUrKtQfeySCczKflrEJKmB13Y6HO1vREhRa05NZsw815vQ4G5QvN7vpQPvQdEHt2iwEzbOFzHV\\\\\\\/yEbLHyQ992P5L01h3UvZunCojMOo4MRNlvKz3e6Du6bUeJQf8r7iLZFD2N0d+pdEHqDZ52kgaGRhkSZ64aj5jY+exPra6GVth3eOeRIktprdhNiFCBYYiTXU9l6Bvey55hD+fgNDgOmhCY+iVg9VpcW\\\\\\\/YKT77T3a5RGw\\\\\\\/uVRjSULIe6+3uAvnps+YZ\\\\\\\/ghdr7OnGvzkiyYNshQpsCF1lKI4hIyklF76wHUaj\\\\\\\/m82pgX+ZfEgVbPWj8Njie7bZxy8OpRvgIj94TllK5ykXm\\\\\\\/Dz0rA+jYxAjxaLlKef8t5LmTvBeqyLEYQTp0KD6liiWaMiQC9rvupgix5W6F379EnBHFLxTNz\\\\\\\/HvK6xvvVC4sTFcMfJ4hZ7VTYcpPLwDvsMkamlbx3UFA+qp7yMbgeGAtSSfmQAd5MT4HlSChuBeQ8egMCFa9D2WePU8YB5nRs9BsQguAC7Jg1wnQMwyxjeFCN3B+xU4YDXHNjXaOeCkp69xCD97hCduqaQ+I6Mv7kxVqoy\\\\\\\/SELbnDAb1vOAHTxRCFEEdphVYphlDBshokTPFjL8Dn\\\\\\\/Z5G1f9r3nznpRX83pavPcIZlEcUQsoBCgsaoGtl6+Ms68LJuqC43NF6OqnzEIxydoJsfExDDgDgwTp5JThSymXf27Ed9wJMTxeGkEG2l0hp3vM\\\\\\\/GnnNLrszMcM+I+KoKSwQBhBxmrr5cQA0dHAKwOwyz2OiTlfNXdCd807OYyQeAi3rGY81G75M7eSS\\\\\\\/Gt8pr99kN2osrmnQPhIjvAXwimwwI9mKiEVgPR5s5YxbFExH8WLc4hsnQNPm2skPDyv+cJw34mfaag\\\\\\\/3Yu5dIt0x08ojNXOo2rlFp0m9O7wu4FaDdSASbHzfCMhWN1nwsfAlS9IO5KmqlcJ7IUai8XrV+IMpnVMuAppLjBdxGUoNrVdVkRFlpduKx3\\\\\\\/ZmImL0uje+LCk6MA4Og47pvZMBGNqSyzTRZa1chB+QFcep8kroi73h24RkoG4xeZyofB5dCOJKYzceSQ74bPjhy3EjAmLhK4p0AouzkVzxKk9NtW8RtAsVmoNayiAydNHuvCcfoYp+oXZoefP3QMVQc0fnifeA6SANW6WRQb8nLSs7pVgA6QNKrUikOIO3Han2rkemJ+WpxdtcDxJphds+0DYTe5DFMUsWb\\\\\\\/x8HipGQPX1dzNGfkEk7PzUncZB1hcaKO8WBoo87gqEK8q4IPKXIb9aJbx41xLlSr9uKtSSRLtD5lyT3Jmg8uilLihffSgqrg7PUHj3fj\\\\\\\/K9bQfPimmtsvw+tJhST9dAPv\\\\\\\/aOC5Mhb82R\\\\\\\/yyu4CRW7Y3SDikghcJHItoqY0Ko\\\\\\\/PUbJDbfEMleXI8Lymmx0txm6Ket457AwdqBVwtY0SmR5jnt9gzIBjAApMHtsAD3LRD8dW3K3RjrrdOq9pQvcdPrNMvCqtQam9rsMH4d+96z\\\\\\\/Nk1bg0JIS8SOuVw79ox9vUB\\\\\\\/Z8BvJhiSfjEFDrJtQXeki32d0VQdms60F2YfRBs+\\\\\\\/DvrI2qEW7e9hNezhDPOjFvj1ZqUSIkO8gqNB3YzlAXCpz8l79quaooXhqK48IcKOMBKP\\\\\\\/3vOFm4YeexIb+Q1gBOxnZCrvkcPJSKivxtt03d6Gd+h0iJ6HvmDHHYTOHfYbAkmB324FPsOkYAsagf3+ho8jo+wr2bmMO7ZCXBOXCgRCnEWD+ig4oidLqIUHNdKnvPRz8eQtlaV5HOKcqyIehfXlMii3aJDfE6S3EfT6xbmzQS7egHT68it84FsWxvl6angJa2A3JOEobvf2o3AgCtazxf7zkq0Uq5NJoWWqQYBP1ZwSJOQXu+79H6aoe3wzeu0zkD8Fn08tibavC81hOglB5eapMF6XzI174Kx1LsN7hlFpvGqGScaWTYSh586yjPUsq1mqXdY6ffNL3Q9LHVdagkWAK6Fd1F6b8zyvVPB6Ulan0oBl5y1Gz+i5j4KnfCd9QctOKtEplsBT37MTsMOMlub4do6nL4ppir8xVibV0i\\\\\\\/DQ9k4Fe6VopKl3H5V3pr1XLv1gyx+QTnfSMV62A+0aZ7QYO+VjBGvXmC8vSrIasq36bP8\\\\\\\/z1PCSrOHgsQTGCqYGbwGaubFEU5XaSlTHR0n5uOqlnRJ9mU97z+7U+Drpu1iyB3lwHovuLer8C7FUj6TV0T2jCVjhV07dBqwtMG1Uz3+NOw2lQ7UfalzubLTvTGR4EK4kYqpyFLFTB\\\\\\\/uuo6j29aFKgkmTEV4QDIhQbA+wmavN\\\\\\\/JnL+6ESbceqTzODkNlcs9IAhjrMDhB7Q5Jzzg\\\\\\\/BUpjn4jqfhHdkH2VP342WxzdSSLo4MlUys4KNs9eJ+IqaCjSoto1HlAA1kd3FKrDu5\\\\\\\/zBGjcXYHnPUP3DIvGC9Fu+0EllswSwvuz7empyGp6pk4to97fFLEGf+fKSuy2E3UU1gEGG0yoEELIzpvsypcKYOJntwjAhGXELfIpyZ21Orv3woDazo2LGclgH5d4T6OwXtnxvQuE4HkuxX5zqUeOTmNe5C8Pa3eCtPaUfrTjzns1onr0j0ME1r\\\\\\\/7sj0XbHacndFN7K7BOF90BtSUAsKfys+rMqoaxIDtYB8JDdfISxMEOCWYiWzOTnxlTyZDA9Xx17NulU2HqswmP7\\\\\\\/FQ529qe7zrkQSnqMQZ6SWvB16G4JoyxdHDpmTNQi8y32dYgj3sY0SlQtYbC0kof4XjOKZoVlPpdBvzQVrxtuLi5ysrs8X4gUoETaLIH6Ds7XJcAXHf0IvPmcHcf34vJQWPf5dgLFyL8C0L8+\\\\\\\/AeuAI\\\\\\\/NOs+zkFfiFuOzMVxqFyLMk03L658i3QKps6R8C8BncxDxfdfVkjIxa1Sz7SlqGEKfJoUH8Zoy8mcbLDYKyOATDOaK2iISaQ2WnTzQsKHWFWzrxZ+ELfUgI9iLTzkRlOHxQouyStyh3Af6teGmCZkyjoT8ixQxXpeiE\\\\\\\/PhurJeoAdaSFa9tpdLPDQgFgwn\\\\\\\/2ncet\\\\\\\/9MDXqNyHK0GRjt3gz+fMheGUhvSitHZpKOeAcQ+WPWm7Asl41P0mVHoZmawLku8DLOMLL\\\\\\\/LUUSQbvnF2fjeMjxSLWZz5KCoD\\\\\\\/4PjWPVCUFi83Ww4crCSKMWyfVV2loEQ7\\\\\\\/J+LdSFv2j8AYxG3WYBt8o5PqelcZRPqTz9mbT2ImChYfit1rfZ\\\\\\\/nJ6FmhZ44IBHN0xQj83JFtAF2wog\\\\\\\/phjfEaeW3VTTQBp8M2QSaRw9QwQtIypZ9qkH9JgkygyPvzT7hy9Dwydmt1y5IvnLgcj2CiFj3E40QAt8YIL4dQR9hI0v\\\\\\\/zWKWm6Yyxqnf8a5EHkrTYQM9jGNxD2abck+VFotuyPkCQKO0OAVvlEuDQ5fSp\\\\\\\/ZfGnf3yrABCp7AW4tx\\\\\\\/xS+iH6YS9w7Zv5cAif3LAcvCvUaAY3wUSLLTHjXdD2xAyoH+9rgSt3H\\\\\\\/ipFMMC5jILDyn4\\\\\\\/WxQrQniS3A8HRVVfBYt\\\\\\\/IRGJrOjS0I0ucFPW5e2IoWh+M8uDthGZbIA7CWaoA9nmtkjvrv6XWU1wqVIte7qB1POF825ReyOLcBCTKp1rvZQXZ35NGdfzluKkcz1069TgMCVb5BqwvcXOYR01A8CMLIPhU0N\\\\\\\/bg6EpgdPH\\\\\\\/cmR1J0cxVOxK4lY3sbSS56CJVgY9trJJxmkXRE4dkmB0\\\\\\\/Q\\\\\\\/HWb3EznXqY+gSH0UKkB5svYgAd0UlbhecZDb0zsMayYiJlLh0Dgpo07atpEPrTMZe26WPvXgN0Zg1g4NEw1gMW1X0Kk9KcIBbzc0fVdGl4nHsA\\\\\\\/3K+GNcgxySPIKSml3TsyHJWiz0eegq9EsZ31KJIvcDK+Sr\\\\\\\/oEh6dcG+J4QMw6w4qnUZvMN5WzusCnQgafGupeGxZpLQZu4o6ndcntfGljjNJutL4PHSztwdtF4RDXt\\\\\\\/qnFUljBv0jlnffvxlM2KCOiISfGCN+GOLPM1dvhoMIz9kXq3X\\\\\\\/Gs3puMcxZu4scWmPNeAlJIIjOQ+ezbNhUHS0+z+yJ0uuhZIYYQytGXOAIT\\\\\\\/HzZzj0+CJD\\\\\\\/V72CWA5VXYnfcYLRbfbxGUXS4oRX5VJfBCmtyYEqrS1fRthCbWdAsljMJJ5czka20FP8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022eJJFLqFIWPGIECyz\\\\\\\/bayzA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220d305bf8cfeaaa18d0da8faa5d45913aa34248e45eb7074384aadc4e017476a2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022I+dbit9ZsZnQ2MhUiCw\\\\\\\/fD2qG+AujY1ht1TYna\\\\\\\/3OReMJJ39Ff\\\\\\\/AAcBGj6XHj\\\\\\\/hrJb1TsCAbeFP+OD7ZD+QjSigK31ZeIW+AwgOQSq8iWiHg5Kh2J1i9gvnH7eElhvaHRyg9K0yr9wQ5q4RlNFufPlLxDXAhoHYaDabtTHLzYAgvNM2nOahfMeZPluLjfSWYpN\\\\\\\/+SNYLn6lfWf2WxNnvKfbHOAtkomvyroVZUhDff56GacWk7gaAm7eZkCimgHOmNtyrvcpWf9RaH+LsQMOMSXHe4TwANb7MbaOkfFDc29Lk+5Ap\\\\\\\/\\\\\\\/e\\\\\\\/NllQQSzTHLfDkUPMXrpdoNizjsXATAASoj\\\\\\\/aLEN63cOtwRYSLnUxVGJRmz9zj\\\\\\\/QRCAnsRg5Aq5XZCL32h6Rurx+DwwoO5p3RmeGQwxZtxSK8JLOUaUodP\\\\\\\/dpGuId94P5N1dAYNBrc\\\\\\\/ULBkzbyNw+RbduF5KAXdDhvcF9BYkRy0C2GHocmNs\\\\\\\/RFGQTaN1eAL7w9xGdEAZK9Nz1UdtOtLTo9XrvDWGDVc9sU9rUcj30i24QemKoWWFUx05u3AJ+mN6l8LOb3iM2oovNeFmGyVBMWpbcRIpdcLkTwZjMAb4V54Xkms+59h2WTRsrGE1x0f4wfOA4G7evX1uR6D8XShRiumK+JcopOhNgoQ9eGDp3ygaCZJF+T5vMaffTIcXv9GuAw6RakFbT0I9Juxi4hJpLcTLWcOP3AOHBu2Q+d+SmZx\\\\\\\/sRGnNl2BC4hYZmTWxsIM6lQunwyF+nfWzU0s2O8n2YbIfLWk5Z8k2sHG\\\\\\\/xp17u4tx5QJ7WsoUzH+WnUeYswYLJqe9qdQ8H7IesjuexsszfDLE0XmNkZfCiVa2tXuQmTmmES7LsCukSPE6AMj0KuHWIBVaJogIhul0mUveVU14XDWr\\\\\\\/sqH4nuXUhy4qz5LOnQsjS+YyQ3IWWaAM8ETO0CJh1P0yxi\\\\\\\/fHJZyvEDn54cLw61vhfYBv87QXayOeT84p0uZlcKP0PdYtkRnoKla\\\\\\\/y4\\\\\\\/8WCUM\\\\\\\/JmTpF18Y0JOngdzjzSC1CaJpeNM4ISpx9GDyDHjM4HVFjTKs5zV3maRELsrS6+3ijwhlNeMG868fuLEOyGWjD1C8P6\\\\\\\/w1UUB31OlghUZMDjp0lW2ebiYIGyurxhd23uLgkUke6h8XyWS3rPjfY28n\\\\\\\/CVVGwTig27jFAi1HzDlxCN6wgoBoTtwrtcZiygh\\\\\\\/vYhQVtOc1b90N0fBIQeORrDgRQv91rHU9j068twNy8QsLvur5o+Zdsv1Yqi7jX3fhugu0o\\\\\\\/jWpsyZdA4ZJbmBXZMsnoV3sl5zgH4b1s9yQB80eBGOLxiMO+bpSaOoo7jTAyfp4jXV88Ip2CPadT\\\\\\\/62wTf2D+WZfF11Y9VUs3J7qWNkOssBOIemNz1PWDLig5zC6+uNjA9KXRktjWvw5Ec8UjOWtxwH7Pf4uOfYKcqTT6e036q6zWKzT2R4xYYb3RAZ4lXP7rXNjZ1koZkItneRd8OUwv1u4ikExrGzC5Tf\\\\\\\/Qgzd+9HrIcO+fTVMBgS5AlGCgy+Fq0RwHaU0UVx3M6GnRuNX\\\\\\\/8vIKDA0jeMX+x9UUSBcciCwampchYRZOCvoBSFqJlh8faKE6XUtLxTjNj9isDuFrXQOhWTtsRnIs8oNTbRw5nbf9wdabza33aMm2v1dANKjx9zhUv6MNptshRLEnIRt91vwgTG+Fd7ffHL1\\\\\\\/tC0cXFkWlmyLGGSz2pEfoJa+AF1eSt4HSS7HxAIh7o+mL6+h0ANhnnt0yqFB6OnWf3FfrQBM1pl\\\\\\\/rpFM5ppyAya18e0oBqJZMaSmrCj8BDpjsRTDjNgFwj2\\\\\\\/5aTS07AOH931BP\\\\\\\/XSEImUxxwFVZhR5bQsHuq3javl0SnkPnAOv2uoNfzCRf7xpPg2w5gsc7eRrdvWARsrOtYU\\\\\\\/96jbDl6JgVQ2C\\\\\\\/u4XBEymOe7lOXjZFpmUT0g7OQdZ42Oqoi0GCfIJ7RaEi6+CjwMT3kEcbtYYeIfS0UXveUbZWSuLbaKE1mm\\\\\\\/lKgPJMWC3pwh1klKq3MHpI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022KRdp2mWsUPe4Tox\\\\\\\/CqRpxw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022087354f830c3ee67562467f30203dcf98f8787d052b33f4d906c42c68ca47b61\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495864 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1113" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FEWNuOS9xDtB53GRvQcmOwIoTZ7i3MpjQoE4FfmGvDY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1113" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vgHVIiSbLA-nqf_zilRZqw94hdVrrIsuk7KOcn5O3fc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/1099" title="Multiple Requirements for Rab GTPases in the Development of Drosophila Tracheal Dorsal Branches and Terminal Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/1125" title="Absence of a Faster-X Effect in Beetles (Tribolium, Coleoptera)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4959-0481" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Madhav Bhatta</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2957-3086" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lucia Gutierrez</span>, <span class="highwire-citation-author" data-delta="2">Lorena Cammarota</span>, <span class="highwire-citation-author" data-delta="3">Fernanda Cardozo</span>, <span class="highwire-citation-author" data-delta="4">Silvia Germán</span>, <span class="highwire-citation-author" data-delta="5">Blanca Gómez-Guerrero</span>, <span class="highwire-citation-author" data-delta="6">María Fernanda Pardo</span>, <span class="highwire-citation-author" data-delta="7">Valeria Lanaro</span>, <span class="highwire-citation-author" data-delta="8">Mercedes Sayas</span> and <span class="highwire-citation-author" data-delta="9">Ariel J. Castro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1113-1124;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400968 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495864" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Madhav Bhatta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, 1575 Linden Dr., WI, 53706,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madhav%2BBhatta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhatta%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMadhav%2BBhatta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4959-0481" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Madhav Bhatta</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lucia Gutierrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, 1575 Linden Dr., WI, 53706,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucia%2BGutierrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gutierrez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucia%2BGutierrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2957-3086" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lucia Gutierrez</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gutierrezcha@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">gutierrezcha@wisc.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lorena Cammarota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of plant production, Facultad de Agronomía, Universidad de la República, Ruta 3, Km363, Paysandú 60000, Uruguay,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Maltería Uruguay S.A. Ruta 55, Km26, Ombúes de Lavalle, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorena%2BCammarota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cammarota%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALorena%2BCammarota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernanda Cardozo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Maltería Uruguay S.A. Ruta 55, Km26, Ombúes de Lavalle, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernanda%2BCardozo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cardozo%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernanda%2BCardozo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Silvia Germán </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Instituto Nacional de Investigación Agropuecuaria, Estación Experimental La Estanzuela, Ruta 50, Km11, Colonia, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Silvia%2BGerm%C3%A1n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Germ%C3%A1n%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASilvia%2BGerm%25C3%25A1n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Blanca Gómez-Guerrero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Latitud, LATU Foundation, Av Italia 6201, Montevideo 11500, Uruguay, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Blanca%2BG%C3%B3mez-Guerrero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=G%C3%B3mez-Guerrero%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABlanca%2BG%25C3%25B3mez-Guerrero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">María Fernanda Pardo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Maltería Oriental S.A., Camino Abrevadero 5525, Montevideo 12400, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mar%C3%ADa%2BFernanda%2BPardo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMar%25C3%25ADa%2BFernanda%2BPardo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Valeria Lanaro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Latitud, LATU Foundation, Av Italia 6201, Montevideo 11500, Uruguay, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valeria%2BLanaro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lanaro%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValeria%2BLanaro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Mercedes Sayas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Maltería Oriental S.A., Camino Abrevadero 5525, Montevideo 12400, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mercedes%2BSayas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sayas%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMercedes%2BSayas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ariel J. Castro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of plant production, Facultad de Agronomía, Universidad de la República, Ruta 3, Km363, Paysandú 60000, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ariel%2BJ.%2BCastro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Castro%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAriel%2BJ.%2BCastro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/3/1113" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495864" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495864/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/1113.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495864" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495864/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/1113.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495864" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495864/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/3/1113.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495864" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495864/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Plant breeders regularly evaluate multiple traits across multiple environments, which opens an avenue for using multiple traits in genomic prediction models. We assessed the potential of multi-trait (MT) genomic prediction model through evaluating several strategies of incorporating multiple traits (eight agronomic and malting quality traits) into the prediction models with two cross-validation schemes (CV1, predicting new lines with genotypic information only and CV2, predicting partially phenotyped lines using both genotypic and phenotypic information from correlated traits) in barley. The predictive ability was similar for single (ST-CV1) and multi-trait (MT-CV1) models to predict new lines. However, the predictive ability for agronomic traits was considerably increased when partially phenotyped lines (MT-CV2) were used. The predictive ability for grain yield using the MT-CV2 model with other agronomic traits resulted in 57% and 61% higher predictive ability than ST-CV1 and MT-CV1 models, respectively. Therefore, complex traits such as grain yield are better predicted when correlated traits are used. Similarly, a considerable increase in the predictive ability of malting quality traits was observed when correlated traits were used. The predictive ability for grain protein content using the MT-CV2 model with both agronomic and malting traits resulted in a 76% higher predictive ability than ST-CV1 and MT-CV1 models. Additionally, the higher predictive ability for new environments was obtained for all traits using the MT-CV2 model compared to the MT-CV1 model. This study showed the potential of improving the genomic prediction of complex traits by incorporating the information from multiple traits (cost-friendly and easy to measure traits) collected throughout breeding programs which could assist in speeding up breeding cycles.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/multi-trait" class="hw-term hw-article-keyword hw-article-keyword-multi-trait" rel="nofollow">Multi-trait</a></li><li class="kwd"><a href="/keyword/multi-environment" class="hw-term hw-article-keyword hw-article-keyword-multi-environment" rel="nofollow">multi-environment</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/malting-quality" class="hw-term hw-article-keyword hw-article-keyword-malting-quality" rel="nofollow">malting quality</a></li><li class="kwd"><a href="/keyword/grain-yield-grain-quality" class="hw-term hw-article-keyword hw-article-keyword-grain-yield-grain-quality" rel="nofollow">grain yield, grain quality</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Barley is the fourth most important cereal crop worldwide with an annual production of more than 143 million tons and is mainly used for animal feed and malting (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">FAOSTAT, 2019</a>). Therefore, barley breeding efforts have focused mainly on grain yield and malting quality traits (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Thorwarth <em>et al.</em>, 2017</a>). However, the collection of phenotypic data such as malting quality traits in large-scale breeding programs is expensive and time-consuming. Additionally, malting quality traits (<em>i.e.</em>, beta-glucan content, malt extract, soluble nitrogen, and protein content) are complex in nature, have quantitative inheritance, and are evaluated mainly by seed destructive methods (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Han <em>et al.</em>, 1997</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Igartua <em>et al.</em>, 2000</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gutiérrez <em>et al.</em>, 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nielsen <em>et al.</em>, 2016</a>). Although there is an advancement in the high-throughput phenotyping techniques, it is still expensive to evaluate complex traits such as grain yield and malting quality traits with high heritability for a large set of genotypes. On the other hand, the cost of genotyping is decreasing, allowing to genotype a large number of lines at a very low cost. Therefore, some commonly used breeding strategies to overcome the challenges of complex traits is to use molecular markers (quantitative trait loci/marker-trait associations/all genomic information) for predicting and transferring these traits (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Igartua <em>et al.</em>, 2000</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Schmierer <em>et al.</em>, 2005</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lorenz <em>et al.</em>, 2012</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sallam <em>et al.</em>, 2015</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gutiérrez <em>et al.</em>, 2015</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Schmidt <em>et al.</em>, 2016</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Nielsen <em>et al.</em>, 2016</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lado <em>et al.</em>, 2018</a>).</p><p id="p-4">The ultimate goal of plant breeding is to improve the genetic gain, which is defined as the gain in the performance of a population per year through selection. In modern plant breeding, the expected genetic gain per year is defined as Δ<em>G</em>=<em>i r</em> σ<em><sub>A</sub></em>/<em>t</em>, where Δ<em>G</em> is the response to selection, <em>i</em> is the intensity of selection, <em>r</em> is the correlation between true breeding values and genotypic values, σ<em><sub>A</sub></em> is the square root of the additive genetic variance (<em>i.e.</em>, standard deviation of the breeding value), and <em>t</em> is the length of breeding cycle (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Xu <em>et al.</em>, 2017</a>). Genomic selection is a recent approach that increases genetic gain per unit of time by accelerating the breeding process through estimating the merit of an individual with the use of all the markers from the entire genome (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Meuwissen <em>et al.</em>, 2001</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Resende <em>et al.</em>, 2012</a>) but without phenotyping it. The first step in genomic selection is to build the prediction model for estimating the marker effect using a training population (<em>i.e.</em>, with phenotypic and genotypic data). Then, the model is validated through cross-validation. Finally, genotypic or genomic estimated breeding values from a breeding or selection population are predicted based on their genotypic information (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Lorenz <em>et al.</em>, 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lado <em>et al.</em>, 2016</a>, <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">2018</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Xu <em>et al.</em>, 2017</a>) and the model. However, increasing prediction accuracies of the testing population remains an area of continuous research for quantitative/complex traits such as grain yield and quality traits (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Sallam <em>et al.</em>, 2015</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lado <em>et al.</em>, 2016</a>, <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">2018</a>). The predictive ability of a genomic selection model is the correlation between the true and the predicted genetic (or breeding) value from a cross-validation scheme (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Lorenz <em>et al.</em>, 2012</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lado <em>et al.</em>, 2016</a>, <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">2018</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Xu <em>et al.</em>, 2017</a>). The predictive ability is affected by population size (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em>, 2013</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zhang <em>et al.</em>, 2016</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Duangjit <em>et al.</em>, 2016</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Berro <em>et al.</em>, 2019</a>), population structure (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Endelman <em>et al.</em>, 2014</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Heslot <em>et al.</em>, 2015</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Duangjit <em>et al.</em>, 2016</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Berro <em>et al.</em>, 2019</a>), relationship between training and testing sets (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em>, 2014</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Duangjit <em>et al.</em>, 2016</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Thorwarth <em>et al.</em>, 2017</a>), marker density (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Poland and Rutkoski 2016</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Duangjit <em>et al.</em>, 2016</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Thorwarth <em>et al.</em>, 2017</a>), trait architecture, heritability (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Sallam <em>et al.</em>, 2015</a>; <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Duangjit <em>et al.</em>, 2016</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lozada and Carter 2019</a>), and the statistical model (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Lado <em>et al.</em>, 2016</a>, <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">2018</a>; <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Xu <em>et al.</em>, 2017</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Lozada and Carter 2019</a>).</p><p id="p-5">Multi-trait (MT) genomic prediction models use the information from individual lines and multiple traits simultaneously. Recent studies have evaluated the impact of MT prediction in genomic prediction models. For instance, a study of 495 wheat advanced breeding lines has included multiple baking quality traits in the genomic prediction model and identified that the predictive ability of unobserved individuals was increased compared to the single-trait (ST) model (<a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Lado <em>et al.</em>, 2018</a>). <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jia and Jannink (2012)</a> and <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jiang <em>et al.</em> (2015)</a> have incorporated genetically correlated multiple traits in prediction models and identified a significant improvement in predictive ability compared to ST models. <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Montesinos-López <em>et al.</em> (2016)</a> have found higher predictive ability when using correlated traits compared to uncorrelated traits in the MT prediction model. Similarly, several other studies have observed an increase in predictive ability when multiple traits were included in the prediction models (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rutkoski <em>et al.</em>, 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Guo <em>et al.</em>, 2014b</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sun <em>et al.</em>, 2017</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hayes <em>et al.</em>, 2017</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Lozada and Carter 2019</a>). The use of correlated traits could be especially valuable for predicting expensive or difficult to measure traits (<a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Lado <em>et al.</em>, 2018</a>) such as malting quality traits in barley. The main objectives of this study were to determine whether incorporating multiple agronomic traits into the genomic prediction models would increase the predictive ability of an agronomic trait of interest such as yield, and whether the inclusion of an agronomic trait or both agronomic and malting quality traits would increase the predictive ability of a malting quality trait of interest in barley.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Experimental materials</h3><p id="p-6">Four crosses using five parents were performed to create a large inter-connected population of 980 doubled haploid lines (INNO1, CLE268 x Kalena; INNO2, Kalena x CLE267; INNO3, Kalena x Conchita; and INNO4, Livia x CLE268) at the <em>Instituto Nacional de Investigacion Agropecuria</em> (INIA), Uruguay (Figure S1). A total of 145 doubled haploid lines from those crosses were used for the model comparisons (33 lines from INNO1, CLE268 x Kalena; 39 from INNO2, Kalena x CLE267; 45 from INNO3, Kalena x Conchita; and 28 from INNO4, Livia x CLE268, Figure S1). The 145 lines were evaluated for agronomic traits for two years (2015 and 2016) in three locations [EELE (57°42’W, 30°20’S), MOSA (58°02’W, 33°16’S in 2015 and 58°04’W, 33°19’S in 2016), and MUSA (57°58’W, 33°54’S in 2015 and 57°53’W, 34°01’S in 2016)] and three years (2015, 2016, and 2017) in another location [EEMAC (58°03’W, 32°22’S)]. This resulted in a total of nine environments (location-year combinations; EELE15, EELE16, EEMAC15, EEMAC16, EEMAC17, MOSA15, MOSA16, MUSA15, and MUSA16) for evaluating agronomic traits. Malting quality traits were evaluated from samples of three of the experiments EEMAC15, EELE15, and MUSA15. All the experiments were conducted in a partially replicated experiment (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cullis <em>et al.</em>, 2006</a>) with eight replicated lines in each experiment augmented in a randomized complete block design. The agronomic traits measured in this study were grain yield (kg ha<sup>-1</sup>), number of grains per square meter, thousand grain weight (g), and grain plumpness (%; the percentage weight of grains retained over a 2.5 mm sieve). The thousand grain weight and number of grains per square meter were not measured in 2016 at MUSA. The malting quality traits evaluated were beta-glucan content (ppm), malt extract (% dry basis), soluble nitrogen (mg per 100 g), and grain protein content (%). Micro-malting was performed at the <em>Laboratorio Tecnológico del Uruguay</em> (LATU) and Malteria Oriental S.A. following a standard malting procedure.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic data analysis</h3><p id="p-7">The estimation of the genotypic best linear unbiased estimates (BLUEs) was obtained from the following model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-1.gif"/></noscript></span></span>where y<em><sub>ijkl</sub></em> is the trait of interest; μ is overall mean; E<em><sub>i</sub></em> is the effect of the i-th environment (location-year combination, location, or year); B<em><sub>j(i)</sub></em> is the effect of the j-th block nested within the i-th environment; <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-2.gif"/></noscript></span></span> is the effect of the k-th genotype; GE<em><sub>jk</sub></em> is the effect of the genotype by environment interaction; and e<em><sub>ijkl</sub></em> is the residual error. Genotype was assumed as a fixed effect, whereas environment, block nested within the environment, and genotype by environment interactions (GEI) were assumed as random effects. For individual experiments, BLUEs were estimated using the above-mentioned model excluding environment (E) and GEI effects. These analyses were performed using PROC MIXED in SAS 9.4 (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">SAS Institute 2018</a>) using restricted maximum likelihood estimates for random effects.</p><p id="p-8">Broad-sense heritability for combined experiments was calculated using the model described by <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cullis <em>et al.</em> (2006)</a>:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-3.gif"/></noscript></span></span>where, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-4.gif"/></noscript></span></span> is the average pairwise variance of the best linear unbiased predictor (BLUP) and <em>σ<sup>2</sup><sub>g</sub></em> is genotypic variance estimated using the ‘sommer’ package in R (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Covarrubias-Pazaran 2018</a>).</p><p id="p-9">Pearson correlations among traits and environments were estimated based on the BLUEs using the <em>rcorr</em> function of ‘Hmisc’ package in R (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Harrell 2019</a>). Principal component (PC) analysis was performed for all traits based on the correlation matrix to further understand the association between traits using the ‘factoextra’ package in R (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kassambara and Mundt 2017</a>).</p></div><div id="sec-4" class="subsection"><h3>DNA extraction and SNP genotyping</h3><p id="p-10">Leaf tissue samples were collected from all 980 lines and DNA was extracted using the CTAB method (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Saghai-Maroof <em>et al.</em>, 1984</a>). Genotyping was performed using the 50K single nucleotide polymorphism (SNP) iSelect array at the Cereal Crops Research, Fargo, ND. SNP alleles were called using GenomeStudio software v2.0.4 (Illumina, San Diego, California). There were two main steps involved in the SNP calling using GenomeStudio. The first step involved performing SNP calls using the default clustering algorithm implemented in GenomeStudio and determining the three distinct clusters corresponding to the AA, AB, and BB genotypes. The second step involved manual curation of SNP allele clusters that were unable to clearly be identified using the default algorithm (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Guo <em>et al.</em>, 2014a</a>). The accuracy of SNP calling was validated by looking at the cluster performed, call rate value (&gt;0.95%), and GenTrain score (&gt;0.24). To further improve the quality of SNPs called, SNPs with call frequency &lt;95%, minor allele frequency &lt;0.05 and SNPs that were unmapped on barley chromosomes were removed from the analysis. This process resulted in 6,482 high-quality polymorphic SNPs that were distributed across the seven barley chromosomes (Figure S2). For genomic prediction models, SNPs were converted to -1, 0, and +1, where -1 indicates minor allele at a given locus, 0 indicates heterozygous loci, and +1 indicates major allele at a given locus. The additive relationship matrix (<em>K</em>) was estimated using the ‘A.mat’ function in the ‘rrBLUP’ package in R (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Endelman 2011</a>).</p></div><div id="sec-5" class="subsection"><h3>Genomic prediction models and cross-validation schemes</h3><p id="p-11">Genomic prediction models were evaluated with the 145 lines that had genotypic and phenotypic information. Single trait (ST) and multi-trait (MT) models for genomic prediction of agronomic and malting quality traits were obtained following <a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Lado <em>et al.</em> (2018)</a>. In brief, the ST model was estimated using a Bayesian ridge regression model for each trait with 1,500 burn-in and 3,000 iterations for the Gibbs sampler algorithm implemented in the ‘BGLR’ R package (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pérez and de los Campos 2014</a>). The predictive ability was obtained for all agronomic and malting quality traits using the following ST model.<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-5.gif"/></noscript></span></span>where <strong>y</strong> is the vector (<em>n</em> x 1) of phenotype on <em>n</em> genotypes for a single trait; <strong>1</strong> is a vector (<em>n</em> x 1) of ones of length <em>n</em>; <strong>μ</strong> is the overall mean; <strong>Z</strong> is the incidence matrix (<em>n</em> x <em>p</em>) with fixed known values of <em>p</em> markers for these genotypes, <strong>u</strong><sub>(</sub><em><sub>n</sub></em><sub>x1)</sub> is a genotypic predictor with u∼<em>N</em>(0, K<em><sub>n</sub></em><sub>x</sub><em><sub>n</sub></em><span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-6.gif"/></noscript></span></span>), where <strong>K</strong> is the realized additive relationship matrix and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-7.gif"/></noscript></span></span> is additive genetic variance; and <strong>e</strong> is the residual errors vector with e∼<em>N</em>(<strong>0</strong>, <strong>R<em><sub>n</sub></em><sub>x</sub><em><sub>n</sub></em></strong><span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-8.gif"/></noscript></span></span>) where <strong>R</strong> accounts for heterogeneity in mean estimate precision. The predictive abilities for the ST model were estimated using only one cross-validation scheme (CV1) as shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prediction scheme for the single trait (ST) and multi-trait (MT) genomic prediction models with two cross-validation schemes (CV1 and CV2) used in this study. ST-CV1 model: single trait prediction model with cross-validation scheme 1 where a trait (e.g., grain yield; YLD) is predicted at a time; we used 60% of individuals as the training set (phenotyped and genotyped, light green) and 40% of the individuals as the testing set (genotyped only, light blue with PRD code for the trait to be predicted, yield as an example here). MT-CV1 model: multi-trait prediction model with cross-validation scheme 1 for new un-phenotyped individuals; we used 60% of individuals as the training set (phenotyped for all traits and genotyped; light green) and 40% of the individuals as the testing set (genotyped but not phenotyped for any trait; light blue with PRD code for the trait to be predicted, yield as an example here). MT-CV2 model: multi-trait prediction model with cross-validation scheme 2 where 100% of the information from other traits are available for the individuals to be predicted; we used 60% of individuals as the training set (phenotyped for all traits and genotyped; light green) and 40% of the individuals as the testing set (phenotyped for associated traits but not for the targeted traits, and genotyped; light blue with PRD code for the trait to be predicted, yield as an example here). Rectangles represent genotypes and colors represent whether the phenotypic information was used (light green) or not (light blue with PRD code for the trait to be predicted, yield as an example here) for the population. Similar scheme was applied for predicting thousand grain weight (TGW), grain plumpness (PLM), number of grains per square meter (GM2), protein content (GPC), beta-glucan content (BGL), malt extract (EXT), and soluble nitrogen (SNI) where PRD was used for TGW, PLM, GM2, GPC, BGL, EXT and SNI one at a time." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-803357555" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prediction scheme for the single trait (ST) and multi-trait (MT) genomic prediction models with two cross-validation schemes (CV1 and CV2) used in this study. ST-CV1 model: single trait prediction model with cross-validation scheme 1 where a trait (&lt;em&gt;e.g.&lt;/em&gt;, grain yield; YLD) is predicted at a time; we used 60% of individuals as the training set (phenotyped and genotyped, light green) and 40% of the individuals as the testing set (genotyped only, light blue with PRD code for the trait to be predicted, yield as an example here). MT-CV1 model: multi-trait prediction model with cross-validation scheme 1 for new un-phenotyped individuals; we used 60% of individuals as the training set (phenotyped for all traits and genotyped; light green) and 40% of the individuals as the testing set (genotyped but not phenotyped for any trait; light blue with PRD code for the trait to be predicted, yield as an example here). MT-CV2 model: multi-trait prediction model with cross-validation scheme 2 where 100% of the information from other traits are available for the individuals to be predicted; we used 60% of individuals as the training set (phenotyped for all traits and genotyped; light green) and 40% of the individuals as the testing set (phenotyped for associated traits but not for the targeted traits, and genotyped; light blue with PRD code for the trait to be predicted, yield as an example here). Rectangles represent genotypes and colors represent whether the phenotypic information was used (light green) or not (light blue with PRD code for the trait to be predicted, yield as an example here) for the population. Similar scheme was applied for predicting thousand grain weight (TGW), grain plumpness (PLM), number of grains per square meter (GM2), protein content (GPC), beta-glucan content (BGL), malt extract (EXT), and soluble nitrogen (SNI) where PRD was used for TGW, PLM, GM2, GPC, BGL, EXT and SNI one at a time.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1113/F1.medium.gif" width="401" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/3/1113/F1.medium.gif" width="401" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495912" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">Prediction scheme for the single trait (ST) and multi-trait (MT) genomic prediction models with two cross-validation schemes (CV1 and CV2) used in this study. ST-CV1 model: single trait prediction model with cross-validation scheme 1 where a trait (<em>e.g.</em>, grain yield; YLD) is predicted at a time; we used 60% of individuals as the training set (phenotyped and genotyped, light green) and 40% of the individuals as the testing set (genotyped only, light blue with PRD code for the trait to be predicted, yield as an example here). MT-CV1 model: multi-trait prediction model with cross-validation scheme 1 for new un-phenotyped individuals; we used 60% of individuals as the training set (phenotyped for all traits and genotyped; light green) and 40% of the individuals as the testing set (genotyped but not phenotyped for any trait; light blue with PRD code for the trait to be predicted, yield as an example here). MT-CV2 model: multi-trait prediction model with cross-validation scheme 2 where 100% of the information from other traits are available for the individuals to be predicted; we used 60% of individuals as the training set (phenotyped for all traits and genotyped; light green) and 40% of the individuals as the testing set (phenotyped for associated traits but not for the targeted traits, and genotyped; light blue with PRD code for the trait to be predicted, yield as an example here). Rectangles represent genotypes and colors represent whether the phenotypic information was used (light green) or not (light blue with PRD code for the trait to be predicted, yield as an example here) for the population. Similar scheme was applied for predicting thousand grain weight (TGW), grain plumpness (PLM), number of grains per square meter (GM2), protein content (GPC), beta-glucan content (BGL), malt extract (EXT), and soluble nitrogen (SNI) where PRD was used for TGW, PLM, GM2, GPC, BGL, EXT and SNI one at a time.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">The MT model was built using a Bayesian multivariate Gaussian model estimating an unstructured variance-covariance matrix between traits (<strong>Σ</strong>) and residual matrix (<strong>R</strong>) with 1,500 burn-in and 3,000 iterations for the Gibbs sample algorithm implemented in the ‘MTM’ R package (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos and Grüneberg 2016</a>) as described by <a id="xref-ref-34-9" class="xref-bibr" href="#ref-34">Lado <em>et al.</em> (2018)</a>. The model used was:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1113/embed/mml-math-9.gif"/></noscript></span></span>where <strong>y</strong> is a vector of <em>n</em> x <em>t</em> length (<em>n</em> genotypes and <em>t</em> traits); <strong>1</strong> is a vector of ones of length <em>n</em>x<em>t</em>; <strong>μ</strong> is the overall mean; <strong>Z</strong><sub>[(<em>n</em>x<em>t</em>)xp]</sub> is the incidence matrix; <strong>u</strong><sub>[</sub><em><sub>n</sub></em><sub>x</sub><em><sub>t</sub></em><sub>)x1]</sub> is a genotypic predictor with <strong>u</strong> ∼ MVN(0, <strong>Σ</strong>⊗<strong>K</strong>) and <strong>e</strong> is the residual errors vector with <strong>e</strong> ∼ MVN(<strong>0</strong>, <strong>R</strong>⊗<strong>I</strong>), where <strong>Σ</strong> is the variance-covariance matrix across traits, <strong>K</strong> is the realized additive relationship matrix among individuals estimated from the markers, <strong>R</strong> is the variance-covariance matrix for the residual effects for each individual in all traits, and I is the <em>n</em>x<em>n</em> identify matrix. <strong>Σ</strong> was estimated as an unstructured matrix and <strong>R</strong> as a diagonal matrix following <a id="xref-ref-34-10" class="xref-bibr" href="#ref-34">Lado <em>et al.</em> (2018)</a>. The predictive abilities for the MT model were estimated using two cross-validation schemes (CV1 and CV2) as described in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>.</p><p id="p-14">Two genomic prediction models (ST and MT) were used to compare two main cross-validation schemes described previously (<a id="xref-ref-34-11" class="xref-bibr" href="#ref-34">Lado <em>et al.</em>, 2018</a>) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Briefly, the first cross-validation scheme (CV1) used a random set of lines (60%; ∼90) with phenotypic and genotypic information (training set) to train the model and the remaining lines (testing set; 40%; ∼60) for prediction using genotypic information only. This process was repeated 100 times, where each iteration included different lines in the training and testing sets. The second cross-validation scheme (CV2) used a random set of lines (60%) with phenotypic and genotypic information for the trait of interest and other multiple traits (<em>i.e.</em>, correlated traits) to train the model. The remaining lines (40%) were predicted for the trait of interest using genotypic and phenotypic information from other correlated traits. This process was repeated 100 times where each iteration included different lines in the training and testing sets. The first cross-validation scheme was used for the ST (designated as ST-CV1) and MT (MT-CV1) models, whereas the second cross-validation scheme was used only for the MT (MT-CV2) model. In short, the CV2 scheme uses full genotypic information on every individual (training and prediction sets), full phenotypic information for correlated traits on every individual (training and prediction sets), and phenotypic information of trait of interest only on the training set. Therefore, CV2 only applies to multiple traits problems. Pearson’s correlation (<em>r<sub>PA</sub></em>) between the genotypic values and genotypic BLUEs was estimated to determine the predictive ability of the models.</p><p id="p-15">Genomic prediction models for agronomic traits were tested using the BLUEs from an individual experiment, experiments across locations (EELE, EEMAC, MOSA, and MUSA), years (2015 and 2016), and all combined (designated as ‘ALL’). Similarly, genomic prediction models for malting quality traits were tested using BLUEs from individual experiments conducted in 2015 (EELE15, EEMAC15, MOSA15, and MUSA15), experiments combined across locations in 2015 (‘ALL15’) and using agronomic trait information from all nine experiments combined (ALL). Agronomic traits used in the genomic prediction models were grain yield, thousand grain weight, number of grains per square meter, and grain plumpness. For the MT model, two strategies were used for predicting agronomic traits of interest. The first strategy included predicting the agronomic trait of interest using all other three agronomic traits (AGRO). The second strategy included predicting the agronomic trait of interest using the remaining three agronomic traits and all four malting quality traits (A+M). Similarly, five strategies were used for predicting malting quality trait of interest as follows: (i) all four agronomic traits (AGRO), (ii) both agronomic and malting traits (A+M), (iii) positively correlated traits with grain protein content (COR1; beta-glucan content, grain plumpness, soluble nitrogen, and grain protein content), and (iv) traits explaining the largest variance in the first principal component (PC1 or COR2: grain yield, thousand grain weight, number of grains per square meter, and grain protein content); and (v) traits explaining the largest variance on the second principal component (PC2 or COR3: beta-glucan content, grain plumpness, soluble nitrogen, and malt extract).</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-16">File S1 contains supplementary Tables S1-S6 and Figures S1-S6. Table S1 contains basic summary statistics with BLUEs, standard error, minimum, maximum, coefficient of variation, broad sense heritability, and proportion of variance component for genotype by the environment and genotypic effect for agronomic and malting quality traits evaluated across multiple locations and years. Table S2 contains an analysis of variance with mean squares for agronomic and malting quality traits. Table S3 contains genomic predictive ability for un-phenotyped environments using MT-CV1 and MT-CV2 models. Table S4-S6 contains predictive ability for agronomic and malting quality traits using MT-CV1 and MT-CV2 models. Figure S1 shows a principal component analysis of all individuals. Figure S2 contains a distribution of the 6,482 single nucleotide polymorphisms (SNPs) across the seven chromosomes of barley in 145 doubled haploid lines. Figures S3-S6 contain Pearson’s correlation among environments (locations-years) for grain yield, number of grains per square meter, grain plumpness, and thousand grain weight, respectively. File S2 contains the BLUEs for the agronomic traits (grain yield, plumpness, thousand grain weight, and number of grains per square meter) from individual experiments (EELE15, EELE16, EEMAC15, EEMAC16, EEMAC17, MOSA15, MOSA16, MUSA15, and MUSA16), experiments across locations (EELE, EEMAC, MOSA, and MUSA), years (2015, 2016, and 2017), and all experiments combined (ALL). File S3 contains the BLUEs for the malting quality traits (beta-glucan content, malt extract, protein content, and soluble nitrogen) from the experiments conducted in 2015 in three locations (EELE, EEMAC, and MOSA) and for the combined experiments (ALL15). Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9172619">https://doi.org/10.25387/g3.9172619</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Multi-trait characterization</h3><p id="p-17">Basic summary statistics and analysis of variance of the agronomic and malting quality traits are in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Tables 1</a>, S1, and S2. A wide range of variation was observed for agronomic and malting quality traits across environments (EELE, EEMAC, MOSA, and MUSA). The highest mean grain yield averaged over years was observed at MOSA (7,658 kg ha<sup>-1</sup>), followed by EELE (7,337 kg ha<sup>-1</sup>), EEMAC (5,134 kg ha<sup>-1</sup>), and MUSA (3,877 kg ha<sup>-1</sup>). Similarly, a wide variation across environments was observed for other traits.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495962/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495962/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Best linear unbiased predictors (BLUEs) and standard deviation (in parenthesis) for each environment (<em>i.e.</em>, combination of location-year) evaluated for agronomic and malting quality traits in in Uruguay</span><div class="sb-div caption-clear"></div></div></div><p id="p-18">Variance component analysis showed that the proportion of the variance explained by the GEI effects to genotypic effects was greater than one for most of the agronomic traits whereas lower than one for malting quality traits (Table S1). Broad-sense heritability estimates for agronomic traits were low to high ranging from 0.20 to 0.80 (Table S1). Low to moderate heritability was observed for grain yield (0.20 to 0.66) and plumpness (0.25 to 0.53) whereas moderate to high heritability was observed for number of grains per square meter (0.46 to 0.71) and thousand grain weight (0.42 to 0.80). Relatively high heritability was observed for malting quality traits ranging from 0.66 (beta-glucan content) to 0.74 (soluble nitrogen) (Table S1).</p><p id="p-19">Grain yield was positively correlated with plumpness and the number of grains per square meter, negatively correlated with grain protein content, and not correlated with grain weight (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Grain protein content affects the malting quality of barley as indicated by the positive correlation with beta-glucan content, soluble nitrogen, and grain plumpness, and negative correlation with malt extract. Additionally, grain protein content was correlated with both yield components (positive correlation with grain weight and negative correlation with the number of grains). The correlation values for all agronomic traits were mostly higher among locations than years (Figures S3-S6).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic correlation between agronomic and malting quality traits using best linear unbiased estimates from combining three experiments (EEMAC, EELE, and MOSA) conducted in 2015. Numbers in bold indicate a correlation significantly different from zero at an alpha level of 0.05. BGL, beta-glucan content; EXT, malt extract; GM2, number of grain m-2; PLM, grain plumpness; TGW, thousand grain weight; GPC, grain protein content; SNI, soluble nitrogen; and YLD, grain yield." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-803357555" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic correlation between agronomic and malting quality traits using best linear unbiased estimates from combining three experiments (EEMAC, EELE, and MOSA) conducted in 2015. Numbers in bold indicate a correlation significantly different from zero at an alpha level of 0.05. BGL, beta-glucan content; EXT, malt extract; GM2, number of grain m&lt;sup&gt;-2&lt;/sup&gt;; PLM, grain plumpness; TGW, thousand grain weight; GPC, grain protein content; SNI, soluble nitrogen; and YLD, grain yield.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1113/F2.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/3/1113/F2.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495936" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Genetic correlation between agronomic and malting quality traits using best linear unbiased estimates from combining three experiments (EEMAC, EELE, and MOSA) conducted in 2015. Numbers in bold indicate a correlation significantly different from zero at an alpha level of 0.05. BGL, beta-glucan content; EXT, malt extract; GM2, number of grain m<sup>-2</sup>; PLM, grain plumpness; TGW, thousand grain weight; GPC, grain protein content; SNI, soluble nitrogen; and YLD, grain yield.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">In the principal component analysis (PCA), the first two PCs explained ∼52% of the total variation (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The PC1 was mostly explained by yield component traits, such as grain yield, protein content, number of grains per square meter, and thousand grain weight. The PC2 was mostly explained by malting quality traits such as beta-glucan content, malt extract, grain plumpness, and soluble nitrogen.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Principal component analysis showing the association between agronomic and malting quality traits. BGL, beta-glucan content; EXT, malt extract; GM2, number of grains per square meter; PLM, grain plumpness; TGW, thousand grain weight; GPC, grain protein content; SNI, soluble nitrogen; and YLD, grain yield. PVE: Proportion of the variance explained by each trait." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-803357555" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Principal component analysis showing the association between agronomic and malting quality traits. BGL, beta-glucan content; EXT, malt extract; GM2, number of grains per square meter; PLM, grain plumpness; TGW, thousand grain weight; GPC, grain protein content; SNI, soluble nitrogen; and YLD, grain yield. PVE: Proportion of the variance explained by each trait.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1113/F3.medium.gif" width="440" height="380"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/3/1113/F3.medium.gif" width="440" height="380"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495897" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Principal component analysis showing the association between agronomic and malting quality traits. BGL, beta-glucan content; EXT, malt extract; GM2, number of grains per square meter; PLM, grain plumpness; TGW, thousand grain weight; GPC, grain protein content; SNI, soluble nitrogen; and YLD, grain yield. PVE: Proportion of the variance explained by each trait.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Genomic prediction for agronomic traits</h3><p id="p-23">The model (MT-CV2) that included other agronomic traits from individuals to be predicted performed better for agronomic traits than either single (ST-CV1) or multi-trait (MT-CV1) models without this information (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and Table S4). For instance, the predictive ability for grain yield using the MT-CV2 model was high ranging from 0.34 to 0.70 for most environments. Low predictive ability was found for MUSA16 and MUSA combined. The MUSA16 environment, where agronomic traits were weakly correlated, was an outlier. The predictive ability was high for all agronomic traits ranging from 0.56 to 0.60 using the MT-CV2 model when all experiments were combined (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and Table S4).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box plots showing the predictive ability for grain yield (YLD), plumpness (PLM), thousand grain weight (TGW), and number of grains per square meter (GM2) using single trait (ST) and multi-trait (MT) models from individual experiments (EELE15, EELE16, EEMAC15, EEMAC16, EEMAC17, MOSA15, MOSA16, MUSA15, and MUSA16), experiments across locations (EELE, EEMAC, MOSA, and MUSA), years (2015, 2016, and 2017), and all experiments combined (ALL). CV1, predicting new lines with genotypic information only and CV2, predicting partially phenotyped lines by using genotypic and phenotypic information from all traits from individuals in the training set, and genotypic and correlated phenotypic traits in the testing set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-803357555" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box plots showing the predictive ability for grain yield (YLD), plumpness (PLM), thousand grain weight (TGW), and number of grains per square meter (GM2) using single trait (ST) and multi-trait (MT) models from individual experiments (EELE15, EELE16, EEMAC15, EEMAC16, EEMAC17, MOSA15, MOSA16, MUSA15, and MUSA16), experiments across locations (EELE, EEMAC, MOSA, and MUSA), years (2015, 2016, and 2017), and all experiments combined (ALL). CV1, predicting new lines with genotypic information only and CV2, predicting partially phenotyped lines by using genotypic and phenotypic information from all traits from individuals in the training set, and genotypic and correlated phenotypic traits in the testing set.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1113/F4.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/3/1113/F4.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495965" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">Box plots showing the predictive ability for grain yield (YLD), plumpness (PLM), thousand grain weight (TGW), and number of grains per square meter (GM2) using single trait (ST) and multi-trait (MT) models from individual experiments (EELE15, EELE16, EEMAC15, EEMAC16, EEMAC17, MOSA15, MOSA16, MUSA15, and MUSA16), experiments across locations (EELE, EEMAC, MOSA, and MUSA), years (2015, 2016, and 2017), and all experiments combined (ALL). CV1, predicting new lines with genotypic information only and CV2, predicting partially phenotyped lines by using genotypic and phenotypic information from all traits from individuals in the training set, and genotypic and correlated phenotypic traits in the testing set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">To identify whether the inclusion of additional traits such as malting quality traits along with the agronomic traits in an MT model could further improve the predictive performance of agronomic traits of interest, we evaluated a multi-trait (MT-CV2) prediction model including both agronomic and malting (A+M) traits and compared it with the MT model including agronomic (AGRO) traits only (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and Table S5). The predictive ability for the agronomic trait of interest was further increased from using both A+M traits compared to the model with AGRO traits only. For instance, the mean predictive ability of the MT-CV2 model at all combined experiments in 2015 for grain yield using only agronomic traits was 0.57 compared to 0.77 when using both agronomic and malting traits.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplots showing the predictive ability for grain yield (YLD), plumpness (PLM), thousand grain weight (TGW), and number of grains square meter (GM2) using single trait (ST) and multi-trait (MT) models from 2015 experiments conducted in three locations (EELE, EEMAC, and MOSA). ALL15, three combined experiments of the year 2015; CV2, predicting partially phenotyped lines by using genotypic and phenotypic information from all traits from individuals in the training set, and genotypic and phenotypic information from other correlated traits in the testing set.; The correlated traits were: AGRO, agronomic traits (YLD, PLM, TGW, and GM2); or A+M, agronomic and malting quality traits (beta-glucan content, malt extract, protein content, and soluble nitrogen)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-803357555" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Boxplots showing the predictive ability for grain yield (YLD), plumpness (PLM), thousand grain weight (TGW), and number of grains square meter (GM2) using single trait (ST) and multi-trait (MT) models from 2015 experiments conducted in three locations (EELE, EEMAC, and MOSA). ALL15, three combined experiments of the year 2015; CV2, predicting partially phenotyped lines by using genotypic and phenotypic information from all traits from individuals in the training set, and genotypic and phenotypic information from other correlated traits in the testing set.; The correlated traits were: AGRO, agronomic traits (YLD, PLM, TGW, and GM2); or A+M, agronomic and malting quality traits (beta-glucan content, malt extract, protein content, and soluble nitrogen).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1113/F5.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/3/1113/F5.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495920" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-26" class="first-child">Boxplots showing the predictive ability for grain yield (YLD), plumpness (PLM), thousand grain weight (TGW), and number of grains square meter (GM2) using single trait (ST) and multi-trait (MT) models from 2015 experiments conducted in three locations (EELE, EEMAC, and MOSA). ALL15, three combined experiments of the year 2015; CV2, predicting partially phenotyped lines by using genotypic and phenotypic information from all traits from individuals in the training set, and genotypic and phenotypic information from other correlated traits in the testing set.; The correlated traits were: AGRO, agronomic traits (YLD, PLM, TGW, and GM2); or A+M, agronomic and malting quality traits (beta-glucan content, malt extract, protein content, and soluble nitrogen).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Genomic prediction for malting quality traits</h3><p id="p-27">A substantial increase in the predictive ability of the malting quality trait of interest was observed using the MT-CV2 model containing A+M and other correlated traits (COR1, COR2, and COR3) in the prediction model for both individual and combined experiments (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and Table S6). Additionally, the predictive ability of grain protein content was increased using the MT-CV2 model containing only agronomic traits. However, the predictive ability of ST-CV1, MT-CV1, and MT-CV2 models was similar when we used only the agronomic traits to predict other malting quality traits. The predictive ability of the MT-CV2 model that uses both agronomic and malting quality traits had the highest predictive ability for malting quality traits followed by the models that included correlated traits. For instance, the predictive abilities for grain protein content using MT-CV2 models with A+M traits ranged from 0.25 to 0.46. Similarly, the predictive abilities for grain protein content using MT-CV2 that included correlated traits ranged from 0.07 to 0.38. The grain protein predictive ability at all combined experiments using the MT-CV2 model with A+M had 76.2% and 75.8% higher predictive ability than using the MT-CV1 model with A+M and ST-CV1 models, respectively.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box plots showing the predictive ability for malting quality traits using single trait (ST) and multi-trait (MT) prediction models for the experiments conducted in 2015 in three locations (EELE, EEMAC, and MOSA) and for the combined experiments in 2015 (ALL15) and all (ALL). ALL, agronomic traits from all nine experiments combined and malting traits from the ALL15; CV1, predicting new lines with genotypic information only; and CV2, predicting partially phenotyped lines by using genotypic and phenotypic information from all traits from individuals in the training set, and genotypic and phenotypic information from other correlated traits in the testing set. Predicted traits were: beta-glucan content; malt extract; protein content; and soluble nitrogen; ST represents single trait, AGRO, agronomic traits (grain yield, plumpness, thousand grain weight, and number of grains per square meter) and trait of interest for multi-trait prediction model; A+M, agronomic and malting quality traits; COR1, denotes correlated traits which are soluble nitrogen, thousand grain weight, beta-glucan content, plumpness, and grain protein content; COR2, denotes traits within principal component 1 (PC1), which are grain yield, thousand grain weight, number of grains per square meter, and grain protein content; COR3, traits within PC2 which are beta-glucan content, plumpness, soluble nitrogen, and extract. Treatment combinations were shown in a 5 &#xD7; 8 rectangle figure where each small rectangle with colors represent the presence (black) or absence (white) of phenotypic information used in the prediction model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-803357555" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box plots showing the predictive ability for malting quality traits using single trait (ST) and multi-trait (MT) prediction models for the experiments conducted in 2015 in three locations (EELE, EEMAC, and MOSA) and for the combined experiments in 2015 (ALL15) and all (ALL). ALL, agronomic traits from all nine experiments combined and malting traits from the ALL15; CV1, predicting new lines with genotypic information only; and CV2, predicting partially phenotyped lines by using genotypic and phenotypic information from all traits from individuals in the training set, and genotypic and phenotypic information from other correlated traits in the testing set. Predicted traits were: beta-glucan content; malt extract; protein content; and soluble nitrogen; ST represents single trait, AGRO, agronomic traits (grain yield, plumpness, thousand grain weight, and number of grains per square meter) and trait of interest for multi-trait prediction model; A+M, agronomic and malting quality traits; COR1, denotes correlated traits which are soluble nitrogen, thousand grain weight, beta-glucan content, plumpness, and grain protein content; COR2, denotes traits within principal component 1 (PC1), which are grain yield, thousand grain weight, number of grains per square meter, and grain protein content; COR3, traits within PC2 which are beta-glucan content, plumpness, soluble nitrogen, and extract. Treatment combinations were shown in a 5 &#xD7; 8 rectangle figure where each small rectangle with colors represent the presence (black) or absence (white) of phenotypic information used in the prediction model.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1113/F6.medium.gif" width="401" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/3/1113/F6.medium.gif" width="401" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1113/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495947" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-28" class="first-child">Box plots showing the predictive ability for malting quality traits using single trait (ST) and multi-trait (MT) prediction models for the experiments conducted in 2015 in three locations (EELE, EEMAC, and MOSA) and for the combined experiments in 2015 (ALL15) and all (ALL). ALL, agronomic traits from all nine experiments combined and malting traits from the ALL15; CV1, predicting new lines with genotypic information only; and CV2, predicting partially phenotyped lines by using genotypic and phenotypic information from all traits from individuals in the training set, and genotypic and phenotypic information from other correlated traits in the testing set. Predicted traits were: beta-glucan content; malt extract; protein content; and soluble nitrogen; ST represents single trait, AGRO, agronomic traits (grain yield, plumpness, thousand grain weight, and number of grains per square meter) and trait of interest for multi-trait prediction model; A+M, agronomic and malting quality traits; COR1, denotes correlated traits which are soluble nitrogen, thousand grain weight, beta-glucan content, plumpness, and grain protein content; COR2, denotes traits within principal component 1 (PC1), which are grain yield, thousand grain weight, number of grains per square meter, and grain protein content; COR3, traits within PC2 which are beta-glucan content, plumpness, soluble nitrogen, and extract. Treatment combinations were shown in a 5 × 8 rectangle figure where each small rectangle with colors represent the presence (black) or absence (white) of phenotypic information used in the prediction model.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Genomic prediction in un-phenotyped environment</h3><p id="p-29">Genomic prediction for a location using the information from other years (<em>e.g.</em>, predict 2017 from 2015 and 2016 information and vice versa) had relatively lower predictive ability compared to the prediction for a location using information from other locations (<em>e.g.</em>, predict EELE from using EEMAC, MOSA, and MUSA) (Table S3). In general, genomic prediction using the MT-CV2 model had a higher predictive ability for all traits compared to the MT-CV1 model for unobserved environments. For instance, grain yield predictive ability across locations using the MT-CV1 was up to 0.26 and the MT-CV2 model was up to 0.41. For prediction across years, the MT-CV1 model was up to 0.27 and the MT-CV2 was up to 0.32. The highest predictive abilities using MT-CV2 for grain plumpness, thousand grain weight, and number of grains per square meter were 0.52, 0.53, 0.52, and 0.46, respectively. For locations, on average, the highest predictive ability was observed at EEMAC for grain yield, MOSA for plumpness, EELE for thousand grain weight, and MUSA for number of grains per sq. m using the MT-CV2 model.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-30">Plant breeders routinely collect multiple traits for evaluating the performance of genotypes in multiple environments. The selection of genotypes based on genome-wide markers and phenotypic information using genomic selection methods is gaining popularity in breeding programs with the advent of cost-effective and robust next-generation sequencing technologies (Poland and Rife 2012). However, only a few studies have been conducted using multi-trait multi-environment genomic selection methods due to complexity of model and requirements of high computing power (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Calus and Veerkamp 2011</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jia and Jannink 2012</a>; Rutkoski <em>et al.</em>, 2015; <a id="xref-ref-34-12" class="xref-bibr" href="#ref-34">Lado <em>et al.</em>, 2018</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ward <em>et al.</em>, 2019</a>). The present study investigated the different strategies for including multiple traits in multi-environment trials for predicting un-phenotyped or partially phenotyped lines.</p><div id="sec-13" class="subsection"><h3>Trait characterization and association</h3><p id="p-31">The first thing to do before utilizing multiple traits in a genomic prediction model is to understand the characteristics of the traits used. This study evaluated four agronomic traits (grain yield, thousand grain weight, grain plumpness, and number of grains per square meters) and four malting quality traits (beta-glucan content, malt extract, protein content, and soluble nitrogen) for use in a multi-trait genomic selection model. The proportion of variance explained by GEI was larger for agronomic traits than malting traits (Table S1), suggesting a large influence of GEI effects on agronomic traits compared to quality traits. The larger influence of GEI on agronomic traits resulted in a lower heritability for agronomic traits compared to quality traits. Lower heritability estimates for agronomic traits such as grain yield was expected as they are governed by many genes and are genetically complex in nature (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Thorwarth <em>et al.</em>, 2017</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bhatta <em>et al.</em>, 2018b</a>). Similar results were observed for agronomic and quality traits in previous studies (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rode <em>et al.</em>, 2012</a>; <a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Sallam <em>et al.</em>, 2015</a>; <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Thorwarth <em>et al.</em>, 2017</a>).</p><p id="p-32">For the MT model for agronomic traits, two sets of traits (set 1: grain yield and number of grains per square meter; and set 2: plumpness and thousand grain weight) were strongly positively correlated to each other. The positive correlation of grain yield with number of grains per square meter and no correlation with grain weight in the current study indicates that the grain yield was largely defined by the number of grains per square meter component. The negative relationship between grain yield and protein content identified in this study is well understood in cereal crops and is mainly associated with the dilution effect (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bertholdsson 1999</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bhatta <em>et al.</em>, 2018a</a>). Similarly, the positive association of thousand grain weight and a negative association of malt extract with grain protein content have been reported previously in malting barley cultivars (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bertholdsson 1999</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Dai <em>et al.</em>, 2007</a>).</p></div><div id="sec-14" class="subsection"><h3>Multi-trait genomic prediction models and their implications in a breeding program</h3><p id="p-33">Increasing genomic predictive ability of quantitative and complex traits such as grain yield and malting quality traits are primary goals for successful utilization of genomic selections in breeding programs (<a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Sallam <em>et al.</em>, 2015</a>; <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Lado <em>et al.</em>, 2016</a>, <a id="xref-ref-34-13" class="xref-bibr" href="#ref-34">2018</a>). In this study, we have evaluated eight traits with different range of heritabilities and correlations to investigate strategies for integrating multiple traits into the prediction model. The main aim of the study was to determine whether incorporating multiple agronomic traits or both agronomic and malting quality traits into the multi-trait models (MT-CV1 and MT-CV2) could further improve the predictive ability of a trait of interest compared to single trait (ST-CV1) model. The present study suggests little to no advantage of the MT-CV1 model, irrespective of different multiple traits inclusion strategies, compared to the ST-CV1 model when predicting new genotypes. Newly introduced genotypes are untested for multiple traits across environments and using the MT-CV1 under this scenario may not provide additional information to leverage for increasing predictive ability (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burgueño <em>et al.</em>, 2012</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Ward <em>et al.</em>, 2019</a>). Also, the predictive ability using MT-CV1 and ST-CV1 was low for grain yield (0.07 to 0.17 for MT-CV1 and 0.05 to 0.15 for ST-CV1). Similar results were observed in previous studies where the MT-CV1 model that included correlated traits did not perform better than the ST-CV1 model (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Calus and Veerkamp 2011</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Schulthess <em>et al.</em>, 2016</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dos Santos <em>et al.</em>, 2016</a>; <a id="xref-ref-34-14" class="xref-bibr" href="#ref-34">Lado <em>et al.</em>, 2018</a>), demonstrating that the multi-trait models are not necessarily advantageous for new individuals compared to the single-trait model. This result may be associated with the trait complexity and wide range of heritability of the multiple traits used (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Jia and Jannink 2012</a>; <a id="xref-ref-34-15" class="xref-bibr" href="#ref-34">Lado <em>et al.</em>, 2018</a>). However, a few studies have identified considerable increase in predictive ability due to the inclusion of correlated traits with high heritability in MT-CV1 models (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Rutkoski <em>et al.</em>, 2012</a>; <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Jia and Jannink 2012</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Guo <em>et al.</em>, 2014b</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Montesinos-López <em>et al.</em>, 2016</a>), but the benefit of MT-CV1 model for complex trait is minimal compared to the ST-CV1 model (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Jia and Jannink 2012</a>).</p><p id="p-34">The MT-CV1 and ST-CV1 models did not show differences, it is, therefore, similar to compare ST-CV1 to MT-CV2. The MT-CV2 model for all studied traits significantly improved the predictive ability for the trait of interest compared to MT-CV1 and ST-CV1 models. Additionally, the MT-CV2 model had higher predictive abilities for all agronomic traits (0.56 to 0.60) across all combined experiments. For instance, the MT-CV2 model for grain yield across all combined experiments increased the predictive ability by 57% and 61% compared to the ST-CV1 and MT-CV1 models, respectively. This may be associated with the higher heritability estimates of combined environments compared to individual environments. The higher predictive ability in genomic prediction models for a trait is expected when the heritability is higher (<a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">Jia and Jannink 2012</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Guo <em>et al.</em>, 2014b</a>; <a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Thorwarth <em>et al.</em>, 2017</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wang <em>et al.</em> 2017</a>). This result suggested that the MT-CV2 model can provide fairly high genomic predictive performance for a complex trait such as grain yield for multi-environment trials when high heritability and correlated traits are used. Similarly, <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Wang <em>et al.</em> (2017)</a> have also reported a higher predictive ability for grain yield when using the multi-trait model in rice. A genomic prediction study in 750 winter barley genotypes identified low predictive ability for grain yield (0.31) and high predictive ability for thousand grain weight (0.71) (<a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">Thorwarth <em>et al.</em>, 2017</a>) comparable to the predictive abilities of the MT-CV2 model in our study. This result suggested that the MT-CV2 model could be useful in predicting traits after partially phenotyping complex traits such as grain yield. Furthermore, the predictive ability of agronomic trait of interest using the MT-CV2 model was improved using both agronomic and malting traits compared to using agronomic traits only. For example, the MT-CV2 model with both agronomic and malting traits for grain yield across all combined experiments in 2015 increased predictive ability by 26% compared to the MT-CV2 model with agronomic traits only. Evaluating malting quality traits for predicting the agronomic traits may not have a practical implication on a breeding program, however, the most interesting aspect is that if researchers are evaluating malting quality traits for their study, using this information will improve the yield prediction. This result showed the potential strategy for further improving complex traits such as grain yield via the inclusion of a larger number of highly heritable and correlated traits. On the other hand, the predictive ability of the MT-CV2 model that included weakly correlated traits such as grain yield and plumpness at MUSA and MUSA16 had similar predictive ability compared to MT-CV1 and ST-CV1 models. This result was expected due to the fact that uncorrelated/weakly correlated traits have rarely useful information from the correlation matrix that goes into the multi-trait prediction model (<a id="xref-ref-29-7" class="xref-bibr" href="#ref-29">Jia and Jannink 2012</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Montesinos-López <em>et al.</em>, 2016</a>; <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Wang <em>et al.</em>, 2017</a>; <a id="xref-ref-34-16" class="xref-bibr" href="#ref-34">Lado <em>et al.</em> 2018</a>), thereby, no additional improvement in the predictive ability was observed.</p><p id="p-35">Malting quality traits are expensive, time-consuming, labor-intensive, and difficult to breed due to their complex genetic architecture (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gutiérrez <em>et al.</em>, 2011</a>). Nevertheless, in malting barley production, they are generally the key and final factor determining if an experimental line can be developed into a commercial variety. Using information from correlated traits could increase the predictive performance of the trait of interest (<a id="xref-ref-29-8" class="xref-bibr" href="#ref-29">Jia and Jannink 2012</a>; <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Wang <em>et al.</em>, 2017</a>). In this study, the highest predictive ability for malting quality traits was observed when using the MT-CV2 model including both agronomic and malting quality traits followed by the model including some correlated traits. These results suggested that the prediction of quality traits in genomic selection could be improved by borrowing information from correlated agronomic traits. A recent study on multi-trait genomic prediction in bread baking quality in 495 advanced wheat lines has reported increased predictive ability from the MT-CV2 model that included one highly correlated trait compared to MT-CV1 and ST-CV1 models (<a id="xref-ref-34-17" class="xref-bibr" href="#ref-34">Lado <em>et al.</em>, 2018</a>). Several other studies have also reported advantages of the multi-trait prediction model for predicting traits of partially phenotyped individuals (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Rutkoski <em>et al.</em>, 2012</a>; <a id="xref-ref-29-9" class="xref-bibr" href="#ref-29">Jia and Jannink 2012</a>; <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Guo <em>et al.</em>, 2014b</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Sun <em>et al.</em>, 2017</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Hayes <em>et al.</em>, 2017</a>). This result suggested that the MT-CV2 model that included correlated traits should be used as a strategic approach to replace phenotyping of labor-intensive and high-cost traits such as malting traits. However, the predictive abilities of ST-CV1, MT-CV1, and MT-CV2 models were similar when we used only the agronomic traits that were uncorrelated with malting quality traits to predict malting quality trait of interest.</p><p id="p-36">Additionally, this study showed that the predictive ability using the MT-CV2 model could be further increased with the increase in the number of associated traits included in the prediction model. Similarly, a previous study on 575 rice hybrids had reported an increase in the predictive ability with the increase in the number of multiple traits included in the prediction model (<a id="xref-ref-58-5" class="xref-bibr" href="#ref-58">Wang <em>et al.</em>, 2017</a>). However, other studies have shown that the contributions of multiple traits in the model would tend asymptotically toward zero, and additional traits could introduce issues in co-linearity; and therefore, fewer traits have been recommended (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Schulthess <em>et al.</em>, 2016</a>; <a id="xref-ref-34-18" class="xref-bibr" href="#ref-34">Lado <em>et al.</em>, 2018</a>; <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Lozada and Carter 2019</a>).</p><p id="p-37">Similarly, genomic prediction in un-phenotyped environments using the MT-CV2 model outperformed the MT-CV1 model. This result suggests that the MT-CV2 model should be used for predicting the performance of genotypes in new environments when phenotyping was performed for other traits. On average, predicting a location using other location data resulted in a higher prediction compared to predicting across the year. This result may be associated with higher correlations among locations compared to correlations among years where high GEI was observed. <a id="xref-ref-58-6" class="xref-bibr" href="#ref-58">Wang <em>et al.</em> (2017)</a> had observed a higher predictive ability of the MT multi-environment model compared to the ST single environment model because MT multi-traits models can borrow information from associated traits across environments. This study showed the usefulness of the MT-CV2 model in predicting a new environment despite the presence of GEI.</p><p id="p-38">However, there are several factors affecting the predictive abilities of MT multi-environment genomic prediction models. Some of the important factors associated with this study were the size and composition of the training population (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em>, 2013</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Zhang <em>et al.</em>, 2016</a>; <a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Duangjit <em>et al.</em>, 2016</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Berro <em>et al.</em>, 2019</a>), the genetic relationship between the training and testing population (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em>, 2014</a>; <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Duangjit <em>et al.</em>, 2016</a>; <a id="xref-ref-56-8" class="xref-bibr" href="#ref-56">Thorwarth <em>et al.</em>, 2017</a>), trait genetic architecture and heritability (<a id="xref-ref-50-6" class="xref-bibr" href="#ref-50">Sallam <em>et al.</em>, 2015</a>; <a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">Duangjit <em>et al.</em>, 2016</a>; <a id="xref-ref-58-7" class="xref-bibr" href="#ref-58">Wang <em>et al.</em>, 2017</a>; <a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Lozada and Carter 2019</a>), trait correlations (<a id="xref-ref-36-6" class="xref-bibr" href="#ref-36">Lozada and Carter 2019</a>), correlations among environments (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Lado <em>et al.</em>, 2016</a>), and GEI (Heslot <em>et al.</em>, 2014; <a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Lado <em>et al.</em>, 2016</a>). Although the population size in our study was small, the predictive ability using correlated traits in the CV2 scheme had high PA, suggesting that correlated traits could somewhat offset the effect of smaller population sizes.</p><p id="p-39">Although poor predictive ability remains a major challenge in implementing genomic selection (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em>, 2014</a>), several studies have shown that genomic selection could be advantageous for complex traits with low heritability such as grain yield (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Belamkar <em>et al.</em>, 2018</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Burgueño <em>et al.</em>, 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lozada <em>et al.</em>, 2019</a>). Genomic selection with low to moderate levels of predictive ability could be used in early-generation testing and selection (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Belamkar <em>et al.</em>, 2018</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Juliana <em>et al.</em>, 2018</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Michel <em>et al.</em>, 2018</a>) and in off-season nurseries where field phenotypic information might be useless. Breeding program do not rely on predictive ability itself, but also on how genomic selection can be leveraged in selecting or discarding lines within a program, selecting parents, and phenotyping efforts (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Belamkar <em>et al.</em>, 2018</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Juliana <em>et al.</em>, 2018</a>; <a id="xref-ref-34-19" class="xref-bibr" href="#ref-34">Lado <em>et al.</em>, 2018</a>).</p></div></div><div class="section conclusions" id="sec-15"><h2 class="">Conclusions</h2><p id="p-40">Multiple traits such as agronomic and malting quality traits are routinely evaluated in barley breeding programs. In general, every experimental line goes through an extensive selection process which includes a high number of experiments, all of them including agronomic phenotyping and a fraction of them including malting quality testing (due to cost and logistical reasons). Genomic selection using multiple traits collected throughout the breeding program could be useful for improving the genomic prediction of complex traits of interest. We evaluated strategies for adding multiple traits and compared their impact on the predictive ability of the single (ST-CV1) and multi-trait genomic prediction models (MT-CV1 and MT-CV2) on the predicted genotypic values for a trait of interest in barley breeding program. We found that including agronomic traits that are relatively easy to score and inexpensive improves the prediction of more complex traits such as grain yield and expensive/difficult to measure traits such as malting quality traits. These results can have a large impact on breeding programs where improving grain yield is a major goal and malting quality evaluations are not performed routinely in all individuals, but agronomic evaluations are. Our results suggest that including genotypic information and a model that includes all the collected data, breeders can improve the identification of superior lines with a small increase in the costs. Using these strategies could significantly change the way that phenotyping trials are set up in the future, and also the way that individuals are chosen to be phenotyped for malting quality traits.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">This work was funded in part by the following grants from ANII (FSA-1-2013-12977), CSIC (CSIC_I+D_ 1131 and CSIC_Movilidad_ 1131). The work was also funded by the Cereals Breeding and Quantitative Genetics group at the University of Wisconsin - Madison. We would like to acknowledge Dr. Juan Diaz at INIA, who developed the double haploid population and also contributed to the planning of the study. Malteria Oriental S.A. (MOSA) contributed with the experiments in their experimental areas and with some of the lab work. Malteria Uruguay S.A. (MUSA) contributed to the experiments in their experimental areas. We would also like to acknowledge: USDA-ARS small grains genotyping lab at Fargo, North Dakota for genotyping service; the Center for High Throughput Computing (CHTC) service at the University of Wisconsin-Madison for providing the high-performance computing resources; and Dr. Bettina Lado for sharing the R scripts. We would like to thank two anonymous reviewers and editors who provided constructive suggestions to this manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-42">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.9172619">https://doi.org/10.25387/g3.9172619</a>.</p></li><li class="fn" id="fn-2"><p id="p-43"><em>Communicating editor: P. Ingvarsson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-07-26"><span class="received-label">Received </span>July 26, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2020-01-22"><span class="accepted-label">Accepted </span>January 22, 2020.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Bhatta <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.1" data-doi="10.1534/g3.118.200415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Belamkar</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.J.</span> <span class="cit-name-surname">Guttieri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Hussain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarquín</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">El-basyoni</span></span>, <span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2018</span>  Genomic Selection in Preliminary Yield Trials in a Winter Wheat Breeding Program. G3:(Bethesda), 8: 2735.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200415</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200415%26rft_id%253Dinfo%253Apmid%252F29945967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvOC8yNzM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.2" data-doi="10.3835/plantgenome2019.04.0028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berro</span>,  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Lado</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. S.</span> <span class="cit-name-surname">Nalin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Quincke</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gutierrez</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Training population optimization for genomic selection.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">190028</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2019.04.0028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D12%26rft.spage%253D190028%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2019.04.0028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2019.04.0028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.3" data-doi="10.1016/S1161-0301(98)00043-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bertholdsson</span>,  <span class="cit-name-given-names">N. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Characterization of malting barley cultivars with more or less stable grain protein content under varying environmental conditions.</span> <abbr class="cit-jnl-abbrev">Eur. J. Agron.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1161-0301(98)00043-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BAgron.%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1161-0301%252898%252900043-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1161-0301(98)00043-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.4" data-doi="10.3390/ijms19103237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhatta</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Baenziger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Waters</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Poudel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Belamkar</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Genome-wide association study reveals novel genomic regions associated with 10 grain minerals in synthetic hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">Int. J. Mol. Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3237</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/ijms19103237</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BMol.%2BSci.%26rft.volume%253D19%26rft.spage%253D3237%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fijms19103237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/ijms19103237&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.5" data-doi="10.3390/ijms19103011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhatta</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Morgounov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Belamkar</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Baenziger</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">Genome-wide association study reveals novel genomic regions for grain yield and yield-related traits in drought-stressed synthetic hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">Int. J. Mol. Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/ijms19103011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BMol.%2BSci.%26rft.volume%253D19%26rft.spage%253D3011%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fijms19103011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/ijms19103011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.6" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype× environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.7" data-doi="10.1186/1297-9686-43-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>,  <span class="cit-name-given-names">M. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Veerkamp</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-26</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D43%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DAccuracy%2Bof%2Bmulti-trait%2Bgenomic%2Bselection%2Busing%2Bdifferent%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-26%26rft_id%253Dinfo%253Apmid%252F21729282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729282&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.1113.8"><div class="cit-metadata unstructured">de los Campos, G., and A. Grüneberg, 2016 MTM Package. <a href="http://quantgen.github.io/MTM/vignette.html">http://quantgen.github.io/MTM/vignette.html</a> (accessed February 28, 2019).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.1113.9"><div class="cit-metadata unstructured">Covarrubias-Pazaran, G., 2018. Quantitative genetics using the sommer package. <a href="https://cran.r-project.org/web/packages/sommer/vignettes/sommer.pdf">https://cran.r-project.org/web/packages/sommer/vignettes/sommer.pdf</a> (accessed February 28, 2019).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.10" data-doi="10.1534/g3.113.008227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Beyene</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kassa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in maize breeding populations with genotyping-by-sequencing. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1903</span>–<span class="cit-lpage">1926</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.008227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D3%26rft.spage%253D1903%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.113.008227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.11" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ornella</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.12" data-doi="10.1198/108571106X154443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>,  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Smith</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. E.</span> <span class="cit-name-surname">Coombes</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">On the design of early generation variety trials with correlated data.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">393</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/108571106X154443</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D11%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1198%252F108571106X154443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/108571106X154443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.13" data-doi="10.1016/j.foodchem.2007.04.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dai</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Xu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genotypic and environmental variation in phytic acid content and its relation to protein content and malt quality in barley.</span> <abbr class="cit-jnl-abbrev">Food Chem.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">606</span>–<span class="cit-lpage">611</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.foodchem.2007.04.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BChem.%26rft.volume%253D105%26rft.spage%253D606%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foodchem.2007.04.019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foodchem.2007.04.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.14" data-doi="10.1371/journal.pone.0152045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dos Santos</span>,  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. C.</span> <span class="cit-name-surname">Vasconcellos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. P.</span> <span class="cit-name-surname">Pires</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Balestre</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Von Pinho</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Inclusion of dominance effects in the multivariate GBLUP model.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0152045</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0152045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0152045%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0152045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0152045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.15" data-doi="10.1007/s11032-016-0453-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duangjit</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Causse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Sauvage</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Efficiency of genomic selection for tomato fruit quality.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">29</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-016-0453-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D36%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-016-0453-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-016-0453-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.16" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.17" data-doi="10.2135/cropsci2013.03.0154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. N.</span> <span class="cit-name-surname">Atlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Beyene</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Semagn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Optimal design of preliminary yield trials with genome-wide markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.03.0154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.03.0154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.03.0154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.1113.18"><div class="cit-metadata unstructured">FAOSTAT, 2019 Food and Agriculture Organization of the United Nations. <a href="http://www.fao.org/faostat/en/#data">http://www.fao.org/faostat/en/#data</a> (accessed February 28, 2019).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.19" data-doi="10.1038/nprot.2014.174"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">He</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhong</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Illumina human exome genotyping array clustering and quality control.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2643</span>–<span class="cit-lpage">2662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2014.174</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D9%26rft.spage%253D2643%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2014.174%26rft_id%253Dinfo%253Apmid%252F25321409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2014.174&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25321409&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.20" data-doi="10.1186/1471-2156-15-30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Du</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Comparison of single-trait and multiple-trait genomic prediction models.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-15-30</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D15%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-15-30%26rft_id%253Dinfo%253Apmid%252F24593261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-15-30&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24593261&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.21" data-doi="10.3835/plantgenome2011.07.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cuesta-Marcos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Castro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">von Zitzewitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schmitt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association mapping of malting quality quantitative trait loci in winter barley: positive signals from small germplasm arrays.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.07.0020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D256%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.07.0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.07.0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000312661700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.22" data-doi="10.1007/s00122-014-2448-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutiérrez</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Germán</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pereyra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Hayes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Pérez</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multi-environment multi-QTL association mapping identifies disease resistance QTL in barley germplasm from Latin America.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">516</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2448-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D501%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2448-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2448-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.23" data-doi="10.1023/A:1009608312385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Romagosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Ullrich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Hayes</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Molecular marker-assisted selection for malting quality traits in barley.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">437</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1009608312385</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D3%26rft.spage%253D427%26rft.atitle%253DMOLECULAR%2BMARKER-ASSISTED%2BSELECTION%2BFOR%2BMALTING%2BQUALITY%2BTRAITS%2BIN%2BBARLEY%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1009608312385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1009608312385&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.1113.24"><div class="cit-metadata unstructured">Harrell, F. E., 2019 Package “Hmisc.” <a href="https://cran.r-project.org/web/packages/Hmisc/Hmisc.pdf">https://cran.r-project.org/web/packages/Hmisc/Hmisc.pdf</a>. (accessed February 28, 2019).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.25" data-doi="10.1007/s00122-017-2972-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Panozzo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. K.</span> <span class="cit-name-surname">Walker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Choy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kant</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Accelerating wheat breeding for end-use quality with multi-trait genomic predictions incorporating near infrared and nuclear magnetic resonance-derived phenotypes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2505</span>–<span class="cit-lpage">2519</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2972-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D2505%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2972-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2972-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.3.1113.26" data-doi="10.1007/s00122-013-2231-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Akdemir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>. <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integrating environmental covariates and crop modeling into the genomic selection framework to predict genotype by environment interactions.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">480</span>. </cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2231-5%26rft_id%253Dinfo%253Apmid%252F24264761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2231-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24264761&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.27" data-doi="10.2135/cropsci2014.03.0249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Perspectives for genomic selection applications and research in plants.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2014.03.0249</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.03.0249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.03.0249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.28" data-doi="10.2135/cropsci2000.4051426x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Igartua</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Edney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. G.</span> <span class="cit-name-surname">Rossnagel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Spaner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Legge</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Marker-based selection of QTL affecting grain and malt quality in two-row barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1426</span>–<span class="cit-lpage">1433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2000.4051426x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DIgartua%26rft.auinit1%253DE.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D1426%26rft.epage%253D1433%26rft.atitle%253DMarker-Based%2BSelection%2Bof%2BQTL%2BAffecting%2BGrain%2Band%2BMalt%2BQuality%2Bin%2BTwo-Row%2BBarley%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2000.4051426x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2000.4051426x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000089612600032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.29" data-doi="10.1534/genetics.112.144246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>. <span class="cit-pub-date">2012</span> Multiple-trait genomic selection methods increase genetic value prediction accuracy. Genetics 192: 1513–1522.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.144246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.144246%26rft_id%253Dinfo%253Apmid%252F23086217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE1MTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzExMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.30" data-doi="10.1038/hdy.2015.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Joint prediction of multiple quantitative traits using a Bayesian multivariate antedependence model.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.9%26rft_id%253Dinfo%253Apmid%252F25873147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25873147&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.31" data-doi="10.3835/plantgenome2018.03.0017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juliana</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. P.</span> <span class="cit-name-surname">Singh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mondal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Prospects and challenges of applied genomic selection—a new paradigm in breeding for grain yield in bread wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">180017</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2018.03.0017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2018.03.0017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2018.03.0017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.1113.32"><div class="cit-metadata unstructured">Kassambara, A., and F. Mundt, 2017 factoextra: Extract and Visualize the Results of Multivariate Data Analyses. <a href="https://cran.r-project.org/web/packages/factoextra/factoextra.pdf">https://cran.r-project.org/web/packages/factoextra/factoextra.pdf</a> (accessed February 28, 2019).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.33" data-doi="10.2135/cropsci2015.04.0207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lado</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. G.</span> <span class="cit-name-surname">Barrios</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Quincke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Silva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gutiérrez</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Modeling genotype × environment interaction for genomic selection with unbalanced data from a wheat breeding program.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">2165</span>–<span class="cit-lpage">2179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.04.0207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D2165%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.04.0207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.04.0207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.34" data-doi="10.1007/s00122-018-3186-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lado</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Vázquez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Quincke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Silva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Aguilar</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Resource allocation optimization with multi-trait genomic prediction for bread wheat (<em>Triticum aestivum</em> L.) baking quality.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">2719</span>–<span class="cit-lpage">2731</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-018-3186-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D2719%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-018-3186-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-018-3186-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.35" data-doi="10.2135/cropsci2011.09.0503"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Smith</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Potential and optimization of genomic selection for fusarium head blight resistance in six-row barley.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1621</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.09.0503</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D1609%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.09.0503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.09.0503&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305804900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.36" data-doi="10.20900/cbgg20190012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lozada</span>,  <span class="cit-name-given-names">D. N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Carter</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Accuracy of single and multi-trait genomic prediction models for grain yield in US pacific northwest winter wheat. Crop Breeding</span>. <abbr class="cit-jnl-abbrev">Genes Genomics</abbr> <span class="cit-vol">1</span>: <span class="cit-elocation-id">e190012</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.20900/cbgg20190012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomics%26rft.volume%253D1%26rft.spage%253D190012e%26rft_id%253Dinfo%253Adoi%252F10.20900%252Fcbgg20190012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.20900/cbgg20190012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.37" data-doi="10.1186/s12863-019-0785-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lozada</span>,  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Mason</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Sarinelli</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Brown-Guedira</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Accuracy of genomic selection for grain yield and agronomic traits in soft red winter wheat.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-019-0785-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D20%26rft.spage%253D82%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-019-0785-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-019-0785-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.38" data-doi="10.1007/s00122-017-2998-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michel</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kummer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gallee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hellinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ametz</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Improving the baking quality of bread wheat by genomic selection in early generations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">493</span>. Erratum: 495.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2998-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D131%26rft.spage%253D477%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2998-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2998-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.39" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>. Erratum: xx.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzExMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>,  <span class="cit-name-given-names">O.A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Montesinos-López</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.H.</span> <span class="cit-name-surname">Toledo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Pérez-Hernández</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> A genomic bayesian multi-trait and multi-environment model. G3: (Bethesda) 6: 2725–2744.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032359</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.41" data-doi="10.1371/journal.pone.0164494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Jahoor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Jensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Orabi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Cericola</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic prediction of seed quality traits using advanced barley breeding lines.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0164494</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0164494</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0164494%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0164494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0164494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.42" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.3.1113.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>,  <span class="cit-name-given-names">J. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. W.</span>, <span class="cit-name-surname">Rife</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Genotyping-by-sequencing for plant breeding and genetics. The Plant Genome 5: 92–102.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rutkoski</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Advances and challenges in genomic selection for disease resistance. annual review of phytopathology 54: 79–98.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-phyto-080615–100056</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.45" data-doi="10.1534/genetics.111.137026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>,  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Muñoz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D. V.</span> <span class="cit-name-surname">Resende</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic selection methods in a standard data set of loblolly pine (<em>Pinus taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.137026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137026%26rft_id%253Dinfo%253Apmid%252F22271763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1MDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzExMTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.46" data-doi="10.1007/s11032-011-9667-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rode</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ahlemeyer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Friedt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Ordon</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of marker-trait associations in the German winter barley breeding gene pool (<em>Hordeum vulgare</em> L.).</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">843</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-011-9667-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D30%26rft.spage%253D831%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-011-9667-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-011-9667-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.47" data-doi="10.3835/plantgenome2012.02.0001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Benson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Jia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Brown-Guedira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Jannink</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evaluation of genomic prediction methods for fusarium head blight resistance in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.02.0001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.02.0001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.02.0001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.3.1113.48" data-doi="10.3835/plantgenome2014.10.0074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. P.</span> <span class="cit-name-surname">Singh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Huerta-Espino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bhavani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Poland</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic gain from phenotypic and genomic selection for quantitative resistance to stem rust of wheat.</span> <abbr class="cit-jnl-abbrev">The Plant Genome,</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.10.0074</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%252C%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.10.0074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.10.0074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.49" data-doi="10.1073/pnas.81.24.8014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saghai-Maroof</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Soliman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Jorgensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Allard</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Ribosomal DNA spacer-length polymorphisms in barley: mendelian inheritance, chromosomal location, and population dynamics.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">8014</span>–<span class="cit-lpage">8018</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.81.24.8014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSaghai-Maroof%26rft.auinit1%253DM.%2BA.%26rft.volume%253D81%26rft.issue%253D24%26rft.spage%253D8014%26rft.epage%253D8018%26rft.atitle%253DRibosomal%2BDNA%2Bspacer-length%2Bpolymorphisms%2Bin%2Bbarley%253A%2Bmendelian%2Binheritance%252C%2Bchromosomal%2Blocation%252C%2Band%2Bpopulation%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.24.8014%26rft_id%253Dinfo%253Apmid%252F6096873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODEvMjQvODAxNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTExMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sallam</span>,  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Endelman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Jannink</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. P.</span> <span class="cit-name-surname">Smith</span></span></li></ol><cite>. <span class="cit-pub-date">2015</span> Assessing genomic selection prediction accuracy in a dynamic barley breeding population. The Plant Genome 8: 1–15.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.05.0020</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.1113.51"><div class="cit-metadata unstructured">SAS Institute, 2018. 2018 SAS 9.4 Product Documentation. <a href="https://support.sas.com/documentation/94/">https://support.sas.com/documentation/94/</a> (accessed February 28, 2019).</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.52" data-doi="10.1007/s00122-015-2639-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kollers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Maasberg-Prelle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Großer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Schinkel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Prediction of malting quality traits in barley based on genome-wide marker data to assess the potential of genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">213</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2639-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D203%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2639-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2639-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.53" data-doi="10.1007/s11032-005-0903-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmierer</span>,  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kandemir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Kudrna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Ullrich</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Molecular marker-assisted selection for enhanced yield in malting barley.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-005-0903-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D14%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-005-0903-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-005-0903-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.54" data-doi="10.1007/s00122-015-2626-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulthess</span>,  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Miedaner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wilde</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Reif</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multiple-trait- and selection indices-genomic predictions for grain yield and protein content in rye for feeding purposes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">287</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2626-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D273%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2626-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2626-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.55" data-doi="10.3835/plantgenome2016.11.0111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Rutkoski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> Multitrait, Random Regression, or Simple Repeatability Model in High-Throughput Phenotyping Data Improve Genomic Prediction for Wheat Grain Yield. The Plant Genome 10: 1–12.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.11.0111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLANT%2BGENOME%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.11.0111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.11.0111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.56" data-doi="10.1007/s00122-017-2917-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorwarth</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ahlemeyer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.-M.</span> <span class="cit-name-surname">Bochard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Krumnacker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Blümel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction ability for yield-related traits in German winter barley elite material.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">1669</span>–<span class="cit-lpage">1683</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2917-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D1669%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2917-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2917-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.57" data-doi="10.2135/cropsci2018.03.0189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>,  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Brown-Guedira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Tyagi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. L.</span> <span class="cit-name-surname">Kolb</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Van Sanford</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Multienvironment and multitrait genomic selection models in unbalanced early-generation wheat yield trials.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">507</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2018.03.0189</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D59%26rft.spage%253D491%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2018.03.0189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2018.03.0189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.58" data-doi="10.1038/hdy.2016.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Predicting rice hybrid performance using univariate and multivariate GBLUP models based on North Carolina mating design II.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2016.87</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D118%26rft.spage%253D302%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2016.87%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2016.87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.59" data-doi="10.1093/jxb/erx135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Xie</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Enhancing genetic gain in the era of molecular breeding.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">2641</span>–<span class="cit-lpage">2666</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/erx135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft.volume%253D68%26rft.spage%253D2641%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferx135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/erx135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1113.60" data-doi="10.1007/s00122-015-2614-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. B.</span> <span class="cit-name-surname">Cregan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.-L.</span> <span class="cit-name-surname">Jiang</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association study, genomic prediction and marker-assisted selection for seed weight in soybean (Glycine max).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">130</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2614-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2614-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2614-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/3/1113.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/1099" title="Multiple Requirements for Rab GTPases in the Development of Drosophila Tracheal Dorsal Branches and Terminal Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/1125" title="Absence of a Faster-X Effect in Beetles (Tribolium, Coleoptera)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/3"><span>Volume 10 Issue 3, March 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/3.cover-source.jpg?itok=8jjPWDA4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1113?rss=1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7Kr-rlrIvB2c0bBfDOAuOP5nnQ1bx6P21fiUmnD6zco" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/3/1113" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/3/1113" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/3/1113" class="active" data-icon-position="" data-hide-link-title="0">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (Hordeum vulgare L.)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495864" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-WevUA6rPOE7GhuS2CkhnsOfA2a2EKsxhSQgLVyBYtso" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/1113.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/3/1113" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-JzkgV2VbQxLuSyh5qu3FxXvIsQZuxrEtMEuCT7tFOpo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/3/1113" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/1113.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864--2" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4959-0481" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Madhav Bhatta</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2957-3086" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lucia Gutierrez</span>, <span class="highwire-citation-author" data-delta="2">Lorena Cammarota</span>, <span class="highwire-citation-author" data-delta="3">Fernanda Cardozo</span>, <span class="highwire-citation-author" data-delta="4">Silvia Germán</span>, <span class="highwire-citation-author" data-delta="5">Blanca Gómez-Guerrero</span>, <span class="highwire-citation-author" data-delta="6">María Fernanda Pardo</span>, <span class="highwire-citation-author" data-delta="7">Valeria Lanaro</span>, <span class="highwire-citation-author" data-delta="8">Mercedes Sayas</span> and <span class="highwire-citation-author" data-delta="9">Ariel J. Castro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1113-1124;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400968 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495864--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Madhav Bhatta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, 1575 Linden Dr., WI, 53706,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madhav%2BBhatta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhatta%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMadhav%2BBhatta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4959-0481" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Madhav Bhatta</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lucia Gutierrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, 1575 Linden Dr., WI, 53706,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucia%2BGutierrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gutierrez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucia%2BGutierrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2957-3086" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lucia Gutierrez</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gutierrezcha@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">gutierrezcha@wisc.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lorena Cammarota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of plant production, Facultad de Agronomía, Universidad de la República, Ruta 3, Km363, Paysandú 60000, Uruguay,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Maltería Uruguay S.A. Ruta 55, Km26, Ombúes de Lavalle, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorena%2BCammarota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cammarota%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALorena%2BCammarota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernanda Cardozo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Maltería Uruguay S.A. Ruta 55, Km26, Ombúes de Lavalle, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernanda%2BCardozo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cardozo%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernanda%2BCardozo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Silvia Germán </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Instituto Nacional de Investigación Agropuecuaria, Estación Experimental La Estanzuela, Ruta 50, Km11, Colonia, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Silvia%2BGerm%C3%A1n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Germ%C3%A1n%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASilvia%2BGerm%25C3%25A1n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Blanca Gómez-Guerrero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Latitud, LATU Foundation, Av Italia 6201, Montevideo 11500, Uruguay, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Blanca%2BG%C3%B3mez-Guerrero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=G%C3%B3mez-Guerrero%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABlanca%2BG%25C3%25B3mez-Guerrero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">María Fernanda Pardo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Maltería Oriental S.A., Camino Abrevadero 5525, Montevideo 12400, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mar%C3%ADa%2BFernanda%2BPardo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMar%25C3%25ADa%2BFernanda%2BPardo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Valeria Lanaro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Latitud, LATU Foundation, Av Italia 6201, Montevideo 11500, Uruguay, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valeria%2BLanaro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lanaro%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValeria%2BLanaro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Mercedes Sayas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Maltería Oriental S.A., Camino Abrevadero 5525, Montevideo 12400, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mercedes%2BSayas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sayas%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMercedes%2BSayas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ariel J. Castro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of plant production, Facultad de Agronomía, Universidad de la República, Ruta 3, Km363, Paysandú 60000, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ariel%2BJ.%2BCastro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Castro%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAriel%2BJ.%2BCastro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/3/1113" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/3/1113&amp;title=Multi-trait%20Genomic%20Prediction%20Model%20Increased%20the%20Predictive%20Ability%20for%20Agronomic%20and%20Malting%20Quality%20Traits%20in%20Barley%20%28Hordeum%20vulgare%20L.%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/3/1113&amp;title=Multi-trait%20Genomic%20Prediction%20Model%20Increased%20the%20Predictive%20Ability%20for%20Agronomic%20and%20Malting%20Quality%20Traits%20in%20Barley%20%28Hordeum%20vulgare%20L.%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/3/1113&amp;title=Multi-trait%20Genomic%20Prediction%20Model%20Increased%20the%20Predictive%20Ability%20for%20Agronomic%20and%20Malting%20Quality%20Traits%20in%20Barley%20%28Hordeum%20vulgare%20L.%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/3/1113&amp;text=Multi-trait%20Genomic%20Prediction%20Model%20Increased%20the%20Predictive%20Ability%20for%20Agronomic%20and%20Malting%20Quality%20Traits%20in%20Barley%20%28Hordeum%20vulgare%20L.%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/3/1113&amp;title=Multi-trait%20Genomic%20Prediction%20Model%20Increased%20the%20Predictive%20Ability%20for%20Agronomic%20and%20Malting%20Quality%20Traits%20in%20Barley%20%28Hordeum%20vulgare%20L.%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/3/1113&amp;t=Multi-trait%20Genomic%20Prediction%20Model%20Increased%20the%20Predictive%20Ability%20for%20Agronomic%20and%20Malting%20Quality%20Traits%20in%20Barley%20%28Hordeum%20vulgare%20L.%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/3/1113&amp;title=Multi-trait%20Genomic%20Prediction%20Model%20Increased%20the%20Predictive%20Ability%20for%20Agronomic%20and%20Malting%20Quality%20Traits%20in%20Barley%20%28Hordeum%20vulgare%20L.%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/3/1113&amp;title=Multi-trait%20Genomic%20Prediction%20Model%20Increased%20the%20Predictive%20Ability%20for%20Agronomic%20and%20Malting%20Quality%20Traits%20in%20Barley%20%28Hordeum%20vulgare%20L.%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495864/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864--3" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4959-0481" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Madhav Bhatta</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2957-3086" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lucia Gutierrez</span>, <span class="highwire-citation-author" data-delta="2">Lorena Cammarota</span>, <span class="highwire-citation-author" data-delta="3">Fernanda Cardozo</span>, <span class="highwire-citation-author" data-delta="4">Silvia Germán</span>, <span class="highwire-citation-author" data-delta="5">Blanca Gómez-Guerrero</span>, <span class="highwire-citation-author" data-delta="6">María Fernanda Pardo</span>, <span class="highwire-citation-author" data-delta="7">Valeria Lanaro</span>, <span class="highwire-citation-author" data-delta="8">Mercedes Sayas</span> and <span class="highwire-citation-author" data-delta="9">Ariel J. Castro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1113-1124;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400968 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495864--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Madhav Bhatta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, 1575 Linden Dr., WI, 53706,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Madhav%2BBhatta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bhatta%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMadhav%2BBhatta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4959-0481" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Madhav Bhatta</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lucia Gutierrez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, University of Wisconsin-Madison, 1575 Linden Dr., WI, 53706,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucia%2BGutierrez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gutierrez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucia%2BGutierrez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2957-3086" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lucia Gutierrez</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gutierrezcha@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">gutierrezcha@wisc.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Lorena Cammarota </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of plant production, Facultad de Agronomía, Universidad de la República, Ruta 3, Km363, Paysandú 60000, Uruguay,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Maltería Uruguay S.A. Ruta 55, Km26, Ombúes de Lavalle, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorena%2BCammarota%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cammarota%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALorena%2BCammarota%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Fernanda Cardozo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Maltería Uruguay S.A. Ruta 55, Km26, Ombúes de Lavalle, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernanda%2BCardozo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cardozo%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernanda%2BCardozo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Silvia Germán </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Instituto Nacional de Investigación Agropuecuaria, Estación Experimental La Estanzuela, Ruta 50, Km11, Colonia, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Silvia%2BGerm%C3%A1n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Germ%C3%A1n%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASilvia%2BGerm%25C3%25A1n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Blanca Gómez-Guerrero </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Latitud, LATU Foundation, Av Italia 6201, Montevideo 11500, Uruguay, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Blanca%2BG%C3%B3mez-Guerrero%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=G%C3%B3mez-Guerrero%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABlanca%2BG%25C3%25B3mez-Guerrero%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">María Fernanda Pardo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Maltería Oriental S.A., Camino Abrevadero 5525, Montevideo 12400, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mar%C3%ADa%2BFernanda%2BPardo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMar%25C3%25ADa%2BFernanda%2BPardo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Valeria Lanaro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Latitud, LATU Foundation, Av Italia 6201, Montevideo 11500, Uruguay, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valeria%2BLanaro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lanaro%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValeria%2BLanaro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Mercedes Sayas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Maltería Oriental S.A., Camino Abrevadero 5525, Montevideo 12400, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mercedes%2BSayas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sayas%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMercedes%2BSayas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ariel J. Castro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of plant production, Facultad de Agronomía, Universidad de la República, Ruta 3, Km363, Paysandú 60000, Uruguay,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ariel%2BJ.%2BCastro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Castro%20AJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAriel%2BJ.%2BCastro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1113&amp;rft.epage=1124&amp;rft.atitle=Multi-trait%20Genomic%20Prediction%20Model%20Increased%20the%20Predictive%20Ability%20for%20Agronomic%20and%20Malting%20Quality%20Traits%20in%20Barley%20%28Hordeum%20vulgare%20L.%29&amp;rft.volume=10&amp;rft.issue=3&amp;rft.date=2020-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bhatta%2C+Madhav&amp;rft.au=Gutierrez%2C+Lucia&amp;rft.au=Cammarota%2C+Lorena&amp;rft.au=Cardozo%2C+Fernanda&amp;rft.au=Germ%C3%A1n%2C+Silvia&amp;rft.au=G%C3%B3mez-Guerrero%2C+Blanca&amp;rft.au=Pardo%2C+Mar%C3%ADa+Fernanda&amp;rft.au=Lanaro%2C+Valeria&amp;rft.au=Sayas%2C+Mercedes&amp;rft.au=Castro%2C+Ariel+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495864/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495864/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495864/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495864/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495864/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495864/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495864/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495864/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495864/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495864/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495864/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495864/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/3/1113" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/3/1113.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/3/1113.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/3/1113.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
