<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/3/951" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/3/951.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/3/951.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/3/951.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/3/951/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/3/951.atom" />
<meta name="HW.pisa" content="ggg;10/3/951" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity" />
<meta name="DC.Identifier" content="10.1534/g3.119.400763" />
<meta name="DC.Date" content="2020-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Zhou et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="There has been extensive discussion of the “Replication Crisis” in many fields, including genome-wide association studies ( GWAS ). We explored replication in a mouse model using an advanced intercross line ( AIL ), which is a multigenerational intercross between two inbred strains. We re-genotyped a previously published cohort of LG/J x SM/J AIL mice (F34; n = 428) using a denser marker set and genotyped a new cohort of AIL mice (F39-43; n = 600) for the first time. We identified 36 novel genome-wide significant loci in the F34 and 25 novel loci in the F39-43 cohort. The subset of traits that were measured in both cohorts (locomotor activity, body weight, and coat color) showed high genetic correlations, although the SNP heritabilities were slightly lower in the F39-43 cohort. For this subset of traits, we attempted to replicate loci identified in either F34 or F39-43 in the other cohort. Coat color was robustly replicated; locomotor activity and body weight were only partially replicated, which was inconsistent with our power simulations. We used a random effects model to show that the partial replications could not be explained by Winner’s Curse but could be explained by study-specific heterogeneity. Despite this heterogeneity, we performed a mega-analysis by combining F34 and F39-43 cohorts (n = 1,028), which identified four novel loci associated with locomotor activity and body weight. These results illustrate that even with the high degree of genetic and environmental control possible in our experimental system, replication was hindered by study-specific heterogeneity, which has broad implications for ongoing concerns about reproducibility." />
<meta name="DC.Contributor" content="Xinzhu Zhou" />
<meta name="DC.Contributor" content="Celine L. St. Pierre" />
<meta name="DC.Contributor" content="Natalia M. Gonzales" />
<meta name="DC.Contributor" content="Jennifer Zou" />
<meta name="DC.Contributor" content="Riyan Cheng" />
<meta name="DC.Contributor" content="Apurva S. Chitre" />
<meta name="DC.Contributor" content="Greta Sokoloff" />
<meta name="DC.Contributor" content="Abraham A. Palmer" />
<meta name="article:published_time" content="2020-03-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;There has been extensive discussion of the “Replication Crisis” in many fields, including genome-wide association studies (&lt;b&gt;GWAS&lt;/b&gt;). We explored replication in a mouse model using an advanced intercross line (&lt;b&gt;AIL&lt;/b&gt;), which is a multigenerational intercross between two inbred strains. We re-genotyped a previously published cohort of LG/J x SM/J AIL mice (F&lt;sub&gt;34&lt;/sub&gt;; n = 428) using a denser marker set and genotyped a new cohort of AIL mice (F&lt;sub&gt;39-43&lt;/sub&gt;; n = 600) for the first time. We identified 36 novel genome-wide significant loci in the F&lt;sub&gt;34&lt;/sub&gt; and 25 novel loci in the F&lt;sub&gt;39-43&lt;/sub&gt; cohort. The subset of traits that were measured in both cohorts (locomotor activity, body weight, and coat color) showed high genetic correlations, although the SNP heritabilities were slightly lower in the F&lt;sub&gt;39-43&lt;/sub&gt; cohort. For this subset of traits, we attempted to replicate loci identified in either F&lt;sub&gt;34&lt;/sub&gt; or F&lt;sub&gt;39-43&lt;/sub&gt; in the other cohort. Coat color was robustly replicated; locomotor activity and body weight were only partially replicated, which was inconsistent with our power simulations. We used a random effects model to show that the partial replications could not be explained by Winner’s Curse but could be explained by study-specific heterogeneity. Despite this heterogeneity, we performed a mega-analysis by combining F&lt;sub&gt;34&lt;/sub&gt; and F&lt;sub&gt;39-43&lt;/sub&gt; cohorts (n = 1,028), which identified four novel loci associated with locomotor activity and body weight. These results illustrate that even with the high degree of genetic and environmental control possible in our experimental system, replication was hindered by study-specific heterogeneity, which has broad implications for ongoing concerns about reproducibility.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/03/01" />
<meta name="citation_mjid" content="ggg;10/3/951" />
<meta name="citation_id" content="10/3/951" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/3/951" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/3/951.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/3/951.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/3/951.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400763" />
<meta name="citation_pmid" content="31974095" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="951" />
<meta name="citation_lastpage" content="965" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Xinzhu Zhou" />
<meta name="citation_author_institution" content="Biomedical Sciences Graduate Program, University of California San Diego, La Jolla, CA, 92092," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6432-0033" />
<meta name="citation_author" content="Celine L. St. Pierre" />
<meta name="citation_author_institution" content="Department of Genetics, Washington University School of Medicine, St. Louis, MO, 63110," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5465-6601" />
<meta name="citation_author" content="Natalia M. Gonzales" />
<meta name="citation_author_institution" content="Department of Human Genetics, University of Chicago, IL, 60637," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5289-0410" />
<meta name="citation_author" content="Jennifer Zou" />
<meta name="citation_author_institution" content="Department of Computer Science, University of California, Los Angeles, CA, 90095," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2045-3272" />
<meta name="citation_author" content="Riyan Cheng" />
<meta name="citation_author_institution" content="Department of Psychiatry," />
<meta name="citation_author" content="Apurva S. Chitre" />
<meta name="citation_author_institution" content="Department of Psychiatry," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1709-9214" />
<meta name="citation_author" content="Greta Sokoloff" />
<meta name="citation_author_institution" content="Department of Psychological &amp; Brain Sciences, University of Iowa, Iowa City, IO, 52242" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3651-1980" />
<meta name="citation_author" content="Abraham A. Palmer" />
<meta name="citation_author_institution" content="Department of Psychiatry," />
<meta name="citation_author_institution" content="Institute for Genomic Medicine, University of California San Diego, La Jolla, CA, 92037 and" />
<meta name="citation_author_email" content="aap@ucsd.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3634-0747" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Abney;citation_title=Identity-by-descent estimation and mapping of qualitative traits in large, complex pedigrees.;citation_volume=179;citation_year=2008;citation_pmid=18622032;citation_doi=10.1534/genetics.108.089912" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=T. B. Bartnikas;citation_author=C. C. Parker;citation_author=R. Cheng;citation_author=D. R. Campagna;citation_author=J. E. Lim;citation_title=QTLs for murine red blood cell parameters in LG/J and SM/J F(2) and advanced intercross lines.;citation_pages=356-366;citation_volume=23;citation_year=2012;citation_issue=5-6;citation_pmid=22322356;citation_doi=10.1007/s00335-012-9393-3" />
<meta name="citation_reference" content="citation_journal_title=Sci. Data;citation_author=A. Baud;citation_author=V. Guryev;citation_author=O. Hummel;citation_author=M. Johannesson;citation_author=R. Hermsen;citation_title=Genomes and phenomes of a population of outbred rats and its progenitors.;citation_pages=140011;citation_volume=1;citation_year=2014;citation_doi=10.1038/sdata.2014.11" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. D. Beavis;citation_author=D. Grant;citation_author=M. Albertsen;citation_author=R. Fincher;citation_title=Quantitative trait loci for plant height in four maize populations and their associations with qualitative genetic loci.;citation_pages=141-145;citation_volume=83;citation_year=1991;citation_pmid=24202349;citation_doi=10.1007/BF00215714" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=W. D. Beavis;citation_author=O. S. Smith;citation_author=D. Grant;citation_author=R. Fincher;citation_title=Identification of Quantitative Trait Loci Using a Small Sample of Topcrossed and F4 Progeny from Maize;citation_pages=882-896;citation_volume=34;citation_year=1994;citation_issue=4;citation_doi=10.2135/cropsci1994.0011183X003400040010x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=F. Besnier;citation_author=P. Wahlberg;citation_author=L. Rönnegård;citation_author=W. Ek;citation_author=L. Andersson;citation_title=Fine mapping and replication of QTL in outbred chicken advanced intercross lines.;citation_pages=3-3;citation_volume=43;citation_year=2011;citation_issue=1;citation_pmid=21241486;citation_doi=10.1186/1297-9686-43-3" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=P. Carbonetto;citation_author=R. Cheng;citation_author=J. P. Gyekis;citation_author=C. C. Parker;citation_author=D. A. Blizard;citation_title=Discovery and refinement of muscle weight QTLs in B6 × D2 advanced intercross mice.;citation_volume=46;citation_year=2014;citation_pmid=24963006;citation_doi=10.1152/physiolgenomics.00055.2014" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=A. M. Carroll;citation_author=R. Cheng;citation_author=E. S. R. Collie-Duguid;citation_author=C. Meharg;citation_author=M. E. Scholz;citation_title=Fine-mapping of genes determining extrafusal fiber properties in murine soleus muscle.;citation_volume=49;citation_year=2017;citation_pmid=28087756;citation_doi=10.1152/physiolgenomics.00092.2016" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=A. C. Cervino;citation_author=G. Li;citation_author=S. Edwards;citation_author=J. Zhu;citation_author=C. Laurie;citation_title=Integrating QTL and high-density SNP analyses in mice to identify Insig2 as a susceptibility gene for plasma cholesterol levels.;citation_pages=505-517;citation_volume=86;citation_year=2005;citation_issue=5;citation_pmid=16126366;citation_doi=10.1016/j.ygeno.2005.07.010" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=R. Cheng;citation_author=M. Abney;citation_author=A. A. Palmer;citation_author=A. D. Skol;citation_title=QTLRel: an R package for genome-wide association studies in which relatedness is a concern.;citation_pages=66-66;citation_volume=12;citation_year=2011;citation_pmid=21794153;citation_doi=10.1186/1471-2156-12-66" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Cheng;citation_author=J. E. Lim;citation_author=K. E. Samocha;citation_author=G. Sokoloff;citation_author=M. Abney;citation_title=Genome-wide association studies and the problem of relatedness among advanced intercross lines and other highly recombinant populations.;citation_volume=185;citation_year=2010;citation_pmid=20439773;citation_doi=10.1534/genetics.110.116863" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=R. Cheng;citation_author=C. C. Parker;citation_author=M. Abney;citation_author=A. A. Palmer;citation_title=Practical considerations regarding the use of genotype and pedigree data to model relatedness in the context of genome-wide association studies.;citation_volume=3;citation_year=2013;citation_pmid=23979941;citation_doi=10.1534/g3.113.007948" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=E. J. Chesler;citation_title=Out of the bottleneck: The Diversity Outcross and Collaborative Cross mouse populations in behavioral genetics research.;citation_pages=3-11;citation_volume=25;citation_year=2014;citation_pmid=24272351;citation_doi=10.1007/s00335-013-9492-9" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=G. A. Churchill;citation_author=D. M. Gatti;citation_author=S. C. Munger;citation_author=K. L. Svenson;citation_title=The diversity outbred mouse population.;citation_pages=713-718;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22892839;citation_doi=10.1007/s00335-012-9414-2" />
<meta name="citation_reference" content="citation_journal_title=GENET MAP POPUL FOR CONDUCT HIGHRESOLUT TRAIT MAP IN PLANT;citation_author=J. Cockram;citation_author=and I. Mackay;citation_title=GENETIC MAPPING POPULATIONS FOR CONDUCTING HIGH-RESOLUTION TRAIT MAPPING IN PLANTS.;citation_pages=109;citation_volume=164;citation_year=2018;citation_doi=10.1007/10_2017_48" />
<meta name="citation_reference" content="citation_journal_title=Cytogenet. Genome Res.;citation_author=P. M. Conneally;citation_author=J. H. Edwards;citation_author=K. K. Kidd;citation_author=J.-M. Lalouel;citation_author=N. E. Morton;citation_title=Report of the committee on methods of linkage analysis and reporting.;citation_pages=356-359;citation_volume=40;citation_year=1985;citation_doi=10.1159/000132186" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=The genome architecture of the Collaborative Cross mouse genetic reference population.;citation_volume=190;citation_year=2012;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Neurobiol. Learn. Mem.;citation_author=J. Coyner;citation_author=J. L. McGuire;citation_author=C. C. Parker;citation_author=R. J. Ursano;citation_author=A. A. Palmer;citation_title=Mice selectively bred for High and Low fear behavior show differences in the number of pMAPK (p44/42 ERK) expressing neurons in lateral amygdala following Pavlovian fear conditioning.;citation_pages=195-203;citation_volume=112;citation_year=2014;citation_doi=10.1016/j.nlm.2013.06.010" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. C. Crabbe;citation_author=D. Wahlsten;citation_author=B. C. Dudek;citation_title=Genetics of Mouse Behavior: Interactions with Laboratory Environment;citation_pages=1670-1672;citation_volume=284;citation_year=1999;citation_issue=5420;citation_pmid=10356397;citation_doi=10.1126/science.284.5420.1670" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Darvasi;citation_author=M. Soller;citation_title=Advanced intercross lines, an experimental population for fine genetic mapping.;citation_pages=1199-1207;citation_volume=141;citation_year=1995;citation_issue=3;citation_pmid=8582624" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=K. Demarest;citation_author=J. Koyner;citation_author=J. McCaughran;citation_author=L. Cipp;citation_author=R. Hitzemann;citation_title=Further characterization and high-resolution mapping of quantitative trait loci for ethanol-induced locomotor activity.;citation_pages=79-91;citation_volume=31;citation_year=2001;citation_issue=1;citation_pmid=11529277;citation_doi=10.1023/A:1010261909853" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=I. A. Diouf;citation_author=L. Derivot;citation_author=F. Bitton;citation_author=L. Pascual;citation_author=M. Causse;citation_title=Water Deficit and Salinity Stress Reveal Many Specific QTL for Plant Growth and Fruit Quality Traits in Tomato.;citation_pages=279;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00279" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Doitsidou;citation_author=S. Jarriault;citation_author=R. J. Poole;citation_title=Next-Generation Sequencing-Based Approaches for Mutation Mapping and Identification in Caenorhabditis elegans;citation_pages=451-474;citation_volume=204;citation_year=2016;citation_issue=2;citation_pmid=27729495;citation_doi=10.1534/genetics.115.186197" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Dupuis;citation_author=D. Siegmund;citation_title=Statistical Methods for Mapping Quantitative Trait Loci From a Dense Set of Markers;citation_pages=373-386;citation_volume=151;citation_year=1999;citation_issue=1;citation_pmid=9872974" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="Falconer, D. S., (1960) Introduction to quantitative genetics. Introduction to Quantitative Genetics. https://www.cabdirect.org/cabdirect/abstract/19601603365" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. J. Fitzpatrick;citation_author=S. Gopalakrishnan;citation_author=E. S. Cogan;citation_author=L. M. Yager;citation_author=P. J. Meyer;citation_title=Variation in the form of Pavlovian conditioned approach behavior among outbred male Sprague-Dawley rats from different vendors and colonies: Sign-tracking vs. goal-tracking.;citation_pages=e75042;citation_volume=8;citation_year=2013;citation_pmid=24098363;citation_doi=10.1371/journal.pone.0075042" />
<meta name="citation_reference" content="citation_journal_title=Blood;citation_author=S. Gardenghi;citation_author=M. F. Marongiu;citation_author=P. Ramos;citation_author=E. Guy;citation_author=L. Breda;citation_title=Ineffective erythropoiesis in β-thalassemia is characterized by increased iron absorption mediated by down-regulation of hepcidin and up-regulation of ferroportin.;citation_volume=109;citation_year=2007;citation_pmid=17299088;citation_doi=10.1182/blood-2006-09-048868" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. M. Gatti;citation_author=K. L. Svenson;citation_author=A. Shabalin;citation_author=L.-Y. Wu;citation_author=W. Valdar;citation_title=Quantitative trait locus mapping methods for diversity outbred mice. G3: Genes, Genomes;citation_pages=1623-1633;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Ghazalpour;citation_author=S. Doss;citation_author=H. Kang;citation_author=C. Farber;citation_author=P.-Z. Wen;citation_title=High-resolution mapping of gene expression using association in an outbred mouse stock.;citation_pages=e1000149-e1000149;citation_volume=4;citation_year=2008;citation_issue=8;citation_pmid=18688273;citation_doi=10.1371/journal.pgen.1000149" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=N. M. Gonzales;citation_author=A. A. Palmer;citation_title=Fine-mapping QTLs in advanced intercross lines and other outbred populations.;citation_pages=271-292;citation_volume=25;citation_year=2014;citation_doi=10.1007/s00335-014-9523-1" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=N. M. Gonzales;citation_author=J. Seo;citation_author=A. I. Hernandez Cordero;citation_author=C. L. St Pierre;citation_author=J. S. Gregory;citation_title=Genome wide association analysis in a mouse advanced intercross line.;citation_pages=5162;citation_volume=9;citation_year=2018;citation_doi=10.1038/s41467-018-07642-8" />
<meta name="citation_reference" content="citation_journal_title=Nat Commun;citation_author=N. M. Gonzales;citation_author=J. Seo;citation_author=A. I. Hernandez-Cordero;citation_author=C. L. S. Pierre;citation_author=J. S. Gregory;citation_title=Genome wide association study of behavioral, physiological and gene expression traits in a multigenerational mouse intercross.;citation_pages=5162;citation_volume=9;citation_year=2017;citation_doi=10.1038/s41467-018-07642-8" />
<meta name="citation_reference" content="citation_journal_title=Journal of Pharmacology and Experimental Therapeutics;citation_journal_abbrev=J. Pharmacol. Exp. Ther.;citation_author=J. H. Graziano;citation_author=R. W. Grady;citation_author=A. Cerami;citation_title=THE IDENTIFICATION OF 2,3-DIHYDROXYBENZOIC ACID AS A POTENTIALLY USEFUL IRON-CHELATING DRUG;citation_pages=570-575;citation_volume=190;citation_year=1974;citation_issue=3;citation_pmid=4416298" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. D. Gruber;citation_author=A. Genissel;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_title=How Repeatable Are Associations Between Polymorphisms in achaete–scute and Bristle Number Variation in Drosophila?;citation_volume=175;citation_year=2007;citation_pmid=17277365;citation_doi=10.1534/genetics.106.067108" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. Han;citation_author=H. M. Kang;citation_author=E. Eskin;citation_title=Rapid and accurate multiple testing correction and power estimation for millions of correlated markers.;citation_pages=e1000456-e1000456;citation_volume=5;citation_year=2009;citation_issue=4;citation_pmid=19381255;citation_doi=10.1371/journal.pgen.1000456" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=L. Han;citation_author=M. Abney;citation_title=Identity by descent estimation with dense genome-wide genotype data.;citation_pages=557-567;citation_volume=35;citation_year=2011;citation_issue=6;citation_pmid=21769932;citation_doi=10.1002/gepi.20606" />
<meta name="citation_reference" content="citation_journal_title=Physiological Reports;citation_journal_abbrev=PHY2;citation_author=A. I. Hernandez Cordero;citation_author=P. Carbonetto;citation_author=G. Riboni Verri;citation_author=J. S. Gregory;citation_author=D. J. Vandenbergh;citation_title=Replication and discovery of musculoskeletal QTLs in LG/J and SM/J advanced intercross lines;citation_pages=e13561-e13561;citation_volume=6;citation_year=2018;citation_issue=4;citation_doi=10.14814/phy2.13561" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=A. I. Hernandez Cordero;citation_author=N. M. Gonzales;citation_author=C. C. Parker;citation_author=G. Sokolof;citation_author=D. J. Vandenbergh;citation_title=Genome-wide Associations Reveal Human-Mouse Genetic Convergence and Modifiers of Myogenesis, CPNE1 and STC2.;citation_pages=1222-1236;citation_volume=105;citation_year=2019;citation_doi=10.1016/j.ajhg.2019.10.014" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Immunology;citation_journal_abbrev=J. Immunol.;citation_author=K. Hoshino;citation_author=O. Takeuchi;citation_author=T. Kawai;citation_author=H. Sanjo;citation_author=T. Ogawa;citation_title=Cutting Edge: Toll-Like Receptor 4 (TLR4)-Deficient Mice Are Hyporesponsive to Lipopolysaccharide: Evidence for TLR4 as the Lps Gene Product;citation_pages=3749-3752;citation_volume=162;citation_year=1999;citation_issue=7;citation_pmid=10201887" />
<meta name="citation_reference" content="citation_journal_title=Cell. Mol. Life Sci.;citation_author=S. Hyun;citation_title=Body size regulation and insulin-like growth factor signaling.;citation_pages=2351-2365;citation_volume=70;citation_year=2013;citation_pmid=23508807;citation_doi=10.1007/s00018-013-1313-5" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Johnsson;citation_author=R. Henriksen;citation_author=A. Höglund;citation_author=J. Fogelholm;citation_author=P. Jensen;citation_title=Genetical genomics of growth in a chicken model.;citation_pages=72;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-4441-3" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. W. J. Joo;citation_author=F. Hormozdiari;citation_author=B. Han;citation_author=E. Eskin;citation_title=Multiple testing correction in linear mixed models.;citation_pages=62;citation_volume=17;citation_year=2016;citation_pmid=27039378;citation_doi=10.1186/s13059-016-0903-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. R. Keele;citation_author=W. L. Crouse;citation_author=S. N. P. Kelada;citation_author=W. Valdar;citation_title=Determinants of QTL Mapping Power in the Realized Collaborative Cross. G3: Genes, Genomes;citation_pages=1707-1727;citation_volume=9;citation_year=2019;citation_doi=10.1534/g3.119.400194" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K.-A. Kim;citation_author=W. Gu;citation_author=I.-A. Lee;citation_author=E.-H. Joh;citation_author=D.-H. Kim;citation_title=High Fat Diet-Induced Gut Microbiota Exacerbates Inflammation and Obesity in Mice via the TLR4 Signaling Pathway.;citation_pages=e47713-e47713;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23091640;citation_doi=10.1371/journal.pone.0047713" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=E. G. King;citation_author=A. D. Long;citation_title=The Beavis Effect in Next-Generation Mapping Panels in Drosophila melanogaster. G3: Genes, Genomes;citation_pages=1643-1652;citation_volume=7;citation_year=2017;citation_doi=10.1534/g3.117.041426" />
<meta name="citation_reference" content="citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. Kislukhin;citation_author=E. G. King;citation_author=K. N. Walters;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_title=The genetic architecture of methotrexate toxicity is similar in Drosophila melanogaster and humans.;citation_pages=1301-1310;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=T. S. Korneliussen;citation_author=A. Albrechtsen;citation_author=R. Nielsen;citation_title=ANGSD: Analysis of next generation sequencing data.;citation_pages=356;citation_volume=15;citation_year=2014;citation_pmid=25420514;citation_doi=10.1186/s12859-014-0356-4" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=K.-O. Lai;citation_author=Y. Chen;citation_author=H.-M. Po;citation_author=K.-C. Lok;citation_author=K. Gong;citation_title=Identification of the Jak/Stat Proteins as Novel Downstream Targets of EphA4 Signaling in Muscle: implications in the regulation of acetylcholinesterase expression.;citation_volume=279;citation_year=2004;citation_pmid=14729671;citation_doi=10.1074/jbc.M313356200" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Genomics;citation_author=A. Lionikas;citation_author=R. Cheng;citation_author=J. E. Lim;citation_author=A. A. Palmer;citation_author=D. A. Blizard;citation_title=Fine-mapping of muscle weight QTL in LG/J and SM/J intercrosses.;citation_volume=42A;citation_year=2010;citation_pmid=20627939;citation_doi=10.1152/physiolgenomics.00100.2010" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. Listgarten;citation_author=C. Lippert;citation_author=C. M. Kadie;citation_author=R. I. Davidson;citation_author=E. Eskin;citation_title=Improved linear mixed models for genome-wide association studies.;citation_pages=525-526;citation_volume=9;citation_year=2012;citation_issue=6;citation_pmid=22669648;citation_doi=10.1038/nmeth.2037" />
<meta name="citation_reference" content="citation_journal_title=Genes, brain, and behavior;citation_journal_abbrev=Genes Brain Behav;citation_author=R. W. Logan;citation_author=R. F. Robledo;citation_author=J. M. Recla;citation_author=V. M. Philip;citation_author=J. A. Bubier;citation_title=High-precision genetic mapping of behavioral traits in the diversity outbred mouse population.;citation_pages=424-437;citation_volume=12;citation_year=2013;citation_pmid=23433259;citation_doi=10.1111/gbb.12029" />
<meta name="citation_reference" content="citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. J. Macdonald;citation_author=A. D. Long;citation_title=A Potential Regulatory Polymorphism Upstream of hairy Is Not Associated With Bristle Number Variation in Wild-Caught Drosophila;citation_pages=2127-2131;citation_volume=167;citation_year=2004;citation_issue=4;citation_pmid=15342546;citation_doi=10.1534/genetics.104.026732" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. Mangin;citation_author=B. Goffinet;citation_author=A. Rebai;citation_title=Constructing confidence intervals for QTL location.;citation_pages=1301-1308;citation_volume=138;citation_year=1994;citation_issue=4;citation_pmid=7896108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Manichaikul;citation_author=J. Dupuis;citation_author=S. Sen;citation_author=K. W. Broman;citation_title=Poor performance of bootstrap confidence intervals for the location of a quantitative trait locus.;citation_volume=174;citation_year=2006;citation_pmid=16783000;citation_doi=10.1534/genetics.106.061549" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. N. Marriage;citation_author=E. G. King;citation_author=A. D. Long;citation_author=S. J. Macdonald;citation_title=Fine-Mapping Nicotine Resistance Loci in Drosophila Using a Multiparent Advanced Generation Inter-Cross Population;citation_pages=45-57;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236448;citation_doi=10.1534/genetics.114.162107" />
<meta name="citation_reference" content="citation_journal_title=Genes, brain, and behavior;citation_journal_abbrev=Genes Brain Behav;citation_author=T. R. Mhyre;citation_author=E. J. Chesler;citation_author=M. Thiruchelvam;citation_author=C. Lungu;citation_author=D. A. Cory‐Slechta;citation_title=Heritability, correlations and in silico mapping of locomotor behavior and neurochemistry in inbred strains of mice.;citation_pages=209-228;citation_volume=4;citation_year=2005;citation_issue=4;citation_pmid=15924554;citation_doi=10.1111/j.1601-183X.2004.00102.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. A. Najarro;citation_author=J. L. Hackett;citation_author=B. R. Smith;citation_author=C. A. Highfill;citation_author=E. G. King;citation_title=Identifying Loci Contributing to Natural Variation in Xenobiotic Resistance in Drosophila.;citation_pages=e1005663;citation_volume=11;citation_year=2015;citation_doi=10.1371/journal.pgen.1005663" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Nicod;citation_author=R. W. Davies;citation_author=N. Cai;citation_author=C. Hassett;citation_author=L. Goodstadt;citation_title=Genome-wide association of multiple complex traits in outbred mice by ultra-low-coverage sequencing.;citation_pages=912-918;citation_volume=48;citation_year=2016;citation_doi=10.1038/ng.3595" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=I. Nikolskiy;citation_author=D. F. Conrad;citation_author=S. Chun;citation_author=J. C. Fay;citation_author=J. M. Cheverud;citation_title=Using whole-genome sequences of the LG/J and SM/J inbred mouse strains to prioritize quantitative trait genes and nucleotides.;citation_pages=415;citation_volume=16;citation_year=2015;citation_pmid=26016481;citation_doi=10.1186/s12864-015-1592-3" />
<meta name="citation_reference" content="citation_journal_title=Behavior research methods, instruments, &amp; computers :  a journal of the Psychonomic Society, Inc;citation_journal_abbrev=Behav Res Methods Instrum Comput;citation_author=L. P. J. J. Noldus;citation_author=A. J. Spink;citation_author=R. A. J. Tegelenbosch;citation_title=EthoVision: a versatile video tracking system for automation of behavioral experiments.;citation_pages=398-414;citation_volume=33;citation_year=2001;citation_issue=3;citation_pmid=11591072;citation_doi=10.3758/BF03195394" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. C. Parker;citation_author=P. Carbonetto;citation_author=G. Sokoloff;citation_author=Y. J. Park;citation_author=M. Abney;citation_title=High-Resolution Genetic Mapping of Complex Traits from a Combined Analysis of F2 and Advanced Intercross Mice;citation_pages=103-116;citation_volume=198;citation_year=2014;citation_issue=1;citation_pmid=25236452;citation_doi=10.1534/genetics.114.167056" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=C. C. Parker;citation_author=R. Cheng;citation_author=G. Sokoloff;citation_author=J. E. Lim;citation_author=A. D. Skol;citation_title=Fine-mapping alleles for body weight in LG/J Ã SM/J F2 and F(34) advanced intercross lines.;citation_pages=563-571;citation_volume=22;citation_year=2011;citation_issue=9-10;citation_pmid=21761260;citation_doi=10.1007/s00335-011-9349-z" />
<meta name="citation_reference" content="citation_journal_title=Genes, brain, and behavior;citation_journal_abbrev=Genes Brain Behav;citation_author=C. C. Parker;citation_author=R. Cheng;citation_author=G. Sokoloff;citation_author=A. A. Palmer;citation_title=Genome-wide association for methamphetamine sensitivity in an advanced intercross mouse line.;citation_pages=52-61;citation_volume=11;citation_year=2012;citation_issue=1;citation_pmid=22032291;citation_doi=10.1111/j.1601-183X.2011.00747.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=C. C. Parker;citation_author=S. Gopalakrishnan;citation_author=P. Carbonetto;citation_author=N. M. Gonzales;citation_author=E. Leung;citation_title=Genome-wide association study of behavioral, physiological and gene expression traits in outbred CFW mice.;citation_pages=919-926;citation_volume=48;citation_year=2016;citation_doi=10.1038/ng.3609" />
<meta name="citation_reference" content="citation_journal_abbrev=Front Genet;citation_author=C. C. Parker;citation_author=A. A. Palmer;citation_title=Dark matter: are mice the solution to missing heritability?;citation_pages=32-32;citation_volume=2;citation_year=2011;citation_pmid=22303328;citation_doi=10.3389/fgene.2011.00032" />
<meta name="citation_reference" content="citation_title=Molecular Dissection of Complex Traits;citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. J. Pruim;citation_author=R. P. Welch;citation_author=S. Sanna;citation_author=T. M. Teslovich;citation_author=P. S. Chines;citation_title=LocusZoom: Regional visualization of genome-wide association scan results.;citation_volume=26;citation_year=2010;citation_pmid=20634204;citation_doi=10.1093/bioinformatics/btq419" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=A. Baud;citation_author=R. Hermsen;citation_author=V. Guryev;citation_author=P. Stridh;citation_author=D. Graham;citation_title=Combined sequence-based and genetic mapping analysis of complex traits in outbred rats.;citation_pages=767-775;citation_volume=45;citation_year=2013;citation_pmid=23708188;citation_doi=10.1038/ng.2644" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=F. Relaix;citation_author=D. Montarras;citation_author=S. Zaffran;citation_author=B. Gayraud-Morel;citation_author=D. Rocancourt;citation_title=Pax3 and Pax7 have distinct and overlapping functions in adult muscle progenitor cells.;citation_volume=172;citation_year=2006;citation_pmid=16380438;citation_doi=10.1083/jcb.200508044" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Environ.;citation_author=L. Rishmawi;citation_author=J. Bühler;citation_author=B. Jaegle;citation_author=M. Hülskamp;citation_author=M. Koornneef;citation_title=Quantitative trait loci controlling leaf venation in Arabidopsis.;citation_pages=1429-1441;citation_volume=40;citation_year=2017;citation_doi=10.1111/pce.12938" />
<meta name="citation_reference" content="citation_journal_title=Genes, brain, and behavior;citation_journal_abbrev=Genes Brain Behav;citation_author=K. E. Samocha;citation_author=J. E. Lim;citation_author=R. Cheng;citation_author=G. Sokoloff;citation_author=A. A. Palmer;citation_title=Fine mapping of QTL for prepulse inhibition in LG/J and SM/J mice using F(2) and advanced intercross lines.;citation_pages=759-767;citation_volume=9;citation_year=2010;citation_issue=7;citation_pmid=20597988;citation_doi=10.1111/j.1601-183X.2010.00613.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=T. Sawada;citation_author=D. Arai;citation_author=X. Jing;citation_author=M. Miyajima;citation_author=S. J. Frank;citation_title=Molecular interactions of EphA4, growth hormone receptor, Janus kinase 2, and signal transducer and activator of transcription 5B.;citation_pages=e0180785;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0180785" />
<meta name="citation_reference" content="citation_journal_title=Journal of Clinical Investigation;citation_journal_abbrev=JCI;citation_author=H. Shi;citation_author=M. V. Kokoeva;citation_author=K. Inouye;citation_author=I. Tzameli;citation_author=H. Yin;citation_title=TLR4 links innate immunity and fatty acid-induced insulin resistance.;citation_pages=3015-3025;citation_volume=116;citation_year=2006;citation_issue=11;citation_pmid=17053832;citation_doi=10.1172/JCI28898" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=L. J. Sittig;citation_author=P. Carbonetto;citation_author=K. A. Engel;citation_author=K. S. Krauss;citation_author=C. M. Barrios-Camacho;citation_title=Genetic Background Limits Generalizability of Genotype-Phenotype Relationships.;citation_pages=1253-1259;citation_volume=91;citation_year=2016;citation_pmid=27618673;citation_doi=10.1016/j.neuron.2016.08.013" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=R. E. Sorge;citation_author=L. J. Martin;citation_author=K. A. Isbester;citation_author=S. G. Sotocinal;citation_author=S. Rosen;citation_title=Olfactory exposure to males, including men, causes stress and related analgesia in rodents.;citation_pages=629-632;citation_volume=11;citation_year=2014;citation_pmid=24776635;citation_doi=10.1038/nmeth.2935" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. L. Svenson;citation_author=D. M. Gatti;citation_author=W. Valdar;citation_author=C. E. Welsh;citation_author=R. Cheng;citation_title=High-resolution genetic mapping using the Mouse Diversity outbred population.;citation_volume=190;citation_year=2012;citation_pmid=22345611;citation_doi=10.1534/genetics.111.132597" />
<meta name="citation_reference" content="citation_journal_title=Immunity;citation_journal_abbrev=Immunity;citation_author=O. Takeuchi;citation_author=K. Hoshino;citation_author=T. Kawai;citation_author=H. Sanjo;citation_author=H. Takada;citation_title=Differential roles of TLR2 and TLR4 in recognition of gram-negative and gram-positive bacterial cell wall components.;citation_pages=443-451;citation_volume=11;citation_year=1999;citation_issue=4;citation_pmid=10549626;citation_doi=10.1016/S1074-7613(00)80119-3" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. J. Talbot;citation_author=A. Nicod;citation_author=S. S. Cherny;citation_author=D. W. Fulker;citation_author=A. C. Collins;citation_title=High-resolution mapping of quantitative trait loci in outbred mice.;citation_pages=305-308;citation_volume=21;citation_year=1999;citation_issue=3;citation_pmid=10080185;citation_doi=10.1038/6825" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=L. Toker;citation_author=M. Feng;citation_author=P. Pavlidis;citation_title=Whose sample is it anyway? Widespread misannotation of samples in transcriptomics studies.;citation_pages=2103;citation_volume=5;citation_year=2016;citation_doi=10.12688/f1000research.9471.1" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=W. Valdar;citation_author=L. C. Solberg;citation_author=D. Gauguier;citation_author=S. Burnett;citation_author=P. Klenerman;citation_title=Genome-wide genetic association of complex traits in heterogeneous stock mice.;citation_pages=879-887;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16832355;citation_doi=10.1038/ng1840" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. M. Visscher;citation_author=W. G. Hill;citation_author=N. R. Wray;citation_title=Heritability in the genomics era--concepts and misconceptions.;citation_pages=255-266;citation_volume=9;citation_year=2008;citation_issue=4;citation_pmid=18319743;citation_doi=10.1038/nrg2322" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=P. M. Visscher;citation_author=N. R. Wray;citation_author=Q. Zhang;citation_author=P. Sklar;citation_author=M. I. McCarthy;citation_title=10 Years of GWAS Discovery: Biology, Function, and Translation.;citation_pages=5-22;citation_volume=101;citation_year=2017;citation_pmid=28686856;citation_doi=10.1016/j.ajhg.2017.06.005" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. C. Vonesch;citation_author=D. Lamparter;citation_author=T. F. Mackay;citation_author=S. Bergmann;citation_author=E. Hafen;citation_title=Genome-wide analysis reveals novel regulators of growth in Drosophila melanogaster.;citation_pages=e1005616;citation_volume=12;citation_year=2016;citation_pmid=26751788;citation_doi=10.1371/journal.pgen.1005616" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. B. Maller;citation_author=G. McVean;citation_author=J. Byrnes;citation_author=D. Vukcevic;citation_author=K. Palin;citation_title=Bayesian refinement of association signals for 14 loci in 3 common diseases.;citation_pages=1294-1301;citation_volume=44;citation_year=2012;citation_pmid=23104008;citation_doi=10.1038/ng.2435" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Xu;citation_title=Theoretical Basis of the Beavis Effect;citation_pages=2259-2268;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704201;citation_doi=10.1038/hdy.1992.131" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Yalcin;citation_author=S. A. Willis-Owen;citation_author=J. Fullerton;citation_author=A. Meesaq;citation_author=R. M. Deacon;citation_title=Genetic dissection of a behavioral quantitative trait locus shows that Rgs2 modulates anxiety in mice.;citation_pages=1197-1202;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15489855;citation_doi=10.1038/ng1450" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Yang;citation_author=S. H. Lee;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_title=GCTA: a tool for genome-wide complex trait analysis.;citation_pages=76-82;citation_volume=88;citation_year=2011;citation_issue=1;citation_pmid=21167468;citation_doi=10.1016/j.ajhg.2010.11.011" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Yang;citation_author=N. A. Zaitlen;citation_author=M. E. Goddard;citation_author=P. M. Visscher;citation_author=A. L. Price;citation_title=Advantages and pitfalls in the application of mixed-model association methods.;citation_pages=100-106;citation_volume=46;citation_year=2014;citation_pmid=24473328;citation_doi=10.1038/ng.2876" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=J. Yang;citation_author=J. Zeng;citation_author=M. E. Goddard;citation_author=N. R. Wray;citation_author=P. M. Visscher;citation_title=Concepts, estimation and interpretation of SNP-based heritability.;citation_pages=1304-1310;citation_volume=49;citation_year=2017;citation_pmid=28854176;citation_doi=10.1038/ng.3941" />
<meta name="citation_reference" content="citation_journal_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_author=N. Zaitlen;citation_author=P. Kraft;citation_title=Heritability in the genome-wide association era.;citation_pages=1655-1664;citation_volume=131;citation_year=2012;citation_issue=10;citation_pmid=22821350;citation_doi=10.1007/s00439-012-1199-6" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Zhou;citation_author=M. Stephens;citation_title=Genome-wide efficient mixed-model analysis for association studies.;citation_pages=821-824;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706312;citation_doi=10.1038/ng.2310" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Zöllner;citation_author=J. K. Pritchard;citation_title=Overcoming the winner's curse: estimating penetrance parameters from case-control data.;citation_pages=605-615;citation_volume=80;citation_year=2007;citation_issue=4;citation_pmid=17357068;citation_doi=10.1086/512821" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=J. Zou;citation_author=J. Zhou;citation_author=S. Faller;citation_author=R. Brown;citation_author=E. Eskin;citation_title=Accurate modeling of replication rates in genome-wide association studies by accounting for winner’s curse and study-specific heterogeneity.;citation_volume=21;citation_year=2019;citation_doi=10.1101/856898" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="twitter:description" content="There has been extensive discussion of the “Replication Crisis” in many fields, including genome-wide association studies ( GWAS ). We explored replication in a mouse model using an advanced intercross line ( AIL ), which is a multigenerational intercross between two inbred strains. We re-genotyped a previously published cohort of LG/J x SM/J AIL mice (F34; n = 428) using a denser marker set and genotyped a new cohort of AIL mice (F39-43; n = 600) for the first time. We identified 36 novel genome-wide significant loci in the F34 and 25 novel loci in the F39-43 cohort. The subset of traits that were measured in both cohorts (locomotor activity, body weight, and coat color) showed high genetic correlations, although the SNP heritabilities were slightly lower in the F39-43 cohort. For this subset of traits, we attempted to replicate loci identified in either F34 or F39-43 in the other cohort. Coat color was robustly replicated; locomotor activity and body weight were only partially replicated, which was inconsistent with our power simulations. We used a random effects model to show that the partial replications could not be explained by Winner’s Curse but could be explained by study-specific heterogeneity. Despite this heterogeneity, we performed a mega-analysis by combining F34 and F39-43 cohorts (n = 1,028), which identified four novel loci associated with locomotor activity and body weight. These results illustrate that even with the high degree of genetic and environmental control possible in our experimental system, replication was hindered by study-specific heterogeneity, which has broad implications for ongoing concerns about reproducibility." />
<meta name="og-title" property="og:title" content="Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/3/951" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="There has been extensive discussion of the “Replication Crisis” in many fields, including genome-wide association studies ( GWAS ). We explored replication in a mouse model using an advanced intercross line ( AIL ), which is a multigenerational intercross between two inbred strains. We re-genotyped a previously published cohort of LG/J x SM/J AIL mice (F34; n = 428) using a denser marker set and genotyped a new cohort of AIL mice (F39-43; n = 600) for the first time. We identified 36 novel genome-wide significant loci in the F34 and 25 novel loci in the F39-43 cohort. The subset of traits that were measured in both cohorts (locomotor activity, body weight, and coat color) showed high genetic correlations, although the SNP heritabilities were slightly lower in the F39-43 cohort. For this subset of traits, we attempted to replicate loci identified in either F34 or F39-43 in the other cohort. Coat color was robustly replicated; locomotor activity and body weight were only partially replicated, which was inconsistent with our power simulations. We used a random effects model to show that the partial replications could not be explained by Winner’s Curse but could be explained by study-specific heterogeneity. Despite this heterogeneity, we performed a mega-analysis by combining F34 and F39-43 cohorts (n = 1,028), which identified four novel loci associated with locomotor activity and body weight. These results illustrate that even with the high degree of genetic and environmental control possible in our experimental system, replication was hindered by study-specific heterogeneity, which has broad implications for ongoing concerns about reproducibility." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
    <title>Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"spknhN-ZunrSiQ0Qp4p983IdwOUm7nQg-YC7lKuMTl0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495847","apath":"\/ggg\/10\/3\/951.atom","pisa":"ggg;10\/3\/951","ac":{"\/ggg\/10\/3\/951.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/3\/951.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/3\/951"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/3\/951"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/3\/951?rss=1":true,"\/content\/10\/3\/951":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ufcWKiVgfnmv5lTYWOnsQZPI8xmlppqtItnRds5H9zhABuuVSar7Cyfv+RHhHYKcc7aS0VLAUyPRs+8k69Ydmcm+ifvJvgGKiCzyT490c01sHcqEBfD4jvGVIBQd3yyXdR2otnmaNW7AS+QshkCV9dZlH\\\\\\\/8TiYSulSxWwAzoSBXq6+69QbK5yaAuRUfufhHRbb39NfCEkbVJ49foEEj\\\\\\\/MoJyIBbMc1NzW4WoUTMQCJ5My+oHVTdLwrsIrzuk7xkWRNGapgtHGDZi\\\\\\\/G+xndcUHMWdP2jUXd2robLvvA3+mdo8sQCOHD1uTXWzpHhdGR\\\\\\\/gXT9I2hluaXNt34DaCzpD1o5szTueHwGeRiD2BF4gqGfeBuCL82UrE1lGgdYX0CbkAFtLxqEuHn\\\\\\\/8vMAXSYggd2SQJeB44roVsXkaAql1IOTcRxKVd0aebasJWL2A0jMRIhmTvJ224qP+t6rf3EQZOLC6kiVm3rTJHemZsN8QqSYw2S0RzCotZ\\\\\\\/GgQxXv+hr9mG7wmKDCDUO+2s+nRa9BC9Syq\\\\\\\/AbC\\\\\\\/98LtD1Ymeotf5J6Pn6fvsN0Ulu6cvfhcf0TI5ydQMVXV+M1J2ZQ8aGDFW36FzX7+iV9BPhMQLv0Wu+vfhYFgCqtljlq6K1\\\\\\\/uVyTt5x4lswnIq7hmo5+fWpjYfcHQV4sZg+PSu88rseON2CviYXKdPx0KV5Pfa5ybBwqAku5BkSLyO0uPL\\\\\\\/1acK5p6BU3z9Cf3eaXC5m3T+0nf8NsdCMduXXPehqlEFuck0rDWVPrOghItfBOePEq5uwNZdLsmwK+OOhVO5fPB55w00C0JcPWe\\\\\\\/UZlTXafYYBHY71BwQPpSreA47fEq0wz2fTt8bRX9YF9A0RS5zUHXa01xhmc7PDh3I0O0d8cFTELsh75WsjChPLBNXlll0G9mN\\\\\\\/u1skiDYSk85KiDk6jZdjJ7A58L8h1sDWM6ObSGhq1TM5\\\\\\\/egb4Y00RSvKGH+\\\\\\\/IsZhxMCkyvT0MK8OFwgoGXR+YjcfKVROXefmKA3jiAmzOzRy\\\\\\\/uGjiUltvmahGzstrPYfwncFNHMmyfHnoGjmniKf3lFsUqagSocqp0BXPc85PC7I0X3R91qMC8bl1k4f6\\\\\\\/tGVCpTu9wmcFkW4DhYIm8+sphOc6gqoD5yN+pL9al\\\\\\\/tBMP6ZH\\\\\\\/m9Y7R+h0Z7gx2t+czYdc62kwynJ1YsatgtKcONbaJpTN2J\\\\\\\/233kKG4qc2jbqHIDdydtcfzumvMj5m6CpB4gUOGkRBtWJ+UOIbiuLXLDJAzzGj05YAFSCVONV0D\\\u0022,\\\u0022iv\\\u0022:\\\u0022cKfUE1Ffcap+IvLZiXVpOA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b475568a2333a13a1025edabf701fb3ce82987559652e2ce8cef3edac3169025\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022N+GYU+y5v4a72nhnYSSiOMbQq5PmPY30DR0eQ7IDlZsz+3R7SOMLxlFCZGIepV\\\\\\\/uh440u7aZm8b+Bo40fg2GJECJ7RWHD7L0dMR\\\\\\\/Rh\\\\\\\/22J0ODTReV24v9nKNBth2BJWqw0CBiGuSXX3AXkhfEiFW8dxTpIAuJ3dXGyF2FfesTAWNTym1N6S7bkuAJDPvHnLJ8eAl6tewIKwmyLmEbdSPH2Y1gf+74DR20\\\\\\\/IIH581flqGZNmAvETlx3epMQP9mA75E6dwIF9vXeJK04cpbYZEmo6EDsGrvVyowS3SvMKCM4QvlhGgGJQusnaK0\\\\\\\/0tOCxfF2dOOVvMhUYYDYfMQMd\\\\\\\/0nvAmVv+NdJy5enxfLHSHgkRHOz6TRTCigBCJ8bzqWHTGkYt6jHcpNAjBtnLm1DYUwbwrHLKfM\\\\\\\/oXHypQ432NB2Db4lftozpK27amlB5mfzuFYWLs20fpZzTnIRpFWwgSFMwU+RJ+U44d1SHrXyaLkJTsTzVET83y+rhBDTcyzUwvyV7HRp8YVY1N\\\\\\\/Ie2NoXeUIeunNGQpqFkE9ejE6EsBRw6Sgs+k7JedVV3X5DxMEH\\\\\\\/kVn2MvrOLiFAeINIqHSuSaM7TRicrV9kVFV+xaX9XJwbjbI9gfhLm6TZlzVm5T0ZR9564DnU9PRZt9Gb6b8Sc8NwBdWJwezMRVbe8PD73i\\\\\\\/tD\\\\\\\/laHukJw4+GCS8R93\\\\\\\/EbnWOr29Or+d6IntLSpd+EGie96dip3sCliuzEo7q4Aejy1QF0bU+oHQ5NVyLQDDJgI80QSA1R6B\\\\\\\/\\\\\\\/+3xkCZBNGgJB\\\\\\\/hIZ\\\\\\\/ixKxVQX2rKkV8W+IVWZrMXsIIXCAG4Rle2F0iGvDx+t8ihfy94PfqAavbRFtrjb5MKCT89HI1qZP5wrD2MS0emQtcmYF6UIqiql95iDApcaZq0SfdMJOa\\\\\\\/OWYm0KBVwEQx7YWQv2QV5jE7bb2RfWE749RzGVzs0qk4Qcqn5bn2UZ2XCQA6sTCHfI0tX72r\\\\\\\/7BGnEbmfsp0QdNW5T\\\\\\\/w+FLmGhEWT13838aoN9k9bNAtP5Lyw\\\\\\\/C4IHDNS9jiqI9sBTvibMCgjOXZDiJ7+Bcnq9v5sHUluvQ0jqmZCtV1NxF9gxuVI1XqmQyZWnD6ohe2eRg76Ik6RhPgJ0ZNhfWzvWxE1XEI5pe9KBhNXBB55uQmHsu75ioUciPqWnbB9aYvuMWxarUZM4LZRBuGMU+fTCBO+Lsp4tC\\\\\\\/2k\\\\\\\/pVhT9IfEvSxs3ZUIFn3jFqUcF00X2xFUbW328T+LQ+fAtFFiVkE1Ibn0rYUwjCd40j1Lk7xalaDrE5IuSMlS9NmaeUzjnKTU0nB059lbfRDgfH\\\\\\\/5Lr3gD81p8RoLUs0RdvdxR5FRstGL33mODNR8LvN6f8LuxiDhI79GnAjlP1zjpQ\\\\\\\/2lhvl9PzbhZc6WUX6nSoeGQweMypXmzRI+aIAnlOtXrWgRvgNwWiXDRGtiVk5X8Fm7GtcSMBFRW7vnIhw\\\\\\\/CgYhhrBDoUKEJ3LGrl+E\\\\\\\/g0Dm6fKc\\\\\\\/ECZgSHx\\\\\\\/hj1dlDSPysrc9KKle3B8auHmk\\\\\\\/saEuaBg41yXFnQHprwna9VdPOiPRNbpmnQoxIcm4eyhTvSQUfjNn+gB8JfciFbuHZS2n6Kv21PP2lvR8FEI45r6XtGU5YBTTs2PuP\\\\\\\/TTZNALZwS6OkxVq3GvdQTZxka4ocJH3ZsxkxntrEqvWesiicJnGZPW4y5X7Ov+9ZBsvg\\\\\\\/1mmmYVAAwgNAVqTmQAfjOeXrSmyoR+b7mmvMoHmkj6HW86eQ048bpbLEl4phkWv8eD7kMbcPNPpCO\\\\\\\/tfEMcHO6WCQ+inU0mjMVnPwRWtws\\\\\\\/zRyMbtPICVm1PHZrCU4qBky5SDVAY4o1mQLCfqLHFw+htzngH18ht2GzMeXmlFHTTQLzBn+9ZRIJ\\\\\\\/ijJ8G+b1kcKpE0Z9u0VDlHg6onluKOo1O0Ussz90twYKwabA5DqV0s2yTie0zyewjTSqCq+hTZDvjjGAv638kwcnhxKmBMCyAqCJtB+NVh2qY6MlKAwyBkf4Wzfnrm3gfoC4YVUvtqoLwbV4e+Dcg35i0ihBhF2xzTya8zpPnHkycrGMxyumb8HwkFrS0OyQbUS641gkjhXiH8BqVIXDBBVIGxJhRRdjspagMDCTIrCEztXRCgF3ONYDcY3mVZ6Hb36Z6Ox0F4l2C0r0nsDf9Or1Ked5bwUc5S6O6LmHKxUDXy+OexdjUyW1Rq1NE57Txvp9X5RR4kGSQmiW+nmEhcZTNz4ryprWAhpbRaoVBIo4t1p64Zsr+0jTNpzMK\\\\\\\/JkIiQCVlfJwkqNA4z0IcC4GYVyD1TjrEAd5E+TadZ+LoX5MjXH8aLXmUJQIahOncDdgB2spCcOQ3n4YOHkIoBmgseDX3ttRffcDwIpf5MQngQKK5M\\\\\\\/CDvglTN\\\\\\\/BTmxYKaJWW89+\\\\\\\/5k4m\\\\\\\/JtJ\\\\\\\/viSA2Jb4m0ocu7i0CbZcCw84R5ZtAXUhe3G207iLXEult5L0wam75\\\\\\\/GbMU6YdGTO0OvDTqg8Ropq+OvAGTUB0ORY2sJB50iXuss8I7S1GJQv+52pwnL\\\\\\\/SkVpHJKo2DVu1C+60uaHm09gfOrYhIDNtS1Hc+mPGDMdO2NcXJfAbOiRFj1iVgeN+ZOHtWJ3w9x\\\\\\\/Vpw4hIZyW6\\\\\\\/N2AqhTD+BAv652AESgJHvgaiczEkcMrHAp7k2IoCsYBZ2T7+RtRcZn8cw848FTNEi7rZeVpSiUuy2dDpeXAHE6ixqlfw6Wc0S5oDO2KoLEeVA12yHnW6eI\\\\\\\/0iXFmzJRAzVopEOtwGEI4wL6MZHKspPq52oHO1c9NwgVJNgVWD54DA5Dl5CJR\\\\\\\/Uh8swuuSXYlHqgJmkaOG9u1vj4Dhae1Cp1Ws8eszkjqNtj075fVFjKet9BbHrNEAG+gspI\\\\\\\/SVPnFIEZtew\\\\\\\/94NSmhgHIvVBoDhPw1HYHXC5dtHf10aW3dKTPiJAhnP+ydcTZBa3iZk66BdGe4djaewzuBPTGA5PM11W+dLNYyC1ebuLuC3wHLzfuj7wnc1cnMBRTc5VXOV2Zzc9Na0uunt4e4FfECZVDI9w6vGNs8GG\\\\\\\/mayJOIPA7B7JMkMM9093LucMOBmQjFZ2d6I+E8X1PdLZAsomHw8r2vrs2tHrtC\\\\\\\/iAKMnBJhvqdhZGjGdzq3XchlCTZT1Pb\\\\\\\/eyDOVFqnzfMg\\\\\\\/t5sK5WXMK+AvnDfztTOZ4J+Un59PkeNmRUY8W8gi49eqI2Rv7nimAOYIhgMm18wntXYL7KjUdRdbv+LifW7F32wj4czuaulE6UZCyWoESqoLN8e7h9P7p1y2BKBmrWPxQTMnau2APYAHTdDeJZs4vF7gKA5hbocWTB63VPD7aPAFTAcfWIc5Cb9lLUhCRVGForhQCPhVKX9924XNqvL3LUBUclVrWPGLoxA4rELO4IcRCLzNCnoe0Md4463u1smB3lstI4hH8nXbQTpnTuqhOIQyw\\\\\\\/QBy9Q4WqY+aNGluUlid9McdqeXK9FLkWFIOTk9jQz4Fvg+0PaCbNVxvd4+qM49Khyjf6OuXuztpOoa6pUwqBZdee3BEj1Cf3sx6zB98ryMsi4zY158s5FtQkQX7lfIiVqUTMhc89y+MftgQWYITxlXjLozTFG9Hlx\\\\\\\/yKCRz9akI8gffOhBkBzEzyZuAn7QoDNAi\\\\\\\/czIzTZc\\\\\\\/PPXNfRLLWwmKfS6CjeHol0h+LD14YDzDGHVdaoThBcI0k0BFPDGKWIw\\\\\\\/ev9DYVjGlguWAMp8v5Vu1pj\\\\\\\/TMPHrqVYmg7CuSQXokYToxYd3lzyqtQXSDB0bOf\\\\\\\/h5g2adgGnkXfmojUfdXQtm4VwfR6jU1BbBWZsEE2xRzfaP7Z89BxLP2skimH5lF\\\\\\\/aejmtMaaRpzDdkD0C\\\\\\\/rhE34vRzvPTJ+2GMpAjngE47gEIZhvtl3naqf70ljCxdOn807Je11g+3SJWQkde42P9yDdtgCyvKuM+dCK0xur29OYOhkvNNKBwM6pq5tcz+\\\\\\\/FhUUR1KlThPGB8F6oO1+rBnDJEJru1yv54l8fw4Y4+afEc1j1asxqInZn1UnAy3N8gzL5rLD+TpzdkLsJS7A1i5CRwBK\\\\\\\/ezYYMNaXnYL6\\\\\\\/LnZvZO7ueIOr9djRki\\\\\\\/FdLyugDDGAl2sya+aBoMqIYiRDHOzh9JkVKCA9gSjy1XTqaxy2HgjHF+xOUweDrVDdM7UFUK+bTr\\\\\\\/JhmKMm3DXSEhv7kOZHgExrOJ9CmRVgflKrt5e8f39qDba1vIeieXdTYWmngm2JeTWmRaXLOhct79G4G1JF5EE3zy0EmpctGknysevvu+9Dzr\\\\\\\/eHtGsecm5CEF3rpTghRlxNOy0Cljti6N4p1OQgskEAKCLxpk4wLSj9599C5rBuBtMJLdaCkZn+KWpGAF6CIy6vgmBV4SW64Vyy2k03fDmG1fX1q8zlMEOBnSI2ospktbkiv3BEznkX7aNTU9dgyGoZia9su+NvGR9FlDmz2jg6gETBmdKhVlJPeJnR9JQZV\\\\\\\/79Vx8a5KVH5PWe8oZ1PiyLIENT8s6S4LG7bTrOuMymCtm+nD3E\\\\\\\/O\\\\\\\/JuHE4uvRKeVtSQz253mFZYlNJHmgdFWhHBEKnZvvun51HquxjS+LrimliP9wiif1\\\\\\\/FxGwUjiJnmviSiD3Z\\\\\\\/U2d2+yMkp1OgAin5cuAt0okGyFPQEYQITYXPuAot2hffbi\\\\\\\/RLFUohIxnG+pUmrlA1TeswWb3w+AHpvWr17ltufVovthp4s6f2sKa+neY6Kj\\\\\\\/fZsyih4O8oUA8DzN\\\\\\\/dCPH97TrIAdF9Uj5bfeFMX\\\\\\\/TQVdNJwFSba8fYuyNdXhOX4tz3ES8WXj9Ewh8aIlMdSDQ4eM5Uv75JZP2QApMB4TkddTrO8fqkq++Xef630evclT9iLssn72q3fjBr2DeEQb71k8tA0wqSgIXwfnVztWtiS24eiBa+L4\\\\\\\/Iia4fcXMeIdsIVrqbxDKs9eXcaxVOy1qnwvBeMOmnzWQQvv5LTQ5smOMRwy0kiBxAFOjE18H8rYHFNQkZHSb+1qYo5W2eDcieOClkOQQXqZekCjPKmiSBwxWaokeUhEzdcti5RZJwdz\\\\\\\/IBZg3y0Ur684GgOkIu0dOyL\\\\\\\/TlGd8UPkFCsfY6MfabY+FtTqG4imZRBeWF\\\\\\\/Ey7aUAYLSqz30zdkAtRr3rb+4Crxmdve+yay1XbaxjFiRSInmocvhMDRxz0ntV8fqKrqQHOo5EZGZnVxthBD0feEpF8C\\\\\\\/TeeZt0JIMaLEgtGvDChbbmwvoASaeAvrysVL1RJVqsJXQJIrHmVho5isRWU8liNeCqiDZp7D+eT3zfpFA8g0eLLQtlRhlfYXfX+0BoqCi5f8LxP5CrAciWK8Xkg9LPr0qYkq+vYe1JX5vs4bIrr9a238bNNY2KXRsDHXaPhEQKzbZzoMV7kNKkDdqpLRp+nwMfTptWhsaPv3zbbfQE9rvMUF4P5NuUjF3\\\\\\\/HDkFBf59ot\\\\\\\/SwyeV+raUSNioWtFRCgHpTBoCneTgJbHRn1AihWDk2boeP+JHEpnszCszo9qecSKvNQjk6lLYW1yW5zLBFK2RXDoHp055DeKsV4TlbK7tvlrRtIpsThVJBlhI+QjgwV5p1XT0EXSXgsLy5NEV7evAl2lIOkhLSKA2BJzebZFVUjznAH5P1Eh4GyDPbdOlx4RJ7y+iKbD6SyPTV3zjvGaiCZjI6sKArLIMDxELLtNzARatfBIlHv4ihHdd1eeXvTpuAAHJotUDfptmjO9yX1Ad3j7qBy6VUeTBzrec\\\\\\\/n\\\\\\\/IkVxsSpMbNbkgeIpiSuShMCawI2FqhhGMMUx1SRXQK784wBM8YZBA0FJxO6V+zWeQdc568XYoxhuCk3msReXNQuZSXZBnadxxVpj49bjdGZnO6OOwWyyHSEwJ41d4xk06Z1i7\\\\\\\/myLlVQ0Cxw1x44ZfbFWrgyjKJd3hCvfEz0TI2rWD7bSfHUpo6QsxekLhAJmxrbxLJzCyaSbxu5bl6yTjxOvAB6Y4CaMBbm4Ln6wJtE3Cwel0Rs9uV7PLY\\\\\\\/aiOLZIGLuOtd0MRJUZyfB4dGyGQrjnFtQ5grm+5oyOflzkYAQSlSVtrwhKu6nIi25hMtcOtdc2DV1XlBAyzGmD0fs7L4yxT0g9Qi8BVShkmYihhDyjYa2i5AiQMjRJ+PxNUn0S15i0YK3NIofUuCELe79ijWbKxZ3dhlJXFaUg\\\\\\\/62tlUZaSSzXmqAb\\\\\\\/04ToTkgi\\\\\\\/\\\\\\\/+JuSQo9xVbIMBkRelkDEdrWUnxdY7mx0gI+\\\\\\\/0xUUc9dIfvKqvrMxUiAUlp\\\\\\\/gnZZ25NxI+IEdtTqihQoEZr35SODGKFy\\\\\\\/l80lC1gSHM1GKKEDcuC\\\\\\\/Brzcce33e4CIYzMQ5EpKc2a7MR0SrBJ41z\\\\\\\/UdsACI+XQCfcFAKvU8HeVg1xNDcfSA6YZemWaJuKBj+00SvHrJM2Z6GB9KML1KDgm7sZEQOQsWcEbRgV1qmseNyv692JFa2G6TkwrSDXlBBC0k6\\\\\\\/\\\\\\\/T3HMg06wY\\\\\\\/NsW5l9vVChZU2sR+dlKGloy\\\\\\\/t3lS0Ja+PVYf5QQv7cKiloUKtWpdtGeLbLlD8Vo1akGWc9lsUQz4L9WmlsPIalLQ5Bo0OKMBjM1UfFUBWurmC4gZUO6DUvHSUB1E\\\\\\\/t+Fds51KQ5tO5qhIHnBx+nCPC6sgzKcaRbRrSgO+g4z2Zi9Gjdmo9UUiF+i0mRTknOyWwYXQGdfn7Ah8hk5XWAkPadaeXep95\\\\\\\/JptqE609Ziis2b+qkZgHPbV4BFOZRbXdwwWoQ7fc69a2eh8+hwfpaE3ePFWbd+Xbyr\\\\\\\/HvX1Ho4XwciZVWA\\\\\\\/8\\\\\\\/fkzJx04H0NfU4PFbx6WSoIRkXm3CylUoPDtFKnN8njn67by3tv+pZor5FptqZ\\\\\\\/7zEbK\\\\\\\/GHmeBakYqlVj\\\\\\\/izbY5TBeAQqirs07QBHC5fuVp6HPXhjngCHEw0NGVwSWaIeNUVcyfjEeyfmr9kkfxoYVnLjG9PSg1ytG5rJLEWDY0\\\\\\\/3s4em\\\\\\\/oIOAByjZ50+99h4yIa\\\\\\\/wultK6JDlGfq+HPNik\\\\\\\/JZA7BUhqrpzSA1vcHDzxQ\\\\\\\/30mg+k7GxXi0xKw73PPxrtFgtfdDkgmCZNJbj33YaG3l0fyYCQYWXMJKKF+f3jfqvZBjsfIfVV6glN4ajuXUTRFTHWgGDqPH8o4hDegTYMicDVLjp2q5xlVJxcbRfdvPO\\\\\\\/VopeUU9tVnMl3dutm7hx+n+4luSrjioFzndJYcYJJyN1HMbUzV92VK9IAPkgT3VU7oGVXZWe15Y4WHY75vO6kWA6dsOaooiK47I65+f4ORUyMY13WR8He9VJPq49R+kcJvZy1jr3D021gFGa3inH9b2a+TdefapKPYtRsKiEDs0JE0rK5MVqHygCma+HhfZEA5KquT5Uagd0p1NS3PQszyUcJhLsK08+hSx0bByM2pGDYpgZZtcMEl91997Zt86qtMZ0hiNm7yEcXEyk5jD5bXOvc7wfgFBT7xq1hWKos0rsAs1vQ5oVLgfeAJKc5CdqDapv69hk8zz85tNRfV2Bf39Fc6BYA8rYYZrP9LCpdUQ364EXmMH9C2GnY4UYLNh3Bn6jjMWVeTdMrNxedjF1k+B7Su6BbFK4VKRER0sS\\\\\\\/qFW\\\\\\\/Q8gDmd87UUFwmOeu4NzORXwGaJ2DrA5QASssp0Zs7y5dVNA28nOqY8CMhUqaPq2qnfv6SjP7f8d6QxHQIsKIdUYoEbGAXrl5IJqIEC3LweL9c0aKZhgno0PKVI5klFXhC9wYomiXhcIE6QY7\\\\\\\/Jf98VZuzA7XNiSf2qaaZ88itduA9IVNScd3ytHUYc0ZIWP0HGIHg7HGG19EEJRRcBMkapH5QoDdyZeJLA\\\\\\\/hPxVBp0OGesvkbiGBGidHCToKtPYnetG3cJbeLMG\\\\\\\/gBGraLRTRva5UWF6SSeD8ue8Gsg1QSZH7ZiITPSaHnC84fzVDjQCIRdLYvLgbG\\\\\\\/ccQ9hls2tVTeZE+NIfFAlbxJCJCCazAyNSjoBsU3EvwYEyx6ayXDpqccnKHAcUrXEpYpasT9f+Mrk2tAVMcwsQsgfcN+Z0fnxal43TOxOwYoL2cI3Bd9jfnuqMKxAsBzgxHDA796O2ZPPy22EWZhEkGrG0h8MX96NlDHe1QtudK4MwzCjRZc++X3aYi3l15RPmY6rJsAq3ocNMM62snBZNtD1gFf7I31qCtNrtpNUZ+vKBLxNtGLosifTqcPmlJU6NloZISlf5i0jEeDj1bvJol+kF31J0d9Dg1HW3pzXaop1vxIOqKTnXpxgY6Aq51YTqT87J+FirbjMyTAmcHVPvjHAq2rxaHfvujKdyO9Jz8zCasuoBKn7bfuWpll+quQc7GvRne06yKSXk6ksMwqk6IIt7Q40mGJnb\\\\\\\/9vmYPEMxipYVrIqr3EbO1Hn94k1RD1R098HriRdgF4lKnL2qIwvkAWcQdTovf6rZoNtz9kCDxn2OCojWqnzn7DPwZrWlrsfZkSkdQLZ0qD1yAyy3Iif4fbf5U7oCuYjq7Sai\\\\\\\/p8zP19KEp4kZxg2NnrwAojB9JDcmhvTIhwOdvwHPClwjIJgId1UcGQba26p4UscY1jqtnFlxUX3XYtLwuv0I7TgNqA4O64NsET+7DxAd8GCg+y9uLdpTfCY650edTvxJ9u8Dzty0OjIL3XfThRc3R1uf3XcC97Xb95iCmERXPFiEW0IdZLMJ6BpvabTwt+C9Jo8ievpVwORcbnVwbcw0KQuew\\\\\\\/edR8Yh4vQX7pJ9zljhJTvmRXPUW0wqWCgUVVo7FWyitD2XI\\\\\\\/wNYObBPGtncjE87tjyqFCtoj2ypAnaEuMgGV4kK1zc6biXdL1Rpon7vb5fkXXULme2lwS41d\\\\\\\/TVJZ16772GCuAGCdx2l3lp0ChzD75Fd0g14mll4++TAygX1Xs0UyxSqLrJ\\\\\\\/szjsEGz9TZyLy9GC77i29oQuLDepojfRPdHZhELX\\\\\\\/DobH313eKE1cm6f6yhQm6wQaF2MelZlIqBDAzujItlGMN9RsMNVc\\\\\\\/zpQmSBOm1s2WN5987VMfats8mdQz8PB6dbxvv8D1X3+jClphymv0NE4WSporuXjRQzWrMuXCjis688W5SQ8hb\\\\\\\/9fmxZz1mSpL3YKbr+UEsoMR47Jd1D93fn\\\\\\\/atrtaaGhom85gbi\\\\\\\/\\\\\\\/8qm9T4HUzML3PTz8fZ3JZpAtyCbIXg7OcXrHIRpDdsjriHBtw8kZJo3JjUmx+LN6Zri3VZJaYj5wJ50cevATjEsjlWhnTkfkqNjx2mbVsOJBFW\\\\\\\/V0IyhV+vxZcYkmWNVMXFpIbXFI6f7R3KoNP4K9iIT4lflIiuOqRYJVKRvLeIzOU3cfomP1TpToNKEs\\\\\\\/TQSY5+Shc6ABcm2qA82Uz98Llxyu6m2ljc1ivcytPf3kiKgA8rPIGxEr6zvQgzJAZdMXZpGX1GPb6wN1dNwOuvU9\\\\\\\/TEfMnw6NsF2SoQE50JBlok2FoJ3OLgQjpSRFoDiJzt4QTAS\\\\\\\/K0FnX53+qPdV6WyfY5ffa1g94HaUljIDoqdXdOd6X72ptLklzYBmtzUTT6MPH7b0axnhjeXnh7ZcbpBDnc7Xjo8NWnBxnT5uwiYfMlDwm77GwEVpp+1vE5uDBRlPFxEOu8afzQDVZ2n1KsyUBjevfoemfyhKdAN0Wme\\\\\\\/HFYTDgOfmFBKaEkTol5pgbsAxjnPR69lJelFahI+z3CH8ilnHuTNelvu7GdbEKeBMclEteZpg\\\\\\\/wagv1REhhhqJzQ2IQfUV2xuC3ohMIojbDn3Q2sVtghG\\\\\\\/E9BPZwhPCzqGLQe5cg+tE595COyO7ec+WYKIpXNbKoqICQJu2CmmBhAX+hQC2mroVsxrGzWIyhZpTjoevwy06tJuBq25PHOttqGaV9kJj3Rsf+VToLG7y0Q4rgZhDySB+m6Z+79aCWLAl4bIz1TAqR0U2aaF91pbLJANJ3d4poXwUge1B842mZ+fftNPm0KPFUcBjegaSID7YfkwP+R0S7r1bqAh3ltmKh9cEjo0cJnnUUK\\\\\\\/XKQO6fxjdQxBBeTj1r\\\\\\\/KHmwS\\\\\\\/uFTwzJ85EgfCF32eufMlTRAspR1De5DAefxPg4O\\\\\\\/iQnfZ8pr0YzmvcBzz\\\\\\\/jphd7uPw5tY\\\\\\\/ZbQLidSvLlUHefStRanKIzIPEB+XgutloMs67SjtOikRWnDOBRTDRIv3vE4Q0LvxFZz30AJ89b3rE5lxNt915Ft75rc5i9botdx3DeEiydhFtG7jHZ7B7SZqsESW0W2KuvoCyCMNX6rT2uNFMQF9gTj6izsRN0LVvyO9iZ+XnQvK3O0pivucrN1etp7wbjHqPxn7KraAZptsrP8XRomcavoRTXDlgHeWQyv+fYqUx97aWXp8lcvbVn5MUZGCcRyUuCRXmXx2XCxclH9oRe6\\\\\\\/D9knjYfntUZzWahEAXsBYyHPHLIgvYsVdgC3j5z7wIIgbVZYJLMyYtdF9\\\\\\\/PKZ05cHlKkWOX4gMoYZ2o8snsS7ElMnE1QqhgUoIFkO4T7ZuZE4x9bTqf4RdzIZVkuLs6IiE45rhSCncmwUqdgljC7PMaPthvN3yOh5FeulHyFrglTyS8jGdibRddmdiSeqQinZV8C0Kw1xkjM3UR4cNDPhFrkC1XXs2HnIjOLU28Xrz0MqUmYORCaLqwMCVVVczga1tq76YwAGel+PRUQdXMw6b0K61cUAy1XeAcVIZsVwNfyTCeDgwbuWNqjWxETUNB5K6a\\\\\\\/xyO7Y+XdXfDamOR1\\\\\\\/9gfjVQfDmZFmz3zJftIqHKQtj\\\\\\\/rd9t\\\\\\\/0YmCn+nkxO9GySGhfVAzc58t13lZY+dXrP9pr8u9Krms3OJ9FvtJz1qTsEd1qHrN0AdPqIdzu4Ol6vdtqVSH8igQFAf4MQqgV89x8fkSyrlOIKFflLcwu3UH71W6D\\\\\\\/mbfwET9PW01oApigPP0wLuWcv1tVZ5BT9PmsPZB4Ja7YaBBdTiHlgLC9F9u3ddnsnnboXBC47\\\\\\\/hrBrsrvG3\\\\\\\/E9mSqNIoR3IxyCbmmZX+128UXnrxCFifc60cfyNrEHuj3eXQJT3tXQFfns5PCgvF0if9AqlQrLfmmqUT4CZSRpRxdH02UZ786nyPUaQPU1e1mL1AacuxY07gvkOHU9JsyRIsYmSb1rUCWIMGvehB3lgqRHcbLb7tz6n3B+TZbkwRtIujxyx5Ncotama5lVOkc3kdcJzAjhC2Z0o53\\\\\\\/c5zJpukbHl7pMpCSPGsvp+cYdOTHLx0OiUb5G0+EMit4f5cSo1VwHXXJIvy4SVAT7AIr0dyBfLxjFZdCVN7Y4EZdgsFQmY2hePE9uX4yVqHnbS86nFbRghAR2gObhOd19j7\\\\\\\/KDmh2q11PlwgDHZP76IQDIkomINIA8Ns8GOyNPSnVOFwLs3242XlPjbYadqKbCcWxf33F9pJHxs+E6fS1PJ8a\\\\\\\/aSGC3UV\\\\\\\/MgUoPsa3nf2Sbg2i+ZYEl+vnNH0GG0qpwtCBq2TIFi6awWHutCQ8qe2PShDjI1gF3WbPmYLXN0rrOcZwQ7m9HAI7TfSGTSIDyPtuub0JpDLueptRIvvGrFIPnXmJNpDqoPwc0RlZdUMXaUHp54rKG0qiS3L91rU\\\\\\\/f7tWTiZjBv9CI2dvUVTwDIjBi8hJ1yPjs3gs\\\\\\\/QaHZUpKfc\\\\\\\/Ht4SciG6zDXazdcEYbUFvu9ZqgywzfatWe+sBMSqbwy6Flr58mjCvsDmdfTnAJZj0WfpfYNDalL05gBJSFcchPSzI\\\\\\\/p94G6We12iI5aojqfvWxPElg+4O76T9CkqE4UAJxA\\\\\\\/zkUil8adhMgJNn9YdyRZPeCt6DLgLT5MWyT9wslRPaoj0ULuA6LkWjrescB84GANdmPuyWJhiF7JI69qU++hZ\\\\\\\/0bwn7SM5yMGIWkY\\\\\\\/krk0e1VsILLol1wharQzVZMaROlBtYFFZ1xgECyrYjCVg+LlIVB30BrYDiAZkmW0YNtH524Nv\\\\\\\/B3HkOYl\\\\\\\/08iQcJslZKyLtcfRSAprdLVvzgQBwcZbJAeNNeS3dEm+vNxTZBoJpSYo1mD42RUhw6h1cowCEP7OJICyClFI5i1qRMG8dSQihKFkZunFxx1tAzUcf98i3\\\\\\\/QiOvwd5nrSekWzaZstqnkPp916cnNfS2bq\\\\\\\/PWBzD\\\\\\\/vjWcvo6dYAV4xzT30Dn+1D+a9pIMcIZ2f0ScDyf431CvcGHVuhlFsozepxSKZLaDlTjCNhDfInd5YZE7c60lynKFUajwh53xOIjblPAl7WY4i0+kcgaqYQ9WqVXtENuDRSvwZ1fCte7P5iqXli5WpnUzbGyF0ZHz+pxXui8VF9F53dJ4QyWQ1nvziB2L7nsU9FpJ6f\\\\\\\/ea8mzV5Zld5g9Xx3+1uZQOJMFTqhjDLUC+lDoQKQlbuvc24\\\\\\\/qRQYgdp6FvVaN0IuhsRbo0W227pjlQONo1f9Xyn69I\\\\\\\/mLuW\\\\\\\/xmPxElN9T1KGUtlSlSkfaseKWDiQOuxI05p2mPpdzdfgViCKzY2wOi5+27HgnUNuL4xjFlCwFc5HzaAKK8+P34\\\\\\\/5PJOPXJBAy0Kv17KD831dWuD7yD1iR\\\\\\\/Rf\\\\\\\/3qhHGN+GNhVQy7qMKLaz+p8YFpVZE1t4PRC+\\\\\\\/DYGUQZ31XM0S8+mzYPhoFn6xGwUm9B4XjNPlVuDzLSXK+uboBa1\\\\\\\/wFtJuDpaFKp9zNmeyukT3Rc223UNX+eMnAm8nBqvRubApmO9rpTcnO79MBacfRHzdvGmOKXev+TKF1GtMBY4SBHD7Qqaa5PIvgcvtKFViatEwQ4XEoc+Qi59s2rdu6IFW5zqvqnkeOt5gXYte6ayDMSryQOuwBTCGGZHPmZJZBtKPebEtWJRzuEcS0yMqCMtczBHAV1sQ\\\\\\\/zJo+6ANVtHOXXFK1JoCnn4Mc7Ah3OyRsoPLIAynacMInfbro+omVC28tS4\\\\\\\/F\\\\\\\/aFs0r9ZN\\\\\\\/SHzm0it9VcnfbVQ15PFVIcsSddm70nCRlE+qklbXblDmBUv3GBNu6foIBZ+DAlk+LEF1hznSudOe2jYKSfaoQ\\\\\\\/wMl3VcSbJD2o6aopoWyaX1ki0+ZPsR6rrZZhgiekJ3sg6zNsnZtM\\\\\\\/uLjvdMB+vsSo8S+EpMJeQJ9U3f3UlUY+PZj+Uigk6x3sDsCptWgA07wBszPCz9nENJH7FiplPkgXKppRwYYoMnlAYkr5+GH3TXaGx8zH08QFhuQUTCFUJfmFAN1N74VGuguKwa67VwasGUpRa7gvUu3dymXkoW9tW1jDTmH41BpEafKnTHKXlKHaRzeIKp0HAD4z5pPQ7hrLiF5UzJxosjA43kN81hY+89vbvEEVDAP6fT8YV9890ZbB+NHTHka5sHs4vknaGfTvOW+6PqeuFXX7jDgRa4F702kN+VEJXKZHsW2Ig\\\\\\\/o+fY0UksTASKDse+f10L6dV1k1lpApSFJdOGrSomcG\\\\\\\/QpobnkSW8+6E\\\\\\\/No8atvFPcHdmHuhObGSGQm2\\\\\\\/YbYkQVWTbA5ryEsK\\\\\\\/LS0ezwg+hmzoymr52H+9ONHNIFXI1EAzXNc\\\\\\\/zVR2mudG5gxX4wA+tNZOKn9pYzbG0pqAfWIyNtPZ3eSbOsssQKWcjNOdwNOQAyuVgGGr34m0QaKQuKRXpELURvorFHzl4MdVrKfYp8EATyrecLJHxLQ32Vg9nNKwliWyhPGmt1UWseRfD9rwDEEMfmLsdI1kDb0o4H0aVO3ZrrXhAhHrnXkkvtGuzpPlqs74uRHT83kiBV8wMWZ1b2G7UwFXs3l5REZxUPeXd72\\\\\\\/HUvBsIWEb5DC2\\\\\\\/97oQMyeqzHwUVsOtyFi3e8Bl46BQ6xNbyZMIVrhugCwgRo5atIHpNEIewHllT+4H8eRZQKOR9PVSVEnukj+hYzP4uta2yqxK+2TxIVXiCRVEzOG44j\\\\\\\/LwZjIGg8W+MTeGsPPENcjplu1MKpMXTMRdHhPTKaapE3r\\\\\\\/a5CzdaRtkv3zfNhQ6zgDQpR+bdw0wLOjVnp+0bmjsgL7MaOkL4lyPX6LeqZ4qpCaV0sm7ycZC34f6gtnNF4TfOVlwdXHjYNdm\\\\\\\/UJyI0DLLG6uyE+RFnZKgGnxXwgXCnLCl27gQT1\\\\\\\/zsLGxpDtN0WlfBymCc3M60ZAZ+mOMmvQcZMDmRWdNc8MSbjYHegD8L2mCKleevTQAIr0cIc+64\\\\\\\/qnHi1XErviKi8tIKxLQ4nPD1tErA\\\\\\\/2ZQ+aQKV\\\\\\\/FH83kPh1\\\\\\\/Kb0vya6UtL560KMCNfP3YR3J4CtSCBzXB4OtTDSWLm9WBmV49YyqaMWKO+BthbjK25JyeVC4U\\\\\\\/LqYgbfUj\\\\\\\/vAExB3ES4bsTnLCGMnN\\\\\\\/9il3Bd7QCMOZfENAp9raXTVdYLBsJ9+nJ3WRnRLxhu3kQpTp7RnDnaZX1aP11nccm+Rz1ddrjTJQJWEsg5eCjsKPIxQoMJfEk2Ww8WzU+vUfTT41qzhNuAh\\\\\\\/dU\\\\\\\/mKVn3HWxVVIGkHadA5NKm5V0I3UddOtda5FaqFIC2lePRbiimj9M22h2aIjjxSzXP1XuFpZNzNCZk8tyfN0smN8c9ADIgL1gLtADNlAaoznIKQ\\\\\\\/C3LpPc8LFyfJU6ha6shQ9QzKKHmfgYMVatKHCApoalS9mqeGVdFpYPLdUZvWGIpWDwLc65cT21N86j9VHaYbCSMeSJs0ZdtQWWbOPu1rAwaxoN0bVnL55GtneDTiJLY59mH5mLcSWh\\\\\\\/zb\\\\\\\/PNZPsoBxZC5jP1uyzIbK3dwl1W8M0DVkjnIzt7ilwAbMn9XCEOjWQ0paRbr\\\\\\\/mFd2J+EMuWWOWdBHmHZlyLHu2tUpQZhpO1j+oEs5VoWB6NnlmDWLQgtkIAzhMtHlyUTT+45LVf7Vkcym7wHQW6NxxOjkgLrpWfsSTmgV1t\\\\\\\/HZPCzwKLHhEohJF53npf91jYgsNfJVf9\\\\\\\/hlQipjPS7dAqcHxUGi7f8pduvcYIxVSoprpG9dRvIHJv+zB0FzoZAg1+WKYL5GOWYLkPlYG1NoCcHUOc5L7EXZNou55+BZv5HjzKwgaibKIVFENv3CAvA596Lg8Jnc3So3aHzSHoE2nCH3CZL6C\\\\\\\/z8THBHxk18mC1arJtFEEpDI2L9KIJov1NSJyOMs9qJVQLtJ4ZjRjPf63YpWLpaBwHXKGd1M8CpylhnLAAWpR7PmGaRmxrRGlZr5dVdP1jtOL+ZpRF9\\\\\\\/vbpU77M5Z6GNII6zSZ+etfXgJ5toymNNAzQzue+OjkP7cLTHYZjnDmoBJd\\\\\\\/4r4xQadfKmMawS4X3DhxDR5RgiCptvlabVKJjyEpGQEJKd25E331SnmR3CId8SaZQP+r5yOTffQEzxlLfIdN7yDMKIlszcuyreGq2CqSoyEpDgcmsdT7hAC6HmM0diEg\\\\\\\/1YVqd1U3I27tvodW5AE6EpLshWeM96dhlRk4ZxgEYZFjNvGQSbS+jbW1rDlrSSJ97k4m844Io6\\\\\\\/E+yT5MedQGTXavlyEkxoegt2hoY2kTnVj+02\\\\\\\/pdThXByDaHUWRsKeeG1RwNoxuUVT2OUqYUaTPSTvomqHAoYVKeljudrZ5SyQJPWLVjLtS\\\\\\\/mLcdabeVuI7VWXrewsYthDLyidiVlavfTRM\\\\\\\/NpHqZKs0Ax4+4JF10pLJSw2JgggaL296A\\\\\\\/8xjJU9T\\\\\\\/8FbfCGp6WwVyqnLDVjbVJQ+BOYbsiYlvMZCaIs2OepxT4XIdeJcOvH9nvA8FWnA+unUc+onGJRsCEOY5xHuu3K5skCBcfJ78WOPmqPAs\\\\\\\/K3Eq0V6\\\\\\\/RKya4CuCp+L6KOEO2TBnp7OdETzqxI7trSIsslZ4\\\\\\\/DDlBSG6lV6DCYDYBVHnxOTEBCH\\\\\\\/gXMYC17O4s2Qg+\\\\\\\/IrtP0FSE1rd5rklPuBl9+tD9S6GbkNQQTIQUcx9xHVUh7qKuzldtQ6o9\\\\\\\/qXxQEYYVsRh7wtHydpuMGuDQSCms3wxI4EhQHzGtenzoUsMy9xWGoJoeolQx7FWV5Yuq06C\\\\\\\/kMe+mOaO38O5H\\\\\\\/GFTUpy0XeKp+wKR2zrdEfYVmy6DtKRTMP0UkGsJYzvAFQrQVjNdxxbqjzo1sjlyAi7vf428iwUMWgx0Qn\\\\\\\/PfsdDyo\\\\\\\/s2ivpZqu\\\\\\\/I+jDE1CIwsmaVCRJ7nIPfn+W2D\\\\\\\/dEd1xVAWbFCrCbp45AWjgW4JR9iVaswu2ivLfq6RzryTr\\\\\\\/VH5+5Dy6L3qvV4kZJQDN3XuLPM4e2lcsX05FhdRBJ4I0BI03X4G+o9dGJvwnh8y2pagy2onDtLq4wg+uRQ6jN0+yagxonzlFbfhYsT8mqvuxNvremo6Yc3J4\\\\\\\/aNnBqhRGM8QeOfSDv87tz1fhwIlzzI5ai71iCzz1JUIQj88\\\\\\\/VfuoomU2sRIdfRrDSAAxbDyNBtrw6Kq12NkO7dtBnkQpe0TpOn9lthAHTezSgKQgIHBPf+IrNvQgQGy94MIGkYtNih0h46nRFatWvVU0eWR6s3TCN5rFsmN6\\\\\\\/PnYW8RijyzPMVKQDDyNVnmZ2awBr0CXHjfmgZo1WMwoWvq+dYTgZKwAxn3PvGfNj8MUnAFjkFbYAT1QvKhCr4HOz01BN8txl\\\\\\\/rdq0Cgpp\\\\\\\/uHV3EiO33aSXwQLeSPfdsupHA+rZwXwSfT73vMuikVhLOBBbUa2e8CFrafhwfN0gkUzVw8hKZHntUC\\\\\\\/wVmdjYquITyNmIzEPkaJoO2QM\\\\\\\/bw6aUfH9dKcYjfrqTqXprftoXG4VLzJ\\\\\\\/883k56wysKO4CyoAnKR5U7zkLDzGj\\\\\\\/gqRmwAT0yg1NprkKogK+v6ovC16VwwLOgz9xauUtdGBZY5JxnAKmpYpo5cW2ZTTpc5o9krZVrHb5osLn1eWC\\\\\\\/Ol5zn3fhcuVaPHNltIk1uyI2TcCguVasTf6h9\\\\\\\/fEfI4P1FnF4onll05lu2d6ZTJQi15Skf1\\\\\\\/kmroP1rc3Bf5FTLiYiJ3GznnmdIrWqv6IvFRPNm8g3eW3AcXEqphYobBQNH9aq5qWur+gykcaPQM\\\\\\\/LQ7rNYv1iK3oE+O3PyZLgMAZRNQ7Z1XPLwLmPq5Z6TTubR\\\\\\\/FJscJN\\\\\\\/8mBV+qwqOIgB9454FxS0kRR5nJ1nrVCv0r2MdInCHOr5NschJ6Z7qJStbPLOuEk5oeBx5\\\\\\\/M2Nm44kBmxd+HPdiWKCp+uEgV38SIyVq7P9gFjKXtJ0hl7UOuyODy5HIKkrzFXhIw+G9fET7vDf2iKKVp4Ct3povblTMQkpfXMC+hNXrjqZiuDdsCXuIYYBj8GnkODhyrnccBAamzOCRQHN3Er12rZ0pP\\\\\\\/bCf07ZT1eOgOYwG\\\\\\\/4Is4Fzux4bjb84pie0A7xOIkF5qE\\\\\\\/PPzCgF7uyA7o7QVpJAyIwomftR9gbGOKnqfaNzX29bjIU\\\\\\\/zowP0nBdOPQ87rBmIs9fGYF30+Q2xl+QGSWnlrtn6XyknQdzc+nV8KB3pPBl+OR\\\\\\\/R949JKEX0pYfRUdk1k2SoSxBdlKke8AimcoXY6iffWTx2aGRrVJXm897XD6PB7raC\\\\\\\/ujHmP6QU99lhaFXj1W87COP5zsj7N8Jpux8iZw0NMruYdrw6db0911+\\\\\\\/7Nd7b1VWo+Hd04VW6gHV1wIRZkMKgmEETKBpqKRfabt7KOzCZgvXCAfjbFeJxAC2GuS2NtfVg4AIJAoRVFYkhpat0YOD7478jQyZ6Z3YzrNcpKPTsy9vfWJXzOpFvcSuS\\\\\\\/aozwhOiOlBPfW2GgHYhXti5TrtTER5zwvZCaQaswtxPRhmFWbtWTeXoacBr9cXbnlTJpWZjsiPT+K1HiejvAmW35FqA5D3fIH\\\\\\\/UBQhxNbeX8mgLW1PA0ywl4Tyz5sm7uphGl8uaiEEDgnGtVqjaJ3pOd0AoGHIWyA7j6IFRU2j9Y+j5c5OxgzZk\\\\\\\/T3LHycZ3iXZ7xDRI\\\\\\\/iQxvQ0fy8MXSQAHmDs+npIwbxqzK6016A7Je8pYU4tmK1b2UNR3EcB48zVJdQuw3WZjcE9zIFfqaH+ZMLB4nRNg0rSoDQyTmVoTwars1LqUriEAQjUHWnPK1xxXjXpkcK4iCytsA7iLZwkjegzG8X1uAWTtuHGobbjCUrUZwjljmB7SCg2hPXGDZBZtKiVS3dwny6pmAX7c1waikqhY5B1eXs61vSkXqAlvXsLjdpO5aFFnFBd7qeYm\\\\\\\/vRMb\\\\\\\/B\\\\\\\/ERo41\\\\\\\/Wm4kgXX1uFJuv8cp1GVpZpdx1hJrwLeLLlEfBPIy8r9DjKYkJvkZeKsFJAcaAAW+oafdYrzFpD1blYJdQeswWMt1KnMWFdPkbSwf2KcxjeUbZlMOL+4Hjf7eXf+8EM5OSimzE8I9v1Y0O1A2qCrbUZfcaS3On1zxMFK3qg\\\\\\\/hTUzPULqo\\\\\\\/7iYHpYGbe6YuTsgdTsNpMkIqnn6+5J1oYbfqAmWzE05zUTQaMPz5utQcMNcbkacGHSYFOGSYLuuZIfcOX3mdOBgHpwGh6dYY7tRxD61vGrsALCgq5fWXmX\\\\\\\/FRXBY6wzXZnazNotdbbd8PkF8KzakE2vLuY+4S78jSyOk\\\\\\\/+IcblbSy1Hhw+6eTR+n5IBYwikAep429ODGRYD93dG1o8ZapJIPQU9EIEjyoNF+4WF\\\\\\\/vR\\\\\\\/7p5zTEYjIkGrIoR1adTJSf3E28cDIEVy+PBg850CRQcVzJDH+ZLbsyfz7kRG2P2sWs7FGhwXNdK7P8WIpduS2x2v1AOOGOlRZ+Q7Yx5AvaB729zt7XISDzwIhUDZEuLgqLku14dhAINS6AR7ucobWnGnFu0gPx\\\\\\\/syu4Se02Dhehp4IaPbKMKfnGjyjieciQCYuwF5pklXNobYGG60+YeNDoUWN462s8mp7vboQ\\\\\\\/NVBUc0tKMabRg0UjMYMvHQAIFGsXDNoocrb\\\\\\\/fKflrm+I+Tg7jO4MiBxsHCMPyCVvNam\\\\\\\/Omz6D\\\\\\\/Up72rOVULW5bzSou9x9agbhVzWdlc6zsL5virihO\\\\\\\/lkV4moViEJrcbe2Te6LwNz3Wea+\\\\\\\/s39pDktZAZ173LwbWOaCEb68Mlop4YyyOHlvU7acJqQMYFC3HnNQJGnRqcLy2DAREGL3gvw8Kujl1P0HTF3hK3QQYr1+ADZsCAyBUavIBXNxlwqROv\\\\\\\/CLxZ1W\\\\\\\/t9oaUHDTslXaqmtWc4WM7MGnJFkTCH5x+h9DMYyM8Zsurf5lnjE74bOJghsboqjQcvMjH7BVk7Je5AftCfQvzsbzkhGZKapDO9Z9eSxXC45FmI5ydQeMgs+hST4WfLDz+8XiDzGP8VQ8MQHgURVMMZRYzDSuMnRGA0xjNsLBcn91hVmTBSf5BNEk3Iy09ius0ZLSNV2o3EN72NXD9p\\\\\\\/tm0JNe35MmcTPN1QOkISuQX2eMr2CbKr\\\\\\\/eBhTq5bOInNtlXWHpw78KPgTsMUGzGFO86OxpjXhhbrMTdpSZtQ683xkQpTzYwqsa3fcstWzojSpW3I1VZmjzUyXquIyZRmLmZJkDdyESfqvrtamsR1\\\\\\\/YbnKg+v2FLcLLRv+mUVRxgB1Iz0HlSfldLzUf2vKgGUFWR6E9f0kJzPfwOo6EpUinlvpfXqzWlg7s826TEd9CKsmP+Lp2QPmfQYlP2+wVpmEA1DxcWtwwolRsk+BUwLu\\\\\\\/k0T0fnIXDmrk4ijIDbX+b+4niF8CGA32PC\\\\\\\/t1owe\\\\\\\/Il4r4LVfvLqcC\\\\\\\/+aSclom\\\\\\\/Zf4+DyHlvnpuC+MNz68oeObohnUaNry7YgVowSrXBo9a2onp9pCr+8wUe7tcnWa9AR9kyAD3PtULNEIozMpAOyzzZOSDNV00rZz8xQ2Lg+QJoB65\\\\\\\/NSHqdy7F8PkYKoNXk4J1uOvcxgho037hFrTdEevK\\\\\\\/Uv\\\\\\\/cvqpXcedzjH1nhDRHYrX3E\\\\\\\/NAOOSivxmxtEBxos5pAEHciPqkQvLDgzBQu9wZincED3Y+by+KtQsfKJqQXHaWN94Zm65qEe3jDu3ObGsnux2Xu2DRtKWAtgUmsYP170YrxnQXGAm0kuMjVzwEI03n7knhwOJ6VbdkbRWWaVU5dQzwY1wPj+eSX\\\\\\\/IIybKKeU93V5THxZy\\\\\\\/HaZtncfTCJPQcJYMaGUDwpuMoDDUNFFfP\\\\\\\/45noTI1EERL9\\\\\\\/LIjZe1eESO6Kf+QJYFm1i4kipK1uhKe4Ltjq8ilPKkvxf3Mvw0hCITKagLaXtWjXanfilU9N8vq3UrMoUIGFCXjqixWvx6Ck1\\\\\\\/+ZrxHfm2wCDZ\\\\\\\/KgdqvjeUpcfvr7qrmzemWxZj3xoGTljX9NXDwyPuPDgKV7KeEEcIlwb\\\\\\\/K3U0YsmCUlKtqzJSoerHw4W+dywhvz2+\\\\\\\/+0uprl46tkrGm\\\\\\\/CdevOWoVg1VNtnP\\\\\\\/KJX3OT13m10sQAor7gRMG+i7uLKY7jujhMqwh97pFHJn9M4iYWh+cphCqwCNQJHPsDPIgAlqOEZ8EfXqg941LAQaPoZk3rZka44gXWqw8LjcYGY+Ye9RB5TjBNbj6gf2obTaKRClk3UFoBlV1o6QtszeSgYQaUqoOvHP7lZE0fOQhD+Q2C3FWF8IYsScEUU9d3RCbvObGPjB2sV9wRT3kd8rA5D\\\\\\\/RQnqirbiJOukyikaHX7P97JxdMhITDYpyl1kAmjAgB+QbSp\\\\\\\/D9XtpddW8NnRnWNKi2ApTh0IVBwBF\\\\\\\/6QAnHwjaFXbmD8MV2vcIgUKUP2CC8FC2XbGpMcQmXvbsA6sgXIqSkcQRHQBer4EPqqHsD\\\\\\\/0pEAGQRWJs9\\\\\\\/xEgFHn5gkaeX9\\\\\\\/o057j048vIweI8lo5u8RofD7L4lhmURmKcUnDSSUwgDS3CXtYndmvGgsbX6isDJGETXcsKCoFMspbY36OjUBSu11RyDFiUrzDJ28CJGE+abIQnGDOPoVKaQ5gXvvfaUC0oPVTtaI2xFWK++BdZvZGMcOmf\\\\\\\/BnHkkJ2omE7ElDlPEmXjGQOAwrnWqp5ReCO9igVkTrd4MKyVVZNxSv1zOD8MkrXYaR9E+zEjVDeyic1cG9C5Nmp0PdkSVP5rlHFylKK028AjaE18AYXFVsXacGuYg4sJzO4B35\\\\\\\/gFI\\\\\\\/YLVPeqAAPZJsRiHTA\\\\\\\/eQMUoHLaAQ9fILwfpLrCQpa73vEUUdmpM7HsYlrBZH3DLf+0tmEb3qjyFEZULAZeSWTpuvScFmskyD2UgL47TZK5KtcLnnvbtRzMaKTn6SrbLgUASHLOCuoSGlXIDKxtG7KlDVuUu\\\\\\\/ggQDXhX054WI1ZSkEhwd+RmA6sNbwabBfCod8H7Ga7G6514Je6\\\\\\\/jQ3WHGRVmMJQf13yeh2up6HwECVKtdjtsiYrRfBwH7xmgidstHW3hEMgakJVgrGo2nM8FYxN+XETTYepa0Ln0aD70yu2sJ0gZaeomnkI8wYxO\\\\\\\/a6Xl2e3eWk7Dmh2KjY6MzRl4Ac4dBxD9IIfa6BD6Wr6hQO2I3CPuOr5wSAVxtmBOFBihExh+YCRQMJM2PLwSQiXTOEbNqmipd1ruYfiCGiDKF332cg1XWg\\\\\\\/P\\\\\\\/3pf6mzNXrZtgpvtUTbg00jp26uyaoR0Pg4CHMHlWFNub9eYyGY0g7nPw1LJF4awALmwmkggzBAS+NQWStarrjcmgyR21JsVENI5B+1yW\\\\\\\/yfZJgGacYRxusRVR3jINQvDycXkXRnR\\\\\\\/ZoZ+O7hHgxRNEnwBHHfffNj\\\\\\\/HimCJGmiYXD2g4hqW6ETO2TEXMvgPSpW0FAar0H6umvso8Cq6F\\\\\\\/rOI1UDRx7ZCH0L1x8UhphlQ7hrjzw4xT0ziSMhV7UJJOM42T3k5rFaRbekkPShiuYbUpPKG+pWDf\\\\\\\/ZjoYvQoCLORJoCbjwmR8aj8n1z0teOzSnf3uUf30djo2kd0N968r7Nth76dkilp0nZDX+TUqx1AhnA+\\\\\\\/gqEJio38XS94Pwg0v\\\\\\\/WcplMJg2VpbjZBRe3HT1KHPJ66O7SiSa1Y2TavuVJQpuYlT+ZV0FqY47ri8NGf1a\\\\\\\/AFRyMYPdoDcm5u9jSCip98btPMJJDWfxZJXNBJSLm4pmD8dsFoQgve7ZZRoEjO9E3JYmFDDAa+0ledWI\\\\\\\/hdIgDKAHPWeEuHxJ7XeqESoVXRBudmTXTjrr\\\\\\\/o4nonb4j80mHkaB5b0OJnjQrAreC\\\\\\\/oJQ1zjY5R5lJyrjMszumd0QiYRFQ0+8FYQ\\\\\\\/NMcAKBAQajU1my3Rnq7wHIDdEM+vPZ6ydOzAK8bUZjXGSWpWel3r6Dbi39kzsGHaGyIh+XFYR3jdnkD1F5veOXpb7Rxfpe8IA3u8cjWQuF2c6To8NSmZ1k8zHyVrJvmGssqGJdhjNsrbHd3HWwLtvS0pFzPmVcfgRG7R8MxBFu19\\\\\\\/qA+KbiHqYfuNYe6creLryNPLu0KPSNfzN1UVflFybXRB2smZO6Xgs7+XTBLfSK0HaED8SVa0MkHMqVXYsyfzxrh+XBh+YLSn46tn8yqFwOCPiKqpPay7EO8bAdPBe9SksAOuFgN3uzHorQSuaJ5KwT7lQb1hCEVsiZ36E4HMzoQA9gWLd1vUPnOS7PdAMBneM2Xxe7+Fpv2PDiwlmGM6s+mCeh335DQvnD8xO5\\\\\\\/ST\\\\\\\/8SFnoW3b7bc25k1yfaAhQ6B2HxoLVqFWe+C5sX6fixA2CKuhDNR1gGJEfVurqnyUbkYNauLsIEQvnVYVjxSBhH6nOYKCv++huUinOlPS4Ysfs1jyeV7XqJrl7LUqlZHYYPzLHJQB6uqN5ql1ElCimEgSVciZUVWHFR5sMgXi+5TEKiJ78rASklzI53z62GUxbYynbUP7IU148KSRvAF8omy5ib2PG4dq7ozo8clISQNr+4wJNW7n\\\\\\\/0WC+fNNtprxE8t+on9hYnR5PdCpbySVuvYEMof\\\\\\\/HSKqdSTx8NIy4jbb9h6OjwIK3jdU+vYu7DSMVCVWGR4B8Xt1\\\\\\\/+\\\\\\\/4hkdVm4JsG1jM8bOlNJiw2qjq3NeZypBuyO6eT14JFRYrea0OkRCoo1uNMAo2zWUvpZnJYGOLRDSgwIOVEEOcnqTCpB1D5GUtWPNZ7BIuG1pQbtp\\\\\\\/frGgYyU8iKqH3v9c2NSvLgJPKuSJrgnLT79YbUwgXC\\\\\\\/xu4rgcqmKSKAbPq3L+HUUxhN8vDWygeb1oFMa8FtcsltpNDSbOZ32rLwLMm42zJdvhL7b4r5eZaGdlt1Bsa3H4PhPpaGpvvvsbwkhX5fD2s37XK7+qqecbcmLBcqWbNI0jsNauF8DIH3\\\\\\\/vYhfZATzV3bq7kBpFvLfEkGqreKhwssL6wDaboVQ+LGJIz17Q3siWbEhwX76+7corNwfB02BG9n2TZMhEtUc1TCadu7uq0xkpmuzm8T\\\\\\\/NduQtwFI3OIHa3jQqyebc\\\\\\\/KT4+IuFtwuBwbmln5qioYVi5LFChM\\\\\\\/LRzi9UOggjJH\\\\\\\/4SjP07yCOsSI3nc0gtpc+4bcaFPb\\\\\\\/0o\\\\\\\/50DaHTjlAv1yJmd3U0INgz73t4BeSD28SiJsKn6SrkENqFS3cvScxCUtX9lY91LE1jay+uBeujtz1QuN6VZvlINEhHwTY9rpqIS66\\\\\\\/NksypjOXpaWn2r3KLys8J263Km1wUgmy3d+prRavLatIWg4fI0H1jqk3JmnMz8UFGS571VDAM1MpO9FMJpL95uYDMcH2U8Am6nnbALMnDD4cXoCgk4yUjbAVNEBwbrB2Nsa3S1QBpIM2E32jTSpPqDqV2zsUcwM1sxztnUMjOfq1S1SDsXKeh\\\\\\\/c\\\\\\\/mXNlBQ5o4YDixSycbrF7+3KzyUCbZP+HadDvvMO+FJjJQoGP0Xl90llgUk3kIC7eI89ty6Jlv+t5qzWj9yxo8jsJrWc\\\\\\\/xL\\\\\\\/jcGrWIzs46j\\\\\\\/KjpOlAYBm6JGmCqYxWd\\\\\\\/z\\\\\\\/I6nwc8WRdxz4LkCy\\\\\\\/NqfxcGzwac2ZPzm1ViziT1HWUmuuWkjyl98hTOTqj9al1ObhxaooxCssx8\\\\\\\/p8lhtu5\\\\\\\/NVzi+Co8Sf5W7cXzDLnj8QEIe7iI4VgpN4Wm\\\\\\\/LVbIjNNOGyFzOkzv4uYEdFZApy1vCorVx6o4oqMbyLjb52tQjMiqmnLK4FLwuXik9BRpVQI7NXrfGwTJn7qPEObNU9o45KvPSkboflrK+pDVDQCg2j6oipy+3QBezTfrm4lVxzgKjeqHGR+ilbj7zt+USx2hcYtSxgoyYSIOC8kgOZmY99JQK6gRY8\\\\\\\/XTXua81veAmxrNBXWIXdu7CMosSDJDRTfkO5iDlnPFIoJo3exxXu0VmNuLyELIk30wOs7l+M4ROMoPKeWA\\\\\\\/ePA5v+MD4h2tZFd1Zh7k1cG5adpYFLayHCU145LkmacoeOOFv32otcPCijaZofkwbkR8fckKtyMVpYbpAcVMxtCy+QbYoqs4Nl0XGGImUmE+UpjSfPQcToVSHDSPtAgTr9AqW\\\\\\\/n0UepheONEwQcJ704zy9x4lcA4M0ttdkJmvF6sqq+YFV4hVcKi7YpmiZGRiOFk+EwvBTxJcG8q9vDKbG23sNonPraj9gYXr5pm4MGtg9srylwDXJ6ptDEi1A3iYQOA5h9kwhwbD6MKK0js4fiTqEVbgAZaWWxC5Pxu2GYLHZjO4y39zW2mWpYsbRvOcAJbCokwjF19kgMLcDcXHrBpGcr\\\\\\\/BQWdWTut2wkEpqSgNQ8O4g9d0tbcLFw9TyrlcKl8aFLQlKTLPAwJbOKCc0s4w9vqfQwjd+jbJ1xwhO6k1Fh1NxCDqWiRL5N0Nr\\\\\\\/ZfQqWKjuKVQCUHy9eoxWnqsi9pyMN5FB\\\\\\\/ZicpWmOLlWCkCDzSwmG1wPHCwZp1fylHffKiEZ3u4Vs0z47CJ4onBFZBjtVpFhYlXhZDN4Mt0zOEf3Y6jg9FZYK4RRXAJjhHf2ZeMQjjndo+mb7oa376Q0XchbfNSAQJHMT78peUP3p\\\\\\\/0mfJk33lCwWyAURpkYgOrGlrPO9LZwmXVwvEyzYnkCa8yhPZsgAmGp1eQajrYAV8qqpLEsU0dSBu86nRIR3wqsRDZjsZEr9bJcLSGYb0KTQ+XkGpHboPt\\\\\\\/Xct\\\\\\\/YN8NNP7sbDR2tAee\\\\\\\/ACMSLxAU7cQW6l3A\\\\\\\/hTtdSHah9wbGK4eep\\\\\\\/IAUdXSttMUllFKb8qFUmjEiJsFEObWHfnTp4nozeM3d4P0UTuTbgV6NXIFBAjZ0FRRXFwbShjdTUsdRN0DS3kVmZfo7TOTv8VprsfrqkECPf+z86IN3jXdCpdhVtXW9NBeJLx+QdZ0fxQw+9RAir2zK0qLIExB\\\\\\\/GCKmwyd\\\\\\\/SSAdOcHXI8tRbebdjmBG09aNZXTJeYRnCx5B9nVb7baxRMkc0GOmA6jTeZZp\\\\\\\/oJWZ84AXyWMFFS9YFfBhGBtOQOvtTjguxx9oil3N\\\\\\\/6aZ7N27jBQqUXW\\\\\\\/ayYmSfxbAqYToIP8lK6u5jSpgEY0TpmRjiY3z4EHtL3pQ1YNBRDHGZaL7kOvNFNFhvwaL16hWjZXUUtQ8+NobQ2sVIqI5BUrI9t6hvDEqHMZ+LjFAwcPjc5hLquLh5IHt3Na8r1T8UNCx2YA6jaH7lTzsqpiv42TEHZObOT913EbzXzU+xk7JUzNTfYGJGzxF9w4Q7CnjtTyXm2b2CensO8pgwUAIJmpyG41mlLWMb6a7WfQFLJ2Z9cYeoQAdJjOKt0OpXDRJUsNi1ZWMdbna0g4A8\\\\\\\/KZEIhB+0tWT+4diktE8Jqt7uN\\\\\\\/Lb0849ClEmxEsc14PwNT9jSN2JmkMdsrVUVoeYR2HytTGP6zJvB38jT7m8zlBSjt52W5Bl5BxSGDhnzmO4nw8EX1sZ2LHzewNqkHssgDbsM6runmSIYKyH7CUdJDjurN9kAAIj19mtQvVdCue\\\\\\\/H0y9Sfy0h6UOkZlOgLCCDhUG16O5nzP3l6aB2vYM1mx8P8n4AjyxYhryANtQGv6VFftJT5g5RxHBYxj3D+\\\\\\\/VyjCcaahVOIXCAjgS5\\\\\\\/Onm4EHvdaXCCmtfDYzJsw5k+J\\\\\\\/Xpq2dOPhND+9gwx+OSYTvoE59Y16gce6WEpUVLOOEQFyL\\\\\\\/Ae8O9j9afKEvZgHTL1VR68GuXWreTLb\\\\\\\/6C4zbGgGF9uhHyvv1fvESWUfDPnyvEr7odh+oR8OTjJFs\\\\\\\/7bR6v7tWPVGERZYTpPC78wHROFA4TBAVGt1SLT\\\\\\\/FlFF21cv70MxfimJIF9bsuxUjg2ad4Fxvb2Vann1HNujiAIyOvQVXD8HvCSFapvBg6PXBIRl04WTsZhq+Wdbavm0JYxF2wHvurSHW4VcA\\\\\\\/QtQvHCvMsEoB6WN7CuYHhSHZbf+cPg2VhJj0w6AMrYTh1bcH+MB8e4+vOA4u3Homk5o0cnAJVLpasYoRyc+aZH6qjnv\\\\\\\/BIdtgLsqz445K18nYf1PCDtaFTwpaBO5vXBtkFekdNKKbZBruLyUkTJEMH6RdbTBOBJl+BMr212C4kkDjAqohvv\\\\\\\/IRIl6GZNzQtMNa3ZBsF6Qboa+zO9vG6yPVy\\\\\\\/Pgax3M+f4ToDcNJ6TTBc6Vlw3hxLQF0FvswsauW9UU\\\\\\\/+rtQ+caCm9wvDLUgqtKmR\\\\\\\/Kfibd4L\\\\\\\/FJ6Bdt1kFid4+Jxx52587j\\\\\\\/zzP8tzmZ+IXuIqhYPPkieoY3KUIpoURzjRDb9C3ODWZ3z1QySD5Gc5ONO451k039mfKBuksowGAFxlq4SiIi9yu23P+ud2OFrRclDlsfgkeOTf84QbcUrL2Qo5QETmZe\\\\\\\/5mUPHIojHyiupc1bhBs4UupPDUtw6Uj0H8uXdLdGkYj593Nce3X2u5GKXAP1PPck6jHE6lzPAlPU56i4S+8WsxkejS6+uhqAL1l8QGalpyh7ugqg1TTZphVGbOWywKhyCT6h9e4tAFIYeAZ2GI7d7STnmdHW70OG872qp1791ayUW4EG9UjfWRXKTcz2LvkD2GfQYcfKpXQLBa1SjW5vs26giHhXeVYZos6e5BRgJ2AzOBezJMMF1J7y4hJNGwnc6sPaniNJ8cD\\\\\\\/ppTAHWE7tkwEjXyEHXnEG\\\\\\\/L2xpxYbodBCF\\\\\\\/Q3\\\\\\\/XH\\\\\\\/iHhuK8Cll6LPjsekZ4ztNzS\\\\\\\/WSkykHwd5Hvral+FvobTERq46pJWeIGZMLcH5EZyDmaoS1VxwOS4HGkQHiDMqidcE+Ek7ZaSC5NoocgtH3xgCYJH9yxzgRwtxv07sXHEI3YRJgo9CRVpDz8H1uCZgb8vuQz9\\\\\\\/8qxZGG91xx4kr6JB0qdzaSC6LwlqJ8HjRoc+Ay\\\\\\\/YgRDcuUSZRV4RW1PNCShVp9l8TOd1VCg\\\\\\\/5cm5IcOqdBHjTMBNcfiuPlMOK5DtEsk5hEiKXbE+XCxQ5xgF9UQB+bmNJ2Xpi3lTY5E29fWSHg7Auw88f3okPrFlAI5+RT0SMJFt53Fg1xroaFW0AJUsIeyeteIzHKCidG++ABsLKjj0xZyLq7\\\\\\\/NMGDNHtaSCVp8ob11wj\\\\\\\/eEsDuMoZaV7FS5AjTv2uHNXjF2tg01qz8Vd5R03V08y1na6reSF2Zvhl2XWXgYaoI4wU\\\\\\\/Cpbvhabr2oYuSuyZVjeQbsf5EqmEwtM4SpNyd8YTsZqQXEA\\\\\\\/k02mln2XM3pePwt8i9qyqwgl6fuiK12H79eRO6XlXO0unLaDTY2xERE+LPaO7y27deObrvHTVYVctR2W2Q0Sfo5gro6\\\\\\\/hp1dyjTopZzTI+00D5NL95HVSYDmLYUZDvQOa+pUbXIr4031MmEfduzntKwMtOXJ43hQZut6hJTCFIa8LtgkinJl4CG9NM08cZu5U87na6SRtb2aAkl8TrES+FsEMnsNKfIRLB99ESENT9wHXie7BPbWiACS1dRIzi\\\\\\\/H8ls6lm2kfz\\\\\\\/erLRkW8sSKxfyfjLya8j4YfuNI3SvRYP6BGO4uTXu9keUX6j1zloc17yPFmeERnMQ3G4VxVQTqQPSArSSCLzZ+28jQZrP9X7vm7ou71RNNx+WSB7vcOgo5q2oY4lLfxS87iPho9mA\\\\\\\/OExaRhIuhyEIIe7Wefq5Hn8S50RUPk6VZCrjWC5Il4LWChRU7waNo\\\\\\\/7YxcLrxJ\\\\\\\/OBG1rMcn\\\\\\\/4vIvnToU3RYs0K3Ec4\\\\\\\/VE3N\\\\\\\/tZ0hXLgc58cGnYZfdszPdqyB437NIa2lv7XQcn6sqHVGdOM+5LR5dlVZwyT90FHvgw+Y\\\\\\\/nAfaxRUrJr9Rcrlx9r1M7DGwkYI+JhcFio8BwIA5Wiy8RklVuEOcqkd\\\\\\\/JCaSHeUUqj6BFjVgTqKO\\\\\\\/4y9zmlyf4eGK9wIFO\\\\\\\/s8cVMyDwbJjl5ccBrkSgTdcY7uH3jSZScbWlb70QHskxnlPjHR1EMKrIxiV0guKH7p1mYzy\\\\\\\/KVQlA7th9ovCAN4\\\\\\\/xe6UBDppvAD9sjos+VEDB0IDXymxeTPIVBshXk+0cuc6wp\\\\\\\/vbm7Q6yFNiog\\\\\\\/7lxqTicapa4TJEMxmag4hZ8fba0kU2ShjkyCavOVMdD7dbpkfiPG76h2ZxLiY2KutiYypdnEghtQAhl2ku8dC6jMDgnduT++B2gyrWkxum7tV8K1dlSfr8eLtKa4W1RoV4RL9g3fue5k9MQR5+CTRlYmslMi64=\\\u0022,\\\u0022iv\\\u0022:\\\u00222Y77zPmXRKaV7H33UFeeuA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002201ae6903e5ff4998045f16c25898be27fe0594eb381f4f0fd4e75de185644e92\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ooOZBm2LXoqSdDP+7aGPPTcfJdmSMAeBU8nnumA1dYrmAAqbIEQm6xi9iWZ1y9NOKRD4kOIs\\\\\\\/Okn5Fch8bDpQN\\\\\\\/DrynWsnkjyHhvI+s8MkW5rFRiSBQm8S3cIBkJr1iQzNGeyAz92V7MhXRXY4NhNxQrdjISH0b\\\\\\\/0H7cfciTEeHULK6ueLTCw2PrmaEWiKFVe3gaNFqHQwkBg+g9OIU972iQ3vuDI412Ob7mql0zeNSSa8JkDjhNbXam1UxppL\\\\\\\/KDKy6rTdhoF4xRK0AbuLzbvtGUaGZUjaIGQtX43a8gpyWDah7jeAk49LEzhna+HMfmiZXAMUuGlym5OIV3MJZciCFhs8TT3A589OPqlgZlNYlcRsjQDmwMB1Q3aF\\\\\\\/QaD\\\\\\\/ilYX8Iv3qse0TO5\\\\\\\/TJPZ82n10WVGI8WR1UuB0CCl6yl2ggfldKM4dq6OYty0zWWPnR2g04\\\\\\\/RxmyvODOYcSCrcoNGBAjtHH6f6Q8eM8Bi0rCEpQuk0SzhKP9RhUTcLFKeItlM8zGO8TbDJe+JhXR\\\\\\\/GgSr4MqibUlTvkJMez0Hbhf9w0QX8MORD2CtqXXDW+beSmFl9U4OPztN9SmGM1Vk4VXKnuJcPmD0qUN5NbV+u26iAYAXJ\\\\\\\/J04SDCu\\\\\\\/OGfAg1NK6MnZyAqihVp4K5V6j1P0tfydQ4y\\\\\\\/Lte9mqi9FtftPALS2HSpOE+hOS7Y9vIDemImZcm5S1hSA7R3MdpxvWd0OnmSqrr2m8aXBQG\\\\\\\/dpest0W1gdQ2chtGVQLIrZDtDmhd3t5hHWkpsdyW+NTP9jCM+LoMyaRC5Eg7RFWEV4QgcpYT\\\\\\\/L2nuk8gdKQPoPf2FP5n+KN\\\\\\\/6Dq1FuZKqHwtubm6McbJJmgtoCeNKLqG8lvOXgVtZF\\\\\\\/Dywflhk+eTiLwXRvft8gklfeqIbtO9joAP1ZFcZTLCmronhz18ppTAK1xAg4KxCk8J\\\\\\\/jEy0V+bxZ3173oyLuF77iBF+ac6kU2TNUqX59ig7LYwZoyv6wXqmbKRhGplpShi3z7gXMPcnDFJauWu5BaCkfWFMzMmQuUscMxEkIb5TgSfy8N2OiqU845ehoj2feFdFqUU7Zu5faomUNh\\\\\\\/5dRaGzLq4PixhxJbsCWXUAdU0z2jT+fxzK2Jc0bbtiASJfPBtMS2T9Rb8NvmPopPaJQJXDMlrNXPT2UooSRhM66rZ1FSPtt2aBYpyYZQQKmjJWY1onXDftmqLFsUnbrtUJS0TuWYBRShmX39\\\\\\\/wE5x9MrZarZrItQOqgQkioH4Fp+5\\\\\\\/E8wwinZjY2X2\\\\\\\/UOEGHpDo3M5BGCQOLCBtY2jwJ0ad3sZqk3mNBrMJKnuVD1jju7ybI1yK1p9me39YJ5N6N64FaE2XUFJYSoRDpeRaCbL+bp7LQYblribL3HHAt9INRi2EjK\\\\\\\/sASjfnIBK\\\\\\\/gpLxQn\\\\\\\/jBxDEIUBzctb9\\\\\\\/27HnkzNifPeJ680Po6J1plnot2AKIdptEepej+lBZpGqFgJ0LZxgmYlBUxrIs\\\\\\\/87KnK\\\\\\\/g8d0gpHbNPkEXIWkIkZqNNZTgdJQYZPWYoCrOPNaOf2cOQCobdtO48jLx3goQVysX4h8lUpHR2SZlwHII00Rj5wNLAQsDZOLyY7kzpcSddic4n9rPKTiZLfcWZCSfH1AdynJMP9AFjVAOU1BhYTxotGxk0geHBwgrPHDVtxEho1Vt+qPCJCP+tKdMsX\\\\\\\/d\\\\\\\/KG0nGNPw2D1kMkhjBBZcgJjcaFuwgbk3kcmd+1bp2sDEMzeRZI51155wjemxeBi53PbI72wj2IAiBeIDnh9dqE1\\\\\\\/N0t5iPMoI2CEOEg9S0SIEku7O07gcqr2z1kh05duvNViv8MdzDvAEdlM8GaYNcp07h9RVC8UXc4\\\\\\\/nYhWQpK0\\\\\\\/qEc0fUijwRVsRjYAp+2ZfKyTncz993Ajy0J35zD9Ex9EvgSojRry9cCeGhOTQ6pYq0LSe0vTk7L6\\\\\\\/crkTF31X1yTB9mOcmsi5DPKIk2VII79Lj8zE3hOLErildRrdXqqDXc7caAhp8u1DKRzCxElce6tBPIRC4eX+0m4xIS8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022n+Wll\\\\\\\/L24pZOw04aWFKLoA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e146b52a4b1152649c3eb536362326defc4e52d551ee849a59ccb1ab76da6cde\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495847 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/951" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-S8hNsEWJwSdc-GvW-BqQACRE_yrLIwce_IeH4w8_YW4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/951" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YoHu_pqpX9Sw2PACniWt5tsmit9YD2pBO-7rguUBBkc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/945" title="CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/967" title="The Drosophila Post-mating Response: Gene Expression and Behavioral Changes Reveal Perdurance and Variation in Cross-Tissue Interactions" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6432-0033" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xinzhu Zhou</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5465-6601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Celine L. St. Pierre</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5289-0410" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Natalia M. Gonzales</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2045-3272" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jennifer Zou</span>, <span class="highwire-citation-author" data-delta="4">Riyan Cheng</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1709-9214" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Apurva S. Chitre</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-3651-1980" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Greta Sokoloff</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3634-0747" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Abraham A. Palmer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">951-965;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400763 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495847" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xinzhu Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biomedical Sciences Graduate Program, University of California San Diego, La Jolla, CA, 92092,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinzhu%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXinzhu%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6432-0033" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xinzhu Zhou</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Celine L. St. Pierre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Washington University School of Medicine, St. Louis, MO, 63110,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celine%2BL.%2BSt.%2BPierre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=St.%20Pierre%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACeline%2BL.%2BSt.%2BPierre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5465-6601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Celine L. St. Pierre</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Natalia M. Gonzales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Human Genetics, University of Chicago, IL, 60637,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2BM.%2BGonzales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzales%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANatalia%2BM.%2BGonzales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5289-0410" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Natalia M. Gonzales</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jennifer Zou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Computer Science, University of California, Los Angeles, CA, 90095,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BZou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJennifer%2BZou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2045-3272" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jennifer Zou</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Psychiatry,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Apurva S. Chitre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Psychiatry,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Apurva%2BS.%2BChitre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chitre%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AApurva%2BS.%2BChitre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1709-9214" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Apurva S. Chitre</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Greta Sokoloff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Psychological & Brain Sciences, University of Iowa, Iowa City, IO, 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Greta%2BSokoloff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sokoloff%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGreta%2BSokoloff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3651-1980" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Greta Sokoloff</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Abraham A. Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Psychiatry,</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Genomic Medicine, University of California San Diego, La Jolla, CA, 92037 and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abraham%2BA.%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbraham%2BA.%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3634-0747" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Abraham A. Palmer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aap@ucsd.edu" class="" data-icon-position="" data-hide-link-title="0">aap@ucsd.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/3/951" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495847" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495847/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/951.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495847" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495847/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/951.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495847" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495847/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/3/951.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495847" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495847/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">There has been extensive discussion of the “Replication Crisis” in many fields, including genome-wide association studies (<strong>GWAS</strong>). We explored replication in a mouse model using an advanced intercross line (<strong>AIL</strong>), which is a multigenerational intercross between two inbred strains. We re-genotyped a previously published cohort of LG/J x SM/J AIL mice (F<sub>34</sub>; n = 428) using a denser marker set and genotyped a new cohort of AIL mice (F<sub>39-43</sub>; n = 600) for the first time. We identified 36 novel genome-wide significant loci in the F<sub>34</sub> and 25 novel loci in the F<sub>39-43</sub> cohort. The subset of traits that were measured in both cohorts (locomotor activity, body weight, and coat color) showed high genetic correlations, although the SNP heritabilities were slightly lower in the F<sub>39-43</sub> cohort. For this subset of traits, we attempted to replicate loci identified in either F<sub>34</sub> or F<sub>39-43</sub> in the other cohort. Coat color was robustly replicated; locomotor activity and body weight were only partially replicated, which was inconsistent with our power simulations. We used a random effects model to show that the partial replications could not be explained by Winner’s Curse but could be explained by study-specific heterogeneity. Despite this heterogeneity, we performed a mega-analysis by combining F<sub>34</sub> and F<sub>39-43</sub> cohorts (n = 1,028), which identified four novel loci associated with locomotor activity and body weight. These results illustrate that even with the high degree of genetic and environmental control possible in our experimental system, replication was hindered by study-specific heterogeneity, which has broad implications for ongoing concerns about reproducibility.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/mouse-genetics" class="hw-term hw-article-keyword hw-article-keyword-mouse-genetics" rel="nofollow">Mouse Genetics</a></li><li class="kwd"><a href="/keyword/replication" class="hw-term hw-article-keyword hw-article-keyword-replication" rel="nofollow">Replication</a></li><li class="kwd"><a href="/keyword/multiparent-advanced-generation-inter-cross-magic" class="hw-term hw-article-keyword hw-article-keyword-multiparent-advanced-generation-inter-cross-magic" rel="nofollow">Multiparent Advanced Generation Inter-Cross (MAGIC)</a></li><li class="kwd"><a href="/keyword/multiparental" class="hw-term hw-article-keyword hw-article-keyword-multiparental" rel="nofollow">multiparental</a></li><li class="kwd"><a href="/keyword/populations" class="hw-term hw-article-keyword hw-article-keyword-populations" rel="nofollow">populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">Genome-wide association studies (<strong>GWAS</strong>) in model organisms can use genetically identical cohorts phenotyped under extremely similar conditions, which would be expected to enhance the success of replication. We sought to investigate replication in model organism GWAS using a mouse advanced intercross line (<strong>AIL</strong>). The use of GWAS in model organisms such as mice (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Talbot <em>et al.</em> 1999</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Demarest <em>et al.</em> 2001</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Yalcin <em>et al.</em> 2004</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Valdar <em>et al.</em> 2006</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ghazalpour <em>et al.</em> 2008</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Samocha <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Churchill <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Parker <em>et al.</em> 2012</a>, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">2016</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Svenson <em>et al.</em> 2012</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carbonetto <em>et al.</em> 2014</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chesler 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Coyner <em>et al.</em> 2014</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gatti <em>et al.</em> 2014</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Nicod <em>et al.</em> 2016</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hernandez Cordero <em>et al.</em> 2018</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">2019</a>), rats (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baud <em>et al.</em> 2014</a>), chickens (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Besnier <em>et al.</em> 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Johnsson <em>et al.</em> 2018</a>), fruit flies (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">King <em>et al.</em> 2012</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Mackay <em>et al.</em> 2012</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Kislukhin <em>et al.</em> 2013</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Marriage <em>et al.</em> 2014</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Vonesch <em>et al.</em> 2016</a>), <em>C. elegans</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Doitsidou <em>et al.</em> 2016</a>) and various plant species (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Rishmawi <em>et al.</em> 2017</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cockram and Mackay 2018</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Diouf <em>et al.</em> 2018</a>) has become increasingly common over the last decade. These mapping populations can further be categorized as multi-parental crosses, which are created by interbreeding two or more inbred strains, and various outbred populations, in which the founders are of unknown provenance. An F<sub>2</sub> cross between two inbred strains is the prototypical mapping population; however, F<sub>2</sub>s provide poor mapping resolution (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Parker and Palmer 2011</a>). To improve mapping resolution, Darvasi and Soller (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Darvasi and Soller 1995</a>) proposed the creation of advanced intercross lines (<strong>AILs</strong>), which are produced by intercrossing F<sub>2</sub> mice for additional generations. AILs accumulate additional crossovers with every successive generation, leading to a population with shorter linkage disequilibrium (<strong>LD</strong>) blocks, which improves mapping precision, albeit at the expense of power (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Parker and Palmer 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gonzales and Palmer 2014</a>).</p><p id="p-4">The longest running mouse AIL was generated by crossing LG/J and SM/J inbred strains, which had been previously selected for large and small body size prior to inbreeding and subsequent intercrossing. We obtained this AIL in 2006 at generation 33 from Dr. James Cheverud (Jmc: LG,SM-G<sub>33</sub>). Since then, we have collected genotype and phenotype information from multiple generations, including F<sub>34</sub> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Lionikas <em>et al.</em> 2010</a>; <a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Samocha <em>et al.</em> 2010</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Parker <em>et al.</em> 2011</a>, <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">2014</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bartnikas <em>et al.</em> 2012</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carroll <em>et al.</em> 2017</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Gonzales <em>et al.</em> 2018</a>) and F<sub>39</sub>-F<sub>43</sub>. Our previous publications using the F<sub>34</sub> generation employed a custom Illumina Infinium genotyping microarray to obtain genotypes for 4,593 SNPs (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>; <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Parker <em>et al.</em> 2014</a>); we refer to this set of SNPs as the ‘sparse markers’. Those genotypes were used to identify significant associations for numerous traits, including locomotor activity in response to methamphetamine (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>), pre-pulse inhibition (<a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">Samocha <em>et al.</em> 2010</a>), muscle weight (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Lionikas <em>et al.</em> 2010</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Hernandez Cordero <em>et al.</em> 2019</a>), body weight (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Parker <em>et al.</em> 2011</a>), open field (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Parker <em>et al.</em> 2014</a>), conditioned fear (<a id="xref-ref-68-4" class="xref-bibr" href="#ref-68">Parker <em>et al.</em> 2014</a>), red blood cell parameters (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bartnikas <em>et al.</em> 2012</a>), and muscle weights (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Carroll <em>et al.</em> 2017</a>). Although not previously published, we also collected phenotype information from the F<sub>39-43</sub> generations, including body weight, fear conditioning, locomotor activity in response to methamphetamine, and the light dark test for anxiety.</p><p id="p-5">While the prior GWAS using the F<sub>34</sub> generation detected many significant loci, the sparsity of the markers likely precluded the discovery of some true loci and also made it difficult to clearly define the boundaries of the loci that we did identify. For example, Parker <em>et al.</em> conducted an integrated analysis of F<sub>2</sub> and F<sub>34</sub> AILs (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Parker <em>et al.</em> 2011</a>). One of their body weight loci spanned from 87.93–102.70 Mb on chromosome 14. Denser markers might have more clearly defined the implicated region.</p><p id="p-6">In the present study, we used genotyping-by-sequencing (<strong>GBS</strong>), which is a reduced-representation sequencing method (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Davey <em>et al.</em> 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Fitzpatrick <em>et al.</em> 2013</a>), to obtain a much denser set of SNPs in the F<sub>34</sub> cohort and, for the first time, genotyped mice from the F<sub>39-43</sub> generations. With this denser set of SNPs, we attempted to identify novel loci in the F<sub>34</sub> cohort that were not detected using the sparse SNPs. We also performed GWAS using the mice from the F<sub>39-43</sub> AILs. We explored whether imputation from the array SNPs could have provided the additional coverage we obtained using the denser GBS genotypes. Because F<sub>39-43</sub> AILs are the direct descendants of the F<sub>34</sub>, they are uniquely suited to serve as a replication population for GWAS in the F<sub>34</sub> generation. For the subset of traits measured in both cohorts, we attempted to replicate the results discovered in one cohort in the other. To set our expectations for replication, we performed simulations to estimate the power for these replication studies. Because the actual rate of replication was lower than predicted by the power analysis, we used a random effects model to evaluate the role of Winner’s Curse and study-specific heterogeneity in the low rate of replication. Finally, we also performed a mega-analysis on a subset of traits common to both cohorts.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Animals</h3><p id="p-7">All mice used in this study were members of the LG/J x SM/J AIL that was originally created by Dr. James Cheverud (Loyola University Chicago, Chicago, IL). This AIL has been maintained in the Palmer laboratory since generation F<sub>33</sub>. Age and exact number of animals tested in each phenotype are described in Table S1. Several previous publications (<a id="xref-ref-79-4" class="xref-bibr" href="#ref-79">Samocha <em>et al.</em> 2010</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>; <a id="xref-ref-68-5" class="xref-bibr" href="#ref-68">Parker <em>et al.</em> 2014</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Lionikas <em>et al.</em> 2010</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Carroll <em>et al.</em> 2017</a>; <a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Parker <em>et al.</em> 2011</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bartnikas <em>et al.</em> 2012</a>) have reported association analyses of the F<sub>34</sub> mice (n = 428). No prior publications have described the F<sub>39-43</sub> generations (n = 600). The sample size of F<sub>34</sub> mice reported in this study (n = 428) is smaller than that in previous publications of F<sub>34</sub> (n = 688) because we only genotyped a subset of F<sub>34</sub> animals using GBS.</p></div><div id="sec-3" class="subsection"><h3>F<sub>34</sub>, F<sub>39-43</sub> Phenotypes</h3><p id="p-8">All phenotypes are listed in Table S1. We have previously described the phenotyping of F<sub>34</sub> animals for locomotor activity and locomotor response to methamphetamine (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>), fear conditioning (<a id="xref-ref-68-6" class="xref-bibr" href="#ref-68">Parker <em>et al.</em> 2014</a>), open field (<a id="xref-ref-68-7" class="xref-bibr" href="#ref-68">Parker <em>et al.</em> 2014</a>), coat color, body weight (<a id="xref-ref-69-5" class="xref-bibr" href="#ref-69">Parker <em>et al.</em> 2011</a>), complete blood counts (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bartnikas <em>et al.</em> 2012</a>), heart and tibia measurements (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Lionikas <em>et al.</em> 2010</a>), muscle weight (<a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Lionikas <em>et al.</em> 2010</a>). Iron content in liver and spleen, which have not been previously reported in these mice, was measured by atomic absorption spectrophotometry, as described in Gardenghi <em>et al.</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gardenghi <em>et al.</em> 2007</a>) and Graziano, Grady and Cerami (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Graziano <em>et al.</em> 1974</a>). Although the phenotyping of F<sub>39-43</sub> animals has not been previously reported, we followed previously published protocols for locomotor activity and locomotor response to methamphetamine (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>), coat color, body weight (<a id="xref-ref-69-6" class="xref-bibr" href="#ref-69">Parker <em>et al.</em> 2011</a>), and light/dark test for anxiety (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Sittig <em>et al.</em> 2016</a>). We point out here that even though “locomotor activity” was measured in both the F<sub>34</sub> and F<sub>39-43</sub> using the Versamax software (AccuScan Instruments, Columbus, OH), “open field” in the F<sub>34</sub> cohort was also measured using Versamax, whereas “open field” in the F<sub>39-43</sub> cohort was measured using the EthoVision XT software (Noldus system; (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Noldus <em>et al.</em> 2001</a>)). Because there are meaningful differences in these experimental procedures, we did not attempt to use “open field” data for replication. In summary, we performed GWAS on all traits collected in individual cohorts. For the replication analysis between the F<sub>34</sub> and F<sub>39-43</sub> cohorts, we only directly compared a number of traits that had been measured in both cohorts: body weight, two Mendelian coat color traits (albino and agouti), and three locomotor activity traits (locomotor activity on day 1 and on day 2, and activity on day 3 following a methamphetamine injection).</p></div><div id="sec-4" class="subsection"><h3>F<sub>34</sub> AIL Array Genotypes</h3><p id="p-9">F<sub>34</sub> animals had been genotyped on a custom SNP array on the Illumina Infinium platform (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>; <a id="xref-ref-68-8" class="xref-bibr" href="#ref-68">Parker <em>et al.</em> 2014</a>), which yielded a set of 4,593 SNPs on autosomes and X chromosome that we refer to as ‘sparse SNPs’.</p></div><div id="sec-5" class="subsection"><h3>F<sub>34</sub> and F<sub>39-43</sub> GBS Genotypes</h3><p id="p-10">F<sub>34</sub> and F<sub>39-43</sub> animals were genotyped using genotyping-by-sequencing (<strong>GBS</strong>), which is a reduced-representation genome sequencing method (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Parker <em>et al.</em> 2016</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gonzales <em>et al.</em> 2017</a>). We used the same protocol for GBS library preparation that was described in Gonzales <em>et al.</em> (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Gonzales <em>et al.</em> 2017</a>). We called GBS genotype probabilities using ANGSD (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Korneliussen <em>et al.</em> 2014</a>). GBS identified 1,667,920 autosomal and 43,015 X-chromosome SNPs. To fill in missing genotypes at SNPs where some but not all mice had calls, we ran within-sample imputation using Beagle v4.1, which generated hard call genotypes as well as genotype probabilities (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Browning and Browning 2007</a>). After imputation, only SNPs that had dosage <em>r<sup>2</sup></em> &gt; 0.9 were retained. We removed SNPs with minor allele frequency &lt; 0.1 and SNPs with <em>P</em> &lt; 1.0×10<sup>−6</sup> in the Chi-square test of Hardy–Weinberg Equilibrium (<strong>HWE</strong>) (Table S2). All phenotype and GBS genotype data are deposited in GeneNetwork2 (<a href="http://gn2.genenetwork.org/">http://gn2.genenetwork.org/</a>).</p></div><div id="sec-6" class="subsection"><h3>QC of individuals</h3><p id="p-11">We have found that large genetic studies are often hampered by cross-contamination between samples and sample mix-ups. We used four features of the data to identify problematic samples: heterozygosity distribution, proportion of reads aligned to sex chromosomes, pedigree/kinship, and coat color. We first examined heterozygosity across autosomes and removed animals where the proportion of heterozygosity was more than 3 standard deviations from the mean (Figure S1). Next, we sought to identify animals in which the recorded sex did not agree with the sequencing data. We compared the ratio of reads mapped to the X and Y chromosomes. The 95% CI for this ratio was 196.84 to 214.3 in females and 2.13 to 2.18 in males. Twenty-two F<sub>34</sub> and F<sub>39-43</sub> animals were removed because their sex (as determined by reads ratio) did not agree with their recorded sex; we assumed this discrepancy was due to sample mix-ups. To further identify mislabeled samples, we calculated kinship coefficients based on the full AIL pedigree using QTLRel. We then calculated a genetic relatedness matrix (<strong>GRM</strong>) using IBDLD (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abney 2008</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">L. Han and Abney 2011</a>), which estimates identity by descent using genotype data. The comparison between pedigree kinship relatedness and genetic kinship relatedness identified seven pairs of animals that showed obvious disagreement between kinship coefficients and the GRM, these animals were excluded from further analysis. Lastly, we excluded 14 F<sub>39-43</sub> animals that showed discordance between their recorded coat color and their genotypes at markers flanking <em>Tyr</em>, which causes albinism in mice. The numbers of animals filtered at each step are listed in Table S2. Some animals were detected by more than one QC step, substantiating our evidence that these samples were erroneous.</p><p id="p-12">At the end of SNP and sample filtering, we had 59,561 autosomal and 831 X chromosome SNPs in F<sub>34</sub>, 58,966 autosomal and 824 X chromosome SNPs in F<sub>39-43</sub>, and 57,635 autosomal and 826 X chromosome SNPs in the combined F<sub>34</sub> and F<sub>39-43</sub> set (Table S2). GBS genotype quality was estimated by examining concordance between the 66 SNPs that were present in both the array and GBS genotyping results (Figure S3).</p></div><div id="sec-7" class="subsection"><h3>LD decay</h3><p id="p-13">Average LD (<em>r<sup>2</sup></em>) was calculated using allele frequency matched SNPs (MAF difference &lt; 0.05) within 100,000 bp distance, as described in Parker <em>et al.</em> (<a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Parker <em>et al.</em> 2016</a>).</p></div><div id="sec-8" class="subsection"><h3>Imputation to LG/J and SM/J reference panels</h3><p id="p-14">F<sub>34</sub> array genotypes (n = 428) and F<sub>34</sub> GBS genotypes (n = 428) were imputed to LG/J and SM/J whole genome sequence data (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Nikolskiy <em>et al.</em> 2015</a>) using BEAGLE (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Browning and Browning 2007</a>). For F<sub>34</sub> array imputation, we used a large window size (100,000 SNPs and 45,000 SNPs overlap). Imputation to reference panels yielded 4.3 million SNPs for F<sub>34</sub> array and F<sub>34</sub> GBS imputed sets. Imputed SNPs with r<sup>2</sup> &gt; 0.9, MAF &gt; 0.1, HWE p-value &gt; 1.0×10<sup>−6</sup> were retained, resulting in 4.1M imputed F<sub>34</sub> GBS SNPs and 4.3M imputed F<sub>34</sub> array SNPs.</p></div><div id="sec-9" class="subsection"><h3>Genome-wide association analysis (GWAS)</h3><p id="p-15">We used the linear mixed model, as implemented in GEMMA (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Zhou and Stephens 2012</a>), to perform a GWAS that accounted for the complex familial relationships among the AIL mice (<a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Gonzales <em>et al.</em> 2017</a>). We used the leave-one-chromosome-out (<strong>LOCO</strong>) approach to calculate the GRM, which effectively circumvented the problem of proximal contamination (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cheng <em>et al.</em> 2013</a>). We used the univariate linear mixed model described in Zhou and Stephens (<a id="xref-ref-99-2" class="xref-bibr" href="#ref-99">Zhou and Stephens 2012</a>):</p><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-1.gif"/></noscript></span></span><p id="p-16">where y is a n-vector of traits for n individuals; W is a n×c matrix of covariates (fixed effects); α is a c-vector of the corresponding coefficients; x is an n-vector of genotypes; β is the effect size of the genotype; u is an n-vector of random effects; ε is an n-vector of errors; <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-2.gif"/></noscript></span></span>is the variance of the residual errors; λ is the ratio between the two variance components; K is a known n × n relatedness matrix and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-3.gif"/></noscript></span></span> is an n × n identity matrix. <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-4.gif"/></noscript></span></span> stands for the n-dimensional multivariate normal distribution (<a id="xref-ref-99-3" class="xref-bibr" href="#ref-99">Zhou and Stephens 2012</a>).</p><p id="p-17">Separate GWAS were performed using the F<sub>34</sub> array genotypes, the F<sub>34</sub> GBS genotypes, and the F<sub>39-43</sub> GBS genotypes. Apart from coat color (binary trait), raw phenotypes were quantile normalized prior to analysis. Coat color traits were coded as follows: albino: 1 = white, 0 = non-white; agouti: 1 = tan, 0 = black, NA = white. Because F<sub>34</sub> AIL had already been studied, we used the same covariates as described in Cheng <em>et al.</em> (<a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>) in order to examine whether our array and GBS GWAS would replicate their findings. We included sex and body weight as covariates for locomotor activity traits (see covariates used in (<a id="xref-ref-12-10" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>)) and sex, age, and coat color as covariates for fear conditioning and open field test in F<sub>34</sub> AILs (see covariates used in (<a id="xref-ref-68-9" class="xref-bibr" href="#ref-68">Parker <em>et al.</em> 2014</a>)). We used sex and age as covariates for all other phenotypes. Covariates for each analysis are shown in Table S1. Finally, we performed mega-analysis of F<sub>34</sub> and F<sub>39-43</sub> animals (n = 1,028) for body weight, coat color, and locomotor activity, since these traits were measured in the same way in both cohorts. We quantile transformed all continuous phenotypes in each cohort and then combined the transformed phenotypes for the mega-analysis (coat color traits were not quantile normalized because they are binary).</p></div><div id="sec-10" class="subsection"><h3>Identifying dubious SNPs</h3><p id="p-18">Some significant SNPs in the F<sub>34</sub> GWAS were dubious because the flanking SNPs, which would have been expected to be in high LD with the significant SNP (a very strong assumption in an AIL), did not have high -log10(p) values. We only examined SNPs that obtained significant p-values; close examinations revealed that these SNPs had dubious ratios of heterozygotes to homozygotes calls and had corresponding HWE p-values that were close to our 1.0×10<sup>−6</sup> threshold (Table S3). We chose the 1.0×10<sup>−6</sup> as the filter threshold of the HWE p-values based on a gene-dropping exercise. We used the F<sub>33-34</sub> family pedigree and the F<sub>34</sub> genetic map to simulate the genotypes in F<sub>34</sub> (QTLRel; (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cheng <em>et al.</em> 2011</a>)). The p-value of the chi-square test for Hardy-Weinberg equilibrium in the simulated F<sub>34</sub> population was 7.24329×10<sup>−06</sup>, which was close to the HWE threshold used in Gonzales <em>et al.</em> (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Gonzales <em>et al.</em> 2018</a>). To avoid counting these as novel loci, we removed those SNPs prior to summarizing our results as they likely reflected genotyping errors.</p></div><div id="sec-11" class="subsection"><h3>Selecting independent significant SNPs</h3><p id="p-19">To identify independent “lead loci” among significant GWAS SNPs that surpassed the significance threshold, we used the LD-based clumping method in PLINK v1.9. We empirically chose clumping parameters (<em>r<sup>2</sup></em> = 0.1 and sliding window size = 12,150kb) that gave us a conservative set of independent SNPs (Table S4). For the coat color phenotypes, we found that multiple SNPs remained significant even after LD-based clumping, presumably due to the extremely significant associations at these Mendelian loci. In these cases, we used a stepwise model selection procedure in GCTA (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Yang <em>et al.</em> 2011</a>) and performed association analyses conditioning on the most significant SNPs.</p></div><div id="sec-12" class="subsection"><h3>Significance thresholds</h3><p id="p-20">We used MultiTrans to set significance thresholds for GWAS (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">B. Han <em>et al.</em> 2009</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Joo <em>et al.</em> 2016</a>). MultiTrans is a method that assumes multivariate normal distribution of the phenotypes, which in LMM models, contain a covariance structure due to various degrees of relatedness among individuals. We were curious to see whether MultiTrans produced significance thresholds that were different from the thresholds we obtained from a standard permutation test (‘naïve permutation’ as per Cheng <em>et al.</em> (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cheng <em>et al.</em> 2013</a>)). We performed 1,000 permutations using the F<sub>34</sub> GBS genotypes and the phenotypic data from locomotor activity (days 1, 2, and 3). We found that the 95<sup>th</sup> percentile values for these permutations were 4.65, 4.79, and 4.85, respectively, which were very similar to 4.85, the threshold obtained from MultiTrans using the same data. Thus, the thresholds presented here were obtained from MultiTrans but are similar (if anything slightly more conservative) to the thresholds we would have obtained had we used permutation. Because the effective number of tests depends on the number of SNPs and the specific animals used in GWAS, we obtained a unique adjusted significance threshold for each SNP set in each animal cohort (Table S5).</p></div><div id="sec-13" class="subsection"><h3>Credible set analysis</h3><p id="p-21">We followed the method described in (Wellcome Trust Case Control Consortium <em>et al.</em> 2012). Credible set analysis is a Bayesian method of selecting an interval of SNPs that are likely to contain the causal SNPs; we used LD r<sup>2</sup> threshold = 0.8, posterior probability =0.99. The R script could be found on GitHub: <a href="https://github.com/hailianghuang/FM-summary/blob/master/getCredible.r">https://github.com/hailianghuang/FM-summary/blob/master/getCredible.r</a></p></div><div id="sec-14" class="subsection"><h3>Power analysis</h3><p id="p-22">To estimate the power of replication of a SNP from the discovery set in the replication set, we simulated GWAS with 50 varying effect sizes for the discovery SNP using the LMM model. We first fit the trait in a null model (<em>i.e.</em>, no genotype effect), and obtained estimates of model parameters including the intercept and the genetic variance component. Using these model parameters, we added the genotype effect to the random numbers generated from the null model to recreate a trait. For each simulated effect size, we scanned every simulated trait 2,500 times and examined the ratio of association tests whose test statistics surpassed the significance thresholds (both the genome-wide significance threshold for the cohort and the nominal p-value of 0.05).</p></div><div id="sec-15" class="subsection"><h3>Replication analysis between F<sub>34</sub> and F<sub>39-43</sub> GWAS studies</h3><p id="p-23">We modeled the replication between F<sub>34</sub> and F<sub>39-43</sub> GWAS studies using two random effects models (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Zou <em>et al.</em> 2019</a>). Both models take as input a set of z-scores for variants computed from an association study (“summary statistics”).</p><p id="p-24">The <strong>WC</strong> model accounts only for Winner’s Curse. We assume that there is a shared genetic effect (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-5.gif"/></noscript></span></span> that is responsible for the observed association signal in both studies. To model random noise contributing to Winner’s Curse, we model the summary statistics for each variant k from the discovery and replication studies as normally distributed random variables (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-7.gif"/></noscript></span></span>, respectively). We define the prior probability of the true genetic effect to be <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-8.gif"/></noscript></span></span>, where the variance in the true genetic effect is learned through a maximum likelihood procedure. We correct for the effect of Winner’s Curse in the discovery study by computing the conditional distribution of the replication summary statistic given the discovery summary statistic.</p><p id="p-25">The <strong>WC+C</strong> model accounts for Winner’s Curse and study-specific heterogeneity. In this model, we partition the total effect sizes observed into genetic effects (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-9.gif"/></noscript></span></span> and study-specific effects (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-11.gif"/></noscript></span></span>). We model the statistics for each variant k from the initial and discovery studies as normally distributed random variables (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-12.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-13.gif"/></noscript></span></span>, respectively). In addition to the prior on the genetic effect defined in the <strong>WC</strong> model, we define the prior probabilities of the study-specific effects to be <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-14.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/951/embed/mml-math-15.gif"/></noscript></span></span>, where the variance parameters are learned through a maximum likelihood procedure. We correct for the effect of Winner’s Curse in the discovery study and study-specific effects by computing the conditional distribution of the replication summary statistic given the discovery summary statistic.</p><p id="p-26">We applied each of these models once using F<sub>34</sub> as the discovery study and once using F<sub>39-43</sub> as the discovery study. We used the genome-wide significance thresholds in Table S5 to identify variants in each discovery study and used the results as input to the random effects models. We then used a Bonferroni corrected threshold (<em>P</em> = 0.05/M) for the replication study, where M is the number of genome-wide significant variants in the initial study. We computed the “empirical replication rate” as the proportion of variants passing the genome-wide significant threshold in the discovery study that also passed this Bonferroni corrected threshold in the replication study. Since the estimation of the model parameters requires at least two variants, we only applied this method to phenotypes with at least two genome-wide significant variants in the discovery study.</p><p id="p-27">To assess how well the <strong>WC</strong> and <strong>WC+C</strong> models explained the observed patterns of replication, we computed the predicted replication rates under each model. For each variant that passed the genome-wide significant threshold in the discovery study, we used the conditional distributions previously learned to compute the probability that the variant passed the Bonferroni corrected threshold in the replication study. For each phenotype, we computed the average of these predicted replication rates and compared this average to the empirical replication rates.</p></div><div id="sec-16" class="subsection"><h3>Genetic correlation and heritability estimates between F<sub>34</sub> and F<sub>39-43</sub> phenotypes</h3><p id="p-28">Locomotor activity, body weight, and coat color traits had been measured in both F<sub>34</sub> and F<sub>39-43</sub> populations. We calculated both SNP heritability and genetic correlations between F<sub>34</sub> and F<sub>39-43</sub> animals using GCTA-GREML analysis and GCTA bivariate GREML analysis (<a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">Yang <em>et al.</em> 2011</a>).</p></div><div id="sec-17" class="subsection"><h3>LocusZoom Plots</h3><p id="p-29">LocusZoom plots were generated using the standalone implementation of LocusZoom (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Pruim <em>et al.</em> 2010</a>), using LD scores calculated from PLINK v.1.9–ld option and mm10 gene annotation file downloaded from UCSC Genome Browser.</p></div><div id="sec-18" class="subsection"><h3>Data availability</h3><p id="p-30">All relevant data are within the paper and its Supporting Information files. Genotypes and phenotypes of F<sub>34</sub> (“LGSM AI G34 Palmer (Array)”: GN655; “LGSM AI G34 Palmer (GBS)”: GN656), F<sub>39-43</sub> (“LGSM AI G39-43 Palmer (GBS)”: GN657), and mega-analysis cohort (“LGSM AI G34 G39-43 Palmer (GBS)”: GN654) of AIL are uploaded to GeneNetwork2 (<a href="http://gn2.genenetwork.org/">http://gn2.genenetwork.org/</a>). Code used to perform the analyses is included in the supplementary materials on figshare: <a href="https://doi.org/10.25387/g3.11674221">https://doi.org/10.25387/g3.11674221</a>.</p></div></div><div class="section results" id="sec-19"><h2 class="">Results</h2><p id="p-31">We used 214 males and 214 females from generation F<sub>34</sub> (Aap:LG,SM-G34) and 305 males and 295 females from generations F<sub>39-43</sub>. For the F<sub>34</sub> AIL 79 traits were available from previous published and unpublished work; for the F<sub>39-43</sub> AIL 49 unpublished traits were available (Table S1). F<sub>34</sub> mice had been previously genotyped on a custom SNP array (<a id="xref-ref-12-11" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>; <a id="xref-ref-68-10" class="xref-bibr" href="#ref-68">Parker <em>et al.</em> 2014</a>). The average minor allele frequency (<strong>MAF</strong>) of those 4,593 array SNPs was 0.388 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). To obtain a denser set of SNP markers, we used GBS in F<sub>34</sub> and F<sub>39-43</sub> AIL mice. Since data on the F<sub>39-43</sub> AIL mice had been collected over the span of approximately two years, we carefully considered the possibility of sample contamination and sample mislabeling (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Toker <em>et al.</em> 2016</a>) and removed these samples (see Methods; Figure S1 and S2). The final SNP sets included 60,392 GBS-derived SNPs in 428 F<sub>34</sub> AIL mice, 59,790 GBS-derived SNPs in 600 F<sub>39-43</sub> AIL mice, and 58,461 GBS-derived SNPs that existed in both F<sub>34</sub> and F<sub>39-43</sub> AIL mice (Table S2). The MAF for the GBS SNPs was 0.382 in F<sub>34</sub>, 0.358 in F<sub>39-43</sub>, and 0.370 in F<sub>34</sub> and F<sub>39-43</sub> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). There were 66 SNPs called from our GBS data that were also present on the genotyping array. The genotype concordance rate for those 66 SNPs, which reflects the sum of errors from both sets of genotypes, was 95.4% (Figure S3). We found that LD decay rates using F<sub>34</sub> array, F<sub>34</sub> GBS, F<sub>39-43</sub> GBS, and F<sub>34</sub> and F<sub>39-43</sub> GBS genotypes were generally similar to one another, though levels of LD using the GBS genotypes appear to be slightly reduced in the later generations of AILs (Figure S4).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/951/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Minor allele frequency (MAF) distributions for F34 array, F34 GBS, F39-F43 GBS, and F34 and F39-F43 GBS SNP sets. The average MAF of those 4,593 array SNPs was 0.388; the average MAF of the 60,392 GBS-derived SNPs in 428 F34 AIL mice was 0.382; the average MAF of the 59,790 GBS-derived SNPs in 600 F39-43 AIL mice was 0.358; the average MAF of the 58,461 GBS-derived SNPs that existed in both F34 and F39-43 AIL mice was 0.370 (Table S2). MAF distributions are highly comparable between AIL generations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1535083459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Minor allele frequency (MAF) distributions for F&lt;sub&gt;34&lt;/sub&gt; array, F&lt;sub&gt;34&lt;/sub&gt; GBS, F&lt;sub&gt;39&lt;/sub&gt;-F&lt;sub&gt;43&lt;/sub&gt; GBS, and F&lt;sub&gt;34&lt;/sub&gt; and F&lt;sub&gt;39&lt;/sub&gt;-F&lt;sub&gt;43&lt;/sub&gt; GBS SNP sets. The average MAF of those 4,593 array SNPs was 0.388; the average MAF of the 60,392 GBS-derived SNPs in 428 F&lt;sub&gt;34&lt;/sub&gt; AIL mice was 0.382; the average MAF of the 59,790 GBS-derived SNPs in 600 F&lt;sub&gt;39-43&lt;/sub&gt; AIL mice was 0.358; the average MAF of the 58,461 GBS-derived SNPs that existed in both F&lt;sub&gt;34&lt;/sub&gt; and F&lt;sub&gt;39-43&lt;/sub&gt; AIL mice was 0.370 (Table S2). MAF distributions are highly comparable between AIL generations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/951/F1.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/3/951/F1.medium.gif" width="440" height="331"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/951/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/951/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495904" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-32" class="first-child">Minor allele frequency (MAF) distributions for F<sub>34</sub> array, F<sub>34</sub> GBS, F<sub>39</sub>-F<sub>43</sub> GBS, and F<sub>34</sub> and F<sub>39</sub>-F<sub>43</sub> GBS SNP sets. The average MAF of those 4,593 array SNPs was 0.388; the average MAF of the 60,392 GBS-derived SNPs in 428 F<sub>34</sub> AIL mice was 0.382; the average MAF of the 59,790 GBS-derived SNPs in 600 F<sub>39-43</sub> AIL mice was 0.358; the average MAF of the 58,461 GBS-derived SNPs that existed in both F<sub>34</sub> and F<sub>39-43</sub> AIL mice was 0.370 (Table S2). MAF distributions are highly comparable between AIL generations.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-20" class="subsection"><h3>GBS genotypes produced more significant associations than array genotypes in F<sub>34</sub></h3><p id="p-33">We used a linear mixed model (<strong>LMM</strong>) as implemented in GEMMA (<a id="xref-ref-99-4" class="xref-bibr" href="#ref-99">Zhou and Stephens 2012</a>) to perform GWAS. We used the leave-one-chromosome-out (<strong>LOCO</strong>) approach to address the problem of proximal contamination, as previously described (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Listgarten <em>et al.</em> 2012</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Cheng <em>et al.</em> 2013</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Yang <em>et al.</em> 2014</a>; <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Gonzales <em>et al.</em> 2017</a>). We performed GWAS using both the sparse array SNPs and the dense GBS SNPs to determine whether additional SNPs would produce more genome-wide significant associations. Autosomal and X chromosome SNPs were included in all GWAS. We obtained a significance threshold for each SNP set using MultiTrans (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">B. Han <em>et al.</em> 2009</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Joo <em>et al.</em> 2016</a>).To select independently associated loci (“lead loci”), we used an LD-based clumping method implemented in PLINK to group SNPs that passed the adjusted genome-wide significance thresholds over a large genomic region flanking the index SNP (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Purcell <em>et al.</em> 2007</a>). Applying the most stringent clumping parameters (<em>r<sup>2</sup></em> = 0.1 and sliding window size = 12,150kb, Table S4), we identified 109 significant lead loci in 49 out of 79 F<sub>34</sub> phenotypes using the GBS SNPs (Table S7). In contrast, we identified 83 significant lead loci in 45 out of 79 F<sub>34</sub> phenotypes using the sparse array SNPs (Table S6, Table S7). Among the loci identified in the F<sub>34</sub>, 36 were uniquely identified using the GBS genotypes, whereas 11 were uniquely identified using the array genotypes. These unique loci could be explained by the disparity of the marker density between the GBS and array genotypes. Some unique loci captured haplotype blocks that were not picked up in the other SNP set. Other unique loci were only slightly above the significance threshold in one SNP set where the corresponding loci in the other SNP set had sub-threshold significance (<em>i.e.</em>, p-value ∼10<sup>−5</sup> but below the significance threshold of the cohort; Table S7). Overall, GBS SNPs consistently yielded more significant lead loci compared to array SNPs regardless of the clumping parameter values (Table S4), indicating that a dense marker panel was able to detect more association signals compared to a sparse marker panel.</p><p id="p-34">To determine the boundaries of each locus, we performed a Bayesian-framework credible set analysis, which estimated a posterior probability for association at each SNP (<em>r<sup>2</sup></em> threshold = 0.8, posterior probability threshold = 0.99; (Wellcome Trust Case Control Consortium <em>et al.</em> 2012)). The physical positions of the SNPs in the credible set were used to determine the boundaries of each locus. As expected, the greater density of the GBS genotypes allowed us to better define each interval. For instance, the lead locus at chr17:27130383 was associated with distance traveled in periphery in the open field test in F<sub>34</sub> AILs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). However, no SNPs were genotyped between 26.7 and 28.7 Mb in the array SNPs, which makes the size of this LD block ambiguous. In contrast, the LocusZoom plot portraying GBS SNPs in the same region shows that SNPs in high LD with chr17:27130383 are between 27 Mb and 28.3 Mb. The more accurate definition of the implicated intervals allowed us to better refine the list of the coding genes and non-coding variants associated with the phenotype (Table S6).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/951/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Significant loci on chromosome 17 for open field, distance traveled in periphery in F34 AIL. As exemplified in this pair of LocusZoom plots, GBS SNPs defined the boundaries of the loci much more precisely than array SNPs. GBS SNPs that are in high LD (r2 &gt; 0.8, red dots) with lead SNP chr17:27130383 resides between 27 &#x223C;28.3 Mb. In contrast, too few SNPs are present in the array plot to draw any definitive conclusion about the boundaries or LD pattern in this region. Purple track shows the credible set interval. LocusZoom plots for all loci identified in this paper are in Figure S8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1535083459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Significant loci on chromosome 17 for open field, distance traveled in periphery in F&lt;sub&gt;34&lt;/sub&gt; AIL. As exemplified in this pair of LocusZoom plots, GBS SNPs defined the boundaries of the loci much more precisely than array SNPs. GBS SNPs that are in high LD (r&lt;sup&gt;2&lt;/sup&gt; &gt; 0.8, red dots) with lead SNP chr17:27130383 resides between 27 &#x223C;28.3 Mb. In contrast, too few SNPs are present in the array plot to draw any definitive conclusion about the boundaries or LD pattern in this region. Purple track shows the credible set interval. LocusZoom plots for all loci identified in this paper are in Figure S8.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/951/F2.medium.gif" width="440" height="173"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/3/951/F2.medium.gif" width="440" height="173"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/951/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/951/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495991" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Significant loci on chromosome 17 for open field, distance traveled in periphery in F<sub>34</sub> AIL. As exemplified in this pair of LocusZoom plots, GBS SNPs defined the boundaries of the loci much more precisely than array SNPs. GBS SNPs that are in high LD (r<sup>2</sup> &gt; 0.8, red dots) with lead SNP chr17:27130383 resides between 27 ∼28.3 Mb. In contrast, too few SNPs are present in the array plot to draw any definitive conclusion about the boundaries or LD pattern in this region. Purple track shows the credible set interval. LocusZoom plots for all loci identified in this paper are in Figure S8.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">In our prior studies using the sparse marker set, we did not attempt to increase the number of available markers by using imputation. Therefore, we examined whether the disparity between the numbers of loci identified by the two SNP sets could be resolved by imputation, which should increase the number of markers available for GWAS. We used LG/J and SM/J whole genome sequencing data as reference panels (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Nikolskiy <em>et al.</em> 2015</a>) and performed imputation on array and GBS SNPs using Beagle v4.1 (<a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Browning and Browning 2007</a>). After QC filtering, we obtained 4.3M SNPs imputed from the array SNPs and 4.1M SNPs imputed from the GBS SNPs. More imputed GBS SNPs were filtered out because GBS SNPs were called from genotype probabilities, thus introducing uncertainty in imputed SNPs. We found that imputed array genotypes and imputed GBS genotypes did not meaningfully increase the number of loci discovered, presumably because the utility of imputation is inherently limited in a two-strain cross.</p><p id="p-37">Under a polygenic model where a large number of additive common variants contribute to a complex trait, heritability estimates could be higher when more SNPs are considered (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Yang <em>et al.</em> 2017</a>). Given that there were more GBS SNPs than array SNPs, we used autosomal SNPs to examine whether GBS SNPs would generate higher SNP heritability estimates compared to the sparse array SNPs. Heritability estimates were similar for the two SNP sets, with the exception of agouti coat color, which showed marginally greater heritability for the GBS SNPs (Figure S5; Table S8). Our results show that while the denser GBS SNP set was able to identify more genome-wide significant loci, greater SNP density did not improve the polygenic signal.</p></div><div id="sec-21" class="subsection"><h3>Partial replication of loci identified in F<sub>34</sub> or F<sub>39-43</sub> and mega-analysis</h3><p id="p-38">We identified 25 genome-wide significant loci for 21 phenotypes in the F<sub>39-43</sub> cohort (Table S9). A subset of those traits: coat color, body weight, and locomotor activity, were also phenotyped in the F<sub>34</sub> AILs. To assess replication, we determined whether the loci that were significant in one cohort (either F<sub>34</sub> or F<sub>39-43</sub>) would also be significant in the other. We termed the cohort in which a locus was initially discovered as its “discovery set” and the cohort we attempted replication in as the “replication set” (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Coat color phenotypes (both albino and agouti) are Mendelian traits and thus served as positive controls. All coat color and body weight loci were replicated. The three body weight loci identified in the F<sub>34</sub> were replicated at nominal levels of significance (<em>P</em> &lt; 0.05) in F<sub>39-43</sub>; similarly, one body weight locus identified in F<sub>39-43</sub> was replicated in F<sub>34</sub> (<em>P</em> &lt; 0.05). However, none of the locomotor activity loci were replicated in the reciprocal (replication) cohorts.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495898/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495898/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Replication of significant SNPs between F<sub>34</sub> and F<sub>39-43</sub> AIL association analyses. “Discovery set” indicates the AIL generation that significant SNPs were identified. “Replication set” shows the association p-value, β estimates, etc. of the “discovery set” significant SNPs in the replication AIL generation. SNPs that replicated (<strong>p</strong> &lt; 0.05, same sign for the beta) between F<sub>34</sub> and F<sub>39-43</sub> are in bold italics, SNPs that replicated at the genome-wide threshold (see Table S5) are bold, italic and underlined. Genetic correlations (rG) for phenotypes measured in both F<sub>34</sub> and F<sub>39-43</sub> are listed (see also Table S11)</span><div class="sb-div caption-clear"></div></div></div><p id="p-39">We found that using a broader definition of an association region rather than a single SNP did not improve replication between the F<sub>34</sub> cohort and the F<sub>39-43</sub> cohorts. Confidence intervals (<em>e.g.</em>, (Baud <em>et al.</em> 2013; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Nicod <em>et al.</em> 2016</a>)) and the LOD support interval (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Conneally <em>et al.</em> 1985</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Lander and Botstein 1989</a>) have been used to define a QTL. LOD support interval is very sensitive to the density of the SNPs where sparse markers would produce misleadingly large support intervals. In contrast, the credible set interval is an estimate of the posterior probability for association at markers neighboring the discovery SNP, and thus defines the size of the association region. As a result, we extended the replication comparison from the discovery SNP position to the credible set interval. We found that in the replication cohort, the p-value at the discovery SNP and that at the top SNP within the credible set interval (defined by the discovery QTL) were generally similar (Table S10). The replication of the locus chr14.79312393 (discovered in the F<sub>34</sub> cohort) in the F<sub>39-43</sub> cohort was more successful using the discovery QTL region defined by the credible set interval; the p-value at the top SNP within the credible set interval was noticeably more significant (chr14.82586326; p-value = 1.48×10<sup>−6</sup>) than the p-value at the discovery SNP (chr14.79312393; p-value = 0.0237; Table S10). Our results suggest that for the most part, the discovery SNP accurately represented the association strength of the loci, presumably because of its strong linkage with the neighboring SNPs. In our case, defining a QTL region by the credible set interval did not increase the count of replicated sites between the two cohorts.</p><p id="p-40">We then considered the more liberal “sign test”, a statistical method to test for consistent differences between pairs of observations, to determine whether the directions of the effect (beta) of the coat color, body weight and activity loci were in the same direction between the discovery and replication cohorts. Specifically, we compared whether the sign (direction) of the beta estimates are consistently above or below zero. We found that 11 of 12 comparisons passed this much less stringent test of replication. The one locus (at chr15.67627183) that did not pass the sign test was the locomotor locus “discovered” in F<sub>39-43</sub> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-41">In light of the failure to replicate the locomotor activity findings, we conducted a series of 2,500 simulations per trait to estimate the expected power of our replication cohorts. For each phenotype we used the kinship relatedness matrix and variance components estimated from the replication set. For the coat color traits, we found that we had 100% power to replicate the association at either genome-wide significant levels or the more liberal p &lt; 0.05 threshold (Figure S6). For body weight and locomotor activity, power to replicate at a genome-wide significance threshold ranged from 20 to 85%, whereas power to replicate at the p &lt; 0.05 threshold was between 80% and 100% (Figure S6). These power estimates were inconsistent with our empirical observations for the locomotor activity traits, none of which replicated at even the p &lt; 0.05 threshold, where we should have had almost 100% power (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>; Figure S6). However, our power simulations did not account for Winner’s Curse (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Zöllner and Pritchard 2007</a>) or study-specific heterogeneity (<a id="xref-ref-101-2" class="xref-bibr" href="#ref-101">Zou <em>et al.</em> 2019</a>).</p><p id="p-42">To determine whether these factors could explain the lower than expected rate of replication, we applied a statistical framework that jointly models Winner’s Curse and study-specific heterogeneity in two GWAS studies of the same phenotype (<a id="xref-ref-101-3" class="xref-bibr" href="#ref-101">Zou <em>et al.</em> 2019</a>). This framework proposes two random effects models. The first model (<strong>WC</strong>) only accounts for Winner’s Curse, while the second model accounts for both Winner’s Curse and study-specific heterogeneity due to confounding (<strong>WC+C</strong>). In this context, we define confounding as any biological or technical effect present in one study but not the other. We applied each of these models once using F<sub>34</sub> as the discovery study and once using F<sub>39-43</sub> as the discovery study. The models can be used to assess how well Winner’s Curse explains the observed levels of replication. For example, when F<sub>34</sub> is used as the replication study for the albino coat color phenotype, the expected value of the replication summary statistics after accounting for winner’s curse is the same as the expected value after accounting for Winner’s Curse and confounding (Figure S7). While the 95% confidence intervals for the <strong>WC+C</strong> model are larger than the <strong>WC</strong> model, both models seem to explain the observed data well. However, when F<sub>34</sub> is used as the discovery study for the locomotor activity on day 1 or body weight, the <strong>WC+C</strong> model explains the data better than the <strong>WC</strong> model.</p><p id="p-43">In order to quantitatively assess how well each of these models explain the observed patterns of replication, we computed the predicted replication rates under each model (Methods) and compared these with the empirical replication rates. In this analysis, we defined the empirical replication rate to be the proportion of variants passing the genome-wide significance threshold in the discovery study that also pass the Bonferroni corrected threshold in the replication study. We used this definition of replication for this analysis instead of replication of lead SNPs to allow for a larger number of variants to be included in the model fitting process. For all phenotypes tested, the <strong>WC</strong> model predicts that all the variants passing the genome-wide significance threshold in the discovery study should pass the Bonferroni corrected threshold in the replication study, which is dramatically different from the observed replication of body weight and locomotor activity on day 1 and 2 phenotypes (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). While the replication in the agouti coat color phenotype is not well predicted by the <strong>WC+C</strong> model, this may be due to the fact that the agouti phenotype is a dominant trait, while our model assumes additive allele effects. These results suggest that the sample sizes are sufficiently large that Winner’s Curse cannot account for the lack of replication. However, in these cases, the <strong>WC+C</strong> model has predicted replication rates that are much closer to the true (observed) values, indicating that the lack of replication in these phenotypes is more likely to be due to study-specific heterogeneity that is potentially caused by confounding.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495895/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495895/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Predicted replication rates. We applied the replication analysis to phenotypes with at least two genome-wide significant variants in the discovery study. These phenotypes include body weight, albino coat color, agouti coat color, locomotor test day 1, and locomotor test day 2. We computed the true replication rate as the fraction of variants that were genome-wide significant in the discovery study that also passed the Bonferroni significance threshold in the replication study (“Empirical replication rate”). The model accounting for Winner’s Curse and confounding (“Predicted replication rate WC+C”) explains the true replication rate more accurately than the model accounting for only Winner’s Curse (“Predicted replication rate WC”)</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">We evaluated whether or not the traits showed genetic correlations across the two cohorts. High genetic correlations would indicate a high degree of additive genetic effect that is shared between the two cohorts, and the low genetic correlations would indicate limited potential for replication. We used all autosomal SNPs to calculate genetic correlations between the F<sub>34</sub> and F<sub>39-43</sub> generations for body weight, coat color, and locomotor activity phenotypes (Table S11), using GCTA-GREML (<a id="xref-ref-95-3" class="xref-bibr" href="#ref-95">Yang <em>et al.</em> 2011</a>). Albino and agouti coat color, body weight and locomotor activity on days 1 and 2 were highly genetically correlated (r<sub>G</sub> &gt;0.7; Table S11). In contrast, locomotor activity on day 3 showed a significant but weaker genetic correlation (r<sub>G</sub> = 0.577), perhaps reflecting variability in the quality of the methamphetamine injection, which were only given on day 3. Overall, these results suggest that genetic influences on these traits were largely similar in the two cohorts; however, the genetic correlations were less than 1, suggesting an additional barrier to replication that was not accounted for in our power simulations.</p><p id="p-45">We also calculated the SNP heritability for all traits using GCTA. SNP heritability was consistently lower in the F<sub>39-43</sub> cohort compared to the F<sub>34</sub> cohort, including the Mendelian traits of coat color. The ± 1 × standard error intervals of the F<sub>34</sub> and F<sub>39-43</sub> SNP heritability estimates for the coat color trait albino overlapped. This observation indicates that SNP heritability for albino in the two cohorts is comparable. In contrast, the ± 1 × standard error intervals of the F<sub>34</sub> and F<sub>39-43</sub> SNP heritability estimates for the coat color trait agouti did not overlap. We could not explain the differential SNP heritability for the binary trait agouti in the two cohorts. The lower SNP heritability in F<sub>39-43</sub> for the rest of the quantitative traits could be a result of increased experimental variance (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>; Table S12; (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Falconer 1960</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Lynch and Walsh 1996</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Mhyre <em>et al.</em> 2005</a>; <a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Zöllner and Pritchard 2007</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Visscher <em>et al.</em> 2008</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Zaitlen and Kraft 2012</a>)).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/951/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SNP-heritability estimates in F34 and F39-43 AILs. Square dots represent the SNP heritability estimated by the GCTA-GREML analysis (Yang et al. 2011). The whiskers flanking the square dots show the &#xB1; 1 &#xD7; standard error of the heritability estimate. All heritability estimates are highly significant (p &lt; 1.0&#xD7;10&#x2212;05; see Table S12)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1535083459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;SNP-heritability estimates in F&lt;sub&gt;34&lt;/sub&gt; and F&lt;sub&gt;39-43&lt;/sub&gt; AILs. Square dots represent the SNP heritability estimated by the GCTA-GREML analysis (Yang &lt;em&gt;et al.&lt;/em&gt; 2011). The whiskers flanking the square dots show the &#xB1; 1 &#xD7; standard error of the heritability estimate. All heritability estimates are highly significant (p &lt; 1.0&#xD7;10&lt;sup&gt;&#x2212;05&lt;/sup&gt;; see Table S12).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/951/F3.medium.gif" width="375" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/3/951/F3.medium.gif" width="375" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/951/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/951/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/496004" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-46" class="first-child">SNP-heritability estimates in F<sub>34</sub> and F<sub>39-43</sub> AILs. Square dots represent the SNP heritability estimated by the GCTA-GREML analysis (<a id="xref-ref-95-4" class="xref-bibr" href="#ref-95">Yang <em>et al.</em> 2011</a>). The whiskers flanking the square dots show the ± 1 × standard error of the heritability estimate. All heritability estimates are highly significant (p &lt; 1.0×10<sup>−05</sup>; see Table S12).</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">Due to the relatively high genetic correlations (Table S11), we suspected that a mega-analysis using the combined sample set would allow for the identification of additional loci; indeed, mega-analysis identified four novel genome-wide significant associations (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>; Table S13). The significance level of five out of six loci identified by the mega-analysis was greater than that in either individual cohort. For instance, the p-values obtained by mega-analysis for chr14:82672838 (p-value = 7.93×10<sup>−9</sup>) for body weight were lower than the corresponding p-values for the same loci for F<sub>34</sub> (chr14:79312393, p-value = 7.53×10<sup>−6</sup>) and F<sub>39-43</sub> (chr14.82586326, p-value = 2.63×10<sup>−6</sup>; Table S13; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/951/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Manhattan plots comparing F34 GBS, F39-43 GBS, and mega-analysis on locomotor day 1 test using 57,170 shared SNPs in all AIL generations. We performed mega-analysis of F34 and F39-43 animals (n = 1,028) for body weight, coat color, and locomotor activity, the set of traits that were measured in the same way in both cohorts." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1535083459" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Manhattan plots comparing F&lt;sub&gt;34&lt;/sub&gt; GBS, F&lt;sub&gt;39-43&lt;/sub&gt; GBS, and mega-analysis on locomotor day 1 test using 57,170 shared SNPs in all AIL generations. We performed mega-analysis of F&lt;sub&gt;34&lt;/sub&gt; and F&lt;sub&gt;39-43&lt;/sub&gt; animals (n = 1,028) for body weight, coat color, and locomotor activity, the set of traits that were measured in the same way in both cohorts.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/951/F4.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/3/951/F4.medium.gif" width="440" height="366"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/951/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/951/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495955" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-48" class="first-child">Manhattan plots comparing F<sub>34</sub> GBS, F<sub>39-43</sub> GBS, and mega-analysis on locomotor day 1 test using 57,170 shared SNPs in all AIL generations. We performed mega-analysis of F<sub>34</sub> and F<sub>39-43</sub> animals (n = 1,028) for body weight, coat color, and locomotor activity, the set of traits that were measured in the same way in both cohorts.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-49">We used F<sub>34</sub> and F<sub>39-43</sub> generations of a LG/J x SM/J AIL to perform GWAS, SNP heritability estimates, genetic correlations, replication and mega-analysis. We had previously performed several GWAS using a sparse marker set in the F<sub>34</sub> cohort. In this study we used a denser set of SNPs, obtained using GBS, to reanalyze the F<sub>34</sub> cohort. We found 109 significant loci, 36 of which had not been identified in our prior studies using the sparse marker set. We used a new, previously unpublished F<sub>39-43</sub> cohort for GWAS and showed that genetic correlations were high for the subset of traits that were measured in both cohorts. Despite this, we found that many loci were not replicated between cohorts, even when we used a relatively liberal definition of replication (p &lt; 0.05). The failure to replicate some of our findings was not predicted by our power simulations. Therefore, we performed an analysis to determine whether Winner’s Curse and study-specific heterogeneity could account for the lower than expected replication rate. Winner’s Curse alone could not explain the failure to replicate. However, modeling both Winner’s Curse and study-specific heterogeneity better explained the observed replication rate. Finally, mega-analysis of the two cohorts allowed us to discover four additional loci. Taken together, our results provide a set of refined regions of association for numerous physiological and behavioral traits in multiple generations of AILs. These loci could serve as benchmarks for future GWAS results in intercross mouse lines. More broadly, this study illustrates the difficulty of replication even when using a highly controlled model system.</p><p id="p-50">Previous publications from our lab used a sparse set of array genotypes for GWAS of various behavioral and physiological traits in 688 F<sub>34</sub> AILs (<a id="xref-ref-12-12" class="xref-bibr" href="#ref-12">Cheng <em>et al.</em> 2010</a>; <a id="xref-ref-54-6" class="xref-bibr" href="#ref-54">Lionikas <em>et al.</em> 2010</a>; <a id="xref-ref-79-5" class="xref-bibr" href="#ref-79">Samocha <em>et al.</em> 2010</a>; <a id="xref-ref-69-7" class="xref-bibr" href="#ref-69">Parker <em>et al.</em> 2011</a>, <a id="xref-ref-68-11" class="xref-bibr" href="#ref-68">2014</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Carroll <em>et al.</em> 2017</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Hernandez Cordero <em>et al.</em> 2018</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Gonzales <em>et al.</em> 2018</a>). In this study we obtained a much denser marker set for 428 of the initial 688 AIL mice using GBS. The denser genotypes allowed us to identify most of the loci obtained using the sparse set, as well as many additional loci. For instance, using the sparse markers we identified a significant locus on chromosome 8 for locomotor day 2 activity that contained only one gene: <em>Csmd1</em> (CUB and sushi multiple domains 1). Gonzales <em>et al.</em> (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Gonzales <em>et al.</em> 2018</a>) replicated this finding in F<sub>50-56</sub> AILs and identified a <em>cis</em>-eQTL mapped to the same region. <em>Csmd1</em> mutant mice showed increased locomotor activity compared to wild-type and heterozygous mice, indicating that <em>Csmd1</em> is likely a causal gene for locomotor and related traits (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Gonzales <em>et al.</em> 2018</a>). We replicated this locus in the analysis of the F<sub>34</sub> cohort that used the denser marker set (Figure S8). We also replicated a locus on chromosome 17 for distance traveled in the periphery in the open field test (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>; (<a id="xref-ref-68-12" class="xref-bibr" href="#ref-68">Parker <em>et al.</em> 2014</a>)), three loci on chromosomes 4, 6, and 14 for body weight (Figure S8; (<a id="xref-ref-69-8" class="xref-bibr" href="#ref-69">Parker <em>et al.</em> 2011</a>)), one locus on chromosome 7 for mean corpuscular hemoglobin concentrations (MCHC, complete blood count; Figure S8; (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Bartnikas <em>et al.</em> 2012</a>)), and numerous loci on chromosome 4, 6, 7, 8, and 11 for muscle weights (Figure S8; (<a id="xref-ref-54-7" class="xref-bibr" href="#ref-54">Lionikas <em>et al.</em> 2010</a>)). We noticed that even using original sparse markers, some previously published loci were not replicated in the current GWAS. The most likely explanation is that we had only 428 of the 688 mice used in the previous publications.</p><p id="p-51">QTL mapping studies have traditionally used a 1.0∼2.0 LOD support interval to approximate the size of the association region (see (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cervino <em>et al.</em> 2005</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Logan <em>et al.</em> 2013</a>)). The LOD support interval, proposed by Conneally <em>et al.</em> (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Conneally <em>et al.</em> 1985</a>) and Lander and Botstein (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Lander and Botstein 1989</a>), is a simple confidence interval method involving converting the p-value of the peak locus into a LOD score, subtracting “drop size” from the peak locus LOD score, and finding the two physical positions to the left and to the right of the peak locus location that correspond to the subtracted LOD score. Although Mangin <em>et al.</em> (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Mangin <em>et al.</em> 1994</a>) showed via simulation that the boundaries of LOD support intervals depend on effect size, others observed that a 1.0 ∼2.0 LOD support interval accurately captures ∼95% coverage of the true location of the loci when using a dense set of markers (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Lander and Botstein 1989</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Dupuis and Siegmund 1999</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Manichaikul <em>et al.</em> 2006</a>). In the present study, we considered using LOD support intervals but found that the sparse array SNPs produced misleadingly large support intervals. Various methods have been proposed for calculating confidence intervals in analogous situations (<em>e.g.</em>, (Baud <em>et al.</em> 2013; <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Nicod <em>et al.</em> 2016</a>)). We performed credible set analysis and compared LocusZoom plots of the same locus region between array SNPs and the GBS SNPs (Figure S8; (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Pruim <em>et al.</em> 2010</a>)). For example, the benefit of the denser SNP coverage is easily observed in the locus on chromosome 7 (array lead SNP chr7:44560350; GBS lead SNP chr7:44630890) for the complete blood count trait “retic parameters cell hemoglobin concentration mean, repeat”; denser SNPs delineate the start and the end of an association block much more clearly. Thus, there are advantages of dense SNP sets that go beyond the ability to discover additional loci.</p><p id="p-52">LD in the LG/J x SM/J AIL mice is more extensive than in the Diversity Outbred mice and Carworth Farms White mice (<a id="xref-ref-71-4" class="xref-bibr" href="#ref-71">Parker <em>et al.</em> 2016</a>). Some of the loci that we identified are relatively broad, making it difficult to infer which genes are responsible for the association. We focused on loci that contained five or fewer genes (Table S6). We highlight a few genes that are supported by the existing literature for their role in the corresponding traits. The lead SNP at chr1:77255381 is associated with tibia length in F<sub>34</sub> AILs (Table S6; Figure S8). One gene at this locus, <em>EphA4</em>, codes for a receptor for membrane-bound ephrins. EPHA4 plays an important role in the activation of the tyrosine kinase JAK2 and the signal transducer and transcriptional activator STAT5B in muscle, promoting the synthesis of insulin-like growth factor 1 (IGF-1) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Lai <em>et al.</em> 2004</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hyun 2013</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Sawada <em>et al.</em> 2017</a>). Mice with mutated <em>EphA4</em> shows significant defect in body growth (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Hyun 2013</a>). Curiously, another gene at this locus, <em>Pax3</em>, has been shown as a transcription factor expressed in resident muscle progenitor cells and is essential for the formation of skeletal muscle in mice (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Relaix <em>et al.</em> 2006</a>). It is possible that both <em>EphA4</em> and <em>Pax3</em> are associated with the trait tibia length because they are both involved in organismal growth. Another region of interest is the locus at chr4:66866758, which is associated with body weight (Table S6; Table S13). The lead SNP is immediately upstream of <em>Tlr4</em>, Toll-like receptor 4, which recognizes Gram-negative bacteria by its cell wall component, lipopolysaccharide (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hoshino <em>et al.</em> 1999</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Takeuchi <em>et al.</em> 1999</a>). TLR4 responds to the high circulating level of fatty acids and induces inflammatory signaling, which leads to insulin resistance (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Shi <em>et al.</em> 2006</a>). Kim <em>et al.</em> showed TLR-4-deficient mice were protected from the increase in proinflammatory cytokine level and gained less weight than wild-type mice when fed on high fat diet (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Kim <em>et al.</em> 2012</a>). The association between <em>Tlr4</em> and body weight in the AILs corroborates these findings.</p><p id="p-53">We considered both the F<sub>34</sub> and the F<sub>39-43</sub> as both “discovery” and “replication” cohorts. Significant loci for coat color, which are monogenic and served as positive controls, were replicated, between the two cohorts, as expected. One locus for body weight was replicated (p &lt; 0.05) between F<sub>34</sub> and F<sub>39-43.</sub> However, the loci for locomotor activity were not replicated. Power analyses predicted a much higher rate of replication, which led us to conduct additional analyses to better understand the lower than expected rate of replication.</p><p id="p-54">First, we used a newly introduced method to determine whether Winner’s Curse (<a id="xref-ref-100-3" class="xref-bibr" href="#ref-100">Zöllner and Pritchard 2007</a>)) which has also been termed the Beavis Effect (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beavis <em>et al.</em> 1991</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">1994</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Xu 2003</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">King and Long 2017</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Keele <em>et al.</em> 2019</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Paterson 2019</a>) could account for the lower than expected rate of replication. Beavis’ original report described a lack of replication of QTL for agronomic traits between small populations of maize (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Beavis <em>et al.</em> 1991</a>). Using progeny sizes ranging from 100 to 1000, Beavis simulated interval mapping to evaluate the accuracy of the estimates of phenotypic variance explained at the statistically significant QTL (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Beavis <em>et al.</em> 1994</a>; <a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">Xu 2003</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Paterson 2019</a>). Simulations showed that progeny sizes greatly influenced the estimates of phenotypic variance explained; smaller progeny sizes (n = 100) generated highly overestimated estimates of phenotypic variances, whereas larger progeny sizes (n = 1000) generated estimates of phenotypic variances similar to the actual value (<a id="xref-ref-93-3" class="xref-bibr" href="#ref-93">Xu 2003</a>; <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Paterson 2019</a>). King and Long (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">King and Long 2017</a>) further examined the Beavis Effect in the next-generation mapping populations in <em>Drosophila melanogaster</em>. The authors found that sample size was the major determinant for the overestimation of phenotypic variance explained at the significant QTL in both the GWAS-based Drosophila Genetic Reference Panel (DGRP) and the multi-parental Drosophila Synthetic Population Resource (DSPR). When sample size remained constant and the true phenotypic variance explained at the significant QTL was small, the estimation bias was more pronounced. In contrast, estimates for the phenotypic variance explained at all simulated QTL, significant or not, were generally centered at the true values. In an analogous study of power and replication in Collaborative Cross mice, Keele <em>et al.</em> (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Keele <em>et al.</em> 2019</a>) found that the Beavis Effect was most striking when the number of strains and true effect size of the QTL were small. This estimation bias indicates that mapping statistically significant QTL across experiments, populations, and panels can be problematic (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Macdonald and Long 2004</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Gruber <em>et al.</em> 2007</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Najarro <em>et al.</em> 2015</a>). The analyses we performed indicated that Winner’s Curse alone could not explain the lack of replication, but a model that also included study-specific heterogeneity could.</p><p id="p-55">Our analysis does cannot explain the source of the study-specific heterogeneity. Possible sources of confounding could include maternal effects, which could differentiate the F<sub>34</sub> cohort and the F<sub>39-43</sub> cohort because F<sub>33</sub> animals were transported to the University of Chicago from Washington University in St. Louis. In contrast, the breeder animals of the F<sub>39-43</sub> cohort have already acclimated to the environment for multiple generations. Another possible source of confounding is that the phenotyping of the F<sub>39-43</sub> occurred over five generations (more than a year) during which time numerous environmental factors may have changed (<em>e.g.</em>, several technicians performed the data collection). Such factors are known to be an important potential source of confounding; (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Falconer 1960</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Lynch and Walsh 1996</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Crabbe <em>et al.</em> 1999</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Mhyre <em>et al.</em> 2005</a>; <a id="xref-ref-89-2" class="xref-bibr" href="#ref-89">Visscher <em>et al.</em> 2008</a>; <a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">Zaitlen and Kraft 2012</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Sorge <em>et al.</em> 2014</a>). Our analyses did not correct for the fact that six phenotypes were examined, thus somewhat increasing the chances that at least one of our significant associations could have been a false positive that would not be expected to replicate.</p><p id="p-56">Interestingly, we found that the genetic correlations between the discovery and replication samples were relatively high for all traits; however, some traits replicated well and others replicated poorly. Our subsequent analysis showed that study-specific heterogeneity was low for the coat color traits, but higher for the body weight and locomotor traits. This makes an important point, namely that a strong genetic correlation can exists in the presence or absence of study-specific heterogeneity. Finally, it was notable that replication (at p &lt; 0.05) was relatively successful for body weight, despite the significant evidence of study-specific heterogeneity and low predicted replication (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Power analyses predicted that the body weight loci should replicate at the genome-wide significance threshold, whereas we only observed replication when at the less stringent p &lt; 0.05 level (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). The lack of replication at the genome-wide significance threshold for the body weight phenotype was likely due to study-specific heterogeneity due to confounding that was not accounted for in the power analyses. In <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, “predicted replication” refers to replication using a Bonferroni significance threshold that accounts for the number of significant SNPs in the discovery study. The low predicted replication rate under the <strong>WC+C</strong> model for the body weight phenotype is consistent with the low replication (genome-wide) reported in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>. Thus, both body weight and locomotor traits were strongly impacted by study specific confounding; however, nominal replication was still possible for body weight but not for the locomotor traits.</p><p id="p-57">Finally, we performed a mega-analysis using F<sub>34</sub> and F<sub>39-43</sub> AIL mice. The combined dataset allowed us to identify four novel genome-wide significant associations that were not detected in either the F<sub>34</sub> or the F<sub>39-43</sub> cohorts presumably because of the increased sample size in the mega-analysis (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Visscher <em>et al.</em> 2017</a>). As is true for all GWAS, the loci identified in the mega-analysis could be false positives.</p><p id="p-58">In addition to performing many GWAS and related analyses that led to the identification of dozens of novel loci for locomotor activity, open field test, fear conditioning, light dark test for anxiety, complete blood count, iron content in liver and spleen, and muscle weight, our study also tested our expectations about replication of GWAS findings. We did not obtain the expected rate of replication. We used a method that can distinguish between Winner’s Curse and study-specific heterogeneity to show that the lower than expected rate of replication was due to study-specific heterogeneity. This indicates that study-specific heterogeneity can have a major impact of replication even when in a model system when a genetically identical population is tested under conditions that are designed to be as similar as possible.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-59">AAP and XZ designed the study, oversaw data collection and analysis, and co-wrote the manuscript. XZ imputed genotypes, performed SNP- and subject-level QC, and conducted GWAS in F<sub>34</sub> and F<sub>39-43</sub> AILs under supervision of AAP. CS prepared GBS libraries for sequencing, as well as organizing portions of the F<sub>39-43</sub> phenotypes. NMG de-multiplexed GBS sequencing results and performed alignment and variant calling. JZ performed statistical analyses to model replication between F<sub>34</sub> and F<sub>39-43</sub> cohorts. RC helped with kinship relatedness matrix calculated from AIL pedigree and with power analysis. AC provided technical support for running programs and scripts. GS oversaw breeding and phenotyping of the F<sub>39-43</sub>. We would like to recognize Jackie Lim and Kaitlin Samocha for collecting F<sub>34</sub> AIL phenotype data and Ryan Walters for collecting F<sub>39-43</sub> AIL phenotype data. We wish to acknowledge Alex Gileta for input on a draft of this manuscript. All procedures were approved by the Institutional Animal Care and Use Committee (IACUC protocol: S15226) Euthanasia was accomplished using CO<sub>2</sub> asphyxiation followed by cervical dislocation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-60">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11674221">https://doi.org/10.25387/g3.11674221</a>.</p></li><li class="fn" id="fn-2"><p id="p-61"><em>Communicating editor: S. Macdonald</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-09-23"><span class="received-label">Received </span>September 23, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-10-17"><span class="accepted-label">Accepted </span>October 17, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Zhou <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.1" data-doi="10.1534/genetics.108.089912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abney</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identity-by-descent estimation and mapping of qualitative traits in large, complex pedigrees.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1577</span>–<span class="cit-lpage">1590</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.089912</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089912%26rft_id%253Dinfo%253Apmid%252F18622032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzE1NzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.2" data-doi="10.1007/s00335-012-9393-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartnikas</span>,  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Parker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Campagna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Lim</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">QTLs for murine red blood cell parameters in LG/J and SM/J F2 and advanced intercross lines.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">366</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-012-9393-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DBartnikas%26rft.auinit1%253DT.%2BB.%26rft.volume%253D23%26rft.issue%253D5-6%26rft.spage%253D356%26rft.epage%253D366%26rft.atitle%253DQTLs%2Bfor%2Bmurine%2Bred%2Bblood%2Bcell%2Bparameters%2Bin%2BLG%252FJ%2Band%2BSM%252FJ%2BF%25282%2529%2Band%2Badvanced%2Bintercross%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9393-3%26rft_id%253Dinfo%253Apmid%252F22322356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9393-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22322356&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.3" data-doi="10.1038/sdata.2014.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baud</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Guryev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Hummel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Johannesson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hermsen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomes and phenomes of a population of outbred rats and its progenitors.</span> <abbr class="cit-jnl-abbrev">Sci. Data</abbr> <span class="cit-vol">1</span>: <span class="cit-elocation-id">140011</span>. Erratum: 140016.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/sdata.2014.11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BData%26rft.volume%253D1%26rft.spage%253D140011%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsdata.2014.11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sdata.2014.11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.4" data-doi="10.1007/BF00215714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>,  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Grant</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Albertsen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fincher</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Quantitative trait loci for plant height in four maize populations and their associations with qualitative genetic loci.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00226242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00215714%26rft_id%253Dinfo%253Apmid%252F24202349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00215714&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24202349&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.5" data-doi="10.2135/cropsci1994.0011183X003400040010x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>,  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. S.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Grant</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fincher</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Identification of Quantitative Trait Loci Using a Small Sample of Topcrossed and F4 Progeny from Maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">882</span>–<span class="cit-lpage">896</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci1994.0011183X003400040010x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBeavis%26rft.auinit1%253DW.%2BD.%26rft.volume%253D34%26rft.issue%253D4%26rft.spage%253D882%26rft.epage%253D896%26rft.atitle%253DIdentification%2Bof%2BQuantitative%2BTrait%2BLoci%2BUsing%2Ba%2BSmall%2BSample%2Bof%2BTopcrossed%2Band%2BF4%2BProgeny%2Bfrom%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1994.0011183X003400040010x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1994.0011183X003400040010x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994NV51600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.6" data-doi="10.1186/1297-9686-43-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Besnier</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wahlberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Rönnegård</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Ek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Andersson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fine mapping and replication of QTL in outbred chicken advanced intercross lines.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DBesnier%26rft.auinit1%253DF.%26rft.volume%253D43%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DFine%2Bmapping%2Band%2Breplication%2Bof%2BQTL%2Bin%2Boutbred%2Bchicken%2Badvanced%2Bintercross%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-3%26rft_id%253Dinfo%253Apmid%252F21241486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21241486&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.7" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>,  <span class="cit-name-given-names">S. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Browning</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/521987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.8" data-doi="10.1152/physiolgenomics.00055.2014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carbonetto</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Gyekis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Parker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Blizard</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Discovery and refinement of muscle weight QTLs in B6 × D2 advanced intercross mice.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">582</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1152/physiolgenomics.00055.2014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00055.2014%26rft_id%253Dinfo%253Apmid%252F24963006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/physiolgenomics.00055.2014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24963006&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.9" data-doi="10.1152/physiolgenomics.00092.2016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S. R.</span> <span class="cit-name-surname">Collie-Duguid</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Meharg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Scholz</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Fine-mapping of genes determining extrafusal fiber properties in murine soleus muscle.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1152/physiolgenomics.00092.2016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00092.2016%26rft_id%253Dinfo%253Apmid%252F28087756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/physiolgenomics.00092.2016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28087756&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.10" data-doi="10.1016/j.ygeno.2005.07.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cervino</span>,  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Edwards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Laurie</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Integrating QTL and high-density SNP analyses in mice to identify Insig2 as a susceptibility gene for plasma cholesterol levels.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">505</span>–<span class="cit-lpage">517</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ygeno.2005.07.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DCervino%26rft.auinit1%253DA.%2BC.%26rft.volume%253D86%26rft.issue%253D5%26rft.spage%253D505%26rft.epage%253D517%26rft.atitle%253DIntegrating%2BQTL%2Band%2Bhigh-density%2BSNP%2Banalyses%2Bin%2Bmice%2Bto%2Bidentify%2BInsig2%2Bas%2Ba%2Bsusceptibility%2Bgene%2Bfor%2Bplasma%2Bcholesterol%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2005.07.010%26rft_id%253Dinfo%253Apmid%252F16126366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2005.07.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16126366&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232869900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.11" data-doi="10.1186/1471-2156-12-66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Abney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Palmer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Skol</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">QTLRel: An R package for genome-wide association studies in which relatedness is a concern.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-12-66</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DCheng%26rft.auinit1%253DR.%26rft.volume%253D12%26rft.spage%253D66%26rft.epage%253D66%26rft.atitle%253DQTLRel%253A%2Ban%2BR%2Bpackage%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bin%2Bwhich%2Brelatedness%2Bis%2Ba%2Bconcern.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-12-66%26rft_id%253Dinfo%253Apmid%252F21794153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-12-66&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21794153&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.12" data-doi="10.1534/genetics.110.116863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Lim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. E.</span> <span class="cit-name-surname">Samocha</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sokoloff</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Abney</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association studies and the problem of relatedness among advanced intercross lines and other highly recombinant populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1033</span>–<span class="cit-lpage">1044</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.116863</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116863%26rft_id%253Dinfo%253Apmid%252F20439773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.13" data-doi="10.1534/g3.113.007948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Parker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Abney</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Palmer</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Practical considerations regarding the use of genotype and pedigree data to model relatedness in the context of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1861</span>–<span class="cit-lpage">1867</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.007948</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007948%26rft_id%253Dinfo%253Apmid%252F23979941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTAvMTg2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzMvOTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.14" data-doi="10.1007/s00335-013-9492-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>,  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Out of the bottleneck: The Diversity Outcross and Collaborative Cross mouse populations in behavioral genetics research.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-013-9492-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D25%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-013-9492-9%26rft_id%253Dinfo%253Apmid%252F24272351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-013-9492-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24272351&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.15" data-doi="10.1007/s00335-012-9414-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>,  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Gatti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Munger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Svenson</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The diversity outbred mouse population.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">718</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-012-9414-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D713%26rft.epage%253D718%26rft.atitle%253DThe%2Bdiversity%2Boutbred%2Bmouse%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9414-2%26rft_id%253Dinfo%253Apmid%252F22892839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9414-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892839&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.16" data-doi="10.1007/10_2017_48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockram</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">and I.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> Genetic Mapping Populations for Conducting High-Resolution Trait Mapping in Plants.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/10_2017_48</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGENET%2BMAP%2BPOPUL%2BFOR%2BCONDUCT%2BHIGHRESOLUT%2BTRAIT%2BMAP%2BIN%2BPLANT%26rft.volume%253D164%26rft.spage%253D109%26rft.atitle%253DGENETIC%2BMAPPING%2BPOPULATIONS%2BFOR%2BCONDUCTING%2BHIGH-RESOLUTION%2BTRAIT%2BMAPPING%2BIN%2BPLANTS.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F10_2017_48%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/10_2017_48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.17" data-doi="10.1159/000132186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conneally</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Edwards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. K.</span> <span class="cit-name-surname">Kidd</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-M.</span> <span class="cit-name-surname">Lalouel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. E.</span> <span class="cit-name-surname">Morton</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Report of the committee on methods of linkage analysis and reporting.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000132186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenet.%2BGenome%2BRes.%26rft.volume%253D40%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000132186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000132186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.18" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the Collaborative Cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132639</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.19" data-doi="10.1016/j.nlm.2013.06.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coyner</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">McGuire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Parker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Ursano</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Palmer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mice selectively bred for High and Low fear behavior show differences in the number of pMAPK (p44/42 ERK) expressing neurons in lateral amygdala following Pavlovian fear conditioning.</span> <abbr class="cit-jnl-abbrev">Neurobiol. Learn. Mem.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">203</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.nlm.2013.06.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurobiol.%2BLearn.%2BMem.%26rft.volume%253D112%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.nlm.2013.06.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.nlm.2013.06.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.20" data-doi="10.1126/science.284.5420.1670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crabbe</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Wahlsten</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Dudek</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genetics of mouse behavior: Interactions with laboratory environment.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">1670</span>–<span class="cit-lpage">1672</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.284.5420.1670</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCrabbe%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D284%26rft.issue%253D5420%26rft.spage%253D1670%26rft.epage%253D1672%26rft.atitle%253DGenetics%2Bof%2BMouse%2BBehavior%253A%2BInteractions%2Bwith%2BLaboratory%2BEnvironment%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.284.5420.1670%26rft_id%253Dinfo%253Apmid%252F10356397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODQvNTQyMC8xNjcwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Soller</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Advanced intercross lines, an experimental population for fine genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1199</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDarvasi%26rft.auinit1%253DA.%26rft.volume%253D141%26rft.issue%253D3%26rft.spage%253D1199%26rft.epage%253D1207%26rft.atitle%253DAdvanced%2Bintercross%2Blines%252C%2Ban%2Bexperimental%2Bpopulation%2Bfor%2Bfine%2Bgenetic%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F8582624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MS8zLzExOTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.22" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>,  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Hohenlohe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. Q.</span> <span class="cit-name-surname">Boone</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Catchen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.23" data-doi="10.1023/A:1010261909853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Demarest</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Koyner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">McCaughran</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Cipp</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hitzemann</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Further characterization and high-resolution mapping of quantitative trait loci for ethanol-induced locomotor activity.</span> <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">91</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1010261909853</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DDemarest%26rft.auinit1%253DK.%26rft.volume%253D31%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D91%26rft.atitle%253DFurther%2Bcharacterization%2Band%2Bhigh-resolution%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci%2Bfor%2Bethanol-induced%2Blocomotor%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010261909853%26rft_id%253Dinfo%253Apmid%252F11529277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010261909853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11529277&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170584500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.24" data-doi="10.3389/fpls.2018.00279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diouf</span>,  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Derivot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Bitton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pascual</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Causse</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Water Deficit and Salinity Stress Reveal Many Specific QTL for Plant Growth and Fruit Quality Traits in Tomato.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00279</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D279%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00279%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.25" data-doi="10.1534/genetics.115.186197"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doitsidou</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jarriault</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Poole</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Next-generation sequencing-based approaches for mutation mapping and identification in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">474</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.186197</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDoitsidou%26rft.auinit1%253DM.%26rft.volume%253D204%26rft.issue%253D2%26rft.spage%253D451%26rft.epage%253D474%26rft.atitle%253DNext-Generation%2BSequencing-Based%2BApproaches%2Bfor%2BMutation%2BMapping%2Band%2BIdentification%2Bin%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.186197%26rft_id%253Dinfo%253Apmid%252F27729495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA0LzIvNDUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dupuis</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Siegmund</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Statistical methods for mapping quantitative trait loci from a dense set of markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDupuis%26rft.auinit1%253DJ.%26rft.volume%253D151%26rft.issue%253D1%26rft.spage%253D373%26rft.epage%253D386%26rft.atitle%253DStatistical%2BMethods%2Bfor%2BMapping%2BQuantitative%2BTrait%2BLoci%2BFrom%2Ba%2BDense%2BSet%2Bof%2BMarkers%26rft_id%253Dinfo%253Apmid%252F9872974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUxLzEvMzczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.27" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Glaubitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawamoto</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.951.28"><div class="cit-metadata unstructured">Falconer, D. S., (1960) Introduction to quantitative genetics. <em>Introduction to Quantitative Genetics</em>. <a href="https://www.cabdirect.org/cabdirect/abstract/19601603365">https://www.cabdirect.org/cabdirect/abstract/19601603365</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.29" data-doi="10.1371/journal.pone.0075042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fitzpatrick</span>,  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gopalakrishnan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Cogan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Yager</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Meyer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Variation in the form of Pavlovian conditioned approach behavior among outbred male Sprague-Dawley rats from different vendors and colonies: Sign-tracking <em>vs.</em> goal-tracking.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e75042</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0075042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De75042%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0075042%26rft_id%253Dinfo%253Apmid%252F24098363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0075042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24098363&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.30" data-doi="10.1182/blood-2006-09-048868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardenghi</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Marongiu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ramos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Guy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Breda</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Ineffective erythropoiesis in β-thalassemia is characterized by increased iron absorption mediated by down-regulation of hepcidin and up-regulation of ferroportin.</span> <abbr class="cit-jnl-abbrev">Blood</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">5027</span>–<span class="cit-lpage">5035</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1182/blood-2006-09-048868</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBlood%26rft_id%253Dinfo%253Adoi%252F10.1182%252Fblood-2006-09-048868%26rft_id%253Dinfo%253Apmid%252F17299088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6ImJsb29kam91cm5hbCI7czo1OiJyZXNpZCI7czoxMToiMTA5LzExLzUwMjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>,  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Svenson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Shabalin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.-Y.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Valdar</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait locus mapping methods for diversity outbred mice. <em>G3: Genes</em>, <em>Genomes</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1623</span>–<span class="cit-lpage">1633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D4%26rft.spage%253D1623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.32" data-doi="10.1371/journal.pgen.1000149"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Doss</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Farber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.-Z.</span> <span class="cit-name-surname">Wen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution mapping of gene expression using association in an outbred mouse stock.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e1000149</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000149</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGhazalpour%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De1000149%26rft.epage%253De1000149%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bgene%2Bexpression%2Busing%2Bassociation%2Bin%2Ban%2Boutbred%2Bmouse%2Bstock.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000149%26rft_id%253Dinfo%253Apmid%252F18688273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000149&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18688273&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.33" data-doi="10.1007/s00335-014-9523-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzales</span>,  <span class="cit-name-given-names">N. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Palmer</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fine-mapping QTLs in advanced intercross lines and other outbred populations.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-014-9523-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D25%26rft.spage%253D271%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-014-9523-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-014-9523-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.34" data-doi="10.1038/s41467-018-07642-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzales</span>,  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Seo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. I.</span> <span class="cit-name-surname">Hernandez Cordero</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">St Pierre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Gregory</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genome wide association analysis in a mouse advanced intercross line.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">5162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-07642-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D9%26rft.spage%253D5162%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-07642-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-07642-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.35" data-doi="10.1038/s41467-018-07642-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzales</span>,  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Seo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. I.</span> <span class="cit-name-surname">Hernandez-Cordero</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L. S.</span> <span class="cit-name-surname">Pierre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Gregory</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome wide association study of behavioral, physiological and gene expression traits in a multigenerational mouse intercross.</span> <abbr class="cit-jnl-abbrev">Nat Commun</abbr> 9: 5162.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-07642-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat%2BCommun%26rft.volume%253D9%26rft.spage%253D5162%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-07642-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-07642-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graziano</span>,  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Grady</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Cerami</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The identification of 2, 3-dihydroxybenzoic acid as a potentially useful iron-chelating drug.</span> <abbr class="cit-jnl-abbrev">J. Pharmacol. Exp. Ther.</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">570</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BPharmacology%2Band%2BExperimental%2BTherapeutics%26rft.stitle%253DJ.%2BPharmacol.%2BExp.%2BTher.%26rft.aulast%253DGraziano%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D190%26rft.issue%253D3%26rft.spage%253D570%26rft.epage%253D575%26rft.atitle%253DTHE%2BIDENTIFICATION%2BOF%2B2%252C3-DIHYDROXYBENZOIC%2BACID%2BAS%2BA%2BPOTENTIALLY%2BUSEFUL%2BIRON-CHELATING%2BDRUG%26rft_id%253Dinfo%253Apmid%252F4416298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoianBldCI7czo1OiJyZXNpZCI7czo5OiIxOTAvMy81NzAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.37" data-doi="10.1534/genetics.106.067108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>,  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Genissel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Macdonald</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Long</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">How Repeatable Are Associations Between Polymorphisms in achaete–scute and Bristle Number Variation in Drosophila?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1987</span>–<span class="cit-lpage">1997</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.067108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067108%26rft_id%253Dinfo%253Apmid%252F17277365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS80LzE5ODciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.38" data-doi="10.1371/journal.pgen.1000456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. M.</span> <span class="cit-name-surname">Kang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Eskin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Rapid and accurate multiple testing correction and power estimation for millions of correlated markers.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e1000456</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000456</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHan%26rft.auinit1%253DB.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253De1000456%26rft.epage%253De1000456%26rft.atitle%253DRapid%2Band%2Baccurate%2Bmultiple%2Btesting%2Bcorrection%2Band%2Bpower%2Bestimation%2Bfor%2Bmillions%2Bof%2Bcorrelated%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000456%26rft_id%253Dinfo%253Apmid%252F19381255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000456&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19381255&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.39" data-doi="10.1002/gepi.20606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Abney</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identity by descent estimation with dense genome-wide genotype data.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">567</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20606</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DHan%26rft.auinit1%253DL.%26rft.volume%253D35%26rft.issue%253D6%26rft.spage%253D557%26rft.epage%253D567%26rft.atitle%253DIdentity%2Bby%2Bdescent%2Bestimation%2Bwith%2Bdense%2Bgenome-wide%2Bgenotype%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20606%26rft_id%253Dinfo%253Apmid%252F21769932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20606&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21769932&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.40" data-doi="10.14814/phy2.13561"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez Cordero</span>,  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Carbonetto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Riboni Verri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Gregory</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Vandenbergh</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Replication and discovery of musculoskeletal QTLs in LG/J and SM/J advanced intercross lines.</span> <abbr class="cit-jnl-abbrev">Physiol. Rep.</abbr> <span class="cit-vol">6</span>: e13561.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14814/phy2.13561</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiological%2BReports%26rft.stitle%253DPHY2%26rft.aulast%253DHernandez%2BCordero%26rft.auinit1%253DA.%2BI.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De13561%26rft.epage%253De13561%26rft.atitle%253DReplication%2Band%2Bdiscovery%2Bof%2Bmusculoskeletal%2BQTLs%2Bin%2BLG%252FJ%2Band%2BSM%252FJ%2Badvanced%2Bintercross%2Blines%26rft_id%253Dinfo%253Adoi%252F10.14814%252Fphy2.13561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6InBoeXNyZXBvcnRzIjtzOjU6InJlc2lkIjtzOjEwOiI2LzQvZTEzNTYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.41" data-doi="10.1016/j.ajhg.2019.10.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez Cordero</span>,  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Gonzales</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Parker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sokolof</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Vandenbergh</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genome-wide Associations Reveal Human-Mouse Genetic Convergence and Modifiers of Myogenesis, CPNE1 and STC2.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">1222</span>–<span class="cit-lpage">1236</span>. Erratum: 138.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2019.10.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D105%26rft.spage%253D1222%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2019.10.014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2019.10.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoshino</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Takeuchi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kawai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Sanjo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ogawa</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Cutting Edge: Toll-Like Receptor 4 (TLR4)-Deficient Mice Are Hyporesponsive to Lipopolysaccharide: Evidence for TLR4 as the Lps Gene Product.</span> <abbr class="cit-jnl-abbrev">J. Immunol.</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">3749</span>–<span class="cit-lpage">3752</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DHoshino%26rft.auinit1%253DK.%26rft.volume%253D162%26rft.issue%253D7%26rft.spage%253D3749%26rft.epage%253D3752%26rft.atitle%253DCutting%2BEdge%253A%2BToll-Like%2BReceptor%2B4%2B%2528TLR4%2529-Deficient%2BMice%2BAre%2BHyporesponsive%2Bto%2BLipopolysaccharide%253A%2BEvidence%2Bfor%2BTLR4%2Bas%2Bthe%2BLps%2BGene%2BProduct%26rft_id%253Dinfo%253Apmid%252F10201887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi83LzM3NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.43" data-doi="10.1007/s00018-013-1313-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyun</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Body size regulation and insulin-like growth factor signaling.</span> <abbr class="cit-jnl-abbrev">Cell. Mol. Life Sci.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">2351</span>–<span class="cit-lpage">2365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00018-013-1313-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell.%2BMol.%2BLife%2BSci.%26rft.volume%253D70%26rft.spage%253D2351%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-013-1313-5%26rft_id%253Dinfo%253Apmid%252F23508807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-013-1313-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23508807&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.44" data-doi="10.1186/s12864-018-4441-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnsson</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Henriksen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Höglund</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fogelholm</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Jensen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetical genomics of growth in a chicken model.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-4441-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-4441-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-4441-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.45" data-doi="10.1186/s13059-016-0903-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joo</span>,  <span class="cit-name-given-names">J. W. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Hormozdiari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Han</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Eskin</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multiple testing correction in linear mixed models.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-016-0903-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D17%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-0903-6%26rft_id%253Dinfo%253Apmid%252F27039378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-0903-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27039378&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.46" data-doi="10.1534/g3.119.400194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keele</span>,  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. L.</span> <span class="cit-name-surname">Crouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. N. P.</span> <span class="cit-name-surname">Kelada</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Valdar</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Determinants of QTL Mapping Power in the Realized Collaborative Cross. <em>G3: Genes</em>, <em>Genomes</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1707</span>–<span class="cit-lpage">1727</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.119.400194</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D9%26rft.spage%253D1707%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.119.400194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.119.400194&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.47" data-doi="10.1371/journal.pone.0047713"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">K.-A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Gu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.-A.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.-H.</span> <span class="cit-name-surname">Joh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.-H.</span> <span class="cit-name-surname">Kim</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High Fat Diet-Induced Gut Microbiota Exacerbates Inflammation and Obesity in Mice via the TLR4 Signaling Pathway.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e47713</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0047713</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKim%26rft.auinit1%253DK.%2BA.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De47713%26rft.epage%253De47713%26rft.atitle%253DHigh%2BFat%2BDiet-Induced%2BGut%2BMicrobiota%2BExacerbates%2BInflammation%2Band%2BObesity%2Bin%2BMice%2Bvia%2Bthe%2BTLR4%2BSignaling%2BPathway.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0047713%26rft_id%253Dinfo%253Apmid%252F23091640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0047713&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23091640&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.48" data-doi="10.1534/g3.117.041426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">E. G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Long</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The Beavis Effect in Next-Generation Mapping Panels in Drosophila melanogaster. <em>G3: Genes</em>, <em>Genomes</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1643</span>–<span class="cit-lpage">1652</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.041426</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D7%26rft.spage%253D1643%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.041426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.117.041426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">E. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>,  <span class="cit-name-given-names">S. J.</span></span>, and </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>,  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite> (<span class="cit-pub-date">2012</span>) Properties and power of the Drosophila Synthetic Population Resource for the routine dissection of complex traits. Genetics, 112: 935–949.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kislukhin</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. G.</span> <span class="cit-name-surname">King</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. N.</span> <span class="cit-name-surname">Walters</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Macdonald</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Long</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genetic architecture of methotrexate toxicity is similar in Drosophila melanogaster and humans.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1301</span>–<span class="cit-lpage">1310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D1301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.51" data-doi="10.1186/s12859-014-0356-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>,  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Albrechtsen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ANGSD: Analysis of next generation sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-014-0356-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-014-0356-4%26rft_id%253Dinfo%253Apmid%252F25420514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-014-0356-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25420514&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.52" data-doi="10.1074/jbc.M313356200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>,  <span class="cit-name-given-names">K.-O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-M.</span> <span class="cit-name-surname">Po</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.-C.</span> <span class="cit-name-surname">Lok</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Gong</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Identification of the Jak/Stat Proteins as Novel Downstream Targets of EphA4 Signaling in Muscle: implications in the regulation of acetylcholinesterase expression.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">13383</span>–<span class="cit-lpage">13392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M313356200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M313356200%26rft_id%253Dinfo%253Apmid%252F14729671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzkvMTQvMTMzODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>,  <span class="cit-name-given-names">E. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Botstein</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.54" data-doi="10.1152/physiolgenomics.00100.2010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lionikas</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Lim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Palmer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Blizard</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fine-mapping of muscle weight QTL in LG/J and SM/J intercrosses.</span> <abbr class="cit-jnl-abbrev">Physiol. Genomics</abbr> <span class="cit-vol">42A</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1152/physiolgenomics.00100.2010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fphysiolgenomics.00100.2010%26rft_id%253Dinfo%253Apmid%252F20627939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/physiolgenomics.00100.2010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20627939&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.55" data-doi="10.1038/nmeth.2037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Listgarten</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lippert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Kadie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. I.</span> <span class="cit-name-surname">Davidson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Eskin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Improved linear mixed models for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">526</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DListgarten%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D525%26rft.epage%253D526%26rft.atitle%253DImproved%2Blinear%2Bmixed%2Bmodels%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2037%26rft_id%253Dinfo%253Apmid%252F22669648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22669648&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304778500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.56" data-doi="10.1111/gbb.12029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Logan</span>,  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Robledo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Recla</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. M.</span> <span class="cit-name-surname">Philip</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Bubier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-precision genetic mapping of behavioral traits in the diversity outbred mouse population.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">424</span>–<span class="cit-lpage">437</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/gbb.12029</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2Bbrain%252C%2Band%2Bbehavior%26rft.stitle%253DGenes%2BBrain%2BBehav%26rft.aulast%253DLogan%26rft.auinit1%253DR.%2BW.%26rft.volume%253D12%26rft.spage%253D424%26rft.atitle%253DHigh-precision%2Bgenetic%2Bmapping%2Bof%2Bbehavioral%2Btraits%2Bin%2Bthe%2Bdiversity%2Boutbred%2Bmouse%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fgbb.12029%26rft_id%253Dinfo%253Apmid%252F23433259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/gbb.12029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23433259&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">and B.</span> <span class="cit-name-surname">Walsh</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> Genetics and Analysis of Quantitative Traits. Sunderland, MA: Sinauer.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.58" data-doi="10.1534/genetics.104.026732"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macdonald</span>,  <span class="cit-name-given-names">S. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Long</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A Potential Regulatory Polymorphism Upstream of hairy Is Not Associated With Bristle Number Variation in Wild-Caught Drosophila.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">2127</span>–<span class="cit-lpage">2131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.026732</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMacdonald%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D167%26rft.issue%253D4%26rft.spage%253D2127%26rft.epage%253D2131%26rft.atitle%253DA%2BPotential%2BRegulatory%2BPolymorphism%2BUpstream%2Bof%2Bhairy%2BIs%2BNot%2BAssociated%2BWith%2BBristle%2BNumber%2BVariation%2Bin%2BWild-Caught%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.026732%26rft_id%253Dinfo%253Apmid%252F15342546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ny80LzIxMjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.59" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>,  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Richards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Stone</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Barbadilla</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Ayroles</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Drosophila melanogaster genetic reference panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10811</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mangin</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Goffinet</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rebai</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Constructing confidence intervals for QTL location.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1301</span>–<span class="cit-lpage">1308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMangin%26rft.auinit1%253DB.%26rft.volume%253D138%26rft.issue%253D4%26rft.spage%253D1301%26rft.epage%253D1308%26rft.atitle%253DConstructing%2Bconfidence%2Bintervals%2Bfor%2BQTL%2Blocation.%26rft_id%253Dinfo%253Apmid%252F7896108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzOC80LzEzMDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.61" data-doi="10.1534/genetics.106.061549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dupuis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. W.</span> <span class="cit-name-surname">Broman</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Poor performance of bootstrap confidence intervals for the location of a quantitative trait locus.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">489</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.061549</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061549%26rft_id%253Dinfo%253Apmid%252F16783000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc0LzEvNDgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.62" data-doi="10.1534/genetics.114.162107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marriage</span>,  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. G.</span> <span class="cit-name-surname">King</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Long</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Macdonald</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fine-mapping nicotine resistance loci in Drosophila using a multiparent advanced generation inter-cross population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.162107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D57%26rft.atitle%253DFine-Mapping%2BNicotine%2BResistance%2BLoci%2Bin%2BDrosophila%2BUsing%2Ba%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.162107%26rft_id%253Dinfo%253Apmid%252F25236448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk4LzEvNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.63" data-doi="10.1111/j.1601-183X.2004.00102.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mhyre</span>,  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Chesler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Thiruchelvam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lungu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Cory‐Slechta</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Heritability, correlations and in silico mapping of locomotor behavior and neurochemistry in inbred strains of mice.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">228</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1601-183X.2004.00102.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2Bbrain%252C%2Band%2Bbehavior%26rft.stitle%253DGenes%2BBrain%2BBehav%26rft.aulast%253DMhyre%26rft.auinit1%253DT.%2BR.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D209%26rft.epage%253D228%26rft.atitle%253DHeritability%252C%2Bcorrelations%2Band%2Bin%2Bsilico%2Bmapping%2Bof%2Blocomotor%2Bbehavior%2Band%2Bneurochemistry%2Bin%2Binbred%2Bstrains%2Bof%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-183X.2004.00102.x%26rft_id%253Dinfo%253Apmid%252F15924554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-183X.2004.00102.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15924554&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229562200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.64" data-doi="10.1371/journal.pgen.1005663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Najarro</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Hackett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Highfill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. G.</span> <span class="cit-name-surname">King</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identifying Loci Contributing to Natural Variation in Xenobiotic Resistance in Drosophila.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005663</span>. Erratum: e1005830.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005663</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005663%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.65" data-doi="10.1038/ng.3595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicod</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Davies</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Cai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hassett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Goodstadt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association of multiple complex traits in outbred mice by ultra-low-coverage sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">912</span>–<span class="cit-lpage">918</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3595</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D912%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.66" data-doi="10.1186/s12864-015-1592-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nikolskiy</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Conrad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Chun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Fay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Cheverud</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Using whole-genome sequences of the LG/J and SM/J inbred mouse strains to prioritize quantitative trait genes and nucleotides.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">415</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1592-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D415%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1592-3%26rft_id%253Dinfo%253Apmid%252F26016481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1592-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26016481&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.67" data-doi="10.3758/BF03195394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noldus</span>,  <span class="cit-name-given-names">L. P. J. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Spink</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A. J.</span> <span class="cit-name-surname">Tegelenbosch</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">EthoVision: A versatile video tracking system for automation of behavioral experiments.</span> <abbr class="cit-jnl-abbrev">Behav. Res. Methods Instrum. Comput.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">398</span>–<span class="cit-lpage">414</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3758/BF03195394</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bresearch%2Bmethods%252C%2Binstruments%252C%2B%2526%2Bcomputers%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bthe%2BPsychonomic%2BSociety%252C%2BInc%26rft.stitle%253DBehav%2BRes%2BMethods%2BInstrum%2BComput%26rft.aulast%253DNoldus%26rft.auinit1%253DL.%2BP.%26rft.volume%253D33%26rft.issue%253D3%26rft.spage%253D398%26rft.epage%253D414%26rft.atitle%253DEthoVision%253A%2Ba%2Bversatile%2Bvideo%2Btracking%2Bsystem%2Bfor%2Bautomation%2Bof%2Bbehavioral%2Bexperiments.%26rft_id%253Dinfo%253Adoi%252F10.3758%252FBF03195394%26rft_id%253Dinfo%253Apmid%252F11591072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3758/BF03195394&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11591072&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171221000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.68" data-doi="10.1534/genetics.114.167056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Carbonetto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sokoloff</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. J.</span> <span class="cit-name-surname">Park</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Abney</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">High-resolution genetic mapping of complex traits from a combined analysis of F2 and advanced intercross mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.167056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DParker%26rft.auinit1%253DC.%2BC.%26rft.volume%253D198%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D116%26rft.atitle%253DHigh-Resolution%2BGenetic%2BMapping%2Bof%2BComplex%2BTraits%2Bfrom%2Ba%2BCombined%2BAnalysis%2Bof%2BF2%2Band%2BAdvanced%2BIntercross%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.167056%26rft_id%253Dinfo%253Apmid%252F25236452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMTAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.69" data-doi="10.1007/s00335-011-9349-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sokoloff</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Lim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Skol</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fine-mapping alleles for body weight in LG/J × SM/J F<sub>2</sub> and F(34) advanced intercross lines.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">571</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00335-011-9349-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DParker%26rft.auinit1%253DC.%2BC.%26rft.volume%253D22%26rft.issue%253D9-10%26rft.spage%253D563%26rft.epage%253D571%26rft.atitle%253DFine-mapping%2Balleles%2Bfor%2Bbody%2Bweight%2Bin%2BLG%252FJ%2B%25C3%2597%2BSM%252FJ%2BF2%2Band%2BF%252834%2529%2Badvanced%2Bintercross%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-011-9349-z%26rft_id%253Dinfo%253Apmid%252F21761260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-011-9349-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21761260&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295130800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.70" data-doi="10.1111/j.1601-183X.2011.00747.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sokoloff</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Palmer</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association for methamphetamine sensitivity in an advanced intercross mouse line.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1601-183X.2011.00747.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2Bbrain%252C%2Band%2Bbehavior%26rft.stitle%253DGenes%2BBrain%2BBehav%26rft.aulast%253DParker%26rft.auinit1%253DC.%2BC.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D52%26rft.epage%253D61%26rft.atitle%253DGenome-wide%2Bassociation%2Bfor%2Bmethamphetamine%2Bsensitivity%2Bin%2Ban%2Badvanced%2Bintercross%2Bmouse%2Bline.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-183X.2011.00747.x%26rft_id%253Dinfo%253Apmid%252F22032291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-183X.2011.00747.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22032291&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298989400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.71" data-doi="10.1038/ng.3609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gopalakrishnan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Carbonetto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. M.</span> <span class="cit-name-surname">Gonzales</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Leung</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide association study of behavioral, physiological and gene expression traits in outbred CFW mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">919</span>–<span class="cit-lpage">926</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3609</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D919%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.72" data-doi="10.3389/fgene.2011.00032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>,  <span class="cit-name-given-names">C. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Palmer</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Dark matter: Are mice the solution to missing heritability?</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2011.00032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFront%2BGenet%26rft.aulast%253DParker%26rft.auinit1%253DC.%2BC.%26rft.volume%253D2%26rft.spage%253D32%26rft.epage%253D32%26rft.atitle%253DDark%2Bmatter%253A%2Bare%2Bmice%2Bthe%2Bsolution%2Bto%2Bmissing%2Bheritability%253F%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2011.00032%26rft_id%253Dinfo%253Apmid%252F22303328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2011.00032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22303328&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.951.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>,  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-source">Molecular Dissection of Complex Traits</span>, <span class="cit-publ-name">CRC Press</span>. Boca Raton, FL.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.74" data-doi="10.1093/bioinformatics/btq419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pruim</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. P.</span> <span class="cit-name-surname">Welch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sanna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Teslovich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Chines</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">LocusZoom: Regional visualization of genome-wide association scan results.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2336</span>–<span class="cit-lpage">2337</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq419</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq419%26rft_id%253Dinfo%253Apmid%252F20634204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20634204&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281714100054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.75" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Neale</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Todd-Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Thomas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Ferreira</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: A tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.3.951.76" data-doi="10.1038/ng.2644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Rat Genome Sequencing and Mapping Consortium</span>; </li><li><span class="cit-auth"><span class="cit-name-surname">Baud</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hermsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Guryev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Stridh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Graham</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Combined sequence-based and genetic mapping analysis of complex traits in outbred rats.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">775</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2644</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D767%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2644%26rft_id%253Dinfo%253Apmid%252F23708188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23708188&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.77" data-doi="10.1083/jcb.200508044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Relaix</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Montarras</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zaffran</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gayraud-Morel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Rocancourt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Pax3 and Pax7 have distinct and overlapping functions in adult muscle progenitor cells.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1083/jcb.200508044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200508044%26rft_id%253Dinfo%253Apmid%252F16380438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6IjE3Mi8xLzkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.78" data-doi="10.1111/pce.12938"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rishmawi</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bühler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Jaegle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hülskamp</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Koornneef</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Quantitative trait loci controlling leaf venation in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1429</span>–<span class="cit-lpage">1441</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pce.12938</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BEnviron.%26rft.volume%253D40%26rft.spage%253D1429%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpce.12938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pce.12938&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.79" data-doi="10.1111/j.1601-183X.2010.00613.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samocha</span>,  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Lim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Sokoloff</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Palmer</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fine mapping of QTL for prepulse inhibition in LG/J and SM/J mice using F2 and advanced intercross lines.</span> <abbr class="cit-jnl-abbrev">Genes Brain Behav.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">767</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1601-183X.2010.00613.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%252C%2Bbrain%252C%2Band%2Bbehavior%26rft.stitle%253DGenes%2BBrain%2BBehav%26rft.aulast%253DSamocha%26rft.auinit1%253DK.%2BE.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D759%26rft.epage%253D767%26rft.atitle%253DFine%2Bmapping%2Bof%2BQTL%2Bfor%2Bprepulse%2Binhibition%2Bin%2BLG%252FJ%2Band%2BSM%252FJ%2Bmice%2Busing%2BF%25282%2529%2Band%2Badvanced%2Bintercross%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-183X.2010.00613.x%26rft_id%253Dinfo%253Apmid%252F20597988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1601-183X.2010.00613.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20597988&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282634300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.80" data-doi="10.1371/journal.pone.0180785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawada</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Arai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Jing</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Miyajima</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Frank</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Molecular interactions of EphA4, growth hormone receptor, Janus kinase 2, and signal transducer and activator of transcription 5B.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e0180785</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0180785</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0180785%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0180785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0180785&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.81" data-doi="10.1172/JCI28898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Kokoeva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Inouye</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Tzameli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Yin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">TLR4 links innate immunity and fatty acid–induced insulin resistance.</span> <abbr class="cit-jnl-abbrev">J. Clin. Invest.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">3015</span>–<span class="cit-lpage">3025</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1172/JCI28898</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BClinical%2BInvestigation%26rft.stitle%253DJCI%26rft.volume%253D116%26rft.issue%253D11%26rft.spage%253D3015%26rft.epage%253D3025%26rft.atitle%253DTLR4%2Blinks%2Binnate%2Bimmunity%2Band%2Bfatty%2Bacid-induced%2Binsulin%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1172%252FJCI28898%26rft_id%253Dinfo%253Apmid%252F17053832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1172/JCI28898&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17053832&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241810900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.82" data-doi="10.1016/j.neuron.2016.08.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sittig</span>,  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Carbonetto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Engel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Krauss</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Barrios-Camacho</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic Background Limits Generalizability of Genotype-Phenotype Relationships.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">1253</span>–<span class="cit-lpage">1259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.neuron.2016.08.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D91%26rft.spage%253D1253%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2016.08.013%26rft_id%253Dinfo%253Apmid%252F27618673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2016.08.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27618673&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.83" data-doi="10.1038/nmeth.2935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorge</span>,  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. J.</span> <span class="cit-name-surname">Martin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Isbester</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. G.</span> <span class="cit-name-surname">Sotocinal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Rosen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Olfactory exposure to males, including men, causes stress and related analgesia in rodents.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">632</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2935</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D629%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2935%26rft_id%253Dinfo%253Apmid%252F24776635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24776635&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336899300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.84" data-doi="10.1534/genetics.111.132597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Svenson</span>,  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Gatti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Valdar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Welsh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cheng</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping using the Mouse Diversity outbred population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132597</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132597%26rft_id%253Dinfo%253Apmid%252F22345611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDM3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85NTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.85" data-doi="10.1016/S1074-7613(00)80119-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takeuchi</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Hoshino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kawai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Sanjo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Takada</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Differential Roles of TLR2 and TLR4 in Recognition of Gram-Negative and Gram-Positive Bacterial Cell Wall Components.</span> <abbr class="cit-jnl-abbrev">Immunity</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">443</span>–<span class="cit-lpage">451</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1074-7613(00)80119-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DImmunity%26rft.stitle%253DImmunity%26rft.aulast%253DTakeuchi%26rft.auinit1%253DO.%26rft.volume%253D11%26rft.issue%253D4%26rft.spage%253D443%26rft.epage%253D451%26rft.atitle%253DDifferential%2Broles%2Bof%2BTLR2%2Band%2BTLR4%2Bin%2Brecognition%2Bof%2Bgram-negative%2Band%2Bgram-positive%2Bbacterial%2Bcell%2Bwall%2Bcomponents.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1074-7613%252800%252980119-3%26rft_id%253Dinfo%253Apmid%252F10549626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1074-7613(00)80119-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10549626&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083343100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.86" data-doi="10.1038/6825"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>,  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nicod</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Cherny</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Fulker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Collins</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">High-resolution mapping of quantitative trait loci in outbred mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/6825</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTalbot%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D305%26rft.epage%253D308%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bquantitative%2Btrait%2Bloci%2Bin%2Boutbred%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F6825%26rft_id%253Dinfo%253Apmid%252F10080185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/6825&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10080185&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078977900026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.87" data-doi="10.12688/f1000research.9471.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toker</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Feng</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pavlidis</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whose sample is it anyway? Widespread misannotation of samples in transcriptomics studies.</span> <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/f1000research.9471.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft.volume%253D5%26rft.spage%253D2103%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.9471.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.9471.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.88" data-doi="10.1038/ng1840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Solberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gauguier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Burnett</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Klenerman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome-wide genetic association of complex traits in heterogeneous stock mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">887</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1840</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DValdar%26rft.auinit1%253DW.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D879%26rft.epage%253D887%26rft.atitle%253DGenome-wide%2Bgenetic%2Bassociation%2Bof%2Bcomplex%2Btraits%2Bin%2Bheterogeneous%2Bstock%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1840%26rft_id%253Dinfo%253Apmid%252F16832355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16832355&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.89" data-doi="10.1038/nrg2322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Hill</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Wray</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Heritability in the genomics era—Concepts and misconceptions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">266</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D255%26rft.epage%253D266%26rft.atitle%253DHeritability%2Bin%2Bthe%2Bgenomics%2Bera--concepts%2Band%2Bmisconceptions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2322%26rft_id%253Dinfo%253Apmid%252F18319743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18319743&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254084100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.90" data-doi="10.1016/j.ajhg.2017.06.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>,  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Wray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sklar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. I.</span> <span class="cit-name-surname">McCarthy</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">10 Years of GWAS Discovery: Biology, Function, and Translation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2017.06.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D101%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2017.06.005%26rft_id%253Dinfo%253Apmid%252F28686856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2017.06.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28686856&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.91" data-doi="10.1371/journal.pgen.1005616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vonesch</span>,  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lamparter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F.</span> <span class="cit-name-surname">Mackay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bergmann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Hafen</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide analysis reveals novel regulators of growth in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1005616</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005616</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005616%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005616%26rft_id%253Dinfo%253Apmid%252F26751788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26751788&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.3.951.92" data-doi="10.1038/ng.2435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Wellcome Trust Case Control Consortium</span>; </li><li><span class="cit-auth"><span class="cit-name-surname">Maller</span>,  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">McVean</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Byrnes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Vukcevic</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Palin</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bayesian refinement of association signals for 14 loci in 3 common diseases.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1294</span>–<span class="cit-lpage">1301</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2435</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D44%26rft.spage%253D1294%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2435%26rft_id%253Dinfo%253Apmid%252F23104008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104008&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.93" data-doi="10.1038/hdy.1992.131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Theoretical Basis of the Beavis Effect.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">2259</span>–<span class="cit-lpage">2268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DXu%26rft.auinit1%253DS.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D2259%26rft.epage%253D2268%26rft.atitle%253DTheoretical%2BBasis%2Bof%2Bthe%2BBeavis%2BEffect%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1992.131%26rft_id%253Dinfo%253Apmid%252F14704201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzIyNTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzk1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.94" data-doi="10.1038/ng1450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Willis-Owen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fullerton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Meesaq</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Deacon</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic dissection of a behavioral quantitative trait locus shows that Rgs2 modulates anxiety in mice.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1197</span>–<span class="cit-lpage">1202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1450</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYalcin%26rft.auinit1%253DB.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1197%26rft.epage%253D1202%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Ba%2Bbehavioral%2Bquantitative%2Btrait%2Blocus%2Bshows%2Bthat%2BRgs2%2Bmodulates%2Banxiety%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1450%26rft_id%253Dinfo%253Apmid%252F15489855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15489855&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.95" data-doi="10.1016/j.ajhg.2010.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GCTA: A tool for genome-wide complex trait analysis.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2010.11.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D88%26rft.issue%253D1%26rft.spage%253D76%26rft.epage%253D82%26rft.atitle%253DGCTA%253A%2Ba%2Btool%2Bfor%2Bgenome-wide%2Bcomplex%2Btrait%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2010.11.011%26rft_id%253Dinfo%253Apmid%252F21167468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2010.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21167468&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.96" data-doi="10.1038/ng.2876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Zaitlen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Price</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Advantages and pitfalls in the application of mixed-model association methods.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2876</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2876%26rft_id%253Dinfo%253Apmid%252F24473328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24473328&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.97" data-doi="10.1038/ng.3941"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zeng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Wray</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Visscher</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Concepts, estimation and interpretation of SNP-based heritability.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1304</span>–<span class="cit-lpage">1310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3941</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D1304%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3941%26rft_id%253Dinfo%253Apmid%252F28854176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3941&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28854176&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.98" data-doi="10.1007/s00439-012-1199-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kraft</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Heritability in the genome-wide association era.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1655</span>–<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00439-012-1199-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DZaitlen%26rft.auinit1%253DN.%26rft.volume%253D131%26rft.issue%253D10%26rft.spage%253D1655%26rft.epage%253D1664%26rft.atitle%253DHeritability%2Bin%2Bthe%2Bgenome-wide%2Bassociation%2Bera.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-012-1199-6%26rft_id%253Dinfo%253Apmid%252F22821350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-012-1199-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22821350&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.99" data-doi="10.1038/ng.2310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>,  <span class="cit-name-given-names">X.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide efficient mixed-model analysis for association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhou%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D821%26rft.epage%253D824%26rft.atitle%253DGenome-wide%2Befficient%2Bmixed-model%2Banalysis%2Bfor%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2310%26rft_id%253Dinfo%253Apmid%252F22706312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706312&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.100" data-doi="10.1086/512821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zöllner</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Overcoming the Winner’s Curse: Estimating Penetrance Parameters from Case-Control Data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">605</span>–<span class="cit-lpage">615</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/512821</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DZollner%26rft.auinit1%253DS.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D605%26rft.epage%253D615%26rft.atitle%253DOvercoming%2Bthe%2Bwinner%2527s%2Bcurse%253A%2Bestimating%2Bpenetrance%2Bparameters%2Bfrom%2Bcase-control%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F512821%26rft_id%253Dinfo%253Apmid%252F17357068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/512821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17357068&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F951.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.951.101" data-doi="10.1101/856898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Faller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Brown</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Eskin</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Accurate modeling of replication rates in genome-wide association studies by accounting for winner’s curse and study-specific heterogeneity.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-vol">21</span><span class="cit-pub-id-sep cit-pub-id-doi-sep">: </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/856898</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F856898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI4NTY4OTh2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzMvOTUxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/3/951.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/945" title="CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/967" title="The Drosophila Post-mating Response: Gene Expression and Behavioral Changes Reveal Perdurance and Variation in Cross-Tissue Interactions" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/3"><span>Volume 10 Issue 3, March 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/3.cover-source.jpg?itok=8jjPWDA4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/951?rss=1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8ww2SWHU50xI9gSWPq6mg0pdreh1yAjVMd42vW_sHC0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/3/951" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/3/951" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/3/951" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495847" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-BRSqCEdNaolsRBiJObJklwF4itP2f57hVwabsH4wktg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/951.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/3/951" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RSucgbpqOxNtV3sA795cCxt4gk5-nsiOVVzN8RLL20M" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/3/951" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/951.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847--2" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6432-0033" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xinzhu Zhou</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5465-6601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Celine L. St. Pierre</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5289-0410" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Natalia M. Gonzales</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2045-3272" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jennifer Zou</span>, <span class="highwire-citation-author" data-delta="4">Riyan Cheng</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1709-9214" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Apurva S. Chitre</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-3651-1980" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Greta Sokoloff</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3634-0747" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Abraham A. Palmer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">951-965;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400763 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495847--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xinzhu Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biomedical Sciences Graduate Program, University of California San Diego, La Jolla, CA, 92092,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinzhu%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXinzhu%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6432-0033" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xinzhu Zhou</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Celine L. St. Pierre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Washington University School of Medicine, St. Louis, MO, 63110,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celine%2BL.%2BSt.%2BPierre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=St.%20Pierre%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACeline%2BL.%2BSt.%2BPierre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5465-6601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Celine L. St. Pierre</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Natalia M. Gonzales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Human Genetics, University of Chicago, IL, 60637,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2BM.%2BGonzales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzales%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANatalia%2BM.%2BGonzales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5289-0410" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Natalia M. Gonzales</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jennifer Zou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Computer Science, University of California, Los Angeles, CA, 90095,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BZou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJennifer%2BZou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2045-3272" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jennifer Zou</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Psychiatry,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Apurva S. Chitre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Psychiatry,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Apurva%2BS.%2BChitre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chitre%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AApurva%2BS.%2BChitre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1709-9214" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Apurva S. Chitre</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Greta Sokoloff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Psychological & Brain Sciences, University of Iowa, Iowa City, IO, 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Greta%2BSokoloff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sokoloff%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGreta%2BSokoloff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3651-1980" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Greta Sokoloff</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Abraham A. Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Psychiatry,</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Genomic Medicine, University of California San Diego, La Jolla, CA, 92037 and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abraham%2BA.%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbraham%2BA.%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3634-0747" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Abraham A. Palmer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aap@ucsd.edu" class="" data-icon-position="" data-hide-link-title="0">aap@ucsd.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/3/951" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/3/951&amp;title=Genome-Wide%20Association%20Study%20in%20Two%20Cohorts%20from%20a%20Multi-generational%20Mouse%20Advanced%20Intercross%20Line%20Highlights%20the%20Difficulty%20of%20Replication%20Due%20to%20Study-Specific%20Heterogeneity" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/3/951&amp;title=Genome-Wide%20Association%20Study%20in%20Two%20Cohorts%20from%20a%20Multi-generational%20Mouse%20Advanced%20Intercross%20Line%20Highlights%20the%20Difficulty%20of%20Replication%20Due%20to%20Study-Specific%20Heterogeneity" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/3/951&amp;title=Genome-Wide%20Association%20Study%20in%20Two%20Cohorts%20from%20a%20Multi-generational%20Mouse%20Advanced%20Intercross%20Line%20Highlights%20the%20Difficulty%20of%20Replication%20Due%20to%20Study-Specific%20Heterogeneity" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/3/951&amp;text=Genome-Wide%20Association%20Study%20in%20Two%20Cohorts%20from%20a%20Multi-generational%20Mouse%20Advanced%20Intercross%20Line%20Highlights%20the%20Difficulty%20of%20Replication%20Due%20to%20Study-Specific%20Heterogeneity" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/3/951&amp;title=Genome-Wide%20Association%20Study%20in%20Two%20Cohorts%20from%20a%20Multi-generational%20Mouse%20Advanced%20Intercross%20Line%20Highlights%20the%20Difficulty%20of%20Replication%20Due%20to%20Study-Specific%20Heterogeneity" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/3/951&amp;t=Genome-Wide%20Association%20Study%20in%20Two%20Cohorts%20from%20a%20Multi-generational%20Mouse%20Advanced%20Intercross%20Line%20Highlights%20the%20Difficulty%20of%20Replication%20Due%20to%20Study-Specific%20Heterogeneity" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/3/951&amp;title=Genome-Wide%20Association%20Study%20in%20Two%20Cohorts%20from%20a%20Multi-generational%20Mouse%20Advanced%20Intercross%20Line%20Highlights%20the%20Difficulty%20of%20Replication%20Due%20to%20Study-Specific%20Heterogeneity" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/3/951&amp;title=Genome-Wide%20Association%20Study%20in%20Two%20Cohorts%20from%20a%20Multi-generational%20Mouse%20Advanced%20Intercross%20Line%20Highlights%20the%20Difficulty%20of%20Replication%20Due%20to%20Study-Specific%20Heterogeneity" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495847/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847--3" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6432-0033" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Xinzhu Zhou</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5465-6601" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Celine L. St. Pierre</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5289-0410" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Natalia M. Gonzales</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-2045-3272" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jennifer Zou</span>, <span class="highwire-citation-author" data-delta="4">Riyan Cheng</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1709-9214" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Apurva S. Chitre</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-3651-1980" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Greta Sokoloff</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3634-0747" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Abraham A. Palmer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">951-965;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400763 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495847--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Xinzhu Zhou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Biomedical Sciences Graduate Program, University of California San Diego, La Jolla, CA, 92092,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinzhu%2BZhou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhou%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AXinzhu%2BZhou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6432-0033" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Xinzhu Zhou</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Celine L. St. Pierre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Washington University School of Medicine, St. Louis, MO, 63110,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celine%2BL.%2BSt.%2BPierre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=St.%20Pierre%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACeline%2BL.%2BSt.%2BPierre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5465-6601" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Celine L. St. Pierre</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Natalia M. Gonzales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Human Genetics, University of Chicago, IL, 60637,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Natalia%2BM.%2BGonzales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonzales%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANatalia%2BM.%2BGonzales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5289-0410" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Natalia M. Gonzales</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jennifer Zou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Computer Science, University of California, Los Angeles, CA, 90095,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BZou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJennifer%2BZou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2045-3272" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jennifer Zou</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Riyan Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Psychiatry,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Riyan%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARiyan%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Apurva S. Chitre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Psychiatry,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Apurva%2BS.%2BChitre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chitre%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AApurva%2BS.%2BChitre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1709-9214" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Apurva S. Chitre</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Greta Sokoloff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Psychological & Brain Sciences, University of Iowa, Iowa City, IO, 52242</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Greta%2BSokoloff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sokoloff%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGreta%2BSokoloff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3651-1980" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Greta Sokoloff</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Abraham A. Palmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Psychiatry,</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Institute for Genomic Medicine, University of California San Diego, La Jolla, CA, 92037 and </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abraham%2BA.%2BPalmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palmer%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAbraham%2BA.%2BPalmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3634-0747" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Abraham A. Palmer</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aap@ucsd.edu" class="" data-icon-position="" data-hide-link-title="0">aap@ucsd.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=951&amp;rft.epage=965&amp;rft.atitle=Genome-Wide%20Association%20Study%20in%20Two%20Cohorts%20from%20a%20Multi-generational%20Mouse%20Advanced%20Intercross%20Line%20Highlights%20the%20Difficulty%20of%20Replication%20Due%20to%20Study-Specific%20Heterogeneity&amp;rft.volume=10&amp;rft.issue=3&amp;rft.date=2020-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhou%2C+Xinzhu&amp;rft.au=St.+Pierre%2C+Celine+L.&amp;rft.au=Gonzales%2C+Natalia+M.&amp;rft.au=Zou%2C+Jennifer&amp;rft.au=Cheng%2C+Riyan&amp;rft.au=Chitre%2C+Apurva+S.&amp;rft.au=Sokoloff%2C+Greta&amp;rft.au=Palmer%2C+Abraham+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495847/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495847/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495847/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495847/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495847/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495847/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495847/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495847/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495847/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495847/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495847/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495847/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/3/951" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/3/951.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/3/951.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/3/951.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
