<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/3/1063" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/3/1063.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/3/1063.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/3/1063.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/3/1063/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/3/1063.atom" />
<meta name="HW.pisa" content="ggg;10/3/1063" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Selective Sweeps Under Dominance and Inbreeding" />
<meta name="DC.Identifier" content="10.1534/g3.119.400919" />
<meta name="DC.Date" content="2020-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Hartfield, Bataillon. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A major research goal in evolutionary genetics is to uncover loci experiencing positive selection. One approach involves finding ‘selective sweeps’ patterns, which can either be ‘hard sweeps’ formed by de novo mutation, or ‘soft sweeps’ arising from recurrent mutation or existing standing variation. Existing theory generally assumes outcrossing populations, and it is unclear how dominance affects soft sweeps. We consider how arbitrary dominance and inbreeding via self-fertilization affect hard and soft sweep signatures. With increased self-fertilization, they are maintained over longer map distances due to reduced effective recombination and faster beneficial allele fixation times. Dominance can affect sweep patterns in outcrossers if the derived variant originates from either a single novel allele, or from recurrent mutation. These models highlight the challenges in distinguishing hard and soft sweeps, and propose methods to differentiate between scenarios." />
<meta name="DC.Contributor" content="Matthew Hartfield" />
<meta name="DC.Contributor" content="Thomas Bataillon" />
<meta name="article:published_time" content="2020-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Selective Sweeps Under Dominance and Inbreeding" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A major research goal in evolutionary genetics is to uncover loci experiencing positive selection. One approach involves finding ‘selective sweeps’ patterns, which can either be ‘hard sweeps’ formed by &lt;i&gt;de novo&lt;/i&gt; mutation, or ‘soft sweeps’ arising from recurrent mutation or existing standing variation. Existing theory generally assumes outcrossing populations, and it is unclear how dominance affects soft sweeps. We consider how arbitrary dominance and inbreeding via self-fertilization affect hard and soft sweep signatures. With increased self-fertilization, they are maintained over longer map distances due to reduced effective recombination and faster beneficial allele fixation times. Dominance can affect sweep patterns in outcrossers if the derived variant originates from either a single novel allele, or from recurrent mutation. These models highlight the challenges in distinguishing hard and soft sweeps, and propose methods to differentiate between scenarios.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/03/01" />
<meta name="citation_mjid" content="ggg;10/3/1063" />
<meta name="citation_id" content="10/3/1063" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/3/1063" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/3/1063.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/3/1063.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/3/1063.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400919" />
<meta name="citation_pmid" content="31974096" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1063" />
<meta name="citation_lastpage" content="1075" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Matthew Hartfield" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Toronto, Ontario M5S 3B2, Canada," />
<meta name="citation_author_institution" content="Bioinformatics Research Centre, Aarhus University, Aarhus 8000, Denmark, and" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology, The University of Edinburgh, Edinburgh EH9 3FL, United Kingdom" />
<meta name="citation_author_email" content="m.hartfield@ed.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6941-0647" />
<meta name="citation_author" content="Thomas Bataillon" />
<meta name="citation_author_institution" content="Bioinformatics Research Centre, Aarhus University, Aarhus 8000, Denmark, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4730-2538" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. C. Andersen;citation_author=J. P. Gerke;citation_author=J. A. Shapiro;citation_author=J. R. Crissman;citation_author=R. Ghosh;citation_title=Chromosome-scale selective sweeps shape Caenorhabditis elegans genomic diversity.;citation_pages=285-290;citation_volume=44;citation_year=2012;citation_issue=3;citation_pmid=22286215;citation_doi=10.1038/ng.1050" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. J. C. Anderson;citation_author=S. Nair;citation_author=M. McDew-White;citation_author=I. H. Cheeseman;citation_author=S. Nkhoma;citation_title=Population parameters underlying an ongoing soft sweep in southeast asian malaria parasites.;citation_volume=34;citation_year=2016;citation_pmid=28025270;citation_doi=10.1093/molbev/msw228" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=H. Badouin;citation_author=P. Gladieux;citation_author=J. Gouzy;citation_author=S. Siguenza;citation_author=G. Aguileta;citation_title=Widespread selective sweeps throughout the genome of model plant pathogenic fungi and identification of effector candidates.;citation_pages=2041-2062;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.13976" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=R. D. H. Barrett;citation_author=D. Schluter;citation_title=Adaptation from standing genetic variation.;citation_pages=38-44;citation_volume=23;citation_year=2008;citation_issue=1;citation_pmid=18006185;citation_doi=10.1016/j.tree.2007.09.008" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=N. H. Barton;citation_title=The effect of hitch-hiking on neutral genealogies.;citation_pages=123-133;citation_volume=72;citation_year=1998;citation_doi=10.1017/S0016672398003462" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. J. Berg;citation_author=G. Coop;citation_title=A coalescent model for a sweep of a unique standing variant.;citation_volume=201;citation_year=2015;citation_pmid=26311475;citation_doi=10.1534/genetics.115.178962" />
<meta name="citation_reference" content="citation_journal_title=Biological Reviews (Cambridge; Print);citation_journal_abbrev=Biological Reviews (Cambridge; Print);citation_author=S. Billiard;citation_author=M. López-Villavicencio;citation_author=B. Devier;citation_author=M. E. Hood;citation_author=C. Fairhead;citation_title=Having sex, yes, but with whom? Inferences from fungi on the evolution of anisogamy and mating types.;citation_pages=421-442;citation_volume=86;citation_year=2011;citation_issue=2;citation_pmid=21489122;citation_doi=10.1111/j.1469-185X.2010.00153.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Boitard;citation_author=C. Schlötterer;citation_author=A. Futschik;citation_title=Detecting selective sweeps: A new approach based on hidden markov models.;citation_volume=181;citation_year=2009;citation_pmid=19204373;citation_doi=10.1534/genetics.108.100032" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Bonhomme;citation_author=S. Boitard;citation_author=H. San Clemente;citation_author=B. Dumas;citation_author=N. Young;citation_title=Genomic signature of selective sweeps illuminates adaptation of Medicago truncatula to root-associated microorganisms.;citation_volume=32;citation_year=2015;citation_pmid=25901015;citation_doi=10.1093/molbev/msv092" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. R. Booker;citation_author=P. D. Keightley;citation_title=Understanding the factors that shape patterns of nucleotide diversity in the house mouse genome.;citation_pages=2971-2988;citation_volume=35;citation_year=2018;citation_doi=10.1093/molbev/msy188" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Braverman;citation_author=R. R. Hudson;citation_author=N. L. Kaplan;citation_author=C. H. Langley;citation_author=W. Stephan;citation_title=The hitchhiking effect on the site frequency spectrum of DNA polymorphisms.;citation_pages=783-796;citation_volume=140;citation_year=1995;citation_issue=2;citation_pmid=7498754" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Caballero;citation_author=W. G. Hill;citation_title=Effects of partial inbreeding on fixation rates and variation of mutant genes.;citation_pages=493-507;citation_volume=131;citation_year=1992;citation_issue=2;citation_pmid=1644283" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. L. Campos;citation_author=B. Charlesworth;citation_title=The effects on neutral variability of recurrent selective sweeps and background selection.;citation_volume=212;citation_year=2019;citation_pmid=30923166;citation_doi=10.1534/genetics.119.301951" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. L. Campos;citation_author=L. Zhao;citation_author=B. Charlesworth;citation_title=Estimating the parameters of background selection and selective sweeps in drosophila in the presence of gene conversion.;citation_volume=114;citation_year=2017;citation_pmid=28559322;citation_doi=10.1073/pnas.1619434114" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=B. Charlesworth;citation_title=Evolutionary rates in partially self-fertilizing species.;citation_pages=126-148;citation_volume=140;citation_year=1992;citation_pmid=19426068;citation_doi=10.1086/285406" />
<meta name="citation_reference" content="citation_title=Elements of Evolutionary Genetics;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L.-M. Chevin;citation_author=S. Billiard;citation_author=F. Hospital;citation_title=Hitchhiking both ways: Effect of two interfering selective sweeps on linked neutral variation.;citation_volume=180;citation_year=2008;citation_pmid=18716333;citation_doi=10.1534/genetics.108.089706" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. DeGiorgio;citation_author=C. D. Huber;citation_author=M. J. Hubisz;citation_author=I. Hellmann;citation_author=R. Nielsen;citation_title=SweepFinder2: increased sensitivity, robustness and flexibility.;citation_volume=32;citation_year=2016;citation_pmid=27153702;citation_doi=10.1093/bioinformatics/btw051" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. M. Desai;citation_author=D. S. Fisher;citation_title=Beneficial mutation-selection balance and the effect of linkage on positive selection.;citation_volume=176;citation_year=2007;citation_pmid=17483432;citation_doi=10.1534/genetics.106.067678" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. Elyashiv;citation_author=S. Sattath;citation_author=T. T. Hu;citation_author=A. Strutsovsky;citation_author=G. McVicker;citation_title=A genomic map of the effects of linked selection in Drosophila.;citation_pages=e1006130;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pgen.1006130" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Ewing;citation_author=J. Hermisson;citation_title=MSMS: a coalescent simulation program including recombination, demographic structure and selection at a single locus.;citation_volume=26;citation_year=2010;citation_pmid=20591904;citation_doi=10.1093/bioinformatics/btq322" />
<meta name="citation_reference" content="citation_journal_title=Journal of mathematical biology;citation_journal_abbrev=J Math Biol;citation_author=G. Ewing;citation_author=J. Hermisson;citation_author=P. Pfaffelhuber;citation_author=J. Rudolf;citation_title=Selective sweeps for recessive alleles and for other modes of dominance.;citation_pages=399-431;citation_volume=63;citation_year=2011;citation_issue=3;citation_pmid=21076829;citation_doi=10.1007/s00285-010-0382-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Fay;citation_author=C.-I. Wu;citation_title=Hitchhiking Under Positive Darwinian Selection;citation_pages=1405-1413;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880498" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Ferrer-Admetlla;citation_author=M. Liang;citation_author=T. Korneliussen;citation_author=R. Nielsen;citation_title=On detecting incomplete soft or hard selective sweeps using haplotype structure.;citation_volume=31;citation_year=2014;citation_pmid=24554778;citation_doi=10.1093/molbev/msu077" />
<meta name="citation_reference" content="citation_journal_title=Proc. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=On the dominance ratio.;citation_pages=321-341;citation_volume=42;citation_year=1922;citation_doi=10.1017/S0370164600023993" />
<meta name="citation_reference" content="citation_journal_title=Genes Genet. Syst.;citation_author=N. T. Fujito;citation_author=Y. Satta;citation_author=T. Hayakawa;citation_author=N. Takahata;citation_title=A new inference method for detecting an ongoing selective sweep.;citation_pages=149-161;citation_volume=93;citation_year=2018;citation_doi=10.1266/ggs.18-00008" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Fulgione;citation_author=M. Koornneef;citation_author=F. Roux;citation_author=J. Hermisson;citation_author=A. M. Hancock;citation_title=Madeiran Arabidopsis thaliana reveals ancient long-range colonization and clarifies demography in Eurasia.;citation_pages=564-574;citation_volume=35;citation_year=2018;citation_doi=10.1093/mol-bev/msx300" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. A. Fustier;citation_author=J. T. Brandenburg;citation_author=S. Boitard;citation_author=J. Lapeyronnie;citation_author=L. E. Eguiarte;citation_title=Signatures of local adaptation in lowland and highland teosintes from whole-genome sequencing of pooled samples.;citation_pages=2738-2756;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14082" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. R. Garud;citation_author=P. W. Messer;citation_author=E. O. Buzbas;citation_author=D. A. Petrov;citation_title=Recent Selective Sweeps in North American Drosophila melanogaster Show Signatures of Soft Sweeps.;citation_pages=e1005004;citation_volume=11;citation_year=2015;citation_pmid=25706129;citation_doi=10.1371/journal.pgen.1005004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. R. Garud;citation_author=D. A. Petrov;citation_title=Elevated linkage disequilibrium and signatures of soft sweeps are common in Drosophila melanogaster.;citation_volume=203;citation_year=2016;citation_pmid=27098909;citation_doi=10.1534/genetics.115.184002" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=S. Glémin;citation_title=Extinction and fixation times with dominance and inbreeding.;citation_pages=310-316;citation_volume=81;citation_year=2012;citation_issue=4;citation_pmid=22425679;citation_doi=10.1016/j.tpb.2012.02.006" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=S. Glémin;citation_author=J. Ronfort;citation_title=Adaptation and maladaptation in selfing and outcrossing species: new mutations versus standing variation.;citation_volume=67;citation_year=2013;citation_pmid=23289574;citation_doi=10.1111/j.1558-5646.2012.01778.x" />
<meta name="citation_reference" content="citation_journal_title=Math. Proc. Camb. Philos. Soc.;citation_author=J. B. S. Haldane;citation_title=A mathematical theory of natural and artificial selection, part V: Selection and mutation.;citation_pages=838-844;citation_volume=23;citation_year=1927;citation_doi=10.1017/S0305004100015644" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. C. Haller;citation_author=P. W. Messer;citation_title=SLiM 3: Forward Genetic Simulations Beyond the Wright–Fisher Model.;citation_volume=36;citation_year=2019;citation_doi=10.1093/molbev/msy228" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=A. M. Harris;citation_author=M. DeGiorgio;citation_title=Identifying and classifying shared selective sweeps from multilocus data.;citation_year=2018;citation_doi=10.1101/446005" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=A. M. Harris;citation_author=M. DeGiorgio;citation_title=A likelihood approach for uncovering selective sweep signatures from haplotype data.;citation_year=2019;citation_doi=doi: 10.1101/678722" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. M. Harris;citation_author=N. R. Garud;citation_author=M. DeGiorgio;citation_title=Detection and classification of hard and soft sweeps from unphased genotypes by multilocus genotype identity.;citation_volume=210;citation_year=2018;citation_pmid=30315068;citation_doi=10.1534/genetics.118.301502" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. B. Harris;citation_author=A. Sackman;citation_author=J. D. Jensen;citation_title=On the unfounded enthusiasm for soft selective sweeps II: Examining recent evidence from humans, flies, and viruses.;citation_pages=e1007859;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pgen.1007859" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=M. Hartfield;citation_author=T. Bataillon;citation_author=S. Glémin;citation_title=The evolutionary interplay between adaptation and self-fertilization.;citation_pages=420-431;citation_volume=33;citation_year=2017;citation_doi=10.1016/j.tig.2017.04.002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Hartfield;citation_author=S. Glémin;citation_title=Hitchhiking of deleterious alleles and the cost of adaptation in partially selfing species.;citation_volume=196;citation_year=2014;citation_pmid=24240529;citation_doi=10.1534/genetics.113.158196" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Hartfield;citation_author=S. Glémin;citation_title=Limits to adaptation in partially selfing species.;citation_volume=203;citation_year=2016;citation_pmid=27098913;citation_doi=10.1534/genetics.116.188821" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. W. Hedrick;citation_title=HITCHHIKING: A COMPARISON OF LINKAGE AND PARTIAL SELFING;citation_pages=791-808;citation_volume=94;citation_year=1980;citation_issue=3;citation_pmid=6931092" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Hermisson;citation_author=P. S. Pennings;citation_title=Soft sweeps: Molecular population genetics of adaptation from standing genetic variation.;citation_volume=169;citation_year=2005;citation_pmid=15716498;citation_doi=10.1534/genetics.104.036947" />
<meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.;citation_author=J. Hermisson;citation_author=P. S. Pennings;citation_title=Soft sweeps and beyond: understanding the patterns and probabilities of selection footprints under rapid adaptation.;citation_pages=700-716;citation_volume=8;citation_year=2017;citation_doi=10.1111/2041-210X.12808" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. D. Huber;citation_author=M. DeGiorgio;citation_author=I. Hellmann;citation_author=R. Nielsen;citation_title=Detecting recent selective sweeps while controlling for mutation rate and background selection.;citation_pages=142-156;citation_volume=25;citation_year=2016;citation_pmid=26290347;citation_doi=10.1111/mec.13351" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. D. Huber;citation_author=M. Nordborg;citation_author=J. Hermisson;citation_author=I. Hellmann;citation_title=Keeping It Local: Evidence for Positive Selection in Swedish Arabidopsis thaliana.;citation_volume=31;citation_year=2014;citation_pmid=25158800;citation_doi=10.1093/molbev/msu247" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=B. Igic;citation_author=J. R. Kohn;citation_title=The distribution of plant mating systems: study bias against obligately outcrossing species.;citation_pages=1098-1103;citation_volume=60;citation_year=2006;citation_issue=5;citation_pmid=16817548;citation_doi=10.1111/j.0014-3820.2006.tb01186.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Innan;citation_author=Y. Kim;citation_title=Pattern of polymorphism after strong artificial selection in a domestication event.;citation_volume=101;citation_year=2004;citation_pmid=15249682;citation_doi=10.1073/pnas.0401720101" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Innan;citation_author=M. Nordborg;citation_title=The Extent of Linkage Disequilibrium and Haplotype Sharing Around a Polymorphic Site;citation_pages=437-444;citation_volume=165;citation_year=2003;citation_issue=1;citation_pmid=14504248" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=P. Jarne;citation_author=J. R. Auld;citation_title=Animals mix it up too: the distribution of self-fertilization among hermaphroditic animals.;citation_pages=1816-1824;citation_volume=60;citation_year=2006;citation_issue=9;citation_pmid=17089966;citation_doi=10.1111/j.0014-3820.2006.tb00525.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. D. Jensen;citation_title=On the unfounded enthusiasm for soft selective sweeps.;citation_pages=5281;citation_volume=5;citation_year=2014;citation_pmid=25345443;citation_doi=10.1038/ncomms6281" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. L. Kaplan;citation_author=R. R. Hudson;citation_author=C. H. Langley;citation_title=The &quot;hitchhiking effect&quot; revisited.;citation_pages=887-899;citation_volume=123;citation_year=1989;citation_issue=4;citation_pmid=2612899" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. Karasov;citation_author=P. W. Messer;citation_author=D. A. Petrov;citation_title=Evidence that adaptation in Drosophila is not limited by mutation at single sites.;citation_pages=e1000924-e1000924;citation_volume=6;citation_year=2010;citation_issue=6;citation_pmid=20585551;citation_doi=10.1371/journal.pgen.1000924" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. D. Kern;citation_author=D. R. Schrider;citation_title=diploS/HIC: An updated approach to classifying selective sweeps.;citation_volume=8;citation_year=2018;citation_pmid=29626082;citation_doi=10.1534/g3.118.200262" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Kim;citation_author=R. Nielsen;citation_title=Linkage Disequilibrium as a Signature of Selective Sweeps;citation_pages=1513-1524;citation_volume=167;citation_year=2004;citation_issue=3;citation_pmid=15280259;citation_doi=10.1534/genetics.103.025387" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Kim;citation_author=W. Stephan;citation_title=Detecting a Local Signature of Genetic Hitchhiking Along a Recombining Chromosome;citation_pages=765-777;citation_volume=160;citation_year=2002;citation_issue=2;citation_pmid=11861577" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. Kim;citation_author=W. Stephan;citation_title=Selective Sweeps in the Presence of Interference Among Partially Linked Loci;citation_pages=389-398;citation_volume=164;citation_year=2003;citation_issue=1;citation_pmid=12750349" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=V. Laporte;citation_author=B. Charlesworth;citation_title=Effective Population Size and Population Subdivision in Demographically Structured Populations;citation_pages=501-519;citation_volume=162;citation_year=2002;citation_issue=1;citation_pmid=12242257" />
<meta name="citation_reference" content="Laval, G., E. Patin, P. Boutillier, and L. Quintana-Murci, 2019 A genome-wide approximate bayesian computation approach suggests only limited numbers of soft sweeps in humans over the last 100,000 years. bioRxiv p. 2019.12.22.886234." />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Q. Long;citation_author=F. A. Rabanal;citation_author=D. Meng;citation_author=C. D. Huber;citation_author=A. Farlow;citation_title=Massive genomic variation and strong selection in Arabidopsis thaliana lines from Sweden.;citation_pages=884-890;citation_volume=45;citation_year=2013;citation_pmid=23793030;citation_doi=10.1038/ng.2678" />
<meta name="citation_reference" content="citation_journal_title=Theor. Popul. Biol.;citation_author=G. Martin;citation_author=A. Lambert;citation_title=A simple, semi-deterministic approximation to the distribution of selective sweeps in large populations.;citation_pages=40-46;citation_volume=101;citation_year=2015;citation_doi=10.1016/j.tpb.2015.01.004" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=J. Maynard Smith;citation_title=What determines the rate of evolution?;citation_pages=331-338;citation_volume=110;citation_year=1976;citation_doi=10.1086/283071" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. Maynard Smith;citation_author=J. Haigh;citation_title=The hitch-hiking effect of a favourable gene.;citation_pages=23-35;citation_volume=23;citation_year=1974;citation_issue=1;citation_pmid=4407212;citation_doi=10.1017/S0016672300014634" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. A. T. McVean;citation_title=The structure of linkage disequilibrium around a selective sweep.;citation_volume=175;citation_year=2007;citation_pmid=17194788;citation_doi=10.1534/genetics.106.062828" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=P. W. Messer;citation_author=D. A. Petrov;citation_title=Population genomics of rapid adaptation by soft selective sweeps.;citation_pages=659-669;citation_volume=28;citation_year=2013;citation_pmid=24075201;citation_doi=10.1016/j.tree.2013.08.003" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Nielsen;citation_title=Molecular signals of natural selection.;citation_pages=197-218;citation_volume=39;citation_year=2005;citation_pmid=16285858;citation_doi=10.1146/annurev.genet.39.073003.112420" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=R. Nielsen;citation_author=S. Williamson;citation_author=Y. Kim;citation_author=M. J. Hubisz;citation_author=A. G. Clark;citation_title=Genomic scans for selective sweeps using SNP data;citation_pages=1566-1575;citation_volume=15;citation_year=2005;citation_issue=11;citation_pmid=16251466;citation_doi=10.1101/gr.4252305" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Nordborg;citation_title=Linkage Disequilibrium, Gene Trees and Selfing: An Ancestral Recombination Graph With Partial Self-Fertilization;citation_pages=923-929;citation_volume=154;citation_year=2000;citation_issue=2;citation_pmid=10655241" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=M. Nordborg;citation_author=B. Charlesworth;citation_author=D. Charlesworth;citation_title=Increased levels of polymorphism surrounding selectively maintained sites in highly selfing species.;citation_volume=263;citation_year=1996;citation_doi=10.1098/rspb.1996.0152" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Nordborg;citation_author=P. Donnelly;citation_title=The Coalescent Process With Selfing;citation_pages=1185-1195;citation_volume=146;citation_year=1997;citation_issue=3;citation_pmid=9215919" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. A. Orr;citation_author=A. J. Betancourt;citation_title=Haldane's Sieve and Adaptation From the Standing Genetic Variation;citation_pages=875-884;citation_volume=157;citation_year=2001;citation_issue=2;citation_pmid=11157004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. Pavlidis;citation_author=D. Živković;citation_author=A. Stamatakis;citation_author=N. Alachiotis;citation_title=SweeD: Likelihood-Based Detection of Selective Sweeps in Thousands of Genomes.;citation_volume=30;citation_year=2013;citation_pmid=23777627;citation_doi=10.1093/molbev/mst112" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. S. Pennings;citation_author=J. Hermisson;citation_title=Soft Sweeps II – Molecular Population Genetics of Adaptation from Recurrent Mutation or Migration.;citation_volume=23;citation_year=2006;citation_pmid=16520336;citation_doi=10.1093/molbev/msj117" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. S. Pennings;citation_author=J. Hermisson;citation_title=Soft sweeps III: the signature of positive selection from recurrent mutation.;citation_pages=e186-e186;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17173482;citation_doi=10.1371/journal.pgen.0020186" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=P. S. Pennings;citation_author=S. Kryazhimskiy;citation_author=J. Wakeley;citation_title=Loss and Recovery of Genetic Diversity in Adapting Populations of HIV.;citation_pages=e1004000;citation_volume=10;citation_year=2014;citation_pmid=24465214;citation_doi=10.1371/journal.pgen.1004000" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. M. Peter;citation_author=E. Huerta-Sanchez;citation_author=R. Nielsen;citation_title=Distinguishing between Selective Sweeps from Standing Variation and from a De Novo Mutation.;citation_pages=e1003011-e1003011;citation_volume=8;citation_year=2012;citation_issue=10;citation_pmid=23071458;citation_doi=10.1371/journal.pgen.1003011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. Pollak;citation_title=On the Theory of Partially Inbreeding Finite Populations. I. Partial Selfing;citation_pages=353-360;citation_volume=117;citation_year=1987;citation_issue=2;citation_pmid=3666446" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. Price;citation_author=B. T. Moyers;citation_author=L. Lopez;citation_author=J. R. Lasky;citation_author=J. G. Monroe;citation_title=Combining population genomics and fitness QTLs to identify the genetics of local adaptation in Arabidopsis thaliana.;citation_volume=115;citation_year=2018;citation_pmid=29686078;citation_doi=10.1073/pnas.1719998115" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. K. Pritchard;citation_author=A. Di Rienzo;citation_title=Adaptation - not by sweeps alone.;citation_pages=665-667;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20838407;citation_doi=10.1038/nrg2880" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. Przeworski;citation_author=G. Coop;citation_author=J. D. Wall;citation_title=The signature of positive selection on standing genetic variation.;citation_pages=2312-2323;citation_volume=59;citation_year=2005;citation_issue=11;citation_pmid=16396172;citation_doi=10.1554/05-273.1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Rettelbach;citation_author=A. Nater;citation_author=H. Ellegren;citation_title=How linked selection shapes the diversity landscape in Ficedula flycatchers.;citation_volume=212;citation_year=2019;citation_pmid=30872320;citation_doi=10.1534/genetics.119.301991" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=D. Roze;citation_title=Diploidy, population structure, and the evolution of recombination.;citation_pages=S79-S94;citation_volume=174;citation_year=2009;citation_pmid=19476412;citation_doi=10.1086/599083" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Roze;citation_title=Background selection in partially selfing populations.;citation_volume=203;citation_year=2016;citation_pmid=27075726;citation_doi=10.1534/genetics.116.187955" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=D. E. Reich;citation_author=J. M. Higgins;citation_author=H. Z. P. Levine;citation_author=D. J. Richter;citation_title=Detecting recent positive selection in the human genome from haplotype structure.;citation_pages=832-837;citation_volume=419;citation_year=2002;citation_issue=6909;citation_pmid=12397357;citation_doi=10.1038/nature01140" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=D. J. Schoen;citation_author=M. T. Morgan;citation_author=T. Bataillon;citation_title=How Does Self-Pollination Evolve? Inferences from Floral Ecology and Molecular Genetic Variation.;citation_volume=351;citation_year=1996;citation_doi=10.1098/rstb.1996.0111" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=S/HIC: Robust identification of soft and hard sweeps using machine learning.;citation_pages=e1005928;citation_volume=12;citation_year=2016;citation_pmid=26977894;citation_doi=10.1371/journal.pgen.1005928" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=Soft sweeps are the dominant mode of adaptation in the human genome.;citation_pages=1863-1877;citation_volume=34;citation_year=2017;citation_pmid=28482049;citation_doi=10.1093/molbev/msx154" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Schrider;citation_author=F. K. Mendes;citation_author=M. W. Hahn;citation_author=A. D. Kern;citation_title=Soft shoulders ahead: Spurious signatures of soft and partial selective sweeps result from linked hard sweeps.;citation_volume=200;citation_year=2015;citation_pmid=25716978;citation_doi=10.1534/genetics.115.174912" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=D. Setter;citation_author=S. Mousset;citation_author=X. Cheng;citation_author=R. Nielsen;citation_author=M. DeGiorgio;citation_title=Volcanofinder: genomic scans for adaptive introgression.;citation_year=2019;citation_doi=doi: 10.1101/697987" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=S. Sheehan;citation_author=Y. S. Song;citation_title=Deep learning for population genetic inference.;citation_pages=e1004845;citation_volume=12;citation_year=2016;citation_doi=10.1371/journal.pcbi.1004845" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=W. Stephan;citation_title=Signatures of positive selection: from selective sweeps at individual loci to subtle allele frequency changes in polygenic adaptation.;citation_pages=79-88;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13288" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=W. Stephan;citation_title=Selective sweeps.;citation_volume=211;citation_year=2019;citation_pmid=30626638;citation_doi=10.1534/genetics.118.301319" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=EVOLUTIONARY RELATIONSHIP OF DNA SEQUENCES IN FINITE POPULATIONS;citation_pages=437-460;citation_volume=105;citation_year=1983;citation_issue=2;citation_pmid=6628982" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. M. Teshima;citation_author=G. Coop;citation_author=M. Przeworski;citation_title=How reliable are empirical genomic scans for selective sweeps?;citation_volume=16;citation_year=2006;citation_pmid=16687733;citation_doi=10.1101/gr.5105206" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. M. Teshima;citation_author=M. Przeworski;citation_title=Directional positive selection on an allele of arbitrary dominance.;citation_volume=172;citation_year=2006;citation_pmid=16219788;citation_doi=10.1534/genetics.105.044065" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. Thomson;citation_title=THE EFFECT OF A SELECTED LOCUS ON LINKED NEUTRAL LOCI;citation_pages=753-788;citation_volume=85;citation_year=1977;citation_issue=4;citation_pmid=863244;citation_doi=10.1534/genetics.118.301018" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=H. Uecker;citation_title=Evolutionary rescue in randomly mating, selfing, and clonal populations.;citation_pages=845-858;citation_volume=71;citation_year=2017;citation_doi=10.1111/evo.13191" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=O. A. van Herwaarden;citation_author=N. J. van der Wal;citation_title=Extinction time and age of an allele in a large finite population.;citation_pages=311-318;citation_volume=61;citation_year=2002;citation_issue=3;citation_pmid=12027617;citation_doi=10.1006/tpbi.2002.1576" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=A. I. Vatsiou;citation_author=E. Bazin;citation_author=O. E. Gaggiotti;citation_title=Detection of selective sweeps in structured populations: a comparison of recent methods.;citation_pages=89-103;citation_volume=25;citation_year=2016;citation_doi=10.1111/mec.13360" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=J. J. Vitti;citation_author=S. R. Grossman;citation_author=P. C. Sabeti;citation_title=Detecting natural selection in genomic data.;citation_pages=97-120;citation_volume=47;citation_year=2013;citation_pmid=24274750;citation_doi=10.1146/annurev-genet-111212-133526" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=B. F. Voight;citation_author=S. Kudaravalli;citation_author=X. Wen;citation_author=J. K. Pritchard;citation_title=A map of recent positive selection in the human genome.;citation_pages=e72-e72;citation_volume=4;citation_year=2006;citation_issue=3;citation_pmid=16494531;citation_doi=10.1371/journal.pbio.0040072" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=H. M. T. Vy;citation_author=Y.-J. Won;citation_author=Y. Kim;citation_title=Multiple Modes of Positive Selection Shaping the Patterns of Incomplete Selective Sweeps over African Populations of Drosophila melanogaster.;citation_pages=2792-2807;citation_volume=34;citation_year=2017;citation_doi=10.1093/mol-bev/msx207" />
<meta name="citation_reference" content="citation_title=Coalescent theory: an introduction;citation_volume=Vol. 1;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=K.-A. Williams;citation_author=P. S. Pennings;citation_title=Drug resistance evolution in HIV in the late 1990s: hard sweeps, soft sweeps, clonal interference and the accumulation of drug resistance mutations.;citation_year=2019;citation_doi=doi: 10.1101/548198" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. A. Wilson;citation_author=P. S. Pennings;citation_author=D. A. Petrov;citation_title=Soft selective sweeps in evolutionary rescue.;citation_volume=205;citation_year=2017;citation_pmid=28213477;citation_doi=10.1534/genetics.116.191478" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. A. Wilson;citation_author=D. A. Petrov;citation_author=P. W. Messer;citation_title=Soft Selective Sweeps in Complex Demographic Scenarios.;citation_volume=198;citation_year=2014;citation_pmid=25060100;citation_doi=10.1534/genetics.114.165571" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=S. Wright;citation_title=The genetical structure of populations.;citation_pages=323-354;citation_volume=15;citation_year=1951;citation_doi=10.1111/j.1469-1809.1949.tb02451.x" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=A. T. Xue;citation_author=D. R. Schrider;citation_author=A. D. Kern;citation_title=Discovery of ongoing selective sweeps within Anopheles mosquito populations using deep learning.;citation_year=2019;citation_doi=doi: 10.1101/589069" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Yang;citation_author=J. Li;citation_author=T. Wiehe;citation_author=H. Li;citation_title=Detecting recent positive selection with a single locus test bipartitioning the coalescent tree.;citation_volume=208;citation_year=2018;citation_pmid=29217523;citation_doi=10.1534/genetics.117.300401" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=Y. Zheng;citation_author=T. Wiehe;citation_title=Adaptation in structured populations and fuzzy boundaries between hard and soft sweeps.;citation_pages=e1007426;citation_volume=15;citation_year=2019;citation_doi=10.1371/journal.pcbi.1007426" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=L. Zhong;citation_author=Q. Yang;citation_author=X. Yan;citation_author=C. Yu;citation_author=L. Su;citation_title=Signatures of soft sweeps across the Dt1 locus underlying determinate growth habit in soya bean;citation_pages=4686-4699;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.14209" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Selective Sweeps Under Dominance and Inbreeding" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="twitter:description" content="A major research goal in evolutionary genetics is to uncover loci experiencing positive selection. One approach involves finding ‘selective sweeps’ patterns, which can either be ‘hard sweeps’ formed by de novo mutation, or ‘soft sweeps’ arising from recurrent mutation or existing standing variation. Existing theory generally assumes outcrossing populations, and it is unclear how dominance affects soft sweeps. We consider how arbitrary dominance and inbreeding via self-fertilization affect hard and soft sweep signatures. With increased self-fertilization, they are maintained over longer map distances due to reduced effective recombination and faster beneficial allele fixation times. Dominance can affect sweep patterns in outcrossers if the derived variant originates from either a single novel allele, or from recurrent mutation. These models highlight the challenges in distinguishing hard and soft sweeps, and propose methods to differentiate between scenarios." />
<meta name="og-title" property="og:title" content="Selective Sweeps Under Dominance and Inbreeding" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/3/1063" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A major research goal in evolutionary genetics is to uncover loci experiencing positive selection. One approach involves finding ‘selective sweeps’ patterns, which can either be ‘hard sweeps’ formed by de novo mutation, or ‘soft sweeps’ arising from recurrent mutation or existing standing variation. Existing theory generally assumes outcrossing populations, and it is unclear how dominance affects soft sweeps. We consider how arbitrary dominance and inbreeding via self-fertilization affect hard and soft sweep signatures. With increased self-fertilization, they are maintained over longer map distances due to reduced effective recombination and faster beneficial allele fixation times. Dominance can affect sweep patterns in outcrossers if the derived variant originates from either a single novel allele, or from recurrent mutation. These models highlight the challenges in distinguishing hard and soft sweeps, and propose methods to differentiate between scenarios." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
    <title>Selective Sweeps Under Dominance and Inbreeding | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"qddVq0b90mLOSRvSFTaP-qO0vLDWfb3IoPp9N3JBepY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495861","apath":"\/ggg\/10\/3\/1063.atom","pisa":"ggg;10\/3\/1063","ac":{"\/ggg\/10\/3\/1063.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/3\/1063.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/3\/1063"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/3\/1063"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/3\/1063?rss=1":true,"\/content\/10\/3\/1063":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LSboMuXsrW0O6vbeSsbx5Ra0ehHVSFdOYM0OcVObGc\\\\\\\/dvIDPznjShl1hdTtV0mDcFCgjccqYdHROqG+5D24wnvQP4OYW2GzC9keYFlzs3eU7YjFWODrI840bD3KJiWHvFTUmQKjFl\\\\\\\/bXKpOOCsMp2DR4A3KhPnJ029h9heDcoOxKVGCum4qQv1J+yN0T\\\\\\\/LWebF8m8YrVkYkuxBBNPBNGkTr8itFELAhEycG14DEXpt\\\\\\\/hlMSq9DoLjBETAC5bsmK9yjhjIV6aWfTjWQCTBiL9WoInLH25IUZsOd7lEWqj6StHv\\\\\\\/8Et32RFrsrVOGZCkstl4dmmenpC7q+K4zAAZgGrZdUFgdUso7cbOv7EVjkLauGCARnw\\\\\\\/fz5SIaGGLyCPZ0pvN6c46Wghdc9\\\\\\\/wd39prtBj6sB6An7aOAnuB8yM7Hxn2sG2+Ok+w\\\\\\\/oLL2YJ9sIyNPztXoufthN23kEemBrUJAk1to8du9dFqDv66s4mzcoZDeqL93u6aDYdvjeTlmUsG2\\\\\\\/Ai4yFhHGOaLbPgeMc5SLMHsrA0NByGIt6Uy9ZgpZ3z58KYL2hzdqdPpqHGFU5WYcAwIgycDs\\\\\\\/BqSe\\\\\\\/NGQzhHqDAGtqB+efRpxFj8Y5KjY\\\\\\\/1gMbxgpJmMfOsJHhvNaqQimbV2kCcTowOeBvYlxkIMzRw0M4onnJnUn4WvFd+SYActUbgCIQxWmhHW0tunCe5RFinbQWIGvJRjvNr6Q7D4TOBujjfTj7oIzafYYAkwFiPp\\\\\\\/1pt8kraU2kGeF\\\\\\\/h7Zk+lc\\\\\\\/vfu\\\\\\\/00Mc7P8Kv3op6gNgZxkhLAtZ2XqlD2YDcaA9klpC7rtPcLFMv\\\\\\\/h8I\\\\\\\/ipuxDRMmKpuBlUBrVVgMHs8si5\\\\\\\/cRgFgWnh9rk2hB2bCsBBzhzu34QndCPjdMfWMr9JFnUw3t\\\\\\\/tQlColas\\\\\\\/gstcizQbWcHel+peNFuSp+MkGBDWG+BfjFV6Hp\\\\\\\/yIGy8tBb9FcBC3inegMtfPfEPvBvt18oaT1zaO5vZhwoII2IwInKCtUYjxJGDGjL8Eu3Cl2RuWsGXzeTO7GGEprXXLvin48Fg+ViPVP7Y8\\\\\\\/KFgGUGxN+tKVSyisJqeOuohPri5aDqFnPfTdnIkTgLJLweT+IWPAV2rIJlJlztM++rLnlS2sDZ+w6UyzyLzKvXaa0pqjjzxr3FnT2mvjCLHLuzhjgZ0GMfpPNURxO2udAG3CQHdHOrxhd6Pi0j23Y\\\\\\\/bNKyW+ViY3OYLS8kQ+fWvuE0IXVMd2XelRS1uWEfTEbBYizyPdxKcqRaLxFCkRhL+37dc0\\\u0022,\\\u0022iv\\\u0022:\\\u0022G7u5LHFbKOqXiFtKFvVvQw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002203a891b5f41e4282ef96e390ab7913a75ee9c3be6ddd112246c51c56b3ca312e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qzipMXWQX8sJ0o6lKyFUSbFNLd0ybuR4xtsyt3p9vmUo3n+HD++auVD4EDjx3PIrBf4B2h6S40acnsymYatr1zj5hIkQtkte+PpUj8ibLHEzSuXWSxCbZ4pIxswbGTzDyJDWd65x1SG5eypdoPr9uZ0AnJHR8r3EgE7HVX0ShVhlrzu171k8OB6uYEGtRMMVYUeIIULy6mcHXJtPEHXbIMmLyPkYx2Sd3f8wd50\\\\\\\/Nqh183vvW9U6yDb++qc7fGJysn1UGprmUjr8sibyj\\\\\\\/z6HwpLBjI5T6Y6aw87R0Vost083v5ycMWZlnVuuoa\\\\\\\/j3BbhpnlCmCWMTgRcxbqpbHM2FhfK6z\\\\\\\/CxD7gpabo4VI\\\\\\\/Z3AJZYav6jJYfUcVmGZEv39HCL1fk62OGTp+L\\\\\\\/sZCsvIST8PreGT0\\\\\\\/VNneo0uG9rzoi5XnNieCYot9yjI7idP+\\\\\\\/Vw3aGHrDw8we8JFCnK7YF\\\\\\\/\\\\\\\/2GEozs1LcrtS+8VSiHs9fhPyxi8+oOKIznc2DOqLZoVquASKrhBp2ZO2VI7RVgId\\\\\\\/+jpWbMzvx4oWM5tfldbDfHc3B67MJR1ru9oIkRaWGrwr+59kGMerj1iUDXou1KMV8LMkfxapm5glwHwf1DpNBZ9iyPGsa73WE5JWclGQ\\\\\\\/fpYtZCkAQpNuQi49+DV4mkqJ0fepuD2lJPSd+NECYrfO6+Gf3V8+9bNK9yaGwRex2s8gX1M\\\\\\\/losAzxE6jpZwnvNU8pzB3f1Ze1oPo7gub5ByRNl1TnjuRdqnkJIznV9aufxmlu5W\\\\\\\/CqDTy\\\\\\\/l+J9BMbcsoTYjoeLue5qbSV2efQ\\\\\\\/8TgN8mY3RBge54\\\\\\\/JCUWN5Vq2xXMHzA0DZBcifqtG1Ej6C0ps57UTDmPOjEK40qbe433LdNoh2X+Uw\\\\\\\/TsNx9C08ptCjEzOr7ah9v3T0oJl9VXDh4pph7yMJXWxeAYCWdq3pQLGDLKhEE4rwBPwccmDtYv+J3jTDcdlHhQ5gbsOTX9ANJefXLUUCEz8kLm7r4ySQHyz1IdEJEizO0IJyy1iBXoK1px0HV78y0E5h\\\\\\\/os6AN\\\\\\\/ez\\\\\\\/AmI83R26uo35bmDIjug7ZZZEB+lSQDZPQmJdg34o6fkLPFikEJcmtAYIlDUOIVJ9lFeRhyUAFCdM7tPOSQfecmOBdZ\\\\\\\/UukCh0G0rpyKCMPQIdDPuqqpmOK4fkeXg5hGHPx9mo18IcekZczIS2QhuTt4PEJpK+4jIo5ZvfemDITErxdiEgk6gsiLy2uBzRPLLJ6IxGw9VcDlNYJ0ZC9xqI5gaxE3KKP8Q1FLJI36kJWn7hzv3W1eTXSwe9REYlkaG3QGUXHIBOYbTrpXDMllR6s69AqeSXwQXgi\\\\\\\/MY2VSgH5whwJkZBAtfg+JalHHg3TlZeS3oTGvoEHjmbEqoI14HRxmWoHWH+N2+bvSk+gVlnSoGspY5jevwpTV\\\\\\\/i+7truKlotnZboQPRDJBd7ZXwdKj93PshLIY2vM4tqFhgAYWXNJRV+BOkMmqwxJAGU5l9UkboP\\\\\\\/a1csC9N+Wzsisf0J+rf4d8DXQtnAFS5oekGDCHehHUrzFPJ9WKD5rOO1bJTKcnXMvzXabsk9+7wW4wtcIniQHKr8KjfMbR8XH3i5T6kUK3iSstISz8w9wq3B6y+RSw3wjqDIcEWEJQ3j5XwZhChMN40p\\\\\\\/EdR+f+rrtIzLHLkOEXTVswMt\\\\\\\/05RjsM5LvqfmFYRkAgCQ4NSSDFOJxNXNbNdFxOn0tZ+kHZ7gZe935vlSpb5Lyps7kN8Eu6MuWoPOqE0b5ZZccMlA\\\\\\\/1mqK19G0HT+4tpsFHTANzZupRB5P6AO9JmuNCZG7C5LxuM8Ji\\\\\\\/i5XDckxj23KP4pdnDL7U015v9iCuqZ\\\\\\\/NU\\\\\\\/dVk44WAfvU9YYazE3tq1Zx6+R7f8VYMmgJZA+RGjQg\\\\\\\/bqtCPRso0MWAN2fAv4kVwBwHeqt+vn5vIzb29KMGf4qY4kEmZVyiTwLV3yH47PIarFHzTtA1lZPtn5DEE8M+GbQp5SLWZe2d2yErw8TbMOERmUKla\\\\\\\/S6pXyIvkbfJbn7CA2\\\\\\\/Kq0X+iGtYtlPUHnHuGJNBt+ry5+ZZbhJU2NXNiJywPS6YRcEtAozSsSGKFtBIsCipCe83fYqhXKNAh9DDwsdwPg5SRrRxs4DPcRXagIdq8ai4kFCLDTTzpbpC2lphYSgwvnYMt6htHeTlTdoPzgb6oW2rzdJ7s8qsR9nFstHNdPkRD8nL0YuZcrD8BvXNeygDix2umpt\\\\\\\/Q9PxM2PDiDUHqdPm1HGgo5d5Yu6X3CRiTyFc6gjkFe7RwU3iNQ5EkErotopwqjQiLdKqEtdkeoELGIrxbqUYTzooAQVdyFzOkkUbji+XwBtNrAorGT8cQoTHelbeCBuoh\\\\\\\/wu2QjJVpqq19xE\\\\\\\/fygQt3JD+juLPxW75XLOr0rGCCJM07usiil9mGxBiGTArOTggQdVWyNkpwzC8eMtu2F7o8rlHz995m3vwS68t3At7UCe5yz5ZJ37b4mzA1oPb\\\\\\\/Nxq+OKL0CEID0SoTZvC3c4xsvZsgdxhDW1JOmw6nUd6gU3imaBgC1pgF+bBa5AaZGhHLXDTBOso3usXCZXTQxuwxZNUWNJ9ZEYfFy3wxR975SO3Chs2UkrJ7wT78t94LI2kbQIhHXmWY688AbmA2DoBaIVYyi58hmrRqU4je6Z9Xv\\\\\\\/+Nfb+mqp7faUeneBcmzWXRSq40K1IFC0Rn6r42OI9PB3UO5OqsLQKX4\\\\\\\/MguqVp9wpaWQ+3TFIPrycsOtFhhnaGl32t1DrYMqwADYy9ueAlE4NEERluRf0z9LRm99zSSKbDvNKIIPBAp2EOCHEJRxm84\\\\\\\/WcjNv7ubUbIop6+RuAPok5KA114GGYji846\\\\\\\/9jVw6P0GY1fGqVoSxwkazLU3RGASNUVE5urndGBSnWZIMIAK\\\\\\\/s0aqViz93g6bev9s9K62yBGzX806Uyc+a06A1TUzxDLbIrNkyYfGSkaB7QfpMLq\\\\\\\/LWSq1LfbR79iV0TjsaiSmiq+Bj80BEbJyLpZjsLHT\\\\\\\/soQ17\\\\\\\/8S\\\\\\\/56GrORZcSNnfwINevcSGXU8l0oYawBD5tsXO3ZnHeXohm5AeR24gNAO3KAEhtSHblO+wk3LnoURjhKWDdvktKFRkQevGWywdKMI6msigRYSVpkB+9SnSKL0XmmrihSOa3nVDcqpM92yEFya8UyYmMQ2DHVjnuzqz6GVbnjou4i8AYmDWqx3eMAuav3GeRknQ8f1itpuQpV5nP2Jyu3rhZ3Qh+f3b0y07i7EcVV+hHteG8aK8JvAf3k7VrLTmqRaba5Tfp5iF8\\\\\\\/7lK4f7MTi5yqc\\\\\\\/iSpE2j2SSREIUMT\\\\\\\/MQ8S44m111CtE\\\\\\\/T\\\\\\\/Dap87wGwW58O6kBxWxWTQAi8l1PlhU7FcY7SYo+P40l+PxnQ2HJuCbOoK+4QZRE5yY\\\\\\\/1sbO7BuiCV0qGQe9Nqucoz+54PmKDcwGnTvIiCE45q2r0QCxpX68AcII0c7SBlvuPwrSVX67PzbnoVrBCEgNnK0fB8koJ627lPhyW0dDye9GiALmPdKm4HbETCY2FugBXEHgapyMdZxcJ6QzIQlCVaQ6Y2Z\\\\\\\/YejnuzwjvzEWeD0rPF4wsMBfwe4ktPStEaX69eNyEPa\\\\\\\/W5jHZsRKHToWsmvetpjU3FgcUSBLgSglYN8CJw3nVJif9NAaGbGSlaex30GmzAilsvEQbbzxOwxoeYhe4VjOyKPlVKh+XxvRMb\\\\\\\/d5meW\\\\\\\/xV7M6SzAZgNad\\\\\\\/Nz8hd8lkohltzCHX384qDQ1a9goTOBybFljJsq2IlNVfuY34tFdz4ksm0O+2SVfpg1D9yKTk7tPbSsb2tzEL6nqdZKsSTJy0Tv7Z9pJAkqmvrw96O8qktadvVZlbaKiiOJal1knJn+BZ11ia3YUOXe9sEH0Migk2yw3Di8zHCNBWhR\\\\\\\/QphRSVEjZaWdLxYdRn5G9ofaQrbyNNPmlnBj0WL65EgNGiwDJogAzkB9nRIjm39EL4cil2ODYA4LIPW4KMvaGGv35\\\\\\\/YURXjxP4nATu+Ifg2QKvDrdLOkXj2\\\\\\\/dFkXDRm9Ny+clQWV0mvtuVxvembXxlpJeO8YRJdR7OboIyVwdEd+1ziChEXW3wFVLmVItpc7u8mt8YqFOKlvgxARQPkcZu1zgNXsDiYeJPHaUJhAfjA7U4TvWGuTMJI\\\\\\\/DwX\\\\\\\/ZsNJhpdrTe6pdQcuj2Jzq6tcxiH9WhaS1VNTwFTEaYolPo2HGfeoMXt6CeMrmdeUvyYXb5JML7C0NTY7VCcUgRoA5c4SekDaZ8WbR\\\\\\\/XmC9hdc4JVn8LsIHyhXDcmdICDHo6Z6j67xL7jamdYa7BClksP4YJAObGLBFb8Faki5oy7FsJ5a6s+0Vno3rzibZAlb3RpGgOUtCozmm51qH+nvO068VDfRZzF5qaeHijXzjbaH0iGoodoOCdYfpTTgFaKVFl3wpFwMsHapWjXxNOuD8lWepFE6hfP+I3TrdDCXzm+Il5ttrhqcLFNalCl2Q0VOzfiLqhk8FXslYk+Y1VoyHXpn8eIuoKM40w\\\\\\\/ztiqIYJQEVXc8xj3wXP4GYOmY\\\\\\\/8bWSdjXTn1DzOq2nKTQD65MIZxsc\\\\\\\/869ELOwS8xScCPaMf4Y42eh06iMYYPru0vQc1CcTcKsKB0GBVUc1dI3TH0Hxu5pC\\\\\\\/+et7hacmHkGVKQ7j2ifZ2pjhPeqgoIxO+ok2oTLRnF+u1cBWuFhaNCW72WnGCeF8nDImxtWuaM2j1ULNjWeL7VPxMYrRRD1Mbv5CjBpkC+yoDLDUvUu9QTi83SdgvrKJ3\\\\\\\/AvoaCsmWM6dKP5TXnfWTL\\\\\\\/pybibWRsxtHqflIHw5msRF0ebgxDKKmMSZrEX\\\\\\\/cre6nvw8yzhAe2a3Cup5h+SKvNAC1+kbnOC8d9vvFra5jr+WUoLGxetJiLxNse08aTUELdlzSriAa3vr4flgqXhbktEcktPvcNfpZ9ygCzjJSKdAPfSEoZVcJ7o4aboJpS8L+8QpvLuPjAAerChtpdew36G8YPO0LYyd3DrFt4d5wYQM5su5nkExTMJSi5AoaRJgP0rdQQN5uWtzReN\\\\\\\/sl+bHgdXNuC\\\\\\\/s0IwnlwH+Ilr1gIH2jnWV9UudMbLksqmsxwwBfJZyuKphCAtjAba0qjcGq5xwNFE9LwiYZOIOjASJfc+2Xi4ut2r6vGcW0zPCm2iaj4lMJplQ7gk1zJc2Xo517bQgH0dDtj4L2yGt+CV3C5zPAJYjeZNkLcfG+y0GD9ClxO9SCwNphLYpmQ1hPX25qC6pVl+\\\\\\\/31PWwkeVE8GmAQwz+Vnc90FLcZT8CK17FWyvVMvPvXAOrIXDlNblJDUZ6\\\\\\\/Tw5iD5Qoj4ydwnV195FGwsDdmuPtigeFWECTGQvlFbRFc9zyYQoTWvSxOgb\\\\\\\/WAVhTg8y8DVHeayCsxcL5cJfB5SlTjVcqx+sh+PFbos35x6pY86TtX8SvRczHxlcQLefb7OWnhJKCIItUb+YTYxa6p1JqnJf19sTpdW8U1T5bnxkaSMtQaka4ANDlvF\\\\\\\/Bu3qH2ay19j\\\\\\\/CKBYfLrmXjHi3W4ATOrFkpH9Y6DvAguvP3OWhi770i2R17Ps1hhnhT\\\\\\\/O7y9H1H6bmjfB0GQOzMyghTvHxan6dgBY9Ui0klalrxwRivldelGJT2KobdUsnh3JBTnjAS4ZMVKkEQSDVxSS8a8AY+xeHrzTVWlE1cwZDE3fHydg7uubZWvpllyZQ8YV3x\\\\\\\/fGQkcoEER4uXTuyE6tntqbStCVnhYhFUXtc03ywT+fuS55J9oXcrXT596DpN\\\\\\\/ag2h21yVM0miLCUwNDHW9AvMV0zwtW8amEIHj4SzoVFrXjQg96oIzKQajKt5amyLcsVSW6ycthTkbiShbKkJaaas80YmE60\\\\\\\/zWie0oqGw1fxAbwfH5SQrXNhx+a6rbz5fC2PYj9IWbxuDidNQs\\\\\\\/ga69FNZbqoJovtCIFb5I8orvIRToyGYROxyMkpAN5qaV4MSbSWBVQtLMC1OBylgy7gjDMDXTM6nyyt7DI0bLkFcCYFofsUqMzpYdjrEh4RWRJDCKsV2dMHuwq\\\\\\\/4pgZYYrC9ZzRo7vjhxDCPgFyVKAqpTWlpwTFcVlWnnfn97GgVSfSRCorQ2aejXuI85FV1ek4tCWLUSOnSjgy6thhNELzK+iTCGGXNxZKsga8boBh2vxUkHFR47z927Uu\\\\\\\/OMA2HaqY0LALjzkmUcDesfl6EtA5g6arLest2HmAxlRY\\\\\\\/M4CaCmyL0\\\\\\\/I\\\\\\\/uAb5aERwN6964ZE8BjeSGg08PVl\\\\\\\/+oamHuXEEr+T89DgY704e3RVF0IlolReCzZz\\\\\\\/BiXjH1+LMTuIZ7u8eRxIYI62vfKz1kjGVsCb2KV57d2KjzD4OO1I+GwmI1+56pKsW7MVaFVyzNqlPQWKIbcItddrO7rXbCN4GGfJD8BGH\\\\\\\/CBzA4PF\\\\\\\/NXhtsiZEP4oKMm9PgEPDT6cIEZGkWSGPHC1xwPOS9OhZj0cWI3EIkhIDvlDOvd6ge3\\\\\\\/Bdy4koOvqDXcCLvmv0TrGqe4I5Dcc\\\\\\\/CDl\\\\\\\/qKaKIEMirEWKyuC6ntTAeLN\\\\\\\/kcnNoDCz0PkT9NaY9MsHNkcxkWq55B0CjKRFVcYR9CaBNpTE6XoZd60K6EZhvhc9Yz58ENYMgVKHwtBjiKsxOQ6a8ZZdOaNm3LgV\\\\\\\/pC21rz2OCf4xTuqBDnaFslgMe7d0NdZPmkt8uD0dBNnVRx0qeSsIK\\\\\\\/Wb0x7LSh8gBwBlRzx26X2luWc57G8COjQZnBTRiD3pKFGFveW8I4PJy6CySFqQvoCQZf8vvE4R\\\\\\\/cCS25Xj2uvSBztYf6FHYwXT1RVZzZgFjoNGM7EM27l4VxBqtxmv\\\\\\\/cTP9DDfYJu1PF1OqdAlKrikKiAu02fxMA8SGUFVq7iVkR8+wIzXNJVhf1ix3N7I8TTXoMf5OTdt3ZXqPElM2xn67oQolB9Ce8f0m0nR+6+ODjfW6JoQQcoecjI\\\\\\\/GRumQ6qoFVRnJYXeGbFioZ68mKZ0b8XXd6t22YhAVNoqwj\\\\\\\/j8+AuL0hTgj2MnER4DM3PeWX3GWhvP3kl6TGxujJvg4TKHCkp09PPhOe071CfM0ljBedtCNPL2nCJIXV1QUeHIkjYT\\\\\\\/NpxmZRz5yN1GRb+a+ZkeccD17GItc9St5yq7qLGua3jJzf8bJKuKV3YqfFMEuZ4H+ynTpSUiXcBr2bZAgen9\\\\\\\/VcJlrh8DQDUK79L1bPFav4ai3G2k9153kSYwf\\\\\\\/R2SJcPFuHjKtvbVD+frqu9hMyYkJdR+etOJaXM2FBGTEJfFRNfPMY2o3ESY11eDLe6SdJLxFjcfySZkn6FaMrqqNYDBr2T+13Rf8rX9hPDrGAU0LLBPjPZ5G7UyVHyftaM2+x2cWZ11T\\\\\\\/0YlDOTsqcKystcOEsHPVZaP65RZvoGePlXdliSDumykZPqtFf9yQYl\\\\\\\/4mlcXwci0EG4vmbDeJD4tTr\\\\\\\/Etva8s30QP6ZNGNDtz9CUsWb15Z2cNxqaroKf9G7GhmzpI9fzpBND2jqIj5yPMmIT+J3FRSX5zwJ6KHnH4zJnUBqX4qRZmd\\\\\\\/Ok9z5T1dnzn+9qirwgEQ+C\\\\\\\/5sbd61uIn57b7e8j5P9SU8D5pZgK4dxG\\\\\\\/zSzwtBrX4Tk84S+0nbT1kHQOGPe1lug0dRnTVhKOJWjy1mJgiaw3ac9daSRgwQnN7hXvrzvpy3kJdGYPbgBEBKhs5CR3tgVgcs0YrHUTNU+0hfumc9M\\\\\\\/y2Ke+nXKp8JvGuwGEyRsawxt5AVhSslaDikWfKQ\\\\\\\/yXqQ69wEK1e5vol2o5e38+70HLvCxIy5p3UszuUwPrcqL35LL7OWLDaVHGd9r2iczXseH3u7NKF879zk9\\\\\\\/KcWFAx7MbDlibeKZ6J4potLtUoIXfWeDP28k2HG6IQpEkLKzfkT9L63MLMPoq95UyZeQeHCszgRIphIRWeJY76C3u5wu\\\\\\\/di5dgK8xC6hzTbj1\\\\\\\/uPFQq\\\\\\\/kXM\\\\\\\/IdQqaPWauojrJ9kmV1YUVUA2Ueli47M2I5IQ7cOncq+4CLWzFKfhXBZMXFzqIVKZj5A3Ar9a1\\\\\\\/HhH6y5UVSXOyzSn5bNmWg3mqWfOPVHCQgJxpcyP2GIOisPVxsBTO2pdc8MvpOX4TavnyMloi8UjlcsmJFt7Njz4UKj4Q9JPCaVmkeQ9\\\\\\\/6X31OLNH24jge16oW8E8Ma95ahcl1SVavI48FQ0fYbZx63U8ETAoopFeJifjmjecdt6gWsEnD+kOlbi\\\\\\\/cF5c6aa0XVD2ZhXLpyGi6A7hnd8oK3KS9FRVBBS\\\\\\\/xmMn\\\\\\\/0hZzf0Cb9\\\\\\\/\\\\\\\/est2DfhERivEKXtatf0xu8WLqBZ3Boen6wU9\\\\\\\/2Ybh5rcNFthI049BB231Yl5MgKREBXtiT9dxc1ulfxPfeI6bS562CwHEPJRulDFXnt2GdsQFxM\\\\\\\/lsISb7TSJlyOf1KkPlQcs7fevjNaN6IvNhjk0Nw7pmnkVYACu7IizI6sT+5GydHhiBjkvjmvNDzE310NNgfqDZlkjVf5OXM4fy92OEeqYDQNEt4ncEsbHrzgVI1PYG31TexzmRo9XsNyWzGYOg7MOKNdqtZdoUTMf9J59Atoou8sHcXi66vMHhmIHGNHN6RKgaEKT4XWFKaj1MFFKIsxxVrLUtmC6b2gb2Ep\\\\\\\/XsiL2tnzmxhmqRE8JT3hH0+TvhHQpUljfdZo75JZ27aj2Kr3GgfRjYPGAmernGqg1y0MPIORkmyAdNn2+VH2kONlkC+Z9E0ezPADlllFDUlNAnZvvJ7qN3Y9zD8EEeZUSz0RI5WattrBgyFwPJeQB5kkl4DgarJreK3xNipji+bR2FFAbxRp4vT12KQF6fQWVuQJTSq3MFNeIRDwcY7E8M09+X2zoiiqTAEdtdMVezc1cLWo0hNoKcDoQMzx8EKhDWJXtIyRUxeTrILefJzWxQlyLLjyB3rCuqBIp8tDWJKBTRGl0QLcDidVc+thQ5kXWGM7tuZsfDB353EEhABnJTXPvMnEYscNBXEI\\\\\\\/NWj0LfBDLSCuKdOAfLPY7drwzRo2Ig\\\\\\\/7PyVA4u4uCpleXhTZYOcnGGBZy3au+LsPHTG2FYtBGstbRXg3iBqWbpi9X99HJ6UsXYZJVpnW1ca\\\\\\\/xl1fsUwUlLV6hkSwntoEkHM+N\\\\\\\/urgR5yh2gSgbecOJ+Qstoykgb7wJ7BlWI2Co9g\\\\\\\/bW0299Czp997Qo6tXPnLTwgP07qLmJExPeuolgVnmTw8L04bX1WwbvS9vbSXh0l2PleLINxxUn5J5\\\\\\\/RoKnoQ2TU1cJpIQVZW\\\\\\\/XCNVdwME3xema381cf38Rj3eq58QhuIy5nALjeFe3KB17mDXqCyumAmiPnvMjKuPAyibjqhqs3hjpUAPqFrTujj3HLGJfInasxRcPTIEWkVc3y4ZIAwR9+gkB8APALUAqscUCOWuwWbp4j+OTx20hCETJza2tcVv6NaDATEa8aqMrKqo1g8wy7pp+BNUwg20dwVnLc\\\\\\\/0eo00s+VrNQi1RTmNfrqTJ1cxv5SY3pFVyUmAMVQ\\\\\\\/rqqcyWBkonQqlLT6eoPvs5DhY2km+Lu+EOvas6X0bdV0DDqgyu4679AfwSpfXZJ71GfE6l4pAMMuovwfU2LUQ2xfyI+bId99\\\\\\\/axK5QxE7ndWEcl0C3TMymIEHBq7Q6V3tO+XwLIPgQJYwQmg5CD\\\\\\\/EecCQeKoZ8y8QHWZPadrFz59UuNKnxVbY9qqX2qB4LZWqme3kthzMA3LBu5LLxw2iHnYsR3M8evmoiLujY+cERkbyucP6ps4TNxhc4uacRbSVnsMfF6617OPFVA8ZbNVpSA7JSReUPPjAYY7EgIFwEtSM3kt\\\\\\\/44zSewP\\\\\\\/uY9VBstSugz4wMalSp4UNk29JjN5OFBWs1V2vfZAupyxFyvPYY8B83X53L0+EpBZqKvi5y25k1shnhgYiisI2uBphpgp2ATsPToVx60n6SNgXuO0pimt4eWR18lB8OM3\\\\\\\/E3KcA7aKPncqXtKYYrUe7UwHxlnP2vEWpaLeCbTCsYdr2WLTyYbi8AFS+AbCHCXCuzRTtI0yE+3+aseA1vIIORmkBOla3rAPzhNkWM3tDBVhebXV7uQJ7n9n6+s+60ao0ScuF\\\\\\\/36b6IpcrXdSOTTyKZAv\\\\\\\/QbbH3Zsj+kdST0w1QtHQcVxX2+nFZA8hNoLDkk4WtXLNs+4BWclRQchGsQtkPupKDwlrKuyV93+j\\\\\\\/Hr0dJoSs3yBs77qA1hRpzTwYuN5vdihcwfZHrraG1AvDwkmVNKqiWmBl2m5ri9XvsMkEeJFU5je4AsuBBwdmLVPGxV6JI0R6iyhQYtqRCFeJeTaTSKkjVjVuydyFxS2aSv6mGNPv+Jtnor9PObasBHdIsJO2xl3QLcOmYF1LHQlD1r\\\\\\\/eLva7zirwE3G7fi2s2CXccGh6K6lIYCsq+VA651tVIaTVd8X3ize3DtcJ80ywJg2KHz\\\\\\\/vwIsKD\\\\\\\/AqlDWpHyent0zowccZMpwGvNrrt4Le5TvuSrTv6h6GY2Hbd1BWeFUTvlFpXkquGtjmuL\\\\\\\/4r60LJfZoueGZ+ukSYhqvL+KnoApiVcCuy5obN5bEIvHCTnnIkiDNZrQU8048KwwmIwaZdeVp8KuwAcdBW4XnLWz8EnONWuLDbsHRg3CiIYaALYCwPRdUfT2Cmz2I59N0Lcbw\\\\\\\/au6UwE2W9MjX3h0PErknzVqQx8vi09FDuRJrSl1W2ib9pfPL7Et5k4jRFsKxjBFEdpx\\\\\\\/o2+U1mJtvu2eCDkE50JVwlGW46CzbCE9vIlGNgAceQFddK9k1AYsrehAVVDI3xGDX67cytdDnImAYL5HXjlMlmuxEbEzI+Q2BSz+ZjstAl5KVXnTK4rHiSHrD2qAoz48Dr\\\\\\\/eqhw6n6Gm4H\\\\\\\/Ut07W8aM6SoIcVnmYsRNnpXPZP7Bxysf2Vj0TxSmMPg+BYhC6OhdFxry7A0C4qUs47Hwb222hDN9jTGdEp0urBqWpsfe0Mk6RfR6ZcshJpQXlSeTXimAcZWz\\\\\\\/l6pQUjhWzgOhb\\\\\\\/SBdXYrBOFQ\\\\\\\/\\\\\\\/cD7vTZkhuOzilxCCdQwGJllj7lrn\\\\\\\/f7TiX6zfxSKON3MPgRlxUHLEKZX6\\\\\\\/nAWO5vrYudOLWHNm+6adSrz\\\\\\\/JlrxHqPFMRifh5KBwXvUlmK\\\\\\\/Lm9+UzUyCrbrbZMZrkm2Pkmj1pA0nVZRsoVN9rxElDgpCrzpBLejd31Wrba1\\\\\\\/2Kh0BQv9\\\\\\\/3l7S0pzlzTc+5zfD9KrP4m0wy3c3awkcApvY5I7NYwfpax3ckTdDiNN1LIkNo9l5vk\\\\\\\/b9VcFL8s5pvzAVvvnrGJjX\\\\\\\/NO8p7szjo0Ymcemr9GAIISr1J7k1edi6SJkEv1aENE84mYNXJerCLp8pbtTjCZ1RUoKbGIhzTS1Ha35Q8rJMepTkVJhEGXmJvAHCVhUXWZS647YKojhvZNP9ArvlnCUNR1sDocPJvrXBt03ewIwIHl13d3ES7mWmXS5W+Inex7exjlO56yW+GjisAEdkhg76H+5zpSUd0eqwcJXqBXTu7bqmjjcxGxshTMenPkt8dO75JAsGM3DDMk2F14mq1uLdltbuwGccqqA6pQ3pGwYgobdKHiH\\\\\\\/i9Jrzx3OWKhX26WHXwNbhDXYDVH+i4e8jQ1fLWuB1Im85VXbaUT7bN1EQCxPwGLML7pdSh5Nusx7sQk7a8lKKVPJ7Y\\\\\\\/7vvPyGkTXw4Z+yMdWp8Id+XQEpqOPbhB0Q9R28nnQo77StGE35L6Z\\\\\\\/wBoKSXE3q5tb4XtKWgisGQCfI0ORXX05wIMCm3htD4wfR++9z6jv77oAeHHuRwHcL0Uvd0sPhuWop2tNno0pYz0H2VMfcSazzFKJOHoIT+\\\\\\\/RsJH6ZOi7l\\\\\\\/oUt1qnqv9TNgtcczTavhE6S8mF93+ju4USopNlXuXVL+MC6mnLuy1Sz3SlBKR4ggBSJrmCHgQk3rImMuVYFdQKt6a2xp6cXI6gw6fmdcfxGTLl9upqCqdyMEtQLuT1OUwzxrtFrEbwxzj95uIMpwPyvoflJZY6Sgqp4Vs0aoIBYoifvnh74yuVukZoNOZy990AolJjpT3qEixRpUHMTL81EOydLE2gD45KUqp2wRmUit6aiYDhMyAOlTOe53VchuVMLZO5wIWJCCaKOHcnjIbUWsG3R0dficG+usY8kXxOPXLKqIpFPfZ1RmtV7iC6D+XJCE1bZs\\\\\\\/3f2t45gO13VjdXvBwZqIZh6sbqHOXEF4hW8nY16OmuhYbF6qUbnUvGJFX5TJH4vnp1tAPI7FXkBy5gPt2DVvOJOoY6A5ep9YQ6wJk8jkLXFXAlVGpVMMiZs9DGRcXbD3ierp4hJfUnUvK+gSvW\\\\\\\/YGNbHmf1nCPDn5xLpaULSGSf48UiKXxQovod4KLHOs5ojl0zSbMTPPMTcj+4QdjezTX9D9L\\\\\\\/t6qDgziL6CE7j6n+U8GdoC2s0pX+VNdX1C+fobhx1wj76n49zzywPetqFKLfsab+4puva12FfyRXSFPA+avmWJj51pLMT6H9KjAoh30n4IB1ni1VTHZGYcg0EAjuQAoanpa44aK115uj1HWxBzpaT3sAPjROsJsGOjAMno728igNyUZX6424WBPsJP6OiZDs2iNcInGJ8OAQsee5wCUWMT21144UUDoP8qlrKSXGAhM2YRnRjTUA8Zsnq\\\\\\\/HLDJuibliFAnOAqO52WIsm+D2ckY\\\\\\\/dzp\\\\\\\/VNQ\\\\\\\/6NBH2SHf7KjrOcEJR5QPxK0\\\\\\\/IOZxzV+btEKq\\\\\\\/JtbL\\\\\\\/t1\\\\\\\/pZejbWcSKtSmA7tQcM0cGiZPPwlF25MJuUfr7FCiBXpnMhy+WdPGq9yj6OSH\\\\\\\/64qrm0GIyMIxmNcDl8rEWujnDa+0Y4tV+7RACtjOmq0zVDw7tVHx+djRnSYDjwM21SlrdqXH5bzijUfZhxFpzYeVg8+7d10V6k20ZYuMtiagElI7MYT2Q0MZPr0dWVwFebS5yzcCmZo4dsqFCBzEmFYyHmU6F57ckevAMRgVzxe\\\\\\\/VcZeErHugxBCz\\\\\\\/6jCElCGKVXPr9xcGLkDRELO3cvpUYn3EJ3gtynPykPcYXafRFRTytJ6ugxu2JXIDVHL\\\\\\\/by4ELKmCgUApHBNkmXg9Gq479Tov+6eURu1U8Dr0frYC4Fb8zNsvjbD88Csl\\\\\\\/8gfcO\\\\\\\/tWNkqnBh478uZJvXXHbYttaG3v3KjdSosdEy87HV58IaOgIEuIOtObwEmBvFDObYn3XI9F6RsRsdmQGcmH4xDlZqcK9Fge1cjB2D8pu8+UMLDma3q4O\\\\\\\/N\\\\\\\/IuWFkuBEcsz4MCE6TrzvO\\\\\\\/AnHBw1rxhHU+Uy9gNHXo0hPncR74ZoHs4HietXx5z84P02sQWV5uOiYAMbLSJU7TyBOikt9Y+h4eIwocOLE6pSXRQO5AR342flDPa\\\\\\\/ocU\\\\\\\/g8bsrjx\\\\\\\/UQyqcG6uOCpZ+Dv5yJl9R\\\\\\\/D6LZPRlR+2blnG7DAR1dbZdBpCXlAVNBgPrORyGE9yG6d4xHtCFFpLXtRVsSPZ+FDT86meMvyOXTSbew\\\\\\\/IwSllRFWKor3uGoxWQK9CtOauoLvUeOlK+HHAG2nVpnBtD40JHER6hiJeshIl1nnHpYKWRWkNVdOBuHd9scoaUpS+gqPDeQxPp6j4JjPiDUy8WqDhnx2BKFYGv9FeaB3OOKrldiDBcDw+Cv45Bmpa0ZYR4Hue7srO8RW\\\\\\\/B+UbOMv+KxyhmPg5VGUGCrN1Ggox0OM9NsTswSNHpVqZMFjQWFbTU9RkL+jzqCnz9T23d1FZLdwIP6AAqTvO7oXzv0sndZp4MRTSeP0dlYtAZnCvJT\\\\\\\/WcjVOLmlS9cXROFcc\\\\\\\/4rbawXM2iLXkLI3GRjHbzJb57gDTOSAh+fbpqP1TR1koj6ycqOJ+CHnQTZIO1rsPV6mDz6Y5ba2NPxf+JGAkkItOHBu2n47mI0gdqVwM1D5SBBbXQ82OSepSg+JzsXJluScKtbZeIM9eFOGlSH2tGp8m3e6G5GASWDi\\\\\\\/kRfbcLpvxO9SSYbinF\\\\\\\/ehqeWyvEx1+rFgkNrQFtsqTmmRuQQsZqotM4tA\\\\\\\/nRBItI8mTYkrti5CxsOzAChgRjqtDx7TZtJ0\\\\\\\/8BflMH9CEPIZ6A0EtbEPtV\\\\\\\/ZIos\\\\\\\/DDW6vmaeFmht6cbh\\\\\\\/c4epvblzgPX1I\\\\\\\/bSkwTF20q1l6whtw+eiW0gJg7FAJJ\\\\\\\/bFfCNsquCH7XgIEVaE5l3HJmIRwuNL6kCg1VIwd8t7cUZfZ6CLQtoDplldVnklbEYy51L4bCMrlM+RIg\\\\\\\/LpnAy+WMbAZM78JNGox\\\\\\\/P8VhqjRJVdQRAqUgg\\\\\\\/rh6zpcc8QUfBMNrnxIhY7xqeGhwcbx8PcdEdiZhFZCKcVn2p+NW2Jns8fgQTiMECaKRtL2ASWwwl+OW9a\\\\\\\/ADzzJWFCeMlUdZeAsy7syQYdzAThaJKoB+FSfylmS2MNFnmeGK\\\\\\\/lfURAyQT\\\\\\\/dvgyebViy0Iea3YG0ISeNwwEsUzf1eb38Vj0dV2ELkGcRHwxx6SJWi4VBIC7M4nHFjX1PcFozH6mXiph6iA963Y3D2TceAONAhQa4pa0d49YbY5BRDcORmrSjv1oMYhViHmSVX636cT8NSEG7m0\\\\\\\/Hq866HDmuJeE9jS3wBfqIiC65ffbPyW0yTZj3lTn2gm8VIU9anbkx4mbmFp47u4jcIJJhZWPKVPA5jYcabuUpUM8nb94Ptuwgpr\\\\\\\/d344mbbjH4XDZ7dnxee5EoBtiy5EEPow3M4lB+O8hdqxs6eku7nVkSVudduYggcz\\\\\\\/VcPgr45Br\\\\\\\/VcJnyNN9TfOLVhmDqJgQaHhwn4PnaXOtzLZQE8JduMVeifRq1+aBCAoxTKv1Tf1h4UP8DepxGqFuOAsWcmV\\\\\\\/O8MfPkwNqczPG5aOgJs26PPlSIIb\\\\\\\/hg9FHqYEaOM8RZxPUuDZGqPHdl86r7A1cz9q3xMaR5PDzDUiV1SPlF1EDlOurUVJXJArV4wB2YnJtg6SZpDTpC7s4Qv4+GKunVODl+KLOm7NdYIEQ3OJI6jHUzE6guptgK48rOHqfhYsxfAaD24eqS7JZkwKckh+4sxOFpc+0g2CE0v8YWyUkDC9FQ+sClMUBynupTcJ6l71GvTX\\\\\\\/9NWBL1RsLvQJ\\\\\\\/kh+6UaKtQqIdDeybrzwhQJUM0Jk6K1a7L+izRCoDZZ2aISfS8tAf+Yq3uyxUIN\\\\\\\/+WwA+NdkrsbfIB+Up7lMfU\\\\\\\/eeOAg8BYZPX23U01Qfaumjo5Gv\\\\\\\/6jAzdn2Y1rx3XhRGsrpdH2IOiSOn7jvYbTtjc9lxe2t0v8u0\\\\\\\/EQm\\\\\\\/AAXkk2qUha1ZzCfF6WfHzqw4Gr8bRMR2675litmSFzVdDtQVkIcADok9Lt4WuFbyNaX91B5EP6tyvmsIOWXC6yl4dVJJlpNFVoyKuLOXGinRTp7Kbw\\\\\\\/+MSuYx5dTwcEMI0NhFyIDkve33GXRJ+wjX3lrmCEF1ElsHvq77MRw319R79A8I\\\\\\\/HpXTeZY\\\\\\\/Af05WNbeEMbIkhzUYRaurmPZdyRvScc69SXhACm6kz+LBkjlcq+mo9Tn203jy5Ty+51j5hoEAgnen6pJQl\\\\\\\/MsKfQ3Iva\\\\\\\/EZnyVDxp7ZkI5HzLJxo8wIT84gPhr6WxrsJpCkxpeiHoFwSnfL3ixp1ym+NMf40r19NbPcmvDMqX7cEPNC6457TI\\\\\\\/rN8WQOQwMpu\\\\\\\/U3u42GvNwk2E++lf6mGV\\\\\\\/XQtd7r0yXAKE0guy6+Em556FB+eVGjA50aJoscq2d9QSgXhENE8HFfUkjLIkn4pkOhHIpVRGnQWktboXpNz4eW\\\\\\\/KWpRzsg4Ih1RZLm8uo6o1ATgVs1SkAyMZSwfO+svnPn2R+Zt56Efj\\\\\\\/C6O5PkVcj2w1OgOP6laTnnSY+QIiXspOrX3GaCJkPPmzZZ\\\\\\\/S4GflfueEzMDRvmw3YvIjXY3VVu6NcKHyiP9a90htdbPCn7+6T3N9f9qt+Q\\\\\\\/Z7a\\\\\\\/jDZSLjK2F5Z4\\\\\\\/Hdh5PMys02BF7OH0cjDk1U9VBbsiJNVQox2YQZZd1z5axh82lQUxVDj9O\\\\\\\/dWvtwnExfI8IqAUCD9ooRcYaCT4ISP3OtNyP7fLIWVOf580P\\\\\\\/S2AL8Qy6tdSIctFPXJhnGWXcFamEhBgDBW5s4wtkp2HLMSdmdrY\\\\\\\/kiG85JWxV0rrutRFegsxOlsS5VNxAWcDJe4m3x4qgY4vCM9UGqbexDkmw2LGhCe\\\\\\\/E5laXTy0QjMQSH5vUSKwTD5gYf+7k00udHcNEOMoFNdQGQFVvpPhaPYoejOF9RZsC7PpTEAnP2hf\\\\\\\/l4yGwt5jcNln36ifI4xa1nrtXCbeqn1pS4SSbA4yBMR1tL78CfxbXhg\\\\\\\/LaQIM2LcboLhToWksz5HA1O2SWYs96kpSxTp6mR3+Od4qDKqIPPGefNF5r8nY2u57ZMQTfDNaXY0knObiCUUOooHPmqibTBJ0OtoV8e0iKUDsIj24Kv16HYRGd1ueUt2Hsopzmq5jdp3OYVDUlJtP65d\\\\\\\/8D03jsha2Wnff\\\\\\\/Ugdrmh8+F48hsmlY9E+1b2Yc0\\\\\\\/MlV5mSiMcWj8gdO8QyWPABE+t+adCr+68n40oQyceoQeAVHoenJ4rcLeru3VQikXuysZUD+XTz3NI2qwWx2nJevgu8KN\\\\\\\/wxIS3eZicXhpKr8onytxZhQ8V0xz9wK7Q7BsCHy\\\\\\\/s6fcnyuS9oE3ytLyCWy8mr+s3A1DXAVfv4yyZE+q+WN4yvxMssXfX4KlOvKLyqyNwroiYfc3iwWDV0lmN5oegRucB8+5Q2OwR0WTya8OU3SbDqkLCEPG4gbzvT09wKhNaCR8CCW2FqgP0zNGcpZsgUeNr8v8zp58n60mR+khi\\\\\\\/eNU1YkjtVt9OSAJ\\\\\\\/DyvUsQ6LgVujoNe5NG+dnCC8Rez4TWIOErYZs\\\\\\\/1K7\\\\\\\/xKyOoHGPCjWMnwd2nODzGrK9pI+BoJTRhqCJxLPfMWmolIJo+gOKEwYB8EuwLvtU1pK9Lle6TPmLZL\\\\\\\/J\\\\\\\/SJ41QSIj+btQDgGajPe7BKRgEWsYVgqxLlK5vMDX78iullynqPSQmH5FxN5jilljtOlVyVT005iXBL+mSQT4mJxYId8TA6DXZ2MwcK\\\\\\\/CNineCf4AwJwNOcmZSqH6+1AtHDgvKvOnOa7NXPqkuwaLt\\\\\\\/gGjPQJORxW4hzDO\\\\\\\/40WDL2Hz3V6bfgcHwH6RkXsvRZ5YYSJurC3HgZwR8QauQGHulApcfo2D7IdPnDEoS8ALRyeFUljAcBA5BzVGKS5b\\\\\\\/jfE5gdRNHWKIqXiGnDk2kkpnhMncGPu3MhvPE2k2njkPz6qpBnzPdf3w8dZ5TF8agmfK6UJTc10pFluy20FNdJXCUK0K5mVMI1orUL6AopK4pTXMco3s9HoYbmg8XJVGr0r3B6tBWFbOHW03nGTc3XDnz9EplCms+mQ0U3\\\\\\\/kkHJXoypP8iywbij3PI3fBEOJWZ\\\\\\\/qWq4GjuiWTKaXMIQMVlsAZ4PkHCExPZNbuKTB3NScjRE85Z5vjd6qLH5\\\\\\\/g22vuNK1qMzOlnpIRLV\\\\\\\/f8NRUeK5hJ7fUt2dccfIPIJ1+GTx7Rf4rsRAWByWukcO6Egs3yHVeqztzycCmL\\\\\\\/cux90Q4lp8o3h89U\\\\\\\/tFUBrZo8SsCC1bRAsoYGo99Tz2\\\\\\\/BT8wLsAAWKClrttMqmSgHNslIIrA6IQbxP0arR81ujLMsxD7iP99ijN8b8LNhXN1JJ+M92DI\\\\\\\/ypKRjgXoW\\\\\\\/91yqLfqYaXL4iL4Y2wIY\\\\\\\/y7RNsuEXuzstu9KXZ8jn6djlu+rwxvBEIMRd2ndBa7h7l7b\\\\\\\/33bt1la7qYn0Tan\\\\\\\/7JqWkIUFRG6txgi84cScy0SxGgyIHzSdlZ7e34Mjm7EJYo\\\\\\\/q8WL+qkQqwj8I0FtL9yjv04rJVt\\\\\\\/6DjsbvqqOfXyuIa+ETu6jDMw597lj\\\\\\\/gp+aNfp\\\\\\\/RNo8X0GFh1K5aBpBPnTsqSh7Hj4yVJ38e6njIjt8uXbo8F1dfxu\\\\\\\/Hbbd3kaFbDbdBEJODBTdKsxrUUoQ7EUSnECKw4C4mn\\\\\\\/pSuFoK+Tp6d\\\\\\\/TvIm58dyTdHclEAVzTkMHbOMK0R8qWkekEKGPFNXTxIB\\\\\\\/EK5JjGJnNIZPoTG1yVKd+JZ+CB4F6Dn6rccmurg8yD8RhiAwkQGpJ9CUTYCGImUsajzdnkzBtRWzbmX\\\\\\\/4Zk2E8Opf+Qp3aaReXJbtwAkHfdbALc3JPS9v+glhj\\\\\\\/H20eqonVs7YjQdu41DDzXEyNUge3L+L3G4hSYsqniFzMztCCcqEM\\\\\\\/WSOobxnI30pS+kYVwqvpKLa0VYLwjk+qVxKXQnBoBWF4u3lrlpZ48lqwjwq2fc2Gzqxm00YjC8p0VcHamZw7URaBS5hn0\\\\\\\/1HDsgRdB4KXLTMf+ylRj\\\\\\\/Y7+sBPt4H5jsy5dC5DPw2nJO7vguxWdfqBZ7GWcVRox7XcAqFWY3Sp5YOtNZt7q029\\\\\\\/Ng9+B0BfsibSY2SOu+M4vQl1d2uLwpHI8HShOqCPIIETjxwSoNrSuojt6Ivrn+mrFiuvWX2WJFH1IbKcf6xCMRE\\\\\\\/HM\\\\\\\/CXTs+1h2mNrQ+mSlxbyLgPjqkTcjHrXYDRyqV6DixpdLD7IVVU35ndO2wSFU8z69RfcXwcoXXfXMUZzvJOcQCxKL2bABPI4xyP\\\\\\\/sQ0Lq1nqzNuP+leEDnMye1osaQm4FOBWr0GWmUy7N4lPHYJZFRe9UpLTRQ9F1ZBJrpGyzEMqZOZIbBZ\\\\\\\/dCPVvzyiCX2lSXdlz9v8taLnv6PDRagSATTBa4dwYyDvkLQndOVCqve0Itss7efKHm9CzaPn74vrIOsJ+bzExD7UFIl+xR38y0ERPOf4wRYHFrfp1NJTeyqU6uHnU8qEZCnG2yHOOrjPfv\\\\\\\/T3z8s9Ow+1r6SjLc6KDmsD5+uQU\\\\\\\/++3jsJ\\\\\\\/4QriLcPqr2CGU6Bd\\\\\\\/kS6kJw0QXvaJ5FHmhQOi4uGa1zbUN+8y90XAAGKlZ1gCUMMbUsD8F4zO+e8wxRtS9iUnJ3boilYH0RSTtuI+d9b2HyR57XApGHrMWygY2GzxQk9pRyrsSNCFp8xsjkaFDHtQ\\\\\\\/CzqlAxSpu2CFgHSG8+1QMyrt55MkTG57XjGD34wJKBTvRw6TG+XHnyBHCxSPD2peUclR5Jm1dEafk0bAJeT2v7iTs9bgkFW+ii8olpkxqdLkbv0IO2eihdzDt2RV0ZHH0\\\\\\\/day\\\\\\\/tgmJ\\\\\\\/Jc0JW0+MbggBlo7w8T12BgxvXcDypJqQRNxsK4ECQsgzMX6LCu0lYl14fRqGgaL3G\\\\\\\/k01\\\\\\\/gsT9eImeLaHqBgV+uVkybz49Zgltsm\\\\\\\/Mu5o66VealbEcu67LaYfP2ot7G6EUfSWD+Cb9ZsvrBrAghYwuMfRQizmXk2ZC8p7OxfXl1BJ\\\\\\\/XcJP8wyfCblLsz+xmhjulRWR5sMfUF7F9UGAX9j\\\\\\\/yrsc50TtNK5+9nb+Qgor4uknPu7PcIkWRgFqGMMeUYnogJl5xDRfpzYf9zu\\\\\\\/5IihUqQWkSrHu\\\\\\\/vvknTSVkDDyUjI\\\\\\\/Hr2pi5VzS5yj6NTivQtaacGrx2UXHcuQuH4PdBae1Dq5nwlPGkZjMsulDgNJ1L624MoLtDnE1J0ldFM+pT4aucfVYdo5H4rqDxznCQ94PDfxWSRCRj9Hf9wnXyuM0v4e72Cr0gnVJVCymYb1qGHRAKe\\\\\\\/cQ9ojWvrFkrI+3AlMkiZKRoFvsdkd5Pl0\\\\\\\/DzkodHB6XJhpxJGxPl6wersjocZBhsTQX+Bxt9Lpn9LvMSiL39icGhxFg8Gu7vPAWT9u3sseal+HwKVe7w8Q+GHkpfWCRTxuvjMUKjKNfwKL0jTMk2TaexBW8PYuRkYpsblASaN1zmCol7AHAzISX+TyaOwt33tHT9ZAOBjsumhjSzX1U90QB4RQYD19Cjlnp1nkeQG\\\\\\\/a6\\\\\\\/aBpzWy+OAzNCkTKXdmNCrEFJOBkjSNnJk8nCQ183CxlA1wHrcJNVJNyK20EbMxc+Rujpv15Bc2qnpLUrI7\\\\\\\/lsr7f5Agq1JnPaSI5ZTGC+xbT4bHPdeRpEpp6Zz4dJLFkUVsDJSWr99OaRP04dsXA9plEuQzOZXSqWjhnzB13xwisPsliqAAMpukD3DKkrXETwd1A4A+SL+mBO4D+SzFZoI9wGR0M0uZ+H3pWIrMT25WoJO+JZ7B1YXfZk\\\\\\\/GTMqlY2xzN2zcIFjPqbcPSgH\\\\\\\/KgfFIxLMIcXh+sEoyerkFPvmKTsLiYFJmU6w3B32xD06SDHRqpz0Xr79IyqAR+mAyz5uUOKyZj0B04snAFifQsCg6E3s5o4pPJC2J+fs0hUhSd+e2g4Q2c8TlHBck9Er8DIWCSQaK6QtbOxQiBBLb+3wp54AH6AzEd3oPMQWOdHKEJ2dLsCjJ+SCFTGnb4zv\\\\\\\/pjSS77H6n\\\\\\\/aDotjGzlCoXI7yous5xWPxLTJ8bfImfQjFfp4NXEa154q4EAL522DMnfyt71oGR3vnl7nogtPRwiYcATnwxek7NrJ3n1DlOQcAv2o+pmYrAulY6r8xun712dTSVuxvRcsh00fYxlkAzzpq8j7IDWf4yjW584MaMbtFZJt8z3xdgB6PQRv1WD94rETVUSnsbdZoz0b7cSRC\\\\\\\/SHS8CRH\\\\\\\/DCY1EcTviCqlyveGKxbO2fYvOkEOvaqsSJcYOu261oD6fD6JiFhzfg2DSDakfdWWYMc0AEIsWMfwkpU0l\\\\\\\/eyN+XGGp8Ch4zA\\\\\\\/KUDPOmLhc09VZuvMT\\\\\\\/+5dBoIAt\\\\\\\/5ZIxMFYziZ4bsZC2fhngai1pjefvilrbFwbVdzEueFEMYMlQIiyOTaz1JL3\\\\\\\/EhrcMw23oRNIKHTlwFIxYfwpHB+Z+CwW8uD9noK6LMgJzzYubNN35U1zZzFpca5sYPyuAc7J+95SVjqlTNGM7edXMdV60mcyzTe+sfxOrZvsrUL8G4B5Xys+CLyuRHjO3CIc9KRzr2ToswkpobOUYWlbhnOJ6wDip1rcj4EnrFgtAmG2r1NMdpMslN7ldC5fw7m+mECg0YVBWQBhAD4bMo49HC49s5pV8fT781Yw6virZaxxXF1yCltQ1hqgFRQNfJB\\\\\\\/P016pl5H0w+GPeab\\\\\\\/ByRlwh80aXJ6qOMPD04ew8XT5XHUZ3tHrWB\\\\\\\/qU45jsyK+7M89hW7B1q7TsI1w3Ykjrswo2L51WL4UFrajkJVhB5Kbz2LuGoUlBg0tZxdCYXN3IzugI4KNRXzrZpFHW26m1jRQj3dK\\\\\\\/LuMLF\\\\\\\/RVTa8UbJrUobQlJNqxvtyWxhBoqHYKEky8EWiDK7UFQHDzpoxhtt5eFO6bEPW\\\\\\\/KSuT1ja3mrCkCZ8Ec1OBAeg1FsFpkMWm1jefgXImIdjUMblWR1u4pXeKQzoqJEgK3iUQ\\\\\\\/K9\\\\\\\/lQcc6ATkzu0eoY90NSFth29mjjoTOA3p0R68sIVP5u92zQO24ax6omRpxyCR0hvkTlU3uP6tis9bW+7WgHmAPpLQNo9Ips6T+rEw4U2qHlefvH4N4Lv6f9SGUFxfaMhWAwoN\\\\\\\/les1uc0I4LkmWr3BsTXei\\\\\\\/uuixlliRH4t\\\\\\\/Wq7GYobUy4ufutO1Iu6GuwZeAkNrAb4iIS3l9zekMkcJ62h2qqNCCOJbiHe6xKF+5FYML2iLrHT0oRJKKVmktEzB6q7Ggeqc6WNq1o7Zj4mr6vKAC1v2dh\\\\\\\/KaiBBTBEGLj2BAjCVZFA8TAj8xKVpHkCxTVkX\\\\\\\/4xKa8ikED8ADAS5uQcSUciIMdFwnUHFozZC2I2ObtsIqtKboiWYt6fCu1Orv\\\\\\\/ekdKp+kYCRUBZBcqrqNHHeu+5to4UYBENCKaKW3zaK38KwE2rWFxAYFRmEXTirakGh0PATz\\\\\\\/mUzk0cokbbV++U9XEMwA2B+POhWDWfLz9HwraprQaduO8ubgPF3kPuPFMhMzVDZ2Q\\\\\\\/4+xLt24EQ9Q3HDU1YwBP8UAHh9TwgVDbs6h362vQ4m9sdbnGWIAcp+bCGxDmi5dp5SzMGmQepjCGHMfyf7uDKP8227RUc42SxEWKKQXSCBaawFtPNgfxCgwe1MlyDIOuEPhUivUjCd74jxHzg1lMNPendeDtbMZMWsF+xEew6d3jPP3um7hpovcarb+7EgjGrOuptVQPwsDJSInt096PzwLyg2tBiKEViOP9rA0GjY+RSwLs8fme51GwhRDEcwyGlpd\\\\\\\/rXwd0Og1p9vu3T1s7+uHHnpTeXNWEOHfSidb\\\\\\\/SXpClQIJhsUcc6ViC2DaCH2yA2I1EF1Gjillm\\\\\\\/VS8blxLApWuVPuWQlZuOe77R7ZyzhKuD2XUZNqTcOTAF49+v5kWD+ruVyuyA5eT4ISwLBhkWCURnYaEV4+ZPn4+K1LgUm3nE+RzwJJzZj5ZOK02rP8HW7rhHdFPH9OyqnVDoHo0tPpucZBfiky3k7dVGP6vhhFDZRDv+4MdbGE+qn3xZQW4ofh7s5V1+dXglYlIa+nQ71fvf0eAm3nc5YX6wJtag9XZ0rcrAVvBe4Pl0v7GQSxprg29EGxTFJWWKfFXISSXKRhbHzTQD8WZcNL28oyDEFBXdWjL\\\\\\\/tHQT2rfLU8UMwXSJK6bSx2rrjs4ncuaBo6KRP2T9HSUiQltHFvlTbkFXf0324KDuZe5SKgaDIbl4vy488PUjqGOSeWZvfjJ+6LorWEZbC5tzETjf35IZJhSnM\\\\\\\/qDnthuot65bcoQmCZReKn7xlgI8FQfG7wf9tnPNYbpZX6DRe85RJ3v\\\\\\\/Ovg\\\\\\\/oLoKxkNrzpW1Gl1N3RNPVUy9WnPcuBBhPv9fcPbk75T+uqqk4WrAbPzULdZ\\\\\\\/R0mjRTLiMl1rINyySMM8nxzArOWuy55t6psA95\\\\\\\/8X\\\\\\\/qWqHXl7aiaaW4rYEvDiomhenwaqxhamnAIQ13gwSGma57OFXIMIp5YcpGha2E9sv82vq3akJ9XyTtLrK1Nv3clM21M9HMrJIRWOaKYGVg7jX\\\\\\\/Izs0+ytjXiMCb7SlNByr+FxeXOeGyeyeH14NJjj83enFf6E4IPeTg78dDiewbq42o2B+zJv9ifjzcEqFYikoJBYsQieeMGdgIddG+qw6gkecdpd0HD9Co9si6Zo4RVz4O3ExMN+AHjveKW2Zgs7FJW5LE7nk8B7JAxEjdAZ8SfzbekiXMjZ6zgf0u123hhBf3B5Dic3MsaBfTJARX\\\\\\\/qtKwvADgLJD\\\\\\\/FtKZpGto7\\\\\\\/PeGyiNv9R0o0H1bk9KbSbwMerclGP0I5kEuhBfQZHeYwMCyV9KHJufjHkORGNZBdq342tXzrUTGLQfmC\\\\\\\/wrVSZUFAomPY1XlRL\\\\\\\/Y6ScL1p8zO3XZ7tX6zjGG\\\\\\\/yQJuQE\\\\\\\/4emuS0HFh9iV4YzhZAHgmFx9lmDPpNglwoL6bcsBJ1KH4OJjC3UqisolFrLDP8Nhv7B6LnvIVBpKqhUpSCwWyNbJ+D6weL4lxMmd4A5aEu4UOLAtHtViBndfixDpDz+QhScVO6Pyr970EqgGMWBUVU7rSh5uXZ2ha4xKqRTrUhso86XsqDXHToI9arjof4\\\\\\\/YIBqTvd8gsZQwD9Cns\\\\\\\/WCX5BeqwnuahFIhwI3U1eaw8iJWDRNL\\\\\\\/+CDTPaz9bV4iPiFoow7ZdBqLJ84FvPoO2sL2T0jvTZDVbYvunG\\\\\\\/D3ePKkc3MT1ktDjqfYMe614ULl5QG773MgTVxJGuxmBExtdM+jwJ1JIkF3IyNxegXThBfO0Bs5m7vVNRkSBVDe3LVmm1aIeaOoCMFr4EdvCTlhg8WiWOhKeu3b5lZu8neM2sF28yaZL5vb+KiDTW4Ug75hPAe\\\\\\\/TH+StZd5IQ126NuqyQe676ITgbvwp8EScSr8nmqx6wkRG6VCzLhAFZr2Q0lcN+X9qO+8TnORvjA5TD0R3kiHwyWz6ow3nouPXek9Lzmk0CPl3hxrC+f3T060RxGrv+2Xq5tbNpT9YgvBifKTFoEa6qN2Gbc5w3CBZxHw9iUdA6\\\\\\\/E2e64DMxMLIqXW3iPHveV7UaM7WHaf4H9bd2p3XX8d\\\\\\\/OSEC563wC+T\\\\\\\/BM4+ns+Z4AWCr7laTsicfRFofro+oOsW6eWYc5iv220FSwu3jhrjU5FqGAhTklZNeqmxhHdysNf4oIwPMnINCwGRh+aYWf3MU4+8rufYubfL9BVGqAh12CU3UL3Q3YyINZf\\\\\\\/FO9UFA\\\\\\\/FV72cb\\\\\\\/y6DF6UR2Tz8ofUesVjrnLSwF7vWg3HAGMIX1oTg7rlwyXMBlNr+IdqyDYLiWv45\\\\\\\/tRb+zN3AobAmmGmuMt9IlwqZ6qGXlQM\\\\\\\/s2Pv8UTMTKrGEuERdHdrRaubQkYd53oG6KR+8gQhCvlaNK08fCQdpdMUPGedum3phW+IEKrgi5xvaUyO1uqM6CGnHjS385wIiUif4vzGsMWPdYLJD5UAvxFMC4dCO+FMEZWZ11GrbnYY+3YEZ9d593D8ncioxJIJv+2Nul1mmyd0ewq7hOfRuvoVb0qpRjReNGgnvkZG1d2CfgDGJFOM5ANP5W1ExBJRIaYX1H4Q9aBZkZ50\\\\\\\/\\\\\\\/csVVYtm3jg3j419w3NPvvj38Vx4GBut8Ws9xQ\\\\\\\/649NB7AqusYaFNP4KJhTvaNdgSal2NlIJLaJQ+DiU2pR0JNeTpVQ\\\\\\\/tBAKG\\\\\\\/wolTKVGYDPjrdIGvE3pt8jiPXMm8mefNJRzAoChURKe1st9AxyNJUa1F6Vaj9QnkJKftLhXANLkaBA6soVtRGlrWHAD5nS9Spdr5CxWFGHC5l1vw1gMLgvCMRx5fBEKrKjnLBsti0rCyCI36xLe2iE+It9+SALaUhVmixqm1oJh0SfhZF5QKvDqZbQhY6S8UeYu8YWojAiRj94qfZ64k+4WJgg\\\\\\\/6yxChHF1HAxJOHVcnviElcxaMfiPX63Z4KY96Af+vtwFj2SjIXYP2EqaiunxbNiXu+0ZqWPZj01YzHy7fOvP2n0ahFM5e5TzG\\\\\\\/z+asStjA7TrOnOKh6fv4PK+9swYKlJMe0l1Weykk8JXz2GxjqWnLA3JNG6IQzMSfUQomj3FFW6scKqCkJg6dTaX8JNDemeHV15+kQ\\\\\\\/huG19i5SQrwKagNC6MdpWl+mNfykjjUT1jKk81VWZ5w36wJCRjWNPr9WDxGMHWYXktD85OfBjxhnn2lOc7F2t0JLF1iBHsWluxE8XWms+or742JJX99OBA0gXEDZTiTs3YppqFPnMKSc7R+c5G8YoIP0n8nH9qHpP3CXGPIbddyLFjWOsPph1B1hKACu1H\\\\\\\/qJZg5voUALrtO9X4BrVbgO9tVSsCMk7p3Z4SuY9Gv8sULO+JKx+edG0BzNIB0KTCcJvYPHUoMYuICJXrpozqDZsOWN25AIcApKIbAaOZkwxLkS00oBVLidBcQ8Zxz486UFAaRkskOGf9BrZO7htyoxJ5iMFaqKIXj9MyBovXv6Zaq7m+aEEkv0Pq4SM9+Hx\\\\\\\/ShIV1LxVwWc2wu2Trh3vffYtlBwo04zfiVSWtjctrZptxzJdc\\\\\\\/erShwxxKXpDGWbBS4FrUjhyeRZjisrmqruRNH3GzX+ZcOQH2OloiaKLE5veJI02JH9S9K3HduwrasWF3bTcQoKE+b0ZJ23AtNfY6nm8pvBX3gZOKFN7BQtKSRD3F4qamF3Ji4LvVG5UCkYf6YAQmyfTl7Dn0Mm3HtunLSFBEA9KrluCfr3zMvIuShBRpq0Bs4BO9j9uzYo7CQJPoHs2PxuqlYDUn2EgmO\\\\\\\/WXFVr40+gpK7uoA9raPYtn0rC0RXXgAZ7LKP23f0I0fwouBvcThpeI+xCo2OyeJLSR8nUaM9GYtpjBRUgtEsP6CfhvI1dlz4IxOBO5SbkOWgx+ZNW33bKI8IDuD5JK2pxwhs3XwH6BF9j4C90VtazymbUR0k5zBB0kzbyGb5VGzqqEIxIc4p7069I57eKKgJcLlPXjNDSQxcRteLqBhx1kExl0fMClugK86U7rFXtRp0c0MqgLp5CpuQhgjTbFPAlhIroAfk5W4jR1zMyZWq2FkyzN8upAylTSUyd5lNkbXSP3GfPvHDzK6TJA\\\\\\\/xmulcCVymz3d1qnGHbVVqykPAniR9rEjGQPKX89P7swKrfxDc9DlqI9s9VnomYcJdk5dra2ulqz\\\\\\\/DMqoJwSURbYpdiE\\\\\\\/afs\\\\\\\/d2IuSBXug3VYOr8kdLB8rPxQlxVjMV\\\\\\\/DGwDb5nqW+a1zyiOIlCfcCxbvB+fndTRYgDKMRlJq8G1cKrAWMGRWXUxsVt439Ryp9XWKwWjxvnVXUGRN2svD3ISpC2NSQAeZYAsVI\\\\\\\/IsV\\\\\\\/hOHv+9EsSXJx0IpQXVUuo\\\\\\\/KV9JREbCDw2KZMeOh5c0MhwIs\\\\\\\/gRiffs7ZGePEvkgBNvN\\\\\\\/SGJk4B0hKeEn98QCMCch\\\\\\\/IgoTHkEKdtNhmGoWNWHiquAcw3v4q1KzlKpFCJhIa9ORHk+tPov0i7RmuyuSyM\\\\\\\/hRtt1T9m\\\\\\\/gcACqNu+M0C264pG9kNbd0JTEguuXfmlDOnQ5NWWd7OTq0a3i1Fgez3Tg5qKLwZMb3gYWu0n+eWnd6AZRJEcj+eHtF9307Rn9j5+DvBet2WOaKbNpszlDYZL4PBc+aQA3Gnxy2GAeZqay\\\\\\\/1yMB1cO7rWMcVpF\\\\\\\/F7+8IW7WuAwIdpNTSKSLIphx7V6NYjH37gLBMzGGZ3NcardBN1CTYD5IQF7f+5cFSHOZdzSXiVqJW5K5lzum8yxtof046+Or\\\\\\\/T7XYMOUI0cTuowV2egEW\\\\\\\/KWEmIGnNdOW1IOV2v8qI8\\\\\\\/gyKWwBTRpRlQV3wioonuZvW6MVnICR1gpBDFI\\\\\\\/xk+VvPdaV7Qkf14n4iBrP6pOeU1F4aryw\\\\\\\/skOF8jg6Ii7wAK\\\\\\\/WUEb5EgxXnuqnjqcZPkc75FqVhRHbb2hcMv5dC9xnHcqBvGrasfXlsbZj7jEV9xIOcrvGcVB1FSEjEVlCrDSFTm0hZO3Mg563d8AjlDKnmTjhDsJDw+EEBn0V+Ohw0encgHJUrTp23QO2PEaWcgkmJs0y7OS8diqF\\\\\\\/YIBoX7cpmpcslbX60aFknwnGsOMxfsE8DRnLnWcro2SxfwJHS1nV0iIDhtAfm38NWxSEmAOoAtRKLIfc5Z9h5d2q9atSeLyfxwyWcWFNQYIXx6tRwFeG92t6gtm2HFGqpe4MVruH7TKfYRSg0ROPexnwE72bTRBqrDsyv1BF1Y81m2PHmAViHaJe\\\\\\\/NZzRlDSuiAX416u6PBmwQKhIzfvQ0ZU6q6odWHTgGaKEavMfgFKxl16VkctrrJmVg12vUCRYzghndToUUsj1ZTgDxcpWZVzC8q4l97Ni2iYl8p56lH+bpTGaB6t3BD4CGTe0dLtxB6i36pONqjZq8Jb2YgqDJh3DUaSnSj3x7egBmryfLL0p6tpa\\\\\\\/tdisBDATVKaanFs6J\\\\\\\/UnotSq6fzSEjeK5sSenmfnirVuYqawMmMWciDLx0msmNRawwFMrqwnRGFxQY5FnF6E72BfMaeJ1\\\\\\\/Tqv9WiFJk2hpCqLFualFeQnv66KFW+rWoq3RDGAyYaoDmFBkac\\\\\\\/ijyRH\\\\\\\/ZZfmAOdPxEidY1T3U6H820d5BbTtvdsPerVo3fq7bbQ+qQ4013Q6eJNyVx\\\\\\\/351BwT9jVVU24IMTasjOTb7rqGHOdrhLfa5mO2tsJrwJ0wbmCQwK1lWPtkig3nYUFwg\\\\\\\/SQXCTZV8a4FUEziWlNqM4DHmwLMVtF74mhm69QvgFYqbrBm9oTS4iiiFyDs0faoaFacBE2+Mf1XLphkec0fPI+dapCGnocd9fCsr8Cn\\\\\\\/0\\\\\\\/lu8ILSGnLO0hLE+stZuYFhT7Hz58IhrEMom3fa+\\\\\\\/2UcWgtaxI7H3BGf7BBbqmLYym+BvAbWEo1CPe75hEbaJbko3vnx6e0O4l9fzCWVK1HvgoyoBCdMz2JxXH4m8lLk73609PCuXWl+fPz2RNoYVrcP7ZhyARTLS8Pp79GVDQP\\\\\\\/dDFBerpgummOeiMuL5RVtXYJ0Aizc+OtRYZTkVXFO2YiJSvZ2OaJZK7y+OYwWPfCMD3DpXl+MFJk1sEkAuEMo2kS5vr53WjJ49z1zUX5pMo35j9\\\\\\\/33cfAFfugZSIOxOZSotKXA70B167S6rkmrsFV15sYBHeuif9UUDJ2abDff\\\\\\\/LqWnsvTLqvWLtbmiMRW5Fj9xXVFtfzx3wuXJWuozyFw75xEBQFpWBIarzbLrRcTSOPkpHHapAMQDpaAgmJ1XqUNIvUXCWXNUx3tXplsMYmV\\\\\\\/bVtMujea9muv4pWcoAKo9Rc7gRYzlLJ8Hy3qSzsJ8McXE6VJzYJMwDJxiVxh2JoV4yVDQIYJZ\\\\\\\/RMkRJNBzOi4yLZ2ugCOp\\\\\\\/OGyyyHBOgUqiw+AgtUyS\\\\\\\/vmVGtResjHkImjJXpZ\\\\\\\/LVi6Re+wgOlarPaCQEdEnZ\\\\\\\/21Ey+I9jEams8wML7wUOHbf0ZCZKjSKIbZ0A0=\\\u0022,\\\u0022iv\\\u0022:\\\u0022d+8AnjTRf8v5M6WmgI1ZTQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b5a8c92bca9a16ebe6c1968f157936e228430c5afb083b11f96f23fe8a3b44b7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VaAW38B2CEs9cc1V\\\\\\\/9uJg72QSp\\\\\\\/9yQQucMpe4zR1gBOfXKZvB4rGWL9uZm9dJq1moCdqJDQtOWapcPuQaqAZXygvDQhKWrwAMooB3KGurHhKiIhIiFBMuwWdSm59yabn84J67xbf\\\\\\\/y4bof3fyCazvD3hdjMb2OQlSnZszBPj3CgWAZkXbMNa9rN6c3fwrG3h6mduxCnuaGRncInV9FgjsD+hNdNVAZnp95Z5te9+vxBOA4uSGCE9g36mycKp5XsPm9ox9f+m+RwR7rA3UXnTDTEPX+QGkDbV+FE4662+uuUBqbnUQzLHacEbN0RyZh6fO+jaRlpn5EbQGGlt83rwoSwJl3DmqU\\\\\\\/+bhpKE7jnmf+QpxiIl\\\\\\\/0YPXDtXGv91VPN1hvQvuCetIqIUwR\\\\\\\/Ouxz1XpnKl4HEg7LCsJkVfJ0Lcql1H1+jucC14nOWKX6+rOAHf2nhOgiNEE8BaJMMSNTPK8Ge71lqpVH8I+TcZf79IPBvI7uzHvcJMffdr8jwiLg\\\\\\\/xKLaXbFW1HCGLvxLtCZ\\\\\\\/xLIVoMWOcqrmI2OZhf0SnlbkiT0VaS5Kn3HgYdgUPo0uWDlX7wvpWhNNjLnG1ASWRUF+lUA9Ay+yxgZ3H09F6jwu5flots\\\\\\\/PVm3vqPW5n8\\\\\\\/n3LEeE0d4hZOK0C0Eeu5uYBetraq+yz\\\\\\\/DfCOZJ1nuUGXAWeJWvf++IOaIodH0z8k658emKc7EO3JG9sejTmzDYnuHjR5ac4Om74jIW5miXryL0nQw563XSGqGTnq2Lnjx+rQLSTUswDL72ZCI3Dc+djZongZOgHmwrF9pktekMGxe+jowi7geEBP0WZoa4deSBqoR9KEjVfQ2Pl4Dy+N6ivMaHXSf8BEKHUSP4Wend\\\\\\\/b1TnOvcYBjp\\\\\\\/eMFgJZtjZKLOkIbvSwmPGpy5vxx7Xuvq5Y\\\\\\\/Y1v2XQ\\\\\\\/xNGMD8EkS3+PDneiSo+qAQnBHN4ORov1sh2NLGcGyr0pCXSaSv2S2TrIERG+APWVjSieYJfpjYdoUaRpYKsXiFLKv8whdnHg1axH13o7BWB3XiffNpV40Beqh+mLme07i8Gq2SvJIrUsMSZFtNpdFzpVjYgZ4CI5yiW5GXcEXanur0Nnt\\\\\\\/Ka+QuxwWgZH0reMsbnBiFt4ZYk3LC5pborNaadywTh6SEo7yduZ5DkbZVoLuHiavQWcVB3dJ+pk02ibN\\\\\\\/astwHCEbI2sTZWqTHg4g+AWsGCEMBQ8OTtoMl3WgSRFrJRn2W\\\\\\\/Wc\\\\\\\/hRGPgGzt8xfP+uSwzISfCx\\\\\\\/owbTQnE+1foLrA0NDa+2IKWfK\\\\\\\/jnLvGjimdvkbH8RfTLc24bJkp+fFJDg+PXF8c\\\\\\\/c4ceB3m5\\\\\\\/c8TXRv+LLtf4ahC6UUWEIGxXSs93uAybtGhns67vGgM9BzBka+d5UsrZa62UCAu7ozPLGB6tKcBOZs5AoTDuIteSm3BXsz7xvGHMO47QtHlFXQ5COoObnAgnXIS+45TDYeRPgyhnPAn7iEvcSgs7gLCS5hyipnQ2hL8tzoeI\\\\\\\/aZrLQJepbKJe+kGj84s3lEu8c9zCamG8FmZH78du\\\\\\\/zKhGJ3isTd10MDb26qDX\\\\\\\/jwyp5hULzCdFuLAzYAfE74CxKSlCi1CJDujH1J9Dr2UMb81y0ykWs\\\\\\\/GBOPZoSiKXxD\\\\\\\/ccNsiIw8YX2oMMveePkX0+WUKGmF\\\\\\\/CTtGhXFDqwC\\\\\\\/OnSpLGdlx0OTEm7IG6\\\\\\\/M6cNOnYs5lGTCt6Jww14K2A+6m5wjCXo7s7RB5Jp4YgMKVUVoavgrKB7QNtZCpg9DC6ecME9hF4aTjFyfm1KACbpGQRtWbyBmfudGq2WPkfhHIdv7xYlxER+0I7dcSO7Xl1vf916QJ5Mx1k7FmVHY741mC3zSHT6z82j8+Zttqx3aC0jaaVCNIpUBTqKCXOs\\\\\\\/GcCfGQ0mML8NjwG7ZGmkwmsMHYrDk94oVUriJEBWtBr5F0l3\\\\\\\/JC4F5pCA8Mprkcz3WAd9BKUbpaANimu1BrV2HDSCY2jLBPMRwROB5WgiX4utsbwKEO2jzLFeq6AUOdxE70hUoYk2h8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022GzpiRE2v7r8Ei70UGRJ9Bg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225bea7cfc024c2af36951eadf5b9e509ee5baf16c6b71195d3fa2d57fd124d898\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495861 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1063" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9UENhlNW4uiKSmooNcrwvUkRfBQHBKGJzpeOoER5yVI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1063" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nTKnA2G-2dz80VuG6FA5J3p5C0xyE5xW2g2BOdv8X10" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/1051" title="The Developmental Transcriptome of Aedes albopictus, a Major Worldwide Human Disease Vector" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/1077" title="The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in Arabidopsis thaliana" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495861" id="node495861" data-pisa="ggg;10/3/1063" data-pisa-master="ggg;g3.119.400919" data-apath="/ggg/10/3/1063.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selective Sweeps Under Dominance and Inbreeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6941-0647" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew Hartfield</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4730-2538" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas Bataillon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1063-1075;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400919 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495861" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew Hartfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Toronto, Ontario M5S 3B2, Canada,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Research Centre, Aarhus University, Aarhus 8000, Denmark, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Evolutionary Biology, The University of Edinburgh, Edinburgh EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BHartfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hartfield%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BHartfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6941-0647" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew Hartfield</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.hartfield@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">m.hartfield@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Bataillon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Research Centre, Aarhus University, Aarhus 8000, Denmark, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBataillon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bataillon%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BBataillon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4730-2538" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas Bataillon</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/3/1063" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495861" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495861/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/1063.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495861" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495861/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/1063.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495861" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495861/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/3/1063.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495861" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495861/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">A major research goal in evolutionary genetics is to uncover loci experiencing positive selection. One approach involves finding ‘selective sweeps’ patterns, which can either be ‘hard sweeps’ formed by <em>de novo</em> mutation, or ‘soft sweeps’ arising from recurrent mutation or existing standing variation. Existing theory generally assumes outcrossing populations, and it is unclear how dominance affects soft sweeps. We consider how arbitrary dominance and inbreeding via self-fertilization affect hard and soft sweep signatures. With increased self-fertilization, they are maintained over longer map distances due to reduced effective recombination and faster beneficial allele fixation times. Dominance can affect sweep patterns in outcrossers if the derived variant originates from either a single novel allele, or from recurrent mutation. These models highlight the challenges in distinguishing hard and soft sweeps, and propose methods to differentiate between scenarios.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/adaptation" class="hw-term hw-article-keyword hw-article-keyword-adaptation" rel="nofollow">Adaptation</a></li><li class="kwd"><a href="/keyword/dominance" class="hw-term hw-article-keyword hw-article-keyword-dominance" rel="nofollow">Dominance</a></li><li class="kwd"><a href="/keyword/self-fertilisation" class="hw-term hw-article-keyword hw-article-keyword-self-fertilisation" rel="nofollow">Self-fertilisation</a></li><li class="kwd"><a href="/keyword/selective-sweeps" class="hw-term hw-article-keyword hw-article-keyword-selective-sweeps" rel="nofollow">Selective Sweeps</a></li><li class="kwd"><a href="/keyword/population-genetics" class="hw-term hw-article-keyword hw-article-keyword-population-genetics" rel="nofollow">Population Genetics</a></li></ul><p id="p-3">Inferring adaptive mutations from nucleotide polymorphism data are a major research goal in evolutionary genetics, and has been subject to extensive modeling work to determine the footprints they leave in genome data (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Stephan 2019</a>). The earliest models focused on a scenario where a beneficial mutation arose as a single copy before rapidly fixing. Linked neutral mutations then ‘hitchhike’ to fixation with the adaptive variant, reducing diversity around the selected locus (Maynard Smith and Haigh 1974; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Kaplan <em>et al.</em> 1989</a>). Hitchhiking also increases linkage disequilibrium in regions flanking the selected site, by raising the haplotype carrying the selected allele to high frequency (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Thomson 1977</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Innan and Nordborg 2003</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">McVean 2007</a>). These theoretical expectations have spurred the creation of summary statistics for detecting sweeps, usually based on finding genetic regions exhibiting extended haplotype homozygosity (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Sabeti <em>et al.</em> 2002</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Kim and Nielsen 2004</a>; <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Voight <em>et al.</em> 2006</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ferrer-Admetlla <em>et al.</em> 2014</a>; <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Vatsiou <em>et al.</em> 2016</a>), or an increase in high frequency derived variants (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fay and Wu 2000</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Kim and Stephan 2002</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Nielsen 2005</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boitard <em>et al.</em> 2009</a>; <a id="xref-ref-110-1" class="xref-bibr" href="#ref-110">Yang <em>et al.</em> 2018</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Fujito <em>et al.</em> 2018</a>).</p><p id="p-4">Classic hitchhiking models consider ‘hard’ sweeps, where the common ancestor of an adaptive allele occurs after the onset of selection (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Hermisson and Pennings 2017</a>). Recent years have seen a focus on ‘soft’ sweeps, where the most recent common ancestor of a beneficial allele appeared before it became selected for (reviewed by <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Barrett and Schluter (2008)</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Messer and Petrov (2013)</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Hermisson and Pennings (2017)</a>). Soft sweeps can originate from beneficial mutations being introduced by recurrent mutation at the target locus (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Pennings and Hermisson 2006a</a>, <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">b</a>), or originating from existing standing variation that was either neutral or deleterious (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Orr and Betancourt 2001</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Innan and Kim 2004</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Przeworski <em>et al.</em> 2005</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Hermisson and Pennings 2005</a>; <a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Wilson <em>et al.</em> 2014</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Berg and Coop 2015</a>; <a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Wilson <em>et al.</em> 2017</a>). A key property of soft sweeps is that the beneficial variant is present on multiple genetic backgrounds as it sweeps to fixation, so different haplotypes may carry the derived allele. This property is often used to detect soft sweeps in genetic data (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Peter <em>et al.</em> 2012</a>; <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Vitti <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Garud <em>et al.</em> 2015</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Garud and Petrov 2016</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Schrider and Kern 2016</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Sheehan and Song 2016</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Harris <em>et al.</em> 2018a</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Kern and Schrider 2018</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Harris and DeGiorgio 2018</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">2019</a>). Soft sweeps have been reported in <em>Drosophila</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Karasov <em>et al.</em> 2010</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Garud <em>et al.</em> 2015</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Garud and Petrov 2016</a>; <a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Vy <em>et al.</em> 2017</a>), humans (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Peter <em>et al.</em> 2012</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Schrider and Kern 2017</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Laval <em>et al.</em> 2019</a>), maize (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Fustier <em>et al.</em> 2017</a>), <em>Anopheles</em> mosquitoes (<a id="xref-ref-109-1" class="xref-bibr" href="#ref-109">Xue <em>et al.</em> 2019</a>), and pathogens including <em>Plasmodium falciparum</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson <em>et al.</em> 2016</a>) and HIV (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Pennings <em>et al.</em> 2014</a>; <a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Williams and Pennings 2019</a>). Yet determining how extensive soft sweeps are in nature remains a contentious issue (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Jensen 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Harris <em>et al.</em> 2018b</a>).</p><p id="p-5">Up to now, there have only been a few investigations into how dominance affects sweep signatures. In a simulation study, <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Teshima and Przeworski (2006)</a> explored how recessive mutations spend long periods of time at low frequencies, increasing the amount of recombination that acts on derived haplotypes, weakening signatures of hard sweeps. Fully recessive mutations may need a long time to reach a significantly high frequency to be detectable by genome scans (<a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Teshima <em>et al.</em> 2006</a>). <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ewing <em>et al.</em> (2011)</a> have carried out a general mathematical analysis of how dominance affects hard sweeps, finding that recessive beneficial mutations have markedly different signatures compared to those with other dominance values. Yet the impact of dominance on soft sweeps has yet to be explored in depth.</p><p id="p-6">In addition, existing models have so far focused on randomly mating populations, with haplotypes freely mixing between individuals over generations. Different reproductive modes alter how alleles are inherited, potentially changing the hitchhiking effect. Self-fertilization, where male and female gametes produced from the same individual can fertilize one another, can alter adaptation rates and selection signatures (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Hartfield <em>et al.</em> 2017</a>). This mating system is prevalent among angiosperms (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Igic and Kohn 2006</a>), some animals (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Jarne and Auld 2006</a>) and fungi (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Billiard <em>et al.</em> 2011</a>). As the effects of dominance and self-fertilization become strongly intertwined, it is important to consider both together. Dominant mutations are more likely to fix than recessive ones in outcrossers, as they have a higher initial selection advantage (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Haldane 1927</a>). Yet recessive alleles can fix more easily in selfers than in outcrossers as homozygote mutations are created more rapidly (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Charlesworth 1992</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Glémin 2012</a>). Furthermore, a decrease in effective recombination rates in selfers (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Nordborg <em>et al.</em> 1996</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Nordborg 2000</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Charlesworth and Charlesworth 2010</a>) can interfere with selection acting at linked sites, making it likelier that deleterious mutations hitchhike to fixation with adaptive alleles (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Hartfield and Glémin 2014</a>), or that rare mutations are lost by drift due to competition between adaptive mutations (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hartfield and Glémin 2016</a>).</p><p id="p-7">In a constant-sized population, beneficial mutations can be less likely to fix from standing variation (either neutral or deleterious) in selfers as they maintain lower diversity levels (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Glémin and Ronfort 2013</a>). Yet adaptation from standing variation becomes likelier in selfers compared to outcrossers under ‘evolutionary rescue’ scenarios, where swift adaptation is needed to prevent population extinction following environmental change. Here, rescue mutations are only present in standing variation as the population size otherwise becomes too small (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Glémin and Ronfort 2013</a>). Self-fertilization further aids this process by creating beneficial homozygotes more rapidly than in outcrossing populations (<a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Uecker 2017</a>).</p><p id="p-8">Little data currently exists on the extent of soft sweeps in self-fertilizers. Many selfing organisms exhibit sweep-like patterns, including <em>Arabidopsis thaliana</em> (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Long <em>et al.</em> 2013</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Huber <em>et al.</em> 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Fulgione <em>et al.</em> 2018</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Price <em>et al.</em> 2018</a>); <em>Caenorhabditis elegans</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andersen <em>et al.</em> 2012</a>); <em>Medicago truncatula</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bonhomme <em>et al.</em> 2015</a>); and <em>Microbotryum</em> fungi (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Badouin <em>et al.</em> 2017</a>). Soft sweeps have also been reported in soya bean (<a id="xref-ref-112-1" class="xref-bibr" href="#ref-112">Zhong <em>et al.</em> 2017</a>). Detailed analyses of these cases has been hampered by a lack of theory on how hard and soft sweep signatures should manifest themselves under different self-fertilization and dominance levels. Previous studies have only focused on special cases: <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hedrick (1980)</a> analyzed linkage disequilibrium caused by a hard sweep under self-fertilization, while <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Schoen <em>et al.</em> (1996)</a> modeled sweep patterns caused by modifiers that altered the mating system in different ways.</p><p id="p-9">To this end, we develop a selective sweep model that accounts for dominance and inbreeding via self–fertilization. We determine the genetic diversity present following a sweep from either a <em>de novo</em> mutation, or from standing variation. We also determine the number of segregating sites and the site frequency spectrum, while comparing results to an alternative soft-sweep model where adaptive alleles arise via recurrent mutation. Note that we focus here on single sweep events, rather than characterizing how sweeps affect genome-wide diversity (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Elyashiv <em>et al.</em> 2016</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Campos <em>et al.</em> 2017</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Booker and Keightley 2018</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Rettelbach <em>et al.</em> 2019</a>).</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Model outline</h3><p id="p-10">We consider a diploid population of size <em>N</em> (carrying <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-1.gif"/></noscript></span></span> haplotypes in total). Individuals reproduce by self-fertilization with probability <em>σ</em>, and outcross with probability <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-2.gif"/></noscript></span></span>. A derived allele arises at a locus, and we are interested in determining the population history of neutral regions that are linked to it, with a recombination rate <em>r</em> between them. We principally look at the case where the beneficial allele arises from previously–neutral standing variation, and subsequently look at a sweep arising from recurrent mutation. The derived allele initially segregates neutrally for a period of time, then becomes advantageous with selective advantage <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-3.gif"/></noscript></span></span> when heterozygous and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-4.gif"/></noscript></span></span> when homozygous, with <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-5.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-6.gif"/></noscript></span></span>. We further assume that the population size is large and selection is large enough so that the beneficial allele’s change in frequency can be modeled deterministically (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-7.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-8.gif"/></noscript></span></span>). <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> lists the notation used in the analysis.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495964/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495964/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Glossary of Notation</span><div class="sb-div caption-clear"></div></div></div><p id="p-11">Our goal is to determine how the spread of the derived, adaptive allele affects genealogies at linked neutral regions. For a sweep originating from standing variation, we follow the approach of <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Berg and Coop (2015)</a> and, looking backward in time, break down the selected allele history into two phases. In the recent past is the ‘sweep phase’ where the derived allele was selectively favored, with its frequency decreasing from 1 to <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-36.gif"/></noscript></span></span>. Prior to that phase is the ‘standing phase’, which assumes that the derived allele is present at an approximate fixed frequency <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-37.gif"/></noscript></span></span>. During both phases, a pair of haplotypes can either coalesce, or one of them recombines onto the ancestral background. A schematic is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A schematic of the model. The history of the derived variant is separated into two phases; the &#x2018;standing phase&#x2019; (shown in light gray), and the &#x2018;sweep phase&#x2019; (shown in dark gray). Axis on the left-hand side show allele frequency on a log-scale. Dots on the right-hand side represent a sample of haplotypes taken at the present day, with lines representing their genetic histories. Solid lines represent coalescent histories for the derived genetic background; dotted lines represent coalescent histories for the ancestral, neutral background. Note the allele trajectory is an idealised version as assumed in the model." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760725932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A schematic of the model. The history of the derived variant is separated into two phases; the &#x2018;standing phase&#x2019; (shown in light gray), and the &#x2018;sweep phase&#x2019; (shown in dark gray). Axis on the left-hand side show allele frequency on a log-scale. Dots on the right-hand side represent a sample of haplotypes taken at the present day, with lines representing their genetic histories. Solid lines represent coalescent histories for the derived genetic background; dotted lines represent coalescent histories for the ancestral, neutral background. Note the allele trajectory is an idealised version as assumed in the model.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1063/F1.medium.gif" width="440" height="154"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/3/1063/F1.medium.gif" width="440" height="154"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495983" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">A schematic of the model. The history of the derived variant is separated into two phases; the ‘standing phase’ (shown in light gray), and the ‘sweep phase’ (shown in dark gray). Axis on the left-hand side show allele frequency on a log-scale. Dots on the right-hand side represent a sample of haplotypes taken at the present day, with lines representing their genetic histories. Solid lines represent coalescent histories for the derived genetic background; dotted lines represent coalescent histories for the ancestral, neutral background. Note the allele trajectory is an idealised version as assumed in the model.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">During the sweep phase, the derived allele will also cause the spread of linked haplotypes that it appeared on. Over the course of the sweep, haplotypes are broken down by recombination; the total number of recombination events is proportional to <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-38.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-39.gif"/></noscript></span></span> is the fixation time of the beneficial allele, given an initial frequency <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-40.gif"/></noscript></span></span> (Smith and Haigh 1974). Dominance and self–fertilization have different effects on <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-41.gif"/></noscript></span></span>, and therefore the number of fixing haplotypes. If <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-42.gif"/></noscript></span></span> is low (<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-43.gif"/></noscript></span></span>) then highly recessive or dominant mutations take longer to go to fixation (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Glémin 2012</a>), which can increase the number of recombination events. Dominance also affects the nature of the sweep trajectory. For example, recessive mutations spend more time at a low frequency compared to dominant mutations. These different sweep trajectories can also affect the final sweep profile (<a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">Teshima and Przeworski 2006</a>). Self–fertilization leads to decreased fixation time of adaptive mutations through converting heterozygotes to homozygotes (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Glémin 2012</a>). Recombination is likelier to act between homozygotes under self-fertilization, so its effective rate is reduced by a factor <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-44.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-45.gif"/></noscript></span></span> the inbreeding coefficient (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Nordborg <em>et al.</em> 1996</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Nordborg 2000</a>) and Φ the joint probability of identity-by-descent at the two loci (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Roze 2009</a>, <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">2016</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Hartfield and Glémin 2016</a>), defined as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-46.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>Note that <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-47.gif"/></noscript></span></span> approximates to <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-48.gif"/></noscript></span></span> (as <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-49.gif"/></noscript></span></span>), unless <em>σ</em> is close to one and <em>r</em> is high (approximately greater than 0.1).</p><p id="p-14">During the standing phase, the amount of initial recombinant haplotypes that are swept to fixation depend on the relative rates of recombination and coalescence. The latter occurs with probability proportional to <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-50.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-51.gif"/></noscript></span></span> the effective population size. Under self–fertilization <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-52.gif"/></noscript></span></span> (<a id="xref-ref-108-1" class="xref-bibr" href="#ref-108">Wright 1951</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Pollak 1987</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Charlesworth 1992</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Caballero and Hill 1992</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Nordborg and Donnelly 1997</a>), so self–fertilization increases the coalescence probability. This scaling factor will change if there is a large non-Poisson variation in offspring number (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Laporte and Charlesworth 2002</a>). Although we focus on inbreeding via self-fertilization, the scalings <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-53.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-54.gif"/></noscript></span></span> should also hold under other systems of regular inbreeding (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Caballero and Hill 1992</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Charlesworth and Charlesworth 2010</a>, Box 8.4).</p><p id="p-15">We will outline how both coalescence and recombination act during both of these phases, and use these calculations to determine selective sweep properties. Previous models tended to only determine how lineages recombine away from the derived background during the sweep phase, without considering how two lineages coalesce during the sweep phase. If lineages coalesce during the sweep, then the total number of unique recombination events, and hence the number of linked haplotypes, are reduced. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barton (1998)</a> showed that these coalescent events are negligible only for very strong selection (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-55.gif"/></noscript></span></span>; and B. Charlesworth, unpublished results). Hence, accounting for these coalescent events is important for producing accurate matches with simulation results.</p><p id="p-16">Throughout, analytical solutions are compared to results from Wright-Fisher forward-in-time stochastic simulations that were ran using SLiM version 3.3 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Haller and Messer 2019</a>). Results for outcrossing populations were also tested using coalescent simulations ran with <em>msms</em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ewing and Hermisson 2010</a>). The simulation methods are outlined in Supplementary File S2.</p></div><div id="sec-3" class="subsection"><h3>Data availability</h3><p id="p-17">File S1 is a <em>Mathematica</em> notebook of analytical derivations and simulation results. File S2 contains additional methods, results and figures. File S3 contains copies of the simulation scripts, which are also available from <a href="https://github.com/MattHartfield/SweepDomSelf">https://github.com/MattHartfield/SweepDomSelf</a>. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11687949">https://doi.org/10.25387/g3.11687949</a>.</p></div></div><div class="section results" id="sec-4"><h2 class="">Results</h2><div id="sec-5" class="subsection"><h3>Probability of events during sweep phase</h3><p id="p-18">We first look at the probability of events (coalescence or recombination) acting during the sweep phase for the simplest case of two alleles. Looking back in time following the fixation of the derived mutation, sites linked to the beneficial allele can either coalesce or recombine onto the ancestral genetic background. Let <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-56.gif"/></noscript></span></span> be the adaptive mutation frequency at time <em>t</em>, defined as the number of generations prior to the present day. Further define <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-57.gif"/></noscript></span></span> (<em>i.e.</em>, the allele is fixed at the present day), and <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-58.gif"/></noscript></span></span> the time in the past when the derived variant became beneficial (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-59.gif"/></noscript></span></span>).</p><p id="p-19">For a pair of haplotype samples carrying the derived allele, if it is at frequency <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-60.gif"/></noscript></span></span> at time <em>t</em>, this lineage pair can either coalesce or one of the haplotypes recombine onto the ancestral background. Each event occurs with probability:</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-61.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><p id="p-20">Equation 2 is based on those obtained by <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Kaplan <em>et al.</em> (1989)</a>, assuming that <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-62.gif"/></noscript></span></span> due to self-fertilization (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Pollak 1987</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Charlesworth 1992</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Caballero and Hill 1992</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Nordborg and Donnelly 1997</a>), and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-63.gif"/></noscript></span></span> is the ‘effective’ recombination rate after correcting for increased homozygosity due to self-fertilization (<a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Nordborg <em>et al.</em> 1996</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Nordborg 2000</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Charlesworth and Charlesworth 2010</a>; <a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Roze 2009</a>, <a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">2016</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Hartfield and Glémin 2016</a>). Equation 2 demonstrates how each event is differently influenced by <em>p</em>. In particular, the per–generation coalescence probability <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-64.gif"/></noscript></span></span> can be small unless <em>p</em> is close to <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-65.gif"/></noscript></span></span>. The total probability that coalescence occurs during the sweep phase increases if the beneficial allele spends a sizeable time at low frequency, <em>e.g.</em>, when it is recessive. The terms in Equation 2 can also be defined as functions of <em>p</em>.</p><p id="p-21">We are interested in calculating (i) the probability <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-66.gif"/></noscript></span></span> that no coalescence or recombination occurs in the sweep phase; (ii) the probability <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-67.gif"/></noscript></span></span> that recombination acts on a lineage to transfer it to the neutral background that is linked to the ancestral allele, assuming that no more than one recombination event occurs per generation (see <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Campos and Charlesworth (2019)</a> for derivations assuming multiple recombination events). We will go through these probabilities in turn to determine expected pairwise diversity. For <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-68.gif"/></noscript></span></span>, the total probability that the two lineages do not coalesce or recombine over <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-69.gif"/></noscript></span></span> generations equals:</p><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-70.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span><p id="p-22">Here <em>ϵ</em> is a small term and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-71.gif"/></noscript></span></span> is the upper limit of the deterministic spread of the beneficial allele. We will discuss in the section ‘Effective starting frequency from a <em>de novo</em> mutation’ what a reasonable value for <em>ϵ</em> should be. Also note that we switch from a discrete–time calculation to a continuous–time calculation, which can give simplifying results. To calculate <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-72.gif"/></noscript></span></span> we insert the deterministic change in allele frequency <em>p</em> (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Glémin 2012</a>):<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-73.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Note the negative factor in Equation 4 since we are looking back in time. By substituting Equation 4 into Equation 3, we obtain an analytical solution for <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-74.gif"/></noscript></span></span>, although the resulting expression is complicated (Section A of Supplementary File S1).</p><p id="p-23">To calculate <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-75.gif"/></noscript></span></span>, the probability that recombination acts during the sweep, we first calculate the probability that recombination occurs when the beneficial allele is at frequency <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-76.gif"/></noscript></span></span>. Here, no events occur in the time leading up to <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-77.gif"/></noscript></span></span>, then a recombination event occurs with probability <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-78.gif"/></noscript></span></span>. <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-79.gif"/></noscript></span></span> is obtained by integrating this probability over the entire sweep from time 0 to <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-80.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-81.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where:</p><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-82.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span><p id="p-24">Note that the exponential term of <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-83.gif"/></noscript></span></span> is different from <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-84.gif"/></noscript></span></span> (Equation 3) since the upper integral limit is to <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-85.gif"/></noscript></span></span> rather than <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-86.gif"/></noscript></span></span>. That is, it only covers part of the sweep phase. Equation 5 is evaluated numerically. In Supplementary File S2, we provide a ‘star–like’ analytical approximation to <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-87.gif"/></noscript></span></span> that assumes no coalescence during the sweep phase.</p></div><div id="sec-6" class="subsection"><h3>Probability of coalescence from standing variation</h3><p id="p-25">The variant becomes advantageous at frequency <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-88.gif"/></noscript></span></span>. We assume that <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-89.gif"/></noscript></span></span>, and hence event probabilities, remain fixed over time. <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Berg and Coop (2015)</a> have shown this assumption provides a good approximation to coalescent rates during the standing phase. The outcome during the standing phase is thus determined by competing Poisson processes. The two haplotypes could coalesce, with an exponentially-distributed waiting time with rate <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-90.gif"/></noscript></span></span>. Alternatively, one of the two haplotypes could recombine onto the ancestral background with mean waiting time <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-91.gif"/></noscript></span></span>. For two competing exponential distributions with rates <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-92.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-93.gif"/></noscript></span></span>, the probability of the first event occurring given an event happens equals <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-94.gif"/></noscript></span></span> (<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Wakeley 2009</a>, Chapter 2). Hence the probability that recombination occurs instead of coalescence equals:</p><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-95.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span><p id="p-26">The probability of coalescence rather than recombination is <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-96.gif"/></noscript></span></span>. Here <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-97.gif"/></noscript></span></span> is the population-scaled recombination rate. The final approximation arises as <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-98.gif"/></noscript></span></span> if <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-99.gif"/></noscript></span></span>. This term reflects how increased homozygosity reduces both effective recombination and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-100.gif"/></noscript></span></span>, with the latter making coalescence more likely. In addition, it also highlights how the signature of a sweep from standing variation, as characterized by the spread of different initial recombinant haplotypes, is spread over an increased distance of <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-101.gif"/></noscript></span></span> under self–fertilization.</p></div><div id="sec-7" class="subsection"><h3>Effective starting frequency for a de novo mutation, and effective final frequency</h3><p id="p-27">When a new beneficial mutation appears as a single copy, it is highly likely to go extinct by chance (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fisher 1922</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Haldane 1927</a>). Beneficial mutations that increase in frequency faster than expected when rare are more able to overcome this stochastic loss and reach fixation. These beneficial mutations will hence display an apparent ‘acceleration’ in their logistic growth, equivalent to having a starting frequency that is greater than <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-102.gif"/></noscript></span></span> (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Maynard Smith 1976</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Barton 1998</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Desai and Fisher 2007</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Martin and Lambert 2015</a>). Correcting for this acceleration is important to accurately model hard sweep signatures, and inform on the minimum level of standing variation needed to differentiate a hard sweep from one originating from standing variation.</p><p id="p-28">In Section B of Supplementary File S1, we determine that hard sweeps that go to fixation have the following effective starting frequency:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-103.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-104.gif"/></noscript></span></span> is the effective dominance coefficient for mutations at a low frequency. This result is consistent with those of <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Martin and Lambert (2015)</a>, who obtained a distribution of effective starting frequencies using stochastic differential equations. This acceleration effect can create substantial increases in the effective <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-105.gif"/></noscript></span></span>, especially for recessive mutations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of the effective starting frequency. Equation 8 is plotted as a function of F for different dominance values, as shown in the legend. Other parameters are , . The dashed line shows the actual starting frequency, ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760725932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Examples of the effective starting frequency. Equation 8 is plotted as a function of &lt;em&gt;F&lt;/em&gt; for different dominance values, as shown in the legend. Other parameters are &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-106&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-107&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. The dashed line shows the actual starting frequency, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-108&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1063/F2.medium.gif" width="440" height="198"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/3/1063/F2.medium.gif" width="440" height="198"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/496013" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Examples of the effective starting frequency. Equation 8 is plotted as a function of <em>F</em> for different dominance values, as shown in the legend. Other parameters are <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F2/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F2/embed/mml-math-106.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F2/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F2/embed/mml-math-107.gif"/></noscript></span></span>. The dashed line shows the actual starting frequency, <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F2/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F2/embed/mml-math-108.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">The effective final frequency of the derived allele <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-109.gif"/></noscript></span></span>, at which its spread is no longer deterministic, can be obtained by setting <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-110.gif"/></noscript></span></span>; that is, by substituting <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-111.gif"/></noscript></span></span> to <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-112.gif"/></noscript></span></span> in Equation 8. This final frequency is always used, even if <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-113.gif"/></noscript></span></span>. <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">van Herwaarden and van der Wal (2002)</a> determined that the sojourn time for an allele with dominance coefficient <em>h</em> that is increasing in frequency, is the same for an allele decreasing in frequency with dominance <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-114.gif"/></noscript></span></span>. <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Glémin (2012)</a> showed that this result also holds under any inbreeding value <em>F</em>. See <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Charlesworth (2020)</a> for a fuller discussion of effective final frequencies and their impact on sweep fixation times.</p></div><div id="sec-8" class="subsection"><h3>Expected pairwise diversity</h3><p id="p-31">We use <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-115.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-116.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-117.gif"/></noscript></span></span> to calculate the expected pairwise diversity (denoted <em>π</em>) present around a sweep. During the sweep phase, the two neutral sites could either coalesce, or one of them recombines onto the ancestral background. If coalescence occurs, since it does so in the recent past then it is assumed that no diversity exist between samples, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-118.gif"/></noscript></span></span> for <em>π</em> the average number of differences between two alleles (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Tajima 1983</a>). In reality there may be some residual diversity caused by appearance of mutations during the sweep phase; we do not account for these mutations while calculating <em>π</em> but will do so when calculating the site-frequency spectrum. Alternatively, if one of the two samples recombines onto the neutral background, they will have the same pairwise diversity between them as the background population (<span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-119.gif"/></noscript></span></span>). If the two samples trace back to the standing phase (with probability <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-120.gif"/></noscript></span></span>) then the same logic applies. Hence the expected diversity following a sweep <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-121.gif"/></noscript></span></span>, relative to the background value <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-122.gif"/></noscript></span></span>, equals:</p><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-123.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span><p id="p-32">The full solution to Equation 9 can be obtained by plugging in the relevant parts from Equations 3, 5 and 7, which we evaluate numerically. Equation 9 is undefined for <em>h</em> = 0 or 1 with <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-124.gif"/></noscript></span></span>; these cases can be derived separately.</p><p id="p-33"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> plots Equation 9 with different dominance, self-fertilization, and standing frequency values. The analytical solution fits well compared to forward-in-time simulations, yet slightly overestimates them for high self-fertilization frequencies. It is unclear why this mismatch arises. One explanation could be that drift effects are magnified under self–fertilization, which causes a quicker sweep fixation time than expected from deterministic spread, if conditioning on a sweep going to fixation. Although <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-125.gif"/></noscript></span></span> (Equation 8) captures these drift effects for rare alleles, there may be additional effects that are not accounted for. Under complete outcrossing, baseline diversity is restored (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-126.gif"/></noscript></span></span> goes to 1) closer to the sweep origin for recessive mutations (<span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-127.gif"/></noscript></span></span>), compared to semidominant (<span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-128.gif"/></noscript></span></span>) or dominant (<span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-129.gif"/></noscript></span></span>) mutations. Sweeps caused by dominant and semidominant mutations result in a similar genetic diversity, so these cases may be hard to differentiate from diversity data alone.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expected relative pairwise diversity following a selective sweep. Plots of  as a function of the recombination rate scaled to population size . Lines are analytical solutions (Equation 9), points are forward-in-time simulation results. , ,  (note &#x3BC; is scaled by N, not ), and dominance coefficient  (red lines, points), 0.5 (black lines, points), or 0.9 (blue lines, points). Values of  and self-fertilization rates &#x3C3; used are shown for the relevant row and column; note the axis range changes with the self-fertilization rate. For  we use  in our model, as given by Equation 8. Further results are plotted in Section C of Supplementary File S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760725932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Expected relative pairwise diversity following a selective sweep. Plots of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-130&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; as a function of the recombination rate scaled to population size &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-131&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Lines are analytical solutions (Equation 9), points are forward-in-time simulation results. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-132&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-133&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-134&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (note &lt;em&gt;&#x3BC;&lt;/em&gt; is scaled by &lt;em&gt;N&lt;/em&gt;, not &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-135&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;), and dominance coefficient &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-136&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (red lines, points), 0.5 (black lines, points), or 0.9 (blue lines, points). Values of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-137&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and self-fertilization rates &lt;em&gt;&#x3C3;&lt;/em&gt; used are shown for the relevant row and column; note the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-138&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;axis range changes with the self-fertilization rate. For &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-139&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; we use &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-140&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in our model, as given by Equation 8. Further results are plotted in Section C of Supplementary File S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1063/F3.medium.gif" width="440" height="243"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/3/1063/F3.medium.gif" width="440" height="243"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/496010" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-34" class="first-child">Expected relative pairwise diversity following a selective sweep. Plots of <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-130.gif"/></noscript></span></span> as a function of the recombination rate scaled to population size <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-131.gif"/></noscript></span></span>. Lines are analytical solutions (Equation 9), points are forward-in-time simulation results. <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-132.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-133.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-134.gif"/></noscript></span></span> (note <em>μ</em> is scaled by <em>N</em>, not <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-135.gif"/></noscript></span></span>), and dominance coefficient <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-136.gif"/></noscript></span></span> (red lines, points), 0.5 (black lines, points), or 0.9 (blue lines, points). Values of <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-137.gif"/></noscript></span></span> and self-fertilization rates <em>σ</em> used are shown for the relevant row and column; note the <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-138.gif"/></noscript></span></span>axis range changes with the self-fertilization rate. For <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-139.gif"/></noscript></span></span> we use <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F3/embed/mml-math-140.gif"/></noscript></span></span> in our model, as given by Equation 8. Further results are plotted in Section C of Supplementary File S1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">These results can be better understood by examining the underlying allele trajectories, using logic described by <a id="xref-ref-96-3" class="xref-bibr" href="#ref-96">Teshima and Przeworski (2006)</a> (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). For outcrossing populations, recessive mutations spend most of the sojourn time at low frequencies, maximizing recombination events and restoring neutral variation. These trajectories mimic sweeps from standing variation, which spend extended periods of time at low frequencies in the standing phase. Conversely, dominant mutations spend most of their time at high frequencies, so most recombination events are between haplotypes that carry the derived allele. Hence, there is a reduced chance for linked neutral alleles to recombine onto the ancestral background.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Beneficial allele trajectories. These were obtained by numerically evaluating the negative of Equation 4 forward in time. , , and h equals either 0.1 (red lines), 0.5 (black lines), or 0.9 (blue lines). Values of  and self-fertilization rates &#x3C3; used are shown for the relevant row and column. Note the different axis scales used in each panel. Further results are plotted in Section C of Supplementary File S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760725932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Beneficial allele trajectories. These were obtained by numerically evaluating the negative of Equation 4 forward in time. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-141&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-142&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and &lt;em&gt;h&lt;/em&gt; equals either 0.1 (red lines), 0.5 (black lines), or 0.9 (blue lines). Values of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-143&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and self-fertilization rates &lt;em&gt;&#x3C3;&lt;/em&gt; used are shown for the relevant row and column. Note the different &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-144&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;axis scales used in each panel. Further results are plotted in Section C of Supplementary File S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1063/F4.medium.gif" width="440" height="176"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/3/1063/F4.medium.gif" width="440" height="176"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495992" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Beneficial allele trajectories. These were obtained by numerically evaluating the negative of Equation 4 forward in time. <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F4/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F4/embed/mml-math-141.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F4/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F4/embed/mml-math-142.gif"/></noscript></span></span>, and <em>h</em> equals either 0.1 (red lines), 0.5 (black lines), or 0.9 (blue lines). Values of <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F4/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F4/embed/mml-math-143.gif"/></noscript></span></span> and self-fertilization rates <em>σ</em> used are shown for the relevant row and column. Note the different <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F4/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F4/embed/mml-math-144.gif"/></noscript></span></span>axis scales used in each panel. Further results are plotted in Section C of Supplementary File S1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">As self-fertilization increases, sweep signatures become similar to the co-dominant case as the derived allele is more likely to spread as a homozygote, weakening the influence that dominance exerts over beneficial allele trajectories. Increasing <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-145.gif"/></noscript></span></span> also causes sweeps with different dominance coefficients to produce comparable signatures, as beneficial mutation trajectories become similar after conditioning on starting at an elevated frequency.</p><p id="p-38">An analytical approximation can be obtained by using the ‘star-like’ result for <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-146.gif"/></noscript></span></span> (described in Supplementary Files S1, S2). In this case the expected pairwise diversity approximates to:</p><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-147.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span><p id="p-39">Note that Equation 10 instead uses the probability of coalescence during the standing phase, <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-148.gif"/></noscript></span></span>. This approximation reflects similar formulas for diversity following soft sweeps in haploid outcrossing populations (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Pennings and Hermisson 2006b</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Berg and Coop 2015</a>). There is a factor of two in the power term to account for two lineages. In Supplementary File S2 we demonstrate that this equation overestimates the relative diversity following a selective sweep. This mismatch arises since the star-like assumption of no coalescence during the sweep phase is only accurate for very strongly selected mutations (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Barton 1998</a>; B. Charlesworth, unpublished results). Hence it is important to consider coalescence during the sweep phase to accurately model selective sweeps that do not have an extremely high selection coefficient.</p></div><div id="sec-9" class="subsection"><h3>Site frequency spectrum</h3><p id="p-40">The star-like approximation can be used to obtain analytical solutions for the number of segregating sites and the site frequency spectrum (<em>i.e.</em>, the probability that <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-149.gif"/></noscript></span></span>, 2 … <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-150.gif"/></noscript></span></span> of <em>n</em> alleles carry derived variants). The full derivation for these statistics are outlined in Supplementary File S2, which uses the star-like approximation. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> plots the SFS (Equation A12 in Supplementary File S2) alongside simulation results. Analytical results fit the simulation data well after including an adjusted singleton class, which accounts for recent mutations that arise on the derived background during both the standing and sweep phases (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Berg and Coop 2015</a>). Including this new singleton class improves the model fit, but there remains a tendency for analytical results to underestimate the proportion of low- and high-frequency classes (<em>l</em> = 1 and 9 in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>), and overestimate the proportion of intermediate-frequency classes. Additional inaccuracies could have arisen due to the use of the star-like approximation, which assumes that there is no coalescence during the sweep phase.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expected site frequency spectrum, in flanking regions to the adaptive mutation, following a selective sweep. Lines are analytical solutions (Equation A12 in Supplementary File S2), points are simulation results. , , , and dominance coefficient  (red lines, points), 0.5 (black lines, points), or 0.9 (blue lines, points). The neutral SFS is also included for comparisons (gray dashed line). Values of , self-fertilization rates &#x3C3; and recombination distances R are shown for the relevant row and column. Results for other recombination distances are in Section E of Supplementary File S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760725932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Expected site frequency spectrum, in flanking regions to the adaptive mutation, following a selective sweep. Lines are analytical solutions (Equation A12 in Supplementary File S2), points are simulation results. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-151&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-152&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-153&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and dominance coefficient &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-154&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (red lines, points), 0.5 (black lines, points), or 0.9 (blue lines, points). The neutral SFS is also included for comparisons (gray dashed line). Values of &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-155&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, self-fertilization rates &lt;em&gt;&#x3C3;&lt;/em&gt; and recombination distances &lt;em&gt;R&lt;/em&gt; are shown for the relevant row and column. Results for other recombination distances are in Section E of Supplementary File S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1063/F5.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/3/1063/F5.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495939" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Expected site frequency spectrum, in flanking regions to the adaptive mutation, following a selective sweep. Lines are analytical solutions (Equation A12 in Supplementary File S2), points are simulation results. <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F5/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F5/embed/mml-math-151.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F5/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F5/embed/mml-math-152.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F5/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F5/embed/mml-math-153.gif"/></noscript></span></span>, and dominance coefficient <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F5/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F5/embed/mml-math-154.gif"/></noscript></span></span> (red lines, points), 0.5 (black lines, points), or 0.9 (blue lines, points). The neutral SFS is also included for comparisons (gray dashed line). Values of <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F5/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F5/embed/mml-math-155.gif"/></noscript></span></span>, self-fertilization rates <em>σ</em> and recombination distances <em>R</em> are shown for the relevant row and column. Results for other recombination distances are in Section E of Supplementary File S1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Hard sweeps in either outcrossers or partial selfers are characterized by a large number of singletons and highly-derived variants (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>), which is a typical selective sweep signature (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Braverman <em>et al.</em> 1995</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Barton 1998</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Kim and Stephan 2002</a>). As the initial frequency <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-156.gif"/></noscript></span></span> increases, so does the number of intermediate-frequency variants (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). This signature is often seen as a characteristic of soft sweeps (<a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Pennings and Hermisson 2006b</a>; <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Berg and Coop 2015</a>). Recessive hard sweeps (<span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-157.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-158.gif"/></noscript></span></span>) can produce SFS profiles that are similar to sweeps from standing variation, as there are an increased number of recombination events occurring since the allele is at a low frequency for long time periods (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). With increased self-fertilization, both hard and soft sweep signatures (<em>e.g.</em>, increased number of intermediate-frequency alleles) are recovered when measuring the SFS at a longer recombination distance than in outcrossers (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>, bottom row). This is an example of how signatures of sweeps from standing variation are extended over an increased recombination distance of around <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-159.gif"/></noscript></span></span>, as demonstrated by Equation 7.</p></div><div id="sec-10" class="subsection"><h3>Soft sweeps from recurrent mutation</h3><p id="p-43">So far, we have only focused on a soft sweep that arises from standing variation. An alternative type of soft sweep is one where recurrent mutation at the selected locus introduces the beneficial allele onto different genetic backgrounds. We can examine this case by modifying existing results. Below we derive the expected relative diversity between two alleles following this type of soft sweep, and outline the SFS for more than two samples in Supplementary File S2.</p><p id="p-44">In this model, derived alleles arise from recurrent mutation and are instantaneously beneficial (<em>i.e.</em>, there is no ‘standing phase’). During the sweep phase, lineages can escape the derived background by recombination, or if they are derived from a mutation event. If the beneficial allele is at frequency <em>p</em> then the probability of being descended from an ancestral allele by mutation is <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-160.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-161.gif"/></noscript></span></span> the mutation probability (<a id="xref-ref-75-4" class="xref-bibr" href="#ref-75">Pennings and Hermisson 2006b</a>). Denote the probability of a lineage experiencing recombination or mutation during this sweep phase by <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-162.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-163.gif"/></noscript></span></span> respectively. In both these cases the expected diversity present at linked sites is <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-164.gif"/></noscript></span></span>. If none of these events arise with probability <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-165.gif"/></noscript></span></span>, then remaining lineages can either coalesce, or they arise from independent mutation events. If they coalesce then they have approximately zero pairwise diversity between them; alternatively, they have different origins and thus exhibit the same pairwise diversity <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-166.gif"/></noscript></span></span> as the neutral background. Let <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-167.gif"/></noscript></span></span> denote the probability that mutation occurs at the sweep origin, as opposed to coalescence.</p><p id="p-45">Following this logic, the expected relative diversity for a sweep arising from recurrent mutation equals (with additional details in Supplementary File S1):</p><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-168.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span><p id="p-46"><span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-169.gif"/></noscript></span></span> denotes the diversity around a soft sweep from recurrent mutation. <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-170.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-171.gif"/></noscript></span></span> are similar to the equations used when modeling a sweep from standing variation. They are both modified to account for additional beneficial mutation arising during the sweep phase:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-172.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>where:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-173.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>and:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-174.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span></p><p id="p-47">Note that Equation 14 has an upper integral limit of <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-175.gif"/></noscript></span></span>, as opposed to a general <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-176.gif"/></noscript></span></span> used in the sweep from standing variation model, reflecting that there is no standing phase.</p><p id="p-48"><span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-177.gif"/></noscript></span></span> is the mutation probability during the sweep phase, and is similar to Equation 13 except that <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-178.gif"/></noscript></span></span> is replaced by <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-179.gif"/></noscript></span></span>, for <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-180.gif"/></noscript></span></span> is the derived allele frequency when the event occurs. <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-181.gif"/></noscript></span></span> is the probability that, at the sweep origin, the derived allele appears by mutation instead of coalescing, and is defined in a similar manner to <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-182.gif"/></noscript></span></span> (Equation 7):<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-183.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span>where <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-184.gif"/></noscript></span></span>. The coalescence probability is <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-185.gif"/></noscript></span></span>. Equation 15 implies that self–fertilization makes it more likely for beneficial mutations to coalesce at the start of a sweep, rather than arising from independent mutation events. Hence the signatures of soft sweeps via recurrent mutation will be weakened under inbreeding.</p><p id="p-49"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> compares <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-186.gif"/></noscript></span></span> in the standing variation case, and <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-187.gif"/></noscript></span></span> for the recurrent mutation case, under different levels of self-fertilization. While dominance only weakly affects sweep signatures arising from standing variation under outcrossing, it more strongly affects sweeps from recurrent mutation in outcrossing populations, as each variant arises from an initial frequency close to <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-188.gif"/></noscript></span></span> (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Second, the two models exhibit different behavior close to the selected locus (<em>R</em> close to zero). The recurrent mutation model has non–zero diversity levels, while the standing variation model exhibits zero diversity. As <em>R</em> increases, diversity eventually becomes higher for the standing variation case compared to the recurrent mutation case. We can heuristically determine when this transition occurs as follows. Assume a large population size but weak recombination and mutation rates. Hence, it is unlikely that any events occur during the sweep phase, so <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-189.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-190.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-191.gif"/></noscript></span></span>. Then the expected relative diversity (Equation 11) equals <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-192.gif"/></noscript></span></span> for a sweep from standing variation, and <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-193.gif"/></noscript></span></span> for one from recurrent mutation. To find the recombination rate <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-194.gif"/></noscript></span></span> at which a sweep from recurrent mutation yields higher diversity than one from standing variation, we find the <em>R</em> value needed to equate the two probabilities, giving:</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparing sweeps from recurrent mutation to those from standing variation. Top row: comparing relative diversity following a soft sweep, from either standing variation (Equation 9 with , solid lines) or recurrent mutation (using Equation 11 with , dashed lines). , , and dominance coefficient  (red lines), 0.5 (black lines), or 0.9 (blue lines). Bottom row: the ratio of the diversity following a sweep from standing variation to one from recurrent mutation. Parameters for each panel are as in the respective plot for the top row. Vertical dashed black line indicates  (the approximate form of Equation 16); horizontal dashed line in the bottom-row plots show when the ratio equals 1. Note the different axis between left- and right-hand panels. Results are also plotted in Section F of Supplementary File S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-760725932" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparing sweeps from recurrent mutation to those from standing variation. Top row: comparing relative diversity following a soft sweep, from either standing variation (Equation 9 with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-195&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, solid lines) or recurrent mutation (using Equation 11 with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-196&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, dashed lines). &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-197&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-198&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, and dominance coefficient &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-199&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (red lines), 0.5 (black lines), or 0.9 (blue lines). Bottom row: the ratio of the diversity following a sweep from standing variation to one from recurrent mutation. Parameters for each panel are as in the respective plot for the top row. Vertical dashed black line indicates &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-200&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (the approximate form of Equation 16); horizontal dashed line in the bottom-row plots show when the ratio equals 1. Note the different &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-201&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;axis between left- and right-hand panels. Results are also plotted in Section F of Supplementary File S1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1063/F6.medium.gif" width="440" height="284"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/3/1063/F6.medium.gif" width="440" height="284"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1063/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/496012" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-50" class="first-child">Comparing sweeps from recurrent mutation to those from standing variation. Top row: comparing relative diversity following a soft sweep, from either standing variation (Equation 9 with <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-195.gif"/></noscript></span></span>, solid lines) or recurrent mutation (using Equation 11 with <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-196.gif"/></noscript></span></span>, dashed lines). <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-197.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-198.gif"/></noscript></span></span>, and dominance coefficient <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-199.gif"/></noscript></span></span> (red lines), 0.5 (black lines), or 0.9 (blue lines). Bottom row: the ratio of the diversity following a sweep from standing variation to one from recurrent mutation. Parameters for each panel are as in the respective plot for the top row. Vertical dashed black line indicates <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-200.gif"/></noscript></span></span> (the approximate form of Equation 16); horizontal dashed line in the bottom-row plots show when the ratio equals 1. Note the different <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/F6/embed/mml-math-201.gif"/></noscript></span></span>axis between left- and right-hand panels. Results are also plotted in Section F of Supplementary File S1.</p><div class="sb-div caption-clear"></div></div></div><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-202.gif"/></noscript></span><span class="disp-formula-label">(16)</span></span><p id="p-51">The last approximation arises as <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-203.gif"/></noscript></span></span>. Hence for a fixed <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-204.gif"/></noscript></span></span>, the window where recurrent mutations create higher diversity near the selected locus increases for lower <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-205.gif"/></noscript></span></span> or higher <em>F</em>, since both these factors reduces the potential for recombination to create new haplotypes during the standing phase. Equation 16 is generally accurate when sweeps from standing variation have higher diversity than sweeps with recurrent mutations (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, bottom row), but becomes inaccurate for <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-206.gif"/></noscript></span></span> in outcrossing populations, as some events are likely to occur during the sweep phase. In Supplementary File S2 we show how similar results apply to the SFS.</p></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><div id="sec-12" class="subsection"><h3>Summary of theoretical findings</h3><p id="p-52">While there has been many investigations into how different sweep processes can be detected from next-generation sequence data (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Pritchard and Di Rienzo 2010</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Messer and Petrov 2013</a>; <a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Stephan 2016</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Hermisson and Pennings 2017</a>), these models generally assumed idealised randomly mating populations and beneficial mutations that are semidominant (<span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-207.gif"/></noscript></span></span>). Here we have created a more general selective sweep model, with arbitrary self-fertilization and dominance levels. Our principal focus is on comparing a hard sweep arising from a single allele copy to a soft sweep arising from standing variation, but we also consider the case of recurrent mutation (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>).</p><p id="p-53">We find that the qualitative patterns of different selective sweeps under selfing remain similar to expectations from outcrossing models. In particular, a sweep from standing variation still creates an elevated number of intermediate-frequency variants compared to a sweep from <em>de novo</em> mutation (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figures 5</a>, <a id="xref-fig-6-4" class="xref-fig" href="#F6">6</a>). This pattern is standard for soft sweeps (<a id="xref-ref-75-5" class="xref-bibr" href="#ref-75">Pennings and Hermisson 2006b</a>; <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Messer and Petrov 2013</a>; <a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Berg and Coop 2015</a>; <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Hermisson and Pennings 2017</a>) so existing statistical methods for detecting them (<em>e.g.</em>, observing an higher than expected number of haplotypes; <a id="xref-ref-101-2" class="xref-bibr" href="#ref-101">Vitti <em>et al.</em> (2013)</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Garud <em>et al.</em> (2015)</a>) can, in principle, also be applied to selfing organisms. Under self-fertilization, these signatures are stretched over longer physical regions than in outcrossers. These extensions arise as self-fertilization affects gene genealogies during both the sweep and standing phases in different ways. During the sweep phase, beneficial alleles fix more rapidly under higher self-fertilization as homozygous mutations are created more rapidly (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Charlesworth 1992</a>; <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Glémin 2012</a>). In addition, the effective recombination rate is reduced by approximately <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-208.gif"/></noscript></span></span> (<a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Nordborg <em>et al.</em> 1996</a>; <a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Nordborg 2000</a>; <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Charlesworth and Charlesworth 2010</a>), and slightly more for highly inbred populations (<a id="xref-ref-83-3" class="xref-bibr" href="#ref-83">Roze 2009</a>, <a id="xref-ref-84-3" class="xref-bibr" href="#ref-84">2016</a>). These two effects mean that neutral variants linked to an adaptive allele are less likely to recombine onto the neutral background during the sweep phase, as reflected in Equation 3 for <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-209.gif"/></noscript></span></span>. During the standing phase, two haplotypes are more likely to coalesce under high levels of self-fertilization since <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-210.gif"/></noscript></span></span> is decreased by a factor <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-211.gif"/></noscript></span></span> (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Pollak 1987</a>; <a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Charlesworth 1992</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Caballero and Hill 1992</a>; <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Nordborg and Donnelly 1997</a>). This effect, combined with a reduced effective recombination rate, means that the overall recombination probability during the standing phase is reduced by a factor <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-212.gif"/></noscript></span></span> (Equation 7). Hence intermediate-frequency variants, which could provide evidence of adaptation from standing variation, will be spread out over longer genomic regions (this result can be seen in the site–frequency spectrum results, <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>). The elongation of sweep signatures means sweeps from standing variation can be easier to detect in selfing organisms than in outcrossers. Conversely, sweeps from recurrent mutation will have weakened signatures under self–fertilization. This result is due to a reduced effective population size, making it likelier that lineages trace back to a common ancestor rather than independent mutation events.</p><p id="p-54">We have also investigated how dominance affects soft sweep signatures, since previous analyses have only focused on how dominance affects hard sweeps (<a id="xref-ref-96-4" class="xref-bibr" href="#ref-96">Teshima and Przeworski 2006</a>; <a id="xref-ref-95-2" class="xref-bibr" href="#ref-95">Teshima <em>et al.</em> 2006</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Ewing <em>et al.</em> 2011</a>). In outcrossing organisms, recessive mutations leave weaker sweep signatures than additive or dominant mutations as they spend more time at low frequencies, increasing the amount of recombination that restores neutral variation (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figures 3</a>, <a id="xref-fig-4-4" class="xref-fig" href="#F4">4</a>). With increased self-fertilization, dominance has a weaker impact on sweep signatures as most mutations are homozygous (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). We also show that the SFS for recessive alleles can resemble a soft sweep, with a higher number of intermediate-frequency variants than for other hard sweeps (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>). Dominance only weakly affects sweeps from standing variation, as trajectories of beneficial alleles become similar once the variant’s initial frequency exceeds <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-213.gif"/></noscript></span></span> (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3</a>, <a id="xref-fig-4-6" class="xref-fig" href="#F4">4</a>). Yet different dominance levels can affect sweep signatures if the beneficial allele is reintroduced by recurrent mutation (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>). Hence if one wishes to understand how dominance affects sweep signatures, it is also important to consider which processes underlie observed patterns of genetic diversity.</p><p id="p-55">These results also demonstrate that the effects of dominance on sweeps are not necessarily intuitive. For example, both highly dominant and recessive mutations have elongated fixation times compared to co–dominant mutations (<a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">Glémin 2012</a>). Based on this intuition, one could expect both dominant and recessive mutations to both produce weaker sweep signatures than co-dominant ones. In practice, dominant mutations have similar sweep signatures to co–dominant mutations (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figures 3</a>, <a id="xref-fig-5-9" class="xref-fig" href="#F5">5</a>), and recessive sweeps could produce similar signatures as sweeps from standing variation (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5</a>). Dominance also has a weaker impact on sweeps from standing variation (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figures 3</a>, <a id="xref-fig-5-11" class="xref-fig" href="#F5">5</a>).</p></div><div id="sec-13" class="subsection"><h3>Soft sweeps from recurrent mutation or standing variation?</h3><p id="p-56">These theoretical results shed light onto how to distinguish between soft sweeps that arise either from standing variation, or from recurrent mutation. Both models are characterized by an elevated number of intermediate-frequency variants, in comparison to a hard sweep. Yet sweeps arising from recurrent mutation have non–zero diversity at the selected locus, whereas a sweep from standing variation exhibits approximately zero diversity. Hence a sweep from recurrent mutation shows intermediate-frequency variants closer to the beneficial locus, compared to sweeps from standing variation (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figures 6</a> and C in Supplementary File S2). Further from the selected locus, a sweep from standing variation exhibits greater variation than one from recurrent mutation, due to recombinant haplotypes being created during the standing phase. Equation 16 provides a simple condition for <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/1063/embed/mml-math-214.gif"/></noscript></span></span>, the recombination distance needed for a sweep from standing variation to exhibit higher diversity than one from recurrent mutation; from this equation, we see that the size of this region increases under higher self-fertilization. Hence it may be easier to differentiate between these two sweep scenarios in self–fertilizing organisms.</p><p id="p-57">Differences in haplotype structure between sweeps from either standing variation or recurrent mutation should be more pronounced in self-fertilizing organisms, due to the reduction in effective recombination rates. However, when investigating sweep patterns over broad genetic regions, it becomes likelier that genetic diversity will be affected by multiple beneficial mutations spreading throughout the genome. Competing selective sweeps can lead to elevated diversity near a target locus for two reasons. First, selection interference increases the fixation time of individual mutations, allowing more recombination that can restore neutral diversity (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Kim and Stephan 2003</a>). In addition, competing selective sweeps can drag different sets of neutral variation to fixation. Selective sweep signatures in data tend to be asymmetric, and this effect will exacerbate this asymmetry (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Chevin <em>et al.</em> 2008</a>). Further investigations of selective sweep patterns across long genetic distances will prove to be a rich area of future research.</p><p id="p-58">Finally, we have assumed a fixed population size, and that sweeps from standing variation arose from neutral variation. The resulting signatures could differ if the population size has changed over time (<a id="xref-ref-107-2" class="xref-bibr" href="#ref-107">Wilson <em>et al.</em> 2014</a>), if populations are structured (<a id="xref-ref-111-1" class="xref-bibr" href="#ref-111">Zheng and Wiehe 2019</a>), or if the beneficial allele was previously deleterious (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Orr and Betancourt 2001</a>). Both issues could also affect our ability to discriminate between soft and hard sweeps.</p></div><div id="sec-14" class="subsection"><h3>Potential applications to self-fertilizing organisms</h3><p id="p-59">Existing methods for finding sweep signatures in nucleotide polymorphism data are commonly based on finding regions with a site-frequency spectrum matching what is expected under a selective sweep (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Nielsen <em>et al.</em> 2005</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Boitard <em>et al.</em> 2009</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Pavlidis <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">DeGiorgio <em>et al.</em> 2016</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Huber <em>et al.</em> 2016</a>). The more general models developed here can be used to create more specific sweep-detection methods that include self-fertilization. However, a recent analysis found that soft-sweep signatures can be incorrectly inferred if analyzing genetic regions that flank hard sweeps, which was named the ‘soft shoulder’ effect (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Schrider <em>et al.</em> 2015</a>). Due to the reduction in recombination in selfers, these model results indicate that ‘soft-shoulder’ footprints can arise over long genetic distances and should be taken into account. One remedy to this problem is to not just classify genetic regions as being subject to either a hard or soft sweep, but also as being linked to a region subject to one of these sweeps (<a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Schrider and Kern 2016</a>). These more general calculations can also be extended to quantify to what extent background selection and sweeps jointly shape genome-wide diversity in self-fertilizing organisms (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Elyashiv <em>et al.</em> 2016</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Campos <em>et al.</em> 2017</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Booker and Keightley 2018</a>; <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Rettelbach <em>et al.</em> 2019</a>), or detect patterns of introgression (<a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Setter <em>et al.</em> 2019</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">We would like to thank Sally Otto for providing information on the elevated effective starting frequency of beneficial mutations; Brian Charlesworth on providing advice on modeling selective sweeps, sharing unpublished results, and providing comments on the manuscript; Ben Haller for answering questions about SLiM; Nick Barton, Jeffrey Ross-Ibarra and other anonymous referees for providing feedback on the manuscript. MH was supported by a Marie Curie International Outgoing Fellowship (MC-IOF-622936) and a NERC Independent Research Fellowship (NE/R015686/1). MH and TB also acknowledge financial support from the European Research Council under the European Union’s Seventh Framework Program (FP7/20072013, ERC Grant 311341).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-61">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11687949">https://doi.org/10.25387/g3.11687949</a>.</p></li><li class="fn" id="fn-2"><p id="p-62"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-11-18"><span class="received-label">Received </span>November 18, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2020-01-18"><span class="accepted-label">Accepted </span>January 18, 2020.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Hartfield, Bataillon</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.1" data-doi="10.1038/ng.1050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>,  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Gerke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Shapiro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Crissman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ghosh</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosome-scale selective sweeps shape <em>Caenorhabditis elegans</em> genomic diversity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">290</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.1050</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAndersen%26rft.auinit1%253DE.%2BC.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D290%26rft.atitle%253DChromosome-scale%2Bselective%2Bsweeps%2Bshape%2BCaenorhabditis%2Belegans%2Bgenomic%2Bdiversity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1050%26rft_id%253Dinfo%253Apmid%252F22286215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1050&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22286215&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.2" data-doi="10.1093/molbev/msw228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>,  <span class="cit-name-given-names">T. J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nair</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">McDew-White</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. H.</span> <span class="cit-name-surname">Cheeseman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nkhoma</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Population parameters underlying an ongoing soft sweep in southeast asian malaria parasites.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw228</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw228%26rft_id%253Dinfo%253Apmid%252F28025270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28025270&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.3" data-doi="10.1111/mec.13976"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badouin</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gladieux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gouzy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Siguenza</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Aguileta</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Widespread selective sweeps throughout the genome of model plant pathogenic fungi and identification of effector candidates.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2041</span>–<span class="cit-lpage">2062</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13976</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D2041%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13976&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.4" data-doi="10.1016/j.tree.2007.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>,  <span class="cit-name-given-names">R. D. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Schluter</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Adaptation from standing genetic variation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">44</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2007.09.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DBarrett%26rft.auinit1%253DR.%2BD.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D44%26rft.atitle%253DAdaptation%2Bfrom%2Bstanding%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2007.09.008%26rft_id%253Dinfo%253Apmid%252F18006185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2007.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18006185&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252767300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.5" data-doi="10.1017/S0016672398003462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>,  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The effect of hitch-hiking on neutral genealogies.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">133</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672398003462</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D72%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672398003462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672398003462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000077703600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.6" data-doi="10.1534/genetics.115.178962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>,  <span class="cit-name-given-names">J. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coop</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A coalescent model for a sweep of a unique standing variant.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">725</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.178962</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.178962%26rft_id%253Dinfo%253Apmid%252F26311475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNzA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.7" data-doi="10.1111/j.1469-185X.2010.00153.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Billiard</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">López-Villavicencio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Devier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Hood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fairhead</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Having sex, yes, but with whom? Inferences from fungi on the evolution of anisogamy and mating types.</span> <abbr class="cit-jnl-abbrev">Biol. Rev. Camb. Philos. Soc.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">442</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-185X.2010.00153.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.stitle%253DBiological%2BReviews%2B%2528Cambridge%253B%2BPrint%2529%26rft.aulast%253DBilliard%26rft.auinit1%253DS.%26rft.volume%253D86%26rft.issue%253D2%26rft.spage%253D421%26rft.epage%253D442%26rft.atitle%253DHaving%2Bsex%252C%2Byes%252C%2Bbut%2Bwith%2Bwhom%253F%2BInferences%2Bfrom%2Bfungi%2Bon%2Bthe%2Bevolution%2Bof%2Banisogamy%2Band%2Bmating%2Btypes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-185X.2010.00153.x%26rft_id%253Dinfo%253Apmid%252F21489122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-185X.2010.00153.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21489122&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.8" data-doi="10.1534/genetics.108.100032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boitard</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Schlötterer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Futschik</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detecting selective sweeps: A new approach based on hidden markov models.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1578</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.100032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.100032%26rft_id%253Dinfo%253Apmid%252F19204373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MS80LzE1NjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.9" data-doi="10.1093/molbev/msv092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonhomme</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Boitard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">San Clemente</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Dumas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Young</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic signature of selective sweeps illuminates adaptation of <em>Medicago truncatula</em> to root-associated microorganisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2097</span>–<span class="cit-lpage">2110</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msv092</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv092%26rft_id%253Dinfo%253Apmid%252F25901015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25901015&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.10" data-doi="10.1093/molbev/msy188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Booker</span>,  <span class="cit-name-given-names">T. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Keightley</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Understanding the factors that shape patterns of nucleotide diversity in the house mouse genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">2971</span>–<span class="cit-lpage">2988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D35%26rft.spage%253D2971%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braverman</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Hudson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. L.</span> <span class="cit-name-surname">Kaplan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Langley</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The hitchhiking effect on the site frequency spectrum of DNA polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBraverman%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D140%26rft.issue%253D2%26rft.spage%253D783%26rft.epage%253D796%26rft.atitle%253DThe%2Bhitchhiking%2Beffect%2Bon%2Bthe%2Bsite%2Bfrequency%2Bspectrum%2Bof%2BDNA%2Bpolymorphisms.%26rft_id%253Dinfo%253Apmid%252F7498754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQwLzIvNzgzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caballero</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Hill</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Effects of partial inbreeding on fixation rates and variation of mutant genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCaballero%26rft.auinit1%253DA.%26rft.volume%253D131%26rft.issue%253D2%26rft.spage%253D493%26rft.epage%253D507%26rft.atitle%253DEffects%2Bof%2Bpartial%2Binbreeding%2Bon%2Bfixation%2Brates%2Band%2Bvariation%2Bof%2Bmutant%2Bgenes.%26rft_id%253Dinfo%253Apmid%252F1644283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMxLzIvNDkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.13" data-doi="10.1534/genetics.119.301951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>,  <span class="cit-name-given-names">J. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">The effects on neutral variability of recurrent selective sweeps and background selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.119.301951</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.119.301951%26rft_id%253Dinfo%253Apmid%252F30923166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEyLzEvMjg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.14" data-doi="10.1073/pnas.1619434114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhao</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Estimating the parameters of background selection and selective sweeps in drosophila in the presence of gene conversion.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">E4762</span>–<span class="cit-lpage">E4771</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1619434114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1619434114%26rft_id%253Dinfo%253Apmid%252F28559322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE0LzI0L0U0NzYyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.15" data-doi="10.1086/285406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Evolutionary rates in partially self-fertilizing species.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">148</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/285406</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D140%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.1086%252F285406%26rft_id%253Dinfo%253Apmid%252F19426068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/285406&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19426068&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JA50800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.1063.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2020</span> How long does it take to fix a favorable mutation, and why should we care? Am. Nat. Early Online.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/708187</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.1063.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Elements of Evolutionary Genetics</span>, <span class="cit-publ-name">Roberts &amp; Company Publishers</span>, <span class="cit-publ-loc">Greenwood Village, Colo.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.18" data-doi="10.1534/genetics.108.089706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chevin</span>,  <span class="cit-name-given-names">L.-M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Billiard</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Hospital</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Hitchhiking both ways: Effect of two interfering selective sweeps on linked neutral variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.089706</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089706%26rft_id%253Dinfo%253Apmid%252F18716333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvMzAxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.19" data-doi="10.1093/bioinformatics/btw051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeGiorgio</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Huber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Hubisz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Hellmann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">SweepFinder2: increased sensitivity, robustness and flexibility.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1895</span>–<span class="cit-lpage">1897</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btw051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw051%26rft_id%253Dinfo%253Apmid%252F27153702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27153702&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.20" data-doi="10.1534/genetics.106.067678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>,  <span class="cit-name-given-names">M. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Fisher</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Beneficial mutation-selection balance and the effect of linkage on positive selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1798</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.067678</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067678%26rft_id%253Dinfo%253Apmid%252F17483432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE3NTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.21" data-doi="10.1371/journal.pgen.1006130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elyashiv</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sattath</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. T.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Strutsovsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">McVicker</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A genomic map of the effects of linked selection in Drosophila.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1006130</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006130</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1006130%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006130&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.22" data-doi="10.1093/bioinformatics/btq322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MSMS: a coalescent simulation program including recombination, demographic structure and selection at a single locus.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2064</span>–<span class="cit-lpage">2065</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq322%26rft_id%253Dinfo%253Apmid%252F20591904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20591904&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280703500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.23" data-doi="10.1007/s00285-010-0382-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pfaffelhuber</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rudolf</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Selective sweeps for recessive alleles and for other modes of dominance.</span> <abbr class="cit-jnl-abbrev">J. Math. Biol.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">431</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00285-010-0382-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DJ%2BMath%2BBiol%26rft.aulast%253DEwing%26rft.auinit1%253DG.%26rft.volume%253D63%26rft.issue%253D3%26rft.spage%253D399%26rft.epage%253D431%26rft.atitle%253DSelective%2Bsweeps%2Bfor%2Brecessive%2Balleles%2Band%2Bfor%2Bother%2Bmodes%2Bof%2Bdominance.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00285-010-0382-4%26rft_id%253Dinfo%253Apmid%252F21076829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00285-010-0382-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21076829&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>,  <span class="cit-name-given-names">J. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.-I.</span> <span class="cit-name-surname">Wu</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Hitchhiking Under Positive Darwinian Selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1405%26rft.epage%253D1413%26rft.atitle%253DHitchhiking%2BUnder%2BPositive%2BDarwinian%2BSelection%26rft_id%253Dinfo%253Apmid%252F10880498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzE0MDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.25" data-doi="10.1093/molbev/msu077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrer-Admetlla</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Liang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Korneliussen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">On detecting incomplete soft or hard selective sweeps using haplotype structure.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1275</span>–<span class="cit-lpage">1291</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu077</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu077%26rft_id%253Dinfo%253Apmid%252F24554778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu077&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24554778&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335914400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.26" data-doi="10.1017/S0370164600023993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>,  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1922</span> <span class="cit-article-title">On the dominance ratio.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Edinb.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0370164600023993</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D42%26rft.spage%253D321%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0370164600023993%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0370164600023993&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.27" data-doi="10.1266/ggs.18-00008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fujito</span>,  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Satta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hayakawa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Takahata</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A new inference method for detecting an ongoing selective sweep.</span> <abbr class="cit-jnl-abbrev">Genes Genet. Syst.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1266/ggs.18-00008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenet.%2BSyst.%26rft.volume%253D93%26rft.spage%253D149%26rft_id%253Dinfo%253Adoi%252F10.1266%252Fggs.18-00008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1266/ggs.18-00008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.28" data-doi="10.1093/mol-bev/msx300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fulgione</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Koornneef</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Roux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Hancock</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Madeiran <em>Arabidopsis thaliana</em> reveals ancient long-range colonization and clarifies demography in Eurasia.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">574</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D35%26rft.spage%253D564%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmol-bev%252Fmsx300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mol-bev/msx300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.29" data-doi="10.1111/mec.14082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fustier</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Brandenburg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Boitard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lapeyronnie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. E.</span> <span class="cit-name-surname">Eguiarte</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Signatures of local adaptation in lowland and highland teosintes from whole-genome sequencing of pooled samples.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2738</span>–<span class="cit-lpage">2756</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14082</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D2738%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.30" data-doi="10.1371/journal.pgen.1005004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garud</span>,  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. O.</span> <span class="cit-name-surname">Buzbas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recent Selective Sweeps in North American <em>Drosophila melanogaster</em> Show Signatures of Soft Sweeps.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005004</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005004%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005004%26rft_id%253Dinfo%253Apmid%252F25706129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25706129&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.31" data-doi="10.1534/genetics.115.184002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garud</span>,  <span class="cit-name-given-names">N. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Elevated linkage disequilibrium and signatures of soft sweeps are common in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">863</span>–<span class="cit-lpage">880</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.184002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.184002%26rft_id%253Dinfo%253Apmid%252F27098909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzIvODYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.32" data-doi="10.1016/j.tpb.2012.02.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glémin</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extinction and fixation times with dominance and inbreeding.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tpb.2012.02.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DGlemin%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D4%26rft.spage%253D310%26rft.epage%253D316%26rft.atitle%253DExtinction%2Band%2Bfixation%2Btimes%2Bwith%2Bdominance%2Band%2Binbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2012.02.006%26rft_id%253Dinfo%253Apmid%252F22425679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2012.02.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22425679&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.33" data-doi="10.1111/j.1558-5646.2012.01778.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glémin</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ronfort</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Adaptation and maladaptation in selfing and outcrossing species: new mutations versus standing variation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">240</span>. Erratum: 3381.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.2012.01778.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01778.x%26rft_id%253Dinfo%253Apmid%252F23289574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01778.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23289574&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.34" data-doi="10.1017/S0305004100015644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>,  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1927</span> <span class="cit-article-title">A mathematical theory of natural and artificial selection, part V: Selection and mutation.</span> <abbr class="cit-jnl-abbrev">Math. Proc. Camb. Philos. Soc.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">838</span>–<span class="cit-lpage">844</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0305004100015644</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMath.%2BProc.%2BCamb.%2BPhilos.%2BSoc.%26rft.volume%253D23%26rft.spage%253D838%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0305004100015644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0305004100015644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.35" data-doi="10.1093/molbev/msy228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>,  <span class="cit-name-given-names">B. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">SLiM 3: Forward Genetic Simulations Beyond the Wright–Fisher Model.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">632</span>–<span class="cit-lpage">637</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msy228</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsy228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msy228&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.36" data-doi="10.1101/446005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">A. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">DeGiorgio</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Identifying and classifying shared selective sweeps from multilocus data.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: 446005<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/446005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F446005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0NDYwMDV2MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTA2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.37" data-doi="doi: 10.1101/678722"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">A. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">DeGiorgio</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">A likelihood approach for uncovering selective sweep signatures from haplotype data.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: 678722<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>doi: 10.1101/678722</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252Fdoi%253A%2B10.1101%252F678722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi: 10.1101/678722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.38" data-doi="10.1534/genetics.118.301502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. R.</span> <span class="cit-name-surname">Garud</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">DeGiorgio</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">a</span> <span class="cit-article-title">Detection and classification of hard and soft sweeps from unphased genotypes by multilocus genotype identity.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">1429</span>–<span class="cit-lpage">1452</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301502</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301502%26rft_id%253Dinfo%253Apmid%252F30315068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIxMC80LzE0MjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.39" data-doi="10.1371/journal.pgen.1007859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sackman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Jensen</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span><span class="cit-comment">b</span> <span class="cit-article-title">On the unfounded enthusiasm for soft selective sweeps II: Examining recent evidence from humans, flies, and viruses.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e1007859</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007859</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D14%26rft.spage%253De1007859%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.40" data-doi="10.1016/j.tig.2017.04.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartfield</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bataillon</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Glémin</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The evolutionary interplay between adaptation and self-fertilization.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">420</span>–<span class="cit-lpage">431</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2017.04.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D33%26rft.spage%253D420%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2017.04.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2017.04.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.41" data-doi="10.1534/genetics.113.158196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartfield</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Glémin</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Hitchhiking of deleterious alleles and the cost of adaptation in partially selfing species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.158196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.158196%26rft_id%253Dinfo%253Apmid%252F24240529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzEvMjgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.42" data-doi="10.1534/genetics.116.188821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartfield</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Glémin</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Limits to adaptation in partially selfing species.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">974</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.188821</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.188821%26rft_id%253Dinfo%253Apmid%252F27098913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzIvOTU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hedrick</span>,  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Hitchhiking: A comparison of linkage and partial selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">791</span>–<span class="cit-lpage">808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHedrick%26rft.auinit1%253DP.%2BW.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D791%26rft.epage%253D808%26rft.atitle%253DHITCHHIKING%253A%2BA%2BCOMPARISON%2BOF%2BLINKAGE%2BAND%2BPARTIAL%2BSELFING%26rft_id%253Dinfo%253Apmid%252F6931092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiOTQvMy83OTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.44" data-doi="10.1534/genetics.104.036947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Pennings</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Soft sweeps: Molecular population genetics of adaptation from standing genetic variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">2335</span>–<span class="cit-lpage">2352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.036947</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.036947%26rft_id%253Dinfo%253Apmid%252F15716498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS80LzIzMzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.45" data-doi="10.1111/2041-210X.12808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Pennings</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Soft sweeps and beyond: understanding the patterns and probabilities of selection footprints under rapid adaptation.</span> <abbr class="cit-jnl-abbrev">Methods Ecol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">700</span>–<span class="cit-lpage">716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/2041-210X.12808</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEcol.%2BEvol.%26rft.volume%253D8%26rft.spage%253D700%26rft_id%253Dinfo%253Adoi%252F10.1111%252F2041-210X.12808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/2041-210X.12808&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.46" data-doi="10.1111/mec.13351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>,  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">DeGiorgio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Hellmann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Detecting recent selective sweeps while controlling for mutation rate and background selection.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13351%26rft_id%253Dinfo%253Apmid%252F26290347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26290347&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.47" data-doi="10.1093/molbev/msu247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>,  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nordborg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Hellmann</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Keeping It Local: Evidence for Positive Selection in Swedish <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3026</span>–<span class="cit-lpage">3039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu247</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu247%26rft_id%253Dinfo%253Apmid%252F25158800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25158800&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.48" data-doi="10.1111/j.0014-3820.2006.tb01186.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Igic</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Kohn</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The distribution of plant mating systems: study bias against obligately outcrossing species.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">1098</span>–<span class="cit-lpage">1103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2006.tb01186.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DIgic%26rft.auinit1%253DB.%26rft.volume%253D60%26rft.issue%253D5%26rft.spage%253D1098%26rft.epage%253D1103%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bplant%2Bmating%2Bsystems%253A%2Bstudy%2Bbias%2Bagainst%2Bobligately%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2006.tb01186.x%26rft_id%253Dinfo%253Apmid%252F16817548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2006.tb01186.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16817548&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238181700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.49" data-doi="10.1073/pnas.0401720101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kim</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Pattern of polymorphism after strong artificial selection in a domestication event.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">10667</span>–<span class="cit-lpage">10672</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0401720101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0401720101%26rft_id%253Dinfo%253Apmid%252F15249682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzI5LzEwNjY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nordborg</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The extent of linkage disequilibrium and haplotype sharing around a polymorphic site.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D165%26rft.issue%253D1%26rft.spage%253D437%26rft.epage%253D444%26rft.atitle%253DThe%2BExtent%2Bof%2BLinkage%2BDisequilibrium%2Band%2BHaplotype%2BSharing%2BAround%2Ba%2BPolymorphic%2BSite%26rft_id%253Dinfo%253Apmid%252F14504248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY1LzEvNDM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.51" data-doi="10.1111/j.0014-3820.2006.tb00525.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarne</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Auld</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Animals mix it up too: the distribution of self-fertilization among hermaphroditic animals.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">1816</span>–<span class="cit-lpage">1824</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2006.tb00525.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DJarne%26rft.auinit1%253DP.%26rft.volume%253D60%26rft.issue%253D9%26rft.spage%253D1816%26rft.epage%253D1824%26rft.atitle%253DAnimals%2Bmix%2Bit%2Bup%2Btoo%253A%2Bthe%2Bdistribution%2Bof%2Bself-fertilization%2Bamong%2Bhermaphroditic%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2006.tb00525.x%26rft_id%253Dinfo%253Apmid%252F17089966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2006.tb00525.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17089966&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241226800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.52" data-doi="10.1038/ncomms6281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>,  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">On the unfounded enthusiasm for soft selective sweeps.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5281</span>. <a href="https://doi.org/10.1038/ncomms6281">https://doi.org/10.1038/ncomms6281</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5281%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6281%26rft_id%253Dinfo%253Apmid%252F25345443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25345443&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>,  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Hudson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Langley</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The “hitchhiking effect” revisited.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">887</span>–<span class="cit-lpage">899</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKaplan%26rft.auinit1%253DN.%2BL.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D887%26rft.epage%253D899%26rft.atitle%253DThe%2B%2522hitchhiking%2Beffect%2522%2Brevisited.%26rft_id%253Dinfo%253Apmid%252F2612899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzQvODg3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.54" data-doi="10.1371/journal.pgen.1000924"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karasov</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evidence that adaptation in <em>Drosophila</em> is not limited by mutation at single sites.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e1000924</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000924</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKarasov%26rft.auinit1%253DT.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De1000924%26rft.epage%253De1000924%26rft.atitle%253DEvidence%2Bthat%2Badaptation%2Bin%2BDrosophila%2Bis%2Bnot%2Blimited%2Bby%2Bmutation%2Bat%2Bsingle%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000924%26rft_id%253Dinfo%253Apmid%252F20585551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000924&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20585551&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.55" data-doi="10.1534/g3.118.200262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kern</span>,  <span class="cit-name-given-names">A. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Schrider</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">diploS/HIC: An updated approach to classifying selective sweeps.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1959</span>–<span class="cit-lpage">1970</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200262%26rft_id%253Dinfo%253Apmid%252F29626082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvNi8xOTU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.56" data-doi="10.1534/genetics.103.025387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Linkage disequilibrium as a signature of selective sweeps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1524</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.103.025387</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKim%26rft.auinit1%253DY.%26rft.volume%253D167%26rft.issue%253D3%26rft.spage%253D1513%26rft.epage%253D1524%26rft.atitle%253DLinkage%2BDisequilibrium%2Bas%2Ba%2BSignature%2Bof%2BSelective%2BSweeps%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.025387%26rft_id%253Dinfo%253Apmid%252F15280259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ny8zLzE1MTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Detecting a local signature of genetic hitchhiking along a recombining chromosome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">765</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKim%26rft.auinit1%253DY.%26rft.volume%253D160%26rft.issue%253D2%26rft.spage%253D765%26rft.epage%253D777%26rft.atitle%253DDetecting%2Ba%2BLocal%2BSignature%2Bof%2BGenetic%2BHitchhiking%2BAlong%2Ba%2BRecombining%2BChromosome%26rft_id%253Dinfo%253Apmid%252F11861577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYwLzIvNzY1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Stephan</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Selective sweeps in the presence of interference among partially linked loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">398</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKim%26rft.auinit1%253DY.%26rft.volume%253D164%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D398%26rft.atitle%253DSelective%2BSweeps%2Bin%2Bthe%2BPresence%2Bof%2BInterference%2BAmong%2BPartially%2BLinked%2BLoci%26rft_id%253Dinfo%253Apmid%252F12750349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzEvMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laporte</span>,  <span class="cit-name-given-names">V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Effective population size and population subdivision in demographically structured populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">519</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLaporte%26rft.auinit1%253DV.%26rft.volume%253D162%26rft.issue%253D1%26rft.spage%253D501%26rft.epage%253D519%26rft.atitle%253DEffective%2BPopulation%2BSize%2Band%2BPopulation%2BSubdivision%2Bin%2BDemographically%2BStructured%2BPopulations%26rft_id%253Dinfo%253Apmid%252F12242257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYyLzEvNTAxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.60"><div class="cit-metadata unstructured">Laval, G., E. Patin, P. Boutillier, and L. Quintana-Murci, 2019 A genome-wide approximate bayesian computation approach suggests only limited numbers of soft sweeps in humans over the last 100,000 years. bioRxiv p. 2019.12.22.886234.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.61" data-doi="10.1038/ng.2678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Rabanal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Meng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Huber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Farlow</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Massive genomic variation and strong selection in <em>Arabidopsis thaliana</em> lines from Sweden.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">884</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2678</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D884%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2678%26rft_id%253Dinfo%253Apmid%252F23793030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23793030&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.62" data-doi="10.1016/j.tpb.2015.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lambert</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A simple, semi-deterministic approximation to the distribution of selective sweeps in large populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tpb.2015.01.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPopul.%2BBiol.%26rft.volume%253D101%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2015.01.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2015.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.63" data-doi="10.1086/283071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maynard Smith</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">What determines the rate of evolution?</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/283071</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D110%26rft.spage%253D331%26rft_id%253Dinfo%253Adoi%252F10.1086%252F283071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/283071&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976BZ01600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.3.1063.64" data-doi="10.1017/S0016672300014634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maynard Smith</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Haigh</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The hitch-hiking effect of a favourable gene.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300014634</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D35%26rft.atitle%253DThe%2Bhitch-hiking%2Beffect%2Bof%2Ba%2Bfavourable%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014634%26rft_id%253Dinfo%253Apmid%252F4407212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4407212&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974T216900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.65" data-doi="10.1534/genetics.106.062828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>,  <span class="cit-name-given-names">G. A. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The structure of linkage disequilibrium around a selective sweep.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1395</span>–<span class="cit-lpage">1406</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.062828</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.062828%26rft_id%253Dinfo%253Apmid%252F17194788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzEzOTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.66" data-doi="10.1016/j.tree.2013.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>,  <span class="cit-name-given-names">P. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomics of rapid adaptation by soft selective sweeps.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">669</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2013.08.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D28%26rft.spage%253D659%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2013.08.003%26rft_id%253Dinfo%253Apmid%252F24075201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2013.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24075201&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326666200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.67" data-doi="10.1146/annurev.genet.39.073003.112420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Molecular signals of natural selection.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.39.073003.112420</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.volume%253D39%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.39.073003.112420%26rft_id%253Dinfo%253Apmid%252F16285858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.39.073003.112420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16285858&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234685200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.68" data-doi="10.1101/gr.4252305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Williamson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Hubisz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Clark</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genomic scans for selective sweeps using SNP data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1566</span>–<span class="cit-lpage">1575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.4252305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1566%26rft.epage%253D1575%26rft.atitle%253DGenomic%2Bscans%2Bfor%2Bselective%2Bsweeps%2Busing%2BSNP%2Bdata%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4252305%26rft_id%253Dinfo%253Apmid%252F16251466%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS8xNTY2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Linkage disequilibrium, gene trees and selfing: An ancestral recombination graph with partial self-fertilization.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">929</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D154%26rft.issue%253D2%26rft.spage%253D923%26rft.epage%253D929%26rft.atitle%253DLinkage%2BDisequilibrium%252C%2BGene%2BTrees%2Band%2BSelfing%253A%2BAn%2BAncestral%2BRecombination%2BGraph%2BWith%2BPartial%2BSelf-Fertilization%26rft_id%253Dinfo%253Apmid%252F10655241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU0LzIvOTIzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.70" data-doi="10.1098/rspb.1996.0152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Charlesworth</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Charlesworth</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Increased levels of polymorphism surrounding selectively maintained sites in highly selfing species.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">263</span>: <span class="cit-fpage">1033</span>–<span class="cit-lpage">1039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.1996.0152</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1996.0152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.1996.0152&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Donnelly</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The coalescent process with selfing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D146%26rft.issue%253D3%26rft.spage%253D1185%26rft.epage%253D1195%26rft.atitle%253DThe%2BCoalescent%2BProcess%2BWith%2BSelfing%26rft_id%253Dinfo%253Apmid%252F9215919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0Ni8zLzExODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>,  <span class="cit-name-given-names">H. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Betancourt</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Haldane’s sieve and adaptation from the standing genetic variation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D157%26rft.issue%253D2%26rft.spage%253D875%26rft.epage%253D884%26rft.atitle%253DHaldane%2527s%2BSieve%2Band%2BAdaptation%2BFrom%2Bthe%2BStanding%2BGenetic%2BVariation%26rft_id%253Dinfo%253Apmid%252F11157004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU3LzIvODc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.73" data-doi="10.1093/molbev/mst112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlidis</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Živković</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stamatakis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Alachiotis</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SweeD: Likelihood-Based Detection of Selective Sweeps in Thousands of Genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2224</span>–<span class="cit-lpage">2234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mst112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst112%26rft_id%253Dinfo%253Apmid%252F23777627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mst112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23777627&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323616600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.74" data-doi="10.1093/molbev/msj117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pennings</span>,  <span class="cit-name-given-names">P. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">Soft Sweeps II – Molecular Population Genetics of Adaptation from Recurrent Mutation or Migration.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1076</span>–<span class="cit-lpage">1084</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msj117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj117%26rft_id%253Dinfo%253Apmid%252F16520336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msj117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16520336&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237320800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.75" data-doi="10.1371/journal.pgen.0020186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pennings</span>,  <span class="cit-name-given-names">P. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">Soft Sweeps III: The Signature of Positive Selection from Recurrent Mutation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0020186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPennings%26rft.auinit1%253DP.%2BS.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De186%26rft.epage%253De186%26rft.atitle%253DSoft%2Bsweeps%2BIII%253A%2Bthe%2Bsignature%2Bof%2Bpositive%2Bselection%2Bfrom%2Brecurrent%2Bmutation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020186%26rft_id%253Dinfo%253Apmid%252F17173482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17173482&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.76" data-doi="10.1371/journal.pgen.1004000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pennings</span>,  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kryazhimskiy</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wakeley</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Loss and Recovery of Genetic Diversity in Adapting Populations of HIV.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e1004000</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004000%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004000%26rft_id%253Dinfo%253Apmid%252F24465214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24465214&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.77" data-doi="10.1371/journal.pgen.1003011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>,  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Huerta-Sanchez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Distinguishing between selective sweeps from standing variation and from a <em>De Novo</em> mutation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1003011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPeter%26rft.auinit1%253DB.%2BM.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1003011%26rft.epage%253De1003011%26rft.atitle%253DDistinguishing%2Bbetween%2BSelective%2BSweeps%2Bfrom%2BStanding%2BVariation%2Band%2Bfrom%2Ba%2BDe%2BNovo%2BMutation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003011%26rft_id%253Dinfo%253Apmid%252F23071458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071458&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pollak</span>,  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">On the theory of partially inbreeding finite populations. I. Partial selfing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPollak%26rft.auinit1%253DE.%26rft.volume%253D117%26rft.issue%253D2%26rft.spage%253D353%26rft.epage%253D360%26rft.atitle%253DOn%2Bthe%2BTheory%2Bof%2BPartially%2BInbreeding%2BFinite%2BPopulations.%2BI.%2BPartial%2BSelfing%26rft_id%253Dinfo%253Apmid%252F3666446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE3LzIvMzUzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.79" data-doi="10.1073/pnas.1719998115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. T.</span> <span class="cit-name-surname">Moyers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Lopez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Lasky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Monroe</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Combining population genomics and fitness QTLs to identify the genetics of local adaptation in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">5028</span>–<span class="cit-lpage">5033</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1719998115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1719998115%26rft_id%253Dinfo%253Apmid%252F29686078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzE5LzUwMjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.80" data-doi="10.1038/nrg2880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J. K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Di Rienzo</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Adaptation - not by sweeps alone.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">665</span>–<span class="cit-lpage">667</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2880</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D665%26rft.epage%253D667%26rft.atitle%253DAdaptation%2B-%2Bnot%2Bby%2Bsweeps%2Balone.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2880%26rft_id%253Dinfo%253Apmid%252F20838407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838407&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281911300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.81" data-doi="10.1554/05-273.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coop</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Wall</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The signature of positive selection on standing genetic variation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">2312</span>–<span class="cit-lpage">2323</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1554/05-273.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DPrzeworski%26rft.auinit1%253DM.%26rft.volume%253D59%26rft.issue%253D11%26rft.spage%253D2312%26rft.epage%253D2323%26rft.atitle%253DThe%2Bsignature%2Bof%2Bpositive%2Bselection%2Bon%2Bstanding%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F05-273.1%26rft_id%253Dinfo%253Apmid%252F16396172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/05-273.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16396172&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233769000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.82" data-doi="10.1534/genetics.119.301991"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rettelbach</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nater</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ellegren</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">How linked selection shapes the diversity landscape in <em>Ficedula</em> flycatchers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">285</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.119.301991</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.119.301991%26rft_id%253Dinfo%253Apmid%252F30872320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEyLzEvMjc3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.83" data-doi="10.1086/599083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roze</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Diploidy, population structure, and the evolution of recombination.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">S79</span>–<span class="cit-lpage">S94</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/599083</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D174%26rft.spage%253DS79%26rft_id%253Dinfo%253Adoi%252F10.1086%252F599083%26rft_id%253Dinfo%253Apmid%252F19476412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/599083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19476412&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266564600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.84" data-doi="10.1534/genetics.116.187955"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roze</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Background selection in partially selfing populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">937</span>–<span class="cit-lpage">957</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.187955</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.187955%26rft_id%253Dinfo%253Apmid%252F27075726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzIvOTM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.85" data-doi="10.1038/nature01140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>,  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">Reich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Higgins</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. Z. P.</span> <span class="cit-name-surname">Levine</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Richter</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Detecting recent positive selection in the human genome from haplotype structure.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">419</span>: <span class="cit-fpage">832</span>–<span class="cit-lpage">837</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature01140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D419%26rft.issue%253D6909%26rft.spage%253D832%26rft.epage%253D837%26rft.atitle%253DDetecting%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome%2Bfrom%2Bhaplotype%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01140%26rft_id%253Dinfo%253Apmid%252F12397357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12397357&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178769800045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.86" data-doi="10.1098/rstb.1996.0111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schoen</span>,  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. T.</span> <span class="cit-name-surname">Morgan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Bataillon</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">How Does Self-Pollination Evolve? Inferences from Floral Ecology and Molecular Genetic Variation.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">351</span>: <span class="cit-fpage">1281</span>–<span class="cit-lpage">1290</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.1996.0111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1996.0111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.1996.0111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.87" data-doi="10.1371/journal.pgen.1005928"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Kern</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">S/HIC: Robust identification of soft and hard sweeps using machine learning.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1005928</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005928</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005928%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005928%26rft_id%253Dinfo%253Apmid%252F26977894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005928&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26977894&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.88" data-doi="10.1093/molbev/msx154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Kern</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Soft sweeps are the dominant mode of adaptation in the human genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1863</span>–<span class="cit-lpage">1877</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D1863%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx154%26rft_id%253Dinfo%253Apmid%252F28482049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28482049&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.89" data-doi="10.1534/genetics.115.174912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. K.</span> <span class="cit-name-surname">Mendes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Kern</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Soft shoulders ahead: Spurious signatures of soft and partial selective sweeps result from linked hard sweeps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">284</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.174912</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.174912%26rft_id%253Dinfo%253Apmid%252F25716978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzEvMjY3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.90" data-doi="doi: 10.1101/697987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Setter</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mousset</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">DeGiorgio</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Volcanofinder: genomic scans for adaptive introgression.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> 697987<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>doi: 10.1101/697987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252Fdoi%253A%2B10.1101%252F697987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi: 10.1101/697987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.91" data-doi="10.1371/journal.pcbi.1004845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheehan</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. S.</span> <span class="cit-name-surname">Song</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Deep learning for population genetic inference.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e1004845</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1004845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D12%26rft.spage%253De1004845%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1004845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1004845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.92" data-doi="10.1111/mec.13288"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>,  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Signatures of positive selection: from selective sweeps at individual loci to subtle allele frequency changes in polygenic adaptation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13288</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13288&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.93" data-doi="10.1534/genetics.118.301319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephan</span>,  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Selective sweeps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">13</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301319</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301319%26rft_id%253Dinfo%253Apmid%252F30626638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMjExLzEvNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTA2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>,  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Evolutionary Relationship of DNA Sequences in Finite Populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D105%26rft.issue%253D2%26rft.spage%253D437%26rft.epage%253D460%26rft.atitle%253DEVOLUTIONARY%2BRELATIONSHIP%2BOF%2BDNA%2BSEQUENCES%2BIN%2BFINITE%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F6628982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA1LzIvNDM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.95" data-doi="10.1101/gr.5105206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teshima</span>,  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coop</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Przeworski</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">How reliable are empirical genomic scans for selective sweeps?</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">702</span>–<span class="cit-lpage">712</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.5105206</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5105206%26rft_id%253Dinfo%253Apmid%252F16687733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzYvNzAyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.96" data-doi="10.1534/genetics.105.044065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teshima</span>,  <span class="cit-name-given-names">K. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Przeworski</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Directional positive selection on an allele of arbitrary dominance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">718</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.044065</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044065%26rft_id%253Dinfo%253Apmid%252F16219788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcyLzEvNzEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.97" data-doi="10.1534/genetics.118.301018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">The effect of a selected locus on linked neutral loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThomson%26rft.auinit1%253DG.%26rft.volume%253D85%26rft.issue%253D4%26rft.spage%253D753%26rft.epage%253D788%26rft.atitle%253DTHE%2BEFFECT%2BOF%2BA%2BSELECTED%2BLOCUS%2BON%2BLINKED%2BNEUTRAL%2BLOCI%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301018%26rft_id%253Dinfo%253Apmid%252F863244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiODUvNC83NTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.98" data-doi="10.1111/evo.13191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uecker</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evolutionary rescue in randomly mating, selfing, and clonal populations.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">845</span>–<span class="cit-lpage">858</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/evo.13191</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D71%26rft.spage%253D845%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fevo.13191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/evo.13191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.99" data-doi="10.1006/tpbi.2002.1576"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Herwaarden</span>,  <span class="cit-name-given-names">O. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">van der Wal</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Extinction time and age of an allele in a large finite population.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">318</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/tpbi.2002.1576</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253Dvan%2BHerwaarden%26rft.auinit1%253DO.%2BA.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D311%26rft.epage%253D318%26rft.atitle%253DExtinction%2Btime%2Band%2Bage%2Bof%2Ban%2Ballele%2Bin%2Ba%2Blarge%2Bfinite%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2002.1576%26rft_id%253Dinfo%253Apmid%252F12027617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2002.1576&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12027617&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.100" data-doi="10.1111/mec.13360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vatsiou</span>,  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bazin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. E.</span> <span class="cit-name-surname">Gaggiotti</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Detection of selective sweeps in structured populations: a comparison of recent methods.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13360</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D25%26rft.spage%253D89%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.101" data-doi="10.1146/annurev-genet-111212-133526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitti</span>,  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Grossman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Sabeti</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Detecting natural selection in genomic data.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-genet-111212-133526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft.volume%253D47%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-111212-133526%26rft_id%253Dinfo%253Apmid%252F24274750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-111212-133526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24274750&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329816000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.102" data-doi="10.1371/journal.pbio.0040072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>,  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kudaravalli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A map of recent positive selection in the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e72</span>. Erratums: e154; e147.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0040072</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DVoight%26rft.auinit1%253DB.%2BF.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253De72%26rft.epage%253De72%26rft.atitle%253DA%2Bmap%2Bof%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040072%26rft_id%253Dinfo%253Apmid%252F16494531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16494531&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1063.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.103" data-doi="10.1093/mol-bev/msx207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vy</span>,  <span class="cit-name-given-names">H. M. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.-J.</span> <span class="cit-name-surname">Won</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kim</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Multiple Modes of Positive Selection Shaping the Patterns of Incomplete Selective Sweeps over African Populations of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2792</span>–<span class="cit-lpage">2807</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D2792%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmol-bev%252Fmsx207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mol-bev/msx207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.1063.104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">Coalescent theory: an introduction</span>, <span class="cit-vol">Vol. 1</span>. <span class="cit-publ-name">Roberts &amp; Company Publishers</span>, <span class="cit-publ-loc">Greenwood Village, Colorado</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.105" data-doi="doi: 10.1101/548198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>,  <span class="cit-name-given-names">K.-A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Pennings</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Drug resistance evolution in HIV in the late 1990s: hard sweeps, soft sweeps, clonal interference and the accumulation of drug resistance mutations.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> 548198<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>doi: 10.1101/548198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252Fdoi%253A%2B10.1101%252F548198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi: 10.1101/548198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.106" data-doi="10.1534/genetics.116.191478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Pennings</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Soft selective sweeps in evolutionary rescue.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">1573</span>–<span class="cit-lpage">1586</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.191478</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.191478%26rft_id%253Dinfo%253Apmid%252F28213477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNS80LzE1NzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.107" data-doi="10.1534/genetics.114.165571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>,  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Soft Selective Sweeps in Complex Demographic Scenarios.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">684</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.165571</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165571%26rft_id%253Dinfo%253Apmid%252F25060100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNjY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-108-1" title="View reference  in text" id="ref-108">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.108" data-doi="10.1111/j.1469-1809.1949.tb02451.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">The genetical structure of populations.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">354</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1949.tb02451.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D15%26rft.spage%253D323%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1949.tb02451.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1949.tb02451.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-109-1" title="View reference  in text" id="ref-109">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.109" data-doi="doi: 10.1101/589069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>,  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Schrider</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Kern</span></span>, and <span class="cit-auth cit-collab">Ag1000G Consortium</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Discovery of ongoing selective sweeps within <em>Anopheles</em> mosquito populations using deep learning.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: 589069<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>doi: 10.1101/589069</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252Fdoi%253A%2B10.1101%252F589069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi: 10.1101/589069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-110-1" title="View reference  in text" id="ref-110">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.110" data-doi="10.1534/genetics.117.300401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wiehe</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Detecting recent positive selection with a single locus test bipartitioning the coalescent tree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">791</span>–<span class="cit-lpage">805</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300401</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300401%26rft_id%253Dinfo%253Apmid%252F29217523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA4LzIvNzkxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-111-1" title="View reference  in text" id="ref-111">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.111" data-doi="10.1371/journal.pcbi.1007426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Wiehe</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Adaptation in structured populations and fuzzy boundaries between hard and soft sweeps.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-elocation-id">e1007426</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1007426</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D15%26rft.spage%253De1007426%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1007426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1007426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-112-1" title="View reference  in text" id="ref-112">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1063.112" data-doi="10.1111/mec.14209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Su</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Signatures of soft sweeps across the Dt1 locus underlying determinate growth habit in soya bean</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4686</span>–<span class="cit-lpage">4699</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.14209</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D4686%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.14209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.14209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/3/1063.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/1051" title="The Developmental Transcriptome of Aedes albopictus, a Major Worldwide Human Disease Vector" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/1077" title="The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in Arabidopsis thaliana" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/3"><span>Volume 10 Issue 3, March 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/3.cover-source.jpg?itok=8jjPWDA4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1063?rss=1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sjfYOthneFOXLhvqE2usIo-goqICvmHdIoMPVlHa8XY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/3/1063" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/3/1063" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/3/1063" class="active" data-icon-position="" data-hide-link-title="0">Selective Sweeps Under Dominance and Inbreeding</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495861" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-QzbvLSE1FIyxCACMgGAXw2ip9LJfNJdPlKnLdVGOcLQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/1063.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/3/1063" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-LrHoTd1G1g10LjSDW-3_xYjSHocI1JfYueyFUU6v__M" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/3/1063" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/1063.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495861" id="node495861--2" data-pisa="ggg;10/3/1063" data-pisa-master="ggg;g3.119.400919" data-apath="/ggg/10/3/1063.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selective Sweeps Under Dominance and Inbreeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6941-0647" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew Hartfield</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4730-2538" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas Bataillon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1063-1075;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400919 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495861--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew Hartfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Toronto, Ontario M5S 3B2, Canada,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Research Centre, Aarhus University, Aarhus 8000, Denmark, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Evolutionary Biology, The University of Edinburgh, Edinburgh EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BHartfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hartfield%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BHartfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6941-0647" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew Hartfield</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.hartfield@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">m.hartfield@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Bataillon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Research Centre, Aarhus University, Aarhus 8000, Denmark, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBataillon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bataillon%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BBataillon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4730-2538" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas Bataillon</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/3/1063" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/3/1063&amp;title=Selective%20Sweeps%20Under%20Dominance%20and%20Inbreeding" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/3/1063&amp;title=Selective%20Sweeps%20Under%20Dominance%20and%20Inbreeding" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/3/1063&amp;title=Selective%20Sweeps%20Under%20Dominance%20and%20Inbreeding" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/3/1063&amp;text=Selective%20Sweeps%20Under%20Dominance%20and%20Inbreeding" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/3/1063&amp;title=Selective%20Sweeps%20Under%20Dominance%20and%20Inbreeding" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/3/1063&amp;t=Selective%20Sweeps%20Under%20Dominance%20and%20Inbreeding" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/3/1063&amp;title=Selective%20Sweeps%20Under%20Dominance%20and%20Inbreeding" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/3/1063&amp;title=Selective%20Sweeps%20Under%20Dominance%20and%20Inbreeding" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495861/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495861" id="node495861--3" data-pisa="ggg;10/3/1063" data-pisa-master="ggg;g3.119.400919" data-apath="/ggg/10/3/1063.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Selective Sweeps Under Dominance and Inbreeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6941-0647" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Matthew Hartfield</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-4730-2538" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas Bataillon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1063-1075;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400919 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495861--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew Hartfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Toronto, Ontario M5S 3B2, Canada,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Research Centre, Aarhus University, Aarhus 8000, Denmark, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Evolutionary Biology, The University of Edinburgh, Edinburgh EH9 3FL, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BHartfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hartfield%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMatthew%2BHartfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6941-0647" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Matthew Hartfield</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.hartfield@ed.ac.uk" class="" data-icon-position="" data-hide-link-title="0">m.hartfield@ed.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Bataillon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Bioinformatics Research Centre, Aarhus University, Aarhus 8000, Denmark, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BBataillon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bataillon%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BBataillon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4730-2538" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas Bataillon</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1063&amp;rft.epage=1075&amp;rft.atitle=Selective%20Sweeps%20Under%20Dominance%20and%20Inbreeding&amp;rft.volume=10&amp;rft.issue=3&amp;rft.date=2020-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hartfield%2C+Matthew&amp;rft.au=Bataillon%2C+Thomas"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495861/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495861/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495861/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495861/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495861/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495861/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495861/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495861/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495861/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495861/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495861/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495861/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/3/1063" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/3/1063.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/3/1063.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/3/1063.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
