<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/3/933" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/3/933.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/3/933.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/3/933.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/10/3/933/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/3/933.atom" />
<meta name="HW.pisa" content="ggg;10/3/933" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Interactively AUDIT Your Growth Curves with a Suite of R Packages" />
<meta name="DC.Identifier" content="10.1534/g3.119.400898" />
<meta name="DC.Date" content="2020-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Coutin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Bottlenecks often occur during data analysis when studying microbial growth in liquid culture at large scale. A researcher can collect thousands of growth curves, repeated measures of a microbial liquid culture, at once in multiple micro titer plates by purpose-built robotic instruments. However, it can be difficult and time-consuming to inspect and analyze these data. This is especially true for researchers without programming experience. To enable this researcher, we created and describe an interactive application: Automated Usher for Data Inspection and Tidying (AUDIT). It allows the user to analyze growth curve data generated from one or more runs each with one or more micro titer plates alongside their experimental design. AUDIT covers input, pre-processing, summarizing, visual exploration and output. Compared to previously available tools AUDIT handles more data, provides live previews and is built from individually re-usable pieces distributed as R packages." />
<meta name="DC.Contributor" content="Nicolas P. J. Coutin" />
<meta name="DC.Contributor" content="Guri Giaever" />
<meta name="DC.Contributor" content="Corey Nislow" />
<meta name="article:published_time" content="2020-03-01" />
<meta name="article:section" content="Software and Data Resources" />
<meta name="citation_title" content="Interactively AUDIT Your Growth Curves with a Suite of R Packages" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Bottlenecks often occur during data analysis when studying microbial growth in liquid culture at large scale. A researcher can collect thousands of growth curves, repeated measures of a microbial liquid culture, at once in multiple micro titer plates by purpose-built robotic instruments. However, it can be difficult and time-consuming to inspect and analyze these data. This is especially true for researchers without programming experience. To enable this researcher, we created and describe an interactive application: Automated Usher for Data Inspection and Tidying (AUDIT). It allows the user to analyze growth curve data generated from one or more runs each with one or more micro titer plates alongside their experimental design. AUDIT covers input, pre-processing, summarizing, visual exploration and output. Compared to previously available tools AUDIT handles more data, provides live previews and is built from individually re-usable pieces distributed as R packages.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/03/01" />
<meta name="citation_mjid" content="ggg;10/3/933" />
<meta name="citation_id" content="10/3/933" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/3/933" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/3/933.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/3/933.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/3/933.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400898" />
<meta name="citation_pmid" content="31974098" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Software and Data Resources" />
<meta name="citation_firstpage" content="933" />
<meta name="citation_lastpage" content="943" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nicolas P. J. Coutin" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Faculty of Medicine, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9336-226X" />
<meta name="citation_author" content="Guri Giaever" />
<meta name="citation_author_institution" content="Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver, B.C., Canada, V6T 1Z3" />
<meta name="citation_author" content="Corey Nislow" />
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Faculty of Medicine, and" />
<meta name="citation_author_institution" content="Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver, B.C., Canada, V6T 1Z3" />
<meta name="citation_author_email" content="corey.nislow@ubc.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4016-8874" />
<meta name="citation_reference" content="citation_journal_title=Current Opinion in Biotechnology;citation_journal_abbrev=Current Opinion in Biotechnology;citation_author=A. Blomberg;citation_title=Measuring growth rate in high-throughput growth phenotyping.;citation_pages=94-102;citation_volume=22;citation_year=2011;citation_issue=1;citation_pmid=21095113;citation_doi=10.1016/j.copbio.2010.10.013" />
<meta name="citation_reference" content="citation_journal_title=Q. J. Econ.;citation_author=A. Brody;citation_title=A simplified growth model.;citation_volume=80;citation_year=1966;citation_doi=10.2307/1880584" />
<meta name="citation_reference" content="Chang, W., J. Cheng, J. Allaire, Y. Xie, and J. McPherson, 2019 shiny: Web Application Framework for R. R package version 1.3.2." />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=M. Cokol;citation_author=H. N. Chua;citation_author=M. Tasan;citation_author=B. Mutlu;citation_author=Z. B. Weinstein;citation_title=Systematic exploration of synergistic drug pairs;citation_pages=544-544;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=22068327;citation_doi=10.1038/msb.2011.71" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=L. Fernandez-Ricaud;citation_author=O. Kourtchenko;citation_author=M. Zackrisson;citation_author=J. Warringer;citation_author=A. Blomberg;citation_title=PRECOG: a tool for automated extraction and visualization of fitness components in microbial growth phenomics.;citation_pages=249;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12859-016-1134-2" />
<meta name="citation_reference" content="Garnier, S., 2018 viridis: Default Color Maps from ‘matplotlib’. R package version 0.5.1." />
<meta name="citation_reference" content="Hughes, S. M., 2016 plater: Read, tidy, and display data from microtiter plates. The Journal of Open Source Software 1." />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=M. Kahm;citation_author=G. Hasenbrink;citation_author=H. Lichtenberg-Fraté;citation_author=J. Ludwig;citation_author=M. Kschischo;citation_title=grofit: Fitting Biological Growth Curves with R.;citation_pages=1021;citation_volume=33;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Analytical biochemistry;citation_journal_abbrev=Anal Biochem;citation_author=A. L. Koch;citation_title=Turbidity measurements of bacterial cultures in some available commercial instruments.;citation_pages=252-259;citation_volume=38;citation_year=1970;citation_issue=1;citation_pmid=4920662;citation_doi=10.1016/0003-2697(70)90174-0" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Y. Lee;citation_author=R. P. S. Onge;citation_author=M. J. Proctor;citation_author=I. M. Wallace;citation_author=A. H. Nile;citation_title=Mapping the Cellular Response to Small Molecules Using Chemogenomic Fitness Signatures;citation_pages=208-211;citation_volume=344;citation_year=2014;citation_issue=6180;citation_pmid=24723613;citation_doi=10.1126/science.1250217" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=N. Malo;citation_author=J. A. Hanley;citation_author=S. Cerquozzi;citation_author=J. Pelletier;citation_author=R. Nadon;citation_title=Statistical practice in high-throughput screening data analysis.;citation_pages=167-175;citation_volume=24;citation_year=2006;citation_issue=2;citation_pmid=16465162;citation_doi=10.1038/nbt1186" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=J. Monod;citation_title=The Growth of Bacterial Cultures.;citation_pages=371-394;citation_volume=3;citation_year=1949;citation_doi=10.1146/annurev.mi.03.100149.002103" />
<meta name="citation_reference" content="citation_journal_title=Methods in cell biology;citation_journal_abbrev=Methods Cell Biol;citation_author=J. R. Pringle;citation_author=J.-R. Mor;citation_title=Methods for monitoring the growth of yeast cultures and for dealing with the clumping problem.;citation_pages=131-168;citation_volume=11;citation_year=1975;citation_pmid=1102845;citation_doi=10.1016/S0091-679X(08)60320-9" />
<meta name="citation_reference" content="citation_title=Strain Engineering: Methods and Protocols;citation_title=The Automated Cell: Compound and Environment Screening System (ACCESS) for Chemogenomic Screening;citation_pages=239-269;citation_year=2011" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=K. Stevenson;citation_author=A. F. McVey;citation_author=I. B. N. Clark;citation_author=P. S. Swain;citation_author=T. Pilizota;citation_title=General calibration of microbial growth in microplate readers.;citation_pages=38828;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep38828" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=J. Warringer;citation_author=A. Blomberg;citation_title=Automated screening in environmental arrays allows analysis of quantitative phenotypic profiles in Saccharomyces cerevisiae.;citation_pages=53-67;citation_volume=20;citation_year=2003;citation_issue=1;citation_pmid=12489126;citation_doi=10.1002/yea.931" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Warringer;citation_author=E. Ericson;citation_author=L. Fernandez;citation_author=O. Nerman;citation_author=A. Blomberg;citation_title=High-resolution yeast phenomics resolves different physiological features in the saline response.;citation_volume=100;citation_year=2003;citation_pmid=14676322;citation_doi=10.1073/pnas.2435976100" />
<meta name="citation_reference" content="Wickham, H., 2016 ggplot2: Elegant Graphics for Data Analysis. Springer-Verlag New York." />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=M. H. Zwietering;citation_author=I. Jongenburger;citation_author=F. M. Rombouts;citation_author=K. van ’t Riet;citation_title=Modeling of the Bacterial Growth Curve;citation_pages=1875-1881;citation_volume=56;citation_year=1990;citation_issue=6;citation_pmid=16348228;citation_doi=10.1111/j.1472-765X.2008.02537.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Interactively AUDIT Your Growth Curves with a Suite of R Packages" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="twitter:description" content="Bottlenecks often occur during data analysis when studying microbial growth in liquid culture at large scale. A researcher can collect thousands of growth curves, repeated measures of a microbial liquid culture, at once in multiple micro titer plates by purpose-built robotic instruments. However, it can be difficult and time-consuming to inspect and analyze these data. This is especially true for researchers without programming experience. To enable this researcher, we created and describe an interactive application: Automated Usher for Data Inspection and Tidying (AUDIT). It allows the user to analyze growth curve data generated from one or more runs each with one or more micro titer plates alongside their experimental design. AUDIT covers input, pre-processing, summarizing, visual exploration and output. Compared to previously available tools AUDIT handles more data, provides live previews and is built from individually re-usable pieces distributed as R packages." />
<meta name="og-title" property="og:title" content="Interactively AUDIT Your Growth Curves with a Suite of R Packages" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/3/933" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Bottlenecks often occur during data analysis when studying microbial growth in liquid culture at large scale. A researcher can collect thousands of growth curves, repeated measures of a microbial liquid culture, at once in multiple micro titer plates by purpose-built robotic instruments. However, it can be difficult and time-consuming to inspect and analyze these data. This is especially true for researchers without programming experience. To enable this researcher, we created and describe an interactive application: Automated Usher for Data Inspection and Tidying (AUDIT). It allows the user to analyze growth curve data generated from one or more runs each with one or more micro titer plates alongside their experimental design. AUDIT covers input, pre-processing, summarizing, visual exploration and output. Compared to previously available tools AUDIT handles more data, provides live previews and is built from individually re-usable pieces distributed as R packages." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
    <title>Interactively AUDIT Your Growth Curves with a Suite of R Packages | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"lk2_ApVPjuCG_jEyAXYSt85Kfg_zPYVGEoxBnGjU0xY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495851","apath":"\/ggg\/10\/3\/933.atom","pisa":"ggg;10\/3\/933","ac":{"\/ggg\/10\/3\/933.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/3\/933.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/3\/933"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/3\/933"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/3\/933?rss=1":true,"\/content\/10\/3\/933":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222Lk2KzVFwvxfzdbGopd\\\\\\\/yFcr7KJHD\\\\\\\/LwGqmuxy9VxlviO7NagPLG0ZCE1hNOMoHEdvpR7BpJ+6sSytiSX04fB8CjD1+5AfVwX7EGyjN\\\\\\\/pPKjOpCpKl7+SxQHAcERfglZ8RejPy7tLcCtsAyqFYDR2573YgfyGLA0HWUJOql\\\\\\\/foPrI05dIN7FUSvno2RgGzIaAnizDajQhxNjLL5SkB8AxYe2lNHGatwSo35Cg0dPSws2+g717zoTVjpEQoEICpE9l11qmRwTwqRvNdSzlX6RtM9SP12NYewhXdYYHB4OiIVFYxMo8cnhW2KMhPAdRGMZoqDtzcbQ2FuYOWZOWEta+VuFAdd1rJSF8iY56P3R55YvsyIlFQeq8SjqvHjJrb3+zFFFwtGHWeE7aetFMDpNGweWE29jizMyMsFBEejILBjaj5Mpo9Li9ya5npwlLOD2McP4f3lUkOrao80NvdMlw8TCzT32QVL9Zx9RQ9oMlL6rL1t6G5pOEOvdc9wlSdVEH3diNGr2jzUbi\\\\\\\/TecocrW\\\\\\\/ReIYERn5WbJRy7wl\\\\\\\/XQWpH9hO9Q5Bz8Pj4sObsm+27\\\\\\\/mCwLFQi5V8rwxsqcaOR5uNqgcIlDxeRRI1vaRfkeoNEkYyzvaFEM8fR\\\\\\\/6kybE4CdaHRxo1PONibm7LiXpwS99v8pdlb3Y88u0b6kx5hcm5TmEsnIztQugN\\\\\\\/9pYcWcxdRV5Ptl+F6tjiGk5BI2GDrt0GuXStgO1JKLWePUO43GOTxwYEl41qYT90QiZRn8N1+P\\\\\\\/OF3sQGqQnj2ZAHKfE5Pz4gNjKLLL1s4fxhUJAcmE14jVmEVUO2g1m9RUhNERPc+qy1pyMKz5lIVYsCc3NQWQPzXOeynzdkzx6FDzlP9diYQj+T\\\\\\\/6pTMwc7oxsObggdO0Q9exurEItgwxLrXQEATz2Jfu3A+owr3ZYtqgrUuPk7ZLjATwr2c\\\\\\\/clJqMfN6gZpQjEBo9YoZ8RtemNMo1ah70dZ2EyJ0ZiUr1ol2AGCi0hbVh+BpVO0vdwzQrChLuwfpQk6SnkCzf7BOoqY9kUpoUOI7mTNRFKJbndsE4HzVROtn8KgUrbw9KX1PmSYX05zz14o7MHPKiVSv4dZibhhLdvAQTlCnTXCuFg3fu8UUV9ePK2UECNeo4uwtEqqkXjYcPf2qIpKivStTNaxz48tWmaU1mbF1QhEMDLN\\\\\\\/xLlortz3REQ10rDUsEGu23SZ1COC7NfpL8+1a97f0112hWVxV8qyLbOhwN3Dt2IdkzHVr8Koi2flHEG3WRaa21VJT\\\u0022,\\\u0022iv\\\u0022:\\\u0022G0W\\\\\\\/LuzogZtIvIc7+qehIA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227b7c94aa9815f7152d786af6043b59fbccfccf87a68bfd7adf4614b9bd8e85eb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Vnvg2NxNVBA4FfAfv5OxGLEZGBLs0bhGSAHi3Pl0MWkcYc4kycFa9tumCPwcY4zi9nzwit+q+cvh1DLm2vkjkYmmK2A6wJwAEXkIcdy\\\\\\\/kaGrpR1W\\\\\\\/t9SoVovLmSJ\\\\\\\/tyYvcd6FPTD193ZnMQQmqh+VssjY3k9Xm5kJ4yOC4FgydKSGLcTBSxlspCgPi112tn8MjZoKAc35BTyIisqpmUWAR5wxZ52R7KvOSYaC9QWjBqXLdk6dT41GwaypzTwziq4p5\\\\\\\/+kNFll9uLOMnpP\\\\\\\/TWHv7m4FW3BVlbEDQkhs+4KupkBUQrdxhBZdMDYIyCVrdDkzqBvTrToWzwgon9Iu2hxzjN2lsV1uaV7RcExM351942fmJyuKg4hz\\\\\\\/yiklbU2YX8gGZCPDbbcg7WJxcVsJLlS1Y+Z5b40QJdx8jxsIVbTbKSQ3oikTXGNGSfrxZi5cLzA5vesfdPiqGIBkw6ouxTi8H+XNDh9h\\\\\\\/i1Hed6Wz2\\\\\\\/m\\\\\\\/59B\\\\\\\/\\\\\\\/GAx5C+SUWklNEkwXsleQ+CLUsg9V3oDvM7xxirByVnKY7\\\\\\\/oQfTfh6GuNL\\\\\\\/XeKWdlxHyphiDSoxbzp8H1VO5qHtavSlboilg1RTabXfocbcvaReRhlnaQgwjJtz\\\\\\\/LawfdpdARfHsGvrWoZ0Jkin0SZa7wArbQ54gcHaSG0JZnpKnd0HXke0JPq+XOy+EUnpBPqP\\\\\\\/tgUGqEEcZYvaTxZNwmMp59ci3rZ9JOrcXIdaSRQDJ+oKW6dtkc7GfFmg5nJKJ7J3k0\\\\\\\/bWYHmGBox0LDC+8WbJVTVmLSUrtZF8MLmoEZEMyiLrmWVqZPGtNkkO2FW2JkJjM8f2DaoYQNxs94EbKBzDTFYdonMZDEUU\\\\\\\/X3jxX4Qjf9AaQMumNGjp5OvzTdyBaeLlXEyg7Av6rjWiOEMMtzmR6pXneZ0S1iwZbwH6oANE1GGTa3kh6Eq5OyPGl4g0CMnbYxizkY2yfRmRW6ZSEDI4f0KRe031iB6O2rvzNxr4GkwVr2v9Em4YAptmtbeW1ZXClBqQ4XG9RpnPOAEf7l5XTiZIgD2EEGQddEj9tk9Q2bbFYNBIhz00qghL+O7wH3nS32yERBOdk+f84orJv+SzeHvAd3zTQ0EObM16w13oWHFYFWHQdelIMh+heE\\\\\\\/4CFPVy8IQnsYUxOhNYLP+26GvoQ+iFxSrUZmxx0jue+7naK4ty6SBxhtYYYXOtExzRNcbDa2VjVCbTSwkmd4rY5cMPSVsqCyeEKVEpeNKT\\\\\\\/1UnEvkg94fMhaDsAQ9xqt8\\\\\\\/0F8s7\\\\\\\/1jzhj6Y13jKBUtYRVlKIFVEb4\\\\\\\/vMdumzK5sbjEvuq1S8ja4KKKibABgKyqdXi9eYPQaZ8zKS6xTqoErptOt\\\\\\\/v1R\\\\\\\/JfNZMaEunAEe6tngmerKLiOAMrkPSIoivu3Dr4QP6FX9JKUpOoiPQ8jDL+GOn3oghCNCScrIsZkAHnfCnDAT\\\\\\\/NXckZPgy\\\\\\\/0rJO4jf88a5eW4n\\\\\\\/\\\\\\\/MsuTAaKgdEA6UpomemOIfoQxJLKYuk23z9EUF0NnQmA12JSFi4zJYhz3af2EmkerDvEYBF9RPTO7tfWPaFY53BLZOOH2MVRNIZXGIa9C5ux8aixAcl4muLnfiuoo0mdtMPoEJbZvt0t4NmCyUvPGO0ZTZ3erJO4hB5RAYiyzqUW7fCo5DLUJlSUr2S3wzTWTGeJLI3wvWwPwzw6ussXKXi6vtgr9eyELzQeEx3+RS2AOX8PYjNn5htaE\\\\\\\/8RvHpE8yd86oPjaeipuaWyTYr2jKGutJO4ODjznNdFoVn7W7gX74ku2eBeEDFLxmvbQ7Gfq+UEwxHhFeoz6V00AYTtOGMomYlCERTWSqiTgBrZpRTWzPJkDuBr4Q4GwlGqAW3khNJMArG5cXPJCXm+w5T8jAKS+\\\\\\\/4g9lV+Efg4yg7hKh\\\\\\\/nLLjd+o0nYwOVC\\\\\\\/hjE9gO7\\\\\\\/+8Ofls7USzzNcnYFS+9e4d27qqjFVplUgFkjXT8nXG9V\\\\\\\/r\\\\\\\/woIX+094ACSprynpmhw5SeIvc362iuilqxCuh7vKYeF15OmORLFOdIcSAgsysKbRGlnP58YOcGtbSE4n90BOMdMUGtBPoXDJn6dnFfHz9uwJZhSBCUqT4Y9B6xKzfGQjLxoGjpfjCkfPf3idicl4om2Y63N1qR3IVRHxInr8WQ+zq1aeOlNy7xPMJtpI4d\\\\\\\/\\\\\\\/uwxi8PdYmAFzp189UgC0OWv5TYXuRriW7XErtiMDqAP95Iwbd3fTHk9Bequ22MMqEcHuljjncMYka\\\\\\\/tD+iNfKXRb7GilYxkFgDLKRC6wJb3qkzQSdmwe5wJDc99nU8++7zojSD\\\\\\\/lx1lvBfqHL5\\\\\\\/N+I1\\\\\\\/7IoVAVewSdbbbWgPtRyGTBvt\\\\\\\/\\\\\\\/sF2KO1h\\\\\\\/vzU9rJuFaTAwaZhg\\\\\\\/5bPIYnkLONpAEVeo6TRWIvJUBby9Cvx+NGEoYWk\\\\\\\/z2GJ4qLz9+YTN1n4V4RlqE1qtrIUUyulTVksaiCXMmBS0VsoorIbMBL+hHTmMo7UoPuZHOX6h1CAzj6IceBUCWzhyKeIvCpHHlXnLo8OiedeJPSE9OrQkZCPCQb2nRsljEhrk2PZMTDHhFXBNXMBua9dWSGc4+zkPETrV0PtRqzEC1hHlI5HX77Q\\\\\\\/mYhy819zhbFnKAqqFOFvdS\\\\\\\/SOHciKg2vOwOVCtHlVAnx2y22\\\\\\\/Oa6k2jdlcd2PC4OeJE0Yt21aUzGMekGU+qUkMX5Gv50bPssH0SLbltH8nW8eVHTGPqAEahIJaBXP\\\\\\\/t5KzxN+WX1RdEDHrHuCwKxriT3LzwKiQw2XgvsIuiAXiW2M1B8NmGsNJWt1iGPA1\\\\\\\/9e9ngUWsznJnJw1O9lG\\\\\\\/FbToq+MH12piXonwyB\\\\\\\/L9JlW\\\\\\\/uxEpuAb0CtaU7rEr15TT\\\\\\\/whHXax3Th76T3t8WnAqrqvybeb58tb85o2HBASLT+0QlcERRi\\\\\\\/iy4IzIvx2wJwgh\\\\\\\/rRiegzCSkkd4VDPPgoHNgxAUx4SnG94Iw2t00eMJURCvqb16MgQz1L\\\\\\\/U8j5AOGAbpe4UjjwW8WATLpMge\\\\\\\/R+6tUvnGM4OYXN2cGxhN27scHlUJNxEckI6wHHexqWBrqHmsVq2vqSKrYNXIGwUVfxvtez9IFQZ6U7\\\\\\\/SwYBCddvU36\\\\\\\/m0jurt8zeRn42cHIi8Lonp\\\\\\\/I3gMrWgTthQj5ZRRYZU36NnQkN1neskt1WwYnXyk\\\\\\\/N3D9IyHOca7Jl6ZsbVcCTc35e0+fr9rS8fITGGuzwZEFMirOzOMeomas+LzOHVD8hwNTEwvP5MzPU6V0iARRrYbWIeEVE76U05p0RpWxZOUShLUQzYmsj6tzuHdBKCPooT0QYK0Yo+IbwFmS5IQr+FylM96Ydgc6\\\\\\\/EUyoIzF6MetwWcrrHSgWLkzHKXwEeTSK\\\\\\\/ENiDFbmECce6HjtnxfLxrhqqTweaGofqF3f4ZgrkLveX6RlarBx9RSklT8x1NgxT5krDnVNksNfrXngqNjtSGN1Na7VmrMH8G31euy+zJNNb7YvD3V9abMYfVkCvzy4005PcBXwze46IyQbVxI9sg0cml+rYfmJ1tbDvF47\\\\\\\/jaELHCN5sIVyJY8Yr9\\\\\\\/lOTs3KdZhYo\\\\\\\/zZDiH96pqd8z4ql3TJRX0LHVD96REjZfV\\\\\\\/24YMT6BlEHCroA4g7qZA7OcVXe60HOokl+4QSF+PgGnt5VkgiQEZVIukp9+ccG4JTJSu7giYuU0ntEsGhesEdttfjJvwcjJ7Vbuof9m7IqpK9rD1l2KgPnqy7CxokmBsI6r4ulJGM754H6hVAQ5DIzo6W9Ud6zrs3cYCzfxvcHzj4D0XMCb4xeRtngYdDXQKyAB9Xiw1Apaibp10cwcK1BiOK2Ckol4kJVsNxLu\\\\\\\/cZo+WD6X1NHPlGrq7x7KhFBZOI1Bak1VDWdTxyJMVlpQ0v7v1aiGpsZQVJjrEwK8yha8EM8Ec2OKEG5dEdwfZ8aqmFE7U3vEgKa2qxTLY56hldMyobYyrwp6xQo3MBPgZV\\\\\\\/PaZG9tb2XFY\\\\\\\/nxLf7TqQ60bNGVj2YCq40v1MnGrIAOXeMpeLJblzigjtUPWbUd8O6+ZoGHd7RRavipi4sE9d5Y82gjdVH6O59PPmR+nV7onOiq0niJGdzIjm6uv2u9PGPW0rS+SDi0kor2GcUdZra+Eo891tiUT59lW16DziKLKM\\\\\\\/pALMLvYeomjyIkoonqF9fuFsfsvR6GVmYI+LUEDmQc2QdPQ9VeAV0N0FSRt\\\\\\\/8KgV+bftdQ7ZbpvhwBRXTHtYPxd6p+pJJD7ymCPg9tetD+6awsb\\\\\\\/0BJj5wE5OjuqIEEBtVksoolbYpSLHROFNixfFtRK8lgphKcMAamDjPxZm7vVoRndtq5kAqczyivowXApGffP7X6WO74ZAwiBv54GShxTegHh6bbpFm4lI6\\\\\\\/z\\\\\\\/ekA3iDHV8lEHJA55nDlxdChlM2ntsSU\\\\\\\/eg9uiH3yPtpRLF5JKdM\\\\\\\/liSJ+3BbcE5dczZ9FD394ltfWu1UeA3cZTQytuInmHb7kYJuCHkZ4ZZm0iRsPz5xD1LFIaebFHtGVKVd7wuzWeRuc+OpGC3HQQ4c7+gUKXl2633VHT3+AE303RFzjHl9s8e8h84TRTNC6ODatjMgsOLWz9TQJoo9rQ4q2Xt2Xf2RRfqMNSDDYRkxS3iUVHU0MtPqzXnKYEV1UxEkkctiIzSPn2G8p57y0HfWPwVOET2PbLFeWBbuIbwIgBtumrHrngQFl1L33pqEkGZ9YN\\\\\\\/+dFTxHO1vyn+HMoKpPSHJ7PcIq\\\\\\\/xlE6\\\\\\\/C0+HymnRKoB3EvFMxGI+C3nR\\\\\\\/KOxxj\\\\\\\/QjWv4DEt9p6ZIHnVHCt3sXcYPWmzStd42Z0ir5BKeUVwMXobLotlC7vlplGzL0wSLK1SmaSY7JEP4eIm5XaQxuP4L3mrs4aQ\\\\\\\/Y+7WB3IXyttTx9R7tUxSDNrz3CelevmU1CY8spf8Lw7mKWGNmFYuse8EgjZxXXeOwB61lON\\\\\\\/V5jOBD9NRgWMQpMZLuV+2gmvM9+5QaViGO5Z8tQsBRZYA5LKaoIrQpTM697C9d\\\\\\\/5RaTh7kZ\\\\\\\/NwTf3BPPQPHwvEapFBlrjQC3jsNnLbGJ3SAMmt\\\\\\\/XtMmVomiBQB0c5gYOnOIGZriJhcT63qfBKZ42SjkQ7nOq+KHTlDKl2J8z2cZAa94sfqMSAusAbQpZUhEOKEVG03ay0EvXDdMfUpc5\\\\\\\/hk8hf2ynVsMKq5tNsLdHC6WNqi9PjgFueGX\\\\\\\/wE9f+PSaICD6pRH7nzl7xFkEhvEJDXBKEeYi6pbpHjM0uNj6IQNGIT2Oqt+syvdG+suJBnWTAsM8HrnephfZje7ZgWWYtdRE+BwktZgg2YcrdxInvV+VzjEM8JY9okiKy3ppBPx\\\\\\\/IhgMcHFS20lPF7RI4jmWjS35tUpRTD2YVMk2hokKZNbpuj7+g3M\\\\\\\/7XRolwoHChO7OKt+j\\\\\\\/1aq5H6hDtp2F49+FtzKXzrVyD7EK5cos+XiZ1TPRGtFmuL+whpOqRGdm\\\\\\\/zO5D2xFGRPlGj2Q2xiMCL5jDG4DtANK9pTgoF9TBlX82rei8H\\\\\\\/sa02qGw1M48Bw13Cfc1DSYdKFu+lW4\\\\\\\/91bEmqHb6rDQpk8kLy3Ewp6dXreFR5JcuNw7U0FVyq6J+6oqNECnonxrb3arsxvf3DoI\\\\\\\/K39FvrNi11SAneBvVCGdqQLYtg7GUadLF3+NoI3LjwNNLtuNDuyW4MMwQHYPrh+Y\\\\\\\/yNk4Udp9AZpPl45d1MsXA7KBYL1jl3XXfTXOmM03yVT51lQCZ8by\\\\\\\/BdqYSh3A6Mo6cXiNYlw8sa9US2A+aLyAMakskAhY4mZC6nOT7X2pEe6xZmM\\\\\\\/AMIOIawkg6XND88LoaSzyiFbuKY5qlTz01+10\\\\\\\/QnXY6tlK8pioJHb\\\\\\\/XezRE3IuLqIJONIF+R+Rj63O5VqhzdrivBYx4kv9G9kwSGHuHt\\\\\\\/VwjP4qzF9Ta1GgCBzvjtk0qNlshGRgRYkL2dDTyp04xWhYLzqTstnvCZuUUIWhA7\\\\\\\/8A6PAmkTlI1fhM\\\\\\\/hr+l078n7pSL\\\\\\\/L5iqV0pEg3PkBLOrqFf8DajYv3RSm12OZSNnfOMicBwYSjryoxn\\\\\\\/yx1xgdzMW3KFffSo0170jXA1aO8qAMpfsMUhwXkqokAbFz4lOymth8xuVbm9+USquVoabU8wST9B4qg3S7bmGrQc+RlVb3HQJn4BLeOJuWr9FLaTSktHZDnMI3CN+BtqHzOy0HRYf3nFwySRgW\\\\\\\/77+tW7\\\\\\\/uER+5n4GNhyDabndUWxBCZLCu6Db2yuCcdxvRFhtnPOvNlsrMH3EEJkUNFtdHGou3FLKikxqqQC84hcRcuv6y9Yucg62KLvalvUt1dL4zgO4hY6ckjcvbLRqZMg661QdBUoJ4gUNYK11e8mvkD3iCLzONyoCaQjGjvRKSYSrD5XZcyamSkZzSVUSwaVymuLbuO6oulLRM0Ksjxkw2zJ0XNzrke0V5iSsqnYe65gj5ROZ0U5vE6wzLHhSji4lVUpsv2UC3Vyz08BcFnIiBe998RLmuaXIxayN3lOOciJEBS\\\\\\\/Ehw1cPU+zPIA5NEvUQhDdiFnCeuxl8l5UHBnbab8PlbcyX9o\\\\\\\/swQJ3WFK5O3aYUDE4+YTGuxcCYdUDjsUqii1Q0eyegPj4IIfaeaa7A\\\\\\\/Uj2KUzFMg1\\\\\\\/fZHqy2D8rRTYLNo2NLT7ir7CSIW8pzHaqWrxBY9pgw0e7kNC\\\\\\\/bymL5930rBBBEGolPXMW2VwcHTlOl7\\\\\\\/lsDXsAAUqPG\\\\\\\/jY2pHPLvl9Iy+Q\\\\\\\/YD0RABAPGorcuxCzTIg+AEGhzFRaW7t0ca6ZqCeheub83HS52eNqlOmyCch2IpGDXQgaWH1qB1PHqrs3LukQbSbuFMuIUgBFq9fcZ83XIKA+Z5gJS1E+YNt4hOZNP9gXzexYK72X3Ziv1ht+kGLC1iiSRdTxm2sL+Xv5LC1ywrQ0ocAPlwQKxiqzT50zCUQygVrM1oRbBwHqpwdhPjst47RCutScx7RNslL46OPBolsbV2NszjKpTVBdT5eD\\\\\\\/LxhjW7mhOGUYsCNS19dj\\\\\\\/b4JFOI+0VArB0bHztlDxijsfBIup5h7NXXJgIqIRfjk2syR3gJlCmdKNIs5Crb9edqNjLyePUUgHhpYjTx7ZDc4gl9huDv1TsElDwJy1g7XygEO9eFRrWljzMlCfg4z3qMyrwGgZ1vAD3x39QGCeZigk0Ejdr5iLcao3npqqqNJzPA8OeuWad48gxI5dLe+lfhKLGss6lOqeKUAcAIg+t9MW\\\\\\\/ciVdtIC5007R3bQD5B2hv\\\\\\\/1odEBjh3FklZmzz3OpoU0mNWX5Y06FMAVEEQGLrs9hZRYbTAARFsbBdvNsm1YLrny4BZzZYlOxgKVW+L0MSq6tEinGhNNt\\\\\\\/0GTNx\\\\\\\/08M\\\\\\\/F3jNwqAZTVNIQE4FmCeoVDJJ+iaWtoLRFMVR+Qpu3V2IArriuTiS\\\\\\\/0FF+\\\\\\\/bQK3wEhFEF+RY8e9J\\\\\\\/Om5QPLK2ZUMgJBI9j+2ClpH4+UN5EKEmPJEnsJok6fR\\\\\\\/i\\\\\\\/B1K4UjZIx1N6zLDmlfDYDOS1L\\\\\\\/YwDpL2h+JHA+EakhvRhki125sgBg92um3pyeWdvciCAdnh+cy0GcivuoMioeV6kvMhtNsO1OseX0xyn9yxOSd9r36DYWkRwk\\\\\\\/Rq53YafAFXQXUY2SNcyjkzYeWhLK5R8wSyNYF0ODWK1oKYkFD\\\\\\\/6wAAdbJSp3TAoHTipsg5O+tYU4ZKba35AXMNkZrPG90hn7Zo8vJkaD73dRTo+9y4duUVy44veUv2u6zFsHNMAYW3EFswKvZaTLD7GmTSEpvPM8ctG0RHAQfaALDSIczFIZgVdK2Qgbp77WyEagYuB1wGDuR8rNiBUJ7WQ0cJQnVVP0d6+k0sukSKESvF+DSxTmAHhPBx8scrjU2LENUYvzF6QAW4JuD8v\\\\\\\/KTxk3kq2V52BjTV6FHl4u9\\\\\\\/2YUiBVNLeT4WmAjNjeME1H2aN\\\\\\\/SxGfJj+gMwKHz\\\\\\\/pxz4sP8sOzt1iwvYJxcUgCeJ5XeuWtF47PT2Rsjq5UGPYf544H1caORbAQitPIv3OL3CcIqrbMrnbNxSxdr8\\\\\\\/kaHnnILkio1166BpDPg1aRAXdyMey6VpJxySJDqit7FzGQdTRqJlG58g142W5NMqOpGPwe1VLQpRwQ6yjqMqnUGWrq6OsKGTByIgOhoBI6YceRw39FoaYpwpnsXrg59\\\\\\\/WEtdBZRblo2PBR1UB5CtSSzXtN\\\\\\\/FCY3\\\\\\\/gr4zhtXhaKOjdGPlce0gkHrshVYrto8\\\\\\\/0vIlRmCoGHZfFRIZwitqI4ASZBnfENi05ApKhW1Fapze8r8acJXfwHwR9V7GZFJvVsto\\\\\\\/2+HNSG0K5s7owRQXB0z3ncVqFR8JWoA90dC9H406OPc9Lh\\\\\\\/S\\\\\\\/CxycFz3rbY7QblNu4rcY2u6Ham4I\\\\\\\/9W01IpEeFCXITEbUv4nYEhxWoZw\\\\\\\/r\\\\\\\/xe0mtZvZ5VY3lGJIGMYI29E8OqmHjdpv0HsiiVWAz3ZuqR24bj9Yd1dEjzWfpHMZTfJA3U+gNv17uYzNZ2UCXBLpk0AiXw+\\\\\\\/\\\\\\\/AqcJ+41RA0YeJwgSYyB45CdCha1Grp+B3fDQ0X3AvQrZgy179EY3tkAWeYNWS6KGlI8xlL0dWv3ebCP6rfd3Vjq5J+cYsrfly5Z4oiJmkyNGgbyE0CwauPRwGeQKEeqFmyAIPWFBVFj0AbVYKggpF6wTELByHYIZp+iDpJVez7oW\\\\\\\/3LLlCXtmmUN75XzIWK8psX8YxJUudOI0fi4+BxeiwNKD8WYem41VOKM+5fYTWscsEwnhIlccgyNyP96\\\\\\\/QdIuawTU51rFkmcXndYt3YUbFyKKKWN60sMLMZ+HI3uk3OdrzWNXmWv84VHUIucKIGO5EtmJSh2NZdX1jizh6+2JVH5bUmpozY2zsJG\\\\\\\/JGp4UQDMSLuILoszotpHgGqUM3GFXskBtVWqglKKERJyWMieDb1mWW\\\\\\\/O8rfu+1vgoHfR\\\\\\\/RZ6v8hBrXv\\\\\\\/lmrq4gjz9cRWqDqgEp\\\\\\\/MfQ2g2y\\\\\\\/343X\\\\\\\/aVWrkWC4yoSjLYEX\\\\\\\/NQ8l1mGi3fklZRZqHoyFY0M1TghaqJ67E4uajbHUiT\\\\\\\/ywlRG1gpCj3Pz1D\\\\\\\/0CALuXcsqzvnvcVa8vtDKIvrw8fBE716HOR8TSQNeZiFXRw85N4acVYUDb2+e4CD74H+nc5mUc63any0hpLyekw26r9gl97Pkc+Xxq79csgm3wAh+owFPeJBPeuVRbINbEg8xJzqUyR6+1mYSOtC\\\\\\\/THBwomhgYuSXl4jMreMhgcYoy2cF1g+uq0+00cZNt5mJAOX2YE\\\\\\\/c8pKs505mYWUmzvUEeShpahNh0KKGci0S2f+lP6l\\\\\\\/2KFMSURZs9h+UMwqDTRFEx0IlwI7Hg1+O+8QFbeBKLNGV4k\\\\\\\/wiOFgcMP+s+Q6Sf699aSn6hdSvIv2vV6I6nGC+TSi+5hU2wGcJlfmcCQCFk8wJNp7yAvzxZuMVbfu+rz4anDaui\\\\\\\/\\\\\\\/L8eNJMr4kiwNwkI8+mO8X3GR7fxK0QeZflECYytPZMca+VO8IdnTsHOjKs4F8\\\\\\\/8QcRCOo\\\\\\\/ZznDG\\\\\\\/FOf0mpsQnBy1yi1jszLiD18XUunmPSqZSwZsXRsCellmNH4O1AwFi1zRb37t89zrHv2\\\\\\\/\\\\\\\/0N2xBMApu93XX8Zn\\\\\\\/3tZuZFrwQD2r2DWbMrzJEUGqxge6F+hSLVC40sg99mO4W0+2VZ9SN2DrUT2DwMnxgwXCgcTrFGPV7IyJYclkbMg24c7U1h4A+SYP+lFJEHfPzF8yOjm71aAkchOX7hsGg0b\\\\\\\/MLpv\\\\\\\/fFooBrA79GG+CroKd+nmV01UdR9dIqz3VhEeAWG6A1vFOestu+3GTxAZR3t6MtYkDsKOkF5RgNunjQoSp+z\\\\\\\/VmcNa8v92hJ4fKzYPZOZruUhh6xh9GI8eGeEnditvZvVxEcDWS3JjyLlzaNpaXevy7jGqU4+bPwAiRivSiwCb1ligicmAv70TPAx2dyZoTuPmm85wk3qe1UUwQupxr7OYYJ6N4fLx6Yd9f9o7s\\\\\\\/fzorLGpc1zeNNGq2Bcnm6qN8OpDqAMwH0KmyQuBF5jJvgmm\\\\\\\/\\\\\\\/wlmRGpDy\\\\\\\/R7l8GnmVEsooZ38MhFHjDirbdH+aMy+9p8vmv6tmJv4yGQqsK7HjQPrZ3CAaXWDIZQmkxXExnUXAdwxEDjebpOUgPcypjyt3s6hhn6avxMkR\\\\\\\/FXbj3TGJmhA6XKjKUJ3QBbtT7b2DSwtC4L9+jJ44UrqR+unFazfhmsQiutQ2byz770Rn1BRNstd6zcxDgXxTuUSacwNJi9DlxGr6VHs9Gcd0q4x8CkwIJfMHbod9soBQ3Av+PT9X1iuHfWkkWnprJLdA\\\\\\\/sHTlqyplAPaKIVRqpUzzBqqkPLjEunCLm5OHuDWVppKPLaND5x0gENnqEk2vcNOIWEOIxwvRveqfwqXlzj+ElTXd+jJc\\\\\\\/\\\\\\\/260bPcLVx70brLlbbrLLip2hs3LKwT7yGxqKMnfYZOBGSDGqt3g1+XfKYxi0GxlOLIQazUffLWSVDmma5mbH7K21UJ1UMbCZQj8FBVGeusfawlLCFhxqFNGIJLeCqpfBI0JPU9Jjh5CwOhoac2VxfvcbdHz5WMkqta9rdYBAs78tEEJ4VyBduTsW+sarx3p88g1NI0UW8Sn\\\\\\\/MalC8b+0evmfcFF7rExj6ssitjNp+FzinU1cmYvefRFyadTrmbpndN7jzEJmwLrqVuHwNzI9kDz0EpLHcNFQXoUFZIYX+NFiwQyV7DWazOYMkXIaod78S\\\\\\\/keNPPjgQRz5hW3NWDYXEGUJO62mfb7dYrMMkqoVxmFUcMQFybBFVefqGWzsIrp+f0mFJrNRlLEemjb9nB5K3omlOYWO7skcBkbws5p6bOZvXCvE5cHP5M2O4eMJUfkufvTwoxz9S98xNah4jyt4KMseydVQqF1eQcFXzC7HnsXADO\\\\\\\/skfLSywzU+sjqh\\\\\\\/NjMlYsyMn0nvdWjRZa\\\\\\\/yA6Pxqot6qz0CQ9PU5mLR\\\\\\\/5rmCyqY+Hu5Aa0JAPiN8iTywnsGbihfMylS+5NDy68h2IetqKsCbrV9e3xw3oizFwaMRHSzpUNN6wR5vD8cndNXf5rp7v7UzhR4cOPn\\\\\\\/3D\\\\\\\/Qb5JQ9sBhZAkZtJLDVR1leACFxUcfq6x03NAS+SCNR0HEa3eN42t9svzV6Msc\\\\\\\/XJG5BPD6b0bc0D6vKHTW0lFSkJB2kj4ybt4uPb4EZFkDKNG60Z4iyGs7t69aOIT6I6my46AjD5eQd24RhQG5bzZWYYZFDZY241n\\\\\\\/lwhnzhc2qbPBV6OsqFl4MoUxfkCM3BIGwdkQsLlUQwqGtva0z7LIY+vsiWnzHq4AAJbqr2ick\\\\\\\/vteM0O4uExugwyVUbqwulIXfB3x4seHUxydOhdIji3pex5J275JT3+d4bZTaMDWmUBUFBCd28HfRaUOocsm6CC3f2mwDyFxmnhjvTo41b6yUt\\\\\\\/FazF+GDg\\\\\\\/1ah+vTq11oBg+KLT4OiS3JrVrGhQOqmnTtspJ0qA6\\\\\\\/\\\\\\\/mq1trdNOIp1lyyKOT+zIt10lAuUHiIwiDFkN\\\\\\\/0nbjCXNRSWikWXi87Se6662xtR85GHF3x7qDJ6wknq5K4k4LfEM\\\\\\\/m0OHX2VpKC0OBcoZ3axJsv9JKp4JikWzXZtnizo\\\\\\\/RhkAwJpjuMTPkqjH6YM3+yEpiic\\\\\\\/lP1jPjG7OE9J8p8NgCSB3JwNlQTGoyhvo1RikEzHLg0QFxQGLPG63YqjLwPbySssPovCCoENPr0rODTbFtNN+lrB2G4PLGagzvwvsdJzfe4012OJ9qcgqLyLUTgcH6l0tZMyT11jIOI2Gc8GY5gM4iCHBnXp2uVZeDYAK6uv4Iqq6rO4zIVFaJTIVwwWCytflokPPWQOK7PTPSrLw3plu86\\\\\\\/61yJvw\\\\\\\/tQKoCo\\\\\\\/xN\\\\\\\/WBD8lVFfnBsUqtVLKNQAW9fkPU5SG+RIke\\\\\\\/JrcN5X9YGQG28Wtzgh24OdtB5L4Fe\\\\\\\/0MacPyrgqH\\\\\\\/sz9xRyAZBl34DklIWQ3OsEFmsEtFDRvmbmp25dsDNgDANuvGmkZc6TJYCGa+BdRPeE06FJkaWLve4GufxHpxj+mOmnMjwzeMYY0\\\\\\\/kYQIMKHRfcz8ZfWIX9LPYzXkf3fgT4vm42j+IE+Qo4V2Z8HKsEKKXZTXbh5QcOQErS2z5mosLf6swUbiDTGzmZGm7f0XwvB9NOoXLGPghMFp96FBS825ixtjAnGAt+hPk44QsSzgHp276nYDItjbqgPfa5aKHr8BEOzK0fSJB\\\\\\\/fasIj4+P98e381jWKQ2jByycKv8SnfJeVRcfrUnig1iaTrq4mb9yN\\\\\\\/tYHpcx7cacdcwkA9\\\\\\\/u2gZBMmNR\\\\\\\/q0OOmRQeNUuFaLllYWYof8Lmcj6beYs8TddKtfSp\\\\\\\/ocGW+ymlsUrnkwZX7RPSgGdUmT20EXpIq3QIK0A58XmiKJVWRbLONKKsoOFs+Vam461a3zNDORDt2pAZk6qeAXMKcpf8hzuAA2K1kMXSYyLNPRvkzHiVpaeZaZtFbN7TOYX1N64r5ODgU3XBRxIqmo3P8nAf7Q1wdyEitpixDpS1Rb0dfkeFy3BIp6gdGrPvamPyVeEXUepBy8RWZe9YM2fhiOcH7knVj358BQO9ztZXyQ9mmCqYRSI2G32z0zh+5GrspTnv3gTomU6kuK1fKUIR7ho8LnAoboaoWebi8IELLI0PBla+ByC7JirVGObiIFSJeo4bKOEcpQ1tBGn4uS08K1nbLewkj\\\\\\\/EpZZtJnK8UlXiMqd3O3uM4z9uwnEYgDAZmmrQX8b5i0AsEyRCQX24tfTMPP5ozjKcVGZ0ZLhjN+gohjurkVVVHBsWRQV2rRXO5ggyh8xIX7Xm8OTO7i741ii3e6EiBrZDKirSPtTqLc4MnKxEB1EjxoImD3UgDNuWiP10kJYAcWFwiqfplc11DKgEtTaJj0OMxyGdIBNKNkoYM5ypnc52JYhZd2uCDt5lq8jatJYxaHqbq3KLFm7PLTCF+POgT\\\\\\\/ehNibQrZL5FmR3q1UjLkvt84jhoiBUl641kBcgE1j\\\\\\\/5E\\\\\\\/G5r7mmQ2gCzr25odQNvVKxYRTSIunljnH7ToSwoRRf1V2H2YYiK515NHXniBUQ\\\\\\\/xM\\\\\\\/fV4ctNqu1QHPB+VAgIqUCHOGrcD2dtNGjNSX3MiDQD3C19GXso8suUcUkWjf9WwTYccHNa1RfenN11owsiflJszmfZYjEnNcj8X8Q4\\\\\\\/a2rR2LHyU7l5ahUMivdn8jZgQwPoqH1MiSuh\\\\\\\/O7rWEY82nHfAosZCkWNVdL6Fivb7T4SNEAbsnuEc+pQPb23GcFr95P0WL3w4kLxq3xSfpim2j4hRMCRKuK1hweTChSFMIBSxxSnMvUAlLGmIq\\\\\\\/FC+JmVzbP507LJXZtdWU4DWJeymDIAF87n6IaDMILbZpM4ekbrippD8rQpL\\\\\\\/GdBUGdS2kJSSuLGaLUmLAigG\\\\\\\/eoTq6tuS84Yg+XzrNXd9N7yVBe1szct77w+FwwDlGhhpUnnJpnJdJXvlHb+E8KTqp645bpiaddDK0klaqKPbuQxvtEG1PeVo+NizTv\\\\\\\/4AYuCcQfDvtwYRJBkDqu85uKI8CHyUGdthgm\\\\\\\/dP85JO+HB2CVoYx6Q\\\\\\\/xfkymvKyHIZ2w50j6\\\\\\\/5mZe1OHVg2if39f9uS058vJiZfRhiLO0j6o3ZvQ3AOj683msdPoWO0x3h4WelhqH3fJfYSGSH55ApQqCmeGq21AVEUzJvMDr603cIndtJiJTxEP4DsjmKOs08DdE42Fav9THpTZUjCJU1hV\\\\\\\/q8S5sNNnzxO3j4Og0Px0RGtj3lU8vkb\\\\\\\/utnqyKofk2H8MF5l\\\\\\\/Vcjo7p5hJqM9MibiYS22nZ797KFtC09SXnYldbdFx4sEdlp1jMgZC+OJsfULHHOHITzrXUu56vhscWeAgI61G+oJ+czExm319\\\\\\\/XMdoBZNJHm5KLqDybHOkH6FNwa7TgHMNRpLxZ5SDshUXgCZ\\\\\\\/ud6HWyXrr0qIR2PCe1RW5U9IEXRzRJQUOVX0NoIMx7LYgEul1fxIRJ5fdh7LPtmIkrZ\\\\\\\/oT3oDkFp5ODLpK0OQK6QZ4oHY1IjZN8LaQ6FrjaQyzFYk7FS96wO01D03XUhR2dUiDCrvUPKjMgOApM0lXfv2IJsK1o30bHSrM97uTMdNAZZapqJmu26sgDRssXzuRyQUUOKnkuLuYq0i97I2X88Wrc6jyuudx8\\\\\\\/zj6WpPTXFuQNEO0oEhQzgqOU7ulujyS4emvNgfatajiOJz9op+GFwVX9Av5FDRd11unzvGxp4P0Xe5ZHpEqHbJZlltUCpJAkRMpK9Y4VAZvT8th1jk7iiWEuKXZtNye5r32LfRlqcO65aTvv65RsryinzHbM\\\\\\\/6stxCZmIxKIkkIPPmU5\\\\\\\/umRG68fiuESeRNTRr+Ps1tUD7ynVFFK3EZDDDlRKwxQVMVDI04NpmqL6JX55QFFmvhfLOba2xMU+IC4m93BtSfNO4iND3pH9M1w0US9ftmi6wAsnNR67Z3832mJb60Ri9wIcHhv54N3yGf68XiL\\\\\\\/2h1MlrEjjjYvIeEiQI\\\\\\\/Ltmg8hIMqyLYCbWSvQbfwgR2N9gJXeEmyvhJ5Aj3qtsWqS4sZKbdv0pgTWZhkv+hGnLA\\\\\\\/VMtlwJdOlJkUzU7r\\\\\\\/QH1XBLFWmwZLa0JGOncZXpDJz4aRmR61z11xf39X11quvW3AvvvAJ4WRlPRsZO1osyMPWTWfFLIIavoPkKjTlDDwy\\\\\\\/MC85VkT01PdpzrfxsH5shAPYnlFjjDOYyhyisWZd\\\\\\\/SkdInw6AY2Se4PecalOTgSCkQW1h5Yaszvo6XTPJYYo5FinIZMrtn3U2Xr5y+saZjIqoTgf7Z2qHzr7pUhH1LUiy7YjVW30yk19hI\\\\\\\/DftPxeh+oUnlhT2Mt4zz9xTKx5bguaAtEKDToW6LAzb0kGy3UaVI8jPAwokKSldhcIY5BGdBoXIEilKhNQJoh\\\\\\\/\\\\\\\/gqGwILS\\\\\\\/CQPglULwJz\\\\\\\/ou8HhvJpMbhrv66pdlCZ\\\\\\\/CEQHPdNLihAvX3BtQLugVSgobTjuj6Jo2\\\\\\\/tQFVKemDU8ei1WLSMMXMMWqlbjFLUg6MMSlaM0aLUFVo9OVTACDonQ6kYsB3i1ZSOKYY9d915hE\\\\\\\/jag36OYYVpn6GleSGEYwLAHuzu+4dGManDjiKyydjbP9NdM3Bg3pZcEp\\\\\\\/JJbJr+ZuW3SgKqCA8r84XDEAWcRxZgV9d4NzlT7ru0Kxf30fjo8pr\\\\\\\/j42T3B0IqxBZVuYPY5R+Iwx9lPiqXteF4zxnJkhqe7\\\\\\\/34JNrY9HF3Mk83+1sCmdt5\\\\\\\/llzQxXCVy2nsACBEHVkF9dfFzc0A3CQaOOlHfazA74HNNeQuatqJ04KxW2fGW6JX1cih61XbP1QRuv3wCvpT+OCruZHCHqDcNqpGHZH5GOJYpOHTiCjNHuLlHoybUIs6zq4PZ+iwWbi4WEuc3agYI3wqM2hcZ+41WAHoFSuo8mx0AaoLz2egfUSvzqdDpbJt8GfaB1FIRn1+oHhMp\\\\\\\/b0q3PaFhkzeAfBzrNNLj\\\\\\\/6SHUH4vgKT2FULlnEyljhceKGoKejljh1zautSeYBYbr68+SL\\\\\\\/roxrq0fAYP7L8CZqp2bqpTXdfeK4dyQ4xNU5egFIbs0hA8XLC2ti\\\\\\\/esXodEwfk7fAmJfBM+HFdzbOC8MfBBhXDCBERBVvhA\\\\\\\/pYIkMi7m9XckID9NRHIUNCvw9hz1Az1wg\\\\\\\/tI03L2Hi012B+go6QENdviZQx\\\\\\\/ncfmDdBZe+f30QwFlyWa6ddYR0gL1FUYmrWpFHoZPLtpfbSD+ucz9WXJmjSAfWeUNfjhg3BXf7JycAIYbcDzWSc\\\\\\\/yIHKMUOxDofD6j\\\\\\\/XFnQ+SrAtdyaVEYf7KTmw4cSKqoo43vwBIPc4q9g502T54DPggMSPgFJZXyQ4RHAlpLwCMVbFqYIzHcIzeUrE7FCtJ9y\\\\\\\/wc4vdNjUrcuGYhCtfTU4Rj7xSC9mpFmC1XtFQM1B0kmcy5RsXyh3ieAA0hNRNxamFbZ9p9B7iokCdOzTIuXSqeWE6u\\\\\\\/UbFRhpUmJMnz5Ry6BE5IdOcSE6w5yG40a26od2eFTy0jj\\\\\\\/bVjAZeocc4l9jPDqC2fsOj1cb0fiKGShzipltx8rRPORni5aGBmeciuFpOhTNPJ7aU1H4ZedONkpsvNKBx6DJm\\\\\\\/kGffi0GCeAItrkWo2jLZmxw1Lu5xEfj4HlwZ83WkfxiIJxQUBCfh+vv5yP3EW0Kt6vKPUaHtYeMesn0qE7gUoh4BmZDI2uLgUVmw1lA7fc6n8Z4Y7pdZ3+a08oi9VZ\\\\\\\/h9+jdIOU8tBMsSSIcmE9n29l1WZjRJBxVSMQ0cEZO8r65L64HBidzWDqt9zh+6NV28R4Oncx\\\\\\\/T9rsLjw\\\\\\\/YtDMEx9F0eVQOwICmP3ipPlNfbb2eFM6PttD5O8M9OljjRKZkMKYU\\\\\\\/2HyX91C4AvBWMJU8WQtpc9te8SF6TQ06qAc0HdihHxEJPOWbkCcqUGAuj\\\\\\\/GcCZhs8S3fqW5rFIUCqrNFRDMlw8RqclgK7BJ2PUnjRNi+8E\\\\\\\/V4Jxjk\\\\\\\/HxmCyQYFmR9iPWSgSW5eXuxwR45uXpTCExRboK0g0Sp9IooVgTETqmAH+5+10ty62zm17r1e9Vco7XNWaItrHm1pjqokxhggP7zknFNttwaRczCuigGxiq9F6XLqgSiAfryZ3ZGJU9domg0ihDJQtGiICR1m\\\\\\\/z52k5y4DCPE6xOJF0Q9KMdbwDZVMil\\\\\\\/JAr4GIqJXICElqpitSCTxeeTeyYr3Jh\\\\\\\/j4md83eUmiJUgS3VKnHFsVPIOypEbRQ8bjwEeIlH\\\\\\\/gQ\\\\\\\/NDZei8Or+AjHY08HJE6gjymhaMERtZnSWHMFjwz1VnR6gX09dfWbtneedMEm\\\\\\\/1uHpdkyUJt2Vkc5hd3GXhfyXvS768R7AFfVnuaZv\\\\\\\/qzpqT8oSvSBL+c+YzpZ20Nmuga94CHJSUzEmeEoe2N00PmtCQp9n\\\\\\\/ZTYnlm53k0B23vPf7G9UBvWp7uYy15il0dOFdsAKO4Rz4MRky+6yXZCGqNT+kvjq3wz4Uh9\\\\\\\/q9yPbS5pmfqd0Eg1CoXMBoxDf9N9byDbhkJpAoYjXhow6qJfBfz00cLtX2rHMhlLNJGLhMzZ0\\\\\\\/k65sdhac+O+ypghUQKdbHT1dK1zLNcsBpY17+QGnR56BRQI0GRGKaS4J4ZA8qINgS4Wf79eCxuA801urs2fv5BXb+3KK77kFPAxQJv9E3IpVjasw+qvTTOeJHV4a\\\\\\\/rHZO2YTpsrA7IGi0ecl+TzyHHZCRzZT4T5cCKgM1vxRBY310TR2f7xKa7HhlxnAG36JBp7HzP2U8Ixf7PFJLfwOFbD\\\\\\\/yrl2CXzmEVhPl46hbyGuqBATM2jH98OJPdZn8C7cW46LvJoIyPCKQp5v4qz6UDTk7YeM2aBRseE0sO6evsa9MolWfcm3aV6m1GsRn84AeNaWMVnljNKFk7YAjk\\\\\\\/bWvYSQoKhSu6BrBdgM8AR\\\\\\\/oaHDex3xbdICoq+53mznDbGcEYoZmDDqfdllOb+IX\\\\\\\/tTemNfGh\\\\\\\/HqCjv6ngghN9yN+4mXTVD\\\\\\\/1Oni2P1293iv+YnEAdVHfPUilnamhD1+noxvYJLuCdVeSd79\\\\\\\/2cwg6WJbkRmHBi5Ds9vRTNtm9XG31tZSkwG1hxzpeRVAYFuKwM\\\\\\\/Ja8QTTcnnq2nIhXC5gpTDvnNEkZlxD6N3lAKaaVVxCYl7uLbpCxIlcTr+T9y3NY\\\\\\\/SvL7W2y4EA2lIQbRHCUXVu8p8Zhwax4MVXG11hra0pqi32AaYdxOoGHx2i2UDHLSraODlNKj+x3YOh60JZrNT4slhHCNxo7zhaV0T4AmIQs8IC6LnKeh+K2XXggMEOp7AQ\\\\\\\/7kqF71akVVWJ96sL9pQJcEziMUP\\\\\\\/CdBt3OsN5AY38n4IVpvWGdhwUGaQxwSv4n8+j4BU04ZnbVLNUY7tyDTUBy2V+umXbYSbdNivSKXOuc1jjEIze+vigrFlyQPoKky7SlL6cOoJC0QkNpI97O0hArMh8zNzA6yJ8XjU6BbLd5odcaonlpzR6lJNguLz21NYTXKlSaGZmckx06j49lzx\\\\\\\/0pnSuH\\\\\\\/npc9WYP2E27gKhOrdTys8wga46g+ix3M9Xce94gWze6WqB3CVAW7UiMwFdMIsoAgk9wO6Q49r44AMAcASbL0ymLKanTKL1Ojm+DO\\\\\\\/JVGbkWxu8WvVXFIZR6D7GyY4XGwd6SQpoiaagYPh9fo7F51WwD7IQKbz27e\\\\\\\/qd5jQw92154LaoDSMuXjBxr\\\\\\\/FL0spy429du5HeO6lBc+8WfoptF+ooR8ehyL\\\\\\\/a8r+cHE+fleAJ2FrRYyJMSc9NBBH1cvdh\\\\\\\/MWyW2JpnG7vBhWC4hWr0TXNYbHkl1x0AUbXMDY16agYbVBPCzgk9yEkvhNXzK1CaebwPLP8CcoiL4e712qnDzzds6CswsdLVGLW3svJwsem\\\\\\\/PXjy8fjpOxG6rdGXQ3WCs3U+uJGOye8t++wwjib9xQ1akkyArbm9Ec1Ryy\\\\\\\/13HW9ui8Fy1Kf\\\\\\\/fh5zlJZqi53On3yWnAmbT93u7q6+OjUJhRi\\\\\\\/h4iyUfslc5GKfPpEZf8MhdLQEhwbo3RNY57DTCmr5hNwjF\\\\\\\/ULeAsQ3bJXJHWq\\\\\\\/99VAK14bYGh6klQ2gnD+aTeWQdOmNK61OXTjk4fgh9FfjQyRstvZIlgcSO7+LtkmzEDzeQBCmjkg6hYo3ZirJHpNDrImfR8GCsA6smrUpgMDeU4yBupBYM8yai0cQ4DUfQdu\\\\\\\/K7xd5kXNy2QkWhNoCP8ZQupoHCSp0i791HSKgSy+hm52c4kRLaOYlTUwBkHIr9dmEm5pZZZdsdQRzufb7eTuGQ8MaSyDKLEZnuy0HVciha6UpfUEYOI9542BHTyyE23BF2a6P1rPs+O+X5nQMFUfyQVY+zpNQgny49Uj2IQJnat3ieKBiTA9olTt35oU+X4lvXtEuyTP+Yss0f625c5rWODiy6RyKl0COrPdwWG+ZqUzob0e9CF9ObojFGaw2uELTkx\\\\\\\/rWQmd6+2J4CiP7Gu0yMxA5bFYiDRyArqy\\\\\\\/rJNBhyLG17pMx6TzzKroPDQoxz2+iiwbIwzqBzo5ygbKfiHlTJh2p\\\\\\\/6nl0PJNG8ebaw66KUGFOuH7p5f7HjHBFxZEuuOBafevlXsYAcI5dTShsFkn1HoUAvXQDlUT2cSYsJ2iKXBGyUBBpNOuieTEwEU+HohXDXaCEjp4UO5wYWvOzc52QzXAK\\\\\\\/visYdarKZZ\\\\\\\/fq7xL\\\\\\\/rZ8H2+q5ceB+rqx+Wq7itTIgDfZ4+Vf3IJdCkwKAZ8palR9iMBjkwehcCOCdmefyG3lkj6NlZDxyeqZZRvYRPK6IoejjuNJIWwravVmA1VUuF0D8TMT91xCHWjs8fMndzz2nUuR2nS\\\\\\\/j1oGaYlF0TapEFhs\\\\\\\/XROcOQxuX8PWxr8BxcF5ISbRzW5f3ybiskTtAFQMLRn9qGWilHVo4TE0sE3jMfcypRKHQdBRKTkZkWaG9JOgPsJ+yT+p3If8iUqBYa\\\\\\\/AGeHYCKzVOXuMiP\\\\\\\/hcfga4clc4JuJgtumZhmrgiDnbE0q91ATS5ScoGMkCOQ5LyPlCk+0dYr\\\\\\\/8npRVy6Kbk33yv\\\\\\\/kMWPm\\\\\\\/AeQg\\\\\\\/9uhrEWeZOCNP7GFqGOcMNtjJ\\\\\\\/Ef8pqllCnlsZoglTRDuIp+XhV0mD8UuMKGKXuidFS02gsqYR1K+tegy\\\\\\\/f1W0ICudR9qImi\\\\\\\/4Uao467jxeSliBDU9jvEDVm8sMZvSwyLdauvdMia5CRQUtx4FGaBwzeWiYPUTBzZx0ijt5JlrkGagWAEGaV0vSWQuBAys2g7VVgr4bEJSShdY3lF\\\\\\\/vf807hKHuP+ymaLwo9gaRBcaLEH3HVH\\\\\\\/Oz8jvOJjRSjjnZS7m+z6Rv09PDWV5XrCjLFXrOhAtOXAPki0G2OSmeDHEjbs89yBURZycl4ukfuNHzdy4l9s3xssQ1xZQ+ioWz6dka+PVZiH\\\\\\\/IGpP1A1v71GOLzRU0xcQQOizSRPe2AYcqX2N\\\\\\\/iaj\\\\\\\/2mGXloKxwPRTlImdefNOvmAtY+Vly4E65U03kX1aic\\\\\\\/EO5u1ucXsBvw8CL9FVlyDnLgJWSPsfqE4JrjtMPA5LtNR\\\\\\\/1Beuu8tKIFsvroukP6AdzAXXehPFwJ7qOQ0+m2sHWNWGqSJtW+Dp9sKz9IvwbQWBHilAzMAQOye3uWI2Lb3bvoN\\\\\\\/Q8CzOtzq017OPU8myWmNdkd0yrwQC8hlNAFIlWbUzntHEaS3saMym5yv4jKMggmWH9jOruwA3wBezrbgyHDjo15gPOH3hYT7nWviFrURkLSYPTkKiOj0CMfheHlGzYkhxmSt9wuiYNyvfPKr0FYY0XOD6nVXTx9JdwsXBkBjChewGdhdelSUJkUWP\\\\\\\/OShdZ6vie5CyCzK2SZzid+2vmaCKSAWlmxrwlkeC1zQptBsCsWwUoINj8+qLJC2xPGh4zGs93XxwSl7JzMD5uN2vhvnBcyZ4K+rJW9\\\\\\\/ieNbToCvq\\\\\\\/IFXnmetDVwGd0yOCT+PG07\\\\\\\/mezY7XShi\\\\\\\/oSgQ7R\\\\\\\/oX3JYtj8hZfeofmGMpIEJTJxC0RKpe8jLLBHSRVpnvBYmzk4ObMUY0Ki1Z8DF+WII\\\\\\\/5mwMUDcBy6DgQhaexv03vfcvQHO3iad83RHmv711ifsJ4gfjQ\\\\\\\/cvK0UOma58eikBrvwdzPkQ+iNpjLaXk\\\\\\\/istpfClUnFH4ocwtVSNPirC0i7gODndkgTkFTNF\\\\\\\/x4fFqEWAaBhjOFyW6darU+FNzkgp9hDEV1hUH0q2qrcPwBkqHFQQxltAy9DcLVNhxR8eO7ZJdAJCEEkekb3bv+iburEky7SBkG\\\\\\\/PlpCaUAiFb\\\\\\\/4y1lvaIqQT52mwjicXmcobaXoEf\\\\\\\/FSXAQnJbXB9+XBu5eEGDWJmEt++i6nlK8sI0I8FAjYtHd\\\\\\\/JfaS3cVnjaCgl3y0F5Izk3yylRN6xXLH4t5FYuOAKqtoi48XmxDxttpu5yjvS8uJuqoS\\\\\\\/QGfhuUbR\\\\\\\/wEcuriDiAs4SDUpuokIi4gsml2DQZqBs\\\\\\\/I+yjaYei0M42ug6gtwwgazuGNyauzbbve2pBVyKhygLwKwWuEz5vCx2jRL9BsAIrQfJzatlVFZEZmwjRcDR3WbISkO\\\\\\\/AOb5iz9jyg0mMFK9lWg43cUWJafx85g5zLgasaWllHSul\\\\\\\/8cDxxJouQ64IekP\\\\\\\/92phK8v9s1\\\\\\\/yVc4hC98TtU3Y\\\\\\\/AusQewEjqH7qmclGotfLt7MxEH9pT6GUs4MIxmi03yhtCGs6z41J14RLadsL6c7mkeQQ8xmccigE26dSz4QkzFzkHwHMDnI\\\\\\\/0hBTBJsnLhsshXCeZPIjPBQbfGzLkZwV4N5iq2mCcAk1eRTGbPVwjlRDEC4bvrWCzweGMU4vHgJM80k10dBF0n0\\\\\\\/\\\\\\\/YCWJo0m8Ay9nIUdH7AYT3CLBhLA1uHf7smtii0ZFl46dlrX8Pz54zkMbnU5tyir+tM7txAetTJiWSvBTXgvTgjBstrh0PJFxSfSYJbSQjf2Xwr9WU\\\\\\\/zdBi8cO2NFid7svWPb82MmWmJ0ef1gTc8SNAYJF3XAAw\\\\\\\/m+lOz6KNk2UxVHRfKIT5oFRnQemfIIwh5svG+dxgqSiOI4J9hFpbaME9+i81nsuEnM7qAAi\\\\\\\/vQvsAI6UeyTMfl1ZRTRGIbbPs4K0faxJxLr4ZstuJsA0LQ7YDFidNQZy81DoEbTw0bNVUJ9YSbKqGNZMBY89VLG5jVf8Q3KTS\\\\\\\/6VwA0MMBrhE69gfdr4d1OU5ArkeHac9umzrCLpC7PYtPqOfzdVPdODE98Hypom+dRJgsuA\\\\\\\/Zq++v\\\\\\\/VlMZ3hYM+rulsOMmE9JExSpE0isyFVZn6n5\\\\\\\/KCn4TnGiUZhtwGKj8M3zsJhhNQHZghVTb5I8pqde+xcGV9cszWFkTBHVpFax6OmOMg4e6oHu2QN8jDwwrmEn8VO8DBclsY0qrLPZffQaaPGw4IWhFUCDFrpfKe27Q8qML3S3ULz6RMvNx2MFwSFzd2RnKNow5SAMnbXWQsdJ0JqpEUaBpBMiAojtmFFsfHMnP27+DB8UxaAX5Wjkmfh0SX1tDMeUKN3oP6LiLK1SEZHRKUCz4W+1n1h6OmMNXEUL0nd1\\\\\\\/eiAdlx+b+lh3q5Lzg9L1B6mGN1QipiW1GH2jugHKFI7cS6075y6EuP\\\\\\\/krvy84JfzNp7e\\\\\\\/ML5ppyenm1SPH6Gul2W5vQNxtewfIhKsRhNsS59xAbTDCDOoxRiaaNvCAwy2lxoCOPfKV3zJro3BnlaMf\\\\\\\/ol0oiYu6ny9TToaWjHiCPFL6Pr70UvcdBXU0JtApNyeN1KRzuZk34I1UKHnARH5PZ8h3GB+iTThMAUR98YZms8Fgeq2D6CnRBeoZBcayK33Xp3NU\\\\\\\/rQJZ8yQd8cXvdBi2la0XdOsRwjxFOM+TQNQmVzWQgGTmZLMW9EeqONCOGX0P+X6pGfIPd\\\\\\\/frQlK2kCMOF8vouE+WBhiu1XmJUHHerMARDtIgcZSaqv6hDlj12AqfHAOzi1K4t7dCT1xrLMCtOpQPlPn9prjdfoq\\\\\\\/BFc6+XCR\\\\\\\/YdmT\\\\\\\/iEk6x4MpbF5YR178sDl96r14PiHXrzlDZ+k0tryjzpH5XbJ9MeEfmxXbnLgg4qnnOQS\\\\\\\/ITcd5PtfU\\\\\\\/6cAfqYZ+8+IuUKwpsxf7hVO+q4CGvdUJwYOWeosxeZxxAMJ5swbWrSD0mC4M+Av5s7EKhmhtf9TT2OQFPvT\\\\\\\/BVrtch3amO2bhgAZn8EYZjs9lICSigKE3J2o9FpTjyjWdzvVyEoA2WuBHx2UWt44Rrg+93jCIDbsTRDVI0twVM+HOOLf5yEYj74Y6FmI3\\\\\\\/tFFpu6TaCRkdvPsLyupB3e+LqBsB2XeLybn2hJW7VIeQZma6VTbJFJhxVLDsBI\\\\\\\/muIi6kKIo6Hqg3qhSEr69xya3CzDpR40a\\\\\\\/ttrLghPRwPIvKatGEqlIsWl48Sr6yoPohCcfVf0qRWjJZTQaOMgaY480XfTHuCdlwo3\\\\\\\/XCrvDfjsysOIP2YdeauCA0G6LIX2tg5VyF+oha1+0P9qk+9rKbL+qAcb4UlUVTwFE+y4gC3vjS\\\\\\\/LfS+ccwIj1jPGoh3Y32KiTFcw8HD4FzLtEvF6tEBQAO7XdtqtcFNxP1cmMGhcPD4VcfrEuiwNJwyAB36xwBvUjLaSZ\\\\\\\/njxsVL9jbNpB\\\\\\\/yd45tCNTerj22e4aUidfnMsBF14nx\\\\\\\/Szt\\\\\\\/Dxq2tV6AgOB4cVuOCL4r679tYqnK6CVrDTJ4cmSiyZcd75zwkjb26ZxTDhLfoBuA4CwjtKQmSjYV0PN5e5pbM02G\\\\\\\/tTaxbFXMF+icAL38XeJzvlRD9M3L1ae+uCvd42lpOi1zXex4MOrQCS6SGBDNBj+YI4X3ha1Oad3IXvHAVmIzv4TiaS0Hmll1aJNSvGjw27ezzUt7bREfmunGzUyE73RQWmzjnaEBxQJfAeJ+N5DagkSTVsvSM6s43Y+mU8tXC23+iwa7TjOKYEw9ZK3n9cvBgOAJJF+dRL1lb9E+UtbtZFck9NTHns\\\\\\\/3MFq\\\\\\\/dkA7WzAF8E7Tm9\\\\\\\/vvE0dEEB2pYF\\\\\\\/WGdtMWDZd70JwvYnx2Lf9Zu8Mk7KeQjsEY1XwEuLbkgdD3HVvj8mDjx06NlO3xlxD+aUqt8S7R2rmj++NO1LDQ4UL7twERswf7ySNIMSChmokunHVJJnoujl4NIDYFKT\\\\\\\/I3k1bXdDJpp\\\\\\\/bI9yuBgAeZjweCmTIaiRlEpCCpuwDAEF0wIf9CiZtxOA0DAFut1ftG6uAt2T+g2azsRKecWsrmJwNXHKvJMF1Zvnf5ywJbKi31V3wdkEJyHFOVELnBMGxQb3dOA1QqpUiZY8yv3PuUzXWjt+tCyt3jZaohH\\\\\\\/5dVhQ19LhiOS7RAOZYxNcpRnyMZjATvAiteub5sjk0monR07oSJs9Z+UiF+qny3qxw4t71IH8HS3gK8O7EZyrkrqbKJdJ8BwWjvzu3YWo\\\\\\\/smsnY0zUdha4W5sw\\\\\\\/Ed+pr6qQeOk03FNOmq3NJBAyOFk9WHVa5J3yGkMh+rSluJ3\\\\\\\/YTPZxWEJutgquoufmcDrXPSH5qtmziT+TUyEpjqGMoknLweVLiZ1hdDHr9RfJQdcV4hw10XYUQibFduGXo7Csf4O6lMjX29V3OsrQRRQPM9OPveLv0TPvrIpVuqWrHl4pngfzm\\\\\\\/UABaEvzsjOVgaDlr6NAKvaGkSi8t43HUcP5R4uUdofOLblYPcNXi0+ZExMM4OXM9TsqeUh9KpKE25XUBiZdE7xiXL8JOb3RRZaqETPn2EWaSJv26GIdRtRCxPIYcVAdPjtRSY5+EVKoQsDSJUR31uwdV8xrAYIGRMq\\\\\\\/y9t0diBgiC+3AIomD\\\\\\\/r5Pv8TyyZzd+lseoRTnduenoOhmvY+folHVHho4fry0uH3sVmWuVjH\\\\\\\/canOuelAuDq+5+Ozm7vf4q9xNax9cUrW2ONnOlGOnw9b7TuX6NY+tr+rFDf1jgBGHGM\\\\\\\/3ZkSY4rnYuiQX6burMMgO\\\\\\\/gUQ7J5dGIsadsdQUQ4XV4ZyuRENPlJaGPH5RGURO\\\\\\\/utg4Wy4cBVqRpP7Id4w15t8gl0fwjwte8Wkm5kJcLexCxvz5FqxDOP24HK0NqJnIW3Fa\\\\\\\/735tV\\\\\\\/2xD7PwZD0DFpIVbbk3sz8pTlPABoxaZ9YN2S9nTYOxH8Tjs76xtLaDqmmJJIWGGoU\\\\\\\/QcFmapbl9Ab1wN8H2E+xCHjmRQMaIll9OSft5VW5lydWID\\\\\\\/6wjpN\\\\\\\/Q1kxowzKgHBiAIiDuMOHXiIthu71b053H+UDi\\\\\\\/erXMZu8ueoy5Fq7UUhgKCWRd+CB1t0nCTDjB\\\\\\\/vOFC5zMoeCgEpQdVknQ1V5er38JuO0EGuWE9LixU4kxuOsXjJ3PsLpJTVbDrou\\\\\\\/rnws7yISqZYkrp3NFs9sGQREAnhAsHv9E7NbVTvRm+KvO8344hhux6m8UUMDosnJEM6Ej8R9Uk4I9hOb0ZoFyeVN1R8H1eAiIU\\\\\\\/oUIlJY8HmMBakXqCD6p2v7rtfi2gE946jG52J8v+ovHFNHzGhfLWUAJ+mdKXvxM5xctxlFnvEKiYFq6XUKJI+9I0d9OMT43lsMBv1vcRPj6mQUZSHgZMnoxQD0EigczCv0QkrIzZmeZmIcictycboNT\\\\\\\/F+1\\\\\\\/oLdgP+HsMTH+F7F71LvqH2F1tO7rU+HiUXd1z034CfXfUJzoJEC+niYWbZVbpDhVXA0b1mQOVbYIxJ5TZPpeUuJqbGXAjJgRDn3urvO0DHPEjOO3HKHnz14g5E9AfecmIeeOXoW58yF5uezB+bDIgeYNxdo5eKxDYf5JeO5hG+zbk6HcTJ+7wyKO2P+5s5W0HGMvJI\\\\\\\/bijOeK0PfYOA1ZlYU5lXxDzIqYgctTeF1GJuIO+QRubM9brdwAON5z4hkGQQ8ZoiJdFXPaB14bVlAWo2XMNucImSHhRrB6CByaaSiu\\\\\\\/nVEcd48kBbsN3Fr8+rLCxuRSDgW09bfmLfND16SAVvA41TI3HEYQTNKOU+w2f1A0FqvF3aMY2Ng9j\\\\\\\/J4SAqRoCLnlKoCnhD9m0uurW6xxV8nl4Bjdy7TJjYsKi6T2jA2Ow48HyP6wovncC5A7ZnG893Bbvl2b1yq2HsvBRBhjcfT\\\\\\\/LJSeH3RdfC9T18FFcXHx92gTOBMxeYwMSOWtoQ+V6fK9ynAWTzV+ONAwaoSxyGZk2RMse11mDX7X4v71vgGeVC3Bed053gr0Un5y2JeovRMZaaMRI9LrVp5j5uo1nRMpdTYEj6H2Lbn0qQpe9gbgKnlWtdBlFoL9NoB9XtTC9gO1eXaVJo\\\\\\\/p2sytSiEBAHSid9eNPzCb79RBCO4LsPyfWEjqfpdIG2W5Da\\\\\\\/NLmknTUQSKH2lzSvkulDYfi305E7hGrrUmme6L\\\\\\\/5wS0PlvK+UVUhgRRJ8NVaLOio4BPoqBZnAjcoLHJtnMksue6Cq8Lr1XFm36jw7jpEslbe1wHJ\\\\\\\/jUHnHnOD2cXAXeZ2jR2SGDqmG0dunwkMSSUIJapDS6C\\\\\\\/PP5qw4CcnrDoXAllljWRSDtVNmI87n799H2XJRMD0fEmV5GOa9U8JfF8WvsEciIIvZsF\\\\\\\/WzS\\\\\\\/j+154cifH1rjG\\\\\\\/CPUd0IeqpQ3xE0FDcjYnhO2HJIinj5gegMqoalykWXCPauvT8fKWmmSHVQX9Bggn4CcAEyuQ2KP9wK\\\\\\\/LB1jOOKcgX\\\\\\\/yVKo6KE0+lROysATSL7FQWaIgQs8NNJQLap5QfrzIT\\\\\\\/xW+5B22kjCvUuwanKzF5OuNaKhUIpqFaO81W3NXmTdEqKsjUggiR4kvZilisS5fsBCBPkY3+s\\\\\\\/jdSESLFctJfn+vSQjNI59LRgrrhAaNttQo\\\\\\\/\\\\\\\/yuT\\\\\\\/0BUVyuMUkejlnLOP14lVUu+yEdPskKSVYRlut0aMez3J7XGtI1wy74MC7iuesmllRs8IF8wEv3KIP7nC\\\\\\\/kORvWPYSZ1Elgx4g6WPCSLk68dYsNtZ1vL7+iXokaJLePMGpjdy4MNo2yk6Ga6MkAXBM7QMAlfLbWgJBob7uQJW1rbf1tKh95c6vOEFl4fxENicUu+gxKg8nZOuRKN9T7loT8HkNOmW85TJ0i8mGNUeCeW1D\\\\\\\/YbKzULfrVvMLLQJFZGZJIly5zvaImWUWTg+i5jiD\\\\\\\/pS6D6eqg6sMPQOvJnjrSGwYIdKYAYNUr5KMhpyS8X12w7BmwuOqd8KKuoQ55B58tPywZA2fXu\\\\\\\/zO1HkqbdqF5dDUV0gWq01YNFlL7mnxN8WrivpaLeybAPdypmcAM0FwAcoJWHD3gZI0YJachxcNv2E3MK10R+ety2A0tlwKr0sZHM0CrboIi1K12QxEyB2kS\\\\\\\/adm1ygVtcMlu8TEk0vR2DsiW4CGiS5CpXFNo94HJ8U9ZSJAT9NDnWyUZ52iVwf\\\\\\\/AOnaCrFWoQMc5ivAOLmtEXC1I3euMtz9hz9Zwq3Np4s9M5Yb2JqJp2EpQC5zv\\\\\\\/9hb4TIVyk86WJptPAlh4FbEcO\\\\\\\/tNHOrfj498UWiC4Mt9AvKi7K2BemOkDvTbzPvQKwY18C2rzARzrdEp4iIwIL4sj8Q0ArdRMks+tzT+hjjAxvyRRAEwlLd6zIkqG3rJDPftvGrqQ2ADPl5Ce+hUdDPMwaG7UPHhk9Lo3m3J6qnM\\\\\\\/6b6HLXLvTzYZ2XNRBF5tjS9IAQowWPp\\\\\\\/MCPdMDD3JYH2U5rmEcN4DZYncnE861ex5ItapgDLC+7MR7gul40CxnPvATv\\\\\\\/nqyNI2jdsL\\\\\\\/CbyXHbGUvFYeJHnTOgX84+bEZ4WhLVJXd3DDKyR9Bsa5w7F4sUA\\\\\\\/\\\\\\\/CHKTlYGg9ofCI3t0R8z6tGVws8ms0D5QlRkSLy7xmI3FcWYGz4DzX+waoQ4xvrzTr7hSDRfsJanw0M4QcCc7nga+z3xfxmnCrbdNWaUs\\\\\\\/Bq0+RycMThPaoroX\\\\\\\/90v5iS4Gopcy2K5TzCQmJjIOzWNlozNSiUyQwB8yYrtUapUQOy0xeJVqwzzQT0HwTutP0NaW\\\\\\\/mxlyPCkhPqJx6oRV15KxZszQaWQ0BEW3Jb8VWT2ns9dWnyjjt3uXuQh6FRNWWJVwS4WEUmGZb73+yCLr3GNkFeNjc+gY\\\\\\\/30z3WVujo8EiX75nkyybBYyIrvk\\\\\\\/hxuUrF6GkqAJY40Iqr98GVID332lp2cjLFvTm3w9jkWw5TIXhu0EEhJiLZuCsPNcST2OtgZEkYKatYhUtHIuVy2kbb572\\\\\\\/2lx6LOA\\\\\\\/EA8P8PrsiWPT02qE0Ca1cRtG55z2+g3LVvKHKktuHr122TZXe6LJtelH8hfseMq\\\\\\\/GaOyVXlhYfyn\\\\\\\/axcV9V4+AtgAxSZdAmmbozyECE3sDrEyifpRJqeFiNuSDazp9zEd20fifTg2G\\\\\\\/9jz\\\\\\\/drORxWxS3D8\\\\\\\/coX2DnqsTzenyceyslfLozi1H2cANtasN8oTOHqiMtmJfP2\\\\\\\/MsvAYN7B4vKLabwBUC9igPKFiFD2yEvrbKERXVH4E9cdutF2eE2E47kz\\\\\\\/nnU7+kkuU6tBvrOBX4tuqYVOttCNplTo0ndiriazyJ88jA8K9bnNZS9USmZjfxMUiph88n4jayxYi05tDR1sJkrQ+kNX4B\\\\\\\/rbzTAXWWRD8hZAdzNPtXDcUxxynau+QS4DBTANhPKdlPpXxt2P0rt82CbWrCrOqSdJB2dXpiJSkvAFdRjzVC3MEHnAARsRV\\\\\\\/rBv8IxHMUqHQIj14wRRJ5KPd5Mn4S1\\\\\\\/sWcctyDn7X5Ol9sQRLsXqCLWABE8fr+iyUmUbrW46woSFcvM1TpG9xbXXO1UL7fa7qhDzcqHZi2ws=\\\u0022,\\\u0022iv\\\u0022:\\\u0022S0F0Whte9CnTAPCbRr5Ohg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e5cb483bb03496c6db656fb3617a9b4af882da03b6ea1f9ac089b9e2867a18be\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00221VUIdR80387w51tI5Rp60CfbbTNMb+\\\\\\\/W1qm2rtxJGrvp\\\\\\\/QyaGBTpJ8LCMy0uIV0+kKU\\\\\\\/3wJeYjvuM1G+f8cXJH4pn0YJ9vE5gngHOvFuFdWKFqtSFYz4OiJudyhYOaITwLIZ0CPGfCe+rV2MsZHu6G3ouUx0nmpYCF1pSBOzB8JgVHJWLV8guu+DeG\\\\\\\/ABKBPYVhWUbHob4cp\\\\\\\/7MyR3o2pMgSwDLL8y3JAKyscadh6bNCAPrePHVJ0ItVHaQZ0bJArqFhbXEi3KvqpulW+cSL\\\\\\\/2jUNQs71erxZYED3wMi3J34Z4uHOPpU9a1cmDQlGz13WtNjEIg+uTzcZWReXaVgWMl\\\\\\\/\\\\\\\/j003QK0igXun5AJm\\\\\\\/dR2Buiw8VFyHxcpbymytDf7w+7SvvfcoYCJ9AlrLGRjiE57\\\\\\\/dBd1SwEvG\\\\\\\/FKNl\\\\\\\/aCK4f0Sho8nXP1I3OYmLKZiJT1xlVzzIFlNmyRNXjiwdINYhn3mvf0NVAHQLqAxDbbFL2zjobPE06A26eKNwN19qctyyTgfjTjzAQ0\\\\\\\/ONCo1CyS1nQ8J8GPOtAO0DpwNBO3dynjzav9ECd4pZGTbtX7JUrZgSntDmDFb0\\\\\\\/p7omciYaDbQD1xpBSZFMNpoyNDglOiO3IR7xc07saCF57O4IPFwvAAIdWauXwNkpGifNWlvoWLCTHJLeIpdx1jIQiLKbFi8guAP\\\\\\\/wD118mYCfOJgjBT70RS\\\\\\\/xvQyaZTYvSSyhRdLJs9unvDxSNMoBxG1cWFOnyWisVzwjVj41R3UZXXWrSHy86bBhgdy29wuoo33G3W6J4pYwQklPwxnKrSzgZUlgpz6mWJmMc4aDkv\\\\\\\/hir7hROow91H+jyZgKBubOYKS0aJ99SvSCf+bRiTJbRgweiCoiynT5Y+gIcSRG5pigY3hBB1BC9yjDDunXsqbfi4dsLb35VIKDrzRwsucuTc+pt533EJ24dJ0+rdXq0JMP0JWHHeje9SGJX62\\\\\\\/elgRFCD5wpdKJ01dAzQgTRL9lFPoUljaEXQ3IgOoB+B7jWNVD57dwnJSTjKVWfOXO153wU66rbat6\\\\\\\/o7fOinKoTNAQBA6nP8JzdTPmXjmQQP+b\\\\\\\/rWVhjovLlRNk3jhCe\\\\\\\/ZZS25w0e\\\\\\\/Vaqrkg\\\\\\\/q9eGnswWzuhV2Lwx4+f0nOmhZHtWZCj9J9q6EwfisjLPI8\\\\\\\/Asso0804kVB1m4PSzwbIqBg4Aega06eppbYawlO6S\\\\\\\/Gx8Ku1uI5lJD6ndzvslVMJvp94DEcl50VS29Xf3AV7ZJn4JKHA0cJnvcqTh1Eyx9QAhLHd8VV3MotcaoVH0+se7r8a8Mci1EW1IX9H5OldUedQoRwM9YgSYegz91PbM\\\\\\\/hYvKGDTseC1FjJ\\\\\\\/aSECDxkDpAp37wDLa+c4t+D+BSXVE5GRL+byYzZxmupBEGcmQcGditbzDgJEEVi7rE\\\\\\\/H8MaSf4LPaweOPLjiZQ4AnFQoa50Jde42QNGL7XReTBKbUfB6oWuWPzM\\\\\\\/ruQbxB74BaNXK024G7PD9fnh9NKNy3yCBZh0jWyZqaD3M+r7SqtkFF1CDPb62eevH7lQ9NZoGew0+M1weejzEPsZW6Ly2leHLHgijXOzUoB8Bc\\\\\\\/RYRaExy4aW9cjw6oslg6aKgzZo9s8Rihgwk14wqXqP0+zPHhAEWlfBtabO8qmpBpcUAiW\\\\\\\/kMO\\\\\\\/iHmqe\\\\\\\/izXCKMDyHl0+fjYYCJDey48VZ6DfP2FCXkMmGeiMG\\\\\\\/0f30jk3cvJ26Q96+lbcO3dBd7MEgK+\\\\\\\/gbtlamzSwqEpqUS0ZlzOPJEuBXPjItg8hZv33e4ZYDFCqZMPcb7dSHSCmQSj41S7KjXlWMgEmlwBMtvO\\\\\\\/Npi\\\\\\\/yY6unjA+I2xBt738t9cf+ltziO8q\\\\\\\/k+DK1iVGMVOaMEyXJSl+tu\\\\\\\/2hlAoEuIigyT1CZ2jDQm6xUPIioSQ3F1rDXwPaQW8qkBiIxuHGbCQdFrEMTtslXBrykSbDgRFp4t5l7CBKeg54JdXN8xty6vbngnkjT+fmz9tZGb3CmqVFraWpcJMZiK4MP6tSBfGldMoyT18bNQNUoI=\\\u0022,\\\u0022iv\\\u0022:\\\u0022mrUQfXEsPdgnY4VUolRC4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bbe59b1c244ffd382b90ee9936980a0e372d4aae3726117f0e85898df6d1a088\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495851 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-software-and-data-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/933" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JyIfNCZtADdMlHmZynWeE2Li59KjYJ3TnFDZSamglvw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/933" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fWhtK8PgHAdJ9NuccfiJy1MvslItyh4YZM3yRTjKuBg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/925" title="An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/945" title="CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495851" id="node495851" data-pisa="ggg;10/3/933" data-pisa-master="ggg;g3.119.400898" data-apath="/ggg/10/3/933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Interactively AUDIT Your Growth Curves with a Suite of R Packages</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9336-226X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicolas P. J. Coutin</span>, <span class="highwire-citation-author" data-delta="1">Guri Giaever</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Corey Nislow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">933-943;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400898 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495851" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas P. J. Coutin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biology, Faculty of Medicine, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BP.%2BJ.%2BCoutin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coutin%20NP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BP.%2BJ.%2BCoutin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9336-226X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicolas P. J. Coutin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guri Giaever </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver, B.C., Canada, V6T 1Z3</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guri%2BGiaever%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giaever%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuri%2BGiaever%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Corey Nislow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biology, Faculty of Medicine, and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver, B.C., Canada, V6T 1Z3</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corey%2BNislow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nislow%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorey%2BNislow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Corey Nislow</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:corey.nislow@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/3/933" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495851" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/933.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495851" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/933.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495851" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/3/933.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495851" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Bottlenecks often occur during data analysis when studying microbial growth in liquid culture at large scale. A researcher can collect thousands of growth curves, repeated measures of a microbial liquid culture, at once in multiple micro titer plates by purpose-built robotic instruments. However, it can be difficult and time-consuming to inspect and analyze these data. This is especially true for researchers without programming experience. To enable this researcher, we created and describe an interactive application: Automated Usher for Data Inspection and Tidying (AUDIT). It allows the user to analyze growth curve data generated from one or more runs each with one or more micro titer plates alongside their experimental design. AUDIT covers input, pre-processing, summarizing, visual exploration and output. Compared to previously available tools AUDIT handles more data, provides live previews and is built from individually re-usable pieces distributed as R packages.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/growth-curve" class="hw-term hw-article-keyword hw-article-keyword-growth-curve" rel="nofollow">growth curve</a></li><li class="kwd"><a href="/keyword/chemical-genetics" class="hw-term hw-article-keyword hw-article-keyword-chemical-genetics" rel="nofollow">chemical genetics</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a href="/keyword/fitness" class="hw-term hw-article-keyword hw-article-keyword-fitness" rel="nofollow">fitness</a></li></ul><p id="p-3">The study of microbial growth in liquid culture is now applied at large scale for diverse applications. The quantitative study of the growth of microbial cultures dates back more than 70 years to the work of Monod and others (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Monod 1949</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Koch 1970</a>). Repeated measures of the density of cells in a suspension over time yield a growth curve. Researchers summarize these curves to a small set of parameters. These values are then compared to yield the relative fitness of the microbial system when probed. Robotic manipulation of Micro Titer Plates MTPs in purpose-built instruments has allowed studying thousands of conditions in parallel. For example, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Lee <em>et al.</em> (2014)</a> validated hits from a genomic approach in large scale drug discovery. Others performed High Throughput Screening HTS directly to study drug interactions and the saline response in yeast (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cokol <em>et al.</em> 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Warringer and Blomberg 2003</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Warringer <em>et al.</em> 2003</a>).</p><p id="p-4">A limiting step in HTS is often data analysis. Specifically, manipulating, inspecting and tidying growth curve data across multiple MTPs and multiple runs can be tedious or impossible. Further, different preprocessing and summarization methods are appropriate to apply depending on the experiment and instrument setup (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Malo <em>et al.</em> 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Blomberg 2011</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Stevenson <em>et al.</em> 2016</a>). Custom requirements complicate data analysis in HTS. They also suggest that automated software pipelines may not be appropriate for all applications. The custom tooling already available for growth curve analysis highlights this fact (Table S1).</p><p id="p-5">We describe an Automated Usher for Data Inspection and Tidying (AUDIT) for growth curves. The AUDIT application allows the user to analyze growth curve data, joined with their experimental design, for one or more MTPs across one or more runs. AUDIT guides the user through data input, preprocessing, summarization, visual exploration and output. AUDIT automatically tidies raw experimental data in different formats and joins experimental design data. This interactive web application runs locally or over the web. We wrote it in the shiny R and Javascript (JS) framework (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chang <em>et al.</em> 2019</a>). This application’s code relies on independently written modules and is open-source. AUDIT’s design enables modification and re-use.</p><p id="p-6">To demonstrate AUDIT, we study the effect of two chemical probes on the growth of <em>Saccharomyces cerevisiae</em> reference and gene deletion strains. The complete source code for AUDIT, along with example data, is freely distributed as an R package for easy installation and is available online at <a href="https://github.com/npjc/audit">https://github.com/npjc/audit</a>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Reading and tidying raw input</h3><p id="p-7">AUDIT considers two types of input files. Measures files contain the raw repeated measures data, usually produced directly by the growth instrument’s built-in software. The measure is the property being measured about each well (<em>e.g.</em>, Optical Density (OD), Absorbance at 620 nanoMeters (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-1.gif"/></noscript></span></span>)) at repeated time intervals. Design files contain other information that may vary across wells and makes up the experiment design (<em>e.g.</em>, strain, drug, media). These files are usually produced by the researcher. These files are usually produced manually in a spreadsheet application (<em>e.g.</em>, plater format (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Hughes 2016</a>)).</p><p id="p-8">AUDIT will accept input files in six formats. Raw measures data can be in Yeast Grower (YG) format (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Proctor <em>et al.</em> 2011</a>), Bioscreen C format (Oy Growth Curve), Corey Guri Twelve (CG-12) (S and P Robotics) format and Genome Processor One (GP1) (S and P Robotics) format. For a description of the CG-12 and GP1 high throughput microbial growth instruments see Figure S1. Design files can be in plater format (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Hughes 2016</a>) or in generic comma-separated values Comma Separated Values (CSV) format. File formats are automatically detected for each file using the regular expressions in Table S2. For details on how we defined file formats, see Defining file formats in Supplementary Materials.</p><p id="p-9">AUDIT reports runtime in seconds, minutes or hours based on user input. Some instruments report absolute time in addition to or instead of runtime (elapsed time). Where appropriate, reading modules can report datetime in ISO8601 format. Similarly, parsers can output the optional measure_type column to differentiate more than one measurement value taken at the same runtime interval.</p><div id="sec-3" class="subsection"><h4>Multiple files:</h4><p id="p-10">If multiple files are uploaded, AUDIT will join measures and design files together. To do this it applies a heuristic based on file type and number. If given two files, if one is a design file, join onto the measures file and create a single run. If given more than two files, and only one design file, join onto each of the measures files and create one run per measures file. If given more than two files and there are multiple design files, join measures and designs by partial name matching (<em>e.g.</em>, run1_design.csv would match to measures file run1.txt).</p></div><div id="sec-4" class="subsection"><h4>Built-in example data:</h4><p id="p-11">AUDIT provides example data files in each format and for the case study described herein. Users can load these files by clicking the links in input panel. AUDIT parses example data files identically to user uploaded content. They are also distributed as part of AUDIT’s source repository in the example-data directory (see Data Availability).</p></div></div><div id="sec-5" class="subsection"><h3>Preprocessing Measures</h3><p id="p-12">Preprocessing options in AUDIT allow for log transformation, smoothing, background correction and calibration. First, data can be log-transformed and then smoothed sequentially with a running median and mean filter. Median smoothing reduces high amplitude noise at single time points and mean smoothing reduces low amplitude high frequency noise (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Fernandez-Ricaud <em>et al.</em> 2016</a>). AUDIT can transform curves so that they always increase. This transformation is to reduce bias introduced by bubble formation, cell aggregation or drug precipitation (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Fernandez-Ricaud <em>et al.</em> 2016</a>). Lastly, because OD measurements increase non-linearly with cell densities (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Stevenson <em>et al.</em> 2016</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Fernandez-Ricaud <em>et al.</em> 2016</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Blomberg 2011</a>), an arbitrary calibration function can be applied.</p><div id="sec-6" class="subsection"><h4>Log transformation and smoothing:</h4><p id="p-13">Sigmoid growth models assume that the measured signal has been log-ratio transformed according to: <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-2.gif"/></noscript></span></span>, where N is the current number of organisms and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-3.gif"/></noscript></span></span> is the initial number of organisms (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Zwietering <em>et al.</em> 1990</a>). To enable this kind of transformation generally, raw measures (y) may be log-ratio transformed: <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-4.gif"/></noscript></span></span> . We use the minimum measure (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-5.gif"/></noscript></span></span>) instead of the first measure (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-6.gif"/></noscript></span></span>) in case of noisy initial measurements. AUDIT can smooth growth curves with a running median filter followed by running mean smoothing (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Fernandez-Ricaud <em>et al.</em> 2016</a>). The width of each filter can be any odd integer less than the length of the input values. Finally, it is possible to enforce that the transformed measures be increasing. Here, the software tests if each measure is equal or larger than the previous value. If this is not the case, the current measure is replaced with the previous value against which it was compared.</p></div><div id="sec-7" class="subsection"><h4>Background correction and calibration:</h4><p id="p-14">Background correction and calibration in AUDIT use functional interfaces. The user inputs a formula into each text box and AUDIT applies it each well. The variable y represents the incoming values, therefore a calibration function y would output the incoming values unchanged. This flexible interface makes it possible to interactively execute a wide variety of methods.</p><p id="p-15">To increase the specificity of the measured value to the intended signal the user can perform background subtraction. For example, to subtract a fixed background value: <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-7.gif"/></noscript></span></span>. Dynamic background subtraction is also possible. Specifying <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-8.gif"/></noscript></span></span> will remove the minimum measure from each well. Alternatively, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-9.gif"/></noscript></span></span> will remove the mean of the first 5 values.</p><p id="p-16">To correct for a non-linear relationship between the observed (measured) signal and the actual (true) signal, the user can provide a calibration function. Users can perform a calibration to relate the absorbance or OD measured by the spectrophotometer directly to the cell number or dry weight (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Pringle and Mor 1975</a>). Table S3 provides examples of how to input a calibration function in AUDIT. We recommend empirically determining a calibration function for each experimental setup. Figure S2 gives an example. For more details see Calibration procedures in Supplementary Materials and <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Stevenson <em>et al.</em> (2016)</a>.</p></div></div><div id="sec-8" class="subsection"><h3>Summarizing Growth Curves</h3><div id="sec-9" class="subsection"><h4>Smooth spline interpolation and self-starting model fits:</h4><p id="p-17">After preprocessing, the user chooses one of five methods to estimate summary metrics. These are smooth spline (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">R Core Team 2019</a>), Gompertz (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Zwietering <em>et al.</em> 1990</a>), logistic (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Zwietering <em>et al.</em> 1990</a>), Richards (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Zwietering <em>et al.</em> 1990</a>) and manual (see Arbitrary model specification via manual mode). AUDIT fits each curve via non-linear (weighted) least-squares estimation. For a visual comparison of these methods on the same data see Figure S4.</p><p id="p-18">After fitting, the software computes general summary statistics for each curve (Table S4, Figure S3). It estimates the maximum measure value reached during the assay and the time at which it (first) occurs: <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-10.gif"/></noscript></span></span> . From the <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-11.gif"/></noscript></span></span> curve, we take the maximum to yield the maximum rate: <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-12.gif"/></noscript></span></span> . Similarly, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-13.gif"/></noscript></span></span> is the point on the growth curve where the slope is at its (first) global maximum. We also estimate the x- and y-intercepts for the maximum slope line: <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-14.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-15.gif"/></noscript></span></span> . Last, we compute the area under the curve by integrating over the range of the curve fit with respect to time: <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-16.gif"/></noscript></span></span> . We refer to this value as fit_int in AUDIT.</p><p id="p-19">The application will also estimate model specific coefficients, via non-linear least-squares, for each curve. For example, metrics for the built-in Gompertz, logistic, and Richards methods: the asymptotic growth limit (A), the maximum growth rate (μ) and the lag time (λ). In an alternative implementation, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Kahm <em>et al.</em> (2010)</a> compared the performance of model and model-free methods on example data. They concluded that a spline fit produced more accurate estimates of the characteristic growth parameters and that model fits may lead to unreliable predictions. For the original derivation of A, μ and λ see <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Zwietering <em>et al.</em> (1990)</a>.</p><p id="p-20">The growr module that implements these methods provides additional functions not available interactively. It is possible to define and estimate additional user-defined metrics at the R console. For example, we implemented an additional metric: the minimum doubling time (Figure S5). See Extending AUDIT with new metrics in Supplementary Materials for details.</p></div><div id="sec-10" class="subsection"><h4>Arbitrary model specification via manual mode:</h4><p id="p-21">AUDIT provides an extensible model interface that makes it possible to interactively specify and fit arbitrary models. To engage this mode the user selects the ‘manual’ estimation method in the summarize panel. They can then specify any model that is expressible as an R formula. Specifically, taking the form ‘y ?x’, where y is the preprocessed measures data and x is the runtime. AUDIT generates model fits and estimates coefficients as above, but it requires a user-supplied initial estimate of each model parameter. To do this, the user supplies starting values as named comma-separated values in the form: ‘a1 = 0.05,...’. As an example, Figure S6 shows the output of manually specifying the Brody growth model (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-17.gif"/></noscript></span></span>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brody 1966</a>). The formula for this model is ‘y ?a - (a - w0) * exp(-k * x)’ and the starting estimates chosen are ‘a = 1,w0 = 0.5, k = 0.00003’. See Extending AUDIT with new models in Supplementary Materials for implementation details.</p></div><div id="sec-11" class="subsection"><h4>Curve quality and model goodness of fit:</h4><p id="p-22">To summarize the quality of curve fits AUDIT computes common model statistics. R’s built-in smooth.spline() and nls() methods provide these quality metrics. In the case of cubic (smooth) splines: the smoothing parameter (spar), the equivalent degrees of freedom (df), the underlying criterion minimized (crit), the (weighted) residual sum of squares (penalized criterion) and cross-validation score (cv.crit). For all other models, including custom models fits: the square root of the residual variance (sigma), whether the fit successfully converged (isConv), the achieved convergence tolerance (finTol), the data’s log-likelihood under the model (logLik), the Akaike Information Criterion (AIC), the Bayesian Information Criterion (BIC), the deviance and the residual degrees of freedom (df.residual).</p></div></div><div id="sec-12" class="subsection"><h3>Exploring Results</h3><div id="sec-13" class="subsection"><h4>Experiment-at-a-glance:</h4><p id="p-23">To visualize a summary metric across an experiment, the user first chooses which metric to display with a drop-down menu in the explore panel. The software then plots each well in each plate of the experiment as a rectangle and maps data to plate or well properties. We call this an MTP view plot. AUDIT’s experiment-at-a-glance fills each well with a color representing the value of the summary metric. The viridis package provides a color scale where the user perceives equal steps in data as equal steps in color. This type of scale enables accurate visual comparison (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Garnier 2018</a>).</p><p id="p-24">To export this graphic as a Portable Network Graphics (PNG) file, the user can right-click on the image and select the ‘Save Image As...’ item from the contextual menu. All other images produced by AUDIT inherit this behavior.</p></div><div id="sec-14" class="subsection"><h4>Applying reference/target groups:</h4><p id="p-25">To define reference and target wells, separate them with a ‘dash and greater than’ arrow (-&gt;). Each side can contain multiple sets of wells in a comma separated list. Rectangular selection is also possible by using a colon (:) to define the range of the selection. Finally, to apply the grouping to a specific plate, prefix the grouping by its name and an exclamation point (!).</p><p id="p-26">Users can apply one or more groups with the shorthand syntax [plate name]![ref wells]-&gt;[target wells]. For example 1!A01-&gt;A01:H12 will make well A01 the reference for all 96 wells of a MTP named 1 with 8 rows and 12 columns. Omitting the plate name (A01-&gt;A01:H12) applies the grouping to each plate in the run. To apply multiple groups, the user specifies one per line. Groups can use wells multiple times and they are not mutually exclusive. Currently, all groups must apply to plates of the same dimensions.</p><p id="p-27">After parsing the grouping shorthand, AUDIT adds two additional columns to the summarized data to encode the grouping information into tabular format. The group column contains the group name while the is_ref column indicates which wells in that group are the reference wells with TRUE or FALSE.</p></div></div><div id="sec-15" class="subsection"><h3>Data output</h3><p id="p-28">AUDIT can output tidied raw and preprocessed measures data, summary metrics data, and model quality statistics. The user can download these CSV files from the output pane. The outputted measures data contains all experimental design data and/or grouping joined during the session. Outputted results are consistent for a given set of input data, preprocessing parameters and summary method.</p></div><div id="sec-16" class="subsection"><h3>Data availability</h3><p id="p-29">AUDIT is freely available at <a href="https://nicolascoutin.shinyapps.io/audit/">https://nicolascoutin.shinyapps.io/audit/</a> and is distributed as an R package from <a href="https://github.com/npjc/audit">https://github.com/npjc/audit</a>. Step-by-step installation instructions are available in the repository’s README.md document.</p><p id="p-30">AUDIT is an interactive interface to multiple, independently useful, modules. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> describes and links to their freely available source code and documentation.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495933/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495933/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description and Repositories for AUDIT Modules</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">To make it easy to test AUDIT, we distribute example data as part of the application’s source directory: <a href="https://github.com/npjc/audit/tree/master/inst/audit-main/example-data">https://github.com/npjc/audit/tree/master/inst/audit-main/example-data</a>. The user can also load these files, without downloading them first, within the application’s input panel. Output data produced from the movie and figures presented herein are also distributed with AUDIT’s source: <a href="https://github.com/npjc/audit/tree/master/inst/audit-main/example-data-output">https://github.com/npjc/audit/tree/master/inst/audit-main/example-data-output</a>.</p><p id="p-33">Additionally, the following supplementary files have been deposited at figshare <a href="https://doi.org/10.25387/g3.11691870">https://doi.org/10.25387/g3.11691870</a>:</p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-34">Movie S1.avi: Movie file demonstrating AUDIT session matching figures in main text.</p></li><li id="list-item-2"><p id="p-35">supplementary-materials.pdf: Contains all supplementary tables, figures, listings and text referenced but not presented in the main article.</p></li></ul></div></div><div class="section results" id="sec-17"><h2 class="">Results</h2><div id="sec-18" class="subsection"><h3>Overview of AUDIT</h3><p id="p-36">AUDIT is an interactive web application that provides a high-level interface for large-scale growth curve analysis (Movie S1). Researchers do not need programming experience to use it. AUDIT guides the user through data input, preprocessing, summarization, visual exploration and data output. We wrote AUDIT in the Shiny framework using the R and JS programming languages (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Chang <em>et al.</em> 2019</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">R Core Team 2019</a>). Researchers can host the application online or run it locally on their machine. Making the application available over an internal or public website makes for the easiest end-use but requires maintaining a dedicated Shiny server. In addition, an administrator needs to manage user permissions to permit uploading and downloading files from the server. Installing AUDIT locally, on the same machine where it will be used, removes the need for a network connection. AUDIT will be more responsive locally, because it will not have to wait on data crossing the network. In our testing AUDIT is fast enough to enable interactive use: with each step completing in fractions of a second.</p><div id="sec-19" class="subsection"><h4>Architecture and conventions:</h4><p id="p-37">AUDIT is a high-level interface to individual software modules (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). This modular design makes it easy to reuse a subset or superset of the functionality provided. The software modules, individually available as R packages, each serve a different purpose (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The readbioscreen, readyg, readcg12, readgp1 implement functions that read and tidy raw repeated measures data from Bioscreen, YG (ACCESS), CG-12 and GP1 instruments, respectively. The growr package provides functions related to the preprocessing, modeling and summarization of growth curve data. Finally, the mtpview package provides functionality to visualize growth curves, summary metrics and experimental variables within one or more MTPs by extending the plotting library ggplot2 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Wickham 2016</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/933/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview Architecture and Conventions. A. AUDIT is an interactive, higher level, interface (i) atop a set of software modules/packages (ii) written and distributed via the underlying software platforms in R and JS (iii). The individual function groups (ii) are available as individual packages to enable re-use outside of the context of AUDIT. B. The stratification of individual measures across an experiment.  contains the set of all runs from 1 to i (). Each Run contains the set of all plates from 1 to j (). Each plate contains the set of all wells from 1 to k (). Each well contains the set of measures from 1 to l ()." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491042873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Overview Architecture and Conventions. A. AUDIT is an interactive, higher level, interface (i) atop a set of software modules/packages (ii) written and distributed via the underlying software platforms in R and JS (iii). The individual function groups (ii) are available as individual packages to enable re-use outside of the context of AUDIT. B. The stratification of individual measures across an experiment. &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-18&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; contains the set of all runs from 1 to &lt;em&gt;i&lt;/em&gt; (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-19&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Each Run contains the set of all plates from 1 to &lt;em&gt;j&lt;/em&gt; (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-20&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Each plate contains the set of all wells from 1 to &lt;em&gt;k&lt;/em&gt; (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-21&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). Each well contains the set of measures from 1 to &lt;em&gt;l&lt;/em&gt; (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-22&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/933/F1.medium.gif" width="299" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/3/933/F1.medium.gif" width="299" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/933/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/933/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495953" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-38" class="first-child">Overview Architecture and Conventions. A. AUDIT is an interactive, higher level, interface (i) atop a set of software modules/packages (ii) written and distributed via the underlying software platforms in R and JS (iii). The individual function groups (ii) are available as individual packages to enable re-use outside of the context of AUDIT. B. The stratification of individual measures across an experiment. <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/F1/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/F1/embed/mml-math-18.gif"/></noscript></span></span> contains the set of all runs from 1 to <em>i</em> (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/F1/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/F1/embed/mml-math-19.gif"/></noscript></span></span>). Each Run contains the set of all plates from 1 to <em>j</em> (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/F1/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/F1/embed/mml-math-20.gif"/></noscript></span></span>). Each plate contains the set of all wells from 1 to <em>k</em> (<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/F1/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/F1/embed/mml-math-21.gif"/></noscript></span></span>). Each well contains the set of measures from 1 to <em>l</em> (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/F1/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/F1/embed/mml-math-22.gif"/></noscript></span></span>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">AUDIT defines a convention for grouping repeated measures across an experiment (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). An experiment contains one or more runs. Each run contains one or more MTPs with its own set of wells. Each well has one or more measures. Therefore, we define each experiment as a set of observations each belonging to precisely one run, plate and well. This convention makes the manipulation and output of data consistent across a variety of input sources.</p></div><div id="sec-20" class="subsection"><h4>Input: Automatic parsing and merging of measures data with experimental design files:</h4><p id="p-40">Uploaded files are automatically parsed and previewed in AUDIT (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Measures files contain repeated measures (growth curve) data. Design files describe the distribution of experimental variables across the wells of MTPs in one or more runs. AUDIT consistently tidies varied input data to the same tabular format from one or more files (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). AUDIT records each measure in its own row with columns for the run, plate, well, runtime and measure (the value of the measurement, unit varies). Each measures file is automatically joined to one or more matching experimental design files, if provided. Design files contain the value of experiment variables for each run, plate and well. For details on the accepted formats, their specifications and joining rules see Reading and tidying raw input in Methods. The automatic parsing and merging of measures data and experimental design files makes for easier downstream analysis, distribution and archiving. This maximizes the chance that said data can be usefully re-analyzed to improve upon the findings and/or methods in the future.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/933/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Automatic parsing and tidying of design and measures files uploaded to AUDIT. Screenshot of application&#x2019;s input panel. Users upload one or more files via the file browser (arrow, top-left). Most inputs in AUDIT have a clickable &#x2018;I&#x2019; that, when clicked, will display modal help box with additional information (arrow, second from top-left). After upload, files are automatically parsed and joined into runs. AUDIT logs parsing and joining actions for user review (arrow, bottom left). Alternatively, the user can choose example data to load by selecting one of the links provided. Users can choose seconds, minutes or hours for runtime units. An experimental summary table displays the resulting run(s) (arrow). For each run, the number of plates (n_plates), the number of wells per plate (n_wells) and the number of measures (n_measures) are reported. Selecting one or more rows of the experiment summary table will output a line plot for each well in the MTP&#x2019;s physical layout. The limits of the x and y scales are per plate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491042873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Automatic parsing and tidying of design and measures files uploaded to AUDIT. Screenshot of application&#x2019;s input panel. Users upload one or more files via the file browser (arrow, top-left). Most inputs in AUDIT have a clickable &#x2018;I&#x2019; that, when clicked, will display modal help box with additional information (arrow, second from top-left). After upload, files are automatically parsed and joined into runs. AUDIT logs parsing and joining actions for user review (arrow, bottom left). Alternatively, the user can choose example data to load by selecting one of the links provided. Users can choose seconds, minutes or hours for runtime units. An experimental summary table displays the resulting run(s) (arrow). For each run, the number of plates (n_plates), the number of wells per plate (n_wells) and the number of measures (n_measures) are reported. Selecting one or more rows of the experiment summary table will output a line plot for each well in the MTP&#x2019;s physical layout. The limits of the x and y scales are per plate.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/933/F2.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/3/933/F2.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/933/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/933/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/496000" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-41" class="first-child">Automatic parsing and tidying of design and measures files uploaded to AUDIT. Screenshot of application’s input panel. Users upload one or more files via the file browser (arrow, top-left). Most inputs in AUDIT have a clickable ‘I’ that, when clicked, will display modal help box with additional information (arrow, second from top-left). After upload, files are automatically parsed and joined into runs. AUDIT logs parsing and joining actions for user review (arrow, bottom left). Alternatively, the user can choose example data to load by selecting one of the links provided. Users can choose seconds, minutes or hours for runtime units. An experimental summary table displays the resulting run(s) (arrow). For each run, the number of plates (n_plates), the number of wells per plate (n_wells) and the number of measures (n_measures) are reported. Selecting one or more rows of the experiment summary table will output a line plot for each well in the MTP’s physical layout. The limits of the x and y scales are per plate.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/933/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="AUDIT reading modules transform varied raw input to consistent output. Examples of plain-text rendering of Bioscreen (top left), YG (top right), CG-12 (middle left) and GP1 (middle right) raw input files parsed to common tabular form. The diagram highlights optional fields with a question mark suffix. Following parsing of raw measures AUDIT joins experimental design data to the parsed measures, if available. Design files may be in generic CSV files or in plater format." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491042873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;AUDIT reading modules transform varied raw input to consistent output. Examples of plain-text rendering of Bioscreen (top left), YG (top right), CG-12 (middle left) and GP1 (middle right) raw input files parsed to common tabular form. The diagram highlights optional fields with a question mark suffix. Following parsing of raw measures AUDIT joins experimental design data to the parsed measures, if available. Design files may be in generic CSV files or in plater format.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/933/F3.medium.gif" width="394" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/3/933/F3.medium.gif" width="394" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/933/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/933/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495882" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-42" class="first-child">AUDIT reading modules transform varied raw input to consistent output. Examples of plain-text rendering of Bioscreen (top left), YG (top right), CG-12 (middle left) and GP1 (middle right) raw input files parsed to common tabular form. The diagram highlights optional fields with a question mark suffix. Following parsing of raw measures AUDIT joins experimental design data to the parsed measures, if available. Design files may be in generic CSV files or in plater format.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h4>Preprocess: Live preview provides immediate feedback on flexible user options:</h4><p id="p-43">AUDIT provides a live line plot preview to compare raw and preprocessed measures for any number of wells selected in the experiment (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The user specificies the preprocessing routine with interactive selectors. First, the base for log ratio transformation, if applied. Second, the size of the windows when performing running median and mean smoothing. Next, if the data should be transformed to always be increasing. Last, background subtraction and calibration are possible with the formula execution interface provided. AUDIT binds incoming measures data for each well to the variable y. Specifying <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-23.gif"/></noscript></span></span> therefore translates each growth curve to start at <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-24.gif"/></noscript></span></span> by subtracting the minimum value. Other background correction and calibration methods can be specified, for more details and examples see Background correction and calibration in Methods. Live preview allows the researcher to rapidly iterate and compare preprocessing options before applying them to the entire experiment.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/933/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Live preview and flexible specification of varied preprocessing methods. Screenshot of preprocess panel. AUDIT makes six independent preprocessing options available (arrow, top-left). The Background Subtraction and Calibration input fields support arbitrary formula (code) execution (arrow, middle-right). The well preview selector table (arrow, top-right) contains data from selected runs in the experiment. When clicked, the &#x2018;Preprocess Experiment&#x2019; button applies the preprocessing to the whole experiment (arrow, middle-left). To display a live plot preview, users select rows in the table. Then, AUDIT plots each well&#x2019;s raw (solid line) and preprocessed measures (dashed line) (arrow, bottom-right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491042873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Live preview and flexible specification of varied preprocessing methods. Screenshot of preprocess panel. AUDIT makes six independent preprocessing options available (arrow, top-left). The Background Subtraction and Calibration input fields support arbitrary formula (code) execution (arrow, middle-right). The well preview selector table (arrow, top-right) contains data from selected runs in the experiment. When clicked, the &#x2018;Preprocess Experiment&#x2019; button applies the preprocessing to the whole experiment (arrow, middle-left). To display a live plot preview, users select rows in the table. Then, AUDIT plots each well&#x2019;s raw (solid line) and preprocessed measures (dashed line) (arrow, bottom-right).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/933/F4.medium.gif" width="440" height="289"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/3/933/F4.medium.gif" width="440" height="289"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/933/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/933/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495971" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">Live preview and flexible specification of varied preprocessing methods. Screenshot of preprocess panel. AUDIT makes six independent preprocessing options available (arrow, top-left). The Background Subtraction and Calibration input fields support arbitrary formula (code) execution (arrow, middle-right). The well preview selector table (arrow, top-right) contains data from selected runs in the experiment. When clicked, the ‘Preprocess Experiment’ button applies the preprocessing to the whole experiment (arrow, middle-left). To display a live plot preview, users select rows in the table. Then, AUDIT plots each well’s raw (solid line) and preprocessed measures (dashed line) (arrow, bottom-right).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h4>Summarize: Fit a smooth function to measures and estimate growth curve metrics with a live preview:</h4><p id="p-45">AUDIT enables arbitrary model fitting to preprocessed growth curves with an interactive live preview (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). It provides five estimation methods, including a ‘model-free’ cubic spline (default) and a ‘manual’ mode which enables the user to specify any growth model expressible with R’s built in formula method. From the fitted data, AUDIT computes summary metrics and estimates each model coefficient. For details on the extensible model interface and the summary metrics computed see Smooth spline interpolation and self-starting model fits in Methods. By examining a live preview of the fitted curve and the residual error, users can interactively examine the performance of different fitting procedures. As an example, we fit a Brody growth model (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Brody 1966</a>) (originally used to model economic expansion) to the example data (Figure S6).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/933/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summarize growth curves by curve fit and group wells into reference and target sets. Screenshot of summarize panel in manual mode. AUDIT enables interactive specification of arbitrary growth models. Users specify a model via R&#x2019;s formula interface. AUDIT estimates model coefficients for each curve, using non-linear least-squares optimization, from user-supplied initial estimates. When clicked, the &#x2018;Summarize Experiment&#x2019; button applies the preprocessing to the whole experiment (arrow, middle-left). The well preview selector table (arrow, top-right) contains data from selected runs in the experiment. Selecting rows in the table, will cause a live line plot preview of each curve fit (solid line) atop the underlying measures data (points). Fit residual error is also plotted (red area segment). The application displays a tabular preview of the estimated summary metrics, including model specific coefficients, and the curve quality metrics." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491042873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summarize growth curves by curve fit and group wells into reference and target sets. Screenshot of summarize panel in manual mode. AUDIT enables interactive specification of arbitrary growth models. Users specify a model via R&#x2019;s formula interface. AUDIT estimates model coefficients for each curve, using non-linear least-squares optimization, from user-supplied initial estimates. When clicked, the &#x2018;Summarize Experiment&#x2019; button applies the preprocessing to the whole experiment (arrow, middle-left). The well preview selector table (arrow, top-right) contains data from selected runs in the experiment. Selecting rows in the table, will cause a live line plot preview of each curve fit (solid line) atop the underlying measures data (points). Fit residual error is also plotted (red area segment). The application displays a tabular preview of the estimated summary metrics, including model specific coefficients, and the curve quality metrics.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/933/F5.medium.gif" width="440" height="382"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/3/933/F5.medium.gif" width="440" height="382"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/933/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/933/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495841" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-46" class="first-child">Summarize growth curves by curve fit and group wells into reference and target sets. Screenshot of summarize panel in manual mode. AUDIT enables interactive specification of arbitrary growth models. Users specify a model via R’s formula interface. AUDIT estimates model coefficients for each curve, using non-linear least-squares optimization, from user-supplied initial estimates. When clicked, the ‘Summarize Experiment’ button applies the preprocessing to the whole experiment (arrow, middle-left). The well preview selector table (arrow, top-right) contains data from selected runs in the experiment. Selecting rows in the table, will cause a live line plot preview of each curve fit (solid line) atop the underlying measures data (points). Fit residual error is also plotted (red area segment). The application displays a tabular preview of the estimated summary metrics, including model specific coefficients, and the curve quality metrics.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-23" class="subsection"><h4>Explore: Inspect the distribution of summary metrics across an experiment and within groups:</h4><p id="p-47">Estimated metrics can be visualized across an entire experiment with mtpview plots. <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows an annotated screenshot of the explore panel. mtpview powers the experiment-at-a-glance functionality, which scales to hundreds of plates. Figure S7 shows a 144 plate experiment at a glance. This summary view provides a qualitative comparison of wells, plates and runs.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/933/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Explore the distribution of summary metrics across an experiment and by group. Screenshot of AUDIT&#x2019;s explore panel. The researcher chooses a summary metric and AUDIT creates a color map across the wells in an MTP view plot (arrow, top). Users can specify Reference-target groups interactively. AUDIT will generate line plots of the fitted growth curves along with a summary table of per-group means for each numeric summary metric computed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491042873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Explore the distribution of summary metrics across an experiment and by group. Screenshot of AUDIT&#x2019;s explore panel. The researcher chooses a summary metric and AUDIT creates a color map across the wells in an MTP view plot (arrow, top). Users can specify Reference-target groups interactively. AUDIT will generate line plots of the fitted growth curves along with a summary table of per-group means for each numeric summary metric computed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/933/F6.medium.gif" width="387" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/3/933/F6.medium.gif" width="387" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/933/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/933/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495951" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-48" class="first-child">Explore the distribution of summary metrics across an experiment and by group. Screenshot of AUDIT’s explore panel. The researcher chooses a summary metric and AUDIT creates a color map across the wells in an MTP view plot (arrow, top). Users can specify Reference-target groups interactively. AUDIT will generate line plots of the fitted growth curves along with a summary table of per-group means for each numeric summary metric computed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">To enable within-plate comparisons and group wells in reference and target groups, AUDIT also provides an interactive shorthand. Users specify groups with ‘A01’ style notation in the following syntax: [plate name]![ref wells]-&gt;[target wells]. AUDIT automatically generates line plots of fitted measures for each group. The application also computes a table of per-group means for all numeric summary metrics. See Table S5 for examples and Applying reference/target groups in Methods for details.</p></div><div id="sec-24" class="subsection"><h4>Output: Download tidied and preprocessed measures and summary data as CSV files:</h4><p id="p-50">Measures, summary metrics, and quality statistics are available for download from AUDIT’s output panel (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). We chose this CSV format output for easy distribution and interoperability. These data are amenable to further analysis, for example filtering based on quality of fit, in any spreadsheet-type application.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/933/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Download tidied and preprocessed measures and summary data as CSV files. Screenshot of AUDIT&#x2019;s output panel. Three data files are available for download: Measures data, containing tidied raw and preprocessed measures (left), summary-level data containing estimated metrics and model components/coefficients (middle), and summary-level fit quality data (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491042873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Download tidied and preprocessed measures and summary data as CSV files. Screenshot of AUDIT&#x2019;s output panel. Three data files are available for download: Measures data, containing tidied raw and preprocessed measures (left), summary-level data containing estimated metrics and model components/coefficients (middle), and summary-level fit quality data (right).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/933/F7.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/10/3/933/F7.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/933/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/933/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495925" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-51" class="first-child">Download tidied and preprocessed measures and summary data as CSV files. Screenshot of AUDIT’s output panel. Three data files are available for download: Measures data, containing tidied raw and preprocessed measures (left), summary-level data containing estimated metrics and model components/coefficients (middle), and summary-level fit quality data (right).</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-25" class="subsection"><h3>Anticipated Results</h3><p id="p-52">To demonstrate AUDIT, we sourced a raw data file from our internal archive. We chose the first raw measures data file sampled for which we had information on the experimental design. In this experiment, we probed three Saccharomyces cerevisiae with each of two chemical compounds in triplicate. BY4743 and matched deletion strains <a href="https://identifiers.org/bioentitylink/SGD:S000003834?doi=10.1534/g3.119.400898">opi3</a>Δ and <a href="https://identifiers.org/bioentitylink/SGD:S000000379?doi=10.1534/g3.119.400898">swd3</a>Δ were treated with compounds identified as 5673113 (1) and 6772625 (2). An eight step two fold dilution covered a final concentration range from 50 to 0 micro Molar (uM). A GENios spectrophotometer (Tecan, Switzerland) measured Optical density at 595 nanoMeters (<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3/933/embed/mml-math-25.gif"/></noscript></span></span>) every 15 min for 94 cycles.</p><p id="p-53">Prior to the AUDIT session, we copied this raw measures file and renamed it cs1.csv. We also transferred the experimental design into plater format and saved the resulting file with a _design.csv suffix. We uploaded the resulting files to AUDIT and processed them as demonstrated in Movie S1 and in the annotated screenshots of each AUDIT panel: input (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), preprocess (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), summarize (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>) and output (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). We renamed the downloaded output files with the prefix cs1_output-.</p><p id="p-54">Further analysis is possible with additional functions in AUDIT’s individual modules. This functionality is not available interactively. It requires the user to call them in R, for example with the additional MTP view plots. First, visualizing the experimental design in place can speed the qualitative inspection (auditing) of growth curves (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A,B</a>). Second, overlaying duplicate growth curves from reference wells onto target wells, using opacity to differentiate reference from target curves (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>). We provide the code required to produce these plots in Listing S5 and Listing S6, respectively.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/933/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Case Study Overview. A. MTP view plot maps the values of the strain variable provided in the experimental design to the well fill color. A line plot of the measures data are plotted for each well. B. MTP view plot maps the drug identifier to the fill color and the dose in micromolar to the opacity of the rectangular wells. Line plots are according to the group specifications provided to the left of the plot. For each group, each reference curve is plotted in light, semi-transparent gray, in the target well while each the data for each target well is plotted in black. C. MTP view plot maps the values of the maximum growth rate reported in the summary metrics data. Key is provided on the right for each figure panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-491042873" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Case Study Overview. A. MTP view plot maps the values of the strain variable provided in the experimental design to the well fill color. A line plot of the measures data are plotted for each well. B. MTP view plot maps the drug identifier to the fill color and the dose in micromolar to the opacity of the rectangular wells. Line plots are according to the group specifications provided to the left of the plot. For each group, each reference curve is plotted in light, semi-transparent gray, in the target well while each the data for each target well is plotted in black. C. MTP view plot maps the values of the maximum growth rate reported in the summary metrics data. Key is provided on the right for each figure panel.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/933/F8.medium.gif" width="361" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/10/3/933/F8.medium.gif" width="361" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/933/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/933/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/496006" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-55" class="first-child">Case Study Overview. A. MTP view plot maps the values of the strain variable provided in the experimental design to the well fill color. A line plot of the measures data are plotted for each well. B. MTP view plot maps the drug identifier to the fill color and the dose in micromolar to the opacity of the rectangular wells. Line plots are according to the group specifications provided to the left of the plot. For each group, each reference curve is plotted in light, semi-transparent gray, in the target well while each the data for each target well is plotted in black. C. MTP view plot maps the values of the maximum growth rate reported in the summary metrics data. Key is provided on the right for each figure panel.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-26"><h2 class="">Discussion</h2><p id="p-56">Growth curve analysis can be a fast and cost-effective means of HTS, particularly when dealing with a large number of chemical probes. That said, this technique requires manipulating and inspecting data from hundreds of MTPs across dozens of runs in parallel. We present AUDIT (and the suite of R packages on which it is built) to make this possible across multiple instrument types.</p><p id="p-57">Abstracting an entire growth curve to a single value has advantages and limitations. In High Throughput (HT) experiments where it becomes impractical to inspect growth curves individually, metrics provide a compressed representation. Although these summary statistics can ease comparison and ranking across thousands of conditions, they are also lossy. They do not necessarily capture the underlying biology. For example, if the preprocessed value used to measure growth (<em>e.g.</em>, OD) does not perfectly relate to cellular concentration (or density), the metrics reported (<em>e.g.</em>, maximum growth rate) do not represent the population growth rates. Therefore, researchers should carefully preprocess their data for metrics to directly map to the biological system being studied. Nonetheless, a quantitative approach that summarizes growth curves to one or more metrics can be useful in HTS applications.</p><p id="p-58">We designed AUDIT to be extended. We therefore mark two areas for near term development. First, writing parsing modules for other data formats not currently part of AUDIT. Parsed and tidied data from custom or commercial instruments should adhere to AUDIT’s conventions for storing repeated measures experiment data. For this purpose, the output format should be plain-text human- and computer-parsable where possible. Second, writing new modules to help the researcher perform quality control within AUDIT. For example, integrating heuristics that display or check instrument-specific parameters, such as temperature control and spatial effects. We already include a basic temperature check module in the explore panel.</p><p id="p-59">We designed AUDIT’s underlying modules to be remixed. We highlight new directions for potential future developments. It is possible that better tracking of experimental design variables could enable automated modeling of the ‘history of the inoculum’ and perhaps minimize spurious repetitions. For example, If an experiment calls for a previously characterized inoculum, it could be omitted. Next, the integration of historical databases or archives could enable automated outlier detection and interactive browsing by collaborators or non-specialized researchers. Specifying the experimental design ahead of time and performing Just-in-time analysis during the run could replace post-hoc analysis and provide a valuable in-run diagnostic. This could enable direct instrument feedback. AUDIT could shorten, lengthen or repeat a run to meet a predefined clearance between the reference and target wells of the specified groups.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">We thank the members of the Giaever/Nislow lab for their feedback on the software and this manuscript. Work in Corey Nislow’s lab is supported as a Canada Research Chair.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-61">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11691870">https://doi.org/10.25387/g3.11691870</a>.</p></li><li class="fn" id="fn-3"><p id="p-62"><em>Communicating editor: D. Gresham</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-11-26"><span class="received-label">Received </span>November 26, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2020-01-02"><span class="accepted-label">Accepted </span>January 2, 2020.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Coutin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.1" data-doi="10.1016/j.copbio.2010.10.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Measuring growth rate in high-throughput growth phenotyping.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">94</span>–<span class="cit-lpage">102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.copbio.2010.10.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.stitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.aulast%253DBlomberg%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D94%26rft.epage%253D102%26rft.atitle%253DMeasuring%2Bgrowth%2Brate%2Bin%2Bhigh-throughput%2Bgrowth%2Bphenotyping.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2010.10.013%26rft_id%253Dinfo%253Apmid%252F21095113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2010.10.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21095113&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287840700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.2" data-doi="10.2307/1880584"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brody</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">A simplified growth model.</span> <abbr class="cit-jnl-abbrev">Q. J. Econ.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/1880584</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQ.%2BJ.%2BEcon.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1880584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1880584&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.3"><div class="cit-metadata unstructured">Chang, W., J. Cheng, J. Allaire, Y. Xie, and J. McPherson, 2019 shiny: Web Application Framework for R. R package version 1.3.2.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.4" data-doi="10.1038/msb.2011.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cokol</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. N.</span> <span class="cit-name-surname">Chua</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tasan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Mutlu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. B.</span> <span class="cit-name-surname">Weinstein</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Systematic exploration of synergistic drug pairs.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">544</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/msb.2011.71</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DCokol%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D544%26rft.epage%253D544%26rft.atitle%253DSystematic%2Bexploration%2Bof%2Bsynergistic%2Bdrug%2Bpairs%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.71%26rft_id%253Dinfo%253Apmid%252F22068327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS81NDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzkzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.5" data-doi="10.1186/s12859-016-1134-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Ricaud</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Kourtchenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zackrisson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Warringer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Blomberg</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">PRECOG: a tool for automated extraction and visualization of fitness components in microbial growth phenomics.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-1134-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D17%26rft.spage%253D249%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-1134-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-1134-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.6"><div class="cit-metadata unstructured">Garnier, S., 2018 viridis: Default Color Maps from ‘matplotlib’. R package version 0.5.1.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.7"><div class="cit-metadata unstructured">Hughes, S. M., 2016 plater: Read, tidy, and display data from microtiter plates. The Journal of Open Source Software 1.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kahm</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Hasenbrink</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lichtenberg-Fraté</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ludwig</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kschischo</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">grofit: Fitting Biological Growth Curves with R.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1021</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D33%26rft.spage%253D1021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.9" data-doi="10.1016/0003-2697(70)90174-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>,  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Turbidity measurements of bacterial cultures in some available commercial instruments.</span> <abbr class="cit-jnl-abbrev">Anal. Biochem.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">259</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0003-2697(70)90174-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DKoch%26rft.auinit1%253DA.%2BL.%26rft.volume%253D38%26rft.issue%253D1%26rft.spage%253D252%26rft.epage%253D259%26rft.atitle%253DTurbidity%2Bmeasurements%2Bof%2Bbacterial%2Bcultures%2Bin%2Bsome%2Bavailable%2Bcommercial%2Binstruments.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0003-2697%252870%252990174-0%26rft_id%253Dinfo%253Apmid%252F4920662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0003-2697(70)90174-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4920662&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1970H834700025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.10" data-doi="10.1126/science.1250217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. P. S.</span> <span class="cit-name-surname">Onge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Proctor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Wallace</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Nile</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mapping the Cellular Response to Small Molecules Using Chemogenomic Fitness Signatures.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">344</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">211</span>. erratum: Science 344(6186): 1255771.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1250217</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLee%26rft.auinit1%253DA.%2BY.%26rft.volume%253D344%26rft.issue%253D6180%26rft.spage%253D208%26rft.epage%253D211%26rft.atitle%253DMapping%2Bthe%2BCellular%2BResponse%2Bto%2BSmall%2BMolecules%2BUsing%2BChemogenomic%2BFitness%2BSignatures%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1250217%26rft_id%253Dinfo%253Apmid%252F24723613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDQvNjE4MC8yMDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzkzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.11" data-doi="10.1038/nbt1186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malo</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Hanley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Cerquozzi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pelletier</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nadon</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Statistical practice in high-throughput screening data analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">175</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt1186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMalo%26rft.auinit1%253DN.%26rft.volume%253D24%26rft.issue%253D2%26rft.spage%253D167%26rft.epage%253D175%26rft.atitle%253DStatistical%2Bpractice%2Bin%2Bhigh-throughput%2Bscreening%2Bdata%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1186%26rft_id%253Dinfo%253Apmid%252F16465162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16465162&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235232300032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.12" data-doi="10.1146/annurev.mi.03.100149.002103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monod</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-article-title">The Growth of Bacterial Cultures.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">394</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.mi.03.100149.002103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DMonod%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.mi.03.100149.002103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.mi.03.100149.002103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1949XS94800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.13" data-doi="10.1016/S0091-679X(08)60320-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pringle</span>,  <span class="cit-name-given-names">J. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-R.</span> <span class="cit-name-surname">Mor</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Methods for monitoring the growth of yeast cultures and for dealing with the clumping problem.</span> <abbr class="cit-jnl-abbrev">Methods Cell Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">168</span> <span class="cit-comment">Elsevier.</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0091-679X(08)60320-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bcell%2Bbiology%26rft.stitle%253DMethods%2BCell%2BBiol%26rft.aulast%253DPringle%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D11%26rft.spage%253D131%26rft.epage%253D168%26rft.atitle%253DMethods%2Bfor%2Bmonitoring%2Bthe%2Bgrowth%2Bof%2Byeast%2Bcultures%2Band%2Bfor%2Bdealing%2Bwith%2Bthe%2Bclumping%2Bproblem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0091-679X%252808%252960320-9%26rft_id%253Dinfo%253Apmid%252F1102845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0091-679X(08)60320-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1102845&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.933.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Proctor</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Urbanus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Fung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. F.</span> <span class="cit-name-surname">Jaramillo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Davis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Automated Cell: Compound and Environment Screening System (ACCESS) for Chemogenomic Screening</span>, pp. <span class="cit-fpage">239</span>–<span class="cit-lpage">269</span> in <span class="cit-source">Strain Engineering: Methods and Protocols</span>, <span class="cit-publ-name">Humana Press</span>, <span class="cit-publ-loc">Totowa, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.933.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.16" data-doi="10.1038/srep38828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevenson</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. F.</span> <span class="cit-name-surname">McVey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. B. N.</span> <span class="cit-name-surname">Clark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Swain</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Pilizota</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">General calibration of microbial growth in microplate readers.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">38828</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep38828</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D38828%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep38828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep38828&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.17" data-doi="10.1002/yea.931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Blomberg</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Automated screening in environmental arrays allows analysis of quantitative phenotypic profiles in saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.931</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D67%26rft.atitle%253DAutomated%2Bscreening%2Bin%2Benvironmental%2Barrays%2Ballows%2Banalysis%2Bof%2Bquantitative%2Bphenotypic%2Bprofiles%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.931%26rft_id%253Dinfo%253Apmid%252F12489126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12489126&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F933.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180219900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.18" data-doi="10.1073/pnas.2435976100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Ericson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Fernandez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Nerman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Blomberg</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">High-resolution yeast phenomics resolves different physiological features in the saline response.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">15724</span>–<span class="cit-lpage">15729</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.2435976100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2435976100%26rft_id%253Dinfo%253Apmid%252F14676322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzI2LzE1NzI0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85MzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.19"><div class="cit-metadata unstructured">Wickham, H., 2016 ggplot2: Elegant Graphics for Data Analysis. Springer-Verlag New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.933.20" data-doi="10.1111/j.1472-765X.2008.02537.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zwietering</span>,  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Jongenburger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. M.</span> <span class="cit-name-surname">Rombouts</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">van ’t Riet</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Modeling of the Bacterial Growth Curve.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1875</span>–<span class="cit-lpage">1881</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.56.6.1875-1881.1990</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DZwietering%26rft.auinit1%253DM.%2BH.%26rft.volume%253D56%26rft.issue%253D6%26rft.spage%253D1875%26rft.epage%253D1881%26rft.atitle%253DModeling%2Bof%2Bthe%2BBacterial%2BGrowth%2BCurve%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1472-765X.2008.02537.x%26rft_id%253Dinfo%253Apmid%252F16348228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjU2LzYvMTg3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzEwLzMvOTMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/3/933.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/925" title="An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/945" title="CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/3"><span>Volume 10 Issue 3, March 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/3.cover-source.jpg?itok=8jjPWDA4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/933?rss=1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5Ksqm_J8X3NKFLlu2RQdrRy8K-Xz0VbYcHzh_01ZwS8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/software-and-data-resources" class="" data-icon-position="" data-hide-link-title="0">Software and Data Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/3/933" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/3/933" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/3/933" class="active" data-icon-position="" data-hide-link-title="0">Interactively AUDIT Your Growth Curves with a Suite of R Packages</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495851" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9A89gL19AioUtRJQVRLUQBjQUXlJCqndMOjDf0MvfyA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/933.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/3/933" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-MbHzhlgcEoaPZK92L98UuHtAO0WBpmHG8br70aSFuRo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/3/933" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/933.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495851" id="node495851--2" data-pisa="ggg;10/3/933" data-pisa-master="ggg;g3.119.400898" data-apath="/ggg/10/3/933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Interactively AUDIT Your Growth Curves with a Suite of R Packages</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9336-226X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicolas P. J. Coutin</span>, <span class="highwire-citation-author" data-delta="1">Guri Giaever</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Corey Nislow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">933-943;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400898 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495851--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas P. J. Coutin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biology, Faculty of Medicine, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BP.%2BJ.%2BCoutin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coutin%20NP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BP.%2BJ.%2BCoutin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9336-226X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicolas P. J. Coutin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guri Giaever </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver, B.C., Canada, V6T 1Z3</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guri%2BGiaever%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giaever%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuri%2BGiaever%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Corey Nislow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biology, Faculty of Medicine, and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver, B.C., Canada, V6T 1Z3</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corey%2BNislow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nislow%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorey%2BNislow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Corey Nislow</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:corey.nislow@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@ubc.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/3/933" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/3/933&amp;title=Interactively%20AUDIT%20Your%20Growth%20Curves%20with%20a%20Suite%20of%20R%20Packages" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/3/933&amp;title=Interactively%20AUDIT%20Your%20Growth%20Curves%20with%20a%20Suite%20of%20R%20Packages" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/3/933&amp;title=Interactively%20AUDIT%20Your%20Growth%20Curves%20with%20a%20Suite%20of%20R%20Packages" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/3/933&amp;text=Interactively%20AUDIT%20Your%20Growth%20Curves%20with%20a%20Suite%20of%20R%20Packages" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/3/933&amp;title=Interactively%20AUDIT%20Your%20Growth%20Curves%20with%20a%20Suite%20of%20R%20Packages" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/3/933&amp;t=Interactively%20AUDIT%20Your%20Growth%20Curves%20with%20a%20Suite%20of%20R%20Packages" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/3/933&amp;title=Interactively%20AUDIT%20Your%20Growth%20Curves%20with%20a%20Suite%20of%20R%20Packages" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/3/933&amp;title=Interactively%20AUDIT%20Your%20Growth%20Curves%20with%20a%20Suite%20of%20R%20Packages" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495851/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495851" id="node495851--3" data-pisa="ggg;10/3/933" data-pisa-master="ggg;g3.119.400898" data-apath="/ggg/10/3/933.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Interactively AUDIT Your Growth Curves with a Suite of R Packages</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9336-226X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicolas P. J. Coutin</span>, <span class="highwire-citation-author" data-delta="1">Guri Giaever</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Corey Nislow</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">933-943;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400898 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495851--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas P. J. Coutin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biology, Faculty of Medicine, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BP.%2BJ.%2BCoutin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coutin%20NP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BP.%2BJ.%2BCoutin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9336-226X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicolas P. J. Coutin</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Guri Giaever </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver, B.C., Canada, V6T 1Z3</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guri%2BGiaever%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giaever%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuri%2BGiaever%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Corey Nislow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and Molecular Biology, Faculty of Medicine, and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Faculty of Pharmaceutical Sciences, University of British Columbia, Vancouver, B.C., Canada, V6T 1Z3</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Corey%2BNislow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nislow%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACorey%2BNislow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4016-8874" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Corey Nislow</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:corey.nislow@ubc.ca" class="" data-icon-position="" data-hide-link-title="0">corey.nislow@ubc.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=933&amp;rft.epage=943&amp;rft.atitle=Interactively%20AUDIT%20Your%20Growth%20Curves%20with%20a%20Suite%20of%20R%20Packages&amp;rft.volume=10&amp;rft.issue=3&amp;rft.date=2020-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Coutin%2C+Nicolas+P.+J.&amp;rft.au=Giaever%2C+Guri&amp;rft.au=Nislow%2C+Corey"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495851/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495851/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495851/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495851/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495851/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495851/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495851/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495851/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495851/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495851/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495851/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495851/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496031" id="node496031" data-pisa="ggg;g3.120.401086v1" data-pisa-master="ggg;g3.120.401086" data-apath="/ggg/early/2020/03/09/g3.120.401086.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/09/g3.120.401086" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The UCSF Mouse Inventory Database Application, an Open Source Web App for Sharing Mutant Mice within a Research Community</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495860" id="node495860" data-pisa="ggg;10/3/925" data-pisa-master="ggg;g3.119.400983" data-apath="/ggg/10/3/925.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/925" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">An Automated Method To Predict Mouse Gene and Protein Sequences Using Variant Data</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495845" id="node495845" data-pisa="ggg;10/3/945" data-pisa-master="ggg;g3.119.400945" data-apath="/ggg/10/3/945.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/945" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">CGSEA: A Flexible Tool for Evaluating the Associations of Chemicals with Complex Diseases</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Software%20and%20Data%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Software and Data Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/3/933" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-26" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/3/933.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/3/933.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/3/933.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
