<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/3/1039" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/3/1039.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/3/1039.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/3/1039.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/3/1039.atom" />
<meta name="HW.pisa" content="ggg;10/3/1039" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Type I-F CRISPR-Cas Distribution and Array Dynamics in Legionella pneumophila" />
<meta name="DC.Identifier" content="10.1534/g3.119.400813" />
<meta name="DC.Date" content="2020-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Deecker et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In bacteria and archaea, several distinct types of CRISPR-Cas systems provide adaptive immunity through broadly similar mechanisms: short nucleic acid sequences derived from foreign DNA, known as spacers, engage in complementary base pairing with invasive genetic elements setting the stage for nucleases to degrade the target DNA. A hallmark of type I CRISPR-Cas systems is their ability to acquire spacers in response to both new and previously encountered invaders (naïve and primed acquisition, respectively). Our phylogenetic analyses of 43 L. pneumophila type I-F CRISPR-Cas systems and their resident genomes suggest that many of these systems have been horizontally acquired. These systems are frequently encoded on plasmids and can co-occur with nearly identical chromosomal loci. We show that two such co-occurring systems are highly protective and undergo efficient primed acquisition in the lab. Furthermore, we observe that targeting by one system’s array can prime spacer acquisition in the other. Lastly, we provide experimental and genomic evidence for a model in which primed acquisition can efficiently replenish a depleted type I CRISPR array following a mass spacer deletion event." />
<meta name="DC.Contributor" content="Shayna R. Deecker" />
<meta name="DC.Contributor" content="Alexander W. Ensminger" />
<meta name="article:published_time" content="2020-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Type I-F CRISPR-Cas Distribution and Array Dynamics in Legionella pneumophila" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In bacteria and archaea, several distinct types of CRISPR-Cas systems provide adaptive immunity through broadly similar mechanisms: short nucleic acid sequences derived from foreign DNA, known as spacers, engage in complementary base pairing with invasive genetic elements setting the stage for nucleases to degrade the target DNA. A hallmark of type I CRISPR-Cas systems is their ability to acquire spacers in response to both new and previously encountered invaders (naïve and primed acquisition, respectively). Our phylogenetic analyses of 43 &lt;i&gt;L. pneumophila&lt;/i&gt; type I-F CRISPR-Cas systems and their resident genomes suggest that many of these systems have been horizontally acquired. These systems are frequently encoded on plasmids and can co-occur with nearly identical chromosomal loci. We show that two such co-occurring systems are highly protective and undergo efficient primed acquisition in the lab. Furthermore, we observe that targeting by one system’s array can prime spacer acquisition in the other. Lastly, we provide experimental and genomic evidence for a model in which primed acquisition can efficiently replenish a depleted type I CRISPR array following a mass spacer deletion event.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/03/01" />
<meta name="citation_mjid" content="ggg;10/3/1039" />
<meta name="citation_id" content="10/3/1039" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/3/1039" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/3/1039.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/3/1039.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/3/1039.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400813" />
<meta name="citation_pmid" content="31937548" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1039" />
<meta name="citation_lastpage" content="1050" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Shayna R. Deecker" />
<meta name="citation_author_institution" content="Department of Biochemistry and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1234-1123" />
<meta name="citation_author" content="Alexander W. Ensminger" />
<meta name="citation_author_institution" content="Department of Biochemistry and" />
<meta name="citation_author_institution" content="Department of Molecular Genetics, University of Toronto, Ontario, Canada" />
<meta name="citation_author_email" content="alex.ensminger@utoronto.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0824-3704" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Barrangou;citation_author=C. Fremaux;citation_author=H. Deveau;citation_author=M. Richards;citation_author=P. Boyaval;citation_title=CRISPR Provides Acquired Resistance Against Viruses in Prokaryotes;citation_pages=1709-1712;citation_volume=315;citation_year=2007;citation_issue=5819;citation_pmid=17379808;citation_doi=10.1126/science.1138140" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=A. Biswas;citation_author=R. H. J. Staals;citation_author=S. E. Morales;citation_author=P. C. Fineran;citation_author=C. M. Brown;citation_title=CRISPRDetect: A flexible algorithm to define CRISPR arrays.;citation_pages=356;citation_volume=17;citation_year=2016;citation_pmid=27184979;citation_doi=10.1186/s12864-016-2627-0" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Annals of Internal Medicine;citation_journal_abbrev=ANN INTERN MED;citation_author=D. J. Brenner;citation_author=A. G. Steigerwalt;citation_author=J. E. McDade;citation_title=Classification of the Legionnaires' Disease Bacterium: Legionella pneumophila, genus novum, species nova, of the Family Legionellaceae, familia nova;citation_pages=656-658;citation_volume=90;citation_year=1979;citation_issue=4;citation_pmid=434652;citation_doi=10.7326/0003-4819-90-4-656" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. J. J. Brouns;citation_author=M. M. Jore;citation_author=M. Lundgren;citation_author=E. R. Westra;citation_author=R. J. H. Slijkhuis;citation_title=Small CRISPR RNAs Guide Antiviral Defense in Prokaryotes;citation_pages=960-964;citation_volume=321;citation_year=2008;citation_issue=5891;citation_pmid=18703739;citation_doi=10.1126/science.1159689" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=D. Burstein;citation_author=F. Amaro;citation_author=T. Zusman;citation_author=Z. Lifshitz;citation_author=O. Cohen;citation_title=Genomic analysis of 38 Legionella species identifies large and diverse effector repertoires.;citation_pages=167-175;citation_volume=48;citation_year=2016;citation_pmid=26752266;citation_doi=10.1038/ng.3481" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=K. C. Cady;citation_author=J. Bondy-Denomy;citation_author=G. E. Heussler;citation_author=A. R. Davidson;citation_title=The CRISPR/Cas adaptive immune system of Pseudomonas aeruginosa mediates resistance to naturally occurring and engineered phages.;citation_volume=194;citation_year=2012;citation_pmid=22885297;citation_doi=10.1128/JB.01184-12" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Cazalet;citation_author=S. Jarraud;citation_author=Y. Ghavi-Helm;citation_author=F. Kunst;citation_author=P. Glaser;citation_title=Multigenome analysis identifies a worldwide distributed epidemic Legionella pneumophila clone that emerged within a highly diverse species.;citation_volume=18;citation_year=2008;citation_pmid=18256241;citation_doi=10.1101/gr.7229808" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Cazalet;citation_author=C. Rusniok;citation_author=H. Brüggemann;citation_author=N. Zidane;citation_author=A. Magnier;citation_title=Evidence in the Legionella pneumophila genome for exploitation of host cell functions and high genome plasticity.;citation_pages=1165-1173;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15467720;citation_doi=10.1038/ng1447" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Couvin;citation_author=A. Bernheim;citation_author=C. Toffano-Nioche;citation_author=M. Touchon;citation_author=J. Michalik;citation_title=CRISPRCasFinder, an update of CRISPRFinder, includes a portable version, enhanced performance and integrates search for Cas proteins.;citation_pages=W246-W251;citation_volume=46;citation_year=2018;citation_doi=10.1093/nar/gky425" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=G. D’Auria;citation_author=N. Jimenez-Hernandez;citation_author=F. Peris-Bondia;citation_author=A. Moya;citation_author=A. Latorre;citation_title=Legionella pneumophila pangenome reveals strain-specific virulence factors.;citation_pages=181-181;citation_volume=11;citation_year=2010;citation_pmid=20236513;citation_doi=10.1186/1471-2164-11-181" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=K. A. Datsenko;citation_author=K. Pougach;citation_author=A. Tikhonov;citation_author=B. L. Wanner;citation_author=K. V. Severinov;citation_title=Molecular memory of prior infections activates the CRISPR/Cas adaptive bacterial immunity system.;citation_pages=945-945;citation_volume=3;citation_year=2012;citation_pmid=22781758;citation_doi=10.1038/ncomms1937" />
<meta name="citation_reference" content="citation_author=S. David;citation_author=L. Sánchez-Busó;citation_author=S. R. Harris;citation_author=P. Marttinen;citation_author=C. Rusniok;citation_title=Dynamics and impact of homologous recombination on the evolution of Legionella pneumophila.;citation_pages=e1006855;citation_volume=13;citation_year=2017;citation_pmid=28650958;citation_doi=10.1371/journal.pgen.1006855" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=H. Deveau;citation_author=R. Barrangou;citation_author=J. E. Garneau;citation_author=J. Labonte;citation_author=C. Fremaux;citation_title=Phage response to CRISPR-encoded resistance in Streptococcus thermophilus.;citation_volume=190;citation_year=2008;citation_pmid=18065545;citation_doi=10.1128/JB.01412-07" />
<meta name="citation_reference" content="citation_journal_title=Viruses;citation_author=M. Dion;citation_author=S. Labrie;citation_author=S. Shah;citation_author=S. Moineau;citation_title=CRISPRStudio: A User-Friendly Software for Rapid CRISPR Array Visualization.;citation_pages=602-611;citation_volume=10;citation_year=2018;citation_doi=10.3390/v10110602" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. C. Edgar;citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity.;citation_pages=113-113;citation_volume=5;citation_year=2004;citation_pmid=15318951;citation_doi=10.1186/1471-2105-5-113" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Elmore;citation_author=T. Deighan;citation_author=J. Westpheling;citation_author=R. M. Terns;citation_author=M. P. Terns;citation_title=DNA targeting by the type I-G and type I-A CRISPR–Cas systems of Pyrococcus furiosus.;citation_volume=43;citation_year=2015;citation_pmid=26519471;citation_doi=10.1093/nar/gkv1140" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. C. Fineran;citation_author=M. J. H. Gerritzen;citation_author=M. Suarez-Diez;citation_author=T. Kunne;citation_author=J. Boekhorst;citation_title=Degenerate target sites mediate rapid primed CRISPR adaptation.;citation_volume=111;citation_year=2014;citation_pmid=24711427;citation_doi=10.1073/pnas.1400071111" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=L. Gomez-Valero;citation_author=C. Rusniok;citation_author=S. Jarraud;citation_author=B. Vacherie;citation_author=Z. Rouy;citation_title=Extensive recombination events and horizontal gene transfer shaped the Legionella pneumophila genomes.;citation_pages=536-536;citation_volume=12;citation_year=2011;citation_pmid=22044686;citation_doi=10.1186/1471-2164-12-536" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=L. Gomez-Valero;citation_author=C. Rusniok;citation_author=M. Rolando;citation_author=M. Neou;citation_author=D. Dervins-Ravault;citation_title=Comparative analyses of Legionella species identifies genetic features of strains causing Legionnaires’ disease.;citation_pages=505;citation_volume=15;citation_year=2014;citation_pmid=25370836;citation_doi=10.1186/s13059-014-0505-0" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=J. P. J. Hall;citation_author=M. A. Brockhurst;citation_author=E. Harrison;citation_title=Sampling the mobile gene pool: innovation via horizontal gene transfer in bacteria.;citation_volume=372;citation_year=2017;citation_pmid=29061896;citation_doi=10.1098/rstb.2016.0424" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. E. Haurwitz;citation_author=M. Jinek;citation_author=B. Wiedenheft;citation_author=K. Zhou;citation_author=J. A. Doudna;citation_title=Sequence- and Structure-Specific RNA Processing by a CRISPR Endonuclease;citation_pages=1355-1358;citation_volume=329;citation_year=2010;citation_issue=5997;citation_pmid=20829488;citation_doi=10.1126/science.1192272" />
<meta name="citation_reference" content="citation_author=G. E. Heussler;citation_author=J. L. Miller;citation_author=C. E. Price;citation_author=A. J. Collins;citation_author=G. A. O’Toole;citation_title=Requirements for Pseudomonas aeruginosa Type I-F CRISPR-Cas Adaptation Determined Using a Biofilm Enrichment Assay.;citation_volume=198;citation_year=2016;citation_pmid=27573013;citation_doi=10.1128/JB.00458-16" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=P. Horvath;citation_author=D. A. Romero;citation_author=A. C. Coute-Monvoisin;citation_author=M. Richards;citation_author=H. Deveau;citation_title=Diversity, activity, and evolution of CRISPR loci in Streptococcus thermophilus.;citation_volume=190;citation_year=2008;citation_pmid=18065539;citation_doi=10.1128/JB.01415-07" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=A. P. Hynes;citation_author=M. Villion;citation_author=S. Moineau;citation_title=Adaptation in bacterial CRISPR-Cas immunity can be driven by defective phages.;citation_pages=4399;citation_volume=5;citation_year=2014;citation_pmid=25056268;citation_doi=10.1038/ncomms5399" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=Y. Ishino;citation_author=H. Shinagawa;citation_author=K. Makino;citation_author=M. Amemura;citation_author=A. Nakata;citation_title=Nucleotide sequence of the iap gene, responsible for alkaline phosphatase isozyme conversion in Escherichia coli, and identification of the gene product.;citation_pages=5429-5433;citation_volume=169;citation_year=1987;citation_issue=12;citation_pmid=3316184;citation_doi=10.1128/jb.169.12.5429-5433.1987" />
<meta name="citation_reference" content="citation_journal_title=Cell Host Microbe;citation_author=S. A. Jackson;citation_author=N. Birkholz;citation_author=L. M. Malone;citation_author=P. C. Fineran;citation_title=Imprecise Spacer Acquisition Generates CRISPR-Cas Immune Diversity through Primed Adaptation.;citation_pages=250-260.e4;citation_volume=25;citation_year=2019;citation_pmid=30661951;citation_doi=10.1016/j.chom.2018.12.014" />
<meta name="citation_reference" content="Jackson, S. A., R. E. McKenzie, R. D. Fagerlund, S. N. Kieper, P. C. Fineran et al., 2017 CRISPR-Cas: Adapting to change. Science 356: eaal5056–11. 10.1126/science.aal5056" />
<meta name="citation_reference" content="citation_journal_title=Omics :  a journal of integrative biology;citation_journal_abbrev=OMICS;citation_author=R. Jansen;citation_author=J. D. van Embden;citation_author=W. Gaastra;citation_author=L. M. Schouls;citation_title=Identification of a novel family of sequence repeats among prokaryotes.;citation_pages=23-33;citation_volume=6;citation_year=2002;citation_issue=1;citation_pmid=11883425;citation_doi=10.1089/15362310252780816" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=R. Jansen;citation_author=J. D. van Embden;citation_author=W. Gaastra;citation_author=L. M. Schouls;citation_title=Identification of genes that are associated with DNA repeats in prokaryotes.;citation_pages=1565-1575;citation_volume=43;citation_year=2002;citation_issue=6;citation_pmid=11952905;citation_doi=10.1046/j.1365-2958.2002.02839.x" />
<meta name="citation_reference" content="citation_author=W. Jiang;citation_author=I. Maniv;citation_author=F. Arain;citation_author=Y. Wang;citation_author=B. R. Levin;citation_title=Dealing with the evolutionary downside of CRISPR immunity: bacteria and beneficial plasmids.;citation_pages=e1003844;citation_volume=9;citation_year=2013;citation_pmid=24086164;citation_doi=10.1371/journal.pgen.1003844" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=M. M. Jore;citation_author=M. Lundgren;citation_author=E. van Duijn;citation_author=J. B. Bultema;citation_author=E. R. Westra;citation_title=Structural basis for CRISPR RNA-guided DNA recognition by Cascade.;citation_pages=529-536;citation_volume=18;citation_year=2011;citation_issue=5;citation_pmid=21460843;citation_doi=10.1038/nsmb.2019" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Microbiol.;citation_author=M. Juhas;citation_title=Horizontal gene transfer in human pathogens.;citation_volume=41;citation_year=2013;citation_doi=10.3109/1040841X.2013.804031" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Microbiol.;citation_author=E. V. Koonin;citation_author=K. S. Makarova;citation_author=F. Zhang;citation_title=Diversity, classification and evolution of CRISPR-Cas systems.;citation_pages=67-78;citation_volume=37;citation_year=2017;citation_pmid=28605718;citation_doi=10.1016/j.mib.2017.05.008" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Krzywinski;citation_author=J. Schein;citation_author=I. Birol;citation_author=J. Connors;citation_author=R. Gascoyne;citation_title=Circos: an information aesthetic for comparative genomics.;citation_volume=19;citation_year=2009;citation_pmid=19541911;citation_doi=10.1101/gr.092759.109" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=G. Stecher;citation_author=K. Tamura;citation_title=MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets.;citation_volume=33;citation_year=2016;citation_pmid=27004904;citation_doi=10.1093/molbev/msw054" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Kuznedelov;citation_author=V. Mekler;citation_author=S. Lemak;citation_author=M. Tokmina-Lukaszewska;citation_author=K. A. Datsenko;citation_title=Altered stoichiometry Escherichia coli Cascade complexes with shortened CRISPR RNA spacers are capable of interference and primed adaptation.;citation_volume=44;citation_year=2016;citation_pmid=27738137;citation_doi=10.1093/nar/gkw914" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Levy;citation_author=M. G. Goren;citation_author=I. Yosef;citation_author=O. Auster;citation_author=M. Manor;citation_title=CRISPR adaptation biases explain preference for acquisition of foreign DNA.;citation_pages=505-510;citation_volume=520;citation_year=2015;citation_pmid=25874675;citation_doi=10.1038/nature14302" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Li;citation_author=R. Wang;citation_author=H. Xiang;citation_title=Haloarcula hispanica CRISPR authenticates PAM of a target sequence to prime discriminative adaptation.;citation_volume=42;citation_year=2014;citation_pmid=24803673;citation_doi=10.1093/nar/gku389" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Li;citation_author=R. Wang;citation_author=D. Zhao;citation_author=H. Xiang;citation_title=Adaptation of the Haloarcula hispanica CRISPR-Cas system to a purified virus strictly requires a priming process.;citation_volume=42;citation_year=2014;citation_pmid=24265226;citation_doi=10.1093/nar/gkt1154" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Li;citation_author=L. Gong;citation_author=D. Zhao;citation_author=J. Zhou;citation_author=H. Xiang;citation_title=The spacer size of I-B CRISPR is modulated by the terminal sequence of the protospacer.;citation_pages=4642-4654;citation_volume=45;citation_year=2017;citation_pmid=28379481;citation_doi=10.1093/nar/gkx229" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=M.-J. Lopez-Sanchez;citation_author=E. Sauvage;citation_author=V. Da Cunha;citation_author=D. Clermont;citation_author=E. Ratsima Hariniaina;citation_title=The highly dynamic CRISPR1 system of Streptococcus agalactiae controls the diversity of its mobilome.;citation_pages=1057-1071;citation_volume=85;citation_year=2012;citation_issue=6;citation_pmid=22834929;citation_doi=10.1111/j.1365-2958.2012.08172.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Magoc;citation_author=S. L. Salzberg;citation_title=FLASH: fast length adjustment of short reads to improve genome assemblies.;citation_volume=27;citation_year=2011;citation_pmid=21903629;citation_doi=10.1093/bioinformatics/btr507" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=S. Majumdar;citation_author=P. Zhao;citation_author=N. T. Pfister;citation_author=M. Compton;citation_author=S. Olson;citation_title=Three CRISPR-Cas immune effector complexes coexist in Pyrococcus furiosus.;citation_volume=21;citation_year=2015;citation_pmid=25904135;citation_doi=10.1261/rna.049130.114" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=K. S. Makarova;citation_author=D. H. Haft;citation_author=R. Barrangou;citation_author=S. J. J. Brouns;citation_author=E. Charpentier;citation_title=Evolution and classification of the CRISPR-Cas systems.;citation_pages=467-477;citation_volume=9;citation_year=2011;citation_issue=6;citation_pmid=21552286;citation_doi=10.1038/nrmicro2577" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Microbiol.;citation_author=K. S. Makarova;citation_author=Y. I. Wolf;citation_author=O. S. Alkhnbashi;citation_author=F. Costa;citation_author=S. A. Shah;citation_title=An updated evolutionary classification of CRISPR–Cas systems.;citation_pages=722-736;citation_volume=13;citation_year=2015;citation_pmid=26411297;citation_doi=10.1038/nrmicro3569" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=L. A. Marraffini;citation_author=E. J. Sontheimer;citation_title=CRISPR Interference Limits Horizontal Gene Transfer in Staphylococci by Targeting DNA;citation_pages=1843-1845;citation_volume=322;citation_year=2008;citation_issue=5909;citation_pmid=19095942;citation_doi=10.1126/science.1165771" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=P. R. McAdam;citation_author=C. W. Vander Broek;citation_author=D. S. Lindsay;citation_author=M. J. Ward;citation_author=M. F. Hanson;citation_title=Gene flow in environmental Legionella pneumophila leads to genetic and pathogenic heterogeneity within a Legionnaires’ disease outbreak.;citation_pages=1-10;citation_volume=15;citation_year=2014;citation_pmid=24393533;citation_doi=10.1186/gb-2014-15-1-r1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. W. Modell;citation_author=W. Jiang;citation_author=L. A. Marraffini;citation_title=CRISPR–Cas systems exploit viral DNA injection to establish and maintain adaptive immunity.;citation_pages=101-104;citation_volume=544;citation_year=2017;citation_pmid=28355179;citation_doi=10.1038/nature21719" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_journal_abbrev=Microbiology;citation_author=F. J. M. Mojica;citation_author=C. Díez-Villaseñor;citation_author=J. García-Martínez;citation_author=C. Almendros;citation_title=Short motif sequences determine the targets of the prokaryotic CRISPR defence system;citation_pages=733-740;citation_volume=155;citation_year=2009;citation_issue=3;citation_pmid=19246744;citation_doi=10.1099/mic.0.023960-0" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=F. J. M. Mojica;citation_author=C. Díez-Villaseñor;citation_author=E. Soria;citation_author=G. Juez;citation_title=Biological significance of a family of regularly spaced repeats in the genomes of Archaea, Bacteria and mitochondria.;citation_pages=244-246;citation_volume=36;citation_year=2000;citation_issue=1;citation_pmid=10760181;citation_doi=10.1046/j.1365-2958.2000.01838.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Ochman;citation_author=J. G. Lawrence;citation_author=E. A. Groisman;citation_title=Lateral gene transfer and the nature of bacterial innovation.;citation_pages=299-304;citation_volume=405;citation_year=2000;citation_issue=6784;citation_pmid=10830951;citation_doi=10.1038/35012500" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. J. Page;citation_author=C. A. Cummins;citation_author=M. Hunt;citation_author=V. K. Wong;citation_author=S. Reuter;citation_title=Roary: rapid large-scale prokaryote pan genome analysis.;citation_volume=31;citation_year=2015;citation_pmid=26198102;citation_doi=10.1093/bioinformatics/btv421" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=C. Rao;citation_author=D. Chin;citation_author=A. W. Ensminger;citation_title=Priming in a permissive type I–C CRISPR–Cas system reveals distinct dynamics of spacer acquisition and loss.;citation_volume=23;citation_year=2017;citation_pmid=28724535;citation_doi=10.1261/rna.062083.117" />
<meta name="citation_reference" content="citation_journal_title=Cell. Microbiol.;citation_author=C. Rao;citation_author=C. Guyard;citation_author=C. Pelaz;citation_author=J. Wasserscheid;citation_author=J. Bondy-Denomy;citation_title=Active and adaptive Legionella CRISPR-Cas reveals a recurrent challenge to the pathogen.;citation_pages=1319-1338;citation_volume=18;citation_year=2016;citation_pmid=26936325;citation_doi=10.1111/cmi.12586" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Richter;citation_author=R. L. Dy;citation_author=R. E. McKenzie;citation_author=B. N. J. Watson;citation_author=C. Taylor;citation_title=Priming in the Type I-F CRISPR-Cas system triggers strand-independent spacer acquisition, bi-directionally from the primed protospacer.;citation_volume=42;citation_year=2014;citation_pmid=24990370;citation_doi=10.1093/nar/gku527" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Microbiol.;citation_author=J. E. Samson;citation_author=A. H. Magadan;citation_author=M. Sabri;citation_author=S. Moineau;citation_title=Revenge of the phages: defeating bacterial defences.;citation_pages=675-687;citation_volume=11;citation_year=2013;citation_pmid=23979432;citation_doi=10.1038/nrmicro3096" />
<meta name="citation_reference" content="citation_journal_title=RNA Biol.;citation_author=E. E. Savitskaya;citation_author=E. Semenova;citation_author=V. Dedkov;citation_author=A. Metlitskaya;citation_author=K. V. Severinov;citation_title=High-throughput analysis of type I-E CRISPR/Cas spacer acquisition in E. coli.;citation_pages=716-725;citation_volume=10;citation_year=2013;citation_pmid=23619643;citation_doi=10.4161/rna.24325" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=L. Sánchez-Busó;citation_author=I. Comas;citation_author=G. Jorques;citation_author=F. González-Candelas;citation_title=Recombination drives genome evolution in outbreak-related Legionella pneumophila isolates.;citation_pages=1205-1211;citation_volume=46;citation_year=2014;citation_pmid=25282102;citation_doi=10.1038/ng.3114" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Seemann;citation_title=Prokka: rapid prokaryotic genome annotation.;citation_volume=30;citation_year=2014;citation_pmid=24642063;citation_doi=10.1093/bioinformatics/btu153" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Semenova;citation_author=M. M. Jore;citation_author=K. A. Datsenko;citation_author=A. Semenova;citation_author=E. R. Westra;citation_title=Interference by clustered regularly interspaced short palindromic repeat (CRISPR) RNA is governed by a seed sequence.;citation_volume=108;citation_year=2011;citation_pmid=21646539;citation_doi=10.1073/pnas.1104144108" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=S. Silas;citation_author=P. Lucas-Elio;citation_author=S. A. Jackson;citation_author=A. Aroca-Crevillén. ;citation_author=L. L. Hansen;citation_title=Type III CRISPR-Cas systems can provide redundancy to counteract viral escape from type I systems.;citation_pages=e36853;citation_volume=6;citation_year=2017;citation_doi=10.7554/eLife.27601" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=J. M. Solomon;citation_author=A. Rupper;citation_author=J. A. Cardelli;citation_author=R. R. Isberg;citation_title=Intracellular Growth of Legionella pneumophila in Dictyostelium discoideum, a System for Genetic Analysis of Host-Pathogen Interactions;citation_pages=2939-2947;citation_volume=68;citation_year=2000;citation_issue=5;citation_pmid=10768992;citation_doi=10.1128/IAI.68.5.2939-2947.2000" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=R. H. J. Staals;citation_author=Y. Agari;citation_author=S. Maki-Yonekura;citation_author=Y. Zhu;citation_author=D. W. Taylor;citation_title=Structure and Activity of the RNA-Targeting Type III-B CRISPR-Cas Complex of Thermus thermophilus.;citation_pages=135-145;citation_volume=52;citation_year=2013;citation_pmid=24119403;citation_doi=10.1016/j.molcel.2013.09.013" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=R. H. J. Staals;citation_author=S. A. Jackson;citation_author=A. Biswas;citation_author=S. J. J. Brouns;citation_author=C. M. Brown;citation_title=Interference-driven spacer acquisition is dominant over naive and primed adaptation in a native CRISPR-Cas system.;citation_pages=12853;citation_volume=7;citation_year=2016;citation_pmid=27694798;citation_doi=10.1038/ncomms12853" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=R. H. J. Staals;citation_author=Y. Zhu;citation_author=D. W. Taylor;citation_author=J. E. Kornfeld;citation_author=K. Sharma;citation_title=RNA Targeting by the Type III-A CRISPR-Cas Csm Complex of Thermus thermophilus.;citation_pages=518-530;citation_volume=56;citation_year=2014;citation_pmid=25457165;citation_doi=10.1016/j.molcel.2014.10.005" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=S. Y. Stanley;citation_author=K. L. Maxwell;citation_title=Phage-Encoded Anti-CRISPR Defenses.;citation_volume=52;citation_year=2018;citation_doi=10.1146/annurev-genet-120417-031321" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. C. Swarts;citation_author=C. Mosterd;citation_author=M. W. J. van Passel;citation_author=S. J. J. Brouns;citation_title=CRISPR interference directs strand specific spacer acquisition.;citation_pages=e35888-e35888;citation_volume=7;citation_year=2012;citation_issue=4;citation_pmid=22558257;citation_doi=10.1371/journal.pone.0035888" />
<meta name="citation_reference" content="citation_journal_title=Biol. Direct;citation_author=A. Toms;citation_author=R. Barrangou;citation_title=On the global CRISPR array behavior in class I systems.;citation_pages=20;citation_volume=12;citation_year=2017;citation_doi=10.1186/s13062-017-0193-2" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=S. van Houte;citation_author=A. Buckling;citation_author=E. R. Westra;citation_title=Evolutionary Ecology of Prokaryotic Immune Mechanisms;citation_pages=745-763;citation_volume=80;citation_year=2016;citation_issue=3;citation_pmid=27412881;citation_doi=10.1128/MMBR.00011-16" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Vorontsova;citation_author=K. A. Datsenko;citation_author=S. Medvedeva;citation_author=J. Bondy-Denomy;citation_author=E. E. Savitskaya;citation_title=Foreign DNA acquisition by the I-F CRISPR–Cas system requires all components of the interference machinery.;citation_volume=43;citation_year=2015;citation_pmid=26586803;citation_doi=10.1093/nar/gkv1261" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=E. R. Westra;citation_author=P. B. G. van Erp;citation_author=T. Künne;citation_author=S. P. Wong;citation_author=R. H. J. Staals;citation_title=CRISPR immunity relies on the consecutive binding and degradation of negatively supercoiled invader DNA by Cascade and Cas3.;citation_pages=595-605;citation_volume=46;citation_year=2012;citation_issue=5;citation_pmid=22521689;citation_doi=10.1016/j.molcel.2012.03.018" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. Wiedenheft;citation_author=E. van Duijn;citation_author=J. B. Bultema;citation_author=S. P. Waghmare;citation_author=K. Zhou;citation_title=RNA-guided complex from a bacterial immune system enhances target recognition through seed sequence interactions.;citation_volume=108;citation_year=2011;citation_pmid=21536913;citation_doi=10.1073/pnas.1102716108" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. Xue;citation_author=A. S. Seetharam;citation_author=O. Musharova;citation_author=K. V. Severinov;citation_author=S. J. J. Brouns;citation_title=CRISPR interference and priming varies with individual spacer sequences.;citation_volume=43;citation_year=2015;citation_pmid=26586800;citation_doi=10.1093/nar/gkv1259" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. Yosef;citation_author=M. G. Goren;citation_author=U. Qimron;citation_title=Proteins and DNA elements essential for the CRISPR adaptation process in Escherichia coli.;citation_volume=40;citation_year=2012;citation_pmid=22402487;citation_doi=10.1093/nar/gks216" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Type I-F CRISPR-Cas Distribution and Array Dynamics in Legionella pneumophila" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="twitter:description" content="In bacteria and archaea, several distinct types of CRISPR-Cas systems provide adaptive immunity through broadly similar mechanisms: short nucleic acid sequences derived from foreign DNA, known as spacers, engage in complementary base pairing with invasive genetic elements setting the stage for nucleases to degrade the target DNA. A hallmark of type I CRISPR-Cas systems is their ability to acquire spacers in response to both new and previously encountered invaders (naïve and primed acquisition, respectively). Our phylogenetic analyses of 43 L. pneumophila type I-F CRISPR-Cas systems and their resident genomes suggest that many of these systems have been horizontally acquired. These systems are frequently encoded on plasmids and can co-occur with nearly identical chromosomal loci. We show that two such co-occurring systems are highly protective and undergo efficient primed acquisition in the lab. Furthermore, we observe that targeting by one system’s array can prime spacer acquisition in the other. Lastly, we provide experimental and genomic evidence for a model in which primed acquisition can efficiently replenish a depleted type I CRISPR array following a mass spacer deletion event." />
<meta name="og-title" property="og:title" content="Type I-F CRISPR-Cas Distribution and Array Dynamics in Legionella pneumophila" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/3/1039" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In bacteria and archaea, several distinct types of CRISPR-Cas systems provide adaptive immunity through broadly similar mechanisms: short nucleic acid sequences derived from foreign DNA, known as spacers, engage in complementary base pairing with invasive genetic elements setting the stage for nucleases to degrade the target DNA. A hallmark of type I CRISPR-Cas systems is their ability to acquire spacers in response to both new and previously encountered invaders (naïve and primed acquisition, respectively). Our phylogenetic analyses of 43 L. pneumophila type I-F CRISPR-Cas systems and their resident genomes suggest that many of these systems have been horizontally acquired. These systems are frequently encoded on plasmids and can co-occur with nearly identical chromosomal loci. We show that two such co-occurring systems are highly protective and undergo efficient primed acquisition in the lab. Furthermore, we observe that targeting by one system’s array can prime spacer acquisition in the other. Lastly, we provide experimental and genomic evidence for a model in which primed acquisition can efficiently replenish a depleted type I CRISPR array following a mass spacer deletion event." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
    <title>Type I-F CRISPR-Cas Distribution and Array Dynamics in Legionella pneumophila | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"8S-LCmFRdKkDMRWUEdXVIm1o2PjMrcvW-jlZ2PxArsE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495857","apath":"\/ggg\/10\/3\/1039.atom","pisa":"ggg;10\/3\/1039","ac":{"\/ggg\/10\/3\/1039.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/3\/1039.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/3\/1039"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/3\/1039"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/3\/1039?rss=1":true,"\/content\/10\/3\/1039":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tD25XkZB7vvWUTnu3r\\\\\\\/FBC4t3Fb4rO+UMRxZatm9qcotw0itgGDvcwlfLItmL9tGoIab\\\\\\\/yEadhZZhStFEmVoONSAbxHRuS9uFwnOwQ6Boyp\\\\\\\/hRqetPC3\\\\\\\/aunCEgR6cq6i5JXsfL2e91MFSfwaQIU+hKyqVfYaL25iN1GcldfKrRrpnb89gh9wdDdF5Dp80dPjs0bctj1VMzCdwlBbeZdJGOdsh48NQPZpW7PQK8\\\\\\\/QDjdB4fpIVQntxpaZ3O1dIrAMHxTXhEMa8\\\\\\\/JyQQ6SXyaZoJtXwricS7riEoIrZWI2cyfuIP+kWj2kY7lSVMqC4CLcDVmrV51DMPrC9Awky0yyP5NxevXTbCOmefHlmEaYbcfliW0ViXu0NOwIYy1NxKVK1WDOW4xnn01gQ07klzIKLjdBo3s7NF29XmufrzVHeMw8XM3z\\\\\\\/FqAyJroNY\\\\\\\/z1pomwuHC9rrxZRMO6d3tWKjYItTQGsgiwmH+IN23Vmnwdwok8hKjqwzjlZTCYDoi6iyH2zHtCFfrARK+mSzVczskzc1ZJOWs4jQpytg9x\\\\\\\/RiiAEN80JdKToYv4j7cB7bu4xJSqzpSLPJGgntmE9mPDh4elmPZvT1b1+4TrEf4xq4EJZJn0OUaPNUQas5xI2IGsa9djxd\\\\\\\/DCeWdCAdHKfMc5hSu6SrGCnAv46T+th6b\\\\\\\/xgR204xJk0Eivd\\\\\\\/klKvI3sR6RxYFIeBkQPODrIuseTRlkOdR5HBqmD5F3pqI9pjLlaKYg974mPHfBbQuXTf+QVlE2sTHQS6iiqTyd5deX42uFrqDtnbfMceQOk6xrgAqT0ZhlgHs0Uxx8V4qNVgQN4CY4PiIPbfh\\\\\\\/UqUQ7959NO\\\\\\\/+in3P2r5OhJSRszswXKn6Zmyq1cjD5wl35NDaQnrEu2\\\\\\\/Ln79aSk+LVeNM9OoVDMn+vlQledFaieDfdDqavCQMNG+CbPjlM7JxWNmz4+eIbOxqD0kBzAtT86sQRcfOLXOogg4b1HtqE5jyawrEK\\\\\\\/yMVgLRvqr16bQWObckF+jE5WCnWtYPwXsX6Or1zeusrO78CRujhXJEYtOwj47rUy6Bm0QgYYt0vsa1rQEgti0ZRSXugENwvbYIEhEEU92A8GCgV6Ldd5u3UsDdUJJEe0nTnzkJsRlWuCSoChskGmCT4EorBaLxqSGEeUJrOJ98hKNshlakjfGsmYHS0nBFg8UkVoWPy3JMeyOcQlqFoeczXoqgUztpoTTlOTjuknOC4iZGSsVJeacLLSbi9OHE6AevjlJCP6xk8RVtFHrnTbYlAqS\\\u0022,\\\u0022iv\\\u0022:\\\u0022pfH0xRoAf48AMD87WW36gg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eea968501ca4519f025ab8a9c777b5c7045809ce687c826a9b2286289320ce8a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022npBc2rkR7iYOv+6By236sZxaLcH\\\\\\\/ZEIuAx7zQ\\\\\\\/HVV43Wz93S9ej5TmW0J\\\\\\\/l5\\\\\\\/IUGI8iLCX4TaT7z+dg2c3AG7upaAkxCgosDfDcgLh8cwT8rSfLGQ2qgk6NiuG76QUHMHLraqY9dA5X\\\\\\\/a9+1UMRY15hsOPX5JpkC4BOVXYwYX0T9RKu2kCWwGcD2ku8Cq1imLA9zH3WBZQ\\\\\\\/NQeXctTV6DEF+57zZYLTRsp4oC3RnK1Gtamk1vffqZry4Y9BbulfESTEN7AVbJ6m9QTzjDh+6E0lsmHq9HzALbLry20UKOBA6m21wYdhoJxNZJi1Bn\\\\\\\/PppCPiiVJyHthWh9pxf5W+EUxwOcMKpoT0T4j6YaQ7OSWRQYV5oIpWuDX83kpJudw1gqHeFOYDowd93dzhFW1dTrUviHwdom67kh2rjYeG2SDxpksaYyOs3fE+wSJNXrwoqK+DL+KFpLIoccFzq1A3GZDXtIfBAZ71myNmFnyu76HGfTj9AdtVvZNDao5eOceSibSsDBdpDa5gFZHm4E1+fnyFtLAkxRCqzrQ3frPxy6cGbGerIANRuXznyfI5VrXytujjWqBKi62WGty3vKnoO\\\\\\\/AtqpYSGPRlE5mvlkqs7A5zJMnGeSa+QdtSL48uWqT4iXLeLufmZP2LcrHH0RUbienav8+0cs2UqISuHh5swkN1DP9G+niJNM0778GiP1FfRnFYZK4ZQ\\\\\\\/zGtTzuic68S0mYiaFlyEQHvASmE6lRctU18pAoQ4JyxT2X+Q4D+qLIL6efI4CcLt6fOS3VYW3hM6iHrA2V8w8ApH\\\\\\\/y3tffEdKh4gxe99yqGxqjggPA6IdOw6nmp7ZmgURRKPignKBhnj3\\\\\\\/s8L\\\\\\\/+FTNoRhWEPL8zOu\\\\\\\/I\\\\\\\/I0agQKK703PiGWIz1mpqv6GXQQHx\\\\\\\/RwJs507g6Kl6Wtmg1y0OTDg5NjoudivNRqasQELo3SkZndG1utlkMfHZq76khkj2Gb5tRzUn4tOeQvtAsZc0P29tkpLE8g7Ckz4JcKq90xGz+xpK2HZ\\\\\\\/NM+vUtuHubNajR3zVY18NO4VvIAkfQpr6a0bPLk5jSPuCb5oyFN8CHJ5c0u550RqguMJZmyxCrX1CAueiifJL4+zStePTeJp5dX02u8LbIIobibCys22XA6TAFNARlfg69Nqb9Gdu9f+p8jNQcbwGNlU0IlWEO06QR\\\\\\\/1nxLsTl0kTDdDdHzu4AZOa2jOoswf+beNfrT27caKudek2sLMPiMYOmMMzcnNGP3PDRvTcU8yR1XnWLxKzSb5EKloLf5HLN1tfRnDt\\\\\\\/3iVHrgZfxlSfPnQOImSFIaewMz7M+HQC7bhRqgbwpmcyBbnq9U7nh\\\\\\\/8Hc6RyOKuWB7C1IcnPpmw5SjolZlm5rovQWqbn6rlFkXC6TXHejU+vSvzvcic45q4U0jqxC21ZxTTxys8LtCHo+IEuZBoCnbH9vnSDDrEkvDkhWrKnMChfYA9JrUGAiD1FYbDr4UwPY+9gR8UVl6hrLWQ7LcVyDy0aS\\\\\\\/9KxTCTr5bEcwHQ9G4ZSAVNd2zw0mef107KTKHprAK\\\\\\\/11TvfXqgKYp\\\\\\\/n7hjqJoX+8ws90hU0t8ZSEt3A+FvoiitO38CTNWTgSRO2x2NmEe7mOmcAtZNuneb\\\\\\\/jTLM1nEVxIX1d4RJkXltULUn17AILFvp9B+HOCEYe9KTF17ww3DfFMDXJ8QZBVTlRf80yUrauBgaVVI19wokEQVtIIHXRoez5wSGIDqIn2Z55lTnACuIaWIDeL6lD4+I3oGFdQ+ET+oy\\\\\\\/PyO14dNl4+C+zluPWqJSqfvk1kEdJTiTDaBgVw3dviELXxuTKsG328MEFy06JoBkILwPZIxgwDF5aa\\\\\\\/rwnUihjc601U3ZWiLIX+lzvENo7UZsZXxAyhYoRmJOuPvA0DeTCW06K2eWYwt0QWAMLgcFLRj1QQ\\\\\\\/X+lvgA3bZsZmsXY3lAMKPkVGf9NAzaApUWeHxrd+ipbdwstfpSWqgCIYBqjouL4FFUms3Zhmlw1Aew3HrQLv\\\\\\\/mW3fV1qFNabNDHGd94N0wCCH0XK7DaBQvQol0Df2gMTb7QksLOFZqGEyJ5NAhdTMXKal5XIrYHgogPxTJbVPJzHsG7dmFfcMLhU1cho9I9pf6SrR24\\\\\\\/o\\\\\\\/1XuLw3EL8mL061ZCH383U\\\\\\\/IP+Hs2\\\\\\\/5F1pz7GYi2Z\\\\\\\/UCUae6lXSoFC6rF4dTpUVKmlI5678jLNpzPoZ8J8wcsRirNZ1o\\\\\\\/+eMvqFWI2gP06pyQUp5FH4zv2z4JlS6hUMih49Jvf5X1ydyiAx0X87Tg6OkVkFBQz4zePtsTJAu6zsxjFBE0mWSk5XJT4jR43xsK39oEJg6RtC33MwWhGWXdsY5ufigsdjn6a+IpQY6P\\\\\\\/DS6zoClxY\\\\\\\/vEhenWUvBxvkFSOiaPXQkXn3+keVpHX58i23WHR+4IWYnyV3QfI8kkkHl5Cekd8uLtdtdXBj5Ka5H5uIQPVW7Wnp4FU6\\\\\\\/7oIVe4nLgY7GHqrOd4ExourFMIH9foh85QofHdJPgXNdVKI\\\\\\\/2zryfHLlCooFRG8\\\\\\\/gIFy4NsHWduR3aDhjV4CW2tSVIGzB1\\\\\\\/158umAEGW9NCrMZr\\\\\\\/DbQE2DVUpFn2pHdjEkyXX8CE0nbQiDO7e7QwChDXUbmTE745VyTdceOV6YyJ5C2QfamsmSlAT+X6fA4nTSrpSpd1BWzBmPC3elfKJkvC3tV00xJxyesd4qE85ra+tmH7xBw4vzXsKtW04jbCnMfkeCL4hVr7BNdALUheiBHbEDaFQQGTgzJdexS6IVAnmKfmSgB90u0XcU4rGlSR8tvPXZ04YWnFk3IvZlzv8EBor7voYUSpHYQ\\\\\\\/VeVHT+V5JUb2aKKb33DOHJ9ltFBlp9AvyBaaFXeuQlI9EYBWjvrXsSyAsMV2eut5mUR37tTII3d97WNHElTBCAcnYsLl3sQWd4NYhIpFyU9uWKsJJFLEipDozdjL56JpbAFyiDr0dIcEcYeU3+gOiG5pACStUKywcy25TQYo3JP9pcK+eWFDBiI9N4LeUekPY2Z\\\\\\\/8aSczU9lPpWruYXypFW1i\\\\\\\/XYeXN8Yr9cOuGo5AN3PN4fSRoPCf\\\\\\\/ByBRa0lP\\\\\\\/8DNbzAzCscfSeePjZaxyZivXNrC6Ygb2RbuK2iVzH+g2SRsQjunnO5GHpLK5QA1xiiPEr0iRXZpnJlCtA3fsWCFG3tcsGDNCukDVVfxhhwQdWMXfbIo8AMB6ZFP14q\\\\\\\/pZn3fkzVpDcLK42xo1bR2cdIWBDyHHd+2Boh0MAGbATIi7ITx8EyaX7y5u7SWO6yDQlWXge0QqdcRhYX2cCxdeb4VKlP\\\\\\\/TtqzhCrRM0AUBBO1NGTwQhu\\\\\\\/lKGhqBYXKkIO22bK9QfEcgV7ehhyPkwnd\\\\\\\/buH4Ckzu1e9PqdprK3oEgRmbbfN6qzmNxvMVtrFRcn7ACfV5b+W0FqwC3TXVN3K48F9Kx6Enwy\\\\\\\/OKDxCljACIpXVRbKg1NjE5yWRz7lNW4kotsi8BqHnl1bcl3urjB326erN5\\\\\\\/c6+LqObtQNlV4TCaV0T+TVrXQm81QUjinUYXtuhLd\\\\\\\/wiAh9lxS5cyhR+gPIBVPPngmKdFgSCgM4RMqv11OY88dzAwsjdMwYxIBIuMw05PwfPvIFPhOM5zGYTQs9LNoyqAyc\\\\\\\/9ceVyI6UwIo7XmuI0YMPY0Du6lBu\\\\\\\/VmiqH5ohbahtEe5XrJpt4fDN1yodL1\\\\\\\/asr7E8EDpMyJ5em88PKltnvQzEP1UQJDdl3wbLkPb8E4obbwpBhTrPSmbzxkP9MU2T+UG1nTSoa9lmBKZrdG9G4YCJWxSWLsOULM\\\\\\\/QgY8ygiNZGspsrNMc2hlfZI6p7X+QNYBD7257wIw+gpVFL0o5Mfi3Xrd2UzhGQzugQd6arh+xuHGTlopFKYU0udu1\\\\\\\/kYbP4Z6hIFMcLnbH29hlldVl2\\\\\\\/hsN+mT8FgqnDupPWzCJFeql8\\\\\\\/PYpLpDvM87Zl64PH34BY72u1a4w3rZcUaEogwQaADsprdQBmFUwjxk1RZZVF4u7JDPrjSUR6NevQAd49oAc6k2HtuJPph85EPCGhxuFCfbqPyc+8KfyvR5H15Xlb\\\\\\\/GAJOoW0OEG7pMFKZs5jvDN6EGGfrTWs+YRYK5D5KOjwOU0z0jlo785cBjRrYLvDNWGjK7lXX0MZ8kW5YnVpi1L8IzTeJwKGYGW5RLf9f3QQ9LHt1257D2D2J0Uiig7RNHNwRjS\\\\\\\/W1n\\\\\\\/rCz+hDk3BBdhCbBMnwEPOGteAB2h20MpAfIt+MLwE\\\\\\\/nWKDUmGDQ5q3ju9CzDfcwKsZ74TVRFpYPg\\\\\\\/2mxYSOKDSne7Lqp2EHgpBfv5MnNo3tsZY1LA67ixBSRlIUVycGofCvCyRf9oowgj8YaG1R28z0t5TjzYTxLyBnxzbBOCF2Fn98flTg8XBeV1OQmy8F0u15Bd9TFM5N1ULtDV9H1h0X30alGHEAK5+WRan\\\\\\\/aomUeKLJnW5y1yysW77vc0xStDTMBNo376T+Lndagcn3Hd1gDfozje\\\\\\\/mmrbsdCuGC+rDGisNxVD7UY+i7vDRTX3\\\\\\\/ZbS78i9w2WDKPa8NcXjDQdcZ80EFik3Kuhtkk2bEA5noMdHET1bImW16\\\\\\\/8XH\\\\\\\/sHUt4fH\\\\\\\/26Ape1jqSGyy3QZ5jK7uBcMNiONLzSLBjfBW+9moAbUbEx\\\\\\\/bvV4QxapxXKGWS7gty82zk1SaKvJr5RboGLC2B+RPIoCXNBZeYjwmPYTfiybBQGqXlUNfzipi+45Ln4SbjeygrWi9tSdWpmFgMLcbKzHte8Le63Zux\\\\\\\/Bg5L+G4iKRKy4jtbpiookNFhs6pUUVL1vxVbnlkriVZrartV54oqK35so48ZeEm4iocrRoLAW0TK4atwSG2bU0CZYFTyfBsvEzoUC0R0zfMfQuadadX0roMvAO2HjFEbp7jqO+s\\\\\\\/jay2Ntbh1k0qJygNrU3CUspvh\\\\\\\/0nmE+O1hKHXY9HGYhIFa7I70zOpBBv+9tikJUiVNKavSTlgCu6nxzBwxAiFM70jGSar6cIQVgKxSXVRC7G3OTEpsrMD6e9qroKcEjOVPf5gne8m7ESqkXeNooT+14Y0oS9LeeS8GYWIzXA6DnvUdFZ48FRVeloKjSdXkTjqz80R7UR6xKKvgycPRb3xjX5r\\\\\\\/RPLza4NtGet4SVckmIs7VCkm4RC5ywUdaoNf3rUe7k90ohdwwzrmtbgXbN\\\\\\\/bBImGCHlF0yeDnH2YsG4VwbydZw\\\\\\\/U3Y2c0q6P+6kS\\\\\\\/BMXs2\\\\\\\/BkSspEg97FCS74JiqGDZAhYkyJ94EfhdBQRLOOgev7ski7VWltSCnVRBIRSNck+\\\\\\\/YBtyvu5KBot+LqOD3id5ORsRw7+vieyqWTYKV+jnipr+ksCkkIBuBRuJ0R8xNVlCCgadYbH09sFDTEV7si0pIwkshrwBCbtt58beQwwI\\\\\\\/m72cl+0EbAJw3gTFolazaKR+3G7ujvGfrUjZHD+rr9t9P+eFF0Swb7DDx1Mk8c0wbYqL4NfaVh31gv73PkBW0M88GehRhF0VUz5YUfTUCaT4HvASdv0KFE9BfAxKNI6pAAgmAdNFplBv42rmbXSUv0PZikZkueanRUWE1hlEn8vudOHcwLWP7MA2mRiyZJAQkESRx2OMV9lLeExGmhxXi0CEjQnyDXbLIdaevbv7VcIa1GV1BMjAiMModkHcX5w0\\\\\\\/CV5vHdyikBv2cOE\\\\\\\/KsdreV11QsBdWFtas4PojhD3R+362xTA5IR25sn+YZGDsWhOdvHB0kOgm4uHBxA3WHKoquM7DjDoM\\\\\\\/ac2ga8IjNhjAODpYkd0lmzyejlWxlgLiKaI+pOW9Oi0qeGvulkW0KLGrxOSY3KLPjLZiWWJHw+oHrHCzrqcS\\\\\\\/v+FY0A4Cc0LwXNUVOgB2ejoC3qtc4A8xY5mT7C6ENF\\\\\\\/ObOBwm7x9dJQknjueM00JWDh5YFbdqd1PUgGepUlcUm7r\\\\\\\/ZR5Cko\\\\\\\/D4zTENsEx\\\\\\\/6xExoc75gaNgRDGHxLLb4K+B7BnNXbbig5FS4IXsa9v1UIzUwGbICGy4DDXWgeCE9cnrX8TstQKHL\\\\\\\/\\\\\\\/dA4nVJ6MvazlDS33FxhvXiq\\\\\\\/iyeAkrwVZ35+f691BDGxm\\\\\\\/ECurmJp9T9ijPumnGyH65l3N6EAFxnVHenL5KMV1ItanG57PJJspU9W0pGAIKBf2UzLi9WPgKE5uGVUF0N7lwxGSuAf5O1i472FGrwNBdMd4xdiqT+68ZlibPifmzTX7Gx3\\\\\\\/9JXg6LCT\\\\\\\/IFa1CQQ1SdUS9LjxAEqDEm4w8Pf8upghsliVGJBc1ll4dCtJU1ozSYluRGGGe9QOA3AmhAN4UwUeSfSZvg8q23PPUXqwH2LUeEpK3bi1Mn1vSsDjjXwD2bBfVcbYN0iI2wcFRTIcwOQAX27XgFJVMxWhLOrIgJTaYf2MIEGcqDlROsme2p9xVC72IVkiORSbRnEQ92dUh9qcpzqqWkh3P7D\\\\\\\/AJePL1PZWMi93zqCeWZF4I+Z3v\\\\\\\/ZCYBNCc9eBaLL2hoOM02GuXU6hTqni9FwwKOmYYx5V2ACBS0YBSJrnzRIBXSJWvW0ma5fNf6Tqxa62sZeu6E6Gi2YcgI8bPifnQQ\\\\\\\/MFsDW29HFFZqv\\\\\\\/NELMtGilhcHdzdG4+TDykPj7sjys2o3\\\\\\\/Eb90ZxliiBYxXfpQjV5A5i+bpaLqezEiqvxvjCAy0tFoOaNtH2tfPfHhBllcIhtdto0vb5PsTRNSTnycLkho7GNU1iPCgrIRW20iaj2BOv4MnMA3+e+j2O1De19pxG1jtd3iUHc0Hr\\\\\\\/W\\\\\\\/+scnLCJOYYmhbGU4L2Hoz63zT+x6pfTnqZfd1ca7TTG6CcpqhHc6IkSzj3UuRe5OexTdxqEk+JJJxpnLMAISvY+w63uaEcq7gi9pxcddC+yFfbkVg3N\\\\\\\/YnoHu+Xiv3oR35CC6yigwFbE2JI6KW+xi3jf5Rws32YrZFA1TxoBaXpnzDfizMHyCQBnjd2WkbwPc9Dik0lUk4HdvD1f\\\\\\\/42V8M4I4r8cTeD4gF5JxZWWYq0H51AbaGawFrAd6GYRequa7wLHZ1gSnF7JtgVx9ifJMpFH0KutARseN9Lks0K0znE3YXDyF7QgqlHS7dPQS1SdXzNME+DJ9Cz4AG0OvFXzpMEkYhmpDOsJpXJOQi56jVO2UI8ycM2hUXaKr2+6tJwyHLVVj4nIPzSTpE7NjQR1aktkZICJ\\\\\\\/+ODa0AXHHfh7CJ6FUg6p687fqPPrNyat0QH57zcCKXzQTfLGlBiw22hHx7NsudLvdyzbyxEPb374sr\\\\\\\/MQnMSIw7fvK2zFWlJKQuJf2540MyaMBqTBC8a3vX5+6AP8DUxAJH7c2FNSEy3C7kHnmn8s8TJvvALUtv471rrNBeD7UEeyq\\\\\\\/HIBbkf3XhpD3aEfd0j2EQ8R0t3UgYYR+fUz90K8ydFaihQaXWzZYBR1W\\\\\\\/gW9VpcE+jZL8lh+gnspwhxPqxAIrC+YGVnrvDkKMhtrcAzkmESUtT5KdASguM+lkvk19Rd40AfUSV\\\\\\\/mE0f1JpsXnqSutvgZVQ2rq+1Qz5\\\\\\\/2pJt3jCPxWR+6Of3uaDY0YcApTJVU0WwCFg3w+7ymVmy1BNNbYX0NpxmXIqMYw9+ggzLqhXMz+eevBV0ZBXHVvgyt5FhE5TQwt9Bnm6B9Y1OPFMq7G42MLwupgJ950tlaZEZRsZ4pbCmTcQrkPlOBOtklNUI+Uinv1jQIpd53O6It8DIdiYXlcGk+JACuEUfw8Ut1QbgtdKSt8ba0OuJmZf469oCGJ7yJ12DLDUZUJgtSA4uPPEevdEQNc6TeQNc5BrMlVybi4HT5BdbwbV5Bj4c2RzGYHSXNfpMiCx8uqR+W2IhisxwMoVdkXHLOqc2vsqh5DmQv82GbxktCEMh0TfE5QIUAjaQyUe5F+YNZttCOH4Ftt64TBIlUjbdOA5PmrGls0rceI6biKDarwT+5ReJDorNQkfV3GGSTcpQfJZdUoLXhKkl+QGvc1jBaJ\\\\\\\/H6SF9qsiGC\\\\\\\/ebAzRLRaN6JfYeiLashFTNlFdhGvNHXSLeuaMzOblhRnGHZQsNO0htJkZmDib9WIQ1Lv0RHPY6rLbGrSX17NUhyXSFWlL20v\\\\\\\/BCVjQ6s1vjMoLKhQojDiiPzr2GGdmYtKmKmE7R9P1QNtntEcjoHy9Mf86\\\\\\\/eteIZrEgQ2ulj79l6bsa84ot1xw4tgHn7ougrjg9ucd++jKy2JL1fv0Fc3TFjGIF04C6VnogXFqYMeyK877UlNF2JjJBiDdkM7Z5\\\\\\\/g76Y+7esctDLOMPx0U6tisZyJmrqUNVqMxWXexpRrskm+sHxBjVxozEFhYxhDqXRv5b+ujakDtgSiDlu0bZy77s2o+mZHPqTdnP0gZ1MF1eKJVurql29HLbLrW6jSDMn+XGiZOxLBTgA+z7+38e7ZeJ6GXUXSN4EGtRF8W0aR1f8r8l6FqAV15QuYuDPDid55yCvvHd+kVMiUKF7effw9pSo2roYTFrz7Rh1fR6bc9xXNHHOapPbkAFbAS\\\\\\\/Kb+XI4kFgPZWlQ4rqjj5sXRFDg8WJQ8IOhRdoalRksIQLvLJiDXlARc0ZePrjMZ\\\\\\\/tLBHnx2jHzSDLUDldYVm\\\\\\\/Up9NXCDj8mPhuPdSWVi785BdPUKSauniJWqjpbT3UlmtjX5y+pWlK4HOws24eHQWR8akRxNYEdlMA6uUmI6RudIRY71WgtigQIoflO+HV56DnPZ4QnpVknDwuMQXFUudLWvqV30zqEDXtd31ZT\\\\\\\/5pK7q6vdphVzztuI8rcNtGtHQdPpLKih5L5bxSYaDiSL8G098wXYRJowYmVWSgUMVILknhLBn8Eki5Gwu8VM+fUDfV5BfLhEX0\\\\\\\/kz7Oh6vUD9FPG01\\\\\\\/8wPDjASt4Lpp40Ni1JrvJ48Yw80AN5F76nPHLNCC7x+V3fBRfyzzYmtNPIOfYSmTSKHIaG\\\\\\\/TU1G7X52\\\\\\\/5cUCr6eD8rGxkWrJ8WE5mh99E7+4YUsUbGzkpTi3HfQmvTxVAwd5QqHdMreyG6X1Tbmu21OLHP1DFZOUTlnwnYpolAW5nDQg5lBEme2+9VEyQM\\\\\\\/uoaWgkIDa1zaPwwzAcRwILnkHYwIvHHpobUxWOxyE12oAGxAKEqfcK1SJ+OKuAGEj+LLGJFWAgScN69ZyhxOCWKfMUy8jqqLote\\\\\\\/uXEzegz90ZlHeYoLlDqjKEQCF4gj47OjK6xaHH\\\\\\\/\\\\\\\/l8JQQTT491StTrI0hCStIg+OjLfugmv1x6zw5KZf+\\\\\\\/o8vVLIORQFyBURm\\\\\\\/rySGGQxIjpHwvg++xkO2nssoCIUZlbNS9BiKTVNGPBKDOWwjIOb97IojTAbYj+7QNw3N0WIvAharngZ\\\\\\\/4QIut6Qkh0jaVwPoK+EJUMTPNGlzdGRGmy\\\\\\\/vUX0NpoK\\\\\\\/YKlunkqZvxRdJ\\\\\\\/OvHhAUXcpZwrZmjFd6z9H4or8TK\\\\\\\/vgNSV8xSMozNhPeatvgYHv1TBr1liMgRqiQ9nFnwDKgN6tyAjJ5+rTF5OK\\\\\\\/E5z1GcrGXX\\\\\\\/afHalw9pVlueSVY+NeIanpXuDEA+qFehy646MqcpMf3pCXklM5+G6rQtB\\\\\\\/GlNTjxMMZY9r6Ugz2vtJ5iLXctLvQwAief55ajyfb4Dgzp71GEOk2rc2073H7Lug8JqNcX4DcuL15rstt3limZhFchDW6iE2hFvrNZPVdzPUYKKIjTUc4Lq+JzZEaLsplqBnfzGyhx4Ya8C4DrcKalYKU5kYMEIPVsePTIt25IiukNaMKvi1v0U1OHR3rD2EwEZwKFMGJDxCepJonLNEYvT3phKmybCtXPTTI\\\\\\\/oyEFg9AouKsmxN2SfBwkuL9zINUK0Q0wFiBlDA0G2H45wEUNZzkCo0uqRpCmdIkm1BdsfJo6UUrvWdDwzJaoeFGGZcLateN6aipiHbDaVxVCkpv8eakeuOdMDTha6aCdosNnHQyf9N2AwCsgrAcJ777dRjftRuQBrXCzoe1uUEGZyBZShykSume1B5ypykXDrm8pd8ub4\\\\\\\/PXoRIWnIz71+nGP4mpDacDzs5NXNxxANHvFBxhYxaJwzYzVcWdvqK7g47YYp+qz+pTq31+TxSzmFLi\\\\\\\/rP0xgFZDBxnWVRJTBfLLB1AfFTe5H5p1uWSPiVlgRGxjkOVfyGL95eEIVP9jvdbTLnNSefrijTXbP2K0b8qpZeHTmZbu+AX+jtQhAce8BUVgx8tVsee4Y0v8H55Ixw08DJOgVre0pTRu8hmF6KtgHXn0aKEr2E8cbdKMRuO3QJOs3BlHkDNBttDP8WVaDPNWGcsurNeVKuLtmPtaINioSf3dLhFFIZelXzdmJMzScCFgSO3RqBRkNlfotdKbH9Utxtoc8ac+iftnte4Vx7k8\\\\\\\/LQVFR8zgrxVNgckymAJpEra1v+xDWPJsUl2ZV0xiDoVhZAEYWdfmkSYWiQgQzD5YyvSesjyvEVO3ymoXca5FzLqvBJ6\\\\\\\/EC1zHKOCQ\\\\\\\/JM7gZcJ9nyrLW4Modw4j52GZuEray64f4DdTJFJHlJMO\\\\\\\/UYynVM8Cjpl1W3HmabfDA40G4P4uBkw\\\\\\\/YD18HbApVt1Cq9h87tSAeP4z2mtGP7\\\\\\\/CpBkWHD2R8DI8UKT1x\\\\\\\/j839mO1LRoaBSkq0ujToHYewwbRLAeQpx5b22O7i8DPBLjm2f00m2stVvlLnsGTwYT\\\\\\\/PQoEStjE\\\\\\\/B\\\\\\\/u8CMCNuJVyQt3gSHYjV1x8p+Fay5lcUURAZ0DYnnZgclhRxyl8dQLV8yCWEamIF8E+MmweQ5SergOPKjzhYRfGKmJXAKSBCa+vYsUUmIjgPD6qzhIpXrrqvrg3YBTfZJ\\\\\\\/bMNN2E30Y6jRXQw5GboshEB70z1QmwbmStFv0cGrjUCVOpn1Ii5NyLrBux0NmIkiQwiab\\\\\\\/SvBmqTKzaC+pA0lDZkWSaThr1raMcO+JMKePI7Pv6zpQcoXlA4gNGQWX8rSVcWNZJ6TxBzQ\\\\\\\/RZBAascSWb1CU7OGXEejmPKCqsKZ\\\\\\\/2phQ2VCAIOD8Pks8fdZKnTxnyTwuO6HDcP9Xm8rUv5hdO4nBQxy9MOc4oa7rIYj6qCE+YWYihHl2zpobusjMUkMsKWIcJrxrykeLMQYpX432aH4+iu6X8RN08a2Uas\\\\\\\/B+qxhHsWPkFk5cHQqPb\\\\\\\/RijX3VgyI\\\\\\\/7ljfP22o\\\\\\\/ONgLuDMEetZKGIBidEmrQ3OvLg0FevDCCLeliiEpkZ1skhVez6HKxOYo9uCW1s0SrGalQF8QLX0aK9dtWi9adSdk4ZuQfcz6T+OqDtpEf8BZbjD1d34PLAloEOF6IGgfq21B53TGPU1QCi9cVWA35mclUnLAoZUpDpdN0EwCTv7R+QqxFvMa6BBRriyulEOQgG9mJ+C\\\\\\\/NCIeg3w\\\\\\\/+\\\\\\\/\\\\\\\/fIObr6tlraOHYnUTKovQkpVS7+fzwqC7KzKkBoudc4Ypi8LY58Y2yKXFpbypgQro3WZgWYqqStVZlE1W764Z55xIQlU2rMbKTIpwSeBLEaMR+kerHcF2X\\\\\\\/8qFr4\\\\\\\/BNSX6FbCLx6RgZrcJrmrwnO6Wk6hh6oENn5V5XNymsfE6mwJYZ8YgXNu9AmNX13jnqcD6vWLaW7pdQFOstpMD3vvwJlV5bUe72LmyztuMJ6yafPDB3GYqZiAB\\\\\\\/TFpNGlBiKaAZNZr8zE3VeeEK2CYCjcSQkxAT85Sw6niU17DGY98aLsuBJucS7jFAE+7\\\\\\\/ilyjCqisycFi\\\\\\\/OhShkbHrs+PS0lSxHiMSfa103HbxQ05OhLf0PEzuNwfPN3Kb2wnbhAESKmPWpGV4DSC9dg5sxlG3nN0DqO6Sz5gYgzPqlOiMVOw8xwZcO1WZiUDLFxVdta5UeUAMx2BIxTqRo93tjH4wjkkfc+ehxCapHu5Ec5fERPnju4HPMrUobeAZvf2zU1Bp7XM96fO0ZS03sg6y1+EH3mw7+6x46U8jaKyXaFv0n9YXdOOX4xYWiPrEKCJr71pqW30qGydeBK1KMirrcY7k1YNroGKsuWV3dORY8U+V53SiusutHD0J6Ryyh7uju5tOWt0FwqmA\\\\\\\/PZolfUfquAC8H714UUryDYjvqL9cCUx7wPqWjngFEW\\\\\\\/tVu6Bu5w+InjEDzo6mB6Vw6gwbtO7fKdI9CciXRNTpislL136XCaII1o5PnImWzAR2fzkAcL2VzHHSbsNH+tnGjCvI2GxD+Quv7fQ1mDR4\\\\\\\/7maGsuHNk8a9VFbs68JhL7+r8g02N1iKdcMR68cjkv23XyMGuYhKKMSHbM1wXLtg+haps3GdLlQ4aSqr3IUdkSeEntp25DdrACL3Ktyl2ta60bOqSlT9J7mNTLFPbVskDEGPnl10Uo21Zsr69usPi\\\\\\\/9z5eKexyQUK2wGytK6j\\\\\\\/RxqWoWQnK4E5J\\\\\\\/wMbfP7CuVPBMMv5xtpifqAhf5WLrhuH7g3U77qvUEf8ixBjPjT2OFbBzjff5YQ8yl6DLXWAmNcvw8N9THWI4sBEJkxWD2Q95BkZDwhAfQ0WwYPHxTXcaJMG\\\\\\\/gpTA7w3yNJwl1IlQ\\\\\\\/cUo18bU\\\\\\\/0MKJAC93g+xQekrAEkTDVrCGUGfz\\\\\\\/N85T2c1o+qy2F8XAvsC5piCjGsYhuBDO9FSOZQrvE9tw5aUHZ0o7qoNZo03arTe7FtvVC+6gnaL8d7JL6Z1XQX5nTOCxP79XKfkofkoK\\\\\\\/AsBe\\\\\\\/\\\\\\\/zf3GlMkZW3aty7wse\\\\\\\/a453luc8sr4DcfUgwcSKN\\\\\\\/O8u9VkF+t9mYv8h0p5pUYUV4\\\\\\\/cCgxitUna0vxJDmNgBVqzbPUq8Taw9D72pfXJ0PDfhhMDgpYKOQPgcaK0TPRUg4GFNdEkeIgzyedHddHENXvl7pBZfvAeT4asjm4zVY3u6el1OxuT8jq4ue6kpG0gqH7yoTvka1cVlDA0vpmSeMdalAj\\\\\\\/Ta4PZGw1m2NhR8G08EsttNyB3BUO6pSV6M1xIUVn7krZyZZDwmQ8KYwh8YDSkaPVhxG4ZTEppv30hcu9WXdJ5\\\\\\\/Eb6Ln2jMvMjw6wEs7AV\\\\\\\/4l8WTlvl+elkcymIJmAucs3hbpHZe2BvUnvBGQyK3tLgK55\\\\\\\/zJMHwpKBJtzeTWY6vZgN3TnchLw9hSOJcFpKf0B6H4Vahu2WbYaGqjfFpcmAvPmiPKwKtT6StMOtYZgqVfZElursGDbZGry9YYBbD5dA+CHFA06LRosIkpNPr1BefPXOyijxf1It424lDdIBhEmUqfgUbNSO81nZASLlt2uwHUeMrL7o2gRaA8ej9QPBD7mITYobCXala0Dnt6tGBv99Px9buhQGkDtj7udUgFoYJEcldTrsZ42hknfmEBufD+a1L7rbmOxXMx1i+rTtoki9PEhcYRwHkmJctzNy75edJg2Od3KzcRV\\\\\\\/Lpom1bFHpk4bxPirQWcQLhhhE2CJqWse2aeoMxYNFVVnHUmeiYPa83El8essEFQhjx4aX1hnxcQyYJamUOA8XDp7li0CBNqZZo7igAOt0wG13rdVB0QiFotNh\\\\\\\/jcseObwb+vVDXydhafAkHmOSJ6ZqbNQIQmVaCC6lKCixaKwu4DXbJG6bN8ILVv+p8z7V\\\\\\\/HSXb5smSGvoH+I1YPNXeSGwJfBvjN1DMprZ+HYquXkc54nW8JIfUd9EHTinY33w5Cq0AgL3l8MynZfGSPHtZAqkZWoOW87+1Ek9pPmN3M1QIt0BzilOUW4B\\\\\\\/LIphXmpe+SFmZtlXsETpYX6ka5jjo+hbPoIlYSMgYIZ2WJC5cd+gXYfffdU\\\\\\\/uk3Mc4yj1DqJFysnhR4\\\\\\\/1HPpaWV2EbVztHB9kkNlrtoSylJmKSraNf22Ld+gwYIT7FPv9UXw2yWSPhfmDTWDFoKByLrvDxzv0vL\\\\\\\/FMGzEdM61oDStjbHo4u5wVWithLwnOnztRnn7F0u1G7ct1PndYSjAiBHOCceaA9oh6Ju+ambIaazqCEDRUAzleqIWLGBJSazPp4jIlpp7KlXONe5aCWo0nJ3QUMn5kHk0Btw\\\\\\\/81IFXcSRM+vpraYjwQYePLqV3USusCW84E7yBPkhr3UMczVRbBsK5P0KVOSNidqESgxtxHlpn3A\\\\\\\/s0tCEs\\\\\\\/lD28F90AISm7+DOk96Xljw96eioHd7zqcfgb27xCKgBBAMUOq0QiHFlgRFPhMlzqE6V0ErSnXOZN8+hhgAORf+PSd9eiFdF9GDP2U1ueIVhDd+hj0y\\\\\\\/PQ\\\\\\\/Hun8U8WYVZbCx1lLPi\\\\\\\/z\\\\\\\/TFZDfnNTI3qMp\\\\\\\/EBAxKNxHZxsEChQJO0\\\\\\\/0wwhlolo0Bkhe22fsC15Hz+G6hLdg5mnCRCwBHUyoHSsCrbW0oLm6omARKZoqu0goam5EMgyF1vtA2s9XjcSFmEvPBoLgCpfop94zPnfhTPCv6AuTKgIY4XuIg4WfGUxc8TKdXiDvkvblP53QP0t06zpM9ywUwRB1UnpyW5WnnbZq0cy+hVFdMrIKSqwK4vGSX1ixxANwtOjnaQqMGlMWEJ2uSt7ZLvtZBdq\\\\\\\/neTTZT\\\\\\\/KalYcvioPUo+uamgsTVtnVlIDZaUnE4mIXSSBbjrHblCq3+NxZ\\\\\\\/MNhdgdtQbiy70AvQRt9cn4366+vOXwrMLL5MjFiGnV7VIrgc6zFArRhsdwi5q2YCgU+1lTDhwVDskrtv6KZ9qMtGx4B41YCZwIHX\\\\\\\/5OweHtSXkX+pxVQRgLi1rrZ1NOT5cd6ZURf7rZ8oicLUN9P53m7ZUQXoyVSobgidDeETeDKNfV6+fDukVQL7HDvjPfrOTJZESOpi0Ylwtb7sHbixgn6SHjE1jMFkjt7sFHbk9O9K7DgyCSbCkJQlIQob8Z6owfP48Jdbv\\\\\\\/yqTYguEPt28VAGc+DBxB5qgvwyx0n5xmeRktcOPdX8JL+LKOr0YRCXY5+dJSiz\\\\\\\/kiTJjHiY6kwuy0hLYYBOhioLNdWctCIlJfkHG3hakf7Dux3fpLn1lmnawJEMNNcisSWHp92DhqSe9nbX6zdwT\\\\\\\/xgfD5uVkuQJqL\\\\\\\/U\\\\\\\/g4MISfLPvswih77ZYv15h9UcJwIAVi2HXQp6RIvC6snRqEFM5wUNotcW\\\\\\\/yWzfcY\\\\\\\/I+7eVY6qtLsdUpvcmsglm1rDebL+P\\\\\\\/+lb6HDe7Sgw1+vDk5hFbMOP7wSCgEpqHtVkeQs\\\\\\\/vMZXv6+zCgQrFG+oWoZwyg3bVYV1fhSRhSl3cBh\\\\\\\/3rDxicIJkyMLfhglLnskaoq08rKG5yrZRQO59Mliv\\\\\\\/DnZ\\\\\\\/PBC2rnFFwiTSEIrPb992PlkBjkkYmdEIfKib7fNPBhKJ1yfqwNRyiJh9L7Ey+Py86ONkv3kd7IZfxl2daa4ylvTn6X90pmk2PEipAjRGOlNwgCVF45sbw0VDjIPEZSwysZLUygMHh\\\\\\\/AIPzyRtijllNqeadvVCgfFKfiwDDvmUOWj25EjYMOh8IDbSpZ+jfOhq+WkXxo4Qik3192sit2T2JPP7kjG7YjPGxdLJkBEAu9DlaPRv2TcoQZqwC\\\\\\\/mmjIIlVs\\\\\\\/gwLB9wzVc5YqJro637+xc\\\\\\\/RbISbDthxRpjdEezF2BWsuobSlh0YQqxekUQ\\\\\\\/DypTKV8bCdULz9VTfhQvlysPPkjWKHwLLOnFx4V9DLJ+QrJYMsUdoYNmNR9+UIo+LmJN5tD3hYa9KWk6MXqJtDzqS9zdOAa3erUgLdqhD4Cfzd+QJdPt7P4b3j1T0D0difjY0I1X04Jm3rS1rIYeMdocU\\\\\\\/mG5xsJpyw9qgfGqJzX0DqR+S8GJUGX3zC0S+SC6n5Q4HSilkJtKs3b6nAANOiD7fydh5LRTxaWNpUwPGKPQwVBVhfY1laQJ58AeO6kas7\\\\\\\/i6m4+WkRAQSkwFM3bg6dsdQ49XAjRpzPO99yKc8UDtMrLAXuIJh452vF6xhTPAdASGJf5UqykNr1kTRty+xzLlVgBeeY244maeVrjebnY198PsL5y4FflOCkf0c2txVl+6N6V1oZxiwkxM7KSnMo\\\\\\\/x\\\\\\\/NK9yUR+Q2rD2VCpVZo2bkkYrw7gjPvmsk1yCBjRFtFNc0L4HdAn96kcaOz6vdxEXdNvDncPH2P9KEDn\\\\\\\/hh\\\\\\\/HgQa0oeoCDus1B+jQpqIiPEhnznn2aGIyzvAZBRlWGnPSak3VOO0hueqqaLU\\\\\\\/znu\\\\\\\/DVwKSwTADCxzHQ9qnGuMmacZYBgtjmvjUCEkJoPm7wh0Gp9bP1APdytGRQuzyxGYiMzrbqLMXGHBaVgfVJ0mWRBsHq8Xqk90KffJHC+b7qxjND5hmEe9MYmM1XD\\\\\\\/OgxN2FoZWqoDCpfEp3YV8gaztYlTHoSKd2toJc1IXzPK4eVFwZHcT9UB1XxQ4EjR1boqsSbcjrflueUarCLVs4QJKaW5\\\\\\\/D8Swlu75Ef3oR0Em83Nfc5sRqVFjlMYmbSJxlv\\\\\\\/74zqys9wu1Ycf7JvT2kBClIX7Vvd+46J6ur+cy2Mq2YyGVXnjfcTOvqhMtYeOMUIHPoZI4sjKL0DoVpxSPv313Q5ClOf1XEQyU1WTM7mO4sbqz3IPXt7wa\\\\\\\/AfNIP9dY1ZVgptfUsZH08FI1HhOzHdUQzhbMc3jKejx31pmiK+SB4+RAfPbFkwzdPfC7u40xm51qMCYhcdKa8g3KOvlR9SsojEob44WZGcWG7cAWWBYc0W\\\\\\\/0h0cJhmUigty+LgBjA8OwJ9v\\\\\\\/NZ6KqWnYpmIl4tG7y8NrXwIyl22PTfrteOuz2AI\\\\\\\/YN0t8tmjdJoY951XCpKxvQUAfAHMrn7X5xFw6u\\\\\\\/QJrUkfD6fn6eOkLAMNpAehWs6OK8fPtAskwbjm6av3\\\\\\\/tZeunxtDA\\\\\\\/ilp1JcbHUkJM8iXd3FhM9kTwc\\\\\\\/A6PnLa7Zm5RYK3MkJ76M6ghlEtRm6gN6Pe6hwEORixwv\\\\\\\/5nlMXevCVSHlCO94UGHFJvHG3UtH1xlaXRxzq1HEuBG7PCsuyOl+VIH5qvvVHfptFWC4NFXlJylFLrZvqT2Gnjpy3AV0ueWePlPzu15YFPwEFaFEWpYo3o3CRZesL2jb4I7Zn+Kckx\\\\\\\/XnGDuKvSKt6uPBjAr0CwIjnguRcslXJWC6Xn57cCRGBQnbECdQnDV91nQIEgVTi0SYPw1x7MO1tucD8yRsklLc4mB9S63ASJki0Lp2xhlLvePFHP4EvUqd9SgpRlvx9KfER7MwQgAIFessXHYT0ad24QHrjTH16yxxSZcR0Cmjry8nDTzAb8NCtgBoWBuDbmpLbAd1OvpWcHAryrubd8D3XVmInFBcj1oowpRE0gA64Kw6qZivDL7l\\\\\\\/zruE609NIXElKPWW+0Svtz2YR73b3sfNXIFE0BF+eFFNkD\\\\\\\/zTOMaT0pMbdzP9nXnCBEL6v+gW7SxT40jcQ2Cj5OhPSmR+t5V8fMGkGuPe+2GylC+tIo9re6agY9gW1XcgELwJpGsVWJ7JOtGL1ONLAhm1SUSQBBFd9XJuRPe\\\\\\\/RxUFY9yIfuMnY+RoMtshHJEMpMGizjlUqnA+kRI8niaY12cBePel+K74PIhDzlCnQsomqrQrT6MNALa80cXN5NtUEEjG9B\\\\\\\/kL1PgqKu74GN8ybc05CPPGsRlPwTVYPk1nNlfOoPWPZHFyycaj2P4W3HCrLkc5lD+rLBMG02p784aZl1RaTKhcNMmU2Mug0vvZkrVzIuk5bFHSX1e34p0gaggSAytmFHt7taG5OduwDBNCNrcua6Gd\\\\\\\/hnhIKj3GFWZtQUm9AytSn0YG6w4D892CJiL1Lz07Q6AcT\\\\\\\/YXjW8g5j1bkps08sJxJmGWLXPZdn89iGrI\\\\\\\/A6rvf8FAULfpPzqX\\\\\\\/mJp\\\\\\\/9eZ3u2YLHjbGDTjWfYXlzU\\\\\\\/DMn0ICT71R3JwQF4CBRvk3FSW5Zo3UXDYPqHrBThhfvO672RTdGjfYQPicaUYBqHba7hfdMG9OWd51r0wsi+8\\\\\\\/y\\\\\\\/htkMhJgXm+UiuClxCwQIct4hMwafwPuNunv9f8JkyWy8AIEWykXtN38qP2yDczYygqqEOMvpr8SrW9ZOFXiEsX9UB8ktifUJSHZdADMMyECvM0cPKkZlegl\\\\\\\/mm1biuMr\\\\\\\/YvqvpvNVTo3zxOLbvcBqBKf4xbLBsiJPx4zFLTntr1iAdl27Jvfos623BkYE4IVX6J\\\\\\\/xa3jegOPxVUxtgC2Sm0STxc2FH34UG9kt9It7p\\\\\\\/\\\\\\\/evvQ9l7eSwkLCdVatmmGyUhdL1io6CBLE+7Fm29sWpX6Gjq5PpDA3GVD5YaTogrWlXwjjha9EJd6ibbIO5rg3nDYxsqg5RqPgekcO\\\\\\\/uxCL5xXA3CyJmBXmxbfTQPAeFBfZupWlcEPPeHYCSOyJFAqY\\\\\\\/R6oFeYbx\\\\\\\/Oe74MLmRtdTQQDssdLKe5vvTjDgn04nfFa2oCyCrBdiTDIfpgidrdi4g+emfjZ8VADJhdCV4bJkh4UmzGLifGfqeB7xtCE097W0lRIQyJIKNimwbS273ljSoH3pNbQJol95d0DvEm2muM8+4y390HTCFrbI0xLviNNzW\\\\\\\/9pMBS8Nk86wzPonjDeYME6VaDIHnLKeI1yK+8mFUMgGZeicqeaDbDimO04xXRghy+6VKGHo6FESLe7NVdaz+Bf01MeOL63uqPj5jzKdwKN+inzIxgxWcH2lHlsXz38JVlOzLPwyo9dcCnp\\\\\\\/NahbOMcJxe2WEcvYg6S1jdB7jnNhiWq0O5ncqKNWJ9dMO\\\\\\\/BKIICD8p3ODy4ahhO4VrKD1YOD0vb04xAEu5MlGJaYHmbLQzXDLkW8d\\\\\\\/FlpVHdABwZzsHiVqKNYfKUOWmWreXJ3HshWoZwumvuPyQKYozqQKKDSi\\\\\\\/tzIcQ3rr39ksVP86\\\\\\\/DBV6TnBYlYNcaE2SywvDF7irjnxCE8LWmebzwlR9YGgAvs\\\\\\\/YHiRJ0zfGQvOFJv\\\\\\\/hCCnWDLejSG1cnSLuc\\\\\\\/OYofOxgR1Y2xCaDWRAzgqfGHsI+nU3+qW3N4\\\\\\\/0aAJCeo77zXIvN6PBabkny83dQuoM\\\\\\\/MJ2CCg8jAy6MnhHRc8F+KJ2D+z8sUCE5TIzDT2bgclYidV6i5REmDRLg00TK\\\\\\\/mWqLgfZka7v+5J0MyMiuHEzYd1HZutVm94oZTsw2o3a0Mp7SAqJnN07LihCDe+mb1kTFmtaeZIPW2nm8YKxaRyYTdf7+8bAJr+OJGRnHl9\\\\\\\/jvkIG\\\\\\\/\\\\\\\/aV9AABf2\\\\\\\/r3nU47U42yigdk6\\\\\\\/HJxL4nRUuxqewz0vO1ffydnzKmJxerID2WSmzk5f9pD7q1t0bQ2RG3sWLt\\\\\\\/N8hwGPYV5wNTCe1AnkWTQs4kCBU7H36zOlwg9wp9f\\\\\\\/TO4Z\\\\\\\/\\\\\\\/3jrr+jjsUX8G3mDlS5VHT8cpmeoyMd10rEeesQMKq1BbSLh5PLeYMkj1QMrLJ6ugIsCozALQ3a8S\\\\\\\/d\\\\\\\/\\\\\\\/ZmtYfcnMwcGonh+w6IiY4pBBg+IITcsyKVoWaFRMI+JtIZYiLmqJnX+o7R8DF\\\\\\\/D8lnpQXrCp+dD0oLm7V9VwcAH4es3zlVCVEB7bt3i7y1EG3+mWWwSs2\\\\\\\/VSBchdyxVpyEY3r3LS7AxAiGqniZE2ZQNPE4NoKgHe2MSl8RYZULWnZQmospBxVvrxwX4JV+ZDDF3lDWbTQsCjsURO9UoRJQ+0b59I03aDLi273ph2PC4v4gkqLk1DZa4RWbbAxlg8BcgirVqIl+KCcY56MTR3u0L57Rn9Zc1rwBw4IRxawTo\\\\\\\/nRfrdgQ0VSpu0nvjAfhqtdYhkmmRQvN+jCffRDHDAK3DfYUyXGWXAMaHZP3AG0pvhTl8ONDO5pg7UoHSVKJtRKLifgAbnhIHyZahs3Uf5Ics0b5zk57jpfqrANwjivfBjdXL\\\\\\\/PRIF+fn\\\\\\\/AzHIb\\\\\\\/F2sMF+WaOQntL7zjqXG2unMehhcR\\\\\\\/DXRTdhDAk5ZZ6EqyK43cGAfVTw+mEOTangVoFDBF85EBaA12rz\\\\\\\/b71DNOakgNSyc2jslefxJMlMHS11GcPsr88n2grZdZooiuXSbSRPg\\\\\\\/OUg2\\\\\\\/A2TbQJLJiTfheuxnkX5JichmM9Uir1hks\\\\\\\/\\\\\\\/D+vJl+FnZW30xCi4pqnytP4JwX5qOlliJATzBYtHFI0UDWEpXKePYk2Lx13AcF14VTx6dkc+dq2UTBykUrAJ5JxMHvBGTcMypfblLBkrOHmjR8o0BdFIc9K5It6xpq8dfHldOkFXW80MU9qGG\\\\\\\/qhNt8Dgg0yOvFYDh\\\\\\\/5udoUlxpmZMECpX5ERZadjvMQlqvCKxnjsEjzNQ5E+\\\\\\\/KpjzG5S+\\\\\\\/I1CLUnjU5c38RiiAR61GUUyuf8EM3EHITjIRLAlO\\\\\\\/A\\\\\\\/3ygLNCGpX8LsP\\\\\\\/5b6ng8gxTDS0sBFUK7ybDSr3pbQk+N\\\\\\\/YW2e67IYoXHLCbSF7MkP\\\\\\\/AUmb90M8QYtbJkq0BS0fI5jemnlP3\\\\\\\/yRphkJ8lE5gx3bOOLOFftI+bs\\\\\\\/bPdB4LlUUpyAP\\\\\\\/u9eqlSs6yMJU34lzREDo4f4jjnM+FkbhhD0JpK4vOxLlmQOVemlaGYz6mCwQ+Ngj\\\\\\\/yb9awh5pPnZrnVZBZ5rkEejquggK2uww\\\\\\\/kSWyAaIpW3tK9P9cYtQ9zo7cQs2biDYBUfJhyf0VVIjNPhO2N7sV0sqhxYd1B4nTJh+7WqcJqoB\\\\\\\/M04n\\\\\\\/bC+w8nU9HtxR3WCMYlkMq7wzKkAEHJX\\\\\\\/5tW\\\\\\\/pv2cv8cLOR81KDc3Fre6Q\\\\\\\/Vc4g3k8JgzDaKAnajhvktHhXZp995X1f2foGe+G+pUswPu\\\\\\\/FUeJ7LJ1tKk4Xnxb9G00c23iGAKmLkWne6y9xsdMp0SjiFuE5xXsy9\\\\\\\/g6hmabNGi9YwKpNzE8rbf1ZUMjX7X5TDL2VQktPDZe2gT5Mu0b94Yavv8toQjQcXgF3BXRtatIxdKCOKPj8vkQyXf+7CfgDx6tHVEnSXqnPAbrcqHlI3BleKb\\\\\\\/AeCdMst8brXCAYEFv1yr1Zr6vTebjmCkTYUSHTnLmgPUOIY\\\\\\\/8b\\\\\\\/VGpV5HeAyHM4mnnvmYOsTXhZtAgkk8Evz+ctf00bpDwyJFZM3ojvm0mmg84eEwzGSQCFh3pbUxfkfAUyrC2pZ7lpaYPWx\\\\\\\/2x1n4Gip+reJWdJP70BrffpnrwVtqxxHpiL5PwPVT4VvgAA8YW3QKd3o0TwoNTXl8je528vyyxFilPbs1yqLImAz5rXzzwrGCUemIAlTXN5ag2R6UQkAwpzB6BeQwmPMktT7OX5ng75Ubk2dfZ8wdRUe8xC04Ytxj6hRbhjqg+kz8UZ6zFrszf\\\\\\\/Ksw7ospTvAAjgb5kbWSage+JpQ8PoughnNoXeI11aaoeOCsRyvQ\\\\\\\/rAN6Ms0G8H+lyxGr24cfYlAofKfpcLZa0rUByO1d89kOJfEaKoCWfUbXQm96NIX8XHFkg7d\\\\\\\/GVFEK9uo8RiWe9mnJnoFcW\\\\\\\/zf3pm3KemTN1S3ACx\\\\\\\/3g9xb8RkYD\\\\\\\/6MLc8XaQCOxHwojdHR3Bn\\\\\\\/EYRkefYXApQ7\\\\\\\/MGNMS3IGpLQsEf68VDHKPIh1R7yfqCE\\\\\\\/EIAVX1HYD5wCYzJ+EqpRrRgnAuyMT4snkCsUiaLCxAHMiMITDEKqjMuta2d64P4NDoh\\\\\\\/wUtruyv39hkdyZeh13X8stNBBs23mB9+xny+j5Nux1DGEV+be3KKTgXbYbQhUdOnmXBwN+XI3Y3cVoefpp\\\\\\\/4bTdS51pFwoMuU5PhMoajfw6VkfrV9OMV+nPCEfRRQa5tzHT69jgGvSVR3ODnwFiC4yvjzMitR+ZIAMD3+AMxFLE+swYTvQ96AASh6dvZCdWuRdWsWEuaPsQLrdPX4RH3Won58kaH0xmZlbRVT3q96N9LPo4gTyVvgSEOK11iO8jz\\\\\\\/5xts2M+\\\\\\\/FBigC\\\\\\\/vmAEwZcPW7DHIyGD7NfO\\\\\\\/w7bCsBpMqtQ34bo0+TCHoT2wlRmrC90qTcCAH5R+VgTXslyVHeGLJ+\\\\\\\/lTj2sNaULNFV8SXFfoWzIdQfayy3F8uKtJHcr8Sjd3cVDlEJEBMS3n92wjpeQzyLzm\\\\\\\/CPaS00t5nypQXpqlMZZDEcpo2JNhFXtyRg8LQxwbPtvsewNKl3j\\\\\\\/tIAOjT5\\\\\\\/hNudxnhJNOyHA9YyJBXGIqo5ml+p3rCqccdEQtYWS775OWz2xkMzB\\\\\\\/1YN0Wn6oTH2wz2NQLFQKnxrOw+KZq56mSdIrqc4uWRfyht\\\\\\\/AHNGlqvAANuU\\\\\\\/I\\\\\\\/RhSqVczRfOPXU7J6Kf5qIailSsqM9eyRTrxrWbPOMr9W0EddODq3zJ1mh3hKd\\\\\\\/6dK9VBRzVRkAuTFOq1\\\\\\\/XUbRnRksgJhFVtVAI+0i9fPv3pjwpIRrH\\\\\\\/PSj8HCSTX8wX8XeH99WS1FTT\\\\\\\/C72Rh8hh7XwQIFj\\\\\\\/ajaw+zK5sokNfRS5rUn6zHkqhL+FpKvrHVTbSTbosM5ki\\\\\\\/YtGctVdL5r9OOMb\\\\\\\/hkaJs4HhmLKRNFrlnTVM3ZH\\\\\\\/WEjwPCefYXxtzSlZ0VdK99tbSxjEI6rLHN\\\\\\\/FWI3JSfdru1s5mJCqk9NScXm4d8sDm1on1\\\\\\\/ZyKqoZAviFC33B\\\\\\\/kzV02QfkLbUweCqQt3\\\\\\\/DI2bW7\\\\\\\/d3okqE5iwO1YLfGqIkKG5BjGH\\\\\\\/h+c2yXBglhUIyQAHTGPrw3GERUuLYSZJNEB3ABIqUfxPR5tjzBfcQoRCCPnWSCjqiW4g8ntbflCtZE11gV2FJ961cNgZi+FkoRTTzg4ruCTm4zIrrJxaeq8++0WBrKx4RBdXbb1K79Fs5ee+JV9AwRSAhx87NSWIW5Oi68YcJkuth\\\\\\\/weuaK3unvAFIus+v0TyhHfhWaBySenhnVCjaXJDVvnOYEfwQb3yC+iFrXkz8VvPZHGcNTX27IK62WjGZRH3\\\\\\\/Jb32E0KWGWHLw1611yOvMStGWd3iopaHwBOAwY\\\\\\\/27fBBZEijCEGkHyNl8DE4RcOERn0X9fBqQOyV\\\\\\\/YRsDYKJQK9Uwdj+7BvsWrnIH0LNHmhI8w2M6dPy6QAoW\\\\\\\/jvX+\\\\\\\/9CmBd4ezYf2gEtATLKjZw7W4dvVdXykHsrB\\\\\\\/ZXDs5KHNVlrCFQLGLagFvBd33NvQbb8GBsKb+NxVeKxVXTD1VuD2H0idCW1zsLnv7VDdPjo1YuosLa9gOkgZYkDDpvAXtxZpdmQwi1Oayzwehqqs6VSIFilhxuktP\\\\\\\/iqd8NaAQfCU\\\\\\\/wL7jjv8v5zAjTJPZgKbbjlKunZo07ORLPxGxS9y883KYCwIt0jgoE1hV8Nac4\\\\\\\/qBby4mxNKF88ex+RWKL+D2cvL9Gzf9CelF9GlpxsMyU6U2OFuz5Tuu1uGRJ0tQRbd1q932X8LK8KeHcs1\\\\\\\/gCcowfaz0fsSZuFK4KunXH2EFefK4fD6EzL6nh5XUK9jSlSDOrRBf2hmOBEomjcTrLfTpP4vSC5jzBcY2rlVxSvLn4JZY9LxMolyHS9mT\\\\\\\/8myOkIIHlh1ByUX9pqQzytmkMhzWVmckYkfyp87D86tk\\\\\\\/RQrxMoGYlC5sRM0gl\\\\\\\/DStgsNdTF3X0zQ3pdTHlj0glGqNeO4Fq3LXNdB4r\\\\\\\/aNUkwXK62Dd\\\\\\\/vo3U9dwRfOBfsZimZo4mcHu25Zn543NDRCwulukJ6Nq+y4fYJ\\\\\\\/f03jUs\\\\\\\/IItv41OZW5ZgHTxnMBunC8Lk796MgV0R5yKf2klXJT8mgSILnHS+b69moAyUcz\\\\\\\/CbhnayPcOirAJWJ1uEecf8OHFcPKwiM\\\\\\\/GD\\\\\\\/RgExYNwD9Lw\\\\\\\/lx34kLwuvNGAMOtnJrwebEzCBVMb\\\\\\\/nGL4yP0m\\\\\\\/ie3foSJiGYF9HrAhpOSygmPZIPWkvZNPcjO+lIAFWG0Bw1O09IsGrVrNVUgiSBXRIiKCLsEBM9KvFqxmvJzgJq\\\\\\\/Z7D\\\\\\\/z5mM+URnd0c4k0G7KK3xBNbgcO0EdBRMdnX5i2xyWClgSWBc2\\\\\\\/h6rkxhlriJkuiZWSTD1gkmtjR3Egx\\\\\\\/wRGbrCm5XZbz6yy0cWua+ncU633WtZ\\\\\\\/02ql0xs88gnvW3ivm8wH7my0Fs1rEAJwT6tcUXZ9r3ImStnqPQHnHFXq5emZ+iG7zYMXs\\\\\\\/uC6D\\\\\\\/RhPJ5lKLzlreO6NBOu0XWxF4FioO4lGBqE7TT7p\\\\\\\/3u8KSOJLZNbMNwnr9\\\\\\\/JQIVIaqIhsv0y8V\\\\\\\/hAakJS+EDKuj7DrGB1oU73YnUlRhs1Xc+0Rqoj1j6dTAt8DddT1oJzBOjB02\\\\\\\/WnH0+ubK3zJuDm+5y84A\\\\\\\/uGcrkm+ptIZDhKWQEQKHerDKufMY9Cm8jCK4evIiQcYDPlJGEICpZnEuMAZuh0yaxKG8aF7jJkY4kmNs3x\\\\\\\/3q2uKVxpgk+\\\\\\\/Uqll4oPerYsIRTfetjsOt2Da3J8WIZwj3ieL6uRwgGWDParm6Z\\\\\\\/h+vdMfDdjNivTvd1tmt\\\\\\\/IaIS41NvfdzYWvsSbfDyFpVstGR4EoyeNEElp1Ece2gUu+l3n+v1jd3ye9rUpbWGLbPTCm5HhD5l5fwM9g2czv4q+3MHuNNq2hyNwJ5iiMaBuQHns6qA1QPdOF+kwID2ip6osf9XddfBznR02LLkYv9Z12DxHZabSC8godzmxUQ\\\\\\\/0LmoCHq4WcNiXfyeVSydeBpmeXDaGtc9AuSpm5IMakocS+4lPaAHoS0XJZhhWiZf+2n\\\\\\\/WT7hq7WXjq5do3tXK4OH4sPgXvVA20BqAWBtqRLQPklxNRWwUJ0HkK+lfRmJDzDBwuFdwDoRnx61dVWLPB4cuJ4y9eM2qtC5cQ6jJue0iiBSqk23+BNGso0ixhxb+ZyHuVfYJFW5Ce4YNHWVFThKroUbz\\\\\\\/t4YiP0x8mAHuVaNaClJJ5xisa5c6v+Nn7xQ+5IuHi1lYuBFG+1FbpzDp\\\\\\\/GL+wmhFGkD6EbgDHstuR4tjwUziTr9pDyyAzoQjScBv3gh2eei8eJ2zon9R4JeKh43IZWZARXokplCv1OgYqg0wzkMJbOkYRmmNGLS5Bdt6PFEBVMvvTeTwhDtLSVfCfM\\\\\\\/WUoKSYjYYyWe5\\\\\\\/FwtUsFf\\\\\\\/g+zbPFwtkslkUgY06r5C9QH7Vc5LY8AIuBg0pBn6T2qF2\\\\\\\/XMzP+1ADhd7CJaBR8A6pJKZaxSIiRhgDFokwSEbSvY8WXnVVNl54UmwBBkcqDsEpDyFY\\\\\\\/Z5cwhH58l\\\\\\\/Nvu5l\\\\\\\/GyZsNhbLtNXSgVwiu\\\\\\\/pddTA838xal5ByhGH5HO\\\\\\\/2WYzneRX2qhUkNKQUcgzyNyHcpgM4B2cHu9BRUZympgwVGEVSs4A5u3VpEW\\\\\\\/7Qj4HX4cJMuFtS5pFjpOf46UYjEP4t+h0eqyPCpI+bceHk9TpV51SL\\\\\\\/oLT\\\\\\\/VotenAhdrkaXSLleH7cJXZ19jmRTf1MATToE7a1631DFz+3RX1soRYfmWGdYPuDmrCW0aBvFVh7LicfZPTiRkBAZ4u6eur5r\\\\\\\/+4YdDfs2EMvSxerWtShC+KoTdcU8jf6cw\\\\\\\/aTiPzvpxcGzkqpc7ytQeE0tFfahMp\\\\\\\/OhKxoc53kwUmsxKLz65\\\\\\\/PK0g27u\\\\\\\/m9aWC8+ekZ1y7j2Y6ve53qFV9Sgkyc+pLsgpoof4SKfN6lS+T37XF8nCAfjecaQHl+Js3BiG9jNBTE01qhaU2f5y+TrIz3O0VxLpEbTv63agnst\\\\\\\/ZXfgBSIKVL87D+m1PBgzv2q4XI3bJM0xKLeILJjDTI9gtiUZ7qunAT+Fke8rLayfVistIeqTK257IioNSwv8OtVmIcscQTHKXG+rpBfdNAbCA1tyml4H+xp+pEWK4PV9OnooHb7XDuPMKJtPjm9J2\\\\\\\/0gSUyH14wGttwqrlx69xkOOQgcpxGeaWb+sZ276n\\\\\\\/EmgzJ7dJQ8bPsOF+WF6Ad+0ZAQunoHK5zzJtgyORm6tYDLZ\\\\\\\/hv4KcsT9aGhhUl\\\\\\\/6D7sxdm9GQfDVuFrVWr9kYikmnV9kbtr0GSiRb+xALGkb0eXKE3ZNzlqQlJupaGdO36XhKbQwaq1X\\\\\\\/TfmgO\\\\\\\/fUkWza+SUZJ9g9sQS+iu6wH2yPLgIzneumsByxFtWbleBVII\\\\\\\/Mn\\\\\\\/Gc9eaO5CSW7BQiwv2OehAJuXVbC9THrlK72qi23DIi3q9pFmC0+2RWNJ6TKBCYhV\\\\\\\/+QxIMz0GmxW0Rh\\\\\\\/mmXPeUPSYDNt6vC3DBsh77JPVF7J+mE\\\\\\\/k1ZilUH6XMn\\\\\\\/egJ1nChHqmc6mvfe0lC8QTX+HIUqewMkJ961QrDY2MLGUQl0TWv2GiHILi3rcNyr4Vn7\\\\\\\/kHH11R+IDq+zzK0DWwili6d2p8J7jcz0FatgsJleZhDt5qb4WwFQM4jIg3w6HPIPzxOs066td7yU1scbIpBVZcChH2GSZn\\\\\\\/PHH5odHFIUZujw7jFV\\\\\\\/arpZZHD+dpj\\\\\\\/B3n7uEbZ6G99uwozVh0mXlrvV\\\\\\\/Eq4JjF6xoMyqp5OcKGDRB3rlSClFfvg2CR64hO2kt\\\\\\\/bkQSWdExqHIBB0dhcDa1bPrJD+76NTrP68R4JjCHajQllxJ7xQM6wNx7XXBOvUd\\\\\\\/gfMkCqIZ7EXTjd1dQGDNHgE9QRHsEiNitjNKzSF6gklgfIbxhPhkT4DegzZxVJPXAZXY4KNAIHoXIXmCM9SchkFBUadLEHg+tneSQnAwolR4b6QT0irY8R517ghjdZcdRhf9rUHgITuNsKpOwqWyNPeufxPMTUeopa2NayGhMDaFZhxKuOCs1MwdkGh3aZM+QULsQ7oKTy5iKkIHIaTybrR0waZ\\\\\\\/eDHlYdfKziN\\\\\\\/AiOq8E0z95Kamlg02AMffvxABaX0V1deJgN\\\\\\\/StUV1pPlQtewYLd1KRN2O7HNqgjUxbluQYTJfwSkmQCCVCvyFMdRTjSBgVloTkDrFQhA23MqWRlxHTX7G1fpiVZukEda3urWTt18mpQI8IhCP2OQs\\\\\\\/4OyMselR4MqBPabLvZwGh4zg3+LQcrKqKGFpN7Wlz4y3KjASfvap9OMTi5cfODoyF\\\\\\\/olNykkj6eJ4QSD7yGjI6oBOeFkbGusEdHwf3dqJ4jqasfH9k2M8YdKmd29hkn8W0mMYBElzo5og3hWx2VX9hExWAHDm2DlkdUBQzck9jPeNtGn30FH2Eq2yZPnmplGy7yoZ\\\\\\\/z4J\\\\\\\/m5ogLero7uVh+YVQ1cjbUO2Z1vyKMMRod4Vz5rcR1MdI6SxO9sAUFtRcWChjBNyyQmPSxW+HDvHXaTLky0O2ekti7ahAHbgVBHV5fadRKUg\\\\\\\/TY3nYe8utYxXovizxCGTNaRQK2HxmKvfSPsgz\\\\\\\/twoYCbSTw5sBM9WmIltz+xTH+8hsI7YuSJwJT64mgyfXGKUcmu6XKgkLOOJCiDgLnJ2qaJt0JxlnwxD7z0iu5NVpMNrDGE1UJMB2rSK+qpNbTgMRa4eTegzWM55uo04DLunQFAj36w93N5DGgF8LWWSrSuC01MhTK2LHtN9xMLee3ynJH9scRJzwbDQfAKIMZkbh5luhG\\\\\\\/JgDx7P7EIXEBe+OOHm86WG0b4QUOYixOc+GswbqNjnPYeM7azYUYCsDu2B3w\\\\\\\/s\\\\\\\/ZOGrLL09JHVBEhUFI3XPZgEYupWZR\\\\\\\/Kh0DAj8XxlcWZ3svgJtW7YSZdFRXIwrY6GbO0DttEQqNvQIkfhfU992Wuzzeu9Gkwn3pQqVYbdBbJAAykRaqwV++JeHnQU4MdLxkLC3qEeG1CWse1\\\\\\\/K4kZcuBC\\\\\\\/dLgdKY1iHfhI5pWS0BWB21YDjpwT\\\\\\\/KVI9Q\\\\\\\/PF+TSTmNSzBYA5nHHj5kx4GKsP4RGBl3yNr4PSw6N8X\\\\\\\/bksG6Px\\\\\\\/jQHP1Fr5wyOzIHDmnAKcGOt5MJDK5RUGUOgqAINLTJaA6MmF8UoJ5q53kZTXfRGMOjftxD7xAXj6b9M1BlnyelK\\\\\\\/fPIcezeuGi+iiQJF67mBw5hjn1I4JvHS6gaMtEcsjIl6426YBYh0\\\\\\\/B6qIKlPKs55tAAGuX24nRp0EcynBWOKcgcTOSiFna2fe0R7vwku5QqagR5S9eTHaerAgxUXIK9wezBO27gxSa6+yP61+SeM+sZIWiRRSYVo9VZc5+2r\\\\\\\/H8Y1BHriuiyP\\\\\\\/OlivRTbFgYxhekq4VONaz9JemGDNQBaZGF2MxQkUlyVR+qvjRq9NFXf05NjPGwP2g9ploYCyUIbipeKXmdpGsbb31sbJ0O7KHAeyFZJ5l+2L7U8bHvriEX5mvVrpLgtt1zNWAeZyh\\\\\\\/sZqUW62wG\\\\\\\/ORwO9jtNpgPlZqPfD9QHIMEuvdtCf8Wd4dOGZxTHQyOnvoJueXO7pUjk47J7zbGdGfeOijKIVGxa0yDRh1VT8lsGdDKNkZ0a6Q58jv2fM6Vmd7AkivOYnDXwmvITa1i\\\\\\\/TrdW7hlYl5JjdEufqPUkqYtw2a8fD6X8YpRuYM06po58dWP9\\\\\\\/h8XlVw8Z9pjHHFCg1iGeC6uqLlyIm\\\\\\\/hqCmEK8NBqDha+J8xgDdFfalPp1vwC8wgfBP5\\\\\\\/4m4khIxxnR\\\\\\\/MpCaHfay+0+xunUxtJAeYWuXpLBmPDjv1AfMzgbefWrTeQo49RFfIlY5eob9\\\\\\\/JxWBua7fDNqYa3Ky+Ic48RFDpSrREIEpuj2g5mo8E+ZUzI1yYZw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BCY66p67\\\\\\\/KyqF9g+3HiTGA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225a0134180c99cae0047945d09cef7aea0b1726ce04e2d8f4426423ffee15e8ee\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022L5GeFVtrHEOMQEUhskBv6LOJ+jv5CZ5Bk2hEGlkYlX90taZHMvgsuJXn4rw8KZ6Egs80CLzKLq\\\\\\\/epNU7b6ZvlKJP8TZQT3g4taA0Y3MFzNuVfkEm3ZwjejyJc6a51\\\\\\\/\\\\\\\/JSjqsE3CT\\\\\\\/n2r8e++9V18yPLQsnPb+PixEUy2ytSgJ5qQw\\\\\\\/ekP8QuR0qmKTus4dk\\\\\\\/GWaWSk5RcxrnRDDUe8RslCS6oBuGI6Z13a10nuMfXxWRcYJ80W4Pec96TaRm6wQ3C6yeZHwtbqR\\\\\\\/JRd0j10TmB52XogX53Zpz6tbXflJkwcqzb2OXr3DKW2rRkS8rlA1Z3mLTTuz4nPGvf3z6WblJV4mvxSGt53lRGdM6cYlnJWIbADvG6Mi7ugDF\\\\\\\/i5AEUP79kzxQBQ\\\\\\\/wIrXfWpz9qgnP6qHeFb6QPXKGmqAIa\\\\\\\/47PKv2Cgc\\\\\\\/7+8JaHzd8xd36LbmTWACyXdgTNbF18ItnAzJn0NDSX6eSUIedAUl2RRM06OSO6Q418py0wonV5zsXtFs6buiMKp5YkywHyUDBa6Xl5oL9Z2amojoawgvaexGzPFVqYsA6mf3ANl92l5kVBCtJS7grLoIYhGMPm5LdxRrOePZ\\\\\\\/XPM+Z555wiL7pv8L\\\\\\\/NUJOGP7KDD2yGFnDpWW\\\\\\\/1TFgaqawtZkGckUeL90pG2yc\\\\\\\/H3wIW0BahsQ7YFRYTZCOSuuS23Bj7zsfPrH2ofDQxCrPLwd1HNVz7zteRWONTQ1gtcktdHXkxcUlvLTf\\\\\\\/uMKDf070ZmzuCrb83DKc1S9rHmtYp5urTmA4Dzh3r1c+nlYl2TYaALeIqxK0LNPNNrjPsOQpUvc6q3vAcDCCYQeo1vrRY2Oux7npdsy0aW1Atx10mbY3x27uXEONfHnzsz868I0n\\\\\\\/n+F0Dfom63+9yedEwXT5yPO\\\\\\\/uvEQTAvPHWVIvEdD0+jfnuz3xrt7prtvQL0L8v6T0hu1ejJukGj5hOh90+q3FyZGGPw0hCKhe7wWGLhrJ9bQ3JWqO5gKdbrOwyq2bKoLkiQ758OxeV2R2raioU+I3xhhFV3dRcx0xme6flLJW6msbHexjOzEauP2foOev33Ei4wAYMb7f6gtPUzxiyD\\\\\\\/EWUh8tM7lNe3TyNKshN4jDvCdQjSWhWwMci54YMe+ca1kqetPkCmC4wZcf2+mnSl\\\\\\\/T0\\\\\\\/BrO+HxAvAE7v6FjYtQ6lO7Far0+5LLlH9OMi\\\\\\\/2\\\\\\\/qMH98F5969wHDlImpAQxtaJelIxMsQIqkigbkJ83Caet275dkYFaoWbm2L+0paUxLBhgPCofAUJfe8KWuMgLUPQEA8i+g1dCkVHsvLywKwbAtr\\\\\\\/JpPRDTT31yWt7hHJ3LdPK97Rt4KLvDoGcTGED\\\\\\\/\\\\\\\/56MiInoxzBp8+CrDzLMPuozLCCgzIAO5gHei6zGGmOPxshmBVUHWtCV4U8IGUyhIgBdqGJxMt980APISvzApedUXTgujv1tCiH9\\\\\\\/NH1FphuUiSZx0nLQNlepn5M2VFJH7npY3lYpfGzi7kdjyss9Allf+dOl084JuV3wlflFp84vGGH40nv0ol5veJqTvhwj9AFpn62xT\\\\\\\/yDRZ8y9mvryRpknRj2noszlTDkOLHAqefW6FVFUz+0ysiOjLfZlRSmf0wbxCjG4e0iJpXg1TN3oLWMO6NLxMa\\\\\\\/s5Z6tINTplaF1lZriM1UkMa68QOTeA0Cbj4QjiuRRJiBHVPVCAZvJOLEuLAaikzbtleIrmaGTM41ghjRTCfLvTpI7l5mtOjM8zQQlWL4iJ1k2ao6pli2X9jPJm2bpQJ2RVYhfqnBXSkHnisCeL01+fHFVIMGhY2t2LsQlBP87nmBug1OsJtf\\\\\\\/ttP417tuHxKMZGDlK38oPHva0krgiDmoH1wYYm+7LgYQj2RoWyLFbVFecADU\\\\\\\/txXSvNfdaZZ6Wf0jiQx0LphLOUPMVh1\\\\\\\/sNHZbGtDubZ3vMWo75Tu\\\\\\\/VpnenxeH0aeOY+xuN\\\\\\\/Nzq7sVkhRY1vjOu0h40PGzUmdOBUaRJPvR3MnNWHRbB5IU3otvx9g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022A1H9a5svQExjSdV\\\\\\\/czY0mg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002257d6106199a63b2408a7a5d1904d72dc0fb1cf85ad66f57bba825a57650d1ae0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495857 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1039" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-MZKLa7GFflB2lxfaHf8xGZ6Emw28pUpkx0FrpqoVBAY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1039" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IDJ_mrS_F8rckcbJlnDv6UQCFcdVdsUOF33N9jnUzIQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/1029" title="Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/1051" title="The Developmental Transcriptome of Aedes albopictus, a Major Worldwide Human Disease Vector" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495857" id="node495857" data-pisa="ggg;10/3/1039" data-pisa-master="ggg;g3.119.400813" data-apath="/ggg/10/3/1039.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Type I-F CRISPR-Cas Distribution and Array Dynamics in <em>Legionella pneumophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1234-1123" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shayna R. Deecker</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0824-3704" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander W. Ensminger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1039-1050;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400813 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495857" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shayna R. Deecker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shayna%2BR.%2BDeecker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deecker%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShayna%2BR.%2BDeecker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1234-1123" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shayna R. Deecker</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander W. Ensminger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics, University of Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BW.%2BEnsminger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ensminger%20AW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BW.%2BEnsminger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0824-3704" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander W. Ensminger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alex.ensminger@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">alex.ensminger@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/3/1039" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495857" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/1039.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495857" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/1039.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495857" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/3/1039.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495857" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495857/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In bacteria and archaea, several distinct types of CRISPR-Cas systems provide adaptive immunity through broadly similar mechanisms: short nucleic acid sequences derived from foreign DNA, known as spacers, engage in complementary base pairing with invasive genetic elements setting the stage for nucleases to degrade the target DNA. A hallmark of type I CRISPR-Cas systems is their ability to acquire spacers in response to both new and previously encountered invaders (naïve and primed acquisition, respectively). Our phylogenetic analyses of 43 <em>L. pneumophila</em> type I-F CRISPR-Cas systems and their resident genomes suggest that many of these systems have been horizontally acquired. These systems are frequently encoded on plasmids and can co-occur with nearly identical chromosomal loci. We show that two such co-occurring systems are highly protective and undergo efficient primed acquisition in the lab. Furthermore, we observe that targeting by one system’s array can prime spacer acquisition in the other. Lastly, we provide experimental and genomic evidence for a model in which primed acquisition can efficiently replenish a depleted type I CRISPR array following a mass spacer deletion event.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bacteriology" class="hw-term hw-article-keyword hw-article-keyword-bacteriology" rel="nofollow">bacteriology</a></li><li class="kwd"><a href="/keyword/crispr-cas" class="hw-term hw-article-keyword hw-article-keyword-crispr-cas" rel="nofollow">CRISPR-Cas</a></li><li class="kwd"><a href="/keyword/legionella" class="hw-term hw-article-keyword hw-article-keyword-legionella" rel="nofollow">Legionella</a></li><li class="kwd"><a href="/keyword/microbiology" class="hw-term hw-article-keyword hw-article-keyword-microbiology" rel="nofollow">microbiology</a></li></ul><p id="p-3">Microorganisms have evolved over millions of years to survive in harsh environments, backed in part by immune strategies that protect against foreign genetic elements, such as viral phages and foreign DNA elements (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">van Houte <em>et al.</em> 2016</a>). Clustered regularly interspaced short palindromic repeats (CRISPR) coupled with associated <em>cas</em> genes form a potent adaptive immune response in numerous prokaryotic species (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barrangou <em>et al.</em> 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brouns <em>et al.</em> 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Marraffini and Sontheimer 2008</a>). These systems have been classified into six major types, which are further divided into various subtypes, based on their mechanism of action and Cas protein content (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Makarova <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">2015</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Koonin <em>et al.</em> 2017</a>).</p><p id="p-4">A CRISPR-Cas response to invading DNA occurs in three distinct phases: adaptation, expression and interference (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Barrangou <em>et al.</em> 2007</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Brouns <em>et al.</em> 2008</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Marraffini and Sontheimer 2008</a>). In the adaptation phase, the CRISPR-Cas system acquires a DNA sequence (spacer) from the invader and integrates it into an array of spacers interspersed with repetitive sequences (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ishino <em>et al.</em> 1987</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Mojica <em>et al.</em> 2000</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jansen <em>et al</em>. 2002b</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Barrangou <em>et al.</em> 2007</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jackson <em>et al.</em> 2017</a>). The spacers can be derived from foreign elements whose infection was unsuccessful, such as defunct phage (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hynes <em>et al.</em> 2014</a>), from free DNA ends generated during double strand breaks (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Levy <em>et al.</em> 2015</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Modell <em>et al.</em> 2017</a>) or following phage DNA injection into the host cell (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Modell <em>et al.</em> 2017</a>). Once acquired, these spacers form the basis of immunological memory for the bacterium (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Barrangou <em>et al.</em> 2007</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jackson <em>et al.</em> 2017</a>). During the expression phase, the array is transcribed and processed to form CRISPR RNA (crRNA) molecules which form a surveillance complex with Cas proteins (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Brouns <em>et al.</em> 2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Haurwitz <em>et al.</em> 2010</a>). Infection by an invading genetic element initiates the interference step, wherein the surveillance complex recognizes and binds the foreign DNA via base-pairing with the complementary crRNA and cleaves it, effectively neutralizing the threat to the host (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Brouns <em>et al.</em> 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Jore <em>et al.</em> 2011</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wiedenheft <em>et al.</em> 2011</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Westra <em>et al.</em> 2012</a>).</p><p id="p-5">Despite the sophistication of CRISPR-Cas systems, phages and other foreign DNA elements can still escape CRISPR-Cas targeting (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Samson <em>et al.</em> 2013</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Stanley and Maxwell 2018</a>). A common mechanism of escape is the accumulation of random mutations within the foreign element that either prevent complementary base pairing with crRNAs during interference or abolish target recognition (<em>i.e.</em>, through protospacer adjacent motif [PAM] mutations) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Deveau <em>et al.</em> 2008</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Semenova <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Datsenko <em>et al.</em> 2012</a>). CRISPR-Cas systems can overcome this escape by acquiring new spacers; in fact, imperfect CRISPR targeting often leads to a highly efficient “primed” acquisition response, providing an intrinsic mechanism to protect against mutational escape (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Swarts <em>et al.</em> 2012</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Datsenko <em>et al.</em> 2012</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Savitskaya <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fineran <em>et al.</em> 2014</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Richter <em>et al.</em> 2014</a>). Primed acquisition has been studied in type I-B (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2014a</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">2014b</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">2017</a>), I-C (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Rao <em>et al.</em> 2017</a>), I-E (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Swarts <em>et al.</em> 2012</a>; <a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Datsenko <em>et al.</em> 2012</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Savitskaya <em>et al.</em> 2013</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Fineran <em>et al.</em> 2014</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Xue <em>et al.</em> 2015</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jackson <em>et al.</em> 2019</a>) and I-F (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Richter <em>et al.</em> 2014</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Vorontsova <em>et al.</em> 2015</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Staals <em>et al.</em> 2016</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Heussler <em>et al.</em> 2016</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Jackson <em>et al.</em> 2019</a>) CRISPR-Cas systems. Interference-driven acquisition, or targeted acquisition, has also been observed in type I-C (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Rao <em>et al.</em> 2017</a>), type I-E (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kuznedelov <em>et al.</em> 2016</a>) and I-F systems (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Staals <em>et al.</em> 2016</a>), wherein a primed acquisition response occurs against a target with a perfect match to a spacer already within the array.</p><p id="p-6"><em>Legionella pneumophila</em> is a Gram-negative bacterium and the causative agent of Legionnaires’ disease (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brenner <em>et al.</em> 1979</a>). Most isolates possess any of three different CRISPR-Cas systems: type I-C, I-F and/or II-B (D’Auria <em>et al.</em> 2010; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>). Our lab has recently shown that all three types of CRISPR-Cas systems found in <em>L. pneumophila</em> isolates are active (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>) and we have characterized the targeted acquisition response for the type I-C system (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Rao <em>et al.</em> 2017</a>). While much of this work to date has focused on the I-C systems of <em>L. pneumophila</em>, the type I-F systems of this pathogen are highly protective, remarkably diverse with respect to spacer content, and are frequently found on plasmids – suggesting that they may be circulated via horizontal gene transfer (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>). In this study, we perform the first comprehensive phylogenetic analysis of the <em>L. pneumophila</em> type I-F systems and test a model by which horizontal acquisition of a mobile type I-F CRISPR-Cas system could replenish a collapsed chromosomal array.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Bioinformatic analyses</h3><p id="p-7">Bioinformatic analyses of the Illumina sequence data were performed as described previously (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Rao <em>et al.</em> 2017</a>). Briefly, the raw paired-end reads were merged using FLASH (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Magoc and Salzberg 2011</a>), and any unpaired reads were subsequently quality trimmed using Trimmomatic (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>). These processed reads were then combined and analyzed using a Perl script (available upon request) that annotated existing spacers (S), newly acquired spacers (X), repetitive sequences (R) and the downstream sequence (D). The newly acquired spacers were aligned to the priming plasmid, the <em>L. pneumophila</em> str. Lens chromosome or the <em>L. pneumophila</em> str. Lens plasmid using BLASTN (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>). The results from the BLASTN alignment for the priming plasmid were then processed to obtain coverage per nucleotide, and plotted on the reference sequence using Circos (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Krzywinski <em>et al.</em> 2009</a>).</p><p id="p-8">For the bioinformatic analyses of the <em>L. pneumophila</em> type I-F system diversity, <em>L. pneumophila</em> draft genomes and completed genomes were downloaded from the European Nucleotide Archive and NCBI respectively (Table S1). Type I-F CRISPR-Cas systems were identified using CRISPRCasFinder (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Couvin <em>et al.</em> 2018</a>) and CRISPRDetect (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Biswas <em>et al.</em> 2016</a>). All genomes with type I-F systems present were annotated using Prokka (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Seemann 2014</a>). For the core genome phylogeny, pan-genome analysis was performed with Roary using the default settings and the MAFFT aligner (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Page <em>et al.</em> 2015</a>). The <em>cas1</em> genes were extracted from each CRISPR-Cas system and aligned with MUSCLE (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Edgar 2004</a>). The <em>cas1</em> gene alignment and the core genome alignment were used to create phylogenies with RAxML using the rapid bootstrapping and search for best maximum likelihood tree algorithm with 1000 bootstrap iterations (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Stamatakis 2014</a>). The RAxML trees were condensed with MEGA7 using a bootstrap support cut-off of 50% (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kumar <em>et al.</em> 2016</a>). For the <em>cas1</em> tree, each phylogroup possesses 100% nucleotide identity in the <em>cas1</em> sequence. CRISPR array alignments, clustering and visualization were performed with CRISPRStudio (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dion <em>et al.</em> 2018</a>). Four isolates possessed split CRISPR-Cas arrays (found at the end of separate contigs) in the draft genomes, so they were excluded from both phylogenetic analysis and CRISPRStudio analysis to minimize potential problems with array assembly (Table S1).</p></div><div id="sec-3" class="subsection"><h3>Bacterial strains, plasmids and oligos used</h3><p id="p-9">The bacterial strains and plasmids used in this study are listed in supplementary table 2, and the oligos used in this study are listed in supplementary table 3.</p><p id="p-10">The priming plasmids were created by annealing oligos (see Table S3) to create the protospacer insert with the canonical GG PAM (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Mojica <em>et al.</em> 2009</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cady <em>et al.</em> 2012</a>; <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Richter <em>et al.</em> 2014</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Vorontsova <em>et al.</em> 2015</a>; <a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Staals <em>et al.</em> 2016</a>) and subsequently ligating the insert into an ApaI/PstI-digested pMMB207 vector (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Solomon <em>et al.</em> 2000</a>). The scrambled control plasmid was created in the same manner, except it contained a 32-nt scrambled sequence in place of a targeted protospacer sequence.</p><p id="p-11">Lens (chromosome) array deletion mutants were generated through allelic replacement. Briefly, 1 kb of DNA upstream and 2 kb downstream of the CRISPR array were amplified by PCR and stitched together to create an insert where the entire array, save for the last repeat, was deleted. The insert was ligated into a pJB4648 plasmid. Overnight cultures of <em>L. pneumophila</em> str. Lens were grown in ACES-buffered yeast extract (AYE) medium to an OD<sub>600</sub> of ∼4.0 using two-day patches that were grown on charcoal-buffered ACES yeast extract (CYE) plates. Pellets from 4.0 ODU of culture underwent three washing steps: twice with 1 mL of ice-cold ultrapure water and once with 1 mL of ice-cold 10% glycerol. The pellet was then re-suspended in 200 μL of ice-cold 10% glycerol and 400 ng of plasmid was added to the sample. The solution was transferred to an ice-cold electroporation cuvette with a 2 mm gap and electroporated with the following settings: 2500 kV, 600 Ω and 25 mF. After electroporation, 800 μL of AYE medium was added to each sample and the samples recovered for 3 hr at 37° at 600 RPM in a shaking incubator. The samples were plated on CYE plates supplemented with 15 μg mL<sup>-1</sup> of gentamycin and incubated at 37° for 3 days. Surviving colonies were patched onto CYE + gentamycin plates and grown at 37° for 2 days. Patches were subsequently struck onto CYE plates supplemented with sucrose and incubated at 37° for 3 days. Surviving colonies were patched onto CYE + sucrose plates, grown at 37° for 2 days and screened by PCR to confirm the deletion. Two independent clones were Illumina sequenced and used for subsequent replenishment assays.</p></div><div id="sec-4" class="subsection"><h3>Transformation efficiency assay and population pool generation</h3><p id="p-12">The transformation efficiency assay was performed as we have described previously (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>) with some modifications. Briefly, <em>L. pneumophila</em> str. Lens was electroporated as described above. The samples were plated in a dilution series on CYE plates supplemented with 5 μg mL<sup>-1</sup> of chloramphenicol and incubated at 37° for 3 days. The relative transformation efficiency for each targeted plasmid was calculated as a percentage of the transformation efficiency obtained from the scrambled control plasmid. Three biological replicates were performed for each transformation efficiency assay.</p><p id="p-13">Population pools for spacer acquisition experiments were generated by mixing together ≥ 50 colonies per population from a newly transformed wild-type strain on CYE plates supplemented with 5 μg mL<sup>-1</sup> of chloramphenicol using AYE medium supplemented with 5 μg mL<sup>-1</sup> of chloramphenicol. Population pools were made in triplicate for each transformed plasmid.</p></div><div id="sec-5" class="subsection"><h3>Serial passaging on an automated liquid handler</h3><p id="p-14">The serial passaging of transformed <em>L. pneumophila</em> str. Lens populations was performed as described previously (<a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>). Briefly, overnight cultures of the population pools in AYE medium supplemented with 5 μg mL<sup>-1</sup> of chloramphenicol for plasmid maintenance were grown to an OD<sub>600</sub> of ∼2.0. The culture was then back diluted to an OD<sub>600</sub> of ∼0.0625 and grown in a flat-bottom 48-well plate (Greiner) in a shaking incubator at 37°. A Freedom Evo 100 liquid handler (Tecan) connected to an Infinite M200 Pro plate reader (Tecan) measured the optical density of the plate every 20 min, until an OD<sub>600</sub> of ∼2.0 was reached. The cultures were then automatically back diluted to an OD<sub>600</sub> of ∼0.0625 in the adjacent well to continue growth, and the remaining culture was transferred to a 48-well plate that was kept at 4°. In this manner, each saved culture represented ∼5 generations of growth. The passaging was done without selection in AYE medium to allow for plasmid loss during passaging.</p></div><div id="sec-6" class="subsection"><h3>Genomic DNA extraction, PCR and agarose gel screening</h3><p id="p-15">Genomic DNA was extracted from the passaged cultures and the parental chromosome Lens array deletion strains using a Machery-Nagel Nucleospin Tissue kit according to the manufacturer’s protocol. The extracted samples from passaged cultures were used as a template in a 30-cycle PCR reaction with PaCeR HP Polymerase (GeneBio Systems) to amplify the leader end of the CRISPR array using primers listed in Table S3. The PCR products were then separated on a 3% agarose gel to determine if spacer acquisition (or spacer loss) had occurred based on the presence of an upper (or lower) band relative to the control sample.</p></div><div id="sec-7" class="subsection"><h3>Nextera library prep and Illumina sequencing</h3><p id="p-16">The extracted genomic DNA from passaged cultures was prepared for leader-end array sequencing by performing a 20-cycle PCR using Kapa HiFi Polymerase (Kapa Biosystems) and the primers listed in Table S3. The PCR products were purified using a Machery-Nagel Nucleospin Gel and PCR Clean-up kit as per the manufacturer’s instructions and normalized to 1 ng using the Invitrogen Quant-iT PicoGreen dsDNA assay. The DNA was then tagmented using a Nextera XT tagmentation kit as per the manufacturer’s instructions. The tagmented products were sequenced with a paired-end (2 × 150 bp) sequencing run on an Illumina NextSeq platform at the Centre for the Analysis of Genome Evolution and Function (CAGEF) at the University of Toronto.</p><p id="p-17">The genomic DNA from the parental chromosome Lens array deletion strains was normalized to 1 ng using PicoGreen. The DNA was then tagmented using a Nextera XT tagmentation kit as per the manufacturer’s instructions. The tagmented products were sequenced with paired-end (2 × 150 bp) sequencing in-house on an Illumina MiniSeq platform.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-18">Strains and plasmids are available upon request. Raw Illumina reads have been deposited into the NCBI sequence read archive under the BioProject PRJNA433194. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11367821">https://doi.org/10.25387/g3.11367821</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Phylogenetic analyses suggest widespread horizontal exchange of L. pneumophila type I-F systems</h3><p id="p-19">In order to explore the hypothesis that plasmid-based type I-F CRISPR-Cas systems in <em>L. pneumophila</em> could be circulated via horizontal gene transfer, we bioinformatically examined the diversity of type I-F CRISPR-Cas systems within this species. Using both CRISPRCasFinder (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Couvin <em>et al.</em> 2018</a>) and CRISPRDetect (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Biswas <em>et al.</em> 2016</a>), we surveyed 525 draft and 6 completed <em>L. pneumophila</em> genomes. In total, we identified 47 <em>L. pneumophila</em> isolates that possessed type I-F systems (Table S1), including 5 that we had described previously (<a id="xref-ref-57-6" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>). Four of these isolates were subsequently excluded from further analyses to minimize potential problems with array assembly (Table S1). We next performed two types of phylogenetic analysis: <em>cas1</em> phylogeny (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), which placed each CRISPR-Cas system into one of six phylotypes; and core-genome phylogeny (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>), which reflects the overall relatedness between each of the 43 isolates. A comparison of the two trees indicates a clear phylogenetic incongruence suggesting that horizontal acquisition has impacted the distribution of type I-F CRISPR-Cas systems within the species. Additionally, three of the type I-F systems were present on annotated plasmids (str. Lens pLPL, str. Mississauga-2006 and str. C8_S), with two type I-F CRISPR-Cas systems occurring in the same isolate (str. Lens). There were also two isolates that possessed both a type I-C and a type I-F CRISPR-Cas system (str. Mississauga-2006 and str. FJAD01).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic analysis of type I-F CRISPR-Cas system diversity in L. pneumophila reveals a horizontal distribution across isolates. L. pneumophila draft and completed genomes were analyzed using CRISPRDetect and CRISPRCasFinder to identify type I-F systems present within the genomes. Isolates that possessed type I-F systems were subjected to phylogenetic analyses of their cas1 gene and core genome. The core genome alignment for the examined isolates was determined by Roary. Isolate names are color-coded based on the cas1 gene phylogeny to allow for comparison between the analyses. The isolates used in this analysis and their accession numbers can be found in supplementary table 1. A) The cas1 gene phylogeny for L. pneumophila isolates with type I-F systems reveals six different cas1 groups. B) The core genome phylogeny of the examined isolates is not congruent with the cas1 phylogeny, suggesting that many of the type I-F systems were horizontally acquired rather than vertically inherited. Note that Lens possesses two type I-F CRISPR-Cas systems, one on a plasmid (group &#x398;) and one on its chromosome (group &#x3A6;). Also note that isolates with 100% nucleotide identity in their core genome, as well as a shared cas gene group and redundant CRISPR array, have been collapsed to one representative in the phylogeny. Isolates with unique CRISPR arrays but shared core genomes are listed separately. Bootstrap support values for each node are indicated in both trees. Collapsed isolates include: +: FJEK01, FJEL01, FJEM01, FJEN01, FJEO01, FJEP01, FJEQ01, FJER01, FJES01, FJET01, FJEU01, FJEV01, FJEX01, FJEY01, FJEZ01, FJFA01; ++: FFI105, FFI337; +++: LAXR01; and ++++: FJBI01, FJBU01." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-239459410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic analysis of type I-F CRISPR-Cas system diversity in &lt;em&gt;L. pneumophila&lt;/em&gt; reveals a horizontal distribution across isolates. &lt;em&gt;L. pneumophila&lt;/em&gt; draft and completed genomes were analyzed using CRISPRDetect and CRISPRCasFinder to identify type I-F systems present within the genomes. Isolates that possessed type I-F systems were subjected to phylogenetic analyses of their &lt;em&gt;cas1&lt;/em&gt; gene and core genome. The core genome alignment for the examined isolates was determined by Roary. Isolate names are color-coded based on the &lt;em&gt;cas1&lt;/em&gt; gene phylogeny to allow for comparison between the analyses. The isolates used in this analysis and their accession numbers can be found in supplementary table 1. A) The &lt;em&gt;cas1&lt;/em&gt; gene phylogeny for &lt;em&gt;L. pneumophila&lt;/em&gt; isolates with type I-F systems reveals six different &lt;em&gt;cas1&lt;/em&gt; groups. B) The core genome phylogeny of the examined isolates is not congruent with the &lt;em&gt;cas1&lt;/em&gt; phylogeny, suggesting that many of the type I-F systems were horizontally acquired rather than vertically inherited. Note that Lens possesses two type I-F CRISPR-Cas systems, one on a plasmid (group &#x398;) and one on its chromosome (group &#x3A6;). Also note that isolates with 100% nucleotide identity in their core genome, as well as a shared &lt;em&gt;cas&lt;/em&gt; gene group and redundant CRISPR array, have been collapsed to one representative in the phylogeny. Isolates with unique CRISPR arrays but shared core genomes are listed separately. Bootstrap support values for each node are indicated in both trees. Collapsed isolates include: +: FJEK01, FJEL01, FJEM01, FJEN01, FJEO01, FJEP01, FJEQ01, FJER01, FJES01, FJET01, FJEU01, FJEV01, FJEX01, FJEY01, FJEZ01, FJFA01; ++: FFI105, FFI337; +++: LAXR01; and ++++: FJBI01, FJBU01.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1039/F1.medium.gif" width="320" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/3/1039/F1.medium.gif" width="320" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495905" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Phylogenetic analysis of type I-F CRISPR-Cas system diversity in <em>L. pneumophila</em> reveals a horizontal distribution across isolates. <em>L. pneumophila</em> draft and completed genomes were analyzed using CRISPRDetect and CRISPRCasFinder to identify type I-F systems present within the genomes. Isolates that possessed type I-F systems were subjected to phylogenetic analyses of their <em>cas1</em> gene and core genome. The core genome alignment for the examined isolates was determined by Roary. Isolate names are color-coded based on the <em>cas1</em> gene phylogeny to allow for comparison between the analyses. The isolates used in this analysis and their accession numbers can be found in supplementary table 1. A) The <em>cas1</em> gene phylogeny for <em>L. pneumophila</em> isolates with type I-F systems reveals six different <em>cas1</em> groups. B) The core genome phylogeny of the examined isolates is not congruent with the <em>cas1</em> phylogeny, suggesting that many of the type I-F systems were horizontally acquired rather than vertically inherited. Note that Lens possesses two type I-F CRISPR-Cas systems, one on a plasmid (group Θ) and one on its chromosome (group Φ). Also note that isolates with 100% nucleotide identity in their core genome, as well as a shared <em>cas</em> gene group and redundant CRISPR array, have been collapsed to one representative in the phylogeny. Isolates with unique CRISPR arrays but shared core genomes are listed separately. Bootstrap support values for each node are indicated in both trees. Collapsed isolates include: +: FJEK01, FJEL01, FJEM01, FJEN01, FJEO01, FJEP01, FJEQ01, FJER01, FJES01, FJET01, FJEU01, FJEV01, FJEX01, FJEY01, FJEZ01, FJFA01; ++: FFI105, FFI337; +++: LAXR01; and ++++: FJBI01, FJBU01.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Given the results of these phylogenetic analyses, we next examined the spacer distribution across each of the arrays to determine the level of array diversification within each of the six <em>L. pneumophila</em> type I-F <em>cas1</em> phylotypes. We aligned, clustered, and visualized each of the 23 distinct <em>L. pneumophila</em> type I-F CRISPR arrays using CRISPRStudio (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Dion <em>et al.</em> 2018</a>) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). These analyses revealed patterns consistent with both spacer acquisition and spacer loss, suggesting that both processes contribute to <em>L. pneumophila</em> type I-F CRISPR array diversity.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CRISPR array analysis suggests that spacer acquisition and spacer loss has contributed to array diversification in L. pneumophila type I-F systems. L. pneumophila isolates were subjected to a CRISPRStudio analysis to look at the spacer composition of their CRISPR arrays. Gray boxes denote unique spacers, colored boxes denote shared spacers and a dashed line denotes spacer loss. The isolate color coding scheme is based on the cas1 grouping from Figure 1. Isolates denoted with a &#x201C;+&#x201D; within the same cas gene group have 100% nucleotide identity in their core genomes. Three strains have systems that reside on contigs previously annotated as plasmids (str. Lens, str. Mississauga-2006, and str. C8_S). Notably, an additional six strains (FJAD01, FJAJ01, FJAM01, FJBU01, FJBW01 and FJMD01) have systems that reside on contigs with overlapping sequence (78-89 nt) on either end, suggesting that they may also reside on a plasmid or other mobile element." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-239459410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;CRISPR array analysis suggests that spacer acquisition and spacer loss has contributed to array diversification in &lt;em&gt;L. pneumophila&lt;/em&gt; type I-F systems. &lt;em&gt;L. pneumophila&lt;/em&gt; isolates were subjected to a CRISPRStudio analysis to look at the spacer composition of their CRISPR arrays. Gray boxes denote unique spacers, colored boxes denote shared spacers and a dashed line denotes spacer loss. The isolate color coding scheme is based on the &lt;em&gt;cas1&lt;/em&gt; grouping from Figure 1. Isolates denoted with a &#x201C;+&#x201D; within the same &lt;em&gt;cas&lt;/em&gt; gene group have 100% nucleotide identity in their core genomes. Three strains have systems that reside on contigs previously annotated as plasmids (str. Lens, str. Mississauga-2006, and str. C8_S). Notably, an additional six strains (FJAD01, FJAJ01, FJAM01, FJBU01, FJBW01 and FJMD01) have systems that reside on contigs with overlapping sequence (78-89 nt) on either end, suggesting that they may also reside on a plasmid or other mobile element.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1039/F2.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/3/1039/F2.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495934" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">CRISPR array analysis suggests that spacer acquisition and spacer loss has contributed to array diversification in <em>L. pneumophila</em> type I-F systems. <em>L. pneumophila</em> isolates were subjected to a CRISPRStudio analysis to look at the spacer composition of their CRISPR arrays. Gray boxes denote unique spacers, colored boxes denote shared spacers and a dashed line denotes spacer loss. The isolate color coding scheme is based on the <em>cas1</em> grouping from Figure 1. Isolates denoted with a “+” within the same <em>cas</em> gene group have 100% nucleotide identity in their core genomes. Three strains have systems that reside on contigs previously annotated as plasmids (str. Lens, str. Mississauga-2006, and str. C8_S). Notably, an additional six strains (FJAD01, FJAJ01, FJAM01, FJBU01, FJBW01 and FJMD01) have systems that reside on contigs with overlapping sequence (78-89 nt) on either end, suggesting that they may also reside on a plasmid or other mobile element.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>The plasmid and chromosomal Lens CRISPR-Cas systems are active and adaptive</h3><p id="p-23">Given the evidence for spacer gain and loss that we observe in <em>L. pneumophila</em> type I-F arrays (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), we next decided to examine array dynamics experimentally. We focused on <em>L. pneumophila</em> str. Lens, which possesses two type I-F CRISPR-Cas systems: one on its chromosome and one on an endogenous 60 kb plasmid, pLPL (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">D’Auria <em>et al.</em> 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gomez-Valero <em>et al.</em> 2011</a>; <a id="xref-ref-57-7" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>). The two systems have a 97.6% Cas protein identity and the repeat units between the spacers in the CRISPR array differ by only a single nucleotide (<a id="xref-ref-57-8" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The CRISPR arrays themselves are of different lengths (64 spacers for the chromosomal system and 53 for that contained on the pLPL plasmid). Despite a high degree of identity between the Cas proteins, each array contains a completely unique set of spacers (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">D’Auria <em>et al.</em> 2010</a>; <a id="xref-ref-57-9" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>). The presence of two remarkably similar I-F systems in <em>L. pneumophila</em> str. Lens provided us with an opportunity to examine targeted spacer acquisition in both of these largely uncharacterized CRISPR-Cas systems and the interplay between them.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A comparison of the Lens chromosome and pLPL CRISPR-Cas systems. A) The overall pairwise amino acid identity across the Cas proteins of the two systems is approximately 97%, with individual pairwise Cas protein identities ranging from 96 to 99%. B) A CRISPRStudio alignment of the arrays for two systems shows completely unique spacer content and a differing number of spacers. C) Analysis of the repeat sequences shows that the Lens pLPL and chromosome systems have one SNP between their consensus repeats, in addition to possessing a mutated last repeat in their arrays. Mutations are denoted in red." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-239459410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A comparison of the Lens chromosome and pLPL CRISPR-Cas systems. A) The overall pairwise amino acid identity across the Cas proteins of the two systems is approximately 97%, with individual pairwise Cas protein identities ranging from 96 to 99%. B) A CRISPRStudio alignment of the arrays for two systems shows completely unique spacer content and a differing number of spacers. C) Analysis of the repeat sequences shows that the Lens pLPL and chromosome systems have one SNP between their consensus repeats, in addition to possessing a mutated last repeat in their arrays. Mutations are denoted in red.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1039/F3.medium.gif" width="440" height="230"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/3/1039/F3.medium.gif" width="440" height="230"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495981" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">A comparison of the Lens chromosome and pLPL CRISPR-Cas systems. A) The overall pairwise amino acid identity across the Cas proteins of the two systems is approximately 97%, with individual pairwise Cas protein identities ranging from 96 to 99%. B) A CRISPRStudio alignment of the arrays for two systems shows completely unique spacer content and a differing number of spacers. C) Analysis of the repeat sequences shows that the Lens pLPL and chromosome systems have one SNP between their consensus repeats, in addition to possessing a mutated last repeat in their arrays. Mutations are denoted in red.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">To assess CRISPR-Cas activity in both Lens type I-F systems, we performed an established transformation efficiency assay (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Marraffini and Sontheimer 2008</a>) using two different targeted protospacer sequences: one matching the most recently acquired spacer and one matching a spacer further downstream in the array (chromosomal spacer 23 and pLPL spacer 50). Consistent with active CRISPR-Cas protection, each of the protospacer-containing plasmids exhibited reductions in transformation efficiencies relative to a scrambled protospacer control (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). These relative transformation efficiencies ranged from 1 × 10<sup>−2</sup> to 1 × 10<sup>−4</sup>, with the most recently acquired spacers providing ∼100-fold greater protection than spacers located further downstream in each array.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The L. pneumophila str. Lens chromosome and pLPL type I-F CRISPR-Cas systems are active against plasmids containing protospacers. A) L. pneumophila str. Lens was transformed with plasmids containing targeted protospacer sequences matched to the first spacer (sp1) or a downstream spacer (spacer 23 or spacer 50, respectively) of the Lens (chromosome) CRISPR-Cas system or the Lens (pLPL) CRISPR-Cas system. After plating on selective media and incubating for three days, transformation efficiencies were calculated as a percentage of the transformation efficiency of a control plasmid with a scrambled targeted sequence. The average for three biological replicates is shown where the error bars represent the standard error of the mean. B) Spacer acquisition and loss were analyzed using a PCR-based screen in which the leader-end of the CRISPR array for both the control samples and the transformed samples was amplified with system-specific primers to differentiate between the chromosomal Lens and the plasmid Lens arrays and visualized on an agarose gel. Products from the transformed samples were compared to the control, which contained untransformed genomic DNA. Bands representing spacer acquisition and loss are indicated." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-239459410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &lt;em&gt;L. pneumophila&lt;/em&gt; str. Lens chromosome and pLPL type I-F CRISPR-Cas systems are active against plasmids containing protospacers. A) &lt;em&gt;L. pneumophila&lt;/em&gt; str. Lens was transformed with plasmids containing targeted protospacer sequences matched to the first spacer (sp1) or a downstream spacer (spacer 23 or spacer 50, respectively) of the Lens (chromosome) CRISPR-Cas system or the Lens (pLPL) CRISPR-Cas system. After plating on selective media and incubating for three days, transformation efficiencies were calculated as a percentage of the transformation efficiency of a control plasmid with a scrambled targeted sequence. The average for three biological replicates is shown where the error bars represent the standard error of the mean. B) Spacer acquisition and loss were analyzed using a PCR-based screen in which the leader-end of the CRISPR array for both the control samples and the transformed samples was amplified with system-specific primers to differentiate between the chromosomal Lens and the plasmid Lens arrays and visualized on an agarose gel. Products from the transformed samples were compared to the control, which contained untransformed genomic DNA. Bands representing spacer acquisition and loss are indicated.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1039/F4.medium.gif" width="357" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/3/1039/F4.medium.gif" width="357" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495959" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">The <em>L. pneumophila</em> str. Lens chromosome and pLPL type I-F CRISPR-Cas systems are active against plasmids containing protospacers. A) <em>L. pneumophila</em> str. Lens was transformed with plasmids containing targeted protospacer sequences matched to the first spacer (sp1) or a downstream spacer (spacer 23 or spacer 50, respectively) of the Lens (chromosome) CRISPR-Cas system or the Lens (pLPL) CRISPR-Cas system. After plating on selective media and incubating for three days, transformation efficiencies were calculated as a percentage of the transformation efficiency of a control plasmid with a scrambled targeted sequence. The average for three biological replicates is shown where the error bars represent the standard error of the mean. B) Spacer acquisition and loss were analyzed using a PCR-based screen in which the leader-end of the CRISPR array for both the control samples and the transformed samples was amplified with system-specific primers to differentiate between the chromosomal Lens and the plasmid Lens arrays and visualized on an agarose gel. Products from the transformed samples were compared to the control, which contained untransformed genomic DNA. Bands representing spacer acquisition and loss are indicated.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">To determine whether spacer acquisition occurs within the context of a perfectly matched protospacer target - as we previously observed for a relatively permissive type I-C CRISPR-Cas system (<a id="xref-ref-57-10" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>) - we pooled the transformed populations, passaged them on an automated liquid handler for 20 generations without selection, extracted their genomic DNA, and screened the leader end of the CRISPR array by PCR and agarose gel electrophoresis. While the populations transformed with plasmids encoding either protospacer 23 (chromosome) or protospacer 50 (pLPL plasmid) exhibited spacer acquisition in both Lens systems (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>), the populations transformed with protospacer 1 plasmids exhibited spacer loss, with spacer acquisition undetectable on a gel (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). This is consistent with the level of protection we see against these plasmids (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>): in addition to spacer loss, we may have also selected for host <em>cas</em> or plasmid protospacer mutants that could preclude spacer acquisition. Regardless, these data are consistent with our bioinformatic analyses, which indicates that both spacer acquisition and spacer loss contribute to type I-F CRISPR array diversity in <em>L. pneumophila</em> isolates (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-28">Our observation that spacers downstream of spacer 1 in each type I-F system provided relatively modest protection led us to ask whether these spacers could nevertheless drive primed acquisition of new, more protective spacer sequences. To characterize the patterns of targeted spacer acquisition in the chromosomal and pLPL CRISPR-Cas systems, we amplified the leader-proximal region of each CRISPR array from wild-type populations that had been transformed with the plasmids targeted by their relatively permissive spacers (chromosomal: spacer 23; pLPL: spacer 50). We Illumina sequenced these PCR products and used an established bioinformatics pipeline (<a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Rao <em>et al.</em> 2017</a>) to identify newly acquired spacer sequences within each read (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We then mapped the target of each new spacer to the priming plasmid (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Krzywinski <em>et al.</em> 2009</a>) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495960/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495960/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>The number of acquired spacers for the Lens (pLPL) and Lens (chromosome) CRISPR-Cas systems and their respective targets during self-priming</strong></span><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of self-primed spacer acquisition in the two Lens CRISPR-Cas systems. Bacterial transformants with targeted plasmids were passaged for 20 generations without antibiotic selection to enrich for spacer acquisition; the leader end of the CRISPR array was amplified and the amplicons were Illumina sequenced. Newly targeted protospacers were obtained from the raw reads using an in-house bioinformatics pipeline and visualized with Circos. The arrows in the simplified array schematic show the primer location for PCR amplification prior to sequencing, while the star denotes the priming spacer. The priming protospacer sequence is indicated by a colored box in the Circos plot. All data are the average of three biological replicates. A) The distribution of newly targeted protospacers mapped to the priming plasmid on the Circos plot reveals a strand bias in self-primed spacer acquisition within the Lens (pLPL) CRISPR-Cas system. The height of the bars indicates the number of spacers mapped to the position on the plasmid, up to 5% of total acquired spacers. B) The distribution of newly targeted protospacers mapped to the priming plasmid shows a similar pattern of self-primed spacer acquisition within the Lens (chromosome) CRISPR-Cas system to that of its pLPL CRISPR-Cas counterpart. Labeling as in (A)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-239459410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Characterization of self-primed spacer acquisition in the two Lens CRISPR-Cas systems. Bacterial transformants with targeted plasmids were passaged for 20 generations without antibiotic selection to enrich for spacer acquisition; the leader end of the CRISPR array was amplified and the amplicons were Illumina sequenced. Newly targeted protospacers were obtained from the raw reads using an in-house bioinformatics pipeline and visualized with Circos. The arrows in the simplified array schematic show the primer location for PCR amplification prior to sequencing, while the star denotes the priming spacer. The priming protospacer sequence is indicated by a colored box in the Circos plot. All data are the average of three biological replicates. A) The distribution of newly targeted protospacers mapped to the priming plasmid on the Circos plot reveals a strand bias in self-primed spacer acquisition within the Lens (pLPL) CRISPR-Cas system. The height of the bars indicates the number of spacers mapped to the position on the plasmid, up to 5% of total acquired spacers. B) The distribution of newly targeted protospacers mapped to the priming plasmid shows a similar pattern of self-primed spacer acquisition within the Lens (chromosome) CRISPR-Cas system to that of its pLPL CRISPR-Cas counterpart. Labeling as in (A).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1039/F5.medium.gif" width="181" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/3/1039/F5.medium.gif" width="181" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495839" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-30" class="first-child">Characterization of self-primed spacer acquisition in the two Lens CRISPR-Cas systems. Bacterial transformants with targeted plasmids were passaged for 20 generations without antibiotic selection to enrich for spacer acquisition; the leader end of the CRISPR array was amplified and the amplicons were Illumina sequenced. Newly targeted protospacers were obtained from the raw reads using an in-house bioinformatics pipeline and visualized with Circos. The arrows in the simplified array schematic show the primer location for PCR amplification prior to sequencing, while the star denotes the priming spacer. The priming protospacer sequence is indicated by a colored box in the Circos plot. All data are the average of three biological replicates. A) The distribution of newly targeted protospacers mapped to the priming plasmid on the Circos plot reveals a strand bias in self-primed spacer acquisition within the Lens (pLPL) CRISPR-Cas system. The height of the bars indicates the number of spacers mapped to the position on the plasmid, up to 5% of total acquired spacers. B) The distribution of newly targeted protospacers mapped to the priming plasmid shows a similar pattern of self-primed spacer acquisition within the Lens (chromosome) CRISPR-Cas system to that of its pLPL CRISPR-Cas counterpart. Labeling as in (A).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Both of the Lens type I-F CRISPR-Cas systems exhibited a biased distribution of acquired spacers (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, S1) consistent with what has been seen previously for the type I-F systems of <em>Pectobacterium atrosepticum</em> (<a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Richter <em>et al.</em> 2014</a>; <a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Staals <em>et al.</em> 2016</a>) and <em>Pseudomonas aeruginosa</em> (<a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Vorontsova <em>et al.</em> 2015</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Heussler <em>et al.</em> 2016</a>). The majority of the newly targeted protospacers clustered around the priming sequence on the targeted plasmid, with the non-primed strand of DNA (the plus (+) strand) containing ∼75% of these protospacers (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). Consistent with observations in other type I-F systems (<a id="xref-ref-58-5" class="xref-bibr" href="#ref-58">Richter <em>et al.</em> 2014</a>; <a id="xref-ref-67-5" class="xref-bibr" href="#ref-67">Staals <em>et al.</em> 2016</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Heussler <em>et al.</em> 2016</a>), switching the target sequence to the opposite strand led to an acquisition pattern that mirrored the original distribution observed when the (-) strand contained the targeted protospacer (Fig. S2). The spacer length distribution and PAM usage were also consistent with previous observations of other type I-F systems (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Mojica <em>et al.</em> 2009</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cady <em>et al.</em> 2012</a>; <a id="xref-ref-58-6" class="xref-bibr" href="#ref-58">Richter <em>et al.</em> 2014</a>; <a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">Vorontsova <em>et al.</em> 2015</a>; <a id="xref-ref-67-6" class="xref-bibr" href="#ref-67">Staals <em>et al.</em> 2016</a>) (Fig. S2B, C, S3, S4). Taken together, these data suggest that spacer acquisition is qualitatively similar between the chromosomal and pLPL CRISPR-Cas systems.</p></div><div id="sec-12" class="subsection"><h3>Permissive targeting by one system can lead to primed acquisition in the other system</h3><p id="p-32">Since the chromosomal and pLPL CRISPR-Cas systems function in a very similar manner during targeted acquisition and share a high degree of homology within their <em>cas</em> genes and repeat sequences, we hypothesized that the priming of one system might lead to spacer acquisition in the other. Specifically, we tested whether introducing a protospacer-containing plasmid targeted by one CRISPR-Cas system would initiate a primed acquisition response in the second system. Indeed, this led to efficient spacer acquisition on the second array (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>), with patterns largely indistinguishable from what we previously observed on the cognate array (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Characterization of cross-primed spacer acquisition in the two Lens CRISPR-Cas systems. The experimental set-up is the same as described in Figure 5. All data are the average of three biological replicates. The distribution of newly targeted protospacers mapped to the priming plasmid reveals cross-priming between the Lens (chromosome) and Lens (pLPL) CRISPR-Cas systems. (A) shows Lens (chromosome) primed, Lens (pLPL) array examined while (B) shows Lens (pLPL) primed, Lens (chromosome) array examined." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-239459410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Characterization of cross-primed spacer acquisition in the two Lens CRISPR-Cas systems. The experimental set-up is the same as described in Figure 5. All data are the average of three biological replicates. The distribution of newly targeted protospacers mapped to the priming plasmid reveals cross-priming between the Lens (chromosome) and Lens (pLPL) CRISPR-Cas systems. (A) shows Lens (chromosome) primed, Lens (pLPL) array examined while (B) shows Lens (pLPL) primed, Lens (chromosome) array examined.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1039/F6.medium.gif" width="155" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/10/3/1039/F6.medium.gif" width="155" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495916" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-33" class="first-child">Characterization of cross-primed spacer acquisition in the two Lens CRISPR-Cas systems. The experimental set-up is the same as described in Figure 5. All data are the average of three biological replicates. The distribution of newly targeted protospacers mapped to the priming plasmid reveals cross-priming between the Lens (chromosome) and Lens (pLPL) CRISPR-Cas systems. (A) shows Lens (chromosome) primed, Lens (pLPL) array examined while (B) shows Lens (pLPL) primed, Lens (chromosome) array examined.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Primed repopulation of collapsed arrays</h3><p id="p-34">Based on our observations suggesting widespread horizontal inheritance of <em>L. pneumophila</em> type I-F systems (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>), the diversity of spacer sequences (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>), and the ability of closely related systems to prime each other (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>), we next asked whether coincident CRISPR-Cas might provide a mechanism for replenishing collapsed chromosomal arrays. Spacer loss is one of several outcomes when the targeting of a particular sequence becomes detrimental to bacterial survival. In the lab, this occurs when we artificially “force” the coexistence of an efficiently targeted plasmid and an active CRISPR-Cas system through selection (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jiang <em>et al.</em> 2013</a>) (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). Similar events are also likely to occur randomly or when CRISPR-Cas systems acquire self-targeting spacers at a low, but detectable rate (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>) (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Yosef <em>et al.</em> 2012</a>; <a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Datsenko <em>et al.</em> 2012</a>; <a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Savitskaya <em>et al.</em> 2013</a>; <a id="xref-ref-74-5" class="xref-bibr" href="#ref-74">Vorontsova <em>et al.</em> 2015</a>; <a id="xref-ref-67-7" class="xref-bibr" href="#ref-67">Staals <em>et al.</em> 2016</a>; <a id="xref-ref-56-6" class="xref-bibr" href="#ref-56">Rao <em>et al.</em> 2017</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cross-priming between two similar CRISPR-Cas systems can repopulate a collapsed CRISPR array. A bacterium with a CRISPR-Cas system could undergo a mass spacer deletion event through homologous recombination between the first (black) and last (white) repeat sequences. The remaining locus carries a single repeat (white). While such &#x201C;catastrophic collapse&#x201D; events are likely to occur randomly at a certain rate, a driver of such a collapse could be the acquisition of a self-targeting spacer (yellow), selecting for spacer loss. Horizontal acquisition of a second CRISPR-Cas array (e.g., on a plasmid) is a first step toward replenishing the primary array. If cross-priming can occur between this secondary array and the collapsed array, the original CRISPR array is replenished, but bears an observable molecular scar - conversion of all the repeats to the sequence of the last repeat (white)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-239459410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cross-priming between two similar CRISPR-Cas systems can repopulate a collapsed CRISPR array. A bacterium with a CRISPR-Cas system could undergo a mass spacer deletion event through homologous recombination between the first (black) and last (white) repeat sequences. The remaining locus carries a single repeat (white). While such &#x201C;catastrophic collapse&#x201D; events are likely to occur randomly at a certain rate, a driver of such a collapse could be the acquisition of a self-targeting spacer (yellow), selecting for spacer loss. Horizontal acquisition of a second CRISPR-Cas array (&lt;em&gt;e.g.&lt;/em&gt;, on a plasmid) is a first step toward replenishing the primary array. If cross-priming can occur between this secondary array and the collapsed array, the original CRISPR array is replenished, but bears an observable molecular scar - conversion of all the repeats to the sequence of the last repeat (white).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1039/F7.medium.gif" width="141" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/10/3/1039/F7.medium.gif" width="141" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495950" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-35" class="first-child">Cross-priming between two similar CRISPR-Cas systems can repopulate a collapsed CRISPR array. A bacterium with a CRISPR-Cas system could undergo a mass spacer deletion event through homologous recombination between the first (black) and last (white) repeat sequences. The remaining locus carries a single repeat (white). While such “catastrophic collapse” events are likely to occur randomly at a certain rate, a driver of such a collapse could be the acquisition of a self-targeting spacer (yellow), selecting for spacer loss. Horizontal acquisition of a second CRISPR-Cas array (<em>e.g.</em>, on a plasmid) is a first step toward replenishing the primary array. If cross-priming can occur between this secondary array and the collapsed array, the original CRISPR array is replenished, but bears an observable molecular scar - conversion of all the repeats to the sequence of the last repeat (white).</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">In the most extreme instance, an entire array of spacers could be lost through recombination between the first and last repeats. Normally, such a loss could only be reversed by the relatively inefficient mechanism of naïve spacer acquisition (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Yosef <em>et al.</em> 2012</a>; <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Datsenko <em>et al.</em> 2012</a>; <a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Savitskaya <em>et al.</em> 2013</a>). However, if such a collapsed array could be restored through primed acquisition driven by a coincident system, strains with multiple arrays would be inherently more resistant to the loss of CRISPR-Cas protection. Leveraging the two experimentally tractable type I-F systems of <em>L. pneumophila</em> str. Lens, we sought to bioinformatically and experimentally test some of these predictions.</p><p id="p-37">First, we used allelic replacement to generate an <em>L. pneumophila</em> Lens strain in which the entire chromosomal array was replaced by a single copy of its last repeat, mimicking what would occur after complete spacer loss. Next, we transformed two independently derived array deletion strains with the pLPL type I-F priming plasmid (pLPL protospacer 50 plasmid) as above. Using PCR and Illumina sequencing, we observed robust spacer acquisition in the formerly depleted chromosomal CRISPR array (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>, Table S4), indicating that primed acquisition can replenish the completely collapsed array of a coincidental type I-F system. As expected, the consensus repeat sequence of this replenished array adopted the same alternate sequence as the last repeat of the array.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="An experimentally depleted chromosomal CRISPR array can be replenished through the activity of a plasmid-based array. A) Replenishment of a depleted array in the lab. Allelic replacement was used to remove the entire array from the chromosomal Lens I-F system, leaving behind a single, last repeat sequence (see materials and methods). This strain was then transformed with a plasmid targeted by the pLPL (Lens plasmid-based) I-F array previously shown to drive primed acquisition (pLPL protospacer 50; see Figure 4). Spacer acquisition by the empty chromosomal array was analyzed using a PCR based screen where the leader-end of the CRISPR array was amplified and visualized on an agarose gel. Products from the transformed samples (samples 2 and 4) were compared to untransformed controls (samples 1 and 3). Samples 1 and 2 are from depleted array clone #1 and samples 3 and 4 are from depleted array clone #2. B) Repeat signatures of depletion/replenishment. The repeat structure of the experimentally replenished Lens CRISPR arrays resembles that of L. pneumophila str. Alcoy, suggesting a similar array depletion/replenishment event may have occurred within the Alcoy lineage. The frequency of acquiring one new spacer vs. two new spacers following replenishment in the Lens array depletion isolates was determined by Illumina sequencing." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-239459410" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;An experimentally depleted chromosomal CRISPR array can be replenished through the activity of a plasmid-based array. A) Replenishment of a depleted array in the lab. Allelic replacement was used to remove the entire array from the chromosomal Lens I-F system, leaving behind a single, last repeat sequence (see materials and methods). This strain was then transformed with a plasmid targeted by the pLPL (Lens plasmid-based) I-F array previously shown to drive primed acquisition (pLPL protospacer 50; see Figure 4). Spacer acquisition by the empty chromosomal array was analyzed using a PCR based screen where the leader-end of the CRISPR array was amplified and visualized on an agarose gel. Products from the transformed samples (samples 2 and 4) were compared to untransformed controls (samples 1 and 3). Samples 1 and 2 are from depleted array clone #1 and samples 3 and 4 are from depleted array clone #2. B) Repeat signatures of depletion/replenishment. The repeat structure of the experimentally replenished Lens CRISPR arrays resembles that of &lt;em&gt;L. pneumophila&lt;/em&gt; str. Alcoy, suggesting a similar array depletion/replenishment event may have occurred within the Alcoy lineage. The frequency of acquiring one new spacer &lt;em&gt;vs.&lt;/em&gt; two new spacers following replenishment in the Lens array depletion isolates was determined by Illumina sequencing.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1039/F8.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/10/3/1039/F8.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1039/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495958" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-38" class="first-child">An experimentally depleted chromosomal CRISPR array can be replenished through the activity of a plasmid-based array. A) Replenishment of a depleted array in the lab. Allelic replacement was used to remove the entire array from the chromosomal Lens I-F system, leaving behind a single, last repeat sequence (see materials and methods). This strain was then transformed with a plasmid targeted by the pLPL (Lens plasmid-based) I-F array previously shown to drive primed acquisition (pLPL protospacer 50; see <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>). Spacer acquisition by the empty chromosomal array was analyzed using a PCR based screen where the leader-end of the CRISPR array was amplified and visualized on an agarose gel. Products from the transformed samples (samples 2 and 4) were compared to untransformed controls (samples 1 and 3). Samples 1 and 2 are from depleted array clone #1 and samples 3 and 4 are from depleted array clone #2. B) Repeat signatures of depletion/replenishment. The repeat structure of the experimentally replenished Lens CRISPR arrays resembles that of <em>L. pneumophila</em> str. Alcoy, suggesting a similar array depletion/replenishment event may have occurred within the Alcoy lineage. The frequency of acquiring one new spacer <em>vs.</em> two new spacers following replenishment in the Lens array depletion isolates was determined by Illumina sequencing.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">We next examined the CRISPR (repeat) sequences in each of the isolates used in our earlier bioinformatic analyses for similar evidence of complete collapse followed by replenishment. In many of the arrays, there is a consensus repeat that is found throughout the majority of the array, with the last repeat in the array carrying a mutation (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). This has been observed before in other systems (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jansen <em>et al</em>. 2002a</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">2002b</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Horvath <em>et al.</em> 2008</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lopez-Sanchez <em>et al.</em> 2012</a>). We have previously shown that for the <em>L. pneumophila</em> I-C system, complete loss of an array leads overwhelmingly to a recombination product that resembles the alternate last repeat sequence (<a id="xref-ref-56-7" class="xref-bibr" href="#ref-56">Rao <em>et al.</em> 2017</a>). This is presumably because recombination is favored to occur between the region of identity that falls upstream of the mutated nucleotides of the last repeat. A similar sequence structure exists in the type I-F repeats, predicting a similar outcome after complete array loss. In <em>L. pneumophila</em> isolates Alcoy, JFIM01, LBAN01 and LBAV01, their consensus repeat (found throughout the CRISPR array) is identical to their last repeat. Intriguingly, the sequence that remains is identical to the last repeat found in other type I-F isolates – what one would predict if they were the product of complete array collapse followed by subsequent replenishment (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495918/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495918/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><strong>The repeat sequences of L. pneumophila type I-F CRISPR-Cas systems</strong><sup><em><sup>a</sup></em></sup></span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-43">Horizontal gene transfer is a driving force in shaping bacterial biology and pathogenicity (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ochman <em>et al.</em> 2000</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Juhas 2013</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hall <em>et al.</em> 2017</a>). With respect to <em>L. pneumophila</em>, comparative genomic studies have highlighted the importance of horizontal gene transfer, mobile genetic elements, and homologous recombination in shaping the bacterium’s evolutionary trajectory (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cazalet <em>et al.</em> 2004</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">2008</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">D’Auria <em>et al.</em> 2010</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gomez-Valero <em>et al.</em> 2011</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Sánchez-Busó <em>et al.</em> 2014</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gomez-Valero <em>et al.</em> 2014</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">McAdam <em>et al.</em> 2014</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Burstein <em>et al.</em> 2016</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">David <em>et al.</em> 2017</a>). This aspect of <em>L. pneumophila</em> biology extends to the presence and maintenance of plasmid-based CRISPR-Cas systems (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">D’Auria <em>et al.</em> 2010</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Gomez-Valero <em>et al.</em> 2011</a>; <a id="xref-ref-57-11" class="xref-bibr" href="#ref-57">Rao <em>et al.</em> 2016</a>). Our bioinformatic analyses suggest that type I-F CRISPR-Cas systems are horizontally distributed in this species (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). These CRISPR arrays have also undergone extensive spacer acquisition and some spacer loss (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). While only three of the type I-F CRISPR-Cas systems we describe are present on annotated plasmids (str. Lens, str. Mississauga-2006 and str. C8_S, <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>), this is likely an underestimation due to the nature of analyzing draft genomes. Notably, an additional six strains (FJAD01, FJAJ01, FJAM01, FJBU01, FJBW01 and FJMD01, <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>) have systems that reside on contigs with overlapping sequence (78-89 nt) on either end, suggesting that they may also reside on plasmids or other mobile elements. Indeed, horizontal acquisition of type I-F systems is well-established in other species. For instance, in <em>Vibrio</em> species, 97% of identified type I-F systems are encoded on mobile genetic elements (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">McDonald <em>et al.</em> 2019</a>). Like <em>L. pneumophila</em>, it’s hypothesized these systems have been acquired by their host through horizontal gene transfer (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">McDonald <em>et al.</em> 2019</a>). Our data, however, suggest that such transfer may not be merely a mechanism by which isolates acquire CRISPR-Cas protection, but could also maintain existing defensive capabilities through the unique spacer dynamics provided by two inter-priming arrays. These inter-priming arrays can be part of two different systems, as demonstrated by our data, but could also occur in a system where two (or more) different arrays share a set of <em>cas</em> genes (<a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Swarts <em>et al.</em> 2012</a>; <a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Datsenko <em>et al.</em> 2012</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Staals <em>et al.</em> 2013</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">2014</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Majumdar <em>et al.</em> 2015</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Elmore <em>et al.</em> 2015</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Silas <em>et al.</em> 2017</a>). For example, in <em>E. coli</em> str. K12, it has been shown that two different CRISPR arrays can be populated by the same set of <em>cas</em> genes (<a id="xref-ref-71-4" class="xref-bibr" href="#ref-71">Swarts <em>et al.</em> 2012</a>; <a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Datsenko <em>et al.</em> 2012</a>).</p><p id="p-44">We propose that when a bacterium acquires a second, closely related CRISPR-Cas system, it gains a mechanism by which depleted CRISPR arrays can be repopulated (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). We have modeled such an event in the lab, made predictions about what the signatures of such events would be, and provide genomic data to suggest it may have occurred on several occasions within our collection of sequenced isolates. One obvious line of future investigation would be to observe whether such patterns are present in other species with mobilized type I CRISPR-Cas and perhaps absent in instances where CRISPR-Cas is acquired primarily through vertical inheritance. Such signatures may be a way to detect these events even if they are rare or transient. Notably, we have observed only one strain with two type I-F CRISPR-Cas systems (<em>L. pneumophila</em> str. Lens). Two additional strains (<em>L. pneumophila</em> str. Mississauga-2006 and FJAD01) possesses a type I-C system on its chromosome and a type I-F system on a plasmid. These data suggest that <em>stable</em> co-occurrence of CRISPR-Cas systems appears to be rare in the <em>L. pneumophila</em> isolates described so far.</p><p id="p-45">Lastly, while we think of spacer loss as a predominantly negative event (loss of protection), it is likely to play a more nuanced role in the maintenance of CRISPR-Cas activity. Jiang and colleagues have presented work that demonstrated loss of CRISPR-Cas to maintain beneficial plasmids and discussed the trade-offs of CRISPR immunity <em>vs.</em> beneficial genetic elements (<em>i.e.</em>: confers resistance, etc.) (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Jiang <em>et al.</em> 2013</a>). Using a conjugation assay with a targeted resistance plasmid, they observed loss of the spacer targeting the plasmid (13% of the transconjugants), mutations in CRISPR-Cas that abolished its function (37% of the transconjugants) or partial/complete deletion of the CRISPR-Cas locus (50% of the transconjugants) (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Jiang <em>et al.</em> 2013</a>). Their competition data suggested that even with loss of the entire system, there was little to no fitness cost associated with this loss (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Jiang <em>et al.</em> 2013</a>).</p><p id="p-46">Array length is the product of a dynamic process whose impact on adaptation, expression, and interference remains largely unexplored. Many of the type I-F systems in <em>L. pneumophila</em> have different array lengths, ranging from 8 spacers to 129 spacers, with an average length of 61 spacers (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). A global analysis of class I CRISPR arrays found that the average array length for type I-F systems was 33 spacers, with statistically significant differences between the array lengths of different type I subtypes (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Toms and Barrangou 2017</a>). Accordingly, if spacer acquisition is a driving force in array divergence, it is likely coupled to spacer loss. Close examination of the mechanisms driving spacer loss in these systems - and their subsequent impact on CRISPR-Cas functionality - will be crucial to further testing the model of array diversification in <em>L. pneumophila</em>.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">The authors thank Griffin Deecker (a volunteer high school student) for his assistance in bioinformatically examining the diversity of I-F repeat sequences in <em>L. pneumophila</em>, Chitong Rao for his contributions to experimental design, Kamran Rizzolo for discussions regarding phylogenetic analyses and Harley Mount for help with initial phylogenetic analyses. We also thank the Centre for the Analysis of Genome Evolution and Function (CAGEF) at the University of Toronto for performing Illumina sequencing. We thank members of the Ensminger laboratory for their suggestions and careful reading of the manuscript, in particular Beth Nicholson and Malene Urbanus. SRD is supported by a fellowship from the Department of Biochemistry, University of Toronto and an Ontario Graduate Scholarship. This work was supported by a Project Grant from the Canadian Institutes of Health Research (PHT-148819), the Connaught Fund (NR-2015-16), and an infrastructure grant from the Canada Foundation for Innovation and the Ontario Research Fund (30364) to AWE.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-48">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11367821">https://doi.org/10.25387/g3.11367821</a>.</p></li><li class="fn" id="fn-6"><p id="p-49"><em>Communicating editor: A. Wong</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-10-07"><span class="received-label">Received </span>October 7, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2020-01-10"><span class="accepted-label">Accepted </span>January 10, 2020.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Deecker <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>,  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Gish</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. W.</span> <span class="cit-name-surname">Myers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Lipman</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic Local Alignment Search Tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.2" data-doi="10.1126/science.1138140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrangou</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fremaux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Deveau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Richards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Boyaval</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CRISPR provides acquired resistance against viruses in prokaryotes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">315</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1712</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1138140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBarrangou%26rft.auinit1%253DR.%26rft.volume%253D315%26rft.issue%253D5819%26rft.spage%253D1709%26rft.epage%253D1712%26rft.atitle%253DCRISPR%2BProvides%2BAcquired%2BResistance%2BAgainst%2BViruses%2Bin%2BProkaryotes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1138140%26rft_id%253Dinfo%253Apmid%252F17379808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTUvNTgxOS8xNzA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.3" data-doi="10.1186/s12864-016-2627-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biswas</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H. J.</span> <span class="cit-name-surname">Staals</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Morales</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Fineran</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Brown</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">CRISPRDetect: A flexible algorithm to define CRISPR arrays.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2627-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2627-0%26rft_id%253Dinfo%253Apmid%252F27184979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2627-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27184979&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.4" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.5" data-doi="10.7326/0003-4819-90-4-656"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>,  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Steigerwalt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">McDade</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Classification of the Legionnaires’ Disease Bacterium: <em>Legionella pneumophila</em>, genus novum, species nova, of the Family Legionellaceae, familia nova.</span> <abbr class="cit-jnl-abbrev">Ann. Intern. Med.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">658</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7326/0003-4819-90-4-656</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2BInternal%2BMedicine%26rft.stitle%253DANN%2BINTERN%2BMED%26rft.aulast%253DBRENNER%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D90%26rft.issue%253D4%26rft.spage%253D656%26rft.epage%253D658%26rft.atitle%253DClassification%2Bof%2Bthe%2BLegionnaires%2527%2BDisease%2BBacterium%253A%2BLegionella%2Bpneumophila%252C%2Bgenus%2Bnovum%252C%2Bspecies%2Bnova%252C%2Bof%2Bthe%2BFamily%2BLegionellaceae%252C%2Bfamilia%2Bnova%26rft_id%253Dinfo%253Adoi%252F10.7326%252F0003-4819-90-4-656%26rft_id%253Dinfo%253Apmid%252F434652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7326/0003-4819-90-4-656&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=434652&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.6" data-doi="10.1126/science.1159689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brouns</span>,  <span class="cit-name-given-names">S. J. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Jore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lundgren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Westra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J. H.</span> <span class="cit-name-surname">Slijkhuis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Small CRISPR RNAs guide antiviral defense in prokaryotes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">321</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">964</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1159689</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBrouns%26rft.auinit1%253DS.%2BJ.%2BJ.%26rft.volume%253D321%26rft.issue%253D5891%26rft.spage%253D960%26rft.epage%253D964%26rft.atitle%253DSmall%2BCRISPR%2BRNAs%2BGuide%2BAntiviral%2BDefense%2Bin%2BProkaryotes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1159689%26rft_id%253Dinfo%253Apmid%252F18703739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjEvNTg5MS85NjAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.7" data-doi="10.1038/ng.3481"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burstein</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Amaro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Zusman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Lifshitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Cohen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic analysis of 38 <em>Legionella</em> species identifies large and diverse effector repertoires.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">175</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3481</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3481%26rft_id%253Dinfo%253Apmid%252F26752266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3481&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26752266&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.8" data-doi="10.1128/JB.01184-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cady</span>,  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bondy-Denomy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. E.</span> <span class="cit-name-surname">Heussler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Davidson</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The CRISPR/Cas adaptive immune system of <em>Pseudomonas aeruginosa</em> mediates resistance to naturally occurring and engineered phages.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">5728</span>–<span class="cit-lpage">5738</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.01184-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01184-12%26rft_id%253Dinfo%253Apmid%252F22885297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5NC8yMS81NzI4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.9" data-doi="10.1101/gr.7229808"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cazalet</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jarraud</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ghavi-Helm</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Kunst</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Glaser</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multigenome analysis identifies a worldwide distributed epidemic <em>Legionella pneumophila</em> clone that emerged within a highly diverse species.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.7229808</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.7229808%26rft_id%253Dinfo%253Apmid%252F18256241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzMvNDMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.10" data-doi="10.1038/ng1447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cazalet</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Rusniok</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Brüggemann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Zidane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Magnier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Evidence in the <em>Legionella pneumophila</em> genome for exploitation of host cell functions and high genome plasticity.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1173</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1447</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCazalet%26rft.auinit1%253DC.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1165%26rft.epage%253D1173%26rft.atitle%253DEvidence%2Bin%2Bthe%2BLegionella%2Bpneumophila%2Bgenome%2Bfor%2Bexploitation%2Bof%2Bhost%2Bcell%2Bfunctions%2Band%2Bhigh%2Bgenome%2Bplasticity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1447%26rft_id%253Dinfo%253Apmid%252F15467720%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15467720&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.11" data-doi="10.1093/nar/gky425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Couvin</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bernheim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Toffano-Nioche</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Touchon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Michalik</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">CRISPRCasFinder, an update of CRISPRFinder, includes a portable version, enhanced performance and integrates search for Cas proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">W246</span>–<span class="cit-lpage">W251</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky425</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DW246%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.12" data-doi="10.1186/1471-2164-11-181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’Auria</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Jimenez-Hernandez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Peris-Bondia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Moya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Latorre</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Legionella pneumophila</em> pangenome reveals strain-specific virulence factors.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">181</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-11-181</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DD%2527Auria%26rft.auinit1%253DG.%26rft.volume%253D11%26rft.spage%253D181%26rft.epage%253D181%26rft.atitle%253DLegionella%2Bpneumophila%2Bpangenome%2Breveals%2Bstrain-specific%2Bvirulence%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-181%26rft_id%253Dinfo%253Apmid%252F20236513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20236513&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.13" data-doi="10.1038/ncomms1937"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Datsenko</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Pougach</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tikhonov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Wanner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. V.</span> <span class="cit-name-surname">Severinov</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Molecular memory of prior infections activates the CRISPR/Cas adaptive bacterial immunity system.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">947</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms1937</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DDatsenko%26rft.auinit1%253DK.%2BA.%26rft.volume%253D3%26rft.spage%253D945%26rft.epage%253D945%26rft.atitle%253DMolecular%2Bmemory%2Bof%2Bprior%2Binfections%2Bactivates%2Bthe%2BCRISPR%252FCas%2Badaptive%2Bbacterial%2Bimmunity%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1937%26rft_id%253Dinfo%253Apmid%252F22781758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1937&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22781758&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.1039.14" data-doi="10.1371/journal.pgen.1006855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sánchez-Busó</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Harris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Marttinen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Rusniok</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Dynamics and impact of homologous recombination on the evolution of <em>Legionella pneumophila</em>.</span> PLoS Genet <span class="cit-vol">13</span>: <span class="cit-fpage">e1006855</span>. Erratum: 13:1007116.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1006855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D13%26rft.spage%253De1006855%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1006855%26rft_id%253Dinfo%253Apmid%252F28650958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1006855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28650958&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.15" data-doi="10.1128/JB.01412-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deveau</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Barrangou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Garneau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Labonte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fremaux</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phage response to CRISPR-encoded resistance in <em>Streptococcus thermophilus</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1390</span>–<span class="cit-lpage">1400</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.01412-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01412-07%26rft_id%253Dinfo%253Apmid%252F18065545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzEzOTAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.16" data-doi="10.3390/v10110602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dion</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Labrie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Shah</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Moineau</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">CRISPRStudio: A User-Friendly Software for Rapid CRISPR Array Visualization.</span> <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">602</span>–<span class="cit-lpage">611</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/v10110602</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D10%26rft.spage%253D602%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv10110602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v10110602&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.17" data-doi="10.1186/1471-2105-5-113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-5-113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D5%26rft.spage%253D113%26rft.epage%253D113%26rft.atitle%253DMUSCLE%253A%2Ba%2Bmultiple%2Bsequence%2Balignment%2Bmethod%2Bwith%2Breduced%2Btime%2Band%2Bspace%2Bcomplexity.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-113%26rft_id%253Dinfo%253Apmid%252F15318951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15318951&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.18" data-doi="10.1093/nar/gkv1140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elmore</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Deighan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Westpheling</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Terns</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Terns</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">DNA targeting by the type I-G and type I-A CRISPR–Cas systems of <em>Pyrococcus furiosus</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">10353</span>–<span class="cit-lpage">10363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1140</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1140%26rft_id%253Dinfo%253Apmid%252F26519471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26519471&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.19" data-doi="10.1073/pnas.1400071111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fineran</span>,  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J. H.</span> <span class="cit-name-surname">Gerritzen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Suarez-Diez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Kunne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Boekhorst</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Degenerate target sites mediate rapid primed CRISPR adaptation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E1629</span>–<span class="cit-lpage">E1638</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1400071111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1400071111%26rft_id%253Dinfo%253Apmid%252F24711427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzE2L0UxNjI5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.20" data-doi="10.1186/1471-2164-12-536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Valero</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Rusniok</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jarraud</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Vacherie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Rouy</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extensive recombination events and horizontal gene transfer shaped the <em>Legionella pneumophila</em> genomes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">536</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-12-536</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGomez%2BValero%26rft.auinit1%253DL.%26rft.volume%253D12%26rft.spage%253D536%26rft.epage%253D536%26rft.atitle%253DExtensive%2Brecombination%2Bevents%2Band%2Bhorizontal%2Bgene%2Btransfer%2Bshaped%2Bthe%2BLegionella%2Bpneumophila%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-536%26rft_id%253Dinfo%253Apmid%252F22044686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22044686&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.21" data-doi="10.1186/s13059-014-0505-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gomez-Valero</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Rusniok</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Rolando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Neou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dervins-Ravault</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative analyses of <em>Legionella</em> species identifies genetic features of strains causing Legionnaires’ disease.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">505</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/PREACCEPT-1086350395137407</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D505%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0505-0%26rft_id%253Dinfo%253Apmid%252F25370836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0505-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25370836&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.22" data-doi="10.1098/rstb.2016.0424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>,  <span class="cit-name-given-names">J. P. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Brockhurst</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Harrison</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Sampling the mobile gene pool: innovation via horizontal gene transfer in bacteria.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">372</span>: <span class="cit-fpage">20160424</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2016.0424</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2016.0424%26rft_id%253Dinfo%253Apmid%252F29061896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2016.0424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29061896&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.23" data-doi="10.1126/science.1192272"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haurwitz</span>,  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jinek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Wiedenheft</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Zhou</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Doudna</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Sequence- and structure-specific RNA processing by a CRISPR endonuclease.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">1355</span>–<span class="cit-lpage">1358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1192272</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHaurwitz%26rft.auinit1%253DR.%2BE.%26rft.volume%253D329%26rft.issue%253D5997%26rft.spage%253D1355%26rft.epage%253D1358%26rft.atitle%253DSequence-%2Band%2BStructure-Specific%2BRNA%2BProcessing%2Bby%2Ba%2BCRISPR%2BEndonuclease%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1192272%26rft_id%253Dinfo%253Apmid%252F20829488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjkvNTk5Ny8xMzU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.1039.24" data-doi="10.1128/JB.00458-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heussler</span>,  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Price</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Collins</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">O’Toole</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Requirements for <em>Pseudomonas aeruginosa</em> Type I-F CRISPR-Cas Adaptation Determined Using a Biofilm Enrichment Assay.</span> J. Bacteriol. <span class="cit-vol">198</span>: <span class="cit-fpage">3080</span>–<span class="cit-lpage">3090</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.00458-16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00458-16%26rft_id%253Dinfo%253Apmid%252F27573013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5OC8yMi8zMDgwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.25" data-doi="10.1128/JB.01415-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horvath</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Romero</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Coute-Monvoisin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Richards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Deveau</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Diversity, activity, and evolution of CRISPR loci in <em>Streptococcus thermophilus</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1401</span>–<span class="cit-lpage">1412</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.01415-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01415-07%26rft_id%253Dinfo%253Apmid%252F18065539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE0MDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.26" data-doi="10.1038/ncomms5399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hynes</span>,  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Villion</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Moineau</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Adaptation in bacterial CRISPR-Cas immunity can be driven by defective phages.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms5399</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D4399%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms5399%26rft_id%253Dinfo%253Apmid%252F25056268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms5399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25056268&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.27" data-doi="10.1128/jb.169.12.5429-5433.1987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishino</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Shinagawa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Makino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Amemura</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nakata</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Nucleotide sequence of the <em>iap</em> gene, responsible for alkaline phosphatase isozyme conversion in <em>Escherichia coli</em>, and identification of the gene product.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">5429</span>–<span class="cit-lpage">5433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.169.12.5429-5433.1987</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DIshino%26rft.auinit1%253DY.%26rft.volume%253D169%26rft.issue%253D12%26rft.spage%253D5429%26rft.epage%253D5433%26rft.atitle%253DNucleotide%2Bsequence%2Bof%2Bthe%2Biap%2Bgene%252C%2Bresponsible%2Bfor%2Balkaline%2Bphosphatase%2Bisozyme%2Bconversion%2Bin%2BEscherichia%2Bcoli%252C%2Band%2Bidentification%2Bof%2Bthe%2Bgene%2Bproduct.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.169.12.5429-5433.1987%26rft_id%253Dinfo%253Apmid%252F3316184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE2OS8xMi81NDI5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.28" data-doi="10.1016/j.chom.2018.12.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Birkholz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Malone</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Fineran</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Imprecise Spacer Acquisition Generates CRISPR-Cas Immune Diversity through Primed Adaptation.</span> <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">260.e4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.chom.2018.12.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BHost%2BMicrobe%26rft.volume%253D25%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2018.12.014%26rft_id%253Dinfo%253Apmid%252F30661951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2018.12.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30661951&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.29" data-doi="10.1126/science.aal5056"><div class="cit-metadata"><cite>Jackson, S. A., R. E. McKenzie, R. D. Fagerlund, S. N. Kieper, P. C. Fineran <em>et al.</em>, 2017 CRISPR-Cas: Adapting to change. Science 356: eaal5056–11.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aal5056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJackson%26rft.auinit1%253DS.%2BA.%26rft.volume%253D356%26rft.issue%253D6333%26rft.spage%253Deaal5056%26rft.epage%253Deaal5056%26rft.atitle%253DCRISPR-Cas%253A%2BAdapting%2Bto%2Bchange%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aal5056%26rft_id%253Dinfo%253Apmid%252F28385959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE3OiIzNTYvNjMzMy9lYWFsNTA1NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTAzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.30" data-doi="10.1089/15362310252780816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">van Embden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Gaastra</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Schouls</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">a</span> <span class="cit-article-title">Identification of a novel family of sequence repeats among prokaryotes.</span> <abbr class="cit-jnl-abbrev">OMICS</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/15362310252780816</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOmics%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bintegrative%2Bbiology%26rft.stitle%253DOMICS%26rft.aulast%253DJansen%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D23%26rft.epage%253D33%26rft.atitle%253DIdentification%2Bof%2Ba%2Bnovel%2Bfamily%2Bof%2Bsequence%2Brepeats%2Bamong%2Bprokaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F15362310252780816%26rft_id%253Dinfo%253Apmid%252F11883425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/15362310252780816&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11883425&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.31" data-doi="10.1046/j.1365-2958.2002.02839.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">van Embden</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Gaastra</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Schouls</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">b</span> <span class="cit-article-title">Identification of genes that are associated with DNA repeats in prokaryotes.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1565</span>–<span class="cit-lpage">1575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2958.2002.02839.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DJansen%26rft.auinit1%253DR.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D1565%26rft.epage%253D1575%26rft.atitle%253DIdentification%2Bof%2Bgenes%2Bthat%2Bare%2Bassociated%2Bwith%2BDNA%2Brepeats%2Bin%2Bprokaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2002.02839.x%26rft_id%253Dinfo%253Apmid%252F11952905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2002.02839.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11952905&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174710000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.1039.32" data-doi="10.1371/journal.pgen.1003844"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Maniv</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Arain</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Levin</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dealing with the evolutionary downside of CRISPR immunity: bacteria and beneficial plasmids.</span> PLoS Genet <span class="cit-vol">9</span>: <span class="cit-fpage">e1003844</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003844</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D9%26rft.spage%253De1003844%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003844%26rft_id%253Dinfo%253Apmid%252F24086164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003844&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24086164&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.33" data-doi="10.1038/nsmb.2019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jore</span>,  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lundgren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">van Duijn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Bultema</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Westra</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Structural basis for CRISPR RNA-guided DNA recognition by Cascade.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">536</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nsmb.2019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DJore%26rft.auinit1%253DM.%2BM.%26rft.volume%253D18%26rft.issue%253D5%26rft.spage%253D529%26rft.epage%253D536%26rft.atitle%253DStructural%2Bbasis%2Bfor%2BCRISPR%2BRNA-guided%2BDNA%2Brecognition%2Bby%2BCascade.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2019%26rft_id%253Dinfo%253Apmid%252F21460843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21460843&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290205200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.34" data-doi="10.3109/1040841X.2013.804031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Juhas</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Horizontal gene transfer in human pathogens.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Microbiol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3109/1040841X.2013.804031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.3109%252F1040841X.2013.804031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3109/1040841X.2013.804031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.35" data-doi="10.1016/j.mib.2017.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>,  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Makarova</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Diversity, classification and evolution of CRISPR-Cas systems.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mib.2017.05.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BMicrobiol.%26rft.volume%253D37%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2017.05.008%26rft_id%253Dinfo%253Apmid%252F28605718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2017.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28605718&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.36" data-doi="10.1101/gr.092759.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krzywinski</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Birol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Connors</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Gascoyne</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Circos: an information aesthetic for comparative genomics.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.092759.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092759.109%26rft_id%253Dinfo%253Apmid%252F19541911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTYzOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTAzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.37" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Stecher</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tamura</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1870</span>–<span class="cit-lpage">1874</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.38" data-doi="10.1093/nar/gkw914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuznedelov</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Mekler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lemak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tokmina-Lukaszewska</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Datsenko</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Altered stoichiometry <em>Escherichia coli</em> Cascade complexes with shortened CRISPR RNA spacers are capable of interference and primed adaptation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">10849</span>–<span class="cit-lpage">10861</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw914</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw914%26rft_id%253Dinfo%253Apmid%252F27738137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw914&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27738137&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.39" data-doi="10.1038/nature14302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Goren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Yosef</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Auster</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Manor</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CRISPR adaptation biases explain preference for acquisition of foreign DNA.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">520</span>: <span class="cit-fpage">505</span>–<span class="cit-lpage">510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature14302</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D520%26rft.spage%253D505%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14302%26rft_id%253Dinfo%253Apmid%252F25874675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25874675&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.40" data-doi="10.1093/nar/gku389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Xiang</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title"><em>Haloarcula hispanica</em> CRISPR authenticates PAM of a target sequence to prime discriminative adaptation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">7226</span>–<span class="cit-lpage">7235</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku389</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku389%26rft_id%253Dinfo%253Apmid%252F24803673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24803673&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000338769400042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.41" data-doi="10.1093/nar/gkt1154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhao</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Xiang</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Adaptation of the <em>Haloarcula hispanica</em> CRISPR-Cas system to a purified virus strictly requires a priming process.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2483</span>–<span class="cit-lpage">2492</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt1154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1154%26rft_id%253Dinfo%253Apmid%252F24265226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1154&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24265226&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332381000040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.42" data-doi="10.1093/nar/gkx229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhou</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Xiang</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The spacer size of I-B CRISPR is modulated by the terminal sequence of the protospacer.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">4642</span>–<span class="cit-lpage">4654</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx229</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253D4642%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx229%26rft_id%253Dinfo%253Apmid%252F28379481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28379481&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.43" data-doi="10.1111/j.1365-2958.2012.08172.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Sanchez</span>,  <span class="cit-name-given-names">M.-J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Sauvage</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Da Cunha</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Clermont</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Ratsima Hariniaina</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The highly dynamic CRISPR1 system of <em>Streptococcus agalactiae</em> controls the diversity of its mobilome.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">1057</span>–<span class="cit-lpage">1071</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.2012.08172.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DLopez-Sanchez%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D85%26rft.issue%253D6%26rft.spage%253D1057%26rft.epage%253D1071%26rft.atitle%253DThe%2Bhighly%2Bdynamic%2BCRISPR1%2Bsystem%2Bof%2BStreptococcus%2Bagalactiae%2Bcontrols%2Bthe%2Bdiversity%2Bof%2Bits%2Bmobilome.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2012.08172.x%26rft_id%253Dinfo%253Apmid%252F22834929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2012.08172.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22834929&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.44" data-doi="10.1093/bioinformatics/btr507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magoc</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FLASH: fast length adjustment of short reads to improve genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2957</span>–<span class="cit-lpage">2963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr507</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr507%26rft_id%253Dinfo%253Apmid%252F21903629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr507&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903629&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.45" data-doi="10.1261/rna.049130.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Majumdar</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. T.</span> <span class="cit-name-surname">Pfister</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Compton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Olson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Three CRISPR-Cas immune effector complexes coexist in <em>Pyrococcus furiosus</em>.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1147</span>–<span class="cit-lpage">1158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.049130.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.049130.114%26rft_id%253Dinfo%253Apmid%252F25904135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjIxLzYvMTE0NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTAzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.46" data-doi="10.1038/nrmicro2577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makarova</span>,  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Haft</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Barrangou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J. J.</span> <span class="cit-name-surname">Brouns</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Charpentier</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution and classification of the CRISPR–Cas systems.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrmicro2577</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DMakarova%26rft.auinit1%253DK.%2BS.%26rft.volume%253D9%26rft.issue%253D6%26rft.spage%253D467%26rft.epage%253D477%26rft.atitle%253DEvolution%2Band%2Bclassification%2Bof%2Bthe%2BCRISPR-Cas%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2577%26rft_id%253Dinfo%253Apmid%252F21552286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21552286&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290614100019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.47" data-doi="10.1038/nrmicro3569"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makarova</span>,  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. I.</span> <span class="cit-name-surname">Wolf</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. S.</span> <span class="cit-name-surname">Alkhnbashi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Costa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Shah</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">An updated evolutionary classification of CRISPR–Cas systems.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrmicro3569</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol.%26rft.volume%253D13%26rft.spage%253D722%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro3569%26rft_id%253Dinfo%253Apmid%252F26411297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro3569&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26411297&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.48" data-doi="10.1126/science.1165771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marraffini</span>,  <span class="cit-name-given-names">L. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Sontheimer</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">CRISPR interference limits horizontal gene transfer in Staphylococci by targeting DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">1843</span>–<span class="cit-lpage">1845</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1165771</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMarraffini%26rft.auinit1%253DL.%2BA.%26rft.volume%253D322%26rft.issue%253D5909%26rft.spage%253D1843%26rft.epage%253D1845%26rft.atitle%253DCRISPR%2BInterference%2BLimits%2BHorizontal%2BGene%2BTransfer%2Bin%2BStaphylococci%2Bby%2BTargeting%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1165771%26rft_id%253Dinfo%253Apmid%252F19095942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjIvNTkwOS8xODQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.49" data-doi="10.1186/gb-2014-15-1-r1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McAdam</span>,  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Vander Broek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Lindsay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Ward</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Hanson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Gene flow in environmental <em>Legionella pneumophila</em> leads to genetic and pathogenic heterogeneity within a Legionnaires’ disease outbreak.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0504-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2014-15-1-r1%26rft_id%253Dinfo%253Apmid%252F24393533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2014-15-1-r1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24393533&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.50"><div class="cit-metadata"><cite>McDonald, N. D., A. Regmi, D. P. Morreale, J. D. Borowski, and E. F. Boyd, 2019 CRISPR-Cas systems are present predominantly on mobile genetic elements in <em>Vibrio</em> species. BMC Genomics. 20:1–23.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-019-5439-1</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.51" data-doi="10.1038/nature21719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Modell</span>,  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Jiang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. A.</span> <span class="cit-name-surname">Marraffini</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CRISPR–Cas systems exploit viral DNA injection to establish and maintain adaptive immunity.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">544</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">104</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature21719</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D544%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature21719%26rft_id%253Dinfo%253Apmid%252F28355179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature21719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28355179&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.52" data-doi="10.1099/mic.0.023960-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mojica</span>,  <span class="cit-name-given-names">F. J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Díez-Villaseñor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">García-Martínez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Almendros</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Short motif sequences determine the targets of the prokaryotic CRISPR defence system.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">733</span>–<span class="cit-lpage">740</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/mic.0.023960-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.stitle%253DMicrobiology%26rft.aulast%253DMojica%26rft.auinit1%253DF.%2BJ.%2BM.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D733%26rft.epage%253D740%26rft.atitle%253DShort%2Bmotif%2Bsequences%2Bdetermine%2Bthe%2Btargets%2Bof%2Bthe%2Bprokaryotic%2BCRISPR%2Bdefence%2Bsystem%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.023960-0%26rft_id%253Dinfo%253Apmid%252F19246744%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.023960-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19246744&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264515100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.53" data-doi="10.1046/j.1365-2958.2000.01838.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mojica</span>,  <span class="cit-name-given-names">F. J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Díez-Villaseñor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Soria</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Juez</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Biological significance of a family of regularly spaced repeats in the genomes of archaea, bacteria and mitochondria.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">244</span>–<span class="cit-lpage">246</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2958.2000.01838.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DMojica%26rft.auinit1%253DF.%2BJ.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D244%26rft.epage%253D246%26rft.atitle%253DBiological%2Bsignificance%2Bof%2Ba%2Bfamily%2Bof%2Bregularly%2Bspaced%2Brepeats%2Bin%2Bthe%2Bgenomes%2Bof%2BArchaea%252C%2BBacteria%2Band%2Bmitochondria.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2000.01838.x%26rft_id%253Dinfo%253Apmid%252F10760181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2000.01838.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10760181&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086338200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.54" data-doi="10.1038/35012500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Lawrence</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Groisman</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Lateral gene transfer and the nature of bacterial innovation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">405</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35012500</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOchman%26rft.auinit1%253DH.%26rft.volume%253D405%26rft.issue%253D6784%26rft.spage%253D299%26rft.epage%253D304%26rft.atitle%253DLateral%2Bgene%2Btransfer%2Band%2Bthe%2Bnature%2Bof%2Bbacterial%2Binnovation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35012500%26rft_id%253Dinfo%253Apmid%252F10830951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35012500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10830951&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087085700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.55" data-doi="10.1093/bioinformatics/btv421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>,  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Cummins</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hunt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. K.</span> <span class="cit-name-surname">Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Reuter</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Roary: rapid large-scale prokaryote pan genome analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3691</span>–<span class="cit-lpage">3693</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv421%26rft_id%253Dinfo%253Apmid%252F26198102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26198102&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.56" data-doi="10.1261/rna.062083.117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Chin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. W.</span> <span class="cit-name-surname">Ensminger</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Priming in a permissive type I–C CRISPR–Cas system reveals distinct dynamics of spacer acquisition and loss.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1525</span>–<span class="cit-lpage">1538</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.062083.117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.062083.117%26rft_id%253Dinfo%253Apmid%252F28724535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMC8xNTI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.57" data-doi="10.1111/cmi.12586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Guyard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Pelaz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wasserscheid</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bondy-Denomy</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Active and adaptive <em>Legionella</em> CRISPR-Cas reveals a recurrent challenge to the pathogen.</span> <abbr class="cit-jnl-abbrev">Cell. Microbiol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1319</span>–<span class="cit-lpage">1338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/cmi.12586</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell.%2BMicrobiol.%26rft.volume%253D18%26rft.spage%253D1319%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fcmi.12586%26rft_id%253Dinfo%253Apmid%252F26936325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/cmi.12586&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26936325&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.58" data-doi="10.1093/nar/gku527"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Dy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">McKenzie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. N. J.</span> <span class="cit-name-surname">Watson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Taylor</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Priming in the Type I-F CRISPR-Cas system triggers strand-independent spacer acquisition, bi-directionally from the primed protospacer.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">8516</span>–<span class="cit-lpage">8526</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku527</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku527%26rft_id%253Dinfo%253Apmid%252F24990370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24990370&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000341515400034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.59" data-doi="10.1038/nrmicro3096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samson</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Magadan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sabri</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Moineau</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Revenge of the phages: defeating bacterial defences.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">687</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrmicro3096</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol.%26rft.volume%253D11%26rft.spage%253D675%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro3096%26rft_id%253Dinfo%253Apmid%252F23979432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro3096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23979432&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.60" data-doi="10.4161/rna.24325"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savitskaya</span>,  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Semenova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Dedkov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Metlitskaya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. V.</span> <span class="cit-name-surname">Severinov</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-throughput analysis of type I-E CRISPR/Cas spacer acquisition in <em>E. coli</em>.</span> <abbr class="cit-jnl-abbrev">RNA Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">725</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.4161/rna.24325</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiol.%26rft.volume%253D10%26rft.spage%253D716%26rft_id%253Dinfo%253Adoi%252F10.4161%252Frna.24325%26rft_id%253Dinfo%253Apmid%252F23619643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/rna.24325&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23619643&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323176900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.61" data-doi="10.1038/ng.3114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sánchez-Busó</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Comas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Jorques</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">González-Candelas</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Recombination drives genome evolution in outbreak-related <em>Legionella pneumophila</em> isolates.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1205</span>–<span class="cit-lpage">1211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D1205%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3114%26rft_id%253Dinfo%253Apmid%252F25282102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25282102&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.62" data-doi="10.1093/bioinformatics/btu153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seemann</span>,  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Prokka: rapid prokaryotic genome annotation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2068</span>–<span class="cit-lpage">2069</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu153%26rft_id%253Dinfo%253Apmid%252F24642063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24642063&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339814300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.63" data-doi="10.1073/pnas.1104144108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Semenova</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Jore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Datsenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Semenova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Westra</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Interference by clustered regularly interspaced short palindromic repeat (CRISPR) RNA is governed by a seed sequence.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">10098</span>–<span class="cit-lpage">10103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1104144108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1104144108%26rft_id%253Dinfo%253Apmid%252F21646539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI1LzEwMDk4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.64" data-doi="10.7554/eLife.27601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silas</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Lucas-Elio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Jackson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. Aroca-Crevillén</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Hansen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Type III CRISPR-Cas systems can provide redundancy to counteract viral escape from type I systems.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>:e27601. Erratum: <span class="cit-pub-date">2018</span>;<span class="cit-vol">7</span>: <span class="cit-fpage">e36853</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.27601</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D7%26rft.spage%253De36853%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.27601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.27601&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.65" data-doi="10.1128/IAI.68.5.2939-2947.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solomon</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rupper</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Cardelli</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Isberg</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Intracellular growth of <em>Legionella pneumophila</em> in <em>Dictyostelium discoideum</em>, a system for genetic analysis of host-pathogen interactions.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">2939</span>–<span class="cit-lpage">2947</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/IAI.68.5.2939-2947.2000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DSolomon%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D68%26rft.issue%253D5%26rft.spage%253D2939%26rft.epage%253D2947%26rft.atitle%253DIntracellular%2BGrowth%2Bof%2BLegionella%2Bpneumophila%2Bin%2BDictyostelium%2Bdiscoideum%252C%2Ba%2BSystem%2Bfor%2BGenetic%2BAnalysis%2Bof%2BHost-Pathogen%2BInteractions%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.68.5.2939-2947.2000%26rft_id%253Dinfo%253Apmid%252F10768992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6IjY4LzUvMjkzOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTAzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.66" data-doi="10.1016/j.molcel.2013.09.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staals</span>,  <span class="cit-name-given-names">R. H. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Agari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Maki-Yonekura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Taylor</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Structure and Activity of the RNA-Targeting Type III-B CRISPR-Cas Complex of <em>Thermus thermophilus</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">145</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2013.09.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D52%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.09.013%26rft_id%253Dinfo%253Apmid%252F24119403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.09.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24119403&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326195800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.67" data-doi="10.1038/ncomms12853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staals</span>,  <span class="cit-name-given-names">R. H. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Jackson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Biswas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J. J.</span> <span class="cit-name-surname">Brouns</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Brown</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Interference-driven spacer acquisition is dominant over naive and primed adaptation in a native CRISPR-Cas system.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">12853</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms12853</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D12853%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms12853%26rft_id%253Dinfo%253Apmid%252F27694798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms12853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27694798&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.68" data-doi="10.1016/j.molcel.2014.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staals</span>,  <span class="cit-name-given-names">R. H. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Taylor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Kornfeld</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sharma</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RNA Targeting by the Type III-A CRISPR-Cas Csm Complex of <em>Thermus thermophilus</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">518</span>–<span class="cit-lpage">530</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2014.10.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D56%26rft.spage%253D518%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.10.005%26rft_id%253Dinfo%253Apmid%252F25457165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25457165&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.69" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.70" data-doi="10.1146/annurev-genet-120417-031321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanley</span>,  <span class="cit-name-given-names">S. Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Maxwell</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Phage-Encoded Anti-CRISPR Defenses.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-genet-120417-031321</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-120417-031321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-120417-031321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.1039.71" data-doi="10.1371/journal.pone.0035888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>,  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Mosterd</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W. J.</span> <span class="cit-name-surname">van Passel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J. J.</span> <span class="cit-name-surname">Brouns</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">CRISPR interference directs strand specific spacer acquisition.</span> PLoS One <span class="cit-vol">7</span>: <span class="cit-fpage">e35888</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0035888</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSwarts%26rft.auinit1%253DD.%2BC.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De35888%26rft.epage%253De35888%26rft.atitle%253DCRISPR%2Binterference%2Bdirects%2Bstrand%2Bspecific%2Bspacer%2Bacquisition.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0035888%26rft_id%253Dinfo%253Apmid%252F22558257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0035888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22558257&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.72" data-doi="10.1186/s13062-017-0193-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toms</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Barrangou</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">On the global CRISPR array behavior in class I systems.</span> <abbr class="cit-jnl-abbrev">Biol. Direct</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13062-017-0193-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BDirect%26rft.volume%253D12%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13062-017-0193-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13062-017-0193-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.73" data-doi="10.1128/MMBR.00011-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Houte</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Buckling</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Westra</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evolutionary ecology of prokaryotic immune mechanisms.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">763</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MMBR.00011-16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253Dvan%2BHoute%26rft.auinit1%253DS.%26rft.volume%253D80%26rft.issue%253D3%26rft.spage%253D745%26rft.epage%253D763%26rft.atitle%253DEvolutionary%2BEcology%2Bof%2BProkaryotic%2BImmune%2BMechanisms%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00011-16%26rft_id%253Dinfo%253Apmid%252F27412881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI4MC8zLzc0NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTAzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.74" data-doi="10.1093/nar/gkv1261"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vorontsova</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Datsenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Medvedeva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bondy-Denomy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. E.</span> <span class="cit-name-surname">Savitskaya</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Foreign DNA acquisition by the I-F CRISPR–Cas system requires all components of the interference machinery.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">10848</span>–<span class="cit-lpage">10860</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1261</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1261%26rft_id%253Dinfo%253Apmid%252F26586803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26586803&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.75" data-doi="10.1016/j.molcel.2012.03.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westra</span>,  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. B. G.</span> <span class="cit-name-surname">van Erp</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Künne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. H. J.</span> <span class="cit-name-surname">Staals</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">CRISPR immunity relies on the consecutive binding and degradation of negatively supercoiled invader DNA by Cascade and Cas3.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">605</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2012.03.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DWestra%26rft.auinit1%253DE.%2BR.%26rft.volume%253D46%26rft.issue%253D5%26rft.spage%253D595%26rft.epage%253D605%26rft.atitle%253DCRISPR%2Bimmunity%2Brelies%2Bon%2Bthe%2Bconsecutive%2Bbinding%2Band%2Bdegradation%2Bof%2Bnegatively%2Bsupercoiled%2Binvader%2BDNA%2Bby%2BCascade%2Band%2BCas3.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.03.018%26rft_id%253Dinfo%253Apmid%252F22521689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.03.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22521689&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305095400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.76" data-doi="10.1073/pnas.1102716108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiedenheft</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">van Duijn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Bultema</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. P.</span> <span class="cit-name-surname">Waghmare</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Zhou</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNA-guided complex from a bacterial immune system enhances target recognition through seed sequence interactions.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">10092</span>–<span class="cit-lpage">10097</span>. Erratum: <span class="cit-vol">108</span>: <span class="cit-fpage">15010</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1102716108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1102716108%26rft_id%253Dinfo%253Apmid%252F21536913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI1LzEwMDkyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.77" data-doi="10.1093/nar/gkv1259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xue</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Seetharam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Musharova</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. V.</span> <span class="cit-name-surname">Severinov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J. J.</span> <span class="cit-name-surname">Brouns</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">CRISPR interference and priming varies with individual spacer sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">10831</span>–<span class="cit-lpage">10847</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1259%26rft_id%253Dinfo%253Apmid%252F26586800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26586800&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1039.78" data-doi="10.1093/nar/gks216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yosef</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Goren</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Qimron</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Proteins and DNA elements essential for the CRISPR adaptation process in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">5569</span>–<span class="cit-lpage">5576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gks216</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks216%26rft_id%253Dinfo%253Apmid%252F22402487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22402487&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1039.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305829000040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/3/1039.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/1029" title="Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/1051" title="The Developmental Transcriptome of Aedes albopictus, a Major Worldwide Human Disease Vector" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/3"><span>Volume 10 Issue 3, March 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/3.cover-source.jpg?itok=8jjPWDA4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1039?rss=1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y_2-thos4s0o_2LokF8pEVGN6OjgLkV60Mt_VtYkZmM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/3/1039" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/3/1039" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/3/1039" class="active" data-icon-position="" data-hide-link-title="0">Type I-F CRISPR-Cas Distribution and Array Dynamics in Legionella pneumophila</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495857" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-_V9MqN2-m3yea_WGEDRFt4EP6_qKTMrM7hB9KJECDqQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/1039.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/3/1039" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-QGFP8u_Asgo1RGrIPfYrs9dUt1E2G7noaVRWyP8GW4w" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/3/1039" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/1039.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495857" id="node495857--2" data-pisa="ggg;10/3/1039" data-pisa-master="ggg;g3.119.400813" data-apath="/ggg/10/3/1039.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Type I-F CRISPR-Cas Distribution and Array Dynamics in <em>Legionella pneumophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1234-1123" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shayna R. Deecker</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0824-3704" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander W. Ensminger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1039-1050;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400813 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495857--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shayna R. Deecker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shayna%2BR.%2BDeecker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deecker%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShayna%2BR.%2BDeecker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1234-1123" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shayna R. Deecker</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander W. Ensminger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics, University of Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BW.%2BEnsminger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ensminger%20AW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BW.%2BEnsminger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0824-3704" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander W. Ensminger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alex.ensminger@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">alex.ensminger@utoronto.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/3/1039" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/3/1039&amp;title=Type%20I-F%20CRISPR-Cas%20Distribution%20and%20Array%20Dynamics%20in%20Legionella%20pneumophila" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/3/1039&amp;title=Type%20I-F%20CRISPR-Cas%20Distribution%20and%20Array%20Dynamics%20in%20Legionella%20pneumophila" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/3/1039&amp;title=Type%20I-F%20CRISPR-Cas%20Distribution%20and%20Array%20Dynamics%20in%20Legionella%20pneumophila" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/3/1039&amp;text=Type%20I-F%20CRISPR-Cas%20Distribution%20and%20Array%20Dynamics%20in%20Legionella%20pneumophila" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/3/1039&amp;title=Type%20I-F%20CRISPR-Cas%20Distribution%20and%20Array%20Dynamics%20in%20Legionella%20pneumophila" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/3/1039&amp;t=Type%20I-F%20CRISPR-Cas%20Distribution%20and%20Array%20Dynamics%20in%20Legionella%20pneumophila" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/3/1039&amp;title=Type%20I-F%20CRISPR-Cas%20Distribution%20and%20Array%20Dynamics%20in%20Legionella%20pneumophila" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/3/1039&amp;title=Type%20I-F%20CRISPR-Cas%20Distribution%20and%20Array%20Dynamics%20in%20Legionella%20pneumophila" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495857/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495857" id="node495857--3" data-pisa="ggg;10/3/1039" data-pisa-master="ggg;g3.119.400813" data-apath="/ggg/10/3/1039.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Type I-F CRISPR-Cas Distribution and Array Dynamics in <em>Legionella pneumophila</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1234-1123" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shayna R. Deecker</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0824-3704" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alexander W. Ensminger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1039-1050;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400813 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495857--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shayna R. Deecker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shayna%2BR.%2BDeecker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deecker%20SR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShayna%2BR.%2BDeecker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1234-1123" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shayna R. Deecker</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexander W. Ensminger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biochemistry and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular Genetics, University of Toronto, Ontario, Canada</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexander%2BW.%2BEnsminger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ensminger%20AW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlexander%2BW.%2BEnsminger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0824-3704" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alexander W. Ensminger</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alex.ensminger@utoronto.ca" class="" data-icon-position="" data-hide-link-title="0">alex.ensminger@utoronto.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1039&amp;rft.epage=1050&amp;rft.atitle=Type%20I-F%20CRISPR-Cas%20Distribution%20and%20Array%20Dynamics%20in%20Legionella%20pneumophila&amp;rft.volume=10&amp;rft.issue=3&amp;rft.date=2020-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Deecker%2C+Shayna+R.&amp;rft.au=Ensminger%2C+Alexander+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495857/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495857/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495857/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495857/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495857/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495857/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495857/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495857/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495857/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495857/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495857/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495857/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/3/1039" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/3/1039.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/3/1039.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/3/1039.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
