<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/3/907" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/3/907.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/3/907.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/3/907.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/3/907.atom" />
<meta name="HW.pisa" content="ggg;10/3/907" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome Sequence of the Euryhaline Javafish Medaka, Oryzias javanicus: A Small Aquarium Fish Model for Studies on Adaptation to Salinity" />
<meta name="DC.Identifier" content="10.1534/g3.119.400725" />
<meta name="DC.Date" content="2020-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Takehana et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The genus Oryzias consists of 35 medaka-fish species each exhibiting various ecological, morphological and physiological peculiarities and adaptations. Beyond of being a comprehensive phylogenetic group for studying intra-genus evolution of several traits like sex determination, behavior, morphology or adaptation through comparative genomic approaches, all medaka species share many advantages of experimental model organisms including small size and short generation time, transparent embryos and genome editing tools for reverse and forward genetic studies. The Java medaka, Oryzias javanicus , is one of the two species of medaka perfectly adapted for living in brackish/sea-waters. Being an important component of the mangrove ecosystem, O. javanicus is also used as a valuable marine test-fish for ecotoxicology studies. Here, we sequenced and assembled the whole genome of O. javanicus , and anticipate this resource will be catalytic for a wide range of comparative genomic, phylogenetic and functional studies. Complementary sequencing approaches including long-read technology and data integration with a genetic map allowed the final assembly of 908 Mbp of the O. javanicus genome. Further analyses estimate that the O. javanicus genome contains 33% of repeat sequences and has a heterozygosity of 0.96%. The achieved draft assembly contains 525 scaffolds with a total length of 809.7 Mbp, a N50 of 6,3 Mbp and a L50 of 37 scaffolds. We identified 21454 predicted transcripts for a total transcriptome size of 57, 146, 583 bps. We provide here a high-quality chromosome scale draft genome assembly of the euryhaline Javafish medaka (321 scaffolds anchored on 24 chromosomes (representing 97.7% of the total bases)), and give emphasis on the evolutionary adaptation to salinity." />
<meta name="DC.Contributor" content="Yusuke Takehana" />
<meta name="DC.Contributor" content="Margot Zahm" />
<meta name="DC.Contributor" content="Cédric Cabau" />
<meta name="DC.Contributor" content="Christophe Klopp" />
<meta name="DC.Contributor" content="Céline Roques" />
<meta name="DC.Contributor" content="Olivier Bouchez" />
<meta name="DC.Contributor" content="Cécile Donnadieu" />
<meta name="DC.Contributor" content="Celia Barrachina" />
<meta name="DC.Contributor" content="Laurent Journot" />
<meta name="DC.Contributor" content="Mari Kawaguchi" />
<meta name="DC.Contributor" content="Shigeki Yasumasu" />
<meta name="DC.Contributor" content="Satoshi Ansai" />
<meta name="DC.Contributor" content="Kiyoshi Naruse" />
<meta name="DC.Contributor" content="Koji Inoue" />
<meta name="DC.Contributor" content="Chuya Shinzato" />
<meta name="DC.Contributor" content="Manfred Schartl" />
<meta name="DC.Contributor" content="Yann Guiguen" />
<meta name="DC.Contributor" content="Amaury Herpin" />
<meta name="article:published_time" content="2020-03-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Genome Sequence of the Euryhaline Javafish Medaka, Oryzias javanicus: A Small Aquarium Fish Model for Studies on Adaptation to Salinity" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The genus &lt;i&gt;Oryzias&lt;/i&gt; consists of 35 medaka-fish species each exhibiting various ecological, morphological and physiological peculiarities and adaptations. Beyond of being a comprehensive phylogenetic group for studying intra-genus evolution of several traits like sex determination, behavior, morphology or adaptation through comparative genomic approaches, all medaka species share many advantages of experimental model organisms including small size and short generation time, transparent embryos and genome editing tools for reverse and forward genetic studies. The Java medaka, &lt;i&gt;Oryzias javanicus&lt;/i&gt;, is one of the two species of medaka perfectly adapted for living in brackish/sea-waters. Being an important component of the mangrove ecosystem, &lt;i&gt;O. javanicus&lt;/i&gt; is also used as a valuable marine test-fish for ecotoxicology studies. Here, we sequenced and assembled the whole genome of &lt;i&gt;O. javanicus&lt;/i&gt;, and anticipate this resource will be catalytic for a wide range of comparative genomic, phylogenetic and functional studies. Complementary sequencing approaches including long-read technology and data integration with a genetic map allowed the final assembly of 908 Mbp of the &lt;i&gt;O. javanicus&lt;/i&gt; genome. Further analyses estimate that the &lt;i&gt;O. javanicus&lt;/i&gt; genome contains 33% of repeat sequences and has a heterozygosity of 0.96%. The achieved draft assembly contains 525 scaffolds with a total length of 809.7 Mbp, a N50 of 6,3 Mbp and a L50 of 37 scaffolds. We identified 21454 predicted transcripts for a total transcriptome size of 57, 146, 583 bps. We provide here a high-quality chromosome scale draft genome assembly of the euryhaline Javafish medaka (321 scaffolds anchored on 24 chromosomes (representing 97.7% of the total bases)), and give emphasis on the evolutionary adaptation to salinity.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/03/01" />
<meta name="citation_mjid" content="ggg;10/3/907" />
<meta name="citation_id" content="10/3/907" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/3/907" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/3/907.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/3/907.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/3/907.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400725" />
<meta name="citation_pmid" content="31988161" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="907" />
<meta name="citation_lastpage" content="915" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yusuke Takehana" />
<meta name="citation_author_institution" content="Department of Animal Bio-Science, Faculty of Bio-Science, Nagahama Institute of Bioscience and Technology, 1266 Tamura, Nagahama 526-0829, Japan," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9965-8499" />
<meta name="citation_author" content="Margot Zahm" />
<meta name="citation_author_institution" content="Plate-forme bio-informatique Genotoul, Mathématiques et Informatique Appliquées de Toulouse, INRAE, Castanet Tolosan, France," />
<meta name="citation_author" content="Cédric Cabau" />
<meta name="citation_author_institution" content="SIGENAE, GenPhySE, Université de Toulouse, INRAE, ENVT, Castanet Tolosan, France," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8560-4003" />
<meta name="citation_author" content="Christophe Klopp" />
<meta name="citation_author_institution" content="Plate-forme bio-informatique Genotoul, Mathématiques et Informatique Appliquées de Toulouse, INRAE, Castanet Tolosan, France," />
<meta name="citation_author_institution" content="SIGENAE, GenPhySE, Université de Toulouse, INRAE, ENVT, Castanet Tolosan, France," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7126-5477" />
<meta name="citation_author" content="Céline Roques" />
<meta name="citation_author_institution" content="INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France," />
<meta name="citation_author" content="Olivier Bouchez" />
<meta name="citation_author_institution" content="INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France," />
<meta name="citation_author" content="Cécile Donnadieu" />
<meta name="citation_author_institution" content="INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5041-6843" />
<meta name="citation_author" content="Celia Barrachina" />
<meta name="citation_author_institution" content="MGX, Biocampus Montpellier, CNRS, INSERM, University of Montpellier, Montpellier, France," />
<meta name="citation_author" content="Laurent Journot" />
<meta name="citation_author_institution" content="MGX, Biocampus Montpellier, CNRS, INSERM, University of Montpellier, Montpellier, France," />
<meta name="citation_author" content="Mari Kawaguchi" />
<meta name="citation_author_institution" content="Department of Materials and Life Sciences, Faculty of Science and Technology, Sophia University, 7-1 Kioi-cho Chiyoda-ku, Tokyo 102-8554, Japan," />
<meta name="citation_author" content="Shigeki Yasumasu" />
<meta name="citation_author_institution" content="Department of Materials and Life Sciences, Faculty of Science and Technology, Sophia University, 7-1 Kioi-cho Chiyoda-ku, Tokyo 102-8554, Japan," />
<meta name="citation_author" content="Satoshi Ansai" />
<meta name="citation_author_institution" content="Laboratory of Bioresources, National Institute for Basic Biology, 38 Nishigonaka, Myodaiji-cho, Okazaki 444-8585, Japan," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2683-0160" />
<meta name="citation_author" content="Kiyoshi Naruse" />
<meta name="citation_author_institution" content="Laboratory of Bioresources, National Institute for Basic Biology, 38 Nishigonaka, Myodaiji-cho, Okazaki 444-8585, Japan," />
<meta name="citation_author" content="Koji Inoue" />
<meta name="citation_author_institution" content="Atmosphere and Ocean Research Institute, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8564, Japan," />
<meta name="citation_author" content="Chuya Shinzato" />
<meta name="citation_author_institution" content="Atmosphere and Ocean Research Institute, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8564, Japan," />
<meta name="citation_author" content="Manfred Schartl" />
<meta name="citation_author_institution" content="University of Wuerzburg, Developmental Biochemistry, Biocenter, 97074 Wuerzburg, Germany," />
<meta name="citation_author_institution" content="Comprehensive Cancer Center Mainfranken, University Hospital, 97080 Wuerzburg, Germany," />
<meta name="citation_author_institution" content="Hagler Institute for Advanced Study and Department of Biology, Texas A&amp;M University, College Station, Texas 77843, USA," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9882-5948" />
<meta name="citation_author" content="Yann Guiguen" />
<meta name="citation_author_institution" content="INRAE, UR 1037 Fish Physiology and Genomics, F-35000 Rennes, France, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5464-6219" />
<meta name="citation_author" content="Amaury Herpin" />
<meta name="citation_author_institution" content="INRAE, UR 1037 Fish Physiology and Genomics, F-35000 Rennes, France, and" />
<meta name="citation_author_institution" content="State Key Laboratory of Developmental Biology of Freshwater Fish, College of Life Sciences, Hunan Normal University, Changsha, 410081, Hunan, P. R. of China" />
<meta name="citation_author_email" content="amaury.herpin@inra.fr" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0630-4027" />
<meta name="citation_reference" content="citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. Benson;citation_title=Tandem repeats finder: a program to analyze DNA sequences.;citation_volume=27;citation_year=1999;citation_pmid=9862982;citation_doi=10.1093/nar/27.2.573" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=M. Bernt;citation_author=A. Donath;citation_author=F. Jühling;citation_author=F. Externbrink;citation_author=C. Florentz;citation_title=MITOS: improved de novo metazoan mitochondrial genome annotation.;citation_pages=313-319;citation_volume=69;citation_year=2013;citation_pmid=22982435;citation_doi=10.1016/j.ympev.2012.08.023" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=M. L. Borowiec;citation_title=AMAS: a fast tool for alignment manipulation and computing of summary statistics.;citation_pages=e1660;citation_volume=4;citation_year=2016;citation_doi=10.7717/peerj.1660" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=B. Buchfink;citation_author=C. Xie;citation_author=D. H. Huson;citation_title=Fast and sensitive protein alignment using DIAMOND.;citation_pages=59-60;citation_volume=12;citation_year=2015;citation_pmid=25402007;citation_doi=10.1038/nmeth.3176" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=F. Cabanettes;citation_author=C. Klopp;citation_title=D-GENIES: dot plot large genomes in an interactive, efficient and simple way.;citation_pages=e4958;citation_volume=6;citation_year=2018;citation_doi=10.7717/peerj.4958" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=C. Cabau;citation_author=F. Escudié;citation_author=A. Djari;citation_author=Y. Guiguen;citation_author=J. Bobe;citation_title=Compacting and correcting Trinity and Oases RNA-Seq de novo assemblies.;citation_pages=e2988;citation_volume=5;citation_year=2017;citation_doi=10.7717/peerj.2988" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Capella-Gutiérrez;citation_author=J. M. Silla-Martínez;citation_author=T. Gabaldón;citation_title=trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.;citation_volume=25;citation_year=2009;citation_pmid=19505945;citation_doi=10.1093/bioinformatics/btp348" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. A. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: ultrafast universal RNA-seq aligner.;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=K. Eilbeck;citation_author=B. Moore;citation_author=C. Holt;citation_author=M. Yandell;citation_title=Quantitative measures for the management and comparison of annotated genomes.;citation_pages=67-67;citation_volume=10;citation_year=2009;citation_pmid=19236712;citation_doi=10.1186/1471-2105-10-67" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. El-Gebali;citation_author=J. Mistry;citation_author=A. Bateman;citation_author=S. R. Eddy;citation_author=A. Luciani;citation_title=The Pfam protein families database in 2019.;citation_volume=47;citation_year=2019;citation_pmid=30357350;citation_doi=10.1093/nar/gky995" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. M. Emms;citation_author=S. Kelly;citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.;citation_pages=157;citation_volume=16;citation_year=2015;citation_pmid=26243257;citation_doi=10.1186/s13059-015-0721-2" />
<meta name="citation_reference" content="citation_journal_title=Molecular and cellular endocrinology;citation_journal_abbrev=Mol Cell Endocrinol;citation_author=A. Herpin;citation_author=M. Schartl;citation_title=Molecular mechanisms of sex determination and evolution of the Y-chromosome: insights from the medakafish (Oryzias latipes).;citation_pages=51-58;citation_volume=306;citation_year=2009;citation_issue=1-2;citation_pmid=19481684;citation_doi=10.1016/j.mce.2009.02.004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=D. T. Hoang;citation_author=O. Chernomor;citation_author=A. von Haeseler;citation_author=B. Q. Minh;citation_author=L. S. Vinh;citation_title=UFBoot2: Improving the Ultrafast Bootstrap Approximation.;citation_pages=518-522;citation_volume=35;citation_year=2018;citation_pmid=29077904;citation_doi=10.1093/molbev/msx281" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. J. Hoff;citation_author=S. Lange;citation_author=A. Lomsadze;citation_author=M. Borodovsky;citation_author=M. Stanke;citation_title=BRAKER1: Unsupervised RNA-Seq-Based Genome Annotation with GeneMark-ET and AUGUSTUS.;citation_volume=32;citation_year=2016;citation_pmid=26559507;citation_doi=10.1093/bioinformatics/btv661" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Holt;citation_author=M. Yandell;citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.;citation_pages=491-491;citation_volume=12;citation_year=2011;citation_pmid=22192575;citation_doi=10.1186/1471-2105-12-491" />
<meta name="citation_reference" content="citation_journal_title=Bull. Environ. Contam. Toxicol.;citation_author=Y. Horie;citation_author=N. Kanazawa;citation_author=T. Yamagishi;citation_author=K. Yonekura;citation_author=N. Tatarazako;citation_title=Ecotoxicological Test Assay Using OECD TG 212 in Marine Java Medaka (Oryzias javanicus) and Freshwater Japanese Medaka (Oryzias latipes).;citation_pages=344-348;citation_volume=101;citation_year=2018;citation_doi=10.1007/s00128-018-2398-1" />
<meta name="citation_reference" content="citation_journal_title=Comparative Biochemistry and Physiology. Part B, Biochemistry &amp; Molecular Biology;citation_journal_abbrev=Comparative Biochemistry and Physiology. Part B, Biochemistry &amp; Molecular Biology;citation_author=K. Inoue;citation_author=Y. Takei;citation_title=Asian medaka fishes offer new models for studying mechanisms of seawater adaptation.;citation_pages=635-645;citation_volume=136;citation_year=2003;citation_issue=4;citation_pmid=14662290;citation_doi=10.1016/S1096-4959(03)00204-5" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=K. Inoue;citation_author=Y. Takei;citation_title=Diverse adaptability in oryzias species to high environmental salinity.;citation_pages=727-734;citation_volume=19;citation_year=2002;citation_issue=7;citation_pmid=12149572;citation_doi=10.2108/zsj.19.727" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=M. Kawaguchi;citation_author=S. Yasumasu;citation_author=J. Hiroi;citation_author=K. Naruse;citation_author=T. Suzuki;citation_title=Analysis of the exon-intron structures of fish, amphibian, bird and mammalian hatching enzyme genes, with special reference to the intron loss evolution of hatching enzyme genes in Teleostei.;citation_pages=77-88;citation_volume=392;citation_year=2007;citation_issue=1-2;citation_pmid=17222522;citation_doi=10.1016/j.gene.2006.11.012" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Biol.;citation_author=M. Kawaguchi;citation_author=S. Yasumasu;citation_author=A. Shimizu;citation_author=N. Kudo;citation_author=K. Sano;citation_title=Adaptive evolution of fish hatching enzyme: one amino acid substitution results in differential salt dependency of the enzyme.;citation_volume=216;citation_year=2013;citation_pmid=23348940;citation_doi=10.1242/jeb.069716" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT–the BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Kirchmaier;citation_author=K. Naruse;citation_author=J. Wittbrodt;citation_author=F. Loosli;citation_title=The Genomic and Genetic Toolbox of the Teleost Medaka (Oryzias latipes);citation_pages=905-918;citation_volume=199;citation_year=2015;citation_issue=4;citation_pmid=25855651;citation_doi=10.1534/genetics.114.173849" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Koren;citation_author=B. P. Walenz;citation_author=K. Berlin;citation_author=J. R. Miller;citation_author=N. H. Bergman;citation_title=Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.;citation_volume=27;citation_year=2017;citation_pmid=28298431;citation_doi=10.1101/gr.215087.116" />
<meta name="citation_reference" content="citation_journal_title=Environmental toxicology;citation_journal_abbrev=Environ Toxicol;citation_author=J. Koyama;citation_author=M. Kawamata;citation_author=S. Imai;citation_author=M. Fukunaga;citation_author=S. Uno;citation_title=Java medaka: a proposed new marine test fish for ecotoxicology.;citation_pages=487-491;citation_volume=23;citation_year=2008;citation_issue=4;citation_pmid=18246546;citation_doi=10.1002/tox.20367" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.;citation_volume=27;citation_year=2011;citation_pmid=21903627;citation_doi=10.1093/bioinformatics/btr509" />
<meta name="citation_reference" content="Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv:1303.3997v2 [q-bio.GN]" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Marçais;citation_author=C. Kingsford;citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.;citation_volume=27;citation_year=2011;citation_pmid=21217122;citation_doi=10.1093/bioinformatics/btr011" />
<meta name="citation_reference" content="Martin, M., 2011 Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal 17: 10–12. 10.14806/ej.17.1.200" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=D. F. Mokodongan;citation_author=K. Yamahira;citation_title=Origin and intra-island diversification of Sulawesi endemic Adrianichthyidae.;citation_pages=150-160;citation_volume=93;citation_year=2015;citation_pmid=26256644;citation_doi=10.1016/j.ympev.2015.07.024" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=A. Morgulis;citation_author=E. M. Gertz;citation_author=A. A. Schäffer;citation_author=R. Agarwala;citation_title=A fast and symmetric DUST implementation to mask low-complexity DNA sequences.;citation_pages=1028-1040;citation_volume=13;citation_year=2006;citation_issue=5;citation_pmid=16796549;citation_doi=10.1089/cmb.2006.13.1028" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Myosho;citation_author=H. Otake;citation_author=H. Masuyama;citation_author=M. Matsuda;citation_author=Y. Kuroki;citation_title=Tracing the emergence of a novel sex-determining gene in medaka, Oryzias luzonensis.;citation_volume=191;citation_year=2012;citation_pmid=22367037;citation_doi=10.1534/genetics.111.137497" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=T. Myosho;citation_author=H. Takahashi;citation_author=K. Yoshida;citation_author=T. Sato;citation_author=S. Hamaguchi;citation_title=Hyperosmotic tolerance of adult fish and early embryos are determined by discrete, single loci in the genus Oryzias.;citation_pages=6897;citation_volume=8;citation_year=2018;citation_doi=10.1038/s41598-018-24621-7" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=L.-T. Nguyen;citation_author=H. A. Schmidt;citation_author=A. von Haeseler;citation_author=B. Q. Minh;citation_title=IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies.;citation_volume=32;citation_year=2015;citation_pmid=25371430;citation_doi=10.1093/molbev/msu300" />
<meta name="citation_reference" content="citation_journal_title=Zool. J. Linn. Soc.;citation_author=L. R. Parenti;citation_title=A phylogenetic analysis and taxonomic revision of ricefishes, Oryzias and relatives (Beloniformes, Adrianichthyidae).;citation_pages=494-610;citation_volume=154;citation_year=2008;citation_doi=10.1111/j.1096-3642.2008.00417.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Rastas;citation_title=Lep-MAP3: robust linkage mapping even for low-coverage whole genome sequencing data.;citation_pages=3726-3732;citation_volume=33;citation_year=2017;citation_doi=10.1093/bioinformatics/btx494" />
<meta name="citation_reference" content="Ruan, J., 2019 Ultra-fast de novo assembler using long noisy reads: Ruanjue/smartdenovo. Available at: https://github.com/ruanjue/smartdenovo. Accessed: September 13, 2019." />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. H. Schulz;citation_author=D. R. Zerbino;citation_author=M. Vingron;citation_author=E. Birney;citation_title=Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels.;citation_volume=28;citation_year=2012;citation_pmid=22368243;citation_doi=10.1093/bioinformatics/bts094" />
<meta name="citation_reference" content="citation_journal_title=Biol. Lett.;citation_author=D. H. E. Setiamarga;citation_author=M. Miya;citation_author=Y. Yamanoue;citation_author=Y. Azuma;citation_author=J. G. Inoue;citation_title=Divergence time of the two regional medaka populations in Japan as a new time scale for comparative genomics of vertebrates.;citation_volume=5;citation_year=2009;citation_pmid=19586967;citation_doi=10.1098/rsbl.2009.0419;citation_georef=2013074188" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=G. S. C. Slater;citation_author=E. Birney;citation_title=Automated generation of heuristics for biological sequence comparison.;citation_pages=31-31;citation_volume=6;citation_year=2005;citation_pmid=15713233;citation_doi=10.1186/1471-2105-6-31" />
<meta name="citation_reference" content="Smit, A. F. A., and R. Hubley, 2010 RepeatModeler Open-1.0." />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Suyama;citation_author=D. Torrents;citation_author=P. Bork;citation_title=PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments.;citation_pages=W609-W612;citation_volume=34;citation_year=2006;citation_issue=Web Server issue;citation_pmid=16845082;citation_doi=10.1093/nar/gkl315" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Takehana;citation_author=D. Demiyah;citation_author=K. Naruse;citation_author=S. Hamaguchi;citation_author=M. Sakaizumi;citation_title=Evolution of different Y chromosomes in two medaka species, Oryzias dancena and O. latipes.;citation_volume=175;citation_year=2007;citation_pmid=17194774;citation_doi=10.1534/genetics.106.068247" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=Y. Takehana;citation_author=S. Hamaguchi;citation_author=M. Sakaizumi;citation_title=Different origins of ZZ/ZW sex chromosomes in closely related medaka fishes, Oryzias javanicus and O. hubbsi.;citation_pages=801-811;citation_volume=16;citation_year=2008;citation_issue=5;citation_pmid=18607761;citation_doi=10.1007/s10577-008-1227-5" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=Y. Takehana;citation_author=M. Matsuda;citation_author=T. Myosho;citation_author=M. L. Suster;citation_author=K. Kawakami;citation_title=Co-option of Sox3 as the male-determining factor on the Y chromosome in the fish Oryzias dancena.;citation_pages=4157;citation_volume=5;citation_year=2014;citation_pmid=24948391;citation_doi=10.1038/ncomms5157" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_journal_abbrev=Chromosoma;citation_author=Y. Takehana;citation_author=K. Naruse;citation_author=S. Hamaguchi;citation_author=M. Sakaizumi;citation_title=Evolution of ZZ/ZW and XX/XY sex-determination systems in the closely related medaka species, Oryzias hubbsi and O. dancena.;citation_pages=463-470;citation_volume=116;citation_year=2007;citation_issue=5;citation_pmid=17882464;citation_doi=10.1007/s00412-007-0110-z" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=Y. Takehana;citation_author=K. Naruse;citation_author=M. Sakaizumi;citation_title=Molecular phylogeny of the medaka fishes genus Oryzias (Beloniformes: Adrianichthyidae) based on nuclear and mitochondrial DNA sequences.;citation_pages=417-428;citation_volume=36;citation_year=2005;citation_issue=2;citation_pmid=15955519;citation_doi=10.1016/j.ympev.2005.01.016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Tanaka;citation_author=Y. Takehana;citation_author=K. Naruse;citation_author=S. Hamaguchi;citation_author=M. Sakaizumi;citation_title=Evidence for different origins of sex chromosomes in closely related Oryzias fishes: substitution of the master sex-determining gene.;citation_volume=177;citation_year=2007;citation_pmid=17947439;citation_doi=10.1534/genetics.107.075598" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=H. Tang;citation_author=X. Zhang;citation_author=C. Miao;citation_author=J. Zhang;citation_author=R. Ming;citation_title=ALLMAPS: robust scaffold ordering based on multiple maps.;citation_pages=3;citation_volume=16;citation_year=2015;citation_pmid=25583564;citation_doi=10.1186/s13059-014-0573-1" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. W. Vurture;citation_author=F. J. Sedlazeck;citation_author=M. Nattestad;citation_author=C. J. Underwood;citation_author=H. Fang;citation_title=GenomeScope: fast reference-free genome profiling from short reads.;citation_pages=2202-2204;citation_volume=33;citation_year=2017;citation_pmid=28369201;citation_doi=10.1093/bioinformatics/btx153" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Wittbrodt;citation_author=A. Shima;citation_author=M. Schartl;citation_title=Medaka--a model organism from the far East.;citation_pages=53-64;citation_volume=3;citation_year=2002;citation_issue=1;citation_pmid=11823791;citation_doi=10.1038/nrg704" />
<meta name="citation_reference" content="citation_journal_title=Comparative Biochemistry and Physiology. Part B, Biochemistry &amp; Molecular Biology;citation_journal_abbrev=Comparative Biochemistry and Physiology. Part B, Biochemistry &amp; Molecular Biology;citation_author=K. Yamagami;citation_title=Some enzymological properties of a hatching enzyme (chorionase) isolated from the fresh-water teleost, Oryzias latipes.;citation_pages=603-616;citation_volume=46;citation_year=1973;citation_issue=3;citation_pmid=4201938;citation_doi=10.1016/0305-0491(73)90100-4" />
<meta name="citation_reference" content="citation_journal_title=J. Biochem.;citation_author=S. Yasumasu;citation_author=M. Kawaguchi;citation_author=S. Ouchi;citation_author=K. Sano;citation_author=K. Murata;citation_title=Mechanism of egg envelope digestion by hatching enzymes, HCE and LCE in medaka, Oryzias latipes.;citation_volume=148;citation_year=2010;citation_pmid=20679367;citation_doi=10.1093/jb/mvq086" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Yeo;citation_author=L. Coombe;citation_author=R. L. Warren;citation_author=J. Chu;citation_author=I. Birol;citation_title=ARCS: scaffolding genome drafts with linked reads.;citation_pages=725-731;citation_volume=34;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Environ. Biol. Fishes;citation_author=S. Yusof;citation_author=A. Ismail;citation_author=T. Koito;citation_author=M. Kinoshita;citation_author=K. Inoue;citation_title=Occurrence of two closely related ricefishes, Javanese medaka (Oryzias javanicus) and Indian medaka (O. dancena) at sites with different salinity in Peninsular Malaysia.;citation_pages=43-49;citation_volume=93;citation_year=2012;citation_doi=10.1007/s10641-011-9888-x" />
<meta name="citation_reference" content="Zulkifli, S. Z., F. Mohamat-Yusuff, A. Ismail, and N. Miyazaki, 2012 Food preference of the giant mudskipper Periophthalmodon schlosseri (Teleostei : Gobiidae). Knowl. Managt. Aquatic Ecosyst. 07." />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome Sequence of the Euryhaline Javafish Medaka, Oryzias javanicus: A Small Aquarium Fish Model for Studies on Adaptation to Salinity" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="twitter:description" content="The genus Oryzias consists of 35 medaka-fish species each exhibiting various ecological, morphological and physiological peculiarities and adaptations. Beyond of being a comprehensive phylogenetic group for studying intra-genus evolution of several traits like sex determination, behavior, morphology or adaptation through comparative genomic approaches, all medaka species share many advantages of experimental model organisms including small size and short generation time, transparent embryos and genome editing tools for reverse and forward genetic studies. The Java medaka, Oryzias javanicus , is one of the two species of medaka perfectly adapted for living in brackish/sea-waters. Being an important component of the mangrove ecosystem, O. javanicus is also used as a valuable marine test-fish for ecotoxicology studies. Here, we sequenced and assembled the whole genome of O. javanicus , and anticipate this resource will be catalytic for a wide range of comparative genomic, phylogenetic and functional studies. Complementary sequencing approaches including long-read technology and data integration with a genetic map allowed the final assembly of 908 Mbp of the O. javanicus genome. Further analyses estimate that the O. javanicus genome contains 33% of repeat sequences and has a heterozygosity of 0.96%. The achieved draft assembly contains 525 scaffolds with a total length of 809.7 Mbp, a N50 of 6,3 Mbp and a L50 of 37 scaffolds. We identified 21454 predicted transcripts for a total transcriptome size of 57, 146, 583 bps. We provide here a high-quality chromosome scale draft genome assembly of the euryhaline Javafish medaka (321 scaffolds anchored on 24 chromosomes (representing 97.7% of the total bases)), and give emphasis on the evolutionary adaptation to salinity." />
<meta name="og-title" property="og:title" content="Genome Sequence of the Euryhaline Javafish Medaka, Oryzias javanicus: A Small Aquarium Fish Model for Studies on Adaptation to Salinity" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/3/907" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The genus Oryzias consists of 35 medaka-fish species each exhibiting various ecological, morphological and physiological peculiarities and adaptations. Beyond of being a comprehensive phylogenetic group for studying intra-genus evolution of several traits like sex determination, behavior, morphology or adaptation through comparative genomic approaches, all medaka species share many advantages of experimental model organisms including small size and short generation time, transparent embryos and genome editing tools for reverse and forward genetic studies. The Java medaka, Oryzias javanicus , is one of the two species of medaka perfectly adapted for living in brackish/sea-waters. Being an important component of the mangrove ecosystem, O. javanicus is also used as a valuable marine test-fish for ecotoxicology studies. Here, we sequenced and assembled the whole genome of O. javanicus , and anticipate this resource will be catalytic for a wide range of comparative genomic, phylogenetic and functional studies. Complementary sequencing approaches including long-read technology and data integration with a genetic map allowed the final assembly of 908 Mbp of the O. javanicus genome. Further analyses estimate that the O. javanicus genome contains 33% of repeat sequences and has a heterozygosity of 0.96%. The achieved draft assembly contains 525 scaffolds with a total length of 809.7 Mbp, a N50 of 6,3 Mbp and a L50 of 37 scaffolds. We identified 21454 predicted transcripts for a total transcriptome size of 57, 146, 583 bps. We provide here a high-quality chromosome scale draft genome assembly of the euryhaline Javafish medaka (321 scaffolds anchored on 24 chromosomes (representing 97.7% of the total bases)), and give emphasis on the evolutionary adaptation to salinity." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
    <title>Genome Sequence of the Euryhaline Javafish Medaka, Oryzias javanicus: A Small Aquarium Fish Model for Studies on Adaptation to Salinity | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"WAwUPCctQ9TrFvUbx8QKwkuas0-CweIYJkZRYf7Ljuw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495848","apath":"\/ggg\/10\/3\/907.atom","pisa":"ggg;10\/3\/907","ac":{"\/ggg\/10\/3\/907.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/3\/907.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/3\/907"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/3\/907"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/3\/907?rss=1":true,"\/content\/10\/3\/907":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FT5ibScmpQNJUMc6ZMs3K\\\\\\\/wQ0AmudccdjboBa8TX2AvJwYv9XdhJDiUVqszhzoWvaSTzu4wqbHwScVFRSaEgj16DotpR8zQFuxM0cRKkbXe08B+vnZnkJsz0CTZkwGjcJDTGK579HOf2FwLrCIN1CS5i\\\\\\\/okl\\\\\\\/7azxufoNzZ8zpwl6F9\\\\\\\/Kt6ilcDjN0lZaoJAUmKGFPbZi9x1fjMSz14r6wqkZa+az4AwNidzbvh5fZriMRLqy5ZHI5jBQ7MhmmtfTq\\\\\\\/75Zusp8cz\\\\\\\/\\\\\\\/aOBDeeSxWBUQG1mUVUCc2KwA2HyedMAQJePqlABgbbV9sCExqXeiZsalxJyLSm+4ErVOhuCld3M8uTsr0BvlMi8rYJtnbz+C41r4Yzu9Maut25Qv5r3Sg85rZBcBjXyrrslppskXZEdwtRPYI4oYgdGHl1jwguJbFgXhH8AB8qoWyHnTk6yXZa+gVI0iH2I5n7AVoX4vkVA6er\\\\\\\/lxCfj1SNNC4CvE3ZvtO8Axs1\\\\\\\/A85j4PgVBbinM6ZYMvo5FzTivz1zx9NJbLIjN8L\\\\\\\/jI+ZZmP4QwR5EFCujQl6fJ6sDpNsE\\\\\\\/Jnc6SMmrOAdJTGllodQOyOumqc7vaF4rdV0YUtMl6um71DOZImsHHPU7bSsJFusio8fJ7t+D6Cbg5m+3W4I+2cVSCyLDZWPEe\\\\\\\/0tnD91L1LugxIeECB9o0UZ1\\\\\\\/cNynGPxa4T6VX0dCmfJHk1wxrru5sCg8drgw9+ZuZ7wzUNJciYhIwXRYvf0WehcpG4myrjLZK6IgdCQePdTKLgdewWbRVU2020YxUvg7iY12gmFVVRQnM3UmC1wahK5M9hQ1IF0\\\\\\\/\\\\\\\/N7wvwdz4fZkT50tED1biN7UGu5Ko7872m2ZnjH+iW11Ky3zDYaTHc1+q6rScuU2HDC+1DRiqEZsAM46ogQq34y8J2gF0VLSDT18GqAbj5ZkG5PHN3eIQ5Okp7CF4C5CCe1nAPWZnUfcV9am7weaIKPLDhpBDYQkUkJMOyGEvDic0YbLCbCJ+9xjpCTHn+LyQODh1HLP+r9Vez4Yohy2ysi0N4ISws9hOo2xl4jaxE25GE+OzUWuWWyWE+w3LtT31W8fxCQHT+RLSHrV2TkjMwED0Mw5AgVALbT12mQlwbFsPUMr4BjKmicg9vxRmRzSXk+Rkg8WBwOgp1swkU8O3ZUHxYCJEZaN62UUbsEEzjdX\\\\\\\/EVsi3ZU28mmkJeRtWySNtmRqSLwIjcBj6uNGHjcGYA\\\\\\\/NJ6ISFL\\\\\\\/eQLcsR87HRQUaFZ0dzY7YY0GtR7XPUSqw0\\\u0022,\\\u0022iv\\\u0022:\\\u0022TR4Oz+PWHQ0+D00FHXAPEw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222fcc617e8e67b1ab1a47723260efcb892233d1188a81510d05b78e6f4f364b62\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022f3S3yYhj23Hu1I3V4P45P+lOeNEbJKG7rw4zhOn16+1mEFaJd96okUlcYu2aSNNaqGdzwAG0L0VfzQexD5dMHWBoJIwDhtZYXtEUqFTJ9em0tf05UJo3BAjpSwHNN3x5CylgJZ5SFoxhdX+ld64F7+7OJdYuIBEfom8PG8Pbh7lqq8C7naSqX6ZVQ0yYCG7j+fx5pysOdQgaYHCwOQhZfoSvLRqyyh3BwzTiY3rO2bAJAtBJ0kzMIfj1VP5lM+9IgXwkfSmgbF+WoNImnw5Hczv2Gd9bQCT+jrubCgMe7AQkqYRzIDu7lrAl0KGw4R+cpYoGGQlfp9vOUdoVnJr5CjlVw1oO30vwsSJ8+Hiddp7tUWxmRd3vogQ3c5\\\\\\\/yW0\\\\\\\/LvN\\\\\\\/XSaHzgDhBiqIeOdWKeuQ6iiuti0e6IvZbaKZwXI31JxRvveGpw0GBb0bFDLCjruE0tpZPJngr\\\\\\\/26hgbik51x7OgjSzQt6gPESgjSKj0REu+UxBshOaoc4WrlYr187NX5utMKog1FoFZNzaxalnq6yro5qz5gbRVHuWI9AvEuUvX24aoOZk8HHRi+n8ht2TOkicOW9Ld6QD1ocW0\\\\\\\/M0c5OVDQFUADsACqt13\\\\\\\/lu7TticJrleHcOOst3nXyhgVZW7MhsiQfdXJCj2lP8WMiKvbbg\\\\\\\/5TUs2jZHR1wY09ZNpU+BXZJG91IVLU3NQaIHI+RNoMycSbxSaMt0V9mSmZ7JORGH+5w74HtoTu0ejuGfz9w9Zp08eV6Xu4TfafJuUrftyWf3PvNILNFaqqe\\\\\\\/XQfID48r82aULVPaeVcTCD7y3+N8FlitLERHeqD8QHQGxuC94CvPErOit8aYP\\\\\\\/d50Gs3ZRGw9BHnpgCnoZK6JVjUBVZdHflFVVdVVHsgUUMGGIR6hHgh97jQXlwVi6mBf5+\\\\\\\/W523XWDyGe4iRj3vZhzp\\\\\\\/whkh2SPyuH2Fe2pRLkqW3v3vBSITg55qodTNIzMEsR1+mdq8vGIMKeYRB6dkQ5a0\\\\\\\/pz\\\\\\\/MqbXUL+TonBwZ2yNfEf9tOxu4zSXg6NLT25f7tImb1m6b2+5K7aS0fntFm1dP8EpHURPxSN1n6GIl7NRkOAhUEP4oecYyvsckVT5UdRDPzloWsJfS1m9pcgEgynao1pgS0\\\\\\\/kskwxnoJNhCawPSiAEtryZNKtBULGwflzMOVvnF7DNk+OxRPX8yQEUdQDLDH2tD1n36gyXSZotaQSYsJowOPSCjJ+H0HgYWaIRpkFJ6s0n9qqJgAY8renDqgcZQ7QiZWYrJSJHeBIsvoowsvlLV00ubUr5oV\\\\\\\/+3K7WJDWAIAzcJiQIxsCNmpAjmo+XzCYJb+NgOZ\\\\\\\/O34CisuFk1+b36wqjh5vilnUcyaIA7Ni8Dc5MGf0HBA9VHUkDmq00PO2xrh7STehzKSL94MjvfWltzW8JjiXPbBEmJ9TdPSz4haqFuXIFfgGI5gj0MOk2iSikCd5m42vkbE4jT8LIKSQXeuqrLO9NW0PCyTbf5L6RZdm518lWq85aXzLSdwykW4knTsnMR0IKCrOgRevb40PrppAXN7N8sJhWna2q4LHbs8U\\\\\\\/N2fyr2W9CrQlx+ugRZvqrUUH6zcxXMvsSc+L1df+vQDSb9oHbYQFIzMQZiYne0HjYd8c0Z4yUDozdwhxRtMpMhc3q1MuotLLVGMWNhl1qKohGQPBkwTNGsYzl4tXgWcCuXnbzJX+t3KJis9YthYIwsiTus5Pl7ttoeEGF\\\\\\\/r0joWO2xpUI57\\\\\\\/5rBMywsCQ1hUns96qLutAHkVa\\\\\\\/tJ7u24omF1bpCqTAq0qLzu7siNyLglp\\\\\\\/9CJ+BAIolyWOvcUGV\\\\\\\/rEIoWIs3HjazXdh12vPj9MnrB0Dfaf3djEMrqY2jrCLFvArCNsqlwLMhRRM\\\\\\\/SoyUdZ3qn3OTPPnzakQ3pKbeJDBLBUpgtHY4q+jvjS+7oB1HmVaSG7fxeIp4SYsC+TCTp9kU2NQDt4H1TbAxxsDmc0Dog1dKl1ojTRzpjAIsFuBprRYsl4N+8Zu36DfpC0s8k0sm0MgdeEoS125HsTMG+0w11uys0ygz9fM9DUPuQbcpTIpOlhZhfcZjKgU\\\\\\\/5+iOx0gSb2PVbyF4+LIH7rRrofujTJpQZp5PI0emgYJZs8tohjTMaGJ6htT5sA6eatGfdW074vKMIutBGAe42RELBQgYyQf4RaZxND3EFY1XqnDE37bW05sok2exgg4F0i9hkzJD7CSlHpiGMr8o8bkGd9QvUhHvKXJC2ykF7+wYE3Qk8REDx9pACyz9LUvb1M+o\\\\\\\/Zm2iaZOwVDZfV0qPL\\\\\\\/kWrWr8eczZs3wBMl6+cYq12Obudv1kqfsc0GhQyUn8C4GE41HG\\\\\\\/5IgonM9P6y8yBMUQFpYYxf4c1pyI0dtBFhK2uY4OR7O3a7RNN4hp2X+iyQ1cT5CPLxwXPhjh\\\\\\\/SRWEqfrLhm7OPXf0fwFFF1E4oDP4wSk32Gs7QeR\\\\\\\/rOiXbK0nj4bqENBycNT4JdFMJmo4Wde75F1cVQHo2f4GzrkoBpfJN06U7EYTtLmT5dwu8xS6cLNcmf\\\\\\\/DWi7iDqoOe3W6zyJfLrmTu6Qjjopf2gOPTBh+kGFmI\\\\\\\/P73Kn6H8e50vW3kI+OBKRK70RpLQIKjNhPGqr4kZmVhTvKtkmCicsiJy4Vy48B5IGo5pW4rr4jYDrpTbjOX6Zl9VDl8eGfhZjfeK0Lo60JyI5wrkDw+\\\\\\\/xyiNJ9sy\\\\\\\/kCNGW4J5MwRGu1osVKtwpUWqckeOOSUf8AB0y2cP48rmtbqkl7KosN+goQ0kqJyECMCaDg2Zbhde2ogjAFNXOrzu2tMAhKahgVPq0Rn6rqf3Hw0LOOhVzRUH4+u0Eg9Cjsd+5o6enZNjvVlJLmZN2XxOwt7XpRTHHGvHo9PyAmEGl061rFgTGAu0e8x4HYjSPze1DZlox5EFzqSz0Nbx9a9L5xLGgwSIrUexGsdGtf7vdIAzMl5cQIDKkVNJEyu8u57tf4S3xuDTpInKYl6H8zvzvGqp564JQxJqUfoDjKi4QkKAhSxfgV5SljUcbVWtI9I+RafjUuDvfKDeJZ27lzKS3F6EmFHMI8u3do+a++pShAUAcIHAuPnNOiNq64jxS+27NRrL1s9pSvPr471B8wg+x6Z9Xj1TxqxpXkcIcnM2suvkUTkEtJWtX2CcgEJ8o93Lyy\\\\\\\/0if7Lh08ZDAtnmHDEbY2i\\\\\\\/BDRxSCGlqmeDrx2VjCo0xyQIyP4k\\\\\\\/xkRsKw8nNhFKUO0j4GXB0LpqnILJuNUIrKkfhwg4MB5FPWdBk+sHdnrZWNmFLSVk2fo5Tqs55DvrGOvh1cXaOsEUQp9qmBH1tdUA9Qrsy9r6JOV4RSISTrbb2SQbWwrnxJB2WAjHmu+bIGnDvCptSZzBRDMk8LW06V8T0\\\\\\\/argxLCIFLr8D+6mGz7sy0dBlWGKbkGzUDTgS9BVNjfN7Gio8RqfvMd395amNdOE3tXSZTtjpguYo3sfCuCRq5eKEAUxbK9ziKp+2QyGlNWIjlt2u79Q9TjGIOJVnvcHA\\\\\\\/+CLMfhXup5s6VOAZiNccZUVHdRgTT23ujrNTojd0puwuSJ2qvRvTQrvhLihbltW4\\\\\\\/8p8RDrEXwZArBRAwNpXS2zmERC4e4Y6xUSslZje9DGSmyY94CLqzOD0RNhCaGijZk0PsugARnQa3swHW5VYE2nMs9cKU1XM8zq2wAMOQFjK5H9Y2SGnYzrPN9682sdmX4DKWDcPpQhSKeA1t0PlrLAIbylnRLdEok9bZmym1MakXpANDNmPwSdRs7mmOytkJApWROWX65HF2X5XqRgXVyX4DyF4RRQ4nEN+AM+QPq8glagHxVEJfnctLNWfne7dw6Yk4Wcl1XFKBT1x9qpK8mlc2xrCfpKXCAk\\\\\\\/ha+u\\\\\\\/p\\\\\\\/kzBFRxDxlY4MbFcexRe09uyca2AeqeEAjySYyOj361Lxhxd\\\\\\\/Ic6hyxfhYZCUh4p7MrDnnJISLagUNCHJg2YUQ1sxYIW1pWVwF00aG6E01n1OfFLq5JHRgGeNuG4T3IlPW\\\\\\\/QR2F98zmOfs55aRK6ggHqh0EnBwEy\\\\\\\/kLrPPl1Of4Q+GqVBj++tZNSrkmODeBhdqXinNLOxW\\\\\\\/BmsvI8woAvADfg3+AJf88mrs8I63ZuYb4hHNtIv7Yxm2up4+kAcK7QD4cU4Bt92kIHQl3pSPaU6yej9qU\\\\\\\/lZKfubqQOiNIpkK+vwQy6L8+hWzP1hz4p0UuSfBsldNCxWXdCMQBS\\\\\\\/Rm\\\\\\\/+LhswQzS+8I+uK5gAaq3Qf6vltYaNizGau\\\\\\\/Q3NMFgJBP0ayjhdjduCWLs7KOl40sF5gsSw11pMr+7esP2inIVkDys0PeR17TaOBWRTToD0Mb6h9vI0pLIozrIFyLh5C+x7X0yLIpn0NIomX+XeyP5qSWZ4MLESyw850A9BC4kineZPgQ8hGLC\\\\\\\/5SueZ46co4hlA58C3zaT6s9vpIsvRt+\\\\\\\/1n2HuYnDORr8HCvQ5x8DRUZnKasMdSZFmecCwxsbzOE3uRYaucbqka9idDRxwAP\\\\\\\/votJenU5Wmv+nfu\\\\\\\/JqY5eeAmb0kHO22TjQUy73J7Is78VSuZQfSHn4NIXurVulqfT3NgegHo40h5jTghejO+cw3FCx8sHS\\\\\\\/FHILOa35oW8JwUWVqnY8ZZLruSmZss5MdvK6rfUtSkQ+xSe3fVmiNJ3u1VKLAmRSuvHd\\\\\\\/Ow8RMwDRB+8IVSGBBZK\\\\\\\/TyCcHOcBLdPQlFFAT6eGTl2dFtiERGBwbOuYzL5qI+yCWsDYLQvzOGNWDGS2zMHlvj1CzUytbE7hIJu24WJpZQc\\\\\\\/ct4k+qHo8dv0B1fkbGO7m5OsLrOb7puq1OF9GQ6WamUCOTlzMsdfiRkWGS6x9fXG6Ll452H+xAdQpRtQASELSUcYXa2iXt4hLD5pl5BcZIbma4LNsJ5lgH3Nv4FgydpxPgvbt2pP3IX7De9nnXQdGZPDfVKnj5B3MSf6JrvGvs\\\\\\\/chwC7a\\\\\\\/h66uuncp16oF0tUNb+HY+6Q\\\\\\\/VIZAO1SISSbKCMD2dby7D5vGgxlEzINzUcO7ovI9V71EAF9A82P7V8d+uu9HtPtn0kMbotD9liJZXKBGYk+0JAu8E5zV6ztYKMe6HJ3OrY0n8pcLhhi0jq\\\\\\\/vK1u5LXh7BGUmc4DavVJCv9Pq7IpfLyGv+I4ZC5dCCtmmgSxUkZPy2v7Lt\\\\\\\/\\\\\\\/QfKaYtpk1myQn0coSuULPb8Ln\\\\\\\/kksNQHajOhOfCrmdTpq2WwDc7tJxsmDa0RsyDwuoBUe9mvPHjI5CTLrAP\\\\\\\/Co8TIDZDN4y6yo9wM\\\\\\\/iA8mV2t9uo8C\\\\\\\/\\\\\\\/JHjZyKcv+ba4ONeKtYnlO3AhwAxUUtkSlu5AP3zxOCHR39TFpe4DAD3PlOjSkPZNWuoXBl4TWTJKcH+JUvfVJ1suAU8xTTn+n4ZSkd7iHGA9mD9AvrHQPxyyyy5W0buAoUWm2pWDDDIXDlMa5J4ptW3CQxZ+WhoDmgEcEG\\\\\\\/G9KYuJwJcPPCLBAoxbEZCgKH5G0tN0kA5sSajOunSPPPAX9z3+3m6FKWD8CJfzOLo82f0qWgdjLBDC+dgxRocI9C6AkKmsQMEau3wvx8ZX\\\\\\\/qEaE8QKnkpvd8D8QhxcIm4biQrJmdCiK6b0g7SUgzFczYj4wc3OLWPMVS0P3AtO+d6RYILIjU340OF1CCG9lk1xEb6uPtGFBAa1L5RgLgdOhsnrn8auAu1\\\\\\\/7ffVF+BK+y5GkrtjwxMJupPsE9QVphqk7dwJIKtp6r23xAenO\\\\\\\/KG9pnql3IqCdu2sKi0gQ9\\\\\\\/B9sTIGosVjatUwWj5ywrFpw0J0d7INn33wCdr0H0atHeMP+WaSAFtMtg6UjSEP686RVA7MD57c8j9LnBKizzbYE37PaSrlRrOQGD5eR+lPsrKdTw3riAKWFWvLhY4V\\\\\\\/d6OcpXNxqw+L9Or7Is3fzsqYd4BgaHj1MuytSmXe1+daOp8paAgXLoJiaJcqD1Ady1AOn8od7VF4qeSLWEHblSdPccK\\\\\\\/tewEJjvSO0rKcUTTpbZRynQ\\\\\\\/4dv92Lr4OD9hpSo3aTqviQRSxZVG5uA0+kVXGsTyfDtHJLQaf3nPiSyx+XTr5jAS8uvdv1rC8qFNelshRh3VIUKOV9Geyy3oSIyHe3CEUpqeBNUNW\\\\\\\/Yi1pQnj8Ee39McVrAie\\\\\\\/yuR5p3dovBQWnXxxrdPDFoFrcRtlKZ\\\\\\\/HDn\\\\\\\/LlTEMbThp2IvuJS5m\\\\\\\/x2IqEnWsIOMcSkJf\\\\\\\/9d6jg0b0I3oaCPPj+6Lh7QBqr0lXFZxS8us0GKSib7LT+Kuy2gYwRN7eyIepPNOW897oZRx92n4D2MHVLIoVCjbwFiNofMZ4H1jwuctZP6pe\\\\\\\/hLP2Ey+1IF\\\\\\\/7v1FRz4G6VPmThhqLcrRTJ33xGoQalDwKX6P8vD02qbvvSHwbNMK9jH5Jyh9qlkTG4ZzEEE8pG0WI0rnNmWgmEEVF37BqFqlyQT8HLU2hmalZdYByqbEl6E2fYpxPJSKSDSjYvju7TShfyWv0JXpkGNts3K3Gizz+nBDtVUrANgKWfXIvnhDyJF4qOU1CH9MEk8cqtO+GcoTW53q+OaO1f9Zyf7vGq9vA1sQ1AUO2Ek+XK6z6KMoEgWTfUiP9zKCz3zwLYkawWOi7ZPFMHCDGK6cSVonwX4cCSrULigI8gm2iqCjpXh+glDtsw5SzknsuwlrRlEG\\\\\\\/FIzqfoOdM6GX366Qe1U2Rd7MvW1TJ6BGj6Xx619XT4azRSFKeADOUUqvA\\\\\\\/bqqyaILsrQmvrlpYDRiQrcclimE33yyFqdQWgrrMfSeglqwJ+3NSxFED4njnqzxfwX9bOBTkxSkzL3OPnjiHRfokJRq2DJGKC9QCeWpg7NGZtdbqL2Va4z73\\\\\\\/AdQJDjHj1RIE+miChvDEJHc14AoCzXu14S4LOg8i4M7Thu1h+CmlEUQpAphWwvlLGkmJdIxwFQq3BWrHHu8s30oJ0Bk1Gbqhdopa\\\\\\\/1wFH5njBe3jz8krQTiA+c7J1uDFyBMck+vc4ZHWP4Z7nymJ6cRHebnlANa6N1O7WyGNAySrbx9Nv6yWTu7WFoJFDNbp6HU7i4vq8rr6kkBJp8RPAu+O8z7bLE8iairv6uegkVFrZcSCpqEBYejsx5dDGXkrK+qEDNsSvG\\\\\\\/o6mtDTI6qjE1oRcINjdd7iAcAfTu88NH\\\\\\\/bylIjsovcUknh0+H2NOjFhFzDAya639wHFGDlpzmi\\\\\\\/JuEbZUkBiDtpXlqzNx6TtJw5Q2xJwE0\\\\\\\/1mbXasbjXPjkF6ryu\\\\\\\/bmd6UmJfQ5gFdRkH3Q5wUzbI2+OTRRDiUr2T6qt1cI2EXny0xQ9uAKwsg1dtTaRTBTMaEi2zUzwjKbUu4Kb7XR1wQR2xDELrpibYX5YAW1WT1nrprIfNC+UFzF1\\\\\\\/\\\\\\\/Ynng2EwTNNIhH7LSYL02KXTHTcQha\\\\\\\/F8DrNyuXTYIkdeUcScB\\\\\\\/AzgnW60WMXTL90btItMzF8mr\\\\\\\/LHd9ez\\\\\\\/lOor+P2C6qhZC40RuzymATPNR+NyQCB5lb5AlraicC6ru8hy43TWA8x2H9yDxwzRvVnC5\\\\\\\/32qLPgylruhPqGKL5zFXBL1X22TvI90R6uK2EbAyK09Om531ng1DqcD4BeFzeVSCByy+YIJHIkJvVWMVtT\\\\\\\/zQSnuKh+PsNevS\\\\\\\/HaQ4dbVg0P3pQGXSm3cgnj9YuA\\\\\\\/SJZzaFECHsR0RwApwoM6wuePNSp22QtvJ4s4WF1F3NFMn8hjo6kQeob3kGV7Rc\\\\\\\/u8IvNZG6fwdHr2d1NrQeSy9pSx34I1LZqzSbxCUu5pAU9OIDag7aH5TOIZL\\\\\\\/x2nJygWjDCCfrQ4x0YeoY67tMoIdr\\\\\\\/aWgc+9nu7XbK7khQH1IPM+aCUheIuUD5OwjIC5S9IJzPaobGS5ZAk3qtdUuLYTkVF57\\\\\\\/WJmToO2rHZZD9ddBzWpfs\\\\\\\/T6PmFNocsSK5K0VF6o4BNF2MaCmjJmGuQCHIsHxQoAZk6r1\\\\\\\/kdI8mdDHuyq2\\\\\\\/HSDEc6bWQ4NtSxaFi5xM1N4Bv9KEwhOJ2mvAF+ZL\\\\\\\/kCLsZj0eoJ7SXIXCYuw8A38mS\\\\\\\/paGVrl9Y+bCXljyxYGgPkw+\\\\\\\/W7IjBjJtAvEMcQhULhwXmUioQCEmFkQ4k3+SFOgk6t7BiOinCvE9VJGxHQs0caSc8T9V3Riy5v4zTBBHOJmFqOoNbmI9oCP3rRQxyl11mCofqNx2TIndgYBKu6KqyJi+Z3MFC2nYU4IyL1vgMnji4mof2cw4i6ESH7Da1pgJsS8X3pxJvwFu+ld8hyT3zZYW6GJPxAGAbRILjoV4qMaalLiPzT2R8cnLYYI+s0IA1L62CgaHMDpJWLBo1z1S\\\\\\\/Z4BKfx\\\\\\\/rZewIqayxNUyBt23CerKLkg\\\\\\\/KJi0PzcTrmPLOK\\\\\\\/+1\\\\\\\/uzxdgwoe273em2XlKGpEuqTZVwdvzV9xPboQPzhfsC70S75PrCmPVup3DuYSTWe2hNJyrkCFHo5Hb3JG+sDRg5zNe\\\\\\\/34rrK5HtKXSr6pzwapcdcYVEXcClw\\\\\\\/QWeNDe8L1irtYQzkrvmz+xWx8dyiC5ZxvKxO\\\\\\\/R7ggo36ACvXFhMaYKJdjnA5VKAxWbYtcWGmxonqvbKNrE6fBORKble25fl\\\\\\\/9VLCzkhUALJ+awcMSb+KZWTelD8sb8WaLfd2MsjPyCkpgElmEqM\\\\\\\/p4GzCMX7k6BsMfbY53C7MM2Nk+Yfeb2l8hO4X58gC+nPJfVEHE1\\\\\\\/jLPNCetMioYBjreayJzpd6X1u38DXyvyUXdEiDeyW+UTrUQlGgSrruJAyamXXfmxDbV6Qs+YcpuIXjmMTkCjJhFK4wbYsdL5E8FrnKqtWQtPGgvK\\\\\\\/QkbkyZIDooOJ9b5qgC+rj2MQutSelTPSo+nUQUpQhi4ARKr4PA6CrWJnloP3I\\\\\\\/GrOcw0B8+eQvOLsngd00Nv6HgLsAyZgox1fSSnW2iVdU2XF+7AQHthkqHa+om7LnLgNOz4duI+qSNXm54iwmOxZRvJY0\\\\\\\/IfG+F+ztofRitkC\\\\\\\/C8wU9ZNhcAi87AVQbO8x1vQrH32+j7rNIStu7HQw270bAVonZtrUGEFBz60dHRf5txY7UWS3HW3EoLe\\\\\\\/ZOVqYUVllCDgAvu8r9l4MyDROFwh3t3R65qZyGTPVQMeXPq0bi+9hN\\\\\\\/8zFm86yf3lMj1eqAO3adw5f2wdDT72XcXFBwPsL1r+XXsZXzFk\\\\\\\/lZfgMEM4Nj1HA7VBcIUwteRvQEThtQtrdmzXun1JT3JEZ4OIbSqwKXiJ\\\\\\\/SyWpbs+8MaxwTtiImmYu+x5+\\\\\\\/YjD7HH\\\\\\\/o54K+fdbu56IFm1n+HriZD3PYGFN6dCX0YuMIUYVWod9RsqbmKkvsMQHaO31MN3q9ODAfCb8U+1Hd0qWboLAxdeiPANtaOcc6al\\\\\\\/\\\\\\\/nLygRKWObVxRThHwgaVKBN\\\\\\\/TN3dkXlDp9ZUflYbADi0uy+iFSqwwnw5keEDtQZtYpg8bssclKegAhJu7BDj2zFoJJczG5or1gg\\\\\\\/nK7EYOd8qblxDYIByrWj8uiwg99jLDn6t\\\\\\\/toFHaDrC1CMlkG8S4eBq5psVPSaRRTGcTDperXdeDkQahSDiAztm8fFf0GM0tzmRMB2p\\\\\\\/HGTy0KDpIGFHOfi8kh\\\\\\\/Ka7LLDXK+e0aRsTlC3h+i4ZfGCDlRBEcvZZ3pc4W\\\\\\\/z9F55jvYXC4v9F0tSSCCohQP8jcI8\\\\\\\/j8S+ja2XFt6jI+oJBJDLMGuyMI\\\\\\\/gKeM7A4in5jkpX\\\\\\\/3JIsIzP792AqJ21K31N6LCgeT21EBHM8QVMK6XlVFU+qjaUNWxd6YIoKesidcKD2t9ORzircN1ANbFhbbezvAHbCTxmcfuepb4CQJVLQLa8bXLM83ttIA78vwSFIe1LxO91nhYPh2vpw4GUdQMxP5f2qWx6RoFei9wMAh2i2qG\\\\\\\/BAfHyEGEQoVA\\\\\\\/GUccxuehcmD7Tu7In9RURSWpUa5qLVruR8l4TOwbO8CqQPO\\\\\\\/vMNVzQ47piP6hsx\\\\\\\/flDcD\\\\\\\/K40z123QPOuHIBxXg7oaYcFtu1tqiQW3XF3\\\\\\\/77lbSISgpRzgu7IcDljqNgiOR5s5fKVhFjdBxfZk+4su2kjByF+8AeeBtvBurTe+LnRqVkR2gn6vYaDhUdeujm131MgdYvvh1UfPhZXLWYNoT7yuUdUuaU2VIeexcCl1dB87BNcDNyAnegpYXYPnHDHYWpNfzCYdStGYxAYbxds\\\\\\\/EX6Ke8mlzU9oO9HbttD0i6EMEw509QjNKcjP1ve\\\\\\\/QfscMT0JsbT3lfaVeKf3XSXxJ6H1ZH\\\\\\\/zfXD6nWvAonu14ZA9cjZ+jq1tc0TITPg4083kG4evAvQlLzQT5Hc\\\\\\\/Sbks4ygOI\\\\\\\/W77O+HrNKeliDBQL+H+6h4lYFuwsvow7aiXuVm4ToPhHIz86YyvMG+DYe5HxeAS9hmwNXISRdoRqO+jMA5zZB4xk4WyjAhoz2H6pG5vqJSoKx4XuIMua5q\\\\\\\/Yfa8\\\\\\\/7V0\\\\\\\/k74EYn8SYvwsw4ctxhhFH4deNaSJo62PYoqyz5ixZ1DgeatcutxtIXNnzLTPctzufYX5dUVZa+EI7frI+jsycormJ5MuN7qeeNAsjrMsqYzay13UgdZouKu55gDcyKu4AblmDXfaAw5asE9GTjhqKPUjoiiRsFTrM8o1CI77Xa0mrmfr1WepytfZgkci5Wt7zg6rUO+4d5WsEHTYktrN2KBhDP5HuYdPpZ6qmQE3CgTuaZABophEHihX1qL43Q+FBS0XnYfmo4DjsNiZL2H7NnmBKqtK1W1CHnGVGjIjhzOkX0px04MiKy2UAOL+xuq7l+VtXuxkSNN3oVEzuXUPqTfsmKfXO0YFgsYoS3BBI5FT94xFDgzso7pWYfsPZf9+LfXVZAMJoPAXW\\\\\\\/tTpI5iMz5NZTx27+Vg4GNKl1sSiFT8x83jerL\\\\\\\/wfmx105Sr+xXI\\\\\\\/ZLckFGThhkRguoOxBj9yZz0+xAKIZRgrPN2VZK95HiSMHausfsiWOnS8lPCi4tCUdM29kE4ypqKB3eyX18OWY31\\\\\\\/HCoNxmRvVFc6hb6NG5R\\\\\\\/UHA5s3ebCFj9wShSNMjGr0oXRO4RKYK1jZy4\\\\\\\/EoMebkUCL5SmoNnf3Mkt6oh84GFQ\\\\\\\/FxF29bDGoevO7mFnWwChvon9RGFR\\\\\\\/byqyuxqpBkWAYjdPfh3NS2FblvyJA02cmS9QYyg+DWHMJvrREAKzL7PPvuS6kt7NNApIycH+mLtS5OcX6uWgiExNEExllKH16Nw72a+hwfL8keB7M+ivntudKx4c2J1ZDIk9KNJgnHbIqmS8fVqkstF4bjaUUVfQDOCPKuym7V4vNCiMBlUnNe3Dm8a612MU\\\\\\\/wUp5aQuVhh9OgrYIW8CxSGfcTfBr1fcsSaUOwuQK7D8V87YMYuUqmvJXd4+cCASWSpXObQZDq93I6mb5p+DjdojK8BNy\\\\\\\/bKHpNBXs92GCBIKWmYa6LAoaJ5tTYim39k\\\\\\\/OTzKSKE+52ioIsU1UI8cjdbu3rQ1\\\\\\\/em07tPWcZgYioYXWV91Mtc3DVKHHvnhvH07K\\\\\\\/hux2Y5gQuPmgQjY4ospL89pG5ozmOqHWMJCHHJ\\\\\\\/zOVZ99K4hT3\\\\\\\/CHeAPRDyLFn5BCg5VXfKweGpGxc3+NJG12xVpEhS68qdCUU3SDrDezDMHXJD6KOOSpCCQymePQyfd3G7dZznUN\\\\\\\/4N\\\\\\\/oDc9ZAHMgdUI\\\\\\\/jS9jYIN+p1ZKCOWMZvBZaRb\\\\\\\/UhO7Pl05zfx9NqD9G0lD\\\\\\\/DvX4IzFSXhi+zb0ms1OlCw9xKiqsvtB\\\\\\\/4\\\\\\\/L0rSt9ZSpAfEFJIfYAJRZJEb5oUqQwW1ZlRHA0TDCom2HNx\\\\\\\/9KnxHzN3rtE6RybJXHn4KnRWYsXFV2fddvKkcnAxibxa\\\\\\\/IbLGw7t0paiyZR0HZ\\\\\\\/dfL3J3Wm3gKvKTGNSlyJ7c9zD8kjBnIKmH724kmD0c307tf5tFD\\\\\\\/OQSP5bXoTDWIMaOUaePlSZskrF6LmbkFlhKIKmNCtrIOGv6FSJnt+4c+EqHglcU2Ot98Q8fcEZ5jpUcP1aBBO5lcZKrppH6lX1jSY+89OqR4ZkJFHlcUko8aKw0TxvNHq+CtyhZVMqiza2mvwwUGFatKIYoEtE0P0jtdlMXOSi8rCiJZDEwVfRaxzy9wQheQzMopP7+qbQB7b8bkCXAV\\\\\\\/MPu66WWjYw1mkO3Vx+L6\\\\\\\/781uxPK3OFxlldwVFy7s\\\\\\\/rpBv7KDQC7rwg5NYkTBNxW7SZkXHQV\\\\\\\/antn6lsXaVcBnTUWSwfyiEtmi7goDWZzASRFYOEMwqMXVVGVl5D9R3oJHKabholpXbRnCkQjoiN6kPmb\\\\\\\/nFOdXjWtRvwFRz1fdsRAmrwyiReX5CBAce7QfEV7iUNp\\\\\\\/03e4RRIA4zNuNcRmO\\\\\\\/orVgCUTfF0MeTc1jQAzCUbCW1aEZp4OgI2n6aN8W2WAKL2i8myQ09sosUKov96v3aMraVuf\\\\\\\/lFNjFpRFm80iqagQq7k9efB05q\\\\\\\/8wBzO9nwuS88SOGfbRVgKS+CUMJKbRwiDzgyH3EcgySWbFAlHAGy1RS8fsZWg7Ec38JRdjFyhgfsd8d88yclCOYUdh2b3xFlXqHpSykvmXIK4U63Z2Y4P8ySJ3BVaX9GGXXtIZlWoSJWlztIrsTCEAO4B0sZvXfZ\\\\\\\/m2lKL9w61ScjcTSGBrpoxDbW42K2YwrdCAlCJkvUZnITlosc3kyiLL+WAovumeGGTatJQYSPxuQpqRg568OmLYeVi5Q+Uw6p2wiCT+C8KxLhLSuR1UwETC0MCpxHm8EluE9svHWifUbph6idJeuHjcPovXrUm4fSK2TXHRP8EVySCyszm2lEz7XZTadphHaT31opya9R4GcM9w4F1vzqgIcacHc3TJLGr\\\\\\\/M3qahxEzsyAnKDGwpR0VQ540mX0pwE+XFItav98ytV8iovkcVL+Mr0UdhK+g0\\\\\\\/ag+HqsM3ITvchm4HcEntvCrkVwgB7QpM748Jn42tAU5TtwWH7\\\\\\\/ui+D\\\\\\\/TZFcF6xteOtAUudBWwA3s8T2UiblNvKHnQR78tSAeHqsjFuwiAK+vbZQbdT3IZRh8AZ1OOOnfQ+9hSabqHAAjP83EOJ80zhfyRwZqBLYgHGxbUE+D4d4NDXqbxMpZ+Lv4EuJ50ATIPX7udybnbYPZMQyiNLgqEk45Q5kfKq1cVgwc67cAPQaylaZknZT8Qkm88avv+0bxYX6GjQ54iWNGkYLJNxvQD5sRgXGFt29cVbXMBdPdupXbSXmNmcUg5d6zJun2evI62iTMpVOodtvrhU3oanuvwwIiw6A+52dxY\\\\\\\/RCvq9t7330qDA0hNQMdzYZJ807c9I+gpxybhPIKPT3R7e2Epy3sP+CvUOMOVr93KRG7S6Pf6jegBJoc0IY+\\\\\\\/vb3cFFLCrEYsjjpHovmtinrF\\\\\\\/JSOcB9QniIG5PjsksEIog7Z2T\\\\\\\/HqbFEuGLrn7ADF55+YW1SEnmwthy7E17GL+psgH62QQ8PC3OlM1waadb0XNOU5+k7GSq8foj064q8+dsVDqN9ojNVaVdZrK8cDsklb260R6emAjNvCbzyBGYUXrzc2IvLzOWRezFtWkWAuxpKUHGu79nDigKO4vJVfZXb51duWWPmDi68blv1UTClqNCicYKkucCcAwgs0q5U44oQIbDz2pc231MF7f4Zgcip9io4gI8VL0jLHKkxqjJcVFGVjRzMrwo4UXkVG+nYse+P1IgdHTC2z4trdFTwp4lrl9gS9Rw2gtDRvXmCn1p7+3zFEr0pQET9gvItX2y8BkQ0LNlO3\\\\\\\/P3lspZHg8ebKCJHk4I\\\\\\\/Y49sueNX8fUTArrLWb26fV83aHUJ2GYviET2qspiNWIXII62bWSU1VCtE27PNHQzLXLjk\\\\\\\/vJB8RTWiv7NZvGS+P48uJNw6Faa6S6t5cNTYusp+kaCY+dOJSTdg+qxe5UuB4sOTXFVS0kwZHa01U+ulxA7atnf1Vz97pGB4m9aTC0tHusMq7ycDVCIf3pRCtXKTJ++xsYWH\\\\\\\/eQKwbHfbwGRCotNyZx7wVnT+IdJzZX\\\\\\\/q8Sjr8sjqH1StSgZIYd40WosgGKEO9BwjGSAdetkd1xiivBCZgOYM666ZWjNs+u9YTI3aP8J0BHlry+73MDZSWolpDN+iaR45zCIxQjOdy+H73Y+Fj\\\\\\\/inC\\\\\\\/ZThMNKB8J3vTDZaH8KRz0GFA2vyqpWy4C2HfQMmA19G3MKkLkLJAYxES0uk+ezEuYCmjcvnVTJWwq2OERx8TBShmqsx+4zbQX6MaIxO5Aou\\\\\\\/5llhPtvZnBGpq9D8naOh8PbQ3MvClzShcBI+BmTGe3kwcc4J8h91WVEacFLZ24OkecZLh1SBlr+So\\\\\\\/HEe2TvbNAcIIUogKbPZqD86Dc1ssSvnm3\\\\\\\/sbfpIbBPNoC7aasNaZef\\\\\\\/CfIcPZ7KbKVLsTnLex7XlNwtNAOqX5OKcnUnWZzrXU\\\\\\\/abp+qz2PupRi9nyF679hrjpiodm3obisWcLbgYFkBm6HYJnyC3r2Vk4APJlL5ppVyEdv6sJzHCpuLRKqho2aqRz3vi7fQ\\\\\\\/fm2zQ2jxtf98qT74eXiSAUEkOEeamu\\\\\\\/fC0NiIasnh3yaUQVko9rmD\\\\\\\/F8FkLsNh5jDE5yxPCmcVA8ZbBoqjHSiOVL6ovrySRqDW\\\\\\\/BfRpeMGViA5iBeENWhT1vxtvrizji8mSbaXVKOj5sUn3J4nqtwNpkxPDdbaj2BlDgzqpIhoj6Dl1sslLCH2gfWQBhUXJFbNC\\\\\\\/WTxjJ9bDEFuT3+ZT7mbgWsyZqeRWu\\\\\\\/t8tvxESjBDr1e6f8eMxQ0HLFCAS4BY\\\\\\\/6EbnK1+ZjUNZVjM7N6MwPhF+PyGFQhxSba5NjeaGTEHJNyzILcqYwRxfaJsSBd+mDGsm+YFOacUoirAlTNsblM1MNJyEGJTd3\\\\\\\/yKDAJYKn56aWug6+K\\\\\\\/3aMz1qRpCr4QD1ophBTUGvjnA7KNMUCSU+NAeqqqv4qQ\\\\\\\/qZAV4H5Z3QQ+WTp6GaT9ofvyM0QxM9sUp6Jj2zHwOu0MWIctt\\\\\\\/1QmjxPkWWDzVK4OX6joqtPqOEQBSeK9ge9vZIOmEKyk+\\\\\\\/x6zJytJzAECbOnNeJkHQ91tgrpcSJaDbgrqE3MQQDMaBc7Nelda51NaQpohFbA6bvmoLXpQvKCU8rZtOU5CYEQRIq2KDeAK+phZ6\\\\\\\/2IFvsT305DuSkbHPvibTcvfYa+8QM+Fm9Kf2Uawl7iZCiosZtvv5cUwntrax0Ocb8GWEluTZoT9G5ih2m8Z4CwEFcnUXHMa+en9qmiekN9o8MlAv8nE8cMsNJAd2ZwdO3cbje8nqFt439og0xQG3ralIica9ddYn9rtR3441ZSW2iMV\\\\\\\/MbYTzM6gSt8qbbBm34\\\\\\\/r0d73Xeizun5ThLOTFub6DDp4rZFOPm96CFfsZEOGFp+BOEvqOjLvyPTypvluOggBfMtNgnX9Ao\\\\\\\/xeIezCDQi81dGpuWB+BLAxhCPrqRQm8ajRdUYQ1l5AfDpRn\\\\\\\/XATOfNEMb61sdBYNwvKUlg6SJAYNMRmF4HCzqQEzkRAyV3RYYdlSURVxxSIyUcYI5JVuI05ckctm6YbvA6BMhDcM0gcUhRvKAxrILxkNOc57Nofa5DG0SITmpBbs3lrRPoOJjaNacFMLo27Y7fCaq7YvQINyOyP2QAi96DRMVICZDmYSI4jCdqsNdZQwa8A8FUh1qrGpJSJDyb94\\\\\\\/9o16MO7AfJcDpu0briAYIN87ODtDhYU9jSDRHnSHOg\\\\\\\/\\\\\\\/ozPP43vEcH6t+4R+5LY73+uWCIZzj+iuoM53wvfpQTQeqIHIrxVb0skC6AGfmM8NguMkInE+p6DOHANQ08rrvNQcKPY4Hwbe6\\\\\\\/wbZ6IuvsXuyReVfs5E8ks2mlzsn\\\\\\\/wAFRz\\\\\\\/5UkESGAJR5vVLzeVxz3w9pDYAoFSYF9I8T6aNBCQvCoWfP1AdwmVbJff3qkMXkg7MBevmEN\\\\\\\/sFafUcfQSTWygzUJo3fAMdZvvt5tzcjVCpv2fNTDPEOnqI3lSZ8hZQB4BVGpQmfaHqvWMyg4ly1Q4WrTiawWjk0o\\\\\\\/o+LP4I4t\\\\\\\/Cd2nZZLpg6zrc3k98LIijDFmGyMqr9jdQZQsDL3b2fxSV8px+9ySz5qPbu+XxyajpOsSQj2PNGYOQ+vt8hKU1BcvKf4V2NkiLO1sLehI4RioO\\\\\\\/6yg0E6t406k57FyWv1TE9TAvvhrtDoUVw6F+P+NXE8CvisQFNaQDi1YcVZXYC2Mf290vFjkWXUGnGMNvyVdOEr1oITo+lVKDtlly+nl5h2yHnJwNNom0KyyFqG2Xfq15RTjUkm+azbUb87adqM7EWWag6NNXIoMQ8i5FBSGxNzydRUQGcNu7MWie\\\\\\\/UxsO5b\\\\\\\/MfEL+2HeVoES72dV64vgokej24CRVFyo5XZf4mf\\\\\\\/OXWCPFMatEfhXvaY0IO0vFGv4j\\\\\\\/agxs4KK4b\\\\\\\/J9OfcX0Cqx8E5hP9LrnSizJiEI\\\\\\\/JyJrfOnb9IdmWEfedGWMvZVkyRC34YZgL8DeOzhZqpDb0iDKnuLKSeRc8CziNLqWV8K6f0HGfhtgpj+lozlGtcu4yMF0NIcO0C8L4VO8UJX40Dx\\\\\\\/qbHGMQI0gxhGzRY6NyClVSCF+GVmdbc0al6X4XEvHn5abr008eAVEUrDK3N1r\\\\\\\/q73lbI+KH0GNX8AS8CYNMM+WTgwHJhNcZtitVbEsEOn1TzkGd5ftDAyBzBn\\\\\\\/AeHYONvMfhKUWWQv4E3cdjtp4GA66hc+ZCDZPww3Wfvp14i860S6Mav4U+TA5LEZVlBXOFw5TyXlihNL4xt4ynX7d99aQobVvFnxFOyugcYY7EML8xyk9O5aiogdYTKGviHhqGCx9eVVZMcSnGcgmG0zItJzskXyYTndPtn3aZrTXHF37Nt0wcxPusoLVt\\\\\\\/VEDOmxwbceDETcDYE7q3jjlMy\\\\\\\/wihX1OuXldICXpn3n4ssoUlTvBQhjcfszIlB8Zl6+47jbx1ioLkJyUmAuKIDAHCM4lqc7mry2t19muCsg8S9AgB4A3VHSZbanByII3W4ONa8\\\\\\\/WNfKJ4Wg\\\\\\\/O\\\\\\\/LZBCsKbqILyiQ\\\\\\\/q6QCdaV9UrxkG5ceMSynRQ1t4aW+cLRH0WiK1MwA7TkPmVqWGNbI7TyegF2O1BESNU6vhShVHAmuB7iLDNLcBjeAkCRoiIOBxiHbugRZu1iMtj3BgROmIQT7aXXXWvwaXVcPZClibYCbdKP84Q0AuFMJNyP3n9IQD1EYSXL2BPbEHzbPFluJQC31Ah2XCpfcgnA2wiYWjaNDpVhYhaXeYPTVXhKaMBT8lgH4fEsLrefpEAB\\\\\\\/T+sa3PANf+EqUZDBlEcaOohop1QMV1gtr8GgMMUsfqmIXDdHLBKTeu7Isndy5U25mFcVzUQzywTfXKXws1ikm5XrBKEm5FmmVsC+xtMnOgw68kYKyV8Y8zNze6vSXYbvFoG31ww1+jS02kgZnX5YHdB10MiTYf1CoOjf60\\\\\\\/Mlq6T9WrVxUPxS++9BCYYFUrZcjPaUpsn+\\\\\\\/rYP3sWOnPBeAlN9qiv34VorjZtjUWOsU9zsCxIj1a+xN6932J8Jd7n3QV9tIOnp79wh+CwLsBR5\\\\\\\/7YbhjHlzCDMmrO4IECq6pJBAi7QiqWJ5w+iJ6xF\\\\\\\/VSol0puopx+NCKK7e0PYmpjfqFISdV+T1weLOfyJ\\\\\\\/lUuxHjcfiU+NE1k1Jac7iALFIdLD4TwRDWy9DxPbh9rytzkXswpsghyZW6tGOVvFti+JAS5AuzgpqxNU7O1Lge3g8CF7YXIjGXI3XIzuhzwa8+NXCoRwcLp5\\\\\\\/IkL54fzAmQjKY5OCbtenkMcVXJUaLWWzngzSoduMLv8\\\\\\\/BN6mjIDJSUhzCcxUcpADkT+CoBzzBFawmwDhf92UL04wlXmps21DiQwqCWsIdQroo\\\\\\\/DppVDDBjXZqhmuxCjkiD9yLs7LKrz1ovtJfQzAuO0Vn7wQ9vzpkUSbCsxIbOzdiplHXamH1w8xR7moqR7vpreyAwzq8ZHkyHy9dwZTuvydTWGPPaThPn8FzmpVOhVrNnKaL5evOZUrD09RbKntx7jqI94HdcPekvj+n\\\\\\\/qBcvQzgsposmH\\\\\\\/13JSqG1rVCKw27QTiH+sV07XqI55paDpLOchfvWPfSxKEZCV+jzB3FY9nHF+uEsEcvwgzRpPV7OqXSuRv5wIEozf0DJpcxfW0cFwXDg2r9x9USbQ\\\\\\\/vHlAyQv21XL33P+DV8FB1HMoABnHtDpbo0XsWKJ33Tpew+W+TcE2TXpv5jE0kVNErgvrKdyYlrESSTs2pOGk5ICdFMz3niV\\\\\\\/HC+5asxAF2eEVdaePjbnkJl7Spzl93wRbvBQu1xDIBofaFMBvLmpNtz\\\\\\\/91czxMTWxTAIAOxdMDcUA3EMLnwJ9BRiVFThOhEG2\\\\\\\/Vv8zXFMn2ykh5S0OGavbeNRvmenwfEyU+S7DeLMwjUnkgrwIMsGb\\\\\\\/lPY1dZxnWj9nsbiT2tza1fI9gzz5mvSOEl8t6CUhvnhRYSSB2N4Rs\\\\\\\/nm84lq+jgmI0sunKsVonOIX5HS4dpnxgARLFszAkI9vbiNd5wYOcB5M2QUcYk0b\\\\\\\/sNF3K7rFj9CgwNW3B5Zph0Ng5Cohn2GArbsPTmFqtszSLjLFbWRmunSQHSKBQUrmyUPV1Ad2mALxGG\\\\\\\/H2+\\\\\\\/cCBjE822nwqUSrPgklVAg2Mm+OQfDiI5R\\\\\\\/R\\\\\\\/JpHwHfPWdAGHfT7B\\\\\\\/baj5NDM5b5KjYIdNtMmH8FjqJOmTRxXDf4Y4GCJGLSfypK0l9JcvEBEO4aSclcT2tLjYRsAKNeSl1+262MiPfoBMycEx3Ui27YGJrzKOgIvInV9WnUbWpNFsESy3oqdVDmG7Q4sOqRtCl7x8L3jUMXJVeredgb0zYzk\\\\\\\/fFqrImmCbcv0viQ6XiNudZbbDhq37oo4BH8ESkb09+Sdx9B4GwyrV5DnaFoPTIiTznICMV8TOpbfbO5AOUK25l0YEmZtOLn2wGiJDta\\\\\\\/XpXISZvBKcLbX15a7NuW3txEVdal4scpszH6WLZqbcjQdlC3GU6vWG67fz1Jj2lJRxtD562IXn9IBmpqp5X\\\\\\\/LbCKdPnoM7uKgFvIQcyighEYeGq27Axqp7\\\\\\\/oGGr39ggAVQRb0RpSKy8BA6ELWSNbBsWeXiMgwr8IriIgI5+4eochRXb8nbMNUJRQ6O4AtJo03vvDZB67+kmDApbnqcmnYuFjtN+vk4U1+xBRpn94AvEI8F8pkjCR56ZZ2JIlxT19eoaPMuwdG6LSmelBksWDoNexQjiCO90GXeyFchHqY4txr1TGkBU6vUYA9yDQRvmSXuc8yntQ9IQZwLiJNxy8CMw4biMDVSYqSLDHJNn+wSdZs3JdthHNfVehEEl\\\\\\\/jtfehsONKHZRji7D21QCVRLreP6o+UEXMeIaoDcWXr9Ul9kNdwKXCtBwFWN33iI4JwIkVwKsPSduKtEbtAxKXwQPx01hLndsXru7ikBO7JvFvrAIa\\\\\\\/2Fvu8cCTRlMHwSQbDZPuDg+wJ9rZpW8ZHgoHJCU32U7JAhjbfArZpdEUEc8T2BX8v73SjrNcld3cSCITQtLz13iThGgzcScZS7X+C9xNzy0MYlcJYwgEIvJ87e6GIRVVhE0CxK1sOK0yu17iEuohtQ47BJbvTi32itmytXngke75grJqWUQNXEM6orpcafh2Tn0POGaep1r\\\\\\\/2HiU4t\\\\\\\/pIQiR+C2\\\\\\\/hR9\\\\\\\/cLpVvDddSTzOvmnC2dj5c5LwSQ9IiZ8mhomxlCHZ2xnTeYS6Rxgs2p4aCRLnzm1D5NIOXjcqnmgavVPm\\\\\\\/dkRmWjSUgcxD3FxHTxuM6DE+YkFtXnB2IbV0qQ7jxCTd60oKTYatvGmy1GoW0zJRfBZWMugG8XqIjhEh7+MTDJqX1AbLzC5fBYxvITQY4lIaZ1TXsM7zCN\\\\\\\/zz0ViGfgyPx1+Cvd3yb5WAd6nuHFhwAATAa0lL\\\\\\\/2LyBoZN4WqmUSgZzo8qyr7Pxb\\\\\\\/IUyVdp05tczDmanik9BBYbaJA4xVb2Yk+RKzvCOCPyHPbOYoxUw0cIzEWLkYUEApDybq\\\\\\\/Jwa0rVEMO9j5KVzIr\\\\\\\/gZ+Ne8SMTYEZ1j5iape0oUWPlu84+rRiucvluLqKeoPyzOLnZIbBIrnZeumz6z+5TMe6HIHkDLWO9pt1dT4zrACQ3HsmhFsxCCNLtAj41qfTJTxH8stOc4D\\\\\\\/1gcAwtvQpnkZ\\\\\\\/bm2lJ4R7T\\\\\\\/9GSJBdMqLuWluazfacGBiayJ1L+yp2qntgn4ODLN4BpdH3DXBnjbSbzzVTBXgxVT0BevG6a0ohcU0J+F35LlWE9eEmsN1Mq6ULv0Ms0NcOqthJhQeEE0MkkbSEPM72vzxzUEox5ajwoZnwrFJAltfmzvkMm1W2iaXZUz\\\\\\\/CD6J0CZHyAvrBltkGNI9djaYrlW44do0MKwHJ6NmVNWMsE9sKThWXsC97XwmNvQy611t2asfohzoodktaBgbFrlFDlIKbaC\\\\\\\/Gc6YgZAJfRKpHUnDIOKFx6xt4Ym+HOSvfDBR\\\\\\\/YrxS5y82LJ0GbTzX1VhEUEKVdrCa33\\\\\\\/hexVKLtEfuM923TxOcrkRNeHk9PgHCIYs+1Zx6DTFcQZ+vMEsOfkv09MuvR5up\\\\\\\/vemK7ZhCo23qkqArTr8GsIW8a\\\\\\\/728dPWjvrltKs9moiPK1Z4m5nQB5f5r88825bxBVG0IOb9sPiQUfMrndq4QhUBaaUOgruKLpTf4TPgLbbCJFMjwjyBVZB1qs8vYkqmPADRvOSUIlaUArLHYfkEpQQvq7eLL1HtKMazGxfSn9eLdbP762UC+clEUXT1yrYfLH4J81C9oeITKvaG2MJFlb3tIskli9Y5gSIsMPPjI2bagAqOm3HqThvMd30MdeXJb3YiaF6bnOSkVIVEFJXD1kcxCBwzZDXfUdPAOkwApgCzRGyZtrdOgRh8\\\\\\\/qy1xQIM0c9xws1BY18AAhecsf4Ok23qcBnUMV+8Y5UBIF4RDF7F5DUqGgkrUyhZLlAB9zEoSQwXJM1sk9ZRrU8ADQ4H6N3BkLmKs+Wxa5PZ0WQ516vZnlmKzDffhtgqDzCSmaSmo6kn2vkLtPN4+JfmMcK0s6wTm0YKbML5YqV7Slr6aqXuJL+BBtpBpD+QBqeQhT1ssvq1Djo9ZvcNtGrK7uADU177kmF4G9Wq4AhJjUK7qWmSqexPRSQsnhU6KMmVh6t87+34pe1AFL3TC\\\\\\\/gtf5wt5WQ8K0gGAa3Xn90FKL5msz4u25Cq1kg7ilhA9Z1MvAjbPj49I8koFf4NVvqSnACT5FEanFd1xFlUj9zIuZhxEqYqT6ySCE\\\\\\\/IrajUrYqV7js3iBWhUXnZBmWLsKAY8o3F4\\\\\\\/GH4K\\\\\\\/63uAu2Yse\\\\\\\/Rp7xMLlmchXFZ1c0dRmTTPNiQp1jmEmPHbfDDtCuVa4+yO76wTAde65OgQVNK5L6NRs+fouqJ7T2tFKpfKyuF1wifxGE3ikXr6wZIL0TeWhf2iHQbn241Q7mU9L84pHc4o78cyD8FR9ZRWRAAbU7zKSojuSmVJMlPfX88cRYj3VCKywKikIztS7FGSD+hDs8Ydw8ZCzaRRoRq+IPFK\\\\\\\/zactkYu7Hf+uTmEZRK3TdWHCaFSleEhbaAkOeh7202LDxL6fdrVz89XIdzHsD3HWvTU5RvNKvP1aFqTVO4rOaGQ7LlXGNQe+Rhk0o35lPYmanIEyd2e\\\\\\\/+OnU2YpnLnnNG2pg2+OHMvZJglR+xuA1sHBDiay6iViOfi8LLabsx7Al5jMHPqjLmK13tYx8bYce2ywAUVMlG8YXsK11v\\\\\\\/We5z2gCWY9ZR3qx0H\\\\\\\/Vcggte1aLQEK\\\\\\\/Dh+G6+GKJ\\\\\\\/olAP0dEALRaaC4Y5Bq\\\\\\\/gU5vlrUr67\\\\\\\/ymDx68vyWCrLlenHPR3ubFI6oRT2VcmU5bUXmbVSHJsFQKU3QVb9RKaT29yXy+1s3TPyT3htAF4YeqkoDzB16kjjNJ2wrXi3W4q6XHbWuj3py4u7RlRqvF+YShZdlp0k64XiT6Rwds56OmVa7WuF3olMCJcF7rrd6vmp4szjBZT1WpfAKkBQ6iTdFZzmhfvYUtpJLAVjkQTL5kdctdi9m\\\\\\\/xtO+ZBYWDdOCmtVWQuFBrCjzbntDytSb+iJmBaSLCPQ8KlT7sMs8ssYqkubMOa5V6gNvmbaJE6YQH+Z3q+FvEEV6feH5JgbxYATvrbxXHg3+OYy5AQUODtCjKRroOlaHHaGW2zGC4aKYT0NUEwggkkTnWspqqpf8CPfG+W2T7JMn5fpwr2BDtp1ePpv0Fe3zw7sLFXxXhhTBaG2jS+UVABbNmoZsQNFiYP7\\\\\\\/uLagCkT3C0+W9g5nQWyHZrYw+CB0zDmxMxwPisuwbNJS6ZyO1Lv5zRAQNChMPQyF8ldjNdCVXtQ56xLHJzZK18BQc9LdkmErzBzuVmrI8612\\\\\\\/xJNhfQB3AFYIRqiBHsDVlKsKeTEbBNA2uoEw2tRylq9Lc7v0XJiA5e5yUo+vr38oM2ozmc7ZGPnYSkw9RamkSD1lWAUtJQXvgFH91IP4K2sdNzzqfqJBbbW8fIckeB\\\\\\\/GTawXgVnJtJkKeD\\\\\\\/vyslWuV1LzvkRWutrG66ZqpDP3AqwMdk0+oNLROl7oMUuMS6rGHNtgHBnSlsiBfYS7jzJTL5bPNemW0yqtIttGwgRYVWt75YvfDZSUJk57LQ7idFDpysv8vh0pTgJJICilIa6tsZ4VYdDnnwHl1waXaLgk5akjita1D0Iywk03K4WhUpHRFYdtYKkB0rRq2l6gkRJo1uju2e3dJY0TEzs8icX4nBjbpCMgwQcEhVvAZXfxFmPwgB++TMoA6hB6uIsVC9GLhsWBQ75BPOelNXM\\\\\\\/yHqrxkzFEdS4h9p0PcVOgwpeUwst8QbFbQVXjgruVdsEQWGu5Y2M1SFmY9xmFSVMn5d2\\\\\\\/WefslAcak\\\\\\\/QXpyuV43WSfgpTqsK2g3dG2HwKFooRdFmigyQfGul+Si+vNoniTsV09apQ26Qtdq4Ahdc5OGSpNwpy3RgeMjO27KMu5ejZ0WsQQ2HuM6z7tL0PFjV5XSTe7HF0eZKTYUQqaW4iVVPOabChLzTY9rqrjtsThLsrRWOs3rYBsYHsx1pwB+H6o9Dp19RkmUChmM4AAbXymrxttKAva95UvO2PWGueN6gBh6tdddai2ePSNp+28ByWViYXS85cBAOMRRayjj50JVnqQBnF2+1PMG2nn3J+Vx075t+NZBOCrz4wXJoawjsxOjJws1WgI3ZOvcavcHMOFIOCF\\\\\\\/bqsCEGa\\\\\\\/UnhJ9IdYwUTAzXI3IMZC1Al2n3oRS1TBqNOK7PhXzl84B8sRmUgx8ifooKzdgrIGKw7+Xj\\\\\\\/KZknzMp2UONt0fHl5H1yycbNDOs1pEiaaMIvqkmdK3R6p+zA65acNPbKycZXGz8qUeXIY+oBWwYhicRIdAW6ODklz\\\\\\\/garkexElfIqSHJInXnTFrllP0i3Yz+ULLl1pGQlEJouaK9bPqIMbRHeaYRLdHz8R9s8AQtXjVRsMAonOUIm4x3i8sYvs0h5XTvi4GiijPbdYzQ3RXyqRYBZbo3IJ1zL+AS+\\\\\\\/ZDTtnE7BGh5GYRTUQoYhuYulgCNku4TvYAo4Wgtp5bI3R4xpCQAjSqLM5TRAXuxIrZoXU6NNe3ieF85lV3LrEX9Dzlzs191KzmL8SO+CAMWbqpTj8WDzu0pxQfSs2RXb+KAYhJ+5CcPT1ggXR\\\\\\\/g4sGbSJ1tCVPvReaBQqf6BUyPDnq2U4CAmC9+EbuYrPD8TMcAjt5BCro7D5TlyQlkk5DpfHO+sA5Ch4LdSqgJXu6wIrahYwHcywVBAq7uuZqz03qQmAITs61sLIbJl+LfCgy8YUVjaab52jv3R8V1L3tAPVYaGr7vPd48KiHBG+T55aiujGVRssDZmDIm4BaTU9bprwGCimlDKnmsFEx6dlb9rgBamH1kNwwk\\\\\\\/L\\\\\\\/T2muzuHNmS6k5jc5chnTuHs3qq\\\\\\\/sh\\\\\\\/Xgou6UDH2dUyg\\\\\\\/K3p5wlS3W5S9oEEiV49oQcUlGcPG5TlTF8LWCDJjt9AUYwzg80FLx6hcJjIPJaJywuwyJPctDKYL7BxfEvffkAB\\\\\\\/9RBivfc\\\\\\\/fHiEyPnEUCWkes4ru7XLA7M6\\\\\\\/ZIPsxWJIBpasDv\\\\\\\/K69iCCHPWsTZ0+bXCBnPYqxI\\\\\\\/kq86nLnCRgVwsioyYuWOwtxjtJBi1+rJ1aPE3XTkj2\\\\\\\/NJv9m5zWkeruTis7JaNzdDKvuDOkHD9PbGUU5NKOdLV2ElaijZRGnQsYnpxH\\\\\\\/fIoBqvdLeoMHnyEIZfR2wf5Dmnh\\\\\\\/W561EC1afO9gPLqzr5B1ESxF\\\\\\\/iiWq2Jso+T+m\\\\\\\/7NQ3wfeq1uY8\\\\\\\/df10W\\\\\\\/as7wks4GOXpdMGpdSMF8m\\\\\\\/qPYYDS4Htuaxrkhj+LeBiR0BwZ4JJ1vLtaSyrpU5\\\\\\\/hGmdCh9MkVY+w5HIeryLToZIy1a342qyUMwdYYebBNbXJ8EHGrbvHipdmbnzloLA2FsGv1DczqPAF3admF6pyJj0ruw\\\\\\\/\\\\\\\/G2B+tQ2OymH8vl60kkyuTJ3tC5vJRX2UsUzWUkAwm2F9bx5a8a5ZBczOWKNOmejewl70a8JgVuTc9Nft3gYlw3BNAzXGwEBkkZ\\\\\\\/JU+cGNfnEA9ZeBgWpLswZZ2HeWNduu\\\\\\\/aVEnv5jLlKR1BMuFZow50hudxFA9mSE2JDsWqh8ZyNKz7MWe1qocODhvaACQUpG61QtsFgOfp9KtIoKS6cC88dmpptGvcFTt5DF85Sn9sfVRRAjENHNpUeF7m36LaqwEk8pFAFgAtlGnBOi+TcRh1cwKrMDgXvXpNsjTaoR6tWx3mEbSaWBCu3SmpM+5vcqqCvQ3FZyAZUKR2D8XF+dyf0TWj5POOLGIOJs14Y28q6kCCtjCfrFB+hhVz0jsKOK4aZS319fXiOxwsdZE8NhXPBa2CBL24rMPokwCHmq9tsPeQ4rmJKgNikmF2CL+TF5SWWn61eF8FwaIAzQBnCuss1t5MlM9qiLQp6ZhirMp2lC55i2R2QRZdW4RkChUijqA2YFn4CBvmSLgQESf1RiiZQsPa8SkzPZyRQuSkdUQPm+h\\\\\\\/ZDufyVIKySZaavjZLcSVM3LA0joarqd377z3mfZBZPFD23Q5C1T3Hsm7S\\\\\\\/xfQlstTMdLRNFkF7CNWpEGu7+KLOjrbbq4rq+kl4WGP9n8Te7YgfDWWFmwMHBWoi0jifZLs4M6fI5WqAHmXjw3+\\\\\\\/KpOxdQaNNYoekl0fljbZj4aQEYK9\\\\\\\/ccFxWsAdpbivxw5qDQYjhEjMepWhkK8yP6+yUTqTnKSYXRmSL16gRbqa4VYK2xK41xwilXdTyKJ53IAW1ccBTxkvHjvnTahgqqMJi9i7KrxtjwCLrC681ERDzerobyT9Hf\\\\\\\/8up8\\\\\\\/pWuvRw1zJ5bdqUW6ypmys15DLFUcQrp8yDIgOzXR4YyxWUvto3TD4Spjq5p0+f3jIeGA6TSM4kEj2BFPkVWR8zA7qTIV8w6jfslp9kSVJ8YqbQmQEEOCA6Y1CRQQ1o3c9Rydrx5pHaDtc+2Q18z+p3AK+50rYJ35zF3i3GrvblPDnMPZtZQtKsrokrFt7FiVyp6kfU9Qj3OjmDAysexVbvsSNhRlC43xRoAr1pinVTp\\\\\\\/h8\\\\\\\/Ud39ux4cW4HC9flkmsDHQucc49vxM0xwREyZ0Q94NF+4XXKV7ULLYc8vItZeW9dBnAiIBX\\\\\\\/TnT7hs5SDQvJUoG+cKcDSjkHNy566DD85Pr\\\\\\\/VTmTm4I6EpUesl34JyyuIVXH0XZC8CGGbM2rTtef8xmyqNnFSsr4\\\\\\\/D58HJ5LuRd0odqV+39DudnmNquV\\\\\\\/XwXTy59AXlzWMCpUXI+BRKuDZ5sMd8nOXshfCxAROwShANgZ6XQpxFaU0sz5zwMiai+BA6idN6hPV8PRAzEWxno058c4crMkoLxHQ1U3ZqyvL67iiQ42GbTJDE2XfL9XZsqO1kC2ypvKHwF+3\\\\\\\/wFXHgb+PidneibCoXVnJXofX3aX0cdQEEBoGZoX3CUbvx3W2gGQOguwZmSMfYDSQ+h4ZcRkfYX6E\\\\\\\/8Qq5AU8emU1YaBTmDE5Qxb2i0B55B5O6k4mnqN5tV\\\\\\\/rSZj6qVz2QFmpE8ZwBYbFhEZmEW1DxjuqjALe\\\\\\\/rP01cOoHOkAVc\\\\\\\/TU4QdYzF53Uwo7tpQNnYPy\\\\\\\/GX\\\\\\\/3jVEVk+0voB6Hj9v+ScQG54xYBe01\\\\\\\/WTLbpczQp8OadjbtBfDpOovjwjjK6F7l\\\\\\\/8aJM5CjaNFnLnJ0i0bnHWEnD3ObCYggHVBsSS5Wq3dMpbYSeOinR7byGQ5gv+jb7+uvi51ZtihX24Gax0g1f+S6RilWPMRpg6sNZvO+EWEfrA4G9A6cGqzWk\\\\\\\/UCoFBgOiFKS4hyBlewHGgR5Qr0yZi2aBuulS8wrlwFFRxHyi26kEVcB5T5pJD\\\\\\\/l7OsLjn3oG+nJGJkSDDBKI+QcZDBV1X7L6VubpAAv6yV0njezGciO8qXxZEGfj9BcWudFB1zWuywLfTi\\\\\\\/CB4sFeyOEAMeBcI+gdZ6wEyzEMtT7Fgi\\\\\\\/nb1IkyHyVMp2KX6EDqprkos4YSDjgUg4YwpaZKb9vLX3NhCYc64AC7cbcFiT671S2tf70t5OTC0IVHyGvP6ChDJR78NDSOvrhIQtDZaImeDku\\\\\\\/8hZOd0Bdx8osKjUkKqB54c0ywG\\\\\\\/Bv9X57DmL8CHi3sxnEA14VW3OIRl8LdpwB\\\\\\\/JFkL8LWf72LPv6KSW2DZ5yO6TY+4wENHn9Aix63+jk\\\\\\\/0FbzB+kwlUqfd80rUevWW+69C7\\\\\\\/lD0vlGMGtE6YXGBzTrUiQATG0IevOw5qDxIb5bi5NeTYPlwoynw7O8zmVoq1tAePgoquNbygLavSeTk0tXldKjLWRHzqYrifWBZhf3DlRxMM8R8MYw\\\\\\\/x353257fjdk+fmz6tSkBQAzAFqA2tRDTWu7KeT5Jl2NV7MX1DM6DD0elFNupVLQqgOilTDxdnvOaJwGHBxDf08jihBjxlcVOuxeAfb\\\\\\\/3A9cWsn+x4NhScLLuCwHDKU29bbLeVNwr4JZ9bGaC1MgE3Efy75zCe59HS42BJSfdOXDqZaYfteeEHnSGD2xb5trxfR\\\\\\\/OKFT4zQLXgesb7S5oG5tU\\\\\\\/4E9svOmFUX7buEHj7VGMcahmOOsUGveJxE\\\\\\\/mLEctA6DBmqUJzBhpAg1XOFtnLnBXqRzO9iEt7H9MphrcTv9NkYRdCauEH272ciu\\\\\\\/PTGghGOvk9lbsnY7QMB\\\\\\\/6tbS6dOECH4VpWqt4ePrn3lY2Iw16I\\\\\\\/zlloPIA9v\\\\\\\/Ke3qOb9EEGrV+iRK6XPXgtUQ0Ib785cJGiG79nIgGxmKOGfq5qMBJLlJeoF+8YtTclX37FSqxaqhEjgGlc0cDd95VPU\\\\\\\/v0n+2IftWOoJdUOmla4fBRM\\\\\\\/ry+SllXM+Uw5HnZeoL7C7VeMTXm2coPPS5qcasyvab7KPHtDal30NkJjTT7cdMR5ICYWqGT85nLkrBoMch3ibyvKUxSTGQlUOwde\\\\\\\/87FHthCSN6g7y2aFv3EwCSnSsPPTMEfWY23JsJwhC4NQuEK402Kuoz9FmGvkF7mrNkuUH0R+dbKuaKyFDMgf6mNlTXzIdDBHAgaAMDzAStFrCQyL99fEr7faqTfpHh2G\\\\\\\/ETHxzjGOzS\\\\\\\/Ye5AAvdUlReY6GHBSoyPeMP+1um6Xki7KrpE7mOvTFg2jBpIy+wHSgnKshlodrdt+yebzB0SmKEXAwJudoEDbEO+HmGTcRJ+\\\\\\\/4PbkBCIIi5ILuHglOw7hyiR7ohpVSl0tApvU8gaXoRivvGWKhoZzuUgBeO8lmrbMvawAeyxZhXd7NXFre0UG24Xy8C36BokBdmAzFA8+GyscLZCdKb7dQSdIC4PJ9Ktpm++bNSk1hnNng5ZnfVht4LiUhx0x0tZIygV7JA4d+xy6Ivsx5r8HlYTelfT\\\\\\\/6NVHP\\\\\\\/jdAnIQn5Bc4fJ3pFMTvigA1HkqTfwsPOl\\\\\\\/yoaZu3LkZ3gLI953FaZrJaD7j9KpKo8vadASv3ECHA9bnX8BJb4u6tI+gd0VI3tfxsYxCUZ8peJtNed\\\\\\\/kJ9F7Yf\\\\\\\/SM6vKmmhW\\\\\\\/EOqSF30blXZmJCdH2FxE0+IxNGK+4KYVUiG7IXFtDMXMsvhKJ+NOvMKwHQBVUr3u7RujWZDe2S+611gmVDnrbnOU2Xw\\\\\\\/6ZnmpVoDyEUi8I9LiFwks70qjLpJg9SChtOdOffp8EOHuLRi910moIloIc\\\\\\\/YgXk4u3wpWiiMISiKW5hdQAT\\\\\\\/c6jbGsNzIqomS6RIkgsqURJYC5ydeDeelFB6KUFtpm3k1iZR4VhShcZlow=\\\u0022,\\\u0022iv\\\u0022:\\\u0022k8nB5iT8\\\\\\\/eLK9KkYpWIO5g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002224a20a8a761b163e2b46f87bde07495908ea34607ad3e076a15d76cf2f60d5a0\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qrXbv3PH7bAe55AUIaWtGUFF\\\\\\\/GQoTfdhDwW4HtTfXsywh9jwBuAGKPFm7Bl7gaoTHKvwyZqz4m8T3yQPvQrVaJ1oJc7om6Kf1LQLIrVaHVTR83en1l1wUpwYHZyi+X6Rlh\\\\\\\/49CxOwvsACehuEFpBe+lUDCsCZy0WIQkjLqoBXHpwGU3qyWMjo2vpqrrIpjapLhvyFvx11++qFCT+6VsqtzARmVuCzHmfD7\\\\\\\/HTDaX4fUVZ7t7o1LcjFKWTKaTm1gmfLuxYcQaQg2fT7UTYRewljLfTyFpT6er2g1wLpDYGJ0AVDkr0KltHLN1jDma2aUpPQkBVTsdTFT0etVR2EuwUclq4i7MpxDm6IRVeGbl4mW\\\\\\\/VbGKrxEpTXZyWNXdMwA813jfkja0AyTL6Ipjv0xmRJaNk1oMN0jLAHw5zuoi1\\\\\\\/7kYCp7G7jka\\\\\\\/1LyYHEXFLSVAVChKCpapCizjLESdF\\\\\\\/C2\\\\\\\/lMBRfegbIyav0U152cyyGJZtYCx\\\\\\\/f8HPlEsIP369HlnXePhX\\\\\\\/AllqWo0sknPNEbNDUqVDNw7AtQe3h\\\\\\\/ps2dWp7qCQ4Cjkm9xpLrnsHXUGJukRcISpi3AO2kCAC\\\\\\\/R\\\\\\\/3cOkxxdjyjrfaCzOks0zQ3JkS\\\\\\\/CNSuK0uZcI8i5oHVyV9zXHMQR1U3XByERLIOim09+xfAfAa8LATzoqVx7po+qkYI6hVU\\\\\\\/nlSUmc1HI7kfol9w6G\\\\\\\/xb5a9ml\\\\\\\/bnsDIAP\\\\\\\/8Nyk+20S6vmf0X8eJfmHh1kxJateUlHNlucUOLwHQpo9oXw4w\\\\\\\/eVqjXShp864k2vpnTdWp9Z140uwK4EUqppgdj9easp8FscdPcJtsn9Fi\\\\\\\/dlWwztVWAS5m4JjGJmr4O5OIVwdldak4MoNiN5Vq90YLGD8Qd8evTRGdM2+PMkDtImdu9\\\\\\\/UPhFCvgrQ1962B6OMSuVUODolJGr078nMnjnpkadEdijuuJFAVk4eK13x15rNO7ybZ7syBxIT\\\\\\\/JjwrKzIjPBDW\\\\\\\/eIWhEpqHUIKssq+Djg\\\\\\\/Kcbx2jnWamqDU+gdM0FFkkx7I5pPYjJTeC5HL4wVKOXPoFyC7xKXKKQTQN+WzR1I2QPUmPUYXOop3aMNVWQvi13R913OykTgJt5UJwfPvTSQTFicW4guNHLChjO9PUBAlsKlE448hgPKhv8bSrtdFW374Xj+JugAI1\\\\\\\/3tCZ0zMBr\\\\\\\/wn87Bf9ENzy5U6ta1KmeBQ1BBM++YnZh1U8J2A2e42mBZ6Oof5dkLWVX+WGldr2ZA7ZCX3wwcJu33uHBtqRQoBlNZudqyvopzqZOO2BRzQT5t5KBguVRiitCy5ZZid+v6iHICcGkRwywEBGwEQsyRMiMccjcszVRxBBDFbHNBIz7cTnik4Xzm0+Fn0z3qsjafi9u2YAPkSP6h1U6KTDhFa\\\\\\\/3Iif81fot1lBtXNWulWGmaezJ565hV5\\\\\\\/o8dZOGesYRbyL2FNLZvepucR\\\\\\\/GZQTAPVFbJQJGMG60rGLj2HReNdbjMSj9nvG5cvgNgGHWYm6w5xZJwynhHpyjjxJxcfVyg0WMU8aXVxypABsQpkNFo8akLY5S7wRyGbIfM9F600WNaVzB2Wdpzj0McR46ilSXLNwONjm8AiRSZsrOR60dtlx8kWOrUwnQ1O6sBMLur+ilbhx\\\\\\\/Y8HH\\\\\\\/Dmkjq0wjFhf7Dut1jNYUZ0VDRBTVWmqL9E3DaxURont2qEnZu7MNir6hPNrGiXzWqUWHTpt\\\\\\\/afD\\\\\\\/zzcLG05RffnTM\\\\\\\/b0QVDpQZS7Rxv3lvbWT3IYfAgFGbLfmfwPpQ\\\\\\\/hCHdEqrG\\\\\\\/rdzrVattBPD+hGksCofzEQMpotbOGu65Ztf1V5Q9ycb3eX02lWKhZiLzUa0L4sJ1G5vIkR9ig1tc1iq7eiZRF6+LbkK\\\\\\\/NXfvYpKWekag1PEn8RbPeAoA87TlM2NQTZChbcJAR9J8ek127UgTSTrDg7q+MuW+vCSvgEmn7tcseMV9yuixch\\\\\\\/1SuHsV9g1oLDlj82ByOI\\\\\\\/EfSKz7HZuIXW4ntKjX\\\\\\\/g\\\\\\\/kf8+Hd7lgBYsoTG18gQntCT7I1aZoA=\\\u0022,\\\u0022iv\\\u0022:\\\u0022cg14CANHoA4asLeyVD4NQg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220cdd5e65e54351463b24c0da737ea3233ee71f69b3a5de1c958b6c341f4bcb4a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495848 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/907" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vNBHtZyMEJA8ejIDEutvrB_cdKC7ROzgQkDeQo1ChIc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/907" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-m6ASHb--zG0knD2KIVSXnl-g869j1j-wVnPbeaaHx4o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/899" title="Improved Genome Assembly and Annotation of the Soybean Aphid (Aphis glycines Matsumura)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/917" title="Mutational Analysis of N-Ethyl-N-Nitrosourea (ENU) in the Fission Yeast Schizosaccharomyces pombe" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495848" id="node495848" data-pisa="ggg;10/3/907" data-pisa-master="ggg;g3.119.400725" data-apath="/ggg/10/3/907.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Sequence of the Euryhaline Javafish Medaka, <em>Oryzias javanicus</em>: A Small Aquarium Fish Model for Studies on Adaptation to Salinity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yusuke Takehana</span>, <span class="highwire-citation-author" data-delta="1">Margot Zahm</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-8560-4003" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cédric Cabau</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7126-5477" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christophe Klopp</span>, <span class="highwire-citation-author" data-delta="4">Céline Roques</span>, <span class="highwire-citation-author" data-delta="5">Olivier Bouchez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-5041-6843" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cécile Donnadieu</span>, <span class="highwire-citation-author" data-delta="7">Celia Barrachina</span>, <span class="highwire-citation-author" data-delta="8">Laurent Journot</span>, <span class="highwire-citation-author" data-delta="9">Mari Kawaguchi</span>, <span class="highwire-citation-author" data-delta="10">Shigeki Yasumasu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-2683-0160" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Satoshi Ansai</span>, <span class="highwire-citation-author" data-delta="12">Kiyoshi Naruse</span>, <span class="highwire-citation-author" data-delta="13">Koji Inoue</span>, <span class="highwire-citation-author" data-delta="14">Chuya Shinzato</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15"><a href="http://orcid.org/0000-0001-9882-5948" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Manfred Schartl</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yann Guiguen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="17"><a href="http://orcid.org/0000-0002-0630-4027" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amaury Herpin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">907-915;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400725 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495848" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yusuke Takehana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Bio-Science, Faculty of Bio-Science, Nagahama Institute of Bioscience and Technology, 1266 Tamura, Nagahama 526-0829, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yusuke%2BTakehana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takehana%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYusuke%2BTakehana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yusuke Takehana</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Margot Zahm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plate-forme bio-informatique Genotoul, Mathématiques et Informatique Appliquées de Toulouse, INRAE, Castanet Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Margot%2BZahm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zahm%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMargot%2BZahm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cédric Cabau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>SIGENAE, GenPhySE, Université de Toulouse, INRAE, ENVT, Castanet Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9dric%2BCabau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabau%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC%25C3%25A9dric%2BCabau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8560-4003" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cédric Cabau</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christophe Klopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plate-forme bio-informatique Genotoul, Mathématiques et Informatique Appliquées de Toulouse, INRAE, Castanet Tolosan, France,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>SIGENAE, GenPhySE, Université de Toulouse, INRAE, ENVT, Castanet Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2BKlopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Klopp%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristophe%2BKlopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7126-5477" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christophe Klopp</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Céline Roques </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9line%2BRoques%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roques%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC%25C3%25A9line%2BRoques%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Olivier Bouchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BBouchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouchez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlivier%2BBouchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Cécile Donnadieu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9cile%2BDonnadieu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Donnadieu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC%25C3%25A9cile%2BDonnadieu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5041-6843" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cécile Donnadieu</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Celia Barrachina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>MGX, Biocampus Montpellier, CNRS, INSERM, University of Montpellier, Montpellier, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celia%2BBarrachina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barrachina%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACelia%2BBarrachina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Laurent Journot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>MGX, Biocampus Montpellier, CNRS, INSERM, University of Montpellier, Montpellier, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurent%2BJournot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Journot%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaurent%2BJournot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mari Kawaguchi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Materials and Life Sciences, Faculty of Science and Technology, Sophia University, 7-1 Kioi-cho Chiyoda-ku, Tokyo 102-8554, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mari%2BKawaguchi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kawaguchi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMari%2BKawaguchi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Shigeki Yasumasu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Materials and Life Sciences, Faculty of Science and Technology, Sophia University, 7-1 Kioi-cho Chiyoda-ku, Tokyo 102-8554, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shigeki%2BYasumasu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yasumasu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShigeki%2BYasumasu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Satoshi Ansai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Laboratory of Bioresources, National Institute for Basic Biology, 38 Nishigonaka, Myodaiji-cho, Okazaki 444-8585, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satoshi%2BAnsai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ansai%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASatoshi%2BAnsai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2683-0160" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Satoshi Ansai</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Kiyoshi Naruse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Laboratory of Bioresources, National Institute for Basic Biology, 38 Nishigonaka, Myodaiji-cho, Okazaki 444-8585, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiyoshi%2BNaruse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naruse%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiyoshi%2BNaruse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Koji Inoue </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Atmosphere and Ocean Research Institute, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8564, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koji%2BInoue%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Inoue%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKoji%2BInoue%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Chuya Shinzato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Atmosphere and Ocean Research Institute, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8564, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chuya%2BShinzato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shinzato%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChuya%2BShinzato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Manfred Schartl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>University of Wuerzburg, Developmental Biochemistry, Biocenter, 97074 Wuerzburg, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Comprehensive Cancer Center Mainfranken, University Hospital, 97080 Wuerzburg, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>‡‡‡</span>Hagler Institute for Advanced Study and Department of Biology, Texas A&M University, College Station, Texas 77843, USA,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manfred%2BSchartl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schartl%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AManfred%2BSchartl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9882-5948" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Manfred Schartl</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Yann Guiguen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>INRAE, UR 1037 Fish Physiology and Genomics, F-35000 Rennes, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yann%2BGuiguen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guiguen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYann%2BGuiguen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yann Guiguen</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Amaury Herpin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>INRAE, UR 1037 Fish Physiology and Genomics, F-35000 Rennes, France, and</div><div class='author-affiliation'><span class='nlm-sup'>****</span>State Key Laboratory of Developmental Biology of Freshwater Fish, College of Life Sciences, Hunan Normal University, Changsha, 410081, Hunan, P. R. of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amaury%2BHerpin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Herpin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmaury%2BHerpin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0630-4027" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amaury Herpin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:amaury.herpin@inra.fr" class="" data-icon-position="" data-hide-link-title="0">amaury.herpin@inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/3/907" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495848" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495848/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/907.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495848" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495848/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/907.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495848" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495848/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/3/907.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495848" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495848/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The genus <em>Oryzias</em> consists of 35 medaka-fish species each exhibiting various ecological, morphological and physiological peculiarities and adaptations. Beyond of being a comprehensive phylogenetic group for studying intra-genus evolution of several traits like sex determination, behavior, morphology or adaptation through comparative genomic approaches, all medaka species share many advantages of experimental model organisms including small size and short generation time, transparent embryos and genome editing tools for reverse and forward genetic studies. The Java medaka, <em>Oryzias javanicus</em>, is one of the two species of medaka perfectly adapted for living in brackish/sea-waters. Being an important component of the mangrove ecosystem, <em>O. javanicus</em> is also used as a valuable marine test-fish for ecotoxicology studies. Here, we sequenced and assembled the whole genome of <em>O. javanicus</em>, and anticipate this resource will be catalytic for a wide range of comparative genomic, phylogenetic and functional studies. Complementary sequencing approaches including long-read technology and data integration with a genetic map allowed the final assembly of 908 Mbp of the <em>O. javanicus</em> genome. Further analyses estimate that the <em>O. javanicus</em> genome contains 33% of repeat sequences and has a heterozygosity of 0.96%. The achieved draft assembly contains 525 scaffolds with a total length of 809.7 Mbp, a N50 of 6,3 Mbp and a L50 of 37 scaffolds. We identified 21454 predicted transcripts for a total transcriptome size of 57, 146, 583 bps. We provide here a high-quality chromosome scale draft genome assembly of the euryhaline Javafish medaka (321 scaffolds anchored on 24 chromosomes (representing 97.7% of the total bases)), and give emphasis on the evolutionary adaptation to salinity.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/medaka" class="hw-term hw-article-keyword hw-article-keyword-medaka" rel="nofollow">Medaka</a></li><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">evolution</a></li><li class="kwd"><a href="/keyword/whole-genome-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequencing" rel="nofollow">whole genome sequencing</a></li><li class="kwd"><a href="/keyword/long-reads-0" class="hw-term hw-article-keyword hw-article-keyword-long-reads" rel="nofollow">long reads</a></li><li class="kwd"><a href="/keyword/genetic-map" class="hw-term hw-article-keyword hw-article-keyword-genetic-map" rel="nofollow">genetic map</a></li><li class="kwd"><a href="/keyword/transcriptome" class="hw-term hw-article-keyword hw-article-keyword-transcriptome" rel="nofollow">transcriptome</a></li><li class="kwd"><a href="/keyword/adaptation" class="hw-term hw-article-keyword hw-article-keyword-adaptation" rel="nofollow">adaptation</a></li><li class="kwd"><a href="/keyword/salinity" class="hw-term hw-article-keyword hw-article-keyword-salinity" rel="nofollow">salinity</a></li></ul><p id="p-4">Medaka fishes belong to the genus <em>Oryzias</em> and are an emerging model system for studying the molecular basis of vertebrate evolution. This genus contains approximately 35 species, individually exhibiting numerous morphological, ecological and physiological differences and specificities (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Inoue and Takei 2002</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2003</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Parenti 2008</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mokodongan and Yamahira 2015</a>). In addition, they all share many advantages of experimental model organisms, such as their small size, easy breeding, short generation time, transparent embryos, transgenic technology and genome-editing tools, with the “flag ship” species of this genus, the Japanese rice fish, <em>Oryzias latipes</em> (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wittbrodt <em>et al.</em> 2002</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kirchmaier <em>et al.</em> 2015</a>). Such phenotypic variations, together with cutting edge molecular genetic tools make it possible to identify major loci that contribute to evolutionary differences, and to dissect the roles of individual genes and regulatory elements by functional tests. For example, a recent genetic mapping approach using interspecific hybrids identified the major chromosome regions that underlie the different hyperosmotic tolerance between species of the <em>Oryzias</em> genus (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Myosho <em>et al.</em> 2018</a>). Medaka fishes are also excellent models to study evolution of sex chromosomes and sex-determining loci among species (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Takehana <em>et al.</em> 2007a</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">2007b</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Tanaka <em>et al.</em> 2007</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Herpin and Schartl 2009</a>), with the advantage of being also suitable models for providing functional evidences for these novel sex-determining genes by gain-of-function and/or loss-of-function experiments (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Myosho <em>et al.</em> 2012</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Takehana <em>et al.</em> 2014</a>).</p><p id="p-5">Among these species, the Java medaka, <em>Oryzias javanicus</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), is unique as being the prototypic species of this genus with respect to adaptation to seawater. Previous phylogenetic studies divided the genus <em>Oryzias</em> into three monopyletic groups: <em>(i) javanicus</em>, <em>(ii) latipes</em> and <em>(iii) celebensis</em> species groups (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Takehana <em>et al.</em> 2005</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Mokodongan and Yamahira 2015</a>). Most of the <em>Oryzias</em> species inhabit mainly freshwater biotopes, while only two species belonging to the <em>javanicus</em> group live in sea- or brackish water. One is <em>O. javanicus</em>, found in mangrove swamps from Thailand to Indonesia, and the other is <em>O. dancena</em> (previously named <em>O. melastigma</em>) living both in sea- and freshwaters from India to Malaysia. Although both species are highly adaptable to seawater, <em>O. javanicus</em> prefers hyperosmotic conditions while <em>O. dancena</em> favors hypoosmotic conditions at the west coast of Malaysian peninsula where their distribution ranges overlap (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Yusof <em>et al.</em> 2012</a>). In addition, <em>O. javanicus</em> is an important component of the mangrove ecosystem (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zulkifli <em>et al.</em> 2012</a>), and has been used as a valuable marine test fish in several ecotoxicology studies (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Koyama <em>et al.</em> 2008</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Horie <em>et al.</em> 2018</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/907/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A couple of Java medakas, Oryzias javanicus. Picture from K. Naruse, NBRP Medaka stock center (https://shigen.nig.ac.jp/medaka/top/top.jsp)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943044791" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A couple of Java medakas, Oryzias javanicus. Picture from K. Naruse, NBRP Medaka stock center (https://shigen.nig.ac.jp/medaka/top/top.jsp).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/907/F1.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/3/907/F1.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/907/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/907/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495919" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">A couple of Java medakas, Oryzias javanicus. Picture from K. Naruse, NBRP Medaka stock center (<a href="https://shigen.nig.ac.jp/medaka/top/top.jsp">https://shigen.nig.ac.jp/medaka/top/top.jsp</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">In this study, we sequenced and assembled the whole genome of <em>O. javanicus</em>, a model fish species for studying molecular mechanisms of seawater adaptation. In teleost fish, the major osmoregulatory organs <em>i.e.</em>, gills, intestine and kidney, play different roles for maintaining body fluid homeostasis. Many genes encoding hormones, receptors, osmolytes, transporters, channels and cellular junction proteins are potentially involved in this osmotic regulation. In addition to osmoregulation, hatching enzyme activity dramatically fluctuates and adjusts at different salt conditions. At hatching stage, fish embryos secrete a specific cocktail of enzymes in order to dissolve the egg envelope, or chorion. In the medaka <em>O. latipes</em>, digestion of the chorion occurs through the cooperative action of two kinds of hatching enzymes, <strong><em>(i)</em></strong> the high choriolytic enzyme (HCE) and <strong><em>(ii)</em></strong> the low choriolytic enzyme (LCE) (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Yasumasu <em>et al.</em> 2010</a>). The HCE displays a higher activity in fresh- than in brackish waters (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kawaguchi <em>et al.</em> 2013</a>). Thus, availability of a high-quality reference genome in <em>O. javanicus</em> would facilitate further research for investigating the molecular basis of physiological differences, including the osmotic regulation and the hatching enzyme activity, among <em>Oryzias</em> species.</p><div class="section methods-materials" id="sec-1"><h2 class="">Methods and Materials</h2><div id="sec-2" class="subsection"><h3>Animal samplings</h3><p id="p-8">The wild stock of <em>O. javanicus</em> used in this study was supplied by the National Bio-Resource Project (NBRP) medaka in Japan. This stock (strain ID: RS831) was originally collected at Penang, Malaysia, and maintained in synthetic seawater (ca 3% of NaCl equivalent; although using half seawater is also possible) in aquaria under an artificial photoperiod of 14 hr light:10 hr darkness at 27 ± 2°. Genomic DNA was extracted from the whole body of a female (having ZW sex chromosome) using a conventional phenol/chloroform method, and was subjected to PacBio and 10X Genomics sequencings. For RNA-sequencing, total RNAs were extracted from nine female tissues (brain, bone, gill, heart, intestine, kidney, liver, muscle and ovary), and one male tissue (testis) using the RNeasy Mini Kit (Qiagen). For genetic mapping, we used a DNA panel consisting of 96 F1 progeny with their parents (originally described in a previous study (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Takehana <em>et al.</em> 2008</a>)). Phenotypic sex was determined by secondary sex characteristics of adult fish (six-month-old and sexually mature fish), namely, the shapes of dorsal and anal fins. All animal experiments performed in this study complied with the guideline of National Institute for Basic Biology, and have been approved by the Institutional Animal Care and Use Committee of National Institute of Natural Science (16A050 and 17A048).</p><div id="sec-3" class="subsection"><h4>PacBio genome sequencing:</h4><p id="p-9">Library construction and sequencing were performed according to the manufacturer’s instructions (Shared protocol-20kb Template Preparation Using BluePippin Size Selection system (15kb size Cutoff)). When required, DNA was quantified using the Qubit dsDNA HS Assay Kit (Life Technologies). DNA purity was assessed by spectrophotometry using the nanodrop instrument (Thermofisher), and size distribution and absence of degradation were monitored using the Fragment analyzer (AATI) (8–11). Purification steps were performed using 0.45X AMPure PB beads (PacBio). 80µg of DNA was purified and then sheared at 40kb using the megaruptor system (diagenode). DNA and END damage repair step was further performed for 5 libraries using the SMRTBell template Prep Kit 1.0 (PacBio). Blunt hairpin adapters were then ligated to the libraries. Libraries were subsequently treated with an exonuclease cocktail in order to digest unligated DNA fragments. Finally, a size selection step using a 15kb cutoff was performed on the BluePippin Size Selection system (Sage Science) using 0.75% agarose cassettes, Marker S1 high Pass 15-20kb. Conditioned sequencing primer V2 was annealed to the size-selected SMRTbell. The annealed libraries were then bound to the P6-C4 polymerase using a ratio of polymerase to SMRTbell set at 10:1. After performing a magnetic bead-loading step (OCPW), SMRTbell libraries were sequenced on 48 SMRTcells (RSII instrument at 0.25nM with a 360-min movie resulting in a total of 61.8Gb of sequence data (1.28Gb/SMRTcell).</p></div><div id="sec-4" class="subsection"><h4>10X Genomics genome sequencing:</h4><p id="p-10">Chromium library was prepared according to 10X Genomics’ protocol using the Genome Reagent Kits v1. Sample quantity and quality controls were further validated on Qubit, Nanodrop and Femto. Optimal performance has been characterized on input gDNA with a mean length greater than 50 kb. The library was prepared using 3 µg of high molecular weight (HMW) gDNA (cut off at 50kb using BluePippin system). In details, for the microfluidic Genome Chip, a library of Genome Gel Beads was combined with HMW template gDNA in Master Mix and partitioning oil in order to create Gel Bead-In-EMulsions (GEMs) in the Chromium. Each Gel Bead was functionalized with millions of copies of a 10x Barcoded primer. Upon dissolution of the Genome Gel Bead in the GEM, primers containing (<em>i</em>) an Illumina R1 sequence (Read 1 sequencing primer), (<em>ii</em>) a 16 bp 10x Barcode, and (<em>iii</em>) a 6 bp random primer sequence were released. Read 1 sequence and the 10x Barcode were added to the molecules during the GEM incubation. P5 and P7 primers, Read 2, and Sample Index were added during library construction. 8 cycles of PCR were performed for amplifying the library. Library quality was assessed using a Fragment analyzer. Finally, the library was sequenced on an Illumina HiSeq3000 using a paired-end read length of 2x150 pb with the Illumina HiSeq3000 sequencing kits resulting in 101.6Gb of raw sequence data.</p></div><div id="sec-5" class="subsection"><h4>Genome assembly and annotation:</h4><p id="p-11">PacBio reads were corrected and trimmed using Canu v1.5 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Koren <em>et al.</em> 2017</a>). Contigs were then assembled using SMARTdenovo version of May 2017 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ruan 2019</a>). The draft assembly produced contains 729 contigs with a total genome size of 807.5 Mbp, an N50 of 3,9 Mbp and a L50 of 59 contigs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). To improve the assembly base pair quality two polishing steps were run. First, BLASR aligned PacBio reads were processed with Quiver from the Pacific Biosciences SMRT link software v.4.0.0. Second, 10X reads were realigned to the genome using Long Ranger v2.1.1 and the alignment file was processed with Pilon v1.22 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Walker <em>et al.</em> 2014</a>). Third, the same 10X reads were aligned to the genome with BWA-MEM v0.7.12-r1039 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li 2013</a>) and the alignment file was processed with ARCS v1.0.1 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Yeo <em>et al.</em> 2018</a>) to scaffold the genome. Both tools were run with default parameters. For genome annotation, the MAKER3 pipeline was employed ((<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Holt and Yandell 2011</a>); Maker 3.01.02-beta in mpi mode to merge data from gene models and cDNA/protein evidences). Maker has been running with entries est_gff, protein_gff and pred_gff in run_evm = 1, est2genome = 0 and protein2genome = 0 mode. No AED cut-off was applied but AED scores have been used to select the best supported transcript for each gene.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/907/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Oryzias javanicus assembly pipeline. Sequencing data are represented by colored rectangles with waved bases. Tools used are in gray rectangles. Assembly metrics are in gray and white rectangles. This pipeline is divided in stages symbolized by the frame." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943044791" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Oryzias javanicus assembly pipeline. Sequencing data are represented by colored rectangles with waved bases. Tools used are in gray rectangles. Assembly metrics are in gray and white rectangles. This pipeline is divided in stages symbolized by the frame.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/907/F2.medium.gif" width="440" height="293"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/3/907/F2.medium.gif" width="440" height="293"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/907/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/907/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495943" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-12" class="first-child">Oryzias javanicus assembly pipeline. Sequencing data are represented by colored rectangles with waved bases. Tools used are in gray rectangles. Assembly metrics are in gray and white rectangles. This pipeline is divided in stages symbolized by the frame.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h4>Transcriptome RNA-seq sequencing and assembly:</h4><p id="p-13">RNA-seq libraries were prepared according to Illumina’s protocols using the Illumina TruSeq Stranded mRNA sample prep kit. Briefly, mRNAs were selected using poly-T beads, reverse-transcribed and fragmented. The resulting cDNAs were then subjected to adaptor ligation. 10 cycles of PCR were performed for amplifying the libraries. Quality of the libraries was assessed using a Fragment Analyzer. Quantification was performed by qPCR using the Kapa Library Quantification Kit. RNA-seq libraries were sequenced on an Illumina HiSeq3000 using a paired-end read length of 2x150 pb with the Illumina HiSeq3000 sequencing kits resulting in 95Gb of sequence data (28.9M reads pairs/library). The read quality of the RNA-seq libraries was evaluated using FastQC (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andrew S. 2010</a>). <em>De novo</em> and reference-based transcriptome assemblies were produced. Reads were cleaned, filtered and <em>de novo</em> assembled using the DRAP pipeline v1.91 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cabau <em>et al.</em> 2017</a>) with the Oases assembler (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Schulz <em>et al.</em> 2012</a>). Assembled contigs were filtered in order to keep only those with at least one fragment per kilobase of transcript per million reads (FPKM). In the reference-based approach, all clean reads were mapped to the chromosomal assembly using STAR v2.5.1b (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dobin <em>et al.</em> 2013</a>) with outWigType and outWigStrand options to output signal wiggle files. Cufflinks v2.2.1 (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Trapnell <em>et al.</em> 2010</a>) was used to assemble the transcriptome. All tissues have been <em>de novo</em> assembled separately and the 1 FPKM cut-off has been set on each library. The min, mean, median and max number of transcripts after the cufflinks assembly of each tissue are respectively 26223, 44992, 47804 and 59234.</p></div><div id="sec-7" class="subsection"><h4>RAD-library construction:</h4><p id="p-14">RAD-seq library was built following the Baird <em>et al.</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baird <em>et al.</em> 2008</a>) protocol with minor modifications. Briefly, between 400 to 500 ng of gDNA per fish were digested with SbfI-HF enzyme (R3642S, NEB). Digested DNA was purified using AMPure PX magnetic beads (Beckman Coulters) and ligated to indexed P1 adapters (1 index per sample) using concentrated T4 DNA ligase (M0202T, NEB). After quantification (Qubit dsDNA HS assay kit, Thermofisher) all samples were pooled in equal amounts. The pool was then fragmented on a S220 sonicator (Covaris) and purified with Minelute column (Qiagen). Finally, the sonicated DNA was size selected (250 to 450 bps) on a Pippin HT (Sage science) using a 2% agarose cassette, repaired using the End-It DNA-end repair kit (Tebu Bio) and adenylated at its 3′ ends using Klenow (exo-) (Tebu-Bio). P2 adapters were then ligated using concentrated T4 DNA ligase, and 50 ng of the ligation product were engaged in a 12 cycles PCR for amplification. After AMPure XP beads purification, the resulting library was checked on a Fragment Analyzer (Agilent) using the HS NGS kit (DNF-474-33) and quantified by qPCR using the KAPA Library Quantification Kit (Roche, ref. KK4824). Ultimately the whole library was denatured, diluted to 10 pM, clustered and sequenced using the rapid mode v2 SR100nt lane of a Hiseq2500 device (Illumina).</p></div></div><div id="sec-8" class="subsection"><h3>Identification of genes associated with osmoregulation</h3><p id="p-15">We listed known <em>O. latipes</em> genes that encode proteins associated with osmoregulation (<em>e.g.</em>, hormones, pumps, transporters, channels, osmolytes-related and cellular junction proteins) based on literatures, and used HMMER version 3.1b2 (<a href="http://hmmer.org/">http://hmmer.org/</a>) to identify specific Pfam domains (Pfam 32, (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">El-Gebali <em>et al.</em> 2019</a>) included in these proteins (Supplemental Table S1). Using our gene model of <em>O. javanicus</em> together with Ensembl gene models of the <em>O. dancena</em> and <em>O. latipes</em> species complex (Hd-rR, HNI-II and HSOK), we counted the number of proteins containing the Pfam domains in each species (Supplemental Table S2).</p><div id="sec-9" class="subsection"><h4>Salt dependency of OjHCE:</h4><p id="p-16">The mature enzyme regions of OjHCE3 was amplified from their full-length cDNA using primers designed to contain suitable restriction enzyme sites (BamHI and NdeI) at the 5′ region. After digestion with BamHI and NdeI, the fragments were inserted into pET3c vector. The plasmid was transformed into <em>E. coli</em> BL21 (DE3) pLysE strain cells. The cells were cultivated, and recombinant protein was harvested as inclusion body as described in (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Kawaguchi <em>et al.</em> 2013</a>). After the inclusion body was dissolved in denaturing buffer (50 mM Tris-HCl (pH 8.0), 8 M urea, 0.1 M 2-mercaptoethanol and 1 mM EDTA), the recombinant protein was refolded as described in (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Kawaguchi <em>et al.</em> 2013</a>). The egg envelope digestion activity of OjHCE was determined by turbidimetric methods (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Yamagami 1973</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Kawaguchi <em>et al.</em> 2013</a>). The isolated egg envelopes of medaka <em>O. latipes</em> were minced into fine fragments, and suspended in distilled water (DW). The suspension was allowed to stand overnight to remove rough fragments, and the supernatant containing fine fragments was used as substrate. The enzyme reaction was carried out in 400 μl of a reaction mixture containing 50 mmol/l Tris-HCl (pH 8.0), 0−0.75 mol/l NaCl, egg envelope suspension and recombinant HCE. The initial turbidity at 610 nm (T610) of the mixture was adjusted to approximately 55% when that of DW was 100%. Increment in transmission caused by the digestion of the fragmented envelopes was monitored for 3 min. The relative enzyme activity was expressed as the percentage of the highest activity under various salt concentrations.</p></div></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-17">All genome and transcriptome information was deposited under the NCBI Bioproject number PRJNA505405. The Illumina sequencing data for the RAD-tag genetic map were deposited in the Sequence Read Archive at NCBI with accession numbers SRX5326271 to SRX5326366. The genomic PacBio sequencing data were deposited in the Sequence Read Archive at NCBI with accession numbers SRX5274121 to SRX5274138 and SRX5274139 to SRX5274169. The 10X genomics Illumina sequencing data were deposited in the Sequence Read Archive at NCBI with accession number SRX5274139. The transcriptome Illumina sequencing data were deposited in the Sequence Read Archive at NCBI with accession numbers SRX5017469 to SRX5017479. The final chromosome assembly and genome annotation were deposited in GenBank at NCBI RWID00000000.1. Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10310498">https://doi.org/10.25387/g3.10310498</a>.</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>Genome Characteristics</h3><p id="p-18">To estimate size and other genome characteristics, 10X reads were processed with Jellyfish v1.1.11 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Marçais and Kingsford 2011</a>) to produce 21-mer distribution. The k-mer histogram was uploaded to GenomeScope (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Vurture <em>et al.</em> 2017</a>) with the max k-mer coverage parameter set to 10,000. Genome size was estimated around 908 Mbp, which is slightly higher than the 850 Mbp (0.87pg) estimated size reported on the Animal Genome Size Database (“Animal Genome Size Database:: Home”). Furthermore, this analysis estimates that the <em>O. javanicus</em> genome contains 33% of repeat sequences (around 303 Mbp) and has a heterozygosity of 0.96% (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495888/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495888/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">GenomeScope outputs on <em>O. javanicus</em> genome statistics</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Genome assembly</h3><p id="p-19">Draft assembly contains 525 scaffolds with a total length of 809.7 Mbp, a N50 of 6,3 Mbp and a L50 of 37 scaffolds. This represents 89.1% of the k-mer estimated genome size. Given the high percentage of repeats in the <em>O. javanicus</em> genome (33%), it is possible that the PacBio assembly did not totally succeed in completing all repeated regions. The genome completeness was estimated using Benchmarking Universal Single-Copy Orthologs (BUSCO) v3.0 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Simão <em>et al.</em> 2015</a>) based on 4,584 BUSCO orthologs derived from the Actinopterygii lineage leading to BUSCO scores of 4,327 (94.4%) complete BUSCOs, 176 (3.8%) fragmented BUSCOs and 81 (1.8%) missing BUSCOs.</p></div><div id="sec-14" class="subsection"><h3>Integration with the genetic map</h3><p id="p-20">RAD reads were trimmed by Trim Galore 0.4.3 (“Trim Galore”) with Cutadapt 1.12 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Martin 2011</a>) and then mapped to the assembled scaffolds using BWA-MEM v0.7.17 (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Li 2013</a>). Uniquely mapped reads were extracted from the read alignments, and then called variant bases using uniquely mapped reads by samtools <em>mpileup</em> and bcftools <em>call</em> (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li 2011</a>). Indels and variants with a low genotyping quality (GQ &lt; 20), a low read depth (DP &lt; 5), a low frequency of the minor allele (&lt; 5%), more than four alleles in the family, no more than 5% individuals missing were removed by vcftools v0.1.15 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Danecek <em>et al.</em> 2011</a>). After quality filtering, 6,375 variant sites were kept for the following analysis. Linkage map was constructed using this genotype information using Lep-MAP3 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rastas 2017</a>). Briefly, the filtered vcf file was loaded and the markers removed with high segregation distortion (<em>Filtering2</em>: dataTolerance = 0.001). Markers were then separated into 24 linkage groups with a LOD score threshold set at 9 and a fixed recombination fraction of 0.08 (<em>SeparateChromosomes2</em>: lodlimit = 9 and theta = 0.08). Two linkage groups were then excluded because of their small numbers of contained markers (less than 10). Classification of the markers was determined after maximum likelihood score indexing with 100 iterations (<em>OrderMarkers2</em>: numMergeIterations = 100) in each linkage group. The final map had 5,738 markers dispatched among 24 linkage groups spanning a total genetic distance of 1,221 cM.</p><p id="p-21">The linkage map exhibited discrepancies between genomic scaffolds and genetic markers. Among 525 genomics scaffolds, 32 were linked to more than one linkage group. To split chimeric scaffolds with a higher precision and to rebuild chromosomes with a higher fidelity, we used a cross-species synteny map between the Java medaka (<em>O. javanicus</em>) scaffolds and the medaka <em>(O. latipes</em>) chromosomes in order to combine marker locations from genetic and synteny maps. To build the synteny map, medaka cDNAs were aligned to the Java medaka scaffolds using BLAT v36 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kent 2002</a>), and a list of pairwise correspondence of gene positions on Java medaka scaffolds and medaka chromosomes was established. 13,796 markers were added to the 5,738 markers of the genetic map. Java medaka chromosomes were then reconstructed using ALLMAPS from the JCVI utility libraries v0.5.7 (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Tang <em>et al.</em> 2015</a>). This package was used to combine genetic and synteny maps, to split chimeric scaffolds, to anchor, order and orient genomic scaffolds. The resulting chromosomal assembly consists of 321 scaffolds anchored on 24 chromosomes (97.7% of the total bases) and 231 unplaced scaffolds</p></div><div id="sec-15" class="subsection"><h3>Annotation results</h3><p id="p-22">The first annotation step was identifying repetitive DNA content using RepeatMasker v4.0.7 (“RepeatMasker Home Page”), Dust (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Morgulis <em>et al.</em> 2006</a>) and TRF v4.09 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benson 1999</a>). A species-specific <em>de novo</em> repeat library was built with RepeatModeler v1.0.11 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Smit and Hubley 2010</a>). Repeated regions were located using RepeatMasker with the <em>de novo</em> and the Zebrafish (<em>Danio rerio</em>) libraries. Bedtools v2.26.0 (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Quinlan and Hall 2010</a>) was used to merge repeated regions identified with the three tools and to soft mask the genome. Repeats were estimated to account for 43.16% (349 Mbp) of our chromosomal assembly. The MAKER3 genome annotation pipeline v3.01.02-beta (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Holt and Yandell 2011</a>) combined annotations and evidences from three approaches: similarity with known fish proteins, assembled transcripts and <em>de novo</em> gene predictions. Protein sequences from 11 other fish species (<em>Astyanax mexicanus</em>, <em>Danio rerio</em>, <em>Gadus morhua</em>, <em>Gasterosteus aculeatus</em>, <em>Lepisosteus oculatus</em>, <em>Oreochromis niloticus</em>, <em>Oryzias latipes</em>, <em>Poecilia formosa</em>, <em>Takifugu rubripes</em>, <em>Tetraodon nigroviridis</em>, <em>Xiphophorus maculatus</em>) found in Ensembl were aligned to the masked genome using Exonerate v2.4 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Slater and Birney 2005</a>). Previously assembled transcripts were used as RNA-seq evidence. A <em>de novo</em> gene model was built using Braker v2.0.4 (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hoff <em>et al.</em> 2016</a>) with wiggle files provided by STAR as hints file for training GeneMark and Augustus. The best supported transcript for each gene was chosen using the quality metric Annotation Edit Distance (AED) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Eilbeck <em>et al.</em> 2009</a>). The genome annotation gene completeness was assessed by BUSCO using the Actinopterygii group (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Finally, the predicted genes were subjected to similarity searches against the NCBI NR database using Diamond v0.9.22 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Buchfink <em>et al.</em> 2015</a>). The top hit with a coverage over 70% and identity over 80% was retained.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495980/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495980/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Java medaka assembly and annotation statistics</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Mitochondrial genome and annotation</h3><p id="p-23">The previously sequenced <em>Oryzias javanicus</em> mitochondrial genome (NC_012981) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Setiamarga <em>et al.</em> 2009</a>) was aligned to the chromosomal assembly using Blat. All hits were supported by a single scaffold. This scaffold was removed from the assembly, circularised and annotated using MITOS (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernt <em>et al.</em> 2013</a>). This new <em>Oryzias javanicus</em> mitochondrial genome is 16,789 bp long and encodes 13 genes, 2 rRNAs and 19 tRNAs.</p><div id="sec-17" class="subsection"><h4>Phylogenetic relationship:</h4><p id="p-24">To precisely determine the phylogenetic position of <em>O. javanicus</em> within the genus <em>Oryzias</em>, we estimated the phylogenetic relationship using published whole genome datasets as references. Reference assemblies and annotations of <em>O. latipes</em> (Hd-rR: ASM223467v1), <em>O. sakaizumii</em> (HNI-II: ASM223471v1), <em>Oryzias</em> sp. (HSOK: ASM223469v1), <em>O. dancena</em> (Om_v0.7.RACA), and southern platyfish <em>Xiphophorus maculatus</em> (X_maculatus-5.0-male) were obtained from Ensembl Release 94 (<a href="http://www.ensembl.org/">http://www.ensembl.org/</a>). Among the six genomes, orthologous groups were classified and 10,852 single-copy orthologous genes were identified using OrthoFinder 2.2.6 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Emms and Kelly 2015</a>). For every single gene, codon alignment based on translated peptide sequences was generated by PAL2NAL (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Suyama <em>et al.</em> 2006</a>) and then trimmed by trimAl with ‘-autometed1’ option (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Capella-Gutiérrez <em>et al.</em> 2009</a>). All multi-sample fasta files were concatenated into a single file using AMAS <em>concat</em> by setting each gene as a separate partition (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Borowiec 2016</a>). A maximum likelihood tree was then inferred using IQ-TREE v1.6.6 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nguyen <em>et al.</em> 2015</a>) with the GTR+G substitution model for each codon, followed by an ultrafast bootstrap analysis of 1,000 replicates (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hoang <em>et al.</em> 2018</a>). This tree (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) indicates that <em>O. javanicus</em> forms a monophyletic group with <em>O. dancena</em> but not with the <em>O. latipes</em> species complex (Hd-rR, HNI-II, and HSOK), being consistent with previous trees inferred from two mitochondrial genes and a nuclear gene (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Takehana <em>et al.</em> 2005</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/907/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic position of O. javanicus. Maximum likelihood tree was inferred from the concatenated codon-alignment of 10,852 single-copy genes among 5 reference assemblies of Oryzias species with Southern platyfish (Xiphophorus maculatus) as outgroup. All nodes were supported by 100% bootstrap values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943044791" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic position of &lt;em&gt;O&lt;/em&gt;. &lt;em&gt;javanicus&lt;/em&gt;. Maximum likelihood tree was inferred from the concatenated codon-alignment of 10,852 single-copy genes among 5 reference assemblies of &lt;em&gt;Oryzias&lt;/em&gt; species with Southern platyfish (&lt;em&gt;Xiphophorus maculatus&lt;/em&gt;) as outgroup. All nodes were supported by 100% bootstrap values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/907/F3.medium.gif" width="440" height="97"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/3/907/F3.medium.gif" width="440" height="97"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/907/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/907/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495963" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Phylogenetic position of <em>O</em>. <em>javanicus</em>. Maximum likelihood tree was inferred from the concatenated codon-alignment of 10,852 single-copy genes among 5 reference assemblies of <em>Oryzias</em> species with Southern platyfish (<em>Xiphophorus maculatus</em>) as outgroup. All nodes were supported by 100% bootstrap values.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The D-GENIES (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cabanettes and Klopp 2018</a>) genome-wide comparison of this <em>O. javanicus</em> genome compared to the <em>O. latipes</em> reference genome [Ensembl version ASM223467v1 (GCA_002234675.1)] shows that these two genomes are extremely colinear at the whole genome scale (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). At a chromosome scale the comparison with <em>O. latipes</em> shows that most of the <em>O. javanicus</em> chromosomes are strongly colinear with their single <em>O. latipes</em> chromosome counterparts (Supplemental Figure 1). Only a few <em>O. Javanicus</em> chromosomes are more deeply reorganized compared to their <em>O. latipes</em> chromosome counterparts with for instance, the <em>O. javanicus</em> LG04, LG14, LG23 and LG24 that display multiple intra chromosomal rearrangements and the <em>O. javanicus</em> LG10, LG11 and LG14 that show small inter chromosomal rearrangements (<em>i.e.</em>, the insertion of a small region from an <em>O. latipes</em> different chromosome). With regards to their sex chromosomes <em>O. latipes</em> has a male heterogametic system (XX/XY) and the LG01 is the Y sex chromosome (see (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Herpin and Schartl 2009</a>) for review). <em>O. javanicus</em> has a female heterogametic sex determination system and LG16 is the W sex chromosome (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Takehana <em>et al.</em> 2008</a>). In <em>O. javanicus</em> both the LG01 (the <em>O. latipes</em> Y chromosome) and the LG16 (the <em>O. javanicus</em> W chromosome) display a strong chromosome collinearity with respectively the <em>O. latipes</em> LG01 and LG16. Indeed, and according to previous reports (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Takehana <em>et al.</em> 2008</a>) the <em>dmrt1bY</em> Y specific duplication/insertion on <em>O. latipes</em> LG01 is absent from the <em>O. javanicus</em> LG01 sequence.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/907/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide comparison of the Oryzias javanicus and Oryzias latipes genome assemblies. This dot-plot alignment of the O. javanicus genome with the O. latipes reference genome [Ensembl version ASM223467v1 (GCA_002234675.1)] was produced with the D-GENIES software (Cabanettes and Klopp 2018) available online at http://dgenies.toulouse.inra.fr/ with default visualization parameters and a slight filtering of the small matches. The different ranges of percentage of identity are shown in different colors (legend panel with the identity percentage color range)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943044791" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide comparison of the &lt;em&gt;Oryzias javanicus&lt;/em&gt; and &lt;em&gt;Oryzias latipes&lt;/em&gt; genome assemblies. This dot-plot alignment of the &lt;em&gt;O. javanicus&lt;/em&gt; genome with the &lt;em&gt;O. latipes&lt;/em&gt; reference genome [Ensembl version ASM223467v1 (GCA_002234675.1)] was produced with the D-GENIES software (Cabanettes and Klopp 2018) available online at http://dgenies.toulouse.inra.fr/ with default visualization parameters and a slight filtering of the small matches. The different ranges of percentage of identity are shown in different colors (legend panel with the identity percentage color range).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/907/F4.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/3/907/F4.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/907/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/907/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495902" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Genome-wide comparison of the <em>Oryzias javanicus</em> and <em>Oryzias latipes</em> genome assemblies. This dot-plot alignment of the <em>O. javanicus</em> genome with the <em>O. latipes</em> reference genome [Ensembl version ASM223467v1 (GCA_002234675.1)] was produced with the D-GENIES software (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cabanettes and Klopp 2018</a>) available online at <a href="http://dgenies.toulouse.inra.fr/">http://dgenies.toulouse.inra.fr/</a> with default visualization parameters and a slight filtering of the small matches. The different ranges of percentage of identity are shown in different colors (legend panel with the identity percentage color range).</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-18" class="subsection"><h3>Adaptation to salinity and hatching enzymes</h3><p id="p-28">To gain insight into gene family evolution associated with osmoregulation, we used HMMER version 3.1b2 to identify Pfam domain containing proteins in the <em>O. javanicus</em> genome. We used protein sequences based on our gene model of <em>O. javanicus</em> combined with Ensembl genes of the <em>O. latipes</em> species complex (Hd-rR, HNI-II and HSOK) and <em>O. dancena</em> for the Pfam search, and focused on 147 domains found in 224 proteins whose functions were related to osmoregulation (Supplemental Tables S1 and S2). Similar numbers of proteins were observed among species for each domain, suggesting that the osmoregulation gene repertoires are relatively conserved in <em>Oryzias</em> species. However, further detailed comparisons are required because gene annotation methods are different among data.</p><p id="p-29">We then also focused on specific genes encoding hatching enzymes. In the genome of <em>O. latipes</em>, five copies of <em>hce</em> genes -including one pseudogene- are clustered tandemly with the same transcriptional direction on chromosome 3 (chr. 3), while only one single copy of the <em>lce</em> gene is located on chromosome 24 (chr. 24) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kawaguchi <em>et al.</em> 2007</a>). In <em>O. javanicus</em> 5 copies of the <em>hce (Ojhce)</em> gene are located on chromosome 3 and one <em>lce (Ojlce)</em> gene was found on chromosome 24. The amino acid sequence similarities in the mature enzyme region of the 5 <em>Ojhce</em> genes are between 89–99%. Only in comparison to <em>O. latipes</em>, within the five <em>O. javanicus hce</em> genes, the fourth one (<em>Ojhce4</em>) displays an opposite orientation compared to the others (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>) suggesting a re-arrangement within the <em>hce</em> gene cluster that has likely been occurring during the evolution of <em>Oryzias</em> lineage. Phylogenetic analyses indicated that all the cloned <em>hce</em> and <em>lce</em> genes were orthologous to other euteleosteans <em>hce</em> and <em>lce</em> respectively.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/907/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hatching enzyme of Oryzias javanicus. (A) hce gene cluster of O. latipes (mhce1-5) and O. javanicus (Ojhce1-5). Arrowheads indicate direction of transcription. (B) Salt dependency of O. javanicus hce (black circle) and O. latipes (white circle). Activities are shown as % of relative activity with respect to highest activity, which is considered as 100% in each species." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-943044791" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Hatching enzyme of Oryzias javanicus. (A) &lt;em&gt;hce&lt;/em&gt; gene cluster of &lt;em&gt;O. latipes&lt;/em&gt; (&lt;em&gt;mhce1-5&lt;/em&gt;) and &lt;em&gt;O. javanicus&lt;/em&gt; (&lt;em&gt;Ojhce1-5&lt;/em&gt;). Arrowheads indicate direction of transcription. (B) Salt dependency of &lt;em&gt;O. javanicus hce&lt;/em&gt; (black circle) and &lt;em&gt;O. latipes&lt;/em&gt; (white circle). Activities are shown as % of relative activity with respect to highest activity, which is considered as 100% in each species.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/907/F5.medium.gif" width="401" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/10/3/907/F5.medium.gif" width="401" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/907/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/907/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495994" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-30" class="first-child">Hatching enzyme of Oryzias javanicus. (A) <em>hce</em> gene cluster of <em>O. latipes</em> (<em>mhce1-5</em>) and <em>O. javanicus</em> (<em>Ojhce1-5</em>). Arrowheads indicate direction of transcription. (B) Salt dependency of <em>O. javanicus hce</em> (black circle) and <em>O. latipes</em> (white circle). Activities are shown as % of relative activity with respect to highest activity, which is considered as 100% in each species.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">While LCE’s activity remains constant over various salinities, HCEs have been reported to show salt-dependent activity (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Kawaguchi <em>et al.</em> 2013</a>). In contrast to other <em>Oryzias</em> species, <em>O. javanicus</em>, being a euryhaline species, specifically adapted its physiology to higher water salinities. In order to test whether such adaptive evolution would translate at the level of HCE activity, recombinant OjHCE3 (rOjHCE3) was generated in an <em>E. coli</em> expression system, refolded, and its activity regarding to the digestion of the egg-envelope determined at various salt concentrations based on the method described in Kawaguchi <em>et al.</em> (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Kawaguchi <em>et al.</em> 2013</a>). Although rOjHCE3 showed virtually no activity at 0 M NaCl, an increased activity was apparent at elevated salt concentrations. Furtheron rOjHCE3 activity was recorded to be highest at 0.25 M NaCl, while still maintaining high activity up to 0.75 M NaCl (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). In contrast, it has been reported that <em>O. latipes</em> HCEs show highest activity at 0 M NaCl, and drastically decrease when salt concentrations increase ((<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Kawaguchi <em>et al.</em> 2013</a>), <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). These results suggest that salt preference of HCE enzymes is a species-specific adaptation to different salt environments at hatching.</p><p id="p-32">The Java medaka, <em>Oryzias javanicus</em>, is one of the two species of medaka living in brackish/sea-waters. Being an important component of the mangrove ecosystem, <em>O. javanicus</em> is also used as a valuable marine test-fish for ecotoxicology studies. Here, we sequenced and assembled the whole genome of <em>O. javanicus</em>. Complementary sequencing approaches and data integration with a genetic map allowed the final assembly of the 908 Mbp of the <em>O. javanicus</em> genome. The final draft assembly contains 525 scaffolds with a total length of 809.7 Mbp, a N50 of 6,3 Mbp and a L50 of 37 scaffolds. Providing here a high-quality draft genome assembly of the euryhaline Javafish medaka, we anticipate this resource will be catalytic for a wide range of comparative genomic, phylogenetic and functional studies within the genus <em>Oryzias</em> and beyond.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">This work was supported by a “Projet Incitatif PHASE department 2015” grant (Grant ID ACI_PHASE, INRAE) to AH, a NIBB Collaborative Research Initiative to KN, NIBB individual Collaboration Research Project to MK, and Grants-in-Aid for Young Scientists to YT (Grant ID 16K18590) and MK (Grant ID 16K18593). AH was additionally funded by the project AquaCRISPR (ANR-16-COFA-0004-01). The GeT and MGX core facilities were supported by France Génomique National infrastructure, funded as part of “Investissement d’avenir” program managed by Agence Nationale pour la Recherche (contract ANR-10-INBS-09). The GeT core facility was also supported by the GET-PACBIO program (« Programme operationnel FEDER-FSE MIDI-PYRENEES ET GARONNE 2014-2020 »).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-34">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.10310498">https://doi.org/10.25387/g3.10310498</a>.</p></li><li class="fn" id="fn-3"><p id="p-35"><em>Communicating editor: D. J. Grunwald</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-09-13"><span class="received-label">Received </span>September 13, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2020-01-23"><span class="accepted-label">Accepted </span>January 23, 2020.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Takehana <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.907.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrew</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> FastQC: a quality control tool for high throughput sequence data. Available online at: <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc">https://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.3.907.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Animal Genome Size Database</span></li></ol><cite>. Available at: <a href="http://www.genomesize.com/index.php">http://www.genomesize.com/index.php</a>. Accessed: September 13, 2019.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.3" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>,  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Etter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Atwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Currey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Shiver</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e3376</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0003376</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.4" data-doi="10.1093/nar/27.2.573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benson</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Tandem repeats finder: a program to analyze DNA sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/27.2.573</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.2.573%26rft_id%253Dinfo%253Apmid%252F9862982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.2.573&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9862982&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078181500028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.5" data-doi="10.1016/j.ympev.2012.08.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernt</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Donath</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Jühling</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Externbrink</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Florentz</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MITOS: improved de novo metazoan mitochondrial genome annotation.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">319</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ympev.2012.08.023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D69%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2012.08.023%26rft_id%253Dinfo%253Apmid%252F22982435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2012.08.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22982435&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.6" data-doi="10.7717/peerj.1660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borowiec</span>,  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">AMAS: a fast tool for alignment manipulation and computing of summary statistics.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e1660</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7717/peerj.1660</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D4%26rft.spage%253De1660%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.1660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.1660&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.7" data-doi="10.1038/nmeth.3176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchfink</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Xie</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Huson</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fast and sensitive protein alignment using DIAMOND.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3176</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3176%26rft_id%253Dinfo%253Apmid%252F25402007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25402007&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.8" data-doi="10.7717/peerj.4958"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabanettes</span>,  <span class="cit-name-given-names">F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Klopp</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">D-GENIES: dot plot large genomes in an interactive, efficient and simple way.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e4958</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7717/peerj.4958</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D6%26rft.spage%253De4958%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.4958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.4958&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.9" data-doi="10.7717/peerj.2988"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabau</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Escudié</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Djari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Guiguen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bobe</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Compacting and correcting Trinity and Oases RNA-Seq de novo assemblies.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e2988</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7717/peerj.2988</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D5%26rft.spage%253De2988%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.2988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.2988&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.10" data-doi="10.1093/bioinformatics/btp348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capella-Gutiérrez</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Silla-Martínez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Gabaldón</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1972</span>–<span class="cit-lpage">1973</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp348</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp348%26rft_id%253Dinfo%253Apmid%252F19505945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp348&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505945&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268107100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.11" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Auton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Abecasis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Albers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.12" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Schlesinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Drenkow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zaleski</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.13" data-doi="10.1186/1471-2105-10-67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eilbeck</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Moore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Holt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yandell</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantitative measures for the management and comparison of annotated genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-67</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEilbeck%26rft.auinit1%253DK.%26rft.volume%253D10%26rft.spage%253D67%26rft.epage%253D67%26rft.atitle%253DQuantitative%2Bmeasures%2Bfor%2Bthe%2Bmanagement%2Band%2Bcomparison%2Bof%2Bannotated%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-67%26rft_id%253Dinfo%253Apmid%252F19236712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-67&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19236712&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.14" data-doi="10.1093/nar/gky995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Gebali</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mistry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bateman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Luciani</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">The Pfam protein families database in 2019.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">D427</span>–<span class="cit-lpage">D432</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky995</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky995%26rft_id%253Dinfo%253Apmid%252F30357350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky995&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=30357350&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.15" data-doi="10.1186/s13059-015-0721-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emms</span>,  <span class="cit-name-given-names">D. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kelly</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-015-0721-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0721-2%26rft_id%253Dinfo%253Apmid%252F26243257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0721-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26243257&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.16" data-doi="10.1016/j.mce.2009.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herpin</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schartl</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Molecular mechanisms of sex determination and evolution of the Y-chromosome: insights from the medakafish (Oryzias latipes).</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Endocrinol.</abbr> <span class="cit-vol">306</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">58</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mce.2009.02.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bcellular%2Bendocrinology%26rft.stitle%253DMol%2BCell%2BEndocrinol%26rft.aulast%253DHerpin%26rft.auinit1%253DA.%26rft.volume%253D306%26rft.issue%253D1-2%26rft.spage%253D51%26rft.epage%253D58%26rft.atitle%253DMolecular%2Bmechanisms%2Bof%2Bsex%2Bdetermination%2Band%2Bevolution%2Bof%2Bthe%2BY-chromosome%253A%2Binsights%2Bfrom%2Bthe%2Bmedakafish%2B%2528Oryzias%2Blatipes%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mce.2009.02.004%26rft_id%253Dinfo%253Apmid%252F19481684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mce.2009.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19481684&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.17" data-doi="10.1093/molbev/msx281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoang</span>,  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Chernomor</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">von Haeseler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. Q.</span> <span class="cit-name-surname">Minh</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. S.</span> <span class="cit-name-surname">Vinh</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">UFBoot2: Improving the Ultrafast Bootstrap Approximation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">518</span>–<span class="cit-lpage">522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx281</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D35%26rft.spage%253D518%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx281%26rft_id%253Dinfo%253Apmid%252F29077904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29077904&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.18" data-doi="10.1093/bioinformatics/btv661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoff</span>,  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lange</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lomsadze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Borodovsky</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stanke</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">BRAKER1: Unsupervised RNA-Seq-Based Genome Annotation with GeneMark-ET and AUGUSTUS.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv661</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv661%26rft_id%253Dinfo%253Apmid%252F26559507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26559507&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.19" data-doi="10.1186/1471-2105-12-491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Yandell</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-491</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHolt%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D491%26rft.epage%253D491%26rft.atitle%253DMAKER2%253A%2Ban%2Bannotation%2Bpipeline%2Band%2Bgenome-database%2Bmanagement%2Btool%2Bfor%2Bsecond-generation%2Bgenome%2Bprojects.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-491%26rft_id%253Dinfo%253Apmid%252F22192575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192575&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.20" data-doi="10.1007/s00128-018-2398-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horie</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kanazawa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Yamagishi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Yonekura</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Tatarazako</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Ecotoxicological Test Assay Using OECD TG 212 in Marine Java Medaka (Oryzias javanicus) and Freshwater Japanese Medaka (Oryzias latipes).</span> <abbr class="cit-jnl-abbrev">Bull. Environ. Contam. Toxicol.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00128-018-2398-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBull.%2BEnviron.%2BContam.%2BToxicol.%26rft.volume%253D101%26rft.spage%253D344%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00128-018-2398-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00128-018-2398-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.21" data-doi="10.1016/S1096-4959(03)00204-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Takei</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Asian medaka fishes offer new models for studying mechanisms of seawater adaptation.</span> <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. B Biochem. Mol. Biol.</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">645</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1096-4959(03)00204-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BBiochemistry%2Band%2BPhysiology.%2BPart%2BB%252C%2BBiochemistry%2B%2526%2BMolecular%2BBiology%26rft.stitle%253DComparative%2BBiochemistry%2Band%2BPhysiology.%2BPart%2BB%252C%2BBiochemistry%2B%2526%2BMolecular%2BBiology%26rft.aulast%253DInoue%26rft.auinit1%253DK.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D635%26rft.epage%253D645%26rft.atitle%253DAsian%2Bmedaka%2Bfishes%2Boffer%2Bnew%2Bmodels%2Bfor%2Bstudying%2Bmechanisms%2Bof%2Bseawater%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1096-4959%252803%252900204-5%26rft_id%253Dinfo%253Apmid%252F14662290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1096-4959(03)00204-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14662290&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.22" data-doi="10.2108/zsj.19.727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Inoue</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Takei</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Diverse adaptability in oryzias species to high environmental salinity.</span> <abbr class="cit-jnl-abbrev">Zool. Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">727</span>–<span class="cit-lpage">734</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2108/zsj.19.727</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DInoue%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D727%26rft.epage%253D734%26rft.atitle%253DDiverse%2Badaptability%2Bin%2Boryzias%2Bspecies%2Bto%2Bhigh%2Benvironmental%2Bsalinity.%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.19.727%26rft_id%253Dinfo%253Apmid%252F12149572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.19.727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12149572&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178165300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.23" data-doi="10.1016/j.gene.2006.11.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawaguchi</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yasumasu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hiroi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Naruse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Suzuki</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Analysis of the exon-intron structures of fish, amphibian, bird and mammalian hatching enzyme genes, with special reference to the intron loss evolution of hatching enzyme genes in Teleostei.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">392</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">88</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gene.2006.11.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DKawaguchi%26rft.auinit1%253DM.%26rft.volume%253D392%26rft.issue%253D1-2%26rft.spage%253D77%26rft.epage%253D88%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bexon-intron%2Bstructures%2Bof%2Bfish%252C%2Bamphibian%252C%2Bbird%2Band%2Bmammalian%2Bhatching%2Benzyme%2Bgenes%252C%2Bwith%2Bspecial%2Breference%2Bto%2Bthe%2Bintron%2Bloss%2Bevolution%2Bof%2Bhatching%2Benzyme%2Bgenes%2Bin%2BTeleostei.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2006.11.012%26rft_id%253Dinfo%253Apmid%252F17222522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2006.11.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17222522&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.24" data-doi="10.1242/jeb.069716"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawaguchi</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yasumasu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Shimizu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Kudo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sano</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Adaptive evolution of fish hatching enzyme: one amino acid substitution results in differential salt dependency of the enzyme.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">1609</span>–<span class="cit-lpage">1615</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jeb.069716</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.069716%26rft_id%253Dinfo%253Apmid%252F23348940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjEwOiIyMTYvOS8xNjA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.25" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>,  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT–the BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.229202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.26" data-doi="10.1534/genetics.114.173849"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirchmaier</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Naruse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wittbrodt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Loosli</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genomic and genetic toolbox of the teleost medaka (Oryzias latipes).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">918</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.173849</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKirchmaier%26rft.auinit1%253DS.%26rft.volume%253D199%26rft.issue%253D4%26rft.spage%253D905%26rft.epage%253D918%26rft.atitle%253DThe%2BGenomic%2Band%2BGenetic%2BToolbox%2Bof%2Bthe%2BTeleost%2BMedaka%2B%2528Oryzias%2Blatipes%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.173849%26rft_id%253Dinfo%253Apmid%252F25855651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzQvOTA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.27" data-doi="10.1101/gr.215087.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">Walenz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Berlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Bergman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.215087.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.28" data-doi="10.1002/tox.20367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koyama</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kawamata</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Imai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Fukunaga</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Uno</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Java medaka: a proposed new marine test fish for ecotoxicology.</span> <abbr class="cit-jnl-abbrev">Environ. Toxicol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/tox.20367</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2Btoxicology%26rft.stitle%253DEnviron%2BToxicol%26rft.aulast%253DKoyama%26rft.auinit1%253DJ.%26rft.volume%253D23%26rft.issue%253D4%26rft.spage%253D487%26rft.epage%253D491%26rft.atitle%253DJava%2Bmedaka%253A%2Ba%2Bproposed%2Bnew%2Bmarine%2Btest%2Bfish%2Bfor%2Becotoxicology.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Ftox.20367%26rft_id%253Dinfo%253Apmid%252F18246546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/tox.20367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18246546&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.29" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903627&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.907.30"><div class="cit-metadata unstructured">Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv:1303.3997v2 [q-bio.GN]</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.31" data-doi="10.1093/bioinformatics/btr011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marçais</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kingsford</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">764</span>–<span class="cit-lpage">770</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr011%26rft_id%253Dinfo%253Apmid%252F21217122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217122&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288277300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.32" data-doi="10.14806/ej.17.1.200"><div class="cit-metadata"><cite>Martin, M., 2011 Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal 17: 10–12.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14806/ej.17.1.200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCUTADAPT%2BREMOVES%2BADAPTER%2BSEQUENCES%2BFROM%2BHIGHTHROUGHPUT%2BSEQUENCING%2BREADS%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252Fej.17.1.200%26rft_id%253Dinfo%253Apmid%252F25452271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14806/ej.17.1.200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25452271&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.33" data-doi="10.1016/j.ympev.2015.07.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mokodongan</span>,  <span class="cit-name-given-names">D. F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Yamahira</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Origin and intra-island diversification of Sulawesi endemic Adrianichthyidae.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">160</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ympev.2015.07.024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D93%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2015.07.024%26rft_id%253Dinfo%253Apmid%252F26256644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2015.07.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26256644&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.34" data-doi="10.1089/cmb.2006.13.1028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgulis</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Gertz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Schäffer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Agarwala</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and symmetric DUST implementation to mask low-complexity DNA sequences.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1028</span>–<span class="cit-lpage">1040</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/cmb.2006.13.1028</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DMorgulis%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D1028%26rft.epage%253D1040%26rft.atitle%253DA%2Bfast%2Band%2Bsymmetric%2BDUST%2Bimplementation%2Bto%2Bmask%2Blow-complexity%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2006.13.1028%26rft_id%253Dinfo%253Apmid%252F16796549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2006.13.1028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16796549&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.35" data-doi="10.1534/genetics.111.137497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myosho</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Otake</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Masuyama</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Matsuda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Kuroki</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Tracing the emergence of a novel sex-determining gene in medaka, Oryzias luzonensis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.137497</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137497%26rft_id%253Dinfo%253Apmid%252F22367037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzEvMTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy85MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.36" data-doi="10.1038/s41598-018-24621-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myosho</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Takahashi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Yoshida</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Sato</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hamaguchi</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Hyperosmotic tolerance of adult fish and early embryos are determined by discrete, single loci in the genus Oryzias.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">6897</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-018-24621-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D8%26rft.spage%253D6897%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-24621-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-24621-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.37" data-doi="10.1093/molbev/msu300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">L.-T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. A.</span> <span class="cit-name-surname">Schmidt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">von Haeseler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. Q.</span> <span class="cit-name-surname">Minh</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu300</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu300%26rft_id%253Dinfo%253Apmid%252F25371430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25371430&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.38" data-doi="10.1111/j.1096-3642.2008.00417.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parenti</span>,  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A phylogenetic analysis and taxonomic revision of ricefishes, Oryzias and relatives (Beloniformes, Adrianichthyidae).</span> <abbr class="cit-jnl-abbrev">Zool. J. Linn. Soc.</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">494</span>–<span class="cit-lpage">610</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1096-3642.2008.00417.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZool.%2BJ.%2BLinn.%2BSoc.%26rft.volume%253D154%26rft.spage%253D494%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1096-3642.2008.00417.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1096-3642.2008.00417.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.39" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>,  <span class="cit-name-given-names">A. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Hall</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.40" data-doi="10.1093/bioinformatics/btx494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>,  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Lep-MAP3: robust linkage mapping even for low-coverage whole genome sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">3726</span>–<span class="cit-lpage">3732</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx494</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D3726%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.3.907.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">RepeatMasker Home Page</span></li></ol><cite>. Available at: <a href="http://www.repeatmasker.org/">http://www.repeatmasker.org/</a>. Accessed: September 13, 2019.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.42"><div class="cit-metadata unstructured">Ruan, J., 2019 Ultra-fast de novo assembler using long noisy reads: Ruanjue/smartdenovo. Available at: <a href="https://github.com/ruanjue/smartdenovo">https://github.com/ruanjue/smartdenovo</a>. Accessed: September 13, 2019.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.43" data-doi="10.1093/bioinformatics/bts094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>,  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Zerbino</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Vingron</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Birney</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1086</span>–<span class="cit-lpage">1092</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts094</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts094%26rft_id%253Dinfo%253Apmid%252F22368243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts094&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22368243&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302806900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.44" data-doi="10.1098/rsbl.2009.0419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Setiamarga</span>,  <span class="cit-name-given-names">D. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Miya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Yamanoue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Azuma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Inoue</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Divergence time of the two regional medaka populations in Japan as a new time scale for comparative genomics of vertebrates.</span> <abbr class="cit-jnl-abbrev">Biol. Lett.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">812</span>–<span class="cit-lpage">816</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsbl.2009.0419</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsbl.2009.0419%26rft_id%253Dinfo%253Apmid%252F19586967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsbl.2009.0419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2013074188&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=19586967&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.45" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.46" data-doi="10.1186/1471-2105-6-31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>,  <span class="cit-name-given-names">G. S. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Birney</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Automated generation of heuristics for biological sequence comparison.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">31</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-6-31</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DSlater%26rft.auinit1%253DG.%2BS.%26rft.volume%253D6%26rft.spage%253D31%26rft.epage%253D31%26rft.atitle%253DAutomated%2Bgeneration%2Bof%2Bheuristics%2Bfor%2Bbiological%2Bsequence%2Bcomparison.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-6-31%26rft_id%253Dinfo%253Apmid%252F15713233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-6-31&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15713233&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.47"><div class="cit-metadata unstructured">Smit, A. F. A., and R. Hubley, 2010 RepeatModeler Open-1.0.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.48" data-doi="10.1093/nar/gkl315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suyama</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Torrents</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bork</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W609</span>–<span class="cit-lpage">W612</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSuyama%26rft.auinit1%253DM.%26rft.volume%253D34%26rft.issue%253DWeb%2BServer%2Bissue%26rft.spage%253DW609%26rft.epage%253DW612%26rft.atitle%253DPAL2NAL%253A%2Brobust%2Bconversion%2Bof%2Bprotein%2Bsequence%2Balignments%2Binto%2Bthe%2Bcorresponding%2Bcodon%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl315%26rft_id%253Dinfo%253Apmid%252F16845082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845082&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245650200122&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.49" data-doi="10.1534/genetics.106.068247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Demiyah</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Naruse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hamaguchi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sakaizumi</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span> <span class="cit-article-title">Evolution of different Y chromosomes in two medaka species, Oryzias dancena and O. latipes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1335</span>–<span class="cit-lpage">1340</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.068247</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.068247%26rft_id%253Dinfo%253Apmid%252F17194774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzEzMzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzkwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.50" data-doi="10.1007/s10577-008-1227-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hamaguchi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sakaizumi</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Different origins of ZZ/ZW sex chromosomes in closely related medaka fishes, Oryzias javanicus and O. hubbsi.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">801</span>–<span class="cit-lpage">811</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10577-008-1227-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DTakehana%26rft.auinit1%253DY.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D801%26rft.epage%253D811%26rft.atitle%253DDifferent%2Borigins%2Bof%2BZZ%252FZW%2Bsex%2Bchromosomes%2Bin%2Bclosely%2Brelated%2Bmedaka%2Bfishes%252C%2BOryzias%2Bjavanicus%2Band%2BO.%2Bhubbsi.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-008-1227-5%26rft_id%253Dinfo%253Apmid%252F18607761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-008-1227-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18607761&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258315200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.51" data-doi="10.1038/ncomms5157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Matsuda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Myosho</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Suster</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawakami</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Co-option of Sox3 as the male-determining factor on the Y chromosome in the fish Oryzias dancena.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms5157</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D4157%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms5157%26rft_id%253Dinfo%253Apmid%252F24948391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms5157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24948391&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.52" data-doi="10.1007/s00412-007-0110-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Naruse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hamaguchi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sakaizumi</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">Evolution of ZZ/ZW and XX/XY sex-determination systems in the closely related medaka species, Oryzias hubbsi and O. dancena.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">470</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00412-007-0110-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.stitle%253DChromosoma%26rft.aulast%253DTakehana%26rft.auinit1%253DY.%26rft.volume%253D116%26rft.issue%253D5%26rft.spage%253D463%26rft.epage%253D470%26rft.atitle%253DEvolution%2Bof%2BZZ%252FZW%2Band%2BXX%252FXY%2Bsex-determination%2Bsystems%2Bin%2Bthe%2Bclosely%2Brelated%2Bmedaka%2Bspecies%252C%2BOryzias%2Bhubbsi%2Band%2BO.%2Bdancena.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00412-007-0110-z%26rft_id%253Dinfo%253Apmid%252F17882464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00412-007-0110-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17882464&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249620100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.53" data-doi="10.1016/j.ympev.2005.01.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takehana</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Naruse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sakaizumi</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Molecular phylogeny of the medaka fishes genus Oryzias (Beloniformes: Adrianichthyidae) based on nuclear and mitochondrial DNA sequences.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ympev.2005.01.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DTakehana%26rft.auinit1%253DY.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D417%26rft.epage%253D428%26rft.atitle%253DMolecular%2Bphylogeny%2Bof%2Bthe%2Bmedaka%2Bfishes%2Bgenus%2BOryzias%2B%2528Beloniformes%253A%2BAdrianichthyidae%2529%2Bbased%2Bon%2Bnuclear%2Band%2Bmitochondrial%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2005.01.016%26rft_id%253Dinfo%253Apmid%252F15955519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2005.01.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15955519&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230265900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.54" data-doi="10.1534/genetics.107.075598"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Takehana</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Naruse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hamaguchi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sakaizumi</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evidence for different origins of sex chromosomes in closely related Oryzias fishes: substitution of the master sex-determining gene.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2075</span>–<span class="cit-lpage">2081</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.075598</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.075598%26rft_id%253Dinfo%253Apmid%252F17947439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIwNzUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzkwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.55" data-doi="10.1186/s13059-014-0573-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Miao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ming</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ALLMAPS: robust scaffold ordering based on multiple maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-014-0573-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0573-1%26rft_id%253Dinfo%253Apmid%252F25583564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0573-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25583564&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.56" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. A.</span> <span class="cit-name-surname">Williams</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Pertea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mortazavi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kwan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1621</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-10.3.907.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Trim Galore</span></li></ol><cite>. Available at: <a href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>. Accessed: September 13, 2019.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.58" data-doi="10.1093/bioinformatics/btx153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vurture</span>,  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. J.</span> <span class="cit-name-surname">Sedlazeck</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nattestad</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Underwood</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Fang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">GenomeScope: fast reference-free genome profiling from short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2202</span>–<span class="cit-lpage">2204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2202%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx153%26rft_id%253Dinfo%253Apmid%252F28369201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28369201&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.59" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Abeel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Shea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Priest</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Abouelliel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e112963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.60" data-doi="10.1038/nrg704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wittbrodt</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Shima</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schartl</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Medaka–a model organism from the far East.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg704</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWittbrodt%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D64%26rft.atitle%253DMedaka--a%2Bmodel%2Borganism%2Bfrom%2Bthe%2Bfar%2BEast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg704%26rft_id%253Dinfo%253Apmid%252F11823791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg704&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11823791&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173267200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.61" data-doi="10.1016/0305-0491(73)90100-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamagami</span>,  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Some enzymological properties of a hatching enzyme (chorionase) isolated from the fresh-water teleost, Oryzias latipes.</span> <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. B</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">603</span>–<span class="cit-lpage">616</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0305-0491(73)90100-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BBiochemistry%2Band%2BPhysiology.%2BPart%2BB%252C%2BBiochemistry%2B%2526%2BMolecular%2BBiology%26rft.stitle%253DComparative%2BBiochemistry%2Band%2BPhysiology.%2BPart%2BB%252C%2BBiochemistry%2B%2526%2BMolecular%2BBiology%26rft.aulast%253DYamagami%26rft.auinit1%253DK.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D603%26rft.epage%253D616%26rft.atitle%253DSome%2Benzymological%2Bproperties%2Bof%2Ba%2Bhatching%2Benzyme%2B%2528chorionase%2529%2Bisolated%2Bfrom%2Bthe%2Bfresh-water%2Bteleost%252C%2BOryzias%2Blatipes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0305-0491%252873%252990100-4%26rft_id%253Dinfo%253Apmid%252F4201938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0305-0491(73)90100-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4201938&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.62" data-doi="10.1093/jb/mvq086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yasumasu</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kawaguchi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ouchi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Sano</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Murata</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mechanism of egg envelope digestion by hatching enzymes, HCE and LCE in medaka, Oryzias latipes.</span> <abbr class="cit-jnl-abbrev">J. Biochem.</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiochem.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjb%252Fmvq086%26rft_id%253Dinfo%253Apmid%252F20679367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jb/mvq086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20679367&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F907.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282752400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeo</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Coombe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Warren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Birol</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">ARCS: scaffolding genome drafts with linked reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.64" data-doi="10.1007/s10641-011-9888-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yusof</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ismail</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Koito</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kinoshita</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Inoue</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Occurrence of two closely related ricefishes, Javanese medaka (Oryzias javanicus) and Indian medaka (O. dancena) at sites with different salinity in Peninsular Malaysia.</span> <abbr class="cit-jnl-abbrev">Environ. Biol. Fishes</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10641-011-9888-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BBiol.%2BFishes%26rft.volume%253D93%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10641-011-9888-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10641-011-9888-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.907.65"><div class="cit-metadata unstructured">Zulkifli, S. Z., F. Mohamat-Yusuff, A. Ismail, and N. Miyazaki, 2012 Food preference of the giant mudskipper Periophthalmodon schlosseri (Teleostei : Gobiidae). Knowl. Managt. Aquatic Ecosyst. 07.</div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/3/907.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/899" title="Improved Genome Assembly and Annotation of the Soybean Aphid (Aphis glycines Matsumura)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/917" title="Mutational Analysis of N-Ethyl-N-Nitrosourea (ENU) in the Fission Yeast Schizosaccharomyces pombe" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/3"><span>Volume 10 Issue 3, March 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/3.cover-source.jpg?itok=8jjPWDA4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/907?rss=1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gowHQWNa-sIMQep6GunmUekltfSWHmgj1yep-nIhcgg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/3/907" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/3/907" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/3/907" class="active" data-icon-position="" data-hide-link-title="0">Genome Sequence of the Euryhaline Javafish Medaka, Oryzias javanicus: A Small Aquarium Fish Model for Studies on Adaptation to Salinity</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495848" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kyeffmStf2xbBykrBVVGOMqcH6j7EE5BjjjB3lKwPY0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/907.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/3/907" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Gp5d_yVHcFtwzkrAf-gk8qxYsVYFC6RB4VS7w5UjlMY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/3/907" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/907.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495848" id="node495848--2" data-pisa="ggg;10/3/907" data-pisa-master="ggg;g3.119.400725" data-apath="/ggg/10/3/907.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Sequence of the Euryhaline Javafish Medaka, <em>Oryzias javanicus</em>: A Small Aquarium Fish Model for Studies on Adaptation to Salinity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yusuke Takehana</span>, <span class="highwire-citation-author" data-delta="1">Margot Zahm</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-8560-4003" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cédric Cabau</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7126-5477" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christophe Klopp</span>, <span class="highwire-citation-author" data-delta="4">Céline Roques</span>, <span class="highwire-citation-author" data-delta="5">Olivier Bouchez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-5041-6843" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cécile Donnadieu</span>, <span class="highwire-citation-author" data-delta="7">Celia Barrachina</span>, <span class="highwire-citation-author" data-delta="8">Laurent Journot</span>, <span class="highwire-citation-author" data-delta="9">Mari Kawaguchi</span>, <span class="highwire-citation-author" data-delta="10">Shigeki Yasumasu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-2683-0160" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Satoshi Ansai</span>, <span class="highwire-citation-author" data-delta="12">Kiyoshi Naruse</span>, <span class="highwire-citation-author" data-delta="13">Koji Inoue</span>, <span class="highwire-citation-author" data-delta="14">Chuya Shinzato</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15"><a href="http://orcid.org/0000-0001-9882-5948" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Manfred Schartl</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yann Guiguen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="17"><a href="http://orcid.org/0000-0002-0630-4027" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amaury Herpin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">907-915;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400725 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495848--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yusuke Takehana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Bio-Science, Faculty of Bio-Science, Nagahama Institute of Bioscience and Technology, 1266 Tamura, Nagahama 526-0829, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yusuke%2BTakehana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takehana%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYusuke%2BTakehana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yusuke Takehana</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Margot Zahm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plate-forme bio-informatique Genotoul, Mathématiques et Informatique Appliquées de Toulouse, INRAE, Castanet Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Margot%2BZahm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zahm%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMargot%2BZahm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cédric Cabau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>SIGENAE, GenPhySE, Université de Toulouse, INRAE, ENVT, Castanet Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9dric%2BCabau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabau%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC%25C3%25A9dric%2BCabau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8560-4003" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cédric Cabau</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christophe Klopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plate-forme bio-informatique Genotoul, Mathématiques et Informatique Appliquées de Toulouse, INRAE, Castanet Tolosan, France,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>SIGENAE, GenPhySE, Université de Toulouse, INRAE, ENVT, Castanet Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2BKlopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Klopp%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristophe%2BKlopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7126-5477" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christophe Klopp</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Céline Roques </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9line%2BRoques%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roques%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC%25C3%25A9line%2BRoques%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Olivier Bouchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BBouchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouchez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlivier%2BBouchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Cécile Donnadieu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9cile%2BDonnadieu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Donnadieu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC%25C3%25A9cile%2BDonnadieu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5041-6843" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cécile Donnadieu</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Celia Barrachina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>MGX, Biocampus Montpellier, CNRS, INSERM, University of Montpellier, Montpellier, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celia%2BBarrachina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barrachina%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACelia%2BBarrachina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Laurent Journot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>MGX, Biocampus Montpellier, CNRS, INSERM, University of Montpellier, Montpellier, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurent%2BJournot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Journot%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaurent%2BJournot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mari Kawaguchi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Materials and Life Sciences, Faculty of Science and Technology, Sophia University, 7-1 Kioi-cho Chiyoda-ku, Tokyo 102-8554, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mari%2BKawaguchi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kawaguchi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMari%2BKawaguchi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Shigeki Yasumasu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Materials and Life Sciences, Faculty of Science and Technology, Sophia University, 7-1 Kioi-cho Chiyoda-ku, Tokyo 102-8554, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shigeki%2BYasumasu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yasumasu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShigeki%2BYasumasu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Satoshi Ansai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Laboratory of Bioresources, National Institute for Basic Biology, 38 Nishigonaka, Myodaiji-cho, Okazaki 444-8585, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satoshi%2BAnsai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ansai%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASatoshi%2BAnsai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2683-0160" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Satoshi Ansai</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Kiyoshi Naruse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Laboratory of Bioresources, National Institute for Basic Biology, 38 Nishigonaka, Myodaiji-cho, Okazaki 444-8585, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiyoshi%2BNaruse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naruse%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiyoshi%2BNaruse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Koji Inoue </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Atmosphere and Ocean Research Institute, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8564, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koji%2BInoue%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Inoue%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKoji%2BInoue%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Chuya Shinzato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Atmosphere and Ocean Research Institute, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8564, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chuya%2BShinzato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shinzato%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChuya%2BShinzato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Manfred Schartl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>University of Wuerzburg, Developmental Biochemistry, Biocenter, 97074 Wuerzburg, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Comprehensive Cancer Center Mainfranken, University Hospital, 97080 Wuerzburg, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>‡‡‡</span>Hagler Institute for Advanced Study and Department of Biology, Texas A&M University, College Station, Texas 77843, USA,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manfred%2BSchartl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schartl%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AManfred%2BSchartl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9882-5948" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Manfred Schartl</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Yann Guiguen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>INRAE, UR 1037 Fish Physiology and Genomics, F-35000 Rennes, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yann%2BGuiguen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guiguen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYann%2BGuiguen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yann Guiguen</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Amaury Herpin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>INRAE, UR 1037 Fish Physiology and Genomics, F-35000 Rennes, France, and</div><div class='author-affiliation'><span class='nlm-sup'>****</span>State Key Laboratory of Developmental Biology of Freshwater Fish, College of Life Sciences, Hunan Normal University, Changsha, 410081, Hunan, P. R. of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amaury%2BHerpin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Herpin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmaury%2BHerpin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0630-4027" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amaury Herpin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:amaury.herpin@inra.fr" class="" data-icon-position="" data-hide-link-title="0">amaury.herpin@inra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/3/907" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/3/907&amp;title=Genome%20Sequence%20of%20the%20Euryhaline%20Javafish%20Medaka%2C%20Oryzias%20javanicus%3A%20A%20Small%20Aquarium%20Fish%20Model%20for%20Studies%20on%20Adaptation%20to%20Salinity" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/3/907&amp;title=Genome%20Sequence%20of%20the%20Euryhaline%20Javafish%20Medaka%2C%20Oryzias%20javanicus%3A%20A%20Small%20Aquarium%20Fish%20Model%20for%20Studies%20on%20Adaptation%20to%20Salinity" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/3/907&amp;title=Genome%20Sequence%20of%20the%20Euryhaline%20Javafish%20Medaka%2C%20Oryzias%20javanicus%3A%20A%20Small%20Aquarium%20Fish%20Model%20for%20Studies%20on%20Adaptation%20to%20Salinity" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/3/907&amp;text=Genome%20Sequence%20of%20the%20Euryhaline%20Javafish%20Medaka%2C%20Oryzias%20javanicus%3A%20A%20Small%20Aquarium%20Fish%20Model%20for%20Studies%20on%20Adaptation%20to%20Salinity" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/3/907&amp;title=Genome%20Sequence%20of%20the%20Euryhaline%20Javafish%20Medaka%2C%20Oryzias%20javanicus%3A%20A%20Small%20Aquarium%20Fish%20Model%20for%20Studies%20on%20Adaptation%20to%20Salinity" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/3/907&amp;t=Genome%20Sequence%20of%20the%20Euryhaline%20Javafish%20Medaka%2C%20Oryzias%20javanicus%3A%20A%20Small%20Aquarium%20Fish%20Model%20for%20Studies%20on%20Adaptation%20to%20Salinity" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/3/907&amp;title=Genome%20Sequence%20of%20the%20Euryhaline%20Javafish%20Medaka%2C%20Oryzias%20javanicus%3A%20A%20Small%20Aquarium%20Fish%20Model%20for%20Studies%20on%20Adaptation%20to%20Salinity" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/3/907&amp;title=Genome%20Sequence%20of%20the%20Euryhaline%20Javafish%20Medaka%2C%20Oryzias%20javanicus%3A%20A%20Small%20Aquarium%20Fish%20Model%20for%20Studies%20on%20Adaptation%20to%20Salinity" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495848/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495848" id="node495848--3" data-pisa="ggg;10/3/907" data-pisa-master="ggg;g3.119.400725" data-apath="/ggg/10/3/907.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome Sequence of the Euryhaline Javafish Medaka, <em>Oryzias javanicus</em>: A Small Aquarium Fish Model for Studies on Adaptation to Salinity</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yusuke Takehana</span>, <span class="highwire-citation-author" data-delta="1">Margot Zahm</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-8560-4003" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cédric Cabau</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7126-5477" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Christophe Klopp</span>, <span class="highwire-citation-author" data-delta="4">Céline Roques</span>, <span class="highwire-citation-author" data-delta="5">Olivier Bouchez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-5041-6843" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Cécile Donnadieu</span>, <span class="highwire-citation-author" data-delta="7">Celia Barrachina</span>, <span class="highwire-citation-author" data-delta="8">Laurent Journot</span>, <span class="highwire-citation-author" data-delta="9">Mari Kawaguchi</span>, <span class="highwire-citation-author" data-delta="10">Shigeki Yasumasu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0003-2683-0160" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Satoshi Ansai</span>, <span class="highwire-citation-author" data-delta="12">Kiyoshi Naruse</span>, <span class="highwire-citation-author" data-delta="13">Koji Inoue</span>, <span class="highwire-citation-author" data-delta="14">Chuya Shinzato</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="15"><a href="http://orcid.org/0000-0001-9882-5948" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Manfred Schartl</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="16"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yann Guiguen</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="17"><a href="http://orcid.org/0000-0002-0630-4027" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amaury Herpin</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">907-915;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400725 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495848--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yusuke Takehana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Bio-Science, Faculty of Bio-Science, Nagahama Institute of Bioscience and Technology, 1266 Tamura, Nagahama 526-0829, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yusuke%2BTakehana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Takehana%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYusuke%2BTakehana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9965-8499" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yusuke Takehana</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Margot Zahm </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plate-forme bio-informatique Genotoul, Mathématiques et Informatique Appliquées de Toulouse, INRAE, Castanet Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Margot%2BZahm%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zahm%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMargot%2BZahm%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cédric Cabau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>SIGENAE, GenPhySE, Université de Toulouse, INRAE, ENVT, Castanet Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9dric%2BCabau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabau%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC%25C3%25A9dric%2BCabau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8560-4003" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cédric Cabau</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christophe Klopp </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plate-forme bio-informatique Genotoul, Mathématiques et Informatique Appliquées de Toulouse, INRAE, Castanet Tolosan, France,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>SIGENAE, GenPhySE, Université de Toulouse, INRAE, ENVT, Castanet Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2BKlopp%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Klopp%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChristophe%2BKlopp%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7126-5477" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Christophe Klopp</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Céline Roques </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9line%2BRoques%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roques%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC%25C3%25A9line%2BRoques%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Olivier Bouchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BBouchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouchez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlivier%2BBouchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Cécile Donnadieu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>INRAE, US 1426, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C%C3%A9cile%2BDonnadieu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Donnadieu%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AC%25C3%25A9cile%2BDonnadieu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5041-6843" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Cécile Donnadieu</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Celia Barrachina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>MGX, Biocampus Montpellier, CNRS, INSERM, University of Montpellier, Montpellier, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Celia%2BBarrachina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barrachina%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACelia%2BBarrachina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Laurent Journot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>MGX, Biocampus Montpellier, CNRS, INSERM, University of Montpellier, Montpellier, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Laurent%2BJournot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Journot%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALaurent%2BJournot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mari Kawaguchi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Materials and Life Sciences, Faculty of Science and Technology, Sophia University, 7-1 Kioi-cho Chiyoda-ku, Tokyo 102-8554, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mari%2BKawaguchi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kawaguchi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMari%2BKawaguchi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Shigeki Yasumasu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Materials and Life Sciences, Faculty of Science and Technology, Sophia University, 7-1 Kioi-cho Chiyoda-ku, Tokyo 102-8554, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shigeki%2BYasumasu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yasumasu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShigeki%2BYasumasu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Satoshi Ansai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Laboratory of Bioresources, National Institute for Basic Biology, 38 Nishigonaka, Myodaiji-cho, Okazaki 444-8585, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Satoshi%2BAnsai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ansai%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASatoshi%2BAnsai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2683-0160" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Satoshi Ansai</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Kiyoshi Naruse </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Laboratory of Bioresources, National Institute for Basic Biology, 38 Nishigonaka, Myodaiji-cho, Okazaki 444-8585, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiyoshi%2BNaruse%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Naruse%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiyoshi%2BNaruse%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Koji Inoue </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Atmosphere and Ocean Research Institute, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8564, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Koji%2BInoue%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Inoue%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKoji%2BInoue%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Chuya Shinzato </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§</span>Atmosphere and Ocean Research Institute, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8564, Japan,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chuya%2BShinzato%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shinzato%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChuya%2BShinzato%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Manfred Schartl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>***</span>University of Wuerzburg, Developmental Biochemistry, Biocenter, 97074 Wuerzburg, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>†††</span>Comprehensive Cancer Center Mainfranken, University Hospital, 97080 Wuerzburg, Germany,</div><div class='author-affiliation'><span class='nlm-sup'>‡‡‡</span>Hagler Institute for Advanced Study and Department of Biology, Texas A&M University, College Station, Texas 77843, USA,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Manfred%2BSchartl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schartl%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AManfred%2BSchartl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9882-5948" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Manfred Schartl</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Yann Guiguen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>INRAE, UR 1037 Fish Physiology and Genomics, F-35000 Rennes, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yann%2BGuiguen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guiguen%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYann%2BGuiguen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5464-6219" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yann Guiguen</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Amaury Herpin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§§§</span>INRAE, UR 1037 Fish Physiology and Genomics, F-35000 Rennes, France, and</div><div class='author-affiliation'><span class='nlm-sup'>****</span>State Key Laboratory of Developmental Biology of Freshwater Fish, College of Life Sciences, Hunan Normal University, Changsha, 410081, Hunan, P. R. of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amaury%2BHerpin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Herpin%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmaury%2BHerpin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0630-4027" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amaury Herpin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:amaury.herpin@inra.fr" class="" data-icon-position="" data-hide-link-title="0">amaury.herpin@inra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=907&amp;rft.epage=915&amp;rft.atitle=Genome%20Sequence%20of%20the%20Euryhaline%20Javafish%20Medaka%2C%20Oryzias%20javanicus%3A%20A%20Small%20Aquarium%20Fish%20Model%20for%20Studies%20on%20Adaptation%20to%20Salinity&amp;rft.volume=10&amp;rft.issue=3&amp;rft.date=2020-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Takehana%2C+Yusuke&amp;rft.au=Zahm%2C+Margot&amp;rft.au=Cabau%2C+C%C3%A9dric&amp;rft.au=Klopp%2C+Christophe&amp;rft.au=Roques%2C+C%C3%A9line&amp;rft.au=Bouchez%2C+Olivier&amp;rft.au=Donnadieu%2C+C%C3%A9cile&amp;rft.au=Barrachina%2C+Celia&amp;rft.au=Journot%2C+Laurent&amp;rft.au=Kawaguchi%2C+Mari&amp;rft.au=Yasumasu%2C+Shigeki&amp;rft.au=Ansai%2C+Satoshi&amp;rft.au=Naruse%2C+Kiyoshi&amp;rft.au=Inoue%2C+Koji&amp;rft.au=Shinzato%2C+Chuya&amp;rft.au=Schartl%2C+Manfred&amp;rft.au=Guiguen%2C+Yann&amp;rft.au=Herpin%2C+Amaury"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495848/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495848/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495848/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495848/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495848/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495848/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495848/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495848/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495848/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495848/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495848/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495848/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/3/907" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods and Materials</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/3/907.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/3/907.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/3/907.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
