<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/3/899" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/3/899.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/3/899.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/3/899.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/3/899.atom" />
<meta name="HW.pisa" content="ggg;10/3/899" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Improved Genome Assembly and Annotation of the Soybean Aphid (Aphis glycines Matsumura)" />
<meta name="DC.Identifier" content="10.1534/g3.119.400954" />
<meta name="DC.Date" content="2020-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Mathers. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Aphids are an economically important insect group due to their role as plant disease vectors. Despite this economic impact, genomic resources have only been generated for a small number of aphid species. The soybean aphid ( Aphis glycines Matsumura) was the third aphid species to have its genome sequenced and the first to use long-read sequence data. However, version 1 of the soybean aphid genome assembly has low contiguity (contig N50 = 57 Kb, scaffold N50 = 174 Kb), poor representation of conserved genes and the presence of genomic scaffolds likely derived from parasitoid wasp contamination. Here, I use recently developed methods to reassemble the soybean aphid genome. The version 2 genome assembly is highly contiguous, containing half of the genome in only 40 scaffolds (contig N50 = 2.00 Mb, scaffold N50 = 2.51 Mb) and contains 11% more conserved single-copy arthropod genes than version 1. To demonstrate the utility of this improved assembly, I identify a region of conserved synteny between aphids and Drosophila containing members of the Osiris gene family that was split over multiple scaffolds in the original assembly. The improved genome assembly and annotation of A. glycines demonstrates the benefit of applying new methods to old data sets and will provide a useful resource for future comparative genome analysis of aphids." />
<meta name="DC.Contributor" content="Thomas C. Mathers" />
<meta name="article:published_time" content="2020-03-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Improved Genome Assembly and Annotation of the Soybean Aphid (Aphis glycines Matsumura)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Aphids are an economically important insect group due to their role as plant disease vectors. Despite this economic impact, genomic resources have only been generated for a small number of aphid species. The soybean aphid (&lt;i&gt;Aphis glycines&lt;/i&gt; Matsumura) was the third aphid species to have its genome sequenced and the first to use long-read sequence data. However, version 1 of the soybean aphid genome assembly has low contiguity (contig N50 = 57 Kb, scaffold N50 = 174 Kb), poor representation of conserved genes and the presence of genomic scaffolds likely derived from parasitoid wasp contamination. Here, I use recently developed methods to reassemble the soybean aphid genome. The version 2 genome assembly is highly contiguous, containing half of the genome in only 40 scaffolds (contig N50 = 2.00 Mb, scaffold N50 = 2.51 Mb) and contains 11% more conserved single-copy arthropod genes than version 1. To demonstrate the utility of this improved assembly, I identify a region of conserved synteny between aphids and &lt;i&gt;Drosophila&lt;/i&gt; containing members of the &lt;i&gt;Osiris&lt;/i&gt; gene family that was split over multiple scaffolds in the original assembly. The improved genome assembly and annotation of &lt;i&gt;A. glycines&lt;/i&gt; demonstrates the benefit of applying new methods to old data sets and will provide a useful resource for future comparative genome analysis of aphids.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/03/01" />
<meta name="citation_mjid" content="ggg;10/3/899" />
<meta name="citation_id" content="10/3/899" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/3/899" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/3/899.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/3/899.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/3/899.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400954" />
<meta name="citation_pmid" content="31969427" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="899" />
<meta name="citation_lastpage" content="906" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Thomas C. Mathers" />
<meta name="citation_author_institution" content="Department of Crop Genetics, John Innes Centre, Norwich Research Park, Norwich, Norfolk, NR4 7UH, UK" />
<meta name="citation_author_email" content="thomas.mathers@jic.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8637-3515" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Bansal;citation_author=M. Mian;citation_author=O. Mittapalli;citation_author=A. P. Michel;citation_title=RNA-Seq reveals a xenobiotic stress response in the soybean aphid, Aphis glycines, when fed aphid-resistant soybean.;citation_pages=972;citation_volume=15;citation_year=2014;citation_pmid=25399334;citation_doi=10.1186/1471-2164-15-972" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=N. L. Bray;citation_author=H. Pimentel;citation_author=P. Melsted;citation_author=L. Pachter;citation_title=Near-optimal probabilistic RNA-seq quantification.;citation_pages=525-527;citation_volume=34;citation_year=2016;citation_pmid=27043002;citation_doi=10.1038/nbt.3519" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Camacho;citation_author=G. Coulouris;citation_author=V. Avagyan;citation_author=N. Ma;citation_author=J. Papadopoulos;citation_title=BLAST+: architecture and applications.;citation_pages=421-421;citation_volume=10;citation_year=2009;citation_pmid=20003500;citation_doi=10.1186/1471-2105-10-421" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. J. P. Chaisson;citation_author=J. Huddleston;citation_author=M. Y. Dennis;citation_author=P. H. Sudmant;citation_author=M. Malig;citation_title=Resolving the complexity of the human genome using single-molecule sequencing.;citation_pages=608-611;citation_volume=517;citation_year=2015;citation_pmid=25383537;citation_doi=10.1038/nature13907" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. J. Chaisson;citation_author=G. Tesler;citation_title=Mapping single molecule sequencing reads using Basic Local Alignment with Successive Refinement (BLASR): Theory and Application.;citation_pages=238-238;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22988817;citation_doi=10.1186/1471-2105-13-238" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Chakraborty;citation_author=J. G. Baldwin-Brown;citation_author=A. D. Long;citation_author=J. J. Emerson;citation_title=Contiguous and accurate de novo assembly of metazoan genomes with modest long read coverage.;citation_volume=44;citation_year=2016;citation_pmid=26621913;citation_doi=10.1093/nar/gkv1289" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Chakraborty;citation_author=N. W. VanKuren;citation_author=R. Zhao;citation_author=X. Zhang;citation_author=S. Kalsow;citation_title=Hidden genetic variation shapes the structure of functional elements in Drosophila.;citation_pages=20-25;citation_volume=50;citation_year=2018;citation_doi=10.1038/s41588-017-0010-y" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=W. Chen;citation_author=S. Shakir;citation_author=M. Bigham;citation_author=A. Richter;citation_author=Z. Fei;citation_title=Genome sequence of the corn leaf aphid (Rhopalosiphum maidis Fitch).;citation_pages=giz033;citation_volume=8;citation_year=2019;citation_doi=10.1093/gigascience/giz033" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Current Opinion in Biotechnology;citation_journal_abbrev=Current Opinion in Biotechnology;citation_author=N. Ferry;citation_author=M. G. Edwards;citation_author=J. A. Gatehouse;citation_author=A. Gatehouse;citation_title=Plant-insect interactions: molecular approaches to insect resistance.;citation_pages=155-161;citation_volume=15;citation_year=2004;citation_issue=2;citation_pmid=15081055;citation_doi=10.1016/j.copbio.2004.01.008" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. J. Hawthorne;citation_author=S. Via;citation_title=Genetic linkage of ecological specialization and reproductive isolation in pea aphids.;citation_pages=904-907;citation_volume=412;citation_year=2001;citation_issue=6850;citation_pmid=11528477;citation_doi=10.1038/35091062" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. J. Hoff;citation_author=S. Lange;citation_author=A. Lomsadze;citation_author=M. Borodovsky;citation_author=M. Stanke;citation_title=BRAKER1: Unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS.;citation_volume=32;citation_year=2015;citation_pmid=26559507;citation_doi=10.1093/bioinformatics/btv661" />
<meta name="citation_reference" content="citation_title=Gene Prediction: Methods and Protocols;citation_title=Whole-genome annotation with BRAKER;citation_pages=65-95;citation_year=2019" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=S. A. Hogenhout;citation_author=J. I. B. Bos;citation_title=Effector proteins that modulate plant--insect interactions.;citation_pages=422-428;citation_volume=14;citation_year=2011;citation_issue=4;citation_pmid=21684190;citation_doi=10.1016/j.pbi.2011.05.003" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_title=Genome sequence of the pea aphid Acyrthosiphon pisum.;citation_pages=e1000313-e1000313;citation_volume=8;citation_year=2010;citation_issue=2;citation_pmid=20186266;citation_doi=10.1371/journal.pbio.1000313" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. Jaquiéry;citation_author=J. Peccoud;citation_author=T. Ouisse;citation_author=F. Legeai;citation_author=N. Prunier-Leterme;citation_title=Disentangling the causes for faster-X evolution in aphids.;citation_pages=507-520;citation_volume=10;citation_year=2018;citation_doi=10.1093/gbe/evy015" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Jaquiéry;citation_author=C. Rispe;citation_author=D. Roze;citation_author=F. Legeai;citation_author=G. Le Trionnaire;citation_title=Masculinization of the X chromosome in the pea aphid.;citation_pages=e1003690;citation_volume=9;citation_year=2013;citation_pmid=23950732;citation_doi=10.1371/journal.pgen.1003690" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=X. Jiang;citation_author=Q. Zhang;citation_author=Y. Qin;citation_author=H. Yin;citation_author=S. Zhang;citation_title=A chromosome-level draft genome of the grain aphid Sitobion miscanthi.;citation_volume=8;citation_year=2019;citation_doi=10.1093/gigascience/giz101" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. P. Johnson;citation_author=C. H. Dietrich;citation_author=F. Friedrich;citation_author=R. G. Beutel;citation_author=B. Wipfler;citation_title=Phylogenomics and the evolution of hemipteroid insects.;citation_volume=115;citation_year=2018;citation_pmid=30478043;citation_doi=10.1073/pnas.1815820115" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=I. Julca;citation_author=M. Marcet-houben;citation_author=F. Cruz;citation_author=C. Vargas-chavez;citation_author=J. Spencer;citation_title=Phylogenomics identifies an ancestral burst of gene duplications predating the diversification of Aphidomorpha.;citation_year=2019;citation_doi=10.1093/molbev/msz261" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=D. Kim;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=HISAT: A fast spliced aligner with low memory requirements.;citation_pages=357-360;citation_volume=12;citation_year=2015;citation_pmid=25751142;citation_doi=10.1038/nmeth.3317" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Koren;citation_author=B. P. Walenz;citation_author=K. Berlin;citation_author=J. R. Miller;citation_author=N. H. Bergman;citation_title=Canu: Scalable and accurate long-read assembly via adaptive κ-mer weighting and repeat separation.;citation_volume=27;citation_year=2017;citation_pmid=28298431;citation_doi=10.1101/gr.215087.116" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Koutsovoulos;citation_author=S. Kumar;citation_author=D. R. Laetsch;citation_author=L. Stevens;citation_author=J. Daub;citation_title=No evidence for extensive horizontal gene transfer in the genome of the tardigrade Hypsibius dujardini.;citation_volume=113;citation_year=2016;citation_pmid=27035985;citation_doi=10.1073/pnas.1600338113" />
<meta name="citation_reference" content="Krueger, F., 2015 Trim galoreA wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files. A wrapper tool around Cutadapt FastQC to consistently apply Qual. Adapt. trimming to FastQ files. www.bioinformatics.babraham.ac.uk/projects/trim_galore/." />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=S. Kumar;citation_author=M. Jones;citation_author=G. Koutsovoulos;citation_author=M. Clarke;citation_author=M. Blaxter;citation_title=Blobology: exploring raw genome data for contaminants, symbionts, and parasites using taxon-annotated GC-coverage plots.;citation_pages=1-12;citation_volume=4;citation_year=2013;citation_doi=10.3389/fgene.2013.00024" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=D. R. Laetsch;citation_author=M. L. Blaxter;citation_title=BlobTools: Interrogation of genome assemblies.;citation_pages=1287;citation_volume=6;citation_year=2017;citation_doi=10.12688/f1000research.12232.1" />
<meta name="citation_reference" content="Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. https://arxiv.org/abs/1303.3997" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=Minimap2: pairwise alignment for nucleotide sequences.;citation_pages=3094-3100;citation_volume=34;citation_year=2018;citation_pmid=29750242;citation_doi=10.1093/bioinformatics/bty191" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=D. Liu;citation_author=M. Hunt;citation_author=I. J. Tsai;citation_title=Inferring synteny between genome assemblies: A systematic evaluation.;citation_pages=26;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12859-018-2026-4" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Lomsadze;citation_author=P. D. Burns;citation_author=M. Borodovsky;citation_title=Integration of mapped RNA-Seq reads into automatic training of eukaryotic gene finding algorithm.;citation_volume=42;citation_year=2014;citation_pmid=24990371;citation_doi=10.1093/nar/gku557" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Mapleson;citation_author=G. G. Accinelli;citation_author=G. Kettleborough;citation_author=J. Wright;citation_author=B. J. Clavijo;citation_title=KAT: A K-mer analysis toolkit to quality control NGS datasets and genome assemblies.;citation_pages=574-576;citation_volume=33;citation_year=2017;citation_pmid=27797770;citation_doi=10.1093/bioinformatics/btw663" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=G. Marçais;citation_author=A. L. Delcher;citation_author=A. M. Phillippy;citation_author=R. Coston;citation_author=S. L. Salzberg;citation_title=MUMmer4: A fast and versatile genome alignment system.;citation_pages=e1005944;citation_volume=14;citation_year=2018;citation_pmid=29373581;citation_doi=10.1371/journal.pcbi.1005944" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=T. C. Mathers;citation_author=Y. Chen;citation_author=G. Kaithakottil;citation_author=F. Legeai;citation_author=S. T. Mugford;citation_title=Rapid transcriptional plasticity of duplicated gene clusters enables a clonally reproducing aphid to colonise diverse plant species.;citation_pages=27;citation_volume=18;citation_year=2017;citation_doi=10.1186/s13059-016-1145-3" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=F. McLean;citation_author=D. Berger;citation_author=D. Laetsch;citation_author=H. Schwartz;citation_author=M. Blaxter;citation_title=Improving the annotation of the Heterorhabditis bacteriophora genome.;citation_volume=7;citation_year=2018;citation_doi=10.1093/gigascience/giy034" />
<meta name="citation_reference" content="citation_journal_abbrev=J Proteomics;citation_author=S. J. Nicholson;citation_author=S. D. Hartson;citation_author=G. J. Puterka;citation_title=Proteomic analysis of secreted saliva from Russian Wheat Aphid (Diuraphis noxia Kurd.) biotypes that differ in virulence to wheat.;citation_pages=2252-2268;citation_volume=75;citation_year=2012;citation_issue=7;citation_pmid=22348819;citation_doi=10.1016/j.jprot.2012.01.031" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=S. J. Nicholson;citation_author=M. L. Nickerson;citation_author=M. Dean;citation_author=Y. Song;citation_author=P. R. Hoyt;citation_title=The genome of Diuraphis noxia, a global aphid pest of small grains.;citation_pages=429;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1525-1" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=L. C. Orantes;citation_author=W. Zhang;citation_author=M. A. R. Mian;citation_author=A. P. Michel;citation_title=Maintaining genetic diversity and population panmixia through dispersal and not gene flow in a holocyclic heteroecious aphid species.;citation_pages=127-134;citation_volume=109;citation_year=2012;citation_issue=2;citation_pmid=22549514;citation_doi=10.1038/hdy.2012.21" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Peccoud;citation_author=A. Ollivier;citation_author=M. Plantegenest;citation_author=J.-C. Simon;citation_title=A continuum of genetic divergence from sympatric host races to species in the pea aphid complex.;citation_volume=106;citation_year=2009;citation_pmid=19380742;citation_doi=10.1073/pnas.0811117106" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Entomol.;citation_author=J. Peccoud;citation_author=J. C. Simon;citation_title=The pea aphid complex as a model of ecological speciation.;citation_pages=119-130;citation_volume=35;citation_year=2010;citation_doi=10.1111/j.1365-2311.2009.01147.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. N. Price;citation_author=P. S. Dehal;citation_author=A. P. Arkin;citation_title=FastTree 2--approximately maximum-likelihood trees for large alignments.;citation_pages=e9490-e9490;citation_volume=5;citation_year=2010;citation_issue=3;citation_pmid=20224823;citation_doi=10.1371/journal.pone.0009490" />
<meta name="citation_reference" content="citation_journal_title=Insect Biochem. Mol. Biol.;citation_author=Q. Quan;citation_author=X. Hu;citation_author=B. Pan;citation_author=B. Zeng;citation_author=N. Wu;citation_title=Draft genome of the cotton aphid Aphis gossypii.;citation_pages=25-32;citation_volume=105;citation_year=2019;citation_doi=10.1016/j.ibmb.2018.12.007" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdóttir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=F. J. Sedlazeck;citation_author=H. Lee;citation_author=C. A. Darby;citation_author=M. C. Schatz;citation_title=Piercing the dark matter: bioinformatics of long-range sequencing and mapping.;citation_volume=19;citation_year=2018;citation_doi=10.1038/s41576-018-0003-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. J. Sedlazeck;citation_author=P. Rescheneder;citation_author=A. Von Haeseler;citation_title=NextGenMap: fast and accurate read mapping in highly polymorphic genomes.;citation_volume=29;citation_year=2013;citation_pmid=23975764;citation_doi=10.1093/bioinformatics/btt468" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=N. Shah;citation_author=D. R. Dorer;citation_author=E. N. Moriyama;citation_author=A. C. Christensen;citation_title=Evolution of a large, conserved, and syntenic gene family in insects. G3 Genes, Genomes;citation_pages=313-319;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=C. R. Smith;citation_author=C. Morandin;citation_author=M. Noureddine;citation_author=S. Pant;citation_title=Conserved roles of Osiris genes in insect development, polymorphism and protection.;citation_pages=516-529;citation_volume=31;citation_year=2018;citation_doi=10.1111/jeb.13238" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Stanke;citation_author=M. Diekhans;citation_author=R. Baertsch;citation_author=D. Haussler;citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding.;citation_volume=24;citation_year=2008;citation_pmid=18218656;citation_doi=10.1093/bioinformatics/btn013" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Protoc Bioinformatics;citation_author=M. Tarailo-Graovac;citation_author=N. Chen;citation_title=Using RepeatMasker to identify repetitive elements in genomic sequences.;citation_pages=Unit 4.10-Unit 4.10;citation_volume=Chapter 4;citation_year=2009;citation_pmid=19274634;citation_doi=10.1002/0471250953.bi0410s25" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=P. Thorpe;citation_author=C. M. Escudero-Martinez;citation_author=P. J. A. A. Cock;citation_author=S. Eves-Van Den Akker;citation_author=J. I. B. B. Bos;citation_title=Shared transcriptional control and disparate gain and loss of aphid parasitism genes.;citation_pages=2716-2733;citation_volume=10;citation_year=2018;citation_doi=10.1093/gbe/evy183" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdóttir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2012;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Thurmond;citation_author=J. L. Goodman;citation_author=V. B. Strelets;citation_author=H. Attrill;citation_author=L. S. Gramates;citation_title=FlyBase 2.0: the next generation.;citation_pages=D759-D765;citation_volume=47;citation_year=2018;citation_doi=10.1093/nar/gky1003" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Pest Manag.;citation_author=K. J. Tilmon;citation_author=E. W. Hodgson;citation_author=M. E. O’Neal;citation_author=D. W. Ragsdale;citation_title=Biology of the soybean aphid, Aphis glycines (Hemiptera: Aphididae) in the United States.;citation_pages=1-7;citation_volume=2;citation_year=2011;citation_doi=10.1603/IPM10010" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=T. J. Treangen;citation_author=S. L. Salzberg;citation_title=Repetitive DNA and next-generation sequencing: computational challenges and solutions.;citation_pages=36-46;citation_volume=13;citation_year=2011;citation_pmid=22124482;citation_doi=10.1038/nrg3117" />
<meta name="citation_reference" content="citation_author=H. F. Van Emden;citation_author=R. Harrington;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. Veltri;citation_author=M. M. Wight;citation_author=J. A. Crouch;citation_title=SimpleSynteny: a web-based tool for visualization of microsynteny across multiple species.;citation_volume=44;citation_year=2016;citation_pmid=27141960;citation_doi=10.1093/nar/gkw330" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Wang;citation_author=H. Tang;citation_author=J. D. Debarry;citation_author=X. Tan;citation_author=J. Li;citation_title=MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity.;citation_volume=40;citation_year=2012;citation_pmid=22217600;citation_doi=10.1093/nar/gkr1293" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R. M. Waterhouse;citation_author=M. Seppey;citation_author=F. A. Simao;citation_author=M. Manni;citation_author=P. Ioannidis;citation_title=BUSCO applications from quality assessments to gene prediction and phylogenomics.;citation_pages=543-548;citation_volume=35;citation_year=2018;citation_pmid=29220515;citation_doi=10.1093/molbev/msx319" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=N. I. Weisenfeld;citation_author=S. Yin;citation_author=T. Sharpe;citation_author=B. Lau;citation_author=R. Hegarty;citation_title=Comprehensive variation discovery in single human genomes.;citation_pages=1350-1355;citation_volume=46;citation_year=2014;citation_pmid=25326702;citation_doi=10.1038/ng.3121" />
<meta name="citation_reference" content="citation_journal_title=Insect Biochem. Mol. Biol.;citation_author=J. A. Wenger;citation_author=B. J. Cassone;citation_author=F. Legeai;citation_author=J. S. Johnston;citation_author=R. Bansal;citation_title=Whole genome sequence of the soybean aphid, Aphis glycines.;citation_year=2017;citation_doi=10.1016/j.ibmb.2017.01.005" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Yassin;citation_author=V. Debat;citation_author=H. Bastide;citation_author=N. Gidaszewski;citation_author=J. R. David;citation_title=Recurrent specialization on a toxic fruit in an island Drosophila population.;citation_volume=113;citation_year=2016;citation_pmid=27044093;citation_doi=10.1073/pnas.1522559113" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=C. Ye;citation_author=C. M. Hill;citation_author=S. Wu;citation_author=J. Ruan;citation_author=Z. Ma;citation_title=DBG2OLC: Efficient assembly of large genomes using long erroneous reads of the third generation sequencing technologies.;citation_pages=1-9;citation_volume=6;citation_year=2016;citation_pmid=27503664;citation_doi=10.1038/s41598-016-0001-8" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=B. H. Zhu;citation_author=J. Xiao;citation_author=W. Xue;citation_author=G. C. Xu;citation_author=M. Y. Sun;citation_title=P_RNA_scaffolder: A fast and accurate genome scaffolder using paired-end RNA-sequencing reads.;citation_pages=175;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-4567-3" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Improved Genome Assembly and Annotation of the Soybean Aphid (Aphis glycines Matsumura)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="twitter:description" content="Aphids are an economically important insect group due to their role as plant disease vectors. Despite this economic impact, genomic resources have only been generated for a small number of aphid species. The soybean aphid ( Aphis glycines Matsumura) was the third aphid species to have its genome sequenced and the first to use long-read sequence data. However, version 1 of the soybean aphid genome assembly has low contiguity (contig N50 = 57 Kb, scaffold N50 = 174 Kb), poor representation of conserved genes and the presence of genomic scaffolds likely derived from parasitoid wasp contamination. Here, I use recently developed methods to reassemble the soybean aphid genome. The version 2 genome assembly is highly contiguous, containing half of the genome in only 40 scaffolds (contig N50 = 2.00 Mb, scaffold N50 = 2.51 Mb) and contains 11% more conserved single-copy arthropod genes than version 1. To demonstrate the utility of this improved assembly, I identify a region of conserved synteny between aphids and Drosophila containing members of the Osiris gene family that was split over multiple scaffolds in the original assembly. The improved genome assembly and annotation of A. glycines demonstrates the benefit of applying new methods to old data sets and will provide a useful resource for future comparative genome analysis of aphids." />
<meta name="og-title" property="og:title" content="Improved Genome Assembly and Annotation of the Soybean Aphid (Aphis glycines Matsumura)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/3/899" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Aphids are an economically important insect group due to their role as plant disease vectors. Despite this economic impact, genomic resources have only been generated for a small number of aphid species. The soybean aphid ( Aphis glycines Matsumura) was the third aphid species to have its genome sequenced and the first to use long-read sequence data. However, version 1 of the soybean aphid genome assembly has low contiguity (contig N50 = 57 Kb, scaffold N50 = 174 Kb), poor representation of conserved genes and the presence of genomic scaffolds likely derived from parasitoid wasp contamination. Here, I use recently developed methods to reassemble the soybean aphid genome. The version 2 genome assembly is highly contiguous, containing half of the genome in only 40 scaffolds (contig N50 = 2.00 Mb, scaffold N50 = 2.51 Mb) and contains 11% more conserved single-copy arthropod genes than version 1. To demonstrate the utility of this improved assembly, I identify a region of conserved synteny between aphids and Drosophila containing members of the Osiris gene family that was split over multiple scaffolds in the original assembly. The improved genome assembly and annotation of A. glycines demonstrates the benefit of applying new methods to old data sets and will provide a useful resource for future comparative genome analysis of aphids." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
    <title>Improved Genome Assembly and Annotation of the Soybean Aphid (Aphis glycines Matsumura) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"c0OQNxEWYWTCHl5Ug19FYk-HjUertDhJbJwcWNjxIpA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495869","apath":"\/ggg\/10\/3\/899.atom","pisa":"ggg;10\/3\/899","ac":{"\/ggg\/10\/3\/899.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/3\/899.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/3\/899"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/3\/899"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/3\/899?rss=1":true,"\/content\/10\/3\/899":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229Iix7MlZrR9YieYYu7lKVY1AR05mYHUbrHJJFFezszEx+qQJJNao4rkBCYSyL6FlNeVIN5SUDxCfG1Xlx4MU1dPZTy4zAgWxZJVIwN4Zvf25gWGicD6xwl3mXJG86JgTHCZPkv8y2bjPbRyjUjMQMBGQufCbx4K0kdatH\\\\\\\/rULsf6UW9BiETbLaD6rD2DwppuIVsr5vXId\\\\\\\/KIPMbSRuOgMi02o\\\\\\\/gjomU882PIGPyJVxE\\\\\\\/kNllN7b6AsfwgyRuMGM9tniwz4H6HqPmlFGaYOb9+I1Ihqt1XIblFp5KJ0pNHWfaMADrXt2qtzYucIxKvm9DxJGps3\\\\\\\/uD2Mimyb8X2QSJhPf6rqyuPa2Jb6NWCZ1tTM\\\\\\\/EreAoG\\\\\\\/6MemU1ytec3tj+tIsgyX8T7euHALpzDnxKHSLguOlsJbTjFDAbqhilNWkeIM0EfXC0MUaIROtedHPu7pT9a3QIcjLpx0ArQewhgyNsYoleXBEqxZwJIotyWJ6S0Th8ms8m6oVT1RgajPdyPpNSUqwoaezNmtkg6OeI31bzW8n8muVM0RePpIQmpvthft24sdbJsp76eXyj1\\\\\\\/r2J5lD8itCilZWxgj6I1on0UpoTddOpvPN8xbcBCZKBYbumjBNz6djcOIh\\\\\\\/BSwg2ZB32ZDKbjq3NAZmkjSjeV\\\\\\\/toNEO4XY9P6qGyhGw3YGUyWOglTSQtts7jE2SCJrVv5FEoKa4f7y4Re7vKJRPZHgboP6ts0yite\\\\\\\/tfFqD9dL4AD43vnxiWKNbr44aB78n6kI8Dl5RHkQt67EVr21yyyCdZf9tSb3tGTX8IlivHezkdpi\\\\\\\/B+f2r0mUqEpVnZYi4vbEcGCr4ePxryNP6MpcCONFwM8462a6doCXR\\\\\\\/MWX26msx0oN8ctaEAgVFhWl9CWFfY+YyQ1miv7mTfr1QhtPu8f9b7nfv0sR8YXyYmGcYcH70pHENT7AHA1xwTbiuWbva0TrGdKZAd3U6qto7ACNOUd\\\\\\\/GbgBgcE+QIYqe0FB6O4Vqxw\\\\\\\/i+Oe466nSRdvj1EDWrqx\\\\\\\/ZG06DsVI2jo1EiIB3bjeGwNEpfs52PKd0M+lIJhQyUwbVUaGEkd4oxf+FgfK9jJQStz4\\\\\\\/1EX47AOFsu9w474QQTH8TPs2BGBuHExiJ2LAgg0UiLoIhiUS\\\\\\\/3EPcBeriTfhvrNiPabOaE36IiUVlDavvLc7xOT1sYYAIBSME0FVTWjAwPHSfVL3dIQkFELLfUkgZYPEAxz9g6L8u70xprbCGJy5RDE4GlDHWyGwPxtUUApflNvGtW0P7V7vDfE\\\u0022,\\\u0022iv\\\u0022:\\\u0022yBjjzZp6iiA9o47NTPWwBg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cae7fddb32d0445b43255c87524dbca3097015324773e607122c3ffbf302ad98\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002261ete8JiM1dUpHdMagALaNT63BfNIo1X\\\\\\\/2dJNQMiB02cKI4JXSSKGzOPhXPMWdgY8NgM9f+4fwbsketV3iqNkn+uA3rvlYXC+LY4ut0xttx4Tfa8I87aOXADMLuwIxniZ8Yt+OOwLAdvi14UeL1XxLytfXM39nSIBhBs6ZDSY2n+OyVlP4hEo1kq2JkcXCxR+NPbpKtB8GlPoyO0Vg2kOfXeR3qctGAgLfZRE7HwIH2fieQqHwIJO8TqIaXidtSTbe9WddQp5WGDz1xAPoc50YQpTp\\\\\\\/e2l\\\\\\\/YnugQSzHbtLIOJN\\\\\\\/HX\\\\\\\/AL7pBzTerfVH42vXLe7Ohbaq7WxAXLEBUs\\\\\\\/ElYdT62inF+3gVN0VaQKPDaBre5lPREGeHRDNquhXX94xwaY4WGTH5ww7VkjrR4\\\\\\\/dkAUuy9SHmup57mk1lHpD467WtDH3wecVLyHCLeVSqTxdoww0Mfsoa9BIUVSvJOVMUvLvbHW2PGyjkZCOjgrwH3LKGFrB9ATGI5xBpDwzz0az1o\\\\\\\/fyLqIvoWHoQj3rMTXtGPVGuIFi3ThAgg6GyB4gwOannXeaHBFcupP4z5H28lhC9XEhQOJ5Lo2rG0q\\\\\\\/l09OjuQrBobRvVN61JB4TuIT\\\\\\\/ZjdPjbsnRBEgDzgv\\\\\\\/eZexGQZMStWnl8RwEANX+opU+PyS2dGCcK9\\\\\\\/T2+Lrwif9GiJJt2fCWCkD8lYJ0YUlQUWY9NTMARthOZsNvxFXQXEDdwBeudYOTQ7K6mdP3l309eMaIuTtS8pm4YErgfeyG0+JNpF3fV4PXxaqcea7\\\\\\\/L6GlpbC\\\\\\\/4H54zZnFtKUueiXvP7T884ELsXHxrJpYW85sDnCbV3WCeQljSemvKUxz6nZQO8ncQjyyi4zFKca9vZqNqPL6V9ogo6wrJQg2SfTrEEfV32PwdhNnM9cE\\\\\\\/5fDl25X67KzHnwOmNAZOZ7Pz1Mmc+zwZUzEdjh1cDYDaad2Pn7XkNUqVarCqHA9dBCh7pQ8\\\\\\\/Ogi1KRCzNLZEstuSuXfyOLjGLSeeFjkoGpipSZI1iCbtb\\\\\\\/vmUCf\\\\\\\/9\\\\\\\/rX5rAQM4dozv1frPn5nCsyrWSMFcNFJWgN1cCBIWyoBL82xbVBU9MwGg3895LCvamn\\\\\\\/h\\\\\\\/O9qDQBABDbKEupVGmTyHHVyhQUea9k7VZMWEdlsvRs8ud4mAKE6QOaTwk+agqQRe2wrYOElyP454YiW5CkxWyoCdfMah7RLxV5dMi2uTNnlyalE4htkvqhlGgiwqUNtTDz0RtJ2MOjYOw4vsOVniy2WL5rqF84n1lI+IDOGOgFXZ8NukRtm8eSvENmKM7TXrVQEl0IQ2s8FQdmDq4k4AigZE9Gcvod9XJydUHWizSmo\\\\\\\/ROSTvEoGQx6olztBJvpHsEEx7jfhwPdwJwlU4wMar5jIsCYplj+xV6J\\\\\\\/WjoboleDCxPSHD+cNLWLOMIc6KWL+beCK+KF5faYIqpqpGz9pTk9n4tHOSyGXK7AUfQjvmynvWOrU3GI6z8DxrN4QgXqnOa4dP04yUhZX6MbAAJpZrSb3OI0u0oWylmG\\\\\\\/Sg1Tb3oDCh2eIJewjntKPVKZB5gvlxA63lYetDcXPbIk+F8cZSWuTNq2JraucLtoEEUvGoSNPxU1zfR4ojXPG\\\\\\\/q3Zh2A8bYW0p+0cRlzv8iPm1wp0\\\\\\\/4gDidSfqhn9H2qN30tKJCRyEHMCklVrlY8ygfOK\\\\\\\/2Lbv5JZLzXEL3CI5S1nHAFeTCSfs4YuFsmNLkJWAhC4OWsLoTXKuUOgmx5kislCjC8OXvnPH1UcUxmbxFDuE7LlHTQavijIBxAfyQIjHcGKQL7hNl\\\\\\\/oB3YtysQl1+JUELyQ\\\\\\\/+kOgKiKfPAxoYrd\\\\\\\/5P4nsazqFsqgXzNmIPl8a6FTvocSnF\\\\\\\/yujSAUnsTnLNru1jC7qSbpX54MELiLvk\\\\\\\/GVmMTgL8NgpZUbwBoFFzZUwzzuCa0zgPUPkZ5375XolWOR0S2hArs1ensuD5b3qGVC0sahNMFPzb3h\\\\\\\/KtwSqQyIV2tpoWmO0gFso4OoiurkKWJ5Zf8\\\\\\\/iuhHMRbKJ5VpK7BlukO69Zrra0oxOciXcTgM9PJrgQXVFz4JawWAbpeHNUyNr\\\\\\\/r6PVGoTzGZNb6zjX2bxnU4NVGMxKF7eFpe+zgS04JY7Ch+\\\\\\\/BJY74wCqPHcg4zZNCRDB02LYt5D8QvGcobw1VcC4SvNggSyJ3B4yOJNAnJCf\\\\\\\/AkpfOygEcraCacdyPyHRrRLRHsv6R\\\\\\\/h5uK8MSTSkUhUOk9RD3l8bmq8xYvn2IVEdz6hnEtWy5RAXPVJ1KSDWv1dXwHasGxLcYLjqdwqQMnuTqjuS3V6AdQR\\\\\\\/CIqTC3cVxVBYX83J34hEeh+OT\\\\\\\/qBgm2ZnAbwx94JtEXPaSMXaSSxgZhZayptovhGzCCY4Xmzp8vtqCJIAD1ud88pYpjKsRn5m8PwHFtLERfxthWdsBVl33Jnou6eu6O9qMDXPZWi5SsUuzl9UWkot3TNqGV0bJbTRuemlVVKbLXmlMYdFFFBdRXPIm4inbiTeO55wD6BFdjZJMNQrDHA9uk3bpH0OW4EARSIyxNa\\\\\\\/rXKAZQT\\\\\\\/DDPJc5iaUCVAfRa1gzHcE30Nylff1KaT1FybMsY2VxlEMkXcw3by461MHv\\\\\\\/fQ95JlYMyN5HrJv4EgBugRwl\\\\\\\/nl\\\\\\\/X7eY+yArLJ76TI9FXjZ389uSbyE7QyB7fuhfJUO9+1iiubaSJKDrBnag\\\\\\\/a3mZMTA+Y9s0n4177bdKzteLpDu30ERd0n9uvVk\\\\\\\/R2qn1\\\\\\\/l3Gz26YV54vj0Z9yvLzC49dQa0TT9x3mWq52pA8uIBErRzNZZgJC9tqHI7EfxCVt726N75vrR5UX7SBuYKcUI6fJlWXTII\\\\\\\/Ia0P0NDSSwgJw1UY3kEE2z\\\\\\\/v\\\\\\\/Xlbu0B\\\\\\\/Il1JVtbCWIJmaXYrKVKFfQGDvEerTN2k6kWOoQYWOdA4+udGc\\\\\\\/S1K5ekxCYWOT4hL9auJ0OFwBoz\\\\\\\/irbC3rNr67pcDldvjkT7koh0Guc1JiKipyEaOoTyATCJroF9cq+cJJswcGZAVB1GHznB4zBVLhfSIEgEHV3Q6XZPpt8QCiqB5E9vzW1Nu64oYyryCVMd\\\\\\\/3dxB1\\\\\\\/DenVLetsCqmXNYrXuXPWM0hfVMhgAPZHnP1GRLhJZaxG5A1aGeysDeYVJ2NtculC627dPQRWVqrq2eonZsSIOc2evz2OmqzHA+4V+iHcS78us6Wxj4V\\\\\\\/k6rDmwjLxDHUTdDiIwB6jYXq5D+6fcCXgHnvkSsg\\\\\\\/0BzRQQhds75gJxSJXZWUskzi9jRrHIUCXF+33XOSSwJZB8AvtPPGxbHBqxrMMKHHjpvf1aKHTxO6xWW7hheN+w2QvMZm1MRN6Q9goKGa0q+xwbikMwjs4zd733LN7X+vkmfh7K8uw7nTblOYk\\\\\\\/JK10vczIXPG7JFYYw5A3RTqNoDk8HxHnQ2NFf0\\\\\\\/996TKHTsyFeve0CncYNklcQ50sRavzZoFvXOZADVhS5SdEYajQ2\\\\\\\/IkcqPTSx8Nny7a45tzbtNgEZ7citFaY\\\\\\\/oTKvN91nlElpi6jaxPchBXA\\\\\\\/uj74bCdJSTpkuVt262B+hYOFIpkAQlWdJAX\\\\\\\/hpIOmB4lBbH5d+353ut57bWgL3rIZmpfn+gsj99E+cKcGLxMngjBKlEtE+3LZ0Ho2CW2VmQiz9fnkgTkQLhNBy8GvkbpXhadRS0WiknVHZ3r\\\\\\\/GZ43tvDuWm3pW97k1pqqn4ivsqHSNiFHQ+TGOUseGHckbWUXOEaZIeDSK3rTYaZYDfylEcnJfBP8pod0RVXquH0XKPa2KincQItuTkLsTJizTdrQNXGSgTLxFpR2T8WtTx9eXJNW3VXxgZa47SliLW3TAsk1e1xJ01un2HSDB0J8ixBrkFaXVCkfEvvMJ8ve+0Ae7UKtP3fkai8sDBME5L763Bpr65r\\\\\\\/XrynkYnHL5X6Uk6TAoEl4NugpIb0fUQs0BkY7xAk2Ss3LaB\\\\\\\/xIr34aGYDyEAAdYFr\\\\\\\/sEkYPhKCnvxWuLfRjQKNISaq0HdZR3\\\\\\\/e8H9PUysMSyGTYZXn03tzC1tgrr7Md2gcwQYp6Q555H86Ia+wvj8KoVHo5qg47VoiFKK\\\\\\\/ikg28MoOgDjieTMULBVDdpZKNCwFw1\\\\\\\/JFDtr1stUl\\\\\\\/KyC5boF3an4s6gZfhUopChK1wAcFFp33ADN4\\\\\\\/1U8gubAKazYXBqoJWoHVhZTsBC7wLMMGCnzv43pTRO3Zx8n+nANWbZd\\\\\\\/+\\\\\\\/bJ2tr+AL3eI5wZb+u5mg+y7mkbPojiwaXjA7+aX9YUGrk89CuT2mlBtiEE2OfhPaUaoDI2kgEUMKzH2EwOoLGr1fPLfv0yTrTNP0kJVaJu1zDYjfAqRz+xQWZiEqCPldvwlrLsvOAxmjHKfbrAj7WmQycWieMsQde1FYgkCjCl\\\\\\\/Yg9+69ZrOod3qc9j7Og3cR\\\\\\\/6yY4zMyNScUuoGAnlWKmQxJIribT1yGqppNGirufCZyLXdgrGo08+jEQ1TkcNNgRQv+9MrdMmZ0mhSI4wVLgTp04425090ae8n9p7ype2ztK+h4nSwsPQ\\\\\\\/DsCfaZzylQM1nFaSGjpzTOumfBaPvkxGhJoB7l9HEIskjVsuupCrPJN4oBLoBcoKeTrbIOd7BjHpnjxLFh3R1gVoJjgMC5mApdkyNGISI\\\\\\\/q89rizyEi6znaQbzmW0Ynxo1N+eha15prW8XvUzCrpu+zrdPCcje258BV2KFQMg+FiG2Zk6SpGM24B\\\\\\\/dz4ZdOB4d2887fViDy\\\\\\\/hEnOTxkGVbaNTu2OG+2b\\\\\\\/Z5vxxvrSKC4JZxD7UUCF6XOH+JXZ9xmFjPBsBbO4HQp91mhPLmTKjpvuEl3B9RH0oAiH9wXl3mIk82NZ\\\\\\\/AtRHAHci2dJ0oPILz0WX10N1ZhXXFIIYxXdNBEGTHSkL9QRPsO\\\\\\\/mGte2Mh0DSqQrnBAUuql1Z3M5GCaEiPry2ckAJfo1wQobEcuB7QgMG6RyLTzOj81wt3clSuA18FFTp767LUlCCaDv+2+Z2Hnp4DcSa2aDmAtZQvFrz22RMqOAIo0xWvLDOfdPLRN4Jp5NFLeAEloWNoSUP4kqpiJiS6Yv8rlZhGjuJXs0ZQuDSGSG1iDkUfWwgRqHkxCQBuW8vFKQX1CR6Y1BRt7gGqXRh4CecERUjkC0zldaovLUy5YhL\\\\\\\/LrnD0PMTr8RSGtT6iJ2fZzNOJoWXiHxTKMj3WhvAmYKpOw2f4ZhEf05S2JG11QQEQYHjWd9gGX4cco\\\\\\\/779eaR90ahOh2hZq+gbdxOgTxcWikVNDfJ5QH4IYdWR\\\\\\\/2YZ\\\\\\\/n5UeZ8bAfgBSIJv2+4zKn+qqXYo3hX9fUSLiud3+W\\\\\\\/7z\\\\\\\/sXx0GKeSgQ1iNGxEhTzHs2T4wN3IIQrUpINQEqRDk\\\\\\\/kuWlXm+qkz+LffYGRXiBAiWQ8HvUTfslYYf0pA3uGutP1qjgFKai0Fif9\\\\\\\/0ej3b+RC6lT7CNCy2BE8nDUa9R2ifVbiauYyrpLlSFt21ZRW2L2J7i82HbF4+MS9vk\\\\\\\/WncHNDZx3ju6TlVReDIwgnWOzTHTCPTltvKmTBPshz6Zfmhm5hXKtTAg7Q+z5urg6Op8G4G7LNuT+43vrJdkPsy3Je0ujPObEXg3lUGHnPVBFHzr3dgZ5dcT1fLxBX8JMjaf+MpQUcIU3hPLC3ttorhswgXhvOVdYSiuPpRAx29dgQaP7Vrg5FjfBW4LWi+By2zRC+dQKWFNGFCxqM7iajgOZIJhnjqec0sFLtaSIOMfNABX0R21DTCXhppwDsY\\\\\\\/vUfobmKDbBIAkU5PnOO\\\\\\\/iZpZaQv\\\\\\\/tHN\\\\\\\/uNIyY5ncDP2xktkdR7t7E9lDxLQLLAcupBj86Qx6PPQWko\\\\\\\/b2xQZz0yt8i6pamEkTL1voFSPGYk3Q4FWlzICOdGHuw1ZRcHZz9EEtk0DSR\\\\\\\/BCjnY9JNBaugTUfsb4bHla5S8t0ea7Mw8pSrIXM1h+J6YQd3VO+4a9l6U6RqX3JozqHXe1X43TGp4F9Ig24nM+rL8rgfrrramQSXKDfLeDlZR0ES0eydFs5v38UE\\\\\\\/wVTlB4drWCdanZ15j0hcGRFlIy3cuLHVhlMfWjdsyb8dfYhMHgcJdl\\\\\\\/QGUB7DIwBW6wajdgGHzOBgiYIUaU5GdYj+9Az2V8iBiPtp0vED5Szy3GqSLg9K8k9\\\\\\\/SPJGMI2qS\\\\\\\/m\\\\\\\/\\\\\\\/b0OhrC\\\\\\\/9uvRGIX18hdhIvEZ7rlZ1Iz1qFFO2lwXDglUBILVZxftQvQT11wHChK0JMzDCr1CMlbcnmsd6Ylb8dNKgwX73P18DUQaihiPNyZuahRe7UK00Ky8rzW7dnqC5NY0u6sKAbfdieMvEWMch8NqL1NRNLPyZ4oivSyTFRnYwilsEKm7FE4b84f+mcg9eOLn4VYQ+VKsP6Bfp2j9iCeVDE4QFgCXZilgRLwShx4+AFUcLx1uMkA9Me7U4dEpEy0o8YeEBkQ6ICXOjFAcy8NYu5j7ZlYzKDv77eYGiAmoqQ5fk9zbuspbpAWFTUx4kZI+qVYk8S2ZJNs8Z2ErG8SxpwDkSRK8C8ILe7I9LrUoEYCM+mEf\\\\\\\/ctr1GVMQ+rM\\\\\\\/tBV6LmbITor+bAGt\\\\\\\/zPY5reXpMD7BniqrnNLdrU7k9DejhCu1qcqmhJWvE4ZkU0oOyobPGq2\\\\\\\/rm7cC7qZItvdd6+Vv9y0iXfvus8QMo9Fd\\\\\\\/HKu\\\\\\\/ltAls2B9bKPWHfFLZgWJ4y9RUv7mzTVWIdS1jdvDgfk+WbAINHuRqtaSyjInMocXOPcJ0TYvUxbvBj5rgCFNnvdpwsE3aLn65N8zAxp6FvbOSsN91HDGm2hXfbd8W9XYjZiRbzSO+Mg\\\\\\\/CCNHfSt9XD6QhxfWD9EpHSEXJnwebkEoB+4ZcVigI+\\\\\\\/J8ZHUTLgHCeCjgVgXxrsmKd71QU0vK0JhSU93P7ox3B6lcQfcGW685BH5w6dR+8sd3IrY72cWn2eZ5ip1fTLvDhiUEimJQiF1dH93EvxMtTrQP4VRtehS9\\\\\\\/mJmdRu5jro\\\\\\\/ub8BG8ouUhxPa2kRcCWW+IKpDTwayHB3\\\\\\\/4Y+y7w5BhetIxIdht1aeO8l9NrI4v0sIPV3xQu8KL94evTA84nYEqWcBeejZVsRm+VWZtDUgGq\\\\\\\/LIcDc6NrWu\\\\\\\/+rUnk2vYVsyBlxVpxYUbJ94X1w\\\\\\\/9gNycqxqi1DWaHG8uXUd+CiT2C96vYljh77D3Qg49NRQyeS5IpLvE35Mst6UL42lFPy7ybP1lQdQh\\\\\\\/BalFNGmOl8yyUdWJNjlp3xGZiWkBNMt+tny3PEUT6N9q4sgJ8fnmMzQexQfA34lGB9Gnk3L\\\\\\\/fDTZgluCcHOjSKRWIHy6snWn9x+LVfuBHPDClz7av65RPbkQDGpj3yaxC\\\\\\\/TBHaXnSayQIgkzYxoE8uIq\\\\\\\/rJsUUikWnffEgFI4Iaxtdz+Vg9thRoW0cMJd28zag3GTfxN8mtyuMctSpIJdPvRHZ0AwWso\\\\\\\/OwfBFFaIsgJZh8InDLxm4MqxvSA\\\\\\\/XB9RYIuxwVaUxAQGlpeU\\\\\\\/9KesTqelt+LKnro+oM0WKW\\\\\\\/+VHpmHCw9jwdwmg5iO+jlHOmephNALqk\\\\\\\/RFfMqjv5qwF0iy8KnV8V3y+P0FmWOnukv9UDE01giZ9\\\\\\\/nnGxcuCVuOnS7l0u++hv0sMV7YFc4lV2cC3UtH74+FSs3Ofz9kQT\\\\\\\/IZdqxqLGkaraFD\\\\\\\/Z05mPsPW3w5nBuPkaaKDxYR9SffO3gqqYjbK8a03sNkeeOyS7ow2z\\\\\\\/YlIvcwLb8Y2xH9vGg60+4Zusdxacq3XR3PffgygHwVFFUpJrrc\\\\\\\/Du8QL\\\\\\\/JCepGiRew6S933OLtQhORpmtf+KT1evV1jwNF179mrW7BjjkJAvtwbzRM8a1IoU7LPysYtoiiy8AMKsBjl+VD8SNiauwU1NpLHyD5U\\\\\\\/k\\\\\\\/VbWWaCW97tbUtx1Jt1Pk6006wi7G9sNoPz2UJYpnWma2spCeaZSNWlH0n0ZYr2Qrvde8CezMUHmPVLqAXKdihaG+EFZHO2mCgRlqgcUc\\\\\\\/O5TDvSe9L1BGWccvx3nimBTV36IdSrQtC+zCTEyXpmzcVBfMMRwj9NkZ+g8\\\\\\\/n1YyHBrnDO0IDVTTAjgUCiqUq1C1M2usDllkP9Y0kMZdAhGVBpBWs8fewItAVv\\\\\\\/VC2uzfyhkvdYlYgTsHdZvkb8lWB1MlXui2XXYRnDdD\\\\\\\/9g5nZYSQQOJeYvss\\\\\\\/bNPjuXENXu27PkeCgpNc76h2w1PyyIW\\\\\\\/G0t\\\\\\\/1S9QNv1uVW4xkiVwM7\\\\\\\/d0iuu+ZmRarCiqjYc\\\\\\\/uJ8dY2fOI8NhEoMOgEEun2s4o3F489\\\\\\\/bojTxey9WFvFC5HE1PjMUrjLbln6GRuipBTDvdBjObvLQGx7swWn5ltK00bEvktYZe\\\\\\\/QhTgnEdlCOtZa7CZH8M9vrJeM4iX8VzLoT5Zn41fhOl0YybVGgUPVdiO3QJz9NzHy2D4jf50XPylg9GYzN8CkwsoAREi1Owo3JinMcexg0em3TSjKEfhBw8ObtnpgiuSwvAbivyCdsE5cWc1jXncQ9Z8CuUVjwcuzTd9yB+fmzd0BUdmdN1z520AcscdNjp74aaAMHterz3yeXr9VPtwVAOUYK+9tLQrZY+M7o9806QMWg0VDupWsHj\\\\\\\/ohjY0poQrQZczOj\\\\\\\/RRbddidjaH6yn7v+R91S2+CHB5QeWOyzHnvKJCAPb7Abex4G\\\\\\\/\\\\\\\/VZnYp4MNBpNKOtLGSy6meF5RbEClN9ikTZmhAdTyPmM5jtcHOgUfS1LoHl1FaYS6b3W\\\\\\\/DlrA4o4IEBJorm3rWC7YDyDK7VYDKMoEAcGxcewXW1w+9jNX9zMO5ulvSsEAqdki7prtBnXyI59xV1JC73yO+WHrRVKm9jRxV0gkbD07c\\\\\\\/5roOGEhFPNFy+ppLjImoH+CEg4iV5rtEn1Rlck1jUQ2BRc+KcYu32RoZCKzYSVPpI3BiIItRkhYc1i3ojNrffuzGThB5Fi+GLYOxHDIwH0ZZtd2Gft8nZu+FbribVGKIIA0m4DoaNSc9Q+kfWUn2ZhEY\\\\\\\/AbMb5D1axAYkH6gDo1dj3uxfUQ2UD8gly+XSU8Y5m5tfyg8FNkbM4r76QKvQohRjlvqjo6X67iWS3A5cOABjUkeEsojf3LsBW9jefp9+mpkNShj84fwmxFpRiU1iNXW02q1yG33cVqkjUNWJX5RjP8q\\\\\\\/AMd4052SXmuKOqkNckmuYGL8fCC+XZyyBt\\\\\\\/u8Ep+DUAq+9Exvpw\\\\\\\/8kIe9jHinZLTNhQgvLuh9TM7ePT061powNoOQFbiA5Ss5aRn4OUNtCmOPtNuVcqR1YTxbggoo9rNvEybiNnxHt1vbI\\\\\\\/PW9Spzp06uFEvpHL7BS5M0MhiXccnjVV32S0dhSb2MGYbFn9fpBaEAB6FZQ+a+Vr2lsruKfTpu9Dc5rg8rYMznpLDtf8XvfHhk+3s\\\\\\\/+mk0\\\\\\\/ocuP+Zf+L5mxEjeW+z6bkCz4HsTQ2mz6wF10mlvkJP5viamyq4FrX+j9MeDaN2AXdGWS1\\\\\\\/UBMDuedrkLFncSDZftppyuuiAa+FsyWLIton0pvscRtqu9vtiZ8wcPBZnpzvCFSbpKBGGfvUnVfrEyXrIQY09NzHL6GCuWmPim9i10EGYcEyUGj174qgRYBzp2R9tIXrS1hGJVAbNYFcSdOfXSLLf\\\\\\\/F0LdWBhcdTcRkbU1lFSLhJYx439cr2paa4zjEMKv6kbB6rxv\\\\\\\/lvhKZHA1pCAn9txePzoKapmKX1BD5nUhslRPhuHzlla6I\\\\\\\/WjfXhppitl1rj5aTw7qubqFHPrz0HOTmtFWrpOx8vxS4VHamK+QI9xROA\\\\\\\/1E0zU54TJRsMi4QZEOCrS8pj3SSk5ccg8Mf+oijzHiDXhnqlEDacC0meVDtjcHilFX7U\\\\\\\/idDjGsbPBF\\\\\\\/HvL9wNqWirLofzGsK8+SEuttWUXzUZQO0q0doXYs\\\\\\\/Ix632d5EyT4VW1Ul22vmFWX\\\\\\\/QrHAyN3wuazzmL5hUnPqNVGHnelt0SQ1\\\\\\\/kSPYaT2XwdxKj3cSTLnrJAD+qdStVamhwTTWaHW6PiDKSTNXrmlel\\\\\\\/mmODhQTjHTu5Mq0O4g5\\\\\\\/p06Eky\\\\\\\/yHMggJIpAiJzCVjblCvs00xeQqnsPSVX0NprsteWIKvJQuRJ1ajudUC212lTJYfoFBeKIrL7YBG0iinOxRMqlTddk9ItxvecsChMLfj0SBbCZ9XWt2YKTKtXJMOWKvaYdwtIce0pSd56IpXLZEcsjV4S6azcFW8ymiNuJjSg4pM4IgCjNyushfqkBSyUieSj09cw98dCbeX1orv1aj5BsEf0azrF0k8Ih8Xpz9pkuxVbcsfwKK8\\\\\\\/8EjGjozXFs0Qx6X3p0pmRaCTVy3SFk1RKmISu5XtRd+5rF3+z3qqh3kvkBHZnqDm3f+6TzOVXJ4nmpyJqK9m4mS9mZvw4hxROMix9Yoq0qeQ+Lgk0cEHqNHpQUPTY33i2hI1EIns0zCe37QERErbbZr4dSAPaDOfRwwZvOoqCjIUiQYNJiIyHaVHCwh0Q3cmS5Ie8nAhnCtSoTLwmYt47omdbtzr7zKzvKI\\\\\\\/5jL7bbi4tHUqb3gWZwDbq7l07L084BKeom0\\\\\\\/bvTjw53p8nUJltkVo6KJnOOdWNJdnTdwm6z5UNQ92HUx+tL7trLNJY1J6cMVcONbpb6Y4naYYMJzliisbU3zIY2EZhM+z81LRKfQr4I8RvvSDYS9JVGyxjybCzMJC2VKTpGTtRY4vws8ZYwbmml\\\\\\\/I0pmEowAvbSL1zB38a1zRDVbNSHVw9r8FR5heEIueJpfm\\\\\\\/8Otxe8fuFXuPd8QRVww\\\\\\\/E5HZt3DqWU9LC8Rl+VhtnV14GYBgsItUJh2OkDwq0dueD6d\\\\\\\/wQ+F6qfG549O+pEHtlk9Phs0kk0qYkhCnmf\\\\\\\/x2FDlTXAqg6K0d8v2Bjy2u9psAmrjeghMdNgzeYVcch1JDav7XTYVpzI0veXvgLGsWTWC1f43KKX0vwAo9LCM2cfj827XrY\\\\\\\/ra+7drRXRgf7sHtLxmqALTIaX\\\\\\\/59M1ycGle2dUNYUfv+koUH8pAUXVF0NOQKn4pLvAQYHiq+D4FcyVtCm5jHgM2QQZpgILMQUoSs8t3q+NYHdLcwfLAvPkM7ebQWI9uF3FWMxFCnrb8zMNFWVeqQN4PjdcgbPskkjvbQvDXG00iF3NwbrOm\\\\\\\/1K4zAElqwkRohiIABYBSZfNyFFMdrjRWces0DOeZCBYbGF+f4IMxisM0htf+qyGhGEYDmUUpcUNSBe\\\\\\\/MMHf5KUz4tpWbxn8T\\\\\\\/VBu8u6VP1keb7BD4MzMN0Z2eGbe5I8kon1tNdev2227IqYXfvI2v0UdXj05RxPNINxOwZoxJyEMpI8SceG+TKUhHI2Atxi8bjI+IgE1MyfEAUCJnLDljkW38wIkIkUvyI35tekluhcH4QvMJlvfLLF+yn1uwiK5VeSuFwwnVpnvSqpuywSpq6i08LTaXVUonIvWCfiJ470FvTd46YD8sNO3+VRxlnQnsD2rZ6znDqCvS4Lk0o\\\\\\\/T8Swiz01YIarRVgmJsP28HTC4A9x5UOPlBl4LT7nskRvmfGI9\\\\\\\/gohMJvs5o9pJXZEfWx224292pm56aFlh26+NPx6V+BR63pi2N1voJiEk3W\\\\\\\/EaIYTtNziGex3KfJHA4iiUtamts9XfuafE47d5zjEM1GcDFt6ZJIRO8BD\\\\\\\/peVH5KxW8IO+nQmFpuwJNxB4J4E98WZi\\\\\\\/nws6msG7gBqq9GFQ79yigMzAckjw4jNVi\\\\\\\/bRHoCeuHXkG4xisN+y+WayWo7JrTtwQi1Et9w0NZ\\\\\\\/e+poY64tRiBcgXHrBUerv\\\\\\\/9Ck7b9IjFKSz9Bdf\\\\\\\/0stOp2Jm\\\\\\\/QKOPpQxNGZfkncAzRmPZRQ\\\\\\\/g06BTP4AZlrObvZqMmriTaNgUNSUyKIRz92VXkg9+6iPp8fyz5R5up0fTpJ9BMyTF\\\\\\\/NuvUeCIktLVmYUVy+wIg4YLqFKJhyuogwWERiOzUFqDdPuMPSZ4PoY1jXJJJLHmFXyIuugh\\\\\\\/lUOZt4yp9tymrPPrSm10tQzlM3GdOsEuOnbN+Dv\\\\\\\/G2nMShXwLfLn4AFJzyZ0YTB0+VPM+NM94WhlWMr7WmWrV9KjvNmT\\\\\\\/UHRlQUFtz+9NbiWEh85wnyvxnrekvhnGnqb7BpS6TAiIKh9uOZC+PFNFbOOHVHLGevtSCEVpYDpTeZlP1xGZwroBzB8juemUuHwpVUQKtd+qDj\\\\\\\/oxJAcp22FOFA6j+onbhx9HpxiC8s+ifnNF8vdjVMPM6yUTcFH9OHQ6NWZ\\\\\\\/YEv7jyqdgCUU1WZ3XR2pLvM6+pNnjOHD58D4v2EZkMU9A2ig7KbcawkQpPGz2w\\\\\\\/U5SaKgEOQ5E8RIt14S7E3+ZXduUpS7m3hu9+UyCqzvC91b13VMJIPitidq3Nwg4Rh6qQAy3o3xB0PCy6SNd3V\\\\\\\/anHXLqsmRmc+1sIGSSJSt13uqBi9VIF+txYLZDKifDcWQNRAMkEzPYQeTfjG7kaBVzVvM6VEGXLF5xruyv0fyDzZlgfBjQyyLmryeyR2TEOgc1RaIkWbBVH3wAcuU8Mpg5TE3dOEdK562baaqJD+iup2Wsar6qcbPtCAtwYdqYQMuXKX5E40gP+J8fFZ6tTBnsVouzd5Dt4p0ug2b6Kz6FaoZwysZ4nxR3Ttyu+PLzArsrn6l\\\\\\\/rWVq92jYV5Ro15zhvYveAaU37VFhLpctzabSkaadweqh+Kmns+jLUWqxK29HMC4qOuL25wKEHPcBSMACF\\\\\\\/8NpRQGCqaAu+bDUIjVByofHR1j8vC4HimYvD+UXDqjKCvIrmajMbcpLBRNyfaqHkCezcpES10V\\\\\\\/EX6PLMUm0aT1jecDM9\\\\\\\/e7ihZIdXcfAKzWSbupwS7l+9okIdpGHLtrxG3odtNyOcnyFPINaSz355wAw5MUfRAVanJn2NPCk41eP\\\\\\\/4nT+yT0BHVoB+S\\\\\\\/qw60z0O+1bxGVEckC9I3Y9vbcO6mn05TdR1zdK8DwEuBARaG4SR+oc+ad5pnkl3sNaM8X9jjTJTWiAE0AKcHVGGa4P\\\\\\\/6EAzytnCRmjWZRQYVhh\\\\\\\/2VwAHOm0MdSjFlZDFP+l3hchkmWthe4moewkw84A9LMB+NG4LjAwxQd0FSzTeJpQsOpleIHD4WM3b24jJnqjV2BvX6ShgyWqlMIO2U0kVGsGSX0bAp+4iXxvU8Dta8I2XNyBwyC6o3LucGIKc6+aVbQ9Q3YxgykmzRSQTExm8ewe7D3JB+7Yxd3Mx6yamdr6cWGh0ENic19BAykIrx\\\\\\\/d+qDKRrJKzBBPWw8MesOauSmy0cfsSfzfpxullr5acRnUcimnx6kzhgQleTeXAfNhmal\\\\\\\/zn+ouQiiRwKNoo4x\\\\\\\/yOeCIePN8hDJ7tEYmn6acdHbRFChBEwoKBNGroLVkoFiVc1qtAHzN8nIofRLXppRHip0RGm7XcEVZu66S\\\\\\\/UbxSbGVnf255mjOMBywaTOg5MMr+gWfAGwO79WDmEa0B5k3SWGejLdrj1Iue0ZBImP6qXK34tOjybEPqT8ch9kaghog7\\\\\\\/VktsAgAcG8EMl1Uy7qAcoy28nlWEjzLEU8JWhgBTQ22778sZeHHbPXYmcllXHT2v+JEBo\\\\\\\/YLt9sNb03oVfo4Hp0qXbSuxgsc06ZFu4BVlDDdQT2QovGqMN9XE\\\\\\\/HtyeTyAFdOULSphdDYUX3O9CjsgpP\\\\\\\/Krkxk8bYVZ4eMtiUQ471RvoUCXZIc5CP9TTVvJT\\\\\\\/EYchMLpKKbjixrp2a7jiKxoI2nzOfkVocFmOcV4ksbbUDjNYRVdJmFgfrurA3Zq5c9MFCVdjkC1kZA0ndl1WdqVskA6p8Er\\\\\\\/3SnpyYSqHdf4GZEmXoXMchD77kDL40GM6\\\\\\\/\\\\\\\/EVmIlNO1fAOPguWuvEPBVpz9D722Ha77Q\\\\\\\/HI+CLfrJ4F7432zNDFcfgug9LD7iFrpxoBCYZfds4kKzheJKzAPsyc7kMlWS5OEvWYiM3YtINjmQShZbfCUZHjyqf5\\\\\\\/Vj1wSOGUHdNtQjgNoQd2D3MtOzrB9V43bzoSLTb0Sr2F2ulsWngWdeQ0SomuCBmiyU0ruh\\\\\\\/XtjqPq70FvTT7QWJiJTJXomIKuHTpNRtOkWJWijiI4hc1LFC9DXJde1bfNcwkMxLKWiqPuvWbK2n+sA6hp8AbtBKkagrX6yLrqailqImY68mKxywRIAADZcNqyYZ+\\\\\\\/df3xH0DsSm\\\\\\\/Ib83oe+NBNldABS+DjJq5dF\\\\\\\/K4a1rZ9S3HRseoNxYX3qXeG+TCwxnhexZXOWz1erLoCYLQj9NxYEpEbCwdtxSi1a4X7dmbt4AuGw\\\\\\\/9B8cSpRrhb0uEmP+\\\\\\\/L1y9bEDnRSPiitX7nO\\\\\\\/uV3X4iyQ7TOett9sKYZhHYhnyi0ipDKUOSOBLmLF4Qeyn243ngju\\\\\\\/nXIUZ5q2IQt60FuaXJOJalhcmbadeHxTJ9HGSLK1N3BBVP\\\\\\\/Ev2su3fnuWS2Umb4j45YK90IZ9ez9AwuuPzpEywyMaVAQ+8vJvHyEUo6bKp\\\\\\\/qptSLThYWOa1FfAIqQyqUJUYbOyUjJ8OlBHKqlKr6z4yaeNC4K6b5AUhpPISM5l6\\\\\\\/q+PdXjkx3hPY6zb+StzrmdtgHKy65Bec\\\\\\\/bRFkiWAUDs5DXx6i4EJNvY+ciDRi10lJBRNr908BhKYuF12uYfq11uCWcGOSr6h9NKZy1CwXr4IKa8ttVkAi9s7RZjIOUQp2Us4BmEFSjdmRDbJZkL4VR5ZrW3GsZPB\\\\\\\/uqRvRvlRBwbk7hwQu6Jy2SLhaU\\\\\\\/Tqrp6HHp8TOBMq8dApuZGgPJK7iho9GX+TdVTCTL6V\\\\\\\/JXoNyVOhqL2bUqqT5fCiPuQBHg3r1hhRl4iX8GJiYD010m97tiqlvZSVU+VjMkA1ud7nk39LUV4FvW6I7TlEelPWrnHvTcOlfhMQzPEottz5W2fsfmhCuYvcJW7I4VWkvcq\\\\\\\/Lm8Zn4vEoxwC4vMeXhG+OD2xFVkwWv8z9wJu7UPOWSIpjYJY9DIduO14cMSWvipSZkIySYSmFi6tu4lij6bh+H1mPgCSwtMaxvyt2xtM1uBDPH1M8RyKuU7xF8X3lXIeXQXfnGXWyYhUYIMFVpnyT2LMmeNQjCzPNkbZXuUNgoTeH7odx83TFnldppWGDhCYidOn0QIuh3xmobz+XKNU8Zgc5Vwix2PXzH8MLJm5LyJFTOi1Tt1pPpR6RViaithv9OQmwNHw3jOplgHOgRM\\\\\\\/eBTBzSYqfVVXh3\\\\\\\/TH8M6TOvKlWyZM5+7dDc6T2gxlsmRCi5VMJWyhasvhzxYrI2FenkpmNOVaiUEKVHU4TOhgXYkedFpZvOx2S5Hz5T2I3xsnwqJ1x5b8FRV\\\\\\\/XFUXdTdYJ\\\\\\\/R5gjthSP6Xv88TgfwgOCx0biWprRJ9z4eA00l+xi4UiOs2wp9jVnHZAML85Cgz6lik2LJaC4SFkHA6pLWsAejkSmgjzqq8hI4bvGLUJ6sxrNdgRcB7T2g1SrR8\\\\\\\/O2NQJjxgLPBp8E4zxPn+DzviwdvQ1EiFIxZOFT+bViQI38IWafkc+qPP0lVzjMUOSi3EDSOLzmrlIHr5XDSwqjyKg5l2fDPOlGFw3IrntzR9jc+jOsBl8bvnnKBaxMPKunBzcJxYFDJTBna5ayAOU2BBIBW5P+HH1hW5JipgAJZ4RPE2L+n7kBR\\\\\\\/GWBtUAiEsHVBP8Wr13p3uMANmH8nO\\\\\\\/HtqJxAFBMWtGJxRbKf+LsdfbVwe+Q9jko5j4yZUj+U1J6EPfbQa2nVH3PV\\\\\\\/kIhKsG2meUaVgCq0n9tt4iesMqaqYSlbA6dN6x3x4I2xqj\\\\\\\/4CaebZjhZC6s\\\\\\\/ssKiEYNaZroKraISax6mlJ8H+U8fjF8GOLtjCfh7TQuvABcCZ+VuDIrl5LfQyL7dcWmfg3h3t62QAosNwZZM3uMY+T2tiBncCjVZzZm8c+WiYmNqqzbA70kqBWf5yJUc7LFG4qVyetWdgJhG\\\\\\\/HC62PHiFYx6WO1HRnPKi8z6Pe9cY0S9m2DUXARt9ZcuBXF7zRzkGXSVj19w8buisCNSEs8OcDDWeUwWaAK4kXtZs\\\\\\\/rnGOzJpDDm\\\\\\\/QTblvW\\\\\\\/kJlBYP6HF3Lzfz2E280wzRsDIJ0u6X9v5oIRD5iy7AXl3yxrLhpGlElWSlfX7MFeSmVnzdbKlU8F5DwOKA1TYEYNNvTX9b\\\\\\\/xCuqLuFPwIcKvowWDxcqytUgXTvuryAmH+ZNFZzYoppSCxbEnOT\\\\\\\/I2SBZd\\\\\\\/8v4ejIAK\\\\\\\/e5\\\\\\\/Wbpl32LU2ZjVBdSUrKIEEbqX0wCxxMyhMGBDiLngQKX9C2\\\\\\\/qZIsKr6qgQCDKIcccGT8hqHX4C1HXgI4\\\\\\\/NGlPiBiPLL0VibKUuEkVJu+L36TcwOpYN\\\\\\\/olq8IMnTkvC3FBR+o5ATzmIBXScwFqrWOz0IUVCYSSspGOn9t2SNUPEqI5GR8B5MfsF\\\\\\\/C8JaLSvRyi\\\\\\\/a2W574kzWY4ArMktCD5DK\\\\\\\/c2d2UytxblUPbzEKvgkzqLFU0WS8IK056F\\\\\\\/FKM1Yv5eWgbq1EfpyS0UtlSlFqIapRKAinV\\\\\\\/aF3xSIpj1WR7temU5CJhr5dwIq4wn8DFBztvIvnGlDbZigZ9OIHmUQsLDKWhnpn3\\\\\\\/Va\\\\\\\/UY+SleyjzlwrmtpEIt03dzKmi7WiX4+FHUbF+5JVU71wgq+YOxR5y\\\\\\\/w0E0\\\\\\\/9j\\\\\\\/3Ew1E1C8IgxWMihHnj8KZcoYawbXxTXZEP4LAqnqylx9IAWMB8kgvoliXf0NtDO0UzDs6CqlDHyH5E0ymNAHTmWXoMufsfa\\\\\\\/zvz0I+UqYmE0BOrxW2yWebg4whsZF6V0gcewswMgYDOPFGzC9PvQcy+WRUxvoIfviJK000O9uOweO1bBk3099ZYfpkDtGJkzE4DKh1FmNN6RM0j5a45KwRtk\\\\\\\/v5veY1sVT3NdCjnibanrRWFmZd8gRmNoDlrGY1pa6wWBywwLtn\\\\\\\/IbLlGW\\\\\\\/34qHcFjr6DMOwVRBapivz1exT\\\\\\\/Dqs+tjOmUEzagI2ym90UQQaX00OiD0FtQHmy6BJiCCpAdeUKd9oAl3utf8n8jFqJ+OF3OYmKGut6ddEgCNn4DE823Lxx7Jz1bkWp5T4p5tunpZIIuwsS0GpoM6Bs7B1Y1wv8ESCpGLjRxTkepNnX5iso2qFBwSvjmGzea6mJaTBSfwIZbfrwP8JGRjLAPkGkPYjOoSytOLqH1CIzGSIEfNtHbThmu8d\\\\\\\/GtpNaXyhU9RxHjqMgtSfAowcm7VanYMelyTwdTYduOmA4GTKZBywfdHt3MdpLo\\\\\\\/Y+ZkAqK\\\\\\\/oOpIMdLQPDCTBuukJUWDtFaoyP8KQJl8Oz9C9igSCnoE1ESegqIt\\\\\\\/jRRsqd\\\\\\\/NHyT4DseA\\\\\\\/MeMlxw4VsesGQKHCWmtGO19qSx1wphB4\\\\\\\/eSWaq7XXLRC12HkNTTHg0gfS9mvKCIaQq0CTFOZi5Lk2PHIfs7ujyehMSfiHPrxkEPhIuRPzQLNrHGc3bSO+oXsxM91b4WvqvqQXz+OqJuP6M1n78\\\\\\\/KWNxvgzvGivIJSAgLm1LLdFtLAJibZUvyvmPeehSZby0QBv5nUnJqqmEkfqD9pObppFkc8fNKGEstLPWOuc9C0mEWAD+kanTANl9xmRX0wRRwlM1hg0rLvb28QmMBZvCVtC9DK7Tqs8zlXOKEKkQKJPI2AkW94WYdq26K3ojazMiZt4qjDvBEyNsIq7NwDINya3Pr5+BTkMNr4ixHDeeB6+Lo\\\\\\\/WCsKQEGdUvmzVXHOSr3QhIqZ+9dJAhYxsZorpb83s\\\\\\\/wUJK3vf9mZZa7BIDUaluTOwz3x2qDiJTVSf398WbvTJyGcFIo6lMOR3uu6KHeRmvLvUYrNEV4vwgfw23ejUHSQIb4+qiXr3VpPRozPwUcxVm850Qn61XT4\\\\\\\/5pMtwKLO3NA0VZ6ymdZXItqeVzJih6msuXJFNOuHnM7+VFAg2aeqBr8bTe4G6P4jFi59eynRUY\\\\\\\/e1nDK3Gr4wsekpTKSedPG75t7E+eh9y937SXbV\\\\\\\/RKJ520\\\\\\\/GrpOjdioI01R1uCLWLrmkIHdbc\\\\\\\/M1uMYDd+W9zSBUUGdHfwtKSsDTLofG+Dp+zderJGjnDTKyjfGhox+ivt7kC5w9Izj50XdzrB8fp+c6nz5ZMK1NXfARgsdDmBfhylSteQG52AtAWz9WjUtMKljmsKSl96ini8awuoJj+RH5\\\\\\\/KM\\\\\\\/03fNJhZpmoMIbQRX\\\\\\\/LLZQkIhHLqWPHZs+8RBa2SLDvzCDl49ksbFU6396+ErubTJCsBpdsMSMvu4yGEHS\\\\\\\/aADlO1CYl07\\\\\\\/VAlSpBSkEuGBXJvEigzTV9DXoSxFuOiduS2nKvRk9Fu9z0Uo40ieQQ\\\\\\\/rLU7WGgic7Q+gAwFxEe+KfElQVPl8dW9R9K76Usv2oUcvPGcuObtz2CADhwSsOluF6+et11pvtOYd4pC4XSseca4XU2z\\\\\\\/h8\\\\\\\/AOYwl8Erc5dYMwupHQrBbUistRnkCm0+2JhAtCMffK76IMas3GAXuoIUDD23F0dfsbBf2ma\\\\\\\/qx1BR9nfOErIkoZ0qPkE0lRSUWEUy87D3pKGa7F+W\\\\\\\/UDgFeNAJD5V6u30jL6iWeR1+XnbEn+34SpnrYzOBGlVTuEmXSyNBPPYmcYJuGIgFllCp+WeaXpmmiaoCNUukJxyUBNr\\\\\\\/dSI4F3YfJG0NEBq\\\\\\\/QfX\\\\\\\/zHteVhXjNIvJhsU7UfW+IuuJtPG5yMy953sejZS7NVBK0efPig9JB3xTCRoYO9AZ2MVzJlRkYXngl\\\\\\\/jvZ3ALjZCOf+J8jM7iZbc6DHxFuSNQdaHYIzyMikx+AV6YPWcPs9DWjwaiWTN4IVXPmaT11gaGNT2aaUuNGCNE+IDVMQ888+vd+Mt\\\\\\\/0Pptk8wMbyqPMDd5g+KzzEK6WYjmZ+GPq4F4hS1RCImbOj68aU7TzltOrR83vGN6Pwfg94IP7PRXmV3pBKiQZVM4iBBzGtVhNtv7IpKdZbHV4V48jV3n4qSF3mA1\\\\\\\/Oyo+H4MxIyOwrrTmvelg13uML9RUE5qWJyl29a5gJKUFSshgW66pyiS6Hzv3psPdOyr1ZGzcq0ayl\\\\\\\/qXueWFatxQLiEtW8n4EOJ0cbkEJY7g0\\\\\\\/mpxObFWmUzWxgkovHIrzPYhnYr7gapniRP7krowTZRbqRLKk7bWLdGHbUHXTvDDi7p\\\\\\\/ajlj8TBPh2otzo264+QhSvXj795mp3MCWp9rEch51ibWLVSwBPh0e0t+j0xnQcBIXvGtq7T0nHogBXpzeYJgbB\\\\\\\/5pVnGC9lzsJIxSO0IYL0En68soOcITmSJfz\\\\\\\/15P0wqOHLwz54qwsodsRAEJTjq6yWrWTcFEvQnvTO9ZWT+P\\\\\\\/lov1UYhit45mOEnEsLf60RPNV3xfVJxWqnq9SftqNOnBl0dc+NwbUtXDoHlba4J6BkjFt9jgSI+HkmghcwNMRrMIIveLyMbu6uITtSJpRSSYh+HsKeOmn7sB\\\\\\\/O0fOjQzipeSBLYW1x0Ht8Mee93vrpHZxkfue8ZINCRc76mXhwYHObkSNYYvX+91Aln6HUXCePUuuGg8vVRfo40O4tKw1g0xcEMVHyW3LgIUtn3kJo6LgtG680wIG2C+oKCQNHL\\\\\\\/tnz+ndZQ6oodg2yGc24fRmRfbm4GQPPHX9KhKlGRgnyk4JEyQ3C6tv7wlWrimi4E3swiyV03VW7U9ln4F4j7qn8niUF8mQOTfDscWm47sraWRU81wxv1eYyt+Dzp+RLWPhAo0EnSAJZjqB+M7MjdXLYCzlRygE0iWH2X3JGe14d32eQsylp99VL3ksyO\\\\\\\/Gvx61z+PFap2ZuOrpV59aC2iEVwij4UU\\\\\\\/tBRtij1o2ww66qLwzEIBCpvya2X+NHQC\\\\\\\/muEgXPyt2fAlF4R+1IfBhbz5nA\\\\\\\/\\\\\\\/5bxhShWjt1gTQO11bNkte4rjG3uJcq+P7FS39vv16hFHInvbFmB64kGpPRSfAF\\\\\\\/LYpG+Hh6aWNqo4jbsf\\\\\\\/AqNSL\\\\\\\/5Civ4VykOe2L1lF6zbLa2q6iUqFsS21HM4Y91nEuPopxRiIzj\\\\\\\/IudB0TmvcmFdvykpZP5RBT6zX3CkyjDqvX4Qux7HQqCXvj5V8oq06i0ENSxJ4PW5g76ZMQkuwi96TAeo0f5yYmyXGOCJucfC6C6ww1W7BHNSTGXsMFVh+PPDPAGFd6R\\\\\\\/kI0EngLmlN6d1w1fX47jxZSzvuNNpKPvD9+1VWdh9UKbuPzMJiblEIlOWej0AS0A7J\\\\\\\/MoWKcqfbQ2dPdHq1PNq6tnQ7aYuRS16wwWr1lLKktebotPVOJyMRlncLIsb8mRy\\\\\\\/SUqc6qAAKnb605PtGdvuVwtjzRyS8TG3WEV\\\\\\\/dEb7hlg8ywSld+7pTtZGkGASGuZmzgf1p0MePyn6oF36UahZUH\\\\\\\/79MfY9g5ijb+PYWAFoREna\\\\\\\/ZOJFTeRK9Dam6Jsj19J+u5A0GlVxIcJCSL+YfqJtd4G6\\\\\\\/JEd8zwvBIwueDFuu5wLpwEtDm2iOkoaQOWmQFli67E1Nz2QxbN6690fkLt2VD2Dm3Sznje+J9d73ub1vtjzB+SToFIyFadZ2sAZf8zeZZ41G7TC4I24a4TBBwkCWoYAeUyENumHYFFiK+5qJ1yFYX5euOO4G+jrLd2AFQJa8W8k0x\\\\\\\/qU+OZgYW4bnzBGoQVaTKhkikvYBAB3zFnQLRyJKOiqzpk\\\\\\\/6p+6TLAflwqISpdIijpFa68mrU3Bvlx4sr4VrvxXlC3kW37IEkfu4pnSQZbyZX2Ma6aLiQ\\\\\\\/SablNDEkCDd7udeTSueIWYKwwu59BWzATaQu1HK\\\\\\\/zR1ozstsCaElPtNyMwSufAB7fgMdvzBOBgX\\\\\\\/8PWxmAG\\\\\\\/HXfRNffpYeG4owrMxHtE2sXFC9pIICWat9XnqHCAF3ILvu1IqtbG28F48xfJSJXZxJxPVxnOrnGcz4syePZAS8fVRgxZEKa0lSGi8CZL5RsXtNHdSyvJaSHCzJkY1sH82k7qlqvKdpB3pAliGIVygI5eW6LMsgdm7xwJid\\\\\\\/tkiRZqk45n+f+VbLIRCMllY6PE5dFQIqq\\\\\\\/hyHbTU8sQ+VB2Dvw+vAi8Yz\\\\\\\/NH3KPrCHqUIQaZ9RDv1lYX8C\\\\\\\/lxYN5Dj1BEusexjx231dM6Nemm0nScohcfUmQXKA5dUce\\\\\\\/Wc4dKklh+7kqmlU7tiGVXfzBPOD9leyIhRbHYAiTViWt0pNCgDU9ijoQUFCkX\\\\\\\/bi5nA0JmUqhnTWJZyyDJksAoEYX\\\\\\\/7RrLeVzDUoDch9R8l0nGPSGgeVnaD\\\\\\\/9pN\\\\\\\/4zovsR6Hn\\\\\\\/ngGSvtJC3OEoH0PtmIWHvnxV5+3GoWT13obIe3XX\\\\\\\/RuI2hT\\\\\\\/Oil4EIOQm8cSMdECDXKKhldpH6tfapY+s0lhJNOf2DEbacqG+o+1h2H+qsmuOaf1O8XGV0G+eVhD13Scgx\\\\\\\/o1ylIu19eZ7MeoHgKHnD6pMNwcCHCrjuighQ9MUOIgEppahDtPZPEcIxKP2FYWkuilIpIVSbwNtOgdhVURkxP5htcCifvow6FtI1X4gJ2Pzc9sUP57YZq70v0caxgUVxj41ZxBDPViKi\\\\\\\/cWTkpUQc7jRRxoZJkEarJn7jG3R+gB9V2fdWDessKDHSFHHqgzRZotwYgcmevzIFnkenX2R91+A4a57zHpwLI7f6WA9URJnrlacapkW25ylq8sjYy+0HYjVCzHTdA8CthlzM\\\\\\\/Lztkw67vWLh+JaqJMej+s7ifatsoqbB8LvEE7NuUMRFoifZafIr43W4G\\\\\\\/5Qnl9uhmvGVfxFG+JtwRO78ThH9OSNAT3lBaqLXzY0eV40d5ZCfYxJ8MYtGwpOOXqEft+silawixhz\\\\\\\/Y5TEpn9XJZo3jsp\\\\\\\/YI\\\\\\\/yKeezg0EBoEIO5cdupLeSUbAYN7EHl4tzZuwDDeSQOh5sNUFYm+34NjwIPbo4cqz40xfo+XO2FskTRSx7XZSSQeo3\\\\\\\/iSOfNycCovJwPjPOBpNKHvxRQBRXJi6wJY5afQpOSvKC0UccmjPSvLrzuctVpwWh6F73X4X4m6d15N\\\\\\\/CXxa2nmeozlw4Tnfasuqe5t2SQoVQcb5OtwiEz+q8OBCZ5XO5B3JVX3NuRbaGmh3Wny9jQdxxT7ms+jGYAQ1Cmmv+EymYIYvOfuwAa65fuymK9rt2FYPkDGg3iZWW0WBPGzQs7PLQqUHRkZ3kitKQEPY2oDJhMaNLqeCyOjY7S3vJNEqtc\\\\\\\/J+bCbQLv+FxrXiyShrDPerptlr1lnmkT1dUvKjweTGSaqAeHhJw7G5p3OqynXxoQrc3SPC1R\\\\\\\/rlMFf\\\\\\\/lVKii7waNRuraCRXVglxTmEEqF6iYnDlEMD+grLxe3kSVn3IrrDANSbMM9J9IyD3XReS0YmbOPUkUvfX+dfRxiL0VoxA23nChrTkWm+6YLkLOhjd3dFn9iHbs02jo1+jM6tjZqQkl\\\\\\\/sJGFyKImAkCRgiBYq1i1FnY39QkjZJD6rbOjxmgh8wsG\\\\\\\/U7c3P1sRZBDVOPHZ0WmXNI5Mp\\\\\\\/fy54zlcPrLD2qdcJStraD2e0FrT4rZZYpJg2zEwuFMWINrVn3vxHEatpsRwG9rQqq+7cisFPLvqDsmzyNNl63AyOlxRb8GO4x8AhyMdYR9q2TdSYy4oO\\\\\\\/+qWZZsKqUOrjq4LJg6uwOt9A0+TWOwoljaYkqqgajHtiI0TWmPxiH9OrNZC48yg5XxlO1Mlh4GjkDGgBRbI1pYwi87RtKRs9oG8n9gYgMdOU1sr6+Ku7B8+1KFkM3MNrqqj56WvfnNoTKRPMJnDfLE2CnGWax48iRNHNRbVa5rgIErbFd66REpQ+1Q1PvynAGxaBf74V2LrekSkFalUk2GL3oGVFKEp4\\\\\\\/uVZ4R6AFY2b7WKToeXbI4daptgLkyIqaM2mMD5XbSandPRYRgzl7eBsWj6ORWwh7Rj3UjDOikVZuTZUImk1foR5WQrST0eR8tBZUu11xpKTbaroAHHV2Lb4baWUTegkwQqt0DqlXXXpsgpZWAA5ytNCGiN8oYscrgfrtMFIDokHKvGjfecSZuoNtTCGCYhXszJFK0iXaT6ujo5q2KQ02zLnBfhKW4QBSaPeELfmidP1YbiA1jvEShA6WmmRsZh\\\\\\\/ic3xWeU3JQbiEXBPzHvOtl8bTpQ\\\\\\\/BeYjJe01TGJ\\\\\\\/TABlf7nkLag52JbWuKjrbSThwQE+OmO6W7UtNcTEx6b+byM6pqVrT4aZEBxaQys6HbJ4t3i0xjMsLXY3Cq\\\\\\\/wX1PkKP3fsmsMfO8DWPedW5NfmQmwWVziioBYtz3c+sFiMoN4kvlvwNMQiA3VFi+4SQWixmrBJEZ35IVPOfi5IwFvS39drnsYsa44sY61aaSa6aoVkHGO+yU53hATDlfP8n8Rzk3hx\\\\\\\/0ZgAnVaDnpMF4eI5WaLtqyznQeeykXijmslSyetHcCLsE5QcE89JxMiBvAhfh5AH3u2\\\\\\\/ln8y+tkaSwdbggMsRIPHU8N8Wm79rxa1bof+MCcSSZp\\\\\\\/7AzAMWmMj5p5135cBvwaGBBfyaQEOGJYgb+ul9lV8C0UG7ZIPFqM6NP8VYiiZ7+9mr43O4WwHP8jdRppdPSsANQQ2QANYTDv0FJVoAw1\\\\\\\/jqe0HAf7m\\\\\\\/xtYpazwSW5He\\\\\\\/AtzXXR0KgFIAlG8mfetx\\\\\\\/4254w4gxN01kjUExAMPgtp5z9JjXz8EiU+wUP6PO6gNJ6Fb2M3mK5S+3IyGnwCBEs+WFUxKkQ\\\\\\\/gpp16+7BS4\\\\\\\/M1m0H3vGjbaMkaXgi1gU0TWs37nwH8cQy0hlkEzVA5IqUG5WZ1zh3+J\\\\\\\/HR4INhujb8svXUWNk49aJMLUX3CmkFHsR1Pm16lXDkbHPtvGGX+Tsar8zbJpfjjcqcrNJZImksyxnQrKMM\\\\\\\/fdcNJr1MyTWvfB6B69+Am0M4q2lY62X79Ww\\\\\\\/VLum14QKTwVBxNYmTLsOO3T9H2GjGvdl3bP5p8MEuDaC3AqCpbRMc35\\\\\\\/BV3BAz2jSGM0lohfYCZOOMCHBK7FRWW2an1Ve8UW0GOzx7g0yCe8tOFUOokLtoGS1mBlqlRcwvF0b72FwUBNDxdmI6peJGSeQGerU\\\\\\\/zYrozzDLNqKpfKbVHA9Tm69XPq72+x2\\\\\\\/Q9501N91fy65MWXxYujQP5DXXL3r0g4IUId5S13JPgfKAW9TUfq8KJrDaXUoVE1P0cUFYiEZUZpFujprJtvdCsXl4FWIjtRlUALKCOMoHSgYRV\\\\\\\/Q6QVXnqdKD4RIquvM\\\\\\\/dnzY\\\\\\\/Lk8UVmL9EkpBBMUYCbzCT9uxi6HbxG0WPgMpK+Yf1p\\\\\\\/XxoaOLtV0JBNMupn21S2hN2unPV3K0ya0I2JD0PpPmJbevNySXhbj4U7\\\\\\\/C5YfYbQ\\\\\\\/CUycFy7EbQE71u7EwshYoV6+X2aWWuDFiB0ximoyXD7+ILO\\\\\\\/S2qxsqM98VHfaWE+XN4tDGNYbKLMN\\\\\\\/qO+jtlhfxNS7lDHlHpEQy4pkcqlQieb3hrEpZrMkBL8mdg0o3WCiEpSpHz1gtne\\\\\\\/hjCFW1y5C+HL1fLX69+Px2u9+hO79+IcGfB4+ngpdFerFkNjzV7u2Fsipim+OCILhe4e\\\\\\\/7E6gq7AMppziyylgEV8t8lh\\\\\\\/Vym3k2CgaTWvjrLpwTzF2vqX93PzJztk1qtJjk3AhsPtVooUDqlmJry0KJ9GT+veyGr9nzKXg6zR8Q+rurhhLK54ON1cORfFPN4wbYnCD1KAa7+cQ\\\\\\\/xCzMIBLUhFDsBq+V47+e67yyBgjszKVj37o0tJ2lqKJ8t3k5LeN9oJj77XEoqSv7+VAIZ\\\\\\\/Ho1uHh1YSY73uMbjKPqPN4NNXrDMe4Q0VSaLaduVA6AsuZauscNHOCwUvVML2TW\\\\\\\/BHasPK0bUpOXi+VlaEzL\\\\\\\/cuszc06AVxj2\\\\\\\/uxhFj0gT+nShQWF0+q3Jlw4\\\\\\\/o55lDfzLIhRc0y5G8ThpL\\\\\\\/6sk9Sef4CqL+Bx479faGK00\\\\\\\/AW8y7PB+Ohg5EoxlvusFZWmiqN2KMIfy7teGFQjbSeY6+evcwRA2Ou3\\\\\\\/z0k852VogGeSw0106hZHnwIHghITpgo\\\\\\\/sydDf79zOxp70R4DOPPNPgPxCGau3i\\\\\\\/UgixyJWyVD5YTg2dg7Q53KSErCr12LIhdJyirboTfB+Y40Jt+28YOciRw7xl8EVbwAiIp\\\\\\\/k7d+xqXtNntPFWIGtMTkHbW1RFduAUeOj9kNVbORD\\\\\\\/S+uvM3o9stflWj+F7bTYL9qI5HTI39etM7aAvpCg9PVNyAnE8WeFP1AlAOWbRMuUDQOeEFknvw4xFXpwpmU5E6H3Z6BzgkQuy+kkm9xFLNNC4CGI9YwEzi8cbPbf56+xahHQzJHEYNKcMS44m3UgTHp0815sKa8C5\\\\\\\/G6R3OTNiu2Jnahk99cTQM82a+QSHu8Ma0gpTKD3oF\\\\\\\/yOUADyCDIPWadRQMPJsCbrdcbrpz9wgSHhFpAdIvrCPsG\\\\\\\/uhjNHSlUkTKkSIvqGfyNXX\\\\\\\/1bCQDc2f3j7rXTXFa9UefcyV\\\\\\\/2J\\\\\\\/OKBrBOwjWMz+waPL9xxrAnOyVWEfx45Y1jLDPRzJH+ohdE2TASr6RnNyrQbgsVM7ptPk4xfsZBDTMjF5zmXyCvW\\\\\\\/UFWb0EyzlgL59JCNsn7+y4sSt4Y9Q+++jLX3JTTNHlW0ttuXJDm4yTxkdcHOj8b4m3vLuopCz+ZpSNOMzfq\\\\\\\/KmfzOl19gypMwPSt9wfqC8Za2efYg6GdOcUtiBQk0H9F7U352URXMuwNMKIsUesZZl1HhYkFgvXkdgQ9n6FbYvIGkIxmTCpw3igrgIyaGYgrmszFEXJjmIa1k6m9QKaSvqXEXauZgl89ORygtxj+7jj6qCKFZni6SqvmZofNWRjMC8O+oiUl2uduK1VMGG1IqqAOvrK9kDzw3XBYjboUfjdHKPDpdGz1U6ATvFirpFPhwqjG57Abf++jbw7oX6V4tYkvJW7ALu4xUrUIG2lWVauL2Gofb9RmyjO5uJiO5eX0krUMDiVDvz8otSOuSvDEkUauhI7XidpyTipUKQVmCbn54JS\\\\\\\/7KJmMGedUD9lBCTHJifyjQid3ZHNNfCuBo69XbTHRhIFkMnt\\\\\\\/zyI5wv6XKMV2B5vURLnYrqts3I4XTi\\\\\\\/E93D74KH+7I0rC\\\\\\\/lC5BbmwF6ZkWT1XPJEqhg5OHMXdee79CoJTBv6BLOZV3\\\\\\\/\\\\\\\/Lh2oQQ3zeTwS7iCuitUox8+gpcHFh+Qf2PxyE+B2ZDDWpbBq483h82AlM7kPoc+gxumh+66av4nTY5yEME8ebh7K9iJqN3LUgRH9O7JdZqBEyTvo9AQiJehsEG3zeEe1Ktcgu1TaLsH22gQDnDQLfBSnobCrbrxV5ItUdgFd41MdRh6i\\\\\\\/f0FZ3hpf7cVjt\\\\\\\/NAY5OJKWnq\\\\\\\/MMwEfdJd+jLBfnKm0stkccmb5Bq1kzVcxVbjAgr9mC3MrrPzzIb3H6b0PZHq0364rK\\\\\\\/FRUNFB4uDlfGqRkfd3uYmRvK7A9JmLIIoJJbXl\\\\\\\/AUXmcfPdm\\\\\\\/4pQGZbQFA72BHdUaTFUo7iecCiAjxPiOkg97rkHb7hLvRb1RTuqIgDM9laI6Ftq8LbeRx4OaFuq9q7L9dkUJ++Hh0iAdSYrh\\\\\\\/Erl3SUJa2R9aYpIqkpYKIKBBEPFIZY5ZNQTjhrXmNZh7oOtm+quhxdsZcBNVn4sORItjZTvcp\\\\\\\/f03K0CRC5L5DzqQ6YnQi2wDPXdYL+TcW+jUaQXACEYvdat\\\\\\\/1APz0I34P+AjAAp28FgiI8\\\\\\\/EM5VzjbnM\\\\\\\/P+MaQxXKVv69BYiDcdfhYW4oPtZSRA8vjznK\\\\\\\/f0IMk63Naa5nel\\\\\\\/q0k24pZc6xuz5gnqKw2ekzEnk2672rzYSdQaIzOoLc\\\\\\\/1riObFOVOSrfiUOEwSPz1Aqa+194FO3qezRPQySIYlkBmRlXEq7N+FaB7FUFKTGkaAcGFqpuxCOJ3F09g86HJSxQEI6d8aMUPQqg+3TztAYsw74usRZOl4WJ9dCVQVb0iDcePBDj8MO11h\\\\\\\/pCReDLynQipR4n2KGtQosUhBLEXGhvLLzNMZ5Jl+iNOYZMzl7xn2gLmIOhffQpjjAM8ScnnvWVacJpF0+5EX1ted6gUQsmdeS3UBSDXQ5QklffHs18yMLKqs0gLbGdF7k8scoahWHCQkKOFMig5UMPF3yq9ACtS4\\\\\\\/62qXNIVSdHiH0F+RzTABz0i7Wt2Rd5Ap6ot2flpKSMnifGaAJippD4jztxwNaGV54CkemlGqjRVv3GDiIKCe\\\\\\\/fF7F71k6zKwq8voOfSk9fNNKsxmSOSNKuKeidMmGSCyfUO7LiRNQ98FyN0dm1A8R7d+h5Uf1dm+CWEv8YoCHzyuYN7k8Je\\\\\\\/wfkGLvPahx7iRfnJj2j7ReN6Iw48\\\\\\\/8j\\\\\\\/9NZ1gdEDsvT6GKHYEweDbD1SN2F8FiUkzcNqFYdipH9O52UhDOsYm9sxkxWwC2JM8dX104AcZeGGnayidZ4kDsFnPuAK+t+djQ+BJ8UBtMvR7hn48S1Gp9ki58ZPVFagzjec1+F7wkodOsXc117b51E7x7savgUuYEzLPkIdO25Q9x2VDpmhoa3PxLVLXcfVgkvY7XQAeukQ\\\\\\\/+B1f1mE++uPfllI0WNIWdrDF885Tj5lpf6NdxT5cCP4VD9PJGPplMyp\\\\\\\/XQwmqxuF1vOeof5y39ICES5ZwY0gNvVZ64BTewbfUJJ08ceJI9NfuBz0t12s41aMY56r+YuT4nYVuJ1yMMHnLuXeAY6Xd8BsctELior24iAYG\\\\\\\/3GnLAhgomV0xdzN0G1uXMGeWLAc2CVcHUOe36JseT8+miW9lwAmjHtb9Bdv8CMuP\\\\\\\/jxWqQ1a\\\\\\\/nGrfLY4HY9b49YUhXaUDrtpTkwPnlsl1ysgwEmblWQ44wKfRsdKBbLaLtSLOQS9WkJdfL+hYxY0T0BBMLZxxItzSknsCY+7yRg+qjHxfjr1r7Um2sMIrJJI0WC\\\\\\\/vA+CohRS6qTNQ\\\\\\\/0Psw74Vf6nbbn2pfXtEGTLxe8asSqiY773cGVCVxfGDgoxhXd0o8XSKuvQ1Iy1oGTw\\\\\\\/d1BqvWMwQs9PXccEBi4WYatIhPX5xUfSIUP3RjciJJENdiomVPR7uoMCoXAkUmWCLhE6MEG6HIciHgko5ow90x6bJCBWr3qPdvOpfam6aDYCjlgpnA12H8hoqBrcqA796lKJdL3bO8pmZ9gggz3KVlt7cOlIgjgy+k4A0L0bOCbxgccVf\\\\\\\/NWc7guoVSF19Nw0UpfxW84yTcAN8cex3XwqTwFsLWXdirPaHmDcgWh27nUe9uiL2KB3MHZZhKDfaEBapAdxd4b2UvWcAiQZH+v+55e8nfhpukhC04hS5GtAm3sazj\\\\\\\/iD7DqXu892DrdP\\\\\\\/+JW3x2rDxKZiOHx9N2a6Gl0yCiBVBbrDCM+4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022TfX664M2LUE+oTLE+6zG1g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002260d4982f4e9269b787e8f1963aee3326c0b80b7f63a7a7b23235b0ed9eb661ab\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VgAqYOhByxKhdXfrHmgO2iAbZjLzcgs8Y34LIYC69S6hbPFTk5FupGW3gpGVK3+4kOIfkHlOQHa15H8Bkg17CMgmsstBBiuFANg1JGA8b772ya8x0idTRxa5WO4lvorYq4OPkw5KlTWUGhaiG2U01ydPvS73EOP5cbF9eh\\\\\\\/ZyH9jGojGUIWs+dNSGgIYmMP6yyo6AN3bckUdJ64wAFtejDxA2oYHRyMbOcr90eJFdjZwvWZih+9oBv1inMvUBXdsB9omwbNBfmcWkjUmP1dlzxn\\\\\\\/p9ref+JdfUrqyWNWJz4rzLU3tLtwn0UwslsmqMa+FY7qAfvhjECaybCieDZkSUrbewReVFInXYFnNwfymm+BDqL7uetNg74EhMbBqVTz6gazVQ1V5KkkYHeii7MgWe1bwCVHW8hSff\\\\\\\/ZPmpYKlEt0dhwPwteVuUZUYOGqq9d1RnVOH9607OxE65Nrfv0drXsZckOFGKTvjnFKA0eO\\\\\\\/6Onf2p5RXO8PrYsuoa91J8IkUkRMt5nITCRw6wfb06QirSNPD7uwa78GcUiVHTx44acWJ2d0U\\\\\\\/Xf48bv5d7GGHgOaA6unWMiwHCeC4Kvnpg+SdxgIeq66XbGPy0vXQUasYa5jjHUpvazfoD+8dwR8qi7HTVdgIDl4tIpJd+H97CiKYIIt0z+0FYuLRRIs2hj3LdfgBrsKvXp0GywPjA4jYlI4Y5EQ6+0VHjt6NaptaV4V34hJlh3AUh+w\\\\\\\/kI4n5p1uw+oLJPOeHB9+IufbecmvfA2t91XNL2msR7XvwrytwiogPgbko2ljCShK\\\\\\\/n4gwB15X2rtZCXA1W\\\\\\\/6Bn5byS9nPMnied+Mp9jh7A3Dgum+OZcIhJ10C9y8vcVXhE6rEwLRpiez1Kus1AwEG6XpLXmZYMRO1\\\\\\\/aMJWNRULBbaAUbsLmJdRjNUWEElUQEUJLPJWMuh6djS\\\\\\\/7Od35CxnN0KhyhrsKPrs4pMG3hkhaFtOmYBsj83LYRgRuNs9NvZq1eoamalcS+ZrLDQOmDQJs3nm7BIbabk6PjVGmDQ1RQMUKATQkX+IiYC8NR3HPuAF\\\\\\\/nIiYfPixRz6jhOMJ6AS7rSoW6a16XEXXb0I6pAFGzag6w22ItYmwrY69cvyh9tDEDL\\\\\\\/hfNwkFTGd9YvsUDXVwfcN3LWisISJL8n6wBHZXyUavt1mhQ4l0Zdf6lo5XHKDVqCUfQudSIuB8QWgVC14UFtmuTUdoYyNhkYaw0RKYvBWuyAl\\\\\\\/WTENU5H43XkicVCqd1NlY3kqRObMl+Q9L\\\\\\\/2u5lLJcrhbIIeXm3ZArmvS67qESnL15pNVb1u0rNcR+uSm\\\\\\\/7YJ0FaUYxD96qJWe1F3dCGVgtXZLZnPoUnVMwKZWTD0k49Pp5m6K4op+QlavD7nCXphrwFdUDjP76JU3n4m\\\\\\\/BLEFUSSlDW0U54veCxAzI0QE+a7OlVbDEEXlCfFOz5VslpUbM13p2o2tVvJ3TTtmwSDH8zBGIr9EzwtbpsrRGr2wNZN\\\\\\\/UEifoA640W0iuoDNPulBzX9TtDZu01WKHdxhbHF3uINWIpV33XUKKfh2eW7TK\\\\\\\/2Epjh6amKpETFNuE\\\\\\\/Ev\\\\\\\/Oba7L8nFFANL59CUuMHCkMfVUZBy4k+QDDmhW9WpnEwHVW0X13BfaDeoMezt3GJBLguy+beLrmAvcIRbFccF6srufDV4bOSqvIxWEYZwLp9HXBu3FlFrE3NHFWXewkdMhmkInvllO9MaQW8WKgxVK13gm8xNhho47q1rR+\\\\\\\/f1Sa6yLtmp2Dm8x\\\\\\\/Sgvg4f7I8thpG2j6f+VQVDgYZ0SKdQt8UIsd5O9u4\\\\\\\/35rM6uyDcaV1VUD7VYhIR97UvdfJpnG2t+yj62tEiWlWfPE2pGLbJsNE74uvVz8pCav6x6BLSKda7yeyjmRrPv8XSw4Dh9SPyMf0p9bf4y0d2YGiHFxov+LxO600yr18dVeXL32vtPOOb1t6P+2JJnxnkDqNijd8zAK4JoM9DcgMEqSCBlUn1SVKpu5kOgpLfAhWDcsB7Czhm\\\\\\\/l539\\\\\\\/Sc3TeXrylQ8M=\\\u0022,\\\u0022iv\\\u0022:\\\u0022pKdBR+89rWl7osVYFS9IMQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022abd423780feb12c872de7e8536d4a074a12caf9cdcdb8772d49f66b556d8c322\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495869 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/899" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GFKpKDKydX5QKMtoiIymvb40l8sksbmzHAbKLSQA4N4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/899" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7KoNcnqaFdoZjM0basdCzhZdelUFbtnA6A3tIcNT1AI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/891" title="Chromosome-Level Assembly of Drosophila bifasciata Reveals Important Karyotypic Transition of the X Chromosome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/907" title="Genome Sequence of the Euryhaline Javafish Medaka, Oryzias javanicus: A Small Aquarium Fish Model for Studies on Adaptation to Salinity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495869" id="node495869" data-pisa="ggg;10/3/899" data-pisa-master="ggg;g3.119.400954" data-apath="/ggg/10/3/899.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improved Genome Assembly and Annotation of the Soybean Aphid (<em>Aphis glycines</em> Matsumura)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8637-3515" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas C. Mathers</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">899-906;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400954 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495869" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas C. Mathers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Genetics, John Innes Centre, Norwich Research Park, Norwich, Norfolk, NR4 7UH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BC.%2BMathers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mathers%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BC.%2BMathers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8637-3515" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas C. Mathers</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:thomas.mathers@jic.ac.uk" class="" data-icon-position="" data-hide-link-title="0">thomas.mathers@jic.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/3/899" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495869" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495869/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/899.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495869" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495869/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/899.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495869" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495869/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/3/899.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495869" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495869/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Aphids are an economically important insect group due to their role as plant disease vectors. Despite this economic impact, genomic resources have only been generated for a small number of aphid species. The soybean aphid (<em>Aphis glycines</em> Matsumura) was the third aphid species to have its genome sequenced and the first to use long-read sequence data. However, version 1 of the soybean aphid genome assembly has low contiguity (contig N50 = 57 Kb, scaffold N50 = 174 Kb), poor representation of conserved genes and the presence of genomic scaffolds likely derived from parasitoid wasp contamination. Here, I use recently developed methods to reassemble the soybean aphid genome. The version 2 genome assembly is highly contiguous, containing half of the genome in only 40 scaffolds (contig N50 = 2.00 Mb, scaffold N50 = 2.51 Mb) and contains 11% more conserved single-copy arthropod genes than version 1. To demonstrate the utility of this improved assembly, I identify a region of conserved synteny between aphids and <em>Drosophila</em> containing members of the <em>Osiris</em> gene family that was split over multiple scaffolds in the original assembly. The improved genome assembly and annotation of <em>A. glycines</em> demonstrates the benefit of applying new methods to old data sets and will provide a useful resource for future comparative genome analysis of aphids.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/aphid" class="hw-term hw-article-keyword hw-article-keyword-aphid" rel="nofollow">aphid</a></li><li class="kwd"><a href="/keyword/crop-pest" class="hw-term hw-article-keyword hw-article-keyword-crop-pest" rel="nofollow">crop pest</a></li><li class="kwd"><a href="/keyword/genome-evolution" class="hw-term hw-article-keyword hw-article-keyword-genome-evolution" rel="nofollow">genome evolution</a></li><li class="kwd"><a href="/keyword/hemiptera" class="hw-term hw-article-keyword hw-article-keyword-hemiptera" rel="nofollow">Hemiptera</a></li><li class="kwd"><a href="/keyword/synteny" class="hw-term hw-article-keyword hw-article-keyword-synteny" rel="nofollow">synteny</a></li></ul><p id="p-3">Aphids are an economically important insect group due to their role as plant disease vectors (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Van Emden and Harrington 2017</a>). They are also important models used to study plant-insect interactions (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ferry <em>et al.</em> 2004</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hogenhout and Bos 2011</a>), speciation genomics (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hawthorne and Via 2001</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Peccoud <em>et al.</em> 2009</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Peccoud and Simon 2010</a>) and sex chromosome evolution (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jaquiéry <em>et al.</em> 2013</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">2018</a>). Despite their importance, only a small number of the approximately 5,000 described aphid species have had their genomes sequenced (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">International Aphid Genomics Consortium 2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nicholson <em>et al.</em> 2015</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mathers <em>et al.</em> 2017</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> 2017</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Thorpe <em>et al.</em> 2018</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2019</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jiang <em>et al.</em> 2019</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Julca <em>et al.</em> 2019</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Quan <em>et al.</em> 2019</a>), limiting genomic insights into their diversity and evolution. Furthermore, although highly contiguous assemblies have recently been published for two aphid species (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2019</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Jiang <em>et al.</em> 2019</a>), the majority of publicly available aphid genomes were sequenced using second-generation short-read sequencing technology, resulting in fragmented assemblies that contain thousands of genomic scaffolds. Although these assemblies may be accurate at the gene-level, and have facilitated many important discoveries, they likely underrepresent repetitive genome content (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Treangen and Salzberg 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sedlazeck <em>et al.</em> 2018</a>) and may be unsuitable for analyses such as the detection of large-scale structural variants (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chaisson <em>et al.</em> 2015</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chakraborty <em>et al.</em> 2018</a>) and genome-wide synteny analysis (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> 2018</a>). To gain a fuller understanding of aphid evolution and adaptation many more high-quality genomes are required. This will primarily be achieved through new genome sequencing projects. However, as improved informatic approaches are developed, reuse of existing data sets will also make a useful contribution to improving genomic resources available for aphids and other taxa.</p><p id="p-4">The soybean aphid (<em>Aphis glycines</em> Matsumura) is an important introduced crop pest in North America and was the third aphid species to have its genome sequenced and the first to use long-read sequence data (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Tilmon <em>et al.</em> 2011</a>; <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> 2017</a>). Version 1 of the <em>A. glycines</em> genome assembly (herein referred to as A_gly_v1) was assembled using genomic libraries prepared from wild-caught samples identified as biotype 1 or 4 (n = 21) from across the USA, and from a single library derived from a lab strain of biotype 4 (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> 2017</a>). The wild-caught samples were sequenced using short-read technology (Illumina MiSeq, 300 bp PE, ∼147 × genome coverage) and the biotype 4 lab colony was sequenced using the Pacific Biosciences (PacBio) single molecule real time sequencing (SMRT) platform to generate long-read data (∼25x genome coverage). <a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> (2017)</a> combined all the short-read sequence data from both biotypes to generate an initial <em>de novo</em> assembly that was subsequently scaffolded using the PacBio long reads. This produced a fragmented genome assembly containing 8,397 scaffolds totalling 301 Mb of sequence with a scaffold N50 of 174 Kb (contig N50 = 57 Kb).</p><p id="p-5">Genome assembly algorithms are a source of constant innovation and improvement (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Sedlazeck <em>et al.</em> 2018</a>). This is particularly true in the field of long-read genome assembly and in the integration of short- and long-read data (hybrid genome assembly). However, despite the public deposition of data, genome assemblies of non-model organisms are rarely revisited. Here, I leverage recently developed methods to reassemble the soybean aphid genome using the original sequence data. The version 2 genome assembly is highly contiguous, containing half of the genome in only 40 scaffolds (contig N50 = 2.00 Mb, scaffold N50 = 2.51 Mb), and has improved accuracy at the gene-level, with the representation of conserved single-copy arthropod genes (n = 1,066) increased by 11% compared to version 1 (983 <em>vs.</em> 888). To demonstrate the utility of the updated genome assembly, I investigate synteny of the insect-specific gene family <em>Osiris</em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Shah <em>et al.</em> 2012</a>) between <em>Drosophila melanogaster</em> and <em>A. glycines</em>. The updated assembly of <em>A. glycines</em> resolves the complete <em>Osiris</em> gene cluster and reveals conserved synteny between aphids and flies over approximately 400 million years (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Johnson <em>et al.</em> 2018</a>). The updated genome assembly and annotation of <em>A. glycines</em> will provide a solid foundation to understand the biology of <em>A. glycines</em> and other aphid species.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Assessment of A. glycines v1 and other aphid genomes</h3><p id="p-6">I assessed gene-level completeness of <em>A. glycines</em> v1 (A_gly_v1) and other published aphid genome assemblies using Benchmarking Universal Single-Copy Orthologs (BUSCO) v3.0 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Simão <em>et al.</em> 2015</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Waterhouse <em>et al.</em> 2018</a>) with the Arthropoda gene set (n = 1,066) using default settings. I obtained the publicly available genome assemblies of A_gly_v1 (<a id="xref-ref-62-5" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> 2017</a>), <em>Acyrthosiphon pisum</em> (LSR1) v2.1b (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">International Aphid Genomics Consortium 2010</a>), <em>Diuraphis noxia</em> (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Nicholson <em>et al.</em> 2015</a>), <em>Myzus cerasi</em> (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Thorpe <em>et al.</em> 2018</a>), <em>Myzus persicae</em> (G006) v1.1 (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Mathers <em>et al.</em> 2017</a>) and <em>Rhopalosiphum padi</em> (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Thorpe <em>et al.</em> 2018</a>) from AphidBase (bipaa.genouest.org/is/aphidbase/). I also included the recently published high-quality assembly of <em>Rhopalosiphum maidis</em> (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Chen <em>et al.</em> 2019</a>).</p><p id="p-7">To check for contamination in A_gly_v1, I generated a taxon-annotated GC content-coverage plot using BlobTools v0.9.19 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kumar <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Laetsch and Blaxter 2017</a>). Each scaffold in A_gly_v1 was annotated with taxonomy information based on BLASTN v2.2.31 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Camacho <em>et al.</em> 2009</a>) searches against the NCBI nucleotide database (nt, downloaded 13/10/2017) with the options “-outfmt ‘6 qseqid staxids bitscore std sscinames sskingdoms stitle’ -culling_limit 5 -evalue 1e-25”. To calculate average coverage per scaffold, I mapped Illumina MiSeq paired-end sequence data derived from wild-caught <em>A. glycines</em> biotype 4 samples (13 libraries) from <a id="xref-ref-62-6" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> (2017)</a> to A_gly_v1 using BWA-MEM v0.7.7 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li 2013</a>) with default settings. The resulting BAM file was sorted with SAMtools v1.3 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2009</a>) and passed to BlobTools along with the table of BLASTN results.</p></div><div id="sec-3" class="subsection"><h3>Reassembly of A. glycines biotype 4</h3><p id="p-8">The BlobTools screen of A_gly_v1 revealed contamination likely derived from parasitoid wasp larvae (see Results and Discussion). To create an initial contamination free assembly of <em>A. glycines</em>, I <em>de novo</em> assembled the PacBio long-read sequence data from <a id="xref-ref-62-7" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> (2017)</a> with Canu v1.6 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Koren <em>et al.</em> 2017</a>) using recommended settings for low coverage sequence data: “genomeSize=317m corMinCoverage=0 correctedErrorRate=0.105 gnuplotTested=true ovsMethod=sequential -pacbio-raw”. These data were derived from a lab reared colony of biotype 4 which was unlikely to be contaminated by parasitoid wasp larvae. The Canu assembly was checked for contamination by creating a GC-content coverage plot using KAT sect from the K-mer analysis toolkit (KAT) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mapleson <em>et al.</em> 2017</a>). For this analysis, all biotype 4 Illumina MiSeq libraries from <a id="xref-ref-62-8" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> (2017)</a> were used to calculate median K-mer coverage per scaffold.</p><p id="p-9">To generate a hybrid PacBio-Illumina assembly <em>of A. glycines</em> biotype 4, I first identified libraries that were likely the source of contamination in A_gly_v1 using the contamination-free Canu assembly. I mapped each biotype 4 Illumina MiSeq library to the Canu assembly with NextGenMap v0.5.5 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sedlazeck <em>et al.</em> 2013</a>) using default settings and discarded libraries where less than 75% of reads aligned as valid pairs. The retained libraries were then trimmed for adapter sequence with Trim_galore! v0.4.5 (“–quality 0 –paired–length 150”) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Krueger 2015</a>), concatenated, and assembled with DISCOVAR <em>de novo</em> (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Weisenfeld <em>et al.</em> 2014</a>) with default settings. The discovar assembly was then used as the input for hybrid genome assembly with DBG2OLC (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Ye <em>et al.</em> 2016</a>). DBG2OLC was run using all the PacBio data from <a id="xref-ref-62-9" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> (2017)</a> and with the following settings: “k 17 KmerCovTh 2 MinOverlap 20 AdaptiveTh 0.002 LD1 0 MinLen 200”. A consensus of the resulting backbone assembly was then generated using BLASR (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chaisson and Tesler 2012</a>) and PBDagCon (<a href="http://github.com/PacificBiosciences/pbdagcon">github.com/PacificBiosciences/pbdagcon</a>) as per recommendations in <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chakraborty <em>et al.</em> (2016)</a>. The DBG2OLC consensus assembly was then checked for contamination using a GC-content coverage plot with KAT sect as for the Canu assembly.</p><p id="p-10">To create the final <em>A. glycines</em> biotype 4 contig assembly I merged the Canu and DBG2OLC hybrid assemblies with quickmerge v0.2 (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chakraborty <em>et al.</em> 2016</a>). I used the Canu assembly as the “query” and the DBG2OLC assembly as the “reference” so that the majority of content in the merged assembly would be derived from the Canu assembly (see quickmerge documentation). The length cut-off for anchor contigs (“-l”) was set to the N50 of the Canu assembly (409,248) and the minimum alignment length (“-lm”) was set to 10 kb as per recommendations in the quickmerge documentation, all other settings were left as default. The merged assembly was then polished with the retained biotype 4 Illumina MiSeq libraries using two rounds of Pilon v1.22 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Walker <em>et al.</em> 2014</a>).</p></div><div id="sec-4" class="subsection"><h3>Assembly filtering and detection of endosymbionts</h3><p id="p-11">The polished quickmerge assembly was checked for contamination using two runs of BlobTools, one with per-contig coverage calculated using all the available biotype 4 Illumina MiSeq libraries mapped to the assembly with BWA-MEM v0.7.7 (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Li 2013</a>), and a second run using the PacBio long-reads mapped to the assembly with minimap2 r672 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li 2018</a>). BLASTN searches for the quickmerge assembly BlobTools runs were performed as per A_gly_v1 against the NCBI nt database. The BlobTools analyses revealed the presence of contigs assigned to <em>Buchnera</em> and <em>Wolbachia</em> endosymbionts (Supplementary Table 1), these were filtered from the final assembly. Manual inspection of the BlobPlot also revealed a 1.40 Mb contig assigned to Rickettsiales that had GC content and coverage patterns similar to contigs assigned to Hemiptera (Supplementary Figure 1). Visualization of coverage patterns and BLASTN hits along this contig with IGV v2.5.3 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Robinson <em>et al.</em> 2011</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Thorvaldsdóttir <em>et al.</em> 2012</a>) showed a clear drop in PacBio coverage at 1,097,915 bp coinciding with a transition in BLAST hits from aphid sequences to Wolbachia sequences (Supplementary Figure 2). Given that the intersection between higher and lower coverage regions of the contig was only spanned by a single PacBio read, this contig was considered to be chimeric and was split at 1,097,915 bp and the Wolbachia section removed from the assembly. Additionally, contigs with less than 15x average MiSeq coverage and less than 10x average PacBio coverage were flagged as low coverage contaminants and removed. Finally, the mitochondrial (mt) genome was identified and removed based on alignment to the <em>M. persicae</em> mt genome (NCBI accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KU877171.1&amp;atom=%2Fggg%2F10%2F3%2F899.atom">KU877171.1</a>) with nucmer v4.0.0.beta2 (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Marçais <em>et al.</em> 2018</a>), and patterns of coverage and GC content obtained from BlobTools.</p></div><div id="sec-5" class="subsection"><h3>RNA-seq scaffolding</h3><p id="p-12">To further increase the contiguity of the updated <em>A. glyicnes</em> biotype 4 assembly, I performed RNA-seq based scaffolding of the filtered quickmerge assembly with P_RNA_scaffolder (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Zhu <em>et al.</em> 2018</a>). 5 Gb of <em>A. glycines</em> paired-end RNA-seq data from <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bansal <em>et al.</em> (2014)</a> were processed with Trim_galore! v0.4.5 to remove adapter sequences and trim bases with quality scores below 20, retaining paired reads that were at least 50 bp long after trimming. The trimmed RNA-seq data were aligned to the filtered quickmerge assembly with HISAT2 v2.0.5 “-k 3 –pen-noncansplice 1000000” (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kim <em>et al.</em> 2015</a>) and the resulting SAM file passed to P_RNA_scaffolder which was run with default settings on the filtered quickmerge assembly. The scaffolded assembly was then sorted by scaffold length and scaffolds named numerically from longest to shortest to create a final release (A_gly_v2). A_gly_v2 was then checked for completeness and contamination with BUSCO and BlobTools respectively, following the procedures outlined for A_gly_v1.</p></div><div id="sec-6" class="subsection"><h3>RNA-seq informed gene prediction with BRAKER2</h3><p id="p-13">Prior to gene prediction, I soft-masked A_gly_v2 with known Insecta repeats from the RepeatMasker library using RepeatMasker v4.0.7 (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tarailo-Graovac and Chen 2009</a>) with default settings. I then mapped the quality and adapter trimmed RNA-seq reads from <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bansal <em>et al.</em> (2014)</a> (also used for genome scaffolding) to the soft-masked assembly with HISAT2 with the following parameters: “–max-intronlen 25000 –dta-cufflinks”. BRAKER2 v2.0.4 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hoff <em>et al.</em> 2015</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">2019</a>) was then used to train AUGUSTUS (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Stanke <em>et al.</em> 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lomsadze <em>et al.</em> 2014</a>) and predict protein-coding genes, incorporating evidence from the RNA-seq alignments. The new and previously published annotations were assessed based on BUSCO completeness and by pseudoalignment of RNA-seq reads from <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Bansal <em>et al.</em> (2014)</a> to the respective transcript sets with kallisto v0.43.1 (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bray <em>et al.</em> 2016</a>) using default settings.</p></div><div id="sec-7" class="subsection"><h3>Synteny analysis with A_gly_v1, A_gly_v2 and D. melanogaster</h3><p id="p-14">I investigated synteny between A_gly_v1 and A_gly_v2, and between A_gly_V2 and <em>D. melanogaster</em>, based on the identification of colinear blocks of genes with MCScanX v1.1 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012</a>). For the <em>D. melanogaster</em> comparisons, I downloaded the R6.22 version of the genome assembly and gene predictions from FlyBase (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Thurmond <em>et al.</em> 2018</a>). In all cases, where multiple transcripts were annotated, I used the longest transcript per gene as the representative transcript. For each comparison, I carried out an all <em>vs.</em> all BLAST search of annotated protein sequences using BLASTALL v2.2.22 with the options: “-p BLASTP -e 1e-10 -b 5 -v 5 -m8”. I then ran MCScanX, requiring at least three consecutive genes to call a colinear block (“-s 3”) and only searching for inter-species blocks (“-b 2”), all other settings were left as default. The MCScanX results were visualized with SynVisio (synvisio.github.io/#/) and the dual_synteny_plotter application from the MCScanX package.</p></div><div id="sec-8" class="subsection"><h3>Annotation and analysis of Osiris genes</h3><p id="p-15">I used a combination of synteny, BLAST searches and phylogenetic analysis to annotate <em>Osiris</em> genes in A_gly_v1 and A_gly_v2. I extracted protein sequences of all annotated <em>Osiris</em> genes from the R6.22 <em>D. melanogaster</em> gene set and carried out BLASTP searches against A_gly_v1 and A_gly_v2 proteins with an e-value cut-off of 1 × 10<sup>−5</sup>. I then combined all identified <em>Osiris</em> proteins and aligned them with MUSCLE v3.8.31 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Edgar 2004</a>) using default settings. Based on the MUSCLE alignment, I carried out Maximum Likelihood (ML) phylogenetic analysis with FastTree v2.1.7 (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Price <em>et al.</em> 2010</a>) using default settings, and visualized the resulting tree with FigTree (<a href="http://github.com/rambaut/figtree">github.com/rambaut/figtree</a>). A_gly_v1 and A_gly_v2 <em>Osiris</em> genes were named based on their closest relative to <em>D. melanogaster Osiris</em> genes in the ML tree. <em>A. glycines Osiris</em> genes without clear <em>D. melanogaster</em> ortholog were given and N(n) suffix. Syntenic relationships between A_gly_v1, A_gly_v2 and <em>D. melanogaster</em> with visualized using the SimpleSynteny web server (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Veltri <em>et al.</em> 2016</a>).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-16">A summary of sequence data used in this manuscript is given in Supplementary Table 2. The final and intermediate genome assemblies (including <em>A. glycines</em> mitochondrial and endosymbiont genomes) and genome annotations generated for this study are available from Zenodo (doi: 10.5281/zenodo.3453468). The final A_gly_v2 assembly is also available from NCBI (PRJNA312843). Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11674005">https://doi.org/10.25387/g3.11674005</a>.</p></div></div><div class="section results-discussion" id="sec-10"><h2 class="">Results and Discussion</h2><div id="sec-11" class="subsection"><h3>Assessment of Aphis glycines v1</h3><p id="p-17">I assessed the quality of A_gly_v1 and a selection of published aphid genome assemblies by searching for conserved single-copy genes using BUSCO (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Simão <em>et al.</em> 2015</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Waterhouse <em>et al.</em> 2018</a>) with the Arthropoda gene set (n = 1066). A_gly_v1 contains full length copies of 93.9% of arthropod BUSCOs (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), indicating a high level of genome completeness. However, compared to other aphid genome assemblies, A_gly_v1 has more than twice as many duplicated BUSCO genes (10.6% <em>vs.</em> 2.3–4.7%). This unusual result could be a genuine biological phenomenon or an artifact of the assembly process caused by fragmentation, un-collapsed heterozygosity (separately assembled alleles) or contamination.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/899/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessment of the Aphis glycines v1 (A_gly_v1) genome assembly. (A) BUSCO analysis of published aphid genome assemblies using the Arthropoda gene set of 1,066 conserved single-copy genes. Bars show the proportions of genes found in each assembly as a percentage of the total gene set. (B) Taxon-annotated GC content-coverage plot of A_gly_v1. Each circle represents a scaffold in the assembly, scaled by length, and colored by order-level NCBI taxonomy assigned by BlobTools. The X axis corresponds to the average GC content of each scaffold and the Y axis corresponds to the average coverage based on alignment of pooled A. glycines Illumina MiSeq short-read libraries from Wenger et al. (2017). Marginal histograms show cumulative genome content (in Kb) for bins of coverage (Y axis) and GC content (X axis)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1722013220" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Assessment of the &lt;em&gt;Aphis glycines&lt;/em&gt; v1 (A_gly_v1) genome assembly. (A) BUSCO analysis of published aphid genome assemblies using the Arthropoda gene set of 1,066 conserved single-copy genes. Bars show the proportions of genes found in each assembly as a percentage of the total gene set. (B) Taxon-annotated GC content-coverage plot of A_gly_v1. Each circle represents a scaffold in the assembly, scaled by length, and colored by order-level NCBI taxonomy assigned by BlobTools. The X axis corresponds to the average GC content of each scaffold and the Y axis corresponds to the average coverage based on alignment of pooled &lt;em&gt;A. glycines&lt;/em&gt; Illumina MiSeq short-read libraries from Wenger &lt;em&gt;et al.&lt;/em&gt; (2017). Marginal histograms show cumulative genome content (in Kb) for bins of coverage (Y axis) and GC content (X axis).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/899/F1.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/3/899/F1.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/899/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/899/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495840" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Assessment of the <em>Aphis glycines</em> v1 (A_gly_v1) genome assembly. (A) BUSCO analysis of published aphid genome assemblies using the Arthropoda gene set of 1,066 conserved single-copy genes. Bars show the proportions of genes found in each assembly as a percentage of the total gene set. (B) Taxon-annotated GC content-coverage plot of A_gly_v1. Each circle represents a scaffold in the assembly, scaled by length, and colored by order-level NCBI taxonomy assigned by BlobTools. The X axis corresponds to the average GC content of each scaffold and the Y axis corresponds to the average coverage based on alignment of pooled <em>A. glycines</em> Illumina MiSeq short-read libraries from <a id="xref-ref-62-10" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> (2017)</a>. Marginal histograms show cumulative genome content (in Kb) for bins of coverage (Y axis) and GC content (X axis).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">Given that few (0.9%) BUSCO genes are fragmented in A_gly_v1, and that <em>A. glycines</em> has low genetic diversity across its introduced range in North America (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Orantes <em>et al.</em> 2012</a>; <a id="xref-ref-62-11" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> 2017</a>), I focused on contamination as the likely source of elevated duplication levels. To identify contamination in A_gly_v1 I generated a taxon-annotated GC content-coverage plot (known as a “blob plot”) of all A_gly_v1 scaffolds with BlobTools (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">McLean <em>et al.</em> 2018</a>). This revealed two distinct “blobs”, and an additional group of scaffolds with low coverage and high GC content, indicating the presence of contamination (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>; Supplementary Table 3). Scaffolds in the primary “blob” account for the majority of A_gly_v1 sequence and are mostly assigned to Hemiptera as expected. Scaffolds in the secondary “blob” are assigned to Hymenoptera and have lower average coverage (28x <em>vs.</em> 118x) and higher average GC content (32.5% <em>vs.</em> 27.8%) than those assigned to Hemiptera, indicating they are derived from different genomes (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kumar <em>et al.</em> 2013</a>). The likely explanation for this is that some sequence libraries used in the original assembly were derived from wild-caught aphids infected with parasitoid wasp larvae. In total, 1,001 out of 8,397 A_gly_v1 scaffolds (totalling 3.93 Mb of sequence) are assigned to Hymenoptera. This likely represents an underestimation of the hymenopteran content in A_gly_v1 as there are many scaffolds with unannotated taxonomy also clustering with the Hymenoptera scaffolds due to a lack of sequenced aphid parasitoid wasp genomes. Consistent with this, BLAST hit identities for Hymenoptera scaffolds are significantly lower than for Hemiptera scaffolds (Mann-Whitney U Test: <em>P</em> &lt; 2.2x10<sup>−16</sup>, U = 1552300000; Supplementary Figure 3). Nonetheless, inspection of the A_gly_v1 official gene set (v1.0) reveals that scaffolds assigned to Hymenoptera contain 806 genes previously thought to be derived from <em>A. glycines</em>. These scaffolds also account for 68 out of 113 of the duplicated BUSCO genes. This is a serious problem as contaminated genome assemblies have the potential to significantly affect downstream comparative analysis (<em>e.g.</em>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Koutsovoulos <em>et al.</em> 2016</a>).</p></div><div id="sec-12" class="subsection"><h3>Reassembly of A. glycines biotype 4</h3><p id="p-20">A_gly_v1 was assembled from data derived from wild-caught aphids (short-read data, ∼147x coverage) and from a lab-reared biotype 4 colony (PacBio long-read data, ∼25x coverage) (<a id="xref-ref-62-12" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> 2017</a>). I reasoned that the lab-reared colony was unlikely to be contaminated with parasitoid wasps and could be used to generate a clean <em>A. glycines</em> genome assembly. I therefore took advantage of improvements to the Canu genome assembler (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Koren <em>et al.</em> 2017</a>) that enable assembly of low coverage long-read datasets, to generate an initial <em>de novo</em> assembly of the <em>A. glycines</em> PacBio dataset. This resulted in 1,967 contigs, totalling 301 Mb of sequence, with an N50 of 409 Kb. Contigs from this assembly formed a single blob around the expected GC content, with no evidence of parasitoid wasp contamination (Supplementary Figure 4).</p><p id="p-21">Although the Canu assembly is approximately 7 times more contiguous than A_gly_v1 (409 Kb <em>vs.</em> 57 Kb; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>), further gains may still be possible using alternative assembly strategies. It has been shown that combining PacBio-only assemblies with hybrid assemblies that use both long- and short-read data can dramatically improve contiguity due to the complementary assembly of different genome regions (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Chakraborty <em>et al.</em> 2016</a>). To produce a hybrid assembly of <em>A. gylcines</em> biotype 4, I first identified contamination free <em>A. glycines</em> short-read libraries sequenced by <a id="xref-ref-62-13" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> (2017)</a> to be used to create an accurate de Bruijn graph-based short-read assembly to act as the seed for a PacBio hybrid genome assembly. Excluding contamination before assembly is preferable to post-assembly filtering in this instance as the parasitoid wasp and aphid have similar GC content (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>), making it difficult to distinguish between target species contigs and contamination. To identify libraries that contain high levels of contamination, I mapped the Illumina libraries derived from wild-caught biotype 4 aphids (n = 13) to the Canu assembly and set aside libraries with low mapping efficiency (&lt; 75% of reads mapped). Eleven biotype 4 Illumina libraries passed this filtering step (Supplementary Figure 5) and were assembled with DISCOVAR <em>de novo</em> (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Weisenfeld <em>et al.</em> 2014</a>). This short-read assembly was then used as an input to DBG2OLC (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Ye <em>et al.</em> 2016</a>) to generate a hybrid assembly with the PacBio dataset. The DBG2OLC assembly contained 824 contigs totalling 284 Mb of sequence with a contig N50 of 703 Kb. Next, I used quickmerge (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Chakraborty <em>et al.</em> 2016</a>) to combine the Canu and DBG2OLC assemblies. This further increased contiguity, producing a merged assembly containing 1,068 contigs totalling 306 Mb of sequence with a contig N50 of 2.06 Mb, a 35-fold increase in contig-level N50 compared to A_gly_V1 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). This assembly was then subjected to two rounds of polishing with Pilon (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Walker <em>et al.</em> 2014</a>) followed by filtering to remove contigs derived from mitochondrial DNA, and <em>Buchnera</em> and <em>Wolbachachia</em> endosymbionts (Supplementary Figure 1). Finally, the polished and filtered assembly was scaffolded using <em>A. glycines</em> RNA-seq data (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Bansal <em>et al.</em> 2014</a>) with P_RNA_scaffolder (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Zhu <em>et al.</em> 2018</a>) to produce version 2 of the <em>A. glycines</em> genome assembly (A_gly_v2).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/899/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The updated assembly of A. glycines (A_gly_v2) is contiguous, complete, and free from contamination. (A) Cumulative scaffold (or contig) length for the original assembly of A. glycines (A_gly_v1), the PacBio only assembly (Canu), the hybrid assembly (DBG2OLC), the merged PacBio only and hybrid assembly (quickmerge) and the final updated assembly that includes scaffolding with RNA-seq data and removal of contaminants (A_gly_v2). Summary statistics for each assembly are given in Table 1. (B) BUSCO analysis of A_gly_v1 and A_gly_v2 using the Arthropoda gene set (n = 1,066). (C) Taxon-annotated GC content-coverage plot of A_gly_v2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1722013220" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The updated assembly of &lt;em&gt;A. glycines&lt;/em&gt; (A_gly_v2) is contiguous, complete, and free from contamination. (A) Cumulative scaffold (or contig) length for the original assembly of &lt;em&gt;A. glycines&lt;/em&gt; (A_gly_v1), the PacBio only assembly (Canu), the hybrid assembly (DBG2OLC), the merged PacBio only and hybrid assembly (quickmerge) and the final updated assembly that includes scaffolding with RNA-seq data and removal of contaminants (A_gly_v2). Summary statistics for each assembly are given in Table 1. (B) BUSCO analysis of A_gly_v1 and A_gly_v2 using the Arthropoda gene set (n = 1,066). (C) Taxon-annotated GC content-coverage plot of A_gly_v2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/899/F2.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/3/899/F2.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/899/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/899/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495940" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">The updated assembly of <em>A. glycines</em> (A_gly_v2) is contiguous, complete, and free from contamination. (A) Cumulative scaffold (or contig) length for the original assembly of <em>A. glycines</em> (A_gly_v1), the PacBio only assembly (Canu), the hybrid assembly (DBG2OLC), the merged PacBio only and hybrid assembly (quickmerge) and the final updated assembly that includes scaffolding with RNA-seq data and removal of contaminants (A_gly_v2). Summary statistics for each assembly are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. (B) BUSCO analysis of A_gly_v1 and A_gly_v2 using the Arthropoda gene set (n = 1,066). (C) Taxon-annotated GC content-coverage plot of A_gly_v2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">A_gly_v2 is contiguous, free from obvious contamination and is highly complete. Half of the genome is contained in only 40 scaffolds (941 scaffolds in total, longest scaffold = 7.28 Mb) and the scaffold N50 is increased by 1,342% (2.51 Mb <em>vs.</em> 0.17 Mb) compared to A_gly_v1 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>;<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>). After exclusion of parasitoid wasp scaffolds from A_gly_v1, A_gly_v2 contains 4 Mb more sequence than A_gly_v1 (303 Mb <em>vs.</em> 299 Mb), and is close to the predicted genome size of 317 Mb based on flow cytometry (<a id="xref-ref-62-14" class="xref-bibr" href="#ref-62">Wenger <em>et al.</em> 2017</a>). Furthermore, A_gly_v2 contains 95 more single-copy and complete BUSCO genes than A_gly_v1 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>; 983 <em>vs.</em> 888) and is free from obvious contamination of non-target species (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). This improvement in gene content is the result of a reduction in the number of fragmented or missing genes (45 <em>vs.</em> 65) and greatly reduced duplication levels (38 <em>vs.</em> 113). Using BRAKER2 (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Hoff <em>et al.</em> 2015</a>; <a href="http://github.com/Gaius-Augustus/BRAKER">github.com/Gaius-Augustus/BRAKER</a>) with RNA-seq evidence from <a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Bansal <em>et al.</em> (2014)</a>, I annotated 19,750 protein-coding genes in A_gly_v2 to generate a new gene set release to accompany the updated genome assembly. As for the genome assembly, the A_gly_v2 gene set is more complete than the A_gly_v1 gene set, containing 31 more complete BUSCO genes (1016 <em>vs.</em> 985; Supplementary Table 4) and has a greater number of genes expressed (kallisto estimated counts &gt; 10) in the <a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Bansal <em>et al.</em> (2014)</a> RNA-seq data set (12,637 <em>vs.</em> 11,925; Supplementary Table 5).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495872/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495872/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>Assembly and annotation statistics for the original <em>A. glycines</em> genome assembly (A_gly_v1) and alternative assemblies of <em>A. glycines</em> generated in this study (see main text)</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Resolution of the aphid Osiris gene cluster</h3><p id="p-26"><em>Osiris</em> is a large insect-specific gene family that has retained high levels of synteny during insect evolution (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Shah <em>et al.</em> 2012</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Smith <em>et al.</em> 2018</a>), making it an ideal case study for assessing genome assembly quality. I annotated <em>Osiris</em> gene family members in A_gly_v1 and A_gly_v2 based on genome-wide synteny analysis with <em>Drosophila melanogaster</em>, BLAST searches with <em>D. melanogaster Osiris</em> genes and phylogenetic analysis (Supplementary Figure 6; Supplementary Table 6). In total, I identified 29 <em>Osiris</em> genes in A_gly_v2. Of these, 19 are located on a single 5 Mb scaffold (scaffold_7). In comparison, scaffold_7 is broken up into 19 parts in A_gly_v1, with the <em>Osiris</em> gene cluster split across 2 scaffolds (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A and B</a>). Synteny with <em>D. melanogaster</em> on A_gly_v2 scaffold_7 is limited to the region containing <em>Osiris</em> genes (Supplementary Figure 7) and, given that A_gly_v2 scaffold_7 extends up- and down-stream several Mb, it appears that the entire <em>A. glycines Osiris</em> gene cluster has been resolved (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A and B</a>). The increased contiguity of A_gly_v2 reveals rearrangements in gene order at the extremity of the <em>Osiris</em> gene cluster, where <em>Osiris</em> 2 and 24 have been shuffled to the 3′ end in A_gly_v2 relative to <em>D. melanogaster</em> (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). This rearrangement was previously hidden by genome assembly fragmentation in A_gly_v1, and in the original pea aphid genome assembly, where <em>Osiris</em> gene order has also been investigated (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Shah <em>et al.</em> 2012</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/899/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The A. glycines Osiris gene cluster is resolved in A_gly_v2 and shares synteny with D. melanogaster. (A) MCScanX gene-level colinearity between A_gly_v2 scaffold_7 and multiple scaffolds from A_gly_v1. The position of the Osiris gene cluster on A_gly_v2 scaffold_7 is indicated by a red arrow. This region is split across two scaffolds in A_gly_v1. (B) Schematic showing synteny between D. melanogaster (Dmel), A_gly_v2 and A_gly_v1 for the Osiris gene cluster. A. glycines Osiris genes are named according to the annotation in Supplementary Table 6. Links are shown between one to one Osiris orthologs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1722013220" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &lt;em&gt;A. glycines Osiris&lt;/em&gt; gene cluster is resolved in A_gly_v2 and shares synteny with &lt;em&gt;D. melanogaster&lt;/em&gt;. (A) MCScanX gene-level colinearity between A_gly_v2 scaffold_7 and multiple scaffolds from A_gly_v1. The position of the &lt;em&gt;Osiris&lt;/em&gt; gene cluster on A_gly_v2 scaffold_7 is indicated by a red arrow. This region is split across two scaffolds in A_gly_v1. (B) Schematic showing synteny between &lt;em&gt;D. melanogaster&lt;/em&gt; (Dmel), A_gly_v2 and A_gly_v1 for the &lt;em&gt;Osiris&lt;/em&gt; gene cluster. &lt;em&gt;A. glycines Osiris&lt;/em&gt; genes are named according to the annotation in Supplementary Table 6. Links are shown between one to one &lt;em&gt;Osiris&lt;/em&gt; orthologs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/899/F3.medium.gif" width="440" height="262"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/3/899/F3.medium.gif" width="440" height="262"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/899/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/899/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495886" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">The <em>A. glycines Osiris</em> gene cluster is resolved in A_gly_v2 and shares synteny with <em>D. melanogaster</em>. (A) MCScanX gene-level colinearity between A_gly_v2 scaffold_7 and multiple scaffolds from A_gly_v1. The position of the <em>Osiris</em> gene cluster on A_gly_v2 scaffold_7 is indicated by a red arrow. This region is split across two scaffolds in A_gly_v1. (B) Schematic showing synteny between <em>D. melanogaster</em> (Dmel), A_gly_v2 and A_gly_v1 for the <em>Osiris</em> gene cluster. <em>A. glycines Osiris</em> genes are named according to the annotation in Supplementary Table 6. Links are shown between one to one <em>Osiris</em> orthologs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Genome-wide synteny between <em>A. glycines</em> and <em>D. melanogaster</em> is restricted to two regions, one containing the <em>Osiris</em> gene cluster, and another containing a cluster of GMC oxidoreductase genes (Supplementary Figure 7), further highlighting the extraordinary conservation of <em>Osiris</em> gene order within insects over hundreds of millions of years (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Shah <em>et al.</em> 2012</a>). Factors that may have selected for the long term conservation of <em>Osiris</em> gene order remain elusive (<a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Shah <em>et al.</em> 2012</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Smith <em>et al.</em> 2018</a>). However, several lines of evidence indicate <em>Osiris</em> genes may be important in the biology of aphids and other plant-feeding insects. <em>Osiris 7</em> has been identified in the saliva of the wheat-feeding aphid <em>Diuraphis noxia</em>, indicating it could be involved in insect-plant interactions (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Nicholson <em>et al.</em> 2012</a>), and <em>Osiris</em> 5 is associated with recurrent adaptation to toxic fruit in <em>Drosophila</em> (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Yassin <em>et al.</em> 2016</a>). In <em>A. glycines</em>, phylogenetic analysis shows expansion of genes related to <em>Osiris 5</em> and <em>Osiris 16</em>, with six copies present (Supplementary Figure 6). This expansion is likely a conserved feature of aphid genomes as six copies of <em>Osiris 5 / 16</em> are also found in <em>A. pisum</em> (<a id="xref-ref-46-6" class="xref-bibr" href="#ref-46">Shah <em>et al.</em> 2012</a>). The role of <em>Osiris</em> gene family members in aphid-plant interactions warrants further study.</p></div><div id="sec-14" class="subsection"><h3>Conclusions</h3><p id="p-29">Reassembly of the soybean aphid genome highlights the potential to gain new insights by revisiting old datasets and applying new analysis approaches. Through curation and reanalysis of existing data I have achieved a large increase in genome contiguity and completeness relative to A_gly_V1. This new genome assembly of <em>A. glycines</em> adds to the small but growing collection of high-quality aphid genome assemblies and will provide a solid foundation for future studies of <em>A. glycines</em> biology, and for comparative genomic analysis with other aphid species.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-30">The described work was supported by a BBSRC Future Leader Fellowship (BB/R01227X/1) awarded to the author and CEPAMS grant 17.03.2 International Strategy Fund of the John Innes Centre. This research was supported in part by the NBI Computing Infrastructure for Science Group, which provides technical support and maintenance to the John Innes Centre’s high-performance computing cluster and storage systems. I thank Prof. Andy Michell and Dr. Jacob Wenger for providing access to sequence data from the original <em>A. glycines</em> genome assembly project. I also thank Prof. Saskia Hogenhout for her support throughout the project and Prof. Cock Van Oosterhout for valuable comments on the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-31">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11674005">https://doi.org/10.25387/g3.11674005</a>.</p></li><li class="fn" id="fn-4"><p id="p-32"><em>Communicating editor: P. Ingvarsson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-11-28"><span class="received-label">Received </span>November 28, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2020-01-22"><span class="accepted-label">Accepted </span>January 22, 2020.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Mathers</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.1" data-doi="10.1186/1471-2164-15-972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Mittapalli</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Michel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RNA-Seq reveals a xenobiotic stress response in the soybean aphid, <em>Aphis glycines</em>, when fed aphid-resistant soybean.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">972</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-972</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D972%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-972%26rft_id%253Dinfo%253Apmid%252F25399334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-972&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25399334&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.2" data-doi="10.1038/nbt.3519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bray</span>,  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Pimentel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Melsted</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pachter</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Near-optimal probabilistic RNA-seq quantification.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">527</span> (erratum: Nat. Biotechnol. 34: 888).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3519</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3519%26rft_id%253Dinfo%253Apmid%252F27043002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27043002&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.3" data-doi="10.1186/1471-2105-10-421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Camacho</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Coulouris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Avagyan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Papadopoulos</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">BLAST+: architecture and applications.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">421</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-10-421</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DCamacho%26rft.auinit1%253DC.%26rft.volume%253D10%26rft.spage%253D421%26rft.epage%253D421%26rft.atitle%253DBLAST%252B%253A%2Barchitecture%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-421%26rft_id%253Dinfo%253Apmid%252F20003500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-421&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20003500&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.4" data-doi="10.1038/nature13907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span>,  <span class="cit-name-given-names">M. J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Huddleston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. Y.</span> <span class="cit-name-surname">Dennis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">Sudmant</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Malig</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Resolving the complexity of the human genome using single-molecule sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">517</span>: <span class="cit-fpage">608</span>–<span class="cit-lpage">611</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13907</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D517%26rft.spage%253D608%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13907%26rft_id%253Dinfo%253Apmid%252F25383537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25383537&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.5" data-doi="10.1186/1471-2105-13-238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span>,  <span class="cit-name-given-names">M. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tesler</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR): application and theory.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">238</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-13-238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DChaisson%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D238%26rft.epage%253D238%26rft.atitle%253DMapping%2Bsingle%2Bmolecule%2Bsequencing%2Breads%2Busing%2BBasic%2BLocal%2BAlignment%2Bwith%2BSuccessive%2BRefinement%2B%2528BLASR%2529%253A%2BTheory%2Band%2BApplication.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-238%26rft_id%253Dinfo%253Apmid%252F22988817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22988817&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.6" data-doi="10.1093/nar/gkv1289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. G.</span> <span class="cit-name-surname">Baldwin-Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Long</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Emerson</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Contiguous and accurate de novo assembly of metazoan genomes with modest long read coverage.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1289%26rft_id%253Dinfo%253Apmid%252F26621913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26621913&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.7" data-doi="10.1038/s41588-017-0010-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. W.</span> <span class="cit-name-surname">VanKuren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kalsow</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Hidden genetic variation shapes the structure of functional elements in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41588-017-0010-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D50%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41588-017-0010-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41588-017-0010-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.8" data-doi="10.1093/gigascience/giz033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Shakir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bigham</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Richter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Fei</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Genome sequence of the corn leaf aphid (<em>Rhopalosiphum maidis</em> Fitch).</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">giz033</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/giz033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D8%26rft.spage%253Dgiz033%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiz033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/giz033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.9" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.10" data-doi="10.1016/j.copbio.2004.01.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferry</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Edwards</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Gatehouse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Gatehouse</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Plant – insect interactions : molecular approaches to insect resistance.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.copbio.2004.01.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.stitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.aulast%253DFerry%26rft.auinit1%253DN.%26rft.volume%253D15%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D161%26rft.atitle%253DPlant-insect%2Binteractions%253A%2Bmolecular%2Bapproaches%2Bto%2Binsect%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2004.01.008%26rft_id%253Dinfo%253Apmid%252F15081055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2004.01.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15081055&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221127800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.11" data-doi="10.1038/35091062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawthorne</span>,  <span class="cit-name-given-names">D. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Via</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetic linkage of ecological specialization and reproductive isolation in pea aphids.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">412</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">907</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35091062</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHawthorne%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D412%26rft.issue%253D6850%26rft.spage%253D904%26rft.epage%253D907%26rft.atitle%253DGenetic%2Blinkage%2Bof%2Becological%2Bspecialization%2Band%2Breproductive%2Bisolation%2Bin%2Bpea%2Baphids.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35091062%26rft_id%253Dinfo%253Apmid%252F11528477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35091062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11528477&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170689000043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.12" data-doi="10.1093/bioinformatics/btv661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoff</span>,  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lange</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lomsadze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Borodovsky</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stanke</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BRAKER1: Unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv661</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv661%26rft_id%253Dinfo%253Apmid%252F26559507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26559507&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.899.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoff</span>,  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lomsadze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Borodovsky</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stanke</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Whole-genome annotation with BRAKER</span>, pp. <span class="cit-fpage">65</span>–<span class="cit-lpage">95</span> in <span class="cit-source">Gene Prediction: Methods and Protocols</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kollmar</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.14" data-doi="10.1016/j.pbi.2011.05.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hogenhout</span>,  <span class="cit-name-given-names">S. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. I. B.</span> <span class="cit-name-surname">Bos</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effector proteins that modulate plant – insect interactions.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">422</span>–<span class="cit-lpage">428</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.pbi.2011.05.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DHogenhout%26rft.auinit1%253DS.%2BA.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D422%26rft.epage%253D428%26rft.atitle%253DEffector%2Bproteins%2Bthat%2Bmodulate%2Bplant--insect%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2011.05.003%26rft_id%253Dinfo%253Apmid%252F21684190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2011.05.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21684190&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.15" data-doi="10.1371/journal.pbio.1000313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Aphid Genomics Consortium</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome sequence of the pea aphid <em>Acyrthosiphon pisum</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1000313</span> (erratum: PLoS Biol. 16: e3000029).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1000313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253De1000313%26rft.epage%253De1000313%26rft.atitle%253DGenome%2Bsequence%2Bof%2Bthe%2Bpea%2Baphid%2BAcyrthosiphon%2Bpisum.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000313%26rft_id%253Dinfo%253Apmid%252F20186266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20186266&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.16" data-doi="10.1093/gbe/evy015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaquiéry</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Peccoud</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ouisse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Legeai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Prunier-Leterme</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Disentangling the causes for faster-X evolution in aphids.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">520</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evy015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D10%26rft.spage%253D507%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevy015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evy015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.17" data-doi="10.1371/journal.pgen.1003690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaquiéry</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Rispe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Roze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Legeai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Le Trionnaire</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Masculinization of the X chromosome in the pea aphid.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1003690 </span>.<span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003690</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003690%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003690%26rft_id%253Dinfo%253Apmid%252F23950732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23950732&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.18" data-doi="10.1093/gigascience/giz101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Qin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Yin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhang</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">A chromosome-level draft genome of the grain aphid <em>Sitobion miscanthi</em>.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">8</span>: giz101.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/giz101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiz101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/giz101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.19" data-doi="10.1073/pnas.1815820115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>,  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Dietrich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Friedrich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Beutel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Wipfler</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Phylogenomics and the evolution of hemipteroid insects.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">12775</span>–<span class="cit-lpage">12780</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1815820115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1815820115%26rft_id%253Dinfo%253Apmid%252F30478043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTE1LzUwLzEyNzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy84OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.20" data-doi="10.1093/molbev/msz261"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Julca</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Marcet-houben</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Cruz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Vargas-chavez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Spencer</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Phylogenomics identifies an ancestral burst of gene duplications predating the diversification of Aphidomorpha.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msz261</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsz261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msz261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.21" data-doi="10.1038/nmeth.3317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Langmead</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HISAT: A fast spliced aligner with low memory requirements.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">360</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.3317</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D12%26rft.spage%253D357%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.3317%26rft_id%253Dinfo%253Apmid%252F25751142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.3317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25751142&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.22" data-doi="10.1101/gr.215087.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">Walenz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Berlin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Miller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. H.</span> <span class="cit-name-surname">Bergman</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Canu: Scalable and accurate long-read assembly via adaptive κ-mer weighting and repeat separation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.215087.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMTAvMy84OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.23" data-doi="10.1073/pnas.1600338113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koutsovoulos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kumar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Laetsch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Stevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Daub</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">No evidence for extensive horizontal gene transfer in the genome of the tardigrade <em>Hypsibius dujardini</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">5053</span>–<span class="cit-lpage">5058</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1600338113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1600338113%26rft_id%253Dinfo%253Apmid%252F27035985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzE4LzUwNTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.24"><div class="cit-metadata unstructured">Krueger, F., 2015 Trim galoreA wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files. A wrapper tool around Cutadapt FastQC to consistently apply Qual. Adapt. trimming to FastQ files. <a href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.25" data-doi="10.3389/fgene.2013.00024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Koutsovoulos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Clarke</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Blaxter</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Blobology: exploring raw genome data for contaminants, symbionts, and parasites using taxon-annotated GC-coverage plots.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2013.00237</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D4%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2013.00024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2013.00024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.26" data-doi="10.12688/f1000research.12232.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laetsch</span>,  <span class="cit-name-given-names">D. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Blaxter</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">BlobTools: Interrogation of genome assemblies.</span> <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1287</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/f1000research.12232.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft.volume%253D6%26rft.spage%253D1287%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.12232.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.12232.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-10.3.899.27"><div class="cit-metadata unstructured">Li, H., 2013 Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. <a href="https://arxiv.org/abs/1303.3997">https://arxiv.org/abs/1303.3997</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.28" data-doi="10.1093/bioinformatics/bty191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Minimap2: pairwise alignment for nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">3094</span>–<span class="cit-lpage">3100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty191</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D34%26rft.spage%253D3094%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty191%26rft_id%253Dinfo%253Apmid%252F29750242%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29750242&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.29" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.30" data-doi="10.1186/s12859-018-2026-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hunt</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. J.</span> <span class="cit-name-surname">Tsai</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Inferring synteny between genome assemblies: A systematic evaluation.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-018-2026-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D19%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-018-2026-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-018-2026-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.31" data-doi="10.1093/nar/gku557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lomsadze</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Burns</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Borodovsky</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Integration of mapped RNA-Seq reads into automatic training of eukaryotic gene finding algorithm.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-elocation-id">e119</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku557</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku557%26rft_id%253Dinfo%253Apmid%252F24990371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku557&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24990371&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.32" data-doi="10.1093/bioinformatics/btw663"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mapleson</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. G.</span> <span class="cit-name-surname">Accinelli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kettleborough</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wright</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Clavijo</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">KAT: A K-mer analysis toolkit to quality control NGS datasets and genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">574</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D574%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw663%26rft_id%253Dinfo%253Apmid%252F27797770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw663&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27797770&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.33" data-doi="10.1371/journal.pcbi.1005944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marçais</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Delcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Phillippy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Coston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">MUMmer4: A fast and versatile genome alignment system.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e1005944</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1005944</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D14%26rft.spage%253De1005944%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1005944%26rft_id%253Dinfo%253Apmid%252F29373581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1005944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29373581&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.34" data-doi="10.1186/s13059-016-1145-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathers</span>,  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Kaithakottil</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Legeai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. T.</span> <span class="cit-name-surname">Mugford</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Rapid transcriptional plasticity of duplicated gene clusters enables a clonally reproducing aphid to colonise diverse plant species.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">27</span> (erratum: Genome Biol. 18: 63).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13059-016-1145-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D18%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-016-1145-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-016-1145-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.35" data-doi="10.1093/gigascience/giy034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLean</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Berger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Laetsch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Schwartz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Blaxter</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Improving the annotation of the <em>Heterorhabditis bacteriophora</em> genome.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gigascience/giy034</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgigascience%252Fgiy034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gigascience/giy034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.36" data-doi="10.1016/j.jprot.2012.01.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicholson</span>,  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. D.</span> <span class="cit-name-surname">Hartson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Puterka</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Proteomic analysis of secreted saliva from Russian Wheat Aphid (<em>Diuraphis noxia</em> Kurd.) biotypes that differ in virulence to wheat.</span> <abbr class="cit-jnl-abbrev">J. Proteomics</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">2252</span>–<span class="cit-lpage">2268</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jprot.2012.01.031</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BProteomics%26rft.aulast%253DNicholson%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D75%26rft.issue%253D7%26rft.spage%253D2252%26rft.epage%253D2268%26rft.atitle%253DProteomic%2Banalysis%2Bof%2Bsecreted%2Bsaliva%2Bfrom%2BRussian%2BWheat%2BAphid%2B%2528Diuraphis%2Bnoxia%2BKurd.%2529%2Bbiotypes%2Bthat%2Bdiffer%2Bin%2Bvirulence%2Bto%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jprot.2012.01.031%26rft_id%253Dinfo%253Apmid%252F22348819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jprot.2012.01.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22348819&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303136600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.37" data-doi="10.1186/s12864-015-1525-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicholson</span>,  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Nickerson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dean</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Hoyt</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genome of <em>Diuraphis noxia</em>, a global aphid pest of small grains.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">429</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1525-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D429%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1525-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1525-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.38" data-doi="10.1038/hdy.2012.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orantes</span>,  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. R.</span> <span class="cit-name-surname">Mian</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Michel</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maintaining genetic diversity and population panmixia through dispersal and not gene flow in a holocyclic heteroecious aphid species.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">134</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2012.21</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DOrantes%26rft.auinit1%253DL.%2BC.%26rft.volume%253D109%26rft.issue%253D2%26rft.spage%253D127%26rft.epage%253D134%26rft.atitle%253DMaintaining%2Bgenetic%2Bdiversity%2Band%2Bpopulation%2Bpanmixia%2Bthrough%2Bdispersal%2Band%2Bnot%2Bgene%2Bflow%2Bin%2Ba%2Bholocyclic%2Bheteroecious%2Baphid%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2012.21%26rft_id%253Dinfo%253Apmid%252F22549514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2012.21&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22549514&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306567400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.39" data-doi="10.1073/pnas.0811117106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peccoud</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ollivier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Plantegenest</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-C.</span> <span class="cit-name-surname">Simon</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A continuum of genetic divergence from sympatric host races to species in the pea aphid complex.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">7495</span>–<span class="cit-lpage">7500</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0811117106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0811117106%26rft_id%253Dinfo%253Apmid%252F19380742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzE4Lzc0OTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.40" data-doi="10.1111/j.1365-2311.2009.01147.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peccoud</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Simon</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The pea aphid complex as a model of ecological speciation.</span> <abbr class="cit-jnl-abbrev">Ecol. Entomol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">130</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2311.2009.01147.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEntomol.%26rft.volume%253D35%26rft.spage%253D119%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2311.2009.01147.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2311.2009.01147.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.41" data-doi="10.1371/journal.pone.0009490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>,  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Dehal</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Arkin</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">FastTree 2 - Approximately maximum-likelihood trees for large alignments.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-elocation-id">e9490</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0009490</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPrice%26rft.auinit1%253DM.%2BN.%26rft.volume%253D5%26rft.issue%253D3%26rft.spage%253De9490%26rft.epage%253De9490%26rft.atitle%253DFastTree%2B2--approximately%2Bmaximum-likelihood%2Btrees%2Bfor%2Blarge%2Balignments.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009490%26rft_id%253Dinfo%253Apmid%252F20224823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20224823&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.42" data-doi="10.1016/j.ibmb.2018.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quan</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Pan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Zeng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Wu</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Draft genome of the cotton aphid <em>Aphis gossypii</em>.</span> <abbr class="cit-jnl-abbrev">Insect Biochem. Mol. Biol.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ibmb.2018.12.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2BBiochem.%2BMol.%2BBiol.%26rft.volume%253D105%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ibmb.2018.12.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ibmb.2018.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.43" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Thorvaldsdóttir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Winckler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Guttman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Lander</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative Genomics Viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.1754</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.44" data-doi="10.1038/s41576-018-0003-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sedlazeck</span>,  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lee</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Darby</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Schatz</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Piercing the dark matter: bioinformatics of long-range sequencing and mapping.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41576-018-0003-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41576-018-0003-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41576-018-0003-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.45" data-doi="10.1093/bioinformatics/btt468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sedlazeck</span>,  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Rescheneder</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Von Haeseler</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">NextGenMap: fast and accurate read mapping in highly polymorphic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2790</span>–<span class="cit-lpage">2791</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt468%26rft_id%253Dinfo%253Apmid%252F23975764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23975764&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325997500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Dorer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. N.</span> <span class="cit-name-surname">Moriyama</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Christensen</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of a large, conserved, and syntenic gene family in insects. G3 Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D2%26rft.spage%253D313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.47" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.48" data-doi="10.1111/jeb.13238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>,  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Morandin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Noureddine</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pant</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Conserved roles of Osiris genes in insect development, polymorphism and protection.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">529</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jeb.13238</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D31%26rft.spage%253D516%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.13238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.13238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.49" data-doi="10.1093/bioinformatics/btn013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Diekhans</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Baertsch</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Haussler</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Using native and syntenically mapped cDNA alignments to improve de novo gene finding.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btn013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn013%26rft_id%253Dinfo%253Apmid%252F18218656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18218656&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253746400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.50" data-doi="10.1002/0471250953.bi0410s25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tarailo-Graovac</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Chen</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> Using RepeatMasker to identify repetitive elements in genomic sequences. Curr. Protoc. Bioinforma. 1–14.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/0471250953.bi0410s25</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BProtoc%2BBioinformatics%26rft.aulast%253DTarailo-Graovac%26rft.auinit1%253DM.%26rft.volume%253DChapter%2B4%26rft.spage%253DUnit%2B4.10%26rft.epage%253DUnit%2B4.10%26rft.atitle%253DUsing%2BRepeatMasker%2Bto%2Bidentify%2Brepetitive%2Belements%2Bin%2Bgenomic%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F0471250953.bi0410s25%26rft_id%253Dinfo%253Apmid%252F19274634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471250953.bi0410s25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19274634&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.51" data-doi="10.1093/gbe/evy183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorpe</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Escudero-Martinez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J. A. A.</span> <span class="cit-name-surname">Cock</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Eves-Van Den Akker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. I. B. B.</span> <span class="cit-name-surname">Bos</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Shared transcriptional control and disparate gain and loss of aphid parasitism genes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2716</span>–<span class="cit-lpage">2733</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evy183</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D10%26rft.spage%253D2716%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevy183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evy183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.52" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. T.</span> <span class="cit-name-surname">Robinson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Mesirov</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bib/bbs017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.53" data-doi="10.1093/nar/gky1003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thurmond</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Goodman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. B.</span> <span class="cit-name-surname">Strelets</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Attrill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. S.</span> <span class="cit-name-surname">Gramates</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">FlyBase 2.0: the next generation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">D759</span>–<span class="cit-lpage">D765</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky1003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D47%26rft.spage%253DD759%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky1003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky1003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.54" data-doi="10.1603/IPM10010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tilmon</span>,  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. W.</span> <span class="cit-name-surname">Hodgson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">O’Neal</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. W.</span> <span class="cit-name-surname">Ragsdale</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Biology of the soybean aphid, Aphis glycines (Hemiptera: Aphididae) in the United States.</span> <abbr class="cit-jnl-abbrev">J. Integr. Pest Manag.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1603/IPM10016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BIntegr.%2BPest%2BManag.%26rft.volume%253D2%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1603%252FIPM10010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1603/IPM10010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.55" data-doi="10.1038/nrg3117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Treangen</span>,  <span class="cit-name-given-names">T. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Repetitive DNA and next-generation sequencing: computational challenges and solutions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">46</span> (erratum: Nat. Rev. Genet. 13: 146).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D13%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3117%26rft_id%253Dinfo%253Apmid%252F22124482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22124482&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.899.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Emden</span>,  <span class="cit-name-given-names">H. F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Harrington</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Aphids as Crop Pests. CAB International, Oxford, UK.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1079/9781780647098.0000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.57" data-doi="10.1093/nar/gkw330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veltri</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Wight</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Crouch</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">SimpleSynteny: a web-based tool for visualization of microsynteny across multiple species.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">W41</span>–<span class="cit-lpage">W45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw330%26rft_id%253Dinfo%253Apmid%252F27141960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27141960&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.58" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Abeel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Shea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Priest</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Abouelliel</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: An integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e112963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.59" data-doi="10.1093/nar/gkr1293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Debarry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Tan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-elocation-id">e49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr1293</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1293%26rft_id%253Dinfo%253Apmid%252F22217600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22217600&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.60" data-doi="10.1093/molbev/msx319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>,  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Seppey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Simao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Manni</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">BUSCO applications from quality assessments to gene prediction and phylogenomics.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">543</span>–<span class="cit-lpage">548</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx319</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D35%26rft.spage%253D543%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx319%26rft_id%253Dinfo%253Apmid%252F29220515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29220515&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.61" data-doi="10.1038/ng.3121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weisenfeld</span>,  <span class="cit-name-given-names">N. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Yin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Sharpe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Lau</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Hegarty</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comprehensive variation discovery in single human genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1350</span>–<span class="cit-lpage">1355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3121</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D1350%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3121%26rft_id%253Dinfo%253Apmid%252F25326702%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25326702&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.62" data-doi="10.1016/j.ibmb.2017.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wenger</span>,  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Cassone</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Legeai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Johnston</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bansal</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Whole genome sequence of the soybean aphid, <em>Aphis glycines</em>.</span> <abbr class="cit-jnl-abbrev">Insect Biochem. Mol. Biol.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ibmb.2017.01.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInsect%2BBiochem.%2BMol.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ibmb.2017.01.005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ibmb.2017.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.63" data-doi="10.1073/pnas.1522559113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yassin</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Debat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Bastide</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Gidaszewski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">David</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Recurrent specialization on a toxic fruit in an island <em>Drosophila</em> population.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">4771</span>–<span class="cit-lpage">4776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1522559113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1522559113%26rft_id%253Dinfo%253Apmid%252F27044093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzE3LzQ3NzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8xMC8zLzg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.64" data-doi="10.1038/s41598-016-0001-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Hill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Ma</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">DBG2OLC: Efficient assembly of large genomes using long erroneous reads of the third generation sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-016-0001-8%26rft_id%253Dinfo%253Apmid%252F27503664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-016-0001-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27503664&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.899.65" data-doi="10.1186/s12864-018-4567-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xiao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Xue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. C.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. Y.</span> <span class="cit-name-surname">Sun</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">P_RNA_scaffolder: A fast and accurate genome scaffolder using paired-end RNA-sequencing reads.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">175</span> (erratum: BMC Genomics 20: 524).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-4567-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-4567-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-4567-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/3/899.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/891" title="Chromosome-Level Assembly of Drosophila bifasciata Reveals Important Karyotypic Transition of the X Chromosome" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/907" title="Genome Sequence of the Euryhaline Javafish Medaka, Oryzias javanicus: A Small Aquarium Fish Model for Studies on Adaptation to Salinity" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/3"><span>Volume 10 Issue 3, March 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/3.cover-source.jpg?itok=8jjPWDA4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/899?rss=1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ODRijerqaliquTssDVNoWpvbL7yOBHQ6-Zal653eImY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/3/899" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/3/899" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/3/899" class="active" data-icon-position="" data-hide-link-title="0">Improved Genome Assembly and Annotation of the Soybean Aphid (Aphis glycines Matsumura)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495869" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2OTG5UKZ5IwcRrWQZPJzOjd_C_FLTs4qCUvjr_w14iA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/899.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/3/899" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-HCr3WczUxQxM1rh7Mmmv6WRIXwaMmT-8qINRl5zjCOc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/3/899" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/899.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495869" id="node495869--2" data-pisa="ggg;10/3/899" data-pisa-master="ggg;g3.119.400954" data-apath="/ggg/10/3/899.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improved Genome Assembly and Annotation of the Soybean Aphid (<em>Aphis glycines</em> Matsumura)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8637-3515" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas C. Mathers</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">899-906;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400954 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495869--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas C. Mathers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Genetics, John Innes Centre, Norwich Research Park, Norwich, Norfolk, NR4 7UH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BC.%2BMathers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mathers%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BC.%2BMathers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8637-3515" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas C. Mathers</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:thomas.mathers@jic.ac.uk" class="" data-icon-position="" data-hide-link-title="0">thomas.mathers@jic.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/3/899" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/3/899&amp;title=Improved%20Genome%20Assembly%20and%20Annotation%20of%20the%20Soybean%20Aphid%20%28Aphis%20glycines%20Matsumura%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/3/899&amp;title=Improved%20Genome%20Assembly%20and%20Annotation%20of%20the%20Soybean%20Aphid%20%28Aphis%20glycines%20Matsumura%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/3/899&amp;title=Improved%20Genome%20Assembly%20and%20Annotation%20of%20the%20Soybean%20Aphid%20%28Aphis%20glycines%20Matsumura%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/3/899&amp;text=Improved%20Genome%20Assembly%20and%20Annotation%20of%20the%20Soybean%20Aphid%20%28Aphis%20glycines%20Matsumura%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/3/899&amp;title=Improved%20Genome%20Assembly%20and%20Annotation%20of%20the%20Soybean%20Aphid%20%28Aphis%20glycines%20Matsumura%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/3/899&amp;t=Improved%20Genome%20Assembly%20and%20Annotation%20of%20the%20Soybean%20Aphid%20%28Aphis%20glycines%20Matsumura%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/3/899&amp;title=Improved%20Genome%20Assembly%20and%20Annotation%20of%20the%20Soybean%20Aphid%20%28Aphis%20glycines%20Matsumura%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/3/899&amp;title=Improved%20Genome%20Assembly%20and%20Annotation%20of%20the%20Soybean%20Aphid%20%28Aphis%20glycines%20Matsumura%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495869/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495869" id="node495869--3" data-pisa="ggg;10/3/899" data-pisa-master="ggg;g3.119.400954" data-apath="/ggg/10/3/899.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improved Genome Assembly and Annotation of the Soybean Aphid (<em>Aphis glycines</em> Matsumura)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8637-3515" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas C. Mathers</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">899-906;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400954 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495869--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas C. Mathers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Crop Genetics, John Innes Centre, Norwich Research Park, Norwich, Norfolk, NR4 7UH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BC.%2BMathers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mathers%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BC.%2BMathers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8637-3515" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas C. Mathers</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:thomas.mathers@jic.ac.uk" class="" data-icon-position="" data-hide-link-title="0">thomas.mathers@jic.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=899&amp;rft.epage=906&amp;rft.atitle=Improved%20Genome%20Assembly%20and%20Annotation%20of%20the%20Soybean%20Aphid%20%28Aphis%20glycines%20Matsumura%29&amp;rft.volume=10&amp;rft.issue=3&amp;rft.date=2020-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Mathers%2C+Thomas+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495869/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495869/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495869/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495869/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495869/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495869/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495869/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495869/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495869/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495869/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495869/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495869/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/3/899" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/3/899.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/3/899.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/3/899.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
