<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/10/3/1077" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/10/3/1077.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/10/3/1077.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/10/3/1077.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/10/3/1077.atom" />
<meta name="HW.pisa" content="ggg;10/3/1077" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in Arabidopsis thaliana" />
<meta name="DC.Identifier" content="10.1534/g3.119.401023" />
<meta name="DC.Date" content="2020-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2020 Wu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The mechanisms of sequence divergence in angiosperm mitochondrial genomes have long been enigmatic. In particular, it is difficult to reconcile the rapid divergence of intergenic regions that can make non-coding sequences almost unrecognizable even among close relatives with the unusually high levels of sequence conservation found in genic regions. It has been hypothesized that different mutation and repair mechanisms act on genic and intergenic sequences or alternatively that mutational input is relatively constant but that selection has strikingly different effects on these respective regions. To test these alternative possibilities, we analyzed mtDNA divergence within Arabidopsis thaliana , including variants from the 1001 Genomes Project and changes accrued in published mutation accumulation (MA) lines. We found that base-substitution frequencies are relatively similar for intergenic regions and synonymous sites in coding regions, whereas indel and nonsynonymous substitutions rates are greatly depressed in coding regions, supporting a conventional model in which mutation/repair mechanisms are consistent throughout the genome but differentially filtered by selection. Most types of sequence and structural changes were undetectable in 10-generation MA lines, but we found significant shifts in relative copy number across mtDNA regions for lines grown under stressed vs. benign conditions. We confirmed quantitative variation in copy number across the A. thaliana mitogenome using both whole-genome sequencing and droplet digital PCR, further undermining the classic but oversimplified model of a circular angiosperm mtDNA structure. Our results suggest that copy number variation is one of the most fluid features of angiosperm mitochondrial genomes." />
<meta name="DC.Contributor" content="Zhiqiang Wu" />
<meta name="DC.Contributor" content="Gus Waneka" />
<meta name="DC.Contributor" content="Daniel B. Sloan" />
<meta name="article:published_time" content="2020-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in Arabidopsis thaliana" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The mechanisms of sequence divergence in angiosperm mitochondrial genomes have long been enigmatic. In particular, it is difficult to reconcile the rapid divergence of intergenic regions that can make non-coding sequences almost unrecognizable even among close relatives with the unusually high levels of sequence conservation found in genic regions. It has been hypothesized that different mutation and repair mechanisms act on genic and intergenic sequences or alternatively that mutational input is relatively constant but that selection has strikingly different effects on these respective regions. To test these alternative possibilities, we analyzed mtDNA divergence within &lt;i&gt;Arabidopsis thaliana&lt;/i&gt;, including variants from the 1001 Genomes Project and changes accrued in published mutation accumulation (MA) lines. We found that base-substitution frequencies are relatively similar for intergenic regions and synonymous sites in coding regions, whereas indel and nonsynonymous substitutions rates are greatly depressed in coding regions, supporting a conventional model in which mutation/repair mechanisms are consistent throughout the genome but differentially filtered by selection. Most types of sequence and structural changes were undetectable in 10-generation MA lines, but we found significant shifts in relative copy number across mtDNA regions for lines grown under stressed &lt;i&gt;vs.&lt;/i&gt; benign conditions. We confirmed quantitative variation in copy number across the &lt;i&gt;A. thaliana&lt;/i&gt; mitogenome using both whole-genome sequencing and droplet digital PCR, further undermining the classic but oversimplified model of a circular angiosperm mtDNA structure. Our results suggest that copy number variation is one of the most fluid features of angiosperm mitochondrial genomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2020/03/01" />
<meta name="citation_mjid" content="ggg;10/3/1077" />
<meta name="citation_id" content="10/3/1077" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/10/3/1077" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/10/3/1077.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/10/3/1077.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/10/3/1077.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.401023" />
<meta name="citation_pmid" content="31964685" />
<meta name="citation_volume" content="10" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1077" />
<meta name="citation_lastpage" content="1086" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zhiqiang Wu" />
<meta name="citation_author_institution" content="Department of Biology, Colorado State University, Fort Collins, CO 80523 and" />
<meta name="citation_author_institution" content="Guangdong Laboratory of Lingnan Modern Agriculture, Genome Analysis Laboratory of the Ministry of Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518124, China" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4238-7317" />
<meta name="citation_author" content="Gus Waneka" />
<meta name="citation_author_institution" content="Department of Biology, Colorado State University, Fort Collins, CO 80523 and" />
<meta name="citation_author" content="Daniel B. Sloan" />
<meta name="citation_author_institution" content="Department of Biology, Colorado State University, Fort Collins, CO 80523 and" />
<meta name="citation_author_email" content="dbsloan@rams.colostate.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3618-0897" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. V. Abdelnoor;citation_author=R. Yule;citation_author=A. Elo;citation_author=A. C. Christensen;citation_author=G. Meyer-Gauen;citation_title=Substoichiometric shifting in the plant mitochondrial genome is influenced by a gene homologous to MutS.;citation_volume=100;citation_year=2003;citation_pmid=12730382;citation_doi=10.1073/pnas.1037651100" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. Aird;citation_author=M. G. Ross;citation_author=W. S. Chen;citation_author=M. Danielsson;citation_author=T. Fennell;citation_title=Analyzing and minimizing PCR amplification bias in Illumina sequencing libraries.;citation_pages=R18-R18;citation_volume=12;citation_year=2011;citation_issue=2;citation_pmid=21338519;citation_doi=10.1186/gb-2011-12-2-r18" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. O. Allen;citation_author=C. M. Fauron;citation_author=P. Minx;citation_author=L. Roark;citation_author=S. Oddiraju;citation_title=Comparisons among two fertile and three male-sterile mitochondrial genomes of maize.;citation_volume=177;citation_year=2007;citation_pmid=17660568;citation_doi=10.1534/genetics.107.073312" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=C. Alonso-Blanco;citation_author=J. Andrade;citation_author=C. Becker;citation_author=F. Bemm;citation_author=J. Bergelson;citation_title=1,135 genomes reveal the global pattern of polymorphism in Arabidopsis thaliana.;citation_pages=481-491;citation_volume=166;citation_year=2016;citation_pmid=27293186;citation_doi=10.1016/j.cell.2016.05.063" />
<meta name="citation_reference" content="citation_title=Plant Mitochondria;citation_author=M. P. Arrieta-Montiel;citation_author=S. A. Mackenzie;citation_title=Plant mitochondrial genomes and recombination;citation_pages=65-82;citation_volume=1;citation_year=2011;citation_doi=10.1007/978-0-387-89781-3_3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. P. Arrieta-Montiel;citation_author=V. Shedge;citation_author=J. Davila;citation_author=A. C. Christensen;citation_author=S. A. Mackenzie;citation_title=Diversity of the Arabidopsis mitochondrial genome occurs via nuclear-controlled recombination activity.;citation_volume=183;citation_year=2009;citation_pmid=19822729;citation_doi=10.1534/genetics.109.108514" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=S. Backert;citation_author=T. Borner;citation_title=Phage T4-like intermediates of DNA replication and recombination in the mitochondria of the higher plant Chenopodium album (L.).;citation_pages=304-314;citation_volume=37;citation_year=2000;citation_issue=5;citation_pmid=10853767;citation_doi=10.1007/s002940050532" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=A. J. Bendich;citation_title=Reaching for the ring: the study of mitochondrial genome structure.;citation_pages=279-290;citation_volume=24;citation_year=1993;citation_issue=4;citation_pmid=8252636;citation_doi=10.1007/BF00336777" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A. J. Bendich;citation_title=Structural analysis of mitochondrial DNA molecules from fungi and plants using moving pictures and pulsed-field gel electrophoresis.;citation_pages=564-588;citation_volume=255;citation_year=1996;citation_issue=4;citation_pmid=8568898;citation_doi=10.1006/jmbi.1996.0048" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.;citation_pages=289-300;citation_volume=57;citation_year=1995;citation_pmid=16632515;citation_doi=10.1093/biostatistics/kxj037" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=C. T. Brown;citation_author=M. R. Olm;citation_author=B. C. Thomas;citation_author=J. F. Banfield;citation_title=Measurement of bacterial replication rates in microbial communities.;citation_pages=1256-1263;citation_volume=34;citation_year=2016;citation_doi=10.1038/nbt.3704" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=W. M. Brown;citation_author=M. George;citation_author=A. C. Wilson;citation_title=Rapid evolution of animal mitochondrial DNA.;citation_pages=1967-1971;citation_volume=76;citation_year=1979;citation_issue=4;citation_pmid=109836;citation_doi=10.1073/pnas.76.4.1967" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. V. Chamary;citation_author=J. L. Parmley;citation_author=L. D. Hurst;citation_title=Hearing silence: non-neutral evolution at synonymous sites in mammals.;citation_pages=98-108;citation_volume=7;citation_year=2006;citation_issue=2;citation_pmid=16418745;citation_doi=10.1038/nrg1770" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. C. Christensen;citation_title=Plant mitochondrial genome evolution can be explained by DNA repair mechanisms.;citation_volume=5;citation_year=2013;citation_pmid=23645599;citation_doi=10.1093/gbe/evt069" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. C. Christensen;citation_title=Genes and junk in plant mitochondria-repair mechanisms and selection.;citation_volume=6;citation_year=2014;citation_pmid=24904012;citation_doi=10.1093/gbe/evu115" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. F. Conrad;citation_author=D. Pinto;citation_author=R. Redon;citation_author=L. Feuk;citation_author=O. Gokcumen;citation_title=Origins and functional impact of copy number variation in the human genome.;citation_pages=704-712;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=19812545;citation_doi=10.1038/nature08516" />
<meta name="citation_reference" content="citation_journal_title=Mitochondrion;citation_author=J. D. Cupp;citation_author=B. L. Nielsen;citation_title=DNA replication in plant mitochondria.;citation_pages=231-237;citation_volume=19;citation_year=2014;citation_pmid=24681310;citation_doi=10.1016/j.mito.2014.03.008" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. Darracq;citation_author=J. S. Varre;citation_author=L. Marechal-Drouard;citation_author=A. Courseaux;citation_author=V. Castric;citation_title=Structural and content diversity of mitochondrial genome in beet: a comparative genomic analysis.;citation_volume=3;citation_year=2011;citation_pmid=21602571;citation_doi=10.1093/gbe/evr042" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. Darracq;citation_author=J. S. Varre;citation_author=P. Touzet;citation_title=A scenario of mitochondrial genome evolution in maize based on rearrangement events.;citation_pages=233-233;citation_volume=11;citation_year=2010;citation_pmid=20380689;citation_doi=10.1186/1471-2164-11-233" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=J. I. Davila;citation_author=M. P. Arrieta-Montiel;citation_author=Y. Wamboldt;citation_author=J. Cao;citation_author=J. Hagmann;citation_title=Double-strand break repair processes drive evolution of the mitochondrial genome in Arabidopsis.;citation_pages=64-64;citation_volume=9;citation_year=2011;citation_pmid=21951689;citation_doi=10.1186/1741-7007-9-64" />
<meta name="citation_reference" content="citation_journal_title=Phytochem. Bull.;citation_author=J. J. Doyle;citation_author=J. L. Doyle;citation_title=A rapid DNA isolation procedure for small quantities of fresh leaf tissue.;citation_pages=11-15;citation_volume=19;citation_year=1987;citation_doi=10.2307/4119796" />
<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution (Online);citation_journal_abbrev=Molecular Phylogenetics and Evolution (Online);citation_author=G. Drouin;citation_author=H. Daoud;citation_author=J. Xia;citation_title=Relative rates of synonymous substitutions in the mitochondrial, chloroplast and nuclear genomes of seed plants.;citation_pages=827-831;citation_volume=49;citation_year=2008;citation_issue=3;citation_pmid=18838124;citation_doi=10.1016/j.ympev.2008.09.009" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Ellis;citation_title=Promiscuous DNA--chloroplast genes inside plant mitochondria.;citation_pages=678-679;citation_volume=299;citation_year=1982;citation_issue=5885;citation_pmid=7121600;citation_doi=10.1038/299678a0" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=V. V. Goremykin;citation_author=P. J. Lockhart;citation_author=R. Viola;citation_author=R. Velasco;citation_title=The mitochondrial genome of Malus domestica and the import-driven hypothesis of mitochondrial genome expansion in seed plants.;citation_pages=615-626;citation_volume=71;citation_year=2012;citation_issue=4;citation_pmid=22469001;citation_doi=10.1111/j.1365-313X.2012.05014.x" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=J. M. Gualberto;citation_author=K. J. Newton;citation_title=Plant mitochondrial genomes: dynamics and mechanisms of mutation.;citation_pages=225-252;citation_volume=68;citation_year=2017;citation_doi=10.1146/annurev-arplant-043015-112232" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=D. L. Halligan;citation_author=P. D. Keightley;citation_title=Spontaneous mutation accumulation studies in evolutionary genetics.;citation_pages=151-172;citation_volume=40;citation_year=2009;citation_doi=10.1146/annurev.ecolsys.39.110707.173437" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=P. C. Hanawalt;citation_author=G. Spivak;citation_title=Transcription-coupled DNA repair: two decades of progress and surprises.;citation_pages=958-970;citation_volume=9;citation_year=2008;citation_issue=12;citation_pmid=19023283;citation_doi=10.1038/nrm2549" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. Harris;citation_author=R. Nielsen;citation_title=Error-prone polymerase activity causes multinucleotide mutations in humans.;citation_volume=24;citation_year=2014;citation_pmid=25079859;citation_doi=10.1101/gr.170696.113" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. Hazkani-Covo;citation_author=R. M. Zeller;citation_author=W. Martin;citation_title=Molecular poltergeists: mitochondrial DNA copies (numts) in sequenced nuclear genomes.;citation_pages=e1000834-e1000834;citation_volume=6;citation_year=2010;citation_issue=2;citation_pmid=20168995;citation_doi=10.1371/journal.pgen.1000834" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Jiang;citation_author=A. Mithani;citation_author=E. J. Belfield;citation_author=R. Mott;citation_author=L. D. Hurst;citation_title=Environmentally responsive genome-wide accumulation of de novo Arabidopsis thaliana mutations and epimutations.;citation_volume=24;citation_year=2014;citation_pmid=25314969;citation_doi=10.1101/gr.177659.114" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Kozik;citation_author=B. Rowan;citation_author=D. Lavelle;citation_author=L. Berke;citation_author=M. E. Schranz;citation_title=The alternative reality of plant mitochondrial DNA: One ring does not rule them all.;citation_pages=e1008373;citation_volume=15;citation_year=2019;citation_doi=10.1371/journal.pgen.1008373" />
<meta name="citation_reference" content="citation_journal_abbrev=Mitochondrion;citation_author=T. Kubo;citation_author=K. J. Newton;citation_title=Angiosperm mitochondrial genomes and mutations.;citation_pages=5-14;citation_volume=8;citation_year=2008;citation_issue=1;citation_pmid=18065297;citation_doi=10.1016/j.mito.2007.10.006" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=R. A. Kumar;citation_author=D. J. Oldenburg;citation_author=A. J. Bendich;citation_title=Changes in DNA damage, molecular integrity, and copy number for plastid DNA and mitochondrial DNA during maize development.;citation_volume=65;citation_year=2014;citation_pmid=25261192;citation_doi=10.1093/jxb/eru359" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=D. M. Lonsdale;citation_author=T. Brears;citation_author=T. P. Hodge;citation_author=S. E. Melville;citation_author=W. H. Rottmann;citation_title=The plant mitochondrial genome: homologous recombination as a mechanism for generating heterogeneity.;citation_volume=319;citation_year=1988;citation_doi=10.1098/rstb.1988.0039" />
<meta name="citation_reference" content="Martin, M., 2011 Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal 17: 10–12. 10.14806/ej.17.1.200" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. P. Mower;citation_author=A. L. Case;citation_author=E. R. Floro;citation_author=J. H. Willis;citation_title=Evidence against equimolarity of large repeat arrangements and a predominant master circle structure of the mitochondrial genome from a monkeyflower (Mimulus guttatus) lineage with cryptic CMS.;citation_volume=4;citation_year=2012;citation_pmid=22534162;citation_doi=10.1093/gbe/evs042" />
<meta name="citation_reference" content="citation_title=Plant Genome Diversity;citation_author=J. P. Mower;citation_author=D. B. Sloan;citation_author=A. J. Alverson;citation_title=Plant mitochondrial diversity – the genomics revolution;citation_year=2012;citation_doi=10.1007/978-3-7091-1130-7_9" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Nielsen;citation_title=Molecular signatures of natural selection.;citation_pages=197-218;citation_volume=39;citation_year=2005;citation_pmid=16285858;citation_doi=10.1146/annurev.genet.39.073003.112420" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. D. Palmer;citation_author=C. R. Shields;citation_title=Tripartite structure of the Brassica campestris mitochondrial genome.;citation_pages=437-440;citation_volume=307;citation_year=1984;citation_doi=10.1038/307437a0" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=T. Preuten;citation_author=E. Cincu;citation_author=J. Fuchs;citation_author=R. Zoschke;citation_author=K. Liere;citation_title=Fewer genes than organelles: extremely low and variable gene copy numbers in mitochondria of somatic plant cells.;citation_pages=948-959;citation_volume=64;citation_year=2010;citation_issue=6;citation_pmid=21143676;citation_doi=10.1111/j.1365-313X.2010.04389.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. W. Rice;citation_author=A. J. Alverson;citation_author=A. O. Richardson;citation_author=G. J. Young;citation_author=M. V. Sanchez-Puerta;citation_title=Horizontal Transfer of Entire Genomes via Mitochondrial Fusion in the Angiosperm Amborella;citation_pages=1468-1473;citation_volume=342;citation_year=2013;citation_issue=6165;citation_pmid=24357311;citation_doi=10.1126/science.1246275" />
<meta name="citation_reference" content="citation_journal_title=Cancer Research;citation_journal_abbrev=Cancer Res.;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdóttir;citation_author=A. M. Wenger;citation_author=A. Zehir;citation_author=J. P. Mesirov;citation_title=Variant Review with the Integrative Genomics Viewer;citation_pages=e31-e34;citation_volume=77;citation_year=2017;citation_issue=21;citation_pmid=29092934;citation_doi=10.1158/0008-5472.CAN-17-0337" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=D. R. Schrider;citation_author=J. N. Hourmozdi;citation_author=M. W. Hahn;citation_title=Pervasive multinucleotide mutational events in eukaryotes.;citation_pages=1051-1054;citation_volume=21;citation_year=2011;citation_issue=12;citation_pmid=21636278;citation_doi=10.1016/j.cub.2011.05.013" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=V. Shedge;citation_author=M. Arrieta-Montiel;citation_author=A. C. Christensen;citation_author=S. A. Mackenzie;citation_title=Plant mitochondrial recombination surveillance requires unusual RecA and MutS homologs.;citation_volume=19;citation_year=2007;citation_pmid=17468263;citation_doi=10.1105/tpc.106.048355" />
<meta name="citation_reference" content="citation_journal_title=Hortic. Res.;citation_author=J. Shen;citation_author=Y. Zhang;citation_author=M. J. Havey;citation_author=W. Shou;citation_title=Copy numbers of mitochondrial genes change during melon leaf development and are lower than the numbers of mitochondria.;citation_pages=95;citation_volume=6;citation_year=2019;citation_doi=10.1038/s41438-019-0177-8" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=D. B. Sloan;citation_title=One ring to rule them all? Genome sequencing provides new insights into the ‘master circle’ model of plant mitochondrial DNA structure.;citation_pages=978-985;citation_volume=200;citation_year=2013;citation_pmid=24712049;citation_doi=10.1111/nph.12395" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=D. B. Sloan;citation_author=J. C. Havird;citation_author=J. Sharbrough;citation_title=The on-again, off-again relationship between mitochondrial genomes and species boundaries.;citation_pages=2212-2236;citation_volume=26;citation_year=2017;citation_doi=10.1111/mec.13959" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=D. B. Sloan;citation_author=K. Muller;citation_author=D. E. McCauley;citation_author=D. R. Taylor;citation_author=H. Storchova;citation_title=Intraspecific variation in mitochondrial genome sequence, structure, and gene content in Silene vulgaris, an angiosperm with pervasive cytoplasmic male sterility.;citation_pages=1228-1239;citation_volume=196;citation_year=2012;citation_issue=4;citation_pmid=23009072;citation_doi=10.1111/j.1469-8137.2012.04340.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. B. Sloan;citation_author=D. R. Taylor;citation_title=Testing for selection on synonymous sites in plant mitochondrial DNA: the role of codon bias and RNA editing.;citation_pages=479-491;citation_volume=70;citation_year=2010;citation_issue=5;citation_pmid=20424833;citation_doi=10.1007/s00239-010-9346-y" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=D. B. Sloan;citation_author=Z. Wu;citation_title=History of plastid DNA insertions reveals weak deletion and at mutation biases in angiosperm mitochondrial genomes.;citation_volume=6;citation_year=2014;citation_pmid=25416619;citation_doi=10.1093/gbe/evu253" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=D. B. Sloan;citation_author=Z. Wu;citation_author=J. Sharbrough;citation_title=Correction of persistent errors in Arabidopsis reference mitochondrial genomes.;citation_volume=30;citation_year=2018;citation_pmid=29519893;citation_doi=10.1105/tpc.18.00024" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=I. D. Small;citation_author=P. G. Isaac;citation_author=C. J. Leaver;citation_title=Stoichiometric differences in DNA molecules containing the atpA gene suggest mechanisms for the generation of mitochondrial genome diversity in maize.;citation_pages=865-869;citation_volume=6;citation_year=1987;citation_issue=4;citation_pmid=16453759;citation_doi=10.1002/j.1460-2075.1987.tb04832.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. E. Stajich;citation_author=D. Block;citation_author=K. Boulez;citation_author=S. E. Brenner;citation_author=S. A. Chervitz;citation_title=The Bioperl Toolkit: Perl Modules for the Life Sciences;citation_pages=1611-1618;citation_volume=12;citation_year=2002;citation_issue=10;citation_pmid=12368254;citation_doi=10.1101/gr.361602" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. M. Stupar;citation_author=J. W. Lilly;citation_author=C. D. Town;citation_author=Z. Cheng;citation_author=S. Kaul;citation_title=Complex mtDNA constitutes an approximate 620-kb insertion on Arabidopsis thaliana chromosome 2: implication of potential sequencing errors caused by large-unit repeats.;citation_volume=98;citation_year=2001;citation_pmid=11309509;citation_doi=10.1073/pnas.091110398" />
<meta name="citation_reference" content="citation_journal_title=Exp. Cell Res.;citation_author=E. L. van Dijk;citation_author=Y. Jaszczyszyn;citation_author=C. Thermes;citation_title=Library preparation methods for next-generation sequencing: tone down the bias.;citation_pages=12-20;citation_volume=322;citation_year=2014;citation_pmid=24440557;citation_doi=10.1016/j.yexcr.2014.01.008" />
<meta name="citation_reference" content="citation_journal_title=Nat. Ecol. Evol.;citation_author=A. Venkat;citation_author=M. W. Hahn;citation_author=J. W. Thornton;citation_title=Multinucleotide mutations cause false inferences of lineage-specific positive selection.;citation_volume=2;citation_year=2018;citation_doi=10.1038/s41559-018-0584-5" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. Wallet;citation_author=M. Le Ret;citation_author=M. Bergdoll;citation_author=M. Bichara;citation_author=A. Dietrich;citation_title=The RECG1 DNA translocase is a key factor in recombination surveillance, repair, and segregation of the mitochondrial DNA in Arabidopsis.;citation_volume=27;citation_year=2015;citation_pmid=26462909;citation_doi=10.1105/tpc.15.00680" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. M. Warren;citation_author=M. P. Simmons;citation_author=Z. Wu;citation_author=D. B. Sloan;citation_title=Linear plasmids and the rate of sequence evolution in plant mitochondrial genomes.;citation_volume=8;citation_year=2016;citation_pmid=26759362;citation_doi=10.1093/gbe/evw003" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=K. H. Wolfe;citation_author=W. H. Li;citation_author=P. M. Sharp;citation_title=Rates of nucleotide substitution vary greatly among plant mitochondrial, chloroplast, and nuclear DNAs.;citation_pages=9054-9058;citation_volume=84;citation_year=1987;citation_issue=24;citation_pmid=3480529;citation_doi=10.1073/pnas.84.24.9054" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Evol.;citation_author=E. L. Wynn;citation_author=A. C. Christensen;citation_title=Are synonymous substitutions in flowering plant mitochondria neutral?;citation_pages=131-135;citation_volume=81;citation_year=2015;citation_doi=10.1007/s00239-015-9704-x" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv 427500v3;citation_author=E. Wynn;citation_author=E. Purfeerst;citation_author=A. Christensen;citation_title=Mitochondrial DNA repair in an Arabidopsis thaliana uracil N-glycosylase mutant.;citation_year=2019;citation_doi=10.1101/427500" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=Z. Yang;citation_author=A. D. Yoder;citation_title=Estimation of the transition/transversion rate bias and species sampling.;citation_pages=274-283;citation_volume=48;citation_year=1999;citation_issue=3;citation_pmid=10093216;citation_doi=10.1007/PL00006470" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Zhu;citation_author=W. Guo;citation_author=K. Jain;citation_author=J. P. Mower;citation_title=Unprecedented heterogeneity in the synonymous substitution rate within a plant genome.;citation_volume=31;citation_year=2014;citation_pmid=24557444;citation_doi=10.1093/molbev/msu079" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in Arabidopsis thaliana" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
<meta name="twitter:description" content="The mechanisms of sequence divergence in angiosperm mitochondrial genomes have long been enigmatic. In particular, it is difficult to reconcile the rapid divergence of intergenic regions that can make non-coding sequences almost unrecognizable even among close relatives with the unusually high levels of sequence conservation found in genic regions. It has been hypothesized that different mutation and repair mechanisms act on genic and intergenic sequences or alternatively that mutational input is relatively constant but that selection has strikingly different effects on these respective regions. To test these alternative possibilities, we analyzed mtDNA divergence within Arabidopsis thaliana , including variants from the 1001 Genomes Project and changes accrued in published mutation accumulation (MA) lines. We found that base-substitution frequencies are relatively similar for intergenic regions and synonymous sites in coding regions, whereas indel and nonsynonymous substitutions rates are greatly depressed in coding regions, supporting a conventional model in which mutation/repair mechanisms are consistent throughout the genome but differentially filtered by selection. Most types of sequence and structural changes were undetectable in 10-generation MA lines, but we found significant shifts in relative copy number across mtDNA regions for lines grown under stressed vs. benign conditions. We confirmed quantitative variation in copy number across the A. thaliana mitogenome using both whole-genome sequencing and droplet digital PCR, further undermining the classic but oversimplified model of a circular angiosperm mtDNA structure. Our results suggest that copy number variation is one of the most fluid features of angiosperm mitochondrial genomes." />
<meta name="og-title" property="og:title" content="The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in Arabidopsis thaliana" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/10/3/1077" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The mechanisms of sequence divergence in angiosperm mitochondrial genomes have long been enigmatic. In particular, it is difficult to reconcile the rapid divergence of intergenic regions that can make non-coding sequences almost unrecognizable even among close relatives with the unusually high levels of sequence conservation found in genic regions. It has been hypothesized that different mutation and repair mechanisms act on genic and intergenic sequences or alternatively that mutational input is relatively constant but that selection has strikingly different effects on these respective regions. To test these alternative possibilities, we analyzed mtDNA divergence within Arabidopsis thaliana , including variants from the 1001 Genomes Project and changes accrued in published mutation accumulation (MA) lines. We found that base-substitution frequencies are relatively similar for intergenic regions and synonymous sites in coding regions, whereas indel and nonsynonymous substitutions rates are greatly depressed in coding regions, supporting a conventional model in which mutation/repair mechanisms are consistent throughout the genome but differentially filtered by selection. Most types of sequence and structural changes were undetectable in 10-generation MA lines, but we found significant shifts in relative copy number across mtDNA regions for lines grown under stressed vs. benign conditions. We confirmed quantitative variation in copy number across the A. thaliana mitogenome using both whole-genome sequencing and droplet digital PCR, further undermining the classic but oversimplified model of a circular angiosperm mtDNA structure. Our results suggest that copy number variation is one of the most fluid features of angiosperm mitochondrial genomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/10/3.cover-source.jpg" />
    <title>The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in Arabidopsis thaliana | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"NCFoX-8IJdPu8RYwA5_qEQI2kfsjJOf5OybvL1J3wwI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"495846","apath":"\/ggg\/10\/3\/1077.atom","pisa":"ggg;10\/3\/1077","ac":{"\/ggg\/10\/3\/1077.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/10\/3\/1077.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;10\/3\/1077"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;10\/3\/1077"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/10\/3\/1077?rss=1":true,"\/content\/10\/3\/1077":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PFHw6aNnCiF\\\\\\\/Lbr32YEB\\\\\\\/s1VLlw5zAIzNZiORKWC2rD7YA8WuH2\\\\\\\/A6yj0hce1jD6HtAxwb+z5GTKGauevFgUJ+JNaUGxQY0vGnRVSsWtloGI6Z9jEzurTy91tv6maKykPNXfeOAZRrUkdFicOeRW5yAsAuSK526vtZVUMF2ffBu2hWaorpk1jb9xu3jES4nBS14YtW3AqJJZddSIFvWOgoEEyXz59YFDF45g5W15XTMiv8o10Uuq2w9n9JLzb\\\\\\\/+BCuaJ1HsRy7cG6eCsl0bPEuvJoQ6Zn+7\\\\\\\/BWvQ0k7jxyL4KGAuYnqoxGuU0QnSoq9Y34Gthw8+G6HznzoZ+ilwFGryNv9rV4VFAunLSH+9LUFXHRFZ22Bu+1iSRoHTBvGBZBVWvqZ3CXZrR\\\\\\\/44rg\\\\\\\/Yv+NKT1wjhjO2AjoBbVg6nnENcUQSpdsnCUPR058CmiimIYApgF0eGuLXkD5k7OLMkbciTI5202VBUkNrvnEah6MmPe+SXlKVet2kRr9JdmHgJA2TqvpJkCnyOy6PeJjfgT8+euHZTvnt1mW9QtKKKM8mEby0ArTG3Ce\\\\\\\/zOoNYpCuxVkuF7uyjsR5s6328BzSSV8SsAa1rfEzbI4qqaIQM72OP9fHOIJgnZf0uXF+smQu3H004Obp6g3wCVI\\\\\\\/l63QhMuizkyLDJ3tXAUmrVe1xzPdJjYUli9cXO8cL+cQXHp\\\\\\\/Ghgiuh8qOrDIsk73VrJD1HaIRi5e9go4NqjC85GeBoO2\\\\\\\/2+gKUYDZDA3T7ECitwoAukDdh\\\\\\\/FSLFNgEWDS6rZZZ1dJ6B4K2pTFTIBni8OqTpWytKHh3c50zv5lYS+pLplJwvlcXBqU8sKhsiFbt58\\\\\\\/wo2nrqYzYLKHe1wvHqEBUilrR0+O9uExCTFzCgotG40kosTbj\\\\\\\/4dEN+fU8AunKUvkMGMw8khymoHBbXVxn\\\\\\\/H8ZTeq\\\\\\\/ffkFXNvAN56LRdfGv2rvo56i8vKcj1Lb8rnIYeZvGOJ6\\\\\\\/KtLKMCipHfJ8WnX3uJ9XNXHChPt4TdBbvTRpJZoiVspP1TP6NxeeLMvy1Fp9Jldy5MyKqBOykPoI1oATUmuO4z\\\\\\\/5pAQGbti4Yph39BcIwF\\\\\\\/AmwPrGVdst0hPsS7Os2gpfXajFNanGFcyA4CrI7clPL41Ab5S1b3D4MA7a\\\\\\\/wnk9oTr9JlXDXPx5nIxGRMPIa1p+IsqloS5CXMk\\\\\\\/K9SCgh5wxcbbTZ0qtePt8Yl+S6kjTrzOE78rrNQQbVmNFUGPRrJ0SvEDh2cCDFOcw2lKS3ZpNwkDHyF4VZapPO\\\u0022,\\\u0022iv\\\u0022:\\\u0022avz13tyohhnN0Sx4KKOSTw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229d1ac4d1e3ac1a6fc4dea3a53e47b0e6a4957bc8ffb3ea74591f0883f1123b55\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IooPkbMixPiCPpORRm4wIVnfSsb1197ECcXArIcPKOpger3lCI9h3eB7CN8sEfapsKpiyX5I1lkwrjC+fu3fpRHYP\\\\\\\/9NE5jGIq2GWDBBrhKZk+as+i1ldBOKhWcTroEWStmg\\\\\\\/pNoxDjU8L2+zQDNQMbCOQPJ2sUXUsGAgHanCCi+0X4MtAFaQ78lPJiY3NikQKm1UIbIqqi8SFd263gUZ8otiQUw6VhHFLqSbx7cYs9S6MtzbeCIVcjqGId61puBENqe9i4NPhUTAQqDaJg5AOyUCi9JToMgADLwqRdgIloShJPnfA3mGm8c\\\\\\\/nHn2dDy6DCRBNNw8ZR0deE5nkpeXQsfaq\\\\\\\/z3ORMfvLQxmBlGv1jL3ADSDgl4QAZ4Xcg89Lpr3YrdIHqP0YVdkq4ruX922clRZXc2appmDT5XVs440eaBC88WD9Q3Jm2hXoA2v9eWeQKkaG\\\\\\\/bkx792AMAOgjttv9cqUsfK3mefk1gIjhbEOEwVJsTqnASmrswEuXxEqHSzBOFr\\\\\\\/l6JnvKBCVJYymB7CKE1YL9F1PUwpQeAMYhkrS6IPIEJa9ckecdiFK8mkjqWR4X97WtVjTknDUDjR+0Har8Bbh4XGvZCbeIYr5wxuj3HD3cET6ZTNcfjzwSK3dYJYlzqYOA\\\\\\\/enW\\\\\\\/vNuU\\\\\\\/G3mIFMq\\\\\\\/\\\\\\\/tv\\\\\\\/lXSwxbU8dPAz8IORvQ+PeRxLay9EWLXa5q62uPAdRRHYc3aT8aMBXMauuL475qnhcYjH5tydGKtyfNBUHgDDFFYwamUNycIPp9cje0LZmMvGryaMQS6Z6KJBCWBNK49rS8n9cE6U6d4HvG14AIk9bE+ep6GhdYswKyO5Q4uU+92U9cI++0iX9MSCf9azHugCb2mryejnbvTd6+YV0aLcl9yXMkh3uDfo1I0Xq\\\\\\\/lvtmWtBLcxJMfxLLGWOVkU9RdUJzzDGBGhuILwl9Z0lyjNqyJrXTMNPpwg75UUGGvET23GjzcL+N6GTfcXlcxFDzUnSGNLloUELVTwJZeRL1fi5pDZU+NB6KIe3cPGmEp3hXzpkRwFOCGLz7MBLqTRsMDnlXCrEGkP+E5P5cbKkN9yzA805FSwguvBbqOP19WNwfrLqPYV4yUll\\\\\\\/eARU2osDG\\\\\\\/yGKyr17smeijPMKyurNjT8M6psd7li+MeDIq8IRfnYdHB3lbxD5WbYXfBmJSALkhQt5hn52+xyRx6bEkifv4PpihBnPmqTJ\\\\\\\/m\\\\\\\/DMVsIqxdU16NwxQCrhKX\\\\\\\/l8kaJ789s24\\\\\\\/F92dUBbNT0+RtQBrERST7O2mczH7wN63H3ofBrrAQAZrjKrMYnxgazpdWdhNlMhg3uQfQ5Ln1mNSidyfeKr9kTusIWZ\\\\\\\/OOBX1NFbij3iVGvlXQKCjqe67k0goxOQkOO74MfIxlF8jnXswZUuCw\\\\\\\/p46eObv7xU\\\\\\\/rScjzi\\\\\\\/WZmDyZrI\\\\\\\/4mr95B1btqFXzElHBh5inAgojbkXWLDnxUr1MiTLDyFVePFE0VmrLJNgoD\\\\\\\/Gul830fyS9atdhVL+lhHGhzOhs+2FabXXXB+wViu+xrYs7pv4zlw8VeDhT\\\\\\\/HO8RBjrR5PbKmhooD4xgL5mYunI7S1ggFBYfbfBqz0kGNSkHkuwk0eqTDn5B03UtTz13zewDVqeAPgeAvSJFqJMVnPhZcqrxXifMLnQ6FBJo3xiQ6wQGfiyXHSvDNM+Mi+ZV+oPgwvdeIuPFbhZHKgKd437oztdNcHmDC+F8zLYaMX4T7go4sbNlOLSVA\\\\\\\/CIkLYCge1Vq8yEUbn2pM2eNH38BuEKj6ERJYLf2GFvmlcBTB\\\\\\\/yR9ai2jnhBOgrgU2f5aeChLDqyjrSIN+9Kok+uJZvSEVjs+RjA3iJBL1k0J0mXlERG83m4nbg3FRo+oVQ9f9Vu2p8PcHo3Q2gMXdObKzcNhU\\\\\\\/DKFpC6eR8E\\\\\\\/5ti6wpBnHRpuAXtAeZvVyDuWAx3fGt8rnRrXOQnGDkreQUObamTCO1deOBcC7oF4lmto+IDRs4SJWFryHktiY9NkJlfMjvuGG3LAbl\\\\\\\/bv5deNic6H0xPAQ2jegf0vGpKn4nrEhRNYniU7J2MES3MCkYS3ijXWh+sh70gTIVAdwzK8EYo9R+iJEsKM9O0LHlAT2uXflGyz9+DLboNz47Pd\\\\\\\/+Ah8MP\\\\\\\/Ursx7U4V8zE7jExP9bFFUU6i\\\\\\\/3n1LEc1OUWqffTxUbTrI\\\\\\\/8H6Kufx\\\\\\\/y0HXxjvZ8FG0KopbrvIasQljxPzo5PVISLymNAPiBmL0Toylsl+HN9CDsh\\\\\\\/liXCBH\\\\\\\/Q8f4FG\\\\\\\/+xB71qRG4+Konm22OZ19lbGXEna2M3f3aQpZPPT4s214b+Iggl+BEa9Thewjvydr+7HMmAOTF\\\\\\\/hOAOpV81ih54Pi2wPhSBdGobA8IrdESSR1pSGOWzwDHDHEbx\\\\\\\/g4vkB9O31xBgMmAy\\\\\\\/+NdjZbKeYIDRNl1j3zX4srN3Ctkth1O4OkdEj+SWLCFLzD5ImvMt20+SOpv6WSNQkVHOdCDCNy+VSAiIvAmZMKCOJbq+q9zkqi0imCaH5UbAhr29ttKV2HTjZJeZVRPMBjyguWhFofS9Pw9iVgpnbMKRZ7Nh6lFtnR3IWO2Ba9rFZN1ThhJGqQhbq6aaFY0TZpcP9S67aL8amUGN3RCd8BHXYOJCQZMeo50zuY8ZPaSyiowfh3dK4Q7jMbDyq\\\\\\\/3oEv0UvgC2vdSjnHwKq3QksnmKX5uz7bWQ4krFGV5JvjnqDXx+1X4YyxGUpMPE85X6lWkZXlq5O99TZNyqXM9piwODIjjWcfL68ABhi8dsAXMNjllhtsaGlG21\\\\\\\/SnjY4Paz7UtXaWYDsPU5FudlSnOX7Pu2rPT3pmGC90m0kPHgUoMI\\\\\\\/6+Trt88cz9pxDUwHenSsvfHuONyYz4UjXRPucSq\\\\\\\/A+HQGinJxK7TWJDoWQM2dcYAh8xBlCrQ+PZc7u81FkhUtkKAIbwJtC2H2euBZ22vJi4+I7duG91MDFUelf5rXENXrqjSB16l5+IukAVC0KPJJv+bcoSVBm6r5owJZNJwgG5yMt3bQIFNrZ6K5DLXPgEe4wrA2o+FOl7qWDeKxeXHFlJdbb6i0kbroa0XRXG1uK\\\\\\\/TkNF2QnoF0wDulw0kY2XiGqb2PqLQ3pJ6B\\\\\\\/A0hh4+5Ictz6Jp76TPNuoZNQnDrGsLvvq7DAGV\\\\\\\/e6jhYxAJFv\\\\\\\/FMq6bduTWaq89Gw2a39gP+3yYk7BZY2nEjMwEEJohb23DXw5EkAS7+11nd750xL0VthY9Ff3yVDyMP7km2z023gFB+wKiuXhxe4yTQ8QvZDwPNhVpWUQgxApdeV0VLsO+HtW4XnpAwxUTlbs\\\\\\\/5ANx2P1lPeSmXGWgaCPri+apcdhAF7jK1EP8lUZ8MwCqJkdMvRPmNF\\\\\\\/eEyQzQ8JTl+gedQrra\\\\\\\/CyjK7c\\\\\\\/LA0U5Lv0jzZUfApC2gWVvIGJE8DB1WdJG6+cY7xdL85a8hDFpfxuGxRxQEaKnilnxyXOZXmYpQKUU484iOdcdDyBUgD4VNcPbI08n86Ww94FKIn8Es9hlyepyJS0BZqXjRBj6U2iu9aIjDKlbuHKQ3GVdeuAotVIX3QLwkWlim08ieXj4lvxxcBVjy6PbdXSbS0ICKX7LR1DQkCN+nrLxNuyC8qZCVHWlZAI65LomKP5gCJyZJCj9AN+Ca6pDS92KVlUWzencWztzw4LuyuINkcbnflh1sOeBSMG63T5rjXFL8k1YGE0eNd7VqBjS9fxHYfcrcJBGwj5PJerYHT7YHZPpamNfsXARZkIjsdpLk1ZGDUATBV1fyZjQjIjs07BS6OyKB4td\\\\\\\/BiqyPK5BrT6pcqEBzbsmbDg9PnJC8NMRSZJobfgZO4zxjDCeRgRvDhop9RPrRKojPSVIo4mvGaJflIv3SIPTclX+RzGs39Bwz6eToFjKE3ME9CsYBZkg6+iv59wRigBWIWKd0125aQacNJ3\\\\\\\/fP721YP0+SnjvAj96bCcxPTO2BIBBRjk082bYhb0IJ+RlzansjiTyVG3zVU0QAHUBckJFi7nRQgRoCBUpqAVC1OguKpHEu5GhmLSUvVwvvujq2NFpxcpEq4M1pUc57ezMgcE\\\\\\\/QyjtnIMOsMA0Rvg2RmgMtNdsCJKnvIzH3RRwkZrHvj1HUHnnfSA2c+WuhDlT3WmPGMF3th\\\\\\\/p17rCNWR7VyxGmMYh3qiMdsWj6dpzunPCgSDbabABnWxksZqmTw8jVa0NhUuzGZeYfd1p+ETk5du2mm0r1dTfDeIQm\\\\\\\/DMJ\\\\\\\/psVlZvJKhzanGPOlsrJAvh8nklhGS8OeOsY3NEccSU1VV3MnUVvY7R8giTgz+i8K\\\\\\\/0dWyA901o6AoLEF1GVBHUpr5gquvngg2fD9UyPObBOUYBibsteQMxGEpZVuc8UOh38BQZzYqj8yL4LGKf3opxOmthnbmBd+vp5lA+phmJ19G5ns32vGTTRZ9N0PirhD6g6lFNwHYVMgNkSsFjdEm+LRtuCcV85uqlqg9VsmLjjJ3BjS4K31LTpmQCDAvStbQS8xO\\\\\\\/C8DHV17NziIPTrBpTjAtyghdVZ4gWJkuAnJMq5AylGhqllp3UBCWU2C2LV5KYatvLFl4Lnu5atoOy+cJc7LrVX+U4qAQ2RKPaOfc5fNCkYVq4AXtOW+U5rABEV1mnwurcMCCnLfaNEe62+mWUyjnsYEgI2MgDlbAbVAglfexAel4eQrlSXeh\\\\\\\/b4rznlsAK66KnNchFIgPMb6\\\\\\\/EXjfSofNMZyXmZVXV+rQ7Tof+fLoyhO5eahcA9yg3zp6DmTGgynPViRsquGDOKbeXadsk4fTI1W98TPFkqz\\\\\\\/IRXE9sGoo4CwHBlsIG7+5KO2qMFlTqz4YR9b8K3IA6jeC0K6pimPWdJrfs2xB\\\\\\\/5Bd4NYWDHw2dT3snPBew4p3L7hcq8VHDRKbwW1Rmr2OAAsFrlJt+daHirVf1Ku8trIwJ\\\\\\\/LZsGX5tVY\\\\\\\/grH9uwuaVD8eA8gw3n02oSoR0n6kx3\\\\\\\/QIaRWE1DGcjbT7PuPRhqlFvnMb5H2ZS7JlRaxkwByZsCIAuKM25B\\\\\\\/fNGX2bWkNaF8+ifQkoaiGGrbPHHZvjwu7T2oT0+09oCJXwYGhwFJuCQPNU2RObmKe2QkF\\\\\\\/F1he+kJrA+IZVI6j+9dTp0krXfUAWdZXoCQ3xxvCmjMPuTrEvamogiW2BaJJ2ievpz9JE8gTa6wARsFXMWG4pK4mOlerIGVtIprZLYdzcXV3JRzD2l\\\\\\\/X2ovL7mXUfgdqDo2kQrBGQ\\\\\\\/Qmo\\\\\\\/3mIlpFM+gCHO\\\\\\\/kPRFZMRCnzsEvemgqI7FNb8s4OuqresIJ8y5mTpa3KlVH7QRLIrwuTLX0UJcGHNYQUM7CTn22RHashZHQJ+NTNKEUl4KZwTaf7Y2y6wEs+z7rJWezxP3nOCICY0B6fjXAm\\\\\\\/JJCKdXoL5CzYZt4BogdoHW+575S+ynSB8JKbUvzAQDvzYb2G+l7k1I\\\\\\\/Msq9ktHqCWJbYdCR2JJibOLCrngPMILa3wYSc9kiIbK3PumjU3Yc2Xmgg6Wf0lNchooQ8VaBs8xrClaoF+n6YjAqbHXvJ2cZe2vmRrOVHY2YBVPZHcTjKkpRwg4mTuqBBJu5I91oHWihZ3RcZuHFdRDQRTh1QfK6Xu2qBJdPxIHRogBrgKcMwYnoXUoV0YsgxYr9axJa8viCA3QZEMm55MO93Wdz4IoKMzkw\\\\\\\/4bjPP3IiF0h41kAKx9xmY4ojWCVZcU0YH0TCOZfjtjaBGVaXR93nu2oe6kfQX4+4smFX52k4yR8bGf7X5kBZBPq7bEvvR8wGqzBSHxFiRRKMFAO8axXT4mHXxlJ6NFTVTfKIDOu2wb5wzdUWJA9+aVlnw9qiGENMnM0vpg8YicDlu9GBZ06DUNZk9AslJlJ5W4p+LyQGrkjRMwjMDdYPNC9WMmkLPU+bjAxE1YL0OQnW1t4za1JyUwJY0\\\\\\\/cV7CrRvE8Z1mc+3rvJASpXG13h7YnhQOQOC668bdBNEZjdlvBkL48suKRT75ddMKxsC1YSyRtMgjGd\\\\\\\/Bd03HnmS9CdkZNCIB6QyDu0STL3kOZOvPFeQYTYnPxNPa\\\\\\\/mg2wzBBkbvYRG5+jzgJJKWMYJaFo97SnWOfKXDTFy5N5ttfju5LmvJ9cxJfVDL2fhOKb1hIdAt6jNfFRsxjlJkUxS1X0gicfSiM39Q5W7jBAPK2289wElLQOq8NbZrbe7CnGy02v8WmR9uMXLqCoZ8+oVktip7OgWUQ6OM3evxd8Ry4CLIgq08jDGkIv4IZWL7F5VbUPvr9AjrbQjGHLNsiSnoXQz\\\\\\\/EgpsCAjYF8MysJqMWEOOS7o6Nx9YQJUybuUGuFOQG6mxaAyyu11G6SY5SguEFDpdNGTsioU2d8VCo26SLohXrPPnfsS1DFX1RuBYR\\\\\\\/LJRzyXdAhvz53CI3BWiY\\\\\\\/huXa2mnTkEtgr+9jSIUxKvunxFud4uyBaj+DReFDWIA+Vo\\\\\\\/plA18TKgenKotLM86Y8zJsv\\\\\\\/2mkua9Oce+U0oE\\\\\\\/y8a+WJ1Ij2ZBlLJl\\\\\\\/oCI\\\\\\\/8EgPinYivY6n8IeEQW9YRdGAgVmh\\\\\\\/HasjupbIN1Q8SI7uiyomYwdCfpCawbD2kxmt19Bmq+1UG3P5g7hWNU\\\\\\\/X+TCFustFkWg\\\\\\\/pukP28NL3keyaJE24bLwYtWbwfK+7U0t2yoOdL44rlpC0TzW9LeIo3OGn28K\\\\\\\/d\\\\\\\/qKZ3dQl6hKHZun1MWmJYi8m0UsAgStryQsJIzc\\\\\\\/bwvw8q941jJR79nPajxmIRqZG2opMoJk6OXoL78Lh\\\\\\\/ZN1TF4BPqAjV2w\\\\\\\/oHE9db3OZbzxresYlrFkaz1Jd7KW11Wtp83C34SkyseDpzBgfZgK66UgfSjUDdy+YDRkDZQnIDaeGKO4lfJkzHAV2eT5gTOYFE5jp5QgGGmWznB55xcIU8aG3fMVKabwfejKR6nR3SigQ7lFHYFMsmRghSqYEEUwZ\\\\\\\/XKfbTfTnAH97qUVVkN4Y3ClgU7VYdeKYI7LKL6sVxZ1cQpJRjRAIHn3QUNSSVYD1v3krnQd4++4jIirh35CTMynzy+bQHQaDfGtY+LDcXFrWnVRzYJ730R0VMcqMk41v6JfeZ6ehPctsahJ9tfW8ZxxD8mhqk0t9xg5MTaXz2isbR1MRYZHjTyNzpUDLhiXwoVJLDvU\\\\\\\/msVe8uiQiTlFFsoxSuh79FSm0vwYcYeUqarEyC85nebyMx60fKNek27yunbRVfyrx9jxJ96lWIZyC1F8qVguzaJn6NrPu8iUwTRwLprTJ2JUI1PVmGGwUKnV2LO16RiREX3oMNqSTXWhEmXzvy\\\\\\\/sAqxA7gv2ZOHR3gSUZD4vl46hoEfD3cS3W\\\\\\\/5FYmNIm2FX2L4ppblhDP8Ppb\\\\\\\/+P7e5vkp25mcUvIQ6WIlzUlFsgJPl25c5Tzu00e8Y7cGoNWCksfv7aVryZMpb8hqSzlqq3GqtOyAy4Md3LQ86bsvH6nFzGDveT4rTypkRDxK3Quzs07pQHH8dzf4yXopu0whVjmvuDjF71mIXcBlg5HYCmB8Tdi7v3Zq6QtxZ5n49krH9W2EdpU+aA3uATkwEkrDqD5g5r4i2YkP1gN7rAYffBZnRKn7FVlE+Mly+tRleUShrQMhJkx52sFMc2vIW4QSHTCa6ap\\\\\\\/GkXAHg29QqmPkFVrFxiS27pkOWXigzyIWUWCf5\\\\\\\/GjFAX94aHxFbYNQ1aSMRPBA00xu04ETe74Ge8vXyHlnfH9xhyiDCGoHXR71J6+6e5TERqNwjM9QyPZMJeBg8EiBZBC8VNnRgLmb\\\\\\\/hG6xMIDTorNvLdqK+dfQS9vzI4hQR\\\\\\\/7Na5l0cwzlPU1Ty8K6jpQK9S7VoVeDKnwwXwQIkLFFwrHaU\\\\\\\/gNkEJTiiZi1MmQ5NgEV0oB\\\\\\\/wSMwem130YPECN4vAgc1kcmFKgBSnXyc1ropI4KbHziLkNOCdaF8jtxiuIpaTKdWLubhwRUAJ+KpM1RZIKx48tCNEA5aTSNr5LzxF6HvAX4IMTyv9xanl0Y5yYY8d2Q341cJ7F2euW5CYxWlI+YQChcf9XK7tjYvKofhYLbIblwp60iOPvSYxIFXefgoemiYChjLMO3fm1kmSVSpKx\\\\\\\/sQ4drxRSvrdgs4olkA8eLDi4sotmKq1cT3xTC8EA5I65cuw\\\\\\\/F5vYlQvaUMGXdEuqBzDtakgHuUtKhuDHHW60tB4c8A9vB9hKIySw\\\\\\\/BEBWiFWyd2oGW5hTcRzomcsx5Q4JnD57+NGRNQ5v3MjfVd3IpTR8RY1ZoJrSq4MqykMExXk1jxB+4hgIp7LeUHYNTyHlGbGPb3gsbjLxLW3ycA08S\\\\\\\/e7sSZnmidJiNtPdUITs+LN5199jMEIirWoZTIhAQaYagbzrZzsNDWm0PH7G++LoObTvBIAR+eorbZEg8PWmCAya9T7XmYknAP4CQH6lskqQROfCoeXGUHlrQ33sIbtUalmJJKYH1mNm1VxkAT\\\\\\\/lonsIm7Bn\\\\\\\/7F7TxzUP07WzgYxDo0423kVntA1UvdrNif0qUQNNamsXdTEV2L5CJjuAbEGHXmjiZl+18zER3rcQBLTi8XQCYEgBmYhlseT\\\\\\\/owaxymfyY4lFCmRaxoc167BNtGygdQHwccu4xV16kd1HsJ0SnnXGOpWVtyFDGuHOY9u6MR6qpTIMZoz0CaZ1dmRCRxof5Gicx2Oad8ukdb\\\\\\\/bP3ZkG\\\\\\\/EGikdvA2yNw71HOeepZUwlca39qwwH1PUjQBEet3aaJmv\\\\\\\/lIg7lXCDx41JY7HcMYCDbDGy8hGThJmFPYpZ1uTEj+BWy4mcQguWuGyHANMGLl864cwDTWq87GNHHoIYVPbnd4HS+7SHEyA6ZVMGXGJ1wh7QlY7Nsdc8+LZWLlqNNMJ3J0BZnm5PvIClids2PYBh4y7Lq8Eikn\\\\\\\/NY0VH13QGIztzP4Q7itctiIw05yO5WqHMIZvb1iSwawlUbB85SGH5JYAQx5Frxqq++tn5tkYaZX\\\\\\\/KJjdi3loCaDko037dVG39GTnCtC\\\\\\\/STF+i2nYHdxLYUqIpG6fuOX2M+KYsDAJM6b5qi+hmBVOGctMl+vAklXJgiThX8YmZM5GtN+iBY+GkwHXaRHOxFXhc9oyHn7SjkkU\\\\\\\/++HrFqsV2sSF9zgplYYZtesHrtoj6uBc\\\\\\\/If7a0ekaJpiGzVx7ImjhyoIv2a2mjnbdFJYWFbKA\\\\\\\/ytL95SFU\\\\\\\/LskjlUIYVjGLnbuZ909kLbP+1o2xJJzye3rI\\\\\\\/RbTphTJEFTVM+oojYnzyYWqlKNCoKIk8FpR3tlXNm21b\\\\\\\/2zImdB5svJTZXB3URQ8yc56nvo635gGwME42kayM2CeRQgTLZbrviViiy7KAGvB8jGyLqSELFCO4DD3v7OzaNKY+HYrVvPzJKyeZ+oKp92ElvGAQ76pT2QUdkJGz4gslB3O1pIA+EjpMyspLdv4mcPZNdHSAjZ5+mPvm7iyjdiZhm0TTMnP5MBxvADlN1FSTXCKB6AzdPye7kHMZif9KdH2sTWlSXG\\\\\\\/dQ\\\\\\\/aO7YGe1kMRn7Jjj+oYCKHMayPqcKNVl\\\\\\\/StRP279aw099jfV8Dcz1XRccJZdcrwAhyijkw4\\\\\\\/+9P3evhphkMqvUJh+ESSBSU9uwZ+f4pNmgQ6ofNNiQq0W2nM4kQi8TxdF3\\\\\\\/oC81SHrKbpbNuD8h0I+GjFcC7dtcZLg2bvFk6djvNokRHquVYxTWny6eJflyeytlSmbmmHSWbeWtiPMvse3H73h877xBM9MCDriIoeTVtTikuP+3WkjXHh+7PEvH0IYXpsXkanJ4RXoLQvvyplK4dE73dLHgacXr8vCbUjPDbV2g+GDvo3\\\\\\\/T8sRFu55zsQRzd+PmmL7G9YeGuGGnV6eP0XkcDPxITelA4jD0oaZVwDnBk1Ntbwm0IqXSt21M4OzJ4PMQkrp8o6scwflXil81rXStpzPBKxQ8vyjITzsow9ywxPGKwg1TIuOY+gVt+LQ3jde7YHioPxMupaGf5dFFUUxdUKZFBZ135SZmP7pNafecbgcWc5bdUp0C9ULqvEu+j3zSGr0tpfOs\\\\\\\/tN38UYF9\\\\\\\/AGfscTOkqU6cnTN+VtrbK9ooqOsDd1L7pfJDbmEuBQoNvjzavLZqN2TnlAjcL097+KSmZZ2f1K1dpgv2nbglxMSLAODaUVWotg52QLUS0hX7UNGqxEkTOQlfpmJsgNhTtkPf3x25GKR81ctRLQ1ZRq7xA7cycIZ+mE8FsODve7YmrgdT2Z0WH\\\\\\\/\\\\\\\/53NaGirU8q+xaLZR9jnUeLxOefvVYl+r+T13qMp4FDi1g7E1RvGNz7MhC6VGhiYERYx5RLyhElQEFEAj\\\\\\\/d8YfONAv0+8eFHdTOUqvy4ziHy5ayxdzvHntfTt4S\\\\\\\/BbPAJZ1JlRwaj0+oqEtTO8sy706vYS0BSC3xiCjiUlzcq4kDdgsBYHoHFEovZqfOdGJSHBclu8I+Bx2jB+caOzRn5u5Ia\\\\\\\/LpdDvT931vllOZt7nHKLnPOaiQnOR3k6NZnlhXMVXv527W+Xb5XVDRLvGMmrjJfjXqHV2C4Lh7I8mpqxt5Dt5P\\\\\\\/giPyyb0H1xJ3rn1Ewmk3R+ZWHO1QKrYgukBJ3QBdJOdwAS+onyri6i1KkTvvGvFFrc77vgnurbwBYURzm2BMIKYWTwGZH6V7fG9Y854nqXgJqacJUyILAzaChWaoFGin77jqEMtMIMXEd51A7qiYWrClNxuhK\\\\\\\/52nHy5t5UF\\\\\\\/uAaAg7mE5PVX2gQlz+blShJQgIZyPskN\\\\\\\/j9joTdF6N4SFFDVY\\\\\\\/uprOPn3UsRet9rnnwmumlIVwWrSG0oJNrzOrg6fUGXxMqIinm8DKf2kWqvzA34tbsCj40la4p50CG9y1z3uX6vXi2wfb46hLxzVXXWSFJ43SaXtl86f+PT6WJEPusPTZU2+xVDb2y\\\\\\\/LYKkLAcF4+DAANNw4gqV8MP90+UEpA4kOO1gAVxWao2E8Yr\\\\\\\/wD47tzYTvBeQSqTLWxT7VhQWTgA7A42\\\\\\\/7U\\\\\\\/lFgMdUpEUjfo6rZP1XG1V1ms4jGni0DrHZ7nZLpHQspu3Cm6XP+sdfHxu\\\\\\\/JABEio40aHpgQgKOdrfgWXFhDmcwKeF2Ya0eknkg9LjTuOXUviP0EB13U3wUnW92IQtJXewudcLCRkSVh2cNo7pA3o8ukdoclLiuCak95BRe1G9FWmrCVsdq9vUWx5Q33f8YpArwLKCp352dAJerLB7OsURArtdgy9lmOEDr4wi\\\\\\\/FLadXbB8bFVt+5BZZHpNPjR4EAMv6L5oUtj6VVq3++9KJ5VNICsuDTljhSd8z1SX+t5nj3JeCFizIOlt8FfjwRM9js8eSRHkcrG5QBGxruFXpLLnq6XXnmIcG\\\\\\\/Oh0pe+CuweuzcGkpdc1L\\\\\\\/7TXObTQEFUAm9C8wWs+lCEsdHrw3KWGbXIAnpuPyb1pbB9MgkHv6uZF9P0SUgpdWE7P4EuHyQXgl7EvX\\\\\\\/IQdya55se\\\\\\\/s8\\\\\\\/1mp57blZYWU6TEjz8neGhr7iFgoBgGHBB4XXZLbRNy5\\\\\\\/iWhVGyAG1p7nNbzC4n7w1hZ1Hi0024hqYvuKIBOXAXOlWamfXIanVYVPzJlWs7OmYMBuwycGo45kPTiI\\\\\\\/ox2UWo7B2GnzLxXeLt0vPixwrAJuTCuTFXz2wf7NCDBZI7K8jWa4RsF3Sj3Kl94unL05zwTalaFdJAou\\\\\\\/SlH+5FYf9YdzsSiQI5pabyS50wTCJpu1sbx44sTev635DVdg6nMhDOPg3OcAfN7Sb4c5Xb6LmtBvKpB+U7W+pSzfajaZ6Ww8A6oGkkhMsRCFXROqPFJQFhhALLBieYv50cwx+pEQYVZSjGqydt4h943lnzVWTeVnyj+mZfQHasZB6S8OcUZWovkGl5iu1KqQeCUVblN7dKHx3t5Ns3w\\\\\\\/A5E791ouyLNTc2tHgqMg6nA3yGlS5oIQIfiVWoFGBwY5nLs6OjIXxbTf2H5+cwHdMjGxKFkw+7UUgiNfRl9RZxG+wVcY6hX7IOKpokL24ATVX0AZy12yidvqThUXZK0y+zum+KfAYzybqlkLo\\\\\\\/pb8k9+0E3yKLNEA9nWQ26GTaX3TyAmGWAcQJZCL+8x+2S0HFvlCtGG+CasA74td69qiaDLKFWjGzbm5Ma66HeNOrZjKx7DmX0ZPwkKRpBb82s\\\\\\\/hWnVSYylK5sfEn\\\\\\\/UJSK6J+jHFSevitaczKP72sKQttw3JFxQynVfdC\\\\\\\/hozTUrEBqn2NuP3PeJoIoN0mJZBUAztWIottZX+LEuvDOPhlEWR0cdKKs2x2YdrRg8O\\\\\\\/0LBZwJLa9aU4AvD1ADzxbT\\\\\\\/tb1yPylJMIzIou\\\\\\\/uxWD3RLNsUvnDalThwHgRkSiRE5jvG9nFhCIOevCTeCXXuNF2LBv0DEhPct74X6kphXvq\\\\\\\/HSJJN37E1iZSjm6XVNJb8yz5o0oKcBdWdE3LqK36NEIGOoYGdBxTMtupWgtPUk3QeHbs2zJhZymg2hoBx9hjkROlOlBdNfeKV7MSdl96eg9FVHurCxzCr3WvK5Ew3yQI6l34TXKtL2rY3SjKP0thNG30ftb4MDN8DxoqZiJrGXT699HL\\\\\\\/Vd32WCQe0vGheD69Pkc2OUNv4Ivblt9ExXZLqVCo9DIRhM8OI7FvDbjihhjAmFcEMCsr07HuhRrmEZFXIOtOxP5meRrHa5wZAtAV8fnIk1XjHtm+J+cqLH+k1bh4rEAgaL8Fyavh1EbzhamN6EAD6aDw2oQZ5qR7hnj1yOn3AftrRg+I22lPI2CFyycCgCkZfHQbJ2UTCUYdrEOiz4715GJ6ntDzdh\\\\\\\/2xKv6z5TYfGhOhC9\\\\\\\/wvtfbKdE0h7bw8zNAfMCUgCeO8TvuWSEycPeZdVOw9HO3KPDl4+Sv6ZdjaIZyqSxOAT40YNjby\\\\\\\/TVtDFBzfVWr1YZczuEQEl4t5nXt7XJcDoq8MJyS3C7oq3wtOBDL8WthQjvRSAbasJSiqyGoc0YrDTCok6fQ5GxzWO42GeFTV0ZdsrrbmI68nSOXNUnT8IHfJCTQXfz6sO3yXqSBiv1hStzzrFllLyeEdPpTZtx00VGEM4l+ZeRdeVKLFj+Kh+NoW0sgIBY1DcQk5bgqFuHx6pMN7\\\\\\\/MOBAfvTeqviuJNar0LeXgOdkllBHjogh0VUzQLGKHjXgSspPXN+mToGgbozw0LnHJeeFnrsGlVOMcjgVBwl18H4fxdOAVGJLzJMLDEqAYcaYrWZaNftlj8eDRsZtv5fofMuUuD59VVvqAuQ+e7MtAaT55FM2AsfltTtfmn1iXeY0eTSREy9jyMZi9byYZAVMqKxbku75bQR4GP3zzbOrIMkurdZ23FA6jetljQ\\\\\\\/c0A3d1c4lFchhawNyxnLyCzUhLksjzFsA6f8IGlzNa2SWerjnTIF\\\\\\\/jyJwTCuK1toZYWD2buo9Dq0lKfSK4NjVhbZxjC39in6flVNtj8SHyqTtsi\\\\\\\/A0vk6rPWwt3Wf9U8Z0L\\\\\\\/2cEgtbV65a1v90KzwlbkcuI+\\\\\\\/9VFzT58vjehFYZPlYD1vGZHCarCmbBK0MEusGwzig2v1\\\\\\\/pDsb\\\\\\\/p1eUu64J8BZ+uIy2hRKqSdTQxECFoE7eJH3c\\\\\\\/MGw6BZtAw5+uzzCG4SzoI\\\\\\\/1+P+Lz6xsgwErPYf9T0JHXNE4jFbogerjgiB3GOL5SAAXKjNQQKSAhvX3+mvsXT17miEWtMD\\\\\\\/0+L59wCh90jwKdIsItHrYbpS25fGnxCj5TFrwcRKalKyhq8\\\\\\\/nPwn++CesmnbHzSoRm2sBqyGJnB8Jxa26XgLlAkgQf5j1fgr4cB3E0MJ7NEoj3CW3lat1E66oGz4pkRnDxp2gZ\\\\\\\/RIVZxuugloxG3XUipx+xo6KggF6FYwcWLgPsf8\\\\\\\/lx1oxboPcpKJxL2jEjppms3\\\\\\\/HC3zB7gioKM9esKdxDoFdhKzpkA\\\\\\\/nI\\\\\\\/xYNQUjBMCkSHI+iG56GFB137ERp9Xxpen9oh+HJAHfBM4Cg+5bw0p84h9wDkb4Gdn1deLDr9vJ1eFMTT4kc3QbrnVj7jVUtygNDF1\\\\\\\/aeJ7T+RK8rf7gTvqFYEZixqRwKL1U3KLt+fP8s2JTgWPuPfcwTIOfcbkUgkFPYHK\\\\\\\/kz2vsztMbBYJpq3DGR054jw+yHEq8QnX9lh6kwCmCK\\\\\\\/fNGhHEvK4Uoe9tvV0DhGv7EXdwFHpWLNNctoogD9aPtFQ1P8wowneMRPIsZBClaPHcjB8mGEO2Sopve9JoU2sii9u6iMV3XIhQpCr5UULC87WLTkXmwwOeNGVr8XUL8u8xxKqoZ5Zgx+GlY\\\\\\\/mSvYPUZjKxvybN4Ue9\\\\\\\/oWWQvFORG19\\\\\\\/yfRl8AJlHXzbihi1AGy8kHRWUjLLsxPZTLnRhzsPBKN3XyInFiMUyYUWCPLZFu46hOa8fQXYR\\\\\\\/ptcaOcRcBUyhShaX8lovQbNxBUxWZ4cRLy8PN1+6BJWXpJppKrT5uLeuBI3cQL6J4dCfldy3chLN7peGLE9Uya0TsRWd5W6b+WQzfS5Evab6lth6ghVSpzsXKaKnvMK9xCA6\\\\\\\/XUaMlucCrjWje8imzm3wywc+GFDJT0lVmVgspllfz2EDiEDuNqJz6FHcdtVOuKD\\\\\\\/r2C0q4xSlOFxmiEWlWO9OaOKdfP+C5RqFvB6N\\\\\\\/y03s\\\\\\\/YVqh7t6ZDzGLdmjepdhUaugaPB3juk+JvZ38bhHV3qzjG0fM1npoWikBq4E7lU7VblpKBvk9JAk4qu0H\\\\\\\/3bXa5DtpCfR1Yhpmdkmu+P+bJ0Fol7\\\\\\\/\\\\\\\/zsqXiXK40SZrLP0deoaP+GXDwJO3lLwMJRXdEvEi3YeZfGOh+Y+VQ1Do9nw1PacMcITWx72FygJjJ31yBaXN4XaXewDOb6ZoTa5GAzg5hVg+hx5PirpH24F7aIgHEVTk+dpXUnBg\\\\\\\/COjuVinnI+VEJoIeR6aYxyvmCbydvKf9HjRjaKbv7K8eRlasRVXp6OFy1eepnZ5QHPDymtNoDa12ZkuCKHYMvbDlXq1v0CwdZfCAzPzuuDHxF0LogQYKJfVCuGvPS\\\\\\\/KiLxOubG9wJLFFwaZwvG37594mTFZBmgCE0hBv9OLwUEbO3LV1IQBPaXhSpdoD\\\\\\\/IOdhKkNMDGJlMBw+IXhFYhEs\\\\\\\/M9+Wn5hqIFLdmZ4Ao12+TM9bicQ6xWsnb52cENReDqyLDUOQAp2WBz6qaCvoZcPKfjF7D0WmlaM1Fc+d+sEr0naZFOVurlEQ7MkwnVJMjFqqBy\\\\\\\/Q6wZQYe2LZYC2e6uewnvduWPK+RtMIJDFDTBWkNlvOR12B4uw3tIhodU1fTRTUjOpdM7CL+G39F76Nhb3BwlPRfHeJDH1sArsgI0VzeSyQbEcxwS9\\\\\\\/dSvhqWIxsEV8GGnP6Fn8jIgEu3\\\\\\\/ADdIvR71tGYOKQyZUVg+GF1\\\\\\\/OuZcp7w05QHd8zEQPp9toNSGel9XWESlMpNwRcR\\\\\\\/8LlYWe0\\\\\\\/S3aYUBWNNVgcbOogmtWmg\\\\\\\/h3fhXUYgg9O+iNbzYA0\\\\\\\/MaozkeP1441LroR7g0Sz0X8Rl0RaHKczD2bsptlTqtvZuqgwu0yIMt9k4OTpbU4uH2NThIsdHAcQWferBHSgeYqcYYXPOWhw6hv1r\\\\\\\/+Hi7o+Nuc8H2Y1KwSuHy8Y0cm1jCJ2iia91KQVpfQgkXL2GUIVKcDe0lHP1RHeqNLIrDvp0QybehtTu9eidFrOBUZll5qoNU6rPre5sJaKfhWjQUcBj8BkBgThwWI6zBkah8FWf8r\\\\\\\/ohMYkwSBM860ufJ\\\\\\\/MbtBF9zGD+CzIXJcL86Jt\\\\\\\/gksuhES5\\\\\\\/ubbCRi8le7h3KD4vHRHPz+LK6Haz7pRxbW\\\\\\\/SXx2RqLM81gHbMMUgP8OMsAwQ6fszJEAYbjKWJI23FmVimcKmIpXuyubLkR9KPC36aPxErKlMq2n+hVhn6a1Z+Q5+RlivhkNJclLQkbqFQ2l3HTE56D2N2WgnqStyGNOI645WHP2C+VdXLpmSsAHW5NahP4iyn4K9bX+hvEteKZxtSAZPou0+TvbfT9BbAAC73cc2d9Znz+mMq5PpaFuszjDxq2GEy3iI8kNST7pgNIt5YtbjIEBtxtQy0IHkqWcbMOQ2OAVpoOpvEGqmrTE9LcvYorhSu\\\\\\\/q8xjj3wOCqgdeWh1Rkz5IvEBX+DtmFjNpH9z4hoQG3T3ERFAPkDOZu0q9UH9Ivu7T+BUvIzK3UUh5mX2KTJZ3XMEvh9zsP+whJbhC2B5T53l\\\\\\\/+R5o9oouMLaDQnzCTfQAxAO7xwXGv81LdimKQJqdWPnIKH8BWWAdGUfGbWRQOglqDMETwFDcM\\\\\\\/gHCBECDrv4IyMfbYh5x4NkjWTDAfiFKUcY5qaqhpGB5LJIf7m59R2\\\\\\\/Cq3FZIJNA5lO9zfcGl2LWw+Y1GdE46U27JcVSOuHHlMI41y8SGs8ZSiGMqpUB6DUpbNOud3aqlbHIbdIas3ips\\\\\\\/Woib+5VUabOeyBenDbk\\\\\\\/BeJJ5jLlOFWS+wWIWqLfTxi1hp\\\\\\\/P+wDSC5Q6kBmk8rlhn66jjCZ9oz+xmbUx2iAsgCJd08xif7MjIj4gtSrFRKLFfO5EKdktFwlYgcJW+WWlaIHsu84Ae3Fdh8vOQOiXDlZRlMwgmwVtKbuLd5CNFUnI44zEk7r5HJ61Oq294jo52vXgji\\\\\\\/XbKDzg+urhhsThWxxfUVSzhgVRiC\\\\\\\/P1Bq3V+Rtc9eLNDgZHFlkihx6EVa9OH2Hhf9So\\\\\\\/Jy32cRvQ69G35gBiogVdx3\\\\\\\/hqPOaL7z9bwlx3MUGZI9XvncqPdiGq7enuft48u6q6MJx8Sr0EZoqGDV840mRdTBFg4QhrU14OpEUoPCdQqI7nVFfcRe4l9Avgfc\\\\\\\/0UtSErxNwwAB25KC4bsvAesNmlX+cYUwsL40AEEFKrp7AoyhkG3FssrjYsRHOa2FGcrqDmcy2j3I7wTRxiwtUv8vR\\\\\\\/9wIYhtKtmKb9v3Zoe\\\\\\\/MzqovFZzF\\\\\\\/1VbWHhmgtrrsSgz79GcSA4PILKzd7o0FmsAVT7qrvYPv83SSo25+tMLhRfwqwOjH5si5rfN3yPR\\\\\\\/goKqhmmNHOkZEAt3yEV+gwxum1f\\\\\\\/SRHoAIY5Sl2\\\\\\\/ya6IS7157YmXPSF7E4RqiXTD40doM7EQP0C6YHha8mPDzBlDWABvukdcHzbJ+qsh\\\\\\\/uUxIrLLeWQFLSZayHr+me4AU3bTaM2q9WuFE71YVfkndhfCP3K2V18XPPIvEDAHQmDqHDIZvBzQJZhDObFUWXaX3tUhU0i4msB2aAhpwz37pwFWCUNS8UKpIKq7hGbJxLXZnFErEDmRm3YHyyY\\\\\\\/JlZ+Z4MEMY4a7jHlipUs7XdXDRvMiblg31983TztQ0ykG6QczbnLh2tA1x32\\\\\\\/4YYcYZN1QvUoglT0J3Nrz3cmZGUCSvoHkBWDCWeGSnbOHQ1AORBndABZic53E1g5H0Wb7eeSzkNJMdlaUH3JL2vHurFgkwy3oyHbdLeJfzk8pRs7jCyUcAisSq\\\\\\\/eiAkhLe6fDu8z3h++RDZhcT90yse\\\\\\\/bATmMjGjsGOkRztaEpfuRue3JAN5FxLe6+QEZDPd74z89ghqJo2FuhM4Z05WSd20sgGcA4ikTlhGVSrokVkRxIq+YyuI5lPyCMn6XOGzz1+LIW9BgF88xa+57HaqWoIjSqY7wIGdDMWs7NzTVeKbz7GHExeZoyP2pydduHvYV9OTmFp0tCht1gExPmgVeeI74DMaH491TLCBsOOW\\\\\\\/gHFmCQ2ovl+\\\\\\\/Jg4xcmdpNj1E5BaJSZ0xW\\\\\\\/C01NNVXC\\\\\\\/NrjpugZofuvxS5E5m89VXC1Jel7twpsmTPD\\\\\\\/2PKPHU3rFOFbwu6Ry4HOr5ozND5\\\\\\\/atWEThHR26JqrO4DKa4z\\\\\\\/saV8kQB+LRqgC7jSNJbTIDo0722oGUSfk7k9wvmp8HNjf7KZwQZQ5TvSm8xvG74OMtInzrMo5Fzwa4W3RMSVWfNmgjNeI\\\\\\\/O\\\\\\\/HflANMv3jJo\\\\\\\/rbh5BzNpABdkZHKL2WZPqk4j7pldpZWu3NRkd7912vb0M4ClDv\\\\\\\/v5YVo6fkhFI3f0Gtf8i7t6zXT\\\\\\\/0\\\\\\\/2TB\\\\\\\/i84dDwCIYx5vsuTqkVyaEDxjpCkDP+l5tmzdvAnfShD6iC9bweFiFa04B7fA58bR3R1C9s5AcNF+LbTop23qxE0j88EAamJoo\\\\\\\/Mot4de3jmhFSKNK75tqHJJOfPPxEREvkVvedvcpUtAUe+NNgr038fKft0ErBHbIBS3GMhR3e0rUwfE0OL+9ZyHw5ijzNE4bpTrjfR5INNzuZTGmwNfyqIha2jvEnTL3wY3jlOL9hs5Np0WULP+6Xz+KoVjRvBsMmUZFcZjeBR0h16Nz56hro5ttEQiK7JG3iWQyCGBGiIxobjB81IwIDn6aJ6Z63YinbILLMxZtVsf\\\\\\\/fraiVzQOdgRp6sJFbhMHjMvau9yP85bdx83yIMkedBE7eWgHtQksu52AbNJMYsrIiHdT2x4lKW2CgZdMxrFMd37KjQtnEsG3DNCBnV6Jzkl00G4rDW9Y6TiQ8YJbfBp+pFtSvBUOzG7bg2WiUviYBhumIoaLG6GqUhoQkMT\\\\\\\/gB1YGW8HzNsSipSKGKjKJT4P86my+yHoTsfs77Fhk95G7YCyfiiEGM9\\\\\\\/bgIuLZqIuVtU3+125Wb6uK0Kpcm\\\\\\\/UY71kM\\\\\\\/1Ikab3iL0Z9xWCuVQu1aYWjbGBeBZwOv3yg2Sf499acLeRyrRO91w4l3pH6d5NhcHqMBl\\\\\\\/xeqqrEKL9yH7Gyyc9RbubkU3S8oWq3OxqaUujAf2FfBcxJlcSJQ6bYQ+G+vgf6L9mNlZQ3jEc+2qwvOVGYuiLtWD6bUe7iphaXUnA+mQWGng3PY0CrLOCp+WP+9ZiG\\\\\\\/Z+dT3o1xR5ZbBXaoO0ijztBqr4xijW9Nq1F4CQQ0ilG\\\\\\\/sjNyuXRwGtuECkYh1lScj++jBBuXihcBkbZ4xYTPiYwG1nwY1d4pKlkpwA2iKScth0ncytIleHQFxJE0DJCAQhYyAQa80BqEQIX2kNcC0dBd3IM+EHRD7ypx1DCZ3YUMjp\\\\\\\/wRmsdXSDFjFFU4sFvoZVXsMmLFIAAKL+YLv0wadxDYRTLEFLPyTFRvqP5mgUcWN0Nw0IJnIaHr7+Ay0LDL+VmHCM+1WeayloywALyf\\\\\\\/6hKGZIAjnWLVl7etJtStXSYqdLyt57cRsMflzwvQGSinsGZuZkzGbuDi3aXsbEu8XsZwj3NQkQyH442+FVQgifQJ7KH\\\\\\\/1cJTmIVz6ykw2CqzXqk129JDiz+FKAw0MrBu2vkW6GZv+OyDwl7Git2gpcNFngVRHqOjwWbl2z94mGvNpGLg+ykVFxFOiFe\\\\\\\/fJ8TT3606cCxIp\\\\\\\/xLQ2Yi\\\\\\\/VaJkWLU3HIzSuRxciyS2h7NsojifQN2vzVkFfbKi9rrBKLVJiyu9wsJ8nnXesB7cksO3sLnu\\\\\\\/XB6cFXLCHNS5Cq51w1mw+Y0C9ITghgp8yz5rka1suzaQFYetTmGpbwQ7rmaQ2P7\\\\\\\/q53pO4+OFdQnSIzvcyIFyLgBwrKwoIpsj9ev+NQGrqatTWVfY0ETGYxvKVOjvN8Er+6mhvRytFfM8LixlDxN2Vj68Fx\\\\\\\/pZS7RVgkrmTDSbIQAvlyz5XYwrlHe7ujCKzSCZltZrI7y6SuYDBWLzWaMsoVq6tmwpo3C4taosBUAOZjZCef+VkGFXTSOGoZvtPh6NdjH+oXX\\\\\\\/jGisUAj+mrzOE5NFdUAh9chV4gDvCfnRPV6eBSxPIA4yN\\\\\\\/4aPZwf1nHMvlS\\\\\\\/nGVcOdbnfMt+JmVFGkI8UsuhLpVO9\\\\\\\/Sh9BQsCCWnUiKFe7iP\\\\\\\/J5tvz4FUW2olkqTYndJIa5dbPb8TblT3jvAmLgyZNnUFMyMzPDHO5+pKeP+GRdByedxqRSXXBwAfQHiw2aNxC9zraCoi7gEJhykne3QBrSmTjLjDMfjwX83ZGZNqmU5NnIk3kTPy8n+avBoY+lIKc\\\\\\\/VnTyWujfHDmGOdPwK5nTSNRtDV\\\\\\\/9d6+eFR33+23J4s5bLPYh1LYJc4wg4TPSb2KdPAmuPe7fUsZ70ljIuvzBqTx31OwkGAlSsyBioAXDp9hxuFldGcbLWvT7lISJZhRwi+FYi34a2Nn+xqqaNOebWZ5Pt4dM8MD6yWnZWM8zbJvdiFjBLAOAzzea\\\\\\\/pmJjHMlvW3jNkqx9guLkyOeIrJjHGpVlPD70TQLeqBtZyYjSdLkUQBiSNH5hu0MTAqxBGDyCWAD+DfYRd07S7Bj+G7EgNqk9naKZ\\\\\\\/shJ1i0Fd58KxbPZNJ20zLgox\\\\\\\/GRVA1XWePek+udFhKIcfIDAkBABxFc5YgnNu3doOn1\\\\\\\/FbMPndoWa8S83GknBtuBGF51vPW93I6\\\\\\\/7aRJrz2f7aKalTSIm+S4jQnOynm7H8jLy\\\\\\\/kaU2yUZRsMuz+\\\\\\\/UXUsZI7tdncwVaoovnb3ld\\\\\\\/rgZ2je5twiwfvGbZZSIVtV4+k4UFP85EqdR5\\\\\\\/BZY3M9LrEBAbNP4pj35whL6yz\\\\\\\/Vr3mWMuk+QEVh\\\\\\\/R6lTm7wI1HzShRoYdw2I3mvN8ePj8oDI7Gpv7kuflp45IoQETf3YHVFxf8oXTEGTQeaoeQMv9csD2Zf0RzHTZTa0lA5tTrK7sCcGyydj\\\\\\\/unTNCtPNUadxa8bE2hU1lNJ4+RPEuCcucvp4LojCB3nskKDeIG8GOS67dTgxtvFssKMi5RdcpQBDe34YWJehC1ASlHo6GzPhUtcCMgJ1aSwwj6ZHM0zb6lM4y6qCt64ATCPQjc+nQP60wBXJUPF8NVjmh4QedoDDxVhaMG2MzI0YppKF9+XCQkEhFgnXk6EHP4Au8yFMBxxYx4tW1k17A+amRNB4dVwhm5iTzmvsm2I9McpS7+l7mp3a7soTiYGZiAirPCPycav9ImhVidvewm7huaKZZuDSkJy\\\\\\\/+IZLLLCeuAEX1ctUP86tSc\\\\\\\/Bt9r7Bi559aQiDayqxMZ2VcIEu7Svi3k40IqRG2GlgJwOkKhBzVJIdbgmNUb8ge\\\\\\\/KnuvUgE2NHKroaj4QcbzUblHTskzK0sRJWa4F084O0ODGpaCNMJ4tREOb3XPn5zEf68CYgWL1LH1+EyPDiJWIKeVfYKRlef5q6kkLP7Q44ANp2ScMgfsezko5iS9rV73W3tk6alsWr6J8eHVvMeOVXnePNe0qErsQ67bqxZ4WW\\\\\\\/ZU3hOhp\\\\\\\/wB2RwlyMAxfoy42LnUPS3BsiQE1vt0EY8HoG4qSj4z88G0d74wwG7X18WUAAru1yFJoqOmROoDw2xmN02wLfnctqUaaI8eqNqUPJMwRrxaEoMrDhjFRkWlacfZS3lyusbhDobTFNb\\\\\\\/aWtBjNq0IbI1VQwFavaIK37OjGLQpJgVoJI\\\\\\\/ThlWautxh7bum7K+x393HyVmGjdltodlrD9JTIFIInOy+DX6YBCkT\\\\\\\/5EgJ6kphjJO7vcMssnc5lOq1t7KMUm+l37c76DSt0a0mq5kkCZ\\\\\\\/oB5QFJpB2dHhkBh3ApNJ5MH1srfOhypYdjfp4RtMIhRiLzdivVAZfVn7j8Fvk+6oml1KVBY59L+WaRMkpw4DTJzbOc+Uj6wBSVEUOTlBHcXOQzaII2JqWPvjo0eSgV\\\\\\\/Q1JV+LXwdVDN\\\\\\\/LUNQhhCPOXDfnoDonxnPz3bJnpsoUqMcEdCICFBjrTwBdaOFwFXAwV+u+AyBPMa8L1Pe1c6d4vJZB6IpdcdRAvhsgMdL3ezCHtwbBAMjbakMmYqcPelkQbzkWkz\\\\\\\/DcbXGWN+d7c9AoDU+DHLRzKODvyOx6KS5KVUMhlDfY2R3s4gOJWvUR+9JFkCz0DPHZLxS2rfy0ARZi+Tol7IT6VZuoCvHxUS0bS\\\\\\\/T0RYShcu8mm92L8wlXeaxMylHQSg0l9Trm+JAa08hAjgLYmKG6Hc8a5Qt0XNdkEtKa8R5qjq4YFuBwvt2\\\\\\\/mT1\\\\\\\/mhsTgj4XoWn7C+Z0NDLIQ3X5lOi86uhMXiRWJ1iRYU+QPXKxtanCSoExHRqsjZ0J17FvKVJcM7bg1ziB+bO2cK5nE6NaSLY70aRFo+BdMf9aHASEiIARNFatSnkRFmLdNXKBGgSArRINU\\\\\\\/b+PFG7sEB9UFRIzsoh5HQzAvdw6EvnVEcliSHyAaQGaVXWIi3G3Pwgue\\\\\\\/CV\\\\\\\/SK\\\\\\\/qYGIre9EcAa02iKvwwYm2EkP7AuQBIiMlHfte1IzoLeGMsb7KUEU2G4\\\\\\\/H0NX3cR0YgbNxoHOwCEZtXi5SsWg\\\\\\\/6ZF0\\\\\\\/Pb8VwNcJWtXMGzZCs2vqQuh7R1RZq6IMmQphuyy+s6armNNjMqhZzue\\\\\\\/lXUdDNwAqBwp1N9i87KgKcR09chVGrdI\\\\\\\/i22FQXuRm5fEAeirvOmot5rbmDkPKlQylPdYgZfglneQzjKVc9Rl\\\\\\\/R8+xyjaZo3owK148Tl2tifa7WQsx7KOjJnQ6zNUXuSjmxR9uI6YF6I5Z+KW8H4EuMXhwhFj5dsHsdxdF5a0U9jXfkS5ru6+zA1v3HmfyoQWq5wZK9XIU982cMCPm3wexaz4bmqtqQghHsw6VJT3dwQorOSEQvsmywQDuS1coL\\\\\\\/nYisIhEoDS7EXFpsmtyJ+fqeIacB3Iicv4bYQlYqKSpiWaHTS4f3nQKky\\\\\\\/C65JU1Fi\\\\\\\/ZXdY6p46BZq6B0\\\\\\\/uO8xuDQo+OSYg3pATSAOc7SnkZaLZxqItcEfUVRM40yLv+2e3y3Tq1HMZrHCwSvVYE3750dK\\\\\\\/MDGlTeU8ljpjUdiMzruJk5vEwC3\\\\\\\/4i4Upbi0y3NLuRBI6yLZVvZ0a5Kf4MtxdL2qHpRXVEwDfKFFjAxLIrTbVOj05mGZRlpvm9mu6lhWxSNfCHloYVGJl9QoazCh6JP8lLVrZdMCf\\\\\\\/qKU713rFoRchNiuoIzbouleTAaVg1Lnk87F6cpw\\\\\\\/9\\\\\\\/bupAGwjbx6lgzxNXLRgrDn8PYEaLQhIqFaq170d4lzJjLHgjbu3e2a8FFcpn2Rh3yE9BqTCl+zP4L2ItwfhUFIFvPX6oF1Pxy5gGD9XGj5NantEWbfMoRnIMGZuTykv7xwutc9jgs8TUUuAKET3+ySPmHenZEj\\\\\\\/XTi7pBaScfm++w6FSyQF7nyBnvejaj9mZSUjNNIBiL3CpuF1basMprfwsRRvHbJvkOxkDuKyWwWhLY+KASy8PEKUvNnXb2RvQdxNOAwWd5jOMmJlrBDaLJmEOpf5QiCD3SXEUHQCkdhgKPpoQJV4mXrz30rb8Z6sz8iWiCAgFbgdEG6es028C3z5tBXRbFW1BIJXydRJg0mgwGqilxalQ71vuu4owqTodJEzW+MoTy9u+UTl6ZvYPowyZW9WvO4nfq4aGKyONq\\\\\\\/B5eykyQ\\\\\\\/eIn6uyppAFDibI\\\\\\\/x\\\\\\\/UQc3aX3AITD127685sROeJEWJfgH6VY0b\\\\\\\/dkOuBVz8SiH1kzj4zCbCeYzqhAjE6rAJ4wxG5h1kZkssPX2MWJI03qBInIxQjnF59Gz\\\\\\\/24\\\\\\\/yXMN+ewUPdjzAfG0H18\\\\\\\/Twqk6lA1TvkVge6N7muRGqEYcQ2QRs2SPdKpGIU1YS8MMn6Vokt7CNxN6\\\\\\\/\\\\\\\/gne2BOd3ZK0xAv1yEDyiiZApc0A1zyLXDf8l8G2nwPEwZGI4tVzvBuOi8Zs9PNqBITXZiARf2lCNt3YDk15TkviJAAyxANUVuUqz0XD3uo6ezgQys0L9H8iCKbsMtUvMUWEtVeUu+hV0H0M9yfMvsuO1I3mwRs9pa1xgJYDEZ2gUT9bsOj5qhtVuPYeIXLLYuGhpToXweXCozOxlK9BsJyFrhktD5LTreV4WU2rwCkIz8A+L5iyDeKCjvNIGoqbkUzyidPBDk4GIZCxKEu8qhTefHgMjQFrag7WuyAoWHJ+VhcGgs3IyfJFGm34YswxAxDMN3EIlThDl7WPQtXWY1tMAudxaWuVdFiqoHSOrKGgx0jWuJn4w0X8F3\\\\\\\/eRz1AYsqSk4xgrOWZy49pVFmq6NR393s7NPzfxlLzuZQHCzQs7\\\\\\\/hXjmUTCvZR3cNZHfzlUMKlqI88TW3FUL6k\\\\\\\/QQ0xe4uU9uUSA\\\\\\\/U7HUO1yoJ2Qh7eY7WT3qZOLcC5sfM9DGs1\\\\\\\/mzsoTt4nRu4hUKs4fqfxiL8FGEmhaEMk0gNFWW0mL3klxG8sLErkXDuyjlxXOR+4pi7yAGoQKJzYoL1DQLWJjihHD0DxxSHa5\\\\\\\/tQulmJK\\\\\\\/eWNNNT1u8oeipU6C10Qki4uac1TUZtobUli\\\\\\\/675RSwOzh5RTiQZsX4k+HFe7pHQYu3tKY+Et0qGppyugD\\\\\\\/z2nRti1l+vXm+ioXqpZiTk0fpTQeTtbWVELvLgdkRnGbQ3oh4HxxZWT59UYs5yHWvqCsBPltagVQjdAe67rKkXwzRpIua90zkk3\\\\\\\/QpzWa4cr48nXMlshmTNWjmbOtePDbBzt\\\\\\\/GY\\\\\\\/1fYztv8JAmoJKO\\\\\\\/WAPMnUIBMnGMbDpchH6sNlcTYxYGJerlhaDQdrAUGoTujBJ1P8f+sOrFtpkjQyznp8xivfhZKEjZ78JhBjFUXbftxrDi3mE4KRIC3\\\\\\\/lHbdY1WqndlIuf5z8aLT23iZvpAOrN4h84PdlOB16QtKd5Dfc4D5j4onUeLqeLhKAzgxyDaPpZQS0R3Ux5\\\\\\\/KxU3D\\\\\\\/FlP+grdO\\\\\\\/6jJ3xXENN1boow6MMJycSMfg2yNwJfFmizrhYM2CfmjkJfBvH9rvuiQmILX+PoBsObJIzUZMEyQW8BGck3xkKt3gBpg1UDTzjuVE1hmSiNlb9DEpVaiDElEwicxhZhB1fzL41WtPGZIELpcCgyUoFYX1RESEkKehE9nV0R34r27NVvs8t2EMlaK\\\\\\\/X64Sk0jeHnfRc8qHV\\\\\\\/MqKf4HjXHi\\\\\\\/7R6LaFA6CSv9Kl6W5vryWRhbCLZMBTruaDn\\\\\\\/VGz\\\\\\\/gzCl1TiINwEb+WRmP43W8MEQNSAxJNZuY23A3dFs3Veap7Hb229KJH84B5YvxdcRFB8XmcmRhqmNq8uF1ulSN842DEboAWQ3UCbs+PiPeY4RnSycRiD4kaWq1jJdhxJChJnfZXSDH\\\\\\\/pjlfafkzyJY4PMnTQIIDQ6EioZY02ADdzTZhFf26vzQII0ccl5M0J5JJhoFqaRrOKK7h1BMCcn0sl6HzT9WlMl9\\\\\\\/3wADYDJbC7jbniiUJRlov+tjkPk2gyrisSZNiREAXjfL4KhZaDnLOWD10v6eogHe6296Opf0l7waY\\\\\\\/ZpznyLuk5yr8GjiMRRRYXCqtvyOgXSiS8WGcqKf4nk\\\\\\\/M8BY18U0muWtwgLQdOTFBz8UStTu4yaxTmih2Mrdphq+5Nx+XF5ww3izzRUAkyfa9V2toAJgllkWNEOJNgxUsXM8aUYytzlGzkgHqHsNLLsqiMD+\\\\\\\/e6We\\\\\\\/P+cRST7sptjYjuF2VdRGnf\\\\\\\/pIaPhRzRuojRv+\\\\\\\/DSRUyZFPvA8qYw3HeL78Ngzb1yIczcK9Uy9PNLk3452el5cL4XkCH0LSSvVB7E8pLzIDgfVvfEacyeV9tlNLiGiZ5jtTGLerlFjxr0wJhLU3qvG4EqoGx5MU+tTzku\\\\\\\/33KF7BYBrG\\\\\\\/y8M\\\\\\\/0WWFMuTSpjDxx+cVtC\\\\\\\/EQQadN30lqEv49DF+xzEkkuzMCOkoxe6nw5OXBDWAGs79KrpKSKLFxUCQthzDRCQvphdGb5dpKJBMIuiINGw1Y1A4ccqwpCWMoZaiyL1KCGKvSoSNqmdUYE3MwQe0koZpD2GFrJ8I0SqEMjI4QGhFY6uiUGIhg6oIM6PqCJT40fFzD4Hiip3RFzznHYDLZgn\\\\\\\/zeFaCTZ6K4LqP5e5aVcR0QNOQupxt9oI9yRbfzx0cF72gmibJIuw0wewevTbipfOQgq2iPQ0cfPZLOZkuisSYgCQNvrkHtW\\\\\\\/PuFQnO6zZlVYfc6Gn+YdTmcMmQGVKXhc3nLhQlPwT8OZqFn6M9mrXq08eCwQo+IDhX8afkbp4TH2YMjZqOZBKVN24cjEoDfuoCYDK0pkiFLdo2cGUeDWb0nkrjOyrQcaEJ\\\\\\\/1C\\\\\\\/6OlIg51+3jxtWazfy79nM52mGxIsgZU1nEcwnEZNyaRBJj9dP6Ue\\\\\\\/P37Zz1IPBytJstUO3\\\\\\\/PMYZlLrqZQjETh5WErPaF767cwgFuWQUO7ypKc97zy+jL90e1MH1a5we9IBZtdyMYE0FcV6s+k1rpFerwXtJ3OZfNAVyNKCmmZHPrr9FdidbSu5f4J3gHegCpqSuzIYL9v8qRc5cCiCk5E21CbdYeDqpWIEEDf4WN5XVxCM0eEG3zohRGKaqwihuqEEBL\\\\\\\/4w\\\\\\\/h5LMBsmMkaPVIG3VuOA3U74HVIlb9ExTv8cHDwQO\\\\\\\/jNPYrzJBqZ7q02uBdhyE\\\\\\\/fuH1XcQInPiq0NXWhREnVHSOWvM4Muom7rNQwRNILh0IgCMzEOkJniiZT4iem2ixr4FeumC8PRYJ+UsI4hh0C1KpNG7AQOkRB0dGw4ddyFkJTiHSsSKhgrr85awbtnyG3EPldjQ671z9jwbqm4wlo7HGB7jSKS\\\\\\\/aPOVZmorogcHuOQiwlCLGaAu9gF+j0WI6ZpUGvN\\\\\\\/turWi\\\\\\\/NPKQ6xYuGpQqTRJ8ZooM80O89LYmCcMlH16n6latwSDUWDxQo5D3nGcxkLV8hiJyBrJd349AjNJEKJnzEUsLWf3e63FA1b6kR2ZkvW5bh23yfX\\\\\\\/0DoGRpiRtFRdiGtqSyBvddqKpZlDmLj89ecICMcVSAjel7hBDnbfY9y0gtvXDhccqJ8flq4Y\\\\\\\/uqNN+gN9xDHK4CnrrQAp1uqXhuVfaYRllvAcAr1RXj\\\\\\\/jecQu4rC4ezZF39f\\\\\\\/m0ki4\\\\\\\/T9ln3iJoRz8mM9deSSfckmRooM0CcsrS2O3c1BKToPPzq1C0BIHnZzKOP1zOECMuWGtFPZIvgRP3zw4fo41HN6NBnaaeBLIZ95r1KTAKU22HhFT7h4ii2JEuMa4qbd65p2xomgN+gwvWSZZPvDBH0\\\\\\\/\\\\\\\/uLRgKSPdwa5H5cVlFtxdwvdNYail4U0lcCEQvZBJWMO2+yE0CJZRtxef\\\\\\\/pnJL44ybX0CZjC1dZC+nOQgQ1t5\\\\\\\/Q38QkNfZBSEtEjYVbfR2qaPW8V3aewCIOx9plo4uj7k95zuQIh2KHVL00m15zkTQrHXYVr8r9MCSvbfOXCc3iWY0QowfPC28BPo\\\\\\\/RZqGcuCSpX8d6dAE5cgQUzxcWUWLDMk+K8Cwu7gnnVNJZGfg5gMUcyfmVS1c7VQL\\\\\\\/9S3niq2dTmiE3prMn0H5B3x8CcGv4l\\\\\\\/O4zTRSrgvnbkT98eR01EXNlvrHyLAranshRw1wWWNSN+3\\\\\\\/1jO3GUzfuKBKlX5ARiQUmfHFSwbEAiK4p6TeY4e7dY4l+UckfmmciC8kWNfNCbFLjb0bFxGChBtAPPupxbu+uWFAOGcLZkF3ryFYVLgrNM5Z24qWtndh8HgNY4aw6jMM2GqTK4FJZdjJkGoawX7ubEadKbrFjthwwqFzdtlneJn2\\\\\\\/Sjz+704tMGXKqLUleia6RNpGT4oGyBWN1izaHVfB3nP6X1aTpu7vAddiEqJ\\\\\\\/gV5QsJSxjnuO3JlEjVRVZO1VsORoymeIV7oT1pEqLwzDaRkrGxsP0QSXET8GT0LBoXYMNgpw4O+a4ehHpVKguSpQw8SfJK0QotBTjSWsj9X7c87g4BmUWshZXQC4hoya2ndpQV0Q7vshY1IipU4TCdH3W9FE0csKyEh45IcmxF4dGApTpppTLc3PiIHtJ34Rmrr3DrIX38eE2RNpEQbuHNTTMfMKQWtfq7P\\\\\\\/Q7fkKe0UscR8V5wbBt9f9lol+c3dJLcH2MNLTR+Y\\\\\\\/Wbs6e8IxJOl4UQpNGe6mMW1oK5fXH7FLCD4X+7PUVoybVxGWb9ghLmplpx6eWUOxE8UmljpovZoNlpVz2NjCHZs38\\\\\\\/XqgLhuN\\\\\\\/UjkTbSFmBujNY3FgnmEEgEloJZYTVtsGaBLp3aaDQ3noVXGGRXci5cBA2axo3uaRj861\\\\\\\/hUWxcBLJN35sBly85e7dK0NZYbBoS685JCpwvD6dxi7oT+TNyJe3dhC\\\\\\\/GnRmcKra7g\\\\\\\/Tb7qf3XtlNoiFwNntMz4\\\\\\\/VjkM+OWtmsa7ZxzVmIFs+WCOUDXAj9OlFiRgNTEm1QymYYOCRGof7sI4pusM8cnL8jYOolxwW\\\\\\\/uesYGTEjwPioK5TApmAgxosAfXrEdMR7\\\\\\\/ICyuwVZatGIdAG\\\\\\\/m8YVbVN\\\\\\\/6rd1kD80V39j4xxjEPyIimNQbgd7W\\\\\\\/HACIPqwt8lZ+60a\\\\\\\/pt303AkmKbACK\\\\\\\/tAAjxTb3ItM3A0qyppidooslLJoBwhv5kVKiC8Waas+NZGrcaR\\\\\\\/nPwdTSXqMn0i++dO32EaI0UPbXl8xaGZU4zsln655jfe9Z4NxaE4ZyV6EbszFEdkLe8U2Kdrq5twtYPGks18y\\\\\\\/GN0r6CqUfIhUdKfm8sR0Rip4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022TlFZP6qpxg0L+eeJj2WB0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222a018c3d14fbca6cf527c8515f70595d7cd59d8022f8333e87c659e529cdeda0\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022iUxiFfYatUYnRXXymL98eiHtVcK5Ty1\\\\\\\/FUV\\\\\\\/pDJTMR79JFBQJCe3g0\\\\\\\/8pYLQppkooEV63OOMjinVQkoDLgKBKAS0JcnK6EzOq4PcHLCpPLN+6oJp8BbZLQMa0bzJJ7Jt4mDHUw0do2SpzvtwIxPUeoYaCL0pnTeo+ZR\\\\\\\/8INhoD7OUECVbsRBrt\\\\\\\/nm2\\\\\\\/as3fJaV1PV6T12Z2dUhlfjlCYo6pLyZPlR7TuQ0wNAgMWdrIR1I\\\\\\\/nRHE9Y7tY25F5TV6vAxeageotgz4amlgvxElJXM5pVfS9jbk9kMLtc56bIjCQGSjhRjoN4tKVk7YEGuIyf9+pkjccUeUTCB2Wa8Ii0fw+l2sSGffmbZyZ100m8H1hPe6NJVzZ0xMm4jL2k+Ekg2y5WNoTgKSVKjCyv3LcQJWD626EnMBYndnpIGWfmHFuIv5Ot3MKmAbzNaeN2gX1QOgg90B6hzbSz1w8LaJR5sqz7AP8BtsnRPf5RrDXZ777vaWI2TWBC5MZx\\\\\\\/9ExOIyqXG\\\\\\\/\\\\\\\/OcXujz1is57P6JbFBOQuBTqIyRW6PhnFHLjqsVOnSgP08MD\\\\\\\/xLz651exadzCGrj0OQ3Uu8KUaGUZerZtGS4sJpbiT7ZlP5wOqmGEWsOk6xxubH8TM5QQLQkkeFA8rt\\\\\\\/TbvzNMF4PeaaG4\\\\\\\/5k9xa3Zyg4W8mqa3DNoA67SMnPMiv438dFKQoWLQxDPzhIbGeSSF83swC8eMjFDhPOALeT0vXFuQJj\\\\\\\/Hy0YVYuQNv7ixZFGb16uVSlO8tQjtJZpEPfFYlq8rSzTOo4VAM4+1OFvTBN921CS063afCUjt+VaJYZu8h1HZqWKgU47yveu3ewAVimmqX1acS\\\\\\\/zAqybPQJcScSMiTajnwRw4wE76yYn45s8uv87i26KjrnT05bN9eAzIH3V84tEdJff+RWn7NJDeJxqDIxSdWICTPcxIBgdtbrc3BsXfaUGVsOTiR4+vo5FFz4B5PXlaKQKNew9dEONsIt+dIIfo0KbvmRjx01jWQb\\\\\\\/nXhO45BpssTcQCofPf+zvu3XLzUgXhnX3mgMpRsu90gn0EDKEDj\\\\\\\/zNk6EV0I+Dt3g1AYH+cQwqoizzMU+xnumrXngG4HLFfMlyOuy\\\\\\\/SpI1BdXK4m4UKhd83cOnLC\\\\\\\/OuaUZu2dJvScXs2bepcuVLZXd42bcn5iQqgGQvMetyHtDeCuul\\\\\\\/rMI3qgWAlYVF+mSN7S8MyyxJqHWiPhR5MfPKuBxzaAzXHLN4uFGB6uF09E5+d1eTmMG1vZUtGcAfrR5vOicg0XZPU+QcCKLimPhLUwLK\\\\\\\/dv3hs8nZp91MwxM363hDYJU1TYQICak7KfU3G1L9NNGiK6DIdrCfJH1YKk\\\\\\\/O5UlgFxIW36NW0dHDYzgFWPyzrzzVsCaxiWyZ49nnWT5j4xxrUFGcC+d8Tlv3QOkyUA3KQZPzSU1tblE5G4NBBt6bItOBNbkXivawQJ4nxG6fA8YyqLfZEpQwy98z1gJUJvhH1eFU5tjzDxUWGLj2hu191bbSrrMf0+DCGmNr4RzW0V1j0slMf0I26Vj76ym0qtGHi+7rzfYBUTd7ZAgNtGdes3sbTiqCSv\\\\\\\/60Vfr3N5A3eYedjkgqDYqEGd0AgKl0rJTN4QocvDKFkumzUwWg2EeZqGfWtqbV0EcnlKpG8gIgrIlShEyFMAQEYu8S6jPIJoy4WJTGuX+cM7ZOdJMWlpJlwy39cB7Hpnk5F+pbQPuTYcSLjzYLvl3FHJ0O1TAotpWQ3I7GN0HjFtR9vju6DvXS\\\\\\\/Xi0ouO8oRJ4Ytku0IMwWCTd+BWrzuNmGeu44ItxCcKrBosL0y0fR9k6XolYl9uda\\\\\\\/yh+fnQLTPGlHUhXbri3EwiOGknXLDVygmvko4WxwA3vZNI1PxRuGzEU\\\\\\\/3WzdkwWDSS25\\\\\\\/o0P+niFNjJQTiIbu2epDaJCnprs3oBp+H4CCFZynhkTq7+mkPNyXV8nv1jD4W3mWXZCSuLl9BuQCWtpoGtqpkkhlaqS0EIettFHpVSg3NhffU\\\\\\\/5fF87\\\\\\\/8wOZNFZ96Fh9vVb8=\\\u0022,\\\u0022iv\\\u0022:\\\u0022MMaf9s4dR4f4i9fU8oGrpA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c9ddb1d1aacfa96ce40531ee7413007cd4753ce5e1a3d4b04baefe710064bc7f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-495846 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1077" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-maC--ZYFS_288p6fuBSgHaA1B2WRBZ7PfPfkFdiHnZ0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1077" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--zn-rNFpZU_hAXVYll11DZlEU3Ns9g-L_xJqFy0jX-Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/1063" title="Selective Sweeps Under Dominance and Inbreeding" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/1087" title="Genetic Basis of Increased Lifespan and Postponed Senescence in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495846" id="node495846" data-pisa="ggg;10/3/1077" data-pisa-master="ggg;g3.119.401023" data-apath="/ggg/10/3/1077.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in <em>Arabidopsis thaliana</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4238-7317" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhiqiang Wu</span>, <span class="highwire-citation-author" data-delta="1">Gus Waneka</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3618-0897" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel B. Sloan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1077-1086;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.401023 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495846" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhiqiang Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, CO 80523 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Guangdong Laboratory of Lingnan Modern Agriculture, Genome Analysis Laboratory of the Ministry of Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518124, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiqiang%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhiqiang%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4238-7317" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhiqiang Wu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gus Waneka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, CO 80523 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gus%2BWaneka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waneka%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGus%2BWaneka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel B. Sloan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, CO 80523 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BB.%2BSloan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sloan%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BB.%2BSloan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3618-0897" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel B. Sloan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbsloan@rams.colostate.edu" class="" data-icon-position="" data-hide-link-title="0">dbsloan@rams.colostate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/10/3/1077" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:495846" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:495846/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/1077.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:495846" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:495846/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/10/3/1077.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:495846" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:495846/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/10/3/1077.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:495846" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:495846/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The mechanisms of sequence divergence in angiosperm mitochondrial genomes have long been enigmatic. In particular, it is difficult to reconcile the rapid divergence of intergenic regions that can make non-coding sequences almost unrecognizable even among close relatives with the unusually high levels of sequence conservation found in genic regions. It has been hypothesized that different mutation and repair mechanisms act on genic and intergenic sequences or alternatively that mutational input is relatively constant but that selection has strikingly different effects on these respective regions. To test these alternative possibilities, we analyzed mtDNA divergence within <em>Arabidopsis thaliana</em>, including variants from the 1001 Genomes Project and changes accrued in published mutation accumulation (MA) lines. We found that base-substitution frequencies are relatively similar for intergenic regions and synonymous sites in coding regions, whereas indel and nonsynonymous substitutions rates are greatly depressed in coding regions, supporting a conventional model in which mutation/repair mechanisms are consistent throughout the genome but differentially filtered by selection. Most types of sequence and structural changes were undetectable in 10-generation MA lines, but we found significant shifts in relative copy number across mtDNA regions for lines grown under stressed <em>vs.</em> benign conditions. We confirmed quantitative variation in copy number across the <em>A. thaliana</em> mitogenome using both whole-genome sequencing and droplet digital PCR, further undermining the classic but oversimplified model of a circular angiosperm mtDNA structure. Our results suggest that copy number variation is one of the most fluid features of angiosperm mitochondrial genomes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/copy-number-variation" class="hw-term hw-article-keyword hw-article-keyword-copy-number-variation" rel="nofollow">copy number variation</a></li><li class="kwd"><a href="/keyword/mutation-accumulation-line" class="hw-term hw-article-keyword hw-article-keyword-mutation-accumulation-line" rel="nofollow">mutation accumulation line</a></li><li class="kwd"><a href="/keyword/mutation-rate" class="hw-term hw-article-keyword hw-article-keyword-mutation-rate" rel="nofollow">mutation rate</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphisms-0" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphisms" rel="nofollow">single-nucleotide polymorphisms</a></li></ul><p id="p-4">The evolution of angiosperm mitochondrial genomes (mitogenomes) is a study in contrasts. On one hand, they exhibit exceptionally low nucleotide substitution rates, including at synonymous sites even though such sites are likely subject to relatively low levels of functional constraint (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wolfe <em>et al.</em> 1987</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Drouin <em>et al.</em> 2008</a>). These low levels of sequence divergence are generally assumed to reflect unusually slow point mutation rates, especially when compared to high mitochondrial mutation rates in many other eukaryotic lineages (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Brown <em>et al.</em> 1979</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sloan <em>et al.</em> 2017</a>). However, direct measures of plant mitochondrial mutation rates are generally lacking, and the mechanisms that maintain such low levels of nucleotide substitutions are not known.</p><p id="p-5">On the other hand, angiosperm mitogenomes are remarkably diverse at a structural level (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mower <em>et al.</em> 2012b</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gualberto and Newton 2017</a>). They are large and variable in size and subject to extensive rearrangements via recombination-mediated mechanisms, which may be accelerated under conditions of plant stress (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Arrieta-Montiel and Mackenzie 2011</a>). Although they typically map as circular structures, their actual physical form appears to be far more complex and variable (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bendich 1993</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Sloan 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kozik <em>et al.</em> 2019</a>).</p><p id="p-6">Comparisons among angiosperm mitochondrial genomes often find that large fractions of intergenic sequence are unalignable between species and seemingly unique to individual lineages (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kubo and Newton 2008</a>). In the most extreme cases, only about half of intergenic sequence content may be shared even between two different mitochondrial haplotypes from the same species (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Sloan <em>et al.</em> 2012</a>). There are likely at least two mechanisms responsible for this phenomenon. First, angiosperm mitogenomes are frequent recipients of large quantities of horizontally transferred DNA from the plastid genome, nucleus, and other sources (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ellis 1982</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Goremykin <em>et al.</em> 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rice <em>et al.</em> 2013</a>). As such, many intergenic sequences are recently acquired and lack homologous sequences in mitogenomes of other angiosperms. It is unlikely, however, that horizontal transfer can provide a full explanation because a lot of intergenic content cannot be traced to any potential donor source. A second possible mechanism is that rates of sequence and structural evolution are so fast in the intergenic regions of angiosperm mitogenomes that homologous sequences can become essentially unrecognizable even among closely related species. But this latter explanation presents a paradox when juxtaposed with the observation that genic regions in plant mitogenomes can exhibit some of the slowest known rates of nucleotide substitutions.</p><p id="p-7"><a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Christensen (2013</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">2014</a>) has proposed alternative models to explain the striking contrast in evolutionary rates between genic and intergenic regions in angiosperm mitogenomes, which are based either on differences in mutational input or differences in selection between these two types of regions. Under the mutational-input model, the contrasting rates of divergence would reflect systematic differences between genic and intergenic sequences with respect to DNA polymerase errors during replication, exposure to DNA damage, and/or the efficacy of DNA repair processes. It was hypothesized that transcription-coupled repair (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hanawalt and Spivak 2008</a>) could have such an effect in altering mutation rates in expressed <em>vs.</em> non-expressed regions in angiosperm mitogenomes (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Christensen 2013</a>), but subsequent analysis of substitution rates in transcribed non-coding regions did not find support for this hypothesis (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Christensen 2014</a>). Nevertheless, the possibility of systematic differences in mutational input among regions within plant mitochondrial genomes remains largely untested, and it has been observed that some species can exhibit substantial rate variation even from one gene to the next for reasons that remain unclear (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zhu <em>et al.</em> 2014</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Warren <em>et al.</em> 2016</a>).</p><p id="p-8">An alternative and perhaps more conventional model is that mutational input is relatively consistent across the genome but that genic <em>vs.</em> intergenic regions are subject to very different selection pressures. For example, structural and sequence variation introduced by error prone repair pathways may be filtered out in gene regions but largely neutral and tolerated in non-coding regions (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Christensen 2014</a>). This may be especially true for any repair mechanisms that lead to structural rearrangements or indels that would truncate protein-coding genes. One prediction from this model is that rates of single-nucleotide substitutions in intergenic regions should largely match those at relatively neutral sites in protein-coding sequences (<em>e.g.</em>, synonymous sites). However, this prediction has been difficult to test because finding sets of genomes that have enough divergence in coding regions to estimate substitution rates and still retain enough similarity in intergenic structure and content to align these non-coding regions is a challenge.</p><p id="p-9">In this sense, variation at an intraspecific scale may be informative, as comparisons between patterns of recent and long-term evolutionary change can be powerful in separating effects of mutation and selection (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nielsen 2005</a>). A previous pairwise comparison between two different <em>Arabidopsis thaliana</em> accessions was used to measure mitochondrial sequence divergence, but this analysis only identified a single synonymous nucleotide substitution in protein-coding genes and thus could offer little precision in quantifying the frequency of single nucleotide polymorphisms (SNPs) in different functional sequence categories (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Christensen 2014</a>). The study was further complicated by the large number of sequencing errors that were later identified in the early <em>A. thaliana</em> mitogenome reference sequences (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sloan <em>et al.</em> 2018</a>).</p><p id="p-10">Here, we take advantage of the ever-growing amount of genomic resources in <em>A. thaliana</em>, including the sequencing of complete genomes from the 1001 Genomes Project (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Alonso-Blanco <em>et al.</em> 2016</a>) and from mutation accumulation (MA) lines in this species (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Jiang <em>et al.</em> 2014</a>), to generate more robust polymorphism datasets for investigating the mechanisms of mitogenome divergence. Our goal is to distinguish among alternative explanations for the contrasting rates of genic <em>vs.</em> intergenic sequence evolution and identify the genomic changes that accrue most rapidly during angiosperm mitogenome evolution.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Identification of intraspecific mitogenome variation from the Arabidopsis 1001 Genomes Project</h3><p id="p-11">To analyze standing mitochondrial polymorphisms within <em>A. thaliana</em>, raw Illumina reads from the 1001 Genomes Project (which actually contains 1135 sequenced individuals; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Alonso-Blanco <em>et al.</em> 2016</a>) were downloaded from the NCBI Sequence Read Archive (SRA) under the project accession SRP056687 using the fastq-dump tool in the NCBI SRA Toolkit v2.9.6. For larger datasets, only the first 20 million read pairs were downloaded. Illumina adapter sequences were trimmed with Cutadapt v2.1 (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Martin 2011</a>), applying a q20 quality cutoff, a 15% error rate for matching adapter sequences, and a minimum trimmed read length of 50 bp. As such, 88 of the 1135 sequenced individuals were excluded entirely from the analysis because their original read lengths were shorter than 50 bp. Trimmed reads were mapped to the <em>A. thaliana</em> Col-0 GenBank RefSeq accessions for the mitochondrial (<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=NC_037304.1&amp;atom=%2Fggg%2F10%2F3%2F1077.atom">NC_037304.1</a>) and plastid genomes (NC_000932.1) using Bowtie v2.3.5 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Langmead and Salzberg 2012</a>). By competitively mapping sequence reads against both organelle genomes, we avoided erroneously mapping plastid-derived reads to related regions in the mitogenome resulting from historical plastid-to-mitochondrial DNA transfers (<em>i.e.</em>, <em>mtpts</em>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Ellis 1982</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sloan and Wu 2014</a>). The resulting alignment files were sorted with SAMtools v1.9 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2009</a>), and variants were called using the HaplotypeCaller tool in GATK v4.1.0.0 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mckenna <em>et al.</em> 2010</a>) with ploidy level set to 1 after removing duplicate reads with the GATK MarkDuplicates tool. Coverage depth at each position in the mitogenome was calculated with the SAMtools depth function. The resulting variant sets were filtered to require a minimum site-specific coverage depth of 50. Variants were also excluded if their coverage was less than half or more than three times the median genome-wide coverage. These thresholds were applied to avoid erroneously identifying variants based on low-frequency sequences such as nuclear insertions (<em>i.e.</em>, <em>numts</em>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stupar <em>et al.</em> 2001</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hazkani-Covo <em>et al.</em> 2010</a>) or based on mis-mapping to repeats within the genome.</p><p id="p-12">To distinguish between ancestral and derived alleles that are segregating within <em>A. thaliana</em>, we aligned the <em>A. thaliana</em> reference genome against the <em>Brassica napus</em> mitogenome (NC_008285.1), using NCBI BLASTN v2.2.30+, applying a minimum alignment length of 400 bp and a minimum nucleotide identity of 90%. The <em>B. napus</em> allele for all alignable <em>A. thaliana</em> SNP positions was extracted from the BLAST output with a custom BioPerl script (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Stajich <em>et al.</em> 2002</a>), which is available via GitHub (<a href="https://github.com/dbsloan/polymorphism_athal_mtdna">https://github.com/dbsloan/polymorphism_athal_mtdna</a>). An alternative approach to distinguish between ancestral and derived alleles is based on the fact that derived alleles are typically at low frequency. As such, even when it is not possible to polarize a variant with an outgroup because it is found in an unalignable region, reasonable predictions of ancestral <em>vs.</em> derived state can still be based on current allele frequencies. Therefore, we calculated allele frequencies at each variable site to identify the minor allele, using all samples within the 1001 Genomes Project that met our coverage requirements for variant calling (see above).</p><p id="p-13">Positions within the <em>A. thaliana</em> reference mitogenome were partitioned into functional categories (protein-coding, rRNA, tRNA, introns, pseudogenes, and intergenic) based on the RefSeq annotation (NC_037304.1). PAML v4.9a was used to approximate the total number of synonymous and nonsynonymous ‘sites’ within protein-coding sequence (accounting for the partial degeneracy at some positions owing to two- and three-member codon families).</p></div><div id="sec-3" class="subsection"><h3>Analysis of mitogenome divergence in Arabidopsis mutation accumulation lines</h3><p id="p-14">To analyze short-term divergence in <em>A. thaliana</em> mitogenomes, we obtained raw Illumina reads from the MA lines generated by <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Jiang <em>et al.</em> (2014)</a> from NCBI SRA (SRP045804). MA lines involve bottlenecking each generation through single-seed descent to limit selection on organismal fitness and obtain a relatively unfiltered view of <em>de novo</em> mutation accumulation (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Halligan and Keightley 2009</a>). This dataset consisted of a total of six MA lines, each propagated for 10 generations. Three lines were propagated under benign growing conditions, while the other three were subjected to salt stress each generation, except in the final generation in which all lines were grown under the same benign conditions. Three biological replicates from each of the six lines were sequenced in the original study (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Jiang <em>et al.</em> 2014</a>).</p><p id="p-15">To test for <em>de novo</em> nucleotide substitutions and indels in the mitogenomes of these MA lines, we applied the same variant calling pipeline as described above for the 1001 Genomes samples. The only modification was that we set the ploidy level to 10 so that we could potentially detect any novel variants that were heteroplasmic at a frequency of ∼10% or greater. There are many causes that can lead to erroneous identification of <em>de novo</em> mitochondrial variants, including mapping artifacts, <em>numts</em>, and heteroplasmies inherited from the original parent. To avoid such errors, we focused on variants that were unique to one or more replicates from a single MA line. For all such variants predicted by our pipeline, we manually inspected read alignments using IGV (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Robinson <em>et al.</em> 2017</a>) to determine whether they were detectable in samples from other MA lines.</p><p id="p-16">We analyzed copy number variation across the <em>A. thaliana</em> mitogenome by normalizing site-specific data for depth of sequence coverage as counts per million mapped read (CPMM) values and averaging them into non-overlapping windows of 500 bp. To avoid any effects of cross-mapping from plastid-derived reads, which are highly abundant in total-cellular DNA samples, we excluded any windows that overlapped with previously identified <em>mtpts</em> (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Sloan and Wu 2014</a>). We also excluded the first and last windows because of potential bias in mapping at the edges where the circular mitogenome map was arbitrarily cut into a linear sequence. To try to account for coverage bias introduced during the sequencing process because of differences in local nucleotide composition (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aird <em>et al.</em> 2011</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Van Dijk <em>et al.</em> 2014</a>), we fit these data to a linear model that included GC content and a count of homopolymers of greater than 7 bp in length as independent variables to predict CPMM in each window. This model was implemented in R v3.6.0 using the lm function. The subsequent analyses of copy number variation described below were performed with both the raw CPMM values and the residuals from this model.</p><p id="p-17">To test for associations in coverage values between adjacent windows across the mitogenome, we performed a Wald–Wolfowitz runs test, using the runs.test function in the R randtests package. To test for significant divergence in coverage values among the MA lines, we fit a model with treatment (salt-stressed <em>vs.</em> control) as a fixed effect and MA line as a nested random effect. This test was implemented in R with the lmer function and the lme4 and lmerTest R packages. We controlled for multiple comparisons by applying a false discovery rate (FDR) correction (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Benjamini and Hochberg 1995</a>).</p><p id="p-18">We also examined the frequency of alternative genome conformations associated with recombination between small repeats by first mapping Illumina reads to the <em>A. thaliana</em> Col-0 reference mitogenome with BWA v0.7.12, using the mem command and the -U 0 option. We then used a custom Perl script (<a href="https://github.com/dbsloan/polymorphism_athal_mtdna">https://github.com/dbsloan/polymorphism_athal_mtdna</a>) to parse the resulting alignment file. For each pair of repeats in the mitogenome, this script calculated the number of read pairs that mapped in a concordant fashion spanning a repeat as well as the number of read pairs that mapped discordantly but in locations that were consistent with a recombination event between the pair of repeats. This analysis was performed on all repeat pairs between 100 and 500 bp in length with a minimum of 80% nucleotide sequence identity. We then tested whether the frequency of recombinant conformations for each repeat pair differed significantly among MA lines by once again fitting a model with treatment as a fixed effect and MA line as a nested random effect (see coverage analysis described above).</p></div><div id="sec-4" class="subsection"><h3>Mitochondrial DNA purification and Illumina sequencing</h3><p id="p-19">Three full-sib families from our <em>A. thaliana</em> Col-0 lab stock were grown in a growth chamber under short-day conditions (10 h of light at 100 μmole m<sup>-2</sup> s<sup>-1</sup>) at 23°. For each family, 30-40g of rosette tissue was harvested from plants after 6-7 weeks of growth. To reduce starch content, plants were kept in the dark for two days prior to collecting leaf tissue, and then the harvested tissue was stored overnight in the dark at a 4°. All subsequent tissue-processing and DNA-extraction steps were carried out in a 4° cold room or refrigerated centrifuge unless stated otherwise.</p><p id="p-20">Leaf tissue was disrupted in high salt isolation buffer (1.25 M NaCl, 50 mM Tris-HCl pH 8.0, 5 mM EDTA, 0.5% polyvinylpyrrolidone, 0.2% bovine serum albumin, 15 mM β-mercaptoethanol), using 10 ml of buffer per g of tissue. Disruption was performed with a standard kitchen blender and a series of five bursts of ∼10 s each with ∼10 s of settling time between each burst, followed by filtration through four layers of cheesecloth and one layer of Miracloth. Filtrates were then centrifuged at 150 rcf for 15 min. The resulting supernatant was transferred to new bottles and centrifuged at 1500 rcf for 20 min. The supernatant was then again transferred to new bottles and centrifuged at 15,000 rcf for 20 min. After discarding the resulting supernatant, the mitochondrial pellets, were gently but thoroughly resuspended in 3 ml of DNase buffer (0.35 M sorbitol, 50 mM Tris-HCl pH 8.0, 15 mM MgCl<sub>2</sub>) with a paintbrush. Then 7 ml of DNase solution (DNase I dissolved in DNase buffer at a concentration of 1 mg/ml) was added to each resuspended pellet. The samples were incubated on ice for 1 h with occasional gentle swirling to digest contaminating plastid and nuclear DNA. Three volumes of wash buffer (0.35 M sorbitol, 50 mM Tris-HCl pH 8.0, 25 mM EDTA) was added to each sample followed by centrifugation at 12,000 rcf for 20 min. The resulting pellets were washed two more times by resuspending in 20 ml wash buffer and centrifuging at 12,000 rcf for 20 min. The final washed pellet was resuspended in 1 ml wash buffer. One-twentieth volume of a 20 mg/ml proteinase K solution was added and incubated at room temperature for 30 min. Mitochondria were lysed by adding one-fifth volume of lysis buffer (5% N-lauryl sarcosine Na salt; 50 mM Tris-HCl pH 8.0, 25 mM EDTA) followed by gentle mixing by inversion for 10 min at room temperature. One volume of phenol:chloroform:isoamyl alcohol (25:24:1) was added followed by vortexing for 5 s and centrifugation at 12,000 rcf for 10 min. The resulting aqueous phase was transferred to a new tube and incubated with 4 µl of a 10 mg/ml RNase A solution. The samples were then treated with two rounds of cleanup with phenol:chloroform:isoamyl alcohol as described above followed by precipitation with one volume of ice-cold isopropanol and incubation for at least 20 min at -20°. Precipitated DNA was pelleted by centrifugation at 12,000 rcf for 10 min and washed twice with 500 µl of ice-cold 70% ethanol. The final DNA pellet was air dried and dissolved in TE buffer (10 mM Tris-HCl pH 8.0, 1 mM EDTA).</p><p id="p-21">Sequencing libraries were produced for each of the three resulting mtDNA samples, using the NEBNext Ultra II FS DNA Library Prep Kit. We used 50 ng of input DNA, with a 15 min fragmentation step, and 5 cycles of PCR amplification. The resulting libraries had an average insert size of approximately 245 bp and were sequenced on a NovaSeq 6000 platform (2×150 bp), producing between 14.1M and 15.4M read pairs per library. The reads were used for coverage-depth analysis by mapping to the <em>A. thaliana</em> reference mitogenome as described above for the MA-line dataset.</p></div><div id="sec-5" class="subsection"><h3>ddPCR copy number analysis</h3><p id="p-22">To confirm variation in copy number that was inferred from deep sequencing data across the mitogenome, we performed droplet digital PCR (ddPCR). Primers were designed to target six regions with high sequencing coverage and six regions with low coverage (Table S1). Analysis, was performed on the same three purified mtDNA samples described above and one sample of total-cellular DNA extracted from the same <em>A. thaliana</em> Col-0 lab line, using a modified CTAB and phenol:chloroform protocol (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Doyle and Doyle 1987</a>). The template quantity for each reaction was either 2 pg of mtDNA or 400 pg of total-cellular DNA, with two technical replicates for each reaction. All ddPCR amplifications were set up in 20-μL volumes with Bio-Rad QX200 ddPCR EvaGreen Supermix and a 2 μM concentration of each primer before mixing into an oil emulsion with a Bio-Rad QX200 Droplet Generator. Amplification was performed on a Bio-Rad C1000 Touch Thermal Cycler with an initial 5 min incubation at 95° and 40 cycles of 30 s at 95° and 1 min at 60°, followed by signal stabilization via 5 min at 4° and 5 min at 95°. The resulting droplets were read on a Bio-Rad QX200 Droplet Reader. Copy numbers for each PCR target were calculated based on a Poisson distribution using the Bio-Rad QuantaSoft package. To assess significant difference in copy-number between the sets of primers from high- and low-coverage regions of the mitogenome, one-tailed <em>t</em>-tests were performed for each of the four DNA samples, using the means for each pair of technical replicates.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-23">All newly generated and previously published sequence data are available via NCBI SRA. Newly generated Illumina data were deposited under accession PRJNA546277. Custom scripts used in data analysis are available via GitHub (<a href="https://github.com/dbsloan/polymorphism_athal_mtdna">https://github.com/dbsloan/polymorphism_athal_mtdna</a>). Data pertaining to identified sequence variants and copy-number variation are provided in supplementary Figures S1-S4 and Tables S1-S4 available at figshare: <a href="https://doi.org/10.25387/g3.11469987">https://doi.org/10.25387/g3.11469987</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Intraspecific mitochondrial sequence variation in the Arabidopsis thaliana 1001 Genomes Project</h3><p id="p-24">Using whole-genome resequencing data from the 1001 Genomes Project, we identified a total of 1105 mitochondrial SNPs that are variable across <em>A. thaliana</em> accessions, including three sites at which three different alleles were detected (Table S2). For a subset of 319 of these sites, we could infer the ancestral state by aligning the nucleotide position to the outgroup <em>Brassica napus</em>. We could also infer the polarity of changes for the entirety of the dataset by assuming that the minor allele represented the derived state. This allele-frequency method produced the same call for 87% of the 319 <em>Brassica</em>-polarized SNPs, suggesting that it had substantial predictive value. Both of these approaches revealed a mutation spectrum that is heavily biased toward increasing AT content. Substitutions that increased AT content were sevenfold more common than those that decreased it based on the <em>Brassica</em>-polarized dataset and fivefold more common in the full dataset based on allele frequency (Table S2). The spectrum did not exhibit the large overrepresentation of transitions that is found in mtDNA of some eukaryotes (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yang and Yoder 1999</a>), with an overall transition:transversion ratio of 422:686 that was only modestly above the null expectation of 1:2 (Table S2). However, AT→TA and GC→CG transversions were rare, representing only 7% and 10% of all transversions, respectively (Table S2). This mutation spectrum is generally consistent with observations from a published pairwise comparison between the <em>A. thaliana</em> Col-0 and C24 ecotypes (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Christensen 2013</a>). The extreme AT bias is also consistent with a previous analysis of inserted plastid sequences (<em>mtpts</em>) as relatively neutral markers in angiosperm mtDNA (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Sloan and Wu 2014</a>). Although that study found that angiosperm mitogenomes generally had weak AT bias, it identified <em>A. thaliana</em> as an outlier with a much stronger bias than most species. Therefore, the inferred mitochondrial mutation spectrum from <em>A. thaliana</em> may not be broadly representative of angiosperms with respect to AT bias.</p><p id="p-25">By comparing the distribution of SNPs across different functional classes within the mitogenome, we found that the presence of base-substitutions is 2.9-fold lower in protein-coding and RNA genes than in intergenic regions (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). However, if only synonymous SNPs in protein-coding genes are considered, the SNP abundance is much more similar but remains slightly lower in genes (0.0027 per synonymous site) than in intergenic regions (0.0034 per site). The average minor allele frequency was also slightly lower for synonymous SNPs (0.016) than for SNPs in intergenic regions (0.026).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/495993/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/495993/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>Variant statistics for 1001 Genomes dataset. SNPs: single nucleotide polymorphisms; MAF: minor allele frequency.</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-26">In contrast to the relatively similar SNP levels between synonymous sites and intergenic regions, there was a radical difference in the distribution of indels across functional classes in the <em>A. thaliana</em> mitogenome. A total of 190 polymorphic indels were identified in the 1001 Genomes dataset, and every one of them was located in either an intergenic region or an intron (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Overall, within gene sequences, we found a large reduction of variants that are expected to be disruptive of gene function (<em>i.e.</em>, nonsynonymous substitutions and indels) but limited evidence of reduced abundance of changes that are likely to be relatively neutral (<em>i.e.</em>, synonymous substitutions).</p></div><div id="sec-9" class="subsection"><h3>Shifts in mitochondrial copy-number variation across mutation accumulation lines</h3><p id="p-27">By analyzing mitochondrial reads from published whole-genome resequencing data of <em>A. thaliana</em> MA lines (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Jiang <em>et al.</em> 2014</a>), we found that most potential mitogenome changes were undetectable over a timescale of 10 generations, regardless of whether the lines had been propagated under salt-stressed or benign conditions. We did not detect any SNPs or indels that reached homoplasmy in individual lines (similar to another recent MA study in <em>A. thaliana;</em> <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Wynn <em>et al.</em> 2019</a>). Our pipeline identified a total of 11 low-frequency variants (seven SNPs, two indels, and two multinucleotide variants with multiple changes clustered at nearby sites) that were unique to a single MA line and thus candidates for <em>de novo</em> mutations. However, manual inspection of read alignments found evidence of these same variants at low frequencies in other lines, indicating that they were unlikely to be true <em>de novo</em> mutations. Therefore, we did not find any convincing evidence of novel substitutions or small indels present in the heteroplasmic state. Angiosperm mitogenomes are known to undergo frequent, homogenizing recombination between large repeat sequences and lower frequency recombination between short repeat sequences (&lt;500 bp), which can lead to shifts in the relative frequency of alternative structures (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Small <em>et al.</em> 1987</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Lonsdale <em>et al.</em> 1988</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Arrieta-Montiel <em>et al.</em> 2009</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Gualberto and Newton 2017</a>). To test for such structural changes, we quantified the frequency of recombinant conformations using read-pairs spanning short repeat sequences. Although we identified minor variation in frequencies of alternative conformation across sequenced lines (Table S3), none of these showed consistent patterns of divergence for either treatment or line effects at an FDR of 0.05.</p><p id="p-28">In mapping MA line reads to the <em>A. thaliana</em> reference mitogenome, we observed variation in coverage across the length of the genome, which was broadly similar in the six different MA lines (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Because Illumina DNA sequencing (and the PCR-based techniques it relies on) can be biased against sequences with extreme GC or AT richness or with low-complexity features like homopolymers (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Aird <em>et al.</em> 2011</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Van Dijk <em>et al.</em> 2014</a>), it is possible that the observed coverage variation was an artifact of amplification/sequencing bias. To investigate this possibility, we fit a model to predict sequencing depth based on GC content and presence of homopolymers. This effort was only able to explain a low percentage of the variance in sequencing depth across the mitogenome (<em>R</em><sup>2</sup> &lt; 0.3 for all datasets), and the general pattern of copy number variation was retained after controlling for this effect (Figure S1), suggesting that bias associated with simple nucleotide-composition features was not the primary cause of the observed variation. For subsequent analyses of coverage depths, we also used the residuals from these models to account for sequencing bias related to nucleotide composition.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequencing coverage variation across mitogenome of Arabidopsis thaliana mutation accumulation lines. Each panel represents an average of three biological replicates. Red vertical lines at the bottom of the figure represent the two pairs of large, identical repeats in the A. thaliana mitogenome. When each Illumina read is mapped to these repeats, bowtie2 randomly assigns the read to one copy, so coverage estimates are not expected to be elevated in these regions. The blue dashed line indicates mean coverage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-457397329" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sequencing coverage variation across mitogenome of &lt;em&gt;Arabidopsis thaliana&lt;/em&gt; mutation accumulation lines. Each panel represents an average of three biological replicates. Red vertical lines at the bottom of the figure represent the two pairs of large, identical repeats in the &lt;em&gt;A. thaliana&lt;/em&gt; mitogenome. When each Illumina read is mapped to these repeats, bowtie2 randomly assigns the read to one copy, so coverage estimates are not expected to be elevated in these regions. The blue dashed line indicates mean coverage.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1077/F1.medium.gif" width="440" height="327"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/10/3/1077/F1.medium.gif" width="440" height="327"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495921" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-29" class="first-child">Sequencing coverage variation across mitogenome of <em>Arabidopsis thaliana</em> mutation accumulation lines. Each panel represents an average of three biological replicates. Red vertical lines at the bottom of the figure represent the two pairs of large, identical repeats in the <em>A. thaliana</em> mitogenome. When each Illumina read is mapped to these repeats, bowtie2 randomly assigns the read to one copy, so coverage estimates are not expected to be elevated in these regions. The blue dashed line indicates mean coverage.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">To assess whether there were any significant shifts in copy-number variation during propagation of MA lines, we scanned the length of the genome in 500-bp windows to test for effects at the level of treatment (salt-stressed <em>vs.</em> control) and individual MA lines. We found that many of the 713 windows in the mitogenome showed small but significant differences between treatments after an FDR correction for multiple comparisons (35 windows when using raw CPMM values and 14 when using residuals from a nucleotide composition model; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figures 2</a> and S2; Table S4). None of the windows were significant for an MA-line effect after the same FDR correction, where line was tested as a nested effect within treatment (Table S4). Adjacent regions tended to show coverage differences in the same direction relative to the genome-wide median (Wald–Wolfowitz runs test; only 201 observed cases in which adjacent windows were on opposite sides of the median compared to a null expectation of 356; <em>P</em> &lt;&lt; 0.001). Therefore, we found evidence that MA lines shifted in consistent ways with respect to region-specific copy number when subjected to stressed <em>vs.</em> benign growing conditions over 10 generations. Although the effect sizes were modest (up to a 20.5% shift in coverage in stressed <em>vs.</em> control samples), they could still be detected with a relatively small sample size because of the consistent patterns across replicate lines.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Divergence in region-specific mitogenome copy number in salt-stressed vs. control mutation accumulation lines. Values are expressed as a ratio of the averages for all salt-stressed and all control lines. Windows that deviate significantly from a ratio of 1 after false-discovery-rate correction are highlighted in red. CPMM: counts per million mapped reads." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-457397329" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Divergence in region-specific mitogenome copy number in salt-stressed &lt;em&gt;vs.&lt;/em&gt; control mutation accumulation lines. Values are expressed as a ratio of the averages for all salt-stressed and all control lines. Windows that deviate significantly from a ratio of 1 after false-discovery-rate correction are highlighted in red. CPMM: counts per million mapped reads.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1077/F2.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/10/3/1077/F2.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495995" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Divergence in region-specific mitogenome copy number in salt-stressed <em>vs.</em> control mutation accumulation lines. Values are expressed as a ratio of the averages for all salt-stressed and all control lines. Windows that deviate significantly from a ratio of 1 after false-discovery-rate correction are highlighted in red. CPMM: counts per million mapped reads.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Sequencing and ddPCR analysis of purified Arabidopsis thaliana mtDNA</h3><p id="p-32">To further test for evidence of copy number variation within the <em>A. thaliana</em> mitogenome, we purified mtDNA from replicate families of our own lab line of the Col-0 ecotype. Illumina sequencing of these samples resulted in 67–68% of reads mapping to the <em>A. thaliana</em> reference mitogenome for each biological replicate, demonstrating substantial enrichment for mtDNA. An additional 6–8% and 1–3% of reads could be mapped to the <em>A. thaliana</em> plastid and nuclear genomes, indicating some contamination from other genomic compartments. The remaining unmapped reads were dominated by known plant-associated bacteria (<em>e.g.</em>, <em>Pseudomonas</em> and <em>Enterobacter</em>), which appear to have been co-enriched in our mitochondrial isolations. As found with the MA lines, this analysis revealed a heterogeneous pattern of coverage across the mitogenome, which was generally consistent among the three replicates (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3</a> and S3). Once again, we found that adjacent regions tended to show coverage variation in the same direction (Wald–Wolfowitz runs test; only 120 observed cases in which adjacent windows were on opposite sides of the median compared to a null expectation of 356; <em>P</em> &lt;&lt; 0.001). However, comparing between our samples and the MA lines found only a modest correlation in copy number variation (<em>r</em> &lt; 0.25; Figure S4).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sequencing coverage variation across the mitogenome for three purified mtDNA samples from Arabidopsis thaliana. The windows chosen for development of ddPCR markers are shown in red and blue dots (high- and low-coverage regions, respectively). Red vertical lines at the bottom of the figure represent the two pairs of large, identical repeats in the A. thaliana mitogenome. When each Illumina read is mapped to these repeats, bowtie2 randomly assigns the read to one copy, so coverage estimates are not expected to be elevated in these regions. The blue dashed line indicates mean coverage." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-457397329" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Sequencing coverage variation across the mitogenome for three purified mtDNA samples from &lt;em&gt;Arabidopsis thaliana&lt;/em&gt;. The windows chosen for development of ddPCR markers are shown in red and blue dots (high- and low-coverage regions, respectively). Red vertical lines at the bottom of the figure represent the two pairs of large, identical repeats in the &lt;em&gt;A. thaliana&lt;/em&gt; mitogenome. When each Illumina read is mapped to these repeats, bowtie2 randomly assigns the read to one copy, so coverage estimates are not expected to be elevated in these regions. The blue dashed line indicates mean coverage.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1077/F3.medium.gif" width="440" height="161"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/10/3/1077/F3.medium.gif" width="440" height="161"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495975" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Sequencing coverage variation across the mitogenome for three purified mtDNA samples from <em>Arabidopsis thaliana</em>. The windows chosen for development of ddPCR markers are shown in red and blue dots (high- and low-coverage regions, respectively). Red vertical lines at the bottom of the figure represent the two pairs of large, identical repeats in the <em>A. thaliana</em> mitogenome. When each Illumina read is mapped to these repeats, bowtie2 randomly assigns the read to one copy, so coverage estimates are not expected to be elevated in these regions. The blue dashed line indicates mean coverage.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">To confirm that the observed heterogeneity in coverage was a product of true variation in copy number rather than an artifact of sequencing bias, we performed ddPCR with two sets of six markers that were selected for either high-coverage or low-coverage regions based on sequencing data (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Unlike sequencing and traditional qPCR, this method is generally insensitive to variation in PCR efficiency or amplification bias because it is based on endpoint PCR (40 cycles) within each ‘micro-reactor’ droplet. We found significant differences in copy number between the sets of high- and low-coverage markers for both the purified mtDNA samples that were used in sequencing and a total-cellular DNA extraction (<em>P</em> &lt; 0.001 for each of the three purified mtDNA samples and <em>P</em> = 0.011 for the total-cellular DNA sample; <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). In all cases, the average difference in copy number between these sets was somewhat smaller (between 17.1% and 20.3% for the purified mtDNA samples and 10.7% for the total-cellular sample) than from sequence estimates (mean of 36.3%), which may reflect some regression to the mean because the high- and low-copy markers were chosen only based on being in the extreme tails of the sequencing-coverage distribution rather than for an <em>a priori</em> reason.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ddPCR comparison of copy number for mitogenome regions identified as either high-copy or low-copy by sequencing analysis. Copy numbers are expressed as per &#x3BC;l of ddPCR reaction volume. Input for the mtDNA samples was diluted 200-fold relative to the total-cellular sample." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-457397329" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;ddPCR comparison of copy number for mitogenome regions identified as either high-copy or low-copy by sequencing analysis. Copy numbers are expressed as per &#x3BC;l of ddPCR reaction volume. Input for the mtDNA samples was diluted 200-fold relative to the total-cellular sample.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/10/3/1077/F4.medium.gif" width="440" height="226"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/10/3/1077/F4.medium.gif" width="440" height="226"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/10/3/1077/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/495956" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">ddPCR comparison of copy number for mitogenome regions identified as either high-copy or low-copy by sequencing analysis. Copy numbers are expressed as per μl of ddPCR reaction volume. Input for the mtDNA samples was diluted 200-fold relative to the total-cellular sample.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><div id="sec-12" class="subsection"><h3>Contrasting rates of evolution in genic and intergenic regions of angiosperm mitogenomes</h3><p id="p-36">Our analysis confirmed dramatic differences in rates of mitogenome structural evolution between genic and intergenic regions at an intraspecific level within <em>A. thaliana</em>, mirroring the extensive observations of this phenomenon based on divergence between angiosperm species (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kubo and Newton 2008</a>). By dramatically expanding the number of sampled accessions with the aid of the 1001 Genomes dataset (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Alonso-Blanco <em>et al.</em> 2016</a>), we were also able to make quantitative comparisons between nucleotide substitution rates in these regions, which was previously difficult because of the limited number of substitutions in an earlier comparison between two <em>A. thaliana</em> accessions (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Christensen 2013</a>). The similar levels of nucleotide substitutions between intergenic regions and synonymous sites in protein-coding genes (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) suggests that mutational input in different functional regions is comparable. As such, the most likely explanation for the divergent evolutionary rates in genic and intergenic regions is a conventional model, under which selection has varying effects in filtering mutations in different region throughout the genome (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Christensen 2014</a>).</p><p id="p-37">Despite the rough similarity between nucleotide substitution rates at synonymous sites and in intergenic regions, we still found that the synonymous rate was slightly lower (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). There are multiple possible explanations for this gap. First, it is possible synonymous substitution rates are suppressed because these sites still experience a larger degree of purifying selection than intergenic regions. For example, even if they do not change amino acid sequences, synonymous substitutions can disrupt the translation efficiency, secondary structure, or binding motifs of mRNAs (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chamary <em>et al.</em> 2006</a>). Indeed, there is evidence for some weak purifying selection acting on synonymous sites in angiosperm mitogenomes (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sloan and Taylor 2010</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Wynn and Christensen 2015</a>). Selection on multinucleotide mutations may also affect observed synonymous substitution rates. There is a growing appreciation that clustered substitutions at adjacent sites can occur in a single mutational event (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Schrider <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Harris and Nielsen 2014</a>) and that they can affect inferences of selection (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Venkat <em>et al.</em> 2018</a>). It is very likely that some of the SNPs observed at adjacent sites in our analysis (Table S1) do not represent independent events. When such events occur in protein-coding genes, synonymous mutations may be removed by selection because they are linked to harmful mutations at adjacent nonsynonymous sites, whereas multinucleotide mutations in intergenic regions may remain relatively neutral. There are also mechanisms that may inflate substitution rate estimates in intergenic regions. For example, these regions often contain short, non-identical repeats that can undergo rare recombination events and create rearrangements (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Arrieta-Montiel <em>et al.</em> 2009</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Gualberto and Newton 2017</a>). Such recombination events can give the false impression that conventional nucleotide substitutions occurred because they create chimeric versions of similar but non-identical sequences.</p><p id="p-38">Regardless of the causes of the small observed gap between substitution rates at synonymous sites <em>vs.</em> intergenic regions, it is clear that the magnitude of this difference is trivial relative to the wildly different rates of overall divergence observed between genes and the rest of the mitogenome in angiosperms. Indeed, it may simply reflect sampling variance as the small difference between intergenic regions and synonymous sites (0.0034 <em>vs.</em> 0.0027) is not even statistically significant (χ<sup>2</sup> = 0.8; <em>P</em> = 0.37). While it is possible that certain mutational mechanisms preferentially act in intergenic regions and make them mutation ‘hotspots’, we favor an explanation based on strong selection on gene function, with region-specific mutation rates playing, at best, a secondary role in <em>A. thaliana</em> mitogenomes.</p></div><div id="sec-13" class="subsection"><h3>Uneven copy number across angiosperm mitogenomes and implications for models of genome structure</h3><p id="p-39">Our analysis of a published sequencing dataset from MA lines (<a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Jiang <em>et al.</em> 2014</a>) and newly sequenced samples of purified mtDNA found evidence that coverage across the mitogenome is not constant and that it can show detectable levels of divergence across MA lines. Patterns of coverage variation were largely continuous (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">3</a>), which contrasts with other commonly studied forms of copy number variation, in which germ-line segmental duplications or losses result in discrete shifts in coverage for specific regions of the genome (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Conrad <em>et al.</em> 2010</a>). Our findings are relevant to previous work in the mitogenome of <em>Mimulus guttatus</em>, in which alternative recombination-mediated conformations showed evidence of heterogenous coverage, even in some cases where they were predicted to be part of the same subgenomic molecules (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mower <em>et al.</em> 2012a</a>). In addition, it has been shown that, disruption of specific nuclear genes involved in mitogenome replication, recombination, and repair can lead to preferential amplification or loss of certain genomic regions (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Shedge <em>et al.</em> 2007</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wallet <em>et al.</em> 2015</a>), and recent evidence indicates that mitogenome copy number can change in gene-specific ways across development in <em>Cucumis melo</em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Shen <em>et al.</em> 2019</a>).</p><p id="p-40">Other analyses of intraspecific mitogenome variation in systems such as <em>A. thaliana</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Davila <em>et al.</em> 2011</a>), <em>Beta vulgaris</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Darracq <em>et al.</em> 2011</a>), and <em>Zea mays</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allen <em>et al.</em> 2007</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Darracq <em>et al.</em> 2010</a>) have generally focused on structural rearrangements resulting from repeat-mediated recombination. Indeed, at an even finer level, angiosperm mitogenomes are really a population of alternative structures that interconvert via recombination and coexist within cells and tissues in a single individual (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Palmer and Shields 1984</a>; <a id="xref-ref-25-4" class="xref-bibr" href="#ref-25">Gualberto and Newton 2017</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kozik <em>et al.</em> 2019</a>). As such, these structural rearrangements are arguably the most dynamic element of plant mtDNA evolution, and rapid shifts in the predominant structure (referred to as substoichiometric shifting) are often observed on very short generational timescales (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdelnoor <em>et al.</em> 2003</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Arrieta-Montiel and Mackenzie 2011</a>). However, when it comes to the MA-line analysis in this study, it is notable that it was copy number variation and not structural rearrangements for which we could detect significant divergence among lines. Therefore, in this case, it appears that copy number variation might be the most rapidly diverging feature of the <em>A. thaliana</em> mitogenome, even though the general pattern of coverage is quite similar across lines (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) and there is known to be a persistent level of recombinational activity that is constantly occurring and interconverting among the population of alternative mitogenome structures. The divergence in copy number among lines did not appear to be entirely random, as we detected significant differences associated with salt-stress treatments, suggesting that the historical environment experienced in recent generations can have an effect in shaping the mitogenome landscape.</p><p id="p-41">When identifying copy number variation among lines, it is important to consider a number of alternative explanations. As described above, we investigated the possibility that PCR or sequencing bias associated with nucleotide composition could explain variation in coverage but found very little explanatory power from such effects. Another possibility is that differences among lines represent heterogeneous sampling, such as different developmental timepoints, as there is evidence of locus-specific mitochondrial copy number variation across development (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Shen <em>et al.</em> 2019</a>). Such differences may contribute to the contrasting patterns of variation between the MA lines of <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Jiang <em>et al.</em> (2014)</a> and our purified mtDNA samples (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figures 1</a>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">3</a>, and S4), as these were grown and sampled at different times and in different labs. It is less likely that developmental differences explain observed divergence between salt-stressed and control MA lines because these were all grown and sampled under common garden conditions in the final generation of the experiment. Nevertheless, we cannot rule out the possibility that lines from different treatments exhibited systematic differences in growth such that sampling in the original MA study effectively represented different developmental stages.</p><p id="p-42">A further assumption made in analyzing copy number variation is that DNA extraction methods representatively sample the entire genome. Although this is likely to be a reasonable assumption in most cases, it is plausible that procedures that rely on mitochondrial isolation may differentially enrich for certain subpopulations of mitochondria that may differ in their genomic content. The low ratio of mitochondrial genome copies to actual mitochondrial organelles in <em>Arabidopsis</em> tissues implies that many mitochondria harbor only partial mitochondrial genomes or no mtDNA at all (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Preuten <em>et al.</em> 2010</a>). This and other characteristics of our mtDNA isolation protocol (<em>e.g.</em>, storage in the dark prior to isolation or use of DNase to remove contaminating nuclear and plastid DNA) may be an additional cause of the contrasting patterns between the MA lines and our purified mtDNA samples. It might also explain why ddPCR found smaller differences between high-copy and low-copy markers in total-cellular DNA than in purified mtDNA samples (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). However, the observed difference between total-cellular DNA and purified mtDNA may also simply reflect another example of regression to the mean, as ddPCR was performed on the exact sample purified mtDNA samples that were sequenced and used to define high-copy and low-copy markers, while the total-cellular sample was an independent extraction from different tissue. Once again, the effects of different DNA extraction methods are unlikely to explain divergence between MA lines because they were all processed with the same total-cellular method (<a id="xref-ref-30-7" class="xref-bibr" href="#ref-30">Jiang <em>et al.</em> 2014</a>).</p><p id="p-43">Angiosperm mitogenome sequencing projects typically report genome assemblies represented as a single circular structure, but it is widely accepted that this is an oversimplification resulting from mapping and that the physical form of angiosperm mtDNA involves complex branching structures (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bendich 1993</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Sloan 2013</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Kozik <em>et al.</em> 2019</a>). These branching structures likely reflect the activity of DNA replication, which is thought to be initiated by recombination-dependent mechanisms and not depend on a single origin of replication (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Cupp and Nielsen 2014</a>). In addition to findings from more direct observations of the physical form of mtDNA molecules (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bendich 1996</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Backert and Borner 2000</a>), coverage patterns in previous sequencing efforts have been interpreted as evidence against a ‘master circle’ as the predominant form of the mitogenome (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Mower <em>et al.</em> 2012a</a>).</p><p id="p-44">By itself, copy number variation is not definitive evidence against a simple circular organization in <em>A. thaliana</em>. Bacterial genomes are circular structures but can still exhibit quantitative variation in coverage across the genome when DNA is sampled from actively dividing cultures, with copy number decreasing from the origin of replication to the terminus of replication. Indeed, analyzing sequencing coverage of bacterial genomes can be an effective way to identify the location of the origin of replication and measure the replication rate of bacteria (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Brown <em>et al.</em> 2016</a>). In addition, it is possible that variation in coverage could reflect differential degradation, either occurring naturally over the course of development (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kumar <em>et al.</em> 2014</a>) or as an artifact of the extraction process. Nevertheless, we contend that the combination of heterogeneous coverage and evidence for shifts in copy number variation among MA lines is unlikely to be explained by a simple circular model with preferential amplification at origin(s) of replication or differential degradation within the circle, especially when viewed in the light of existing evidence against the master circle as a predominant genome form. Instead, our results suggest that the complex physical structure of angiosperm mitogenomes creates opportunities for differential amplification or degradation of subgenomic regions in a dynamic way that does not occur in simpler mitogenomes like those found in bilaterian animals. In addition to the rapid and large changes in the frequencies of mitogenome structural conformations associated with substoichiometric shifting, angiosperms appear to be subject to more pervasive low-level fluctuations in copy numbers of local regions within the genome.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank Jeff Mower for helpful discussion and motivation to use intraspecific variation to analyze contrasting patterns of evolution between genic and intergenic regions in plant mitogenomes. We also thank the Associate Editor and an anonymous reviewer for their insightful comments on an earlier version of this manuscript. This work was supported by a grant from the NIH (R01 GM118046) and an NSF NRT GAUSSI Graduate Fellowship (DGE-1450032).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-46">Supplemental material available at figshare: <a href="https://doi.org/10.25387/g3.11469987">https://doi.org/10.25387/g3.11469987</a>.</p></li><li class="fn" id="fn-3"><p id="p-47"><em>Communicating editor: P. Ingvarsson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-12-27"><span class="received-label">Received </span>December 27, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2020-01-20"><span class="accepted-label">Accepted </span>January 20, 2020.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2020 Wu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.1" data-doi="10.1073/pnas.1037651100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdelnoor</span>,  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Yule</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Elo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Christensen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Meyer-Gauen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Substoichiometric shifting in the plant mitochondrial genome is influenced by a gene homologous to MutS.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">5968</span>–<span class="cit-lpage">5973</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1037651100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1037651100%26rft_id%253Dinfo%253Apmid%252F12730382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzEwLzU5NjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.2" data-doi="10.1186/gb-2011-12-2-r18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aird</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Ross</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. S.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Danielsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analyzing and minimizing PCR amplification bias in Illumina sequencing libraries.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2011-12-2-r18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAird%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253DR18%26rft.epage%253DR18%26rft.atitle%253DAnalyzing%2Band%2Bminimizing%2BPCR%2Bamplification%2Bbias%2Bin%2BIllumina%2Bsequencing%2Blibraries.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-2-r18%26rft_id%253Dinfo%253Apmid%252F21338519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-2-r18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21338519&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.3" data-doi="10.1534/genetics.107.073312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>,  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Fauron</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Minx</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Roark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Oddiraju</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparisons among two fertile and three male-sterile mitochondrial genomes of maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.073312</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.073312%26rft_id%253Dinfo%253Apmid%252F17660568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny8yLzExNzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.4" data-doi="10.1016/j.cell.2016.05.063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Blanco</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Andrade</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Becker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Bemm</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bergelson</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">1,135 genomes reveal the global pattern of polymorphism in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">481</span>–<span class="cit-lpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2016.05.063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D481%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.05.063%26rft_id%253Dinfo%253Apmid%252F27293186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.05.063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27293186&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.1077.5" data-doi="10.1007/978-0-387-89781-3_3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arrieta-Montiel</span>,  <span class="cit-name-given-names">M. P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Mackenzie</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Plant mitochondrial genomes and recombination</span>, pp. <span class="cit-fpage">65</span>–<span class="cit-lpage">82</span> in <span class="cit-source">Plant Mitochondria</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kempken</span>,  <span class="cit-name-given-names">F.</span></span> <span class="cit-publ-name">Springer Verlag</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-89781-3_3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMitochondria%26rft.volume%253D1%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-89781-3_3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-89781-3_3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.6" data-doi="10.1534/genetics.109.108514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arrieta-Montiel</span>,  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Shedge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Davila</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Christensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Mackenzie</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Diversity of the Arabidopsis mitochondrial genome occurs via nuclear-controlled recombination activity.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1261</span>–<span class="cit-lpage">1268</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.108514</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.108514%26rft_id%253Dinfo%253Apmid%252F19822729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzEyNjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.7" data-doi="10.1007/s002940050532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Backert</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Borner</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Phage T4-like intermediates of DNA replication and recombination in the mitochondria of the higher plant Chenopodium album (L.).</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">304</span>–<span class="cit-lpage">314</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s002940050532</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DBackert%26rft.auinit1%253DS.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D304%26rft.epage%253D314%26rft.atitle%253DPhage%2BT4-like%2Bintermediates%2Bof%2BDNA%2Breplication%2Band%2Brecombination%2Bin%2Bthe%2Bmitochondria%2Bof%2Bthe%2Bhigher%2Bplant%2BChenopodium%2Balbum%2B%2528L.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs002940050532%26rft_id%253Dinfo%253Apmid%252F10853767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s002940050532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10853767&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087452300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.8" data-doi="10.1007/BF00336777"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bendich</span>,  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Reaching for the ring: the study of mitochondrial genome structure.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">290</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00336777</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DBendich%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D279%26rft.epage%253D290%26rft.atitle%253DReaching%2Bfor%2Bthe%2Bring%253A%2Bthe%2Bstudy%2Bof%2Bmitochondrial%2Bgenome%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00336777%26rft_id%253Dinfo%253Apmid%252F8252636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00336777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8252636&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LZ27900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.9" data-doi="10.1006/jmbi.1996.0048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bendich</span>,  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Structural analysis of mitochondrial DNA molecules from fungi and plants using moving pictures and pulsed-field gel electrophoresis.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">255</span>: <span class="cit-fpage">564</span>–<span class="cit-lpage">588</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jmbi.1996.0048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBendich%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D255%26rft.issue%253D4%26rft.spage%253D564%26rft.epage%253D588%26rft.atitle%253DStructural%2Banalysis%2Bof%2Bmitochondrial%2BDNA%2Bmolecules%2Bfrom%2Bfungi%2Band%2Bplants%2Busing%2Bmoving%2Bpictures%2Band%2Bpulsed-field%2Bgel%2Belectrophoresis.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1996.0048%26rft_id%253Dinfo%253Apmid%252F8568898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1996.0048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8568898&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TQ97500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.10" data-doi="10.1093/biostatistics/kxj037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>,  <span class="cit-name-given-names">Y.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Hochberg</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiostatistics%252Fkxj037%26rft_id%253Dinfo%253Apmid%252F16632515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/biostatistics/kxj037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16632515&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.11" data-doi="10.1038/nbt.3704"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Olm</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. C.</span> <span class="cit-name-surname">Thomas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F.</span> <span class="cit-name-surname">Banfield</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Measurement of bacterial replication rates in microbial communities.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1256</span>–<span class="cit-lpage">1263</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3704</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D1256%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3704&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.12" data-doi="10.1073/pnas.76.4.1967"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>,  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">George</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Wilson</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Rapid evolution of animal mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">1967</span>–<span class="cit-lpage">1971</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.76.4.1967</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBrown%26rft.auinit1%253DW.%2BM.%26rft.volume%253D76%26rft.issue%253D4%26rft.spage%253D1967%26rft.epage%253D1971%26rft.atitle%253DRapid%2Bevolution%2Bof%2Banimal%2Bmitochondrial%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.76.4.1967%26rft_id%253Dinfo%253Apmid%252F109836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ni80LzE5NjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.13" data-doi="10.1038/nrg1770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chamary</span>,  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Parmley</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. D.</span> <span class="cit-name-surname">Hurst</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Hearing silence: non-neutral evolution at synonymous sites in mammals.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">98</span>–<span class="cit-lpage">108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1770</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChamary%26rft.auinit1%253DJ.%2BV.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253D98%26rft.epage%253D108%26rft.atitle%253DHearing%2Bsilence%253A%2Bnon-neutral%2Bevolution%2Bat%2Bsynonymous%2Bsites%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1770%26rft_id%253Dinfo%253Apmid%252F16418745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16418745&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234714000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.14" data-doi="10.1093/gbe/evt069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>,  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Plant mitochondrial genome evolution can be explained by DNA repair mechanisms.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1079</span>–<span class="cit-lpage">1086</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evt069</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevt069%26rft_id%253Dinfo%253Apmid%252F23645599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evt069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23645599&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.15" data-doi="10.1093/gbe/evu115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>,  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genes and junk in plant mitochondria-repair mechanisms and selection.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1448</span>–<span class="cit-lpage">1453</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evu115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu115%26rft_id%253Dinfo%253Apmid%252F24904012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evu115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24904012&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.16" data-doi="10.1038/nature08516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>,  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Pinto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Redon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Feuk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Gokcumen</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Origins and functional impact of copy number variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">704</span>–<span class="cit-lpage">712</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08516</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D704%26rft.epage%253D712%26rft.atitle%253DOrigins%2Band%2Bfunctional%2Bimpact%2Bof%2Bcopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08516%26rft_id%253Dinfo%253Apmid%252F19812545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812545&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.17" data-doi="10.1016/j.mito.2014.03.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cupp</span>,  <span class="cit-name-given-names">J. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. L.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">DNA replication in plant mitochondria.</span> <abbr class="cit-jnl-abbrev">Mitochondrion</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">237</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mito.2014.03.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMitochondrion%26rft.volume%253D19%26rft.spage%253D231%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mito.2014.03.008%26rft_id%253Dinfo%253Apmid%252F24681310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mito.2014.03.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24681310&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.18" data-doi="10.1093/gbe/evr042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darracq</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Varre</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Marechal-Drouard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Courseaux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Castric</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Structural and content diversity of mitochondrial genome in beet: a comparative genomic analysis.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">736</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evr042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevr042%26rft_id%253Dinfo%253Apmid%252F21602571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evr042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21602571&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.19" data-doi="10.1186/1471-2164-11-233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darracq</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. S.</span> <span class="cit-name-surname">Varre</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Touzet</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A scenario of mitochondrial genome evolution in maize based on rearrangement events.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">233</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-11-233</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DDarracq%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.spage%253D233%26rft.epage%253D233%26rft.atitle%253DA%2Bscenario%2Bof%2Bmitochondrial%2Bgenome%2Bevolution%2Bin%2Bmaize%2Bbased%2Bon%2Brearrangement%2Bevents.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-233%26rft_id%253Dinfo%253Apmid%252F20380689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20380689&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.20" data-doi="10.1186/1741-7007-9-64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davila</span>,  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Arrieta-Montiel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wamboldt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hagmann</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Double-strand break repair processes drive evolution of the mitochondrial genome in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1741-7007-9-64</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DDavila%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D9%26rft.spage%253D64%26rft.epage%253D64%26rft.atitle%253DDouble-strand%2Bbreak%2Brepair%2Bprocesses%2Bdrive%2Bevolution%2Bof%2Bthe%2Bmitochondrial%2Bgenome%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-9-64%26rft_id%253Dinfo%253Apmid%252F21951689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-9-64&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21951689&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.21" data-doi="10.2307/4119796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>,  <span class="cit-name-given-names">J. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Doyle</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">A rapid DNA isolation procedure for small quantities of fresh leaf tissue.</span> <abbr class="cit-jnl-abbrev">Phytochem. Bull.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytochem.%2BBull.%26rft.volume%253D19%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.2307%252F4119796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/4119796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.22" data-doi="10.1016/j.ympev.2008.09.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drouin</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Daoud</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Xia</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Relative rates of synonymous substitutions in the mitochondrial, chloroplast and nuclear genomes of seed plants.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">827</span>–<span class="cit-lpage">831</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ympev.2008.09.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BPhylogenetics%2Band%2BEvolution%2B%2528Online%2529%26rft.stitle%253DMolecular%2BPhylogenetics%2Band%2BEvolution%2B%2528Online%2529%26rft.aulast%253DDrouin%26rft.auinit1%253DG.%26rft.volume%253D49%26rft.issue%253D3%26rft.spage%253D827%26rft.epage%253D831%26rft.atitle%253DRelative%2Brates%2Bof%2Bsynonymous%2Bsubstitutions%2Bin%2Bthe%2Bmitochondrial%252C%2Bchloroplast%2Band%2Bnuclear%2Bgenomes%2Bof%2Bseed%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2008.09.009%26rft_id%253Dinfo%253Apmid%252F18838124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2008.09.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18838124&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261647900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.23" data-doi="10.1038/299678a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Promiscuous DNA–chloroplast genes inside plant mitochondria.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">299</span>: <span class="cit-fpage">678</span>–<span class="cit-lpage">679</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/299678a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DEllis%26rft.auinit1%253DJ.%26rft.volume%253D299%26rft.issue%253D5885%26rft.spage%253D678%26rft.epage%253D679%26rft.atitle%253DPromiscuous%2BDNA--chloroplast%2Bgenes%2Binside%2Bplant%2Bmitochondria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F299678a0%26rft_id%253Dinfo%253Apmid%252F7121600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/299678a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7121600&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982PL84900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.24" data-doi="10.1111/j.1365-313X.2012.05014.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goremykin</span>,  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Lockhart</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Viola</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Velasco</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The mitochondrial genome of Malus domestica and the import-driven hypothesis of mitochondrial genome expansion in seed plants.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">626</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.2012.05014.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DGoremykin%26rft.auinit1%253DV.%2BV.%26rft.volume%253D71%26rft.issue%253D4%26rft.spage%253D615%26rft.epage%253D626%26rft.atitle%253DThe%2Bmitochondrial%2Bgenome%2Bof%2BMalus%2Bdomestica%2Band%2Bthe%2Bimport-driven%2Bhypothesis%2Bof%2Bmitochondrial%2Bgenome%2Bexpansion%2Bin%2Bseed%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2012.05014.x%26rft_id%253Dinfo%253Apmid%252F22469001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2012.05014.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22469001&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307169600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.25" data-doi="10.1146/annurev-arplant-043015-112232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gualberto</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Newton</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Plant mitochondrial genomes: dynamics and mechanisms of mutation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-arplant-043015-112232</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft.volume%253D68%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-043015-112232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-043015-112232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.26" data-doi="10.1146/annurev.ecolsys.39.110707.173437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>,  <span class="cit-name-given-names">D. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. D.</span> <span class="cit-name-surname">Keightley</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Spontaneous mutation accumulation studies in evolutionary genetics.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.ecolsys.39.110707.173437</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D40%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.27" data-doi="10.1038/nrm2549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanawalt</span>,  <span class="cit-name-given-names">P. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Spivak</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Transcription-coupled DNA repair: two decades of progress and surprises.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">958</span>–<span class="cit-lpage">970</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrm2549</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DHanawalt%26rft.auinit1%253DP.%2BC.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D958%26rft.epage%253D970%26rft.atitle%253DTranscription-coupled%2BDNA%2Brepair%253A%2Btwo%2Bdecades%2Bof%2Bprogress%2Band%2Bsurprises.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2549%26rft_id%253Dinfo%253Apmid%252F19023283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19023283&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261126800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.28" data-doi="10.1101/gr.170696.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">K.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Error-prone polymerase activity causes multinucleotide mutations in humans.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1445</span>–<span class="cit-lpage">1454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.170696.113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.170696.113%26rft_id%253Dinfo%253Apmid%252F25079859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzkvMTQ0NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTA3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.29" data-doi="10.1371/journal.pgen.1000834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hazkani-Covo</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Zeller</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Martin</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular poltergeists: mitochondrial DNA copies (numts) in sequenced nuclear genomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e1000834</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000834</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHazkani-Covo%26rft.auinit1%253DE.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De1000834%26rft.epage%253De1000834%26rft.atitle%253DMolecular%2Bpoltergeists%253A%2Bmitochondrial%2BDNA%2Bcopies%2B%2528numts%2529%2Bin%2Bsequenced%2Bnuclear%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000834%26rft_id%253Dinfo%253Apmid%252F20168995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20168995&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.30" data-doi="10.1101/gr.177659.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Mithani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. J.</span> <span class="cit-name-surname">Belfield</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Mott</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. D.</span> <span class="cit-name-surname">Hurst</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Environmentally responsive genome-wide accumulation of de novo Arabidopsis thaliana mutations and epimutations.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1821</span>–<span class="cit-lpage">1829</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.177659.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.177659.114%26rft_id%253Dinfo%253Apmid%252F25314969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMS8xODIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.31" data-doi="10.1371/journal.pgen.1008373"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Rowan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lavelle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Berke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Schranz</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">The alternative reality of plant mitochondrial DNA: One ring does not rule them all.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-elocation-id">e1008373</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1008373</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D15%26rft.spage%253De1008373%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1008373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1008373&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.32" data-doi="10.1016/j.mito.2007.10.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kubo</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. J.</span> <span class="cit-name-surname">Newton</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Angiosperm mitochondrial genomes and mutations.</span> <abbr class="cit-jnl-abbrev">Mitochondrion</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mito.2007.10.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMitochondrion%26rft.aulast%253DKubo%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D14%26rft.atitle%253DAngiosperm%2Bmitochondrial%2Bgenomes%2Band%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mito.2007.10.006%26rft_id%253Dinfo%253Apmid%252F18065297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mito.2007.10.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18065297&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252902200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.33" data-doi="10.1093/jxb/eru359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>,  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Oldenburg</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Bendich</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Changes in DNA damage, molecular integrity, and copy number for plastid DNA and mitochondrial DNA during maize development.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">6425</span>–<span class="cit-lpage">6439</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/eru359</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Feru359%26rft_id%253Dinfo%253Apmid%252F25261192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/eru359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25261192&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000348154300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.34" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.35" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.36" data-doi="10.1098/rstb.1988.0039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lonsdale</span>,  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Brears</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. P.</span> <span class="cit-name-surname">Hodge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Melville</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. H.</span> <span class="cit-name-surname">Rottmann</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The plant mitochondrial genome: homologous recombination as a mechanism for generating heterogeneity.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">163</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.1988.0039</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.1988.0039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.1988.0039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.37" data-doi="10.14806/ej.17.1.200"><div class="cit-metadata"><cite>Martin, M., 2011 Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal 17<strong>:</strong> 10–12.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.14806/ej.17.1.200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCUTADAPT%2BREMOVES%2BADAPTER%2BSEQUENCES%2BFROM%2BHIGHTHROUGHPUT%2BSEQUENCING%2BREADS%26rft.volume%253D17%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.14806%252Fej.17.1.200%26rft_id%253Dinfo%253Apmid%252F25452271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.14806/ej.17.1.200&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25452271&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.38" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hanna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sivachenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Cibulskis</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTA3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.39" data-doi="10.1093/gbe/evs042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mower</span>,  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Case</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. R.</span> <span class="cit-name-surname">Floro</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Willis</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Evidence against equimolarity of large repeat arrangements and a predominant master circle structure of the mitochondrial genome from a monkeyflower (Mimulus guttatus) lineage with cryptic CMS.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">670</span>–<span class="cit-lpage">686</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evs042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevs042%26rft_id%253Dinfo%253Apmid%252F22534162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evs042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22534162&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-10.3.1077.40" data-doi="10.1007/978-3-7091-1130-7_9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mower</span>,  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Sloan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Alverson</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Plant mitochondrial diversity – the genomics revolution</span>, pp. <span class="cit-fpage">123</span>–<span class="cit-lpage">144</span> in <span class="cit-source">Plant Genome Diversity</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Wendel</span>,  <span class="cit-name-given-names">J. F.</span></span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Vienna</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-7091-1130-7_9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%2BDiversity%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-7091-1130-7_9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-7091-1130-7_9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.41" data-doi="10.1146/annurev.genet.39.073003.112420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Molecular signatures of natural selection.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.39.073003.112420</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.volume%253D39%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.39.073003.112420%26rft_id%253Dinfo%253Apmid%252F16285858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.39.073003.112420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16285858&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234685200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.42" data-doi="10.1038/307437a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>,  <span class="cit-name-given-names">J. D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Shields</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Tripartite structure of the Brassica campestris mitochondrial genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">307</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/307437a0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D307%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.1038%252F307437a0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/307437a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.43" data-doi="10.1111/j.1365-313X.2010.04389.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preuten</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Cincu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fuchs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Zoschke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Liere</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fewer genes than organelles: extremely low and variable gene copy numbers in mitochondria of somatic plant cells.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">948</span>–<span class="cit-lpage">959</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.2010.04389.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DPreuten%26rft.auinit1%253DT.%26rft.volume%253D64%26rft.issue%253D6%26rft.spage%253D948%26rft.epage%253D959%26rft.atitle%253DFewer%2Bgenes%2Bthan%2Borganelles%253A%2Bextremely%2Blow%2Band%2Bvariable%2Bgene%2Bcopy%2Bnumbers%2Bin%2Bmitochondria%2Bof%2Bsomatic%2Bplant%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2010.04389.x%26rft_id%253Dinfo%253Apmid%252F21143676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2010.04389.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21143676&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285207900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.44" data-doi="10.1126/science.1246275"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>,  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Alverson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. O.</span> <span class="cit-name-surname">Richardson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Young</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Sanchez-Puerta</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Horizontal transfer of entire genomes via mitochondrial fusion in the angiosperm Amborella.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">1468</span>–<span class="cit-lpage">1473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1246275</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRice%26rft.auinit1%253DD.%2BW.%26rft.volume%253D342%26rft.issue%253D6165%26rft.spage%253D1468%26rft.epage%253D1473%26rft.atitle%253DHorizontal%2BTransfer%2Bof%2BEntire%2BGenomes%2Bvia%2BMitochondrial%2BFusion%2Bin%2Bthe%2BAngiosperm%2BAmborella%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1246275%26rft_id%253Dinfo%253Apmid%252F24357311%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDIvNjE2NS8xNDY4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.45" data-doi="10.1158/0008-5472.CAN-17-0337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>,  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Thorvaldsdóttir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Wenger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Zehir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Mesirov</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variant review with the integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">e31</span>–<span class="cit-lpage">e34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1158/0008-5472.CAN-17-0337</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BResearch%26rft.stitle%253DCancer%2BRes.%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D77%26rft.issue%253D21%26rft.spage%253De31%26rft.epage%253De34%26rft.atitle%253DVariant%2BReview%2Bwith%2Bthe%2BIntegrative%2BGenomics%2BViewer%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-17-0337%26rft_id%253Dinfo%253Apmid%252F29092934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjk6Ijc3LzIxL2UzMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTA3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.46" data-doi="10.1016/j.cub.2011.05.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>,  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. N.</span> <span class="cit-name-surname">Hourmozdi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Pervasive multinucleotide mutational events in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1051</span>–<span class="cit-lpage">1054</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2011.05.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSchrider%26rft.auinit1%253DD.%2BR.%26rft.volume%253D21%26rft.issue%253D12%26rft.spage%253D1051%26rft.epage%253D1054%26rft.atitle%253DPervasive%2Bmultinucleotide%2Bmutational%2Bevents%2Bin%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.05.013%26rft_id%253Dinfo%253Apmid%252F21636278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.05.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21636278&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.47" data-doi="10.1105/tpc.106.048355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shedge</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Arrieta-Montiel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Christensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Mackenzie</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Plant mitochondrial recombination surveillance requires unusual RecA and MutS homologs.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1264</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.106.048355</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.106.048355%26rft_id%253Dinfo%253Apmid%252F17468263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE5LzQvMTI1MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTA3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.48" data-doi="10.1038/s41438-019-0177-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Havey</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Shou</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Copy numbers of mitochondrial genes change during melon leaf development and are lower than the numbers of mitochondria.</span> <abbr class="cit-jnl-abbrev">Hortic. Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">95</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41438-019-0177-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortic.%2BRes.%26rft.volume%253D6%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41438-019-0177-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41438-019-0177-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.49" data-doi="10.1111/nph.12395"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sloan</span>,  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">One ring to rule them all? Genome sequencing provides new insights into the ‘master circle’ model of plant mitochondrial DNA structure.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">985</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.12395</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D200%26rft.spage%253D978%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12395%26rft_id%253Dinfo%253Apmid%252F24712049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12395&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24712049&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.50" data-doi="10.1111/mec.13959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sloan</span>,  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Havird</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sharbrough</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The on-again, off-again relationship between mitochondrial genomes and species boundaries.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2212</span>–<span class="cit-lpage">2236</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.13959</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D26%26rft.spage%253D2212%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.51" data-doi="10.1111/j.1469-8137.2012.04340.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sloan</span>,  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Muller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. E.</span> <span class="cit-name-surname">McCauley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Taylor</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Storchova</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Intraspecific variation in mitochondrial genome sequence, structure, and gene content in Silene vulgaris, an angiosperm with pervasive cytoplasmic male sterility.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1228</span>–<span class="cit-lpage">1239</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2012.04340.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSloan%26rft.auinit1%253DD.%2BB.%26rft.volume%253D196%26rft.issue%253D4%26rft.spage%253D1228%26rft.epage%253D1239%26rft.atitle%253DIntraspecific%2Bvariation%2Bin%2Bmitochondrial%2Bgenome%2Bsequence%252C%2Bstructure%252C%2Band%2Bgene%2Bcontent%2Bin%2BSilene%2Bvulgaris%252C%2Ban%2Bangiosperm%2Bwith%2Bpervasive%2Bcytoplasmic%2Bmale%2Bsterility.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2012.04340.x%26rft_id%253Dinfo%253Apmid%252F23009072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2012.04340.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23009072&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310676400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.52" data-doi="10.1007/s00239-010-9346-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sloan</span>,  <span class="cit-name-given-names">D. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Taylor</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Testing for selection on synonymous sites in plant mitochondrial DNA: the role of codon bias and RNA editing.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00239-010-9346-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DSloan%26rft.auinit1%253DD.%2BB.%26rft.volume%253D70%26rft.issue%253D5%26rft.spage%253D479%26rft.epage%253D491%26rft.atitle%253DTesting%2Bfor%2Bselection%2Bon%2Bsynonymous%2Bsites%2Bin%2Bplant%2Bmitochondrial%2BDNA%253A%2Bthe%2Brole%2Bof%2Bcodon%2Bbias%2Band%2BRNA%2Bediting.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-010-9346-y%26rft_id%253Dinfo%253Apmid%252F20424833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-010-9346-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20424833&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278094800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.53" data-doi="10.1093/gbe/evu253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sloan</span>,  <span class="cit-name-given-names">D. B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wu</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">History of plastid DNA insertions reveals weak deletion and at mutation biases in angiosperm mitochondrial genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evu253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu253%26rft_id%253Dinfo%253Apmid%252F25416619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evu253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25416619&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.54" data-doi="10.1105/tpc.18.00024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sloan</span>,  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sharbrough</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Correction of persistent errors in Arabidopsis reference mitochondrial genomes.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.18.00024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.18.00024%26rft_id%253Dinfo%253Apmid%252F29519893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjMwLzMvNTI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.55" data-doi="10.1002/j.1460-2075.1987.tb04832.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Small</span>,  <span class="cit-name-given-names">I. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. G.</span> <span class="cit-name-surname">Isaac</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Leaver</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Stoichiometric differences in DNA molecules containing the atpA gene suggest mechanisms for the generation of mitochondrial genome diversity in maize.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">869</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/j.1460-2075.1987.tb04832.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DSmall%26rft.auinit1%253DI.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253D865%26rft.epage%253D869%26rft.atitle%253DStoichiometric%2Bdifferences%2Bin%2BDNA%2Bmolecules%2Bcontaining%2Bthe%2BatpA%2Bgene%2Bsuggest%2Bmechanisms%2Bfor%2Bthe%2Bgeneration%2Bof%2Bmitochondrial%2Bgenome%2Bdiversity%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fj.1460-2075.1987.tb04832.x%26rft_id%253Dinfo%253Apmid%252F16453759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/j.1460-2075.1987.tb04832.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16453759&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G768700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.56" data-doi="10.1101/gr.361602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Block</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Boulez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Brenner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Chervitz</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Bioperl toolkit: Perl modules for the life sciences.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1611</span>–<span class="cit-lpage">1618</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.361602</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DStajich%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D1611%26rft.epage%253D1618%26rft.atitle%253DThe%2BBioperl%2BToolkit%253A%2BPerl%2BModules%2Bfor%2Bthe%2BLife%2BSciences%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.361602%26rft_id%253Dinfo%253Apmid%252F12368254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMC8xNjExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.57" data-doi="10.1073/pnas.091110398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stupar</span>,  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Lilly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Town</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kaul</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Complex mtDNA constitutes an approximate 620-kb insertion on Arabidopsis thaliana chromosome 2: implication of potential sequencing errors caused by large-unit repeats.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">5099</span>–<span class="cit-lpage">5103</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.091110398</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.091110398%26rft_id%253Dinfo%253Apmid%252F11309509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OC85LzUwOTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8xMC8zLzEwNzcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.58" data-doi="10.1016/j.yexcr.2014.01.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Dijk</span>,  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Jaszczyszyn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Thermes</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Library preparation methods for next-generation sequencing: tone down the bias.</span> <abbr class="cit-jnl-abbrev">Exp. Cell Res.</abbr> <span class="cit-vol">322</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.yexcr.2014.01.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExp.%2BCell%2BRes.%26rft.volume%253D322%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.yexcr.2014.01.008%26rft_id%253Dinfo%253Apmid%252F24440557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.yexcr.2014.01.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24440557&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332055900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.59" data-doi="10.1038/s41559-018-0584-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venkat</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Thornton</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Multinucleotide mutations cause false inferences of lineage-specific positive selection.</span> <abbr class="cit-jnl-abbrev">Nat. Ecol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1280</span>–<span class="cit-lpage">1288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41559-018-0584-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BEcol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41559-018-0584-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41559-018-0584-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.60" data-doi="10.1105/tpc.15.00680"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallet</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Le Ret</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bergdoll</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Bichara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Dietrich</span></span> <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The RECG1 DNA translocase is a key factor in recombination surveillance, repair, and segregation of the mitochondrial DNA in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2907</span>–<span class="cit-lpage">2925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00680%26rft_id%253Dinfo%253Apmid%252F26462909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMC8yOTA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMTAvMy8xMDc3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.61" data-doi="10.1093/gbe/evw003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Simmons</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Wu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Sloan</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Linear plasmids and the rate of sequence evolution in plant mitochondrial genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">364</span>–<span class="cit-lpage">374</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw003%26rft_id%253Dinfo%253Apmid%252F26759362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26759362&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.62" data-doi="10.1073/pnas.84.24.9054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>,  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. H.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Sharp</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Rates of nucleotide substitution vary greatly among plant mitochondrial, chloroplast, and nuclear DNAs.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">9054</span>–<span class="cit-lpage">9058</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.84.24.9054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWolfe%26rft.auinit1%253DK.%2BH.%26rft.volume%253D84%26rft.issue%253D24%26rft.spage%253D9054%26rft.epage%253D9058%26rft.atitle%253DRates%2Bof%2Bnucleotide%2Bsubstitution%2Bvary%2Bgreatly%2Bamong%2Bplant%2Bmitochondrial%252C%2Bchloroplast%252C%2Band%2Bnuclear%2BDNAs.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.84.24.9054%26rft_id%253Dinfo%253Apmid%252F3480529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODQvMjQvOTA1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTA3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.63" data-doi="10.1007/s00239-015-9704-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wynn</span>,  <span class="cit-name-given-names">E. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Christensen</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Are synonymous substitutions in flowering plant mitochondria neutral?</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00239-015-9704-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BEvol.%26rft.volume%253D81%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-015-9704-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-015-9704-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.64" data-doi="10.1101/427500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wynn</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Purfeerst</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Christensen</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Mitochondrial DNA repair in an Arabidopsis thaliana uracil N-glycosylase mutant.</span> <abbr class="cit-jnl-abbrev">bioRxiv 427500v3</abbr>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/427500</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%2B427500v3%26rft_id%253Dinfo%253Adoi%252F10.1101%252F427500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiI0Mjc1MDB2MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzEwLzMvMTA3Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.65" data-doi="10.1007/PL00006470"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">Z.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Yoder</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Estimation of the transition/transversion rate bias and species sampling.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">274</span>–<span class="cit-lpage">283</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/PL00006470</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DYang%26rft.auinit1%253DZ.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D274%26rft.epage%253D283%26rft.atitle%253DEstimation%2Bof%2Bthe%2Btransition%252Ftransversion%2Brate%2Bbias%2Band%2Bspecies%2Bsampling.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006470%26rft_id%253Dinfo%253Apmid%252F10093216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006470&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10093216&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078808700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-10.3.1077.66" data-doi="10.1093/molbev/msu079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Jain</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Mower</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unprecedented heterogeneity in the synonymous substitution rate within a plant genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1228</span>–<span class="cit-lpage">1236</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu079</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu079%26rft_id%253Dinfo%253Apmid%252F24557444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24557444&amp;link_type=MED&amp;atom=%2Fggg%2F10%2F3%2F1077.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335914400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/10/3/1077.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/10/3/1063" title="Selective Sweeps Under Dominance and Inbreeding" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/10/3/1087" title="Genetic Basis of Increased Lifespan and Postponed Senescence in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/10/3"><span>Volume 10 Issue 3, March 2020</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/10/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/10/3.cover-source.jpg?itok=8jjPWDA4" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 10 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/10/3/1077?rss=1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fScKrS4xc98ocDnXXOg2eTCYbJl6yqW_3s3J5Yt045k" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/10/3/1077" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/10/3/1077" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/10/3/1077" class="active" data-icon-position="" data-hide-link-title="0">The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in Arabidopsis thaliana</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/495846" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-iGbE5cxUQUbRmrXkoXRbKRuLWJRbfbhFvmE2eMntf-k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/1077.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/10/3/1077" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-gqyd8kCwG-XSeWG2Hjsfz3osPglxktO_bLsBxyF0PV4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/10/3/1077" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/10/3/1077.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495846" id="node495846--2" data-pisa="ggg;10/3/1077" data-pisa-master="ggg;g3.119.401023" data-apath="/ggg/10/3/1077.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in <em>Arabidopsis thaliana</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4238-7317" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhiqiang Wu</span>, <span class="highwire-citation-author" data-delta="1">Gus Waneka</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3618-0897" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel B. Sloan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1077-1086;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.401023 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495846--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhiqiang Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, CO 80523 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Guangdong Laboratory of Lingnan Modern Agriculture, Genome Analysis Laboratory of the Ministry of Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518124, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiqiang%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhiqiang%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4238-7317" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhiqiang Wu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gus Waneka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, CO 80523 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gus%2BWaneka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waneka%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGus%2BWaneka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel B. Sloan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, CO 80523 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BB.%2BSloan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sloan%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BB.%2BSloan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3618-0897" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel B. Sloan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbsloan@rams.colostate.edu" class="" data-icon-position="" data-hide-link-title="0">dbsloan@rams.colostate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/10/3/1077" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/10/3/1077&amp;title=The%20Tempo%20and%20Mode%20of%20Angiosperm%20Mitochondrial%20Genome%20Divergence%20Inferred%20from%20Intraspecific%20Variation%20in%20Arabidopsis%20thaliana" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/10/3/1077&amp;title=The%20Tempo%20and%20Mode%20of%20Angiosperm%20Mitochondrial%20Genome%20Divergence%20Inferred%20from%20Intraspecific%20Variation%20in%20Arabidopsis%20thaliana" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/10/3/1077&amp;title=The%20Tempo%20and%20Mode%20of%20Angiosperm%20Mitochondrial%20Genome%20Divergence%20Inferred%20from%20Intraspecific%20Variation%20in%20Arabidopsis%20thaliana" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/10/3/1077&amp;text=The%20Tempo%20and%20Mode%20of%20Angiosperm%20Mitochondrial%20Genome%20Divergence%20Inferred%20from%20Intraspecific%20Variation%20in%20Arabidopsis%20thaliana" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/10/3/1077&amp;title=The%20Tempo%20and%20Mode%20of%20Angiosperm%20Mitochondrial%20Genome%20Divergence%20Inferred%20from%20Intraspecific%20Variation%20in%20Arabidopsis%20thaliana" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/10/3/1077&amp;t=The%20Tempo%20and%20Mode%20of%20Angiosperm%20Mitochondrial%20Genome%20Divergence%20Inferred%20from%20Intraspecific%20Variation%20in%20Arabidopsis%20thaliana" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/10/3/1077&amp;title=The%20Tempo%20and%20Mode%20of%20Angiosperm%20Mitochondrial%20Genome%20Divergence%20Inferred%20from%20Intraspecific%20Variation%20in%20Arabidopsis%20thaliana" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/10/3/1077&amp;title=The%20Tempo%20and%20Mode%20of%20Angiosperm%20Mitochondrial%20Genome%20Divergence%20Inferred%20from%20Intraspecific%20Variation%20in%20Arabidopsis%20thaliana" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/495846/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495846" id="node495846--3" data-pisa="ggg;10/3/1077" data-pisa-master="ggg;g3.119.401023" data-apath="/ggg/10/3/1077.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Tempo and Mode of Angiosperm Mitochondrial Genome Divergence Inferred from Intraspecific Variation in <em>Arabidopsis thaliana</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4238-7317" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhiqiang Wu</span>, <span class="highwire-citation-author" data-delta="1">Gus Waneka</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-3618-0897" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel B. Sloan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2020 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 10 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1077-1086;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.401023 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node495846--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zhiqiang Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, CO 80523 and</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Guangdong Laboratory of Lingnan Modern Agriculture, Genome Analysis Laboratory of the Ministry of Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518124, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhiqiang%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhiqiang%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4238-7317" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhiqiang Wu</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gus Waneka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, CO 80523 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gus%2BWaneka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Waneka%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGus%2BWaneka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Daniel B. Sloan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, CO 80523 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BB.%2BSloan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sloan%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BB.%2BSloan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-3618-0897" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel B. Sloan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbsloan@rams.colostate.edu" class="" data-icon-position="" data-hide-link-title="0">dbsloan@rams.colostate.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1077&amp;rft.epage=1086&amp;rft.atitle=The%20Tempo%20and%20Mode%20of%20Angiosperm%20Mitochondrial%20Genome%20Divergence%20Inferred%20from%20Intraspecific%20Variation%20in%20Arabidopsis%20thaliana&amp;rft.volume=10&amp;rft.issue=3&amp;rft.date=2020-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wu%2C+Zhiqiang&amp;rft.au=Waneka%2C+Gus&amp;rft.au=Sloan%2C+Daniel+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/495846/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/495846/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/495846/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/495846/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/495846/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/495846/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/495846/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/495846/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/495846/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/495846/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/495846/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/495846/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/10/3/1077" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/10/3/1077.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/10/3/1077.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/10/3/1077.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
