<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/7/1853" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/7/1853.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/7/1853.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/7/1853.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/7/1853.atom" />
<meta name="HW.pisa" content="ggg;6/7/1853" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Degradation Signals for Ubiquitin-Proteasome Dependent Cytosolic Protein Quality Control (CytoQC) in Yeast" />
<meta name="DC.Identifier" content="10.1534/g3.116.027953" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Maurer et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Cellular protein quality control (PQC) systems selectively target misfolded or otherwise aberrant proteins for degradation by the ubiquitin-proteasome system (UPS). How cells discern abnormal from normal proteins remains incompletely understood, but involves in part the recognition between ubiquitin E3 ligases and degradation signals (degrons) that are exposed in misfolded proteins. PQC is compartmentalized in the cell, and a great deal has been learned in recent years about ER-associated degradation (ERAD) and nuclear quality control. In contrast, a comprehensive view of cytosolic quality control (CytoQC) has yet to emerge, and will benefit from the development of a well-defined set of model substrates. In this study, we generated an isogenic “degron library” in Saccharomyces cerevisiae consisting of short sequences appended to the C-terminus of a reporter protein, [Ura3][1]. About half of these degron-containing proteins are substrates of the integral membrane E3 ligase [Doa10][2], which also plays a pivotal role in ERAD and some nuclear protein degradation. Notably, some of our degron fusion proteins exhibit dependence on the E3 ligase [Ltn1][3]/[Rkr1][3] for degradation, apparently by a mechanism distinct from its known role in ribosomal quality control of translationally paused proteins. [Ubr1][4] and [San1][5], E3 ligases involved in the recognition of some misfolded CytoQC substrates, are largely dispensable for the degradation of our degron-containing proteins. Interestingly, the Hsp70/Hsp40 chaperone/cochaperones [Ssa1][6],2 and [Ydj1][7], are required for the degradation of all constructs tested. Taken together, the comprehensive degron library presented here provides an important resource of isogenic substrates for testing candidate PQC components and identifying new ones.

 [1]: http://www.yeastgenome.org/locus/S000000747/overview
 [2]: http://www.yeastgenome.org/locus/S000001292/overview
 [3]: http://www.yeastgenome.org/locus/S000004861/overview
 [4]: http://www.yeastgenome.org/locus/S000003416/overview
 [5]: http://www.yeastgenome.org/locus/S000002550/overview
 [6]: http://www.yeastgenome.org/locus/S000000004/overview
 [7]: http://www.yeastgenome.org/locus/S000005008/overview" />
<meta name="DC.Contributor" content="Matthew J. Maurer" />
<meta name="DC.Contributor" content="Eric D. Spear" />
<meta name="DC.Contributor" content="Allen T. Yu" />
<meta name="DC.Contributor" content="Evan J. Lee" />
<meta name="DC.Contributor" content="Saba Shahzad" />
<meta name="DC.Contributor" content="Susan Michaelis" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Degradation Signals for Ubiquitin-Proteasome Dependent Cytosolic Protein Quality Control (CytoQC) in Yeast" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Cellular protein quality control (PQC) systems selectively target misfolded or otherwise aberrant proteins for degradation by the ubiquitin-proteasome system (UPS). How cells discern abnormal from normal proteins remains incompletely understood, but involves in part the recognition between ubiquitin E3 ligases and degradation signals (degrons) that are exposed in misfolded proteins. PQC is compartmentalized in the cell, and a great deal has been learned in recent years about ER-associated degradation (ERAD) and nuclear quality control. In contrast, a comprehensive view of cytosolic quality control (CytoQC) has yet to emerge, and will benefit from the development of a well-defined set of model substrates. In this study, we generated an isogenic “degron library” in &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; consisting of short sequences appended to the C-terminus of a reporter protein, Ura3. About half of these degron-containing proteins are substrates of the integral membrane E3 ligase Doa10, which also plays a pivotal role in ERAD and some nuclear protein degradation. Notably, some of our degron fusion proteins exhibit dependence on the E3 ligase Ltn1/Rkr1 for degradation, apparently by a mechanism distinct from its known role in ribosomal quality control of translationally paused proteins. Ubr1 and San1, E3 ligases involved in the recognition of some misfolded CytoQC substrates, are largely dispensable for the degradation of our degron-containing proteins. Interestingly, the Hsp70/Hsp40 chaperone/cochaperones Ssa1,2 and Ydj1, are required for the degradation of all constructs tested. Taken together, the comprehensive degron library presented here provides an important resource of isogenic substrates for testing candidate PQC components and identifying new ones.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/07/01" />
<meta name="citation_mjid" content="ggg;6/7/1853" />
<meta name="citation_id" content="6/7/1853" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/7/1853" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/7/1853.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/7/1853.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/7/1853.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.027953" />
<meta name="citation_pmid" content="27172186" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1853" />
<meta name="citation_lastpage" content="1866" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Matthew J. Maurer" />
<meta name="citation_author_institution" content="Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205" />
<meta name="citation_author" content="Eric D. Spear" />
<meta name="citation_author_institution" content="Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205" />
<meta name="citation_author" content="Allen T. Yu" />
<meta name="citation_author_institution" content="Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205" />
<meta name="citation_author" content="Evan J. Lee" />
<meta name="citation_author_institution" content="Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205" />
<meta name="citation_author" content="Saba Shahzad" />
<meta name="citation_author_institution" content="Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205" />
<meta name="citation_author" content="Susan Michaelis" />
<meta name="citation_author_institution" content="Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205" />
<meta name="citation_author_email" content="michaelis@jhmi.edu" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Sci.;citation_author=T. Ast;citation_author=N. Aviram;citation_author=S. G. Chuartzman;citation_author=M. Schuldiner;citation_title=A cytosolic degradation pathway, prERAD, monitors pre-inserted secretory pathway proteins.;citation_volume=127;citation_year=2014;citation_issue=Pt 14;citation_pmid=24849653;citation_doi=10.1242/jcs.144386" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. E. Balch;citation_author=R. I. Morimoto;citation_author=A. Dillin;citation_author=J. W. Kelly;citation_title=Adapting Proteostasis for Disease Intervention;citation_pages=916-919;citation_volume=319;citation_year=2008;citation_issue=5865;citation_pmid=18276881;citation_doi=10.1126/science.1141448" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=N. W. Bays;citation_author=R. G. Gardner;citation_author=L. P. Seelig;citation_author=C. A. Joazeiro;citation_author=R. Y. Hampton;citation_title=Hrd1p/Der3p is a membrane-anchored ubiquitin ligase required for ER-associated degradation.;citation_pages=24-29;citation_volume=3;citation_year=2001;citation_issue=1;citation_pmid=11146622;citation_doi=10.1038/35050524" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. H. Bengtson;citation_author=C. A. Joazeiro;citation_title=Role of a ribosome-associated E3 ubiquitin ligase in protein quality control.;citation_pages=470-473;citation_volume=467;citation_year=2010;citation_issue=7314;citation_pmid=20835226;citation_doi=10.1038/nature09371" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=O. Brandman;citation_author=J. Stewart-Ornstein;citation_author=D. Wong;citation_author=A. Larson;citation_author=C. C. Williams;citation_title=A ribosome-bound quality control complex triggers degradation of nascent peptides and signals translation stress.;citation_pages=1042-1054;citation_volume=151;citation_year=2012;citation_issue=5;citation_pmid=23178123;citation_doi=10.1016/j.cell.2012.10.044" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=M. A. Braun;citation_author=P. J. Costa;citation_author=E. M. Crisucci;citation_author=K. M. Arndt;citation_title=Identification of Rkr1, a nuclear RING domain protein with functional connections to chromatin modification in Saccharomyces cerevisiae.;citation_volume=27;citation_year=2007;citation_issue=8;citation_pmid=17283062;citation_doi=10.1128/MCB.01947-06" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. Buchberger;citation_author=B. Bukau;citation_author=T. Sommer;citation_title=Protein quality control in the cytosol and the endoplasmic reticulum: brothers in arms.;citation_pages=238-252;citation_volume=40;citation_year=2010;citation_issue=2;citation_pmid=20965419;citation_doi=10.1016/j.molcel.2010.10.001" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. E. Carpenter;citation_author=T. R. Jones;citation_author=M. R. Lamprecht;citation_author=C. Clarke;citation_author=I. H. Kang;citation_title=CellProfiler: image analysis software for identifying and quantifying cell phenotypes.;citation_pages=R100-R100;citation_volume=7;citation_year=2006;citation_issue=10;citation_pmid=17076895;citation_doi=10.1186/gb-2006-7-10-r100" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=B. Chen;citation_author=M. Retzlaff;citation_author=T. Roos;citation_author=J. Frydman;citation_title=Cellular strategies of protein quality control.;citation_volume=3;citation_year=2011;citation_issue=8;citation_pmid=21746797;citation_doi=10.1101/cshperspect.a004374" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Chu;citation_author=N. A. Hong;citation_author=C. A. Masuda;citation_author=B. V. Jenkins;citation_author=K. A. Nelms;citation_title=A mouse forward genetics screen identifies LISTERIN as an E3 ubiquitin ligase involved in neurodegeneration.;citation_volume=106;citation_year=2009;citation_issue=7;citation_pmid=19196968;citation_doi=10.1073/pnas.0812819106" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Deng;citation_author=M. Hochstrasser;citation_title=Spatially regulated ubiquitin ligation by an ER/nuclear membrane ligase.;citation_pages=827-831;citation_volume=443;citation_year=2006;citation_issue=7113;citation_pmid=17051211;citation_doi=10.1038/nature05170" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=R. J. Deshaies;citation_author=C. A. Joazeiro;citation_title=RING domain E3 ubiquitin ligases.;citation_pages=399-434;citation_volume=78;citation_year=2009;citation_pmid=19489725;citation_doi=10.1146/annurev.biochem.78.101807.093809" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=S. Duttler;citation_author=S. Pechmann;citation_author=J. Frydman;citation_title=Principles of cotranslational ubiquitination and quality control at the ribosome.;citation_pages=379-393;citation_volume=50;citation_year=2013;citation_issue=3;citation_pmid=23583075;citation_doi=10.1016/j.molcel.2013.03.010" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=F. Eisele;citation_author=D. H. Wolf;citation_title=Degradation of misfolded protein in the cytoplasm is mediated by the ubiquitin ligase Ubr1.;citation_pages=4143-4146;citation_volume=582;citation_year=2008;citation_issue=30;citation_pmid=19041308;citation_doi=10.1016/j.febslet.2008.11.015" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_title=Ubiquitin-independent proteasomal degradation.;citation_author=J. Erales;citation_author=P. Coffino;citation_pages=216-221;citation_year=2014;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=N. N. Fang;citation_author=A. H. Ng;citation_author=V. Measday;citation_author=T. Mayor;citation_title=Hul5 HECT ubiquitin ligase plays a major role in the ubiquitylation and turnover of cytosolic misfolded proteins.;citation_pages=1344-1352;citation_volume=13;citation_year=2011;citation_issue=11;citation_pmid=21983566;citation_doi=10.1038/ncb2343" />
<meta name="citation_reference" content="citation_journal_title=Nat. Cell Biol.;citation_author=N. N. Fang;citation_author=G. T. Chan;citation_author=M. Zhu;citation_author=S. A. Comyn;citation_author=A. Persaud;citation_title=Rsp5/Nedd4 is the main ubiquitin ligase that targets cytosolic misfolded proteins following heat stress.;citation_pages=1227-1237;citation_volume=16;citation_year=2014;citation_issue=12;citation_pmid=25344756;citation_doi=10.1038/ncb3054" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=D. Finley;citation_title=Recognition and processing of ubiquitin-protein conjugates by the proteasome.;citation_pages=477-513;citation_volume=78;citation_year=2009;citation_pmid=19489727;citation_doi=10.1146/annurev.biochem.78.081507.101607" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Finley;citation_author=H. D. Ulrich;citation_author=T. Sommer;citation_author=P. Kaiser;citation_title=The Ubiquitin-Proteasome System of Saccharomyces cerevisiae;citation_pages=319-360;citation_volume=192;citation_year=2012;citation_issue=2;citation_pmid=23028185;citation_doi=10.1534/genetics.112.140467" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=O. Foresti;citation_author=V. Rodriguez-Vaello;citation_author=C. Funaya;citation_author=P. Carvalho;citation_title=Quality control of inner nuclear membrane proteins by the Asi complex.;citation_volume=346;citation_year=2014;citation_issue=6210;citation_pmid=25236469;citation_doi=10.1126/science.1255638" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cell &amp; developmental biology;citation_journal_abbrev=Semin Cell Dev Biol;citation_author=E. K. Fredrickson;citation_author=R. G. Gardner;citation_title=Selective destruction of abnormal proteins by ubiquitin-mediated protein quality control degradation.;citation_pages=530-537;citation_volume=23;citation_year=2012;citation_issue=5;citation_pmid=22245831;citation_doi=10.1016/j.semcdb.2011.12.006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=E. K. Fredrickson;citation_author=J. C. Rosenbaum;citation_author=M. N. Locke;citation_author=T. I. Milac;citation_author=R. G. Gardner;citation_title=Exposed hydrophobicity is a key determinant of nuclear quality control degradation.;citation_volume=22;citation_year=2011;citation_issue=13;citation_pmid=21551067;citation_doi=10.1091/mbc.E11-03-0256" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=E. K. Fredrickson;citation_author=P. S. Gallagher;citation_author=S. V. Clowes Candadai;citation_author=R. G. Gardner;citation_title=Substrate recognition in nuclear protein quality control degradation is governed by exposed hydrophobicity that correlates with aggregation and insolubility.;citation_volume=288;citation_year=2013;citation_issue=9;citation_pmid=23335508;citation_doi=10.1074/jbc.M112.406710" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=N. Furth;citation_author=O. Gertman;citation_author=A. Shiber;citation_author=O. S. Alfassy;citation_author=I. Cohen;citation_title=Exposure of bipartite hydrophobic signal triggers nuclear quality control of Ndc10 at the endoplasmic reticulum/nuclear envelope.;citation_volume=22;citation_year=2011;citation_issue=24;citation_pmid=21998200;citation_doi=10.1091/mbc.E11-05-0463" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. G. Gardner;citation_author=Z. W. Nelson;citation_author=D. E. Gottschling;citation_title=Degradation-mediated protein quality control in the nucleus.;citation_pages=803-815;citation_volume=120;citation_year=2005;citation_issue=6;citation_pmid=15797381;citation_doi=10.1016/j.cell.2005.01.016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=N. Ghaboosi;citation_author=R. J. Deshaies;citation_title=A conditional yeast E1 mutant blocks the ubiquitin-proteasome pathway and reveals a role for ubiquitin conjugates in targeting Rad23 to the proteasome.;citation_volume=18;citation_year=2007;citation_issue=5;citation_pmid=17360968;citation_doi=10.1091/mbc.E06-10-0965" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=T. Gilon;citation_author=O. Chomsky;citation_author=R. G. Kulka;citation_title=Degradation signals for ubiquitin system proteolysis in Saccharomyces cerevisiae;citation_pages=2759-2766;citation_volume=17;citation_year=1998;citation_issue=10;citation_pmid=9582269;citation_doi=10.1093/emboj/17.10.2759" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=T. Gilon;citation_author=O. Chomsky;citation_author=R. G. Kulka;citation_title=Degradation Signals Recognized by the Ubc6p-Ubc7p Ubiquitin-Conjugating Enzyme Pair;citation_pages=7214-7219;citation_volume=20;citation_year=2000;citation_issue=19;citation_pmid=10982838;citation_doi=10.1128/MCB.20.19.7214-7219.2000" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=C. J. Guerriero;citation_author=K. F. Weiberth;citation_author=J. L. Brodsky;citation_title=Hsp70 targets a cytoplasmic quality control substrate to the San1p ubiquitin ligase.;citation_volume=288;citation_year=2013;citation_issue=25;citation_pmid=23653356;citation_doi=10.1074/jbc.M113.475905" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=G. Habeck;citation_author=F. A. Ebner;citation_author=H. Shimada-Kreft;citation_author=S. G. Kreft;citation_title=The yeast ERAD-C ubiquitin ligase Doa10 recognizes an intramembrane degron.;citation_volume=209;citation_year=2015;citation_issue=2;citation_pmid=25918226;citation_doi=10.1083/jcb.201408088" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. W. Heck;citation_author=S. K. Cheung;citation_author=R. Y. Hampton;citation_title=Cytoplasmic protein quality control degradation mediated by parallel actions of the E3 ubiquitin ligases Ubr1 and San1.;citation_volume=107;citation_year=2010;citation_issue=3;citation_pmid=20080635;citation_doi=10.1073/pnas.0910591107" />
<meta name="citation_reference" content="citation_journal_title=Statistics in medicine;citation_journal_abbrev=Stat Med;citation_author=Y. Hochberg;citation_author=Y. Benjamini;citation_title=More powerful procedures for multiple significance testing.;citation_pages=811-818;citation_volume=9;citation_year=1990;citation_issue=7;citation_pmid=2218183;citation_doi=10.1002/sim.4780090710" />
<meta name="citation_reference" content="citation_author=E.M. Hol;citation_author=W. Scheper;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=G. Huyer;citation_author=W. F. Piluek;citation_author=Z. Fansler;citation_author=S. G. Kreft;citation_author=M. Hochstrasser;citation_title=Distinct machinery is required in Saccharomyces cerevisiae for the endoplasmic reticulum-associated degradation of a multispanning membrane protein and a soluble luminal protein.;citation_volume=279;citation_year=2004;citation_issue=37;citation_pmid=15252059;citation_doi=10.1074/jbc.M402468200" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. Ismail;citation_author=D. T. Ng;citation_title=Have you HRD? Understanding ERAD is DOAble!;citation_pages=237-239;citation_volume=126;citation_year=2006;citation_issue=2;citation_pmid=16873052;citation_doi=10.1016/j.cell.2006.07.001" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. R. Johnson;citation_author=R. Swanson;citation_author=L. Rakhilina;citation_author=M. Hochstrasser;citation_title=Degradation signal masking by heterodimerization of MATalpha2 and MATa1 blocks their mutual destruction by the ubiquitin-proteasome pathway.;citation_pages=217-227;citation_volume=94;citation_year=1998;citation_issue=2;citation_pmid=9695950;citation_doi=10.1016/S0092-8674(00)81421-X" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=D. T. Jones;citation_title=Protein secondary structure prediction based on position-specific scoring matrices.;citation_pages=195-202;citation_volume=292;citation_year=1999;citation_issue=2;citation_pmid=10493868;citation_doi=10.1006/jmbi.1999.3091" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Khmelinskii;citation_author=E. Blaszczak;citation_author=M. Pantazopoulou;citation_author=B. Fischer;citation_author=D. J. Omnus;citation_title=Protein quality control at the inner nuclear membrane.;citation_pages=410-413;citation_volume=516;citation_year=2014;citation_issue=7531;citation_pmid=25519137;citation_doi=10.1038/nature14096" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=F. Khosrow-Khavar;citation_author=N. N. Fang;citation_author=A. H. Ng;citation_author=J. M. Winget;citation_author=S. A. Comyn;citation_title=The Yeast Ubr1 Ubiquitin Ligase Participates in a Prominent Pathway That Targets Cytosolic Thermosensitive Mutants for Degradation;citation_pages=619-628;citation_volume=2;citation_year=2012;citation_issue=5;citation_pmid=22670231;citation_doi=10.1534/g3.111.001933" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=J. Kraut-Cohen;citation_author=E. Afanasieva;citation_author=L. Haim-Vilmovsky;citation_author=B. Slobodin;citation_author=I. Yosef;citation_title=Translation- and SRP-independent mRNA targeting to the endoplasmic reticulum in the yeast Saccharomyces cerevisiae.;citation_volume=24;citation_year=2013;citation_issue=19;citation_pmid=23904265;citation_doi=10.1091/mbc.E13-01-0038" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. G. Kreft;citation_author=L. Wang;citation_author=M. Hochstrasser;citation_title=Membrane topology of the yeast endoplasmic reticulum-localized ubiquitin ligase Doa10 and comparison with its human ortholog TEB4 (MARCH-VI).;citation_volume=281;citation_year=2006;citation_issue=8;citation_pmid=16373356;citation_doi=10.1074/jbc.M512215200" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=M. R. Lamprecht;citation_author=D. M. Sabatini;citation_author=A. E. Carpenter;citation_title=CellProfiler: free, versatile software for automated biological image analysis.;citation_pages=71-75;citation_volume=42;citation_year=2007;citation_issue=1;citation_pmid=17269487;citation_doi=10.2144/000112257" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. J. Lewis;citation_author=H. R. Pelham;citation_title=Inefficient quality control of thermosensitive proteins on the plasma membrane.;citation_pages=e5038-e5038;citation_volume=4;citation_year=2009;citation_issue=4;citation_pmid=19337370;citation_doi=10.1371/journal.pone.0005038" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=C. Liu;citation_author=J. Apodaca;citation_author=L. E. Davis;citation_author=H. Rao;citation_title=Proteasome inhibition in wild-type yeast Saccharomyces cerevisiae cells.;citation_pages=158- 162;citation_volume=42;citation_year=2007;citation_issue=2;citation_pmid=17373478;citation_doi=10.2144/000112389" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Lyumkis;citation_author=S. K. Doamekpor;citation_author=M. H. Bengtson;citation_author=J. W. Lee;citation_author=T. B. Toro;citation_title=Single-particle EM reveals extensive conformational variability of the Ltn1 E3 ligase.;citation_volume=110;citation_year=2013;citation_issue=5;citation_pmid=23319619;citation_doi=10.1073/pnas.1210041110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Lyumkis;citation_author=D. Oliveira dos Passos;citation_author=E. B. Tahara;citation_author=K. Webb;citation_author=E. J. Bennett;citation_title=Structural basis for translational surveillance by the large ribosomal subunit-associated protein quality control complex.;citation_volume=111;citation_year=2014;citation_issue=45;citation_pmid=25349383;citation_doi=10.1073/pnas.1413882111" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. J. McClellan;citation_author=M. D. Scott;citation_author=J. Frydman;citation_title=Folding and quality control of the VHL tumor suppressor proceed through distinct chaperone pathways.;citation_pages=739-748;citation_volume=121;citation_year=2005;citation_issue=5;citation_pmid=15935760;citation_doi=10.1016/j.cell.2005.03.024" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=A. J. McClellan;citation_author=S. Tam;citation_author=D. Kaganovich;citation_author=J. Frydman;citation_title=Protein quality control: chaperones culling corrupt conformations.;citation_pages=736-741;citation_volume=7;citation_year=2005;citation_issue=8;citation_pmid=16056264;citation_doi=10.1038/ncb0805-736" />
<meta name="citation_reference" content="citation_title=Protein quality control in the endoplasmic reticulum and cytosol;citation_author=M. Metzger;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. B. Metzger;citation_author=M. J. Maurer;citation_author=B. M. Dancy;citation_author=S. Michaelis;citation_title=Degradation of a cytosolic protein requires endoplasmic reticulum-associated degradation machinery.;citation_volume=283;citation_year=2008;citation_issue=47;citation_pmid=18812321;citation_doi=10.1074/jbc.M806424200" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Michaelis;citation_author=I. Herskowitz;citation_title=The a-factor pheromone of Saccharomyces cerevisiae is essential for mating.;citation_pages=1309-1318;citation_volume=8;citation_year=1988;citation_issue=3;citation_pmid=3285180;citation_doi=10.1128/MCB.8.3.1309" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. Nakatsukasa;citation_author=G. Huyer;citation_author=S. Michaelis;citation_author=J. L. Brodsky;citation_title=Dissecting the ER-associated degradation of a misfolded polytopic membrane protein.;citation_pages=101-112;citation_volume=132;citation_year=2008;citation_issue=1;citation_pmid=18191224;citation_doi=10.1016/j.cell.2007.11.023" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=N. B. Nillegoda;citation_author=M. A. Theodoraki;citation_author=A. K. Mandal;citation_author=K. J. Mayo;citation_author=H. Y. Ren;citation_title=Ubr1 and Ubr2 function in a quality control pathway for degradation of unfolded cytosolic proteins.;citation_volume=21;citation_year=2010;citation_issue=13;citation_pmid=20462952;citation_doi=10.1091/mbc.E10-02-0098" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biochemistry;citation_journal_abbrev=J Biochem;citation_author=S. Nishikawa;citation_author=J. L. Brodsky;citation_author=K. Nakatsukasa;citation_title=Roles of Molecular Chaperones in Endoplasmic Reticulum (ER) Quality Control and ER-Associated Degradation (ERAD);citation_pages=551-555;citation_volume=137;citation_year=2005;citation_issue=5;citation_pmid=15944407;citation_doi=10.1093/jb/mvi068" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=S. H. Park;citation_author=Y. Kukushkin;citation_author=R. Gupta;citation_author=T. Chen;citation_author=A. Konagai;citation_title=PolyQ proteins interfere with nuclear degradation of cytosolic proteins by sequestering the Sis1p chaperone.;citation_pages=134-145;citation_volume=154;citation_year=2013;citation_issue=1;citation_pmid=23791384;citation_doi=10.1016/j.cell.2013.06.003" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. M. Pickart;citation_title=Back to the future with ubiquitin.;citation_pages=181-190;citation_volume=116;citation_year=2004;citation_issue=2;citation_pmid=14744430;citation_doi=10.1016/S0092-8674(03)01074-2" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=R. Prasad;citation_author=S. Kawaguchi;citation_author=D. T. Ng;citation_title=A nucleus-based quality control mechanism for cytosolic proteins.;citation_volume=21;citation_year=2010;citation_issue=13;citation_pmid=20462951;citation_doi=10.1091/mbc.E10-02-0111" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=R. Prasad;citation_author=S. Kawaguchi;citation_author=D. T. Ng;citation_title=Biosynthetic mode can determine the mechanism of protein quality control.;citation_pages=689-695;citation_volume=425;citation_year=2012;citation_issue=3;citation_pmid=22842567;citation_doi=10.1016/j.bbrc.2012.07.080" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=T. Ravid;citation_author=M. Hochstrasser;citation_title=Diversity of degradation signals in the  ubiquitin-proteasome system.;citation_pages=679-690;citation_volume=9;citation_year=2008;citation_issue=9;citation_pmid=18698327;citation_doi=10.1038/nrm2468" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=T. Ravid;citation_author=S. G. Kreft;citation_author=M. Hochstrasser;citation_title=Membrane and soluble substrates of the Doa10 ubiquitin ligase are degraded by distinct pathways;citation_pages=533-543;citation_volume=25;citation_year=2006;citation_issue=3;citation_pmid=16437165;citation_doi=10.1038/sj.emboj.7600946" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Mol. Cell Biol.;citation_author=D. W. Reid;citation_author=C. V. Nicchitta;citation_title=Diversity and selectivity in mRNA translation on the endoplasmic reticulum.;citation_pages=221-231;citation_volume=16;citation_year=2015;citation_issue=4;citation_pmid=25735911;citation_doi=10.1038/nrm3958" />
<meta name="citation_reference" content="citation_journal_abbrev=Nucleus;citation_author=J. C. Rosenbaum;citation_author=R. G. Gardner;citation_title=How a disordered ubiquitin ligase maintains order in nuclear protein homeostasis.;citation_pages=264-270;citation_volume=2;citation_year=2011;citation_issue=4;citation_pmid=21941105;citation_doi=10.4161/nucl.2.4.16118" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. C. Rosenbaum;citation_author=E. K. Fredrickson;citation_author=M. L. Oeser;citation_author=C. M. Garrett-Engele;citation_author=M. N. Locke;citation_title=Disorder targets misorder in nuclear quality control degradation: a disordered ubiquitin ligase directly recognizes its misfolded substrates.;citation_pages=93-106;citation_volume=41;citation_year=2011;citation_issue=1;citation_pmid=21211726;citation_doi=10.1016/j.molcel.2010.12.004" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. Sadis;citation_author=C. Atienza;citation_author=D. Finley;citation_title=Synthetic signals for ubiquitin-dependent proteolysis;citation_pages=4086-4094;citation_volume=15;citation_year=1995;citation_issue=8;citation_pmid=7623804;citation_doi=10.1128/MCB.15.8.4086" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=S. Shao;citation_author=A. Brown;citation_author=B. Santhanam;citation_author=R. S. Hegde;citation_title=Structure and assembly pathway of the ribosome quality control complex.;citation_pages=433-444;citation_volume=57;citation_year=2015;citation_issue=3;citation_pmid=25578875;citation_doi=10.1016/j.molcel.2014.12.015" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. S. Shen;citation_author=J. Park;citation_author=Y. Qin;citation_author=X. Li;citation_author=K. Parsawar;citation_title=Rqc2p and 60S ribosomal subunits mediate mRNA-independent elongation of nascent chains;citation_pages=75-78;citation_volume=347;citation_year=2015;citation_issue=6217;citation_pmid=25554787;citation_doi=10.1126/science.1259724" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=A. Shiber;citation_author=W. Breuer;citation_author=M. Brandeis;citation_author=T. Ravid;citation_title=Ubiquitin conjugation triggers misfolded protein sequestration into quality control foci when Hsp70 chaperone levels are limiting.;citation_volume=24;citation_year=2013;citation_issue=13;citation_pmid=23637465;citation_doi=10.1091/mbc.E13-01-0010" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_title=A novel degradation signal derived from distal C-terminal frame-shift mutations of KCNQ2 which cause neonatal epilepsy.;citation_author=J. Su;citation_author=X. Cao;citation_author=K. Wang;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. W. Summers;citation_author=K. J. Wolfe;citation_author=H. Y. Ren;citation_author=D. M. Cyr;citation_title=The Type II Hsp40 Sis1 cooperates with Hsp70 and the E3 ligase Ubr1 to promote degradation of terminally misfolded cytosolic protein.;citation_pages=e52099;citation_volume=8;citation_year=2013;citation_issue=1;citation_pmid=23341891;citation_doi=10.1371/journal.pone.0052099" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=R. Swanson;citation_author=M. Locher;citation_author=M. Hochstrasser;citation_title=A conserved ubiquitin ligase of the nuclear envelope/endoplasmic reticulum that functions in both ER-associated and Matalpha 2 repressor degradation;citation_pages=2660-2674;citation_volume=15;citation_year=2001;citation_issue=20;citation_pmid=11641273;citation_doi=10.1101/gad.933301" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Varshavsky;citation_title=Naming a targeting signal.;citation_pages=13-15;citation_volume=64;citation_year=1991;citation_issue=1;citation_pmid=1986863;citation_doi=10.1016/0092-8674(91)90202-A" />
<meta name="citation_reference" content="citation_journal_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=A. Varshavsky;citation_title=The N-end rule pathway of protein degradation;citation_pages=13-28;citation_volume=2;citation_year=1997;citation_issue=1;citation_pmid=9112437;citation_doi=10.1046/j.1365-2443.1997.1020301.x" />
<meta name="citation_reference" content="citation_journal_title=Biological chemistry;citation_journal_abbrev=Biol Chem;citation_author=A. Varshavsky;citation_author=G. Turner;citation_author=F. Du;citation_author=Y. Xie;citation_title=Felix Hoppe-Seyler Lecture 2000. The ubiquitin system and the N-end rule pathway.;citation_pages=779-789;citation_volume=381;citation_year=2000;citation_issue=9–10;citation_pmid=11076011;citation_doi=10.1515/BC.2000.101" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=S. Vashist;citation_author=D. T. Ng;citation_title=Misfolded proteins are sorted by a sequential checkpoint mechanism of ER quality control.;citation_volume=165;citation_year=2004;citation_issue=1;citation_pmid=15078901;citation_doi=10.1083/jcb.200309132" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=S. S. Vembar;citation_author=J. L. Brodsky;citation_title=One step at a time: endoplasmic reticulum-associated degradation.;citation_pages=944-957;citation_volume=9;citation_year=2008;citation_issue=12;citation_pmid=19002207;citation_doi=10.1038/nrm2546" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. A. Wilson;citation_author=S. Meaux;citation_author=A. van Hoof;citation_title=A genomic screen in yeast reveals novel aspects of nonstop mRNA metabolism.;citation_volume=177;citation_year=2007;citation_issue=2;citation_pmid=17660569;citation_doi=10.1534/genetics.107.073205" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Degradation Signals for Ubiquitin-Proteasome Dependent Cytosolic Protein Quality Control (CytoQC) in Yeast" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="twitter:description" content="Cellular protein quality control (PQC) systems selectively target misfolded or otherwise aberrant proteins for degradation by the ubiquitin-proteasome system (UPS). How cells discern abnormal from normal proteins remains incompletely understood, but involves in part the recognition between ubiquitin E3 ligases and degradation signals (degrons) that are exposed in misfolded proteins. PQC is compartmentalized in the cell, and a great deal has been learned in recent years about ER-associated degradation (ERAD) and nuclear quality control. In contrast, a comprehensive view of cytosolic quality control (CytoQC) has yet to emerge, and will benefit from the development of a well-defined set of model substrates. In this study, we generated an isogenic “degron library” in Saccharomyces cerevisiae consisting of short sequences appended to the C-terminus of a reporter protein, [Ura3][1]. About half of these degron-containing proteins are substrates of the integral membrane E3 ligase [Doa10][2], which also plays a pivotal role in ERAD and some nuclear protein degradation. Notably, some of our degron fusion proteins exhibit dependence on the E3 ligase [Ltn1][3]/[Rkr1][3] for degradation, apparently by a mechanism distinct from its known role in ribosomal quality control of translationally paused proteins. [Ubr1][4] and [San1][5], E3 ligases involved in the recognition of some misfolded CytoQC substrates, are largely dispensable for the degradation of our degron-containing proteins. Interestingly, the Hsp70/Hsp40 chaperone/cochaperones [Ssa1][6],2 and [Ydj1][7], are required for the degradation of all constructs tested. Taken together, the comprehensive degron library presented here provides an important resource of isogenic substrates for testing candidate PQC components and identifying new ones.

 [1]: http://www.yeastgenome.org/locus/S000000747/overview
 [2]: http://www.yeastgenome.org/locus/S000001292/overview
 [3]: http://www.yeastgenome.org/locus/S000004861/overview
 [4]: http://www.yeastgenome.org/locus/S000003416/overview
 [5]: http://www.yeastgenome.org/locus/S000002550/overview
 [6]: http://www.yeastgenome.org/locus/S000000004/overview
 [7]: http://www.yeastgenome.org/locus/S000005008/overview" />
<meta name="og-title" property="og:title" content="Degradation Signals for Ubiquitin-Proteasome Dependent Cytosolic Protein Quality Control (CytoQC) in Yeast" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/7/1853" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Cellular protein quality control (PQC) systems selectively target misfolded or otherwise aberrant proteins for degradation by the ubiquitin-proteasome system (UPS). How cells discern abnormal from normal proteins remains incompletely understood, but involves in part the recognition between ubiquitin E3 ligases and degradation signals (degrons) that are exposed in misfolded proteins. PQC is compartmentalized in the cell, and a great deal has been learned in recent years about ER-associated degradation (ERAD) and nuclear quality control. In contrast, a comprehensive view of cytosolic quality control (CytoQC) has yet to emerge, and will benefit from the development of a well-defined set of model substrates. In this study, we generated an isogenic “degron library” in Saccharomyces cerevisiae consisting of short sequences appended to the C-terminus of a reporter protein, [Ura3][1]. About half of these degron-containing proteins are substrates of the integral membrane E3 ligase [Doa10][2], which also plays a pivotal role in ERAD and some nuclear protein degradation. Notably, some of our degron fusion proteins exhibit dependence on the E3 ligase [Ltn1][3]/[Rkr1][3] for degradation, apparently by a mechanism distinct from its known role in ribosomal quality control of translationally paused proteins. [Ubr1][4] and [San1][5], E3 ligases involved in the recognition of some misfolded CytoQC substrates, are largely dispensable for the degradation of our degron-containing proteins. Interestingly, the Hsp70/Hsp40 chaperone/cochaperones [Ssa1][6],2 and [Ydj1][7], are required for the degradation of all constructs tested. Taken together, the comprehensive degron library presented here provides an important resource of isogenic substrates for testing candidate PQC components and identifying new ones.

 [1]: http://www.yeastgenome.org/locus/S000000747/overview
 [2]: http://www.yeastgenome.org/locus/S000001292/overview
 [3]: http://www.yeastgenome.org/locus/S000004861/overview
 [4]: http://www.yeastgenome.org/locus/S000003416/overview
 [5]: http://www.yeastgenome.org/locus/S000002550/overview
 [6]: http://www.yeastgenome.org/locus/S000000004/overview
 [7]: http://www.yeastgenome.org/locus/S000005008/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
    <title>Degradation Signals for Ubiquitin-Proteasome Dependent Cytosolic Protein Quality Control (CytoQC) in Yeast | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Fi1GgbNqDdQN-HuVfJyZh0EAidtJX5AY13RA9YNGXyY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460402","apath":"\/ggg\/6\/7\/1853.atom","pisa":"ggg;6\/7\/1853","ac":{"\/ggg\/6\/7\/1853.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/7\/1853.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1853"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1853"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/7\/1853.abstract":true,"\/content\/6\/7\/1853":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MBzE1HELp8zIHSwKERErFDX4gmVQNXwoA5RSXu4v+xM+nubEi+O+19+aRQvuJgZ1yGu4cUy7vrwzZkFiZNQGH6U35l91LhSrpuQdpi0NntZChHblzFnkmDWjLP78H0SuCDbhaWSvaDLOaA2UuQza550OeWiy77gW5maVQG7LSFcCDJ98bVZwnv94jrrnV7ytpSIuQ4HO+xILoZ1FSl0pDZmBuVNfxpC4jO9DRiBNBTgMhECae+6DBnNoekIRq\\\\\\\/N3To6yizkdk3AFu3b6jf3NIeDJ3MR6TiYyMCuvD7cSOmSSQsqgpxOM\\\\\\\/fsMwv+IZq5gmW3Y5zbQ6nkhG7DMFcBpFNZQhDlU8pYnEvp\\\\\\\/QnObYcD1yiVYugodCI6zEAo5d9qSoI2NX9arom5eO0oNuUs46eVnNl+YQKgU6GkZhto9ByTrmYtyOpNFM8J1DdXifznN9NcOh+1BWOhoaFm1uj8FRHDC2tileDgMb9+C85RJiZ3PbUwHeAM9aNefnbh5f1+eApZEFUHus3YLI7\\\\\\\/N9phTqDqc1K+lO8x8HmQjxWeg0Qlgq6fpkhREqfoReokUfpa7oNpKyRH\\\\\\\/WGlXmXSPeWxsh+Vuuv1leVSI1A9kw6mfEgtMUPG8iqIgfZxtHdXEbUDbSv1rKIMvhhWFiCtQg0SUPB2MShKFcjR8mW9b1ZCBChucKJXtNm2tw900i6ouURLjU\\\\\\\/5AHwtMOlp1LLiTBDeMkLonWjR6\\\\\\\/rZw2B3RcauITvlJIADgkSzK21gYO36DXc92IVgCf1qRgNuViIDOjfyrmyMG8\\\\\\\/owm7mzoZNWOyAXEqy31FruJUY1Qxk0fP77OjnVLRPG2J2suJm45Jnoqn1lizq52s17H3+C8AYbk6PByQKP64Frhdtudp2juFOUHsN3eC8H7B6JmQt8utA7LJSZJBJoWEKPO8IICK0hdl8ryzAcbPz3qw8I46UXhsJGSYRMi5y0JpvW\\\\\\\/KN48dpAnqs1mRXJDlMIsL9CfkvTCSpjDoqcMBVLyGs2PMAQJsuXRv1YhMypbd3YC7Ub9gdntJQR7Q1iODfR7FSGn0+O\\\\\\\/GPkid30sljkBNJgFgDUbOAV\\\\\\\/p6qVbzm3XYIvG3peEkkV1sKWzf9gWlSSGk\\\\\\\/LB0LjgE1E05bB+ywC2M84iBIi0CtnT5TbACKsQrP9zzWgMgGLjCBstNkGGlauvaMgxd9DfT2MSl+7I5HMpg\\\\\\\/Is3vzRI1cqvkviktC0owzslN3gfPelNygzvDVrb23RNaESxvQ0WezeaPvmXB9kHAdBbN5aio4rDL\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q9vOzRfJl8g3bsL6M7Em9g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c73fc12f2a6f4fae21456881adeac9f6c3f01ec445d732e87754d454189aa790\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SzFHXJ5WRdGOqWlglj6kq5DvhjIvdynGAbRIUL4Ry3BQW6KxVkR8w5j6U2wCzUE2fr6UOD2dtIrwLd35VZ73uCxvzcyjQIcYATeVcnCHZbm4fGreujzE+XkwARuxOmhM3jF8rvhuh+ojJDOvo+ldkfIGD8+ceBSEDC7PdBwmxKQVFKtLWA8\\\\\\\/EdFQF8EA6XFmqdCfMKMaHSFN0SYtNDqeGfhGcL0FhvDt1qkjSlSs8FEx4qsFE43uV3rcvji0KzFxVPq3jGh1pljk\\\\\\\/9Rn0YVe1\\\\\\\/8PWzYZ3MTxUbz9ex+AvIBdY4xsN8E2a2pphmKbhUSM\\\\\\\/NGJoEM9Hg0lF\\\\\\\/t\\\\\\\/ddYBOxdSlrllNFrjUkBPJybct9WvF\\\\\\\/PTxfDGiYEWKgBnTL63w02qqyQDOZYG2xaKtwn8Uk09xU4WizdBiFAtrqFxVG4t6VnVluqhTRrze7RPncIDNvWZE\\\\\\\/9akt3A0z+qZxujAxTuKIYOv7LuyihEFj09GJAqTGaUOcU1t5Sg+TkmI7iNDg\\\\\\\/93vUqycCEDlP55WyyAhcmLiGgXl3R7bIcJUS5F69X141RTCvivgk1aBQoDdfTBb3khHvg0zNEf5Esdq5BqirVsJXHk8rs\\\\\\\/fs7E+5Zqoo7gIiTwYFQ6uLEKlWp0KKlU1FDEi28\\\\\\\/osmnOd88QdzsgzwSFoSZJ1O1Ofk\\\\\\\/GtX90CC2GxXr3fTYB9pEduYXILvkWSgUbLXTzbAHLRiiDrWxxEb+u\\\\\\\/XXa97CL0vEvwvmtTCPWZ5V0TGP6c9z4fFf6NLULzJ5EiFcQ+uTzONhw8K1BeeH3ms2ulK2T1ueYUFqOPENUSCCc3v47akU5PgX50sp2OC6yFHei9ngG0iU7VKEadO9xL9Lhq5oA7atlBOIE3\\\\\\\/wBpoM+GREq5pGQR4pH0tZ9WhXmb4CEish8YDwdy1lyI39XNPdY1yGyz3dJdI6FguGhg70tg+AHIEvYsY\\\\\\\/5IhLfJGzf7vxU2t3Vl8uMynxtSYZP5L3SvIsbjGoNdFHlQwNbKk0rHa29GGf05TodnZBul1bCztonbSq0\\\\\\\/yxxsvt+p2bjVi39g6vmDb5uJmUO46T1J5JFj\\\\\\\/Mi9d7oNKtZLa0C8ez9ACI5CfP6JxMTsk6agYuXliRn2x6IEK40aMz7kiCC\\\\\\\/mdO5HoSIOVSt7OYsOLM0vXBLRIvKG7bqrI9wnfPpTA1i74d\\\\\\\/XbjFgtbmH0guurGHXeN2+eQlHKSryMzHg78h4yTTuy2Vl1TkGwUHI50obW5sYcCsoGPq6tfTNdQLbVkbhLIPGJEoZv5zpOXhr39aVCjbsrNZWqTpzY8WyTF3FsloTgr2W2gNrjSaReR\\\\\\\/jOLc9N5k6m3n8mEEq+CEPZtKNo7\\\\\\\/4+78CGWEbiksr4IKH5Sj3y84zE9lZbBtK+5iszJyFRyI78rcElxM2cM8MznWluuhqYM7yCqLPuRm+Cy\\\\\\\/s806+fws\\\\\\\/4MhzEiuqL2LeLTKRloTrcqHyiWvXufYprekzCuEk7\\\\\\\/OHB7rYP1sE3K5N19PC6hV+IYccwAoZIccz8rmZrKqnpS1xd8CXRfs8pqxVX6jldXXKGDvecKa3JX+hHSjVWY3+CeKFmcnV0pe0mlrc3gIXirXTGL8Cbi8jibPiGshYtBHzdITv6z9tiLQfuZIEzEkvB5oan7WWey9TnUuu\\\\\\\/19mNDj\\\\\\\/3cm\\\\\\\/NmeOqrSMd1lpVh3tWgo7ZXwGbAwGPKPKvaUooxDIPlZ89KaS1FgrmDRAEjm0QijbuTJgHLOlKBkQTVtNhW5dlhkZGu+tRycIrjeAdf+wlQqQxTKufPrlAS6H10wmSoeI1Oayxw+6NgiiG8VKCQpTnr3jLxumhRydiJTExVIxwZsE0NeuSoYgA3G\\\\\\\/7AnADbS7QZTdGOzldWgedAnFoKLcJk+xtR+TSIbo4FK90IE1Q\\\\\\\/8pdyAt410Q8eO+fGhlMMPtX+itO\\\\\\\/TfiGMaPJL9QN8lXIpEEiTqmq5XFf8dYAIhm1LUTpME9nICmzKnokgzKMYmZ8X10XSVIHu6F4nIhcApHdNKKggxO+OWiV\\\\\\\/Fio2x\\\\\\\/ehikR9hPh4M64FPLhMsyiL1UK1TIuAOBvOC5DG13a953zRCNcM1dorf4DVW\\\\\\\/vAqnmx1TcnjstJ1HoL9KZ237sKxTvWlq4oOin3A8ibVMztw\\\\\\\/smqtIQp3jlv5YXdq\\\\\\\/+C6Mh1eTcdliSoblDyq\\\\\\\/umEazwEwO1F15YR+r2yISyFSpqe\\\\\\\/ofwiXhjakvFKm4xUBtEFWou37n3zoD4KzQcFpHrhJUc6r6hkXJKq1PW8vqrW5KqQDd4dwnve7XHcRrFCMTG0VA28p\\\\\\\/jtVuTvnucWU6YaD6q1CW6UICq7vrnayV6QvQ2avsdrGtj2EVpiI1PKpnCAfeoBioOQPAKEMLCQ6OkSidTqfZc3hor6LmMX8OmQPTXb3CGJQiqpPRG8KP61ejMgX9Wz65gXcKVLKQ5bJGimUqn5w9n+mHpgl2wRZs3RyClTROV6XnBQSn4O20SdQWw0AhL+qBL8rvSJLfhTSvn5DJtoATn9tFo7UlL9lMM59ceh3FMZy507nK5Ei0v62gAi\\\\\\\/vhZXO88\\\\\\\/gW9Xd8+aw\\\\\\\/dTEseFW07KgJ8dNvXplcBvpAbLgAVUUeuMnhf9UkuPR0PjFad1NdbGpCXK4gQNp9OBMnDLcawguaRKw+IH8wdV1wCUleNwPqz37h+lwLnVvkAA2Qq6y35tGaSh0FpuAnB5mmwBYutATv8DLzQ3Ju\\\\\\\/V3SC+tXrwZIl5d+9rtSI8r8i0KNrqfdDGM05nk6Kpy8bW1zGXk6Sc5w5q7YkM7jxsJv5Klame4NAKpV2hvldpKKPZrheJYGSjtnlRfbU23oIqEoSyjau82vQ3xRA4qeYUYE44b\\\\\\\/n6xdwvWuadG7rQ7nU+7Vv42xn6eVXdnVSaryyVBhMCIPrHg\\\\\\\/ALRNhJ6E7bsOdi31CrrdJBZ37G56zuFdNa9sOZnmPBMH89pQ5BsFGs7Gtj+ZHEb3ppQssTQksTbx+76\\\\\\\/+jinafW+CL0cNcc9BzuNqFIAbHztIKqtSmtaswoxppFp1cegOpQuj2qoTWrUqDtVAyLLldE87DRAB+3Uep5qGfMDorcDJcq0M0W40rxQCXciLrTkmggchx8QC8ejHZpHpW+h2WbA018prU48Fp0E2+G4kv5sJxPrqvQwpvMJG2zu3vRIUO+JWQgTTMMhXUhR\\\\\\\/MPZ0gQUu5r6dFmtrIRAk3gnmLEa0Qa9vG4cGSmstkytCVk758wBbxmQz2B27rFbt+B9x7rTFe7CuRYqzLRe9xcPP6\\\\\\\/32SJXg1vJ0gX1pYLjuclw0hiFPnVZ1GIz8bhaI4y7YYUrvcAOkxIPw0LcZNj+AvkMGlL1jDFtkOgs\\\\\\\/TWJqAjH7zS\\\\\\\/2+uDxoxEGYtzNqra3KMCAwymGW5+bhOy\\\\\\\/YmBAvwy5bqzFAnQETOkmWF0grRs+o4eJ45EhUTweuR+qa1jsrJkHpuf5mCm45cOnrv3e3Z1hBh9W5tDnmR0TMKEJzyZbs3mTRZuG8UahpGY9WpyxaSbWtv6TVqVX7cqNO7jzH4FYInlUBoQL85U7cUpYJgYP6dp7ZadxP0lEQnnV8fX3ZLDqA60fiIbOD3KxAQkJiYl5lWID92JiOaYWJ07l8piARYr7sxRA5QsGAY7seKCZLKhKxjqUeG4QC5w64mLIcpBu\\\\\\\/DIjJ61gbwBqBRfPPX0CY0IgJh27ANZftcyheWX5540d83AzOWNm47+l0mqhJccqYfVLDzmpEnaASooI3zHcFSEd93iCt4JBShp6aJ\\\\\\\/3RPCXnfZnsYnQlWEsfTAhXxLuVncJsH9VzfFFExkGp1slYBbXgbRpFy91fqtWaBoYCkOe2aV5EIMPgkXf3X9atfGt\\\\\\\/6PIHUypHY6Elq72iyLxbHQZE6HnIranFVIXzuJcNS1R44fFiUcb8\\\\\\\/xev0lIvFEshwLGQxBldSzz68F3nPJSPW22ikB2Qq2RLQqt26nuUCD4bIOdz2o9gNzrL4olXCBYcdENd55yBszY1sa1snBr2RsEne+CNqmVzlg\\\\\\\/BrLOwLVN68tiPsomDlm4dnD9upIuyQnfy+5NIfQeZMivTDdveFPMNcJLlDLrwYhG4kgCY2Vc337bZMT1lCGyUgQY+8xg6GzNud0yVChNxnS\\\\\\\/jH+vWdb+yg5c+xlNDZXMYbVWDgGRynf\\\\\\\/7g9DPyCenR0Pi8eGrIMVctx+LCI+MMZVDVgkT4R6WPfpjjDaf7ubzCKav4w9DP2yhxF4WZ1sGHYaPlFrvNAnvCaYtiXP6xKihe+pfmPeu8R4oH+c13e6ZH5ODSBt2c7zodXZ9kdiY2dxYNpqnDB8bjMvYZ72EDOZ5JsQz7WNDB6pzLPDsT8OjBzJn52\\\\\\\/I1DztTeQwiHRVdkxsmkghTAohpNq1S18u7C3MYrnubNzl3ZHDe63JjTZuOZxh2mDhT3hv1OqqNSb4Am3Gpi0AvP\\\\\\\/EBNQiHbsjoUxZ\\\\\\\/OeSFKMI1MgVAsob+V5X74fFIw82qqS9ebXrVA3+DuWtPBEYRt3dMttMlEdLA+\\\\\\\/k2mzSHbr4ZA1uxhWf1gapeKfzFpl65CTScC4l49MnEWGo8trS9ahUviztoBNk\\\\\\\/fTbVtyMiNEP8chzrJEmahS7pzXl1Zhek6Z0LxjGC37J5DVgw\\\\\\\/HO9Wd9UQ3SnbrVDX8zMHutWWuOiAk1bMj83WhsqX5pw3bCks2\\\\\\\/q6UtJELHDu9EB4d0OPLF04mhl4EMVSHFJ+8nciFp4LJbOevB8xlZX5js3fEHcjzZNfZu269CDoKeYoT\\\\\\\/eVjnCI\\\\\\\/x2xdzrYrAWI1lnQSFJEi+Qy8nAFofy5IMqleJWJTxrwgpNhaTJw2b\\\\\\\/2lDEp+oqNUygbXDKU\\\\\\\/S4JCnEmmrimuC3t5jO5pc1CttNPwDZGr31mTBWYqmhXrrI6e28R81pKGuGa2kqI7G0yF8taHMd+4HPg5yA3yrMHk2yIeRC3saLMVafHSEI49O5Gc4Ep8e+9JKczffRR7+pKufd9If9qKKXhVow+aI72ydVmfrRsZF8Ju4to3kUTN9YSQ2izTfE8NfvREy5+dI6JUnEsvq\\\\\\\/3cKQ\\\\\\\/cCSIIvbLS1GCGaeRtI7+aKIEZVJNjo60kPszD1s74FoUkEYC4UkmdNu9HKggdZCWLzRT3NQc\\\\\\\/AHP0qFOy6eCceG+Le7pwVUZ3db\\\\\\\/IrAvUxUobe6Sf4SMOyzeEkPgvAMCmLW\\\\\\\/L3ITS5HFN+tKfUKUYrJ1V9ZzoAjndbUCHKpt63gxzU2HztsPOTL0E\\\\\\\/I43jPW31M2ifSs1ubIprDOPjfBeDrqewjkjePe7nCvwAUiTixoZx+k+ypxP1NoTyTC+G3LScAvxG1JxobUnoM2OZbyIdltm+UhDjLo9bsl6ZjrQU2pMJ5StiOC7N40drxawZwI3yFXviNFvMefcXZ3P+3DsiCca6yM\\\\\\\/EZP55ib+wPvDVcUjU7nck05lSUo\\\\\\\/ctnyzsEuRuJE9f5JQyyPr03IN6xlLlKu4Wv7VQ\\\\\\\/PQU\\\\\\\/cDz2h7aEYgFLsaT9qC1dZ86seoSE0r8FtD2NsbzNT\\\\\\\/Nhvf\\\\\\\/hyYSCZKTcW+AV+B6l3tVto95OgqKAFBEBpPWRQ80JWqxpX2vgxkG+QfB+R0ZaF8HVq2TpqNJpw2cU9Ad57UDjWi27KzHUqYjzW0imWyZryF5jtbJQNm4kZAFnG1Z3KdL\\\\\\\/f6OCZWSin2g5C44EUZ\\\\\\\/QqlQSHNvZQtBLAXEG24Xww6j0yObl3KrxicSTEFhmAJKnULYNuVg4MB+8pzH3MJVTyGTOpDKifgnVJZAWycLfnYe1d\\\\\\\/qWcswOyPYgU3fN15jsYk1zQEwHT7mUE3EonCahlMpfYt\\\\\\\/Dw7f21OYVh8Lm2x5VChGpl6tEtlMTB+P\\\\\\\/RY\\\\\\\/mooXsAIiZOZRVJUcv+5iqciKR4VqJzJA178rI8CaWfuJmDoUvxBBpKLEKyUXdn3p0OdT3jQ9RzXuB5J\\\\\\\/G9W34X3f+W54Wz8SfEJjeoRh8t1bhHczmN6tgzJojTM3dX52wt\\\\\\\/X1Vgq+rbEEYeCRyQO5NEh0fEMzltZkQW2hHp19MJ2hcs2ANz\\\\\\\/NGQWwlneMuHI+mvjJvdTgU7E\\\\\\\/cCh\\\\\\\/dKsfZWxQypYtG11XbHLmW\\\\\\\/SvuS8+ULhH333K04w8AB4GgSyE8aIcBifV9rBQlyPeLQyarfbpuk2qHva1j1Gf0dqK3HHhrE6V2rl7KNwWrBkOK13lx4Os6H3H32qBSjY4JWW3Yxtm83JIm1CaIPQD66bKPQU5iRFUiBJVZq5+qqo\\\\\\\/74c+RZV4kQVh6x8FILUTBwcB4c09uojXTTr6khveqQK1fuR+01GRy6jF0nQfv73yuwpaY9hA8Zd60oGOvPBR1D6yclEIK\\\\\\\/c8aVOCWGg45I2sru68lZg9VarCTJgwRuFv8m0J173PsIPHQlOHNbyS+petcMM612LIgMcf5PCtwHwjVkqXWwdKLZqWmrrV8jyz0WqFYffGxrv2aZs+nex7F+YMwtt0\\\\\\\/uX+lWMw6XRhD6mkwi4jXcOI8wPRoGo367WCsGZziVQmRHlA1eJPz+cKEyXudvXQO8CIPvdf0ABD1TzNGl1jUJom3OSzGp7tg1YINlxE7ZMn4eeeOe+9fxF+C4Q4jCMwxPNluJYu8lZavaH6YtJy5brW\\\\\\\/MMRNlXBpkV1b36iotPmkbDL5iiBQ6BcdNrltADu7HxlgL296w\\\\\\\/Br8rga73oNrnj79LkSLH64EEzxGv5RdOOIOMCc8K3L+onrEsFzph7TcK\\\\\\\/k\\\\\\\/keC2u8DG5AZz40wiUR5RsEvTXOX9PWvWI0gZsUaog5+Zn0VU7AuwbK6DssG1Cbe0rB\\\\\\\/+SBuU4UW1Vj8rL0fHeFV9s6YV3kE55kOEPT29oEsLNVD22KISlifDWKMIw8HAov7oZG6RVw2lpu3Mty0nHgdq3fQqcOI5l1mD72rEmZt4jYOuosbZpHNf6u9evdh6VPZhI6UZwLOvEy0exYNLQw9ghHEdMYHNESS2wmg9XAuQ6ua\\\\\\\/YDC57KKzVO8M37c6tn3JfI56XkLj2GgJjS+Pb5I\\\\\\\/hYZPcQ\\\\\\\/3t92wf3KuYEFjfkFhUqIa8H0SFJwpNlMxRod9sQNDtOHL84zzUACiikIEDxwrCdv\\\\\\\/qqgWbbzLOGl1NVSsJr\\\\\\\/mTchPgzfNlZYykeukv88XLkCOh0w\\\\\\\/U+dfqAImhFNkO568PbUe2QcqQDh6vc\\\\\\\/FPptv2T1VBT4ubhUdm+4C5gSCuxcW+hsbw58Ezig05GQkcOboUYl26u4B5x+ng\\\\\\\/Wjduo4h\\\\\\\/h13qgf6xjM+ZhdbPxwk\\\\\\\/gsgFmmy\\\\\\\/aNPlOWQzQV7IgRu2oOOvg5cTMoQGGnlePKDBv245VEbsnWxosn1GT7XidYI3TGD4tkeWJvUIrvlmn2gcCtJ0PP46+nw34qlpA91c3WOZ2i0WNKcAM6cOT3\\\\\\\/EDvbhZCirAR2MC\\\\\\\/et0JZf4FnIk\\\\\\\/LVGrHXAnY0aLq\\\\\\\/yVmr7U1iFbTFTxlP4DTq8e2Ub2u2wWBoKUuRjl6vJKeHjEH1ymRsqYbS6oRQW+JGIrqb5EtwWgAHFVd28CMGK+4sJ1EdZhUrvz+kmzs8\\\\\\\/yk\\\\\\\/p79p\\\\\\\/GIcPHA6GBEd+fLUVTL4RHfMsL6sC8K4vuUEdU07FT1qAHQrah7II8snAfp6TY5R2QtMfhY+G1JBymsympJ2KJE1s7TfWi+vKmtIMSbsQM05P\\\\\\\/zVjVeb4ym+V\\\\\\\/Mrmfh0ikizJfPXsC7QqcUvi2L27KC94L+suZTMW93P7l41UpnUJLRaIobQcXACkIKgL2eZYk81F8nE\\\\\\\/Pf1qloPDtyDwFSgVMxFb\\\\\\\/7U2o9fAbJZ8sPav929ESB11BRQkOEq4hNlAcWF0OB1ZPBHMRWequ54o3m49I54T5NobokFFrsPai6r4C\\\\\\\/KPJsf219\\\\\\\/rS7zaSQPC9Cg0RfiOqE9zEmraYB0Kt6TlGcyE38M\\\\\\\/s1dWW13+gTsrghROyRegSjWlXQWjQ0jwG6sNOeDKC8\\\\\\\/w3d9XB62PbjexYOfx6N5OTyOqOS7zSui3ptAuiH4D4nzhUpTrPJOcWJkvqIimWnyD5CG11CU0kY43P8U2d2U34E+JfRp9VAalFKtFtc75\\\\\\\/QSTe9DAimbMFkogCyFiQS8bJpsV1Tk+P7eujPbcZVIdwXrRmQ\\\\\\\/S7xkNubmuOY9xRMTtPd0BX+xt\\\\\\\/IiQwVLDk0jhUGnBYoBb3ccPr3GnOyIUeMLStEsYyHreH7HWdRSZet\\\\\\\/d81tOieivZ3LL5x6dB7BnAcePlNlyHCl2OaWhfM3Az9J9E\\\\\\\/ufRvUiRQsdAgqR+h5x+vnjT3Kj5UKlAUw17fxaQsjg0jgegEybKXN7xxyG2BqM21iLz1Q2q7gK6FCST+txk84XjbCqKoWsCTg093jmU9NTAJJs8N0QN1J5ZkWzjKd2vl22LURSTkaQNpUcC2yoGlS8X4Sa2+Th43QZhAf\\\\\\\/sR1+AYTBTwlLqMOZP1tbVQSiX85RdMqeArupGs8Ugfof4k26a0BXi+0d6i65tWr+tBkAAJsCPaeECOcCgJX4VZZezSKFN33blgwhjfH\\\\\\\/xwbQn\\\\\\\/7TmfGwBTSwQB62vyt2llRENesGG7UloYiPPc0+VjuArVCHdnuNNVef1Ryt8RQIbOfI4trphEr2RYhcP9RpX5A0GJU3aRsGeL5ZaVuR0OLBogk\\\\\\\/cM6b7lLQQYBxy9iHNCgKsA0Oh0mPtKq6Kh\\\\\\\/iwlxPOdXuJiEW\\\\\\\/+wbNEwJ2rmpCTofK\\\\\\\/Ll3808uhSR7mvr8jytHQSvOHK7HX5iE4QYVY3GHvcV+mSOx6Efd4LbRmns20okuW6qczHQrn+vxm9\\\\\\\/9o58JDXENci8FpGVxPqHrkNl2rURC2omf4b7l6ktMV92NFZcZDHtQFyS7zpbEeh6\\\\\\\/kJZcx7iClnJ5RJQ3gXSM33v0jqm4rXht0J4o+REiwGYYYxOUnlq7MyoQRv+1IFgFX09n7ftEzSjUwEJmSkLtC3rQBH+mDkTcspdLRJKszyEWkdh7wo03rH\\\\\\\/GpcxuB\\\\\\\/sJfJzXmXt3mtHiBOUXqQqzaB0dDa3kwT4qGU+eNB+fHwHJouTsQZgx03jpbGLbG6q052JmRD3Zh9lM1mO4mbbJwZNzOgWx2gfzsMq6xaKYabez0wGkgW6oFfERsS5+D\\\\\\\/KgV+DqPf8rRFCZVZdF6puFfYYWr54\\\\\\\/eZ2UQO\\\\\\\/zOIvgIXyLXHY8vIuDupkkHAf1nS1UxklqVP73cA2iScxnv0I4wnarODMyWUUA3VEI9XzRU85LQHjK5evsQNJnGh+QpBC48SHH9ntC3Gv3IbD+ChnGWY4dSCAJ5u4S+GWkCJMheKww\\\\\\\/lPXGj+85teI\\\\\\\/uXeuXyPTLMt03SMqPrYFDBIpaslqn9pi7CLV1pCy5WId2MGOM8LWvOH4x2jTOGc1IBxTSer3tR7xguz4KGY57JzQFQIZ\\\\\\\/P1ENRtij6gN57lXkj9hmFRtXt49\\\\\\\/6xBnGnQV\\\\\\\/FJLKAkKid6Kb2qa39StKw2idkdpUKt1k17uPrT\\\\\\\/OUoaV8mzdko3XsxQYPDRJAgT3Tb4q3JXotb1yYC+8gsZ0rbjfBSw2du8FFXcfxxf4NmGB6NDZevNKFBt30Ainuq9JuIYV+Z6XppFakTXGBUSFYJb36NWTtVycAkmweA0zKXsz+ELFSN4W45siM1RElu6FG+N8uhqlJxn2tjMUaLPw+Satlq9znVkFTlkI0+M2znQ69oex59TVVuB4vXAczUCgcFQzGntMoao6\\\\\\\/tWxZrL\\\\\\\/R3jb9eVTaaCZ+MnUTdCdzaa8LRp4qxc5tshGdqoHTVSPaccxttNUIRCJ+MnZQshitdn6yRf3OYy0SXxYDneZcnAjwQTpuVYZfxX64jrlzwraCvZ2kCjPf2HGzHfAN5rnm4OU+MbEOO\\\\\\\/ugWzNi+34oKvoRFQylFL1agfCvNLC36Xkp5MQMIwO8kMZJmYhmucdCeejAqGqHtVXQhnLvt\\\\\\\/cPiRrkpCt42kiL94FM8rYw+g1jzkg68zMcoD\\\\\\\/Yue+KoQdFySEo6yd6GKQf4UrwJLul0yKeT9\\\\\\\/SNcvUCeHw0vvxSrYr5yOsGiVrWS9uLOaWQWfDsygc9NSa3RArd3BO8lJ2NL4BJcIoDLLiClbRuQG6UBcEoPVdzsYzEpdoYx\\\\\\\/AXtTWq1xylJtHESzhewIZDB+05MjBEIEAkE+CZODMV+NtVBF57HbSXRVYnYeOvD6CfCmNCQTxw5FdAoupCFLZtIfG4sXtimlnbyMJkD5GY8YZGTaUy0mGSntb7cohRTZZnIFzc5TF+TC+R2Re4sSbv7CoKusvDzqfWsqGQHTadoSvuvOrEGs8yIhRgAM8PPNeNKqCHa4bRrU5+9UADAj8kSUmvyu+8XAQslGX4G14K4qNlU1quhoZihRFZYFk7Ry+rcznZM9I0V3AJvSFXOSZTQXu3+pp6+b\\\\\\\/YrGr6SFDPAZV2QI8\\\\\\\/nmZ1c5w7J7b2daSyrdve1iDl5ZuKyT3wKNWQVx+hBRcsy7EKiKJv8on6m2ovO2OF\\\\\\\/st89ACope2AbSyScdwYFbBIBBmXvgHlBEVgvLr5CtQ1eYNK85H8344hB8oVxld1ePZg+bb9u+FBvZKpPLh3SsHIvj5Z2zuYNnTzh3IFcbGxdYdAaUbukweH9KQH\\\\\\\/HSpkhJ\\\\\\\/hkezJ5PPnat6m8+2tygDOAtDh7BuRKLabn20yOi4KMroWh2FFq3W6DP8\\\\\\\/Qk1uQg2G+lAIIHrA5HMeGWpOV493cVQMocmqnQhBxm8yu1bVBiH4r0M9R3gf9XpoJQRCsreMCC3X6GzH9BGpXULAQKtka3jwkxJ\\\\\\\/ls2eRMEs0rt1h1G5y00gcZ+RMSbe8akP3WqOgRQZnf9VKJreRaU0a4gUsfwsbnnJhwh80D6g\\\\\\\/jSFyw28aGVerOwhyhGHHZgNvbIiXetAmIdx9GXfZDSnk5nVYzENjvRFNoVvHviUrim77V1wMfmNtUhOtQavtDuhhxzsqwUuFzIbfLYUYZlMNfrMyD5a6731BymKmkCCVv2N2VD0k5svsE15EtYk3U8v3e3AMwpRaPa8ifmLdS2vQr+4eUflPOximvr7AoBM0f\\\\\\\/ZLHbaE8ltvAuehr67YWUFP0k7BSb2HuiAC2hXFw1RvzTr+Bcfb6XYhab4Bzwg4dhUwTwLBtZyOM5rybEZz3jhOaUCnyZi7JV3VUUDWcbmn0\\\\\\\/uxahqLin93khQIj+GA3vLKS5vYiDs0hH8OVYseanjXuSzjEXyUO3erObC7ALjlKHqjEp4mCbljL2AewnjC4CzO1QdrOU7JaM\\\\\\\/mnXLzT0\\\\\\\/0ZNQ38YQVgaXnMjkW+GyDvoTOQr8lt+yqc+MBb7\\\\\\\/utXuUKFGjYkcKsZ2kkAFjtYQ6ETkgDHUsQuHe6gcLPdNC3NPtjNuxNIoYz+881L+Gx3nUSvEDhBqB9i7rupHHK85h2Lntl3DU9GazFK+d0thDe6TjrfbUEjYaOQfHFwJLl\\\\\\\/e\\\\\\\/xR6Ra2eM41vGuAizNJGwNppajQuYbtXMQJDt3PfSJDUtpT0pjFfe\\\\\\\/9nN34GVL8YUC4Ksr5eBqvt3VkiUri2DH+g2y+tj+vZcvxk03gATMY\\\\\\\/sJAeOiK6K1Jg+xgn1g++ip5kQNjwVMdFkTmacQDYpRJyt2OGRqnQ2soUVjbTpIDlHCKIEDxHG3wRoHfzBXvWLKT+W0yhFnsllWltKTL9KeCLY7LC+kPZ7jbjHvoqgSTWl9KG0s45SRGJUyynQvHSZ00jBSPy\\\\\\\/qOCDm49Y+\\\\\\\/jg\\\\\\\/5B97hiBBh0ZXIMUvc3asjd7uvGOj1wS6\\\\\\\/zAC7jYsM9g0TyJc2ODzrmR1jeXbLCaMszcpSQKcb9yh6ZyyzCEUXMlZsFL8co5zUDt6aN\\\\\\\/qaqwfGLRQ7IlBUpx4QcxCvRvl4gPHmh4XRQhDPHVIohHHofIq69UF7RfwXga+8+IFMC6MLjwiokqg3zTPKY+vHQqQ1WxzWVK26gA9JkcI2A3cHbQyMQWByBjpv2Vr6vMCV\\\\\\\/eQVWxwEdPLFHX1gzX3pFyvycLRihQecpIszlvkfxSqFK01mEX2dV5Z41Wu3d51trIVtKHnOfxm+xJlDQbu7Ce9jFz9zA+BTHHvlU3gNCHobms1oRvm3IxKqE467r45CjY2azmEN1oXmn5aNwtHFFckRd1a7L05nwXun0T7LOUU2EaeKLwt8vKsVeaGmZLB\\\\\\\/CH\\\\\\\/IGcbuUhWi+NGm+rqs3GxZO4pOG\\\\\\\/ca\\\\\\\/MYZ5qc4936eOyKFC1ys1FRX\\\\\\\/GMSJduAaVnDb8mzNde9W8wV3u7PS6xh0Hly3eEZmyb0u02dDuORUrnw00dRqM3\\\\\\\/vmKYOtP5zK1N7dKAuxaG3646pSaBtaU2fhU0gJN8FQKPXz7Cl1QZyp7jVkxI+JnIq1AxValhF4lhn1cAxP3ID\\\\\\\/PlatXXzNsHO0\\\\\\\/fWFmH7uSGE7Udl+5JWTjwSOkUSC1vHCXZ\\\\\\\/nrAAHzXQtBbiRUM2udo8nwf0v2k4p\\\\\\\/oENI1mKJRt0qzcfVtTaor3P6Aac0uSxVsCzdrNHV9m8IjvB5V\\\\\\\/mbfy3Nk6+4cJji4GN6O7M+MVo8D0Jtyj0LQOeHXtCsCKR2pmdPXeWR\\\\\\\/Qtl+HxvcZszBMlBZTXnBIVwlju0u\\\\\\\/5T+gFXKEdn1ELMeEnRDqw5obutImv9UGZ4PARC2JV8UANJ+ttpIP+QKYtB4o0SrL2BAdDTjbcZqnKz7iI2dwaXVXPvWT5h0dVvdM\\\\\\\/H1FpfT1k8EsLDnbRioUwxv5O514wC3BMUr2WPnWHkzCDZtx2xT6dbo7IlQIi9Do6fvTDmFc5Y9MtXxCKxgZryfX7z98WMpen4Lf30kB5upix3yV0X5N6UVBitLlaMFuw6MVLTeohyXTrjS72GB7qmRFifLuKxDCOq3oJW44cSsVjCDXjebEwSC5ZTqWPRj1ZUKiSeq4a6UevNDjsg0G3R1lxsD7JbPecaZ0RYDbZqf+HTPGhJe+Uu\\\\\\\/PABzmpeKJaJwgaVxERe5cMP6X+njoTflEPOLm5isk2xTcrk\\\\\\\/kFOAgKLZKUmtqyO9IAOO2XwVMBz0Nk72u7iN+o8PHeeJfQpaDpw\\\\\\\/XdN64pc1zEsRXFsxelw\\\\\\\/4xmjLCU3CRCgbUDKEAytL6mX7IVJxRzHIkZ407aYyv\\\\\\\/lnPGdYwDoEmFpE7trzEuBxHao\\\\\\\/a2iBYaB4Oh8PK+dkw9NWupkkLJ3ejLGvQF2Ye2wva5CmsWTzmzu4oiFkBSJHzF9hzm3P9BMrM2E97O4eWVd2A+dWt2Fbl7s4mfsb1\\\\\\\/PtCkGYHDQoAqsNVatCXXn93O9cpeuLQ6JtNWEYWJ4stcGak6aTW3kyyWzJnG8QcgbLEPodfBfVXCxbCUN658GgU2pRuxvBwof6Cx3W3HKcNBlyZcoXgf5hDe5uqWdKzbzUTKhzpOVm58YhczV+Ps9ImapQc00gntazx\\\\\\\/WHONFZ+NVsDROfNzezjyQFuwTo1V2O0YxwmwTtU6hDrwRdlFScqD3VCzph8RiLxLPC3PqjuxDhlBvkiJ26jMypgy4BXOu9CawJirhBKts4u+DveznHloBLUOuAKfKLXMyQTdE150f1sCen+ongNJudGIWtR238S5ZJyzjt93Naj4ByY\\\\\\\/g4A1Q1GqSZtFOQRyb2bB98Fv5SQu0Bt8X3Jh8BYQDwtYYtcdurYjhZ26MNDrsDmMZB4aavd0\\\\\\\/YlTGhVuDfdR6No3DV8kBXNfkJ6x8AAUyT0A8zVT2hMs0+P2MI85vfKANl8hwpxexLPcm3zNhiloBQg4hwSgqyJ\\\\\\\/wVTipoaJk2MEW3+jxuXV3CEQOpFwNCeXK7ZCQej4tnCzqikwKtfLQgBE\\\\\\\/wmaefNlX9fdo9fKXnIDnE9rcDXtell8UzE+\\\\\\\/srqU0godR1h\\\\\\\/DMgiv7fPsZZWvm4wEVNYuKXrh6lSTe8SoS6beLmOyzxVRdhkkXHtFiBwMyXkiNhpEYcsKC2Tm3QHl8KRQxtVKktdBQLhNgS8lvtnYFxcLW\\\\\\\/EKKHgU8gLFQI02kBfShkp\\\\\\\/cCXw8dxkTbDRwJTqreXVqefSfKjghtLJ7RRWRPjUm+f38kr0yYHRSNlMmUGhofz2uJMvpe0zHlUPU4pLUDCBuWO1rxXsvh8P77zfYKFBVTbe+nqOOrTxhpu8pVCZI3e1sXNiuw+ZdYwSoO\\\\\\\/un0AERgGO4edcMhO9Nf\\\\\\\/KoUp7nCkDcP+zEwWuyG5notumwkhyxTYVulC4B\\\\\\\/Ry9GFT2jzT+tiHlYtazrJZ+\\\\\\\/rcFoE3ZHenAbxQ\\\\\\\/Hb0R6u555FItXjpXemBNxg+eWBPfTDM\\\\\\\/FMy11XddvlTYusAQOaTHjqe0NFmBI4rKZoNfx\\\\\\\/efuCbte8BEsfLSQC304HKgWzRC8v6oyouf3jYnnywRtoHairi9iiensZSaMi34+159QZIitRa+tdWhxHdH3\\\\\\\/xkbN9SRgvI1TrK1A8iyvlmIGcSC089If7NBv48I\\\\\\\/6tC7ZF906nz6gRFigNj0PV5stYOroR8LvtZSZEJT6dYNZzXfopECXLKjOhB5e4s0oOvWXjGjRsvy+Z\\\\\\\/B91Z9Fh3uQZUmn3+iwdEOYWgyud\\\\\\\/lvBdiI5gJzC+xZ6x4P5fXTuuAOyW1op9zRkxtZcLmxkGSMUN8wq6BouBNWiS\\\\\\\/J91rgGm89dOWP25fj1+UGKqFfCvqSRsYTidxpx45KF9gtbywUEx1aJNM9Zx1JGBR1syaK7sJaWjkQ0R8jKjy4XtABj9G9DY7BEWyzDZq+hGXPLiALpTqJ2bnBxDQNuBtO8J7Tuc+WERM9v\\\\\\\/PlN9+lKPVXmk9EU20\\\\\\\/2wZxor7iaZQhBa\\\\\\\/+IJnx+Q9BooFQZ248+kFACEr+JcesG9MU1yWCnLejHybvuIYmd2PG1zVruj5kTWhihvY0lbzFGUZ90bplrhIzhAy+nXxQ\\\\\\\/Ew7q3jCx8CfkaosoPLyrqG5jtY2lptDex3hNw\\\\\\\/DoslafnoRLiy885nWsHX6L6QHv88HdhnonMCg++ZCIgaCOzEiJ7aSac0+ilMBDMEuR2XS1n2eRmv09GpfGWZpzLSWpjVKVdpkRNOrJMvKy1seHXUIdlPMClA\\\\\\\/kYjSgMtd8Jz7uDPoUee1qAFwERalGGQa85Y8lDpYpzTgsk6KRdzShvpewoDHaCnbxY7pWI3YZyowSdJyU1AaW1HSVorcOU+YJhPa0YP3AqCFwvgQ9d+lG+Bv9eAb\\\\\\\/yqzJwyXjWJhSSkkPabbHKtAa2rWjgRBm\\\\\\\/Q7FPPWjOuezfUcWr5WGnM7ZzBtHicCHtRGjMXiYUwpqC9PpTjGlOV3Yr9h7B5TR8l8Wgjqr6VGKUM+rr4OUvCBoAjve9nmr353fgEl80IPUMKBlgDPiu0Fr\\\\\\\/blgOlaiAvCwxSNiEQCt0I455s4W5Sw\\\\\\\/tG2hAE2lpGsiTLIgChmeW4HFYue42acErHmABHS\\\\\\\/SF5qKlGtR8BWEFFjPwmb9arcAEJl0x2IAPLKsf5Gk8T6AO+SDB40JBu1mvbjl4smYJXJFvaPvxisPsX8Nx2DSa2x7ek65Kg5EB6rjMmxk9TVM7B7LfPT5CHzFDcirWtxPNDD71Bfaoess5CcK0eLOQ7dSQWXJU6lsJjYFqgUpk3LQvbTN2u84nSEIRSW1H\\\\\\\/pBkOAcl01K8Q\\\\\\\/IUyeYMLOhlS3T8jlwIhCH5iEm+gaY3Hka\\\\\\\/X6K\\\\\\\/lZv7fZY7SsFCshsFEi6DMX1sw1CbqKqey9NNKyJ0PaWIjqp2y4nF2LpvYdQQP5WOxQD\\\\\\\/PKAVxnBMx6tQxbfklm8vLVUq03fy3CqLRLiWk1Ylxm7eLkyxqPj4MK8se5nBjz0CF8A2RUtrnU5ustYqLpigxjF5HTcfDtRJP3xLITwadjuzfMjManu9thbzcmivRhytnQUNpMHF4EIZSBD2gmMFTZ6K0ZyNH0ERrbThwh+xXnQbeHRdHmHTYJ9dFQWLursTwqaDymt8YAZBQkymc4CBb5JwQNM7cT3gf+nqyKC8+grEgwVahD3cPu\\\\\\\/GaE\\\\\\\/Vtme+is1cMiDr13wlrC59LLw+Ogdgzo6fLoB0N3SI4xRxWGMcUPooWbNjcZqeAZ0vy+b4Zuu89a\\\\\\\/yOHT6XhbVpU4jRi66UeRw7N7uYCedKxjyoKwd5LIDVKST2o4nRe4U\\\\\\\/LJKR2+7YMlyhtK8n\\\\\\\/Npu7CC6amd5IKOIiheTzeda8HHKySyLak5o9RDZ7tAJL4KUOE12jfn+QOG94Ek\\\\\\\/\\\\\\\/2GH89MKuwZtnM4uTBGrZEG+Md1nY4VuPu8cTED7qhOpus6\\\\\\\/iTtEtPPTbUICbVhwrlGDj6UuuMHjbOEmhrClaWveZAWdktH17JKmSC4av5m57crWfPps1E0Myq3e83rHtzzoEMzxZaY+jC5njn3xkbg4xJk3k536OEzz9qjpCCuLPUIwvpVD\\\\\\\/fW6uGrkUsCBnJq9GxIbwppMGpCnbYb1xBFYPCSwij7x8XROoq+HBwl94uApIIj7B33Y2qxEPLPWZCr8IQcD5OKD9fyVWnC4x11LOSUcsYaEfj7FzxEjMPOsrHn98FxZH5AJ2uQQmmLy1aX3LUy9fQaJBFbomGy\\\\\\\/f3OynVxoZkq5KrxMpvgdQ6Gf+ERocgeUi7bZVeDGjS8pew++5rO0YAvR7ORuCVllFN5syiRgav8hPgNEs\\\\\\\/iTd3fTZb\\\\\\\/vdhYcXCKuFiSKoJwgUPjBQGSscqus1+H6cuJ7I70UPTyP3BXLrM3yr21pGRaNyi0Li6GubbE+blwiW5Ddwt4N5uwK+19OL6g5MQi7aI1fWxKfTnnFaYf\\\\\\\/OyXReThWOtHuXO4Rv5WYaIkWgDD0pE\\\\\\\/OhxMJv1d\\\\\\\/HHYNjkJ6crpHIE96zDLiw+\\\\\\\/M85uZpaBrCks+yqL4TMyM\\\\\\\/pjqd8rUvNN0FyUT3F63chc5w\\\\\\\/DhSFE7HmXSKwthAzRcCuD78ZUOIgM7PhaC98YRCQbBfyyRqhlg0WYza1hU3gLk0DQEvtoh6QMmRetSTAYHainKPC5s9T1w0+NjAKZ9dUY7M24Jp6Rxq9122yh55c21FEC2qv9OEih8qZd6xbygcBTP1jZvK2zKiKbqq4DjPBDvLX+iMSQYG6Pdjl0ci7sAIOU5il60lyIPmduncIyMMztPrdXH5xNslj2v+7kmuAvPKar+9Gnl\\\\\\\/904sWJcTdggurdaX6whV6ngtCOguYpe14aW0E5kr9HIDkVs50tH+0riwuILoSs6loU384fkjwE7VlqGJsBKi\\\\\\\/fdraP3e8hhzBmL62e5s5JlJLVi4wssLVpDStqc3aL9jkg4MDr3G+FiwvnpS8DSnC0PieXpzJQIpacrh979vsjBXIvP9VqyuGDtEPeeNyvRji8HrQzQyoK4o6sLmNPoZr7euDqroch5DytW6cbOQSJf8ddUxWy4029pFVy7Gjmo85G35iRypJZzEGrnRksZfIV3zE8OPqad5EpVzWV8+eM44kmgvdGYjulq+E8fjPFLbSefa+VrJHVdJ8Dh\\\\\\\/RmKhatmD7crtoGCM0HjR1V4\\\\\\\/mOeXoSLL7zBjBQQUtYDM\\\\\\\/BgFWy1aCIYiPyGOLoJKNWsIFSy90Iz8h1dfmpurR2XRy69TuyqQG15bucpyAs0wPRsAei29SteMaeVsk8Ce6xoBNYZinEt8fLgmjJ1lQc5EIbWkvVSQlUxp72KvldTfKp0CI6JkmPL622D4LpZ0paNAS0xR\\\\\\\/t5TjcitpIUaWLx+7kUg0NhA3OibmrWLUROLT7CT+BtGVRQIB5kY8cGISJOhlfL7vqGUn7WPCcSscUld7pECcljm9hyBFtuYDFiK0bVaz0BmKYvEhbhqGjieIEgj7ALfIAVdn5uNCHyvgXKYA0Qjh4IHnk7xSonr++Lwr8b3k7QFcYiRNcNlhOQZkl+48\\\\\\\/QiE\\\\\\\/Jx\\\\\\\/zy+ykapZoRasRt3ERx0MHs30CEAg7698NMOuKng2tuevzN7DBXdTrQol6coChDXv0F5KyIodMcU8j8Pd2N8xn4Y9qAX8hM9UNQfhd2AkbyLpRWYYcINRbWiTY1GCapKzIPgbvrSptuMDWsxec9aE+2HFCSQbFZtsLqpnd3tpIjJpG8vwihWElmcKgqD0ip9eFHnWYp3eAzAw89ZiBxDLgxyIAfpTUx\\\\\\\/E1ZPuuJAPixYZYi8hLxCg1z1gXHMbAT8DWeYJmVZT7hyAs0NoAv\\\\\\\/sZ9by3kK9FaBcrYvc0JbEN3N2SuyVShOfpWMI\\\\\\\/AoNMh31I+ATPWLE1WrgeQ8I0HTIa8jvizRKCMfgS2YYF8DujanXsU+FpbRtGnGzuSA+hjSG+I6BJF+dhjkeOCWwUN35VwqFea5b+mtE37WQoi9vsSOrWRzPeKlef4EuSl+GfX7FxX8x3rkeTK6jxRPPmtzOKCiGKfpDTCboz4a6ClCf+a8wHqaVrHAgkMGWpO90FbvGpOP8mRvE3PeaReHoibr46IGPnkRdaxeeaQltUdxhnkGmGkbqQ6xgTlEPg7haeM1CFwKvRxmBTQqMfvdKCnYpeOC7\\\\\\\/aRWiuLIAVWP95nVUrY0EEqpWkq5xb\\\\\\\/L5JSWbzlghvSFV04Z5J+bdARM4vVXuBLydIUJf25yWUtaZ66NBlKX7ofI1Jf4Eqp8skwXn6nRUPsEWt+nzrMkklwXHQ561JHQ5uOZoh1+nGXODxQGbC82mYscWWIRSC7E8wX1WBK8mApjcm3yb+Da0VMxQsF5kPDdYfoRqcXGhlxNKoimAPB3sCdJY2t9eA0LXMoed0TitfU9yhTo3A\\\\\\\/XSEv2bjbPIF7DoPe7Nn5VVvZrOEOio4sULQ0V+NNUGvjwehaAK51QDOD4Qu7Iw0RYRdM03OywIUsq1HE3nn89zAIh7uvnHnHWCcIIIc6rlvX1D2Gb4bBKaLOdCOGOwcjNlD4eb2oFue+qYQTAtF76n+sNOXW0DK58FEARBOvDzhcx1UBIRRGVyNhz5FgmwD+0LK60KiE0FvY5QZJvLezozqtlvYZhVZArveOTziPsL3Jq80A2ODm2Dr0bqb1TWZV0kRZTe0o9unb9tUOICAdCCTtpJLwBIqXnvfVmjfYnhADKB9U83l6m3fp+wjwE44\\\\\\\/wUfQGeopvbajstpH0ezbGz4cPR98SCrby7JQPJcRToN1RfQtTYodVeWZLkYNg4X5U33RyO90caLjfB9RL3JJZcFtY6otB6BDaAULZrYXRQo+gtImk+3QmgDUzEt6i0HkbzFv9YBJdsE6RfYnCl5qJKBc5VgugmnpvU\\\\\\\/tK1vhHXckw\\\\\\\/fAQd+3+eJBNn+ak6KcCALldxW+NqbpKNtef9l06MImwM9JDRBrV56v0KgwWRtWHrB1WIXYTwheyg\\\\\\\/RZBII6N7t1YvNCbHMlZgueaIzymjpUPGDuHVI6Sg6VTjyfKO+BQ49CAwGeegUWtT\\\\\\\/fv8u2Jum+dWdgoiuMPrGzhn3NHIE3mAxs0Ct3QLiuby+EOzqTJsL7Lt5fVizlNQ84iZ5QbyH4tSVnkBLIPOa4fsv7EFhzsxcdikLgubFH8ESKAAh+0J0C00cMA6qE+hj1qdhqiO2lfi\\\\\\\/ZulU\\\\\\\/D6AiFxBJfvuwh7HxKMVcjYKMt\\\\\\\/32qm3X0a4w+Rmkwr1QFQNzJohkjY5kHYiUGTdgZZPwaaJIVUnPyUrXZplYdAecKg99GJahMZ4xvh1Tw7v4\\\\\\\/z+8YoeOsR1RQeu\\\\\\\/04QiSkLy1zwUpL2RdfjWgzRenm5ofhV29I599dpFxvhUXVmkkMPlzJ6KqcGJQhTXVSVGsdoQoNxxeGIIs71VnOeEyXqMZafAUmjSFoMGc6o7bF8ffQyBvna+QyfThULsgRWqy65Gi5QB\\\\\\\/SBrK2CJZqTAaXDULt41oRpQRy8bnmtznxb6eBblqBHtionguP6i\\\\\\\/ddqw\\\\\\\/SjcS6R1lkVMLJmlviL3ObP\\\\\\\/Wgd305tzs5RBJnx2ecxJritYeFTkk8IuAV5asAKRDezV+1E1dVzkVbI9Y54sCnoTtoF9Hvfj+5\\\\\\\/Iq2NwccBDSY81q5ie+ubEBPHtam61gK4URR8iw8iODcuMaKwXmFHUzyV5UCglU8z+eIJ51USELDok+fr2S36JR6ViSwRWxWedr7U7o1A0HoaRGw2SKYc7LBsQ2jFurJjiuexI0mHNWH+lzEbEcEPLK\\\\\\\/oyrixWPgs8nZXDVDtKsPWmtbBWqkDyT\\\\\\\/+x8Ie2FyIlD8pcGdFuRwpspiuKgccaSMFZSEK6dbUlcHdCUjTQuG6iI8J53W4dVPjuQ4ICfc5Sy5qxI7iHVgegknmkZEuiZ8rsCE1A4L81YVhlY\\\\\\\/gcc\\\\\\\/NfJ2ISIK+zuCXXJOxBX9tzLKtohLjm\\\\\\\/uxnfRbdE04du15Y2q7RuwK7jlbiLedobmcrfIVETVaL4rTBieWBeqSydYJ1j+Aqycrfiblc\\\\\\\/XsKCcJsJl8GK2Vc3KpsbBhB78azvctVU2hHrOvXUD+c\\\\\\\/ZimfJoydpe5qFu+pUBGCwOrIo3JDDLsLvkKjciUc2U+dUjjAUd5626IOTfjUrc+ISaWIeoSE51o39X+0Nd\\\\\\\/HG59dHWvY6xMPgFGWJZfGYtkhg8R2G6O5EdApWyU7CQgEeDwNDd90hSfadQ65rorLMTii\\\\\\\/tlrr0RHRwv+VKJnNWYdYzLZ+b1kvUK+78LptRaao7zb2\\\\\\\/x4JJlTRXzi4CeROG1+aLL4g7Ej5SRG\\\\\\\/swI97ssa9Iudv5ZmRAls+1UrzgcrJoA4MSOHNUVoe6DTXx\\\\\\\/ztp8gpPFA4rFZKGgk7QhIMsdp1YlEbc4EL\\\\\\\/rhokvSGFT6CIZxrqmQSyRDTcOR33loT+7IObSq65GOYaNj5SjdzmBh+4eYbfEyj3BYmw2hbcRShQkgdw+kpX9OfQLU0TUujGxsHl4Lzy6KWSEsf0rTc\\\\\\\/rPu812ge6cqgSiipgS8MZtqTCm1NUZYBq141xktvLowafbBF0vZdtzYI6CCqmEtbrHmYQDmXpTuvcyrdJfIihwz348GgAfJClaCU1KYFIly6cBnRXmr1Nk+bMDh1K9uoNIOaZA0ChLXkjEBgdRDQLGo6CE6Mrc\\\\\\\/\\\\\\\/IuFDHGKsngAqHUh6t\\\\\\\/oNvjqH1D50f4K5NUAB4SFfD3fpy+1024HFOz\\\\\\\/Sq8AvlHMXzI9kSaq+OWdByPkgkc3mk9rnYnaIpBod8lXQas\\\\\\\/buYQgPi2iXPrWw6O9QiAmkC0RSVq7nPvtlyyV0UEzD0K5K6BAuILVxYIkSaj4OaxQpGsOIndnb08s5ZPH79fD1qJ68ssemJ0Lw\\\\\\\/7LMEtEStp5BiLKajqD7USuEcQyq6U81W0BrbEXEvIY9uD8w8yumPhTYy45VLmBRZHJ9iEo+VVCHX5SkSGVXa4Qtwp5Tt+PqoEDOwm0uOZEhKikzuN1xta5hkk0oszDIUnI1Ovwvau9oof6Ha1B1yWIYlxZSKWvrv1DUno0dOaBn2iGykfQK6cqY9SaUQvU\\\\\\\/F7p2SHt\\\\\\\/Ud3n9RXdk+ulV8ci8mmuRNKYSmhHmUMgMedu+7p9HoY\\\\\\\/lScIjTIibntOOxyonpp\\\\\\\/PUINBvEQv1UBwjsqlUX\\\\\\\/PqAqRzFxaQDhKWTBadJ8lwTQH8\\\\\\\/Ow3\\\\\\\/yfCoz6ux5yLUrinD4QYJUnr6RflTXu9PiqVDyisGA42\\\\\\\/28StuREIOL290zNs\\\\\\\/EpBYc2YRpSeAZP8SCMonxlDIXHXlhbE3HHO1iVO978lja4Vn3a+jwPy3PlT8t4GX84M\\\\\\\/o0Z0wK82DVwkZjEMBgNqXukGo4VBfcjLbgUHX2G+eFZshi8GShBcr0vZdOuzuijyP68UPNA5OnI7WNE\\\\\\\/tS3th2s0McyGnO5i10iSG1QTPxHXJA\\\\\\\/GJOi5FnpMB+nfN\\\\\\\/6QttZ+tDvka3NuI8M\\\\\\\/zbwT3UoHf9Szl6OMplSH\\\\\\\/IrXa7Acma70sw5m0cfYq59Nu1xNsBL7pKlTienR6WvqnfHJ6mVfjlDZ8E2of\\\\\\\/GV6vqPZBg9LpmTEUVgW+Bq5tznVn058\\\\\\\/kT9Pn3\\\\\\\/OC7s5EF87HKzq5rnFum\\\\\\\/RaROJtDxXejUqYCCu7F\\\\\\\/J2erVH3cebeLoT+MA9+vqtYbGavnfGVULv+xE1ArgJI1PfCtMV9Kh+NRIzcPIN6feirdwSkrBldKJnU3syaLdXE3CYu37Ea04XkhhWr\\\\\\\/8Vntdwk5YSWi0SC46lwxnmcdGu5unl9SNA39SgwPp36hbzNEPEMJPIgiKAmrRwWCZmNqeQdTrnmkNW5duLSKheHhE9IkNks6nES0y8DllW8npaECfAMM1A1ZsLMkAPG2gHX26gkC9QVRouMrjhABdj3cdN57WdeuYn\\\\\\\/3gZ\\\\\\\/FDWbxpLQI7mH4JBnfDRjclvI6WwC+Sh81vIza6MhdglDCLauH9IrkbwU9lww3XyQmA3GH+Xj8ukKWRh7KZl5c0EOunzF5xJsMsCEIakg9DE1n25BXitQTtvNb64vj2DEoVXrpELgYplY76J\\\\\\\/ECaq5f2OdX6YDK218KBrgZ\\\\\\\/NCUwvylUW+kf0lxCHEbkZXReU1cVgCy0L3jF7mewwrhedyyBmOc+rdLZ3ymcmGs2ECEO\\\\\\\/7sfr1wEFc2qy133IWQMkufKWhTjirPv3QRuwOevjTqYVFV4Qji2weMXJ0Db6yOWYYKhXQsnB\\\\\\\/7pdE085MYeE+5h8L+cOoGb2wDFXomLSJPOrrt9mWRDkzdL6kvjwPgbXD1fwaGXyEmw7AuMx+4GEhqy6x4n9Dj2ozIRLMoL2Uo1H6WKhwOcMBJZjZ8L3B0YVjkjuETNK6kyGt8uhmvaBa0OjmR4to0LE2Rc9MNsvj1YmyXE8Tqw5+iu8m4AbpUGcSxBj54McM+9pSaC1aJtNcSNFGG1F\\\\\\\/ADurFs4OH3vipCt4LJ\\\\\\\/1xURS5yJtfIWytCzhj81cvv+7\\\\\\\/\\\\\\\/Zd\\\\\\\/R5sRA+ht9mt+79ldYwGJ1bIJ0B441DJdWFyn9R\\\\\\\/aNeWLHqj6W6A6ryEH9JTrLrj1u5oAjNjnOcO9ly5Cz\\\\\\\/5UieCL3VHXQAC18Cx5c9btYdhDJwAxTswz0SDZSDQD9yDhvwGZ8g4URVdfaiHwN2cv15zXtcBXCeTZNNJYh2qRq6qxn2DKeMZSO00J9HKtEVItqF4Y2SO8QvpjRtJtcSlbr5z5yaYLjcTlERWOqbJX2fc6hRlzMFJYlZASLw+YuWwac+hEQABbjtN2KCnZHWrjBmTU5dwh7wYt\\\\\\\/GKlwHQC+\\\\\\\/Bghnnmhx5h7Yy+k\\\\\\\/qpOPzj4y84Gk00hKtC5bAPaGBFr2+pcG6Vm5MJFQ6Im\\\\\\\/BBAOlx4uP0khhlZWoJCD81GjGFvZK\\\\\\\/NaIiijY2vmn5TvON4y8YzZAr5oglT7E53bYWTE3iamMgm1gC96JCeGZh7sRWiBGlwO5jhUgn8P5LdL6m2osV0U7mK3au2flmz9\\\\\\\/2IOTF62RiQo5a5b0hcpXu6560Jit0atdGmbtcKND6MEb+zA+B+TixUmsyPLrVVSwwnQ\\\\\\\/YAoBhKfYEsu4fr0lAF9F5EP7hDT4RVW2\\\\\\\/9G4xY0pbOm0nEmSsXKE58dC6G2CysiqOuudTGOwSjUMQVh\\\\\\\/nlzZlnQcYcONAfRCsvIChBzbu7N94wNgxG6Hmv9M+LTEx6r+gNidFVHD2Szz9aq5X7CL0Bd33lLyd+6fJaFXFCWiDfy4dnCUHd+PV4uNCYGOj03hTbkENn3W17eqiU18JM55hcvhXQMVdUZo3UZZGL0qsQ+eA9C5kMDN38OPHcExdEOUrdq9wPrRzOAUy67NRfNC+GlsxIg6wAx\\\\\\\/PIhhE10ZJ\\\\\\\/cFbpDmZK4ghVFD4TLlzjFr3oo5JTS+Dlu59FtiDcOMxh6z7qFyPpBBt2VJ9ikrbp45DGtMLq35gGTxIjDpsDT8bRJJTNe83O5IXzqPq30tcSnHkm2xwkbMDIG0NSXT7l8veYhc87gQ7XkVodxJmTTnvViV3vMQFPyahmiMjLHpAigD1VLJDV\\\\\\\/pGGfmkscv9av7QWOdXcMfivXcgNq6tLeu5WTHsLVn2UZ9ogXwbM5L4xqr2PJ1W1cWXLXbEEGhO78TmVjZB0BaBacRFMlqFM2qQV2FmbvREdqNhSTDV2vTE8D1KpF7EO13K4oHwR4RI84wPJsWm3urK\\\\\\\/Izo6P0ygmFhb57L1YGmaSw1Qw1aWiQ9FkdF6DWqcIxdoJLzCliI40qMCsqCBsJ0JzNVF1LBTU3gMczEGebDJxat96U3OgPuFlJg\\\\\\\/va\\\\\\\/9adpv\\\\\\\/H60fum85V30uWDjjVcDn7Bh390ouRFpgvCnXxXASA9QbXdY0HGZIzOb9mKjPLBMovZWJ3PJfU16voqXXXh7mNcXFVPF4ASIj0ZHo+Ik33nHFYrx+rUYKGpIf8hEWsYkNPXOxPEfQ\\\\\\\/sMsWw16Q\\\\\\\/yDqa8wf8iCpjn1LHoMAmXUuuqG86LALtdf4ETnahdizkljQf7MkxfFmQaxyQ7jsbH1DJYy4gh8a26nPOqT+4HFQX5xZoCICw1IJ9aYjps\\\\\\\/OEvKzvaU88qX3nXXYqN1ltVBZIS1qCIrxvvY8wfFHq5nmgbe\\\\\\\/25jmvQioCl1fgnHa+9oSAR6ZKOKYcZ2uAlMZN72Ygl91Ph9wn1peSADm3vTM6XA8SDrJRRI49+0Z5Q2N1q3g5opGa+PGg3bt68DTtRgxchNme9JvgKSabSiTTk85WbrfEpFIe4Sx3LAQkIzEA5w2F0C+feZaXPuVfXG9PW+PkIEHPg9Gt7PvBB6RudPWAQNpbH8SoW8b4C3UthwcsGvmxOHlMXPDdbM6LbtBdQWofgvbvoWliW7QuepkD9BKiEc+Qj+zQ02LhqZtARykb+rkpQvlBmKm7kTczB4IXWeubkxrebwTD0M\\\\\\\/W1WYJmDTl\\\\\\\/W0g9cUXzNR6uA+uyYsDnrdFW+zwUQdLz++CtdSDmo6z19cFbMRSXxGnXpOO2hGQobcnaFAahDI+LYpFmvxcnbanNh1lFrO9ljCQySgSx33Xzk\\\\\\\/75s5Z6mJWSXVgP4turDsGfkNVOo+jxSQbtraHbiJ9jGDc9PXEwFSbqJe085CDO7iZgJ94YpXdbxu9HizANMUpohIXAcAeUlXktiHZ8iPeraW3eeLplNU4nLJJ8Vk7AKjfteIrdo9qFXzZ47ng6MV7WKqKzqlzIgXsJ3loq0BNExBUVHTqkQfX9kVpMNuIrzzDGezTlXXMnVB3pp7IXaXro7g9mNw0cCVuo905Mqb0NrgVEte+768u09HB2497lxwStkHUAWHaOVol1a8KHV9XirVwud8tiA2KkgT7JPioZ+kJ0QMpqsOB3ylyBRHybO3t4KeTMcO+bF4bFhl7z9Lo4D7siwxP0Ku7nJCwdwugLbCvpwJh9GZfI8gz22u6Vb\\\\\\\/tNqLsTvQdwDG8h4XEl97TBQ\\\\\\\/SoV6FNEsgB2MXzkWPuvA2H0lgi2RV5n8Asj5zvmUvmtZXTpczcGpS\\\\\\\/QiZwV6xdYWqZCPlPssjX5cusB65Fiv6knX2WhNpKAw3BhDmrnNG5ZLOCm7DTDS2cgqsiye0p5leq94fU8oUALKPXtAeKJSV3S4dA5r8noItry+\\\\\\\/p23fgzEeuXoLcFVd4Vqr4dcOw75xZYqA+cn\\\\\\\/ZN0MK4FYuJ11GkVRup3rn0bcZSxY8n07qQqgj+dEmLe8lrsXA4e1riOMr82ss3DFCI7ViTkz+7S\\\\\\\/tyAWtoXQSSrQXxW6wi7qNKy7Z7K5C6nR0D\\\\\\\/uSRirt2a0DizW6cbsfEwboXFiyO8m23C9bxbWBTLTPC8RbEFSGu1UL\\\\\\\/bWmiAfS\\\\\\\/aVPi1o6N9Ba22FZCKKSk40R+O1t93xir1C\\\\\\\/iA1pjpBadXwsUtrm1pSAK\\\\\\\/ApAual8T\\\\\\\/+njarj\\\\\\\/\\\\\\\/HuQLwkB4qe1ncG\\\\\\\/IyeS3tGGUPz7MsLcLOyBtzM8gXbQRhkKLo5dgYf5CYZR1K05p1YR0fPMKTlKtwz3T3z\\\\\\\/+P5iZ0\\\\\\\/KaiOhEJ2VM6iGg\\\\\\\/BJ1hUXt4WekHr7b1sD+2lubnquUYHaXsJSuoqumbkTyXaKdod9HBhMyiPc5daF6yLHw13pw+lmSSpDzTi5zGsFMnaYsZY\\\\\\\/KlW2WRMAYPf+lPVxe\\\\\\\/yLfWyyr6S2Gh21F2Jtk\\\\\\\/f+rAPxIfFJPUk6fWbP2oVShDf9pkYwTIIRtse3E7S+\\\\\\\/92uGNog7zYoCKpA2OopV1asPeuRyUEXG0hMKiDGepe93Gx\\\\\\\/jo2RobG2jOyFkCYlf\\\\\\\/bF+tZKgCA5RzSWKAVDQHbfZ\\\\\\\/gEgNVMJbUSTKU5d+Ji\\\\\\\/K0qo6QO4Apvc4A2HNVeGGIpgIPhzP8KZZQfEnGYFVpAknRKSMeRzAWCOAMfcviAHbpssWW0k5\\\\\\\/BEoxY5ec9gQ69bOmIzndnj+qroqZDAZaR6d1nfrULdEUTQGYQYAF7hN38g9Ii74TiN6ymapiwHkDb3VjjJUxMcI5uoc6x6wp35MRsD89Rey0y40GH6tL6wR4+z8rMnuN1LQNeAz1zEVxszizS5hS8MXaTdX5QK0MWX9ZW1ZiwbsJo6sxTDlo35z7YD46QojoCndTwQXDLMW336V2Es+Dfmsqz2ssKO0goRN4ZIZgjGop5vy23g71aXkXvtV5XvH\\\\\\\/iul868ZarO9Mlt5S4pNoHExha6Yo3qydxvAIr\\\\\\\/O0HGvtTexRvdxyZYl\\\\\\\/in2cupMuJ4LHTHiUqe6UmhuEchAJsbAM99dfNCTJAw9nzEZA\\\\\\\/ACfNEU0xfA8B+vIPO4XyxBKNsKV9oZma1MD2K7TKx8w4WkPE4w1wFPeAjd+b0Vb309gkx2SEFkFcMnfgdv+7WQk8wpI\\\\\\\/K0+psOoQl7WGAR0rLU79OSf9ovN0v3SXqHwr\\\\\\\/wmR8T9WvgsmAGxppEivd1uUy9aHx+\\\\\\\/aQUBu9rEuKG3aqLhjZ+1cUnar4Up5UdXBlDv+VyjSbBY23MklrGncdI8dMjPvMS81PYnkiu9YXdylpx6cIXi6Gnuhqh3e4dUhLRVAuUHecsgwFAFr+GQ8UDKll4uFHqV4BosJfA24S8CZ8myiJkuoQuIDKhPvNe7fefBKkIAJeWRR7g9lVxzNJdCQeXIugxifOtXehokuB5k5IGrzcdiXXBJj3vhRsSmkxTn3So8wagEXOWL3h6VTUOL16kZi4dJ\\\\\\\/C1MiuqqOD3NVX3c1qMgcnIMaw75Png5y+sx\\\\\\\/FbOVIfsOz13YpYdodCvrgFg2abBbUcIFge9afE9zMU1A+xqgHTxr6WzxvW\\\\\\\/vJ4m7iRr6DphKdCvuqTFEVGI\\\\\\\/Dyq\\\\\\\/lbI2Utn55izTT5J9pxPp4nBZib5YLbI\\\\\\\/FvDniLwl\\\\\\\/wuxYt5LYwDz0ye7eAzdLBHN25FAAVKnB3LE4vOYmABGb2KH83H\\\\\\\/rqadGkzmdwK0Xbdv1PaYfkUaTi5HlVO506YCBiUu2TmAHI0L49cZitnM7Xk3KBR+yoM2smtueA8ngKQtbzK+3xLcXzAJUmyueKTT5ryNrNPVCNSFj\\\\\\\/wa3cex1m8ScAmX7mjVYf2PpjQhq+B6jrRi+wwYb+bIT2X7aPKSBMsJtXxlWK0E4PAHgB7\\\\\\\/68zYBQxu688zJxub6gdXO2jRmkwGEHHcvOyUaPBTNJh1KqbCyYEMyLvQl0QyR7ZLTqg7KS1Nrw4CdlZxKklGS1Z1sfB5xr0oMqz7NbcMljtfeceX19w65fDNbqA9zNTSgUyYHyiwT38Li+KLYvtcRKHjyxBR\\\\\\\/Ww2LqxJAzEpU44+W2GHS8ce\\\\\\\/rUfAM+uHo2T8jbJBI5PNUhwuJiGz4s7u617tApONA24TwNDXuu3H5VM4DyqA4akOFXClulZ+3ZBeTeJppQlvPMSlc7U9N1\\\\\\\/yWJoyJDh2IrLAJTbD3H66Op+heb\\\\\\\/sVNKbbHTMJV0\\\\\\\/+3a5hZHpy5HBXbngBBGTRPhLzdXMTk9RbkqznEScUp7dmLl3yxxx3MvCRxZkPYwVSv4ixGrd7rfGmRSna3ta5psV5wNrLKNC7OzZsbHuc9tb6CnPGqE7E6kBqaew53NGOtguUCAxtVMeXOlUzmQkcywloXuapd0wmRji0\\\\\\\/5Dep2F9gBKwqFAahoiZ7ZScycJBQ\\\\\\\/EL+DrP8fv11EFNahXM77SOiy7nW3V6U6sha8YtLdg\\\\\\\/fcRLR8lxrQzbQFAtAHClkqgsUKYEieYJfC4qaTetqc6rO0l2xb7mMDHWgIsX4mOXM1H24nJl7mBkgXNEJ2eKTkgHwpvMBbjitz0DgFwj71TDkIfR90UdsOLkhfr0oqNgtYkHeuxo4o8b7GukokxtkUMClDw7CBYiyaL45l8V9ON0Uvm\\\\\\\/ph7UQ0mrwCHd+f3MtOppq21AQOKy0Frlx7KofwmwArQEhzyugG8CUwi9h1L5HQmNUigxwsK\\\\\\\/wIOQp5Ebq11ZBIp19lYQk5+D8bclQIxS71KFHkj46bsVMxIH9h2sOZF9TOkwhnlhPDlXnh8UT2tC4VpGDved39mCXNIUpB3reokf8wO8X34HYgaDyq0VUes8cW\\\\\\\/0HEAYAaSBDeAVlwO956q5J\\\\\\\/Ed9zzB8fSduzYiFln7\\\u0022,\\\u0022iv\\\u0022:\\\u0022NYuwjv1lBjGwemhDYraXfg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226a431c532553945449d84a8a478d4c014bd9272b451158333019e190934e3ccd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+YFufXiQjpMMb3KtwaRvlAC6dTxqITKPuuk01Edjls94Rj5XaOsX6lAcua3UT000ISihsbReLF53YJwCSk4kV1iTjjaVWMneRyR5Amz8H3N0RA1Tzjlnzd2aKejECLU40rGHRHmwr0oPGnY3c3R7ykKaV50JljIQjHluQfUzAqmTRBKdY\\\\\\\/dS423wYM7F2hfUtYv5ages3gRseMh1lPxmKZYF4R0h4rpduwQZps2h\\\\\\\/VUC3XZ1Jf8AarmgyrE1Cso4opINJDREq\\\\\\\/wPlKN5IMSEcMj8GVAd14LuXYh\\\\\\\/HWwN3UPyP98PDG04jAauCsTiiL0\\\\\\\/3L2PqgqfSlgRp+NaKfcqkfZyLmS6cw\\\\\\\/0IubZd5JwNues+1fkg\\\\\\\/JqnQPii06isG1drktVhBKo8Xz9KidWH8jQ3zgK5sEkhVeag9ACOX8YvplAxnDc9nO2VNL45n9U9NgFw0HbWlyNzoKMnLHw38W1PBe7uZlk2COlL8dJre65xtRomGcq9GawB+WKScy4p8P284jg4JEnMJzj+Lb6lWU7rBWLx3y0PXA7qWR4vpJv9wYFs3OJVnJKLbP1pZeaJvLSMXzEZvwiw1oN7NzSkSSDfJrHmt5aIhPIKYBPWdpsYgbo3e8uXdROJq8+HhXp5ENnDw7oX8tE+aM7G1rnWNXIwt0C4NFs9aZ85f76wcoZHHbR\\\\\\\/Wy21KsMWxchGFf0xg4\\\\\\\/p5nIyEgZTrFAwKNHK4BD1nLA6FL4WNkKmK5zNHkBNrLwMZKdC9qVShmcVMkJnGCGL\\\\\\\/NAHmJq7l\\\\\\\/yF\\\\\\\/YF9mibQfnY1\\\\\\\/UOjB4oiwrXKMXV66h4KQbr+qn7XV67BSZxzWY539u7jgVQO5di\\\\\\\/BxPjjQXsDb4tCLGt7GCZ8sDOy8rsnF6j0mVKXPF0rzlr+jOwMyHbOKXLPIVvt9uRNIvjdnXxnFwlh2SU7RHArdC38H7YGqR0Oq\\\\\\\/DvZivmNOwB5yNfqiOpKqX32NCI0O6H8CZy9xLSYXfW0M+YC\\\\\\\/3dK4EdvmwsC7pR1LGdXyHxz9Xq4OiTtVajQxJXz0x50lWPYt4z0TBBnkrrLeixXKjHdkh6DVvit0h6oPSpsXz6amx7sTS4lQsfG8sMnkqKc1nNgNRJpDyfE6O2dnm1Jll+klMcz7dNEBWfBvZZ96t781tChpR6hpviffVc7ojIgqWeySWpfxB0640Ms54hwus\\\\\\\/fFm6qeyvBf01eHG3bXCYvkByaCA+o\\\\\\\/xAWxWDWKUcwX6Psh9em5+pCdqbes2fHOTC3dYVtc6zuJqYA0RCTAug7EfCHZaDREVvvv4vslPRDkNRQcJnGpTtzLeuJCQKsdMPxA5SDLZsTnSjzHClNvibNruBJhy0v3zYE4YdoIabMwUPQKiKhuWpXdMx\\\\\\\/qoPY6HtFFuizaPIfbczYwREKDQ0Zi2Kt4wVXmLR6eXHNd7R4r5n6qJ8mzzhI5Jq0CaWUU24BFfNccjoqrLl+KvgPq\\\\\\\/khTaPL1rbuzSdd0UuFU1jhaPoKZ3nuvfqCTtIqllbqEDwqZaOL7UT1u9B49qq8b347NcwYFlLHhb9C5EQmHUnDSo7vc2p1mQPtPstF16GMI1Of01WwmgbweTseTDGmeM46QNGvfVnN2OXjqIEGfxPH5toEi6CWcN22LceujILvd0VfUHG9L6SVncEtI4FWqKIf6m+2G3S+Bp4sGRHjqgq1pgG3SRS4sW0AQejQINTlPh0uC2f+XSTiUGLuleMjGL6xyk93jr56uA3oVZODSfyprdrEheoXXlc1z05KBqJqTI35H0wYQwL9sabKFZxkIqlyEF62Uo00GV+gmTG2qtrEqL1627LE\\\\\\\/kKC\\\\\\\/+gxnIUVDh9uU3GaiCG2wll75DRH5CV+SX5JFVY5524KfPWofU2fQvsFrV5\\\\\\\/uzPkr93Ln3TAtyn8hS3B+D1t8cGDdpE5M8KCxUmrg1JEslo10KajcZhF\\\\\\\/\\\\\\\/C1yt+6pMlYRLPuwsIFfYSpUBt+zmUXaNse\\\\\\\/PY6JeYqdrGBXuq3Xy9uxu6w287tI9rEvXJWRGO95ocMYlO+ErgNbAhyZevY=\\\u0022,\\\u0022iv\\\u0022:\\\u0022JyoxaRqyHc+GHp0PBSngIQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022da779e3f13fc4a342bc25a1bcde3e20ccf22d737500b3fa1beff90595765ed98\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460402 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1853" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cKcsNdBlXcAdm6JzGeZZnbTU3lBtnDTexrx3D0pV3Cs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1853" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-D2_cebHC-CQtMnVGDPoiJy7r_rHmSnf4A7xRvG8q0_A" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1841" title="The Drosophila melanogaster Muc68E Mucin Gene Influences Adult Size, Starvation Tolerance, and Cold Recovery" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1867" title="Copy Number Profiling of Brazilian Astrocytomas" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460402" id="node460402" data-pisa="ggg;6/7/1853" data-pisa-master="ggg;g3.116.027953" data-apath="/ggg/6/7/1853.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Degradation Signals for Ubiquitin-Proteasome Dependent Cytosolic Protein Quality Control (CytoQC) in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew J. Maurer</span>, <span class="highwire-citation-author" data-delta="1">Eric D. Spear</span>, <span class="highwire-citation-author" data-delta="2">Allen T. Yu</span>, <span class="highwire-citation-author" data-delta="3">Evan J. Lee</span>, <span class="highwire-citation-author" data-delta="4">Saba Shahzad</span> and <span class="highwire-citation-author" data-delta="5">Susan Michaelis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1853-1866;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027953 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460402" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew J. Maurer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BJ.%2BMaurer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maurer%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BJ.%2BMaurer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eric D. Spear </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BD.%2BSpear%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spear%20ED&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BD.%2BSpear%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Allen T. Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BT.%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BT.%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Evan J. Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BJ.%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BJ.%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Saba Shahzad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saba%2BShahzad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shahzad%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaba%2BShahzad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Susan Michaelis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BMichaelis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michaelis%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BMichaelis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:michaelis@jhmi.edu" class="" data-icon-position="" data-hide-link-title="0">michaelis@jhmi.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/7/1853" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460402" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460402/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1853.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460402" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460402/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1853.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460402" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460402/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1853.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460402" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460402/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/7/1853.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460402" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460402/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Cellular protein quality control (PQC) systems selectively target misfolded or otherwise aberrant proteins for degradation by the ubiquitin-proteasome system (UPS). How cells discern abnormal from normal proteins remains incompletely understood, but involves in part the recognition between ubiquitin E3 ligases and degradation signals (degrons) that are exposed in misfolded proteins. PQC is compartmentalized in the cell, and a great deal has been learned in recent years about ER-associated degradation (ERAD) and nuclear quality control. In contrast, a comprehensive view of cytosolic quality control (CytoQC) has yet to emerge, and will benefit from the development of a well-defined set of model substrates. In this study, we generated an isogenic “degron library” in <em>Saccharomyces cerevisiae</em> consisting of short sequences appended to the C-terminus of a reporter protein, <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>. About half of these degron-containing proteins are substrates of the integral membrane E3 ligase <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>, which also plays a pivotal role in ERAD and some nuclear protein degradation. Notably, some of our degron fusion proteins exhibit dependence on the E3 ligase <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a>/<a href="http://www.yeastgenome.org/locus/S000004861/overview">Rkr1</a> for degradation, apparently by a mechanism distinct from its known role in ribosomal quality control of translationally paused proteins. <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> and <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a>, E3 ligases involved in the recognition of some misfolded CytoQC substrates, are largely dispensable for the degradation of our degron-containing proteins. Interestingly, the Hsp70/Hsp40 chaperone/cochaperones <a href="http://www.yeastgenome.org/locus/S000000004/overview">Ssa1</a>,2 and <a href="http://www.yeastgenome.org/locus/S000005008/overview">Ydj1</a>, are required for the degradation of all constructs tested. Taken together, the comprehensive degron library presented here provides an important resource of isogenic substrates for testing candidate PQC components and identifying new ones.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/proteostasis" class="hw-term hw-article-keyword hw-article-keyword-proteostasis" rel="nofollow">proteostasis</a></li><li class="kwd"><a href="/keyword/ubiquitin-proteasome-system" class="hw-term hw-article-keyword hw-article-keyword-ubiquitin-proteasome-system" rel="nofollow">ubiquitin proteasome system</a></li><li class="kwd"><a href="/keyword/protein-quality-control" class="hw-term hw-article-keyword hw-article-keyword-protein-quality-control" rel="nofollow">protein quality control</a></li><li class="kwd"><a href="/keyword/e3-ligase-doa10" class="hw-term hw-article-keyword hw-article-keyword-e3-ligase-doa10" rel="nofollow">E3 ligase Doa10</a></li><li class="kwd"><a href="/keyword/ltn1" class="hw-term hw-article-keyword hw-article-keyword-ltn1" rel="nofollow">Ltn1</a></li></ul><p id="p-6">Protein quality control (PQC) mechanisms are critical for maintaining cellular proteostasis by recognizing aberrant proteins and degrading them. Irreversibly misfolded proteins are selectively degraded by the ubiquitin-proteasome system (UPS) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Balch <em>et al.</em> 2008</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Buchberger <em>et al.</em> 2010</a>). Misfolded or aggregation-prone proteins are associated with numerous human diseases due to loss of function through degradation (<em>i.e.</em>, cystic fibrosis), or toxic gain of function (<em>i.e.</em>, accumulation of protein aggregates in Huntington’s and Parkinson’s diseases) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hol and Scheper 2008</a>). Thus, understanding how misfolded proteins are targeted for destruction, or specifically evade PQC mechanisms, is critical for developing therapeutic strategies for protein folding diseases.</p><p id="p-7">The UPS ensures that misfolded proteins are committed to degradation through the activities of a ubiquitin activating enzyme (E1), ubiquitin conjugating enzymes (E2s), and ubiquitin ligases (E3s) that append poly-ubiquitin chains to substrates, which in turn directs them to the proteasome. Importantly, the E3 ubiquitin ligases are thought to serve as the specificity factors leading to ubiquitylation of misfolded proteins (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Finley 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Finley <em>et al.</em> 2012</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pickart 2004</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Deshaies and Joazeiro 2009</a>). In addition, Hsp70 chaperones and Hsp40 cochaperones play a role in PQC by preventing substrate aggregation, or by providing a “bridge” between misfolded substrate and a specific E3 ligase (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">McClellan <em>et al.</em> 2005b</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Nakatsukasa <em>et al.</em> 2008</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Nishikawa <em>et al.</em> 2005</a>). The code that pairs specific ubiquitin E3 ligases with particular degradation signals, or “degrons”, within aberrant proteins is not yet fully understood. The budding yeast <em>Saccharomyces cerevisiae</em> has been a useful model organism for beginning to define degron–E3 interactions (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Finley <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Guerriero <em>et al.</em> 2013</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Ravid and Hochstrasser 2008</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Ravid <em>et al.</em> 2006</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Vembar and Brodsky 2008</a>), and its utility for this goal is expanded in the present study.</p><p id="p-8">PQC machinery exists in various cellular compartments, including the endoplasmic reticulum (ER), nucleus, and cytoplasm. Until recently, the ER-associated protein degradation (ERAD) pathway, whereby misfolded or unassembled membrane and secretory proteins are degraded, has garnered the most attention. In yeast, ERAD substrates are recognized by one of two integral membrane E3 ligases, either <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> or <a href="http://www.yeastgenome.org/locus/S000005373/overview">Hrd1</a> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bays <em>et al.</em> 2001</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Swanson <em>et al.</em> 2001</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kreft <em>et al.</em> 2006</a>). The site of the misfolded lesion within an ERAD substrate appears to determine which E3 is involved, such that <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> acts on membrane proteins with cytoplasmic lesions, and <a href="http://www.yeastgenome.org/locus/S000005373/overview">Hrd1</a> ubiquitylates proteins with misfolded luminal lesions (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Huyer <em>et al.</em> 2004</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Vashist and Ng 2004</a>). While it had been thought that aberrant transmembrane spans are exclusively ubiquitylated by <a href="http://www.yeastgenome.org/locus/S000005373/overview">Hrd1</a>, recent evidence suggests that <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> can also recognize misfolded transmembrane spans in some substrates (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Habeck <em>et al.</em> 2015</a>). In any case, <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> and <a href="http://www.yeastgenome.org/locus/S000005373/overview">Hrd1</a> can account for the ubiquitylation and degradation of essentially all ERAD substrates in yeast (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ismail and Ng 2006</a>).</p><p id="p-9">Significant progress has also been made in recent years defining the mechanisms of nuclear PQC in yeast. The well-studied nuclear E3 ligase, <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a>, directly binds and targets many temperature-sensitive (Ts<sup>–</sup>) nuclear proteins or nuclear localization signal (NLS)-directed GFP-degron fusions through its N- and C-terminal unstructured regions (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gardner <em>et al.</em> 2005</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rosenbaum <em>et al.</em> 2011</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Rosenbaum and Gardner 2011</a>). Elegant studies from Gardner and coworkers demonstrated that <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> targets substrates with a high propensity to aggregate, such as those having hydrophobic stretches of at least five contiguous residues (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fredrickson <em>et al.</em> 2011</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2013</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fredrickson and Gardner 2012</a>). The E3 ligase <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> resides not only in the ER membrane, but also in the inner nuclear membrane (INM), and is required for the degradation of several nuclear quality control substrates, including a mutant version of the kinetochore protein <a href="http://www.yeastgenome.org/locus/S000003372/overview">Ndc10</a>, and proteins bearing the MATα2-derived degron <em>Deg1</em> (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Deng and Hochstrasser 2006</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Ravid <em>et al.</em> 2006</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Furth <em>et al.</em> 2011</a>). More recently, the Asi complex that contains the E3s <a href="http://www.yeastgenome.org/locus/S000004725/overview">Asi1</a> and <a href="http://www.yeastgenome.org/locus/S000004953/overview">Asi3</a>, and is also in the INM, has been shown to ubiquitylate membrane proteins that inadvertently mislocalize from the ER to the INM, such as the sterol biosynthesis enzymes <a href="http://www.yeastgenome.org/locus/S000001049/overview">Erg11</a> and <a href="http://www.yeastgenome.org/locus/S000001175/overview">Nsg1</a> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Foresti <em>et al.</em> 2014</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Khmelinskii <em>et al.</em> 2014</a>). Together, the nuclear E3s <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a>, <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>, and <a href="http://www.yeastgenome.org/locus/S000004725/overview">Asi1</a>/3 play diverse but related roles in regulation, removing misfolded or aggregated proteins from the nucleus, and safe-guarding nuclear membrane integrity.</p><p id="p-10">Interestingly, much less is known about cytoplasmic protein quality control (CytoQC) than about ERAD or nuclear quality control. The synthesis of nearly all cellular proteins takes place in the cytoplasm. Thus, cells must distinguish between fully synthesized but irreversibly misfolded proteins that should be targeted for degradation, and nascent polypeptides that are not yet fully folded and are still ribosome-associated. The E3 ligases implicated to date as major players in yeast CytoQC are <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>, <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a>, and <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fredrickson and Gardner 2012</a>). These E3s, and the substrates upon which they act, are discussed further below. In addition, a ribosomal quality control (RQC) complex has recently been described that contains the E3 ligase <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> (formerly called <a href="http://www.yeastgenome.org/locus/S000004861/overview">Rkr1</a>) along with the additional components <a href="http://www.yeastgenome.org/locus/S000002741/overview">Rqc1</a> and <a href="http://www.yeastgenome.org/locus/S000005930/overview">Rqc2</a>/<a href="http://www.yeastgenome.org/locus/S000005930/overview">Tae2</a> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brandman <em>et al.</em> 2012</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Shen <em>et al.</em> 2015</a>). <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> ubiquitylates polypeptides that undergo translational pausing during aberrant synthesis of poly-basic peptides, such as poly-lysine from the poly(A) tail of a nonstop mRNA on the ribosome (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bengtson and Joazeiro 2010</a>). Such stalled proteins are cotranslationally ubiquitylated by <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a>, targeting them for degradation even before their synthesis is complete.</p><p id="p-11">It was surprising when studies by us and others revealed that the E3 ligase <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> is also involved in the degradation of several soluble CytoQC substrates (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>; <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Ravid <em>et al.</em> 2006</a>), given that <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> is membrane-associated and clearly implicated in ERAD. These substrates include the mutant Ts<sup>–</sup> cytosolic proteins <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-2 and <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-3, and the cytosolic reporter proteins <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-CL1 and <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-GFP-CL1 that contain a C-terminal 16-amino-acid degron called CL1 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lewis and Pelham 2009</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>). The finding that <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> is responsible for ubiquitylation of these model CytoQC substrates, in addition to its known role in ERAD, suggests that the cytoplasmic face of the ER membrane may be a common platform for CytoQC of numerous substrates, an issue we address in the present study.</p><p id="p-12"><a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> is an E3 ligase first identified by its ability to target proteins with destabilizing N-terminal amino acids (N-end rule substrates) (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Varshavsky 1997</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Varshavsky <em>et al.</em> 2000</a>). More recently, <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> was shown to be critical for the degradation of immature protein kinases, mutant forms of cytosolic enzymes, as well as a number of engineered CytoQC substrates, including cytosolically localized mutant versions of GFP, proteinase A, carboxypeptidase Y and the ABC transporter <a href="http://www.yeastgenome.org/locus/S000001692/overview">Ste6</a> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Eisele and Wolf 2008</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Guerriero <em>et al.</em> 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Heck <em>et al.</em> 2010</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Nillegoda <em>et al.</em> 2010</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Park <em>et al.</em> 2013</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Prasad <em>et al.</em> 2010</a>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Khosrow-Khavar <em>et al.</em> 2012</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Summers <em>et al.</em> 2013</a>). Interestingly, deletion of <em><a href="http://www.yeastgenome.org/locus/S000002550/overview">SAN1</a></em> also stabilized some of these CytoQC substrates to varying extents, providing evidence that <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> and <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> often act in a redundant fashion in CytoQC (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Fredrickson and Gardner 2012</a>). Surprisingly some of these CytoQC substrates are transported to the nucleus, explaining why nuclear <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> may be involved (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Park <em>et al.</em> 2013</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Prasad <em>et al.</em> 2010</a>). However, others clearly remain in the cytoplasm, suggesting a role for <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> outside the nucleus (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Guerriero <em>et al.</em> 2013</a>). Alternatively, stabilized substrates in these cells could be exported out of the nucleus at a faster rate than nuclear import. Clearly, the basis for the interrelated but distinct preference of the <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> and <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> E3 ligases remains incompletely understood.</p><p id="p-13">Despite the growing list of CytoQC substrates, it has been difficult to identify characteristics within misfolded proteins that are recognized by the different E3 ligases in the cytoplasm. Thus, a comprehensive and unified view of CytoQC has yet to emerge and could benefit from development of a defined set of model substrates. In this study, we have generated an isogenic <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-degron fusion library to begin to systematically address the issue of the “code” for CytoQC. Within this collection of 77 degron-containing proteins, about half are substrates for <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>, while the remaining ones are degraded independently of <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>. The degradation of all of these is dependent on the proteasome, and in a more detailed analysis of a subset of these, all show dependence on the Hsp70 chaperones <a href="http://www.yeastgenome.org/locus/S000000004/overview">Ssa1</a>/2 and Hsp40 <a href="http://www.yeastgenome.org/locus/S000005008/overview">Ydj1</a>. Notably, while only a few of the degron library constructs are spared when <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> or <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> are absent, and only modestly at that, a number of the degron constructs exhibit Ltn1-dependent degradation. We provide evidence that this may occur via a mechanism distinct from ribosomal pausing. Taken together, the degron library presented here will provide a valuable resource of defined substrates for future studies of CytoQC.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains, media, and growth conditions</h3><p id="p-14">The <em>S. cerevisiae</em> strains used in this study are listed in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/TableS1.pdf">Table S1</a>. Standard media (SC-Leu and SC-Leu-Ura) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Michaelis and Herskowitz, 1988</a>), and media used for proteasome inhibition with MG132 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Liu <em>et al.</em> 2007</a>), are described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FileS1.pdf">File S1</a>. Unless otherwise indicated, yeast was grown at 30°. Yeast cultures in mid-log phase were obtained by dilution of overnight saturated cultures to an OD<sub>600</sub> of 0.3, and growth for 3–4 hr, at which point an OD<sub>600</sub> measurement was taken to confirm cultures were in mid-log phase (OD<sub>600</sub> between 0.6 and 0.8). For spot growth tests, overnight saturated cultures were adjusted to 0.2 OD<sub>600</sub>/ml, and then 10-fold serial dilutions were spotted onto SC-Leu and SC-Leu-Ura plates and incubated for 2 days at 30°.</p></div><div id="sec-3" class="subsection"><h3>Plasmids and plasmids constructions</h3><p id="p-15">Plasmids used in this study are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/TableS2.pdf">Table S2</a>. To generate the starting plasmid pSM2697 for the degron library (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), recombinational cloning was used to add a fragment containing a <em>Sma</em>I site and three frame-shifted stop codons in place of the single stop codon in pSM2287 (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>). Plasmid pSM2697 is a <em>CEN <a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> plasmid with <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-3HA</em> followed by the <em>Sma</em>I cloning site and stop codons in three frames. Construction of plasmids pSM2801, pSM2802, pSM2803, pSM2811, and pSM2812 is described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FileS1.pdf">File S1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A genetic screen to isolate proteasome-dependent protein degradation signals (degrons). (A) Schematic of the library cloning vector. Yeast genomic DNA fragments were cloned into the library vector such that they would be expressed as a C-terminal fusion to Ura3-HA, followed by stop codons in all three frames. (B&#x2013;E) Candidate clones isolated in the screen based on conferring a Ura&#x2013; phenotype were arrayed in 96-well format, and retransformed into WT (SM4333) and proteasome mutant (pre1 pre2; SM4334) strains. Growth assays for 84 of the 153 isolates are shown in (B&#x2013;D). Transformants were transferred from liquid growth medium onto the indicated agar plates, and incubated for the times shown. Controls expressing Ura3-HA (green boxes) and Ura3-HA-CL1 (orange box) were present in several positions on the plates. Additional degron-containing constructs generated in a previous study (Metzger et al. 2008) are also present on these plates (positions H4&#x2013;H7: Tom20-Ura3-HA, Tom20-Ura3-HA-CL1, Vma12-Ura3-HA-CL1, and Vma12-Ura3-HA). (F&#x2013;G) Spot dilution growth assay for several Ura3-HA-degron constructs from the library. Serial dilutions of WT yeast (SM4460) expressing Ura3-HA with no degron, Ura3-HA-CL1, and four Ura3-HA degron constructs were spotted to plates and incubated 48&#xA0;hr (F). These constructs were also examined for growth in the proteasome mutant (SM4334; pre1-1 pre2-1) and isogenic WT strain (SM4333; PRE1 PRE2) to determine if growth improves when proteasome function is defective (G). (H) Ura3-HA-degron turnover was analyzed by cycloheximide chase analysis as described in Materials and Methods. A first-order exponential decay function was fit to each chase series, allowing determination of the half-lives (t1/2) for each degron-containing protein. The mean and 1&#xA0;SD for three independent experiments are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1168411460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A genetic screen to isolate proteasome-dependent protein degradation signals (degrons). (A) Schematic of the library cloning vector. Yeast genomic DNA fragments were cloned into the library vector such that they would be expressed as a C-terminal fusion to Ura3-HA, followed by stop codons in all three frames. (B&#x2013;E) Candidate clones isolated in the screen based on conferring a Ura&#x2013; phenotype were arrayed in 96-well format, and retransformed into WT (SM4333) and proteasome mutant (&lt;em&gt;pre1 pre2&lt;/em&gt;; SM4334) strains. Growth assays for 84 of the 153 isolates are shown in (B&#x2013;D). Transformants were transferred from liquid growth medium onto the indicated agar plates, and incubated for the times shown. Controls expressing Ura3-HA (green boxes) and Ura3-HA-CL1 (orange box) were present in several positions on the plates. Additional degron-containing constructs generated in a previous study (Metzger &lt;em&gt;et al.&lt;/em&gt; 2008) are also present on these plates (positions H4&#x2013;H7: Tom20-Ura3-HA, Tom20-Ura3-HA-CL1, Vma12-Ura3-HA-CL1, and Vma12-Ura3-HA). (F&#x2013;G) Spot dilution growth assay for several Ura3-HA-degron constructs from the library. Serial dilutions of WT yeast (SM4460) expressing Ura3-HA with no degron, Ura3-HA-CL1, and four Ura3-HA degron constructs were spotted to plates and incubated 48&#xA0;hr (F). These constructs were also examined for growth in the proteasome mutant (SM4334; &lt;em&gt;pre1-1 pre2-1&lt;/em&gt;) and isogenic WT strain (SM4333; &lt;em&gt;PRE1 PRE2&lt;/em&gt;) to determine if growth improves when proteasome function is defective (G). (H) Ura3-HA-degron turnover was analyzed by cycloheximide chase analysis as described in &lt;em&gt;Materials and Methods&lt;/em&gt;. A first-order exponential decay function was fit to each chase series, allowing determination of the half-lives (&lt;em&gt;t&lt;/em&gt;&lt;sub&gt;1/2&lt;/sub&gt;) for each degron-containing protein. The mean and 1&#xA0;SD for three independent experiments are shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1853/F1.medium.gif" width="383" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/7/1853/F1.medium.gif" width="383" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468610" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">A genetic screen to isolate proteasome-dependent protein degradation signals (degrons). (A) Schematic of the library cloning vector. Yeast genomic DNA fragments were cloned into the library vector such that they would be expressed as a C-terminal fusion to Ura3-HA, followed by stop codons in all three frames. (B–E) Candidate clones isolated in the screen based on conferring a Ura– phenotype were arrayed in 96-well format, and retransformed into WT (SM4333) and proteasome mutant (<em>pre1 pre2</em>; SM4334) strains. Growth assays for 84 of the 153 isolates are shown in (B–D). Transformants were transferred from liquid growth medium onto the indicated agar plates, and incubated for the times shown. Controls expressing Ura3-HA (green boxes) and Ura3-HA-CL1 (orange box) were present in several positions on the plates. Additional degron-containing constructs generated in a previous study (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>) are also present on these plates (positions H4–H7: Tom20-Ura3-HA, Tom20-Ura3-HA-CL1, Vma12-Ura3-HA-CL1, and Vma12-Ura3-HA). (F–G) Spot dilution growth assay for several Ura3-HA-degron constructs from the library. Serial dilutions of WT yeast (SM4460) expressing Ura3-HA with no degron, Ura3-HA-CL1, and four Ura3-HA degron constructs were spotted to plates and incubated 48 hr (F). These constructs were also examined for growth in the proteasome mutant (SM4334; <em>pre1-1 pre2-1</em>) and isogenic WT strain (SM4333; <em>PRE1 PRE2</em>) to determine if growth improves when proteasome function is defective (G). (H) Ura3-HA-degron turnover was analyzed by cycloheximide chase analysis as described in <em>Materials and Methods</em>. A first-order exponential decay function was fit to each chase series, allowing determination of the half-lives (<em>t</em><sub>1/2</sub>) for each degron-containing protein. The mean and 1 SD for three independent experiments are shown.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Degron library construction and screening</h3><p id="p-17">Yeast genomic DNA was digested with DNaseI in 50 mM Tris-HCl (pH 7.5), 5 mM MnCl<sub>2</sub>, 5 mM CaCl<sub>2</sub> for various times and the reaction was quenched by the addition of 0.02 M EDTA (pH 8). A reaction yielding a distribution of DNA fragments ranging from ∼50 to 400 bp, as judged from electrophoresis analysis, was made blunt-ended with T4 DNA polymerase and cloned into the <em>Sma</em>I site of pSM2697. The ligation reaction was transformed into bacteria, and colonies from several plates (∼5000 colonies) were pooled, grown in LB plus carbenicillin, and plasmid DNA was purified to yield a library containing <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-3XHA</em> with random 3′ inserts.</p><p id="p-18">The degron plasmid library DNA (<em>CEN <a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a> <a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-3XHA-insert</em>) was transformed into SM4460 (relevant genotype <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a></em>∆<em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>∆) and plated onto SC-Leu media to select for Leu<sup>+</sup> yeast colonies containing individual library plasmids. Transformants were replica plated onto SC-Leu-Ura media to identify Ura<sup>–</sup> clones. Of ∼3000 Leu+ colonies screened, 153 were Ura<sup>–</sup> (∼5%). Plasmids prepared from these initial clones were passaged through <em>Escherichia. coli</em> to ensure each library clone represented a single plasmid. After DNA sequencing, 113 unique inserts were confirmed.</p><p id="p-19">To determine proteasome-dependence of growth, yeast strains SM4333 (<em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>∆ <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a></em>∆) and SM4334 (<em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a>-1 <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a>-1 <a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>∆ <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a></em>∆) were transformed (without heat shock), with the 113 individual library clones in two 96-deep-well plates, with controls on the bottom row. Transformants were grown to saturation in liquid culture, and then pinned manually onto SC-Leu and SC-Leu-Ura agar media. Plates were incubated for 24 hr and 48 hr at 30° and imaged. The growth of nearly all Ura<sup>–</sup> clones was rescued in the <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a>-1 <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a>-1</em> strain (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, and data not shown).</p></div><div id="sec-5" class="subsection"><h3>Systematic growth analysis for degradation dependencies and quantification of growth</h3><p id="p-20">To simplify analyses and condense the library to a single 96-well plate, a subset of 77 unique plasmids that conferred <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a>,2</em>-dependent growth on SC-Leu-Ura was selected and subjected to further quantitative growth analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>). These 77 plasmids, along with plasmids expressing <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-HA-CL1</em> (pSM2288), <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-HA-CL1*</em> (pSM2337; CL1* is a CL1 variant with nine additional C-terminal residues (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Metzger 2009</a>), the positive control plasmid (pSM2697), which expresses <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-HA</em>, and the empty vector (pSM173) (six replicates of each control), were rearrayed in a randomized order into a single 96-well plate. Control plasmids were placed at various positions to control for differential conditions within a single plate; (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>, A–D). Plasmids were transformed in 96-well format into WT and mutant strains, and selected on SC-Leu plates. Transformants (in quadruplicate) were replica pinned from source plates to solid SC-Leu and SC-Leu-Ura media using a manual pin tool, and incubated for 24 hr at 30°. The relative growth between each mutant and the WT was calculated as the Log<sub>2</sub> ratio of the mean normalized colony size of mutant to WT for each degron. Relative growth is reported in the heatmaps for significant differences between mutant and WT, assessed by a Mann-Whitney test.</p></div><div id="sec-6" class="subsection"><h3>Cycloheximide-chase and steady-state protein analysis</h3><p id="p-21">For cycloheximide chase analysis, saturated cultures were backdiluted to an OD<sub>600</sub> of 0.3, and grown for 3–4 hr to mid-log phase. Cells were harvested and resuspended in fresh SC-medium at 2 OD<sub>600</sub> units/ml. To initiate the chase, cycloheximide was added to 100 μg/ml, and, to terminate it, aliquots of cells were transferred to an equal volume of 20 mM sodium azide on ice at the indicated times. For steady-state protein analysis, samples were processed the same way, except that cycloheximide was omitted prior to collecting cells.</p><p id="p-22">Preparation of protein lysates, SDS-PAGE, western blotting, antibodies used, and imaging are described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FileS1.pdf">File S1</a>.</p></div><div id="sec-7" class="subsection"><h3>Proteasome inhibition with MG132 and ubiquitin E1 inactivation</h3><p id="p-23">Proteasome inhibition was performed based on the method of <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Liu <em>et al.</em> (2007</a>), which optimizes the efficacy of MG132. Yeast were grown overnight in SCP– medium. Cultures were diluted to an OD<sub>600</sub> of 0.5 in SCPS-medium and grown for 3–4 hr. Cells were harvested and resuspended in fresh SCPS-medium at 2 OD<sub>600</sub> units/ml. Cultures were split and DMSO or MG132 (dissolved in DMSO) was added to a final concentration of 75 μM for 2 hr at 30° with shaking. Sodium azide was added to a final concentration of 10 mM, and cells were harvested. Protein extracts were prepared as described previously. To test the requirement of ubiquitylation on degron fusion protein levels, strains SM5923 (<em><a href="http://www.yeastgenome.org/locus/S000001693/overview">UBA1</a></em>) and SM5925 (<em><a href="http://www.yeastgenome.org/locus/S000001693/overview">uba1</a>-204</em>) were grown at 25° and then shifted to 37° for 1 hr prior to addition of cycloheximide. Cells were incubated at 37° for an additional 45 min and processed for western blotting.</p></div><div id="sec-8" class="subsection"><h3>Determination of protein half-lives and steady-state levels</h3><p id="p-24">Quantification of cycloheximide chase experiments was performed using QuantityOne software (BioRad). Signal from <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron and the loading control (hexokinase) was quantified. The amount of <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron at each time point was normalized to the amount of loading control protein at the same time point. The percent remaining was calculated from the normalized protein levels at each time point relative to the 0 min time point.</p><p id="p-25">To determine the half-lives of proteins, we assumed that their degradation follows the first-order exponential decay model, <em>P</em> = 100e<sup>(–λt)</sup>, where <em>P</em> is the percent of protein remaining at time t, and λ is the decay constant. This model was fitted to the data by nonlinear least-squares regression analysis to obtain an estimate of λ. The half-life of the protein is then calculated from the estimate of λ. For comparisons of fitted curves, an <em>F</em>-test was used to generate a <em>P</em>-value. The <em>P</em>-values reported in plots have been adjusted to correct for multiple comparisons (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hochberg and Benjamini 1990</a>) for all comparisons of protein decay from cycloheximide chase analyses reported in this study. Steady-state levels of <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron proteins were quantified using Image J (NIH).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-26">Strains and plasmids are available upon request. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/TableS3.xlsx">Table S3</a> contains peptide sequences of all degrons isolated in this study, with nucleotide sequences available upon request.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>A genetic screen to isolate degradation signals (degrons) that destabilize Ura3</h3><p id="p-27">To identify protein sequences that destabilize <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA in a proteasome-dependent manner, we generated a degron plasmid library by cloning DNA fragments into pSM2697 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a> and <em>Materials and Methods</em>) and transformed into yeast, selecting Leu<sup>+</sup> transformants, followed by replica plating to medium lacking uracil. Our initial screening yielded 153 transformants (∼5% of total tested) that grew poorly, or not at all, after 24 or 48 hr on –Leu –Ura medium, indicative of either a destabilized <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA fusion protein, decreased expression, or lack of <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a> function (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1, B–D</a> and data not shown; see also spot test growth of several examples, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1F</a>).</p><p id="p-28">Because we were interested solely in plasmids expressing <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron proteins whose stabilization is proteasome-dependent, clones were transformed into the proteasome mutant <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a>-1 <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a>-1</em>, and tested for growth on medium lacking uracil. As expected, wild-type (WT) cells expressing the control degron construct <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-CL1 failed to grow on SC-Leu-Ura plates, but growth was restored in the <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a> <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a></em> mutant (orange box, compare <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1, E–D</a>; see also <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1G</a>). We found that nearly all of the library clones were capable of growing on medium lacking uracil in the <em><a href="http://www.yeastgenome.org/locus/S000000814/overview">pre1</a> <a href="http://www.yeastgenome.org/locus/S000006307/overview">pre2</a></em> mutant (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1, E</a> <em>vs.</em> D and data not shown; see also spot test growth of several examples in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1G</a>), indicating that they express <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron fusion proteins targeted for proteasomal degradation.</p><p id="p-29">To examine degradation directly, we assessed whether the <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA fusion proteins in strains spotted in <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1, F and G</a> were rapidly degraded by cycloheximide chase analysis (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1H</a>). Without a degron, <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA is stable over the time course of the chase experiment, with a projected half-life on the order of many hours, whereas the control construct <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-CL1 is rapidly degraded with a half-life of 1.6 min, consistent with our previously published results (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>). Similarly, most clones isolated in our screen are also very rapidly degraded, many with half-lives of &lt; 3 min (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1H</a> and data not shown). Because even a small amount of <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a> might suffice for growth, and, since we screened for a Ura<sup>–</sup> phenotype, we might not have isolated “slower-acting” or weaker degrons that promote half-lives of &gt;5 min. For further analysis, we used a subset of 77 unique <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron plasmid constructs (as determined by DNA sequencing), herein referred as the “degron library” (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/TableS3.xlsx">Table S3</a>). The degron peptides appended to <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA range from seven to 58 amino acids in length.</p></div><div id="sec-12" class="subsection"><h3>Identification of Doa10-dependent and -independent degrons indicates several E3 ligases are involved in CytoQC of our degron constructs</h3><p id="p-30">We previously showed that <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-CL1 is dependent upon <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> for degradation (<a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>), prompting us to ask how common <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-dependent degradation is among our 77-member degron library. Library and control plasmids were arrayed in 96-well format and transformed into WT and <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆ strains. To assess growth, transformants were replica-pinned in quadruplicate from this master plate onto –Leu and –Leu-Ura agar plates, and incubated for 24 hr at 30° (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>, A–C). It should be noted that pinning from solid-to-solid medium gives more quantifiable detection of differential growth than the liquid-to-solid plating shown in <a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1</a>. Areas of the colonies were measured from digital images of the plates using image-analysis software (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carpenter <em>et al.</em> 2006</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lamprecht <em>et al.</em> 2007</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Vokes and Carpenter 2008</a>), and normalized within each plate to the control colonies expressing no <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a> or <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>, A–C, red and green boxes, respectively). The relative difference in growth between WT and the <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆ mutant was calculated from the normalized areas and displayed as a heatmap, where pink signifies more growth in <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆ than WT, and white signifies no difference in growth between these two strains (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>; also repeated for ease of analysis as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>D).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The degron collection contains both Doa10-dependent and Doa10-independent degrons. (A) Heatmap representation of the relative growth for doa10&#x394; and WT strains expressing the same degron. WT and doa10&#x2206; strains (SM4460 and SM 4820) transformed with a subset of 77 clones, referred to here as the degron library, were rearrayed in a randomized order in 96-well format, pinned in quadruplicate from solid medium to solid medium, grown for 24&#xA0;hr, and imaged (growth plates are shown in Figure S1 and also include the degron constructs containing CL1 and CL1*). The difference in growth between WT and doa10&#x2206; strains was quantitated from the images and expressed as a heat map, in which the deeper the pink color the greater the growth differential between WT and doa10&#x2206;; white signifies no statistically significant difference in growth between WT and mutant. The positions of the 13 representative degron constructs comprising the &#x201C;degron tester set&#x201D; used in Figure 3 and later tests are indicated. Controls for growth express Ura3-HA (green boxes), no Ura3 (red boxes), or Ura3-HA-CL1 (CL1). Yeast in corner positions (yellow boxes) express Ura3-HA and were not used in the analysis. (B&#x2013;D) Growth quantification and cycloheximide chase analysis for WT and doa10&#x394; expressing two sample degrons. The degrons examined are 12-86 from well G8, and 10-13 from well E1; these panels are taken from Figure S1 and enlarged. Chases were performed in triplicate, and the mean and 1&#xA0;SD of the mean for each time point is shown. Half-lives and P-values were calculated as described in Materials and Methods. (E) The half-lives of the 13 tester set degrons and CL1 in doa10&#x394; and WT were determined by cyclohexamide chase (Figure S2), and their ratio was plotted along the x-axis of the strip chart. Degron constructs that show DOA10-dependent growth (pink circles), and DOA10-independent growth (black circles), are indicated. The pink dotted lines represents the smallest fold change in half-life between doa10&#x394; and WT that was detectable by growth for the 14 degrons analyzed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1168411460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The degron collection contains both Doa10-dependent and Doa10-independent degrons. (A) Heatmap representation of the relative growth for &lt;em&gt;doa10&lt;/em&gt;&#x394; and WT strains expressing the same degron. WT and &lt;em&gt;doa10&lt;/em&gt;&#x2206; strains (SM4460 and SM 4820) transformed with a subset of 77 clones, referred to here as the degron library, were rearrayed in a randomized order in 96-well format, pinned in quadruplicate from solid medium to solid medium, grown for 24&#xA0;hr, and imaged (growth plates are shown in Figure S1 and also include the degron constructs containing CL1 and CL1*). The difference in growth between WT and &lt;em&gt;doa10&lt;/em&gt;&#x2206; strains was quantitated from the images and expressed as a heat map, in which the deeper the pink color the greater the growth differential between WT and &lt;em&gt;doa10&lt;/em&gt;&#x2206;; white signifies no statistically significant difference in growth between WT and mutant. The positions of the 13 representative degron constructs comprising the &#x201C;degron tester set&#x201D; used in Figure 3 and later tests are indicated. Controls for growth express Ura3-HA (green boxes), no Ura3 (red boxes), or Ura3-HA-CL1 (CL1). Yeast in corner positions (yellow boxes) express Ura3-HA and were not used in the analysis. (B&#x2013;D) Growth quantification and cycloheximide chase analysis for WT and &lt;em&gt;doa10&lt;/em&gt;&#x394; expressing two sample degrons. The degrons examined are 12-86 from well G8, and 10-13 from well E1; these panels are taken from Figure S1 and enlarged. Chases were performed in triplicate, and the mean and 1&#xA0;SD of the mean for each time point is shown. Half-lives and &lt;em&gt;P&lt;/em&gt;-values were calculated as described in &lt;em&gt;Materials and Methods&lt;/em&gt;. (E) The half-lives of the 13 tester set degrons and CL1 in &lt;em&gt;doa10&lt;/em&gt;&#x394; and WT were determined by cyclohexamide chase (Figure S2), and their ratio was plotted along the &lt;em&gt;x&lt;/em&gt;-axis of the strip chart. Degron constructs that show &lt;em&gt;DOA10&lt;/em&gt;-dependent growth (pink circles), and &lt;em&gt;DOA10&lt;/em&gt;-independent growth (black circles), are indicated. The pink dotted lines represents the smallest fold change in half-life between &lt;em&gt;doa10&lt;/em&gt;&#x394; and WT that was detectable by growth for the 14 degrons analyzed.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1853/F2.medium.gif" width="211" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/7/1853/F2.medium.gif" width="211" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468541" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">The degron collection contains both Doa10-dependent and Doa10-independent degrons. (A) Heatmap representation of the relative growth for <em>doa10</em>Δ and WT strains expressing the same degron. WT and <em>doa10</em>∆ strains (SM4460 and SM 4820) transformed with a subset of 77 clones, referred to here as the degron library, were rearrayed in a randomized order in 96-well format, pinned in quadruplicate from solid medium to solid medium, grown for 24 hr, and imaged (growth plates are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a> and also include the degron constructs containing CL1 and CL1*). The difference in growth between WT and <em>doa10</em>∆ strains was quantitated from the images and expressed as a heat map, in which the deeper the pink color the greater the growth differential between WT and <em>doa10</em>∆; white signifies no statistically significant difference in growth between WT and mutant. The positions of the 13 representative degron constructs comprising the “degron tester set” used in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and later tests are indicated. Controls for growth express Ura3-HA (green boxes), no Ura3 (red boxes), or Ura3-HA-CL1 (CL1). Yeast in corner positions (yellow boxes) express Ura3-HA and were not used in the analysis. (B–D) Growth quantification and cycloheximide chase analysis for WT and <em>doa10</em>Δ expressing two sample degrons. The degrons examined are 12-86 from well G8, and 10-13 from well E1; these panels are taken from <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a> and enlarged. Chases were performed in triplicate, and the mean and 1 SD of the mean for each time point is shown. Half-lives and <em>P</em>-values were calculated as described in <em>Materials and Methods</em>. (E) The half-lives of the 13 tester set degrons and CL1 in <em>doa10</em>Δ and WT were determined by cyclohexamide chase (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS2.ai">Figure S2</a>), and their ratio was plotted along the <em>x</em>-axis of the strip chart. Degron constructs that show <em>DOA10</em>-dependent growth (pink circles), and <em>DOA10</em>-independent growth (black circles), are indicated. The pink dotted lines represents the smallest fold change in half-life between <em>doa10</em>Δ and WT that was detectable by growth for the 14 degrons analyzed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">As expected for strains expressing <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-CL1, growth of the <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆ transformant was significantly greater than the WT transformant (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, position C12; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>, B and C). Notably, similar to <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-CL1, approximately half of the degron library transformants showed increased colony size in <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>Δ compared to WT, suggesting that the <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> E3 ligase is involved in the turnover of many <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-degron fusion proteins. Thus, <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> appears to have a widespread role in CytoQC, in addition to its well-characterized roles in ERAD and nuclear quality control. However, since many degrons showed no difference in growth in <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆ <em>vs.</em> WT (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>, white squares; see also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>, A–C), other E3 ligases may be involved in the turnover of those degron-bearing constructs. These could function redundantly with <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>, or may define a completely <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-independent pathway(s) for degradation of many of the <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron constructs. It should be noted that <a href="http://www.yeastgenome.org/locus/S000005373/overview">Hrd1</a>, the other ERAD E3, does not appear to act on any members of the degron library (data not shown).</p><p id="p-33">To confirm that poor growth in our quantitative growth assay indeed reflects protein turnover, we examined degradation of a predicted <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-dependent (12-86; G8) and <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-independent (10-13; E1) <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron protein. Consistent with their growth patterns, 12-86 was strongly stabilized in the <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆ strain by cycloheximide chase, whereas 10-13 was degraded as efficiently as in WT (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2, B–D</a>). Thus, the quantitative growth assay provides a reliable method to determine which gene products are involved in the turnover of constructs in our degron library. Quantitative growth of the entire degron collection in a variety of E3 mutant strains is summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>E.</p></div><div id="sec-13" class="subsection"><h3>A “degron tester set” requires ubiquitylation for proteasome-mediated degradation</h3><p id="p-34">For the remainder of this study, we chose a representative set of 13 degron constructs for in-depth analysis based on different degron lengths, hydrophobicities (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/TableS3.xlsx">Table S3</a>), and growth rates in WT and E3 ligase mutants (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>E). We refer to these as the “degron tester set” (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) and we analyzed them together with the previously characterized degron CL1. As predicted by their growth patterns, the steady-state level of all the <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron proteins was significantly lower in WT cells, as compared to <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA with no degron. In addition, the observed migration by SDS-PAGE correlated with the predicted increase in molecular weight due to the presence of degron peptides of varying lengths (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). Five members of the degron tester set, along with CL1 were predicted by growth to be <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-dependent (10-1, 10-21, 10-31, 12-33, and 12-86) and the remaining eight to be <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-independent (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2, A and E</a>). This <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-dependent or -independent degradation for all 13 members of the tester set was confirmed by cyclohexamide chase (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS2.ai">Figure S2</a>) and steady-state analysis (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3E</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The &#x201C;degron tester set&#x201D; constructs all show ubiquitin- and proteasome-dependent degradation. A representative subset of Ura3-HA-degron proteins from the degron library was examined by western analysis. (A) Ura3-HA-degron fusion proteins isolated in the screen differ in migration rate according to their predicted lengths, and are present at lower levels than Ura3-HA, in WT cells (SM4460), reflecting degradation. Steady-state protein extracts were prepared in triplicate, analyzed by SDS-PAGE and western analysis with anti-HA antibodies, as described in Materials and Methods, and a representative gel is shown. Quantitation of the HA signal was performed using ImageJ software, and normalized to a hexokinase control (not shown). The amount of normalized signal for degron-containing Ura3-HA-degron proteins relative to Ura3-HA with no degron (set to 1) is plotted. (B&#x2013;D) Degradation for all degron constructs exhibit ubiquitin and proteasome dependence. In (B), WT cells expressing Ura3-HA or Ura3-HA-degron fusion proteins were pretreated with drug vehicle (&#x2013;) or MG132 (+) for 2&#xA0;hr prior to sample preparation. Proteins were detected by western blotting and quantified as in (A). In (C), degron constructs were expressed in a WT UBA1 strain (&#x2013;) or a mutant uba1-204 strain (+) that is defective in ubiquitylation at nonpermissive temperature (Ghaboosi and Deshaies 2007). Cells were processed as described in Materials and Methods. (E) Doa10-dependent steady-state levels were measured in the WT strain SM4460 (+) and doa10&#x2206; mutant SM4820 (&#x2013;). Error bars represent 1&#xA0;SD of the mean. Vertical dotted lines indicate where separate gels are spliced together." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1168411460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The &#x201C;degron tester set&#x201D; constructs all show ubiquitin- and proteasome-dependent degradation. A representative subset of Ura3-HA-degron proteins from the degron library was examined by western analysis. (A) Ura3-HA-degron fusion proteins isolated in the screen differ in migration rate according to their predicted lengths, and are present at lower levels than Ura3-HA, in WT cells (SM4460), reflecting degradation. Steady-state protein extracts were prepared in triplicate, analyzed by SDS-PAGE and western analysis with anti-HA antibodies, as described in &lt;em&gt;Materials and Methods&lt;/em&gt;, and a representative gel is shown. Quantitation of the HA signal was performed using ImageJ software, and normalized to a hexokinase control (not shown). The amount of normalized signal for degron-containing Ura3-HA-degron proteins relative to Ura3-HA with no degron (set to 1) is plotted. (B&#x2013;D) Degradation for all degron constructs exhibit ubiquitin and proteasome dependence. In (B), WT cells expressing Ura3-HA or Ura3-HA-degron fusion proteins were pretreated with drug vehicle (&#x2013;) or MG132 (+) for 2&#xA0;hr prior to sample preparation. Proteins were detected by western blotting and quantified as in (A). In (C), degron constructs were expressed in a WT &lt;em&gt;UBA1&lt;/em&gt; strain (&#x2013;) or a mutant &lt;em&gt;uba1-204&lt;/em&gt; strain (+) that is defective in ubiquitylation at nonpermissive temperature (Ghaboosi and Deshaies 2007). Cells were processed as described in &lt;em&gt;Materials and Methods&lt;/em&gt;. (E) Doa10-dependent steady-state levels were measured in the WT strain SM4460 (+) and &lt;em&gt;doa10&lt;/em&gt;&#x2206; mutant SM4820 (&#x2013;). Error bars represent 1&#xA0;SD of the mean. Vertical dotted lines indicate where separate gels are spliced together.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1853/F3.medium.gif" width="296" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/7/1853/F3.medium.gif" width="296" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468691" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">The “degron tester set” constructs all show ubiquitin- and proteasome-dependent degradation. A representative subset of Ura3-HA-degron proteins from the degron library was examined by western analysis. (A) Ura3-HA-degron fusion proteins isolated in the screen differ in migration rate according to their predicted lengths, and are present at lower levels than Ura3-HA, in WT cells (SM4460), reflecting degradation. Steady-state protein extracts were prepared in triplicate, analyzed by SDS-PAGE and western analysis with anti-HA antibodies, as described in <em>Materials and Methods</em>, and a representative gel is shown. Quantitation of the HA signal was performed using ImageJ software, and normalized to a hexokinase control (not shown). The amount of normalized signal for degron-containing Ura3-HA-degron proteins relative to Ura3-HA with no degron (set to 1) is plotted. (B–D) Degradation for all degron constructs exhibit ubiquitin and proteasome dependence. In (B), WT cells expressing Ura3-HA or Ura3-HA-degron fusion proteins were pretreated with drug vehicle (–) or MG132 (+) for 2 hr prior to sample preparation. Proteins were detected by western blotting and quantified as in (A). In (C), degron constructs were expressed in a WT <em>UBA1</em> strain (–) or a mutant <em>uba1-204</em> strain (+) that is defective in ubiquitylation at nonpermissive temperature (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ghaboosi and Deshaies 2007</a>). Cells were processed as described in <em>Materials and Methods</em>. (E) Doa10-dependent steady-state levels were measured in the WT strain SM4460 (+) and <em>doa10</em>∆ mutant SM4820 (–). Error bars represent 1 SD of the mean. Vertical dotted lines indicate where separate gels are spliced together.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468561/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468561/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Degron tester set sequences and properties</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">Proteasome-mediated degradation generally requires ubiquitylation of the substrate; however, ubiquitin-independent signals also exist (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Erales and Coffino 2014</a>). To determine whether degradation of our degron-containing proteins was ubiquitin-dependent, we analyzed levels of the <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron constructs in the <em><a href="http://www.yeastgenome.org/locus/S000001693/overview">uba1</a>-204</em> mutant, which encodes a temperature-sensitive mutation of the sole E1 ubiquitin activating enzyme (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Ghaboosi and Deshaies 2007</a>), and is required for ubiquitin conjugation. At the nonpermissive temperature, the <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron protein levels were around two- to eight-fold higher in the <em><a href="http://www.yeastgenome.org/locus/S000001693/overview">uba1</a>-204</em> strain than in the WT strain at the nonpermissive temperature (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3, C and D</a>, bottom). This trend of stabilization was mirrored in cells treated with the proteasome inhibitor MG132 (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3, C and D</a>, top). Together, these data indicate that degradation of all members of the degron tester set require the full ubiquitin-proteasome pathway.</p></div><div id="sec-14" class="subsection"><h3>All members of the degron-tester set require Hsp40 (Ydj1) and Hsp70 (Ssa1/2) for degradation</h3><p id="p-37">The chaperone Hsp70 and its cochaperone Hsp40 have been shown to play a critical role in the degradation of numerous ERAD-C and CytoQC substrates (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Heck <em>et al.</em> 2010</a>; <a id="xref-ref-51-7" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Nakatsukasa <em>et al.</em> 2008</a>; <a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Prasad <em>et al.</em> 2010</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Shiber <em>et al.</em> 2013</a>). These chaperones may act upstream or downstream of E3 ubiquitin ligase activity (or both), but prior to degradation by the proteasome. Interestingly, deletion of <em><a href="http://www.yeastgenome.org/locus/S000000004/overview">SSA1</a>/<a href="http://www.yeastgenome.org/locus/S000003947/overview">SSA2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005008/overview">YDJ1</a></em> stabilized every degron-containing protein in our tester set, albeit to varying extents (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The observation that every degron construct requires the Hsp70s and Hsp40 for turnover, whereas their E3 ligase dependency for degradation varies, suggests that the discrimination step for these distinct CytoQC substrates is mediated by E3 ligases rather than the Hsp40 or Hsp70 chaperones.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The cochaperone and chaperone proteins Hsp40 and Hsp70 are required for efficient degradation of Ura3-degron proteins. (A, B) Steady-state western blots of Ura3-HA and Ura3-HA-degron proteins in WT (+) or deletion mutants (&#x394;) for ydj1&#x394; (SM 4460 and SM4819) (A) and ssa1&#x394; ssa2&#x394; (SM5755 and SM5774) (B). Quantification of Ura3-HA and Ura3-HA-degron proteins in the deletion mutants relative to WT is shown. Error bars represent 1&#xA0;SD of the mean. Vertical dotted lines indicate where separate gels are spliced together." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1168411460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The cochaperone and chaperone proteins Hsp40 and Hsp70 are required for efficient degradation of Ura3-degron proteins. (A, B) Steady-state western blots of Ura3-HA and Ura3-HA-degron proteins in WT (+) or deletion mutants (&#x394;) for &lt;em&gt;ydj1&lt;/em&gt;&#x394; (SM 4460 and SM4819) (A) and &lt;em&gt;ssa1&lt;/em&gt;&#x394; &lt;em&gt;ssa2&lt;/em&gt;&#x394; (SM5755 and SM5774) (B). Quantification of Ura3-HA and Ura3-HA-degron proteins in the deletion mutants relative to WT is shown. Error bars represent 1&#xA0;SD of the mean. Vertical dotted lines indicate where separate gels are spliced together.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1853/F4.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/7/1853/F4.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468483" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">The cochaperone and chaperone proteins Hsp40 and Hsp70 are required for efficient degradation of Ura3-degron proteins. (A, B) Steady-state western blots of Ura3-HA and Ura3-HA-degron proteins in WT (+) or deletion mutants (Δ) for <em>ydj1</em>Δ (SM 4460 and SM4819) (A) and <em>ssa1</em>Δ <em>ssa2</em>Δ (SM5755 and SM5774) (B). Quantification of Ura3-HA and Ura3-HA-degron proteins in the deletion mutants relative to WT is shown. Error bars represent 1 SD of the mean. Vertical dotted lines indicate where separate gels are spliced together.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>The E3 ligases San1 and Ubr1 only very modestly affect the degradation of a small subset of degron-containing proteins</h3><p id="p-39">In addition to <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>, other E3 ligases have been implicated in CytoQC degradation. Mutations in <em><a href="http://www.yeastgenome.org/locus/S000002550/overview">SAN1</a></em> and/or <em><a href="http://www.yeastgenome.org/locus/S000003416/overview">UBR1</a></em> have been reported to stabilize a variety of CytoQC substrates (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Eisele and Wolf 2008</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Guerriero <em>et al.</em> 2013</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Heck <em>et al.</em> 2010</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Nillegoda <em>et al.</em> 2010</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Park <em>et al.</em> 2013</a>; <a id="xref-ref-58-4" class="xref-bibr" href="#ref-58">Prasad <em>et al.</em> 2010</a>, <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">2012</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Summers <em>et al.</em> 2013</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Khosrow-Khavar <em>et al.</em> 2012</a>). Since many of our degron-containing proteins are degraded efficiently in <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆ cells, we wanted to determine if <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> and <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> are involved in their turnover. A small number of constructs in the degron library showed a minor increase in growth in the <em><a href="http://www.yeastgenome.org/locus/S000002550/overview">san1</a></em>∆ and <em><a href="http://www.yeastgenome.org/locus/S000003416/overview">ubr1</a></em>∆ mutants (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>E). We further examined by cycloheximide chase the stability of two of these that are members of the degron tester set, 10-6 and 12-32 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Loss of <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> or <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> only very slightly increased the half-life of 10-6 and 12-32, as compared to the WT strain (for 10-6 half-life increased from 12.6 min in the WT strain to 16.6 and 25.7 min in the <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> and <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> mutants, respectively, and for 12-32, half-life went from 5.5 min in the WT strain to 7.9 and 9.1 min in the mutants), consistent with the modest differences observed in the growth test. Furthermore, steady-state analysis of these and other members of the tester set in the double mutant <em><a href="http://www.yeastgenome.org/locus/S000002550/overview">san1</a></em>∆ <em><a href="http://www.yeastgenome.org/locus/S000003416/overview">ubr1</a></em>∆ revealed no additional stabilization (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS3.ai">Figure S3</a>). We conclude that the <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> and <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> E3 ligases do not play a major role in the ubiquitylation of our degron constructs. This finding is in contrast to <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆, which almost completely stabilizes some of the degron constructs (cf. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS2.ai">Figure S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS3.ai">Figure S3</a> to <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3E</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Lack of UBR1 or SAN1 has only a minor effect on the degradation of a few Ura3-HA-degron proteins in the tester set. (A) and (B). Cyclohexamide chase and quantitation are shown for degron constructs 10-6 and 12-32, respectively, which showed slightly improved growth in the single mutants ubr1&#x2206; (SM5745) and san1&#x2206; (SM5746) vs. WT (SM4460). Hexokinase (HexK) is the loading control." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1168411460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Lack of &lt;em&gt;UBR1&lt;/em&gt; or &lt;em&gt;SAN1&lt;/em&gt; has only a minor effect on the degradation of a few Ura3-HA-degron proteins in the tester set. (A) and (B). Cyclohexamide chase and quantitation are shown for degron constructs 10-6 and 12-32, respectively, which showed slightly improved growth in the single mutants &lt;em&gt;ubr1&lt;/em&gt;&#x2206; (SM5745) and &lt;em&gt;san1&lt;/em&gt;&#x2206; (SM5746) &lt;em&gt;vs.&lt;/em&gt; WT (SM4460). Hexokinase (HexK) is the loading control.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1853/F5.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/7/1853/F5.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468578" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">Lack of <em>UBR1</em> or <em>SAN1</em> has only a minor effect on the degradation of a few Ura3-HA-degron proteins in the tester set. (A) and (B). Cyclohexamide chase and quantitation are shown for degron constructs 10-6 and 12-32, respectively, which showed slightly improved growth in the single mutants <em>ubr1</em>∆ (SM5745) and <em>san1</em>∆ (SM5746) <em>vs.</em> WT (SM4460). Hexokinase (HexK) is the loading control.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>The ubiquitin E3 ligase Ltn1 affects the degradation of some members of the degron tester set</h3><p id="p-41"><a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> (also called <a href="http://www.yeastgenome.org/locus/S000004861/overview">Rkr1</a>) is an E3 ligase shown to be part of the ribosomal quality control (RQC) complex. It is required for the ubiquitylation and degradation of nonstop polypeptides and proteins containing translation pause-inducing sequences, such as polylysine or polyarginine stretches (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bengtson and Joazeiro 2010</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brandman <em>et al.</em> 2012</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Shen <em>et al.</em> 2015</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wilson <em>et al.</em> 2007</a>). In a separate study, we identified <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">LTN1</a></em> in a screen for deletion mutants that stabilize a degron construct related to <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-CL1, called CL1*, (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Metzger 2009</a>), which is not a nonstop protein and does not contain a polybasic stretch. Given its importance in <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-CL1* PQC, we asked whether the <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ mutant also would stabilize proteins in our degron library. Comparison of the steady-state level of degron constructs in WT <em>vs.</em> <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ suggest that <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> contributes to the turnover of many of the degron constructs (10-13, 10-31, 10-15, 10-34, 10-40, 12-33, and CL1; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS3.ai">Figure S3</a>B), as does the quantitative growth test (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>E). Two of these constructs, 10-40 and 12-33, were further examined by cycloheximide chase analysis. As expected, both, showed significant stabilization in <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆, as compared to WT (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). In contrast, degron construct 10-6, which showed only very slight stabilization in <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ by growth and steady state analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>E and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS3.ai">Figure S3</a>B) also showed only slight stabilization in <a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a>∆ by cycloheximide chase (data not shown). It should be noted that, for some constructs for which improved growth was seen in <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆, in particular 10-43, we observed no difference in stability of the degron construct between WT and <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆, although a high molecular weight (HMW) species was apparent in the <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ mutant (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS3.ai">Figure S3</a>B and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS4.ai">Figure S4</a>; see <em>Discussion</em>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Lack of LTN1 affects the degradation of several Ura3-HA-degron proteins in the tester set. Western blots and quantitation of cycloheximide chase analysis of WT (SM4460) and ltn1&#x394; (SM5559) strains expressing the Ltn1-dependent degrons 12-33 and 10-40 [(A) and (B), respectively]." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1168411460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Lack of &lt;em&gt;LTN1&lt;/em&gt; affects the degradation of several Ura3-HA-degron proteins in the tester set. Western blots and quantitation of cycloheximide chase analysis of WT (SM4460) and &lt;em&gt;ltn1&lt;/em&gt;&#x394; (SM5559) strains expressing the Ltn1-dependent degrons 12-33 and 10-40 [(A) and (B), respectively].&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1853/F6.medium.gif" width="440" height="385"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/7/1853/F6.medium.gif" width="440" height="385"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468696" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-42" class="first-child">Lack of <em>LTN1</em> affects the degradation of several Ura3-HA-degron proteins in the tester set. Western blots and quantitation of cycloheximide chase analysis of WT (SM4460) and <em>ltn1</em>Δ (SM5559) strains expressing the Ltn1-dependent degrons 12-33 and 10-40 [(A) and (B), respectively].</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Ltn1 appears to affect Ura3-HA-degron constructs by a mechanism distinct from translational pausing</h3><p id="p-43">Recently, several groups have provided insight into the mechanism by which the RQC operates. <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> associates with ribosomes that are translationally stalled due to the presence of polylysine/arginine pause signals in the nascent polypeptide, or due to the synthesis of polylysine from the poly(A) tail of an mRNA lacking a stop codon. In both cases, <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> is well-positioned to ubiquitylate the emerging translation product (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Bengtson and Joazeiro 2010</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Brandman <em>et al.</em> 2012</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lyumkis <em>et al.</em> 2013</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Lyumkis <em>et al.</em> 2014</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Shao <em>et al.</em> 2015</a>; <a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Shen <em>et al.</em> 2015</a>). However, our degron constructs differ from the published Ltn1-dependent substrates, by having intact stop codons, and not encoding polylysine or other polymeric repeats (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/TableS3.xlsx">Table S3</a>). Although it remains possible that some degron sequences form stem-loop structures with the 3′ UTR, leading to translational stalling, it seems unlikely given that numerous degrons having diverse sequences are affected in the <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ mutant.</p><p id="p-44">To investigate the basis for Ltn1-dependent degradation of our constructs, we carried out further experiments with degron 12-33. We first examined whether the ubiquitin ligase activity of <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> was required for degradation. <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-12-33 was degraded in the WT strain (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>, lanes 1–12) and in the <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ mutant expressing plasmid-borne WT <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">LTN1</a></em> (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>, lanes 21–24), but stabilized in <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ cells with an empty vector or expressing <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a><sup>C1508A</sup></em>, a RING mutant devoid of E3 ubiquitin ligase activity (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7A</a>, lanes 13–20) (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Bengtson and Joazeiro 2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Braun <em>et al.</em> 2007</a>). We found a similar result for degron 10–40 (data not shown). Thus, the ubiquitin ligase activity of <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> is critical for the degradation of our constructs.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ltn1-mediated degradation of the Ura3-HA-degron protein 12-33 requires E3 ligase activity, and does not involve production of a nonstop protein or a translational pause signal (A) The ubiquitin ligase activity of Ltn1 is necessary for the degradation of an LTN1-dependent degron. Cycloheximide chase analysis of the Ura3-HA-12-33 degron construct expressed from plasmid pSM2743 in an ltn1&#x394; strain (SM5559) bearing plasmid-borne LTN1 (pSM2658), no LTN1 (EV; pSM171), or the Ltn1 RING domain mutation C1508A (ltn1RING; pSM2659). (B) Translation of Ura3-HA-12-33 correctly terminates at the intended stop codon. HA and Myc epitopes were individually probed in steady-state western blots of the original construct (lanes 1-2), an N-terminally Myc-tagged derivative (lanes 3&#x2013;4), or a derivative with Myc placed C-terminal to the stop codon, all expressed in WT (SM4460) or ltn1&#x394; (SM5559). In the merged image, HA and Myc signals are displayed in the green and red channels, respectively; protein containing both the HA and Myc epitopes appears yellow. (C) A stalled, truncated form of the Ura3-HA-Deg-Myc construct (as indicated by the presence of HA staining, but a lack of Myc staining) accumulates in ltn1&#x394; when the degron sequence is replaced with the known stall-inducing sequence (K12), but not with degron 12-33 (lanes 2 vs. 4, respectively). The Myc coding sequence shown in the schematic is immediately downstream of, and in frame with, K12 or degron 12-33." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1168411460" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ltn1-mediated degradation of the Ura3-HA-degron protein 12-33 requires E3 ligase activity, and does not involve production of a nonstop protein or a translational pause signal (A) The ubiquitin ligase activity of Ltn1 is necessary for the degradation of an &lt;em&gt;LTN1&lt;/em&gt;-dependent degron. Cycloheximide chase analysis of the Ura3-HA-12-33 degron construct expressed from plasmid pSM2743 in an &lt;em&gt;ltn1&lt;/em&gt;&#x394; strain (SM5559) bearing plasmid-borne &lt;em&gt;LTN1&lt;/em&gt; (pSM2658), no &lt;em&gt;LTN1&lt;/em&gt; (EV; pSM171), or the Ltn1 RING domain mutation C1508A (&lt;em&gt;ltn1&lt;sup&gt;RING&lt;/sup&gt;&lt;/em&gt;; pSM2659). (B) Translation of Ura3-HA-12-33 correctly terminates at the intended stop codon. HA and Myc epitopes were individually probed in steady-state western blots of the original construct (lanes 1-2), an N-terminally Myc-tagged derivative (lanes 3&#x2013;4), or a derivative with Myc placed C-terminal to the stop codon, all expressed in WT (SM4460) or &lt;em&gt;ltn1&lt;/em&gt;&#x394; (SM5559). In the merged image, HA and Myc signals are displayed in the green and red channels, respectively; protein containing both the HA and Myc epitopes appears yellow. (C) A stalled, truncated form of the Ura3-HA-Deg-Myc construct (as indicated by the presence of HA staining, but a lack of Myc staining) accumulates in &lt;em&gt;ltn1&lt;/em&gt;&#x394; when the degron sequence is replaced with the known stall-inducing sequence (K12), but not with degron 12-33 (lanes 2 &lt;em&gt;vs.&lt;/em&gt; 4, respectively). The Myc coding sequence shown in the schematic is immediately downstream of, and in frame with, K12 or degron 12-33.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1853/F7.medium.gif" width="440" height="402"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/7/1853/F7.medium.gif" width="440" height="402"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1853/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468550" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-45" class="first-child">Ltn1-mediated degradation of the Ura3-HA-degron protein 12-33 requires E3 ligase activity, and does not involve production of a nonstop protein or a translational pause signal (A) The ubiquitin ligase activity of Ltn1 is necessary for the degradation of an <em>LTN1</em>-dependent degron. Cycloheximide chase analysis of the Ura3-HA-12-33 degron construct expressed from plasmid pSM2743 in an <em>ltn1</em>Δ strain (SM5559) bearing plasmid-borne <em>LTN1</em> (pSM2658), no <em>LTN1</em> (EV; pSM171), or the Ltn1 RING domain mutation C1508A (<em>ltn1<sup>RING</sup></em>; pSM2659). (B) Translation of Ura3-HA-12-33 correctly terminates at the intended stop codon. HA and Myc epitopes were individually probed in steady-state western blots of the original construct (lanes 1-2), an N-terminally Myc-tagged derivative (lanes 3–4), or a derivative with Myc placed C-terminal to the stop codon, all expressed in WT (SM4460) or <em>ltn1</em>Δ (SM5559). In the merged image, HA and Myc signals are displayed in the green and red channels, respectively; protein containing both the HA and Myc epitopes appears yellow. (C) A stalled, truncated form of the Ura3-HA-Deg-Myc construct (as indicated by the presence of HA staining, but a lack of Myc staining) accumulates in <em>ltn1</em>Δ when the degron sequence is replaced with the known stall-inducing sequence (K12), but not with degron 12-33 (lanes 2 <em>vs.</em> 4, respectively). The Myc coding sequence shown in the schematic is immediately downstream of, and in frame with, K12 or degron 12-33.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Next, we wished to ask whether the stop codon in <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-12-33 is indeed functional, rather than resulting in a nonstop protein. To do so we placed a Myc epitope immediately downstream of the stop codon, or at the N-terminus as a control. As expected, the parental construct <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-12-33 without a Myc tag was detected with anti-HA antibodies (and not with anti-Myc), and showed increased steady-state levels in <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ <em>vs.</em> WT (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7B</a>, lanes 1 and 2). The addition of Myc at the N-terminus of <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-12-33 resulted in a protein with a noticeable size shift (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7B</a>, lanes 3 and 4) that is stabilized in <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆, and is detectable with both anti-HA and anti-Myc antibodies (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7B</a>, αHA, αMyc, and merge panels, yellow band). Importantly, when Myc was placed immediately downstream of the stop codon in <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-12-33, no size-shifted, Myc-tagged product is evident; and indeed the stabilization pattern is indistinguishable from that of the parental construct (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7B</a>, cf. lanes 5 and 6 to lanes 1 and 2). Similar results were also observed for <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-10-40 (data not shown). Taken together, these findings indicate that translation of <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-12-33 (and <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-10-40), correctly terminates at the intended stop codon, and that its recognition by <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> must therefore not involve nonstop polypeptide production.</p><p id="p-47">Finally, we wished to examine whether degron 12-33 promotes ribosome stalling, which can result in truncated translation products that are rapidly degraded in WT cells, but are stable when <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">LTN1</a></em> is deleted (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Bengtson and Joazeiro 2010</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Brandman <em>et al.</em> 2012</a>; <a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Shen <em>et al.</em> 2015</a>). We placed an in-frame Myc epitope just C-terminal to degron 12-33. As a control, we replaced the 12-33 degron with a stretch of 12 lysine codons (K12) (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7C</a>), known to induce translational arrest and Ltn1-dependent protein degradation (<a id="xref-ref-4-6" class="xref-bibr" href="#ref-4">Bengtson and Joazeiro 2010</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Braun <em>et al.</em> 2007</a>). These constructs allowed us to ask whether we could detect only fully translated proteins containing both the HA and Myc epitopes, or whether we could also detect truncated proteins containing solely HA (and not Myc), indicative of a ribosome stalling sequence upstream of Myc. Consistent with previous results (<a id="xref-ref-4-7" class="xref-bibr" href="#ref-4">Bengtson and Joazeiro 2010</a>; <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Brandman <em>et al.</em> 2012</a>; <a id="xref-ref-67-5" class="xref-bibr" href="#ref-67">Shen <em>et al.</em> 2015</a>) expression of <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-HA-K12-Myc</em> gave rise to two species of proteins: a slower migrating species detectable with both anti-HA and anti-Myc antibodies, and a faster migrating form detectable with anti-HA but not anti-Myc antibodies (<a id="xref-fig-7-9" class="xref-fig" href="#F7">Figure 7C</a>, left panels, lane 2). This faster migrating species (<a id="xref-fig-7-10" class="xref-fig" href="#F7">Figure 7C</a>, labeled “stalled”) results from arrested translation due to the polylysine tract, and is observed only when it is stabilized in <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ (<a id="xref-fig-7-11" class="xref-fig" href="#F7">Figure 7C</a>, lanes 1 and 2). The slower-migrating protein species detected with both anti-HA and anti-Myc antibodies represents the full-length protein that is produced from translation through the pause site. By contrast, we failed to observe any truncated species in either WT or <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ for our degron construct 12-33 (<a id="xref-fig-7-12" class="xref-fig" href="#F7">Figure 7C</a>, lanes 3 and 4) or 12-40 (data not shown), indicating that translation of these degron sequences apparently do not cause the ribosome to stall.</p><p id="p-48">Interestingly, there were equivalent levels of the full-length <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-Deg12-33-Myc in the WT and the <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ strains (<a id="xref-fig-7-13" class="xref-fig" href="#F7">Figure 7C</a>, lanes 3 and 4). Thus, the 12-33 sequence does not function as a degron when it is not the most C-terminal element of the protein. Similar results were seen for degron 10-40 (data not shown). Taken together, our results demonstrate that <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> can mediate the degradation of proteins other than nonstop proteins and ribosome pause-inducing proteins, and suggest that <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> activity may require that the degron be at the very C-terminus of the protein.</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-49">A major focus in the proteostasis field is to define the recognition codes that direct particular substrates to specific E3 ligases (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Deshaies and Joazeiro 2009</a>; <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Fredrickson and Gardner 2012</a>; <a id="xref-ref-51-8" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>; <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Ravid <em>et al.</em> 2006</a>). Studying PQC using naturally occurring misfolded proteins is complicated by the fact that the degradation signal is unknown, could exist anywhere in the protein, and multiple degradation signals are likely to be present in a single unfolded protein. Degrons are signals, often in the form of a peptide, defined by their ability promote a protein’s degradation (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Varshavsky 1991</a>). In previous studies, a library of peptides appended to a nuclear-directed form of GFP revealed fundamental principles of nuclear PQC (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Fredrickson <em>et al.</em> 2011</a>, <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">2013</a>). Here, to facilitate the analysis of CytoQC and to build on an earlier screen by Gilon and coworkers (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gilon <em>et al.</em> 1998</a>, <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">2000</a>), we generated a comprehensive collection of peptides that possess degron function when appended to <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>. Having many different short sequences fused to the same cytosolic reporter protein will simplify the analysis of CytoQC recognition signals, by limiting variation to a small, defined region of the substrate.</p><p id="p-50">For creating this CytoQC degron library, we chose HA-tagged <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a> as the reporter protein because it is cytosolic and genetically tractable. Sheared DNA fragments were cloned into the C-terminus of <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA. Degrons were identified as sequences that confer a defective growth phenotype on –Ura medium in a WT strain, but for which growth is spared in a proteasome mutant. This rescued growth when the proteasome is impaired indicates that <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a> is functional, and thus unlikely misfolded, and that the degradation signal lies within the cloned degron sequence. Our degron collection contains 77 <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron constructs ranging in size from seven to 58 amino acids in length (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/TableS3.xlsx">Table S3</a>). For detailed analyses we focused on a representative “tester set” of 13 degron constructs. For most of these, the half-life is extremely short (&lt;3 min). This degradation rate is considerably faster than that for previously reported model misfolded CytoQC substrates, such as <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-2 and <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-3, ∆2GFP, ∆ssPrA, ∆ssCPY*GFP, slGFP, NBD <a href="http://www.yeastgenome.org/locus/S000001692/overview">Ste6</a>*, and a variety of Ts<sup>–</sup> mutant proteins, which have half-lives ranging from 10 to 30 min (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Eisele and Wolf 2008</a>; <a id="xref-ref-30-5" class="xref-bibr" href="#ref-30">Guerriero <em>et al.</em> 2013</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Heck <em>et al.</em> 2010</a>; <a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Nillegoda <em>et al.</em> 2010</a>; <a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Park <em>et al.</em> 2013</a>; <a id="xref-ref-58-5" class="xref-bibr" href="#ref-58">Prasad <em>et al.</em> 2010</a>, <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">2012</a>; <a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Summers <em>et al.</em> 2013</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Khosrow-Khavar <em>et al.</em> 2012</a>). The consistently rapid degradation we observe for members of our <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-degron library may reflect a bias imposed by our screening criteria for a Ura– phenotype, which could have demanded an exceedingly low level of protein, a situation that may only be achieved for constructs that are very rapidly degraded. Alternatively, the positioning of degrons at the C-terminus of well-folded <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a> may allow ideal exposure to E3 ligases, hastening their ubiquitylation and degradation. Because members of our degron collection are degraded so soon after synthesis, they may prove advantageous for identifying components that act on nascent proteins just after their release from the ribosome, rather than machinery that acts on other types of CytoQC substrates, such as previously well-folded proteins that have sustained damage or become unfolded. It will be of interest to test positional effects of our degrons and their transferability to other reporter proteins. To date, we have examined degrons CL1, 10-1, 10-13, and 10-43, and all retain their degron properties when transferred to the C-terminus of GFP (data not shown).</p><p id="p-51">To determine the E3 recognition pattern for all members of our degron library, we tested for restored growth on –Ura medium in the E3 mutants for the E3s most prominently implicated in CytoQC to date, which include <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>, <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a>, <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a>, and <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>E). For members of the tester set, we also directly determined their half-lives in WT and mutant cells, and their steady-state levels (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS2.ai">Figure S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS3.ai">Figure S3</a>). It is notable that of all the E3 mutants tested, the <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆ mutant shows the most profound effect on stabilization, and acts on about half of our degron library constructs. The <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ mutant also stabilizes numerous constructs, but stabilizes them to a lesser extent than <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆. Notably, only modest effects were seen for <em><a href="http://www.yeastgenome.org/locus/S000003416/overview">ubr1</a></em>∆ (or <em><a href="http://www.yeastgenome.org/locus/S000004014/overview">ubr2</a></em>∆) and <em><a href="http://www.yeastgenome.org/locus/S000002550/overview">san1</a></em>∆, with the growth of far fewer degron constructs affected in these latter mutants, and to a lesser extent. Below, we discuss in depth our findings with these E3s. We note that we have not yet tested the E3s <a href="http://www.yeastgenome.org/locus/S000003109/overview">Hul5</a> and <a href="http://www.yeastgenome.org/locus/S000000927/overview">Rsp5</a>, which were shown to act on cytosolic proteins that are destabilized by heat shock (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fang <em>et al.</em> 2011</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">2014</a>). Nor have we examined the E3 ligase <a href="http://www.yeastgenome.org/locus/S000002674/overview">Hel2</a>, shown to play a broad role in nascent protein CytoQC (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Duttler <em>et al.</em> 2013</a>). However, it will be of interest to study mutants defective for these and many other candidate UPS components.</p><p id="p-52"><a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>, which localizes to the ER membrane and the INM, plays a key role in ERAD-C in which it ubiquitylates membrane proteins with a cytosolic lesion, and it was recently shown to mediate prERAD—a process in which the cytosolic precursors of some post-translationally secreted proteins that fail to be translocated are degraded (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ast <em>et al.</em> 2014</a>). In addition, <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> acts on the nuclear proteins MATα2 and misfolded <a href="http://www.yeastgenome.org/locus/S000003372/overview">Ndc10</a> (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Deng and Hochstrasser 2006</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Furth <em>et al.</em> 2011</a>; <a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Ravid <em>et al.</em> 2006</a>). Previous findings by us and others showing that <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> also mediates the ubiquitylation of the CytoQC substrate <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-CL1 raised the intriguing possibility that the ER or nuclear membrane may serve as a major platform for CytoQC (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Gilon <em>et al.</em> 1998</a>; <a id="xref-ref-51-9" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>; <a id="xref-ref-61-6" class="xref-bibr" href="#ref-61">Ravid <em>et al.</em> 2006</a>). Here, we found here that many (about half), of our degron-containing proteins show improved growth in <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆ (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3E</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>), suggesting that many CytoQC substrates in addition to <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-CL1 may also be ubiquitylated at the face of the ER membrane or possibly the INM. Thus the ER/INM may serve as a convenient way to organize the cellular machinery used for ubiquitylation of a substantial number of substrates, although clearly not all, since importantly, many of our degron constructs show no stabilization in <em><a href="http://www.yeastgenome.org/locus/S000001292/overview">doa10</a></em>∆ mutant (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3E</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS2.ai">Figure S2</a>). The mechanism whereby CytoQC substrates are delivered to the membrane for degradation presents an interesting problem, and could involve specific guide or escort proteins that our degron constructs could help to identify. Alternatively, <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-dependence for the degradation of certain constructs might reflect the possibility that these constructs are synthesized on ER-bound ribosomes (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kraut-Cohen <em>et al.</em> 2013</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Reid and Nicchitta 2015</a>), and, thus, <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> would be among the first E3s that they encounter.</p><p id="p-53">Another E3 ligase that affected the stability of many members of the degron library was <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a>, recently shown to be a key component of the RQC complex (composed of <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a>, <a href="http://www.yeastgenome.org/locus/S000002741/overview">Rqc1</a>, <a href="http://www.yeastgenome.org/locus/S000005930/overview">Rqc2</a>/<a href="http://www.yeastgenome.org/locus/S000005930/overview">Tae2</a>) (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Brandman <em>et al.</em> 2012</a>; <a id="xref-ref-67-6" class="xref-bibr" href="#ref-67">Shen <em>et al.</em> 2015</a>). <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> associates with the 60S ribosomal subunit, and ubiquitylates incompletely synthesized nascent proteins that are translationally stalled due a faulty mRNA, or the presence of a ribosome pause sequence such as a polybasic tract of amino acids (<a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Brandman <em>et al.</em> 2012</a>). Several of the degron constructs in our tester set are stabilized in the <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ mutant, as evidenced by steady-state or cycloheximide chase analysis (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS3.ai">Figure S3</a>). We examined degron 12-33 in depth, and showed that this sequence does not cause ribosome pausing, in contrast to what we see for a classical engineered polylysine pause sequence (K12) (<a id="xref-fig-7-14" class="xref-fig" href="#F7">Figure 7D</a>). Furthermore, the gel mobility of the Ua3-HA-12-33 degron fusion protein is considerably greater than that of <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS3.ai">Figure S3</a>B), suggesting that the entire protein is synthesized (including the degron sequence), and therefore is likely to be completely released from the ribosome. Thus, <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> may have a PQC role for nascent proteins that have been newly released from the ribosome, or may influence certain aspects of cellular proteostasis, in addition to its well-characterized role in handling paused proteins that remain associated with the 60S ribosomal subunit.</p><p id="p-54">It is notable that, for one of our tester set degron constructs, <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-10-43, we saw significantly improved growth in <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ <em>vs.</em> WT, despite observing only minor stabilization of the degron construct. Instead, we observed a HMW protein species that is present in the <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ mutant, but is absent in WT cells (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS4.ai">Figure S4</a>, A–C; note that the HMW species is sometimes observed as a discrete band or as multiple bands). The HMW bands are also absent when the other RQC components, <a href="http://www.yeastgenome.org/locus/S000002741/overview">Rqc1</a> and <a href="http://www.yeastgenome.org/locus/S000005930/overview">Rqc2</a>/<a href="http://www.yeastgenome.org/locus/S000005930/overview">Tae2</a>, are absent (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS4.ai">Figure S4</a>D). The additional forms of <a href="http://www.yeastgenome.org/locus/S000000747/overview">Ura3</a>-HA-10-43 likely account for the improved growth on –Ura medium. We hypothesize this HMW species may correspond to carboxy-terminal Ala and Thr extensions (‘CAT” tails), that have been shown to arise from nontemplated addition of these amino acids, and is mediated by the <a href="http://www.yeastgenome.org/locus/S000005930/overview">Rqc2</a> component of the RQC complex (<a id="xref-ref-67-7" class="xref-bibr" href="#ref-67">Shen <em>et al.</em> 2015</a>). Thus, 10-43 and other degrons in our library that show HMW species in the <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">ltn1</a></em>∆ mutant, may provide insight into signals that prompt this unusual modification. Overall, it is important to understand the nature and breadth of <a href="http://www.yeastgenome.org/locus/S000004861/overview">Ltn1</a> substrates, as a mutation in the mouse <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">LTN1</a></em> homolog, <em>lister</em>, results in a neurodegenerative disorder similar to spinal-bulbar muscular atrophy (SBMA) and amyotrophic lateral sclerosis (ALS) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chu <em>et al.</em> 2009</a>). A more thorough understanding <em><a href="http://www.yeastgenome.org/locus/S000004861/overview">LTN1</a></em> function could provide insight into human neurodegenerative disorders.</p><p id="p-55">Two other E3s that have been implicated in CytoQC are <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> and <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a>. For some well-studied model substrates, full stabilization is observed only in mutants deleted for both, while some CytoQC substrates appear to be ubiquitylated mainly by one or the other of these E3s (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Eisele and Wolf 2008</a>; <a id="xref-ref-30-6" class="xref-bibr" href="#ref-30">Guerriero <em>et al.</em> 2013</a>; <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Heck <em>et al.</em> 2010</a>; <a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Nillegoda <em>et al.</em> 2010</a>; <a id="xref-ref-56-5" class="xref-bibr" href="#ref-56">Park <em>et al.</em> 2013</a>; <a id="xref-ref-58-6" class="xref-bibr" href="#ref-58">Prasad <em>et al.</em> 2010</a>, <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">2012</a>; <a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Summers <em>et al.</em> 2013</a>). While <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> is cytosolic, <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> is nuclear (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Fredrickson and Gardner 2012</a>). The <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a>-dependent degradation observed for many CytoQC substrates, along with the direct observation of nuclear localization of these substrates, has provided evidence that nuclear import contributes to their ubiquitylation and degradation. Interestingly, very few members of our degron library showed any dependence on either of these E3s for degradation, and the two that did (10-6 and 12-32) showed only very modest stabilization in either the <em>ubr</em>∆<em>1</em> or <em><a href="http://www.yeastgenome.org/locus/S000002550/overview">san1</a></em>∆ single or the double mutants (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>, and also <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS1.ai">Figure S1</a>E and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/FigureS3.ai">Figure S3</a>). It is worthwhile pointing out that most <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a> and <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> substrates characterized to date have fairly long half-lives, whereas all of our degron constructs are exceedingly short-lived, as discussed above. It may be that our degron constructs are degraded so soon after being released from the ribosome that they may never have the chance to encounter <a href="http://www.yeastgenome.org/locus/S000003416/overview">Ubr1</a> or be transported to the nucleus, where <a href="http://www.yeastgenome.org/locus/S000002550/overview">San1</a>-dependent degradation occurs.</p><p id="p-56">In addition to E3s, chaperones play a key role in PQC, either by attempting to fold misfolded proteins, or by preventing misfolded proteins from aggregating so that they remain accessible to the PQC machinery (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2011</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">McClellan <em>et al.</em> 2005a</a>, <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">2005b</a>; <a id="xref-ref-51-10" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>). The PQC substrates, Ura3-CL1 and <a href="http://www.yeastgenome.org/locus/S000001692/overview">Ste6</a>* are not ubiquitylated in the absence of the Hsp40 <a href="http://www.yeastgenome.org/locus/S000005008/overview">Ydj1</a> or the Hsp70 <a href="http://www.yeastgenome.org/locus/S000000004/overview">Ssa1p</a>, indicating that this cochaperone/chaperone pair functions upstream of the ubiquitylation machinery (<a id="xref-ref-51-11" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Nakatsukasa <em>et al.</em> 2008</a>), in some way assisting in delivery to the E3. We observed dependency on <a href="http://www.yeastgenome.org/locus/S000005008/overview">Ydj1</a> and <a href="http://www.yeastgenome.org/locus/S000000004/overview">Ssa1</a> for degradation for all of the degron-containing proteins in our tester set (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). This finding suggests that <a href="http://www.yeastgenome.org/locus/S000005008/overview">Ydj1</a> and <a href="http://www.yeastgenome.org/locus/S000000004/overview">Ssa1</a> may not provide specificity for directing substrates to a particular E3, but it clearly indicates that these chaperones are very important for a variety of CytoQC substrates. It will be interesting to determine if the role of <a href="http://www.yeastgenome.org/locus/S000005008/overview">Ydj1</a> and <a href="http://www.yeastgenome.org/locus/S000000004/overview">Ssa1</a> for our Ura3-degron constructs is to maintain their solubility so that they can be delivered to, or recognized by, a particular subset of E3s.</p><p id="p-57">Most of the sequences we isolated as degrons originate from ORFs, but are out-of-frame translations (including reverse complement), or are from nonprotein-coding sequences such as rRNA genes and intergenic sequences. Thus, the majority of the peptides we isolated as degrons are of sequences that do not naturally exist in the yeast proteome, as is also the case for the degron CL1 (<a id="xref-ref-51-12" class="xref-bibr" href="#ref-51">Metzger <em>et al.</em> 2008</a>). Nevertheless, our degrons are still expected to be informative, since degron recognition within aberrant proteins is unlikely to be at the level of primary sequence, but instead probably involves physical characteristics such as exposed hydrophobicity or propensity to aggregate. It is notable that at least one human disease results from a frame shift mutation that creates a novel degron (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Su <em>et al.</em> 2011</a>). Other collections of short peptides with degron function exist, also constructed by fusing synthetic sequences or genomic DNA or cDNA to reporter proteins (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Fredrickson <em>et al.</em> 2011</a>; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Gilon <em>et al.</em> 1998</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sadis <em>et al.</em> 1995</a>). Those studies screened for degrons that exhibited a specific E2 or E3 dependency or filtered out strong degrons. Unlike these previous screens, the present screen did not impose any such filters; we simply asked for defective growth on –Ura medium, anticipating that poor growth would reflect decreased protein half-life. Accordingly, we were successful in isolating a larger number of degrons, for a wider variety of E3s than previously published screens.</p><p id="p-58">Our degron library greatly expands the number of <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-dependent degrons available for analysis. Previous studies of Deg1, CL1, and DegAB have proven valuable to begin to dissect the recognition mechanism of <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>, and led to the suggestion that <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> recognizes the hydrophobic face of an amphipathic α-helix present in its substrates (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Ravid and Hochstrasser 2008</a>). Substituting hydrophobic residues with polar ones in these degrons was shown to destroy degron function, in some but not all cases (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Furth <em>et al.</em> 2011</a>; <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Gilon <em>et al.</em> 1998</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Johnson <em>et al.</em> 1998</a>). However, with only a few examples available up to now, it has remained unclear if all <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-dependent degrons must obligatorily contain an amphipathic α-helix, and our understanding of the recognition motif(s) for <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a> has remained incomplete. Here, we report many new <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-dependent degrons. Secondary structure predictions (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bryson <em>et al.</em> 2005</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Jones 1999</a>) of our sequences (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.027953/-/DC1/TableS3.xlsx">Table S3</a>) suggest that, while some of these conform to the prevailing model, some are predicted to contain α-helices with no clear hydrophobic face, or to contain no helical structure at all. Furthermore, some degrons for which we did not observe any <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-dependence, are predicted to form amphipathic helices, suggesting this feature is not sufficient to dictate specificity for <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>. Our preliminary sequence analysis suggests that a much closer bioinformatic examination, along with mutagenesis experiments to determine minimal length and critical residues of <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>-dependent degrons, will be necessary to more fully understand the general basis for recognition by <a href="http://www.yeastgenome.org/locus/S000001292/overview">Doa10</a>.</p><p id="p-59">The degron library characterized in this study will provide a rich resource for genetic, biochemical, and cell biological studies to identify novel E3s and other UPS components involved in CytoQC, as well as to determine sequence features important for substrate recognition in PQC. Growth on –Ura medium and protein stability assays, together with suppressor screens, can be used to genetically identify new PQC components that act on particular degron constructs. We are currently examining the role of candidate PQC gene products for several of our degron constructs, as well as conducting genome-wide suppressor screens to seek new components. In the work reported here, we looked mainly at single gene deletions or conditional mutations to assess growth, steady state protein levels, and protein half-life. However it is possible that the involvement of a particular E3 or chaperone could be masked by functional redundancy. Thus, we are generating strains deleted for multiple E3s, E2s, or chaperones for further testing. Important questions recently raised for CytoQC are the roles of nuclear import, and aggregation. By tagging our degron constructs with GFP, it will be possible to directly assess their localization and aggregation properties in a broad range of mutants, further extending the utility and versatility of the degrons identified in this study.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">We thank Rich Gardner, Randy Hampton, Karen Arndt, Davis Ng, Ray Deshaies, and Avrom Caplan for strains and plasmids. This work was supported by a grant (R01 GM051508) from the National Institutes of Health (NIH) to S.M., and a Johns Hopkins Institute for Basic Biomedical Science Bridge grant.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-61">Supplemental material is available online at <a href="http://www.g3journal.org/content/suppl/2016/04/26/g3.116.027953.DC1">http://www.g3journal.org/content/suppl/2016/04/26/g3.116.027953.DC1</a></p></li><li class="fn" id="fn-5"><p id="p-62"><em>Communicating editor: C. Boone</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-02-05"><span class="received-label">Received </span>February 5, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-04-21"><span class="accepted-label">Accepted </span>April 21, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Maurer <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.1" data-doi="10.1242/jcs.144386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ast</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aviram</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuartzman</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuldiner</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A cytosolic degradation pathway, prERAD, monitors pre-inserted secretory pathway proteins.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">Pt 14</span>): <span class="cit-fpage">3017</span>–<span class="cit-lpage">3023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.144386%26rft_id%253Dinfo%253Apmid%252F24849653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjEyNy8xNC8zMDE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.2" data-doi="10.1126/science.1141448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balch</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morimoto</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dillin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Adapting proteostasis for disease intervention.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>(<span class="cit-issue">5865</span>): <span class="cit-fpage">916</span>–<span class="cit-lpage">919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBalch%26rft.auinit1%253DW.%2BE.%26rft.volume%253D319%26rft.issue%253D5865%26rft.spage%253D916%26rft.epage%253D919%26rft.atitle%253DAdapting%2BProteostasis%2Bfor%2BDisease%2BIntervention%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1141448%26rft_id%253Dinfo%253Apmid%252F18276881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2NS85MTYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.3" data-doi="10.1038/35050524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bays</span>  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seelig</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joazeiro</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampton</span>  <span class="cit-name-given-names">R. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Hrd1p/Der3p is a membrane-anchored ubiquitin ligase required for ER-associated degradation.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">24</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DBays%26rft.auinit1%253DN.%2BW.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D29%26rft.atitle%253DHrd1p%252FDer3p%2Bis%2Ba%2Bmembrane-anchored%2Bubiquitin%2Bligase%2Brequired%2Bfor%2BER-associated%2Bdegradation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35050524%26rft_id%253Dinfo%253Apmid%252F11146622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35050524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11146622&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166146400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.4" data-doi="10.1038/nature09371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bengtson</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joazeiro</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Role of a ribosome-associated E3 ubiquitin ligase in protein quality control.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>(<span class="cit-issue">7314</span>): <span class="cit-fpage">470</span>–<span class="cit-lpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBengtson%26rft.auinit1%253DM.%2BH.%26rft.volume%253D467%26rft.issue%253D7314%26rft.spage%253D470%26rft.epage%253D473%26rft.atitle%253DRole%2Bof%2Ba%2Bribosome-associated%2BE3%2Bubiquitin%2Bligase%2Bin%2Bprotein%2Bquality%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09371%26rft_id%253Dinfo%253Apmid%252F20835226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09371&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20835226&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282090200045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.5" data-doi="10.1016/j.cell.2012.10.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandman</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart-Ornstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A ribosome-bound quality control complex triggers degradation of nascent peptides and signals translation stress.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">151</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1042</span>–<span class="cit-lpage">1054</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBrandman%26rft.auinit1%253DO.%26rft.volume%253D151%26rft.issue%253D5%26rft.spage%253D1042%26rft.epage%253D1054%26rft.atitle%253DA%2Bribosome-bound%2Bquality%2Bcontrol%2Bcomplex%2Btriggers%2Bdegradation%2Bof%2Bnascent%2Bpeptides%2Band%2Bsignals%2Btranslation%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.10.044%26rft_id%253Dinfo%253Apmid%252F23178123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.10.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23178123&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311423500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.6" data-doi="10.1128/MCB.01947-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crisucci</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Identification of Rkr1, a nuclear RING domain protein with functional connections to chromatin modification in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">2800</span>–<span class="cit-lpage">2811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01947-06%26rft_id%253Dinfo%253Apmid%252F17283062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI3LzgvMjgwMCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-web" id="cit-6.7.1853.7"><div class="cit-metadata unstructured">Bryson, K., L.J. McGuffin, R.L. Marsden, J.J. Ward, J.S. Sodhi <em>et al.</em>, 2005 Protein structure prediction servers at University College London. Nucleic Acids Res. <strong>33</strong>(Web Server issue):W36–W38.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.8" data-doi="10.1016/j.molcel.2010.10.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buchberger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bukau</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Protein quality control in the cytosol and the endoplasmic reticulum: brothers in arms.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">40</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">238</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBuchberger%26rft.auinit1%253DA.%26rft.volume%253D40%26rft.issue%253D2%26rft.spage%253D238%26rft.epage%253D252%26rft.atitle%253DProtein%2Bquality%2Bcontrol%2Bin%2Bthe%2Bcytosol%2Band%2Bthe%2Bendoplasmic%2Breticulum%253A%2Bbrothers%2Bin%2Barms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.10.001%26rft_id%253Dinfo%253Apmid%252F20965419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.10.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20965419&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284028100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.9" data-doi="10.1186/gb-2006-7-10-r100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamprecht</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">I. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">CellProfiler: image analysis software for identifying and quantifying cell phenotypes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">R100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DCarpenter%26rft.auinit1%253DA.%2BE.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253DR100%26rft.epage%253DR100%26rft.atitle%253DCellProfiler%253A%2Bimage%2Banalysis%2Bsoftware%2Bfor%2Bidentifying%2Band%2Bquantifying%2Bcell%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-10-r100%26rft_id%253Dinfo%253Apmid%252F17076895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-10-r100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17076895&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.10" data-doi="10.1101/cshperspect.a004374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Retzlaff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frydman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cellular strategies of protein quality control.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">a004374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a004374%26rft_id%253Dinfo%253Apmid%252F21746797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiIzLzgvYTAwNDM3NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.11" data-doi="10.1073/pnas.0812819106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masuda</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenkins</span>  <span class="cit-name-given-names">B. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelms</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A mouse forward genetics screen identifies LISTERIN as an E3 ubiquitin ligase involved in neurodegeneration.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">2097</span>–<span class="cit-lpage">2103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0812819106%26rft_id%253Dinfo%253Apmid%252F19196968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzcvMjA5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.12" data-doi="10.1038/nature05170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochstrasser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Spatially regulated ubiquitin ligation by an ER/nuclear membrane ligase.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">443</span>(<span class="cit-issue">7113</span>): <span class="cit-fpage">827</span>–<span class="cit-lpage">831</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDeng%26rft.auinit1%253DM.%26rft.volume%253D443%26rft.issue%253D7113%26rft.spage%253D827%26rft.epage%253D831%26rft.atitle%253DSpatially%2Bregulated%2Bubiquitin%2Bligation%2Bby%2Ban%2BER%252Fnuclear%2Bmembrane%2Bligase.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05170%26rft_id%253Dinfo%253Apmid%252F17051211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17051211&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241362700044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.13" data-doi="10.1146/annurev.biochem.78.101807.093809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deshaies</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joazeiro</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">RING domain E3 ubiquitin ligases.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DDeshaies%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D78%26rft.spage%253D399%26rft.epage%253D434%26rft.atitle%253DRING%2Bdomain%2BE3%2Bubiquitin%2Bligases.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.78.101807.093809%26rft_id%253Dinfo%253Apmid%252F19489725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.78.101807.093809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19489725&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268069200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.14" data-doi="10.1016/j.molcel.2013.03.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duttler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pechmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frydman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Principles of cotranslational ubiquitination and quality control at the ribosome.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">50</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">379</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D50%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.03.010%26rft_id%253Dinfo%253Apmid%252F23583075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.03.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23583075&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.15" data-doi="10.1016/j.febslet.2008.11.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisele</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">D. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Degradation of misfolded protein in the cytoplasm is mediated by the ubiquitin ligase Ubr1.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">582</span>(<span class="cit-issue">30</span>): <span class="cit-fpage">4143</span>–<span class="cit-lpage">4146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DEisele%26rft.auinit1%253DF.%26rft.volume%253D582%26rft.issue%253D30%26rft.spage%253D4143%26rft.epage%253D4146%26rft.atitle%253DDegradation%2Bof%2Bmisfolded%2Bprotein%2Bin%2Bthe%2Bcytoplasm%2Bis%2Bmediated%2Bby%2Bthe%2Bubiquitin%2Bligase%2BUbr1.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2008.11.015%26rft_id%253Dinfo%253Apmid%252F19041308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2008.11.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19041308&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261949100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erales</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffino</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ubiquitin-independent proteasomal degradation.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> 1843(<span class="cit-issue">1</span>): <span class="cit-fpage">216</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.17" data-doi="10.1038/ncb2343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Measday</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayor</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hul5 HECT ubiquitin ligase plays a major role in the ubiquitylation and turnover of cytosolic misfolded proteins.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1344</span>–<span class="cit-lpage">1352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DFang%26rft.auinit1%253DN.%2BN.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D1344%26rft.epage%253D1352%26rft.atitle%253DHul5%2BHECT%2Bubiquitin%2Bligase%2Bplays%2Ba%2Bmajor%2Brole%2Bin%2Bthe%2Bubiquitylation%2Band%2Bturnover%2Bof%2Bcytosolic%2Bmisfolded%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2343%26rft_id%253Dinfo%253Apmid%252F21983566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb2343&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21983566&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296737900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.18" data-doi="10.1038/ncb3054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comyn</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Persaud</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Rsp5/Nedd4 is the main ubiquitin ligase that targets cytosolic misfolded proteins following heat stress.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1227</span>–<span class="cit-lpage">1237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCell%2BBiol.%26rft.volume%253D16%26rft.spage%253D1227%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb3054%26rft_id%253Dinfo%253Apmid%252F25344756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb3054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25344756&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.19" data-doi="10.1146/annurev.biochem.78.081507.101607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finley</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recognition and processing of ubiquitin-protein conjugates by the proteasome.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DFinley%26rft.auinit1%253DD.%26rft.volume%253D78%26rft.spage%253D477%26rft.epage%253D513%26rft.atitle%253DRecognition%2Band%2Bprocessing%2Bof%2Bubiquitin-protein%2Bconjugates%2Bby%2Bthe%2Bproteasome.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.78.081507.101607%26rft_id%253Dinfo%253Apmid%252F19489727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.78.081507.101607&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19489727&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268069200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.20" data-doi="10.1534/genetics.112.140467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finley</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulrich</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaiser</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The ubiquitin-proteasome system of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">319</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFinley%26rft.auinit1%253DD.%26rft.volume%253D192%26rft.issue%253D2%26rft.spage%253D319%26rft.epage%253D360%26rft.atitle%253DThe%2BUbiquitin-Proteasome%2BSystem%2Bof%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140467%26rft_id%253Dinfo%253Apmid%252F23028185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvMzE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.21" data-doi="10.1126/science.1255638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foresti</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Vaello</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Funaya</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carvalho</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quality control of inner nuclear membrane proteins by the Asi complex.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>(<span class="cit-issue">6210</span>): <span class="cit-fpage">751</span>–<span class="cit-lpage">755</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1255638%26rft_id%253Dinfo%253Apmid%252F25236469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDYvNjIxMC83NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.22" data-doi="10.1016/j.semcdb.2011.12.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fredrickson</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Selective destruction of abnormal proteins by ubiquitin-mediated protein quality control degradation.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">23</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">530</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.stitle%253DSemin%2BCell%2BDev%2BBiol%26rft.aulast%253DFredrickson%26rft.auinit1%253DE.%2BK.%26rft.volume%253D23%26rft.issue%253D5%26rft.spage%253D530%26rft.epage%253D537%26rft.atitle%253DSelective%2Bdestruction%2Bof%2Babnormal%2Bproteins%2Bby%2Bubiquitin-mediated%2Bprotein%2Bquality%2Bcontrol%2Bdegradation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2011.12.006%26rft_id%253Dinfo%253Apmid%252F22245831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2011.12.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22245831&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.23" data-doi="10.1091/mbc.E11-03-0256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fredrickson</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenbaum</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Locke</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milac</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exposed hydrophobicity is a key determinant of nuclear quality control degradation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">2384</span>–<span class="cit-lpage">2395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E11-03-0256%26rft_id%253Dinfo%253Apmid%252F21551067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMy8yMzg0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.24" data-doi="10.1074/jbc.M112.406710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fredrickson</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallagher</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clowes Candadai</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Substrate recognition in nuclear protein quality control degradation is governed by exposed hydrophobicity that correlates with aggregation and insolubility.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">288</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">6130</span>–<span class="cit-lpage">6139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M112.406710%26rft_id%253Dinfo%253Apmid%252F23335508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODgvOS82MTMwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.25" data-doi="10.1091/mbc.E11-05-0463"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Furth</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gertman</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiber</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alfassy</span>  <span class="cit-name-given-names">O. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exposure of bipartite hydrophobic signal triggers nuclear quality control of Ndc10 at the endoplasmic reticulum/nuclear envelope.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">22</span>(<span class="cit-issue">24</span>): <span class="cit-fpage">4726</span>–<span class="cit-lpage">4739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E11-05-0463%26rft_id%253Dinfo%253Apmid%252F21998200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8yNC80NzI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.26" data-doi="10.1016/j.cell.2005.01.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottschling</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Degradation-mediated protein quality control in the nucleus.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">120</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">803</span>–<span class="cit-lpage">815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGardner%26rft.auinit1%253DR.%2BG.%26rft.volume%253D120%26rft.issue%253D6%26rft.spage%253D803%26rft.epage%253D815%26rft.atitle%253DDegradation-mediated%2Bprotein%2Bquality%2Bcontrol%2Bin%2Bthe%2Bnucleus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.01.016%26rft_id%253Dinfo%253Apmid%252F15797381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.01.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15797381&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228067500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.27" data-doi="10.1091/mbc.E06-10-0965"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghaboosi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshaies</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A conditional yeast E1 mutant blocks the ubiquitin-proteasome pathway and reveals a role for ubiquitin conjugates in targeting Rad23 to the proteasome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1953</span>–<span class="cit-lpage">1963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E06-10-0965%26rft_id%253Dinfo%253Apmid%252F17360968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE4LzUvMTk1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.28" data-doi="10.1093/emboj/17.10.2759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chomsky</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulka</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Degradation signals for ubiquitin system proteolysis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">17</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">2759</span>–<span class="cit-lpage">2766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DGilon%26rft.auinit1%253DT.%26rft.volume%253D17%26rft.issue%253D10%26rft.spage%253D2759%26rft.epage%253D2766%26rft.atitle%253DDegradation%2Bsignals%2Bfor%2Bubiquitin%2Bsystem%2Bproteolysis%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.10.2759%26rft_id%253Dinfo%253Apmid%252F9582269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTcvMTAvMjc1OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.29" data-doi="10.1128/MCB.20.19.7214-7219.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chomsky</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulka</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Degradation signals recognized by the Ubc6p-Ubc7p ubiquitin-conjugating enzyme pair.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">19</span>): <span class="cit-fpage">7214</span>–<span class="cit-lpage">7219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGilon%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D19%26rft.spage%253D7214%26rft.epage%253D7219%26rft.atitle%253DDegradation%2BSignals%2BRecognized%2Bby%2Bthe%2BUbc6p-Ubc7p%2BUbiquitin-Conjugating%2BEnzyme%2BPair%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.20.19.7214-7219.2000%26rft_id%253Dinfo%253Apmid%252F10982838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xOS83MjE0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.30" data-doi="10.1074/jbc.M113.475905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guerriero</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiberth</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodsky</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Hsp70 targets a cytoplasmic quality control substrate to the San1p ubiquitin ligase.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">288</span>(<span class="cit-issue">25</span>): <span class="cit-fpage">18506</span>–<span class="cit-lpage">18520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M113.475905%26rft_id%253Dinfo%253Apmid%252F23653356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODgvMjUvMTg1MDYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.31" data-doi="10.1083/jcb.201408088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habeck</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebner</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimada-Kreft</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreft</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The yeast ERAD-C ubiquitin ligase Doa10 recognizes an intramembrane degron.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">209</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">261</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201408088%26rft_id%253Dinfo%253Apmid%252F25918226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIwOS8yLzI2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.32" data-doi="10.1073/pnas.0910591107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heck</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampton</span>  <span class="cit-name-given-names">R. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cytoplasmic protein quality control degradation mediated by parallel actions of the E3 ubiquitin ligases Ubr1 and San1.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1106</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0910591107%26rft_id%253Dinfo%253Apmid%252F20080635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA3LzMvMTEwNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.33" data-doi="10.1002/sim.4780090710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">More powerful procedures for multiple significance testing.</span> <abbr class="cit-jnl-abbrev">Stat. Med.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">811</span>–<span class="cit-lpage">818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStatistics%2Bin%2Bmedicine%26rft.stitle%253DStat%2BMed%26rft.aulast%253DHochberg%26rft.auinit1%253DY.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D811%26rft.epage%253D818%26rft.atitle%253DMore%2Bpowerful%2Bprocedures%2Bfor%2Bmultiple%2Bsignificance%2Btesting.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fsim.4780090710%26rft_id%253Dinfo%253Apmid%252F2218183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/sim.4780090710&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2218183&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DN97200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hol</span>  <span class="cit-name-given-names">E.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheper</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> Protein quality control in neurodegeneration: walking the tight rope between health and disease. J. Mol. Neurosci. 34(1):23–33.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.35" data-doi="10.1074/jbc.M402468200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huyer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piluek</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fansler</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreft</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochstrasser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Distinct machinery is required in <em>Saccharomyces cerevisiae</em> for the endoplasmic reticulum-associated degradation of a multispanning membrane protein and a soluble luminal protein.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">279</span>(<span class="cit-issue">37</span>): <span class="cit-fpage">38369</span>–<span class="cit-lpage">38378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M402468200%26rft_id%253Dinfo%253Apmid%252F15252059%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzkvMzcvMzgzNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.36" data-doi="10.1016/j.cell.2006.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ismail</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Have you HRD? Understanding ERAD is DOAble!</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">126</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">237</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DIsmail%26rft.auinit1%253DN.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D237%26rft.epage%253D239%26rft.atitle%253DHave%2Byou%2BHRD%253F%2BUnderstanding%2BERAD%2Bis%2BDOAble%2521%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.07.001%26rft_id%253Dinfo%253Apmid%252F16873052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16873052&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239552600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.37" data-doi="10.1016/S0092-8674(00)81421-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rakhilina</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochstrasser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Degradation signal masking by heterodimerization of MATalpha2 and MATa1 blocks their mutual destruction by the ubiquitin-proteasome pathway.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">94</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">217</span>–<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DJohnson%26rft.auinit1%253DP.%2BR.%26rft.volume%253D94%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D227%26rft.atitle%253DDegradation%2Bsignal%2Bmasking%2Bby%2Bheterodimerization%2Bof%2BMATalpha2%2Band%2BMATa1%2Bblocks%2Btheir%2Bmutual%2Bdestruction%2Bby%2Bthe%2Bubiquitin-proteasome%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981421-X%26rft_id%253Dinfo%253Apmid%252F9695950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81421-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9695950&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075020100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.38" data-doi="10.1006/jmbi.1999.3091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Protein secondary structure prediction based on position-specific scoring matrices.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">292</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">195</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DJones%26rft.auinit1%253DD.%2BT.%26rft.volume%253D292%26rft.issue%253D2%26rft.spage%253D195%26rft.epage%253D202%26rft.atitle%253DProtein%2Bsecondary%2Bstructure%2Bprediction%2Bbased%2Bon%2Bposition-specific%2Bscoring%2Bmatrices.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1999.3091%26rft_id%253Dinfo%253Apmid%252F10493868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1999.3091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10493868&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082597400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.39" data-doi="10.1038/nature14096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khmelinskii</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaszczak</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pantazopoulou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omnus</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Protein quality control at the inner nuclear membrane.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">516</span>(<span class="cit-issue">7531</span>): <span class="cit-fpage">410</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D516%26rft.spage%253D410%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14096%26rft_id%253Dinfo%253Apmid%252F25519137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14096&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25519137&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.40" data-doi="10.1534/g3.111.001933"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khosrow-Khavar</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winget</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comyn</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The yeast ubr1 ubiquitin ligase participates in a prominent pathway that targets cytosolic thermosensitive mutants for degradation.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">619</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DKhosrow-Khavar%26rft.auinit1%253DF.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253D619%26rft.epage%253D628%26rft.atitle%253DThe%2BYeast%2BUbr1%2BUbiquitin%2BLigase%2BParticipates%2Bin%2Ba%2BProminent%2BPathway%2BThat%2BTargets%2BCytosolic%2BThermosensitive%2BMutants%2Bfor%2BDegradation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001933%26rft_id%253Dinfo%253Apmid%252F22670231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvNS82MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.41" data-doi="10.1091/mbc.E13-01-0038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kraut-Cohen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Afanasieva</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haim-Vilmovsky</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slobodin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yosef</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Translation- and SRP-independent mRNA targeting to the endoplasmic reticulum in the yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">19</span>): <span class="cit-fpage">3069</span>–<span class="cit-lpage">3084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E13-01-0038%26rft_id%253Dinfo%253Apmid%252F23904265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xOS8zMDY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.42" data-doi="10.1074/jbc.M512215200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kreft</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochstrasser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Membrane topology of the yeast endoplasmic reticulum-localized ubiquitin ligase Doa10 and comparison with its human ortholog TEB4 (MARCH-VI).</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">281</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">4646</span>–<span class="cit-lpage">4653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M512215200%26rft_id%253Dinfo%253Apmid%252F16373356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODEvOC80NjQ2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.43" data-doi="10.2144/000112257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamprecht</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabatini</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CellProfiler: free, versatile software for automated biological image analysis.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">71</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DLamprecht%26rft.auinit1%253DM.%2BR.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D75%26rft.atitle%253DCellProfiler%253A%2Bfree%252C%2Bversatile%2Bsoftware%2Bfor%2Bautomated%2Bbiological%2Bimage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000112257%26rft_id%253Dinfo%253Apmid%252F17269487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000112257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17269487&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248948900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.44" data-doi="10.1371/journal.pone.0005038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pelham</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inefficient quality control of thermosensitive proteins on the plasma membrane.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e5038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLewis%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De5038%26rft.epage%253De5038%26rft.atitle%253DInefficient%2Bquality%2Bcontrol%2Bof%2Bthermosensitive%2Bproteins%2Bon%2Bthe%2Bplasma%2Bmembrane.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005038%26rft_id%253Dinfo%253Apmid%252F19337370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19337370&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.45" data-doi="10.2144/000112389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apodaca</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Proteasome inhibition in wild-type yeast <em>Saccharomyces cerevisiae</em> cells.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">42</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">158</span><span class="cit-lpage">, 160, 162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DLiu%26rft.auinit1%253DC.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D158%26rft.epage%253D%2B162%26rft.atitle%253DProteasome%2Binhibition%2Bin%2Bwild-type%2Byeast%2BSaccharomyces%2Bcerevisiae%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000112389%26rft_id%253Dinfo%253Apmid%252F17373478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000112389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17373478&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.46" data-doi="10.1073/pnas.1210041110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyumkis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doamekpor</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bengtson</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Single-particle EM reveals extensive conformational variability of the Ltn1 E3 ligase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1702</span>–<span class="cit-lpage">1707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1210041110%26rft_id%253Dinfo%253Apmid%252F23319619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEwLzUvMTcwMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.47" data-doi="10.1073/pnas.1413882111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyumkis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira dos Passos</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tahara</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webb</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Structural basis for translational surveillance by the large ribosomal subunit-associated protein quality control complex.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>(<span class="cit-issue">45</span>): <span class="cit-fpage">15981</span>–<span class="cit-lpage">15986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1413882111%26rft_id%253Dinfo%253Apmid%252F25349383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzQ1LzE1OTgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.48" data-doi="10.1016/j.cell.2005.03.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClellan</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frydman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">a</span> <span class="cit-article-title">Folding and quality control of the VHL tumor suppressor proceed through distinct chaperone pathways.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">121</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">739</span>–<span class="cit-lpage">748</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMcClellan%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D121%26rft.issue%253D5%26rft.spage%253D739%26rft.epage%253D748%26rft.atitle%253DFolding%2Band%2Bquality%2Bcontrol%2Bof%2Bthe%2BVHL%2Btumor%2Bsuppressor%2Bproceed%2Bthrough%2Bdistinct%2Bchaperone%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.03.024%26rft_id%253Dinfo%253Apmid%252F15935760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.03.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15935760&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229658000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.49" data-doi="10.1038/ncb0805-736"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClellan</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaganovich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frydman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">b</span> <span class="cit-article-title">Protein quality control: chaperones culling corrupt conformations.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">7</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">736</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DMcClellan%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D736%26rft.epage%253D741%26rft.atitle%253DProtein%2Bquality%2Bcontrol%253A%2Bchaperones%2Bculling%2Bcorrupt%2Bconformations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb0805-736%26rft_id%253Dinfo%253Apmid%252F16056264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb0805-736&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16056264&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230881500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-6.7.1853.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzger</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>Protein quality control in the endoplasmic reticulum and cytosol</em></span>, <span class="cit-publ-name">Johns Hopkins University</span>, <span class="cit-publ-loc">Baltimore</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.51" data-doi="10.1074/jbc.M806424200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzger</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dancy</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Degradation of a cytosolic protein requires endoplasmic reticulum-associated degradation machinery.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">283</span>(<span class="cit-issue">47</span>): <span class="cit-fpage">32302</span>–<span class="cit-lpage">32316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M806424200%26rft_id%253Dinfo%253Apmid%252F18812321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODMvNDcvMzIzMDIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.52" data-doi="10.1128/MCB.8.3.1309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herskowitz</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The a-factor pheromone of <em>Saccharomyces cerevisiae</em> is essential for mating.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1309</span>–<span class="cit-lpage">1318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DMichaelis%26rft.auinit1%253DS%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D1309%26rft.epage%253D1318%26rft.atitle%253DThe%2Ba-factor%2Bpheromone%2Bof%2BSaccharomyces%2Bcerevisiae%2Bis%2Bessential%2Bfor%2Bmating.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.8.3.1309%26rft_id%253Dinfo%253Apmid%252F3285180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjgvMy8xMzA5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.53" data-doi="10.1016/j.cell.2007.11.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakatsukasa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huyer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodsky</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dissecting the ER-associated degradation of a misfolded polytopic membrane protein.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">132</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">101</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DNakatsukasa%26rft.auinit1%253DK.%26rft.volume%253D132%26rft.issue%253D1%26rft.spage%253D101%26rft.epage%253D112%26rft.atitle%253DDissecting%2Bthe%2BER-associated%2Bdegradation%2Bof%2Ba%2Bmisfolded%2Bpolytopic%2Bmembrane%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.11.023%26rft_id%253Dinfo%253Apmid%252F18191224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.11.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18191224&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253427600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.54" data-doi="10.1091/mbc.E10-02-0098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nillegoda</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theodoraki</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandal</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayo</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Ubr1 and Ubr2 function in a quality control pathway for degradation of unfolded cytosolic proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">2102</span>–<span class="cit-lpage">2116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E10-02-0098%26rft_id%253Dinfo%253Apmid%252F20462952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMy8yMTAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.55" data-doi="10.1093/jb/mvi068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishikawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodsky</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakatsukasa</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Roles of molecular chaperones in endoplasmic reticulum (ER) quality control and ER-associated degradation (ERAD).</span> <abbr class="cit-jnl-abbrev">J. Biochem.</abbr> <span class="cit-vol">137</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">551</span>–<span class="cit-lpage">555</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiochemistry%26rft.stitle%253DJ%2BBiochem%26rft.aulast%253DNishikawa%26rft.auinit1%253DS.-i.%26rft.volume%253D137%26rft.issue%253D5%26rft.spage%253D551%26rft.epage%253D555%26rft.atitle%253DRoles%2Bof%2BMolecular%2BChaperones%2Bin%2BEndoplasmic%2BReticulum%2B%2528ER%2529%2BQuality%2BControl%2Band%2BER-Associated%2BDegradation%2B%2528ERAD%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjb%252Fmvi068%26rft_id%253Dinfo%253Apmid%252F15944407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamJpb2NoZW0iO3M6NToicmVzaWQiO3M6OToiMTM3LzUvNTUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.56" data-doi="10.1016/j.cell.2013.06.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kukushkin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konagai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">PolyQ proteins interfere with nuclear degradation of cytosolic proteins by sequestering the Sis1p chaperone.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">154</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">134</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D134%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.06.003%26rft_id%253Dinfo%253Apmid%252F23791384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.06.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23791384&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321327900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.57" data-doi="10.1016/S0092-8674(03)01074-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pickart</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Back to the future with ubiquitin.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">116</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">181</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPickart%26rft.auinit1%253DC.%2BM.%26rft.volume%253D116%26rft.issue%253D2%26rft.spage%253D181%26rft.epage%253D190%26rft.atitle%253DBack%2Bto%2Bthe%2Bfuture%2Bwith%2Bubiquitin.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252803%252901074-2%26rft_id%253Dinfo%253Apmid%252F14744430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(03)01074-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14744430&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188541700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.58" data-doi="10.1091/mbc.E10-02-0111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawaguchi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A nucleus-based quality control mechanism for cytosolic proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">21</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">2117</span>–<span class="cit-lpage">2127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E10-02-0111%26rft_id%253Dinfo%253Apmid%252F20462951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMy8yMTE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.59" data-doi="10.1016/j.bbrc.2012.07.080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prasad</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawaguchi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Biosynthetic mode can determine the mechanism of protein quality control.</span> <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">425</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">689</span>–<span class="cit-lpage">695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DPrasad%26rft.auinit1%253DR.%26rft.volume%253D425%26rft.issue%253D3%26rft.spage%253D689%26rft.epage%253D695%26rft.atitle%253DBiosynthetic%2Bmode%2Bcan%2Bdetermine%2Bthe%2Bmechanism%2Bof%2Bprotein%2Bquality%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2012.07.080%26rft_id%253Dinfo%253Apmid%252F22842567%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2012.07.080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22842567&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.60" data-doi="10.1038/nrm2468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ravid</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochstrasser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Diversity of degradation signals in the ubiquitin-proteasome system.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">679</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DRavid%26rft.auinit1%253DT.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D679%26rft.epage%253D690%26rft.atitle%253DDiversity%2Bof%2Bdegradation%2Bsignals%2Bin%2Bthe%2B%2Bubiquitin-proteasome%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2468%26rft_id%253Dinfo%253Apmid%252F18698327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18698327&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258636900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.61" data-doi="10.1038/sj.emboj.7600946"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ravid</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreft</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochstrasser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Membrane and soluble substrates of the Doa10 ubiquitin ligase are degraded by distinct pathways.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">533</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DRavid%26rft.auinit1%253DT.%26rft.volume%253D25%26rft.issue%253D3%26rft.spage%253D533%26rft.epage%253D543%26rft.atitle%253DMembrane%2Band%2Bsoluble%2Bsubstrates%2Bof%2Bthe%2BDoa10%2Bubiquitin%2Bligase%2Bare%2Bdegraded%2Bby%2Bdistinct%2Bpathways%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600946%26rft_id%253Dinfo%253Apmid%252F16437165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIyNS8zLzUzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.62" data-doi="10.1038/nrm3958"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicchitta</span>  <span class="cit-name-given-names">C. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Diversity and selectivity in mRNA translation on the endoplasmic reticulum.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">221</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMol.%2BCell%2BBiol.%26rft.volume%253D16%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm3958%26rft_id%253Dinfo%253Apmid%252F25735911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm3958&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25735911&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.63" data-doi="10.4161/nucl.2.4.16118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenbaum</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">How a disordered ubiquitin ligase maintains order in nuclear protein homeostasis.</span> <abbr class="cit-jnl-abbrev">Nucleus</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">264</span>–<span class="cit-lpage">270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNucleus%26rft.aulast%253DRosenbaum%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D264%26rft.epage%253D270%26rft.atitle%253DHow%2Ba%2Bdisordered%2Bubiquitin%2Bligase%2Bmaintains%2Border%2Bin%2Bnuclear%2Bprotein%2Bhomeostasis.%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fnucl.2.4.16118%26rft_id%253Dinfo%253Apmid%252F21941105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/nucl.2.4.16118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21941105&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.64" data-doi="10.1016/j.molcel.2010.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenbaum</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fredrickson</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oeser</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrett-Engele</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Locke</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Disorder targets misorder in nuclear quality control degradation: a disordered ubiquitin ligase directly recognizes its misfolded substrates.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">41</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">93</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DRosenbaum%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D41%26rft.issue%253D1%26rft.spage%253D93%26rft.epage%253D106%26rft.atitle%253DDisorder%2Btargets%2Bmisorder%2Bin%2Bnuclear%2Bquality%2Bcontrol%2Bdegradation%253A%2Ba%2Bdisordered%2Bubiquitin%2Bligase%2Bdirectly%2Brecognizes%2Bits%2Bmisfolded%2Bsubstrates.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.12.004%26rft_id%253Dinfo%253Apmid%252F21211726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21211726&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286692400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.65" data-doi="10.1128/MCB.15.8.4086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sadis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atienza</span>  <span class="cit-name-given-names">C.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finley</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Synthetic signals for ubiquitin-dependent proteolysis.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">4086</span>–<span class="cit-lpage">4094</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSadis%26rft.auinit1%253DS.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D4086%26rft.epage%253D4094%26rft.atitle%253DSynthetic%2Bsignals%2Bfor%2Bubiquitin-dependent%2Bproteolysis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.15.8.4086%26rft_id%253Dinfo%253Apmid%252F7623804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE1LzgvNDA4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.66" data-doi="10.1016/j.molcel.2014.12.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santhanam</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegde</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Structure and assembly pathway of the ribosome quality control complex.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">57</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">433</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D57%26rft.spage%253D433%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.12.015%26rft_id%253Dinfo%253Apmid%252F25578875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.12.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25578875&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.67" data-doi="10.1126/science.1259724"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsawar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Protein synthesis. Rqc2p and 60S ribosomal subunits mediate mRNA-independent elongation of nascent chains.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">347</span>(<span class="cit-issue">6217</span>): <span class="cit-fpage">75</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DShen%26rft.auinit1%253DP.%2BS.%26rft.volume%253D347%26rft.issue%253D6217%26rft.spage%253D75%26rft.epage%253D78%26rft.atitle%253DRqc2p%2Band%2B60S%2Bribosomal%2Bsubunits%2Bmediate%2BmRNA-independent%2Belongation%2Bof%2Bnascent%2Bchains%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1259724%26rft_id%253Dinfo%253Apmid%252F25554787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNDcvNjIxNy83NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.68" data-doi="10.1091/mbc.E13-01-0010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiber</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breuer</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandeis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ravid</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Ubiquitin conjugation triggers misfolded protein sequestration into quality control foci when Hsp70 chaperone levels are limiting.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">24</span>(<span class="cit-issue">13</span>): <span class="cit-fpage">2076</span>–<span class="cit-lpage">2087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E13-01-0010%26rft_id%253Dinfo%253Apmid%252F23637465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMy8yMDc2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A novel degradation signal derived from distal C-terminal frame-shift mutations of KCNQ2 which cause neonatal epilepsy.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> 286(50): 42949–42958.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.70" data-doi="10.1371/journal.pone.0052099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Summers</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cyr</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Type II Hsp40 Sis1 cooperates with Hsp70 and the E3 ligase Ubr1 to promote degradation of terminally misfolded cytosolic protein.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">e52099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De52099%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0052099%26rft_id%253Dinfo%253Apmid%252F23341891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0052099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23341891&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.71" data-doi="10.1101/gad.933301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Locher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochstrasser</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A conserved ubiquitin ligase of the nuclear envelope/endoplasmic reticulum that functions in both ER-associated and Matalpha2 repressor degradation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">15</span>(<span class="cit-issue">20</span>): <span class="cit-fpage">2660</span>–<span class="cit-lpage">2674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSwanson%26rft.auinit1%253DR.%26rft.volume%253D15%26rft.issue%253D20%26rft.spage%253D2660%26rft.epage%253D2674%26rft.atitle%253DA%2Bconserved%2Bubiquitin%2Bligase%2Bof%2Bthe%2Bnuclear%2Benvelope%252Fendoplasmic%2Breticulum%2Bthat%2Bfunctions%2Bin%2Bboth%2BER-associated%2Band%2BMatalpha%2B2%2Brepressor%2Bdegradation%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.933301%26rft_id%253Dinfo%253Apmid%252F11641273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE1LzIwLzI2NjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.72" data-doi="10.1016/0092-8674(91)90202-A"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshavsky</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Naming a targeting signal.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">64</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">13</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DVarshavsky%26rft.auinit1%253DA.%26rft.volume%253D64%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D15%26rft.atitle%253DNaming%2Ba%2Btargeting%2Bsignal.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252891%252990202-A%26rft_id%253Dinfo%253Apmid%252F1986863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(91)90202-A&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1986863&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991ET17500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.73" data-doi="10.1046/j.1365-2443.1997.1020301.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshavsky</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The N-end rule pathway of protein degradation.</span> <abbr class="cit-jnl-abbrev">Genes Cells</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">13</span>–<span class="cit-lpage">28</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DVarshavsky%26rft.auinit1%253DA%26rft.volume%253D2%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D28%26rft.atitle%253DThe%2BN-end%2Brule%2Bpathway%2Bof%2Bprotein%2Bdegradation%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2443.1997.1020301.x%26rft_id%253Dinfo%253Apmid%252F9112437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2443.1997.1020301.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9112437&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WR94800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.74" data-doi="10.1515/BC.2000.101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varshavsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Felix Hoppe-Seyler Lecture 2000. The ubiquitin system and the N-end rule pathway.</span> <abbr class="cit-jnl-abbrev">Biol. Chem.</abbr> <span class="cit-vol">381</span>(<span class="cit-issue">9–10</span>): <span class="cit-fpage">779</span>–<span class="cit-lpage">789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiological%2Bchemistry%26rft.stitle%253DBiol%2BChem%26rft.aulast%253DVarshavsky%26rft.auinit1%253DA.%26rft.volume%253D381%26rft.issue%253D9-10%26rft.spage%253D779%26rft.epage%253D789%26rft.atitle%253DFelix%2BHoppe-Seyler%2BLecture%2B2000.%2BThe%2Bubiquitin%2Bsystem%2Band%2Bthe%2BN-end%2Brule%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1515%252FBC.2000.101%26rft_id%253Dinfo%253Apmid%252F11076011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1515/BC.2000.101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11076011&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000089944100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.75" data-doi="10.1083/jcb.200309132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vashist</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">D. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Misfolded proteins are sorted by a sequential checkpoint mechanism of ER quality control.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">165</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">41</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200309132%26rft_id%253Dinfo%253Apmid%252F15078901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6IjE2NS8xLzQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.76" data-doi="10.1038/nrm2546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vembar</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodsky</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">One step at a time: endoplasmic reticulum-associated degradation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">944</span>–<span class="cit-lpage">957</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DVembar%26rft.auinit1%253DS.%2BS.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D944%26rft.epage%253D957%26rft.atitle%253DOne%2Bstep%2Bat%2Ba%2Btime%253A%2Bendoplasmic%2Breticulum-associated%2Bdegradation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2546%26rft_id%253Dinfo%253Apmid%252F19002207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19002207&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1853.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261126800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-book" id="cit-6.7.1853.77"><div class="cit-metadata unstructured">Vokes, M. S., and A. E. Carpenter, 2008 Using CellProfiler for automatic identification and measurement of biological objects in images. Curr. Protoc. Mol. Biol. 109: 14.17.1–14.17.13.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1853.78" data-doi="10.1534/genetics.107.073205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meaux</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Hoof</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A genomic screen in yeast reveals novel aspects of nonstop mRNA metabolism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">773</span>–<span class="cit-lpage">784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.073205%26rft_id%253Dinfo%253Apmid%252F17660569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzIvNzczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/7/1853.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1841" title="The Drosophila melanogaster Muc68E Mucin Gene Influences Adult Size, Starvation Tolerance, and Cold Recovery" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1867" title="Copy Number Profiling of Brazilian Astrocytomas" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/7"><span>Volume 6 Issue 7, July 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/7.cover-source.jpg?itok=E84OOyGI" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1853.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_acXr5qz9oae14a2Fbti8UU2AZ3kXJ6RifJNqR8KOtk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/7/1853" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/7/1853" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/7/1853" class="active" data-icon-position="" data-hide-link-title="0">Degradation Signals for Ubiquitin-Proteasome Dependent Cytosolic Protein Quality Control (CytoQC) in Yeast</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460402" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-bqAp97qcRWfpITwo61HCgfhaRDo90UfiYRD730GFslM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1853.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/7/1853" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-sR0-o7z1jz3Ca1aq2aQvHRy09ryHe8S3EgaTvUDllzI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/7/1853" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1853.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460402" id="node460402--2" data-pisa="ggg;6/7/1853" data-pisa-master="ggg;g3.116.027953" data-apath="/ggg/6/7/1853.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Degradation Signals for Ubiquitin-Proteasome Dependent Cytosolic Protein Quality Control (CytoQC) in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew J. Maurer</span>, <span class="highwire-citation-author" data-delta="1">Eric D. Spear</span>, <span class="highwire-citation-author" data-delta="2">Allen T. Yu</span>, <span class="highwire-citation-author" data-delta="3">Evan J. Lee</span>, <span class="highwire-citation-author" data-delta="4">Saba Shahzad</span> and <span class="highwire-citation-author" data-delta="5">Susan Michaelis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1853-1866;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027953 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460402--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew J. Maurer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BJ.%2BMaurer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maurer%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BJ.%2BMaurer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eric D. Spear </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BD.%2BSpear%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spear%20ED&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BD.%2BSpear%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Allen T. Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BT.%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BT.%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Evan J. Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BJ.%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BJ.%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Saba Shahzad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saba%2BShahzad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shahzad%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaba%2BShahzad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Susan Michaelis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BMichaelis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michaelis%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BMichaelis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:michaelis@jhmi.edu" class="" data-icon-position="" data-hide-link-title="0">michaelis@jhmi.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/7/1853" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/7/1853&amp;title=Degradation%20Signals%20for%20Ubiquitin-Proteasome%20Dependent%20Cytosolic%20Protein%20Quality%20Control%20%28CytoQC%29%20in%20Yeast" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/7/1853&amp;title=Degradation%20Signals%20for%20Ubiquitin-Proteasome%20Dependent%20Cytosolic%20Protein%20Quality%20Control%20%28CytoQC%29%20in%20Yeast" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/7/1853&amp;title=Degradation%20Signals%20for%20Ubiquitin-Proteasome%20Dependent%20Cytosolic%20Protein%20Quality%20Control%20%28CytoQC%29%20in%20Yeast" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/7/1853&amp;text=Degradation%20Signals%20for%20Ubiquitin-Proteasome%20Dependent%20Cytosolic%20Protein%20Quality%20Control%20%28CytoQC%29%20in%20Yeast" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/7/1853&amp;title=Degradation%20Signals%20for%20Ubiquitin-Proteasome%20Dependent%20Cytosolic%20Protein%20Quality%20Control%20%28CytoQC%29%20in%20Yeast" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/7/1853&amp;t=Degradation%20Signals%20for%20Ubiquitin-Proteasome%20Dependent%20Cytosolic%20Protein%20Quality%20Control%20%28CytoQC%29%20in%20Yeast" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/7/1853&amp;title=Degradation%20Signals%20for%20Ubiquitin-Proteasome%20Dependent%20Cytosolic%20Protein%20Quality%20Control%20%28CytoQC%29%20in%20Yeast" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/7/1853&amp;title=Degradation%20Signals%20for%20Ubiquitin-Proteasome%20Dependent%20Cytosolic%20Protein%20Quality%20Control%20%28CytoQC%29%20in%20Yeast" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460402/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460402" id="node460402--3" data-pisa="ggg;6/7/1853" data-pisa-master="ggg;g3.116.027953" data-apath="/ggg/6/7/1853.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Degradation Signals for Ubiquitin-Proteasome Dependent Cytosolic Protein Quality Control (CytoQC) in Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Matthew J. Maurer</span>, <span class="highwire-citation-author" data-delta="1">Eric D. Spear</span>, <span class="highwire-citation-author" data-delta="2">Allen T. Yu</span>, <span class="highwire-citation-author" data-delta="3">Evan J. Lee</span>, <span class="highwire-citation-author" data-delta="4">Saba Shahzad</span> and <span class="highwire-citation-author" data-delta="5">Susan Michaelis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1853-1866;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.027953 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460402--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Matthew J. Maurer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BJ.%2BMaurer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maurer%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BJ.%2BMaurer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eric D. Spear </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eric%2BD.%2BSpear%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spear%20ED&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEric%2BD.%2BSpear%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Allen T. Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Allen%2BT.%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAllen%2BT.%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Evan J. Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evan%2BJ.%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvan%2BJ.%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Saba Shahzad </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saba%2BShahzad%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shahzad%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaba%2BShahzad%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Susan Michaelis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Cell Biology, The Johns Hopkins University School of Medicine, Baltimore, Maryland 21205</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susan%2BMichaelis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michaelis%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASusan%2BMichaelis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:michaelis@jhmi.edu" class="" data-icon-position="" data-hide-link-title="0">michaelis@jhmi.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1853&amp;rft.epage=1866&amp;rft.atitle=Degradation%20Signals%20for%20Ubiquitin-Proteasome%20Dependent%20Cytosolic%20Protein%20Quality%20Control%20%28CytoQC%29%20in%20Yeast&amp;rft.volume=6&amp;rft.issue=7&amp;rft.date=2016-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Maurer%2C+Matthew+J.&amp;rft.au=Spear%2C+Eric+D.&amp;rft.au=Yu%2C+Allen+T.&amp;rft.au=Lee%2C+Evan+J.&amp;rft.au=Shahzad%2C+Saba&amp;rft.au=Michaelis%2C+Susan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460402/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460402/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460402/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460402/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460402/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460402/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460402/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460402/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460402/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460402/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460402/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460402/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/7/1853" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/7/1853.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/7/1853.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/7/1853.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/7/1853.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
