<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/7/2051" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/7/2051.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/7/2051.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/7/2051.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/7/2051.atom" />
<meta name="HW.pisa" content="ggg;6/7/2051" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="CRISPR/Cas9-Mediated Insertion of loxP Sites in the Mouse Dock7 Gene Provides an Effective Alternative to Use of Targeted Embryonic Stem Cells" />
<meta name="DC.Identifier" content="10.1534/g3.116.030601" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Bishop et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Targeted gene mutation in the mouse is a primary strategy to understand gene function and relation to phenotype. The Knockout Mouse Project (KOMP) had an initial goal to develop a public resource of mouse embryonic stem (ES) cell clones that carry null mutations in all genes. Indeed, many useful novel mouse models have been generated from publically accessible targeted mouse ES cell lines. However, there are limitations, including incorrect targeting or cassette structure, and difficulties with germline transmission of the allele from chimeric mice. In our experience, using a small sample of targeted ES cell clones, we were successful ∼50% of the time in generating germline transmission of a correctly targeted allele. With the advent of CRISPR/Cas9 as a mouse genome modification tool, we assessed the efficiency of creating a conditional targeted allele in one gene, dedicator of cytokinesis 7 ( Dock7 ), for which we were unsuccessful in generating a null allele using a KOMP targeted ES cell clone. The strategy was to insert loxP sites to flank either exons 3 and 4, or exons 3 through 7. By coinjecting Cas9 mRNA, validated sgRNAs, and oligonucleotide donors into fertilized eggs from C57BL/6J mice, we obtained a variety of alleles, including mice homozygous for the null alleles mediated by nonhomologous end joining, alleles with one of the two desired loxP sites, and correctly targeted alleles with both loxP sites. We also found frequent mutations in the inserted loxP sequence, which is partly attributable to the heterogeneity in the original oligonucleotide preparation." />
<meta name="DC.Contributor" content="Kathleen A. Bishop" />
<meta name="DC.Contributor" content="Anne Harrington" />
<meta name="DC.Contributor" content="Evguenia Kouranova" />
<meta name="DC.Contributor" content="Edward J. Weinstein" />
<meta name="DC.Contributor" content="Clifford J. Rosen" />
<meta name="DC.Contributor" content="Xiaoxia Cui" />
<meta name="DC.Contributor" content="Lucy Liaw" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="CRISPR/Cas9-Mediated Insertion of loxP Sites in the Mouse Dock7 Gene Provides an Effective Alternative to Use of Targeted Embryonic Stem Cells" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Targeted gene mutation in the mouse is a primary strategy to understand gene function and relation to phenotype. The Knockout Mouse Project (KOMP) had an initial goal to develop a public resource of mouse embryonic stem (ES) cell clones that carry null mutations in all genes. Indeed, many useful novel mouse models have been generated from publically accessible targeted mouse ES cell lines. However, there are limitations, including incorrect targeting or cassette structure, and difficulties with germline transmission of the allele from chimeric mice. In our experience, using a small sample of targeted ES cell clones, we were successful ∼50% of the time in generating germline transmission of a correctly targeted allele. With the advent of CRISPR/Cas9 as a mouse genome modification tool, we assessed the efficiency of creating a conditional targeted allele in one gene, &lt;i&gt;dedicator of cytokinesis&amp;nbsp;7&lt;/i&gt; (&lt;i&gt;Dock7&lt;/i&gt;), for which we were unsuccessful in generating a null allele using a KOMP targeted ES cell clone. The strategy was to insert loxP sites to flank either exons&amp;nbsp;3 and 4, or exons&amp;nbsp;3 through 7. By coinjecting Cas9 mRNA, validated sgRNAs, and oligonucleotide donors into fertilized eggs from C57BL/6J mice, we obtained a variety of alleles, including mice homozygous for the null alleles mediated by nonhomologous end joining, alleles with one of the two desired loxP sites, and correctly targeted alleles with both loxP sites. We also found frequent mutations in the inserted loxP sequence, which is partly attributable to the heterogeneity in the original oligonucleotide preparation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/07/01" />
<meta name="citation_mjid" content="ggg;6/7/2051" />
<meta name="citation_id" content="6/7/2051" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/7/2051" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/7/2051.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/7/2051.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/7/2051.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.030601" />
<meta name="citation_pmid" content="27175020" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2051" />
<meta name="citation_lastpage" content="2061" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kathleen A. Bishop" />
<meta name="citation_author_institution" content="Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074" />
<meta name="citation_author" content="Anne Harrington" />
<meta name="citation_author_institution" content="Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074" />
<meta name="citation_author" content="Evguenia Kouranova" />
<meta name="citation_author_institution" content="Horizon Discovery, St. Louis, Missouri 63146" />
<meta name="citation_author" content="Edward J. Weinstein" />
<meta name="citation_author_institution" content="Horizon Discovery, St. Louis, Missouri 63146" />
<meta name="citation_author" content="Clifford J. Rosen" />
<meta name="citation_author_institution" content="Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074" />
<meta name="citation_author" content="Xiaoxia Cui" />
<meta name="citation_author_institution" content="Horizon Discovery, St. Louis, Missouri 63146" />
<meta name="citation_author" content="Lucy Liaw" />
<meta name="citation_author_institution" content="Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074" />
<meta name="citation_author_email" content="liawl@mmc.org" />
<meta name="citation_reference" content="citation_title=Manipulating the Mouse Embryo: a Laboratory Manual;citation_author=R. Behringer;citation_author=M. Gertsenstein;citation_author=K. V. Nagy;citation_author=A. Nagy;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=A. Bradley;citation_author=K. Anastassiadis;citation_author=A. Ayadi;citation_author=J. F. Battey;citation_author=C. Bell;citation_title=The mammalian gene function resource: the international knockout mouse consortium.;citation_pages=580-586;citation_volume=23;citation_year=2012;citation_issue=9-10;citation_pmid=22968824;citation_doi=10.1007/s00335-012-9422-2" />
<meta name="citation_reference" content="citation_journal_title=Front. Endocrinol. (Lausanne);citation_author=J. L. Coleman;citation_author=K. Brennan;citation_author=T. Ngo;citation_author=P. Balaji;citation_author=R. M. Graham;citation_title=Rapid knockout and reporter mouse line generation and breeding colony establishment using EUCOMM conditional-ready embryonic stem cells: a case study.;citation_pages=105;citation_volume=6;citation_year=2015;citation_pmid=26175717" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=L. Cong;citation_author=F. A. Ran;citation_author=D. Cox;citation_author=S. Lin;citation_author=R. Barretto;citation_title=Multiplex genome engineering using CRISPR/Cas systems.;citation_volume=339;citation_year=2013;citation_pmid=23287718;citation_doi=10.1126/science.1231143" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=L. M. Cotton;citation_author=M. L. Meilak;citation_author=T. Templeton;citation_author=J. G. Gonzales;citation_author=A. Nenci;citation_title=Utilising the resources of the International Knockout Mouse Consortium: the Australian experience.;citation_pages=142-153;citation_volume=26;citation_year=2015;citation_pmid=25645994;citation_doi=10.1007/s00335-015-9555-1" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics Proteomics;citation_author=R. H. Friedel;citation_author=C. Seisenberger;citation_author=C. Kaloff;citation_author=W. Wurst;citation_title=EUCOMM—the European conditional mouse mutagenesis program.;citation_volume=6;citation_year=2007;citation_pmid=17967808;citation_doi=10.1093/bfgp/elm022" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=G. Koscielny;citation_author=G. Yaikhom;citation_author=V. Iyer;citation_author=T. F. Meehan;citation_author=H. Morgan;citation_title=The International Mouse Phenotyping Consortium Web Portal, a unified point of access for knockout mice and related phenotyping data.;citation_volume=42;citation_year=2014;citation_pmid=24194600;citation_doi=10.1093/nar/gkt977" />
<meta name="citation_reference" content="citation_journal_title=Hum. Gene Ther.;citation_title=CRISPRs for optimal targeting: delivery of CRISPR components as DNA, RNA and protein into cultured cells and single-cell embryos.;citation_author=E. Kouranova;citation_author=K. Forbes;citation_author=G. Zhao;citation_author=J. Warren;citation_author=A. Bartels;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=FEBS Open Bio;citation_author=A. Y. Lee;citation_author=K. C. Lloyd;citation_title=Conditional targeting of Ispd using paired Cas9 nickase and a single DNA template in mice.;citation_pages=637-642;citation_volume=4;citation_year=2014;citation_pmid=25161872;citation_doi=10.1016/j.fob.2014.06.007" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=D. Li;citation_author=Z. Qiu;citation_author=Y. Shao;citation_author=Y. Chen;citation_author=Y. Guan;citation_title=Heritable gene targeting in the mouse and rat using a CRISPR-Cas system.;citation_pages=681-683;citation_volume=31;citation_year=2013;citation_pmid=23929336;citation_doi=10.1038/nbt.2661" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=W. Li;citation_author=F. Teng;citation_author=T. Li;citation_author=Q. Zhou;citation_title=Simultaneous generation and germline transmission of multiple gene mutations in rat using CRISPR-Cas systems.;citation_pages=684-686;citation_volume=31;citation_year=2013;citation_pmid=23929337;citation_doi=10.1038/nbt.2652" />
<meta name="citation_reference" content="citation_journal_title=Annals of the New York Academy of Sciences;citation_journal_abbrev=Ann. N. Y. Acad. Sci.;citation_author=K. C. Lloyd;citation_title=A knockout mouse resource for the biomedical research community.;citation_pages=24-26;citation_volume=1245;citation_year=2011;citation_pmid=22211970;citation_doi=10.1111/j.1749-6632.2011.06311.x" />
<meta name="citation_reference" content="citation_journal_title=Transgenic research;citation_journal_abbrev=Transgenic Res;citation_author=L. Longo;citation_author=A. Bygrave;citation_author=F. G. Grosveld;citation_author=P. P. Pandolfi;citation_title=The chromosome make-up of mouse embryonic stem cells is predictive of somatic and germ cell chimaerism.;citation_pages=321-328;citation_volume=6;citation_year=1997;citation_issue=5;citation_pmid=9322369;citation_doi=10.1023/A:1018418914106" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=P. Mali;citation_author=L. Yang;citation_author=K. M. Esvelt;citation_author=J. Aach;citation_author=M. Guell;citation_title=RNA-guided human genome engineering via Cas9.;citation_volume=339;citation_year=2013;citation_pmid=23287722;citation_doi=10.1126/science.1232033" />
<meta name="citation_reference" content="citation_journal_title=J. Bone Miner. Res.;citation_author=K. J. Motyl;citation_author=K. A. Bishop;citation_author=V. E. Demambro;citation_author=S. A. Bornstein;citation_author=P. Le;citation_title=Altered thermogenesis and impaired bone remodeling in Misty mice.;citation_pages=1885-1897;citation_volume=28;citation_year=2013;citation_pmid=23553822;citation_doi=10.1002/jbmr.1943" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=G. Pacholczyk;citation_author=R. Suhag;citation_author=M. Mazurek;citation_author=S. M. Dederscheck;citation_author=P. A. Koni;citation_title=Generation of C57BL/6 knockout mice using C3H x BALB/c blastocysts.;citation_pages=413-416;citation_volume=44;citation_year=2008;citation_issue=3;citation_pmid=18361795;citation_doi=10.2144/000112706" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. J. Pettitt;citation_author=Q. Liang;citation_author=X. Y. Rairdan;citation_author=J. L. Moran;citation_author=H. M. Prosser;citation_title=Agouti C57BL/6N embryonic stem cells for mouse genetic resources.;citation_pages=493-495;citation_volume=6;citation_year=2009;citation_issue=7;citation_pmid=19525957;citation_doi=10.1038/nmeth.1342" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_title=A mouse informatics platform for phenotypic and translational discovery.;citation_author=N. Ring;citation_author=T. F. Meehan;citation_author=A. Blake;citation_author=J. Brown;citation_author=C. K. Chen;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Ringwald;citation_author=V. Iyer;citation_author=J. C. Mason;citation_author=K. R. Stone;citation_author=H. D. Tadepally;citation_title=The IKMC web portal: a central point of entry to data and resources from the International Knockout Mouse Consortium.;citation_volume=39;citation_year=2011;citation_pmid=20929875;citation_doi=10.1093/nar/gkq879" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=E. Ryder;citation_author=D. Gleeson;citation_author=D. Sethi;citation_author=S. Vyas;citation_author=E. Miklejewska;citation_title=Molecular characterization of mutant mouse strains generated from the EUCOMM/KOMP-CSD ES cell resource.;citation_pages=286-294;citation_volume=24;citation_year=2013;citation_pmid=23912999;citation_doi=10.1007/s00335-013-9467-x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. C. Skarnes;citation_author=B. Rosen;citation_author=A. P. West;citation_author=M. Koutsourakis;citation_author=W. Bushell;citation_title=A conditional knockout resource for the genome-wide study of mouse gene function.;citation_pages=337-342;citation_volume=474;citation_year=2011;citation_issue=7351;citation_pmid=21677750;citation_doi=10.1038/nature10163" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. V. Sviderskaya;citation_author=E. K. Novak;citation_author=R. T. Swank;citation_author=D. C. Bennett;citation_title=The Murine Misty Mutation: Phenotypic Effects on Melanocytes, Platelets and Brown Fat;citation_pages=381-390;citation_volume=148;citation_year=1998;citation_issue=1;citation_pmid=9475748" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=G. E. Truett;citation_author=P. Heeger;citation_author=R. L. Mynatt;citation_author=A. A. Truett;citation_author=J. A. Walker;citation_title=Preparation of PCR-quality mouse genomic DNA with hot sodium hydroxide and tris (HotSHOT).;citation_pages=52-52, 54;citation_volume=29;citation_year=2000;citation_issue=1;citation_pmid=10907076" />
<meta name="citation_reference" content="citation_journal_title=FEBS J.;citation_title=Inhibition of nonhomologous end joining to increase the specificity of CRISPR/Cas9 genome editing.;citation_author=S. V. Vartak;citation_author=S. C. Raghavan;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=H. Wang;citation_author=H. Yang;citation_author=C. S. Shivalila;citation_author=M. M. Dawlaty;citation_author=A. W. Cheng;citation_title=One-step generation of mice carrying mutations in multiple genes by CRISPR/Cas-mediated genome engineering.;citation_pages=910-918;citation_volume=153;citation_year=2013;citation_pmid=23643243;citation_doi=10.1016/j.cell.2013.04.025" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. B. West;citation_author=R. K. Pasumarthi;citation_author=B. Baridon;citation_author=E. Djan;citation_author=A. Trainor;citation_title=A lacZ reporter gene expression atlas for 313 adult KOMP mutant mouse lines.;citation_volume=25;citation_year=2015;citation_pmid=25591789;citation_doi=10.1101/gr.184184.114" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=H. Yang;citation_author=H. Wang;citation_author=C. S. Shivalila;citation_author=A. W. Cheng;citation_author=L. Shi;citation_title=One-step generation of mice carrying reporter and conditional alleles by CRISPR/Cas-mediated genome engineering.;citation_pages=1370-1379;citation_volume=154;citation_year=2013;citation_pmid=23992847;citation_doi=10.1016/j.cell.2013.08.022" />
<meta name="citation_reference" content="citation_journal_title=Dev. Biol.;citation_author=S. T. Yen;citation_author=M. Zhang;citation_author=J. M. Deng;citation_author=S. J. Usman;citation_author=C. N. Smith;citation_title=Somatic mosaicism and allele complexity induced by CRISPR/Cas9 RNA injections in mouse zygotes.;citation_pages=3-9;citation_volume=393;citation_year=2014;citation_pmid=24984260;citation_doi=10.1016/j.ydbio.2014.06.017" />
<meta name="citation_reference" content="citation_journal_title=J. Vis. Exp.;citation_author=T. Zangala;citation_title=Isolation of genomic DNA from mouse tails.;citation_pages=246;citation_volume=6;citation_year=2007;citation_pmid=18997894" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="CRISPR/Cas9-Mediated Insertion of loxP Sites in the Mouse Dock7 Gene Provides an Effective Alternative to Use of Targeted Embryonic Stem Cells" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="twitter:description" content="Targeted gene mutation in the mouse is a primary strategy to understand gene function and relation to phenotype. The Knockout Mouse Project (KOMP) had an initial goal to develop a public resource of mouse embryonic stem (ES) cell clones that carry null mutations in all genes. Indeed, many useful novel mouse models have been generated from publically accessible targeted mouse ES cell lines. However, there are limitations, including incorrect targeting or cassette structure, and difficulties with germline transmission of the allele from chimeric mice. In our experience, using a small sample of targeted ES cell clones, we were successful ∼50% of the time in generating germline transmission of a correctly targeted allele. With the advent of CRISPR/Cas9 as a mouse genome modification tool, we assessed the efficiency of creating a conditional targeted allele in one gene, dedicator of cytokinesis 7 ( Dock7 ), for which we were unsuccessful in generating a null allele using a KOMP targeted ES cell clone. The strategy was to insert loxP sites to flank either exons 3 and 4, or exons 3 through 7. By coinjecting Cas9 mRNA, validated sgRNAs, and oligonucleotide donors into fertilized eggs from C57BL/6J mice, we obtained a variety of alleles, including mice homozygous for the null alleles mediated by nonhomologous end joining, alleles with one of the two desired loxP sites, and correctly targeted alleles with both loxP sites. We also found frequent mutations in the inserted loxP sequence, which is partly attributable to the heterogeneity in the original oligonucleotide preparation." />
<meta name="og-title" property="og:title" content="CRISPR/Cas9-Mediated Insertion of loxP Sites in the Mouse Dock7 Gene Provides an Effective Alternative to Use of Targeted Embryonic Stem Cells" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/7/2051" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Targeted gene mutation in the mouse is a primary strategy to understand gene function and relation to phenotype. The Knockout Mouse Project (KOMP) had an initial goal to develop a public resource of mouse embryonic stem (ES) cell clones that carry null mutations in all genes. Indeed, many useful novel mouse models have been generated from publically accessible targeted mouse ES cell lines. However, there are limitations, including incorrect targeting or cassette structure, and difficulties with germline transmission of the allele from chimeric mice. In our experience, using a small sample of targeted ES cell clones, we were successful ∼50% of the time in generating germline transmission of a correctly targeted allele. With the advent of CRISPR/Cas9 as a mouse genome modification tool, we assessed the efficiency of creating a conditional targeted allele in one gene, dedicator of cytokinesis 7 ( Dock7 ), for which we were unsuccessful in generating a null allele using a KOMP targeted ES cell clone. The strategy was to insert loxP sites to flank either exons 3 and 4, or exons 3 through 7. By coinjecting Cas9 mRNA, validated sgRNAs, and oligonucleotide donors into fertilized eggs from C57BL/6J mice, we obtained a variety of alleles, including mice homozygous for the null alleles mediated by nonhomologous end joining, alleles with one of the two desired loxP sites, and correctly targeted alleles with both loxP sites. We also found frequent mutations in the inserted loxP sequence, which is partly attributable to the heterogeneity in the original oligonucleotide preparation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
    <title>CRISPR/Cas9-Mediated Insertion of loxP Sites in the Mouse Dock7 Gene Provides an Effective Alternative to Use of Targeted Embryonic Stem Cells | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"XKUsgiVb3Tk-yjALdOfiy2eDnCMZJAxkvvW0UHgoifs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460434","apath":"\/ggg\/6\/7\/2051.atom","pisa":"ggg;6\/7\/2051","ac":{"\/ggg\/6\/7\/2051.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/7\/2051.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/7\/2051"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/7\/2051"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/7\/2051":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224dz5y9SwFLO+e2a9lziCDSasVNqRK9Uoaa3d\\\\\\\/c\\\\\\\/NmR1rKxhFsJIeZj7vrhLCwPEdFhYnH\\\\\\\/0uz6fdYUgC2D3Z4o7eKRaCxitBOskEB6hf3wj4VX2G4lHg+RdHpF1eikwLcveXhlmvm6d3Ov8dQ960\\\\\\\/CYBtzJUkLjjP5yWqDMyWEyOQbNK5HqhziYY3LQlSK8fSSI3R7RpZDo\\\\\\\/cN6EqcDTOV3ukKc8gzDQtNPkdjKZmPE4Q+9XQpgfxEBzIAnX2P6MZayMfRh6CbgsdhBMBIvYDvg+L1eHM5FcZYVIQa2SMVP3oHAzN4c4Xi9phRGVI5ZizTMTRqoHfhUubsNGvYpJ3N6HXcU66BDHN+JkaalteInpsuCi0SNd0+16VNomGPEU4s5EZ7c9H8UwsIMUzudDQFq6sSpSuLt5mjIgPJ8Dwcvqg2zLQ+tENrTHtsx5k8lowKaVpNUZh6VMBVxIC6AwRFPsE1ay4DWKQSE+EUNbQxYxtdEbVTFlRHBNHqw8u06bs+aDNPxLGxl1PJ\\\\\\\/sCuZQSR\\\\\\\/gaSJHhjU+rnZIU0DUZreGve0j+5tarDiGZaPaomki1CZ2ooHEd5Tp4myWzNLnbf5YdpyIkNx+EHUFQHS6+uteNEsxpT134jYf3\\\\\\\/MM7wyAt789vSj4Lu+zAQX9Id4RiK94xGi\\\\\\\/4rFjvcZOhXJ1qAX6ka6VZoS2Z0jIj2XbF9tyWOeClnbkKq9apfiIR\\\\\\\/HryLSiwh5qiiHr3gnphlFV14akjkQu2wLIFbJDriL3S0rmQxhJx9fAyT\\\\\\\/0dCnw9TqZpEa7uy4ZjyNIajyVHYb33nMyG5YURBNcpUp9NKbERfrr6RJGQbR4xi2\\\\\\\/0PbcVtRp0nnxosMPcrrKk0pRPpQODTy98Xm7PGFJnEmpSOcT6txQLmzJEdlpREG\\\\\\\/EnQGl84NH\\\\\\\/J2l8pMm\\\\\\\/MUxLycMuzryZTrXBwC\\\\\\\/jT+w7dGcAC\\\\\\\/dxNPHpoIMYdTrjk8iHM72ttb4nV1z9zJqHnEz\\\\\\\/aL+yw6gK5Hk\\\\\\\/lFeacfaKoKoMViLAIF9Znk23ebFZb9Wrhm56JbrrvPREbN1LfvJDfgbI50X6hj5CxFPZR+ymwQttmoulQUq9Z07FwakwEGGsFQZxqYz1YJNp03Mjg2+mQrE5anqruu1T9\\\\\\\/4RaBHol7zHwy4u4e7Am\\\\\\\/Yu9W\\\\\\\/q+Y+2C4WveIMQM6a\\\\\\\/1dXaQaP9I5uielHAY1jcBghRgWYIcN2PHyIL9hNPVYln81ksRviFYd7qHNFBBsPpUeauGk7k9+\\\\\\\/TUcWF7kTwbUnoY+lo480tLNjFnP\\\u0022,\\\u0022iv\\\u0022:\\\u0022BL3WuRdEijuZG7iaVUS93Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fce8ac477310de03b104192e2673ee9dfb316b13c38a1cef30ec6972eea017dd\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kmRPxM1n2zhPdFDhQAp6TGEe21qSsEGyF7ZSXF0qCVbXWeBWaOHEbPtbHaT4NZ4f5vq3lPOgw6BjbZ6fyn\\\\\\\/Q97OOzpErwWm1zM7LrSpBUvfgXd74dI+wXSIVvPzAD5tDGEkwpkuf3nMyB9HO3pBgBqC24EXJSTIiWYnQ6oZvJWyXPDUtZXgX\\\\\\\/\\\\\\\/BG8lJnDJQvkUotv97PjC1F2Ah4MqkRiwiJrqni7oBsFmTA5cHu\\\\\\\/zMtOITWUF4x418zPZGCi0KVrHKc6saSESYG4W\\\\\\\/MU+U7CP6OMHtWHW71756SfGmVccnGho5LFKvpEpotjxhan2MgLTO27yaqyozjZEAUxsCafavxJRkiZ56e1qv2J4RQFCqbfOYyJnGbhubets7YboAEf2EvRmCMIl05wFpfoq+Q\\\\\\\/wXcAs8LeSu0IDROBb+1T\\\\\\\/yX7ncdxueAHv+vjEODA6+z1+1QXqoEbB\\\\\\\/+A1Pwhig9W+1GcHkSw6j01h4l8vNAThrTLstBIvcPy4YdtpJnAQog90CtRbDpjy4VppMWDCJXhbOaWtZx9eRGpwQC5S6itAjI\\\\\\\/qmEl5BLQV1NMq\\\\\\\/m7Bj0AcOrI44Hgzu+2uvMSkoAyQB2BuT\\\\\\\/1LtUm0tjB4PgmuV\\\\\\\/FYiANg4wuo4wEoVB+6J4AcUNXgaYcStY49W01o7yI3HOCdsPUO13oDZpsxfnIX1a1FjZLYsjoaUuWkTjMopvn9De5YIQfvRFq1r7aAXI6AmZxbzRYOxkpUIowOH9PnYpOveDU7wUO5lcV4pYEKGKcMPKoRRnJH3mOZV4I0QemnU+FsNPSIC9ft2CqYiHHGfzij\\\\\\\/HRKqz\\\\\\\/yroja2qDnJLnenBxpksQhkVQITxiog7og6jGy9pC\\\\\\\/699bMTX8cEcZBigoPM3LeeDlkt+9rk2MtgasLDfK5i+YgDx9GW+QGW1Td\\\\\\\/UIWwY2dxNKpNOnV2FRHezX4mnnIFdKKNXwsjfT0VJTPQ\\\\\\\/bBl9EnnqSkfy51RL+USN6YZfyab6jcgnkS5rPQpl8VTiKwAi7r5gRLY4PSkU1+SDV\\\\\\\/BG1+iHRN4aUvffnOr\\\\\\\/lhQEqi8C3i1FYKR\\\\\\\/XQSe+IQVsYAKWabQDhVAJUrt\\\\\\\/jrYeOO37EyoFmCCEFL7PFrD+rjnc\\\\\\\/kxwQwnXtw110ari3P+wrkwCxX9swsQkT+K\\\\\\\/1BTmNmK5IZir7qLMUUQbrykECDyLd2ZmHbMG+6rUeq+MflMvlEzHpH+h2hlnQTmcKd522Bp6cWtVL9sM0g8CuxWUcuuUfTn5SpUibyGYpROaqk5v+bD6SF2sepB3pYVQBtTPYCVV3T5W84ts2TL8AWZNinsBmNBbmwQIJQ\\\\\\\/iNJthec14YBs+UWD52EKUMRAwf6pkRiqmlNQtyAxIIRR3d33jMm\\\\\\\/fgpjYux3fLMDnWqwBS\\\\\\\/o6MfJR\\\\\\\/RmwDeT8ASOQ1vnUZjWrkvbj64WrHphS8XrCGjhP3f+MwT6Y0yzlR47LfG8C6\\\\\\\/LUkiVHSF85EiNejHghr5UuGKVx1nBohTB7+0BdH+\\\\\\\/Af3pqnOotwcRjgDS6h\\\\\\\/rhbThjdsTv+FUf1GxilqCeAJT5jzGoUGzk9yrQ1mfEreF17d5wK0UAnMRNf4eZRlskWIQtCsdeceqg02mLrDMT5HCJ\\\\\\\/GA5\\\\\\\/eImP+gO4aDjqhdagDJ91x8DqIg4PxLYLKaYNbpnpgrZYPkhGkCduOO+0E2k0cUGkwH2OK7KGz24mzqKd1rtxrWvJr6STqP9VnGIjcryAtCcEY\\\\\\\/ck7bS5mD+DWt\\\\\\\/4DBus17BJTXst5rFQOZXQibWrDLivxnb+txodXfPTX6sNT54WKbjtsi+\\\\\\\/d4mOPJj\\\\\\\/E69e3EaHya\\\\\\\/Iin8eNHv1U\\\\\\\/FvepTjfHz8x+7GRBa\\\\\\\/NYB+qzbOtiJ+QufI66hSayJ\\\\\\\/1TgETzT6+auuc8ooAba\\\\\\\/Afu0eqd7qbVmPlCD7CvxlXQkDX+FxAw3WLmYB717a\\\\\\\/T8DijlQWW2Wisehkz6kjYwKR2I5VrO5HyZBFT71MIV6\\\\\\\/dkvWrVHd1y2nHV6YPSM1mmdtcCu6WwMYWoiVZIthUi6bwEW2Ayr9RtN96NngKklY5jIPySwHNpvdun7baP5FoEbzCp6wqfWJnIbWTpHPznMW8hhMTVS9C9sNZQmzuXSo3tRXetTp3ipGuBK+kcW0CLRkj1Ya\\\\\\\/\\\\\\\/2cko8EPJtu854kf74HHyl4z9KlsNdS1kL8VT9IKU32C6sSU3QFcZbUYRPVKq7G9JVrBFxd7S0i7iWgXDxV\\\\\\\/g9yku2o1TPe1hVbwc4PZuBog6kFA1yN60Pr7rxwnV+SLbWDAq2QL+gn26\\\\\\\/Z7booHKQB7c8T2GinzNsfzQdf7jaP0N4cflLr4YmykZhaOehhxSfOQ1ZB48\\\\\\\/PTHlAd2EiOpMwSFeERV4q+z3tm4CgFeMI+BPCQFscYQ97mkyhxFOX53igpFLmDicrXy\\\\\\\/xickPuNHkbAeQ5XOl1Um4OcjlObjnQfYqu7oZdcAXmG1956UTaSmkHuOAeDnejsByMem1dF56iHgHsDrPyJVzdsHHX2YG77MkgnuJHZAtjK5X0Nhyer80D8WXd20NsE0Uri0neUi0IdM7ZxIFCwo25TiNrxvavvRsguEU2mSamo\\\\\\\/tIwah3vGaBEGusloKFL9H7Ydn3JycyJarozCL+wWPNZCdQljeDFxEBs9twSrnGAFhiTKkh7dfYMpy9od1GCoK4VT84CR81CF22BR2\\\\\\\/gnlwK+uC3+EJ8Kj\\\\\\\/\\\\\\\/\\\\\\\/x4F2juzax1wpHU0+x0mjZa0pxXYCiqlwnNNZTfhag0aHmi2pavBQuIadzSkp6GX74jEAyufrMMx+mmlr+ge9eFrGzgrTIJBjueacb+abfXNz5wr+J3hib18M4e2zcbyEMUm0QOWnBiuUUoImFinLJvNLpBO0+QHzDQZ5TksSzVYbX4Jm\\\\\\\/k0X01Wxc1PuepkUk2CFQViXEIk1qvp+ftbAqL1LvjZr9\\\\\\\/oDiR\\\\\\\/cLbkdNMMY+VxNS05k5RXFfVF42WWm8a3LqAxisN5edjdyyrcIkbJgjzbU2JwEkaLYPDecHjruY0l4HXiJs7MhKWsXAqrK9wXoUo1IQxmHVDiVttJnMCdpTG91hHtSjPAqwQ6y\\\\\\\/Ho0r1gqcIK4c3935wdMd\\\\\\\/xB7y7boJee2vETxl8KCdm+LW0c2xOKpxyQG8kG9E6GKQX5U\\\\\\\/9g7M0znMfSG7vhyhUs6g7RVLXrLnXHJEYbIrg3pPUkKOG33DxuNopwlkKISrcmC6M2b+tI56UorPwammzDEF5vLZ8xveFmEx27v1OhcFqJrhkDGNDgdZKLIPtiB7XprxLcv1BmTopXbb3oB0VfyG8RqvqrWrz6Rc6xuKbZ4Up3X+yHX0LzZr\\\\\\\/8dKefpnJQiGCnlfZEjllwKUd02NweQBZyz0Y6sXy8LY0\\\\\\\/YbzgSWdrNihjMizGcmyf5HyhJFU\\\\\\\/l8lriR5N\\\\\\\/9oWJQrtdN5ofF0rWpLkSyvaz8kenVz0BC03\\\\\\\/jEzMleWvzMCbYW9f7YY+ghduk68sikh6s9rLcYfMHF9f9Ecpi2A6DVEZzbUb\\\\\\\/aWJFz6zMU9hhuQ5oOPgRxYeRkfECkUenqVj3pmHmnRf0pHxNHwXo2gvw6OY+jk0VWd5XPVAuuZiABZmIUXI8nT8aBD7Gm8PvDxuEJAekl6+0U300MVrJwHG0ejn+18thkz9m9RnvZrMsGlAa9OHjiphWr88B2IinK4R55wMrMolrTBH9KHSw8WsZJXj9g\\\\\\\/3AH\\\\\\\/0xONEAKhdhj3VlBqcDVrc8Xnssf4H8YH76mIwhjv1CDbLW8KKLIMKPtBTruaKQP\\\\\\\/LGvLPZdhnciSXJvPHein0mYTosY5fCXA94ykwoWM9Gr7AobLF1LY633bzY4UWNIuhXPw+tw8XM\\\\\\\/uYTjmQs+fit+MPg7ZrlBss3MTneXwzOtKxedlrCIwuz86TxqYpFmcGkqDZlF243zRA0OO1uZ26ZY5DYLmXP\\\\\\\/jLapL4ewIbAFBYpai3vIZKWKXx2fXCOIEa3aONRMKeWh087u9AE9RTRtzOb0c44k41NEPzS8V3SfiKpQecI6xdFxcHi74lO4qwkHCcfkuWDAmiBsN2MKIqrcGdNtJc1w6vifiuiJprcFCjgtIZoPNG3lEi67\\\\\\\/wiaiHUz9ijo46gMB0DvWm4aJj7UgOJTh6E8BS83SpYwymcUIPHIlXW4uAPd4MhYS3z42xKNwYtl00RfwVyQGGbdyjtAKP6aVsN\\\\\\\/orqKE4h2CXbEaCEwC2ZeDHZt8zqOiGngIhbLcC0e9dDWYZ6UPuTYdJD+165+LzDC5pfwojlifgCcjdoRnVtZvyIdxDbgrUW8T6TcypGlR9ZTaS6Sj5AEQRs3G4kkUxKq4FSNyi2u3lV16k7\\\\\\\/MUJhRhhlJk0T8a5ulTCENlT4TQk6umdkVtTnFcLdGfzHtP6ZG2zJ8aCHVxQkpOukWQaRLMmA2JGOAKI7ZcqYJi1alKPI4rFXM1xvzmbnGsET2i3IHszjt\\\\\\\/GGLZOR\\\\\\\/5wfLjgEzMc8YkXEvbxNlSxIY1VWXPvrgOHBCCltw8\\\\\\\/awLQd8\\\\\\\/eR\\\\\\\/lXZMvi7d3\\\\\\\/IPPpZVb8+ri4NxNd2rbN6jUfA6VsmWd4DyV8ffI\\\\\\\/KWkZlFOfk3ifovFDscbGx\\\\\\\/Rf4G+VeRVEnSQ0RRRfvSjz3NXIpAhaEb0yAFyMuhzp4S9ol\\\\\\\/0ivShwakItvRe7HwVRdWbWpwg5IpI7xpSh6455bR+skuosodoK\\\\\\\/+xDeRnAYUDL\\\\\\\/Vwr+WbEA4uEC6l08yp3X8UmSy1efJhu0rWK1XOesFslA2ClBr9ggqEdwFMf+ejQQH4VXnE\\\\\\\/aQQ7ZzKMiUDhq+l7DhCUwhOyDaLnhmJooOC03WdGM7vBZ+52pjsdQLpE3eFoIgYnunryTjI+cxOgNFHUotYxsMYhkTrLpYGKTT92SOGY33V9aC1cPdMWKi5P4N+USDc2qNyQwmNNdFDx2pRuYUXVszaAZp+9MAHxBvnZVG5EJ5e7CC0K2zRJSEe1x11AmVRr\\\\\\\/NlN+lduH8hkopJA196qXNCQLDtF8w7EPpU2jW0ajALiPEkMhmdei1pwX35vZz\\\\\\\/7tfamvEz4SOzD8MazPTcq2GgJCNGE\\\\\\\/fS605foBvRmxI1kGXy\\\\\\\/olwW\\\\\\\/WLm97xKTl4gY69cAH8ZCpCAudMu1SzdGDk+nx5QxcA+F8mu7pMr5YJNEgkaV6kxc6sz4hhF0jIe0J7A9xO4iv88gc1bNLJquR6Zrr8EdM1+v++d9QHnOkw22BBwE0hOtmCgbI64YP5ur2a3OOZfQ9AibZMomjlc9C2EOq7+2U\\\\\\\/100bJx0EswuraK1ZQRs6\\\\\\\/B6uSvk9tNN2r3mw4qwjehB5P1OHdhWkegB8V+zPnqj0sbGn6yHon00lvrwNlij22+WnBy5BfZBSn3QQNhaav1YtrBZ2GddFOPD2vOg72\\\\\\\/VmgcvaMhJtCuoRg1DZw\\\\\\\/NvzJLPeBtO4M4nBzRuI7hYz0lOWIGwB58akFBNkhO3EgY+f8LA9fg261nOxM7Vjg7A9qHSZs6kXW5Nz6a60YAvCqX0I0ucNIfSqf6Oof2pGWxTl6JHD51Hw+7kV9avID2xLPssHyHSx2tSZxyk2VqDkEa6si5l4KhY93r5pRZX7500nR3ZhserOj0mXNB3PNQtKQSPkXBFDN6LkF6BdQvLmG966UwBM8820D5eER+YsQ3n38KPohW8WVSsGlOMQndjZHxV+KtbEItfYcj8WN74LDr7sBjPxewzcT9T2MnFlEHE\\\\\\\/YolwZxCW6A8LxyL8Gifkz1LfT9nlVlSQ7TU8SXpoESmurntdUNI5EQZBlRJf4\\\\\\\/xiI6xGt3c256Jz5sU5Z5KlbaMZ4tt+bAQlIHH9dybzMuoM+c61kP7DRP5Ccr+VNwj6aRJktrl+9Kki\\\\\\\/Aks\\\\\\\/h+bFENkjpjgzxeipnuZhXMEN+JMFJvptyHiCiPEZIkve\\\\\\\/CVPE2hCeT+sk6NFLtIg\\\\\\\/PzlZj7wqMHLMT9CfalX\\\\\\\/InG4E5lzH9xWYgRKsufRM1r0wgaDTjVnpa4L3FrVgqQ2KoP1LdgHziQNl\\\\\\\/+pls+4bcv5GsrOGYxuThdaGjswPweDA0Gk7chvn5VRS0c\\\\\\\/x9H0H8dDfGYzwAZjokLtoWRuj0jZA5qBfJlHvuIX5yABAqG3BxivvJJvKBQgqfbxacXCSjyWHrKJTnAjAoJal2UtLAIvm4Gh0dAtOnzVeec3N5k50MhN3oAgHrMy26tVj7e0wfuJ2OO3qNfvnnkg5zDbXpWkcvP41EoObUYiVITrlt1Nh5Q\\\\\\\/8MBjkTro5SN7WZmtg\\\\\\\/ug2XH0hz10BaKm\\\\\\\/udKTanXosqqmF4djIHNF8f5CSdsbbDEtUS7UdSf0htZhZBFwLXCbp6qjgZ8z\\\\\\\/uNRb30mBsVFHbb8rh8d0DqS0YI9DDp+3XIXBbZbfx0xyI0fPj\\\\\\\/V+WHeg1CO3n5cuDLXdxHNl7O68NtIAXdWq9VbkQQwJVxxzytuV80WY6EK6h9E30Na8sz5fd0W5zhdTan\\\\\\\/UQrV4Z7hedsAthlla6ayg50Q2GpYLR+gosFhGk4lOCe\\\\\\\/qISN8ryGvPjLnUHcJPhN6HjNpH+z2ir4ZUzH6keZffp\\\\\\\/ZEQFL1QPvnOelXb\\\\\\\/ZjDxEEnXDZib3Jv7NwzmJ4jHTkC9BVzUATC4G8rLx1QsjD1C9ncZNnKUB+2gGXux245XzquqvBkELniZ2jfvAIs4QfREwQBgGiQWrZLjV9AU3cMbkLIFHphlmmscAnWhp9r+yfSX+ScUUMYdnv\\\\\\\/kdXaT7yKLmVpYX\\\\\\\/ARt\\\\\\\/uVJl5yBFQqzoG3+zEOFDjHxm+4XwzokFbuqxv9Xi6GpGACWiSR8NLl+hhMgMkIrfg95Wr5jD4pflNVHsooPR576zIgrY31S9PQcanTOZ6at+750VXl9\\\\\\\/lbaz1g0IKJ7pFY9Uby3e7KU3ePf6esdkdnmxijUtn6soGVopjvkyP3sfVSczMoCN\\\\\\\/Cxjli7EZYMq4hPoztLvREAVE2QoaKEu8Wea08BUn7XkbkEYTOrXrLIBiXhbPG1PM0+yLOvlhyqJ6g+K6x82CxrDGPzrNcn1SYM7TBZ57fwYDfIDXm+jie5iZQAVYLWNphMf5qepCpFruzMPvI+eW+cqPT77VQ7IwqNf8o\\\\\\\/NAlxKDMHNmf+\\\\\\\/MNGRhS\\\\\\\/7V1ekCg\\\\\\\/kYqDei7f\\\\\\\/Gwk7CW4cTD0LzeBdtzfA+\\\\\\\/W1xfxxcxaB9yJ9EOUxI4fkbl9YhSNiBiw4Y64BcAx9SwwWtAk\\\\\\\/KVOMmG58SDfu62SaCg9ip5BQHJhsCu34\\\\\\\/oTaXyxNkHsH\\\\\\\/aoz9PhtZtNv2Qxdq10hRvA+K5VdBZ5SZt9kgsJ8oX\\\\\\\/xgdKODNewBCP+Q5ilL5GdbKxbeJSJYQzCz06z8DWUDHe3ZUQKAoHB4tlgptqcR8W\\\\\\\/LvxSuZbr62AUvDz6M3ESepDn2VbunKMahDeiidLcTEHqip6xNUMgiUdfThzcbNn79kWM9m5YtMXTDfMGORAkvLs8mPXwZb\\\\\\\/FxZKAof3voTLDNLCGaEmIdyqWqRJ2NeNSPq\\\\\\\/7PjPQ\\\\\\\/j98E8Hq8s\\\\\\\/\\\\\\\/GwXxLSmr9xnKg1t7qd9v5VAQE6JmODDA+\\\\\\\/Q9h4tJfjEqhyzCvJAYg0Yv+c3qKrbvvELdkDR6mlo0qJN7Fdww++Xcpe7V1qJYxA10Jqsg0MuM2374BQCUvCRXfdXi2\\\\\\\/bII\\\\\\\/KeDMIgnk2HORq9EK86fA34P8Ie7A7KNsLtyipOY1Xz3eKUku0BjT4mTLlOkDy4BBwQU9038\\\\\\\/HwJpOz\\\\\\\/NRcTpp27hqRot2gYLoVilZEE43v\\\\\\\/IlNIPgqQeDN8jJshkCHPeRjnP2IJPBlb3HanR4hmHqh2f9jfWukQWgakoQS\\\\\\\/I+rOVJwD3IFRYq7QHxXQqiT4n\\\\\\\/xJubGAbmlwIsGoJm2TkSITspWGZwGvf1G7CzYcwdlbjOYuJVtNfOrh7hiA2l\\\\\\\/e5PmMf4dny74l0WyRMTDLOaJTvIMCZcySntZpERxJ9UqcL0JF9hRsv4ZcJYNQc5YPWngufBQKZyWtHNsqZN0MN8CGFUM4FRJ2XQonB07WeoECwV93Xy3+UXWL\\\\\\\/z7U+FyZiMALzbP3aW65aorzQWQRjHhLeY2Raofia0mpxoie029MpkIcDEQzWbi67DYyJRWhKR4Me7zuDVLAP9gJrocdiIuuzde6O5jjXXrAIh10CZyDpjXpfnS+2vvfkb9j5uyeO6wQWUsr9+hivcx7SWWiA7tcWoe66mUQMl9cBr63502pPgTxSp1Ja+iaPguR4izzecdpOxfi+OOId5eSTasFU6R4CLD1ARDty\\\\\\\/sNgpV6Aa+fYgDkMsWMqAb6Dyd8PzGWfgMhgSHLa9H6H0tMvfOE5DYz4OuWvXBe70P4C3C\\\\\\\/oo7wkjDbh4FsMsPvkm03w+WYZifXmHWntjBGegdRPkGu+vxvYgtVAIimfOqUUuBLk7udsIZo+dOtx6SQW49og7i2jHnTWDXHQi3AeRGryzL58z532WYY41TPbEhtVgxaqjqN8R5zBT3JSCOA3liLABoQLAt31NbPXDUZP2bGGhcS4xueNgAWlnNCv9IJ3woU947pSKh2\\\\\\\/F1UJUS86Lho9Np2r+WG1A+oC+IwjBbqoRmVHtiiz7jp2lKp\\\\\\\/O75PUR1GNpW+SH55v8NMkhVzv9nVCnLXn1Z6xflf6d+xAN9j8UaKqBfv7GGRg\\\\\\\/5Nr65NkfUgbVoWOtopgynyU3i272rAGvIeEEjpQ8hBLuuEzrQcpsn0poWLbP2PX7f190vbl43VzV14+1ziQg9WBL6BQIV5QNzCiBwVMlbqLShGAKBycSp4hTJB2T21I\\\\\\\/2SMOTauf3tuxVvRAELIXTr9Ib72WgGlClylf0plefoarq6KGlBkA5a+adHR6Rww+LVdHwM+FQipPq7hbDmISSBvWTZxljrGP3CkzC1QDFhzwos6yyU1A9n55qEEkP43fsv+07McRCgdyQsAVY8QfhZc68ifF5lkPQNOHAU1u89qwthy32bmpUiI9poIN8a+bjfrv8083U54aoHp+sAUOZxfXOyvJ8m4mx1QltT4xE7tzsBvpQ6cD8mdwJnY3hfU65W6esPSjyHTlgMTd0IfZq5FLH1XGExnAZW9\\\\\\\/GHuUc\\\\\\\/7oHZeWdHzkAbASIWQH4wvG\\\\\\\/bKaZvQ1jUkOO8W+OpdrWoS0heoL0P9iwntW1RtqE40ZlljrCl4S4hvPLIDuGFlF4wICmEkEMJiSs715+Pvc6OEzOb6SVGdglMKxqhYq1Iw6teGoGTvGNMDWBol0fbhyzWph4mBWnXOiVYIKomd6ibtopqGzljtpSxK6m0Ee0hBgeD2FeQFbxba4ta5DjcLlKz0oDktM800p+0nuPQZNRSYKOO8VAShROTsmUGydd3pL8pb+uj9smVvBoGFUV9U\\\\\\\/mGjiLXFGC8Q7qIrSkRfgrbHdRddUhpA0NEJLTlKZKi+yhajAyByFzuOjfhiyQlWYtyIgmEQ4p4Qcxhtp3z3xz04T\\\\\\\/ZMeRVIdPA+ghuXC6UQO3OkWpM4H7Ey8zfPNga8TTN5Pm0fbe7IlQ6xTF3dG1\\\\\\\/CApMfhwh4X5wGTyx\\\\\\\/BjTSIq+OpY+jcDV9JtDbt1rCzq8oBK2hFl9dcjDZH\\\\\\\/KyHiYBt4jhUJBB04hhXmWXpqLyVkzsAdblZOXz7vpG96ivHdeNcxKgYUZyxvprnl\\\\\\\/0B+hqLVmu4Nqoac1NOrNuyY\\\\\\\/A3fy\\\\\\\/o9XC0Amx2YRfsh3Hqy5mSYlkWbfellx8qndT2k1lgmcmYjVGtfYTusV3lJn36fju\\\\\\\/NsAjRIqDEx1TIkRxmWQC3eSrW\\\\\\\/c5GKESafuoeR6Orq+bOVq85djSJs7VNNh4vSARn2+RqqnXWxRFB5H4Zj0H2RsTElNzsJSYd1rpXiQP9WYH22KTDD0Uo5aMRYIjEKXnG5+HIjUpWE5GMmbyjgm5Dfx9QGh7WTswZ0BED3WTg2AJRSYeB9xHyUE0P\\\\\\\/w20dxcGakLk9EzimeazddWbuAhkdqxBUfxF1YVFu9kw77zf2qGgibHD6C8NZ\\\\\\\/quUT1IHArMoSHZJh8HSKtK4Z8I9fdSVhlI9D45md9hCUPrcigWIa43ZUQ9m+QhnW2tipPgBd5yQ\\\\\\\/BPo0EjhgLh62NbWhU6zgJexgsKR9wUzB7KgITkKEZE5mq+XwG2lygfUJaYLaqnFAWmsu81Qaa88qHNO5x\\\\\\\/yGrmlg1gVupICt\\\\\\\/6b8tHN3kHVqDyGz4v\\\\\\\/M53+PW\\\\\\\/ReEL6NR0VRXdMmuV4Fnot70qOK81bU8gXwKaH+Am9\\\\\\\/SGOFbXQnTDwpomB0\\\\\\\/0JVBaEzr+x6CWxmAuRultDkgiAnigFgNQQuwHE+sntpYO0i2zeTZFhEW2Q+CvGvR4GPZeCcSvUqXkENBH7byYQyWlE1NSkNFPKdzczZ0rvjqf3pDmVy3TT6mhQ3RNmy1p6P8JaacT9UphDIr0CqzzlX5BdpfUNxI205r68A\\\\\\\/BpFZrcO3QA6jj3bSAaxJUwyrM14xZUa5BRdMOBHjrcVWEm4OBeQf+E+aXERH2hZDSQvh46kFuzWOHcuQlAkQYXkSWtsuYOCAuKylb9ko3\\\\\\\/Dx+YSTCpExIE9ZHLYAkAL\\\\\\\/Y8Pxo47lZd4Oswf6OQrpqTXQ6cfdtBmR8x6Vt6mXUdjqJGhYkz8SWDTLIeHz7NatCkW8Jb61NZhGrySrsk829Rqfwe7dj3\\\\\\\/vaj4ArPw77JjGmv4oKDCpU0epjzqjP8HU5y7OiVMp8iL9auQkr0\\\\\\\/zKrWnxGb5bbhW6paKCOHyRBwi88LvT6jPsBsW3ory1A9d2UBJRAjV9n3igQsqxSeflndaE6w3N4QhzZuIEfXzLnD6HpStPsO7H7weeE86LUVc77L9eXiVbeamjPIhY+L4elBFp5ZcgHRDlsv\\\\\\\/9Xde04vUPhfl9iA54VuLirbRyS8P5v4sS6fQcE+3aMCkWw6xHAoeM9\\\\\\\/XZ1VLHhyLZLtDsGJBcSu9s3r7g\\\\\\\/UXeL5Suz+3srm2RHKY6zRpQFYAuInYbxBzaU0WrnK96kPiW8srZGdb8g34SRr04OvIV5goqEZiOOx9VkKhGAWWQodxpQmv2bK+w4EMPfejtxyG5whQJdUUtiCgeju0n\\\\\\\/3TxIK6kfq8qiXdgzZNwV9k\\\\\\\/sMZaoPSxtAZsIOOcepq2kJizsY3c\\\\\\\/D0nPk5kFrxZ9LEqiJ8SVYsFfNtiktzQnFahGrEyhnwwNJZMn0jgPTylxWj7GBbBymevQFhjusbEbBdrgz\\\\\\\/V7+AyDRlCbaGvuqXNE6JtR7bXYDApindLQtJ20cwNlNtMWyythI48ZB8QWGk8h7IxQtghu\\\\\\\/6rcQ0EUXeu2nk3jlpI6WLzcqfROARE9vslefzxXmDXRvrqYpHw95gNPWZ\\\\\\\/Hw7zMHmXWi+zKGLb428fcDooj3mBkdTIzYn3QgkMdCPTHNBhLB65AIG\\\\\\\/kEg5nFQwS7E1a7lqyO7aIZQXA2SZQBKzp9HL03OKv0PTN6DYz1y5VKPSRCWKPcjrksN3l759DMph3u70\\\\\\\/2r0H3GpsBGVVmiPbzae3NL7Dce4annh2IjhmYmEGdV20eAvc82Zi0w36dKoFj9nkgOXxiTYMH6uEc7frDi+4HXIj6UUgA7hiT8ctathSbZl4GLb4LJGwFavxs2RzfAu52dC2J1CCyruM6hMejLgx94\\\\\\\/Cg5CjrCAt3tjPZ+t8JnHVS4xqh9AStFOZXZ9iB+GvqBcOnO+n\\\\\\\/naOjtkIGDs1WT7YBFF\\\\\\\/5Ix0uCMaFgcvKeetptyl6FjgVL6obQ8jAIpAC13tOJcw\\\\\\\/seyuGjqiwtXX2Xyy7\\\\\\\/lXCHx\\\\\\\/11ford9JPk+4IES+NyJPShIGUZnyrt7nF09zzUITxHxglcxnknCOWrhRRZLgvrzx0ab3Mm\\\\\\\/Bfv0RE81Mw6cZWyh3p9F7OeapKvRycjsKAN1dGndteooR4CYHj84l\\\\\\\/jgfC2jyha7jZtT3vi2bL9nlhDQ\\\\\\\/igeDY0qMZ1wMNtndzOxcOb49dbgQWAw4U7ZFoExHuSs43ZMQpMyXXSt409Ff01f\\\\\\\/KPFKo1UZl5sqiEFUC3f2ZF7GGi\\\\\\\/80yq25QWFQ32WAdiy3Hl+kcsN6TdMKS4unVVNBk6BGRqk3lB8yXuJSxeUCpucWOZTQ+OSey9CexNzX5NPfh8Ae\\\\\\\/C2YDgDU\\\\\\\/Yr0axzw4xQmUGUHEIqjWZL\\\\\\\/hINdW83e7EYwjQSKbrwwYYRhQT5gG59guglmM3vhj8a8npONxskDjWT69Yn7\\\\\\\/0Mg4yD3eI58BjJAjHoRydOf5di9KrvlnE+hWfGJMESEasuvrgOSOewl195duePZZQbJ7ISvEc5rwEve\\\\\\\/NIEAUBI12wxEcPSgOtVio7rRj6RfApxGTiCsHPsE2tyqgB6DPOfq4vt9kUSA0v8ld8ECvXK9EdCxRbdIgqaPg6Zf5or3VL2iZs0BcPUSNh5xoSCbwKkcsbXti+KpJOt2rQ6p+BjaxQ+jL7m6mXF3iLejLvnCe++YdxZ0ufZSx+Cvn+ai+c0msFZvwxMtKDTBdGGj5YVywTdj9abySqTixKknMgwMlN9q2JUD7paCPS98dZx1v9ZVaDRbHB6Eoyeh7vltp+AzitzRuAsbd+5Ew+m+l6BdGJ2mBUfIV8PNOatxeM\\\\\\\/qByy4+mUYAIXPjYlkQkDH2kUPg4kHyrC4llMImsoCMhQpAfCjjm20xtEohqc81a5cXJ+DafUC9G+YdhLdFZMDB7O0Q7ixXUe+g05lqlA6\\\\\\\/pCktX7vmH9GNHhXzge8eiKC6yZAuAlc\\\\\\\/ZJ1xGvjhCgQ2yE1ohEJIhNVqdREaFQrz2ogFngLLFeszb95r8+dPcVa3O6b5nAfoVtl1DUHRNdyuUJQ+4I0T99FCsNaKZTgEcPcaRs1yage9o2Ki4YIYJ3nmDd3ep1HngZHJj8YyewYR4cZzbT1ls0zTNlNJxdZYMyfVRxSfzZl1WO5BktFSr4JLRFjp2cusKhYFpEGgHzE8thb7Op9IwMv7FaRp6CVRWRo9EeubBWoGKK5nMbDU8IIHMU6bBP1hIeGV\\\\\\\/FVGbeHif2BTvVjrF+Qupl54jUvW2BRiTSYUoW3YYKytMNOfZFTlaUoW5Ag8+Zt0nI8oWvarZfpSPeUw+gf5eq2qMO64ZcTxqV5MWXqEkd5X3Yni4n42YNc\\\\\\\/G4uAysly2aA7WwUhUTcuHpCUrnhjZL2K7H28K0xx5fR4\\\\\\\/vvqnBYc5dXDlgXAu\\\\\\\/SMathYopiwh7KI1cRpTW\\\\\\\/o0M2qA5xE4hr65N3jr795xOeDd34fFf+bwh0\\\\\\\/kjehojUclPWgwtWIj3WEvSVjqmAak1tXgkzHCO4Rv\\\\\\\/Oy6J3iRAv8dTghQEtYIaoZqlHS2NJAXCrWrWLLKzu3vwCbB6zC1N4pmW5GRFruEkrCvGmFI08D2irOqG9W5liqG8P714RnIZtF9u46w53rK46aUGhZalwmwp9u\\\\\\\/v+jqFtQx0jhoOIehToq6gQCtTQecXfHuPTsGUaR3A1+53fIknwMffq1+RYdqkracWtH7nlWvcj5fNE6d9LlyYus8JPueEaL+lxbl\\\\\\\/xlxufnrGvuWDsgIbul6tYZAada5gzj8mg7GKFByGjgPESGYifKK6pRM7lc1OuWmpLXxqMRtXgmKkecK4EhogVS2NwOOrydMMgOaCYVRjTKarNI0yY4+Z8AN86URhnZcV+VUkrmGixB\\\\\\\/FuC2Bu5Rdm3HlEZ5fg4p3juZqQ\\\\\\\/IsEykQDe7ndvuqzUN4Ch\\\\\\\/IUGhRQ+7Tji5qPZLnGKTi2t9ErB1dkuc74Z7HMlQuKEsczJFzOHAKP8qxEUorZbrNAMAYBV7Xwe3P6FxGxPJ4gFRlAOMSBTlLGCj8ltMHssJQ3y7Mljr99x+n9IP1tJjsSHu4MxpLUSdG+oO1sFfZnzA0jPmQ2Vvdt1ssUsuVEYb6kcfr8aHMa1hAd9RKIdH6zKcznR36Zw5ny93UjOsepJohrqEEs\\\\\\\/eHZYyDXwokl3TL7xoqeigZjuTuh4Ew9wW6jgz+Vpz1Uz0ncdDl4+\\\\\\\/raCEUEw+0Z2cE7ZG\\\\\\\/ak11nnqfOfQVkDVASiovoi4\\\\\\\/9eWlkH+eOcAJXcSGqNVbX10\\\\\\\/dopsZdgGPwRhWwo2dMsRH8mqUeVPh5eskFspqol7XOcFWrOoilAc70RlUv4kRQ\\\\\\\/sG\\\\\\\/MSkwsYCLh7W3jN20i05FOIhatkxQ2yAZBSipOG7bCTx2k48PBnHq7wEM\\\\\\\/b702Y3V9ht46FWbdCmHpiWd+rdGkxrqnegzkU9cbo5W9Nq4LJfPsQ6BZ5CnwqAtkjOuPpjgPU\\\\\\\/eGEhJMH0gZvVJXGwv641t2GVDASE61W9CmwDqiHcwLWp1MoE0Uh7hKZpGo765hKTzH+Gb7HMltAtd729WEnoR\\\\\\\/NiNgc6glDwbWXQQ1Jc5\\\\\\\/Ha1q32YCIj76yJ31lbZVtZw5qtL+qCCBvwwqIsY6fPXTXm5x2Zwu38qFQEIzFIszvboOZ4BQWiPcimiCF1vbv3xbvEwjZG2V8ilKT9Dq2THRoAxs2MLdFubNZqqih88bzxF7ol8Af1RwnKJVhC44gIAngll7wE1P2t\\\\\\\/oLDnPeNfc2QqPfrqgQmj67mZJU0MXjVQh+nTcb3LecSAH4DU+ytaSpjiBtgIT6+6y9GZsbnEgS6YdSGjlXgiOlaCjH5cjzDs3kaF9uZuh+cnmDXBSBuuumez9DiAzf86CHzzppn4XyvC5H1k4aFx5jPtVCjQHZCOfqL4KjgEDePrKkhMHkj7hLtT8MIop8SLlPGJGCGIfmSSCt6NXWldJ3WXJLkDMxTWFQg8Nuuc9yamtQRt3KvB1yxCkOaVanZKcZPyZWzlAaSXyu+5DechhMCy+LLeP8ZSlU6GdxxQW+VLTTDRwUfaxYZEgoItkegg+vyhySs4dh72lUwfAfusTG0wtVKy29lZLiaBA9cfTWBGw\\\\\\\/KcIQUZZKjct5WE\\\\\\\/4w09HLgaf63kLLfu45y+5Xc\\\\\\\/lTvG7ipMcRdDf3KBEnBrCvwZ7IMWHfyhedJwg5HRKSp10sRXKvjrrY0GeZ5FMi56FTe0Ux3Gly9n13A0eqm4ilMBDDBrZsx2TiklpMPMkEAjNlXxdKhLmsiQNATr5m1jucQcyGsZNFUhW5oJj+vqLEkBMqtV4bMIOCxZS5gl0ZyFVqgxfH\\\\\\\/DB4ic2le04Umtb2NvAbWow9OKkHE1BsnEZwD82r98vY7Nejj9ex\\\\\\\/rkFxM7jvpWNm\\\\\\\/2LyGzHWqiEbvnFXgu6mT9taIvF1rsuqYfR2om6kCajYfeNpzoo+1FzBYr4xL43S\\\\\\\/YYOvRZcPdvzTSH5Qdzxf\\\\\\\/nGbv3Ah91R6kTsMNZhkd1H+vHVN1agtm+WbLM\\\\\\\/O\\\\\\\/ps6v7+GAZjXqItYi91TUaQlhDZqRH+wC07vik6jxyzQCXWAy3lXO+ygQ0balF9EJfCFFYQRNXsQL9HBb7FJIUngRAYKv4PSHuJxf100hnmizsWNtjNvki3zBezOzqYo7nSmAGT8A3KevRxRIK3RoQI+DZxrmBcbIsWU1ckCgy8trM0aflXyFt3h978CYXrWY88rK5LDXUnwAdbbPLfoHpqLO+1fMMkPfqc4fs4dngdT5FUYeCYhmpNjffm1Vcb9Sij2af2\\\\\\\/8ILsw\\\\\\\/yFNfTBID64Dt2\\\\\\\/k\\\\\\\/Y2sC1n3lfrMGasVVSp0L5zt9tGrGuqNI30dKKPmBhNN\\\\\\\/3O91XUzG9Vx0sb8FZynJe7QRWIxZRdl47qMWpctfyAVVroOkNS5Du9P6g1SS8ebHz0Tup+2EV\\\\\\\/nLMx7m40cct5Ww1xNhJdutBN\\\\\\\/b8zpCQkWmsCxMnHA8DAdHrMs0dt0yzxkKV2+ITHoZ8VAbwxp6p+z+Ty6wcpMqYh8xNZRc1p7sb6lkDoMVuMVSnzvvoZnql1lFvoh\\\\\\\/4lj4N7l3j2qvpq+gEnSXqKNNS0QmNqe+lg6G686a0c9QPkhTNwVD9fiiDopnxff7cezzTRNxf2ePH\\\\\\\/6mVKQbkY1frL0dNy0hoF91W9SN1MdQmp9q5SUxF2daDWAxdfYRFp3AhhcW9dn99qXh+r\\\\\\\/\\\\\\\/UEL1O86tOlLWLYih2ijgDeWDBSLrqN9wbdTeEFN5imfiwU2SQDo65PNTWKi0puyvwpAudjGRTj\\\\\\\/Av1hOctofAPexdsZlATC\\\\\\\/MTNhb+gNWkDYOkyepckyVUbijxmvUgaZElp4nEhtAZRSCHomwxwnRPK5exAJq6fvfzfJc8qx6phz4oUG\\\\\\\/O9A\\\\\\\/lhOa3Z0kVwDSY4QwVxwTHn95EdVVPycROWekU2uALvPqIualZCcjd8Ha7u8n1CwgrmYWj9WgaM5276Wjqd1EQhAuzZwc3B3pc0\\\\\\\/QjdrWFVT5Hei1fWhugcyIiJjlzHBKUoipPuk2RdJGSXZYhC7NXt6xPZyh2dWOyWZS3yN\\\\\\\/WD0+QiK0Ty95vOk72JcD4s8MRN7ty0XjVhaQjQcG0AyNwNYYnwlSmz6EvKt4IPzNHFd3kkkBzDtfQQsjWJLlVOqBVF5VaOQM3b+0EqYpQ6Co4j36GjxfLmnCNXCCJlNcRXc2kT77Qn6x3a3Upa+oSrV6h5Rjw1EsVFCY67MQpqZsKFogyO4yYTwu9wnX4qh5BGq+HrCqJkzJxNlYi4+cBROWA1aFvGOk4gmuYHAdjkmuX48hcey7asbl8QWhD9Rl2rvTZnRqDi+zat5QmxRfm5tE8qXaM1GtdOYcuO9YfwjUevcdkz+Sgtmp4THSo2BORqDC9cu8AEDvuGieUhs3VvfuYlD6my8IY2rs9KFYziEjrRfU\\\\\\\/V86NdxMtXlIcDLwSSWi9l6AEf6kOCIzP+ClgetrAv+DfuAjqwE3g72uWUZYwj3y430gG8gCXp36v8essPCs7c4UaI2lge9By\\\\\\\/5cVHvaPpNxWJf0\\\\\\\/BcSzvx9yLKsADwo\\\\\\\/y+oG26UY0CGhp6sBsF1W0IOhJ3THcLYDK492awZyMXS8kwXksbcyE1PkD1iMehQWvYUV1oLeuThh63b0mh5+RN+nnj1+1zAl9aHMjDkpxDsvYA8zg5i2IxC7SZCWlfEh9D+mBNZl9F9vZGsgV\\\\\\\/ZHJwRRRHBbjLqUbii8f6oTlMzEWcSzmjWoGZPdxhz\\\\\\\/Gfsz3eAFspqthDKPKunVZT7qlli8EIxixRtoQnBM3njCKsQZFzSZSWB6GBHPfusBsg8vvq+qgGAF8Rrk58qG54WyoK6uOeJLdHsV1EhaeIlZAAx8gUft+oRH04VjNEGXls\\\\\\\/1HQYZrLvxJTLIQrg9yePWCYEIwAY\\\\\\\/a\\\\\\\/e5o5BR+USbmx2Zxg6g1pU2P4ivVI7+tUwDRa0b95iA7FCcBh5ZIeSTDuFjVXpelBlbv\\\\\\\/oMfrbifPoiGNqRwFYMfdBZI\\\\\\\/LxQZIelroipI8B90MM1VxdYql5JkvSk5AAzm+v20+l+AXDXxPLAZwpOle8qR7mm+aknr6XGOF5NAJ4L4dkz6EOXbzrVZjHezxa7rae8QFZsU7hl2T\\\\\\\/q6ejduxF6Fo9w\\\\\\\/Rm9hFX1\\\\\\\/jv2GdWCtLXytP23TVZHs2ridMwWwfKzzBpd\\\\\\\/HFnspiEKgzxMaWYFb+4ANb04oAtJAUNyKDHiOrxp57ArxfNEyNMiHoKBBBrd\\\\\\\/XRcT6Ci5iyGpEG8nZ6Xtq2WL2HJ8axEpplED9G0PhAo4sLnPTH\\\\\\\/i\\\\\\\/QeoxYVE6\\\\\\\/hR79qM1AFGbEqcP9EnMuWqMnBQFKDscyYBTZJzFksx\\\\\\\/wLsPZwUwjuVxXTY18ZPTK6knL8POM2sEkIRfJQem2NLBpKFeaoJZFqwSKVt5y2wjrElTPMSM7XioeL4Qnmgg7lB8fG2ldFr6V0QiLpc\\\\\\\/K16kTtNcwtVhY5fXXPhnSwgXMeudr8N2Flrw1x4OcpPZrLkRbKuLX7IlGVuVVKWs8dtWk3bMLSOU8UDOauDWyVQDyrJwkckrAmIcKeSA64FwVtKUXXUMKbqsJ7vOpW\\\\\\\/jFHaRfxKigvHt1XTKYoIbJN1sqOwTfAtxdJJSSITSzVn6J\\\\\\\/DuIBwJzwdjM57Z+xxegmEDCi6hI5PqKb9Wnui34Iy7jtzETItLBHF\\\\\\\/U+GLxXkh4sdzILRIMhRxqNdMxoDqR3RXlVT8LmXPBfH7qFsvepFdPkj0WS9noZ1QzmQVfZmAEeJ1bVIbnhA88L8BGQj\\\\\\\/bxhIgBSAxaxUTxikHeRCy\\\\\\\/fJ2CJ8atzuKAvlCp5GQTSmNM4Fjgahv3SP28H7LEzwgEcS8Sg2yU222y4Ehzp20MJSWXh8+DL2KIfR\\\\\\\/r0LvKon9IV2Nz9YO9TrZDpTIRH7yRDEwzuvayt+uDlRC6hKCQ\\\\\\\/Qi57uJ2eM8eqmoq\\\\\\\/htNBhzecSJCLeY8UQKR1ZqJbWd4U1r\\\\\\\/rlSxXejA1uGRdxVBdLkBsnDiY2xCLuc+76Gith1nffGWNMzTXpK5abx3aJ7Ik+0JAFuq2PG4iz9oRsC3M69Iix4BZo2Dr94UcxAofhMyEB6CDLDNrIY35Wdwpojy\\\\\\\/e\\\\\\\/ckzjuMAJJQa4\\\\\\\/9ab\\\\\\\/eGoSuwSUUgTJd9WCPoT41sHG9Aa69io7O+p6bBxRD\\\\\\\/cIlxZ6YOgB\\\\\\\/6kPTu6tvLZapwBoNp2b07wRb68ND13otkw\\\\\\\/D02urZRAs\\\\\\\/UHLNYYe5t6wqmiWn1d5Sc0cWvMhdIlCM2MCkID3SVxyysLVdvtslBP8Sl\\\\\\\/W38NZRPCIsq0VxZ3rZtMsQ9okvfKkLq7FjFUN0ax6yCRC9PB+Zj6Q3ITfbuNBfsjAZ\\\\\\\/Adh+\\\\\\\/bWuNRUJr2vib1YEnOfZq\\\\\\\/fZSGa8xpiv2Q4muW2rgVegWYc8i7FX8oRqlQ5Df5dhD+bjjsW3yf2Q+g174bL90zE7BCtWwvaeZLLDx+CNBdqNEDeH7mMPsZjj0BhGWJW4X6iMTpWXVCeA7iu6SjtAwWULFaOEawaSIuONJ5kuwkcrzhMuUHOZuFGC3XZ8kw0XLjKmyqzGCGCoaPsQoDFJqZI7IIdJ1TFea7xZJj96Lw\\\\\\\/0SN7gtWxBQXyJFOVl3hsiLzR8hyn0WqrOFzPRD3d0\\\\\\\/84he+0Qw13P2waBPGqiMtXyhOcQ5PeeNx\\\\\\\/xaUhNYfvhGq6\\\\\\\/0GlSuj6IM\\\\\\\/FU8kfzpduNRpd\\\\\\\/M433qmzpBp3sqYVAW\\\\\\\/sROb31UW3vyGBQCitcVzw8DBAUDbuICcz9fZcnqgdH8m5rUAsPHcaDUkyJHNjuUQNFrhIlfh49339\\\\\\\/QrLRUAc+nz1jKoM4LSKCJhjHFI7x49X0zEYZskPggfyCjoQHzFFA2ky4R07AT2hYBsLgKDS4uStqGz4PmMdrO0RJK0zJVBIGUaj2UTg4YXntsWQcIe\\\\\\\/jyicENLXtWLLWMjb8nRDntRloh7tWV1ejWW5MxrNtcDllouazbNTQ4Cg55xUdW7JBuRFsDHoiMqSX0HhA4ORVdE\\\\\\\/ghK1Kp9KGiQ0m02N9\\\\\\\/c\\\\\\\/TXgJZKMCr5khZFnexrGnkowPvBe3mDaiLRpi\\\\\\\/xNUKKbklNdrGRNRgfEJeKxXBahbGI0WHFDKwYqdnacEtKdudtoNds\\\\\\\/0KJoBKWgEMThnHrlYoqkdYVaBvP0UhsAbnQdI63k+J9e8WaDdNQAZ7pLohlN5UhzLyfGl3j1tnSMLYFj1yVaATG7aAhaDuhuODabL1eHun3Mlk5hwdShcGp5KR8dodHZ\\\\\\\/JnMafNi\\\\\\\/JKghRPJsVIH2IRAGJmrajdBKCcPw757MQNjCYCvPBjlPaDkvkHdvMNG7dOGPJd+\\\\\\\/\\\\\\\/PNMnfo9Mk9LjeAVVh51s70Cda+LpMOA+cpm3cyprp2hwpYFUyl+oc+ycjB3xK8lsIbIQXjonB4z5ptAnh0ze\\\\\\\/ZOEUCwgIsF6DC\\\\\\\/L+C0wobVKifJQg9EKnRYhUl6WZj\\\\\\\/CidTpayD\\\\\\\/Hc5vFTtzLT9nLgCYoD09+iFkFVnddtdcaiSAHjsz\\\\\\\/kQsx\\\\\\\/ouGZcGfUCxfp9kYE3jqUgXeHCMr5f3HX71mvXzNDDmVxEGZIWqfFMK+Ksbv47h0fuLCo4L50QD\\\\\\\/JTa2u7xELCZLbPEyFFG1HiFvmJKXgy+lAyKsOaVQ8EAosRKLy6\\\\\\\/B4J\\\\\\\/yxy44jCu1F7ePCqg7eCtdIcX4MmDppEBwo1Kwk9LbzZkMQN8mQSSKK860a09uJDH7QmATQQ8myUJk8kAmlr\\\\\\\/8jJnQQGZGFwkEPngEqnVKNh9ChmmXeaCKE8HWvV4nG\\\\\\\/7RlXnImWCOvVdTAM7Gme6yje1P883R+ryMFU\\\\\\\/JCyoVP2EnFPvuUjk2DP0Bu1F\\\\\\\/4QAXu7\\\\\\\/WDWZORCYklJ9kNdL7XOMDpkQcnfL3dQyxOdW0+t7qEHcaZ6RDqjumPBOQYRiWHuo+SgSzjzIQy2SXPTVNKFBAvu\\\\\\\/exD6jNz7wPBSJQhX2TesJV7GpQXI2Olm7bu4AJJBQxNZ74VAnN5f4d5cpc3MO71+9RimmYUTlts8r1rUzPjypFG\\\\\\\/dYJv5hF\\\\\\\/6CLS4JuTSutMZY1v2bd6bAYz0L+8peL+p\\\\\\\/QCFwUlrockDzdUMafgFQbviGo4S\\\\\\\/eo\\\\\\\/eYzjoeUPxIru9ifzZA\\\\\\\/oAsxzSMe1GMUXyOZGsNqM5xDqsTVG6sC520iWFXFi0Mf+UgU93PFoJSt+X86KkXJnlcSyi7OGC1S9KKPVQYX2IR40YQIjBz1QZ8Hg1j8icJiajFbLyx8GUAilugu75tLNGJlCe45+MD+Xhg3V4stpB5kq0tfp+TQrXfVUL7BShvrjX5+pYsQI9A1gA4RLcOctLF1tzT5Ub7GT2b+GsMcfPpU1IAlVnumzS9LI4E152tCDQekgd4HQ6jBV\\\\\\\/Z1SAOznXoHDe6jUmDhuFq5YggG4JPIEUC9MeweqKv21jxRGlHgsGiRJhh2kf38LG7kxKrNrB\\\\\\\/QsiSsvGgxF4iKGeeRPaA53Uucy5gxSIABnE871ISIf3TH+3XDCcA\\\\\\\/AXXMwTW97P7Z786ZL1pHpcs3nRUl2sNis7dGAqstSltTNTWrjmGq2BgGex8tubm5tZyKNTVdrQmyfg9LQ0\\\\\\\/71BiguU2nVIiHVtyz0psOg8XV0M2QwsCFE6O+m+SvoXGHyS8DM+Y7kFSP5CfKmW4E5t6C23Mv8OIPZkY0WhJhLkYjioLQ4e+451CJIXfr0Y+tIfdPdcVDRyT6qzFxgKWWfdy9Ob6PK6p5pYfN2we0zvrNxPB2YAGW8f65zA3Sg84QDjB52dXGkkyymsRfPlY3RtQN6XbhqgYk1FJ7WXsyZ\\\\\\\/avmHoteq4HHIc7N593D073VzbL6X4C9soYNWd+O2Om6105oL3Ckhw3lqvWDLiVx4C4\\\\\\\/xqkqkOaMXsNT4Epiyl9ooq0ieUsfdCR7CHPZzkGrI7L5IIrTidlRulZDjw76o+ThGNZU47Q41DyLyhVcOKk3XpcgQpgyEbRmfmmESowF\\\\\\\/FS0JoJGaaDBpSazYMTYPFnzyni2UvgPAOGQDWDihx3kUbMWb1bgcXOxlKXVE29UMCvw6Fpi88QAHUYyb7PGqaEPcOdqUxi3ivyl8mReYctrfRFo5IJceheGJqCybMhKFXn\\\\\\\/LtERs+elm3vuLt3KzrCICNctDxd62yGqbnoJDUSQEgh5hTvb3B\\\\\\\/KAovbPjQ4h1gBNlR+R+YgTWzrSU3SryhXUQBhkiV5NGLnmnXoDYuud3eN6guKGwx\\\\\\\/WbGOV9Es1el+cp4ZZoAztmI823cdUPAWfRklTd3q8ySKT23adj3k6OPtK3LhyJq8lfC7GvGKeY2SRGchEePwfkyTnDHysH94eqOKgNrWWQND1Uz\\\\\\\/CUAvDzlnQnmHAqzWgIoiv5hIVPuw720p354s3R7KMHRR5+bYj1XQ3PmF6dxoi86BvXM++woLf41LJnD8+Q8qnW+Iq6G3OEtAXDSaLRtNGosP2cfRfXepyXyNhbrDxjMPA83Fhd8+D98UoyMX173DP4nniKV16L01+KWzyZ3nt8yFBv2XQuU+n59s6qMgcVlCoUSTbCsmuzkaFGn\\\\\\\/JjWAC01lvarbq04Gm8LItqn4UCz8vLgYkRCGqkA71MGr7RhHLiS\\\\\\\/bvscrrJ8\\\\\\\/4uokA1M6FE5Tp3\\\\\\\/2LXgqUfyVEmVV6AqpuGmqcN18e2VJ6DNCna18Q0SyOoMQM85kljTiUQycjHVRYH4c4mG9Cv1lyZjsfWHMU1SR3jiJLRQGyyVHdShiKBCjOw3W9a0RjxCiGGcKGORaRU8iSxmVJpccwIDUkECOJKfqFh55QD6jxcC+SV6U0FzSrwPYh5d2xKcjd4AoGEIUZpX7l49c09K\\\\\\\/bUoYsY\\\\\\\/tOrc1Vovy79\\\\\\\/xyuit5oWEBqx5gupETgFnCEMYCJXuzXeuWFDVj\\\\\\\/qkL50G7GdHtVjlHvnLJiZ9RNzO3RftLAVosc3+gqxsr5IjF5vafl61Zr5qr6XqyrRkWhdOhg8QlUrdIBBtFMWIi6Xr9PU2UBgZzTRlVPL6lDWVEG7sb8kVLDUi079ZvCineJnXEJlmoxTnmd8kdztMSRJpZHbLvtNE88Bl49weqwuUm5iPeRE9dH+TuPfiPBx8Gd9NmT+vGZjhMjYQpOPFK5zN9n2qNd\\\\\\\/yzS8ZDd8iTVRVVvR8m8qr3EXaHEVnMp1\\\\\\\/sk1zJyTE4yIjZ9mJ9FbCSScPIMIMzbU41vgOgObrSKJFVXfEqL5SumXaLfN20\\\\\\\/rzttBxLxIHRsNjFs0EuGCHiaLENO3k6UHrZ6uzQgDocrAknhCO3uA0mai38EbZ7542Q32LYU2N7rrZpFRQ3brRO50vqV5MgRvR+2T0YuexsapeQa7bluI\\\\\\\/L5ur2xoRiwLgwT81stALZXoh1snXAijcoDkMWV0\\\\\\\/v4b12qONP7SkgFpo7+oI4kTxrxlq1h9i0q8WKWgqmQEae4oUSIz5tDc4yA\\\\\\\/qZ52E2aGxz5OwUkQIQktaWZ6N5ieGsZkJIBezh+cXf4tIl0hy9Oyr6x4p5KdhkiTdBvgJseqCi+n70EQ5xGqGS4GOk3JcqsqMl7jcAKCScx+D5VJ0lPLADRL3TJ0JmVwjJwWL1EOXRVeRrmZ4a7kvFj5fil0eUzdpOmtBXIwU97uvBmQGwo\\\\\\\/d6TcFme9IquAl+FvCDL9NJEva7vO3NlayG0+1tE4YDkUyvQpib5WHo4v68QbShNDbjcI1jo0lfDT2BE19WRzk9682yaPeCN0Y4NUaL7CuNtDfG1AYW5jdXMK5H70cwpC9bFV0OIpvNUzQC11PuPgWdZxsKxpun8XeBNmcrDXgYRA9QC9I43dGVbNEfKS9gliXiYY794ovHvr5IIZC5NW0iYXhj7lRVo0h+T5cOyO13goj8h5HUtEq37lz0EV8mbsss3clxs3\\\\\\\/TowyJG9fgKaXQDj7HoQ+sERMo9l\\\\\\\/bXUtXvVDLcY8p2YzTnPOVTuz\\\\\\\/Mo9Qp1Z7TcC0k9WsA6RRT6im6rlIHwsdGieuDt5DOz8GvR+rsIRIKu61TE4NpstDZ1SqTR3wzD3Vo3PoZdEoNf336k8r\\\\\\\/XpLxzsY0CdjSES52NJU9Cmr\\\\\\\/UgwRjRHa6maEGjwedhv8SVmtvAg7cF4vClaAnXiIrF2RKXDPYQy4iGgcydo4Bn\\\\\\\/2Eo3kduiN18\\\\\\\/4OSsT0qAa\\\\\\\/NByEM8nvocqkSoa21EdD7PBSrX\\\\\\\/kXKkkQaEmkYwGduxWW9KxAjErv\\\\\\\/GeEq8DJJEyN56zSpzS4zGA7p2\\\\\\\/BMpprmou9\\\\\\\/qW\\\\\\\/\\\\\\\/fjgm4WYU1NKTW+3yZmr8TMR5NQbrzAS2RqrVKOvZYn\\\\\\\/GBVgvnb3tSBnREL5eukWtjH\\\\\\\/c1Ofc+vQM9aHUuxqiagIOhy0b3hmx5uTzPW\\\\\\\/lCM+D+TtwCdN0xMJCLMO339QMfRm3k1roE5pkGsZyyFPvVuF8bYGsh8OR74t3q3GW+rzY1X+VHQ3dITqgr2Z7g47k3\\\\\\\/UIeGIbjwvygW\\\\\\\/Bmeb8uiGcq7tosmQeK6UIqNsDNEvIKHYcr+lH5JMmjDyDCj4O\\\\\\\/3\\\\\\\/TMVBLdAR\\\\\\\/ih9LtT3oqy1aCkLUYQkPIOJsbzJcRKEpKc3TtvBlnz0sBOK9E+P1asev41nv05U7rOKImz7aFDyXHUepMh\\\\\\\/1f5THxUuHULNeJgJ0bvG\\\\\\\/DAQEcakze5F7aF9zA37J2V9CrP8OFy5G\\\\\\\/Duy534CN8GX2moZnhzDiPe65QheO7kaqTGmRetzIQKyjFnX7Rbo5ZsizzYH23SVxqNbY0sNSru\\\\\\\/2AZ59o7cTgGZRi9CgjAHHkti0dMTRi2kBW0c\\\\\\\/3XRUWsoYaPyMpiEu4ZU9UODVxz96OhWMtzbvVbTfbiOecBU8xPoePTh1IG6u1PGA+AcuAK0DSMqB7Jc4FhZBicASWagRb8TnrpEUCLVBhpgqJAtFH5veuKgvKbd7ZB3HzlZkbH7EMCa5KWq38JRNLr\\\\\\\/tBK43soQARLeEkn\\\\\\\/se+aJ3UVb9B\\\\\\\/b1ZRyZBFofayIr7q0CN9DNh6EekX5B2\\\\\\\/4QEa1Pcsw1UIn42gL5SSAFMznpB0i3hgBG0MCH3DJEdgfj7HttjsEfMO2g\\\\\\\/wAC+bExSgIWXRo7XaLBXalmomS4slvNZfRG7ggjGeQCNh8Bzmkt8OVus1uNFHOn+dTPgFEZsuCk7MwCkAQYNM\\\\\\\/Btn8WCutEpuKeYwbTzZp0qmJJUVo8FXK0f5kYg\\\\\\\/uBqTNwDwhN4zs17MYB6oGPtceGYad\\\\\\\/I10ZxNVKr5ihfm+UWQVMQvZeV3xCdVAKcOP1EVrZGYBEMv48s100SOaTHXpvaOeN\\\\\\\/IrqEq\\\\\\\/bZo+JVlvOsFzkAUvQLkpifqVMBbITdPD5Uavh7LPIJwt14j4nFmzH7ecUQK2iXzcqUbm4QBcoogoheC\\\\\\\/O70OtYGtjQNg4SRM7eR4BX6oJgkRCCcKT+7zccnEpVz9+PbLF+lUxLxjawgcUYsqwtBN0pkdnvVlvsmQ6jOXDDlcZ4+Gegonn10YQwDk53L3bUH0VVOalOOcEBA2WPBCy98wWAC1GGoQUmnUSuuIFYd0rjXEZ9QsvZ6956oOxGRWqjFY\\\\\\\/T52o7wJSONS68TYy6cqh1byJSnX1VhjNTlSPJ+qObxx8sutvNQ9\\\\\\\/mE+o9ek2Y9jZBDHSMwe1YndXkOlFxcJvsD+OxggAxVx\\\\\\\/1iJuYn3FzdpGrr6bN1g90nDfeLjJvTffxlr6dkvRi7qG1Y2vAaaD3Y2usI1Q\\\\\\\/gqRUBBJkpEzXlKooNYgzcL1TXWuYJsEpipCjHskxpopz9ryPG1QafCaEILXm8rW9uortJz\\\\\\\/E0OSUOT5bIyBCJwEVT57PXWMpYubM9BHdNQwAlzJ+5wLYtqi1fa1NpJirWnSARsTuTeVv2xYaTS4\\\\\\\/B8LIWVJ0bvmTL4q9+c3oHgqSpftrElZOcfnW5tnUhCwRwb0ZBUH5FCjmgQpwLXIhT0xH3zZF3pgR1yzaBplRarJX7V8LFBJ227RSWHTmAZxoWoqfXTT+ZXQ4JqBL53hw7cn+qnB\\\\\\\/mDTWUB2UWl18zkmR0j3q8e2xvg9AVNhq2+MHyUSNzaHfjn9ik49LEU9ZyW921WTvypuyiyBzCyNzdkDvX4EOgnzV9wKndT1UQlle6Daq4qgwC7YqrW+BL0pEB\\\\\\\/8xwii8Yvvx4Te4oEvMt3+e+iJy\\\\\\\/VD7KAA6NVBxX17XNEyL9CXRLxoWTN4992U6EarIczFouGjsqYiz0dzFdXRzMAZ6f+3CvHyZOFq2tE0bJ0Nz40Tt9PYaNKo\\\\\\\/qLFYtBFyayUVcL1BvKTDrZksOUCpiuciMq0EKyfkr\\\\\\\/GWedrcjBAc1NgtBWahX4WFZbAy+xxVAEMsEGowvk46zEx4+PwOxe4DDg\\\\\\\/0reZpeW9iazsfTXQD2D5M6i2r6ef2NYFTs3\\\\\\\/DQwYsEkPfqJW1FjlCOMHlrgZKl\\\\\\\/2l2q3ry5CDmEdjrZi7s1MGXqnagEe\\\\\\\/dhmsU1IPHJpxTmpXPAVX2eAPVkeuVxMqhcKCcMuKNE4S\\\\\\\/mFYMVwpYH1LoiS4x7biKC5PSLnfjkW\\\\\\\/lA\\\\\\\/dIGLOmbzQ7g+POsMifVh\\\\\\\/ztKXeoKYT16h\\\\\\\/X0sIDrERFSIPuuAQ2XhHXqbTlhOys+\\\\\\\/aFCrkrkJ\\\\\\\/ofP3Ap4e3o7p5KGFGe9qPzpMCUCMSnmbAM16MUTnst5A614N8DVG3TYicwLQI1We4wnW0ymUSIqXvng2SaWx2CcesSGt40uwF+sFGoxj3GSTEFg9ZobJx3KthHzyqoJvOEb54FZkywGhCB2M0uZwlKlmmvHV5DZg\\\\\\\/jp8hDYeuPJbv0sfE\\\\\\\/b+yRo1kkjqFI5O7rDjbF71+Tam0SMyKLKTtwZQMFqkPcDfC6rzho2PkgP0ftYkUzLjbFfG2449My\\\\\\\/0qiUZ0lUTSH5CtJKyhfIIYdgTkDHWKXFtrvNh9wbB6okWmGZXhEj2JnwG\\\\\\\/L+u7Zqlk3U9cubGSevnEjTNbXe+TmUuzip17vJJRSL0s4U+0wQK47H\\\\\\\/lkxtVT\\\\\\\/fsftmuspdyVTfYYijfB0C38NXrxl4rDGv2xR0+LwuleNy6hnzWaj3t1g2XVeYtZpwIrDr+f2SDqrTC1ZOW5AhEckaNjtxNMA00nsHHzXHVoUfzDJZhcl\\\\\\\/LXPD2zBYLoVlm7+TBcqG9huWMyXwMzRZcsjsHzuivmN4EgJSclY9sMjoSk79vXrZHFLGbuGmuxM3Q\\\\\\\/21NIJoWu5cCsQOcmRbiMsky3SriexmBsMeTpTLiplj28yOaHpvCraXym6nkyzK5coAAc4WFQM65hX+J+gUmU81vsUeqLALmC\\\\\\\/LUSU2teSPYBgFmb+yyTE1zNGBl509oFh51YtgLsPRGVVN7G88VuH7RMuMjYRa87K774Gj6pSxHFch81lIJfKCVUAmSpWEtYVpN25thOyRHw0bG0XxTUlz3vxwiYU2RL1HjFKJr+da09922I50PTuJcP7sG84D6QqwECV+EULGGvNiUkpX\\\\\\\/vubZV3fP3KuwHugQd3G3O1ZQQ2Xz0QmKml1Mfeme2PJqZNgHbVvJsvpNS+BjYJfv4jqFAITowHLafBmy2asMM6GO1ih521TbBcCyo\\\\\\\/qeBWV517HDTEVj5ARDAjo0wpO4i+11ttPZY9HUVrvEqN2YMRkpPsksUn+8NFcbF52sU4BG\\\\\\\/O\\\\\\\/YydNx5eXf8+94wYABJ19boTSp36XLQFcE9XsYhzEqc2aouRJipLxqbf1gk0FU8GMQDEjAT3iDnBqBXz2BYElPdz6pEIqZp9uTPUwbAx32gnTj7slQME409yJWR3vfBdtaToJhScXfEdVAib92VgInyW5TRMfZOkBxL4uwy6NEYrj2iJDM+8bp9cYYzZemIEoa10+5YP3STr26dEP4S1A16fTe8XYpHz5vrIPUXoU9vVkPrIf5yHp8L9WF0RSK2ZkN9xQ2bI7N8PIkckB3tObm9v4dZkwKX8zkVCh6qSItivz2AVk5Q+edgxo2umw9bi4h0+wq9v+4GJ3\\\\\\\/WS9yob4mbL27R5YwYO2UxmedabFvDuYg983QDn0WHaHK5Ot58806dGqsJfxFuv2JNCCER7WE1Rnrt7g55V8q3SP7v57HGqlSdDfWCACRg8KImf8IEKdNg+1l\\\\\\\/9T85MmsZ+3obizcjnIjoseVf5nqbTec+qQDDbqk2+NDCurGxYeNiV\\\\\\\/dgx5jilF\\\\\\\/axFCmfKalvrrpF7OjKDYh9IOgIl7wNq6gTFnXIKoUW6kzl5EyGTPCRWD4oLqVRbj4ms1hJZLWGlvJFlyLRVOPtlsNY6Ehg2lcZ1ZzwfFUdtptlRgCl2YZG6IRpEl0nSyFLwHtGM7i\\\\\\\/7kKXkQH0PwU2KV6XqlK3L3\\\\\\\/w9g+2aouWbFYPF6X2bLLKCFmMcPwiu7OetQ7W74YisT\\\\\\\/nqzkf75\\\\\\\/C0uPg9DRehZlcC7yT8NdN3uVsHwD0MBumrt2y1nsgjOVJgVkgqnCO86\\\\\\\/JLeBiivqXc6SP8WwcQMd+M+ZA4eMlrenFHWeTriBWcx4s9yFgTNKbLHLnFcK7SpyvNeaOS5w4Y9124oysMNqXs0ih8BrisACp8gCjMZA7AdvVB0rHB9tC0n0yghmKEzUfFz3DjGomSMLoN2gSUvMsxlleVrCsg4VP1kccEYJXkEbU7L\\\\\\\/ChTVbdbLWxXg8wOhl5kjZvPmclhU2HRaGFRncP6hTkyjiOv2LY+3vqGin5WzCYLwWF1g3F3SmH0oTMq6X\\\\\\\/Wpzt5GdaibC9kQ2HHxspZ1K9\\\\\\\/K0VKNLpSOXG\\\\\\\/XKLq1Y3DamMEZPJ6RB5gAtzC0KF6d1GBimGtRsC1szuwszgw1Wms\\\\\\\/vdY+wAZH5ZDm95FT0WRqdJC5Bevn4TLY5QmPrZQ8eVZ32gYxKAEuiCuLHtlZBPqV5O2dcFeys8YMrnkPDCBKr96DI+buAyE1T8GLcsmhaacRgZzUwZA1u8gvqnxwDwpfagG+Kk5IP9RyGER2Vl6YqB4fNWQgEID9UCknqNM6in2xPlL8EY8It5zRIdzR2y6W8LLrm84ah0bVhELAHZjTSA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022aK4f7sbc2eo7k4LQfpIUSw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225cf5bfb31fd8e4e1bfa75455d3a5acef4a3d6253d9ff5a3a64929eeb9e43e972\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022N3LC\\\\\\\/Tn6rnXoYsxkDIEiBXsLuto249UUy2oLISbmQzly74uJa7lHZ3x4q4Ia6CkH94rzOJoa4jCPxfh4\\\\\\\/9ft74OSC1lVisPyU\\\\\\\/gtaQL6JzNrpqUWF+XbmIFUS11n7KcPVJKKIOT\\\\\\\/xKcGCKUJZ\\\\\\\/Xjhnm9vVk7SHAfRMZXovkk4fFfFrUz\\\\\\\/42tW7ABTj\\\\\\\/cROA\\\\\\\/f3pKm8TqdnNOWd3Nk6u1EBf0LbEye1Xoakm\\\\\\\/3QUhwmn\\\\\\\/rttV8+JXQxiA80TEAXNFD4xC+f0WmpRAN67XuSDfioeNqbVtkycEHNxch0ICq+ldlboy9YFNGIUPo66nQQedRLvMzZ6YCdGk+jkaxKWZKOgzobWBkaHuxVTlT2FhWKuLq2R\\\\\\\/1Ma\\\\\\\/tdPQJfR7ZDfoqWtwAFCAAs9Hmbwf1O51lFtdFm77ysKk\\\\\\\/uiROLriZOVXyFV5SForvQvXZDordxS0BkbSZ1mIY6PJV2Dx9wvZI4hnZpGaRScDJiGZhW\\\\\\\/3+P3vl+GY1HKhFlql7dH5usHA8Ro7l5NqKO1dn9hlQXiPMNyMCahK3zyXcwkHBNSGJuQrYTrUIfhwQ26xvhfiGO9cAEyXeRip4pU6woY32PHwF4tj14cEiqA1cmWNEvlqNRKI4ge3scBQeikdyrtM8ldiDGg2zYPrSXpGDHsVTO34xfL0tONlEE7GC1Ju\\\\\\\/iwzyY+CGo07A7T4\\\\\\\/PU2RL3aAJwF66djDtr7R30pK91lq1VJ8S354\\\\\\\/JHEOwXJmssmUNE6AU9CKuQZeukSZk5RTyq71+F3IB6SnSONX1OiKw1Q6i7NxEFiLBUmnFGoXsPUgIDULr7ZIL15Q3k7trdil483iJTXEwo3RiPTPml6V+WGs4JGvhS8IIiQt7mt7lOUSH2HAaEwM+T11LIBKZI3Ev+4I9NTY\\\\\\\/KLMBuhPU8g+OxCLfllSC6Pb8gstl39XXUMuBXLP3z9c3l62lp32I4qLliih1CveXL5Zr5w\\\\\\\/OWB6W7KwEZ9Kgi+cnivUestAskOL3NqDFpqeKcjDh230asv9ttCprLmud\\\\\\\/9EIGwhjYj1v95+ad9QFzWh4vel3ogVn9LAEEuLDWMKJMxUKpUXZrAuqzv7BCXjNKXJPvn6JFRsXCQiaIv4TjYy++v77FpTuH53EiNCxcJcp5nKG35jS9RLK485PBQ4znaAErJtpPwCvXUANHKCK9T9+8zerWYXjrp2pP0OvjZUmMteiJy9YWI7izQlqNwAG0s8MFxp+4JG3Mkh+6zfPHnVVhZ1JuNcCJBrUqzpLhGN4Fd+DbKINa4R12w7n2EFPUgmWiOz6DKWT9ApwVRyBqtnu6bour0eDQOC8X4UNP8zPaK6t9hOLRznkT8Noy\\\\\\\/FYEWX0fhlv+uuWhYAB3Wz14D45r10DoenQWJ17nx0mPTXYGtNsyNWmB3tA+lKTuxcsY70yfNR4Pe01PxBhovFJLv8YK7FzErGOH9Fvxuy2Mt8e4s7Ja\\\\\\\/ClAIEkO0hzJnqKr8YYhJ857CIi+t5DGUn4o2Nile14PeehAdwzAhDWKVrDdkJIr4nP7XfTAjdSxbGFM8fMM4mrQejF8ekZhnwis2vQO0g30HCE\\\\\\\/TBxiHPqGP3pP853iwAHydIAPqvmtMKzECXC\\\\\\\/bjTMFUu6gIbo13NibCqjn6zoxgW2cHWIscD7KO7YT7W0eFoQ3IJEvQAu8mGFDB7qITsKNEctjbQQLSab4yS+Ay0pJ7eh5doKx+1Rp4mY6BukQMk93cv3O1qLphZ0z0+DCe4BbXENbwe0u7sa1hBQ+XU+XaB3L2jYdfJkBpOdZvMWKpfOv8lATdp3umHGS40aEdVoRo4G39n4a+CiR1adaQa2zwVT2eFVaRodFJOgxwbxwwOYZcbj0CV\\\\\\\/mXnJBtOhJKL+ecwjPilouDtm6x4v6RalQJp3iVax3v1wiBUAtTdAjSAq8CJN8SB0AnKYnQWRcd0GZmWXQRzhZ94dh2IrGkFPjZ2lwgzhHeuLVM8czG8Y7Q53fHAT3B0kzDwBP6GQdR\\\\\\\/ryjlVwmBKWg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022jDEQFDBz7NaLcMOHRdUsGg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022727765d66cb166d2b2cf9e07bf8043954eaa1738dad0ea13c03b9b0ac782d696\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460434 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2051" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZEi1x4BouF-RpDNIqSxqI1l_eGBmB6IYoE7lrxfU42c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2051" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FLrZwToBoUYv_YLTOHBpE9K_dMkbhO_C0Tikdttg_nU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/2043" title="Exhaustive Genome-Wide Search for SNP-SNP Interactions Across 10 Human Diseases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/2063" title="Fitness Effects of Spontaneous Mutations in Picoeukaryotic Marine Green Algae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460434" id="node460434" data-pisa="ggg;6/7/2051" data-pisa-master="ggg;g3.116.030601" data-apath="/ggg/6/7/2051.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CRISPR/Cas9-Mediated Insertion of loxP Sites in the Mouse <em>Dock7</em> Gene Provides an Effective Alternative to Use of Targeted Embryonic Stem Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kathleen A. Bishop</span>, <span class="highwire-citation-author" data-delta="1">Anne Harrington</span>, <span class="highwire-citation-author" data-delta="2">Evguenia Kouranova</span>, <span class="highwire-citation-author" data-delta="3">Edward J. Weinstein</span>, <span class="highwire-citation-author" data-delta="4">Clifford J. Rosen</span>, <span class="highwire-citation-author" data-delta="5">Xiaoxia Cui</span> and <span class="highwire-citation-author" data-delta="6">Lucy Liaw</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2051-2061;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030601 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460434" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kathleen A. Bishop </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BA.%2BBishop%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bishop%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathleen%2BA.%2BBishop%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne Harrington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BHarrington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harrington%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BHarrington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Evguenia Kouranova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Horizon Discovery, St. Louis, Missouri 63146</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evguenia%2BKouranova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kouranova%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvguenia%2BKouranova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Edward J. Weinstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Horizon Discovery, St. Louis, Missouri 63146</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BJ.%2BWeinstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinstein%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BJ.%2BWeinstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clifford J. Rosen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clifford%2BJ.%2BRosen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosen%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClifford%2BJ.%2BRosen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xiaoxia Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Horizon Discovery, St. Louis, Missouri 63146</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoxia%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoxia%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lucy Liaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BLiaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liaw%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucy%2BLiaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liawl@mmc.org" class="" data-icon-position="" data-hide-link-title="0">liawl@mmc.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/7/2051" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460434" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460434/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2051.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460434" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460434/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2051.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460434" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460434/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2051.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460434" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460434/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/7/2051.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460434" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460434/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Targeted gene mutation in the mouse is a primary strategy to understand gene function and relation to phenotype. The Knockout Mouse Project (KOMP) had an initial goal to develop a public resource of mouse embryonic stem (ES) cell clones that carry null mutations in all genes. Indeed, many useful novel mouse models have been generated from publically accessible targeted mouse ES cell lines. However, there are limitations, including incorrect targeting or cassette structure, and difficulties with germline transmission of the allele from chimeric mice. In our experience, using a small sample of targeted ES cell clones, we were successful ∼50% of the time in generating germline transmission of a correctly targeted allele. With the advent of CRISPR/Cas9 as a mouse genome modification tool, we assessed the efficiency of creating a conditional targeted allele in one gene, <em>dedicator of cytokinesis 7</em> (<em>Dock7</em>), for which we were unsuccessful in generating a null allele using a KOMP targeted ES cell clone. The strategy was to insert loxP sites to flank either exons 3 and 4, or exons 3 through 7. By coinjecting Cas9 mRNA, validated sgRNAs, and oligonucleotide donors into fertilized eggs from C57BL/6J mice, we obtained a variety of alleles, including mice homozygous for the null alleles mediated by nonhomologous end joining, alleles with one of the two desired loxP sites, and correctly targeted alleles with both loxP sites. We also found frequent mutations in the inserted loxP sequence, which is partly attributable to the heterogeneity in the original oligonucleotide preparation.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/dock7" class="hw-term hw-article-keyword hw-article-keyword-dock7" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Dock7</em></a></li><li class="kwd"><a href="/keyword/crisprcas9" class="hw-term hw-article-keyword hw-article-keyword-crispr-cas9" rel="nofollow">CRISPR/Cas9</a></li><li class="kwd"><a href="/keyword/komp-es-cells" class="hw-term hw-article-keyword hw-article-keyword-komp-es-cells" rel="nofollow">KOMP ES cells</a></li></ul><p id="p-3">The International Knockout Mouse Consortium (IKMC), and participating organizations, are using both targeted deletion and knockout-first allele strategies to generate embryonic stem (ES) cell lines with both knockout and conditional knockout potential (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Friedel <em>et al.</em> 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Lloyd 2011</a>). To date, investigators have had success with germline transmission of the targeted clones, providing valuable tools to the research community (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Coleman <em>et al.</em> 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cotton <em>et al.</em> 2015</a>). Since initial offerings of targeted ES cell clones, the KOMP project has moved forward to the production, gene expression analysis (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">West <em>et al.</em> 2015</a>), and phenotyping of thousands of mouse strains (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ring <em>et al.</em> 2015</a>), and the corresponding database management system to allow data access for researchers (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ringwald <em>et al.</em> 2011</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Koscielny <em>et al.</em> 2014</a>). Although the majority of the mutant alleles that have generated mouse strains are correctly targeted alleles, thorough molecular characterization has identified limitations that users need to be of aware of in characterizing their own particular lines of interest. Mouse strains generated from the EUCOMM/KOMP-CSD collection have previously been subjected to a rigorous quality control screen to confirm correct cassette targeting and potential for conditional deletion. Analyzing 731 mouse strains from this resource, 86% of resultant mice had the correct targeting and cassette structure, and 97% of these retained sequences required for conditional inactivation of the allele (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ryder <em>et al.</em> 2013</a>). The unexpected events that precluded generation of a successful allele included incorrect targeting, deletions in the 5′ end of the targeting cassette, variability of success depending on the ES cell clone initially used for targeting, and evidence for mixed populations of ES cells in the targeted “clone” (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Ryder <em>et al.</em> 2013</a>). As resource users, it is also important to consider that there are additional limitations that may arise by problematic chimera generation and/or germline transmission (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Pacholczyk <em>et al.</em> 2008</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Coleman <em>et al.</em> 2015</a>). While the study of <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Ryder <em>et al.</em> (2013</a>) characterized alleles that had been transmitted successfully, aggregate germline transmission rates appear to be around 50% [KOMP repository germline transmission rates and reference (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cotton <em>et al.</em> 2015</a>)].</p><p id="p-4">With its practical application and successes with modification of the mouse genome, CRISPR/Cas9 technology is an alternative for efficient conditional gene targeting (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Yang <em>et al.</em> 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cong <em>et al.</em> 2013</a>). The Jaenisch lab demonstrated a 20% targeting efficiency of introducing one loxP site into two separate genes (Tet1 and Tet2), and ∼16% efficiency in targeting two loxP sites into one allele of the <em>Mecp2</em> gene using two targeting small guide RNAs (sgRNA), and two loxP-containing oligonucleotides (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Yang <em>et al.</em> 2013</a>). An alternate strategy was used to introduce two loxP sites into the <em>Ispd</em> locus using a single DNA template containing the floxed exon with flanking homologous arms (1.9 kb each) using paired Cas9 nickase and two sgRNAs (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Lee and Lloyd 2014</a>), although the efficiency was relatively low (8% of live pups born). To generate a conditional <em>dedicator of cytokinesis 7</em> (<em>Dock7</em>) allele, we targeted sgRNAs to insert loxP sites to flank either exons 3 and 4, or exons 3 and 7. Using this strategy, we generated a variety of alleles, including a correctly targeted, novel, conditional null allele of the <em>Dock7</em>gene.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Reagents</h3><p id="p-5">General chemicals and reagents were purchased from Sigma-Aldrich (St. Louis, MO) and ThermoFisher Scientific (Waltham, MA). HEPES-buffered DMEM (12430) and TOPO TA cloning kit (K461020) were purchased from ThermoFisher Scientific (Waltham, MA). Giemsa stain (GS500), Jumpstart Taq ReadyMix (P2893), and mineral oil (M8410) were obtained from Sigma-Aldrich (St. Louis, MO). Fetal bovine serum (35-074-CV), and trypsin (25-052) were purchased from Corning Life Sciences (Corning, NY). The hiScribeTM T7 quick high yield RNA synthesis kit (E2050S), Phusion high fidelity polymerase (M0530S), and proteinase K (P8102) were obtained from New England Biolabs (Ipswich, MA). QIAquick PCR purification kits (28104), QIAquick gel extraction kits (28704), and QIAprep spin miniprep kits (27106) were purchased from Qiagen (Germantown, MD). EmbryoMax M2 medium (MR015P5D), ESGRO leukemia inhibitory factor (ESG1107), and colcemid (234109) were obtained from EMD Millipore (Billerica, MA). Chorionic gonadotropin and pregnant mare serum were purchased from the National Hormone and Peptide Program (NHPP), Harbor-UCLA Medical Center (Torrance, CA). Penicillin/streptomycin (SV30010) from Hyclone (Logan, UT), MessageMAX T7 ARCA-capped message transcription kit (C-MMA60710) from Cellscript (Madison, WI), EMEM media (ATCC-30-2003) from ATCC (Manassas, VA), SF Cell Line 96-well Nucleofector Kit (V4SC-2096) from Lonza (Basel, Switzerland), and the QuickExtract DNA extraction solution (QE09050) from Epicentre (Madison, WI) were obtained. The Surveyor mutation assay kit (S100) and ultramer oligonucleotides were purchased from Integrated DNA Technologies (Coralville, IA). AccuStart II PCR SuperMix (95137) was purchased from Quanta Biosciences (Gaithersburg, MD), MasterTaq (2200230) was obtained from 5 PRIME (Gaithersburg, MD), and Terra PCR direct polymerase mix (639270) was purchased from Clontech (Mountain View, CA).</p></div><div id="sec-3" class="subsection"><h3>Mouse strains</h3><p id="p-6">C57BL/6J and B6.Cg-Tg(Sox2-cre)1Amc/J (Sox2-Cre, stock 8454) mice were purchased from the Jackson Laboratory (Bar Harbor, ME); C57BL/6NCrl (C57BL/6N) mice were purchased from Charles River (Wilmington, MA), and Swiss Webster mice were purchased from Taconic (Taconic, Hudson, NY). Mice were housed in the barrier, AAALAC-accredited animal facility at Maine Medical Center Research Institute. All studies were reviewed and approved by the Institutional Animal Care and Use Committee of Maine Medical Center, and followed National Institutes of Health (NIH) guidelines for the Care and Use of Laboratory Animals.</p></div><div id="sec-4" class="subsection"><h3>Culture of ES cell clones</h3><p id="p-7">Targeted ES cell lines available via KOMP were purchased and cultured according to the protocol provided (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Pettitt <em>et al.</em> 2009</a>; <a href="https://www.komp.org/protocols.php">https://www.komp.org/protocols.php</a>). Briefly, ES cells were cultured in HEPES-buffered DMEM, 5% FBS, and 1% penicillin/streptomycin, and grown on a feeder layer of mitotically inactivated mouse embryonic fibroblasts with leukemia inhibitory factor at 37°, 5% CO<sub>2</sub>. For subculture, ES cell colonies were treated with 0.25% trypsin into a single cell suspension and replated onto a fresh plate with a mouse embryonic fibroblast feeder layer.</p></div><div id="sec-5" class="subsection"><h3>Clonal expansion of ES cell lines and chromosome counts</h3><p id="p-8">Individual ES cell colonies were picked, trypsinized, and propagated for DNA collection and cryopreservation. Chromosome counts were performed by treating exponentially growing cultures with 0.02 mg/ml colcemid in fresh growth medium for 1 hr prior to trypsinization. Cells were pelleted and treated with 0.56% KCl for 6 min, then pelleted, and fixed with 3:1 methanol:acetic acid. After two additional washes with fixative, cells were dropped onto glass slides and stained with Giemsa. Chromosomes were visualized at 1000 × magnification with light microscopy, photographed, and total chromosomes were counted for a minimum of 10 spreads per slide.</p></div><div id="sec-6" class="subsection"><h3>Microinjection of ES cells</h3><p id="p-9">Three-wk-old C57BL/6N female mice were injected with 5 IU/mouse of pregnant mare serum followed 48 hr later with 5 IU/mouse of human chorionic gonadotropin, and mated to C57BL/6N males. At 3.5 d post coitum (dpc), the females were killed and the blastocysts were flushed from the uterine horn with microinjection medium (5% FBS added to HEPES-buffered D-MEM). Blastocysts were kept warm at 37° until injection, when both blastocysts and a single cell suspension of ES cells were placed in 50 μl droplets of microinjection medium covered in mineral oil for injection, similar to the procedure described in <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Behringer <em>et al.</em> (2014</a>). The blastocysts were injected using Narishige oil hydraulic micromanipulators using VacuTip holding capillaries (930-00-101-5), and beveled TransferTips (930-00-104-0) purchased from Eppendorf (Hamburg, Germany). ES cells were aspirated into a TransferTip backfilled with microinjection medium. Each blastocyst received 10–12 ES cells. After injection, blastocysts were incubated at 37°, 5% CO<sub>2</sub> in injection medium until implantation the same afternoon. Finally, 12–16 injected blastocysts were divided equally between two uterine horns of a 2.5 dpc pseudopregnant Swiss Webster female. Injections of morulae followed a similar protocol. Briefly, morulae were collected at 2.5 dpc, injected with 10–12 ES cells, and 12–16 morulae were divided equally between two uterine horns of a 0.5 dpc Swiss Webster female. For ES cell lines derived from the JM8A3 parental line, chimerism was detected based on agouti coat color.</p></div><div id="sec-7" class="subsection"><h3>CRISPR/Cas9 reagents</h3><p id="p-10">Cas9 mRNA was transcribed <em>in vitro</em> using a MessageMax T7 kit and linearized plasmid carrying a T7 promoter and a human codon-optimized Cas9 open reading frame as a template (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Mali <em>et al.</em> 2013</a>). The templates used for <em>in vitro</em> transcription of small guide RNAs (sgRNAs) were assembled by PCR, amplifying two overlapping DNA oligonucleotides, containing a T7 promoter, 20-bp spacer sequence, and common backbone primer. The PCR product was then purified using the QIAQuick PCR purification kit and used as a template to <em>in vitro</em> transcribe sgRNAs with a HiScribe T7 quick high yield RNA synthesis kit. Following <em>in vitro</em> transcription, RNA was purified by ethanol precipitation with 1/10 volume of 3 M sodium acetate. Sequences of sgRNA recognition sites and oligonucleotides used for loxP insertion are listed in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS3.pdf">Table S3</a>. Cas9 mRNA, sgRNAs, and oligonucleotides were mixed immediately prior to injection in TE buffer, and centrifuged for 10 min at 14,000 rpm.</p></div><div id="sec-8" class="subsection"><h3>Validation of CRISPR/Cas9 sgRNAs</h3><p id="p-11">sgRNAs were validated in mouse Neuro2a cells stably expressing Cas9 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Kouranova <em>et al.</em> 2016</a>). The cells were maintained in EMEM medium supplemented with 10% FBS and 1% penicillin/streptomycin at 37° with 5% CO<sub>2</sub>. All cell transfections were performed with a Nucleofector (Lonza) according to the manufacturer’s 96-well shuttle protocol for respective cell lines. After trypsinization, cells were counted, pelleted and washed twice in Hanks balanced salt solution to minimize nuclease carryover from the growth medium. Specifically, for the mouse Neuro-2a cell line, the solution SF solution (SF Cell Line 96-well Nucleofector Kit) and program 96-DS-137 were used to transfect 2–4 μg sgRNA into 200,000 cells per reaction; 1 μg of a GFP plasmid was transfected for each condition as a control. Transfected cells were collected at various time points post transfection, added into 80 µl of QuickExtract DNA extraction solution, and incubated at 65° for 15 min, and 98° for 3 min to release nucleic acids. Target regions were PCR-amplified using the primers listed for genotyping at insertion sites of loxP4, loxP5, and loxP6 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS4.pdf">Table S4</a>), and analyzed using the SURVEYOR mutation assay, in which CRISPR cleavage-mediated modifications via nonhomologous end joining (NHEJ) at a target site result in cleavage of the PCR amplicon of the target region into fragments of predicted sizes. The mixtures were resolved on a 10% polyacrylamide TBE gel Validation of the sgRNAs used in this study are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/FigureS1.eps">Figure S1</a>.</p></div><div id="sec-9" class="subsection"><h3>SURVEYOR mutation assay</h3><p id="p-12">Two Taq polymerase mixes and respective conditions were used interchangeably: AccuStart II PCR SuperMix and JumpStart. The corresponding programs are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS2.pdf">Table S2</a>; 10 μl of the above PCR reactions were incubated under the following program: 95°, 10 min, 95°–85°, at –2°/sec, 85°–25° at –0.1°/sec. According to the protocol for the SURVEYOR mutation detect kit, 1 μl each of nuclease S (Cel-I) and enhancer were added to digest the above reaction at 42° for 20 min.</p></div><div id="sec-10" class="subsection"><h3>Microinjection of CRISPR/Cas9 reagents</h3><p id="p-13">Three-wk-old C57BL/6J female mice were injected with 5 IU/mouse of pregnant mare serum, followed 48 hr later with 5 IU/mouse of human chorionic gonadotropin. The females were then mated to C57BL/6J males. Fertilized oocytes were collected at 0.5 dpc in microinjection medium (5% FBS added to HEPES-buffered D-MEM). Fertilized oocytes were place on a depression slide in 30 µl EmbryoMax M2 medium covered in mineral oil, similar to what was previously described (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Behringer <em>et al.</em> 2014</a>). Using the Eppendorf Transjector 5246, with in-house pulled glass capillaries, fertilized oocytes were injected into either the pronuclei or cytoplasm with the prepared CRISPR/Cas9 reagents using air-regulated compensation and an injection pressure of 90–115 psi in order to create a continuous flow of reagents (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Behringer <em>et al.</em> 2014</a>). Due to the continuous flow of reagents, fertilized oocytes with injections into the pronuclei received CRISPR/Cas9 reagents in both pronuclear and cytoplasmic regions. Injected zygotes were incubated at 37°, 5% CO<sub>2</sub> until transplantation. Approximately nine zygotes were then transferred into each oviduct of the pseudopregnant Swiss Webster females on the afternoon of the injection.</p></div><div id="sec-11" class="subsection"><h3>DNA isolation</h3><p id="p-14">For ES cells, DNA was isolated similarly to what has been previously described (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Zangala 2007</a>). Briefly, cells were centrifuged and resuspended in 300 µl DNA isolation buffer containing 100 mM NaCl, 50 mM Tris, pH 7.5, 10 mM EDTA, 0.5% SDS, and 0.5 mg/ml proteinase K at 56° for 3 hr. DNA was precipitated with an equal volume of isopropanol, and pelleted by centrifugation. The DNA pellet was washed with 750 µl of 70% EtOH, air dried, and resuspended in water. Mouse toe or tail clips from pups derived from injected embryos were incubated in DNA isolation buffer at 56° overnight, precipitated with an equal volume of isopropanol, and pelleted by centrifugation. The DNA pellet was washed with 750 µl of 70% EtOH, air dried, and resuspended in water. For N1, and additional generations, genomic DNA was isolated by incubation of the tail or toe clip in 300 µl of 50 mM NaOH at 95° for 2 hr followed by neutralization with 30 µl Tris-HCl, pH 8.0 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Truett <em>et al.</em> 2000</a>).</p></div><div id="sec-12" class="subsection"><h3>Genotyping</h3><p id="p-15">Genomic DNA isolated from either ES cells or mice generated from ES cell injections was amplified using MasterTaq using the primer pairs are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS1.pdf">Table S1</a> according to the cycling conditions listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS2.pdf">Table S2</a>. DNA isolated from mice generated from the injection of CRISPR/Cas9 reagents was amplified using either JumpStart Taq ReadyMix or Terra Taq, using the primer pairs listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS4.pdf">Table S4</a> according to the cycling conditions described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS2.pdf">Table S2</a>. Both Taq polymerase mixes were found to produce equivalent results. All PCR reactions were run on a 3% agarose gel to visualize the products.</p></div><div id="sec-13" class="subsection"><h3>DNA sequencing of cloned loxP sites in mice generated from CRISPR/Cas9 injection</h3><p id="p-16">DNA isolated from tail/toe clips was amplified using Phusion high fidelity polymerase using primers listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS4.pdf">Table S4</a> according to the cycling condition described in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS2.pdf">Table S2</a>. To add TA overhangs, 1.25 U of Terra Taq was added, and samples were incubated at 72° for 10 min. Amplified DNA was then cloned using the TOPO TA cloning kit according to the manufacturer’s protocol. Clones were screened for insertions according to the previously described genotyping protocols, purified using QIAquick PCR purification kits, and the PCR products were sequenced at either the Maine Medical Center Research Institute or the Dana-Farber/Harvard Cancer Center DNA Resource Core.</p></div><div id="sec-14" class="subsection"><h3>DNA sequencing of cloned oligonucleotides</h3><p id="p-17">Oligonucleotides were amplified using Phusion high fidelity DNA polymerase with primers Oligo4 F/R and Oligo5 F/R as listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS4.pdf">Table S4</a>, and cycling conditions listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS2.pdf">Table S2</a>, then cloned using the TOPO TA cloning kit according to the kit’s protocol. Clones were screened for insertions and sequenced using the T7 forward primer (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS4.pdf">Table S4</a>) at the Dana-Farber/Harvard Cancer Center DNA Resource Core.</p></div><div id="sec-15" class="subsection"><h3>Analysis of off-target Cas9 activity</h3><p id="p-18">The top 10 potential off-target sites for each sgRNA in the <em>Dock7</em> cKO1 and cKO2 models were identified using the Zhang lab algorithm (<a href="http://crispr.mit.edu/">http://crispr.mit.edu/</a>). Flanking PCR primers designed to amplify 300–600 bp fragments are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS5.pdf">Table S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS6.pdf">Table S6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS7.pdf">Table S7</a>, with the sequence and genomic coordinates of each potential off-target region. DNA was extracted from N0 mice and C57BL/6J mice by proteinase K digestion, and amplified using JumpStart Taq ReadyMix. Amplified DNA samples were assayed using the SURVEYOR mutation assay according to the protocol listed above. The mixture was resolved on 3% agarose gel for the predicted cutting pattern.</p></div><div id="sec-16" class="subsection"><h3>Data availability</h3><p id="p-19">Strains are available upon request. The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-17"><h2 class="">Results</h2><div id="sec-18" class="subsection"><h3>Targeted embryonic stem cell approach</h3><p id="p-20">We obtained several targeted ES cell clones generated from multiple members within the International Knockout Mouse Consortium (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Skarnes <em>et al.</em> 2011</a>). These ES cells were grown and expanded according to the instructions for each particular line, and prepared for injection into C57BL/6N morulae or blastocysts. Host C57BL/6 strains were chosen for the ability to distinguish chimerism based on coat color. <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> shows the clones injected, number and sex of chimeras obtained, and success in germline transmission of each allele. Three out of the six lines tested passed through the germline, for a germline transmission rate of 50%. Our results from this small collection of ES cell lines are consistent with other reports using aggregate data (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Cotton <em>et al.</em> 2015</a>) (KOMP germline transmission data, <a href="https://www.komp.org/gltrates2.php">https://www.komp.org/gltrates2.php</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468781/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468781/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Chimera generation and germline transmission from targeted embryonic stem cell clones</span><div class="sb-div caption-clear"></div></div></div><p id="p-24">For the unsuccessful clones, the desired mouse lines were not achieved for a variety of reasons, including lack of germline transmission (Mir199b), and sterility of chimeras (Lgr4). Of particular interest was the <em>Dock7<sup>tm1a(EUCOMM)Wtsi</sup></em> (<em>Dock7<sup>tm1a</sup></em>) knockout-first allele, which we explored in more detail. The <em>Dock7<sup>tm1a</sup></em> chimeric mice were identified by agouti coat color, and genotyped with primers to the neomycin (neo)-loxP2 region (neo-loxP2A F/R, neo-loxP2B F/R) and the loxP3 region (loxP3 F/R) indicated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>. The <em>Dock7<sup>tm1a</sup></em>-targeted ES cell line was used as a positive control for DNA amplification where amplified DNA product was detected for each primer pair tested (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>, ES total). Out of six chimeras generated, only one was identified with amplification from all three primer pairs, suggesting it contained cells with integration of an intact neo-loxP2 boundary (neo-loxP2A F/R, neo-loxP2B F/R) and loxp3 (loxP3 F/R), similar to the ES cell population generated from the EUCOMM stock (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). The other chimeric mice had evidence of cells containing sequences amplified by primer pair neo-loxP2A F/R, but not the other two target sites. This suggested that there could be a deletion of DNA in the cassette, and that there was heterogeneity in our starting ES cell population.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of the Dock7 first knockout allele (Dock7tm1a). (A) Targeting cassette and genotyping primers for the L1L2_Bact_P cassette inserted at the Dock7 gene locus. Primers used for genotyping are indicated, with the expected product size. Figure was adapted from IMPC.org (Ringwald et al. 2011). (B) Left: the original Dock7tm1a ES targeted line was used for genomic DNA isolation to evaluate the Dock7 locus (ES total). Genomic DNA was collected from six chimeric mice generated by injection of the ES cell line, and amplified using the three primer pairs. The Dock7tm1a ES cells, chimeras generated from ES injection, and ES cell subclones were genotyped at the neomycin (neo)-loxP2 (neo-loxP2A F/R, neo-loxP2B F/R) and loxP3 (loxP3 F/R) regions. PCR control is located in exon&#xA0;18, &#x223C;68&#xA0;kb downstream of the cassette insertion. Right: a total of 83 subclones was picked from the parental Dock7tm1a ES cell population, and DNA isolated from the ES cell subclones was analyzed by PCR. A representative example of six subclones is shown, which reflects the amplification patterns found within the subclones." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-145898997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of the &lt;em&gt;Dock7&lt;/em&gt; first knockout allele (&lt;em&gt;Dock7&lt;sup&gt;tm1a&lt;/sup&gt;&lt;/em&gt;). (A) Targeting cassette and genotyping primers for the L1L2_Bact_P cassette inserted at the &lt;em&gt;Dock7&lt;/em&gt; gene locus. Primers used for genotyping are indicated, with the expected product size. Figure was adapted from IMPC.org (Ringwald &lt;em&gt;et al.&lt;/em&gt; 2011). (B) Left: the original &lt;em&gt;Dock7&lt;sup&gt;tm1a&lt;/sup&gt;&lt;/em&gt; ES targeted line was used for genomic DNA isolation to evaluate the &lt;em&gt;Dock7&lt;/em&gt; locus (ES total). Genomic DNA was collected from six chimeric mice generated by injection of the ES cell line, and amplified using the three primer pairs. The &lt;em&gt;Dock7&lt;sup&gt;tm1a&lt;/sup&gt;&lt;/em&gt; ES cells, chimeras generated from ES injection, and ES cell subclones were genotyped at the neomycin (neo)-loxP2 (neo-loxP2A F/R, neo-loxP2B F/R) and loxP3 (loxP3 F/R) regions. PCR control is located in exon&#xA0;18, &#x223C;68&#xA0;kb downstream of the cassette insertion. Right: a total of 83 subclones was picked from the parental &lt;em&gt;Dock7&lt;sup&gt;tm1a&lt;/sup&gt;&lt;/em&gt; ES cell population, and DNA isolated from the ES cell subclones was analyzed by PCR. A representative example of six subclones is shown, which reflects the amplification patterns found within the subclones.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2051/F1.medium.gif" width="440" height="230"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/7/2051/F1.medium.gif" width="440" height="230"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468479" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Analysis of the <em>Dock7</em> first knockout allele (<em>Dock7<sup>tm1a</sup></em>). (A) Targeting cassette and genotyping primers for the L1L2_Bact_P cassette inserted at the <em>Dock7</em> gene locus. Primers used for genotyping are indicated, with the expected product size. Figure was adapted from IMPC.org (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Ringwald <em>et al.</em> 2011</a>). (B) Left: the original <em>Dock7<sup>tm1a</sup></em> ES targeted line was used for genomic DNA isolation to evaluate the <em>Dock7</em> locus (ES total). Genomic DNA was collected from six chimeric mice generated by injection of the ES cell line, and amplified using the three primer pairs. The <em>Dock7<sup>tm1a</sup></em> ES cells, chimeras generated from ES injection, and ES cell subclones were genotyped at the neomycin (neo)-loxP2 (neo-loxP2A F/R, neo-loxP2B F/R) and loxP3 (loxP3 F/R) regions. PCR control is located in exon 18, ∼68 kb downstream of the cassette insertion. Right: a total of 83 subclones was picked from the parental <em>Dock7<sup>tm1a</sup></em> ES cell population, and DNA isolated from the ES cell subclones was analyzed by PCR. A representative example of six subclones is shown, which reflects the amplification patterns found within the subclones.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">The <em>Dock7<sup>tm1a</sup></em> chimeras were bred to C57BL/6J mice, and the offspring were assessed similarly for coat color and genotype to determine germline transmission of the mutant allele (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Of the four chimeras that produced offspring, only chimeras 1 and 5 showed evidence of germline transmission by agouti coat color. Chimera 1 showed evidence of germline transmission of only a partial <em>Dock7<sup>tm1a</sup></em> cassette, while chimera 5 produced four agouti offspring, genotyping showed no evidence of the targeting cassette in any of the pups.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468625/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468625/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Characteristics of N1 offspring from <em>Dock7<sup>tm1a</sup></em> chimeras</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">To address the possibility of a heterogeneous population of ES cells in the <em>Dock7<sup>tm1a</sup></em> line, we established 83 clonal subpopulations by individual colony picking from the parental <em>Dock7<sup>tm1a</sup></em> line. These subclones were expanded, and neo-loxP2 and loxP3 regions were screened with the aforementioned primer pairs. The amplification patterns matched what was found in the chimeric mice: some clones had all three products (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>, subclones 1 and 6), and some had a product only from primer pair neo-loxP2A F/R (subclones 3, 5). We also obtained subclones that yielded no evidence of amplification from any of the three primer pairs (subclones 2 and 4). Collectively, we found evidence for an intact targeting cassette in 38% of the subclones (32/83), and a partial targeting cassette in 55% (46/83) of the subclones. Interestingly, 6% (5/83) were negative using all three primer pairs. The presence of ES cells with multiple genotypes within the <em>Dock7<sup>tm1a</sup></em> subclones confirms the heterogeneous nature of the <em>Dock7<sup>tm1a</sup></em> ES cell population, and explains the generation of chimeras with multiple genotypes.</p><p id="p-29">Two <em>Dock7<sup>tm1a</sup></em> ES cell subclones containing the intact targeting cassette (subclones 1 and 6, <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>), originally isolated from the original 83 subclones, were analyzed for chromosome number. We found that over 50% of cells contained an abnormal number of chromosomes, which is known to significantly decrease the potential for germline transmission (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Longo <em>et al.</em> 1997</a>). Consistent with our observations of abnormal total chromosomal numbers in this particular clone derived from the JM8A3.N1 ES cell line, other investigators have reported that, of the 151 clones (19.8%) tested from the EUCOMM repository, 30 also contained over 50% of cells with abnormal chromosome numbers (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Cotton <em>et al.</em> 2015</a>). Furthermore, the same study reported that injection of ES cells lines from the eight clones with over 50% of cells with abnormal chromosome counts did not result in germline transmission of the lines test (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Cotton <em>et al.</em> 2015</a>). These collective data suggest that the chance of germline transmission of a correctly targeted <em>Dock7<sup>tm1a</sup></em> clone was low. Therefore, we did not pursue the <em>Dock7<sup>tm1a</sup></em> clone or subclones further. Additional clones targeting <em>Dock7</em> with conditional knockout potential were not available through the KOMP project at the time these experiments were performed.</p></div><div id="sec-19" class="subsection"><h3>CRISPR/Cas9 for conditional gene targeting of the Dock7 locus</h3><p id="p-30">Using the CRISPR/Cas9 system as an alternative method to insert loxP sites within the <em>Dock7</em> locus, two conditional deletion strategies were designed (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). One inserted loxP sites to flank exons 3 and 4 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, cKO1), and the second inserted loxP sites to flank exons 3 and 7 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>, cKO2). The loxP insertion sites in the <em>Dock7</em> cKO1 model were located within the same introns as the <em>Dock7<sup>tm1a</sup></em> clone, and were the primary focus of this study. Cas9 mRNA, validated sgRNAs, and ∼195 bp oligonucleotide donors containing loxP sites were coinjected into single-cell C57BL/6J zygotes. The resultant mice were genotyped for the presence of the loxP sites, and assessed for phenotypes associated with biallelic disruption of the <em>Dock7</em> gene in <em>Misty</em> mice (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Previously, a spontaneous mutant mouse, <em>Misty</em>, was identified with a mutation in the <em>Dock7</em> allele. Mice homozygous for the <em>Misty</em> mutation have undetectable levels of the Dock7 protein (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Motyl <em>et al.</em> 2013</a>), a diluted coat color, and a white belly spot (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Sviderskaya <em>et al.</em> 1998</a>) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>), which allowed us to easily phenotype potential homozygous null <em>Dock7</em> alleles that could have been generated by NHEJ, or incorrect targeting. Biallelic disruption of the <em>Dock7</em> gene as indicated by a diluted coat color and a white belly spot was observed in five mice (5/41, 12%) in the cKO1 model (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Interestingly, each of the five mice displaying a diluted coat color phenotype had a complete absence of amplified product for both loxP insertion sites (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, mouse 127 and mouse 137). To understand the nature of these disruptions, we selected mice with a white belly spot, and genotyped for deletion of the targeted exons with primers flanking exons 3 and 4 as depicted in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>. Each of the five mice genotyped positive for a deletion of DNA, examples are shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>. The amplified product deletion products were sequenced, and the results are diagramed in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>. While we found evidence for NHEJ (mouse 126), all others had evidence of some insertion from the oligonucleotide, although mutations or deletions were also noted. Furthermore, these mice showed deletion of exons 3 and 4, likely causing the diluted coat color and white belly spot observed in these mice. Because our focus was on generating a conditional, floxed allele, we did not breed mice containing biallelic disruption of the <em>Dock7</em> gene to evaluate germline transmission. We then assessed the potential deletion of the DNA between the potential Cas9 cleavage sites located at the loxP4 and loxP5 insertion sites. In our N0 cohort, 45% (19/42) of the mice had some variant of DNA deletion between the two Cas9 cleavage sites (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>) as observed in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>. The disruption of the target gene was found to be a frequent event with CRISPR/Cas9-mediated insertions.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Strategy to generate conditional alleles of Dock7 by CRISPR/Cas9-mediated targeting. (A) One sgRNA was validated within intron&#xA0;2 and intron&#xA0;4, a distance of 2.1&#xA0;kb apart. Two corresponding oligonucleotides were synthesized with 79&#x2013;80&#xA0;bp homology arms flanking the loxP sequences. With the translational start site in exon&#xA0;1, Cre-mediated recombination of this allele is predicted to result in a premature stop codon due to deletion of exons&#xA0;3&#x2013;4 and surrounding intronic sequences. (B) The same upstream sgRNA in intron&#xA0;2 was also used in combination with an sgRNA targeting in intron&#xA0;7, with a distance of 5.3&#xA0;kb in between. The downstream oligonucleotide was designed with a 75&#x2013;81&#xA0;bp homology arm flanking loxP and the BamHI restriction site. Cre-mediated recombination of this allele is also expected to generate a premature stop codon upon deletion of exons&#xA0;3&#x2013;7." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-145898997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Strategy to generate conditional alleles of &lt;em&gt;Dock7&lt;/em&gt; by CRISPR/Cas9-mediated targeting. (A) One sgRNA was validated within intron&#xA0;2 and intron&#xA0;4, a distance of 2.1&#xA0;kb apart. Two corresponding oligonucleotides were synthesized with 79&#x2013;80&#xA0;bp homology arms flanking the loxP sequences. With the translational start site in exon&#xA0;1, Cre-mediated recombination of this allele is predicted to result in a premature stop codon due to deletion of exons&#xA0;3&#x2013;4 and surrounding intronic sequences. (B) The same upstream sgRNA in intron&#xA0;2 was also used in combination with an sgRNA targeting in intron&#xA0;7, with a distance of 5.3&#xA0;kb in between. The downstream oligonucleotide was designed with a 75&#x2013;81&#xA0;bp homology arm flanking loxP and the &lt;em&gt;Bam&lt;/em&gt;HI restriction site. Cre-mediated recombination of this allele is also expected to generate a premature stop codon upon deletion of exons&#xA0;3&#x2013;7.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2051/F2.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/7/2051/F2.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468529" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Strategy to generate conditional alleles of <em>Dock7</em> by CRISPR/Cas9-mediated targeting. (A) One sgRNA was validated within intron 2 and intron 4, a distance of 2.1 kb apart. Two corresponding oligonucleotides were synthesized with 79–80 bp homology arms flanking the loxP sequences. With the translational start site in exon 1, Cre-mediated recombination of this allele is predicted to result in a premature stop codon due to deletion of exons 3–4 and surrounding intronic sequences. (B) The same upstream sgRNA in intron 2 was also used in combination with an sgRNA targeting in intron 7, with a distance of 5.3 kb in between. The downstream oligonucleotide was designed with a 75–81 bp homology arm flanking loxP and the <em>Bam</em>HI restriction site. Cre-mediated recombination of this allele is also expected to generate a premature stop codon upon deletion of exons 3–7.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468636/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468636/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Frequency of loxP insertion at the <em>Dock7</em> locus using CRISPR/Cas</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotype and genotyping of Dock7 null alleles in the cKO1 model. (A) A proportion of mice from the Dock7 CRISPR/Cas9 injection displayed a diluted coat color and white belly spot similar to that of the Misty/Misty mice, which have a no detectable Dock7 protein. This phenotype was found in five out of the 35 mice pups assessed for coat color in the Dock7 cKO1-2 injection. (B) Summary of sequencing results from mice with diluted coat color and white belly spot from injection Dock7 cKO1-2. Deletion of exons&#xA0;3 and 4 was detected by amplifying DNA isolated from mice in (A) and the amplified products were cloned and sequenced, and the sequencing results are depicted in the schematic above. In all five mice, two deletion products were identified, and correspond to the deletion of exons&#xA0;3 and 4 of Dock7 gene, similar to what is observed in Figure 4. DNA was amplified with loxP4A F/loxP5A R and sequenced with loxP4B F and/or loxP5B R." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-145898997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phenotype and genotyping of &lt;em&gt;Dock7&lt;/em&gt; null alleles in the cKO1 model. (A) A proportion of mice from the &lt;em&gt;Dock7&lt;/em&gt; CRISPR/Cas9 injection displayed a diluted coat color and white belly spot similar to that of the &lt;em&gt;Misty/Misty&lt;/em&gt; mice, which have a no detectable Dock7 protein. This phenotype was found in five out of the 35 mice pups assessed for coat color in the &lt;em&gt;Dock7&lt;/em&gt; cKO1-2 injection. (B) Summary of sequencing results from mice with diluted coat color and white belly spot from injection &lt;em&gt;Dock7&lt;/em&gt; cKO1-2. Deletion of exons&#xA0;3 and 4 was detected by amplifying DNA isolated from mice in (A) and the amplified products were cloned and sequenced, and the sequencing results are depicted in the schematic above. In all five mice, two deletion products were identified, and correspond to the deletion of exons&#xA0;3 and 4 of &lt;em&gt;Dock7&lt;/em&gt; gene, similar to what is observed in Figure 4. DNA was amplified with loxP4A F/loxP5A R and sequenced with loxP4B F and/or loxP5B R.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2051/F3.medium.gif" width="440" height="380"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/7/2051/F3.medium.gif" width="440" height="380"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468613" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Phenotype and genotyping of <em>Dock7</em> null alleles in the cKO1 model. (A) A proportion of mice from the <em>Dock7</em> CRISPR/Cas9 injection displayed a diluted coat color and white belly spot similar to that of the <em>Misty/Misty</em> mice, which have a no detectable Dock7 protein. This phenotype was found in five out of the 35 mice pups assessed for coat color in the <em>Dock7</em> cKO1-2 injection. (B) Summary of sequencing results from mice with diluted coat color and white belly spot from injection <em>Dock7</em> cKO1-2. Deletion of exons 3 and 4 was detected by amplifying DNA isolated from mice in (A) and the amplified products were cloned and sequenced, and the sequencing results are depicted in the schematic above. In all five mice, two deletion products were identified, and correspond to the deletion of exons 3 and 4 of <em>Dock7</em> gene, similar to what is observed in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>. DNA was amplified with loxP4A F/loxP5A R and sequenced with loxP4B F and/or loxP5B R.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotyping of the Dock7 cKO1 allele. Examples of genotyping are shown for the Dock7 cKO1 allele. N0 mice were generated from the cKO1-2 CRISPR/Cas9 injections. N1 mice resulted from breeding of 122 female founder, designated with (*) to a C57BL/6J male. DNA ladder size is indicated in base pairs (bp). DNA fragment size of the loxP (L), wild type (+), and small deletions due to Cas9 cleavage (&#x394;) are indicated. (A) The loxP4 (loxP4B F/R) and loxP5 (loxP5B F/R), and (B) deletion of exons 3&#x2013;4 (loxP4B F / loxP5B R) primer pairs, were used in genotyping. The corresponding genotypes of the N0 and N1 mice are shown in (A), where (n) indicates no band during genotyping and a potential null allele." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-145898997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotyping of the &lt;em&gt;Dock7&lt;/em&gt; cKO1 allele. Examples of genotyping are shown for the &lt;em&gt;Dock7&lt;/em&gt; cKO1 allele. N0 mice were generated from the cKO1-2 CRISPR/Cas9 injections. N1 mice resulted from breeding of 122 female founder, designated with (*) to a C57BL/6J male. DNA ladder size is indicated in base pairs (bp). DNA fragment size of the loxP (L), wild type (+), and small deletions due to Cas9 cleavage (&#x394;) are indicated. (A) The loxP4 (loxP4B F/R) and loxP5 (loxP5B F/R), and (B) deletion of exons 3&#x2013;4 (loxP4B F / loxP5B R) primer pairs, were used in genotyping. The corresponding genotypes of the N0 and N1 mice are shown in (A), where (n) indicates no band during genotyping and a potential null allele.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2051/F4.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/7/2051/F4.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468588" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">Genotyping of the <em>Dock7</em> cKO1 allele. Examples of genotyping are shown for the <em>Dock7</em> cKO1 allele. N0 mice were generated from the cKO1-2 CRISPR/Cas9 injections. N1 mice resulted from breeding of 122 female founder, designated with (*) to a C57BL/6J male. DNA ladder size is indicated in base pairs (bp). DNA fragment size of the loxP (L), wild type (+), and small deletions due to Cas9 cleavage (Δ) are indicated. (A) The loxP4 (loxP4B F/R) and loxP5 (loxP5B F/R), and (B) deletion of exons 3–4 (loxP4B F / loxP5B R) primer pairs, were used in genotyping. The corresponding genotypes of the N0 and N1 mice are shown in (A), where (n) indicates no band during genotyping and a potential null allele.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Genotyping was also performed for all N0 mice to detect the loxP site insertions. Examples of genotyping are shown in <a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a>. The live-birth rate of the injected embryos was 15% (47/305) for the cKO1 model (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). LoxP4 insertions were observed in 10/47 mice (21%), loxP5 insertions were present in 11/47 mice (23%), and insertion of both loxP4 and loxP5 occurred in 11% (5/47) of the N0 mice from the <em>Dock7</em> cKO1-1,-2 injection (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). Interestingly, founder 117 not only genotyped positive for the wildtype allele and loxP4 insertion, but also had evidence of deletion of exons 3 and 4 (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The presence of three or more alleles, including the deletion of exons 3 and 4, for loxP4 or loxP5 was identified in a total of seven (17%) of the 42 N0 mice genotyped, suggesting that Cas9 cleavage is either inducing major recombination events, or the generation of mosaic mice. Nine N0 founder mice with identified loxP sites were bred, and the N1 generation was analyzed by genotyping and sequencing. Overall germline transmission rates were over 89% (8/9) for the mice bred. Offspring number (N1 generation) and genotypes are recorded in <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>. However, the rate at which loxP sites in the N0 mice were passed to N1 mice varied, and was consistent with mosaicism in the founder mouse. Evidence of mosaicism was also present in <em>Dock7</em> cKO1 founder 122, as offspring consisted of mice with floxed alleles as well as a single loxP site. Additionally, five N0 mice, where loxP sites were not detected by genotyping, were bred. For the mice, one to two litters, totaling five to 10 pups, were genotyped, and no mice containing loxP sites were found in the N1 generation (data not shown). While mosaicism could potentially create germline transmission of a loxP site to N1 generation, even though they were not detected in the N0 mice, this scenario was not observed in our cohort, suggesting it is not a frequent event.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468640/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468640/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Germline transmission rate of loxP sites in the <em>Dock7</em> cKO1 model</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">In a separate set of cytoplasmic injections for the <em>Dock7</em> cKO2 model, similar results were observed compared to the <em>Dock7</em> cKO1 model. The live-birth rate for transferred embryos was 12%, with 20/174 zygotes resulting in pups. LoxP4 insertions were observed in 7/20 mice (35%), loxP6 insertions were present in 3/20 mice (15%), and insertion of both loxP4 and loxP6 occurred in 1/20 mice (5%) from the <em>Dock7</em> cKO2 injection (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS8.pdf">Table S8</a>). Germline transmission of loxP sites occurred in 5/5 (100%) of the mice bred, and mosaicism was detected in one founder mouse that produced offspring containing floxed alleles or a single loxP site (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS10.pdf">Table S10</a>). Biallelic disruption of the <em>Dock7</em> gene occurred in 5/19 mice assessed for coat color, and deletion of DNA between the Cas9 cut sites was present in 10/20 mice (50%). Examples of genotyping for both loxP insertion and deletion of DNA between the Cas9 cut sites are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/FigureS2.eps">Figure S2</a>. While the results were overall similar to those observed in the <em>Dock7</em> cKO1 model, an increase in biallelic disruption of the <em>Dock7</em> gene was observed. It is unclear whether this was due to location of injection (cytoplasmic <em>vs.</em> pronuclear), variation between injection days, or was affected by the distance between the Cas9 cleavage sites. Further studies are necessary to investigate these questions.</p><p id="p-39">The inserted loxP sites and flanking sequences in the N1 and selected N0 mice were cloned and sequenced. Several of the loxP sites sequenced contained small 1- to 2-bp deletions or 1-bp substitutions, whereas some other insertions carried larger deletions within the loxP site itself (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Correctly sequenced and mutated loxP sites are indicated in <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS10.pdf">Table S10</a> for the cKO1 and cKO2 models, respectively. From either model, only founders 122 (cKO1) and 17 (cKO2) produced floxed <em>Dock7</em> mice, with two correctly sequenced loxP sites in <em>cis</em> orientation. These loxP mutations were observed in N0, and their respective offspring. In order to determine if any of these mutations of the insertion sequence were introduced during oligonucleotide synthesis, the oligonucleotide donors for loxP4 and loxP5 were cloned and sequenced. The oligonucleotide donors indeed contained 1- to 2-bp deletions and a 1-bp substitution within the loxP site and/or surrounding sequence (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). These mutations observed in the oligonucleotide donors were similar to the type of mutation observed in the CRISPR-generated transgenic mice, suggesting some of the mutations occurred during the synthesis process. However, no large deletions or insertions in the loxP site were observed in the oligonucleotide donors. These data suggest that some of the small mutations in the oligonucleotides may be responsible for similar mutations observed in the inserted loxP sites. We believe that the larger deletions or insertions of DNA sequences occurred during the recombination process.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutations at loxP insertion sites. LoxP sites in the Dock7 cKO1 line were sequenced in the N1 and selected N0 mice. Several deviations from the loxP consensus sequence were observed in these mice. The loxP sequence is shown in capital letters, and inserted DNA is shown in lowercase letters. Amplification of isolated DNA was performed using loxP4 (loxP4A F/R) and loxP5 (loxP5A F/R) primers pairs. LoxP4 (loxP4B F/R) and loxP5 (loxP5B F/R) primers were used for sequencing. LoxP mutations in mice 3 and 116 were passed via germline transmission. Mice 4 and 6 were not bred." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-145898997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mutations at loxP insertion sites. LoxP sites in the &lt;em&gt;Dock7&lt;/em&gt; cKO1 line were sequenced in the N1 and selected N0 mice. Several deviations from the loxP consensus sequence were observed in these mice. The loxP sequence is shown in capital letters, and inserted DNA is shown in lowercase letters. Amplification of isolated DNA was performed using loxP4 (loxP4A F/R) and loxP5 (loxP5A F/R) primers pairs. LoxP4 (loxP4B F/R) and loxP5 (loxP5B F/R) primers were used for sequencing. LoxP mutations in mice 3 and 116 were passed via germline transmission. Mice 4 and 6 were not bred.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2051/F5.medium.gif" width="440" height="84"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/7/2051/F5.medium.gif" width="440" height="84"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468759" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-40" class="first-child">Mutations at loxP insertion sites. LoxP sites in the <em>Dock7</em> cKO1 line were sequenced in the N1 and selected N0 mice. Several deviations from the loxP consensus sequence were observed in these mice. The loxP sequence is shown in capital letters, and inserted DNA is shown in lowercase letters. Amplification of isolated DNA was performed using loxP4 (loxP4A F/R) and loxP5 (loxP5A F/R) primers pairs. LoxP4 (loxP4B F/R) and loxP5 (loxP5B F/R) primers were used for sequencing. LoxP mutations in mice 3 and 116 were passed via germline transmission. Mice 4 and 6 were not bred.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468680/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468680/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Sequence analysis of oligonucleotide donors</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">Sequencing of the <em>Dock7</em> locus in founders 122 (cKO1) and 17 (cKO2) confirmed correct insertion of both loxP sites creating the floxed <em>Dock7</em> allele. Only these mice produced offspring containing two loxP sites on the same chromosome with the correct sequence. To calculate the success rate in generating the <em>Dock7</em> floxed alleles, we divided the number of correctly targeted founders by the total number of embryos transferred per cKO model. This represents a 0.33% (1/305) and 0.57% (1/174) success rate for the models, respectively, for the <em>Dock7</em> cKO1 and cKO2 models (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS9.pdf">Table S9</a>). Only 2–5% of pups born were capable of establishing a mouse line bearing the correct floxed allele. In order to test off-target Cas9 cleavage activity, the top 10 potential off-target sites were identified as predicted by the algorithm from the Zhang lab, and the cleavage activity at these sites were tested using the SURVEYOR mutation assay. No off-target activity was observed for either the <em>Dock7</em> cKO1 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>) or the <em>Dock7</em> cKO2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/FigureS3.eps">Figure S3</a>) model. DNA isolated from C57BL/6J mice was used as a negative control (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/FigureS4.eps">Figure S4</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of potential off-target effects in the Dock7 cKO1. The SURVEYOR mutation assay was used to evaluate off-target cleavage by Cas9. (A) LoxP4 and loxP5 insertion sites were used as a positive control. Cas9 activity was screened at the top 10 homologous sites in the genome to the sgRNA for both loxP4 (B) and loxP5 (C) in the N0 mouse, using primer pairs listed in Table S5 and Table S6. As a negative control, off-target activity was assessed in the wild-type C57BL/6J mouse at the insertion sites for both loxP4 and loxP6, as shown in Figure S4. Primer pairs for loxP4 (loxP4A F/R) and loxP5 (loxP5B F/R) were used for amplification of control DNA samples." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-145898997" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of potential off-target effects in the &lt;em&gt;Dock7&lt;/em&gt; cKO1. The SURVEYOR mutation assay was used to evaluate off-target cleavage by Cas9. (A) LoxP4 and loxP5 insertion sites were used as a positive control. Cas9 activity was screened at the top 10 homologous sites in the genome to the sgRNA for both loxP4 (B) and loxP5 (C) in the N0 mouse, using primer pairs listed in Table S5 and Table S6. As a negative control, off-target activity was assessed in the wild-type C57BL/6J mouse at the insertion sites for both loxP4 and loxP6, as shown in Figure S4. Primer pairs for loxP4 (loxP4A F/R) and loxP5 (loxP5B F/R) were used for amplification of control DNA samples.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2051/F6.medium.gif" width="440" height="129"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/7/2051/F6.medium.gif" width="440" height="129"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2051/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468607" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">Analysis of potential off-target effects in the <em>Dock7</em> cKO1. The SURVEYOR mutation assay was used to evaluate off-target cleavage by Cas9. (A) LoxP4 and loxP5 insertion sites were used as a positive control. Cas9 activity was screened at the top 10 homologous sites in the genome to the sgRNA for both loxP4 (B) and loxP5 (C) in the N0 mouse, using primer pairs listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS5.pdf">Table S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/TableS6.pdf">Table S6</a>. As a negative control, off-target activity was assessed in the wild-type C57BL/6J mouse at the insertion sites for both loxP4 and loxP6, as shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/FigureS4.eps">Figure S4</a>. Primer pairs for loxP4 (loxP4A F/R) and loxP5 (loxP5B F/R) were used for amplification of control DNA samples.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">As a phenotypic method to analyze the <em>Dock7</em> cKO1 model, and to assess the functionality of the Dock floxed allele, we crossed the <em>Dock7</em> cKO1 model with the Sox2-Cre strain, which expresses Cre protein in the early germline and is efficient in generating germline deletions. Following Cre recombination, excision of exons 3 and 4 was confirmed by DNA sequencing (data not shown). Mice homozygous for the <em>Dock7</em> null allele (<em>Dock7</em><sup>−/−</sup>) generated through Cre recombination showed similar coat color and white belly (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1/FigureS5.tif">Figure S5</a>) as the <em>Misty</em> mice and <em>Dock7</em> null mice generated through biallelic disruption of the <em>Dock7</em> gene by CRISPR/Cas (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). These data confirm that the floxed allele can undergo Cre recombination, that the coat color phenotype is a direct result of disruption of the <em>Dock7</em> gene, and that deletion of exon 3 and 4 in the <em>Dock7</em> gene is sufficient to elicit the null <em>Dock7</em> phenotype.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-45">The establishment of the KOMP, a resource for targeted C57BL/6N ES cell lines, has dramatically enhanced both the speed and quality of research on the C57BL/6 inbred background (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bradley <em>et al.</em> 2012</a>). However, one limitation is the less than ideal success rate of germline transmission of the targeted ES cell lines. Interestingly, even with a small group of six targeted alleles, the success rate observed in this study recapitulated that observed by the KOMP repository of ∼50% [KOMP repository germline transmission rates and (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Cotton <em>et al.</em> 2015</a>)]. Furthermore, additional issues that prevented generation of the <em>Dock7</em> transgenic line from the KOMP-targeted cells included an incomplete targeting cassette and a mixed population of ES cells from the parental “clone” (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Ryder <em>et al.</em> 2013</a>). Our observations indicate that characterization of the targeted ES cell clones prior to injection is an important validation step.</p><p id="p-46">The development and application of the CRISPR/Cas9 system was a major advancement in the generation of genetically modified mice. The recent successful application of CRISPR/Cas9 technology in genome modification, regardless of species and genetic background, has established it as an efficient and powerful alternative to traditional gene targeting. It is now possible to generate an NHEJ-mediated gene disruption in mice in as little as 4 wk. However, the efficiency of targeted genomic insertion was at least two-fold lower than that of deletions (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Yang <em>et al.</em> 2013</a>). The rate at which both loxP sites were incorporated ranged from 5/47 (11%) for cKO1 to 1/20 (5%) cKO2 in our models. However, the rate at which a correctly sequenced floxed allele in cKO1 was generated and passed to the N1 generation was lower, at 1/47 (2%). Of the four additional mice that showed evidence of insertion of loxP4 and loxP5, two were found to have mutated loxP sites, one died prior to weaning, and the final mouse had no germline transmission of either loxP site after four litters.</p><p id="p-47">One of the benefits to CRISPR/Cas9 is that the genome of any mouse strain can be manipulated. Previous studies have demonstrated success in hybrid/outbred strains (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Wang <em>et al.</em> 2013</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Yang <em>et al.</em> 2013</a>), and mice in a C57BL/6N background (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Lee and Lloyd 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Yen <em>et al.</em> 2014</a>). However, it is still unclear if variability between mouse strains can affect the overall outcome of success in developing a floxed allele using CRISPR/Cas9. Since we generated only one functional floxed <em>Dock7</em> mouse in each model using C57BL/6J zygotes, our data suggest that that injection and transfer of at least 300–400 C57BL/6J zygotes may provide the necessary coverage to generate between 40 and 60 N0 mice in the C57BL/6J background, which is a reasonable number to screen.</p><p id="p-48">Genomic manipulation by CRISPR/Cas9 has been associated with robust rates of germline transmission (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Li <em>et al.</em> 2013a</a>), <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2013b</a>). In our study, we chose to primarily breed N0 mice containing either one or two loxP sites. Germline transmission rates of loxP sites were above 93% for the mice bred. Furthermore, germline transmission of loxP sites was generally achieved within one to two litters, requiring screening an average of seven pups per founder. Only founder 117 required screening of 19 pups (three litters). All other founders required screening of only six pups, or one litter, including both founders harboring the correctly sequenced floxed alleles. Founder 142 did not produce germline transmission of single inserted loxP site after 28 pups or four litters; since founder 142 genotyped positive for only one loxP site, breeding was discontinued. In all cases, no offspring genotyped positive for loxP sites that were not detected in the N0 generation from either transgenic or nontransgenic mice. These data suggest that priority should be placed on screening offspring from transgenic founders positive for both loxP sites, and that identification of potential loxP sites should be reliable in the N0 generation. Furthermore, screening of the N1 generation can be accomplished relatively quickly in one to two litters. However, with the necessity of screening the N1 generation to confirm the germline transmission of the floxed allele, and the potential for mutations in the regions of recombination, the length of time necessary to validate a conditional mouse model can be more extensive than expected. As every genetic locus is different, and loxP insertion frequency is likely to be dependent on the surrounding sequence or transcriptional activity of the gene, there are no standard guidelines that can be extrapolated from targeting one particular gene.</p><p id="p-49">Heterogeneity in the N1 generation due to mosaicism has been observed previously (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Yen <em>et al.</em> 2014</a>; <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Yang <em>et al.</em> 2013</a>). Mosaicism in the founder mice was present in both founders that produced the correctly targeted <em>Dock7</em> cKO1 and cKO2 mice, with variation in the number of loxP sites passed from the N0 to N1 generation. Additional evidence of mosaicism could be observed in the N0 mice containing both a wild-type and loxP positive allele as well as a deletion of DNA between the Cas9 cut sites, such as in the loxP4 genotyping in cKO1 founder 117. Therefore with the frequent occurrence of mosaicism with CRISPR/Cas9, screening the N1 generation is necessary to confirm the establishment of the conditional deletion line.</p><p id="p-50">Due to the novelty of the CRISPR/Cas9 system, many of the technical issues related to the optimal strategy of simultaneous insertion of two loxP sites using two sgRNAs and two oligonucleotide donors have not been established. Various mutant alleles can result from this strategy, including deletions at either or both of the target sites, or between the two target sites by NHEJ, a single targeted loxP site, or two targeted loxP sites. In addition, if both targeted loxP sites were inserted, they could be either on the same or a different allele of the gene if located on an autosome. Our screening of a large cohort of N0 and N1 mice revealed all of the aforementioned varieties of modified alleles, as well as mutated and partial loxP sites. Several loxP variants occurred in mice with two correctly inserted loxP sites, thus eliminating their use as a conditional deletion model, and decreasing the efficiency of generating a correctly targeted allele. Further investigation suggested that, in some cases, 1- to 2-bp mutations in the loxP sites themselves were in fact present in the donor oligonucleotides. Therefore, it is likely that use of oligonucleotides of almost 200 bp greatly increases the error rate. We hypothesize that smaller oligonucleotides, or oligonucleotides generated from a different source, potentially a sequenced plasmid, might decrease the frequency of error and increase the correct targeting allele efficiency. However, it is unclear if the partial loxP sites observed here are direct results of a partial insertion or of full insertion followed by a subsequent deletion of DNA within the loxP site. Further work is necessary to identify the mechanisms of DNA deletion and mutation observed in loxP sequences.</p><p id="p-51">Among the mice we obtained from zygote injections, deletion of genomic DNA between the Cas9 cut sites occurred in 42–67% of mice. This was 1.5- to 2-fold higher than the loxP site insertion rate (18–50%), which is similar to what has been previously reported (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Yang <em>et al.</em> 2013</a>), suggesting that NHEJ is still more efficient than homologous recombination. We also observed large biallelic deletions in between the Cas9 cut sites at a rate between 12% and 25% among mice generated from injected zygotes. The use of a visible phenotype greatly enhanced the screening process for these alleles. Although mice with this type of DNA deletion may in fact be a useful complement to conditional null alleles, production of these alleles still reduced the overall efficiency of generation of the desired transgenic strains. As deletion of DNA between the Cas9 cut sites occurred in all three injections at varying rates, it is unclear what variables may increase the rate of these deletions, including the location of injection (pronuclear <em>vs.</em> cytoplasmic), the distance between Cas9 cut sites, and gene-to-gene variability. SCR7—a DNA ligase IV inhibitor—has been used as an inhibitor of NHEJ to promote homology-mediated repair (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Vartak and Raghavan 2015</a>), but that strategy was not utilized in this study. Lastly, it is worth noting that the use of CRISPR/Cas9 to generate conditional deletion mice of heterozygous or homozygous lethal genes would naturally select against embryos containing these global deletion alleles, thus resulting in fewer N0 mice, but increasing targeting efficiency results.</p><p id="p-52">Overall, we succeeded in the generation of two novel, conditional, null alleles of the <em>Dock7</em> gene using the CRISPR/Cas9 system to target two intronic sites. These sites were modified with loxP sequences provided via oligonucleotide donors. Our results showed a greater than expected variability in sequence within and surrounding the loxP sites, highlighting the importance of full characterization of the modified sequence.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We thank Thomas Gridley for critical review of this article as well as Anyonya Guntur, members of the Rosen laboratory, and the Maine Medical Center Research Institute community for their insightful discussion. This study was supported by the Mouse Transgenic and In Vivo Imaging Core facility (Lucy Liaw PhD, Core Director) of grant number P30GM103392 and Phase III COBRE in Vascular Biology (Robert E. Friesel PhD, P.I.), a grant supported by the National Institute of General Medical Sciences. The research reported in this publication was also supported by the National Institute of Aging, grant number AG040217 to Clifford J. Rosen, MD, and the National Institute of Arthritis and Musculoskeletal and Skin Diseases of the National Institutes of Health under Award Number F32AR067071 to Kathleen A. Bishop, PhD. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-54">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030601/-/DC1</a></p></li><li class="fn" id="fn-10"><p id="p-55"><em>Communicating editor: D. W. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-11-23"><span class="received-label">Received </span>November 23, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-05-05"><span class="accepted-label">Accepted </span>May 5, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Bishop <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-6.7.2051.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behringer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gertsenstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Manipulating the Mouse Embryo: a Laboratory Manual</span>, <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.2" data-doi="10.1007/s00335-012-9422-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anastassiadis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayadi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Battey</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The mammalian gene function resource: the International Knockout Mouse Consortium.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">580</span>–<span class="cit-lpage">586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DBradley%26rft.auinit1%253DA.%26rft.volume%253D23%26rft.issue%253D9-10%26rft.spage%253D580%26rft.epage%253D586%26rft.atitle%253DThe%2Bmammalian%2Bgene%2Bfunction%2Bresource%253A%2Bthe%2Binternational%2Bknockout%2Bmouse%2Bconsortium.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-012-9422-2%26rft_id%253Dinfo%253Apmid%252F22968824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-012-9422-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22968824&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309543200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brennan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ngo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balaji</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rapid knockout and reporter mouse line generation and breeding colony establishment using EUCOMM conditional-ready embryonic stem cells: a case study.</span> <abbr class="cit-jnl-abbrev">Front. Endocrinol. (Lausanne)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BEndocrinol.%2B%2528Lausanne%2529%26rft.volume%253D6%26rft.spage%253D105%26rft_id%253Dinfo%253Apmid%252F26175717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26175717&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.4" data-doi="10.1126/science.1231143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barretto</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multiplex genome engineering using CRISPR/Cas systems.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">819</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1231143%26rft_id%253Dinfo%253Apmid%252F23287718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjEyMS84MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjA1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.5" data-doi="10.1007/s00335-015-9555-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotton</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meilak</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templeton</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzales</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nenci</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Utilising the resources of the International Knockout Mouse Consortium: the Australian experience.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D26%26rft.spage%253D142%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-015-9555-1%26rft_id%253Dinfo%253Apmid%252F25645994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-015-9555-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25645994&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.6" data-doi="10.1093/bfgp/elm022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedel</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seisenberger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaloff</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wurst</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">EUCOMM—the European conditional mouse mutagenesis program.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics Proteomics</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">180</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felm022%26rft_id%253Dinfo%253Apmid%252F17967808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiYmZncCI7czo1OiJyZXNpZCI7czo3OiI2LzMvMTgwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.7" data-doi="10.1093/nar/gkt977"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koscielny</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yaikhom</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meehan</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The International Mouse Phenotyping Consortium Web Portal, a unified point of access for knockout mice and related phenotyping data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D802</span>–<span class="cit-lpage">D809</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt977%26rft_id%253Dinfo%253Apmid%252F24194600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0Mi9EMS9EODAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kouranova</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forbes</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartels</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">CRISPRs for optimal targeting: delivery of CRISPR components as DNA, RNA and protein into cultured cells and single-cell embryos.</span> <abbr class="cit-jnl-abbrev">Hum. Gene Ther.</abbr> (in press).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.9" data-doi="10.1016/j.fob.2014.06.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">A. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">K. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Conditional targeting of Ispd using paired Cas9 nickase and a single DNA template in mice.</span> <abbr class="cit-jnl-abbrev">FEBS Open Bio</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BOpen%2BBio%26rft.volume%253D4%26rft.spage%253D637%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fob.2014.06.007%26rft_id%253Dinfo%253Apmid%252F25161872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fob.2014.06.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25161872&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.10" data-doi="10.1038/nbt.2661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Heritable gene targeting in the mouse and rat using a CRISPR-Cas system.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2661%26rft_id%253Dinfo%253Apmid%252F23929336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23929336&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.11" data-doi="10.1038/nbt.2652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Simultaneous generation and germline transmission of multiple gene mutations in rat using CRISPR-Cas systems.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">684</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D684%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2652%26rft_id%253Dinfo%253Apmid%252F23929337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23929337&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.12" data-doi="10.1111/j.1749-6632.2011.06311.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">K. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A knockout mouse resource for the biomedical research community.</span> <abbr class="cit-jnl-abbrev">Ann. N. Y. Acad. Sci.</abbr> <span class="cit-vol">1245</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bthe%2BNew%2BYork%2BAcademy%2Bof%2BSciences%26rft.stitle%253DAnn.%2BN.%2BY.%2BAcad.%2BSci.%26rft.aulast%253DLloyd%26rft.auinit1%253DK.%2BC.%26rft.volume%253D1245%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DA%2Bknockout%2Bmouse%2Bresource%2Bfor%2Bthe%2Bbiomedical%2Bresearch%2Bcommunity.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1749-6632.2011.06311.x%26rft_id%253Dinfo%253Apmid%252F22211970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1749-6632.2011.06311.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22211970&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301289400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.13" data-doi="10.1023/A:1018418914106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bygrave</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grosveld</span>  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandolfi</span>  <span class="cit-name-given-names">P. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">The chromosome make-up of mouse embryonic stem cells is predictive of somatic and germ cell chimaerism.</span> <abbr class="cit-jnl-abbrev">Transgenic Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">321</span>–<span class="cit-lpage">328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTransgenic%2Bresearch%26rft.stitle%253DTransgenic%2BRes%26rft.aulast%253DLongo%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D321%26rft.epage%253D328%26rft.atitle%253DThe%2Bchromosome%2Bmake-up%2Bof%2Bmouse%2Bembryonic%2Bstem%2Bcells%2Bis%2Bpredictive%2Bof%2Bsomatic%2Band%2Bgerm%2Bcell%2Bchimaerism.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1018418914106%26rft_id%253Dinfo%253Apmid%252F9322369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1018418914106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9322369&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XY67800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.14" data-doi="10.1126/science.1232033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esvelt</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNA-guided human genome engineering via Cas9.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">823</span>–<span class="cit-lpage">826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1232033%26rft_id%253Dinfo%253Apmid%252F23287722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzkvNjEyMS84MjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjA1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.15" data-doi="10.1002/jbmr.1943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Motyl</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demambro</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bornstein</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Altered thermogenesis and impaired bone remodeling in <em>Misty</em> mice.</span> <abbr class="cit-jnl-abbrev">J. Bone Miner. Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1885</span>–<span class="cit-lpage">1897</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBone%2BMiner.%2BRes.%26rft.volume%253D28%26rft.spage%253D1885%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjbmr.1943%26rft_id%253Dinfo%253Apmid%252F23553822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jbmr.1943&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23553822&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.16" data-doi="10.2144/000112706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pacholczyk</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suhag</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazurek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dederscheck</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koni</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Generation of C57BL/6 knockout mice using C3H x BALB/c blastocysts.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DPacholczyk%26rft.auinit1%253DG.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D413%26rft.epage%253D416%26rft.atitle%253DGeneration%2Bof%2BC57BL%252F6%2Bknockout%2Bmice%2Busing%2BC3H%2Bx%2BBALB%252Fc%2Bblastocysts.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000112706%26rft_id%253Dinfo%253Apmid%252F18361795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000112706&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18361795&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.17" data-doi="10.1038/nmeth.1342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pettitt</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rairdan</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prosser</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Agouti C57BL/6N embryonic stem cells for mouse genetic resources.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPettitt%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D6%26rft.issue%253D7%26rft.spage%253D493%26rft.epage%253D495%26rft.atitle%253DAgouti%2BC57BL%252F6N%2Bembryonic%2Bstem%2Bcells%2Bfor%2Bmouse%2Bgenetic%2Bresources.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1342%26rft_id%253Dinfo%253Apmid%252F19525957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19525957&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267442900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ring</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meehan</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A mouse informatics platform for phenotypic and translational discovery.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr>. 26: 413–421</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.19" data-doi="10.1093/nar/gkq879"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ringwald</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tadepally</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The IKMC web portal: a central point of entry to data and resources from the International Knockout Mouse Consortium.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D849</span>–<span class="cit-lpage">D855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq879%26rft_id%253Dinfo%253Apmid%252F20929875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzOS9zdXBwbF8xL0Q4NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjA1MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.20" data-doi="10.1007/s00335-013-9467-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ryder</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gleeson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sethi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miklejewska</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Molecular characterization of mutant mouse strains generated from the EUCOMM/KOMP-CSD ES cell resource.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">286</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D24%26rft.spage%253D286%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00335-013-9467-x%26rft_id%253Dinfo%253Apmid%252F23912999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00335-013-9467-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23912999&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000323309900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.21" data-doi="10.1038/nature10163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skarnes</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koutsourakis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bushell</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A conditional knockout resource for the genome-wide study of mouse gene function.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">474</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSkarnes%26rft.auinit1%253DW.%2BC.%26rft.volume%253D474%26rft.issue%253D7351%26rft.spage%253D337%26rft.epage%253D342%26rft.atitle%253DA%2Bconditional%2Bknockout%2Bresource%2Bfor%2Bthe%2Bgenome-wide%2Bstudy%2Bof%2Bmouse%2Bgene%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10163%26rft_id%253Dinfo%253Apmid%252F21677750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21677750&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000291647100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sviderskaya</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novak</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swank</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The murine <em>misty</em> mutation: phenotypic effects on melanocytes, platelets and brown fat.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">390</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSviderskaya%26rft.auinit1%253DE.%2BV.%26rft.volume%253D148%26rft.issue%253D1%26rft.spage%253D381%26rft.epage%253D390%26rft.atitle%253DThe%2BMurine%2BMisty%2BMutation%253A%2BPhenotypic%2BEffects%2Bon%2BMelanocytes%252C%2BPlatelets%2Band%2BBrown%2BFat%26rft_id%253Dinfo%253Apmid%252F9475748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ4LzEvMzgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truett</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heeger</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mynatt</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truett</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Preparation of PCR-quality mouse genomic DNA with hot sodium hydroxide and tris (HotSHOT).</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DTruett%26rft.auinit1%253DG.%2BE.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D52%26rft.epage%253D52%252C%2B54%26rft.atitle%253DPreparation%2Bof%2BPCR-quality%2Bmouse%2Bgenomic%2BDNA%2Bwith%2Bhot%2Bsodium%2Bhydroxide%2Band%2Btris%2B%2528HotSHOT%2529.%26rft_id%253Dinfo%253Apmid%252F10907076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10907076&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088141000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vartak</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Inhibition of nonhomologous end joining to increase the specificity of CRISPR/Cas9 genome editing.</span> <abbr class="cit-jnl-abbrev">FEBS J.</abbr> 282: 4289–4294</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.25" data-doi="10.1016/j.cell.2013.04.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shivalila</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawlaty</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">One-step generation of mice carrying mutations in multiple genes by CRISPR/Cas-mediated genome engineering.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">910</span>–<span class="cit-lpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D910%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.04.025%26rft_id%253Dinfo%253Apmid%252F23643243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.04.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23643243&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318844000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.26" data-doi="10.1101/gr.184184.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasumarthi</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baridon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Djan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trainor</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A lacZ reporter gene expression atlas for 313 adult KOMP mutant mouse lines.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">598</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.184184.114%26rft_id%253Dinfo%253Apmid%252F25591789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzQvNTk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwNTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.27" data-doi="10.1016/j.cell.2013.08.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shivalila</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">One-step generation of mice carrying reporter and conditional alleles by CRISPR/Cas-mediated genome engineering.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">1370</span>–<span class="cit-lpage">1379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D1370%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.08.022%26rft_id%253Dinfo%253Apmid%252F23992847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.08.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23992847&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000324239300023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.28" data-doi="10.1016/j.ydbio.2014.06.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yen</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usman</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Somatic mosaicism and allele complexity induced by CRISPR/Cas9 RNA injections in mouse zygotes.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">393</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BBiol.%26rft.volume%253D393%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ydbio.2014.06.017%26rft_id%253Dinfo%253Apmid%252F24984260%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ydbio.2014.06.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24984260&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2051.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zangala</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Isolation of genomic DNA from mouse tails.</span> <abbr class="cit-jnl-abbrev">J. Vis. Exp.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVis.%2BExp.%26rft.volume%253D6%26rft.spage%253D246%26rft_id%253Dinfo%253Apmid%252F18997894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18997894&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2051.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/7/2051.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/2043" title="Exhaustive Genome-Wide Search for SNP-SNP Interactions Across 10 Human Diseases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/2063" title="Fitness Effects of Spontaneous Mutations in Picoeukaryotic Marine Green Algae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/7"><span>Volume 6 Issue 7, July 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/7.cover-source.jpg?itok=E84OOyGI" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2051" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-GCb8AWO0D6STauZ7u4EPx1jtFGuekP_0Ic84k90t9wE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/7/2051" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/7/2051" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/7/2051" class="active" data-icon-position="" data-hide-link-title="0">CRISPR/Cas9-Mediated Insertion of loxP Sites in the Mouse Dock7 Gene Provides an Effective Alternative to Use of Targeted Embryonic Stem Cells</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460434" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yDN2Yj8-P7daoHcPycMsa9BS0CAcMas7yYphwSxNxnY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/2051.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/7/2051" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-f6SgT14oZTJ6hhTRVL0vFz3QLSFbDNTG1A7pwHLYaDY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/7/2051" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/2051.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460434" id="node460434--2" data-pisa="ggg;6/7/2051" data-pisa-master="ggg;g3.116.030601" data-apath="/ggg/6/7/2051.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CRISPR/Cas9-Mediated Insertion of loxP Sites in the Mouse <em>Dock7</em> Gene Provides an Effective Alternative to Use of Targeted Embryonic Stem Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kathleen A. Bishop</span>, <span class="highwire-citation-author" data-delta="1">Anne Harrington</span>, <span class="highwire-citation-author" data-delta="2">Evguenia Kouranova</span>, <span class="highwire-citation-author" data-delta="3">Edward J. Weinstein</span>, <span class="highwire-citation-author" data-delta="4">Clifford J. Rosen</span>, <span class="highwire-citation-author" data-delta="5">Xiaoxia Cui</span> and <span class="highwire-citation-author" data-delta="6">Lucy Liaw</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2051-2061;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030601 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460434--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kathleen A. Bishop </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BA.%2BBishop%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bishop%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathleen%2BA.%2BBishop%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne Harrington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BHarrington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harrington%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BHarrington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Evguenia Kouranova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Horizon Discovery, St. Louis, Missouri 63146</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evguenia%2BKouranova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kouranova%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvguenia%2BKouranova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Edward J. Weinstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Horizon Discovery, St. Louis, Missouri 63146</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BJ.%2BWeinstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinstein%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BJ.%2BWeinstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clifford J. Rosen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clifford%2BJ.%2BRosen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosen%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClifford%2BJ.%2BRosen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xiaoxia Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Horizon Discovery, St. Louis, Missouri 63146</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoxia%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoxia%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lucy Liaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BLiaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liaw%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucy%2BLiaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liawl@mmc.org" class="" data-icon-position="" data-hide-link-title="0">liawl@mmc.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/7/2051" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/7/2051&amp;title=CRISPR/Cas9-Mediated%20Insertion%20of%20loxP%20Sites%20in%20the%20Mouse%20Dock7%20Gene%20Provides%20an%20Effective%20Alternative%20to%20Use%20of%20Targeted%20Embryonic%20Stem%20Cells" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/7/2051&amp;title=CRISPR/Cas9-Mediated%20Insertion%20of%20loxP%20Sites%20in%20the%20Mouse%20Dock7%20Gene%20Provides%20an%20Effective%20Alternative%20to%20Use%20of%20Targeted%20Embryonic%20Stem%20Cells" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/7/2051&amp;title=CRISPR/Cas9-Mediated%20Insertion%20of%20loxP%20Sites%20in%20the%20Mouse%20Dock7%20Gene%20Provides%20an%20Effective%20Alternative%20to%20Use%20of%20Targeted%20Embryonic%20Stem%20Cells" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/7/2051&amp;text=CRISPR/Cas9-Mediated%20Insertion%20of%20loxP%20Sites%20in%20the%20Mouse%20Dock7%20Gene%20Provides%20an%20Effective%20Alternative%20to%20Use%20of%20Targeted%20Embryonic%20Stem%20Cells" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/7/2051&amp;title=CRISPR/Cas9-Mediated%20Insertion%20of%20loxP%20Sites%20in%20the%20Mouse%20Dock7%20Gene%20Provides%20an%20Effective%20Alternative%20to%20Use%20of%20Targeted%20Embryonic%20Stem%20Cells" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/7/2051&amp;t=CRISPR/Cas9-Mediated%20Insertion%20of%20loxP%20Sites%20in%20the%20Mouse%20Dock7%20Gene%20Provides%20an%20Effective%20Alternative%20to%20Use%20of%20Targeted%20Embryonic%20Stem%20Cells" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/7/2051&amp;title=CRISPR/Cas9-Mediated%20Insertion%20of%20loxP%20Sites%20in%20the%20Mouse%20Dock7%20Gene%20Provides%20an%20Effective%20Alternative%20to%20Use%20of%20Targeted%20Embryonic%20Stem%20Cells" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/7/2051&amp;title=CRISPR/Cas9-Mediated%20Insertion%20of%20loxP%20Sites%20in%20the%20Mouse%20Dock7%20Gene%20Provides%20an%20Effective%20Alternative%20to%20Use%20of%20Targeted%20Embryonic%20Stem%20Cells" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460434/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460434" id="node460434--3" data-pisa="ggg;6/7/2051" data-pisa-master="ggg;g3.116.030601" data-apath="/ggg/6/7/2051.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">CRISPR/Cas9-Mediated Insertion of loxP Sites in the Mouse <em>Dock7</em> Gene Provides an Effective Alternative to Use of Targeted Embryonic Stem Cells</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Kathleen A. Bishop</span>, <span class="highwire-citation-author" data-delta="1">Anne Harrington</span>, <span class="highwire-citation-author" data-delta="2">Evguenia Kouranova</span>, <span class="highwire-citation-author" data-delta="3">Edward J. Weinstein</span>, <span class="highwire-citation-author" data-delta="4">Clifford J. Rosen</span>, <span class="highwire-citation-author" data-delta="5">Xiaoxia Cui</span> and <span class="highwire-citation-author" data-delta="6">Lucy Liaw</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2051-2061;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030601 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460434--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kathleen A. Bishop </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BA.%2BBishop%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bishop%20KA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKathleen%2BA.%2BBishop%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne Harrington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BHarrington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Harrington%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BHarrington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Evguenia Kouranova </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Horizon Discovery, St. Louis, Missouri 63146</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evguenia%2BKouranova%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kouranova%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvguenia%2BKouranova%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Edward J. Weinstein </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Horizon Discovery, St. Louis, Missouri 63146</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BJ.%2BWeinstein%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weinstein%20EJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BJ.%2BWeinstein%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Clifford J. Rosen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clifford%2BJ.%2BRosen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosen%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClifford%2BJ.%2BRosen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xiaoxia Cui </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Horizon Discovery, St. Louis, Missouri 63146</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoxia%2BCui%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cui%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoxia%2BCui%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Lucy Liaw </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Molecular Medicine, Maine Medical Center Research Institute, Scarborough, Maine 04074</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucy%2BLiaw%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liaw%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucy%2BLiaw%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:liawl@mmc.org" class="" data-icon-position="" data-hide-link-title="0">liawl@mmc.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2051&amp;rft.epage=2061&amp;rft.atitle=CRISPR/Cas9-Mediated%20Insertion%20of%20loxP%20Sites%20in%20the%20Mouse%20Dock7%20Gene%20Provides%20an%20Effective%20Alternative%20to%20Use%20of%20Targeted%20Embryonic%20Stem%20Cells&amp;rft.volume=6&amp;rft.issue=7&amp;rft.date=2016-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bishop%2C+Kathleen+A.&amp;rft.au=Harrington%2C+Anne&amp;rft.au=Kouranova%2C+Evguenia&amp;rft.au=Weinstein%2C+Edward+J.&amp;rft.au=Rosen%2C+Clifford+J.&amp;rft.au=Cui%2C+Xiaoxia&amp;rft.au=Liaw%2C+Lucy"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460434/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460434/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460434/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460434/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460434/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460434/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460434/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460434/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460434/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460434/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460434/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460434/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/7/2051" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/7/2051.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/7/2051.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/7/2051.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/7/2051.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
