<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/7/2103" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/7/2103.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/7/2103.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/7/2103.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/7/2103.atom" />
<meta name="HW.pisa" content="ggg;6/7/2103" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Improved Placement of Multi-mapping Small RNAs" />
<meta name="DC.Identifier" content="10.1534/g3.116.030452" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Johnson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="High-throughput sequencing of small RNAs (sRNA-seq) is a popular method used to discover and annotate microRNAs (miRNAs), endogenous short interfering RNAs (siRNAs), and Piwi-associated RNAs (piRNAs). One of the key steps in sRNA-seq data analysis is alignment to a reference genome. sRNA-seq libraries often have a high proportion of reads that align to multiple genomic locations, which makes determining their true origins difficult. Commonly used sRNA-seq alignment methods result in either very low precision (choosing an alignment at random), or sensitivity (ignoring multi-mapping reads). Here, we describe and test an sRNA-seq alignment strategy that uses local genomic context to guide decisions on proper placements of multi-mapped sRNA-seq reads. Tests using simulated sRNA-seq data demonstrated that this local-weighting method outperforms other alignment strategies using three different plant genomes. Experimental analyses with real sRNA-seq data also indicate superior performance of local-weighting methods for both plant miRNAs and heterochromatic siRNAs. The local-weighting methods we have developed are implemented as part of the sRNA-seq analysis program ShortStack, which is freely available under a general public license. Improved genome alignments of sRNA-seq data should increase the quality of downstream analyses and genome annotation efforts." />
<meta name="DC.Contributor" content="Nathan R. Johnson" />
<meta name="DC.Contributor" content="Jonathan M. Yeoh" />
<meta name="DC.Contributor" content="Ceyda Coruh" />
<meta name="DC.Contributor" content="Michael J. Axtell" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Improved Placement of Multi-mapping Small RNAs" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;High-throughput sequencing of small RNAs (sRNA-seq) is a popular method used to discover and annotate microRNAs (miRNAs), endogenous short interfering RNAs (siRNAs), and Piwi-associated RNAs (piRNAs). One of the key steps in sRNA-seq data analysis is alignment to a reference genome. sRNA-seq libraries often have a high proportion of reads that align to multiple genomic locations, which makes determining their true origins difficult. Commonly used sRNA-seq alignment methods result in either very low precision (choosing an alignment at random), or sensitivity (ignoring multi-mapping reads). Here, we describe and test an sRNA-seq alignment strategy that uses local genomic context to guide decisions on proper placements of multi-mapped sRNA-seq reads. Tests using simulated sRNA-seq data demonstrated that this local-weighting method outperforms other alignment strategies using three different plant genomes. Experimental analyses with real sRNA-seq data also indicate superior performance of local-weighting methods for both plant miRNAs and heterochromatic siRNAs. The local-weighting methods we have developed are implemented as part of the sRNA-seq analysis program ShortStack, which is freely available under a general public license. Improved genome alignments of sRNA-seq data should increase the quality of downstream analyses and genome annotation efforts.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/07/01" />
<meta name="citation_mjid" content="ggg;6/7/2103" />
<meta name="citation_id" content="6/7/2103" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/7/2103" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/7/2103.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/7/2103.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/7/2103.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.030452" />
<meta name="citation_pmid" content="27175019" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2103" />
<meta name="citation_lastpage" content="2111" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nathan R. Johnson" />
<meta name="citation_author_institution" content="Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802" />
<meta name="citation_author_institution" content="Department of Biology, Knox College, Galesburg, Illinois, 61401" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5279-9964" />
<meta name="citation_author" content="Jonathan M. Yeoh" />
<meta name="citation_author_institution" content="Department of Biology, Knox College, Galesburg, Illinois, 61401" />
<meta name="citation_author" content="Ceyda Coruh" />
<meta name="citation_author_institution" content="Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802" />
<meta name="citation_author_institution" content="Department of Biology, Penn State University, University Park, PA 16802" />
<meta name="citation_author" content="Michael J. Axtell" />
<meta name="citation_author_institution" content="Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802" />
<meta name="citation_author_institution" content="Department of Biology, Penn State University, University Park, PA 16802" />
<meta name="citation_author_email" content="mja18@psu.edu" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. Allen;citation_author=Z. Xie;citation_author=A. M. Gustafson;citation_author=J. C. Carrington;citation_title=microRNA-directed phasing during trans-acting siRNA biogenesis in plants.;citation_pages=207-221;citation_volume=121;citation_year=2005;citation_issue=2;citation_pmid=15851028;citation_doi=10.1016/j.cell.2005.04.004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Aravin;citation_author=D. Gaidatzis;citation_author=S. Pfeffer;citation_author=M. Lagos-Quintana;citation_author=P. Landgraf;citation_title=A novel class of small RNAs bind to MILI protein in mouse testes.;citation_pages=203-207;citation_volume=442;citation_year=2006;citation_issue=7099;citation_pmid=16751777;citation_doi=10.1038/nature04916" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=M. J. Axtell;citation_title=ShortStack: comprehensive annotation and quantification of small RNA genes.;citation_volume=19;citation_year=2013;citation_pmid=23610128;citation_doi=10.1261/rna.035279.112" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=T. Blevins;citation_author=R. Podicheti;citation_author=V. Mishra;citation_author=M. Marasco;citation_author=H. Tang;citation_title=Identification of Pol IV and RDR2-dependent precursors of 24 nt siRNAs guiding de novo DNA methylation in Arabidopsis.;citation_volume=4;citation_year=2015;citation_pmid=26430765;citation_doi=10.7554/eLife.09591" />
<meta name="citation_reference" content="citation_journal_title=BMC molecular biology [electronic resource];citation_journal_abbrev=BMC Mol Biol;citation_author=Š. Čikoš;citation_author=A. Bukovská;citation_author=J. Koppel;citation_title=Chicken ovalbumin upstream promoter transcription factor II regulates uncoupling protein 3 gene transcription in Phodopus sungorus.;citation_pages=1-1;citation_volume=8;citation_year=2007;citation_pmid=17204145;citation_doi=10.1186/1471-2199-8-1" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Plant Biol.;citation_author=C. Coruh;citation_author=S. Shahid;citation_author=M. J. Axtell;citation_title=Seeing the forest for the trees: annotating small RNA producing genes in plants.;citation_pages=87-95;citation_volume=18;citation_year=2014;citation_pmid=24632306;citation_doi=10.1016/j.pbi.2014.02.008" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. T. Cuperus;citation_author=N. Fahlgren;citation_author=J. C. Carrington;citation_title=Evolution and functional diversification of MIRNA genes.;citation_volume=23;citation_year=2011;citation_pmid=21317375;citation_doi=10.1105/tpc.110.082784" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=G. J. Faulkner;citation_author=A. R. R. Forrest;citation_author=A. M. Chalk;citation_author=K. Schroder;citation_author=Y. Hayashizaki;citation_title=A rescue strategy for multimapping short sequence tags refines surveys of transcriptional activity by CAGE.;citation_pages=281-288;citation_volume=91;citation_year=2008;citation_issue=3;citation_pmid=18178374;citation_doi=10.1016/j.ygeno.2007.11.003" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=D. Kim;citation_author=G. Pertea;citation_author=C. Trapnell;citation_author=H. Pimentel;citation_author=R. Kelley;citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.;citation_pages=R36;citation_volume=14;citation_year=2013;citation_pmid=23618408;citation_doi=10.1186/gb-2013-14-4-r36" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Kozomara;citation_author=S. Griffiths-Jones;citation_title=miRBase: annotating high confidence microRNAs using deep sequencing data.;citation_volume=42;citation_year=2014;citation_pmid=24275495;citation_doi=10.1093/nar/gkt1181" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Li;citation_author=L. E. Vandivier;citation_author=B. Tu;citation_author=L. Gao;citation_author=S. Y. Won;citation_title=Detection of Pol IV/RDR2-dependent transcripts at the genomic scale in Arabidopsis reveals features and regulation of siRNA biogenesis.;citation_volume=25;citation_year=2015;citation_pmid=25414514;citation_doi=10.1101/gr.182238.114" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. D. Malone;citation_author=J. Brennecke;citation_author=M. Dus;citation_author=A. Stark;citation_author=W. R. McCombie;citation_title=Specialized piRNA pathways act in germline and somatic tissues of the Drosophila ovary.;citation_pages=522-535;citation_volume=137;citation_year=2009;citation_issue=3;citation_pmid=19395010;citation_doi=10.1016/j.cell.2009.03.040" />
<meta name="citation_reference" content="citation_author=M. Martin;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=M. A. Matzke;citation_author=R. A. Mosher;citation_title=RNA-directed DNA methylation: an epigenetic pathway of increasing complexity.;citation_pages=394-408;citation_volume=15;citation_year=2014;citation_pmid=24805120;citation_doi=10.1038/nrg3683" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=A. Mortazavi;citation_author=B. A. Williams;citation_author=K. McCue;citation_author=L. Schaeffer;citation_author=B. Wold;citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq.;citation_pages=621-628;citation_volume=5;citation_year=2008;citation_issue=7;citation_pmid=18516045;citation_doi=10.1038/nmeth.1226" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Moxon;citation_author=F. Schwach;citation_author=T. Dalmay;citation_author=D. Maclean;citation_author=D. J. Studholme;citation_title=A toolkit for analysing large-scale plant small RNA datasets.;citation_volume=24;citation_year=2008;citation_pmid=18713789;citation_doi=10.1093/bioinformatics/btn428" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Okamura;citation_author=W.-J. Chung;citation_author=J. G. Ruby;citation_author=H. Guo;citation_author=D. P. Bartel;citation_title=The Drosophila hairpin RNA pathway generates endogenous short interfering RNAs.;citation_pages=803-806;citation_volume=453;citation_year=2008;citation_issue=7196;citation_pmid=18463630;citation_doi=10.1038/nature07015" />
<meta name="citation_reference" content="citation_journal_title=Neuroscience letters;citation_journal_abbrev=Neurosci Lett;citation_author=C. Ramakers;citation_author=J. M. Ruijter;citation_author=R. H. L. Deprez;citation_author=A. F. M. Moorman;citation_title=Assumption-free analysis of quantitative real-time polymerase chain reaction (PCR) data.;citation_pages=62-66;citation_volume=339;citation_year=2003;citation_issue=1;citation_pmid=12618301;citation_doi=10.1016/S0304-3940(02)01423-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. W. Schmid;citation_author=U. Grossniklaus;citation_title=Rcount: simple and flexible RNA-Seq read counting.;citation_volume=31;citation_year=2015;citation_pmid=25322836;citation_doi=10.1093/bioinformatics/btu680" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. B. Stocks;citation_author=S. Moxon;citation_author=D. Mapleson;citation_author=H. C. Woolfenden;citation_author=I. Mohorianu;citation_title=The UEA sRNA workbench: a suite of tools for analysing and visualizing next generation sequencing microRNA and small RNA datasets.;citation_volume=28;citation_year=2012;citation_pmid=22628521;citation_doi=10.1093/bioinformatics/bts311" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=R. S. Taylor;citation_author=J. E. Tarver;citation_author=S. J. Hiscock;citation_author=P. C. J. Donoghue;citation_title=Evolutionary history of plant microRNAs.;citation_pages=175-182;citation_volume=19;citation_year=2014;citation_pmid=24405820;citation_doi=10.1016/j.tplants.2013.11.008" />
<meta name="citation_reference" content="citation_journal_title=Cell Res.;citation_author=D.-L. Yang;citation_author=G. Zhang;citation_author=K. Tang;citation_author=J. Li;citation_author=L. Yang;citation_title=Dicer-independent RNA-directed DNA methylation in Arabidopsis.;citation_pages=66-82;citation_volume=26;citation_year=2016;citation_pmid=26642813;citation_doi=10.1038/cr.2015.145" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. Ye;citation_author=G. Coulouris;citation_author=I. Zaretskaya;citation_author=I. Cutcutache;citation_author=S. Rozen;citation_title=Primer-BLAST: a tool to design target-specific primers for polymerase chain reaction.;citation_pages=134-134;citation_volume=13;citation_year=2012;citation_pmid=22708584;citation_doi=10.1186/1471-2105-13-134" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=R. Ye;citation_author=Z. Chen;citation_author=B. Lian;citation_author=M. J. Rowley;citation_author=N. Xia;citation_title=A dicer-independent route for biogenesis of siRNAs that direct DNA methylation in Arabidopsis.;citation_pages=222-235;citation_volume=61;citation_year=2016;citation_pmid=26711010;citation_doi=10.1016/j.molcel.2015.11.015" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=J. Zhai;citation_author=S. Bischof;citation_author=H. Wang;citation_author=S. Feng;citation_author=T.-F. Lee;citation_title=A one precursor one siRNA model for Pol IV-dependent siRNA biogenesis.;citation_pages=445-455;citation_volume=163;citation_year=2015;citation_pmid=26451488;citation_doi=10.1016/j.cell.2015.09.032" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Improved Placement of Multi-mapping Small RNAs" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="twitter:description" content="High-throughput sequencing of small RNAs (sRNA-seq) is a popular method used to discover and annotate microRNAs (miRNAs), endogenous short interfering RNAs (siRNAs), and Piwi-associated RNAs (piRNAs). One of the key steps in sRNA-seq data analysis is alignment to a reference genome. sRNA-seq libraries often have a high proportion of reads that align to multiple genomic locations, which makes determining their true origins difficult. Commonly used sRNA-seq alignment methods result in either very low precision (choosing an alignment at random), or sensitivity (ignoring multi-mapping reads). Here, we describe and test an sRNA-seq alignment strategy that uses local genomic context to guide decisions on proper placements of multi-mapped sRNA-seq reads. Tests using simulated sRNA-seq data demonstrated that this local-weighting method outperforms other alignment strategies using three different plant genomes. Experimental analyses with real sRNA-seq data also indicate superior performance of local-weighting methods for both plant miRNAs and heterochromatic siRNAs. The local-weighting methods we have developed are implemented as part of the sRNA-seq analysis program ShortStack, which is freely available under a general public license. Improved genome alignments of sRNA-seq data should increase the quality of downstream analyses and genome annotation efforts." />
<meta name="og-title" property="og:title" content="Improved Placement of Multi-mapping Small RNAs" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/7/2103" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="High-throughput sequencing of small RNAs (sRNA-seq) is a popular method used to discover and annotate microRNAs (miRNAs), endogenous short interfering RNAs (siRNAs), and Piwi-associated RNAs (piRNAs). One of the key steps in sRNA-seq data analysis is alignment to a reference genome. sRNA-seq libraries often have a high proportion of reads that align to multiple genomic locations, which makes determining their true origins difficult. Commonly used sRNA-seq alignment methods result in either very low precision (choosing an alignment at random), or sensitivity (ignoring multi-mapping reads). Here, we describe and test an sRNA-seq alignment strategy that uses local genomic context to guide decisions on proper placements of multi-mapped sRNA-seq reads. Tests using simulated sRNA-seq data demonstrated that this local-weighting method outperforms other alignment strategies using three different plant genomes. Experimental analyses with real sRNA-seq data also indicate superior performance of local-weighting methods for both plant miRNAs and heterochromatic siRNAs. The local-weighting methods we have developed are implemented as part of the sRNA-seq analysis program ShortStack, which is freely available under a general public license. Improved genome alignments of sRNA-seq data should increase the quality of downstream analyses and genome annotation efforts." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
    <title>Improved Placement of Multi-mapping Small RNAs | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"whcXT8tAakn5BUaQdTGSaxl-3223YOg4pFKpAVd-u80","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460422","apath":"\/ggg\/6\/7\/2103.atom","pisa":"ggg;6\/7\/2103","ac":{"\/ggg\/6\/7\/2103.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/7\/2103.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/7\/2103"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/7\/2103"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/7\/2103.supplemental":true,"\/content\/6\/7\/2103":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aN1upGwbbV0lNyb+nn8vov3Bhag07SDEzByfnokwYxkTTGUBQNJbqbHn\\\\\\\/NFHA2T3aEJWUnE5Cv9\\\\\\\/\\\\\\\/MKBDq7k4FOptDkkU3mBcVNqfwrdbGu2wc4+2g+xwBoAkBQzhxPGMccVgS3UfNamHXAbpyC3QTRbC9yJw3Y\\\\\\\/Qxir0Qu6Y6HzoeJxzEEKiJP9hE2GrCle7w96AlA73PnKrwzqg\\\\\\\/X+STpBndrC4c1JDuGOhHBWjrbmGx2I2oxv0Ves0zPZLG0jn4JHA6wLXnpzsYNg1wozVhmAcmWtMgzwpybHU4C8wL+owLbd65d+nB0mSfOiBKEZK3jdhQzOoJtEpxh2LMhzTxK6v6lB7sZgjPyeAXyd47G6U7B7d4GcDRbKEaTZQTRnjl4ZXOgTOJcXaBx0v91crBqXSADiD4MCjPh8Nq3A5SYg33V9EkqxCwsxbHuUXNtxrR5DEzMrAtk35zJDViX2J\\\\\\\/+pnqXZC8YLbi1q4qrq4ya9mA9EFzv\\\\\\\/ycp10rpGdPbGq3dLibUAX+XjfYKkwtkROS\\\\\\\/seccSn\\\\\\\/oHE3jpS+z82AUt1wHrBsRrajOG+NAhQujl4HGoFAgBCAzn+rXhP9keJwmvQrupAliOqkp0QooMaacZpIrNw5XHbJtwiOUlGoL3izajuAKMWtInxxkMGQXpS7wuJAistsHdeIt76xBhOyIAiaGX6g2nwm9hSVR5ljm0psBgDgZ6b026hi8ykD1PMVY3Itg7ehicIEJen8dWEhm7MZpz7QhZEjuT1742CrWmkPQJLIdTpMqRnWnTUE4\\\\\\\/6UWuya5lchElbofY7RDPph0cRzp0XDoYVjaxnNxbWY1JLLoTM1\\\\\\\/xE3EbbVLsB8MuOOmMsek1p1NmvQb34X6XGJSXJXPp6TTT2\\\\\\\/cvDfXiICOg0bhOKJGCQ3\\\\\\\/DnrPExaE4kIEQgDd7c22JnmQlzy6LFOkgo1BAzc2iwnqmpbgLP3TdXHlL65ERGJ9gxuku7gm4Vg+\\\\\\\/yOHQjtTAGlSfJjeNz1IBRiSsKxK33kW72pMH+TsqH0zOIqRhkPnWiu9q4o0\\\\\\\/7U4sD8PYTRthiOTr6tifA6dEE4oKr6TftYss6rch73V2aTOEjfSJ9iafHmT4ro3AnVG4WoCJ5hokUCrGjbDUDGaQ86tfmvqVwz\\\\\\\/4T7aPLDGSRDzkEikHV4Q8vKFFBRu4NP65uzq6k1vJuCi4qJdEYXUzygZ0iavuux47eyd5hsh7i+wFZZSdQ3J1QnC0mysHCzgT\\\\\\\/cMk2gN+TEAbOTOW\\\\\\\/ozUct3tkKGG8jt3LfEuWuXi\\\u0022,\\\u0022iv\\\u0022:\\\u0022OOSP2N6ceqtaxrAKylzrjg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220b8721d03c2d0dd94eb19af70ca7d3039f1bdbc049b3ca4d3d12e1aa8922297c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RTqXt7CVkpmI5bk0jqIrRPf5\\\\\\\/AasByngg0dijLRSGSLVlwKtgwjH9k3F+iPOwdPkVlhKi5OtfJOj5FG\\\\\\\/myQOMZ3F0LwiDvskR5\\\\\\\/jT57gy7dMJTxx5JL3FXCMKaHA7Z1KjdaOChXByOO+6+EK4iNERHB5qJXveCBHVlqift8mfBGyjQTAeaoLpc1b1Zcv1l4BH8NBVhv\\\\\\\/Owgmwg2N4RevYzZXyABnTEgShwwr1\\\\\\\/+WCU05RRi3EPijk4Fy\\\\\\\/xy2Cz74++D\\\\\\\/vzL11pb\\\\\\\/uifm2QUXITid+53Hjr+T+TipQJY8JG7KYbE7ctPgd3rnL5iWgN6QZtZwMIntrg\\\\\\\/eSRANNSD00j3NVsCxCa7XOc9yroN7jnIrQsS+iCrl1b5s\\\\\\\/AIk6rRShAz00RPA0rwjKGdwatWnbFgu121iAarynRVElZoPmMKetNMt0M7iARjvQsM7+gZFv\\\\\\\/oLnBMkdD7s4y\\\\\\\/1LgwKiWjoES8IYDIsAgemG68tCHuWirLYuMG82U7uR3zzjDOyks1N8qhYqZXJRUmmP5\\\\\\\/WviaboBT9USPZ8fDwbQgU8AGzebqZ09bCzr5ol2xwcd7e47gyjnPdlgp7cqFPid1nrBSboo8PP\\\\\\\/I1YC\\\\\\\/JUQDUMKGJc+TnxdBhr+OggdohyzYMFOoKAXYpFNuV4ZkTynJcD2xFTaB8TODIDWxjZUFhBV6niqfsQEXLRvjM6wCPZQ\\\\\\\/fGszboeBkPIDlgJSDXSgqVwQBMuClPJ4SSq8Pi6ZPijGFXlrVH+XIWT\\\\\\\/piHxOHzgmj0vWIAAOuAyCTL1UZouCuZ6TQQ7w4M4Jr159JvSVE7xRbKJhVn+NTs7zZXgOtpcz3lJ1F+2ArR2n5OXrn\\\\\\\/OTsfRomQSuiQJCZwSMgxIQ9s+Zqi5\\\\\\\/+uuDN4VAhdk5SUE9ArYNfZ5zRyJ9ZFQqgZAZXo\\\\\\\/1NNF7ehU6H3zOhMhMVPEaux01rBwmnR3BQJ97iFg5Zd3JjBhQzhufLECtFEzEy0TLQloLZe8vXGntQ0RqiRTpi+aBXliPfAOovcZqlO3F7UA0ln75j3HL7KMzbWUu44MvfamKVA0Tsix3zoXHgUVpLV\\\\\\\/FbDr64yMm7RomR79AUtecmir\\\\\\\/Iro63SA\\\\\\\/U0MUHIX15jmPQzvUUuJF9M4oANrS3aBMyWWvC+yvcEfjTM4A8w6yHWu9d2QR71Vo4cJzWYqNgB\\\\\\\/yky\\\\\\\/\\\\\\\/H\\\\\\\/7KB1Db+CZxxMwUaz4Fq74qrM6iU+nYH2QxevWGOZZHGsrNwff+yqMlTSbKh06htdWyb4zsFRZek3cvbXVB94CMsHZAyI6L6cIIbikFG2X71duqBjo6KaOC8\\\\\\\/loh5VYylOvUh6NAGvij\\\\\\\/h+Zynk9y7sr96MmLTnclaXt+BiIrF2dq5kvhfXxtrGRu57iJduLJUQXAqiVEU7F+5f0+e2iFntqBMJsru8AesRZctWPwT1Rym2F0nXQ6o07ejOrIIQ0ztK11jgbOkaDoys9hQOAd\\\\\\\/DqaunqAl+j3T8MLJPPx8YLs2JPFHBYUb5UJLIKfiCToy1r0ndCEo5DsPVQuTeIWrg2nGCfQw0pEpcn3YOoD8Li78Faadd91hcb0OCnck3d\\\\\\\/DZyxN1aKM9HvExrUbzV1P1VnpjsyKhOZGmuJD5IcoQWFjMclJcXB8\\\\\\\/9+4INh8QeK7p6riXjwVR9ZDtaVTdtO\\\\\\\/j2boR0G2z+G4GwPzRXmeTe\\\\\\\/45kKCdr6OI5Gdhn4BiUvN22Gb6Z+fQH6Y9H5\\\\\\\/VgDhSUVmu4wnz1F1MSJbIw9M7pMjlpcgapEPy\\\\\\\/6GpyG\\\\\\\/WA\\\\\\\/QLvWM0j3QuzkXhpx+vOZoOUjRUzq6658LBQ5k9GgTw+zM3flQDYJjG900Sf6HcCQzM1uhUmB2w+0bq\\\\\\\/LJPIrNFJB0WjuQzsdAtEgKZYBMIjHXlnS3pZ3gbKvrDBVOM2VpVudUlV5TDT9lo9ESm0eYvmSnIY8zlYEPjH2bV55Juf5rq5LTLBwRQ1J9ibJQYH1EGT2CuCzJH9OXH7Lbho6j4LbdB3ALxUbt53\\\\\\\/B2tKgENGBpCcxcZ3NeaU+LXCeStmz39brIg4RRAJ4IkXIb+sY7bqeNYmCyHnAumZ9FVcvoO8kCsJuHqKFR05lMWX9Dvq8gdpt4E1701\\\\\\\/\\\\\\\/SE4iQS3Vl06o1NB2BuA8OBIK4PezFXNDfqZc7X6Rh\\\\\\\/d9JXY6q1NDNtcjbI1Db\\\\\\\/3hiZ24oeEKXXTvNqnQQXFtpmY6+JI+UrqZj7fMFEOUZnA6Rn+FsXCWY7GN\\\\\\\/M5MHcVFMklBWQLwF7a\\\\\\\/9M8cx8NZiaoRHqPuqAGISaF0ODmM5C1TpJ4xqLbzPcdHo5X9xRcI6g+PzJ2nak3E9nN1hcwHdZ9AVLygbAPGGafZH8Wq9hG7f4t0nvc0g88KRwCTCglvPiQnv+NQ0cpd1bJSrXEri2rcQdr+uLHgYXPyjAWwk19fKT4DAd\\\\\\\/kQwyX6JcPKHjHpp5cYFV8q4XF17mx\\\\\\\/OqiZCGR8z4Qm1P6wUg6m31CttMqIxEGxx8fhA9EX\\\\\\\/bfDlgg5rVwA4teNU7f4coV+168Dr8X0CuSs88eTIrRZp9U3IIt99vuZwEhkRiO\\\\\\\/TCb6ofHxfIpH6dc24NYVmvAigD6+8pGeEsi17NVNVKshw4HVJu5S30ozQbWkV8za0gvUNkgB47NQbm1RDumYVUFBS9Oek8J67NzRC+Iy7bWQmnvNHd0gPSDxR7sbZ6bY\\\\\\\/Z4YibofKT+g59IfS7oPq3AH5Gy\\\\\\\/Y0AOMHN9oy7cqj7HXV7keb6OpUD6B4KCJQYveYU7DPQP8sC0E6+lnjKGzNj2r2O1gZ2g77LETRc5LHlSMDjPbKdBWGHPH9uF87RwOCMi6cIlkGRYjyVap9AXZdQo3MnhR906qdrmxcJHYPDKfhoRaPwRI576wn6MlWnloZgD+wn8sgODfr1xWw\\\\\\\/neF+e\\\\\\\/gN202U9cUm97Lwvn8r39gk\\\\\\\/pvYBua11oHqXUTVNFdw1gSqSCAj7woi7qz3roUMdbWpd3abJ3hBUxEb7rlJHx5n0GFMoaKbKQMr8Mmu276ZUfYeL7XIcT5ZX7RnfsGwwGnxcI+b\\\\\\\/MJd3xgMSZqJK2E+zvokzHLeLcT2CABwsudywUzSqlVf36IuMdfhgzJpXrcVrRxi9l36+IgzTEh5zL731laltb1N\\\\\\\/FxuK5mZH9es+3fd26ENIOaLduG8iCSiWH1KcRe7JtD2BSMe2+0Yzs9sHSavPJft+56H1r9aTeMFEwoWpFQKHtPWZ7MlespL+CnKy08nJ5UEFyi1iKD7fhHsaI5ino5Y\\\\\\\/c7UcD0wMO+v7KWhv0o3Y3QTTs8MIopBeV\\\\\\\/bvEaYvPpv3d891zO5QeC3jAQI1cDshgj567ck4Zp+34fDoZq8flp0yPRlNlB9smsvBV3Uuc+DsJiXp3k34zMv3663mGePAPszkUaApMxa9xbXps2zniZqYtMIcp\\\\\\\/YlfY5AMRyWpdPrJaXSBZmpI0N7NfwfzvOGDDs0rr45yJ0ynPKao7\\\\\\\/+d0Q9EjnDfs1tmDUH0v3LUQWGLmpwDoEqw6ftxtiUN3+4i\\\\\\\/tRX\\\\\\\/FaZSSV3QQPECKloOPebltaPLlIkNcRdiGK6X0mUywvOri\\\\\\\/cQutf6n1WmjY\\\\\\\/DzLuJ0jepXTJNb3X7z1nvxbyGWPEN2Us+TJS4DVWTB12mmc0Q0inttES+iB1ReuEzXs8x3VijMZUy\\\\\\\/Bc2xg+9PQIbOCc5rjQm5iY8E53EuSRR6D3X4d8cg7kZAGFzP6vt4vYWgLv7Fsq1\\\\\\\/m0H3DXibdqimc\\\\\\\/\\\\\\\/uZJVKOfvSIrbjakL0duoCjujfAv0wsGgPF\\\\\\\/yNtub9bYdkRIWNgx4l+JWTAxQakgyghTuxx\\\\\\\/khYwvaEbZI9ll64DQ8prWMigL1IG9MnGsQHO3g7AR76xDsxYcWAhN7WH4sFe9863CLWSaY+6Jpaa9zKXGFayk8QRQ+PrdtsFIIXx3ldCpvV6j44BlFJu\\\\\\\/z0aIKE8zUCVDAzJjVRY\\\\\\\/JL\\\\\\\/ljvnlrOtFLEOIaIAiOWSpg39UeXfasmNTPmaB2xMOoe3zwvd8X26S8XAE48DJu270wPczrOuml5l906v8luj4FICBweN4708x2jilBYcNb+8Ept8efSLCuA5lDWTFiN\\\\\\\/sm2dmlxAWdcI\\\\\\\/aeA+BUXj7Pwh5r4qytGfwW9tQG7B\\\\\\\/KFTwC4U2ZA+ssXVBulNNn6Aq2sHlV+ViqPCLLheo+fTQGgt+cGLz4QA+MeatOevBp1MNNXQ9DmhIs7Go5Z\\\\\\\/YeQbyv4m7FvXF19MS+xIaO\\\\\\\/S+RMQDpF9KEKzdar8uYmwlxKVbtK4ustn\\\\\\\/TWqqvzZrA+BJQu48b6XI0B\\\\\\\/QDwJHOcuZIP3uZNWtemAHGMZcROE\\\\\\\/IfwdVWFeBriSMaBXlLud0nrDsXoQdH4TDDlgyu70YpsLq8w7Tw0P+4IVSeNHmlIxSN\\\\\\\/6ZYdmFn9YPcwWPCMwmfsd\\\\\\\/XE2k7J9lBSlteJNeIjNj8g33GKStoewd0Vu\\\\\\\/e1ugG38mX7pPiZ5lHP3CgwN\\\\\\\/P29doYm\\\\\\\/I\\\\\\\/40caQxsk8mU360qfwI3ypovwm3XARckZQzzBkPSodlI8uRqw5adi4MSoBE+FAwHQZXTImPYYeCd\\\\\\\/0Nx+NMTptcGYfG0rM8xTGyWEFtOA0PFwebdVRmBFlizaL5l6Rqv3cFNNpezICTNX\\\\\\\/\\\\\\\/NLxr1Y7KxmGZr64Whva2ajU9A9oLxWOJxniRbk2umEkQ2sRELxYT+LPDhi7RzuEVGKestlo0hj1J6t6Iv8YtG9tN1KV6d+dJiVbCmpLN4aDxu\\\\\\\/14Qzl9MXcaOb5BmkG54rcXqHc9tN\\\\\\\/p9vsqySlsQ7HUTWSv\\\\\\\/\\\\\\\/h0DsDACwA5D1IY6aBHp5H92xj2fk2A0jgzhBWx+JtKEsL1yJuthX8JO+24KCpzf5GMmMTHMec8Un3W90vp6StDTFzGFNLl8YQOStS9RFVfta5ZjCSQKUkBvt609T4+AT8fqMjgrODxICS60g7M4LHmmg1ZtG4brBIXUaWZSevxg0JSCHgma4nMnkIYQn8iYKXmoHrKCUjOwAZrBVy3FdFmaNrh2TmHrlCV7yyUmJLO20j8dtlNol\\\\\\\/Aw25VrLFtBN7d\\\\\\\/RS7D5\\\\\\\/io+u+Sd33fizsQWqUuBemn6EomgblPJtlweRcL1JoAcQxXD4YZ\\\\\\\/j2d7NxKMkLSipG2WjTcEBV3YxsAcZ2fUhBUeQvICRh7QCyyxZvn6q38T\\\\\\\/qPmG++njf55tj1qubGelTAGtfC0t8\\\\\\\/tngLD1XBHs\\\\\\\/Os9U7\\\\\\\/Hycb6Aem3FXcF7wAtjAT4zZI2A4rWSd94J4UQksprrpD517vdfF7LWApQArUN9bJViDY6bgD8cQMEoVbIwSxmA0F6c2c1WNxCSFpL5v\\\\\\\/g2UzLwETSZ4wiAnLO8sRBS7fcXtXCPRmQ4TxWmvS7C0AE5ndza0pXWvt7ozgki0Zf5rJnxyT+bvcAv2phIBi2UJGMnhgMUOtdz0+OcTRqCj55WkM8sxE3j4ZS3+3C\\\\\\\/5vk0lBtYee\\\\\\\/LaSHk7irg9bzJ0grcj+gd7XeKdXzTLJsJ5ippoxhnm8RSyorshAWVYTBMwWSGT30whsai0tMZtLlpQMMz1JHjDJKrhzkJGYs52G\\\\\\\/MNFDt1B7kPrXBBW8qP0wtc4fyXxa+IFSiyQMZnBCIoGJEwLMdMwI3tZMh+co1Fb2v7fJG5LD8GazNGfZYOV3GJ+seFkccGYBNAbO\\\\\\\/zdPFouan6wM2i9ytuTmhl7s+yGwT6wnS87JlaaacIZ7wugOYGtgv0+NNC7l+8\\\\\\\/a1krC9ENmLXtlAJOTzAJlaKV01PfsM62oq0IelxmlgGpz45o\\\\\\\/tc\\\\\\\/\\\\\\\/4QHgxVsYz1gpM2N2Viw0by\\\\\\\/d9SpH0H+wHanv9n9le8arNdFzjVHINCH6VRmCLWctbuOsHl8cxskJSMWonn3EafPmOJDucE3uOj0P4za1yqKuoSt7KY0wrW0sIngtf\\\\\\\/vRy8Z9ulNGRyOy2eRl\\\\\\\/j8UAOY94PyimngadWMQBnaRDw9D2GkB8TiUIHPmvzpNvahe3h\\\\\\\/VMkC5OVPoJlmkFIhVQIuO1LeGSLvL5r7Y2Wl7mpbiMZLFEZSxfK3pcfoeR9xsZnyooUfeTA5Srqp8kq1aHiOnPjZmFZgFOwAsWShjoTC74fdEqow9UHrf\\\\\\\/6CM9HzlWBNPf40lauwhcecG0\\\\\\\/FCKZyq9zf61pI4HHZpGfuAKzcnZh4f0Uw+TaREj8NWyN97D0Mh5dCvPpXhBDsf7\\\\\\\/IcPy6vmfkV05HzCat1Hw4yQJzSB\\\\\\\/dxkx5S9\\\\\\\/0pdzGv+oeyH8z5i9\\\\\\\/W9UJSFa3YRQNR5Fx\\\\\\\/VY1pAqjRPCQZm4HNN2fm3x8wwOc7zwY9zo0Sk\\\\\\\/ueAfkkNEL0GLEgR4F5wz3EXjIfToBbnL1XYQOPZFaaVGOScbDLvbTbUMgTHkWWHIxU5fFggtMIoWmdXgpV0CpzitEEE9x2bbIIunvhSRxIaaOIRoj+y1l1q8eDudz3XXtenBzvnByBt7DfMzQJaPNeeNATld1iTreIIIuPU0WbvUi1hj0fTJwRuQk2gnAyT4WEIILBI5rAqWa8JyPkI5pTSr2G2pT+LCVocMj\\\\\\\/+oaBlBnTo\\\\\\\/HwIgQjMg\\\\\\\/FVlRxO8sJmRNfxA0AVcaVfpzx+B4L7+c\\\\\\\/rQEwQt8oNyHPAaawdtuKuUmv8IPr24AId+B5Za7mWkwL1nVZxEF7Vdwgbcg9f+oG0jCtP5VINI6Fs5X36kTGSPzCm0wmDnvLK3d+g9z3Ugii89Hpgdj60WnD+e9YMy5rflenJCaI\\\\\\\/p5WzjECl7qJuiWLhb+TIAtb4bBfGlV4q9Qks0gD3IrG6k7PgMC2LvowboKfwHOyBCYs\\\\\\\/2E3IAdxCyGFZ1HNeD9+tmEWuc4ospk+IEohpMm5zyw6YJGsxZXCF8a6cnN9eHTtdy7ytqH+s\\\\\\\/EKV++Ia3cigWfKY1rfgSIQtIuyYa\\\\\\\/8bhf74nm\\\\\\\/6hbTTHgbQVvEIWd9KkvxjBKIbB1IpotDu8v\\\\\\\/tNyWG+z\\\\\\\/K2jFOxWB4m\\\\\\\/BuDT\\\\\\\/d+E9wt6nqzUVvH8kbNx7+bpYZGt\\\\\\\/TtxpGpMEx7eOrc6mLV6oMLwa9IZbsPkB7Nx1IF3T0E5So35Eu6mZYy0w9JrM3JXDhX2pzK54N3T4QcPpl3qiouL3LujQjyhTmcbuVqnHd\\\\\\\/+VF2iuFlhGKBt0uoSP0gIh0K4JMv5Zdc56OhIZwG1MSxds740OCggJuH120NIkttQul5CT\\\\\\\/R8+Sp+xu5MwIessL9PqIPSDGBTUh4Sz5acaAV5xQtXMDqehUIKieWCGNKtcKWG6Br3kbhMNTJ5PYJ1RPGdq7Z9xt0oE\\\\\\\/YvB1meYXVaCZtxwZP5C7pehoWsw7NuWbxSIxL4iA4NrZMglTTfrhkUtcL5PgjxcUXu865qyVn10ANUFnb7kS4SW0\\\\\\\/+WhpxVV7+t205AlC2ScZ5z80v\\\\\\\/Cqt135V6qZNQd81iwIvfV6SKVY7Ju4R8LjpTfjbFRVYMSpgMmWz9e2zRwanRq2FNgeMwRJxzp6e0Q7d8gP3f8MDTalbZ+rC91pS1SRZPVSuDq6EWebxjtd47yB53ISTJw07rYcR+Mjx+Ez+l4dEhVjtRgvtlNhlBSsjYrs4Z929TeRpoVp2MRVb\\\\\\\/UyZgQHvUEHgWfwZWTI5HWxDfG8i6tSYN\\\\\\\/nH4yQH8Hw1OPu6GdVscYAPqEXZDfrsNAJxIqNJw907CJWb0H3GN2gzooTcC5afPEXrGwZ6jBFc5v9apboFrYQXrUxz4R+Tio2H0ebU6p3tHJbTcWiiJwSTehiJzmUDN5fdk2MmIA8YkdlCSD8NHKzJmVntHDD+EcsQiWiQa+vBcxindIUabgQuAvQVE4yCsHKk9MZXtG\\\\\\\/XN8OMlKbRV8q7+n7zAsqx189WK+ezSTfeEGgZQdZNwJb2JaRXKszRCCP0HxQOgJ+2vmLyfujHl2znH0aqSd99lHuPbglSmT5QH9XNqSNvg3\\\\\\\/jF2uBGD0co0L9dO5gna9ytlHOGAZKc8F19zGtS8EA2H5sFx22Ghz4+0RNoLTKzaVHplfwYP0e2hxRJuDEoSN5aYmJF7dp+usIqdS4uYOTWzhgvrLB\\\\\\\/E7c0y0wl1TbKLx9Rq6q489SIjUPExypswrCIb+Mki\\\\\\\/xCQCxZI4eeykKTuo3dRSt+WVDMXBX+lmabFd3N\\\\\\\/lV0ymiFjP26NQdpbnFbB3zK+nTOaODLiA8WuYZnbay6OcpnPeukmOK9TA85wzi6p0j11aFdf0DOmtyjvBZHCyzI1yHl4prq+PUCOXEHB+j1Rh1YYynvfOw9l6ZWBdKyfZKH0wJEZ7XQyRcvmDLn6xmkGBBFZk3XOH8t26Uw0gfJ0mJ0OjE1yDTqOddLRMOrRKRVYLW103uD841m7KhP65MDgysSq5wYdNgIdG6SjJiikmwRGVac9NhMlg3ffUB2KuMRWKM\\\\\\\/K5nUyiBibDUKmb520Fm\\\\\\\/iBmZn1SJIR84GfY27w56cxcTs0yfSMlXOJBFGnxd9DKNdb8BcyOHhMEzhRITM4NLryOkIIMPpBuCWP45pwP+JpFVrtXnnJgIr+wgXg1M5dew\\\\\\\/mTNVExYQW2OG\\\\\\\/ryY7SM3QTNbctkUTbeDB\\\\\\\/Cpt\\\\\\\/oR25ugMYe3y47beRfnW3sBVlLGydD5LwB82NBFWouOj60HeMXHho5STtQuWvgXmZGkXlMj2BqgrFKw3Q8xtYg13JKFTInde4HOtfkZEX\\\\\\\/STMaDe57cWNdBbs4l\\\\\\\/qlxLYFMKHVbykIPy5oaM5L7MYF5iGBJ3ZWrneg9Ey394bW9JwhwY1OT7eTV254RxEtnra5SHfwRIe0Uh\\\\\\\/Gfc1\\\\\\\/ibOfcMbiur8xGaah3hDlfW1sz70E+dPIqTPZxKB1AD+tiCo4TRc2q8c3li9+pw\\\\\\\/OUSjcZ8YjYmitOEkx9d3b\\\\\\\/q8PblEJseX2li9hUSbk6EVYc\\\\\\\/2EM9dP+UCwWJbcdlATKwtXsjwxVsa17Bm3j7bqyF19Jng1KGkFDfbRMcr9QibARsHM9AYkNu3gbFWyCnUn8mYDleap9KQ4QdjIQUWYfvpCFHwmqU7GNyHFAVvzxQsvsN6mG5P6+zYB1uZdRTHWumiCke++9qxwcpJxOWv598zYfKXsmkNQ\\\\\\\/EylBfMCr0XBMLAEG62C82gf13SGU+m0HBwah6UNdcQYVjFVtX4qRwSfljWF4c8t7fOXbihAQmOLLvdcYJqNqfWaIy\\\\\\\/DgviozESlOirbOziziVWxHZoIkrKKDnezlLRu+7acCIVIwQT4OU8LlamaDzgtXZ5FY2vXfvc+fOSuzqcME7OWIIWeSfAat8gUKHQ1TS+2w+T\\\\\\\/xDqcpdEG1wO\\\\\\\/0Gmmjc\\\\\\\/n72sum1I1BR439CVZJu4jgOmyMrvocBVIo1YYLvT+FGZzaJEjF5H2PQUPVbypm91+b1kKggUvb4ZP85+B353vKWWPhgG9fDvCoL82U3WhQ9duJ7CI4XPcsKKEPo7pvTPNKIfxRT9tnGJrMVVpCOTx5MVX\\\\\\\/JvVz0jsBw2hq0nWM0bAj\\\\\\\/ZnNv7+PSd80TDrsBD0VrERggslSbmIfsKdiIQjtGF1oZnT+hTQnzkEuN6qjBuDXUS985qP8yRgwzCDDViwhDp56rdQrwPB\\\\\\\/xP+clVlZsLMUKn+vZkb\\\\\\\/Uu9iKiqBp59cdmopMw+jgkYiEmfs2NRSs1J7fDA2KPr5Uvu\\\\\\\/852rofBUtqVfTiBOgXPbqtR5S6\\\\\\\/z9or64niKYa4AhZlAOVCdu3lzcSW6SV\\\\\\\/H9KlVa4ZfXe12MnLfYNRZBIUwYDIV6Rw9gjnEKjUuhphiA3ev4d2OrGeBC0weWIyglKFYclKazgtjMgnPwJdERJCKHgk4tdrWfPR1BS9K3AvN5ySKKTiRgHC+mscjd\\\\\\\/zyOYKWuVSG4+zIBklicxXasc5U4+RVyNCPyxMQp6H5GymMzugc4OB8xBSWh8BAsw1vSwV60jACZkONJRe\\\\\\\/TU2MuX1GOgSel1ObSeH3Kc6tMG+\\\\\\\/myuVhmkAwh+ljb5COYLuse7lWI2UEfCr2NKaW5UrUz15ttUW9BMnkDsY4iDx19PeuQck+RhlXkMQdi2cO5a\\\\\\\/ILsfYC97QYK+LKlY1GeMH\\\\\\\/Wu9QGCqfTvps8doAgX+lezRjJh3Bcx4\\\\\\\/19T68ebHbaCv8jZt3Oz6RT\\\\\\\/zVCTM3oq2gwU32PVPLnKJY3CETQNdEa3t381Xkj\\\\\\\/zMxg8lyoJZ4\\\\\\\/A7lnhfzwJIFZg47wFRPs0G1GvtqdLrKjPHviOTRf8ddSQFtALuAfRVKF+qtHP4Qff0eayfHk8+zSBW0Xd6LriH6AvN7DpR65AXh4Mh9wVunbsl6TbFRpSKZHt+rAraxBTRcslKeFy0ZikpARCdVei\\\\\\\/WOI0B5M1e+47u69dIalwy\\\\\\\/bXwYjxJBfJRfbem7YeJSmgCa3kTyE5bchzKJSN3BlIP9TuNCgyHx4IgI1wXznvZuv0ak+UuwyJcc6qgJdD2jLu2q+jwAcEZT8DYd594N8xyFUlA2gsHv8J4hMZ7PO\\\\\\\/rL8oNHxLUk\\\\\\\/lMF13gdW9y99QZzJxvqKejq4zduTyIrsZ2g4lrx3zRMSN\\\\\\\/OIQnOZ25X6Vd\\\\\\\/\\\\\\\/pDZFNPMt\\\\\\\/X5EW\\\\\\\/1YQIBX5Epc+PB+sKrzf2AXJr1O3gQZUiC8HEUTs5myMuHenKcyHH44Rx0Q6gLRUiYpSG0C3ulQ0pIgsPHn0uKqH3MKJFnJezYe5yKLjLPkMmNirORdhA8phF4cbky1\\\\\\\/rnQ4fQKt2\\\\\\\/4H30BGX\\\\\\\/JZYIHgb0WkBlYQiJpdxUWlgQ9Q\\\\\\\/zDkHlHYRpZ5oWYsLpgGTQnF7mM60crZw2XgYOxAd8VAmrJl1TDOLi0md1UjT59KY4eJAxB3MDbarWLQkE3K7mnoxKA8lz5IMgPWyy81yxfd2A1mLHEFTcXZj3lmFBaxZaDqJt2yuQxoG6QTDBsaff4HnnuIiucqMRYeg09crgrOaEsesCvcxdPYS40ZYsJa8h8afvHRTQb0gvnmJkSH8fTe811DulIQ\\\\\\\/Xi0Wfb+HvrZAzR+6VEGh7qjc6rlcikZyEhr7a3NkTejRXjqY8FItn9z2ppi433S3ZijXES0UkH1NWQ4WCe0AjVL0zBo7+XdfPuyBGmHHZ3wRUufeLLKXyYGuY14IvzabbA1rFOKUNuN2BMXmVyJy4AdNAOJKV7tDYDEv5TWM34gtUNY5xiS2oQBUuJCKZUcCal4SafvTfEn2FVC3VIFX4J1yt6o3jr5+i2lJdTgN00dxao3iwpOVyalX7koLErNkevgp8Jhs+YX4R9wAWexWF9Rg+sBS6L8Ci+2GLwQ7Q7b3B+8EkRNIEatmXGQvQoWXusnklv+ZRyvzFTMuRgE1EaQzapZuMTW4qw2diuw9UWvVPO+oBJ\\\\\\\/Ybdo0ChKvsgVXGDwRx2QnvlkenvZKMOwdc7Ht+7epbr6xk490uTNm8b8XPuxBoSrKzwyGpszuyYGrTFhAY7qpdZz1WSi+8CDuzIKLtRpAwrjfNxpHpEcP4zf5kKM3Ef5S9rf4lsnykY\\\\\\\/TvC2EHYbmxRKWf5A7DR9W4fJ8LmKWIP7zulag3U5nSVGfYblzmIPzNKhsgt01T1OgDTL\\\\\\\/E+SYoGLUoOXUHd54OLf56H3aRdk+37\\\\\\\/7KL8OK++ruRNTB5Bm1vqlpnQx16pp8Cwk6CvHiahwtrvBOrK5C+8nlbARQQ1DTYH8AgGqJjTpv+RkwtyjyiAojJ++gt3EiRKCLBwVEDsNuC0pytxVrHlSLP9+yvDSPpYMLM\\\\\\\/QJDETFinP7oXddopn2E31PLtosSixCIsuOhvhusTFV+g2IxzZGIbxjnW4eCrB6WhqTUZ8Rh6D2maQX2tB+BSJxlRauIvp3MSvPPRjzAJKAE2uUKsqa+ihp8dSONpgQPFe\\\\\\\/Dsw6L1ST753DJSYwVIQ2bN29AE3NSibVlR1XYXjKtBCA3hgtCCA62SWdMoQLgP5YhF5EC7l9l0sO20A1JDSALrB5bPgGuZ7roLIlto0nh\\\\\\\/6otYSb0kYfAMu2Fv+Ioz4WNk4UrQXBdY5yX0oAVKZuF8WNAMmxe6lZIRR38LtrcNckaTZvxdy4YYUql+AQaQnDLhHPvX47wU2SWQNDQoJNMhBT2N0tPbRLeR1R+QjzvhAbvcBh1\\\\\\\/sKM8mCVkB7MXXEo6mFJYrcQ6Y4LbKYe5ZDnKUv0bFF3IlXDqHZVJCmHwi3GIKoRxMyGg+m9vjoWXChmZxfYPcLSI2ksJkF1mrOGrB63DnNHa8+j502lap3BEojXqZKOj\\\\\\\/lSyH6xE+m7PioFJP\\\\\\\/saKFVvrlw750Cr\\\\\\\/zw2a036sg8nnDAYZDfqVa\\\\\\\/pMD6UIQPrRtPSR8P2ljatn6a9UZuveC6JhNwnK6Hp9XYfA\\\\\\\/JZ4IgDfZrnHnS6gwJHa21Vz6Wnee3hjEjZdl3CJrjrlvub043pBkUVVhctMRGUcmCQrzRzhiKeT7QUWgMcYl\\\\\\\/GpusjiEIirZjlx8bgGcRzu+YvKBe0toV2H4lVsCqFy+eOLdrOHwP5lClmVve7k\\\\\\\/U255iKqVGuN5yvDkfvFkZYFAnNLTiz1l3X8r7ANbJuGA3w7ktThIeVyoUEpsPcS98zPR4cSq\\\\\\\/BUiETF1+apQTmqbajIn2gTG4fgZFmia+a9WBwfnBIpbacTyBe7gsu6UPMcezg9fHngf0QaPoFw70DUEbxLFp05fjpdP8UH3ERjBIlDVO38iRj5SDeNfpNPvJvfMvhT\\\\\\\/zU+aGQ+ZQi2\\\\\\\/wHTUIDu6+rVT0Vqr7gIfFzHt5GNpnV15bd61QCwcOqi8nUAHphbQZHTKmtWzncjRMSLOF3O\\\\\\\/nHMTvxVtvy4oO0EhRM\\\\\\\/\\\\\\\/g59aRqsGpumbnSbrIHdJv7VeriMV7NnUUp0KawDK9tduG73tsOZjGyy3+pQX+Z3aS5XDH3hxl4tt5fIXz4IX++xEGDIPJ1BsSlywcBNPzjtRicFOXCIYVrAkP86T6HAlhZ5+V0oyGu8P9cdcfWnSuLUGVDU7awZiPDf93CLZ39x6rH2dq73whugEDn3mUTSV7FJzvm4EcKrC\\\\\\\/848X2ep+R575eMkXLmlrwnPoKF5vKjBdpRTsvx2FgH1ox1OyMm04mMcjt1ZhXAibv22SRl\\\\\\\/muaHU\\\\\\\/qfSwRg82cDLv3AmjkHEEQu1x8eZBCbWGkhAKOu8gbJ2kh5gu3rsWDQD\\\\\\\/aNnSpPllfLYrSoIITZF7MnXF\\\\\\\/9qN8OKKMznl1x+w9RaItvTF64QMiscP4pIX1EH3qWdQLfzYMFElo27zq2ZhLdWtff5bBzPHuRVragFybW7jHz8JwkrWvwp3JD5Bp+eqR9veDRKC2aEFlvHynLd+U0NDwm7zPkjHLVORUutZSlY6x71I2OEcQhC25tGQN3eZm1cEyl4fD9MrfVGOCVN6LcCS\\\\\\\/a4QQlUalv2ZQJk3YBt6DGHtkHDTAjw1NEOCgY8PeJNvb\\\\\\\/bGL5Jx1XwNkB0Ld0SfMB0z41NPt1LyK\\\\\\\/JYz88RWSthzM4e3zP5K61cgX9kHYnfXCqxIZsdnQpWsFkm1\\\\\\\/zG6PVQsOCowAeYhG87gcvIxYazESydz4HIFvG\\\\\\\/hscXzgNe7QhsEBZJ3Lpf+tnuxywzgTaAeO7EADcaUL8drKnqZ21DTC4f1aDTL2\\\\\\\/SuWCQnFY4NO08z9vZ\\\\\\\/vfCXXWDxsp\\\\\\\/b+MNObakhgLNc4ZRjjjRkDLJ3tHNtTN0QlCOycD\\\\\\\/wvq7I02+GPxmAL8hkfiF5cXvCDosqmSMVq+kX0aWqlWzViPtHwjWymL1GzXkCJvVYPS3s638HBa0z\\\\\\\/A05Axdc+NFzabrCjd\\\\\\\/3Vhyae3FYykmu6kvvXFHzp182WCXTxH7wwW7\\\\\\\/zoPMc01+ikeskhZyG0L6hNStK+32ozGtiumFRn7pLMyCNvkrao0ZcDmm3p0IZkaCGZ67hmBNpLPkyLnxBNOmZ\\\\\\\/EdNOlY5NgyM2PvpNkt\\\\\\\/F7OU16EXNroPDeLEQDYpEmAnreSdWvdAUY3OZNXx1rc8rcDD32fS8jQMkj1sgonLmLXF+S+75Zs5kU2YMg7jKvp45hu7V\\\\\\\/Wn56T1pBlIusWOZfauV4Z2UYomJ7RiysQ07\\\\\\\/cUPSI6yaU4VukeGhvpYk8UcMmWSrLwRMpFD4USf7tGh9hb22msFdNpAp8mb7Uh49cCpXlEwK5uKeNFFgMh6cu91PMcRANiYCdZ45Vg+SY1+iCwStylF3b8JZD1uEkLgKasmzURYUvOxc3xTUdgcydya6ZkafaundcjQOpSnCGWl6E4EKBS93rMXK5qJeo8+y6FC35ZU2\\\\\\\/CY7mSlob1QO7kia7xvXe4FvbkhLyp3DQzGxUtzEcAf9gvnvWKgu1W6TkOW9nspEy0Qc6XUbJho6pUb1vC84JJ\\\\\\\/PLPqVP2Zam\\\\\\\/l4Ezi7\\\\\\\/xp\\\\\\\/DIb9htI2HQ3Sa3G2otOO3Dba+4GTJBO\\\\\\\/jGwzMxQEvNeh0q\\\\\\\/ZTA\\\\\\\/RciA15gdjdymCgg2xwmoO4tJxn\\\\\\\/YTqMf7u6nvI0o2TjpSO5KIOHs4tXokhmORoSKP6+JOzcWkAT2PkCkLFUBsmdnlprukQEJOKY8Xty8DHaPp3ZnYTM705XfMIGreg2UauLIinc9w9+xOdaHYiht8k83LBeYUUd7DFJdLVEnfF\\\\\\\/oPl3W4\\\\\\\/V31Kv9DprtOaw9z3UiYcWgnfH8r7tHhS7o\\\\\\\/R+33p6djuGcHYbi3xSUqv5dWuV4U091lA9hlntBi7UNg5eGpOdJaX\\\\\\\/4epJVOFPzW3+u2rKwCuiz4FmvBCG74nGsybQxBNf5WBEXaRP0ecHfLBLcPj2MF86D2t9RkY8y2mhr\\\\\\\/\\\\\\\/NI\\\\\\\/xU8NeYXx7aDsQwUyQWvNLrrWqnQbJZ+lyOQZCHF9IIj7CadvMl0yz6A5g9wOE7kmlTAIXSIC8FA7F8CMUBGaC2vJq0Ub+7xKpyZcFxiMLyH0hu2a5kPXdRcPbXjTmaAxEgZ0CXtvr0ZeAqSwdyx8k3BsZlnp88OeHnFKDIQrDqa9ZvAGh6Zi5AYYl12kKegVci0IIYYYFam1XkgPe6DXNYLJgHP7JN59Ngyc+2cpFIS04\\\\\\\/YuOsTDWQ7jWF2osZkzIPGAN7Ykyuz8mm2iiH+UybFvxCFQykUGUpc0\\\\\\\/zbhcc49RuRCkYHVsw\\\\\\\/945KGkmL6r5R1p7N6nyYmYEUEgXf+t4zNwM8abUbN\\\\\\\/AorWdPJ8CRY1tWu6RcLIN43aQKw0jD8AkQrCPIMwhCSedG59DzH7EbHafS0TCObqF2hpsugAtX11VL5W\\\\\\\/1++BRWW42hGwloaVBAwHH457lw0xruHwczNr5i7apNt4m2zwuI+Dq5ncSGSJvlYBb4cauE34o+J3oOiEvdhT6CdGBs82N31geEk0XL1UoSM5caY8QjRGk0BAiwdIgAjO2BXDuJLaiQ0YpZGZftwUZDSniZTQ++j+abgiDfG0jR\\\\\\\/YCggP17G5UFkQnketSIUKMiEH9Inwnw13p7r8EAWkCk0P+o89Y+AHtFu8x\\\\\\\/fei4wle+PJ9sP\\\\\\\/I2YtXQnoPncUu\\\\\\\/a6pnsbFm7Hw8wglY81eb20H8Ij5\\\\\\\/d6RicvNilaXtEMfaY\\\\\\\/b59nBeSPl8xyjMNyQb5haEgN8hClRwtE5Mxa3lhdmrB4+ORZ3jynWOR\\\\\\\/kmMawZHd\\\\\\\/gKVn\\\\\\\/4JZXLqYpV2ZY1dDSajjunpmp0yzgG5gHQP9T\\\\\\\/H1YquJ4OFUKp1LnROmAkbiRPcNOf84EkwNvuJEOj6R4VOlDU9fGysYqa4WwbXzxGuasVGc5TuoqkkYVztQfdeAmIU3CstP6rwaHXgQbEXASTsBXIl4dwkktHHC73rxHJb1tO5rn2zh7OEdSwObveskE1Ndd23kVa27\\\\\\\/0dfwEqvPISJ7iB5ZKahIXDKEchfDPA6g1iSwwVMB\\\\\\\/Rn7x7upril4ikvSuWEEwQun\\\\\\\/lYwY1glT0hrf33qzkHixatDofSZ2E0LDTUNIMQDqBKJCNsS4gW3bIDpcF9N5dau5XtiaJyp1k0NaTEKi4O4RG3lBIu2cWSn\\\\\\\/MGOGlMPyTrD\\\\\\\/yCn2F4yHAVu3VMAfM73OoCOWmZ+q9XqXsEGGHbpPmjFMrxmGGppl7NF2GMosYfV2Rp4ZUeOrV8+BfZWIj\\\\\\\/byedHUS992PesLTjCD\\\\\\\/aiCKLuLBUl4KDWm1WN4DAebNw1lwjF73256hFbvXN4SK64vsSpMyd2OWgjMD2qhj8QCDXTcjbKSF66pGDOlhtcD2LUZNJOttMObgNzqMMnxt1rjK+NJPTDILXPjZn4H+MvUJA6\\\\\\\/H2+nbSa0D3+GhzkxilRxVjkmIxsUcHm1YcDdYBDR6Lyl0QiCZ7OKUpstw7VFucNh97C+rh2kyPiFYdUbYpGjpu8ewjhe7l98rJjF4NMXQz74JSUu9OiLG+xn1wkOGOSeeVX+cedQmd9s3VHHRZTfnbE7Rhg3R2q63R\\\\\\\/AFmxfJlQXyarUKDvi5t8CQgI6VTBb6eUgbA+y6XPtx+QbSU7da18i8sY+1W\\\\\\\/F35IsIT2KDlx++byf7xZ6hS0MZM5Ox1vc0s66ToWSrwke8P9hc+lIYHYdcpgTcGcXjdmu7JDJxvelExFefA0M\\\\\\\/qNPAYUWUDBjrm+UkF5CL\\\\\\\/sRvX7FU+MlWcGMMp6Hx9Ye1L4EFp7frD\\\\\\\/Wq2uEuJKOGDHsTzmx2fZ884G0HcaacpNHK9mrOVMiaXSYko0p+fUB7X6UuCWzUNs3ewYSlQ53lYK9CZ6c4dct2HbYNAbluz7zKyDWJ7yoSXC8fy\\\\\\\/uF\\\\\\\/FXF7gmkMfRfZAsF1\\\\\\\/jnG4njey6J2hTBWc9y2Lp4LCsHVx+PzveL4GAID4PDQJsB2Bt8iR7nhniMoVKFr7NRtkGrEHmojlkmrS9+W4zIanKdriXCjen1r8WCtcifnwjklBhNLSXzoPGycGYkupaXrWO29grj9IIM0efMc\\\\\\\/98Sb\\\\\\\/9Lt0GOtrq9997Irj+u+d0afMqf9IIqYFFl3f7nz2FG4Lt6sQSrzgfncpWDfLOzsu9fxizd723dA4JR7sugnkhTZSUztg3L8Pihq\\\\\\\/kjlqETDRdrhfdBCxkBihKp69xDL7INqNBtZ+UhOfF2gZs9HHzxEbZLCf0pkt7K69ENPobzs5w\\\\\\\/pzOBDwaRcnYq\\\\\\\/eXk5N2Ow0OEzkJd3CXtuBWe9mHraLEzvPv9OzmTysKMq3DfGH4vUjFDQcdN3Y8rw2F2Pv8BRMQtU7NWmE0gAQP2C8VdhOm0nr0b3zdR\\\\\\\/ja+R5axOs+LXp6ZEn5tTsjoonWCqSjWQCgs7m74gH2TqzbqfC\\\\\\\/FoeDoMENlSPRsZK\\\\\\\/GfnUI3jYYqH+uCK3iIXr1nl5Warj0+z8in9reAjiEsnLw7Lk8HqHj\\\\\\\/jGan\\\\\\\/5JVgw8EKphtX7fU28N3GUO6nTQZG\\\\\\\/Sa7wz+GUFpl0dNjcRL8ycRhHWsg\\\\\\\/+04NEvLJlOQRFreydU\\\\\\\/E2dOznZH9VBcEaBbH0AQZ4pHYQO0nti4yPKBWMKki4XihcQMwTBuK1pE+oOYjZRlLvnGhvuT+DjoOIIit+rOXbI1OOdNTfSYO3ARQ1AxAaWfkmxYSlHKdYcNfTjfwda2jTxu4d28iBfOhOr\\\\\\\/mObDMCZ3d0kNKcRp\\\\\\\/rH0b5m1+\\\\\\\/xkofzDMpVPDDz+ClZw32LVLZpUI64glYlfOzF1rS8TYPgyGVzzYEMin5\\\\\\\/xFEq22VjfyRKqGL0H+wY6+\\\\\\\/bWWeQxbgZMLawQPVcUMqBvMB\\\\\\\/PVofqnPAB9StsM6aCAY0WLia5sXA47CdRqB+dHLLDJJz69i9LUh86g602UIdwXbWrTQOBsah\\\\\\\/wO3+twvedrXWVsKaW1TUQMP0EgVTYZP372Xn18oBuGdYIuIgxEVPn7NKJbv5PgNcEPLEkKcBXRGbJ0OLRJOBE2pLqfAG8h+tFMm+C8MTYnU8j7FOky1HFHtF4zdBV16P2+aYHODHdObjFlU4B4fIOET5hzqXDL18\\\\\\\/d5PvO0Uro2L7NWxX4Yl1j\\\\\\\/TqWP1bLlt8w4NaYiQ7kz4nIC8sqZv0d3gqbx7NG8lqvd955C8+Hv5XbCHuxkjGxR3pwefZiUCsXlr+qCSOqf0xtt75Mjp4i+pLRsnfchrWK7gS\\\\\\\/BaJGOJ+0KEVF7Rt\\\\\\\/jqj8DQBHezKRHZ1T2C1GCMKkaf4\\\\\\\/S+6cZrJ3tRj4pi8j21ExFRcVIn7IP2LHwqc+\\\\\\\/8EwAVdp\\\\\\\/oIQYeFD4S2ppuR\\\\\\\/rExevn9YNBj6o3qDbOWFmV+pcdPkeZ239gSNOiQf31xNtIlWJ4zY3gEo250QzZlbZKiyIEu\\\\\\\/Q7CmfJoM2vhxy5Jv\\\\\\\/ckTyv+fI3tS6h5QPYug7nqA95YOs9uo6H9AN1HHbwOaIRbsjaIEZLUlvQl+ZxHLf8rJph5c5m3i\\\\\\\/SXRiFDeXL14Edq7rAMMBrYDOFvw9KdxiUm0sU9teT9+ZSXZaqU8NjsdU83PYU0xM7XCp7sMx9OJhhe\\\\\\\/WAVBXmYm+trhfoHyLeB+oyVDT5vBvWH+Q3hnYuRfxQQ8a0QuBFaupAGfVBKNOxVH7KAytq9x2sHUiNN3HtsNhYp1kiHgCW5+z5O5\\\\\\\/27BYorkbBTNSuee0ecqWq5vN77fZjHFmOCXs7hgbmbYZcZV9DkDZ2NhyuIkHWW0ZxGGESM6xffCwAPH3OYkCUlLFahn3++ak07nTqW+z6ptNjyQe+MB2YuOblckomSS24sAzoSlx8Uf20SXaJqsxhxsvu2Ao\\\\\\\/zVvh2Vf7PVJOOIkcdaum9SnW1Hjz6f0SeC+UF0S5yc7bGRLPdSg67\\\\\\\/U3Ej6WjI+cQNKgJboTotbP+gA5+fRviLZqMsKAOfW0IAmMIDaARKdMECPzoRhacevgX1o6xqJ57fw+puWAgwL+LjS6HdloEAJCS6ZUckUURRfZR1SbxeSOztiGU9f4z+ruIYHWz\\\\\\\/dUodjHByM\\\\\\\/Qw8ThInWf8pQjvScMzkzjTafYI6Wqz2l2ixx0\\\\\\\/GgsEm+uC714Jn2AtmimtRgqXUkl1RmvpFPMGKCBljRzEDpMmohhC9s5RTrySMIFKHVm7ruGZp+OJpMUnrY8nEqZFbjHkyQTlUbw2xQcuLHZTm5dydhlaOy1+LIxEqNrLbU6ml\\\\\\\/zE1eFvQ588xQKqaj05EunUxcMFCFI0HU3QF4LSkaR5kDRA52EWryCVFgnN8erFoZ08htrkTiIzlFpVyNK0lZFJ100K44qMHEVyM97KvCjVoaxrH87azLM1CF+a3WQ8IZRqm2Y37Ea2Aml4YB5cMbohoTm7a5xOkVUKieW+3sKjvyxXKwiqQisfjs0FK2uIzSODH52c9nhcHB\\\\\\\/0L2tlqmn1wvp+2+zN8U9sBAg3RK5jIQmlKcjRlYU1k4jGz6rzMyUM5wDotP7cHMavDBwT76P+Slj+y9Zk3jcE+kAHWgS\\\\\\\/paMDhcuMoSoU2GEcmKNp5vfyrT9n48HauhmqNRdoKxILxASCntD4ps27iMJ+RD\\\\\\\/vI7zTqG\\\\\\\/vC2mc1bqJlsvbeAllydP3y3cnuTepucBcft37lFOGzkHfc3nTdD+fvnSZ6mpFfbFpdCCIf\\\\\\\/N23dOUgayFmKCtNSR7WaU56LqhGYoDzw4zET6TKHkS0L560JjT\\\\\\\/QRIK6+GeU4cnXX0XCsEnwR8cg0I4DlkPk4CRld6m1fEFyropQammhhMW92btngW63mzqj0Bm3QxApwdPeCZpbD0P7Etbtn0JuHy5IpZlRQR9u12KSUkwU3wWTFUY1+XYKBlL+UMbf0sBZaudTJEHV3Hb3juERKNjA5Dgp+1PPuiK75Wjcc7vM4z4wUgYuFI2dIR9+oMjBAha55F1gadfgNgezBNJlHp9uIQg9EnFDqKilMlFsgMyEaFndFG7NhjoNUOVAh5\\\\\\\/OW8Wu+4UkdFJ0TyFzFenPG63HP5bCzwISGIPfCdjGipBa2AO4jCYpsmRf1Py7SwygGLebB+c5WFz+9ikSui4k+1nTfoQqu\\\\\\\/GsV7VFyI0ZfaOmF2STAmsxNymbXnUSqqJtzPj8lwMPWWzzkngDZyUQpYeKjGknRhM+A8LuGsmUdt1sa\\\\\\\/rRILof+H55ejBuTZXqhwgaLYrdHSqJ0dYL4fzR09jeS9z\\\\\\\/TwzbynKUUDylzVzec1PHN2SUizXnk5RDrdQHwy6OjTYUUJbbJMwhibKoLDp6ymmz9zN2Q4ahstWJL0XKsmDeflAMJb+RzS2sPj+A0yA+Yu2piOxtfBbB4ErYyU6Xz1k2aUWQhPVZUSMD4Vvmc7knj3PTcNVyK2pZm5qFwqfMNI17HcAquRRftDj2dO2UGYXhVDcsp1aGAbX8BREKNXdpjDRnz1JtNoTSJWxZZ0\\\\\\\/clEiR3hUW9SRLXzFKUB\\\\\\\/cUIxQeDRf2JxiMB+rjGy6OkMycs8JBJt7l6nSDGvcRcbejPdHwDwxHKFEaQY4XVqrdqtyR2FxpnhC3qIqIwyBAJkJO02RAVJPPXHpZaalsHSWj4fu2ghKBDC2EMICqo2tv9821ejs8bkL31Mfr1ecfSPJk6Jm3DJb+wIKF7h4o9nUSuUxq0f7abtf1CEd6EQqcMKO5y\\\\\\\/wnWmh5o2wZWQWKFdFFJ99ZgN4sRYRD02j15Pv1Uace9PMVTZEcHFAuda794H1yHPS5cOe35MbNkHmx+0kiSqU3fe4rRwN4FJsHYgab0DgtAjLujFLfqhuQ\\\\\\\/8CggwLVg\\\\\\\/uN9N4+yU9jTbEXrZ8IkqiH1S+sXCubWjB4\\\\\\\/RGCSZ\\\\\\\/xCg\\\\\\\/7Ww8IRhjEsZFfJBaJuef9JzrPWzhRTNH6jG3yKb5KQRLOs4lCztPK\\\\\\\/j8f8k38y\\\\\\\/\\\\\\\/vYHmp2pP6cGYxcVXHW6tznErm8WRIseh8peYjct2IWULE6O2v\\\\\\\/wL2t5NERIRGneBaoSlGsxZ7qKmriihBXN3mpND\\\\\\\/SJwIT3vxumSGqTkNtX3L5rE9pOsa5I0rTzMwGQx3ZF7ePTWBr1VG1VJ1Zzk3pKeucjFzfLX\\\\\\\/GTMfWoQ+DwlIxTEr6IyXtTtLUcmIm6YkhFmGj56+Z\\\\\\\/2RaUJbykzCZG2pzJ3MZ6aUuYfCNKvUpfExVUlzMSrxZaGM9o1iWZdGCQVrHW9eG8wg2\\\\\\\/nPS\\\\\\\/M\\\\\\\/5StVBrln6fZT7j7\\\\\\\/JUAnDa5j19Hz9pH3vDoUBU2cHMIqk9UiloDPL8r5ztMA4vWZbdgZWTRzYsI+ou\\\\\\\/i5V6mcIvEjslgzw9bUPd1FGr5E5nM0k4YMcK2nWUjNjq9HQ56uhAfiVSrprai8B88VCtwW2bYdsaSvT610LOreu+sNSEjq3+ouK8KWPrqlBOH7SsazlUI6zatPfjyZu+yewRZ523blXQMSe3LB9JxbwNChsmfzJ6+pHq0MvA2ACpkgumzgNlUIErrkY2FLvIm8FdVR0gQuaVPW3uRhWq5D5GyP4Ts90xuPJvgs7JEbkU2yDE1phOIyJMjEQJYN+aFbDXF+m\\\\\\\/q13Lo\\\\\\\/81gtOGmdBXNvhwEtxfD3PCzsk2GckS+zor4ofdhQ6kAInrlEV0Fo\\\\\\\/74yudQnfsb6KK8x3HZWVF0jKwaOJmWt4CbsvgYypkTwjXaywuZnAiYjZTd+VOseULqW1OhejHl6eGF5QjH4U48IdtcS\\\\\\\/nwVhsd540bOEUJadRpDjpBlwsIiA\\\\\\\/C3spSn3NqLPO2M6q3a87BzK\\\\\\\/5QBHoF3RgjwgZpt\\\\\\\/I7fWiO20oVCdNx2ujU614xH\\\\\\\/qS3m4th\\\\\\\/liRUm\\\\\\\/NC2JwIagCWxs2bpP\\\\\\\/+iMoBc7N9KCthkh4Xi5zitbCQ8KFvLu7VWM6kO9qa8wSVGOULD+ACMHeKMiasDbKF6boCIkazZY3fH9awPvnr2bJVpkMCd2NvbZ\\\\\\\/tbVpNzXicu3A00dPee57fi0XeiklfEl6FIwSYRo1qzRGibuM2V2fqeTuX5BzEahFa70iHoVXIwdZThyioyYNqBi2qF6mRNKZuEY\\\\\\\/JdknTcDBcjlJ4gnzO0VFoOoo6nq\\\\\\\/f5fAvsesG8pvYqBvovj77pmRxmvUJovndTfQh92uFoUbyJ7Mn\\\\\\\/mUkobOBpQOVTiWyzeI+6DyxYTfpa6Z2Os4ug4x7Yg2E3GJvE3g9ToQ22WMT8F2ma2rtonH05\\\\\\\/o4xsITvvFPL7qArK07e6vinqHw7PGQbH6RvoKt4U9IytyRekP2rWxP34dXesCLplixCKUGqTbuwdJ3+sa9jaBKbD+iX69dUkWzOfOmRm0SWNARjOSndW2YSWdZu1QPHudNu+wRlKOdoeRYIB06hWgydiRZu5H8g2EG5PcsuC7PLAPYC0ujZx87KzeGpeKo73obs7a8\\\\\\\/8yEHOifmqIdu5YJeazma8G0Wdy9XSy\\\\\\\/06q9SAAziTED2h9lyr+zYevK4K6z0dMSGNhFPkQ3GLcskAt4V1OTG8BiaFLM++Q\\\\\\\/5g\\\\\\\/\\\\\\\/nxJPOAMdNTD9hA1MP1cZ3\\\\\\\/Pp7U+RQgiXDJhqRrFFX0nsmi4HHlgTR2+nBbQOWXyI4r5fFuIqeNG0N0jhx7zfxd8Sy4ZiBBmtxmT8ruHTVqOyya11qbfWmN6Mnu\\\\\\\/dcWwD3MXZ05XUrx6g3cAhpPuc6veOP6M4HyVrfutUMdYeM6PgDHFxR3CoeKFuHApam2NAWj7xVAH9yY\\\\\\\/MpufGQLrsL0h6Wmr2XMRKRsdkf3oI\\\\\\\/adfcqcTHGHD37YvV3a7sVcRZkh0dO3inOZE\\\\\\\/6vOlYSV9oOWU8rnIZs29aK\\\\\\\/m3UoTAGTVpAKgPBLPEY1Max5lSG18WXUxmXoxf6L7fbTsFgj0ZDRxdtaNZ+1eXcnoXD1htT0q5E23jdIZnXpbymQEJ\\\\\\\/IQo1zQbKITl+Z5gScmJbYeHUv0q4mCLFnmYUOx1LNmnhfe73a1WZZj8X1F4egvzel+vlfG57qf9bo2e2ZB02rXF83ujYik03jzCfBd9G12yvtPDtAbo44sFuo1asQBh8iokJy0Su\\\\\\\/Qg5y5TdWYmmTzhD4P1zfC5flpZAR9yfT5TnmpZeJ8ZaRgR2yCpSFRdS3KnZob8sRDk3DlIuKA489qs92ZIggCNmldEHB2YdKj95h2lD\\\\\\\/EgHsgRMssP5A7S1JPyDt8NZLPpcRYSXofhokjgNPbObQPRpBWj1YZU80jnHPixRSkadf1Ae5lPT99eAHRby4Mly+cQvMfAMdILOPPKU60ULVhYTp25IEbzkVTBIBnWGPM3CeVqRhx82rePPYCDTZyoV+ee2ByiBSCGAit\\\\\\\/NPUJfVWQoy3C5UzA1hF9PSaoII+5sm3bb\\\\\\\/eEl20ukG7N+uYPykHFLJKUu7Qdxi7YB1gt2SWLu4Ud0UfrwPNKvr7pD+zkHD29U1LtAKQu4ywG35f5Qq\\\\\\\/kkW70ElABsoUzoAc+tzSUHOf\\\\\\\/+sZwz7wupuda2HddTi697hQS+bhDpXqYi8jh12a\\\\\\\/l\\\\\\\/0OxjcowaBXbYgoWalI6rNSSTxenGuHrQ63u8K8uUEYnvhWMVxLONeSMQmpQF8XuVlU0PzZElWXipusCPk3CZr7WAZ7D8Nq2Ttul81ldNcBDdnwNLSPF9\\\\\\\/JtQa1p5zLCXo2hVGl8GnTXrqouRrKpMEmeBWeMfosIuqGBvh4taVjQVLKOZ5BWcbQP4mMxIuMGEztmzums6xoEU9ARU8CFoVjNzBfxg1wh4+dJU3MskHOBk0uDH1NcFm0Q4+52WuSh6W4y8geVU\\\\\\\/LBw9byfs6GGdJV6xUFreN9QyZzgxY3MdlyAL04htAwsFRlNDdeykfxd0OhB33d9Rl\\\\\\\/G8okuMDd22dI7do7SCPCY0RFl8Y\\\\\\\/UW+Iea5FNvYeNBEzjsIGkO6LN5dTL215yGv7iMRz\\\\\\\/yILQRBpvLDdGFbBcfHgfbwYgYnwSEQ\\\\\\\/GKRHSNkjks2jPcxFOT5N3mumgnGpbaqSL+f5y+kJvjf4WWrLzUxi7W9gIjJ7rz6yIho0uzXeYlpGhtcWgRhqCyEzm4qAC\\\\\\\/fdOZqVx5ltXvgRLnHXZ3PIOs+ZxkmONGIGLv2I\\\\\\\/0wHTvE86pNpZvZxpmEsb8NUozVMVqFnQj+hh4ShwOVrNcOeV0vfKDoYY5F91LCs4OSRPpoH9KzVQtIDL8NS9qnQVtTumqzcrMd7mb4fRjJ2UUo+n9PELnUbzT4+5kD9OCKtkjki7avu5TT\\\\\\\/pD0ReGVqeUiHJ\\\\\\\/+000DHDSMmafyVuPT7nOkQ2mptVo9SPXkN6QO7hM+hh5AbVrirFg0lkKtSALB0rgQY0yCxp17PTCzpV9pN\\\\\\\/ifvpLClTxKMYt+JpnDB2BlD8\\\\\\\/TKiHbd5TILq5bCQBw1MwWuNjiBUREYnfTpsY6g\\\\\\\/bjaZgp19fc3Qua7A+1QfUZNxSJUlTJ+XtH0ETzwq3S5He73smMoyotm7ChH+7LlJLAh2CEg0N3snkxY\\\\\\\/ayePGkyXdMYMXrR0OcPhsMbYp2cM9VyKSQlVjWahRhflwpFwBA2PC5Hm14DycEa2W9YEH7l1Mpx1NM+9jYlb0A7m0lJGxd89ED5T6ZPeqWspMhCP+trbBLC0O8ebQn0frUt+etEQ1nnki24lGx1m6up7B+euw0NVO3O8N28RwDwC668J8v20oaYU7ciPKOMmcaFmNBaJd600lrPOQplxiCqekjXmk4GgmTA6RnAFGKG\\\\\\\/oMSaP9Vzr6nbVrsVeWRlC0hVmmXzxi4bv7p0Bjpv+D8LSHphCpanFHDaTqF14Kwtj\\\\\\\/+7fMsDGuFC2km\\\\\\\/X6Ou9mX1MZ9TRT+9tCof2UmwLUH5zv\\\\\\\/uALG8OHlRWDqksSROav5WLZy1TyfZNIG8K4Yea+xDgP4\\\\\\\/w6Tq7HzciFux1Ug9cSMptwUEQkOJpkqmPw5EEVY06ruxrb7nlM5wQS+OMGrbxokgnjxZxmvdEIML8jucyferfJgoI63dU0onWkU71dbMwfoEdiXX8GcFriUE5XgGb3hizzGxLd9qKOUSTSeawqZMpgK87ypuwhl9\\\\\\\/9w8Vet1wIUcaxAXq8qkHTxH97Vf7NN6ACn1i4avmI5uhxrxf30kTNIXmvBf+eqD1+xJuGxjJ4ac1\\\\\\\/932lscvbpSuWAsTgHpCivaTQxKjOeQ6RZg7tWJA95lKZ8bTVC9ySFds47lxjnTWD8E0w5N1X8o5hS+xVDKlm+c\\\\\\\/PGKOtKL8WjQuNCSO2zJKX8OIO6hmRtN\\\\\\\/Hn4wYXNURjXUuFOK0Dbb6tAn09DZY4AbuGLR0zbT1hOKUotnq5qA+l6n9sTtFygO\\\\\\\/VSRLh\\\\\\\/dGR7eZwhxZWdrJy2oJ0vrCJnglK6G7ghZrwXx5aUQ2alyxqpyqv1S4h7hPRecNFreoDCg+7WDZBW1iKKjCyVsIx8CxJXDA+CxnrfvUaquSQQztyzHqa+B1gz70DdA2GSB6mY6p6aHvwYMU18fLEKQ94VKjXKMz7HxGcB6sA8LtL8qnDJrJF+5WCJvnIZMxXZYWnnRsuZbGsVuiVXse7I7pdHZQqH1FP0gUWh\\\\\\\/lBD8QZohYZKG6Vzt4awqwNF4HkXu2zBK4QJvoybA\\\\\\\/WGao+kgLiA+XhhIMWsdAXb5O9RBb818FPqvGinIN4As1Z12HNtCbtTmIoL\\\\\\\/rbcVh+GtY1+1N9zv1jZgy\\\\\\\/BOc+7HqtvbW6qYJvvcjS6XXhKoongmzH1vJSRH8wWlzGKgMqD3iKxkfXNAi4ssN+GxaU\\\\\\\/MZuKVBoLuYtH7iCbB3VR\\\\\\\/wOvOU0Tz+GqthSI6h8f\\\\\\\/LCc\\\\\\\/yMQKZbTU5J1AgfV85Dmo4aX8Yn6NQr+fldPJMfVMfYbz21F33I7XXYJ+vMfMwIJNpfhcWyG7U4zEfk1gJ3dWQyVcNDlqIgehmbXSyuobBohiKBObFXQsDvBRgHqyK0yALvdMsLqJ3k4bh\\\\\\\/cDJr6VS1UL81LhaYCn88kNHiGaOycPKuep8OUnLYXoER5U4N4Ephfi88F4tOy9tlPyREsDZURg1uFvD4jsNZ2TFOZW26OUeb55mSXabHVPchme\\\\\\\/GluyZnppKdG2JiZ6cyKfva+HkpPmUDrKsOKOn58\\\\\\\/oh0wHkc7457sOQeLmLfCYFeTSdZqELQ+MIlV\\\\\\\/rH+8G3zQJHsiV2yAkYhsrHZyS7hLuW9YldBEH0ORzNBmTnl+4WreE30xT0DZEtRLEEp0GlO8b3cVDxQkjASYDd24pPFi\\\\\\\/HiZC5EnxUdneT6MeKwDdFGi\\\\\\\/C5b80cUCPMsdRBD9hrBdSnEe5Zh3T6+HNRvVipLgrtOklCtsnUKb5YmP7TR0fcKPe1bZ5mNeQRh\\\\\\\/CJRH4yGo0ehTa+L88Zy4dZIlg1q2guesF0VrTUC0TojPKnidqY+zKKd0GTV9akxOjnRhfRU4hHT1j0GntBsn0c+1ck4BEASdJmS5PYTZJxkdfYrVMTyT8hlG\\\\\\\/kHDjlX9Ysg0UZIpOrcseHVQnx7LlVv\\\\\\\/cIs6A1KUTTta8CnaThaqQiIO3RUcOUrT+IpzbGeZZvVF6LR68RSurx8HTNXhNl97rqwlJjVjvMGuO\\\\\\\/oOeIbZc\\\\\\\/vWtQpVLd1O\\\\\\\/6kmp+njBszxaN3jwxtTjElhl6xsn2cqXlGHaHF5YmXCipuGvO\\\\\\\/H4y3RJ2nKJYIer2uHTllf2ZY4ZR97txlc6OPTiImu+PJCNyW2y2kPd1hhf8sBCIOToHvEyA3NaDPMXBhb9M4ppEmxxK\\\\\\\/4kGnh4R6DzyBH96UUaYoqEDmIWVk5EhTmOwAGU\\\\\\\/qiB8gOw6\\\\\\\/vjOTakPeBRr39G2KcJ0vRQsye1zuPDz5IfOpn\\\\\\\/3+\\\\\\\/UNuJfGsTGiaqjEPyGm7eqB4wxl2gzk9\\\\\\\/c+ankG75CrTIdwERYIkGcPVpoPrnR2osoYHjMUszzH0Jwif00epkjyg8eic54zcUsaA+w4G\\\\\\\/D9Jf6doa9HKR\\\\\\\/XKQfWIeCSrx6B46XNg25w4vtreIHTKv6M7wq8NxSZLOddyfbz\\\\\\\/WAUhn2aI2IkYk\\\\\\\/IcuAfhdBmwUE0CT8BCEP\\\\\\\/cMziHDolqv3AxAeo0wLFBk2x8Yi6T9AvgYcGfS1cKRAEdX2m\\\\\\\/is0v7fh5gPyfXQI5dv2s92R7JUEy35eBjGdj3KPSzQVuuzaNECrYUAiKeXMrGkwLBNmsdY61PCqfwavrUFIyayfuAB8KoHQyQ7WVKHr+RMve9YnvK9Cs8FQVIEhWKavSqEBdknsUVt\\\\\\\/LJvILQ9OwuMkDA9VtdUkMwRL7t81mUVt4zpGlUQc+Acf1uybma2gpPA6FAb5urzP0OTYqbnPtHw6VZqYQqOICMJVOxgTO6qcaaT1zZNPdq298j2dYoIBXdv3nczV9LWWjt8x9NpViN2pI9XSM2xmol3c+EWFbmO2If6fBYSEkZugc8QWlwBWQC1ZRYuWjD3GfkOLilQMbYgb2eTJOIsNeQj+JzjwrhSF3tXE6hFK5w3GGfGBpSZCSLByObYXoZStmPr4EbQO8Je7gPJI9MG7ow6V+zOkTHb7FQKmun\\\\\\\/BRQQhpYpwCJmobjmPZnIEo8oncNUD2juL\\\\\\\/up6Olsh8dP+oQbl67IGaPdpwe28LaePEmIds4P6Mmkyp0SkuG3RAkF4d7b3nI0XUfb6Y7czSxkCTbcB0XLRWRZ1z0WDym5UuGbCpk1tC\\\\\\\/X\\\\\\\/trm3h0uXVskgHi13P4rnn9\\\\\\\/M0ImbC7HMZLcA2sPTJotStXRE58kh71UJJzyIsUZufoyeMgaUjeoKErxY1GI1cbuNNiIbbTLVnqWIleWcsQqz+rWdK1W3YxlLjy7TbJL4lLzDZRknxb990EcpoPo1A6P4LVbC86z3ILEoXyW+oaE\\\\\\\/yAJFFnZ87wFgoQXVfNADmSKSznMQB50JwVqr7bbxm2Nup5ejD\\\\\\\/I0PuDsI3SJDxW2uNFvxBdQy0Kv3liqz3L0fzWnawbqUwOSthjbgD1Fivv\\\\\\\/Ma56+587VTLi1HlsTgntRmcqo4yUysqawbGMJUtiTv452wkUMnXKdLG60ASbEQBsEMAlqxNfmD\\\\\\\/NVthJN0qaQShNw4+otPB3NpMp2pfSTAYcblem4pR0bmOkn4gKTxl\\\\\\\/A595fdBkP1alBBZpSn125CoAJcnxQhsCrIw9xXm1CCMXx1O3zXeYMy1cPomfrnJuDHI9oMTEAA+j\\\\\\\/XyS3mn3mS5eZLZVPGDIqS3qMIkzt7dyU+3hL2a3AgLPC3c7tW+8sn258\\\\\\\/l4N4YVzO3P+VnwJilzU6EFf49WdOLI\\\\\\\/y6FD6j3tvW7xMntLq\\\\\\\/EMTDSS3v67CtMyPMsUyGPNyTWW0uJoOZZ0za0GCbde7VCiy9ofdNWZozmI1foSGJDcj3jCk8ZD5thGrUziCgBFf1ebbB8fUPK8zovOShCnwiuzO\\\\\\\/RBE\\\\\\\/s3c6DJ1zM4mOvA\\\\\\\/UfeiEpdxi49ZwlgJM30979XE+pd0KQBVMg1WdFpvKZCvcE8vPcxdTkl\\\\\\\/xvnKrDHKKx5W96urwO\\\\\\\/kldKzRfhRn6nKjoX4s5PDSzrlQKC9f47lDaT7YoTFirDjvkuUltvcT+5zmmwWTraALZfaxWKGbhQlDMuLcnYjx6xuPYVHuRPkwWHe0pptGDFc96eTXuesTVID7Or0SiHvnANPumYpz7TwPSP7\\\\\\\/qomtYomz72Ml3YsJcGT7jfS1FUFNJiVnNYRjQhaijKb+l\\\\\\\/yCioDWV\\\\\\\/vNEFmfzrVcU71xpFz1Nf0gShMcgb8DW3uLP498ykpTUozpwV70FLFxFqVhavAU74g4cjZ7naOjnUl7oU=\\\u0022,\\\u0022iv\\\u0022:\\\u00221\\\\\\\/j8RrGA00\\\\\\\/JpLYPTSFUJA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223f58c9600a20979dd1cda28358905ab0bad28a6d4f3edba6bc6e66477e7214c2\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LgGzp4uXco7EagQFifHkc4Kj5jmDee6c1QvOr\\\\\\\/9FaNUuzr9umQ2ELR\\\\\\\/HFwotqcanD2uh6okPKoGMZGfIoITOtkA\\\\\\\/hiTrWlbhQZwQgvJYEpKg2uRPOzZO4ejelv2w\\\\\\\/Zf9nxThujnCdHjR7SDO97MdyaYANsIbGh90jREQDE\\\\\\\/2rVPsaMSZhs\\\\\\\/f8ToqXgLjyK1SX42ZYfnTI276dUmr1uFqk6+jjz6P47KBCHjoeIj4eldZzsDyLqmYdeLj6PTS0wcPO\\\\\\\/R5iPy6wxeElszZFE7vY6xCUqXbEBpF8+OEgPyRDDDwPXokikrSTbPnfFRWwniRvzwtM4rQc1ZjZZ8XONt2yItOgHAzGULWorkvCJ\\\\\\\/QlHEp8zXvpBwhNaurOrYfY1l2G0kdCbQFzz0dM6U5dWWzIJn1+nZh5HmQE\\\\\\\/ZelCvBCsCuXI8CiTsrjfOSsWdYT0AVqd+OTEAdCuIbgbGz1dFAlD5whLqwXixF44xPkWJzezDVLRa9L9GpTrx2neU2p\\\\\\\/psER1eoyyLOPl+u9YENXMqtM4yXuH8Ak3HM692\\\\\\\/aVdZ1\\\\\\\/GPX94E0hFDlNvy9R993Sj1hpF99ZhKSfLsyL\\\\\\\/FHPTGO0aYGGmo9yFiO7f9WSocyn\\\\\\\/oICEMcHm91mIAtLL1wBsYIoaqwYkBDUCplxEvSB2CZpC95EQMk02hw7oHGw\\\\\\\/NS+BUdzVwLyVuYGE79l+QE10Ool78aqFO8n4lSEycN\\\\\\\/MRkD10XitEl+gBdaMf8LAApz5j41yj2il+baUnKfw0dFzHK6m3PthBadBq8KYYUPddaom7TVNagX\\\\\\\/SptVn9B+MK11TkQzqAnP4ioohKcgA3ehux6CenEpDeXzC4Eo8eP\\\\\\\/gO4QKO+6yklcG6I5+gD4dpGry83\\\\\\\/xFeHfgYe9DGBbX7w3+j+xqb1UWn4e7RFd4qLSSJ8HLbG9khOCC684vOfmBepxYk4+Y8wKApdxOTD2ADQ3LCIk2IReny93QlukeHNa44M1fGMPOEp9lwx7hpIDlmMd99nIl1+WihrzJk8o0QTaIBYdadC5DGvDhEjgfe35jBrksaOQvV4LDPi1KiaHE3XwSQ8581LgYdv2ue\\\\\\\/zIu9ontYrxWsLx3voHYxeJ\\\\\\\/NESIZR2i96roltW4RSr8F+co2uWN2blO+c7tkN2uZzWZEMSnirT+1LiJQ7vQjO32tDUUtnb9s+dJaByHi9oX502h+651AcEPuyYWFR3kimJC\\\\\\\/vAnDNKh9+HWdQ6eWAaitf9aInURMp44CerygQwIED3lVza7WbRk1w1Jha7+SDJeaWgcFyybgQllq31\\\\\\\/KAhDCYYwnc8H1lccOPwo5b7yEislOwlLBI07a6G4tOQtTSQ3SOIPkGtJChqzb3ZD+8xY2IhzuUKdCeF5bnxjCqPf+YOurZcNNsVNsjODtIc01u21JcETnPXloFKWfgtC3gTXSHTdVC28Ewvre1N\\\\\\\/ruo2XuOvufaqQ2PXQTHiNq3bI359enM0x45C4A5Lm\\\\\\\/PHcQfCkLawH47xbLwbigHjKuWO+GD2MCzeSFTtFkV7l2u+PYQK2TNeSS4xXX2LuRZY1i0DCvgZtZeKBRFRtHpdEkYR3MScriwfnPP46tHuudt25g8NDdJ\\\\\\\/zrfdkcyKf+Ve9OZ0262cW+N928+MbrB5zju6ktbBZOyvS3dM10Q4T8f8VO0ELTcGhMOhiC2DeG8ZtqKJpekFPSYaRThYg3nU+AT1WbJgbuHoNEiIIk4WfgUCx2\\\\\\\/cu1tylQVbArhQHTrNOPPGfOTe3Y91bUV4pgGXk5zaMSUCS5tJrWSbHzSGGN4dDueAFMvZEQidi8UDg8nSC\\\\\\\/Qgx6faeKg3dpVsx2Ug7G8bsK76nFhajPKdQKdVMw\\\\\\\/h8wqVt3oIMp586u0uTPQzVAXQf3rLhdKE2RhbyvVd\\\\\\\/BNzXAfKJD5bo6hXo0dsFDLUHolEWW6uBwlZQa22Ly6XtS\\\\\\\/\\\\\\\/OMpeaEhOZ1LbLq2GKoshu+h2+5Q\\\\\\\/qsEnKeVEY+HUqh6ChKIfX2uY48Kivkjc53x\\\\\\\/XllPD00m8de0+wvxz70OkcHpUMcSJjivTLfUP3TfmEu1mlN6W+afp5tR8wCAF74cqkacIkJLIG9R+BDGUea4wJmeJ9gycGwTZh8fqY7ZbrQLkdYlcD7I0eR\\\\\\\/AbaYWaEuCQg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ifhQt0fnwrgXvbcfnJe0eA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d51b9bc5ec6bdec8ac6e3dc826a9e20cb9c00e3878f26f069f46d74f1719c3ac\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460422 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2103" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-sjcYN3_oanE3SAZYE4yiRVEEPn-wPnJ2fBuI6-fIdqk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2103" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DiVcRmJ3sOht_0gso5AgRfOfGdT5BWT9CHEub_2H0Rg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/2091" title="Dynamics of a Novel Highly Repetitive CACTA Family in Common Bean (Phaseolus vulgaris)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/2113" title="Comparative Genomics of a Parthenogenesis-Inducing Wolbachia Symbiont" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460422" id="node460422" data-pisa="ggg;6/7/2103" data-pisa-master="ggg;g3.116.030452" data-apath="/ggg/6/7/2103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improved Placement of Multi-mapping Small RNAs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5279-9964" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathan R. Johnson</span>, <span class="highwire-citation-author" data-delta="1">Jonathan M. Yeoh</span>, <span class="highwire-citation-author" data-delta="2">Ceyda Coruh</span> and <span class="highwire-citation-author" data-delta="3">Michael J. Axtell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2103-2111;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030452 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460422" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nathan R. Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Knox College, Galesburg, Illinois, 61401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BR.%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathan%2BR.%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5279-9964" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathan R. Johnson</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jonathan M. Yeoh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Knox College, Galesburg, Illinois, 61401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BM.%2BYeoh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeoh%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BM.%2BYeoh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ceyda Coruh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Penn State University, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ceyda%2BCoruh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coruh%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACeyda%2BCoruh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael J. Axtell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Penn State University, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BAxtell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Axtell%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BAxtell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mja18@psu.edu" class="" data-icon-position="" data-hide-link-title="0">mja18@psu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/7/2103" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460422" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460422/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2103.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460422" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460422/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2103.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460422" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460422/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2103.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460422" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460422/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/7/2103.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460422" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460422/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">High-throughput sequencing of small RNAs (sRNA-seq) is a popular method used to discover and annotate microRNAs (miRNAs), endogenous short interfering RNAs (siRNAs), and Piwi-associated RNAs (piRNAs). One of the key steps in sRNA-seq data analysis is alignment to a reference genome. sRNA-seq libraries often have a high proportion of reads that align to multiple genomic locations, which makes determining their true origins difficult. Commonly used sRNA-seq alignment methods result in either very low precision (choosing an alignment at random), or sensitivity (ignoring multi-mapping reads). Here, we describe and test an sRNA-seq alignment strategy that uses local genomic context to guide decisions on proper placements of multi-mapped sRNA-seq reads. Tests using simulated sRNA-seq data demonstrated that this local-weighting method outperforms other alignment strategies using three different plant genomes. Experimental analyses with real sRNA-seq data also indicate superior performance of local-weighting methods for both plant miRNAs and heterochromatic siRNAs. The local-weighting methods we have developed are implemented as part of the sRNA-seq analysis program ShortStack, which is freely available under a general public license. Improved genome alignments of sRNA-seq data should increase the quality of downstream analyses and genome annotation efforts.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/alignment" class="hw-term hw-article-keyword hw-article-keyword-alignment" rel="nofollow">alignment</a></li><li class="kwd"><a href="/keyword/annotation" class="hw-term hw-article-keyword hw-article-keyword-annotation" rel="nofollow">annotation</a></li><li class="kwd"><a href="/keyword/srna-seq" class="hw-term hw-article-keyword hw-article-keyword-srna-seq" rel="nofollow">sRNA-seq</a></li><li class="kwd"><a href="/keyword/sirna" class="hw-term hw-article-keyword hw-article-keyword-sirna" rel="nofollow">siRNA</a></li><li class="kwd"><a href="/keyword/mirna" class="hw-term hw-article-keyword hw-article-keyword-mirna" rel="nofollow">miRNA</a></li><li class="kwd"><a href="/keyword/bioinformatics" class="hw-term hw-article-keyword hw-article-keyword-bioinformatics" rel="nofollow">bioinformatics</a></li></ul><p id="p-4">High-throughput sampling of cDNA libraries derived from endogenous small RNAs (sRNA-seq) is a proven and widely used method. sRNA-seq allows both discovery and quantification of diverse regulatory small RNAs, which can include microRNAs (miRNAs), endogenous short interfering RNAs (siRNAs), and Piwi-associated RNAs (piRNAs), depending on the species and specimen. sRNA-seq data have played critical roles in the advancement of understanding of miRNAs, siRNAs, and piRNAs from numerous species. However, despite the large amounts of available sRNA-seq data, multiple issues remain. For example, the central miRNA database, miRBase, is thought to contain substantial numbers of incorrect annotations (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kozomara and Griffiths-Jones 2014</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Taylor <em>et al.</em> 2014</a>). Additionally, annotation of siRNA loci is much less well developed, especially in plants, where siRNAs frequently represent the majority of expressed small RNAs (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Coruh <em>et al.</em> 2014</a>). Many factors contribute to annotation errors and omissions, including the use of suboptimal methodologies for small RNA-seq alignments to reference genomes.</p><p id="p-5">Alignment of small RNA-seq data to a reference genome remains a persistent, if perhaps under-recognized, problem. A major issue is the prevalence of multi-mapping (MMAP) reads in sRNA-seq data. MMAP reads occur when there are multiple best-scoring alignments to the reference genome. MMAP reads are quite rare in modern polyA+ mRNA-seq data due to their longer read-lengths, and due to the fact that polyA+ mRNAs generally are transcribed from single-copy sequences. In contrast, MMAP reads are much more frequent in sRNA-seq data due both to the short lengths of the reads, and their tendency to originate from higher-copy number regions of the genome. Endogenous siRNAs are known to come from repetitive regions of many genomes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Matzke and Mosher 2014</a>), while identical miRNAs are often encoded by multiple paralogous loci (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cuperus <em>et al.</em> 2011</a>).</p><p id="p-6">MMAP sRNA-seq reads are often dealt with simplistically, either by randomly selecting one the possible alignment positions, or by ignoring them entirely. For instance, the popular bowtie aligner (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Langmead <em>et al.</em> 2009</a>) by default randomly selects one position for MMAP reads, and can also be configured to ignore them. Both of these approaches have the advantage of computational speed, but both also have significant downsides: random selection results in large error rates, while ignoring MMAP reads discards large portions of sRNA-seq libraries. More sophisticated approaches for placing MMAP reads have been described for mRNA-seq data. Expression estimation using the ERANGE method, where the expression of MMAP reads is measured as a proportion of uniquely mapped reads within a particular read-cluster, was shown to improve estimates of mRNA abundance from mRNA-seq data (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Mortazavi <em>et al.</em> 2008</a>). This approach has been applied to sRNA-seq with the SiLoCO method (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Moxon <em>et al.</em> 2008</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Stocks <em>et al.</em> 2012</a>), where loci are identified by clustering, and MMAP reads lend abundance proportional to their MMAP-value. A similar approach applied to cap analysis of gene expression (CAGE) data instead weights only by the number of different species of uniquely aligned reads, preventing highly expressed sequences from lending proportionally higher weight (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Faulkner <em>et al.</em> 2008</a>). Both of these methods improve the accuracy of mRNA quantification. The Rcount method uses similar ideas (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Schmid and Grossniklaus 2015</a>), but produces an alignment output, as opposed to solely a quantification of mRNA expression levels.</p><p id="p-7">sRNA-seq and mRNA-seq data are similar in that both data types are expected to frequently result in local genomic clusters of aligned reads with distinct sequences. For mRNA-seq, clustering of alignment positions results from experimentally induced fragmentation of longer mRNAs in preparation for cDNA synthesis and sequencing. In sRNA-seq, nature performs the fragmentation through various RNA processing events acting on longer precursor RNAs. For miRNAs, the precursor stem-loop RNA often produces multiple variants of the major miRNA, including miRNA*s and isomirs (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Coruh <em>et al.</em> 2014</a>). Endogenous siRNAs from both animals and plants are also often produced from longer hairpin or dsRNA precursors that spawn multiple distinct siRNAs (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allen <em>et al.</em> 2005</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Okamura <em>et al.</em> 2008</a>), or from short dsRNA precursors that are themselves spawned from colocated genomic clusters (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Blevins <em>et al.</em> 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Zhai <em>et al.</em> 2015</a>). Finally, piRNAs are also produced in large genomic clusters in multiple animals (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aravin <em>et al.</em> 2006</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Malone <em>et al.</em> 2009</a>). We thus reasoned that the biologically expected clustering of sRNA-seq reads at their true loci of origin would enable an ERANGE- or Rcount-like algorithm to improve placement of MMAP sRNA-seq reads. All of the previously described implementations of this general read-rescue strategy have specific features and settings particular to mRNAs, and hence are not directly useful for sRNA-seq analysis. Thus, we implemented these general ideas into the alignments performed by an updated version of our previously described sRNA-seq analysis tool, ShortStack (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Axtell 2013</a>). Here, we describe the implementation and testing of sRNA-seq alignment performance using a local-weighting method to better place MMAP small RNA reads.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Alignment methods</h3><p id="p-8">In thinking about treatment of MMAP sRNA-seq reads, we make two assumptions. First, we assume that each read in a sRNA-seq experiment represents a single small RNA molecule, which therefore must have had a single genomic origin. Note that this assumption does not mean we assume that all reads with the same sequence necessarily have the same genomic origin. For instance, if we find 100 reads of identical sequence with a MMAP value of two (<em>e.g.</em>, two possible alignment positions), it is certainly possible that some of those reads came from one location, and the rest from the other. In other words, we treat single reads as indivisible, but multiple reads with the same sequence can each be placed in a different location. Second, we assume that the goal of sRNA-seq alignment to the reference genome is to identify the site of transcriptional origin of the small RNAs, not to list their possible targets. Target predictions, which usually involve lineage-specific parameters and only a subset of the genome (for instance, mature mRNAs only for most miRNAs), are a separate issue than determining the transcriptional origins.</p><p id="p-9">We implemented alternative methods for handling MMAP sRNA-seq reads into version 3 of our general-purpose sRNA-seq analysis software, ShortStack (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Axtell 2013</a>), which is publicly available at <a href="https://github.com/MikeAxtell/ShortStack/releases">https://github.com/MikeAxtell/ShortStack/releases</a>. ShortStack takes in raw sRNA-seq data (in fasta, fastq, or color-space formats), and a corresponding reference genome, and performs alignment, annotation, and quantification of expressed small RNAs (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In the first step of alignment (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>), ShortStack uses bowtie (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Langmead <em>et al.</em> 2009</a>) to identify all possible best-matched alignments for each read, subject to a default, user-adjustable limit of 50 alignments per read. ShortStack will then calculate a probability for each alignment according to one of three alternative methods. Consider an example read with two possible alignments to different loci in the genome (green-colored read in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). Each of the two possible loci in this example have different “neighborhoods” of adjacent alignments of other reads. In random-placement mode, the probability of originating from a locus is simply 1 / <em>n</em>, where <em>n</em> is the number of possible alignments; in the example read from <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, each position thus has a probability of 50% (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). This emulates the default behavior of bowtie (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Langmead <em>et al.</em> 2009</a>) and bwa (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Li and Durbin 2009</a>). In unique-weighting mode (U), the frequencies of uniquely aligned reads mapping within the vicinity of the alignment under consideration are tallied. For the example read in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, locus 1 has seven uniquely aligned reads in the vicinity, while locus 2 has only one (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a>). Thus, in U mode, the probability of the example read originating from locus 1 is calculated as 87.5%, while the probability of originating from locus is calculated as 12.5% (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2D</a>). The fractional method (F) uses all reads mapping within the vicinity of an alignment. Unique reads in the vicinity lend full weight, while MMAP reads provide weights inversely proportional to their MMAP-value. For the example read in <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>, the weighted probability for originating from locus 1 is thus calculated at 73.7%, while the probability for originating from locus 2 is calculated as 26.3% (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2E</a>). The calculated probabilities are then used as weightings in a random number selection to designate the primary alignment for the read (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2F</a>), and all of the other possible alignments are marked as secondary alignments. Alternatively, ShortStack can be instructed to simply ignore (N) all MMAP reads (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2G</a>), emulating the behavior of bowtie with option -m 1 set, or that of Novoalign’s default settings.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of ShortStack methodology." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-270602679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overview of ShortStack methodology.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2103/F1.medium.gif" width="440" height="169"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/7/2103/F1.medium.gif" width="440" height="169"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468559" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Overview of ShortStack methodology.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="ShortStack3 alignment methodology. (A) First step in alignment by ShortStack: initial alignment of sRNA-seq reads to a reference genome. (B) Example of local alignments for a read (green) with an MMAP-value of two. Numbers adjacent to reads indicate their MMAP-value. (C) Weighting scheme for random placement of MMAP reads. (D) Weighting scheme for ShortStack&#x2019;s Unique (U) method. (E) Weighting scheme for ShortStack&#x2019;s fractional (F) method. (F) Final step: choosing a single primary alignment based on calculated probabilities. (G) Alignment tools grouped by MMAP methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-270602679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;ShortStack3 alignment methodology. (A) First step in alignment by ShortStack: initial alignment of sRNA-seq reads to a reference genome. (B) Example of local alignments for a read (green) with an MMAP-value of two. Numbers adjacent to reads indicate their MMAP-value. (C) Weighting scheme for random placement of MMAP reads. (D) Weighting scheme for ShortStack&#x2019;s Unique (U) method. (E) Weighting scheme for ShortStack&#x2019;s fractional (F) method. (F) Final step: choosing a single primary alignment based on calculated probabilities. (G) Alignment tools grouped by MMAP methods.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2103/F2.medium.gif" width="440" height="350"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/7/2103/F2.medium.gif" width="440" height="350"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468676" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-11" class="first-child">ShortStack3 alignment methodology. (A) First step in alignment by ShortStack: initial alignment of sRNA-seq reads to a reference genome. (B) Example of local alignments for a read (green) with an MMAP-value of two. Numbers adjacent to reads indicate their MMAP-value. (C) Weighting scheme for random placement of MMAP reads. (D) Weighting scheme for ShortStack’s Unique (U) method. (E) Weighting scheme for ShortStack’s fractional (F) method. (F) Final step: choosing a single primary alignment based on calculated probabilities. (G) Alignment tools grouped by MMAP methods.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">The vicinities used in methods U and F are obtained by first dividing the reference genome into 50-nt bins. The bin of a given alignment is defined as the location of its left-most aligned nucleotide. The vicinity is then defined as the total number of reads in a five-bin window, with the center bin being that of the alignment in question. Thus, the vicinity is essentially a 250-nt window centered on the alignment in question. When run in modes U or F, MMAP reads for which the computed probabilities at all positions are equal remain a random guess. In those cases, such reads will be suppressed entirely (marked as unmapped) if they had more than three possible alignment positions (this threshold is user-adjustable).</p><p id="p-13">It is important to note that the U and F weighting methods are not ’winner-take-all’. For instance, if the U method was used for the example in <a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2</a>, there would be a 12.5% probability that the less-likely position would be marked as the primary alignment. Similarly, MMAP placement decisions are made on per-read basis, not a per-sequence basis. For example, 100 MMAP sRNA-seq reads of identical sequence, with two possible alignment positions of probabilities 80% and 20%, we expect ∼80 of the reads to be placed in the first position, and ∼20 of them to be placed in the second position.</p></div><div id="sec-3" class="subsection"><h3>Data sources</h3><p id="p-14">sRNA-seq libraries from <em>Arabidopsis thaliana</em>, <em>Oryza sativa</em>, and <em>Zea mays</em> were obtained from the NCBI Sequence Read Archive (SRA) (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/TableS1.xlsx">Table S1</a>). Libraries were selected that had &gt; 5 million raw reads, were available in an unprocessed format, and were derived from an Illumina instrument. 3′-Adapter sequences were discovered using find_3p_adapter.pl (available at <a href="http://sites.psu.edu/axtell/">http://sites.psu.edu/axtell/</a>), and removed using ShortStack’s internal adapter trimming protocol. Simulated sRNA-seq libraries were produced to closely emulate real sRNA-seq data. This process was accomplished through a custom python script and wrapper run under default settings: sRNA-simulator.py (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/FileS1.txt">File S1</a>). This script uses a real sRNA-seq library as the basis for each simulated library. Real sRNA-seq libraries were aligned using bowtie (<a id="xref-ref-11-4" class="xref-bibr" href="#ref-11">Langmead <em>et al.</em> 2009</a>) reporting all alignments. Regions of the genome that had no alignments were removed from consideration as simulated loci, while genomic regions prone to alignments with certain length classes of sRNAs became candidate regions for simulated heterochromatic siRNA (hc-siRNA; 23–24 nt) and <em>trans</em>-acting siRNA (21 nt) loci. miRNA candidate regions were picked based on prior annotated loci, available through miRBase (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Kozomara and Griffiths-Jones 2014</a>). Simulated loci were chosen from these candidate regions at random. Five million reads were then generated from these simulated loci, generating roughly 3.25 M hc-siRNA, 1.5 M miRNA, and 250 k tasiRNA reads. Loci were made to approximate real loci in size and pattern: hc-siRNA as primarily 24 nt RNAs from 200- to 1000-nt loci, from both genomic strands; miRNA as 21-nt RNAs from 125-nt loci with a miRNA and miRNA* pattern; tasiRNA as 21-nt RNAs from 140-nt loci producing a number of phased reads, from both genomic strands. All three loci types produced a realistic distribution of differently sized or shifted reads to simulate misprocessing. Sequencing errors are simulated at a rate of one mis-sequenced base per 10,000 reads. Unlike real data, simulated reads are traceable to their loci of origin, and thus are suitable to discern correct placements from incorrect ones. PolyA+ mRNA-seq data were obtained from SRA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/TableS1.xlsx">Table S1</a>). Reference genome versions were TAIR10 (<em>A. thaliana</em>), IRGSP7 (<em>O. sativa</em>), and B73v3 (<em>Z. mays</em>).</p></div><div id="sec-4" class="subsection"><h3>Alignments and analyses</h3><p id="p-15">sRNA-seq libraries were aligned using ShortStack (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Axtell 2013</a>), bowtie (<a id="xref-ref-11-5" class="xref-bibr" href="#ref-11">Langmead <em>et al.</em> 2009</a>), bwa (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Li and Durbin 2009</a>), and Novoalign (Novocraft.com). Specific versions and settings are specified in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/TableS2.xlsx">Table S2</a>. mRNA-seq libraries were trimmed using cutadapt (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Martin 2011</a>), or processed as needed to depair paired-end datasets, and aligned using tophat2 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Kim <em>et al.</em> 2013</a>) with bowtie (<a id="xref-ref-11-6" class="xref-bibr" href="#ref-11">Langmead <em>et al.</em> 2009</a>) as its alignment algorithm. Data processing details for each analysis are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/TableS1.xlsx">Table S1</a>.</p></div><div id="sec-5" class="subsection"><h3>Analysis of Pol IV / RDR2-dependent siRNAs</h3><p id="p-16">Candidate RNA polymerase IV / RDR2 (P4R2)-dependent siRNA precursors were obtained from RNA-seq data from <em>A. thaliana dcl234</em> triple mutants (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Li <em>et al.</em> 2015</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Blevins <em>et al.</em> 2015</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Zhai <em>et al.</em> 2015</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ye <em>et al.</em> 2016</a>). Alignment was performed using bowtie, tolerating no mismatches, and retaining only uniquely aligned reads between 28 and 60 nt in length. These reads were then used to computationally generate four, 24-nt siRNA ‘daughters’ from each putative precursor, corresponding to both ends of top and bottom strands of the presumed duplex. Computational daughters that were actually sequenced in the corresponding wild-type sRNA-seq libraries (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/TableS1.xlsx">Table S1</a>) were noted. The wild-type sRNA-seq data were aligned using different alignment approaches. The precision of the tracked MMAP daughter reads was assessed using the coordinates of the corresponding uniquely aligned precursor as the known true origins.</p></div><div id="sec-6" class="subsection"><h3>sRNA-seq</h3><p id="p-17">Total RNA was isolated from wild-type Col-0 <em>A. thaliana</em> inflorescences using Tri-Reagent per the manufacturer’s instructions (Thermo Fisher, 4368814). sRNA-seq libraries were constructed using the Tru-Seq sRNA kit (Illumina, RS-200-0012) as per the manufacturer’s instructions, and sequenced on an Illumina Hi-Sequation 2500. sRNA-seq data has been deposited at NCBI GEO under accession GSE76281 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/TableS1.xlsx">Table S1</a>).</p></div><div id="sec-7" class="subsection"><h3>qRT-PCR of primary miRNA transcripts</h3><p id="p-18">Total RNA was isolated from wild-type Col-0 <em>A. thaliana</em> inflorescences using Tri-Reagent as per the manufacturer’s instructions (Thermo Fisher, AM9738). Primers were designed using the NCBI PrimerBlast tool (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ye <em>et al.</em> 2012</a>), and targeted the 3′ region downstream of the Dicer-Like (DCL) cleavage sites. Primer sequences are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/TableS3.xlsx">Table S3</a>. cDNA was synthesized using a reverse transcription kit (Thermo Fisher, 4368814), and qRT-PCR experiments run on a Life Technologies StepOne Plus real-time PCR instrument using SYBR Green-based master mix (Quanta Biosciences, 95073-012) as per the manufacturer’s instructions. Expression is calculated as the fluorescence relative to actin (ACT2), calculated using grouped PCR efficiencies from analysis by LinRegPCR (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ramakers <em>et al.</em> 2003</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Čikoš <em>et al.</em> 2007</a>).</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-19">Information on sequencing libraries generated in this study are available through data in public repositories: NCBI GEO, accession number GSE76281. Information and accessions for all other libraries used are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/TableS1.xlsx">Table S1</a>. The sRNA-seq simulator script is provided as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/FileS1.txt">File S1</a> and all other scripts used in analysis are available upon request. Sequences and annotation information for miRNAs were acquired through miRBase (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Kozomara and Griffiths-Jones 2014</a>).</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Prevalence of MMAP reads in sRNA-seq data</h3><p id="p-20">In a survey of polyA+ mRNA-seq data from three plant species, MMAP percentages above 10% are rare (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). In contrast, sRNA-seq datasets from the same three species frequently have &gt; 50% MMAP reads (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). This illustrates the extent of the MMAP problem specifically in sRNA-seq data. We examined a collection of studies (<em>n</em> = 20) where plant sRNA-seq data were aligned to a reference genome (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/TableS4.xlsx">Table S4</a>). In 80% of cases, MMAP reads were handled by random selection, where a single possible alignment is randomly selected for each MMAP read. Although computationally easy, random selection is highly imprecise, as most random choices will be wrong. Another 10% of the studies examined simply ignored MMAP reads. This method has the clear disadvantage of discarding huge amounts of data, often &gt; 50% (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>). This survey of data and published studies clearly indicates that MMAP reads are a large issue in sRNA-seq data analysis that are often dealt with ineffectively.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prevalence of MMAP reads and methods. (A) MMAP rates for reads from mRNA-seq and sRNA-seq data from A. thaliana (At), O. sativa (Os), and Z. mays (Zm). Horizontal bars mark median values, circles mark values for individual libraries. (B) Proportion of MMAP-selection methods for sRNA alignment in recent literature (n&#xA0;=&#xA0;20; Table S4)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-270602679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prevalence of MMAP reads and methods. (A) MMAP rates for reads from mRNA-seq and sRNA-seq data from &lt;em&gt;A. thaliana&lt;/em&gt; (At), &lt;em&gt;O. sativa&lt;/em&gt; (Os), and &lt;em&gt;Z. mays&lt;/em&gt; (Zm). Horizontal bars mark median values, circles mark values for individual libraries. (B) Proportion of MMAP-selection methods for sRNA alignment in recent literature (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;20; Table S4).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2103/F3.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/7/2103/F3.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468631" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Prevalence of MMAP reads and methods. (A) MMAP rates for reads from mRNA-seq and sRNA-seq data from <em>A. thaliana</em> (At), <em>O. sativa</em> (Os), and <em>Z. mays</em> (Zm). Horizontal bars mark median values, circles mark values for individual libraries. (B) Proportion of MMAP-selection methods for sRNA alignment in recent literature (<em>n</em> = 20; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1/TableS4.xlsx">Table S4</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>High-precision alignments of MMAP sRNA-seq reads</h3><p id="p-22">We designed two alternative modes of proximity-based weighting systems (U and F) to place MMAP reads, and implemented them in the sRNA-seq analysis software ShortStack (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2</a>) (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Axtell 2013</a>). We then conducted a performance analysis comparing these methods to several other aligners with various settings. The alignment methods can be classified into three groups based on their treatment of MMAP reads (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2</a>): ShortStack-U and -F both use a proximity-weighted scheme to influence placement of MMAP reads. In contrast, under their default settings, bowtie (<a id="xref-ref-11-7" class="xref-bibr" href="#ref-11">Langmead <em>et al.</em> 2009</a>) and bwa (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Li and Durbin 2009</a>) randomly place MMAP reads, as will ShortStack when run in the R mode. Finally, Novoalign, bowtie with setting -m 1, and ShortStack in mode N, all will simply ignore MMAP reads. We simulated sRNA-seq data based on the properties of real datasets from <em>A. thaliana</em>, <em>O. sativa</em>, and <em>Z. mays</em>. Unlike real data, the ‘correct’ alignment location for each MMAP is known for simulated data. This allows direct comparison of aligner performance. Specifically, we can count true positives (aligned reads that are put in the correct position; TP), false positives (aligned reads that are put in the incorrect position; FP), true negatives (unaligned reads that have no possible alignment position; TN), and false negatives (unaligned reads where there is at least one valid alignment position; FN). Precision [TP / (TP + FP)] and sensitivity [TP / (TP + FN)] can then be calculated in the standard manner.</p><p id="p-23">In nearly all cases, the highest precisions were obtained with ShortStack’s U method (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). ShortStack-U had significantly (Kruskal-Wallis ANOVA with Dunn multiple comparison test, α = 0.05) higher precisions than any non-ShortStack method. The results were especially striking when examining the precision specifically for MMAP reads. ShortStack’s U method routinely achieved precision values of ≥ 75% (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). In contrast, none of the random-placement, or ignore MMAP, methods ever had MMAP precision values &gt; 50%. The high precision does come at the cost of sensitivity: as described in <em>Materials and Methods</em>, ShortStack-U and -F will not place all MMAP reads. Instead, if a read has too many possible positions (default is &gt; 50), or if the weighted probabilities are equal, MMAP reads are suppressed. Depressed sensitivities are more apparent in the more repetitive genome of <em>Z. mays</em> as compared to <em>A. thaliana</em> (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). We computed F1 scores (harmonic means of precision and sensitivity) to assess the balance between precision and sensitivity. By this metric, ShortStack-U and -F both performed significantly (Kruskal-Wallis ANOVA with Dunn multiple comparison test, α = 0.05) better than any other method in all three species (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance analysis of sRNA-seq alignment methods. (A) Precisions, sensitivities, and F1 scores for alignments of simulated sRNA-seq data with the indicated methods for entire datasets. Boxplots show medians (central bars), the 1st to 3rd quartile range (boxes), other data out to 1.5 the interquartile range (whiskers), and outliers (dots); n&#xA0;=&#xA0;15, 12, and 21 for the At, Os, and Zm data, respectively. Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, &#x3B1;&#xA0;=&#xA0;0.05). (B) MMAP reads only. Same analysis and conventions as in (A). (C) False negative rates for alignments of real sRNA-seq data with the indicated methods for MMAP reads. Plotting conventions as in (A). At, A. thaliana; Os, O. sativa; Zm, Z. mays." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-270602679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Performance analysis of sRNA-seq alignment methods. (A) Precisions, sensitivities, and F1 scores for alignments of simulated sRNA-seq data with the indicated methods for entire datasets. Boxplots show medians (central bars), the 1st to 3rd quartile range (boxes), other data out to 1.5 the interquartile range (whiskers), and outliers (dots); &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;15, 12, and 21 for the At, Os, and Zm data, respectively. Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, &#x3B1;&#xA0;=&#xA0;0.05). (B) MMAP reads only. Same analysis and conventions as in (A). (C) False negative rates for alignments of real sRNA-seq data with the indicated methods for MMAP reads. Plotting conventions as in (A). At, &lt;em&gt;A. thaliana&lt;/em&gt;; Os, &lt;em&gt;O. sativa&lt;/em&gt;; Zm, &lt;em&gt;Z. mays&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2103/F4.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/7/2103/F4.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468547" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">Performance analysis of sRNA-seq alignment methods. (A) Precisions, sensitivities, and F1 scores for alignments of simulated sRNA-seq data with the indicated methods for entire datasets. Boxplots show medians (central bars), the 1st to 3rd quartile range (boxes), other data out to 1.5 the interquartile range (whiskers), and outliers (dots); <em>n</em> = 15, 12, and 21 for the At, Os, and Zm data, respectively. Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, α = 0.05). (B) MMAP reads only. Same analysis and conventions as in (A). (C) False negative rates for alignments of real sRNA-seq data with the indicated methods for MMAP reads. Plotting conventions as in (A). At, <em>A. thaliana</em>; Os, <em>O. sativa</em>; Zm, <em>Z. mays</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Analysis of real libraries is limited to examination of false negative rates [FNR = FN / (FP + TP)], as TP and FP cannot be distinguished. As expected, FNRs are higher in <em>Z. mays</em> and <em>O. sativa</em> relative to <em>A. thaliana</em>, reflecting the higher proportions of reads with high MMAP-values from the more repetitive genomes (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a>). Under default settings, ShortStack’s U and F methods discard ∼30–50% of the MMAP reads from rice and maize, but are likely placing the remainder with very high confidence.</p></div><div id="sec-12" class="subsection"><h3>Precision is a function of MMAP-value</h3><p id="p-26">As discussed above, our methods by default will not attempt weighting or placement of MMAP reads with &gt; 50 possible alignment positions. This is based on the prediction that, for reads with very high MMAP-values, we will have an unacceptably high error rate. We tested this prediction by examining the precision values in our simulated datasets as a function of MMAP value (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A and B</a>). As expected, precision declines as a function of higher MMAP values in all cases (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), with random having the most severe drop-off. Notably, the U method consistently gives the highest precisions at a given MMAP value. Analysis of cumulative precisions showed that plateaus of precision generally became apparent at MMAP-values ∼50 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>), justifying our default settings to ignore reads with &gt; 50 possible alignment positions. Our simulated data had more high MMAP reads than real data (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>), indicating that aligner performance with real data is likely to be superior to that estimated using simulated reads.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Influence of MMAP-value on performance. (A) Precision as a function of MMAP-value for simulated sRNA-seq data from the indicated species and alignment method. MMAP-value is the number of possible alignment positions for a read. Colored lines are standard deviations, black dots are mean values. Heavy dashed line at MMAP&#xA0;=&#xA0;50 indicates the default cutoff value for ShortStack, above which placement of MMAP reads is not attempted. (B) Cumulative precision as a function of MMAP-value for simulated sRNA-seq data from the indicated species and alignment method. Plotting conventions as in (A). (C) Cumulative proportion of real and simulated sRNA-seq data retained by ShortStack alignments under differing MMAP-value cutoffs. Note that simulated libraries have higher proportions of reads with high MMAP values. Plotting conventions as in (A). At, A. thaliana; Os. O. sativa; Zm, Z. mays." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-270602679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Influence of MMAP-value on performance. (A) Precision as a function of MMAP-value for simulated sRNA-seq data from the indicated species and alignment method. MMAP-value is the number of possible alignment positions for a read. Colored lines are standard deviations, black dots are mean values. Heavy dashed line at MMAP&#xA0;=&#xA0;50 indicates the default cutoff value for ShortStack, above which placement of MMAP reads is not attempted. (B) Cumulative precision as a function of MMAP-value for simulated sRNA-seq data from the indicated species and alignment method. Plotting conventions as in (A). (C) Cumulative proportion of real and simulated sRNA-seq data retained by ShortStack alignments under differing MMAP-value cutoffs. Note that simulated libraries have higher proportions of reads with high MMAP values. Plotting conventions as in (A). At, &lt;em&gt;A. thaliana&lt;/em&gt;; Os. &lt;em&gt;O. sativa&lt;/em&gt;; Zm, &lt;em&gt;Z. mays&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2103/F5.medium.gif" width="302" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/7/2103/F5.medium.gif" width="302" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468687" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-27" class="first-child">Influence of MMAP-value on performance. (A) Precision as a function of MMAP-value for simulated sRNA-seq data from the indicated species and alignment method. MMAP-value is the number of possible alignment positions for a read. Colored lines are standard deviations, black dots are mean values. Heavy dashed line at MMAP = 50 indicates the default cutoff value for ShortStack, above which placement of MMAP reads is not attempted. (B) Cumulative precision as a function of MMAP-value for simulated sRNA-seq data from the indicated species and alignment method. Plotting conventions as in (A). (C) Cumulative proportion of real and simulated sRNA-seq data retained by ShortStack alignments under differing MMAP-value cutoffs. Note that simulated libraries have higher proportions of reads with high MMAP values. Plotting conventions as in (A). At, <em>A. thaliana</em>; Os. <em>O. sativa</em>; Zm, <em>Z. mays</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Experimental testing of alignment methods</h3><p id="p-28">Mature miRNAs are frequently encoded by multiple paralogous loci, which causes sRNA-seq reads corresponding to the mature miRNA to be MMAPed. However, the primary transcripts derived from each of the paralogs typically are much longer, and have distinct sequences, such that qRT-PCR approaches can distinguish them. We used accumulation levels of primary transcripts as proxies for the true expression levels of various <em>A. thaliana MIRNA</em> loci that encode identical mature miRNAs, and compared those data to estimates derived from small RNA-seq alignments (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). qRT-PCR indicated sharp differences in primary transcript accumulation for each of the three miRNA families we studied (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). ShortStack’s U method produced mature miRNA alignments that were mostly similar to the results from qRT-PCR of the primary transcripts (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). In contrast, the other alignment methods suggested nearly equal accumulation of mature miRNAs from each of the paralogs. We quantified the fit between the qRT-PCR data and the sRNA-seq alignment data by analyzing the squared residual errors between the scaled expression levels (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6C</a>). By this metric, ShortStack-U predicts the specific contributions of miRNA paralogs significantly (Kruskal-Wallis ANOVA with Dunn multiple comparison test, α = 0.05) better than all other tested methods.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Experimental assessment of sRNA-seq alignment methods using miRNA paralogs. (A) Relative expression of the indicated primary MIRNA transcripts in A. thaliana Col-0 inflorescences assessed via qRT-PCR. Values are normalized to 1&#xA0;/&#xA0;1000 those of ACTIN2. Dots show values from biological replicates (n&#xA0;=&#xA0;3). (B) Accumulation of the indicated mature miRNAs from each of their possible paralogs as determined by different sRNA-seq alignment methods. Values are from three biological replicate sRNA-seq libraries from A. thaliana Col-0 inflorescences. (C) Squared residual errors from comparisons of scaled qRT-PCR data to scaled sRNA-seq alignment results. Boxplots show medians (horizontal bars), the 1st to 3rd quartile range (boxes), data out to 1.5 times the interquartile range (whiskers), and outliers (dots). Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, &#x3B1;&#xA0;=&#xA0;0.05)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-270602679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Experimental assessment of sRNA-seq alignment methods using miRNA paralogs. (A) Relative expression of the indicated primary &lt;em&gt;MIRNA&lt;/em&gt; transcripts in &lt;em&gt;A. thaliana&lt;/em&gt; Col-0 inflorescences assessed via qRT-PCR. Values are normalized to 1&#xA0;/&#xA0;1000 those of &lt;em&gt;ACTIN2&lt;/em&gt;. Dots show values from biological replicates (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;3). (B) Accumulation of the indicated mature miRNAs from each of their possible paralogs as determined by different sRNA-seq alignment methods. Values are from three biological replicate sRNA-seq libraries from &lt;em&gt;A. thaliana&lt;/em&gt; Col-0 inflorescences. (C) Squared residual errors from comparisons of scaled qRT-PCR data to scaled sRNA-seq alignment results. Boxplots show medians (horizontal bars), the 1st to 3rd quartile range (boxes), data out to 1.5 times the interquartile range (whiskers), and outliers (dots). Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, &#x3B1;&#xA0;=&#xA0;0.05).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2103/F6.medium.gif" width="210" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/7/2103/F6.medium.gif" width="210" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468574" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-29" class="first-child">Experimental assessment of sRNA-seq alignment methods using miRNA paralogs. (A) Relative expression of the indicated primary <em>MIRNA</em> transcripts in <em>A. thaliana</em> Col-0 inflorescences assessed via qRT-PCR. Values are normalized to 1 / 1000 those of <em>ACTIN2</em>. Dots show values from biological replicates (<em>n</em> = 3). (B) Accumulation of the indicated mature miRNAs from each of their possible paralogs as determined by different sRNA-seq alignment methods. Values are from three biological replicate sRNA-seq libraries from <em>A. thaliana</em> Col-0 inflorescences. (C) Squared residual errors from comparisons of scaled qRT-PCR data to scaled sRNA-seq alignment results. Boxplots show medians (horizontal bars), the 1st to 3rd quartile range (boxes), data out to 1.5 times the interquartile range (whiskers), and outliers (dots). Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, α = 0.05).</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">When DCL enzymes are absent from <em>A. thaliana</em>, longer double-stranded RNAs accumulate that are dependent on RNA polymerase IV (Pol IV) and RNA-dependent RNA polymerase 2 (RDR2) (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Li <em>et al.</em> 2015</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Blevins <em>et al.</em> 2015</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Zhai <em>et al.</em> 2015</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Yang <em>et al.</em> 2016</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Ye <em>et al.</em> 2016</a>). These are likely to be the direct substrates for the DCL3-mediated production of 24-nt siRNAs by excision from the ends of precursors. We identified a set of longer precursors from <em>dcl2/dcl3/dcl4</em> triple mutant plants that could be uniquely mapped to the reference genome. We then computationally diced this set of precursors to create predicted mature 24-nt siRNAs, and examined the corresponding wild-type sRNA-seq libraries to find cases where the putative mature siRNAs actually were sequenced. The wild-type sRNA-seq libraries were aligned with various methods. Cases where the mature siRNAs of interest were MMAP allowed assessment of alignment precision, as we presume their uniquely aligned precursors represent the correct alignment positions. ShortStack-U significantly (Kruskal-Wallis ANOVA with Dunn multiple comparison test, α = 0.05) outperformed other alignment methods (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>) in this analysis, as measured by precision. Thus, experimental data from both miRNAs and endogenous siRNAs support the hypothesis that ShortStack’s U alignment method is superior.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Precisions from alignments of Arabidopsis thaliana MMAP 24 nt siRNAs whose true origins are known based on a unique precursor alignment. Dots, data from individual libraries; horizontal bars, medians. Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, &#x3B1;&#xA0;=&#xA0;0.05)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-270602679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Precisions from alignments of &lt;em&gt;Arabidopsis thaliana&lt;/em&gt; MMAP 24 nt siRNAs whose true origins are known based on a unique precursor alignment. Dots, data from individual libraries; horizontal bars, medians. Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, &#x3B1;&#xA0;=&#xA0;0.05).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2103/F7.medium.gif" width="440" height="333"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/7/2103/F7.medium.gif" width="440" height="333"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468433" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-31" class="first-child">Precisions from alignments of <em>Arabidopsis thaliana</em> MMAP 24 nt siRNAs whose true origins are known based on a unique precursor alignment. Dots, data from individual libraries; horizontal bars, medians. Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, α = 0.05).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Bowtie shows bias in random read placement</h3><p id="p-32">When run under default settings, bowtie (<a id="xref-ref-11-8" class="xref-bibr" href="#ref-11">Langmead <em>et al.</em> 2009</a>) randomly selects one possible alignment position for MMAP reads. However, the software documentation warns that biased strand selection, where alignment positions from one strand are favored, can occur (<a id="xref-ref-11-9" class="xref-bibr" href="#ref-11">Langmead <em>et al.</em> 2009</a>). For sRNA-seq data, we have observed that this strand bias is quite strong. We observe a strong top-strand bias when bowtie is run under default settings to align sRNA-seq data (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>). Indeed, this can affect estimation of miRNA accumulation from paralogs (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6B</a>, miR166e). This bias is not apparent when other methods are used (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). Small RNA polarity is an important feature used for annotations, so it is conceivable that this strand bias could distort results.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Strand-biased selection of MMAP alignment positions by bowtie. Bias is shown as the overall ratio of top-strand aligned reads to bottom, based on simulated libraries from A. thaliana (n&#xA0;=&#xA0;15). Boxplots show medians (horizontal bars), the 1st to 3rd quartile range (boxes), data out to 1.5 times the interquartile range (whiskers), and outliers (dots). Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, &#x3B1;&#xA0;=&#xA0;0.05)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-270602679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Strand-biased selection of MMAP alignment positions by bowtie. Bias is shown as the overall ratio of top-strand aligned reads to bottom, based on simulated libraries from &lt;em&gt;A. thaliana&lt;/em&gt; (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;15). Boxplots show medians (horizontal bars), the 1st to 3rd quartile range (boxes), data out to 1.5 times the interquartile range (whiskers), and outliers (dots). Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, &#x3B1;&#xA0;=&#xA0;0.05).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2103/F8.medium.gif" width="362" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/6/7/2103/F8.medium.gif" width="362" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468722" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-33" class="first-child">Strand-biased selection of MMAP alignment positions by bowtie. Bias is shown as the overall ratio of top-strand aligned reads to bottom, based on simulated libraries from <em>A. thaliana</em> (<em>n</em> = 15). Boxplots show medians (horizontal bars), the 1st to 3rd quartile range (boxes), data out to 1.5 times the interquartile range (whiskers), and outliers (dots). Treatments sharing a common letter indicate groups that are not significantly different by nonparametric analysis (Kruskal-Wallis ANOVA with Dunn multiple comparison test, α = 0.05).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Speed testing</h3><p id="p-34">We compared the alignment speed of our methods with those of other aligners. To account for differences caused by different genome sizes and/or sRNA-seq libraries, we measured speed in normalized units. Because bowtie under default settings was always the fastest method, we normalized each run to a multiple of the bowtie-default completion speed. The time spent on genome-indexing was not part of this assessment. In general, our favored method, ShortStack-U, was 5–10 times slower than bowtie-default, but comparable to bwa (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>). Given that the results of ShortStack-U are superior to the other methods, we feel that the extra time is an acceptable trade-off. In absolute terms, the wall-time for our ShortStack-U runs on these data ranged from 6 min to 2.5 hr, compared with bowtie-default, which ranged from 1 min to 32 min.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of alignment times for real sRNA-seq libraries with the indicated methods. Boxplots show medians (central bars), the 1st to 3rd quartile range (boxes), other data out to 1.5 the interquartile range (whiskers), and outliers (dots); n&#xA0;=&#xA0;15, 12, and 21 for the At, Os, and Zm data, respectively. At, A. thaliana; Os, O. sativa; Zm, Z. mays." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-270602679" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of alignment times for real sRNA-seq libraries with the indicated methods. Boxplots show medians (central bars), the 1st to 3rd quartile range (boxes), other data out to 1.5 the interquartile range (whiskers), and outliers (dots); &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;15, 12, and 21 for the At, Os, and Zm data, respectively. At, &lt;em&gt;A. thaliana&lt;/em&gt;; Os, &lt;em&gt;O. sativa&lt;/em&gt;; Zm, &lt;em&gt;Z. mays&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2103/F9.medium.gif" width="440" height="212"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/6/7/2103/F9.medium.gif" width="440" height="212"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2103/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468498" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-35" class="first-child">Comparison of alignment times for real sRNA-seq libraries with the indicated methods. Boxplots show medians (central bars), the 1st to 3rd quartile range (boxes), other data out to 1.5 the interquartile range (whiskers), and outliers (dots); <em>n</em> = 15, 12, and 21 for the At, Os, and Zm data, respectively. At, <em>A. thaliana</em>; Os, <em>O. sativa</em>; Zm, <em>Z. mays</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Conclusions</h3><p id="p-36">A particularly important goal of ours is to reduce the likelihood of false discoveries in small RNA gene annotation. Reducing the number of incorrect alignments is likely to reduce spurious annotations. ShortStack-U gives fewer false positives for MMAP reads than all other methods for sRNA-seq alignment (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>, and <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>). The experimental evidence identifying miRNA alignment is striking, as mature miRNAs are remarkably poorly placed through standard approaches (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a>). It is likely that misalignment of this nature is significantly misrepresenting the abundance of many loci, conceivably resulting in misannotations. ShortStack-U’s alignment method reduces these risks, striking a balance between precision and sensitivity.</p><p id="p-37">A method that is capable of identifying the true site of origin for all MMAP sRNA-seq reads is elusive. To achieve reasonable precision, our methods accept reduced sensitivity by ignoring highly MMAP reads. However, it should be kept in mind that these reads are part of the small RNA profile too, and should not be completely ignored. Future development of sRNA-seq alignment methods should focus on increasing precisions to allow confident placement of even the most repetitive reads. Until then, these most highly repetitive reads can be analyzed using methods independent of alignment to the reference genome.</p><p id="p-38">Overall, our work demonstrates that sRNA-seq alignments can be significantly improved by using local weighting to guide placement of multi-mapped reads, and that biases in placement of multi-mapped reads can influence downstream analyses.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-39">This work was supported by an award from the United States National Science Foundation (US NSF) to M.J.A. (award # 1339207). sRNA-seq was conducted on an Illumina HiSeq2500 purchased with funds from a US NSF award to M.J.A. (award #1229046).</p><p id="p-40">Author contributions: M.J.A. developed ShortStack code. N.R.J. and M.J.A. developed alignment algorithm ideas and ideas for performance testing. N.R.J. conducted performance analyses and generated all figures. N.R.J. and J.Y. conducted qRT-PCR experiments. C.C. constructed sRNA-seq libraries. M.J.A. and N.R.J. wrote the manuscript, with comments from J.Y. and C.C.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-41">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030452/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-42"><em>Communicating editor: M. Watson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-03-17"><span class="received-label">Received </span>March 17, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-05-08"><span class="accepted-label">Accepted </span>May 8, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Johnson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.1" data-doi="10.1016/j.cell.2005.04.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gustafson</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrington</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">microRNA-directed phasing during trans-acting siRNA biogenesis in plants.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAllen%26rft.auinit1%253DE.%26rft.volume%253D121%26rft.issue%253D2%26rft.spage%253D207%26rft.epage%253D221%26rft.atitle%253DmicroRNA-directed%2Bphasing%2Bduring%2Btrans-acting%2BsiRNA%2Bbiogenesis%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.04.004%26rft_id%253Dinfo%253Apmid%252F15851028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15851028&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228705000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.2" data-doi="10.1038/nature04916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aravin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaidatzis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeffer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagos-Quintana</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landgraf</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A novel class of small RNAs bind to MILI protein in mouse testes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">442</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAravin%26rft.auinit1%253DA.%26rft.volume%253D442%26rft.issue%253D7099%26rft.spage%253D203%26rft.epage%253D207%26rft.atitle%253DA%2Bnovel%2Bclass%2Bof%2Bsmall%2BRNAs%2Bbind%2Bto%2BMILI%2Bprotein%2Bin%2Bmouse%2Btestes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04916%26rft_id%253Dinfo%253Apmid%252F16751777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16751777&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238979700048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.3" data-doi="10.1261/rna.035279.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Axtell</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">ShortStack: comprehensive annotation and quantification of small RNA genes.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">740</span>–<span class="cit-lpage">751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.035279.112%26rft_id%253Dinfo%253Apmid%252F23610128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE5LzYvNzQwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIxMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.4" data-doi="10.7554/eLife.09591"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blevins</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podicheti</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mishra</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marasco</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of Pol IV and RDR2-dependent precursors of 24 nt siRNAs guiding de novo DNA methylation in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e09591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.09591%26rft_id%253Dinfo%253Apmid%252F26430765%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.09591&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26430765&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.5" data-doi="10.1186/1471-2199-8-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Čikoš</span>  <span class="cit-name-given-names">Š.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bukovská</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koppel</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Relative quantification of mRNA: comparison of methods currently used for real-time PCR data analysis.</span> <abbr class="cit-jnl-abbrev">BMC Mol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bmolecular%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BMol%2BBiol%26rft.aulast%253DFromme%26rft.auinit1%253DT.%26rft.volume%253D8%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DChicken%2Bovalbumin%2Bupstream%2Bpromoter%2Btranscription%2Bfactor%2BII%2Bregulates%2Buncoupling%2Bprotein%2B3%2Bgene%2Btranscription%2Bin%2BPhodopus%2Bsungorus.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2199-8-1%26rft_id%253Dinfo%253Apmid%252F17204145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2199-8-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17204145&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.6" data-doi="10.1016/j.pbi.2014.02.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coruh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shahid</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axtell</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Seeing the forest for the trees: annotating small RNA producing genes in plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BPlant%2BBiol.%26rft.volume%253D18%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2014.02.008%26rft_id%253Dinfo%253Apmid%252F24632306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2014.02.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24632306&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.7" data-doi="10.1105/tpc.110.082784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuperus</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fahlgren</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrington</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution and functional diversification of MIRNA genes.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.110.082784%26rft_id%253Dinfo%253Apmid%252F21317375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvNDMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIxMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.8" data-doi="10.1016/j.ygeno.2007.11.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faulkner</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">A. R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chalk</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroder</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashizaki</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A rescue strategy for multimapping short sequence tags refines surveys of transcriptional activity by CAGE.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DFaulkner%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D91%26rft.issue%253D3%26rft.spage%253D281%26rft.epage%253D288%26rft.atitle%253DA%2Brescue%2Bstrategy%2Bfor%2Bmultimapping%2Bshort%2Bsequence%2Btags%2Brefines%2Bsurveys%2Bof%2Btranscriptional%2Bactivity%2Bby%2BCAGE.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2007.11.003%26rft_id%253Dinfo%253Apmid%252F18178374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2007.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18178374&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253475900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.9" data-doi="10.1186/gb-2013-14-4-r36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-4-r36%26rft_id%253Dinfo%253Apmid%252F23618408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-4-r36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23618408&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.10" data-doi="10.1093/nar/gkt1181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kozomara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths-Jones</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">miRBase: annotating high confidence microRNAs using deep sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D68</span>–<span class="cit-lpage">D73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1181%26rft_id%253Dinfo%253Apmid%252F24275495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24275495&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.11" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.12" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.13" data-doi="10.1101/gr.182238.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandivier</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Won</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Detection of Pol IV/RDR2-dependent transcripts at the genomic scale in <em>Arabidopsis</em> reveals features and regulation of siRNA biogenesis.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.182238.114%26rft_id%253Dinfo%253Apmid%252F25414514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzIvMjM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIxMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.14" data-doi="10.1016/j.cell.2009.03.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malone</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brennecke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCombie</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Specialized piRNA pathways act in germline and somatic tissues of the <em>Drosophila</em> ovary.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">522</span>–<span class="cit-lpage">535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMalone%26rft.auinit1%253DC.%2BD.%26rft.volume%253D137%26rft.issue%253D3%26rft.spage%253D522%26rft.epage%253D535%26rft.atitle%253DSpecialized%2BpiRNA%2Bpathways%2Bact%2Bin%2Bgermline%2Band%2Bsomatic%2Btissues%2Bof%2Bthe%2BDrosophila%2Bovary.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.03.040%26rft_id%253Dinfo%253Apmid%252F19395010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.03.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19395010&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265677200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal 17: 10–12.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.16" data-doi="10.1038/nrg3683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matzke</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RNA-directed DNA methylation: an epigenetic pathway of increasing complexity.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">394</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D394%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3683%26rft_id%253Dinfo%253Apmid%252F24805120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24805120&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.17" data-doi="10.1038/nmeth.1226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCue</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wold</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DMortazavi%26rft.auinit1%253DA.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253D621%26rft.epage%253D628%26rft.atitle%253DMapping%2Band%2Bquantifying%2Bmammalian%2Btranscriptomes%2Bby%2BRNA-Seq.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1226%26rft_id%253Dinfo%253Apmid%252F18516045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516045&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257166700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.18" data-doi="10.1093/bioinformatics/btn428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moxon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwach</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalmay</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maclean</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Studholme</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A toolkit for analysing large-scale plant small RNA datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2252</span>–<span class="cit-lpage">2253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn428%26rft_id%253Dinfo%253Apmid%252F18713789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn428&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18713789&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259581900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.19" data-doi="10.1038/nature07015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">W.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruby</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The <em>Drosophila</em> hairpin RNA pathway generates endogenous short interfering RNAs.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">453</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DOkamura%26rft.auinit1%253DK.%26rft.volume%253D453%26rft.issue%253D7196%26rft.spage%253D803%26rft.epage%253D806%26rft.atitle%253DThe%2BDrosophila%2Bhairpin%2BRNA%2Bpathway%2Bgenerates%2Bendogenous%2Bshort%2Binterfering%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07015%26rft_id%253Dinfo%253Apmid%252F18463630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18463630&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256415300048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.20" data-doi="10.1016/S0304-3940(02)01423-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramakers</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruijter</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deprez</span>  <span class="cit-name-given-names">R. H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moorman</span>  <span class="cit-name-given-names">A. F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Assumption-free analysis of quantitative real-time polymerase chain reaction (PCR) data.</span> <abbr class="cit-jnl-abbrev">Neurosci. Lett.</abbr> <span class="cit-vol">339</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuroscience%2Bletters%26rft.stitle%253DNeurosci%2BLett%26rft.aulast%253DRamakers%26rft.auinit1%253DC.%26rft.volume%253D339%26rft.issue%253D1%26rft.spage%253D62%26rft.epage%253D66%26rft.atitle%253DAssumption-free%2Banalysis%2Bof%2Bquantitative%2Breal-time%2Bpolymerase%2Bchain%2Breaction%2B%2528PCR%2529%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0304-3940%252802%252901423-4%26rft_id%253Dinfo%253Apmid%252F12618301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0304-3940(02)01423-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12618301&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181411500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.21" data-doi="10.1093/bioinformatics/btu680"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossniklaus</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rcount: simple and flexible RNA-Seq read counting.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu680%26rft_id%253Dinfo%253Apmid%252F25322836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu680&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25322836&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.22" data-doi="10.1093/bioinformatics/bts311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stocks</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moxon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mapleson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolfenden</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohorianu</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The UEA sRNA workbench: a suite of tools for analysing and visualizing next generation sequencing microRNA and small RNA datasets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2059</span>–<span class="cit-lpage">2061</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts311%26rft_id%253Dinfo%253Apmid%252F22628521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts311&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22628521&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306686400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.23" data-doi="10.1016/j.tplants.2013.11.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarver</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiscock</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donoghue</span>  <span class="cit-name-given-names">P. C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolutionary history of plant microRNAs.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2013.11.008%26rft_id%253Dinfo%253Apmid%252F24405820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2013.11.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24405820&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332905200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.24" data-doi="10.1038/cr.2015.145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">D.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Dicer-independent RNA-directed DNA methylation in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRes.%26rft.volume%253D26%26rft.spage%253D66%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2015.145%26rft_id%253Dinfo%253Apmid%252F26642813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2015.145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26642813&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.25" data-doi="10.1186/1471-2105-13-134"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coulouris</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaretskaya</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcutache</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Primer-BLAST: a tool to design target-specific primers for polymerase chain reaction.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DYe%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.spage%253D134%26rft.epage%253D134%26rft.atitle%253DPrimer-BLAST%253A%2Ba%2Btool%2Bto%2Bdesign%2Btarget-specific%2Bprimers%2Bfor%2Bpolymerase%2Bchain%2Breaction.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-134%26rft_id%253Dinfo%253Apmid%252F22708584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-134&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22708584&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.26" data-doi="10.1016/j.molcel.2015.11.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowley</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A dicer-independent route for biogenesis of siRNAs that direct DNA methylation in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">222</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D61%26rft.spage%253D222%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.11.015%26rft_id%253Dinfo%253Apmid%252F26711010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.11.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26711010&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2103.27" data-doi="10.1016/j.cell.2015.09.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bischof</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T.-F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A one precursor one siRNA model for Pol IV-dependent siRNA biogenesis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">445</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D163%26rft.spage%253D445%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.09.032%26rft_id%253Dinfo%253Apmid%252F26451488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.09.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26451488&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/7/2103.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/2091" title="Dynamics of a Novel Highly Repetitive CACTA Family in Common Bean (Phaseolus vulgaris)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/2113" title="Comparative Genomics of a Parthenogenesis-Inducing Wolbachia Symbiont" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/7"><span>Volume 6 Issue 7, July 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/7.cover-source.jpg?itok=E84OOyGI" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2103.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cygf4KzhjAIzfOkznGxhIQZILRlsuskap0525SCVZjw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/7/2103" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/7/2103" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/7/2103" class="active" data-icon-position="" data-hide-link-title="0">Improved Placement of Multi-mapping Small RNAs</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460422" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-emnsuIY9VDLZcb3Ps5jUJE44pXk8s5sykj1H05-Ukzo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/2103.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/7/2103" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-tgnc5Xu_aDipqN2e_A927DPGuW8T47dquZo6bm7tFhM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/7/2103" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/2103.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460422" id="node460422--2" data-pisa="ggg;6/7/2103" data-pisa-master="ggg;g3.116.030452" data-apath="/ggg/6/7/2103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improved Placement of Multi-mapping Small RNAs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5279-9964" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathan R. Johnson</span>, <span class="highwire-citation-author" data-delta="1">Jonathan M. Yeoh</span>, <span class="highwire-citation-author" data-delta="2">Ceyda Coruh</span> and <span class="highwire-citation-author" data-delta="3">Michael J. Axtell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2103-2111;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030452 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460422--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nathan R. Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Knox College, Galesburg, Illinois, 61401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BR.%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathan%2BR.%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5279-9964" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathan R. Johnson</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jonathan M. Yeoh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Knox College, Galesburg, Illinois, 61401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BM.%2BYeoh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeoh%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BM.%2BYeoh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ceyda Coruh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Penn State University, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ceyda%2BCoruh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coruh%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACeyda%2BCoruh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael J. Axtell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Penn State University, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BAxtell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Axtell%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BAxtell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mja18@psu.edu" class="" data-icon-position="" data-hide-link-title="0">mja18@psu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/7/2103" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/7/2103&amp;title=Improved%20Placement%20of%20Multi-mapping%20Small%20RNAs" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/7/2103&amp;title=Improved%20Placement%20of%20Multi-mapping%20Small%20RNAs" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/7/2103&amp;title=Improved%20Placement%20of%20Multi-mapping%20Small%20RNAs" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/7/2103&amp;text=Improved%20Placement%20of%20Multi-mapping%20Small%20RNAs" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/7/2103&amp;title=Improved%20Placement%20of%20Multi-mapping%20Small%20RNAs" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/7/2103&amp;t=Improved%20Placement%20of%20Multi-mapping%20Small%20RNAs" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/7/2103&amp;title=Improved%20Placement%20of%20Multi-mapping%20Small%20RNAs" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/7/2103&amp;title=Improved%20Placement%20of%20Multi-mapping%20Small%20RNAs" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460422/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460422" id="node460422--3" data-pisa="ggg;6/7/2103" data-pisa-master="ggg;g3.116.030452" data-apath="/ggg/6/7/2103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Improved Placement of Multi-mapping Small RNAs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5279-9964" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nathan R. Johnson</span>, <span class="highwire-citation-author" data-delta="1">Jonathan M. Yeoh</span>, <span class="highwire-citation-author" data-delta="2">Ceyda Coruh</span> and <span class="highwire-citation-author" data-delta="3">Michael J. Axtell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2103-2111;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030452 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460422--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nathan R. Johnson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Knox College, Galesburg, Illinois, 61401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BR.%2BJohnson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Johnson%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANathan%2BR.%2BJohnson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5279-9964" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nathan R. Johnson</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jonathan M. Yeoh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Knox College, Galesburg, Illinois, 61401</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BM.%2BYeoh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yeoh%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BM.%2BYeoh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ceyda Coruh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Penn State University, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ceyda%2BCoruh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coruh%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACeyda%2BCoruh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael J. Axtell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Huck Institutes of the Life Sciences, Penn State University, University Park, PA 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Biology, Penn State University, University Park, PA 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BAxtell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Axtell%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BAxtell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mja18@psu.edu" class="" data-icon-position="" data-hide-link-title="0">mja18@psu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2103&amp;rft.epage=2111&amp;rft.atitle=Improved%20Placement%20of%20Multi-mapping%20Small%20RNAs&amp;rft.volume=6&amp;rft.issue=7&amp;rft.date=2016-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Johnson%2C+Nathan+R.&amp;rft.au=Yeoh%2C+Jonathan+M.&amp;rft.au=Coruh%2C+Ceyda&amp;rft.au=Axtell%2C+Michael+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460422/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460422/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460422/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460422/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460422/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460422/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460422/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460422/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460422/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460422/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460422/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460422/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/7/2103" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/7/2103.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/7/2103.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/7/2103.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/7/2103.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
