<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/7/1899" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/7/1899.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/7/1899.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/7/1899.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/7/1899.atom" />
<meta name="HW.pisa" content="ggg;6/7/1899" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Patching Holes in the Chlamydomonas Genome" />
<meta name="DC.Identifier" content="10.1534/g3.116.029207" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Tulin, Cross. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Chlamydomonas genome has been sequenced, assembled, and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. However, the current reference genome contains ∼1000 blocks of unknown sequence (‘N-islands’), which are frequently placed in introns of annotated gene models. We developed a strategy to search for previously unknown exons hidden within such blocks, and determine the sequence, and exon/intron boundaries, of such exons. These methods are based on assembly and alignment of short cDNA and genomic DNA reads, completely independent of prior reference assembly or annotation. Our evidence indicates that a substantial proportion of the annotated intronic N-islands contain hidden exons. For most of these, our algorithm recovers full exonic sequence with associated splice junctions and exon-adjacent intronic sequence. These new exons represent de novo sequence generally present nowhere in the assembled genome, and the added sequence improves evolutionary conservation of the predicted encoded peptides." />
<meta name="DC.Contributor" content="Frej Tulin" />
<meta name="DC.Contributor" content="Frederick R. Cross" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Patching Holes in the Chlamydomonas Genome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The &lt;i&gt;Chlamydomonas&lt;/i&gt; genome has been sequenced, assembled, and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. However, the current reference genome contains ∼1000 blocks of unknown sequence (‘N-islands’), which are frequently placed in introns of annotated gene models. We developed a strategy to search for previously unknown exons hidden within such blocks, and determine the sequence, and exon/intron boundaries, of such exons. These methods are based on assembly and alignment of short cDNA and genomic DNA reads, completely independent of prior reference assembly or annotation. Our evidence indicates that a substantial proportion of the annotated intronic N-islands contain hidden exons. For most of these, our algorithm recovers full exonic sequence with associated splice junctions and exon-adjacent intronic sequence. These new exons represent &lt;i&gt;de novo&lt;/i&gt; sequence generally present nowhere in the assembled genome, and the added sequence improves evolutionary conservation of the predicted encoded peptides.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/07/01" />
<meta name="citation_mjid" content="ggg;6/7/1899" />
<meta name="citation_id" content="6/7/1899" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/7/1899" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/7/1899.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/7/1899.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/7/1899.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.029207" />
<meta name="citation_pmid" content="27175017" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1899" />
<meta name="citation_lastpage" content="1910" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Frej Tulin" />
<meta name="citation_author_institution" content="The Rockefeller University, New York, New York 10065" />
<meta name="citation_author_email" content="frej.tulin@slcu.cam.ac.uk" />
<meta name="citation_author_email" content="Frederick.Cross@rockefeller.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6053-4093" />
<meta name="citation_author" content="Frederick R. Cross" />
<meta name="citation_author_institution" content="The Rockefeller University, New York, New York 10065" />
<meta name="citation_author_email" content="frej.tulin@slcu.cam.ac.uk" />
<meta name="citation_author_email" content="Frederick.Cross@rockefeller.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=I. K. Blaby;citation_author=C. E. Blaby-Haas;citation_author=N. Tourasse;citation_author=E. F. Y. Hom;citation_author=D. Lopez;citation_title=The Chlamydomonas genome project: a decade on.;citation_pages=672-680;citation_volume=19;citation_year=2014;citation_pmid=24950814" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. E. Crooks;citation_author=G. Hon;citation_author=J. M. Chandonia;citation_author=S. E. Brenner;citation_title=WebLogo: A Sequence Logo Generator;citation_pages=1188-1190;citation_volume=14;citation_year=2004;citation_issue=6;citation_pmid=15173120;citation_doi=10.1101/gr.849004" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=F. R. Cross;citation_title=Tying down loose ends in the Chlamydomonas genome: functional significance of abundant upstream open reading frames.;citation_pages=435-446;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. C. Edgar;citation_title=Search and clustering orders of magnitude faster than BLAST.;citation_volume=26;citation_year=2010;citation_pmid=20709691;citation_doi=10.1093/bioinformatics/btq461" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. M. Flowers;citation_author=K. M. Hazzouri;citation_author=G. M. Pham;citation_author=U. Rosas;citation_author=T. Bahmani;citation_title=Whole-genome resequencing reveals extensive natural variation in the model green alga Chlamydomonas reinhardtii.;citation_volume=27;citation_year=2015;citation_pmid=26392080;citation_doi=10.1105/tpc.15.00492" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. D. Gallaher;citation_author=S. T. Fitz-Gibbon;citation_author=A. G. Glaesener;citation_author=M. Pellegrini;citation_author=S. S. Merchant;citation_title=Chlamydomonas genome resource for laboratory strains reveals a mosaic of sequence variation, identifies true strain histories, and enables strain-specific studies.;citation_volume=27;citation_year=2015;citation_pmid=26307380;citation_doi=10.1105/tpc.15.00508" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=B. J. Haas;citation_author=A. Papanicolaou;citation_author=M. Yassour;citation_author=M. Grabherr;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.;citation_pages=1494-1512;citation_volume=8;citation_year=2013;citation_pmid=23845962;citation_doi=10.1038/nprot.2013.084" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. S. Merchant;citation_author=S. E. Prochnik;citation_author=O. Vallon;citation_author=E. H. Harris;citation_author=S. J. Karpowicz;citation_title=The Chlamydomonas Genome Reveals the Evolution of Key Animal and Plant Functions;citation_pages=245-250;citation_volume=318;citation_year=2007;citation_issue=5848;citation_pmid=17932292;citation_doi=10.1126/science.1143609" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. E. Prochnik;citation_author=J. Umen;citation_author=A. M. Nedelcu;citation_author=A. Hallmann;citation_author=S. M. Miller;citation_title=Genomic Analysis of Organismal Complexity in the Multicellular Green Alga Volvox carteri;citation_pages=223-226;citation_volume=329;citation_year=2010;citation_issue=5988;citation_pmid=20616280;citation_doi=10.1126/science.1188800" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=F. Sievers;citation_author=A. Wilm;citation_author=D. Dineen;citation_author=T. J. Gibson;citation_author=K. Karplus;citation_title=Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega.;citation_pages=539;citation_volume=7;citation_year=2011;citation_pmid=21988835;citation_doi=10.1038/msb.2011.75" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=E. F. Smith;citation_author=P. A. Lefebvre;citation_title=PF20 gene product contains WD repeats and localizes to the intermicrotubule bridges in Chlamydomonas flagella.;citation_pages=455-467;citation_volume=8;citation_year=1997;citation_issue=3;citation_pmid=9188098;citation_doi=10.1091/mbc.8.3.455" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=F. Tulin;citation_author=F. R. Cross;citation_title=A microbial avenue to cell cycle control in the plant superkingdom.;citation_volume=26;citation_year=2014;citation_pmid=25336509;citation_doi=10.1105/tpc.114.129312" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=F. Tulin;citation_author=F. R. Cross;citation_title=Cyclin-dependent kinase regulation of diurnal transcription in Chlamydomonas.;citation_volume=27;citation_year=2015;citation_pmid=26475866;citation_doi=10.1105/tpc.15.00400" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=W. Zachariae;citation_author=K. Nasmyth;citation_title=Whose end is destruction: cell division and the anaphase-promoting complex;citation_pages=2039-2058;citation_volume=13;citation_year=1999;citation_issue=16;citation_pmid=10465783;citation_doi=10.1101/gad.13.16.2039" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Patching Holes in the Chlamydomonas Genome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="twitter:description" content="The Chlamydomonas genome has been sequenced, assembled, and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. However, the current reference genome contains ∼1000 blocks of unknown sequence (‘N-islands’), which are frequently placed in introns of annotated gene models. We developed a strategy to search for previously unknown exons hidden within such blocks, and determine the sequence, and exon/intron boundaries, of such exons. These methods are based on assembly and alignment of short cDNA and genomic DNA reads, completely independent of prior reference assembly or annotation. Our evidence indicates that a substantial proportion of the annotated intronic N-islands contain hidden exons. For most of these, our algorithm recovers full exonic sequence with associated splice junctions and exon-adjacent intronic sequence. These new exons represent de novo sequence generally present nowhere in the assembled genome, and the added sequence improves evolutionary conservation of the predicted encoded peptides." />
<meta name="og-title" property="og:title" content="Patching Holes in the Chlamydomonas Genome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/7/1899" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Chlamydomonas genome has been sequenced, assembled, and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. However, the current reference genome contains ∼1000 blocks of unknown sequence (‘N-islands’), which are frequently placed in introns of annotated gene models. We developed a strategy to search for previously unknown exons hidden within such blocks, and determine the sequence, and exon/intron boundaries, of such exons. These methods are based on assembly and alignment of short cDNA and genomic DNA reads, completely independent of prior reference assembly or annotation. Our evidence indicates that a substantial proportion of the annotated intronic N-islands contain hidden exons. For most of these, our algorithm recovers full exonic sequence with associated splice junctions and exon-adjacent intronic sequence. These new exons represent de novo sequence generally present nowhere in the assembled genome, and the added sequence improves evolutionary conservation of the predicted encoded peptides." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
    <title>Patching Holes in the Chlamydomonas Genome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"NrvzmaZyAYwJyB2NNPqGjLbk2sHT5r7XjNkZQbPMXA0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460400","apath":"\/ggg\/6\/7\/1899.atom","pisa":"ggg;6\/7\/1899","ac":{"\/ggg\/6\/7\/1899.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/7\/1899.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1899"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1899"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/7\/1899.abstract":true,"\/content\/6\/7\/1899":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BzyqLean\\\\\\\/+B5nc7C0q2ciPxATFns42+J3LYmjlvxPUW2n\\\\\\\/V+1eBphzvUsrRJafctxs+D\\\\\\\/wwiCO1n7Hsm5gft+3F30kFO1\\\\\\\/JhONVjcGDn8jlRLJgdOgMkxvsoMZ804fa6EDwbJ0uT8vWm1FJ0tquCcn5TAFQByi5gLBdWMgAySVxH27WnfLE5eMqLjkjMKCauZmV89uBVSq1HQtTcKulKIXXih2jB\\\\\\\/Ccx8fn0LWQWCr3RwQG5d0oxUaynH8LbfwC10pb+sifCjaQiVYIDTK3w8oTy+PHftDJwC94aVK4bCpqfqQJqk0xxsnwLwqIVkBTzmsFeqQ5Wrz2IpknmDZhlkMeigu\\\\\\\/MJa6G0B33zMKgDYMGxx\\\\\\\/6QtoeFUHbXEWRgDC2fJwW0YuAjpZJ0XonckZNZJhU7cfwETkK5QT+uLPY5rPSyyn4JKPGBM\\\\\\\/fyST\\\\\\\/Ec2VvdaxtTT++so0CArqAGwottzzQBHPR7avSsh9rdSWcJhjd2zkeEm4MvoiQ4Hox\\\\\\\/8A6fc9qOAA2brvZF2zFRCv+m8G75Znn\\\\\\\/iHd6xM0oA6GjnQTYGP+iiwNKXDn8UmgcGOLNPd2YZVF2TB0a4dfstJjym4nFEh+lvPcsEr8oc3\\\\\\\/7kIvYCmF\\\\\\\/UIS7mH4NJR3Fq1fm1oAXPNItZHNua0Df3eVC5WaVRb7Pt\\\\\\\/v4MxQS39XiVyUzYtzl1k3NeW3yUUp886J296YycDYpanRjw\\\\\\\/FVuQb6493wyoZAp1h9ofgC81C\\\\\\\/KxC1vL6g5OHgIBCnCjsDs1JkgcWxmxX1qNvQJmHZNwJI0WsWIx+j+CZ0yfhV1HKRRaPTmXHGl+KbH8\\\\\\\/qkWv+9SlpBnK32CCT2ACzy+nLsezltftiiUj8mKkg4Sw73eeFKM26\\\\\\\/VjKgM4iqSgXOWQH984kF28Fr+Ad2nFcfEjof4BoBjFlvzHBaYaKO+ou\\\\\\\/+IyJ0dhy1Wiy30\\\\\\\/ooOFOPvthr632jZyWyssLVY2MBqZhjOjroV5mlbrP4YcultSmNdp6uqs77AT4dYiFhrD1HtayP9R4E8apliIv5F0rUloAGe1HRpt61+A1KFuDIjs9D63vYpzu4i1zBN\\\\\\\/g5A\\\\\\\/tktXgJ5hvtSbk\\\\\\\/NMFQw3rWk4NhsLfmGr\\\\\\\/bj6Ds0z6dCembCnJFaZn3CGdT208oaxqndCu52vb7deMSDR+SNRu3UWsDXuE++3ez5gs6nJVGDZdxOx6odnBx73nx\\\\\\\/xFCeGjNa58b8fr40M6giNy+puS2TCK2oIyyPrmzG06s0ccxWwqqGSZyW13KW0Avekped7LCdG6Y\\\u0022,\\\u0022iv\\\u0022:\\\u0022lx+lI6KSpN3Kt8HEt1RcwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f174825d62588f5d489982bbf046c72193eb0d0783bd6a5db718f5c8ebc7ed25\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BsEyX5wKdjngQ3ks01+QaAVFMbqS51mvu0Z6YxnqnoMBSu82L2KyK11emENbB1fwHtPP+9P\\\\\\\/raM8HeULSRgpXbp6Q16g9dBKpfHBnHDV7VcvT1AGorn229477\\\\\\\/ZDoxgv7cBEsUQkxigjaM5GwlKVoAVq8ToYLyBiwD8gStCaidzpkGIruLMo8sZk\\\\\\\/Ok6dfK9rewfcZV07s+xIRRirGN4+xseiyDu4tAa6n57YwxEdqwMEN2O3DhQyUJL3PZYO4LWcK+Y5PwYCd0dIpcm\\\\\\\/saUsbY0twFdlL68+k\\\\\\\/NUu0Mw2i3cusfjXFqIXSWhfT1Ti96+ypyoQWkWJazEPFkUomAQzdsNtW4zSiRswvM9Ujhvk2nwgQZrh6DUUSedFbux4o1w79Q+y0QaCyANaLP1Ylz0bgLbTQA\\\\\\\/mx5W4hEyW9XyoBrmJ0oCZs4hKNhLXyhso1FZSE3KX5fyYyEXyYaWWaw2nSOdzX4ULZCQN+yosf3tNIrFlyorTbjVDbISSxONr1YuKRcKMmRPeO0Pki\\\\\\\/SArj8uVv5TP\\\\\\\/CaCf7Ch5jr5GSBh27wXyhEmqb3+4SZvlXcI3rYFsqWSzKAqcQDjbspgATcRYrpDIBQtdvIpy1gziNmznPcIkEnSvXJ+tb+FYPTgY+odkSJY8mMNI0Z+31+wi2HCOpE3Yu7BUrqAHnz6+Db9Cjks15YfRqIlx94KzpdII6TCcq3QvYhOUiqYl2BBLDqJ8vSB5KaXavUNsUmseEwqii+lREMjZzUQIZeuTgwgANQz\\\\\\\/gp0u9TOuAvplZhqUGUlAdiRkbzk1N2Dmv0DKMPYDjX8cCO7Y4osiujxQy\\\\\\\/kALLRLfAxsa3lhzxCScJBzW+rMPiyNNxAI95iqqVFjwEufzM7IaDtBPd+OI0YhOPcgnYwqrW1gyE+j3adFeYPTzAn65ZisrBscbVY4l3DEXheTl44wCr1H2BDnZjgFX4p\\\\\\\/tjRlexsirj14xYQ7b2wcdW6r0flfA1eExm\\\\\\\/7cGrsVJIX2iRfjOHubgkIT+mH+UQ36QKBpWfW3N6P6+ZzgcNtIYGLeBi76s0YFavV4zhqrVyGrh65CVG3LPO5PSKwSpe4uTylvpbarsg2gUu+DSJ5\\\\\\\/W1YPJHbqD3yRlA0Km2TQmQsHtGB0bSt06xXk6kid5gNe59h\\\\\\\/lfJIcJ19tu+xAwyjZpg6L0+kXRyUGyQPkUT1OzM26ZEh67uneMhli3oak1UZu9eVfkNmkNLhCU0mgR2\\\\\\\/JiPtz69uYmCxKESxffv7GPHCr0hG\\\\\\\/eQmf\\\\\\\/zBPG8FOR20jkTkLD42u4rYVOTILTtXCcfdIdrE3XY3V+6p\\\\\\\/VOmiA8odswOqPgjy1hJBAcBfA67SHQmsohlC+\\\\\\\/4k3W2Ysu9j3HtUJu3FjVrJDdBfoNOFBYZRLFZWx\\\\\\\/nsucTukgiukHzpTGPteZg5dGR62Qou71zS2OdE\\\\\\\/GUM9F+RAMQrdJmAqGny9GTapPz3MWxTMJ5dnVTiO7fgSoXWepCx7cF9fKwFp8QCivi05fGd6WSt8lTROqDy9Qm2Dc2obLDNT+FAYua857K4pIczC6ltt3KFKiwjqlpJPTZkw\\\\\\\/je0Dv5FLPVzr5cYnRzVQI+pcUjrKJ5Et531j4Y3ZnSheR1yDhgpUmOw0\\\\\\\/Dyjmm1E4upcP3JprcISjx7spdCh3i80k3+dkcuQbXEUQMwe9jssmDZAjzf2dNKUlWhjDEpp+n1XvzlH66b8\\\\\\\/zSFORyORtdvTvkipB+fjuaW3wBrLIqS7C7NOl4t0X4fRWhno\\\\\\\/MSCIJyQduNRNYz5VIthiLsHZCwcGcLuTtEsNyFMqCaB6vEivU3zEShAEnbyi+SKF0nCm7GPFi94uccnuPsq3lk2aQuAzJ9LsfEUAp+jm8AbXUHyUB3oHQiBaOHUx2P4fiox7MNYvVgvgxQ1XnIkdIbm79PZ5ypwZ6oqc0aP6bOjtgNAoouFPTgyAbj\\\\\\\/GeC98tgbWQQ35\\\\\\\/VK6p578qD0fBlp1qRN9Br1688pViaEIYbwQfdBnflW6Vr6SucY\\\\\\\/b2HxfIN0gaS\\\\\\\/VhFca\\\\\\\/vJT0YgiDa8urhEHbum\\\\\\\/euR2iU7rLt36Y22e54R2+X5gnCInWdrjsVIjSkSR6c5mzhwLMopvGhhD\\\\\\\/z+s+jzktdflxk0tDuz4eAehOVKQlgzPxdE7vADraKgZITO5yU40xl3dt2YxKn6Yubkeu1DM1QpB2DnkWDcajZ7Lmbgs6ObkGY4O44RQPz1eXvzNEcnf\\\\\\\/21ByCLHuPS993vC9pzl3oxcOAw38vPqdrEwXlW83lWo73YGU5ZWBobM72NuULu1xZQ441LpV\\\\\\\/yCsKsFsCh5q8BDivsezgnS\\\\\\\/YJVELEEMoyznogIG0CQ8+EMiti+K\\\\\\\/78ZkehhZO0e2FsU8JzsXCDV2LD03lYRQcUexO69sUr+YLexrA5BSHcPpkorvCWfSn2CMFTcS\\\\\\\/Wa1WJPHahuACZ3bdElpdJM4thgHa1qVu4LxrN6pICva5fz+gKv2jccBlo1TM\\\\\\\/128E3mmLgzPQgGPvnT1yunLOEtZx2MSxqlIDfvXtlkUpYz1GgTk0a9VdOPkkJKhIaNi5nzE+spBNiJDNQNWVsmzKo26ASmxJqZBMTmAlsqPxnHionr3N73jBT1MGO6T0NtAbi1I+hFjX\\\\\\\/rpVj5mUHmbHzdvYCBAYTzvhHdkQoMAPeLvfqSxGA3pKLXV3Bh53nmlwczFWIFp1I2nFAfzB9uN0nbJs3Nx5JSTMWrAEqxx3Ce80ME0aCaPRlt3OwlIzv2ZC9Fn98Br352MmiOffdbm165goU\\\\\\\/hahg42C0iwW+16DEL1tuUhJG\\\\\\\/bbVYer4ylwT1C44ji81xrKm3JdhSt0o3wV7zU+UXHEuAJK6DbbFszoOiJFQDZCbzlsvJLQXEe1f7cYOUplonOGQVlHOsx0TYUyigaDdIbQrn5gPlUELT8Q+auH7T9j9RKhomwAdJMcLEdx9tSCwRc8mdQZdbMlI1uI\\\\\\\/jx8lj1rfqoVJO0NJf1mQ0IRUHbh3W8ihBUdYRrLL5qTacuLv7RDThMslBHoak0g3B9xXI4PkWa3Eyu7bKWM4TatArS9pDvvm9meKb2C4uqAPrtHlEH1zzvpMdLUw2J2r\\\\\\\/AgVenX18dtR8fnSidOVgDsFn9yNc28+D+Ch\\\\\\\/KALYGcXNabucVpXKq6cBIvIpFLm73T5t9lxF3h+dCYwaTJldewXgoF6fNPOo+IPR\\\\\\\/GNg0PmPr5IJj3OknKqm6W1DZzTT1QcsuQQ+eHmbLYPRXFJECMOoCFcucUFKDO60zY8HUtFj+soCt5AvzMO+UrWcG9c6Yf\\\\\\\/gfpppyJPi7Bz6v3hd+i8UH2Mdl6PskPFmDfwtGuKaYfaPctvI8N0nkCaez67TkUk\\\\\\\/qFIE1wngCQlTDmL4DzntD8BMC1EzyqaOVyKTbX0gmEA2\\\\\\\/N1RXI++3YD6b6fD1onwcSqtUj9fppa3mPDRLo\\\\\\\/8jqooDYS8jbj45L49WkO5w99kSRnEoaHdBhE9mfGzOUDmEEPogauAvqgQPrbmdEX6MbJH5k5ZCXuAKOLBH2SQ70lD5mHU3kQfFGgVib2BUa39PtOAMAFXjmDQ6ZanbuAukNNk1fc0ebAgMOdBLXpcGslNchVonE2JzAv4NSELJaZK8c0h02tbUcPk7jbE2hhsj1woMl2jxtmpw3IgWAwKotEO6jOjzbxBODwFHlW7HfAcHDXP1X0KYM\\\\\\\/S9MudDdbZg2SAq58I0RFc222BMPmpYOrLAReVgqLb7ouh2OAbh00ibrahUNQrw8DQDoDKGsHL7lncazU\\\\\\\/SB7pnLQ5HGyU3VPhSENN0WgdHJkpR5Qopaf2vRn6To9F+V7Swpg2BF8guJzHNVrCnhgk0+QZB1zEwVA8BejlQDixcV5nj4ZSQY5xuvumOC7x89uaxuesc4Mg1TbQr5JFgAZ8SY7u5k0S0N0zPdH9G2rGBNNl5Rzrs8xc\\\\\\\/4PXIdS5fyXMMXa2Jo15x4V9zqb6yallsMLiuBc\\\\\\\/KvtY1DoWtuhOfVLxrFU\\\\\\\/DupONEaU1Tlbdlowi0pYq9rFUBO3qobBkkqOAMtCl3llekkO0tKY\\\\\\\/hJPyNDP2CKMKxIWrPTiQlLVP4xu+0nCOf08jdTut3TVNbaNXLPLir6\\\\\\\/q\\\\\\\/fuucLD4vCvzIae3rP\\\\\\\/5vOJwRNN6iAuEJcWL0i5\\\\\\\/6ra\\\\\\\/mWzlEWg4aCX3ssGFAfGFjRwZdYyxsnRFhWYH593G0gr9ogn7N3j8U5E8SfJrxgI6jc2JWi+el+Mk3FO03uFnhKj7LfZV2wx43s\\\\\\\/LS8KdwlWfhBFXEN+fbxuW\\\\\\\/2PlBigMyuDf9\\\\\\\/HOn+8ua2qZtu5ZGPmB0NoXSQD0WZ+kj1PTleSlWlL84WRvQYDLnNVp+lBRmM7AT2Ll1OnP3e95rj45\\\\\\\/6aTGwo2nr1FraruSsWiWF67BPnYm9Bci2ouGg58OQRl2IocTt3tW+IslFtizmzKdYSIdbDSCfaqyUMXngC1XzrQsoD\\\\\\\/0zjZ\\\\\\\/X57MVyAJzHAjYQd3FiSrOPBSSaMPREfV2sUWKupKRjchkJXeyvVhWWzv8xtQytBvJjizjyyEJdBqv1NRyz5G9mLSYdnT7v825hJyIvCY7vWTi3kQsxM+wArHjHjDPtOmZqvbAdoUIX\\\\\\\/y3sXLZPHKpf\\\\\\\/UCwqWAgzubAi+KlgbBHUNcgz1sVUrRUfR0vD89Ir90uEHaJs7IX5wj6SMUs7YbjJKAM7If8qw8zkp\\\\\\\/F2\\\\\\\/Q45EzO9PMLLexgfV9DdW7jS99cd2SwYabQKxOHcrfwlLpE9pzvu42u4U5TwQEIipsIV9AqxmMmNgoFOEBU83Deq\\\\\\\/DN6zbOhfMUh3Kh+nTxEjXxmsSrfM4k15X4YSJsAvcHaXIq0ctPLh90+HPbgKKQzVTEAfQ6iZKhhnssnOHgClfgOHZO5\\\\\\\/um4vspk21aAovRUmBtgz2fPNMjfO+dXCyajpKZiYY3wgBA3d7l9dl8cnANEvNpw37VoqGkgLFtiIv+XMRl+PeYlsl5Ew3GtlFEbPVZ9OahW0FS0iLGThK+qqGkrEaP6vsEBU8gR24Pon0UZj7cCnakuojEaGlh84HSlmz\\\\\\\/7VqCWTmaXkXZQgN0r7f42rHlpgLR37ejB7l65vayg9QyuYXc4MKAXjAj2t+98IJkWFtq1Lvh3\\\\\\\/mzGV3oH+v9rIwBZ9WrDlUjfJsIBiT9n4jKGrtUhYWCD+Kmu\\\\\\\/yBL4VPoDZlHgR\\\\\\\/45+ZbXpMgqFtSdWpwvvrt6B9l9LcDNjyNrbJdeWPTLxV\\\\\\\/aZzJjoi4GMOiKwD\\\\\\\/gBC68mOF4IFtNPIzXt3oXKtKufas2OXMbv4mBupSTM+bl2MvhMGGPWnuQ1dRm1SxtCkwaFJH+COysHRdPKRyd1MEinTBVCELe4T+g3jjRU3QcP5HFKjG7WZetSEqDP2rqEKP3k\\\\\\\/UggFO9kV5EBdG7uxWiMg+Ci4p9CiojXoJM5e8GyvX4ZB3kihvoSF5nHBBPQBnAVsmo1r+UGIKBpcR\\\\\\\/09VCb+oP0H7B8qmUnvFb5L74PaH77J9+LcImOhPg\\\\\\\/aQb9EsZsOPhyuwJQzZ7+QOuRlBCKz6XvhH0noLv7ifYmkGVRNtBabpbu314kBq2FxQYxJZGzyoyCO5NWltJrSmS9lTzc7ERvT8PjXm8K7fNslPF+OufabLQziktoEB8n0yUeCgA2vhYIOAb1bTfODOsmNvmQZAkWXuWt6BKuO5yy8iqKpIXmw6B+wfoydMwtriy99tS\\\\\\\/9WN38F37UUtSDy7Eo31\\\\\\\/v9pADLm8luq8u23jC8x2r1jfWn1QiS+G8lDXZXpCMHf666tzenplTqX8MEdX6R6YClkirwI3h8YxBiTQc9iD8N79F+98FGbN6gGurMrGZ\\\\\\\/0p\\\\\\\/qKX05bdbBiz6KRtGXRazb8pBhv3GiaVgZAufkURrbekcwan1K6tgpwfpVV9phRfTH8zuO7Myh1Ja9GlIs8tEkLA9Xcsb2a\\\\\\\/oU\\\\\\\/SsVi6llmjiGmdngfxGk4YJlDI+oTK8+xWmaeycy0UlIowX5YAjnxhcG1yfnqWlP1QzidBcCZvG+4FTLPBBh32Z\\\\\\\/tmiMr+Xp3fDGiKSlGcr7uz7rhdI7NmBY+pXSkuq+jnClG6G8Roap0kvkmABe3Hyb71fi+Y6bsmH7z\\\\\\\/fOYWmkS1u5Nv\\\\\\\/zrbTD6GYG2Tz1pCoNhu\\\\\\\/gCK8ESOBawzCnMo4vLETETYQMogW8K6aZ0WL8SbJCFAq\\\\\\\/r7kY+CnsiNh50YnRE87BfwEp5EJsrGyM9ydy6Wobc803De7Tp20+eUN1Tswrf6VnspcSGmDb20Iftclw7Hd5kN2vkGCgwpP9RLfYBdVE2AgNovRrOC1K5IAqx\\\\\\\/MBA6IeI\\\\\\\/dG9txHTRlU+G2sas1LHZF8VQjgSRvI\\\\\\\/RHbEyCibrww2LXWV+CLJ4o+8xuQqZEOgBVGghyUl\\\\\\\/p3U4B39BcvIZc3qaKynHLITIsPZ05OcJUyfe2zYZ\\\\\\\/CX79OgF5LkN0uCHSjQ+cTFyDfVx9td9hubSSYvU\\\\\\\/1oFoOY4U3We9QCWvPSbbecybBhhLWyv7mesV7\\\\\\\/3ecRaUZxq4Q36rEEw0i+maHE2EPgdn46HmTO\\\\\\\/o36MNfE+c0UovCJa7RTCH7QP1Spc+XjGUfhnmZksXsBsuyudv7AtHjbrkh5cd2KTPFLUdEo48cXzUVuiOMOJvC6TslR4DjzsJBodSu3ti5K53lFrlsc41ppmE6vDXvVJIJLKBA7dyzvPfOv7T57U57Fnb3ukX024KAdaVn4z\\\\\\\/DXcmrKJQRCSKRZB74YhiwV9sliLuO9BBVemDMFR53+BxHmbvjRK1+q449BGKw3crD3VU76+4aT4\\\\\\\/CDPlA5ZF7ugMnIHnShM1HpKWf78yiK3SvmAs6vGdSZnX6zEqXOKs3SF8qoQKC0trLu5vZsA\\\\\\\/pKcZudlyqlRKkVmtYswPTJk3pKdJkuMjvX97j73ZkTKDpWLYPN3rDpBcRCbjoaVMOmkBBPbH7Mznooq1LByuEzDQg7yD2K6+tHkoYKXeA3Z+Pi1KLZwl0TZBO9K4f+ynu\\\\\\\/BgPfG9DhCgMIuobir6qeuI7wny4knI2K\\\\\\\/mJx2WfceGGEY8RHFCr9LhKRQ9Iyht+en4S+fG05jI8gTDDbiNoTQS7aNn\\\\\\\/2m8FjSjtw0+AhA3Z+0b3zZWZhmkkdM7VSKJQY1Xh+FL05FanSMft3YZx+NDxRQN4mywl73HJl4YXMvAUXu+ATqpJNMFgE\\\\\\\/qS1l2LlWK4h1KTq8CLjguVc2KRwZm6h9qFiuEkuQjTSccPkoXfbdvSLzptUtmEwp+iGATd6OPI0uJAAdyRuaAaOa8gPb\\\\\\\/k2DLst701iHtnSTZheIgMPAJI\\\\\\\/VCTZnQiMCIdoFIV+eizNs+hLZKkB3JrX5Cf8vIWk2GYmc8kXvULv6Kb7Ay+uJ75uac4\\\\\\\/LBVmG8slY3az6sMk5+RNNAlD4tPgO+Ewwkh6suzOxCrGjsoQM6hDTcRYGRnLsMvW1O6DwwK2Zxzr758KlYYdFrFEIi9X6fnZggJPEjgQIFF6DjfAAWdFxGQrA2ZRQcz6hTmbmZy1MJ+CpWzoBC5q\\\\\\\/L63svJMXNLAR2dYHmij3icw25\\\\\\\/8sqRb7uTdO+UVG9qDnks3VDjqwxFzGSX+1hcvODP86PF6evcl6vcnvKKESJt4QfWQo4OqFOmL+FuAnLWpkXdNDJ1DAsiBiHlCe1XdP1pqlnK78GDkNxCBtmaoJWAmCHHTnegovXWDAgRfTrgaoUn7ajf9MXX4MYxFe7QFx5bDOfiCPSrDYgUTp9yNfE6C81ClQYtXlJLGgnm6ZHfATxzEnxgD7gA4nYZM\\\\\\\/UIJUsgLDWl5EpPmifdBY04\\\\\\\/cpt2LzItwxJRXE98uA1MYD9FbKgLdcfVA+JtT5dV5NDFjje1t7RM7ZFYMd23X\\\\\\\/eDOBxsL01sGzo2DNkPhGCEKzNCBF1QU5gxBOubhw6SU7PNnWkeBm3s5xlQsR1iLnWl3NztOcWiQQo\\\\\\\/IY3SrCxgBL64vZ1oEJtLWbtrVrXywnQqhvTzTovzOTYasVeTrCsDmtucbOsSIboHp\\\\\\\/G+2eP0svRhz8ZZ3JkXGYSOe4r+TnZfQQpI0X8IU97dlaHxPIxIfdLBeH6VjcwU1Lc8UemXhKYjTJ+ASFPaplWYYTOI96DJkPkLVguH3gVtuDarCYFc8LtqPLes72Zfkos4qJkQtCApIzKVl4Xh16nzNre8umVE5mihMJt35kkmraF5TSASZf1v0tH0qDZCwdfYxHnAhpAzoWvr5ZScUfJi+oIaN6YnRlNzg2i9yOqYphNjiumOkZMp8jihxtI0NQxw0z\\\\\\\/+Z+Bv+KRpDCMYqMU2SP82VNJ3VAf\\\\\\\/cFHqqS0lHe8wCTsHrkSnqhGh\\\\\\\/mDwcStr1EylEXxV+taZADLWS\\\\\\\/+\\\\\\\/R1xBPayMikJczErl\\\\\\\/IxtTLecQX9eF6E1i9rLJuSAwjT5WCDqXoh5EWftwJU0XpBJQYENqfiMnigayuAExdZAir0L9hfGKdAnWshSr6DFnhkitrz+LnOsnhh\\\\\\\/42vBPSCCZeXVtZ3SvF0GFiU9kjZE2FuVXZhEJIAvzEXhxEi+Ys4Qkcgir6fCDXvbv\\\\\\\/bQaH5+ROqQ2PR4pINEJp9xf\\\\\\\/UZdWEhSOaikDqiiO+F\\\\\\\/Qf4SQ2+5P\\\\\\\/4kkFOWO9ULmvPNIABK8ibk\\\\\\\/6cMFnlM21PBQ0UryE8KNascDdyM+ZeSwHeuaaMQIRwcEvyEv0NfnSH35IrjrfCNFuozD4DYwyzYtAJRyBdyrXkk\\\\\\\/KFUuqg32HdrwsG6RZCh13OKVCNx9aH+RC1usbTZFKnj7ICKJ2F57Nb1UGNulLaZVVEsMqaO+U3n6ihGXpTVREG6VS+XERQWSObWVY1Aav9fzCMdYx9sJhE31YYcF9r\\\\\\\/zBzKSz6azOIgoiFKANW3BDs00wTQ\\\\\\\/oL3C6y7xuq3dG6VtGB\\\\\\\/P14NC\\\\\\\/Y9Cw6XwgtZgWj30UwlX3X\\\\\\\/gmbDGlTM21rWzXT6UX4rOWUqgU3tot3Jzf0eHGWowtXG087q0GJ72JSB6cfDzV52z99xPWxRtc5lnIxK89UgCZs\\\\\\\/5ZZ7A1G\\\\\\\/QT4V3MYJA5QD9ULFkkWuKRhu08z8hfm4x\\\\\\\/aeit7soASR1zgRDypV0td\\\\\\\/uxQX0D8I0dA3y7VzQrQbUtUfzVvAshNexhr3dwi27CHsn8+94lUfjUKngLQ1NU7CMg016D2sNC50I7d\\\\\\\/cD76fQC\\\\\\\/DAkhPXuTFiSK\\\\\\\/6qFuckn7n49jFDEMoVtf4ipE3jwa6t2Vdhm8ld7iInfg4TMrbKbZXccJVnwHFPN3lko+ibb95yPMLXHYT2pGre52HO5LGqJKV089Oqi7M\\\\\\\/ZlsacxluKG4i16sbct5SzGEyp2Qc2DB981p7x63F4gv+99LRcUd47Shr6KWJGT88AwXJFk7t93B1aCQS8qsTXRXGvux8+3FQhqaU0xCOiV4HrxSWjQ9W9iZnDoyTZileBwyWB0AP2I6Elegeu79nLSzPOM0Hva85W2FWOzsWoC1mBycz+JiNCnowJy+0+iiaElcJpS6WYzt+1k55BJ5RqCodAqxgVPKVRLhvXPtE0l3Id19H+t1oYSqduo9OGSV4HEoKy6ehpgMK2TmD51di0q0D9auGCUNrB9fs0er5HbcCp\\\\\\\/jREbsPKyBXnON77FAqbMmrFOsPifW2l00aeYIOWVAZ1mQ4aTsctucw1TXX39JPMwZdWCRahiTGPswFIf1dCFLgcvyjRASIVs9zCwPthHeffhXQ0tnXrVLqX+DDp1el2Jw9u2S\\\\\\\/Z9D8d\\\\\\\/6g+oxoNktSu0BxN5Rr0C2CvSMChwmiQsT11q1GewTMwwA3PGLN6ROrGA57WfB3IewbuxlHIpZwLpdHCiPs+v8xKXXymppb7YZixYPo7FGLHbiIYR9L1Pdvejje6SrYCxzAx3wbzuKOMDisdiyjAZrl6htKVgB6+zFQjVGhaLlhhby\\\\\\\/YlcPl0PyRlJwpZgQ6VPqoTF5EDYaDG3Ga0BxzUCfIwGqzXi+o5ijr0EPhPczrW1O7Bvfgqtj2A844xbTbmfHCl0a7zxNo1qxMiF\\\\\\\/eywkwTPBFPkurZ4q3VWr6igXgfSOB+LhXT6FcYxDnX8mygjIC6LseaFkaGDxd2GIQcSKIdUrV9jh15wTHfP0qzqSq5hL9zWPaGIlZFwv7mMbL32Q+6Oz6XM47MVx6FfKCg6ac+pXAId56fkHF73Tkx5eUivHnd\\\\\\\/NEdpRns+CSjQM1slWJTr3DFDUehoFtQf7u16MfAwgOATG0K\\\\\\\/TXVweEWehgjLJR12ujNrhrkwa1KacsXEeBoCjYzxCZ5YbnTgzpa0fkPrFsevHaAs2E\\\\\\\/of2vHqjWCUrqUlcEDvQgch1JUEiscKQVscHIDeA\\\\\\\/d4adHI80D6SlZqQF+eh0Pdcs1LFC0NdDFWvOM5qyiUF8\\\\\\\/vvxudOsmPMrRZhK0KVSBKc1ZhStKmaIRKGp9ZMG16MfvVcIsOOLiEJRWFBj3RwKnPS1piWJe5HaplJzI0N\\\\\\\/CfeaYWp1JvR\\\\\\\/WzOV0eTkeMbwE6mCULPJh601DxT3csLqnOMPzTKDVi1Pn0eBdIWaDGlE2KZ3dIfv4aINpiNhO9w6iiebRqBb4FzY0h0cwzVErc9BBgA8UJ1g\\\\\\\/q0h8d99jCmDPQf+AxX0uXb7knG3vW15xJ3fG2vTJRlLE7DBWsDVlyvJ1sf20C8BEVFdS7GE3xQtUXXtU58W+tUahGYhybcnV16D2pUnjOk17bkf9B7R9eA75GSxMmcYFKmJIRPCPB4adQsKFzKfxiPXRKbODJUOIppm0nNgybAt79YXIXxu0lvRFq7SPjOzf2cbXle89lxMWOhWsybOdPc0G5JII9RZyHeunnAcD7NcHKJUKfD667F0w0NhqPYBXGQnDqm+R8lhVADOiPXWp9Rj8XvUAuvAVEF5GQoFDgUlHYXgReAARJCiSUlHrMiZaWDxRKsYdPOQK10\\\\\\\/Y64Rwpw\\\\\\\/Z2fZaeCPQtR5T+EWeTO1Q6w4CWw50fZR8AvrvteBggwJ9kjfE2HtdF+EtHvgu84bP0\\\\\\\/5VLZ5mZHCC7jnPXXcYpBRgZk3Apz37ULxk5WqvPbqLPq0ESg5p7XkKIBhOxDaeVWIGZDWX8tucH2qZH0\\\\\\\/RjMW6u9dtDhkH9TD+w2d1q2nEm\\\\\\\/NoxfToBO6dbb6vfVmp+XNJA\\\\\\\/OyFDkCxadpK54ZrS90JTuoxwD4x89obvxRZ1XovGCrohvKCwxyM7\\\\\\\/k07OH9vE0Qxdv9ibSsUKDyTpOfgZ0+SKewWs1WQajH8OabZUUtYr+D+OFu0xdTkywQiCQtlc8jV5nDhs6xsRVsGYn4njYGEDLvyT0BJ3jPzcPnRIWQgCJQEJcLiqZgRtow1w7bI24zwa9SaayZNGJLAPJIvWa4jF6T0iAEBmHdfyzQG4fWJXMZeH+1Gy03Jw\\\\\\\/gnbTMKOVVVnpXHFsMPKstn4lLpGMPK5n++WhHO1pMARxQ5rnsmrZWFdSWjUepmGFGEkz2GBK7PTMbXn8NhIyQ+05KHKXgeWuPb9WspFbDNj12qB4GIvC1gR5wjYIf8AKfZEGR7j\\\\\\\/vNLEKowsLFGCHw2jDBB8yN7m51CA1qDP28cim4a21NxsZEtWG5xtp3+G3heyFvuqXbVAL4qkuqDgd45Y0SdetvUnvXVgv6yJA6FTf6zf4lZpstwaA5EBS3VCm+1zcu7M5LT2k9Hiwx8TfKX6J0FH1BThMPxr5WrqHzO2Mj30HikgARG0RF9+MGXT1jScHNQN7IEgKZ1E1wGIP9zdOXgLyvTEqojZEv73u7aUWIOE8eDy1OvmabPyO2ZkKBBXuVNB42+ZFsJgHw\\\\\\\/CYATuk20GYzz8ESUFtwo9sVSygua9bq1pMZOFv1NgotEpviQ941k7Cl5\\\\\\\/2NUHtPHqvvR44DSX2taUDC2bPWI1EXRNtjbY2bxSuyhK+ZFgZuWweLXqvWr6vjZvA+h0pfDpIiIcffZ1LGeOO2NCepD0GxkEpBu2wJBmRnutxN68x66FWk2VNUYLnYmbLZ\\\\\\\/FkxBO2DKzW3XGmgbJLCv0GV2zrfKN+dvjrTnAsBAoMCnAglckzUuEndNGN6ffGGQTQOXLSSd6Cgwh4mUSl0gztO1y9xjLcgT7u2pnhP6EYptJalvdoxZVqgjE4hZcOL4pJ30uhSUKOkzrxH\\\\\\\/Irli28mymgGasYQKgXHJOAl+zve2vtvLmtnAHxebjspyCF8Z+8hpuBEmWxKgNEra8miwXS3Y+XXgM\\\\\\\/T388ZIkCmP9y129Exp8sPYhjWc2acXz3P7aWkx4e2Qn7z3E99iG1lPvEGumcJuMKq+BHKNh5216hzfF1Ih2fe7KYi6be5KB8azw\\\\\\\/bjACU0gsoA\\\\\\\/41H92TXPyhHdD9IzAaBj4UTwIdnx57qXBIJPpMyUdmiqVNM141Zb9R5DG6GP4GkK1QOi63KaNDs\\\\\\\/C8xqYKMPYa9j5q5Z4mk1VqXLs6cXGvAaGwCffcStJEq8vIqhtCeeNZOepzAqE39j3cuI30mEtBPKRmsQXL6wWCRL3XGidzbZWsUF5L9B8ifbEonQD+hqoIDlbD+u24a9jt\\\\\\\/+eu0IKuSAhG8Ckp37jC+LinVMGGsfU\\\\\\\/5YD1ROBPy\\\\\\\/YkOxSKNVCLiBXqMeu8wrxKK26d9J6I1D1BEF\\\\\\\/9j70LL5cTsqnkXhbYbKiY65kzkwODumImCunTaiL66BiJQN\\\\\\\/ediysI++OepocvENry5dftfGIsw5nABmrEO9XwHrhqyBxCGfWN4qct1Rr3rHzWX7rdqJnWbbkDLKdN8Fg\\\\\\\/HGytwYls3mpKYUV2puCLPafCpRIB8EQupIdGJ491LSk4NYQMwIKlTwWQAYBTlUV92mqLblDMunOl99QVm9wLytegFS4upEcUnVPV29bbj71DCper+TAczfWA+Vhelm3\\\\\\\/CVkaBzPURr2nlBlPfXJXOX04ohQuSeM4Lunt3vtHFHqALVzi8eSTuN3FLr66T19z2D\\\\\\\/zEAzIVa676o\\\\\\\/BZ6hl0sRvtPu57GvJHQCq3MCk8aaohHod7AjlWNOk4jbpVigjhLG475evfzFjbnvzNMUZi7OAIHXOqj4auhxn+egBxv2Fb0\\\\\\\/i\\\\\\\/T8LhXM9lRwh5nrVAlZSmTlEZa1QFLxaFgxyBJ14mVhvu3UoDzIrn4XSN1Z0ulrIRaVr8Tr0sWgpVUjw30KcvdvYSNbcbaSO\\\\\\\/Q8wVUxpncDj7ll8L1D9JXePmoS\\\\\\\/J9PzBtVxJvoYPfWGV0zD+D+gRw59uy87X5M78\\\\\\\/o\\\\\\\/KxgprA\\\\\\\/yq7mVXNwBUAiHg12SqWvOpIxgzp++ngaIlMiKNXC\\\\\\\/OdblJCklq1E82x7cIq72cX\\\\\\\/9XRjnVVm\\\\\\\/LeRRV3Hllzv0btZoqWkrRpc7rZBF2zb9evfcJZg2Hi0vg0xkg8Oy7EbbG+J6fS4JAspexJi1jNWjpA4DeE+1hHJmNal0yDMNJI6Z+jy62jDGlGg7E4pJ4ND2lcDNuK3qlmWXmniDIZsrUZ5qTlBBIJPNp7c4QkWjGkiXgQLb8oYU3glSCQlWJIfhD41A6vQmeAxkwrbJK\\\\\\\/GXe0uP5vD+kPX0RhowmOyDKuibU2Pv0bW\\\\\\\/c6EiLTU8yR6nH5z5JbJG8ykuWxIqcPTRxCE73FZidIOmgVxf\\\\\\\/KdtQ1DbUx4xkO\\\\\\\/SqugkYxkOGU3K6zk\\\\\\\/ghCb4vni43UU3L8u2AiAeTaGYtQhfMfuGeaLxGxbFZZaGkm4zdMJq19Dd8uqPDySXuur7Xo9wOsaCrBWXTWmx1f7fw1R3VqvBJlMhQL9LStyDd94Fp7J39wHFsvdeZDQ+sZvNUFg8Y0m7U4pg3MwHbMuD8cma0Z9lVc9A\\\\\\\/KWfPQT\\\\\\\/SUpNmq2KqMdSdQdkrn2pIZ7cG4HCc4r\\\\\\\/C2kcKIytT\\\\\\\/GwNCWWC77Noq6jAqE9cMGNZP2FgYrfg4hXbIz5HEAyhr5TFgUaxUMiBFbIPZIo4j2r9Ly33mq3k16sg4cqDVYjvhIXb1mh1nLsPCcVks5Fkv6sHCROOvqpvxGKeozt7EjoPNJs3x0GTVV7s6f7RYZalgVfVjOT0F3fYHq1ece3Oxw\\\\\\\/6NQDqY\\\\\\\/XYMzQ7hWDfFBDuPT1939G6BfCf7HlsvU0B60tUo56tU55Pd9OcWHIwWz2PzXW6Bul1hjwe++QFoVO7AquW3075G9IuNszK0V2MEDTiuhnCq\\\\\\\/W1f1fplYo67vPvnnQQCIOqcc\\\\\\\/z8POE9sFlbKijs0Rml3zOYzNRwYTaGH76V4P\\\\\\\/rSFoFC37UrxzDyUhdZigILv2IubwJ7EzNNAuTcAjULhwV43gNsFXTIj4SCtcUkWFLtH9uxxOgIn0sZB06HM9IyQeyLBmX2smg8pNErNwurs9daAC\\\\\\\/ML+76n5rvd\\\\\\\/WgzV+ymGvSpsbOsBwGTm44ogh6WXUbycV+wtl5RVapXlucfmPcS\\\\\\\/R7vfxHQoqXGU4zonXPw24ks77IKrhCjKW4aE1iYkfz+sBk3ZG01hGlwhOTcVcJfRRty7Bktny5TdotUj10s25rr1YTZrJW819hbjf7P\\\\\\\/TrwP2rTPDv7DH2uDdsxPvY4z2+E+kM4\\\\\\\/rpuWh97RjFoZ\\\\\\\/GIDXULGlQxBp+PmF0itPwxT944UOP0VyPbJx18bSybTm1EJSX0WknnUVchnxuoHSS223mMjI6pHKOTpeP8cBCL9MdhUyAMDLEh5QhtIs3H\\\\\\\/A9V8agN3rsoBESq3FhfAgKh5LWCbwazARDdFU77vn6aXoNA0V7vwC6Y3GQLmmAkAvBe\\\\\\\/Jf5N3YoMFI0TGbdABr+fDmtEWcjuSBrqXZlf6cjgV4tqhw0N0qyVM5FkZ5x6nQNducgW3RDUWUOPjgjzxaZTIn66Bq9GMVGFWODCBB0apQsqV46urYDmbX+WjhewWj\\\\\\\/v2dTC\\\\\\\/UCPL0pHcKKWGNnJWINOLHFK3HcEW+HEz5v+NgETW1qI8gNUgnLQmmlBYBcmLn7XbgyAMbwc3YHAMpu4UmYNbzYQLwmNcgjkpZwcRAs7val01j929DByAKgMjBj1+yUWE+SH87kYtcUzBBiBQdA8P+01P3lOsz\\\\\\\/ClB26s2o\\\\\\\/uYuE5zYMiOIlaGpphgHc0PgWvHA951NE12drJIxgx9RmPrTvmgN1MrqA1s0zdivK5cquhInpISu026xT4sipT+QyfBXRtLDJRLrgd935O6hrxv9lLvFpLH7fgxNHLk7u2OCtd7nrkzooHAwYpXsstkDTb0AZHF0+HG+FTsiR+reZ3a3nJLpMXPiw6V2h5ihHMSlMG7FYoE3NpnGMNGW5WXzbq9KE7KzCMiQ1l6qRXTSrZo\\\\\\\/tOVLgFk5+5mKps7uW+gHLxSE4EPoC9bHJUe\\\\\\\/2GMp2P2VqWGg8ld9EPTVHjsBNvMNaKXVO52z8hgylYTfX7A2ta2xiLNEPFaZi8IKm2gfrWC4GUM\\\\\\\/KWWTIBohF50hdII3+q\\\\\\\/QuC6368aHHmorAt2UZnzGKAgkXRoEhJgHIkfbAt9VaKNIfmuoNQW6mERdHtHOEIJL5iK6JcMQfi5QN8KXMPzfBxjqiK70wr4SjBwC4jRflLtz75mX8s7x4RAQaLWT78CrJnAQWOKllygROdyY7SflmOuCAg7SH1m5QdyyOk4WqNe0EpjR8XQ9twLQLiYJ5rhoUPG65FZsdJKtFhIXWoc9yyp5AriNu8+Ze5DoEeLAdh1taqfUdfqOJFg2CIYf20dBUrSeJD6gZElL+pMJ4TyjsfGyvaUoAE9qp8\\\\\\\/qTaoxvDNIM\\\\\\\/riMP2P7JHualXHlYDmxTFJYGAZeGpGFtszQg0BfSEDHrUYpSCMG44LF+6lUkb8oFlmjg5NdudmK3eQA+xDACRL4XTBUkMqeqdTv+Vo4Rjf9BwQsFBm3Xj6RUfRAtX51JbnOlS2iGtMLxL1xz1nYU4GgjVU73iYLBssyGUwqQ4n0PyUYE3jTGOCvlVelUccHUO8frvUy1f+umFIMOdQrnP3OIlIcjDnV92TyagCrLzGnxixsf1V65U9cjY9ttWKGjiciS2lVTM+bn95I7y3Wt20BoG30Mtz7uXRekC8QStICJxwWIIxschHozceX+e96IQbCjyb\\\\\\\/Fi0s5Y0\\\\\\\/NRvfwfVdS5Qj8qfab70cUhS6Mbt2cb+PKPwSsQj0RX\\\\\\\/3faSlFGV9euoy2CW4PaKvZ3sgCXLIBDxnU6bzCffTRpyDfsk\\\\\\\/DCWCemdievmnPNKfZkmNXvmLzkmWf83tRxbZrK4A68zAAMbzfp2OA8zdv\\\\\\\/QdtONl4V8lR35Fcypu9jIgTgjheXuAO0GaUhyzfVHsaha1kup0oPu4tNLiMXnO\\\\\\\/zqj0ZhHFZuN0HZN231N5OIuNGPoAvm2sh3ocw9y7PRGhV9sZIsVTqrgkUsFbB1PqD+CU7Ie7Dd72wFytw0rgYCQ\\\\\\\/kEYXD5g\\\\\\\/TiG\\\\\\\/45ysyVQeeTVe1r5B21Hn8110XVaa3zqtxVtCfCtejiLCILqr6235Cd81IEMygT1Bo+0bL9jNr351mlTx87cul+25Q40Qvh13hXkC4HFSsYRk7LgAD43g4d7ZMWcuNsrfOHx\\\\\\\/G0xlZ8Pb4jJKFZnU76QgeU62RlcTS7y7XjJdbstpQpE\\\\\\\/ayq2sRndNGwLNCeInhjczxhWpWiEzpOqDtFDupJ\\\\\\\/6MiuhktlYKv8pEueOX+vYZDdxIfTp++avEGtughJvojXanfSTnR+\\\\\\\/A2yoJVwf4z6WlZfx7wwqhd5s12bglNrv1MBG68cXg2uOLSyilulCHvAIe24StCsRQ2P4GoUZ5YW\\\\\\\/trmXz7ZCFJHP9kPB6jG3w8yv0KfVbWvERIdyvl2BLBGjNC2UKR6\\\\\\\/4\\\\\\\/Nn9CQznkjkHTPaRQvieigchD9WzLArKEc7R0yVxBYPOdpPQ1Ou7TY2WQ8cYi\\\\\\\/NOiGFq\\\\\\\/6lZeWXtIQ1JahvNGizMhpfvHYBtgWNSBVBidpT4VWebcaGL3iTiHgzJ+YEefluuyseI1WRf653yOgvFdg3N1EDEuTmQizBQsXTFucWNBB2TrfpjcmI8TIDy+Ad6f2duk1VWgLnSfScXe861sqH9WIlrXClScSZBSaf8glG9pCaeILnLK8pEuRtTxWbJC7E4\\\\\\\/gauYU+E2EjRDPyYrVJrjykvcwmzpppl9iu9UeQoU9UrsD3MGCPQgYNoYNCP1ytC8lj0FWXI6w4cndh04AidfP00qzo2OMVVXHqhg65cRxl0xyaPVQeKy2T9J+UROaxyX48tEXil6CXde+twkhPy5yUL0gWffedyByZBXbSeKGUO+peS+E2RdpbZt\\\\\\\/8AnghVHNU9eeI5gAMiZIDAsh7cG3avV9xtkPja9AHSdqzCXtWUCdfFRplY0Roq1LD7VpsuDMCOjk5mKJRqWEWhwza+7Exx6Gv\\\\\\\/MgyJ6+b7augLWNzaYMtqe+ElBpfXbc4XVqfMqMOB+Vuf8Tt8f\\\\\\\/rH1ZQGT9Goun1FEsNmZOl4iu55WiHfhpCgADZdLz849XEqi0gCM3YGsDftYQ0mHyVhv4wdj5exnIADgUgahGZ2Sy8Fbfn5lGtFxcV7RB0u4Ru603IEcbpZV7V3IxdGjDqgEMy8XIuQTg8+STpkj7kjj+NaLalGEpEGDzkr7j1J2gEbrujr0G19OArizeAl45LnIDvHz5bUrQh2oAGb7l83YMdQzfoHgqUrg6emcW4VAi1l8ypF1K4ZI1D1JrQ0A0zJaMWoSGQuLTl2HYfAO+Ug\\\\\\\/Kwh2NIuiDy9Ky8f8OpvJMh6m3PkZzXMf8yw7SgNcFFJb+EZLghRgeitgNt9aS1RKDC\\\\\\\/eM11Auuyda4bAfQEXyv\\\\\\\/0tV4LyMRHwl+JtQefe9Yk4JJCfej8EMEuaUGdcbTcW3ghjLDhtMaLbLgTGxni83lYDc8zwLvRm5tw1Fu1V0pSSh1GvkHBUh1CJo3HZV4GboiI0ZO5gw0AqMv\\\\\\\/dhHDIj5gVUBaH+wsnzwIgsJ5NKPp+VK2s7JExkigOu4mr9solzIg5JjaddER228ZdSTbTY5HcrXFP7K7+07qYILSaqEG6JfvSmGKGHBl8p6Yt7sWpnlG0z8ciOOpplEhRNFN+TSAzB2H08y\\\\\\\/q7mK7gL0YVM2Ogbv3AVDZSvLXNRI5ARgYe4\\\\\\\/N6fojK4zxxOl+FZLZapol9sZyoo1Bhf8zstWLPiYw\\\\\\\/OBcUsUp6fAkmtcHIfjMpTJ1xKmGqOiwUI\\\\\\\/dt1DzmICxIi6M7tkQFYkScMAQsNgYTHnOGjWpj1ISNrwvpr2Jrre1xDkFSlAAf7DaHlecYzkzaWYHh2VsVoYPMdzH1CwwIkLGV+9HRXuGKKKACdiUmrsg\\\\\\\/MAeRnFtKFDFalDReFPMSk9VAmZOC\\\\\\\/5wGuTAiGtnZ5TnyOOnYw8WGhF4\\\\\\\/UOQHUWBZK\\\\\\\/UqyrXG+uV8ThBKpXF9hySex0NBDFnuc9Fx3UJ6uQ+VdDHmaBA6PMN9jWqW8llqup86xvg0InWyvr7+13Qch3GNrzxrNArDAq9YYV6iHcRkFON4cRfPwptshSJuMFl8wlCO5yOai7dSuHVbf+M4e6wjxdyZ0tVgRCyiLg6AzDbE0EQyHrrplF+DkqatLlCK8GkTp+VOb81mnd3W6r0JKDXy+skanW1GB7GqGoxSnKVe6p+P9dGegplmJXmXfy1+hddhdRlNVqXt0UDVItg2Ea7zrIx5OhMRJZzvQAxYIdfkay+7Ii1WPkxDZbMKGG0EsWs9C\\\\\\\/1P3BOrWTwfm3goI8T4\\\\\\\/eyxNi7ZXxHCozqI0O44EAgZHJgBt5BwyBXdhD+EgGLfnut9EVTm\\\\\\\/EAuQ\\\\\\\/9u7o2SUc5BaO0RdWFuPngNRtk5cpL6R6bnzX\\\\\\\/+1ayeE+smBQ0cxm6ehLjVKTPnUcOJIGSOegnJ6fgMQdVYruxVVuRYWdTybZ4X2egmmGZCrUAmoAFXy\\\\\\\/0rFHfEz9\\\\\\\/Jivd+8RGI+YKh93TPoi2aSZtQePsnbaiR61AeLrBIkYtFDklHr1\\\\\\\/pItGsnNHkTEI3ZDoZ+l7Gc6NcKtdXoL3Qhxr6qzcKSqSj2BawZ4M3FheaR8QPM9DFYQ0t0y2R46GhSQp5kvh3TctIMcSTA7DVWP3eG17FooWPX0DoJQNDVMc1d9iGohIUCq71Q8NIJD1RkGF3AEEt7aM0NNYoLS06SpggRz8K9MU1BGUWxqAHwnbbbZbMtvVDfwlo+fD+3DTlRX5qunJJfiilnFTb9NodaMMnUlL5MXb2Id\\\\\\\/1D+3zH681BnTqB44I1TyDCX2lERgRJwS4gdZWPOPtDOJhA3r5Q6sJcl1QB35\\\\\\\/Kn1XZaj1ZW7pNZE2brghWb2LTKeJmPpMBWmMj2zKjXyFS7quXmciKuy6gWG8JBEGnKUGJZMnyiRmMpeEtcSc0zIH\\\\\\\/JdWZxAP8T8Ljr7mtmSYcGBdTLfkk0gh6CMzOC+62q37kV\\\\\\\/mAI3d+T9jW\\\\\\\/RJqgQLC8oqntMDR+W\\\\\\\/263KwzVX6NmEsD9rR9DHxMcXAI08aNgkiE4yAzaJ9n2n+oMCzYgF+oYx7tYcB58dcwSUUMqnVdRPlSrnqmBHpRY2BMR9u7\\\\\\\/mG9Aq+OZ4tTh7RXbfcTFpIoqnF9ftZ7q8FJer2ZSIwvOMiOcg6IwY0G04ZUpbSuZb3rM8TZRxU23tao1EcKeRQwmeC38sAwOlCZzM1CHWWstcCwq9olcbQGFdbfq8xIPl3nedQB73AUaC8DiFr7fV1SlBePtmdyzNsdYIqXj8v8IhnIXfTsFH0+MKNsBKzlPsawps8LKW7dfL5cxgWUF18GeEXg84Aqbpg8yR6edBwCiCritRx4oVVjLcSfyNFmL2bYoW4MFZ4qVsVxeod89J15z3SArV+PLo9Fg83S9bvnFYnZOIoq2LUc2c9qNpfaUj11RyGCGgChPlYRm\\\\\\\/GhKmWvPeJSncAn81SXePUFLm925cxaB3lRlldLtcTh5El2EWLTvuoEhATtrI7qqS42DHNiH9AqAuvwlxOoSRcv8gcXtIDXvPWBqnQI0wBTTQXcVk2Jtx9+gPnzD035hW1ad1+laLcQ3S2Oir8sl1Gc0Gaisxh6rRsVc9lb9+b1eK\\\\\\\/McoAeXuRkd1z4waE1M4J8l9tgJzIrb44QuECf3H4ydpZTRq3cWu9oVW8fqHWfCq7QNaM4UVoGsvX7\\\\\\\/puALLEcvcCGDAcjL7\\\\\\\/hfsHuDHQe+GZqSBh+9r3+91NRINXk2mpxAPupc7V4HxupGVG528G+az9OqjptJiBADwjgUAqWRSPCzCIO5YCL3VluNmL2kbdqsaHTikgWj6v56enLBLhlf0M2uJ5ukvyopmyE2bfqyI6PyUzsuWITGQ9iVJZuuRDqpnNi1x8vSUKOareJP6OOwrAo6zjBYpqRkfPktOUvA3HDXB4\\\\\\\/VrpDTTAltZaqb8st0FVan8GpCHFqhzua+hkb9hGFx7xyr3IBqApdx2v\\\\\\\/+nv12nZL7jcWi1zog4p3U37GPE4RJJ4QqJXWw5\\\\\\\/kuQiWPN8ZvjBQ0ehNpO8p95\\\\\\\/9k7Y4LMmJ+536\\\\\\\/Lod6COtZId5zNGdULN5znLiSmIsANk53Om42rHN4kBMoyxI8I6L6JQ9UO9kYJuUpf2MOwOy+X0M18xAhVzwIaS1Oh+hoBiFQUNOh8CTESL7lSFWgr73m1oBdJ8uVmYwD2GW\\\\\\\/IoFqhIpOXl83l2V95gLIWVd3bGrrEo4lz1FHf8XwLuPdUWURJWKXFFFQuq9sXR5CkFDHeAadDpBU2GkdOeUsDyW96a\\\\\\\/Ddcg29QifJRN4efNpcNa4stKJMvNqfZq\\\\\\\/QkdTkQxuvaZs32f1G39b76nO9jAU9svd9Y7ng0+zwrrtFZfkhYi38CelqSiFRtUui4\\\\\\\/hsk37aW0WIH6g\\\\\\\/f1d+MZ6DMDI2gICr9\\\\\\\/dQVm6xTmUo3sYMacwnAKoU8+t+0k2jtZoUDt4NRN0ymr8XrgaJuLQgsia99euryfD\\\\\\\/3tfdVqPDemSGugR09gHfSxvPolrW4Jc1O6bL7lmuOyNNIKimhLHPYrz0xb4UqVQSIWIRuZsz6R23Lydxh7HWwUq1fwGnvOkEFE2Wpvp6srvjGZy1mHvRM32jbzZy408LYNwi3t8dCsW1XD798KUmnv6huq6RB99jbh3eVFLEewLIH2hg74o\\\\\\\/IYe3TWUPmTJ+xmazBKUzXnS47pL8wdDXZp\\\\\\\/Tg02zfyevRigB2V76UeWQ+ay1Wx63tus8wP3y4QBL4Bgy12RgyryrlUJvjW829SxtT0ZHTjSYxB67DPvN8\\\\\\\/4+wGXcU\\\\\\\/HUsHCs8aD1EPwhwoZhyHWZOpoZbaACHwbLAe9yUvZ4itQSFRye6E0Q\\\\\\\/rHbL1XC0DouCBEIlGKUvmoKGbXHoMS+t4a0CDlaF1uhfRh3A0mZtJyt\\\\\\\/uDYwKIYnwLqJtqWzf0+9Tt0R\\\\\\\/WRvnQJzTyCZGCS9sYkdPadlpENocu6neYaCkgKcmsRlAwW2dkaj6PHhLqe2U5HNZYbHmV5M1OvZMgTcAwF1JwRYllZwRPXcedDPciv2x4zPgzZRoHQK7zxFGR26QNiGoHNs4UKBzASxJ2\\\\\\\/ls6XGePrkQ1qVYXJP1qfzL6mtYsYXx9Vyj9XQYQKRy9vE6bfj6uRcV1zSpnN7+NQnLm74+R\\\\\\\/UVNIjsw\\\\\\\/9HJv6G0XzCaI0kPOHcpW0xoUEyhtjV2Z9cnaRp7St8aQdNn2NGrMo+GJdtr25yWdrRZWHPmqdgIubGxOJoRvXuutrGm3kKU+2ttdU5\\\\\\\/gZaInRqpq9zBICBgrOogx7cNGpiBKVPCcMv8NnFPChxjsSeYpvwqJfEiUhstCV94\\\\\\\/ECw8Iu0SodHdAj5+8+bRVD\\\\\\\/6Onecn45WJt5k+avotjLzTABIGtD\\\\\\\/v7DbypZdcxbkMzMbc9KKrhjufRqhUkQDn4uQ3uoHVI1lUk2ThaUc\\\\\\\/MNv99zaT+9uPOEe\\\\\\\/Zx7CIFWxM35zKCKDGH+t8psSNU2a7E5mRQ2a7E614BQ0pM4yXDO7zSorp\\\\\\\/7FovwsWPamoO8yp4HRSmUpQ2oMs7FU4kR2gp13\\\\\\\/Y7SR8OPYJAK9mx7ofy\\\\\\\/R+E5t52VNdVR+OydX6WP7tiE0H1lyr78IxLAD8gVe3j8AzXySJF0tXAFdGuDgcBEw1SMQK4goZViGFBcEfz74iJXCV1JTzpEK4VAxssRuXdnu4ez1tH4PC8Z1ptOx5SHO7TpAEW+WEfFtMWQeni7wN02DuNMZ7nP947rj5O3JsY+7m3wUibx3BBoGUupRT0upzueJ+yJWMpZ12cgjKMT5pvMXKyca1LHZQcA5d8qO+jVO4ZGcPrRLlL3Bk\\\\\\\/WlZW5bieXR20xZ0BASJFfpJRQoeHvzpWGTYGi0Kb58GUds17Pn77O1CzJUJ83XRj9ortb\\\\\\\/Gc3jJOCqY3u5ew4zKjogbOxVsDCZ69Xa5r3UZyu4kvz+pQG8WC\\\\\\\/pyaefIeeEUtgtHS30qo4HYeMUEfj+zvgEuhfbhP\\\\\\\/UFdme7xDODjdTI5K17HAX5wltfSr65o8UcAiD5iI3BG6hpuS6ILk3BYRoP\\\\\\\/s5HulXSDmPG92pTLmYi9+whvxrmgqP2uD47PrfOytHGYiB3qNWI\\\\\\\/1c\\\\\\\/33\\\\\\\/j3hNyKgmSVy095e3SaCeeRSzulkiHFfUXeomb1w3TKb64SKyyzCLmU9tn2X9zb8w4qcPn8YBSlMbO6XuscqMoq\\\\\\\/h5xm1cKr898XDwCNVrnvJa1Ee0YMLlRPnZwmuS9c5NyAarf\\\\\\\/79ummASo8uZPK33aUcwSI8ev+v+104FtA9KuBHn65ow6KZ19Knid81lPpRw2eXoF2dEFVxOL64dTjodvblHRmVPZM+Q+PeIc\\\\\\\/xYqJ6JSzCAo491Hh7M\\\\\\\/PSfFWXTpX1TDPfp6WGlMF9p1AqpnUdsccqtDOA6J6uAqL7YuD0hgU87IqHyBgYNS8FGuuWoSQleUvP8rMxKBJ1MoPi0gJDvLntphIQ\\\\\\\/gy0EGw0RdS58TEc\\\\\\\/2TzW\\\\\\\/bOJvTuWbRB7k9+40+fX8sNqgpQWpC34mrtvhZzknxwnu8v7Tf3juVWVeNq148EXZfJ1eYeltlWWqODIpscEUee\\\\\\\/sTtpkjmCqwATq95ykr4QtXgz3EerJMFzuMcaHy7GOz3dbdGN\\\\\\\/ot\\\\\\\/gL4qjbDdtBWXyN9cs0cwTuXO8fPI3pDlGXwakkRRGmk27t77FRh+PA4NbkHTBvDWuoibRLtCAjgKmJPL91SN3yGdtOovYIvnu9C0bJRi9YExTCcwUeSfq5bEY7LkbUjIA3TwfGNBb3jl0i3vRBG+IaY6Tdx5yeJsrVfqXf70qPd2WGfTGNRUBiG12fMl1QUUntA9E7GkQs0ZOnIY1XgYYuRmCC9Sswjh9WvrL70bNB2urIg9KxPVj1gmgTl5yHRKGF5gFoaUNKjx3v6OPjl\\\\\\\/CWDitfhiFTTGPlPZncBFs2Qs6WNw5\\\\\\\/I9\\\\\\\/5O4g+smILEFhrsqrFhpG0\\\\\\\/wG9WqJ0npZQYl+E3SUZA+g8cePDqNax5\\\\\\\/pVXXboAQdhyeKtqQardx6shvlF2Wpl2McHqWIf081ByGBMPKXTwx2z\\\\\\\/TqsFfeBO1VPv8hdT\\\\\\\/U5qqEpB69oOAjUJWNkLCII0ofpvHem\\\\\\\/fyxo76gKod+ovvmketqtxrCUwHFtXtzUDKNk4fB9W30y\\\\\\\/FYqQoLNdsxsgN3MtD++ilGnMZcQ7sOce83NkcV8zd6Ub1rJK5qGoahbMfjfGBIk0coxM\\\\\\\/SfXPcjbdAeFHCvxMcOIZLnxD3TImglyzazhZ3NfLZsVp3JQXX6Kcn2XEn1vMa00\\\\\\\/GSPF0N9g909BqoBBQRDMV2PghAdQgtoiT\\\\\\\/Jm1wyW+AvZpkzZviZKbvXAbzXdf8buxEV6TR\\\\\\\/ONNpUvfqaTYA3dVNEDzdT7MFn+DtrngH6f3juPRuMTehFgIGcEEeaPy5qRqA+7h3XmIbbMvzlbv9dmg0EKad23zEr3EmKEGjJTPDmYuAqsk19ZjeKFmxEa5hC3hdnXb8MqyAms9cDrh8lDAPOUX2Twt8ZPLJu79aq4NKpqaaTCfyZ9ZKRsrHfV3tpmCGevpAPL\\\\\\\/XFQ3ZtXv3bXbMgxQgATPkKhfIBkaogpJtxai7iQWm9M3I0Jpsw+XOzj2vEmykytlI7OUbWBWSFdStkSbozpLqZllkDTO0k0hXrIRpSZ0BSw9kfygN4D0sB7p0AjlLTbfAT8ygCK8lHGLkCfurhst09feI+D7uODyUYDBX5+M+afU3gOmLAYXCAFhlkC+fwb9Y5RuBoxFmrmu0Bvdn0i2HYprajHV4qAvq896PBuNU6ce8qVticm2BdjC6Q117DO2ZfNj++OZFZ4p8XwpahFL2xxwoVEk4ZSyUG+zbdx4nIxdHx7YMkouvNNEAk1n8F3z8w9jI4RadbAivPsskzU+LRqGT0rtGWSUpVb9oFoK7mHmkd9bP9R7Fe8U0wBqm2zyoUFJ0gy5OFZyMsrGu9hLhLCLs4RwxYISwsmR7M6RkpRUtsxZS38Ofpw6OhDFMpZoifr4uEHw4gYF+nr0wbHYoK3RR8hdQCfxWOyFOhFZjmkYDV0rMAtWPhwpBN\\\\\\\/ScqwTlW3s8OlVTBScadq11kySI\\\\\\\/eW9z28c0ldWkZV204x75kdydQaiirCU9ykTE9dC8uOhBsCry2fpa49X7RtOTfFtKQOQr0uWIwFkKcccdl43royNZQPh+LYnaXM+0B0oRoEqucBdzzqzvNZQUZkGmOkO0HopKfgMdMrF98aiIexnh0l5ZJAc9+T1AAPF1uMFf7RtlI0\\\\\\\/da76D9Rs9pz+IqXf1MyWOoV7ekeOob3UGE5p6SMEXhicmlntIWEBqqmgHWuYMV77vxhlgje7dnN6nXoWa4hqy\\\\\\\/QLFP4TxRCMrNENetMWhCKwOCOzg9qFQAt5PQPe07PdjQgnOyf6jwXLKsWfgWx0MTYrPZG6HnJUh1SlAIOIbAubU0wAi25tnVr4P\\\\\\\/9CHGrECq1+cvGnQG+TIuzTTG4nzGFDfueoPgZZgtak3oeVdWZ4EJDXxFNVhD\\\\\\\/t9ji06zPFQpY\\\\\\\/abVyfh7w+igkCR\\\\\\\/eiz2kUuQiLU+oD2w2wKpp0uuJP\\\\\\\/dmfLAR181JbMhAw5L7rkrNPywhSQAISyIG\\\\\\\/g8r6jIHkMhk3AOn4A\\\\\\\/\\\\\\\/XP9MRtC+LoTE5wVB3HQZ8HEls12L8mJ5IfVsCvfo\\\\\\\/gwKS75gUbGi8QPcYPep7hnqC2EpXaEHCueqRF+WbePIT6LAp7oByCyYUxtTS4g3DNV0UaaiieHNjWxGyk9JCOSVadorgWRHw0Wqh2QjrozYmUWto34N1VU3xm5FEFDiW2W5F\\\\\\\/NfZz2x7omE4X49hIU2HbOzNSi9Y+sMLjpiHDMhesgX\\\\\\\/\\\\\\\/WK2HQ2XWNFUq4NOMCfK2qnPPqeDqZuHMK+H7ajdWxI45WP4BltFpxff6ijrKs+toLHFLg0H2TdPg6T8U8KHYK2eTLFEDl\\\\\\\/sKebNJ+9H4hx4ad9IH80b8wFNXF8ReB24EkaBYPdC2MDaiQ+SwbXhIWZP2EeghNNBupXAZCDu2pVpN40o3tg9bI4jtkWxt1i2s5TqpoLE2IcM0DThxoDFmlPHhqxKbJc+0XMT4IBEJRozUzGBp53ePNYWe4kSmfPYi7TRzV80iURaMEGLRxQHYgqpxS6QDHHw\\\\\\\/uWUnfN\\\\\\\/3SHA9Jyvs9\\\\\\\/SbXA3mKIHn\\\\\\\/YWcnfExqJVvJl+Q6ZvVFJSRptjjGx4zZuVs3Ul\\\\\\\/389mKh+Z3Nruxa2SSLQ8G+7J9QAVKoYzsILW0nIbmwVoWklia0QULdsgV+uSjjSAxMlo1QIr+HKhelAgigUMU47sgCbjPYJPhB8lgv2scac12PpweFO0OwBtitZd5ROL3mXE5dASzKbInGS30CjoENNn0FKNFmJNrvlMZ2aen\\\\\\\/MuTnkv\\\\\\\/+8x8jKzLL3my2i8NG4E3sp6KgNcVi2abzRtxrCzK7ieYc6AGdX2QoiB7pE8jf6BEqM678wVLHW1Jz1kKR95\\\\\\\/9Yf+nc1FN9RD9BvzBzYLw5ZEbJZF7ICBMVkdrhQLiPhms0lernznIf6Qqwhxv1CSavCNJlpfL6dDFKRAmaUoj9oafA3DTQb2c+k5MPgzRep4BjwYffhiH4AFQHl748EieDa3NxYaUZvPAvGjcAhSXdlnUbATBHxSVDradaCAVSTqXKTy7fr87xVXYYNlv6CXTRWaT0USodMsw0Czqz7c4U+2L1eMLeY8xp3kw7pvliJNYOc51Ar702DDyp5rr02\\\\\\\/4R\\\\\\\/wliWrevSvbNzMfhGIv2pQN+1rG0xru\\\\\\\/pQJ8AoixGo0J8NybJUTBWWSw4uw2SzD1fNdV7BV7YWuowzmGkHmyTbagNrryO\\\\\\\/y+lbr4dxd6kA4rv2UeOOneGsKqkWleUa\\\\\\\/1jN0NB4YPsUuicgzoBxR2kf3nqz9siUsf1Pb1EZ4NUQtcETi66z92lzrcSpWCMtMNp6Uz5dOzS\\\\\\\/rqsnLygwXr6kNtefVrbVyWCagV3x8QUXb9SUOj2RUBzh0L2Bjl0Xt6tp0XR\\\\\\\/9kQ5bQj2IXDPw3Q1b12T7P5tgrfsLA2Zhm14TZ+KAMDoAvX83xo4CL0RKPNjx6T8PdsqIxgaUsTNvqKWsJ44sXkTm8\\\\\\\/lE1MomFtok\\\\\\\/3mn4CKrdoKLEsZAhaKs1aeyEfP7ta6qLGhMWAXx+Zl7\\\\\\\/iUGEvn5kve9d2V0TPUYZK8c63S9kWqS8JbixYxTgzLu0NqhTpRUmJblekMbuInw19N3GMkg2TdzxLuMZ5kJrbsC3fEYMSLmIcnuln2QHp+Iuc6acnxf1rcri6TLo3voIPQiXOBXQvLkhk3lxu+TjQ5nIzus2nEj3P\\\\\\\/CcnQlkmIrKnEaq2GglaPJuD7QZg+ljfOTWeHkLubg+BhG0wSBvzSP+MPRHbNtMVrzhLXWCXxTcO4YApHLjDma0DrSJzMnr+Cu+icQm+h+2iI9mU2+Jf+TUXNa+lpRoZ5a7ZgE1b+QEiMsfFUKq4qHkgVi002y7UJPO3kQ2TVnfGTDJZFHvC4YnHsvg1f1nXU00TvQUNsl4LHKrQS9UIH5tcMFIYcgVcH4NCxj+NbReODUl6GYyxd723z5ZCONgrG7L8etrnEQLEMk6FMvbDb4uaEc5Tgcfs8uHX8SovaF1IUCs0DpJuYGiRBSow+l\\\\\\\/akmiQ0M9O+Gf82339N3oiRG7rS+koLD6IzPx2QhO1\\\\\\\/i3xSndPQHV5nhVbC0jLxniUhqrPvL72JG8FvGZj6t6Iq74Lbl1Ak4KHoSZAKI62r2yHCGdaCjSR2lQ8\\\\\\\/3kyym74au\\\\\\\/U2ZPZUVkRDePmwmrwbX5VKhoKQbE8x53hT9Y7Vy9H7gm4tIVx+fZWNWtzuqrT7tA77oosqdtRiXte\\\\\\\/3U7yE\\\\\\\/ry47ua56tKnpujy7OQpy11CgEKDt2rT4wbIci9bjCYAlU5ZlirLxE2QXmlo0KaRKqQwoIdLqIsUEF2KQ4eV7Np\\\\\\\/O4UZCqGzFInp8DMd+OCJ\\\\\\\/AoQDGRHjXwMAA8CZ9yh4SBVYsM9V+FqoyO1IHAx+aZeYjouEZEn9a\\\\\\\/geA6U3utDn4p4V210kFcVZlcccg2qoUq8IIzrpQLt2xGM2Bb2ETWYZZCB2arBasOG6WV9VwSQdyPSoFRMSXipc8gln+udahjO4YjvD4pqOQuLjsw+WZ3mf+CT5HTs7SctjeMOClYFbVBD2yO4EgjLyE9fJUlu\\\\\\\/28arAnVC+FWNilEpczaMyq8KIzl39+r\\\\\\\/Ut5GMWjUAQthatmfnGyqbLEqfNY609S8DMRhKroAIl5gTcp9KPOwOj2ui8CubqjQ00L0vyZxNDXurwZdJWzAZz0xeY+4wpYIhbD3N3aZ\\\\\\\/o1OAlD09maiATGq9EJGKYq4i8wXOc+DpR8+ZhXRIgltd3U4IjG0P4wDYZGZBduBF2Fpok2CaLZroFyVogn2jD9u7aE+DpVz6Y\\\\\\\/6fOLF\\\\\\\/CHmSpBXYWiwzPB1QNSq9KmUzAS+d6lM4fR6iE9b6ziw3LH6myRHknZJiPAQ+Flgs13KEZ33gqC0IZUTef12\\\\\\\/DxuyGalj5aXPredp22Tf5auMbjisrMEqj6BlOZfnT+sqmyXR\\\\\\\/dLLjlTDnE7DkhgrwMkfLiy53fNh7sxuVfydUVpF3pN+G0sMSHFzqLX4CMIWONWVhl\\\\\\\/iTbPaGfyDtpYnRyft7TgqdEUF6UBFKdyydZRimQ7an\\\\\\\/5YCaY6U0Xb0XI6GZWP323WdhV65cOyPzf72aVhLtpnzF4II9PLpzEByK6YDZ7JXcPSabQD4zCD9ktAOyOpRoPW0K7iu9JLMQdLat71Nmc4Uy0Minmb27kjU\\\\\\\/ATRjDzbhbw3Up73UE5FxNZyd8z0myYTp\\\\\\\/WSN+fSCLGQitSB4kWGTh4100YtCoa9+ac3vOAkhdVfpDXqoVvQXj48UulrA5NaJgI4Vb7P3XUabWW\\\\\\\/SFi\\\u0022,\\\u0022iv\\\u0022:\\\u00228dSQ54xKWV4e4dNmKGHlQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220504bde1a7ee6ba7f4d749ecab2e344132efa1dfdaaf065942a2237cadf1e2b3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dBXCdD\\\\\\\/+b0XgPDIHkUn9+sm1Ld+61CUdJUVHQ35nBt\\\\\\\/oPkN4i4TNpJx3E\\\\\\\/4WR05KEdQelFo8N\\\\\\\/HRl1ZckBaqUQeFgdTfa5NuxnPJvlrdn5ey1mrzsWtcUeF4sc5kdUs8HKXYyQo4OAwNAXzrXiH71VtRXK7sHWmdZgeWzk8q1R81fcTsfTpFlo2QtU0Ydb+SfYhiePGk0ty5l77Z04+WLaO2qUWsZAZBeAKovVxx0MbuWTV4rE5rt1Q20WaUn0lvXVXZzl3Yvlm2di7H\\\\\\\/1x9YlKH6pgqHAxkw\\\\\\\/AKZCroe0hE2PqeP3kUbwN+yvIVy+TBq4S4l+U3fLds9WugzhcgrzNAdXBv5dzTtQVqB6WinLYGT7LZI3VALm8\\\\\\\/svnMQNvDs5uuiYXRlD1jEb\\\\\\\/\\\\\\\/bT+CMQB3xEW5Kl63iPddw3\\\\\\\/v0WisEuSo+1kAYAidbJnBJMzie\\\\\\\/Xd2KJs9xim+auGRp0zYlwAwF0kg5tI1lV4bmxE8wV6uYwIIXoJlmZjKwA+ZvVsijbsK5rg9k7Fuj87WWfEUgyJisSdbsXjmWSMqsZbKpGg7JqBgEBM\\\\\\\/5+lVQxhGObikHwo2iwKCEGUeL56PJiEFAM5usK1bU+BKrk9CZlWTXOolcEHJrJ2OhKcfYAuxTeo3IlJSE1RD+GBEhNNOihKcT1+eOu89W4mwBnia+ImiQ3i9N2pnL3kQantaqC05sYCUeabZOpT1Jade1l8wVRSU7cZ75HOvhMIEYMCCUujZGabmKyg2bxqvj7fWpK+CkSStS0VMn0ElsqtfjBaAoHJNTzfbtzM2Wh1D72mbg77UP\\\\\\\/gK8rkjvi4hEvzk0X4nW\\\\\\\/PHK06bhFthZ7G\\\\\\\/4\\\\\\\/ugHiPLM25v46qNQyt0RtAwdkRliMqhoLZsL02RLxAywMCS7Sda04lJULVsvdt8ulXqcUlhCzorx67VuhDZyGo1RctckUmX9jaF7kgnGjj6x2dXoOaeJWeFUc2CmPR+UBQVLMQ6ACjyPC3bZ0L6QTtXh5w7extXyVkv3eCszalkH9FV5CPGzOu+MXL06YPtUIi9lSyLhQVVJ7JcalID9iX00A7KHZe9cqmeVSDVNv46P8TiopZy6wMe3hcQ908SByzjCNZrJc6r8oOOQ+0X9TZFGLRaHIUr9mdzNRWIAWa+WkJPSvWXSjrB7DEFo3noeCC2pYRkT69rJeTtbA0BGr8LGa+7p7fJ4CeyfifoY\\\\\\\/9AuuR1obKAoeXSeh2htGO9benE9SthMdbuoRlaIEcYgttAu0EGLsoErsjnpjB3IfYqReiNG1CU20a09a+Zb3arsnv+WCtKojhRn4Sw6kLPBbEfv2V9YzlWHJ+PU+P77NU5ok5MJXIpiYa9fR24FRonvspPBnv+vdPJT7NVV6rCILY5Ufr5ClDF2di17V0cfgdLmEhJwm++AZjxe56VBGejwou08VMVZAu7RaDy3Q5ioore5uXIKQM3XLAeRmSLULBdMWQTdW3ncNn0dgXOVJyHzFHFZX5Hfx4eT682cd4\\\\\\\/1SZmtdxdJj7nCQf4qc2FmGGKvacCDf8N0I4IZkMmrsoCNkqtIZZVTB6TVJ7O3YQL1kq9BpRp7UKMTXDRWchHV2yjoWep9ExH95\\\\\\\/rv9b8JQcQ7j\\\\\\\/oXEx36vW356WBhxevNHwthFXffCLYgYZTkBUBqg5UvQxBj3Uu624Q2\\\\\\\/VAyRPpiei5HPTzMw9X6dN\\\\\\\/ZeiGgrTw7D+RMMvQw81pR3nbt2aq8XfVUTcDlia2D1d+jF1ZwkncNbgLDk\\\\\\\/kG1utYQ439KvfYXQ\\\\\\\/h1xqVYW7MqKXOOH+5w1QHnnFnbudYbGZkAilOhSkzGr1UR48dBAxRP7ABF5QHQBfLx4Ze06ny6ii7oLoICeeNhCQE7qwNLcblv1J64TanUN+aAFVORhrlIrWjezXjNQbfUjTjZoQUOKoXnAUV3j3V3a1O5r395MahpT74lKM\\\\\\\/+p5XJk48gLYn0CayhQhk7OrNLaXvpNGrT9iA16I+gahtTJcIoSugYRcijEpeY3ASxNdzw\\\\\\\/zJn1QlAQ3USBwD5eqXg=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Iw04fordMapZ0WCqLJVv4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221ebaf208e737d87a7b10fdbe1508f68d768ce3f00ef0644a9ee5b6ec4a35bcd0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460400 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1899" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vdLHHHB6z4R2m6-YZGok7_oPQ1PZbWPFaY_0FWXrQxI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1899" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8HhWqHZSS7rQtO3Kh3XNh2F9xoufCNUPn5jGpulrD2U" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1891" title="Cytotype Regulation Facilitates Repression of Hybrid Dysgenesis by Naturally Occurring KP Elements in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1911" title="Placental Hypomethylation Is More Pronounced in Genomic Loci Devoid of Retroelements" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460400" id="node460400" data-pisa="ggg;6/7/1899" data-pisa-master="ggg;g3.116.029207" data-apath="/ggg/6/7/1899.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Patching Holes in the <em>Chlamydomonas</em> Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6053-4093" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frej Tulin</span> and <span class="highwire-citation-author" data-delta="1">Frederick R. Cross</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1899-1910;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029207 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460400" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frej Tulin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frej%2BTulin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tulin%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrej%2BTulin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6053-4093" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frej Tulin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frej.tulin@slcu.cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">frej.tulin@slcu.cam.ac.uk</a>
<a href="mailto:Frederick.Cross@rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">Frederick.Cross@rockefeller.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Frederick R. Cross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BR.%2BCross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cross%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BR.%2BCross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frej.tulin@slcu.cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">frej.tulin@slcu.cam.ac.uk</a>
<a href="mailto:Frederick.Cross@rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">Frederick.Cross@rockefeller.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/7/1899" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460400" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460400/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1899.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460400" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460400/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1899.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460400" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460400/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1899.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460400" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460400/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/7/1899.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460400" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460400/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The <em>Chlamydomonas</em> genome has been sequenced, assembled, and annotated to produce a rich resource for genetics and molecular biology in this well-studied model organism. However, the current reference genome contains ∼1000 blocks of unknown sequence (‘N-islands’), which are frequently placed in introns of annotated gene models. We developed a strategy to search for previously unknown exons hidden within such blocks, and determine the sequence, and exon/intron boundaries, of such exons. These methods are based on assembly and alignment of short cDNA and genomic DNA reads, completely independent of prior reference assembly or annotation. Our evidence indicates that a substantial proportion of the annotated intronic N-islands contain hidden exons. For most of these, our algorithm recovers full exonic sequence with associated splice junctions and exon-adjacent intronic sequence. These new exons represent <em>de novo</em> sequence generally present nowhere in the assembled genome, and the added sequence improves evolutionary conservation of the predicted encoded peptides.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/transcriptome-assembly" class="hw-term hw-article-keyword hw-article-keyword-transcriptome-assembly" rel="nofollow">transcriptome assembly</a></li><li class="kwd"><a href="/keyword/trinity" class="hw-term hw-article-keyword hw-article-keyword-trinity" rel="nofollow">trinity</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/chlamydomonas" class="hw-term hw-article-keyword hw-article-keyword-chlamydomonas" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Chlamydomonas</em></a></li></ul><p id="p-4">The assembled <em>Chlamydomonas</em> reference genome is 120 Mb long, 65% GC, and repeat-rich. The assembly contains 17 chromosomes (∼1–10 Mb) and a further 37 ‘scaffolds’ (0.1–0.8 Mb) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Merchant <em>et al.</em> 2007</a>). The reference assembly, and an annotated genome including full transcript models (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blaby <em>et al.</em> 2014</a>), is available on the public access Phytozome website (<a href="http://phytozome.jgi.doe.gov">http/phytozome.jgi.doe.gov</a>); we will refer to the sequence assembly and annotations as ‘Phytozome’ or ‘reference.’ About 3.5% of the nucleotides are indicated as ’N’ (unknown sequence). These Ns are distributed in ‘N-islands’ of varying length, mostly ∼100 nt; most islands are annotated as being present within intronic sequence. These islands are effectively ‘placeholders’, where the length of the island is intended to reflect an estimate of the amount of missing sequence (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Merchant <em>et al.</em> 2007</a>).</p><p id="p-5">Here, we present bioinformatic and experimental methods to determine whether N-islands assigned to introns are entirely intronic, or instead contain exons missing from the assembled sequence. Our methods allow determination of the sequence of these hidden exons along with flanking intronic sequence. While we focus here on the specific set of N-island-containing transcript models in <em>Chlamydomonas</em>, the approach can be used more generally to ‘proofread’ transcript models from publically available or <em>de novo</em> assembled transcriptomes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-6">Genomic sequences and transcript models (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Merchant <em>et al.</em> 2007</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Blaby <em>et al.</em> 2014</a>) were downloaded from Phytozome. Illumina RNAseq libraries were described in <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Tulin and Cross (2015)</a>. Illumina genomic libraries were from parental strains in the CC124 background used for the RNAseq experiments (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Tulin and Cross 2014</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">2015</a>). This background is closely related, but not identical, to the genome reference strain, CC-503 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Flowers <em>et al.</em> 2015</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gallaher <em>et al.</em> 2015</a>). We used these RNAseq libraries because we were especially interested in genes involved in mitotic progression; many such genes are highly enriched for expression in the synchronized libraries we used (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Tulin and Cross 2015</a>). Alignment of RNAseq reads to Phytozome transcript models was described (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Tulin and Cross 2015</a>). We downloaded BLAST (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>), Trinity (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Grabherr <em>et al.</em> 2011</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Haas <em>et al.</em> 2013</a>), clustalo (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Sievers <em>et al.</em> 2011</a>), and the Bowtie2 aligner (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Langmead and Salzberg 2012</a>). The latter was used in ‘local’ mode to align genomic reads to Trinity objects. Trinity and BLAST results were analyzed with shell scripts and Perl code. Analysis of genomic reads aligned to Trinity objects was carried out with custom MATLAB code.</p><p id="p-7">We used the Trinity software suite (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Grabherr <em>et al.</em> 2011</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Haas <em>et al.</em> 2013</a>) to assemble a <em>de novo</em> transcriptome for <em>Chlamydomonas</em>, using RNAseq reads that we generated from multiple diurnal cell cycle time courses (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Tulin and Cross 2015</a>). Since we were primarily interested in evaluating N-islands within gene models, we restricted the RNAseq data input to ∼57 million 50-bp single-end reads that either (1) aligned to a transcript models that contain at least one N-island, or (2) did not align to any transcript model.</p><p id="p-8">To focus on sequences that are likely to contain useful biological information, we filtered the initial 10,134-sequence Trinity output (median sequence length = 315 bp) to retain only sequences that had an e-value (BLASTX) to <em>Volvox</em> or <em>Arabidopsis</em> (TAIR10) peptides of 0.001 or lower. <em>Volvox</em> is a recently diverged multicellular relative of <em>Chlamydomonas</em>, with approximately 50% neutral sequence divergence but high conservation of many predicted encoded proteins (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Prochnik <em>et al.</em> 2010</a>). This reduced the number of sequences to 3473, indicating that many shorter sequences may be either Trinity assembly errors or else too short to generate significant BLASTX alignments. The shortened list contained many similar sequences. To filter out shorter fragments that were potentially partial versions of longer sequences in the set, we used the uclust software (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Edgar 2010</a>). This narrowed the result to a final list of 3114 unique Trinity-generated primary sequences (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS2.zip">File S2</a>), enriched in longer sequences (median length = 446 bp) containing potentially conserved protein coding sequence.</p><p id="p-9">Perl and MATLAB codes and detailed results from the analysis are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS3.zip">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS5.zip">File S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS6.zip">File S6</a>.</p><div id="sec-2" class="subsection"><h3>Data availability</h3><p id="p-10">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article. Complete raw data and accessory files used in the computations are also available on request.</p></div></div><div class="section results" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>Targeted de novo transcriptome assembly</h3><p id="p-11">Blocks of unknown sequence (‘N-islands’, see above) could result from assembly difficulties from blocks of repetitive or low-complexity sequence, which are highly abundant in <em>Chlamydomonas</em>; or from regions of DNA that are intrinsically difficult to sequence (<em>e.g.</em>, inverted-repeats). Such sequences can cause loss of adjacent high-complexity sequence from the assembly. From the list of Phytozome reference ‘primary transcripts’, we recovered a list of 789 intronic N-islands of varying length. An intronic block of unknown sequence could contain unknown exons (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). <em>De novo</em> assembly from RNAseq reads could recover the missing high-complexity sequence, even if it is hidden in low-complexity, or difficult-to-sequence, regions in its intronic context.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="N-islands annotated as intronic can contain hidden exons: the separated high-scoring pair (HSP) test. (A) Top: Gene discovery algorithms confronted with blocks of unknown sequence (&#x2018;N-islands&#x2019;) between likely exons are obliged to interpret the N-island as entirely intronic (since otherwise the algorithm would have to posit splice acceptors, donors, and exonic sequences entirely composed of &#x2018;N&#x2019;). If this assignment is correct, then the mature mRNA will seamlessly join exon 1 to the left, and exon 2 to the right, of the N-island. Bottom: the N-island could harbor an unknown exon X, incorporated into mRNA between exons 1 and 2. (B) &#x2018;Trinity objects&#x2019; are produced by de novo assembly of RNAseq reads (Grabherr et al. 2011). Top: if such an object is derived from an exact exon 1&#x2013;exon 2 splice, then it will align in one continuous HSP in BLASTN analysis to the predicted set of mature transcripts. Bottom: in the case where an N-island contains a hidden exon (A, bottom), the Trinity object assembled from the mRNA should have two HSPs separated by Trinity object sequence; the extra sequence is equivalent to exon X." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;N-islands annotated as intronic can contain hidden exons: the separated high-scoring pair (HSP) test. (A) Top: Gene discovery algorithms confronted with blocks of unknown sequence (&#x2018;N-islands&#x2019;) between likely exons are obliged to interpret the N-island as entirely intronic (since otherwise the algorithm would have to posit splice acceptors, donors, and exonic sequences entirely composed of &#x2018;N&#x2019;). If this assignment is correct, then the mature mRNA will seamlessly join exon 1 to the left, and exon 2 to the right, of the N-island. Bottom: the N-island could harbor an unknown exon X, incorporated into mRNA between exons 1 and 2. (B) &#x2018;Trinity objects&#x2019; are produced by &lt;em&gt;de novo&lt;/em&gt; assembly of RNAseq reads (Grabherr &lt;em&gt;et al.&lt;/em&gt; 2011). Top: if such an object is derived from an exact exon 1&#x2013;exon 2 splice, then it will align in one continuous HSP in BLASTN analysis to the predicted set of mature transcripts. Bottom: in the case where an N-island contains a hidden exon (A, bottom), the Trinity object assembled from the mRNA should have two HSPs separated by Trinity object sequence; the extra sequence is equivalent to exon X.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F1.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/7/1899/F1.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468686" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-12" class="first-child">N-islands annotated as intronic can contain hidden exons: the separated high-scoring pair (HSP) test. (A) Top: Gene discovery algorithms confronted with blocks of unknown sequence (‘N-islands’) between likely exons are obliged to interpret the N-island as entirely intronic (since otherwise the algorithm would have to posit splice acceptors, donors, and exonic sequences entirely composed of ‘N’). If this assignment is correct, then the mature mRNA will seamlessly join exon 1 to the left, and exon 2 to the right, of the N-island. Bottom: the N-island could harbor an unknown exon X, incorporated into mRNA between exons 1 and 2. (B) ‘Trinity objects’ are produced by <em>de novo</em> assembly of RNAseq reads (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Grabherr <em>et al.</em> 2011</a>). Top: if such an object is derived from an exact exon 1–exon 2 splice, then it will align in one continuous HSP in BLASTN analysis to the predicted set of mature transcripts. Bottom: in the case where an N-island contains a hidden exon (A, bottom), the Trinity object assembled from the mRNA should have two HSPs separated by Trinity object sequence; the extra sequence is equivalent to exon X.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">We carried out a <em>de novo</em> transcriptional assembly (see Materials and Methods). The assembled sequences, even after filtering, likely include some assembly artifacts (for example, track-crossing errors at repeat sequences), in addition to authentic <em>in vivo</em> mRNA species. We refer to these sequences as ‘Trinity objects’ to reflect initial neutrality concerning their biological status.</p></div><div id="sec-5" class="subsection"><h3>Coverage of splice junctions by Trinity objects</h3><p id="p-14">To estimate completeness of the Trinity-generated transcriptome, we performed a BLASTN search with Phytozome transcripts against a database of Trinity-generated objects. Since we wanted to resolve intronic N-islands, we used only Phytozome transcripts predicted to splice out N-islands in the search. There are 821 such transcripts; we analyzed 647 of them, for which we found aligned Trinity objects. These 647 transcripts contain 9720 splice junctions, of which 789 are annotated as splicing out N-islands.</p><p id="p-15">From this set, of 8931 junctions not associated with N-islands, 7141 (80%) were ‘covered’: a Trinity object aligns with continuous coverage across the junction (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>, top). This result supports the <em>in vivo</em> splicing event proposed for these junctions. For 20%, we did not recover Trinity sequences aligned to either side of the splice junction (‘no information’). The ‘no information’ category could reflect many causes, including low mRNA abundance, Trinity alignment difficulties due to repetitive sequence, or incorrect gene models, such as intronic or intergenic sequence predicted to encode a splice junction.</p><p id="p-16">In contrast, of 789 junctions predicted to splice out N-islands, only 272 (34%) were covered (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/TableS1.xlsx">Table S1</a>). There was no information for 396 (50%). For 15%, we found evidence that the N-island-associated junction was associated with hidden exons within the N-island (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>, bottom, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/TableS2.xlsx">Table S2</a>).</p><p id="p-17">The striking enrichment of the ‘no information’ category for splice junctions splicing out N-islands suggests that presence of N-islands may correlate with (or cause) difficulties in construction of accurate gene models; this possibility is hard to evaluate directly. We focus below on validating the finding of hidden exons within intronic N-islands. The overall frequency with which intronic N-islands actually contain hidden exons is hard to estimate with confidence due to the large ‘no information’ category; the simplest estimate is 26% (104 with hidden exons out of 393 for which we have Trinity information).</p></div><div id="sec-6" class="subsection"><h3>Hidden exons</h3><p id="p-18">If an intronic N-island harbors hidden exonic sequence (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>, bottom), we may find a Trinity object that ’bridges’ the splice junction using two high-scoring pairs (HSPs) flanking the junction (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>). Blastn alignments were done with the ’ungapped’ option to force generation of distinct HSPs, rather than introducing gaps in a single HSP. This option was particularly important for discovering very short hidden exons. We also required extra sequence from the Trinity object between the HSPs, to spurious detection of BLAST results with two HSPs that overlap due to direct repeat sequences at the HSP termini. A ’half-bridge’ can result if the Trinity object is too short relative to the amount of missing sequence to make a full bridge (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/TableS4.xlsx">Table S4</a>). To score as a half-bridge, we required that an HSP align within 50 bp of the N island, with a long (&gt; 60% of the Trinity sequence) unaligned tail extending in the direction of the N island. We also required evolutionary support by Blastx alignment of the Trinity sequence to <em>Volvox</em> peptides (see below).</p></div><div id="sec-7" class="subsection"><h3>BLAST analysis against Volvox to determine evolutionary conservation of bridging sequences</h3><p id="p-19">Trinity sequences bridging an N-island-containing splice junction (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>) suggest identification of missing exons; however, they could result from artifacts of <em>in silico</em> assembly. A Trinity object bridging an N-island should frequently improve BLAST alignments to the proteome of <em>Volvox</em>, a close relative of <em>Chlamydomonas</em>, if the bridge contains genuine translated exons. Such improvement should be highly unlikely if the bridge structure is the result of a Trinity assembly artifact. A similar analysis was useful in evaluating biological significance of annotated 5′-untranslated open reading frames (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cross 2015</a>). Therefore, we carried out BLASTX with nucleotide sequences corresponding to the aligned HSPs plus the intervening sequence provided by Trinity (’extended’) as query, and the <em>Volvox</em> proteome as source of subjects. As a control, the intervening sequence was scrambled. We required that queries with and without the intervening sequence both hit the same <em>Volvox</em> peptide. To determine how much score increase we should consider significant, we supposed that if the intervening sequence was not translated <em>in vivo</em>, then small random score changes should be similar with scrambled controls as with the intact sequence. We found that the scrambled control never showed a score increase &gt; 10, while such increases happened very frequently with the intact sequence. While a BLAST score of 11 is trivially attained in a search of a query against an entire proteome; the reason such a small score is significant in the present case is that the subject for the query is a single <em>Volvox</em> peptide preselected on the basis of alignment to the flanking regions common to the reference transcript and the Trinity object (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Increases of this magnitude were biologically significant in evaluating upstream open reading frames (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Cross 2015</a>). Note that, since the assay is for improvement of BLAST score, the result is specific to the novel segment in question—it is independent of the quality of alignment elsewhere in the sequence. In general, the newly aligned sequence aligns across its length, and lacks stop codons or other indicators of internal errors, though we have not evaluated this comprehensively.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Authentic missing exons can encode evolutionarily conserved sequence: the Volvox BLASTX test. Assume that a Trinity object, but not the most closely related Phytozome transcript model, contains a hidden exon (Figure 1). Volvox is a close relative to Chlamydomonas. If BLASTX of the Trinity object against Volvox gives a score, S1, due to alignment to segments A, B, and C, then BLASTX of the corresponding best-hit Phytozome model (A&#x2013;C) will give a score S2 &lt; S1, due to lacking alignment from the middle segment lost in the N-island. As a control, scrambling the intervening sequence from the Trinity object (random permutation of the amino acid sequence) should eliminate any scoring advantage, yielding score S3 &#x2248; S2 &lt; S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Authentic missing exons can encode evolutionarily conserved sequence: the &lt;em&gt;Volvox&lt;/em&gt; BLASTX test. Assume that a Trinity object, but not the most closely related Phytozome transcript model, contains a hidden exon (Figure 1). &lt;em&gt;Volvox&lt;/em&gt; is a close relative to &lt;em&gt;Chlamydomonas&lt;/em&gt;. If BLASTX of the Trinity object against &lt;em&gt;Volvox&lt;/em&gt; gives a score, S1, due to alignment to segments A, B, and C, then BLASTX of the corresponding best-hit Phytozome model (A&#x2013;C) will give a score S2 &lt; S1, due to lacking alignment from the middle segment lost in the N-island. As a control, scrambling the intervening sequence from the Trinity object (random permutation of the amino acid sequence) should eliminate any scoring advantage, yielding score S3 &#x2248; S2 &lt; S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F2.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/7/1899/F2.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468608" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Authentic missing exons can encode evolutionarily conserved sequence: the <em>Volvox</em> BLASTX test. Assume that a Trinity object, but not the most closely related Phytozome transcript model, contains a hidden exon (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>). <em>Volvox</em> is a close relative to <em>Chlamydomonas</em>. If BLASTX of the Trinity object against <em>Volvox</em> gives a score, S1, due to alignment to segments A, B, and C, then BLASTX of the corresponding best-hit Phytozome model (A–C) will give a score S2 &lt; S1, due to lacking alignment from the middle segment lost in the N-island. As a control, scrambling the intervening sequence from the Trinity object (random permutation of the amino acid sequence) should eliminate any scoring advantage, yielding score S3 ≈ S2 &lt; S1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Seventy-seven of 104 sequences that bridge intronic N-islands, and 13 of 13 that half-bridge N-islands, passed this criterion, and thus are highly likely to contain additional exonic sequence. This criterion is conservative, since a substantial proportion of <em>Chlamydomonas</em> proteins have either absent or patchy alignment to <em>Volvox</em> peptides.</p><p id="p-22">Peptide multiple sequence alignments of translated Trinity objects to Phytozome peptides from <em>Chlamydomonas</em> and <em>Volvox</em> are found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS4.zip">File S4</a>. Lower-case letters in the Trinity sequence represents novel sequence that was not aligned by Blastn to Phytozome transcripts. The corresponding Trinity nucleotide sequences, with the Blastn-aligned parts in upper case and the unaligned parts in lower-case letters, are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS2.zip">File S2</a>.</p></div><div id="sec-8" class="subsection"><h3>Alignment of genomic reads to Trinity objects to determine consistency with in vivo RNA splicing</h3><p id="p-23">Trinity objects are assembled from short cDNAs. A Trinity object could be: (1) assembled from RNA sequence transcribed from a single contiguous stretch of genomic DNA; (2) assembled from RNA transcribed from discontiguous stretches that are spliced <em>in vivo</em> into a single RNA; (3) assembled from RNA transcribed from discontiguous stretches that are combined by Trinity into a single object by misassembly <em>in silico</em> (most likely by ‘track-crossing’ at repeat sequences).</p><p id="p-24">If a segment of a Trinity object was transcribed from a contiguous stretch of genomic DNA, then genomic reads aligned to the segment should tile over a single continuous matching sequence. In contrast, if the segment was transcribed from discontiguous segments, alignment of genomic reads should break into multiple discrete islands. We aligned 100 bp genomic DNA reads to the collection of Trinity objects with Bowtie2 using ‘local’ alignment (which allows alignment even if only a portion of the read aligns to the template). This leads to a highly specific signature of an alignment discontinuity: an ∼100-fold excess assignment of ‘starts’ and finishes’ of alignment to nucleotide positions exactly at the point where two discontiguous sequences are joined in the template (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B</a>). These positions were used as seeds to grow a contiguous genomic region out of all reads initially aligned to the Trinity object, by iteratively constructing a majority consensus, discarding reads that fit poorly, and recruiting new reads that aligned well and expanded the consensus. The process terminates when no unrecruited reads both align to the Trinity object and also agree with the growing consensus. The latter criterion is highly effective in discarding reads that align solely by an internal repeat. The result for each discontiguous segment of genomic DNA aligning to part of a Trinity object is an island of sequence supported by a tiled series of reads, with sequence not alignable to Trinity extending from either end (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The extensions are typically of length ∼80 nt, equal to the read length 100, minus the minimum 20 required for Bowtie2 to call a local alignment. Ideally, one such island is found for each discontiguous sequence making up the Trinity object.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Local alignment of genomic reads to transcripts gives a signature for junctions of discontiguous genomic encoding segments. (A) Tiled Illumina reads derived from the genome covering discontiguous segments E1, E2, and E3 (top) are aligned locally (not requiring alignment across the complete read) to the fused structure E1&#x2013;E2&#x2013;E3 (bottom). These reads have starting and finishing positions of alignment to the E1&#x2013;E2&#x2013;E3 structure (green and red dots). (B) The result for random starts and finishes of reads in the genome is a huge enrichment of start positions at the beginnings of the genomically discontiguous segments E1, E2, and E3, and for alignment finishes at the ends of E1, E2, and E3 (top). This enrichment is on average the read-length (e.g., 100&#xA0;&#xD7; for 100-bp reads). The plot of starts and finishes of alignments by position then yields a highly sensitive indicator of the segment boundaries." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Local alignment of genomic reads to transcripts gives a signature for junctions of discontiguous genomic encoding segments. (A) Tiled Illumina reads derived from the genome covering discontiguous segments E1, E2, and E3 (top) are aligned locally (not requiring alignment across the complete read) to the fused structure E1&#x2013;E2&#x2013;E3 (bottom). These reads have starting and finishing positions of alignment to the E1&#x2013;E2&#x2013;E3 structure (green and red dots). (B) The result for random starts and finishes of reads in the genome is a huge enrichment of start positions at the beginnings of the genomically discontiguous segments E1, E2, and E3, and for alignment finishes at the ends of E1, E2, and E3 (top). This enrichment is on average the read-length (&lt;em&gt;e.g.&lt;/em&gt;, 100&#xA0;&#xD7; for 100-bp reads). The plot of starts and finishes of alignments by position then yields a highly sensitive indicator of the segment boundaries.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F3.medium.gif" width="329" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/7/1899/F3.medium.gif" width="329" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468516" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Local alignment of genomic reads to transcripts gives a signature for junctions of discontiguous genomic encoding segments. (A) Tiled Illumina reads derived from the genome covering discontiguous segments E1, E2, and E3 (top) are aligned locally (not requiring alignment across the complete read) to the fused structure E1–E2–E3 (bottom). These reads have starting and finishing positions of alignment to the E1–E2–E3 structure (green and red dots). (B) The result for random starts and finishes of reads in the genome is a huge enrichment of start positions at the beginnings of the genomically discontiguous segments E1, E2, and E3, and for alignment finishes at the ends of E1, E2, and E3 (top). This enrichment is on average the read-length (<em>e.g.</em>, 100 × for 100-bp reads). The plot of starts and finishes of alignments by position then yields a highly sensitive indicator of the segment boundaries.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F4/graphic-4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Potential for RNA splicing to assemble consensus islands of genomic sequence into mRNA: the splicing test. (A) Tiled genomic reads (specific segments indicated by different colors) aligned to a Trinity object encoded by two discontiguous segments E1 and E2 yield island consensus sequences (&#x2018;Island 1&#x2019;, &#x2018;Island 2&#x2019;). Each island has a central segment identical to E1 or E2 and flanking sequences not found in the Trinity object. Note that when aligned within the Trinity &#x2018;frame&#x2019;, Islands 1 and 2 have no sequence in common, even though each aligns to a large segment of the Trinity object. (B) Consensus sequences for splicing in Chlamydomonas. Splice sites from the annotated Phytozome reference genome were collected and analyzed using the Weblogo algorithm (Crooks et al. 2004; http://weblogo.berkeley.edu). The human consensus is shown for comparison. (C) The splicing test. Consensus islands assembled as in (B) can be assembled into mRNA only by a join at a highly restricted location [in the absence of any terminal direct repeats, a single nucleotide at the terminus (start or finish) of each island]. This provides a highly constrained condition for searching for splice site homology (donor and acceptor positions; the most likely start and finish positions are indicated by green or red arrows). A series of consensus islands connected by high-confidence splices on the same strand forms a &#x2018;connected island&#x2019;; a candidate hidden exon should be contained within such a connected island to pass the splicing test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Potential for RNA splicing to assemble consensus islands of genomic sequence into mRNA: the splicing test. (A) Tiled genomic reads (specific segments indicated by different colors) aligned to a Trinity object encoded by two discontiguous segments E1 and E2 yield island consensus sequences (&#x2018;Island 1&#x2019;, &#x2018;Island 2&#x2019;). Each island has a central segment identical to E1 or E2 and flanking sequences not found in the Trinity object. Note that when aligned within the Trinity &#x2018;frame&#x2019;, Islands 1 and 2 have no sequence in common, even though each aligns to a large segment of the Trinity object. (B) Consensus sequences for splicing in &lt;em&gt;Chlamydomonas&lt;/em&gt;. Splice sites from the annotated Phytozome reference genome were collected and analyzed using the Weblogo algorithm (Crooks &lt;em&gt;et al.&lt;/em&gt; 2004; http://weblogo.berkeley.edu). The human consensus is shown for comparison. (C) The splicing test. Consensus islands assembled as in (B) can be assembled into mRNA only by a join at a highly restricted location [in the absence of any terminal direct repeats, a single nucleotide at the terminus (start or finish) of each island]. This provides a highly constrained condition for searching for splice site homology (donor and acceptor positions; the most likely start and finish positions are indicated by green or red arrows). A series of consensus islands connected by high-confidence splices on the same strand forms a &#x2018;connected island&#x2019;; a candidate hidden exon should be contained within such a connected island to pass the splicing test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F4/graphic-4.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/7/1899/F4/graphic-4.medium.gif" width="440" height="413"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F4/graphic-4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F4/graphic-4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468628" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F4/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Potential for RNA splicing to assemble consensus islands of genomic sequence into mRNA: the splicing test. (A) Tiled genomic reads (specific segments indicated by different colors) aligned to a Trinity object encoded by two discontiguous segments E1 and E2 yield island consensus sequences (&#x2018;Island 1&#x2019;, &#x2018;Island 2&#x2019;). Each island has a central segment identical to E1 or E2 and flanking sequences not found in the Trinity object. Note that when aligned within the Trinity &#x2018;frame&#x2019;, Islands 1 and 2 have no sequence in common, even though each aligns to a large segment of the Trinity object. (B) Consensus sequences for splicing in Chlamydomonas. Splice sites from the annotated Phytozome reference genome were collected and analyzed using the Weblogo algorithm (Crooks et al. 2004; http://weblogo.berkeley.edu). The human consensus is shown for comparison. (C) The splicing test. Consensus islands assembled as in (B) can be assembled into mRNA only by a join at a highly restricted location [in the absence of any terminal direct repeats, a single nucleotide at the terminus (start or finish) of each island]. This provides a highly constrained condition for searching for splice site homology (donor and acceptor positions; the most likely start and finish positions are indicated by green or red arrows). A series of consensus islands connected by high-confidence splices on the same strand forms a &#x2018;connected island&#x2019;; a candidate hidden exon should be contained within such a connected island to pass the splicing test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Potential for RNA splicing to assemble consensus islands of genomic sequence into mRNA: the splicing test. (A) Tiled genomic reads (specific segments indicated by different colors) aligned to a Trinity object encoded by two discontiguous segments E1 and E2 yield island consensus sequences (&#x2018;Island 1&#x2019;, &#x2018;Island 2&#x2019;). Each island has a central segment identical to E1 or E2 and flanking sequences not found in the Trinity object. Note that when aligned within the Trinity &#x2018;frame&#x2019;, Islands 1 and 2 have no sequence in common, even though each aligns to a large segment of the Trinity object. (B) Consensus sequences for splicing in &lt;em&gt;Chlamydomonas&lt;/em&gt;. Splice sites from the annotated Phytozome reference genome were collected and analyzed using the Weblogo algorithm (Crooks &lt;em&gt;et al.&lt;/em&gt; 2004; http://weblogo.berkeley.edu). The human consensus is shown for comparison. (C) The splicing test. Consensus islands assembled as in (B) can be assembled into mRNA only by a join at a highly restricted location [in the absence of any terminal direct repeats, a single nucleotide at the terminus (start or finish) of each island]. This provides a highly constrained condition for searching for splice site homology (donor and acceptor positions; the most likely start and finish positions are indicated by green or red arrows). A series of consensus islands connected by high-confidence splices on the same strand forms a &#x2018;connected island&#x2019;; a candidate hidden exon should be contained within such a connected island to pass the splicing test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F4/graphic-5.medium.gif" width="400" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/7/1899/F4/graphic-5.medium.gif" width="400" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F4/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F4/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468628" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Potential for RNA splicing to assemble consensus islands of genomic sequence into mRNA: the splicing test. (A) Tiled genomic reads (specific segments indicated by different colors) aligned to a Trinity object encoded by two discontiguous segments E1 and E2 yield island consensus sequences (‘Island 1’, ‘Island 2’). Each island has a central segment identical to E1 or E2 and flanking sequences not found in the Trinity object. Note that when aligned within the Trinity ‘frame’, Islands 1 and 2 have no sequence in common, even though each aligns to a large segment of the Trinity object. (B) Consensus sequences for splicing in <em>Chlamydomonas</em>. Splice sites from the annotated Phytozome reference genome were collected and analyzed using the Weblogo algorithm (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Crooks <em>et al.</em> 2004</a>; <a href="http://weblogo.berkeley.edu">http://weblogo.berkeley.edu</a>). The human consensus is shown for comparison. (C) The splicing test. Consensus islands assembled as in (B) can be assembled into mRNA only by a join at a highly restricted location [in the absence of any terminal direct repeats, a single nucleotide at the terminus (start or finish) of each island]. This provides a highly constrained condition for searching for splice site homology (donor and acceptor positions; the most likely start and finish positions are indicated by green or red arrows). A series of consensus islands connected by high-confidence splices on the same strand forms a ‘connected island’; a candidate hidden exon should be contained within such a connected island to pass the splicing test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Sequential islands of this kind have no aligned sequence at all comparing them to each other, although each has a central segment aligned to sequential segments of the Trinity object. Alignment discontinuities between these islands (the blue-yellow and orange-purple junctions in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>) mark an obligatory and exact position where the genomic segments must be joined in order to reconstitute the Trinity object. This restricts positions of potential RNA splicing to one nucleotide. (This is expanded to a few nucleotides in the case of direct repeats at the termini of the Trinity-homologous segments). Internal repeats will typically not yield a neighboring start–finish pair in close enough proximity to explain the join by splicing.</p><p id="p-28">Because the position of the join is highly constrained, we reasoned that if the segments were joined by RNA splicing, then we should be able to detect sequence signatures permissive for splicing specifically at these junctions. We examined genomic splice site sequences in the reference assembly/annotation to determine a splicing consensus (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, B and C</a>). Consistent with previous work in many eukaryotic organisms, the strongest determinant was the /GT…AG/ dinucleotide pair at intron termini; other flanking sequences also contributed significantly to the consensus, equivalent to almost eight nucleotides of specified sequence. We found that a match score of 11 bits (obtained by summing information content for observed nucleotide at each position) was attained by ∼99% of Phytozome annotated splice junctions, and by only ∼1% of randomly selected genomic sequences, and we chose this value as a cutoff for splice junction detection. Because of the near-absolute requirement for the /GT…(intron)…AG/ dinucleotides for splicing, we imposed a two-step test: presence of the dinucleotides, and a score of at least 11 (the dinucleotides alone give a score of 8). We applied this test to the discontinuities in alignment of genomic sequences to Trinity objects, as detailed above.</p><p id="p-29">The results of this procedure for four typical cases with a ‘bridged’ N-island (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1B</a>, bottom) are shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>. The successive red bars at top represent islands aligned to Trinity; the black lines at left and right of these islands represent non-Trinity-homologous sequence. The blue arrows are high-confidence splice junctions according to the test described above. In the four typical cases in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>, the algorithm has assembled multiple candidate exons. In all cases. these exons could be joined by high-confidence splices; thus, these sets of sequences form ‘connected islands’ that could be assembled into a single mature transcript by splicing. Additional Matlab-generated diagrams, and the code used for their automated generation, are available in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS1.pdf">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS2.zip">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS3.zip">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS4.zip">File S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS5.zip">File S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/FileS6.zip">File S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/TableS2.xlsx">Table S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/TableS3.xlsx">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/TableS4.xlsx">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/TableS5.xlsx">Table S5</a>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F5/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical summary of analysis of Trinity objects. The &#x2018;frame&#x2019; for all data are the sequence of the Trinity object (numbers on x-axis of top graph). At top is the result of analysis of genomic reads aligned to the Trinity object. (A) The red/black bars are alignment islands, where red indicates identical sequence to Trinity and black non-identical sequence. The blue joining arrows indicate high-scoring splice junctions (if detected, junctions on the opposite strand are in yellow). Below is a quantitative trace of splice junction locations and scores (standardized to the maximum possible score of &#x223C;15; the cutoff value for acceptance of 11 is illustrated by the red line). The blue lines are splices in our model; green lines below are splices from the reference transcript model aligned to the Trinity object. Red lines indicate splice junctions on the opposite strand in the Trinity sequence. The green bars indicate regions of alignment (HSPs) to the specified Phytozome reference transcript. The black triangle indicates that the gap in the Blastn alignment to the Phytozome transcript model is the approximate location of putatively spliced-out N-islands in the reference. The bottom graph indicates the positions of Blastx alignment to the Volvox proteome. (A) Typical cases for &#x2018;bridging&#x2019; Trinity objects. In these four cases the entire set of consensus islands are joined into one &#x2018;connected island&#x2019; by same-sense splices. (B) Rare aberrant cases where the Trinity object is due to &#x2018;track-crossing&#x2019; in silico, and contains regions of two distinct reference transcripts (top two examples), or where splice detection fails even on the same reference transcript (bottom two examples). In these cases there are multiple connected islands (gray outlines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Graphical summary of analysis of Trinity objects. The &#x2018;frame&#x2019; for all data are the sequence of the Trinity object (numbers on &lt;em&gt;x&lt;/em&gt;-axis of top graph). At top is the result of analysis of genomic reads aligned to the Trinity object. (A) The red/black bars are alignment islands, where red indicates identical sequence to Trinity and black non-identical sequence. The blue joining arrows indicate high-scoring splice junctions (if detected, junctions on the opposite strand are in yellow). Below is a quantitative trace of splice junction locations and scores (standardized to the maximum possible score of &#x223C;15; the cutoff value for acceptance of 11 is illustrated by the red line). The blue lines are splices in our model; green lines below are splices from the reference transcript model aligned to the Trinity object. Red lines indicate splice junctions on the opposite strand in the Trinity sequence. The green bars indicate regions of alignment (HSPs) to the specified Phytozome reference transcript. The black triangle indicates that the gap in the Blastn alignment to the Phytozome transcript model is the approximate location of putatively spliced-out N-islands in the reference. The bottom graph indicates the positions of Blastx alignment to the &lt;em&gt;Volvox&lt;/em&gt; proteome. (A) Typical cases for &#x2018;bridging&#x2019; Trinity objects. In these four cases the entire set of consensus islands are joined into one &#x2018;connected island&#x2019; by same-sense splices. (B) Rare aberrant cases where the Trinity object is due to &#x2018;track-crossing&#x2019; &lt;em&gt;in silico&lt;/em&gt;, and contains regions of two distinct reference transcripts (top two examples), or where splice detection fails even on the same reference transcript (bottom two examples). In these cases there are multiple connected islands (gray outlines).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F5/graphic-6.medium.gif" width="440" height="339"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/7/1899/F5/graphic-6.medium.gif" width="440" height="339"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F5/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F5/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468643" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F5/graphic-7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical summary of analysis of Trinity objects. The &#x2018;frame&#x2019; for all data are the sequence of the Trinity object (numbers on x-axis of top graph). At top is the result of analysis of genomic reads aligned to the Trinity object. (A) The red/black bars are alignment islands, where red indicates identical sequence to Trinity and black non-identical sequence. The blue joining arrows indicate high-scoring splice junctions (if detected, junctions on the opposite strand are in yellow). Below is a quantitative trace of splice junction locations and scores (standardized to the maximum possible score of &#x223C;15; the cutoff value for acceptance of 11 is illustrated by the red line). The blue lines are splices in our model; green lines below are splices from the reference transcript model aligned to the Trinity object. Red lines indicate splice junctions on the opposite strand in the Trinity sequence. The green bars indicate regions of alignment (HSPs) to the specified Phytozome reference transcript. The black triangle indicates that the gap in the Blastn alignment to the Phytozome transcript model is the approximate location of putatively spliced-out N-islands in the reference. The bottom graph indicates the positions of Blastx alignment to the Volvox proteome. (A) Typical cases for &#x2018;bridging&#x2019; Trinity objects. In these four cases the entire set of consensus islands are joined into one &#x2018;connected island&#x2019; by same-sense splices. (B) Rare aberrant cases where the Trinity object is due to &#x2018;track-crossing&#x2019; in silico, and contains regions of two distinct reference transcripts (top two examples), or where splice detection fails even on the same reference transcript (bottom two examples). In these cases there are multiple connected islands (gray outlines)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Graphical summary of analysis of Trinity objects. The &#x2018;frame&#x2019; for all data are the sequence of the Trinity object (numbers on &lt;em&gt;x&lt;/em&gt;-axis of top graph). At top is the result of analysis of genomic reads aligned to the Trinity object. (A) The red/black bars are alignment islands, where red indicates identical sequence to Trinity and black non-identical sequence. The blue joining arrows indicate high-scoring splice junctions (if detected, junctions on the opposite strand are in yellow). Below is a quantitative trace of splice junction locations and scores (standardized to the maximum possible score of &#x223C;15; the cutoff value for acceptance of 11 is illustrated by the red line). The blue lines are splices in our model; green lines below are splices from the reference transcript model aligned to the Trinity object. Red lines indicate splice junctions on the opposite strand in the Trinity sequence. The green bars indicate regions of alignment (HSPs) to the specified Phytozome reference transcript. The black triangle indicates that the gap in the Blastn alignment to the Phytozome transcript model is the approximate location of putatively spliced-out N-islands in the reference. The bottom graph indicates the positions of Blastx alignment to the &lt;em&gt;Volvox&lt;/em&gt; proteome. (A) Typical cases for &#x2018;bridging&#x2019; Trinity objects. In these four cases the entire set of consensus islands are joined into one &#x2018;connected island&#x2019; by same-sense splices. (B) Rare aberrant cases where the Trinity object is due to &#x2018;track-crossing&#x2019; &lt;em&gt;in silico&lt;/em&gt;, and contains regions of two distinct reference transcripts (top two examples), or where splice detection fails even on the same reference transcript (bottom two examples). In these cases there are multiple connected islands (gray outlines).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F5/graphic-7.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/7/1899/F5/graphic-7.medium.gif" width="440" height="331"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F5/graphic-7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F5/graphic-7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468643" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-30" class="first-child">Graphical summary of analysis of Trinity objects. The ‘frame’ for all data are the sequence of the Trinity object (numbers on <em>x</em>-axis of top graph). At top is the result of analysis of genomic reads aligned to the Trinity object. (A) The red/black bars are alignment islands, where red indicates identical sequence to Trinity and black non-identical sequence. The blue joining arrows indicate high-scoring splice junctions (if detected, junctions on the opposite strand are in yellow). Below is a quantitative trace of splice junction locations and scores (standardized to the maximum possible score of ∼15; the cutoff value for acceptance of 11 is illustrated by the red line). The blue lines are splices in our model; green lines below are splices from the reference transcript model aligned to the Trinity object. Red lines indicate splice junctions on the opposite strand in the Trinity sequence. The green bars indicate regions of alignment (HSPs) to the specified Phytozome reference transcript. The black triangle indicates that the gap in the Blastn alignment to the Phytozome transcript model is the approximate location of putatively spliced-out N-islands in the reference. The bottom graph indicates the positions of Blastx alignment to the <em>Volvox</em> proteome. (A) Typical cases for ‘bridging’ Trinity objects. In these four cases the entire set of consensus islands are joined into one ‘connected island’ by same-sense splices. (B) Rare aberrant cases where the Trinity object is due to ‘track-crossing’ <em>in silico</em>, and contains regions of two distinct reference transcripts (top two examples), or where splice detection fails even on the same reference transcript (bottom two examples). In these cases there are multiple connected islands (gray outlines).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Also indicated are positions of splice sites in the aligned reference annotation compared to those detected in the island termini. The splice junctions are almost always are on the same strand; one case where a splice junction could be called on both strands can be noted in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>, top right (yellow arrow).</p><p id="p-32">The exact lineup of almost all computed and reference splice junctions constitutes strong evidence for accuracy of our procedure, and supports these intron locations in the reference annotation. The exceptions to this rule are essentially all detected at the position of N-islands thought to be spliced out according to the reference, that are in ∼25% of cases associated with additional exons in this analysis. Importantly, almost all of the nucleotides contributing to the splice site consensus are intronic (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>), and thus not present in the Trinity-assembled sequence. Finding these consensus sequences in the genomic tails at the junction is thus entirely independent validation.</p><p id="p-33">Examples of the main types of occasional aberrant patterns are shown in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>. The boxes delineate ‘connected islands’ defined by splice junctions on the same strand. In the top left and top right of the figure, Trinity assembly errors in which two transcripts were fused computationally into single Trinity objects are evident. Note that, in the case at top right, the fusion switched the strand of the sequence, as indicated by splice junction orientation (blue to yellow). The bottom two cases are more likely to be simple failure of detection of splice junctions by the algorithm.</p></div><div id="sec-9" class="subsection"><h3>Case studies: PF20 and CDC27</h3><p id="p-34">We will discuss two cases of exon discovery in detail. <em>PF20</em> is required for flagellar function; its complete cDNA sequence encoding 606 amino acids was described previously (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Smith and Lefebvre 1997</a>), but an intronic N-island in the annotated reference genome (between exon 8 and 9 in the Phytozome gene model Cre04.g227900) eliminated coding sequence for amino acids 357–488.</p><p id="p-35"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a> diagrams our results for <em>PF20</em>, which replace the N-island with four exons. Inclusion of these exons in the predicted mRNA results in a complete coding sequence exactly matching the results of <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Smith and Lefebvre (1997</a>). The bottom dark green bar in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a> indicates regions of homology to <em>Volvox</em> PF20. The bridging sequence is required for optimal alignment to <em>Volvox</em>. This result, recapitulating a careful single-gene study, was generated by unsupervised computations, with the only input being Illumina libraries from bulk cDNA and genomic DNA.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="N-islands can contain hidden exons. Two examples are shown. (A) PF20; (B, C) CDC27. (A, B) Diagrams as in Figure 5. In both cases, one N-island contains hidden exons (four in PF20, two in CDC27). The second N-island in CDC27 is covered; the diagram shows it to be authentically intronic according to the Trinity object analysis. As in Figure 5, black bars indicate the approximate position of the intronic N-island in the Phytozome gene model. Multiple sequence alignments between Trinity and Phytozome peptide sequences from Chlamydomonas and Volvox were generated by clustalo (Sievers et al. 2011). (C) The analysis recovers splice-site-proximal intronic sequences; illustrated for CDC27. The islands from genomic reads, including Trinity nonhomologous sequence, align perfectly with reference genomic sequence for individual exons (green bars), and surrounding intronic sequence (green lines). Similar extensions (red lines) beyond the Trinity-aligned segments (red bars) are detected for the two hidden exons &#x2018;E-X&#x2019; and &#x2018;E-Y&#x2019;. (D) These extensions are characterized by strong terminal splice site homology, which we take as evidence that &#x2018;exons X and Y&#x2019; are in fact spliced into the mature CDC27 mRNA, as reflected in the Trinity object comp4349-c0-seq1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;N-islands can contain hidden exons. Two examples are shown. (A) &lt;em&gt;PF20&lt;/em&gt;; (B, C) &lt;em&gt;CDC27&lt;/em&gt;. (A, B) Diagrams as in Figure 5. In both cases, one N-island contains hidden exons (four in &lt;em&gt;PF20&lt;/em&gt;, two in &lt;em&gt;CDC27&lt;/em&gt;). The second N-island in &lt;em&gt;CDC27&lt;/em&gt; is covered; the diagram shows it to be authentically intronic according to the Trinity object analysis. As in Figure 5, black bars indicate the approximate position of the intronic N-island in the Phytozome gene model. Multiple sequence alignments between Trinity and Phytozome peptide sequences from &lt;em&gt;Chlamydomonas&lt;/em&gt; and &lt;em&gt;Volvox&lt;/em&gt; were generated by clustalo (Sievers &lt;em&gt;et al.&lt;/em&gt; 2011). (C) The analysis recovers splice-site-proximal intronic sequences; illustrated for &lt;em&gt;CDC27&lt;/em&gt;. The islands from genomic reads, including Trinity nonhomologous sequence, align perfectly with reference genomic sequence for individual exons (green bars), and surrounding intronic sequence (green lines). Similar extensions (red lines) beyond the Trinity-aligned segments (red bars) are detected for the two hidden exons &#x2018;E-X&#x2019; and &#x2018;E-Y&#x2019;. (D) These extensions are characterized by strong terminal splice site homology, which we take as evidence that &#x2018;exons X and Y&#x2019; are in fact spliced into the mature &lt;em&gt;CDC27&lt;/em&gt; mRNA, as reflected in the Trinity object comp4349-c0-seq1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-8.medium.gif" width="440" height="368"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-8.medium.gif" width="440" height="368"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468531" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="N-islands can contain hidden exons. Two examples are shown. (A) PF20; (B, C) CDC27. (A, B) Diagrams as in Figure 5. In both cases, one N-island contains hidden exons (four in PF20, two in CDC27). The second N-island in CDC27 is covered; the diagram shows it to be authentically intronic according to the Trinity object analysis. As in Figure 5, black bars indicate the approximate position of the intronic N-island in the Phytozome gene model. Multiple sequence alignments between Trinity and Phytozome peptide sequences from Chlamydomonas and Volvox were generated by clustalo (Sievers et al. 2011). (C) The analysis recovers splice-site-proximal intronic sequences; illustrated for CDC27. The islands from genomic reads, including Trinity nonhomologous sequence, align perfectly with reference genomic sequence for individual exons (green bars), and surrounding intronic sequence (green lines). Similar extensions (red lines) beyond the Trinity-aligned segments (red bars) are detected for the two hidden exons &#x2018;E-X&#x2019; and &#x2018;E-Y&#x2019;. (D) These extensions are characterized by strong terminal splice site homology, which we take as evidence that &#x2018;exons X and Y&#x2019; are in fact spliced into the mature CDC27 mRNA, as reflected in the Trinity object comp4349-c0-seq1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;N-islands can contain hidden exons. Two examples are shown. (A) &lt;em&gt;PF20&lt;/em&gt;; (B, C) &lt;em&gt;CDC27&lt;/em&gt;. (A, B) Diagrams as in Figure 5. In both cases, one N-island contains hidden exons (four in &lt;em&gt;PF20&lt;/em&gt;, two in &lt;em&gt;CDC27&lt;/em&gt;). The second N-island in &lt;em&gt;CDC27&lt;/em&gt; is covered; the diagram shows it to be authentically intronic according to the Trinity object analysis. As in Figure 5, black bars indicate the approximate position of the intronic N-island in the Phytozome gene model. Multiple sequence alignments between Trinity and Phytozome peptide sequences from &lt;em&gt;Chlamydomonas&lt;/em&gt; and &lt;em&gt;Volvox&lt;/em&gt; were generated by clustalo (Sievers &lt;em&gt;et al.&lt;/em&gt; 2011). (C) The analysis recovers splice-site-proximal intronic sequences; illustrated for &lt;em&gt;CDC27&lt;/em&gt;. The islands from genomic reads, including Trinity nonhomologous sequence, align perfectly with reference genomic sequence for individual exons (green bars), and surrounding intronic sequence (green lines). Similar extensions (red lines) beyond the Trinity-aligned segments (red bars) are detected for the two hidden exons &#x2018;E-X&#x2019; and &#x2018;E-Y&#x2019;. (D) These extensions are characterized by strong terminal splice site homology, which we take as evidence that &#x2018;exons X and Y&#x2019; are in fact spliced into the mature &lt;em&gt;CDC27&lt;/em&gt; mRNA, as reflected in the Trinity object comp4349-c0-seq1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-9.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-9.medium.gif" width="440" height="308"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468531" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="N-islands can contain hidden exons. Two examples are shown. (A) PF20; (B, C) CDC27. (A, B) Diagrams as in Figure 5. In both cases, one N-island contains hidden exons (four in PF20, two in CDC27). The second N-island in CDC27 is covered; the diagram shows it to be authentically intronic according to the Trinity object analysis. As in Figure 5, black bars indicate the approximate position of the intronic N-island in the Phytozome gene model. Multiple sequence alignments between Trinity and Phytozome peptide sequences from Chlamydomonas and Volvox were generated by clustalo (Sievers et al. 2011). (C) The analysis recovers splice-site-proximal intronic sequences; illustrated for CDC27. The islands from genomic reads, including Trinity nonhomologous sequence, align perfectly with reference genomic sequence for individual exons (green bars), and surrounding intronic sequence (green lines). Similar extensions (red lines) beyond the Trinity-aligned segments (red bars) are detected for the two hidden exons &#x2018;E-X&#x2019; and &#x2018;E-Y&#x2019;. (D) These extensions are characterized by strong terminal splice site homology, which we take as evidence that &#x2018;exons X and Y&#x2019; are in fact spliced into the mature CDC27 mRNA, as reflected in the Trinity object comp4349-c0-seq1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;N-islands can contain hidden exons. Two examples are shown. (A) &lt;em&gt;PF20&lt;/em&gt;; (B, C) &lt;em&gt;CDC27&lt;/em&gt;. (A, B) Diagrams as in Figure 5. In both cases, one N-island contains hidden exons (four in &lt;em&gt;PF20&lt;/em&gt;, two in &lt;em&gt;CDC27&lt;/em&gt;). The second N-island in &lt;em&gt;CDC27&lt;/em&gt; is covered; the diagram shows it to be authentically intronic according to the Trinity object analysis. As in Figure 5, black bars indicate the approximate position of the intronic N-island in the Phytozome gene model. Multiple sequence alignments between Trinity and Phytozome peptide sequences from &lt;em&gt;Chlamydomonas&lt;/em&gt; and &lt;em&gt;Volvox&lt;/em&gt; were generated by clustalo (Sievers &lt;em&gt;et al.&lt;/em&gt; 2011). (C) The analysis recovers splice-site-proximal intronic sequences; illustrated for &lt;em&gt;CDC27&lt;/em&gt;. The islands from genomic reads, including Trinity nonhomologous sequence, align perfectly with reference genomic sequence for individual exons (green bars), and surrounding intronic sequence (green lines). Similar extensions (red lines) beyond the Trinity-aligned segments (red bars) are detected for the two hidden exons &#x2018;E-X&#x2019; and &#x2018;E-Y&#x2019;. (D) These extensions are characterized by strong terminal splice site homology, which we take as evidence that &#x2018;exons X and Y&#x2019; are in fact spliced into the mature &lt;em&gt;CDC27&lt;/em&gt; mRNA, as reflected in the Trinity object comp4349-c0-seq1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-10.medium.gif" width="440" height="366"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-10.medium.gif" width="440" height="366"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F6/graphic-10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468531" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-36" class="first-child">N-islands can contain hidden exons. Two examples are shown. (A) <em>PF20</em>; (B, C) <em>CDC27</em>. (A, B) Diagrams as in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>. In both cases, one N-island contains hidden exons (four in <em>PF20</em>, two in <em>CDC27</em>). The second N-island in <em>CDC27</em> is covered; the diagram shows it to be authentically intronic according to the Trinity object analysis. As in <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>, black bars indicate the approximate position of the intronic N-island in the Phytozome gene model. Multiple sequence alignments between Trinity and Phytozome peptide sequences from <em>Chlamydomonas</em> and <em>Volvox</em> were generated by clustalo (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Sievers <em>et al.</em> 2011</a>). (C) The analysis recovers splice-site-proximal intronic sequences; illustrated for <em>CDC27</em>. The islands from genomic reads, including Trinity nonhomologous sequence, align perfectly with reference genomic sequence for individual exons (green bars), and surrounding intronic sequence (green lines). Similar extensions (red lines) beyond the Trinity-aligned segments (red bars) are detected for the two hidden exons ‘E-X’ and ‘E-Y’. (D) These extensions are characterized by strong terminal splice site homology, which we take as evidence that ‘exons X and Y’ are in fact spliced into the mature <em>CDC27</em> mRNA, as reflected in the Trinity object comp4349-c0-seq1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">CDC27 (<em>Chlamydomonas</em> Cre17.g740510) is a conserved component of the anaphase-promoting complex (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Zachariae and Nasmyth 1999</a>). The Cre17.g740510 gene model has two intronic N-islands, between exons 11 and 12 (246 N) and between exons 12 and 13 (100 N). We identified a Trinity object that covered exons 9–17 of the <em>CDC27</em> transcript model (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). Two separated HSPs surround the first N-island, with extra sequence between the HSPs in the Trinity object [the signature for possible hidden exons (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>)]. The first HSP aligns to reference exons 9–11, and the second HSP covers reference exons 12–17, including the 3′UTR. The intervening “missing sequence” between exons 11 and 12 is accounted for by splicing in two new exons ‘X’ and ‘Y’ from a previously unknown genomic segment; the extra peptide sequence thereby added to CDC27 is supported by <em>Volvox</em> homology (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). Indeed, this extra peptide sequence is conserved in CDC27 from a wide range of eukaryotic species including <em>Arabidopsis</em>, animals, and yeast. In contrast, the second annotated intronic N-island (triangle in <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6B</a>) is ‘covered’, in agreement that this N-island is completely contained within an intron (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6B</a>).</p><p id="p-38">The Trinity-nonhomologous ‘tails’ at 5′ and 3′ ends of the aligned islands are likely to be intronic. Indeed, these sequences for ‘covered’ introns exactly matched reference splice donor and acceptor, and neighboring intronic sequences, at each covered splice junction (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6D</a>); thus, splice junction-proximal intronic sequences are accurately recovered by local alignment of genomic reads to Trinity objects, independent of any prior reference or annotation.</p></div><div id="sec-10" class="subsection"><h3>Exonic N-islands</h3><p id="p-39">Most N-islands are annotated as intronic. There are, however, 40 exonic N-islands in the reference assembly. Most are assigned to 3′UTR or 5′UTR. We found three cases where a Trinity object bridged an exonic N-island (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/TableS3.xlsx">Table S3</a>). In two of these the bridge resulted in a significant coding sequence improvement in Blastx score against <em>Volvox</em>.</p></div><div id="sec-11" class="subsection"><h3>Flanking N-islands</h3><p id="p-40">The intronic and exonic N-islands examined above were all annotated within the boundaries of a gene. Some N-islands flank an annotated gene; such a gene might actually extend into (or even beyond) the flanking N-island. We defined a flanking N-island as an N-island starting within 100 bp of the 5′ or 3′ borders of a gene model. If the authentic gene extends into the N-island, then there could be a ‘half-bridging’ Trinity object aligning at one end to the reference gene model, with the other end ‘hanging’ in the N-island. We scored 136 flanking N-islands in the genome, 11 of which could be composed as a connected island extending into the N-island, significantly increasing the <em>Volvox</em> BLAST score (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029207/-/DC1/TableS5.xlsx">Table S5</a>). Two examples are presented in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>. Because the Trinity objects are short, it is unlikely that these extensions complete the transcript model, and the lack of a full bridge intrinsically makes these hits less reliable, but it is clear that extension of these methods could significantly extend or even complete gene models by foraging into the adjacent N-islands. We suspect that the low yield of these structures is related to the sequencing difficulties (simple sequence, repeat regions, and inverted repeats) that led to them being called as N-islands in the original annotation.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flanking N islands can truncate a gene model. Diagrams as in Figure 5. Two Phytozome transcript models with flanking N islands are shown, Cre06.g293582.t1.1 and Cre09.g408851.t1.1 (light green bars). The Trinity objects comp2196_c0_seq1 and comp3488_c0_seq1, respectively, align to the body to the Phytozome transcripts, and extend into the N islands (represented by black bars). The Trinity sequence in the extensions (unaligned to Chlamydomonas sequence) can be aligned by Blastx to Volvox proteins (dark green bars)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-821015666" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Flanking N islands can truncate a gene model. Diagrams as in Figure 5. Two Phytozome transcript models with flanking N islands are shown, Cre06.g293582.t1.1 and Cre09.g408851.t1.1 (light green bars). The Trinity objects comp2196_c0_seq1 and comp3488_c0_seq1, respectively, align to the body to the Phytozome transcripts, and extend into the N islands (represented by black bars). The Trinity sequence in the extensions (unaligned to &lt;em&gt;Chlamydomonas&lt;/em&gt; sequence) can be aligned by Blastx to &lt;em&gt;Volvox&lt;/em&gt; proteins (dark green bars).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1899/F7.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/7/1899/F7.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1899/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468740" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-41" class="first-child">Flanking N islands can truncate a gene model. Diagrams as in <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>. Two Phytozome transcript models with flanking N islands are shown, Cre06.g293582.t1.1 and Cre09.g408851.t1.1 (light green bars). The Trinity objects comp2196_c0_seq1 and comp3488_c0_seq1, respectively, align to the body to the Phytozome transcripts, and extend into the N islands (represented by black bars). The Trinity sequence in the extensions (unaligned to <em>Chlamydomonas</em> sequence) can be aligned by Blastx to <em>Volvox</em> proteins (dark green bars).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-42">N-islands are regions of undetermined sequence whose lengths are only estimated. Many N-islands are assigned in the reference to intergenic or intronic regions, and such assignments are problematic (since, for example, intronic N-islands could, in principle, contain any number of unknown exons, or even complete unknown genes). Our results suggest additional internal coding exons, or N- or C-terminal extension of coding sequence, to replace the putatively noncoding N-islands, in a significant minority of N-islands.</p><p id="p-43">We did not attempt to validate the new exons by additional direct experimentation; possibilities would include longer RNAseq reads, paired-end, or strand-specific RNAseq, and different sequencing technologies allowing much longer genomic DNA reads. However, the BLASTX improvement assay, and the finding that high-confidence splice junctions could be formed at almost all the relevant junctions, constitute two entirely independent ways to validate the results, since neither of these results is predicted if the new candidate exons are not spliced and translated as predicted.</p><p id="p-44">The methods used here are useful for ‘proofreading’ a reference assembly/annotations, but the methods do not require a prior reference. In principle, these methods could validate RNAseq-based transcriptome assemblies, including providing genomic splice sites and flanking intronic sequences, even for organisms without any assembled genome or annotation.</p><p id="p-45">While our computations <em>per se</em> did not use the assembled genome or reference annotation, the results could be evaluated with much higher confidence when looking for a revision of the current assembly, rather than entirely <em>de novo</em> sequence. Thus, we have more confidence in the bridged than the half-bridged cases (such as with the flanking N-islands), and we did not so far attempt to evaluate Trinity objects that were completely unalignable to any reference transcript. One limitation to our approach, as noted above, was the use of single-end, nonstrand-specific 50-nt RNAseq reads. More informative libraries might improve resolution and reduce track-crossing sufficiently that a reliable full transcriptome, along with junctional sequences at promoter, termination and splice sites, might be recovered <em>de novo</em>. The <em>Chlamydomonas</em> genome is a favorable case in that it is 1.2 × 10<sup>8</sup> bp, of which around one-third is exonic. It is unfavorable in having a very high density of repeats and low-complexity sequence (likely the source of most alignment difficulties including N-islands, and certainly the source of most difficulties we encountered with our approach).</p><p id="p-46">In principle, the genomic islands recovered to account for Trinity objects could be used iteratively to align genomic reads, ultimately finding the next exon and providing sequence of the entire intron. We have not attempted this because of the clear danger of track-crossing in repeat sequences, which are especially enriched in intronic sequence, but it is likely possible in at least some cases.</p><p id="p-47">The test for splice sites at the borders of the Trinity-nucleated genomic islands also tests the validity of the Trinity object, since it is a sensitive indicator of track-crossing leading to fusion transcripts (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5B</a>). This could be a useful proofreading step when Trinity-generated transcriptomes are used in the absence of a reference genome sequence, since it requires only a single library of genomic reads aligned to the Trinity objects, and requires no prior assembly of a reference genome.</p><p id="p-48">The combination of <em>de novo</em> transcriptome assembly followed by alignment of short genomic reads onto the assembled transcriptome objects in principle represents a massively parallel approach to validating a genome assembly/annotation, equivalent in some ways to checking a near-complete collection of ESTs against the annotation in a single experiment. Our implementation was not quite up to this challenge, because the assembly was only ∼70% complete, and a significant minority of the assembled objects had track-crossing and other issues (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5B</a>). However, these seem likely to be solvable issues, through the use of better RNAseq libraries, and through making use of paired-end information for both RNAseq and DNA libraries.</p><p id="p-49">While our approach did uncover some novel exons missed in the prior assembly/annotation, overall our results document the very high quality of the annotation. A majority of N-islands annotated as intronic most likely are purely intronic, and a large majority of annotated splice junctions lacking N-islands are confirmed: they are covered by continuous Trinity object sequence, and put together from discontiguous genomic segments with high-confidence splices.</p><p id="p-50">The current assembly/annotation reports ∼4 Mb of Ns (3.4% of the genome), distributed in 1489 islands of highly variable size; 64% of these islands are located in introns, and these are primarily the N-islands addressed by the current study. However, these intronic N-islands are short (only 15% &gt; 1 kb, 2% &gt; 10 kb), and account for only 22% of the total Ns; most Ns are found in large islands (45% &gt; 1 kb, 25% &gt; 10 kb) annotated as intergenic. It is likely that at least some of these large regions contain unknown transcripts, which should be represented in our study by Trinity objects that did not align to any current Phytozome transcript model. We have not examined this question in detail, but it is clear that few such objects exist in our dataset, and could account for no more than a very small proportion of the 3.1 Mb of Ns in intergenic islands. Therefore, while we cannot make a strong conclusion on this point, it seems at least plausible that most of the large intergenic N-islands are composed of noncoding sequence; perhaps sequence characteristics leading experimentally to N-island status are relatively rare in sequences transcribed into mRNA. Until the sequences are identified, this remains a speculation.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">Funding support was from Public Health Service grant 5RO1-GM078153.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-52">Supplemental material is available online at <a href="http://www.g3journal.org/content/early/2016/05/19/g3.116.029207/suppl/DC1">http://www.g3journal.org/content/early/2016/05/19/g3.116.029207/suppl/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-53"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-03-15"><span class="received-label">Received </span>March 15, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-04-23"><span class="accepted-label">Accepted </span>April 23, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Tulin, Cross</li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blaby</span>  <span class="cit-name-given-names">I. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaby-Haas</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tourasse</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hom</span>  <span class="cit-name-given-names">E. F. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The <em>Chlamydomonas</em> genome project: a decade on.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">672</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D672%26rft_id%253Dinfo%253Apmid%252F24950814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24950814&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.3" data-doi="10.1101/gr.849004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crooks</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chandonia</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">WebLogo: a sequence logo generator.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1188</span>–<span class="cit-lpage">1190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DCrooks%26rft.auinit1%253DG.%2BE.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1190%26rft.atitle%253DWebLogo%253A%2BA%2BSequence%2BLogo%2BGenerator%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.849004%26rft_id%253Dinfo%253Apmid%252F15173120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE0LzYvMTE4OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Tying down loose ends in the <em>Chlamydomonas</em> genome: functional significance of abundant upstream open reading frames.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.5" data-doi="10.1093/bioinformatics/btq461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Search and clustering orders of magnitude faster than BLAST.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2460</span>–<span class="cit-lpage">2461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq461%26rft_id%253Dinfo%253Apmid%252F20709691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTkvMjQ2MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.6" data-doi="10.1105/tpc.15.00492"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flowers</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazzouri</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosas</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahmani</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Whole-genome resequencing reveals extensive natural variation in the model green alga <em>Chlamydomonas reinhardtii</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2353</span>–<span class="cit-lpage">2369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00492%26rft_id%253Dinfo%253Apmid%252F26392080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI3LzkvMjM1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.7" data-doi="10.1105/tpc.15.00508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallaher</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitz-Gibbon</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaesener</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellegrini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merchant</span>  <span class="cit-name-given-names">S. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title"><em>Chlamydomonas</em> genome resource for laboratory strains reveals a mosaic of sequence variation, identifies true strain histories, and enables strain-specific studies.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2335</span>–<span class="cit-lpage">2352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00508%26rft_id%253Dinfo%253Apmid%252F26307380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI3LzkvMjMzNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODk5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.8" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.9" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papanicolaou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blood</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>De novo</em> transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.10" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.11" data-doi="10.1126/science.1143609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merchant</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prochnik</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallon</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpowicz</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The <em>Chlamydomonas</em> genome reveals the evolution of key animal and plant functions.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">318</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMerchant%26rft.auinit1%253DS.%2BS.%26rft.volume%253D318%26rft.issue%253D5848%26rft.spage%253D245%26rft.epage%253D250%26rft.atitle%253DThe%2BChlamydomonas%2BGenome%2BReveals%2Bthe%2BEvolution%2Bof%2BKey%2BAnimal%2Band%2BPlant%2BFunctions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1143609%26rft_id%253Dinfo%253Apmid%252F17932292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTgvNTg0OC8yNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.12" data-doi="10.1126/science.1188800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prochnik</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Umen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nedelcu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic analysis of organismal complexity in the multicellular green alga <em>Volvox carteri</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DProchnik%26rft.auinit1%253DS.%2BE.%26rft.volume%253D329%26rft.issue%253D5988%26rft.spage%253D223%26rft.epage%253D226%26rft.atitle%253DGenomic%2BAnalysis%2Bof%2BOrganismal%2BComplexity%2Bin%2Bthe%2BMulticellular%2BGreen%2BAlga%2BVolvox%2Bcarteri%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188800%26rft_id%253Dinfo%253Apmid%252F20616280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk4OC8yMjMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.13" data-doi="10.1038/msb.2011.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sievers</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilm</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dineen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karplus</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft.volume%253D7%26rft.spage%253D539%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.75%26rft_id%253Dinfo%253Apmid%252F21988835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2011.75&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21988835&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1899.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.14" data-doi="10.1091/mbc.8.3.455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lefebvre</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">PF20 gene product contains WD repeats and localizes to the intermicrotubule bridges in <em>Chlamydomonas</em> flagella.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DSmith%26rft.auinit1%253DE.%2BF.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D455%26rft.epage%253D467%26rft.atitle%253DPF20%2Bgene%2Bproduct%2Bcontains%2BWD%2Brepeats%2Band%2Blocalizes%2Bto%2Bthe%2Bintermicrotubule%2Bbridges%2Bin%2BChlamydomonas%2Bflagella.%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.8.3.455%26rft_id%253Dinfo%253Apmid%252F9188098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjc6IjgvMy80NTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.15" data-doi="10.1105/tpc.114.129312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tulin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A microbial avenue to cell cycle control in the plant superkingdom.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4019</span>–<span class="cit-lpage">4038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.129312%26rft_id%253Dinfo%253Apmid%252F25336509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMC80MDE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.16" data-doi="10.1105/tpc.15.00400"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tulin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cross</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Cyclin-dependent kinase regulation of diurnal transcription in <em>Chlamydomonas</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2727</span>–<span class="cit-lpage">2742</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.15.00400%26rft_id%253Dinfo%253Apmid%252F26475866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xMC8yNzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4OTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1899.17" data-doi="10.1101/gad.13.16.2039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zachariae</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Whose end is destruction: cell division and the anaphase-promoting complex.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2039</span>–<span class="cit-lpage">2058</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DZachariae%26rft.auinit1%253DW.%26rft.volume%253D13%26rft.issue%253D16%26rft.spage%253D2039%26rft.epage%253D2058%26rft.atitle%253DWhose%2Bend%2Bis%2Bdestruction%253A%2Bcell%2Bdivision%2Band%2Bthe%2Banaphase-promoting%2Bcomplex%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.13.16.2039%26rft_id%253Dinfo%253Apmid%252F10465783%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEzLzE2LzIwMzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg5OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/7/1899.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1891" title="Cytotype Regulation Facilitates Repression of Hybrid Dysgenesis by Naturally Occurring KP Elements in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1911" title="Placental Hypomethylation Is More Pronounced in Genomic Loci Devoid of Retroelements" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/7"><span>Volume 6 Issue 7, July 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/7.cover-source.jpg?itok=E84OOyGI" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1899.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LnAUuqmapBUp7gpLi--PLsHZoTOm28wPq9fy_fVmqIs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/7/1899" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/7/1899" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/7/1899" class="active" data-icon-position="" data-hide-link-title="0">Patching Holes in the Chlamydomonas Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460400" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-ugoNSuil7moXjpnNBKkrDJl91ZicM5uzV0Id4_Nr5_Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1899.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/7/1899" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-zlQqlqbIn2t6ESIbOCs6HVRGsAo1bYQjXl3qG3A9viE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/7/1899" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1899.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460400" id="node460400--2" data-pisa="ggg;6/7/1899" data-pisa-master="ggg;g3.116.029207" data-apath="/ggg/6/7/1899.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Patching Holes in the <em>Chlamydomonas</em> Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6053-4093" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frej Tulin</span> and <span class="highwire-citation-author" data-delta="1">Frederick R. Cross</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1899-1910;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029207 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460400--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frej Tulin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frej%2BTulin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tulin%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrej%2BTulin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6053-4093" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frej Tulin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frej.tulin@slcu.cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">frej.tulin@slcu.cam.ac.uk</a>
<a href="mailto:Frederick.Cross@rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">Frederick.Cross@rockefeller.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Frederick R. Cross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BR.%2BCross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cross%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BR.%2BCross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frej.tulin@slcu.cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">frej.tulin@slcu.cam.ac.uk</a>
<a href="mailto:Frederick.Cross@rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">Frederick.Cross@rockefeller.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/7/1899" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/7/1899&amp;title=Patching%20Holes%20in%20the%20Chlamydomonas%20Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/7/1899&amp;title=Patching%20Holes%20in%20the%20Chlamydomonas%20Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/7/1899&amp;title=Patching%20Holes%20in%20the%20Chlamydomonas%20Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/7/1899&amp;text=Patching%20Holes%20in%20the%20Chlamydomonas%20Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/7/1899&amp;title=Patching%20Holes%20in%20the%20Chlamydomonas%20Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/7/1899&amp;t=Patching%20Holes%20in%20the%20Chlamydomonas%20Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/7/1899&amp;title=Patching%20Holes%20in%20the%20Chlamydomonas%20Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/7/1899&amp;title=Patching%20Holes%20in%20the%20Chlamydomonas%20Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460400/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460400" id="node460400--3" data-pisa="ggg;6/7/1899" data-pisa-master="ggg;g3.116.029207" data-apath="/ggg/6/7/1899.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Patching Holes in the <em>Chlamydomonas</em> Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6053-4093" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Frej Tulin</span> and <span class="highwire-citation-author" data-delta="1">Frederick R. Cross</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1899-1910;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029207 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460400--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Frej Tulin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frej%2BTulin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tulin%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrej%2BTulin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6053-4093" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Frej Tulin</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frej.tulin@slcu.cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">frej.tulin@slcu.cam.ac.uk</a>
<a href="mailto:Frederick.Cross@rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">Frederick.Cross@rockefeller.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Frederick R. Cross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The Rockefeller University, New York, New York 10065</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Frederick%2BR.%2BCross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cross%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFrederick%2BR.%2BCross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:frej.tulin@slcu.cam.ac.uk" class="" data-icon-position="" data-hide-link-title="0">frej.tulin@slcu.cam.ac.uk</a>
<a href="mailto:Frederick.Cross@rockefeller.edu" class="" data-icon-position="" data-hide-link-title="0">Frederick.Cross@rockefeller.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1899&amp;rft.epage=1910&amp;rft.atitle=Patching%20Holes%20in%20the%20Chlamydomonas%20Genome&amp;rft.volume=6&amp;rft.issue=7&amp;rft.date=2016-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tulin%2C+Frej&amp;rft.au=Cross%2C+Frederick+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460400/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460400/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460400/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460400/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460400/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460400/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460400/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460400/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460400/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460400/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460400/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460400/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/7/1899" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/7/1899.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/7/1899.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/7/1899.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/7/1899.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
