<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/7/2203" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/7/2203.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/7/2203.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/7/2203.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/7/2203.atom" />
<meta name="HW.pisa" content="ggg;6/7/2203" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetic Mapping of Millions of SNPs in Safflower (Carthamus tinctorius L.) via Whole-Genome Resequencing" />
<meta name="DC.Identifier" content="10.1534/g3.115.026690" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Bowers et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Accurate assembly of complete genomes is facilitated by very high density genetic maps. We performed low-coverage, whole-genome shotgun sequencing on 96 F6 recombinant inbred lines (RILs) of a cross between safflower ( Carthamus tinctorius L.) and its wild progenitor ( C. palaestinus Eig). We also produced a draft genome assembly of C. tinctorius covering 866 million bp (∼two-thirds) of the expected 1.35 Gbp genome after sequencing a single, short insert library to ∼21 × depth. Sequence reads from the RILs were mapped to this genome assembly to facilitate SNP identification, and the resulting polymorphisms were used to construct a genetic map. The resulting map included 2,008,196 genetically located SNPs in 1178 unique positions. A total of 57,270 scaffolds, each containing five or more mapped SNPs, were anchored to the map. This resulted in the assignment of sequence covering 14% of the expected genome length to a genetic position. Comparison of this safflower map to genetic maps of sunflower and lettuce revealed numerous chromosomal rearrangements, and the resulting patterns were consistent with a whole-genome duplication event in the lineage leading to sunflower. This sequence-based genetic map provides a powerful tool for the assembly of a low-cost draft genome of safflower, and the same general approach is expected to work for other species." />
<meta name="DC.Contributor" content="John E. Bowers" />
<meta name="DC.Contributor" content="Stephanie A. Pearl" />
<meta name="DC.Contributor" content="John M. Burke" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genetic Mapping of Millions of SNPs in Safflower (Carthamus tinctorius L.) via Whole-Genome Resequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Accurate assembly of complete genomes is facilitated by very high density genetic maps. We performed low-coverage, whole-genome shotgun sequencing on 96 F&lt;sub&gt;6&lt;/sub&gt; recombinant inbred lines (RILs) of a cross between safflower (&lt;i&gt;Carthamus tinctorius&lt;/i&gt; L.) and its wild progenitor (&lt;i&gt;C. palaestinus&lt;/i&gt; Eig). We also produced a draft genome assembly of &lt;i&gt;C. tinctorius&lt;/i&gt; covering 866 million bp (∼two-thirds) of the expected 1.35 Gbp genome after sequencing a single, short insert library to ∼21 × depth. Sequence reads from the RILs were mapped to this genome assembly to facilitate SNP identification, and the resulting polymorphisms were used to construct a genetic map. The resulting map included 2,008,196 genetically located SNPs in 1178 unique positions. A total of 57,270 scaffolds, each containing five or more mapped SNPs, were anchored to the map. This resulted in the assignment of sequence covering 14% of the expected genome length to a genetic position. Comparison of this safflower map to genetic maps of sunflower and lettuce revealed numerous chromosomal rearrangements, and the resulting patterns were consistent with a whole-genome duplication event in the lineage leading to sunflower. This sequence-based genetic map provides a powerful tool for the assembly of a low-cost draft genome of safflower, and the same general approach is expected to work for other species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/07/01" />
<meta name="citation_mjid" content="ggg;6/7/2203" />
<meta name="citation_id" content="6/7/2203" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/7/2203" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/7/2203.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/7/2203.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/7/2203.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.026690" />
<meta name="citation_pmid" content="27226165" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2203" />
<meta name="citation_lastpage" content="2211" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="John E. Bowers" />
<meta name="citation_author_institution" content="Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Stephanie A. Pearl" />
<meta name="citation_author_institution" content="Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="John M. Burke" />
<meta name="citation_author_institution" content="Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_email" content="jmburke@uga.edu" />
<meta name="citation_reference" content="citation_journal_title=Turk. J. Biol.;citation_author=H. Amini;citation_author=A. Arzani;citation_author=M. Karami;citation_title=Effect of water deficiency on seed quality and physiological traits of different safflower genotypes.;citation_pages=271-282;citation_volume=38;citation_year=2014;citation_doi=10.3906/biy-1308-22" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=R. Ariyadasa;citation_author=M. Mascher;citation_author=T. Nussbaumer;citation_author=D. Schulte;citation_author=Z. Frenkel;citation_title=A Sequence-Ready Physical Map of Barley Anchored Genetically by Two Million Single-Nucleotide Polymorphisms.;citation_volume=164;citation_year=2014;citation_pmid=24243933;citation_doi=10.1104/pp.113.228213" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. S. Barker;citation_author=N. C. Kane;citation_author=M. Matvienko;citation_author=A. Kozik;citation_author=W. Michelmore;citation_title=Multiple Paleopolyploidizations during the Evolution of the Compositae Reveal Parallel Patterns of Duplicate Gene Retention after Millions of Years.;citation_volume=25;citation_year=2008;citation_pmid=18728074;citation_doi=10.1093/molbev/msn187" />
<meta name="citation_reference" content="citation_journal_title=Molecular biotechnology;citation_journal_abbrev=Mol Biotechnol;citation_author=W. G. Beattie;citation_author=L. Meng;citation_author=S. L. Turner;citation_author=R. S. Varma;citation_author=D. D. Dao;citation_title=Hybridization of DNA targets to glass-tethered oligonucleotide probes.;citation_pages=213-225;citation_volume=4;citation_year=1995;citation_issue=3;citation_pmid=8680928;citation_doi=10.1007/BF02779015" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=D. Botstein;citation_author=R. L. White;citation_author=M. Skolnick;citation_author=R. W. Davis;citation_title=Construction of a genetic linkage map in man using restriction fragment length polymorphisms.;citation_pages=314-331;citation_volume=32;citation_year=1980;citation_issue=3;citation_pmid=6247908" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. E. Bowers;citation_author=E. Bachlava;citation_author=R. L. Brunick;citation_author=L. H. Rieseberg;citation_author=S. J. Knapp;citation_title=Development of a 10,000 Locus Genetic Map of the Sunflower Genome Based on Multiple Crosses;citation_pages=721-729;citation_volume=2;citation_year=2012;citation_issue=7;citation_pmid=22870395;citation_doi=10.1534/g3.112.002659" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. E. Bowers;citation_author=S. Nambeesan;citation_author=J. Corbi;citation_author=M. S. Barker;citation_author=L. H. Rieseberg;citation_title=Development of an Ultra-Dense Genetic Map of the Sunflower Genome Based on Single-Feature Polymorphisms.;citation_pages=e51360;citation_volume=7;citation_year=2012;citation_pmid=23284684;citation_doi=10.1371/journal.pone.0051360" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. A. Chapman;citation_author=M. Mascher;citation_author=A. Buluc;citation_author=K. Barry;citation_author=E. Georganas;citation_title=A whole-genome shotgun approach for assembling and anchoring the hexaploid bread wheat genome.;citation_pages=26;citation_volume=16;citation_year=2015;citation_pmid=25637298;citation_doi=10.1186/s13059-015-0582-8" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=J. Cheema;citation_author=J. Dicks;citation_title=Computational approaches and software tools for genetic linkage map estimation in plants.;citation_volume=10;citation_year=2009;citation_pmid=19933208;citation_doi=10.1093/bib/bbp045" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. Drouaud;citation_author=C. Camilleri;citation_author=P. Y. Bourguignon;citation_author=A. Canaguier;citation_author=A. Berard;citation_title=Variation in crossing-over rates across chromosome 4 of Arabidopsis thaliana reveals the presence of meiotic recombination “hot spots”.;citation_volume=16;citation_year=2006;citation_pmid=16344568;citation_doi=10.1101/gr.4319006" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Asian J. Plant Sci.;citation_author=V. Emongor;citation_title=(CARTHAMUS TINCTORIUS L.) THE UNDERUTILIZED AND NEGLECTED CROP: A REVIEW;citation_pages=299-306;citation_volume=9;citation_year=2010;citation_doi=10.3923/ajps.2010.299.306" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. A. Frazer;citation_author=D. G. Ballinger;citation_author=D. R. Cox;citation_author=D. A. Hinds;citation_author=L. L. Stuve;citation_title=A second generation human haplotype map of over 3.1 million SNPs.;citation_pages=851-861;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943122;citation_doi=10.1038/nature06258" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=T. Garnatje;citation_author=S. Garcia;citation_author=R. Vilatersana;citation_author=J. Valles;citation_title=Genome size variation in the genus Carthamus (Asteraceae, Cardueae): Systematic implications and additive changes during allopolyploidization.;citation_volume=97;citation_year=2006;citation_pmid=16390843;citation_doi=10.1093/aob/mcj050" />
<meta name="citation_reference" content="citation_journal_title=Agric. Water Manage.;citation_author=H. Ghamarnia;citation_author=M. Gholamian;citation_title=The effect of saline shallow ground and surface water under deficit irrigation on (Carthamus tinctorius L.) in semi arid condition.;citation_pages=29-37;citation_volume=118;citation_year=2013;citation_doi=10.1016/j.agwat.2012.10.028" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gnerre;citation_author=I. MacCallum;citation_author=D. Przybylski;citation_author=F. J. Ribeiro;citation_author=J. N. Burton;citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data.;citation_volume=108;citation_year=2011;citation_pmid=21187386;citation_doi=10.1073/pnas.1017351108" />
<meta name="citation_reference" content="citation_author=M. W. Hahn;citation_author=S. V. Zhang;citation_author=L. C. Moyle;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. H. Huang;citation_author=Q. Feng;citation_author=Q. Qian;citation_author=Q. Zhao;citation_author=L. Wang;citation_title=High-throughput genotyping by whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_journal_title=Botany-Botanique;citation_author=N. C. Kane;citation_author=N. Gill;citation_author=M. G. King;citation_author=J. E. Bowers;citation_author=H. Berges;citation_title=Progress towards a reference genome for sunflower.;citation_pages=429-437;citation_volume=89;citation_year=2011;citation_doi=10.1139/b11-032" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Luo;citation_author=B. Liu;citation_author=Y. Xie;citation_author=Z. Li;citation_author=W. Huang;citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.;citation_pages=18;citation_volume=1;citation_year=2012;citation_pmid=23587118;citation_doi=10.1186/2047-217X-1-18" />
<meta name="citation_reference" content="citation_journal_abbrev=Genome Med;citation_author=E. R. Mardis;citation_title=The $1,000 genome, the $100,000 analysis?;citation_pages=84-84;citation_volume=2;citation_year=2010;citation_issue=11;citation_pmid=21114804;citation_doi=10.1186/gm205" />
<meta name="citation_reference" content="citation_journal_title=Veg. Hist. Archaeobot.;citation_author=E. Marinova;citation_author=S. Riehl;citation_title=Carthamus species in the ancient Near East and south-eastern Europe: archaeobotanical evidence for their distribution and use as a source of oil.;citation_pages=341-349;citation_volume=18;citation_year=2009;citation_doi=10.1007/s00334-009-0212-z" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=M. Mascher;citation_author=G. J. Muehlbauer;citation_author=D. S. Rokhsar;citation_author=J. Chapman;citation_author=J. Schmutz;citation_title=Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ).;citation_pages=718-727;citation_volume=76;citation_year=2013;citation_pmid=23998490;citation_doi=10.1111/tpj.12319" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. A. McKusick;citation_author=F. H. Ruddle;citation_title=The status of the gene map of the human chromosomes;citation_pages=390-405;citation_volume=196;citation_year=1977;citation_issue=4288;citation_pmid=850784;citation_doi=10.1126/science.850784" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=S. U. Nambeesan;citation_author=J. R. Mandel;citation_author=J. E. Bowers;citation_author=L. F. Marek;citation_author=D. Ebert;citation_title=Association mapping in sunflower (Helianthus annuus L.) reveals independent control of apical vs. basal branching.;citation_pages=84;citation_volume=15;citation_year=2015;citation_pmid=25887675;citation_doi=10.1186/s12870-015-0458-9" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=R. Bruggmann;citation_author=I. Dubchak;citation_author=J. Grimwood;citation_title=The Sorghum bicolor genome and the diversification of grasses.;citation_pages=551-556;citation_volume=457;citation_year=2009;citation_issue=7229;citation_pmid=19189423;citation_doi=10.1038/nature07723" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=S. A. Pearl;citation_author=J. E. Bowers;citation_author=S. Reyes-Chin-Wo;citation_author=R. W. Michelmore;citation_author=J. M. Burke;citation_title=Genetic analysis of safflower domestication.;citation_pages=43;citation_volume=14;citation_year=2014;citation_pmid=24502326;citation_doi=10.1186/1471-2229-14-43" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Renaut;citation_author=C. J. Grassa;citation_author=S. Yeaman;citation_author=B. T. Moyers;citation_author=Z. Lai;citation_title=Genomic islands of divergence are not affected by geography of speciation in sunflowers.;citation_pages=1827;citation_volume=4;citation_year=2013;citation_pmid=23652015;citation_doi=10.1038/ncomms2833" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Oil Chem. Soc.;citation_author=J. R. Smith;citation_title=Safflower - Due for a rebound?;citation_pages=1286;citation_volume=62;citation_year=1985;citation_doi=10.1007/BF02545945" />
<meta name="citation_reference" content="citation_journal_title=Journal of Experimental Zoology Philadelphia;citation_author=A. H. Sturtevant;citation_title=The linear arrangement of six sex-linked factors in Drosophila, as shown by their mode of association.;citation_pages=43-59;citation_volume=14;citation_year=1913;citation_doi=10.1002/jez.1400140104" />
<meta name="citation_reference" content="citation_author=M. J. Truco;citation_author=H. Ashrafi;citation_author=A. Kozik;citation_author=H. van Leeuwen;citation_author=J. Bowers;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=E. Wijnker;citation_author=G. V. James;citation_author=J. Ding;citation_author=F. Becker;citation_author=J. R. Klasen;citation_title=The genomic landscape of meiotic crossovers and gene conversions in Arabidopsis thaliana.;citation_volume=2;citation_year=2013;citation_pmid=24347547;citation_doi=10.7554/eLife.01426" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Youssoufian;citation_author=H. H. Kazazian;citation_author=D. G. Phillips;citation_author=S. Aronis;citation_author=G. Tsiftis;citation_title=Recurrent mutations in haemophilia A give evidence for CpG mutation hotspots.;citation_pages=380-382;citation_volume=324;citation_year=1986;citation_issue=6095;citation_pmid=3097553;citation_doi=10.1038/324380a0" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genetic Mapping of Millions of SNPs in Safflower (Carthamus tinctorius L.) via Whole-Genome Resequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="twitter:description" content="Accurate assembly of complete genomes is facilitated by very high density genetic maps. We performed low-coverage, whole-genome shotgun sequencing on 96 F6 recombinant inbred lines (RILs) of a cross between safflower ( Carthamus tinctorius L.) and its wild progenitor ( C. palaestinus Eig). We also produced a draft genome assembly of C. tinctorius covering 866 million bp (∼two-thirds) of the expected 1.35 Gbp genome after sequencing a single, short insert library to ∼21 × depth. Sequence reads from the RILs were mapped to this genome assembly to facilitate SNP identification, and the resulting polymorphisms were used to construct a genetic map. The resulting map included 2,008,196 genetically located SNPs in 1178 unique positions. A total of 57,270 scaffolds, each containing five or more mapped SNPs, were anchored to the map. This resulted in the assignment of sequence covering 14% of the expected genome length to a genetic position. Comparison of this safflower map to genetic maps of sunflower and lettuce revealed numerous chromosomal rearrangements, and the resulting patterns were consistent with a whole-genome duplication event in the lineage leading to sunflower. This sequence-based genetic map provides a powerful tool for the assembly of a low-cost draft genome of safflower, and the same general approach is expected to work for other species." />
<meta name="og-title" property="og:title" content="Genetic Mapping of Millions of SNPs in Safflower (Carthamus tinctorius L.) via Whole-Genome Resequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/7/2203" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Accurate assembly of complete genomes is facilitated by very high density genetic maps. We performed low-coverage, whole-genome shotgun sequencing on 96 F6 recombinant inbred lines (RILs) of a cross between safflower ( Carthamus tinctorius L.) and its wild progenitor ( C. palaestinus Eig). We also produced a draft genome assembly of C. tinctorius covering 866 million bp (∼two-thirds) of the expected 1.35 Gbp genome after sequencing a single, short insert library to ∼21 × depth. Sequence reads from the RILs were mapped to this genome assembly to facilitate SNP identification, and the resulting polymorphisms were used to construct a genetic map. The resulting map included 2,008,196 genetically located SNPs in 1178 unique positions. A total of 57,270 scaffolds, each containing five or more mapped SNPs, were anchored to the map. This resulted in the assignment of sequence covering 14% of the expected genome length to a genetic position. Comparison of this safflower map to genetic maps of sunflower and lettuce revealed numerous chromosomal rearrangements, and the resulting patterns were consistent with a whole-genome duplication event in the lineage leading to sunflower. This sequence-based genetic map provides a powerful tool for the assembly of a low-cost draft genome of safflower, and the same general approach is expected to work for other species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
    <title>Genetic Mapping of Millions of SNPs in Safflower (Carthamus tinctorius L.) via Whole-Genome Resequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TRqS2953uXciontfLcj9nL3Hq5XTX6XliL0lFacTut0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460419","apath":"\/ggg\/6\/7\/2203.atom","pisa":"ggg;6\/7\/2203","ac":{"\/ggg\/6\/7\/2203.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/7\/2203.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/7\/2203"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/7\/2203"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/7\/2203":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Z4zIt0LZQEDMtbmyDoXIprQw4OodQChBHG8IQ6yKKMQPYv70OoHCSaQ9VTwOnZTWoyVBZMJrBaccmcuQrkD+0sgi18WvdagBWcSBaKu\\\\\\\/p\\\\\\\/KOzLAIuJdwlHiBqhhtPphhqVlxkdvBeKoh9K0Fcyl\\\\\\\/F0Tl2c5GXEsUTkqF6rV+C7puJ1VquN631zCmAeM23UNsYph9xJTkyZpFzVVZTC5l95a9pA5zO0+lKhvBbiouDpELGAXYz5CoA2CBj9T8y0iKtZT99d5DqqfzyQNJsjvpBw7XfKNtxuNI3bHC+Zi715h7qcF\\\\\\\/+RVR0ZjKqUe8SoRO6vauqEZRUMFivRbAWbV+HtVoYR9VTQkA3gwG4KVVTl6G0oe6OVAB334pcbYkb8Ns0WnHsyKFeEFhkJfb2wdlumx6IGF6LXwtSat8RxWuh4HMLOB375zYLiKVv+VP8pic19HQjTXGzx9ary3zC1VqAc8RErKI1deDRxJzPjQn\\\\\\\/OSseX5RE3GXg5IpJQpQCruSvcawkgEOacPpxftL4tcvlsbQAUQXXV1Wa\\\\\\\/OVpoZPEgeXAK1Ny9mr4SGvnZyGCxyrQvDXy8i4rv9gSXfamlU5rX\\\\\\\/xh\\\\\\\/XdHb7\\\\\\\/P4qA7bfPa1o4Dx+dZcrq8XTA4SzmwbGu7mc\\\\\\\/UtHepGWbU7xz1kAqd2O1BSv2bc2adMw+25e3q9LatBwSVwFjD3odzqoJYypa+LrWE3dQqtfl7j\\\\\\\/y3wRl6Uzija9nn00oxP2MqtdrMXYLlkwZ75n0MJ5OhB19FoQMov9LtyzRUN5mDzgTwCnV6j+OuTyvRRyH8jxCdEJ1VfvaaqDrzfHY8f6ntM8waC\\\\\\\/+RXwkRKIo4ilFBs0GsP3uycEXnz1ItUjzTL5QtaTopdx7xeg+wuC\\\\\\\/prL+7Xb5YGwBWz3pUw7mZPfAhR5327HJ25+Vd9s9HHbpnqvIeb4w9X6c1binVb0GXRwzQ53Fi5sRw\\\\\\\/9cMIPAL3AdlBrv3YaKczpAzTJUTeOQsFOhgoI7BZQrabhJPXi6k0gvpQPPns\\\\\\\/fAINwG5ZZUSzksYevvAKUu\\\\\\\/ic8WKWgLJ76nLqRkGsTjbPpFz\\\\\\\/T1bKrylJrAdyZchneV6OSUt31CY6YDwqAyNsENd\\\\\\\/11RSZsEUnOgQf75CMIXvETcejjFshQ4GmdXcpLufRA3XhzgVQCuVTKNtN8kz1dQluwhFbxVSyXywoyU8mUxakKmCKds4Cwto\\\\\\\/zbQ1QM\\\\\\\/x1XuhP2FsTVdW0iGqU2hF\\\\\\\/yaV\\\\\\\/uXMvLa01ubV45QTBPBzHgYR0+7JiRF7ryTVYxK\\\u0022,\\\u0022iv\\\u0022:\\\u0022vIdmOC2ddj\\\\\\\/k46\\\\\\\/nKtvqAw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002273feb1716c9eb29a3c519a8a8bb6627b3b72ad9e49245f0c40346173d158e600\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IOVau2EvrbPa2lytX\\\\\\\/eKJV4kWqb7fa256CWvM1mffn5jPCjj5o9BwMeKvtK7bZRPQLeLRhZ5Y92sAqBVQyohGBk8MMgag7Sk4ejLRaErlG0rTGZChBhcRzf5LuJRkDfQ\\\\\\\/WMVUwgsJURDmJLijmOof6CLeEFd+k6th0aNmPImk2Z\\\\\\\/dSgckZ8Bb4aoyGKVyNtYJRqtRsYEOJ+GANCchiZ5dEnIFGmIFub\\\\\\\/lSDIgHR6Xfh0DqjkkVDuwVRdXNpJZyndnU0MClHXYY4BTygN1ujhKZa6Dn7DPYBZ7imvcyadZAVZoHHXvrUVi+uFDN+omcLSawPg\\\\\\\/DRGi96MywOPI0hL6wzHYc17uxReZO1QM9w0ixJY30znbE1caa1ho20tNaJRpraHQWZBWeKsKcroTajeLChLvKW8wldfzoprt7sfPEfflHOz3m1NwhFwSM3avTJUHBJRxXKq2Z94yDHXYRqYuiXmSVw3yJBf+HPIDCoVTkdIaA+s+\\\\\\\/RcmRt5T5oZtL4Q11iG4VZbLK1nwH\\\\\\\/mwAasqZuvTc6lINNomNW8wHcb6paVuWrfd963xnLWKf9+enub8V17UQ2tzYH+W8EijrBxvhEr6gLnMjxyY6lzZnkohKRVikQZSBoqc5s3bF67ugzmfAXLnxIQ4zKNycHTrXPSXmqHJD8EUWQ2kKdbsbnI7I92fBoGKNtodOWQ2o4e5ILAGN4Owl6xW0A9cj532y+W1EK83i9lV8W5dFcGRhcwKCeIBnAYpN8k6qNPFVLiE1KMoC3tM0uN7WbPW58hRiJIUA+qcrz3mY4gzF4VP\\\\\\\/4jy07SI8moLTGxL0ZT1ASpZk7A1niQ4bKEODjnuRGbDSGQlSYLccWoMpp3j2ojFD5LP\\\\\\\/kCmJFHZdCCLjq8l044nC+cqNPeYR87Yyj0sTco1pEqW1c3eNgXQBbAeCGGpSAxGH\\\\\\\/a418bGsBr+bXdR1cYPmDrJagrfyLubBbWSX3E0OjgXlycDAdMmiIKbLLo+R\\\\\\\/XJoX0RndKqMb+SHx9OFNcKA4alybwhKvgpXVZb\\\\\\\/jzFnQ4RZ520JGC5XvqtnraC1aMcoD1U\\\\\\\/bZhIbzXsk149KLssZ0OfwvXK\\\\\\\/0kTBmBfkQQ+qTYaTVozZI99MPgXpiKgB7tmumpP0DIScHqi0KFfVvTJN\\\\\\\/gvb+9\\\\\\\/x3xtLuGmbSsa\\\\\\\/wAwJVk55cvgZAvaA3uL4kCKaTJkERNiP2guMN7dASyje340eY3hN4FCqz7jPltH5l7Ns6wcboipyRM0q5Ov9iFN4YJzx8\\\\\\\/9cySQNJhnVqr4DMFuH7aeTVpQSoQ8lijXF6MwcWwI67qjUJCLQlZdWDEKrjET3gOEoMd8dOLdCGJH\\\\\\\/8uV3btX2KOzvvzRUC741oO5\\\\\\\/Tj\\\\\\\/a3OaAScX\\\\\\\/b0WnsJVgRtmuXAe+Y2APpIdAjttMueN0o91aDlktZQgPmv2Lqz45gIRVy+2DbwCopskDhKf5lQ51PUdg9tL4O3az2SssGH2+Vq44ZslYrCmyedirFK+pezQpvCDJEfe2Z\\\\\\\/JZmyzRNDppYRCVsxCeZOuoS\\\\\\\/Fp69nD+fCPFKM99gO3+44cfGc0MP4lQYkEaq3etfST2W5szPGaXwpumJYd\\\\\\\/bD1i+dhczvKRVzUcH49ek+tyhDQ62miWIuq0M0gqtz1QViHf1M\\\\\\\/+4YrlAjXAS5YeNgE26nHDpoUjMDHxNMGi\\\\\\\/5t1DWF5uE0vo0HfZ6f3u1\\\\\\\/Xj6FAwHG7bA5U7NDTp4CvA+3iRCjrY5kSZqCkaX\\\\\\\/yyEtaGWZV+zywMNPA\\\\\\\/k7IMjfjYLtfQQKJQ\\\\\\\/LUV+M8h6veLmoJDWAxcpI7l06LNvo6sKP0kP6UQRjYWhby6ZKGlMQN9Cjnfg5WxWQszOLV89UCPJver4O7DLw77fK7hzLsMJKdSacySc7ZrXMxGJqAX7xcPtiQNxpEHo2cCKfMnTgMchlWvLM+qzlXpd0ECo4REWP5IEzfar93oUyHEIlHRZkPOeyY\\\\\\\/amvmnilvSbcDXkldlpS4XquhLP3Boc\\\\\\\/Vapn0bqTgVYDPpdulYgFefB2Kf0vh8m5Z4c3rqkCiYVrH4HbF4Gz37kL5Mptr9\\\\\\\/v3WljFRAB4StgoLiQVThSzxUN3dCsguzRqWi0W8KZL9QOfKOdOOAxyIBi5pfh5zYScDxMW3GB9NDP6NBgOMjVxra8C4R5aMaRwXAK4RCQZjbjY5whbbhYsCGiifd2eQNtPtHTkPmk5rKXAbjbMDnIwW5aFxZqDkJOYrndHg6RcSJkOqAg3QioHjEPBAytQe4cFVnDuXV8qR1pdf3U1ekPmlLWJjh\\\\\\\/E\\\\\\\/kXbputMB8FXv9BuXN9bRUp8S7zlOMNpJ1wRRwD0laSc0xRGC0EF\\\\\\\/VSdbl1NUzATxb\\\\\\\/szoZypeJ\\\\\\\/tSeN8z5pandbvNLtOY23Y8YPLGGnJ79bsJO65edEzhUQ1Uxa26EfEncJYPcKAgvnugS5kwE99jbEEAJ8kr2sJ8pDuCkXM3E5VbkVtCvnnEsFsBVekDvJWW7jK2DPDdtngKk5gwuaGmXGH+Zq5iGDwjDypvRJZq8u7f5c9H2zA7FH+EhHkPpt3qwjyqTGV91hMQOnKqhqSzCHqBdiLwAezvjjLbSzyguNHQygS2thrplBMEELyga4Rs3hHuyvj\\\\\\\/0UhvGPHV+s4Kx2FQgS7aCQ47e1v4v+3eLIFaQFWs3Vz\\\\\\\/o9M0aO\\\\\\\/zrtwjieTzDRC+33G3UayJ7mptEN+\\\\\\\/Kg5x9hnbk9UJV4Y7NGcXmCBnddeSKKimXUtdfZf6REdzMfZDONuChnLv4F9cDI8vpNB4BlqVSCEoYUejIN4G95nDQT4Iilwyj9311DvU1WOVeZgyibsVmY2HfKhoDFXzf8UIw9eUVCXBmUpzUtjgcUj7RVP5TGeL20N\\\\\\\/GZ9XPGIBgtsR0RKqqpIO4xtY1lwg8WU5uCNBSB41JyySJHMNa0cMvAGvw20XeIpt6LhXcUyRjQySNqvVn0mAAhgC25yj9JhmlKwg8IW2tBFnfWyr0IdMC2M7ixswjKzzAzaaq6yhLN5cDEJYQ4F8ABGTobKv3sKuhblYxvUV00QfGrWMhYQT\\\\\\\/mg+18rOFoWSC4iJBGh+toM8rI7KkFX\\\\\\\/+2xvUMu88F8ZPGVesnmg\\\\\\\/6i0SNOoRZdtT2UsZV06L7kLXYKPOEoy1tiffp2HwrqdgzkmUEDaMNS\\\\\\\/cvjJ4FCda5HM\\\\\\\/zPn6NyoGm1obfFduX\\\\\\\/m6gLXd3PLZZpB\\\\\\\/0ipHelDDCDOdQvknn8Y8oDvzihp6SFz6h0I2+6g6U0AaJf3VuHdz0CHkHJuNhd91zrr2FtCOfm+UYt9CzrCwymgjBBEikGlxZ2F9xaS1vNgpKosbB22YDkHANg9m520ra8SWj\\\\\\\/Qe8u4BI12Qp+hJHF9CqWKyoM8iM6BhG\\\\\\\/U7UIs3iDfFbbqqXmj7QoFJWrC343jqxUeTfw83ylSG5i8Fw1EBTPNIlbbvUaJoOp+bAjuY89pVSUA5DxbeijtU47ZmrbK57lirQdBviN3rRC2Vu7iZSQ76eSO62vLSDj4Te1flBYhVJirRJAyoFYxw1qF69oNZukzu72gnDWcY17DvOGD0v2kNAcBJRyG3XMhncnUnf0\\\\\\\/MDXbIG2ayzvpTA3opy7RmDkD0B\\\\\\\/AZiArPgoyFJFTdcmhM6jMuAMwAK2epNAxcbuj5zlN6TfOS3maueHp1N5\\\\\\\/esM3EFGOWtdBiySIS5oe+ADUwbV5QU70QeuHS91TA4PSsiLbtOWkFdhsOqdbm3yeHvPznrRDkeKENvOH6BmM0L7gQ2J+EaB7TLar\\\\\\\/JDIP4PvyenM8R761S1WqZp7jDtD3jzomYNy15vYFiaXx0332I3Qd+SRvs40o\\\\\\\/a9orNXg+PbPB4Zo4jUroZmkHsElLw+eEUvnUrnCgt99s9F4GmHATcXN2z4\\\\\\\/kxqHUIA2JqyxRXnqjQUY\\\\\\\/SwDyy2g5ooj8DLEMjc4VlUNkEi62KK\\\\\\\/uu2AmAtWBEyZN0AWc14ukOfu8kmfOAXpqaEyXFvLcWgQY2Vl8Di6UtTgDMnHcq3YIgfMeylxBL7bDRsXqebRsNz1wKwOBQNdBwr8j3r9klcWmRT+zVwIlxZUxIGT3OuGnndPJmpkak1OgsHhQCJIUI+8tuKg+hVgHeDpkKhZt\\\\\\\/HqQ4j11G8FpEnKStqoisKnAcF\\\\\\\/LcL9TyUhM2XzH5TnVJnMBXbrewC\\\\\\\/9\\\\\\\/d36swBIN4uhQhaevsoyYXsVBJqKKnn1jTwvW1JKSj+5ZRJEaIAiJypAAz9MhNCx883GqF6gZQ9sh6Ahb7eA0BAbN8xH7LBhKGp3bT85YCkA3Q\\\\\\\/xGQOAWVa8SLs+YJ6tjYcya8E\\\\\\\/bqGd1+GIARBN\\\\\\\/iDThY1GjdiIbEu9WKnwGQhswBgwo6gKSWgHfMwQJ7I0aiYMW7PKq+sbuH3sdgfyBq6QatF2wa0uWQyIVJA1G+0HzL2ortcWNMMJiNnmyq4J6GWaOjbYBmRf5lIKHBD+a7EUp9Pl9nhPKSwQqrl9EPl9kNfD\\\\\\\/SlDWab+opMCeDhAP3kNE4nhEMU5QzCak+NVfYAkCRNt4VoTtk\\\\\\\/qvcAVxbGit91xRpD+kDMFPuZ\\\\\\\/VFOtdeJKc8DuYLQrGf+Ze2+IZRAb\\\\\\\/tQQkjb1Mch2mj92djNH9s4titK3alvTmSpU5OXS\\\\\\\/jp8kVgOxitD\\\\\\\/fy6prrnH18ehlT6ARIgchP0hIdNsZJqs1yaBOSPiVWChoDykeIepHBegzCyoPq6T0M8vVo\\\\\\\/DVYgaYMEi5+r\\\\\\\/jopCilibNJctSVx0hkhU7EBhBBIhp27p4L78mRhNAX0w60N3GuJRaundV2V88HA98iXMwQzZ3OvU8D8I3HDry1ljxSrLkWlxg5rVqJcan+rDTnGTuRAhz0E5q2EvzMy+HHJp62pGF1FEiAetiRwrGNtDmRQfu+bhDZMmYxeDTZn1e1LzqvhYmIxIRj9jbkmqojV4J4X4NGlShO+CX3PsibHvyRPWfOOrvgW1Ee8Yepm2uP1r01V2wRxLuQDe1X5kFgkLhaOyKO1IaESZx3KC3CXpXoPoxC2hhfKoRtGdYnbnxqn821EoFcwvFQSmMfS9R+bkGlDOLUwB8q9fyxN1xR4J1WcfDDE4+f8odRA5L4TikhieKYuN5WDNp7ATJeJBz+IkRo8Nk208O2+FXl5GS2\\\\\\\/FavD9WKXeL37hLgaqaQUMNmMHOEiZvsJHYb9PD4gn316aF5HnHMQMTkaw8qC2Jj0zOp2SNrbzPCJkLPKkwkRkJX98al1ckEelTqASpPnLHdo0nMQoAkNUgXijgJaD75ljYoc7wawnY8V14DlMEvKU455K+V4RglMIaxxgYJTIxgt2PV2yOAdtqOXVNmSf5QfGUnz9CcWuzayaqkA9P9edSvcDXMI8pigliusrwFS8Zcu3CfT9SyvxkJR3qpGo1dzf99qGG6xXYlSvhskakpx3yTaOm+xSRR\\\\\\\/wwtHq8Ut4DXAP+bdC5BHWxYDgFimGkmjqIFfbKlxFwPzUGi0Pz983MSJn9WoMfx+ciKU6UZFlRETHaNr9trAHEJUdU4Ngh7tm6ezC\\\\\\\/oo4c0Nya2\\\\\\\/Spe8YOKJmOieOy0RWN74I6lyf8XVUJT0IMzHTmCx7jK6+Jd0rzCTP3E1NVryzLZgW8d9XvKu50SmvSPJiMubamPEAHEBkN5\\\\\\\/stPwpIMyLcsSyYW+UHbPAN\\\\\\\/FMY8t26\\\\\\\/75hbxsgd5JHjtQPACfaz4n1IW9j1+kX+517WE0Z2EyCxSQEEn5uw77KoeZf5nooA3yP4pNMEuV00WEUP8qWRkbEYH\\\\\\\/2WxdtO8nbPWq5Bz35V2Qxo\\\\\\\/fmD7pMSmyYCUHQrc7fLX0EjBo6pLw5yZqeMNimrBtlyITyWGY\\\\\\\/F7LNyvMcyIME6Sp7au6dDJkIRTb+y8c5y8nqrR91M6lK1IL4RIsrVIChIfc8vclPjY\\\\\\\/+hw4HH+hijXjnf7e7VJYniadBFgO11NC6wLyJXCwHMpR5+yvS3tLHr60HJeZ6Qp7pLsAGoBrzVtHARfa3zxHpc9L5ebm71mBzRw8dwJI1qJzwVcm5MszSBO4uRGXwRfPthF4aOFPacAF\\\\\\\/sNojr\\\\\\\/gKLz2BBszvlspBj8VQ6l6IXUvFSUnODriIST345768xxIDq89h3yuZai00FqQljffAlN2cbLXUwI3+X76TbXrHnMFAfs9\\\\\\\/QIs3Yk9GTwf7bTlSjgOrTQusd7Nyq25b7Cc\\\\\\\/v8HKwhE3KEhNNavh320haDeSB+qUk\\\\\\\/jfv1efMJJe1SXsnthEqhOkmy4Sx5JwG9fFgGiop7iggdz79qoEq0JEBhcFU0qqGNlYw2HNVoLKiBdWWpZJV4o9QaBJdhj7Raj7fVcaYR1J5Jsb0ZS9mfcU95Z6X+RUkZttleqdJwzddvU1rlkDLcCvmT1YbExFlyBxx7RguPw0zTKHMKtBwRUl+s3qZS1aDo6GmByCRXIwdw6C6+HwGa7a9V7bYlkkXkLca55vRKBAUN5x0d6O1oCpI9ogUD7UScWD+LHoKv+jaquOQ7jKWB5fQCaYEcf9KdESvmQ7fYt9+8B\\\\\\\/AHoqyQ+yc\\\\\\\/p22Tm7USoCAr5\\\\\\\/l4HDbRrnsI1dFAl5T5BSRaDbja00+AdnJM+7oruoqiTQmwWjB\\\\\\\/gskpRbRcO7NcdzLItGGsEjdxGTsq4IsAe13AvscTF4GU6jdNXvsz3Ddy3P\\\\\\\/BlNwQDXK4zkgmWnbaCzVM2JTmkHeDm6AW1K93+tsCwq9zRqBytUtCiNgGQhu3yFtfdRfk0nRCWuFeOGkxcaZN\\\\\\\/ZUPjNQYzxqFC27a1pGnFbLENyVl32FzuNNkczxwCGeGQ94fEpm1gV1AyLLDwvmQ0FMDAUY96MN5FRg+4xQdOqk43oStUWLGwlOJfWcYgYy0SIqUqg+\\\\\\\/aA5HB2arbU8lAsMXqrDktE9ce5DYWquD5xap+T14HBGSGXOubaCz5xXvKUf2tzOEtbYfS0T9zbx1cZSblUl9kCZqQ+yNXsjCWNzC6eptqhoI+gTDGPILRBKeQaPJFSCQctFRkTOzpiekVzvDBas4423kOVa5lxa93rckIX8SR03wb17Gh9XjYCmdHTDz48zJlcBJD8Yv07oQlWKr5WzMrSSYuiwo2cOKqpm5vknu6HjCm0xDzf5ejnVn60h989Vni34ewDXWjAgADlFBQI+17TV0f\\\\\\\/XhZcLbn2FRDclOfzbgtQBt24o21ybaTmBh3qkYDAkQU+ew6yZO9dsf5KtSUaGjKaDLGktq3Q3whSkNA+UBjw7OrhHI0SrH\\\\\\\/Qv5cjY3L7qY3iIz019YkjG8LS3X+pm258ylQPvv0RIK0exFK8DVPezY5oifvd9VYMB1ApxBhsgPPlUBFdHHsuRdw016p09hstpzfvJvdlFsdAeNETtXwtnYotVjHQWfciRh7MNKWatBe2FpRTc9F2pSHGW7Lf\\\\\\\/J+cN0uNf6NDQbUEY1uqoHKDfD6UtcyaCRL58Y7AIuspLFqEawZ0IS8tXo2Qy7BVzGBXcz2JfRGd7eweKi+oyJvfYclPBcblnw76CKwB4cUhRN7J9OFAKln\\\\\\\/NC9dIU5aZke4KX2v593z6i0wRViB2dFMAn7EDk8PE\\\\\\\/Aq+L\\\\\\\/J9F6J3ZDLnanhxbJCjGeo9ZvhPgZZJORF8\\\\\\\/jpXkjk1NO1fIbH0mnkIjHsBztOn1fVmWGo3SqiIDbIYApvQPWfck8CKd95rfzG4bwdSc+B6udviQ4iCPLgi3q56iURJpt+jzPqL+r4uzK84Q08sZ0KrD2eKrLIAENdcX8YaJvPwUMlheaGwldQwKR9XcFj\\\\\\\/a+d061q5pAF5Y+5CbPOzBtZW0MN+Tgpcr0aypNuHp9mCvORJeAiS6agbfeUFCJcrFG7L2HK+WvMkxh1f2fc4BFQ1SXOKcKlPpbflbgw1nFSZMipd\\\\\\\/siUXOa2HfVJREXu2H0XdMVTKHkGcwg\\\\\\\/1KRy93ReV835tVeL4dnpxTnm4EuPRsyYuAaaBDyx1prQHP2dhaAjjpQ9N73E0UG+IW2HSzRc\\\\\\\/Q+iu3bp6Ine9mQB92gziYhgqAo\\\\\\\/zJTodNON69OyyHSdnnCpTB+gtxizGuyvP2jzYaKb3VdpGWqgt\\\\\\\/RdZOX1LgqVha8Udaylg4XbEAmJur2GN84R1\\\\\\\/gyvY7RLDhJYEmPdcwdVSLBPqBvGDnWkyDxHKDw\\\\\\\/NqT39rf9Ud9Ob4ovy\\\\\\\/\\\\\\\/QtUa4GkAujjRVT9oDr64T69wgn2YBz\\\\\\\/JVPqlCRu1tpAGEKZJC0x93DQi9\\\\\\\/PuXl4auyVMpRK8fob7+nM8JMeMYH4uGKZWbp713hjUfRT5K9xts5KU7DAXizTasTXHhnYsnNpI9JMHBKD0juV1tpKgofGfRGSQ5MMxBOdTAbwm0+kibPYlS2DgRTznQtB9ggHuZ6X8ebcWhpHA5rBbxHG3nYKcIRH\\\\\\\/9DKxuSGFnrD499ICIY3gwKO3Zaex1h1LpZPI1vlUnjfBUOYWid8F7p5CpkfQSf0myj0z922q8mlMlcMlusdumOqbkykjTUb5kx92HaC5sgFHqTL5o4bRpNnCBZvHN39oLcX8xNaomUC1+kc9CyQ7HoC+sAlXj5VSrOnjd4NDLmfXKYKx9WrhL4qtXIXT3+LUoKN8TGrG2AQbxaoEk3uI46\\\\\\\/3jmcPevlYQ9MkOi31ybIXk0vbZVCk3JyyjB543tUnVDdEmOuOJndzau5VTWC2iYBdQ5t0ZezzpdNxU9y456ELl7nhYUrddO2bXq1ZbGOilGCoIc5l624UPusWArhjL8\\\\\\\/svYrCCusKccQgQsUwV\\\\\\\/VCQCJuT\\\\\\\/c6zLh4EvHTj1YbQsCiau1qJauKyXqc2DKCaiyu\\\\\\\/2xaKM\\\\\\\/9M9FwmZObxIGBqbaOz+ACS5UNjDUem9s8kddWlbVPBBMNgoUYr9cD72R6P27dCaW0JXWTPvOqIWbPrhp3hfy3Hfzb5uw0+pYpCw+XY6B4qBZRXgKPVyTf3v69fd1v48kK\\\\\\\/pr+sQbsr+Wv6Lz02vhW9VwIAD8icN4IcDjoKvWHO6vagLYSyOQlMKqL93Gde2OvXumnCmh5xLagVegPT1ziFh7Q1rtvVJbVd0FCEFikvqUXXfXUO6wI19crlr3jlSxIWlRrYeWPdgxdkxfqzZZM8f\\\\\\\/PiWH1rS2cpZea4Zefd6e2c4IyWNS5iPk7RtkXek8OaZPoiHtW0VrUMHaIMnP+KB+bfCmo0+t0vV4jvXP1WF1o6pVI8KObwmHP\\\\\\\/Wsh7VQmgmGnlHmLTxNhepEZyH5Oi1S4sjuJa8BCx9hugkPGuFTPF5mqWqqo8ufdaVmOprBEvJpjzd3OutMaBp11SWApwwBO8tCBE1i\\\\\\\/0gusqUW\\\\\\\/B8x+7aPBCKRtYofiH26burxZaKWkzOA\\\\\\\/BVNnqGlKH\\\\\\\/\\\\\\\/QiPkXl61\\\\\\\/IsZdHaKF1gjenTcOqC5DoJxP7ZEJeEXbXV1Q5dVuqrfATrRfFXgysaIFsVhJlFstf\\\\\\\/870MjZOSxkikT7X7KNJMBBsYi\\\\\\\/9QLieUsEE2OU0lFjlYoviwIHPfF+owUX44jzjp3W5+sKB0LHuslmXpwMFiet4F7CIHLHZhdbQwVu+qh4ZTKHhe9vzVGAHXo8qn05DCeHv5U\\\\\\\/bFMv3B89IqA3hrK9Vym\\\\\\\/1q6f9FJrXwT1JRZovYdCQZd0qBx4RTT9YQ6UOTwKDJiMzkNWNUVx3\\\\\\\/MJOtBdx+vRpgM4YIRGoh\\\\\\\/DRfDX09ADPebB18MIHbzhGDzrAZSyD02zchtNcQncMNcdcgzrn7G9VVq0zun+SV2NnzRyX1E23Ss7dQ49QLxnDd0zPNI\\\\\\\/XSf2ZfzUJStvwzg3jsGOu7X\\\\\\\/53UFp7+YavVp4xMxi1x2++YJb6gLto\\\\\\\/dolZVp6hqFcPDgsVmEFIeI9NmZsfywO1PTOtkTFhb7NSQJhoFAjn89jcK4TsGGEfhU1CDY5o2fbOHTzP4Ghvi87h\\\\\\\/e\\\\\\\/UaO6F876F0eOWK2ulGZWJI5pveGfd3SIYe1lCiqTvph3zJgJdzIKYJw30\\\\\\\/nqXLUZbKTSM9\\\\\\\/zBhttui9E2VH981z1o2ibXMAyFtimZIUIJ6T0a8aQzfXfrc8wJ3Qjsbzs00LNERbT0\\\\\\\/yNd8Jy85VjeyRcFqyJJ04jAIr0\\\\\\\/0nPzqubzB4I\\\\\\\/szwKQNuXVKfrtlEmBJFYOvxRgoV6tt84\\\\\\\/R+qcpQCfEG0JmfYcMLW03y3OaMSR90jnSy8+9DLTJFbJw+EWh\\\\\\\/tSR0WmTMu4KqEUjruCe2A9Gj4ArTT1FsmQ8iFK86Y6VGQ72x2gBLRH\\\\\\\/sO1H9yvvutK9ScG1+hwzPKPwUZBwo3TA+TJ6ak+Ecnrv4gcNlsb5dK8ZaXXAu\\\\\\\/lDIY9hdggSAigJmGng50CHvAnZvJnnT9y1AKgn917OtmHDVp8iO3blYFxEeE\\\\\\\/nn5gye1phSp\\\\\\\/bIhZ2h5Bd8vZgZ8chtQMRwdqlCq9MXixACZcUsSCgYhbbiBnWCoiZeWJCyJzNdBnykHciI0RXFSqhVwC\\\\\\\/UKh5Lfg2nCUYDjMPxQs6itbr\\\\\\\/BuMmb15C0ZTaDURFkSAtgeK+g81vKIAntFcf26gnp+fv9z5u6dOBOZQBkEAA2k6FAR0cdsD1yW43dq+Za5gLRSo48cEOQjwfVlqQ85AdLAzFy9eWDi56gzzO742nK+oZINvk1m0VBi6CYmAsHgAnyMbbfXAhiB9zftWp+DWHrGNe+8onGRShxv3aSdeDbgchgW8srO4qhS2JeDsbdEJrF5RGB3yPFXltlOmHczzQo6Rw426lMOuUI6JNPr3mUgdoYXswXtsDqdJY1X9E3ZcAoMkR12ur8SagBKjkT2E+af\\\\\\\/h6zloP0+F7RgoytKxgl2zKr7sRC9658BSliOVyXCM+NKq1lvP+D+RevZw8hUTLs2JQXXjA4Y5zo6CRJrqag8k05em0dkudXsRsKVE8xOQawFgU\\\\\\\/EEUKyZs4HQMtiDicuU3RxCRuLbI3Kju24fBvDMat0SnS24ec+z5x\\\\\\\/DyJaBI6\\\\\\\/H4d1QztL9Zmd13k8J0gpvN5hQXJ\\\\\\\/+E\\\\\\\/udo\\\\\\\/HTMUNqgtY8LnJclHeRH71HdG\\\\\\\/eQJ+76Dds5vMOCmgWKuhn0H\\\\\\\/+yRIviFeppUd5lA5LP+WOCO5gj4FQ55i687gzc0WxlpmnP4z6i2dzTrqnztku2eSgozD+nHWA1c0N+Y3ZRazRqvC\\\\\\\/GRern2DM+0uWmyJGr2JFxnNzea266cbksMEGniv1Nm2yBqRy0GeomdypBV31\\\\\\\/zK5eQXPdkP\\\\\\\/M64\\\\\\\/4o\\\\\\\/sjIzW6E6W8Uc9aLaSzpYoBw8HmWfFVJuK5\\\\\\\/ypbHyRNkMEtbpYMLTZkkXXQD2fBG1+k5HJFAEOo4rYKeHHcNhVK3a0zks3nlZdjYLZHaTc3D5E\\\\\\\/fH4m24TDrQfvT2OUZ6mfspAFvInLcg\\\\\\\/bvbQydeLiwQPAw0PdFEpDA+3q4lPpMlBmsypNNvWopSwf9R\\\\\\\/sJVryY\\\\\\\/MEl2TcgWAgPAN8OdvyBPrlspGBsCurBVS1Lv+pIwttc\\\\\\\/aFMa7N+oc6OvuENx+4Gs\\\\\\\/TfDewXbxpIKZ2jhsXA47AoJFPG\\\\\\\/qBTEIm4pn8i7VgukgSPF6gipPzpifJ\\\\\\\/l0Bil4tfArURhHR4Lee\\\\\\\/\\\\\\\/ZPZsVl3\\\\\\\/5hmMyPOnLJ\\\\\\\/8Kwgkecaw1AZ5huZ8SQr8wT111i3UDellt4mDR1mD78RdWj6n5JAAzMgbhVeIsECpd\\\\\\\/z4fQKLiw9naIfhK\\\\\\\/Aks024qMHBtC3dVFBx6IOvVOU8K1tdym0m1jK5lmlf4UwM8wdUtuBqRusBemUwKFWBdyjFghCb2eDJuGoMZKjVT0TdiSw6+JXOkBVk+mAxBoZMjC4XdvfpYfoE2ldDrUnZuTiOER5l7BLxxwR5SMgr2eNXROLxiGTKZcAt+bZGGt3c3fjz1WMcjNthm3xIi2qBrlZsaDpCrRXIb6F8vOEQy2WFwFVSNqqWTIc5ucGbqCR85m1sQXN3CPvcozzO3+DzhmwL9LAQcg5SsgIqD9y\\\\\\\/StzzAKlw0zd5fe0Oz+Xa5OqIXEO5dKYaOJc4LULiDbad+vl0ofKRg9+bHzA7klR0qbgclpr4iCv912RyyQyt2+KofXsu3L2OHKeK6jwP0A0nqlN7i8mS8enOfCLH1z76DfHOlRKtaD4R+ZZG8m1f578JnC5nkSs7gQcvyzU08h\\\\\\\/EdhU0ioVEfiu5uxnb+RoOb+MQ4QZMHElZT\\\\\\\/6S7z7FvQdtddEjeWhZgRz8vvLHpgwBdXDjGRaZ1ZJ0NEu5bw9FHpf2YU3cjIERml8q0fyU8k1YyAAIhYUZCLE0I2z\\\\\\\/CGKYuWAXeJHPY8LGBBiveG0kjJEmok+sUN+o4JAp3vbor8rCOsQjUWO7YKM\\\\\\\/ztcMgp+x73eyMSQQY+ZXJDrkJ7Mg9YDRUhBTyFSfKTKLTr2YIDfGnuKxxUH\\\\\\\/7iGZ4IYy8UaNvnN4GHqZlf0eDc9dxb6HDWOAFNizFHTxKQKwKlVIVz3ybFnOdlc8ohH1jVKeCvOzNh+wGmdaZI5Wf1qpsapfocSf\\\\\\\/zCOAn\\\\\\\/OSBTX0oYAsnOD6c5oxTiUvwCPOzCqEuLqN9502reuhAL73rLHvcLuchwzmT\\\\\\\/3PcHyzNQdQW8mWdSqFtt2Ljvpj9xMZ9ZKkmFgXrx1jCNcHBjFceRNxx+Pp88GgKd3igpNN2Vmr1R0DrAWP1xLspzWsJNEWHCCmlcGyAK82nJF7pjazBX1rbXXKwITMjdKjYjYtvp51AE4peVRZCwU10yMZMt9FzJoyiM6lzE+5ZkaT0FCcaz0M1HW+kHuHgeFJSMUMnLCefCsprWKTAOQuj+cOu8PPse36Kj2QS\\\\\\\/oSfz7zQMT2C8j2dLGJv9XwR3JSU8t83xWq2yOP8StvD4j31kwCmJbjfzgfs1mc0Z813wt+SeLJTUEPI+3XoPo6aQGF0MQX9oh6fGhU+0c93eO7KOpeMT7HgjyyNreaI+UI20FkSRBdBol1h0RMadUv4BNj\\\\\\\/jeG\\\\\\\/Khw6TJ5HyeM5PTeM4p\\\\\\\/pd+\\\\\\\/uWw5JSTBLAtACmvV5B6EwzlcqB+7bPjIkUy3gWAx1piTGlzkIMXOPg\\\\\\\/P0c1KVLhRTI4WF0dui\\\\\\\/7CT+iZbe9N+hKXcyTzPGbv6k1x9QHyvDpv1tQa7v2WMxysNkTulcZa5ENgKQSlBY72MQKNbkAVJKiYnkKBELh40Cl+Le4o6eo79+x+6BfBbjhiBTn4NsLSBMpIcApUfP9nadwdORlgtX0uY\\\\\\\/XMySWv6Eu\\\\\\\/srb2id+9HMc58DjWR8VvgA1kbeGhZmLzDn\\\\\\\/oSoXFuucorRMSssfVEMHiP2UizXl4NQDi2Ey+ZmkC6+b0jHoJHCi34Dpu2WPXKXe7bhvuKinZyG3kvbVZjfVLZg2mTQbr3L3QnDCfZAMUMbpdBnnJfgATiGdJVmTsp14RdVJiNrWoYiYw7r\\\\\\\/3ChwMlfZOuzoFilj3LOYJ41SCQRyCq8Mg0kqRBY\\\\\\\/Wnwd0oHaxvnhWO\\\\\\\/F1Kuc08fDh7+\\\\\\\/mPnD0zF+p42dDCJLppkzGR2nJsRTj1RdUpabdc3wgxtvCGa\\\\\\\/wx0JTVIsq\\\\\\\/3Ns2Z0ei2GhGoFrW7tH7BHa8qRR1xIIeKj7J+PpN4q9g5w2JiDvjjkitjF\\\\\\\/JjRfhLnte\\\\\\\/+RzcX8UVW45pJeVumeQe1aqaCa+Hk1VrIpGqVbd6ejYfm\\\\\\\/85G91GM19j5zjFbphO8f2fTIO9q41vbE\\\\\\\/UT0R\\\\\\\/adfzPNnxmauQwfuNkIscDJcTr2g8oN1kUdCQP+PDgsooid+3L\\\\\\\/AULRGzUv+DBLGKvhTdptuq5S7JJ+Ff2ADLlV7t5tviXWIptiHBQo7YPfXP\\\\\\\/QOgdv5R2rQsVLOmZGK3Co6is2ssthx6dQ6YFphNAb2ml9qblxJRxrmrdEcjw2Zk9iGAN1x7hP69UrXR3t65G91EFoI8w1aK7Lpf4vhe6cA4aTjNsT+PleLHaHE5fCd3onEP+bZxfiNNUM+e\\\\\\\/JF8xWNyKoWGTR49c9GxDjFeMhDu\\\\\\\/LWSTF+jko5Yjj2Jrh3cpcEGA3HHs\\\\\\\/dQSxCCkTEbgayB4ZaDKbcdN21yF4BIXAZ+Ink9i26uV7uoUdzuCoVEAxkxz+UsEaToERUGdbE02hIz\\\\\\\/B4tYJomiY3knqDQmg06A4RswucbSQr1mso5VMIpO99W7R10l3C54Q9ADdxPZX1NHd64vQA7pwtWjhV7Liw4o3MBPeNVPWX40tEK0Ws8F+TD7MRWuDVvIMYRxjC8r3os6BizpyacoDONE0U5Sq3alKYSZhdJFeWdb2le3LDk\\\\\\\/phiNFgX6CGh1knd0jR3VKxpbUXxtZlVvOSP84E42oLQBLqXpxxjtjLd76hJX8nvo3mh3yzJIeyoZvfueYsz+CsDo8f83XbJy2bnRww1ywz7bYnUFPCRSYxiGCB4uZBpHa3iIc8BXvFf1iuONms8Fmoelltmk5xM2gfhew9NXLGgjDeG3oQH8e09PosojOZdRMWBkxOYeZoYmvnWa4+liyQXaBTRYBvD4MyCdQm1b4NrlU9xM6BR5nmd3SQR\\\\\\\/jlG98octSB1eSPCM0J02EAZMs4lCChI3BalkD+vTH1uPyFqOn5ZsrgDewTWsKDmN5Y2VYKksR5xrg31FkHDMXo7TZF15zlza048pxKpXyr+SiE\\\\\\\/fGllAaTiaPIHTqDvtP+XUxPLfpwfhdnkqroKF6GSD+5ruOEhKwgQ3YFodb3sXqWh9tjxW3hyoLLncrT9rm8262+aosAA6M+H9QB2qkUaGP0BY2jVq3DEFJnQIqBuFNBrgsAsQCGSUpu4eeNU5GLaGHBBRPImIl7oIthh\\\\\\\/JIuK3OITK\\\\\\\/6qNP3EEhPdPasAgCDEMJte3FaHezCERmaCiBdw1VJJ9IsabYorvUelAI1U\\\\\\\/RghdbtO2Ej3uR4xafc1Q7xVPfirYdnHR6XRSQ4xZm+QZmbDULSQf9OrJ6GLGvGHKC+kKI7VdEPuTXkWkCIE03ZIPROWH\\\\\\\/eEpr3b5f4n9Woo\\\\\\\/9SczcjbEcPsJA9ckCU7ogGxNHy1lxX70EAzZVeoBNjYCMLaK0uLJw\\\\\\\/4l+agaCg9RwoFjr8lsEqUl+UoPOhwMA0w3QOlcfT2BpoH968OA\\\\\\\/2cCYFqOv1\\\\\\\/rOoSXL7cXzxZ9txa6q96wnjRR14OwnRd\\\\\\\/HxjwOiDoh58GTKas\\\\\\\/vv85ufNdt+uj2mAqS1sM9Ov\\\\\\\/ilgdVsy67Xl1FtbpdMJ9HAH4SdYFydyWKx3D9bbVSXLuIC84a1wDxZZrR3gFp7BWYVl+tyKMzxqJBuWOFWNhlFQoo\\\\\\\/2gLLh5BDGwobSCy7UooJkTCkmDiajZg9V16SDnXqEVmq4N4l0UAXl9SYiK4RelfhFv0nYbeKfJTHaxmWbKUemRX4tccw+c7nckRQmANYnhC+Evq8mpYoVAzvP4MJ2kHuyFnarRFGqLtzsSJMLPRgOsxasMWRKnnwNAMyeeJ9kRAwUCnFvxxA6tmik6wk4jce9S72MkUwPIudUGWRGN2I\\\\\\\/HKfHSvo37\\\\\\\/MIhtv805M1vjycdOoNr9Ef7ER\\\\\\\/fvLusm\\\\\\\/\\\\\\\/lVWpaWOKXsrdTU2elDNy564FRPxYlQWYfv9\\\\\\\/sgzy\\\\\\\/ewi5JPfsigComp9B40khmaTqgk4dvxI3Dz3DKD80dTzra7T5aYEV9TnkbJ78ogUNY1SCd4U66FrvAUICJxDHCKgGj7NQiTK0sTtwOUII0WTREBPN7IMDhdCJ2GQVkUhVVmqB80O6pkmQcwNLoAlFqe3vHK46cDcS\\\\\\\/VLrSL29NV9NBz3pg1g+VJu7OSKC4tqd4wUR99a9+N4oXFi3gWRiTwC+CNCaUBZUS3DJEwLhhFwqupGAGbOIgGvATk\\\\\\\/c\\\\\\\/MXYMAA\\\\\\\/ZwboRLGvUsZTe4G2ZGe4UDMgqIif+3KpxE7m0miU35YcHLhVtwqbKRWkCMZlBnqemHxl1CXPXevwXTvCEd4B0\\\\\\\/3NHel78kyFEdvdntn9i\\\\\\\/nazydQZly+gGoBBfbqKM6d4ufEh61136RiAqQgvIMO3C1pJ7SXtqCztVAPwG2HVp7XT6KBKtF\\\\\\\/jk8SGkCpfh7SiOYobd4zKhbWuihNM8kZ4JAfwBXU2xv3g4L3GC3WFbZSwe83mwXCye+wzv0bitKCDF6c2pLwG03fg5mbBcLMf5j1czKSdzofkIPDsvhKa5vR6GQpdJ92dOYMH95ioTFJONUfe0V+sw\\\\\\\/80vN3AGN36qVvrSqZP6OdAFzABkhHmRn4zKTlKkgitxc5LzE5ZAWHCOTmlelLpDkwRBpDceHi4RBUMrUQvmQKAEQA9rfDOZ\\\\\\\/Y+iTFPQjA6Jg87bJwzzE9hoMilRY1FF5IQsD9tdABEFToG666TQUdmjSMHCEFvply3SeZrUsiz7Pe0ftxG9iR1ylHIUR+QLsV2ApS1+qmx2ABmE0yXd+4KppvSmw4yX9h+SuN0W4sAwzXvLY3br57z0xm\\\\\\\/eb0Pze0qGKgiRXCos26K3feafWyHESO8H\\\\\\\/mqy1haaasbF\\\\\\\/f4Q4vFq56\\\\\\\/7VsX4iAw9sjUL6kVOWd2bvzQlgR+qvOhVTaLmQmVBAGbi+NpmXzxop5QsZcLFjInIepHMi8OuSTfYLuQV8dRUnK4ZX24nx\\\\\\\/wMpkcCG586LyVhdz8wWnIYBHjj8Ksto3TFtP\\\\\\\/afV9B1XXZwTDN3TWNiMLThRiZ39hv5DGKfDMLth78N29p6eRQuCgPH\\\\\\\/WuOxSdWcVfWrzKsiHFCim\\\\\\\/MiQj0ppVwkene\\\\\\\/uOMUCIfKHFq3UvZ4EFGpLGw\\\\\\\/OdeVPkFgw3BYvteoxp2b9Io3NkJkeqb91C9CZWx3l5+NTo1VnzkHJyP+p\\\\\\\/0ULCOyArYi4vxdty9POLsyHeI2YSTEVtXQvgFhORl9ylkTHV32Mub3wptuETNwgHr20YNpKhRXnluvkZQQfVSPtKvRbEN948XrenNc\\\\\\\/DpExSP7+eMz\\\\\\\/YRyMOegj3yrb0Afsruta14miNCkHmY47N9xwuHHXBQ+UvGskm6IqfNhDF2BxxHA9fZ9wR8o9OT7PMvekI0MMKqJfxkfXagoSPIWV4s1dsLfQwyvSKWZ6vg2ax9plgjt8HAxhVTr56vDejPwEIiS3GThpSDj4IEGongMB5ZBcmwpt1rdHnjFEIs7aBNWuuxpDjtIhqzLgosZLrI7qibDqfI1GmXmzFCvVGiJ\\\\\\\/CCR0D0aMvpwm23ERr\\\\\\\/3mMnRoGZEqv0YLYkht22N\\\\\\\/r52dYb4J\\\\\\\/jjSBPeGYw8tMZgvI\\\\\\\/s9RGQxdfpUKHMAqM3+\\\\\\\/GCpujn21Eain7lvdjpCFqqEi1AcYAsoolZUPZarZtvSJmX8zJTTbkPqTSI+aoEvLx3KUbDJoic6HbpaD6V6VOYkVs1WsYwXO268M\\\\\\\/wXooFcYmScANzbbf2uJ7b\\\\\\\/G0lnf6htfMR4KriN0PA3xgdFoS6GUGPN8opoyLXFrXlQiNq5+8mnU3ScsZeukjUAseACoGM5h1n8nkO2fDWUtj5ogbjRuWtfjlSVyMxGJUFvPq7xHEs4vEx+gcB+m7x1Hk5W1X1nOkS47xRw3k1Sagx2eeEm6uKLw43BXzhPaepHg1ChS1v7qyT8mE4+a3hZuEsDTYhnZWswfYWSd68AvsuVLzvXWo\\\\\\\/E+4KUKw4QH3bIaQ0WmtQJFq2rWX0MgvW\\\\\\\/Apbs8yDgGOVvPhazSW9+45SavWEpjSJuWXyvDOh6fvtKRR4PeRnUQeWyISpSY0xKe0ELvlZFWRUFtU+ZvX7CBaH9VH6Y0u1BM0vh1dXNPLHTdN4XkvknhtPlPk0CKHc3HyREPYoJnPvY\\\\\\\/7gxlvEf7IKz2Sb\\\\\\\/0rFtMxIkf0OAVYv8sFi1chHI2ihw6Dw4BQbkL2owf+VQpnqdumLq7261RbF0heGXpOSu+dCTeiWAKQBQIxJYotU629uAMe5HqGLHBbOll6Y06hXUfYQ7zMA7S48M3N75CLnkaCtPZui2WKt\\\\\\\/\\\\\\\/0bbvX\\\\\\\/ki94Ctp8IngJzcC7uJ6uhILN9iaQ8zLTsk85WDnlGG2J0wWSntAiam4MwWVN9fmQeSqyrYz7BNh478PZ4M1j3b6p55Wu4yuw\\\\\\\/0S7eIPeNrvY93FQAIofeXavXGPjdpcgPEw2QWRiiD1ePDIeiVR1oZa23Ws7KjPO6eEKC4X0kFPm8PlDfZ8+tQeQ\\\\\\\/LadZqQpLgdW4ezKpqv3PeSYZKRaj55Qex1j6mnHoIWiE6HxulFb2f6PmD4gdC\\\\\\\/\\\\\\\/+pEZcpG2kXXOZecogDK+oQd+ZgDWfhlvKBLHZV9qUl3+iCrgKwh26xjKxANGw\\\\\\\/lreK2Rm2UDNmO6m8Ul3X+YrjIvqIyvumkjm2k+B51NWnDnx0seHA+q\\\\\\\/sN8JHXGUbh6v\\\\\\\/1sAjH1GP+poUR+06coYZ72j82MaZC8CWpVqYukj1dVcX5GGEPY4rAMCqQHW\\\\\\\/LZN2DodUzLATQwHcYudBacvX6GHzZBMTXHW8+aNOnzFVV0dwRL7YF2+PFn9AfQVsxOqOV9W6XatBB0ssZmJcqL22jWqwi5Yx8c2\\\\\\\/Rm2S5b5Mac3RPlKfy2gm5fgUaN9d\\\\\\\/rAyHlPnfzFNWpm61k6N6r6U9zkFRPxbsH32EueakIdbikFVsAwK8sX6RsAewRkgMXlZ0OkLTa5BJHc7IRcLAeazEfGWDICWbk4EORwrxNmONW+6dx2YkrE8f9nP02Eytinv5N2jX26Bnrn3geAygpESLCtPU3ySVHBtmeZUX1poAxtwKFg68+pSIktpy3XfpSaT84tOTIIitEv7r503XFS+R2Fo\\\\\\\/LuZ\\\\\\\/HA2eQMOeRx1CPKjNDkEUQr7WXSVeN8kqqd6CT5pbasEUASNVy5dJ6Zgv7hA9k0B7qC1HJo1pW3QlN9Y2FRFMh5W45NBQ7hAE5adC1RTls2TyXPU\\\\\\\/PCdnVlNQjnVd2SkfPVqrXQkxY0TpplPYcl0AxMpVg0s7ebv1meDW+Cyip9vkWXk\\\\\\\/qGXUQgDP2HoXeN\\\\\\\/B3HpH51CPTLWVNt9Vx\\\\\\\/mt8n8yvk9jm5TxyzrSzFxpFjjqSv\\\\\\\/Wbp1LKWz+\\\\\\\/PLdzgbliCiI4fTrUnToIvXABP48CojFWBDxK8zbrmq3Da2xfoBCqaJL5aOfxOa9yspILDrN\\\\\\\/sUkofrf5W3xF3\\\\\\\/dxwXvMS3wipXHxXGyoX\\\\\\\/KRJ3RAhS5l8nClz2CQXxEubnNOcWEQ1M3NRi9qDzmYddYOGpeRd87BmV6SQism69jc745AwMyMnrITkfzBnJkzUd+misYGlA+iILj0cqsJFrAWVCzhewPIGCBhV1djYtRcyDvIV\\\\\\\/wdTFCPH5nE0VmYY2LmBd4SHk+ZHlb836VktVq819nohE2jCenM+nx7ns7+rmgJuFrPP\\\\\\\/ACtR\\\\\\\/uU8INHYZurnNd8ZaaZhVC\\\\\\\/X4lHatOOiZnxCusY3pbQYP+oo4BEuq3ZaCfUok2Nxng9Wj4C8nUIwvOjSpqHuYrcbTN94EroBhfWKQm6L6Oj7EYhA+4i2P2U8HcPnJFOUaY8rKc9CcnHimqkOxGPTQgB0BsENscP62w4Ury\\\\\\\/FdRvtmqAkNUuTiqKzLqV4tGU0rHU2Gd0HWgjMkRsVfBjOdlTnPlKBNqaqD091TmHmw52Gq5Zz\\\\\\\/3mItxL6vr2cihG5ibhKk93PLop3WTLgD3HmrURbpzcppT\\\\\\\/tJjoU4MVgrMsO7JBizim7Vw91ITfZwNa0+QNKAZ5TEus4SXSoRBRNOL+423\\\\\\\/iDBn9+5iztX9EvVNVPyfv0oW3IB34mhi0V+O8x4jg2QE8CUmQH1p7apINq1yu09G8moBcLKseMOP9LJz8xzwa6kQfMt3oIDw0KbmHJzSskkksgIIeiT5viq8KqIcijeMft0JV+Lt\\\\\\\/zWV44ynmx2fkNhAKVkfmfx5e3mBFan86WT+Wdv+kLVvS8D1mAgT0aO7hQU8Q8MG14InrLZMmtvaIs6FRIwQlTcUXvyPhI3C4cWQQw\\\\\\\/ak4lUXm4xN1qksA+3Y\\\\\\\/2XvR0GAVFb9vjRcTo+clNDDBqvcP0+lbZjRhu8llkcmQs4\\\\\\\/t+hPknz7cHWfD\\\\\\\/yUKX8egP9AVsjfuIwoh5PVXahEIq3uioXS5CrEZJ+lLs4ZY5n6LWVwFWhwChrU+Rjqd8JHFt+0O0xM63BCJu\\\\\\\/G2NMb3KxFfPhQU7PPMTApkl3FWX3hc8nyDxMIlolbYzE6pBsiH1HPK2WCMAREf9gB9PQRCKR0CH1Y1xGsllmQngOnRCKIkADAiDNSRRtU7W5pBOakO27BkicRMV9bgXIPnOBa7OtXg5X\\\\\\\/vTdWqoEOK4ptW+bJIDNplSAJGMXvt5fKnq6HHNvUz8PXIRLKOH\\\\\\\/QsYiYsaE\\\\\\\/UlcZYbMCNDFQpZXRhXoUUiGj68bgx3x8pKDkESG2RpQBr2xusfiY167N+EHyqzPUAIbjMHgkpG9D52XK1ArZdOIeMkcjYM5dUtV+C\\\\\\\/Ze0tKBR\\\\\\\/gt8x3N+wLc9veoEB6ThCczGW9udBWEPxmDKR6Cb5yQhoOvSbB\\\\\\\/kOlgDdcoVJFbLk1nTcN9Pk\\\\\\\/vOt3j\\\\\\\/G3PjCzLcJZZbTuaXr1qfgLVOL0eTwt0\\\\\\\/WhKubwvvmf08eY6nQXBEBVRwq2bK2ytjxndJ7dTuD6RM5M29g+it836A9eFtGSPOJ5TaUGHoK8kCzZLGVKdHfpveqtQt6QWSSNxrXQrpt+x1QVX930mz3XKrWgUZiiy0OEPoIGovNDwwrDgUEZeFo\\\\\\\/nInuHNEmVLWZ8jn3BQBA1dvcX3kMGzq88rqq6qmyNo024RZ0F8eXkFxyt82UxNjeN0PUkdGXcPOh+qCzG8gXVpcMAFRfcC7gpAMn0EAH\\\\\\\/j4xwlEYX4i\\\\\\\/vwGyMStDK038+SgjlmmMh\\\\\\\/HxP6m6rGf6fIOKZVceFJw3ZsvkEBEgKvHtsnw97xauEbe+Ob8nd4En5FPEkNmXZPQR63fGVi3L\\\\\\\/F0kmYgzmqwFK0\\\\\\\/TNhPsr0xkCWsBLFxl1RQJtw4F0v8QIOXvf\\\\\\\/4JN251li5OL\\\\\\\/y34frxGuaxVI\\\\\\\/Qv845M7H1OuQ4X9UCMKi3dVq7U1E7DAdPxGhQ1OQFceHok7Pq3jL5EdWG1PqWH7UiT0vN2gL2AFo0HmJTySoFC5GE6HpO\\\\\\\/lshuA6vNTJstc\\\\\\\/fEDuoRvtxZti1PLO8NCRLCCyHMfNpkZAxGmLT8yZPI5mv8PNjyWmgZk4l2cCnHCuZhoQ1itbqvQobRWbFEbQhESYhQhgeXVJfORDMckxVs3eskN9PhbJezs6woBzC37xFCzd+xKm9PFYEvqkTWRYQH2eHmy\\\\\\\/wLTvAgs1uG0mzOxVG\\\\\\\/WVkZkvVfRozJqLKDFZ6LRn7VyiloK8HJmRBrM6ywdD25POrQFLCYTwXfdejrtyFNwWOQD00FtWN9f2NX6r1jrXXAIvddu+TFU9HQYn8fqUltbSmCmiTe2vLEc7h2AmoP4joJf6J80zugbjq1bSc4BkgOgLqtxKaesJV8nccWb7Nl5TBEFZ\\\\\\\/lvPK8l6ldOMs7giHGsFENTRxE4FF1cq3Z3rEuPc0BNWTJxKHpXHEEfuVlwlA5qcF6NiiSjTK1jH0su8QCrpiUFxssEz1\\\\\\\/WFPGvh193ZQneLgiFa9zNyZFuxQLkOp6QDRdy6bDeygh8k9s9P35iDjS1rrf2z83pe1q5mut3C6pYzvPfexl1QvUl0hw2+k8qOv1pIWmCrcZQBxNNTACisCzRIgoOTA1IS1oWx+EIw9PkVK5tCLSiidsr9\\\\\\\/ZF6wZO\\\\\\\/FoNf3f5vvosSe58IRt0RDx4Npsga4VtGpV0AUSxHwukblZBHyrmOAVl8FslbcHyFX+8qZam66CtlVN9aD3tZCbIRu7KTZ9OfGRkn+h5AhqBoOb1RV9V2joOMmDRHY2SD9sKjkAll3R2U7V\\\\\\\/bEuAMn7oLPBcdX9el4n3kaqm7\\\\\\\/UiSiWBN3950dqIDisYQ0nsZ7ubfaFLLcMcyMnTBBInBMHO1macJJ3N5P\\\\\\\/BzX5FFz3GUOoRqn4TlqsWHYOqnxnBVWiZWIZsqBfEHLcXJkm0smk8dbJgq24okb9eHupjjxzmz+XJgXT+E5EIiAJyZjNFqnjvY9dP6lZfhQSNBRrQZaNSLHCl\\\\\\\/NBbFCMt5n\\\\\\\/ns7J10OOLZqMhHfEmJ44fjK2gde+rnT3qcNvPHQNZmj9FSGsM8IA+H2irFPPnc+BbopVjx6EQYPztcLvNhz4j6hwt6u9qOHSRGXYeslt2jWY\\\\\\\/TiNQIzsTgVdkxbVXw3iKqJIerULMvP0fZGO\\\\\\\/J5B+8UGUpvfEs\\\\\\\/z3WzlWXmr3Tf1\\\\\\\/SgIkDh5+b2X4mATcUglG\\\\\\\/GvQdf+EViWWuzcVRLRr1oF8IPXomq9x9ogHhFUgN6DFYRp4m\\\\\\\/Iotg1uPbz8+gnIiPIXCdQTcJ\\\\\\\/Tz12fkM+sVhqw4aqBtGSp+4O2GeWqXYRHaPpzo0vdQ23pnukVnot+9bjYXU1HS1DW33xBNCvjLRnm01shTafODvIY7pS1vLHOQopPP2LKSOAT0ccXTdh+nvMI3sVY4Z\\\\\\\/4gIUEs426sz2pKVOhc83ET5tOaKmWyAUcRK9QxJ35Folkeu18xPkXwgd6XYN4u1AsiCzNvmqk0duYWN\\\\\\\/rCk+CBGP7BusNt6X\\\\\\\/0+O7TYldCc\\\\\\\/nxvtryXgo\\\\\\\/AJuCgi6oi8t6sFXRx5DpDZ9UAB5RPIh2Vdrlq0bZheR2Pf6dasUecdDZovJdg6lagONjdZaBA8vyV+uav+MN3sBSCAQeFdA58CJTKyW8+DeBk\\\\\\\/ojP+bF1j0vBVe8q6VCiUFGSVxfCyN5zw3vW\\\\\\\/\\\\\\\/HaAsV9YbLZhWL0rAFy\\\\\\\/GDV\\\\\\\/iGCmZzNdbYntzDBBD3XviH4i85MLFGUkc+plY35OLv9JAE0NjpxZLsnkx1q8FRF8DmPldTJBjml\\\\\\\/GLMw64gMSmtZnT\\\\\\\/MLAsWqw85AUwkZu4cpCJNq0k8Qvp7fDGwV\\\\\\\/c+xThSqCj2dTPNg8wL99j7VbGA0kVlp3uXQjVMFnSM4Y3Yiu2W34Lilvcj4D17EmrkjYmW9oi1ImImmCipMBUaUMSFgmHjKvjqc02cYy42R3f45r2K9UTCq56AuFqmoRKERvnt3G3ER8Vs32lp+Zuw\\\\\\\/AGRpPMAp7ux50p+623uy5FPf17cV6xhgH47s6L5IgzOC9YL0\\\\\\\/3LStAPTHAefd3Zo5w9BdM+ki7IZaddzwU2vJAhGajAAQUcjVGEho0QjVxiPWZOEy2dXW49nanV2StV3jU6x\\\\\\\/ZP9DmavcGg2hvqR5WGNnZ0yIke0DbbaVBMCzqJK3FQk3IByeWeewOBm8IrZ6X+bews9VHic+9KD\\\\\\\/mQDfLr88K1dtat3VJdSRtDV8XbFEyE3tzwTJEWn50CDEY9A0rMmSw3htuRK3367qosF3wH2l7\\\\\\\/HY6f58x8Sta1D7kquWQpiaY5UVGIn6UdaBWa2\\\\\\\/VbdfsLvcc2y5aSAM3WVN2FNe74rZWi15LNxiPCYRdWPfrhlH1Mmfv5wFhLLf3bQy+D8u76ajyWwpYG4U22pOduOFPtKW6jRY083A+GZs5b4vLQWa0Uj5AcmhFYPXnRMbYRJLoGKOpcRyg8NeLVUqEpMiI8YASXnra8YhnxF2GRnNtxgPlV664nMPJh+YnICEqECSGSMcuKdCE7U0OvxkwijbucoocF+USQlDSwkLdreX9urDLWn+hdnDA8zqTMkKUvAL5KrB4HqwctDmKjhZk9zWb7vNke2swa\\\\\\\/kM\\\\\\\/AUVQRHS99kzo0zyoIci6\\\\\\\/ecBFhnmE4erWxMcYP8Di\\\\\\\/FL\\\\\\\/W4KkRVwBkQGOo7n2ZHbigR3+UJbFIyTc2fHHU5fNlvcre8PLBiMxGqAz\\\\\\\/j6FBRVq+cgxkoHvO\\\\\\\/QBUUwqPaZ14DdeWeVw\\\\\\\/4YToOedJga8M4omIZUg+zc84JUq18mFTBG\\\\\\\/Xy8G2AZHorc6XdiHfooIuPA8\\\\\\\/gvuczepOb8gIIhPR\\\\\\\/cI0b2R7O9w4EeVLs75wVcZNV4uK0m7jXWVvLWpYQ1qp5wct3\\\\\\\/ovtLd9FLyJf7f6S8TtQS+B44zaHmdROuxrxcDXvTUhPZbfKG2la1\\\\\\\/EytKHaP1BV1rQKINKEQos1VERgk29HjTWWENoFSRFymOFKHByHBGTkjjXodpowQamtPMvdvy1F3wO9R7huAIQotN+T\\\\\\\/8l4bQUHvcKwHuxA9ZIszvsrACZYLfOoH6MhdQKCbvYNjrPIUUbbGGSX1bqDTIRG7dpvIgOK6CQMgmNAZE2VuPhZ9S548ndgK8j2jFCjcziiYkgjFZ62NjAEQXZWhiMf0dxelwZwEtB1mVaHzPsUVyRa30UtmXK65XvANnvHrkTadzj12HGy5bHr92P4Ue72yyy3XArq7rxfDWjMnqsGqwAs35rX+B0ePHNLOuQfzRUCGzclNfre3L8bMBmNxZs2i5myGugQ+EufAWOMOzbZ7QRIxlA0woMLtH7JNeGXfLuL2sRd1nBXWmR6cCoWlNV\\\\\\\/VlQIRb+vee3YSXYOr1nMjepnVDmwd7WZw2ce34vkdlx84RzFQOFY2tLcc98ML8LGUjMbpf0EFFYCO7RjkWS17bwYDdxAj7vP3wtnKws\\\\\\\/5fu9IYVMBYMMdyXIYylvr5ITLRqdK2Mits3e2W44+ewki2VYR37H3sm\\\\\\\/g12GtCnA8x0+yYTe88p9366u1VHDLRicKPh4FJXyfKzQOea9U0fCNQsyWXcP5mWh3LiaLSLSm1\\\\\\\/Sb4T\\\\\\\/hpD5jo8ximYHBkeWAyV6RuyKj6m2w9rHGQkAa77hQGCL1o\\\\\\\/Xq0tviazrgKgP\\\\\\\/FOKurDKy22vQjhfsioID6B7nk2JFdMzF38mk5h7sOjsU4HhEDF6GirBMtcVGlv8X9daTKJS1sPH6LTMOjf4ew4wvd59p0r1UD2aKIc6phj4PODK0jaQZn4KnVT9CCslMufITeKxNyhquvLVHU0DcvO+5O1rnoR5hpEL10RIGNGzZ45ienRohxY0\\\\\\\/3JiS5rDYpnKajCTGxZEVRhVLTegJNBwddcD8Geo9snMZQ3wiZS6cgsBtwX2CqMot94iqxIO8MLWps8EVsEE5k7iJE5RrkXh12GmaWy3UH\\\\\\\/hYxwkTCzmYVLdTRLfemjP\\\\\\\/1UDSTqHSDKZqWv07y7BWZ7Y8QAgDLjxk\\\\\\\/EpjX628I83ouhDFA+TqX+hnrzpaptQULX5a7kdqBXQSy5dF3B+lfu2cz1kyJ\\\\\\\/3uNoCODCqWZzwPAc2IRIknB3\\\\\\\/a4HuKhTe8vS7IiFhjsgmJliyP8eReUin2YC9rMQaYY5cItnvFZDQd0ouH\\\\\\\/swe33dmNPZWPt72kko2qOpBtW5JRLb\\\\\\\/W6EsM7tC9EjMmKeJePlrWX+LkmyqbH0zIcXGgiOeudXjtGUUNiMy+C4bKt9k4MMw+eJy\\\\\\\/hciXomjwbRjDNhQZI3dL0zbVfjx\\\\\\\/qUpDT3+ptvN+oUQw+\\\\\\\/MmqXkPTdVN4Kok+D1iMuTP\\\\\\\/HorGU+m3xTM8+fOpbly8cAHemmJFyEt5E8gHaZqWPjfr7b1G3wvfgSJHbfEdZstr0\\\\\\\/fhCMlokpzImlxiYz+yvGJHrkDsi6w9D\\\\\\\/7mSLwLAcEn\\\\\\\/zF1A0xuIY102lZy2QyOCkURq9q+qknPoV+C3gsC10exYA5hmLcGqW0vCA3rfFlQPXxoNoEbxA+Rdk4DW7clbT7\\\\\\\/OpJz4QBLMMzHgpp\\\\\\\/\\\\\\\/raZkpw2l+exWKnYof5eGqVnYZNzz6TeVyEpqlgR1HC7JGmNl9LzS7l4ttSOcWCsz6pEsc4PP1\\\\\\\/bvL0FbqVMOlv47kdRDnggad6znHzQ4PD7U5\\\\\\\/YdE3+VdCw0HywhsrcKgkpOgyvYNBupZ36pAyVu0Ujm0+yezRweiVKmz+WxJZ\\\\\\\/FIWmyAsZmmQ5BMHVKVrtnnW\\\\\\\/ELiGLdgC1AgEshglx0ybPO\\\\\\\/n4nOVZTBzG2wsk5U6YfWZi6WDltjtBC0XLOVgh3dRATfjQSEZeXa99T7Z9kZW7bUrL0pboWag1nt\\\\\\\/LlUnRksLPtpTFTSYSII6I6EryqNa3\\\\\\\/wOFBbD+C8efz26+bXcaYzurcIEhq5rwOKhnlJtf5\\\\\\\/kPD\\\\\\\/M16RZ7A9OFldQ0Nur2lhWRy90KLNNzY4iedCgkLpOQkyMWMgHM++cMaSHK3vwWfetRCzx1BWWJmOUnOqQh74BJoqX1WVL5brqh002QzLlinuDysvAiLQgp0tKRxudSROmEE18y2IVzSuIYBJYWmt\\\\\\\/4BOb+jLh4LjkMIMUdNhnkSd1hiwvk6YGfS0ZCZJZ+9S0sqOJULWGTXNrL93zXeycD1lfi+GtAr4hmYDG3FTLAeEKOk6l2ysF209mTY42BXgysS7GCusv+PLjLJJNdDr49AYMBIuiNsj9NzcqV2NdHN8RKXU72BxG4VKXfxALUxCOFd7uhSE1A5bmnWVJIm\\\\\\\/0OrM0DA7rKaKudE0S1ddfjJTAg1Q4Jg\\\\\\\/nBzwGkJVMU9J\\\\\\\/GheImigq1LOFLEjf3UXnvI3udzL8Q\\\\\\\/cVtS7I90Auz92rjHjDR92krfcDCS5zDcGUET8jeQbokQBbDKd5wbjBdqtctzlb2MIxy43xoSqhUrzAO2r3VCM4OMF912NPp4QJfEwZGb\\\\\\\/wHj0fVX5UrsViqEkUswFq4ilAUX2pSHsKGyr6XOjn8Tw+thl5kXv5VSebdXZ8sBkQWl4MIBfIFxFEnbTxl4cOubXNL\\\\\\\/bZWzu+t5WIFRV9HItS+VfuTpmktcNMdniMo6UD81LlJw+WzHVv8HyMXWvxDOaG85oiAfnJU2iPtuprGfvK93nFwjiOfBr091W5NSXuxwhukOiJyvniTY+kTxdx9AVRZFySkYDg58Kwu89k32t\\\\\\\/7QrC0amU3kImS9pRH7Y3yajpk\\\\\\\/+s8EJtVJVoP0lUcedY4PcYD4UHUlRei0WO2YRzUWTTUZCGRYGhsV9zaChJpzHWga9sdrucH666ymElrfYQWsoo7lezh9KDAWmPh3+IkXhA0j8+Ai5CHVQhQCwrQ9RuwZSMshQdueo+9R098d\\\\\\\/iDx\\\\\\\/MCL9iVZtFetDV3TyiBHvkvg2M\\\\\\\/JRXgPOi1rUKRv\\\\\\\/7nRWtK+xW8ROemLd9WaGSZYq74tszGwNyI5kZxjqgkXGy94GSAacuNcuN\\\\\\\/xxLgzsPGBclSGmgMy1rrLwSrfPPC1NneS8RAHUg0gHEu\\\\\\\/\\\\\\\/EJ2rnnxADoB8SJ9lw7cGD6flTHDtlp7GgIQ2zsG91B\\\\\\\/7D3QV+hpaHpb10DxzAtoSaHEe\\\\\\\/DN2Ni\\\\\\\/yvc4Xza7eA+XUh55OMj4Ebtwzn7+iqP8QuRCDAeQQQqAAlddJs1z49K3KqaVv7t+gpvMRl9hD5SqHDgZLnzg6acfHeih6ycvaWffQNU37fEo1jYASLKSOKNacQeV9FP69cOem5ua61K139jqmMT0ALChMsLZMAAfqG2YRPZxvOk9mZDWr6jU0g6rYMCwyUUqrA3S3Sikul6aC221GaowzY5K2MyUVQiHHH9ESqyDq0r+85xBBCQA21S6olKSt41DpIekNsPbFh0dhnkildpSU1fqGp\\\\\\\/vfE6TQj4+zzlvi\\\\\\\/MUzpVagjqYj\\\\\\\/Uwdo2OV5IYPQCREUpb8xB2Mt7Co9gzdT1zlNM\\\\\\\/O3PTzBvRHSYUCUFUlZU9zlnhdDXdTqRNW9W+l2BuwEcv78Td\\\\\\\/Eizm6D9OiZq44kXIE\\\\\\\/S8yQ2jBSbwECYE\\\\\\\/b061m5yiQzrwmSkeAsALMMpURrQNXOEzQPr\\\\\\\/DpkAfEf4XAWLty3XfUwERd1EFYvuvKWtQS60gZlztExaoC\\\\\\\/16bk\\\\\\\/9y1D6a5i\\\\\\\/y3YqsOZfQHJzrOEXAdNQU2wzRMaU2c9Mj\\\\\\\/Ik6ONkI5EBITcHEzX5IzBEdEhZmM43H2zjdk6G3w4CIa6YqZdDqWjPVKEqQ2mlwkSxAs1wOYYtIZRqLwfETFcv6SIudycYf5ORYTaJxakb0GnCM\\\\\\\/ls\\\\\\\/X1FkJ0ibz1zujLiN8zeuHRLlwOD4bmVHRrODfVQhTloUhLE03iHE\\\\\\\/Vx5FHDX7Xoy0PiDUlDjpeF0VlG5QkZx34l5HG49AokahBcbYtAyMbhK2oeiExwo\\\\\\\/aoyGeCylT32G7M2zD7zFkQcbtIOTmaUbLQkeF+wZqcjc6nYs+edw7MKtYeuAJMZtw+DRujaYOGQpQygbLR0euiEP4z11Q9Vs9Nqp+h7yEaHoke8myhYAGW34Gzh24FFHKxXxZud9+NzIOOX+ibIhbzBKvfAPL6YTzCwq66GVwrfR73x5v3jOuNMX\\\\\\\/7bM3sTbrlG+8RysCHTo\\\\\\\/AbGUBwUEQiqzht+467dCJS\\\\\\\/WtBRivg24Ym5n+wwIFBJPq\\\\\\\/gQq7r\\\\\\\/tfh\\\\\\\/pP19y1Wmfqw6P5Wi51hctwQ6bbCOfdxKqx9cIRJ5r\\\\\\\/SkIb1j894d2JF9WeZ6L7jSzh6dSU5\\\\\\\/SBmPNS3dcwkaLOgPu\\\\\\\/w+XmE6oZnYvdI76IlCJnIoHYbEqTiP2bcg1FaKn0fl8hNBzwjG+jZpLCFZaAKXClEGgmogUog==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HUOGZaJXynRMH4tCArWKaQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224c480a9edd58ebbcdaa3bf1f2a5d4e07e54918db1b998b5466e1d421d41a40c7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022c5XW1RooNxCJQskvMabG8wi26mZOFNsnBjhZnOTCI0lkx7bKQtaTTRGNBHUSRVC1FjVTDZV7o04wpJkaBi8j7oqlflKz8CC6znc+p+QgqXeyTcdDKtDSNxVLeQZYhk+8IB7WRQWt5RBG+VoWYzmayeOJy7Vs+dBB\\\\\\\/LHfV5Lawyb3sndKk99e1OBSEr5BoXtCenKgkFo5\\\\\\\/TDRTbcTrKt2Lpz2EBMtzNIRIq5KyFRl\\\\\\\/l9mOA6gxQIRMDkvbqPYsSvURO1NFriK9PbrrgY21Zy7jwvI2+d9PirLofkdZ6cdEoMykUFzV9zaLdns4JfRvPEBYVdD52XAx+PDQRIsIkjjgJB0+yil5FTdHt3bPLYhroWWF3GRzbEypX\\\\\\\/5w1ZJLkck6vBwyTiH8uaJoNksmeRtDth7bjSS4onD0VRgZgw1hC1McrVo3Am+HCQVz4ZKqsb+EFTyz31HBhuLQH9CwkUZZsYZnB6jb9ToIIdtBNrXvieMtk0ibo7zgC8xMfbOAjrymCgwd2g1IIbqic4bHc+vNg7oVl+Zx\\\\\\\/+EeLpuq9exT66BtJXS6VqRJKz0NO0UTeH3gtRIoIhqIfTqZDJhoLRytvg1IKTZMYIxf\\\\\\\/we1+shMXuK8Z0BJh4tnlicqplzTq5E77NMie8u7aoFexVoNfp+BYhsBWst+J\\\\\\\/O7LGH6j7TAyWN8NHo7xjMj5Uzmrs7hGPC5LzleBU5Ki2Y5XO6Xb5++BQwchRNn\\\\\\\/\\\\\\\/OmYffO\\\\\\\/Vvy\\\\\\\/GUmFeHbRgmP2Xj6uzbM9zE6cV2HtvPnR58P7rbPPTlmlh71iJV+GsYMsxI927GNFyLF\\\\\\\/UnGhPrsWnh2H4kTU8VHXGtGgxSQeKa1Zhg876W4u+kNoQVhGHT\\\\\\\/kLYRfO0cJgtgi4P7dXUhqbeLJnir2GxeMsFozoyUnr\\\\\\\/4logTtknmg63l8H3Kbl6xmPBht5QN6SgmAeZTdaI3jbQkovytiNJNRF1qQrGLnorPfeSdu7x1RthmUs5h5FRT5f+noySkD5RxvUcPIoABp1c4xkXwWBdQM\\\\\\\/0uxbDw+kdbZ3QW3PxyJ0Kd9PBE7wKI2a1IrApw2cLxFz1M5txOyLYergs06U\\\\\\\/7iSqmWmH9gkbEkXqqizQhW3XfO7vRxxn2JVmz0Snp4HtL+hY9xDSdLx8H+VYmurKmxYMe+CTAVl5uM1Xj6ezVvYMENhHqTMOri1LiGCjSWbBtMeE86M9Wym6lLH80EBVQgQl20FwWIPUJQHiftZ9abwIMFTPzcrK3qEwJZdkvV2cxbMrKl0Ub22rMu3BokRY3VmJ1NCbV\\\\\\\/lsKZrRmIEp1rtdGFZ8+VsBkZRMXcaHVhxBen8oC+8oT1w5hbFunfByR8hu+seEoiloPP\\\\\\\/b+P1SC7BjU32GaxWc04EL6DHk7aPoWnMLZQfrr6e34kQmSeW63AK0gKqsUfrBufQkJJW+6S8dI9eplNk4HqBkXKja5eA\\\\\\\/n3YV3WNNkzoVoDbbbe+X6aSTNlOBE1pe7IM3918zhGChNhAnSZgEgwV4j2RQ89iEM9lCIJCAdfgQZAGDwWrhLkOVBfXUe5a04YLvd8qHV7FHvS5oK9RqBhIywVkA85IXJA9ld5sFXdl8PH2rfs+bh5Q3iRqvOUAgy0+bPIaN7VbctYSmDvwsgAHY2PcZaO3wNmcd6JS3PjuxjYBxtUQ\\\\\\\/QJOY+7jZ7N\\\\\\\/SkwwdGXCqnDV1zZ+euZhHuZEFeqT7AalReDipsEcbV7PC\\\\\\\/RPCIzMfm9cePZIBZSP4Oy7N0ymrtCXGrbx2Tufj1dpDIuNO2MvJZ3vilGGCSYr1i4beI74IodyC2BkmLPB+CST7um68Mp5DNZZcrl\\\\\\\/m7I6Nia\\\\\\\/wrSedffnqqddT8zwuVvHgvNmzvyv4beAxM19YXsKgvm1qR8TUHS4VNtf5rmo7yRo\\\\\\\/nMqU0mkAb9ChMMNjDYPmiVkGt76KZHG6nev5xpPQBSj1u+q7ZgQK36MmsSwxpI5VE0G8sOq8aLSqNIcSORUmocAVVwhmnX67ubv9qByMaklDnQ==\\\u0022,\\\u0022iv\\\u0022:\\\u002238Ns4VA0ZK90HkHlFmDYvQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022be59158259729cbb2889fec7235c5ea27055cbe019793ba81f199e9969d340ad\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460419 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2203" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KMKeIBmUCDenOxKPR_6p_qsxU-guzGJEroB7KHKCaqA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2203" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ijZJ-u7dD_uPDnFJHHSAAHGzKuYtxu9zfc2vZ2o92EY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/2195" title="An Improved Binary Vector and Escherichia coli Strain for Agrobacterium tumefaciens-Mediated Plant Transformation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/2213" title="Large Deletions at the SHOX Locus in the Pseudoautosomal Region Are Associated with Skeletal Atavism in Shetland Ponies" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460419" id="node460419" data-pisa="ggg;6/7/2203" data-pisa-master="ggg;g3.115.026690" data-apath="/ggg/6/7/2203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Mapping of Millions of SNPs in Safflower (<em>Carthamus tinctorius</em> L.) via Whole-Genome Resequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">John E. Bowers</span>, <span class="highwire-citation-author" data-delta="1">Stephanie A. Pearl</span> and <span class="highwire-citation-author" data-delta="2">John M. Burke</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2203-2211;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026690 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460419" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John E. Bowers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BBowers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowers%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BE.%2BBowers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stephanie A. Pearl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephanie%2BA.%2BPearl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pearl%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephanie%2BA.%2BPearl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John M. Burke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BBurke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burke%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BM.%2BBurke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmburke@uga.edu" class="" data-icon-position="" data-hide-link-title="0">jmburke@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/7/2203" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460419" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460419/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2203.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460419" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460419/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2203.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460419" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460419/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2203.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460419" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460419/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/7/2203.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460419" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460419/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Accurate assembly of complete genomes is facilitated by very high density genetic maps. We performed low-coverage, whole-genome shotgun sequencing on 96 F<sub>6</sub> recombinant inbred lines (RILs) of a cross between safflower (<em>Carthamus tinctorius</em> L.) and its wild progenitor (<em>C. palaestinus</em> Eig). We also produced a draft genome assembly of <em>C. tinctorius</em> covering 866 million bp (∼two-thirds) of the expected 1.35 Gbp genome after sequencing a single, short insert library to ∼21 × depth. Sequence reads from the RILs were mapped to this genome assembly to facilitate SNP identification, and the resulting polymorphisms were used to construct a genetic map. The resulting map included 2,008,196 genetically located SNPs in 1178 unique positions. A total of 57,270 scaffolds, each containing five or more mapped SNPs, were anchored to the map. This resulted in the assignment of sequence covering 14% of the expected genome length to a genetic position. Comparison of this safflower map to genetic maps of sunflower and lettuce revealed numerous chromosomal rearrangements, and the resulting patterns were consistent with a whole-genome duplication event in the lineage leading to sunflower. This sequence-based genetic map provides a powerful tool for the assembly of a low-cost draft genome of safflower, and the same general approach is expected to work for other species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/wgs-map" class="hw-term hw-article-keyword hw-article-keyword-wgs-map" rel="nofollow">WGS map</a></li><li class="kwd"><a href="/keyword/genetic-map" class="hw-term hw-article-keyword hw-article-keyword-genetic-map" rel="nofollow">genetic map</a></li><li class="kwd"><a href="/keyword/safflower" class="hw-term hw-article-keyword hw-article-keyword-safflower" rel="nofollow">safflower</a></li></ul><p id="p-3">Genetic linkage maps have long served as powerful tools for the study of genome structure and the genetic basis of trait variation in both plants and animals. The utility of such maps is largely determined by the number of loci and the accuracy and precision with which they have been placed. Initially, genetic maps were based on small numbers of segregating, single gene morphological traits, and the resulting linkage groups had very few loci (<em>e.g.</em>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sturtevant 1913</a>). Over time, it became possible to produce genotypic data from indirectly observed genetic variants, such as isozymes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">McKusick and Ruddle 1977</a>). Genetic linkage maps remained sparse, however, until the advent of DNA-based techniques in the 1980s (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Botstein <em>et al.</em> 1980</a>), which facilitated the more efficient generation of genotypic data. By testing many DNA samples at a time via RFLP or PCR assays, it became possible to produce genetic maps of hundreds or even a few thousand loci. More recently, this paradigm of testing many samples at once for one or a few loci per assay reversed to the simultaneous interrogation of individual samples for hundreds or thousands of loci on a DNA chip (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beattie <em>et al.</em> 1995</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bowers <em>et al.</em> 2012a</a>,<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">b</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Truco <em>et al.</em> 2013</a>), effectively reducing the lab work needed to generate genetic maps containing thousands of loci to a few days.</p><p id="p-4">The rapidly falling cost of DNA sequencing has enabled new, low cost options for the generation of genetic mapping data. By sequencing a targeted subset of the whole-genome, such as sequences that flank restriction enzyme cut sites, researchers have been able to generate sequence-based genetic maps with tens of thousands of loci from sequence data produced in a few days. This general technique, known as genotyping-by-sequencing (GBS; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elshire <em>et al.</em> 2011</a>) has revolutionized genetic mapping research. But even with these much greater marker densities, more loci are often desired to thoroughly characterize complex genomes. Here, we describe an approach for the construction of ultradense, sequence-based genetic maps using low coverage, whole-genome shotgun (WGS) sequencing data from a set of recombinant inbred lines (RILs). As we near the age of the $1000 genome and the $100,000 analysis (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Mardis 2010</a>), it has become increasingly obvious that the development of new approaches for large-scale assembly and analysis, not raw sequencing costs, will drive future breakthroughs in cost and efficiency. Our work represents a step in that direction.</p><p id="p-5">The generation of a genetic map from the WGS sequencing of segregating progeny presents unique computational challenges. While WGS sequencing has the potential to reveal millions of mappable polymorphisms (three or four orders of magnitude higher than previously practical), the resulting datasets present unique computational challenges as the number of possible locus orders corresponds to the factorial of the number of loci. Low coverage sequencing poses additional complications, as many individuals will be missing data at a substantial fraction of all loci. Moreover, heterozygous genotypes will often be genotyped incorrectly, though this latter problem can be minimized via the use of an inbred mapping population. Finally, due to the very large number of loci tested, errors in sequencing and read placement will result in a large number of erroneous data points, even if the overall rate of such errors is low. Thus, the primary challenge when using WGS data for genetic mapping is determining how to best use the data, as opposed to generating the data in the first place.</p><p id="p-6">While the general approach of using WGS sequencing data for genetic mapping has been previously implemented (<em>e.g.</em>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Frazer <em>et al.</em> 2007</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Huang <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Renaut <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wijnker <em>et al.</em> 2013</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mascher <em>et al.</em> 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ariyadasa <em>et al.</em> 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hahn <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chapman <em>et al.</em> 2015</a>), it has only recently been applied to genomes without existing chromosome-level assemblies. The limited use of WGS sequencing for mapping previously unexplored genomes likely relates to the aforementioned challenges, and the fact that an available genome assembly avoids many of the computational challenges associated with marker ordering. In this paper, we describe the construction of a sequence-based genetic map of the safflower (<em>Carthamus tinctorius</em> L.) genome and its use to explore patterns of genome evolution across the Compositae, which is the largest family of flowering plants and includes other economically important species such as lettuce and sunflower.</p><p id="p-7">Safflower has a long history of human use, with archeological remains of <em>Carthamus</em> spp. dating back to 7500 BC having been found at sites in Syria (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Marinova and Riehl 2009</a>). From there, safflower cultivation is thought to have spread to Egypt, the Aegean, and into southeastern Europe. In ancient times, safflower floral extracts were commonly used as both red and yellow dyes for textiles; hence, the species name <em>tinctorius</em>, derived from the Latin <em>tinctus</em> (dyed, stained, or tinged), which means ‘used in dyeing.’ Safflower seeds have also been directly consumed on a limited basis, dried and ground flowers have been used in cooking in place of saffron, and it is sometimes grown for medicinal uses. Commercialization of safflower in the Americas commenced in the 1950s, where it has primarily been grown as an oilseed crop, as a source of birdseed, and as an ornamental (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Smith 1985</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Emongor 2010</a>). Worldwide, safflower is planted on ∼1 million hectares/yr with a total production value of ∼$260 million (<a href="http://faostat.fao.org/">http://faostat.fao.org/</a>). It is also notable that safflower is well-adapted to growth in moisture-limited environments and can tolerate saline soils. As such, it can be grown on marginal agricultural lands that are suitable for few other crops (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ghamarnia and Gholamian 2013</a>). Despite this relatively low overall value, it has been suggested that the improvement of minor crops such as safflower could help to provide food security in the face of climate change and an ever-increasing global population (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amini <em>et al.</em> 2014</a>).</p><p id="p-8">Because safflower was originally domesticated for an entirely different purpose (<em>i.e.</em>, as a source of dye) and has been subjected to relatively limited modern breeding efforts, it seems likely that selection has unlocked only a small portion of its potential as an oilseed crop. To accelerate the ongoing improvement of safflower, we sought to develop an ultradense, sequence-based genetic map that could be used to guide the assembly of the safflower genome. This resource has the potential to facilitate molecular breeding efforts and, when combined with available mapping resources from other species within the Compositae (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kane <em>et al.</em> 2011</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bowers <em>et al.</em> 2012a</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Truco <em>et al.</em> 2013</a>), can also be used to investigate patterns of genome evolution within one of the world’s most important flowering plant families.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and draft genome assembly</h3><p id="p-9">The initial production of the mapping population was described by <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Pearl <em>et al.</em> (2014)</a>. Briefly, a single individual of safflower (cv AC Sunset; PI 592391) was crossed with a single individual of its wild progenitor, <em>C. palaestinus</em> (PI 235663). The F<sub>1</sub> seeds from this cross were then planted, grown to maturity, and self-pollinated to produce the F<sub>2</sub> generation. The resulting F<sub>2</sub> lineages were then repeatedly grown and self-pollinated to produce F<sub>6</sub> RILs. DNA was extracted from each of 96 RILs and five plants from the same seed packets as each of the mapping parents. The individual samples were then barcoded, combined into a single mix, and sequenced on all eight lanes of a single flow cell of an Illumina Hi-Seq (GenBank PRJNA 313950). A genome assembly of the mostly homozygous <em>C. tinctorius</em> AC Sunset inbred line was made but, due to the difficulties of genome assembly for heterozygous individuals, the same was not done for the outbred <em>C. palaestinus</em>.</p><p id="p-10">We produced a total of 4.7 × 10<sup>9</sup> bp of sequence data using 100 bp paired-end sequencing reads with an average insert size of 306 bp. Each RIL was thus sequenced to an average of 2.26 × (range 0.92 × – 4.44 ×) coverage of the estimated 1350 Mbp genome size of the safflower genome (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Garnatje <em>et al.</em> 2006</a>), while the parents were sequenced to a greater depth. Sequence reads from the homozygous AC Sunset parent totaled ∼21.3 × coverage of the estimated genome size, and a draft genome was assembled from these reads using the program SOAPdenovo2 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Luo <em>et al.</em> 2012</a>) at K = 63, followed by gap filling using SOAP GapCloser.</p></div><div id="sec-3" class="subsection"><h3>Linkage map construction</h3><p id="p-11">Sequence reads from the RILs were mapped onto the draft AC Sunset assembly with Bowtie 2 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Langmead and Salzberg 2012</a>) and processed using SAMtools (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li <em>et al.</em> 2009</a>). Genotype scores for individuals at each SNP were computed from LOD values by assuming homozygous genotypes whenever possible (LOD &gt; 0), not the automatic genotype calls produced by SAMtools. The reason for this is that the assumptions of Hardy-Weinberg equilibrium that are used to assign genotypes in SAMtools do not apply to the RILs studied; with RILs, heterozygous genotypes are far less likely than would otherwise be expected. For an F<sub>6</sub> RIL mapping population, the expected segregation ratio would be 31:2:31 for the AA, AB, and BB genotypes, respectively. Loci that were heterozygous in more than 15% of the RILs were discarded as they likely corresponded to multi-copy sequences. SNPs that segregated with minor allele frequency (MAF) of less than 5% were also discarded, as they were most likely artifacts due to sequencing errors or incorrect placement of sequence reads against the assembly. The final thresholds on MAF and heterozygosity were set empirically based on iterative creation of draft maps with less stringent thresholds. This is because, while the expected levels can be estimated, actual results for a map depend on segregation distortion in the mapping population, which can be highly unpredictable and variable. The observed MAF minimum for haplotypes included in the drafts map was 12.5%, with an observed maximum heterozygosity of 10.5%. The thresholds used for individual SNPs were more liberal than the observed limits on the draft maps to account for sequencing errors and missing data affecting individual SNPs. SNPs with &gt; 50% missing data were also discarded.</p><p id="p-12">As several million SNPs still remained after the above steps, only SNPs with quality scores from SAMtools function Mpileup equal to the 999 maximum score were used. In the next step, SNPs from the same sequence scaffold were merged together to create a consensus scaffold-level haplotype. By constructing a consensus haplotype from all SNPs on a given scaffold, we were able to use low coverage data to unambiguously determine genotypes for a given genetic region, including heterozygous genotypes, and sequencing errors could then be corrected <em>vs.</em> the consensus. The resulting scaffold haplotypes were then used to assemble a consensus template map using Microsoft Excel as described by <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bowers <em>et al.</em> (2012a)</a>, beginning with scaffolds containing &gt;50 SNPs to simplify the ordering problem. The somewhat unorthodox approach of using spreadsheet software for genetic map ordering is outlined in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026690/-/DC1/FileS1.pptx">File S1</a>. The template map was assembled by sorting data columns, where columns correspond to individuals and rows correspond to the consensus SNP haplotypes. This approach provides a simplified and rapid means for ordering a genetic map when most of the genotypic patterns are present in the data (<a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Bowers <em>et al.</em> 2012a</a>). The reason that this approach works so well is that, with extremely high marker density, most haplotype patterns are separated just one or a very small number of recombination events from adjacent patterns in the data. Using the assumption that the best genetic map is the one that contains the fewest recombination events, ordering becomes a relatively simple task of placing the most similar multi-locus haplotypes adjacent to each other.</p><p id="p-13">The initial template map was then used to filter all individual SNPs. Individual SNPs that are the product of multi-copy sequences or sequences where the two alleles correspond to different genomic locations would not fit onto the genetic map. All SNPs that showed more than six differences from the most similar location on the template map were discarded as anomalous SNPs. The genotypes of scaffolds were then recomputed with the filtered SNP data, reducing the error rate of the consensus genotype for the scaffold. For comparison to the template, heterozygous loci were assumed to match genotype scores of AA, AB, or BB without flagging them as being different from the template due to the likelihood that heterozygous genotypes at an individual SNP may not be observed due to sequencing depth. The filtered SNPs were then merged by scaffolds and used to manually edit and improve the template map. Removal of the anomalous SNPs from the scaffold prior to determining the consensus improved the accuracy of the scaffold genotypes. This process was repeated, revising the template map each cycle and refiltering the individual SNPs to combine them by scaffolds, until no further changes to the template map were made (15 iterations). The template map was revised and edited with scaffolds containing progressively fewer SNPs, with the final five iterations involving the examination of all scaffolds with ≥ 5 SNPs.</p></div><div id="sec-4" class="subsection"><h3>Analysis of the linkage map</h3><p id="p-14">Individual scaffolds that contained multiple SNPs mapping to two different linkage groups or &gt; 10 recombination events apart were identified as mis-assembled scaffolds. Scaffolds containing three or more consecutive SNPs of one homozygous genotype followed by three or more consecutive SNPs of the other homozygous genotype, and in which the template map predicted a crossover event in the corresponding RIL, were inferred to contain recombination breakpoints. DNA sequence from the genome assembly between the SNPs indicating recombination events was extracted, along with 200 bp of flanking sequence on each side. While recombination events also exist on the map between homozygous and heterozygous regions, the difficulty of assigning heterozygous genotypes to individual SNPs due to low sequence coverage prevented precise placement of such events. Therefore, recombination events between heterozygous and homozygous regions were only placed at intervals between scaffolds, not between individual SNPs.</p><p id="p-15">Synteny comparisons to previously published genetic maps of lettuce and sunflower (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Bowers <em>et al.</em> 2012a</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Truco <em>et al.</em> 2013</a>) were performed using BLAST to place gene sequences from the other genetic maps onto the safflower draft assembly. The best match for each sequence with e-values &lt; 10<sup>−6</sup> was then included in a dot plot if the best hit corresponded to a genetically mapped safflower scaffold.</p></div><div id="sec-5" class="subsection"><h3>Data availability</h3><p id="p-16">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results-discussion" id="sec-6"><h2 class="">Results and Discussion</h2><div id="sec-7" class="subsection"><h3>Draft genome assembly and SNP identification</h3><p id="p-17">Our draft assembly of the safflower genome consisted of 3,254,412 contigs 100 bp or larger that assembled into 2,195,958 scaffolds 100 bp or larger <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026690/-/DC1/FileS2.zip">File S2</a>, (<a href="http://datadryad.org/resource/doi:10.5061/dryad.635ds">http://datadryad.org/resource/doi:10.5061/dryad.635ds</a>; GenBank LUCG00000000 for sequences ≥ 200 bp). The assembly had an overall GC content of 37.3%. The contig N50 size was 368 bp and the scaffold N50 size was 1976 bp (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026690/-/DC1/FileS3.txt">File S3</a>). The total assembled size was 866 Mbp, corresponding to ∼65% of the expected genome size of 1350 Mbp. The incomplete coverage and fragmented nature of our draft assembly were likely due to the use of a single, small insert library at 21 × coverage, combined with the possibility that repetitive genomic sequences failed to assemble or collapse into single contigs. In fact, recommendations for <em>de novo</em> WGS assemblies from short reads call for substantially higher levels of sequencing coverage as well as libraries with multiple insert sizes (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gnerre <em>et al.</em> 2011</a>). Nonetheless, the suitability of our assembly for polymorphism discovery and genetic mapping demonstrates that even a rudimentary genome assembly can be a valuable resource.</p><p id="p-18">The initial results from calling SNPs after alignment with Bowtie 2 and processing with SAMtools provided 7,822,301 candidate SNPs with a minimum allele frequency &gt; 5%. However, after the removal of SNPs with high levels of heterozygosity, missing data, and/or low quality scores, the number of SNPs was reduced to slightly over two million. With millions of SNPs and missing data rates &gt; 15% at individual SNPs, plus high genotyping error rates at heterozygous sites due to limited sequencing depth, construction of a genetic map by conventional approaches that treat each SNP separately would be exceptionally difficult (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cheema and Dicks 2009</a>). Given the extremely large number of loci, an exhaustive examination of all possible marker orders is all but impossible. Even after merging all SNPs with identical patterns, hundreds of thousands of genotype patterns (due to the segregation of actual polymorphisms as well as sequencing errors and missing data) would still have to be ordered. As such, an alternative approach (outlined above) was required.</p></div><div id="sec-8" class="subsection"><h3>Genetic map description</h3><p id="p-19">The final template map is located in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026690/-/DC1/FileS4.xlsx">File S4</a> (<a href="http://datadryad.org/resource/doi:10.5061/dryad.635ds">http://datadryad.org/resource/doi:10.5061/dryad.635ds</a>), with an example for one chromosome provided in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>; an overall summary is provided in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The genetic map of safflower assembled into 12 linkage groups, matching the expected chromosome number. A list of the genetically placed scaffolds and their locations is presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026690/-/DC1/FileS5.xlsx">File S5</a> (<a href="http://datadryad.org/resource/doi:10.5061/dryad.635ds">http://datadryad.org/resource/doi:10.5061/dryad.635ds</a>). The template map included 2082 recombination events, of which 544 were transitions from homozygous to heterozygous genotypes, and 1538 were transitions between the two different homozygous genotypes. Given that they occurred across 12 chromosomes, the 2082 recombination events would be expected to produce a total of 2094 genotypic patterns; however, only 1178 unique patterns were actually observed on one or more full scaffold, as several intervals in the map had multiple recombination events between loci. A total of 2,008,196 mapped SNPs <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026690/-/DC1/FileS6.csv">File S6</a> (<a href="http://datadryad.org/resource/doi:10.5061/dryad.635ds">http://datadryad.org/resource/doi:10.5061/dryad.635ds</a>) could be placed on 57,270 mapped scaffolds that contained 192,589,591 non-N nucleotides. This is an average of one genetically mapped SNP per 95.9 bp, though the actual differences between the two safflower individuals is likely higher than this, as SNPs near repeats or near the ends of scaffolds are less likely to be mapped. The high levels of sequence divergence between the two mapping parents may have prevented accurate placement of sequence reads in more divergent regions.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Template genetic map for safflower linkage group 2. A total of 65 unique recombinational patterns were observed across the 96 RILs for this linkage group. The rows correspond to distinct chromosomal locations, and the columns on the right correspond to the 96 RIL lines and the genotype of the RIL at each location. The &#x201C;A&#x201D; genotype score corresponds to &#x201C;AA,&#x201D; &#x201C;H&#x201D; to &#x201C;AB,&#x201D; and &#x201C;B&#x201D; to &#x201C;BB,&#x201D; and the genotypes are color-coded for clarity. A total of 3572 sequence scaffolds containing 11,697,443 bp and 108,293 mapped SNPs were placed on this linkage group. RIL, recombinant inbred line; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-135970511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Template genetic map for safflower linkage group 2. A total of 65 unique recombinational patterns were observed across the 96 RILs for this linkage group. The rows correspond to distinct chromosomal locations, and the columns on the right correspond to the 96 RIL lines and the genotype of the RIL at each location. The &#x201C;A&#x201D; genotype score corresponds to &#x201C;AA,&#x201D; &#x201C;H&#x201D; to &#x201C;AB,&#x201D; and &#x201C;B&#x201D; to &#x201C;BB,&#x201D; and the genotypes are color-coded for clarity. A total of 3572 sequence scaffolds containing 11,697,443 bp and 108,293 mapped SNPs were placed on this linkage group. RIL, recombinant inbred line; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2203/F1.medium.gif" width="386" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/7/2203/F1.medium.gif" width="386" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468646" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Template genetic map for safflower linkage group 2. A total of 65 unique recombinational patterns were observed across the 96 RILs for this linkage group. The rows correspond to distinct chromosomal locations, and the columns on the right correspond to the 96 RIL lines and the genotype of the RIL at each location. The “A” genotype score corresponds to “AA,” “H” to “AB,” and “B” to “BB,” and the genotypes are color-coded for clarity. A total of 3572 sequence scaffolds containing 11,697,443 bp and 108,293 mapped SNPs were placed on this linkage group. RIL, recombinant inbred line; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468724/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468724/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary characteristics of genetic map by linkage group</span><div class="sb-div caption-clear"></div></div></div><p id="p-22">Despite the high number of SNPs and mapped scaffolds, several gaps remained in the final map. Eight gaps spanning ≥ 5 cM remained in the final map. The largest gap spanned 7.1 cM, corresponding to 14 of the 96 RILs exhibiting recombination events between the flanking loci. These gaps could be regions that are relatively short in physical size, but have high recombination rates (<em>i.e.</em>, recombination hotspots; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Drouaud <em>et al.</em> 2006</a>). Alternatively, they could correspond to regions that largely contain repetitive DNA, which could limit local assembly quality and reduce the frequency of mappable polymorphisms. Finally, it is possible that they represent large genomic regions that are highly similar or identical due to a recent shared ancestor between the two genotypes used as parents in the cross, precluding any SNPs from being mapped in those regions (<a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Bowers <em>et al.</em> 2012a</a>).</p></div><div id="sec-9" class="subsection"><h3>Genotyping and assembly errors</h3><p id="p-23">For SNPs in mapped scaffolds, the error rates and SNP types could be calculated. The missing data rate for SNPs at individual RILs was 16.9%. For SNPs where the scaffold consensus genotype was heterozygous, 61.2% of the individual SNPs were erroneously scored as being homozygous (<em>i.e.</em>, both alleles were not observed). For SNPs where the scaffold consensus genotype was homozygous, 0.5% of the individual SNPs were incorrectly scored as heterozygous, and 0.4% were scored as the other homozygous genotype. For many individual SNPs, exact locations on the genetic map would have been uncertain due to missing data. However, when the SNPs were grouped into scaffolds, 95.2% of the scaffold consensus genotypes fit into a unique position on the map template with no uncertainty. Even when they were not assigned a unique position, most scaffolds could be placed in a narrow window. Indeed, for 99.9% of the SNPs grouped by scaffolds, the genetic map position assigned was a window of 2 cM or less. Of all 2,008,196 genetically mapped SNPs, 64.7% were C/T polymorphisms, 7.5% were C/G polymorphisms, 16.8% were C/A polymorphisms, and 11.0% were A/T polymorphisms. The high frequency of C/T polymorphisms has been noted in many mutation studies, and is likely related to the high frequency of methylated cytosine mutating to thymidine (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Youssoufian <em>et al.</em> 1986</a>).</p><p id="p-24">Genetic maps such as the one described herein also provide insight into the quality of a genome assembly, particularly with respect to identifying mis-assembled scaffolds while editing and correcting the assembly into chromosome-sized pieces (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Paterson <em>et al.</em> 2009</a>). In the present study, scaffolds that contained five or more SNPs mapping to two different chromosomes were identified as likely errors in the initial shotgun genome assembly. We found 118 such chimeric scaffolds, corresponding to ∼0.2% of all mapped scaffolds. Of course, the highly fragmented nature of our assembly, with many short scaffolds, reduces the likelihood of chimeric assemblies in this case. In addition to error detection, an ultrahigh density genetic map can be used to place/order scaffolds, as well as to orient the scaffolds that are found to contain recombination events. Although that process would be of limited value with the current assembly, it could greatly improve the quality of a more complete genome assembly.</p></div><div id="sec-10" class="subsection"><h3>Read depth as a filtering tool</h3><p id="p-25">The combined sequencing depth across all RILs can be used to identify multi-copy sequences. The combined sequencing depths for the final set of mapped SNPs, as well as the initially identified pool of candidate SNPs, are summarized in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. For genetically mapped SNPs, the total sequencing depth combined from all RILs fell into a narrow range, while the preliminary candidate SNPs showed a much broader distribution. The combined sequencing depth for 96 RILs of mapped reads had a mode of ∼200 ×. The mapped SNPs fell into a relatively narrow range with 99% of the mapped SNPs having a combined mapped read depth between 82–339 reads. Most genetic polymorphisms that correspond to repetitive sequences were removed in the mapping process, as they would not map to a single location on the template map, or they would show unexpected segregation ratios. For many of the candidate SNPs that we initially identified, the depth of coverage was much broader, with a lower peak of low coverage. Most of these lower coverage SNPs were discarded due to missing data at over 50% of the RILs or low sequence alignment scores. The candidate SNPs also contained a substantial number of SNPs that showed depth of coverage &gt; 400×, most likely corresponding to repetitive sequences. Although we did not do this in the current study, filtering candidate SNPs by combined read depth would be an effective approach to remove sequences corresponding to repetitive DNA.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Total mapped sequencing read depth for genetically mapped SNPs, combined across 96 RILs. The modal combined sequencing depth was 200, and 99% of the mapped SNPs were sequenced between 79&#x2013;358 times, combined across all RILs. RIL, recombinant inbred line; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-135970511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Total mapped sequencing read depth for genetically mapped SNPs, combined across 96 RILs. The modal combined sequencing depth was 200, and 99% of the mapped SNPs were sequenced between 79&#x2013;358 times, combined across all RILs. RIL, recombinant inbred line; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2203/F2.medium.gif" width="404" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/7/2203/F2.medium.gif" width="404" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468663" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Total mapped sequencing read depth for genetically mapped SNPs, combined across 96 RILs. The modal combined sequencing depth was 200, and 99% of the mapped SNPs were sequenced between 79–358 times, combined across all RILs. RIL, recombinant inbred line; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Recombination rate variation</h3><p id="p-27">Comparisons of whole-genome assemblies to genetic maps in other plant species have revealed that recombination rates vary across the genome (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Paterson <em>et al.</em> 2009</a>). It is thus not surprising that the number of anchored base pairs per distinct segregation pattern varied greatly in our study. While 44% of the patterns theoretically present in the genetic map (<em>i.e.</em>, between observed genotypes) were not observed, some intervals with no recombination were represented by as many as 213,339 SNPs, corresponding to &gt; 10% of all mapped SNPs and totaling 20,497,082 bp of mapped scaffolds. The exact relationship between the local recombination rate and base pair distances can, however, only be approximated by the portion of the genome that is represented by genetically mapped scaffolds, and it is logical to assume that the gene rich, single copy fraction of the genome would both assemble and be genetically mapped more often than the repetitive fraction. It seems likely that the three loci with the largest number of SNPs (located on chromosomes 4, 5, and 6 and harboring 213,339, 135,029, and 85,996 SNPs, respectively – 21.6% of all mapped SNPs combined) are the result of inhibited recombination along a portion of those chromosomes, possibly due to structural rearrangements such as large inversions. In contrast, linkage groups 7, 8, and 9 did not contain any genetic loci with more than 12,815 SNPs. The total number of SNPs that could be mapped per linkage group ranged from 75,934–339,005. The distribution of SNP loci across the map, and the number of scaffolds and base pairs mapped by position, is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026690/-/DC1/FigureS1.pdf">Figure S1</a>.</p><p id="p-28">The draft genome assembly allowed us to identify the genomic location of a subset of the observed recombination events. A total of 194 recombination events could be placed within sequence scaffolds, and the location of the recombination events could be localized to regions spanning 7–3748 bp in length (average = 343 bp; see <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> for an example). Additionally, there were four cases in which two recombination events were observed within a single scaffold. These events were 255–409, 407–797, 737–958, and 2400–2686 bp apart, suggesting the existence of recombination hotspots within the safflower genome. The scaffolds with recombination events added an additional 23 genotype patterns that were not present in the genetic map template based on whole scaffolds. As noted above, the exact placement of recombination events between homozygous and heterozygous genotypes was not attempted due to the relatively higher error rate in genotyping heterozygous loci at individual SNPs. Comparison of all the sequences where recombination occurred, plus the flanking 200 bp to each other, revealed no recurring sequence motif that could be detected by BLAST. However, The GC content was slightly lower near recombination events than in the assembly as a whole (36.4% <em>vs.</em> 37.3%). While the absolute difference in GC content was slight, it was highly significant (<em>P</em> &lt; 0.000000001).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Sample scaffold showing raw genotype scores for 41 SNPs and the consensus genotype for the scaffold. This scaffold shows a recombination event, indicated by a black bar in the consensus, that occurred between 1291&#x2013;1338 bp. RIL, recombinant inbred line; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-135970511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Sample scaffold showing raw genotype scores for 41 SNPs and the consensus genotype for the scaffold. This scaffold shows a recombination event, indicated by a black bar in the consensus, that occurred between 1291&#x2013;1338 bp. RIL, recombinant inbred line; SNP, single nucleotide polymorphism.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2203/F3.medium.gif" width="420" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/7/2203/F3.medium.gif" width="420" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468753" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Sample scaffold showing raw genotype scores for 41 SNPs and the consensus genotype for the scaffold. This scaffold shows a recombination event, indicated by a black bar in the consensus, that occurred between 1291–1338 bp. RIL, recombinant inbred line; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Comparison to other species</h3><p id="p-30">Comparisons between the safflower genetic map and draft assembly were made to two previously published maps of lettuce and sunflower (<a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Bowers <em>et al.</em> 2012a</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Truco <em>et al.</em> 2013</a>), which are members of different subfamilies within the Compositae. From the lettuce genetic map, the 12,786 mapped genes included 10,593 (83%) that could be placed on the safflower assembly. Of these, 4614 (44%) matched scaffolds that had been placed on the safflower genetic map. Inspection of <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> reveals that the safflower and lettuce genomes have undergone substantial rearrangement since their most recent common ancestor, with only one pair of linkage groups (LGs; safflower LG 8 and lettuce LG 2) retaining synteny across their entire length. All other safflower and lettuce LGs are partially syntenic to multiple LGs from the other species, except safflower LG 2, which is one of three linkage groups from safflower that contributed to lettuce LG 3. In all, 26 large segments of synteny were observed between lettuce and safflower; these segments contained 3611 of the 4614 genes mapped in both species. This suggests that ∼78% of all genes are in corresponding syntenic positions between the two species. This comparison also suggests that the mapped scaffolds represent a higher than expected fraction of single copy gene sequences in safflower. Indeed, while only 14% of the raw expected genome size of safflower is in genetically mapped scaffolds, 36% of the gene sequences from a related species could be placed on the safflower map.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synteny plot between the lettuce genetic map and the safflower draft genome assembly and genetic map depicting the relative positions of 4614 gene sequences. The genes from 26 syntenic segments highlighted in red contain 78% of all gene sequences on both maps." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-135970511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Synteny plot between the lettuce genetic map and the safflower draft genome assembly and genetic map depicting the relative positions of 4614 gene sequences. The genes from 26 syntenic segments highlighted in red contain 78% of all gene sequences on both maps.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2203/F4.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/7/2203/F4.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468428" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Synteny plot between the lettuce genetic map and the safflower draft genome assembly and genetic map depicting the relative positions of 4614 gene sequences. The genes from 26 syntenic segments highlighted in red contain 78% of all gene sequences on both maps.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Sequences from 6994 single copy, mapped sunflower genes (<a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Bowers <em>et al.</em> 2012a</a>) were compared to the safflower genome assembly, and 6419 (92%) could be placed on scaffolds within that assembly. Of these, 2777 (43%) matched genetically mapped scaffolds. Inspection of <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> reveals a more complex pattern of synteny <em>vs.</em> the sunflower genome, with each region of the safflower genome corresponding to two different regions in sunflower. This pattern is due to a previously described polyploidy event that occurred near the base of the Heliantheae on the lineage leading to sunflower (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barker <em>et al.</em> 2008</a>). In part due to this genome duplication, a large number of syntenic segments (50 in total) could be detected between the two species. These syntenic segments contained 74% of the 2777 sequences localized in both genomes. Once again, this result suggests that a higher than random fraction of the single copy gene sequences is present in the successfully mapped safflower scaffolds.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Synteny between the sunflower genetic map and saf flower draft genome assembly and genetic map showing the relative position of 2777 gene sequences. The genes from 50 syntenic segments highlighted in red contain 74% of all gene sequences on both maps." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-135970511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Synteny between the sunflower genetic map and saf flower draft genome assembly and genetic map showing the relative position of 2777 gene sequences. The genes from 50 syntenic segments highlighted in red contain 74% of all gene sequences on both maps.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2203/F5.medium.gif" width="432" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/7/2203/F5.medium.gif" width="432" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2203/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468538" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">Synteny between the sunflower genetic map and saf flower draft genome assembly and genetic map showing the relative position of 2777 gene sequences. The genes from 50 syntenic segments highlighted in red contain 74% of all gene sequences on both maps.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Conclusions</h3><p id="p-34">The general approach outlined above for the sequence-based construction of genetic maps has the potential to efficiently produce maps containing 100–1000 × as many loci as most genetic maps produced in recent years. At the same time, this approach can also offer substantial cost and time savings over older genotyping technologies. However, due to difficulties associated with accurately identifying heterozygous genotypes when dealing with low coverage data, this method is best applied to highly homozygous mapping populations (<em>e.g.</em>, RILs or doubled haploids). Moreover, despite the large number of mapped loci, the resolution of such maps is still limited by the size of the mapping population. Despite these limitations, such maps can be powerful tools for comparative genomic analyses, assisting in the assembly of high quality genomes in otherwise uncharacterized species, and/or identifying candidate genes underlying previously mapped traits (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Nambeesan <em>et al.</em> 2015</a>). If more widely adopted, even minor crops could have better genetic mapping resources than were available in the most well-characterized species a few years ago.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">This work was supported by a grant from the NSF Plant Genome Research Program (DBI-0820451 to JMB) and an NSF Doctoral Dissertation Improvement Grant (DEB-1110350 to JMB and SAP).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-36">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026690/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.026690/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-37"><em>Communicating editor: E. Akhunov</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-12-31"><span class="received-label">Received </span>December 31, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-05-16"><span class="accepted-label">Accepted </span>May 16, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Bowers <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.1" data-doi="10.3906/biy-1308-22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amini</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arzani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karami</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Effect of water deficiency on seed quality and physiological traits of different safflower genotypes.</span> <abbr class="cit-jnl-abbrev">Turk. J. Biol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTurk.%2BJ.%2BBiol.%26rft.volume%253D38%26rft.spage%253D271%26rft_id%253Dinfo%253Adoi%252F10.3906%252Fbiy-1308-22%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3906/biy-1308-22&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.2" data-doi="10.1104/pp.113.228213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ariyadasa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nussbaumer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulte</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frenkel</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Sequence-Ready Physical Map of Barley Anchored Genetically by Two Million Single-Nucleotide Polymorphisms.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">412</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.113.228213%26rft_id%253Dinfo%253Apmid%252F24243933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjQvMS80MTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.3" data-doi="10.1093/molbev/msn187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matvienko</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multiple Paleopolyploidizations during the Evolution of the Compositae Reveal Parallel Patterns of Duplicate Gene Retention after Millions of Years.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2445</span>–<span class="cit-lpage">2455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn187%26rft_id%253Dinfo%253Apmid%252F18728074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMS8yNDQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.4" data-doi="10.1007/BF02779015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beattie</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varma</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dao</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Hybridization of DNA targets to glass-tethered oligonucleotide probes.</span> <abbr class="cit-jnl-abbrev">Mol. Biotechnol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiotechnology%26rft.stitle%253DMol%2BBiotechnol%26rft.aulast%253DBeattie%26rft.auinit1%253DW.%2BG.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D213%26rft.epage%253D225%26rft.atitle%253DHybridization%2Bof%2BDNA%2Btargets%2Bto%2Bglass-tethered%2Boligonucleotide%2Bprobes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02779015%26rft_id%253Dinfo%253Apmid%252F8680928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02779015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8680928&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TL37300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skolnick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Construction of a genetic-linkage map in man using restriction fragment length polymorphisms.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">314</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBotstein%26rft.auinit1%253DD.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253D314%26rft.epage%253D331%26rft.atitle%253DConstruction%2Bof%2Ba%2Bgenetic%2Blinkage%2Bmap%2Bin%2Bman%2Busing%2Brestriction%2Bfragment%2Blength%2Bpolymorphisms.%26rft_id%253Dinfo%253Apmid%252F6247908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6247908&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JX05900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.6" data-doi="10.1534/g3.112.002659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bachlava</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunick</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Development of a 10,000 Locus Genetic Map of the Sunflower Genome Based on Multiple Crosses.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">721</span>–<span class="cit-lpage">729</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DBowers%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253D721%26rft.epage%253D729%26rft.atitle%253DDevelopment%2Bof%2Ba%2B10%252C000%2BLocus%2BGenetic%2BMap%2Bof%2Bthe%2BSunflower%2BGenome%2BBased%2Bon%2BMultiple%2BCrosses%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.002659%26rft_id%253Dinfo%253Apmid%252F22870395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjIvNy83MjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.7" data-doi="10.1371/journal.pone.0051360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nambeesan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Development of an Ultra-Dense Genetic Map of the Sunflower Genome Based on Single-Feature Polymorphisms.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e51360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253De51360%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0051360%26rft_id%253Dinfo%253Apmid%252F23284684%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0051360&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284684&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.8" data-doi="10.1186/s13059-015-0582-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buluc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georganas</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A whole-genome shotgun approach for assembling and anchoring the hexaploid bread wheat genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0582-8%26rft_id%253Dinfo%253Apmid%252F25637298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0582-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25637298&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.9" data-doi="10.1093/bib/bbp045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheema</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dicks</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Computational approaches and software tools for genetic linkage map estimation in plants.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbp045%26rft_id%253Dinfo%253Apmid%252F19933208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjg6IjEwLzYvNTk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.10" data-doi="10.1101/gr.4319006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drouaud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camilleri</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourguignon</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canaguier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Variation in crossing-over rates across chromosome 4 of Arabidopsis thaliana reveals the presence of meiotic recombination “hot spots”.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4319006%26rft_id%253Dinfo%253Apmid%252F16344568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzEvMTA2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.11" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Robust, Simple Genotyping-by-Sequencing (GBS) Approach for High Diversity Species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.12" data-doi="10.3923/ajps.2010.299.306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Emongor</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Safflower (Carthamus tinctorius L.) the Underutilized and Neglected Crop: A Review.</span> <abbr class="cit-jnl-abbrev">Asian J. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAsian%2BJ.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D299%26rft.atitle%253D%2528CARTHAMUS%2BTINCTORIUS%2BL.%2529%2BTHE%2BUNDERUTILIZED%2BAND%2BNEGLECTED%2BCROP%253A%2BA%2BREVIEW%26rft_id%253Dinfo%253Adoi%252F10.3923%252Fajps.2010.299.306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3923/ajps.2010.299.306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.13" data-doi="10.1038/nature06258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frazer</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballinger</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinds</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuve</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A second generation human haplotype map of over 3.1 million SNPs.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFrazer%26rft.auinit1%253DK.%2BA.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D851%26rft.epage%253D861%26rft.atitle%253DA%2Bsecond%2Bgeneration%2Bhuman%2Bhaplotype%2Bmap%2Bof%2Bover%2B3.1%2Bmillion%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06258%26rft_id%253Dinfo%253Apmid%252F17943122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943122&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.14" data-doi="10.1093/aob/mcj050"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garnatje</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilatersana</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valles</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome size variation in the genus Carthamus (Asteraceae, Cardueae): Systematic implications and additive changes during allopolyploidization.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcj050%26rft_id%253Dinfo%253Apmid%252F16390843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk3LzMvNDYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.15" data-doi="10.1016/j.agwat.2012.10.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghamarnia</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gholamian</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The effect of saline shallow ground and surface water under deficit irrigation on (Carthamus tinctorius L.) in semi arid condition.</span> <abbr class="cit-jnl-abbrev">Agric. Water Manage.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgric.%2BWater%2BManage.%26rft.volume%253D118%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.agwat.2012.10.028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.agwat.2012.10.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.16" data-doi="10.1073/pnas.1017351108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gnerre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacCallum</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przybylski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">1513</span>–<span class="cit-lpage">1518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1017351108%26rft_id%253Dinfo%253Apmid%252F21187386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA4LzQvMTUxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moyle</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Sequencing, Assembling, and Correcting Draft Genomes Using Recombinant Populations. G3 (Bethesda) 4: 669–679.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.18" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput genotyping by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.19" data-doi="10.1139/b11-032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berges</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Progress towards a reference genome for sunflower.</span> <abbr class="cit-jnl-abbrev">Botany-Botanique</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBotany-Botanique%26rft.volume%253D89%26rft.spage%253D429%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fb11-032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/b11-032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.20" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.21" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.22" data-doi="10.1186/2047-217X-1-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587118&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.23" data-doi="10.1186/gm205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mardis</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The $1,000 genome, the $100,000 analysis?</span> <abbr class="cit-jnl-abbrev">Genome Med.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenome%2BMed%26rft.aulast%253DMardis%26rft.auinit1%253DE.%2BR.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253D84%26rft.epage%253D84%26rft.atitle%253DThe%2B%25241%252C000%2Bgenome%252C%2Bthe%2B%2524100%252C000%2Banalysis%253F%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgm205%26rft_id%253Dinfo%253Apmid%252F21114804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gm205&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21114804&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.24" data-doi="10.1007/s00334-009-0212-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marinova</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riehl</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Carthamus species in the ancient Near East and south-eastern Europe: archaeobotanical evidence for their distribution and use as a source of oil.</span> <abbr class="cit-jnl-abbrev">Veg. Hist. Archaeobot.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeg.%2BHist.%2BArchaeobot.%26rft.volume%253D18%26rft.spage%253D341%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00334-009-0212-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00334-009-0212-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.25" data-doi="10.1111/tpj.12319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muehlbauer</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokhsar</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmutz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ).</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">718</span>–<span class="cit-lpage">727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D76%26rft.spage%253D718%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.12319%26rft_id%253Dinfo%253Apmid%252F23998490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.12319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23998490&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326138100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.26" data-doi="10.1126/science.850784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKusick</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruddle</span>  <span class="cit-name-given-names">F. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Status of Gene Map of Human Chromosomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">390</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMcKusick%26rft.auinit1%253DV.%26rft.volume%253D196%26rft.issue%253D4288%26rft.spage%253D390%26rft.epage%253D405%26rft.atitle%253DThe%2Bstatus%2Bof%2Bthe%2Bgene%2Bmap%2Bof%2Bthe%2Bhuman%2Bchromosomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.850784%26rft_id%253Dinfo%253Apmid%252F850784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIxOTYvNDI4OC8zOTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.27" data-doi="10.1186/s12870-015-0458-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nambeesan</span>  <span class="cit-name-given-names">S. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mandel</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marek</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Association mapping in sunflower (Helianthus annuus L.) reveals independent control of apical <em>vs.</em> basal branching.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D15%26rft.spage%253D84%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-015-0458-9%26rft_id%253Dinfo%253Apmid%252F25887675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-015-0458-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25887675&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.28" data-doi="10.1038/nature07723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sorghum bicolor genome and the diversification of grasses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D457%26rft.issue%253D7229%26rft.spage%253D551%26rft.epage%253D556%26rft.atitle%253DThe%2BSorghum%2Bbicolor%2Bgenome%2Band%2Bthe%2Bdiversification%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07723%26rft_id%253Dinfo%253Apmid%252F19189423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19189423&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262852200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.29" data-doi="10.1186/1471-2229-14-43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearl</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Chin-Wo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic analysis of safflower domestication.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D14%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-14-43%26rft_id%253Dinfo%253Apmid%252F24502326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-14-43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24502326&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.30" data-doi="10.1038/ncomms2833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grassa</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moyers</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic islands of divergence are not affected by geography of speciation in sunflowers.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D1827%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2833%26rft_id%253Dinfo%253Apmid%252F23652015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms2833&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23652015&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.31" data-doi="10.1007/BF02545945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Safflower - Due for a rebound?</span> <abbr class="cit-jnl-abbrev">J. Am. Oil Chem. Soc.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">1286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BOil%2BChem.%2BSoc.%26rft.volume%253D62%26rft.spage%253D1286%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02545945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02545945&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.32" data-doi="10.1002/jez.1400140104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1913</span> <span class="cit-article-title">The linear arrangement of six sex-linked factors in Drosophila, as shown by their mode of association.</span> <abbr class="cit-jnl-abbrev">Journal of Experimental Zoology Philadelphia</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BExperimental%2BZoology%2BPhiladelphia%26rft.volume%253D14%26rft.spage%253D43%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.1400140104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.1400140104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashrafi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> An Ultra-High-Density, Transcript-Based, Genetic Map of Lettuce. G3 (Bethesda): 617–631.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.34" data-doi="10.7554/eLife.01426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wijnker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klasen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genomic landscape of meiotic crossovers and gene conversions in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e01426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.01426%26rft_id%253Dinfo%253Apmid%252F24347547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjIvMC9lMDE0MjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2203.35" data-doi="10.1038/324380a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Youssoufian</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kazazian</span>  <span class="cit-name-given-names">H. H.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aronis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsiftis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Recurrent mutations in haemophilia A give evidence for CpG mutation hotspots.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">380</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYoussoufian%26rft.auinit1%253DH.%26rft.volume%253D324%26rft.issue%253D6095%26rft.spage%253D380%26rft.epage%253D382%26rft.atitle%253DRecurrent%2Bmutations%2Bin%2Bhaemophilia%2BA%2Bgive%2Bevidence%2Bfor%2BCpG%2Bmutation%2Bhotspots.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F324380a0%26rft_id%253Dinfo%253Apmid%252F3097553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/324380a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3097553&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/7/2203.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/2195" title="An Improved Binary Vector and Escherichia coli Strain for Agrobacterium tumefaciens-Mediated Plant Transformation" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/2213" title="Large Deletions at the SHOX Locus in the Pseudoautosomal Region Are Associated with Skeletal Atavism in Shetland Ponies" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/7"><span>Volume 6 Issue 7, July 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/7.cover-source.jpg?itok=E84OOyGI" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2203" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-V3Hx8VHFyo3vKso3C_Wxvb7rcS-8ECtQCV8U0vMU-y4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/7/2203" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/7/2203" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/7/2203" class="active" data-icon-position="" data-hide-link-title="0">Genetic Mapping of Millions of SNPs in Safflower (Carthamus tinctorius L.) via Whole-Genome Resequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460419" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-4WzSj-2uyEsJtx7e_9TYCpRKCvJiA77DakoQXoOOrvA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/2203.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/7/2203" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-laWlFMo5aM_el3IXMW0SmKxJYqPt-7cat0FGCviMs4s" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/7/2203" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/2203.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460419" id="node460419--2" data-pisa="ggg;6/7/2203" data-pisa-master="ggg;g3.115.026690" data-apath="/ggg/6/7/2203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Mapping of Millions of SNPs in Safflower (<em>Carthamus tinctorius</em> L.) via Whole-Genome Resequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">John E. Bowers</span>, <span class="highwire-citation-author" data-delta="1">Stephanie A. Pearl</span> and <span class="highwire-citation-author" data-delta="2">John M. Burke</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2203-2211;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026690 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460419--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John E. Bowers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BBowers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowers%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BE.%2BBowers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stephanie A. Pearl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephanie%2BA.%2BPearl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pearl%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephanie%2BA.%2BPearl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John M. Burke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BBurke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burke%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BM.%2BBurke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmburke@uga.edu" class="" data-icon-position="" data-hide-link-title="0">jmburke@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/7/2203" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/7/2203&amp;title=Genetic%20Mapping%20of%20Millions%20of%20SNPs%20in%20Safflower%20%28Carthamus%20tinctorius%20L.%29%20via%20Whole-Genome%20Resequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/7/2203&amp;title=Genetic%20Mapping%20of%20Millions%20of%20SNPs%20in%20Safflower%20%28Carthamus%20tinctorius%20L.%29%20via%20Whole-Genome%20Resequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/7/2203&amp;title=Genetic%20Mapping%20of%20Millions%20of%20SNPs%20in%20Safflower%20%28Carthamus%20tinctorius%20L.%29%20via%20Whole-Genome%20Resequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/7/2203&amp;text=Genetic%20Mapping%20of%20Millions%20of%20SNPs%20in%20Safflower%20%28Carthamus%20tinctorius%20L.%29%20via%20Whole-Genome%20Resequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/7/2203&amp;title=Genetic%20Mapping%20of%20Millions%20of%20SNPs%20in%20Safflower%20%28Carthamus%20tinctorius%20L.%29%20via%20Whole-Genome%20Resequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/7/2203&amp;t=Genetic%20Mapping%20of%20Millions%20of%20SNPs%20in%20Safflower%20%28Carthamus%20tinctorius%20L.%29%20via%20Whole-Genome%20Resequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/7/2203&amp;title=Genetic%20Mapping%20of%20Millions%20of%20SNPs%20in%20Safflower%20%28Carthamus%20tinctorius%20L.%29%20via%20Whole-Genome%20Resequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/7/2203&amp;title=Genetic%20Mapping%20of%20Millions%20of%20SNPs%20in%20Safflower%20%28Carthamus%20tinctorius%20L.%29%20via%20Whole-Genome%20Resequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460419/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460419" id="node460419--3" data-pisa="ggg;6/7/2203" data-pisa-master="ggg;g3.115.026690" data-apath="/ggg/6/7/2203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genetic Mapping of Millions of SNPs in Safflower (<em>Carthamus tinctorius</em> L.) via Whole-Genome Resequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">John E. Bowers</span>, <span class="highwire-citation-author" data-delta="1">Stephanie A. Pearl</span> and <span class="highwire-citation-author" data-delta="2">John M. Burke</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2203-2211;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.026690 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460419--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">John E. Bowers </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BE.%2BBowers%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bowers%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BE.%2BBowers%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Stephanie A. Pearl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephanie%2BA.%2BPearl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pearl%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephanie%2BA.%2BPearl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John M. Burke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Miller Plant Sciences, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BBurke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burke%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohn%2BM.%2BBurke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmburke@uga.edu" class="" data-icon-position="" data-hide-link-title="0">jmburke@uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2203&amp;rft.epage=2211&amp;rft.atitle=Genetic%20Mapping%20of%20Millions%20of%20SNPs%20in%20Safflower%20%28Carthamus%20tinctorius%20L.%29%20via%20Whole-Genome%20Resequencing&amp;rft.volume=6&amp;rft.issue=7&amp;rft.date=2016-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Bowers%2C+John+E.&amp;rft.au=Pearl%2C+Stephanie+A.&amp;rft.au=Burke%2C+John+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460419/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460419/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460419/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460419/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460419/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460419/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460419/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460419/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460419/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460419/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460419/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460419/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/7/2203" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/7/2203.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/7/2203.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/7/2203.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/7/2203.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
