<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/7/1819" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/7/1819.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/7/1819.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/7/1819.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/7/1819/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/7/1819.atom" />
<meta name="HW.pisa" content="ggg;6/7/1819" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction of Gene Bank Wheat Landraces" />
<meta name="DC.Identifier" content="10.1534/g3.116.029637" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Crossa et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="This study examines genomic prediction within 8416 Mexican landrace accessions and 2403 Iranian landrace accessions stored in gene banks. The Mexican and Iranian collections were evaluated in separate field trials, including an optimum environment for several traits, and in two separate environments (drought, D and heat, H) for the highly heritable traits, days to heading (DTH), and days to maturity (DTM). Analyses accounting and not accounting for population structure were performed. Genomic prediction models include genotype × environment interaction (G × E). Two alternative prediction strategies were studied: (1) random cross-validation of the data in 20% training (TRN) and 80% testing (TST) (TRN20-TST80) sets, and (2) two types of core sets, “diversity” and “prediction”, including 10% and 20%, respectively, of the total collections. Accounting for population structure decreased prediction accuracy by 15–20% as compared to prediction accuracy obtained when not accounting for population structure. Accounting for population structure gave prediction accuracies for traits evaluated in one environment for TRN20-TST80 that ranged from 0.407 to 0.677 for Mexican landraces, and from 0.166 to 0.662 for Iranian landraces. Prediction accuracy of the 20% diversity core set was similar to accuracies obtained for TRN20-TST80, ranging from 0.412 to 0.654 for Mexican landraces, and from 0.182 to 0.647 for Iranian landraces. The predictive core set gave similar prediction accuracy as the diversity core set for Mexican collections, but slightly lower for Iranian collections. Prediction accuracy when incorporating G × E for DTH and DTM for Mexican landraces for TRN20-TST80 was around 0.60, which is greater than without the G × E term. For Iranian landraces, accuracies were 0.55 for the G × E model with TRN20-TST80. Results show promising prediction accuracies for potential use in germplasm enhancement and rapid introgression of exotic germplasm into elite materials." />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Diego Jarquín" />
<meta name="DC.Contributor" content="Jorge Franco" />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="Juan Burgueño" />
<meta name="DC.Contributor" content="Carolina Saint-Pierre" />
<meta name="DC.Contributor" content="Prashant Vikram" />
<meta name="DC.Contributor" content="Carolina Sansaloni" />
<meta name="DC.Contributor" content="Cesar Petroli" />
<meta name="DC.Contributor" content="Deniz Akdemir" />
<meta name="DC.Contributor" content="Clay Sneller" />
<meta name="DC.Contributor" content="Matthew Reynolds" />
<meta name="DC.Contributor" content="Maria Tattaris" />
<meta name="DC.Contributor" content="Thomas Payne" />
<meta name="DC.Contributor" content="Carlos Guzman" />
<meta name="DC.Contributor" content="Roberto J. Peña" />
<meta name="DC.Contributor" content="Peter Wenzl" />
<meta name="DC.Contributor" content="Sukhwinder Singh" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Prediction of Gene Bank Wheat Landraces" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;This study examines genomic prediction within 8416 Mexican landrace accessions and 2403 Iranian landrace accessions stored in gene banks. The Mexican and Iranian collections were evaluated in separate field trials, including an optimum environment for several traits, and in two separate environments (drought, D and heat, H) for the highly heritable traits, days to heading (DTH), and days to maturity (DTM). Analyses accounting and not accounting for population structure were performed. Genomic prediction models include genotype&amp;nbsp;×&amp;nbsp;environment interaction (G&amp;nbsp;×&amp;nbsp;E). Two alternative prediction strategies were studied: (1) random cross-validation of the data in 20% training (TRN) and 80% testing (TST) (TRN20-TST80) sets, and (2) two types of core sets, “diversity” and “prediction”, including 10% and 20%, respectively, of the total collections. Accounting for population structure decreased prediction accuracy by 15–20% as compared to prediction accuracy obtained when not accounting for population structure. Accounting for population structure gave prediction accuracies for traits evaluated in one environment for TRN20-TST80 that ranged from 0.407 to 0.677 for Mexican landraces, and from 0.166 to 0.662 for Iranian landraces. Prediction accuracy of the 20% diversity core set was similar to accuracies obtained for TRN20-TST80, ranging from 0.412 to 0.654 for Mexican landraces, and from 0.182 to 0.647 for Iranian landraces. The predictive core set gave similar prediction accuracy as the diversity core set for Mexican collections, but slightly lower for Iranian collections. Prediction accuracy when incorporating G&amp;nbsp;×&amp;nbsp;E for DTH and DTM for Mexican landraces for TRN20-TST80 was around 0.60, which is greater than without the G&amp;nbsp;×&amp;nbsp;E term. For Iranian landraces, accuracies were 0.55 for the G&amp;nbsp;×&amp;nbsp;E model with TRN20-TST80. Results show promising prediction accuracies for potential use in germplasm enhancement and rapid introgression of exotic germplasm into elite materials.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/07/01" />
<meta name="citation_mjid" content="ggg;6/7/1819" />
<meta name="citation_id" content="6/7/1819" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/7/1819" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/7/1819.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/7/1819.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/7/1819.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.029637" />
<meta name="citation_pmid" content="27172218" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1819" />
<meta name="citation_lastpage" content="1834" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="suk.singh@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Diego Jarquín" />
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska-Lincoln, 321 Keim Hall, Lincoln, Nebraska 68583-0915" />
<meta name="citation_author" content="Jorge Franco" />
<meta name="citation_author_institution" content="Departamento de Biometría, Estadística y Computación, Facultad de Agronomía, Universidad de la República (Udelar), Paysandú, Uruguay" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Post-Graduados, Montecillos, Edo. de Mexico, 56230 Mexico" />
<meta name="citation_author" content="Juan Burgueño" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Carolina Saint-Pierre" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Prashant Vikram" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Carolina Sansaloni" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Cesar Petroli" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Deniz Akdemir" />
<meta name="citation_author_institution" content="Department of Plant Breeding &amp; Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Clay Sneller" />
<meta name="citation_author_institution" content="Department of Horticulture and Crop Science, Ohio State University, Wooster, Ohio 44691" />
<meta name="citation_author" content="Matthew Reynolds" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Maria Tattaris" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Thomas Payne" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Carlos Guzman" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Roberto J. Peña" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Peter Wenzl" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author" content="Sukhwinder Singh" />
<meta name="citation_author_institution" content="Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_email" content="suk.singh@cgiar.org" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Akdemir;citation_author=J. I. Sanchez;citation_author=J.-L. Jannink;citation_title=Optimization of genomic selection training populations with a genetic algorithm.;citation_pages=38;citation_volume=47;citation_year=2015;citation_pmid=25943105;citation_doi=10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=M. Ayala;citation_author=C. Guzman;citation_author=J. B. Alvarez;citation_author=R. J. Peña;citation_title=Characterization of genetic diversity of puroindoline genes in Mexican wheat landraces.;citation_pages=53-63;citation_volume=190;citation_year=2013;citation_doi=10.1007/s10681-012-0773-2" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=A. H. D. Brown;citation_title=Core collections: a practical approach to genetic resource management.;citation_pages=818-824;citation_volume=31;citation_year=1989;citation_doi=10.1139/g89-144" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=L. Caballero;citation_author=R. J. Peña;citation_author=L. M. Martin;citation_author=J. B. Alvarez;citation_title=Characterization of Mexican Creole wheat landraces in relation to morphological characteristics and HMW glutenin subunit composition.;citation_pages=657-665;citation_volume=57;citation_year=2010;citation_doi=10.1007/s10722-009-9501-8" />
<meta name="citation_reference" content="citation_author=J. Carling;citation_author=K. Uszynska;citation_author=D. Jaccoud;citation_author=A. Machado;citation_author=A. Hopper;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=H. D. Daetwyler;citation_author=K. E. Kemper;citation_author=J. H. J. van der Werf;citation_author=B. J. Hayes;citation_title=Components of the accuracy of genomic prediction in a multi-breed sheep population.;citation_pages=3375-3384;citation_volume=90;citation_year=2015;citation_doi=10.2527/jas2011&amp;ndash;4557" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=G. de los Campos;citation_author=Y. Veturi;citation_author=A. I. Vazquez;citation_title=Incorporating genetic heterogeneity in whole-genome regressions using interactions.;citation_pages=467-490;citation_volume=20;citation_year=2015;citation_pmid=26660276;citation_doi=10.1007/s13253-015-0222-5" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Franco;citation_author=J. Crossa;citation_author=S. Taba;citation_author=H. A. Shands;citation_title=A Sampling Strategy for Conserving Genetic Diversity when Forming Core Subsets;citation_pages=1035-1044;citation_volume=45;citation_year=2005;citation_issue=3;citation_doi=10.2135/cropsci2004.0292" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Franco;citation_author=J. Crossa;citation_author=M. L. Warburton;citation_author=S. Taba;citation_title=Sampling Strategies for Conserving Maize Diversity When Forming Core Subsets Using Genetic Markers;citation_pages=854-864;citation_volume=46;citation_year=2006;citation_issue=2;citation_doi=10.2135/cropsci2005.07-0201" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Franco;citation_author=J. Crossa;citation_author=S. Desphande;citation_title=Hierarchical multiple-factor analysis for classifying genotypes based on phenotypic and genetic data.;citation_pages=105-117;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.01.0053" />
<meta name="citation_reference" content="citation_title=Crop genetic resources: Conservation and evaluation;citation_title=Plant genetic resources today: a critical appraisal;citation_author=O. H. Frankel;citation_author=A. H. D. Brown;citation_pages=249-257;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=A. R. Gilmour;citation_author=B. R. Cullis;citation_author=A. P. Verbyla;citation_title=Accounting for natural and extraneous variation in the analysis of field experiments.;citation_pages=269-293;citation_volume=2;citation_year=1997;citation_issue=3;citation_doi=10.2307/1400446" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. de los Campos;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=G. Gorjanc;citation_author=J. Jenk;citation_author=S. J. Hearne;citation_author=J. M. Hickey;citation_title=Initiating maize pre-breeding programs using genomic selection to harness polygenic variation from landrace populations.;citation_pages=30;citation_volume=17;citation_year=2016;citation_pmid=26732811;citation_doi=10.1186/s12864-015-2345-z" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=Z. Gou;citation_author=D. M. Tucker;citation_author=C. J. Basten;citation_title=The impact of population structure on genomic prediction in stratified populations.;citation_pages=749-762;citation_volume=127;citation_year=2014;citation_issue=3;citation_pmid=24452438;citation_doi=10.1007/s00122-013-2255-x" />
<meta name="citation_reference" content="citation_journal_title=J. Cereal Sci.;citation_author=C. Guzman;citation_author=A. S. Medina-Larque;citation_author=G. Velu;citation_author=H. González-Santoyo;citation_author=R. P. Singh;citation_title=Use of wheat genetic resources to develop biofortified wheat with enhanced grain zinc and iron concentrations and desirable processing quality.;citation_pages=617-622;citation_volume=60;citation_year=2014;citation_doi=10.1016/j.jcs.2014.07.006" />
<meta name="citation_reference" content="citation_journal_title=J. Cereal Sci.;citation_author=C. Guzman;citation_author=R. Ortega;citation_author=M. Yamamori;citation_author=R. J. Peña;citation_author=J. B. Alvarez;citation_title=Molecular characterization of two novel null waxy alleles in Mexican bread wheat landraces.;citation_pages=8-14;citation_volume=62;citation_year=2015;citation_doi=10.1016/j.jcs.2014.11.003" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H. P. Yang;citation_author=M. E. Sorrells;citation_author=J. L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. L. Jannink;citation_title=Dynamics of long-term genomic selection.;citation_pages=35-35;citation_volume=42;citation_year=2010;citation_pmid=20712894;citation_doi=10.1186/1297-9686-42-35" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Janss;citation_author=G. de los Campos;citation_author=N. Sheehan;citation_author=D. Sorensen;citation_title=Inferences from genomic models in stratified populations.;citation_volume=192;citation_year=2012;citation_issue=2;citation_pmid=22813891;citation_doi=10.1534/genetics.112.141143" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. D. Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_author=M. López-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_author=J. L. Jannink;citation_author=R. P. Singh;citation_author=E. Autrique;citation_author=. de los Campos;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_issue=2;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. M. González-Camacho;citation_author=J. Crossa;citation_author=Y. Manes;citation_author=. Dreisigacker;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. C. Reif;citation_author=A. E. Melchinger;citation_author=M. Frisch;citation_title=Genetical and Mathematical Properties of Similarity and Dissimilarity Coefficients Applied in Plant Breeding and Seed Bank Management;citation_pages=1-7;citation_volume=45;citation_year=2005;citation_issue=1;citation_doi=10.2135/cropsci2004.0517" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_title=Genetic gain from phenotypic and genomic selection for quantitative resistance to stem rust of wheat.;citation_author=J. Rutkoski;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=B. Bhavani;citation_author=J. Poland;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. H. Sallam;citation_author=J. B. Endelman;citation_author=J.-L. Jannink;citation_author=K. P. Smith;citation_title=Assessing genomic selection prediction accuracy in a dynamic barley breeding population.;citation_pages=1-15;citation_volume=8;citation_year=2015;citation_issue=1;citation_doi=10.3835/plantgenome2014.05.0020" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=T. Schulz-Streeck;citation_author=J. O. Ogutu;citation_author=A. Gordillo;citation_author=Z. Karaman;citation_author=C. Knaak;citation_title=Genomic selection allowing for marker-by-environment interaction.;citation_pages=532-538;citation_volume=132;citation_year=2013;citation_issue=2;citation_doi=10.1111/pbr.12105" />
<meta name="citation_reference" content="citation_journal_title=Interbull Bull;citation_author=P. M. VanRaden;citation_title=Genomic measures of relationship and inbreeding.;citation_pages=33-36;citation_volume=37;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_title=Unlocking the genetic diversity of Creole wheats.;citation_author=P. Vikram;citation_author=J. Franco;citation_author=J. Burgueno-Ferrera;citation_author=H. Li;citation_author=D. Sehgal;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=J. H. Ward;citation_title=Hierarchical grouping to optimize an objective function.;citation_pages=236-244;citation_volume=58;citation_year=1963;citation_doi=10.2307/2282967" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=V. S. Windhausen;citation_author=G. N. Atlin;citation_author=J. Crossa;citation_author=J. M. Hickey;citation_author=P. Grudloyma;citation_title=Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.;citation_pages=1427-1436;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_title=Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS.;citation_author=X. Zhang;citation_author=P. Pérez-Rodríguez;citation_author=K. Semagn;citation_author=Y. Beyene;citation_author=R. Babu;citation_year=2014" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction of Gene Bank Wheat Landraces" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="twitter:description" content="This study examines genomic prediction within 8416 Mexican landrace accessions and 2403 Iranian landrace accessions stored in gene banks. The Mexican and Iranian collections were evaluated in separate field trials, including an optimum environment for several traits, and in two separate environments (drought, D and heat, H) for the highly heritable traits, days to heading (DTH), and days to maturity (DTM). Analyses accounting and not accounting for population structure were performed. Genomic prediction models include genotype × environment interaction (G × E). Two alternative prediction strategies were studied: (1) random cross-validation of the data in 20% training (TRN) and 80% testing (TST) (TRN20-TST80) sets, and (2) two types of core sets, “diversity” and “prediction”, including 10% and 20%, respectively, of the total collections. Accounting for population structure decreased prediction accuracy by 15–20% as compared to prediction accuracy obtained when not accounting for population structure. Accounting for population structure gave prediction accuracies for traits evaluated in one environment for TRN20-TST80 that ranged from 0.407 to 0.677 for Mexican landraces, and from 0.166 to 0.662 for Iranian landraces. Prediction accuracy of the 20% diversity core set was similar to accuracies obtained for TRN20-TST80, ranging from 0.412 to 0.654 for Mexican landraces, and from 0.182 to 0.647 for Iranian landraces. The predictive core set gave similar prediction accuracy as the diversity core set for Mexican collections, but slightly lower for Iranian collections. Prediction accuracy when incorporating G × E for DTH and DTM for Mexican landraces for TRN20-TST80 was around 0.60, which is greater than without the G × E term. For Iranian landraces, accuracies were 0.55 for the G × E model with TRN20-TST80. Results show promising prediction accuracies for potential use in germplasm enhancement and rapid introgression of exotic germplasm into elite materials." />
<meta name="og-title" property="og:title" content="Genomic Prediction of Gene Bank Wheat Landraces" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/7/1819" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="This study examines genomic prediction within 8416 Mexican landrace accessions and 2403 Iranian landrace accessions stored in gene banks. The Mexican and Iranian collections were evaluated in separate field trials, including an optimum environment for several traits, and in two separate environments (drought, D and heat, H) for the highly heritable traits, days to heading (DTH), and days to maturity (DTM). Analyses accounting and not accounting for population structure were performed. Genomic prediction models include genotype × environment interaction (G × E). Two alternative prediction strategies were studied: (1) random cross-validation of the data in 20% training (TRN) and 80% testing (TST) (TRN20-TST80) sets, and (2) two types of core sets, “diversity” and “prediction”, including 10% and 20%, respectively, of the total collections. Accounting for population structure decreased prediction accuracy by 15–20% as compared to prediction accuracy obtained when not accounting for population structure. Accounting for population structure gave prediction accuracies for traits evaluated in one environment for TRN20-TST80 that ranged from 0.407 to 0.677 for Mexican landraces, and from 0.166 to 0.662 for Iranian landraces. Prediction accuracy of the 20% diversity core set was similar to accuracies obtained for TRN20-TST80, ranging from 0.412 to 0.654 for Mexican landraces, and from 0.182 to 0.647 for Iranian landraces. The predictive core set gave similar prediction accuracy as the diversity core set for Mexican collections, but slightly lower for Iranian collections. Prediction accuracy when incorporating G × E for DTH and DTM for Mexican landraces for TRN20-TST80 was around 0.60, which is greater than without the G × E term. For Iranian landraces, accuracies were 0.55 for the G × E model with TRN20-TST80. Results show promising prediction accuracies for potential use in germplasm enhancement and rapid introgression of exotic germplasm into elite materials." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
    <title>Genomic Prediction of Gene Bank Wheat Landraces | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"EbtxaMXEvJfK635L6Ze98w3Cn3nshOWVLhAqE2i8M5A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460418","apath":"\/ggg\/6\/7\/1819.atom","pisa":"ggg;6\/7\/1819","ac":{"\/ggg\/6\/7\/1819.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/7\/1819.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1819"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1819"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/7\/1819?ijkey=9dbcbbdd3f62da788d54f71ddf03dfdf91f61712\u0026keytype2=tf_ipsecsha":true,"\/content\/6\/7\/1819":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qlncq7XOFAbp5AKirnCxhGqpy89Y9vK4W94JnVu1OUZTZaf\\\\\\\/msf1\\\\\\\/rU6IPvqdSLOc2GWWSbF\\\\\\\/b7Q5aY3BdRH9LvuHNIhMnuGXBW5i5MXMOSUdy7Lg6m9D++7N5fMiJozQcXcMI4BKSZRQYgHCZogxqMtLEWPL+AFQwG+7UYV2xCKG4YszdzEpO9jMDGFc8phjKMc4wLakjUHZx8JcZbVgCGFBhoYxh+Td33ER0AFaX9tKzVJIGPm01+t4EKUwQuDqRXptvF945Nb6RoT9FeWcQDWwClK3dDDE3Mda7NtgGTRTFP4PnedRM1vDCpmMlB93UCFSiFghJNCB88F3QEEIrCgcLPndB8fi7\\\\\\\/DFgEdgfQrL3kswRF4WX+yRQDa89btuYxgcqnUNVVTMdaBvgKSlM3iTqIlqCeHGDyWutbZo0WYuMbEsKRxIVQnqjaHvuE8uesk2TgyAwFSm3AjimHh1OwA2wMsmjLCYazqV02wcE4dbN9Ir+LLNpbVKI4gVXdfQ\\\\\\\/sotNTGFaj+PrILaNeG7yco+fj6nJAKeHxGJa7Zhsq6WfHTDph3YAN7zwZD54DDxkzaKOFiR2Rfi1JqCb40LH4CifZMSYwhkLXbqZxTxXJDRgjvsf7Lh8jFeEaBflhRlnZdCrxezaWbysV6ifLDm7mLJkY2hn9Is2ApO6+bdCej\\\\\\\/gBJFUdu6EUKEeFE\\\\\\\/EcvSuVw+OU6tZUL+8gLlG9k+9SRyaoPK5Axhqu4q73PErTRgroHMBIcS7OkKTQ+O2tZnF9cCRlFh8Au4v3Z1ovRW2Fq9a19jRp43Z4weQXSe78a73S8FDibD4cKUbfzMnPr3qHgROWHGw\\\\\\\/Z0VqbpltFCHwp3ffbD\\\\\\\/otnT\\\\\\\/gVLDaO6uP3UYbRo4xK+XbKPqPsr\\\\\\\/GQPpJY+nstxNnUGbq7kixV8yJ6cTmpmJJqWkHsJPj23PkD6ZDtaSLQvei7iO545YjZ6l9rDW4us4+MHH6\\\\\\\/i2dUwDx6rna6Jd2N0XM80TFeXoVTGOHxg2dOSJR6PdXdEtxlPR3nH8+Ii5E1SMda9SXH3Nz3nTnJZiAnFIOYZx1aTnzKZX4q1IMlMEaM0PpKe6hUEK4Z2WjZwrfCT5keOsC+vMYDthocyLwS0MWQdgPJZ+Bm5FDHkltVC+7r0+QLdBHmtuYL+U2ZZQpJyionZIxQi2x1zxtvi9QwMD3mklP\\\\\\\/J\\\\\\\/FEsp0eIMFhV8oaYKWpHavLmM2l8bnY9MoQ+1mI1+bS0lt1Nk0bKQkP\\\\\\\/hjhERPDC\\\\\\\/xTltEDLxWPf+KzxV8lbXe\\\u0022,\\\u0022iv\\\u0022:\\\u0022EBES2hRIbcohhAyL\\\\\\\/Hf99Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002251c36d42381b7a97fad2f71aeb5b675345764be908fa7646b09d94f2b716938b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Hij4hbb1tfOfFNLRLIEjVIJAaPZsCUKr1cYO3gH+dEYoEyj3jrTGFNHIMw4MPOakNDTH3i4xdaNI3\\\\\\\/NYsthb42TJ18BG47a9MQEdtI58uMf+fUwGozLouJ6o0apmiewhwp9sZOqSbtrTujgVgsdsMTLH+aqj44G3ZR1mp0nWDVr2mtoj5\\\\\\\/eguwK\\\\\\\/I3SgPpd2r5svFPE9utqs0PxszXeFhz+W1JCbZxd5z3EjWeKWoigpap+HO4wBXdQ+7\\\\\\\/khMTsQnWAJfSjHpm\\\\\\\/27GH3kWBxKUmkQX+gbZL2BVfzC4kSISPLDUIDAg\\\\\\\/OR+U7yJ8FT3YiYYy336flCYqn0Zg1MRlit6QJyB0jyYBCLdcNekOG7QhX57m6bdoDB0dlUIDfpfa+RmAkT0F0ks5nrJ28X1bWaVQgCKUkUTfMAd\\\\\\\/hO9HHNcA36sYFoU9jDlod2lnNW1219RtAu1akNPOaxfLWqg1PUJDv8xGtYkucFVKwBqPV0mk3NYx3PXfg3oki3ga76FaBABl354hQKFHQbefu349rSVO3S4TyiQxmCtma4pFhiIhk6GvKTuGtsvbEe8ZF7SJ3M0xdzrXjIVxDmiTNqbOZCsahawxjvUXRbNERgfT7YE1P8IxIWowVCQ\\\\\\\/2KnEFyMHx1yhRUabyIO4P8tx0dZI6vMGBAI0vi+pN\\\\\\\/58TxPtJckeoTH6W6AW8SrTPGI4uWdK7FIWuSxJRh9k4T1aHEiHk7x0EMlpDJ6RV6G0pXzDA3sCAXvAkg5x3OdOu6aAhx\\\\\\\/YqeRZUoJhMyYkENkJ79bel13khknIJPuw4apGLmmHy7YZQILoMI2+xCOGO\\\\\\\/PK\\\\\\\/SK2FWQPlT3qyZB+ntvmSaZsyiJR41aWToSdWm6JsiUqfZnMEY06w7nWOaCyPeePPFOzE11EvZNAdR9YlAH6gY9tV0pGkeHHckJTkhl1uksBBjLC7HTbljj3K22H6Yr36HtGxT5MuAkDbi3\\\\\\\/+K7d5Zr\\\\\\\/HWsa98LIJp2IkmoaR35NNSnQiqM6FyiNUqtQVKifZutIh4P422AIgIBY\\\\\\\/UXXIOjRCtGSqd0q9akUaDziZBs3xRFgl1qcglj17s6IuLRPhJfkIb57+PWP7WMo539vuNOzW0cUNhoAmL2ncl2h3UExnJrML4QLEoxW1evpcEZYt4KvfXqhdXJo4uQ6mS\\\\\\\/S0d6BeTt\\\\\\\/e0XC\\\\\\\/EdwPy1f6akph0GgdaXCo5zhA4fEvbAvIxdfPfghL+VWZxBKOFXlswzalHmMFctwRal3Jb2EdW7Dl5f5Viv7ct3J+z2JNXDe8cUwXTlwHAyyS7xXwc7i4UddEqfgeo40\\\\\\\/1KkkWVmM1YPFb+KC+B0Uq9FqaSNAocWWzbdTvGaugMRwB4YWP9xbTXvm3PpwNealo9FtGsut5NWbytYRH4yd\\\\\\\/ksJ1cskLKFlyTq7p4ZMKBxU++F04ucfgC6ag++Fc4fcnj+ki+Gf6Ltmd0VQ3UyhYdF3AFEc2RsPMwTaKqjyeyXOQ8ugJMIONu2iEYj1NqvrtF9tliA7qHT6e5zDAdXKNQjfSDItrabXNuiwm6Af4ZoGtrzC3VMlpnRU3OwfDBnNYJhVCRulXB28qmXo+CNikHXgiaabycdb07MfWFLOYErRpd+DP7BdXsUozvG26tsRfny0s\\\\\\\/Xl8LOsOtoJf6a32\\\\\\\/zgw++XH9FP\\\\\\\/krhxeU\\\\\\\/iy+veSLv+y3i9H8GzWtGsOXfRVmXoCibD+S9iPtmJZziNG+SgxbDysAHXCSus0OhpY3bk\\\\\\\/XQjAkAzxPHeLwAFcY4A\\\\\\\/4m4xVU6W9Qa2q1NQbS\\\\\\\/Efv+\\\\\\\/xYx+qv8gjBTh111TpXQzfpHtS6eRqeBTPHnzQVfLO8mRqf90svoXxo2zThffKV\\\\\\\/t+IrVvO33LB7CM38btcfIT2tCnQEU68Z\\\\\\\/TDYe5oU7iXoNeTHoP\\\\\\\/CtaJPnmk6DoYw1dTu3oItPwnvdkzbOxPnkSE463qfUM42qflFNlwy3kH0kpqDky6wYC9DnYwBGdL+dxX3zGTMaHmvtjPLChJDNaMH2QMeMxl5ziNTGvrfHKVIahvdcEieArKuudjW8pw9mmedhm0eFS\\\\\\\/Ub1xWvcS46f9Ok5pbaRKGix15JMuL6K75rLgs8JaPDpwT3zaS75nTWwdNEJpA8rc\\\\\\\/rpJmnq4TqAsKvBFWo7at0BjbqsmZFhzQM\\\\\\\/hK6YrYEZxPNnDReWz6MmqKXYrN3HYQvigUpraeudbm3uZMezvl9Xi7AEuvzMs\\\\\\\/uxY05B4\\\\\\\/Ds2HSuYnBgLc\\\\\\\/imLiJZ96iRtLArpkIDQb2bUUasFuz0cJaQOfYLr9fsYQyFiewYKC6gi16RBI06LamDB26GF8qZa1nfzzeApRLKCUuHlJpzXjrcOSm6uQBQrFtc1UvIeZE5x\\\\\\\/a3Lx0gGvk9t7w4NIDCx\\\\\\\/QPG2GLBv1nF8JdQNfI3wiykZ7reikyCU9U2nXmSOL105y0i6\\\\\\\/CWI5lOG8MN1zvxTNd+l7fzgV4paN1b7+96Xea4N4XLAFzligyniVnust3\\\\\\\/G\\\\\\\/LrdUK8D+GgzO1KAeoitjnYId5gC5RXqBvNbkzpQ0SAKJ6Jnx7w7CEp3CAnXht\\\\\\\/rHTurrj9n63M38Y81JYyuzNpIu0hqrA+3sNF7RmLFjz+Lrg0dlYDo2WbOAuJ+dQmCrcDRgR99naB\\\\\\\/aXWLBcz2yKzZqmBZK24vw5gapqR47TxzWpWM5H6+Cms4V8nvoxiVhh42dbhGymd6bJ7FZnwzDmEnbzynv1ZFMGyLH5JI4G1mKja383LyZBpLMVoCI9UAwuAVLpeADz818pqT6zlZ4CY7OQqqjjW0\\\\\\\/bgHDGMosRW\\\\\\\/EEB6d0Oj+dZkDGzvNiO0JLii6zHQLf5h4Ix7nApIYanvhgqxbfQCpDjN0c9LnEMGsyi7nUlXeNAQk0cb7oIIQJFfINRx4HLa4bgmQjLx0fltJH78uesh6Qav44hSxPJEkkaAN\\\\\\\/si6gN+43y6DJQn68f1MrH1FrCZybGLstwDxMJdLmmtWmcCxXgPMo9ZpQnQFBIozrRfTt8HosOpKnDaiBclpvgM1s3BbzimPaIgxh2nTY0emrbnOWzDlB1f555PpffYfoQPqBQ71X7nKMm7dfIAHgukQUsFs91s822qiZPPSmDlHqDT2ExcIynabGrPxspFfuJaBcUJKQg6mIJNKIVdvPwsPF6S06UWnHG03BiiCv1ZOS746L+i7DLzwQqPL56pYPZPwgeNNxcJDR7CwS\\\\\\\/x4SDLT7Spz7gUZ6eQPc6vjk7pfMIG6HaLaqy6qY+mrrMCK4QC8e3HewG9AI1Quhy\\\\\\\/I0f430zuSUubHNsAHSUDJjB7j0YBEEujlpYnePOHt4U860BqYi2TbldCB1Db9IDvVu6TPEejGsrOvt5ocmm4ZGEBmE8Eh8tYBxXmQRaImnO3F+MXx8pWmWzMppb2ruJzk+uclrFgWFtHZfGRWQb3sUmhBdPJ\\\\\\\/enip7YLQ8f0Eyf0kAPAHTqNbFVXW9WOW6wiHlVnfc0sgpRhwmaq3ySydh9NImdQSk7EeP\\\\\\\/xXB1Dzub9rW2rEXY5HUKHySpe6twxO6kkNpo57R\\\\\\\/AIOAH8tj4loJcHbPQcmPSUGKLgnxMdQ+vZ846edh4bHhQbqny8ZWQh7b\\\\\\\/CaY9l\\\\\\\/7xTry\\\\\\\/jRfMJuxm3DgnBoZ010tFmJpFlAOIENtqRGcvkbstHMm9giIUUK4d4rj\\\\\\\/5Pk8RXvqiPAlP7ceSA\\\\\\\/X5slH\\\\\\\/HNZ9RaOTfEVoWgK6H\\\\\\\/rxSMXcMSrh0kCWIEXBGPImBLef3Xl4hB5Lgq1tz\\\\\\\/zoUtydeLuu9h88lokHpx5z3dAYvmtVd5nOi33OwMdP0+vTvHyyI0MCaeVoVy8XzUCDxyllYyQupm+61ILJIp8gAdNhZJ2rFQSSlU0LD7XAuQzKVCHPlyHfqIFdtNj1bl2xv\\\\\\\/Wbx4JN9sWB70qcR8fn8tVRM3h0agBOp+wmgkgHtC+Orc3TZMltiUaal72a2Ib8z8fKtC2UR\\\\\\\/PuEcRXnn7tUc9mM4CtBwrvsAYkvTelZclB7avCm0It9NFPGfDo3B\\\\\\\/D7ocy+px2\\\\\\\/YHaCXJFgNKdvTLSZPpPeE\\\\\\\/Ifc+CBFM3OhS6QsZf3ve6vAxxTIR18Awv36U3jfJP\\\\\\\/vLLrR+4tD+jZYwM3UUxsryKXRaQ20sgid9q6XBeKt0Kl\\\\\\\/CVXZ8J\\\\\\\/Y4FJvR7PnhJmSkeV94V50qbJM6vUG\\\\\\\/ZUi4jrXLejZwcomWncBaNIWHwxJc\\\\\\\/B+B4L1rv9dWq1y3lcLCzqQ2woYvftUdQpOyz\\\\\\\/YkarPZzlTXIpraTag92W1nqHkniUAl0ioTtTKO0wGFKYqGqBVLp14IdJJshQbM26FrwZoZMpPYVM1wYk1QghHdVWoGEZi+nwTKTneEETr+0ulDgM92bRAxliaDHFp53pbfWN0Pz7OVZynAF1zBwUKe9fKwNuWCJcvlBByqM16GBz6AvbCr9qZhq\\\\\\\/ED8K1NxdBa1Sor2Fyaw2PCpdGYc93nJrwh80UHCfyYVbCWmznGbtHhHJaTWa3VV2PSXj9iENLakY2gxfmPd1jYGArsQmlxavJe6DvF6hAAJeyh51eaj3QSJ\\\\\\\/e3arPSbwOS4rQkJekx7sX2XmxKR2cSlyV7NS+0q+jg4IspO\\\\\\\/dVTQTMYxDabfEi\\\\\\\/YW5jvrii\\\\\\\/nOyc67kRreDv0LElqPQZ6mqx\\\\\\\/\\\\\\\/BT1JZLlLrE0+c1AlTqzZUBLtJSMMqTaWpNrZqnaxJ\\\\\\\/7eqRI90zgqchwHuOhjUATbjk5hBuO5YpiGtqKDxFk84VBaw8MuqGs3Mx7gCu4EXzkYPZDKehRcp4QE9IIzLuT8w0p33t+QPjxgb7CzgjsnC3UQKT+\\\\\\\/HQtgRsTS4MM7wOmve0zC5ZJUFXDQsNr6ox9izOc9jID1PggWTdyXl\\\\\\\/yWxbT2Pc1N0S8xHwobVWYrgmnQkdREkKXdgKwbRWkMEIQA6oIPxJwbtN3HRcOxtW153YDoHB2zoyctBWEqesVfNPYQnMCbYnrPwLVY1hCV5ykx8qnnKFmNvfAsbIxLSuq2d7B2rqhGEPBdhdH62bdR1wjADA1WVumvb4e+qOi8fD5SFWy6OqTsXLFeZFWdQyVijYEFjsMS\\\\\\\/CjFCgDEzY3GyPVfSfl1RY7LkxM2yMm6c5ajYS08HM5v4Z9mDozeGGhGoYs1nzfpYo5YGCvZYFAlz3S8DEnUX+C1BfH+xO6WScb+Krj4pylUz\\\\\\\/WJZnGOJcaj0ETndHCPIfO0C9AAaxRxEw7OMTbRJ353OADKK1av0WoCJ72m0LnYARrHdlwfHysxXFmobRvLDZ1WlZTJ6a1pHPkI5djqZ6IhIYRMVO+ObxiwctjES7NkgIBBiIh\\\\\\\/BhYfMWCsb9WsLhohksJ0+3BYUMe4rZW0NMPkcpDnOM6syO0gKjMHorPWFhObRd5iYrW4VYUcACBziJLIRRASN64ckNVL1fVJ+LWAe+eAz3jLXEbcivdpcwS4PfG8Xf3EJ1W9Ol+zl9HvGkwSk5\\\\\\\/\\\\\\\/OtZxlnXUT8Q5V1YSJMiaOWLM4IYvTpRZYCE\\\\\\\/EzBO+FQ0PKP4nkL\\\\\\\/+XA8r7K6+Tr1k3eJMMuEcIPQOrMzZryGZYIFt2rhYd03lAnPGjpCU1hojQqzztXspWxhuLiL\\\\\\\/RSMb\\\\\\\/XWsdwANql7HkntPqlvXaRP+CtHPSIe1PIAWa0ea+Y4TSNwujM0I15B7R32QuWca0dwaq8zbDVhG6cNlkUW2y9OD2ZoenoWCATnkItNYsfyNTSeAohAAmKk4AT7\\\\\\\/b+rp7KpCFsEjG+mPeNHcOMdUrYk1+2iwX1Qnt7E9Js0MkO7pgZIOYORrBt8PqilBQayPRtan7VZeiPz8K+IuarPOTpBLCdYtX+o398QgaO3FbvXFQSfXKXO9o2LA\\\\\\\/xZMQXdx9EJRzOXB3RXFHCqQcxyCLLgrC+Q5jttfkDAglHLzBq62GhnFzyxMTALDwf6Ltp979vqmFDrqFKMxncB4znnACICuBIh4jeuWDFI9LdXe8AkAbE8cbN9V0BIwdUKHsXY3A2o7Gzs8irY7BZjrT5ZjMQgIf+v1wsRagoo8osOTKH0a+B8omCfibu7RsDUHE+0\\\\\\\/XSJabd7K677T4B1+rYwFX3eVMG77eCokevsCzr\\\\\\\/r4N6KJ5is6esE1gwJcDcJYuj6CYhrzqZzZzOxdN4iSQl0QZTm5NfObJNHgk0TQP29G70Mhf1vYJ18+sDRm6ZsNemJMCH0XXLyN9fTmJxGmzTD8SA5SbmR36qaUi+KkRrMDDKHs9tM1mXg9+6iaDb24pk3qbTVn6jiGKhlWziynsIlYn2rPKh7NCwABZi1Fqb3vPKljQibLtkTn2NWRHofMDBxjeyUkX1C6W7N2p195tx0MgL9SAsf62lmXKEg32bh2Zv2rTp7RkCCGPD1oqJInf\\\\\\\/bfsRhAZLGlITRuxo8MlelFj1EsQ+Pil3aqLSXJPbXs3aFl+qipn\\\\\\\/7CoEUesfsL3w4+s4\\\\\\\/UDVPRqFnC3UQLgeCHX7X2yrh4RD3lC106ayaExGreHiBF6IbC2lfhz4rYPfZNZdC99YWkkEFHGfmvqDHf\\\\\\\/wZVHs52hWyP\\\\\\\/q7S5jkJEquu0AQQaTyqIShhAfT0k0V65pWnEMp0xCE+H0QlM4u+lMHglvQpnSipR2d+GkCR+3u8oIvFidPhue+4DDUz2tvQNOsiyg36Rgg7eYGOokA2BG7YM3EPnAoxEUzIysA4nTAWgHkQVbM4aLXkpExfjLLqyF4Lcop4hOlAW8j+ebvCX+xnmTB2UJg70UuKmDA1LHXsauVnryvrJn1pQXkVdRZ2K7CE3iMkf3+ZeXC5cqJO17A4+YIoJihdF6Pe5p9AsQEJAVqacCoiMVuRPConLj8gTz6wu6kwE8GzqrzbMaKGUnL2L6Sgyg0g43CkBUUvBb\\\\\\\/uwpd0hd9sXqRxdOrnOk2N3lDnhXdhj4WDCFtX+D1zIaNJ0+lmMjT6sVJ8g7+StOQPVCNLfkniMYccfHFVJmCe+PclLBSmkbksxeUJb0JfNBUCSpbVRQPieZNSZBYfZ0SuzfYjZTwGovsuB0QReX+qrYKuwr1H8Wa7LV37MAAjj4C0yPy2ScYsowPCnC9sLtbwlBhmG0nPCLNorwOTcHivDTtKfwnJOIN6s9o+2A3j7HG76sQx3LP8r000Mo1cpL3XsdeHDEzQMBy9iSCzX2f3NxJ+jd5SYJLPy9XQYtE+33QN7jXVVTKDNoSB+gogPM3ScG+YDefl3Vp+O1E\\\\\\\/elmCQfPGM31TrLXfDphUFwfTVAuhsvSfD70FqBXXcnIKEeiTToWhOtqK46nhYBwyeAnTWNrEkR12rIFEUqyVU7lRDTF9Qy1HDjz5cyMvD6Hb\\\\\\\/0vEMLvicqVKAwDM0kqu5EeZtT\\\\\\\/ZDtX4hh8LoDOWLs41cHcDGxWhS2iDK5EVFJu73vHPMS1KTXcF2pdlAR9cVZH\\\\\\\/VqqSxM9sxIPsZcWHiOV067CZYHjwq+KT85VtN5EkdSXhPnu8FEwutX3yLMAS\\\\\\\/BXCR5322VG9IiPou2tpvvyhQUelVa5076V7kvaQMB9I3vss2iSIgl\\\\\\\/fiRwg1K2dSlleimhhF9vcFsOdv7WO1iKExPEJqQJWi6xSOCk3OBAsXRe6+4Mgce1SiB2eaY+RgehagGpJdZt0g4\\\\\\\/aAVRpG5qJtXfMQAFjpkYTEH4jP2U5rei6lw+mnEFxA+djDd40sv8M4emM+svuNxQRPFgJMG5pogR18K2keYmIfg9TFZ4Vja2RI78RXeM9OaEz8lr3TohjgwFH3tnAFDctj\\\\\\\/tYqenoxWgDNMtA+MNPINakmNdXSoUrONO8V4+tq18ul63kis+8jqOLkb\\\\\\\/oZhckSeiKGIdsfmj8DMEWKNj218SwRLBL0ZZjCbkcFVmgeq3psdMQ5BaKK0vKHLmBf4LXDHaLj6ayb96nIaHsUzof4wFC1s37ToDb+6kgmFc2ZPTKNhAtFBbNg+VJ4nkg\\\\\\\/wysn62WRmljdm0kzF7\\\\\\\/PrzmyZjx\\\\\\\/HqxIZc042r3znLU8uYvaCTUa0VKrv6sVMrxIM8Js9jBqmzuhAM2gmlt6z2R3o\\\\\\\/g+iupXCJWaPIkBsiloeIO09a1osat7FHi4D05kS4OWqU2VcKu53kI3oh+1RBs0IxhFzMeEadRD6ubmW\\\\\\\/5gKa4rN0sDXOG9nwvES2dXmmN\\\\\\\/Ufn85x7sNLi8js8h6nkBgxM6xmBcF7IrVwtO1Bwg3eVAO0Q0WQ6Sv0cWeXwvkKz0HIg780NyX8Y1UxCUED47CLpnJayj7EAj5H8+QpX83juR6utEh6SVWhEHjtdJLexqCwWzyX0xYWDBEI1wLAG5KmJkJKqy7FlBuoeXZnnurC60Uqj10lea4zoEbdLy2th8ON3UVmiCPTQFeG5cU+OWoobMZTYkuaAInW8u6KYiPJdEtXsxcLpi0ERtqhOY1tMFyQWAzVJ0vUHLn6CSZoWKqk3vD08rZVm6dDazPsSiwCyh2LH251C\\\\\\\/9gzgi3wOZ3yG3d3ILjZISMm7IKkzVNmgzIkuRlrMdTcmlihYrMKW6O9v+6W9KhdO3C1SOemC0zJ4SLWcG47tVF5mvmFg42rfR5YH95vtKpTpi\\\\\\\/56zqUzpvu0owo\\\\\\\/7Wk\\\\\\\/eJAY13hY996ykc0ru8u2Ey2iGiReLGs\\\\\\\/SsMLEA8LYfH9tZhh5kIcw3wTOD7Bp2Dt9xcOdUNAY9HYJHvYzoG6LOuhr94dlaefQmqK0Fsm6VRhklFpimXmZUce2QuY8IjervLupCkGzaG0KazdNjFYwpShH4J1H4XQCAhaVthjyB7x5HB+eyhREBQHrgmcU6fj1xlFEUp\\\\\\\/kFfc0XDXE62pQq3IGeFWrrDoqrM0zX9F5kiFadBO1yFUxRLGmYDFj+G63B+2iD18oINdNA9UH1wdjL1DUVAEeVVkVqpnZFaNmkg7LBeebeA7J2ymZLJYJ\\\\\\\/InP1PWxYTQxwtRA7K2VGL+a+Xpy+EzdYrO0WMj5wDbmvjkCTqdw89hwH+aPuopwRx8do+YbQUJmAWuFDNOZOC+ecXk9HlOgnPpSjzx3oFExl5HwMfhp6wGPSCKZ+P9L\\\\\\\/BW6rkv2JtdpLSJjac0ykeUAtRZWCsIUVcLSFvyAox9wvhgJ7cI6k1Y3+zcHye9tOlcqTn6WRqIyd3Qf3gGhcktQfXwu0+pCim5b6VLNb7\\\\\\\/OK71ePS4VPvK\\\\\\\/U3WqnFAzvfu6teXFqy7re3DZIiXl8GzD7J5uGrf7visKY7iXgDin7z208NmfRvp8DjuqKMXne+EGgZuyCxmqiWZvX69JRaHT1XDks0vgZt81RNun1mzqqIMHPdGYmaQ+kzYUHxiXQKwU9KnMHX2neOEjY4MhT0pcHBsTiVmipF93SwiVGuumxHJUKRA6lXczS9ZQmEj5l9n0MTp5nJJW3gp9vVnJVmSSkN6rXQEIyUMZFEGNkGLM+2S8MepRUEK\\\\\\\/Ofs1g8Nm1H0fXGQIkUpBmNrHVVOuvxKHAk5DS52AFsBvqP2OGemZZJfMeRKNY5Kuc\\\\\\\/tFHVfCwOfxIrmuYPd5ztFqx+cnN0P12a5gJAI2eR5QBkd4pK5lL9XawAgPTVZkT50yooZzRc6fRp7m0zytapmrc99blNyy7ldDZKQ\\\\\\\/6dfc3VQcEnVCSnee4KXjWnlsGiyP0lYLOJCGC7gBzth9aOzK\\\\\\\/nUUixshMwX4lr7OIZ\\\\\\\/TS\\\\\\\/eT865ogemd2mBzl1VmV8sEo1GrpU5svQth+UjRZx68GzDU3jsce8SHEUL95qYSnCeYVj6afFHazLRbtuKfywSfp927Cg2UTipwqR7PK7lfTpXYIZZoJj6sUjUE8L5QURJwjXYZwdD+iQO+0cZtemX+KkNF2tejTZP81IWjGkvzPufBXhV295aux8S6cvYhshNkQezdqtLpcXThwZ763F2DxFaSoIofaQpgYolsRi\\\\\\\/saw+aiWqH6xw8slwDJlbY3tn7LY5Il+G27up6\\\\\\\/s3IOSnJi3tbp02o\\\\\\\/QI4NMyX5QgkkXpaBqu1AzEoPGK7jDa1pm6\\\\\\\/OOL8dzsUwovurOwjrmYQ5fPTkXEFuwVv9UG5WRmogGMGNPUO\\\\\\\/PF9RqzqwPMA73ByuQAoJ4hH2qqG1u\\\\\\\/h+VNgtvS+XLy9f0bs3XSAJwCkqjp8zjI\\\\\\\/qxf1PLHVSGxEMk4fym\\\\\\\/o6Csa8qNMRX4f3qIx7zPmdgt+gj9fqieSloTax5irzuneAupA6arfp7e9KVbgyRKnP30bACbIhIY1I3dxXbyjt04Gfxg6eLdyUc5h07Ykzc2nBSyNVw07D5X9COhm38fe07zcT1NwT7kqA0ECapx98t8COFTHT4Y3s6CGQaDbPuOM1wD36dnv8RsC+G+8mbwRDgYWqNo\\\\\\\/hT2neI\\\\\\\/ultFvFeWnhxbxDL8UlaNJLIH3eO+2M37TVqzBvZh66Y8TLhPnxSP\\\\\\\/Ko\\\\\\\/+zJogDJTCfkcll2r34eVjoW0EA\\\\\\\/mXuWae6fdSp7k5Dxgy+E1X5kLe3JokIMDU4uIkh2DgSU83AZFBJurmDMlBF6DKYOZq3D9o4S0gpg1Tv\\\\\\\/nS491D7o6nMyEz5AkNg1orwXHr+HEhzdQm8rpFKeB9gTrHQSInXxXscTYKRbirS2XWA4eSoxZoMK2BhX8S8TJe52kpYLp6ZfkHJ9Q8W7Nf4CHHR4DJWjRY9Y0fyvqBbxlNOco0UiAIWGqAhCADogf2mVaY74ccaBJRpFUs99uvk5XTtbB2+EpMzFpwdZwL+r7POs1\\\\\\\/gNX8i2npLFEdXeZ+vnr6vJss9hy7XqyqLmJSUTWG5NFYm9jFicoMUi2gJV\\\\\\\/a24gav5PB0EprQcpsa4eA0BT5QhidJqWTvvPdKNYGo1x71oYLuyIZ4DSlkMIFNV3qhppkgCBZqvdNtGEI0Y88jKKbDcAva+8iNz+C995toFR0grxdK\\\\\\\/dI0O5aDSicAYdTNMkjXzQYFBh38OERUV4vo4L5gf5jZMdDZ5sw050EGYox5V9LmdNnKLWOnNLpZ\\\\\\\/6eO+XLd0QU7vp\\\\\\\/U4dJX0r\\\\\\\/QOy7woHZaUqKPIX1GEp8YMk9keZGF8U0BxOWD4MWzZf+saTzPSPwMOgAXuBBnIb6e04ysn\\\\\\\/inmX+oeK74F7cr6BsR+8Obun2RR\\\\\\\/eJiZQ3df\\\\\\\/iceXEj8XV+p5FFCu6SSSql\\\\\\\/RpTL3jaWVn7PbiFLKfiPWnXOmKi4XHjcoErxPphbernrKTJQO6QU6\\\\\\\/QaXsx9GNgzWKJCxs+MRTDCCfsWpbwnx93pk6W38qBuzEj9cuM6qxDrZQvbiPEkY9lzLevR0NXAfOIjnSNLT2kjogxUE7WgSBtXoVn\\\\\\\/rwJ35RxzVTdrWebCOOgRkC79j60gXxD4y+19hzAR\\\\\\\/Wh6Zuzd8paN6sJC5gt2YWlF8WIdQUX5dRZZP2USthmvUb9g4z2NMp9R9n2NfMkc6vw5DP\\\\\\\/bXKkXfUoIos6JigW9i7fLHkKBwYtnuzeShD1LCvsnkbHlDsRkhOvRvyQk+vmuEjQ5ZeyaQiVu0ezL0s4ZB3VKgM62TQhi9qi\\\\\\\/mjPI2heJ7sYrrFPqc4NBrOwYHqLAXPsAHXzEwovgrjRX\\\\\\\/0OqK9H6oXJiJSxBSy\\\\\\\/Vi\\\\\\\/7B\\\\\\\/98i5FqMhp4oYZDs3KLzbvv7EyGO8zC6vhGXgeOBxANyl+XyM2mfdTI3iaXkO78o5CDVAlHK1y6tKb+XMXnxpRPrtks4BKXGCaKcpoqCcrRcOV7O9Skqixou5OzJ9WrQOpWzykqHpR8HRmxxfPBUcZgwCUuPdykVoJMC2QW29TvvxZb+XXdpIFDDoMulXEMgWHrTkZqtBTNYMfq7gpA556Y55sBoFUDQobgaps8kIzuc6uO9SZXfX5BUmIW9DLVPC\\\\\\\/HCQDqryN8HUHX3+PNOCfGoIfMKIBST+kMH2TfO1jgSYCQKYfCN6O8Qfl6QVNr5NPhQx5ndRwZGGSj3YZsMByxKuIp5341q4d4QLSxdKAm1UI7IxqwFh70RDMG1Z46A\\\\\\\/VgYcpQs2FgZiiAoL+cqO6i\\\\\\\/DQjW\\\\\\\/vJcc0ozmarNELzb\\\\\\\/gGyWdQ\\\\\\\/Q9j0zPl1oBmH4i\\\\\\\/Dy7tCcIE7CcjiWZtG\\\\\\\/9Vg3R0Eq2wOWsPFevR3xBP4PuRwze2NbcvmASBanG2hdDHM04wQGzl8HQeimfx\\\\\\\/uItOGJxANjpqvkMIKAPxf6bt9K5csllpPh+CBO7pbjn6GNd\\\\\\\/pOfaVOKsDaC4ObGDRCWn+m02OS\\\\\\\/E\\\\\\\/fQ5so0s17KYs\\\\\\\/P5Q+gcdbIjNXVKwuEvDMP5V64MW7UbKWCcyCfaZ30rlV3piIXaGu2dCrhRwIyp2Mmyn17dAhAKZVNbIpnItViJOyQ+bGGlWl22mOT7Hs+z32n00+EUzyRRRdqjqWDa\\\\\\\/iMytF+cy37NFE7s3BHgvFAG6dBx6c4xSh5yxHgHc6jDUYNk9rqAGven+61KidCEQ\\\\\\\/tVW5\\\\\\\/Q4epHfLP1oR3sVj2z7TQ9KZvp0T7HS32GfAlP7N7o2nVrj+HOx63jxmvoa2xusaVrt99rWm0mONMvdAV7a2R\\\\\\\/fmx43OCqJpnKP0dq79djvtJbgZa+SZXncvh4gpc2iNsbnE6WT4xYZy9AYgfHjQh1E83S5E\\\\\\\/Qox3GOiZiAg9ocZ4IGwnHw+NLDA8JAT\\\\\\\/ly4K1VIFRCeM3i4uSREaZySDSXZDM5AVFu96XhnrfNk0+WR5sRg3AJWnHTVLQRbK+j0mcH3PhG9QZyQ8bnTsqPrdfPHFiKgUKuZUc7Xda9ZZrJg5GbmqO4WMHc840ojzI0QqwKuNhdsE722Ld\\\\\\\/d8WXLOMeoa8zkI51SxiAd8sJmcedFFftXpo8dWCiImcVtsDlBtko2ZHNWXQG0xODasFFiiy7Rw8RXxFoIJOMYdboGqxwIB\\\\\\\/5YHdKbusETFqWSG31dHIaPc3kR0errDRHBaTDD5ATqdJwPSkxE5Gezap20sAOmZeps+Gea0wxDvJ102eUEucJwV2SxyIMGr8RVrDILG5KyC1PmVmmACVu8vGO7nx+wrdIykcElHOglYMxkdzT6bYTt1epWDC1SVT5veo1M7pTaBcs6a94I9wYyptau5L8c+M1RW+9VPjAOaRh7GlkskD31QpCr1+HJrOIJHzgQUrmgU+\\\\\\\/qiaGVCVJLKVzSRgx+Tj88IqDdYoSJqkOmdKBH1\\\\\\\/v0dg6cLtt3X67Vz4L2tfl0\\\\\\\/8RqSxb0ZgppTjLFubyjSIUaANJ\\\\\\\/OKIf71NDKgXDfcpwMzoZFX81ahuS\\\\\\\/ujtFef0x1uSBJQgWAu2Pn1rfIX40HpVlVgYeEuw1g6MtMc5i7KHksXogfFxiVKPNCELkyCK0uQ8Y+pnbYqHvQkgW01oo9HPeQV25MkoyG1EeCqnq8gWFuWXc1jqZvkEqLhT0QkdaJ55oKgUE8bCnoHOra1cHriEImGBslDLQdbWkcGg5Ebw17Ha+yv6xjbc4St048SjOnGx65oG15BETk4WkX538OsGMTKIwEO9RMwkCNfzytVUJ6kG+es47yFIwmnLYqHIQDzdev2hmSyDep9mAvYKPwhKFsvd9yq9\\\\\\\/BhnAg29G0dXcKUzkXmQiAKYN4Z16taTyfKBHrXDks7yqb7uRYY56V9i\\\\\\\/GpGhpi22eL4alWSxx2z4hT25nRfATQHRK+zvw4sYpay42WEWjezl2tFdazbqriXA3KMhM8OcTiEddJqZc6TFkpe5E63SSw2BENrJ\\\\\\\/Lcm1AXmMQRONLo1Oh2OnTMR5BmHmcd5t5MZguQXMSAP51dLf4jiInq3f+g56EPvTQSw2AdrZeqfe5LR4BSSs3fLcZbDxMYuWOgpZmCO\\\\\\\/YPfAdHmv3uYTTOskMUyhNDfQg543W2JIOy55Qa07O\\\\\\\/ugMKU7odS\\\\\\\/kjLiOgAGbsmBHtPGIbXu9g4P4\\\\\\\/fq6P4jPTHhxH9tM+UYZ8DoHUObAUT6Q8tbkCVr\\\\\\\/oNw0TPTN3FVBt0jIokPlD6L3S+DW2sN4UcGz2GnQq8qFMEzYMo5zHfM\\\\\\\/Tu4OmTebg287dOu+VX4u50tMGAKg9jCEBdNB3X6l8f4ktC8oBq8Od2vYU\\\\\\\/YgE7Ayh9\\\\\\\/649rmr4v908Q5THVxFjpWcQHyqZwl\\\\\\\/B6GvjoV23e6n2GErIP76MrrQUjkPqi79Ke0+i54+BKKOUC27fEERCBhXui47R5vF6SYWm3Ps4Hurv1C8Vq6Mj\\\\\\\/EC1OnYmrPRScRen89r8QntAiGg6Mz4isK55\\\\\\\/ThDZKHbxtVBgEndI4O2mGQWB5kSbTuR0verpvmVPuGHa9Ut4t9xJmWDNyiKI2MV1scw7a3R0gSyTibpxANgBu1HYT6S5k9ik5BzahviTryi5AJb+HiVgeyC1In6UrPjJlQ04p4JXswJ1PC03hw8chCCB\\\\\\\/P8xinhT\\\\\\\/MC2VOmO49TzLJR6gWgHp9nj\\\\\\\/CKb9Ke+i0Glw\\\\\\\/Dq2IIz+mu6n1YH0Q6lWt28ku4pEhkDtQ0d2uZp4QGiE8Kib5QtBNvyLWBZJGJd\\\\\\\/XxPBbucr8zNwu1deG8wnYeEU3eJLYGW3Spny1paWuvm1mSt8AETWgGcy0GIWBDeTDDY2aTkDoQgNcZwe9srMvBBuNHC+miCrCzTuMOxu5Eb\\\\\\\/K4an98gVu1prLBDMiDzVLvt2BV43xkabEP+KK3nG+8bCtJETBYall6nGtEsSL2H7eHfeWRlin2jtfeQyGTzCTx9j8FSGs6Ha1cZv6oTmkYtUt\\\\\\\/nGY+DNTeU2iDIiNI+Mv10pQw9ZJJ+KTmvQmfdtVmGELCdgsNd2vm7ebPMxZU0BurhV\\\\\\\/hfi00K+Ijjgldkt25poIFSg7MDOLmkoVQxGITnR+PyU+zKD+ARO0lUxbY4PKomsdhEhsn6049X6Kit3o6tD22YwcTqylGWz6AnDoTDw\\\\\\\/40VZYXVoJAaBm2jVadxihG0jlGXPCLRYnbwet1mq6YEnc5n\\\\\\\/OAjqOCymtdRoKU09mOCfXmvO5C76c6ts+J9AYCahYra\\\\\\\/5uB\\\\\\\/m2UIUuMVsNIk0gySJrhnCM5+WIdmUF\\\\\\\/KJoZ3O4xkvOt4Bwxc4EOE+sF7qmTdyGR6t5mfomfU5Ux4WxTVRSw0Y+xEqVHBGV+eE0OTmKIjDeMk3bo\\\\\\\/yWXjmpiTqyQDMdqRl8VoJWKVoLqdDfPxAmFThMujuz9HLuodDjvpkS2F0gjzYHSsFqru314YUqsiZIl7CkZlj2+fRIxLKaDptZcsrjReqlT6l8UnOdRh+1jO3PC27d8CmthlNU9j8GjiNqwX1Rj1bVeOfJylEEdjiHcr61vCrwhwRss6Q4143xzZm82J4Fs6d\\\\\\\/qwsVOGI8a7ngtS99\\\\\\\/R8gJg+R54M7wxeIOw4KEczigj\\\\\\\/0mJOtKj+6nu2xUQlTYfj2I0UknVHZbRrrdGsNYP5\\\\\\\/6z\\\\\\\/IMnX1qsehDiT4BkPf8x8B8WvrEFTQ+8VObKhE7z02Gv6Psjnp+S13\\\\\\\/+OCNcHVqgtC3lfs3jcQJR8b3xz1ee41GI+RCHU2LYcztrhJ\\\\\\\/rSPUvZsD7JHMUE0I6VyvEjuxWhCyIqxxF3dm0fIkFpir0DNk+5RTnLraq3EONDywlP7yEaWvULcxMvstYbAOKKESUFuVTlt9a8lVwJ2V\\\\\\\/MHhGGev6B5SmEqRMKw8DisLFQWeaRsmbtXOCpVFKkdkcMRdvQZtq+KhXH7WNhunQmFpvjruPkyjM\\\\\\\/FGsNHflS3ItEvjsw5igdKOy1nj+aZcn2RQrdz5oUa7rEEtsbNqmuYXT0J0I0IAqy9utvkqCKD5WgOrzWdL44Jfcpn9J8sLtWjvIAL4BVeHmDg+7z0E1VnR79YTkoeql4sXXw\\\\\\\/DU8+Y5gnxcBrHFr0q2uBuiC3D4YzMFL\\\\\\\/wmqwOXkGUYiguBTrB7n6iOSXg1xHOUXeifoGc2CHsIOfrpr5hytgT16Tfyit7jTLnWcWii1JVqhL9fYV\\\\\\\/Xf0AbYbMRlZhZviod\\\\\\\/l6EENLoGs02DkIyI8h\\\\\\\/VpyiAfEvVjC4Z3WN2AZ2aoPtjwQ\\\\\\\/ZMH9TLbHOMzpX+4r7zWf\\\\\\\/sLbwvEVSyZW+cOtK\\\\\\\/ryDzcpo0Fj6YKeWhXhlaRA7+skr736NqjcFJXvP0td43BXD9ACb1\\\\\\\/wZUJMEV4ayzpZzIYrW2LDmX1ET3kf3J8lB\\\\\\\/Egar1JNYKdRECg5JKQW2zFfodWvoDHEacFArkZ46vLIwsiAfC1UUm6GOfxYOWlGlUyUUXUnMC3Re\\\\\\\/eXwDabr0a\\\\\\\/3MxEz8T7ZVVXEVAhQyBvFocpGy63W2nrscQ\\\\\\\/cEzE51OH\\\\\\\/Fvk6m0TjO9G\\\\\\\/BgV\\\\\\\/yNmtBlJx9KkenIfSettNj9BkYJx+YdlrM9tsCUZCxvrbI9qZ+zob+9Fvy65XCsyBD3M+JwNSF5RaLUyv4f0XY0e16gwi2cypPpv1W9NzoyWXXri2a544ovGtdoSf9kq0j6bs7Z1CuZYkM0N8j486f\\\\\\\/2hZvMGzCeDbSzoOfskRTZyhxtXPnO\\\\\\\/d1xdIkuiRHuBfEQlR7R5qd6w2mfmkTuD5+Gc2xJnnAnDoJqS4\\\\\\\/7vG+z66tmN5jVSPW39SBcu6vR+oLrhRhR9HF6iQFZcMMIEL73lUy6Db9qemx3ShogAkYr6TOQxQT5yeOo+wGDQoW+33zTI7nwJ2gVfmR521lbSfOK+mvzg4W8BWgC93K4eIut10lkYN3F60NboPEL+knZnirhHDP6dXTGXV4NiDfDaKXjMJe58l\\\\\\\/PKxwG9iGIjHUoy\\\\\\\/r3yOrL81PvyFiYhlSzz9x\\\\\\\/ClIMSdomKmjKi+AbETU2WYMEqR8HszOBXXLCXMUfbDeqHjTpxcDafEjQwzwG7CuiujCxLPHyx+JXO+zhS9fMPqLCrGXF6TIT7AJCFtQ8dFnm\\\\\\\/bEOTwgE\\\\\\\/VzLYnyYDNp50Zuy3hE4ytq2OmN6wwoLY0ol78xYNgoG1fG8WhcHVnAmqu\\\\\\\/Lv+EXP8qo6knRlUTJpPePuKmbBVvOjCivstHSELdBlgI9ojxoywYuKVxlNmxfHhsFjrEoSIoP63G+H6eIEYTtVqOM7weldPglLRKwlaNXsCNHti9j91JpuN1tas0Iw6sV94VqU\\\\\\\/VGUf\\\\\\\/qxHwDUN1MYqy0S1\\\\\\\/qTS4Ag+LtyU0R1DCWFrEIko3F646CVIf17XdDedo8luh2q6Bar5qwhFBsOtjjlq98Y8YdZMS+3N4hzmQ46VMhT+NaWw3MdT58+BohhiEw6lsCVAQ7wd5pUZV67vVyXPCyMy9g1JFxA6qhz0DG7xbBEXmZ08h3WN0LXC3rJip58aF+EZW72hDLHAAQ60LZre56QdPD24fIyCk8fwMJTS5cFQ6fa\\\\\\\/JDo3MrKMvhGvZBmnqv\\\\\\\/2yGrCaVOiTtQZCIiyHh\\\\\\\/i3F3HN5+XFRdeFVO9DjH0NWhSd+BdTswNtPg9IZaLHU6bRTsMhHszbSu\\\\\\\/CbfH0ekSlsiek2jvYNq4Ty5zBMJaYdfYJcvtAMlS9uAVGxIgllkEkz6dPOZvlGa29mtA1T+vIOoFRrjr\\\\\\\/569yWwu+C5+BI57Ir412KCHejFNKt6o7J0TlrtATRGUgrqP8i0Rk1goh+q5gfgPq7ksVRAuu3KTCdBUjZRJ\\\\\\\/qfSB6TDZhAnZzsdRfcGeW\\\\\\\/587bCQ7BYgaV39QHc02So25kDNfDmUT3trVfbJGEL+JRVFN0BQlBrA3Okc9BUMWJoPPUn+9Dthigu6p6yGiairmsTO7adtVHzecdUGGjV0hIzJarvGux617\\\\\\\/4zzujFmAjmZixNc4MiO9bMQO9BYXHU5XDKL9TCJ4QDGZjXPejnZxO16WYEnuJ3h5Fm\\\\\\\/t5wtFPdKMwIj23POUZEr8mvbOVdPHE8Dwt23sXwVYNK7P9BEFo+WlvtKybXzH0cctrxKuRh9MObjDl7erWXtSA0nfVePZ5pnS2JXhK5Zv\\\\\\\/GV7sir8oDVFqjXYcZ2Orx+7UvICHsASoeDz\\\\\\\/xYMkpKdjVlAiM95qLgHPV65gH8StiU8dbEYotLCECCU+zcoPc7DUBKNgXz2VhiEvg4I3mr6MSn0G17hmfY4ziMRpKnILdXxgbt93Kr\\\\\\\/A0GOK6kpcI1Xya1Y+EFCn76pgu2CpQI5ChUHtRFU3eX+TZ5+X7QPpGG6t3KDp4FXs8p0oSTHW2aioHl8mYpb84DOvQH1rqPkFfif1oGI1ezyV+GjJl0rNuWa+EgP4W4YGHIPIxXjImz5gCuVst2SdZOuEWggKSJwCXovqak2yxhJEGXFCoi7OYNI09lCNmCv0uhuTHlbqINkXU0oWkpwve1ItGC0FTFxlflBFSO6U9\\\\\\\/BLEBjGbcofsTDLiRrwnJJovFFE7jmmWpWWR4na9yp4S4tEh5oYGOxajQN7pWp2hA014RFhrKuPg4JPQ+gpV4WrNPxhKs1eZrxJG7bIZdg2OAT8QQ1L\\\\\\\/6EkEYcWs3o3oo59qDSU1PQUQp8Xc4v4VQDAQsIy5\\\\\\\/tInXyYNSM7kEJmnjQpHVtPPvdxk06cR6NtZViDqz5xts+eoyVg7bF9lLNfgkGDycCfBUn0VvY8Wy7QGTCEVVcvO9ls0G0zYD3QKpYSM9Dksm2cQjrJCYcl3wUNS6YWgNmuUgcV2+dzh9\\\\\\\/oYgGKQMIMg8PXZ0aUZr2oiptq0a2iN00wT5vIIY9UJBBHjdOKe+\\\\\\\/BXtMNP4ECWak+XTxiOdXXODCm2Et3M8AqKm78WCeuwkJsTfF4vMKQa0BsQVkaSpSRpJPcs6IuU3olzF+IFl2S3lqOSEVQWlfjXPyQ1J5qZUXSiKtrDbsN+QqR1jhSLknKMWnWxYKF++hVrQsSS1R0d\\\\\\\/hdatqzpqPXuRxr1XQn137u+yZhyqoUaqUhEg3ZW5y\\\\\\\/esP4yppzMlEclPAN840KdnPDlvfYFrJdWNFYXOd59Rk3tJZ4HkAuEHFi\\\\\\\/AagOn0adYzQdZBpTjB2a5wyr\\\\\\\/F7nb0zdBC69l5LFDMdUe+pOisLPFVvzlwwgx3xzuTgv83yGnF\\\\\\\/JQWbIDDcq9QvfZZqaerkLKZ8FC6NjqKLS2+BCGlK7Kmm5XNdOFGoLOe4JELnMhf0U37swlCbevFU3ChXM6gn+l9wH9T8j4OSB8cCEc8z3uBVtTAM90eqwZYl34DEncSaEv\\\\\\\/ivANNnCqorffZAaA8EKMfVx+nVTYbmSoVDVO9Xd+k5EV1ujXc07\\\\\\\/nPgKkU4PbYDR8IZSl0k8Z62qcnN+RZ\\\\\\\/pKEgRVEV8\\\\\\\/yohsHz40b4KB1PSxejFYYuuqMBgfUn8xphveib14USQaOQ+RjjkVa57SBGeJbJ6Jhz0HO0M4D62VMoRlXzE\\\\\\\/VLUfRLdA7XvBUl8F9UR2fl2QGVl\\\\\\\/GHsboakrBQUzQuuVknZn8JNgbeHYMtN8e5pE4pYaZVpB7XTxxbxl4z9f5eu6nFpHShlI6ZS+xOC+BViLQCTAebH4tp\\\\\\\/MIoQb+xdpdq+GGR1gbT0abiUl7nT\\\\\\\/jSSzKr7ihIE2aMdg4jESIBjxipimOMY6dDIxtycLhLM612Tqvwx4DI5ox8X4UcADDly0Txkod67acUxlbEjK7fXgIolNk4fi4yuDLHJ1DtG17Jw+wuCCC8XVLYoNWiGkDk9b63NQf5L+m1O3iVSRR0lNsbp+ekvHw+Qt2mcTcC+RlXVHKpLHWeGs3vYjpN4xXMQQMSqGXUT3SlVpMF4rOUtw26j\\\\\\\/bXMemQeVALIcezVKhSbqHft0U3V3VVl3XxDOgXCtD2F4BTA\\\\\\\/TDLrFYH1zaTToD3K9vhPE\\\\\\\/PPqqQ3XTu4VxEUqIadn2kC2EGv4HQSWKGhbWDHULTIkrS7SkROwFsDCjKn2Y6gcHXx0BnGENYxxT9ZgB6Ulfwx9YMxP2vNPTZPf4EcVedfjb1v2qcvy8vS8HcLjGzLE9qPN+nLkf\\\\\\\/Wse6xYkxyBs4SeHZTXxbySGkkn2uaLMbtdVX8RnuspbmWZRaLxiowKpk+Y8ErdlRtRxjalM\\\\\\\/69hE8XOfoMlWS+30F0UMYQ0GR3f3fAwDQFcKcpNWhoIoFaDfAJ\\\\\\\/EX6pRUvNGanfNMnKZlFhmaWJaOKMUWaVa9mpdCBEW\\\\\\\/EiRMFeZ8\\\\\\\/ZZ+nAyTmt7ArdK8kxeE9XRMnAjYr4HXOWD4mWheniiALHYmNlcxGX+a2h8BPR2hUl6NlNTVIT14ZBBMP55yx7DfG7QDRd49TZ8wJBaKfsr0CqY2aBv1CN1Kxclm+6\\\\\\\/yHeMKVQmOVSzcyPiyUHLYkGK6vgOIOkEz4X7k4xlDPxPpSsUIPtpxucSXtyekGjVo2hZL6faGJ\\\\\\\/\\\\\\\/1mh\\\\\\\/E2RW5Gs96njaWj\\\\\\\/3D97zVE3p6OXFg20Safwlo9pTc8AwHMH8JFos8E55tU1Kbb\\\\\\\/ytMOpSBcsbnGWWcgvJUqi\\\\\\\/AAfYrIgvGDwl9FSf5i\\\\\\\/ogcFV5ow+LgBdquSouPy7UFDqfGu\\\\\\\/\\\\\\\/4ELsT4iNMPqldJrBEbzyWyquvpSf5NtflUE04LcIvZ8wdivSDPXjrpB6j5DKQeJbr+b24mByCiqW8D6mpeqncSyXlXozQzjcOYyMRPhO4RM45So+yojFF9mp0701e\\\\\\\/WTOE2W7BROk5ksUS88z++RTrJZysp6CdVOgzdkDFifB8S0w3\\\\\\\/ttL\\\\\\\/HP00LozsKj9d8hZQX\\\\\\\/FJNenYAYhJnBMf0PTRq1TJX7tu8AR46qvDl5T1yEpqq3yAQfWCA82r3ylhGuGIQKP4hNjQCQcbo5DeXRU4QJSZkLJPHKJQ2s8clDVJPcrOTjGXbzMTFIvEUKwSFF78YdLZrKkegfdHi5jlY0e\\\\\\\/C2tiI8dIT6JoRSzRmQPznSO18HcdGvYWFUS4LdyjADPIMmy2fXI\\\\\\\/DH4DYnHVDY69v1egegc5AtH2Eh9JBss\\\\\\\/VwQSyNfsTMPinBlsrafkaKEOiXaHW2PPdYhB2XhnWm9Hfu5ZQI7kbDaQxroXBiCwxGYEMYtduuCYzyHHDM3r87JnD+JJsRsTGC8xqzK2687Cvm8BO6vRzFWl\\\\\\\/+GlqsRc6B31am7FFlpS3bCpP9fn3dy+H52du\\\\\\\/BPoltBqNCp5\\\\\\\/JNtEN552wATPz6mOhuUV7xUtj0o0hb0CnJ52Vn96ADsJdtq8i5JGIrYMwzsLmwFiYCnbAEsbpxyarh1PG3kTmBwOFWnPfKeHRLJcf2f7uyPrUUEtP9Fh\\\\\\\/CBKSOT8SbFRHWT4GgQB6D2hhaLBFzs6YQAofa5rZyDGvxtvuJOZR2adx5JIrPeOTgkmhpW9FtN6ZPys5O07c4laO2Ht37Hf8qQ6bPoE4THbY1Ha\\\\\\\/oTpaI5\\\\\\\/xPtOlfVjAZM6mvE0vYIZ1286fzuOBerm293LPm9lHktUWa+rSxRcoDZX\\\\\\\/b7Za9TUAC07yfpzeon1G6mdZTtwI\\\\\\\/g0kA97BioY2dg6GUTJlIvESUcnoGjV5RFqUogrXtfGDLe21702FObkz0TWyInf9RZCIj9Lrav\\\\\\\/Ieo5uHnGekcJcKVFHSqdF\\\\\\\/fdNapWSPocbLMP+oy0XgUfG9atuVS8CSxnYWNTEddy6t15im93OnEr1t6QFGQCgplqTo6T0UDdUwgBEUf4XKJualoQ+zNETJsS58Q1thfwdXT4Uag73X89R6cOQuKwyM3G+dn1jfwA\\\\\\\/yL\\\\\\\/eRI4VKTeJi9AV+W9Lp3i1bySlye0rk5dz+aHyO6mBFh+619j26+5tV63rkImhJU7IhPrOUTyGLOD1AntVssVEB5wJIT\\\\\\\/uKkHXeAp6yKxw+ibIJX+KTOd5HgZTNfMAl+BRxpTpdxwJ33wVd+B0rProLJL66ahaCdGEUFltI2hRNsJFRjYHEfbndj73HUiPSU3yokbJuufd7VBAGjC2xXKqekjuXsiby4JrL3\\\\\\\/UzwPzrPm0lpQsCjHV2LP9fGNpfc0CLt7E4TpdiG9Wt\\\\\\\/g+08VpDvRzwPafRvfEVc5zc+cgEeHZU1KD3R4mPSEsyxzprTI\\\\\\\/nZEC2ZcHP0mbHJdONvo62UyL8zVaPSvyWvPgn0K1eNxFkcSkwWgKReGFQmDhpA9Io6Gs0EjkAOQCrPWw6ZmfXUyHqzDXlIHFGlGzQ5vOdH\\\\\\\/8oaR1d7jzj89ncJWl9Wrtq8U4COdtz4Y+2xOL1\\\\\\\/qZ+mY9GuTncbJf+bDVZrgBCy1thDMi+ljAoirIXphpsZ6xyM8xvuiLxQnYKcNMOG8Wtz3+Edkwa430HSLbXn2oQfoqAjWnHOWMOo033jzYkN81eVNXxGVAqFOms1OC17zFdeIO3Fg5Ur1H6PyZWYogwF+OgWayXAj\\\\\\\/KoazSMAWyMAH\\\\\\\/vczZYs1WBV3O5l6SMQCgA5W5OZMG5+g4gKWIdvCUfPnXuZGJ06xMWIRafEViws+dEVCWN1R\\\\\\\/BeBuet2zNRwzrKxMLKBh80cw6ozD1HtDZJQSGg9juNemJLFKWqM7oY85jB8QgFE3ISBcfeA88Ukp479Xal+MAZJnXmB6htofVSPZwVNvAbkCBnus0UdL7\\\\\\\/TSGBYr5HtBLKEDLO69Bf1duFhJD+2LDwTC4X7\\\\\\\/pmupA6nasQ4P0hKQDUO2Fbw893mSgPmhDyIRHNBi1Ek4TgMoE5Fte6ZAROfsUxcG9y2Dv7iipu0gbUxQxPS2IKG0fOTx2p88ClnyEhrxYYgLBhrOJNXbaQS0f77DZmJU\\\\\\\/1iSysvlLcupu1UWrnlnO1JwqJ+9MIwqUgRZoxI42i0jEgEzDrr4kp1iz2tclI766xIyX6Udn\\\\\\\/DPkCUQdd7j4AVRkciOj+xEdILae3siW+VHyASdrF6tF9pXYrQvi6ZhO0Od8EDkwPCNvQgG9\\\\\\\/aGdh3opRrGQ399BNfSldIMl2dVcsBqjiPf9kRKXkmCRYN6hcBEnniyIHDmHuITugwxM0syTaYg8xawIzSWHqZXI03vN80wQg5VMzOJjAzF7ZIMMJxhXSVDTEIOeLuVsabtLhYNC+AY0z0SlhCSrnGuPF+XN+qaJlxAU8N0D4dh9PFD6XCYFwzsiyBeEvkOf2vCEBjoMJdODQxOVVXRTbVvDWViIHw+7w4AeWY\\\\\\\/EvXnWLB16Ijf91KiOG1jP4OehzVvUrhEw6P6j6jxznX+5AYkqGH5w85KOunf9ehdctu+wqZyumacmyqRlUunzxOF7ZJh8D0mXn1reTyqE0cbmFoG08dLsmp0jhmwsMr3Nvm6o\\\\\\\/qtZpZfrYQeEVqVUtqQPDeX1kTTIB9Y\\\\\\\/4pSzr9FgnV5LFIeFFow4+BDTpT9vo6l7\\\\\\\/8Hyv\\\\\\\/n+rWIrAZb8nBCrcFKwQR9j43keeK5Dk2alUOBZa\\\\\\\/32Wsl60XrL\\\\\\\/f81sw\\\\\\\/VVBObgLpBKA402FUfRWEAzrPAgdpAECfZdIeVczbwH2nhoRikkFGJtpkaZKXQdxhxj0UBcXYder2aFhbjlhbQmVmpf91mGSSIvaqa05bGhDonEZX2FdtJlxtBKUtSEimtaK0BS2is1olp+o6Cv32nlkz8DFJZdnMwQqF5ASKZAUif4glt\\\\\\\/CPfzeYXddJ7twbceX2xT5bm\\\\\\\/LS0Y+TUd+FiJzxGBTg036LlTlF6ICv4svYdIl1Bh5usG2RB\\\\\\\/VCiVFi1Anj8A\\\\\\\/eGgLnNJmxDzU\\\\\\\/W2q4lbouXWBHUl99jMfjrlb0qGJETsxR3MIJ0VBgWUY24k5ep4z3th1H6F7enT8pioz7VRiFiM\\\\\\\/DBoZnN8vCNnEr9abwVTzotvVuZv+8Fx\\\\\\\/kBaJJzslgnqEWY6UpJvvg0i8XXz\\\\\\\/vLAeZYvIYIXBqDNSzMa1IizavQQGVXJjR5vMlvrsnv\\\\\\\/eioM4haN2le0na68SOz3gYNDlTm3xJrhw4lOzfjwHNVA2m7F0iHHN+9LuxFIre8GYNWvpgqFBw8\\\\\\\/Y65FeEdnDmxryH67FKGW\\\\\\\/jy\\\\\\\/7MNW2c0\\\\\\\/QVuBlSYWBxJ3N1w3\\\\\\\/XVpQOhD\\\\\\\/SKwhhaKu\\\\\\\/YctwvWEi+OX0ho4spLfoY364WYrISJFXWfF5NpK7Uwgt1ejL5JHiHJuUQ11ttJaUmYEE9UrGArnsdUmroszEhUR5O\\\\\\\/B7U9lyLUVVm95SJMErPTnzgQa6yWG9jLYPL2+rcfPHlP2I1\\\\\\\/Y3q7df0+GVNKDwjS\\\\\\\/JgnWGum\\\\\\\/YsE+tQKIE4obGfO4KZnVX\\\\\\\/h2a0UC0KysPNDGNnjeTa37LVGDnX0NomRXlyC\\\\\\\/b7Yn7cDzs\\\\\\\/qYpSXqd5s7wnevEUHQHfzIWeSoZDSyCYgGVgqw\\\\\\\/i74Fdf\\\\\\\/j+qTp9Zc6whkHa2oBXtwYwLjqWdS+22u5xO3ZcdjcZ1Ivh4YrEa2r24vt6L8m0PheoZDIpJvlKOaDJ0q\\\\\\\/aO\\\\\\\/xJkuMkhW4IfXmBscDYHu4Hqb64M3EqxKS0+Mpdk+8g2hx42XMisMnkEK+UQFfEdG8PhZRRksMKBqithdI7\\\\\\\/yYAbvzOCGm0OjLEiWtY11\\\\\\\/otF4ylt8iado3kgiyf5NfrBF0ZwxM2ighJSc7hGn3ioSNiuJuGkkj8Z\\\\\\\/hpWk2r1uNy\\\\\\\/BSItQEIvNmDJ9ldOxNRVn6zBp4Xq5A7dRRVm3PHlwqmwZx7P07b2aLgtOUafXWEYiOgh1wD2xFaY3e\\\\\\\/e4bsLItQEiK3hs8mNLRI0iWGAKKKRmyCgXrGkHPNtQDhLdQ9f4z4CM6MZ3EadULgfGo1yZfpG7fQdisqLbWJwbYPFeT3vEUGAxC29g8p4h9zNP2sGlPH70uq6HcBzSViHCJH2nFH47eg+PgG67s+MmCPB8DCoZCnbs4U20+jE\\\\\\\/CKbtFTcb9Lat2gP8ZMmLQbds6iPDNsVRD1Eb5sB9MIOtELWov4WRnDS3rBiq9MZsVLtiisfaloiCw1oEmYOclLBC6TqG+01VXV8yAQ+Opg80R27Zi\\\\\\\/XUyuyej342lceGb4AogELBcdwDVi3LglU6Uif3er7UwF3idmBSAw7xalFPWIgdghkWU9DhbWMLGQ\\\\\\\/CZ8TO5T+Z+B+Ccj46DNM6dWITjatwi9\\\\\\\/7QcZ9N8+iJ1s5YKlIHc4cEI1WKT92R+X6g1DYdE8WzRN6U+U3p3m7fe8cYNEr76Ud5LdYRPWHHJW\\\\\\\/eEwz+pE5BjEtl09duoLKZBV6zFW2n1kg8TQbp108vFlkFWX4AABgDzAC8vt4w3y24Tha8nE7p7tAm2j9un8mXCR0JaJI1E82sGvulF+e\\\\\\\/lPQWbUoIgiJPfS0qM4C8wF8h7myRJMiQClQes9x0vjNSAs6OHH1MaMvqQwFnBfAa449E6afFN7\\\\\\\/Us9X2K68\\\\\\\/1k6M++DJbNuhZmL2MK5kNMHFx4KVvHs4nyqB2uzQxsc8gymr6o\\\\\\\/5wTEXFcUUIIR0yliWzNtIoAK73eYKuip\\\\\\\/aRPspI6YEjA5x+e7WxjabnIHZ+evSFyI5VS75m5+IRDOA3YU4wQWG\\\\\\\/kUo+xPFSHUDTRj0tEz1Yy+4GOeVRWc3pkxq0SU0d04vR3AlyyXDzOY6fOTbcILivzaEL\\\\\\\/WkwFN5Be3vFt2nyM2rr0XDY552CyHK0NKbEoXQXhX9t0Adq3VSrmZjL1SH4U7YsULAociZqBUiNKvWhXLOO6GoNvynRONzS96O81ukMJ9pIYgN3BuTlNCUy3Zv+JMbUempw54JU\\\\\\\/16Nnqry4qMoNro0lVBB1i9evTH8JkxfingFaEdZjwf58W+vwHNgK46uhiH9nPy9bcsTktZ\\\\\\\/auvbEvdhR8NKYpkr0QI09NAPzsDmUXBAcVUJqM\\\\\\\/RIudrXLlkxlcjvdAqWxelWfZEStxsm3aRL5RV9tVob2eM8hmRWAzUG09uslu+urGZULHR8qqRqS1uw6DNUCG\\\\\\\/nWWwTsvQrVLKONtvHc1hjXFBpGvHbrdix2\\\\\\\/mHLsz9C4as\\\\\\\/Kh1cz7fCEl+1nyACLS3k37ZCVkmaCGElosoM3lgUzW1H0Nmwe9gi2Ztxh8++cklHKmCkfxeFCO2mO\\\\\\\/PsgKvl+YKWw3nsi5y4AB0LUlODjKEREq\\\\\\\/bW6Wg83WXJ\\\\\\\/dc16vpJhJcOyk4kdre+kKGMmS5IoKZdPaViA38rJo1vIomirP+fud1CUU51xnRkerTHFBm2IikSXXxThMv+iEIFSUFAIArQ7At2JUpLnof03Vk6RyIw9jF4wAZS2RdNTO7q2k9ba9dJLQlIbLhLQmmYtPZqbar9bsGQKJUJfn+r90YhmXSGf8xbDllwU\\\\\\\/4lWxlh6GCG04DnBihIwvKnnuYdXxKOjuvREz57qapGllsbhFpNuNwNu1ZirugznvV\\\\\\\/pxYWKCuTfYU2Xcp4XuNeQvaOWtdJ\\\\\\\/OsM0odl5hWVOmsQuWLwQ5pTdnHR5fF3FPknMsoodScY6iEjk8y\\\\\\\/NG73vFChl779w2ZZjM4UqbSbJhN3EztwSzOUPOrkB+T7IDI4\\\\\\\/OzV+FlMenWX1algFTGXLb8RAOWRFK0pKP8Pwwa35PShshZvByJIpe+Spe4Bui1buN4XVepmwXUbVhiHAFfuemJlsl1xktn3jDIXeSnj8h5kRsgMw9MqFQED9+7y2ZXnEzbZRJfhTQg5G24ej05KH39CpW5qLWPtcw+slAxB3IMLN51z5iUQAzxthVqcZca887Av0XW2dPBbttUq82b8+mRGdEhlv7EQw9D2fBHf9q8JWP6rV1sowNWEdMuFVqRGQxyFQX9NkMmtErhcMb9mnrHHvwhZtYTEPLEsr5KMur9qNFZQJNhIMPgnnloQzkWfXsOs+JMCR\\\\\\\/Vo7gCrHctiAtFdaYfNsDt0sHiKF8KCfQEGxcKP+dNvDV8TcVDNaNePA2FuQYyo\\\\\\\/7Ipde28j7+yviYSCgHSbnM8jlNZmxm8p3YEYDn2r4k8M1txtyDL+WHGSAX09BWe2oRvR4q+Vn\\\\\\\/HkzglkKLUGtOP\\\\\\\/X6Q4lnUvxh0PNxrrM99CjpL9Oyj7839R3N510nRVOYM0ck8aqde\\\\\\\/V92o6EFpQqcPWIHIKatENUEzpkJP36w6W9r\\\\\\\/FPsfPcj1ro1Rgx6J1TWBWEY5Di8kw0DTyNdQhB4oIzvbIExkm6UH7L6zOTFLCDIFqZp7T3nVCVlXfYnrGueLGiPj4ZIzYlzjxbIDUqcy8w2hk7bHdOAzchtBDiRDI4CwjBrUVyHH1m86fM5CWKPkDrucIKqXpFRYQRgHyf6yhD++1geHFxZXLprMMZvlVcloSRWp6TpfyJWwHYuW6SOReTXL8UFTnK+XLmYGhhnRVCNmfJhfmq5cmd808tMTokvLUZAV71ekkA7IJlscXbyxXcxa7OduULdKWZ5Bjh2VyDWjQUUI9JH2FSouD1r2NlmCjzMiS18FYh8jck4raf1obE9CgRKDj\\\\\\\/u4tH0pf5JzJsqhgoJPDrvY6jzkJtCybx51NZSLqOV5HFOA8K6GMFmU9oYTdTA+1bopFRufxSw9IFMWTkuZJKspeNrj9UjWMRSaoiXBwNWVAK7egNNHz1byUHwM+psuwdP9keuGvt+yY73icctTNA6POb8W1ElcXPPaqRtb+ZuXuBlPCCuKXYYwWPh0srvOFOEWxeYRxDoqNRr0tCuZITdhcjV9LzU\\\\\\\/2A\\\\\\\/vLovUtyYOO8q\\\\\\\/2qPbewZ49X75mkLn50yGg2vvL8t8cqseLHnHXTwWQ1UX3TcYAK\\\\\\\/gxpkjHySZQQaywTQJhd\\\\\\\/ujfFybSJX+ObQq70nDmR5ZAbo2H1eNnBMbCtCqh7Gcbze2W7mjdB1c\\\\\\\/VrjeJB9x6itbv8W85KInwt4rGTdgSwe\\\\\\\/m5RVAfM6KD2jeda5Y7eK2QM2CX5G62vJq\\\\\\\/lNafnNRa3MONEzzgYipG4h3UtKf7+yU42volbQx56L1DrWoGrWTkWpdROUq4hy+4zF0ISHqaQu3eq4Rysqyj1+UMazoTlpdlfnig9AefGUO+IyAP0Zht7Ai6pTCFwcGypUsTPbx0FjPQRD6NjrzjUVyu7WmjBGWXg2doOfZtSfq94h0TEnj5zBZbbb1HOvFuQapWUIpav11VINLxyLuti5cU7wh2CDBo3RxeVdZHCg1cXkr2jzb+NdH6Fia4DnP6OLrfZL74c46XAKBe1nmLGI4X5HDqCk9Vp8wk4HpsvDXhXrQ1VTgvaX6en+nJcmIcDeyKeKX2XrIR9ITyb6j3d+LZ6pNBKdm\\\\\\\/GvIDVjhfglp524ZFJEo2k0\\\\\\\/4BCYE9fpRMSuJGwPmW+uRXx5sIOpJCGtmR7\\\\\\\/1ZdFyYwaQ6qCQjenZpY72h4W1p1hfiqrzy\\\\\\\/cQNQJTKTfAOnvnBbsT4QUJiaNJgOdb5kjWLbVTAcdLxO9hzY7KaMkqbUD\\\\\\\/j5Sm\\\\\\\/5jZ4ydPMu\\\\\\\/uC\\\\\\\/+9jib8qD5Ipd+sf+CPSW6JzX\\\\\\\/gj9gUEc6P2cuMCQ\\\\\\\/H7e2LSR0bmJLpmcX9xK7r+s816g089FIPSuj1e8NF5Z2f4FlHdbEUpASZUSUaJ8MIaJA2ChSsLAGM7UHpiYnJACcks11j1fzmjbotKV1RLh3SBcwe0rwCu9jlc1jrMRKiTohhDUhbq4A8DrC4P\\\\\\\/GaR1PaW\\\\\\\/MQ4Wp3nfbf+vL1nBFgTNZ4C2RMlGQ5vi0g2Q9oCzisCbGH9qx5enTJN7e6s+Q59NmSfmm667j+\\\\\\\/S1Ws7wRr3wjDb2m9ICp572cU\\\\\\\/bJC8uCZzfTot5jBjV0OLO0mL8EztmI69cF3rf+z6uFnryiZFnIcRFzQB04ttZ7RVb6ltfBUwzJBEPe5Jb2ty180Xi1w==\\\u0022,\\\u0022iv\\\u0022:\\\u00220AHRVfNYCev4+QQLMF8hNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022692ed6e259f1998d3d8e2ad23a56310d30154b4dfc80188f4e246ddbc6e1a009\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+FxoMDqFNm6zhfk2Fg90+ofEEexJIyV\\\\\\\/CYO8760p3uHp11A\\\\\\\/MbF8akulff88YF\\\\\\\/OoMCg2tVuSBGPktNDlR6KcXwMTf1IyRKhrBKUAm1baXkUqrZnxTQMflhWc78ZbVhHxWGUeSSip8Z+amtCNOdECce4q8pTOCvaNwyB0obg+WpXnSAdXO9ZzZyslcGnLyzK60M+jbBpm3RrJOcVFjtw\\\\\\\/VTrmeH68sDoQRx6UfZjmOIbwomb9isImKnoZTrC0mOM1\\\\\\\/T4LpPjrNyaDNKJ\\\\\\\/ecywTNfgbz\\\\\\\/\\\\\\\/bOAgxi2vthX4U2z1QTODnqKCS7ucInCBQTWdsoca94efg503QpkULAR3\\\\\\\/O0c3AkOhYXszSOrROCSSuNSckNWwHgeJ6CiBpK1ea8XZHD5V+BolTpk+2+y5J0ovIEN1QtlTFvKaPytAS7PHdrg4CCgx57WdgCLQvMPRrXeplVD61kwXDWZ4Ni2ufNGy6\\\\\\\/D3dSnAZCZRTLbYzQ6hOFJ2JwR+nBpoE1douAsFAuy6tXsDvLA49iaeUtvXYAP7GB57pKZb812UbMnSLDD7ueU1u101P1hRzQWjxhJkTxmvkNLMvrPlvNnuVRu7nqnzG3HeD9jFX35BziH+KX+LdHxnJ8sZLHT+DSl9xXCHLkSyvAyv+7ThCl0LAYMdoLQJe8Ujdhp4lX74EYsZr6gNh\\\\\\\/ijvO4YIDMCLLrhukofOmaNW5jJK5nQHqwym8Q3UeOWsO5HA25ymA4MTvWVH6O3b8Ulzc6Em\\\\\\\/BZHhFrlSfaQNz0Cdd+92mUdyFgwv17SHojUsb0z6embcnuvXxRCchl8uWrLfCtOFMx2K2yg6sfbQThSPDjVmAac3KAH9NwGYZ7hy4318Kw\\\\\\\/w2DkEjrW3Xq1wCSoj5RFIxr4Pe11yfMfzp1tbbtY\\\\\\\/ol\\\\\\\/LSMfjLv8idYsmXx\\\\\\\/IGftOEsoKL+xHwbzeBH\\\\\\\/Y5jjiDlGZwNDdAXcy5u0+tTPyAlvjwrCVP2lP7vCQPvzwTHTtf9eeSD+oMoCA5ld8MpEziTq2\\\\\\\/Wxw5dDgXkRiYsyxcewtIr1KNyM7XXFxJKQEaPKVdO9QtnUly1MYh5SMg2aFNe5oZm5UXf2PW\\\\\\\/QJ5qLx0jbh2jDC5sgepVUtke91fuUA4Z17Ee3SCpLN3zd4Z62WkuD5L4HWp4zfDHDdUKrXnq+lxo6XxSG12SGFHKIoufGs2mH9I\\\\\\\/gPXDacfosqAtpGUa1QZUAasskhmghRnKxGPuwBIVr1Tic51ndVHP+JF9zpXXr80PiXvZPzNCUjDNZ9OQMcDOqYmaGTbFMqDi7rogWmwsTyITQ2hF0ANPCD2EItN\\\\\\\/Cu11SPDAF2QdOThnEaFYCsEekW6Am4R5npZPcZklEfNypLlpQnqj08\\\\\\\/16mHdhdYL9biSWIqDyU2jbHebebwpPytiam8OT+E11UReuDcpwrH7u\\\\\\\/GZOATfZqy0l5wd5RRv+nqYisbx7y9Tqgr7CvS+JqWTn8+PRmXnAMB\\\\\\\/nZ5Ingc1Z8L0sLBpJDdvZeuaIojJ6FNe5Dc0XxGc4GZpLWEV3YS4JbMBYl3UKRkE4SblDZ7LZ4qWz+jR3LKHe13B3a2O5mDSuqc4njmzX00OVmtnVnk0vM6P\\\\\\\/3wwfrznlPEImd3KNU3z6XR94iAeGR8Tym3U3p\\\\\\\/P6VE6cJviiHD8BYtHvD7fsWy8RJ3vdHZuqszZOxtyB7ntgPKrcir+mH74BEYzPh\\\\\\\/PoSjyw8YDix+M\\\\\\\/EBWMw0B6VvMRJh8AgWuq3svbQLL303qILxE93jzW2M5CnFUNQEKfmlED3t4w5l6td6G1PbePurDu\\\\\\\/4yBELHUG3+yqo3vgpeGWf82xjcWs0owmRbe7j9G3HR5Sub7MGSjtfLvOgnLecGTWtdAKqROrqOi4NmIaDex7iqrTy+YMiZK5esYbVisPuQLzZVrJ4FeY\\\\\\\/0mmPWl0ix20QeiX+IVQwzRMwZtInHazftwq5\\\\\\\/RhZzY0CMc7aNnQ0ESXcrNchFXL37g+nUOa9fk9L0odAQMI0ml\\\\\\\/r9WjMZHFq3DTmihyJESgTMUzNZFXzBaI58WsDC1P\\\\\\\/AGmcVuz5d2\\\\\\\/6EV+RSbRIgn2+X2quQt0qTajpHP8VMxxxS80I2d9Nxq9wUean0fLNawP8GxrGW4m\\\u0022,\\\u0022iv\\\u0022:\\\u00228brRr4NP5zPkcVZ70NJq5A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002290b20e16544239a21b07ac17fb9a247b1156ef58312d3f53d796ab8313a2c3da\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460418 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1819" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-llDy9IsXDUur-MbCJfT10LZddmyLgtAoAoXA0FLD7qQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1819" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-w1S7dPb9TyGJCx9OSr8oMYc8oxWDEtKV-9EB2mGYyaw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1809" title="Fine Mapping of the Barley Chromosome 6H Net Form Net Blotch Susceptibility Locus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1835" title="An Improved Genome Assembly of Azadirachta indica A. Juss." rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460418" id="node460418" data-pisa="ggg;6/7/1819" data-pisa-master="ggg;g3.116.029637" data-apath="/ggg/6/7/1819.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Gene Bank Wheat Landraces</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="1">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="2">Jorge Franco</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="5">Carolina Saint-Pierre</span>, <span class="highwire-citation-author" data-delta="6">Prashant Vikram</span>, <span class="highwire-citation-author" data-delta="7">Carolina Sansaloni</span>, <span class="highwire-citation-author" data-delta="8">Cesar Petroli</span>, <span class="highwire-citation-author" data-delta="9">Deniz Akdemir</span>, <span class="highwire-citation-author" data-delta="10">Clay Sneller</span>, <span class="highwire-citation-author" data-delta="11">Matthew Reynolds</span>, <span class="highwire-citation-author" data-delta="12">Maria Tattaris</span>, <span class="highwire-citation-author" data-delta="13">Thomas Payne</span>, <span class="highwire-citation-author" data-delta="14">Carlos Guzman</span>, <span class="highwire-citation-author" data-delta="15">Roberto J. Peña</span>, <span class="highwire-citation-author" data-delta="16">Peter Wenzl</span> and <span class="highwire-citation-author" data-delta="17">Sukhwinder Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1819-1834;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029637 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460418" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:suk.singh@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">suk.singh@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, 321 Keim Hall, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jorge Franco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Biometría, Estadística y Computación, Facultad de Agronomía, Universidad de la República (Udelar), Paysandú, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jorge%2BFranco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Franco%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJorge%2BFranco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Post-Graduados, Montecillos, Edo. de Mexico, 56230 Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carolina Saint-Pierre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BSaint-Pierre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saint-Pierre%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BSaint-Pierre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Prashant Vikram </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Prashant%2BVikram%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vikram%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrashant%2BVikram%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Carolina Sansaloni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BSansaloni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sansaloni%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BSansaloni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Cesar Petroli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cesar%2BPetroli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petroli%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACesar%2BPetroli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Deniz Akdemir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Breeding & Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deniz%2BAkdemir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akdemir%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADeniz%2BAkdemir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Clay Sneller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Horticulture and Crop Science, Ohio State University, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clay%2BSneller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sneller%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClay%2BSneller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Matthew Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Maria Tattaris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BTattaris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tattaris%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BTattaris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Thomas Payne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BPayne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payne%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BPayne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Carlos Guzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BGuzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guzman%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BGuzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Roberto J. Peña </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BJ.%2BPe%C3%B1a%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pe%C3%B1a%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoberto%2BJ.%2BPe%25C3%25B1a%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Peter Wenzl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BWenzl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wenzl%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BWenzl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Sukhwinder Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sukhwinder%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASukhwinder%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:suk.singh@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">suk.singh@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/7/1819" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460418" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460418/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1819.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460418" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460418/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1819.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460418" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460418/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/7/1819.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460418" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460418/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">This study examines genomic prediction within 8416 Mexican landrace accessions and 2403 Iranian landrace accessions stored in gene banks. The Mexican and Iranian collections were evaluated in separate field trials, including an optimum environment for several traits, and in two separate environments (drought, D and heat, H) for the highly heritable traits, days to heading (DTH), and days to maturity (DTM). Analyses accounting and not accounting for population structure were performed. Genomic prediction models include genotype × environment interaction (G × E). Two alternative prediction strategies were studied: (1) random cross-validation of the data in 20% training (TRN) and 80% testing (TST) (TRN20-TST80) sets, and (2) two types of core sets, “diversity” and “prediction”, including 10% and 20%, respectively, of the total collections. Accounting for population structure decreased prediction accuracy by 15–20% as compared to prediction accuracy obtained when not accounting for population structure. Accounting for population structure gave prediction accuracies for traits evaluated in one environment for TRN20-TST80 that ranged from 0.407 to 0.677 for Mexican landraces, and from 0.166 to 0.662 for Iranian landraces. Prediction accuracy of the 20% diversity core set was similar to accuracies obtained for TRN20-TST80, ranging from 0.412 to 0.654 for Mexican landraces, and from 0.182 to 0.647 for Iranian landraces. The predictive core set gave similar prediction accuracy as the diversity core set for Mexican collections, but slightly lower for Iranian collections. Prediction accuracy when incorporating G × E for DTH and DTM for Mexican landraces for TRN20-TST80 was around 0.60, which is greater than without the G × E term. For Iranian landraces, accuracies were 0.55 for the G × E model with TRN20-TST80. Results show promising prediction accuracies for potential use in germplasm enhancement and rapid introgression of exotic germplasm into elite materials.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gene-bank-accessions" class="hw-term hw-article-keyword hw-article-keyword-gene-bank-accessions" rel="nofollow">Gene bank accessions</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/cross-validations" class="hw-term hw-article-keyword hw-article-keyword-cross-validations" rel="nofollow">cross-validations</a></li><li class="kwd"><a href="/keyword/reference-core-subsets" class="hw-term hw-article-keyword hw-article-keyword-reference-core-subsets" rel="nofollow">reference core subsets</a></li><li class="kwd"><a href="/keyword/%C2%A0%C3%97%C2%A0e-accession-%C3%97-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-a&#xD7;e-accession-&#xD7;-environment-interaction" rel="nofollow">A × E: accession × environment interaction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">shared data resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li></ul><p id="p-3">Breeding gains have depended largely on having access to useful genetic variation in crop gene pools. Gene banks are the repositories of novel and useful genetic variation contained in a crop’s gene pool. The wheat gene bank of the International Maize and Wheat Improvement Center (CIMMYT) has access to various germplasm pools, and preserves an array of wild relatives, landraces, genetic stocks, and cultivar germplasm materials. Wheat landrace germplasm has the advantage over wild relatives of being more easily crossed with cultivated hexaploid wheat. The potential of these landraces should be harnessed for present and future wheat genetic improvement programs, and such efforts have begun (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ayala <em>et al.</em> 2013</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Caballero <em>et al.</em> 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Guzman <em>et al.</em> 2014</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2015</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Vikram <em>et al.</em> 2016</a>). Part of the existing genetic variation in CIMMYT’s wheat and maize gene banks was characterized recently by phenotyping and genotyping thousands of accessions through the Seeds of Discovery (SeeD; <a href="http://seedsofdiscovery.org">http://seedsofdiscovery.org</a>) project funded by the Mexican government through the Sustainable Modernization of Traditional Agriculture program (MasAgro; <a href="http://masagro.mx">http://masagro.mx</a>).</p><p id="p-4">Evaluation and use of the genetic resources stored in gene banks can be facilitated by forming reference core sets. Initially put forward by <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Frankel and Brown (1984)</a> and <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brown (1989)</a>, developing core sets consists of sampling only a small proportion of all the collections of a species that will maximize allele diversity. Several authors have since studied different sampling strategies for forming “diversity” core sets using phenotypic and molecular marker data (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Franco <em>et al.</em> 2006</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">2010</a>). Recently, diversity core sets of Mexican and Iranian bread wheat landraces that maximize allele diversity were prepared by using high-throughput molecular markers, for their enhanced use in wheat breeding through genome-wide association analysis (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Vikram <em>et al.</em> 2016</a>).</p><p id="p-5">Rapid and precise breeding is required by crop improvement programs, and various marker-assisted methods have proven their relevance in different cereal crops. One of these methods is genomic selection (GS), which is becoming a standard approach to achieve genetic progress in plants. GS models often accurately predict the value of nonphenotyped plants, and reduce generation intervals by reducing the need for field testing progeny (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Meuwissen <em>et al.</em> 2001</a>). Further, combining methods involving high-density marker platforms with models that include genotype × environment (G × E) interactions adds power to GS models; however, <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Schulz-Streeck <em>et al.</em> (2013)</a> found contrasting prediction results when using G × E. Genomic prediction models have been proposed that take into account the random effects of markers and their interaction with environments by considering Gaussian processes with covariance functions based on genetic and environmental similarities among individuals (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Jarquín <em>et al.</em> 2014</a>).</p><p id="p-6">Although the accessions stored in gene banks represent a rich asset for breeders, alleles need to be moved from the accessions to cultivar development programs. Lengthy prebreeding programs are required to develop lines that combine favorable alleles from the germplasm bank with good agronomic performance, and thus can be used as parents in a breeding program. In recent years, genomic selection and prediction have been studied in bread wheat using only elite germplasm sets (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">González-Camacho <em>et al.</em> 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pérez-Rodríguez <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">López-Cruz <em>et al.</em> 2015</a>). To date, no study has been reported on the genomic prediction accuracy of traits measured in wheat gene bank accessions, or on the use of genomic G × E models on wheat bank accessions evaluated in different environments. In a recent study, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gorjanc <em>et al.</em> (2016)</a> performed extensive computing simulation to evaluate germplasm enhancement schemes within the SeeD initiative for harnessing polygenic variation from maize landraces using genomic selection. Based on the simulation of various prebreeding options, <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gorjanc <em>et al.</em> (2016)</a> concluded that germplasm enhancement breeding programs can be initiated directly from landraces or landraces crossed with elite testers.</p><p id="p-7">Recently, methods for selecting efficient training populations to be used in GS and prediction (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Akdemir <em>et al.</em> 2015</a>) have been developed to select a training set based on molecular marker genotyped that minimizes the predictive error variance. From the perspective of selecting a subset of individuals from the entire population, this can be considered a method for selecting a ‘prediction’ core set.</p><p id="p-8">In light of the above, the objectives of this study were: (1) to examine the genomic prediction accuracy within a large number of Mexican and Iranian wheat landraces held in CIMMYT’s gene bank for several phenotypic traits; two highly heritable traits were measured in two environments [<em>i.e.</em>, days to heading (DTH), and days to maturity (DTM) evaluated in drought and heat environments], and several other traits were measured in a single optimum environment; and (2) to study two genomic prediction strategies: (i) random cross-validation schemes where 20% of the accessions form the training (TRN20) set, and 80% of the accessions comprise the testing set (TST80) (TRN20-TST80); (ii) studying the prediction accuracy of two types of reference core sets (diversity and prediction) that included 10% and 20%, respectively, of the total collections to predict the remaining 90% and 80% of the accessions, respectively.</p><p id="p-9">For traits measured in a single optimum environment, we used the standard Genomic Prediction Best Linear Unbiased Predictor (GBLUP) model for the TRN20-TST80 partitions and the 10% and 20% diversity and prediction core sets. For the two traits that were measured in two environments, we used a G × E reaction norm model (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Jarquín <em>et al.</em> 2014</a>) for predicting the genetic value of 80% of the accessions for which there were no phenotypic data available in either of the two environments. For core sets, we directly predicted the 80% and 90% of the accessions that were not observed in both environments. Prediction accuracy within each collection (Mexican and Iranian) was performed accounting and not accounting for population structure.</p><p id="p-10">Note that this study focuses on each collection of accessions (Mexican and Iranian separately), and that prediction results from both collections cannot be compared due to differences in population size, number of markers, and traits measured and evaluated in different field experiments. We included results of both collections in the same tables and/or figures for easy presentation, but not for comparing Mexican and Iranian landrace collections.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-11">We used a total of 8416 Mexican and 2403 Iranian bread wheat (<em>Triticum aestivum</em>) landrace accessions held in CIMMYT’s wheat gene bank. All of the landraces were genotyped using genotyping by sequencing (GBS) methods, and more than 40,000 SNPs were detected. The final number of markers used for each trait in both sets of landraces varied (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468489/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468489/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Phenotypic traits of Mexican and Iranian gene bank landrace collections, number of accessions, number of markers, and heritability (<em>h</em><sup>2</sup>) of the trait</span><div class="sb-div caption-clear"></div></div></div><div id="sec-2" class="subsection"><h3>Phenotypic traits in Mexican and Iranian landrace data</h3><p id="p-12">The accessions were evaluated in the field and laboratory for several traits, and the two collections (Mexican and Iranian) were planted in different field experiments. DTH and DTM of the Mexican and Iranian wheat landraces were evaluated in field drought (D) and heat (H) experiments at CIMMYT’s experiment station near Ciudad Obregon, Sonora, northwest Mexico (27°20′ N, 109°54′ W, 38 meters above sea level), during the 2010–2011 Obregon cycle. Heat stress trials were planted in April 2011, and drought stress trials were planted in November 2010. Both heat and drought are common in wheat growing areas, and they considerably affect several important traits. The drought trial was sown during the normal wheat planting cycle and harvested in April. Plots received ∼250 mm of water through drip irrigation during the entire cropping cycle. The heat stress trial was sown on a delayed sowing date in early April (sowing date: 6–8 April; first irrigation: April 10, 2011), and harvested in July 2011. This trial was fully irrigated to keep water limitations from confounding the heat-stress results. The heat-stress trial had an average daily maximum temperature of 36.3° (min 18.1°), compared to 27.8° (min 9.0°) for the drought stress trial.</p><p id="p-13">In both drought and heat stress trials, fertilization, as well as weed, disease, and pest control, were applied as necessary to minimize limitations. The Mexican and Iranian landraces were evaluated in two separate trials using an augmented grid-check field design with three randomized checks distributed along rows and columns. Plots were 0.40 m<sup>2</sup>. For each accession of each collection, DTH was determined as the number of days from emergence to 50% spike emergence. DTM was measured as the number of days when 50% of peduncles were completely yellow.</p><p id="p-14">Other traits were measured on Mexican and Iranian landraces, but only in a single, optimum, well-irrigated environment in Cd. Obregon using an augmented grid-check field design. For Mexican accessions evaluated in the optimum environment, the other traits measured were: thousand-kernel weight (TKW), test weight (TW), grain hardness (GH), grain protein (GP), SDS sedimentation (SDS), grain yield per square meter (GYSM) and plant height (PHT) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Traits measured on Iranian landraces only in the optimum environment in Cd. Obregon were thousand-kernel weight (TKW), test weight (TW), grain width (GW), grain hardness (GH), grain protein (GP), grain length (GL), plant height (PHT), and SDS sedimentation (SDS).</p></div><div id="sec-3" class="subsection"><h3>Spatial analyses of field experiments</h3><p id="p-15">In agricultural field trials, accounting for major and minor sources of spatial variation in plot errors is of paramount importance (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gilmour <em>et al.</em> 1997</a>). In this study, the raw data for all traits in each Mexican and Iranian landrace experiment were corrected for plot-to-plot variability using a separable autoregressive model fitted in the direction of the rows and the columns of the spatial coordinates, in which the data are observed on a regular grid (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gilmour <em>et al.</em> 2009</a>). The autoregressive model is a random process that describes processes that vary in space (or time). In this autoregressive process, the value of the response trait in one experimental plot depends on the trait’s other values in spatially related plots and represents a special case of a time series problem (when one data value depends on other previously observed data values).</p></div><div id="sec-4" class="subsection"><h3>Genotypic data</h3><p id="p-16">The DArT-Seq (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sansaloni <em>et al.</em> 2011</a>) platform was used to generate genomic profiles of Mexican and Iranian wheat landraces. This technology combines DArT complexity reduction methods with next generation sequencing platforms, which allow scanning over 100,000 loci for DNA variation primarily targeting genetic regions. DArTseq integrates silico-DArT markers (based on SNP and methylation variation) with “traditional” SNP markers on the fragments detected in genomic representation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carling <em>et al.</em> 2015</a>). Two enzymes (<em>Pst</em>I and <em>Hpa</em>II) were used to create genomic representations of both populations. The samples were submitted to digestion and ligation of barcode adaptors, which allow multiplexing 96 samples in a single lane of an Illumina Hiseq2500 (Illumina Inc., San Diego, CA). More than 2,000,000 tags per sample generated up to 77 bases. A DArT P/L analytical pipeline was used to generate allele calls for SNP and silico-DArT. From a total of 40,000 markers, a set of filtering parameters was applied to select and provide high quality markers. Marker filtering was done in a trait-specific manner because some traits were not measured in all the lines; thus these lines had to be deleted from the prediction.</p><p id="p-17">The number of markers used in each data set for each trait measured in Mexican and Iranian landraces are shown in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. Genomic heritability (<em>h</em><sup>2</sup>) (shown in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) was computed as the ratio between the genetic variance due to markers over the summation of the genetic variance plus the error variance (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-1.gif"/></noscript></span></span>).</p></div><div id="sec-5" class="subsection"><h3>Defining 10% and 20% diversity core sets of wheat Mexican and Iranian landraces</h3><p id="p-18">To develop a suitable genomic selection training model, we sampled reference core sets of the Mexican and Iranian wheat landraces. Reference core sets were formed solely with genotypic marker information. Two such representative sets with 10% and 20% of the Mexican and Iranian landrace collections were selected to maintain the collections’ genetic diversity based on genetic distance: these reference core sets are named diversity core sets.</p><p id="p-19">To define the reference core sets, allele frequencies were calculated; modified Rogers genetic distances (MRD; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Reif <em>et al.</em> 2005</a>) between pairs of accessions were then computed and the accessions classified into groups by the hierarchical “minimum variance within group” clustering method (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ward 1963</a>) using the “hclust” routine from the “fastcluster” R package (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">R Core Team 2016</a>). Using the MRD and the minimum variance within group strategy guarantees that the individuals within a group have a low average distance, while individuals between groups have a high average distance. Using the MRD genetic distance is justified because it satisfies the condition of being a Euclidean distance (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Franco <em>et al.</em> 2005</a>).</p><p id="p-20">To define the number of groups in each set, different partitions of accessions were analyzed using the “rect.hclus” routine from R; this routine draws rectangles around the different branches of a dendrogram so that corresponding clusters are highlighted. Sample size per group was defined by average MRD distance values calculated for each cluster and the number of accessions to be obtained per cluster was defined as being proportional to the group average distance (D-method, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Franco <em>et al.</em> 2005</a>). Once sample size per group was defined, 1000 independent “candidate subsets” were selected from the collection by the stratified random sampling method using the previously defined number of accessions per group. An average MRD distance value was calculated for each of the “candidate subsets,” and the subset showing the highest average MRD value was selected as the training set.</p></div><div id="sec-6" class="subsection"><h3>Defining 10% and 20% prediction core sets of wheat Mexican and Iranian landraces</h3><p id="p-21">Here the selection of the core set was based on the reliability measure of <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">VanRaden (2008)</a> that is expressed as <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-2.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-3.gif"/></noscript></span></span> is the genomic relationship matrix of the individuals in the training set, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-4.gif"/></noscript></span></span> is the genomic relationship among individuals in the training and testing sets, and <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-5.gif"/></noscript></span></span> is the trait’s genomic heritability. This reliability measure is related to the prediction error variance (PEV), and also to the coefficient of determination.</p><p id="p-22">When sample size increases, computation of that reliability measure is difficult to do in practice. A solution to this problem was recently proposed by <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Akdemir <em>et al.</em> (2015)</a>, who proposed including a set of individuals that form the training set (<em>i.e.</em>, individuals for phenotyping and genotyping), and a second set containing the testing sets on which the prediction model is validated. The authors proposed approximating the PEV as the objective function to be minimized by applying an efficient method that uses the first 100 principal components of the marker data. In this study, the proposed method to approximate the PEV was used to form 10% and 20% prediction core sets that will be the training populations of Mexican and Iranian landraces used to predict the remaining accessions.</p></div><div id="sec-7" class="subsection"><h3>Statistical models</h3><p id="p-23">We used the standard GBLUP genomic prediction model for traits evaluated in a single environment, and a G × E model that extended the GBLUP theory to a reaction norm model (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Jarquín <em>et al.</em> 2014</a>) for traits DTH and DTM measured in D and H environments. Phenotypic data were obtained after correcting for the field experimental design, and doing a spatial field adjustment based on the repeated check using the spatial autoregressive model described above.</p><div id="sec-8" class="subsection"><h4>Genomic prediction accuracy accounting and not accounting for population structure:</h4><p id="p-24">Genomic accuracy is affected by population structure and/or linkage disequilibrium (LD) between markers and QTL (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Daetwyler <em>et al.</em> 2015</a>). Population structure affects genetic variation because some individuals are more related than others, or because a sample may contain individuals from different groups with similar allele frequencies within groups but with different reference allele among them at a particular locus. One way to remove this structural variability due to stratified populations would be to include the first few principal components in the analyses as fixed effects; this will decrease genomic prediction accuracy (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Daetwyler <em>et al.</em> 2015</a>).</p><p id="p-25">As expected, in the extensive gene bank collections of landrace accessions included in this study (Mexico and Iranian), there is a population structure. Although it is difficult to decide whether to adjust for population structure, we performed the prediction (i) accounting for population structure, and (ii) not accounting for population structure. To account for population structure, analyses were based on the relative contribution to molecular variance. The plots of the proportion of total variance explained by the eigenvalues of the genomic relationship matrix <strong>G</strong> for Mexican and Iranian landraces shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1, A–H</a> depicted a substantial population structure. The first five and nine eigenvectors for the Mexican and Iranian landraces accounted for about 25% and 30% of the total variance, respectively. We used the first five eigenvectors to adjust phenotypes for population structure. This number of eigenvectors was also suggested by <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gou <em>et al.</em> (2014)</a> in stratified maize and rice populations.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1819/F1/graphic-1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the (A) first vs. second principal component (PC1 vs. PC2) from the marker data for Mexican landraces; (B) first vs. third principal component (PC1 vs. PC3) from the marker data for Mexican landraces; (C) second vs. third principal component (PC2 vs. PC3) from the marker data for Mexican landraces; (D) cumulative variance of the various principal components; (E) first vs. second principal component (PC1 vs. PC2) from the marker data for Iranian landraces; (F) first vs. third principal component (PC1 vs. PC3) from the marker data for Iranian landraces; (G) second vs. third principal component (PC2 vs. PC3) from the marker data for Iranian landraces; (H) cumulative variance of the various principal components." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1477378175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the (A) first &lt;em&gt;vs.&lt;/em&gt; second principal component (PC1 &lt;em&gt;vs.&lt;/em&gt; PC2) from the marker data for Mexican landraces; (B) first &lt;em&gt;vs.&lt;/em&gt; third principal component (PC1 &lt;em&gt;vs.&lt;/em&gt; PC3) from the marker data for Mexican landraces; (C) second &lt;em&gt;vs.&lt;/em&gt; third principal component (PC2 &lt;em&gt;vs.&lt;/em&gt; PC3) from the marker data for Mexican landraces; (D) cumulative variance of the various principal components; (E) first &lt;em&gt;vs.&lt;/em&gt; second principal component (PC1 &lt;em&gt;vs.&lt;/em&gt; PC2) from the marker data for Iranian landraces; (F) first &lt;em&gt;vs.&lt;/em&gt; third principal component (PC1 &lt;em&gt;vs.&lt;/em&gt; PC3) from the marker data for Iranian landraces; (G) second &lt;em&gt;vs.&lt;/em&gt; third principal component (PC2 &lt;em&gt;vs.&lt;/em&gt; PC3) from the marker data for Iranian landraces; (H) cumulative variance of the various principal components.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1819/F1/graphic-1.medium.gif" width="440" height="417"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/7/1819/F1/graphic-1.medium.gif" width="440" height="417"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1819/F1/graphic-1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1819/F1/graphic-1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468685" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1819/F1/graphic-2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the (A) first vs. second principal component (PC1 vs. PC2) from the marker data for Mexican landraces; (B) first vs. third principal component (PC1 vs. PC3) from the marker data for Mexican landraces; (C) second vs. third principal component (PC2 vs. PC3) from the marker data for Mexican landraces; (D) cumulative variance of the various principal components; (E) first vs. second principal component (PC1 vs. PC2) from the marker data for Iranian landraces; (F) first vs. third principal component (PC1 vs. PC3) from the marker data for Iranian landraces; (G) second vs. third principal component (PC2 vs. PC3) from the marker data for Iranian landraces; (H) cumulative variance of the various principal components." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1477378175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the (A) first &lt;em&gt;vs.&lt;/em&gt; second principal component (PC1 &lt;em&gt;vs.&lt;/em&gt; PC2) from the marker data for Mexican landraces; (B) first &lt;em&gt;vs.&lt;/em&gt; third principal component (PC1 &lt;em&gt;vs.&lt;/em&gt; PC3) from the marker data for Mexican landraces; (C) second &lt;em&gt;vs.&lt;/em&gt; third principal component (PC2 &lt;em&gt;vs.&lt;/em&gt; PC3) from the marker data for Mexican landraces; (D) cumulative variance of the various principal components; (E) first &lt;em&gt;vs.&lt;/em&gt; second principal component (PC1 &lt;em&gt;vs.&lt;/em&gt; PC2) from the marker data for Iranian landraces; (F) first &lt;em&gt;vs.&lt;/em&gt; third principal component (PC1 &lt;em&gt;vs.&lt;/em&gt; PC3) from the marker data for Iranian landraces; (G) second &lt;em&gt;vs.&lt;/em&gt; third principal component (PC2 &lt;em&gt;vs.&lt;/em&gt; PC3) from the marker data for Iranian landraces; (H) cumulative variance of the various principal components.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1819/F1/graphic-2.medium.gif" width="440" height="430"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/6/7/1819/F1/graphic-2.medium.gif" width="440" height="430"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1819/F1/graphic-2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1819/F1/graphic-2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468685" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Plot of the (A) first <em>vs.</em> second principal component (PC1 <em>vs.</em> PC2) from the marker data for Mexican landraces; (B) first <em>vs.</em> third principal component (PC1 <em>vs.</em> PC3) from the marker data for Mexican landraces; (C) second <em>vs.</em> third principal component (PC2 <em>vs.</em> PC3) from the marker data for Mexican landraces; (D) cumulative variance of the various principal components; (E) first <em>vs.</em> second principal component (PC1 <em>vs.</em> PC2) from the marker data for Iranian landraces; (F) first <em>vs.</em> third principal component (PC1 <em>vs.</em> PC3) from the marker data for Iranian landraces; (G) second <em>vs.</em> third principal component (PC2 <em>vs.</em> PC3) from the marker data for Iranian landraces; (H) cumulative variance of the various principal components.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">When accounting for population structure by correcting the phenotypic response variables of all the traits by the first five principal components, prediction accuracies decreased by about 15–20% (on average) as compared with analyses that did not account for population structure. Full results of the prediction accuracy of models fitted without correcting for population structure are given in the <em>Appendix</em>.</p></div><div id="sec-9" class="subsection"><h4>GBLUP single-environment model for traits measured in one environment:</h4><p id="p-28">We applied a single-environment model for several traits that were measured in one environment. This model regresses the phenotype vector containing the records for the response variable, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-6.gif"/></noscript></span></span> (where <em>i</em> indexes wheat landraces accessions), on markers using a linear model of the form <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-7.gif"/></noscript></span></span>, (<em>i</em> = 1,2,…,<em>n</em> accessions; <em>k</em> = 1,2,…,<em>p</em> markers) or, in matrix notation,<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-9.gif"/></noscript></span></span> is an intercept, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-10.gif"/></noscript></span></span> is a matrix of marker genotypes, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-11.gif"/></noscript></span></span> is a vector of marker effects and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-12.gif"/></noscript></span></span> is a vector of model residuals. The assumptions of the GBLUP model (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">VanRaden, 2007</a>, <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">2008</a>) are that <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-13.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-14.gif"/></noscript></span></span>. Setting <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-15.gif"/></noscript></span></span> model (1) can be represented as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>with <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-17.gif"/></noscript></span></span>, with the entries of <strong>G</strong> given by <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-18.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-19.gif"/></noscript></span></span> is the estimated allele frequency whose number of copies at the <em>i<sup>th</sup></em> accession is counted in <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-20.gif"/></noscript></span></span>. Centering (<em>i.e.</em>, subtracting <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-21.gif"/></noscript></span></span> from the genotype codes) and standardization (<em>i.e.</em>, dividing by <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-22.gif"/></noscript></span></span> allows interpreting <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-23.gif"/></noscript></span></span> as a genomic variance. As the number of independently segregating loci increases, the entries of the genomic relationship matrix <strong>G</strong> converge to twice the coefficient of parentage (or coancestry) between lines.</p></div><div id="sec-10" class="subsection"><h4>GBLUP G × E models for traits measured in two environments:</h4><p id="p-29">Response variables (DTH and DTM) measured in two environments (D and H) were analyzed by applying a sequence of multiplicative reaction norm models similar to that used by <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Jarquín <em>et al.</em> (2014)</a> with genomic-based relationship matrices and by <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Pérez-Rodríguez <em>et al.</em> (2012)</a> with pedigree-based relationship matrices. Two models (M1 and M2) included only the main effects of environment, accessions and/or genomic information. Models M3 and M4 included the main effects and different types of interactions. Several articles have used slightly different sequences of models for assessing the prediction accuracy of models including main effects and interaction terms (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zhang <em>et al.</em> 2014</a>), depending on the structure of the main effects and interaction terms of interest. A brief description of the models considered in this study is given below.</p><div id="sec-11" class="subsection"><h5>Main effect model 1 (M1):</h5><p id="p-30">This baseline main effect model considers the response of the <em>j</em><sup>th</sup> accession in the <em>i</em><sup>th</sup> environment (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-24.gif"/></noscript></span></span>as a function of a random effect model that accounts for only the effect of the environment (<span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-25.gif"/></noscript></span></span>), the accession (<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-26.gif"/></noscript></span></span> plus a residual (<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-27.gif"/></noscript></span></span>):<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-28.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-29.gif"/></noscript></span></span> is an intercept, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-30.gif"/></noscript></span></span> is the random effect of the <em>i</em><sup>th</sup> environment, <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-31.gif"/></noscript></span></span> is the random effect of the <em>j</em><sup>th</sup> accession, and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-32.gif"/></noscript></span></span> is a model residual. Here <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-33.gif"/></noscript></span></span>stands for a normally distributed random variable, and <em>iid</em> stands for independent and identically distributed. In this model, the effects of the lines are regarded as independent; therefore, there is no borrowing of information between landrace accessions.</p></div><div id="sec-12" class="subsection"><h5>Main effect model 2 (M2):</h5><p id="p-31">The other main effect model adds to model (equation 3) the random effect of the <em>j<sup>th</sup></em> genome <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-34.gif"/></noscript></span></span>, which is an approximation of the true genetic value of the <em>j</em><sup>th</sup> accession (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Jarquín <em>et al.</em> 2014</a>). This approximation is given by the regression on marker covariates <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-35.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-36.gif"/></noscript></span></span> is the genotype of the <em>j</em><sup>th</sup> accession at the <em>k</em><sup>th</sup> marker, and <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-37.gif"/></noscript></span></span> is the effect of the <em>k</em><sup>th</sup> marker with the assumption that <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-38.gif"/></noscript></span></span> (<em>k=</em>1,…,<em>p)</em> and <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-39.gif"/></noscript></span></span> is the variance of the marker effects. The vector <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-40.gif"/></noscript></span></span> contains the genomic values of all the accessions, and is assumed to follow a multivariate normal density with zero mean and covariance matrix <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-41.gif"/></noscript></span></span>, where <strong>G</strong> is the genomic relationship matrix and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-42.gif"/></noscript></span></span> is the genomic variance, which is proportional to <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-43.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-44.gif"/></noscript></span></span>). Therefore, model (3) becomes<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-45.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where the vector of random effects is assumed <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-46.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-47.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-48.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-49.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-50.gif"/></noscript></span></span> The random effects <strong><span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-51.gif"/></noscript></span></span></strong> are correlated such that model (4) allows borrowing information across <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-52.gif"/></noscript></span></span> accessions; thus predicting accession performance in environments where the lines were not observed is possible. As previously mentioned, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-53.gif"/></noscript></span></span> approximates the true genetic values of the <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-54.gif"/></noscript></span></span> accession.</p></div><div id="sec-13" class="subsection"><h5>Main effect and interaction model 3 (M3):</h5><p id="p-32">In this study, a Gaussian process with a covariance function structured based on a reaction norm model is used to model the interaction between markers and environments. <a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Jarquín <em>et al.</em> (2014)</a> showed that the covariance structure is the Hadamard product of two covariance structures, one describing the relationships between lines based on genetic information, and the other relating the environments. Genetic similarity could be based on a genomic relationship matrix or on a pedigree relationship matrix; environmental similarities could be approximated by using environmental covariates (when available) (<a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">Jarquín <em>et al.</em> 2014</a>).</p><p id="p-33">The response of the trait is measured in the <em>j</em><sup>th</sup> accession in the <em>i</em><sup>th</sup> environments (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-55.gif"/></noscript></span></span>) and explained by including the random main effects of model (2), <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-56.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-57.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-58.gif"/></noscript></span></span>, plus the random effects of the interaction between the <em>i</em><sup>th</sup> environment (<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-59.gif"/></noscript></span></span>and the <em>j</em><sup>th</sup> genomic (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-60.gif"/></noscript></span></span>) <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-61.gif"/></noscript></span></span>. This model is obtained by extending model 2 (Equation 4) to introduce a new random effect due to interaction (<span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-62.gif"/></noscript></span></span>). Thus, predictions can be obtained with the model:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-63.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-64.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-65.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-66.gif"/></noscript></span></span> have already been defined, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-67.gif"/></noscript></span></span> is the genome’s interaction with the environment, where <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-68.gif"/></noscript></span></span> is the incidence matrix for the effects of the genetic values of the genotypes, <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-69.gif"/></noscript></span></span> is the variance component of <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-70.gif"/></noscript></span></span> and ‘<span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-71.gif"/></noscript></span></span>’ stands for the Hadamard product between two matrices. Matrix <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-72.gif"/></noscript></span></span> is the incidence matrix for environments. Note that in Equation (5), the interaction term <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-73.gif"/></noscript></span></span> approximates the interaction of the <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-74.gif"/></noscript></span></span>accession with the <em>i</em><sup>th</sup> environment (<span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-75.gif"/></noscript></span></span>).</p><p id="p-34">As discussed in <a id="xref-ref-28-8" class="xref-bibr" href="#ref-28">Jarquín <em>et al.</em> (2014)</a> and <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Pérez-Rodríguez and de los Campos (2014)</a>, the model of Equation (5) uses the covariance patterns induced by a bilinear reaction norm where the intercepts are implicitly accounted for by the main effects of accessions and the environments, while the slopes are implicitly modeled by the interaction term. The intercepts and the slopes are treated as independent.</p></div><div id="sec-14" class="subsection"><h5>Main effect and interaction model 4 (M4):</h5><p id="p-35">Model 4 is similar to model 3 but includes the interaction of the <em>j</em><sup>th</sup> accession with the <em>i</em><sup>th</sup> environment, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-76.gif"/></noscript></span></span>, such that<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-77.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where the term <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-78.gif"/></noscript></span></span> denotes the interaction of the <em>j</em><sup>th</sup> accession in the <em>i</em><sup>th</sup> environment with <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-79.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-80.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-81.gif"/></noscript></span></span> are the incidence matrices for accessions and environments, respectively, and <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-82.gif"/></noscript></span></span> is the variance component of <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-83.gif"/></noscript></span></span>.</p></div></div><div id="sec-15" class="subsection"><h4>Model prediction using random cross-validation:</h4><p id="p-36">The models were fitted in a cross-validation setting to estimate prediction accuracy. For traits evaluated in one environment using the single-environment GBLUP model, we made 30 random cross-validation partitions with 20% of all accessions in the training set (TRN20) and 80% of the accessions in the testing set (TST80).</p><p id="p-37">For traits DTH and DTM measured in two environments (D and H) using the G × E model, we followed <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> (2012)</a> and <a id="xref-ref-28-9" class="xref-bibr" href="#ref-28">Jarquín <em>et al.</em> (2014)</a>, and considered the prediction of the performance of Mexico and Iranian accessions that have not been evaluated in any field trials (CV1). The CV1 partition was obtained by assigning accessions to folds; hence, when the phenotype of an accession is predicted, the corresponding training set contains no record of this accession. Depending on the trait, for the Mexican collections about 7380 (90%) and 6560 (80%) accessions were predicted, and for the Iranian collection about 1800 (90%) and 1600 (80%) accessions were predicted. For the random cross-validation, a TRN20-TST20 scheme was employed with 30 random partitions. For the 10% and 20% diversity and prediction core sets, the prediction was directly computed for the specific core type and size.</p></div><div id="sec-16" class="subsection"><h4>Model prediction using 10% and 20% diversity and prediction reference core sets:</h4><p id="p-38">The other prediction assessment problems we studied were those posed by using 10% and 20% reference core sets as training populations to predict the remaining 90% and 80% of the Mexican and Iranian landraces, respectively, for the core sets based on diversity or core sets based on prediction. For these prediction problems, the correlation reported for traits DTH and DTH measured in D and H environments is that obtained between the predicted values from the four G × E models, and the observed DTH and DTM values in D and H environments and across both environments. For traits measured in a single environment, the reported correlation is that obtained between the predicted values computed using GBLUP trained with 10% and 20% of the two types of reference core sets and the observed values for 90% and 80% of Mexican and Iranian landraces.</p></div></div><div id="sec-17" class="subsection"><h3>Software</h3><p id="p-39">The models described above were all fitted using the BGLR R-package (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Pérez-Rodríguez and de los Campos 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos and Pérez-Rodríguez 2014</a>). This package can handle both molecular marker and pedigree data in parametric and semi-parametric contexts, and allows including different numbers of random effects with user-defined covariance matrices.</p></div><div id="sec-18" class="subsection"><h3>Data availability</h3><p id="p-40">The complete phenotypic and genotypic data sets for Mexican and Iranian landraces can be downloaded from the link: <a href="http://genomics.cimmyt.org/mexican_iranian/traverse/">http://genomics.cimmyt.org/mexican_iranian/traverse/</a>. The Iranian directory contains the following data sets: G.RData, dth_dtm_cores.RDta, standarizedData_dth_dtm.RData, and the standarizedData_univariate.RData. The Mexican directory has one root directory, Toshare, that includes two sub-directories, G×E.Data and Univariate; the G×EData sub-directory has two files, standarizedData_dth_dtm.RData and standarizedData_core.RData, whereas the Univariate sub-directory has two files: Cores and data.Univariate.</p></div></div><div class="section results" id="sec-19"><h2 class="">Results</h2><div id="sec-20" class="subsection"><h3>Traits evaluated in a single environment</h3><div id="sec-21" class="subsection"><h4>Prediction accuracy for random cross-validation (TRN20-TST80):</h4><p id="p-41">Genomic heritabilities of the traits (<span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1819/embed/mml-math-84.gif"/></noscript></span></span> were substantially higher for the Iranian landraces than for the Mexican landraces (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), but this varies depending on the trait. For both collections, the prediction accuracies of the GBLUP model trained in one single optimum environment under random cross-validation TRN20-TST80, and adjusted for population structure, produced correlations of around 0.40-0.65 (except for PHT, 0.166, for the Iranian collection) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). For Mexican landraces, TRN20-TST80 correlations ranged from 0.407 (PHT) to 0.677 (TKW), whereas for Iranian landraces, prediction accuracies for TRN20-TST80 ranged from 0.166 (PHT) to 0.662 (GL). When not accounting for population structure, prediction accuracy increased and ranged from 0.451 (PHT) to 0.767 (TKW) for Mexican accessions and from 0.260 (PHT) to 0.688 (GW) for Iranian accessions (<a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table A1</a>, <em>Appendix</em>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468616/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468616/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Accounting for population structure</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h4>Prediction accuracy for the 10% and 20% diversity and prediction reference core sets:</h4><p id="p-43">The prediction accuracies of the GBLUP model trained with 10% and 20% diversity, and prediction core sets of the Mexican and Iranian collections are shown in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. It is interesting to note that the correlations between observed and predicted values for the 20% core size (either diversity and/or prediction cores) of the Mexican landraces were slightly lower than for the random cross-validation TRN20-TST80 partition of the entire population (an exception was PHT, 20% diversity core). For example, for the Mexican collection, grain yield per square meter (GYSM) was predicted by TRN20-TST80 with an accuracy of 0.460, whereas the diversity and prediction cores sizes predicted GYSM within a range of 0.422 and 0.451 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Trait TKW was predicted by the 10% and 20% diversity and prediction core sets with accuracies ranging from 0.644 to 0.663, whereas partition TR20-TST80 predicted this trait with an accuracy of 0.677. Interestingly, prediction accuracies of the 20% diverse and prediction core sets were only slightly higher than accuracies of the 10% diverse and prediction core sets and very similar to those achieved by random cross-validation TRN20-TST80. Therefore, for the Mexican collections, prediction accuracies from TRS20-TST80, and 10% and 20% diversity and prediction core sets were similar.</p><p id="p-44">Regarding the prediction accuracy of the Iranian reference core sets, the correlations for TRN20-TST80 ranged from relatively high for GL (0.662) to low for PHT (0.116) (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). The prediction accuracy for the 10% diversity core ranged from 0.112 (PHT) to 0.593 (GL), whereas for the 20% diversity core, accuracies ranged from 0.182 (PHT) to 0.647 (GL). However, for the Iranian collection, the 10% and 20% diversity and prediction core sizes tended to have slightly lower prediction accuracy than those observed in the Mexican collection (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>); for some traits, the prediction core gave lower accuracy than the diversity core for both sizes (10% and 20%).</p><p id="p-45">The correlations increased when not accounting for population structure (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table A1</a>, <em>Appendix</em>) in the TRN20-TST80, 10%, and 20% diversity and prediction core subsets for single environment traits. Prediction accuracies increased up to 0.767 for TKW in Mexican landraces, and up to 0.688 for GW. In general, the 10% and 20% diversity core sets achieved prediction accuracies similar to those of the TRN20-TST80 random partition. However, the 10% and 20% prediction cores did not outperform the diversity core sets or the cross-validation TRN20-TST80.</p></div></div><div id="sec-23" class="subsection"><h3>Traits evaluated in drought and heat environments</h3><div id="sec-24" class="subsection"><h4>Descriptive statistics:</h4><p id="p-46">Box-plots of traits DTH and DTM in D and H environments are depicted in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a> for Mexican and Iranian landraces, respectively. Mexican and Iranian landraces in environment D planted in November headed and matured later than the ones in environment H, which were planted in April. For Mexican landraces, the range in DTM-D and DTM-H was large, ranging from 80 days (DTM-H) to 125 days (DTM-D); similar differences were found in Iranian landraces for DTM-D and DTM-H. In both sets of landraces, traits DTH-D and DTH-H tended to have a smaller range of values than DTM-D and DTM-H.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1819/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box-plot of days to heading (DTH) and days to maturity (DTM) measured in drought (D) and heat (H) environments for (A) Mexican landraces and (B) Iranian landraces." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1477378175" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box-plot of days to heading (DTH) and days to maturity (DTM) measured in drought (D) and heat (H) environments for (A) Mexican landraces and (B) Iranian landraces.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1819/F2.medium.gif" width="440" height="171"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/7/1819/F2.medium.gif" width="440" height="171"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1819/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1819/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468573" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-47" class="first-child">Box-plot of days to heading (DTH) and days to maturity (DTM) measured in drought (D) and heat (H) environments for (A) Mexican landraces and (B) Iranian landraces.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h4>Sample phenotypic correlations:</h4><p id="p-48">Sample phenotypic correlations among the four trait-environment combinations for Mexican and Iranian landraces show that the correlations among traits (DTH-D, DTH-H, DTM-D, and DTM-H) for Mexican landraces are high, ranging from 0.5919 (for DTH-D <em>vs.</em> DTH-H) to 0.8276 (for DTH-H <em>vs.</em> DTM-H) (data not shown). For Iranian landraces, sample phenotypic correlations among the four trait-environment combinations are lower than those obtained for Mexican landraces, ranging from 0.3459 (for DTH-D <em>vs.</em> DTM-H) to 0.7939 (for DTH-D <em>vs.</em> DTM-D) (data not shown).</p></div><div id="sec-26" class="subsection"><h4>Variance component estimates:</h4><p id="p-49"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> gives estimates of the variance components of the four models (M1–M4) for the full data sets of Mexican and Iranian landraces, for each of the traits (DTH and DTM) across environments D and H. The variance of the accessions (A) captures the difference between the accessions. The genomic variance (G) accounts for only the genetic effects captured by markers. Similarly, the interaction variance component of G × E (genomic × environment) quantifies the interaction between the genomic effects and the environments.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468697/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468697/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Accounting for population structure in Mexican and Iranian landraces</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">The aim is to study the ability of the different models to predict the performance of the nonphenotyped accession within each environment, after accounting for the odd term environmental mean effect (<a id="xref-ref-28-10" class="xref-bibr" href="#ref-28">Jarquín <em>et al.</em> 2014</a>). Therefore, the proportional contribution of each random effect to within-environment variance is expressed relative to the total variance corrected by the variance due to the main effects of the environments.</p><div id="sec-27" class="subsection"><h5>Mexican landraces:</h5><p id="p-52">The genomic (G) random main effect, the G × E variances, and the residual variance explained the largest proportion of DTH and DTM variances within environments. The G main effect explained up to 28.15% (M4) of the variance for trait DTH, and up to 37.38% (M2) for trait DTM (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The residual variances also explained a relatively large proportion—33.56% for DTH, and 26.56% for DTM (both for M3)—of the within-environment variance for both traits.</p><p id="p-53">The estimated variance due to accession (A) for all traits and models was much lower than the variance associated with G, suggesting that genomic markers are able to capture a sizable proportion of the total variability due to the main effects of accessions. Estimates from models M3 and M4 for DTH show that roughly 26–28% of within-environment variability can be explained by the main effects of markers (G), 24–26% by G × E interaction terms, and 25–34% by residuals (unaccounted factors). For DTM, the variance components of models M3 and M4 show that 33.65–35.01% of within-environment variability can be explained by main effects of markers, 19.58–20.96% by G × E, and 19.63–26.56% by residuals (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). For both traits, the proportion of within-environment variation that is explained by G × E is not negligible and indicates the importance of considering such interactions in models for genomic-enabled prediction.</p></div><div id="sec-28" class="subsection"><h5>Iranian landraces:</h5><p id="p-54">The magnitudes of the G variance component for models M3 and M4 were similar, and explained 19.94% (model M3) of the within-environment variance for DTH and 31.65% (M3) for DTM. The G × E and residual variances also explained a large proportion of the within-environment variances for both traits (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). These results suggest that markers are able to capture a sizable proportion of the variability due to the main effects of accessions. For both traits, the G × E component explained a sizable percentage of the within-environment variance.</p><p id="p-55">Estimates from model M3 for trait DTH in Iranian landraces indicate that roughly 20% of the within-environment variability can be explained by main effects of markers (G), and 42.77% by interaction terms (G × E), with a residual explaining 28.23%. For trait DTM, the variance components of model M3 indicated that 31.65% of the within-environment variability can be explained by main effects of markers, 30% by G × E interaction terms, and 24.71% by residuals (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). Similar to the Mexican landraces, for DTH and DTM, the proportion of within-environment variation that is explained by G × E is not negligible, and indicates the importance of considering such interactions in models for genomic-enabled prediction.</p><p id="p-56">Estimates of variance components for analyses not accounting for population structure are shown in <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table A2</a> (<em>Appendix</em>). The variance components do not change much when not accounting for population structure <em>vs.</em> variance component estimates when accounting for population structure (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>).</p></div></div><div id="sec-29" class="subsection"><h4>Assessing model prediction accuracy by random cross-validation (TRN20-TST80):</h4><div id="sec-30" class="subsection"><h5>Prediction of Mexican landraces across environments:</h5><p id="p-57">The average GS accuracy of the four models for TRS20-TSN80 for both traits clearly shows that models M3 and M4 gave the highest accuracy across both environments (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>), where prediction accuracies of models M3 and M4 were almost equal for traits DTH and DTM (<em>i.e.</em>, 0.59–0.60). This was expected due to the small variance component of A and A × E as compared with the variances of G and G × E (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>). When not accounting for population structure, prediction accuracies of M3 and M4 models reached 0.74–0.76 for DTHA and DTM for TRN20-TST80 (<a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table A3</a>, <em>Appendix</em>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468755/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468755/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Accounting for population structure in Mexican and Iranian landraces</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-31" class="subsection"><h5>Prediction of Iranian landraces across environments:</h5><p id="p-60">The average GS accuracy of the four models for cross-validations TRN20-TST80 for both traits across environments are reported in <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>. Results show that, in general, correlation values were lower than those achieved for Mexican landraces for all trait-environment combinations. As with the Mexican landraces, models M3 and M4 gave the highest accuracy for both traits across both environments. Prediction accuracies for traits DTM and DTH were similar. Models M3 and M4 reached up to 0.551 and 0.548 prediction accuracy, respectively, for DTM and DTM under partition TRN20-TST80. Without accounting for population structure, prediction accuracies of M3 and M4 models reached 0.58–0.60 for DTH and DTM under cross-validation TRN20-TST80 (<a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table A3</a>, <em>Appendix</em>).</p></div></div><div id="sec-32" class="subsection"><h4>Assessing model prediction accuracy by 10% and 20% diversity and prediction core sets:</h4><div id="sec-33" class="subsection"><h5>Mexican landraces across environments:</h5><p id="p-61">Models M3 and M4 gave the best prediction accuracy across environments for DTH and DTM for the 10% and 20% diversity and prediction reference core sets. When the 10% and 20% diversity and prediction core sets of Mexican landraces were used to predict the remaining 90% and 80% of the Mexican accessions, correlations using models M3 and M4 for DTH and DTM ranged from 0.555 (10% diversity core DTH) to 0.584 (20% prediction core DTM) (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). The accuracy obtained using the diversity core sets and the prediction core sets were very similar.</p><p id="p-62">On average across environments, there is a slight decrease in prediction accuracy (about 4%) when the size of the core decreases from 20 to 10%. Also, the prediction accuracy of the 20% diversity and prediction core sets for M3 and M4 (0.58) were similar to the prediction accuracies obtained from random cross-validation TRN20-TST80 across environments (around 0.596). These results confirmed that the 20% diverse and prediction core sizes produced good accuracy when predicting the remaining 80% of Mexican landraces. For the Mexican landraces, a clear increase in prediction accuracy of models M3 and M4 was achieved when not accounting for population structure, with prediction accuracies ranging from 0.717 to 0.758 for both sizes of diversity and prediction cores (<a id="xref-table-wrap-8-3" class="xref-table" href="#T8">Table A3</a>, <em>Appendix</em>).</p></div><div id="sec-34" class="subsection"><h5>Iranian landraces across environments:</h5><p id="p-63">Models M3 and M4 had the highest prediction accuracies in all cases, <em>i.e.</em>, TRN20-TST80, 10% and 20% diversity and prediction core sets (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>). For the 10% prediction core set, model M3 gave correlations of 0.5111 and 0.5021 for DTH and DTM, respectively, whereas for the 20% prediction core set, these correlations were 0.5446 and 0.5194, respectively. Prediction accuracies of the 10% and 20% diversity and prediction core sets were similar to each other and to those achieved by TRN20-TST80.</p><p id="p-64">The prediction accuracies of the 20% diversity core set of Iranian landraces observed in <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a> for models M3 and M4 are at a similar level (0.55) as the prediction accuracies obtained for TRN20-TST80. Those obtained for the 20% prediction core were slightly lower than those of the diversity core. These results indicate that the diversity and prediction core subsets of 20% size are good predictors of the remaining Iranian accessions, and could be useful for incorporating genomic prediction when introgressing exotic germplasm into elite adapted germplasm. For Iranian accessions, the analysis not accounting for population structure gave superior accuracies of models M3 and M4 over those used when accounting for population structure (<a id="xref-table-wrap-8-4" class="xref-table" href="#T8">Table A3</a>, <em>Appendix</em>) and the 10% and 20% predictive core sets were as good as the 10% and 20% diversity core sets at predicting the testing set. Also, the differences in prediction accuracy between the 10% and 20% diversity and prediction core sets were small.</p></div></div></div></div><div class="section discussion" id="sec-35"><h2 class="">Discussion</h2><p id="p-65">In this research, we studied genome-enabled prediction accuracy in two different collections of gene bank accessions, one Mexican and one Iranian. Prediction accuracy obtained without accounting for population structure was always higher (15–20%) than prediction accuracy obtained when accounting for population structure. However, in both cases, the prediction accuracy for highly heritable traits in wheat gene bank accessions ranged from intermediate to high. These results may stimulate the application of genomic prediction in gene bank evaluation and germplasm enhancement programs to speed up the process of introgressing diversity into elite germplasm. In general, results of this study are in agreement with the simulation study by <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Gorjanc <em>et al.</em> (2016)</a> of maize genetic resources, in the sense that prebreeding can start directly from the large genetic diversity in gene bank landraces. Furthermore, the relatively high accuracies obtained from the 10% and 20% core sets (diversity and prediction) indicate that the methods used to form these core sets resulted in reference training sets that are representative of the whole collection, and can predict the remaining population fairly accurately.</p><div id="sec-36" class="subsection"><h3>Prediction accuracy of the random cross-validation scheme <em>vs.</em> the diverse core set and the prediction core set</h3><p id="p-66">The TRN20:TRT80 analyses represent a random selection of lines for the training population. This set always produced higher accuracy than did the diversity and prediction core sets (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a> and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). For traits measured in one environment, when accounting for population structure, the prediction accuracy of the 20% diversity and 20% prediction core sets were, on average, about 1.2–2.8% lower for Mexican and Iranian landraces, and almost 8.7% lower for the 20% prediction core of the Iranian collection when compared to the results of TRN20:TRT80 (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). On average, the decrease in prediction accuracy of the 10% prediction core set <em>vs.</em> TRN20-TST80 was greater (10.3% and 14.9%) than the percent change in accuracy of the 20% diversity for Mexican and Iranian collections (2.7% and 2.8%). The percent change in accuracy of the 20% prediction core <em>vs.</em> TRN20-TST80 was lower than the percent change in accuracy of the 20% diversity core for the Mexican collection.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468662/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468662/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Average percent change in prediction accuracy of 10% and 20% diversity and prediction cores <em>vs.</em> prediction accuracy of random cross-validation TRN20-TST80 (first four columns) and percent change in prediction accuracy between 10% diversity core <em>vs.</em> 10% prediction core, and between 20% diversity core <em>vs.</em> 20% prediction core for traits measured in one or in two environments for Mexican and Iranian collections</span><div class="sb-div caption-clear"></div></div></div><p id="p-68">For traits measured in two environments (DTH and DTM), in four cases, the 10% and 20% prediction cores had a smaller reduction in their correlations with respect to the TRN20-TST80 set, than those observed for the 10% and 20% diversity core sets for the Mexican and Iranian collections (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>) (6.5 <em>vs.</em> 7.5 7.6 <em>vs.</em> 12.1 5.11 <em>vs.</em> 8.30 1.2 <em>vs.</em> 6.5).</p><p id="p-69">The main advantage of the 10% and 20% diversity and prediction core sets over the random cross-validation is that they generate in one time a good prediction training set. The random cross-validation partitions explore and sample the entire correlation space between observed and predicted values (<em>i.e.</em>, some of these correlations are low or even negative, indicating a low training-testing relationship, while others are high, indicating a close relationship between training-testing sets), while the core sets focus directly on the subset that will produce high correlations between training-testing sets.</p><p id="p-70">As discussed above, the two sizes (20% and 10%) of the Mexican and Iranian landrace core sets gave different prediction accuracies, with the 20% core set being slightly better than the 10% core set. These results confirm the hypothesis that phenotyping and genotyping a small number of accessions and genotyping the entire population can give relatively high prediction accuracies, thereby prompting the application of GS to predict the trait value gene bank accessions for traits with relatively high heritability. The results of this study indicate that one could phenotypically evaluate only 20% of germplasm bank accessions using the reference core set of accessions for GS model training and then the values of the rest of the collection. The 20% core sets of the Mexican landraces were still large (<em>n</em> &gt; 1600), while the 20% core sets of the Iranian landraces were relatively small (<em>n</em> &gt; 400). However, genomic prediction of traits with low heritability will likely require using training sets of greater size, and prediction accuracy would not be expected to reach values as high as those found in this study for highly heritable traits.</p><p id="p-71">As breeding programs begin to implement GS, maintaining genetic variance will be increasingly important because GS has been shown to lead to faster losses of genetic variance compared to phenotypic selection (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jannink 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rutkoski <em>et al.</em> 2015</a>). Although starting with high genetic variance will not prevent its rapid loss, it will allow starting the breeding process at higher levels of allelic diversity and genetic variance while ensuring good prediction accuracies.</p><p id="p-72">Breeders can use predicted values developed from the core set to identify a diverse set of accessions with desired values, and use them to start a prebreeding project. Breeders will likely select only lines with the desired trait value based on phenotypes or GEBVs to initiate a prebreeding program. For example, breeders may only select lines with the highest yield (actual or predicted). It is possible that the GS model built on the entire phenotypically and genetically diverse core set will not predict the value of the progeny from a more restricted set of selected parents. The GS model may need to be retrained with this subset of parents.</p></div><div id="sec-37" class="subsection"><h3>Prediction accuracy of the diverse core sets <em>vs.</em> the prediction core sets</h3><p id="p-73">For genomic selection (GS), a good training set of individuals to be genotyped and phenotyped is crucial for predicting the value of candidates in the testing set. Therefore, designing a core training population that both maximizes genetic diversity and increases the accuracy of GS models is a key issue as phenotyping becomes more expensive. The method used in this study to form the diversity core was developed by <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Franco <em>et al.</em> (2006)</a> using all the markers, and with the objective of maximizing allele diversity. The method proposed by <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Akdemir <em>et al.</em> (2015)</a> to approximate the prediction error variance (PEV) as a reliability measure based solely on the first 100 principal components of the marker data was used to select the prediction core sets. Our results indicated very similar prediction accuracies were obtained when using the diversity <em>vs.</em> the prediction core set (<a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>). Better prediction accuracy of the 10% prediction core over the 10% diversity core was clear for the Iranian collection for the traits across two environments when accounting for population structure (–5.3), than when not accounting for population structure (–3.48). The 20% diversity core was superior to the 20% prediction core for traits in one environment (5.9 and 19.5). For the rest of the comparisons, differences in prediction accuracy between the diversity and prediction core set were negligible, ranging from –1.3 to 2.4. These results indicated similar prediction accuracies for both kinds of cores, regardless of their size.</p></div><div id="sec-38" class="subsection"><h3>Diversity of the diversity and prediction core sets</h3><p id="p-74"><a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Franco <em>et al.</em> (2006)</a> described and applied three diversity indices to compare the diversity of different core sets formed using markers: the Shannon diversity index, the expected proportion of heterozygous loci, and the number of effective alleles. We applied these indices on the diversity and prediction core sets to compare their genetic diversity.</p><p id="p-75">In terms of diversity measured through the Shannon Diversity Index, the expected heterozygosity (with two alleles, the maximum is 0.5) and the number of effective alleles (the maximum value is 2 for two alleles), the two types of cores gave very similar values. For example, for the Mexican collection, the expected heterozygosity was around 0.23 for the diversity cores of both sizes, and 0.22 for the prediction cores of both sizes; the number of effective alleles for both core types of collections was around 1.4, and the Shannon Diversity Index was 0.48 for the diversity core and 0.49 for the prediction core. For the Iranian collections, the expected heterozygosity was around 0.19 for both types of core sets, and the number of effective alleles was 1.30, with a Shannon Diversity Index of around 0.5. These results indicated that forming diversity and/or prediction cores gave stability to the values of the genetic distances among accessions, and thus more diverse alleles were collected in both types of cores.</p></div><div id="sec-39" class="subsection"><h3>Genomic prediction under population structure</h3><p id="p-76">In general, when accounting for population structure, genomic prediction accuracy decreases. In animal and human populations, fitting the first few initial principal components from analysis of marker data to adjust phenotypic values is a common practice for accounting for population structure. <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Daetwyler <em>et al.</em> (2015)</a> pointed out that accounting for spurious population structure (such as that originated from admixtures without affecting relatedness) is required; however, they recognized that this is not easy. Also in breeding populations, high prediction accuracy for small training populations can be caused by population structure. <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Sallam <em>et al.</em> (2015)</a> argued that structure in training populations of great size could also reduce genomic prediction accuracy.</p><p id="p-77">In general, population structure is important when assessing genomic prediction in stratified populations (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Windhausen <em>et al.</em> 2012</a>), especially those comprised of gene bank landrace accessions. The population structure of these landrace collections is expected to be stronger than what can be found in a standard breeding program. The first five principal components accounted for about 25% of the global molecular variance in each landrace set and were used to adjust the phenotypic data for population structure. This seems to be an acceptable number of principal components, and was also suggested by <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Gou <em>et al.</em> (2014)</a>. Five principal components is a reasonable number to account for population structure because using an excessive number of principal components may remove useful genetic relationships between individuals in the training and testing sets (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Daetwyler <em>et al.</em> 2015</a>).</p><p id="p-78">Also, it should be mentioned that GBLUP models capture population structures and substructures within and between families because regressing phenotypic values on all marker values is equivalent to regressing the phenotypes on all principal components derived from molecular data (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">de los Campos <em>et al.</em> 2010</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Janss <em>et al.</em> 2012</a>). The different degrees of genetic similarities and dissimilarities between landrace accessions could also be the result of a combination of different factors such as population structure, substructure, and subtle complex relationships due to additive × additive interaction (epistasis); these factors are captured by marker differences and consequently accounted for by genomic regression GBLUP models (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> 2013</a>).</p><p id="p-79">In a recent article, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos <em>et al.</em> (2015)</a> developed quantitative genetic models that account for stratified genetic populations. Rather than dealing with stratification as a confounding effect and thus developing methods that correct for stratification, such as using marker-derived principal components as fixed covariables, these authors proposed that the approach, from a quantitative genetic perspective, should regard population structure as a modifier effect (not as a confounding effect). They argued that differences in allele frequency between marker-derived groups (population structure) induce heterogeneity of allele substitution effects at different loci. Also, different linkage disequilibrium between markers and QTL may cause heterogeneity of marker effects (population structure).</p></div><div id="sec-40" class="subsection"><h3>Germplasm enhancement of wheat: the way forward</h3><p id="p-80">Our results show that for the two populations of landraces included in this study, genomic predictions were generally of a magnitude that could be very useful for predicting the value of other accessions in the gene bank, and that could be useful in breeding. This occurred despite pronounced population structure and G × E. The first application of this approach would be to predict the value of all genotyped accessions in a gene bank, and then phenotype those that have the highest predicted value to verify their value in breeding. Once their value has been verified, a breeder could begin prebreeding following several strategies. A major decision would be whether to initiate a prebreeding population by crossing among the accessions themselves or whether to cross the chosen accessions to elite materials. The former is a conversion approach (improve the value of exotic germplasm until it becomes elite), while the latter is an introgression approach. The genetic values modeled within the landrace populations are only relevant within those populations, and thus would only be useful in a conversion strategy. Using an introgression strategy will require crossing the best accessions to elite materials, and developing a new model of the genetic effects that would be used to predict the value of future progeny. <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Gorjanc <em>et al.</em> (2016)</a> compared these strategies after cycles of simulated selection and found that crossing only among landraces (conversion) maintained the most diversity (relative to the elite gene base), but produced a low rate of improvement of genetic merit (relative to the elite base). In contrast, the introgression strategy (crossing landraces with elites followed by selection) retained less diversity after cycles of selection but produced greater genetic merit relative to the elite base, and a greater rate of gain. Their results did show that genomic selection was effective at improving genetic merit with either strategy.</p><p id="p-81">Further research is required to clarify several aspects of using genomic selection in prebreeding. Work is underway at CIMMYT to do multiple introgressions of a large number of landraces into elite materials with a large number of parents and progenies that have been initially genotyped and phenotyped. This will provide excellent populations to explore the use of genomic prediction for capturing the value present in gene banks.</p></div><div id="sec-41" class="subsection"><h3>Conclusions</h3><p id="p-82">Results of this study indicate that genomic-enabled prediction of large wheat gene bank collections comprising Mexican and Iranian landraces for highly heritable traits may be a valuable tool for germplasm enhancement. Prediction accuracies based on random cross-validation partitions of traits related to wheat grain quality measured in one environment were intermediate to relatively high (from 0.4 to 0.6) for Mexican (7 traits) and Iranian (8 traits) landraces. Reference diversity and prediction core sets of sizes 10% and 20% of the total number of accessions gave good prediction accuracy, especially cores of size 20% that achieved similar prediction accuracies as those obtained by random cross-validation set (TRT20-TST80). Core sets of 10% had a substantial decrease in prediction accuracy (up to 16%) as compared with the prediction accuracy of the 20% core sets.</p><p id="p-83">Traits DTH and DTM were evaluated in drought and heat environments. Genomic models including genomic × environment (G × E) interaction gave substantial and consistent increases in prediction accuracy over the main effect models though the accession × environment interaction did not account for much variability. For Mexican landraces, models including G × E had the highest prediction accuracies, reaching correlations of 0.59–0.60 for both traits when 80% or 90% of the total number of accessions were predicted across environments when accounting for population structure. Iranian landraces had similar but slightly lower prediction accuracies. When not accounting for population structure, traits DTH and DTM for a large number of Mexican accessions were predicted with accuracies of 0.7–0.75.</p><p id="p-84">This study used extensive wheat landrace data stored in the CIMMYT gene bank, and has shown promising results in terms of prediction accuracies of highly heritable traits that should stimulate further research on utilizing gene bank accessions with high-density markers for future application of genomic prediction in germplasm enhancement programs.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-85">We would like to thank Dr. Ky Mathew, who developed the experimental field designs of the trials analyzed in this study. We would also like to thank Drs. Jessica Rutkoski and Kevin Pixley for their critical reviews and helpful suggestions on an early version of the manuscript, as well as Dr. Gustavo de los Campos for his valuable advice on several aspects of the statistical analyses. This research was supported by the Seeds of Discovery (SeeD) project of MasAgro, a project funded by Mexico’s Secretariat of Agriculture, Livestock, Rural Development, Fisheries and Food (SAGARPA).</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-42"><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468723/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468723/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A1</span> <span class="caption-title">Not accounting for population structure</span><div class="sb-div caption-clear"></div></div></div><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468444/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468444/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A2</span> <span class="caption-title">Not accounting for population structure in Mexican and Iranian landraces</span><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468515/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468515/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A3</span> <span class="caption-title">Not accounting for population structure in Mexican and Iranian landraces</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-86"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-01-21"><span class="received-label">Received </span>January 21, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-04-15"><span class="accepted-label">Accepted </span>April 15, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Crossa <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.1" data-doi="10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Optimization of genomic selection training populations with a genetic algorithm.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">38</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711–015–0116–6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711%2526ndash%253B015%2526ndash%253B0116%2526ndash%253B6%26rft_id%253Dinfo%253Apmid%252F25943105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711&amp;ndash;015&amp;ndash;0116&amp;ndash;6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25943105&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.2" data-doi="10.1007/s10681-012-0773-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ayala</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guzman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peña</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Characterization of genetic diversity of puroindoline genes in Mexican wheat landraces.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D190%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-012-0773-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-012-0773-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000316015600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.3" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.4" data-doi="10.1139/g89-144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Core collections: a practical approach to genetic resource management.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">818</span>–<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D31%26rft.spage%253D818%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg89-144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g89-144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.5" data-doi="10.1007/s10722-009-9501-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caballero</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peña</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Characterization of Mexican Creole wheat landraces in relation to morphological characteristics and HMW glutenin subunit composition.</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D57%26rft.spage%253D657%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-009-9501-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-009-9501-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.7.1819.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uszynska</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaccoud</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Machado</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hopper</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> (<span class="cit-pub-date">2015</span>) DArT and DArTseq genome profiling for breeding, pre-breeding and population genetics. P0052. <em>Plant and Animal Genome Conference</em> XXIII.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.7" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>  <span class="cit-name-given-names">S</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2013</span> Genomic prediction in maize breeding populations with genotyping-by-sequencing. G3 (Bethesda) 3(11): 1903–1926.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.9" data-doi="10.2527/jas2011&amp;ndash;4557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemper</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Components of the accuracy of genomic prediction in a multi-breed sheep population.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">3375</span>–<span class="cit-lpage">3384</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas2011–4557</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft.volume%253D90%26rft.spage%253D3375%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas2011%2526ndash%253B4557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas2011&amp;ndash;4557&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-web" id="cit-6.7.1819.10"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2014 Bayesian generalized linear regression. R package version 1.0.4. Available at: <a href="https://r-forge.r-project.org/R/?group_id=1525">https://r-forge.r-project.org/R/?group_id=1525</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.11" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.12" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.13" data-doi="10.1007/s13253-015-0222-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veturi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span></li></ol><cite> C. Lehermeier, and P. Pérez-Rodríguez, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Incorporating genetic heterogeneity in whole-genome regressions using interactions.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D20%26rft.spage%253D467%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13253-015-0222-5%26rft_id%253Dinfo%253Apmid%252F26660276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13253-015-0222-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26660276&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.14" data-doi="10.2135/cropsci2004.0292"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taba</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shands</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Sampling strategy for conserving genetic diversity when forming core subsets.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1035</span>–<span class="cit-lpage">1044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DFranco%26rft.auinit1%253DJ.%26rft.volume%253D45%26rft.issue%253D3%26rft.spage%253D1035%26rft.epage%253D1044%26rft.atitle%253DA%2BSampling%2BStrategy%2Bfor%2BConserving%2BGenetic%2BDiversity%2Bwhen%2BForming%2BCore%2BSubsets%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0292&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000229105700030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.15" data-doi="10.2135/cropsci2005.07-0201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warburton</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taba</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Sampling strategies for conserving maize diversity when forming core subsets using genetic markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">854</span>–<span class="cit-lpage">864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DFranco%26rft.auinit1%253DJ.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D854%26rft.epage%253D864%26rft.atitle%253DSampling%2BStrategies%2Bfor%2BConserving%2BMaize%2BDiversity%2BWhen%2BForming%2BCore%2BSubsets%2BUsing%2BGenetic%2BMarkers%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.07-0201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.07-0201&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000235991100041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.16" data-doi="10.2135/cropsci2009.01.0053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desphande</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Hierarchical multiple-factor analysis for classifying genotypes based on phenotypic and genetic data.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.01.0053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.01.0053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-6.7.1819.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frankel</span>  <span class="cit-name-given-names">O. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">A. H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Plant genetic resources today: a critical appraisal</span>, pp. <span class="cit-fpage">249</span>–<span class="cit-lpage">257</span> in <span class="cit-source">Crop genetic resources: Conservation and evaluation</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Holden</span>  <span class="cit-name-given-names">J. H. W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">J. T.</span></span>. <span class="cit-publ-name">George Allen and Unwin</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.18" data-doi="10.2307/1400446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Accounting for natural and extraneous variation in the analysis of field experiments.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">269</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D2%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1400446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1400446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-web" id="cit-6.7.1819.19"><div class="cit-metadata unstructured">Gilmour, A. R., B. J. Gogel, B. R. Cullis, and R. Thompson, 2009 ASReml User Guide Release 3.0 VSN International Ltd, Hemel Hempstead, HP1 1ES, UK. Available at: <a href="http://www.vsni.co.uk/downloads/asreml/release3/UserGuide.pdf">www.vsni.co.uk/downloads/asreml/release3/UserGuide.pdf</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.20" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, J. E. Cairns <span class="cit-etal">et al.</span> <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.21" data-doi="10.1186/s12864-015-2345-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jenk</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hearne</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Initiating maize pre-breeding programs using genomic selection to harness polygenic variation from landrace populations.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2345-z%26rft_id%253Dinfo%253Apmid%252F26732811%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2345-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26732811&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.22" data-doi="10.1007/s00122-013-2255-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, H. Gandhi, E. Ersoz <span class="cit-etal">et al.</span> <span class="cit-pub-date">2014</span> <span class="cit-article-title">The impact of population structure on genomic prediction in stratified populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">749</span>–<span class="cit-lpage">762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D749%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2255-x%26rft_id%253Dinfo%253Apmid%252F24452438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2255-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24452438&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.23" data-doi="10.1016/j.jcs.2014.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guzman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medina-Larque</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Santoyo</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2014</span> <span class="cit-article-title">Use of wheat genetic resources to develop biofortified wheat with enhanced grain zinc and iron concentrations and desirable processing quality.</span> <abbr class="cit-jnl-abbrev">J. Cereal Sci.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCereal%2BSci.%26rft.volume%253D60%26rft.spage%253D617%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcs.2014.07.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jcs.2014.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.24" data-doi="10.1016/j.jcs.2014.11.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guzman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortega</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamori</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peña</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Molecular characterization of two novel null waxy alleles in Mexican bread wheat landraces.</span> <abbr class="cit-jnl-abbrev">J. Cereal Sci.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">8</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCereal%2BSci.%26rft.volume%253D62%26rft.spage%253D8%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jcs.2014.11.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jcs.2014.11.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.25" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.26" data-doi="10.1186/1297-9686-42-35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamics of long-term genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DJannink%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D42%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DDynamics%2Bof%2Blong-term%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-35%26rft_id%253Dinfo%253Apmid%252F20712894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20712894&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.27" data-doi="10.1534/genetics.112.141143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheehan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Inferences from genomic models in stratified populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">693</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141143%26rft_id%253Dinfo%253Apmid%252F22813891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNjkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.28" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheyron</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrique</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, and G. <span class="cit-auth"><span class="cit-name-surname">de los Campos</span></span> (<span class="cit-pub-date">2015</span>) Increased prediction accuracy in wheat breeding trials using a marker × environment interaction genomic selection model. G3 (Bethesda) 5(4):569–582.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTgxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.31" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Pérez-Rodríguez</span></li></ol><cite> P., and G. de los Campos, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manes</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, and S. <span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span></span><span class="cit-pub-date">2012</span> Comparison between linear and non-parametric models for genome-enabled prediction in wheat. G3 (Bethesda) <strong>2</strong>:1595–1605.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-6.7.1819.33"><div class="cit-metadata unstructured">R Core Team, 2016 R: A language and environment for statistical computing. Vienna, Austria: R Foundation for Statistical Computing. Available at: <a href="http://www.R-project.org">http://www.R-project.org</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.34" data-doi="10.2135/cropsci2004.0517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic and mathematical properties of similarity and dissimilarity coefficients applied in plant breeding and seed bank management.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DReif%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D45%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D7%26rft.atitle%253DGenetical%2Band%2BMathematical%2BProperties%2Bof%2BSimilarity%2Band%2BDissimilarity%2BCoefficients%2BApplied%2Bin%2BPlant%2BBreeding%2Band%2BSeed%2BBank%2BManagement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.0517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.0517&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000226435300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhavani</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic gain from phenotypic and genomic selection for quantitative resistance to stem rust of wheat.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> (in press).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.36" data-doi="10.3835/plantgenome2014.05.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sallam</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assessing genomic selection prediction accuracy in a dynamic barley breeding population.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.05.0020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.7.1819.37"><div class="cit-metadata unstructured">Sansaloni, C., C. Petroli, D. Jaccoud, J. Carling, F. Detering <em>et al.</em> 2011 Diversity Arrays Technology (DArT) and next-generation sequencing combined: genome-wide, high-throughput, highly informative genotyping for molecular breeding of Eucalyptus. BMC Proceedings 5(Suppl 7): P54.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.38" data-doi="10.1111/pbr.12105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogutu</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordillo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karaman</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knaak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection allowing for marker-by-environment interaction.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">132</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">532</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D132%26rft.spage%253D532%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic measures of relationship and inbreeding.</span> <abbr class="cit-jnl-abbrev">Interbull Bull</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterbull%2BBull%26rft.volume%253D37%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.40" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1819.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vikram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franco</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueno-Ferrera</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sehgal</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2016</span> <span class="cit-article-title">Unlocking the genetic diversity of Creole wheats.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> 6: 23092.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.42" data-doi="10.2307/2282967"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J. H.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">Hierarchical grouping to optimize an objective function.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">236</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D58%26rft.spage%253D236%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2282967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2282967&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1963P102700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grudloyma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1819.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in biparental tropical maize populations in water-stressed and well-watered environments using low-density and GBS.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> 114: 291–299.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/7/1819.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1809" title="Fine Mapping of the Barley Chromosome 6H Net Form Net Blotch Susceptibility Locus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1835" title="An Improved Genome Assembly of Azadirachta indica A. Juss." rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/7"><span>Volume 6 Issue 7, July 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/7.cover-source.jpg?itok=E84OOyGI" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1819?ijkey=9dbcbbdd3f62da788d54f71ddf03dfdf91f61712&amp;keytype2=tf_ipsecsha" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FvfeNaxxUxUBkEK5OaSCud46QF3GH1JCViLhz5Pj0t0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/7/1819" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/7/1819" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/7/1819" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction of Gene Bank Wheat Landraces</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460418" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-iQr7WUEzkIZGpcZe2C_0YJhkLyWFp5GW-8EI6GB2Ibk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1819.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/7/1819" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-a3SE6ymwdwmU10KWYXkYl1PLJOmU87Oh_yAIILo1l6s" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/7/1819" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1819.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460418" id="node460418--2" data-pisa="ggg;6/7/1819" data-pisa-master="ggg;g3.116.029637" data-apath="/ggg/6/7/1819.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Gene Bank Wheat Landraces</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="1">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="2">Jorge Franco</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="5">Carolina Saint-Pierre</span>, <span class="highwire-citation-author" data-delta="6">Prashant Vikram</span>, <span class="highwire-citation-author" data-delta="7">Carolina Sansaloni</span>, <span class="highwire-citation-author" data-delta="8">Cesar Petroli</span>, <span class="highwire-citation-author" data-delta="9">Deniz Akdemir</span>, <span class="highwire-citation-author" data-delta="10">Clay Sneller</span>, <span class="highwire-citation-author" data-delta="11">Matthew Reynolds</span>, <span class="highwire-citation-author" data-delta="12">Maria Tattaris</span>, <span class="highwire-citation-author" data-delta="13">Thomas Payne</span>, <span class="highwire-citation-author" data-delta="14">Carlos Guzman</span>, <span class="highwire-citation-author" data-delta="15">Roberto J. Peña</span>, <span class="highwire-citation-author" data-delta="16">Peter Wenzl</span> and <span class="highwire-citation-author" data-delta="17">Sukhwinder Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1819-1834;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029637 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460418--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:suk.singh@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">suk.singh@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, 321 Keim Hall, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jorge Franco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Biometría, Estadística y Computación, Facultad de Agronomía, Universidad de la República (Udelar), Paysandú, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jorge%2BFranco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Franco%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJorge%2BFranco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Post-Graduados, Montecillos, Edo. de Mexico, 56230 Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carolina Saint-Pierre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BSaint-Pierre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saint-Pierre%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BSaint-Pierre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Prashant Vikram </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Prashant%2BVikram%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vikram%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrashant%2BVikram%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Carolina Sansaloni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BSansaloni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sansaloni%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BSansaloni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Cesar Petroli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cesar%2BPetroli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petroli%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACesar%2BPetroli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Deniz Akdemir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Breeding & Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deniz%2BAkdemir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akdemir%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADeniz%2BAkdemir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Clay Sneller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Horticulture and Crop Science, Ohio State University, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clay%2BSneller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sneller%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClay%2BSneller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Matthew Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Maria Tattaris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BTattaris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tattaris%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BTattaris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Thomas Payne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BPayne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payne%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BPayne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Carlos Guzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BGuzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guzman%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BGuzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Roberto J. Peña </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BJ.%2BPe%C3%B1a%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pe%C3%B1a%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoberto%2BJ.%2BPe%25C3%25B1a%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Peter Wenzl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BWenzl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wenzl%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BWenzl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Sukhwinder Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sukhwinder%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASukhwinder%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:suk.singh@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">suk.singh@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/7/1819" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/7/1819&amp;title=Genomic%20Prediction%20of%20Gene%20Bank%20Wheat%20Landraces" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/7/1819&amp;title=Genomic%20Prediction%20of%20Gene%20Bank%20Wheat%20Landraces" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/7/1819&amp;title=Genomic%20Prediction%20of%20Gene%20Bank%20Wheat%20Landraces" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/7/1819&amp;text=Genomic%20Prediction%20of%20Gene%20Bank%20Wheat%20Landraces" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/7/1819&amp;title=Genomic%20Prediction%20of%20Gene%20Bank%20Wheat%20Landraces" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/7/1819&amp;t=Genomic%20Prediction%20of%20Gene%20Bank%20Wheat%20Landraces" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/7/1819&amp;title=Genomic%20Prediction%20of%20Gene%20Bank%20Wheat%20Landraces" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/7/1819&amp;title=Genomic%20Prediction%20of%20Gene%20Bank%20Wheat%20Landraces" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460418/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460418" id="node460418--3" data-pisa="ggg;6/7/1819" data-pisa-master="ggg;g3.116.029637" data-apath="/ggg/6/7/1819.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Gene Bank Wheat Landraces</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="1">Diego Jarquín</span>, <span class="highwire-citation-author" data-delta="2">Jorge Franco</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="5">Carolina Saint-Pierre</span>, <span class="highwire-citation-author" data-delta="6">Prashant Vikram</span>, <span class="highwire-citation-author" data-delta="7">Carolina Sansaloni</span>, <span class="highwire-citation-author" data-delta="8">Cesar Petroli</span>, <span class="highwire-citation-author" data-delta="9">Deniz Akdemir</span>, <span class="highwire-citation-author" data-delta="10">Clay Sneller</span>, <span class="highwire-citation-author" data-delta="11">Matthew Reynolds</span>, <span class="highwire-citation-author" data-delta="12">Maria Tattaris</span>, <span class="highwire-citation-author" data-delta="13">Thomas Payne</span>, <span class="highwire-citation-author" data-delta="14">Carlos Guzman</span>, <span class="highwire-citation-author" data-delta="15">Roberto J. Peña</span>, <span class="highwire-citation-author" data-delta="16">Peter Wenzl</span> and <span class="highwire-citation-author" data-delta="17">Sukhwinder Singh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1819-1834;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029637 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460418--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:suk.singh@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">suk.singh@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Diego Jarquín </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agronomy and Horticulture, University of Nebraska-Lincoln, 321 Keim Hall, Lincoln, Nebraska 68583-0915</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BJarqu%C3%ADn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarqu%C3%ADn%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BJarqu%25C3%25ADn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jorge Franco </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departamento de Biometría, Estadística y Computación, Facultad de Agronomía, Universidad de la República (Udelar), Paysandú, Uruguay</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jorge%2BFranco%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Franco%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJorge%2BFranco%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Colegio de Post-Graduados, Montecillos, Edo. de Mexico, 56230 Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Carolina Saint-Pierre </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BSaint-Pierre%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Saint-Pierre%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BSaint-Pierre%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Prashant Vikram </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Prashant%2BVikram%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vikram%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrashant%2BVikram%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Carolina Sansaloni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carolina%2BSansaloni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sansaloni%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarolina%2BSansaloni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Cesar Petroli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cesar%2BPetroli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Petroli%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACesar%2BPetroli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Deniz Akdemir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Breeding & Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deniz%2BAkdemir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akdemir%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADeniz%2BAkdemir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Clay Sneller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Horticulture and Crop Science, Ohio State University, Wooster, Ohio 44691</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Clay%2BSneller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sneller%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClay%2BSneller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Matthew Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Maria Tattaris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maria%2BTattaris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tattaris%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMaria%2BTattaris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Thomas Payne </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BPayne%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payne%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BPayne%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Carlos Guzman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Carlos%2BGuzman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guzman%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACarlos%2BGuzman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Roberto J. Peña </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Roberto%2BJ.%2BPe%C3%B1a%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pe%C3%B1a%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARoberto%2BJ.%2BPe%25C3%25B1a%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-16"><div class="author-tooltip-name">Peter Wenzl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BWenzl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wenzl%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BWenzl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-17"><div class="author-tooltip-name">Sukhwinder Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Resources Program and the Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), 06600, Mexico, DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sukhwinder%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASukhwinder%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a>
<a href="mailto:suk.singh@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">suk.singh@cgiar.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1819&amp;rft.epage=1834&amp;rft.atitle=Genomic%20Prediction%20of%20Gene%20Bank%20Wheat%20Landraces&amp;rft.volume=6&amp;rft.issue=7&amp;rft.date=2016-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Jarqu%C3%ADn%2C+Diego&amp;rft.au=Franco%2C+Jorge&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=Burgue%C3%B1o%2C+Juan&amp;rft.au=Saint-Pierre%2C+Carolina&amp;rft.au=Vikram%2C+Prashant&amp;rft.au=Sansaloni%2C+Carolina&amp;rft.au=Petroli%2C+Cesar&amp;rft.au=Akdemir%2C+Deniz&amp;rft.au=Sneller%2C+Clay&amp;rft.au=Reynolds%2C+Matthew&amp;rft.au=Tattaris%2C+Maria&amp;rft.au=Payne%2C+Thomas&amp;rft.au=Guzman%2C+Carlos&amp;rft.au=Pe%C3%B1a%2C+Roberto+J.&amp;rft.au=Wenzl%2C+Peter&amp;rft.au=Singh%2C+Sukhwinder"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460418/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460418/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460418/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460418/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460418/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460418/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460418/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460418/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460418/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460418/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460418/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460418/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/7/1819" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-35" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/7/1819.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/7/1819.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/6/7/1819.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
