<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/7/2033" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/7/2033.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/7/2033.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/7/2033.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/7/2033.atom" />
<meta name="HW.pisa" content="ggg;6/7/2033" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Budding Yeast SLX4 Contributes to the Appropriate Distribution of Crossovers and Meiotic Double-Strand Break Formation on Bivalents During Meiosis" />
<meta name="DC.Identifier" content="10.1534/g3.116.029488" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Higashide and Shinohara. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The number and distribution of meiosis crossover (CO) events on each bivalent are strictly controlled by multiple mechanisms to assure proper chromosome segregation during the first meiotic division. In Saccharomyces cerevisiae , [Slx4][1] is a multi-functional scaffold protein for structure-selective endonucleases, such as [Slx1][2] and [Rad1][3] (which are involved in DNA damage repair), and is also a negative regulator of the [Rad9][4]-dependent signaling pathway with [Rtt107][5]. [Slx4][1] has been believed to play only a minor role in meiotic recombination. Here, we report that [Slx4][1] is involved in proper intrachromosomal distribution of meiotic CO formation, especially in regions near centromeres. We observed an increase in uncontrolled CO formation only in a region near the centromere in the [slx4][1] ∆ mutant. Interestingly, this phenomenon was not observed in the [slx1][2] ∆, [rad1][3] ∆, or [rtt107][5] ∆ mutants. In addition, we observed a reduced number of DNA double-strand breaks (DSBs) and altered meiotic DSB distribution on chromosomes in the [slx4][1] ∆ mutant. This suggests that the multi-functional [Slx4][1] is required for proper CO formation and meiotic DSB formation.

 [1]: http://www.yeastgenome.org/locus/S000004125/overview
 [2]: http://www.yeastgenome.org/locus/S000000432/overview
 [3]: http://www.yeastgenome.org/locus/S000005943/overview
 [4]: http://www.yeastgenome.org/locus/S000002625/overview
 [5]: http://www.yeastgenome.org/locus/S000001197/overview" />
<meta name="DC.Contributor" content="Mika Higashide" />
<meta name="DC.Contributor" content="Miki Shinohara" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Budding Yeast SLX4 Contributes to the Appropriate Distribution of Crossovers and Meiotic Double-Strand Break Formation on Bivalents During Meiosis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The number and distribution of meiosis crossover (CO) events on each bivalent are strictly controlled by multiple mechanisms to assure proper chromosome segregation during the first meiotic division. In &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, Slx4 is a multi-functional scaffold protein for structure-selective endonucleases, such as Slx1 and Rad1 (which are involved in DNA damage repair), and is also a negative regulator of the Rad9-dependent signaling pathway with Rtt107. Slx4 has been believed to play only a minor role in meiotic recombination. Here, we report that Slx4 is involved in proper intrachromosomal distribution of meiotic CO formation, especially in regions near centromeres. We observed an increase in uncontrolled CO formation only in a region near the centromere in the &lt;i&gt;slx4&lt;/i&gt;∆ mutant. Interestingly, this phenomenon was not observed in the &lt;i&gt;slx1&lt;/i&gt;∆, &lt;i&gt;rad1&lt;/i&gt;∆, or &lt;i&gt;rtt107&lt;/i&gt;∆ mutants. In addition, we observed a reduced number of DNA double-strand breaks (DSBs) and altered meiotic DSB distribution on chromosomes in the &lt;i&gt;slx4&lt;/i&gt;∆ mutant. This suggests that the multi-functional Slx4 is required for proper CO formation and meiotic DSB formation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/07/01" />
<meta name="citation_mjid" content="ggg;6/7/2033" />
<meta name="citation_id" content="6/7/2033" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/7/2033" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/7/2033.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/7/2033.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/7/2033.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.029488" />
<meta name="citation_pmid" content="27172214" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2033" />
<meta name="citation_lastpage" content="2042" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mika Higashide" />
<meta name="citation_author_institution" content="Department of Integrated Protein Functions, Institute for Protein Research, Osaka University, 565-0871, Japan" />
<meta name="citation_author_institution" content="Department of Biological Science, Graduate School of Science, Osaka University, 565-0871, Japan" />
<meta name="citation_author" content="Miki Shinohara" />
<meta name="citation_author_institution" content="Department of Integrated Protein Functions, Institute for Protein Research, Osaka University, 565-0871, Japan" />
<meta name="citation_author_institution" content="Department of Biological Science, Graduate School of Science, Osaka University, 565-0871, Japan" />
<meta name="citation_author_email" content="mikis@protein.osaka-u.ac.jp" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Agarwal;citation_author=G. S. Roeder;citation_title=Zip3 provides a link between recombination enzymes and synaptonemal complex proteins.;citation_pages=245-255;citation_volume=102;citation_year=2000;citation_issue=2;citation_pmid=10943844;citation_doi=10.1016/S0092-8674(00)00029-5" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Bani Ismail;citation_author=M. Shinohara;citation_author=A. Shinohara;citation_title=Dot1-dependent histone H3K79 methylation promotes the formation of meiotic double-strand breaks in the absence of histone H3K4 methylation in budding yeast.;citation_pages=e96648;citation_volume=9;citation_year=2014;citation_pmid=24797370;citation_doi=10.1371/journal.pone.0096648" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=F. Baudat;citation_author=A. Nicolas;citation_title=Clustering of meiotic double-strand breaks on yeast chromosome III;citation_pages=5213-5218;citation_volume=94;citation_year=1997;citation_issue=10;citation_pmid=9144217;citation_doi=10.1073/pnas.94.10.5213" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. K. Bishop;citation_title=RecA homologs Dmc1 and Rad51 interact to form multiple nuclear complexes prior to meiotic chromosome synapsis.;citation_pages=1081-1092;citation_volume=79;citation_year=1994;citation_issue=6;citation_pmid=7528104;citation_doi=10.1016/0092-8674(94)90038-8" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=V. Borde;citation_author=N. Robine;citation_author=W. Lin;citation_author=S. Bonfils;citation_author=V. Geli;citation_title=Histone H3 lysine 4 trimethylation marks meiotic recombination initiation sites;citation_pages=99-111;citation_volume=28;citation_year=2009;citation_issue=2;citation_pmid=19078966;citation_doi=10.1038/emboj.2008.257" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=J. Buard;citation_author=P. Barthes;citation_author=C. Grey;citation_author=B. de Massy;citation_title=Distinct histone modifications define initiation and repair of meiotic recombination in the mouse;citation_pages=2616-2624;citation_volume=28;citation_year=2009;citation_issue=17;citation_pmid=19644444;citation_doi=10.1038/emboj.2009.207" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=C. Buhler;citation_author=V. Borde;citation_author=M. Lichten;citation_title=Mapping meiotic single-strand DNA reveals a new landscape of DNA double-strand breaks in Saccharomyces cerevisiae.;citation_pages=e324-e324;citation_volume=5;citation_year=2007;citation_issue=12;citation_pmid=18076285;citation_doi=10.1371/journal.pbio.0050324" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=J. A. Carballo;citation_author=R. S. Cha;citation_title=Meiotic roles of Mec1, a budding yeast homolog of mammalian ATR/ATM.;citation_pages=539-550;citation_volume=15;citation_year=2007;citation_issue=5;citation_pmid=17674144;citation_doi=10.1007/s10577-007-1145-y" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. A. Carballo;citation_author=A. L. Johnson;citation_author=S. G. Sedgwick;citation_author=R. S. Cha;citation_title=Phosphorylation of the axial element protein Hop1 by Mec1/Tel1 ensures meiotic interhomolog recombination.;citation_pages=758-770;citation_volume=132;citation_year=2008;citation_issue=5;citation_pmid=18329363;citation_doi=10.1016/j.cell.2008.01.035" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. A. Carballo;citation_author=S. Panizza;citation_author=M. E. Serrentino;citation_author=A. L. Johnson;citation_author=M. Geymonat;citation_title=Budding yeast ATM/ATR control meiotic double-strand break (DSB) levels by down-regulating Rec114, an essential component of the DSB-machinery.;citation_pages=e1003545;citation_volume=9;citation_year=2013;citation_pmid=23825959;citation_doi=10.1371/journal.pgen.1003545" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=H. Cartagena-Lirola;citation_author=I. Guerini;citation_author=N. Manfrini;citation_author=G. Lucchini;citation_author=M. P. Longhese;citation_title=Role of the Saccharomyces cerevisiae Rad53 checkpoint kinase in signaling double-strand breaks during the meiotic cell cycle.;citation_volume=28;citation_year=2008;citation_pmid=18505828;citation_doi=10.1128/MCB.00375-08" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. R. Chua;citation_author=G. S. Roeder;citation_title=Zip2, a meiosis-specific protein required for the initiation of chromosome synapsis.;citation_pages=349-359;citation_volume=93;citation_year=1998;citation_issue=3;citation_pmid=9590170;citation_doi=10.1016/S0092-8674(00)81164-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. de los Santos;citation_author=N. Hunter;citation_author=C. Lee;citation_author=B. Larkin;citation_author=J. Loidl;citation_title=The Mus81/Mms4 Endonuclease Acts Independently of Double-Holliday Junction Resolution to Promote a Distinct Subset of Crossovers During Meiosis in Budding Yeast;citation_pages=81-94;citation_volume=164;citation_year=2003;citation_issue=1;citation_pmid=12750322" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. De Muyt;citation_author=L. Jessop;citation_author=E. Kolar;citation_author=A. Sourirajan;citation_author=J. Chen;citation_title=BLM helicase ortholog Sgs1 is a central regulator of meiotic recombination intermediate metabolism.;citation_pages=43-53;citation_volume=46;citation_year=2012;citation_issue=1;citation_pmid=22500736;citation_doi=10.1016/j.molcel.2012.02.020" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. L. Diaz;citation_author=A. D. Alcid;citation_author=J. M. Berger;citation_author=S. Keeney;citation_title=Identification of Residues in Yeast Spo11p Critical for Meiotic DNA Double-Strand Break Formation;citation_pages=1106-1115;citation_volume=22;citation_year=2002;citation_issue=4;citation_pmid=11809802;citation_doi=10.1128/MCB.22.4.1106-1115.2002" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=S. Flott;citation_author=C. Alabert;citation_author=G. W. Toh;citation_author=R. Toth;citation_author=N. Sugawara;citation_title=Phosphorylation of Slx4 by Mec1 and Tel1 regulates the single-strand annealing mode of DNA repair in budding yeast.;citation_volume=27;citation_year=2007;citation_pmid=17636031;citation_doi=10.1128/MCB.00135-07" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=W. M. Fricke;citation_author=S. J. Brill;citation_title=Slx1-Slx4 is a second structure-specific endonuclease functionally redundant with Sgs1-Top3.;citation_volume=17;citation_year=2003;citation_pmid=12832395;citation_doi=10.1101/gad.1105203" />
<meta name="citation_reference" content="citation_author=S. K. Gothwal;citation_author=N. J. Patel;citation_author=M. M. Colletti;citation_author=H. Sasanuma;citation_author=M. Shinohara;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Grushcow;citation_author=T. M. Holzen;citation_author=K. J. Park;citation_author=T. Weinert;citation_author=M. Lichten;citation_title=Saccharomyces cerevisiae Checkpoint Genes MEC1, RAD17 and RAD24 Are Required for Normal Meiotic Recombination Partner Choice;citation_pages=607-620;citation_volume=153;citation_year=1999;citation_issue=2;citation_pmid=10511543" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Hayase;citation_author=M. Takagi;citation_author=T. Miyazaki;citation_author=H. Oshiumi;citation_author=M. Shinohara;citation_title=A protein complex containing Mei5 and Sae3 promotes the assembly of the meiosis-specific RecA homolog Dmc1.;citation_pages=927-940;citation_volume=119;citation_year=2004;citation_issue=7;citation_pmid=15620352;citation_doi=10.1016/j.cell.2004.10.031" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=N. M. Hollingsworth;citation_author=L. Ponte;citation_author=C. Halsey;citation_title=MSH5, a novel MutS homolog, facilitates meiotic reciprocal recombination between homologs in Saccharomyces cerevisiae but not mismatch repair.;citation_pages=1728-1739;citation_volume=9;citation_year=1995;citation_issue=14;citation_pmid=7622037;citation_doi=10.1101/gad.9.14.1728" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet;citation_author=D. Iwasaki;citation_author=K. Hayashihara;citation_author=H. Shima;citation_author=M. Higashide;citation_author=M. Terasawa;citation_title=The MRX Complex Ensures NHEJ Fidelity through Multiple Pathways Including Xrs2-FHA-Dependent Tel1 Activation.;citation_pages=e1005942;citation_volume=12;citation_year=2016;citation_pmid=26990569;citation_doi=10.1371/journal.pgen.1005942" />
<meta name="citation_reference" content="citation_journal_title=Current topics in developmental biology;citation_journal_abbrev=Curr Top Dev Biol;citation_author=S. Keeney;citation_title=Mechanism and control of meiotic recombination initiation.;citation_pages=1-53;citation_volume=52;citation_year=2001;citation_pmid=11529427;citation_doi=10.1016/S0070-2153(01)52008-6" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Y. Kim;citation_author=F. P. Lach;citation_author=R. Desetty;citation_author=H. Hanenberg;citation_author=A. D. Auerbach;citation_title=Mutations of the SLX4 gene in Fanconi anemia.;citation_pages=142-146;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21240275;citation_doi=10.1038/ng.750" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Lichten;citation_author=B. de Massy;citation_title=The impressionistic landscape of meiotic recombination.;citation_pages=267-270;citation_volume=147;citation_year=2011;citation_issue=2;citation_pmid=22000007;citation_doi=10.1016/j.cell.2011.09.038" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=G. Mazon;citation_author=A. F. Lam;citation_author=C. K. Ho;citation_author=M. Kupiec;citation_author=L. S. Symington;citation_title=The Rad1-Rad10 nuclease promotes chromosome translocations between dispersed repeats.;citation_pages=964-971;citation_volume=19;citation_year=2012;citation_issue=9;citation_pmid=22885325;citation_doi=10.1038/nsmb.2359" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. R. Mullen;citation_author=V. Kaliraman;citation_author=S. S. Ibrahim;citation_author=S. J. Brill;citation_title=Requirement for Three Novel Protein Complexes in the Absence of the Sgs1 DNA Helicase in Saccharomyces cerevisiae;citation_pages=103-118;citation_volume=157;citation_year=2001;citation_issue=1;citation_pmid=11139495" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=I. M. Munoz;citation_author=K. Hain;citation_author=A. C. Declais;citation_author=M. Gardiner;citation_author=G. W. Toh;citation_title=Coordination of structure-specific nucleases by human SLX4/BTBD12 is required for DNA repair.;citation_pages=116-127;citation_volume=35;citation_year=2009;citation_issue=1;citation_pmid=19595721;citation_doi=10.1016/j.molcel.2009.06.020" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=S. Munoz-Galvan;citation_author=C. Tous;citation_author=M. G. Blanco;citation_author=E. K. Schwartz;citation_author=K. T. Ehmsen;citation_title=Distinct roles of Mus81, Yen1, Slx1-Slx4, and Rad1 nucleases in the repair of replication-born double-strand breaks by sister chromatid exchange.;citation_volume=32;citation_year=2012;citation_pmid=22354996;citation_doi=10.1128/MCB.00111-12" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=K. T. Nishant;citation_author=C. Chen;citation_author=M. Shinohara;citation_author=A. Shinohara;citation_author=E. Alani;citation_title=Identification of new genetic risk variants for type 2 diabetes.;citation_pages=6;citation_volume=6;citation_year=2010;citation_issue=9;citation_pmid=20862305;citation_doi=10.1371/journal.pgen.1001062" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=P. Y. Ohouo;citation_author=F. M. Bastos de Oliveira;citation_author=B. S. Almeida;citation_author=M. B. Smolka;citation_title=DNA damage signaling recruits the Rtt107-Slx4 scaffolds via Dpb11 to mediate replication stress response.;citation_pages=300-306;citation_volume=39;citation_year=2010;citation_issue=2;citation_pmid=20670896;citation_doi=10.1016/j.molcel.2010.06.019" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=P. Y. Ohouo;citation_author=F. M. Bastos de Oliveira;citation_author=Y. Liu;citation_author=C. J. Ma;citation_author=M. B. Smolka;citation_title=DNA-repair scaffolds dampen checkpoint signalling by counteracting the adaptor Rad9.;citation_pages=120-124;citation_volume=493;citation_year=2013;citation_pmid=23160493;citation_doi=10.1038/nature11658" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=J. Pan;citation_author=S. Keeney;citation_title=Detection of SPO11-oligonucleotide complexes from mouse testes.;citation_pages=197-207;citation_volume=557;citation_year=2009;citation_pmid=19799184;citation_doi=10.1007/978-1-59745-527-5_13" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Soc. Trans.;citation_author=J. Rouse;citation_title=Control of genome stability by SLX protein complexes.;citation_volume=37;citation_year=2009;citation_pmid=19442243;citation_doi=10.1042/BST0370495" />
<meta name="citation_reference" content="citation_journal_title=Worm;citation_author=T. T. Saito;citation_author=M. P. Colaiacovo;citation_title=Crossover recombination mediated by HIM-18/SLX4-associated nucleases.;citation_pages=e28233;citation_volume=3;citation_year=2014;citation_pmid=25057454;citation_doi=10.4161/worm.28233" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. T. Saito;citation_author=F. Mohideen;citation_author=K. Meyer;citation_author=J. W. Harper;citation_author=M. P. Colaiacovo;citation_title=SLX-1 is required for maintaining genomic integrity and promoting meiotic noncrossovers in the Caenorhabditis elegans germline.;citation_pages=e1002888-e1002888;citation_volume=8;citation_year=2012;citation_issue=8;citation_pmid=22927825;citation_doi=10.1371/journal.pgen.1002888" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. Sasanuma;citation_author=H. Murakami;citation_author=T. Fukuda;citation_author=T. Shibata;citation_author=A. Nicolas;citation_title=Meiotic association between Spo11 regulated by Rec102, Rec104 and Rec114.;citation_volume=35;citation_year=2007;citation_pmid=17264124;citation_doi=10.1093/nar/gkl1162" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=H. Sasanuma;citation_author=M. S. Tawaramoto;citation_author=J. P. Lao;citation_author=H. Hosaka;citation_author=E. Sanda;citation_title=A new protein complex promoting the assembly of Rad51 filaments.;citation_pages=1676;citation_volume=4;citation_year=2013;citation_pmid=23575680;citation_doi=10.1038/ncomms2678" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. H. Schiestl;citation_author=S. Prakash;citation_title=RAD10, an excision repair gene of Saccharomyces cerevisiae, is involved in the RAD1 pathway of mitotic recombination.;citation_pages=2485-2491;citation_volume=10;citation_year=1990;citation_issue=6;citation_pmid=2188090;citation_doi=10.1128/MCB.10.6.2485" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Shima;citation_author=M. Suzuki;citation_author=M. Shinohara;citation_title=Isolation and characterization of novel xrs2 mutations in Saccharomyces cerevisiae.;citation_volume=170;citation_year=2005;citation_pmid=15716496;citation_doi=10.1534/genetics.104.037580" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Shinohara;citation_author=A. Shinohara;citation_title=Multiple pathways suppress non-allelic homologous recombination during meiosis in Saccharomyces cerevisiae.;citation_pages=e63144;citation_volume=8;citation_year=2013;citation_pmid=23646187;citation_doi=10.1371/journal.pone.0063144" />
<meta name="citation_reference" content="citation_journal_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=A. Shinohara;citation_author=S. Gasior;citation_author=T. Ogawa;citation_author=N. Kleckner;citation_author=D. K. Bishop;citation_title=Saccharomyces cerevisiae recA homologues RAD51 and DMC1 have both distinct and overlapping roles in meiotic recombination;citation_pages=615-629;citation_volume=2;citation_year=1997;citation_issue=10;citation_pmid=9427283;citation_doi=10.1046/j.1365-2443.1997.1480347.x" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Shinohara;citation_author=S. L. Gasior;citation_author=D. K. Bishop;citation_author=A. Shinohara;citation_title=Tid1/Rdh54 promotes colocalization of Rad51 and Dmc1 during meiotic recombination;citation_pages=10814-10819;citation_volume=97;citation_year=2000;citation_issue=20;citation_pmid=11005857;citation_doi=10.1073/pnas.97.20.10814" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Shinohara;citation_author=K. Sakai;citation_author=T. Ogawa;citation_author=A. Shinohara;citation_title=The Mitotic DNA Damage Checkpoint Proteins Rad17 and Rad24 Are Required for Repair of Double-Strand Breaks During Meiosis in Yeast;citation_pages=855-865;citation_volume=164;citation_year=2003;citation_issue=3;citation_pmid=12871899" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Shinohara;citation_author=S. D. Oh;citation_author=N. Hunter;citation_author=A. Shinohara;citation_title=Crossover assurance and crossover interference are distinctly regulated by the ZMM proteins during yeast meiosis.;citation_pages=299-309;citation_volume=40;citation_year=2008;citation_issue=3;citation_pmid=18297071;citation_doi=10.1038/ng.83" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Sci.;citation_title=DNA damage response clamp 9-1-1 contributes to chromosomal assembly of ZMM/SIC proteins for formation of crossovers and synaptonemal complex during meiosis.;citation_author=M. Shinohara;citation_author=K. Hayashihara;citation_author=J. T. Grubb;citation_author=D. K. Bishop;citation_author=A. Shinohara;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=T. Snowden;citation_author=S. Acharya;citation_author=C. Butz;citation_author=M. Berardini;citation_author=R. Fishel;citation_title=hMSH4-hMSH5 recognizes Holliday Junctions and forms a meiosis-specific sliding clamp that embraces homologous chromosomes.;citation_pages=437-451;citation_volume=15;citation_year=2004;citation_issue=3;citation_pmid=15304223;citation_doi=10.1016/j.molcel.2004.06.040" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Stoepker;citation_author=K. Hain;citation_author=B. Schuster;citation_author=Y. Hilhorst-Hofstee;citation_author=M. A. Rooimans;citation_title=SLX4, a coordinator of structure-specific endonucleases, is mutated in a new Fanconi anemia subtype.;citation_pages=138-141;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21240277;citation_doi=10.1038/ng.751" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. Storlazzi;citation_author=L. Xu;citation_author=A. Schwacha;citation_author=N. Kleckner;citation_title=Synaptonemal complex (SC) component Zip1 plays a role in meiotic recombination independent of SC polymerization along the chromosomes.;citation_pages=9043-9048;citation_volume=93;citation_year=1996;citation_issue=17;citation_pmid=8799151;citation_doi=10.1073/pnas.93.17.9043" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=G. W. Toh;citation_author=N. Sugawara;citation_author=J. Dong;citation_author=R. Toth;citation_author=S. E. Lee;citation_title=Mec1/Tel1-dependent phosphorylation of Slx4 stimulates Rad1-Rad10-dependent cleavage of non-homologous DNA tails.;citation_pages=718-726;citation_volume=9;citation_year=2010;citation_pmid=20382573;citation_doi=10.1016/j.dnarep.2010.02.013" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=T. Tsubouchi;citation_author=H. Zhao;citation_author=G. S. Roeder;citation_title=The meiosis-specific zip4 protein regulates crossover distribution by promoting synaptonemal complex formation together with zip2.;citation_pages=809-819;citation_volume=10;citation_year=2006;citation_issue=6;citation_pmid=16740482;citation_doi=10.1016/j.devcel.2006.04.003" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_title=The kinetochore prevents centromere-proximal crossover recombination during meiosis.;citation_author=N. Vincenten;citation_author=L. M. Kuhl;citation_author=I. Lam;citation_author=A. Oke;citation_author=A. R. Kerr;citation_volume=4;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. Wach;citation_author=A. Brachat;citation_author=R. Pohlmann;citation_author=P. Philippsen;citation_title=New heterologous modules for classical or PCR-based gene disruptions in Saccharomyces cerevisiae.;citation_pages=1793-1808;citation_volume=10;citation_year=1994;citation_issue=13;citation_pmid=7747518;citation_doi=10.1002/yea.320101310" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=L. Xu;citation_author=M. Ajimura;citation_author=R. Padmore;citation_author=C. Klein;citation_author=N. Kleckner;citation_title=NDT80, a meiosis-specific gene required for exit from pachytene in Saccharomyces cerevisiae;citation_pages=6572-6581;citation_volume=15;citation_year=1995;citation_issue=12;citation_pmid=8524222;citation_doi=10.1128/MCB.15.12.6572" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. Zakharyevich;citation_author=S. Tang;citation_author=Y. Ma;citation_author=N. Hunter;citation_title=Delineation of joint molecule resolution pathways in meiosis identifies a crossover-specific resolvase.;citation_pages=334-347;citation_volume=149;citation_year=2012;citation_issue=2;citation_pmid=22500800;citation_doi=10.1016/j.cell.2012.03.023" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Budding Yeast SLX4 Contributes to the Appropriate Distribution of Crossovers and Meiotic Double-Strand Break Formation on Bivalents During Meiosis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="twitter:description" content="The number and distribution of meiosis crossover (CO) events on each bivalent are strictly controlled by multiple mechanisms to assure proper chromosome segregation during the first meiotic division. In Saccharomyces cerevisiae , [Slx4][1] is a multi-functional scaffold protein for structure-selective endonucleases, such as [Slx1][2] and [Rad1][3] (which are involved in DNA damage repair), and is also a negative regulator of the [Rad9][4]-dependent signaling pathway with [Rtt107][5]. [Slx4][1] has been believed to play only a minor role in meiotic recombination. Here, we report that [Slx4][1] is involved in proper intrachromosomal distribution of meiotic CO formation, especially in regions near centromeres. We observed an increase in uncontrolled CO formation only in a region near the centromere in the [slx4][1] ∆ mutant. Interestingly, this phenomenon was not observed in the [slx1][2] ∆, [rad1][3] ∆, or [rtt107][5] ∆ mutants. In addition, we observed a reduced number of DNA double-strand breaks (DSBs) and altered meiotic DSB distribution on chromosomes in the [slx4][1] ∆ mutant. This suggests that the multi-functional [Slx4][1] is required for proper CO formation and meiotic DSB formation.

 [1]: http://www.yeastgenome.org/locus/S000004125/overview
 [2]: http://www.yeastgenome.org/locus/S000000432/overview
 [3]: http://www.yeastgenome.org/locus/S000005943/overview
 [4]: http://www.yeastgenome.org/locus/S000002625/overview
 [5]: http://www.yeastgenome.org/locus/S000001197/overview" />
<meta name="og-title" property="og:title" content="Budding Yeast SLX4 Contributes to the Appropriate Distribution of Crossovers and Meiotic Double-Strand Break Formation on Bivalents During Meiosis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/7/2033" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The number and distribution of meiosis crossover (CO) events on each bivalent are strictly controlled by multiple mechanisms to assure proper chromosome segregation during the first meiotic division. In Saccharomyces cerevisiae , [Slx4][1] is a multi-functional scaffold protein for structure-selective endonucleases, such as [Slx1][2] and [Rad1][3] (which are involved in DNA damage repair), and is also a negative regulator of the [Rad9][4]-dependent signaling pathway with [Rtt107][5]. [Slx4][1] has been believed to play only a minor role in meiotic recombination. Here, we report that [Slx4][1] is involved in proper intrachromosomal distribution of meiotic CO formation, especially in regions near centromeres. We observed an increase in uncontrolled CO formation only in a region near the centromere in the [slx4][1] ∆ mutant. Interestingly, this phenomenon was not observed in the [slx1][2] ∆, [rad1][3] ∆, or [rtt107][5] ∆ mutants. In addition, we observed a reduced number of DNA double-strand breaks (DSBs) and altered meiotic DSB distribution on chromosomes in the [slx4][1] ∆ mutant. This suggests that the multi-functional [Slx4][1] is required for proper CO formation and meiotic DSB formation.

 [1]: http://www.yeastgenome.org/locus/S000004125/overview
 [2]: http://www.yeastgenome.org/locus/S000000432/overview
 [3]: http://www.yeastgenome.org/locus/S000005943/overview
 [4]: http://www.yeastgenome.org/locus/S000002625/overview
 [5]: http://www.yeastgenome.org/locus/S000001197/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
    <title>Budding Yeast SLX4 Contributes to the Appropriate Distribution of Crossovers and Meiotic Double-Strand Break Formation on Bivalents During Meiosis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"rOexLUiDGGNdF-xFpbViDbTkxA9ysoqIuhOVpGUmIW0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460406","apath":"\/ggg\/6\/7\/2033.atom","pisa":"ggg;6\/7\/2033","ac":{"\/ggg\/6\/7\/2033.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/7\/2033.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/7\/2033"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/7\/2033"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/7\/2033":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228M2p4j56MDZSWLeacomAfXCAoYIIlbYsMgaQ7xz13VCglHu+MnZSaRRs1l246DIMuKqIOjnnq8HpcefpQhR2DN2UwPTGhGd4YuPIEto4keRloX\\\\\\\/j+tqZw2RZ1ocQvDzAih+Hxw3NS0IfwTyHXulAMF+cvDiWaYwE0XI4QyySkEEZA0wEn3MDiHvKdRyNl8p3opTLeBWi93aCrh0tcseqysRRhAOHJFQ\\\\\\\/Gg3pkzcKeBrMDryLq1wCni1u4sl1FMPjupFOKv2cKFfM34LRnpEu6g2CDqC4l2U6ZIyPFrhIagsVcIrbZGJjNUni84I7Gw3ElxOdzdvXUuUQ8Z+v3b6Wgx6CFNmiJrSACYZl\\\\\\\/jL7oEH5Xspm57FfqzJDTkMXEWRsKOlXDntQ7QM0CZbo\\\\\\\/rp5quNcYOFTEVkIrnPMjPv7kr0LSyclMO\\\\\\\/l2xvlZc+3ZGyNDM+BW3BRETTPAgCZLD2fE3OS\\\\\\\/lHrBTr6DIyLP5hKmmlHLqRBOsz1qd2R8LZQ47BhzbZTDmeVMPMqtklE93T60bHdILP6Z94OYWP6o9KnnMCM+stbf2GcviivgHBfGlvgcy+qDZDhZvv1XzlGVVzXEx8F9hg6plDNOOu7SZYo4PmbOblYfVTg5570q92zeaVpjZMAoANnYdaRXPtviADAg3Ze1nqzD96BUFXv8Sqojr4dWIlMwVv9GbJOXqfgi3xi2gMm1eGZ9qz5U07o\\\\\\\/lu824PS3wi1+8RcHA62oglbyqYAZH52vsuDWVX7MOYFdp\\\\\\\/2wpDEK6UjbVgyfzxxjE5etq0MlcXNNdYmbmbz7rt5nayM01ENKOQX4LAFhQjyfbN1c8FPhSyPcAC+dODmHKPl1VQXjdVF7175nOMTfhtKWYGef8VzhpPF0W3ZTtzUleJ2ncsnD5SGYGXNQnvpZ\\\\\\\/AKHrcxwbu\\\\\\\/a4FgXxEP8\\\\\\\/mLmnU0x\\\\\\\/B2hKSanumZ3VEj0ikZ8uncWe\\\\\\\/0y\\\\\\\/0\\\\\\\/Fh7M\\\\\\\/UDHR9j\\\\\\\/okM+fgWsfevPI3J8LCO9gSAGI1BcfllRDzNvrwO6zXBqHDD00aijbOAyKjtd8XHnNvlCs0ZIn4D+xVaiWvJ6b2qrFzEJYqZj66qVt1h22jXUoyVV5VHFPPTitKgalQFKu+1W0qUfkcxjE5UH0Tw\\\\\\\/Rc7vN7Uz\\\\\\\/Trq9gaJXHEMcUNeXovorQDUp0t6CvGaWX+2XMlcHUEqf6WPeIUCcKtp5lhlBwO8GG2mU1I4hmODkAM7oHLdhV\\\\\\\/MWyaRxHYlI7PEXGEQmjafy9hon4ao6yoGbd7DeoKQY0MkVIi\\\\\\\/wIet\\\u0022,\\\u0022iv\\\u0022:\\\u00229UaDtHWcFznx\\\\\\\/\\\\\\\/ijLjmhnw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228974bf77252f0a695392e117541bd3a47e76e75bb50d8583b78077c5066f92d6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cctT2cLre0SccQcM6ZJpJ5a6kmXekLXqtNLsOsdzu6JAiPutlzvn8dG2Dkzv3dGT7pm5Z4YCFzg\\\\\\\/Ofx81ST03llbmpSej9mlGT13QseNnXrlL8lJal6uG6S0tRcw\\\\\\\/wIf42kgzV4\\\\\\\/8BPGnm0P2kENvwqt25ULKfiP1ana5ZPsJ7UjdmkWa20hO+I5V9mXN5u7E4H7GZc5d0Hn\\\\\\\/VB8VAj6NpIAlTVrZr44EuRf1pJX0j9Mh4NTldH0HzPuov80PuEcCISMGWnNljzxnFdp5EtkmM2tG4tgJ662qYYojKY44gxdcSJC6ngmbe7uKtstj44KLaiKk4pHPdXt0XQk09k1\\\\\\\/SkRQFbIkclyTYbk6Y96sZcBQD7DnzkcOA4R6vVhmdBLs0qJt2cxjaS+bTpYlHaxUafIrt8MpEO02yMsccHo7FMn+m8qqH6u\\\\\\\/JAes133UrRfqg7TwI38R2oxvj5t8hjy6VqVTEuJf\\\\\\\/ZeBUUh+BiJPP46dSlZvKmMnQt9DzJSlG6xGWWnmaPuZD9YeI6C2lV83pD5231SilHo6v48EVfVjqtMQjf4HIjU1OzCVgd3b6eLIIP9k4Dr39C9ius70Gk24lZ7YHX7SQZa37rkjZIkJH0xAofFaE6MKBMe3ZM3MR4K83Wf7AZytlCDm0dtyJv4R97yxRaYlCkPCs\\\\\\\/zFAY2nzb3pCHG7HBpYxWe\\\\\\\/ASb83Fzsd6yb3qnDbiMb9vj9uffq4q65MVn4FaQG40SwG19GNKLavuX9Y9PUO3Zp3OrkEEB1h1upisJGz7meXcfUIKnaevlWwhHFmzyNO57WDFhymtn9YuAvUrcUYOlLINtcIhz4NqxSYmYYzO6G1jWq7IcshM9AUYj\\\\\\\/JQHduTkfx6aJtL2JUVl1CFbgYOyjouG5acZvODUFUtF8jnvIW1CZQzMmXunkfJ76mUBnIW4YgvW2ZW2Sxt8Z8JmX0FqqeuY5K1MthhW3jg3sPULOX19gSXPZFPEN2SeAdtd5ybbR9bnEKlow\\\\\\\/s\\\\\\\/jShguSUJH5vIZcm6JpqROscMOZPBciKCPRQgPzhnfYS9fM2XopJqVaYkV6Z4IziUkkKuW3dYJ+CB2PEWbDnVio1OrkcDP\\\\\\\/AjNhKHKKr7lYQjsjoJ9FZUsIei7NIsm+yedWkoZiCqiztF9nrHMGipENvcP5GYSzgju9qBCeBZru7cHCIbg4Z5G\\\\\\\/hcNiDy+AvoevaOvic8sB5rI26FT6p52AGFO5e2KBOuJssUYRQia8a5DOmHjFbGbKPUnj99PG4ACjzIMpmqJkfz\\\\\\\/sgOZPFoeTbO502qiB7W7ZPsI81fEFTnrnND+cpWpU3\\\\\\\/MurLwLV9qk67oR6Ii8js464ujvOVGeSSWI5IPUftvH3dQ9kaRdKHbQURgDGEhNaSWYTgW2d5xgQNFGOVVjYCb6O44vmbFEcL573VeL8vEi2OprBSI7TJplIS9qAOzGtCdcEhKNilPAew4ed1MTOn18PExr8NuDrhtL+9FyZJeoPRl8dFKjFuoqo2mvAGWVRQlGz5FYxrCxfogBZlM37sExkACTS+hWPdOwILLrL+vHcS+0WVmQSvUwWqAwJeElJ3WfHmPZqAPFg\\\\\\\/SlL0cggduDNFbrilXH3mMoFyvuQzs1hOpAh7+RSIWwRVPkftS\\\\\\\/iWA+QAH1YbnZX9sI6hAN14BzbyuuX8XBW1DgFiGWRNp60Xcdjs8w34I7XGH2ouNHkiRMt3PIuHnfkrQtMQnKCGpKnoiZWGOJ++M38bJ1qXZ2bI2laiA+FKpi563pzC+39n6B7v9kyGY9La+w3tuFD+Eb4mlW9Q02PvISn33K4r59W2W5hpLtwaVmwMSYCzyd4YuB59DIqfeCTXkLwO8gbER48vqR9f0cyNvAAmoT2JsC++MZMTKQJ7FotDtbA4nejLfoXHhlY21ssK1x5xXr0PB4TcwmjfE0ZylrxHpKQWdw6o1r1Aweh5uFV0PcJmEpJE0TgxewWlFAGKD6FTHC3f86pz2KkHNU8Cpz5PJJr2Kwts4+XxKCD6N\\\\\\\/IoAVq9p2HbcmwU0TH1U4pP0aemS1jvi6\\\\\\\/97wWFg2ESJBNVKZy0dpYuNKM8sZVs7qqY91ZaCLMgVyR5UrYpPOlI\\\\\\\/GG5dwcYpBOErLl05m7B2IlOKg7+epIrvgQG+lyNWYqjwPdnxuAS6+RMIEhXcvMejJy+17KzvFm9qbKVBkvJCWFohOebrW1v+Nn8uY12h\\\\\\\/x5g1lwVT0K2s85w1g4o6IuKzbWEYsc9Zmmvajekm13kAKfV5EftGTdWSplNcJAjfuN0nt0cR6tf+uRjUfU24mXIJ0hJFmDrB3b0EzYTQsn9GMe1Qoj9biOQWk+5govquUbx6QliAZ2kzvClFGlIUw\\\\\\\/n1my2pEnLpyrAaHchOeBTTe9BuI1PCv9JQPiCGgKyt1LHPVPK+bICuaIMwCcqtBcEk+yn9mwRcg3bCod8P+2udnDzYIQyJ0wVQOpiCFXoSgy42UnNri3XaxK2I1UnQo4ZVwKbrwonSsMUqvH5rHwuxyREXWpvacz6TB7310BC5Sj4s4LkgXSnbkWQVqEGWtX13ZqV65J7Sy6CX\\\\\\\/tBertZLp891hyf0lCNIdv5KYs2gd5psExiIciyjXFFVpxiV062\\\\\\\/DE4J5RzB0eLPhFcDAeIavV8J3ClXO82KlzyuECfXRfom9BqBESZaHqUWj13BbzvfXQSd3r94G3pQF6ncGkouJWXecmDnYBiUHokQsk44U4xkwTmmE6VnR3PDArtKG\\\\\\\/qXNAiIE3RyjcyzOU5gwvjN6cqeYDySHAkqCpA24aES72AAM8GxjDlq3s\\\\\\\/wwRsl3PxhwNi18WNX2km7WETi0ZRDMSot1T8O936xgtYejUVFo6hxbcj3lr0fp6Z2ieJUwPpw7pZ2+\\\\\\\/ly2FgL5xbb1peBJynYgyiom6Gf8uw+zkkEFx8reLNze2W3PxJGT4ggdBDXUq4uK8FqzVQ4XVErJNTg3j7o0oFbFSgIhaO9cx5qzk4OTt4uSbLiM84yNoM0XEEiiiOpH89kXYc5rq1A7bYAF6dkYgn6GKMANvN5ghK6al4fW9e7U\\\\\\\/C5vevgfK0iLSlCaY+d6RA6OZ77e8nhr6tgwMC\\\\\\\/fA7xTyPKdVaLnX2fSJY4m\\\\\\\/tw+MnS\\\\\\\/f9BJb73\\\\\\\/iHjTCYE0KLkUtDGjQDKQxvLNmEXqml5HcCYXKGMci7jv8C7GICQT+GrHAa1ldQraRYM\\\\\\\/vMoP6qiAGo7vBpHe4Onz1D5gSNZ6EldjRQtyq8INLKK5qIj9cFnAH0d3LXPlj+EWwBWZgUsr6Beo6i+MgOaRMvjK3sm6\\\\\\\/QGpVm7WQyroQXTbgE2PdVg3j+JC3sArKSRsREDLG0DTtEpaNYUM\\\\\\\/YuTEBnHE1eUj2PKktLdlDlY\\\\\\\/57hqaZsI0rf\\\\\\\/L3dAnYHpZnxp9mjR4XhnoFuqzdOWfvy\\\\\\\/C8J4EKvhuUow4Ncr\\\\\\\/l+VrkGW\\\\\\\/30Oi6nINlYWbOCYviwBL1zGLmR3qB33I44Sg5mXAZNt8pWb4KCdqnWgGW9g6Y\\\\\\\/wsUjNi1ZkJEUiLpDIyJahWr5NPSM7CQ2ko8WrOQWk9LfxKqhNbKzp\\\\\\\/uYccIKn0Fery2vUL20my7gJGI9m6RH74NfDPFkY1ZINVWbCP0C8VcX+IkPLU9mVOhH08HRXGFt4V3rkDvd3VJu4bI1NKbszySG34dqqK8j4rTGcgx5++qwq\\\\\\\/3UsAKOchY2QnNvM8+BfiCZsjiTWObBatJnX+c\\\\\\\/pDG40lJ7iSwOUSfSXGn40sutZaOOuOCmf7BXFljNgnw3xEhizvwYup9vmJW1gDB6cvMJmnM11\\\\\\\/tYGTJ\\\\\\\/VBN7a83lAGlnvMLOl8oXxk+nWRKMmvTvb6QUB0w5Myy2MUobPgUjABZoCzxbiOpxw+EqB8\\\\\\\/xP3lJJqj8oLOwBe56SD5WmO\\\\\\\/FkOz+XAZaa817DyB8XopECad8qk2DNE0bDHmEsSMM23KKOfHKw9RINLi7xSKm\\\\\\\/O8jZ\\\\\\\/44GgmNqYVaOiq5IUK7t+\\\\\\\/M3+CiHmWN9rtyb+dH5sh++0pxofQzZZUjjuMCWZs7l8BkjNetMH8lATzUorv7r5oDUZ7dL7PUGL3i0jvA6pY3FXagtLy8OlgWpBSVbHjB7q4RTWjpME4kS+OZtrslTiCuw96+LpjVOTtZvb6ugkOEpqUUy2AS\\\\\\\/QUZT3jRksvPWj2jzbbD8bezpvHiTSGXbgVdd79qRw9pRUfwZOJDBi9\\\\\\\/J0fNpZTJrPWc+UVnHgf8ty+zw7lhgbBDFeQekyx8GVdXkHrmvSaB2VonLiwpz8oKXZPrhvVl2ESkVjfaeXApillJxwoL0z\\\\\\\/ZxSFGYy3syRhb6OjvTQkfcvXGTyBS8gUffEtLB7wFQlRCLWPewByGb32fQbug2DdXsGpaiyQpEQhFucUP6TgR1gh4irP0u4v5TBRVpVv7SaGIj\\\\\\\/dmYyo3NlqWM79yzyMwkXhOdoXCFeCg+UnFeQuFC+JRW5um5dmYEYrt5n8J6fk7KDRSDvn1sb1oQhMwrIN+mbGbTAsM6K6bv6c5aa0tcBz8iVKCEGJBEx+XnTsRXHTaIiY\\\\\\\/eCBPDPOvQaL3sp3yCkfT1HqI752Od6LyfY9ow5tsjtBdti3cpu\\\\\\\/DjHjfNYnZ8MA81G1QCN8j6TA2hlbvcAWqXaAuEFyBmdnwXkK9Wa36goIQ0Psu8q\\\\\\\/ONR\\\\\\\/\\\\\\\/gJ6WYi5l7HJfAHWwCFKRmTk\\\\\\\/\\\\\\\/9ZrbEavoATuW\\\\\\\/ujZ8k9QKBvOW7\\\\\\\/iP1nTHp5V6CPz1M1mk3ijWmmYvxEXRm1HPWutMSLnM7yODKpx0Ye2EHCd2h21\\\\\\\/bdOiN6T7SX4koJObrXajAQ3xJQc1zMNZI5RtJ4gfmMyAiTe0gsPCc+NDlWyLT0Lc19n8xfwtjUuXvfvX5w3IAuFilXNRCze94xALhhxfpOcewqgLpMqeqT9\\\\\\\/7644XfQs1ubGaBfjxNoi7HdN7zm0zKla7opCVJ0cdpxBSYa+rp5LTGR3PwZhIOVFW5JiRKxfts3h8j+kbXTBoiBdfeYyqD469SzbjKUJWAkHWg9ZZyN8zFziQ321cej3JE5dgL1Gy4FXH+xtk0ThIDSYzTObM+vMx\\\\\\\/jxmYbeZndAr7EjHkYPw0BLCBzYblni2L9RHlQAjr8h+fZDq15A8qCFExcanzR7jA24b6lDLGsYlKw0tzOwgyAJl4lh2NXbmGUUe1akNq+DWyP8KYjFUHxfZX\\\\\\\/0zyTbnJz4SB+aLdnrYF9q7ul4eWDhDrh0spYWiSH21w3LqhTjXC66+bWw+dOklikgsoeqScpp4B7Ipcu6D+kvDPVIyrT8bL7neY2CRlYtAwbq+maAmB1faBd482sNugWLDuXjVVTW0UHvP16pXbgLX\\\\\\\/nMjQkn3s7hHN5CIrPqRe6RyyHjwRXvHyG4W\\\\\\\/HbDMDglbhgBEq6n+bynu3k6\\\\\\\/pULdfN71SA8smRBcWZRCMEXd4ZaP\\\\\\\/FOQ422ok5rARuiS9qgsmrXOvHuMCRnpAsFCeF5efc4z7wwtaUbS8ixZJFiaNHku6TQ0OpbKNvKmWOeuRoJqgfUp\\\\\\\/cfZZ2XlsJLowemprAyhb6YAM3xFR8UgpldLpTGcCL3qx9tNpi\\\\\\\/rrgHk2R89q\\\\\\\/2yHCkKTSZhO\\\\\\\/6E9ljLSyDkVKK45PZ3FexMlk0zczQ17bv4rcgV64sGwxD0x6NyTLIpmtEwAzV8oKtDsSzLtnEgVaLIh3AAlebeZkVI3m2LoMmHjY3QxUE73QsRZ44CZrGVxOaXDXbdkK\\\\\\\/6eSzPJooftS341dAgZtkZ6zuea5wlUqjEMYkSAiqKM08B1vsnA1FMn7JFd2F1kzSRtpW+IRuHee3dwEhkoYUaJ5AY4YEi4QHBtdstsiMRDsCulQDmM\\\\\\\/HpGDjWB4hAsnnPlr\\\\\\\/31hC\\\\\\\/txRl3QG9WyiJP7vKhGalZkpA8LLK5TpLgLkc+QSvSARd+GZknJuUV3gvUn12fIercrZe6e8rwJQtClXfY3h\\\\\\\/wdNUVdVC05fHuFQJHtgMJlleoUnWCHa5gXXQ4rdbeFnXW4KH7tH55RL98TCnfHQmsyWMPoIawmw1xZnTntRjlOSC0DZR23LcBT1SlJ61y13zyYVT5HXOKLhjZoMez0ao0qlCQ8zxeJtuP+RNqQtRo9gYg44ihnS9zyzHCdFuaIJNgB9G31MD+Cosk0I+oK33E\\\\\\\/ggvRAlGBuce52ahlsdv7vdg7RCXDT6KUDok7eLMemIf2tJE8+R0a0t7MDKng\\\\\\\/aCFZEkyK7h7lwmJbLNhY4XI46SxbfzpA06RXTc4Mgjnvz6QrqcCPTVpPzTqMYYh2b\\\\\\\/89rkuIj29GDgjKWUywWhcokgi+rn3ZLg7v2Y9J5zFR\\\\\\\/9UGgQfgVszCmZyo3tiZAq9SR+SVy3munkDBXJQ6pr29vZofyXLQBerBhFXwVtcQaug77SB1VdsUR2z34CxUy\\\\\\\/MJbE9ZGlxCZursWE7tZdOejmAmBo4sj9d4NGCuSEQySsMh6CVXPi1++J3qN+GrvtZeC988V\\\\\\\/wWwukjVctdJatTTLr\\\\\\\/qbbHUO4mHBuAsrSitpTUf8MOIFnNNXaLpZzP7pyR6NajOwKH9nQ3D9xNgvIF85OFNYNBqhHOEKIY+RyMKbfCCbGKZrW7ycMtgq0A8yOteX128ytF9wZuVkY4Qng+uafXw9Do66sYlF5fkgUkVwsoaFYG\\\\\\\/ttcoNB7wot0\\\\\\\/9oOX7S9Nj6z3PwLGccinr+7R2iwHCN0aaNGot+7JLHfje55j3N723+xWqCg1GGDeDex17IGoPJXUrfqcxzQt9oAVawVyQsbymmRtCfJX7jCkK00LDk3qqV5GrSt3V7SYFD5twXcCNnc3ZU5GinbyBObB5UJNhohZOkuJOLL\\\\\\\/QEOr\\\\\\\/Xtu+BclFLUWvKZYbUAE4ZqbvxTbqIlpcDqW4gxrQJER3goaiwVrNbWrOCotPhdSm7F2T\\\\\\\/a2AoVmmHJbPj8RiSeB4jzblVIjgQXUnTseY+qaeXWrIrPz9Isp6FmaMv20eeN1m1SDbNglN4cxB0ncLciGA68eRxcQqAeR4irdFVliJDUojxvh83jMF4aUcAUI0x4dAabAhR8o75tMi7Ez4VtYsqiiBSuuVY3Cf8Vgdckvx0kCc+3gdgyhDaD6ZkSj14K0spucMxpfL7CTRoNr7VyW6\\\\\\\/07kxT6wgNqN\\\\\\\/ipWGOaJJxdu4b6l4Sicp2ay9UeNiQHooAaZ3nKL1s5eXmJ7OuxRauRUeKWHNiTbIhArTYDC1XA32x3iK8aSVTkzWqMA94mtaFLHEIgLauqIBsN8\\\\\\\/bb1Pcxs40+dLnyqGhC5cQCvF90Ir8AYRQ94Z4YOxDx+e8PWRA8sNfeIo78YGpNRso0Ywv4es3sElv6o\\\\\\\/3Hn6HlwEDA9ymv3T9pCp9NAj\\\\\\\/dK9SEPpZFyp93\\\\\\\/oslu9pVMOWO6CRbiilZ+VuVmK5T1BhFPdXI5AMCE59zegsr3GfB97NgZgo8N911A4n5nueQldqR++eqToG\\\\\\\/ZaxYBjMnqQaUP0hYlJR\\\\\\\/ANhQK\\\\\\\/0Y7SgfSPatZ9hNCHKcACKucSwxxCMWWM5SnqTtJmzqcXBQp7\\\\\\\/JWUT4VcwO9osn3ZpYHAZ66P9W5B75l2VI7p5\\\\\\\/on7BXHxZIwB1WkIHnZzNeG+JaJQRTMpDVb2Yh6b4NeFTKYDq4ew7ENs3ZpXEYkyV6MjXgKirutXo61p++uPLAOhOBQtlSS6+4mRZGoCcR42eunnRHtkCr0G18xyirgkv56lXPYE3tp8tF330eUR7L\\\\\\\/q3giHAx61pQpyHo2K3\\\\\\\/8H6Xumn8orX5uADQkZGxGHIonGGWlK5Ouv05iIeSLnbiHQqT0YYLYbpv2yPXziu8utUGss99VJnNV\\\\\\\/OgTQjD9VfkcOq6e2z6ZXhFjt+NGCOjwBuh80\\\\\\\/wAjENm8nA7nQuLY0l6Vm6DXGMFDreguPpcF89IKNzAyBWxLKc\\\\\\\/Hgsz2njDgb7fAcIre6Bcr3QLpMh9dvMxDvgs\\\\\\\/3xdXQrcGktRfHiABeJZAj\\\\\\\/mX1BTFgOHwHOo8n1W8BO5ZcCT5duhe55x\\\\\\\/RQGtjas0wOqZgKQdMRDhwaVeRRKsRPqYlAQ9qz7KcG5hpAuTUJXgsx09ZblYQ\\\\\\\/pSMug8aaIsEebLwxfabFbmMr4Irft0yi3+aUYh7ytHeZQ6bt2N4Jc8iifd1mmxycKPqkjyVJ+lyWVa2s8BAlM79uTFNguIkcMxlpbPfpASDUTeF+pwT4jDwGMVbGWuH8m78l\\\\\\\/b3WsjxLf9ZoVyxW051yXtww+9jB1SaQUt4qr4JUW6LdFETp0RsouGHr3XoQ+QVvEnsAMWWQ9Fywt18Nzr42Qlhjiyh3BqhApFxPisg2dCrwswaYqQjXnjRLGUfUoO7XGwOqlPTGv1O05yVdhH8TIbk5T8gKvhFsdiIaIXsCE55+I320ixSliZIcnAR87mk62iQO2Br9wghXJQbq3hxOrEprwWNZC5jukdHDMLdeecK9OjqNvRc68dUdLh3tIr3\\\\\\\/FiuRa19PuXxi841sEpyKmtbUgiImbfFEe3ORq0tpSJucjSKF9TQJU8Az0R7iNd22fttRT+1evG8PfqsI9RhJYubxPcZSYlgvmMG0+ZodbcsvfYNX5DhHb3A\\\\\\\/Acuw\\\\\\\/F+Yg+J+eQgLFQCQ+ah7AOFx1bhdBuA176m9Au8iw0aRxk4WK11\\\\\\\/2w93QE3LEBFUCQA1CzcDuKtWEFWNbSucNpsdUkaiLY1GEtmQba0+CPcdEot7oJ\\\\\\\/vwPa4F8rkRFb4+xSQ3yjzZM0004SV2g5tEEB+Q6PYhxTpD7wuu7G4pBPtEpGn21YWxD4zNJx+Q64y4Z3qISqxF7dtTUzc9dht\\\\\\\/QRwFb5M3nBF+YPmMVuQ+J+EJHZPWCkLiDfyjV9nPxPu1QtjzzXL35gWMrYPV5NP47s6+ixUSDZgOIqM9Mn9bC0Kr1gh28J9fGXFy+HwxNWyoZjarpn8RYjvqan9\\\\\\\/5Xhw9xj1ctidBgx4ZHC4TRFZDDeyVd0N7OgYsjzjHV5ydZYjgSDF+tMQMesU8Q8hWq\\\\\\\/p7ZeXMD8PgYqI9zXEum8hP8Kf7RoSIGSxaZxrQDnDnd7Tl57U8K4nOzQ8Va6Vo28agz8Y+o1wLhLLe9+9Y384Q\\\\\\\/93w8+LzzHMqEPm0ULT4a8t1dmU3DbVKUWonNHPEuuhjWsmtt3teHUFc0akA+Tq2rgj95I35Je8m4+NjE0JkXoI58dzhE9VG0KeS5MEUvg9TJEtt4deCvco5PmmRuWq8rvNpmI7d9SwtvuYwfv75rYCCDtz2GS0XK3MoiqsY7ALBcWOz3OxBnLx8sr3RvFCXKA2QwY5Wx\\\\\\\/Kdi5gudLAqVcO3iKca1A74yTfigOtS68KvR8hObA8kJGo00LpQKtOfHTYpCJmesRMsy2ofAom\\\\\\\/IHEc2W9TSWY6ZUPwW2CuHlFlNDXjdSRJAmN9moMuH69VPQKsoeBqfc82c5Df0G+wazyU0joyc0w7guY1hrPK5q71UhSBh0paV2PGzaV8hpI5LVtrPS9E5HEf+ueHy6qNeMshqUy6fGDOPotLPfNC3T3s+g2PKSkiNv41hracb0rSn+8C4jCVa48xxObwWrmfnoRMDYfgbvjToisAlR0dvXNP+EMSFT8rxh89Dkx2ot\\\\\\\/IaKc2IAnZzSGgAquWYKq4x5LgXoDSJyQUNrRf5VB3AZuiQ3RhigUY1YUnQ6D8conFUBS+eIZPE1QNwjil8PNcbcfsAkmKIasiRRRVlpgiGhzaNxnBYltqmKaAdCbz1b3eG5P1MhFSlgtsDY+6OR9FrNBNeXK0Z6FLT7hyLtt13BHNxClQzWU8i9UUTkHYEWZkUanUbiAn+Vf6wWPddrCqbsLi2FgLvk1OyfHMtTO5qKgzvtA\\\\\\\/WFCH1Nz8GpAZmNs3akSEqC65Egib39oOSOQgWaDLjTBy40xTNAVKfUXfgKY2v0CYBdp0o3QF8ojsse4cSZplXqaeF64v+5WRKdXwcSiIDNBfZSNAUFKwYMhBcLqX0055XB\\\\\\\/cL0tY9qqPgcuPc7Lxw0yg3KhDExdmgVfXIsWPq3ue00Q\\\\\\\/hxwx\\\\\\\/JudD5vhNgb5sUQCSawQFrHytKQxrb+z8j65aqjjxh9NZmZIPhqWMqECoavVCkZKZu+R6MCbuAahJeZuCvb3+rJppsKoZ67nd1P4V8qXOTbrZFMWfkUQZqPWSa9xBpyqzkz11poZwAU3E0u+1swrtrJsBz1eefi8CYtXMeHaOedL8WB5r9F8T+q2zDxTQIheA\\\\\\\/NFhloa4SuewVg78n\\\\\\\/KsIG2QmY+s\\\\\\\/YwHfkMBLI3uDKHESQgp4JgFrKVWrxWJOQS6QbkioPo7R7+9zgyL2QW2S5mjKUfXxNuWUsTStxZ8xkYU7YlHRL72Ds6HjeOtH3nt4dmh2fwMC1kUw+j1eKTqvwHD6\\\\\\\/Y7PTlNz2JrVQNxLSw71gvH1mZrblHc6EM37OQPml77wM\\\\\\\/NEu+3g7ZstmSOPfRkI8p3sA7qTsqEqZcymVYiEYokziEifb9+mCVdX9xe4lgUsjI3tR0S8Us3tPaZ8eWvHr9QQQxBnqvUOr2VOpN2x8kWaHGyCj4sSMp\\\\\\\/dmNo2CaX\\\\\\\/O1GhE6enD3lpC4yAfGdz8R3aZEfzys21D8dufwATBH3KVtPaIOAXyeinaGLgfo9j7aZZfiWB4FxGNsOqtwhsL9Ewwm+YouS9Sv6mSK9kq\\\\\\\/9xY5N0L6fh49uWH\\\\\\\/H6tVAFcWZIovRmKBOJwyrXCHDfK3g92\\\\\\\/o7oGaSD2zuJoiScj\\\\\\\/+TE5nb3gDp1gfi4AfaCMvn84yBjMTUpHI2t6\\\\\\\/Qzxkb74zvDck3E9Myak5\\\\\\\/evpuKv8joftFhoQxUVbcLg+U7xlUMgyX9VamaZAkcdo3+qevXunmvGS7Z\\\\\\\/H7pYmW9qhPuHtPtO\\\\\\\/upZwwnFvsYMpqTSLQsoKEx+VmrQ8CjyIQ\\\\\\\/JiGCTWxdRSMF6sCQDeHivpWgG5RHZRyh7Xwna3fo7DFjs7ujQpnGOjcrBrCFPDyrGlkQuhl33+8BlyZP3DNma\\\\\\\/AXiWAgQWwWJH39p2bXOMjDEZa1BJHgFqzithJvQ8uXYr9aYm67KJN0Pya3wI7t2JV2zPqgHnOc65UwvZ2N+ulc38jImpWhS3RRtw\\\\\\\/k61WzG3ONn7c3x4ZlRPRxb7QJ6rF7BpSbwMUXveVHH48rf1+5oRnRuYGf5gqU1eSLbr0G1DxZPzlkLvt27f3xbhAKVErvHkDsi\\\\\\\/G8ZfhY5cR91i838JdFOt1VIMiXW5cCxoaHqnmeQdTpDI7Pl+fPk5hBjXEZfC05xKI5Jd39foa+16Iz5oW0gLHUVwndfvgK0z8RPsUn7xH2Hk5a2ey6lIctNiXJnAEkmg2ig9Iv7ODoR7zRkishc9L7XmNk5\\\\\\\/DnTpMNPQ8F5oLUnVIrHnQAekSXx5i2J3NrUgw3jS\\\\\\\/D892wbiOVWAGno52a4zzxAOV4KdsQ+i6Kt45BClVDixtw1zlQeYuYwlc+sWu5w2qnjM+tFMTqeQ0aOqbZsXZ0cHn2IDrjJKLCPG1DhoCpbT\\\\\\\/kEx37Muyrv01kx2Ukc\\\\\\\/XAp2fEwLEcRNu97xpUKH\\\\\\\/7wQz+NC49TRvJFjuK18T+rYxhA2tgHBUwDMR4Kkh374B\\\\\\\/Rm2Q4yYTvpglHE0R5lSRmkg6f8KrxwaCzBJMcommmxu77l79TzBgora6Q2k0ziS37v+lmXI4A83G8AJTncsXQesGIMfSuJsXJehM4ppnhiFugqcHsvswm0gkGnDcQSTRS+qYeUQfX44e24uus3F\\\\\\\/jYF\\\\\\\/qypM9Y0yxhyxXRABkb8xvQOhPA0NjZApHHzjkv5cYKQubuIjN5lmW0l7VkqlKC2vV7mtNOVgxnPNNp\\\\\\\/6fpYT+594UrQ5Dz9rcS4L0bC0ch9O4Zka7bJqRGXfbPL3L8xzfIyQ4t0p1tzsz66MvSH\\\\\\\/M745KSnwLiXjXDb+xpAAAvOk6aim\\\\\\\/sDX771CPWURYyN2E9UiMopLIxNeMy5K9\\\\\\\/571t3XVJ54LhR1vxFv8IQhtEWGMK8ba2dIzX\\\\\\\/Q+t8+bzNXaH8zYpzjxbhPzsGMnRca+8rbSweyfrzqLcBmy5BkwJXJQoMj4NOsWCDSwPdLHREOM8rRzh\\\\\\\/tIu07O\\\\\\\/cr5jGp8CJuF3YE\\\\\\\/l7+eMMmPpbUqyA1plKYsEhYz0qBvO3yczzNuOtIxBruzhXWa7uUghaF5GYXIjP\\\\\\\/y26bowJQrSd5TlT8vmeyBRSGjY4nfF+kJes1JBqlUeuDG3Z3ps8DEJ6PbrKSXgCVOvZzfNf3klYxGwDxymYE2rwSRpA\\\\\\\/p2n\\\\\\\/gw5ipaHS35R6Bf0TUgdX0RwqFu\\\\\\\/6WTUngVAsZiHigRExfWDlIdSqyDtBv3VVY1GGERU5Tu241jYna0LF6xEZ7062HWWVoAl2Pri2LybggfH5i5HYBXQwasWUV\\\\\\\/T3X79RICQduZVBBom01HLM0ngnrRvnjPIPEylQqDolt0SArQ1Wz3O8qScamVqYLtBiWzNTkVub+7WAA\\\\\\\/eGfNzLh2ALI0tSqT64jEhMrenSn36kbu5px1F+XeJpI0yRJVAQb4PGoHz0YJbleer0TPHVTecVoiJmS8M\\\\\\\/r9zfgn53sKKS5Re8K8n1cbZK5IxXgR8gjjfCuDKzz9EACmXChfMlv1peOT5G9S1s2dGdHNpfiQTvyVzMGYpbSHE5LYwQyHXjyBHetYagcqLd69aW1djbwnqI8MbB0aD\\\\\\\/a5g3Ly5sZg3MzQWHeCva2ZtQIfvjAztzegacDLgXIn1BbKIOzTFoUBkk+5kHh63nANNWROiyWIGFwlEDb3ygwNZIAMdFb4l1W1xPM0TQx6niivlGBPCgqWvZuYQ1\\\\\\\/lUSLhKddx69H3secoExNB\\\\\\\/ouxOCow1f7wO1ajfKm9AnnMuq4N\\\\\\\/BOhdOO8dQKtvnFAA2CFfTli1AgiwB3rqLzvUOYZLzj4uy9iDCjNUiGJxaXK1gqcN3\\\\\\\/a7GIBV+XZG9XBzhTQu4b8xVnwtvlT1rrv1KhOK4gRQrRiE1JrGlFQ47EZwMVAj9HlH1Zx82g1MZ4LKTuWKe1byqXC5jGIZzWLz0B4eFOTlKQOvUyPcvXLppTdGV8EB7wdFI9ZxkOYl8W9qcWgVvCkh5xATEiK1FD9M+e\\\\\\\/h\\\\\\\/xOeUFYJrXqBhDLeDUzXebsA2gkX156u9wp0vpktVMyqyLcVBsjg71hRU+I3uivIJQEVbGtCfOYoht5fAOLyyE8qfJV7UHAvLVAvD2vd352msJrFjG6VQOETAzMuP1CpqSkAeCF4TfvUtRZiC3cJ8GntDyrgElOnPRWZzWeqzpVZqS0xJ\\\\\\\/umK1NwxGORmDyOoPcVonm\\\\\\\/OWAK4UzG49qu43dn+Vr1ED95jqT9Z1gPuw3ANEGCa889bVAK6C8jJXJg\\\\\\\/FtfjmBV8iqjbu47W1pjv77WnxV48q+w5M2fuSOvFocNrepBzmP71M8TqqIyWrA+foNQPdwW8rACGl3CMRJBm3+Im0TyY8HFrpZOMLKaXS++djCK9LMNO7FGATScxpRCg+cl34kFhHHNMJAO3pCgGh4ECQ5IGIJnEET2xcb7j\\\\\\\/UxAURgeY2uZ\\\\\\\/ivrEuqWbwaX3Jz2DXR9eqo6aDYyIcto5XoH5ELGFPtyd1Xkm9+Jo\\\\\\\/hGok1+XnE9Vk4DEUHhVpD6R+mQ2glOluiuBEBUKU7E+cjWGo6zl24K8eHdy1YDlBsI7+Vc9CGdSWdmYOJb7Fmi+GrWwFulUrVN0KThojgwY9dj4yKfsK\\\\\\\/3qd1Evd6HEf7ojttelX2fPMjBwBshuVq8PAhbsssEbmgaZk1o6xcrsmVy8qCFSh03PjiXcEu2NjI7CPmZimS3plRaZYX4n6hD9f\\\\\\\/Hdw5MgujQVL7tS0NvMjfH9O6kHj33hyV8rDUmH+FSH0d70rYSmDWirktvYEr8FrEeCPHtZMxTDEJaOpkNJuykCyZtVslvygczknc2MpuxuzxLUr9OP+KUQt2unX9DfNsI8yCMTL73ta0Sk4m5haVybjxjdNO6JYZiJxhR9lrnpV\\\\\\\/jo74MZai4ks\\\\\\\/UU+52RTPoct9qbnJl3arqBry0P\\\\\\\/MqOLUaX2WhsncvTKgXWBSEJt4rQfJW5P54wMqbMid03ABh0KNZ94tilHbEaXNnI8kTovhLJVwOsqFWNUOPJSd9h\\\\\\\/g9sB03i8pg6xsPWDNoa50v4rXr6QTABz2PROEkfCGxCr7xByTZiI6fLYW\\\\\\\/Eys2KS1nnrf8c\\\\\\\/+fTsTSAZyR2R\\\\\\\/CQNF4amuPFCLJGJtmppU8Pu5+PR5whooZ+LBYJGcS2LaviePnSgT99GNXWxY8Q9HoK1\\\\\\\/3hnaMh\\\\\\\/B0yFegaPX7zbIEwzQTExXDkGtLd7vqQBPF\\\\\\\/hAWDbRC\\\\\\\/qz315DJSB69D+I1lMOMkz0FG+FOJm6Tz0K85vfTb5YFv2hOpbg\\\\\\\/UBGLuy0KeJFkKQ5yDEwfuakJu\\\\\\\/iHPX6pxboz9x+bvPqcTB+4PazmQACjir59Rjpj8IDvN2xIoTm0zJfCdV\\\\\\\/VPALOK\\\\\\\/C263aeQXNdZmXr1mTrSngnR5hPNs7iMRormrGg2JjZxzF1mPBiOlx++K6NgWn9+WrqtwLK1MR3bxMtiIemeMB9ZLVU4dTWARsD3y7lWkZQQh5RHidP7hDG6Nf3QqD+9JJYkt\\\\\\\/Q8QpAJJICZreZt4sSRGVZfULPwDsBin3rXslOig80Daxi263aNcBwJ6icK3HjZQJ8IWcTNSsSnYBloSDDWlregu\\\\\\\/DAaHU0XrHHINGNjxQ60yp+TG\\\\\\\/G8u9TWvK3AOJCQSC8ZFvQSo7rfVpReQEVnPZ8UC9Fxc7v\\\\\\\/F6Ai4xJm+mvEdJxWTjBm4PAebfMw3VBGRBgRga680rUfHIhXvpyhD7JQ3d3V2RYRU4IX6d2SPM9GAIVXk7V3alMqgyXwl0F+2ByvDqfHDXmIsXkLqc99TuJQ+6ABB0uew2saZL5SLKbsAiMn2LY\\\\\\\/3QqVzcd\\\\\\\/CviNgmZfpnufapmjRZ8LcMB5+qxbiOz1dpVWwE4C\\\\\\\/QE5gulhPvCmGKYhe2gs+3SuHH7eRs\\\\\\\/eBFWKyFndpA4STClrZvoGDeWKozGHnq7UkKgPAXZ1C2VnR9be+tUuLgWU1z40nsl9Y4Aem9ehtzajicTn0zCzU4iXOl3SiJ+Dcg8ie1n6gzs3XQGDmnzhONDKwQvLFxdSKu8bqQ+HsZCxNehHu\\\\\\\/3F428kuJ6oJDjU6MF+K0CLIJD+LVikHd65PaaQUni5wEM7BqwL7E8DW8VCspYgKIfGv+wsv1f84UBIYBHAn\\\\\\\/I5WQyL19HFafqjZXdpI7qfL692W83FgIRHgfVqyvdlHGnwhKviEnbU2L5agUBjv6S0G46IjCPaom\\\\\\\/RWMVstTg58RU4VROOGCCx6eyhQlQOwaSy2VlS9aLFHFac9Ini0bTeJWVVRQ0f2N0WhCSR8O3f+jMP7H2PKGmF9zOWSbfPHz3T5iUzplTlvnCns2xGgy709xmAAdrzKkjqPMLWGxnR8nexQmUT+0JDDK+0sJc0muKqYCuw+i7T2zUGEkIpzYbQKut8FZjH+rZwsw9hbkoGwakcDNG2vborc6mdp1vHyzECIpR\\\\\\\/US9qex6t\\\\\\\/jbF6ZI6FELfm1vHFA\\\\\\\/+o0lT3ZSA6daOs6geLjt1p+MilBRX9EkljBdrpBQCkDB3zDr18WwLhZhxjiZrEP\\\\\\\/iV9AjzmlvZMW9UNOqjbODm0+Epy2Opt90VrfS3pOuSb0NJKyLiVSKOPPaAMJH9nasCBClXayGXnpMGmXCP7bKc5I9pLvmH7BTqQ5kvpyNm+D7H9TbkqXJsFh2V+RIm0NC3bKpLDdHX94\\\\\\\/LpbfbjIokBrNsGS\\\\\\\/0ZHYq91KuEFisaEUgZjHpuzQYo+qy8dqXcOO\\\\\\\/IKuFHVONC5vALT1zf5oLQdInse3nsC8dIg97UR7xD0YzqwRsctrh9de216TXJKYTkj09tYuvPIBrx\\\\\\\/gRW5uHwPKPt6CayNZs88uwa0coLLX1o6dfauqkvyY17TT3L+psA8MQV0UMHhooi6sw20TAHPPbF3MaduJcTgzVMTWE5ZGkV++azSnAyVlDj2RFqeQ\\\\\\\/x\\\\\\\/\\\\\\\/QkBNIOjCRb4yaNx8XFfIVoDdnJUhLBXsWV7doP6RUJhLQYBThA8KG3wk\\\\\\\/7c3\\\\\\\/iYiizDntNplE3KYyLapLF5aPBT2U0JgBQGI3u+Y44L6JJK1eNTOkMggKlGSdjO+AfekF5yGh\\\\\\\/ian8Cvmh9l1eAMcpa09IknaEOgSF4UobQxUShUaDAe9Iy3Eln42mQ9R1RL0n8YbkXjfZPh2+58pGDGlmfofDNkuh+9LZ7YdmrmVYu+COaFjPikM9QPyEF\\\\\\\/9oN5AhGoXmfKlWn0xEduvoFbkXZsn53rYePwey9qsOzKZgKJ0eDK84RSQ8yZIRXOfpEiCAn7xhmSiHm8vlVzgwns72e6JqDne+WL+4H\\\\\\\/+JtzJ0\\\\\\\/xYv6JjgCw7IwILI5PtFgP1hFv5mshWVFjN7kFPuY5bQ\\\\\\\/V8fiN7zWwSYiVf0WiWCTPwHixX\\\\\\\/3ucULcVZ6GlARcz93+kHgPiK+MR883BdXTMbOdLstVyk5iEQmE5Lg+BWoWKJQ2v0PgGJ+Cy1RK+XjB8A6Ave3bjvAeXogoj03gF\\\\\\\/4uA6CQbYfmsc6uCsYsZl36G3lFaII6pN5TrezNjVDn\\\\\\\/m1+9t150\\\\\\\/Oe7y2lsXXPntaU1sRaB9ZSHBDJ13N+4TvhEn10JPQ0ZqwRM+9tD0IhvW69LuGbSVFOwB+36S4dV+921SCpcQetPMq08CKnEArKk3XnIeDkEJEJtY6z0vFQMmrs02V8YPM1+Fi\\\\\\\/E77wqRbhsXRRbAAia5vLyd32utiQNph+nI3A2Y4FRX5llB4ZPHOv4qVR9wm9n\\\\\\\/glGQAd7Ag4ArdXOpcfZRWKqK\\\\\\\/XAzi1rgjuriyS4HThIN\\\\\\\/o73CrttuSuklL5AotRwvpDf6P5dpnDVk3Ueoonz+\\\\\\\/3NSTsPzRgRLcp2XU1UnzIk6fhfpbbFdjYFfNiwSdldbPvJRjhsbhALaMcu5m5zQhaj3Drdhv+j7azCMkWuaU1ZXOr3LJDZwtso3JZkypEMhYz2JMPKNU5PXqIFdHKK99pKABWUvsa+z0n8BdrfdzXIB+VAkq8TaX7onDWw5j5nqJgRsB2lJf34+xUvDtho7Xny5KUrJ7pGxepcO6KOiFLn7BKJqfWYV7vnFZPVew+81ifrAuSig2EXvxjdjUtGg3ZPt4LfAXQtxhpBermBYY254guqeRHYvv6o332ye8KQuT5mWvlHvonQt\\\\\\\/QEGcbAZUYzoi+5aQ2JQGlpU3tHT9ExdM+X7Y6hr3E3isrsaWRnnAdOw3iu7wBPObBrnLblx\\\\\\\/6dj+OJQFMBNKuhPVGgT28LunYskBnzmJkTNp0lualYUCEFpQvBX5a13FUJ4RcQRS61AJMvnaCtrr8\\\\\\\/6vg5YCTBbNlrld4Pfi1B39maYm+5dqfXUa2w18MX\\\\\\\/jhyg2ZpegC4nGM9DfEfBnLW5TwqbWxKnxmrnstekHFdyRv5ljQ5jeaRm3idrKPnY3RFFOfbvnmi72EHi8JIojzkAC0giE76E+IQ8xDjCmul10LfGcPAQjVApkc718xMQ4qWdUoV8y\\\\\\\/wuZ\\\\\\\/iOAm5jVHMgWUTU+nUgXvNKGGmw2WIkyXRmcBLsT9KIUq3VcVfQqAGTJGmEOrZ79ZCt0TWITU2JZhqot\\\\\\\/KcpR3PZp1RvIjp4EUdGEqdU850wCZWEF+O\\\\\\\/Vm\\\\\\\/+Os8\\\\\\\/Vdw1QZgno6ppwiU\\\\\\\/hmIdPkyRPGFeuWwqxroEHOoC2u6oE4z\\\\\\\/Faohtd9dUMVfplXnAIS\\\\\\\/Y0adp6cp1Wa4IX0Tp\\\\\\\/ilX9Zji0loRhJ+hhpFgh2YpN\\\\\\\/yc1hRxOma0GxqQ+KGrG8TBg51vtgjKlHdbMKz0TXHbnVFWlYrrwSOcT71v4sLe0C3\\\\\\\/23\\\\\\\/sW\\\\\\\/FUMfC26yLOKuUYAC\\\\\\\/BrNJCTebvvSbWrYjC7\\\\\\\/9e5Lo\\\\\\\/GwmhZBVnT+WPpDCs+GsHhDkZm2ve6muVRykwJDelWwJ4HeYzTPqPzq0fRphTDPyfzxJ6\\\\\\\/IdDkfzZ4aa9kRCGQ6CUZGKgQhjTFcl8KmzVq9ystXIS7BKGGYviF7HsDgVNCHFt0KufPgYUr8ilvR2ap3r1aNxsck2SwSXMHoSfjXZCAvI5WnzjOiwCTSyLDiiJb4j8NvACfqk7XKZZ+OBK89JfT+yLxu2MlJ\\\\\\\/0G9m0vpKdcxAfm3sqREKaLWcFprbZ8n55PMpQziYv38Mdr1yh1JtKucVS1epS4xB3eORmgYxGGs8EAl1HgEBkuDDK\\\\\\\/jZUZHiw910UeiHEN6w12lipbb4Yh6b92FXMpKdagMZFEP1pgyOxfC+XuHkl2FGdtMgPFWdZ5ME5vUCQg9HnOq2nWlWBdEoHngDE3Gckjo9BZNBb8C2h39ds9AlWx+EXdGwEhXS+dNaMQed1CHNlvaFx\\\\\\\/CiO1iHAk0SY5BTszAGnj\\\\\\\/Zp7DTI8j3H4o2M7g97y7DNbdtXZmo2\\\\\\\/Gu0fdFQp2LsfMIeSFy52K1GcTcu2Kt41vNEBjyo8lDHZr167lFAYV7ypEnzk\\\\\\\/Sg4XVVZ8qrVFZulsU119I+uX+Emj+9NNPcT4BOibwSbVkUe1NHUQF0qmrtE7mc+H13+LFTZdpPocgx4lKgtMfAe\\\\\\\/LIm8rcvsIO4JSuIx98NsfuJt4Vt0ShesKcJzFhyWQuA7rLznBrpjDMSG8EVo+0KgLX4v8ylwoHgEdY8VFnyS\\\\\\\/B3ojCxK\\\\\\\/HprxlWy\\\\\\\/AgtDk9gRs6wWfX0Y+X27VyifrKqaVPf3j9suUARFbVJB+kW6Ul8at7vkLyd\\\\\\\/dBr6eh0wVlJH4c7iFOzXaAUoybyJmo6tz55jJFD8BuaiYYRAyvmhaIv08+p9Ied9IwinkwfWjA1C4X6D53RBZvIEGiEA72rlvYvqZW4L68StnjcFNI3ZZsJ7twhsZuy+WeiBdZjq2+552SiKc+6tSYcHGFmiejQZOY2x+e1j620xg\\\\\\\/f30Jiq3daRdL6AtXbabY+kUc9dbnaKUumX1HSVjZUxWkqbpiRFU3zSigdZyDPOLiuk277E1VeR6Et4v0bXznoYZRPN0e7Le+BsENQ8bT01hPXWuuz6D6s6W9YZZrSnjjJsp2K4w895+0qGZrSDGu6NgiTkcPEF6oibgl0OPzCZp0B\\\\\\\/dBMGhFbDH2oyinKg7OEzugTT8KGCN5Na64mevheuqAkaEekBayjVOGF5MeKOaXf\\\\\\\/xEZsPLZGQxJUr4A8oBR8AW1g07mx7YZ28SwOLSSKF7xXwGOU3pNqMHJEw1UYMQ2yU5pCJ08+uE7SY2MEa3GiFolrGwF4eh1Tr93d4biXUyPRUYyCZEtFhRLPI58SXreiBNEfAXC9Yq3k5JWZiU\\\\\\\/sJvnrtQJ8HB67RvaAZtxtiNlzBmfrEXXFGn0PlXk7d9Eive6\\\\\\\/c3vx5SOPJOsqpSx379mhM3hBC\\\\\\\/fq4hOu4LncoujUf1ikywYDW6rh+Z\\\\\\\/2\\\\\\\/w2JuyWrX\\\\\\\/yD2fLp7RSaZMkEPBbNvCgt0vMGLlpUNm9bsvAGBnEwis373PnLSKPSgGfbc1X1rSdXeTcsCq2D6to7fL8cVrfySB0MnvXjUHuz3ExxuGtANJiARbOciocvieqLrLd0v1\\\\\\\/tUlcnNBqxUDRrcdr+W2d4NLuS3wxQnReN4vvbMIXOj2NvoQolTMbyS\\\\\\\/MzjFZ1kkHwlqs0vZkujTg66v2t08GsMDNj+HY\\\\\\\/dM5h4P3zgTCz+sbjQ0Ml1zVF2IUf4gauO6fxniihYJaFrPrKmSCGJ3D3uG6FDVF1BvWJWG8bnRuvRv6POGgVJAkOLDhF1Y4XJZYEhvtxGzO5sSaXP96zymnKz29wow\\\\\\\/f2xoWp8pKFPtrlKuCIPNZksWBPbiJGbQzu5kBpDEWoTNKcXiikIfGKrU1HlEQaJVudnrYm63Le9h7HG8\\\\\\\/rruOVE83bjAIh7\\\\\\\/juvZdVw1tzhhOj2SjfevOV+\\\\\\\/QpSaJijwP\\\\\\\/Lk4c7avB5P4RcF5HscaPoKOwGCLaro4GcBKTr7uPt348phq9cAiD+\\\\\\\/SohIe3HON2\\\\\\\/9pntD5U1tG9TTbZPJqoAWQSzRPj\\\\\\\/oKyl1s1WXx+ukYreStTo+8h1G8ukgsYt3UKEpwLKFabin32hVl0unkhlt1e8s47isE2M18Atlm7B06U4pPHd5Tgl+uXuzWkwr59cdvbDaUzguYQZeSEmy2AjqgEK+xAF24G+eq+Pxe50EBL1eEiynkYr9\\\\\\\/9CujLMkzhJJzIc\\\\\\\/JYtH+7iBRAZ5f7xagiahf732akW+96nPEoAyZi7Ql+bHFVdoGnH4uKOEQ2O5D78xJR3tvAIyLqcToxyIvfHXi6HyNJw502pEweiuHn9JkjWm3jsmc24XYqCKnThvK\\\\\\\/30rJHw06lKv7bf2nSn1XP4CE0IqoaV2whprm+4SH9nglHersgLUhaJlP5UEf4hOyXzAPdYFEAL+4x3bGPIrKSw1Zlw2gX83DTORuk6JCvRqmj1GcGN5a3D1J8y6tRax0VUV2vvv28mViA2cyCizc4D9a3Cup2NCvz9xiwQKBIA8mLCS0nC832RCGtaSQMTbOGhSlZGAdAbijNJR355ZmmHN\\\\\\\/v39Q48p3h3eH9bcDRxZYTaSC5mQcVJuXWKbES5JjrmDCRb9LF8XHEIPo9qILVBzK5NSgbYczCDmdxn0Mhofma0vU7XaNV+vkf\\\\\\\/A5y0TRonPoKYbDmC5CDwdmM7VquPe1Jht\\\\\\\/YFjdTHinwrqJ\\\\\\\/hbnyzwbx6EvoQcVJ6OCyD0X\\\\\\\/0nAWYGiyt5nhiwYryZVg2kZP4cNtiM84YGcNTlQRKMOi4cfeMm5Pz4KiMKRJjPol8obxIC1oKfILXPVJ7xrXHznSlclqVuNFceYnaK\\\\\\\/I\\\\\\\/NHT79pwViE88XykaGicGP5e0r59ny6Yc\\\\\\\/WTwktSGZQL3HRnoT6cx8a9qTsiM2JCVw34EzbparInX1tTUpMYEBq+nKrf1AGk+Ep9sZjDILur06Cixa5IF\\\\\\\/s6FY4pz3bJESblv\\\\\\\/UcaDNyq2dGnkoXtxyFgy6VpOabYX2kobGJAUhLS8F\\\\\\\/XbmTI7qNO3Kz57TIu+307QSYvg64XIGxyiPpbR1adX3tPu7cRtGWJCuXACpxH5yyNBEq7X6Gcxd9+qDjkl4OqiARSHQDreojX4NRn7qIcbg4DHNLhVjC3MO7e\\\\\\\/K\\\\\\\/AKQKTg1R6E4YnvBpJI\\\\\\\/+q1FBOFVDOjx6GtF6pa9PqL4cStZXdVIfvDOW1V3ZAyCrxl4Lh4wsdUzfEmJbl6qx+6IC+McC5cKupADOycqLZ582JwZSW7TPqEjzobEezhT4LtkT4C8wsyPp+skDRW5P1ojgeN\\\\\\\/dk\\\\\\\/98pAtOd8687qdaIsMf4zLwu0nHp3b\\\\\\\/iIc979faf4YtDurnHb807XKgeKT+KnQrtBMqmUbD2zH6QJtE7OcZg6NZJZPzPuOcE5493Oa5q2JNrhvmvKkpqrvpndWMT+ixk1OBQsAa1En+vLkV5\\\\\\\/Ut60CuZ\\\\\\\/2+OyEuRynV3+iZQTYPUiARj1rpyuM2f8KYXDsq3HU8AQ2YYbtTmzDf8bFGa\\\\\\\/RVRL284MSUOGPrSmfO6TSqnUKdoTrOmrvsXKDWk3jB7qk2NH1KXutdwTWQcVpvHeSVXSFwURDiKu9s7jcbeS23WNu97KXutov5p2YxK1GtI3xyaUP+D5mo3sY+0ZdHE5maB\\\\\\\/ugzW3uUg1zC4hmAyaUphQ23Z5sljbRsn3EAtTMNdD4\\\\\\\/B749ZFSY5+3\\\\\\\/p33YUyFonTnQKJX3xtKWzflNLNSpX7mxc2E3jU9dSzfg0S7BJ0E2bVbZzSlR6ycwoBFGJvWsH0wJn1pbh+m+HRk3JaL6iddbICpDqV727L4qNkywYsxLCMAlsULYf0MEbbp9sgGKK+mrDuidfwjsZDP74PuPgNpHMxGaRZWkS8PL2h3BZtzGhaWYaICmSnxpdC1kHvuZ9T0z4OKGIF1wDLaWpbmO6mmlqGOhKosVz6CHKZ7au+w7v\\\\\\\/Dbtmb2SLbKakV2LHiVBtQNuMftBDmoRu9CfvKYkqlfj\\\\\\\/qYtjt9ygJBRPJnoJBUDvo5ABsuclRw4gCcfN3cTqECY3eSUbqHhSUqYozN3p1kIKgr+NmH+i4PPbatOq2\\\\\\\/FpGpObEppgUNlhBlnPWJhmUu69TJVvbqIaXxV5juWTKyS3AqTEnNJ3BwZkW0pmlTdwaheAs4V+bjwk\\\\\\\/9wQof8i+cq4i4487EkoDuKFv200at7ziFhx6im3Ro\\\\\\\/OOGBtQBL31tP7Vv5j9JL4vU8ngtGZsxtEAk5SGGf3ashIWumQ\\\\\\\/PclalLJ75r2TnwvlOU++oPej82vKM+dYIST\\\\\\\/TgAaehOLy8UCc5birATyaMqxpL19Dwsb\\\\\\\/1OdMk\\\\\\\/xGPliSgIwxoMFcLxYYF4H5Mu7Wgq\\\\\\\/uvwwtXTBl9AKk5656wmyEc385CgQ22lKg\\\\\\\/F8rMgqy3I2m05DlOnkejjKzHXp0l4x2q5Oxpo90amy6Da65LRFU0CBSALg57oM9xAUaHO0PtJuapOE4DLSztS+eM8zESFwzvS\\\\\\\/ZvWofjJ1LAU58W9XBxjjPcRFRiFsk16VB0l+knGmDCWQApzGXdsVq3RYAXlALD4FsScJl4vY4FIWoXWKrSzkbNVfvI21vWj857YdL\\\\\\\/wIg4YNPhrY3BP0FHLH38XCoyEbA5KVMXbS\\\\\\\/ZyO3\\\\\\\/Vs+U6OdHCjx2K1myls9J1J\\\\\\\/+21s9SqjNM+ZHlEVihjdQ8tvjT\\\\\\\/zPbK\\\\\\\/4WnhCexyP448IwMOEXpGwTfWvgQiG+eVV22khRiehyQfScDMkOC1OOYp1Vdr6bgfi52lkoSLTPs\\\\\\\/KzIzSBm5m2LVLkdoOfSL0EY+15E+xyRL4s+u1JrlmFB8y62Yai6zahd9DtBRYMNuHwq5z+AqV+zyU+kDuTqLon91fcVNBTF9PjjxGlYHnBo8Ww7CLpjDgYckOuvPDZ\\\\\\\/oKU499ATVgsVssPL7OvpHY92P\\\\\\\/Gs6IslkQiEjxEAjgRZkWbfWoI\\\\\\\/udakl26W6n\\\\\\\/g3EKkyqfO3SwQBue42UD\\\\\\\/YVnqINKuiAIZ0Qjmdv8hZF20sH4F++VkrSFaYm1LpgdwdRxgiIacL8W7dsfQf+812rCvZW0ku0vsaErXl6cwr6gD9FZAI6ftyhEkJw62t2zyEI6TX5BLb96pQbvzXDg\\\\\\\/C83fuPLH\\\\\\\/7PY\\\\\\\/+oz\\\\\\\/87FJBzgJJnGNr\\\\\\\/Sjlu0F2y17z7HpzxQA+Cegea2KWM0MtvHhjfnWCd0DSu1Eh0wSMK7YSLCsI0DfrMYDt2qfUXAS\\\\\\\/ba3Z4PRYfIbYMwGuW\\\\\\\/EQLrFdxfzVjKfuc1zlsi2edV6a1lw1Zhyw15echvX61f\\\\\\\/Why0r4jsKk\\\\\\\/oisXe5mZ4gF\\\\\\\/Q\\\\\\\/7MjjVA2zxheTOQHSGVP20mEAwsfKArPDoSLI1zDlPYdSzdrwMPFcxMhrYFraNhpz13jemUNNq9M3HoGh8M9bx0CijCvLd3wCI\\\\\\\/uL62b2DKGVdvOzZzAutKrmkXQHXswVLijI7GkVKIHbEuH\\\\\\\/HGHpAHs+BjKacvrKLdRpw75fHhikGYBxlLWr6wwekyzQCM+A\\\\\\\/dbFbU3tEkadefi+ncj0qO\\\\\\\/5MTjxqVqbgz9VyHKf1e05yHLS+hpW8qsCyPQxfz09wxrUD+8fqtTL0e\\\\\\\/16Rbb4E0OT7JFeD0I8uoDOsJkinrUiiFII9uaISCBsxusJSUWVVmlEcNEtoiBZI5upXvBDsAYTqmjFnOdJ2VH2FbsqMrP5GEPdSn1VGJ4Y+PKfDLurfQVYvhan+1QKkFDlwDzV3u+Pg2YdEDzMtH44yvZ2\\\\\\\/FoP9jyb4BMpNh3DCZWGPlhdAt6VZSyjUSPxZiuN0aklc3pzDEpWGaGSMMVceb2Gcf6qLbiO9FsXfzmJAJXD9bwxR\\\\\\\/ROMfWjib1ekz4cgD\\\\\\\/J80hVquMEmHYDiG8smWI+px9Inmpjv4JaqQmNmnoaXis4yz0Rg+dJtYXOZpontQOXNGjy9gBFY6JRXRAmf89UsnDlcNEGjzwGxwMAgP4WhObDgvpzh+pbooxjOaT2JZX+wIlPxGq7DDVVJnYHTtzHS7qYi0ndC+dI8POo2TOm5DhaqJCOCaozaHcnKp\\\\\\\/wOxd3+NJUZno8ybv\\\\\\\/zSWBxOQc1CZtECM920rckNVQdh3X4ijZcbjmnGrE4vMtpfZTr3uot4OGXr8IZ7\\\\\\\/jNZj2DxVeeqQyrcwINHBZhShyfPAJvi+MPc7JGiR6reqR8ivjvHY6kZoHo7lHppOmQU2W6BZZC1ULZGXScMMPw\\\\\\\/f4S+BCAH7O2qNQVMqxdFab7NP\\\\\\\/m1e7utLRduRd2K2SuR+fMz8ggYA6egYgtxK3eMXjSkCesgsXUP5QPdQr3gP2bEH7JzTegOJR0SpEIhYoPJ5QncQLW1v1DgQQlJ3T7ArDRwM5sfQfzphI9E0yfjzWhhme7+wPiIkwC8Mcc9AivYgvXzFjk8F2kyVjyTYEAM5mLKAP1GHkmRpnVB\\\\\\\/8+c9gBHqX9\\\\\\\/KjkPyGbS+DjxpTDhr8yvVoQ2gcbU\\\\\\\/Wkl6AbZEH39bI3XxX88j3lBT8InpwbcBEa4vr67gNafXyY7xHJ2fhVGvHIwZnQcda2ZV8vFd\\\\\\\/oQnd7glwvpByPhuXGYsCGFybneiIVE3E5Xb8TU0B5TtvzuYqsga24hh6gaC6Ur6PzEm8oUomxO+EIzyUYR5+siagDhrM5fmU58GvoIcbNc+VDvkDO+WbeuBoG5dkhGFWlUkT6koPgVL0k0hUru44kNGMbyYEb1g69PQFh6BLcKE5+I\\\\\\\/fGZMJQhWrr1JbRBh9k3MPm1DsSHr82wSM2GynIfgDHkYBTKJDumxwkVwHZkFh\\\\\\\/B7Gl5ps64OuxHCiPfkgfhfpfogdTm62aOVzPEbvuOW97K7O6tbZgc78sc+xsNI\\\\\\\/FyyS8B4OV5drAxO3m8OO7vXW9gEUhbNZsEglfxiR0yOmKi9hKZaZCdthOc6\\\\\\\/Q2mIkAlIKNx7shp7xXhLDIBVLBw9GoTGxRtxcbUzvZNmtGG4bZd3HfydhvGJoqqF985MgGLYe8rzzLVfcc7scbzKtdVYZwE\\\\\\\/7tzrTFds6Yq4MgWCmIxkeHKKrYhHHd7Yz9KyNvWBc8d3LBo8cXCJXUVksvx9OIVBkRu39zowYPiEUs5lFhufh0HpPXNMxLOydJqi3nw\\\\\\\/z2vfFIgiiG3TXXeSSw3ha2ycgK6hdZ8\\\\\\\/pah79k2PEUpxOwUTAs24De4gVFDFv6sJXKN8K5yhsEyOpPJA\\\\\\\/BXupHoBa+d0\\\\\\\/acWn2lwAPpjvmE2bBllPr40ueC8b4BTrZ9usuJtNdlI1mu4du41o+mG\\\\\\\/f09C0TpCS5W\\\\\\\/+mo2H00CbjE2QWsLsq6LZqt606zoZsgXreDIyzDsQkE2EedVRrqmzIfmZpgQFiPFi4c3CYpt3tMaaRu5lFmwMyMpe37apufpsZdMEmrZY4EcP55luxO7PZ4Uq8j+ZQjJ1ZS239jWynpT0yXgQ9ThLJq8AzeqtHEBsvfsPaCj0onBooyK7I4ESyTvxUqWrhDXdFh8OC1p2A6GsNUla0akSetdgyUpaI5kYhlmLLMc4iKBkN5a4hdYHgQ7Q9GhEzn\\\\\\\/Nnk8YaUhfUqiGD7fSH8nXrrkclpSe2BkzLuPnmfSur0viFdx4Yo2yZ\\\\\\\/E2EYIEBbcvEcRpD3LaKhxfQLyyqIalROCi5sjc1muxoEFxkpStuUxq1VVU6sBIjFsZPiLYG5lnCc+z26u8+Tf8M+MROmeZdoF++vFFDd\\\\\\\/DlUh5Dh21Jtbh0nGSScKSC4lgE8DGkft481xIVwQiUNEWZwJFC0G0SpAjVDWdVrcklDxrZxoeqy6k\\\\\\\/tw8lbslrDApvq3gVGNAzEPQMCVO3It0UbMpYQgDfqayI0ioeLJuA9Hc8HfB9Fom2ncxX5gGNr52+ncQZsSYClb\\\\\\\/3b84qMr\\\\\\\/0+q3udUyxyn9ppBFsiWn1Hp0JRC7GqOYnS0eI7sPAa+0OIE3pgsUTZKunVTWsXCpUh+CozdUB2m8Q3zJ8uqBDthzbZnUF+eJ0Wf2xCxiVWLaARjAUoxOIsJhU1NodlJkKIX28ai76xzdX6WWQcgtn8kPzYK5MwX5OuXjPZWx67D\\\\\\\/ajt60duuKGvcerY322a\\\\\\\/Pd\\\\\\\/soshU73xaxEz3bfoXQgSsGxhs0ZOCmVQZbruH2ZiU+HwO\\\\\\\/6pseuZKtMEIb2DBoLLlSr6tAWPwn92NDk\\\\\\\/htyMnCOq6pxwHHbZ7\\\\\\\/SOVFISFZnHHnfaaW9+3x6TPC7YwqNjQOuZhEy\\\\\\\/9JbCosk7zZtvC6AHhJrYYhwvnGnNRz5O+BX0PmpmXx9JOPOCt7f\\\\\\\/VIm+n4BhD+diJvuAnusA7yQ90KyREZHTh5yLMuWKhGJHP7Ppd4SYtjbp8rBWVPpRYZDjEm+JplZMz8190pHAIP6TmjRU+dS14VyzoQyeF+XrJssKKbRT68810IRWVYyArobjuo92plsb6o4UBmCatXOeAOkgssKvfs44NzGKAgA8Wn2oFLqpcRqZupM+99TiQfdwQV7Co\\\\\\\/QhbidWN4SedWvYAoxpqC3qn9IoItkN3emz+TxISYZmmCKbBg\\\\\\\/PLn0PkKx5h84MkQrno1NRMumgcLGx4Rs3AQWYOMNqoxjcHoNdrL2eizE+uo3fLEWfD8ZoxXgIKlvCVfk\\\\\\\/ecmqwRdNMjxk3KByk6KDecUUvjhGOulQYOd1ahcM5bCNdww9A6U6CK2scvmkZ8zNdZYSkAE3hkxbzCjCbUWdkW4cxOgJtJ0Y0AuTP6yvor9SpkKWhBNuHB+MA1IpsNZcY02WdfJhPv90pMN56Si9821RZKtkDkut+MTFlgp7KEWxN2JyjfF1nzQQ0mjj1clv9fZgi3vUYXMyNVmrAqeqGMK66MFgHKT3b8eGTeUkz2CY\\\\\\\/fok1QZoZRQb8taNNvhdoDq5E3MqDyw3UDJivSIr2hFx4LTSZqqIS\\\\\\\/snR0NB7W360ic5Fm3ELWAS0wbg3swnzMtSik\\\\\\\/HpbajqPHupszWrXEH4A4pxgu0\\\\\\\/10qmSZdQtrWYF8\\\\\\\/CGNbepRGS\\\\\\\/Mh6QWnAPnOFNA5xVn12815UsSwNkN5eosLjFCuAqoAPsjsLek59Pteb30hfUHQyl5WSakUVzWg26ldXKCwf54m50Po\\\\\\\/Qy4TEcZEfu7A78f31vocuBaCP5qeHFkGjfuYqNbA2kIkFKHehe4K6mJJpBILEhjuHn6tYTIefQ+Bed3twu8OONuJc9lUHAg4CvhAhti5866oZNQatp4DtLLD6OFP0OXwqYYqcX1Nv4sCXDh7ofu9VGlEEhaHXmidaXEmfz\\\\\\\/OuxWwMERpDyjnmpysQUjcNUzkP38itI8okAdXBYRflkN2ZwkkNOfuWgzk+UxxNDJWoWb6q8tC5XF849qervHKa8XlwHZDvCQW\\\\\\\/P3tRBBate8cxRP+Tu56BX0SdOAm+g03Tw0JOdtN+zWSYLJFEEUA+f9aiE2Cy64zsQcbmfyxnkuqsE5Qz+GE2DsWIAIeu26sUgVURy6m6XH2Iz9QRlz4g0QfT+QOWdChviGk1Sy13Uqp+87MtfzCCgd3cv87P9F\\\\\\\/zNYL8P2Y0YGRzzpoFlHG\\\\\\\/SCO4fcFSXpjzu16Hh2YWG9tp5eJSEqcCswwgyzxNsP7CSr\\\\\\\/njHF469oLYNm0Mh\\\\\\\/pyQrH21eQIh4S4lMJYA0TFISN6Ly18AtyIUAaWEVHJ3fcKOFpzBLbqSrfDSDMyLeUr7IvBYOlbUC\\\\\\\/pjtty3n7sD0wi0Gpf7m1VlE+X3k6GdAaUL9NETMezQJe0numZSUeK4GLIAugqs3b5tw\\\\\\\/peNpftdjSbbjBmU\\\\\\\/R\\\\\\\/Ch0iPV7bUxfIlydeKVVh6EymAFkRjNs4JToRwr4wEe36tognG3ZusyWtS+++Qhayu16TlbnCBcDYQKXHwFmsQ8pEM9kW+oVyeTjkq4yvNsMJE+Ot9G0uJW2izJWadjDlNHtmICl6NQ6ytvgUHzEphTGViJ\\\\\\\/mWS+LPcUPNQ5Rd9G\\\\\\\/CR6GS5pMeD33\\\\\\\/zaGCeASzQoqXrLL+jkqxHlkOGtIqu7OaPSOyLa4vnDRssKz57oJVg3bPYurT1Zrr\\\\\\\/46EkGluQGyRfEQx3aj84yZFNdV\\\\\\\/qLDD1lHXTYs+akyvBoCVjJND\\\\\\\/58J3PVQ38W7IvYhRl3\\\\\\\/MaQ4HgvU8PO91CAC7VqO8fNcgT6ogfB51EoDjUGtDXsAwGBroObMKwQCI9Z0dhrqkP+4u8i21w9zaSDKkIU8FNzlYfbrmWBkBuHeQwm+HhFMGwVqXdIQXlBPJ2NLkrujM6z5Y4lSnbOPmFGU0qD6eTOKX5f31vTrJ8sa2Y1P5G4SxHqA255wfiw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022y6yD\\\\\\\/ZJ\\\\\\\/mTel\\\\\\\/aUGhIQuMw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d354b90df442a9e564505663d993749d107dcec8755a8dff87fb56a4c92bb3e6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229Pxk09o+Fqk7LeZ2P\\\\\\\/sINfBcMtfwlSY9LuJkSzzGYZKDzcgb0CP4PtEoA\\\\\\\/PVVXn7D60Pf+SKODflVUXF\\\\\\\/6KtFMOaTXG\\\\\\\/rp+kTPDc6K1par1CLFwEufX+YS4CjxFmWPKfSe0Eoc2ZfgeYfKPc5JKDr6AGiZbCDdiSbGLj3L+dtkDhtAhZeNfgNirDjH8iLALdVi2njwDbtONWn9evpBpm0tqwS6mP1bqtAcLyHhCbb+sTdT4dluiq6H7suSFJgIi5W712lfZktpjkRHlsQDLYAQZ4ulGBTwvutkNODWsFJV7\\\\\\\/tkVgcz4W4JuXWFClxYV1r7P6jUW5SJBoW0ATYa\\\\\\\/Ca7t3Wcyw5zzBpPMgxRZEcX+hDTOhMaVxZnFeLLwjhrImYv9MVaA6549Zt43YAQwsqUrNmsM5BsOJShwUzmFXMBPsEixvC+KDoZ+iEYzGTuZJ2Of26uU51BqksOl8OrahCHXsXx6+92eOUZ+t1cQBczP4mpzn0V\\\\\\\/dOdz4DYZa0Dqzijq200Ljc2h3s8q28FmYXWj0bCBJ3FU+ags0uc+Ry8vupl5eQLKgR2Cm0ic4GXh\\\\\\\/xX1rml\\\\\\\/aty+cIPVHZnqiqStoPLGvjm0\\\\\\\/vCgXi3Tt9xUIh6kI2ykqrrylEFCLyq3B6GLTtZ\\\\\\\/HMjTRFp7mpksbSXT+70PVZpexanboViY78kZypJ34fPzE2McEnukFDLiAh5IUXuIPccG4B8jbDN3pFCHHgYGw7SwqwLakZ5I\\\\\\\/KhVDjCJ+MXotBChHZnarzkqN5u37f0jUWrjeFYd3q3CCxNP8U8zhydUhzlx\\\\\\\/H9MpqXXPes3AIzQiWQ8\\\\\\\/iMTRJiN5z4Ca3Mx+s7mYkO+TPJgUbGUj1y\\\\\\\/Q3KzzIzEZnQvHPCu6uMXybX4NXd8BAZ4eoOlecOQiOo982N+8GpqsCMvbiS1+xU+yVuA\\\\\\\/46FOrD26GD9Tp7Vme1lHuaKpZD84Nz+f4EDGijETuHxQxrvNPvBZc+HeQqCd7vYdkHvTG2fOZMSj4Bxef6YU9IYcioZOmaUIxdmkuDgDgJbiiOJDq0AMsfL2kpuj5UsoRODYGS0E1jfVpi2+6HCjHyhSWta34EaXGbUd5LK6P\\\\\\\/HSZF8BV+rr2sE9PCrq35WjrduwRustBVXaizUitB5B\\\\\\\/1hIR\\\\\\\/ht9LlvdjPPCnsMsYnydFtslSACoKDVtG106eeKqFH+dQk66HBkELq3miauGSCJCm6dcL4pbLhPto36LHDeL37b2DGy4lUYJumAeq1pP2OuvHNAg8BQwVpnDyUbO+TI8RV0Fl88aDTArF8pSz+qX7ju2b4oYZa1vXpxC0tIVBqkSVLgUW4xQOWw7M\\\\\\\/y3G8MIe0aar8YrqCQZQHr5osVfxQ1FeLtkWx64q2C0ggkcBqOxrEKicN8cAqZQSwwLlDJx9Kl54+XVZ9OCgl\\\\\\\/+qcCIQ6Fhu6\\\\\\\/7BPRtWrShWmsVHAE3MJJGJRfOQ\\\\\\\/9MVTuX7EL9eQieQoTNuTdh7Qg2J8PUdr+lVbMwkPGefXtko4phQztoV1Z2YyvCA4e002jhy+5O4c5gxQHAx9Uv9ekk\\\\\\\/Ky8Lc6bX37CisjV6i74dry6DKjoBRf59Zx\\\\\\\/AJ4pqpTVMLSuNE8p+7XUObhYeNyChBPmuLiVlDNepsrAZqCJ9TSRU6NBoo+4y\\\\\\\/QYoPxPqjq04gCCxIZa+qO05Zy81PNWz6mJYdjRsCmOO\\\\\\\/FfzBQNimR6Aqgc\\\\\\\/P2fEwrZOgj474hRe78Zz7iSDU+YdkRnmhdbrE9x56vj06m\\\\\\\/gj\\\\\\\/g9buDm8yrSVH3\\\\\\\/2Ic+Gn867vSiz15sQ\\\\\\\/WosNNMiSOLCuS4i1pBFBFCvcwUYBfHW6tYKuvy07FlLJAM2WsYTqF9UhoWV6RJirIsOvD8vhJx\\\\\\\/WQF4UwJ5wC8VA15YverJPIlEPedqjXXp3\\\\\\\/7XrJ9VTDTqU8nQx9h1ABQ\\\\\\\/bL7xEJOE1CDVZhOBencEAZqrsXVmCnKak6QTLCK2BS0cVyHI5jnw733lOMPnMKuYRwW+2GjlhE1F8PpriFb4ZV8zV6g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022KRuWhtbD8uREAO09ZvCQLg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002241830064c5fc84b9904639eef787ef818b351f5bfd0450405acf2eaf12af6cf6\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460406 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2033" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tRWXTikkNrFjKqhtf2r-zmbkIq9CST2D2bZqB2F4Qzc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2033" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Lo22QDCRxDdbk0xGyVMTdGlGxFNeoREqUR7N8kaf5tc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/2023" title="Functional Genetic Screen to Identify Interneurons Governing Behaviorally Distinct Aspects of Drosophila Larval Motor Programs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/2043" title="Exhaustive Genome-Wide Search for SNP-SNP Interactions Across 10 Human Diseases" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460406" id="node460406" data-pisa="ggg;6/7/2033" data-pisa-master="ggg;g3.116.029488" data-apath="/ggg/6/7/2033.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Budding Yeast <em>SLX4</em> Contributes to the Appropriate Distribution of Crossovers and Meiotic Double-Strand Break Formation on Bivalents During Meiosis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mika Higashide</span> and <span class="highwire-citation-author" data-delta="1">Miki Shinohara</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2033-2042;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029488 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460406" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mika Higashide </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrated Protein Functions, Institute for Protein Research, Osaka University, 565-0871, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biological Science, Graduate School of Science, Osaka University, 565-0871, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mika%2BHigashide%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Higashide%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMika%2BHigashide%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Miki Shinohara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrated Protein Functions, Institute for Protein Research, Osaka University, 565-0871, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biological Science, Graduate School of Science, Osaka University, 565-0871, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miki%2BShinohara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shinohara%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiki%2BShinohara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mikis@protein.osaka-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">mikis@protein.osaka-u.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/7/2033" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460406" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460406/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2033.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460406" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460406/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2033.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460406" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460406/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/2033.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460406" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460406/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/7/2033.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460406" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460406/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The number and distribution of meiosis crossover (CO) events on each bivalent are strictly controlled by multiple mechanisms to assure proper chromosome segregation during the first meiotic division. In <em>Saccharomyces cerevisiae</em>, <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is a multi-functional scaffold protein for structure-selective endonucleases, such as <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a> and <a href="http://www.yeastgenome.org/locus/S000005943/overview">Rad1</a> (which are involved in DNA damage repair), and is also a negative regulator of the <a href="http://www.yeastgenome.org/locus/S000002625/overview">Rad9</a>-dependent signaling pathway with <a href="http://www.yeastgenome.org/locus/S000001197/overview">Rtt107</a>. <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> has been believed to play only a minor role in meiotic recombination. Here, we report that <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is involved in proper intrachromosomal distribution of meiotic CO formation, especially in regions near centromeres. We observed an increase in uncontrolled CO formation only in a region near the centromere in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant. Interestingly, this phenomenon was not observed in the <em><a href="http://www.yeastgenome.org/locus/S000000432/overview">slx1</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000005943/overview">rad1</a></em>∆, or <em><a href="http://www.yeastgenome.org/locus/S000001197/overview">rtt107</a></em>∆ mutants. In addition, we observed a reduced number of DNA double-strand breaks (DSBs) and altered meiotic DSB distribution on chromosomes in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant. This suggests that the multi-functional <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is required for proper CO formation and meiotic DSB formation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/meiotic-recombination" class="hw-term hw-article-keyword hw-article-keyword-meiotic-recombination" rel="nofollow">meiotic recombination</a></li><li class="kwd"><a href="/keyword/crossover-control" class="hw-term hw-article-keyword hw-article-keyword-crossover-control" rel="nofollow">crossover control</a></li><li class="kwd"><a href="/keyword/chromosome-segregation" class="hw-term hw-article-keyword hw-article-keyword-chromosome-segregation" rel="nofollow">chromosome segregation</a></li><li class="kwd"><a href="/keyword/endonuclease" class="hw-term hw-article-keyword hw-article-keyword-endonuclease" rel="nofollow">endonuclease</a></li></ul><p id="p-3">Meiotic crossover (CO) formation is essential for proper segregation of homologous chromosomes during meiosis I, and the number of COs per bivalent is strictly regulated. CO formation originates from <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-dependent meiotic double-strand break (DSB) formation at recombination hot spots (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Keeney 2001</a>). A hot spot is generally defined not by specific DNA sequences, but by an open chromatin structure and epigenetic marks of histone modification (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Borde <em>et al.</em> 2009</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Buard <em>et al.</em> 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lichten and de Massy 2011</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bani Ismail <em>et al.</em> 2014</a>). In addition, whole-genome mapping of meiotic DSBs indicates that the distribution of meiotic DSBs is not uniform, and there are regions with few DSBs (cold spot), including those near centromeres and telomeres (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Buhler <em>et al.</em> 2007</a>).</p><p id="p-4">Meiotic DSBs are repaired by <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a>- and <a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a>-mediated homologous recombination (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bishop 1994</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shinohara <em>et al.</em> 1997</a>). Zip-Mer-Msh (ZMM) components, especially <a href="http://www.yeastgenome.org/locus/S000004386/overview">Zip3</a>, <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>, <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>, and <a href="http://www.yeastgenome.org/locus/S000002313/overview">Msh5</a>, are essential for both CO formation and its control, such as CO assurance and CO interference (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hollingsworth <em>et al.</em> 1995</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agarwal and Roeder 2000</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Snowden <em>et al.</em> 2004</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Shinohara <em>et al.</em> 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Nishant <em>et al.</em> 2010</a>). In contrast, other ZMM components, especially <a href="http://www.yeastgenome.org/locus/S000001335/overview">Spo22</a>, <a href="http://www.yeastgenome.org/locus/S000003218/overview">Zip2</a>, and <a href="http://www.yeastgenome.org/locus/S000001196/overview">Spo16</a>, are involved in the elongation of transverse element, which consists of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> polymerization, during synaptonemal complex (SC) formation (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chua and Roeder 1998</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tsubouchi <em>et al.</em> 2006</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Shinohara <em>et al.</em> 2008</a>).</p><p id="p-5"><a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> functions as a scaffold for various structure-selective endonucleases that are involved in repairing many kinds of DNA lesions. <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> forms a complex with <a href="http://www.yeastgenome.org/locus/S000005943/overview">Rad1</a>-<a href="http://www.yeastgenome.org/locus/S000004560/overview">Rad10</a> (ERCC1-XPF in mammals), with <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a> in budding yeast and mammals, and with Mus81-Eme1, an ortholog of yeast <a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a>, in mammals (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Munoz <em>et al.</em> 2009</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rouse 2009</a>). <a href="http://www.yeastgenome.org/locus/S000005943/overview">Rad1</a>-<a href="http://www.yeastgenome.org/locus/S000004560/overview">Rad10</a> is a 3′-flap end nuclease, and is involved in nucleotide excision repair and recombination (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Schiestl and Prakash 1990</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mazon <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Munoz-Galvan <em>et al.</em> 2012</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Saito <em>et al.</em> 2012</a>). <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> cleaves the 5′-flap as well as the replication fork structure <em>in vitro</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fricke and Brill 2003</a>), and plays a minor role as a resolvase of Holliday junctions during meiosis (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">De Muyt <em>et al.</em> 2012</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zakharyevich <em>et al.</em> 2012</a>). In addition, <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is involved in interstrand cross-link (ICL) repair, and is also known as FANC-P, which is responsible for one subgroup of Fanconi anemia in humans (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kim <em>et al.</em> 2011</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Stoepker <em>et al.</em> 2011</a>). In addition to its interactions with nucleases, <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> also interacts with the DNA damage response (DDR) component <a href="http://www.yeastgenome.org/locus/S000001197/overview">Rtt107</a>. The <a href="http://www.yeastgenome.org/locus/S000001197/overview">Rtt107</a>-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> complex is involved in suppression of the <a href="http://www.yeastgenome.org/locus/S000003626/overview">Dpb11</a>-<a href="http://www.yeastgenome.org/locus/S000002625/overview">Rad9</a>-related signaling pathway (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Ohouo <em>et al.</em> 2010</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2013</a>).</p><p id="p-6">In yeast meiotic CO formation, <a href="http://www.yeastgenome.org/locus/S000004777/overview">Mlh1</a>-<a href="http://www.yeastgenome.org/locus/S000006085/overview">Mlh3</a> functions as a main player in pro-CO intermediate joint molecule resolution (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Zakharyevich <em>et al.</em> 2012</a>). Thus, <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> has a redundant role with <a href="http://www.yeastgenome.org/locus/S000002794/overview">Mus81</a>-<a href="http://www.yeastgenome.org/locus/S000000302/overview">Mms4</a> and <a href="http://www.yeastgenome.org/locus/S000000843/overview">Yen1</a>, as well as a very minor role in meiotic recombination (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">De Muyt <em>et al.</em> 2012</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Zakharyevich <em>et al.</em> 2012</a>). In contrast, in <em>Caenorhabditis elegans</em>, SLX-1–HIM-18/SLX4 is involved in suppression of CO formation at the center region of the chromosomes through a function of the plant homeodomain (PHD) finger in the SLX-1 protein (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Saito and Colaiacovo 2014</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains</h3><p id="p-7">All genotypes of <em>Saccharomyces cerevisiae</em> strains used in this study are shown in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS1.pdf">Table S1</a>. Deletion alleles of <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">SLX4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000432/overview">SLX1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001197/overview">RTT107</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000005943/overview">RAD1</a></em> were constructed using PCR-mediated gene disruption (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wach <em>et al.</em> 1994</a>). The <em><a href="http://www.yeastgenome.org/locus/S000003134/overview">cup2</a>-B</em> and <em><a href="http://www.yeastgenome.org/locus/S000003293/overview">ade6</a>-B</em> mutations for MSY4304 were introduced by insertion of a <em>Bam</em>HI linker at the first ATG site of each gene by using site-direct mutagenesis. The <em><a href="http://www.yeastgenome.org/locus/S000003093/overview">met13</a>-B</em> and <em><a href="http://www.yeastgenome.org/locus/S000002994/overview">trp5</a>-S</em> mutations were introduced by crossing with strain NHY942 (a gift from Dr. Neil Hunter) which is <em>MAT</em>a parent of NHY957 (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Santos <em>et al.</em> 2003</a>). The original <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">SPO11</a>-3FLAG</em> and <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-Y135F</em> strains were gifts from Dr. K. Ohta and Dr. S. Keeney, respectively (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Diaz <em>et al.</em> 2002</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sasanuma <em>et al.</em> 2007</a>).</p></div><div id="sec-3" class="subsection"><h3>Yeast meiosis time course analysis</h3><p id="p-8"><em>S. cerevisiae</em> strains derived from SK1 background NKY1551 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Storlazzi <em>et al.</em> 1996</a>) were used for meiotic cytological analysis, and western blot and Southern blot analyses. Meiotic time course experiments were carried out as described (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Shinohara <em>et al.</em> 2003</a>).</p></div><div id="sec-4" class="subsection"><h3>Cytological analysis</h3><p id="p-9">Cytological analysis by immunostaining of yeast meiotic nuclear spreads was performed as described (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Shinohara <em>et al.</em> 2008</a>). Stained samples were observed using an epifluorescence microscope (Axioskop2, Zeiss), with LED fluorescence light sources (X-Cite; Excelitas Technologies), and a 100 × objective (Axioplan, NA1.4, Zeiss). Images were captured with a CCD camera (Retiga, Qimaging), and processed using iVision (BioVision Technologies) and Photoshop (Adobe) software. More than 100 nuclei were counted for each sample, and more than five foci-positive nucleus in a cell indicated a focus-positive cell. Antibodies used for this study were anti-Zip1 [rat, 1:500 (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Shinohara <em>et al.</em> 2008</a>)], anti-<a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> [rabbit, 1:500 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Shinohara <em>et al.</em> 2015</a>) or guinea pig, 1:500 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shinohara <em>et al.</em> 2000</a>)], and anti-<a href="http://www.yeastgenome.org/locus/S000000981/overview">Dmc1</a> [rabbit, 1:500 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hayase <em>et al.</em> 2004</a>)].</p></div><div id="sec-5" class="subsection"><h3>Western blotting</h3><p id="p-10">Whole-cell lysates of meiotic cells were extracted with the TCA precipitation method (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sasanuma <em>et al.</em> 2013</a>), and then proteins were separated on SDS-PAGE gels and transferred to PVDF membranes (Immobilon-FL, Millipore). The following antibodies were used for western blotting: anti-DYKDDDDK tag (1E6, Wako), anti-<a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> ((<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Iwasaki et al. 2016</a>), guinea pig, 1:1000), anti-<a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a>-pT318 ((<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Iwasaki et al. 2016</a>), rabbit, 1:1000), and anti-tubulin (MCA77G, AbD Serotec). Primary antibodies were visualized with Alexa Fluor 680-conjugated (Molecular Probes) or IRDye 800-conjugated (LI-COR Biosciences) secondary antibodies using an Odyssey infrared imaging system (LI-COR Biosciences). The density of each signal was determined by using ImageStudio v3.1 software (LI-COR Biosciences).</p></div><div id="sec-6" class="subsection"><h3>Yeast genetic analysis of meiotic recombination</h3><p id="p-11">For tetrad analysis, zygotes were generated by 3-hr matings of each parental haploid strain derived from MSY4304 or MSY4245 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS1.pdf">Table S1</a>), and then transferred to a sporulation medium plate (0.3% potassium acetate, 0.02% raffinose, and 2% agar) and incubated for 48 hr at 30°. Genetic distances between markers and CO interference were analyzed by using the MacTetrad 6.9.1 program (<a href="ftp://130.14.250.7/repository/yeast/mactetrad/">ftp://130.14.250.7/repository/yeast/mactetrad/</a>) as described (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Shinohara <em>et al.</em> 2008</a>). Map distances were determined using Perkins equation, and SEMs were calculated using the Stahl Lab online tool (<a href="http://www.molbio.uoregon.edu/~fstahl">http://www.molbio.uoregon.edu/~fstahl</a>). At least four independent crosses were analyzed for each strain.</p></div><div id="sec-7" class="subsection"><h3>Southern blotting</h3><p id="p-12">Southern blotting was carried out as described (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Storlazzi <em>et al.</em> 1996</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Shima <em>et al.</em> 2005</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Shinohara and Shinohara 2013</a>). Genomic DNA from NKY1551-derived yeast strains was digested using <em>Pst</em>I for DSB detection; <em>Xho</em>I for inter-homolog CO recombination (IHR) detection: and <em>Mlu</em>I, <em>Xho</em>I and <em>Bam</em>HI for hetero-duplex (HD) detection. DNAs were transferred onto nylon membranes (ClearTrans, Wako) by capillary transfer. Probes for Southern blotting were prepared using pNKY291 for DSB and pNKY155 for CO/NCO (non-CO) detection (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Xu <em>et al.</em> 1995</a>). Detection of DSBs at the <em><a href="http://www.yeastgenome.org/locus/S000000630/overview">ELO2</a></em> locus was carried out as described (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gothwal <em>et al.</em> 2016</a>). Probes were labeled with α-[<sup>32</sup>P]-dATP using random labeling with the Klenow fragment (3′-5′ -exo) (NEB) and random dN6 (NEB). Blots were detected using a Phosphorimager BAS5000 (Fuji film) and quantified using ImageQuant software (GE Healthcare).</p></div><div id="sec-8" class="subsection"><h3>Contour-clamped homogeneous electrical field (CHEF) analysis</h3><p id="p-13">Pulsed-field gel electrophoresis (PFGE) to detect the whole-chromosome distribution of meiotic DSBs was performed as described (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bani Ismail <em>et al.</em> 2014</a>). Genomic DNA from NKY1551-derivatived yeast strains was prepared in agarose plugs and run under the following conditions: 120° angle, 6 V/cm, and 48 hr with the CHEF DR-III (Bio-Rad), with 25 sec to 125 sec as the switch time. Signals were visualized by Southern blotting as described above. Probes for Southern blotting were prepared using <em><a href="http://www.yeastgenome.org/locus/S000000569/overview">CHA1</a></em> for chromosome III and <em><a href="http://www.yeastgenome.org/locus/S000003134/overview">CUP2</a></em> for chromosome VII (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bani Ismail <em>et al.</em> 2014</a>).</p></div><div id="sec-9" class="subsection"><h3>Spo11-bound oligo DNA detection</h3><p id="p-14">The DNA fragment covalently bound to <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> protein was isolated as described (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Pan and Keeney 2009</a>). Briefly, <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-3FLAG was immunoprecipitated with anti-DYKDDDDK (1E10, Wako) and Dynabeads Protein G (Veritas) from TCA-treated whole meiotic cell extract. DNA fragments in the immunoprecipitates were labeled with α- [<sup>32</sup>P]-dCTP (NEG531Z, Perkin Elmer) by using terminal transferase (NEB). <sup>32</sup>P signals were detected with a Phosphorimager BAS5000 after separation by SDS-PAGE. <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-3FLAG protein in the immunoprecipitates was detected by western blotting with TrueBlot HRP-conjugated anti-Mouse Ig (Rockland), and then signals were visualized with the ImageQuant LAS4000 (GE healthcare) after treatment with ECL Prime Western blotting detection reagent (GE Healthcare).</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-15">The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>slx4∆ cells are delayed in meiosis progression in a meiotic DSB-dependent manner</h3><p id="p-16">Although <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is involved in a minor pathway to resolve Holliday junctions with <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>, in contrast to the <em><a href="http://www.yeastgenome.org/locus/S000000432/overview">slx1</a></em>∆ mutant, the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant has delayed meiosis progression (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">De Muyt <em>et al.</em> 2012</a>; <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Zakharyevich <em>et al.</em> 2012</a>). We confirmed that the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant showed a 1.3-hr delay in meiosis I entry (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). We then analyzed a <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a></em> catalytic mutation, <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-Y135F</em>, which suppressed the delayed meiosis in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). This suggested that the delay in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ is caused by a post-DSB event.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="slx4&#x2206; cells are delayed in meiosis progression with a defect in Zip1 elongation and Rad51 assembly. (A) Meiosis progression was analyzed in wild type (NKY1551) and slx4&#x2206; (MHY24) strains. The percentage of cells containing two, three, and four nuclei per ascus (post-MI %) is shown in the graph. Error bars indicate the SD from at least three independent trials. (B) Meiosis progression was analyzed in spo11-Y135F (MSY3699) and spo11-Y135F slx4&#x2206; (MHY365) strains. Error bars indicate the SD from at least three independent trials. (C) SC elongation was analyzed in wild type and slx4&#x2206; by immunostaining for Zip1 at each time point. A representative graph showing percentages for each class&#x2014;class&#xA0;I (leptotene), class&#xA0;II (zygotene), and class&#xA0;III (pachytene)&#x2014;for SC elongation for each time point is shown. The percentages of cells with poly complex structures of Zip1 staining are shown in red. (D) Representative images of meiotic nuclear spreads from each indicated time point that were costained with anti-Rad51 (green) and anti-Zip1 (red) in wild type and slx4&#x2206;. Scale bar indicates 2 &#xB5;m. (E) Percentages of Rad51-positive nuclei in wild type and slx4&#x2206; at each time point during meiosis. Error bars indicate the SD from at least three independent trials. (F) Average numbers of Rad51 foci per nucleus at each time point were analyzed in wild type and slx4&#x2206;. At least 100 nuclei were analyzed for each time point. Error bars show the SEM from at least three independent trials. (G) Representative 32P-labeled DNA fragments covalently bound to Spo11-3FLAG (upper) and Spo11-3FLAG protein (lower) in immunoprecipitates from SPO11-3FLAG (wild type, MSY5089) and slx4&#x2206; SPO11-3FLAG (slx4&#x2206;, MHY471) are shown. Average of relative DNA fragment signals at each time point, which was shown as percent of peak amount of signal in wild type (5&#xA0;hr), were shown in graph. Error bar shows SD from three independent trials." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1170113899" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;slx4&lt;/em&gt;&#x2206; cells are delayed in meiosis progression with a defect in Zip1 elongation and Rad51 assembly. (A) Meiosis progression was analyzed in wild type (NKY1551) and &lt;em&gt;slx4&lt;/em&gt;&#x2206; (MHY24) strains. The percentage of cells containing two, three, and four nuclei per ascus (post-MI %) is shown in the graph. Error bars indicate the SD from at least three independent trials. (B) Meiosis progression was analyzed in &lt;em&gt;spo11-Y135F&lt;/em&gt; (MSY3699) and &lt;em&gt;spo11-Y135F slx4&lt;/em&gt;&#x2206; (MHY365) strains. Error bars indicate the SD from at least three independent trials. (C) SC elongation was analyzed in wild type and &lt;em&gt;slx4&lt;/em&gt;&#x2206; by immunostaining for Zip1 at each time point. A representative graph showing percentages for each class&#x2014;class&#xA0;I (leptotene), class&#xA0;II (zygotene), and class&#xA0;III (pachytene)&#x2014;for SC elongation for each time point is shown. The percentages of cells with poly complex structures of Zip1 staining are shown in red. (D) Representative images of meiotic nuclear spreads from each indicated time point that were costained with anti-Rad51 (green) and anti-Zip1 (red) in wild type and &lt;em&gt;slx4&lt;/em&gt;&#x2206;. Scale bar indicates 2 &#xB5;m. (E) Percentages of Rad51-positive nuclei in wild type and &lt;em&gt;slx4&lt;/em&gt;&#x2206; at each time point during meiosis. Error bars indicate the SD from at least three independent trials. (F) Average numbers of Rad51 foci per nucleus at each time point were analyzed in wild type and &lt;em&gt;slx4&lt;/em&gt;&#x2206;. At least 100 nuclei were analyzed for each time point. Error bars show the SEM from at least three independent trials. (G) Representative &lt;sup&gt;32&lt;/sup&gt;P-labeled DNA fragments covalently bound to Spo11-3FLAG (upper) and Spo11-3FLAG protein (lower) in immunoprecipitates from &lt;em&gt;SPO11-3FLAG&lt;/em&gt; (wild type, MSY5089) and &lt;em&gt;slx&lt;/em&gt;4&#x2206; &lt;em&gt;SPO11-3FLAG&lt;/em&gt; (&lt;em&gt;slx&lt;/em&gt;4&#x2206;, MHY471) are shown. Average of relative DNA fragment signals at each time point, which was shown as percent of peak amount of signal in wild type (5&#xA0;hr), were shown in graph. Error bar shows SD from three independent trials.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2033/F1.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/7/2033/F1.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468658" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child"><em>slx4</em>∆ cells are delayed in meiosis progression with a defect in Zip1 elongation and Rad51 assembly. (A) Meiosis progression was analyzed in wild type (NKY1551) and <em>slx4</em>∆ (MHY24) strains. The percentage of cells containing two, three, and four nuclei per ascus (post-MI %) is shown in the graph. Error bars indicate the SD from at least three independent trials. (B) Meiosis progression was analyzed in <em>spo11-Y135F</em> (MSY3699) and <em>spo11-Y135F slx4</em>∆ (MHY365) strains. Error bars indicate the SD from at least three independent trials. (C) SC elongation was analyzed in wild type and <em>slx4</em>∆ by immunostaining for Zip1 at each time point. A representative graph showing percentages for each class—class I (leptotene), class II (zygotene), and class III (pachytene)—for SC elongation for each time point is shown. The percentages of cells with poly complex structures of Zip1 staining are shown in red. (D) Representative images of meiotic nuclear spreads from each indicated time point that were costained with anti-Rad51 (green) and anti-Zip1 (red) in wild type and <em>slx4</em>∆. Scale bar indicates 2 µm. (E) Percentages of Rad51-positive nuclei in wild type and <em>slx4</em>∆ at each time point during meiosis. Error bars indicate the SD from at least three independent trials. (F) Average numbers of Rad51 foci per nucleus at each time point were analyzed in wild type and <em>slx4</em>∆. At least 100 nuclei were analyzed for each time point. Error bars show the SEM from at least three independent trials. (G) Representative <sup>32</sup>P-labeled DNA fragments covalently bound to Spo11-3FLAG (upper) and Spo11-3FLAG protein (lower) in immunoprecipitates from <em>SPO11-3FLAG</em> (wild type, MSY5089) and <em>slx</em>4∆ <em>SPO11-3FLAG</em> (<em>slx</em>4∆, MHY471) are shown. Average of relative DNA fragment signals at each time point, which was shown as percent of peak amount of signal in wild type (5 hr), were shown in graph. Error bar shows SD from three independent trials.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">To confirm the cause of the delay, we analyzed <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> elongation. <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> is a component of the central element of the SC, which is visible as dotty (class I; leptotene), partially elongated (class II; zygotene), and fully elongated (class III; pachytene) structures according to the progression of prophase I in wild type (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>). In the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant, whereas the timing of appearance of class I <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> was normal, the appearance of the partially elongated <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> signal, and also the disappearance of the <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> signal that occurs soon after 6 hr in the wild type, was delayed in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/FigureS1.pdf">Figure S1A</a>). In addition, elongation of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> was affected as compared with wild type, and also the poly complex structure, which is a marker of an SC elongation defect (<a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Shinohara <em>et al.</em> 2008</a>), was increased in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, C and D</a>). This suggested that meiosis progression before leptotene would be normal in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant.</p><p id="p-19">Although the initial timing of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> assembly was normal, we observed a 0.8-hr delay in the appearance of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/FigureS1.pdf">Figure S1B</a>). In addition, disappearance of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci was also delayed (0.7 hr) in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant. There was, however, no significant difference in the average numbers of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci per nucleus between wild-type and <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ cells at each meiotic chromatin at their peak abundance, after 4 hr in meiosis (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1F</a>). In addition, there was no difference in the life span of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> focus positive nuclei in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant compared with that in wild type (3.04 ± 0.64 hr and 2.95 ± 0.57 hr, respectively, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/FigureS1.pdf">Figure S1B</a>), thus this suggests that turnover of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci was not affected in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆. These observations suggest that <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> plays a role in the leptotene-to-zygotene transition, perhaps specifically in the formation of DSBs, and their extensive resection to promote <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> assembly. Then, we observed reduced amount of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-bound oligomeric DNA fragment (<a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-oligo) in the early phase (at 3–5 hr) in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1G</a>), similar to the <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> focus number (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1F</a>). In contrast, the amount of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-oligo at the peak point (6 hr) was distinguishable from that in wild type. This strongly suggests that <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is required for efficient DSB formation.</p></div><div id="sec-13" class="subsection"><h3>Delay of meiosis progression in the slx4∆ mutant occurs independently of the Slx4-related components</h3><p id="p-20"><a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is phosphorylated by the <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> and <a href="http://www.yeastgenome.org/locus/S000000184/overview">Tel1</a> kinases after DNA damage (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Flott <em>et al.</em> 2007</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Toh <em>et al.</em> 2010</a>). We analyzed <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> phosphorylation during meiosis. We conjugated <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> protein with 3 × FLAG epitope at the N -terminus (3FLAG-Slx4), and confirmed that the tagging does not affect meiosis progression (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/FigureS2.pdf">Figure S2A</a>). We analyzed <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> protein during meiosis by western blotting with an antibody against FLAG. We detected an increase in 3FLAG-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> expression during meiosis, and also multiple slower-migrating signals at 2.5 to 4 hr after meiosis entry, such that most of the 3FLAG-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> protein was hyper-shifted at 3 to 4 hr (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) when the appearance of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci peaks (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1E</a>). The hyper-shifted 3FLAG-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> was undetectable in <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-Y135F</em> mutant (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). In addition, we confirmed that mobility of the hyper-shifted signal was indistinguishable from the signals induced by DNA damage in both mitotic and meiotic cells (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/FigureS2.pdf">Figure S2B</a>). This indicated that <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is phosphorylated as a result of not only accidental DSBs induced during vegetative growth but also programmed DSBs induced by <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a> during meiosis.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The multiple functions of Slx4 are required for proper meiosis. (A) Phosphorylation of 3FLAG-Slx4 was analyzed during meiosis. Whole-cell extracts from wild type (untagged; NKY1551), 3FLAG-SLX4 (MHY129), and spo11-Y135F 3FLAG-SLX4 (MHY187) meiotic cells were analyzed by western blotting with anti-FLAG and anti-tubulin antibodies. The asterisk indicates the phosphorylated signal. (B) Meiosis progression in wild type (NKY1551), slx4&#x2206; (MHY24), slx1&#x2206; (MHY68), rad1&#x2206; (MHY96), and rtt107&#x2206; (MHY235) was analyzed as described in Figure 1A. (C) Phosphorylation of Hop1 at T318 in wild type (NKY1551) and slx4&#x2206; (MHY24) was analyzed by western blotting with anti-Hop1 (green) and anti-Hop1-pT318 (red) (left). The relative ratio of pT318 signal to the Hop1 signal is shown in the graph (right). Error bars show SD from more than three independent trials." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1170113899" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The multiple functions of Slx4 are required for proper meiosis. (A) Phosphorylation of 3FLAG-Slx4 was analyzed during meiosis. Whole-cell extracts from wild type (untagged; NKY1551), &lt;em&gt;3FLAG-SLX4&lt;/em&gt; (MHY129), and &lt;em&gt;spo11-Y135F 3FLAG-SLX4&lt;/em&gt; (MHY187) meiotic cells were analyzed by western blotting with anti-FLAG and anti-tubulin antibodies. The asterisk indicates the phosphorylated signal. (B) Meiosis progression in wild type (NKY1551), &lt;em&gt;slx4&lt;/em&gt;&#x2206; (MHY24), &lt;em&gt;slx1&lt;/em&gt;&#x2206; (MHY68), &lt;em&gt;rad1&lt;/em&gt;&#x2206; (MHY96), and &lt;em&gt;rtt107&lt;/em&gt;&#x2206; (MHY235) was analyzed as described in Figure 1A. (C) Phosphorylation of Hop1 at T318 in wild type (NKY1551) and &lt;em&gt;slx4&lt;/em&gt;&#x2206; (MHY24) was analyzed by western blotting with anti-Hop1 (green) and anti-Hop1-pT318 (red) (left). The relative ratio of pT318 signal to the Hop1 signal is shown in the graph (right). Error bars show SD from more than three independent trials.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2033/F2.medium.gif" width="376" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/7/2033/F2.medium.gif" width="376" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468471" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">The multiple functions of Slx4 are required for proper meiosis. (A) Phosphorylation of 3FLAG-Slx4 was analyzed during meiosis. Whole-cell extracts from wild type (untagged; NKY1551), <em>3FLAG-SLX4</em> (MHY129), and <em>spo11-Y135F 3FLAG-SLX4</em> (MHY187) meiotic cells were analyzed by western blotting with anti-FLAG and anti-tubulin antibodies. The asterisk indicates the phosphorylated signal. (B) Meiosis progression in wild type (NKY1551), <em>slx4</em>∆ (MHY24), <em>slx1</em>∆ (MHY68), <em>rad1</em>∆ (MHY96), and <em>rtt107</em>∆ (MHY235) was analyzed as described in <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1A</a>. (C) Phosphorylation of Hop1 at T318 in wild type (NKY1551) and <em>slx4</em>∆ (MHY24) was analyzed by western blotting with anti-Hop1 (green) and anti-Hop1-pT318 (red) (left). The relative ratio of pT318 signal to the Hop1 signal is shown in the graph (right). Error bars show SD from more than three independent trials.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22"><a href="http://www.yeastgenome.org/locus/S000000184/overview">Tel1</a>/<a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a>-dependent phosphorylation of <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is related to the <a href="http://www.yeastgenome.org/locus/S000005943/overview">Rad1</a>-<a href="http://www.yeastgenome.org/locus/S000004560/overview">Rad10</a> <a href="http://www.yeastgenome.org/locus/S000000432/overview">endonuclease</a> activity of cleaving nonhomologous tails in the single-strand annealing (SSA) pathway (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Toh <em>et al.</em> 2010</a>). We thus examined the contribution of <a href="http://www.yeastgenome.org/locus/S000005943/overview">Rad1</a>-<a href="http://www.yeastgenome.org/locus/S000004560/overview">Rad10</a> nuclease in meiosis progression. In contrast to the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant, we did not observe any delay in meiosis in the <em><a href="http://www.yeastgenome.org/locus/S000005943/overview">rad1</a></em>∆ mutant (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). We also analyzed mutations in two additional <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a>-related components: <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a> and <a href="http://www.yeastgenome.org/locus/S000001197/overview">Rtt107</a>. <a href="http://www.yeastgenome.org/locus/S000001197/overview">Rtt107</a> is involved in the regulation of <a href="http://www.yeastgenome.org/locus/S000006074/overview">Rad53</a> activity through a mechanism referred to as dampens checkpoint adaptor-mediated phosphor-signaling (DAMP) during mitosis (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Ohouo <em>et al.</em> 2010</a>). However, we did not observe any delay in meiosis in these mutants (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). On the other hand, it is known that <em><a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002794/overview">mus81</a></em> mutants show a delay in meiosis progression, and, then, the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ <em><a href="http://www.yeastgenome.org/locus/S000000302/overview">mms4</a></em> meiotic null double mutant shows additive delays (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">de los Santos <em>et al.</em> 2003</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">De Muyt <em>et al.</em> 2012</a>).</p><p id="p-23">We next analyzed <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> expression and phosphorylation to determine whether <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is required in the meiotic DSB-related checkpoint pathway. <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a>, a multi-functional protein, is a meiosis-specific component of the axial structure of the SC, and <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a>/<a href="http://www.yeastgenome.org/locus/S000000184/overview">Tel1</a>-dependent phosphorylation of <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> is essential for its function (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carballo <em>et al.</em> 2008</a>). We used an antibody against whole <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> protein, and an antibody specific for phospho-T318 of <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> to monitor <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a>/<a href="http://www.yeastgenome.org/locus/S000000184/overview">Tel1</a>-dependent <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> phosphorylation. Expression of <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> was detected from 2 hr after meiosis entry in both wild type and the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). This also indicates that meiosis progression before leptotene would be normal in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant. Phosphorylation of <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> began to appear after 3 hr in meiosis, with robust phosphorylation detected at 4 hr in wild type and the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant. Interestingly, quantification of the signals indicated that <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> phosphorylation level was increased in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant cells (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>).</p></div><div id="sec-14" class="subsection"><h3>slx4∆ cells have altered intrachromosomal distribution of COs on chromosomes III and VII</h3><p id="p-24">A previous genetic analysis in budding yeast revealed that <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant cells show as significant increase in CO frequency in the <em>HIS4LEU2-MAT</em> interval, but not in the interval <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-HIS4LEU2</em> in the strain that includes the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> hot spot on chromosome III (<a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Zakharyevich <em>et al.</em> 2012</a>). We reanalyzed the CO frequency in additional intervals including the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-MAT</em> interval on chromosome III and also several intervals on chromosome VII (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) to compare COs in different chromosomes of different length. We compared the genetic length (in centimorgans) of each interval among wild type, and the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000000432/overview">slx1</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000001197/overview">rtt107</a></em>∆, and <em><a href="http://www.yeastgenome.org/locus/S000005943/overview">rad1</a></em>∆ mutants. First, we confirmed that <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ does not show any defect in spore viability (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mullen <em>et al.</em> 2001</a>); in addition, the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">SLX4</a></em>-related mutants <em><a href="http://www.yeastgenome.org/locus/S000000432/overview">slx1</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000005943/overview">rad1</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000004560/overview">rad10</a></em>∆, and <em><a href="http://www.yeastgenome.org/locus/S000001197/overview">rtt107</a></em>∆ showed no changes in spore viability compared with wild type (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS2.pdf">Table S2</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="CO distribution and CO interference are affected in slx4&#x2206; cells in the chromosomal region that includes the centromere. (A) Schematics show physical maps of genetic markers used for the genetic analysis and centromere (Cen) on chromosomes&#xA0;III or VII. (B) Genetic CO frequencies for each interval on chromosome&#xA0;III in wild type (NKY4304/4245), slx4&#x2206; (MSY4930/4910), slx1&#x2206; (MSY5314/5282), rad1&#x2206; (MSY5624/5622), and rtt107&#x2206; (MSY5622/5159) are shown. (C) Genetic CO frequencies for each interval on chromosome&#xA0;VII are shown. (D) NPD ratios on chromosome&#xA0;III were analyzed using Papazian&#x2019;s method in wild type (NKY4304/4245, n&#xA0;=&#xA0;1341), slx4&#x2206; (MSY4930/4910, n&#xA0;=&#xA0;1682), slx1&#x2206; (MSY5314/5282, n&#xA0;=&#xA0;1362), rad1&#x2206; (MSY5624/5622, n&#xA0;=&#xA0;1724), and rtt107&#x2206; (MSY5622/5159, n&#xA0;=&#xA0;1192). Error bars show the SEM. Details are shown in Table S3. (E) NPD ratios on chromosome&#xA0;VII are shown. Error bars in (B) to (E) indicate the SEM, and asterisks indicate a significant difference between the values in wild type based on Perkins formula (**&#xA0;P&#xA0;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1170113899" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;CO distribution and CO interference are affected in &lt;em&gt;slx4&lt;/em&gt;&#x2206; cells in the chromosomal region that includes the centromere. (A) Schematics show physical maps of genetic markers used for the genetic analysis and centromere (Cen) on chromosomes&#xA0;III or VII. (B) Genetic CO frequencies for each interval on chromosome&#xA0;III in wild type (NKY4304/4245), &lt;em&gt;slx4&lt;/em&gt;&#x2206; (MSY4930/4910), &lt;em&gt;slx1&lt;/em&gt;&#x2206; (MSY5314/5282), &lt;em&gt;rad1&lt;/em&gt;&#x2206; (MSY5624/5622), and &lt;em&gt;rtt107&lt;/em&gt;&#x2206; (MSY5622/5159) are shown. (C) Genetic CO frequencies for each interval on chromosome&#xA0;VII are shown. (D) NPD ratios on chromosome&#xA0;III were analyzed using Papazian&#x2019;s method in wild type (NKY4304/4245, &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1341), &lt;em&gt;slx4&lt;/em&gt;&#x2206; (MSY4930/4910, &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1682), &lt;em&gt;slx1&lt;/em&gt;&#x2206; (MSY5314/5282, &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1362), &lt;em&gt;rad1&lt;/em&gt;&#x2206; (MSY5624/5622, &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1724), and &lt;em&gt;rtt107&lt;/em&gt;&#x2206; (MSY5622/5159, &lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;1192). Error bars show the SEM. Details are shown in Table S3. (E) NPD ratios on chromosome&#xA0;VII are shown. Error bars in (B) to (E) indicate the SEM, and asterisks indicate a significant difference between the values in wild type based on Perkins formula (**&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.01, *&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.05). All values were calculated using the Stahl Laboratory on-line tool.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2033/F3.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/7/2033/F3.medium.gif" width="354" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468623" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">CO distribution and CO interference are affected in <em>slx4</em>∆ cells in the chromosomal region that includes the centromere. (A) Schematics show physical maps of genetic markers used for the genetic analysis and centromere (Cen) on chromosomes III or VII. (B) Genetic CO frequencies for each interval on chromosome III in wild type (NKY4304/4245), <em>slx4</em>∆ (MSY4930/4910), <em>slx1</em>∆ (MSY5314/5282), <em>rad1</em>∆ (MSY5624/5622), and <em>rtt107</em>∆ (MSY5622/5159) are shown. (C) Genetic CO frequencies for each interval on chromosome VII are shown. (D) NPD ratios on chromosome III were analyzed using Papazian’s method in wild type (NKY4304/4245, <em>n</em> = 1341), <em>slx4</em>∆ (MSY4930/4910, <em>n</em> = 1682), <em>slx1</em>∆ (MSY5314/5282, <em>n</em> = 1362), <em>rad1</em>∆ (MSY5624/5622, <em>n</em> = 1724), and <em>rtt107</em>∆ (MSY5622/5159, <em>n</em> = 1192). Error bars show the SEM. Details are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS3.pdf">Table S3</a>. (E) NPD ratios on chromosome VII are shown. Error bars in (B) to (E) indicate the SEM, and asterisks indicate a significant difference between the values in wild type based on Perkins formula (** <em>P</em> &lt; 0.01, * <em>P</em> &lt; 0.05). All values were calculated using the Stahl Laboratory on-line tool.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We confirmed that the CO frequency on chromosome III within the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-MAT</em> interval, which includes the centromere, was significantly increased in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ (58 ± 2.0 cM) as compared with wild type (37 ± 1.5 cM) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS3.pdf">Table S3</a>). In contrast, CO frequency was significantly reduced in the <em><a href="http://www.yeastgenome.org/locus/S000029214/overview">HML</a>-<a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> interval, and slightly decreased in the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> interval. Changes in CO frequencies were not observed in other <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">SLX4</a></em>-related mutants, meaning CO frequencies in the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-MAT</em>, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000029214/overview">HML</a>-<a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> intervals in <em><a href="http://www.yeastgenome.org/locus/S000000432/overview">slx1</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000001197/overview">rtt107</a></em>∆, and <em><a href="http://www.yeastgenome.org/locus/S000005943/overview">rad1</a></em>∆ mutants were indistinguishable from those of wild type. When the CO frequencies were summed, the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant showed an increase in CO frequency in the <em><a href="http://www.yeastgenome.org/locus/S000029214/overview">HML</a></em>-though-<em>MAT</em> region on chromosome III (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>).</p><p id="p-27">Similar increases and decreases in CO frequencies were observed on chromosome VII, one of the largest chromosomes in budding yeast. In the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant, CO frequency was significantly increased in the <em><a href="http://www.yeastgenome.org/locus/S000002994/overview">TRP5</a>-<a href="http://www.yeastgenome.org/locus/S000003293/overview">ADE6</a></em> interval, which includes the centromere. It was unchanged in the <em><a href="http://www.yeastgenome.org/locus/S000003093/overview">MET13</a>-<a href="http://www.yeastgenome.org/locus/S000003071/overview">CYH2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003071/overview">CYH2</a>-<a href="http://www.yeastgenome.org/locus/S000002994/overview">TRP5</a></em> intervals, but was significantly decreased in the <em><a href="http://www.yeastgenome.org/locus/S000003134/overview">CUP2</a>-<a href="http://www.yeastgenome.org/locus/S000003093/overview">MET13</a></em> interval. However, these tendencies were not observed in the other <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">SLX4</a></em>-related mutants (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS3.pdf">Table S3</a>). Again, the total CO frequency across these intervals on chromosome VII was increased in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆. In contrast, there were no significant differences in the noncrossover (NCO) frequencies, observed as non-Mendelian segregation, at each genetic locus in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ and in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">SLX4</a></em>-related mutants, and also that in wild type (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS4.pdf">Table S4</a>). These results suggested that <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">SLX4</a></em> is involved in the regulation of CO distribution along each chromosome.</p><p id="p-28">To determine the function of <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> and <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a>-related components in CO control, we analyzed CO interference for each interval on both chromosome III and chromosome VII by an analysis with Papazian’s equation (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3, D and E</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS3.pdf">Table S3</a>), as well as the coefficient of coincidence (COC) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS5.pdf">Table S5</a>). In wild type, the ratios of the observed to expected number of nonparental di-types (NPDs) were &lt; 0.5 for all intervals (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3, D and E</a>), indicating the presence of CO interference. In the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant, we observed abolished CO interference in the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-MAT</em> and <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a>-MAT</em> intervals (0.86 ± 0.1 and 1.13 ± 0.15, respectively) on chromosome III and in the <em><a href="http://www.yeastgenome.org/locus/S000002994/overview">TRP5</a>-<a href="http://www.yeastgenome.org/locus/S000003293/overview">ADE6</a></em> interval (1.52 ± 0.2) on chromosome VII (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3, D and E</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS3.pdf">Table S3</a>). Interestingly, all three intervals include the centromere. Although the ratio of the observed number of NPDs to the expected number of NPDs was almost the same for the <em><a href="http://www.yeastgenome.org/locus/S000029214/overview">HML</a>-<a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em> interval, it was not significant because of a small number of NPD tetrads in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS3.pdf">Table S3</a>). In addition, we observed compromised CO interference in the <em><a href="http://www.yeastgenome.org/locus/S000003134/overview">CUP2</a>-<a href="http://www.yeastgenome.org/locus/S000003093/overview">MET13</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003093/overview">MET13</a>-<a href="http://www.yeastgenome.org/locus/S000003071/overview">CYH2</a></em> intervals in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ as compared with that in wild type (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS3.pdf">Table S3</a>). In contrast, there was a significantly greater amount of CO interference in the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> interval in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ than in wild type. In support of this finding, we observed weakened CO interference in the <em><a href="http://www.yeastgenome.org/locus/S000029214/overview">HML</a>-<a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> region, for which the ratios of observed to expected consecutive COs were 0.491 (<em>P</em> &lt; 0.001) and 0.724 (<em>P</em> = 0.41) in wild type and <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆, respectively, based on the COC method (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS5.pdf">Table S5</a>).</p></div><div id="sec-15" class="subsection"><h3>The slx4∆ mutant has a slight delay in meiotic DSB formation</h3><p id="p-29">To determine the cause of altered intrachromosomal CO distribution in the <a href="http://www.yeastgenome.org/locus/S000004125/overview"><em>slx</em>4</a>∆ mutant, we analyzed the physical products of meiotic recombination at the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> hot spot (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Storlazzi <em>et al.</em> 1996</a>) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). We observed a slight decrease in the genetic CO frequency in the overlapping intervals <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-<a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a></em> (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS3.pdf">Table S3</a>). First, we analyzed programmed DSB formation as an initial event of meiotic CO formation by Southern blotting (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/FigureS3.pdf">Figure S3</a>). We observed a delay in the appearance and also the disappearance of meiotic DSBs in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant as compared with wild type (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, B and C</a>), which corresponded temporally with the appearance and disappearance of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1E</a>). In addition, we observed a slight decreased in the peak amount of DSB formation in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ (10.6 ± 1.1% at 4 hr) as compared with that in wild type (12.2 ± 2.7% at 3 hr) (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>). This result suggested two possibilities: (i) the total amount of DSB formed at this hot spot was decreased, or (ii) DBSs were repaired more rapidly in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant. We observed the same amount of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-oligo in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ at a peak point (5 hr) as compared with that in wild type (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1G</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Physical analysis of meiotic recombination products at the HIS4-LEU2 hot spot. (A) Schematic showing the HIS4-LEU2 meiotic recombination hot spot. Diagnostic restriction enzyme sites are shown as X (XhoI), Pst (PstI), M (MluI), and B (BamHI). The sizes of meiotic DSBs (DSBs&#xA0;I and II) and parental band (P) detectable by probe 291, or of interhomolog COs (R1 and R2) and parental (P1 and P2) and heteroduplex (HD) intermediates (HD1&#x2013;4) of homologous recombination and parental (P1 and P2) detectable by probe 155 are shown. (B) Representative Southern blotting of DSB detection in wild type (NKY1551) and slx4&#x2206; (MHY24). Time in meiosis (in hours) is shown above the blot. (C) Quantification of DSB&#xA0;I signals in wild type and slx4&#x2206; from Southern blotting as in (B). Error bars represent the SD (n&#xA0;=&#xA0;4). (D) Representative Southern blotting of CO detection in wild type and slx4&#x2206; from the same DNA samples as in (B). (E) Quantification of IHR signals (i.e., R1&#xA0;+&#xA0;R2) in wild type and slx4&#x2206; from Southern blotting as in (D). Error bars represent SD (n&#xA0;=&#xA0;3). (F) Representative Southern blotting of HD detection in wild type and slx4&#x2206; from the same DNA samples as in (B). HD1 and HD4 correspond to NCO products, HD2 and HD3 correspond to the CO products as shown in (A). EC1 and EC2 are produced by ectopic recombination between HIS4-LEU2 and leu2::hisG (Shinohara et al. 2003). (G) Quantification of HD-CO signals (i.e., HD2&#xA0;+&#xA0;HD3 signals, which correspond to CO intermediates) in wild type and slx4&#x2206; from Southern blotting as in (F). Error bars represent the SD (n&#xA0;=&#xA0;3). (H) Quantification of HD-NCO signals (i.e., HD1&#xA0;+&#xA0;HD4, which correspond to NCO intermediates) in wild type and slx4&#x2206; from Southern blotting as in (F). Error bars represent the SD (n&#xA0;=&#xA0;3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1170113899" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Physical analysis of meiotic recombination products at the &lt;em&gt;HIS4-LEU2&lt;/em&gt; hot spot. (A) Schematic showing the &lt;em&gt;HIS4-LEU2&lt;/em&gt; meiotic recombination hot spot. Diagnostic restriction enzyme sites are shown as X (&lt;em&gt;Xho&lt;/em&gt;I), Pst (&lt;em&gt;Pst&lt;/em&gt;I), M (&lt;em&gt;Mlu&lt;/em&gt;I), and B (&lt;em&gt;Bam&lt;/em&gt;HI). The sizes of meiotic DSBs (DSBs&#xA0;I and II) and parental band (P) detectable by probe 291, or of interhomolog COs (R1 and R2) and parental (P1 and P2) and heteroduplex (HD) intermediates (HD1&#x2013;4) of homologous recombination and parental (P1 and P2) detectable by probe 155 are shown. (B) Representative Southern blotting of DSB detection in wild type (NKY1551) and &lt;em&gt;slx4&lt;/em&gt;&#x2206; (MHY24). Time in meiosis (in hours) is shown above the blot. (C) Quantification of DSB&#xA0;I signals in wild type and &lt;em&gt;slx4&lt;/em&gt;&#x2206; from Southern blotting as in (B). Error bars represent the SD (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;4). (D) Representative Southern blotting of CO detection in wild type and &lt;em&gt;slx4&lt;/em&gt;&#x2206; from the same DNA samples as in (B). (E) Quantification of IHR signals (&lt;em&gt;i.e.&lt;/em&gt;, R1&#xA0;+&#xA0;R2) in wild type and &lt;em&gt;slx4&lt;/em&gt;&#x2206; from Southern blotting as in (D). Error bars represent SD (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;3). (F) Representative Southern blotting of HD detection in wild type and &lt;em&gt;slx4&lt;/em&gt;&#x2206; from the same DNA samples as in (B). HD1 and HD4 correspond to NCO products, HD2 and HD3 correspond to the CO products as shown in (A). EC1 and EC2 are produced by ectopic recombination between &lt;em&gt;HIS4-LEU2&lt;/em&gt; and &lt;em&gt;leu2&lt;/em&gt;::&lt;em&gt;hisG&lt;/em&gt; (Shinohara &lt;em&gt;et al.&lt;/em&gt; 2003). (G) Quantification of HD-CO signals (&lt;em&gt;i.e.&lt;/em&gt;, HD2&#xA0;+&#xA0;HD3 signals, which correspond to CO intermediates) in wild type and &lt;em&gt;slx4&lt;/em&gt;&#x2206; from Southern blotting as in (F). Error bars represent the SD (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;3). (H) Quantification of HD-NCO signals (&lt;em&gt;i.e.&lt;/em&gt;, HD1&#xA0;+&#xA0;HD4, which correspond to NCO intermediates) in wild type and &lt;em&gt;slx4&lt;/em&gt;&#x2206; from Southern blotting as in (F). Error bars represent the SD (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2033/F4.medium.gif" width="254" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/7/2033/F4.medium.gif" width="254" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468509" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Physical analysis of meiotic recombination products at the <em>HIS4-LEU2</em> hot spot. (A) Schematic showing the <em>HIS4-LEU2</em> meiotic recombination hot spot. Diagnostic restriction enzyme sites are shown as X (<em>Xho</em>I), Pst (<em>Pst</em>I), M (<em>Mlu</em>I), and B (<em>Bam</em>HI). The sizes of meiotic DSBs (DSBs I and II) and parental band (P) detectable by probe 291, or of interhomolog COs (R1 and R2) and parental (P1 and P2) and heteroduplex (HD) intermediates (HD1–4) of homologous recombination and parental (P1 and P2) detectable by probe 155 are shown. (B) Representative Southern blotting of DSB detection in wild type (NKY1551) and <em>slx4</em>∆ (MHY24). Time in meiosis (in hours) is shown above the blot. (C) Quantification of DSB I signals in wild type and <em>slx4</em>∆ from Southern blotting as in (B). Error bars represent the SD (<em>n</em> = 4). (D) Representative Southern blotting of CO detection in wild type and <em>slx4</em>∆ from the same DNA samples as in (B). (E) Quantification of IHR signals (<em>i.e.</em>, R1 + R2) in wild type and <em>slx4</em>∆ from Southern blotting as in (D). Error bars represent SD (<em>n</em> = 3). (F) Representative Southern blotting of HD detection in wild type and <em>slx4</em>∆ from the same DNA samples as in (B). HD1 and HD4 correspond to NCO products, HD2 and HD3 correspond to the CO products as shown in (A). EC1 and EC2 are produced by ectopic recombination between <em>HIS4-LEU2</em> and <em>leu2</em>::<em>hisG</em> (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Shinohara <em>et al.</em> 2003</a>). (G) Quantification of HD-CO signals (<em>i.e.</em>, HD2 + HD3 signals, which correspond to CO intermediates) in wild type and <em>slx4</em>∆ from Southern blotting as in (F). Error bars represent the SD (<em>n</em> = 3). (H) Quantification of HD-NCO signals (<em>i.e.</em>, HD1 + HD4, which correspond to NCO intermediates) in wild type and <em>slx4</em>∆ from Southern blotting as in (F). Error bars represent the SD (<em>n</em> = 3).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">We analyzed interhomolog CO products (IHRs) at this locus (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4A</a>). We observed a delay in the appearance of the product, and a slight but significant reduction in the total amount of IHRs in the <em>slx</em>4∆ mutant (<em>P</em> = 0.006, at 6 hr) (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4, D and E</a>), although a previous study indicated that the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ single mutation has little effect on meiotic recombination (<a id="xref-ref-55-6" class="xref-bibr" href="#ref-55">Zakharyevich <em>et al.</em> 2012</a>). In addition, we analyzed both the CO products and NCO products separately with additional digests at the hetero-allelic restriction enzyme site <em>Mlu</em>I (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4A</a>). We observed a slight decrease in both CO and NCO products in the <em>slx</em>4∆ mutant as compared with wild type (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4, F–H</a>). This result corresponds to the result that the efficiency of DSB formation at the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> hot spot was reduced in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4A</a>). In addition, we observed an increase in extra bands, which are caused by ectopic recombination (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Shinohara <em>et al.</em> 2003</a>), in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant. An increase in ectopic recombination is also observed in the checkpoint mutants <em><a href="http://www.yeastgenome.org/locus/S000000340/overview">mec1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000975/overview">rad24</a></em>, or <em><a href="http://www.yeastgenome.org/locus/S000005895/overview">rad17</a></em>, and meiotic recombination, such as <em><a href="http://www.yeastgenome.org/locus/S000000981/overview">dmc1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000277/overview">tid1</a>/<a href="http://www.yeastgenome.org/locus/S000000277/overview">rdh54</a></em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grushcow <em>et al.</em> 1999</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Shinohara and Shinohara 2013</a>). These results suggest that the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutation compromises the strand-invasion process during meiotic recombination.</p></div><div id="sec-16" class="subsection"><h3>Slx4 regulates the chromosome-wide distribution of meiotic DSB formation</h3><p id="p-32">Because we observed altered genetic CO distribution for both chromosomes III and VII, and a reduction in the physical CO products corresponding to the reduced formation of DSBs at the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> hot spot, we hypothesized that altered CO distribution may be caused by altered DSB formation in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant cells. To confirm this possibility, we analyzed chromosome-wide meiotic DSB distribution on chromosomes III (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A and B</a>) and VII (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, C and D</a>) by using CHEF electrophoresis analysis. We used the <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a>S</em> background to make DSB signals more clear and to prevent DSB disappearance to facilitate quantification.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The distribution of meiotic DSBs is affected in slx4&#x2206; on chromosomes&#xA0;III and VII. (A) Distribution of DSBs along chromosome III was analyzed by Southern blotting after separation by PFGE. (B) Average of quantified signals (#1&#x2013;#5) shown in (A) at each time point in rad50S (blue) and slx4&#x2206; rad50S (red). Error bars represent SD (n&#xA0;=&#xA0;3). (C) Distribution of DSBs along chromosome&#xA0;VII was analyzed by Southern blotting after separation by PFGE. (D) Average of quantified signals (#6&#x2013;#9) shown in (C) at each time point in rad50S (blue) and slx4&#x2206; rad50S (red). Error bars represent SD (n = 3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1170113899" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The distribution of meiotic DSBs is affected in &lt;em&gt;slx&lt;/em&gt;4&#x2206; on chromosomes&#xA0;III and VII. (A) Distribution of DSBs along chromosome III was analyzed by Southern blotting after separation by PFGE. (B) Average of quantified signals (#1&#x2013;#5) shown in (A) at each time point in &lt;em&gt;rad50S&lt;/em&gt; (blue) and &lt;em&gt;slx4&lt;/em&gt;&#x2206; &lt;em&gt;rad50S&lt;/em&gt; (red). Error bars represent SD (&lt;em&gt;n&lt;/em&gt;&#xA0;=&#xA0;3). (C) Distribution of DSBs along chromosome&#xA0;VII was analyzed by Southern blotting after separation by PFGE. (D) Average of quantified signals (#6&#x2013;#9) shown in (C) at each time point in &lt;em&gt;rad50S&lt;/em&gt; (blue) and &lt;em&gt;slx4&lt;/em&gt;&#x2206; &lt;em&gt;rad50S&lt;/em&gt; (red). Error bars represent SD (&lt;em&gt;n&lt;/em&gt; = 3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/2033/F5.medium.gif" width="327" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/7/2033/F5.medium.gif" width="327" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/2033/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468507" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-33" class="first-child">The distribution of meiotic DSBs is affected in <em>slx</em>4∆ on chromosomes III and VII. (A) Distribution of DSBs along chromosome III was analyzed by Southern blotting after separation by PFGE. (B) Average of quantified signals (#1–#5) shown in (A) at each time point in <em>rad50S</em> (blue) and <em>slx4</em>∆ <em>rad50S</em> (red). Error bars represent SD (<em>n</em> = 3). (C) Distribution of DSBs along chromosome VII was analyzed by Southern blotting after separation by PFGE. (D) Average of quantified signals (#6–#9) shown in (C) at each time point in <em>rad50S</em> (blue) and <em>slx4</em>∆ <em>rad50S</em> (red). Error bars represent SD (<em>n</em> = 3).</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">On chromosome III, we observed significant differences in DSB distribution (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>, #1–5, arrow marked). In some instances, the number of DSBs was reduced (#2 and #5, blue arrow) or elevated (#1, #3, and #4, red arrows) in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ as compared with wild type. Especially at sites #2, #4, and #5, the differences were notable (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>). In contrast, although we observed an increase in the genetic CO frequency in the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-MAT</em> interval, we did not observe a notable increase in DSB formation in this region with the CHEF analysis (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5, A and B</a>). We then analyzed site-specific DSB formation at the <em><a href="http://www.yeastgenome.org/locus/S000000630/overview">ELO2</a></em> locus, which is known as a cold spot for DSB formation in wild type (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baudat and Nicolas 1997</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gothwal <em>et al.</em> 2016</a>), and is located in the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-MAT</em> interval. We observed a slight increase in DSB formation in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant in the <em><a href="http://www.yeastgenome.org/locus/S000005194/overview">rad50</a>S</em> background at this locus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/FigureS3.pdf">Figure S3</a>). On chromosome VII, we observed a significant reduction in DSB formation in the region near the <em><a href="http://www.yeastgenome.org/locus/S000003134/overview">CUP2</a></em> locus in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5, C and D</a>; #9) as well as at site #8 (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5, C and D</a>). We originally considered region #9 to be a cold spot, as we did not observe any apparent DSB bands. We observed a significant reduction of genetic CO frequency in this region, <em><a href="http://www.yeastgenome.org/locus/S000003134/overview">CUP2</a>-<a href="http://www.yeastgenome.org/locus/S000003093/overview">MET13</a></em>, in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3C</a>). In contrast, we observed a significant increase in the number of DSBs at sites #6 and #7 in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5, C and D</a>). These results indicated that <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is involved in DSB formation, and is required for the normal distribution of DSBs across each chromosome. In addition, we detected a reduced amount of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-oligo DNA in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant compared with that in wild type in the early phase of meiotic recombination (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1G</a>).</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-35">SLX-1, together with SLX-4/HIM-18, is required for suppression of CO formation at the center region of the <em>C</em>. <em>elegans</em> chromosome (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Saito and Colaiacovo 2014</a>). We observed an increase in CO formation in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant as compared with wild type, specifically in those intervals that contain the centromere, on both chromosomes III and VII in budding yeast. In contrast to <em>C</em>. <em>elegans</em>, we did not observe this phenomenon in the <em><a href="http://www.yeastgenome.org/locus/S000000432/overview">slx1</a></em>∆ mutant, nor in <em><a href="http://www.yeastgenome.org/locus/S000005943/overview">rad1</a></em>∆ or <em><a href="http://www.yeastgenome.org/locus/S000001197/overview">rtt107</a></em>∆. The Slx4-Slx1 complex plays a minor role in the resolution of Holliday junctions during a late step of meiotic recombination (<a id="xref-ref-55-7" class="xref-bibr" href="#ref-55">Zakharyevich <em>et al.</em> 2012</a>). In this study, we observed defects during an early step of meiosis in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant, such as the delayed formation of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> foci (<a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1E</a>) and of meiotic DSB formation (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4C</a>), as well as a reduced amount of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-oligo (<a id="xref-fig-1-16" class="xref-fig" href="#F1">Figure 1G</a>). In addition, no elevation of <a href="http://www.yeastgenome.org/locus/S000000897/overview">Rad51</a> focus number at peak point in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant, even in the delayed recombination reaction (<a id="xref-fig-1-17" class="xref-fig" href="#F1">Figure 1F</a> and <a id="xref-fig-4-11" class="xref-fig" href="#F4">Figure 4, D–H</a>), also suggested reduced DSB formation or asynchronous DSB formation in the cells. In contrast, we did not observe any delay in the appearance of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> foci (<a id="xref-fig-1-18" class="xref-fig" href="#F1">Figure 1D</a>) and expression of <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2C</a>). This indicates that the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant has a defect in DSB formation but not in entry to meiosis. This result strongly suggests that <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is involved in meiotic DSB formation and its regulation.</p><p id="p-36">In meiotic DSB formation, the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ mutant showed an altered distribution of DSBs on chromosomes III and VII (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5</a>). Recently, it was reported that the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a>/CCAN subcomplex of the kinetochore protein complex is required to suppress centromere-proximal COs via DSB formation independently from the homologous chromosome pairing mediated by centromere-located <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Vincenten <em>et al.</em> 2015</a>). In the case of <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆, we did not observe a clear correlation between the accumulation of meiotic DSBs and increase in CO formation. For example, we observed a high CO frequency in the <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-MAT</em> interval, but we did not observe distinguishable differences in DSB distribution between <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ and wild type. In addition, we did not observe any defect in the appearance of class I <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/FigureS1.pdf">Figure S1A</a>), which is the centromere-located form of <a href="http://www.yeastgenome.org/locus/S000002693/overview">Zip1</a>, and is not dependent on the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a>/CCAN subcomplex (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Vincenten <em>et al.</em> 2015</a>). Thus, the functional relationship between <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> and the kinetochore complex in the suppression of centromere-proximal COs still remains unknown.</p><p id="p-37">The finding that CO interference was abolished in the <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a></em>∆ cells, specifically in the intervals that contain the centromere in two different chromosomes (<em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-MAT</em> on chromosome III and <em><a href="http://www.yeastgenome.org/locus/S000002994/overview">TRP5</a>-<a href="http://www.yeastgenome.org/locus/S000003293/overview">ADE6</a></em> on chromosome VII), suggests that (i) abnormal CO formation was promoted in the centromere-proximal region in the absence of <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a>, or (ii) recruitment of the <a href="http://www.yeastgenome.org/locus/S000001891/overview">Msh4</a>/<a href="http://www.yeastgenome.org/locus/S000002313/overview">Msh5</a> complex, which is an essential factor for CO control (<a id="xref-ref-45-7" class="xref-bibr" href="#ref-45">Shinohara <em>et al.</em> 2008</a>), to the DSBs might have been affected in this region. It is important to note that genetic NCO frequency was not affected in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆ (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1/TableS4.pdf">Table S4</a>). Thus, control-free CO formation would be activated in the centromere-proximal region in the absence of <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a>. In contrast, we observed stronger interference in the <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> region not only in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆, but also in <em><a href="http://www.yeastgenome.org/locus/S000000432/overview">slx1</a></em>∆. COs in this interval originate from DSBs within a strong artificial <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a>-<a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em> hot spot. This suggests that <a href="http://www.yeastgenome.org/locus/S000000432/overview">Slx1</a>-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> function might be involved in CO control, specifically at quite strong hot spots.</p><p id="p-38"><a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a>, with <a href="http://www.yeastgenome.org/locus/S000001197/overview">Rtt107</a> as a binding partner, functions as a negative regulator of <a href="http://www.yeastgenome.org/locus/S000002625/overview">Rad9</a> through competitive interaction with <a href="http://www.yeastgenome.org/locus/S000003626/overview">Dpb11</a> in the <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> pathway (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Ohouo <em>et al.</em> 2013</a>), and <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> phosphorylation is required for this function (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Ohouo <em>et al.</em> 2010</a>). However, <a href="http://www.yeastgenome.org/locus/S000002625/overview">Rad9</a> and <a href="http://www.yeastgenome.org/locus/S000006074/overview">Rad53</a> activities are excluded from the <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-dependent programmed DSB-related <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> activation pathway (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cartagena-Lirola <em>et al.</em> 2008</a>). We observed an accumulation of <a href="http://www.yeastgenome.org/locus/S000001334/overview">Hop1</a> phosphorylation at T318, which is a <a href="http://www.yeastgenome.org/locus/S000000184/overview">Tel1</a>/<a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> phosphorylation site (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carballo and Cha 2007</a>), in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆, even with the slightly decreased amount of DSB formation at early time points. This suggested that Slx4-Rtt107 functions as a negative regulator of <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> activation even in the absence of <a href="http://www.yeastgenome.org/locus/S000002625/overview">Rad9</a> activation. <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> activation is required for negative regulation of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-dependent meiotic DSB formation through <a href="http://www.yeastgenome.org/locus/S000000340/overview">Mec1</a> activation (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Carballo <em>et al.</em> 2013</a>). As we observed a slight reduction in the amount of <a href="http://www.yeastgenome.org/locus/S000001014/overview">Spo11</a>-oligo in <em><a href="http://www.yeastgenome.org/locus/S000004125/overview">slx4</a></em>∆, <a href="http://www.yeastgenome.org/locus/S000001197/overview">Rtt107</a>-<a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> might be involved in regulating the formation of meiotic DSBs.</p><p id="p-39">Thus, <a href="http://www.yeastgenome.org/locus/S000004125/overview">Slx4</a> is required for the normal distribution of COs on each homolog-pair through meiotic DSB formation and CO control, especially in the centromere-proximal region.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">We are grateful to Ms. K. Tsunezumi, Ms. A. Tokumura, and Ms. A. Murakami for technical assistance. We thank Dr. A. Shinohara, Dr. M. Terasawa, Dr. K. Matsuzaki, and Dr. K. Challa for critical reading of the manuscript, Dr. T.T. Saito for helpful discussions, and all members of the Shinohara Lab for daily discussions and support. This work was supported by the Japan Society for the Promotion of Science (JSPS) through the NEXT program to M.S.</p><p id="p-41">Author contributions: M.H. and M.S. designed and performed the experiments. M.H. and M.S. analyzed the data. M.S. wrote and M.H. checked the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-42">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029488/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-43"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-02-18"><span class="received-label">Received </span>February 18, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-05-04"><span class="accepted-label">Accepted </span>May 4, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Higashide and Shinohara</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.1" data-doi="10.1016/S0092-8674(00)00029-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agarwal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Zip3 provides a link between recombination enzymes and synaptonemal complex proteins.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAgarwal%26rft.auinit1%253DS.%26rft.volume%253D102%26rft.issue%253D2%26rft.spage%253D245%26rft.epage%253D255%26rft.atitle%253DZip3%2Bprovides%2Ba%2Blink%2Bbetween%2Brecombination%2Benzymes%2Band%2Bsynaptonemal%2Bcomplex%2Bproteins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252900029-5%26rft_id%253Dinfo%253Apmid%252F10943844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)00029-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10943844&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088366800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.2" data-doi="10.1371/journal.pone.0096648"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bani Ismail</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Dot1-dependent histone H3K79 methylation promotes the formation of meiotic double-strand breaks in the absence of histone H3K4 methylation in budding yeast.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e96648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De96648%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0096648%26rft_id%253Dinfo%253Apmid%252F24797370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0096648&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24797370&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.3" data-doi="10.1073/pnas.94.10.5213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baudat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Clustering of meiotic double-strand breaks on yeast chromosome III.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">5213</span>–<span class="cit-lpage">5218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBaudat%26rft.auinit1%253DF.%26rft.volume%253D94%26rft.issue%253D10%26rft.spage%253D5213%26rft.epage%253D5218%26rft.atitle%253DClustering%2Bof%2Bmeiotic%2Bdouble-strand%2Bbreaks%2Bon%2Byeast%2Bchromosome%2BIII%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.94.10.5213%26rft_id%253Dinfo%253Apmid%252F9144217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTQvMTAvNTIxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.4" data-doi="10.1016/0092-8674(94)90038-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">RecA homologs Dmc1 and Rad51 interact to form multiple nuclear complexes prior to meiotic chromosome synapsis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1081</span>–<span class="cit-lpage">1092</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBishop%26rft.auinit1%253DD.%2BK.%26rft.volume%253D79%26rft.issue%253D6%26rft.spage%253D1081%26rft.epage%253D1092%26rft.atitle%253DRecA%2Bhomologs%2BDmc1%2Band%2BRad51%2Binteract%2Bto%2Bform%2Bmultiple%2Bnuclear%2Bcomplexes%2Bprior%2Bto%2Bmeiotic%2Bchromosome%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990038-8%26rft_id%253Dinfo%253Apmid%252F7528104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90038-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7528104&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PY08600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.5" data-doi="10.1038/emboj.2008.257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robine</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonfils</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geli</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Histone H3 lysine 4 trimethylation marks meiotic recombination initiation sites.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DBorde%26rft.auinit1%253DV.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D99%26rft.epage%253D111%26rft.atitle%253DHistone%2BH3%2Blysine%2B4%2Btrimethylation%2Bmarks%2Bmeiotic%2Brecombination%2Binitiation%2Bsites%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2008.257%26rft_id%253Dinfo%253Apmid%252F19078966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo3OiIyOC8yLzk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.6" data-doi="10.1038/emboj.2009.207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barthes</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Massy</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Distinct histone modifications define initiation and repair of meiotic recombination in the mouse.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2616</span>–<span class="cit-lpage">2624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DBuard%26rft.auinit1%253DJ.%26rft.volume%253D28%26rft.issue%253D17%26rft.spage%253D2616%26rft.epage%253D2624%26rft.atitle%253DDistinct%2Bhistone%2Bmodifications%2Bdefine%2Binitiation%2Band%2Brepair%2Bof%2Bmeiotic%2Brecombination%2Bin%2Bthe%2Bmouse%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2009.207%26rft_id%253Dinfo%253Apmid%252F19644444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTcvMjYxNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.7" data-doi="10.1371/journal.pbio.0050324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buhler</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping meiotic single-strand DNA reveals a new landscape of DNA double-strand breaks in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DBuhler%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De324%26rft.epage%253De324%26rft.atitle%253DMapping%2Bmeiotic%2Bsingle-strand%2BDNA%2Breveals%2Ba%2Bnew%2Blandscape%2Bof%2BDNA%2Bdouble-strand%2Bbreaks%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050324%26rft_id%253Dinfo%253Apmid%252F18076285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076285&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.8" data-doi="10.1007/s10577-007-1145-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carballo</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cha</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Meiotic roles of Mec1, a budding yeast homolog of mammalian ATR/ATM.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DCarballo%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D539%26rft.epage%253D550%26rft.atitle%253DMeiotic%2Broles%2Bof%2BMec1%252C%2Ba%2Bbudding%2Byeast%2Bhomolog%2Bof%2Bmammalian%2BATR%252FATM.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-007-1145-y%26rft_id%253Dinfo%253Apmid%252F17674144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-007-1145-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17674144&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249011500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.9" data-doi="10.1016/j.cell.2008.01.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carballo</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sedgwick</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cha</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phosphorylation of the axial element protein Hop1 by Mec1/Tel1 ensures meiotic interhomolog recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">758</span>–<span class="cit-lpage">770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCarballo%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D132%26rft.issue%253D5%26rft.spage%253D758%26rft.epage%253D770%26rft.atitle%253DPhosphorylation%2Bof%2Bthe%2Baxial%2Belement%2Bprotein%2BHop1%2Bby%2BMec1%252FTel1%2Bensures%2Bmeiotic%2Binterhomolog%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.01.035%26rft_id%253Dinfo%253Apmid%252F18329363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.01.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18329363&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253818000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.10" data-doi="10.1371/journal.pgen.1003545"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carballo</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panizza</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serrentino</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geymonat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Budding yeast ATM/ATR control meiotic double-strand break (DSB) levels by down-regulating Rec114, an essential component of the DSB-machinery.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003545%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003545%26rft_id%253Dinfo%253Apmid%252F23825959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003545&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23825959&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.11" data-doi="10.1128/MCB.00375-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cartagena-Lirola</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guerini</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfrini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucchini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longhese</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Role of the <em>Saccharomyces cerevisiae</em> Rad53 checkpoint kinase in signaling double-strand breaks during the meiotic cell cycle.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">4480</span>–<span class="cit-lpage">4493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00375-08%26rft_id%253Dinfo%253Apmid%252F18505828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xNC80NDgwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.12" data-doi="10.1016/S0092-8674(00)81164-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Zip2, a meiosis-specific protein required for the initiation of chromosome synapsis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChua%26rft.auinit1%253DP.%2BR.%26rft.volume%253D93%26rft.issue%253D3%26rft.spage%253D349%26rft.epage%253D359%26rft.atitle%253DZip2%252C%2Ba%2Bmeiosis-specific%2Bprotein%2Brequired%2Bfor%2Bthe%2Binitiation%2Bof%2Bchromosome%2Bsynapsis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981164-2%26rft_id%253Dinfo%253Apmid%252F9590170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81164-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9590170&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073471500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Santos</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loidl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Mus81/Mms4 endonuclease acts independently of double-Holliday junction resolution to promote a distinct subset of crossovers during meiosis in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253Dde%2Blos%2BSantos%26rft.auinit1%253DT.%26rft.volume%253D164%26rft.issue%253D1%26rft.spage%253D81%26rft.epage%253D94%26rft.atitle%253DThe%2BMus81%252FMms4%2BEndonuclease%2BActs%2BIndependently%2Bof%2BDouble-Holliday%2BJunction%2BResolution%2Bto%2BPromote%2Ba%2BDistinct%2BSubset%2Bof%2BCrossovers%2BDuring%2BMeiosis%2Bin%2BBudding%2BYeast%26rft_id%253Dinfo%253Apmid%252F12750322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTY0LzEvODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjAzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.14" data-doi="10.1016/j.molcel.2012.02.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Muyt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jessop</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sourirajan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BLM helicase ortholog Sgs1 is a central regulator of meiotic recombination intermediate metabolism.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DDe%2BMuyt%26rft.auinit1%253DA.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D53%26rft.atitle%253DBLM%2Bhelicase%2Bortholog%2BSgs1%2Bis%2Ba%2Bcentral%2Bregulator%2Bof%2Bmeiotic%2Brecombination%2Bintermediate%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.02.020%26rft_id%253Dinfo%253Apmid%252F22500736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.02.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22500736&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302838000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.15" data-doi="10.1128/MCB.22.4.1106-1115.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diaz</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alcid</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Identification of residues in yeast Spo11p critical for meiotic DNA double-strand break formation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1106</span>–<span class="cit-lpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DDiaz%26rft.auinit1%253DR.%2BL.%26rft.volume%253D22%26rft.issue%253D4%26rft.spage%253D1106%26rft.epage%253D1115%26rft.atitle%253DIdentification%2Bof%2BResidues%2Bin%2BYeast%2BSpo11p%2BCritical%2Bfor%2BMeiotic%2BDNA%2BDouble-Strand%2BBreak%2BFormation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.4.1106-1115.2002%26rft_id%253Dinfo%253Apmid%252F11809802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjIyLzQvMTEwNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.16" data-doi="10.1128/MCB.00135-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flott</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alabert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toh</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toth</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugawara</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Phosphorylation of Slx4 by Mec1 and Tel1 regulates the single-strand annealing mode of DNA repair in budding yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">6433</span>–<span class="cit-lpage">6445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00135-07%26rft_id%253Dinfo%253Apmid%252F17636031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNy8xOC82NDMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.17" data-doi="10.1101/gad.1105203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fricke</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brill</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Slx1-Slx4 is a second structure-specific endonuclease functionally redundant with Sgs1-Top3.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1768</span>–<span class="cit-lpage">1778</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1105203%26rft_id%253Dinfo%253Apmid%252F12832395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE3LzE0LzE3NjgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjAzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gothwal</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colletti</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasanuma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> The double-strand break landscape of meiotic chromosomes is shaped by the Paf1 transcription elongation complex in <em>Saccharomyces cerevisiae</em>. Genetics 202: 497–512.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grushcow</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holzen</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinert</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> checkpoint genes <em>MEC1</em>, <em>RAD17</em> and <em>RAD24</em> are required for normal meiotic recombination partner choice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">607</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGrushcow%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D153%26rft.issue%253D2%26rft.spage%253D607%26rft.epage%253D620%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BCheckpoint%2BGenes%2BMEC1%252C%2BRAD17%2Band%2BRAD24%2BAre%2BRequired%2Bfor%2BNormal%2BMeiotic%2BRecombination%2BPartner%2BChoice%26rft_id%253Dinfo%253Apmid%252F10511543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUzLzIvNjA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.20" data-doi="10.1016/j.cell.2004.10.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayase</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takagi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyazaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oshiumi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A protein complex containing Mei5 and Sae3 promotes the assembly of the meiosis-specific RecA homolog Dmc1.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">927</span>–<span class="cit-lpage">940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHayase%26rft.auinit1%253DA.%26rft.volume%253D119%26rft.issue%253D7%26rft.spage%253D927%26rft.epage%253D940%26rft.atitle%253DA%2Bprotein%2Bcomplex%2Bcontaining%2BMei5%2Band%2BSae3%2Bpromotes%2Bthe%2Bassembly%2Bof%2Bthe%2Bmeiosis-specific%2BRecA%2Bhomolog%2BDmc1.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2004.10.031%26rft_id%253Dinfo%253Apmid%252F15620352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2004.10.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15620352&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226109700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.21" data-doi="10.1101/gad.9.14.1728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponte</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halsey</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">MSH5, a novel MutS homolog, facilitates meiotic reciprocal recombination between homologs in <em>Saccharomyces cerevisiae</em> but not mismatch repair.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1728</span>–<span class="cit-lpage">1739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHollingsworth%26rft.auinit1%253DN%2BM%26rft.volume%253D9%26rft.issue%253D14%26rft.spage%253D1728%26rft.epage%253D1739%26rft.atitle%253DMSH5%252C%2Ba%2Bnovel%2BMutS%2Bhomolog%252C%2Bfacilitates%2Bmeiotic%2Breciprocal%2Brecombination%2Bbetween%2Bhomologs%2Bin%2BSaccharomyces%2Bcerevisiae%2Bbut%2Bnot%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.14.1728%26rft_id%253Dinfo%253Apmid%252F7622037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOS8xNC8xNzI4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.22" data-doi="10.1371/journal.pgen.1005942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwasaki</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashihara</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shima</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higashide</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terasawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The MRX Complex Ensures NHEJ Fidelity through Multiple Pathways Including Xrs2-FHA-Dependent Tel1 Activation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet%26rft.volume%253D12%26rft.spage%253De1005942%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005942%26rft_id%253Dinfo%253Apmid%252F26990569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26990569&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.23" data-doi="10.1016/S0070-2153(01)52008-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Mechanism and control of meiotic recombination initiation.</span> <abbr class="cit-jnl-abbrev">Curr. Top. Dev. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Btopics%2Bin%2Bdevelopmental%2Bbiology%26rft.stitle%253DCurr%2BTop%2BDev%2BBiol%26rft.aulast%253DKeeney%26rft.auinit1%253DS.%26rft.volume%253D52%26rft.spage%253D1%26rft.epage%253D53%26rft.atitle%253DMechanism%2Band%2Bcontrol%2Bof%2Bmeiotic%2Brecombination%2Binitiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0070-2153%252801%252952008-6%26rft_id%253Dinfo%253Apmid%252F11529427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0070-2153(01)52008-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11529427&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171181800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.24" data-doi="10.1038/ng.750"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lach</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desetty</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanenberg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auerbach</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutations of the SLX4 gene in Fanconi anemia.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKim%26rft.auinit1%253DY.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D142%26rft.epage%253D146%26rft.atitle%253DMutations%2Bof%2Bthe%2BSLX4%2Bgene%2Bin%2BFanconi%2Banemia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.750%26rft_id%253Dinfo%253Apmid%252F21240275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.750&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21240275&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.25" data-doi="10.1016/j.cell.2011.09.038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Massy</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The impressionistic landscape of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLichten%26rft.auinit1%253DM.%26rft.volume%253D147%26rft.issue%253D2%26rft.spage%253D267%26rft.epage%253D270%26rft.atitle%253DThe%2Bimpressionistic%2Blandscape%2Bof%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.09.038%26rft_id%253Dinfo%253Apmid%252F22000007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.09.038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22000007&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296199300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.26" data-doi="10.1038/nsmb.2359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mazon</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kupiec</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symington</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Rad1-Rad10 nuclease promotes chromosome translocations between dispersed repeats.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">964</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DMazon%26rft.auinit1%253DG.%26rft.volume%253D19%26rft.issue%253D9%26rft.spage%253D964%26rft.epage%253D971%26rft.atitle%253DThe%2BRad1-Rad10%2Bnuclease%2Bpromotes%2Bchromosome%2Btranslocations%2Bbetween%2Bdispersed%2Brepeats.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2359%26rft_id%253Dinfo%253Apmid%252F22885325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2359&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22885325&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mullen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaliraman</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibrahim</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brill</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Requirement for three novel protein complexes in the absence of the Sgs1 DNA helicase in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMullen%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D157%26rft.issue%253D1%26rft.spage%253D103%26rft.epage%253D118%26rft.atitle%253DRequirement%2Bfor%2BThree%2BNovel%2BProtein%2BComplexes%2Bin%2Bthe%2BAbsence%2Bof%2Bthe%2BSgs1%2BDNA%2BHelicase%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F11139495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU3LzEvMTAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.28" data-doi="10.1016/j.molcel.2009.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hain</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Declais</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toh</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Coordination of structure-specific nucleases by human SLX4/BTBD12 is required for DNA repair.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DMunoz%26rft.auinit1%253DI.%2BM.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D116%26rft.epage%253D127%26rft.atitle%253DCoordination%2Bof%2Bstructure-specific%2Bnucleases%2Bby%2Bhuman%2BSLX4%252FBTBD12%2Bis%2Brequired%2Bfor%2BDNA%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.06.020%26rft_id%253Dinfo%253Apmid%252F19595721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19595721&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268003300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.29" data-doi="10.1128/MCB.00111-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Munoz-Galvan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tous</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehmsen</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Distinct roles of Mus81, Yen1, Slx1-Slx4, and Rad1 nucleases in the repair of replication-born double-strand breaks by sister chromatid exchange.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1592</span>–<span class="cit-lpage">1603</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00111-12%26rft_id%253Dinfo%253Apmid%252F22354996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjMyLzkvMTU5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.30" data-doi="10.1371/journal.pgen.1001062"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic analysis of baker’s yeast Msh4-Msh5 reveals a threshold crossover level for meiotic viability.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: e1001083<span class="cit-fpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DShu%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253D6%26rft.atitle%253DIdentification%2Bof%2Bnew%2Bgenetic%2Brisk%2Bvariants%2Bfor%2Btype%2B2%2Bdiabetes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001062%26rft_id%253Dinfo%253Apmid%252F20862305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001062&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20862305&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.31" data-doi="10.1016/j.molcel.2010.06.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohouo</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastos de Oliveira</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smolka</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">DNA damage signaling recruits the Rtt107-Slx4 scaffolds via Dpb11 to mediate replication stress response.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">300</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DOhouo%26rft.auinit1%253DP.%2BY.%26rft.volume%253D39%26rft.issue%253D2%26rft.spage%253D300%26rft.epage%253D306%26rft.atitle%253DDNA%2Bdamage%2Bsignaling%2Brecruits%2Bthe%2BRtt107-Slx4%2Bscaffolds%2Bvia%2BDpb11%2Bto%2Bmediate%2Breplication%2Bstress%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.06.019%26rft_id%253Dinfo%253Apmid%252F20670896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.06.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20670896&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280574800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.32" data-doi="10.1038/nature11658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohouo</span>  <span class="cit-name-given-names">P. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastos de Oliveira</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smolka</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">DNA-repair scaffolds dampen checkpoint signalling by counteracting the adaptor Rad9.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">493</span>: <span class="cit-fpage">120</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D493%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11658%26rft_id%253Dinfo%253Apmid%252F23160493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11658&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23160493&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.33" data-doi="10.1007/978-1-59745-527-5_13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detection of SPO11-oligonucleotide complexes from mouse testes.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">557</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DPan%26rft.auinit1%253DJ.%26rft.volume%253D557%26rft.spage%253D197%26rft.epage%253D207%26rft.atitle%253DDetection%2Bof%2BSPO11-oligonucleotide%2Bcomplexes%2Bfrom%2Bmouse%2Btestes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-527-5_13%26rft_id%253Dinfo%253Apmid%252F19799184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-527-5_13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19799184&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.34" data-doi="10.1042/BST0370495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouse</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Control of genome stability by SLX protein complexes.</span> <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSoc.%2BTrans.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBST0370495%26rft_id%253Dinfo%253Apmid%252F19442243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoicHBiaW9zdCI7czo1OiJyZXNpZCI7czo4OiIzNy8zLzQ5NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.35" data-doi="10.4161/worm.28233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colaiacovo</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Crossover recombination mediated by HIM-18/SLX4-associated nucleases.</span> <abbr class="cit-jnl-abbrev">Worm</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e28233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWorm%26rft.volume%253D3%26rft.spage%253De28233%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fworm.28233%26rft_id%253Dinfo%253Apmid%252F25057454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/worm.28233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25057454&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.36" data-doi="10.1371/journal.pgen.1002888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohideen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colaiacovo</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SLX-1 is required for maintaining genomic integrity and promoting meiotic noncrossovers in the <em>Caenorhabditis elegans</em> germline.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSaito%26rft.auinit1%253DT.%2BT.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002888%26rft.epage%253De1002888%26rft.atitle%253DSLX-1%2Bis%2Brequired%2Bfor%2Bmaintaining%2Bgenomic%2Bintegrity%2Band%2Bpromoting%2Bmeiotic%2Bnoncrossovers%2Bin%2Bthe%2BCaenorhabditis%2Belegans%2Bgermline.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002888%26rft_id%253Dinfo%253Apmid%252F22927825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22927825&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.37" data-doi="10.1093/nar/gkl1162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sasanuma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murakami</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shibata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Meiotic association between Spo11 regulated by Rec102, Rec104 and Rec114.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1119</span>–<span class="cit-lpage">1133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl1162%26rft_id%253Dinfo%253Apmid%252F17264124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM1LzQvMTExOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.38" data-doi="10.1038/ncomms2678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sasanuma</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tawaramoto</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lao</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanda</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A new protein complex promoting the assembly of Rad51 filaments.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D4%26rft.spage%253D1676%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms2678%26rft_id%253Dinfo%253Apmid%252F23575680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms2678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23575680&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.39" data-doi="10.1128/MCB.10.6.2485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schiestl</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakash</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title"><em>RAD10</em>, an excision repair gene of <em>Saccharomyces cerevisiae</em>, is involved in the <em>RAD1</em> pathway of mitotic recombination.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2485</span>–<span class="cit-lpage">2491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DSchiestl%26rft.auinit1%253DR%2BH%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D2485%26rft.epage%253D2491%26rft.atitle%253DRAD10%252C%2Ban%2Bexcision%2Brepair%2Bgene%2Bof%2BSaccharomyces%2Bcerevisiae%252C%2Bis%2Binvolved%2Bin%2Bthe%2BRAD1%2Bpathway%2Bof%2Bmitotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.10.6.2485%26rft_id%253Dinfo%253Apmid%252F2188090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEwLzYvMjQ4NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.40" data-doi="10.1534/genetics.104.037580"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shima</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Isolation and characterization of novel xrs2 mutations in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.037580%26rft_id%253Dinfo%253Apmid%252F15716496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTcwLzEvNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjAzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.41" data-doi="10.1371/journal.pone.0063144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multiple pathways suppress non-allelic homologous recombination during meiosis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e63144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De63144%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0063144%26rft_id%253Dinfo%253Apmid%252F23646187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0063144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23646187&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.42" data-doi="10.1046/j.1365-2443.1997.1480347.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasior</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> recA homologues <em>RAD51</em> and <em>DMC1</em> have both distinct and overlapping roles in meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Genes Cells</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DShinohara%26rft.auinit1%253DA%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253D615%26rft.epage%253D629%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BrecA%2Bhomologues%2BRAD51%2Band%2BDMC1%2Bhave%2Bboth%2Bdistinct%2Band%2Boverlapping%2Broles%2Bin%2Bmeiotic%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2443.1997.1480347.x%26rft_id%253Dinfo%253Apmid%252F9427283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2443.1997.1480347.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9427283&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YJ05700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.43" data-doi="10.1073/pnas.97.20.10814"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasior</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Tid1/Rdh54 promotes colocalization of rad51 and dmc1 during meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">10814</span>–<span class="cit-lpage">10819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DShinohara%26rft.auinit1%253DM.%26rft.volume%253D97%26rft.issue%253D20%26rft.spage%253D10814%26rft.epage%253D10819%26rft.atitle%253DTid1%252FRdh54%2Bpromotes%2Bcolocalization%2Bof%2BRad51%2Band%2BDmc1%2Bduring%2Bmeiotic%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.20.10814%26rft_id%253Dinfo%253Apmid%252F11005857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTcvMjAvMTA4MTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMjAzMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The mitotic DNA damage checkpoint proteins Rad17 and Rad24 are required for repair of double-strand breaks during meiosis in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">855</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DShinohara%26rft.auinit1%253DM.%26rft.volume%253D164%26rft.issue%253D3%26rft.spage%253D855%26rft.epage%253D865%26rft.atitle%253DThe%2BMitotic%2BDNA%2BDamage%2BCheckpoint%2BProteins%2BRad17%2Band%2BRad24%2BAre%2BRequired%2Bfor%2BRepair%2Bof%2BDouble-Strand%2BBreaks%2BDuring%2BMeiosis%2Bin%2BYeast%26rft_id%253Dinfo%253Apmid%252F12871899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzMvODU1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.45" data-doi="10.1038/ng.83"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oh</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Crossover assurance and crossover interference are distinctly regulated by the ZMM proteins during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DShinohara%26rft.auinit1%253DM.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D309%26rft.atitle%253DCrossover%2Bassurance%2Band%2Bcrossover%2Binterference%2Bare%2Bdistinctly%2Bregulated%2Bby%2Bthe%2BZMM%2Bproteins%2Bduring%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.83%26rft_id%253Dinfo%253Apmid%252F18297071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.83&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18297071&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253548400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashihara</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grubb</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">DNA damage response clamp 9-1-1 contributes to chromosomal assembly of ZMM/SIC proteins for formation of crossovers and synaptonemal complex during meiosis.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> 128: 1494–1506.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.47" data-doi="10.1016/j.molcel.2004.06.040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snowden</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berardini</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishel</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">hMSH4-hMSH5 recognizes Holliday Junctions and forms a meiosis-specific sliding clamp that embraces homologous chromosomes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DSnowden%26rft.auinit1%253DT.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D437%26rft.epage%253D451%26rft.atitle%253DhMSH4-hMSH5%2Brecognizes%2BHolliday%2BJunctions%2Band%2Bforms%2Ba%2Bmeiosis-specific%2Bsliding%2Bclamp%2Bthat%2Bembraces%2Bhomologous%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2004.06.040%26rft_id%253Dinfo%253Apmid%252F15304223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2004.06.040&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15304223&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223312300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.48" data-doi="10.1038/ng.751"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stoepker</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hain</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuster</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hilhorst-Hofstee</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooimans</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SLX4, a coordinator of structure-specific endonucleases, is mutated in a new Fanconi anemia subtype.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">138</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DStoepker%26rft.auinit1%253DC.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D138%26rft.epage%253D141%26rft.atitle%253DSLX4%252C%2Ba%2Bcoordinator%2Bof%2Bstructure-specific%2Bendonucleases%252C%2Bis%2Bmutated%2Bin%2Ba%2Bnew%2BFanconi%2Banemia%2Bsubtype.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.751%26rft_id%253Dinfo%253Apmid%252F21240277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.751&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21240277&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.49" data-doi="10.1073/pnas.93.17.9043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storlazzi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwacha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Synaptonemal complex (SC) component Zip1 plays a role in meiotic recombination independent of SC polymerization along the chromosomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">9043</span>–<span class="cit-lpage">9048</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DStorlazzi%26rft.auinit1%253DA.%26rft.volume%253D93%26rft.issue%253D17%26rft.spage%253D9043%26rft.epage%253D9048%26rft.atitle%253DSynaptonemal%2Bcomplex%2B%2528SC%2529%2Bcomponent%2BZip1%2Bplays%2Ba%2Brole%2Bin%2Bmeiotic%2Brecombination%2Bindependent%2Bof%2BSC%2Bpolymerization%2Balong%2Bthe%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.17.9043%26rft_id%253Dinfo%253Apmid%252F8799151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTMvMTcvOTA0MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8yMDMzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.50" data-doi="10.1016/j.dnarep.2010.02.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toh</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugawara</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toth</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mec1/Tel1-dependent phosphorylation of Slx4 stimulates Rad1-Rad10-dependent cleavage of non-homologous DNA tails.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">718</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D9%26rft.spage%253D718%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2010.02.013%26rft_id%253Dinfo%253Apmid%252F20382573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2010.02.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20382573&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.51" data-doi="10.1016/j.devcel.2006.04.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsubouchi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The meiosis-specific zip4 protein regulates crossover distribution by promoting synaptonemal complex formation together with <em>zip2</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DTsubouchi%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D809%26rft.epage%253D819%26rft.atitle%253DThe%2Bmeiosis-specific%2Bzip4%2Bprotein%2Bregulates%2Bcrossover%2Bdistribution%2Bby%2Bpromoting%2Bsynaptonemal%2Bcomplex%2Bformation%2Btogether%2Bwith%2Bzip2.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2006.04.003%26rft_id%253Dinfo%253Apmid%252F16740482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2006.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16740482&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238244700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vincenten</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhl</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oke</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The kinetochore prevents centromere-proximal crossover recombination during meiosis.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: e10850.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.53" data-doi="10.1002/yea.320101310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brachat</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pohlmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippsen</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">New heterologous modules for classical or PCR-based gene disruptions in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1793</span>–<span class="cit-lpage">1808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWach%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.issue%253D13%26rft.spage%253D1793%26rft.epage%253D1808%26rft.atitle%253DNew%2Bheterologous%2Bmodules%2Bfor%2Bclassical%2Bor%2BPCR-based%2Bgene%2Bdisruptions%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320101310%26rft_id%253Dinfo%253Apmid%252F7747518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320101310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7747518&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994QJ42800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.54" data-doi="10.1128/MCB.15.12.6572"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ajimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padmore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title"><em>NDT80</em>, a meiosis-specific gene required for exit from pachytene in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">6572</span>–<span class="cit-lpage">6581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DXu%26rft.auinit1%253DL.%26rft.volume%253D15%26rft.issue%253D12%26rft.spage%253D6572%26rft.epage%253D6581%26rft.atitle%253DNDT80%252C%2Ba%2Bmeiosis-specific%2Bgene%2Brequired%2Bfor%2Bexit%2Bfrom%2Bpachytene%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.15.12.6572%26rft_id%253Dinfo%253Apmid%252F8524222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMi82NTcyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzIwMzMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.2033.55" data-doi="10.1016/j.cell.2012.03.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zakharyevich</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Delineation of joint molecule resolution pathways in meiosis identifies a crossover-specific resolvase.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">334</span>–<span class="cit-lpage">347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DZakharyevich%26rft.auinit1%253DK.%26rft.volume%253D149%26rft.issue%253D2%26rft.spage%253D334%26rft.epage%253D347%26rft.atitle%253DDelineation%2Bof%2Bjoint%2Bmolecule%2Bresolution%2Bpathways%2Bin%2Bmeiosis%2Bidentifies%2Ba%2Bcrossover-specific%2Bresolvase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.03.023%26rft_id%253Dinfo%253Apmid%252F22500800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.03.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22500800&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F2033.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302805600013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/7/2033.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/2023" title="Functional Genetic Screen to Identify Interneurons Governing Behaviorally Distinct Aspects of Drosophila Larval Motor Programs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/2043" title="Exhaustive Genome-Wide Search for SNP-SNP Interactions Across 10 Human Diseases" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/7"><span>Volume 6 Issue 7, July 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/7.cover-source.jpg?itok=E84OOyGI" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/2033" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DitibcB3oWUT_RUt66YTtEKGckvO9shv1cpY93OKCco" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/7/2033" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/7/2033" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/7/2033" class="active" data-icon-position="" data-hide-link-title="0">Budding Yeast SLX4 Contributes to the Appropriate Distribution of Crossovers and Meiotic Double-Strand Break Formation on Bivalents During Meiosis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460406" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-AzS3F7iHzc5TA9_7kh2wScivGbPw0q1YnJXm3UDoZns" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/2033.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/7/2033" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-E4LzOefaFJSetQ3A-sqoW6ckoFvuLxvd69mDm14I9wU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/7/2033" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/2033.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460406" id="node460406--2" data-pisa="ggg;6/7/2033" data-pisa-master="ggg;g3.116.029488" data-apath="/ggg/6/7/2033.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Budding Yeast <em>SLX4</em> Contributes to the Appropriate Distribution of Crossovers and Meiotic Double-Strand Break Formation on Bivalents During Meiosis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mika Higashide</span> and <span class="highwire-citation-author" data-delta="1">Miki Shinohara</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2033-2042;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029488 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460406--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mika Higashide </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrated Protein Functions, Institute for Protein Research, Osaka University, 565-0871, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biological Science, Graduate School of Science, Osaka University, 565-0871, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mika%2BHigashide%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Higashide%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMika%2BHigashide%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Miki Shinohara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrated Protein Functions, Institute for Protein Research, Osaka University, 565-0871, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biological Science, Graduate School of Science, Osaka University, 565-0871, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miki%2BShinohara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shinohara%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiki%2BShinohara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mikis@protein.osaka-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">mikis@protein.osaka-u.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/7/2033" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/7/2033&amp;title=Budding%20Yeast%20SLX4%20Contributes%20to%20the%20Appropriate%20Distribution%20of%20Crossovers%20and%20Meiotic%20Double-Strand%20Break%20Formation%20on%20Bivalents%20During%20Meiosis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/7/2033&amp;title=Budding%20Yeast%20SLX4%20Contributes%20to%20the%20Appropriate%20Distribution%20of%20Crossovers%20and%20Meiotic%20Double-Strand%20Break%20Formation%20on%20Bivalents%20During%20Meiosis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/7/2033&amp;title=Budding%20Yeast%20SLX4%20Contributes%20to%20the%20Appropriate%20Distribution%20of%20Crossovers%20and%20Meiotic%20Double-Strand%20Break%20Formation%20on%20Bivalents%20During%20Meiosis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/7/2033&amp;text=Budding%20Yeast%20SLX4%20Contributes%20to%20the%20Appropriate%20Distribution%20of%20Crossovers%20and%20Meiotic%20Double-Strand%20Break%20Formation%20on%20Bivalents%20During%20Meiosis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/7/2033&amp;title=Budding%20Yeast%20SLX4%20Contributes%20to%20the%20Appropriate%20Distribution%20of%20Crossovers%20and%20Meiotic%20Double-Strand%20Break%20Formation%20on%20Bivalents%20During%20Meiosis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/7/2033&amp;t=Budding%20Yeast%20SLX4%20Contributes%20to%20the%20Appropriate%20Distribution%20of%20Crossovers%20and%20Meiotic%20Double-Strand%20Break%20Formation%20on%20Bivalents%20During%20Meiosis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/7/2033&amp;title=Budding%20Yeast%20SLX4%20Contributes%20to%20the%20Appropriate%20Distribution%20of%20Crossovers%20and%20Meiotic%20Double-Strand%20Break%20Formation%20on%20Bivalents%20During%20Meiosis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/7/2033&amp;title=Budding%20Yeast%20SLX4%20Contributes%20to%20the%20Appropriate%20Distribution%20of%20Crossovers%20and%20Meiotic%20Double-Strand%20Break%20Formation%20on%20Bivalents%20During%20Meiosis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460406/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460406" id="node460406--3" data-pisa="ggg;6/7/2033" data-pisa-master="ggg;g3.116.029488" data-apath="/ggg/6/7/2033.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Budding Yeast <em>SLX4</em> Contributes to the Appropriate Distribution of Crossovers and Meiotic Double-Strand Break Formation on Bivalents During Meiosis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Mika Higashide</span> and <span class="highwire-citation-author" data-delta="1">Miki Shinohara</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2033-2042;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029488 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460406--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mika Higashide </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrated Protein Functions, Institute for Protein Research, Osaka University, 565-0871, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biological Science, Graduate School of Science, Osaka University, 565-0871, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mika%2BHigashide%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Higashide%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMika%2BHigashide%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Miki Shinohara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrated Protein Functions, Institute for Protein Research, Osaka University, 565-0871, Japan</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biological Science, Graduate School of Science, Osaka University, 565-0871, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miki%2BShinohara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shinohara%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiki%2BShinohara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mikis@protein.osaka-u.ac.jp" class="" data-icon-position="" data-hide-link-title="0">mikis@protein.osaka-u.ac.jp</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2033&amp;rft.epage=2042&amp;rft.atitle=Budding%20Yeast%20SLX4%20Contributes%20to%20the%20Appropriate%20Distribution%20of%20Crossovers%20and%20Meiotic%20Double-Strand%20Break%20Formation%20on%20Bivalents%20During%20Meiosis&amp;rft.volume=6&amp;rft.issue=7&amp;rft.date=2016-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Higashide%2C+Mika&amp;rft.au=Shinohara%2C+Miki"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460406/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460406/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460406/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460406/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460406/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460406/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460406/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460406/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460406/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460406/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460406/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460406/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/7/2033" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/7/2033.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/7/2033.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/7/2033.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/7/2033.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
