<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/7/1835" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/7/1835.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/7/1835.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/7/1835.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/7/1835.atom" />
<meta name="HW.pisa" content="ggg;6/7/1835" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="An Improved Genome Assembly of Azadirachta indica A. Juss." />
<meta name="DC.Identifier" content="10.1534/g3.116.030056" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Krishnan et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="10.1534/g3.119.400327" />
<meta name="DC.Description" content="Neem ( Azadirachta indica A. Juss.), an evergreen tree of the Meliaceae family, is known for its medicinal, cosmetic, pesticidal and insecticidal properties. We had previously sequenced and published the draft genome of a neem plant, using mainly short read sequencing data. In this report, we present an improved genome assembly generated using additional short reads from Illumina and long reads from Pacific Biosciences SMRT sequencer. We assembled short reads and error-corrected long reads using Platanus, an assembler designed to perform well for heterozygous genomes. The updated genome assembly (v2.0) yielded 3- and 3.5-fold increase in N50 and N75, respectively; 2.6-fold decrease in the total number of scaffolds; 1.25-fold increase in the number of valid transcriptome alignments; 13.4-fold less misassembly and 1.85-fold increase in the percentage repeat, over the earlier assembly (v1.0). The current assembly also maps better to the genes known to be involved in the terpenoid biosynthesis pathway. Together, the data represent an improved assembly of the A. indica genome." />
<meta name="DC.Contributor" content="Neeraja M. Krishnan" />
<meta name="DC.Contributor" content="Prachi Jain" />
<meta name="DC.Contributor" content="Saurabh Gupta" />
<meta name="DC.Contributor" content="Arun K. Hariharan" />
<meta name="DC.Contributor" content="Binay Panda" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="An Improved Genome Assembly of Azadirachta indica A. Juss." />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Neem (&lt;i&gt;Azadirachta indica&lt;/i&gt; A. Juss.), an evergreen tree of the Meliaceae family, is known for its medicinal, cosmetic, pesticidal and insecticidal properties. We had previously sequenced and published the draft genome of a neem plant, using mainly short read sequencing data. In this report, we present an improved genome assembly generated using additional short reads from Illumina and long reads from Pacific Biosciences SMRT sequencer. We assembled short reads and error-corrected long reads using Platanus, an assembler designed to perform well for heterozygous genomes. The updated genome assembly (v2.0) yielded 3- and 3.5-fold increase in N50 and N75, respectively; 2.6-fold decrease in the total number of scaffolds; 1.25-fold increase in the number of valid transcriptome alignments; 13.4-fold less misassembly and 1.85-fold increase in the percentage repeat, over the earlier assembly (v1.0). The current assembly also maps better to the genes known to be involved in the terpenoid biosynthesis pathway. Together, the data represent an improved assembly of the &lt;i&gt;A. indica&lt;/i&gt; genome.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/07/01" />
<meta name="citation_mjid" content="ggg;6/7/1835" />
<meta name="citation_id" content="6/7/1835" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/7/1835" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/7/1835.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/7/1835.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/7/1835.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.030056" />
<meta name="citation_pmid" content="27172223" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="6" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1835" />
<meta name="citation_lastpage" content="1840" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Neeraja M. Krishnan" />
<meta name="citation_author_institution" content="Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India" />
<meta name="citation_author" content="Prachi Jain" />
<meta name="citation_author_institution" content="Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India" />
<meta name="citation_author" content="Saurabh Gupta" />
<meta name="citation_author_institution" content="Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India" />
<meta name="citation_author" content="Arun K. Hariharan" />
<meta name="citation_author_institution" content="Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India" />
<meta name="citation_author" content="Binay Panda" />
<meta name="citation_author_institution" content="Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India" />
<meta name="citation_author_institution" content="Strand Life Sciences, Bangalore 560024, India" />
<meta name="citation_author_email" content="binay@ganitlabs.in" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=Bioinf.;citation_author=A. Gurevich;citation_author=V. Saveliev;citation_author=N. Vyahhi;citation_author=G. Tesler;citation_title=QUAST: quality assessment tool for genome assemblies.;citation_volume=29;citation_year=2013;citation_pmid=23422339;citation_doi=10.1093/bioinformatics/btt086" />
<meta name="citation_reference" content="Haas, B. J., 2010 Available at: http://transposonpsi.sourceforge.net. Accessed: May 15, 2015." />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. J. Haas;citation_author=A. L. Delcher;citation_author=S. M. Mount;citation_author=J. R. Wortman;citation_author=R. K. Smith;citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies;citation_pages=5654-5666;citation_volume=31;citation_year=2003;citation_issue=19;citation_pmid=14500829;citation_doi=10.1093/nar/gkg770" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=B. J. Haas;citation_author=A. Papanicolaou;citation_author=M. Yassour;citation_author=M. Grabherr;citation_author=P. D. Blood;citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.;citation_pages=1494-1512;citation_volume=8;citation_year=2013;citation_pmid=23845962;citation_doi=10.1038/nprot.2013.084" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Han;citation_author=S. R. Wessler;citation_title=MITE-Hunter: a program for discovering miniature inverted-repeat transposable elements from genomic sequences.;citation_volume=38;citation_year=2010;citation_pmid=20880995;citation_doi=10.1093/nar/gkq862" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Kajitani;citation_author=K. Toshimoto;citation_author=H. Noguchi;citation_author=A. Toyoda;citation_author=Y. Ogura;citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads.;citation_volume=24;citation_year=2014;citation_pmid=24755901;citation_doi=10.1101/gr.170720.113" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Kanehisa;citation_author=S. Goto;citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes;citation_pages=27-30;citation_volume=28;citation_year=2000;citation_issue=1;citation_pmid=10592173;citation_doi=10.1093/nar/28.1.27" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Kanehisa;citation_author=S. Goto;citation_author=Y. Sato;citation_author=M. Kawashima;citation_author=M. Furumichi;citation_title=Data, information, knowledge and principle: back to metabolism in KEGG.;citation_volume=42;citation_year=2014;citation_pmid=24214961;citation_doi=10.1093/nar/gkt1076" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=V. V. Kapitonov;citation_author=J. Jurka;citation_title=A universal classification of eukaryotic transposable elements implemented in Repbase.;citation_pages=411-412;citation_volume=9;citation_year=2008;citation_issue=5;citation_pmid=18421312;citation_doi=10.1038/nrg2165-c1" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=S. Koren;citation_author=M. C. Schatz;citation_author=B. P. Walenz;citation_author=J. Martin;citation_author=J. T. Howard;citation_title=Hybrid error correction and de novo assembly of single-molecule sequencing reads.;citation_pages=693-700;citation_volume=30;citation_year=2012;citation_issue=7;citation_pmid=22750884;citation_doi=10.1038/nbt.2280" />
<meta name="citation_reference" content="citation_journal_title=Curr Sci (India);citation_author=N. M. Krishnan;citation_author=S. Pattnaik;citation_author=S. A. Deepak;citation_author=A. K. Hariharan;citation_author=P. Gaur;citation_title=De novo sequencing and assembly of Azadirachta indica fruit transcriptome.;citation_pages=9;citation_volume=101;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=N. M. Krishnan;citation_author=S. Pattnaik;citation_author=P. Jain;citation_author=P. Gaur;citation_author=R. Choudhary;citation_title=A draft of the genome and four transcriptomes of a medicinal and pesticidal angiosperm Azadirachta indica.;citation_pages=464-464;citation_volume=13;citation_year=2012;citation_pmid=22958331;citation_doi=10.1186/1471-2164-13-464" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Li;citation_author=H. Zhu;citation_author=J. Ruan;citation_author=W. Qian;citation_author=X. Fang;citation_title=De novo assembly of human genomes with massively parallel short read sequencing.;citation_volume=20;citation_year=2010;citation_pmid=20019144;citation_doi=10.1101/gr.097261.109" />
<meta name="citation_reference" content="citation_journal_title=Bioinf.;citation_author=W. Li;citation_author=A. Godzik;citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences.;citation_volume=22;citation_year=2006;citation_pmid=16731699;citation_doi=10.1093/bioinformatics/btl158" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Luo;citation_author=B. Liu;citation_author=Y. Xie;citation_author=Z. Li;citation_author=W. Huang;citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.;citation_pages=18;citation_volume=1;citation_year=2012;citation_pmid=23587118;citation_doi=10.1186/2047-217X-1-18" />
<meta name="citation_reference" content="citation_journal_title=Bioinf.;citation_author=W. H. Majoros;citation_author=M. Pertea;citation_author=S. L. Salzberg;citation_title=TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders.;citation_volume=20;citation_year=2004;citation_pmid=15145805;citation_doi=10.1093/bioinformatics/bth315" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=T. P. Michael;citation_author=S. Jackson;citation_title=The first 50 plant genomes.;citation_pages=7;citation_volume=6;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. W. Myers;citation_author=G. G. Sutton;citation_author=A. L. Delcher;citation_author=I. M. Dew;citation_author=D. P. Fasulo;citation_title=A Whole-Genome Assembly of Drosophila;citation_pages=2196-2204;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731133;citation_doi=10.1126/science.287.5461.2196" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=S. Pattnaik;citation_author=S. Gupta;citation_author=A. A. Rao;citation_author=B. Panda;citation_title=SInC: an accurate and fast error-model based simulator for SNPs, Indels and CNVs coupled with a read generator for short-read sequence data.;citation_pages=40;citation_volume=15;citation_year=2014;citation_pmid=24495296;citation_doi=10.1186/1471-2105-15-40" />
<meta name="citation_reference" content="citation_journal_title=Bioinf.;citation_author=L. Salmela;citation_author=E. Rivals;citation_title=LoRDEC: accurate and efficient long read error correction.;citation_volume=30;citation_year=2014;citation_pmid=25165095;citation_doi=10.1093/bioinformatics/btu538" />
<meta name="citation_reference" content="Smit, A. F. A., R. Hubley, and P. Green, 2014 RepeatMasker Open-4.0. Available at: http://www.repeatmasker.org. Accessed: May 10, 2015." />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=R. Thimmappa;citation_author=K. Geisler;citation_author=T. Louveau;citation_author=P. O’ Maille;citation_author=A. Osbourn;citation_title=Triterpene biosynthesis in plants.;citation_pages=225-257;citation_volume=65;citation_year=2014;citation_pmid=24498976;citation_doi=10.1146/annurev-arplant-050312-120229" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Z. Xu;citation_author=H. Wang;citation_title=LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons.;citation_pages=W265268;citation_volume=35;citation_year=2007" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="An Improved Genome Assembly of Azadirachta indica A. Juss." />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="twitter:description" content="Neem ( Azadirachta indica A. Juss.), an evergreen tree of the Meliaceae family, is known for its medicinal, cosmetic, pesticidal and insecticidal properties. We had previously sequenced and published the draft genome of a neem plant, using mainly short read sequencing data. In this report, we present an improved genome assembly generated using additional short reads from Illumina and long reads from Pacific Biosciences SMRT sequencer. We assembled short reads and error-corrected long reads using Platanus, an assembler designed to perform well for heterozygous genomes. The updated genome assembly (v2.0) yielded 3- and 3.5-fold increase in N50 and N75, respectively; 2.6-fold decrease in the total number of scaffolds; 1.25-fold increase in the number of valid transcriptome alignments; 13.4-fold less misassembly and 1.85-fold increase in the percentage repeat, over the earlier assembly (v1.0). The current assembly also maps better to the genes known to be involved in the terpenoid biosynthesis pathway. Together, the data represent an improved assembly of the A. indica genome." />
<meta name="og-title" property="og:title" content="An Improved Genome Assembly of Azadirachta indica A. Juss." />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/7/1835" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Neem ( Azadirachta indica A. Juss.), an evergreen tree of the Meliaceae family, is known for its medicinal, cosmetic, pesticidal and insecticidal properties. We had previously sequenced and published the draft genome of a neem plant, using mainly short read sequencing data. In this report, we present an improved genome assembly generated using additional short reads from Illumina and long reads from Pacific Biosciences SMRT sequencer. We assembled short reads and error-corrected long reads using Platanus, an assembler designed to perform well for heterozygous genomes. The updated genome assembly (v2.0) yielded 3- and 3.5-fold increase in N50 and N75, respectively; 2.6-fold decrease in the total number of scaffolds; 1.25-fold increase in the number of valid transcriptome alignments; 13.4-fold less misassembly and 1.85-fold increase in the percentage repeat, over the earlier assembly (v1.0). The current assembly also maps better to the genes known to be involved in the terpenoid biosynthesis pathway. Together, the data represent an improved assembly of the A. indica genome." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
    <title>An Improved Genome Assembly of Azadirachta indica A. Juss. | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"M2YJkIv8gYNw88fOqrmrx-mAQOnnB96u9NLIgYRzysg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460415","apath":"\/ggg\/6\/7\/1835.atom","pisa":"ggg;6\/7\/1835","ac":{"\/ggg\/6\/7\/1835.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/7\/1835.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1835"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1835"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/7\/1835":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Nwx\\\\\\\/SULQ1Y1gM7q0pevkt\\\\\\\/0J4+LuH7r1nxK5LcHjAb7hK2aTPKB3tuF0vPFmi2gXN1uLMr0zh9qSAQyOvQfJDvZV3RHhS66RQo3YAAqrdgDQ\\\\\\\/heb9XpUM3YH5LCiulDLKuuHz4q5mRlgIzNsTi+W5\\\\\\\/trjAOKMA2RV6aZFp9vMkE1OAH93M8ukT4kDDgEsu3QR0hN+mxEp4M8kGfO03OONbogNhtDl7I\\\\\\\/Nq74tyYJHSXaIlT98zz8at5\\\\\\\/ugsGiSZdZQcpLu0ORVVlcAQ2y4Cpo8dnezgyjcDSGhJ617Mi927aiWqvqkFeDXDtJDjtaGwdoBNRJvqQiMjZeeyQLu\\\\\\\/F9WLlSN1Ws0lTYaULC7h3jHLHnmNExC8D156HP8HkCxPI\\\\\\\/V0Kue2MsxMwhuPtd+I\\\\\\\/oxp65SCG81NfDJxskdfPSgasDOQBvQ+GcsKfOpcMNL23mCXFHevqLoPP9d2OGRxdRo98C5bNumZGQ7yh0UBE5\\\\\\\/H\\\\\\\/R5R0h6cu+9RARhpBdLsbciIKwt0p\\\\\\\/baL2CV\\\\\\\/Mp0S3qxK4Du+CUGg6\\\\\\\/PYV\\\\\\\/nJuBXAWYwkYP88FHKGKnCiMasjIp+eb2DeOsM6+sSNODwwFnOOEzCNE7A7sgEX7GarV0Vdk9ACAu+MSwoEXubQXe76enJY6K1VVd3hBSOmr\\\\\\\/ohvU8KqZEvVROLQyY3fCth34YmXBpVgTApHF2ynSNepdjPodZZQz9QTc7lvR6H8rZwqWOh9Za0D7vy+o5ybiEoE\\\\\\\/3SxLoWHlMu+X0ChCQBuavIo0CR6uxIZ7ZUSWwbLC2J6zGyH7ldtMPn3pEeuy\\\\\\\/I6KS6KCblEk1PgRJ07mzxWSwp9Bc5bn7qHg7S3TQIgLjku5ScEyaBpZJ3UDTynf8icWujQHVsByRxqtMgLEhzJxqa5AkxRIP9TXBarPiEusovVZHr4oyEs5\\\\\\\/l4A9FqHUgirAo5bidkv3JcW5sLZz2Y32xtFokBDFTrFkeXlPO+aB\\\\\\\/jYjw7BhxLPTF3ykS00o7\\\\\\\/boPuyZJK1Meh7IbQMXV7beh6Bm1bMHFbQc+LRhBx2VtFc9Ri2axnvedi4UHozZ+2kr94gRrfxdjLm3nQU4VeJ9HE7X6GBxTuzrgFJALTZEnzm8\\\\\\\/Gag1yRexSm9A1hsu\\\\\\\/LOUyNRJbL41KZerBs7anSBnsTeHc+fdRG1Bc5Ru\\\\\\\/2KgO+w3pq9wNeTuZDh76QadpGVkZwxf6rLhzrZeIIynJ\\\\\\\/AcPSSEOEE68LM+HZ58lGvyEIdoe7DXFn+4xyq0Xiop4EO8xFrzS6FM0eQurbQchHla\\\u0022,\\\u0022iv\\\u0022:\\\u0022YnVPEB59rZQHy8Tn06MNuQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002275517c7d778ebb4d8adc1136b3051b4007809f33256aedce0c2f537c877b70f9\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022t8+\\\\\\\/9qaU0rk4rqumO2ljO37fPa+Ou6+KKTVHcZo2sXxXSz3eNxhrWUWEdCgWoBMDYygulw0tUtxexOgkzB4UMrCNd+8i\\\\\\\/TZuSm37DKkUdBd9IruWvGQ24ixNBAuAB6jme1ix\\\\\\\/Mws5RzvOYXjPFbDiu91RrlTCI3OSIa89TJp2meubUWfhelXZSgbq3tUJxyoKVl8CTDK3whCN8LcjWBpt\\\\\\\/ngiL\\\\\\\/LeHeVrU9rx5Nr2XobdQkcAv+fbLabZk0Cq36Bu8y8qkeiW67g5pHECe3ho821jjg7dVkXGJSaR++ZqiJ41EF25ixA5\\\\\\\/53rbkVnW1N2ahS3FshUjK7e9PCWoooJBxazDA93Z1o2QKTRpLqSf+9z0Gt4KU6maqT8ghr0VQ9\\\\\\\/V8AAiAEwWbSxt4bzdEixCjTcizbp6+rkVeKdh8QuI\\\\\\\/ohbXh7JOnrwd+SMqPKVjihWx0O353XvybVtb9rHQxO2CEZ3g40pXFkp5fs6njx12ZP6p3C8fm+tk+5hpHwr1h5Es0u78ptUZ0yhNYltpFTIbiZT9Q36mMgH0T427A1yteth91BtKEBQvkPwFBHOxLv9chm5ELuu2YTtmgFu47NU3O+5T4Ad1fkwkxTeajwgxNh7wGyZCKU3XHmGylH8FhVRVgdXx4Ye46NgIRgGIDOLbfe5u4P+Cw9g652S3K9TALeP9SAg48Qd\\\\\\\/r7KO2WKIq9cEanxpuwfos797OzYbv1cy4QmyJmhGBRijOsRYZd34KPwBx0wgY1ymQR8jX9pWd8RQ+18OCWMYPI0lWsCmnv8oj+HNAYd8xKWuV6R03G4FRt8mCc21ycCtu5BhZsCYA4PSH0VvibiHJAB7L\\\\\\\/2mzs4kzjxlXcsZKNXDlQuThvPnRyrxVcnztANNqg5aQ1IDT7yd1t3FT7bCJkLw2J1CWcu1p87XT3+9pqEyW+bp5mbatW\\\\\\\/JncryddqalhYvBAB\\\\\\\/QKiIcqUtcuAABbYxYwfScvSin2gtBMuPHgZ1SgCd\\\\\\\/VFpiRsc\\\\\\\/M9PYDo0ENIAh6qu4S0KjHt4Po47rGvDFkYOZDOI80wyH\\\\\\\/TSVyq4vZpPGeD6cY3f3ZuQZPvWoVg6OdJw76AiE0D74g9ghateuRIrdvEEbGcfkT26RbwdZ3tqOQipUM0cZ7RJm5OASvDMr0\\\\\\\/0064vbqzWrsashIibFKPmqT0579rEGlacCDhOzeJnrOC+1J4VdC486QZePTC3wuIk9zrzCMNDb9iPKfLB9os3GCk9It8vpfpth69KWE431NVXvXODrG2mJxy8WI\\\\\\\/Hc5pcBW4BKB8tuTbLj1agmYdrg4KwytIJCD41bXP6TfpM5pquo85JhjxOt3doTBxGR0H2a9tCZ4JMt\\\\\\\/OXaXtHBlWZT93\\\\\\\/usfgeSBddwzZNi\\\\\\\/DOw1UqHgxdiYyYqiP0wkmRUhppWt8c7YpLHpuOvuxjm1jCX9aXKZa4f7YwBIwmfj81akH4by4mAQ8OCmZ2ADrGPrDxAhJxZeP5WHO01ouy\\\\\\\/MwT9zL3BKa+X05YexXh9DBtqojYsXcgDbGF0iJJIGMOUGp+Huk9ff5i7azgeJVxNqfbyIgE3nors58iemkP\\\\\\\/Z\\\\\\\/fTdCft9SNVUjSSxBJow2fC\\\\\\\/Dm1cj3V+sQGbzBGoIzkj\\\\\\\/IAmBOuO3ba7gdLSAb5ZQcHRGIbUQ+Qi8OAKjrcQK6RdAoZXL9VQKPUPOeoSfS83xCkWcYk5zK5+hrSzV7ugA3sY714RDkBPzg2GCPoBvtKV94MUtxf8mcS6r9Vpg3C0GvC\\\\\\\/ViSVwBveQ3eFsUVk3Ivd2KuFmiHQyrySlFnXgnKt0YIpqd\\\\\\\/cczDbsfnMEADEloSq093A8GQ1BBGhg+tZhgf5Pq\\\\\\\/5OB4c0hpI5XNeQZ62yFnXh+E56vzhr0OcsZ1osP8348dBWP8lDXLXSvqeO3WvFaUcrAxqktFPxla2uTYI7\\\\\\\/+2o0a04MCy+4uJ4gCx6flE4ChaMdy82Alzr4FftY\\\\\\\/bS1kgq3juFTj1Uhl4ktiMMGujmKRSfUbLvkSdjn2l6C+by9VGbnXmL9atxbfTsEOliXKrJK64zUtVdQAcvcQb2GPUr8ZSgAGhfCMZrQCyhDN6dx01vUylRktfnUUWMP8EDIkTCGCZkMytxuN08QqZUxaDAnneU\\\\\\\/S5QEY48P9Gw3T17yxKcLlXX0iTqrgwnRs6bs8ftHjqddApz8RMMOdhPtZuuJsA84t6pNf\\\\\\\/7MK5FEn1JC15Cx1IRFsJ8gPEDi6gvNIZJpf+PpypwaaBCmvvG7M30qAXl8E81YxbAI7CbR5CXatzKF1fD2XHXhlyjOKeCtOAd\\\\\\\/NmS1B01bk+5+ABSb6cnwAk\\\\\\\/rG2eUe1Vbrthbvi91MRkI2YZVfcHJzYMM+mw4CfeO85\\\\\\\/UOsTZRgnZ+3UsCAiblxk4vbB\\\\\\\/BzjYHse\\\\\\\/iWfzAPW58raai12ngwrF4YOcKlT4\\\\\\\/FLxfZ2fqSllO9cnmRhH2IMPT0W8tFEPqycwdR35WvXebS2NwsqKqv2\\\\\\\/6fzsA4GIRMTs6HZoSeFfW\\\\\\\/bnz+SQwW4q3jlZyBw1P2GSusILbTpOWWJjbDlBuFq+H7thKgnnpaNcK8nyTADSmUOyiY9bkg9seir+IDZ0zYo\\\\\\\/qiL5yJcpepaYCswWplSp34xOenA9Ohi63FSgXZawQ0gIMN9VRXyxru5SPc\\\\\\\/JaUIHjN3OQAw3tvmVuROovvrtbNpDcDjL48cI6zHXInsiCJtbNVHxPLcxwOVL3KIgxjDbw+Nm+kSvKmW+RNbebII3snjAkdsr1tYZ\\\\\\\/FWvt+Fax1gjTpAYnZxADKXbtRPFIjXpObq\\\\\\\/rhHkoF+89fw9JMk9LST0g85EOgcZkyahdkAv6GESl4HaaTvEBDXIwDLEq43aAgnRU7qT6I051QF4U7QUGnySsjNGRRY8aKbS79hvttvDci7DB4IdOV2P17EAfwb6wTKzT5JGj3ztTBknVIywjMeNXN9X7nsIPdOwBCWaz14pk2yaHJDn05iQN3vk9qRdaF\\\\\\\/RTrvonnxZfZAGxI4ZxOy6B60WdgH7Q+99aXT04p5PylDs6nKRj2kGtIvTcquTT47TBYkEmh1y2+1xcZLAsUbhrKd8Rs3HpL2Cmh5WIMBRqNAOA8x8HX3nBLk1Rz5R56Ym7n4VVrSrbtcEMfXNNiIwWW1KtBjWiGrvOJpTgkSool2DnBJw6uoeyU\\\\\\\/QQKZFmy6tashfOq2BSMqlpOku1PrLs7jUvr76\\\\\\\/bAwOThShCGKl6g\\\\\\\/pZKIus7buWZ5MsH7Cb8sB6msokYXzpSfOS7Zv3Ffsu\\\\\\\/GlCMC\\\\\\\/8gc2i3Eqhg78tLZ2UioWKAcialbEl4H5svNtN2hPO6KaFzO+VRlp990lVJZloo2YN2R0bogrk8NrXWynvzk8\\\\\\\/aotzsItqVaVKNuxC9uwbbFFQGDBv4eSTlpJb9fx990jNOIljWxPSCJGvbTKsRzbSk3dqySmg4o9NWmXvVEjh1Po7iFN7C6PzT6eSWdh8VNDxP46GWCH+3tc6cothyrDwmbPbD161zgdwDGQmwKPrmjsIRpXOD6fOGYz0f0TqFuE61iMp4w2l\\\\\\\/WV+lny5lIBCN6ZQQK3tJVS8en8+NpvqWeh4UISuZ6umKpDLra\\\\\\\/4viQIYqYzBNNnNEn5e6oZpTo\\\\\\\/IwSlQhKxWz6z9TUOx66HrsUU0pieYu1sKWna5C8\\\\\\\/OAx94UBFTYW7iNAw7iiMuUaRgXH\\\\\\\/6\\\\\\\/AnA9JlcU\\\\\\\/GTq1kJWbUE\\\\\\\/Jwr6N\\\\\\\/zbXAsKPbRzTWWvRMKpCv0QrMof\\\\\\\/+0\\\\\\\/3DLkgEoIyq37wtOoFrmlrzIvAgIQ6UV7V4hpvnQJfd7dSwX4wg98ybjYHNkwEm32fUk3wIdmL9tzLtN1d\\\\\\\/0Gu9\\\\\\\/n41MOCbJYLJNfwN5LFSDjIsJc618OxWSI8F4LC4ckKtCRc80HSAvdv3N63vMhuw4QLmVgPXv6gfwKxoyIXs0oQMfj4TyQ5ETUJoxpIwT+qtsjarCO1PBq1Irl27jiwNf40lNpZ39NqLA\\\\\\\/yMsZ3Cc9K64G2qiWIf5s3MIpMLx\\\\\\\/27OWL21LkmuNILL+W2hPA6PT1jrBuNd7f+U8LK9cprzOB\\\\\\\/8bHdAaxgNEz40oPPmaXuNMWgrjyEwcR1gxZ9k6fYqdKqs7Mf6t39OXaprPW5NqufA\\\\\\\/n00MOLHQ9jDvPN+lLtQcYAS4XKWCkZijLiHrKkgQLUAZNLWeOEPo6NqRt5aQrUXLISImnbonke8EGWtAT7J6O2Ee8TlFubX0aAsMgV7PuiIVhfNChFsdV8JPoFGJrrKCKtmiwBkl0VpfCc8tbNSvdfNst2mCcTzBbWCRujba1aW2caJKm\\\\\\\/6YDxmaOdWqva5iu+LE+IgP572xKJEC+CtXRZRJ7HSCI\\\\\\\/pIrqUWDvSWGvJBspw6a0VPWhoVgWnQDihLEGMGt+u2\\\\\\\/Q2+4jOVKC78n3iLGlpNw3s8PlsVKVYZdyvDxgdLIj9Q\\\\\\\/Ar06zKcFn6ahGbyC3YSzBzz5v0iwxHrLGyI7\\\\\\\/qMnmDI4e0B1CTqLQPrUm4NF+DSxiSVJ8iqDK\\\\\\\/KrUSX1lSspFhnTO\\\\\\\/oWTjw5HN3YtZNGxWwBQEWX4jbxTGX30QrH5MINaqNsm28LvU2otxP3ElLoVQA6U3MJ8qfCFtip9i+Ev1hWRuX66bZE3SyS96mr334xcSofzID95yJDpBGMz7VQlvmLqsgsBxPNOvGKx5cKtXAvbkji7a4H5WVRlJ1caLy5nefoBk1QUXHTOf8iOWIYl1kXLNwfBdRNxZQ8I1sAIcURH\\\\\\\/dBNHoVFPAPaG+\\\\\\\/JTOy+o0Nd4fWzU79EwpfVwY4JhGO\\\\\\\/snKxssRuLQYqZy+fbT2sIFqFFNUby2NQ+Hk1XK2\\\\\\\/grPeBPdKk296\\\\\\\/xNq5du2cSZJQtJWWhCP7ZhxRGagbT3aMT45w6jPY8ENHN9JmIZG6bVjQN6UVz3Pku0Fvy+64ngZySXgvGU94n65q0L9WbtB0o77POPCo7zc4Yag0TrYo8AEWsVpf7Jiu6E93iOb6TSssE3RHQvkqwjrxR6w08x7SS\\\\\\\/11n9n9P7ouFeVscXAKacMgza9WIJz1zteZzdJljPe35HHz+d8FyLyOP1mcJb6tU\\\\\\\/1YJAnT+LA75mEzvqO1m7WqmviUbWREk5j\\\\\\\/OsdAKJIhLaazZAQfFvxS+BsJ3MCcz+z\\\\\\\/y3hiIuABsEROuI170plkUds9OjpDWLKLpS9TTm7+aZ2DggTCHBJrr+2znpbMSDy5w8zB\\\\\\\/5L5MDu9Ecq2HpJkFL8VM3A9yLvC+uNMEcMvL+YQYGQBhZ190fk\\\\\\\/EBsCkzGQziKCNExQC9ggvu4gjq6lV6u\\\\\\\/nBXO8vlob1eQIQfQhxNqV1WX1C0RNEIDQKNwSWqODSILHLULeewWgY8X5GYOtHqXPnDCjKrP6Df4zD3n3I+mOtgy7kSEHFjjlYGAWqK6szXtvNBv9gmhg\\\\\\\/9Kg2Ng3T4+ysJrQDc9wb62bu0movSElKgHEQsuGUiMu24B1x3a98Cg3pvEYjSCQZpd4xF77l9JBdYsM3EdTyUPt2wMieJbFV2Br\\\\\\\/WVwNqyU0wguC9Q8RmEM5LC4Fnu8PC9uG0pgX3Odo7r2KFNPXvy\\\\\\\/99shzt7\\\\\\\/bjxbN6KtkW3XvgAvg8qgMDFTUd8qLvEr0fu2jZffS8C0vs62tYwABocGQ2E+c8o9EZU6RpwzhEq3qZ7ZO6fmAuWkGKFm8uu2xfk7pZLWxfA0+929im9v7pg\\\\\\\/GxJ7x5\\\\\\\/Whbw1Ac3nCHm5Hk7hSD5UyB1UtQQeOLJ3pAAawQj1fgQMK0gw5lIC2jIQYDHkoXMHm+FHd3Y7m3vlf0ZhjpRHPPoTT11\\\\\\\/fxWk4mV4+\\\\\\\/I3sA0dAlEppQJa63ErE9HtNOZMEmtOzQjI1MKb8NAZA+Cci7fs74Vydt8PZkH8mumsW0jhmzBz8lHAUpfOs1sceLoPuex2j9UojtEgKbBtN+MFPnbxwWZW63GFEoLMLl90ZPkNPvqeXmWRCCAKyUsNK3Tm2lrMtHUL3PS88SoSdRdQhXvMVF1anv2vHSb+vpj0L\\\\\\\/EivOy5K3X04E6TZ2Sd41n\\\\\\\/q\\\\\\\/cgSQ0gjF7JnOtnLx0\\\\\\\/cTxUK6BsbU8ty+fdC6BiUKNrcWbtwJsE7fHuBByktkk\\\\\\\/m\\\\\\\/Im1hRpu0d8rn4OSUfl\\\\\\\/SjmLIr5ZFPtqrSMZ0fxWUi6hi+uf0aQA+176C8\\\\\\\/Av1AnQUptY8amGxwrZhjD77VFoILRl65ks0\\\\\\\/QBGGgkz6B5FC6E3sq3jjIgTuFs\\\\\\\/oCKedsN9zL75nCPUCxBnGg22vaMVsD+YNFDDZXblIgcSzsDtPVtqqwj+B3BOV+S5rp0Rm1JpvGq0vVLZH9N42nevmMOikxVjF3+l47ZCf66A\\\\\\\/uS4eWuwFvHPkH\\\\\\\/9mxJ\\\\\\\/m\\\\\\\/BWEZOQ1W4at6Mk57JfHfnjXGiPlR45VfAanxqY1+vw64VJ3sILMBJKj+ejx3zn3+ZLJP+Tc48FRN45Ijb4XrobkASSfhfknYEsHQ8H+YMnnGAkjlRjfPbNYsXnUFxNFt+Qj5J1OiR1a4fLfcusf4VCUeBgCx0eUENvRINK\\\\\\\/Ngm34GF9MCIibyP7SCJksak3MnAjzABiEpQ1b4qgNAtW4pMeVq1G2z7I6mMOFiWNfyoslQJgJCEzutiR1uyHWzvuDchhFB4Y08LhtImS0YNiDuWHF0TV2pZEdxnT3jOW0uHTVlLz8GM8HgxLRS\\\\\\\/SJExcQFii+cg9klEbIUIMusfKJMVIyiwexAsyW\\\\\\\/fG4OGxsT6TxIfMWwbU+RiXmM6hJcF1K3cGfSWlCrlzzAs8yVwzLyv9Tuv\\\\\\\/kmh71dKWAH2R0crA7sG+RUdbvreWT4VdgIEgpd7YUZgpn09BXpgDEINzLQDn1V7aOlUllNthDyzUAT0RVLczts9ct40EcEOOjE+UcAbhSQNcEdBsw82yO94\\\\\\\/yOBXFnoH+VYpQ3m39FQx6cFO2DjzlviFzI2k5CsRYekFgXEDU7gJaalPQbErAXl8vsoug3tZlAhHdc0\\\\\\\/GrdJXpQexrPLAFpKN9k9lZUZ0kLvHOPqsMC5WuTD3oC3xCnIqR\\\\\\\/OVt+FZxNRp8EhmEtZQ\\\\\\\/ScAyYqCPyvu1NI0Xuv8aN+5hVLSUvzae9Q68v4zX7Z9Mh5FeyedxSvtDrvs8e1T3L5Jro7p\\\\\\\/0DusNca+zSvsrbmlHE4KMrDVvhED9GTYsY4KkWuZNGFvYKFG810bQWHVunE1IO77PLcDtQu\\\\\\\/cyjGzSBrhevieVdtujZ\\\\\\\/00DCmSxLWA4v+d8aqPDqITE1K\\\\\\\/JELOt+GFig7hcfjbOZUxJWrdPu9iWdh8jUmUxE7N\\\\\\\/HUKvJ0KLujFFhw1S+nJSf8v\\\\\\\/ougZCE8qXnxuf\\\\\\\/SkeHkRi\\\\\\\/Yysmq9b4GPFiesxGDo7ggG5ljWJBFJaoeKqhuxP5CLC+VG3CJgP8JHCNz\\\\\\\/Jm+w2X2dg\\\\\\\/WYeO1EP0bD7fDLDrVVb37Hvy4sioGs3dYXkPJ6KwLGTAm\\\\\\\/U5u9DYgmyHr1vP\\\\\\\/EaR84tLHk1iYpZKF4CpuRx6K4Bqh8+0CL2SApHbFowd8MTmE1piPLhQdPgClsIM06reWXoS5PUTCbKLzZiecPnFS7USE9qOEFURmHjpRYmvmrUl\\\\\\\/cE0czzKie5PC2Gii9fsnm5aqi0hXjyDvKgqdyWADrlcndjGndRvzz7WqiDGCScJDEZ5xrr0OsKUK6+pVWp\\\\\\\/Q2BLNfWW1qCAK8iBlYPjXbiSx5Li1qjHNEgiQU8GhDn3F0Y4wrNRyPJndO4friX28YaDGsdx+62Htlz46YS0ZkI8xmxAh7NQj0avEJQOixXmDRlyucpT3VXtYUTLi1oA9GLzfX\\\\\\\/Rs1Q6iFOmK+F67LyoujsamRooc483J42i1Lf8qPNBELAlt2A1LEb8G1wWXFOKSPP7+hQyfLNcBTVTRNmdtDxPFBswkrH0DhXVkEBBjCszlbszAmvJii05bJ+vxZucdggoa3e3qn13cuh\\\\\\\/16L8SnW96\\\\\\\/6KLeKlfnIZNekdaSfsgiqZbkhLoSameBEZORbUZnuir7L7zMbGD22YSoK7oNKaTTiiOLZXUyNU87EwHmzVpCLQa1DZ6b7spEtMkrSTMHckIaA9q4JmiHa76+\\\\\\\/+KtsQtk+1IQQnoMXx\\\\\\\/lnnWvVHLxy93Za59vAl+v1EMdHzL9baBrIMqALNTp+HLQuyJMeEJwBwLYueN8qtdazboa7nEvNRty9k67aa4H2tpUXnN\\\\\\\/wJbmLzchJhDStKt3RYHZkGZtn5uyPTDPbLWND3rPpLQAEoMfvJPlvKh0ndltdUuw+KQJMNKLsDarm8MGLbIlIFHJgMCP1MAoIiaWZZ0dDned\\\\\\\/CvPPIiqazIp2P+PJamfjo+y2mDbcWptWFm0nYLTV93n\\\\\\\/ttgGn3CvSBU59Z8YymzPyeustDvLUttg7yIm22133wL8j9dUVdqRkg\\\\\\\/pN2bJJmRPAZv1gL+xAL\\\\\\\/Y6089ZCSGJXnM4DslnouB2dKMGOZUkR5Dl\\\\\\\/y8LLWtvWLcoK2Th4ta8ZcraVMBGw6z8KFbWiBhWZPJv5frP4o98Kenz487jvAtLrRetcj33klPZdzf7wgPLhdvmpHwhrym8jQj9zf52YfHFOMPy26NZpxS\\\\\\\/ngy197Yi+aSy8zBlh+5xWD11qqV2PR59Fw3nHQ1NtnadN1ZuVgqmO4BJscTh0XjVRZo2X8xh4yHTOMTmcJ9bjqNVP7esaOjOjMm+ry\\\\\\\/gkLJE5lKc7SAJSRNp36k+\\\\\\\/mCyHfGlQYjyJkyvQf08fwOyB1vlwhIjEeJ6R7BPa8E6XJ9LWWdP0AVGf4mRM18WgtsxpD4MgbcKk+cjACM4pGtwMV4sTh8C8qIkVCuivrXvp0gw86\\\\\\\/F29ED0XyOlIomoBiTWYjJFjhF5rzjTYJAu\\\\\\\/hGmNKnrqN4RNLytpRwOKenc7O5Le47A6ZtLawP4UfdGIIk0Iu9g2joMGszOuh3RrwzWiqVvRk2xA9FBN8y2AKZMuFwYUfEr1L9AdkqPvwTMBp\\\\\\\/fRMKnisfeTlUuDxkNGBP55HxWKolqB4yXBZOUfv9LgngkuoDFqwHKy+N+4nu0R4Rvv3mdJufkshzBJ23BcArCfS+3xDn9muLYVudtnw4v7ZShcs7Ix9w7NelYlDgjYLI1A3XcJ7eBePsxKDABc+rKDSwYqFtiFw2vRMv3qBUi1SvputlCHViYoIuVPCyzgq84JUel7ER5cckrfh7MzDk9uBeu7Onmfj4ar8L6qwRxkkRf+78Lxc83xwEZHDmvYlwIO1QQ6t\\\\\\\/8DSUDGZ8FglailCm\\\\\\\/QFTZX\\\\\\\/jK4ZLAkpCOSYeDzcd7poM8Kl2o0KHBTYlG9bMJJRLO4zvA6dG3YenA4dPO+Ivy3yJKP2\\\\\\\/CtOPwMVbweb82SDQk+9Wavk07La0Z8AWAMLpAerMSCz7Tw1crFkcSnUES0P+Kjuj+Cl1jJeA9TIgH6fsnollT9bpp4GH13QS+h+NmrKFa18VOrRU+faZkR3Tqj1hWyVoEZ\\\\\\\/ddl3wNCBvjrp0jVaJeSQgJiONLpPLsOHY096K0HVD5lyA9SnCxoOCiSHkYUJ8nKFnVW5gDVl8rO3f1Ox3mv6j9PWDmIGIl30WR5sHvpyXpH+XZ3Z6WJgNmAAShOK1QYyDzByk+9FaFUVDShCjyfeCYtZfHtQNypQUApymkiC0tbX3Jl3kw65k6kVc\\\\\\\/2M1Cx9vaKBtl2z8DPAWPTAtevM0i+EU4wvbcjCivkfMNrfleuX76cWUhq8t92rNx3HYl5ZUgaiRPuF8flRMjiDOi8DulUjb6Sxx139ZbtQUXQZrDVa13w1RCkqhOSEM6tgwrty5q2W4UmT2ZWO8Eon+zER+nfaWJkQzRKcYno\\\\\\\/9D6TvBlxH\\\\\\\/eKXiRTKtRcCzjJ\\\\\\\/aoA9b4f\\\\\\\/klHVbFdmRahxmrjxudk2YF9JgAKxoy1xp9W70\\\\\\\/HB6qo2FWjT8\\\\\\\/Ikm\\\\\\\/0ucTSDw1pVIgMFkeVnuM08ql\\\\\\\/BITG0IdRX+5BVRu6S7ahAFSI45nQNGYFk4k5JkS7hK08YHJYQvnAWIzQrnuz41jzhDhLbMjuoq3QUmAGXZAniiM9bi4Yyc5HQFE1toy9SAfc6cmwFS+5a50L7ivbaCs773Z\\\\\\\/jrS2jbjumnOc3pfsvHCRDFQe3kPCsjSIrVQZ1xMof8yIkJeWBZ6uMmOZ5hdDYebHNaXSOS7YozNmi83n8SsvSLAHyc12LLTx1VVbPq9Hyxp7VE8KyxtY5bXkfhamOQoxZiJyPAj8xm5zXWg208n7QyMkIWAnXMzgW5Qo6V\\\\\\\/Y4OACMC9ABCnqZlIdlW5HXODF+gYQ2KgJ+K27bvW4stanNMs0crpV3XtPkwxBbHRkMnwEwbo1jYUgeYBwtQc6pY6YjBsckJSzui1piXexf8R2v+hc+ntzvsSxI7QANsHPjkHxPnIERONZwQtxG\\\\\\\/PnTU9ve2mJMEYPfhuBV2Q8iTANfdxkFApd7yrQvnrDXpd8TJXq3KWAZbCvPmHDMUml2yhetXg1DXgthV1rLP5MR6zEp1MKFvqnhJapnf75Tp9P8F5vcth3ZJcC95XgkyJ7SsZx4g7KL86Y7L+p6mrZYEpuB\\\\\\\/jNg+Yvn7drUkwYyt2\\\\\\\/3BmkBY8uj+D7hrwdX7rGIZdhNPUTAjGUAr6ZxqV0g+G8K2peEVo27qRnUvYWsfkrN3S7ZUWPA5UGX9fhluBfBMcBV6K7ClLO9xcPrhLO1r80HJrOnUI1EtvcQflvz0KOpHdqlNdWRPLidqglRyRwm25lM5fNQERvpqjpBF6eeq\\\\\\\/tp+X+MtkVVo8ozWkgqrlqQmz4cbu9ZEypK9+Bt\\\\\\\/27qZHdO86a2CSa+TOtpKyPN3HWta5dLWUWGO+M0KciIjnIhlKncpTAH9yYnAOLVTrPILHVJ8T+VIK7idWXEtv\\\\\\\/BM2FzNSA2IiF51ApqwGGnwrvCtBDlVOhWUVDklrE033\\\\\\\/FrBaDnDZpasKuQuIrFPpCfk8E0YwAJT\\\\\\\/PB97vUJbtLftGBGiNzNJXPp7ZqDbC+R8J2u6rh6qIE6L7Cw3A5tkpr\\\\\\\/l4HLNMw1BzKI1mFmsCv888Ju+dwkH+EBSL5S9WERanyRvZi+sSmYL5w6VHeTBclyUNOTlndXg6vWy6w99jkeaIaaeI4jstm9IRKMyIOj1IW4IQ9qFTHv1HO19x0Rjq8T1YGgviXLSYRijoQwgEcN8xs732svm+lOKbd45gkHTxpeqGduieQnHP23cVoRPPYoOGJP7mLCtLOIuVsv44wHm84aaD6r+Z6FRRxDZvWdpfwHdjksWEZo2WbWkgbgoe6Peu3R2k1lLWp5tgCRMNadWccdm3vwp8+O7NJa0+gLUfqt6fiZ4cZylxDJ+DL0BRY0iBb0LBO3xBPaXklY0t2J2UbC3dPbgTD\\\\\\\/afUlmBJP9EASoYMIjjuzCViltmH+HRAUvI+UpCoNzP2JXYcPF2s2\\\\\\\/MQbnyAUvU3Nn+xF5IHD2kZ2POSpzTf2bdJdhWZSHj2oF91slVSVyg9gDg4xToX17pKcH3JmGCR+zuNSk4uLUpumk1ol59\\\\\\\/Od2coqJ3r7f74e2yI+5Py\\\\\\\/UCpuSteYokkZDoOVDSH2u2lrENTGyhZXpDp+gbR5WLFnefFx3JtMduDSFtSKbJc86ZqREPdvdQXaZF7eK6jAZzQWWY3fk20p\\\\\\\/akKXc7FpptHKJb+bzoqejaRh\\\\\\\/8W4gndyKGUU3WBOJAKvI8s6X7BYVtsKy5A14P4OWNmYPOiNApILD3VDRMHxfIWK+17UAscuKvMFaWee\\\\\\\/9O8duhCsO2jydIsAsn1TJsRBVh\\\\\\\/+ZLo1xOlkhJ\\\\\\\/AvuRqi9ZLvfNGXu8aB7P5sabrl1bpflaVjnUt7D3AMWH4jF+WNDQW+Y9+tvaO\\\\\\\/gVppoqU3oyI6pG3atednfhd0yustqX28jbPs9G1pHmdocsTlpm0ZW4XobNc8ffVaEWzfyZ1JVTVy30w5QJ\\\\\\\/4j9ZGqekmcbCqcFwLGYaIWRp4BlBM4Rjgmfk9unqekTSuUYA45gDGegcttJNPHdiRFFKemv7FdalV77JbqtT7KS4YS88pwFsB1R797\\\\\\\/lcYpAe1yoVJJiiOi3TCAfCMbxrugqhgvxUg2ULG0sTc+1O7Taqi41Vw9RNd9gjabxy1guxueFEf71HC4IopmZD8To8WWyms352fedUkQMm06WZRm1y4yZT0PU+1KNVyZ8z4p9adwuOOxOo79BiJmzUK2NfExNkIbzDjEArA5zYwdlG7jGMj+HkXYztELM0NR2tUh4hK7ZRpuSoBS4t+z\\\\\\\/gjXavMUBQC3RKLWx\\\\\\\/nnnx7Ka4jrd4gRjv5VZlG8c83QIoRbE+fnKF4jUDh\\\\\\\/uWW4OmreW2lGvZ79VRVpCEA8nWwloiRrqftbiPaCWWSiHfsHJalaxaAerhO7j4fxl2RFyqeURdrmqBpFkbjnDW2V6GFbFEZujOsb4G+sfHAU+6WQnYYYV96lLaeACj+2\\\\\\\/Tq39ztE5r73E3bU\\\\\\\/QfXLOzzy4VPJv0abTjE\\\\\\\/gu8mP7Dpq5cLpR+AG8T4At11TrYwCTbb6hQa1mrrcsSrxCJ9k2z1zGw7V6LT3hc0rZorZaTQ+RslyyapKc9MNUYivGMORnTnTqhy2enhHUlcMIE7NtvohLLMcYP0YFFuSupH+JoX59xbYMev\\\\\\\/nftCQSHw8Klg1YADXBEyv5mhvMjKDKaEctsDrFh5zfQUDbKV7aQnDlSQs+ucfQU47npz0kVO549gvNAfCWolGJIEGezaHBEAzcRDxFHbbVdH3G\\\\\\\/89mA46IUzcUriLMX1k2lDF8+ybzYQXJxzHcpSYxGVIJEISBr\\\\\\\/GdyTwtxj7TjJIKHEeVt21nVloSE8xNhhAPPwvxlVzQi4GsVeXaYM0lFVWhtFP\\\\\\\/6plfE2RQ222U8VLF1ze0RR1LxxSa6s12TtEpGP+ZRhIW0I6VvYcwGFpGF4h8B9+itmKdOLMEzSje6+UVnHyA3XuryAKx7\\\\\\\/30kLpijoay0O38AlBSYVqflrNJYNErAKfRyYn2pYIb5eQ2kQ\\\\\\\/A\\\\\\\/q\\\\\\\/XQk9mhg63Bjsf8ftU7fULg84mfnkiuxGWYjo4D9G9y4EA3ypfKyxp+yK8S+B88c3wX1qHHwapK7BMN6wBC2XLR8e\\\\\\\/+kUe4ZwjRy\\\\\\\/2FJKjoX\\\\\\\/bpWp7051PjxnX\\\\\\\/jbzf2H1wLFe0nqJEI9LoQ27Jza0rIvLTx3kRjq\\\\\\\/IDhUrnrsIklibSY12wpmH2ukQ7TG2+EkcqZBuEhPjN8S\\\\\\\/d8ADk12iIOWkqIlwUnBTen0USRApZi7I45kQme3NABemeW1UidX0ZxzBzibf2HNH+r0d1Y7p623PsnK8Wj+Si5yzMdbBZLDdDQjlBkZq3N0krGfdnSgl6RWfLpPEBgCPuJK43RhQME5vnQDBtJDqjPIFJakBYsK\\\\\\\/NXXt6\\\\\\\/k9\\\\\\\/HzujvA5ufmadQoHVzmsBRqUAWmVbTLjBkCfFBW\\\\\\\/y2NS65+UzXGg+TIpM68+kJ7H2kq6abp93lPT3IzHh1e3FsJU9eADWnKiTx\\\\\\\/DgoLRe4pTnQBDO9UBQcy6hsbJqd\\\\\\\/ohHaYJqcoLJJTE36iE3BwO0ah7mRXHtSHJhe6V9bjA61ugVPea+nBU7gPbDc\\\\\\\/b9dOaBKjqloCbb7dPj5lWzJXFugw8kHTI9CEyQCpha35IyaBB+rPosOeBefUJ+hSQE+SSBGHp3e2Q3EH\\\\\\\/wgfYcNWVxXL\\\\\\\/c765maeckvAZH3RG53RppUJDRrBi87s+m8y6M7xaNnQBXfTXCTiAsjbGJsGDnxcbEl22a81QL3pHYTP9ViHAdP4YN6I\\\\\\\/wnJzGuLcTtymCLGtuzMSN6juyj09O8m4w8CKLAUu3p+NVyUYGOGxtajzHiD6rzvPFjvvx2S\\\\\\\/F2OWt80zBGA4PWjtDQAcAvQVoZMuzPjmg\\\\\\\/cbsTFwhroht4M\\\\\\\/mzaA2i4CJNV3OMcnmpWMWVB9G5tiZfFa02BYyiH+8dPXukjlCbvnP\\\\\\\/JIAyqTLrmGUC1HoeMhTCVUh61UJSNo\\\\\\\/QPflXedgISHYoi8BqpmkvpcusSKyBJJfFykLYA36\\\\\\\/ML02Zdsia+XPSkzYk899ljroaPQWPR1o211CcRHRm8p4cZV6hIqJnS2TuG1Fl8it+ukw2TZPprewDGMIRaYK1IZnQROdgOlPPP2rXM7okw2MsWA2paQs7Hics6gvN+ikf4UN2RSTI34MUIk8ojgsw+Ihf17hFFJdtu+XCxEGtdwiLC\\\\\\\/N7ZW1qJ9L+i83dMKM3eGT1RAQ3os2hx6r3pu\\\\\\\/51W30RmFCFgLirS1I97z2VunTFnRYYWsG1Z3JW2krfkQBKvoHDylbHu61dCCx6+dtfuB7CuNU6kH1GE3eUbQG1gGfIVGSIAf8yQu+D1hf3GCh43NTbISzdK98G5wZFSV873l+kn6jZ04g6O9Zo77XCc67vgPoPPko5V9Hnrf\\\\\\\/6kan5udLEcfVK1EJpN4jIjrHifujFifX+tj9x+z83QXgIZJO7mkJ3ecy+RlRiFB7zTvM4+JAqOynWZHr04BuPd\\\\\\\/FC\\\\\\\/nBugkn5VjGgnAXqbtP+Lk9FEHLy51pZqzwALcNUzkcdBc7zLjWF1NehfzFlmcMW0kXor6Ro+XWGOwom0etF1F86VUjuQpt8hx3YsibFfTYROxPkr1wmHJUtv3mDXRUX1SgJvnQp51VTHpDGbAYTj9cDFqCZo+hMe8gboJcHweUT2vJt7DTMsVEiX3UqAmb80Yfr3K4\\\\\\\/jveZ2uECjUYoomwtDTIvO9Xtr6sjynz6Twtysj+ytL138I9MvxBl05GRgzEd\\\\\\\/kTdBSH1lk\\\\\\\/raTBASB5cx7CGm7LFSTDgIdfSbjzzYrjzaGpRiLygVQAhb\\\\\\\/8+GeP\\\\\\\/nDDYEMLKYP+qNKDzEjkPGz8CIoYisPn0WHQ+UAea9p32jBJ60u0JlwGFvjuOhJwSn06gbV4B49xsJYyhD7tZ8\\\\\\\/NWb9LwoTpn4S9vSEYZ\\\\\\\/qkRiE6RbBVZyel6ocOHlyCgDzSpDgNiMti7m3EH1HpX6WSppCKkw1JLR5HJJkkU43bULEq6XOfWF3Y\\\\\\\/A16ZEEmrogqDvkyPxO17vEUhA5KgrtXRRvwgo2mtw\\\\\\\/e+iAX2NKE7TAXUbCx5rPINS3z55fvJFLLLikEhZCfpbnSAQ8x\\\\\\\/B\\\\\\\/9+YJoziq4ywa0wABaTqo6TAq7yKjC0Mg1wEG1SlRtJDQumsSjYFVfMABHacDdnWt7Sz\\\\\\\/hdty7Aevr365wLIpf9sm3rw90OQaipmjfFATWB+8wTAu5IcI2dIGGDsbz\\\\\\\/E9jAumOo3HTOg9kq13rGkVGINRa8ckixmeCRES7OAv\\\\\\\/8c2zG5TMR6mgumjO068Xl7\\\\\\\/AnxvPfdoVIrCdJDYfsBLRlXaZVgGsXvRCemJU9UcywbU1\\\\\\\/smM4Eaty8DIKeJqzEjv0LbCT6uNhk5sr0q3g4pYce4C021Nm+eeYRx6RrDK93zV+nC4YPAMDdxbpwM0kMNCEFN46eLZd3Ga+m5hNxoeOhlOSmjAZOdfdSjRh1wR3eSuxYfY20EqlQffkNJDlQxxa8DiNy0\\\\\\\/bVdy9lHErBdFv9xIS2+IDEd7bGUdEUJQmwNQZ8InGwdZFtjsbb9hM7DduX0Mxk22i5HlIjKYTxZWQsYq92yb2IO\\\\\\\/KFHuzPdSQtVIZxNLQhMPJYJUVfTyCQ248a\\\\\\\/yE1DoCC\\\\\\\/yiPh0otWi+KfTyCxWZ48n6MITuMMa7XtLez+DBcp6emalQCto4y85CE5mry3XzgdPPTHNhabDahcUg0w\\\\\\\/yzuCOvhczzLNkyILDZ2vOzwTr7YiMbm6KcDDPlsWp7oWgFlK8tzojghjFR+ZJ27\\\\\\\/bNiz0TY+p9IuPIQHtgtP+wDFnPnRUPG\\\\\\\/0UWI0KNctqM5sfAG9c1v\\\\\\\/Ye7EqN9WoAXASPmk+JFZXlzV+Bu3WWvOt5n8VfDQhBmkc5KuQbAm6r4dzPrL5V\\\\\\\/\\\\\\\/j\\\\\\\/paJmYvIdwgCnC6uxWh7g8w1zkOkl3YB81Zkblr5sa6hpaK2ezUYZ++gzFpYjGj691iPAZ0bfbqd1hhok1ikf1a3K+S1i38RFJM6x3HYqXtscdlevKJlAs3TJ+hjdCRVBqgMOmkqCE6uHJCFjlBhOe3ErHmM7B+FUn6xC8Xol2G7yMNDr8K\\\\\\\/upY76q3MZNALPbljW9MIMAgUH7ggu9uQvyzrEeVLMx3RIC0+leYDPDgVJtBjPUBj2SJ8zmHiHLRviPRRtLu91t8HVFIjhOe46ICmkBg90bLGRtLwzSgCs3fq++WPVOszG7Q3hyh3+DaAs4YqkBo+O4LC47jbI8Zcv8jNJXdr5Cr1cOjznrfdddN6cNJg7jItNsAVif+ExS7oq1X\\\\\\\/jVNu\\\\\\\/nJ7DtG\\\\\\\/3SlFckXUIUWqkRL0eJenr0Q9XDceFzONqjp9Gc4cKQ\\\\\\\/uxXQOqMHJ\\\\\\\/OKWQNavpzDfTOQwCkM8zb0z8LQJXW2MzeFmISCVKOO1\\\\\\\/DgNE1lZimgaTbJTdA8Eqh1hnCQQti9wAZiAkFULRArc1UNnv7ZqykdnpHRamFbeGSqqr3CDCBYbKU2GXCkh8jIiJU2mjQb0rLnQpo+XGgPf7QpYHN\\\\\\\/EG0UbKDho9HoH3B6e6uMZUvUqLnQ+zH1Viw5z7W\\\\\\\/9iFbfgzaSbKDl2HJjkjmtFX8GujDrN93JWfkyvNkztKu6GhONWVEqonzPZmIPuZFFtetB8BPCzja+LY1HPYukTDyvTMdABzltiYUl8F8MW0Ehll6nGYAxzLLAqOG+sv9hMemahijaCn42MBg9GvCTv\\\\\\\/Vhu2zNkEgHyPvzMbXxNIc+ndwFLbrboQ2HEYXqdqHhhNe3qJp3Om900giDRpp1JTxvicmjUL1sF\\\\\\\/DaLTw2hpexpKFxVidNm8Xb4\\\\\\\/6bjfFOCPEHof4pjnP2pBHo5Vac4es8EHr3xNwe4XwSBYrmJD5Ie\\\\\\\/8zzFgsPZryDgf\\\\\\\/QNMUC3xMSoOnxOCkx0Y3wTEAYT1yqXWAbi91VV0EH4cP5Hp+ZwVqw8Vbqu\\\\\\\/PPF1T1Q7xeybWeNgiqGuL9LKsSkYtYgGFVHYkvhZVzwvEJ4KtZzbez3NToQe0bm99QCeVcWgDypo9vDxRpesFQjdc3Q+9bshCx4dLau0LGEaS5TGf9qexo0K1SkeQHr3SllVksymKeRF7fwQ2Bf7\\\\\\\/z4j39V2p6VPFny91bKbm09qVhEmhwDRbcYAjs9GCMh+7IXRpNpk9VK8VXhz1r2pQYxTdbw\\\\\\\/qkyyVLwxfzEtugCoBhXIvw\\\\\\\/aWCrINYuRoam2Fs9mNnnR0miOcHfVBUsKYfZ39mlYdGm8KxIt2tcoK8FVrKLvP99n1wFXlBtwcfw2F0KOXj2ke1myUn1ZEp1UkG4xeXvNLXgRXDsL7zp0Gnh9BGqHpImdLyhU+0ZYu1\\\\\\\/YRcLEkFQZ3Ey\\\\\\\/zneMuhDpjwYCWp+Z+8lbCjkq+83goANsruPu9rjLSI\\\\\\\/qCRsJkdSf9Igby+rXHFU2cvlD0vh+6+N9+Uy9NDYQal9p8ymBzXZyYy4fHV\\\\\\\/yo8LN5Jt04kzC8wd16qK6tG0C0qJCKavkci6YqDaiNnBI3nhXV1vq7u3dvlPhkPIUj0exiIFUwX50jUgZAxG8quUnuyevwRHWzjIc7\\\\\\\/k78MtZlx3bRAGxfY7H7tOJgzaz\\\\\\\/FgY3u4mYJUL2R9yWoRHX72+NlIIPoCQw5NHmTCQBMh6DxjZsWGmfcOruL\\\\\\\/QYggrd7wtJ0mAwjmCOhbWXKy3+NT0l9zMzU1QbgTq3iCZxrx9gl2VR6wRjlw5r\\\\\\\/x4LqjRlUHAWzUh57G9zEIhicS7QSkoCibY1Yj2iBaUSLs57EESmlMQcHIMs0dqgxMlPE2KZmYFrh8MDaeNr7TlLuENpOtehBw42CWjO42MA8Y8rUqAi9rVkuC\\\\\\\/CnbPuWSsn5A9zKzKlKsYAHeEyBBI8HMzyqSfI7O1c7Iw6ejuTlTseMuGOaRJV880i40wQ7EAtvGyfAOeZnwQLRqQV4o5EN4vHKmiGfYxRvYo7wg6gNU0Mipz6Gba9SJO\\\\\\\/Ja\\\\\\\/2v88gJlnhJoUjncSSX\\\\\\\/hYlx20P\\\\\\\/lmiVxuQ1CPyQ6Jpt93g68V9+8hGOv2SHkrLe6IvnczkpnE2B1GbVisJ4Rxap+fG2VjXdgi1uL7U2asQ4jtKXxQISJVB1W1liFuTyNcwuQrREwQydKvUBPOZ2161FZOk5+rJg8bCPjRV2WOTPAGpw1vsOhEhx+XyDe0e6GuPN1r00YhpX3A5CpXvonyorMjYVmAH35pidqWJzVmNylIkJEq4Ul7MBqWoAJLdufNdfbWHsmbSSYvqX3HOgnkXn4l23mlGdPqTMx35StjCjNAMgROnge2bvPU3g9VaXn+al1Jvq9XBFRK7LqYmMmSCwy1wi\\\\\\\/18mQJukwilKiQOmKKpriCVv4xhuPQDohum+H4wdcgxrquwHs\\\\\\\/jwrKM05Ku7gI\\\\\\\/NPRjo8gQBW11Z1drJ\\\\\\\/7kn74HrB+nto3U+fbBKvWVFK3DLCGTQHXyoJ7\\\\\\\/TmIfAFeinK9fNXszCjvqdGsUYlh56RwmFsAh0NloicjnGtLK\\\\\\\/xodLXSj5Yof0pVg123iliyEm18rqvEEElINFMGRaLTWCwiX737doT8iiqlGeMecFj4hK9LsJNxh6IKR9a7+GwMJ7jcTV\\\\\\\/aVgUh+oBALjakroMUmvIps6FzG1ke38+Sd1Dy+xEMlDhhumwg0eQ0mb3esmbTUZy\\\\\\\/e7RXrd\\\\\\\/Q7OZWDPjxie72rCgj89SoKfBViHWo9kcOcf5vgGvvhY9BAUwzCdcGLCt7SK+aOtFtzRPUjlM1WLtszreJJ6pHZbv21RxXBU3h2ukgg9j6BRtx9LgTrd8dW7BIDZaENY50OmS5IPx4rc6uBQ32atcbNqAHUN+N+wZnkjRvZ+3wNuvtmH4hmQ6n+T0YRCfd3o9pWAUpRw\\\\\\\/5NLnbFluHCO9qEUYEEBJB5B4n07HlpFSL8rtCQ2qiQJHOvgkPavOonr5cu9o14Qyasnyk9+TXuo1ybqCo3p5jQks9PPyZppO9lCinBcEA9P963Xl0Bw9p1lui68s\\\\\\\/Mb7u94wYJcIPzz6E9E3R1LwmSEZzmaAEh\\\\\\\/WQBmjsx+bad6W8DALz\\\\\\\/BFe5sSrX+EFAqXfmlJ1XfxF3vfI29xwSDTpKWzN\\\\\\\/2X0nSG8BjxBZTxpDjixuku7kwrgmc\\\\\\\/DaGMvT2Rrt7V8WSfvxPCw7AkqPaqE0R+jz5RuW5yFaQpC62KCFDxyHHezUB4CqgzasktvwWwVubSz4F1TuhHLAdZJcxKaxDPDExmsx+L9B0tw2LPduVntwPeQ5zdBaV9BPx6WknmA9inqAR3xlteZKwzqO7kQ+47iaxZMi\\\\\\\/rcdYUeBo1aq9RYGy2vCcPfKqxvgqZ53CJyuobIXY0fesKYwvF7YaWt1WJXdRyB+yRMYT3f65fjNvEaLa1FLa0wP3NT79HSGFTFUZOWzekcsCjAuHf6wj3et8C290OVzKRD+LLlZBvJyNHa4f2RU2cBE\\\\\\\/qH4vuqVwlzyWn0KMDqqd1mQ5ZTdk4RZF4eiNOelP78Ye7IYFR9flAipO8SyUnVMY32F9JzSJyTPmDzsr5qNx4YyVwD7E8LaSM1xlqLPWlBVVerb6kEJf9p6xk69gXfC5jEqjT54u615X2R6zhB1VzIBXBgDjMRB1vjt550imp7hPSbxkdNKZ1nG8lzmes8PMx4BSXezCKbf+KXznTPpg3IdhMGusI9eSa+C+ymWFwbdB25rRx\\\\\\\/BcB5wDmO4lmcsDc2IIVu2gFuR5+Msos0T52Xca0JyAb7L\\\\\\\/YUKaAQn1MKKVvqD7T49uzVuySbUYOfT2XL+B36GWdRHzMhRjlOk81MqpbuC4tnbRY5dyjrkki\\\\\\\/r+XY6eS3oxYceQxe4IIHf7c1\\\\\\\/NNI9+13RWeU6uk\\\\\\\/rab0hbNEIWb2dMrr2u+5n\\\\\\\/NUrBjuac4cN0iBbkbeHronbL\\\\\\\/1fMmcO4fVE6NPjiCA1PpqkgIJUNHf02gy9ZFzxYs0MGgjo8l5EhdXV1Ra9Go2Zr8zSigh41JhqLmukUieSPl8Y5DgfLC\\\\\\\/jqDeFtQJvDafj7MXyvZXPZOiiLNLBBAoPva0pDErA7GUUd1+bRLOYoV4gHe0648DswTisEAqAi\\\\\\\/+RsGCG1np\\\\\\\/u1wH31LzU++oboi2MQd1Ch231bkf\\\\\\\/wqxU5PliwxZQ5VdNLk7j2TBFZXFMr6fkWmx2ZJ2KsyLDkSt2pyw1WKNbdXR7WL+69RT\\\\\\\/dEOKqIpbVRRKPmL1ZuPrKzf3AS38sBTASJnFBc8wbVWII2o+aNItxvwz+y+7Yo\\\\\\\/6gVKWo48W5x3xu+FBfmHTm\\\\\\\/USpwSbw3zwfe2ZzSEk0eHQoVulyW9Z4JIS7MKYKlkBTzFCLY1ZhTj6PfXqdU5BdSduBO5H8Qulrild+6VknDx5VhVOkxLWLj2rh4cmlMs9V1Cc+J\\\\\\\/HEE8QXWDtLnY2csipUlL8fnKzLWbjkE9wBv6iPNEPmq\\\\\\\/0I64b2MVGb1TLceSqHVtfB6+EG6xgm5jVqB3hSCwYEj\\\\\\\/QNAbwec3znkFfdDxgUcRWDvimq34BcFiEJvwsKwFm3Y97Aq8\\\\\\\/DLfq+scRSqQ1Yej0R3bHhDdpyJ8csFvIggQ+p1tgBLOWBCS+63zFBylgd\\\\\\\/1hySn\\\\\\\/S\\\\\\\/Rov2GPqPw\\\\\\\/3Hw+\\\\\\\/bL831919Fm2Lfoldf7W7eH9eHn\\\\\\\/2otKT17vNaPB4h52hBa88ZpbC9QZppr85L3jnDdnpdYCWXKvQQABIsIbzMOEVpkI1G1S3NArezGbhhOZraOS5g6pLXBlVejt\\\\\\\/+iRWeJxr5olhwbRw0YJ3hw+INROZ4prn\\\\\\\/CrfcDxxvi9s8Acg\\\\\\\/u\\\\\\\/byd3DlqE5GkJX+H116AsBGaXhRPdx7Q8QMRHo2j6BV\\\\\\\/37L98qfX20N64BisGp6ENXCu1YzFvjiMOetKFq8dL\\\\\\\/+2Njxw1fpUnCUF3dmDffwtOkQHpx\\\\\\\/af9HR+qchWcyUKkQzuJTPlxjWRpi7asUF4HFcex7KnOiYbLYCY1S86hQHnhjLasH7MiL1IVGJ7\\\\\\\/GtLpkqUE57oQbDPFthSABnpcVK52cvTIVDpPtQ+yCKUFW+RZNoBFxLi+Qo1+xqXM2HLk0sKNx4su619yWylWzvccG8R1syg51hiSh6\\\\\\\/oGXi+bwxh0KNL9NpDw62RW6DLdelRkwuQliw0iuj7WIU4CZZqmaxn7ihxuU+g7M9DMWHSy2u5TsQPoyUgqllbmGb7G0y2cvshkY12ClpRBXvtkRwtfa7Fm24yuNnYaTjjyqzQGcIO5DJJQrSHMClcAVS4VLp4aSa4AZ9ND5z86Nd+YFc3hpBDVOy+MTmIXMvWCJcwkrhEuLgGSmKkVhMtNDu\\\\\\\/AaARAMLBWn0AoFMl2Xjx\\\\\\\/aywVHks\\\\\\\/wCEUyRqmWm5CmNoakA57kJwD4CyvohtDhfafR3x2MoZ4aMKFwW8TUNfLvO6mn4dsQfg7yNNBiV8y1HjJvIxBYFxVE8fyOCz3xwh0wshJaO1sNKNxmBg67bWumfRZZuyb37MfKpiLIZO7T7jqCuLiJoAVoEwwcAczfYss6vPOayBeXgoPu5hkm0tYyLk8k7x5d2jKcoVMy9nm0Nsy4rpYtJDZkrw5Po4VmVGeXbxdD4lWZ0xYeoebmZvXjDXepPLH\\\\\\\/DA\\\\\\\/si5LzO2qV2DltUSUO1dqIPGKb0J+BCr8MDGYFXSJJgTrDknvM2ASaDDQZ7OueH6B\\\\\\\/cJSv+05QE86zXDOpOGDXwEhu2mZKXXQKufzatdWbMRzxsFn50ARAo8fcbG0jBzhZye4kOfyXiyBRPsBtvGHoqBjZaoJGBoHoVN8zMXQ2hMheDuBstO+8BS7cMQNhB7YWe6hUIJ2uU2z5\\\\\\\/X8GlhcqjHxE6VyX4+U4xMgbW8xWA0wAL67wuLW9Hn8Wb7kvGFjo7pOz52F8cQ2hLF\\\\\\\/m+EZ1WjU83EhPN0kKHZhkZZqblGEXvSfI\\\\\\\/ZX0mQgeOt7tzwENo8EVvl8OCWTY5prREvW+5PZ1Ya3p8p\\\\\\\/YKxyeamzJJrwgWruK8i9p4E0ThIT06u9VsNvAqnmx+bMNQ8ksJfGJ98M\\\\\\\/9ElR\\\\\\\/YJ2ejk1EmhxGx2aXGyiDTuO5sb7j3dswnFB2SnHaYIqNT3Q3dqOQSYJFiQ\\\\\\\/pFgnWV+MK99\\\\\\\/F4RFsXPJLtZdjaxZfm4KQkFEb3t+R3O1U4mC\\\\\\\/MheoFMnFq79uV9b5l7eEvcLT8cD+AYSZyadlfhB8ALb6WgZEA2uBjC4gTqPiC\\\\\\\/j7SlA3YvD10P9ZrnhIjI0cNb2NLfP16uL3xyVINPkQwnwKS+DY7rRmTnWslesbvVKZxGQTT\\\\\\\/EeHMUPnFYYYW768JWhPTiuczgXP6cqDAoe2sKtEaivMDxKf08JF3yJiwIs2njUfGX7nIFulZGf\\\\\\\/OI0C596EfhQlw1serl5jjrC\\\\\\\/OAJnoVzsReMet10Q7KjzBDN+SAUkJIJWg1orHTHkQDKlMA+Wh6Erinoq+ul6kVAouUUneEHjQV9qfYfFc6nlbJmS7d9Xe47ZZTjdHzb4pTL7TPGs2mtKD4Ystx6KeUUOh8m0wnJ2WsX2re81\\\\\\\/9nO39wDGe5hc+zMxQWbUfjOWKavlhM+2uXNMyzERZq\\\\\\\/AYvnnbUlEjxrO\\\\\\\/0h7NLeC0AGFLLE2ebrrC4tdyI8PxfEyDWXeI9llRLqxxoqf+\\\\\\\/AhWTzp1FZDXYDeg9LV5nbPE\\\\\\\/cBghOfyfnKE\\\\\\\/yngwj7BmrYvjaoWV3ZWzqR\\\\\\\/BXmDRgkAXUn6Sz+IbXOAwz3CYTHomCIdizxFTZ37ScoFbNtZpuEV8ZuBw9BG3gIlXNV4FF+RX6deMn2znyywamsrCnrqjpPzlEYkiBcQhjs95uBjnxuV1eJkO63ngahY1eETjbYrWh0EhnXWj38JzwoPhTSnsYlEednQD4PoPqjJqHJ4uk92ioZ3egk2O1kMIGHAAVqQmPX82+s2Dig6573erAhpK61rLNvelvFfvRKQ4duDLG\\\\\\\/CWAVQLcRNX9SzZrFjOBVnpCYYfiTLoLrputKxpOPCXhFsBcAkn25G9OkO4CLtA+x5EdgNrUGo7LhO80LTmJu0lCiPVaOJg8HGynbta6vKNgKnP3W0r5RFX5LxvcaD6cLfCJG9ik4+h6dhSYMKfKTsk5jOMW8kh4TOGwJH+DLgEUZdhk+mkR03IL2rHrTxWjsrCBDR2G15N4D8DnOQWGc3rePHp\\\\\\\/jyTT5Y0btmVkk+7g5LrdNmIM54AkXxPc12aL2iRljABZCqZAKxqxb8VUlA1Ajx4ahRr2qolNWc5YqvClDlTx+EfC7qiUBoqNNV0CSSDb\\\\\\\/ohap7QThuTS2RlJBkDfgWudso2BZNI4jWJntrKjou+kF4LauuQlb\\\\\\\/1eIEf7sHov7J+Uu4X1UCzw4V\\\\\\\/Qx4azc0gJhecD2kAMBMUmnKmS0Ax1pzSuOr6su8Stw4fA7kD2+l\\\\\\\/usPXl1xzC58b7DoNPl9sEZWvEI7+lsUmonL3JHz+0jqcgl2pA\\\\\\\/vhEtD9o0h1mZBq8eQj3QO0DmrRWgOc51bDoCja8CmiXPt4dHtI0DxJGIbTJJH9BwCyps7CcTONW1SoXEPPYw9r1QxVFhrNebsvOYPRCckrqLRAXPhpdS5KVBNwUl+tmvMu48dDahT6cQDBcq8yD8tdPN+FHjS0ibgd8bWpjJdOAvZ9AnHlw2oNBfty1shu+r4Cm94AJyVkyCsItLVijiqJUlGSPp\\\\\\\/H38Y\\\\\\\/qfO3b7+bdj3rOlsSBQEYcnjg1bQN\\\\\\\/iCvCLGwEpd\\\\\\\/QBQWHT181Gn8OBNxLndjEPUz0lr0KTGkVwbVoOSzoPohheaz9nnC70MAwEyU0U2oQDCrIh3EfT4+k9L6kKeSCI9bp4Ux9etTeTTrRU9jH\\\\\\\/t71aTR4pVYRWg+clCTOKsvRDuoVTk8pa+DRPP3a13q0isau20PhpDjIWOlPrWy91bg6KFwAiAKTrAhARat4FStgCsPUQK7vSv05pJohiTH4v+Z\\\\\\\/XakT4KTzPx3RpGIWEwBYEs4BoZWEnabykTR0PrI9AWHvOs7JZ3nRhRGiPNFPV0jcn6QvqqNStizuctd6X8mD0kFNbGIvujs8M5gMtFHaGsUrIvZ9+I6pxc\\\\\\\/KgHY86o7S5Jh1UmttanvP1Cr1Ny5ryokXDbglcIFyz3x5jb3DRICk7vsUbefLVsbjM\\\\\\\/1T1sLc1tcP29nDmlzPlOY76tE1cWbbmi0GmH4v1azlCOXB+0MzMuSkDEUkFm7UfF+4q2x66Z\\\\\\\/7q\\\\\\\/7nPQxlGVI5XtGmJO7og1sc\\\\\\\/vdA0uSfEpqIJJX+y+NznedRcOSg0bXJlYmMZJchHY0JSlj6Zz9W\\\\\\\/b6c23MP1U7FU0v9JSNxQnF1JyX5Dr4Dfy\\\\\\\/g16NWjbc6bYNj1zDQ7ielwJ4dyXPk7vtOHQvAULyB\\\\\\\/ZjA8fdG6O45iPkNC4Ee8mskbkFqx9R3DAcsqCQkLoRFZZyxcMlWN9WpyyjGgmd2NP7j2xPnrjj8COQnafcZ6uhoUcXqmeEU2Dfuk69t8LFlyCET6CONcZnEJnABoKp9Csm7YVSQxEjiHBg7More667J6yquyH\\\\\\\/cq0TGeEhRuWQ6AYQHLLFtEsFOST\\\\\\\/Hte5nGkvMgfcfhtIUA+AwuHd4Eb\\\\\\\/5r8pua4pcK+DRw+oVHgp8U9ayliahGo9p5uPBmVAGtZbjJsHj02cpu3Gno3\\\\\\\/I9wYdzx+1eBl1ct3bEX45w1qOA0rEhBXGqTqF6w1Oj7JIJG9alekajNoNHWu3FDza1doKzH9pfywaqOXToki08disIY009RzYpCcqM7PzvzDeYm\\\\\\\/rVb7Pw5bkkDGedIpimNvD5VB6h1a3H0ed0yICjTMEY8LtXislwm+u3rHCR\\\\\\\/Vi2q+1vauJotkIhcay2OxXiKc0v48EFtMNzM3tex3Wr8L7TpteDXGp+U\\\\\\\/bxDLlcDZdfeT9Z13gHSKdnH08J67FxdMdhmEdts0sN38FSVEvc0mIKrabTbEfE+AqorOrM+dKCz1IuduKXxrUnoAtWt6mBKC+LQ0HizPPVcZvY646DWgoido5tqwGxpvBWjMeAnwx8Xf04O9jkcUAphKBtjrMeeb7WlV\\\\\\\/E6OtmNjUXs2VMe00oZ7aETMQ1nX2YTJmpnfPV5B1fOgaxCxlw4SnJdEXahN96skn927dqfQ83dYBLMtXOoXFJL5rWX2J3hq2yvrDD\\\\\\\/w061jpUt7DH2+q\\\\\\\/GnGOSL8PZjGf2OBFj2fVNxVCL1A3LzD9vOe0L25QqcnPA1bhM3lWy2MSqxlUNSIE4K6yrJ6i4hFc1HYBuPWDGksL6W+JYT3BQp6MynLjDABwGVEB\\\\\\\/5GuEdGPls8vG3Z\\\\\\\/pegF4WWmYE0fO9b\\\\\\\/AkIaPj8NLX0zr4RvD01GW5goUKAgLSoyHgut9TEYHp91bbMXNPwxGUhMspruWdN4SrcGWzLsZoTsoNFVfpM251QEF9YZqBnS8UJwQ+X\\\\\\\/ctq5meWOiBsMOxOp0JIib3bOImhH0l5L6QqOkXm3CSK7k0m5qjkBSAL3y8OTx5lN4M2J8KLJGO1fdWu2S7l2oFRhD6tWxqU2SBOb\\\\\\\/gbAgyuI8yUodEMsiM67Hn0du+CAchU0T+rr3qqGW8oQEm54RchjlPejZeqF5ouXZPQn7TLROrhf0VO3mGtN3ooX\\\\\\\/3\\\\\\\/XngyzAm1qV1Z0e+g4wCJ3gGf48t0aJqUptnKGuM3jkfnzjqwaBEg3xuLjw5Cu1ls5ALRSirxF16XH8tmu97Gm6VfKmAEZSQgOxUZsoE24f6Q0lt+78iLm\\\\\\\/Zd7pzdyY9yWdIT5wNh6KRG80psnXCCIM85lpwl0t2uFFo7TJkIv8gIFoucuo\\\\\\\/odJ0e+vcib+dGgPh4TVTeO5G\\\\\\\/HwqgtY9XXKgAb\\\\\\\/N\\\\\\\/6\\\\\\\/UMYuoTEQ98dCdOf6D8nPzUoYSkAc9dAKaSRGLr6ZNlwpW\\\\\\\/dHai8VED1kllPRuiwRSQOPqqvjk86cLLSJpGRorR8cAIId+iLzVZQMAbyaVMsHwr6zURPuwbNWCHoCDcSrrKKQegJPm5XetOXDkYQfKG3yy0\\\\\\\/gLifWq3vqCO25FQetnYjlrUre8UjqIEpcJt8eolh4Eu7thzqz27UnDFiwoKqUM7JNbQ9Z6rzWLNpdtv21DpQDUqMs\\\\\\\/Nb\\\\\\\/SkTGI8S\\\\\\\/\\\\\\\/RFEup4hKgchIDtPMjNqBXJ4n2DlO+oPC+0iw\\\\\\\/RaSxtvUJCImCKhuCTRV78jWkJGr75LghlIQUOY9jW\\\\\\\/DgwhsMg33PZhm44UciDKsRYxDM3jvZGO1DmOwD3HnlM6jbAaGlOy5Bi2gNOpZNOazFqlj\\\\\\\/EPCAOALxwhTEQ69C6wfw8vaVptURlqrFNF4D00ZyCJQXlC7MzgOr0NzMV8EfBWeW\\\\\\\/RciYJSWHSywGRvpC6oIs\\\\\\\/XM+qz6ABgkQctoDygi1Uyh+u4PS+vOyafqcn8\\\\\\\/tZLbSwfdxXZAMsNxp0RpA1N04Kc5OTBfb97IZaR3aIbZZlFCObYC\\\\\\\/SjaCbJ+85aPx5Io23hJnBu236bfmekI5C50Jox7j5CS5nQB\\\\\\\/yJlFM\\\\\\\/ce9Evt1ER0TFDa4wIkvws58VtSMDpZBWJ1qm2Cmf2Q98U1JeCLs9e2JKcNuX9XQY6GJn5B0FZ+LZqa4w6NMpYT5Ewf6uZfp+Tpnc\\\\\\\/8v6ZndbtACPmAUY\\\\\\\/OE5FSujj6EdP2bt1I52kdRnL0JQ9Fo1d7BtK228KM1GckZruZZSt9VxUSq77DrEXiDxhl5Bs3dmSJ\\\\\\\/edbakegYlJQfl3rRWfZyTS3Gv1EFKAubuQCxw7NvknWvAqXVujigVrk63juV6xgqr2pXW6Qk5FxDpEmyYJB6cCHC\\\\\\\/JhLD8xs3yuLG77QIKXE\\\\\\\/UAdlXZkrP8OTV1mRuoKesjyK0bGgXR+l1t4XNzcj4N\\\\\\\/Arao0NMIIhfiLjgQGKAIOD3JvP2obRcQttf4rfQBfAgbxpZz1S2CDgFKIWq2n5kJu8hvrn5sdFKP4pv4uuEx77NHfT\\\\\\\/3TVmLcDCxl8AEz8ngajnM3qFBUa\\\\\\\/owhRLnah2GJwTL6eWIDo8j1\\\\\\\/1a\\\\\\\/5oCWs3zBXcPiSSVHel929Ya+0NQ9ZNA9GmD0Bh1mQZp4P5LKvSmLqDHv0GT6lHlw5C0kvbmUf4Fc4wMiVVyC3m7iH0vr9jKguaG8CVZhEC2r8RfPElIfkQMNBRQjWVG4ytxzQgXV6OM18QAr6Z884Y65RS1cqbK4vTXmw2n+6h77udkGcehLlciYWfvS3SITZx8KgwxIdZSiIuTt\\\\\\\/l+zc4Adt+mMXaf23NutMS+UtLkSquRQ02jnZSl2ECdz3WlsVkAJYTiDsYz7L21lsiMaQUwvCwguiara22afRciMeoiibwRduOqY4qi8MY8XZ6BizPxbJwb1DAVLSUzPY3W7kPin3xQWgr2\\\\\\\/IRKLz5bF\\\\\\\/AkJmNmih\\\\\\\/qUkqW62+RBxBiotWsUXU9zyt2n59iH18Irq4DCd6EusUyz4xMiJewZxRyIhaIMXBubgkmb7tMbcov8yDz5KbDpq6m+KWp1n3CKD4etQKKz7jvLJjYWP7Tmep3ZZovi6HCjpH3IlBgUnTHz2lQ1TRzTEnCYceDGInXs5S0wKbcshJQdv7KpL+yYF2OgOaZmCdCImwFVZthBtGoyIIC1YqUYIPlL7iFKtNCm6RTB4kiSLe1lPgfEuW75p5Ejky0jSjaQBmv\\\\\\\/qkb+DELI4e\\\\\\\/FIxlEUPs6x7+sn0H4\\\\\\\/RlVPX\\\\\\\/yqZAbrpSYbffnCywzV+kaqEU6ID4o\\\\\\\/jQ5lOS2aZekAycco4WGq4c5whHd8qL98yTAyFAltMUYAoi\\\\\\\/p0UtYuSm3NxQovGohZHf8LtUMYn30CDEs5jkyGEkH21BdlPmSFFCIESv2bFIjPk9SibDqBzV9rYdDvm3vFeEnEzLyoea\\\\\\\/rXsx5jr91nXImWbfPrqcnVHX9vQDR0xZWsIursCz2R4JvVOxvLa9EO8SD+7leaZl6uqadOw\\\\\\\/Ji9JzZkdAfVxySBuRHRtwHrh8HYoIPtGg+i0jQK+zRyN0rdmQHt58tgy\\\\\\\/oiiq3\\\\\\\/hdV0LAr+4UFO8CKHbR15N\\\\\\\/ubFJc+LHilgT\\\\\\\/MwyNCd7DnRXbLLW\\\\\\\/1SoUHYfU92NuSbbchkWDU+wky2TyOfLVDMHKWEMCFVua\\\\\\\/qndiwgKeBL9l5t72YwGmGPTso2JNacTYqI4RM3GoAtrKgYyMmtjyx6TgxQDolXTbrGyTbV6+m67NOmu4qZrKZHsX9Mj5r6YbmyarvYStZy2K\\\\\\\/3UW8bPN8yrRKgcN4lbNmuB0Y8w2UrWbdvDk8uBavb0YwS7u+1ncqXHcTpfDxT6lF0nzcBEdT\\\\\\\/u8yd4MWpC\\\\\\\/FQL3EDfquEFZwrAWI3iR6uoSCIVMC9zWS1DwQvZug0bTUFj\\\\\\\/wVPllHkzyfr7pt66g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022E8resGIweN9iVHXK3Y7xCw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223969d12e99fd8d66a258c64efc49f00ea76f7546a71015d57debeb845abada57\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225jFQoEZnzxMY6Dmtt2atEGqjLbZ1pPVTVa9T\\\\\\\/XdEFD+t++jtxOV4M+9BrZrQ0X+BPXIIke4gw5lkfy38YieNcakbtOoza\\\\\\\/psmNomK3F8+i9vtcdUKXVxbI9nEMo4WWc8heN8aXfBGGXkPSDJ3+4B97JEI8I+QEWjThxu16\\\\\\\/nPDoyE6n8rJY4q60nKBXXNpikxS\\\\\\\/wYcyxK0zXnfWbpw3zQV7q0J2Noi6YfWkCmi3kDcX1++1fV48MAxrlB+7JhQHJK6YztkMdow0wr+T75KGWx5wX1bjG1QDTkMOj2ahCB\\\\\\\/nuBKRzzEGWgkSTRPMinBLNnF8YlxAffFe3mhWyB3nHSXrlCW05sTlAfUvI3y3aXetaqAMliw+Ex6vpa9nHF30+V85zMONerCGswptB0bQp6IN5iBt\\\\\\\/e5mD230YbFQnp\\\\\\\/YnDKtWCc3ePd4wDqIN9kEJOFhFjUrGRjcU8sa1wndFv63eyvEPhc7Rp+2AkDU4iUGOIMjBEsrF3ptL8Wo74TjYuAR0yUtkyr4kww0yBXHwagBZJibwPrzkqd\\\\\\\/bMyeSMTEz2rDOvjI76WJEL2XTkyew0xErf3UZ1C\\\\\\\/q5yAPsSCYYU0aqPRxdA0nAMwoGGyxxXE9ETDusd2eGq+yWR6L\\\\\\\/IcO86fp7B0363Na3dIXnpJNaO62N3WNTrOSfM0S2ueAU2M5MhinJSur4l0TAbFZ22OjquaANfzvJyLnI+eI94gfrwKow62hlOSPwuOmf0Vdavb+jUz+6hMv+ww\\\\\\\/d9kv\\\\\\\/8NkCFHWhEbYfx8BPFxD82gai9MX+g+wOXfrwLQc1h1yMfy6efm0O99q9QVPAAPpJ4YziW47YllVnB7K9EjbiXOzxxYcdlYyA6FRgiFZvTfqgon6fbG2yIzavc+EYvfbn2f1NNqnH+5zrwY1sd\\\\\\\/D5PP97qD4oV9yXW2ZFzGWmFLISqbIPuUksiJqWXJOTVXAr5G\\\\\\\/FKiNfh6nzRlg73wq856FZf5DS3KSo5s\\\\\\\/3JgfcY0EqHsUOu5xAy8lD1sif7w0+szZSQvLylSKTxrHF+Prl70WCvMLVR3gwvJHdvIk6SPsgHWIzfk6f0gMU4cSWoQhpqse1AeglDdUQmOymJ62nS9pLc7qapkikzPfnDBJEKlGJGun0jK\\\\\\\/J5iJRKmBPc7QkzyS79D607uRpM\\\\\\\/+TS+h7Rh3tEdMRLn07F2RYe4qeIAlEDfVgiLbcGleOPa7h118SEK+SHWBFYSms1g+axuT41PvcjGrE6HVHU6NJr1B\\\\\\\/db5hy481b9\\\\\\\/qvGzTcqjAt6kAe7+UfK9gYPBsLEOya1RL4U1fJB0GPOnWNCTssIrU5IpR1Uwg6nP\\\\\\\/BbZad5t4yoQsLUVLFuNzgvJViYEENg8s1q44n7N6fEAsv4Uq8Rb9EHAw0TCR+Tx41sOueaA3B3hZAhaJcLUfQYPhMTl2c+gvsLq2\\\\\\\/chrU9iswLDmUf7c+GNs3X+vykSZiZEfeG06wf6yPD4HEJXHr0duT0y6HvM9c9ePb\\\\\\\/S70c4j4YNMAJ8AwaLE+d+tc5RrXWPwq\\\\\\\/oFfmDgtXx72+qiYxzEmkgXWwsU9Ru9bYBcxHQw0nGPj6O75rXwYd8sBn\\\\\\\/si9Bm80r7jyC9hLfVhpN54kfb2XUXJJx+pNpf4T3FjDM5t6D6x1cDMsLk1tx9EW7533gnuMEJL4hvDXuFmvrUur8zTuVGeGB8JmemMqIF2QUZ7ioKheXKqUbKytt2S9w0TJUZRBKYLd\\\\\\\/bmQxmsXYUGkzLd7qexoseF6+8SPDYy2ixAR4PhCP8XB8vzq0KAuQ\\\\\\\/SnCLyTvTso8v0qLxwbyELKfuX3M33L6f5pBHyypWg49GXK4Yt0MrKnoWCFpJ1Wdas6nRwc6wz2c6j80aE5HOOCSSccB5gA0PetjlJ20zd9pnYyttE8OC6l6GBMOCNoWgp4TWYEKOYIEwhK60kqXtWSzi2SKP+smJh0KJyVAEMPOT\\\\\\\/4x0gL\\\\\\\/3tlSE9wHGTsYNNqQnWW4HR+aAg0xGhVM+19QGmaF7CQMAGzq5Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022B2\\\\\\\/DmH8qAFEVhIbE5tVPdQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022050b485064b5feb3791491c29d933e9f63acc9ca10fa31955c505382dd09ec79\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460415 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1835" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-j2RQityyb6JNApBZSdoQMCMIP4hS_BHK1Q_LbHHRb8g" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1835" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-c4Gg8Xp3jtzZKiwxhuTJnnE7TrzozvCEe30XDEip22g" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1819" title="Genomic Prediction of Gene Bank Wheat Landraces" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1841" title="The Drosophila melanogaster Muc68E Mucin Gene Influences Adult Size, Starvation Tolerance, and Cold Recovery" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460415" id="node460415" data-pisa="ggg;6/7/1835" data-pisa-master="ggg;g3.116.030056" data-apath="/ggg/6/7/1835.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Improved Genome Assembly of <em>Azadirachta indica</em> A. Juss.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Neeraja M. Krishnan</span>, <span class="highwire-citation-author" data-delta="1">Prachi Jain</span>, <span class="highwire-citation-author" data-delta="2">Saurabh Gupta</span>, <span class="highwire-citation-author" data-delta="3">Arun K. Hariharan</span> and <span class="highwire-citation-author" data-delta="4">Binay Panda</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1835-1840;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030056 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460415" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Neeraja M. Krishnan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neeraja%2BM.%2BKrishnan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krishnan%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANeeraja%2BM.%2BKrishnan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Prachi Jain </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Prachi%2BJain%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jain%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrachi%2BJain%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Saurabh Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saurabh%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaurabh%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Arun K. Hariharan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arun%2BK.%2BHariharan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hariharan%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArun%2BK.%2BHariharan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Binay Panda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Strand Life Sciences, Bangalore 560024, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Binay%2BPanda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Panda%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABinay%2BPanda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:binay@ganitlabs.in" class="" data-icon-position="" data-hide-link-title="0">binay@ganitlabs.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/7/1835" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460415" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460415/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1835.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460415" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460415/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1835.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460415" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460415/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1835.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460415" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460415/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/7/1835.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460415" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460415/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-corrections" >
  
      
  
  <div class="pane-content">
    <div class="messages highwire-corrections highwire-correction-type-correction"><h3 class="highwire-corrections-title">This article has a correction. Please see:</h3><ul class="highwire-corrections-list"><li class="relation-75004 first last"><a href="/content/9/7/2375" class="" data-icon-position="" data-hide-link-title="0">Corrigendum - July 01, 2019</a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Neem (<em>Azadirachta indica</em> A. Juss.), an evergreen tree of the Meliaceae family, is known for its medicinal, cosmetic, pesticidal and insecticidal properties. We had previously sequenced and published the draft genome of a neem plant, using mainly short read sequencing data. In this report, we present an improved genome assembly generated using additional short reads from Illumina and long reads from Pacific Biosciences SMRT sequencer. We assembled short reads and error-corrected long reads using Platanus, an assembler designed to perform well for heterozygous genomes. The updated genome assembly (v2.0) yielded 3- and 3.5-fold increase in N50 and N75, respectively; 2.6-fold decrease in the total number of scaffolds; 1.25-fold increase in the number of valid transcriptome alignments; 13.4-fold less misassembly and 1.85-fold increase in the percentage repeat, over the earlier assembly (v1.0). The current assembly also maps better to the genes known to be involved in the terpenoid biosynthesis pathway. Together, the data represent an improved assembly of the <em>A. indica</em> genome.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/pasa" class="hw-term hw-article-keyword hw-article-keyword-pasa" rel="nofollow">PASA</a></li><li class="kwd"><a href="/keyword/mate-pair" class="hw-term hw-article-keyword hw-article-keyword-mate-pair" rel="nofollow">mate-pair</a></li><li class="kwd"><a href="/keyword/assembly" class="hw-term hw-article-keyword hw-article-keyword-assembly" rel="nofollow">assembly</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/fdft1" class="hw-term hw-article-keyword hw-article-keyword-fdft1" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">FDFT1</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/sqle" class="hw-term hw-article-keyword hw-article-keyword-sqle" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">SQLE</em></a></li></ul><p id="p-3">High-throughput sequencing platforms, especially those based on short read technology, have enabled sequencing of many plant genomes (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Michael and Jackson 2013</a>). This has substantially improved our understanding of genome organization, evolution and complexity in different plant species. However, most first generation genome assemblies are draft and incomplete assemblies. The correctness and accuracy of genome assembly depends on the length of the sequencing reads, errors generated during sequencing, and the accuracy of the computational tools (assemblers and downstream annotation pipelines) used. Additionally, most genome assemblers are not suitable to assemble genomes of heterozygous plants, a characteristic feature of most plants in the wild (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Kajitani <em>et al.</em> 2014</a>). Draft assemblies often bear significant gaps and errors, yielding less accurate gene predictions and annotations. This is compounded by the usage of incomplete training sets with gene prediction algorithms and absence of a representative transcriptome that can correctly anchor to the genome. Therefore, it is imperative to improve the quality of draft genome assemblies with the help of longer reads using genome assemblers tailored to handle heterozygosity, and make gene predictions using updated training sets and gene annotations using combinatorial approaches not fully reliant on sequence similarity such as Basic Local Alignment Search Tool (BLAST).</p><p id="p-4">Neem (<em>Azadirachta indica</em> A. Juss.), belonging to the order Rutales, family Meliaceae, is an important woody angiosperm, given its many medicinal and agrochemical uses. We had previously sequenced and reported the draft genome and five organ-specific transcriptomes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Krishnan <em>et al.</em> 2011</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">2012</a>) of a neem tree. The neem genome was the 38th plant genome to be sequenced (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Michael and Jackson 2013</a>). The genome assembly was generated using short paired-end reads (76 bases or shorter) from Illumina GAIIx with a first generation genome assembler, SOAPdenovo (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Li <em>et al.</em> 2010</a>). This was followed by genome annotation and gene prediction analysis, analysis of repeat elements, phylogenetic analysis, and gene expression studies (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Krishnan <em>et al.</em> 2012</a>). In the current report, we have improved the quality of the neem genome assembly by (a) using additional long-insert libraries from Illumina Hiseq, (b) using long reads from a third generation sequencer by Pacific Biosciences (PacBio), (c) using LoRDEC (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Salmela and Rivals 2014</a>), an algorithm that takes short reads from Illumina and uses those to correct errors in the PacBio reads, and (d) assembling the genome with short and error-corrected long reads using Platanus (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Kajitani <em>et al.</em> 2014</a>), which is better suited to assemble heterozygous genomes. We reassembled all five organ-specific RNA libraries into a pooled representative transcriptome, using Trinity (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Grabherr <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Haas <em>et al.</em> 2013</a>), and employed the Program to Assemble Spliced Alignments (PASA, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Haas <em>et al.</em> 2003</a>) to benchmark the completeness of previous (v1.0), intermediate, and current (v2.0) genome assemblies based on their mappability to this transcriptome. We also performed gene prediction analyses with GlimmerHMM (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Majoros <em>et al.</em> 2004</a>, v3.0.4) using updated training sets from <em>Citrus</em> species, which were found to be evolutionarily closer to neem by our earlier phylogenetic analyses (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Krishnan <em>et al.</em> 2012</a>). Building on our draft assembly, here, we present data on different assembly parameters, accuracy, gaps, gene predictions, and the total repeat content as evidence toward an improved neem genome assembly.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Assembly</h3><p id="p-5">In addition to the Illumina read libraries used for assembling the previously published draft neem genome (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Krishnan <em>et al.</em> 2012</a>), four more libraries were used for updating the assembly. We included reads from three Illumina mate-pair (with insert sizes 4 kb, 6 kb, and 10 kb) and one PacBio (average read length &gt; 2kb, varying up to 17.64 kb) libraries. Details of all libraries used are presented in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS1.xls">Table S1</a>.</p><p id="p-6">We preprocessed all the libraries as follows. In the case of Illumina libraries, exact read duplicates were removed using the ’in silico normalization’ utility from Trinity. For PacBio, reads were error-corrected using LoRDEC v0.4.1 based on the two paired-end Illumina libraries (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS1.xls">Table S1</a>). kmers ranging from 19 to 36 were tested for error-correction. We made an effort to assemble intact PacBio reads following error-correction using the PacBioToCA (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Koren <em>et al.</em> 2012</a>) pipeline and Celera WGS assembler v7.1 (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Myers <em>et al.</em> 2000</a>). However, this process was CPU- and RAM-intensive, and also resulted in a suboptimal assembly (data not shown). We, therefore, converted the PacBio reads, with and without error-correction, into Illumina-like paired-end reads (read lengths of 100 bases and average insert size of 350 bases) using SInC’s read generator (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Pattnaik <em>et al.</em> 2014</a>), which could be easily assembled using SOAPdenovo, SOAPdenovo2 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Luo <em>et al.</em> 2012</a>), and Platanus. Converting PacBio reads to Illumina-like reads did not nullify the advantage of the long reads, in terms of contiguity (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-7">We produced 13 intermediate assemblies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS2.xls">Table S2</a>) for quality comparison, as follows: (a) reassembly of the published version using SOAPdenovo with Illumina short reads (R.S1/v1.0); (b) assembly using additional Illumina libraries using SOAPdenovo2 (S2.DUP); (c) assembly of all Illumina duplicate-removed libraries using SOAPdenovo2 (S2); (d) assembly, using SOAPdenovo2, of all Illumina duplicate-removed libraries along with the error-corrected PacBio reads (S2.ecPB.21 and S2.ecPB.32, using kmers 21 and 32, respectively); (e) assembly using Platanus of all Illumina duplicate-removed libraries alone (P), or along with either the error-corrected PacBio reads using 19- (P.ecPB.19), 21- (P.ecPB.21), 32- (P.ecPB.32), and 36-mers (P.ecPB.36), or along with uncorrected PacBio reads (P.ucPB) f) assembly and gap-closing, using Platanus, of all Illumina duplicate-removed libraries and the PacBio library with (P.ecPB.32.gc/v2.0; kmer = 32) or without (P.ucPB.gc) error-correction.</p><p id="p-8">All assembly QCs were performed using QUAST v2.3 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Gurevich <em>et al.</em> 2013</a>). The assembly NG50 was estimated assuming the neem genome size to be 364 Mb (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Krishnan <em>et al.</em> 2012</a>). We refer to the R.S1 assembly as v1.0 (previous) and the P.ecPB.32.gc assembly as the improved v2.0 (current) in our comparisons statistics below.</p></div><div id="sec-3" class="subsection"><h3>Assembly mapping to transcriptome using PASA</h3><p id="p-9">PASA r20140417 was used to compare and evaluate all the assemblies. The representative neem transcriptome was assembled <em>de novo</em> using Trinity v2.0.6 with five tissue-specific published RNA-seq libraries. This transcriptome was mapped to various genome assemblies using PASA and the numbers and lengths of valid alignments, failed alignments, and transcript assemblies were compared. In addition, the numbers and lengths of exon-only regions of the valid alignments were also extracted and compared across the assemblies.</p></div><div id="sec-4" class="subsection"><h3>Gene prediction using GlimmerHMM</h3><p id="p-10">GlimmerHMM was used for benchmarking the assemblies. We created training sets based on <em>Citrus sinensis</em> and <em>C. clementina</em> (genes.gff3 files downloaded from <a href="http://phytozome.jgi.doe.gov/pz/portal.html">http://phytozome.jgi.doe.gov/pz/portal.html</a>), and used the inbuilt <em>Arabidopsis thaliana</em> training set to predict genes and gene structures in the neem assemblies. Both <em>Citrus</em> species were used here since they were found to be the evolutionarily closest to neem, among sequenced species (<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Krishnan <em>et al.</em> 2012</a>).</p></div><div id="sec-5" class="subsection"><h3>Repeat analyses</h3><p id="p-11">RepeatModeler v1.0.8 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Smit <em>et al.</em> 2014</a>), employing Repeat Scout, Tandem Repeat Finder, and Recon modules, was used to construct a library of novel repeats entirely based on the neem genome. Other tools such as LTR_finder v1.0.5 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Xu and Wang 2007</a>), TransposonPSI v08222010 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Haas 2010</a>), and MITE-hunter v11-2011 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Han and Wessler 2010</a>), were used to identify Long Terminal Repeats (LTRs), retrotransposons, and Miniature Inverted repeat Transposable Elements (MITEs), respectively. The neem genome assembly was masked using RepeatMasker v4.0.5 (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Smit <em>et al.</em> 2014</a>) with all these repeats and the updated plant (Viridiplantae) libraries from Repbase (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kapitonov and Jurka 2008</a>), to estimate the nonredundant genomic repeat content. This was further classified using the RepeatClassifier module of RepeatModeler.</p></div><div id="sec-6" class="subsection"><h3>Identification of FDFT1 and SQLE gene structures across assemblies</h3><p id="p-12">We obtained the transcript sequences corresponding to <em>FDFT1</em> and <em>SQLE</em> genes in <em>C. clementina</em> from KEGG (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Kanehisa and Goto 2000</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kanehisa <em>et al.</em> 2014</a>), and created a database of these sequences using the makeblastdb utility in the BLAST package v2.2.29 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>). These genes belong to the sesqui- and tri-terpenoid biosynthesis pathways, involved in the synthesis of the commercially important compound, azadirachtin, and hence were chosen for comparative analyses here. The neem transcriptome was mapped against the database using BLAST with an Expect (E) value threshold of 0.001. The mapped neem transcripts were traced to their PASA alignments in various genome assemblies. In cases where the identified transcripts for the same reference gene aligned to multiple neem scaffolds, consensus exon−intron structures were inferred individually for each scaffold, and the one agreeing best with the <em>C. clementina</em> gene structure was considered. The gene structures for all assemblies were plotted along with the corresponding gene structure in <em>C. clementina</em> using ‘Structure Draw’ (<a href="http://www.compgen.uni-muenster.de/tools/strdraw/index.hbi?">http://www.compgen.uni-muenster.de/tools/strdraw/index.hbi?</a>). Regions of gaps (Ns) in the assembly were highlighted in red.</p><p id="p-13">All scripts used in the assembly, QC, evaluation, genome-to-transcriptome mapping, gene prediction, and repeat analyses pipeline are presented under in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FileS2.pdf">File S2</a>.</p></div><div id="sec-7" class="subsection"><h3>Experimental validation of FDFT1 and SQLE genes</h3><p id="p-14">We synthesized primers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS3.xls">Table S3</a>) for <em>FDFT1</em> and <em>SQLE</em> genes to confirm whether the v2.0 assembly is indeed improved over the previous one. The primers were designed using <a href="http://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information</a> (NCBI) Primer-BLAST (<a href="http://www.ncbi.nlm.nih.gov/tools/primer-blast/">http://www.ncbi.nlm.nih.gov/tools/primer-blast/</a>) in genic regions using v1.0 and v2.0 assemblies. We amplified the genes using the conditions (denaturation at 94° for 30 sec, followed by 35 cycles of denaturation at 94° for 30 sec, annealing at 58° for 30 sec, and extension at 68° for 8 min, followed by a final extension at 68° for 10 min). The amplified products were loaded onto a 0.8% agarose gel to visualize DNA bands.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-15">The raw data described in this manuscript are submitted to the NCBI Short Read Archive under the accession numbers SRX1074131, SRX1074132, SRX1074133, and SRX1074134 (SRP013453). The assembly is deposited in FigShare: <a href="https://figshare.com/articles/AZ_fa/8080733">https://figshare.com/articles/AZ_fa/8080733</a>. The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Quality comparison across all versions of neem genome assembly</h3><p>We compared the correctness and completeness of all the assembly versions based on three measures:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-17">Assembly statistics using QUAST</p></li><li id="list-item-2"><p id="p-18">Metrics from transcriptome-to-assembly alignment using PASA</p></li><li id="list-item-3"><p id="p-19">Gene and gene structure prediction based on three different training sets using GlimmerHMM</p></li></ol><p>The first measure strictly quantifies the completeness of the assembly, while the middle one mainly quantifies the correctness of the assembly, and its completeness to the extent that the draft transcriptome is complete, and the last measure quantifies the completeness of the assembly, but also its correctness, with the assumption that the genes and gene structures in the organisms used as training sets are present, as is, in the neem genome. Detailed metrics from all the benchmarking tools are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS2.xls">Table S2</a>.</p></div><div id="sec-11" class="subsection"><h3>Comparison of assembly statistics</h3><p id="p-20">Overall, assembly statistics improved with Platanus over SOAPdenovo or SOAPdenovo2 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS2.xls">Table S2</a>), with the best assembly (v2.0) produced by Platanus using a combination of all duplicate-removed Illumina read libraries and error-corrected (kmer = 32) PacBio library in all three stages—assembly, scaffolding, and gap-closing. The scaffold numbers and the assembly size here were reduced by 2.6- and 3-fold, respectively, over those from the earlier draft assembly (v1.0; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The assembly using uncorrected PacBio reads, in combination with Illumina libraries (P.ucPB), resulted in the longest scaffold (12,211,325 bases). However, other important quality metrics were compromised for this assembly. N50 and N75 were highest for Platanus assembly using all Illumina-only reads (P; 4,002,232 and 1,489,583 bases, respectively). The v2.0 assembly revealed a 13.4-fold reduction in gaps over the v1.0 assembly (an average of 5414.21 Ns per 100 kb, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>) and a 2.26-fold lowered NG50. Incidentally, the NG50 for the Platanus assembly using Illumina-only reads (P; 1,587,838 bases) was comparable to that using SOAPdenovo (v1.0; 1,663,167 bases). Almost 60% of each assembly was covered at 5X when PacBio reads were assembled, along with Illumina read libraries, using SOAPdenovo2 or Platanus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS2.xls">Table S2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Improvements (fold change between current, v2.0, over the previous, v1.0, assembly) in various (A) assembly statistics and (B) PASA mapping statistics. The Y-axis is plotted on a logarithmic scale and the minor grids conform to uniform intervals on positive and negative Y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1584298486" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Improvements (fold change between current, v2.0, over the previous, v1.0, assembly) in various (A) assembly statistics and (B) PASA mapping statistics. The Y-axis is plotted on a logarithmic scale and the minor grids conform to uniform intervals on positive and negative Y-axis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1835/F1.medium.gif" width="396" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/7/1835/F1.medium.gif" width="396" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468440" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Improvements (fold change between current, v2.0, over the previous, v1.0, assembly) in various (A) assembly statistics and (B) PASA mapping statistics. The Y-axis is plotted on a logarithmic scale and the minor grids conform to uniform intervals on positive and negative Y-axis.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Comparison of transcriptome-to-genome alignment metrics</h3><p id="p-22">The numbers and cumulative lengths of all valid alignments and PASA assemblies were highest at 77,635 and 61,292, ∼100 Mb and ∼99 Mb, respectively, for the v2.0 assembly (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS2.xls">Table S2</a>). The cumulative size of valid exonic alignments was also highest at ∼48 Mb for this assembly, and the corresponding numbers and lengths of all failed alignments were least at 6584 and ∼32 Mb, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS2.xls">Table S2</a>). The overall valid alignments increased 1.25-fold, and the ones in exons increased by 1.95-fold for the updated (v2.0) assembly over the old one (v1.0) (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). Failed alignments went down by 3.5- and 5.9-fold in number and cumulative size, respectively (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>).</p></div><div id="sec-13" class="subsection"><h3>Comparison of predicted genes</h3><p id="p-23">We found the highest number of predicted genes and exons using training sets from any of the three organisms (<em>A. thaliana</em>, <em>C. sinensis</em>, <em>C. clementina</em>), with the v2.0 assembly (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS2.xls">Table S2</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The cumulative length of all predicted genes was highest for this assembly (68,723,917 bases) when <em>A. thaliana</em> was used as the training set. When <em>Citrus</em> species were used as training sets, however, the v1.0 assembly resulted in the highest cumulative predicted gene lengths (473,787,912 and 431,305,649 bases, respectively, with <em>C. sinensis</em> and <em>C. clementina</em>). The predicted gene lengths were comparable between both the assemblies after excluding gaps, suggesting this to be mostly a result of misassembly (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Improvements (fold change between current, v2.0, over the previous, v1.0, assembly) in the numbers (#s) and sizes (bases) of gene and exon predictions from GlimmerHMM. The Y-axis is plotted on a logarithmic scale and the minor grids conform to uniform intervals on positive and negative Y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1584298486" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Improvements (fold change between current, v2.0, over the previous, v1.0, assembly) in the numbers (#s) and sizes (bases) of gene and exon predictions from GlimmerHMM. The Y-axis is plotted on a logarithmic scale and the minor grids conform to uniform intervals on positive and negative Y-axis.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1835/F2.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/7/1835/F2.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468615" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Improvements (fold change between current, v2.0, over the previous, v1.0, assembly) in the numbers (#s) and sizes (bases) of gene and exon predictions from GlimmerHMM. The Y-axis is plotted on a logarithmic scale and the minor grids conform to uniform intervals on positive and negative Y-axis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">We found an abundance of smaller (&lt;100 bases) mRNAs and exons in gene predictions in the v1.0 assembly, especially with <em>Citrus</em> training sets, which were substantially reduced in the v2.0 assembly (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). In contrast, the longer mRNAs were more abundant in the latter assembly, with <em>Citrus</em> training sets, an indication of improvement in the assembly.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportion (%) of gene-bearing scaffolds/contigs with gene predictions of lengths 100 bases, for A. thaliana, C. sinensis, and C. clementina training sets." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1584298486" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportion (%) of gene-bearing scaffolds/contigs with gene predictions of lengths &lt;10 bases, 10&#x2212;100 bases, and &gt;100 bases, for &lt;em&gt;A. thaliana&lt;/em&gt;, &lt;em&gt;C. sinensis&lt;/em&gt;, and &lt;em&gt;C. clementina&lt;/em&gt; training sets.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1835/F3.medium.gif" width="253" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/7/1835/F3.medium.gif" width="253" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468751" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Proportion (%) of gene-bearing scaffolds/contigs with gene predictions of lengths &lt;10 bases, 10−100 bases, and &gt;100 bases, for <em>A. thaliana</em>, <em>C. sinensis</em>, and <em>C. clementina</em> training sets.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Comparison of gene structures of FDFT1 and SQLE across various assemblies</h3><p id="p-27">In order to demonstrate the biological significance of the improved assembly, we used <em>FDFT1</em> and <em>SQLE</em> genes, two important genes involved in the sesqui- and triterpenoid biosynthesis pathways. We observed that the gene structures of <em>FDFT1</em> and <em>SQLE</em> were more complete and accurate in the improved v2.0 assembly when compared to the v1.0 assembly (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FigureS2.tif">Figure S2</a>). Using Platanus alone, and augmenting the libraries with additional short Illumina mate-pair libraries yielded a better <em>FDFT1</em> gene structure. Similarly, using Platanus as an assembler along with pre-and postprocessing yielded a better assembly of the multi-isoform <em>SQLE</em> gene.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of v1.0 and v2.0 assemblies for (A) FDFT1 and (B) SQLE genes. The FDFT1 and SQLE transcripts from C. clementina were mapped to the representative Trinity-assembled A. indica transcriptome using NCBI BLAST (E-value 0.001). The transcripts were traced to their neem genomic scaffold mappings from PASA, in order to extract the exon&#x2212;intron structures of the corresponding genes. In the figure, boxes and lines denote exons and introns, respectively, and the red regions denote gaps in the assemblies. The scales are different for FDFT1 and SQLE and are, therefore, indicated individually." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1584298486" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of v1.0 and v2.0 assemblies for (A) &lt;em&gt;FDFT1&lt;/em&gt; and (B) &lt;em&gt;SQLE&lt;/em&gt; genes. The &lt;em&gt;FDFT1&lt;/em&gt; and &lt;em&gt;SQLE&lt;/em&gt; transcripts from &lt;em&gt;C. clementina&lt;/em&gt; were mapped to the representative Trinity-assembled &lt;em&gt;A. indica&lt;/em&gt; transcriptome using NCBI BLAST (E-value 0.001). The transcripts were traced to their neem genomic scaffold mappings from PASA, in order to extract the exon&#x2212;intron structures of the corresponding genes. In the figure, boxes and lines denote exons and introns, respectively, and the red regions denote gaps in the assemblies. The scales are different for &lt;em&gt;FDFT1&lt;/em&gt; and &lt;em&gt;SQLE&lt;/em&gt; and are, therefore, indicated individually.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1835/F4.medium.gif" width="440" height="144"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/7/1835/F4.medium.gif" width="440" height="144"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1835/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468730" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Comparison of v1.0 and v2.0 assemblies for (A) <em>FDFT1</em> and (B) <em>SQLE</em> genes. The <em>FDFT1</em> and <em>SQLE</em> transcripts from <em>C. clementina</em> were mapped to the representative Trinity-assembled <em>A. indica</em> transcriptome using NCBI BLAST (E-value 0.001). The transcripts were traced to their neem genomic scaffold mappings from PASA, in order to extract the exon−intron structures of the corresponding genes. In the figure, boxes and lines denote exons and introns, respectively, and the red regions denote gaps in the assemblies. The scales are different for <em>FDFT1</em> and <em>SQLE</em> and are, therefore, indicated individually.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">We found that the read support offered by Illumina and PacBio libraries for <em>FDFT1</em> and <em>SQLE</em> genes to be stronger and more contiguous in the case of the v2.0 assembly as compared to the v1.0 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FigureS3.pdf">Figure S3A and 3B</a>). Additionally, we used Integrative Genomics Viewer (IGV) to visualize the mapped reads to earlier and current scaffolds containing these two genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FigureS3.pdf">Figure S3</a>) to demonstrate the superiority of the v2.0 assembly over the earlier version (v1.0). As shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FigureS3.pdf">Figure S3</a>, additional short and long reads along with the usage of the assembler, Platanus, resulted in gene assemblies that are more contiguous (gray boxes) and with lesser gaps (white boxes) than the earlier (v1.0) assembly. We further experimentally verified both versions of the assemblies by designing primers to amplify two key genes, <em>FDFT1</em> and <em>SQLE</em>. We expected to obtain amplified products with sizes of 4 kb, 7.3 kb, and 3.8 kb (for partial <em>FDFT1</em>, full <em>FDFT1</em>, and partial <em>SQLE</em> genes, respectively) as per our v2.0 assembly (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS4.xls">Table S4</a>). Had the earlier version of the assembly (v1.0) been correct, we were expecting to obtain much higher sizes of the bands (11 kb or higher) for both <em>FDFT1</em> and <em>SQLE</em> genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FigureS4.tif">Figure S4A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS4.xls">Table S4</a>). As shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FigureS4.tif">Figure S4B</a>, it is clear that the v2.0 assembly is indeed an improved and correct one for these two genes over the previous assembly.</p></div><div id="sec-15" class="subsection"><h3>Estimation of repeat content</h3><p id="p-30">The repeat content was estimated to be 54,375,206 bases (24.15% of v2.0), which is higher than the 47,427,034 bases reported earlier (13.03% of v1.0). We further classified the repeats into distinct classes, as shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS5.xls">Table S5</a>.</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-31">Here, we report an improved genome assembly of <em>A. indica</em> and provide quantitative evidence on various parameters in support of the improved assembly. The current assembly benefits from using additional Illumina mate-pair reads and long reads from PacBio, a third generation sequencing platform. Additionally, we have used Platanus, a tool designed to assemble heterozygous genomes, such as that of neem (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FigureS1.tif">Figure S1</a>), better, and an algorithm that uses short reads to correct the errors in long reads. Finally, we have used updated and near complete training sets from closely related plant species to predict gene structures, and an equally enriched and updated repeat library to predict repeat sequences in the neem genome.</p><p id="p-32">In our study, we employed PASA and GlimmerHMM to benchmark the assemblies, both of which have their limitations in the current context. PASA assumes that the transcriptome is free of misassembly errors. The caveat with GlimmerHMM, is that the gaps and errors in the genomic assembly extends to the predictions (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). We found that the number of gene predictions decreased across assemblies, postredundancy removal using cd-HIT-EST (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Li and Godzik 2006</a>). Additionally, the gene predictions are only as good as the training sets used. Presence of a large number of very short, possibly spurious, exons in the <em>C. clementina</em> training set manifested in a large number of similar predictions in the neem assembly (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). However, as expected, either these did not align to the neem transcriptome, or a large fraction of those that aligned did not meet the validity criteria set by PASA, suggesting incorrect predictions. This implied a larger number of gene predictions not to be an indicator of correct or complete assembly in neem. Instead, integration of results from multiple tools, preferably using additional information from orthogonal high-throughput platforms such as RNA-seq, and experimental validation offered better benchmarking.</p><p id="p-33">The presence of duplicate reads may give false assurance to the assembler in terms of artificially inflated read depth. Hence, removing exact read duplicates reduced the number of misassemblies. Interestingly, we found that the assembly with SOAPdenovo2, after duplicate removal (S2), displayed worse statistics, but much improved transcriptome-genome mappings using PASA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS2.xls">Table S2</a>). SOAPdenovo, using fewer Illumina libraries, and without a duplicate removal step (v1.0), also displayed suboptimal assembly statistics but a good NG50 number (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). This, most likely, is due to an abundance of gaps in the assembly, inflating the assembly size. Incidentally, the NG50 numbers for assemblies using libraries from the same platform were comparable (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/TableS2.xls">Table S2</a>). Such observations caution against deriving conclusions regarding best assembly based solely on assembly statistics tools, such as QUAST. Exploring the finer details of individual genomic features, instead of macro-level statistics like NG50, may provide a better estimate of the improvement in the assembly quality, as exemplified by the improved assembly of two enzymes, <em>FDFT1</em> and <em>SQLE</em> catalyzing key stages of the biosynthetic route to sterols and triterpenes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Thimmappa <em>et al.</em>, 2014</a>), in the improved neem assembly (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1/FigureS4.tif">Figure S4</a>). Relying solely on sequence similarity-based approaches for gene identification can result in incomplete and/or inaccurate structural annotations. Using BLAST against <em>C. clementina</em> transcripts, with a stringent E-value threshold of 0.001, identified only portions of the <em>FDFT1</em> and <em>SQLE</em> genes in our scaffolds, making us falsely deduce that we had assembled only certain exons from these genes. This would be particularly true for structurally conserved genes, which have few very important, and, therefore, conserved domains. In such genes, variable domains might not have significant sequence homology to the reference database(s) that include sequences from other species, causing the genes to not be annotated in their entirety. Therefore, our approach of using the sequence similarity between <em>C. clementina</em> and neem transcripts to trace back the entire gene sequence structure and combining both reference- and <em>de novo</em>-based identification techniques is a better one (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><p id="p-34">In conclusion, genome assemblies need to be updated continuously by implementing accurate computational algorithms and supplementing with experimental evidence to obtain error-free and near complete assemblies. The process of obtaining accurate genome assembly is a dynamic and continuous process that needs to be undertaken, in our opinion, by groups or communities that have produced the first draft sequence of various genomes. This will facilitate research in genomics and create public resources to understand gene structure and function in plants better.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">We thank Manisha and Manju from Ganit Labs for performing PCR on <em>FDFT1</em> and <em>SQLE</em> genes. Research presented in this article is funded by the Department of Electronics and Information Technology, Government of India (ref no: 18(4)/2010-E-Infra., 31-03-2010) and Department of Information Technology, Biotechnology, and Science and Technology, Government of Karnataka, India (ref no: 3451-00-090-2-22). </p><p id="p-36">Author contributions: B.P. was responsible for overall planning, conception, and design of the study, data interpretation, and writing the manuscript; N.M.K. was involved in the conception, analysis, and interpretation of data, and writing of the manuscript; P.J. and S.G. were involved in the analysis and interpretation of data, and writing of the manuscript; and A.K.H. was responsible for sequencing data production.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-37">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030056/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-38"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-12-17"><span class="received-label">Received </span>December 17, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-04-16"><span class="accepted-label">Accepted </span>April 16, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Krishnan <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.2" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.3" data-doi="10.1093/bioinformatics/btt086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saveliev</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vyahhi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesler</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QUAST: quality assessment tool for genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinf.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1072</span>–<span class="cit-lpage">1075</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinf.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23422339&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318109300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-web" id="cit-6.7.1835.4"><div class="cit-metadata unstructured">Haas, B. J., 2010 Available at: <a href="http://transposonpsi.sourceforge.net">http://transposonpsi.sourceforge.net</a>. Accessed: May 15, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.5" data-doi="10.1093/nar/gkg770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mount</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">R. K.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Improving the <em>Arabidopsis</em> genome annotation using maximal transcript alignment assemblies.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">5654</span>–<span class="cit-lpage">5666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHaas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D31%26rft.issue%253D19%26rft.spage%253D5654%26rft.epage%253D5666%26rft.atitle%253DImproving%2Bthe%2BArabidopsis%2Bgenome%2Bannotation%2Busing%2Bmaximal%2Btranscript%2Balignment%2Bassemblies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg770%26rft_id%253Dinfo%253Apmid%252F14500829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14500829&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185600400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.6" data-doi="10.1038/nprot.2013.084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papanicolaou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blood</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>De novo</em> transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1494</span>–<span class="cit-lpage">1512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D1494%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.084%26rft_id%253Dinfo%253Apmid%252F23845962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845962&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.7" data-doi="10.1093/nar/gkq862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wessler</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MITE-Hunter: a program for discovering miniature inverted-repeat transposable elements from genomic sequences.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">e199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq862%26rft_id%253Dinfo%253Apmid%252F20880995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq862&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20880995&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.8" data-doi="10.1101/gr.170720.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kajitani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toshimoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noguchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyoda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient <em>de novo</em> assembly of highly heterozygous genomes from whole-genome shotgun short reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.170720.113%26rft_id%253Dinfo%253Apmid%252F24755901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTM4NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.9" data-doi="10.1093/nar/28.1.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>  <span class="cit-article-title">KEGG: Kyoto Encyclopedia of Genes and Genomes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res</abbr>. <span class="cit-vol">28</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKanehisa%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D30%26rft.atitle%253DKEGG%253A%2BKyoto%2BEncyclopedia%2Bof%2BGenes%2Band%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.27%26rft_id%253Dinfo%253Apmid%252F10592173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.1.27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10592173&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084896300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.10" data-doi="10.1093/nar/gkt1076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furumichi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Data, information, knowledge and principle: back to metabolism in KEGG.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D199</span>–<span class="cit-lpage">D205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1076%26rft_id%253Dinfo%253Apmid%252F24214961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24214961&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.11" data-doi="10.1038/nrg2165-c1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kapitonov</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A universal classification of 329 eukaryotic transposable elements implemented in Repbase.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">412, 414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKapitonov%26rft.auinit1%253DV.%2BV.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D411%26rft.epage%253D412%26rft.atitle%253DA%2Buniversal%2Bclassification%2Bof%2Beukaryotic%2Btransposable%2Belements%2Bimplemented%2Bin%2BRepbase.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2165-c1%26rft_id%253Dinfo%253Apmid%252F18421312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2165-c1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18421312&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.12" data-doi="10.1038/nbt.2280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schatz</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walenz</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Hybrid error correction and de novo assembly of single-molecule sequencing reads.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DKoren%26rft.auinit1%253DS.%26rft.volume%253D30%26rft.issue%253D7%26rft.spage%253D693%26rft.epage%253D700%26rft.atitle%253DHybrid%2Berror%2Bcorrection%2Band%2Bde%2Bnovo%2Bassembly%2Bof%2Bsingle-molecule%2Bsequencing%2Breads.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2280%26rft_id%253Dinfo%253Apmid%252F22750884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22750884&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krishnan</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pattnaik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deepak</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hariharan</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaur</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>De novo</em> sequencing and assembly of <em>Azadirachta indica</em> fruit transcriptome.</span> <abbr class="cit-jnl-abbrev">Curr Sci (India)</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr%2BSci%2B%2528India%2529%26rft.volume%253D101%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.14" data-doi="10.1186/1471-2164-13-464"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krishnan</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pattnaik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaur</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choudhary</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A draft of the genome and four transcriptomes of a medicinal and pesticidal angiosperm <em>Azadirachta indica</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKrishnan%26rft.auinit1%253DN.%2BM.%26rft.volume%253D13%26rft.spage%253D464%26rft.epage%253D464%26rft.atitle%253DA%2Bdraft%2Bof%2Bthe%2Bgenome%2Band%2Bfour%2Btranscriptomes%2Bof%2Ba%2Bmedicinal%2Band%2Bpesticidal%2Bangiosperm%2BAzadirachta%2Bindica.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-464%26rft_id%253Dinfo%253Apmid%252F22958331%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-464&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22958331&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.15" data-doi="10.1101/gr.097261.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">De novo assembly of human genomes with massively parallel short read sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.097261.109%26rft_id%253Dinfo%253Apmid%252F20019144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzIvMjY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.16" data-doi="10.1093/bioinformatics/btl158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godzik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">Bioinf.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1658</span>–<span class="cit-lpage">1659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinf.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl158%26rft_id%253Dinfo%253Apmid%252F16731699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16731699&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238905700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.17" data-doi="10.1186/2047-217X-1-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587118&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.18" data-doi="10.1093/bioinformatics/bth315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Majoros</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders.</span> <abbr class="cit-jnl-abbrev">Bioinf.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2878</span>–<span class="cit-lpage">2879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinf.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth315%26rft_id%253Dinfo%253Apmid%252F15145805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15145805&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225250100057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The first 50 plant genomes.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D6%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.20" data-doi="10.1126/science.287.5461.2196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutton</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dew</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fasulo</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A whole-genome assembly of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2196</span>–<span class="cit-lpage">2204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMyers%26rft.auinit1%253DE.%2BW.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2196%26rft.epage%253D2204%26rft.atitle%253DA%2BWhole-Genome%2BAssembly%2Bof%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2196%26rft_id%253Dinfo%253Apmid%252F10731133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4MzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.21" data-doi="10.1186/1471-2105-15-40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pattnaik</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panda</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SInC: an accurate and fast error-model based simulator for SNPs, Indels and CNVs coupled with a read generator for short-read sequence data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-15-40%26rft_id%253Dinfo%253Apmid%252F24495296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-15-40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24495296&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.22" data-doi="10.1093/bioinformatics/btu538"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmela</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rivals</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">LoRDEC: accurate and efficient long read error correction.</span> <abbr class="cit-jnl-abbrev">Bioinf.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3506</span>–<span class="cit-lpage">3514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinf.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu538%26rft_id%253Dinfo%253Apmid%252F25165095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25165095&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-web" id="cit-6.7.1835.23"><div class="cit-metadata unstructured">Smit, A. F. A., R. Hubley, and P. Green, 2014 <em>RepeatMasker Open-4.0</em>. Available at: <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>. Accessed: May 10, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.24" data-doi="10.1146/annurev-arplant-050312-120229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thimmappa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geisler</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louveau</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’ Maille</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osbourn</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Triterpene biosynthesis in plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft.volume%253D65%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-050312-120229%26rft_id%253Dinfo%253Apmid%252F24498976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-050312-120229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24498976&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1835.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340193000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1835.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">LTR_FINDER: an efficient tool for the prediction of full-length LTR retrotransposons.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">W265–268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D35%26rft.spage%253DW265268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/7/1835.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1819" title="Genomic Prediction of Gene Bank Wheat Landraces" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1841" title="The Drosophila melanogaster Muc68E Mucin Gene Influences Adult Size, Starvation Tolerance, and Cold Recovery" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/7"><span>Volume 6 Issue 7, July 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/7.cover-source.jpg?itok=E84OOyGI" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1835" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fECda0VkDKrHww9__cxOtNzz_03zRxY9NNjwglElMHE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/7/1835" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/7/1835" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/7/1835" class="active" data-icon-position="" data-hide-link-title="0">An Improved Genome Assembly of Azadirachta indica A. Juss.</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460415" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-osLolxJhCiN-0d6cjAI41UP-JcahykSsxweM4CIFQ50" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1835.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/7/1835" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Gh7yND65L17Oj8D3C6-r0eJPvd_oK0mAfTFxqdOqLII" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/7/1835" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1835.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460415" id="node460415--2" data-pisa="ggg;6/7/1835" data-pisa-master="ggg;g3.116.030056" data-apath="/ggg/6/7/1835.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Improved Genome Assembly of <em>Azadirachta indica</em> A. Juss.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Neeraja M. Krishnan</span>, <span class="highwire-citation-author" data-delta="1">Prachi Jain</span>, <span class="highwire-citation-author" data-delta="2">Saurabh Gupta</span>, <span class="highwire-citation-author" data-delta="3">Arun K. Hariharan</span> and <span class="highwire-citation-author" data-delta="4">Binay Panda</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1835-1840;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030056 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460415--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Neeraja M. Krishnan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neeraja%2BM.%2BKrishnan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krishnan%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANeeraja%2BM.%2BKrishnan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Prachi Jain </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Prachi%2BJain%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jain%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrachi%2BJain%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Saurabh Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saurabh%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaurabh%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Arun K. Hariharan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arun%2BK.%2BHariharan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hariharan%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArun%2BK.%2BHariharan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Binay Panda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Strand Life Sciences, Bangalore 560024, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Binay%2BPanda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Panda%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABinay%2BPanda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:binay@ganitlabs.in" class="" data-icon-position="" data-hide-link-title="0">binay@ganitlabs.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/7/1835" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/7/1835&amp;title=An%20Improved%20Genome%20Assembly%20of%20Azadirachta%20indica%20A.%20Juss." id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/7/1835&amp;title=An%20Improved%20Genome%20Assembly%20of%20Azadirachta%20indica%20A.%20Juss." id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/7/1835&amp;title=An%20Improved%20Genome%20Assembly%20of%20Azadirachta%20indica%20A.%20Juss." id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/7/1835&amp;text=An%20Improved%20Genome%20Assembly%20of%20Azadirachta%20indica%20A.%20Juss." id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/7/1835&amp;title=An%20Improved%20Genome%20Assembly%20of%20Azadirachta%20indica%20A.%20Juss." id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/7/1835&amp;t=An%20Improved%20Genome%20Assembly%20of%20Azadirachta%20indica%20A.%20Juss." id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/7/1835&amp;title=An%20Improved%20Genome%20Assembly%20of%20Azadirachta%20indica%20A.%20Juss." id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/7/1835&amp;title=An%20Improved%20Genome%20Assembly%20of%20Azadirachta%20indica%20A.%20Juss." id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460415/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460415" id="node460415--3" data-pisa="ggg;6/7/1835" data-pisa-master="ggg;g3.116.030056" data-apath="/ggg/6/7/1835.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">An Improved Genome Assembly of <em>Azadirachta indica</em> A. Juss.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Neeraja M. Krishnan</span>, <span class="highwire-citation-author" data-delta="1">Prachi Jain</span>, <span class="highwire-citation-author" data-delta="2">Saurabh Gupta</span>, <span class="highwire-citation-author" data-delta="3">Arun K. Hariharan</span> and <span class="highwire-citation-author" data-delta="4">Binay Panda</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1835-1840;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030056 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460415--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Neeraja M. Krishnan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neeraja%2BM.%2BKrishnan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krishnan%20NM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANeeraja%2BM.%2BKrishnan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Prachi Jain </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Prachi%2BJain%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jain%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APrachi%2BJain%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Saurabh Gupta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Saurabh%2BGupta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gupta%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASaurabh%2BGupta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Arun K. Hariharan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arun%2BK.%2BHariharan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hariharan%20AK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArun%2BK.%2BHariharan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Binay Panda </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Ganit Labs, Bio-IT Centre, Institute of Bioinformatics and Applied Biotechnology, Bangalore 560100, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Strand Life Sciences, Bangalore 560024, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Binay%2BPanda%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Panda%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABinay%2BPanda%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:binay@ganitlabs.in" class="" data-icon-position="" data-hide-link-title="0">binay@ganitlabs.in</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1835&amp;rft.epage=1840&amp;rft.atitle=An%20Improved%20Genome%20Assembly%20of%20Azadirachta%20indica%20A.%20Juss.&amp;rft.volume=6&amp;rft.issue=7&amp;rft.date=2016-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Krishnan%2C+Neeraja+M.&amp;rft.au=Jain%2C+Prachi&amp;rft.au=Gupta%2C+Saurabh&amp;rft.au=Hariharan%2C+Arun+K.&amp;rft.au=Panda%2C+Binay"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460415/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460415/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460415/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460415/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460415/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460415/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460415/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460415/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460415/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460415/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460415/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460415/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/7/1835" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/7/1835.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/7/1835.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/7/1835.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/7/1835.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
