<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/7/1879" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/7/1879.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/7/1879.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/7/1879.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/7/1879/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/7/1879.atom" />
<meta name="HW.pisa" content="ggg;6/7/1879" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Histone H1 Limits DNA Methylation in Neurospora crassa" />
<meta name="DC.Identifier" content="10.1534/g3.116.028324" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Seymour et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Histone H1 variants, known as linker histones, are essential chromatin components in higher eukaryotes, yet compared to the core histones relatively little is known about their in vivo functions. The filamentous fungus Neurospora crassa encodes a single H1 protein that is not essential for viability. To investigate the role of N. crassa H1, we constructed a functional FLAG-tagged H1 fusion protein and performed genomic and molecular analyses. Cell fractionation experiments showed that H1-3XFLAG is a chromatin binding protein. Chromatin-immunoprecipitation combined with sequencing (ChIP-seq) revealed that H1-3XFLAG is globally enriched throughout the genome with a subtle preference for promoters of expressed genes. In mammals, the stoichiometry of H1 impacts nucleosome repeat length. To determine if H1 impacts nucleosome occupancy or nucleosome positioning in N. crassa , we performed micrococcal nuclease digestion in the wild-type and the ![Formula][1]&lt;/img&gt; hH1 strain followed by sequencing (MNase-seq). Deletion of hH1 did not significantly impact nucleosome positioning or nucleosome occupancy. Analysis of DNA methylation by whole-genome bisulfite sequencing (MethylC-seq) revealed a modest but global increase in DNA methylation in the ![Formula][2]&lt;/img&gt; hH1 mutant. Together, these data suggest that H1 acts as a nonspecific chromatin binding protein that can limit accessibility of the DNA methylation machinery in N. crassa .

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif" />
<meta name="DC.Contributor" content="Michael Seymour" />
<meta name="DC.Contributor" content="Lexiang Ji" />
<meta name="DC.Contributor" content="Alex M. Santos" />
<meta name="DC.Contributor" content="Masayuki Kamei" />
<meta name="DC.Contributor" content="Takahiko Sasaki" />
<meta name="DC.Contributor" content="Evelina Y. Basenko" />
<meta name="DC.Contributor" content="Robert J. Schmitz" />
<meta name="DC.Contributor" content="Xiaoyu Zhang" />
<meta name="DC.Contributor" content="Zachary A. Lewis" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Histone H1 Limits DNA Methylation in Neurospora crassa" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Histone H1 variants, known as linker histones, are essential chromatin components in higher eukaryotes, yet compared to the core histones relatively little is known about their &lt;i&gt;in vivo&lt;/i&gt; functions. The filamentous fungus &lt;i&gt;Neurospora crassa&lt;/i&gt; encodes a single H1 protein that is not essential for viability. To investigate the role of &lt;i&gt;N. crassa&lt;/i&gt; H1, we constructed a functional FLAG-tagged H1 fusion protein and performed genomic and molecular analyses. Cell fractionation experiments showed that H1-3XFLAG is a chromatin binding protein. Chromatin-immunoprecipitation combined with sequencing (ChIP-seq) revealed that H1-3XFLAG is globally enriched throughout the genome with a subtle preference for promoters of expressed genes. In mammals, the stoichiometry of H1 impacts nucleosome repeat length. To determine if H1 impacts nucleosome occupancy or nucleosome positioning in &lt;i&gt;N. crassa&lt;/i&gt;, we performed micrococcal nuclease digestion in the wild-type and the &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me1&quot; overflow=&quot;scroll&quot;&gt;&lt;mi&gt;Δ&lt;/mi&gt;&lt;/math&gt;&lt;i&gt;hH1&lt;/i&gt; strain followed by sequencing (MNase-seq). Deletion of &lt;i&gt;hH1&lt;/i&gt; did not significantly impact nucleosome positioning or nucleosome occupancy. Analysis of DNA methylation by whole-genome bisulfite sequencing (MethylC-seq) revealed a modest but global increase in DNA methylation in the &lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me2&quot; overflow=&quot;scroll&quot;&gt;&lt;mi&gt;Δ&lt;/mi&gt;&lt;/math&gt;&lt;i&gt;hH1&lt;/i&gt; mutant. Together, these data suggest that H1 acts as a nonspecific chromatin binding protein that can limit accessibility of the DNA methylation machinery in &lt;i&gt;N. crassa&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/07/01" />
<meta name="citation_mjid" content="ggg;6/7/1879" />
<meta name="citation_id" content="6/7/1879" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/7/1879" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/7/1879.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/7/1879.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/7/1879.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.028324" />
<meta name="citation_pmid" content="27172195" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1879" />
<meta name="citation_lastpage" content="1889" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Michael Seymour" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Lexiang Ji" />
<meta name="citation_author_institution" content="Institute of Bioinformatics, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Alex M. Santos" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Masayuki Kamei" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Takahiko Sasaki" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Evelina Y. Basenko" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Robert J. Schmitz" />
<meta name="citation_author_institution" content="Department of Genetics, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Xiaoyu Zhang" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Zachary A. Lewis" />
<meta name="citation_author_institution" content="Department of Microbiology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_email" content="zlewis@uga.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1735-8266" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=I. Annalisa;citation_author=S. Robert;citation_title=The role of linker histone H1 modifications in the regulation of gene expression and chromatin dynamics.;citation_pages=486-495;citation_volume=1859;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. P. Baldwin;citation_author=P. G. Boseley;citation_author=E. M. Bradbury;citation_author=K. Ibel;citation_title=The subunit structure of the eukaryotic chromosome.;citation_pages=245-249;citation_volume=253;citation_year=1975;citation_issue=5489;citation_pmid=1167623;citation_doi=10.1038/253245a0" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. L. Barra;citation_author=L. Rhounim;citation_author=J. L. Rossignol;citation_author=G. Faugeron;citation_title=Histone H1 Is Dispensable for Methylation-Associated Gene Silencing in Ascobolus immersus and Essential for Long Life Span;citation_pages=61-69;citation_volume=20;citation_year=2000;citation_issue=1;citation_pmid=10594009;citation_doi=10.1128/MCB.20.1.61-69.2000" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=E. Y. Basenko;citation_author=M. Kamei;citation_author=L. Ji;citation_author=R. J. Schmitz;citation_author=Z. A. Lewis;citation_title=The LSH/DDM1 homolog mus-30 is required for genome stability, but not for dna methylation in neurospora crassa.;citation_pages=e1005790;citation_volume=12;citation_year=2016;citation_pmid=26771905;citation_doi=10.1371/journal.pgen.1005790" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=J. Bednar;citation_author=A. Hamiche;citation_author=S. Dimitrov;citation_title=H1-nucleosome interactions and their functional implications.;citation_pages=436-443;citation_volume=1859;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=D. Bell-Pedersen;citation_author=J. A. Y. C. Dunlap;citation_author=J. J. Loros;citation_title=Distinct cis-acting elements mediate clock, light, and developmental regulation of the Neurospora crassa eas (ccg-2) gene;citation_pages=513-521;citation_volume=16;citation_year=1996;citation_issue=2;citation_pmid=8552078;citation_doi=10.1128/MCB.16.2.513" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. Cao;citation_author=N. Lailler;citation_author=Y. Zhang;citation_author=A. Kumar;citation_author=K. Uppal;citation_title=HIGH-RESOLUTION MAPPING OF H1 LINKER HISTONE VARIANTS IN EMBRYONIC STEM CELLS;citation_pages=e1003417;citation_volume=9;citation_year=2013;citation_pmid=23633960;citation_doi=10.1371/journal.pgen.1003417" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=C. Cerf;citation_author=G. Lippens;citation_author=S. Muyldermans;citation_author=A. Segers;citation_author=V. Ramakrishnan;citation_title=Homo- and heteronuclear two-dimensional NMR studies of the globular domain of histone H1: sequential assignment and secondary structure.;citation_pages=11345-11351;citation_volume=32;citation_year=1993;citation_pmid=8218199;citation_doi=10.1021/bi00093a011" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. V. Colot;citation_author=G. Park;citation_author=G. E. Turner;citation_author=C. Ringelberg;citation_author=C. M. Crew;citation_title=A high-throughput gene knockout procedure for Neurospora reveals functions for multiple transcription factors.;citation_volume=103;citation_year=2006;citation_pmid=16801547;citation_doi=10.1073/pnas.0601456103" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=R. H. Davis;citation_author=M. B. Lawless;citation_author=L. A. Port;citation_title=Arginaseless Neurospora: Genetics, Physiology, and Polyamine Synthesis;citation_pages=299-305;citation_volume=102;citation_year=1970;citation_issue=2;citation_pmid=5419257" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. a. Downs;citation_author=E. Kosmidou;citation_author=A. Morgan;citation_author=S. P. Jackson;citation_title=Suppression of homologous recombination by the Saccharomyces cerevisiae linker histone.;citation_pages=1685-1692;citation_volume=11;citation_year=2003;citation_issue=6;citation_pmid=12820979;citation_doi=10.1016/S1097-2765(03)00197-7" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Y. Fan;citation_author=A. Sirotkin;citation_author=R. G. Russell;citation_author=J. Ayala;citation_author=A. I. Skoultchi;citation_title=Individual Somatic H1 Subtypes Are Dispensable for Mouse Development Even in Mice Lacking the H10 Replacement Subtype;citation_pages=7933-7943;citation_volume=21;citation_year=2001;citation_issue=23;citation_pmid=11689686;citation_doi=10.1128/MCB.21.23.7933-7943.2001" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=Y. Fan;citation_author=T. Nikitina;citation_author=E. M. Morin-Kensicki;citation_author=J. Zhao;citation_author=T. R. Magnuson;citation_title=H1 Linker Histones Are Essential for Mouse Development and Affect Nucleosome Spacing In Vivo;citation_pages=4559-4572;citation_volume=23;citation_year=2003;citation_issue=13;citation_pmid=12808097;citation_doi=10.1128/MCB.23.13.4559-4572.2003" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Y. Fan;citation_author=T. Nikitina;citation_author=J. Zhao;citation_author=T. J. Fleury;citation_author=R. Bhattacharyya;citation_title=Histone H1 depletion in mammals alters global chromatin structure but causes specific changes in gene regulation.;citation_pages=1199-1212;citation_volume=123;citation_year=2005;citation_issue=7;citation_pmid=16377562;citation_doi=10.1016/j.cell.2005.10.028" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=H. D. Folco;citation_author=M. Freitag;citation_author=A. Ramon;citation_author=E. D. Temporini;citation_author=M. E. Alvarez;citation_title=Histone H1 Is Required for Proper Regulation of Pyruvate Decarboxylase Gene Expression in Neurosporacrassa;citation_pages=341-350;citation_volume=2;citation_year=2003;citation_issue=2;citation_pmid=12684383;citation_doi=10.1128/EC.2.2.341-350.2003" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. E. Galagan;citation_author=S. E. Calvo;citation_author=K. a. Borkovich;citation_author=E. U. Selker;citation_author=N. D. Read;citation_title=The genome sequence of the filamentous fungus Neurospora crassa.;citation_pages=859-868;citation_volume=422;citation_year=2003;citation_issue=6934;citation_pmid=12712197;citation_doi=10.1038/nature01554" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=G. Geeven;citation_author=Y. Zhu;citation_author=B. Kim;citation_author=B. Bartholdy;citation_author=S.-M. Yang;citation_title=Local compartment changes and regulatory landscape alterations in histone H1-depleted cells.;citation_pages=289;citation_volume=16;citation_year=2015;citation_pmid=26700097;citation_doi=10.1186/s13059-015-0857-0" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. W. Harshman;citation_author=N. L. Young;citation_author=M. R. Parthun;citation_author=M. A. Freitas;citation_title=H1 histones: current perspectives and challenges.;citation_volume=41;citation_year=2013;citation_pmid=23945933;citation_doi=10.1093/nar/gkt700" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=S. Heinz;citation_author=C. Benner;citation_author=N. Spann;citation_author=E. Bertolino;citation_author=Y. C. Lin;citation_title=Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities.;citation_pages=576-589;citation_volume=38;citation_year=2010;citation_issue=4;citation_pmid=20513432;citation_doi=10.1016/j.molcel.2010.05.004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=S. Honda;citation_author=E. U. Selker;citation_title=Direct interaction between DNA methyltransferase DIM-2 and HP1 is required for dna methylation in Neurospora crassa.;citation_volume=28;citation_year=2008;citation_pmid=18678653;citation_doi=10.1128/MCB.00823-08" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Honda;citation_author=E. U. Selker;citation_title=Tools for fungal proteomics: multifunctional Neurospora vectors for gene replacement, protein expression and protein purification.;citation_volume=182;citation_year=2009;citation_pmid=19171944;citation_doi=10.1534/genetics.108.098707" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=S. Honda;citation_author=Z. A. Lewis;citation_author=K. Shimada;citation_author=W. Fischle;citation_author=R. Sack;citation_title=Heterochromatin protein 1 forms distinct complexes to direct histone deacetylation and DNA methylation.;citation_pages=471-477;citation_volume=19;citation_year=2012;citation_issue=5;citation_pmid=22504884;citation_doi=10.1038/nsmb.2274" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=A. Izzo;citation_author=K. Kamieniarz-Gdula;citation_author=F. Ramírez;citation_author=N. Noureen;citation_author=J. Kind;citation_title=The genomic landscape of the somatic linker histone subtypes H1.1 to H1.5 in human cells.;citation_pages=2142-2154;citation_volume=3;citation_year=2013;citation_pmid=23746450;citation_doi=10.1016/j.celrep.2013.05.003" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=L. Ji;citation_author=T. Sasaki;citation_author=X. Sun;citation_author=P. Ma;citation_author=Z. A. Lewis;citation_title=Methylated DNA is over-represented in whole-genome bisulfite sequencing data.;citation_pages=341;citation_volume=5;citation_year=2014;citation_pmid=25374580" />
<meta name="citation_reference" content="citation_journal_title=The FASEB Journal;citation_journal_abbrev=FASEB J.;citation_author=H. E. Kasinsky;citation_author=J. D. Lewis;citation_author=J. B. Dacks;citation_author=J. Ausió;citation_title=Origin of H1 linker histones;citation_pages=34-42;citation_volume=15;citation_year=2001;citation_issue=1;citation_pmid=11149891;citation_doi=10.1096/fj.00-0237rev" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. D. Kornberg;citation_author=J. O. Thomas;citation_title=Chromatin Structure: Oligomers of the Histones;citation_pages=865-868;citation_volume=184;citation_year=1974;citation_issue=4139;citation_pmid=4825888;citation_doi=10.1126/science.184.4139.865" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Kotliński;citation_author=K. Rutowicz;citation_author=L. Kniźewski;citation_author=A. Palusiński;citation_author=J. Olędzki;citation_title=Histone H1 variants in Arabidopsis are subject to numerous post-translational modifications, both conserved and previously unknown in histones, suggesting complex functions of h1 in plants.;citation_pages=e0147908;citation_volume=11;citation_year=2016;citation_pmid=26820416;citation_doi=10.1371/journal.pone.0147908" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=E. Kouzminova;citation_author=E. U. Selker;citation_title=dim-2 encodes a DNA methyltransferase responsible for all known cytosine methylation in Neurospora;citation_pages=4309-4323;citation_volume=20;citation_year=2001;citation_issue=15;citation_pmid=11483533;citation_doi=10.1093/emboj/20.15.4309" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. a. Lever;citation_author=J. P. Th’ng;citation_author=X. Sun;citation_author=M. J. Hendzel;citation_title=Rapid exchange of histone H1.1 on chromatin in living human cells.;citation_pages=873-876;citation_volume=408;citation_year=2000;citation_issue=6814;citation_pmid=11130728;citation_doi=10.1038/35048603" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Levy;citation_author=M. Eyal;citation_author=G. Hershkovits;citation_author=M. Salmon-Divon;citation_author=M. Klutstein;citation_title=Yeast linker histone Hho1p is required for efficient RNA polymerase I processivity and transcriptional silencing at the ribosomal dna.;citation_volume=105;citation_year=2008;citation_pmid=18687885;citation_doi=10.1073/pnas.0709403105" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Z. a. Lewis;citation_author=S. Honda;citation_author=T. K. Khlafallah;citation_author=J. K. Jeffress;citation_author=M. Freitag;citation_title=Relics of repeat-induced point mutation direct heterochromatin formation in Neurospora crassa.;citation_volume=19;citation_year=2009;citation_pmid=19092133;citation_doi=10.1101/gr.086231.108" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=C. Li;citation_author=J. E. Mueller;citation_author=M. Elfline;citation_author=M. Bryk;citation_title=Linker histone H1 represses recombination at the ribosomal DNA locus in the budding yeast Saccharomyces cerevisiae.;citation_pages=906-919;citation_volume=67;citation_year=2008;citation_issue=4;citation_pmid=18179596;citation_doi=10.1111/j.1365-2958.2007.06101.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with burrows-wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J.-Y. Li;citation_author=M. Patterson;citation_author=H. K. A. Mikkola;citation_author=W. E. Lowry;citation_author=S. K. Kurdistani;citation_title=Dynamic distribution of linker histone H1.5 in cellular differentiation.;citation_pages=e1002879-e1002879;citation_volume=8;citation_year=2012;citation_issue=8;citation_pmid=22956909;citation_doi=10.1371/journal.pgen.1002879" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=X. Li;citation_author=K. Liu;citation_author=F. Li;citation_author=J. Wang;citation_author=H. Huang;citation_title=Structure of c-terminal tandem BRCT repeats of Rtt107 protein reveals critical role in interaction with phosphorylated histone H2A during DNA damage repair.;citation_volume=287;citation_year=2012;citation_pmid=22262834;citation_doi=10.1074/jbc.M111.311860" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=X. Lu;citation_author=S. N. Wontakal;citation_author=H. Kavi;citation_author=B. J. Kim;citation_author=P. M. Guzzardo;citation_title=Drosophila H1 Regulates the Genetic Activity of Heterochromatin by Recruitment of Su(var)3-9;citation_pages=78-81;citation_volume=340;citation_year=2013;citation_issue=6128;citation_pmid=23559249;citation_doi=10.1126/science.1234654" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Luger;citation_author=W. Mäder;citation_author=R. K. Richmond;citation_author=D. F. Sargent;citation_author=T. J. Richmond;citation_title=Crystal structure of the nucleosome core particle at 2.8 A resolution.;citation_pages=251-260;citation_volume=389;citation_year=1997;citation_issue=6648;citation_pmid=9305837;citation_doi=10.1038/38444" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=B. S. Margolin;citation_author=M. Freitag;citation_author=E. U. Selker;citation_title=Improved plasmids for gene targeting at the his-3 locus of Neurospora crassa by electroporation.;citation_pages=34-36;citation_volume=44;citation_year=1997;citation_doi=10.1128/EC.00244-07" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=R. Mayor;citation_author=A. Izquierdo-Bouldstridge;citation_author=L. Millán-Ariño;citation_author=A. Bustillos;citation_author=C. Sampaio;citation_title=Genome distribution of replication-independent histone H1 variants shows H1.0 associated with nucleolar domains and H1X associated with RNA polymerase II-enriched regions.;citation_volume=290;citation_year=2015;citation_pmid=25645921;citation_doi=10.1074/jbc.M114.617324" />
<meta name="citation_reference" content="citation_journal_title=Journal of biosciences;citation_journal_abbrev=J Biosci;citation_author=K. McCluskey;citation_author=A. Wiest;citation_author=M. Plamann;citation_title=The Fungal Genetics Stock Center: a repository for 50 years of fungal genetics research.;citation_pages=119-126;citation_volume=35;citation_year=2010;citation_issue=1;citation_pmid=20413916;citation_doi=10.1007/s12038-010-0014-6" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Misteli;citation_title=Dynamic binding of histone H1 to chromatin in living cells.;citation_pages=877-881;citation_volume=408;citation_year=2000;citation_issue=6814;citation_pmid=11130729;citation_doi=10.1038/35048610" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=N. Nalabothula;citation_author=G. McVicker;citation_author=J. Maiorano;citation_author=R. Martin;citation_author=J. K. Pritchard;citation_title=The chromatin architectural proteins HMGD1 and H1 bind reciprocally and have opposite effects on chromatin structure and gene regulation.;citation_pages=92;citation_volume=15;citation_year=2014;citation_pmid=24484546;citation_doi=10.1186/1471-2164-15-92" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Noll;citation_author=R. D. Kornberg;citation_title=Action of micrococcal nuclease on chromatin and the location of histone H1.;citation_pages=393-404;citation_volume=109;citation_year=1977;citation_issue=3;citation_pmid=833849;citation_doi=10.1016/S0022-2836(77)80019-3" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. L. Olins;citation_author=D. E. Olins;citation_title=Spheroid Chromatin Units (ngr Bodies);citation_pages=330-332;citation_volume=183;citation_year=1974;citation_issue=4122;citation_pmid=4128918;citation_doi=10.1126/science.183.4122.330" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Newsl.;citation_author=M. L. Pall;citation_title=The use of ignite (basta; glufosinate; phosphinothricin) to select transformants of bar-containing plasmids in Neurospora crassa.;citation_pages=58;citation_volume=40;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=H. G. Patterton;citation_author=C. C. Landel;citation_author=D. Landsman;citation_author=C. L. Peterson;citation_author=R. T. Simpson;citation_title=The Biochemical and Phenotypic Characterization of Hho1p, the Putative Linker Histone H1 of Saccharomyces cerevisiae;citation_pages=7268-7276;citation_volume=273;citation_year=1998;citation_issue=13;citation_pmid=9516420;citation_doi=10.1074/jbc.273.13.7268" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=K. R. Pomraning;citation_author=K. M. Smith;citation_author=M. Freitag;citation_title=Genome-wide high throughput analysis of DNA methylation in eukaryotes.;citation_pages=142-150;citation_volume=47;citation_year=2009;citation_issue=3;citation_pmid=18950712;citation_doi=10.1016/j.ymeth.2008.09.022" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=Bedtools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=V. Ramakrishnan;citation_author=J. T. Finch;citation_author=V. Graziano;citation_author=P. L. Lee;citation_author=R. M. Sweet;citation_title=Crystal structure of globular domain of histone H5 and its implications for nucleosome binding.;citation_pages=219-223;citation_volume=362;citation_year=1993;citation_issue=6417;citation_pmid=8384699;citation_doi=10.1038/362219a0" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_title=Deletion of the unique gene encoding a typical histone H1 has no apparent phenotype in Aspergillus nidulans.;citation_pages=223-233;citation_volume=35;citation_year=2000;citation_issue=1;citation_pmid=10632892;citation_doi=10.1046/j.1365-2958.2000.01702.x" />
<meta name="citation_reference" content="citation_author=M. Rea;citation_author=W. Zheng;citation_author=M. Chen;citation_author=C. Braud;citation_author=D. Bhangu;citation_author=T. N. Rognan;citation_author=W. Xiao;citation_year=2012" />
<meta name="citation_reference" content="citation_title=Molecular Cloning: A Laboratory Manual;citation_author=J. Sambrook;citation_author=E. F. Fritsch;citation_author=T. Maniatis;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=C. Sancar;citation_author=N. Ha;citation_author=R. Yilmaz;citation_author=R. Tesorero;citation_author=T. Fisher;citation_title=Combinatorial control of light induced chromatin remodeling and gene activation in Neurospora;citation_pages=e1005105;citation_volume=11;citation_year=2015;citation_pmid=25822411;citation_doi=10.1371/journal.pgen.1005105" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=T. Sasaki;citation_author=K. L. Lynch;citation_author=C. V. Mueller;citation_author=S. Friedman;citation_author=M. Freitag;citation_title=Heterochromatin controls gammaH2A localization in Neurospora crassa.;citation_volume=13;citation_year=2014;citation_pmid=24879124;citation_doi=10.1128/EC.00117-14" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. Schäfer;citation_author=C. R. E. McEvoy;citation_author=H.-G. Patterton;citation_title=The Saccharyomyces cerevisiae linker histone Hho1p is essential for chromatin compaction in stationary phase and is displaced by transcription.;citation_volume=105;citation_year=2008;citation_pmid=18799740;citation_doi=10.1073/pnas.0806337105" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. J. Schmitz;citation_author=Y. He;citation_author=O. Valdes-Lopez;citation_author=S. M. Khan;citation_author=T. Joshi;citation_title=Epigenome-wide inheritance of cytosine methylation variants in a recombinant inbred population.;citation_volume=23;citation_year=2013;citation_pmid=23739894;citation_doi=10.1101/gr.152538.112" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. D. Schultz;citation_author=R. J. Schmitz;citation_author=J. R. Ecker;citation_title='Leveling' the playing field for analyses of single-base resolution DNA methylomes.;citation_pages=583-585;citation_volume=28;citation_year=2012;citation_issue=12;citation_pmid=23131467;citation_doi=10.1016/j.tig.2012.10.012" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. D. Schultz;citation_author=Y. He;citation_author=J. W. Whitaker;citation_author=M. Hariharan;citation_author=E. A. Mukamel;citation_title=Human body epigenome maps reveal noncanonical DNA methylation variation.;citation_pages=212-216;citation_volume=523;citation_year=2015;citation_pmid=26030523;citation_doi=10.1038/nature14465" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=C. Schwerdtfeger;citation_author=H. Linden;citation_title=Blue light adaptation and desensitization of light signal transduction in Neurospora crassa.;citation_pages=1080-1087;citation_volume=39;citation_year=2001;citation_issue=4;citation_pmid=11251826;citation_doi=10.1046/j.1365-2958.2001.02306.x" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. R. Shaw;citation_author=T. M. Herman;citation_author=R. T. Kovacic;citation_author=G. S. Beaudreau;citation_author=K. E. Van Holde;citation_title=Analysis of subunit organization in chicken erythrocyte chromatin.;citation_pages=505-509;citation_volume=73;citation_year=1976;citation_issue=2;citation_pmid=1061151;citation_doi=10.1073/pnas.73.2.505" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=X. Shen;citation_author=M. Gorovsky;citation_title=Linker histone H1 regulates specific gene expression but not global transcription in vivo.;citation_pages=475-483;citation_volume=86;citation_year=1996;citation_issue=3;citation_pmid=8756729;citation_doi=10.1016/S0092-8674(00)80120-8" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=X. Shen;citation_author=L. Yu;citation_author=J. W. Weir;citation_author=M. a. Gorovsky;citation_title=Linker histones are not essential and affect chromatin condensation in vivo.;citation_pages=47-56;citation_volume=82;citation_year=1995;citation_issue=1;citation_pmid=7606784;citation_doi=10.1016/0092-8674(95)90051-9" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=A. Stützer;citation_author=S. Liokatis;citation_author=A. Kiesel;citation_author=D. Schwarzer;citation_author=R. Sprangers;citation_title=Modulations of DNA contacts by linker histones and post-translational modifications determine the mobility and modifiability of nucleosomal H3 tails.;citation_pages=247-259;citation_volume=61;citation_year=2016;citation_pmid=26778125;citation_doi=10.1016/j.molcel.2015.12.015" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=H. Tamaru;citation_author=E. U. Selker;citation_title=A histone H3 methyltransferase controls DNA methylation in Neurospora crassa.;citation_pages=277-283;citation_volume=414;citation_year=2001;citation_issue=6861;citation_pmid=11713521;citation_doi=10.1038/35104508" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Teytelman;citation_author=D. M. Thurtle;citation_author=J. Rine;citation_author=A. van Oudenaarden;citation_title=Highly expressed loci are vulnerable to misleading chip localization of multiple unrelated proteins.;citation_volume=110;citation_year=2013;citation_pmid=24173036;citation_doi=10.1073/pnas.1316064110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=Tophat: discovering splice junctions with RNA-seq.;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=C. Trapnell;citation_author=A. Roberts;citation_author=L. Goff;citation_author=G. Pertea;citation_author=D. Kim;citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.;citation_pages=562-578;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22383036;citation_doi=10.1038/nprot.2012.016" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=M. A. Urich;citation_author=J. R. Nery;citation_author=R. Lister;citation_author=R. J. Schmitz;citation_author=J. R. Ecker;citation_title=Methylc-seq library preparation for base-resolution whole-genome bisulfite sequencing.;citation_pages=475-483;citation_volume=10;citation_year=2015;citation_pmid=25692984;citation_doi=10.1038/nprot.2014.114" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Y. Wang;citation_author=K. M. Smith;citation_author=J. W. Taylor;citation_author=M. Freitag;citation_author=J. E. Stajich;citation_title=Endogenous small RNA mediates meiotic silencing of a novel DNA transposon.;citation_volume=5;citation_year=2015;citation_pmid=26109355;citation_doi=10.1534/g3.115.017921" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=J. P. Whitlock;citation_author=R. T. Simpson;citation_title=Removal of histone H1 exposes a fifty base pair DNA segment between nucleosomes.;citation_pages=3307-3314;citation_volume=15;citation_year=1976;citation_pmid=952859;citation_doi=10.1021/bi00660a022" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. T. Wierzbicki;citation_author=A. Jerzmanowski;citation_title=Suppression of histone H1 genes in Arabidopsis results in heritable developmental defects and stochastic changes in DNA methylation.;citation_volume=169;citation_year=2005;citation_pmid=15489532;citation_doi=10.1534/genetics.104.031997" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S.-M. Yang;citation_author=B. J. Kim;citation_author=L. Norwood Toro;citation_author=A. I. Skoultchi;citation_title=H1 linker histone promotes epigenetic silencing by regulating both DNA methylation and histone H3 methylation.;citation_volume=110;citation_year=2013;citation_pmid=23302691;citation_doi=10.1073/pnas.1213266110" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=A. Zemach;citation_author=M. Y. Kim;citation_author=P.-H. Hsieh;citation_author=D. Coleman-Derr;citation_author=L. Eshed-Williams;citation_title=The Arabidopsis nucleosome remodeler DDM1 allows DNA methyltransferases to access H1-containing heterochromatin.;citation_pages=193-205;citation_volume=153;citation_year=2013;citation_pmid=23540698;citation_doi=10.1016/j.cell.2013.02.033" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Zhang;citation_author=M. Cooke;citation_author=S. Panjwani;citation_author=K. Cao;citation_author=B. Krauth;citation_title=Histone h1 depletion impairs embryonic stem cell differentiation.;citation_pages=e1002691-e1002691;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22589736;citation_doi=10.1371/journal.pgen.1002691" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Zhang;citation_author=Z. Liu;citation_author=M. Medrzycki;citation_author=K. Cao;citation_author=Y. Fan;citation_title=Reduction of Hox gene expression by histone H1 depletion.;citation_pages=e38829-e38829;citation_volume=7;citation_year=2012;citation_issue=6;citation_pmid=22701719;citation_doi=10.1371/journal.pone.0038829" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=Y. Zheng;citation_author=S. John;citation_author=J. J. Pesavento;citation_author=J. R. Schultz-Norton;citation_author=R. L. Schiltz;citation_title=Histone H1 phosphorylation is associated with transcription by RNA polymerases I and II.;citation_volume=189;citation_year=2010;citation_pmid=20439994;citation_doi=10.1083/jcb.201001148" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Histone H1 Limits DNA Methylation in Neurospora crassa" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="twitter:description" content="Histone H1 variants, known as linker histones, are essential chromatin components in higher eukaryotes, yet compared to the core histones relatively little is known about their in vivo functions. The filamentous fungus Neurospora crassa encodes a single H1 protein that is not essential for viability. To investigate the role of N. crassa H1, we constructed a functional FLAG-tagged H1 fusion protein and performed genomic and molecular analyses. Cell fractionation experiments showed that H1-3XFLAG is a chromatin binding protein. Chromatin-immunoprecipitation combined with sequencing (ChIP-seq) revealed that H1-3XFLAG is globally enriched throughout the genome with a subtle preference for promoters of expressed genes. In mammals, the stoichiometry of H1 impacts nucleosome repeat length. To determine if H1 impacts nucleosome occupancy or nucleosome positioning in N. crassa , we performed micrococcal nuclease digestion in the wild-type and the ![Formula][1]&lt;/img&gt; hH1 strain followed by sequencing (MNase-seq). Deletion of hH1 did not significantly impact nucleosome positioning or nucleosome occupancy. Analysis of DNA methylation by whole-genome bisulfite sequencing (MethylC-seq) revealed a modest but global increase in DNA methylation in the ![Formula][2]&lt;/img&gt; hH1 mutant. Together, these data suggest that H1 acts as a nonspecific chromatin binding protein that can limit accessibility of the DNA methylation machinery in N. crassa .

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif" />
<meta name="og-title" property="og:title" content="Histone H1 Limits DNA Methylation in Neurospora crassa" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/7/1879" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Histone H1 variants, known as linker histones, are essential chromatin components in higher eukaryotes, yet compared to the core histones relatively little is known about their in vivo functions. The filamentous fungus Neurospora crassa encodes a single H1 protein that is not essential for viability. To investigate the role of N. crassa H1, we constructed a functional FLAG-tagged H1 fusion protein and performed genomic and molecular analyses. Cell fractionation experiments showed that H1-3XFLAG is a chromatin binding protein. Chromatin-immunoprecipitation combined with sequencing (ChIP-seq) revealed that H1-3XFLAG is globally enriched throughout the genome with a subtle preference for promoters of expressed genes. In mammals, the stoichiometry of H1 impacts nucleosome repeat length. To determine if H1 impacts nucleosome occupancy or nucleosome positioning in N. crassa , we performed micrococcal nuclease digestion in the wild-type and the ![Formula][1]&lt;/img&gt; hH1 strain followed by sequencing (MNase-seq). Deletion of hH1 did not significantly impact nucleosome positioning or nucleosome occupancy. Analysis of DNA methylation by whole-genome bisulfite sequencing (MethylC-seq) revealed a modest but global increase in DNA methylation in the ![Formula][2]&lt;/img&gt; hH1 mutant. Together, these data suggest that H1 acts as a nonspecific chromatin binding protein that can limit accessibility of the DNA methylation machinery in N. crassa .

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
    <title>Histone H1 Limits DNA Methylation in Neurospora crassa | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"sTkl1bAHHeTl7m7pQCx0RgSDhHfkwA4wcQWZnVb0x4A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460436","apath":"\/ggg\/6\/7\/1879.atom","pisa":"ggg;6\/7\/1879","ac":{"\/ggg\/6\/7\/1879.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/7\/1879.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1879"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1879"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/7\/1879.long":true,"\/content\/6\/7\/1879":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qAhfc1+9DA+BQwRp8qykHj9pXhh0\\\\\\\/JpSw14M5JZuDpffbGgoFdk8DWLB7KxD3OhHGx0+\\\\\\\/nHD\\\\\\\/Bcopmq9JRW9EOtlCE18r1BiOrrnTlm4qUjkGDo4JDttdi8wiycbpoKqP3+HymQbxMf+D\\\\\\\/U8p9fRiSE0fG6+NUdtI6pPrhE20QnyZ0PZuX8LOYkQQxkuwKDlM5idGoDWOLOp2hQK37LwS5SujcyAdbQqdKcvgNjQNEUlUDNct2DWG2bQaXkGl2j+KjvijUF4Bf1smka6ONDVetgIf+g6lHTLcewFYDw0mVh1GUVAntUf06+Gt+wT9pJkK27LhCUGBw1uZ3150gYzjoRuN2wLKl2ohzunNQhRQfPZWijefZkzFvWY8uVxuMio8qTYI\\\\\\\/9b3e56Onmi\\\\\\\/aEgOvx6CLUclI+OPGhEfBjNhYAjeTEUIM9oHOTu6o3zXi1bga70rvQvZW6qoQhsTXMotu35xIKt1I1\\\\\\\/OnA5HyXDAg2sHRNFprV3cdbs6JUBNVpj3xjW219Mrhv+RWNw4XjMBmyLNQAnEUkB2AJ8msaMN4dnh9PsPwHBCFGsMtha2BG9ocfywIZxKHvQBU4sW6v+juxEamBJjcB5FpPUPF8p2w1MnP2wuQ77OsegnEHfi0\\\\\\\/hSjtgwo6VXSEhf5tMMKkSBTosOSDXt4sXgpbcKhPiiz\\\\\\\/osviSqcepiptFBOJG7VMP\\\\\\\/FY5JwAX1S\\\\\\\/+fUNe\\\\\\\/5K0eBlZQ6wtpjivVHPChxZ2\\\\\\\/2rRBXGBLwjsw4bD18GSWVkgb7Ss1Fdp5rn0jtb7AoTupGJWXsKt70\\\\\\\/Y1KNrPIJtbFcahmJMWSXklfDPAlQxiCtjijdBG38a3P4o\\\\\\\/jBfxTjpNGPZOUGVcHoJvFDYr3z1ngWJ8rjhWrVTnL+tJ6siMZRUaShK8LGLrxsbnr264F\\\\\\\/2lFNE7e1q495EKG3SHxBkTAPwkUCtUqt4Djne3n+wKlqdIlbOe6U7vpuw9j8+Ogmt02pjhZ3uLEDHd0DfXJB6hz2as8HDHYPw\\\\\\\/B33SZeASVlx8TRH4W1Ve5mgTGgCnW4BLEqqn0gtmsNSAfD\\\\\\\/EBAzgyI+9hIR101NMLVflp9HN9cevkEMyNtIrrNN8iE7nSW1bouW2b5jIp86ikdWlxHA+kRkI6hiujHWS9eBGaW18sB3lDANusTmJcp5vyhqFfUIcYZdqEu0g9PjDgVw6thif5LUspaliLFfg1HOvEIeRtxWmAZqiwsU8XRbjiMt0UGMiTVsPI0Awcqej26ZH6VX6\\\\\\\/nYxOdK6Jk9VMgTi4mLokbn\\\u0022,\\\u0022iv\\\u0022:\\\u00228WgUvdKlsiAXl4ir825RcQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002246797fa1d87f7de11cc1c0e3895c85585e377825ca69b86450c57005b155b94d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cln3u4lC0H1gb5Ujn1f7sFA2tscwUYgiT\\\\\\\/67Qo20ukT77+Ww4fnS5rlRKVfOOBb9fDURhuqgWjDQUJvQzwekVPpskbuuJt8tVxqysSM1SyNB3PI7H+Wvj3SG3Jm4cfaufb8Hv6qcpUiHkcFNGjp2GxDnsj2\\\\\\\/CU6MlKEIPYCUB\\\\\\\/GnxTO1kDPnSODFm\\\\\\\/CmzbIvob8TxNRYyJaiADt2HfDt3Aww2xrqdiL2IcLCkqv8uLZxfNiZfZK4VtyxYai2K7cbJ7pHZS+QvVDX6soCtsFS2HJpWOT7XCjuYelZd7CvNlYwW0+vu08OZrHlRPrkmQKHqANpkKl7DqiH4m+kmfrCP2w96CkvPq1xsJvfNvOPhpQ1vuvHPig7exd4Btl5lcesc+wNYootYmM686jbz\\\\\\\/ehOLakswLWgzE2TxobWymd7RqM7XapmeyYe31IMbIAbVLObEOpS7uH9YHuTLeMRaasGUvvI4R6PvTeBWbl3GMxrw04t4ZdpSxwXt\\\\\\\/AGn1B\\\\\\\/+higakBhSasAEqPnyddz1Ni9ZrR2xOUlnkXG12D5zcKffMZItE1ERZ1s6O93pQFgWgpFMTT4KKasg80PKYacQJnaXb26sb5+\\\\\\\/YYrR41AE1DTIbviSRCSbwl9Iz9D\\\\\\\/+0+UOlYKNcdKjwlbY0d2ca7uBt2QIjQimllwKN\\\\\\\/sFsjPDN99rJtUHK36NUrAzBZ\\\\\\\/CqOSoHtU7sq7FqReRYamT0YsaCwHzkkIraLvWp8J1pJR7zTbDGLllbykOrkQr5zdJWFn7lxRuFAQ9Fr258BbukZC430ESulEMuljkq1bU\\\\\\\/kK5Ks2HqNjth6QSgeAgmmRxseh5sMSNkOPShdxTS07Iuh\\\\\\\/rZ8WllLGgyIGacn3htfRuLxINI4mSVqpkV5TeXBsm1stc8z9sCZuBmyWnOazcFiLOnDI5OaQJE\\\\\\\/REiSMRTVi16RmG2JcXQNH5\\\\\\\/f+3Z6XJkjk0pDnA6aio8V0ggNXDI8ojuD4UolS9cbizEqUzLO3pRPW7gezTIXiXBoguQqFNgGwXmaNe77VFwi4y0pcxjwvfNydXp7qLmYOiUo7gTsMBrb\\\\\\\/KtGJmnQm6EEkGY2HTXgzOclrs\\\\\\\/KXdaE8Z1PYUjfZ8GE6O9XzQxD7wJZGyM9uxMjYZowQtJ548gv3+0Xgom\\\\\\\/JxejG\\\\\\\/SSLM2V93z7eRdzvqL3QYU6dwvG78xpCjSi\\\\\\\/SDaXFuqSC4QZMmjPAPnXotIY6QVJi6GlMd7Az9Ef23qzKihpGVAtK1gdxfzJ9WcwyljgUYv821eV4UFfW\\\\\\\/2znjaYjQxltzYKJOtdcIs0UPIRTLEVXxOgoNwOGXYwMfd6QRfm3MMQf88JkbQi2DyCoQwA\\\\\\\/P4GMX8HkgxW969GEVRSvK5\\\\\\\/xbT8+YHtNxMDSnmD3ezodHldMuBCjzruGzDhf7v9i+o2In2gPM6H5EZCiLb5mpORxyxLecS2pDna3DazsUXXIz5LA3RoR4f\\\\\\\/otP0LwuDimwOghBGPzSGH\\\\\\\/EnhCJNB3f\\\\\\\/yb0gQj4vgUAMuvyJQzzDeq2WsfNLzkG0MQXKMCBIZOtpKK0X4LZbaVJg65uOzF6x1UZeUSgBHdz09NBkSAuIyzQZVdDlQGzPIT4\\\\\\\/TdSuZkBhenSq02agfZ0XumTUID\\\\\\\/i3HMvDGplleqTCPRUI7zVIIU21e4hB6o866RcvYdZSEOYGdI8QHwy2zeg+jvxong3y\\\\\\\/XXkgWRq8o2CfJB9byU3iHkPP2aazebc\\\\\\\/yXUP\\\\\\\/R5\\\\\\\/MHK\\\\\\\/eKO9UglqPUbKFLrx5PvuYvrKmw5p7DDJnIbp9rhYtbrm4tZDFyegvBS8+QEOGjA0DbNAvljhqxRksC+ZgcCFa5Nsn3wQk\\\\\\\/rQNxCUm+XC92tGgAo8F\\\\\\\/YUV2A2bupqwGFze00dnVLvaDNYWweSH\\\\\\\/7bxzBDFjns33KXJQWyNh2sqZG\\\\\\\/fCKz91GJFd5akcojJSWstgsiThDEEApCU00miITpyfjAG94lyNAIFQlomrh2x3YmFxfJWjjZIAVsQG18XhqaNb0hapfGQpYKNF1Ton+VqcYdl8nT2RAbRbLvlnK7gzXuEa2\\\\\\\/ksENUvnlFJKn8jNJAc9Pmmx0tyRjH8Q7ZUu4VfjGqI+0h5ppj++oo05nzsic2OR0bwy1uLaOni3iWkNJgJJkgoeBH+efmI3iPRS3++QyJaoLfpy3ZVywXtcs5mRM\\\\\\\/iqWZoQtDrTqhTwzKt+W7eTYxcSMvukwIvsnz6v7UGxm4rqAhMzFn3pDoqq1i1vtOiPkytwt2pdtDogEQi\\\\\\\/DGScsVuWDx4NuXg8rx8R+HuEdVKczK9sFMVG2Yp\\\\\\\/An791BeAbBHpNjqCFCcxcSyBU5i2Z1DLtYd\\\\\\\/V1lBRmcjn35cEYlP46wWY7RZUItlohVUGFscr5VdaVH2Q33XItE6xB3uM9S\\\\\\\/Ba6hjcpYkDaPD\\\\\\\/dOI\\\\\\\/U+fGt+kn5goIMzGnfv3OMr6856GEGt9vBns7f9d0rglTsxQ9rPFW3WOBcetTYi+cKJB2Mq1VSTaSgP6H+Q7mxaR7QOrgd3KMx3HmKBH8TPyvvEsfaQ44IgL38nZSUKvRs4DXplZDVfjRzAAj3NJY2ZA\\\\\\\/djv4CCSMqnFw4ZdAVhRnZu8Cr+nwKUIOE2Q8bZTIRked351njw\\\\\\\/SHBRUbl5BSKQkW9LEPN9Xk+iq4YCQqCWprulw6GD5nOQTNgud8q5WrZL4tOcrDVakKRxMAi65A3dx\\\\\\\/rc2IFmOU9Zh6tZI6lpvMtu6i4cGUzOByuJ2FgIluqYyiEvxyc8hYd5wBwGvrb3vqgK6NPNNhjbc+6FpwyXaRmt4mhwnVxPG9MtYOn1eNX5gKpCXMXPoLM\\\\\\\/DKvbzPcoI4MU53jjLxHUJ43WpwshSFergNkzEU6W1JjGyC2DJ+eW84aKWp92hgkxB\\\\\\\/zZjGZRIn+Zj4H0KqML8MW52FfWkGkaMwyFiQGQEsvC24H1YjlzwQQLraY409lvwX0+g5rIK8aC2kZcmCx2LLbIzOY9ZLg5zEifwfao9LsFz3YgUmk5kHb6Ux+xF8PqeuGKvXJLrhGKoLRQHql45213WY6usW9PrT+Irtq413+cucE+M70VLIulvWz\\\\\\\/5rlexKFvQGBIxavOWs06WeZmQ5oYir\\\\\\\/WQyBYUvMJmiRvshdeRl1JOw02C+zdtTQvW9HqPNGPTDgzDyFlC4TgwcOFTOjGH9k+evrFYbwrPbfSd2pdAzRKlLmL6DR+DdEhiKUNYM2STgKhxzWKYtWFUGVpeLAdBpWQw3t+0epc90RxIUk37jVmi2DuZj2DQKQ\\\\\\\/ZohSm8hfTO\\\\\\\/a5fFn8zsaYLPmhABMPIXUxCGul85jwSJqVWS\\\\\\\/Q5UJQUHt26nWXM1sWijdMQjocVYARMA7fkY0FTADxw7Okx95XaHwmdvYW7qHv\\\\\\\/Rv8hrU1eO\\\\\\\/fzGe3EgK+LIG2OUwSGs9ewzfcHUOwboHmjJA8bwIHsMYsQ68X15IcmIZ2HNygUe15azodL072s+4XVrFqwCt426J6Cj5pHjAu2nr0jDX7Dt43GUIz09Lc6MVfONgN8bI6UODi0ZSjCP9DrBDzOm1vX9qX60VL35nkThACNoL4DehgKr\\\\\\\/hZvbP3z7GwDGB+qRRjkbd0wUPJ2Ffm75Z0Wu8cuWKsHqcnyg5O4BORYEtzM02tSxFedRl3dRHDogmDvHB8HcLbsZ98VbHcyCJxio\\\\\\\/QVjJU0RIXeLrymRboJRVKnKnNZw8KwKwTdj\\\\\\\/RSZd8hd7atBXtWNs0K9VqcXhMXIngeI7GWC6C+4x\\\\\\\/KplemFjebGa\\\\\\\/W6U7qOB\\\\\\\/yFNmC72WBAL0wsmD0a9Cv8EiMlrH9XFlqcKHv3WurV7ShdenAeMv6rfYQI5+w10LkRQ5NLYiTy65vdUTScAArLQWnKd6wT8cDhyI4w7kmwPlNNlAFCViazJSsTC2MRmN0yc6L+ZkDePBsUhhaHC7NTkCD6D115\\\\\\\/IBhz4V8x9md3aTjcUtqUX7Kbhqm2YWSUzMCVLF\\\\\\\/PjAhxjofoEMdbAuA\\\\\\\/cZ+I2yCfpwR7j5pr7\\\\\\\/Ms14vPqN846IhvWU\\\\\\\/fU\\\\\\\/HmL5qtONcu6KWCNetc54xca4Pi17Sq7LxzcErxXlzVARfTnpnGqTunSJgA6EzVHAher4GN9r7x4mMLA3iRKBtXv9RkKPdi9vaJsOboHjjugfdkHDewvLy6yTohDHt69OEIbjAQjb0Rt97SC2EereooTtY2OJy9VX3N1LrMMPDVlELfpuRTDpizoFjeoI9CHqofPR2QCSXCZmZMGlJYjFHFmEKUgUIjbT0xsVoJjJp4W6tJDR45tflhBF+x39R+hLM\\\\\\\/FWO2+6c29Oa7j8U2W77C04YO3E0QcV0XGv6gw3EewWKMFUNjR8kMyN1XPmnVw0n4as\\\\\\\/wulryhv7rKmX9dk4Jrm9VTaEuIacg3+6V2HLX7zX3NKrZ2XkcMWvAnlxlEFH2o5p4LaVm24iACsnehCt8OKyRaMxBBrOqITkxpnpjg2zanxbBUbbgs2YgT7\\\\\\\/\\\\\\\/V35njLsH3QWeZkMgJ1Q1CY+Wi1KV8lkwEifSmUC5VtOUDNv79zrSbGkVNr3YGTjIQPMhVHBLZX24am3WCaTZv\\\\\\\/M++EpE\\\\\\\/wBZyhMpOymsHRhxEGZaPNis48o+7YAvxkVxip1En7xjXXRWF1mlXHG+LYnw1DaOrqU9Prr+I2fs0Z7SeNfiY96\\\\\\\/LipUiycmRNKp16FtDRLyjQNjppPvBHPNOuzLTUWi1C\\\\\\\/AtlOo8oPTjB2ml25nPZTNm8NL1smj+ScbulQTDQucbaYhfBqWDeondCiBmHVZXWvoJJX2ls3BxBBXzsgLK3PxH0nnEg0MY3LXrue7yvuElABHE1j5uPvwcCl8WRodpxJ\\\\\\\/6B9936P83c3hE82n2TosLT8w7L8ZR09uCX5p1fPY++3Hqo2V5QwRGedB1i892EpLegg73S1TDbbIjjTUtS+v4FUgF2Re+0zz7kj\\\\\\\/TmANDVLMdOE6Btfg6w4od97nPnoItRFieCAo4JAwHs5wARdikKQttYD01Vov8M4myWDtAOiOYlCIyPzKsukBvxqGfddQgoXmPMG4dcSNN\\\\\\\/AAeBtyxiSgCunbBG9uGf0ltpgbzlVyEK\\\\\\\/h1QqCyGz9aDJ5HJ1j+xUVQ3tDaBtEcVrMJGYaEcZM9ArO\\\\\\\/BVYKa7tOykOaq74CzT7Q06LKWFPbqDaerLAtUkxmCCeMy45v4J\\\\\\\/Y0VwWAoY7Vq7yoGY4im1QquU\\\\\\\/IwvCEtZ3kZzBU4\\\\\\\/8aKydDuPQ\\\\\\\/Ddb3kR05fCGiO3l9GV5XR9srf6wTbAn8WlJ8Vj9L3IkR11kE6G\\\\\\\/CZCfa9munqRkS5JdQzI+44i1agAQtD0n1q948\\\\\\\/pBIRDvXJjTcR\\\\\\\/bpaLLW4RMh+b6jgD2WNEFJ\\\\\\\/jVyliyAJAyockRmf3xa0EHuHNCP\\\\\\\/0ikjnqIZFl\\\\\\\/Kr09PGMtDdWao1Llnzqp6qSWceL1X2Hrq3O0v5Xxmiv5ce2vy97U3t8lrIvEj3\\\\\\\/12tzNOIjH3NimNPyUlY317BbAbxEGwiqX8ZBxD1ccoCcyF3IgebXxckRAtx6TXOLg\\\\\\\/drvtYyJkU3ed+eUKx42G6PBE8HIAhANe2f6VqD70byeHwqVGAGazFyP0lzDxJ02mVHxnN7czk47DID9n732l5NKP2UvLbJU2PNaYHrVfHfRBO7ANKoQV0WbOdb4mGb+sWwgYxVwCM0bpiXuwDsWgZuX0yjP9HSjMclqXn0Vt82qAFFxEzfHMS2P\\\\\\\/feNJqIEU4jtRV801D9MDA15U2eeX+XBf4sSodOKrd9kV+ZEHPg1dsUVlYsdtWNHtCjf9e0uMwWavQXaX6PM9boYuHyQMr1Sc6DOp5BKdo283D8pbZHE+H0\\\\\\\/bDa5\\\\\\\/egZUIWejM8V0VTgsC5kqbTZ4qMoRrFwzd+NRGgm1sEjOcyS\\\\\\\/W2jO4Rwda2UigWZCWNR+uUjaR\\\\\\\/xkp3xCGXSfFfB4Ks0Guil9elS3lwgfr1HQ5STpGS42HHJ5F8pR5NG9GsO83GPizFSPlSNJ2I\\\\\\\/kIid5BGaALYIngWlLlsQBwzzSf93oQf7zsewOfVJCH1D2wackp0YYbomYkZoMNP5i8nXWk6NyGbjIgLpco5CV1KSo2koIG7DRFN9efTnWnV8RQKcSX\\\\\\\/kBMDx+DCmLA59TMxxyxT9JtiHReO7rV8IE9EvUuoR8Bg75Yz1UIVICaDzCcGmblk\\\\\\\/8lZH8M4I\\\\\\\/lkInshnBdxdnQPpaVtqJftQ\\\\\\\/nNX6T22UIZ0K837zk4KR3gziWKNhPHGTuZInirhJO7QiSTA0uLkj3PDYrlrYj0fjr9ZHR7Vw8II68R03unRQRs9fNyZsQ2pB2rBRj5sBfHXORbNc8BUVWjIum8ZltnPR8\\\\\\\/3jxy08AiamHhJkkOvHNnJk+\\\\\\\/i7+ZtJfP4Z+2+I1hM07k3yK+G4FG0DxDfVtQulG1Gh4ZpJUtEsQ6wSViN9sTwiEVrqBS5nWaMdFetPurnQAgHtPhTYRR2gET2\\\\\\\/6Yom6+KwD58AUq\\\\\\\/YpiLp5scMo913v6TL2Uid+iPpBvXt+RpBXXPRM9UkkZ2nLjnrnTlAKqvCc6vm5TLBEpQ5nv8T03bk3DchPdv5pSOzCAPe8SJ8rAK2iPZ5c7NJNk6RJ3D4yIAvgKZqADNdPTVNT21EByAGActwRmdbEU1+9EP5n2zZaueoD3aZrjX0qz4IjimhfP4lTAvkupukN1CX+iea8F++JxW7L2dnsAqMFPfzQzv2k5KSVZuV8o8DIHmsLNUJ93jUsvm0FgO0nbuLj7lLFRPwCR\\\\\\\/hgXpFaf+zedfzHrJTJky7c7An5ntr5\\\\\\\/hHs4aN6ZPEO5GDiBrHGHDWB7CEKT\\\\\\\/zOWGlHUVMxOkKYA6VT7xe8+Uqb4+L9bWyZAQLU4u+GpD+ElsJiKWjkAdjlJDwTNIN3OTgG9pWRaxsHnFkvo7tsBOnlB6ZlOBzgr38xS3IANiy5aYxvwfRGDoEXBZASjPF9FWYqoGQF4\\\\\\\/8ZUV78nk01LhMyPfqVe7L7cUgREE6\\\\\\\/acwcAp\\\\\\\/r93XhITJ1DvscoW\\\\\\\/AmwzNlSHC+CvoAMbpjTyoW7Sa39eZfaQ3uLfkTYpjQC9qYQH7IdOAsuIYUMU2Mn7GzRnh\\\\\\\/VJk1eGiRNb6\\\\\\\/XvRrYNk4T4ry8YwSP6be9v94CZd5bqwo34Kgp8Crz9Rs5GaAJg9Ys+sJ4O2GFOf34eV5F6ZDGWRhEjarEWflEgt1CbLahbYmXnUH+sLw+yLrr9UPaO80CPX3YKaLNjlR5eOGI3FujGnSbSUyEHmi8Z5+55YA+97xM5mjS214ZdA1Ck3vD9m8eTFzNC6fiLD\\\\\\\/cJesmohFxehbXMPpKR\\\\\\\/wj2pgZm18pPofqa00usf8DF9GEl+Y58MautsLFZg9XspPoNXy6gp2FWt7KIJ0l9TQbppaB3kXUcX59IovXECNm9s3EMOMMFWyOwtVvpjNs0c1wustw\\\\\\\/I0u4SN8NDYVgha8wsJlUoNh0CqailDDq39Up7vq7j8wQnvBgIWtjBJFCqBReEsod31a0p8fqs1n97Qo8H9wmZH95bcGA\\\\\\\/Q0g3tmOaxmLF8GTc6llA7zlI78PwNGEL5QY7ts6Tesxs4aVO4dMYFIjhvrXgfHMioRtZBLMCK0Dc7rF2n\\\\\\\/jU0n\\\\\\\/80CZXdkRmaUhJE8KaRkdQM5DSDf4ffUNN1BeIK7nP373PP+5vRoZ9Na0zdfmn3R3Czx2OxoQHJhEVxunDPCkgQzZicDUHzNGNE474ibFDZjDVulaBgFmicHYW7bCf\\\\\\\/OYQoQCT8T83o+enyjywTdQ7KDadvMKVKpUyNKTJaO+A9uKOF9W1qEEuY5\\\\\\\/TT69imH8kWb0GFfHe3Fy5Pz0+EByjYmNSR0TumzdRPvpVV+GXimCtFaDXuhqkyWzI3gaEvAz6cuMpXYl+M52hkWbeqLrhGkNnMeciH2\\\\\\\/8TishPOKO3orj79yvN33+QTaHlqtCq6H9Juhx6SYpIUylTGoxwRFNxqzaS\\\\\\\/7S8aHR10cidUwpr0SaZ\\\\\\\/bshKLU6fR6Gspzq7dDhIVOcX9jjKmQKszkGUHplniZ5fkHNkeVu1JCKO0\\\\\\\/c2iEQq0\\\\\\\/mcFHNVQodEQlHsjxMh5oIZZfFE+UyYeG1lwdKAwSeutfieSuX\\\\\\\/GfWnsBuRLs6j+Oui8JIXXgcy3DV1jLdXK33YoGtjjj0H0WxEhCOhEBAdEelCofCFQ+5dVtCUimKj8trO66oGoLXVvsHRml1ULmQjZKhfijN\\\\\\\/AXA9FnGQ6qC6KZUeDIgYBvUCSC1UIOFr9bW8YY+kmHXcGc0jHTv6ItIVzCZcDP+2TkIXbcmd7M7TMpY0UALDUjhaoPuZtCw2cp+8PV8mPfeBiosJ0spMUGETQI+m\\\\\\\/NYTJZK6OgTZMbif0WljZhpzQNWXFR+dQRTXzY\\\\\\\/94syeCZ9C5xsmGGnwaKU9mwgQiQS9sU1l9U1QYu\\\\\\\/gilq4jqmpCVuzHLo8nOCDzt7OVrwEg2w2Q31Cq99hO4bmQeIfzOBcnxqM7nAUhamimYlJ6AwLG8OuLk31Oo+Hy7suvV+B8fd7U0HHe2\\\\\\\/p6pOWh298ZA+UFyrgpIkQuTR4OgYC9H5+z2GSC61mkKxCwx+l2k7JQq4yvcHOm8bkAJVJaN58aMSqbiycu70m+I4xZsVaDa86cyovBprEk2BFXWJBsc8xcIHTONw+b5+lX\\\\\\\/OHzDN9IcP59IeH6x68df9ighlmpNtFbNMAr69U2sQqpyrglBGK5mYRuFmPXMeiiDvpfaTIQpOHac+TtqV6XvS+eUYYXlr\\\\\\\/K2lTg613vvb8\\\\\\\/D+X6Gwj635abFLg5+awqB161Y6PVE2chCEXiJqGVhYLSs75DKS1ggIrjZMfrQ1d0lXgszCn4miWqweBVzQGEUOhfdtLDdST9FfqSox6042XuI1zwMXhaSsHrWKudtq5PXrDWcDFkLMq+QhTVNPjMVuXJUTYeOwE+RPXQZDkiy18ZGnBcoFTpEWnN17j3U6KDTPqCWnG2Msxfef+N4MeBEe+PmHeBhHb\\\\\\\/wlQVpkSK9\\\\\\\/ATlUhnMJKm5YgGaDXgZLtnVKzNWXtXjB\\\\\\\/tUdlm8rFvjQrmgucmRHVsHL2K0EQSPczJ7L5HL+\\\\\\\/j8wi\\\\\\\/kKHRhL8iCDdXUHFrcsALjUnaJQ6FeTtahkxsre5ePwj2ho2nYNrhRccDgkdpiu7PbfnCcV82WcXhmSsiAZoQXg72sos7fE\\\\\\\/5rG0JO6PKqWxTvsHuxc6p247TOLppHFSjiRi5qsLKCCOECkMrKeFOvRd+uH6bcO3AtOL4c8qeYGPH5vnNzXazAFytoWVhJ8wXvzcWQuDWD2JNjswJhrfU+2MUfPcuNNDDmT1vNDXl0dvVZExRtG82FpXEvSZAcuhQnVqsFhPO6cah7rGfKxiUqFzjLIyFsWOn3OZbyUL8TOUlRKpagmloFnY7Lc0UWG8+B\\\\\\\/k3+2Kpa0G4eG2x2O\\\\\\\/l96ae+c6F+3Jwd\\\\\\\/j7OOJxXRIe3QHLsCu2qg0d7uTq5Dmqwm3hgNPmsbARj\\\\\\\/fk+onvjZyRDsHR7Lu1OFKxZztUeloj4c7z9mNAaIn191Bxpqsp1GnqDjFYn7YoSz6\\\\\\\/kuXJh6jZnQ0syI955okiYBMUXbpcMEUjekAqGqHZscHjvb2IWYEJxynXOCU28EFa0MMdnVsxWK44JOflgY07TXHxrn8Q4u6lTam9YD6Fnee7OF73\\\\\\\/Uyn+8lCgVjqTXF5opOJV\\\\\\\/w9ff5g5B00Z1WO3I+7Hhj7ia54foU\\\\\\\/nPZgFR7lXe\\\\\\\/6XKP4I3T3YEH94b9pgPxxUmhzkqtKSZA32nMdoADlDErBnBa66BEJwHzO9xAYHiZnZdLJE6NQfOMdvIqtuOSLpoES4o5iLdVP+d4+pa8TQjqxKlYRUkyNgFBzuDtnRFK3XC0YrJD7VEcGan7kU6dmxJG6NGMEuI5ZMlJaQQmLFLitci8FA7RELKuEb0TE9RELo6tipCoo8jUJga2j3X6dlVahTP1Vi8Bj4EWJA\\\\\\\/SQMvahcjuBK68jGDZ62V57w+DMlR4NNrn\\\\\\\/rdPiJbNLLZVRYZMmcVxiKl40xZtquN\\\\\\\/Yzp6UrgkCXJVZhIhvIkFBWYTkXDEmkKY53BuY2HRBqrVllcvO8aQJcm1VD3EZCId57Qfbc\\\\\\\/z\\\\\\\/E99cG+wgaF2spF0K+xJ94qumWVvChlboES7iPzh9UeqW087yySLdOZ3olP3sgOy1or8w+JjlTesFGY6ZwF1697A+tzIo8AdEW2KfrUK8WinVfm33wD2rlYa1ChfyDgK6YauG+2E9wS3mKWRGFbtpsMwk9kp4AjgVGg7djkvMaRXXhz7zPQw2I8h\\\\\\\/DG0rSNXhATq3UrkeFo9Qr\\\\\\\/fhpMCgu9tzsh+qL8t9w+TKdAaN\\\\\\\/1maCwbuxHtjpqI4LffWGLsOz9dVMMlehzYdxPqhsn9AkFI8wykgC1DCo9WUP9103T2sx9TKqOUPfAICU41gttqpKGT\\\\\\\/w6uln3f5bOp4PkDiYc2ekncVQ+C4fiCWPfHMxemjh5OlQX+HjkSJcJKmCQLquzydPaLjwZwnYuKm0Z7YgHCYr6OwsS60l0qloYrYOM8eV5ohK5NcMH7ZUg29ulhYCs2b+YDQpZEzcbtrhqSIZ+m3TkOhA4CvU0e5HtHz\\\\\\\/gAmjGxqQI3LRh5cMKuxmB25zKryMYkAeRDyAtETBGDNFrwdw+hM2Lsjtm86aCoR8JXJw+5N2qHxI\\\\\\\/XDU8DFTavO2IgH56qpPabaVRpYFl2QmeLocZ1DHUFVmsBmY0H1tUdl5MacbVjWbp7Zg25lTkKjT4Wo0Nbd4\\\\\\\/WzTr9xhwIrY5mXMDXxSBfXVckyvFWoGnftbvTrqaEAnhAd8gFXKXRpK0C1IAM\\\\\\\/YmHSUBmGBZi0LFTiXOk\\\\\\\/vy4XodE6Hbu+LpkcRN+61gBxwU++Fi7LaIam0HuKdrzBG8pWbDUPFcl+r3f5S0Q64g9atHYEuPOiImWi2qbA9gKeTPoD4khb6HAqbbWPNE2hiBNX015FYxhdfzgL\\\\\\\/XVcXE1bijeZzRd942Blyp1Jui09BSz8AUFmrcni9VjWhe8LUE9G1OfUYbjr852r2+jQfqIWNd2I93opIwXVYX5gk0Tb\\\\\\\/xZvZEEKSZglG4krxMzW3g01rUkpy\\\\\\\/kb27szVo4sE1fkN9x6ch7akdAFnbPlEgJZ7nof3AHv6lBXMKlWivjbXcJWTO5aDGbufmZ57HtaHQ7rmcF3PdYZ8OVdf1ufamhguCKWpWimx8USUJ+POpyuBURxNOa+oMom94GxI4\\\\\\\/MIW3\\\\\\\/rczNZFHqalWPNquQbNzXicpLvquWlzlhJXSvG8l+2MbwRHo3xyJVO54gvk9h0ow1WWDo01YNxxAOpOHfH8N6ScInuPoVQEepTA6bKZ9i6iLRwmW8J9tZSTKQbPMTp3kLohG0\\\\\\\/3LSWIFTDWnIzAWaNG5Nrw0mzqKANFIEY7zSCPktScfQUDiYIas\\\\\\\/USFQrM0PgxXnKN2OGwXmeRAOzAa6puZOLp0ivy70yHKeNDWO24zlZlXooQDxugx0LXs49iB5P1QEyYgAR4p1q9gS8PwnygEHrhZBUAijV6nHHyiT4jSG9VAvMkHQwi7ljPRrsPCwUZqJ7Rq\\\\\\\/UmXue370uHXWPibr3ot3nui3NWwNAnmu0VfkSUa4T3YkY9UX7KAFYRCeYsO2Yx+Hh4vifykI2ZLYNqwKFRCdwnvaaEJwJkFr9C1Jrhx9V6ymFDMVD1LXP+NNixkFviOwYWqJ0meurYvclLCpiQ4KsqmC5KQR3gx2rhhbHPuDDC92FFP+kG6R2OXBg\\\\\\\/fxB6mxdmF20QGTM2uCKYimxLQDWTBvMFMZdDpvkC+t\\\\\\\/wsdYYL8rFakyG978WQ6v7Yrzn2EUWQmGitG9AVa1BrPGrb0buqjzTHGNVl5nVIlj0i1+1Y2v5ZUYkucPbYOqW6HmqlRRq0uXGYoSpVoSjIk0QkvRWcnD1x4PTm0nLm\\\\\\\/qQvWyqZGX3edMC6h015tIBVmzew8Z25UHV3WeaVbpScLZO6O6Y7Zb2LyMj7tZYPaVsYSIYns08+TLApXPgd2k1NdJzr\\\\\\\/GAaMoSrCBb18IPnWZKaxERyQP1qVRMxBn0nYE3Vh\\\\\\\/2EGCB9wuPsvGMQ+e9h6JNayjYjW59g7qz51RPL4Bp5XdaMfAELcfY+nOnSQEi8Y+QgcZJ\\\\\\\/2HiAI7PWPfLvpjdhRbJIFqnPZD9IlyZh4ik\\\\\\\/eJ\\\\\\\/6hnN050ww9iNq\\\\\\\/hnuDWEuTysvkmdRiWEhzFif418FOvq82XfF7AqECY6LPDW962MKLnyopDPrgmLYcn\\\\\\\/YnJyFJyX\\\\\\\/VmqluQluCn8Xl6K50hykDHSHWgk5VyuG5caMJ9+zJV+alwnZdhGuWxnTLvMB3hZpMZODwcktplW97bqTecJKas3kAg+4sJ+X8KwWh6ddpe5oTsmmJBBJRkvJ\\\\\\\/R03DxAipJIjPTGpXV1\\\\\\\/u6bKoVxeMRoATZ4EXod8XJ7O+UJmydzfptGvIyHxAMzqrYpV5jYNTWaRaqE9Nu8D9t3KpWi+bqrEiMuPiy\\\\\\\/vmfb7mM6lwS+oyWaRJ9cxbp8PaTxD24wxOkZJFg2Fmyy7Teld30Ai1WKMX2TCSE0Z2oTzFD1672LHQM3vjW1dcWNqVBMKSKCgCRgF11Y1e3etGC3sdHw8LHKmeA9QKk3o4Pg+nSUZZt164rd8fQjY4FOnPr3bwCTvRVC3m2Rnd\\\\\\\/wUZ5ceOK12hTB6mu5oWvjZCWNmI4yTtmKDPJeVzdjAWJ8yU3EO14rdtgB5irWT42Uv5i5kMHPg9rLF7f+4Yq8S0Z5zBX2ydfk9VX+CrLUDu0TUMv7xN\\\\\\\/ZN0pUHXE+4CjOmwgvIuxryueEpChpOjvTf7NNzFuwe9A\\\\\\\/sxw9fduCCczgqpvOv8LkjTFPf1ouUfoVC0f3g5Yy3F8xN0t+9r1yVq4tZFCKjhX80jFBzYQn+7RYo1S+mIF76nUZAmF1MOhJ3u8xZt1zdrn7ABGyXYQXgpgh3e80VMUvhl6xB6gDwC2mj9\\\\\\\/mYqxtOCO4Y40uYxZcybl0Zyy\\\\\\\/pfnpexuUJRGeUbICHjr+uXX5WHyWvTa\\\\\\\/1y8ptlV7BXcf8escgjarun7x2v0Lsx41YQII9cAHRn14tAHcljlWeeVL4QC1QxlTHn3e3KED4a7sY3tqdlxv5DSkrqWtgVS2IyroWrBJny0woXnuaYBbDiD0tR438Q37h2Mw8drm98G+dCqpeSrfxHuXutDgek8ZwiYecS84LcLbrq9lbAJcLbY6+q4VV6ogIscwUqWF18HhbAkoQF4d6lFDFVxeeUXX0WvnwbinQXANXLUjtFKGo3KM5EUsaQJiTcB5rLHW+M5SjFxjKj10YZCgB0b54KbULIawX5nDpVWZkcOLHDmPmSJcIZxyyp0pC8HSta\\\\\\\/nk5rVAozVSbSrwSyh\\\\\\\/VhWg+WKNgMcTW5m91DC3G+vcgVWvWZ7HhD8qqhbr0yjpTt9C13znEP16hvrU6fDIuH1ushfGe5CqWk4Cuvd\\\\\\\/AZ41MlX1pUSYp1TJlBTvcnElV3xNDqMe4rdaZe4lhX7Etk8T4E4WdGQ+kC6PTnzjYxQMQ4oiENRSeVSBGCdJjyvjFZcv2FvoFEjM8N5hQgAGqezRWpyK\\\\\\\/0Ojt7XVN+uu9iGk\\\\\\\/KI1bW7tOvyxGjTtGPyyQ0vCL0IDs9VRqPMthMPY90hHTfT92wTlNF1heeqAmLqjUEZqnBF+MDYK7EzAMXqK6b82li3JBCxd5sQrkibj9K77f\\\\\\\/86TdTwWmvP9tYTVxJeOkvrxeyNkm9AfzLol9jiVyPxVALxVZ8B74HeH9N3ksuEyZQtOM2hpKcvf2A4bR56AdIu04GiJP3MWLRoKAlnIw3CjYF\\\\\\\/06NAv0SGkSL+jgVgEFC\\\\\\\/yw04DUk2sieuoAQmyx1hA8NeNYoQLYnHt5RczkNez8k0P19BCw3Zve9ACrCQd9w62e7nqyshpWGAxXdNa1MF3O2+vfiSPlQ0ViySPBcx3cC6kjEFgPqXSsizkRjFbyoEy4CNbKUR2Y8EvZfIoll1ZCsGQOkuhQycS+vHIXXUkK3eeC\\\\\\\/D4y9F1+3ZdHZdov19jxt8mIpSydRiAfucHTMRZ\\\\\\\/7RIS3qqf2hGaWvGR3D1YfpvI9TkMb6A+AA70cSHSW1jQ\\\\\\\/SMK\\\\\\\/Tl5DlJQzAmt+WpCDWQ0g93NaMbFLx5iKjOwM\\\\\\\/W9x3U\\\\\\\/riSaNKinSJD3b0fkn+7XyDwb2oncIkbPeU03eRarfZt7A7uAoisITUq1mEuU7ND03h+vFHv2wYoQjF5uuTDjIpMjHqLKCxEYcgYr1wnQKtnJYlchaM\\\\\\\/qCIhf0SiGNMuWKwrkfYaEYP6rGjqfGBd72nl9gIBNDW5Vhwxq+Eh3QLrQyMFD4S8vUEQ4GkZ+YqEbVdQC6AvDvMKWTFFDutP96PW8WhkAHOvcO3i9fD7VQAyfyjxNHcSBMWkdDBQBsbmuNV9mVx4WoeoRXRZx7MjQwkmrQCLO\\\\\\\/arVGyMcitTfgW+m+9NEr\\\\\\\/UhgGITiGHHQ\\\\\\\/17xb5O3sVfTX4P2JbR6bdoqx\\\\\\\/d7IfWN8O969quRhXUr0DA1IXX8dN8sIKNwg4re+s6K69AH5o6twLqPylG51DW9+FWvkOj3YDDrwvj1dnICA+nfHhzbI9lrIyYG0mdf1A\\\\\\\/qhNTtacM2wXC\\\\\\\/xcsmvBjZgrnP3GYgec5V2kGinAOzBZrZuOTicTh\\\\\\\/RNDHqZ2ekKz0PbOiCl5PjoDaFGREUQs6q1736LzW3yKQyPCd48mDgjbjstgDg2RiwS6yDcFE0HvJOCldf7atqlzia48xK412KWo+JxxRNlPzvM2D6Eechq\\\\\\\/NFQ8\\\\\\\/6qAUTlDGlNsiiPDV6XHVc5M4hXNTbTknHqA2YY6PTk8wMgAnmBJzcdFF3\\\\\\\/macqgEfeDvzaT\\\\\\\/fZFgv48Vl2x+hodN1qVUjAQ8tvy\\\\\\\/AGqAgLxb+VW\\\\\\\/c3da30VbB8g15HMed8aUyVI\\\\\\\/iTJa3vewAPKoYa3cFKHsJT7N+RfBPqR5MQDCxvkcuZabdQYHTtNczOHk\\\\\\\/UkzQ1\\\\\\\/\\\\\\\/EYrJnsyo1udyZaDjQLRLTG+iHsznKS6bU6J3krfaglQtVKf8gVuT8tthkR54HyYaRBFvwJMwAxgBs29fXIgqLo1U82N3rGgsagkuzFdnR4tG+XIFFq6HvPzf1Q3MOzNonWHsU7WvVMCLDEkuiwDDP9SHQq57OtTbyGMkJg31XjVwiMVg7ShK+Q7f02q2BOZXtrKuiF7lfefbB1br5ox0Zcc\\\\\\\/fswrElSXE3TpisuE5PBgRLxCn+VZM4uPU1OtV5F7sye9vSTLqjpG45g8OHv0yateyxtWKCHgEcgCzE4V+pMTDXicLFqIhFKpJQ6DkxO0mXFboZuiOWvrgaXXqOgsV2V8bnccGxIvOrEjeAYkv0ijGMvt008xsyAGUzt5yjy09o62LPE5cegN0TvJ06L+fUITAR+we9RtTCd8Hjm7Y+zRy\\\\\\\/v+Aqhzvtc005RcnnHGpwevYSgbxSs3WCcl2c4Fce34JwgW+FPR6kmfMddFgZS2BX3XlTl92R8l\\\\\\\/fKsxdZN\\\\\\\/kcJkVXZU0Mx2Zg0q5aMou5wcn\\\\\\\/KESOB2cZPvK4uwbJMscRC+mCsXeu6qjvP6NhFz2wpCKCg\\\\\\\/47egAFaEIWZvGeAtnkYX6SN0gkQCpLpOnA8tJs1M0v0FYPxSiphIGJMrCYZ4gdcDmEKsdHNFDUhRoYvkdzOITHT99JQktkbKc5jz4\\\\\\\/NRcHq+Er5WPlmwUKj3IiwU+khhTi6wwyHw+Ns4EGz1C\\\\\\\/5vmfNy1tSBmxePOzz4pxR\\\\\\\/FkT9p6WAlaWZgKgLFfq1QOmZh3v5\\\\\\\/slhjZ9y3VnPUwuEK6b+fkOGzm0pMMcxlLaEOkuWnGevj\\\\\\\/h3\\\\\\\/mjp6bBgMGCJaGNGQjHfV3T+jIqQSXQrm5blMmE9vRwQLNnKgVKinpH+WCYqI3DBh9uIUt8mJNpT\\\\\\\/f3MbCLJcF+aPORqPy6fAjINRFQ5PJds5VpTsXfEp4N+mSaK2MSEgJXUXdIEOQSC15DgZX71CNULUK+lLPF0IftQqn4qccyxh0nWv+u2IW7iDFoWl8U1tSdgKrU+87wcPWpQKGO2mwSSs+NcUgQJbfoaI\\\\\\\/Nae2Db+AJ6L\\\\\\\/Qspnkr\\\\\\\/FJwpJcE7Fs5Os3TV+HsH+x+0\\\\\\\/1qqVfzSek462prh4iziiWoOp6qCXKw2ZCPaI8UakvpMjUcbx3a9WcgDwt3yNWfgqJMxzpk0WFwkuT2e76YMIkatJ+6quajhFYpDmLv38cavGMmEAyMeba9IJAwOEzHSzYz7ufF\\\\\\\/r9\\\\\\\/HRfxOusGAlwQV\\\\\\\/hIKPp8tL\\\\\\\/SHrKkQpW4UAJCL2R3zAAneT7EGi1\\\\\\\/+UUIwgJ6aRFwE2GXnD+Sl+k9KiF9a94ZLdfJYfVqhqKClx41tYLJdcsV+gKlpA8gy4kZVIQVt8Ano8kLweYHGr+V+eaQROrrranrTl4ozvPySY\\\\\\\/cEtgunMwMCcODiWN4nFO3PmtXnnzK2S3KCCTzTnQtLYt4\\\\\\\/0H\\\\\\\/CeCrbK88Bh9yH8YMsmyQTwFZRiMx21lKKbzruvTxBcUnM9Q+xj7yq4OJECgP9Z9SQtIx212YSEBN0Ioc474S5mt7cJvKzsik\\\\\\\/ksvHkF6p9wosCOLZ\\\\\\\/cpCmj7+kfz19K9vWD\\\\\\\/X4NsSXEzdigF9CLR76ltibCZX\\\\\\\/LcmLdMJ+0QTZlTFvR3ZBq5YPqtR+oEPi9jDXgqRTagftsWDwB+9XZKGT7\\\\\\\/IE\\\\\\\/6bS4h\\\\\\\/9S7wmBqSvnA6MqPAAl3ljCNOoOBNN6Pk5FsDfehOXmLah1Pw6TzCiL7ARH3Ts1x8fNrGl8QaVpNuBWBvG7HFobmfjD+SMkfx3K\\\\\\\/n+eLe6ZeCpIbj+rR25uhUVWVwUKnTf3MeQ9p5M1842GseB93QN1B\\\\\\\/Echfph7IGNUOFd0I1xWpkGUcLYAi6T7iVWUAjJTNxSVP22a5TIdgE+CZvv4GiiowuOQbtqOAaEeT9OePZ9qhS4xQasKRrGj\\\\\\\/QT6VQEg315ooWVMo\\\\\\\/8USmoSkxD7VfXrdVLcBDK0RsrpGl\\\\\\\/0cpar1ffTvD73yuEJ8RRf1+bugXDuv+eA+Zqt\\\\\\\/uTmauM9pbouu\\\\\\\/QVM930TPKc4CBkLBTzNUrYboILbe901F7pRW4fTg23yt3m1ZFVpUyNgxKPBJouaONT1PTObdKJXEZoH9CUcHSLDPGs55Do4GJDxq6IwmWQ8bmICwlIBtz8qnL2i4mG0AR5hqeFR0V7\\\\\\\/FYDVpxUJZ\\\\\\\/aXgUnhsg3bn9Km2vtWHNIeyNpiBx0OT7ynZGYrbe8yDtI1J+yvY4sZoTGjFkyrxW0k7G5dOE+3+uoR609+ISoivuxY\\\\\\\/NbPwvECxHAepsr121nZnWnCPeLb5Qn+1MMHFb+k\\\\\\\/XCZbrZbxWJsgyyR+vOcEecwfV9SGKFxB2C5VB20Fj9uBFySR2Y9j2baWIuo7m60IJmVmu+\\\\\\\/GlwXl6xDw7vd7l\\\\\\\/X\\\\\\\/JQLDlGdam260qqPumVm46pTqaSior0OKFEwb5eLEV2s1kX+pSzT34t2SZq6z4X+ctnOjlVg8A4o4fP4MHhPx2BON2E5Wco4mm7S3XHT\\\\\\\/vUM30tdubCzDlXriqwlDwkaF6PAU5INRn8OK+RabKtTynOEehqyAEzYqIH20NiiAyIKruzJGfPuPkUXKXQcfFoswIr0djhP9P31TiIlplAOM31QaxztsDIjEK8YW7W5JYexhGHfVQLoBkqK2qUsib+483BGRmw6iQtfWSgOENjvhmzAMCmqilhLlyKVsOPrTBR7p+iysBwLu7I+pmDkMTITv\\\\\\\/Xu4nYuHEb7od0bqoRAMMNgW482s+TnFzIo3jgcaYbjvGVX0a6y\\\\\\\/xZdakhlqz304zrC4k0m6UKN1dXYMeU3yfjd7V3MLMXYBWaxj0C9Uo8yseAkoiIhAevzsJ\\\\\\\/EbcxaDZ6wNIbDXCFbKJXNeFZ9+jGzVhDHLP5DnR063hmG2Z4RdpKk052pUo9vLQGxdj2WjowyxHhwT9sgE1Tttj0LyCGZ9kxjD22x1y6NLRHD0s96XuN3KNNhAvQJyV4KHxiWiltXXXgA+4uqDill70W3\\\\\\\/TQd2BHFdCqwyRZj0b3P1Ybq7tsKpUYNau33RZj\\\\\\\/lX4m7nGAXJ4f\\\\\\\/Aqh6pHvAmqhJG33vsUsLEY3ckAulEW9Ge0qS35Ue4H1HZ6\\\\\\\/dzG9xrVm5XiFz2\\\\\\\/1TFzyO0rtwe7OvFc4H+z948CqkdWeh1N+MBSpZoqErxJ8R5RZUpHxaK41FXjshNU6j6BbgBdyZ3A8JwCSJEuzO+ugHdRyhVv+KOIhHMCjTD60tQop5CH+lpX7vY2z+zVMba1zyxBRl4uGGlspf2faAU1sHzr0xCAvsIWIntHov130qpR18EhmNh0FG1SH8LH+togYhFS7t9mvAxDPWMEc9vQGLPwwwXigANLkP\\\\\\\/+SlAV2WazLRXRlXx6nHGx5CAplP9BOgOtTXdUqBRiHfP3M+36HffdPKEaChlp1k5RkKA7qNyCW576x+XXcTbFrB\\\\\\\/OpJ9OITVnanAU+ZaMhn\\\\\\\/j\\\\\\\/nFAWbVJLXxSi1dhkZTPjeaxglMRswzdfEPPLyglzoOw7RgdTK8t4FEP8rlS61GOwPKajI8LzxDOctyzeGO4YMNcduYclnx6o0g49nL1H04tZJrkANGjbmt16eKeCEnXWWM\\\\\\\/ZEVzu8qGZTTty5bU4eLbkMRfVvoR6g1zuVqUE2wroiQA7hymI\\\\\\\/AVIIT\\\\\\\/w6Uydg1ndt7rnLBSmamdSO9rLfaCyLkYcfXp2ig1Dup5Rn6oXN5KD6cDQAFIz3cbKqdTvYRpnT3vTonjQk3hZCmcPDtpu1VUf+Qs7C83oTRjBKGvho31oEvCecUTBgLRAmEu6r5f\\\\\\\/FQqu6+LUuQkxdXiA1My5F6LZJstefDXzbXoXQlCalsruevIc5iFSPKfO1hHHPDRUGLG1zobpmHi2BcoqfSFgZTgo\\\\\\\/zYfL5y8UySdea7tWyEG2nxsVTex3D0OnINkgHyowUd5ZzmxMByRde0ZoYjLEhgrlX7JFLHKDwNPbXLlKeJ05h\\\\\\\/Akr+4TPs+B9Rsraf\\\\\\\/ajT0BtZrEp+Otm7I4121nyir1x4Mvq0baOxCdmsRiLKbEPNXONoHj5dLlx6MBACFkR764BynC5hmpsJQX0zjoDLeRyhAo1fOyLi0p3fkt3wJYFYeYgtjjHSAeRH\\\\\\\/PUgAEyQtbLvNY8Aby8G0UES6aSucGyw9Fka6R0+ZW4Q+JnzWjQ0+PGIPRFY8XQKPrlrlkNpt5YOlfT4PPSyPO1EC3Q3FFvPZHTML5rXqzqUoLZWI0P1BQdnsNrm3MTmX03lK8xzLLriFOL\\\\\\\/Y+AULJsbn90thHZOYRJ1IJe+3Lw6lfS00vILRPH8Lsfc+GadsAzV950Up9YFeMfzjL90OHHAXkolEkSyMp8vuWaXFEahllsjToab6iTt\\\\\\\/skYxmyInSIRjbqQTMuZBjkiXmEof1I6ZNiK2\\\\\\\/NDuivqZsegEMbAWZ4CgLlc2LUK71vbsNDZH1sIEL22TuXVNsa1mtvr1FDqfEvs4OZsIYHBVBE0eZA4Tjf1lHH81IOGqrhG4HZh6xhvAFS3sSEDdMzyLqy1+zw8LNpOB+pU+DuX9eZKXY3Kkmp2OV\\\\\\\/6waJ8FTR1gt4RSLA4FkWMn1GLxef4dkyxUb2lYReiYYPxvh3ZLWn6Wti6mxpC001ONfCnAxI+8G4\\\\\\\/ExNKgKqL71XebTu67L7Z2i\\\\\\\/Drsh2lxUmIk+0nIYjPpTILLAlyKDh5ub6XoKdZuxcVvjjShTiJu8k6vpj0mIn1lALQknyUA0PeAoRna6jPteV9ggNbwLNx1\\\\\\\/8PaL5R6JN3q7pVmC1Xggru7GZKt3IcPMps3KLiYChlcSOus5jftpGtbgm\\\\\\\/0\\\\\\\/kpDZLKdFnN9Alxkxsw97My3RtzrRR3s7\\\\\\\/d0zJn22hkDwTTTo8DSP1wAPUEQMkSOCiNqIgW1MDEXggvHkIr5peTJZBRgZaW9aQZmT09hAmy3IlLZxvTFy0fNpAweycoGJtDd5A9Q6dWaonZo9KMCa\\\\\\\/NeoG6l06X2EQ3s7ordPZkWZYHPwGvlXYEbNIV09hJR0cFJgQ5aske+ySKTf6U0GzhDd5g\\\\\\\/JzFLZFLIXa\\\\\\\/AkgA7YXePZgTT8GB0\\\\\\\/LYZ56g2V+Pr9v6EDkuy638MwqJwlH1c76MjoNAN2I2rYWMgjDoDFAQqlXYgYwS0UnshG6frpQOk0kUAicvG1Q3KHApYO6pg5neNdCtfbarRRZDYsBf2FZRHxdf0LeVN8uamPrFSOtljsqjCqct\\\\\\\/EIvd3YF4EJafNT0iUw9QznFdSCCxZZwElW8\\\\\\\/eKYz5891JByuSA6v6893l7XssV1rhh6umzyBgRmL1SJYrzBaHbPyqdpA3nk\\\\\\\/5wXtNaUREK8aUJvakxdmORudIzkl7e0r6OiHvd0l8vahe35A+X3uhb1gt6VnpGfGr72rgpQ7TYYZjCGvx08H\\\\\\\/DnL4C7C2pNBxtkDRkvPSZB7olwIQ46R6r3CZNS\\\\\\\/R4rn4J\\\\\\\/kImmxc3ifju0gG8B4PLPjXXRgcBLClvpBic1q4gfKiQqSOtId8ekfU\\\\\\\/JW36awbIU\\\\\\\/9mJdTe1SWPZVA3m6A2qgF\\\\\\\/D\\\\\\\/G+rDwB34gJa3SqtE158p0HvP5ePdmix28SMKtmPP2UQuelIXIjGAAOE0ogFLz1M\\\\\\\/\\\\\\\/43jbvEHgdu79NsOZ9nRUBTEdl5ErDS3Qx3NoVuq7hd49YSv\\\\\\\/ynsLP0F+ecekRTB55HpE8\\\\\\\/BmA8ZNzG7C0KlrIwdbtZ8s7\\\\\\\/Ut31d7INi89FLNfwNcbcDdG+5wO70R1b5SIMbmJQfF0YwGvwOywt4iX5L8G8ngOWEJFy+S59\\\\\\\/hvYU9rz7FxCddlf+RwMm12tD\\\\\\\/SA\\\\\\\/4A59b3rRKvJAu\\\\\\\/3niUBwiTuEPk\\\\\\\/Nq9KpPeROghIATqcu2i4ljbzxqmTxCxWJ5tCNlPiLN8ccwGANJhoyKidjlVdQdMrAxc6q0ITrUEokpapo8hOAPt+VuAAxXI77DQ6fJY\\\\\\\/chOWjR\\\\\\\/4z+rLDehq5QvYrC8epDLkeO4Conl00Yq1AKLKmVGypkrV15IALgb9e4IBofCbx6\\\\\\\/fE8BxJqob3yvuqNPqQmwqvP2D2WJTizIfo6Ci7ZvcbLwXtAilqYwfxXvcCbQI+XGT1otCyw+YtP0MkNtWzwyL+iaiXdn\\\\\\\/pPjogedO3aan41levMT8TSlD7x8gUYe\\\\\\\/dvj1Klxi6DySakek+zJpU8b9GNRzeT99lColgXvr+WU1caLN2A5uU5OkbMH8KGXOLGLtpiWbHrdyupZozt6\\\\\\\/i2zrcoEnQGfcK41fNyt5NMFtDBrPn5Ugy4\\\\\\\/rS\\\\\\\/IWhJMuNmsxhp0JbC41jgdp5CsU1wQ8HExwTf\\\\\\\/iTJiLRaLOuD9figc8iENyfk68ibC\\\\\\\/j5X\\\\\\\/sl2pAYBKCfM+2Hpaj2cQKgpIypwgD7BEzl5L7TVfrUdgw5\\\\\\\/PWzQ6CQQvoueJ3g\\\\\\\/xIBYvksWsX3PUBT7V+lp8ubeyb2rO7LSEw7oWPcDz90FU3r1pPh0FcNhmtHXsw83sOY1kc2kn\\\\\\\/wFCymT4Xw8PJJCjGsa9J6oUbD5bEV9w57SLJhwEc3m1WdVEXtOe1+KW3fAaYLv5UVpMxEHcGpHsM1Clz3VW1Jqbu7SsOB9wjEmfg3a3DJcIUjchPWokfKy9Nd8ET4pcfb7H61RcQvaCh\\\\\\\/Q1g4fF14dNMfLuXSD0MZYNCixR8jv+tVjD5pYsE+4gdFME6jIyQw9F+GfKp4acVZTEKkvqqnZIea\\\\\\\/t+tKuTZZW4yhxzXyEH+ZiiuIv0rnEsvK\\\\\\\/SXnZhohpRqKskCS2CygZo2Xqim87LP7OUHXJtmUCzA1kOaskAuQRVZ+4l6mvJwyXcb\\\\\\\/EZnKPOA1tON8zPluMeJmZFWjDFkF0p20DRPU+JuhQiEARNDehDhc+un9ReG1vFfpgF9YQoWUjvxBpDG6KnyFol5Kdwnqk8vqKlEXZWNPRyBWIztBRg\\\\\\\/SrCvA5BdYAYWLzwfNQv2BQ6Wi9WsYAuvHfggaxsPUDvD8n\\\\\\\/qPDeI79GXFmqazDsi6CYMeUL6l9jRM7YRYxsL3YZydJjt3TX28kYv1ukhBsPOLHB0CKRkbOOuzwHftoj3V0iPrZKtnTWPS3naWhUjiMSKpDie5aUnO4DOUJ7IbX2dOwKMQUJiqY49io0JMA0bixI2ogwO9hDooSTIGELB6VL7KNZAbOYlqVQOGAG4QMl9mcEZVT0tsDdEwLQ3+s7T88Cb0r3N58Qs01s5AjRNjm6mVKa7qrYaXeFSG1KjCjvp5Acjwa5fDVQbJEPFJsDh6O3ofDbRTvtXrjLQoTUPxqkPoS8INCnuiTVcUXSvs6STzdZwGkSmhTY8HAkK9nuaMY94iKjoQfX3xtVOsldcQf0juesIeHNfnlCPsk3fbsaspC3mZGjF33ZUVizG29BMVzwCH9+WzSckgrlbT1UIeaRJluPLdjqe6qVOzs1kmpjI+uXDzW\\\\\\\/4bZPg43pUSDr2pZUASXQWDEwydRUZABm0IlbSY19hRmnG1FDPp\\\\\\\/Og\\\\\\\/S05ovoi20liyjtwvsvNB6tP46hAa+2ccYOnjf0t1\\\\\\\/8yy3DNvcutIED6ULs1zhadLKpJptEirruFE8dIKUUCRPnI\\\\\\\/sxFWfNM66df2IuiZ9DZHoGwrkaU3NAAIY+fA7w+dx0zjzGxpld0VIf+WZRa9Qdm4o7+eAMcq\\\\\\\/7brD2cA6TgwQNcdu+DHv0TmSU+NiIQlubPE2RrSBrzLgZRc196fZYPZKawH1XHiHr9GjU4l2lzEkQGOy7LNGGLPJk21ko0EUZtR5lhHSHBNOXrqczSbbrvk37fHp\\\\\\\/8+wND69g9iQReQMshFH7BdlUI6r8maV7452mwsLv7nfdVdElvni57GJDQ98VBW0pT4hxwzZOuxT4PCgqTivjqFvYNfM+86YLpgsBZTUlLuUHEg17Kb\\\\\\\/RolWAgMqoNxQSfXsb6KLLxH7lop1tAHR6aVN5FYo0dHuLIrHwcxGM1Vp2vut8txcpcbKqo\\\\\\\/0QaRfA\\\\\\\/BFnA5TiGDTOVl9Q1VsYp9ndTSGDWeCQheG4ihH0KjLnoy2DC\\\\\\\/t8u3cmALBHt0zzsKpts3iLd5frNWtCw5Tc\\\\\\\/\\\\\\\/W1VjqSM+VEySMA4Res0cD5aEf9frrswzz5mWbXwDpe2cQ8nHRpNrrN+\\\\\\\/h\\\\\\\/pIXeAlWMUIDi5xyXUxGFg8OIYxfRzRrpz0X5lU\\\\\\\/ywZFpCLLNC0HCfYM\\\\\\\/KqKM3A+XOHDqFN7Y\\\\\\\/Vq7HZPQpeQbCf65nr3cPIgUAE1QHNV7erN+rwiuT21hBXVYt4yLNG+UHxefJrrYn3n9HgOtamupKQKlkNmraRRQxfAtz\\\\\\\/UPv7P+8th8MUtQUasGWYCw\\\\\\\/oRG7ySgdP8bpbXtJEKFlhpCgwchfgVpjZQCSEsH2j0qyZDvNDxDu4JDeWoGgkP2SUgxbDp806gILBMyiLW13D6km75LL42L4c2s8BQ2QLevhOjR0mMCmanuPI47s\\\\\\\/Jt9deYQ\\\\\\\/CIy6j87MkKaXD2eXg7yxDzTFeBtzkp\\\\\\\/Bxrl3X7A482WoF6QDNCXHZZlriWvJvz3Ez8hWfdLCK46rV+fB3QnNZKetNuzr5Uf0FIaAqDesKHkeHEFMGuiaCeNqumGXk74sqbu5wTWZhTChkn\\\\\\\/oAlNZqf2\\\\\\\/ZM5AGadkFvLMmkXad5UMN89xml9lmg0w9h5QjJBX9\\\\\\\/Z5817vygrddskkhwtYt4qhhEtyyUAN7yB5xYbwkQzWSMSpCmm8FGsrUcbMZ5o3\\\\\\\/FJ8gDbNTdu\\\\\\\/q7bQmU4wGzKZEAhqvOZUk\\\\\\\/i2BbupMs7GKvhIN9dEdkDYqMxqIwY2i1zUlatTaEyeWYx6xDJOXHcnOQIq+BtWsYGs91liVTTvOAHwW1QCpZ4HuQt2ZDfojhTooFqz\\\\\\\/jURr2J5jAdL\\\\\\\/bjyfDHF4NxboUFvVf\\\\\\\/96pSPZ\\\\\\\/imozJiGO8sH512lVkQSEyp731zNxa680evJc+vzTDq0k5Y28EKl54AV7XIajU0h+az7819BSyqGju9r2AjBu5xwsCQwdH8UgAXNP6PRaGma\\\\\\\/rgsb5QRtx4PwBI1a0blFGmmDZsCqpbFVfZ7ihsznmmORryGZAzT9YYtxL4Tx0xFsdhX74nJ4A2CoW3sB+FhptXH6jbg2H1NJ9pA7G5a7B\\\\\\\/kIeTVvHEbK+pylKg\\\\\\\/Nr8V56AwcOH0NnFUT89QeExf1SPkXMmnMeCD7HamjzX74V4nY2pTo0peo3IbMVLWvMh4ZdUPKGlndAMobzFleK66UwdSdU8rtW9Awj2vsJdZSwN1V+OY4i1qslt5q6vqRpt+UWl4nTt2VI7mN8Ou1yf\\\\\\\/fLVCz1fjxcD\\\\\\\/ZDd\\\\\\\/tJV\\\\\\\/HEAUgr6GZAH4uBdJ2w5dVS2sukqlRW8cbYx6O7bKbt1x0X7ScgHjnilLhDcj7J12RfS5DDuQcSuTT8OTatdzK48dfG5a+pDz71hnDawg5pE4lrsbFJg08cjDN+GSmvyTB9zdfrkTrC+JQTbTE5DTlFOFXI2y5LnuuEQhv0zAdreWCQtO18yukRAE+DLu+Mfrq7fK3rX2fE+DJEI7MJzvlZ57H4QaZRzq3+tXYeK5gm1qRnv5ztqMP8ZrWQKyVNfQDEkPQJVN+olmSXYrp\\\\\\\/tiMuPpyjQv6\\\\\\\/cs+o9p0emROAhJPPWDgBW6yi\\\\\\\/byon6Z4edOXz+VkDVk9vKCjydUWCm9GhB8pVs4BGESmxWaoQbQlVZueLnyK3pMTQRM6PND1uIHeCRbW9LkdpymzrbOoGrD33GRvVFNppstYBNIL4yHBoCTlHZm0jsjvqUsj3godLKc3hNhHciyEWo7qdtyFAD5q5ANo+t8pm2pxB50kgmACeRG567qofrLBc47rm39KT7QLk5LW\\\\\\\/Ps+IgwhIOqQJN3TaM\\\\\\\/VRl\\\\\\\/FLczIsDwSRpjqElrWuTrNGFnHmoFMNqlcE\\\\\\\/DJdZPn7qRkJdGNCrx4ge2JuM6GS+vc6t5aa5UNISfY3pyOKsJXyW3DLkBoKp3rr4b54nQ5x09Rf\\\\\\\/CdUYBUryVgXAyzr2X4\\\\\\\/VekTMqcM8Waf56nJP9dhF7OyKrx+SVS+8xVjDYuWU+oxxS4F9FRUqlx1EXAytHLnNvtAX+mJAJ2m40HxrMJzfnzhUZaA4Q3r+fJsCph8jAzGPOiEA5lIIIoLEA4QFi278jCIsL\\\\\\\/NRJrbrc2EQw+MVZuN4\\\\\\\/EVMH75r1BEp+4bdecMNkHbTf+wVGlGy2v63q7wHt0qyorRY3qCNvZ5LybKc3c6IEzImiaSvF7rGrNSKVNTD9UWSDRzdEP36TqHJodcViS0U7++q0uVIc+RithC8igcLOMnDf1SN749E4\\\\\\\/NTHp+u0\\\\\\\/HBIxyUJj3GtZgKFnXxx\\\\\\\/KjTG9Zxqow41i2MoF4VNyLha3RAxeyvsOFYMiwSwFM7wqwQyqdw81tQBrbC1Ov\\\\\\\/PFXxZUVc\\\\\\\/VsP6gE2KusoQc2aVpuEHGuMiPVCPAo4B9QIKU6tS\\\\\\\/Nqsz7mpCJ3AnbMSqoMZDXjv3pXnugCb4KL8A6czhvrjjWPvWBq4+eGycHcww0PkkTueWoVyjd2aTExYoEzBkdbYiYMcfuuX\\\\\\\/RkNaq\\\\\\\/oEXi3EWxEOS\\\\\\\/T95sFDFhgwOjZewr43H9yWZaOa+sSiqDRBgqPnYk6eWtMMae0bt5gLNCER307zRRMBa9LAcRoVZrHXuDzmDi2pnCF9zWgo+pxbrYFp8vXC0ssGZJ7i0FYzQA7LcKvEf8QD+xXKgtAcOBqOqdKRzbW+RcCDO6SiA\\\\\\\/mBc3jFTQtiCDoc5ZRJFyYdiYhNEH\\\\\\\/3J4kvmNKNRTNpvAKlMgpR+IUfQoT4rWSwo\\\\\\\/EpaF2hcrOQ2k414U8mKgD2OyufeRoiCK7miJBI0u\\\\\\\/2qTBUKNBJwreM7Dcw3RPaaE8q2jUPlShSFwHNzONwz8m\\\\\\\/Pb1BmrXVe\\\\\\\/SWGlHs3qFRiwo1EwOqzu+C4tCiTMRAGoe9AWFH4HVImow052O4A0VsVul4Jdp+oW7kiKMPm+4VT2dceKuRNO6OzOQms4um0PPX9g\\\\\\\/uy5kYa2aQTrSZfrsRr+mNmqRr8Zw3nGW5ixNpBYv3KKG+MAIXOfLDnbpEY7lTMArc6bFC5GtNzqtMP3e6FF659WAw2GQV\\\\\\\/6s9HlNskPrfv1Ii1FtVBy07cMxRJB1P\\\\\\\/pnhAiLSxFvOJfigD7k0sNL7h8K\\\\\\\/wGc33+Idi634N\\\\\\\/N\\\\\\\/Kp1DK7cnVKkO0KQDYghgkIOvxdxg6MnyG9IkjmirnbyNPHMCH35+rt2x15HD6dLz3aG37hOYEmczFozcmZd2dUuGgtQWHlVCr3DQ0W8FwLkQSHYZAgU8lsMfFLGdpotkpbK99RDy4xZVgd8dGwLlxLil0HHgizLbxVPqcNM+nf9\\\\\\\/+m9nxgnEZh8Cp9obcAzfblrjUmpXjmDDrbu7NHyohGh18b\\\\\\\/kZrJhBHMGgdylhCm+yONmusuKsuO369U8TbYzEXJYJyuIMt9\\\\\\\/i7934Xqhu7tf5ws9IPCaBVoVy9AHwcTHeExJjKtU0RKxh4kVhwZa+7G0sBWr6C3nr3REiqCZiv8NGvgBwMAcVxZdVMVGzF+YDRHk6jrvHJk9TMR3+UzREkkTqmrx4xRuw4m1\\\\\\\/7dkPtp8W2TxzfX+lj7VtfwJUKnexhgLD3\\\\\\\/f2xsyGRxEVk+VX5nI0skaulfyLV082AnWxfJuMN+cJMaytgzrXSV7eunc1WE8gO0fKwXTshdaVGX+PB3RVJ\\\\\\\/EPm8waTP26a0yZcawd1CAHiwsjPPvsydMoGyIz6YXnScKYEYv6oA+0wGIp5KxvrmBjOdb8UuQHw0keqM6r\\\\\\\/mLbDrLhD3PYhW8s\\\\\\\/ToV2hPTeD6yOMgn\\\\\\\/UrW9Ui5L\\\\\\\/gaGlO5vnUEoN\\\\\\\/Tr5GmdV50lYhk+nP\\\\\\\/oo2BwBkHKIuy2O0A+Ao3In11jNBSMtHXgcClt4JFXZYguo8JKC06Qu\\\\\\\/6dwVKYpQkPPaJ94KwpFWO37U0qVKIUC0RqOlRrT3f8z4rYDxPXJrAbE+NLnDKdvZN6+NvbUcxeDwvGr1oaY8zjLsWsITa5hzmmwvzF\\\\\\\/EH+mhHTxs5Ms7JyiIqYTgjz37B47fa5QXv3kMXKotePHX4FPuVHpwydFzxJQ654JKy+xFnzzdkFEwneTILsV+Iey\\\\\\\/g85Zb2me2VgkC1gicea59zRYykvaN5Ewx9vS9dWQ2iJFI8DPZN3l1ZJWefxYTUvEi6Te0uS7k+3g5A\\\\\\\/1ZfcPHaTKIgppSm2kGQAA6jb3H199ztSFrHLJ7iNtkIaDII2HaendsAllRNlAthb4xech9U91IHULcl7CoIYoIgzYl8FO0waf0VZtAiKqvFcVYMoc69Q9MFR2kGue5DKbCiU79687z\\\\\\\/wooHqgb6U2LAUqdeE+k4pkk6xMo7xut8YEmrZMsY9yrAtCywqDGh7\\\\\\\/zm+lhyO79FJ7lOMrvsY45KrOasMO2I5ajD9FJbcHqBVWQv4v87j7Z7Zpof2fevFin2zgj0h6F8nxgdA1svBIqBxOfhD5Rn3rFWE+zm5pdXcqAAzTPR3besIlX312z4DF\\\\\\\/cMpkGIp\\\\\\\/vhjFQHq3EfYip+7nuriUawSV8AK3WDvjsI94baNhmYX2\\\\\\\/Rp\\\\\\\/457WrkaYvtnVQAjCaH53Mo52xki16xXkX6Mf95iKXoNYYxp1LskZilulezBP\\\\\\\/EBnwJffL9CqOC5TKE3MQ2BwvmVCBzEUjp67Y5pl77Fj99m24pR59fb+eJ6+zY5rzVpQysW1K3xfJc33\\\\\\\/v5bqvW6IECNrN3XeNOgdqVUJvCVHom8pp2MFWkF47kwEVUyrVdPpknt89J9fHBmpWdLVW\\\\\\\/cX3VRYyxXQ9tB5iggSVzLvG0mTN5c\\\\\\\/m8CTSOFwMBBMiulGOzoqeVwDju0PtkpYEIKj0or\\\\\\\/2X3WKyMF\\\\\\\/Jn2pSTF\\\\\\\/ttvAqMCccbXToOtuiHUKsQtKcTYVzW\\\\\\\/PJIDIHb2+sj26irNGr2zFbZarDw0eOC3p1B1+1ySXeQ9x1yzZGDCI52+0mhS0wj6xcqsa4AAa4ZlWgflOf1c\\\\\\\/bd24UQ\\\\\\\/HTFQRlnPF4UA5BtK4RHUsmIzKKqVW4WUE1B750F7UBNNn7RWKTbsnhlt9T4evS7JYoihfCG+dK9Npm8+fgUHZBxlEr6fJRPeq2N7DDR21QYWn53r+aiPsqkNiPvFDaSgJxGC5DadBSOqjlb6Gr\\\\\\\/okIWiKVuKtWRUcr4ixNfVdk4Li\\\\\\\/aHvtjhTVcOpyORgksFtv8dF2Dg6l30oYSrevLzUtWWTTV7MybWi8D2Mf095g497DjjIZFGUmseDE+cbqnov9P\\\\\\\/HrSn6mAg1wW8RbHAjlOeoE+532tyQGwk32Lasjh3w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022GUENKRmusektgn8s1MMP1w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d81b27302f862a88a4556d0f5eece39ef7739374fdb0e1a21cd897ef3177b692\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IiO8SrlOx9fQDK90Rfwhcuz6PCS0vNTxEZaE8xKdmJH6tmFLV9mwHvngeBvb0eHnnWzslZDXJssfYODyQknaDG\\\\\\\/mZmPLuOZiiiWpP6JaCRbpwqff7Khyj9ZvlvRm5xyhb30+v0lfkldrJ3I7ulLC4nFcHruSO+pIezFMgjJty1HZwmlOeMxMeA\\\\\\\/OQpF\\\\\\\/pZ3336lIicnsOjM+5VVZ6Yl8R5gF1cPGMFSz1tJaoWUxXTNC0A1Q1zqXCNHXX8yPYOyRx+m\\\\\\\/oJ2ygeX42hqUdv0pOvTT30CS\\\\\\\/v547rzTK5iNU5MiYnjUzDDQsHASK9SN8UdEAe4d5OeXuGveKtI38k8\\\\\\\/rBXkPmO7NS8P0fU9dTUcK8sVuqr6KRDYQu7jxVfGQze1B1pAgTTsbwO\\\\\\\/UIdKZVu+DgY\\\\\\\/OB8zlr8jHr0Uv+BqRxj+v6rkW\\\\\\\/QSaDENvRvYQNL0cHUyIkdgqR42fTCy34OLGTBGudZA41sYHgOJ0q8wPXgg6l43rTXouvZ7P95+4J95Nx289Oh\\\\\\\/FIi7\\\\\\\/+0jF1vJGWTg4ZeQWLgYr5NoikUvEJPtcMancoTQERn6v9ZpeGHw4pqVKZiyjdy6VIiW3WYolbHR7wVcIEaoQQKq\\\\\\\/q2Gz8qNQwQx0II9nCu\\\\\\\/CgKreMwbritcVVq2dO2JjwlYz7scnCPt\\\\\\\/roWqM38UXXaV+94sDsHm3C5jjlpoliW4unuMyMypVnF6wrytjCOIQBRIUQz5hOUIXu4V4zaprlM8ODWg55B9ImkcB8hhZzpQLTu9OQSNXgpEQu0Ep4nJNi0xJeb5wm\\\\\\\/moVguT7cB50ihK+U5\\\\\\\/Rq8ILdvDpMmIwYJ4zOOg3sG46SHxKG3dIJz9vOXcQFvRNaqpt25a9uWczIQ00t7FsBrpaplrHIwAYfz2tgUfrvltYU89KXhTao8cU9\\\\\\\/e+pPpkPGdlfLN76YqGljXFdPIum5FBlqt4nRhp5GHv98ZOQGMtFL8ra2VeGJTNJUzmqS9CrvacNo8E0LLvMPh\\\\\\\/wArFws65N3wRokW6KlXCEZDZfvkjRScMKiHQ1hZkzgvQR0hagn7KnVk9MNFW82IjLUz\\\\\\\/t\\\\\\\/DGO7EHLtRKl9MlUjzIUF3RuNvfXk3BXZ+k4HcxyHN2qgmFqCP+e17vTqC79Y8ZKFIdTUlAuC8poxTkSj4tanm6B\\\\\\\/E\\\\\\\/LVnFSTr1o6GGWKrj2Mw4eZ5M\\\\\\\/z0TJW54XOq340XlfGAwZi5O32AG5726hkEZT+6BUYBR9AcubCP9yOa0gA2vuUw\\\\\\\/xd7i6yDxGGO0xUApxN9hqJ4e4kswQS1xjs\\\\\\\/JGKG9TBiRFp\\\\\\\/x\\\\\\\/v9pRYaSRunXlPYOw\\\\\\\/S4sK3gCGcT4zEaBYLWEQBeNIt9J8Zh8KqwRrQVenhg7zfFpWVYcchYMKESmz57QcKl1X\\\\\\\/gs4MdigypD5LXtYQQ6IFJVZKsxuWknmFq6YPAlyQKiQHEeTFcfis6ong5atpcNKffkVrFyUrhyUSqqTyx3UbjNk2n2dCTud6kXFw0fQ0uGsFoN1KQ77GeeCtm9S\\\\\\\/q04H+okGRWJttSYEFNydN0pkwuz7+ivzZ8cq92JmEBiC5Zcbx4YzauV1v9fghhzXIhDpcAxVbJRegcUSyFSXsb\\\\\\\/znr6rnXAgMtez1mjVQNEdXSL40dBgAX6rdULFA5RXHr\\\\\\\/1aMgy+tLAlz78ASMOmiogYv7sEQMMepA3V9Iy0Q\\\\\\\/yUWg1G\\\\\\\/R878JELtDyY63\\\\\\\/q8hGkLCDB7tb9WyieSHoSR7zCSQD\\\\\\\/hN+jk7qdmiH\\\\\\\/9iqsCc50CH+qwiTUEudjM9UJY1sF9esZ9nt7xgq3t2XmPian7SKr4mCj7lfR9pH53\\\\\\\/z8tP2AMoUL2Xn2rHCmZ4r6ewXyTdtIuNEGeMybgROZg0cqesEwcVu8vtrJQ+NxTewbwHLoKAp+pBZYX1zLfQYf3t\\\\\\\/+lRMhKE+6iYIpY73ZThH+L\\\\\\\/db3BvBdPKA7xQnGZGk9VZ\\\\\\\/+Cc2sVo47sMx0YvYIHUisnakT4VOE40GKzd+YxJIMh65Lr2O5yH3JW9Q6EcKQnrgp+g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022l3GXk2svKA0A9ll2Y97JRw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224e46f39d7aae860af4d3b49c087745c139a8ebf7775cb6fed31656e6b8c1cbac\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460436 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1879" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uKadc0DATQ3q13_SpAafP0ALjFm-CVw9NltdUgIxSyc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1879" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LIpQbP4XntIbPKdT3jaok_KEKnEGrx7DD0CgSQpZ-3s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1867" title="Copy Number Profiling of Brazilian Astrocytomas" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1891" title="Cytotype Regulation Facilitates Repression of Hybrid Dysgenesis by Naturally Occurring KP Elements in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460436" id="node460436" data-pisa="ggg;6/7/1879" data-pisa-master="ggg;g3.116.028324" data-apath="/ggg/6/7/1879.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Histone H1 Limits DNA Methylation in <em>Neurospora crassa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael Seymour</span>, <span class="highwire-citation-author" data-delta="1">Lexiang Ji</span>, <span class="highwire-citation-author" data-delta="2">Alex M. Santos</span>, <span class="highwire-citation-author" data-delta="3">Masayuki Kamei</span>, <span class="highwire-citation-author" data-delta="4">Takahiko Sasaki</span>, <span class="highwire-citation-author" data-delta="5">Evelina Y. Basenko</span>, <span class="highwire-citation-author" data-delta="6">Robert J. Schmitz</span>, <span class="highwire-citation-author" data-delta="7">Xiaoyu Zhang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-1735-8266" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zachary A. Lewis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1879-1889;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028324 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460436" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael Seymour </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSeymour%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seymour%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BSeymour%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lexiang Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Bioinformatics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lexiang%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALexiang%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alex M. Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BM.%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santos%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlex%2BM.%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Masayuki Kamei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Masayuki%2BKamei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kamei%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMasayuki%2BKamei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Takahiko Sasaki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takahiko%2BSasaki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sasaki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATakahiko%2BSasaki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Evelina Y. Basenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evelina%2BY.%2BBasenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Basenko%20EY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvelina%2BY.%2BBasenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Robert J. Schmitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BSchmitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmitz%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BSchmitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xiaoyu Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoyu%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoyu%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Zachary A. Lewis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BA.%2BLewis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lewis%20ZA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZachary%2BA.%2BLewis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1735-8266" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zachary A. Lewis</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zlewis@uga.edu" class="" data-icon-position="" data-hide-link-title="0">zlewis@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/7/1879" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460436" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460436/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1879.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460436" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460436/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1879.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460436" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460436/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1879.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460436" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460436/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/7/1879.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460436" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460436/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Histone H1 variants, known as linker histones, are essential chromatin components in higher eukaryotes, yet compared to the core histones relatively little is known about their <em>in vivo</em> functions. The filamentous fungus <em>Neurospora crassa</em> encodes a single H1 protein that is not essential for viability. To investigate the role of <em>N. crassa</em> H1, we constructed a functional FLAG-tagged H1 fusion protein and performed genomic and molecular analyses. Cell fractionation experiments showed that H1-3XFLAG is a chromatin binding protein. Chromatin-immunoprecipitation combined with sequencing (ChIP-seq) revealed that H1-3XFLAG is globally enriched throughout the genome with a subtle preference for promoters of expressed genes. In mammals, the stoichiometry of H1 impacts nucleosome repeat length. To determine if H1 impacts nucleosome occupancy or nucleosome positioning in <em>N. crassa</em>, we performed micrococcal nuclease digestion in the wild-type and the <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-1.gif"/></noscript></span></span><em>hH1</em> strain followed by sequencing (MNase-seq). Deletion of <em>hH1</em> did not significantly impact nucleosome positioning or nucleosome occupancy. Analysis of DNA methylation by whole-genome bisulfite sequencing (MethylC-seq) revealed a modest but global increase in DNA methylation in the <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-2.gif"/></noscript></span></span><em>hH1</em> mutant. Together, these data suggest that H1 acts as a nonspecific chromatin binding protein that can limit accessibility of the DNA methylation machinery in <em>N. crassa</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/chromatin" class="hw-term hw-article-keyword hw-article-keyword-chromatin" rel="nofollow">chromatin</a></li><li class="kwd"><a href="/keyword/histone-h1" class="hw-term hw-article-keyword hw-article-keyword-histone-h1" rel="nofollow">histone H1</a></li><li class="kwd"><a href="/keyword/nucleosome-positioning" class="hw-term hw-article-keyword hw-article-keyword-nucleosome-positioning" rel="nofollow">nucleosome positioning</a></li><li class="kwd"><a href="/keyword/dna-methylation" class="hw-term hw-article-keyword hw-article-keyword-dna-methylation" rel="nofollow">DNA methylation</a></li></ul><p id="p-4">In eukaryotes, packaging of genomic DNA into chromatin is essential for genome function. The most basic unit of the chromatin fiber is the nucleosome core particle (NCP), made up of <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-3.gif"/></noscript></span></span>146 bp of DNA wrapped around a core of histones H3, H4, H2A, and H2B (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kornberg and Thomas 1974</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Olins and Olins 1974</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Luger <em>et al.</em> 1997</a>). In addition to the core histones, many organisms encode one or more H1 proteins, also known as linker histones. H1 proteins are evolutionarily unrelated to the core histones and are characterized by a central winged helix domain, or globular domain, flanked by unstructured N- and C-termini (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cerf <em>et al.</em> 1993</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ramakrishnan <em>et al.</em> 1993</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kasinsky <em>et al.</em> 2001</a>). Early studies showed that animal H1 proteins bind outside of the NCP (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Baldwin <em>et al.</em> 1975</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Shaw <em>et al.</em> 1976</a>) and can protect an additional 20 bp of DNA from nuclease digestion (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Whitlock and Simpson 1976</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Noll and Kornberg 1977</a>). Subsequent studies revealed that H1 binds near the NCP dyad axis and can interact with DNA as it enters and exits the NCP (recently reviewed in <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bednar <em>et al.</em> 2016</a>). Although the interactions between H1 and the NCP have been extensively investigated, H1’s roles in the cell remain poorly understood.</p><p id="p-5"><em>In vivo</em> studies of mammalian H1 are complicated by the existence of 11 H1 variants that appear to be partially redundant (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pan and Fan 2015</a>). Deletion of single H1 variants failed to produce significant phenotypes in mice (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fan <em>et al.</em> 2001</a>), but mice lacking three H1 variants are inviable (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fan <em>et al.</em> 2003</a>) and triple-knockout embryonic stem cells (ESCs) are unable to differentiate (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Zhang <em>et al.</em> 2012a</a>). These and other data suggest that animal H1 variants cooperate to perform critical functions, influencing gene regulation (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fan <em>et al.</em> 2005</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2012b</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Zhang <em>et al.</em> 2012b</a>), establishment, and/or maintenance of chromatin modification patterns (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2012b</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Zhang <em>et al.</em> 2012a</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Yang <em>et al.</em> 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lu <em>et al.</em> 2013</a>) and formation of higher order chromatin structures (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Fan <em>et al.</em> 2005</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Geeven <em>et al.</em> 2015</a>).</p><p id="p-6">Less is known about the functions of H1 in other groups of organisms, but genetic studies have been carried out in a handful of microbial model systems. H1 is not essential for viability in the single-celled <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Patterton <em>et al.</em> 1998</a>) or <em>Tetrahymena thermophila</em> (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Shen <em>et al.</em> 1995</a>). Similarly, H1-deficient mutants are viable in several filamentous fungi including <em>Neurospora crassa</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Folco <em>et al.</em> 2003</a>), <em>Aspergillus nidulans</em> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ramón <em>et al.</em> 2000</a>), and <em>Ascobolus immersus</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barra <em>et al.</em> 2000</a>). The yeast H1 homolog Hho1p suppresses homologous recombination (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Downs <em>et al.</em> 2003</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Li <em>et al.</em> 2008</a>), impacts ribosomal RNA processing (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Levy <em>et al.</em> 2008</a>), and influences chromatin compaction during stationary phase (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Schäfer <em>et al.</em> 2008</a>). In <em>T. thermophila</em>, H1 is required for normal chromatin compaction in macronuclei and influences expression of a small number of genes (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Shen <em>et al.</em> 1995</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Shen and Gorovsky 1996</a>). It is important to note that both yeast Hho1p and <em>T. thermophila</em> H1 have atypical protein structures. The yeast protein contains two globular domains, whereas the <em>T. thermophila</em> protein lacks a globular domain altogether. Thus, it is not clear if these proteins are functionally analogous to H1 in other organisms.</p><p id="p-7">The filamentous fungi <em>N. crassa</em>, <em>A. nidulans</em>, and <em>A. immersus</em> encode H1 proteins with a canonical tripartite structure, raising the possibility that these genetic systems can be used to gain insights into H1 function in plants and animals. In <em>N. crassa</em>, an <em>hH1</em>-deficient strain displayed reduced growth and H1 was required for repression of the <em>cfp</em> gene in the presence of ethanol (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Folco <em>et al.</em> 2003</a>). In <em>A. immersus</em>, <em>H1</em> gene silencing led to increased nuclease accessibility and increased DNA methylation (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Barra <em>et al.</em> 2000</a>). In contrast, deletion of <em>hhoA</em> in <em>A. nidulans</em> failed to produce a phenotype (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Ramón <em>et al.</em> 2000</a>). In general, the functions of H1 in filamentous fungi remain poorly understood. Moreover, it is not clear if H1 plays similar roles in fungal and animal cells. In the present study, we utilized molecular and genomic approaches to investigate the functions of H1 in the model fungus <em>N. crassa</em>. We confirmed that <em>N. crassa</em> H1 is a chromatin component <em>in vivo</em>, and we found that H1 is not a major determinant of nucleosome positioning, nucleosome repeat length, or nucleosome occupancy. We report that an H1 fusion protein exhibits enhanced enrichment at nucleosome-free regions and is depleted from coding sequences of expressed genes in a ChIP-seq assay. We also show that loss of H1 causes a global increase in DNA methylation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains, growth media, and molecular analyses</h3><p id="p-8">All <em>Neurospora</em> strains used in this study are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Knockout strains of <em>hH1</em> were generated by the <em>N. crassa</em> gene knockout consortium (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Colot <em>et al.</em> 2006</a>) and obtained from the Fungal Genetics Stock Center (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">McCluskey <em>et al.</em> 2010</a>). <em>Neurospora</em> cultures were grown at 32° in Vogel’s minimal medium (VMM) + 1.5% sucrose (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Davis <em>et al.</em> 1970</a>). Crosses were performed on modified synthetic cross medium at 25° (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Davis <em>et al.</em> 1970</a>). For plating assays, <em>N. crassa</em> conidia were plated on VMM with 2.0% sorbose, 0.5% fructose, and 0.5% glucose. When relevant, plates included 200 μg/ml hygromycin or 400 μg/ml basta (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Pall 1993</a>). <em>N. crassa</em> transformation (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Margolin <em>et al.</em> 1997</a>), DNA isolation (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Pomraning <em>et al.</em> 2009</a>), protein isolation, and western blotting (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Honda and Selker 2008</a>) were performed as previously described. 3X-FLAG knock in constructs were made by introduction of linear DNA fragments constructed by overlapping PCR using described plasmid vectors (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Honda and Selker 2009</a>) and <em>N. crassa</em> fragments generated using the following primers: H1 CDS FP 5′-GAG GTC GAC GGT ATC GAT AAG CTT AT ATC CAC CGA CAA CAT GTT CGA CTC-3′; H1 CDS RP 5′-CCT CCG CCT CCG CCT CCG CCT CCG CCT GCC TTC TCG GCA GCG GGC TC-3′; H1 UTR FP 5′-TGC TAT ACG AAG TTA TGG ATC CGA GCT CGA CTC GTT CCT TTG GGA TGA T-3′; H1 UTR RP 5′-ACC GCG GTG GCG GCC GCT CTA GAA CTA GTT CAT CAA ACC AAA TTC TCG G-3′. To separate soluble nuclear proteins from the chromatin fraction, cultures were grown overnight and cells were collected, ground in liquid nitrogen, and resuspended in 1 ml of low salt extraction buffer [50 mM HEPES-KOH pH 7.5, 150 mM NaCl, 2 mM EDTA, plus protease inhibitor tablets (cat # 11836153001; Roche, Indianapolis, IN)]. Extracts were centrifuged at 14,000 rpm and the supernatant containing soluble proteins was saved. The pellet was resuspended in 1 ml of high salt extraction buffer [50 mM HEPES-KOH pH 7.5, 600 mM NaCl, 2 mM EDTA, plus protease inhibitor tablets (cat # 11836153001; Roche, Indianapolis, IN)] and subjected to sonication. Extracts were centrifuged at 14,000 rpm in a microfuge and the supernatant was saved as the chromatin fraction. Both fractions were analyzed by western blotting using anti-FLAG antibodies (cat # F1804; Sigma-Aldrich) and anti-H3 antibodies (cat # 06-755 Millipore).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468738/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468738/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Strains used in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>DNA sequencing and data analysis</h3><div id="sec-4" class="subsection"><h4>ChIP-seq:</h4><p id="p-9">For chromatin immunoprecipitation (ChIP) experiments, 5 × <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-6.gif"/></noscript></span></span> conidia/ml were inoculated into 50 ml of VMM and incubated at 32° for 5 hr. Cells were harvested and ChIP was performed as previously described (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Sasaki <em>et al.</em> 2014</a>) using anti-FLAG antibodies (cat # F1804; Sigma-Aldrich) or antibodies to the unphosphorylated C-terminal repeat of RNA polymerase II (8WG16; cat# MMS-126R; Covance). Two biological replicates were performed for each experiment. Libraries were prepared using the TruSeq ChIP sample prep kit (Illumina, cat # IP-202-1012) according to the manufacturer’s instructions with the following modification. Library amplification was performed using only four cycles of PCR to reduce biased enrichment of GC-rich DNA (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ji <em>et al.</em> 2014</a>). Libraries were sequenced at the University of Georgia Genomics Facility on an Illumina NextSeq 500 instrument. Reads were aligned to version 12 of the <em>N. crassa</em> genome (Refseq Accession # GCF_000182925.2; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Galagan <em>et al.</em> 2003</a>) using the Burrows-Wheeler Aligner (BWA version 0.7.10) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Li and Durbin 2009</a>). To determine if H1-3XFLAG was enriched over background, coverage was normalized to mitochondrial DNA as follows. We used BEDtools (version 2.25.0) ‘coverage’ to calculate read coverage for 1000 bp windows across the genome (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Quinlan and Hall 2010</a>). We then used BEDtools ‘map’ to calculate the median coverage for mitochondrial DNA. The coverage for each 1000 bp window was then divided by the median coverage for mitochondrial DNA. As a positive control, data from a previously published ChIP experiment for methylated lysine-9 of H3 was analyzed (Accession #SRX550120; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Sasaki <em>et al.</em> 2014</a>).</p><p id="p-10">The Hypergeometric Optimization of the Motif EnRichment (HOMER version 4.7.2) software package (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Heinz <em>et al.</em> 2010</a>) was used to generate metaplots and heatmaps of enrichment data (annotatePeaks.pl module; using the -hist and -ghist option, respectively). We first created a custom HOMER genome annotation for <em>Neurospora</em> using a fasta file and a GTF file (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028324/-/DC1/FileS1.zip">File S1</a>) containing the version 12 genome assemblies and annotations, respectively (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Galagan <em>et al.</em> 2003</a>). All plots were centered on transcriptional start sites or transcriptional termination sites, and a window size of 10 bp was specified for all histograms (-hist 10). HOMER was also used to construct metaplots of expression-ranked gene groups using the -list option. Genes were assigned into expression-ranked groups by expression level determined by RNA-seq (see below). RPKM values for each quintile group were: Q1 (12927.9–36.8); Q2 (36.8–13.8); Q3 (13.8–4.9); Q4 (4.9–0.33); Q5 (0.33–0). Thus, most genes in expression group 5 were silent or rarely expressed.</p></div><div id="sec-5" class="subsection"><h4>MNase digestion:</h4><p id="p-11">For micrococcal nuclease (MNase) experiments, 5 ×<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-7.gif"/></noscript></span></span> conidia/ml were inoculated into 50 ml of VMM and incubated at 32° for 5 hr. The cell suspension was transferred to a 50 ml conical flask and centrifuged at 1000 × <em>g</em> for 5 min to pellet germinated conidia. Cell pellets were washed with 10 ml Phosphate Buffered Saline (PBS) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sambrook <em>et al.</em> 1989</a>) and then resuspended in 10 ml of PBS containing 1% formaldehyde. The cell suspension was transferred to a 125 ml flask and incubated with gentle shaking for 30 min at room temperature before the cross-linking agent was quenched by addition of 500 ml 2.5 M glycine. The cell suspension was transferred to a 50 ml conical tube and germinating conidia were pelleted by centrifugation for 5 min at 1000 × <em>g</em>. Cells were washed once in 40 ml of PBS and resuspended in 1 ml of ice-cold PBS. Cells were pelleted by centrifugation for 5 min at 5000 × <em>g</em>, and each cell pellet was resuspended in NPS buffer with calcium chloride [50 mM HEPES-KOH, pH 7.5, 140 mM NaCl, 5 mM MgCl<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-8.gif"/></noscript></span></span>, 1 mM CaCl<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-9.gif"/></noscript></span></span>, 1% Triton-X 100, 0.1% deoxycholate, 0.5 mM spermidine, 1 mM PMSF plus Roche EDTA-free protease inhibitor tablets (catalog # 05892791001)]. Cells were then lysed by gentle sonication using a Heat Systems, Ultrasonics W-380 sonicator with micro tip (Output 2.0, Duty Cycle 80%; 30 1 sec pulses). The chromatin fraction was pelleted by centrifugation for 5 min at 14,000 × <em>g</em>. The supernatent was discarded and each pellet was resuspended in 1 ml of NPS buffer and transferred to a 15 ml conical tube. NPS buffer with calcium chloride was added to raise the volume to 6 ml and the chromatin sample was mixed by pipetting. Then, 700 μl aliquots were transferred to 1.5 ml tubes and 2 units of micrococcal nuclease (cat #2910A; Clontech) were added to each tube. Individual samples were incubated at 37° for 5, 10, 20, 40, or 60 min. MNase digestions were stopped by addition of 15 μl 0.5M EDTA and 25 μl 4M NaCl, and samples were incubated overnight at 65° to reverse cross-links. A total of 6 μl RNase A (10 mg/ml; Fisher Scientific, cat # BP2529250) was added and samples were incubated for 2 hr at 50°. A total of 6 μl of 10% SDS and 10 μl Proteinase K (10 mg/ml; Fisher Scientific, cat # BP1700) was then added and samples were incubated for 2 hr at 65°. The digested DNA was isolated by phenol-chloroform extraction and precipitated overnight at −20° in ethanol and sodium acetate (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Sambrook <em>et al.</em> 1989</a>). Digested DNA was resolved by gel electrophoresis to confirm that digestion was successful.</p></div><div id="sec-6" class="subsection"><h4>MNase-seq:</h4><p id="p-12">We constructed sequencing libraries from mononucleosomes generated by 20 or 60 min MNase digestion. We first performed a gel extraction (Qiagen, cat #28706) of mononucleosomal DNA (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-10.gif"/></noscript></span></span>150 bp) and constructed libraries using an Illumina TruSeq Sample preparation kit (Illumina) according to the manufacturer’s instructions. A total of 50 bp paired-end sequencing reads were generated on an Illumina HiSeq 2500 instrument at the Oregon State University genomics core facility. Due to a technical problem during the sequencing run, only 44 bp of sequence were obtained for the read 2 sequence. Sequences were mapped to the <em>N. crassa</em> version 12 genome assembly (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Galagan <em>et al.</em> 2003</a>) using bowtie2 (version 2.2.3) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Langmead and Salzberg 2012</a>). To analyze the size distributions in the wild-type and the <em>hH1</em> strain, the Picard software package (<a href="http://broadinstitute.github.io/picard">http://broadinstitute.github.io/picard</a>) was used to remove duplicate reads and determine insert size metrics (using ‘CollectInsertSizeMetrics’). HOMER was used to create metaplots of MNase data as described above. In all cases, a window size of 10 bp was used (-hist 10). Metaplots depict only plus strand reads (using the ‘-strand +’ option) and thus peaks indicate the left edge of nucleosomes.</p></div><div id="sec-7" class="subsection"><h4>RNA-seq:</h4><p id="p-13">For RNA-seq experiments, 5 × 10<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-11.gif"/></noscript></span></span> conidia/ml were inoculated into 50 ml of VMM containing 2% glucose and grown for 5 hr at 32°. RNA isolation was performed as described (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bell-Pedersen <em>et al.</em> 1996</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Schwerdtfeger and Linden 2001</a>), and strand-specific RNA-seq libraries were prepared from 5 μg total RNA. Ribosomal RNAs were depleted using the yeast Ribo-zero kit (cat # MRZY1324 Epicentre) and RNA libraries were generated with the Illumina Stranded RNA-seq kit (cat # RS-122-2101). Reads were aligned to version 12 of the <em>N. crassa</em> genome sequence using TopHat (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Trapnell <em>et al.</em> 2009</a>) and expression levels (RPKM) were determined using Cufflinks (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Trapnell <em>et al.</em> 2012</a>).</p></div><div id="sec-8" class="subsection"><h4>MethylC-seq:</h4><p id="p-14">For DNA methylation analysis, conidia were inoculated into 5 ml of VMM and cultures were grown for 48 hr at 32°. Genomic DNA was isolated using described procedures (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Pomraning <em>et al.</em> 2009</a>). MethylC-seq libraries were prepared as previously described (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Urich <em>et al.</em> 2015</a>). Illumina sequencing was performed at the University of Georgia Genomics Facility using an Illumina NextSeq 500 instrument. Sequencing reads were trimmed for adapters, preprocessed to remove low quality reads, and aligned to the <em>N. crassa</em> version 12 genome assembly (<a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Galagan <em>et al.</em> 2003</a>) as described in <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Schmitz <em>et al.</em> (2013)</a>. Mitochondrial DNA sequence (which is fully unmethylated) was used as a control to calculate the sodium bisulfite reaction nonconversion rate of unmodified cytosines. Only cytosine sites with a minimum coverage (set as 3) were allowed for subsequent analysis. Binomial test coupled with Benjamini-Hochberg correction was adopted to determine the methylation status of each cytosine. Identification of DMRs (Differentially Methylated Regions) was performed as described (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Schultz <em>et al.</em> 2015</a>). Methylated regions in the wild-type were defined previously (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Basenko <em>et al.</em> 2016</a>). For metaplots, both upstream and downstream regions were divided into 20 bins each of 50 bp in length for a total 1 kb in each direction. Methylated regions were separated every 5%, for a total of 20 bins. Weighted methylation levels were computed for each bin as described previously (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Schultz <em>et al.</em> 2012</a>).</p></div></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-15">All strains are listed in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and available upon request or from the Fungal Genetics Stock Center (Manhattan, KS). All sequencing data have been deposited into the NCBI SRA/GEO databases. ChIP-seq, MNase-Seq, and RNA-seq data generated during this study have been deposited under accession #GSE78157. Control data from a previously published ChIP-seq experiment for methylation of H3 lysine-9 was deposited under accession #SRX550120 (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Sasaki <em>et al.</em> 2014</a>). MethylC-seq data are deposited under accession #GSE76982 (this study) and #GSE70518 (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Basenko <em>et al.</em> 2016</a>).</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Construction of an H1-3XFLAG fusion protein</h3><p id="p-16">To investigate the role of H1 in <em>N. crassa</em> cells, we constructed an epitope-tagged version of the protein by introducing coding sequence for a 3X-FLAG tag at the 3′ end of the native <em>hH1</em> locus (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Primary transformants were crossed to obtain a homokaryon that was analyzed further. To confirm that the H1-3XFLAG fusion protein is functional, we first compared the growth rate of the <em>hH1</em>::<em>hH1-3xflag-Hph+</em> strain to the wild-type and to an <em>hH1</em> deletion strain obtained from the <em>Neurospora</em> gene knockout consortium (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Colot <em>et al.</em> 2006</a>). The <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-12.gif"/></noscript></span></span><em>hH1</em> strain displayed a reduced growth rate, as reported previously for an H1 loss-of-function allele generated by repeat-induced point mutation (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Folco <em>et al.</em> 2003</a>). The <em>hH1</em>::<em>hH1-3xflag-Hph+</em> grew similar to the wild-type (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1, B and C</a>). We also asked if the H1-3XFLAG protein associates with chromatin. We isolated soluble and chromatin-containing fractions (see <em>Materials and Methods</em>) and performed western blot analyses using anti-FLAG and anti-H3 antibodies. Western blots probed with an anti-FLAG antibody revealed a single band. The apparent molecular weight was larger than expected based on amino acid sequence prediction, but the apparent size was consistent with previous analysis of <em>N. crassa</em> purified by extraction with perchloric acid (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Folco <em>et al.</em> 2003</a>). We detected H1-3XFLAG in both soluble and chromatin fractions with higher levels of H1-3XFLAG observed in the chromatin fraction (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1D</a>). As expected, H3 was exclusively detected in the chromatin fraction. Together, these data demonstrate that the H1-3XFLAG construct is functional and that <em>N. crassa</em> H1 is a component of chromatin.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="H1-3XFLAG is functional and binds to chromatin. (A) A cartoon illustrating the strategy for introducing a 3x-flag sequence into the 3&#x2032; end of the native hH1 locus by homologous recombination is shown. (B) The linear growth rate of the indicated strains was measured using &#x2018;race tubes&#x2019; for three replicates of each strain. The direction of growth is left to right. (C) Quantification of the linear growth rate data from panel B as distance (y-axis; cm) vs. time (x-axis; hr). The three lines shown for each genotype represent three independent race tubes. (D) The soluble (Sol) and the chromatin fractions (Chr) were isolated from wild-type and the hH1-3xflag-Hph+ strain, and both fractions were analyzed by western blotting using anti-FLAG and anti-H3 antibodies, as indicated. MW, positions and sizes in kDa of a prestained protein ladder; WT, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1606160464" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;H1-3XFLAG is functional and binds to chromatin. (A) A cartoon illustrating the strategy for introducing a &lt;em&gt;3x-flag&lt;/em&gt; sequence into the 3&#x2032; end of the native &lt;em&gt;hH1&lt;/em&gt; locus by homologous recombination is shown. (B) The linear growth rate of the indicated strains was measured using &#x2018;race tubes&#x2019; for three replicates of each strain. The direction of growth is left to right. (C) Quantification of the linear growth rate data from panel B as distance (&lt;em&gt;y&lt;/em&gt;-axis; cm) &lt;em&gt;vs.&lt;/em&gt; time (&lt;em&gt;x&lt;/em&gt;-axis; hr). The three lines shown for each genotype represent three independent race tubes. (D) The soluble (Sol) and the chromatin fractions (Chr) were isolated from wild-type and the &lt;em&gt;hH1-3xflag-Hph+&lt;/em&gt; strain, and both fractions were analyzed by western blotting using anti-FLAG and anti-H3 antibodies, as indicated. MW, positions and sizes in kDa of a prestained protein ladder; WT, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1879/F1.medium.gif" width="160" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/7/1879/F1.medium.gif" width="160" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468675" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">H1-3XFLAG is functional and binds to chromatin. (A) A cartoon illustrating the strategy for introducing a <em>3x-flag</em> sequence into the 3′ end of the native <em>hH1</em> locus by homologous recombination is shown. (B) The linear growth rate of the indicated strains was measured using ‘race tubes’ for three replicates of each strain. The direction of growth is left to right. (C) Quantification of the linear growth rate data from panel B as distance (<em>y</em>-axis; cm) <em>vs.</em> time (<em>x</em>-axis; hr). The three lines shown for each genotype represent three independent race tubes. (D) The soluble (Sol) and the chromatin fractions (Chr) were isolated from wild-type and the <em>hH1-3xflag-Hph+</em> strain, and both fractions were analyzed by western blotting using anti-FLAG and anti-H3 antibodies, as indicated. MW, positions and sizes in kDa of a prestained protein ladder; WT, wild-type.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>H1 is moderately enriched in promoters and depleted from coding sequences of actively expressed genes</h3><p id="p-18">To determine the genome-wide distribution of H1-3XFLAG, we performed chromatin-immunoprecipitation followed by sequencing (ChIP-seq). Inspection of the H1-3XFLAG enrichment data in a genome browser revealed a relatively uniform distribution for all seven chromosomes. Given that high levels of H1-3XFLAG were detected in the chromatin fraction by western blotting, we reasoned that the uniform enrichment pattern observed for H1-3XFLAG might reflect global binding of H1 across the genome. To determine if this was the case, we normalized read counts obtained for each 1000 bp window in the nuclear genome to the median read count obtained for all 1000 bp windows covering the mitochondrial genome. This allowed us to calculate enrichment over background because mitochondrial DNA should not be enriched by immunoprecipitation of a nuclear protein. We plotted normalized enrichment data for all 1000 bp windows on Linkage Group VII (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). ChIP-seq experiments using anti-FLAG antibodies led to global enrichment of chromosomal DNA from the <em>hH1</em>::<em>hH1-3xflag-Hph+</em> strain but not from a wild-type negative control strain. As a positive control, we normalized read counts obtained in a previously published ChIP-seq experiment performed with antibodies to H3 methylated at lysine-9 (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Sasaki <em>et al.</em> 2014</a>). Results for other chromosomes were comparable. This confirmed that normalization to mitochondrial DNA is an effective method for quantifying enrichment over background. Moreover, these data demonstrate that H1 is a general chromatin architectural protein in <em>N. crassa</em>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="H1-3XFLAG is globally enriched throughout the N. crassa genome. Normalized read coverage is shown for all 1000 bp windows across N. crassa Linkage Group VII for anti-FLAG ChIP-seq experiments performed with a WT negative control strain (no FLAG protein) and the hH1-3xflag-Hph+ strain (two replicates for each strain are shown). Normalized read coverage is also plotted for an anti-H3K9me3 ChIP-seq experiment performed with the WT. Each spot depicts the read count for a single 1000 bp window normalized to the median enrichment obtained for mitochondrial DNA windows. ChIP-seq; Chromatin-immunoprecipitation combined with sequencing; WT, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1606160464" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;H1-3XFLAG is globally enriched throughout the &lt;em&gt;N. crassa&lt;/em&gt; genome. Normalized read coverage is shown for all 1000 bp windows across &lt;em&gt;N. crassa&lt;/em&gt; Linkage Group VII for anti-FLAG ChIP-seq experiments performed with a WT negative control strain (no FLAG protein) and the &lt;em&gt;hH1-3xflag-Hph+&lt;/em&gt; strain (two replicates for each strain are shown). Normalized read coverage is also plotted for an anti-H3K9me3 ChIP-seq experiment performed with the WT. Each spot depicts the read count for a single 1000 bp window normalized to the median enrichment obtained for mitochondrial DNA windows. ChIP-seq; Chromatin-immunoprecipitation combined with sequencing; WT, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1879/F2.medium.gif" width="440" height="381"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/7/1879/F2.medium.gif" width="440" height="381"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468703" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">H1-3XFLAG is globally enriched throughout the <em>N. crassa</em> genome. Normalized read coverage is shown for all 1000 bp windows across <em>N. crassa</em> Linkage Group VII for anti-FLAG ChIP-seq experiments performed with a WT negative control strain (no FLAG protein) and the <em>hH1-3xflag-Hph+</em> strain (two replicates for each strain are shown). Normalized read coverage is also plotted for an anti-H3K9me3 ChIP-seq experiment performed with the WT. Each spot depicts the read count for a single 1000 bp window normalized to the median enrichment obtained for mitochondrial DNA windows. ChIP-seq; Chromatin-immunoprecipitation combined with sequencing; WT, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">We did not detect prominent peaks that are typical of transcription factors or previously analyzed histone modifications such as H3K4me2 or H3K9me3 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lewis <em>et al.</em> 2009</a>). However, we did detect a subtle enrichment of H1 in the promoters of many genes along with a corresponding depletion of H1-3XFLAG within many coding sequences. To determine if this pattern occurred broadly across the <em>N. crassa</em> chromatin landscape, we created metaplots to analyze the average H1 distribution across the transcriptional start sites (TSS) or transcriptional termination sites (TTS) of all <em>N. crassa</em> genes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="H1-3XFLAG is depleted from gene bodies and modestly enriched in promoters of expressed genes. (A) Metaplots depict the average ChIP-seq enrichment pattern across all N. crassa genes for two replicate FLAG ChIP-seq experiments performed with the hH1-3xflag-Hph+ strain and a WT negative control strain (no FLAG-tagged protein). Metaplots are centered on the transcriptional start (TSS; left) or the transcriptional termination site (TTS; right). (B) All N. crassa genes were ranked by expression level and split into quintile groups. Quintile 1 (Q1) corresponds to genes with the highest expression level, whereas Q5 corresponds to genes with the lowest expression. The metaplot depicts the H1-3XFLAG enrichment pattern for each expression group across the TSS or TTS. (C) Heatmaps show the distribution of H1-3XFLAG across all N. crassa genes centered on the TSS (left) or TTS (right). Genes are ordered by expression level from highest to lowest. (D) Heatmaps show the distribution of RNA polymerase II across all N. crassa genes centered on the TSS (left) or TTS (right). Genes are ordered as in C. Data in B&#x2013;D are from replicate one. ChIP-seq; Chromatin-immunoprecipitation combined with sequencing; WT, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1606160464" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;H1-3XFLAG is depleted from gene bodies and modestly enriched in promoters of expressed genes. (A) Metaplots depict the average ChIP-seq enrichment pattern across all &lt;em&gt;N. crassa&lt;/em&gt; genes for two replicate FLAG ChIP-seq experiments performed with the &lt;em&gt;hH1-3xflag-Hph&lt;sup&gt;+&lt;/sup&gt;&lt;/em&gt; strain and a WT negative control strain (no FLAG-tagged protein). Metaplots are centered on the transcriptional start (TSS; left) or the transcriptional termination site (TTS; right). (B) All &lt;em&gt;N. crassa&lt;/em&gt; genes were ranked by expression level and split into quintile groups. Quintile 1 (Q1) corresponds to genes with the highest expression level, whereas Q5 corresponds to genes with the lowest expression. The metaplot depicts the H1-3XFLAG enrichment pattern for each expression group across the TSS or TTS. (C) Heatmaps show the distribution of H1-3XFLAG across all &lt;em&gt;N. crassa&lt;/em&gt; genes centered on the TSS (left) or TTS (right). Genes are ordered by expression level from highest to lowest. (D) Heatmaps show the distribution of RNA polymerase II across all &lt;em&gt;N. crassa&lt;/em&gt; genes centered on the TSS (left) or TTS (right). Genes are ordered as in C. Data in B&#x2013;D are from replicate one. ChIP-seq; Chromatin-immunoprecipitation combined with sequencing; WT, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1879/F3.medium.gif" width="214" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/7/1879/F3.medium.gif" width="214" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468570" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">H1-3XFLAG is depleted from gene bodies and modestly enriched in promoters of expressed genes. (A) Metaplots depict the average ChIP-seq enrichment pattern across all <em>N. crassa</em> genes for two replicate FLAG ChIP-seq experiments performed with the <em>hH1-3xflag-Hph<sup>+</sup></em> strain and a WT negative control strain (no FLAG-tagged protein). Metaplots are centered on the transcriptional start (TSS; left) or the transcriptional termination site (TTS; right). (B) All <em>N. crassa</em> genes were ranked by expression level and split into quintile groups. Quintile 1 (Q1) corresponds to genes with the highest expression level, whereas Q5 corresponds to genes with the lowest expression. The metaplot depicts the H1-3XFLAG enrichment pattern for each expression group across the TSS or TTS. (C) Heatmaps show the distribution of H1-3XFLAG across all <em>N. crassa</em> genes centered on the TSS (left) or TTS (right). Genes are ordered by expression level from highest to lowest. (D) Heatmaps show the distribution of RNA polymerase II across all <em>N. crassa</em> genes centered on the TSS (left) or TTS (right). Genes are ordered as in C. Data in B–D are from replicate one. ChIP-seq; Chromatin-immunoprecipitation combined with sequencing; WT, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">On average, H1-3XFLAG was enriched upstream of the TSS and was depleted from gene bodies. Similar enrichment patterns were observed for both biological replicates, but not in a negative control experiment in which we used anti-FLAG antibodies to perform ChIP-seq in a wild-type strain (no FLAG-tagged protein). A slight depletion of reads just upstream of the TSS was observed in negative control experiments. Given that these regions are typically depleted for nucleosomes (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sancar <em>et al.</em> 2015</a>), this result may indicate that the FLAG antibody exhibits weak background binding to nucleosomal DNA. We next asked if H1-3XFLAG enrichment was correlated with the level of transcription. We binned <em>Neurospora</em> genes into five groups based on expression level and constructed metaplots to visualize the average H1-3XFLAG distribution pattern for each group (see <em>Materials and Methods</em>). The level of enrichment in gene promoters was positively correlated with the level of transcription (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Similarly, depletion of H1-3XFLAG from coding regions was correlated with the level of transcription. To explore this further, we performed ChIP-seq for RNA polymerase II and constructed heatmaps of H1-3XFLAG and RNA polymerase II occupancy across all <em>Neurospora</em> genes ordered by expression level (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, C and D</a>). These data further support the idea that H1-3XFLAG enrichment is highest in the promoters and lowest in the coding sequences of highly expressed genes. We did not detect significant enrichment or depletion of H1-3XFLAG in heterochromatin domains.</p></div><div id="sec-13" class="subsection"><h3>H1 is not a major determinant of nucleosome positioning in N. crassa</h3><p id="p-23">Given that H1-3XFLAG displayed a subtle preference for promoters of actively expressed genes, we hypothesized that H1 may impact chromatin structure at promoters. Like many eukaryotes, <em>N. crassa</em> promoters are characterized by nucleosome-free regions (NFRs) upstream of the TSS followed by well positioned “+1” nucleosomes (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Sancar <em>et al.</em> 2015</a>). To determine if H1 is important for the establishment of this characteristic promoter structure, and to determine if <em>N. crassa</em> H1 is important for nucleosome occupancy or nucleosome positioning at specific sites in the genome, we performed micrococcal nuclease digestion followed by sequencing (MNase-seq). Nuclei were incubated with MNase for increasing times (up to 60 min) before the DNA from each digestion reaction was purified and resolved on an agarose gel. Wild-type and <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-13.gif"/></noscript></span></span><em>hH1</em> showed similar global MNase sensitivity across multiple experiments (representative gels are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a>), with slightly enhanced MNase accessibility observed in the H1-deficient strain treated with enzyme for 5 or 10 min. This subtle increase in MNase accessibility in the <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-14.gif"/></noscript></span></span><em>H1</em> is in agreement with previously published work (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Folco <em>et al.</em> 2003</a>). We next performed gel extractions to isolate the DNA band corresponding to mononucleosomes (∼150 bp) from samples digested with MNase for 20 or 60 min and constructed Illumina sequencing libraries. For each digestion time, libraries were prepared from two independent biological replicates (four samples in total for each strain) and paired-end sequencing was performed. We observed no differences in the average length of the mononucleosomal fragments in wild-type and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-15.gif"/></noscript></span></span><em>hH1</em> (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). As expected, longer digestion times produced shorter DNA fragments (compare 20 min to 60 min digestion times).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="H1 is not a major determinant of nucleosome size or positioning in Neurospora. (A) Wild-type or (B) hH1 nuclei were isolated and treated with micrococcal nuclease (MNase) for the indicated times (min). DNA was purified, resolved on an agarose gel, and visualized by staining with Ethidium Bromide. Wild-type and hH1 display similar digestion kinetics. (C) Metaplots depict the average sequencing depth on the + strand across all N. crassa genes obtained by sequencing mononucleosomal DNA from a 60 min MNase digest. Plots show data from two biological replicates each for WT and hH1. (D&#x2013;H) All N. crassa genes were ranked by expression level and split into quintile groups, ranging from highest expression (Quintile 1) to lowest expression (Quintile 5). Metaplots depict the average sequencing coverage on the + strand for each group. Plots show data from two biological replicates each for WT and hH1. The legend in the top right panel corresponds to plots (C&#x2013;H). For all plots, the coverage of + strand reads from an H1-3XFLAG ChIP-seq experiment is shown as a dashed line. ChIP-seq; Chromatin-immunoprecipitation combined with sequencing; WT, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1606160464" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;H1 is not a major determinant of nucleosome size or positioning in &lt;em&gt;Neurospora&lt;/em&gt;. (A) Wild-type or (B) &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-16&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;hH1&lt;/em&gt; nuclei were isolated and treated with micrococcal nuclease (MNase) for the indicated times (min). DNA was purified, resolved on an agarose gel, and visualized by staining with Ethidium Bromide. Wild-type and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-17&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;hH1&lt;/em&gt; display similar digestion kinetics. (C) Metaplots depict the average sequencing depth on the + strand across all &lt;em&gt;N. crassa&lt;/em&gt; genes obtained by sequencing mononucleosomal DNA from a 60 min MNase digest. Plots show data from two biological replicates each for WT and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-18&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;hH1&lt;/em&gt;. (D&#x2013;H) All &lt;em&gt;N. crassa&lt;/em&gt; genes were ranked by expression level and split into quintile groups, ranging from highest expression (Quintile 1) to lowest expression (Quintile 5). Metaplots depict the average sequencing coverage on the + strand for each group. Plots show data from two biological replicates each for WT and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-19&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;hH1&lt;/em&gt;. The legend in the top right panel corresponds to plots (C&#x2013;H). For all plots, the coverage of + strand reads from an H1-3XFLAG ChIP-seq experiment is shown as a dashed line. ChIP-seq; Chromatin-immunoprecipitation combined with sequencing; WT, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1879/F4.medium.gif" width="440" height="340"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/7/1879/F4.medium.gif" width="440" height="340"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468668" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">H1 is not a major determinant of nucleosome size or positioning in <em>Neurospora</em>. (A) Wild-type or (B) <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F4/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F4/embed/mml-math-16.gif"/></noscript></span></span><em>hH1</em> nuclei were isolated and treated with micrococcal nuclease (MNase) for the indicated times (min). DNA was purified, resolved on an agarose gel, and visualized by staining with Ethidium Bromide. Wild-type and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F4/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F4/embed/mml-math-17.gif"/></noscript></span></span><em>hH1</em> display similar digestion kinetics. (C) Metaplots depict the average sequencing depth on the + strand across all <em>N. crassa</em> genes obtained by sequencing mononucleosomal DNA from a 60 min MNase digest. Plots show data from two biological replicates each for WT and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F4/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F4/embed/mml-math-18.gif"/></noscript></span></span><em>hH1</em>. (D–H) All <em>N. crassa</em> genes were ranked by expression level and split into quintile groups, ranging from highest expression (Quintile 1) to lowest expression (Quintile 5). Metaplots depict the average sequencing coverage on the + strand for each group. Plots show data from two biological replicates each for WT and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F4/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F4/embed/mml-math-19.gif"/></noscript></span></span><em>hH1</em>. The legend in the top right panel corresponds to plots (C–H). For all plots, the coverage of + strand reads from an H1-3XFLAG ChIP-seq experiment is shown as a dashed line. ChIP-seq; Chromatin-immunoprecipitation combined with sequencing; WT, wild-type.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468585/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468585/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Paired-end read insert size</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">We next asked if H1 impacted the occupancy and/or positioning of nucleosomes in promoters or gene coding sequences. We created metaplots to analyze the average nucleosome distribution across all <em>N. crassa</em> genes (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4C</a>). In this case, only plus strand reads were plotted to generate peaks corresponding to the edge of each nucleosome. The average gene profile revealed a characteristic nucleosome-free region upstream of the TSS, as reported previously for <em>N. crassa</em> (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Sancar <em>et al.</em> 2015</a>). The average nucleosome position profiles were similar in the wild-type and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-33.gif"/></noscript></span></span><em>hH1</em>. Because H1 was most enriched in the promoters of highly expressed genes, we next plotted average MNase-seq enrichment profiles for genes grouped by expression level. Although we detected a subtle increase in the size of the NFR from the most expressed genes in the two 60 min digestion samples, this difference was not apparent in the samples subjected to a 20 min MNase digestion (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028324/-/DC1/FigureS1.pdf">Figure S1</a>). The difference was also not apparent in a third independent replicate subjected to single-end Illumina sequencing (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028324/-/DC1/FigureS1.pdf">Figure S1</a>). Thus, we conclude that H1 is not a major determinant of nucleosome positioning or occupancy in <em>Neurospora</em>. Because it appeared that enrichment of H1-3XFLAG was inversely correlated with nucleosome occupancy, we plotted plus strand reads from replicate one of the H1-3XFLAG ChIP-seq experiment described above to allow direct comparison of nucleosome occupancy and H1-3XFLAG enrichment (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, C–H</a>). This confirmed that the highest levels of H1 enrichment correspond to sites with the lowest nucleosome occupancy, raising the possibility that H1-3XFLAG binds to DNA that is free of nucleosomes as well as to nucleosomal DNA in <em>N. crassa</em>. The group of genes including silent or rarely expressed genes (Quintile 5) did not exhibit an obvious nucleosome-free region or an ordered array of nucleosomes. This may suggest that transcription is important for establishment of chromatin architecture at these genes. However, we cannot rule out the possibility that this result is due to incorrect annotation of the TSS for many genes in this group.</p></div><div id="sec-14" class="subsection"><h3><em><span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-34.gif"/></noscript></span></span></em>hH1 exhibits increased DNA methylation in N. crassa heterochromatin domains</h3><p id="p-27">H1 impacts DNA methylation levels in animals (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Fan <em>et al.</em> 2005</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Zhang <em>et al.</em> 2012a</a>), plants (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Wierzbicki and Jerzmanowski 2005</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Zemach <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Rea <em>et al.</em> 2012</a>), and the fungus <em>A. immersus</em> (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Barra <em>et al.</em> 2000</a>). It was previously reported that H1 did not impact DNA methylation in <em>N. crassa</em> (<a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Folco <em>et al.</em> 2003</a>); however, Folco and colleagues analyzed only a single methylated region. It remained possible that H1 impacts DNA methylation in a region-specific manner or, alternatively, that H1 has a subtle impact on DNA methylation that may have been overlooked. We performed MethylC-seq to analyze DNA methylation across the entire genome at single base pair resolution. Genomic DNA was isolated from two replicates each of a wild-type <em>mat A</em> strain, the <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-35.gif"/></noscript></span></span><em>hH1</em> mutant, and a negative control <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-36.gif"/></noscript></span></span><em>dim-2</em> strain, which lacks DNA methylation altogether (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Kouzminova and Selker 2001</a>). All strains were grown simultaneously, but the wild-type and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-37.gif"/></noscript></span></span><em>dim-2</em> data were published previously as part of another study (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Basenko <em>et al.</em> 2016</a>).</p><p id="p-28">To determine if H1 impacts the level or distribution of DNA methylation in <em>N. crassa</em>, we first plotted DNA methylation levels across Linkage Group VII, a 4 Mb chromosome corresponding to <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-38.gif"/></noscript></span></span>10% of the genome. The overall pattern of DNA methylation was similar in the wild-type and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-39.gif"/></noscript></span></span><em>hH1</em> strains. However, we noted that DNA methylation levels were higher in the <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-40.gif"/></noscript></span></span><em>hH1</em> mutant at most regions along the chromosome (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). We next constructed metaplots to quantify the average methylation level for all genomic regions that are normally methylated in the wild-type <em>mat A</em> strain (see <em>Materials and Methods</em>). Both <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-41.gif"/></noscript></span></span><em>hH1</em> replicates displayed higher average DNA methylation levels when compared to the wild-type strain (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). To confirm that this was not an artifact due to differences in strain backgrounds, we compared the level of methylation in the <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-42.gif"/></noscript></span></span><em>hH1</em> strain to a wild-type <em>mat a</em> strain, and analyzed DNA methylation levels in a second <em>hH1</em> loss of function strain in which the <em>hH1</em> gene was inactivated by repeat-induced point mutation (<a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">Folco <em>et al.</em> 2003</a>). In both cases, the level of DNA methylation was higher in the <em>hH1</em> mutant strains (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, C and D</a>). A search for differentially methylated regions between wild-type and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-43.gif"/></noscript></span></span><em>H1</em> identified only a single methylated region that was specific to the <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-44.gif"/></noscript></span></span><em>H1</em> strain (Linkage Group VI; 309,590–319,133). This hypermethylated sequence corresponds to <em>Sly1-1</em>, a DNA transposon found in the genomes of some <em>N. crassa</em> isolates (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Wang <em>et al.</em> 2015</a>). It is possible that the observed increase in DNA methylation at the <em>Sly1-1</em> locus is due to a site-specific increase in DNA methylation. However, it is also possible that this is actually due to a change in copy number or location of the transposon in the <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-45.gif"/></noscript></span></span><em>H1</em> strain background. Additional work is needed to test these possibilities.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Increased DNA methylation is observed at most N. crassa heterochromatin domains in H1-deficient strains. (A) The DNA methylation level (weighted DNA methylation level (%); (see Materials and Methods) is shown for 10 kb windows across N. crassa Linkage Group VII for wild-type, the hH1 mutant, and the dim-2 strain, which lacks all DNA methylation. (B&#x2013;D) The metaplots show the average DNA methylation level across all previously identified wild-type methylated domains: (B) the wild-type mat A strain and the hH1 strain; (C) the wild-type mat a strain and the hH1 strain; and (D) the wildtype mat A strain and the hH1RIP2 strain. Data for at least two independent biological replicates of each strain are shown. (E) More cytosines are methylated in hH1. The plot shows the total number of methylated cytosines (see Materials and Methods) identified in each wild-type and hH1 replicate. (F) The level of methylation at individual cytosines is higher in hH1. The percentage of total shared methylated sites (y-axis) vs. the level of methylation at individual cytosines (x-axis) is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1606160464" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Increased DNA methylation is observed at most &lt;em&gt;N. crassa&lt;/em&gt; heterochromatin domains in H1-deficient strains. (A) The DNA methylation level (weighted DNA methylation level (%); (see &lt;em&gt;Materials and Methods&lt;/em&gt;) is shown for 10 kb windows across &lt;em&gt;N. crassa&lt;/em&gt; Linkage Group VII for wild-type, the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-46&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;hH1&lt;/em&gt; mutant, and the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-47&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;dim-2&lt;/em&gt; strain, which lacks all DNA methylation. (B&#x2013;D) The metaplots show the average DNA methylation level across all previously identified wild-type methylated domains: (B) the wild-type &lt;em&gt;mat A&lt;/em&gt; strain and the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-48&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;hH1&lt;/em&gt; strain; (C) the wild-type &lt;em&gt;mat a&lt;/em&gt; strain and the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-49&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;hH1&lt;/em&gt; strain; and (D) the wildtype &lt;em&gt;mat A&lt;/em&gt; strain and the &lt;em&gt;hH1&lt;sup&gt;RIP2&lt;/sup&gt;&lt;/em&gt; strain. Data for at least two independent biological replicates of each strain are shown. (E) More cytosines are methylated in &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-50&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;hH1&lt;/em&gt;. The plot shows the total number of methylated cytosines (see &lt;em&gt;Materials and Methods&lt;/em&gt;) identified in each wild-type and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-51&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;hH1&lt;/em&gt; replicate. (F) The level of methylation at individual cytosines is higher in &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-52&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;&lt;em&gt;hH1&lt;/em&gt;. The percentage of total shared methylated sites (&lt;em&gt;y&lt;/em&gt;-axis) &lt;em&gt;vs.&lt;/em&gt; the level of methylation at individual cytosines (&lt;em&gt;x&lt;/em&gt;-axis) is shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1879/F5.medium.gif" width="440" height="410"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/7/1879/F5.medium.gif" width="440" height="410"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1879/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468496" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-29" class="first-child">Increased DNA methylation is observed at most <em>N. crassa</em> heterochromatin domains in H1-deficient strains. (A) The DNA methylation level (weighted DNA methylation level (%); (see <em>Materials and Methods</em>) is shown for 10 kb windows across <em>N. crassa</em> Linkage Group VII for wild-type, the <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-46.gif"/></noscript></span></span><em>hH1</em> mutant, and the <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-47.gif"/></noscript></span></span><em>dim-2</em> strain, which lacks all DNA methylation. (B–D) The metaplots show the average DNA methylation level across all previously identified wild-type methylated domains: (B) the wild-type <em>mat A</em> strain and the <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-48.gif"/></noscript></span></span><em>hH1</em> strain; (C) the wild-type <em>mat a</em> strain and the <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-49.gif"/></noscript></span></span><em>hH1</em> strain; and (D) the wildtype <em>mat A</em> strain and the <em>hH1<sup>RIP2</sup></em> strain. Data for at least two independent biological replicates of each strain are shown. (E) More cytosines are methylated in <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-50.gif"/></noscript></span></span><em>hH1</em>. The plot shows the total number of methylated cytosines (see <em>Materials and Methods</em>) identified in each wild-type and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-51.gif"/></noscript></span></span><em>hH1</em> replicate. (F) The level of methylation at individual cytosines is higher in <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/F5/embed/mml-math-52.gif"/></noscript></span></span><em>hH1</em>. The percentage of total shared methylated sites (<em>y</em>-axis) <em>vs.</em> the level of methylation at individual cytosines (<em>x</em>-axis) is shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Higher methylation levels can result when individual cytosines are methylated at a higher frequency in a population of nuclei, or when new cytosines become methylated (or a combination of both). To determine which of these possibilities was the case in H1-deficient strains, we determined the number of methylated cytosines in wild-type and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-53.gif"/></noscript></span></span><em>hH1</em> using a binomial test in combination with multiple testing correction (see <em>Materials and Methods</em>). The number of methylated cytosines increased by <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-54.gif"/></noscript></span></span>25% in the <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-55.gif"/></noscript></span></span><em>H1</em> strain. We next compared methylation frequency at cytosines that were scored as methylated in both strains. This revealed that shared sites are methylated at higher frequency in nuclei from the <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-56.gif"/></noscript></span></span><em>hH1</em> strain. Thus, loss of H1 leads to a subtle, but global increase in DNA methylation at relics of repeat-induced point mutation.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-31">We applied genomic and molecular methods to investigate H1 in the model fungus <em>N. crassa</em>. We first confirmed that <em>Neurospora</em> H1 is a chromatin component by constructing a functional epitope-tagged H1 fusion protein and analyzing its localization in cell fractionation and ChIP-seq experiments. This revealed that H1 is globally enriched throughout the genome, occupying both heterochromatic and euchromatic regions. Surprisingly, we observed the highest enrichment of H1-3XFLAG in promoters of expressed genes. Although the overall amplitude of enrichment was low, preferential enrichment of H1-3XFLAG was clearly correlated with expression level. These results may indicate that H1 plays different roles in fungi and animals. In mouse ESCs, H1c and H1d are depleted from strong promoters and enhancers (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cao <em>et al.</em> 2013</a>). It should be noted that H1 protein levels are significantly reduced in ESCs compared to differentiated cells (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Fan <em>et al.</em> 2003</a>), which might explain why H1c and H1d are not detected at ESC promoters. This seems unlikely, however, as ChIP-seq analysis of H1b from ESCs and from differentiated cells revealed that this protein was similarly enriched at repressed genes and depleted from active promoters (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2012a</a>). On the other hand, given that mammals encode multiple H1 variants, it is possible that certain H1 variants will bind preferentially to nucleosome-depleted promoter regions, similar to the case for <em>Neurospora</em> H1. Indeed, analysis of H1 variants by DamID showed that H1.1 exhibits a distinct localization compared to other histones and is not excluded from promoters like the other somatic H1 variants (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Izzo <em>et al.</em> 2013</a>). Similarly, the more divergent H1.X variant was recently found to be enriched at regions with high RNA polymerase II occupancy (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Mayor <em>et al.</em> 2015</a>), suggesting that these H1 variants may play roles within active chromatin in higher eukaryotes. Additional work to define the specific roles of linker histones in fungal and animal systems is needed to determine their specific functions at open chromatin.</p><p id="p-32">One possibility is that <em>N. crassa</em> H1 functions more like animal HMG (High Mobility group) proteins. The chromatin architectural protein HMGD1 is enriched in active chromatin, for example (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nalabothula <em>et al.</em> 2014</a>), similar to the results obtained for H1-3XFLAG here. It is important to note that we cannot rule out the possibility that the subtle promoter enrichment observed for H1-3XFLAG is an experimental artifact. This could be the case if the basic H1 protein interacts nonspecifically with DNA sequences originating from open chromatin during the ChIP procedure. It was shown that certain highly expressed loci in yeast are prone to artifactual enrichment in ChIP-chip experiments for reasons that are not understood (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Teytelman <em>et al.</em> 2013</a>). However, we do not think this is the case here because the nonspecific enrichment patterns observed in yeast are qualitatively different from the enrichment patterns we observe for H1-3XFLAG. Thus, it is likely that the patterns observed here reflect the <em>in vivo</em> occupancy of H1. It will be interesting to determine if promoter regions interact with a subpopulation of H1 in which specific residues are post-translationally modified. It will also be interesting to determine if depletion of H1 from coding sequences of expressed genes depends on post-translational modification. This seems likely given that H1 proteins in plants and animals are extensively modified, much like the core histones (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Harshman <em>et al.</em> 2013</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bednar <em>et al.</em> 2016</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Annalisa and Robert 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kotliński <em>et al.</em> 2016</a>). Moreover, it was shown that phosphorylation of H1 was linked to transcription by RNA polymerase I and II in humans (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Zheng <em>et al.</em> 2010</a>). An important goal for future studies will be to determine if <em>N. crassa</em> H1 is post-translationally modified and to determine if different forms of H1 exhibit distinct localization and/or distinct functions.</p><p id="p-33">We found here that deletion of <em>hH1</em> from <em>N. crassa</em> did not substantially alter global MNase accessibility or nucleosome positioning. Moreover, H1 did not impact the size of protected DNA fragments produced by MNase treatment or the distance between adjacent nucleosomes in genes. These results point to clear differences in how H1 interacts with chromatin in <em>N. crassa</em> and in animals. Indeed, H1 depletion caused increased MNase accessibility, altered nucleosome spacing lengths, and reduced chromatin compaction in H1 triple-knockout ESCs (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Fan <em>et al.</em> 2005</a>). Our results could indicate that <em>N. crassa</em> H1 does not bind to the linker DNA and the dyad axis of the NCP as demonstrated for animal H1 (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Bednar <em>et al.</em> 2016</a>). Another possibility is that <em>N. crassa</em> H1 is more dynamic than H1 in higher eukaryotes. FRAP studies revealed that mammalian H1 variants exist in high-mobility and low-mobility pools, and that the half-life of fluorescence recovery after H1-GFP bleaching was significantly shorter than for the core histones (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Misteli <em>et al.</em> 2000</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lever <em>et al.</em> 2000</a>). Interactions between H1 and the NCP may be even more transient in <em>N. crassa</em>, such that H1 does not interfere with MNase digestion even though it interacts with the same region of the nucleosome protected by animal H1.</p><p id="p-34">We found increased DNA methylation in H1-deficient cells of <em>N. crassa</em>. H1 affects DNA methylation in both <em>A. thaliana</em> and animal cells, but the relationship between H1 and DNA methylation is different in these systems. In animals, H1 variants promote repressive modifications, including DNA methylation in mammals (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Yang <em>et al.</em> 2013</a>) and H3K9me2 in both mammals and <em>Drosophila</em> (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Li <em>et al.</em> 2012a</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Lu <em>et al.</em> 2013</a>). The observation that H1-deficient cells exhibit hypermethylation demonstrates that <em>N. crassa</em> H1 is not required to promote DNA methylation or H3K9 methylation, which directs DNA methylation in <em>Neurospora</em> (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Tamaru and Selker 2001</a>). Similar hypermethylation was reported in <em>A. immersus</em> (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Barra <em>et al.</em> 2000</a>). Moreover, the DNA methylation phenotypes of <em>N. crassa</em> and <em>A. immersus</em> are reminiscent of <em>A. thaliana</em> H1 depletion lines, where a global increase in DNA methylation was observed in heterochromatin domains along with loss of DNA methylation in euchromatic transposon sequences (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Wierzbicki and Jerzmanowski 2005</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Zemach <em>et al.</em> 2013</a>). These results indicate that H1 can limit DNA methylation in plants and fungi. Indeed, depletion of <em>A. thaliana</em> H1 rescued the reduced DNA methylation phenotype of <em>ddm1</em> plants, leading to the conclusion that DDM1 promotes DNA methylation by removal of H1. The <em>Neurospora</em> LSH/DDM1 homolog MUS-30 does not impact DNA methylation levels, therefore we predict that an <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-57.gif"/></noscript></span></span><em>hH1 <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-58.gif"/></noscript></span></span></em>mus-30 double mutant would resemble the <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1879/embed/mml-math-59.gif"/></noscript></span></span><em>hH1</em> single mutant strain (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Basenko <em>et al.</em> 2016</a>). Taken together, the data from fungi, plants, and animals may indicate that H1 evolved a new function to promote heterochromatic modifications in the animal lineage.</p><p id="p-35">It is possible that <em>N. crassa</em> H1 limits access of the DNA methyltransferase DIM-2 or the H3K9 MTase DIM-5<sup>KMT1</sup>. A recent study showed that binding of H1 to the NCP limited the dynamics and modifiability of the H3 tail <em>in vitro</em> (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Stützer <em>et al.</em> 2016</a>), consistent with this possibility. In addition, increased accessibility of the DIM-2 DNA methyltransferase was linked to hypermethylation in the <em>N. crassa</em> Histone Deacetylase-1-deficient strain (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Honda <em>et al.</em> 2012</a>). On the other hand, in <em>A. thaliana</em>, H1 was required for imprinting of the <em>MEDEA</em> locus, which involves active removal of methyl cytosine bases by DNA glycosylases (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Rea <em>et al.</em> 2012</a>). A mechanism for DNA demethylation has not been described in <em>N. crassa</em>, but it is possible that H1 promotes removal of methylated cytosines in heterochromatin domains. Additional studies are needed to understand exactly how H1 impacts DNA methylation levels in <em>N. crassa</em>. Overall, this work adds to the diverse set of phenotypes that have been reported following depletion of H1 in plants, animals, and fungi. Future work to investigate H1 in fungal systems is likely to yield new insights into the evolution and the functions of this important group of proteins.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">We would like to thank Cameron Prybol for technical contributions to the project. This work was funded by a grant from the American Cancer Society to Z.A.L. (RSG-14-184-01-DMC), a grant from the National Institutes of Health to R.J.S. (R00GM100000), and a grant to X.Z. from the National Science Foundation (#0960425).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-37">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028324/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028324/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-38"><em>Communicating editor: J. Rine</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-02-24"><span class="received-label">Received </span>February 24, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-04-22"><span class="accepted-label">Accepted </span>April 22, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Seymour <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Annalisa</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The role of linker histone H1 modifications in the regulation of gene expression and chromatin dynamics.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1859</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1859%26rft.spage%253D486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.2" data-doi="10.1038/253245a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baldwin</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boseley</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibel</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">The subunit structure of the eukaryotic chromosome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">253</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBaldwin%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D253%26rft.issue%253D5489%26rft.spage%253D245%26rft.epage%253D249%26rft.atitle%253DThe%2Bsubunit%2Bstructure%2Bof%2Bthe%2Beukaryotic%2Bchromosome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F253245a0%26rft_id%253Dinfo%253Apmid%252F1167623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/253245a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1167623&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.3" data-doi="10.1128/MCB.20.1.61-69.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barra</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rhounim</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossignol</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faugeron</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Histone H1 is dispensable for methylation-associated gene silencing in ascobolus immersus and essential for long life span.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBarra%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D61%26rft.epage%253D69%26rft.atitle%253DHistone%2BH1%2BIs%2BDispensable%2Bfor%2BMethylation-Associated%2BGene%2BSilencing%2Bin%2BAscobolus%2Bimmersus%2Band%2BEssential%2Bfor%2BLong%2BLife%2BSpan%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.20.1.61-69.2000%26rft_id%253Dinfo%253Apmid%252F10594009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjc6IjIwLzEvNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.4" data-doi="10.1371/journal.pgen.1005790"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basenko</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">Z. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The LSH/DDM1 homolog mus-30 is required for genome stability, but not for dna methylation in neurospora crassa.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005790%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005790%26rft_id%253Dinfo%253Apmid%252F26771905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005790&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26771905&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bednar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamiche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dimitrov</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">H1-nucleosome interactions and their functional implications.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1859</span>: <span class="cit-fpage">436</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1859%26rft.spage%253D436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.6" data-doi="10.1128/MCB.16.2.513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell-Pedersen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunlap</span>  <span class="cit-name-given-names">J. A. Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loros</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Distinct cis-acting elements mediate clock, light, and developmental regulation of the <em>Neurospora crassa</em> eas (ccg-2) gene.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBell-Pedersen%26rft.auinit1%253DD.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D513%26rft.epage%253D521%26rft.atitle%253DDistinct%2Bcis-acting%2Belements%2Bmediate%2Bclock%252C%2Blight%252C%2Band%2Bdevelopmental%2Bregulation%2Bof%2Bthe%2BNeurospora%2Bcrassa%2Beas%2B%2528ccg-2%2529%2Bgene%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.16.2.513%26rft_id%253Dinfo%253Apmid%252F8552078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE2LzIvNTEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.7" data-doi="10.1371/journal.pgen.1003417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lailler</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uppal</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-resolution mapping of H1 linker histone variants in embryonic stem cells.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003417%26rft.atitle%253DHIGH-RESOLUTION%2BMAPPING%2BOF%2BH1%2BLINKER%2BHISTONE%2BVARIANTS%2BIN%2BEMBRYONIC%2BSTEM%2BCELLS%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003417%26rft_id%253Dinfo%253Apmid%252F23633960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23633960&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.8" data-doi="10.1021/bi00093a011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cerf</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lippens</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muyldermans</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segers</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishnan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Homo- and heteronuclear two-dimensional NMR studies of the globular domain of histone H1: sequential assignment and secondary structure.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">11345</span>–<span class="cit-lpage">11351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D32%26rft.spage%253D11345%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00093a011%26rft_id%253Dinfo%253Apmid%252F8218199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00093a011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8218199&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MD71000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.9" data-doi="10.1073/pnas.0601456103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colot</span>  <span class="cit-name-given-names">H. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ringelberg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crew</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high-throughput gene knockout procedure for Neurospora reveals functions for multiple transcription factors.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">10352</span>–<span class="cit-lpage">10357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0601456103%26rft_id%253Dinfo%253Apmid%252F16801547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzI3LzEwMzUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Port</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Arginaseless Neurospora: genetics, physiology, and polyamine synthesis.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DDavis%26rft.auinit1%253DR.%2BH.%26rft.volume%253D102%26rft.issue%253D2%26rft.spage%253D299%26rft.epage%253D305%26rft.atitle%253DArginaseless%2BNeurospora%253A%2BGenetics%252C%2BPhysiology%252C%2Band%2BPolyamine%2BSynthesis%26rft_id%253Dinfo%253Apmid%252F5419257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTAyLzIvMjk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.11" data-doi="10.1016/S1097-2765(03)00197-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Downs</span>  <span class="cit-name-given-names">J. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosmidou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Suppression of homologous recombination by the <em>Saccharyomyces cerevisiae</em> linker histone.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1685</span>–<span class="cit-lpage">1692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DDowns%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D1685%26rft.epage%253D1692%26rft.atitle%253DSuppression%2Bof%2Bhomologous%2Brecombination%2Bby%2Bthe%2BSaccharomyces%2Bcerevisiae%2Blinker%2Bhistone.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252803%252900197-7%26rft_id%253Dinfo%253Apmid%252F12820979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(03)00197-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12820979&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183719700026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.12" data-doi="10.1128/MCB.21.23.7933-7943.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirotkin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayala</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skoultchi</span>  <span class="cit-name-given-names">A. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Individual somatic H1 subtypes are dispensable for mouse development even in mice lacking the H1(0) replacement subtype.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">7933</span>–<span class="cit-lpage">7943</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DFan%26rft.auinit1%253DY.%26rft.volume%253D21%26rft.issue%253D23%26rft.spage%253D7933%26rft.epage%253D7943%26rft.atitle%253DIndividual%2BSomatic%2BH1%2BSubtypes%2BAre%2BDispensable%2Bfor%2BMouse%2BDevelopment%2BEven%2Bin%2BMice%2BLacking%2Bthe%2BH10%2BReplacement%2BSubtype%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.23.7933-7943.2001%26rft_id%253Dinfo%253Apmid%252F11689686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8yMy83OTMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.13" data-doi="10.1128/MCB.23.13.4559-4572.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikitina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morin-Kensicki</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnuson</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">H1 linker histones are essential for mouse development and affect nucleosome spacing in vivo.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4559</span>–<span class="cit-lpage">4572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DFan%26rft.auinit1%253DY.%26rft.volume%253D23%26rft.issue%253D13%26rft.spage%253D4559%26rft.epage%253D4572%26rft.atitle%253DH1%2BLinker%2BHistones%2BAre%2BEssential%2Bfor%2BMouse%2BDevelopment%2Band%2BAffect%2BNucleosome%2BSpacing%2BIn%2BVivo%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.13.4559-4572.2003%26rft_id%253Dinfo%253Apmid%252F12808097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMy80NTU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.14" data-doi="10.1016/j.cell.2005.10.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nikitina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleury</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharyya</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Histone H1 depletion in mammals alters global chromatin structure but causes specific changes in gene regulation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">1199</span>–<span class="cit-lpage">1212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DFan%26rft.auinit1%253DY.%26rft.volume%253D123%26rft.issue%253D7%26rft.spage%253D1199%26rft.epage%253D1212%26rft.atitle%253DHistone%2BH1%2Bdepletion%2Bin%2Bmammals%2Balters%2Bglobal%2Bchromatin%2Bstructure%2Bbut%2Bcauses%2Bspecific%2Bchanges%2Bin%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.10.028%26rft_id%253Dinfo%253Apmid%252F16377562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.10.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16377562&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234584500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.15" data-doi="10.1128/EC.2.2.341-350.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Folco</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Temporini</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Histone H1 is required for proper regulation of pyruvate decarboxylase gene expression in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">341</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DFolco%26rft.auinit1%253DH.%2BD.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D341%26rft.epage%253D350%26rft.atitle%253DHistone%2BH1%2BIs%2BRequired%2Bfor%2BProper%2BRegulation%2Bof%2BPyruvate%2BDecarboxylase%2BGene%2BExpression%2Bin%2BNeurosporacrassa%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.2.2.341-350.2003%26rft_id%253Dinfo%253Apmid%252F12684383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIyLzIvMzQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.16" data-doi="10.1038/nature01554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galagan</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calvo</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borkovich</span>  <span class="cit-name-given-names">K. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The genome sequence of the filamentous fungus <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGalagan%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D422%26rft.issue%253D6934%26rft.spage%253D859%26rft.epage%253D868%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bfilamentous%2Bfungus%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01554%26rft_id%253Dinfo%253Apmid%252F12712197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01554&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12712197&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182432600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.17" data-doi="10.1186/s13059-015-0857-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geeven</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartholdy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Local compartment changes and regulatory landscape alterations in histone H1-depleted cells.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0857-0%26rft_id%253Dinfo%253Apmid%252F26700097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0857-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26700097&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.18" data-doi="10.1093/nar/gkt700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harshman</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parthun</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitas</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">H1 histones: current perspectives and challenges.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">9593</span>–<span class="cit-lpage">9609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt700%26rft_id%253Dinfo%253Apmid%252F23945933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS8yMS85NTkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.19" data-doi="10.1016/j.molcel.2010.05.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heinz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertolino</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and b cell identities.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHeinz%26rft.auinit1%253DS.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D576%26rft.epage%253D589%26rft.atitle%253DSimple%2Bcombinations%2Bof%2Blineage-determining%2Btranscription%2Bfactors%2Bprime%2Bcis-regulatory%2Belements%2Brequired%2Bfor%2Bmacrophage%2Band%2BB%2Bcell%2Bidentities.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.05.004%26rft_id%253Dinfo%253Apmid%252F20513432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20513432&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278448100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.20" data-doi="10.1128/MCB.00823-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Honda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Direct interaction between DNA methyltransferase DIM-2 and HP1 is required for dna methylation in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">6044</span>–<span class="cit-lpage">6055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00823-08%26rft_id%253Dinfo%253Apmid%252F18678653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xOS82MDQ0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.21" data-doi="10.1534/genetics.108.098707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Honda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Tools for fungal proteomics: multifunctional Neurospora vectors for gene replacement, protein expression and protein purification.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098707%26rft_id%253Dinfo%253Apmid%252F19171944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTgyLzEvMTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.22" data-doi="10.1038/nsmb.2274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Honda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischle</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sack</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Heterochromatin protein 1 forms distinct complexes to direct histone deacetylation and DNA methylation.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DHonda%26rft.auinit1%253DS.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D471%26rft.epage%253D477%26rft.atitle%253DHeterochromatin%2Bprotein%2B1%2Bforms%2Bdistinct%2Bcomplexes%2Bto%2Bdirect%2Bhistone%2Bdeacetylation%2Band%2BDNA%2Bmethylation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2274%26rft_id%253Dinfo%253Apmid%252F22504884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22504884&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.23" data-doi="10.1016/j.celrep.2013.05.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Izzo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamieniarz-Gdula</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramírez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noureen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kind</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genomic landscape of the somatic linker histone subtypes H1.1 to H1.5 in human cells.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2142</span>–<span class="cit-lpage">2154</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D3%26rft.spage%253D2142%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2013.05.003%26rft_id%253Dinfo%253Apmid%252F23746450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2013.05.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23746450&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Methylated DNA is over-represented in whole-genome bisulfite sequencing data.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D341%26rft_id%253Dinfo%253Apmid%252F25374580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25374580&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.25" data-doi="10.1096/fj.00-0237rev"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kasinsky</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dacks</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ausió</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Origin of H1 linker histones.</span> <abbr class="cit-jnl-abbrev">FASEB J.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BFASEB%2BJournal%26rft.stitle%253DFASEB%2BJ.%26rft.aulast%253DKASINSKY%26rft.auinit1%253DH.%2BE.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D34%26rft.epage%253D42%26rft.atitle%253DOrigin%2Bof%2BH1%2Blinker%2Bhistones%26rft_id%253Dinfo%253Adoi%252F10.1096%252Ffj.00-0237rev%26rft_id%253Dinfo%253Apmid%252F11149891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmFzZWJqIjtzOjU6InJlc2lkIjtzOjc6IjE1LzEvMzQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.26" data-doi="10.1126/science.184.4139.865"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kornberg</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Chromatin structure; oligomers of the histones.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKornberg%26rft.auinit1%253DR.%2BD.%26rft.volume%253D184%26rft.issue%253D4139%26rft.spage%253D865%26rft.epage%253D868%26rft.atitle%253DChromatin%2BStructure%253A%2BOligomers%2Bof%2Bthe%2BHistones%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.184.4139.865%26rft_id%253Dinfo%253Apmid%252F4825888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjE4NC80MTM5Lzg2NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.27" data-doi="10.1371/journal.pone.0147908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kotliński</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutowicz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kniźewski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palusiński</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olędzki</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Histone H1 variants in Arabidopsis are subject to numerous post-translational modifications, both conserved and previously unknown in histones, suggesting complex functions of h1 in plants.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0147908</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0147908%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0147908%26rft_id%253Dinfo%253Apmid%252F26820416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0147908&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26820416&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-2" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.28" data-doi="10.1093/emboj/20.15.4309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kouzminova</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title"><em>dim-2</em> encodes a DNA methyltransferase responsible for all known cytosine methylation in Neurospora</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">4309</span>–<span class="cit-lpage">4323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DKouzminova%26rft.auinit1%253DE.%26rft.volume%253D20%26rft.issue%253D15%26rft.spage%253D4309%26rft.epage%253D4323%26rft.atitle%253Ddim-2%2Bencodes%2Ba%2BDNA%2Bmethyltransferase%2Bresponsible%2Bfor%2Ball%2Bknown%2Bcytosine%2Bmethylation%2Bin%2BNeurospora%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F20.15.4309%26rft_id%253Dinfo%253Apmid%252F11483533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjAvMTUvNDMwOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.29" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.30" data-doi="10.1038/35048603"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lever</span>  <span class="cit-name-given-names">M. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Th’ng</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendzel</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Rapid exchange of histone H1.1 on chromatin in living human cells.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLever%26rft.auinit1%253DM.%2BA.%26rft.volume%253D408%26rft.issue%253D6814%26rft.spage%253D873%26rft.epage%253D876%26rft.atitle%253DRapid%2Bexchange%2Bof%2Bhistone%2BH1.1%2Bon%2Bchromatin%2Bin%2Bliving%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35048603%26rft_id%253Dinfo%253Apmid%252F11130728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35048603&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130728&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165831300054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.31" data-doi="10.1073/pnas.0709403105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hershkovits</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmon-Divon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klutstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Yeast linker histone Hho1p is required for efficient RNA polymerase I processivity and transcriptional silencing at the ribosomal dna.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">11703</span>–<span class="cit-lpage">11708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0709403105%26rft_id%253Dinfo%253Apmid%252F18687885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzMzLzExNzAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.32" data-doi="10.1101/gr.086231.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">Z. a.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Honda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khlafallah</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffress</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Relics of repeat-induced point mutation direct heterochromatin formation in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">437</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.086231.108%26rft_id%253Dinfo%253Apmid%252F19092133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzMvNDI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.33" data-doi="10.1111/j.1365-2958.2007.06101.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elfline</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryk</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Linker histone H1 represses recombination at the ribosomal DNA locus in the budding yeast saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">906</span>–<span class="cit-lpage">919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DLi%26rft.auinit1%253DC.%26rft.volume%253D67%26rft.issue%253D4%26rft.spage%253D906%26rft.epage%253D919%26rft.atitle%253DLinker%2Bhistone%2BH1%2Brepresses%2Brecombination%2Bat%2Bthe%2Bribosomal%2BDNA%2Blocus%2Bin%2Bthe%2Bbudding%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2007.06101.x%26rft_id%253Dinfo%253Apmid%252F18179596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2007.06101.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18179596&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253312400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.34" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with burrows-wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.35" data-doi="10.1371/journal.pgen.1002879"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mikkola</span>  <span class="cit-name-given-names">H. K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowry</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurdistani</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Dynamic distribution of linker histone H1.5 in cellular differentiation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DJ.%2BY.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002879%26rft.epage%253De1002879%26rft.atitle%253DDynamic%2Bdistribution%2Bof%2Blinker%2Bhistone%2BH1.5%2Bin%2Bcellular%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002879%26rft_id%253Dinfo%253Apmid%252F22956909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002879&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22956909&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.36" data-doi="10.1074/jbc.M111.311860"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Structure of c-terminal tandem BRCT repeats of Rtt107 protein reveals critical role in interaction with phosphorylated histone H2A during DNA damage repair.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">9137</span>–<span class="cit-lpage">9146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.311860%26rft_id%253Dinfo%253Apmid%252F22262834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyODcvMTIvOTEzNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.37" data-doi="10.1126/science.1234654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wontakal</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kavi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guzzardo</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Drosophila h1 regulates the genetic activity of heterochromatin by recruitment of Su(var)3-9.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">340</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLu%26rft.auinit1%253DX.%26rft.volume%253D340%26rft.issue%253D6128%26rft.spage%253D78%26rft.epage%253D81%26rft.atitle%253DDrosophila%2BH1%2BRegulates%2Bthe%2BGenetic%2BActivity%2Bof%2BHeterochromatin%2Bby%2BRecruitment%2Bof%2BSu%2528var%25293-9%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1234654%26rft_id%253Dinfo%253Apmid%252F23559249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzNDAvNjEyOC83OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.38" data-doi="10.1038/38444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mäder</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sargent</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richmond</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Crystal structure of the nucleosome core particle at 2.8 a resolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">389</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLuger%26rft.auinit1%253DK.%26rft.volume%253D389%26rft.issue%253D6648%26rft.spage%253D251%26rft.epage%253D260%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bthe%2Bnucleosome%2Bcore%2Bparticle%2Bat%2B2.8%2BA%2Bresolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F38444%26rft_id%253Dinfo%253Apmid%252F9305837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/38444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9305837&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XW77200038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.39" data-doi="10.1128/EC.00244-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margolin</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Improved plasmids for gene targeting at the <em>his-3</em> locus of <em>Neurospora crassa</em> by electroporation.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D44%26rft.spage%253D34%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00244-07%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/EC.00244-07&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.40" data-doi="10.1074/jbc.M114.617324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayor</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Izquierdo-Bouldstridge</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millán-Ariño</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustillos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sampaio</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome distribution of replication-independent histone H1 variants shows H1.0 associated with nucleolar domains and H1X associated with RNA polymerase II-enriched regions.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">7474</span>–<span class="cit-lpage">7491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M114.617324%26rft_id%253Dinfo%253Apmid%252F25645921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyOTAvMTIvNzQ3NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.41" data-doi="10.1007/s12038-010-0014-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiest</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plamann</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The fungal genetics stock center: a repository for 50 years of fungal genetics research.</span> <abbr class="cit-jnl-abbrev">J. Biosci.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiosciences%26rft.stitle%253DJ%2BBiosci%26rft.aulast%253DMcCluskey%26rft.auinit1%253DK.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D126%26rft.atitle%253DThe%2BFungal%2BGenetics%2BStock%2BCenter%253A%2Ba%2Brepository%2Bfor%2B50%2Byears%2Bof%2Bfungal%2Bgenetics%2Bresearch.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12038-010-0014-6%26rft_id%253Dinfo%253Apmid%252F20413916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12038-010-0014-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20413916&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277022500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.42" data-doi="10.1038/35048610"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Misteli</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, A. Gunjan, R. Hock, M. Bustin, and D. T. Brown, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Dynamic binding of histone H1 to chromatin in living cells.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">877</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMisteli%26rft.auinit1%253DT.%26rft.volume%253D408%26rft.issue%253D6814%26rft.spage%253D877%26rft.epage%253D881%26rft.atitle%253DDynamic%2Bbinding%2Bof%2Bhistone%2BH1%2Bto%2Bchromatin%2Bin%2Bliving%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35048610%26rft_id%253Dinfo%253Apmid%252F11130729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35048610&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130729&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165831300055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.43" data-doi="10.1186/1471-2164-15-92"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nalabothula</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVicker</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maiorano</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The chromatin architectural proteins HMGD1 and H1 bind reciprocally and have opposite effects on chromatin structure and gene regulation.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">92</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D92%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-92%26rft_id%253Dinfo%253Apmid%252F24484546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-92&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24484546&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.44" data-doi="10.1016/S0022-2836(77)80019-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornberg</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Action of micrococcal nuclease on chromatin and the location of histone H1.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DNoll%26rft.auinit1%253DM.%26rft.volume%253D109%26rft.issue%253D3%26rft.spage%253D393%26rft.epage%253D404%26rft.atitle%253DAction%2Bof%2Bmicrococcal%2Bnuclease%2Bon%2Bchromatin%2Band%2Bthe%2Blocation%2Bof%2Bhistone%2BH1.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252877%252980019-3%26rft_id%253Dinfo%253Apmid%252F833849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(77)80019-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=833849&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1977DB93700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.45" data-doi="10.1126/science.183.4122.330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olins</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olins</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Spheroid chromatin units (v bodies).</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">330</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DOlins%26rft.auinit1%253DA.%2BL.%26rft.volume%253D183%26rft.issue%253D4122%26rft.spage%253D330%26rft.epage%253D332%26rft.atitle%253DSpheroid%2BChromatin%2BUnits%2B%2528ngr%2BBodies%2529%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.183.4122.330%26rft_id%253Dinfo%253Apmid%252F4128918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIxODMvNDEyMi8zMzAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pall</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The use of ignite (basta; glufosinate; phosphinothricin) to select transformants of <em>bar</em>-containing plasmids in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Newsl.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BNewsl.%26rft.volume%253D40%26rft.spage%253D58%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Role of H1 linker histones in mammalian development and stem cell differentiation. Biochim. Biophys. Acta</span><abbr class="cit-jnl-abbrev">.</abbr> <span class="cit-vol">1859</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.48" data-doi="10.1074/jbc.273.13.7268"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterton</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landel</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landsman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">R. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The biochemical and phenotypic characterization of Hho1p, the putative linker histone H1 of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">7268</span>–<span class="cit-lpage">7276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DPatterton%26rft.auinit1%253DH.%2BG.%26rft.volume%253D273%26rft.issue%253D13%26rft.spage%253D7268%26rft.epage%253D7276%26rft.atitle%253DThe%2BBiochemical%2Band%2BPhenotypic%2BCharacterization%2Bof%2BHho1p%252C%2Bthe%2BPutative%2BLinker%2BHistone%2BH1%2Bof%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.273.13.7268%26rft_id%253Dinfo%253Apmid%252F9516420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNzMvMTMvNzI2OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.49" data-doi="10.1016/j.ymeth.2008.09.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pomraning</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide high throughput analysis of DNA methylation in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">142</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DPomraning%26rft.auinit1%253DK.%2BR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D142%26rft.epage%253D150%26rft.atitle%253DGenome-wide%2Bhigh%2Bthroughput%2Banalysis%2Bof%2BDNA%2Bmethylation%2Bin%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2008.09.022%26rft_id%253Dinfo%253Apmid%252F18950712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2008.09.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18950712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263695700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.50" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bedtools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.51" data-doi="10.1038/362219a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishnan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finch</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graziano</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweet</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Crystal structure of globular domain of histone H5 and its implications for nucleosome binding.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">362</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRamakrishnan%26rft.auinit1%253DV.%26rft.volume%253D362%26rft.issue%253D6417%26rft.spage%253D219%26rft.epage%253D223%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bglobular%2Bdomain%2Bof%2Bhistone%2BH5%2Band%2Bits%2Bimplications%2Bfor%2Bnucleosome%2Bbinding.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F362219a0%26rft_id%253Dinfo%253Apmid%252F8384699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/362219a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8384699&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KT02600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.52" data-doi="10.1046/j.1365-2958.2000.01702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Ramón</span></li></ol><cite>, A., M. I. Muro-Pastor, C. Scazzocchio, and R. Gonzalez, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Deletion of the unique gene encoding a typical histone H1 has no apparent phenotype in Aspergillus nidulans.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DRam%25C2%25BFon%26rft.auinit1%253DA.%26rft.volume%253D35%26rft.issue%253D1%26rft.spage%253D223%26rft.epage%253D233%26rft.atitle%253DDeletion%2Bof%2Bthe%2Bunique%2Bgene%2Bencoding%2Ba%2Btypical%2Bhistone%2BH1%2Bhas%2Bno%2Bapparent%2Bphenotype%2Bin%2BAspergillus%2Bnidulans.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2000.01702.x%26rft_id%253Dinfo%253Apmid%252F10632892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2000.01702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10632892&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000084867000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braud</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhangu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rognan</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> Histone H1 affects gene imprinting and DNA methylation in Arabidopsis Plant J. 71: 776–786.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-6.7.1879.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sambrook</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritsch</span>  <span class="cit-name-given-names">E. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maniatis</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-source">Molecular Cloning: A Laboratory Manual</span>, <span class="cit-publ-name">Cold Spring Harbor Laboratory Press, Cold Spring Harbor</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.55" data-doi="10.1371/journal.pgen.1005105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sancar</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ha</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yilmaz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tesorero</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Combinatorial control of light induced chromatin remodeling and gene activation in Neurospora</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005105%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005105%26rft_id%253Dinfo%253Apmid%252F25822411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25822411&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.56" data-doi="10.1128/EC.00117-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">C. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Heterochromatin controls gammaH2A localization in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">990</span>–<span class="cit-lpage">1000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00117-14%26rft_id%253Dinfo%253Apmid%252F24879124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiIxMy84Lzk5MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.57" data-doi="10.1073/pnas.0806337105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schäfer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">C. R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterton</span>  <span class="cit-name-given-names">H.-G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The <em>Saccharyomyces cerevisiae</em> linker histone Hho1p is essential for chromatin compaction in stationary phase and is displaced by transcription.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">14838</span>–<span class="cit-lpage">14843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0806337105%26rft_id%253Dinfo%253Apmid%252F18799740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzM5LzE0ODM4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.58" data-doi="10.1101/gr.152538.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdes-Lopez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Epigenome-wide inheritance of cytosine methylation variants in a recombinant inbred population.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1663</span>–<span class="cit-lpage">1674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.152538.112%26rft_id%253Dinfo%253Apmid%252F23739894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMC8xNjYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.59" data-doi="10.1016/j.tig.2012.10.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Leveling the playing field for analyses of single-base resolution DNA methylomes.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DSchultz%26rft.auinit1%253DM.%2BD.%26rft.volume%253D28%26rft.issue%253D12%26rft.spage%253D583%26rft.epage%253D585%26rft.atitle%253D%2527Leveling%2527%2Bthe%2Bplaying%2Bfield%2Bfor%2Banalyses%2Bof%2Bsingle-base%2Bresolution%2BDNA%2Bmethylomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2012.10.012%26rft_id%253Dinfo%253Apmid%252F23131467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2012.10.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23131467&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312054900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.60" data-doi="10.1038/nature14465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitaker</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hariharan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukamel</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Human body epigenome maps reveal noncanonical DNA methylation variation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">523</span>: <span class="cit-fpage">212</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D212%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14465%26rft_id%253Dinfo%253Apmid%252F26030523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26030523&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.61" data-doi="10.1046/j.1365-2958.2001.02306.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwerdtfeger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linden</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Blue light adaptation and desensitization of light signal transduction in neurospora crassa.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1080</span>–<span class="cit-lpage">1087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DSchwerdtfeger%26rft.auinit1%253DC.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D1080%26rft.epage%253D1087%26rft.atitle%253DBlue%2Blight%2Badaptation%2Band%2Bdesensitization%2Bof%2Blight%2Bsignal%2Btransduction%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2001.02306.x%26rft_id%253Dinfo%253Apmid%252F11251826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2001.02306.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11251826&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167237500022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.62" data-doi="10.1073/pnas.73.2.505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovacic</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beaudreau</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Holde</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Analysis of subunit organization in chicken erythrocyte chromatin.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">505</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DShaw%26rft.auinit1%253DB.%2BR.%26rft.volume%253D73%26rft.issue%253D2%26rft.spage%253D505%26rft.epage%253D509%26rft.atitle%253DAnalysis%2Bof%2Bsubunit%2Borganization%2Bin%2Bchicken%2Berythrocyte%2Bchromatin.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.73.2.505%26rft_id%253Dinfo%253Apmid%252F1061151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI3My8yLzUwNSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.63" data-doi="10.1016/S0092-8674(00)80120-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Linker histone H1 regulates specific gene expression but not global transcription in vivo.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DShen%26rft.auinit1%253DX.%26rft.volume%253D86%26rft.issue%253D3%26rft.spage%253D475%26rft.epage%253D483%26rft.atitle%253DLinker%2Bhistone%2BH1%2Bregulates%2Bspecific%2Bgene%2Bexpression%2Bbut%2Bnot%2Bglobal%2Btranscription%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980120-8%26rft_id%253Dinfo%253Apmid%252F8756729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80120-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8756729&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VC30900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.64" data-doi="10.1016/0092-8674(95)90051-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorovsky</span>  <span class="cit-name-given-names">M. a.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Linker histones are not essential and affect chromatin condensation in vivo.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DShen%26rft.auinit1%253DX.%26rft.volume%253D82%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D56%26rft.atitle%253DLinker%2Bhistones%2Bare%2Bnot%2Bessential%2Band%2Baffect%2Bchromatin%2Bcondensation%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252895%252990051-9%26rft_id%253Dinfo%253Apmid%252F7606784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(95)90051-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7606784&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RK42400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.65" data-doi="10.1016/j.molcel.2015.12.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stützer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liokatis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiesel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarzer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sprangers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Modulations of DNA contacts by linker histones and post-translational modifications determine the mobility and modifiability of nucleosomal H3 tails.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D61%26rft.spage%253D247%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2015.12.015%26rft_id%253Dinfo%253Apmid%252F26778125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2015.12.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26778125&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.66" data-doi="10.1038/35104508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamaru</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A histone H3 methyltransferase controls DNA methylation in <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">414</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DTamaru%26rft.auinit1%253DH.%26rft.volume%253D414%26rft.issue%253D6861%26rft.spage%253D277%26rft.epage%253D283%26rft.atitle%253DA%2Bhistone%2BH3%2Bmethyltransferase%2Bcontrols%2BDNA%2Bmethylation%2Bin%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35104508%26rft_id%253Dinfo%253Apmid%252F11713521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35104508&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11713521&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172150700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.67" data-doi="10.1073/pnas.1316064110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teytelman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurtle</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Oudenaarden</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Highly expressed loci are vulnerable to misleading chip localization of multiple unrelated proteins.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">18602</span>–<span class="cit-lpage">18607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1316064110%26rft_id%253Dinfo%253Apmid%252F24173036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzQ2LzE4NjAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.68" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Tophat: discovering splice junctions with RNA-seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNS85LzExMDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTg3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.69" data-doi="10.1038/nprot.2012.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Differential gene and transcript expression analysis of RNA-seq experiments with tophat and cufflinks.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D562%26rft.epage%253D578%26rft.atitle%253DDifferential%2Bgene%2Band%2Btranscript%2Bexpression%2Banalysis%2Bof%2BRNA-seq%2Bexperiments%2Bwith%2BTopHat%2Band%2BCufflinks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2012.016%26rft_id%253Dinfo%253Apmid%252F22383036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2012.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22383036&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.70" data-doi="10.1038/nprot.2014.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Urich</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nery</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Methylc-seq library preparation for base-resolution whole-genome bisulfite sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D10%26rft.spage%253D475%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2014.114%26rft_id%253Dinfo%253Apmid%252F25692984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2014.114&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25692984&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.71" data-doi="10.1534/g3.115.017921"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stajich</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Endogenous small RNA mediates meiotic silencing of a novel DNA transposon.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1949</span>–<span class="cit-lpage">1960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.017921%26rft_id%253Dinfo%253Apmid%252F26109355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjUvMTAvMTk0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.72" data-doi="10.1021/bi00660a022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitlock</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">R. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Removal of histone H1 exposes a fifty base pair DNA segment between nucleosomes.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">3307</span>–<span class="cit-lpage">3314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D15%26rft.spage%253D3307%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00660a022%26rft_id%253Dinfo%253Apmid%252F952859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00660a022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=952859&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1976BY98300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.73" data-doi="10.1534/genetics.104.031997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wierzbicki</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jerzmanowski</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Suppression of histone H1 genes in Arabidopsis results in heritable developmental defects and stochastic changes in DNA methylation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">997</span>–<span class="cit-lpage">1008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.031997%26rft_id%253Dinfo%253Apmid%252F15489532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY5LzIvOTk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE4NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.74" data-doi="10.1073/pnas.1213266110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norwood Toro</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skoultchi</span>  <span class="cit-name-given-names">A. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">H1 linker histone promotes epigenetic silencing by regulating both DNA methylation and histone H3 methylation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1708</span>–<span class="cit-lpage">1713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1213266110%26rft_id%253Dinfo%253Apmid%252F23302691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEwLzUvMTcwOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.75" data-doi="10.1016/j.cell.2013.02.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zemach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">P.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman-Derr</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eshed-Williams</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Arabidopsis nucleosome remodeler DDM1 allows DNA methyltransferases to access H1-containing heterochromatin.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D193%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.02.033%26rft_id%253Dinfo%253Apmid%252F23540698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.02.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23540698&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316853700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.76" data-doi="10.1371/journal.pgen.1002691"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panjwani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krauth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Histone H1 depletion impairs embryonic stem cell differentiation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002691</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002691%26rft.epage%253De1002691%26rft.atitle%253DHistone%2Bh1%2Bdepletion%2Bimpairs%2Bembryonic%2Bstem%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002691%26rft_id%253Dinfo%253Apmid%252F22589736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002691&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22589736&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.77" data-doi="10.1371/journal.pone.0038829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medrzycki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Reduction of hox gene expression by histone H1 depletion.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e38829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De38829%26rft.epage%253De38829%26rft.atitle%253DReduction%2Bof%2BHox%2Bgene%2Bexpression%2Bby%2Bhistone%2BH1%2Bdepletion.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0038829%26rft_id%253Dinfo%253Apmid%252F22701719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0038829&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22701719&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1879.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1879.78" data-doi="10.1083/jcb.201001148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pesavento</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz-Norton</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiltz</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Histone H1 phosphorylation is associated with transcription by RNA polymerases I and II.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201001148%26rft_id%253Dinfo%253Apmid%252F20439994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE4OS8zLzQwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xODc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/7/1879.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1867" title="Copy Number Profiling of Brazilian Astrocytomas" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1891" title="Cytotype Regulation Facilitates Repression of Hybrid Dysgenesis by Naturally Occurring KP Elements in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/7"><span>Volume 6 Issue 7, July 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/7.cover-source.jpg?itok=E84OOyGI" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1879.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0COKdAGOhTwp9fs6EclTblbiUprTp-_tNMvU2by1ZVM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/7/1879" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/7/1879" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/7/1879" class="active" data-icon-position="" data-hide-link-title="0">Histone H1 Limits DNA Methylation in Neurospora crassa</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460436" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-SqUIFCIrp4dsoyO-q6B0YS2ZmpQ85gq5oDAINlsBbFo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1879.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/7/1879" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-0ZI_FGUiZ46DaweHxEya-7UzzYq9pL_P9MuqElZvM8g" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/7/1879" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1879.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460436" id="node460436--2" data-pisa="ggg;6/7/1879" data-pisa-master="ggg;g3.116.028324" data-apath="/ggg/6/7/1879.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Histone H1 Limits DNA Methylation in <em>Neurospora crassa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael Seymour</span>, <span class="highwire-citation-author" data-delta="1">Lexiang Ji</span>, <span class="highwire-citation-author" data-delta="2">Alex M. Santos</span>, <span class="highwire-citation-author" data-delta="3">Masayuki Kamei</span>, <span class="highwire-citation-author" data-delta="4">Takahiko Sasaki</span>, <span class="highwire-citation-author" data-delta="5">Evelina Y. Basenko</span>, <span class="highwire-citation-author" data-delta="6">Robert J. Schmitz</span>, <span class="highwire-citation-author" data-delta="7">Xiaoyu Zhang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-1735-8266" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zachary A. Lewis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1879-1889;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028324 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460436--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael Seymour </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSeymour%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seymour%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BSeymour%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lexiang Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Bioinformatics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lexiang%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALexiang%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alex M. Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BM.%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santos%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlex%2BM.%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Masayuki Kamei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Masayuki%2BKamei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kamei%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMasayuki%2BKamei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Takahiko Sasaki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takahiko%2BSasaki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sasaki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATakahiko%2BSasaki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Evelina Y. Basenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evelina%2BY.%2BBasenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Basenko%20EY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvelina%2BY.%2BBasenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Robert J. Schmitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BSchmitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmitz%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BSchmitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xiaoyu Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoyu%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoyu%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Zachary A. Lewis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BA.%2BLewis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lewis%20ZA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZachary%2BA.%2BLewis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1735-8266" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zachary A. Lewis</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zlewis@uga.edu" class="" data-icon-position="" data-hide-link-title="0">zlewis@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/7/1879" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/7/1879&amp;title=Histone%20H1%20Limits%20DNA%20Methylation%20in%20Neurospora%20crassa" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/7/1879&amp;title=Histone%20H1%20Limits%20DNA%20Methylation%20in%20Neurospora%20crassa" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/7/1879&amp;title=Histone%20H1%20Limits%20DNA%20Methylation%20in%20Neurospora%20crassa" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/7/1879&amp;text=Histone%20H1%20Limits%20DNA%20Methylation%20in%20Neurospora%20crassa" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/7/1879&amp;title=Histone%20H1%20Limits%20DNA%20Methylation%20in%20Neurospora%20crassa" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/7/1879&amp;t=Histone%20H1%20Limits%20DNA%20Methylation%20in%20Neurospora%20crassa" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/7/1879&amp;title=Histone%20H1%20Limits%20DNA%20Methylation%20in%20Neurospora%20crassa" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/7/1879&amp;title=Histone%20H1%20Limits%20DNA%20Methylation%20in%20Neurospora%20crassa" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460436/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460436" id="node460436--3" data-pisa="ggg;6/7/1879" data-pisa-master="ggg;g3.116.028324" data-apath="/ggg/6/7/1879.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Histone H1 Limits DNA Methylation in <em>Neurospora crassa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael Seymour</span>, <span class="highwire-citation-author" data-delta="1">Lexiang Ji</span>, <span class="highwire-citation-author" data-delta="2">Alex M. Santos</span>, <span class="highwire-citation-author" data-delta="3">Masayuki Kamei</span>, <span class="highwire-citation-author" data-delta="4">Takahiko Sasaki</span>, <span class="highwire-citation-author" data-delta="5">Evelina Y. Basenko</span>, <span class="highwire-citation-author" data-delta="6">Robert J. Schmitz</span>, <span class="highwire-citation-author" data-delta="7">Xiaoyu Zhang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-1735-8266" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zachary A. Lewis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1879-1889;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028324 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460436--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael Seymour </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BSeymour%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seymour%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BSeymour%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lexiang Ji </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Bioinformatics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lexiang%2BJi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ji%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALexiang%2BJi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Alex M. Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BM.%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santos%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlex%2BM.%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Masayuki Kamei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Masayuki%2BKamei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kamei%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMasayuki%2BKamei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Takahiko Sasaki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Takahiko%2BSasaki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sasaki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATakahiko%2BSasaki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Evelina Y. Basenko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Evelina%2BY.%2BBasenko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Basenko%20EY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEvelina%2BY.%2BBasenko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Robert J. Schmitz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BJ.%2BSchmitz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmitz%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BJ.%2BSchmitz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xiaoyu Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaoyu%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaoyu%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Zachary A. Lewis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, University of Georgia, Athens, Georgia 30602 </div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BA.%2BLewis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lewis%20ZA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZachary%2BA.%2BLewis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1735-8266" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zachary A. Lewis</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:zlewis@uga.edu" class="" data-icon-position="" data-hide-link-title="0">zlewis@uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1879&amp;rft.epage=1889&amp;rft.atitle=Histone%20H1%20Limits%20DNA%20Methylation%20in%20Neurospora%20crassa&amp;rft.volume=6&amp;rft.issue=7&amp;rft.date=2016-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Seymour%2C+Michael&amp;rft.au=Ji%2C+Lexiang&amp;rft.au=Santos%2C+Alex+M.&amp;rft.au=Kamei%2C+Masayuki&amp;rft.au=Sasaki%2C+Takahiko&amp;rft.au=Basenko%2C+Evelina+Y.&amp;rft.au=Schmitz%2C+Robert+J.&amp;rft.au=Zhang%2C+Xiaoyu&amp;rft.au=Lewis%2C+Zachary+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460436/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460436/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460436/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460436/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460436/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460436/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460436/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460436/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460436/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460436/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460436/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460436/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/7/1879" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/7/1879.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/7/1879.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/7/1879.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/7/1879.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
