<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/7/1979" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/7/1979.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/7/1979.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/7/1979.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/7/1979/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/7/1979.atom" />
<meta name="HW.pisa" content="ggg;6/7/1979" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Identifying Genetic Signatures of Natural Selection Using Pooled Population Sequencing in Picea abies" />
<meta name="DC.Identifier" content="10.1534/g3.116.028753" />
<meta name="DC.Date" content="2016-07-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Chen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The joint inference of selection and past demography remain a costly and demanding task. We used next generation sequencing of two pools of 48 Norway spruce mother trees, one corresponding to the Fennoscandian domain, and the other to the Alpine domain, to assess nucleotide polymorphism at 88 nuclear genes. These genes are candidate genes for phenological traits, and most belong to the photoperiod pathway. Estimates of population genetic summary statistics from the pooled data are similar to previous estimates, suggesting that pooled sequencing is reliable. The nonsynonymous SNPs tended to have both lower frequency differences and lower FST values between the two domains than silent ones. These results suggest the presence of purifying selection. The divergence between the two domains based on synonymous changes was around 5 million yr, a time similar to a recent phylogenetic estimate of 6 million yr, but much larger than earlier estimates based on isozymes. Two approaches, one of them novel and that considers both FST and difference in allele frequencies between the two domains, were used to identify SNPs potentially under diversifying selection. SNPs from around 20 genes were detected, including genes previously identified as main target for selection, such as PaPRR3 and PaGI ." />
<meta name="DC.Contributor" content="Jun Chen" />
<meta name="DC.Contributor" content="Thomas Källman" />
<meta name="DC.Contributor" content="Xiao-Fei Ma" />
<meta name="DC.Contributor" content="Giusi Zaina" />
<meta name="DC.Contributor" content="Michele Morgante" />
<meta name="DC.Contributor" content="Martin Lascoux" />
<meta name="article:published_time" content="2016-07-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Identifying Genetic Signatures of Natural Selection Using Pooled Population Sequencing in Picea abies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The joint inference of selection and past demography remain a costly and demanding task. We used next generation sequencing of two pools of 48 Norway spruce mother trees, one corresponding to the Fennoscandian domain, and the other to the Alpine domain, to assess nucleotide polymorphism at 88 nuclear genes. These genes are candidate genes for phenological traits, and most belong to the photoperiod pathway. Estimates of population genetic summary statistics from the pooled data are similar to previous estimates, suggesting that pooled sequencing is reliable. The nonsynonymous SNPs tended to have both lower frequency differences and lower &lt;i&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/i&gt; values between the two domains than silent ones. These results suggest the presence of purifying selection. The divergence between the two domains based on synonymous changes was around 5&amp;nbsp;million&amp;nbsp;yr, a time similar to a recent phylogenetic estimate of 6&amp;nbsp;million&amp;nbsp;yr, but much larger than earlier estimates based on isozymes. Two approaches, one of them novel and that considers both &lt;i&gt;F&lt;sub&gt;ST&lt;/sub&gt;&lt;/i&gt; and difference in allele frequencies between the two domains, were used to identify SNPs potentially under diversifying selection. SNPs from around 20 genes were detected, including genes previously identified as main target for selection, such as &lt;i&gt;PaPRR3&lt;/i&gt; and &lt;i&gt;PaGI&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/07/01" />
<meta name="citation_mjid" content="ggg;6/7/1979" />
<meta name="citation_id" content="6/7/1979" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/7/1979" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/7/1979.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/7/1979.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/7/1979.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.028753" />
<meta name="citation_pmid" content="27172202" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="7" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1979" />
<meta name="citation_lastpage" content="1989" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jun Chen" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden" />
<meta name="citation_author" content="Thomas Källman" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden" />
<meta name="citation_author" content="Xiao-Fei Ma" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden" />
<meta name="citation_author" content="Giusi Zaina" />
<meta name="citation_author_institution" content="Department of Agricultural, Food, Environmental and Animal Sciences, University of Udine, 33100 Italy" />
<meta name="citation_author" content="Michele Morgante" />
<meta name="citation_author_institution" content="Department of Agricultural, Food, Environmental and Animal Sciences, University of Udine, 33100 Italy" />
<meta name="citation_author" content="Martin Lascoux" />
<meta name="citation_author_institution" content="Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1699-9042" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=V. Bansal;citation_title=A statistical method for the detection of variants from next-generation resequencing of DNA pools.;citation_volume=26;citation_year=2010;citation_pmid=20529923;citation_doi=10.1093/bioinformatics/btq214" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=L. B. Barreiro;citation_author=G. Laval;citation_author=H. Quach;citation_author=E. Patin;citation_author=L. Quintana-Murci;citation_title=Natural selection has driven population differentiation in modern humans.;citation_pages=340-345;citation_volume=40;citation_year=2008;citation_issue=3;citation_pmid=18246066;citation_doi=10.1038/ng.78" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. A. Beaumont;citation_author=D. J. Balding;citation_title=Identifying adaptive genetic divergence among populations from genome scans.;citation_pages=969-980;citation_volume=13;citation_year=2004;citation_issue=4;citation_pmid=15012769;citation_doi=10.1111/j.1365-294X.2004.02125.x" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=M. A. Beaumont;citation_author=R. A. Nichols;citation_title=Evaluating Loci for Use in the Genetic Analysis of Population Structure;citation_pages=1619-1626;citation_volume=263;citation_year=1996;citation_issue=1377;citation_doi=10.1098/rspb.1996.0237" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=C. Becquet;citation_author=M. Przeworski;citation_title=Learning about modes of speciation by computational approaches.;citation_pages=2547-2562;citation_volume=63;citation_year=2009;citation_issue=10;citation_pmid=19228187;citation_doi=10.1111/j.1558-5646.2009.00662.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Bhatia;citation_author=N. Patterson;citation_author=S. Sankararaman;citation_author=A. L. Price;citation_title=Estimating and interpreting F-ST: the impact of rare variants.;citation_volume=23;citation_year=2013;citation_pmid=23861382;citation_doi=10.1101/gr.154831.113" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Birol;citation_author=A. Raymond;citation_author=S. D. Jackman;citation_author=S. Pleasance;citation_author=R. Coope;citation_title=Assembling the 20 Gb white spruce (Picea glauca) genome from whole-genome shotgun sequencing data.;citation_volume=29;citation_year=2013;citation_pmid=23698863;citation_doi=10.1093/bioinformatics/btt178" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Boitard;citation_author=C. Schlotterer;citation_author=V. Nolte;citation_author=R. V. Pandey;citation_author=A. Futschik;citation_title=Detecting selective sweeps from pooled next-generation sequencing samples.;citation_volume=29;citation_year=2012;citation_pmid=22411855;citation_doi=10.1093/molbev/mss090" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=B. Charlesworth;citation_title=Measures of divergence between populations and the effect of forces that reduce variability;citation_pages=538-543;citation_volume=15;citation_year=1998;citation_issue=5;citation_pmid=9580982;citation_doi=10.1093/oxfordjournals.molbev.a025953" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=J. Chen;citation_author=T. Källman;citation_author=N. Gyllenstrand;citation_author=M. Lascoux;citation_title=New insights on the speciation history and nucleotide diversity of three boreal spruce species and a Tertiary relict.;citation_pages=3-14;citation_volume=104;citation_year=2010;citation_issue=1;citation_pmid=19639012;citation_doi=10.1038/hdy.2009.88" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Chen;citation_author=T. Källman;citation_author=X. F. Ma;citation_author=N. Gyllenstrand;citation_author=G. Zaina;citation_title=Disentangling the roles of history and local selection in shaping clinal variation of allele frequencies and gene expression in Norway spruce (Picea abies).;citation_volume=191;citation_year=2012;citation_pmid=22542968;citation_doi=10.1534/genetics.112.140749" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Chen;citation_author=S. Uebbing;citation_author=N. Gyllenstrand;citation_author=U. Lagercrantz;citation_author=M. Lascoux;citation_title=Sequencing of the needle transcriptome from Norway spruce (Picea abies Karst L.) reveals lower substitution rates, but similar selective constraints in gymnosperms and angiosperms.;citation_pages=589-589;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23122049;citation_doi=10.1186/1471-2164-13-589" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Chen;citation_author=Y. Tsuda;citation_author=M. Stocks;citation_author=T. Källman;citation_author=N. Xu;citation_title=Clinal variation at phenology-related genes in spruce: parallel evolution in FTL2 and Gigantea?;citation_volume=197;citation_year=2014;citation_pmid=24814465;citation_doi=10.1534/genetics.114.163063" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Ecography;citation_author=I. Ekberg;citation_author=G. Eriksson;citation_author=I. Dormling;citation_title=Photoperiodic reactions in conifer species.;citation_pages=255-263;citation_volume=2;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_title=Robust demographic inference from genomic and SNP data.;citation_author=L. Excoffier;citation_author=I. Dupanloup;citation_author=E. Huerta-Sanchez;citation_author=V. C. Sousa;citation_author=M. Foll;citation_volume=9;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Foll;citation_author=O. Gaggiotti;citation_title=A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective.;citation_volume=180;citation_year=2008;citation_pmid=18780740;citation_doi=10.1534/genetics.108.092221" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Fracassetti;citation_author=P. C. Griffin;citation_author=Y. Willi;citation_title=Validation of pooled whole-genome re-sequencing in Arabidopsis lyrata.;citation_pages=e0140462;citation_volume=10;citation_year=2015;citation_pmid=26461136;citation_doi=10.1371/journal.pone.0140462" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Futschik;citation_author=C. Schlotterer;citation_title=The next generation of molecular markers from massively parallel sequencing of pooled DNA samples.;citation_volume=186;citation_year=2010;citation_pmid=20457880;citation_doi=10.1534/genetics.110.114397" />
<meta name="citation_reference" content="citation_journal_title=Theor. Pop. Biol.,;citation_author=D. W. Goodsman;citation_author=B. Cooke;citation_author=D. W. Coltman;citation_author=M. A. Lewis;citation_title=The genetic signature of rapid range expansions: How dispersal, growth and invasion speed impact heterozygosity and allele surfing.;citation_pages=1-10;citation_volume=98;citation_year=2014;citation_pmid=25201435;citation_doi=10.1016/j.tpb.2014.08.005" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. N. Gutenkunst;citation_author=R. D. Hernandez;citation_author=S. H. Williamson;citation_author=C. D. Bustamante;citation_title=Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.;citation_pages=e1000695-e1000695;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19851460;citation_doi=10.1371/journal.pgen.1000695" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Heuertz;citation_author=E. De Paoli;citation_author=T. Källman;citation_author=H. Larsson;citation_author=I. Jurman;citation_title=Multilocus patterns of nucleotide diversity, linkage disequilibrium and demographic history of Norway spruce [Picea abies (L.) Karst].;citation_volume=174;citation_year=2006;citation_pmid=17057229;citation_doi=10.1534/genetics.106.065102" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=J. A. Holliday;citation_author=K. Ritland;citation_author=S. N. Aitken;citation_title=Widespread, ecologically relevant genetic markers developed from association mapping of climate-related traits in Sitka spruce (Picea sitchensis).;citation_pages=501-514;citation_volume=188;citation_year=2010;citation_issue=2;citation_pmid=20663060;citation_doi=10.1111/j.1469-8137.2010.03380.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. D. Huber;citation_author=M. Nordborg;citation_author=J. Hermisson;citation_author=I. Hellmann;citation_title=Keeping it local: evidence for positive selection in Swedish Arabidopsis thaliana.;citation_volume=31;citation_year=2014;citation_pmid=25158800;citation_doi=10.1093/molbev/msu247" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Hudson;citation_author=M. Slatkin;citation_author=W. P. Maddison;citation_title=Estimation of levels of gene flow from DNA sequence data.;citation_pages=583-589;citation_volume=132;citation_year=1992;citation_issue=2;citation_pmid=1427045;citation_doi=10.1371/journal.pgen.1005397" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=B. C. Jackson;citation_author=J. L. Campos;citation_author=K. Zeng;citation_title=The effects of purifying selection on patterns of genetic differentiation between Drosophila melanogaster populations.;citation_pages=163-174;citation_volume=114;citation_year=2014;citation_pmid=25227256" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_title=Patterns of nucleotide diversity at photoperiod related genes in Norway spruce [Picea abies (L.) Karst.].;citation_author=T. Källman;citation_author=S. De Mita;citation_author=H. Larsson;citation_author=N. Gyllenstrand;citation_author=M. Heuertz;citation_volume=9;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Keinan;citation_author=J. C. Mullikin;citation_author=N. Patterson;citation_author=D. Reich;citation_title=Measurement of the human allele frequency spectrum demonstrates greater genetic drift in East Asians than in Europeans.;citation_pages=1251-1255;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17828266;citation_doi=10.1038/ng2116" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. R. Keller;citation_author=N. Levsen;citation_author=M. S. Olson;citation_author=P. Tiffin;citation_title=Local adaptation in the flowering-time gene network of Balsam poplar, Populus balsamifera L.;citation_volume=29;citation_year=2012;citation_pmid=22513286;citation_doi=10.1093/molbev/mss121" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=U. Lagercrantz;citation_author=N. Ryman;citation_title=Genetic-structure of Norway spruce (Picea abies)—concordance of morphological and allozymic variation.;citation_pages=38-53;citation_volume=44;citation_year=1990;citation_doi=10.2307/2409523" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=H. Larsson;citation_author=T. Källman;citation_author=N. Gyllenstrand;citation_author=M. Lascoux;citation_title=Distribution of long-range linkage disequilibrium and Tajima’s D values in Scandinavian populations of Norway spruce (Picea abies).;citation_volume=3;citation_year=2013;citation_pmid=23550126;citation_doi=10.1534/g3.112.005462" />
<meta name="citation_reference" content="citation_journal_title=American Naturalist;citation_author=R. G. Latta;citation_title=Differentiation of allelic frequencies at quantitative trait loci affecting locally adaptive traits.;citation_pages=283-292;citation_volume=151;citation_year=1998;citation_pmid=18811359;citation_doi=10.1086/286119" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=S. Le;citation_author=J. Josse;citation_author=F. Husson;citation_title=FactoMineR: an R package for multivariate analysis.;citation_pages=1-18;citation_volume=25;citation_year=2008;citation_doi=10.18637/jss.v025.i14" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=V. Le Corre;citation_author=A. Kremer;citation_title=The genetic differentiation at quantitative trait loci under local adaptation.;citation_pages=1548-1566;citation_volume=21;citation_year=2012;citation_issue=7;citation_pmid=22332667;citation_doi=10.1111/j.1365-294X.2012.05479.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. R. Li;citation_author=H. P. Li;citation_author=M. Jakobsson;citation_author=S. Li;citation_author=P. Sjödin;citation_title=Joint analysis of demography and selection in population genetics: where do we stand and where could we go?;citation_pages=28-44;citation_volume=21;citation_year=2012;citation_issue=1;citation_pmid=21999307;citation_doi=10.1111/j.1365-294X.2011.05308.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=J. D. Lockwood;citation_author=J. M. Aleksic;citation_author=J. B. Zou;citation_author=J. Wang;citation_author=J. Liu;citation_title=A new phylogeny for the genus Picea from plastid, mitochondrial, and nuclear sequences.;citation_pages=717-727;citation_volume=69;citation_year=2013;citation_pmid=23871916;citation_doi=10.1016/j.ympev.2013.07.004" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. Lynch;citation_author=D. Bost;citation_author=S. Wilson;citation_author=T. Maruki;citation_author=S. Harrison;citation_title=Population-genetic inference from pooled-sequencing data.;citation_volume=6;citation_year=2014;citation_pmid=24787620;citation_doi=10.1093/gbe/evu085" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. Maruki;citation_author=S. Kumar;citation_author=Y. Kim;citation_title=Purifying selection modulates the estimates of population differentiation and confounds genome-wide comparisons across single-nucleotide polymorphisms.;citation_volume=29;citation_year=2012;citation_pmid=22826460;citation_doi=10.1093/molbev/mss187" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Nei;citation_author=W. H. Li;citation_title=Mathematical model for studying genetic variation in terms of restriction endonucleases.;citation_pages=5269-5273;citation_volume=76;citation_year=1979;citation_issue=10;citation_pmid=291943;citation_doi=10.1073/pnas.76.10.5269" />
<meta name="citation_reference" content="citation_journal_title=Planta;citation_journal_abbrev=Planta;citation_author=L. Nilsson;citation_author=A. Carlsbecker;citation_author=A. Sundas-Larsson;citation_author=T. Vahala;citation_title=APETALA2 like genes from Picea abies show functional similarities to their Arabidopsis homologues.;citation_pages=589-602;citation_volume=225;citation_year=2007;citation_issue=3;citation_pmid=16953432;citation_doi=10.1007/s00425-006-0374-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Nystedt;citation_author=N. R. Street;citation_author=A. Wetterbom;citation_author=A. Zuccolo;citation_author=Y. C. Lin;citation_title=The Norway spruce genome sequence and conifer genome evolution.;citation_pages=579-584;citation_volume=497;citation_year=2013;citation_pmid=23698360;citation_doi=10.1038/nature12211" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. S. Olson;citation_author=N. Levsen;citation_author=R. Y. Soolanayakanahally;citation_author=R. D. Guy;citation_author=W. R. Schroeder;citation_title=The adaptive potential of Populus balsamifera L. to phenology requirements in a warmer global climate.;citation_pages=1214-1230;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12067" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=O. Savolainen;citation_author=T. Pyhajarvi;citation_author=T. Knurr;citation_title=Gene flow and local adaptation in trees.;citation_pages=595-619;citation_volume=38;citation_year=2007;citation_doi=10.1146/annurev.ecolsys.38.091206.095646" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=J. D. Storey;citation_title=The positive false discovery rate: a Bayesian interpretation and the q-value.;citation_pages=2013-2035;citation_volume=31;citation_year=2003;citation_doi=10.1214/aos/1074290335" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=P. Tiffin;citation_author=J. Ross-Ibarra;citation_title=Advances and limits of using population genetics to understand local adaptation.;citation_pages=673-680;citation_volume=29;citation_year=2014;citation_pmid=25454508;citation_doi=10.1016/j.tree.2014.10.004" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. M. Tollefsrud;citation_author=J. H. Sonstebo;citation_author=C. Brochmann;citation_author=Ø. Johnsen;citation_author=T. Skrøppa;citation_title=Combined analysis of nuclear and mitochondrial markers provide new insight into the genetic structure of North European Picea abies.;citation_pages=549-562;citation_volume=102;citation_year=2009;citation_issue=6;citation_pmid=19259114;citation_doi=10.1038/hdy.2009.16" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_title=The extent and meaning of hybridization and introgression between Siberian spruce (Picea obovata) and Norway spruce (P. abies): cryptic refugia as stepping stones to the west?;citation_author=Y. Tsuda;citation_author=J. Chen;citation_author=M. Stocks;citation_author=T. Källman;citation_author=J. H. Sønstebø;citation_author=L. Parducci;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Wright;citation_title=EVOLUTION IN MENDELIAN POPULATIONS;citation_pages=97-159;citation_volume=16;citation_year=1931;citation_issue=2;citation_pmid=17246615" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=S. Nacu;citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads.;citation_volume=26;citation_year=2010;citation_pmid=20147302;citation_doi=10.1093/bioinformatics/btq057" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=Z. Zhang;citation_author=S. Schwartz;citation_author=L. Wagner;citation_author=W. Miller;citation_title=A greedy algorithm for aligning DNA sequences.;citation_pages=203-214;citation_volume=7;citation_year=2000;citation_issue=1-2;citation_pmid=10890397;citation_doi=10.1089/10665270050081478" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Identifying Genetic Signatures of Natural Selection Using Pooled Population Sequencing in Picea abies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
<meta name="twitter:description" content="The joint inference of selection and past demography remain a costly and demanding task. We used next generation sequencing of two pools of 48 Norway spruce mother trees, one corresponding to the Fennoscandian domain, and the other to the Alpine domain, to assess nucleotide polymorphism at 88 nuclear genes. These genes are candidate genes for phenological traits, and most belong to the photoperiod pathway. Estimates of population genetic summary statistics from the pooled data are similar to previous estimates, suggesting that pooled sequencing is reliable. The nonsynonymous SNPs tended to have both lower frequency differences and lower FST values between the two domains than silent ones. These results suggest the presence of purifying selection. The divergence between the two domains based on synonymous changes was around 5 million yr, a time similar to a recent phylogenetic estimate of 6 million yr, but much larger than earlier estimates based on isozymes. Two approaches, one of them novel and that considers both FST and difference in allele frequencies between the two domains, were used to identify SNPs potentially under diversifying selection. SNPs from around 20 genes were detected, including genes previously identified as main target for selection, such as PaPRR3 and PaGI ." />
<meta name="og-title" property="og:title" content="Identifying Genetic Signatures of Natural Selection Using Pooled Population Sequencing in Picea abies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/7/1979" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The joint inference of selection and past demography remain a costly and demanding task. We used next generation sequencing of two pools of 48 Norway spruce mother trees, one corresponding to the Fennoscandian domain, and the other to the Alpine domain, to assess nucleotide polymorphism at 88 nuclear genes. These genes are candidate genes for phenological traits, and most belong to the photoperiod pathway. Estimates of population genetic summary statistics from the pooled data are similar to previous estimates, suggesting that pooled sequencing is reliable. The nonsynonymous SNPs tended to have both lower frequency differences and lower FST values between the two domains than silent ones. These results suggest the presence of purifying selection. The divergence between the two domains based on synonymous changes was around 5 million yr, a time similar to a recent phylogenetic estimate of 6 million yr, but much larger than earlier estimates based on isozymes. Two approaches, one of them novel and that considers both FST and difference in allele frequencies between the two domains, were used to identify SNPs potentially under diversifying selection. SNPs from around 20 genes were detected, including genes previously identified as main target for selection, such as PaPRR3 and PaGI ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7.cover-source.jpg" />
    <title>Identifying Genetic Signatures of Natural Selection Using Pooled Population Sequencing in Picea abies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"HVHmUoPTZjZoJTUaaKBUtk2R12rQ888q6i8mBRXtj9I","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460427","apath":"\/ggg\/6\/7\/1979.atom","pisa":"ggg;6\/7\/1979","ac":{"\/ggg\/6\/7\/1979.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/7\/1979.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1979"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/7\/1979"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/7\/1979":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022O9I4F1R4HrsDkcXA7eV2nFPHGLHKMnBq5DBhwrj5ZrLdn\\\\\\\/8aETheiz\\\\\\\/YK\\\\\\\/HtmMtNap+cu+MQ4P2HNduuiCtTlHAc8fAmSL0duj8CMlJED7QbuecNHuBKyt1emLavrNPZAmzTsBp5xc+OxoRAlVoNo91EoH608tesbWWqIg5PJK+NAt4U3iW39MM5PCGVerWQflGYMU+cMV9Wdokrh2iXIDx\\\\\\\/bzxK1z8CHpQx38uKZRLdmzxcQ0F6GQQPZaARsMXJoS2eXYXiUc9ew6b5bcEq34IpQHhE7KqjTwmmEy3KlFytQ0lWjmfhfnpaxHWDQ2Nx877teCr5yJ4cukbU0MCkAHzOJEYLZs6cbeLoMuBeyjbkkXoKStbcdduq4pgiFr6EfGQ3f6aNUir\\\\\\\/rE99OOdbteCjCAMkPx\\\\\\\/uk1dS5iRb1h69L3V7jjjxc4WG306hUb1QTolDBAsaZb3O\\\\\\\/dKVkW8xEqOtW5CdFKValbHBlzafM5zAKeGe12PY1fGLUQS+DLIOAwQgvsewEQVLYKbXJsZ0C3lSoL+OKcYCECv4y09U1iuNeZMLth7RfEUnxiMkBGHkOKmmWnJ8gzWzHBp9xYORwfEs0\\\\\\\/K11MqAvsSolsRqKS0iGhEcpmndiVl6d76CqdNi9uDXJkSJO5iYVGJYlm+CKF3swp3AormDruK0w\\\\\\\/MFS96upBdA7C4Jtoxm0JdJ\\\\\\\/IQ1bW8ALvv+\\\\\\\/enKuseOz\\\\\\\/Va2U\\\\\\\/2W6negPuTCsKNtNt9PPJxuZFe6957YOyz4JggcCsitlSqQ2GUKbiNEab\\\\\\\/BrAL0XoDz8mFu2YEyBryHnG9F69PJKbRIhbZtC1hjHZS1+iNVFWikgMBQ4mFJHSKpCuH49xeVMHeIHXLxAMb2LHYVwPNvqh0hWpRr9bYKXW91m5AXeiBwLLLbZyneIz3nPShKY1Ek3StpDaEywXvdd1FYY5NCnfPEmnb57mz0QbgLpD9bSVnln1zGI++38875fuOpr4WuvTr\\\\\\\/+DbmS0h1eSx8f0yR8xGyrxow4daAQfqkSp3VP3+xI2408H3Zcm\\\\\\\/5dd7crkEMn+u0\\\\\\\/ljGq7w0zfriZH\\\\\\\/RU2MxLu4zeYaL+Tm50lR8+yhiaNf9XQcFYCDBbWXthfLBGbsZ2b8kEg4O6CwTzWDLJvinbsjtAIpBjvS9YbkOeOf7CPAMaikoxRkz5LqNarAsGNqkj82x5spJtBOdC6bTEpzqx+wOu8JWmnGzjqGW27JJRcaW94HDNW0iAGUT2Uxx1i3oCDhfRzwpMfj5ifADRekIYb1GJ1LscOy+0jY\\\u0022,\\\u0022iv\\\u0022:\\\u0022P7nCRDJBVlMqSvSRKEPglA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002217e01f57071cfaaff299c1721d30f0d90ce9c8db6d060c32bc18910efca086b7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sBGXTZ9OlkmprIlQ014ajHfnYpTbq36y+GIj7Rtr\\\\\\\/PvE+PUZicCb6ETyHVm1LWvM6Ju+VBRWFOPBY9YKY71D7hZat14oyAVPEFN3BJPKsATpVRxzsFzog8wyBUeWPEu6L4aEiNs6m6dysxj8MjNAdBfHB6ZrUcQxTBE95KGlKigH6AY9K+Jzk4VhGANYt40Tc0GuHY1OEfipu4SyweWjRU6KhcVNoWQmaP8DD25F9i6EXMA2HgDBHeT6ZYSfS8nS4ntsZDZekQsY9pk3GC1z9GFogbcZdwvY6p4dLlFtPZ59kKjk08rCcboH8H5A7gkWS6\\\\\\\/CUT\\\\\\\/I3Ew2B5qOkGF44EWbXR9CgiY95B4u2R9JG6vOrzwKdZ7\\\\\\\/L7JYAOTOvJuOOSt\\\\\\\/rASrG\\\\\\\/UDffsmPEOtW6rvFRKDYZktmRuCMkab9Q\\\\\\\/IqYvugMQwFLfhIWYskSlrYPDFPAsAnAYDC9aLIU1vnv48SETBVBvZFp3MWFRgkR3hUbuLNUk+pNEGYol85PX8JfUSpNfHwlEB19kKSDv2VLRyxt2YrA4CYkEaq51LG7kz6bsSd3V5oTAJOfcLWf+DByQEzGwZ48NnQLf1b6Pnmri3Wh\\\\\\\/X1C8bexqhgo4Bu0vGmcCNQDA+CML2T3izBWaVCZ79BT3ccAukEA9WMlEgMAAfFny1yt9Q6tVO93jejQxzgGZxLt8bHMJo8msEsa\\\\\\\/nU2+y+TUiwwWZRTkbbpmHMCMf\\\\\\\/hasFsak3JUMDGKkG1xN3UDScMgKTPmVmiYTvQjPJrX5R7Bqq0iQ0llSGFBtPDN4VNl2HzOckEwWlyBpck0MiuddkUBvYIu5C32rd5o\\\\\\\/\\\\\\\/CUTkDAIQs7AGi0oyw\\\\\\\/Gy5Urg3Cg+CLtC6vEIq7t2J4heUlyLGHlguAFQe\\\\\\\/xbiTVjPC2LC2gHV2bLuymkh5L7nk2\\\\\\\/opf1uF1mV1dJBcEm65xvKDsMN7OEq86lhmv6qvXNJjoRFbUG\\\\\\\/\\\\\\\/KjxFqUQGehV3QMl+IOpFgsY59grr6bXF5cbblZwCa6MMC7sEWuQtnscvTYstI1JWTPFDB\\\\\\\/Z4GF4f72GLm+O3QHL183n7yeN6ncQ7UMh5grgoMiELVxJs2UWZxglBJfhxb0BJw8qb+dkqDEFVrF7fSBPXPMmVRjY583sbl82qAxQ+pTINDazC537S4AZN6W9XlTYmFRvMbWT0KNBjQNhDkd5vqacA8ckbY16h8q3qOqQXuPr5HQS9AGuRJJigpbZ92Vo9XoVbCB9jRyXkQm+qIUXNTKzMcpqr4KK2JoJidU1S2IppuQI66bVF4ZoLp1LFhfYsyqL6sLJLw3gK1dxpNMtxGPsbATygLsN6d3dORY\\\\\\\/3iTqY6LNCPJrHYn+4zBtMnbEGeFt80a4Fnckku9YefVxW4GFEins6DLwzT6H\\\\\\\/xOdloqWH8YsdGnGrGxPwzuccV9ywIeURtzHJgHlORUSfDG\\\\\\\/hPnj0ObMuHgnRbtt2bMZtCWGWF26bk7QSCbZq6DmqAWA+G+blWDSV6iuLsOw86WS3g5FEb+Skkja58n8rzLcPu92qWD0ZZ0AOktaZMlkxrs+SqoHhBheHIx8yF7lFkijSsHCdkpEd4akg7lMkUAvK\\\\\\\/w\\\\\\\/1ob16D2WrSQiq4JSj8WsWaPdhV4hgcJnVKZHdvTR4A2PFPXKW3rg7PrgN5oSsm8icSi8ZfgH8X6JuKuQtOxZmk1SbhGWXukp2rbdZnY0z22oR01yz7asNCGbHivl5I\\\\\\\/ynVbFUqUmBYbtplCJl9FkFjOD2z\\\\\\\/a8Ex7yz+vvAH3Tli8sPDOpZ8eIfKBuSp2fa0EqMPHX+2rA9LNq9nryS1CN\\\\\\\/qZY1jpGIAlYDCi7UYr8KqJj2SFcVj2m1iKUifrRIOqqcy8YklyFTdl1q3\\\\\\\/KoepdsJJY+SA75EPqBA1yP\\\\\\\/G4sEWY7x0mTvQyNdoaaNxsL3V\\\\\\\/0q1q4+IZho1gVwD5ygHAAflmqJaMjRYWxxpMM5QBZagAK2b7w+5FC82VUEdyDjSs\\\\\\\/lm+AUDhfg\\\\\\\/2nGELys8nvW7PpydUwCIvmC3n7h0WH9V6aSc9yFJnVzJcjez42j0LwbFn6P6N0aoYvhL4lI+ZrSb7icPXOMNYnaNWK3JqAdbkqyUs7zhxZhhTKJNhKty4tywmItTE+SbeTTV\\\\\\\/kYuqIogsAPinwwausl0eMlIJ0KnFIvNSonVoHTCAPcyuDis7VouDzlhW\\\\\\\/P0R3wIc2YHsl+FUU74CEWAiy0DrVcoKdlKppNd6TALcUy9NWe7FseVjxtvjLsYuqWD4\\\\\\\/OfeRrcM20LgMfFXMvfQVRGZ2MGV3hk+OsxvoKvrLcwrNWxPIXEne6Ez8EiBNQfIEb6Wo5Ut4L6tr5EF8m1twywPgXkonzFjGcKTf5EyFuZQ8D4osqnXsP9OMH9w+GVKSTZEtluy5uIwGqxLXxLZof4C1MfjA+b17Xy2Wz4sLrqiSCuZS5rB5bA74q586Fi+aS\\\\\\\/wrOc4OVgjsvbUuzJdfL1e\\\\\\\/T4U\\\\\\\/mZZHbKubfTYEO7OdsJl1bSHBbjXGWe8usX2Ir512Ov7UOF1l13K3gpoZi8eJyRpd\\\\\\\/bnHLZWg17\\\\\\\/0lpZYfROczmBMjGEojfHxLe5E+853QM7dMwvZyEX95Pq3UsP7rDisdm4iSb1vPYXM2Jf1q+zZUdhzkVn9jwEHh9qrIUZA3Bn0RJ2gC+cmwCLyCBQq8O5AAeJCX42KNbQrJdHqDLeH2PJ6AZirdDNEWcAu0CjG3umwxXDHdJE1c8MHTa28H6Z6678jvVZs5hpBar\\\\\\\/HZ+gm0yLoFBaGhsPbtCdUnKK+qm++nV6+UJtXkqCauApw9MpkAmBhU+rwJFjC7yyK8qQmu42kVo5WFwNZWec\\\\\\\/+zvek8Nak40GKM3bIod53fMalHp\\\\\\\/ElJi7aWSq1+q2YTKYeE5WAO+P0S79YKJsF\\\\\\\/ZWNOQO1PBkgIh\\\\\\\/GMHIfoh6WzJ1fI9ofmlSxqtXN995cQK8xCWtc9xX46hjsgdFQz11YU1mMe6Q5D08zEOhIthIj\\\\\\\/rKJN9qPxZ6mqznuxFHL1rUL7HFMXnNXMSq++ieJUKcZz7h0mvUVK8UM5GVpdOMthS9iGOfeifWtqoM4D5QEMWd\\\\\\\/l\\\\\\\/uR0ThCH38eP5CID8cOxQ8lMme5\\\\\\\/6uW9uY2tP2jo8T3Lw+LnKjxeBV8AhzUqdkEpxwvlFG7W45cm4S+5SXv8j28RYIRcTD4ldmHhoiVfc4Z\\\\\\\/PwzLCQ3PFrBoCJF5XM4p4fxEWKJHNUhOXhbyoC1nyWcS\\\\\\\/+UKm2L0P0bdAzNVOxV4ZUlGx5rYfs1d00rofhSa7yH1LYvBdnNq19kk81WyPyvk0VOd2qxubEkMKFSDv8I\\\\\\\/QJt4cI+8uRU8H8jXtNVOpQR1gkkbxpUnWpda1B6aTogBm+VmXFCXvIn+U\\\\\\\/pxVPupe0vWqoqOogThhX5KULlYkFoIhLQN5\\\\\\\/yNyPUnT0r1xgIowoRQFeIJnPkt6svcsN9GT7vQjek9nqCijEuPv\\\\\\\/YXKOu35N\\\\\\\/3tnvSCmdNE8GdQwRE6YV\\\\\\\/GWHNSzhZqChWVp\\\\\\\/q6uGGnnXT0kyL7\\\\\\\/jOjjLdvkxxa7uUlrSw4n7AM62\\\\\\\/BHNYHdCitbvWyL2NI7WskBDkwHODkQCLncq3ijOZDCrMqQ6jY6\\\\\\\/Ny56OApD4L7XQQ04dAWBdkwo0a0vp9k3cTMQlXJy1+VQUQJwUH1WptHv7esCc0IV0J\\\\\\\/gjDTnwWaTu2nDk6COZyPP2v5EKOInf7V8bCK1ODOTfxgnGP3P\\\\\\\/\\\\\\\/WB4nmb3c3y\\\\\\\/LrkHGzXDYSUhwzvVr181g8PIrqfoYjQdR8JdcZznvTOZTDg8CoVxwCdTPpV2qPI8wFQ3YwqL3Pg6srMurs2+Z8zLWxjKuN92wzDgknyMP+QWdk+x3bjiEb3nvNYNYxoumH43TxZc\\\\\\\/CpXdC\\\\\\\/QUJpMe4dRfb0KjOYpET4kkSfxD6JLirIVLfT0KP0KH67KFnKHybP638BSEYxTB5qDbZF8Jk+x141X6ctNW+kXXklxLjexOpWTkwLSucywhw4svQ6M68XGd2UgGqUmTtBRLu6RdCDFA9Alb+\\\\\\\/6mB5lwwUkVE8\\\\\\\/Kzrnvd40zzVNIpGs99ed23mDCRC8\\\\\\\/SiZuqx6go9upOSQh\\\\\\\/vQu9B0eFbXcZ0\\\\\\\/CI5KBjhu3qU0voUdOdEUTuGyzEWoh7AM+5\\\\\\\/I9qzhllzVEjYFwYvkPjU3u8Een3orgZMm7MTUqYJBJzDh4YT4YxPFxMSU32tDd0y+0WPDU0VMcxdbsmC2Fz8BvC7lex5QGvNBxowTZ09rSfpr8gM+icUjlK8x0SjPZwjCUj2uVwr\\\\\\\/mFuZYIn9mAfGiaHx6O3tc5I64JrO4j4t4IItH4qvGOggEAJ0g\\\\\\\/5dXOubG3pGUPeV0S8PaVOJ+5EsMec8t3QHNjf5bxlZQxQKxYL6FCEhcioSEH4jIY7Mq7Znd5iu+WaWVlZJrqMtA0FJloxMqLnZXsqV7+WIT4R3ll8YxJroJgKup6RxfOEqXeJFGtRnSdMQeL9IF8W5o138JMNvMJ7xpgNTLOIvLjFKzWdJYjOMKTek5XtQ\\\\\\\/de6T\\\\\\\/G7XcrOO+YNfw93Ye10Kknb1k+Lh7u9+QjEkFfEZyf+KjzzW0KAhRqebb41QafNeejtxAh25sPaasq1qA7PsGlSmOuR4luzTGJLsej+DDJxry4KWrXXEFJlGyj56aAqIkqtfPD6nNSIwIkyaORTKbdjMKsYZWvHvKE1PWfGPAFDs\\\\\\\/oj5TcM8rBqjaCX6xjla8DYbUdQOEQlRo6k72PGJC0O3Fapx4qhq+LLQI2D4sBWdox4C18yCLvsOHxLAYTKAdo7exyyg\\\\\\\/RDnP4Mcvdq99QsXTiYYzJthL3rPF87u6ZDSkefw1YXdiuBp34MXb5C2FdeuMoU9K3EtInsNDMkrgoEZmwrAQMRAL396b3KkhuRAnr0v4l9SMWllew8fmLZRng0+KvidWsgQXx8GPbsDfUwKB9qMqabbzOh6P8nvlc4Wt35Wpz+wcYKlKDrkdnKQy2hnPaw\\\\\\\/1IpFgvTHP4BAhugg\\\\\\\/TsxdDXC7ItfV3DF6vIubKn2ZLRddV7gB5l92N2U+a2fNr9Vs2uOuLyCjvqWJi1YV4vpWrIT\\\\\\\/sCFlEy5xAQVz+lTanfE0AIVsJcAVxp8DKmsctII9saSS35XgWMzuQXFEZSa5qChqcSFrLXpOoiSSJ6d82Bp4xLheXst8tEu61WIePcVPLQb4t1kYeuMJvrRnmiVsWiUiMIJ2Cfit0lo93DkCeQA7Y\\\\\\\/XYDEM24syFzDNvy63G\\\\\\\/Fm23qc5RN6iR2uCisrD21HI+wWbJfEcr2dev+ixzpNEsxUYs7F8BhNYu9Rb5T2nwB0st+\\\\\\\/T2t146Pn2RhajlQAztoYInZ4UVGlamM3VEgagv6oNM5wd1og2f4DTQZiP10r3SoOp9hW1\\\\\\\/T\\\\\\\/+5U8huUUrldr\\\\\\\/eE8yW1eLBq\\\\\\\/4IAmB6wNcWgdEi8vxmWhHiP3szZPS0XecE6FQjvejiQRT1LSVV0hURC+aIs7cT5cRCSa7Hvw2tAgVLJ9ZloYawwlzFMb+BxTgYzOtFZvP1rf02pbf0W7xPAWXIDK4dpaBQtuTg1GRi7eZR4X1\\\\\\\/aO\\\\\\\/Qto54DeWMuO+oLXXyrRSwxGLW05L5Svx+6WBjfis98DcCAaG\\\\\\\/mgS+ZSRsjnED9\\\\\\\/BV1HwdftCouskAsST28nu\\\\\\\/C\\\\\\\/okwRuEow2ik4d778VMLuUeTYC57w4lZ5OGxqSo82+sja38tAecoBPo2fERB8ft9upJatTL+hij7L9cfyY26VsNJRpdu49dbioSeiCYysSOy7W6KF9CexG8yaKgHc94rKMCSDP+GjKiHy95pvXCi6pBTLH0nncJukA6fxJ7vTmub6r9IFlOkmodBDEDM5N2erJiQPIGbdwnECTGdy0bn\\\\\\\/w\\\\\\\/WsDzPIg1eCGWPCfGQgBH9rDTZ30VJeGKYcQmAFMuz1NDL54bAwcsGH2PzgmkfIHiqUBpSRr5fQvGzNxO8DYPAPl0rbCqijbWUYYc1ZiGh+Mj67nLtkCp4ryQM9HowUEUHI9hpAfb82Ggt\\\\\\\/XDMS1jMGxh9lvQpedM8+oU1NbhAQyNHqx1+3w9hapXS3t5syHuLJ8KKnuLbBgL6rOvDzhnKmOhFjfRnPQtQQCxbT4As6+w4VSGRI1N23HIfsqQ\\\\\\\/tJxvTc4uc3P\\\\\\\/fEYOLJhnTHXlQ66skStW7k0sCsDs0eEjfIO6XayhiZM2ZJbGslcKqfU8mdXfycsC13syretzTh3ndhXLMrZiDcZNGVPnl55HeIV0hJOGhpmYklNPWup7LvolT8AR5qNgHlgN+oe7aSqaQY9kEuxag0BYByWgjPp4gBbAeonCtX12h5mwMrbOaqANPCQ9wlFcuIwn84TmTWWwxWOZyrDlPNnBUUPBfxt95T5qupYiy0hJCkRvrwWNyd8ntVGD1HyJyUgfjCJp5v97x9EX7OGLrCIu3IJiPReWyC9rTO9epZoKSr4jScU7uk9odbAM0sJzy5F2JG4mMEqbt4ozfaOKEa0G3G+UxgBfswHqYmobP0Ae0NQHJTVaG3CiBO3g8KZGkknLPzlMoWLRYRmBnsAxYDgqsOQRuApKJ169P1jOfpyezQrS6i6KJPCLYBJ8qA6Q+wP9pG5hFxWwJMKXu0+0V50b2l3jNWV2gjEF6cCofPX5F9m1S9jWtfg+uZ\\\\\\\/qkgx1yu16IEUQ\\\\\\\/RtTqz1JRFY8XSWISCxCyjDTWFbqxthppxMlHc\\\\\\\/zO0oyzcHY9YWnq0kcqyr1q7Ua4XAZ29V76ARgJItCPE3Ps2NkxXcQY0WUuOdnx37lLmbmmca4CwRmPmFOQvNKyXOJO2apjxtwdCmBIoKYzvIaawYtNNG4wBCuoaiVJu5r1\\\\\\\/nN2sp37rcITcE+LJVbFeMc0khJXgNnt1KeCOzJN0TE5vcMFHqU3BPxcM\\\\\\\/5qODdnurvS5XVeIaiPDrBWZ5rPARVQepDIXd5uMSAeODuZSOCpNgzrC98CCoNLkd\\\\\\\/plv1K\\\\\\\/nq4GLcVJywDzva6Uq\\\\\\\/8KVpk0G4UIATVoufPo6dJnMAKIxuAogxetKTVyir+7ZCsR2wHcG163V5NAOMcYKMSPI5SOmoaUg483Je\\\\\\\/8NlPJ\\\\\\\/2svoxvLDmwhiGlINLvXNJnUvQ3i7d81PMH1O15ppUTD9YtUEXtWDFxH7eDa8e7G9zhSKG\\\\\\\/MTg1vGmtmOXI0KghzHw8+s22BhJljw+pg3ME4ghDJy5ZzsS5oLJr3Dx7bcqpXoOp0UPDWYliavUFgr+HdBAS+n1tbySxnDRObuAOr3urPwSmYiwrxFaCaXiQb+VksTveGrHonrgHNK9MWEMtcCrLVr\\\\\\\/NU9ei0HW\\\\\\\/CzT4b2sdmnikJWy63jygDY8ryN+d4I9XcT65kqbwgIOrQ\\\\\\\/rUzjh6LEM4NpaTpI2oncDjnlyjeT1dNghAqJDZxZ0LjvWN\\\\\\\/w94K9qYYC+WBD+fBV9W2fyXFHuaYkXmlQD8wHQ2E+xo+McNUNzBjuZpRUs7gaXV\\\\\\\/EqrkMUJ\\\\\\\/abwtDPe9fDMWOLDCjDfnXH1iR3+PgkzT1bNLXhljLDvPoLVuEsmfGpJL5V9jldHYWeaF0ipskUSSAMYl2KyqdHS0rvdXbORcSqYLEayRdpzvs2eLVRJNJ2\\\\\\\/i1fqI3U793r43ZxXUbYoKNujNU+qTiGcGYlhlmatlxzKpb9eVTWXIvrlLN04aaC6FBiihBgdYHIXNKyrQxjh+wEuzhWKgbQ02qCKFXOdO3mNvbgd8pMm1QqmEgUNHVmesB6r5TSJ0EwB5kGm6cD2LMfTzGlzB+TCrOV0ObkrQRTRR1Ni6MWQAs\\\\\\\/1uzvgv+WZ9Mp+d1Q1FHt+kdUbBXPVpJ40yqlLHrbNBjCC4wwSr2fo1uqHd9Hh+N\\\\\\\/v0bB6sPfslKRnIJ6lqepHUFKsGIldHRpKIzACJ7CFHJMZ00ya5fvEIAdhYgK9JSTur6NLDefobXXGKyfnrXRKNvs+6DqHamvAuteOURv5YAjq99CQ22hAZM76BKtKZwcFPASmqu86XNEzrhOStZk7baYhDMNKAh11u4HDY0UcY0bS7MipIXSwhjIluStESWcoxv24JlcUrTC1B1NJ1\\\\\\\/\\\\\\\/5sKu2VpL6NT0K1axFq8oD0dl6HOm13NHDGjwqmplOzKjsHt2f7oHH8XZkpGLoRuHZP0pbPVY24AapGWSkhIGH6JQWcHJIaWt6c\\\\\\\/Xmwban+6Zv1HzQqzziNEj2Q13l8rWL4XzWszzKcThfEkj+VSKYF8SKoBWAv2BivMplewnUmfB4mc7JbjHYWR1WDD0T2smBTQrPuEM2YnTbNUyMM3w1NhCrnDXm4SzW0pG2iZqW+BaZ1BPgzgwKmBLBT+njufxwIc2WdTlyibYgBADsPoEfURsRJc4jGGZGiwf72Inkyk3wJDmiMxtMU0A8bZAaVYf3luLJONOw00u7FkKvRtzobeUeo2c6lEcY0eGh0E0uCKbf51kMaSjtksDSKgDrpcE6A8cFBNQQG\\\\\\\/o\\\\\\\/2Vmfvt1wXNg6+YMdbqpQVF6kX30AnAzSOh8Pve3rdhE7VUbEpdQJAKSlXEXL76TsNt4PPuTTFt9JFOuouxsX4Fn5tMU0e7or7qtxEeYd6Md9pcF\\\\\\\/lZ36cNv6c35NtleJf1sJ3lqNuhCfHd\\\\\\\/2tSf5ycHUS4qip29IFyEUGk4l0kdRZbD1nxTRv0411X6hFKzTqsX0SzUnwDNKulgbNWE7vwVI5PCUIGjcIbcA\\\\\\\/IdNF0L2h8mVhXk4zyeT6Rn0AQ80cnDIlG9To7j0uTdprj2CHoSQeNCjOsBHuF5GGYMCM+20ZDRxXVs\\\\\\\/E5teQ4dVYumB9mjRfTJ6YBlsXFSoDKME4HoBYwf18YZZJ5FQdaBqQy+OwIQWvP5J7ZMjTnoKdHeFDndbm\\\\\\\/wzVTIdkY40Ozhm2Wwa71Isa6Rlr+NN6yb6lQsdv17hEPj0ak9DSKeopOnbRnllYD8ozSaZAruS6qNijimx8XUtrxhy4dpY9LM4HWFyWD4IOHhATMunczjqBbjZNteyb9fJ8gsxpmpj6IrKPyui6gUrHZ\\\\\\\/LEd3uAsJlYNZDtbPS+Ul6ZmBMZdq6IcZJKyVsHlUhRaH0NLnx+2DhYmPa6hpZ8x4dVxBDynTKVkWPiYEDLIUoh\\\\\\\/66PnG2ig00moU1O1PpfGvQGkBwUCog7mtIX8G+H1Ox2PurvScTjWaaj95GNdmR+cBQrYuqpfXHgCTF2HS6n1peCpirvaVKp61TQqXk8JMh2UlhglsCwL+ea0Lqh7oJLs81MUGLSKg82h56IcJRp48lyxBk82UDPF745\\\\\\\/+MvEoKvwHleQFtkfo9uWMXx4q4rznkft\\\\\\\/WWYDjW93PYZ1ZLLnNf1BoFgCqfHXKqLZhMaFbzYcSxgOAtsVQjA4lrUHgDK3zrEKx5FVoDZsJXN8SES4JzZs1+PTiDQuOX8yoqq8AIP7H6VsAsu1godtuBnYTRH4fUsYvYwJm\\\\\\\/dbmQh12TLdLxWP67+9nOLpYbxm8YrV6KDja0+lsPxE5hDBrOPtwOSU5cRwBYBVmRQ2pzj06Z5CrN3POB3DIuKxB+JW9SP3bRy10ofdUeHD+vC9r+JuvKir82wBJ77yvi5wlp1oLETbLUQ4TH\\\\\\\/bnqZQPvbrgM\\\\\\\/BPadqt9eFwt3RN9yFCogDQAfCLKrLNLrifZtpWqxVPWm9F0hC3bbU7D2YK8MLCjn7Pi07OHDxPSgYm7OGmL3k0WUSXr1Nyw8KCsbLIYij8xSVOkKIMS4nwG0udQQeRluJLbAbdclfGV9KFG6PmFabcYaEyuPvdfJgT6A4Hy2X9J\\\\\\\/nTRUuzXJmlrUOnyZdFdSukTAkWJkPUgZPwS21woMNYgpsTS5m8u7xotSBE8H521nYy4drsYDckXPdXG2M5uV6VFyvcv\\\\\\\/6aCbNghjMVO8TiFK4tZi3qMuCFd5Yx7STuUZtvVBPSMR4LPOpmFaWbu9NzFAM7JOHU6k+ql1HSD2lId3NOV7BjYK1c9PRmRk51iPvRIam+aQijAu2tGmYetyM7ZpIyNuepN9B1aSETN67eK50iXerEQvbfKtB\\\\\\\/ZQ\\\\\\\/PBQ\\\\\\\/zp+NdX5\\\\\\\/ClCIL2m7jm+v16eDA8kcLXfwtnl5OVNYQ9RU32xFwGng+pzY477VtD0gK7V8TP8I3\\\\\\\/VarNpg6MvsuyYk0EN0\\\\\\\/rUSkhAdBw+jJcAEbcheIbmACU8KETpzs7XkIGd9cScvrEf2QnMrlZzw\\\\\\\/jM716gjFBbyDsMv+zDQ+FHVjpSi9uATbv\\\\\\\/cDCklFM+LTe5tsVRBYhp7eWwXh2Byj372zKk4uLN9n2nvwPnBgTjdg2oAxHSDT52TcY+YTLF6pxL5zCDrcS\\\\\\\/YDIArpygi5UcVWNcjHAQtKgDHZUfoMPJtYagl4nRY1xlUMj36mOUVMUYFBOrtN9NSiGeKEauElTj8hQaZY4qh7e8tmDhtalzXxyhq+h4BjLdqfAo9M3CuwTvJ+\\\\\\\/p85n259OZ6z8d6IoSLDzWQ4y3V2wT3XuDKNGvTptwPLm2Vd\\\\\\\/\\\\\\\/UrVInlmCNiG3Kii\\\\\\\/bDXSWDvXRqPH6rv6f1ziiADHeMrZyM3rdJzziawQPqXEwAZ9CoIDjv51iTfNTS0b16rM397I3W09PvuQ+2nTW2aiLK5cWJYiV9kFrs3sDYasfWDbcAUxcHXbRs3ScTmwjCZe4nHJFGnp6A2R3Ss2e8Pj5AKK1auQkBRB6Q6Lu+kA820CtRT34F4cSr5QMZl19sII1jROf9SKOpDPOBakuYev58Wyo0yBNL++69nZHHqBFJAEtzZpA4ZmExIqOQUHnjVS66tJv9\\\\\\\/Uqaxd7NQb0nlF8VRwfeuUYLntR4I4+cpTrQxi512hSvfPboCaJlcuG5ulB9n0U2cMKbUuzwuTWpa34ANodk8pr4HeC7pqqAY5tyVrUxsSWpDxU7iVi83xZkXvmgiwC9EQFNYyDAQwsmYppfxAkpW7Xasis1MpjeXzeHDEI74VTreKIKlYl3qwZAs3vSXz3ARkbNsTkkU8p5xt96Y3aQzU6AxHOtVQxNH73L5hUKN+R5w\\\\\\\/uI6ARyk3069N0KYnfoSL\\\\\\\/dt9\\\\\\\/HOtRgicvpvquNq9PFpAL93m2GWv1QVeeugMeFJyYRgpSVEKESj9NV9mZyMlP+ReDqP53uo+IZriQD8MYaATz7D26JS2678Tii0Tg8\\\\\\\/1yxDE2bHzqCndG3Rh6sn878inW2ZhdqAUX9AdCLTlm5XD1ZZgBzNRWNiQST9MW\\\\\\\/3IYI6pC6fC1J43rrtqiVf\\\\\\\/T5nJ7\\\\\\\/1WMV0y+yNO1FSpf3oPibETd\\\\\\\/WslNlzLuWMTGpwGl\\\\\\\/owiyJwqnpuADQYRxzoNKpy7pjk8dWzjBOhsFTeK\\\\\\\/nUx5Ck2xh4S3bAvZfyV5ph7Ycp+52wknuZRe0TlKJFq63817RKf2nCYK\\\\\\\/l3YvjLSH0wm1vHBQzgAReoPoztn9FxgoW6PiWWG7LwLJ1SPi1fpqsnDOZdJGw5utgcNKFSv+Jm3hH\\\\\\\/zAYXGivBQBRqVXM9V8fm1ZZpKGAe0UwGynN679k0SXhXUj7Rga1FuuIn6AY6hyt03bpjYeIXeq76+nPkAl0hHaux5q5Ml\\\\\\\/LyQ+hgUn8A+hcDemV+KEsiteYFGES0rRgSkqkY79TKOH+d+7fLSNBaVF+5VsTV\\\\\\\/LFBswKeIyqEcEiNskksumuR4eaZcXBp7cRZJsJ1okHKdD1NNNS3R5UcNR8HuBbCmoZRwayLlokLGPQyJT9BCG67sx+cvEZp4IVpwa6WtoeQP8xsODo2PLFhvNTPlK3f+icg4xW+RRj\\\\\\\/XPh7D\\\\\\\/Dft93BFydDyUIH5PQ2T5zrg+MolSmJgoAh6KD4gfBrO9w6HxmajmgxB5Rq13m3ILeJbJmBCGchi+Lp964RabvkOwgxOYCRiTE3H6j3csW0OSwNkdC0P0b7RP7prD0\\\\\\\/4\\\\\\\/F1Wui8VWGH1MDh0Etbp4Io8ScbeEczRDaJpy3LM8jU43oS9nUqPcFbXxHI2t5VVg+eZWqaBcLsyY5xNx0zsrfmLtPu2r0sw0I6zYvLrY2tfKoQo6r8rivd0j4aI9IlrxkwR1G2a+xxq9T7YLxLn\\\\\\\/5q+z9m8N3XWy5ydkEfQiAKIJnNHnqSc9fYmVjiwDmaIwvlWLYJLxqIIHnxZ5sbCH0cxQ1ZDnD4Smgc+l2s41ppSt9bpZOxErLdt5Gi\\\\\\\/JLeXdHkHsfRYz9lWYBRn2LQDPAIpk+6eCN5w6oizKIsSDYvOkSzdJzV37qemDH3Jco6pgw5cjFkmqJlp\\\\\\\/d\\\\\\\/2TIdfup3PaEEidNdNSS+dqWSYFFNvaHs0hVVse27rdFRCE1KyV9xovCVcYjXy0AUTYqXlYmrcKNWeDfbe7gAaz5hJMHjS+SPvcDvxSeirgopBaoAJl5ShaZPWYHo+3ahTdvF8S+XjfCn0Iwl2OpQoqOd2fw746sDhzEZgIHmRUcH7tM0kSc09nyuK4bHfG005yLflanHVxSJ6+n00iuEiGZVZTxsMMd\\\\\\\/biYJqRsnazskRqhTJ8nkrVd4tBDSCARQoSKTphUo2F2gLdIOHqLkaXlbeJSba8aA4HSYVr1n95n+QMw\\\\\\\/4J1HEZSH5\\\\\\\/IhaQcXuj5Qqio5buXMKs+UTYExLsh\\\\\\\/EULgreVbBIQlrQd\\\\\\\/lQNcmEuu\\\\\\\/uZlp+xJIqEid3w1Wl52ZAX0T5br2r+prJJNNPljJPj0DWci7OHU4esiWmFmKgIekkX504ytCGe0Ptp7DJcqm7BF7AOUH8CCcVpAC1BgrxPNxxNwNEBwxMFQz4EMfXDsKE+5kmyK+uWYlPEQ+0r64vRdk95jDAJ0TD88EU+iE8p+rNtretKDwNtQof9ZvTk2PllabLcsVRGprF1YTQ1E55E7Bk8K2pyI3Ylbo4KNyqmLbSKfTnFPxTeHT95K0EOG\\\\\\\/2U0ZEcZzBSBz3mDJinFXIvPGH2s4t9sCt\\\\\\\/dcW3S4BizlonnYrhtsuYl\\\\\\\/K7r917M8Pmyguwe5P6xZPfqg4guxSWSVayzLaTv7gbyAIE6sLXTYXb3qbJqHAF91MB977MLYWsFyTxqTf34utNZrlHf+08oLo50Ajxg8wQETg6Xi7XZTP0h4swhzLo9pKHx4AbMYsiEkm2GVi+2U0Ua5YBUq1iFWSWg8eGKmIMwxGerK\\\\\\\/TxLJ8TQ2SfcF6NjbGAvWlqZiDBcotyVQ9BR7LZlZYWbqPGt3G4Z06UWYTfmbIwWQYvX757k+KASNcjXqoOlHsprI12F+h3e97C2jWnAsHxVcW0kHVY5N3flTgV7aQhInCdMOyN7vFeXXeDBjYDGwvJEmjOSzN76AXkYIKsjzL\\\\\\\/ceccKtkM3nmSt8omT0QfSInyZHNEEgWzIaVdyJrMP5AVFFQwF7UocVcG9t\\\\\\\/CDoQI33z+OFDOg1JlAQ6LETTbbTgY\\\\\\\/j40aIj9Li3GcQWby0bu74nzUjjjDkF76WHhj+snkugYFcJGypBwRr7TyMBgHrU5i+6I1hWJQgFOkysvCREdfn95xhJSdtHAq151ORPjxktD9YeoK93Pk9s34n2oFkN18\\\\\\\/R2tWhtv4R0CY5+VMppJe618dq3Bcuu\\\\\\\/Z0xBLYKgoEawvmjVT8bhwYdIAbAOyBTUWzichw04yjHzCikjOj02fs6K0viQQ4oVdanwIzchvPGiQ0OV4EkMEPTmCUQMISHPjqVezpAqhlsnkbdZiSqN94hqBFzexDkOlILI8F2vW3+h6Oscd7n4jvK4vRPvZ7bhs66l+NjQy2WhJSswJ7oOX2KNZJ3ykSwaw9VOpufdLVua9dHekla1dE5SiLT8NJ+x9bt0PHUl\\\\\\\/d8XvGcxRWmqZVdvKfG8FmVQ8gWWPwVBF8Ht+GiBAb3zx0L5sLJHYkWAHEgQpvh7P7V+MGzRRVtq46pW5VsuRxRRIke3fK3\\\\\\\/v\\\\\\\/hPwCuFdvJXfL87V8aEKWP1MOKq9tMVBaI5cbHd02TPHJC5KiD6WJAW7HZV+H9SR3GjA3TEJpVyTTSeCACsle4QJzSrCkYjcc1EurlGX1htgtVI\\\\\\\/\\\\\\\/VjSaHtaVSvucJlo8tjzPD1eKTD6jxlNzmJ7iOIQTq4Kpwqc7Z\\\\\\\/fQ9crEZfjPpxgkDK1Ygll5Ylz6xhQVTB8c8OWPsNfBevWddeoKNK3kWP+Gr2oS+6fGHeWKTbXZMcspeWMlvmXMeeWGiH0VdXSOejalL5cKI0rLZE4dDRWWvRP8K2UEDgMzL69RHMdbUJtwPQ2fLR3yO9qO5fyroJYBwVBEGQYAedCUOPa3MACHvdTdgj8TcmKFX\\\\\\\/9TeMFK0eM6UWu4L6NwP0zwVkNeOfhLsAZzjz+QfIrz4KUGNZvzEyQa70QYMrZHN05iYwHqFTMDEOg6kGP7kiaG+qGV+RpURNb+1Ifn74t9gy55hLB8mpU0SoZftpPuejHEjF1P+p\\\\\\\/dNx+4gZT3JySFUeBDZ1vt+sHYBaHRbDBt8gOLq26hDk4CPBaq0cZ5Q9jylatGlPMa7Cf\\\\\\\/0Sdocc1vhqAk6KDq9mWrzBYeVRS10NYiJAAixtZglyBZp+vbMf5lkxfn5hTwNvJQiPf+nqYVz1UltUFl+Sso9s2\\\\\\\/1QSNfVmCs31yBKqRkNRmDhhgdXxBbftTlErUPVnoNkKyjbEzygLk9N\\\\\\\/bLqUFpFQDBvr7q3Mmig9A0qICYJY0CcTlCMXqTZxTQSpUwc3URAwS4zggq6i2be9slJUbSHqIAAnneEMfnD7lpyCbNhVvBARZnsC+\\\\\\\/WPOsP0q8LEuzOKYsNRuGZ1gm\\\\\\\/K16yWT+jREq90yOOhEY4qX\\\\\\\/QXwnXffGA2JoxjUcNkArvAhjnts1IRVde5TPSIW8tWQH+ilQ1Wojh2J6B7SxJJ6nyDd1A9MmemRFLWr1UxgCy3nNl1WMh09b8sjCcE3yRf0HLWTKygDZ9LvGs4ZexaJxRIsoLcHzmRypyVnkCn\\\\\\\/lo9AOG2TdpkmLuHWrRWBXaYXpiRKtQC7F3f225ySCQqiHPyXz1s75mYD+poC\\\\\\\/zgjAOLV\\\\\\\/1NF6Bs8rccdb1NrzYXamH3mNOqtYJncpeEo9si26K4SyyP0OuV8w+yyyEhlAuIDG09mXk7wd958LHeHYa\\\\\\\/bxIt1hrx6TaBWGhSJs1y\\\\\\\/jwJ79tBD3CWU2d61RZAzxRG5GCtphQOLuzzVdOxA6L8Q4i6WXgZ0nUNF4heEC7HX2VW\\\\\\\/aGQpBGy68y6i1M6VvhigWREXRC4LUhLusYpTtZjDUXSrQfEX6+G20qEaaan0FE+EH8M4fpJVqsGCQkSAKI54XY1NUFt3Atbs8vEFlzoIDRO0ZzMlxEbm2PyEwsQGsXlkKqJPtQZj+KEH3aQWCuw5EZkp3MISi9WgkuKt9TaorwMjn5GupG7AaO0Pa04HoF7eBFjYDGkRLNThtYw5B0C5K\\\\\\\/vK\\\\\\\/sKtuOMBbE\\\\\\\/jF6ePNQXJ8o\\\\\\\/2HxZoUKkIWXjG7A2zlrRwOQwTsT4fkPswoZPP\\\\\\\/Kvry37ORBhYgHnJxw2v6tiJzq3EE21YH+xEeOLQEjNq9UBcU4zribNp+LDY43IGhgwc1T6LpJQeq6qbc52\\\\\\\/nhZTykBSuqU9AJ4i0GHFcXr6KnxeQia++f3hvYLcjoyGFHNfqH+FOGtoLBWXU7D4WDnxuOk6foTJIhViDPAkbAHQv1fWYmuWqKvMe5BOATOz4RNh8bsuNLMRh+oHsIjTgKc\\\\\\\/Trraejs9rY\\\\\\\/V4pEH1aMziX7blURzRHhmIFlKDnIjFWymPiSTzPT1UvIC0ARHr1FXn3q4hmvkFVCJ2DnqSrr\\\\\\\/0xAuehU8BeTzQkBPw1y5KMemhfJNCTH4\\\\\\\/Wsd16Q9VP9b01TnqadNQrE7arqSE+4E0bDlucccmG\\\\\\\/rDoQ08UMZbap2\\\\\\\/LYqBFFOXfTNyObSFIeO47dIEHT7aODkiKY\\\\\\\/SkmQTmB3swRVGsI8GY3OhY2kO2xnR+LyVPY8gaIFEcNi+SYPhCfR58uHdEwVN9e6By9XRRxO7TwihtPyGsWMXfiFfactQMk9d4T\\\\\\\/ewIW2tWutcqfP2fcys5sXGZeajdQSxiI47cliaux0CryxgB\\\\\\\/Vf1v81Tazd6XVVx0fC1ZxoWIVmQ0YRAreyp1p4l+RegGRtP32I5\\\\\\\/uyfmZHEGII3qGI9PUJT4F2ZjQKhv6JhGVhSwV2QTeMUh2L9Jp\\\\\\\/AUACb8JEj+5JT447dsJJ7PB1oUwgM2iogJ1WZcQDC\\\\\\\/yYNJqlHSxEBmem3pDp+5puj5HH0T9vbawFjLtoNaIC9JYM4unfskGnQY+zbBndZ0ZHkOroucjqHSDj7x145GR8tdsKTFW6T36Sx0+zTYSn2BDirSQAiiVbrnd\\\\\\\/\\\\\\\/3cO8e9w2wqJ\\\\\\\/0wSnl1UmS32O1xH9ocHfQhcnTFterNxHX1apllYLEKXpuFGnu4as1RiwkiMFB6GscTsEqFQFuH3le2LLjax0bWjeq+jXNZQKdZ\\\\\\\/Edc7j\\\\\\\/NDY02baioyi\\\\\\\/lZrzAxnlRRQ80skl4zR4pNPqmizAAmsYd4RTtIMcHp7eDpxRFKm7U+gM0sydlC+Ei9yI8jYQO9Rq6NJsujQz88XnD+fbJVG5cwYRFEaI+P58kRzTHk5+y2GQOttkCyUn+GbJkriqKud09YslyhgcABiQ+Adn91kf1NpBK5Bi6vdmbCXE3ieukoTKit1XTtWQf8ohGh0O46sG3Uk9XUGkH+jQoK8SBhKt4x9jikREwjxR4EQsgqE8hzM+gYZ9SKFdvT6ZAruM2V12iC13HRJ+M9JUL92tAjBFIJ+wv\\\\\\\/UicDyI+JFmTXtSY4mmAqoae4ShKalwozV3nPpdSi0kXH1cw4B2BGEzJeLJH4ZGizWlTV0gR1T+3a8zKN4ZAkoqoS9+IF4jIT+iKu+R4QGx59d03mORE52VjfMrSKFUtB7FJ6qkaXlXAdpXG70ij9DOqjez3r6RdFm\\\\\\\/n32\\\\\\\/FXWkr2K5T8Kja0cynCj7jBL4SfNJQzgtPpRcX2Z+GKEBVf+\\\\\\\/j4+hYhAj9ziUi9oUk7dzb0u6t1UCdjQGJ7yLZ3hTqrPhffDwzxj1jDcAid9VRz+b4JT2NciinLmYY+c5AiWC+E+BxkYGb3PG4poLf3mXgir515gFRZ2yfRj\\\\\\\/LrZl88P727qIHqee+4eMzBD0ZRbxDdX30xYIfUKNW97hwdi\\\\\\\/x89Ugry2JXb9tCfdsz2JRdZ3NQSm0E0vHO+4w+rcoX1x76KKFRCiYytIS9MeGNNiWVL2ll0E\\\\\\\/HKnRI0joYW157mTjkoIvCbOXhcjbzFcEJhDriXQm0\\\\\\\/J88q4EyDu6oOG9E1JCFpgwjPEHOGfYY9Bekhc9NSX21ieugL13H6ZySLjSXivxv\\\\\\\/myUXO69195sm9o4n3jM5PXlasrTbwhWGE1Ey3g50ZtnK8JkMO9l0zZtwhoz7bn+TeLxVSVFDf9vUyugtbDuosZvMX5tUzend25FFKgWZdfAm8K8b4RGHC4NRxJxuZ9nWoKVGcPw\\\\\\\/eohVBzj9Oy0YY4fTuXezYvWlYkQhkBOOKZv7+5nkV2LRMiKe805xzQARti1DsiuvW5EQc4JrLpGAgdnVXwm36xcmomIkWvUl8BEjPQKQZkoGTUH5iauL5NitHFP8IQe3080TCxb+NI71aQWWAvVxyBe\\\\\\\/dQLuYJhsrzIn+t0Oh0xnp5AO5YN9rqaeS2cqwnoVf5unIdqQ3zZfkgLwp6EWWZwhUa5WS6IKUMxDg8rVw9rgg2GPPTVJ8MhVjq7GRt+KJ3F8nSr2zIJmZgeptvrlEJSuse0rW8Ki1DfJE8DqUd6rSlSVdN01DDZDGELLft3DhpRDFXOwssOQne6x7Tp9ulJxcT7qA0HboNBx5ENx7g2maOjawXRv7SG2cI+EyIsD\\\\\\\/LDeODWj4uLl\\\\\\\/qgBEVuevMVhT1Vb5Ff8xnkemwTPPWtWogSF5UhBxVv4KRtHJTT5ndNTfj8U35XoEA2TAX4rF0JiidMSCEsH1ZEo3mwYoyF6Y19MepV0sX3fXu4beH\\\\\\\/GuI7idTtQD2Om7UvXJW4\\\\\\\/Sz1tnF3WYj50it4wfsnwddOo0xZtOnfQHQHBEjgd3DPqJnkjJTqo\\\\\\\/PPlYaec4iryPPZtEHTJn88tPr6GzTyHxeJ8adMP+wvhzVHABf1XUlSP93gik1T96SbTjgWXiKaFVsDcTyDdk5kefbz1yXlAvITMH3rDR5j6sdXaKXLdPafs00iNDqHOPtuDUuZBaYBafVvLVZ8A0xSB\\\\\\\/jKHKkhdt5Fgo3cTfDk0nYKt8hFD2w55tAhWnsyJxJ0ghQsMKY98gZjh3AJIimZLCb+03lKNfSzogrVw1IdMgxW9gYA+yqSDC2auGx1rpMn11R6A\\\\\\\/iBg95RaIDiWlw2tGRfdKusWGGJikNKdVItdZ7LOd60cX89D+ae\\\\\\\/zdiGL6oqmxjxbujq40VSZR1FrawpwXZf8Qnhjwc0VCsuS46g+zbbZpjihLWkE10QWv2O1bIYwK7T\\\\\\\/IwQYmZ5sl3ZZkxmOCAXBf3aXQ0x5PxNjj2hgWJPm9cTh\\\\\\\/0z1+Bsmf3b960eaO4W2aaNVYxcNR7eJUTfEN4QCWDw+gDH9JVIcWi7hgt0hpX5bNz2gZ1IcgKbxbRGQmMZOQv0ukVIkDiRfdG2eAXaZrOgRcrvZYk15elCeYDcitUqBJPdZ32cOwGkWuMpOMuvqGDbvyo9l5F0ciBnb6BHzmUuQvrLXL3S1sy7R\\\\\\\/HasKVwN88bCWQPbuivttAlQt21uFrBlsWxOFbJKeHaaTXJYNGbCmJ0xxODH3mqcVOzYSFC9WuurPUMQO4NGRourIi4++MHe79B4aAg0sSNwd2oyeVEJWlR4nkwGRyhRcehnOdryrOSTqibAY0xsVepbHuMheBz7ZiXtCoeB2Z7SU1DJn76clTWCiFMjkVEfjDe2mAxetSXmk8r5H8vbzqg54aEcnJ7TYJ18smamTUTtJ4Kk0AR9DWb4DFML5ZpUE9lcps5wqXtf8WXT5hgFBSsFj3oG1oxiKqH6+DeNlrtR0yDo\\\\\\\/zJpfpb9VIPmOQBhKOSaCQRYCDqrfHobswRy46\\\\\\\/ZYC6h3gk6pqS3uSpVRyPykP4MfBGJQMJmUDfJ\\\\\\\/kH3LI4N0SBMqTxvOKoZRDsSCa4xVOfocH4Cv2W4i+7yJBdog7N\\\\\\\/oJrzvVA\\\\\\\/mcSwNbRHiM0\\\\\\\/TV9LUYCgH10xOOMCGpsv\\\\\\\/1iVdp91GUwuxgiIG6jxOBswFEtK0YZVag1+BpR\\\\\\\/wwAqyHtthliZbs5vHmndj19t40NCoNVXaSuNiLOzDhylH6W3+DEFgQahpizNs7XLON6GDccw79OVAwN+32BlfFOSlMcjDybT83G\\\\\\\/VevuHqWvGQsbEWdAEy1RMpZm9dtcl\\\\\\\/N3e\\\\\\\/KCkv4M0j12jBu2QnNdykSiovXim0qnsKwZUvdueigxXOeZktjvJWsEekhYiKsTWH4YaevB0IMEcuawlnE\\\\\\\/tyNyTgvOjlr6p1VXD\\\\\\\/q3UnIGu5NVduWQ8A\\\\\\\/yOLrCgVfizwgqhQPDijCGDvSsrQteFgvaRQOevSaG\\\\\\\/6wEm6MBmucXBIvvMMOFRqcTjTZxgqOKXU9ncA0ESut1kZFhfvMIb60X5qaxDE+4Eu+N4TwsWKFU4xwCcS1xNZru\\\\\\\/oEGhZWYxUSLGev+VXi12pKtnakFzp40g+dFVlfI18Ns7tvknrndKa2HDW0KbgGXLQEx7g1kD9B4lqHTLkqMXyoShwu8+6j0XLknCtrm53qEqcEunayjbFV+oiX6rf0n8Q7lJhzajHVPD8GTt9S39xpRHq7TVFGGi7kZeCSA2Mmlww6EB065AC1Li0bAKcvMdgRjc0SjTYSeDRHPkCe3UsZ7K63nLQZDmpOO0SCxlYCFsSgsjraSUEBBRsnxD8e+BEZKfWAUHwd\\\\\\\/+hWfK80WsPCa6AdMc\\\\\\\/wgG3aO3WzTUgtfsRxxTruvgZ3MIO\\\\\\\/YoAOZHvBemuetpmhJGkjJgeGreFxb9cmIltJ7xmyrUqar1OnyEnGrBZzEkZHKQ2NsASwFaEa2JqgLKo722A7h2sm8LBO5JkqHMZaVE9rkDWt5coV38QeTPvPJnoK5et8akIKtHsUVZiAn6AwwijSZjWarKy6JQQ1SZCwHKriBdk0+1oTavgx0KoRGaEu0OZyCx9QOvrZMNECvW+WIvoVN0PZ\\\\\\\/pfhPJ8c74nK5yP6ODnmwQHpc2wsCWV09lRWaX7bLvJKaOGb1+3\\\\\\\/\\\\\\\/87eKMZgpD1tGxMIV1clT6ozMjiIUQxx8TKPgsdXCgWx8VLc+a3cOdMLGJoJmosS4IA45Xl86IT+5XP9Xs1hemlaUSW0qp2ynfH+0yDhdBpNQzMv2cQvrs4RAn7N5v1b7UxYLs3SmKVQUiMZwgVMnejlHZISvt76QWCGN6FWPNqtodcOYxuIvH5+zi2feVGR9x0GKYBX1YeGpH9ZLRh058DtRaNoMFllJni18x1+prLJWGxHPXPC8HbUG3DPXu6QeIz8H3ILOKzgCfwfrZeJtzuaBaDsojWR4hx51aKSmySWLVyWvW1LYebotiYSvNbF1czxf2pa57lY\\\\\\\/jkmZ+YfqeyqLHszaIx46fLaxwhUk4EvVWsrnqC2ti0PmzXQ1OxGvXRS+zbjVDzgA4AOXsVuRMlAvV3igXq7hqToM8RPRzCEfSqbbqtmcdfJjiDC+2QoGgrkd7vfS8DfYeozPvEzhrzcoEdUijH\\\\\\\/bpthgB4VQplwxZJ+za3iuHiq06Tq9x3nf5GCjngh7t\\\\\\\/I+UkLrmPyAWDn6+K8p8K8vHIJB2v7y5KRvm+tPvICDufgmNOQHg3vg2byZaNvtDgik5YlLBWCrt3ScSZThf9bSrCF1ltZ2nRmySDPlbPKi+UnMdEWe7pvfGV5z796njEeA1GcsHkbsDvjlmlAW3Ls7tNIsTvuudZllQPbFDHpnxWsu4Lyg6elL8CqyEhavFVIq8R10\\\\\\\/QpZSg9+YHhXNwUGS91cKaOdLwpVdSReTnNl9UqUMeG2NKhZR7SL4n6HaTSiKyEBBjw+Dyo4B3W0AaAIU3iE6iTU0XB939Jws5Tdx6k1ndGHqYk9wZ5sssBcZdogTt\\\\\\\/g9TEzZoTDjVknRfkHbMoDYr4LV6nlbutccp7HJKEIdHajhHLSAqgCIwGhJHjn8EDooAYdZuJqybwhe422SXUL8rkqWC7XshJDkNLYOhQ3zYyBXETG9r4vJizs5OPueTiKCSoLx2uFpMrkGltDvbi3lkXIbgcsBX9uAvndxxikoP\\\\\\\/rMCy3BSx8Yq7NQABElraGYFw8Pt00ccFAzdXZmLOL1g7qrMqjrxBXb2lxxQRbChnOOGaxMHb5flywkzXOoLxW+P9RuIKGxRZaLsgeV1tTQHtMXwFnET7slXpzuUGzkNFH2u1my6+Z63Hs2fpLMqyV7TPCV\\\\\\\/q37pbRWbQBSxizWckZPN2DOgSTdatW3di\\\\\\\/+JbWCDsSNTGMq5x1SenjYYz49Y3KBj35sr\\\\\\\/EZfFsec2byj0U4tK11SPfULGjaPHQBu5ktFBamK1JX03oO8Y74ZWMHPKcpQe9O\\\\\\\/FfgAttTndgeYQrknGJ46mi4z22KpEgNp4c4TrdV04EWy2\\\\\\\/VIkqgodfbrVZTcocQItp\\\\\\\/Vek84WE0eZ6TF001EQCpPDZ3HBnQOjuGCQynL1daxR0mb3wXXDMuy8uKmAxgD9EzoEe20cw0QmuUCCL9ArQ1DI8PLefPC001WqIyWQEMigSsjBfMnGVNq7uCSs\\\\\\\/9HJijzmx3sl9ZQY8ZSJBlX5FMNhQHeNxvh4pRHqZeaiLv54UQ51mahJUKZnb5MDxrkwIayzaWoCoxcEWZLK5\\\\\\\/J2LX1AfK4mNqxEzfFIpLqczztdB+Ohh2OUzDMjj39hrx9nlR1RdamO9pBCefq3ae4wtfoHxL\\\\\\\/oWwp\\\\\\\/MLdPOsBQV5GYCI28fW42y4fEplMPU7FIemXuH6V5WsV8K960iCEwqZQA+WVRiDHRSHA5fVn+Rcu3X31tsFEchJluTkZOoujteVyoNck6rpSwJi2ne5neL8EyTnx1kDvuQ45J51XM590uZY4z1XpUxzV3xrqvv7DTcegQo5++aNCCFpa2LnUQ9aGMsOCSDqAHob\\\\\\\/8PVYOdXn8k+k3+F5K\\\\\\\/3RJj+jlE3I\\\\\\\/v3xYXeWWD7KJbLj2VdYN4T3dau0u+6b21pA6GFz5U1iH7ZO7iyQ3++njyGPHNkQ1FK6vdMVBQ+sn+KVdrC4vlQ\\\\\\\/tqIyKYBEsh3JanSsmHv3rJg3+Ldqxky6T13S2qrASue+GQXsVF2n7MEZlAs4yDDpvvXakOx+\\\\\\\/TCMJDFs9rMYFBLmFqn8OwtRUZ2xaYWQpthuzhQqFGKVNnQTtPD5SC8YsBi6S+jVAVmovEgerzwhxg9+Ac2ikZx+qtM1Yq\\\\\\\/31SS\\\\\\\/BIE3cjsZcFyfWM4Utn6hjvAtMLbN9oaLYzT8IhyVM+SnZFyggC+P3IPaJPPAyWISUdyUtQ3ho+EjbesvDiOzM34HWTE6M9ci7N\\\\\\\/Vo1KZA5YyQnbzAGXlXBor6Sn82TNuA7zh6Ava6warSHv6A9kwaQB4UCRHzFN8845xsYfgyVPEQrmq3XEdrU4z3RmHEzR5k+8oInKmOr38WlUDf3UjsVgJoYR4tPRut3+PkQk6iEdcIBVXMctpZ\\\\\\\/c8baxrH33T5+9tr7LG\\\\\\\/Bg9acHWEN5Xr4S2Ztn6KySrfwrWZIP0rfsJRutAWEDDVlXrcRpzLuHQtYcTBwTTgSTvZyANj020M\\\\\\\/wZ1Qjadgnr+xpFdWsAY\\\\\\\/khfqN0Z9jezRjaVD0GM+u6cw\\\\\\\/\\\\\\\/tJEUNV+A4v9+CRPnQnDn0aFUXjdWn91da6hdJFzybGeOg2BNSkPl9FCYyh2xS2GuuMpRhT8tSAuZ9yf3xvbGsARpvjTNPllqTjeTK3AWjWcOq8SmcfsziIkRMLCleHKzo6Vx1f\\\\\\\/uHhYRVFl7sLiAFmod0r\\\\\\\/tkTujWibd7hni0iMrV9YwfWhw0qaj73W+togr6Bf2l6Oy03SpKcrWvX5x8XPVjLk0rrmcG+oCKq8g6D\\\\\\\/m25C101t1ZIeEOikVFD+SEIvwwZqU0K2jjanDyDQKkAybF0kEKXa2g217UPQqIvH+5DJ\\\\\\\/uhk8VOFGBcDs8FaEkN8IqgV819NTVx+cfXZ\\\\\\\/ntdUyrdOw89zyK4gePadeT1VlhvCq95eedK3pzB74WbRahDjGRq15hsdJz\\\\\\\/yogoMybZqGvI6GfFm+Ero3gJhFFIua50NQ0Wlv9YcIBKOFS87T+zdiK1t2aYqJUG6qnI86X7\\\\\\\/eIhkYVKiuESI5VB62rOyzdRe5+ywWiEGCO\\\\\\\/JQhcBrF6qG6+CPy\\\\\\\/eGKCGPpAPjq5xkoXG+AYH+ABtZrQ36pDTg6CfpIPxk5vZzC54qZ4ZmvIYqViN5HwTk46lOAxrk8cSWbA1VyZK9PULr5qsi3hNeghm8LKk5HekKZto+1vIJXl64L\\\\\\\/j74ADiBocVx2dOvn4zaEn8aCexwcgMgGHPvCaXOSLz76Bb+OR35KJYP2vBzTheSy5KqbHJTnfCyydVJNnnvsCZtTbutg2wNox5KtjlFfgTC4al18rKWMn38HXbfUc3knwnTRwuWs1y7IM8UUG\\\\\\\/GS5ybNz\\\\\\\/OTOt+R9u9o2VC7wTVpiYe75tCrNJHCjybOzxUq9PtaukYrfdIsBy8hCibkl8CPJj0BnvatO6iuR218o3YDluCBQxPgAQqoCytQBqaXbeC1liTCftaOXIBUY10P0pbO008VhMx2qj2EM4R\\\\\\\/\\\\\\\/epfvNIoF5VhHmxOKS39XO7kQyKrIUK\\\\\\\/VI9Tz5chjrleHKlAy1CBeRvAG7RNasnWv7Zrhgy+qLa1LXvjP\\\\\\\/HuSRSbuHaS9T282\\\\\\\/BsD\\\\\\\/fHAdVLFJFlIfeq9dFbxuRYUcge4FF+zwJ3SlvL+zTnY8PAFgO3QxpMQLnLJO65pM5c\\\\\\\/D2Zuk+ssbr+3v2ExzC6xlTEaRUIrZWatnCyfu\\\\\\\/uWU42yZmAE\\\\\\\/sDXHt7GIkbOcpjygiFu5JFaMSYSTMqiWFbYQZUMJ47o7N8YQeTRIg\\\\\\\/wKaM7uBI1CJL9HOb7eRkVU78ncvNBFpA4x+D85rnW7IW9kVBEnsXnPE3xqEYvKYAgsllRX7glCqcSAvQwsk2shq86LObL2i7N5lT7yDbMibATMrxQXeNu7tsKbPdbCQtfXEPLYuohYO7WgQOk9TpVzM3lKl9N5WglWhj\\\\\\\/yo+4vbBBiRTZk3i+sepKxaSrwLmlCuE+whUr6JpPMAFnncYgY9rTIgRpmukPdXb5wReNoRKX83PUXxaE3gQdzl9rE+PD2soIzReoHtYyw4GAPMQQRJZx4YowH6Vbmq5iSGpcWJN0VSvR0jxYa07JdY91XEmYuyU+LmvrhdNIvf3NKSH0swabCazpT+2zboKGSvgdIP7gbZ81PdqhpBwpB2AdEBDjnul5jE2cRCTf78CVIU4xI8bMOcY3kZO1xpFrVdmVTuzqXUh4MLIczVE2NkYRaHLg8YEz0fDnABScdMU5XEs3gueEAjtJx6PjGaiLxmO3HzFIOrCtJt9UFimVU84rCi7mf46i4JSl9bSR6038\\\\\\\/36pPVRkqsU5r648L1vrNUV4byE5FZOpI4WOqCzyzAQRsHzZQF0C0hCy1TnfF57rgZrYvICLFuCZ6yizvRLXsMltyITjBG7fsmDFNeNhv8oEsSOVveXln4Cap86Zo1gALDzPF5VBeJNAtWn2SqcM8mwi5JertlOjJk15hakKEPlMIBBMgvy0km2RD1V\\\\\\\/mLd2TzEwlNkYBpE8ZBCCyUNLVVvi8B\\\\\\\/aYB3zOA56zDoVtTMqsoE993e6uakjrBuSKiP5Tv+l7pHt6EV+EU0jeNt23G9QRAkCmJ1az0YWzr40PFgFcvRgynaQky9AazIGHhiplDTswguJEDP7wsEVktjYp43vHcFZF5HFh1jWlB7fwJHMmj\\\\\\\/9zW86UurvQbA0UOF3uieGZJbtsiZPzzxoHR9hzWeeVP1NQhRgfQV0v4sSJK76pUZGiNULF3\\\\\\\/9QflN+CAk5CtwYCagOfL9\\\\\\\/w4NOXJgbp9n9l32DfUkfi2ybpEsHjYZjgi4qR9dfoj\\\\\\\/G46cZJloSMcodQiDBTxQFeUrced7HLDclTb9SiehRVMO8UvKHnHd2MsL+3HPQuoqmjNOjPixKdtbgcLrkDHt6uVnS+fK7QmvU2Kn0BQvl4W9dRZ6TbKFInP0t30FaAC2reDFyxtxDXTeECgp2RBToQNc8RguYOIlsV22EqqQGJRs7xoklDrBRClppG7NEE9\\\\\\\/oc4LpsyFDPyxKNP1+IU753KU8WmelKDwZRO\\\\\\\/2jeysC0ZWQrPQ5xqCRRh\\\\\\\/PQDHq\\\\\\\/ukaZdxM+O+AeRbkExC6kNW70mApILKVaBL9SdlveSzAtUqtiJfUbRf8KSxpuDubOVfS7QqGzQtIzpF1KGMRwNsc6jQ2ekdroCsZUlnjsg9hoQ+nrvFJATqw+\\\\\\\/ZhA2Dy9Jk6TzNvn82g7C7fkBzNTAsWzvP6EQWLSdtz\\\\\\\/hPbhzgsRjiyhYSWHNpjdXXieWQR2bdEYvUTSG0C4eVhLC5jCX8k3CNQReUoYPJ20gm\\\\\\\/5EPqs72g6mnDS0g4muWuBqqyc7naL6WneF\\\\\\\/CQ\\\\\\\/I13sTcdvYoDI0vaX4VAeFjaoXkesd8X5XH41Grpdmg9T6iUdXX4jFgHa6krYhCHsCASV0rvlpcM5WOZ6gpOWeTxvPXXnuQzTgQo\\\\\\\/B33tv5l+MLsmuQf\\\\\\\/QO4QUie\\\\\\\/Jiet2ULp12tKfhO2WbIAcRDHnL\\\\\\\/LagXoKaxuDJ2QgzttRmfeMf2Ak20z4n9LS1rw4IkHJUPDgYzfK\\\\\\\/PRvivOS7dB7pyqcA\\\\\\\/Yh+iqIZ233URZAbMDbn5OHACs7VBbwzo0Js24D+OtHqgpCQ1z55rl\\\\\\\/DTAeNLpKXMH9Im22RKOo4af9yQ1kI8F\\\\\\\/O8Ep8ULgabNA6h3+6u4J9hfGkn8gFPbHLDQoYfFlbH3g1O1MjZQFrsBwr9TwIVsjiBe14TXyfKkJ02uLZnoK7rTO8TQW4NPQLP6F3F10YimilsT8uEjJopB+Mlnzb7e6btX8IH1XaOTx9tzi0IgLyxcEvQlPsWE98JzdZAvxNYXNeMO6E7B\\\\\\\/cwkJWRGQVqRHire8LiKSFuzMiOADmTVLjY\\\\\\\/9pPEyq6lgxo4TSRPll6a4rjgQ5skL98nOTfQDdmuOYHTkJumBwXcv2rYuXVfCYzq0t0\\\\\\\/Ih4hDuTsDwEMuL8\\\\\\\/ZKeKoOnh5QPP7NPzDBAk1CdtCyf1FP1niCeZJYeYvtLJTz\\\\\\\/vXZEgjS\\\\\\\/17zVvRuL\\\\\\\/TUG1yMcWCir0ErYZhNyCifDI2tFtiHrQtYkpq57QmUb5EOFaKPtZV7gH9XjuNCBw\\\\\\\/5SDgwpG7WQDHfk9uFG2ws1Rm9RC9Rur9hCbIda+98+CznFG9zL233+pakYD0TQQxsCFPijM2x1ioNu5GuoBjD1vIy8+jPZqZkQRhs2NWqRPlXe\\\\\\\/tXX\\\\\\\/64xhinOBzbh58t5t9kbT8fyAZCnGtd3ZUWmrCDMSdwNI9cA+KpvDosqaZv\\\\\\\/cQlSHIiWExhFqGMTDGCCc8hs3Ypq0Az2DTc8h35CtWSQDOn67NueHfmKSuaOx5ofEBv3jBpeZIKalV1z\\\\\\\/7ZQTgPjK\\\\\\\/z4s3D7pN5M\\\\\\\/UdKWmOYsM787IaNFajrtdsSYocu8x9CfKQKJa8\\\\\\\/h2FYWf8K0sNcl+HSYKXeuYq93NacZrqSek4YGa3MrbTJbfw83agrsV7lfrFUqkTtw4Uca2\\\\\\\/pWiUZlmNvOiuDFQJa\\\\\\\/PCk8XWVjlsmHvckiJLTafCFlEB1KSc1Ife\\\\\\\/05n001g0EX9Z0iKQPxoGlDRFKk4p9Ouxww99z+MoaIuDhGxchDDk7zPTF6sBWkeV3BgdELzcU3Pt+BDpd4hWwBqJKUu4esDLS2rQHeCxYcM9dbbniJq8dSQvcEZVyzC5Z+0EXTjvnsPChtfVglRZfyjEyqAko5eYRfsPFuy9Y1BPfSgSr+N3bxLcczeTNhiSUxdhrgShdjCH1fQXxIITNbCH54DiUFVlgVE1UWn9KwQg6ZJ6lCa7Zk7g3woZ3XCwMKn41mWroRqMOVTm92mo6qCPpYn1dlQM9bsmwBopTpzDiuYeRe7w2qvfShXY94e85r3p5WrU\\\\\\\/U\\\\\\\/nMg9wqYlXITSbHB66dcsP3soMNZTQV7vx7ZLV0q2VgktrbU4GtDFdo8eejnhcrgxdkzZhij2pCTJtOZXiJJ4nxEjK+EuzEptD+aTXjARtYDw\\\\\\\/fj5hJPljonqB8YEuZyfQGaYSQqSg67ohFvHIJHslpwUvRgulwBoThDesg+9xplQxL6SVobRbGp2TAHZKSX284P70iI06UQtuQR1qA2mLq99jgs\\\\\\\/qi0WxTMzDWunP\\\\\\\/LayVza6cOBWOM\\\\\\\/ohkfxpYoL4chwFUKnjVxPMaWUTEGKZ9S0nMKByYhjZHvBC+ehNufo7eTEsUX+QrOD+9fB+iicpKcIoNA7fbnLrSJFjNbiHKse9f62LoDcARVAFZJdMIdUQ2tj\\\\\\\/GM+q5nYHGWwq282SZIpye4xDGcv2mAyokrbIaDzjBbAMi7N8vejE+04L3CvEGDtCTcMEF5UwN+LQ8FHejo92\\\\\\\/H1mGCuul3r\\\\\\\/pe50pajwbx3wOTmpaQRRN7iLliiKGg7zgsQgVrxRw37hsj950t9fJXIk9CNWQumcWPNOtUtyF+WQMykiziirAKU5JsEwzx9xC82zV9uoQgA8nI+RI84y6mRYp6NOGj1L\\\\\\\/xZ6r0lPfuVL7SCchLsOXLnW61ixIvn3RA4rOs12u4Z2rjrIj6ZYTJr4vhPMEelBqAScJE+x3dqHbkyT95osVW6kBi1P7D\\\\\\\/D1dTzDYgcaBelpO+zKB\\\\\\\/NH9EECk4ngz38nPLPAK2Mbgg\\\\\\\/OMQ7O30f+6UaMJ2jhxuEHm5GCOIah+buQwE9YVNSxWvzt4pfUD\\\\\\\/RxSQS8\\\\\\\/rOvDX1JxPL4p1jh7F6tq\\\\\\\/Xh+uryzfgs9dfkfzNX9ZZkUEyrj\\\\\\\/4Tj4Lgii8w8oB1RPUyWHmH3q8EgHLG6WjpPLqyT41mhf0+VHxvxcxxGO50+NKWTz8KDeryk1cON0a+KNxyYCH8k5biLzF4AnKimVoPQj7MfdlRFCEkw1bl5SNGbZd\\\\\\\/cw5rRTZ0SOaz3NzQDO+agCubC2tAD61pIFv+S2dD8KyfR692s+gnfzBvWsgyPqCBEu2cebq5ZLGpiJRZkb1gfAzoja5MuYQY6uYVGhokS0KV3d9Z+tNjjmcf1yTtc4YH5zfTdwUXSNUegghS0EnFLIrVN5J9UFwg+FqZt5YImZBcjDNNoTZfz7qIejzpaLOj1dX7Wm3xfYOMP3mz4J1qMZCMlLfWAwGNFONJQ59ibFZEx\\\\\\\/LmSBGg==\\\u0022,\\\u0022iv\\\u0022:\\\u00225MG\\\\\\\/WW2udwGnyiFCVbeXAw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e9174eabedabe936aeef8e5aefaf3fa397aba993e74bca60193323e4c27631cd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ma16i0M1W3vRxDl\\\\\\\/hccd06U0kpwMF0HbuPCxPmWakOUxYNmQ2Pc1rASty8xTgnxjzVRJMbFUmIrR4skRVoy+jWWXShQSlurB1LAlgxdojmIO9l1K898+z7XYxTluCXk4NllLpDOxGx58BsjelfQHhjwp48IZ\\\\\\\/OoYD1nGT9V2gfG6ZyE04wblukc2vTLc7RKhOJBHth3pQ79RJ0kyohwOt3LeUO53w5+vB+tfHJ26edl+aJieqxMJqAGPyk4guS93RdN8gjKtCSuFFO7pEv6FyV7kknC+4Xo1XsE2P0V6+8OajwawkVrQ0Va+aKjJbHpmcu9XP16TkDAcpDCF4LpZa9OoLjoaOCPtwmWKY1hB7kIb80IIzDdfmgJrtAST0tXXjWUpGrLbZ0QLhwnk+WXVyRs44u7sjG0iYm3LH4tRH\\\\\\\/q3ED0g8XBJqR3IFzGZj\\\\\\\/XKic6WNfZVAeVmjzhdBKC1oTvae3icxfGgZFdymlWw9Glvp3bXYtjuGiw4eDo2a9Fqyluqzd09zaabkVQD1RoTT4Mq5xPajG6R7nbzrNqfj5\\\\\\\/ZS6glH2+Lr2dqQtze3imL1eQGtUkEVQgajiK5FOro2xCc3aVuXAWmFGev5L7g9J0\\\\\\\/XEATEDclYkY4C\\\\\\\/hkCPdGH4wR2tpQoVlRd+HpJR114xl7rdh6lgMAr8AT9Nlq59inQ8FPwnar8NoDMVwdQED+LVSEcJ6Wks77osqeb99UPe1T+\\\\\\\/bmXFHl2SaveyEBtQCsfNaXGD8j0ulu1So6QBXmmrju3OdGWIC+wAz2Th05scZxD6lqLW\\\\\\\/HbrjnzT7XoKRVX+ULg\\\\\\\/\\\\\\\/yBb6cIrMrBmpr20lsXayAHg4VxXE7pGxVa77jfjwSdGhF5newkgQluMHdNy9sS1ozLlTqWFgM1qiGTCG1HPhHCBCYg7KSycp17VrxwtOXEApCeihJK7ntYcZeWOe1HfpQWhLNqEvjwqJZdqAniW2AFA4o+z6kFX1GplqloAv2BrD\\\\\\\/sweC4bKKVcoE7iMItg\\\\\\\/dXwKYE4ofNVEcodJshUIfjGfjZqcL4m+BDR7djLbOc5yBLHu4w1LAE2UzYcNCTKJYoAXXaTgSGatDwUeUxdqu2LFFGmICnLOZHj8Uvi6j\\\\\\\/VkBX1JLVoFHFNNxDEn0uq2fgNRcY3ixg7VIFlKOB5zYIjiq1zJ6O33FzCCBW\\\\\\\/q4ucJNizrgieNkZotFlcCQJCNMcQ+0CP7LnmqMFUUdTGAWLlYv0XK2GzH367n8\\\\\\\/+ofR+YBge++GX1MktKaw6yf8FZwdNZZUky15+fboYW5q+q9UqAVl3IPSP1M6k4lDLFHVTQJ5yMiB01iAnFXYmnUOS7MoezmopNxFs3l32ncY7LF6t47cGHFLOjCwHtvBiz82cipe4dwRRANDHDnvLlQfwtvJMN+flpf3ZnHa9mrw3tt5qhUN5GmeZMnq1OCvybfpJRem6kU6fl\\\\\\\/6aN7TxV7o21C90DjcNkD6YARnSoexlVmmVsr71wMqFqhpjglMLOussSFt+GSstxdI8BChwUxsGCOE1zP30s5E3ev0KBji\\\\\\\/gpNXGIT+teoV0ONijBDT9wmahUxvOup8ou59bkWRy7L602hWbbzbrBdx7J6w99kcH62bk+gIC\\\\\\\/AQEm5u\\\\\\\/ApcKpaJi5VLd1FzODKmSaYTKL974vqdz9lBNYQJbohfOBGU06LQGxEBO0BJtugd8FdXqPvuIEm6BaNMM\\\\\\\/mR9v91mO6oF\\\\\\\/A95E5VM64MUZr\\\\\\\/frDDX5Wfh5jV1pDVP\\\\\\\/cEO+kxZHMDwwpkf\\\\\\\/92AWtltiSY+0WoJFMuSzv5n7U7KPIGEIw4YpT01H1RDOnNHcY9DZsxVvkxEECP6JSI5Rh57TmPYdkGTP3TJK9fRJZZ9KiA2EqbIgnKeHiuSeHoA6V9dzK37MWc\\\\\\\/Kar9Yh6vwyDOr8IF7jZ9h\\\\\\\/JjAeXLQW8XPaUTD+zBMKNKjgStb9KAbq+oSUc6JjYzFHgg1Mlq5TuVtFv1n03tMbGj2aWVk9b0G4Mx82b8qBAaE6lXY1Wzr8w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/c6PRRQknzqJS5AVls4qsw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225073a786b6abf88c1c007c4f1ba79db7224a0f0bdbf928cd90c1047200571f72\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460427 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1979" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Rv-LSq89pGxoi7La8OMgxscsZqJEBQoyjbF_J8Pu5ws" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1979" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-19MbrI4jiaQvGUDKwT6zYl4osWhV6-MMe8rSJbbQRLQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1969" title="Whole-Genome Sequencing Reveals Genetic Variation in the Asian House Rat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1991" title="Identification of Novel Regulators of the JAK/STAT Signaling Pathway that Control Border Cell Migration in the Drosophila Ovary" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460427" id="node460427" data-pisa="ggg;6/7/1979" data-pisa-master="ggg;g3.116.028753" data-apath="/ggg/6/7/1979.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identifying Genetic Signatures of Natural Selection Using Pooled Population Sequencing in <em>Picea abies</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jun Chen</span>, <span class="highwire-citation-author" data-delta="1">Thomas Källman</span>, <span class="highwire-citation-author" data-delta="2">Xiao-Fei Ma</span>, <span class="highwire-citation-author" data-delta="3">Giusi Zaina</span>, <span class="highwire-citation-author" data-delta="4">Michele Morgante</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1699-9042" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martin Lascoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1979-1989;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028753 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460427" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jun Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jun%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJun%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Källman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BK%C3%A4llman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4llman%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BK%25C3%25A4llman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiao-Fei Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiao-Fei%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiao-Fei%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Giusi Zaina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agricultural, Food, Environmental and Animal Sciences, University of Udine, 33100 Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Giusi%2BZaina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zaina%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGiusi%2BZaina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michele Morgante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agricultural, Food, Environmental and Animal Sciences, University of Udine, 33100 Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michele%2BMorgante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgante%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichele%2BMorgante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Martin Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1699-9042" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin Lascoux</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/7/1979" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460427" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460427/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1979.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460427" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460427/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1979.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460427" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460427/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/7/1979.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460427" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460427/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/7/1979.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460427" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460427/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">The joint inference of selection and past demography remain a costly and demanding task. We used next generation sequencing of two pools of 48 Norway spruce mother trees, one corresponding to the Fennoscandian domain, and the other to the Alpine domain, to assess nucleotide polymorphism at 88 nuclear genes. These genes are candidate genes for phenological traits, and most belong to the photoperiod pathway. Estimates of population genetic summary statistics from the pooled data are similar to previous estimates, suggesting that pooled sequencing is reliable. The nonsynonymous SNPs tended to have both lower frequency differences and lower <em>F<sub>ST</sub></em> values between the two domains than silent ones. These results suggest the presence of purifying selection. The divergence between the two domains based on synonymous changes was around 5 million yr, a time similar to a recent phylogenetic estimate of 6 million yr, but much larger than earlier estimates based on isozymes. Two approaches, one of them novel and that considers both <em>F<sub>ST</sub></em> and difference in allele frequencies between the two domains, were used to identify SNPs potentially under diversifying selection. SNPs from around 20 genes were detected, including genes previously identified as main target for selection, such as <em>PaPRR3</em> and <em>PaGI</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/pooled-sequencing" class="hw-term hw-article-keyword hw-article-keyword-pooled-sequencing" rel="nofollow">pooled sequencing</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/fst" class="hw-term hw-article-keyword hw-article-keyword-fst" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">F<sub>ST</sub></em></a></li><li class="kwd"><a href="/keyword/allele-frequencies" class="hw-term hw-article-keyword hw-article-keyword-allele-frequencies" rel="nofollow">allele frequencies</a></li><li class="kwd"><a href="/keyword/local-adaptation" class="hw-term hw-article-keyword hw-article-keyword-local-adaptation" rel="nofollow">local adaptation</a></li></ul><p id="p-5">Identifying loci under selection and inferring past species history remain two of the main challenges in evolutionary biology. To a large extent, this is due to the difficulty of teasing apart the effects of selection and demography at the genetic level, and to the dearth of methods to estimate the effect of these two processes simultaneously (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2012</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tiffin and Ross-Ibarra 2014</a>). This is especially true for methods based on the detection of outliers from a null demographic model since an incorrect demographic model can lead to a large number of false positives or false negatives (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huber <em>et al.</em> 2014</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Tiffin and Ross-Ibarra 2014</a>). The difficulty is further compounded by the fact that selection can act at different spatial scales, and detection of local selection can simply be missed if one uses a sample of individuals drawn indiscriminately from a large population. Conversely, selection can leave a signature in some genes at the species range level, but no detectable signature at a more local scale, and vice versa. This is what is suggested by our recent studies in Norway spruce (<em>Picea abies</em>) and Siberian spruce (<em>P. obovata</em>), two Eurasian conifer species with a joint distribution extending from Norway eastwards to the Sea of Okhotsk. We have identified loci putatively under selection in two types of studies. First, using populations located along latitudinal clines, we combined tests of association to environmental variables, detection of <em>F<sub>ST</sub></em> outliers and gene expression studies to detect evidence of local adaptation in genes related to variation in phenology, a trait known to exhibit strong latitudinal variation (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Ekberg <em>et al.</em> 1979</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Savolainen <em>et al.</em> 2007</a>). In both species, two photoperiod pathway related genes, <em>PaFTL2</em> (<em>P. abies</em> Flowering Locus T like 2) and <em>PaGI</em> (<em>P. abies</em> Gigantea), appear to have been under local selection (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2012a</a>, <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">2014</a>). Second, using individuals sampled across the range of Norway spruce, we looked for departure from neutrality under different demographic models in 19 genes from the photoperiodic pathway. Only the circadian clock gene <em>PaPRR3</em> (<em>P. abies</em> Pseudo Response Regulator 3) deviated consistently from neutrality for all tested demographic scenarios, and the signature of selection observed at <em>PaPRR3</em>, a positive Tajima’s <em>D</em> value, indicative of an excess of common variants, was suggestive of local selection, at least at broad geographic scale (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Källman <em>et al.</em> 2014</a>). However, no evidence of selection at <em>PaPRR3</em> was detected in the aforementioned analyses of clines, which considered a more local or finer spatial scale.</p><p id="p-6">In comparison to these previous studies, which either sampled rather densely along a cline, or used a scattered sampling across the whole natural range, the present study is based on an intermediate sampling strategy. The current study also differs in its resequencing approach. Here, we use next generation sequencing of two pools of individuals to assess polymorphism at 88 nuclear genes. Pooled sequencing reduces costs, and, if analyzed carefully, can provide reliable estimates of allele frequencies (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boitard <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Futschik and Schlotterer 2010</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2014</a>). These polymorphism data are used to (i) infer the divergence time and migration rate between the two domains, using silent sites; and (ii) test for the presence of both purifying and adaptive selection in those genes. The two pools of individuals were sampled in the two main phylogeographic domains of Norway spruce, the Alpine domain, and the Fennoscandian domain (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). These two domains are today geographically separated and genetically differentiated (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heuertz <em>et al.</em> 2006</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lagercrantz and Ryman 1990</a>), especially for mitochondrial DNA markers (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lockwood <em>et al.</em> 2013</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tollefsrud <em>et al.</em> 2009</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tsuda <em>et al.</em> 2016</a>). The strong divergence observed at mtDNA markers, and the polyphyly <em>of P. abies</em> for mtDNA even led <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lockwood <em>et al.</em> (2013)</a> to suggest that the two domains may deserve to be considered as two different species. However, divergence between the two domains is not exceptionally high for 22 allozymes (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lagercrantz and Ryman 1990</a>), and 16 nuclear genes (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Heuertz <em>et al.</em> 2006</a>), and certainly lower than the divergence from Siberian spruce (<em>P. obovata</em>) at both types of markers (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Tsuda <em>et al.</em> 2016</a>). With 88 nuclear loci the present dataset provides an opportunity to obtain a better estimate of divergence time and migration between the two domains compared to earlier studies. We also show that both purifying and directional selection have shaped polymorphism at some of the genes related to the photoperiodic pathway.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Map of population samples included in this study. The areas shaded in green are the distribution of Norway spruce (www.euforgen.org). Squares and dots represent the populations in FEN and ALP domains, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1632398004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Map of population samples included in this study. The areas shaded in green are the distribution of Norway spruce (www.euforgen.org). Squares and dots represent the populations in FEN and ALP domains, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1979/F1.medium.gif" width="440" height="419"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/7/1979/F1.medium.gif" width="440" height="419"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468551" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-7" class="first-child">Map of population samples included in this study. The areas shaded in green are the distribution of Norway spruce (<a href="http://www.euforgen.org">www.euforgen.org</a>). Squares and dots represent the populations in FEN and ALP domains, respectively.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sampling and sequencing</h3><p id="p-8">We germinated seeds from 48 maternal trees sampled in four Fennoscandian populations (coming from latitudes ranging from 61°N to 67°N), and from 48 maternal trees sampled in three Alpine populations (coming from latitudes ranging from 46°N to 47°N) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Diploid DNA was extracted from needles individually, using a DNeasy Plant Mini Kit (Qiagen, Germantown, MD). We PCR-amplified 88 gene fragments using the proofreading enzyme Phusion (Finnzymes, Espoo, Finland). We selected genes in two ways: (i) 30 of them come from our resequencing efforts; they were chosen since they are homologous to <em>Arabidopsis</em> or <em>Populus</em> genes that belong to the photoperiodic pathway, and, in particular, to the circadian clock; (ii) 58 of them originate from a previous unpublished microarray expression study. They had a rapid response to day length changes, and were highly divergent in gene expression during the bud set period (T. Källman <em>et al.</em>, unpublished data; Paper I in <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Källman 2009</a>). Gene sequences and their best BLASTN hits in the Norway spruce genome have been deposited in Dryad (see <em>Data availability</em>). We blasted the reference sequences against the <em>P. abies</em> and <em>P. glauca</em> genomes and found them to be unique. We controlled for paralogs in two ways. First, we checked the gel bands in the lab. We also Sanger-sequenced genes using the megagametophytes of two individuals from the same origin as the pooled samples, and this did not suggest the presence of paralogs. Second, a BLAST search of reference sequence against <em>P. abies</em> and <em>P. glauca</em> genomes produced significant similarity differences between best hit and second hit. The bias introduced by possible paralogs depends on the percentage of paralogs distributed in individuals of both pools, and the percentage of paralog reads mapped/unmapped to the reference. This remains a difficult problem in nonmodel organisms whose genome is incomplete since exhaustive search is impossible for homolog/paralog/gene duplication.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468553/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468553/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sampling information</span><div class="sb-div caption-clear"></div></div></div><p id="p-9">Amplification products of all genes were diluted to a concentration of 10 ng/μl and mixed in equal proportions. Individuals from the different populations were pooled into two sequencing libraries corresponding to the Fennoscandian and Alpine domains, respectively. Mixed DNA samples were sheared into fragments of size between 150–200 bp using Bioruptor. Paired-end libraries were prepared and sequenced using next generation sequencing (NGS) technology on Illumina HiSeq2000 platform by Istituto di Genomica Applicata (IGA), Udine (Italy), with the read length equal to 109 bp. In order to identify errors arising from sequencing, two sequencing replicates were run for both pooled libraries. The Sanger-sequenced genes using the megagametophytes of two individuals from the same origin as the pooled samples were also used as references for short-read alignment.</p></div><div id="sec-3" class="subsection"><h3>Read mapping and SNP calling</h3><p id="p-10">We trimmed sequencing reads using the program Trimmomatic v. 0.32 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bolger <em>et al.</em> 2014</a>), retaining only high-quality sequence pairs where each pair was at least 34 bp long after trimming. Read alignment was performed using a two-step SNP-tolerant mapping protocol. First, the trimmed reads were aligned to reference Sanger sequences using GSNAP (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wu and Nacu 2010</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wu and Watanabe 2005</a>) with default settings for genomic mapping. Aligned reads were filtered with a mapping quality ≥ 20. We only kept those properly paired reads that were mapped to a unique position in the same gene (<em>i.e.</em>, concordant unique read pairs). Indel-realignment was performed using the Genome Analysis Toolkit (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">DePristo <em>et al.</em> 2011</a>). We identified single nucleotide polymorphisms (SNPs) with Sanger-sequenced references in all sequencing libraries using samtools’ mpileup (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2009</a>). Missing sites in reference sequences were corrected by BLAST (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zhang <em>et al.</em> 2000</a>) against the Spruce Genome Project (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nystedt <em>et al.</em> 2013</a>), as well as the aligned sequence in both pools before the next step. In the second step, we used identified variants to perform SNP-tolerant read alignment using GSNAP. In order to have a uniform number of reads per gene, reads of each library were subjected to two steps of downsampling before and after aligning to references. We randomly picked reads without replacement to generate a down-sampled library of size equal to the smallest library, which contains 1,807,545 read pairs. We then repeated the quality filtering and indel-realignment as in the first step. Before SNP calling, we performed a further downsampling of aligned reads in each gene to reduce possible bias in frequency estimation caused by uneven distribution of read depth across gene body, as well as different amplification cycles among genes. We randomly picked 100 reads on both strands without replacement that covered the first base at 5′ end of the gene, and recalculated read depth in each position. We then kept downsampling at the next base toward the 3′ end until the number of reads at all positions reached 100 on both strands. Thus, in total, we expected the mean depth at each position to be around 200. At positions with a depth lower than 200, all reads were retained. Finally, SNP identification and allele frequency estimation were performed in all sequencing libraries using CRISP, a multi-sample variant caller for high-throughput pooled sequence data (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bansal 2010</a>). To minimize the number of possible sequencing errors, the inference of an allele and its frequency was considered to be reliable only if: (i) the allele was supported by both sequencing replicates, and (ii) minor alleles could be identified in at least two reads in each replicate, or five reads in total across the two pools. Allele frequencies were polarized by comparing to the <em>P. glauca</em> homolog (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Birol <em>et al.</em> 2013</a>). Since we expected the minimum allele frequency to be at least one in 96, we considered rare alleles of frequency &lt; 0.01 (which is also the 5% tail of frequency distribution of all alleles) to be biased estimates of unbalanced mixture of amplification products or possible mismatches introduced during the amplification and/or sequencing processes. However, we did not simply exclude them from our study since those estimates still represented sampling probability of rare alleles. Instead, we performed our analyses on the total dataset, as well as on subsets of SNPs with different frequency cutoffs. Two frequency cutoffs were used: ≥ 0.01 and ≥ 0.1.</p></div><div id="sec-4" class="subsection"><h3>Nucleotide diversity and population divergence</h3><p id="p-11">We estimated pairwise nucleotide diversity (π) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nei and Li 1979</a>), Tajima’s <em>D</em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tajima 1989</a>), and Wright’s fixation index <em>F</em><sub>ST</sub> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wright 1931</a>) values for each locus based on estimates of allele frequencies using custom R scripts. We chose Hudson’s estimator of <em>F</em><sub>ST</sub> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hudson <em>et al.</em> 1992</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Keinan <em>et al.</em> 2007</a>) for each SNP, and an overall <em>F</em><sub>ST</sub> across all SNPs as an estimate of population divergence using the formulae derived by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bhatia <em>et al.</em> (2013)</a>:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1979/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1979/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>n</em><sub>1</sub>, <em>n</em><sub>2</sub> and <em>p</em><sub>1</sub>, <em>p</em><sub>2</sub> are the sample size and allele frequencies in the two samples, respectively. Hudson’s estimator is a better choice for pairs of populations since it has been shown to be more robust to sample size difference than other estimators, and <em>F</em><sub>ST</sub> is not systematically overestimated. To combine information across SNPs, two methods were used: we took the average of individual SNP <em>F</em><sub>ST</sub> values, or we calculated the ratio of the expectations of the denominator and numerator across all SNPs. The first estimator is more sensitive to the presence of SNPs with low minimum allele frequency (MAF), and the second is dominated by SNPs that are on average more polymorphic (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jackson <em>et al.</em> 2014</a>).</p></div><div id="sec-5" class="subsection"><h3>Inference of population demographic history</h3><p id="p-12">To infer the population divergence history we used the program fastsimcoal2, (ver. 2.5.2.8), which implements a maximum likelihood method based on composite likelihoods of the two-dimensional joint site frequency spectra (SFS) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Excoffier <em>et al.</em> 2013</a>). We considered three demographic models. All three models start with an ancestral population (<em>N</em><sub>0</sub>) that split into two descendant populations with effective population size <em>N</em><sub>FEN</sub> and <em>N</em><sub>ALP</sub>, respectively, at time <em>T</em> before present. Asymmetrical migration is allowed between the two descendant populations after divergence with a number of migrants equal to <em>M</em> = 2<em>Nm</em>, where <em>N</em> is the effective population size of one of the two descendant populations, and <em>m</em> is the migration rate that can differ for each descendant population. Three different demographic models were applied to the ancestral population: (i) constant population size; (ii) population expansion with a growth rate α; and (iii) bottleneck (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In the latter, to reduce the complexity, we assumed that a bottleneck size of 0.001N<sub>0</sub> occurred <em>T</em><sub>BOT</sub> generations ago, and lasted for 100 generations. We used all silent SNPs to calculate the site frequency spectrum, and projected it on a 10 × 10 grid. Maximum likelihood estimates of the model parameters were obtained using sequential Markov coalescent simulations, and an extension of the EM algorithm where each parameter of the model is maximized in turn, keeping the other parameters at their last estimated value (see <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Excoffier <em>et al.</em> 2013</a> for details). To estimate the confidence intervals around parameter estimates, we generated 1000 bootstrap datasets by binomial-sampling of alleles with the observed frequencies for all loci. Model comparisons were carried out using Akaike’s weight of evidence, as suggested by <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Excoffier <em>et al.</em> (2013)</a>. To assess the performance of the models, we applied a multinomial comparison between expected and observed SFS by calculating Anscombe residuals (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gutenkunst <em>et al.</em> 2009</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Models analyzed with fastsimcoal2. From left to right, constant ancestral population size, bottlenecked ancestral population, and expanding ancestral population. The constant ancestral population size model is the most likely as indicated by the percentage of bootstraps supporting each of the three models given above each tree. Figures within each branch are estimates of the effective population sizes of the descendent and ancestral populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1632398004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Models analyzed with fastsimcoal2. From left to right, constant ancestral population size, bottlenecked ancestral population, and expanding ancestral population. The constant ancestral population size model is the most likely as indicated by the percentage of bootstraps supporting each of the three models given above each tree. Figures within each branch are estimates of the effective population sizes of the descendent and ancestral populations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1979/F2.medium.gif" width="440" height="168"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/7/1979/F2.medium.gif" width="440" height="168"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468748" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-13" class="first-child">Models analyzed with fastsimcoal2. From left to right, constant ancestral population size, bottlenecked ancestral population, and expanding ancestral population. The constant ancestral population size model is the most likely as indicated by the percentage of bootstraps supporting each of the three models given above each tree. Figures within each branch are estimates of the effective population sizes of the descendent and ancestral populations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>SNP divergence analysis based on F<sub>ST</sub> and allele frequency difference</h3><p id="p-14">In this analysis, we combined Hudson’s <em>F</em><sub>ST</sub> and allele frequency difference (AFD) as a distance measurement of SNP divergence between populations. By comparing to the expected distribution of the most likely demographic scenario, we identified outliers with higher divergence as the clue of either diversifying selection.</p><p id="p-15">We first dispersed all SNPs on orthogonal coordinates of <em>F</em><sub>ST</sub> and AFD. The distance of each SNP to the origin (<em>F</em><sub>ST</sub> = 0, AFD = 0) was calculated as a new summary statistics for SNP divergence distance (div<em>D</em>) that combines <em>F</em><sub>ST</sub> and difference in allele frequency (AFD):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1979/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/7/1979/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>We then dispersed SNPs on a third dimension by grouping them into nonsynonymous or silent (noncoding and synonymous) sites using a variable transformation based on factor analysis of mixed data (FAMD). We applied FAMD to our data using functions implemented in the R package ‘FactoMineR’ (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Le <em>et al.</em> 2008</a>). Divergence distance for a given gene was calculated by averaging values at SNPs within that gene.</p><p id="p-16">To identify candidate SNPs under diversifying selection, we calculated SNP distance distribution from the simulation datasets: namely, we first generated 1000 expected SFS under the most likely demographic history using parameters estimated by 1000 bootstrap runs. For each expected SFS, we drew the same number of SNPs, and calculated the expected allele frequencies and <em>F</em><sub>ST</sub> values. Simulated SNPs were also randomly grouped as if they belonged to the same genes as in the observed dataset. For each simulated dataset, we then calculated div<em>D</em> for SNPs and genes and calculated the 95%, 97.5%, and 99% envelopes based on the ranked SNP distance to the origin. We used the maximum values of these three envelopes of the 1000 bootstrap runs as the cutoffs for outlier selection at 5%, 2.5%, and 1% significance levels.</p></div><div id="sec-7" class="subsection"><h3>F<sub>ST</sub> outliers</h3><p id="p-17"><em>F</em><sub>ST</sub> has been used extensively as a powerful statistic to identify loci under natural selection (<em>e.g.</em>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beaumont and Balding 2004</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beaumont and Nichols 1996</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Foll and Gaggiotti 2008</a>). Compared to neutral SNPs, outlier SNPs with higher <em>F</em><sub>ST</sub> values could indicate the presence of diversifying selection, while lower <em>F</em><sub>ST</sub> values could hint at balancing selection. We performed the <em>F</em><sub>ST</sub> outlier test implemented in the program BayeScan version 2.1, which assumes that allele frequencies follow a multinomial Dirichlet distribution that arises under a wide range of demographic models (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Foll and Gaggiotti 2008</a>). To account for the difference in numbers of putative neutral and selected loci, we chose a prior odds ratio of 10, which should be conservative in our case. We performed 20 pilot runs with 5000 iterations and 500,000 iterations MCMC, with an additional 50,000 iterations as burn-in. We calculated Q-values (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Storey 2003</a>) as posterior probabilities corrected for multiple tests to estimate the likelihood of the model with selection compared to the neutral model.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-18">All short-read libraries have been deposited in NCBI Sequence Read Archive (SRA) under accession number PRJNA272968. The reference sequences, gene annotation, SNPs, as well as custom Perl and R scripts used in this study have been deposited in Dryad <a href="http://dx.doi.org/10.5061/dryad.c5250">http://dx.doi.org/10.5061/dryad.c5250</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Sampling and sequencing</h3><p id="p-19">We isolated diploid DNA from 48 individuals drawn from populations sampled in the Fennoscandian domain (FEN) at latitudes ranging from 61 to 67°N, and 48 individuals drawn from populations of the Alpine domain (ALP) with latitudes ranging from 46°N to 47°N (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We selected 88 different gene fragments covering in total 111,297 bp. On average, because of the filtering each base was covered by approximately 200 reads (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028753/-/DC1/FigureS1.pdf">Figure S1</a>), and no significant difference was detected between the two pools (two-sample Kolmogorov-Smirnov test <em>p</em>-value = 0.99).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468710/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468710/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of aligned reads and number of alleles identified in both pooled population data sets</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Site frequency spectrum and nucleotide diversity</h3><p id="p-21">In total, we identified 2791 SNPs from the 88 gene fragments; 2436 SNPs were shared between the two pools, and 60 and 295 SNPs were private to the FEN and ALP pools, respectively. The private SNPs tended to have a low frequency in the pool where they were found (the mean frequency of private SNPs in the FEN and ALP domains were 0.06 and 0.1, respectively). A total of 1600 SNPs as located in noncoding regions, and 546 SNPs were synonymous; 632 SNPs caused amino acid replacement (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). We also found 11 premature stop-codons in open reading frames, and two mutations from stop-codons to amino acids. The site frequency spectra of both pools deviated from neutrality, showing an excess of minor alleles at low frequencies (&lt; 0.1), possibly indicating population expansion after the two pools diverged.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of summary statistics calculated based on all SNPs in both pools. (A) Site frequency spectra on silent and nonsynonymous SNPs; (B) Nucleotide diversity (&#x3C0;w); (C) Tajima&#x2019;s D; (D) FST." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1632398004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distributions of summary statistics calculated based on all SNPs in both pools. (A) Site frequency spectra on silent and nonsynonymous SNPs; (B) Nucleotide diversity (&#x3C0;&lt;em&gt;&lt;sub&gt;w&lt;/sub&gt;&lt;/em&gt;); (C) Tajima&#x2019;s &lt;em&gt;D&lt;/em&gt;; (D) &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1979/F3.medium.gif" width="440" height="407"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/7/1979/F3.medium.gif" width="440" height="407"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468653" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Distributions of summary statistics calculated based on all SNPs in both pools. (A) Site frequency spectra on silent and nonsynonymous SNPs; (B) Nucleotide diversity (π<em><sub>w</sub></em>); (C) Tajima’s <em>D</em>; (D) <em>F</em><sub>ST</sub>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">On average, we observed a higher number of segregating sites per gene in the ALP dataset (mean <em>S</em> = 31) than in the FEN dataset (mean <em>S</em> = 28). We noticed slight difference in nucleotide diversity between the two datasets using paired Wilcoxon Rank Sum test (mean π = 0.0056 and 0.0051, for ALP and FEN, respectively, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Similarly, estimates of Tajima’s <em>D</em> between ALP and FEN also differed, though the difference is unlikely to be biologically meaningful (mean <em>D</em> = –1.1 and –1.04, respectively, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). Finally, because alleles at very low frequency could simply be false SNPs (<em>e.g.</em>, sequencing errors), we also calculated the summary statistics for all alleles with frequencies ≥ 0.01. Results based on this subset of 2062 SNPs gave similar estimates for both datasets: π = 0.0049 and 0.005, <em>D</em> = –1.03 and –1.03 for FEN and ALP datasets, respectively.</p></div><div id="sec-12" class="subsection"><h3>Population divergence</h3><p id="p-24">Population divergence between the two domains was estimated for each single SNP. We chose Hudson’s estimator of <em>F</em><sub>ST</sub> since it is less sensitive to sample size difference than other estimators (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bhatia <em>et al.</em> 2013</a>). More than half of the SNPs had an estimate of <em>F</em><sub>ST</sub> value less than 0.01, and close to 90% of the SNPs had an estimate of <em>F</em><sub>ST</sub> below 0.1 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>). Twenty-seven highly divergent SNPs had <em>F</em><sub>ST</sub> values above 0.3 (99% quantile). About half of these high <em>F</em><sub>ST</sub> SNPs were distributed over four genes, <em>PaGI</em> (2 SNPs), <em>PaAP2L3</em> (5), and <em>Pa02776N18</em> (5) and <em>Pa02728N15</em> (5). An overall estimate of population divergence was calculated based on combining <em>F</em><sub>ST</sub> values across all SNPs. We first averaged the numerator and denominator of Hudson’s estimator separately and used the ratio as the genome-wide estimate of <em>F</em><sub>ST,</sub> which gave a value of 0.047. The overall estimate of <em>F</em><sub>ST</sub> obtained by averaging the individual <em>F</em><sub>ST</sub> values of all SNPs gave an estimate of 0.032. This smaller value is expected since this second estimator of <em>F</em><sub>ST</sub> is more sensitive to ascertainment bias and private SNPs (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bhatia <em>et al.</em> 2013</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Jackson <em>et al.</em> 2014</a>). If allele frequency was restricted to values ≥ 0.01, <em>F</em><sub>ST</sub> was 0.037 between the two domains. So, all estimates show that population divergence between the two domains is limited. To find out whether this is due to a recent divergence, or to gene flow, we estimated migration and divergence time.</p></div><div id="sec-13" class="subsection"><h3>Joint demographic history inference</h3><p id="p-25">We simulated the joint site frequency spectrum (SFS) based on an “Isolation with migration” model, using all silent SNPs. Three ancestral population models were tested: a constant, expanding, or bottlenecked ancestral population size (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Demographic inference was conducted using 1000 bootstraps of the SFS, and we reported the 95% confidence interval for each parameter (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028753/-/DC1/FigureS2.pdf">Figure S2</a>). Model comparison using Akaike’s weight of evidence (<em>w</em>) showed that 83.8% of bootstraps supported the constant size model (mean <em>w</em> = 0.53), while only 8.7% and 7.5% supported the bottleneck (mean <em>w</em> = 0.24) and growth models (mean <em>w</em> = 0.23), respectively. Two-dimensional multinomial comparison between the constant size model and observed data also showed a good fit with the maximum Anscombe residual less than 2.8 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028753/-/DC1/FigureS3.pdf">Figure S3</a>). Assuming a mutation rate of 1.1 × 10<sup>−9</sup> per site per year (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chen <em>et al.</em> 2012b</a>), and an average generation time of 25 yr, we obtained an effective population size around 41,000 (95% C.I.: 13,000–47,000) for FEN and a slightly larger one, at 51,000 for ALP (95% C.I.: 45,000–65,000). The ancestral population size was nearly ten times larger (∼400,000). As noted by <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Becquet and Przeworski (2009)</a>, isolation–migration models often yield an estimate of the ancestral effective population size that is much larger than that of either descendant population. Simulations indicate that this large value of the ancestral population size could reflect the fact that the ancestral population was actually structured. The two populations started to diverge around 5 million yr ago (Mya) (95% C.I.: 3.6 Mya–14 Mya, for the constant size model). We also found more migrants in the direction from north to south (M∼15 per generation, 95% C.I.: 1.7–19.5) than in the opposite direction (M &lt; 2 per generation; 95% C.I.: 0.9–8.9).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/468666/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/468666/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Posterior distribution modes and 95% confidence intervals of demographic parameters estimated with fastsimcoal2</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>SNP divergence analyses based on F<sub>ST</sub> and AFD</h3><p id="p-29"><em>F</em><sub>ST</sub> is a fixation index and measures genetic divergence between populations, but is not an absolute measure of population differentiation: two populations can have no alleles in common and yet have a very low <em>F</em><sub>ST</sub> value, if each of them is highly variable (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Charlesworth 1998</a>). Hence, part of the information present in the difference in allele frequencies between populations is not captured by <em>F</em><sub>ST</sub>. We therefore calculated a new summary statistic div<em>D</em> from both AFD and <em>F</em><sub>ST</sub> values (see <em>Materials and Methods</em>). While over half of the SNPs with AFD &lt; 0.1 and <em>F<sub>ST</sub></em> &lt; 0.01 clustered tightly around the origin, SNPs spread quickly as AFD increased (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We also separated the SNPs into nonsynonymous and silent sites (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028753/-/DC1/FigureS4.pdf">Figure S4</a>). The nonsynonymous SNPs in general tend to be less diverged (less spread in <em>F</em><sub>ST</sub> and AFD values) between FEN and ALP populations than silent ones. The distance distribution between these two groups of SNPs differed significantly (Kolmogorov-Smirnov test <em>p</em>-value = 1.9e–4) in both mean (0.87 and 1.1 for nonsynonymous and silent SNPs, respectively, Wilcoxon Rank Sum test <em>p</em>-value = 7e–4) and variance (0.55 and 1.1, F-test <em>p</em>-value &lt; 2.2e–16). This difference could reflect the fact that nonsynonymous mutations are more likely to be under purifying selection and therefore exhibit much lower divergence between geographically isolated populations than silent mutations that are primarily affected by genetic drift (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Barreiro <em>et al.</em> 2008</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Jackson <em>et al.</em> 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Maruki <em>et al.</em> 2012</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scatterplot of divergence analysis based on FST and allele frequency difference. Gray dots show the silent SNPs, and pink dots show the nonsynonymous SNPs. Divergence distance of SNP based on FST and AFD. Cutoff envelopes for different significance levels are shown in colored ellipses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1632398004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Scatterplot of divergence analysis based on &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; and allele frequency difference. Gray dots show the silent SNPs, and pink dots show the nonsynonymous SNPs. Divergence distance of SNP based on &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; and AFD. Cutoff envelopes for different significance levels are shown in colored ellipses.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1979/F4.medium.gif" width="440" height="412"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/7/1979/F4.medium.gif" width="440" height="412"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468742" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">Scatterplot of divergence analysis based on <em>F</em><sub>ST</sub> and allele frequency difference. Gray dots show the silent SNPs, and pink dots show the nonsynonymous SNPs. Divergence distance of SNP based on <em>F</em><sub>ST</sub> and AFD. Cutoff envelopes for different significance levels are shown in colored ellipses.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">On the other hand, SNPs involved in local adaptation may show higher divergence level due to differences in environmental stress across populations. To control for the effect of genetic drift, we took advantage of the simulated SFS to estimate the variation in <em>F</em><sub>ST</sub> and AFD that could be generated by the demographic history. We then built significance envelopes at 5%, 2.5%, and 1% to select SNP outliers based on their ranked distance to the origin (see details in <em>Materials and Methods</em>). This resulted in 132, 68, and 15 SNPs in the 5%, 2.5%, and 1% significance level tails, respectively (see the detailed list in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028753/-/DC1/FileS1.xlsx">File S1</a>). Remarkably, the majority (&gt; 70%) of these outliers came from nine genes: <em>PaAP2L3</em> (39), <em>PaPRR3</em> (12), <em>Pa02728N15</em> (13), <em>Pa02745J16</em> (11), <em>Pa02776N18</em> (10), <em>Pa0273A22</em> (8), <em>PaGI</em> (3), <em>Pa00931A07</em> (3), and <em>Pa02749P05</em> (3). Another interesting observation is that there were 68 outlier SNPs (5% tail), where the allele frequencies in the northern populations (FEN) were close to fixation or loss (freq ≥ 0.99 or freq ≤ 0.01, 30 were found within the 2.5% significance tail, and four were found within the 1% significance tail). This was most obvious for the gene <em>PaAP2L3</em>, which showed a clear enrichment in such SNP (35 SNPs). In contrast, only two SNPs were found in the southern populations (ALP), with frequencies close to 0, and none close to fixation, while this ratio was 518/269 in the total dataset. On the other hand, we also noticed that more outlier loci in the ALP populations had intermediate allele frequencies (0.3–0.7) than in FEN (55 <em>vs.</em> 21 at 5% tail, 40 <em>vs.</em> 9 at 2.5% tail, 13 <em>vs.</em> 1 at 1% tail, but 357 <em>vs.</em> 303 in the total dataset). The enrichment of fixed/lost alleles, and lack of intermediate alleles in northern populations, could indicate a stronger purifying selection pressure in northern populations than in southern ones, if one assumes that the two domains had similar demographic histories. Alternatively, Fennoscandian populations were much more affected than Alpine ones by glacial cycles during the Quaternary, and the difference could also be the result of processes such as loss of heterozygosity or allele surfing during postglacial range expansion (<em>e.g.</em>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Goodsman <em>et al.</em> 2014</a>, and references therein).</p><p id="p-32">By using the same strategy as for individual SNPs, we calculated div<em>D</em> for genes by averaging values of SNPs, and selected outlier genes as genes outside the simulated 95% envelope. Seven genes showed a higher average level of divergence including <em>PaAP2L3</em>, <em>PaGI</em>, <em>PaEBS</em>, <em>PaARFL</em>, <em>PaAtCUL3a</em>, <em>Pa02717F24</em>, and <em>Pa02728N15</em>.</p><p id="p-33">To account for the difference between synonymous and silent SNPs, we also applied FAMD, which incorporated the nonsynonymous/silent information as a categorical variable. The first three components explained 41%, 32% and 27% of the variance, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028753/-/DC1/FigureS4.pdf">Figure S4</a>). We observed 126 outlier SNPs for separate div<em>D</em> analyses, and 125 outlier SNPs for the FAMD analysis. No additional outliers were identified by either method compared to the analysis of mixed silent and nonsynonymous SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028753/-/DC1/FileS1.xlsx">File S1</a>).</p><p id="p-34">Low frequency alleles could still be sequencing errors even though we performed cross-validation among replicates and filtered the data carefully during SNP calling (see <em>Materials and Methods</em>). Thus, we also assessed the robustness of our results by redoing the analyses in subsets of data with minor allele frequency ≥ 0.01 and 0.1. The subsets included 2082 and 963 SNPs, respectively. When only SNPs with minor allele frequency ≥ 0.01 are retained, we obtained 94 outliers at 5% tail, 52 at 2.5% tail, and 20 at 1% tail, 89, 49, and 12 of which overlapped with those identified in the total dataset, respectively. When the threshold was at frequency ≥ 0.1, we identified 39 outliers in the 5% tail, 15 at 2.5%, and 2 at 1%. All of them had been reported in the total dataset. The distance distributions of the three datasets [all SNPs (full spectrum), minor allele frequency ≥ 0.01, and minor allele frequency ≥ 0.1] were highly correlated (Pearson’s correlation test, <em>p</em>-value &lt; 2e–16, coefficient = 0.99). The relative proportions of outliers selected at 5% remained similar in all three datasets (4–4.7%). Compared to the complete dataset, a decrease of the number of outliers was observed in the most stringent subset (frequency ≥ 0.1) at 2.5% tail (from 2.4 to 1.56%), and at 1% tail (from 0.5 to 0.2%).</p></div><div id="sec-15" class="subsection"><h3>Testing for local adaptation: F<sub>ST</sub> outliers</h3><p id="p-35">To identify candidate SNPs under local adaptation, we used BayeScan v2.1 (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Foll and Gaggiotti 2008</a>). The method decomposes <em>F</em><sub>ST</sub> into a population-specific effect shared by all SNPs, and a locus-specific effect shared by all populations. The presence of selection is inferred if the locus-specific effect is necessary to explain the observed divergence. The log probabilities of choosing alternative models were calculated and corrected for multiple tests as Q-values (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). By assuming a prior odd of neutral to selected mutations at 10, we identified 24 outlier SNPs at Q-value ≤ 0.05 containing four nonsynonymous (<em>PaGI_F4_233</em>, <em>PaPRR3_2066</em>, <em>PaAP2L3_1420</em>, and <em>Pa0279G15_694</em>) and one synonymous (<em>PaAP2L3_1621</em>) SNP. Outliers came mainly from <em>PaAP2L3</em> (11), <em>Pa02728N15</em> (4), and <em>Pa0273A22</em> (3), but also <em>PaGI</em> (2), <em>PaPRR3</em> (1), <em>PaAtCUL3a</em> (1), <em>Pa02711F10</em> (1), and <em>Pa0279G15</em> (1). All 24 outliers at the 5% cut-off of Q-values had been also identified by div<em>D</em> analyses at 2.5% significant tail. Only four of them were nonsynonymous changes. If we chose a Q-value cutoff at 1%, six outliers would be identified, and four of them were also picked by div<em>D</em> at 1% significant tail (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028753/-/DC1/FileS1.xlsx">File S1</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between FST and allele frequency difference. The color strips show the FST contours, and the gray dash lines parallel to the diagonal show the AFD contour lines. SNPs used in this study were mapped using points: small black dots are SNPs with divergence distance under 97.5% tail of simulated distribution, while outliers are shown in difference shapes, and colors corresponding to their q-values calculated by BayeScan. The red dots correspond to outliers for FST values, and allele frequencies that were also outliers with BayeScan with Q-value &#x2264;&#xA0;0.05. The analysis based on FST and allele frequency difference also picks outliers with BayeScan Q-value ranging from 0.05 to 0.8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1632398004" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; and allele frequency difference. The color strips show the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; contours, and the gray dash lines parallel to the diagonal show the AFD contour lines. SNPs used in this study were mapped using points: small black dots are SNPs with divergence distance under 97.5% tail of simulated distribution, while outliers are shown in difference shapes, and colors corresponding to their q-values calculated by BayeScan. The red dots correspond to outliers for &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; values, and allele frequencies that were also outliers with BayeScan with Q-value &#x2264;&#xA0;0.05. The analysis based on &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; and allele frequency difference also picks outliers with BayeScan Q-value ranging from 0.05 to 0.8.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/7/1979/F5.medium.gif" width="440" height="434"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/7/1979/F5.medium.gif" width="440" height="434"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/7/1979/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/468579" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Relationship between <em>F</em><sub>ST</sub> and allele frequency difference. The color strips show the <em>F</em><sub>ST</sub> contours, and the gray dash lines parallel to the diagonal show the AFD contour lines. SNPs used in this study were mapped using points: small black dots are SNPs with divergence distance under 97.5% tail of simulated distribution, while outliers are shown in difference shapes, and colors corresponding to their q-values calculated by BayeScan. The red dots correspond to outliers for <em>F</em><sub>ST</sub> values, and allele frequencies that were also outliers with BayeScan with Q-value ≤ 0.05. The analysis based on <em>F</em><sub>ST</sub> and allele frequency difference also picks outliers with BayeScan Q-value ranging from 0.05 to 0.8.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-37">In previous studies, we have detected departures from neutrality among phenology-related genes using scattered samples across the whole range of Norway spruce (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Källman <em>et al.</em> 2014</a>), and evidence of local adaptation by analyzing variation along latitudinal clines in Norway spruce and Siberian spruce (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2012a</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">2014</a>). In the present study, we used an intermediate sampling strategy as well as a different sequencing approach, and sequenced 88 genes in two pools of individuals drawn from the two main historical domains of Norway spruce: the Alpine and Fennoscandian domains. We then used silent polymorphisms to infer the demographic history of the two domains, and all polymorphic sites to test for the presence of selection. Different approaches, including a novel one, were used for the latter. The main conclusions are that, based on the silent variation in this sample of 88 genes, the two domains diverged some 5 Mya and that purifying and positive selection both played an important role in the divergence of the two domains. In particular, we confirmed the importance of selection at a large geographic scale on genes related to the circadian clock, such as <em>PaPRR3</em> and <em>PaGI</em>. Below we shall discuss those results in turn, but we start by addressing some of the technical issues encountered when analyzing pooled sequence data.</p><div id="sec-17" class="subsection"><h3>Estimate population genetic parameters from pooled ngs data</h3><p id="p-38">Pooled sequencing technology has proven to be not only cost-effective but, more importantly, a fast and reliable access to estimates of the site frequency spectrum and population divergence if a proper sampling and statistical strategy is used (<em>e.g.</em>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Boitard <em>et al.</em> 2012</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Futschik and Schlotterer 2010</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fracassetti <em>et al.</em> 2015</a>). To minimize the error rate in this study, we took a number of precautionary steps at both the data collection and analysis phases of the project. We first sequenced both pools twice in order to increase the detection power of rare alleles, while securing a lower false positive rate (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bansal 2010</a>). Since the distribution of short-read coverage could be a major source of error in estimation of allele frequency and population divergence (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Futschik and Schlotterer 2010</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2014</a>), we then applied a two-step down-sampling to minimize the unequal coverage across libraries and genes, as well as different regions of the amplicons. Finally, subsampling at frequencies ≥ 0.01 and ≥ 0.1 indicates that estimates of polymorphism and divergence are not very sensitive to rare alleles. Although it is hard to estimate the accuracy of allele frequency estimates, congruence with previous studies lends support for the pooled approach. Earlier estimates of multi-locus nucleotide diversity, Tajima’s <em>D</em> and <em>F</em><sub>ST</sub> in <em>P. abies</em> populations are of the same order of magnitude (π = 0.0038, <em>D</em> = –0.90 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2010</a>); <em>F</em><sub>ST</sub> = 0.035–0.05 between populations of 61–68°N and populations of 47–53°N (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2012a</a>); mean π = 0.0021, <em>D</em> = –0.92, <em>F</em><sub>ST</sub> ∼0.12 in comparisons between populations of 66°N and populations of 46–47°N (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Heuertz <em>et al.</em> 2006</a>) as the ones obtained in this study (π = 0.005, <em>D</em> = –1.0, and <em>F</em><sub>ST</sub> = 0.047). The smaller nucleotide diversity and larger <em>F</em><sub>ST</sub> estimates obtained in <a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Heuertz <em>et al.</em> (2006)</a> could simply be due to the difference in sample sizes and choice of genes, since the present study and studies that are of more similar sizes (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2012a</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Larsson <em>et al.</em> 2013</a>) gave highly similar results.</p></div><div id="sec-18" class="subsection"><h3>Population divergence history</h3><p id="p-39">According to the analysis of the joint frequency spectrum of synonymous sites, the two domains diverged around 5 Mya, and gene flow between them has been extensive and mostly in a north to south direction. This divergence estimate is much larger than that proposed earlier by <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Lagercrantz and Ryman (1990)</a> who, based on Nei’s genetic distance calculated from 22 allozyme loci, obtained an approximate divergence time of 40,000 yr. Our estimate of 5 Mya, however, is of the same order of magnitude as the 6 million yr obtained by <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Lockwood <em>et al.</em> (2013)</a> from plastid (matK, trnk, and rbcL), mitochondrial (nad1 and nad5), and nuclear (4CL) sequences. Given that the latter primarily reflects divergence at cytoplasmic loci, one would expect to obtain shorter estimates of divergence since the effective population sizes are twice as large for nuclear as for cytoplasmic markers in monoecious species. On the other hand, neither <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Lockwood <em>et al.</em> (2013)</a> nor <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Lagercrantz and Ryman (1990)</a> considered migration when estimating divergence time, and hence their estimates and that obtained here are difficult to compare. Finally, one major caveat of the present estimate is the fact that synonymous sites come from genes that might have been under selection, and thereby could have been affected by nearby selection on nonsynonymous sites. Although we could not estimate linkage disequilibrium in these pooled data, a rapid decay of LD within genes has generally been found in previous studies (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Heuertz <em>et al.</em> 2006</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Larsson <em>et al.</em> 2013</a>), so polymorphism at the silent sites is unlikely to have been strongly affected by selection across the 88 genes. Also, when using Bayescan, a single synonymous SNP was detected. So, altogether, linked selection is unlikely to have had a major impact on demographic inferences.</p></div><div id="sec-19" class="subsection"><h3>Evidence of selection</h3><p id="p-40">Previous studies of genetic differentiation have focused either on quantifying population divergence and past demography from variation at neutral sites, or on detecting genomic region under diversifying selection by using genome scans. Despite overwhelming evidence of its importance in evolution, the effect of purifying selection on population divergence has seldom been considered explicitly. Recently, <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Jackson <em>et al.</em> (2014)</a> compared genetic differentiation between French and Rwandan populations of <em>Drosophila melanogaster</em> at 0-fold sites, four-fold sites and introns. They showed that, in agreement with the expected effect of purifying selection, sites in 0-fold sites in conserved genes are less differentiated than those in less conserved genes. In the present study, the nonsynonymous SNPs tend to have both lower frequency differences, and lower <em>F</em><sub>ST</sub> values between the two domains than silent ones. Assuming that both synonymous and nonsynonymous sites are otherwise affected in the same way by demography, purifying selection seems the most likely explanation to this observation. We note that this does not contradict the enrichment in nonsynonymous outliers in sites with high <em>F</em><sub>ST</sub>. These outliers are, by definition, a very small proportion of all sites, and here we have been looking at the overall effect of divergence on those sites. Using Bayescan, we also detected a handful of outliers. <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> illustrates the difference between the outliers detected by the divergence analysis, and those detected by Bayescan. In our specific case, Bayescan preferentially picks outliers on the edges of the SFS matrix (where alleles are close to fixation or loss in one of the two populations), while the divergence analysis retains SNPs over a larger spectrum of frequencies in both populations. Bayescan’s outlier SNPs have higher <em>F</em><sub>ST</sub> values than SNPs with the same allele frequency difference (AFD) between populations but that are far from fixation in either populations (SNPs with the same AFD are distributed along the gray dashed lines in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). However, this also means that given SNPs of same <em>F</em><sub>ST</sub> values (located within the same color strip in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>) Bayescan outliers have lower AFD values (since <em>F</em><sub>ST</sub> contours are bent, and are not parallel with AFD lines except near the diagonal). Few SNPs with Q-value lower than 0.2 in Bayescan are in nonmarginal areas, and only one with intermediate frequencies has a Q-value lower than 0.05. Therefore, in this study, and perhaps more generally, Bayescan tends to target old mutations that went to fixation or new mutations that have been under strong selective sweep. For the genomic areas where these loci are located, migration barriers may have long been established, even if the average migration rate across all loci has been high between the two domains of Norway spruce. However, this is not always the case, and, for loci under weaker selection, for example, loci underlying a quantitative trait controlled by a large number of loci, each of which is weakly affected by selection (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Latta 1998</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Le Corre and Kremer 2012</a>), gene flow will keep introducing new variants and prevent the locus to go to fixation. In such cases, AFD would be high, but neither allele will be fixed (lower-right or upper-left areas of <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>). These loci will be identified by the divergence distance method as putatively under selection, but not by Bayescan, which is more stringent. Additionally for pooled samples, to be able to select Bayescan outliers large sample sizes are required as the outliers usually have a frequency close to fixation or loss in one of the populations (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Lynch <em>et al.</em> 2014</a>). Many outliers of our divergence analysis have intermediate frequency, and thus can lower the requirement for the number of individuals. Finally, one caveat is that, as for many outlier methods, the present method can lead to false positives and false negatives if the wrong demographic model is used it the analysis.</p><p id="p-41">The two most interesting loci detected in the present study are the circadian clock genes <em>PaGI</em> (<em>Gigantea</em>) and <em>PaPRR3</em>, since they were also identified as top candidate genes for local adaptation in previous studies. Gigantea homologs consistently show evidence of non-neutral evolution, both in conifers (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2012a</a>, <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">2014</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Holliday <em>et al.</em> 2010</a>), and in angiosperms (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Keller <em>et al.</em> 2012</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Olson <em>et al.</em> 2013</a>). Its pattern of polymorphism within spruce species is intriguing, but has been hard to analyze because of the lack of silent polymorphism in the parts of the gene that were sequenced so far. Repeated selective sweeps have been proposed for <em>Populus balsamifera</em>, where a similar pattern of polymorphism is observed (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Keller <em>et al.</em> 2012</a>), but this hypothesis remains to be confirmed. <em>PaPRR3</em> has not been included as often in candidate gene panels, and less is known about it. In Norway spruce, <em>PaPRR3</em> departed from neutrality when departure from neutrality was tested from a range-wide sample (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Källman <em>et al.</em> 2014</a>), but <em>PaPRR3</em> did not exhibit latitudinal cline in allele frequency as one might have expected from a circadian clock gene if it were directly involved in adaptation to local conditions (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Chen <em>et al.</em> 2012a</a>). Finally, it is also worth mentioning <em>AP2L3</em> as a gene of special interest. While genetic variation in this gene has not been studied as extensively as that of <em>GI</em>, <em>PRR3</em>, or <em>FTL2</em>, <em>AP2L3</em> exhibits an intriguing pattern in the present study, and its involvement in development in Norway spruce (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nilsson <em>et al.</em> 2007</a>) makes it an interesting candidate gene.</p><p id="p-42">In conclusion, this study shows that combining PCR with a pooled sequencing strategy is a cost-effective and reliable approach for generating multilocus population genetic data sets in nonmodel organisms. The inferred pattern of diversity and population divergence clearly shows that the Alpine and Fennoscandian domains of Norway spruce remain genetically closely related although they diverged 5 Mya. This is probably due to pollen flow, as the two domains are clearly differentiated for mitochondrial DNA, which is maternally inherited (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Lockwood <em>et al.</em> 2013</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Tsuda <em>et al.</em> 2016</a>). In addition we detected a set of genes deviating from neutral expectations. Together with earlier studies of candidate genes for adaptation to local light conditions, both <em>PaPRR3</em> and <em>PaGI</em> are emerging as central genes that would certainly deserve further attention. In particular, it would be very informative to resequence a larger part of the PaGI gene in a larger sample in order to test for selection, and to conduct more extensive functional studies in both genes.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">We would like to thank Kerstin Jeppsson for help in the lab. Financial support from the EU (Noveltree project), Formas, the Erik Philip Sörensen Foundation, and BioDiversa (Projects LinkTree and Tiptree) is acknowledged.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-44">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028753/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.028753/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-45"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-03-05"><span class="received-label">Received </span>March 5, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-04-28"><span class="accepted-label">Accepted </span>April 28, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Chen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.1" data-doi="10.1093/bioinformatics/btq214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A statistical method for the detection of variants from next-generation resequencing of DNA pools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">i318</span>–<span class="cit-lpage">i324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq214%26rft_id%253Dinfo%253Apmid%252F20529923%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTIvaTMxOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.2" data-doi="10.1038/ng.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barreiro</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laval</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quintana-Murci</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Natural selection has driven population differentiation in modern humans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">340</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBarreiro%26rft.auinit1%253DL.%2BB.%26rft.volume%253D40%26rft.issue%253D3%26rft.spage%253D340%26rft.epage%253D345%26rft.atitle%253DNatural%2Bselection%2Bhas%2Bdriven%2Bpopulation%2Bdifferentiation%2Bin%2Bmodern%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.78%26rft_id%253Dinfo%253Apmid%252F18246066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18246066&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253548400026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.3" data-doi="10.1111/j.1365-294X.2004.02125.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Identifying adaptive genetic divergence among populations from genome scans.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">969</span>–<span class="cit-lpage">980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBeaumont%26rft.auinit1%253DM.%2BA.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D969%26rft.epage%253D980%26rft.atitle%253DIdentifying%2Badaptive%2Bgenetic%2Bdivergence%2Bamong%2Bpopulations%2Bfrom%2Bgenome%2Bscans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02125.x%26rft_id%253Dinfo%253Apmid%252F15012769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02125.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15012769&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220153000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.4" data-doi="10.1098/rspb.1996.0237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Evaluating loci for use in the genetic analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">263</span>: <span class="cit-fpage">1619</span>–<span class="cit-lpage">1626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DBeaumont%26rft.auinit1%253DM.%2BA.%26rft.volume%253D263%26rft.issue%253D1377%26rft.spage%253D1619%26rft.epage%253D1626%26rft.atitle%253DEvaluating%2BLoci%2Bfor%2BUse%2Bin%2Bthe%2BGenetic%2BAnalysis%2Bof%2BPopulation%2BStructure%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1996.0237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMzoiMjYzLzEzNzcvMTYxOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.5" data-doi="10.1111/j.1558-5646.2009.00662.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Becquet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Learning about modes of speciation by computational approaches.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">2547</span>–<span class="cit-lpage">2562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBerner%26rft.auinit1%253DD.%26rft.volume%253D63%26rft.issue%253D10%26rft.spage%253D2547%26rft.epage%253D2562%26rft.atitle%253DLearning%2Babout%2Bmodes%2Bof%2Bspeciation%2Bby%2Bcomputational%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00662.x%26rft_id%253Dinfo%253Apmid%252F19228187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00662.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19228187&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270651800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.6" data-doi="10.1101/gr.154831.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhatia</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Estimating and interpreting F-ST: the impact of rare variants.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1514</span>–<span class="cit-lpage">1521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.154831.113%26rft_id%253Dinfo%253Apmid%252F23861382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIzLzkvMTUxNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.7" data-doi="10.1093/bioinformatics/btt178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackman</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pleasance</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coope</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assembling the 20 Gb white spruce (<em>Picea glauca</em>) genome from whole-genome shotgun sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1492</span>–<span class="cit-lpage">1497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt178%26rft_id%253Dinfo%253Apmid%252F23698863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjkvMTIvMTQ5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.8" data-doi="10.1093/molbev/mss090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boitard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlotterer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolte</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandey</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futschik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Detecting selective sweeps from pooled next-generation sequencing samples.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2177</span>–<span class="cit-lpage">2186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss090%26rft_id%253Dinfo%253Apmid%252F22411855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI5LzkvMjE3NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.9" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTUvMjExNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.10" data-doi="10.1093/oxfordjournals.molbev.a025953"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Measures of divergence between populations and the effect of forces that reduce variability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">538</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DCharlesworth%26rft.auinit1%253DB%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D538%26rft.epage%253D543%26rft.atitle%253DMeasures%2Bof%2Bdivergence%2Bbetween%2Bpopulations%2Band%2Bthe%2Beffect%2Bof%2Bforces%2Bthat%2Breduce%2Bvariability%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025953%26rft_id%253Dinfo%253Apmid%252F9580982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE1LzUvNTM4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.11" data-doi="10.1038/hdy.2009.88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyllenstrand</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New insights on the speciation history and nucleotide diversity of three boreal spruce species and a tertiary relict.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DChen%26rft.auinit1%253DJ.%26rft.volume%253D104%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D14%26rft.atitle%253DNew%2Binsights%2Bon%2Bthe%2Bspeciation%2Bhistory%2Band%2Bnucleotide%2Bdiversity%2Bof%2Bthree%2Bboreal%2Bspruce%2Bspecies%2Band%2Ba%2BTertiary%2Brelict.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.88%26rft_id%253Dinfo%253Apmid%252F19639012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19639012&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272794600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.12" data-doi="10.1534/genetics.112.140749"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyllenstrand</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaina</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Disentangling the roles of history and local selection in shaping clinal variation of allele frequencies and gene expression in Norway spruce (<em>Picea abies</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.140749%26rft_id%253Dinfo%253Apmid%252F22542968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzMvODY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.13" data-doi="10.1186/1471-2164-13-589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uebbing</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyllenstrand</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagercrantz</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Sequencing of the needle transcriptome from Norway spruce (<em>Picea abies</em> Karst L.) reveals lower substitution rates, but similar selective constraints in gymnosperms and angiosperms.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DChen%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D589%26rft.epage%253D589%26rft.atitle%253DSequencing%2Bof%2Bthe%2Bneedle%2Btranscriptome%2Bfrom%2BNorway%2Bspruce%2B%2528Picea%2Babies%2BKarst%2BL.%2529%2Breveals%2Blower%2Bsubstitution%2Brates%252C%2Bbut%2Bsimilar%2Bselective%2Bconstraints%2Bin%2Bgymnosperms%2Band%2Bangiosperms.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-589%26rft_id%253Dinfo%253Apmid%252F23122049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23122049&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.14" data-doi="10.1534/genetics.114.163063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsuda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stocks</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Clinal variation at phenology-related genes in spruce: parallel evolution in FTL2 and Gigantea?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">1025</span>–<span class="cit-lpage">1038</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.163063%26rft_id%253Dinfo%253Apmid%252F24814465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ny8zLzEwMjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTk3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.15" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ekberg</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eriksson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dormling</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Photoperiodic reactions in conifer species.</span> <abbr class="cit-jnl-abbrev">Ecography</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcography%26rft.volume%253D2%26rft.spage%253D255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupanloup</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Sanchez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Robust demographic inference from genomic and SNP data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: e1003905.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.18" data-doi="10.1534/genetics.108.092221"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">977</span>–<span class="cit-lpage">993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092221%26rft_id%253Dinfo%253Apmid%252F18780740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzIvOTc3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.19" data-doi="10.1371/journal.pone.0140462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fracassetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffin</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willi</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Validation of pooled whole-genome re-sequencing in <em>Arabidopsis lyrata</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0140462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0140462%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0140462%26rft_id%253Dinfo%253Apmid%252F26461136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0140462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26461136&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.20" data-doi="10.1534/genetics.110.114397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Futschik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlotterer</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The next generation of molecular markers from massively parallel sequencing of pooled DNA samples.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.114397%26rft_id%253Dinfo%253Apmid%252F20457880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvMjA3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.21" data-doi="10.1016/j.tpb.2014.08.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodsman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooke</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coltman</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genetic signature of rapid range expansions: How dispersal, growth and invasion speed impact heterozygosity and allele surfing.</span> <abbr class="cit-jnl-abbrev">Theor. Pop. Biol.,</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>. Available at: <a href="http://doi.org/10.1016/j.tpb.2014.08.005">http://doi.org/10.1016/j.tpb.2014.08.005</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPop.%2BBiol.%252C%26rft.volume%253D98%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2014.08.005%26rft_id%253Dinfo%253Apmid%252F25201435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2014.08.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25201435&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.22" data-doi="10.1371/journal.pgen.1000695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGutenkunst%26rft.auinit1%253DR.%2BN.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000695%26rft.epage%253De1000695%26rft.atitle%253DInferring%2Bthe%2Bjoint%2Bdemographic%2Bhistory%2Bof%2Bmultiple%2Bpopulations%2Bfrom%2Bmultidimensional%2BSNP%2Bfrequency%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000695%26rft_id%253Dinfo%253Apmid%252F19851460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19851460&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.23" data-doi="10.1534/genetics.106.065102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heuertz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Paoli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurman</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Multilocus patterns of nucleotide diversity, linkage disequilibrium and demographic history of Norway spruce [<em>Picea abies</em> (L.) Karst].</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">2095</span>–<span class="cit-lpage">2105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.065102%26rft_id%253Dinfo%253Apmid%252F17057229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC80LzIwOTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTk3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.24" data-doi="10.1111/j.1469-8137.2010.03380.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Widespread, ecologically relevant genetic markers developed from association mapping of climate-related traits in Sitka spruce (<em>Picea sitchensis</em>).</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DHolliday%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D188%26rft.issue%253D2%26rft.spage%253D501%26rft.epage%253D514%26rft.atitle%253DWidespread%252C%2Becologically%2Brelevant%2Bgenetic%2Bmarkers%2Bdeveloped%2Bfrom%2Bassociation%2Bmapping%2Bof%2Bclimate-related%2Btraits%2Bin%2BSitka%2Bspruce%2B%2528Picea%2Bsitchensis%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03380.x%26rft_id%253Dinfo%253Apmid%252F20663060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03380.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20663060&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282225000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.25" data-doi="10.1093/molbev/msu247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Keeping it local: evidence for positive selection in Swedish <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3026</span>–<span class="cit-lpage">3039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu247%26rft_id%253Dinfo%253Apmid%252F25158800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8xMS8zMDI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.26" data-doi="10.1371/journal.pgen.1005397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maddison</span>  <span class="cit-name-given-names">W. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Estimation of levels of gene flow from DNA sequence data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D583%26rft.epage%253D589%26rft.atitle%253DEstimation%2Bof%2Blevels%2Bof%2Bgene%2Bflow%2Bfrom%2BDNA%2Bsequence%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005397%26rft_id%253Dinfo%253Apmid%252F1427045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMyLzIvNTgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effects of purifying selection on patterns of genetic differentiation between <em>Drosophila melanogaster</em> populations.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D114%26rft.spage%253D163%26rft_id%253Dinfo%253Apmid%252F25227256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25227256&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-other" id="cit-6.7.1979.28"><div class="cit-metadata unstructured">Källman, T. 2009 Adaptive evolution and demographic history of Norway spruce (<em>Picea abies</em>). Doctoral Dissertation, Acta Universitatis Upsaliensis.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Mita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyllenstrand</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heuertz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Patterns of nucleotide diversity at photoperiod related genes in Norway spruce [<em>Picea abies</em> (L.) Karst.].</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: e953069.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.30" data-doi="10.1038/ng2116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keinan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullikin</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Measurement of the human allele frequency spectrum demonstrates greater genetic drift in East Asians than in Europeans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1251</span>–<span class="cit-lpage">1255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKeinan%26rft.auinit1%253DA.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1251%26rft.epage%253D1255%26rft.atitle%253DMeasurement%2Bof%2Bthe%2Bhuman%2Ballele%2Bfrequency%2Bspectrum%2Bdemonstrates%2Bgreater%2Bgenetic%2Bdrift%2Bin%2BEast%2BAsians%2Bthan%2Bin%2BEuropeans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2116%26rft_id%253Dinfo%253Apmid%252F17828266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17828266&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249737400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.31" data-doi="10.1093/molbev/mss121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levsen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Local adaptation in the flowering-time gene network of Balsam poplar, <em>Populus balsamifera</em> L.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3143</span>–<span class="cit-lpage">3152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss121%26rft_id%253Dinfo%253Apmid%252F22513286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMC8zMTQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.32" data-doi="10.2307/2409523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lagercrantz</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryman</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Genetic-structure of Norway spruce (<em>Picea abies</em>)—concordance of morphological and allozymic variation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D44%26rft.spage%253D38%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2409523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2409523&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.33" data-doi="10.1534/g3.112.005462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gyllenstrand</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Distribution of long-range linkage disequilibrium and Tajima’s D values in Scandinavian populations of Norway spruce <em>(Picea abies)</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.005462%26rft_id%253Dinfo%253Apmid%252F23550126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvNS83OTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTk3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.34" data-doi="10.1086/286119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Latta</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite> <span class="cit-pub-date">1998</span> <span class="cit-article-title">Differentiation of allelic frequencies at quantitative trait loci affecting locally adaptive traits.</span> <abbr class="cit-jnl-abbrev">American Naturalist</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">292</span>. Available at: <a href="http://doi.org/10.1086/286119">http://doi.org/10.1086/286119</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BNaturalist%26rft.volume%253D151%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1086%252F286119%26rft_id%253Dinfo%253Apmid%252F18811359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/286119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18811359&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072128600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.35" data-doi="10.18637/jss.v025.i14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Josse</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Husson</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">FactoMineR: an R package for multivariate analysis.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D25%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v025.i14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v025.i14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.36" data-doi="10.1111/j.1365-294X.2012.05479.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Corre</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genetic differentiation at quantitative trait loci under local adaptation.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1548</span>–<span class="cit-lpage">1566</span>. Available at: <a href="http://doi.org/10.1111/j.1365-294X.2012.05479.x">http://doi.org/10.1111/j.1365-294X.2012.05479.x</a></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLE%2BCorre%26rft.auinit1%253DV.%26rft.volume%253D21%26rft.issue%253D7%26rft.spage%253D1548%26rft.epage%253D1566%26rft.atitle%253DThe%2Bgenetic%2Bdifferentiation%2Bat%2Bquantitative%2Btrait%2Bloci%2Bunder%2Blocal%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05479.x%26rft_id%253Dinfo%253Apmid%252F22332667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05479.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22332667&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301933600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.37" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.38" data-doi="10.1111/j.1365-294X.2011.05308.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakobsson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sjödin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Joint analysis of demography and selection in population genetics: where do we stand and where could we go?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">28</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLi%26rft.auinit1%253DJ.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D28%26rft.epage%253D44%26rft.atitle%253DJoint%2Banalysis%2Bof%2Bdemography%2Band%2Bselection%2Bin%2Bpopulation%2Bgenetics%253A%2Bwhere%2Bdo%2Bwe%2Bstand%2Band%2Bwhere%2Bcould%2Bwe%2Bgo%253F%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05308.x%26rft_id%253Dinfo%253Apmid%252F21999307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05308.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21999307&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.39" data-doi="10.1016/j.ympev.2013.07.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lockwood</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aleksic</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A new phylogeny for the genus <em>Picea</em> from plastid, mitochondrial, and nuclear sequences.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D69%26rft.spage%253D717%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2013.07.004%26rft_id%253Dinfo%253Apmid%252F23871916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2013.07.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23871916&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.40" data-doi="10.1093/gbe/evu085"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bost</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maruki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Population-genetic inference from pooled-sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1210</span>–<span class="cit-lpage">1218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu085%26rft_id%253Dinfo%253Apmid%252F24787620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjg6IjYvNS8xMjEwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.41" data-doi="10.1093/molbev/mss187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maruki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Purifying selection modulates the estimates of population differentiation and confounds genome-wide comparisons across single-nucleotide polymorphisms.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">3617</span>–<span class="cit-lpage">3623</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss187%26rft_id%253Dinfo%253Apmid%252F22826460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xMi8zNjE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.42" data-doi="10.1073/pnas.76.10.5269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Mathematical-model for studying genetic variation in terms of restriction endonucleases.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">5269</span>–<span class="cit-lpage">5273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D76%26rft.issue%253D10%26rft.spage%253D5269%26rft.epage%253D5273%26rft.atitle%253DMathematical%2Bmodel%2Bfor%2Bstudying%2Bgenetic%2Bvariation%2Bin%2Bterms%2Bof%2Brestriction%2Bendonucleases.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.76.10.5269%26rft_id%253Dinfo%253Apmid%252F291943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzYvMTAvNTI2OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.43" data-doi="10.1007/s00425-006-0374-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nilsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlsbecker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundas-Larsson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vahala</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">APETALA2 like genes from <em>Picea abies</em> show functional similarities to their Arabidopsis homologues.</span> <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">225</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlanta%26rft.stitle%253DPlanta%26rft.aulast%253DNilsson%26rft.auinit1%253DL.%26rft.volume%253D225%26rft.issue%253D3%26rft.spage%253D589%26rft.epage%253D602%26rft.atitle%253DAPETALA2%2Blike%2Bgenes%2Bfrom%2BPicea%2Babies%2Bshow%2Bfunctional%2Bsimilarities%2Bto%2Btheir%2BArabidopsis%2Bhomologues.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00425-006-0374-1%26rft_id%253Dinfo%253Apmid%252F16953432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00425-006-0374-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16953432&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243964000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.44" data-doi="10.1038/nature12211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nystedt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wetterbom</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuccolo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Norway spruce genome sequence and conifer genome evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D579%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12211%26rft_id%253Dinfo%253Apmid%252F23698360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23698360&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319556100035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.45" data-doi="10.1111/mec.12067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levsen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soolanayakanahally</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">W. R</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The adaptive potential of <em>Populus balsamifera</em> L. to phenology requirements in a warmer global climate.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1214</span>–<span class="cit-lpage">1230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D1214%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12067&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.46" data-doi="10.1146/annurev.ecolsys.38.091206.095646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyhajarvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knurr</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene flow and local adaptation in trees.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D38%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.38.091206.095646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.38.091206.095646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000252359800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.47" data-doi="10.1214/aos/1074290335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The positive false discovery rate: a Bayesian interpretation and the q-value.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2013</span>–<span class="cit-lpage">2035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D31%26rft.spage%253D2013%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1074290335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1074290335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000188780400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical-method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.49" data-doi="10.1016/j.tree.2014.10.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tiffin</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Advances and limits of using population genetics to understand local adaptation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">680</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D29%26rft.spage%253D673%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2014.10.004%26rft_id%253Dinfo%253Apmid%252F25454508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2014.10.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25454508&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.50" data-doi="10.1038/hdy.2009.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tollefsrud</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstebo</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brochmann</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">Ø.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skrøppa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Combined analysis of nuclear and mitochondrial markers provide new insight into the genetic structure of North European <em>Picea abies</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DTollefsrud%26rft.auinit1%253DM.%2BM.%26rft.volume%253D102%26rft.issue%253D6%26rft.spage%253D549%26rft.epage%253D562%26rft.atitle%253DCombined%2Banalysis%2Bof%2Bnuclear%2Band%2Bmitochondrial%2Bmarkers%2Bprovide%2Bnew%2Binsight%2Binto%2Bthe%2Bgenetic%2Bstructure%2Bof%2BNorth%2BEuropean%2BPicea%2Babies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.16%26rft_id%253Dinfo%253Apmid%252F19259114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19259114&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266199100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsuda</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stocks</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Källman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sønstebø</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parducci</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The extent and meaning of hybridization and introgression between Siberian spruce <em>(Picea obovata)</em> and Norway spruce <em>(P. abies):</em> cryptic refugia as stepping stones to the west?</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> (<span class="cit-comment">in press</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1931</span> <span class="cit-article-title">Evolution in Mendelian populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D159%26rft.atitle%253DEVOLUTION%2BIN%2BMENDELIAN%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F17246615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiIxNi8yLzk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNi83LzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.53" data-doi="10.1093/bioinformatics/btq057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nacu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq057%26rft_id%253Dinfo%253Apmid%252F20147302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi83Lzg3MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzYvNy8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.54" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzE4NTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy82LzcvMTk3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.7.1979.55" data-doi="10.1089/10665270050081478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A greedy algorithm for aligning DNA sequences.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D7%26rft.issue%253D1-2%26rft.spage%253D203%26rft.epage%253D214%26rft.atitle%253DA%2Bgreedy%2Balgorithm%2Bfor%2Baligning%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1089%252F10665270050081478%26rft_id%253Dinfo%253Apmid%252F10890397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/10665270050081478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10890397&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F7%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087833300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/7/1979.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/7/1969" title="Whole-Genome Sequencing Reveals Genetic Variation in the Asian House Rat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/7/1991" title="Identification of Novel Regulators of the JAK/STAT Signaling Pathway that Control Border Cell Migration in the Drosophila Ovary" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/7"><span>Volume 6 Issue 7, July 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/7.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/7.cover-source.jpg?itok=E84OOyGI" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (7)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/7/1979" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-R2EC9Wqyo1tfNmEWlWSNlLdKtHCAC7ikSjDjhPvu7n0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/7/1979" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/7/1979" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/7/1979" class="active" data-icon-position="" data-hide-link-title="0">Identifying Genetic Signatures of Natural Selection Using Pooled Population Sequencing in Picea abies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460427" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-sC7K7yI6AiBrKC813dqMGu21nRfaIIlzgVoiNm-uMhA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1979.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/7/1979" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-5tmpVYNdeJ6I7Wpi5dvLS3u4Cib4x9GBi6OOeOVCY5M" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/7/1979" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/7/1979.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460427" id="node460427--2" data-pisa="ggg;6/7/1979" data-pisa-master="ggg;g3.116.028753" data-apath="/ggg/6/7/1979.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identifying Genetic Signatures of Natural Selection Using Pooled Population Sequencing in <em>Picea abies</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jun Chen</span>, <span class="highwire-citation-author" data-delta="1">Thomas Källman</span>, <span class="highwire-citation-author" data-delta="2">Xiao-Fei Ma</span>, <span class="highwire-citation-author" data-delta="3">Giusi Zaina</span>, <span class="highwire-citation-author" data-delta="4">Michele Morgante</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1699-9042" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martin Lascoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1979-1989;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028753 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460427--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jun Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jun%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJun%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Källman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BK%C3%A4llman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4llman%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BK%25C3%25A4llman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiao-Fei Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiao-Fei%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiao-Fei%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Giusi Zaina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agricultural, Food, Environmental and Animal Sciences, University of Udine, 33100 Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Giusi%2BZaina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zaina%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGiusi%2BZaina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michele Morgante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agricultural, Food, Environmental and Animal Sciences, University of Udine, 33100 Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michele%2BMorgante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgante%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichele%2BMorgante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Martin Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1699-9042" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin Lascoux</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/7/1979" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/7/1979&amp;title=Identifying%20Genetic%20Signatures%20of%20Natural%20Selection%20Using%20Pooled%20Population%20Sequencing%20in%20Picea%20abies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/7/1979&amp;title=Identifying%20Genetic%20Signatures%20of%20Natural%20Selection%20Using%20Pooled%20Population%20Sequencing%20in%20Picea%20abies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/7/1979&amp;title=Identifying%20Genetic%20Signatures%20of%20Natural%20Selection%20Using%20Pooled%20Population%20Sequencing%20in%20Picea%20abies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/7/1979&amp;text=Identifying%20Genetic%20Signatures%20of%20Natural%20Selection%20Using%20Pooled%20Population%20Sequencing%20in%20Picea%20abies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/7/1979&amp;title=Identifying%20Genetic%20Signatures%20of%20Natural%20Selection%20Using%20Pooled%20Population%20Sequencing%20in%20Picea%20abies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/7/1979&amp;t=Identifying%20Genetic%20Signatures%20of%20Natural%20Selection%20Using%20Pooled%20Population%20Sequencing%20in%20Picea%20abies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/7/1979&amp;title=Identifying%20Genetic%20Signatures%20of%20Natural%20Selection%20Using%20Pooled%20Population%20Sequencing%20in%20Picea%20abies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/7/1979&amp;title=Identifying%20Genetic%20Signatures%20of%20Natural%20Selection%20Using%20Pooled%20Population%20Sequencing%20in%20Picea%20abies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460427/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460427" id="node460427--3" data-pisa="ggg;6/7/1979" data-pisa-master="ggg;g3.116.028753" data-apath="/ggg/6/7/1979.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Identifying Genetic Signatures of Natural Selection Using Pooled Population Sequencing in <em>Picea abies</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jun Chen</span>, <span class="highwire-citation-author" data-delta="1">Thomas Källman</span>, <span class="highwire-citation-author" data-delta="2">Xiao-Fei Ma</span>, <span class="highwire-citation-author" data-delta="3">Giusi Zaina</span>, <span class="highwire-citation-author" data-delta="4">Michele Morgante</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1699-9042" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Martin Lascoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">July 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 7 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1979-1989;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.028753 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460427--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jun Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jun%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJun%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Thomas Källman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BK%C3%A4llman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=K%C3%A4llman%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BK%25C3%25A4llman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Xiao-Fei Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiao-Fei%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiao-Fei%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Giusi Zaina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agricultural, Food, Environmental and Animal Sciences, University of Udine, 33100 Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Giusi%2BZaina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zaina%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGiusi%2BZaina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Michele Morgante </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agricultural, Food, Environmental and Animal Sciences, University of Udine, 33100 Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michele%2BMorgante%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgante%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichele%2BMorgante%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Martin Lascoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Genetics, Evolutionary Biology Centre and Science for Life Laboratory, Uppsala University, 75236 Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BLascoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lascoux%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMartin%2BLascoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1699-9042" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Martin Lascoux</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1979&amp;rft.epage=1989&amp;rft.atitle=Identifying%20Genetic%20Signatures%20of%20Natural%20Selection%20Using%20Pooled%20Population%20Sequencing%20in%20Picea%20abies&amp;rft.volume=6&amp;rft.issue=7&amp;rft.date=2016-07-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chen%2C+Jun&amp;rft.au=K%C3%A4llman%2C+Thomas&amp;rft.au=Ma%2C+Xiao-Fei&amp;rft.au=Zaina%2C+Giusi&amp;rft.au=Morgante%2C+Michele&amp;rft.au=Lascoux%2C+Martin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460427/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460427/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460427/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460427/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460427/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460427/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460427/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460427/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460427/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460427/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460427/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460427/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/7/1979" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/7/1979.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/7/1979.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/7/1979.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/7/1979.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
