<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/1/183" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/1/183.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/1/183.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/1/183.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/1/183.atom" />
<meta name="HW.pisa" content="ggg;6/1/183" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Gene Expression and Chromatin Modifications Associated with Maize Centromeres" />
<meta name="DC.Identifier" content="10.1534/g3.115.022764" />
<meta name="DC.Date" content="2016-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Zhao et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Centromeres are defined by the presence of CENH3, a variant of histone H3. Centromeres in most plant species contain exclusively highly repetitive DNA sequences, which has hindered research on structure and function of centromeric chromatin. Several maize centromeres have been nearly completely sequenced, providing a sequence-based platform for genomic and epigenomic research of plant centromeres. Here we report a high resolution map of CENH3 nucleosomes in the maize genome. Although CENH3 nucleosomes are spaced ∼190 bp on average, CENH3 nucleosomes that occupied CentC , a 156-bp centromeric satellite repeat, showed clear positioning aligning with CentC monomers. Maize centromeres contain alternating CENH3-enriched and CENH3-depleted subdomains, which account for 87% and 13% of the centromeres, respectively. A number of annotated genes were identified in the centromeres, including 11 active genes that were located exclusively in CENH3-depleted subdomains. The euchromatic histone modification marks, including H3K4me3, H3K36me3 and H3K9ac, detected in maize centromeres were associated mainly with the active genes. Interestingly, maize centromeres also have lower levels of the heterochromatin histone modification mark H3K27me2 relative to pericentromeric regions. We conclude that neither H3K27me2 nor the three euchromatic histone modifications are likely to serve as functionally important epigenetic marks of centromere identity in maize." />
<meta name="DC.Contributor" content="Hainan Zhao" />
<meta name="DC.Contributor" content="Xiaobiao Zhu" />
<meta name="DC.Contributor" content="Kai Wang" />
<meta name="DC.Contributor" content="Jonathan I. Gent" />
<meta name="DC.Contributor" content="Wenli Zhang" />
<meta name="DC.Contributor" content="R. Kelly Dawe" />
<meta name="DC.Contributor" content="Jiming Jiang" />
<meta name="article:published_time" content="2016-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Gene Expression and Chromatin Modifications Associated with Maize Centromeres" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Centromeres are defined by the presence of CENH3, a variant of histone H3. Centromeres in most plant species contain exclusively highly repetitive DNA sequences, which has hindered research on structure and function of centromeric chromatin. Several maize centromeres have been nearly completely sequenced, providing a sequence-based platform for genomic and epigenomic research of plant centromeres. Here we report a high resolution map of CENH3 nucleosomes in the maize genome. Although CENH3 nucleosomes are spaced ∼190&amp;nbsp;bp on average, CENH3 nucleosomes that occupied &lt;i&gt;CentC&lt;/i&gt;, a 156-bp centromeric satellite repeat, showed clear positioning aligning with &lt;i&gt;CentC&lt;/i&gt; monomers. Maize centromeres contain alternating CENH3-enriched and CENH3-depleted subdomains, which account for 87% and 13% of the centromeres, respectively. A number of annotated genes were identified in the centromeres, including 11 active genes that were located exclusively in CENH3-depleted subdomains. The euchromatic histone modification marks, including H3K4me3, H3K36me3 and H3K9ac, detected in maize centromeres were associated mainly with the active genes. Interestingly, maize centromeres also have lower levels of the heterochromatin histone modification mark H3K27me2 relative to pericentromeric regions. We conclude that neither H3K27me2 nor the three euchromatic histone modifications are likely to serve as functionally important epigenetic marks of centromere identity in maize.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="ggg;6/1/183" />
<meta name="citation_id" content="6/1/183" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/1/183" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/1/183.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/1/183.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/1/183.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.022764" />
<meta name="citation_pmid" content="26564952" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="183" />
<meta name="citation_lastpage" content="192" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hainan Zhao" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Xiaobiao Zhu" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Kai Wang" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="Jonathan I. Gent" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Wenli Zhang" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author" content="R. Kelly Dawe" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author" content="Jiming Jiang" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706" />
<meta name="citation_author_email" content="jjiang1@wisc.edu" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=P. S. Albert;citation_author=Z. Gao;citation_author=T. V. Danilova;citation_author=J. A. Birchler;citation_title=Diversity of chromosomal karyotypes in maize and its relatives.;citation_pages=6-16;citation_volume=129;citation_year=2010;citation_issue=1-3;citation_pmid=20551613;citation_doi=10.1159/000314342" />
<meta name="citation_reference" content="citation_journal_title=Developmental cell;citation_journal_abbrev=Dev Cell;citation_author=M. D. Blower;citation_author=B. A. Sullivan;citation_author=G. H. Karpen;citation_title=Conserved organization of centromeric chromatin in flies and humans.;citation_pages=319-330;citation_volume=2;citation_year=2002;citation_issue=3;citation_pmid=11879637;citation_doi=10.1016/S1534-5807(02)00135-1" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. P. Cam;citation_author=T. Sugiyama;citation_author=E. S. Chen;citation_author=X. Chen;citation_author=P. C. FitzGerald;citation_title=Comprehensive analysis of heterochromatin- and RNAi-mediated epigenetic control of the fission yeast genome.;citation_pages=809-819;citation_volume=37;citation_year=2005;citation_issue=8;citation_pmid=15976807;citation_doi=10.1038/ng1602" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=Y. Dalal;citation_author=H. Wang;citation_author=S. Lindsay;citation_author=S. Henikoff;citation_title=Tetrameric structure of centromeric nucleosomes in interphase Drosophila cells.;citation_pages=1798-1809;citation_volume=5;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. K. Dimitriadis;citation_author=C. Weber;citation_author=R. K. Gill;citation_author=S. Diekmann;citation_author=Y. Dalal;citation_title=Tetrameric organization of vertebrate centromeric nucleosomes.;citation_volume=107;citation_year=2010;citation_pmid=21059934;citation_doi=10.1073/pnas.1009563107" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=H. R. Drew;citation_title=Structural specificities of five commonly used DNA nucleases.;citation_pages=535-557;citation_volume=176;citation_year=1984;citation_issue=4;citation_pmid=6086935;citation_doi=10.1016/0022-2836(84)90176-1" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=O. Flores;citation_author=M. Orozco;citation_title=nucleR: a package for non-parametric nucleosome positioning.;citation_volume=27;citation_year=2011;citation_pmid=21653521;citation_doi=10.1093/bioinformatics/btr345" />
<meta name="citation_reference" content="citation_journal_title=Dev. Cell;citation_author=T. Fukagawa;citation_author=W. C. Earnshaw;citation_title=The centromere: chromatin foundation for the kinetochore machinery.;citation_pages=497-509;citation_volume=30;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Gassmann;citation_author=A. Rechtsteiner;citation_author=K. W. Yuen;citation_author=A. Muroyama;citation_author=T. Egelhofer;citation_title=An inverse relationship to germline transcription defines centromeric chromatin in C. elegans.;citation_pages=534-537;citation_volume=484;citation_year=2012;citation_issue=7395;citation_pmid=22495302;citation_doi=10.1038/nature10973" />
<meta name="citation_reference" content="citation_journal_abbrev=Epigenetics Chromatin;citation_author=J. I. Gent;citation_author=K. L. Schneider;citation_author=C. N. Topp;citation_author=C. Rodriguez;citation_author=G. G. Presting;citation_title=Distinct influences of tandem repeats and retrotransposons on CENH3 nucleosome positioning.;citation_pages=3-3;citation_volume=4;citation_year=2011;citation_pmid=21352520;citation_doi=10.1186/1756-8935-4-3" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. I. Gent;citation_author=Y. Z. Dong;citation_author=J. M. Jiang;citation_author=R. K. Dawe;citation_title=Strong epigenetic similarity between maize centromeric and pericentromeric regions at the level of small RNAs, DNA methylation and H3 chromatin modifications.;citation_volume=40;citation_year=2012;citation_pmid=22058126;citation_doi=10.1093/nar/gkr862" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=J. I. Gent;citation_author=T. F. Madzima;citation_author=R. Bader;citation_author=M. R. Kent;citation_author=X. Y. Zhang;citation_title=Accessible DNA and relative depletion of H3K9me2 at maize loci undergoing RNA-directed DNA methylation.;citation_volume=26;citation_year=2014;citation_pmid=25465407;citation_doi=10.1105/tpc.114.130427" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. I. Gent;citation_author=K. Wang;citation_author=J. M. Jiang;citation_author=R. K. Dawe;citation_title=Stable patterns of CENH3 occupancy through maize lineages containing genetically similar centromeres.;citation_volume=200;citation_year=2015;citation_pmid=26063660;citation_doi=10.1534/genetics.115.177360" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=Z. Y. Gong;citation_author=Y. F. Wu;citation_author=A. Koblizkova;citation_author=G. A. Torres;citation_author=K. Wang;citation_title=Repeatless and repeat-based centromeres in potato: implications for centromere evolution.;citation_volume=24;citation_year=2012;citation_pmid=22968715;citation_doi=10.1105/tpc.112.100511" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=M. Guenatri;citation_author=D. Bailly;citation_author=C. Maison;citation_author=G. Almouzni;citation_title=Mouse centric and pericentric satellite repeats form distinct functional heterochromatin.;citation_volume=166;citation_year=2004;citation_pmid=15302854;citation_doi=10.1083/jcb.200403109" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=D. Hasson;citation_author=T. Panchenko;citation_author=K. J. Salimian;citation_author=M. U. Salman;citation_author=N. Sekulic;citation_title=The octamer is the major form of CENP-A nucleosomes at human centromeres.;citation_pages=687-695;citation_volume=20;citation_year=2013;citation_pmid=23644596;citation_doi=10.1038/nsmb.2562" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. G. Henikoff;citation_author=J. A. Belsky;citation_author=K. Krassovsky;citation_author=D. M. MacAlpine;citation_author=S. Henikoff;citation_title=Epigenome characterization at single base-pair resolution.;citation_volume=108;citation_year=2011;citation_pmid=22025700;citation_doi=10.1073/pnas.1110731108" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=J. G. Henikoff;citation_author=J. Thakur;citation_author=S. Kasinathan;citation_author=S. Henikoff;citation_title=A unique chromatin complex occupies young {alpha}-satellite arrays of human centromeres;citation_pages=e1400234-e1400234;citation_volume=1;citation_year=2015;citation_issue=1;citation_doi=10.1126/sciadv.1400234" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Henikoff;citation_author=J. G. Henikoff;citation_title=“Point” centromeres of Saccharomyces harbor single centromere-specific nucleosomes.;citation_volume=190;citation_year=2012;citation_pmid=22234856;citation_doi=10.1534/genetics.111.137711" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Henikoff;citation_author=K. Ahmad;citation_author=H. S. Malik;citation_title=The Centromere Paradox: Stable Inheritance with Rapidly Evolving DNA;citation_pages=1098-1102;citation_volume=293;citation_year=2001;citation_issue=5532;citation_pmid=11498581;citation_doi=10.1126/science.1062939" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=W. Horz;citation_author=W. Altenburger;citation_title=Sequence specific cleavage of DNA by micrococcal nuclease;citation_pages=2643-2658;citation_volume=9;citation_year=1981;citation_issue=12;citation_pmid=7279658;citation_doi=10.1093/nar/9.12.2643" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=X. Q. Huang;citation_author=A. Madan;citation_title=CAP3: A DNA Sequence Assembly Program;citation_pages=868-877;citation_volume=9;citation_year=1999;citation_issue=9;citation_pmid=10508846;citation_doi=10.1101/gr.9.9.868" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=W. W. Jin;citation_author=J. R. Melo;citation_author=K. Nagaki;citation_author=P. B. Talbert;citation_author=S. Henikoff;citation_title=Maize centromeres: organization and functional adaptation in the genetic background of oat.;citation_volume=16;citation_year=2004;citation_pmid=14973167;citation_doi=10.1105/tpc.018937" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=W. W. Jin;citation_author=J. C. Lamb;citation_author=W. L. Zhang;citation_author=B. Kolano;citation_author=J. A. Birchler;citation_title=Histone modifications associated with both A and B chromosomes of maize.;citation_pages=1203-1214;citation_volume=16;citation_year=2008;citation_issue=8;citation_pmid=18987983;citation_doi=10.1007/s10577-008-1269-8" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=O. Kohany;citation_author=A. J. Gentles;citation_author=L. Hankus;citation_author=J. Jurka;citation_title=Annotation, submission and screening of repetitive elements in Repbase: RepbaseSubmitter and Censor.;citation_pages=474-474;citation_volume=7;citation_year=2006;citation_pmid=17064419;citation_doi=10.1186/1471-2105-7-474" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Krassovsky;citation_author=J. G. Henikoff;citation_author=S. Henikoff;citation_title=Tripartite organization of centromeric chromatin in budding yeast.;citation_volume=109;citation_year=2012;citation_pmid=22184235;citation_doi=10.1073/pnas.1118898109" />
<meta name="citation_reference" content="citation_author=H. Li;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Magoc;citation_author=S. L. Salzberg;citation_title=FLASH: fast length adjustment of short reads to improve genome assemblies.;citation_volume=27;citation_year=2011;citation_pmid=21903629;citation_doi=10.1093/bioinformatics/btr507" />
<meta name="citation_reference" content="citation_author=M. Martin;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. A. Ribeiro;citation_author=P. Vagnarelli;citation_author=Y. M. Dong;citation_author=T. Hori;citation_author=B. F. McEwen;citation_title=A super-resolution map of the vertebrate kinetochore.;citation_volume=107;citation_year=2010;citation_pmid=20483991;citation_doi=10.1073/pnas.1002325107" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. S. Schnable;citation_author=D. Ware;citation_author=R. S. Fulton;citation_author=J. C. Stein;citation_author=F. S. Wei;citation_title=The B73 Maize Genome: Complexity, Diversity, and Dynamics;citation_pages=1112-1115;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965430;citation_doi=10.1126/science.1178534" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. H. Shi;citation_author=R. K. Dawe;citation_title=Partitioning of the maize epigenome by the number of methyl groups on histone H3 lysines 9 and 27.;citation_volume=173;citation_year=2006;citation_pmid=16624902;citation_doi=10.1534/genetics.106.056853" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Shivaraju;citation_author=J. R. Unruh;citation_author=B. D. Slaughter;citation_author=M. Mattingly;citation_author=J. Berman;citation_title=Cell-cycle-coupled structural oscillation of centromeric nucleosomes in yeast.;citation_pages=304-316;citation_volume=150;citation_year=2012;citation_issue=2;citation_pmid=22817893;citation_doi=10.1016/j.cell.2012.05.034" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=K. M. Smith;citation_author=J. M. Galazka;citation_author=P. A. Phatale;citation_author=L. R. Connolly;citation_author=M. Freitag;citation_title=Centromeres of filamentous fungi.;citation_pages=635-656;citation_volume=20;citation_year=2012;citation_issue=5;citation_pmid=22752455;citation_doi=10.1007/s10577-012-9290-3" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=B. A. Sullivan;citation_author=G. H. Karpen;citation_title=Centromeric chromatin exhibits a histone modification pattern that is distinct from both euchromatin and heterochromatin.;citation_pages=1076-1083;citation_volume=11;citation_year=2004;citation_issue=11;citation_pmid=15475964;citation_doi=10.1038/nsmb845" />
<meta name="citation_reference" content="citation_journal_title=The International journal of biochemistry;citation_journal_abbrev=Int J Biochem;citation_author=D. J. Telford;citation_author=B. W. Stewart;citation_title=Micrococcal nuclease: its specificity and use for chromatin analysis.;citation_pages=127-137;citation_volume=21;citation_year=1989;citation_issue=2;citation_pmid=2663558;citation_doi=10.1016/0020-711X(89)90100-6" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=TopHat: discovering splice junctions with RNA-Seq.;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=J. S. Verdaasdonk;citation_author=K. Bloom;citation_title=Centromeres: unique chromatin structures that drive chromosome segregation.;citation_pages=320-332;citation_volume=12;citation_year=2011;citation_issue=5;citation_pmid=21508988;citation_doi=10.1038/nrm3107" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. Wang;citation_author=Y. F. Wu;citation_author=W. L. Zhang;citation_author=R. K. Dawe;citation_author=J. M. Jiang;citation_title=Maize centromeres expand and adopt a uniform size in the genetic background of oat.;citation_volume=24;citation_year=2014;citation_pmid=24100079;citation_doi=10.1101/gr.160887.113" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=X. F. Wang;citation_author=A. A. Elling;citation_author=X. Y. Li;citation_author=N. Li;citation_author=Z. Y. Peng;citation_title=Genome-wide and organ-specific landscapes of epigenetic modifications and their relationships to mRNA and small RNA transcriptomes in maize.;citation_volume=21;citation_year=2009;citation_pmid=19376930;citation_doi=10.1105/tpc.109.065714" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. K. Wolfgruber;citation_author=A. Sharma;citation_author=K. L. Schneider;citation_author=P. S. Albert;citation_author=D. H. Koo;citation_title=Maize centromere structure and evolution: sequence analysis of centromeres 2 and 5 reveals dynamic Loci shaped primarily by retrotransposons.;citation_pages=e1000743-e1000743;citation_volume=5;citation_year=2009;citation_issue=11;citation_pmid=19956743;citation_doi=10.1371/journal.pgen.1000743" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=Y. F. Wu;citation_author=S. Kikuchi;citation_author=H. H. Yan;citation_author=W. L. Zhang;citation_author=H. Rosenbaum;citation_title=Euchromatic subdomains in rice centromeres are associated with genes and transcription.;citation_volume=23;citation_year=2011;citation_pmid=22080597;citation_doi=10.1105/tpc.111.090043" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=H. H. Yan;citation_author=W. W. Jin;citation_author=K. Nagaki;citation_author=S. L. Tian;citation_author=S. Ouyang;citation_title=Transcription and histone modifications in the recombination-free region spanning a rice centromere.;citation_volume=17;citation_year=2005;citation_pmid=16272428;citation_doi=10.1105/tpc.105.037945" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=H. H. Yan;citation_author=H. Ito;citation_author=K. Nobuta;citation_author=S. Ouyang;citation_author=W. W. Jin;citation_title=Genomic and genetic characterization of rice Cen3 reveals extensive transcription and evolutionary implications of a complex centromere.;citation_volume=18;citation_year=2006;citation_pmid=16877494;citation_doi=10.1105/tpc.106.043794" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=H. H. Yan;citation_author=P. B. Talbert;citation_author=H. R. Lee;citation_author=J. Jett;citation_author=S. Henikoff;citation_title=Intergenic locations of rice centromeric chromatin.;citation_pages=2563-2575;citation_volume=6;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Zang;citation_author=D. E. Schones;citation_author=C. Zeng;citation_author=K. Cui;citation_author=K. Zhao;citation_title=A clustering approach for identification of enriched domains from histone modification ChIP-Seq data.;citation_volume=25;citation_year=2009;citation_pmid=19505939;citation_doi=10.1093/bioinformatics/btp340" />
<meta name="citation_reference" content="citation_journal_title=Chromosome research :  an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology;citation_journal_abbrev=Chromosome Res;citation_author=H. Zhang;citation_author=R. K. Dawe;citation_title=Total centromere size and genome size are strongly correlated in ten grass species.;citation_pages=403-412;citation_volume=20;citation_year=2012;citation_issue=4;citation_pmid=22552915;citation_doi=10.1007/s10577-012-9284-1" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Zhang;citation_author=P. B. Talbert;citation_author=W. L. Zhang;citation_author=Y. F. Wu;citation_author=Z. J. Yang;citation_title=The CentO satellite confers translational and rotational phasing on cenH3 nucleosomes in rice centromeres.;citation_volume=110;citation_year=2013;citation_pmid=24191062;citation_doi=10.1073/pnas.1319548110" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. L. Zhang;citation_author=Y. F. Wu;citation_author=J. C. Schnable;citation_author=Z. X. Zeng;citation_author=M. Freeling;citation_title=High-resolution mapping of open chromatin in the rice genome.;citation_volume=22;citation_year=2012;citation_pmid=22110044;citation_doi=10.1101/gr.131342.111" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=Y. Zhang;citation_author=T. Liu;citation_author=C. A. Meyer;citation_author=J. Eeckhoute;citation_author=D. S. Johnson;citation_title=Model-based analysis of ChIP-Seq (MACS).;citation_pages=R137-R137;citation_volume=9;citation_year=2008;citation_issue=9;citation_pmid=18798982;citation_doi=10.1186/gb-2008-9-9-r137" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C. X. Zhong;citation_author=J. B. Marshall;citation_author=C. Topp;citation_author=R. Mroczek;citation_author=A. Kato;citation_title=Centromeric retroelements and satellites interact with maize kinetochore protein CENH3.;citation_volume=14;citation_year=2002;citation_pmid=12417704;citation_doi=10.1105/tpc.006106" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Gene Expression and Chromatin Modifications Associated with Maize Centromeres" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="twitter:description" content="Centromeres are defined by the presence of CENH3, a variant of histone H3. Centromeres in most plant species contain exclusively highly repetitive DNA sequences, which has hindered research on structure and function of centromeric chromatin. Several maize centromeres have been nearly completely sequenced, providing a sequence-based platform for genomic and epigenomic research of plant centromeres. Here we report a high resolution map of CENH3 nucleosomes in the maize genome. Although CENH3 nucleosomes are spaced ∼190 bp on average, CENH3 nucleosomes that occupied CentC , a 156-bp centromeric satellite repeat, showed clear positioning aligning with CentC monomers. Maize centromeres contain alternating CENH3-enriched and CENH3-depleted subdomains, which account for 87% and 13% of the centromeres, respectively. A number of annotated genes were identified in the centromeres, including 11 active genes that were located exclusively in CENH3-depleted subdomains. The euchromatic histone modification marks, including H3K4me3, H3K36me3 and H3K9ac, detected in maize centromeres were associated mainly with the active genes. Interestingly, maize centromeres also have lower levels of the heterochromatin histone modification mark H3K27me2 relative to pericentromeric regions. We conclude that neither H3K27me2 nor the three euchromatic histone modifications are likely to serve as functionally important epigenetic marks of centromere identity in maize." />
<meta name="og-title" property="og:title" content="Gene Expression and Chromatin Modifications Associated with Maize Centromeres" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/1/183" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Centromeres are defined by the presence of CENH3, a variant of histone H3. Centromeres in most plant species contain exclusively highly repetitive DNA sequences, which has hindered research on structure and function of centromeric chromatin. Several maize centromeres have been nearly completely sequenced, providing a sequence-based platform for genomic and epigenomic research of plant centromeres. Here we report a high resolution map of CENH3 nucleosomes in the maize genome. Although CENH3 nucleosomes are spaced ∼190 bp on average, CENH3 nucleosomes that occupied CentC , a 156-bp centromeric satellite repeat, showed clear positioning aligning with CentC monomers. Maize centromeres contain alternating CENH3-enriched and CENH3-depleted subdomains, which account for 87% and 13% of the centromeres, respectively. A number of annotated genes were identified in the centromeres, including 11 active genes that were located exclusively in CENH3-depleted subdomains. The euchromatic histone modification marks, including H3K4me3, H3K36me3 and H3K9ac, detected in maize centromeres were associated mainly with the active genes. Interestingly, maize centromeres also have lower levels of the heterochromatin histone modification mark H3K27me2 relative to pericentromeric regions. We conclude that neither H3K27me2 nor the three euchromatic histone modifications are likely to serve as functionally important epigenetic marks of centromere identity in maize." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
    <title>Gene Expression and Chromatin Modifications Associated with Maize Centromeres | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__34Arr8QetTzezDWW4Etyv7gbgH-x-ZTNo3yO15Q-gHU__Mk-lyyG5WoK4SRy56eUq5eomxN7OGXjpDv-Ob1a1Q1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"EDnHoytGROouY_bp17V2IzH2na3fk-v02WhWwvkquHA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460246","apath":"\/ggg\/6\/1\/183.atom","pisa":"ggg;6\/1\/183","ac":{"\/ggg\/6\/1\/183.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/1\/183.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/1\/183"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/1\/183"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/1\/183.abstract":true,"\/content\/6\/1\/183":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fF7DD5K1xeyus4ybyffH+N49ZIphXVvt7eacJB5azn87z8D31S4Jmj54JgFoOxfTuzK7R6Xt7+Vbs3wI0AF6Zlvcz6+GmcIWb7PNgXX+KYJ\\\\\\\/Dy9nY0R\\\\\\\/RclH0nWMUEFTFiMGBtp7ZXCQhDLgMHZWpwZK2ktpE6X4Ler3IYK\\\\\\\/YQ+PjqrCkUzrF6KHbnOVxGeiKgsfokMeNBkzJ54blnPyOU1CofOHKeLF8kgTlERfrhrqpsNnGVFgrTeF1LZtaX8OmbLkwxrG0SYKiPB+MX3FLAEWxuv5xfHzUNAvFuSLy2aJ5l3u3I9pBB\\\\\\\/DmHRlvwauhcq0p11PvHsbiL9dWpjFBlKf5QhgyVwidH+AoHCpU5IQTUR+nLR9J8uMuPSdaLNWEIECn0m7w5cvNZ+doC02+IaGAwy2wiI60U91JEb5hRLELpXRd+fRfkrKDF8utiDeCpLumJcPggmL3WH9gSD3bNkSon0QTJwtW6uc7x8sI0emYw4i0\\\\\\\/xwYUBkc7hyr51WGPHhVP3QO3QVeDxc9Bfspcc2ffV5K0+SgbBvHa\\\\\\\/3jcHbpx9SxkeCr5u\\\\\\\/ztWHROjvYNozGgZoOt95C6onn\\\\\\\/c0fjh87ZqRi+p+mbP5+oZaebhFnc6ZsVEFodXP9ait2v8SneEttUAwItv6Aw9X6PI1CpvOv8H7ogpaOb5fWHbOorSMqVBS6CMTVixxYoDfynH6jZN\\\\\\\/Jmd9uoRYEhNNZZeBC0kAOWEqweFGHZUUiNR+sYZmStif\\\\\\\/cI6WEPfODc6AVs1mPdU+Juoc7hOsZuRVU5Aza\\\\\\\/Q4SPDzK4jcFtutDXqyybZtZyWrStfxMxwbOuU4W5oPPAc73F3meppMAP+ElNUZQxpVayrYi\\\\\\\/k1p6u7uYsGnuVfi2aNg2B7OFu2e9RouQ43NyXFVRn3u6rWNt+lE+XnUG\\\\\\\/HbQfkfgHYaeIyL7QeoQ9UIT641P6POSAnQLSLGyVIBSZSzc0b000PN+bogmImIKho+dhUA9L4fRwiv+rWF9IMq4KgRweBoKqHSv\\\\\\\/zymb6k+COp6WI0bfjFhF0Qs60fnci99aFKlGcFvzy9Y3ISsni8eyiQR6i3b5FjDQyy+6q0HKxgXBxAOfBfw\\\\\\\/wIBIcp\\\\\\\/Ellf9oNkyiz1EqRlR6IKBXoPLft4eL+f\\\\\\\/0swA4f2jMR\\\\\\\/CQ0QOvpIDpgcXDpqR0wuN3uYROcKfDKqeKNYm+SAl2r9bnNtP7gxwsgOhppU6CqADhQwrIXTbpyD4WpjuKmIRUaI8b3dVMMfoTmhK9qAZ1TVZ\\\\\\\/KgNvZmuJ7i9invt\\\u0022,\\\u0022iv\\\u0022:\\\u0022F6LRulijjkZ+BfD1f1QffA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224e137effe0829015f8c76c4671fbc3b46621b02f8e685063eb92a7cfd0c16655\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uxpOm8JPnFcjheqP0tUoFPNgX5iG\\\\\\\/TurE+0JDe0+\\\\\\\/el7fwaHX22CdpXzkXIiQk+G42cSZBbgN0StjFb9XdC9anI0Iaohc\\\\\\\/PpOgGnInSseD2tjPv2tXwfmnk9aDY9g7yPRmOZGVPB+a59XvOX2N8F1TtiPzDA9Bts+c8rmVdTDP8kfHxNEC\\\\\\\/2kghxhKBBvtWdZoj+DujfW49L+dG+5O1oGukvwU5TvJQk+X1M23FsGA3s10z2jXylSl0gzGf\\\\\\\/0fWWzyof4bmE0yntv\\\\\\\/gEbQJCdyuDpXCjWrII8PLDD4Zqk+vHpDJBcimFgc7BXk1IiJmwa0Xjc453EkAdTPYiC2Zvbe\\\\\\\/n8ck+fqPnXNRiFufj\\\\\\\/Ule51GOKtxwY2q4j+ZuvHKnFI\\\\\\\/381\\\\\\\/zK429Cpe004MRzhGfmuXQneMVUYsgqGiQMY9R+5smrwFP2Gwud\\\\\\\/FtCB5Gyjvlq6NqHVZRBGzRjf4RQFnXP9MgWP4uAEZ03eBN8nMbZKN\\\\\\\/Y7kLEkJxq4\\\\\\\/HaLvYkAifD1TwhFpERoDTxR3UGipBeFB6VQ2W6JPXCsR6KuwLc7QOQ55KundGRkOFAlDJz9IKRXa6RTzx9F+DOS2u2dTEts9cMWPDaLj7OnXuXskOj16txW05x2t7Z9FnrGefwGrw+Oq4rdXKWMhLNa+VbBT8qGtOaO4Cinwqo9IEcJ8lVsy8Go4DjFz+a9b2rDmEhsaMYfMQZq4\\\\\\\/2ICc6t81AwGsXBu4gJHV6Yy\\\\\\\/xBwRKBlgSh9i4msI4tuC9ta3Wzpi8Su7NLU8XW71EYRo0tMtHM72PdvjMGmHMV40s09A4e\\\\\\\/7pASc98GFBjGEWqjEqHi4Eji4IBS1\\\\\\\/YkxyQW7HpHCXlPneTG8i0X5slxKNx1iSpdGW5DWiPyJOtiNPEIH4Dl0R9mGhQzOqVYZRA5Sys1jkSeBspqcsGaPH8G8EIjJOjTr8YGugyx769Fkfdo9\\\\\\\/mFfslHjsTOzWFEqGVNG81XXvJUM\\\\\\\/ysX8yFoCTh40yys4agEHlgDkY6+Meswtrk26xSdgFRyaYASpmO4AyB3SuZE+sGqWxV0B9\\\\\\\/gXkHv0\\\\\\\/WvZ1SmV23qf7iEH1UVBwh\\\\\\\/r7fn9Jimh3d1XLULG0JKDFYY7K6jvJcI0J4\\\\\\\/NGXAEGOq4M9BYs7xBGpW5nLvs0mzbRRtl3t8hWLH0cRSc+vXMjsrAlyebXJv1HV\\\\\\\/YSWABo8eGOYLjY9V7yXFpKL1+mWd1hrAgkpHOkpJ94EgM599tD1hfugTXlg3ESkNqIog+tQw5OjxVqN9VyJzcPLJ5CzIp1S7jKhLEym4B2hCv7vVEcrmW6EbD7f+D\\\\\\\/dpG9HBc4TbjG5rZsdExkej0wsOOPYmpI7bneI7mwSbxllL2JzXKyycOU57LQIwRGULX0\\\\\\\/AtpNrxINYNhyQMBratMvrviP0ZzTv1X3TXP5C0aDIKPc2F\\\\\\\/znOYcVtcnSXDWMC7jIpJvecYRVWaLpu1YS67ENwiQ86bdzogO5DrFD6bpBIX\\\\\\\/Od5v1JKXkQSh8xBn2uFXnIwhnlU36NY\\\\\\\/owFpy+fAEaZJqUkqQs4pLn2xO3IxIjRuHNrr7pOPg8WC3O4lypaMZbGLhPHpzWx7x7rPHWqQ5u0Hiah3EkEsSLfd9foMDplu9fB5IB1ELrzu7WX0Qi4xsNzNXBNM8QyTelP46DY766JDbI8Ky4RzO6us+AobNumheH4WCIOjXzK8VAl9DXnvgJyKN7NEx5hSTP2Io4GG+tX7gXZp+zDoOtC7tFKltJ5tgRIT79h8Fnsf5THmHAJbEOBHVgdwZjcpT5Ddt54nccp4z9+wPr3uhLKtR\\\\\\\/m4mfDr2fGYxipFktkDfZnMS5YdpatIEkA9e4wlBsX9MpMZQyFA+FbDUekP0R1Ywczz4HPTpbvusbQVd2wT+bwq58UjwIOysxiiNi6sGH4MSZ4\\\\\\\/hmnguZxlRV97rZWsDL9xqjQ8LJMfE10O50Gql4E7nLo3z1mw5cVPkP2+Dkre5W4TR0vlFjaI8TEtmt1EyZnNqlKyUfZvs28q61Cnff+i3m7dsfTsYj1m2vUYFsvw\\\\\\\/rqRdpICYmFmsAaRdjVtsCSzyrMcshm3UMfrn\\\\\\\/wqjvkctwpLWbDbV5SPN\\\\\\\/W3TxbLtPHy46CvrmaeYlqwk8nUsLzuBWreDsZpp7CmVUGTrumpzHZIjB7+sv1XV9mjLDqAyLQlI1\\\\\\\/0CseMVjv8mQYCX6JcncK\\\\\\\/mymxRvEN9vH+7NrnmtjbW1WN0loxLQ9jUB21uliwHTuxffyuQp3VyDTEUecNcAS+Eht0y5fdX68X05vZh59zS5WDsHuRwAcSI\\\\\\\/sl+6KsWzXqggfnS4P6BOccKJx4RlBBONzfUFZVR19vQPotBnkLU0gU9EY+G+JedTsFOKE06WPsjcqLW+iX8Sr\\\\\\\/Qoe43pAnKvlvmCxcEGzSuSaa0YjuoNEiAlAnVolWRum5nj\\\\\\\/3bd91FqRdrrEApnZbP7xqCru7fu0l40MsJMhSEsNL2E+jbSBP7IYQ0Kmn7QqXyn1xJcnCKYUumaaXxlwwyIVXUlgEQxBn92oBHD0YEtxWEuBBljD5WAvNALoeLiBJXamidyvNCjvhDKJlIeo26\\\\\\\/tGB3b0wn+Eg38cV5H7pb4PSS7i+U2aZDQdWC3RjSZlcyZel1vsGYwhPu+lGdqlgsZC3opLF+cQsdRvTqFt2hicW+feQtNqYPpC2JmFniDaA6rd9weGHn0P1tzDN38K0UcbRu5NqeR2uZ79nBQllADxjmhfKAGvamtpuojQnmxgPj7V2L0ebanFYNy55ECq+u++zPUU80Ocer9uD66RnFp3SJ+3EAzDsVgtAN2d4rrHd1YDbXRaSbZbIKDlW+Nsi2VzOObUUjibqFi7l8aniEsCwgH98O3YrAXu5IXojK9xsPcdD5BQwKezGgcO2oHDLb5pyk7p9cC+tOvb9J4bOXhYuNOwv5UfvKZusUxDdE8Bt72rW+\\\\\\\/NZWNyILbYNgadeVGc8CZBj5hpkdb3fs\\\\\\\/pieEdo53S46csOqO29RErCZ0+qPgwgD4OMi8Kdm2hVcbzWvHamG05iFlw3MgbErzo+NDr4n1\\\\\\\/gTMTKXpxyjtCYxl4m6rFr+qXYLuqYDxjmh14XYeSltTVEstiuaXonBig7uHOz+6mARXJHrI6aLUIxhqqXrPPHEM\\\\\\\/CGUi1ZDhLgiZ0QxNMQQIvGZ+33w50eo26FnB9q7lxh40gstFKm0yFiuoyFlaFbTz7pCdq8PTQ8N1rVCDe8LHAV\\\\\\\/\\\\\\\/CZANX7RcBNsw0ojBMa2wm1MtmDbi9MXFYF+BM+qBKKhqUTppCHcA\\\\\\\/pjUwxnLiMx+FQTyYB\\\\\\\/Y7qcPUoaR3WOPivRl9tBiL+A4LDvXX840FL6DDROYdpRFJb9Dii9g7S7mg06uExSrLRcLA5J\\\\\\\/wxIbGGZl\\\\\\\/9fhUNWvGhxBWBQFqanL1C9PcD0wmaZRIRJASwr7hFP5d15OYxCvbJADqHLvB3g1mSnbmKTEnYKPYfVNjkFjmQvaSqk7QWwhYHKoyaNgk8osVJWjeqJBDPwa0eSFA5fa9IabSg0qF8jU80BXCXdEBIXsv0\\\\\\\/WjkC0tdx8eeD35IqwnmDZlXTQQXqBe3bpsPlu\\\\\\\/z1njWq9pIllN0xza76vnToouV6S11BdBYVZ5TKeHBBI9Le3Ay\\\\\\\/bECWiPmqEi5\\\\\\\/3kRV\\\\\\\/ATh8KJD1Ia+\\\\\\\/NzhUlmZ9NUO32tozSTKy\\\\\\\/lAdu8r36Usibh2zqSc0c5T93+J1FPT2sffZ6gjmfMK8VmZj1K6S8qE+7oEPC3glPrDGciGOAM99POkfMtyKl+HZ5do8vZvETrWcFA9TV0BodtR3sOoQkI71BKHfSaevyaUv8W52FiIkyZsDStYsRzByGyHqBFuyxz1oF\\\\\\\/tKWeeCKgjIULulzizT34gSgUg8a3Vb+4O0PYBZ0cm7MhUhxEGOjyb+GpiZk4hfd+m\\\\\\\/rW9H0riF0OFG3hwKY4aeLsSbcp8U8McR\\\\\\\/285aqt4AqdOK92gfGoMfC7N0opqlGF\\\\\\\/1SjqjnaocAzKI6HNReNBTrNGtdVxH173vPhNld3OJPf2klFeAbkH8WZpxoTaJ2qZo6wfJrLn2ZvsrD\\\\\\\/Xn+wbFV347L+nJn\\\\\\\/9a+SqcBZWwVVGXaHvW8DVBfHuUclyLB9jNY5Et50\\\\\\\/xzMprxC1VnewCbbZpaEZsLlxtm4tEdJ2jB4KJ4RubAsx+PlxqBnILC2wnyYqG2fAlaPikCAxtA87jp3mykJFxz0jg0e+OblEQJnc+thnicnHgvqdueoxJJSFQOivYTBEB8\\\\\\\/290LJ8Xd0Q3nQ5FTqsHhKiUgNFq6rMdn2mjkIj\\\\\\\/dCCETH9kbUe\\\\\\\/Dmyhc6JFAisi5Xg9pgzVcxfo0i3qKkKj3xyd6JVbLE\\\\\\\/WFQdOH2nExU\\\\\\\/YwVkW3JL\\\\\\\/hTSSt7V4xPrdcamn2bAVY\\\\\\\/l7WSlBNUFkwhvLWzskX551DL\\\\\\\/V\\\\\\\/8wp5SBWbRfUMI4uI1F0W2Pi9uF7saYAIXA68kVebFJPG6GEu6kcF8NDxV25LXVC3cXL2OUJkbGxPnw\\\\\\\/alZkfqRhPt9pXgOWYG0+NO8W\\\\\\\/uMKUWA0eG4nt2adKnNmnvHhLssSeOh6pnIw\\\\\\\/rlXA7gKpHKjuMJYY3X\\\\\\\/YO61b3Yeh2lRW7R9E80QLY\\\\\\\/OO5uVtXR\\\\\\\/BmnhrkmSdMxzovNj1KfxPXaU6ZTwoUKrNf2sGM9tuD4Jbmpz2mKKRN7TTZoX67CtKqSk9eNP+qY5w3zmUgQaVPDALBdVnfnSfyEqYXC7k7nCjAQcC1zGn50tQq1KhQIDW0mCNRl2wONFvLVO7v9GEPjplaUlEY8ehvtWVD\\\\\\\/BF7Yl8BuZMTe8tOxBSvjSQVn9tpREk10DfSobVs9Rk3ojL1c3Y3dVS66GrL6evkJsf1pvognxEhz83VovIJaH6l96Gw0xW1fiXkM8X7WhzDbvm1hGylfZq1uCHZHU28tsBcReyxMJ5pPtqi5Owcbe86k4bu8rBZAv1modsE6hqFLP\\\\\\\/cMkpco4h69KPnDW1z5OrraCy02034CUqpOeKzM\\\\\\\/JVEBIhNz4fJwvGvS3CthqthqCgtiDQsR\\\\\\\/hYjp4gsjbXxzbD+dImoUZuFJEcJnNdxiH3LlsEK5a0wnkTEY64QojvhbYxmOGvi5Cvnntku0iOAFhszOtHbDd9Qi5wt+KkFNPnynifV3lYRV4irkNHf5frLjllOuB1j3Dqd2OwX515p7d2gUp30NhrIqxsA4hxoyXid6o6Qa+MoLeEWspLOq\\\\\\\/Qg0Hs\\\\\\\/WNoyGDzzviw3Q6HepiZxyyjG8CvI+kY25g\\\\\\\/KF8hQ3Kbi1dfz04V8\\\\\\\/E3nCXd2Q5cKqJy+bDHCm2YHLId83NXN\\\\\\\/vX72HjTFExo8Kx73tFesmv8wpU7czK47gfF5VM0tF82mdViOhWOfTFbhz09QV2W7UCfO2PiSV80MRQAzantESUVAwGm2v+k1F9qjaaWt0RVpeZWn11liq4jezVAr64kUdg8aJw2VJerOvRUU8BchVYUolONPY0lV6HQeWhlBHdgP8DrQxbTuKyxXd1NbWtoPgX5vacpMm8ezdr0O8AQWOeFH7BU8lfxOo\\\\\\\/IRvtyFFIUJPxRDtq9yi2inWYb\\\\\\\/z2Cu3rOS7u0mC0aVK57GArHmenRowCh33kkTF\\\\\\\/2RYC5r1kYuPZHSaF1cBbmUavoKXppLfUqpWGaI9T9YSs1iZBMiBTiiUnxltnpNOTcyxezNanegJ8zZ4XE9ZD+L27G4iwJ5WG+Sl25r14+CrfeChAtI0O82W2xvo8F6gQI9aNnLIRjd+oohtPF8SMtjZTIVCXWgOf8yNYDe2uNENBaXtzYuEqnQV5Lp\\\\\\\/hCYAGlHMfPgke3nScPDTzKl3gN2ANRu2xakp8ZF7fGDVEkpl0H5Yu0C2BURDtoJeRwT6glp3DcZgh2BYD+usf6a1JuMHEjM8RsZsVliI6H0xiy+sWgCRmuNeQ\\\\\\\/3ygt7iSwQ\\\\\\\/JfkGAJAYvqdTqrdfrkR3rrgw\\\\\\\/NV1lj0nxMsvOAJH5zLfUdXCUk\\\\\\\/LM4pke8744s6wgCHURZU3ZRlEj0RULVPQZ\\\\\\\/GvDv1iXZkNjXPsvK6APNo3RmZaRdpyOx7j9KKXp3d\\\\\\\/+5BPylDDslGXYzDrm8UzveoPtdVS4lsHh74CMmwaZf1yPbcTHhPa4f8uHPcSXo9VqupRHVTx6SHMVUBur4Sy1hOhYZSJ\\\\\\\/huSLWQJaByRJdVSfNpkrMktglnopKsFkzs\\\\\\\/c\\\\\\\/2XgG\\\\\\\/NUdXIdpfIyIx4qeXlW7Nq0V25gdU6HbgaN9dH9uzEP8FrN\\\\\\\/sjwKm2\\\\\\\/AF9oEjetOOyTCYqqUbnuFcfLRYk3Px+RHLbsQ6M17vdXOMS5cGa0c4toDGH8ItqqNgjMkkTQHd5PnOhZfeaiB0Cm9aJVoCtttQUPs1\\\\\\\/8FJlDLQjOm0UinQ4KTene0fZX56Zsn5elmF+WHYRJZL6DhbSgsPnN1La752IfJ9741qVnnkQe94AIub+G9Ou++avrFriFZejOZh9sKY4Q6\\\\\\\/t8SqYCxlW8mIMJHMHvQkefzvdLLpvpGS1PpDOZ6bCsisclI45B0r570LOM4R\\\\\\\/KdzElY\\\\\\\/5vTCA6hpAadjylmrCzDMHAUEPYaF7D6wuLbUYZSy0nI9poinQYBRDdUUP5dGibXquQHutCq2AqyGtjFBVrmfJPZqnUW+YVF6hg8bHxbYw1xvYJuJ0IpiR9pQBHw9G7W+H6yMLK2pFZ97Zpz8ilS1E3BOtBoTyb\\\\\\\/bzV1r7G9UJE5QsWO5d2Zyz6QvnnKwSrHDGxjtkloQpv3AXXXJ7aHOHO\\\\\\\/5KE2eHT3Qjf4ZHjWWumNFz4yZPc8qJ9ZreLTunGAAL2o8YtFO9ggHMTjaPf52IRQA41vifvQaMP4ZY11krGnBD4AjTGm4Vtt9XrLCOmFb3JO3z2OKAQ58gkr1rFLqWv8ku\\\\\\\/1VM2FGwoOhc\\\\\\\/vweUodUCFAYOEYlpaF9J6mV3m\\\\\\\/y2j4FV7N82FyoH07ZXnE5SD9Elg61+8BvAyFU3zJTc5eCRB4EYL09SXMz+jLdrCbKfzqP8py0kYJ2wNOO+ggfN\\\\\\\/I51AsZWdOlX1TDyQvy7QiYBgCxnUXJcaAUMdW6wkLv4m67fxjKR8XQygjl+ltDGz3vrstuJKdarkLq6FrtviVzoU4HKU6YYU9nLKlUeybfSkuGYkebSsB9LYJCAZK4JG7kK8J22lyJA2gkOmqK\\\\\\\/ouQH9MBB2aqsiwD1hVvwi6yj7q+xg\\\\\\\/FZUCKXhZOobSDR7RMD+0q8IZf1yXYfRJli2eDyD4iiLOoi0UOMWCiXG+9wZZrQmxrfT8GhzO6vhmF2F+fjpbUjbqqi9kC8z9hQKDcg0aOJT2ALsMs5HM9cxFOZ43bu7Vq77wIfoUx+Ro7gVFgAIavH87I4isQGt32pRUyVvNN9ag0TuB6e4g+sX\\\\\\\/WAtogSOVSAUH3qtCXa4sqlMq01htQoBurfRVk6O\\\\\\\/4Kj7ZQHtRlweBKnRaPJyQDSnhBJt0yQgcC\\\\\\\/OuorPaOtfGdtrZOV0DzG\\\\\\\/RnBMpDYvol+4ZRgf6zf+sosStyW1sopuRx6Dc8EEMwVjQ1j5EHvTOaPJpwh9qB6sYXWnwnpmFakxw9oRX1N+\\\\\\\/8ne51nywFh\\\\\\\/MKGcoe1qmE4bYQkmQAazAl+5VrGaf9sY4DqT3k6TMQlQE1qceIduGRIOlwXSZmloC\\\\\\\/G8dmdI63UEuADAeCwRrPINqGOSOCuBt3HW4SKctdU+22vk5LNy\\\\\\\/mUPknoV9AfYN+yTd37iDp0AHyGdeJKQ68+qYd0w\\\\\\\/tKIinat+VYL+3z4dbtSTJhj0GHX9VQ1uLks9x+AMyuRbhcybgq4tny2gIJuICT3n0ki2jHip0sCdQ2qbRrYslHvlihNcBSZfbaaBwsH55Tftiw4Dmlmip0nVvHQ\\\\\\\/4y7748Wb0\\\\\\\/msGJ7PED7j3XiYd5Occ\\\\\\\/9TQwdJQFXFbvjbE1+muq4\\\\\\\/8oceEzP+bhVbd\\\\\\\/o3ye3FadMrcRu1ONUE5O\\\\\\\/zOKU4bvjGQNKTpYbZZdqgsQzON5NkLkdhIJpQQtLMSNpX8h2PkJEAU9\\\\\\\/xrB79GbqCFTue2866ng5jhW\\\\\\\/2nDuUZF9Vlv967C8ypjqAJvVmymriNUZ+xDcMaE9FFKogBdg+jmUyqWfS6hzCbyhIYd31pJLujr0xoq7gM5yXR81I6Y1JUIT7+aTLeJmWnb9zUmosRSxhbTWU6tfq1PCSmzamskahvxu2ASpuNqIcN3Let6xnql2gVXv8lFkhjPLdlLO\\\\\\\/U1e9gP+YLMd+S8RLi7sc75vFrAl4geMwA+e15QlRtzX\\\\\\\/cs44Vm0e+\\\\\\\/oEkPs9R6DTqh2gbogh7vdxjdt4ZFKqU8XhM6+QPzkjsM0ugwUX2Og+CqZgDCYnXoG0foW6EBSiIppE6HPK+N8hQQVHH5NRbHVIsX47jCtDB+UdrIzJaUIc8Li\\\\\\\/TF\\\\\\\/gl\\\\\\\/r1UsMo14ocDHuhGchJ+S4bWKD5AY21CAvIvnLix6hj6LhBvs+8UXnoME9Xn+A\\\\\\\/k\\\\\\\/fYxtHiuR1QT2rWAWZcu6+6HESVA337nGqxwlE2\\\\\\\/TwY1+TiFuDfNfrXXLmde\\\\\\\/Mq6a7pz\\\\\\\/xY3iGDmj+M3N9H5\\\\\\\/iBAoVMWUiOb0\\\\\\\/h2aleXW\\\\\\\/nSvb9D0vpSpE05hSNTeaXwz6GiqiSLUX+vcQ6VlB1fM8NS\\\\\\\/xoR\\\\\\\/LLYwOH+7lfk1TZaAIzt6NWpQ2\\\\\\\/EL7V2dqicP8q3tlseLmMubQUU0Kzo76f+OaLsLcOnz7avaN7GiAAJONZ7NTvZ+neIOfxATUmjRV+FAhOhb8OP9yAG12WfCEY1FU2QSVyqhUaA2SvMIQCsGtTuI5FJuE1M0Ui8QyYyVkg1bQYR1E980+y3FEiQc98B+refLUiNHkmroXtMtZ6LcyskzhKx2uqetZPgrCl3njrdONTRGjQjJctezxc8+SV0vBRee9eXaxoP6uffDTpbta\\\\\\\/5XufOcxI8Ndo9L9IdhNAY24YvcA+sFz2z56kMIH\\\\\\\/t0wqGi5ebTNRsUOJWL20Aa+b6h2FmpPyjSdzgvBGPNiy+yRPOTSrSz+QQWnXDmLhhU30g+\\\\\\\/fRS2jAVr9jZKlbTTY+NYMv4A4mlXoBXeiyegHqshkXVs\\\\\\\/x9GS+Fa7hB2R38mp08KrUgQyifdkX2vIL2uCuNlW10dK5LkN1mGy6YiMn+QnSK29NY\\\\\\\/wT40urr1mIn7iaX9qNht9fEME0XVSGyMKMo0cjMbkCavLc6oCgbWtjw0P1I\\\\\\\/oZz2ftm\\\\\\\/1j6ou1efL+tMpASvctIKgcbdJ6YIEqXlJqkrMHCYjP1HweqA6RtqagMyWzOnLh+nM7Jgi\\\\\\\/9PtHN2\\\\\\\/uzXINie74b67AcJ54ImqRJa1CBJxyVdedpAYoDTvXpylSeYBij2067aXQc68GEZP0KO10y66VvqTYKbkyjHm4lWwhXlaAL1Hgj3bkYr7YMm4WjfPNHa5WLEHiq1aQpjYjD\\\\\\\/wQoblmNzBTGu7KHgni2eFcD\\\\\\\/THkxVVrlCwLCEFLxauLUSz\\\\\\\/LLpY0iU64b9OW72IjNYFFrCUusu3WNl24G4Ce6mfJQFnCsetkjPzwxOdG\\\\\\\/3op+SA\\\\\\\/EnbZa77Y6QlM5hN4JohTkogGnGMgRmMieVofrfGNR3AZh26jCLQ48GkhUONn3\\\\\\\/kky1ar+8cJnDQkUWL\\\\\\\/2ArlQkMO5R9iJjdvVHGC0Snu7mHF\\\\\\\/fs2NemNLD7pTsSwHWJQuHXBHCRtQaWokRFZ4K9aRW1L9PXd3GQ68a+MZxKIwPP4cmW0trCUXNqMAh1RUGv\\\\\\\/fBp8tDr21DqiVyT9FB1BRJO\\\\\\\/74a4jJHk55KLwqHrMjcaW39Rad5ToJ+k0Zd73Hw0c2AWfFwrHpoBUL7Anhym9KVcCQysqMVGL7+tOmieSs3nVp06pNbA9yFFvwbskYaODjtRrRDMf+VuXjTX\\\\\\\/fWhqLLLpD08J2FnRVqk\\\\\\\/DbvZaUkR7A\\\\\\\/o1sIkcfadYmHWUpzuz2xGEPPNQcHjcBALTcOGO3MshcUisBj6dOyspGbZFcnoguagdqNN82Y4fjAdx8KoK5VopWQnRZ+zT5lABe7WQeL8QtIaHsGfV0IbxMpsfjOBPlm8ccsCKVMnqd5r2bwyXx6g1IDS8V08a08IMmeTuVfKa0pOmE1y0pklr\\\\\\\/xi9i1jyy9p6Nv0LtPH2PbXZF+q3hmZcG6LXotzb2Gn5zdHyf2D3gbhSoTRWJMC8E3FskAnfh\\\\\\\/Xt5PTIyqiWeTT5R241t7ItldN\\\\\\\/WIkL8K\\\\\\\/cxPlbISv\\\\\\\/0yQeLF9SyvENPsoULFtYwqmhUCupvdIVL5+LQqjdPmu1OkMYdFq\\\\\\\/PXn+RGesaOP8EVB1pfye9jge1A5qew1SlK9DVdcdsnBeIgOcr6HlOk09MIn\\\\\\\/Ggqh3+yMTDrGp6+SYrkAa4fY8mddv3OUNtf5aBmndPzhWnvPv15CLpHV0a2Mr9o5i4aXsgtFv2cj6lwfp7cxmGu2zGdJY\\\\\\\/D4WJh7Pj9+2BomRM1VyG1r9V7ocdzwYrn54hOMKmhbh8oiwXKCCFITSCG60Dep84\\\\\\\/hdgtlV0WkJhefdGJhuSAvy2SCh+6USFK0frM2XWKSAVIvye7WT8gcTJKy20+htbb3m19SJx9tP7zDGuAa3cHChvFX3Wqi0A7qoaOgYrYUMq+1oe+ZVU1eOmgQuUa2E6Cg+gXcOIFiDGTI9tUaF4joCeqvlqu\\\\\\\/hPeIXQfgdiwRI4m46DsHA0KjihUp+B6NwzUrHrCxkXactiXYuzm38Q1bJccDNLays18B5GjO1erTtLjuJLN8nNfhOLp\\\\\\\/D+a5bNeEVfokSd\\\\\\\/7\\\\\\\/d6gYYCTQ4tCoajD1Ywx2jSci6+dslkY6BqycinwKqdvorUb3A6ez6cexPUPI\\\\\\\/EBA0ANXLBwIq7iOAEp81gXcIMPAPqiXc8Aj2zY8wXqx8BVCyVeWEa30rdbgXxnbbfIlZskqc5lEoTfNufw76dsPBWBFtSn+IYOaeoq65lr9hlTgE+WKTWbEcVNL2yRSHEAtYtYCiOSOWaARX2b2pPSK5+cUimLQ2gLYyqKL3LF3kL4brc8hH2Fm8uaD9QTUCAyRmRDJZjkdfjlHygltle\\\\\\\/Gg6J8qRvsnoYo\\\\\\\/VwGU019VjBRhq8qN+0dL5or8YmqpDdVdYCdCP8inkPWMKFzcBT7ReWomfnXt3outfuEBgqfdmApRlm7foGMt5Vvxzyji+gMRlIk5gI3FozFlW5XRHsoF2AkcDWRmcLZTw96DWWn4+ddb6FfSs0e1tYcYbjcCoUvkG2z39cFW8Bz9B00eyq7VdInWo8PxWCSluQ86\\\\\\\/4vRg80VwU70hpayF9w1tswCr1yO\\\\\\\/ZSt9krP9PJbGXeJEEo+BzOdOUfr5DiZNvS9cqfCb7M6MHZ4XdcT1IB1QPWdEAujnBGFxVrD8Hc2d2Qi4XRsbFT5LQ6W4QLHGyLJkf7AhZshMw9azKHKCqCETYizGEwxekolMOba6H\\\\\\\/DSpsNzmzju3j1rDnDypSCEnmiwblpYJMc4ONMznPq365ikH\\\\\\\/oHLMPdulGF3J2r+Kol2tVdnRN5PV2r10ethTSMwrYIsu8GN9X9YPXQRX0Pv0INDvWhCZXrAc5FQCynSMVUF6Xyjo877F9axApDehUHXl+IDaPwFzC+rQd7uIOUxd53hrG\\\\\\\/2ed6MUsl6Fz62rcAC+qI2dFbo8izRKdBjGk4+CNk+KsduoeatetPsZ9Usb3UNUk8oOuy5MiV1xk\\\\\\\/oTQ9uGRKEux3DmTsx46IUXtM8\\\\\\\/UVS7SF7kyBrIgNuPplri8nOo\\\\\\\/SVQz1PP0EE0zA2kinMhXKGrNMqfb5GJOsUhL8nsIL\\\\\\\/TmXDIqUSwZrn9xvI44Rde8rTo1ukZYrG3FUSkYBpv0r8WFzBolGjAyRzBY2mtA1HdEMhZ1IdbQD24mOeEYv3Wg4xd6ucw7\\\\\\\/JQ53OpES00JFVORT2c\\\\\\\/T7N3lDVNdBwUTVGFUFVehFl71kv8siX4m5d5\\\\\\\/DVjhiukduk6Jol\\\\\\\/LfpXVlxCLJd15zf36NZKkrrdtNtDxenZfWNKs6+OQgWtVmD8vhXtGvntnNCrj3jspu3vCdXsZAfW633iC0+xJ3FN1RBry\\\\\\\/d3IcVW0ZtReaxh97vP7Ciz5jJ+Hy4UPMmo+7i6jlBlzH73aY2PSDDrfO29Qh+VHdI1eeENIfFxgphLHx4JG14C921M7wWYwSOEFy6T6S9sKvE5badFLisD2tSCsimtGrNxMVKyDMLJyisMtZ\\\\\\\/SXXgZ2WXQS+kqosH90xAa8YtvHNhuM+O57njcB8E9Koxfa3dHVW2AhpQ\\\\\\\/COBQfaAZ1648I63Ujjn4MhW\\\\\\\/GsRUM7WkxY4RaSjAA8lAwB1O44OdUcswW8wSh\\\\\\\/d1sj7OgLvw0cDM1vW4S+pnVeVErPN6s7pZUgXnZyHRc7zUDwxDl1GUuN4TmXAyG9WH4haBr2qpJcyNDZDnGHABNnlKQQLf+1JVicZCKuPKcTGoeE9kk2hbVTOmebwQXZGQth8c+KIMOVPGcH5avxmujeR87BAq0qEBam8BpFM56H5KedyY9Jq4aOe2aE9Xpgzmgy95y1aJUHDFIkVidq2nJ4NH+qULMhkCwMZNJjPUIMQws4zpL6OnLxr9zYgYZ5IiNm+LXLhEVDg4mXJuKQR3SVXk6TJgZn8OBxRU8bdjUXFy4mx\\\\\\\/RINf6uEZ46bsdbMYldbXJfC3XG+toZPLCxL0b1EXFeRVk5JYCzq20lwAdRhOCNT8\\\\\\\/CMra2hGfWfYTISaBYP2qp\\\\\\\/L36Pwv\\\\\\\/sLwhU3uk9XfXazLv7XvImj1g4Z1jy5qmjfrSI6l7endbZwcQWMR0jErm3u3VxWtkSGSWV5v1jl+FL24BopZ\\\\\\\/5LlCvvQ3WBBw\\\\\\\/81oGCgyRN+tKqAMaquz4pz003xyPxr2KmWSJAqrupUURuhUtl6wYb05lLStuOQopGujHQoRRQDLADAJvFoayG+rn1C6h19iuYmNFEjVH9f4kZwP7wkgKePx+oAm4O+91+vCNgJciob6vR2YDco6o6PaVPgZH1BjAe36DTGpn0tKGy53Nsls0zFF8Ivn\\\\\\\/o4HWvBKLXWloU0t+U7wVaPa7Iy02wiGyvnKluk8xCRZL2yFOR0H250SUqp\\\\\\\/Vd85zh5C2\\\\\\\/Jy4MW1WZaplp\\\\\\\/KN0BbBt5zFek4pfFXvG3l06YkE+jcQsAPi9fK\\\\\\\/Mg+m3N59ZkCMxL0C1uYWzGZub0a2glr61kVqJdR6e6lIHubEHXA3bqa80Nv5HY6qb0mzTSzAP+EJqf+Wd\\\\\\\/4lTXxBTNfkgP9DrztksCzu92XRB8+VkodqFpgDDfqxn320lv08PcKWo25lAMs700wOUV+\\\\\\\/ppc+nQXkEi2H0U2eNUgiiRcfFzJkNTL\\\\\\\/NMDRoMRnR5DpRPupJoxKur9Kb9l+gII0dhxtMXdGuC8M3wTUPRIEZ1fW3WrRlVxSYKRP9gxQxJm\\\\\\\/gPXMyZtEDhhJtfmV2WNctMYcvM9zg+59dJRjA0bh4FN\\\\\\\/L81GMDOMaSC8eXL77Wh60lItdob1Z+mvyYjXMIxt0DJMvb9UrSyPqMrBeTIJe7XLEIOkKk0CL1+U\\\\\\\/yinQDf8v32SjS5213XPEOyfRMd4drntiLTkLWl0zPVChEeISEoPO8u597sMOm\\\\\\\/j7AFwxmgpQbZXEfAor8NrJak4eFFEKLBIDKJXhsbMXAxiq+Ph9qLSGW9CmbqiXgWtl82szUMAjrYsjEnIS45uVcpt7DCdv1ZZavjo9u8LIVdkgXFtpD9WtUilJuShtZp45H8lCPYnxge72kWiWDH2okLmAtxE675mxTB8WzzC1gmJ+GWNLtZc\\\\\\\/K+meSR9Pq+WXxe7UcQ71DDR5jj7wvb0J\\\\\\\/GvpvR3tRZ\\\\\\\/2okr9ZzzPnm+bN1NxJZJgjQ6VB0bEqQMlWhrpazkZNEp54exO1Wu1bgdcI4OjcqkJTjrh3C2uCMqQPYsOQFfwLHSC+\\\\\\\/VsOklOoGqYmlYkXcptpdnkRrG3ycMjKVkkHjnp\\\\\\\/AghMUOHFb7L+zV0jn9BwOXtU0GLMC8fFGZ5gmxcKYH4Zx\\\\\\\/+w43pr48wygT\\\\\\\/hGQBUlz7ld15rg8o7YNG3sx+XNv\\\\\\\/oTDsR2ZkLcA8RpVcT3njj07LcYXZLBvVmK64G9QseZ\\\\\\\/8A+bxR8Qs8nNtPWuqtRzn+ywf9To8oeyW11dVHGy6OE8U+IlLA9O3YDStahlHSo8cwSTmH2S58VXGoktnLTMnKLm+IGBZVbg3Tr\\\\\\\/YxGEm+HWAfOs81SlrSXuMJe1nSxihLa2XcwU+21OaKqkXzqySqdYAsNFAAJCbvxKsIsBizDauVVOpHBjeOXif5RnHFKlYMvoWIkPpXuMbTSAEgYhTLbSznahwopYTqb75R61LfF9hpiVlX4wEwqnvnnGqey09+F6bBfxxvvWjqFiVttJz+g9B9mzKIjLQWY3mwWVe3D9u6aIiTA\\\\\\\/ERK9\\\\\\\/+TSvqiNOiSaM2mA6PWymt\\\\\\\/aHnztfJB3wgRDQtJ3Scji5ArImgotz0R9tD+0uPglfhxTZdXQTlHtKhwREjTS0vvoZIBgsjfSsWN3WOkZ1Ot1mTgPu6umWOGg6kNXJoczwcAQzUAwWwJ3o0D9n98GxtpTvVgcy0Ts9qinmTM9BzOVkt05E8WY8BV7wCPGvQH++ng3ij4FGy3ng+hqFQKHcMMXYzuCwl8CK9YLxquCtd1BVl1MPaKIjT4z6BPeKbCfUKm3AY1KrJEWpasSSo01cPukOP38yHI9DaO4EutbkbubJNTfxh88aGDohCpXP96SYqpcMOOA1ONC13DRz0d77AV3jg+5yzTKwu+M\\\\\\\/bNrL220HWB+hR3KyGM3SvVB8Ixb\\\\\\\/33tT\\\\\\\/PrJTdpdPjPaFqnFBicJOX8qXW6Vs6dpwFSEw\\\\\\\/fmajaKlwCgP9uLvuer1s2Jbr3w\\\\\\\/OJv4xHo6o4IiiPtq8ljA7tqA8qA3IBebq+NvJv8AZJCKNT8ctYYe7mikxd0sZ30RXU37\\\\\\\/FYCDdREvieBKnO\\\\\\\/cygb98c5M7AZBPkjjTZPFi2blpE\\\\\\\/itupUVTLYllsM2hzygeC8AHhgHK2pEKZUg2TfQ1X3XDHRPq4Qz36s6cdn1kqaoaCC2ca01hL3mASOjDnxdn45dSIOsNdNEzeKNqaYBrwx3wNJYetQgaBUlugrmXAURZk+OjlfRA8BQ5Z8rUznO9k3jocPjYok0rSzigB9MJoW6msrl4OOFaGp0Wfjrr4Cvu1SFdM+Ig5Cqe+XGK+v9qQUBWHBG8UyevEJyJ13ki0pJV\\\\\\\/CAOKXj6M2M9qEBIXufmTvM00mIqMbKPpxDOmky3JUbNihcjq2UxurIybXHUS8rWiNELzGUZUzI0GlIEYuhLo7REP+HU9DiGllgP4wl3rHNk830IrfsO+BJITSjmFMsEp6wY9lrps7Rlg8V5riHh0Yiw5Fyzduaf1Yn+zIoGisGgdR93UdvUBQXoXcFFSEvqDw9Ha3xiMQOcRx62wiHciOPguoCwOgwvb88Abu0tcvyc1l0SxGrGA\\\\\\\/+7siGBRDXAPFgvwlpjkhcUIvr5nu4Pzf1YeQAYXmNCZUhGQMmbKIN+doAyOOS+wcD01vBfkgLSYYW11o7rjkYOsRLHw\\\\\\\/7SXipHp7sdtYZdWaNVIYd1eAmX\\\\\\\/mDhvlwVBac1qi3U34Nq78K0FCG2Y2mNPlvFLCsyxUEtr+QH39rctinmK7lCLMPJEMrP1lyTdOe\\\\\\\/ByCZzeJqTJXtljZEhk0c6e1ye7DyKMJsKt4U2U5SrPl1pZKViXf9XUWJjgwYGkbWwOMF9p\\\\\\\/qn+5gmGS+p00LO37jYd8yhTTseHm3iVcoGSvQNVpOa58Oes1V1UxgJzUh1qzSlZJHmAOvnfHmvgoQ+B02TIysVXyIUGhFTtZQDJVyRphZ+Ag+meG9iYy+t4J3vys016iW6nRnR5wervsl2fLWpaKiqVChT8CZccXX5m7VpdJSUUJcn+Mb+Cvhr\\\\\\\/yLoIZmWSLILw1tynAnuwYfJPGCEX8JiJCOJgpRaSb45KAUP\\\\\\\/I\\\\\\\/5HRHQQDuTiGY9CJlIJoopTYtSV7iPtLfV068flmGcGAg\\\\\\\/hWIwN1CUCfeYaHqPIARvssd2e7Np2iOZ8ADFBIdTWbo5iBQ+FaIyoD7LtoCeA1TnNT3EH\\\\\\\/w4VQJrSU7AihtVuxuxXV7yNC4AljTIG9FvQPoCMlSCXsrfXMGa2kkaMhRWQe0LVEtp4oaTCgAnva1+ZUjfQs1iro7i9AsV6oz9FFzG\\\\\\\/sDMub7k86X0TJb\\\\\\\/jGK8dVk7iB5OEIvb0aFYitkyCKacO9AenanAR6lUjYF5jd4A6TbH0XOY1iEfHM3J2TRors1k5we2QqTCe+MqGGJhciCx28VfUkSKiOtK+IMUeyqzWA1DA6rg3scQ1yVdio9dZSvLkeGuhLkg1gJOmfIAJjdLsxoZ7DKA5Hd4amTnRGaIY5WQIxFnQmwx27bLV\\\\\\\/gjRY9e6AuLhHC37DnDVNyF\\\\\\\/vCAx0tZo16ti5KmaEDtpxn\\\\\\\/16vYMjNm7TTRa7OzAksI4oxSMPwZZdLRIutJJ2K4rxj5jqGnSLGvLWoRHrUDKDt8PZibrb6WjRjhziAuBcHbjNhDHbZc389wK4tlnrgDcmmYyAF8lQUHM79Upr2+TuKUB4L+nmIasdvUtRe6HJ8DsgP7FpMDemUppdw9711VUBbNNIjYXqPXhGbDh8eCBbk2UOvuzMRqS8kKSHpQ\\\\\\\/0o5LTL5Bd6S93UHVO2XBLthbsfF8ZDEnYBT\\\\\\\/9Jyzpbq6hN7g4\\\\\\\/JHhKYDNCOZv5RcNmSeW86zhK2mN0ApGXAHMZgLvEdjHgxOvXfFEUowJNVuS2WHGeK+Vq5EKG3JgwgT15Mo5VdJo1erJ2FlCVB5wA9C4P4YsW8ttp4mpBUH62bcsru3IBXQmNp3suVl57P3uVke2IhaO82a+Lvt5iOyjrXOAQFzUsiwrtY\\\\\\\/snLrgrWMJnlCtWNYnZZo5W+PFJRWHup7Ned8bJChshrPDGAdP1RXzJWd85yX066WHYdUtVoGOTQs9qOEttpkm6J6kx0n2+s1vIumeFGTLf1uH3j9tTzxmo\\\\\\\/NMrpiTBtleT6s38UVt0OY51vbyCU7V7g1Pl1AvQWXv0WpTH+v6THjyw18zCaADHNB55ROAD7eBXckpI5ClmUgV5XlliUa1B+MZ9LcAwfLCAY6ssT6oWyI7mAPUFbjgVEEp\\\\\\\/isT+kDWAkvxwGsMOAu8JJNojCbICQat+9yHtO4BrSPT9oZgT+JKUhiQaHVkGJjh2HkJgq6NFCK7YfOd5b2W2DDHwrpumXXDPVJK1FooGfsJqLVlt3Cp4U1Jb8HNisS922f6XZpNth3WWFcoGxobo\\\\\\\/dC6tdAyUMjb9griJzaFaqcIB3zSiGdV5PSv5EiHi5cw5EYW1ZtccWKZaf08bsVJ5xXi2MZ6kfeCM8FqV6qat3tjJ4s0TBpIWKpi\\\\\\\/eJU9LmOMNdDLrrdpePUYyEc4h413TrdrcGfa9CchLKGqsDaOcc5ZHszIumWlFAME6\\\\\\\/i\\\\\\\/40ZBraR93jv\\\\\\\/ek7sHv4y4r2Y\\\\\\\/UT8kKn8TZSskvOPgSvlfN4GCmXSVouogRhRKny5PggE6MCHDwsW4Ijq+8VM7r4rhYGQHs6D87X5CkU0HQg\\\\\\\/LEVaNGDZlDH3lZ6vWMEWPriCx3eCmzDfW76MLwh5\\\\\\\/sTig9WvGyYrz2CBzppaYlDd\\\\\\\/2kFLWbtRoynwNI8kEUr8ArDMZWP0qhOq+PgXqHYfAtlUERvYRnLi3+MrwrV5+rjQgLJ8uBpmAT2ldCFOIxx8\\\\\\\/bOgNXbmIs+FtsHle8rDuTF2ngVK8cQo3Kper5HveNaFNVLyZALqlqECPFphW4i9z35IrmW256zS9dOwSXDqQc8F0iPRCDbPnVBeOV19sCGLCs9rmRbe\\\\\\\/aXAmMnkRMJHhHNUIslYGhZupsFP86PdQ\\\\\\\/H0h7zdTJxUPf4+spuA\\\\\\\/qHcAzqbWVuD3UeLFq7OfIODD+Zn3MuAFo7\\\\\\\/CfxjDnN4n31LWYdMWnWDKyj3OWy4H2at+N6qms+W1zFQoa8rksoqb4+Ab7f6CmQv+OsrY7D1OqlMNSMKDt51g8\\\\\\\/tQwx5l6HxxOMbNXPyG7Zqi6S8J05J+DzbeImkN8BD4t\\\\\\\/EJWLUbtGE1JpLkKQ\\\\\\\/uuLY07fXnb2KuGB930yOH2yJmOvvlQsl1CNh4fIKszBvHFAPYhMNjJ+OoIVjikXP1PsSG9tLqGXMpfp7ciCcwxo2VMwdDbhhIUY8kSob38m3QhHF3eR1bo8FK0aelShTfJXe392bjL0MN4fzTi5iHmo9VJbaM5PRGyNnbscSsKUpP64TJ29eIMPImiWyK2LymeTBOvtX+xUQGeYnj5vMn7HQ6krIp6si4YCwOT7gRxAcVy4Vzn+K47I+5BqbfWz2lPwzVm3IapX+\\\\\\\/ahrK7NuPpWzcmUEFspAw\\\\\\\/zqjVfUQ7GB7IS\\\\\\\/kWy1aXIYjlNq+Cwc57ZuYqpEx42jOCVXquCzHYppRxeF5qpHNnxLcNrQtV5Nga41T85lXDDlXAtM4D8MmdKwkMr9ces3rt5VP1WZH6JksMGtO27h4HWWVggTIlV8knvoYZo3qN6P2X9gR9Ry46b+VjTPGPIGKHdsXvStbsAHRHXMquTvG\\\\\\\/uusYXi4q6PPR7BxEYaO730ps0DuTKBYOdYXpsUz0FiTtYKF02UzfxIX2Ixyfcna+Atzg1WsypqSMmGa1Oi+x0CoV9243zITmanfsryNIYYQ+C4FhGSLQcHxNuJh83XK3bqkdsuXVTgkhHUmXRX9bbIkNRSHQo+is3V7qYpaB1hO+k7DTF\\\\\\\/eqvfzIjtIkLKQnQJn+jJ8w\\\\\\\/SQqqow\\\\\\\/KC6zMN+YSVDv6IE7FIDB0FdIsTo23szkUlZVYpDRT89ctfWrrG48AMWE5Fb9Ize0KuNTbqMNpbJ8Z\\\\\\\/Hgmo4sODKDIhgjgtKKlmSHsPS9bvd3Zc0DhHtlFNTYwjsW\\\\\\\/455S1SeY95g3hby42JCA2sJFaPWGiLYWYDQ\\\\\\\/8TmlkpXm02g18rt8+lizSjrjaq++qc7UirK97NQZb1ZAu2uVH658ZK6Fncvx88lWFRuln+TelH6MT9zNC5rW8DVsbexh\\\\\\\/p99a+LUOlLNY+O9tOJZSloFXRIoz4co\\\\\\\/sz\\\\\\\/zNnzXkanIpzqPUJ4MIKsx8hDHSKc0ogI8yqYNIl8d+GwHZVwtwAztUW6Gp4wj8SjFRXloIA3b\\\\\\\/gikZ069raHAkcqFzC6Ek6f1kv3Qh7Gl9TsUyWXvvZMPiwRVQfK3ZacKVgJoFEwtZF8vDbA\\\\\\\/Bei1Fq2ahndlKDyr\\\\\\\/arH5rU1jyYZO+bf1LK\\\\\\\/ERkM\\\\\\\/tWi0DwTh7cr4eLBrlEjnty8qDhmStoDoZrytgSEZxpg9vRx8kvW+h0\\\\\\\/jvU8mB7NRSpgnuqZ30qCoH6oWviuFZU5RKJO31arVF1AcqaNUSlnRgbzrSVBUFpvNyy\\\\\\\/ZN3uIzh7vH25YaauB1wwO8JbRNe8M8qq2IsX7zmp15oJJBM8zC8oYm4t+8IMcXFEgXZ\\\\\\\/X4ZrsVzczE5YYO9+Mx4IaqyVTuFeWdc1mE8Trfp6z5m8P4HhlscFifsX+pPEDA4Uu9idl+95noZmR0Jwkf67haTqXXZx3upgS9HA9YCK0nkK2hZtnD1BpNBce\\\\\\\/9Nfiyexfjm7rk\\\\\\\/HHuA8+vhrzMc8AZ+7d9aqi11WK30hkzYiHKta4sTtKLXlgAcjbyoxcvE+W7aGwgr0Jt96wVOvrQirogEKWes5d2OherRGroTAnxXJ3\\\\\\\/9wtTeEjwnxjRSI8kjbXcLClIUPI2hYr9gZqCxKW3YNqP5QeXdkNfTMKPpTlWY8HqydsTM7kTxkTbjPpuB1kx9Khthoyhgf0MjYHfptv4co8+OOFEfJ\\\\\\\/okoViT0Zc0Zpld\\\\\\\/XQ97ZbeUdnYrqTzoWew+9dQ3pfGstNeiQlt5cdmKGXQvAm00XfvU6o+FjAIYrPdtgNDxmRbi4gTIQkWX0mMbmKPiWnAxuoUCNHnIC4EDtPu5tmJWipKpBmJ5UHsnkMnZFEYrPkeZYxON0cemREfT2nSAnpi3\\\\\\\/4gZKOTowGSpmdlCh12eW\\\\\\\/OHjg82ZH8UyXNBRf0ea20RdlJjnd5hDMxIKBVeTRvL7R9WqaXGuGltbQz6PLMpM955xx5UMGNC8Ldp27c9jtmunr3phkaM1zVPCL0s4SCFzobkg4DwGMfO1dSHOXyyVTXnJpLrtFLrUqGhJ6Utrc6VMIDCXdEWpzKNMaVLFNNr7H4k6gbpk7O4R5+VxJWQluUrBotpXXtCmAfOj4zLUFKk3U235unCCUZVZ841DxQ7wvSNikpmrTJT1J\\\\\\\/qeRUUD9jlIe\\\\\\\/ShU1c4rIbkeAUOd0Lv9sg2AeCdE3+baThZK6o1wEfCvJA8\\\\\\\/4ltpXFiXNSXlbVJR0NioCir5MIPKGDzPfEOA6WYVmLR1wamE1BBwjaY5RYjBsN0vJxzKNlHDDqD9Z1n\\\\\\\/sBdQE7Hto\\\\\\\/eIigjVTHBSak1arAzImjG4GLFkp5a78qJWRJBK\\\\\\\/DLBRy207vdOIoUSfmgRzI9rOReHI8uwxMq4vqZ8vfExpe64m38MB7bNnNhdEu8GJty29WiAJS3fVhUmx6qCrSW\\\\\\\/6Ps6ZAKb7o1+NEux9RPg4FD0o2C5rtgaVPcJxfS8D92ysGeU5Y7QiKrwyMdJ\\\\\\\/Qqy1xaXer4VJQGT56zw9BxIthKsDq9fIgJ4TKLJqwQSMBypoowJwsfkaXsOVD7Yzn\\\\\\\/SmGfcWzJTYGbHFRACYPTqDYKPts\\\\\\\/QYQHqvTd2\\\\\\\/wzw6DEjqw1Iqom0tSUtM2Fd8qB1xAc\\\\\\\/sPQkiwP\\\\\\\/JWJYGdwgJmTVIDnSL9oCE80uW7zO4sdBHdBbMJ2SaiPczTDsS02mBFoMEUgZLRfg6QFa\\\\\\\/hCbGQLdH6uUleaV6OXpdZXdjXeqcA7oKyRUstWjJzu7VSeK6U69xTDgmHMP+6i4WuJpAoBTb6d\\\\\\\/W9Inmp7D41mxKg6jPE61bfz++3qrKZEkiEyjkEqpvljlfGr\\\\\\\/7ou+miR8o+uWtjkAGP\\\\\\\/ZQE\\\\\\\/NZtghvb1tPFBQQWA\\\\\\\/KNa5z4KbgZ6eZoett0gesocczzpacDBXkTUfoDnfvLf377cjrqhg0T2PgWMOlTlx+wUUr8ztrxFuIwnx6QE+54THuDsOMyt4\\\\\\\/aUl2P8lNb68A6qCGms73xKbLohsom0xdQJUNq+epC\\\\\\\/aR5WBcqOUNTjPHNMtHvZIaPTJgSesZHEEWHmnjR4KKYVtK\\\\\\\/x8tTGG6KWXmBabHp9MtpzYbYKwEzQQAOy2To1ZFJt8LtVgygJqzcGWlMPFJETWXmqNA0TWr8MGDRQvOq0W+feokQ2cWrFlMC+T64DPjlEd99v0Nyp57P4HP08ZN24dhAOSLfUGKF2basq0aFFDbtEhk7ZcJAxaTbSgyaAJBIylyI1n2ZUr5KN3jS00aKKuP7Fqg6TTbnHQ1molsz6uYhjHnhfB6lAG\\\\\\\/Hxc7uLSvPhFeha9xhPiXS8FtgYKbVn64HnHTdb2iFFPH4ju2w+K7kdEUQxz7j8tEA1+ojcvZiaiFaEieAOkjTHbLVSVyjpPDZCfwEbh0eG+C4qJHPI2PquhxmiYAOHVxRTUZDI1qpq5TprlqZYLHFz3llvt2gO7IO5YMyRsCvYGOxhvLeLLG4Hxwuqyyp+syItgC8CBYP82Ixd3lJ4zFJDoQUyTCLrn6XCimVV9ZZLh3+l+2sPbWbAG\\\\\\\/OXU\\\\\\\/amLxE3CPB49A3yOqfQwmnsoBmwuwTMULPzU4LpXtiV4BXjd9q4ObF+f6+ozWQCAQQj1eJxtVSW4jv7QkVT03\\\\\\\/MeHDet70rd269fXBnU8r1mu0rGQ+kLdjmUK80YZU2shI5e3U2NMHT2BHXsrgHAsKvNVV3fJgRA8EeEWolQE1e1FOT74InkgSxvPJAR4W20lvAlBCelJPZND9HOBZKmlwNccxP95Nl+9ITGIGB1XoOn7qPPWS\\\\\\\/Vle60vcfjxZCEvXGzLbCTVIChQmahAs7ACoqG0pzOXWZorJjUxb9CZSFOcBAM6XPkC6U15oFgLZKSYylej+6OXCu0x7ZXJPu7ah9GIS3TQDxKaMOYIMcWqDIZURLHKO0poUEg9DDGCd6Vr1B390SODm+ZdGLBmEINdDHLrLyvWGD2KPF7rkEiHj7\\\\\\\/zE+Nm\\\\\\\/eEho9m55gow7rjINFMaPK5pztTCbigfJG1mOCQa3Esx\\\\\\\/BOxtALXAHYcqkx6BmioBzUTT1aA9cSatuZnlaM+iNF4\\\\\\\/5fFtdwPoIMab9Rq5HYiyPjBo7LIm+iOSNqmOrLcV4YGBuv4r9XtaMgmquZ07SmouZ7YVfz4\\\\\\\/Alccrw+Ils5\\\\\\\/CxyyvJg2XakoHGtJBTrIH8arZRjRJDO+TYvhIXNTJPjfXsSbVa1vLsAR97Coqe1y\\\\\\\/JdhcWWkx1w4ltwtgW2VDroeH8FInk5ZCECXvQGotjpcumOEN+bSpCqW6MwXN0eKh\\\\\\\/4Xg7OFbkBciXR8XwzGkv5rBlOEi99NOfHzFMl4o2kCwv\\\\\\\/KY2i2NxpjLPky\\\\\\\/G2j906vNzen3rtyDvAs3MVpkfvms+qlSMH8jZ7xVonaucc9lUECFJqY0MgXPym8CHPDUj38mVODZFHyDgbreNsYX\\\\\\\/vaDlkC9K8NaGh2PISexv2BfrDlNqR952AHDImG063aKNkgdG86fhTXsM5Q1xyhU96cmziOxRB9XRSxBvYDWMASTSIwSfwiokXQnWBv\\\\\\\/qI9tEfI7uU8Tb3bFaL+fC1\\\\\\\/bACHFc0i36GtEfwwErlCcwXVi+05l0RPiC26Fy8iwY0wIquqznd4ivtxZAqqtSJr1CaN\\\\\\\/RB2pe9gA97PE0V\\\\\\\/Z8sLluZle5P7LVefLkwTDAwLa5cGC1KjThQkqcVkGuemS5BfpOwUt1XMCUtiQRDfhqglv1+vVb4uASNhWTPqFO2nkgGIM6kMz8fFb9LEIBMPTHlMWzKKv0jNUNv3D+rxzehpqpIce9qncHgDZ\\\\\\\/AtwZThpz+jRh8sgXg5FkG6G6LMiPDe\\\\\\\/SavNUXx5bKITnuZ76U\\\\\\\/CwP11KSR+q97B5p\\\\\\\/H4nKuA+mHKsX0GVIKFALjemhQfwSQu\\\\\\\/gW2BVCEUsN7CfOO9XoXwwn8cZmcvYPMNf56GCQF1Vjfbb718uHxPtnAMG44PpokNt8sz3fe8QEUWf4Tm4E04xa7GRQDkvTvkzdVIi0dOUGa+wO4tlVwV56WBU9w0VZYukcRvy6uYucu0y72FMfsnogAwuS6eTyN2o60MlDJFRkOC39yP+XY4K5ovPbO++nVUt6BbsQaqF5BbNbmX2GvNjOY2eQEPSDNbQf8YDDVS39a7KDoRbamVECMy4cauamUzWnoy+NrTT2afC1xHyUvqYVhi6xQK4i5Txe+mSDHFm\\\\\\\/lfcMPFWZsXu9BfeSUWESKxitMonLRFloy4Po2XjfKZvPkhy4thwDdbpQUYhlU7dT7FQ2L3JfwLcC8m0mTY0lze7e0pXwXvxWkpUsgwpMFNFxsdkRePSnLYn3zVr3H4PDozPWsxtkSD\\\\\\\/vpWc032GrD+B3FZ\\\\\\\/FAlWo53yhtwSUErrF7dLXAO8OpDeoJCXm5BFFGZWOKOBWW9bMNwv74eVJfPoiVYl+RuHjNNs2rMym8cOgTRn4EXCEHTXgENmnSSpq8Spp3QXILIjt92r6H3AuFrHoKqkNBFJbZAO1N38ZuuQ8DJpwcXIi3yqkgdb4aFflMdXF6cmT5bl20cxEVYfCj6r4e6dxGlADR7V8w8\\\\\\\/1zm+oWIxXVD2v+pJ\\\\\\\/JfkSG6PKvIYBQPVthQV8HUEdlt1+jIGdsILHyYl7y\\\\\\\/UYgzzOgAgPLpC73Vfa1Qk\\\\\\\/3RxyrD6OGWkItMWwu2vWCCSF2n7r4UlAOKXlTeQY+cxAIIxehXnlVnWWIEJaRpi7eFaLh7LcgngCGazcnSPeVrViNSRkT70i6\\\\\\\/udvvkXqxB48e7P\\\\\\\/tWojrOAoJp8xbOyCX2x6IPZjCR\\\\\\\/cxCF+if5XwxPhm8ZUlyOBeAtGW+ggTV0UJmZk7m4uWa4k8OL5nYQMwaXbuFdIzY1fa8Cgrv4i6gctZs+RM7Y6h4Jcw+GmwdYaUmJK8acjJPr7OfPgMTeZRRl2h+\\\\\\\/pvQV+8dtDDZcnQ1Hqn1j+kxF4lfcmtCub5kP0yYueTleoG\\\\\\\/OZwIivmWGrwh4UgJf1yqVvPdPz8D87QMBf3Tv6TJZqPh69\\\\\\\/l00keT2Nnn20V1bVrIQbt4TlrwlBO7zIgBVimtlKAvYtYVCbLsGYuXtfol2YJzQeEkMsVZaEDzMkEGDHLG5e6dOpskrGE2DgrfvOHZFg5nYiLNYrsRn5OF\\\\\\\/npCVPWDUQebvyZhFIK3Z+3fHaNBDynQT2NyXLd8FZsys65UUdo\\\\\\\/NBjWVhZBd+yiBlbHUJJa0G+vmykN6V6vBD0FzVHdAfGbHc6EtpkOQHAVdMuZ\\\\\\\/HM9SFKhJVrWVCD7gXW6x74eKs9tIqCOYlUkXQ6TxyKpjkwodpzSYBJwiBeadQO1VJnBmwS8jitoPEKffl78SpjJZQxXFzECNlua+PrlOpq9fm4+3wvXYMWSGr+9hq3ACOb4OrM\\\\\\\/ffL4PV0Ry\\\\\\\/B0EVt5Rkny+zesT5UbaCPr9guYHq5AWs8iEtkt2biJfss4rituhPrfxePlN6\\\\\\\/\\\\\\\/IMg\\\\\\\/WGBcATDvXbDvNMFr3R4UG4Hxa3zozM5yGIiLkdOaHKx4TE3t6L84reAr9e0TO+GaUlfKK3DhQarmzVS4cutNcXPInz+O1QKQr0OAOjKwS9IyYIXFimXJa\\\\\\\/ANMhHcN4c0UsLdV8fBe8E4ohiJjyJuT5ntJvYAcwy44U0tXXc9hTYVMHfArsCIrcobUp\\\\\\\/uzucbqwStBhIFXQypzjkemeVoSKBHZdHS2NzTH9t8vfHanj7qPQLcWBPmLOMsPqqGrW0ej3L4WDQUHioGG26nT2kMx5RicT9lc9K6ErJUNhK1Q33Pjsve5j9B+sbEhT5vNBHy5uzg2Codn4+lFuJoG0WvOh17odbbDbsnx8\\\\\\\/V8ZZ0yDbbZXswNHT0xga1lIycBFYpbW9aK5Xucm33Q5hta7AC\\\\\\\/5S2+3BvAvClTUfd8GROXs+KT05GmvXNi8t+khmtBmoTxBN9G0pE5RET7dCiiEos+vyVhYHIzZo\\\\\\\/DmyIy7Rl8ktniCBXCOxz780Jh1hSElBCK0suuZyx\\\\\\\/NRsNQDt6IAlXzR4ZFaJ1TSr8OH1Z+tm92aWjB8DDaqFn+iLISc9RQwOq3UFLfYMK76B9g4ZI6LWdCxKxxNeIZzyxiFUk0VWkSEZyK5RFqopAGh0210VVZthxfE8RHOPNzxFBlTtDDSYqZrdnitJW6VPy+TQ\\\\\\\/jdAjCVjOBFURTmvX9J82Hfqz1FtMHhU7m4UXymUl+Iy\\\\\\\/xF9MJKOByY9MX5T1HWhx8C9iFxGe\\\\\\\/bwXclfrk94ZqKeG0LTxkCE8B6ahjlO4nnKHBQ0EFstTV8aEtXoLOIcHLsIhy5ZBqHbC9hxAKut63UC0y8JS8XmlEX2MCm7vyALWOv+9hzWU5UXMGEFl1I1CMyiXqO3U4\\\\\\\/sbzfrOypVfsr5GECcDwInJjS+8ZawGddxijUksAtakd55MHc2JKArfSIevtwZJWzn63BZEAq7lCYmW8oTPbmnNpOciXLCvSJ2WDfm6BMpr2OtZ+Q67WmauRKUuGsZ80rz10qQ59bjFiC\\\\\\\/P\\\\\\\/kdwY5Q2KXzoTZ\\\\\\\/Ii08Z+c3skwV\\\\\\\/ptqrKz437++LaOax00Vvv7pSB0IBeQJja3vfl+EkSDboAH4ekuTPR6C+KxbuYpEXRlkf1hBTBb\\\\\\\/9AI4LGr8QGqkLA006HbtEE38GVvTmheWF\\\\\\\/rKX7KJmsMwYdzWCidbQ2sxydl8hOfxbcOe\\\\\\\/HH0UIlt6D79+KBR60kHr31emSwzMwFcEVFYo2IHktxqQ36T58aYFPQLA1WUwVm2ZrxF4cSGK6eDifraTNciE0O+N0pYCnpH4EFGd2NtCWy8dRKMUGdzc3hvWFiaLfEwcrXsDT2pev5i1mbLVpIeGdhmDGhl0KpRc69UaReQ6VYD96Pl0um4Tcn5OaN9QJiQ8saPRwfk8BycD4\\\\\\\/FiRj\\\\\\\/5vZCwvi+LavnmGB7iXbyKyuyZnkjfNv+ECVgmReqQipvPkIL3Gl2S0GEuGDtAkCcUdGg7e0KvuTMdz+TyTQF5hPoBaeeOQXdu9r5\\\\\\\/UyoL\\\\\\\/Y8VHObrb+t3nbd0RQGpqxObX0gjaj22ZbA6jbENaIUMZl6afHvsgjJ0XWgJDeBxLL6eFXNP\\\\\\\/Hmg1ynXoLGdbLZT5vL3brhXsfPrGM\\\\\\\/nBQ+uy81m1DMJd1imb8wPhMYqUENA0TKuqfiIavhIYGwqT+34cQGDPhIQNdcRozYLQbWVbSSfU+emmPAPnDJor4nFzJzREcT3f5QlQpU91UOho40NiW3ugMi1nRJM3Nn+Pn4k8KbODa1r4Bs4u0WRrYZidv7n54UnK53qirXLSiFnmcHfnL2zMu\\\\\\\/Mc62LXxDn\\\\\\\/nzglI9FIDXyppMxGYP4RZvLF\\\\\\\/JB2phFfoS76J6f2+2+gBHTJlnTblTMbax4t93dHYi3zo9mR+YaOPxEk\\\\\\\/CYE7KC4\\\\\\\/mccBMSi2I2XzXG8fD\\\\\\\/X8h\\\\\\\/vIebLund2SgJ6rQwAKUQjw2dnCqvcAXr7ug1\\\\\\\/akumwbTw7Qf\\\\\\\/RO2IgHUom5RCxtaIYrppt3t7Zthy3v8CeR0YENOkLQIvg6MBM3jxIqmcJJLZXbn\\\\\\\/D3Gq6WkvY84wd7us6rnLlb9DVUykxmPeKaCPXGX3l7dvenbxbs3QM1OnhcXUACV01jkGv1JE4vlyMwQu95INgRyPCluCyTj7ZnbfTa81Am3+AMmv0nsgVKZBZ5gr3FIqYlxuHeT69OKCcE2qusKLjhBoDF5gCnwyLdwqlNhbvjY0Wk9iLHD1DYBVgPbCNgBDEpX\\\\\\\/ZNieZFQocfpy5FB4vFiCiVuxuk7EiIf90AVQFqYLHUQiXmWnBj8i6OWqDWKSfOeoRUtliy0I6g5Fv\\\\\\\/0GqqvO5mUivOTNsXy4QPRsrQ056ncFdcb4+uu+VsAcstdIcSR\\\\\\\/7rbfKnX2mYzzmfnmyL8iU5O0ieFaLcBWMw24+Un0cmdetERol5BK9d9nWNuAL0NGuNIGd2XXFY6wgUxsCkcy\\\\\\\/JPRTsSLol+yWUYMIaUyeQL6e2OLdl33Rdwq5xNmV70CLsS2djF3d69QkVyDbZ49y\\\\\\\/IA4ERln1fmgMbWGZ0ZrhtyusL0teo6S3H3f7tIB74MoonyB34rev761B+\\\\\\\/qCkT2k+8ze\\\\\\\/ImnxzAvLZOEI1NG3lYUc5KN42ljC7vRKSXFw0YeV6sBj0iD3vRGKvPQTZ9Bkz77miGQjr3Oce72K5232fGA2GtkmJGAmD+QgrNrBKonWwYbT1SIc2atAiINqoqPYKD+CmhGQcv6\\\\\\\/hSSOLWYzBi\\\\\\\/i4zROcOEpwbnMA\\\\\\\/rQ1uukOuX1idqhGzPL3fPqdjhHgrIoRM\\\\\\\/ZDAosXmLZc8KnkNF87BV3A+j5GCXvEJpqhAGYkss\\\\\\\/gmWxjO2Rf2aDd00wSdzhdPEgsOBtKGVK8HGpKB47S8rf5gGvVACD1ALAXh+wxKMFp+A380uHDVYh+YwWnocxh8eU4HBniNP5H7NLmdYADqF6dVrNIC2QvNGyEK3JSjZevHzf8S35o8MIXHJsQba3NlMDRdumVoKgbwbo1hYkxI28zvDSDfgBF6Xca24ZXeEL9XYgUF7UrxZi8sNu5WRMLzWHWnwJxH5RMwbP8uA3JHp3vNehye2wP90cWuD10bu9JEGF8jtOW1OziUUpa3f4Vkdmc61ev4Vxa7n5WaH0IRKb4fW3lHF9oYSvKermTV\\\\\\\/7YeU4V5GyPOqHmuHEnK3\\\\\\\/xsxVsh4oTUgaNb+NrYyGbpKq47kYi9ecdrw7rSCz\\\\\\\/FrFRppODrLB\\\\\\\/NMl4H37tPnVpjaLY6EObJDJY02ZKLvne2IA0MrU7z+9sGsg4uOe+mQSRY0dDJpDaUrR8rK+nvGsKR654RvKZ31qWEfUJKdEd\\\\\\\/gPFxMOOuBCRA\\\\\\\/Qv3o47LZY4brDhSa8on\\\\\\\/wMhf\\\\\\\/OCQMZzPADgRkiSGsz8LkPEQBrG0rjHtB5evvCqnfybv72rQCgSrlVjEaU7NfHaeoFcRU3\\\\\\\/PsizFR0i8K5l58D0ni6JfJDqT+RDtSd0x5ezsl90kULPc\\\\\\\/9Tycf9ns+uekD4F1hmGa7iUSUBMiuh9uVkDrev5gMX3yjWbKtbnQ262vmYCDmNAXV+KPPOYXm1zRJ6ZdrP+JQBIkYKXe9hSBImvD8kZVJeiYAg3Glg2vQVMUQYd4MXxYRVadRmpkDQjnzLSSY2LQJQgJqnpbh8YSUVCfBMJx0WvvCFQnOGIrQ1sHElxcLvCFW9mhrESoItQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022bqIO2NHWCCZmc658lXPPJg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022679473844b63a711cbb5666148fbcb855af6c142ec8bf525f3e20b262551ec6b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wGuF7RWrhRV8LLjEU2vhXt9fchYmHRiQz\\\\\\\/qmjxdLKabBSqj3LIufVuK00ArfYuP2vYNcCdFyiLPrybnV\\\\\\\/AzKIaHCxRZqqC9HjMQCOAXQuhX9Q84CGXuWUTL94tefsydhb20QL\\\\\\\/Lu9i4kGD4dkM4+xA91lQgpltVl8zWwsCzEQfgkgLQpjcR4PVsi6kj3YvTwfCZJCSlpCe2wCNTl6U2RNFev9ti6zGILTLq8PfhrUmrxCIiqoH5i+t9T4Hb71do7tRPPXrYSxgXF2FoIFEvYNFkN4BuWWpsygbanOEhdVDRTNchDLAKnWeru\\\\\\\/gc2L3JEe5prJu6UgS4g2NeAcKfSnMJiapvGqAmJXgnZVQF0NlXaZlKLEHe5msG4SzeoKcLut4WNPBl\\\\\\\/lWNpomK+ilaGt\\\\\\\/nwy484qU5aQc6RASb1hzNDprfm5okaEiKG\\\\\\\/6BolLjbVWQDw4Y1dz3Zk2HJ1bJQn+ALUbv+nf5ns3KtGqRnfw4BREzVsE\\\\\\\/F+zg7pd+32ZbXsA7BsP73XFSO5S4xP6I06NgXhlkyWCobNMOUf1HjsevtBDI2MAXWCao9yP4Y6lt1zvDq4+N3H5Lp9gWiPPcAWu6oZDbBXM5PkIPeaNecH6AeXrEtGntIhvlys3dEJ01J3G1\\\\\\\/dyFkuHY3+zRIjz+4SKVStozitEBs034\\\\\\\/FwBQ7dkYlVP68PIpDGWjGinWMwPjTgXqLQyHuGf3MM2qJRZ++WNBBEFEiKKzQxQ7gwevtG88yv9E4ZO4ceBPTA2OJNOXLoZpHn0YTwyK3DfUHvGitO4WeLLH74oG8PUeqOipysK4aXK+KHWuq8NPYnki7WGu0bidYZhGoU2e+D1\\\\\\\/CbGwRq0iHG8vZji6\\\\\\\/KjLTgnAS2k1Qw2O5NvmpMdL4+JheWRKI6sHI7SOc9sTpigk930STEcwQ+E5NlNatKfW\\\\\\\/vBlp4b0A90P2mqABDEaW0QVbhPdGdXM7SVtM6+YXOvMRr3Ve2arWjRAIPk99VyIdGInBS7D9w9dg3O1x\\\\\\\/r0Ec4NAZ6EwRdmJyRPsaXkzeT9E8OZKBnm3ePkfHxhS065Htof9KuTSs52NjQUyHAmavmVd4WRZoEtPWUo7vKbPgi55Frt0jmO3Tn53YVzYqGNDQXIyjYm9Svj5lU9\\\\\\\/0MGci9mT\\\\\\\/C4fESD6kgl1acpURZggagenK6NZY6ewL7Fkv8baCPt8R7bPC0pzaJqJ1MHBTxK41q2XCJTYWeN0F\\\\\\\/WKDp5s3y2PT72miYVLs6rIyF565ArZKaokvqhNRav+9vIhPwhR1CHqFG9rv5HDS6DI3UK8YMXZKzRc5P+zyIrK7S0oe1dP7vHAAJRcsb62vYLalWT\\\\\\\/BreWZOGKPMY9OyfFeXk0szKJZvqeXgaZn6DwxfuKKQaV9XNXMmkrpgKyc+4Vx2CACSuZDw4oCwEMHDNMjM1JT9OHd+bc+tBzuOUfwHPpGfFaLADHDenT1\\\\\\\/lY3+dsLqzn1ABtuexiCUNVdhcYcU6YTBXyF31MVV8lrzlvRd6x1o4iNu4zikI\\\\\\\/aAAK5sWpd54pl86+vpsBKMGvSIMFM1sXZlzKubzc9sLu\\\\\\\/mQYDVRqWsKR8wxBYWEbaVXibMFcE+9xG3Cszu5oWwMl3rEEcTDq9Gw0nXVwKq3S1g+poQDOoX6psE4oSEo2F6ySN4JUkArZSBXhv6q8HU1SyLhuzBOq2KGoSUbHUM4OdS96mByjzQynn543y0d1qMK1ULwjYr8Vogs7eBwXNKzLI4wWk5C62EOVXccS+HV+p9LWfy3Ur9hTJxgNt70jhKhShazzNGHUTOe8sVJg\\\\\\\/AE\\\\\\\/VkCVpnEl\\\\\\\/NpMXjSRtQb2YdYa+\\\\\\\/gIjQfVs4KsnTTQmVi9cmR+Rnd8JrrhSJqysyGG68c9P\\\\\\\/sRiO8ZB27sjLV2uQytpIrEs3FGjcbiAtIUJbKDnMZFcreYqLI5KNHsN5hLJxKmgw3CP2u0nJPWET0u6FhBgMNAhKi\\\\\\\/NGlpQlCYCVyaUgwchfn2xV9AEoYGj6MPUvzZoRlwVv51Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022tLOnBQoaKI6ke5+0sOfHvQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022abe624c6f7169f66b86afe020e9ff5ab2752be3573e7b78580ce851f78318913\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460246 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/183" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-maXuWI_1jr9vKuqiVmIfm1DkIXe7dmo5ZHU9wq8rZYo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/183" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mSzIduox3mnF5kLK8B2XdO6m3w6F0Oe3IBqSmCDXZ64" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/171" title="On the Consequences of Purging and Linkage on Fitness and Genetic Diversity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/193" title="Identification of a Classical Mutant in the Industrial Host Aspergillus niger by Systems Genetics: LaeA Is Required for Citric Acid Production and Regulates the Formation of Some Secondary Metabolites" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460246" id="node460246" data-pisa="ggg;6/1/183" data-pisa-master="ggg;g3.115.022764" data-apath="/ggg/6/1/183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gene Expression and Chromatin Modifications Associated with Maize Centromeres</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hainan Zhao</span>, <span class="highwire-citation-author" data-delta="1">Xiaobiao Zhu</span>, <span class="highwire-citation-author" data-delta="2">Kai Wang</span>, <span class="highwire-citation-author" data-delta="3">Jonathan I. Gent</span>, <span class="highwire-citation-author" data-delta="4">Wenli Zhang</span>, <span class="highwire-citation-author" data-delta="5">R. Kelly Dawe</span> and <span class="highwire-citation-author" data-delta="6">Jiming Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">183-192;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022764 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460246" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hainan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hainan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHainan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaobiao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaobiao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaobiao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kai Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKai%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jonathan I. Gent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BI.%2BGent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gent%20JI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BI.%2BGent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wenli Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenli%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWenli%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">R. Kelly Dawe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BKelly%2BDawe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dawe%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BKelly%2BDawe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jiming Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiming%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiming%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jjiang1@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">jjiang1@wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/1/183" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460246" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460246/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/183.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460246" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460246/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/183.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460246" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460246/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/183.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460246" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460246/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/1/183.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460246" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460246/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Centromeres are defined by the presence of CENH3, a variant of histone H3. Centromeres in most plant species contain exclusively highly repetitive DNA sequences, which has hindered research on structure and function of centromeric chromatin. Several maize centromeres have been nearly completely sequenced, providing a sequence-based platform for genomic and epigenomic research of plant centromeres. Here we report a high resolution map of CENH3 nucleosomes in the maize genome. Although CENH3 nucleosomes are spaced ∼190 bp on average, CENH3 nucleosomes that occupied <em>CentC</em>, a 156-bp centromeric satellite repeat, showed clear positioning aligning with <em>CentC</em> monomers. Maize centromeres contain alternating CENH3-enriched and CENH3-depleted subdomains, which account for 87% and 13% of the centromeres, respectively. A number of annotated genes were identified in the centromeres, including 11 active genes that were located exclusively in CENH3-depleted subdomains. The euchromatic histone modification marks, including H3K4me3, H3K36me3 and H3K9ac, detected in maize centromeres were associated mainly with the active genes. Interestingly, maize centromeres also have lower levels of the heterochromatin histone modification mark H3K27me2 relative to pericentromeric regions. We conclude that neither H3K27me2 nor the three euchromatic histone modifications are likely to serve as functionally important epigenetic marks of centromere identity in maize.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cenh3" class="hw-term hw-article-keyword hw-article-keyword-cenh3" rel="nofollow">CENH3</a></li><li class="kwd"><a href="/keyword/centromere" class="hw-term hw-article-keyword hw-article-keyword-centromere" rel="nofollow">centromere</a></li><li class="kwd"><a href="/keyword/gene-expression" class="hw-term hw-article-keyword hw-article-keyword-gene-expression" rel="nofollow">gene expression</a></li><li class="kwd"><a href="/keyword/histone-modification" class="hw-term hw-article-keyword hw-article-keyword-histone-modification" rel="nofollow">histone modification</a></li><li class="kwd"><a href="/keyword/nucleosome" class="hw-term hw-article-keyword hw-article-keyword-nucleosome" rel="nofollow">nucleosome</a></li></ul><p id="p-3">Centromeric chromatin contains the histone H3 variant cenH3 (CENP-A in humans, CENH3 in maize)—a defining feature of centromeres (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Henikoff <em>et al.</em> 2001</a>). While this H3 variant makes up only a small percentage of the total H3 population in centromeres (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fukagawa and Earnshaw 2014</a>), it plays a central role in recruiting other centromere proteins. The patterns and functional significance of other histone variants and histone modifications in centromeres are much less well understood, and are likely variable between organisms. For example, centromeres of the filamentous fungi <em>Neurospora crassa</em> have heterochromatic histone modifications (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Smith <em>et al.</em> 2012</a>), while centromeres of the fission yeast <em>Schizosaccharomyces pombe</em> have euchromatic histone modifications (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cam <em>et al.</em> 2005</a>). Similarly, both euchromatic and heterochromatic histone modifications have been found in the centromeres of different plant and animal species (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sullivan and Karpen 2004</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Guenatri <em>et al.</em> 2004</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yan <em>et al.</em> 2005</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ribeiro <em>et al.</em> 2010</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wu <em>et al.</em> 2011</a>).</p><p id="p-4">Centromeres contain both H3 and cenH3 nucleosomes (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Blower <em>et al.</em> 2002</a>). Immunofluorescence staining of animal centromeres has revealed the presence of distinct domains within centromeres, some with reduced cenH3 occupancy (reviewed by <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Fukagawa and Earnshaw 2014</a>). Genome-wide mapping of sequences associated with cenH3 nucleosomes in several plant species revealed distinct cenH3-enriched and cenH3-depleted subdomains in individual centromeres (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yan <em>et al.</em> 2008</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gong <em>et al.</em> 2012</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gent <em>et al.</em> 2012</a>). In rice centromeres, several euchromatic histone modification marks were associated with subdomains that are depleted of cenH3 nucleosomes. These subdomains also contain a low density of transcribed genes (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Yan <em>et al.</em> 2005</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">2006</a>). The CENH3-binding domains of maize centromeres expand from ∼1.8 Mb to ∼3.6 Mb when the maize chromosomes are transferred into the genetic background of oat (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>). Interestingly, the centromeres appeared to expand preferentially into flanking regions that lack genes. These results suggest that cenH3 chromatin is not compatible with transcription. Another striking character associated with centromeric chromatin is that cenH3 nucleosomes appear to protect less DNA from MNase digestion than canonical nucleosomes. The “point” centromeres of budding yeast <em>Saccharomyces cerevisiae</em> contain a single cenH3 nucleosome consisting of ∼120 bp centromeric DNA sequence (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Krassovsky <em>et al.</em> 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Henikoff and Henikoff 2012</a>). Similarly, the cenH3 nucleosomes in humans and rice protect ∼90–130 bp sequences from MNase digestion (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hasson <em>et al.</em> 2013</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhang <em>et al.</em> 2013</a>). In addition, cenH3 nucleosomes were found to be highly phased on the centromeric satellite repeats in both humans (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hasson <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Henikoff <em>et al.</em> 2015</a>) and rice (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Zhang <em>et al.</em> 2013</a>).</p><p id="p-5">The maize genome has been sequenced based on a minimum tilling path of bacterial artificial chromosomes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Schnable <em>et al.</em> 2009</a>). Several maize centromeres contain only limited amounts of the centromeric satellite repeats, and represent some of the best sequenced centromeres in multicellular eukaryotes (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wolfgruber <em>et al.</em> 2009</a>). These well sequenced centromeres provide us a platform to profile chromatin modifications and RNA expression patterns associated with maize centromeres. We developed a high resolution genome-wide map of maize CENH3 nucleosomes based on Illumina sequencing of DNA samples prepared from chromatin immunoprecipitation (ChIP) using a maize anti-CENH3 antibody. This map has delineated fine boundaries of alternating CENH3-enriched and CENH3-depleted subdomains in seven of the 10 maize centromeres. We identified a set of 34 annotated genes in the centromeres. However, only 11 genes located in CENH3-depleted subdomains were transcriptionally active. These active genes were associated with euchromatic histone modification marks. In addition, maize centromeres lacked both euchromatic histone modifications (except for the regions associated with active genes) and H3K27me2, a mark of heterochromatin in maize. We demonstrate that deep CENH3 ChIP-seq datasets allow mapping of CENH3 nucleosome positioning and dynamics in maize.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>MNase digestion, ChIP, and ChIP-seq</h3><p id="p-6">Ten-day-old maize (B73) seedlings grown in the greenhouse were collected and ground into fine powder in liquid nitrogen for nuclei extraction. The input chromatin for anti-CENH3 ChIP experiments was prepared from purified nuclei digested with 0.2 unit (U) and 5 U of micrococcal nuclease (MNase; Sigma-Aldrich, N5386_200U), respectively. ChIP experiments, including nuclei purification, followed published protocols (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Zhang <em>et al.</em> 2012</a>). Both ChIPed DNA and input DNA corresponding to 0.2 U and 5 U MNase trimmed chromatin input were used for ChIP-seq library preparation following published methods (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Henikoff <em>et al.</em> 2011</a>) to recover and sequence small DNA fragments. The DNA libraries were sequenced using the Illumina HiSequation 2000 platform. For MNase digestion of naked DNA, DNA was extracted from a B73 developing ear using a DNeasy Plant Mini Kit (Qiagen), and digested with MNase (New England Biolabs, #M0247S) as follows: a master mix consisting of 140 ng/μl DNA, 82 gel units/μl MNase, 150 ng/μl BSA in 1X micrococcal nuclease buffer was prepared then divided into five 150-μl aliquots on ice. All aliquots were simultaneously transferred to a 37° water bath then removed one at a time after 1, 4, and 8 min (at which time the bulk of DNA had been digested to fragments of less than 200 bp in length). The digested was purified and prepared for Illumina sequencing as described (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gent <em>et al.</em> 2014</a>).</p></div><div id="sec-3" class="subsection"><h3>Gene expression analysis using quantitative real-time PCR</h3><p id="p-7">RNA was extracted from 10-day-old maize (B73) seedlings using an RNeasy Plant Mini Kit (Qiagen, 74904) following the manufacturer’s instructions. RNA samples were treated with TURBO DNase (Ambion, AM1907), and reverse transcribed with SuperScript III (Invitrogen, 18080-051). The expression levels of a set of seven centromeric genes (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS1.pdf">Table S1</a>) relative to the <em>Actin</em> reference gene were validated by both semiquantitative real-time PCR (semiqRT-PCR) and quantitative real-time PCR (qRT-PCR) carried out on a Thermal Cycler (Bio-Rad, C1000) using rTaq DNA polymerase (Takara, R001A), and on MJ Research Opticon 2 (Bio-Rad Laboratories) using the SYBR Advantage qPCR Premix (Clontech, 639676), respectively. Annealing temperature was set to 60°. SemiqRT-PCR and qRT-PCR primers are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS1.pdf">Table S1</a>.</p></div><div id="sec-4" class="subsection"><h3>Sequence mapping and identification of CENH3-binding domains</h3><p id="p-8">Raw sequencing reads were trimmed by Cutadapt (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Martin 2011</a>) to remove low quality nucleotides (with quality score less than 30) and adapters. Pairs with reads of less than 26 bp in length for either read after quality and adapter trimming were discarded. Reads were mapped to the B73 reference genome (RefGen_v3, <a href="http://plants.ensembl.org/Zea_mays">http://plants.ensembl.org/Zea_mays</a>) by BWA-MEM using default parameters (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li 2013</a>). Only uniquely-mapped reads were retained for subsequent analyses (mapping quality of at least 20 and perfect matches) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2009</a>). CENH3 ChIP-seq data from the 0.2 U and 5 U MNase-digested samples were analyzed independently. To analyze CENH3 enrichment, each chromosome was divided into 10-kb nonoverlap windows. The number of ChIP-seq reads in each window was normalized using input reads. CENH3 enrichment of each window was determined by comparing the number of reads from ChIPed and input data. CENH3-binding domains were identified by comparing the ChIPed and input data using SICER (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zang <em>et al.</em> 2009</a>), which is designed to identify ChIP-enriched signals that tend to form specific subdomains. The parameters for SICER included a window size of 1000 bp, gap size of 3000 bp, effective genome fraction 0.7, redundancy threshold 1, fragment size 150 bp, and false discovery rate (FDR) 0.05. To determine the sizes and positions of centromeres, we merged CENH3-binding domains that were separated by less than 100 kb to eliminate the gaps that arise because of insufficient mapping at highly repetitive regions. Small CENH3-binding subdomains of less than 30 kb were discarded.</p></div><div id="sec-5" class="subsection"><h3>Transcriptome, histone modification, and CENH3-depleted subdomains</h3><p id="p-9">Transcriptome (SRR445245 and SRR445382) and ChIP data of H3K27me3, H3K36me3, H3K4me3 and H3K9ac (GSE1528) (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2009</a>), and H3K27me2 (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gent <em>et al.</em> 2014</a>) (SRR1584368 and SRR1584369) were downloaded from NCBI (<a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>). Transcriptome reads were mapped using TopHat (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Trapnell <em>et al.</em> 2009</a>), and ChIP reads were mapped using BWA-MEM with default parameters (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Li 2013</a>). Only uniquely mapped reads were retained. Gene expression in centromeres was analyzed by Cufflinks (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Trapnell <em>et al.</em> 2010</a>) using gene annotation AGPv3.21 (<a href="http://plants.ensembl.org/Zea_mays">http://plants.ensembl.org/Zea_mays</a>) with default parameters. We used MACS2 (-B –broad –g 740000000–trackline) (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zhang <em>et al.</em> 2008</a>) and SICER (window size 200 bp, gap size 400 bp, effective genome fraction 0.4, redundancy threshold 1, fragment size 150 bp, and FDR 0.05) to identify histone modification peaks in centromeres, and retained peaks that were consistently discovered by both algorithms for further analysis. To analyze the distributions of each histone modification, the genome was divided into 100-kb windows. Mappable regions within the windows were determined by mapping simulated reads to genome sequence. Read density was calculated as the number of reads in the mappable regions of each window.</p><p id="p-10">To identify CENH3-depleted subdomains in the centromeres, we divided the B73 genome into 10-kb windows and counted uniquely-mapped reads from ChIP and input (0.2 U and 5 U libraries combined). ChIP read counts were then normalized by input to calculate enrichment for each 10-kb window. Read count distribution of the input data were generated. For each window, read count of the ChIPed data was tested against the input distribution. Windows with a <em>p</em>-value less than 1 × 10<sup>−5</sup> were considered as CENH3-depleted subdomains. Some regions may have fewer reads due to inefficient mapping at repetitive regions rather than paucity of CENH3. To overcome a potential mapping bias caused by repetitive DNA sequences, we generated 150-bp reads randomly from the B73 genome, and aligned these reads to the genome to identify mappable 10-kb windows in centromeres. CENH3-depleted subdomains overlapped to mappable regions were retained. Portions of CENH3-depleted subdomains that overlapped with unmappable regions were discarded. In addition, we discarded CENH3-depleted subdomains located at the ends of centromeres (less than 200 kb to the ends of centromere boundaries) as these regions may not represent CENH3 depletion.</p><p id="p-11">To analyze the relationship between histone modification and maize centromere expansion in oat, we divided the expanded regions of <em>Cen2</em>, <em>Cen5</em>, <em>Cen9</em>, and <em>Cen10</em> into 1-kb windows and calculated read density of H3K4me2, H3K36me3, H3K9ac, H3K27me3, and H3K27me2 in each window. Read densities were then normalized by the mappability of each window. Read densities of nonexpanded regions were calculated in the same manner. The significant differences of each histone modification between expanded regions and nonexpanded regions were determined by bootstrap resampling of read densities from expanded regions and nonexpanded regions.</p></div><div id="sec-6" class="subsection"><h3>Nucleosome positioning and phasing on <em>CentC</em></h3><p id="p-12">We analyzed the nucleosome positioning using nucleR (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Flores and Orozco 2011</a>) with slight modification. Reads mapped to centromeres were extracted and trimmed to 40 bp around the center of each pair. Read depth of each position in centromeres was then determined by BEDtools (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Quinlan and Hall 2010</a>). Read depth was analyzed by nucleR with parameter “pcKeepComp” set to 0.01. “pcKeepComp” was set to 0.02 to identify closely located peaks on <em>CentC</em> arrays. We annotated the maize satellite repeat <em>CentC</em> using CENSOR (-nofilter –s –bprm ‘-filter = none’) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kohany <em>et al.</em> 2006</a>). The consensus <em>CentC</em> sequence was generated by CAP3 (-j 200 -s 1000 -o 100 -p 80) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Huang and Madan 1999</a>) using ∼9850 (82% of total <em>CentC</em> monomers) <em>CentC</em> monomers. Paired-end reads were merged by FLASH (-M 100 –z –t 1) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Magoc and Salzberg 2011</a>), and then mapped to the B73 genome and a <em>CentC</em> tetramer using BWA-MEM (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Zhang <em>et al.</em> 2013</a>). Phasing of CENH3 nucleosomes was analyzed using previously described methods (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Zhang <em>et al.</em> 2013</a>).</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-13">The CENH3 ChIP-seq sequencing data are available from NCBI sequence read archive (SRA) under accession SRP057754, the MNase-digested maize genomic DNA sequencing data are under accession SRP049952 (<a href="http://www.ncbi.nlm.nih.gov/sra">http://www.ncbi.nlm.nih.gov/sra</a>).</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>High-resolution mapping of CENH3-binding domains of maize centromeres</h3><p id="p-14">We conducted two independent ChIP experiments using anti-CENH3 antibody (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zhong <em>et al.</em> 2002</a>). Chromatin isolated from B73 seedling tissue was digested by either 0.2 U or 5 U of MNase. The sample treated with 5U MNase was nearly fully digested (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Mono-nucleosomal DNA fragments of approximately 100–200 bp were gel-purified from both samples for library construction. A total of four libraries, two from ChIPed DNA and two from input DNA, were sequenced using the Illumina HiSequation 2000 platform, producing a total of 283 million 100-bp paired-end reads from the four libraries (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS2.pdf">Table S2</a>). Sequence reads were mapped to maize B73 genome (version 3) (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Schnable <em>et al.</em> 2009</a>). Only uniquely mapped reads with perfect matches to the B73 genome were retained for further analysis.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/183/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromatin immunoprecipitation (ChIP) and sequencing of maize centromeres. (A) Agarose gel electrophoresis of DNAs isolated from maize chromatin digested by 0.2&#xA0;U and 5&#xA0;U micrococcal nuclease (MNase), respectively. Libraries were developed from both ChIPed and input DNA samples of both chromatin samples. (B) Distribution of fragment lengths from 0.2&#xA0;U and 5&#xA0;U ChIPed and input sequencing libraries. The x-axis represents the length of fragments (bp). The y-axis represents the percentage of reads in specific length." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-656974181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromatin immunoprecipitation (ChIP) and sequencing of maize centromeres. (A) Agarose gel electrophoresis of DNAs isolated from maize chromatin digested by 0.2&#xA0;U and 5&#xA0;U micrococcal nuclease (MNase), respectively. Libraries were developed from both ChIPed and input DNA samples of both chromatin samples. (B) Distribution of fragment lengths from 0.2&#xA0;U and 5&#xA0;U ChIPed and input sequencing libraries. The &lt;em&gt;x&lt;/em&gt;-axis represents the length of fragments (bp). The &lt;em&gt;y&lt;/em&gt;-axis represents the percentage of reads in specific length.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/183/F1.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/1/183/F1.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/183/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/183/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467463" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Chromatin immunoprecipitation (ChIP) and sequencing of maize centromeres. (A) Agarose gel electrophoresis of DNAs isolated from maize chromatin digested by 0.2 U and 5 U micrococcal nuclease (MNase), respectively. Libraries were developed from both ChIPed and input DNA samples of both chromatin samples. (B) Distribution of fragment lengths from 0.2 U and 5 U ChIPed and input sequencing libraries. The <em>x</em>-axis represents the length of fragments (bp). The <em>y</em>-axis represents the percentage of reads in specific length.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">Approximately 23% of the uniquely-mapped reads from the 0.2 U and 11% from the 5 U libraries were mapped to the centromeres (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS2.pdf">Table S2</a>), which represented 45-fold and 27-fold CENH3 ChIP sequence enrichments in centromeres compared to input. The ChIP-seq read density across the maize genome was highly correlated between the two libraries (<em>R</em> = 0.90, <em>p</em> &lt; 2.2 × 10<sup>−16</sup>, Spearman’s rank correlation). The lengths of the uniquely-mapped sequence reads from 0.2 U ChIP and input libraries peaked at 158 bp and 159 bp, respectively, <em>i.e.</em>, longer than the 147 bp that would wrap the canonical nucleosome core particle. These results indicate that the 0.2 U chromatin sample was underdigested and most reads contained linker sequences. The lengths of the sequence reads from the 5 U ChIP and input libraries peaked at 147 bp, with two minor peaks at 123 bp and 133 bp, respectively (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The minor peaks were also clearly visible in the input, of which only a small percentage of the reads is expected to be derived from CENH3-containing nucleosomes. Thus, these smaller fragments are abundant independently of CENH3, but are particularly enriched by CENH3 ChIP.</p><p id="p-17">We divided each chromosome into 10-kb nonoverlapping windows and calculated ChIP-seq read enrichment within each window. Unambiguous enrichment peaks were observed in centromeric regions of chromosomes 2, 3, 4, 5, 8, 9, and 10 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/FigureS1.pdf">Figure S1</a>), consistent with previous centromere mapping results (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Wolfgruber <em>et al.</em> 2009</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gent <em>et al.</em> 2015</a>). Several minor peaks were also observed in chromosomal arms (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/FigureS1.pdf">Figure S1</a>), which may result from misassembly or misplacement of centromere-associated repetitive DNA sequences. We observed very narrow peaks in centromeres 6 and 7, and no unambiguous peak in centromere 1, suggesting that these three centromeres contain mainly highly repetitive DNA sequences. Indeed, fiber-FISH mapping revealed that both <em>Cen1</em> and <em>Cen7</em> contain megabase-sized arrays of the <em>CentC</em> satellite repeats (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jin <em>et al.</em> 2004</a>). We excluded <em>Cen1</em>, <em>Cen6</em>, and <em>Cen7</em> in further analysis.</p><p id="p-18">We used SICER (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Zang <em>et al.</em> 2009</a>) to identify CENH3-binding domains using the 0.2 U and 5 U ChIP-seq data, independently. The CENH3-binding regions identified by the two datasets were highly correlated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS3.pdf">Table S3</a>). The consensus CENH3-binding regions identified in both datasets were then considered as the functional centromeres. We considered the SICER-defined peaks of CENH3 enrichment as the centromere of each chromosome (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The CENH3-binding domains ranged from 0.93 Mb of <em>Cen4</em> to 1.88 Mb for both <em>Cen5</em> and <em>Cen8</em>. This range, however, may not accurately reflect the true variation of centromere size because highly repetitive sequences in the centromeres are either missing in the reference genome or/and the relevant ChIP-seq reads were not mappable. <em>Cen2</em> and <em>Cen5</em>, the two best assembled maize centromeres, spanned 1.82 and 1.88 Mb, respectively.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467506/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467506/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The positions of the centromeres in maize (B73) genome</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>CENH3-depleted subdomains in maize centromeres</h3><p id="p-20">The CENH3 ChIP-seq reads were not evenly enriched in maize centromeres, which could be related to the lack of unique sequence in some regions in centromeres, and/or could indicate the existence of intermingled CENH3-enriched and CENH3-depleted subdomains as reported in several animals and plant species (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Blower <em>et al.</em> 2002</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Yan <em>et al.</em> 2008</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gong <em>et al.</em> 2012</a>). To profile CENH3 binding in maize centromeres, we first determined each 10-kb window of each centromere as mappable or unmappable based on mapping 150-bp sequence reads that were generated computationally from the maize genome. Unmappable 10-kb windows in the centromeres were excluded because the CENH3-binding in these regions cannot be determined. A total of 1350 Mb mappable regions were identified in maize genome, including 6.5 Mb in centromeres, which comprised 53% of the identified centromeric sequences. We then divided the centromeres into 10-kb windows, and determined the CENH3 read enrichment of each window by comparing CENH3 ChIP-seq data to the input data (see <em>Materials and Methods</em>). We identified a total of 846 kb of CENH3-depleted subdomains (see <em>Materials and Methods</em> for definition of a CENH3-depleted subdomain) within the seven maize centromeres, which consisted of 13% of mappable centromeric regions (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The CENH3-depleted subdomains were interspersed among CENH3-enriched regions in centromeres (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The percentages of CENH3-depleted subdomains were different among centromeres, ranging from 9.9% (<em>Cen5</em>) to 26.2% (<em>Cen3</em>) of the mappable sequences (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). For this and all subsequent analyses, we combined both the 0.2 U and 5 U ChIP datasets into a single set.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467392/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467392/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of CENH3-depleted subdomains and histone modifications in maize centromeres</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/183/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Position and structure of maize Cen3. (A) The position of Cen3 was mapped between 99.80&#xA0;Mb and 100.79&#xA0;Mb. The y-axis represents fold enrichment of CENH3 ChIP-seq reads. (B) A CENH3-depleted subdomain (100.26&#x2013;100.32&#xA0;Mb) within Cen3 is exemplified. This subdomain is mainly occupied by an active gene GRMZM2G409893. ChIP-seq reads from euchromatic histone modification marks H3K36me3, H3K9ac, and H3K4me3 within this subdomain were exclusively associated with the gene." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-656974181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Position and structure of maize &lt;em&gt;Cen3&lt;/em&gt;. (A) The position of &lt;em&gt;Cen3&lt;/em&gt; was mapped between 99.80&#xA0;Mb and 100.79&#xA0;Mb. The &lt;em&gt;y&lt;/em&gt;-axis represents fold enrichment of CENH3 ChIP-seq reads. (B) A CENH3-depleted subdomain (100.26&#x2013;100.32&#xA0;Mb) within &lt;em&gt;Cen3&lt;/em&gt; is exemplified. This subdomain is mainly occupied by an active gene &lt;em&gt;GRMZM2G409893&lt;/em&gt;. ChIP-seq reads from euchromatic histone modification marks H3K36me3, H3K9ac, and H3K4me3 within this subdomain were exclusively associated with the gene.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/183/F2.medium.gif" width="440" height="367"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/1/183/F2.medium.gif" width="440" height="367"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/183/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/183/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467350" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Position and structure of maize <em>Cen3</em>. (A) The position of <em>Cen3</em> was mapped between 99.80 Mb and 100.79 Mb. The <em>y</em>-axis represents fold enrichment of CENH3 ChIP-seq reads. (B) A CENH3-depleted subdomain (100.26–100.32 Mb) within <em>Cen3</em> is exemplified. This subdomain is mainly occupied by an active gene <em>GRMZM2G409893</em>. ChIP-seq reads from euchromatic histone modification marks H3K36me3, H3K9ac, and H3K4me3 within this subdomain were exclusively associated with the gene.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Nucleosome positioning in maize centromeres</h3><p id="p-24">Although CENH3 ChIP-seq datasets were developed previously (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Wolfgruber <em>et al.</em> 2009</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gent <em>et al.</em> 2015</a>), we produced the highest volume of ChIP-seq reads from a specific maize genotype. This high volume of paired-end reads provided an opportunity to investigate the positioning of individual maize CENH3 nucleosomes. Each pair of sequence reads was first merged into a single DNA fragment, which provided information on the size and boundaries of individual DNA fragment that wrap single CENH3 nucleosomes. Approximately 93–95% of the paired reads were merged to a single fragment from the two libraries. Mapping merged fragments to the B73 genome revealed many sharp peaks in centromeres, which represent well-positioned CENH3 nucleosomes (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/183/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Positioning of CENH3 nucleosomes and MNase cutting patterns associated with CentC repeat. (A) Well-positioned CENH3 nucleosomes in Cen2, as indicated by CENH3 ChIP-seq reads. Top panel, a region containing an array of 20 CentC monomers marked by green bars. Most peaks were phased with individual CentC monomers. Bottom panel: a region containing Ty3/gypsy LTR retrotransposon sequences. Most CENH3 nucleosomes were well positioned but were not phased. (B) MNase cutting pattern on a consensus CentC repeat monomer. Vertical bars represent the frequency of cutting at each position on the CentC monomer. The y-axis shows the number of cuts in ChIPed DNA and naked DNA datasets. The x-axis shows each site of the consensus CentC monomer (156&#xA0;bp). Black dashed lines mark the top 20% cutting frequency. (C) Phasogram of CENH3 nucleosomes in Cen2. Total Cen2: all fragments mapped to Cen2 were used for the phasogram; CentC: only fragments mapped to CentC arrays were calculated for the phasogram. The y-axes is the frequency of distance between the midpoints of two fragments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-656974181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Positioning of CENH3 nucleosomes and MNase cutting patterns associated with &lt;em&gt;CentC&lt;/em&gt; repeat. (A) Well-positioned CENH3 nucleosomes in &lt;em&gt;Cen2&lt;/em&gt;, as indicated by CENH3 ChIP-seq reads. Top panel, a region containing an array of 20 &lt;em&gt;CentC&lt;/em&gt; monomers marked by green bars. Most peaks were phased with individual &lt;em&gt;CentC&lt;/em&gt; monomers. Bottom panel: a region containing &lt;em&gt;Ty3&lt;/em&gt;/&lt;em&gt;gypsy&lt;/em&gt; LTR retrotransposon sequences. Most CENH3 nucleosomes were well positioned but were not phased. (B) MNase cutting pattern on a consensus &lt;em&gt;CentC&lt;/em&gt; repeat monomer. Vertical bars represent the frequency of cutting at each position on the &lt;em&gt;CentC&lt;/em&gt; monomer. The &lt;em&gt;y&lt;/em&gt;-axis shows the number of cuts in ChIPed DNA and naked DNA datasets. The &lt;em&gt;x&lt;/em&gt;-axis shows each site of the consensus &lt;em&gt;CentC&lt;/em&gt; monomer (156&#xA0;bp). Black dashed lines mark the top 20% cutting frequency. (C) Phasogram of CENH3 nucleosomes in &lt;em&gt;Cen2&lt;/em&gt;. Total &lt;em&gt;Cen2&lt;/em&gt;: all fragments mapped to &lt;em&gt;Cen2&lt;/em&gt; were used for the phasogram; &lt;em&gt;CentC&lt;/em&gt;: only fragments mapped to &lt;em&gt;CentC&lt;/em&gt; arrays were calculated for the phasogram. The &lt;em&gt;y&lt;/em&gt;-axes is the frequency of distance between the midpoints of two fragments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/183/F3.medium.gif" width="440" height="231"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/1/183/F3.medium.gif" width="440" height="231"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/183/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/183/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467391" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Positioning of CENH3 nucleosomes and MNase cutting patterns associated with <em>CentC</em> repeat. (A) Well-positioned CENH3 nucleosomes in <em>Cen2</em>, as indicated by CENH3 ChIP-seq reads. Top panel, a region containing an array of 20 <em>CentC</em> monomers marked by green bars. Most peaks were phased with individual <em>CentC</em> monomers. Bottom panel: a region containing <em>Ty3</em>/<em>gypsy</em> LTR retrotransposon sequences. Most CENH3 nucleosomes were well positioned but were not phased. (B) MNase cutting pattern on a consensus <em>CentC</em> repeat monomer. Vertical bars represent the frequency of cutting at each position on the <em>CentC</em> monomer. The <em>y</em>-axis shows the number of cuts in ChIPed DNA and naked DNA datasets. The <em>x</em>-axis shows each site of the consensus <em>CentC</em> monomer (156 bp). Black dashed lines mark the top 20% cutting frequency. (C) Phasogram of CENH3 nucleosomes in <em>Cen2</em>. Total <em>Cen2</em>: all fragments mapped to <em>Cen2</em> were used for the phasogram; <em>CentC</em>: only fragments mapped to <em>CentC</em> arrays were calculated for the phasogram. The <em>y</em>-axes is the frequency of distance between the midpoints of two fragments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We observed sharp peaks associated with individual <em>CentC</em> monomers in <em>Cen2</em> (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>), indicating that CENH3 nucleosomes preferentially bind to <em>CentC</em> repeats such that the same histone-DNA contacts are formed; this phenomenon is termed nucleosome phasing and is related to innate sequence preferences of histones. Phasing of CENH3 on <em>CentC</em> arrays was further evidenced by phasograms of <em>CentC</em> arrays with a periodicity of approximately the same length as the <em>CentC</em> monomer ∼156 bp (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a><strong>)</strong>. In contrast, we did not detect a similar nucleosome phasing associated with non-<em>CentC</em> sequences in <em>Cen2</em>. For example, although positioned CENH3 nucleosomes were associated with retrotransposon-related sequences (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>), these nucleosomes did not show any specific periodicity as <em>CentC</em>-associated nucleosomes.</p><p id="p-27">We next examined if the cutting site pattern associated with the <em>CentC</em> repeats was caused by sequence preference of MNase using a large sequence dataset derived from MNase-digested B73 genomic DNA. The distribution patterns of the cutting sites on the <em>CentC</em> repeat were clearly different between naked DNA and ChIPed DNA (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>). The cutting sites on naked <em>CentC</em> biased to A/T sites, consistent with the sequence preference of MNase (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Horz and Altenburger 1981</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Drew 1984</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Telford and Stewart 1989</a>). By contrast, the cutting sites of ChIPed DNA did not show A/T preferences (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>). Among the top 20% most frequently cut sites on the <em>CentC</em> repeat, 92% were A or T on naked DNA, but only 45% were A or T on ChIPed DNA.</p></div><div id="sec-12" class="subsection"><h3>Transcription of genes in maize centromeres</h3><p id="p-28">CENH3-depleted subdomains in rice centromeres are occupied by canonical nucleosomes and contain active genes (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Wu <em>et al.</em> 2011</a>). We next investigated the transcriptional activity associated with maize centromeres. A total of 34 protein-coding genes were annotated in seven maize centromeres, of which 21 were located in CENH3-depleted subdomains, and 13 located in CENH3-enriched regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS4.pdf">Table S4</a>). Eleven of the 21 genes (52%) in CENH3-depleted subdomains were expressed in shoot and/or root tissues based on RNA-seq datasets (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2009</a>). In contrast, transcription was not detected for any of the 13 genes located in CENH3-enriched regions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS4.pdf">Table S4</a>). Six of 13 inactive genes located in the CENH3-enriched regions were shorter than 100 amino acids, and lack homologs (≥70% identity) in other plant species, suggesting that these are likely nonfunctional genes.</p><p id="p-29">Maize centromeres also included a total of nine genes annotated as “low confidence genes” or “novel transcript” that lack homologs in other plant species, or lack evidence of transcription and/or intact coding capacity (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS4.pdf">Table S4</a>). Analysis of RNA-seq datasets revealed potential transcription associated with eight of the low confidence genes as well as one “novel transcript” in <em>Cen2</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS4.pdf">Table S4</a>). Only two of these genes, which showed low levels of transcription (FPKM of 1.17 and 1.23, respectively) in roots (but not in leaf), were located in CENH3-enriched regions.</p><p id="p-30">We selected seven centromeric genes to validate their transcription. These genes have different expression levels based on RNA-seq data. Transcription of all seven genes in seedlings was detected by both qRT-PCR and semiqRT-PCR, including genes <em>GRMZM2G175425</em> and <em>GRMZM2G137715</em> with FPKM 1.1 and 2.3, respectively (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The expression of a low confidence gene <em>GRMZM5G820434</em> (FPKM 5.6) was also confirmed.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/183/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Validation of expression of centromeric genes in maize. (A) qRT-PCR validation of a set of seven centromeric genes. RNAs were isolated from 10-day-old seedlings of B73. The y-axis represents the relative expression level normalized by setting the expression of the Actin gene to 100. (B) SemiqRT-PCR analysis of the same set of centromeric genes. Validated genes include: Lane 2: GRMZM2G040843 (FPKM 22.8); Lane 3: GRMZM2G175425 (FPKM 1.1); Lane 4: GRMZM2G137715 (FPKM 2.3); Lane 5: GRMZM5G820434 (FPKM 5.6); Lane 6 GRMZM2G071042 (FPKM 13.7); Lane 7: GRMZM2G409893 (FPKM 5.7); and Lane 8: GRMZM2G083935 (FPKM 3.0). Note: the levels of expression of each gene correlated in qRT-PCR and semiqRT-PCR experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-656974181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Validation of expression of centromeric genes in maize. (A) qRT-PCR validation of a set of seven centromeric genes. RNAs were isolated from 10-day-old seedlings of B73. The &lt;em&gt;y&lt;/em&gt;-axis represents the relative expression level normalized by setting the expression of the &lt;em&gt;Actin&lt;/em&gt; gene to 100. (B) SemiqRT-PCR analysis of the same set of centromeric genes. Validated genes include: Lane 2: &lt;em&gt;GRMZM2G040843&lt;/em&gt; (FPKM 22.8); Lane 3: &lt;em&gt;GRMZM2G175425&lt;/em&gt; (FPKM 1.1); Lane 4: &lt;em&gt;GRMZM2G137715&lt;/em&gt; (FPKM 2.3); Lane 5: &lt;em&gt;GRMZM5G820434&lt;/em&gt; (FPKM 5.6); Lane 6 &lt;em&gt;GRMZM2G071042&lt;/em&gt; (FPKM 13.7); Lane 7: &lt;em&gt;GRMZM2G409893&lt;/em&gt; (FPKM 5.7); and Lane 8: &lt;em&gt;GRMZM2G083935&lt;/em&gt; (FPKM 3.0). Note: the levels of expression of each gene correlated in qRT-PCR and semiqRT-PCR experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/183/F4.medium.gif" width="440" height="422"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/1/183/F4.medium.gif" width="440" height="422"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/183/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/183/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467344" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Validation of expression of centromeric genes in maize. (A) qRT-PCR validation of a set of seven centromeric genes. RNAs were isolated from 10-day-old seedlings of B73. The <em>y</em>-axis represents the relative expression level normalized by setting the expression of the <em>Actin</em> gene to 100. (B) SemiqRT-PCR analysis of the same set of centromeric genes. Validated genes include: Lane 2: <em>GRMZM2G040843</em> (FPKM 22.8); Lane 3: <em>GRMZM2G175425</em> (FPKM 1.1); Lane 4: <em>GRMZM2G137715</em> (FPKM 2.3); Lane 5: <em>GRMZM5G820434</em> (FPKM 5.6); Lane 6 <em>GRMZM2G071042</em> (FPKM 13.7); Lane 7: <em>GRMZM2G409893</em> (FPKM 5.7); and Lane 8: <em>GRMZM2G083935</em> (FPKM 3.0). Note: the levels of expression of each gene correlated in qRT-PCR and semiqRT-PCR experiments.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Histone modifications associated with CENH3-depleted subdomains</h3><p id="p-32">We next investigated the histone modification patterns associated with maize centromeres. Genome-wide datasets of active histone marks H3K4me3, H3K36me3, and H3K9ac, and repressive histone mark H3K27me3 were developed using maize shoot tissue (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2009</a>). We identified 64 peaks in seven centromeres (see <em>Materials and Methods</em>), which were associated with one to four of these histone marks (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS5.xlsx">Table S5</a>). Most of these peaks (62 of 64) were associated with the three active marks, and only two peaks associated with the repressive mark H3K27me3. In addition, most peaks (54 of 64) were found in CENH3-depleted subdomains. Interestingly, 47 of the 54 peaks located in CENH3-depleted subdomains were associated with annotated genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS5.xlsx">Table S5</a>). H3K36me3 was the most prevalent modification found in the CENH3-depleted subdomains (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). Of the 20 transcribed centromeric genes, 17 (85%) were associated with active histone modifications (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/TableS5.xlsx">Table S5</a>). H3K4me3 and H3K9ac are generally enriched around transcription start sites of most maize genes. By contrast, H3K36me3 is more broadly associated with gene bodies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/FigureS2.pdf">Figure S2</a>). Thus, the centromeric genes showed similar histone modification patterns as most maize genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/FigureS2.pdf">Figure S2</a>). These results indicate that CENH3-depleted subdomains in maize centromeres are occupied by canonical nucleosomes and share similar chromatin properties with euchromatic regions in the maize genome.</p><p id="p-33">H3K27me2 is a heterochromatic histone mark and is enriched with repetitive elements in the maize genome (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Shi and Dawe 2006</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Gent <em>et al.</em> 2014</a>). Interestingly, H3K27me2 was generally depleted in centromeres compared to the flanking pericentromeric regions (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/FigureS3.pdf">Figure S3</a>). The reduced H3K27me2 in centromeres relative to pericentromeres was not due to mappability of the sequences in centromeric regions because there was no mapping bias of random reads generated from the B73 genome (see <em>Materials and Methods</em>). These results were also in agreement with previous cytological studies, in which H3K27me2 signals were detectable in centromeres, but were relatively weaker than in pericentromeric regions (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Shi and Dawe 2006</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jin <em>et al.</em> 2008</a>). In addition, H3K27me2 was also significantly reduced in the CENH3-enriched subdomains relative to the CENH3-depleted subdomains (Kolmogorov-Smirnov test, one-tailed) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/FigureS4.pdf">Figure S4</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/183/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of sequencing reads associated with histone modifications and CENH3 in the centromeric and pericentromeric regions of maize chromosome 5. Chromosome 5 was divided into 100-kb windows, and read numbers of CENH3, H3K4me3, H3K27me2, H3K9ac, H3K27me3, H3K36me3 and randomly generated 150-bp reads were calculated in each window. Note the depletion of H3K27me2 within Cen5 as compared to the pericentromeric region. The left y-axis represents the numbers of CENH3 ChIP-seq and random reads in each 100-kb window. The right y-axis represents the relative read density associated with different histone modifications." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-656974181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of sequencing reads associated with histone modifications and CENH3 in the centromeric and pericentromeric regions of maize chromosome 5. Chromosome 5 was divided into 100-kb windows, and read numbers of CENH3, H3K4me3, H3K27me2, H3K9ac, H3K27me3, H3K36me3 and randomly generated 150-bp reads were calculated in each window. Note the depletion of H3K27me2 within &lt;em&gt;Cen5&lt;/em&gt; as compared to the pericentromeric region. The left &lt;em&gt;y&lt;/em&gt;-axis represents the numbers of CENH3 ChIP-seq and random reads in each 100-kb window. The right &lt;em&gt;y&lt;/em&gt;-axis represents the relative read density associated with different histone modifications.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/183/F5.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/1/183/F5.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/183/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/183/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467416" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-34" class="first-child">Distribution of sequencing reads associated with histone modifications and CENH3 in the centromeric and pericentromeric regions of maize chromosome 5. Chromosome 5 was divided into 100-kb windows, and read numbers of CENH3, H3K4me3, H3K27me2, H3K9ac, H3K27me3, H3K36me3 and randomly generated 150-bp reads were calculated in each window. Note the depletion of H3K27me2 within <em>Cen5</em> as compared to the pericentromeric region. The left <em>y</em>-axis represents the numbers of CENH3 ChIP-seq and random reads in each 100-kb window. The right <em>y</em>-axis represents the relative read density associated with different histone modifications.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Chromatin features associated with expanded maize centromeres in the genetic background of oat</h3><p id="p-35">The CENH3-binding domains of maize centromeres expand from ∼1.8 Mb to 3.6 Mb in the genetic background of oat to adapt to a similar size as oat centromeres (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>). Interestingly, several centromeres, including <em>Cen2</em>, <em>Cen5</em>, <em>Cen9</em>, and <em>Cen10</em>, expand exclusively toward a single direction, either in the short or in the long arm, suggesting that the expansion direction is not random. The expanded regions appear to be relatively deficient for active genes (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>).</p><p id="p-36">To further explore the potential barriers that prohibit maize centromere expansion in oat background, we compared the histone modification patterns associated with the expanded region flanking one side of the original centromere (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6, A–D</a>, regions marked by red bars) with the patterns associated with nonexpanded region flanking the other side of the centromere (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6, A–D</a>, region marked by blue bars). We analyzed the average numbers of ChIP-seq reads, using in 1-kb windows, of H3K4me3, H3K36me3, and H3K9ac and H3K27me3 associated with expanded and nonexpanded regions in four centromeres (<em>Cen2 Cen5</em>, <em>Cen9</em>, and <em>Cen10</em>). The expanded regions contained significantly lower numbers of reads (<em>p</em>-value &lt; 1 × 10<sup>–5</sup>, two-tailed) of H3K4me3, H3K36me3, and H3K9ac than nonexpanded regions (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6, E–G</a>). Thus, the expanded regions were less euchromatic than the nonexpanded region. A similar read density of H3K27me3 was observed in expanded and nonexpanded region (<em>p</em>-value = 0.69, two-tailed), probably due to the lack of H3K27me3 in the pericentromeric regions (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6H</a>). We also found similar read density of a heterochromatin mark H3K27me2 in expanded and nonexpanded regions (<em>p</em>-value = 0.50, two-tailed) (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6I</a>), suggesting that the heterochromatin marked by H3K27me2 is not likely to play a role in the one-directional expansion of the centromeres.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/183/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Centromere expansion and histone modifications. (A&#x2013;D) Diagrammatic illustrations of expansion of maize Cen2, Cen5, Cen9, and Cen10, respectively, in the genetic background of oat (Wang et al. 2014). Green lines represent CENH3 ChIP-seq read density in maize, red lines represent CENH3 ChIP-seq read density in oat. Green horizontal bars show the locations of the four centromeres in maize. Red horizontal bars show the locations of expanded regions in oat. Each blue horizontal bar represents a nonexpanded region of the same size as the red bar of the same centromere. (E&#x2013;I) Quantification of H3K4me3, H3K36me3, H3K9ac, H3K27me3, and H3K27me2, respectively, in original maize centromeres (green lines), expanded centromeric regions in oat (red lines) and nonexpanded regions (blue lines). The histone modification data are presented as the average of four centromeres (Cen2, Cen5, Cen9, and Cen10). The y-axis represents the number of 1-kb windows. The x-axis represents the average number of reads in each 1-kb window." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-656974181" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Centromere expansion and histone modifications. (A&#x2013;D) Diagrammatic illustrations of expansion of maize &lt;em&gt;Cen2&lt;/em&gt;, &lt;em&gt;Cen5&lt;/em&gt;, &lt;em&gt;Cen9&lt;/em&gt;, and &lt;em&gt;Cen10&lt;/em&gt;, respectively, in the genetic background of oat (Wang &lt;em&gt;et al.&lt;/em&gt; 2014). Green lines represent CENH3 ChIP-seq read density in maize, red lines represent CENH3 ChIP-seq read density in oat. Green horizontal bars show the locations of the four centromeres in maize. Red horizontal bars show the locations of expanded regions in oat. Each blue horizontal bar represents a nonexpanded region of the same size as the red bar of the same centromere. (E&#x2013;I) Quantification of H3K4me3, H3K36me3, H3K9ac, H3K27me3, and H3K27me2, respectively, in original maize centromeres (green lines), expanded centromeric regions in oat (red lines) and nonexpanded regions (blue lines). The histone modification data are presented as the average of four centromeres (&lt;em&gt;Cen2&lt;/em&gt;, &lt;em&gt;Cen5&lt;/em&gt;, &lt;em&gt;Cen9&lt;/em&gt;, and &lt;em&gt;Cen10&lt;/em&gt;). The &lt;em&gt;y&lt;/em&gt;-axis represents the number of 1-kb windows. The &lt;em&gt;x&lt;/em&gt;-axis represents the average number of reads in each 1-kb window.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/183/F6.medium.gif" width="347" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/1/183/F6.medium.gif" width="347" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/183/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/183/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-37" class="first-child">Centromere expansion and histone modifications. (A–D) Diagrammatic illustrations of expansion of maize <em>Cen2</em>, <em>Cen5</em>, <em>Cen9</em>, and <em>Cen10</em>, respectively, in the genetic background of oat (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>). Green lines represent CENH3 ChIP-seq read density in maize, red lines represent CENH3 ChIP-seq read density in oat. Green horizontal bars show the locations of the four centromeres in maize. Red horizontal bars show the locations of expanded regions in oat. Each blue horizontal bar represents a nonexpanded region of the same size as the red bar of the same centromere. (E–I) Quantification of H3K4me3, H3K36me3, H3K9ac, H3K27me3, and H3K27me2, respectively, in original maize centromeres (green lines), expanded centromeric regions in oat (red lines) and nonexpanded regions (blue lines). The histone modification data are presented as the average of four centromeres (<em>Cen2</em>, <em>Cen5</em>, <em>Cen9</em>, and <em>Cen10</em>). The <em>y</em>-axis represents the number of 1-kb windows. The <em>x</em>-axis represents the average number of reads in each 1-kb window.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-38">We determined patterns of maize CENH3 nucleosome occupancy across centromeres by combining CENH3 ChIP and deep sequencing. Consistent with previous studies in rice (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Yan <em>et al.</em> 2008</a>; <a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Zhang <em>et al.</em> 2013</a>), we found that regions with high CENH3 enrichment were interspersed with regions of low CENH3 enrichment, at least some of which contain expressed genes. Although the chromatin environment of centromeres differs from the rest of the genome (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Verdaasdonk and Bloom 2011</a>), we found that expressed genes located in centromeres exhibit histone modification associated with typical active genes, including H3K36me3, H3K4me2, and H3K9ac (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/FigureS2.pdf">Figure S2</a>). The fact that expressed genes in centromeres have lower CENH3 enrichment than other parts of centromeres suggests that transcription may inhibit CENH3 accumulation. We also speculate that very high levels of CENH3 within genes would be incompatible with transcription because of a different N-terminal tail structure that cannot be modified with the H3 modifications associated with transcription regulation (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gassmann <em>et al.</em> 2012</a>; <a id="xref-ref-42-7" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2014</a>). Regardless of how CENH3 affects or is affected by transcription, it is clear that the expressed genes can exist inside of functional centromeres in maize.</p><p id="p-39">Comparison of CENH3 nucleosome distribution in maize and rice reveals several interesting differences. First, maize centromeres are much larger than rice centromeres (∼2 Mb <em>vs.</em> ∼750 kb). Second, maize centromeres appear to have greater CENH3 nucleosome occupancy across their centromeres: 87% of maize centromeres were made of domains of high CENH3 enrichment, compared with 38–49% of rice centromeres (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Yan <em>et al.</em> 2008</a>). These numbers should be interpreted with caution, however, as the rice study measured enrichment using a normalization-free method that does not allow precise measurements of CENH3 enrichment. Further supporting evidence for greater CENH3 nucleosome occupancy in maize centromeres than in rice comes from the different numbers of genes in centromeres in each species: while maize centromeres contain from one to eight genes in each centromere, rice centromeres contain from 17 to 21 genes (<a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Yan <em>et al.</em> 2008</a>).</p><p id="p-40">High occupancy of CENH3 nucleosomes in maize is not limited to regions containing the <em>CentC</em> repeats, nor is it an artifact from mapping reads to areas of low sequence complexity. First, <em>Cen2</em> and <em>Cen5</em> contain the lowest amounts of <em>CentC</em> repeats among B73 centromeres (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albert <em>et al.</em> 2010</a>), but regions of high CENH3 enrichment make up nearly 90% of these two centromeres. Second, <em>CentC</em> and the two dominant centromeric retrotransposons <em>CRM1</em> and <em>CRM2</em> of maize are highly enriched for CENH3 nucleosomes (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Zhong <em>et al.</em> 2002</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Jin <em>et al.</em> 2004</a>; <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Wolfgruber <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gent <em>et al.</em> 2011</a>). Differences in centromere size and CENH3 abundance between maize and rice may be a consequence of maize having much a larger genome and chromosome size. An elegant immunofluorescence-based comparison of centromere and genome size revealed that centromere size correlates with genome size among the genetically related grass species, suggesting large amounts of cenH3 benefit the stabilization of the spindle and separation of large chromosomes (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhang and Dawe 2012</a>), Based on these data, we propose that abundant CENH3 nucleosomes are important for centromere function in species with large chromosomes, and we predict that characterization of additional large-chromosome species will reveal maize-like centromeres that are either large or have a high percentage of CENH3-enriched subdomain within the centromeres.</p><p id="p-41">Consistent with prior studies of nucleosome organization for the maize centromeric repetitive elements <em>CentC</em>, <em>CRM1</em>, and <em>CRM2</em> (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Gent <em>et al.</em> 2011</a>), we found that the dominant pattern of nucleosome spacing across centromeres revealed by micrococcal nuclease digestion was approximately 190 bp between nucleosome midpoints (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3C</a>). However, we also found evidence for reproducible positioning, or phasing, of nucleosomes relative to the <em>CentC</em> repeat (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3A</a>). Phasing of CENH3 nucleosomes on the 156-bp <em>CentC</em> repeats mirrors the recently reported phasing of cenH3 nucleosomes along the 155-bp <em>CentO</em> repeats in rice (<a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Zhang <em>et al.</em> 2013</a>). Thus, nucleosome phasing mediated by centromeric tandem repeats may be a common mechanism that contributes toward centromere/kinetochore function. The existence of distinct fragment sizes following MNase digestion in both input and ChIP (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) could reflect flexibility of DNA wrapping around cenH3 nucleosomes (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Hasson <em>et al.</em> 2013</a>). Another potential explanation for smaller DNA fragments is that nucleosomes are not limited to the classical eight-histone and 147-bp wrapping DNA structure, and such alternative histone complexes are particularly abundant among cenH3 containing nucleosomes. Indeed, multiple experimental approaches in diverse species have indicated that cenH3 nucleosomes can exist in a complex of histone tetramers rather than the canonical histone octamer (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Dalal <em>et al.</em> 2007</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dimitriadis <em>et al.</em> 2010</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Shivaraju <em>et al.</em> 2012</a>).</p><p id="p-42">Centromeric chromatin is associated with distinct histone modification patterns as compared to classical euchromatin and heterochromatin in several model eukaryotes. In humans and <em>Drosophila melanogaster</em>, CENP-A nucleosomes are interspersed with H3 nucleosomes marked by H3K4me2. These H3 nucleosomes are associated neither with the euchromatic mark H3K4me3 nor with the heterochromatic marks H3K9me2 and H3K9me3 (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Sullivan and Karpen 2004</a>). H3 nucleosomes marked with H3K4me were also detected in <em>S. pombe</em> centromeres (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Cam <em>et al.</em> 2005</a>). However, this distinct association of H3K4me2 with centromeres was not found in rice centromeres. Transcribed genes in rice centromeres, however, were associated with classical euchromatic histone modifications marks, including H3K4me2, H3K4me3, H3K36me3, and H3K4ac (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Wu <em>et al.</em> 2011</a>). We detected a similar association of euchromatic histone modifications with active genes in maize centromeres (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Maize centromeres had reduced levels of the heterochromatic mark H3K27me2 relative to pericentromeric regions (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1/FigureS3.pdf">Figure S3</a>), which agrees with cytological observations that maize centromeres were weakly stained with immunofluorescence from heterochromatic histone modification marks (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Shi and Dawe 2006</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Jin <em>et al.</em> 2008</a>). Thus, maize centromeres appear to be dominated by subdomains enriched with CENH3 and have a relative depletion of histone modifications associated with H3 histone. We find no evidence that histone H3 modifications studied in this work serve as a major epigenetic mark for centromere identity.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-43">This work was supported by grant IOS-0922703 from the National Science Foundation to R.K.D. and J.J.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-4"><p id="p-44">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022764/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-45">Sequence data from this article have been deposited with the NCBI sequence read archive under accession nos. SRP057754 and SRP049952.</p></li><li class="fn" id="fn-6"><p id="p-46"><em>Communicating editor: J. Wendel</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-09-17"><span class="received-label">Received </span>September 17, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-11-09"><span class="accepted-label">Accepted </span>November 9, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Zhao <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.1" data-doi="10.1159/000314342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danilova</span>  <span class="cit-name-given-names">T. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Diversity of chromosomal karyotypes in maize and its relatives.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">6</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DAlbert%26rft.auinit1%253DP.%2BS.%26rft.volume%253D129%26rft.issue%253D1-3%26rft.spage%253D6%26rft.epage%253D16%26rft.atitle%253DDiversity%2Bof%2Bchromosomal%2Bkaryotypes%2Bin%2Bmaize%2Band%2Bits%2Brelatives.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000314342%26rft_id%253Dinfo%253Apmid%252F20551613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000314342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20551613&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280519500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.2" data-doi="10.1016/S1534-5807(02)00135-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blower</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpen</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Conserved organization of centromeric chromatin in flies and humans.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bcell%26rft.stitle%253DDev%2BCell%26rft.aulast%253DBlower%26rft.auinit1%253DM.%2BD.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253D319%26rft.epage%253D330%26rft.atitle%253DConserved%2Borganization%2Bof%2Bcentromeric%2Bchromatin%2Bin%2Bflies%2Band%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1534-5807%252802%252900135-1%26rft_id%253Dinfo%253Apmid%252F11879637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1534-5807(02)00135-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11879637&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175387900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.3" data-doi="10.1038/ng1602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cam</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugiyama</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">FitzGerald</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Comprehensive analysis of heterochromatin- and RNAi-mediated epigenetic control of the fission yeast genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCam%26rft.auinit1%253DH.%2BP.%26rft.volume%253D37%26rft.issue%253D8%26rft.spage%253D809%26rft.epage%253D819%26rft.atitle%253DComprehensive%2Banalysis%2Bof%2Bheterochromatin-%2Band%2BRNAi-mediated%2Bepigenetic%2Bcontrol%2Bof%2Bthe%2Bfission%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1602%26rft_id%253Dinfo%253Apmid%252F15976807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1602&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15976807&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230880400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dalal</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindsay</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Tetrameric structure of centromeric nucleosomes in interphase Drosophila cells.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1798</span>–<span class="cit-lpage">1809</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D5%26rft.spage%253D1798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000249124800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.5" data-doi="10.1073/pnas.1009563107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dimitriadis</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diekmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalal</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Tetrameric organization of vertebrate centromeric nucleosomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">20317</span>–<span class="cit-lpage">20322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1009563107%26rft_id%253Dinfo%253Apmid%252F21059934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQ3LzIwMzE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.6" data-doi="10.1016/0022-2836(84)90176-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drew</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Structural specificities of five commonly used DNA nucleases.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDrew%26rft.auinit1%253DH.%2BR.%26rft.volume%253D176%26rft.issue%253D4%26rft.spage%253D535%26rft.epage%253D557%26rft.atitle%253DStructural%2Bspecificities%2Bof%2Bfive%2Bcommonly%2Bused%2BDNA%2Bnucleases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252884%252990176-1%26rft_id%253Dinfo%253Apmid%252F6086935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(84)90176-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6086935&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TB56000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.7" data-doi="10.1093/bioinformatics/btr345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flores</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">nucleR: a package for non-parametric nucleosome positioning.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2149</span>–<span class="cit-lpage">2150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr345%26rft_id%253Dinfo%253Apmid%252F21653521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE0OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fukagawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Earnshaw</span>  <span class="cit-name-given-names">W. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The centromere: chromatin foundation for the kinetochore machinery.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BCell%26rft.volume%253D30%26rft.spage%253D497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.9" data-doi="10.1038/nature10973"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gassmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rechtsteiner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuen</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muroyama</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egelhofer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An inverse relationship to germline transcription defines centromeric chromatin in <em>C. elegans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">484</span>: <span class="cit-fpage">534</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGassmann%26rft.auinit1%253DR.%26rft.volume%253D484%26rft.issue%253D7395%26rft.spage%253D534%26rft.epage%253D537%26rft.atitle%253DAn%2Binverse%2Brelationship%2Bto%2Bgermline%2Btranscription%2Bdefines%2Bcentromeric%2Bchromatin%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10973%26rft_id%253Dinfo%253Apmid%252F22495302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10973&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22495302&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303200400055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.10" data-doi="10.1186/1756-8935-4-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gent</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Topp</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Presting</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Distinct influences of tandem repeats and retrotransposons on CENH3 nucleosome positioning.</span> <abbr class="cit-jnl-abbrev">Epigenetics Chromatin</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEpigenetics%2BChromatin%26rft.aulast%253DGent%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D4%26rft.spage%253D3%26rft.epage%253D3%26rft.atitle%253DDistinct%2Binfluences%2Bof%2Btandem%2Brepeats%2Band%2Bretrotransposons%2Bon%2BCENH3%2Bnucleosome%2Bpositioning.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-8935-4-3%26rft_id%253Dinfo%253Apmid%252F21352520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-8935-4-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21352520&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.11" data-doi="10.1093/nar/gkr862"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gent</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawe</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Strong epigenetic similarity between maize centromeric and pericentromeric regions at the level of small RNAs, DNA methylation and H3 chromatin modifications.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1550</span>–<span class="cit-lpage">1560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr862%26rft_id%253Dinfo%253Apmid%252F22058126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQwLzQvMTU1MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.12" data-doi="10.1105/tpc.114.130427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gent</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madzima</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accessible DNA and relative depletion of H3K9me2 at maize loci undergoing RNA-directed DNA methylation.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4903</span>–<span class="cit-lpage">4917</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.130427%26rft_id%253Dinfo%253Apmid%252F25465407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi80OTAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.13" data-doi="10.1534/genetics.115.177360"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gent</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawe</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Stable patterns of CENH3 occupancy through maize lineages containing genetically similar centromeres.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177360%26rft_id%253Dinfo%253Apmid%252F26063660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMC80LzExMDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.14" data-doi="10.1105/tpc.112.100511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koblizkova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Repeatless and repeat-based centromeres in potato: implications for centromere evolution.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">3559</span>–<span class="cit-lpage">3574</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.112.100511%26rft_id%253Dinfo%253Apmid%252F22968715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI0LzkvMzU1OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.15" data-doi="10.1083/jcb.200403109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guenatri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maison</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almouzni</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mouse centric and pericentric satellite repeats form distinct functional heterochromatin.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200403109%26rft_id%253Dinfo%253Apmid%252F15302854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE2Ni80LzQ5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.16" data-doi="10.1038/nsmb.2562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hasson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panchenko</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salimian</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salman</span>  <span class="cit-name-given-names">M. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekulic</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The octamer is the major form of CENP-A nucleosomes at human centromeres.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">687</span>–<span class="cit-lpage">695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D20%26rft.spage%253D687%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2562%26rft_id%253Dinfo%253Apmid%252F23644596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23644596&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.17" data-doi="10.1073/pnas.1110731108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belsky</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krassovsky</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacAlpine</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Epigenome characterization at single base-pair resolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">18318</span>–<span class="cit-lpage">18323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1110731108%26rft_id%253Dinfo%253Apmid%252F22025700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzQ1LzE4MzE4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.18" data-doi="10.1126/sciadv.1400234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thakur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasinathan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A unique chromatin complex occupies young α-satellite arrays of human centromeres.</span> <abbr class="cit-jnl-abbrev">Sci. Adv.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e1400234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DHenikoff%26rft.auinit1%253DJ.%2BG.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253De1400234%26rft.epage%253De1400234%26rft.atitle%253DA%2Bunique%2Bchromatin%2Bcomplex%2Boccupies%2Byoung%2B%257Balpha%257D-satellite%2Barrays%2Bof%2Bhuman%2Bcentromeres%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.1400234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiMS8xL2UxNDAwMjM0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.19" data-doi="10.1534/genetics.111.137711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">“Point” centromeres of Saccharomyces harbor single centromere-specific nucleosomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1575</span>–<span class="cit-lpage">1577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137711%26rft_id%253Dinfo%253Apmid%252F22234856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1NzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.20" data-doi="10.1126/science.1062939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmad</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malik</span>  <span class="cit-name-given-names">H. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The centromere paradox: stable inheritance with rapidly evolving DNA.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">293</span>: <span class="cit-fpage">1098</span>–<span class="cit-lpage">1102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHenikoff%26rft.auinit1%253DS.%26rft.volume%253D293%26rft.issue%253D5532%26rft.spage%253D1098%26rft.epage%253D1102%26rft.atitle%253DThe%2BCentromere%2BParadox%253A%2BStable%2BInheritance%2Bwith%2BRapidly%2BEvolving%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1062939%26rft_id%253Dinfo%253Apmid%252F11498581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTMvNTUzMi8xMDk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.21" data-doi="10.1093/nar/9.12.2643"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altenburger</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Sequence specific cleavage of DNA by micrococcal nuclease.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2643</span>–<span class="cit-lpage">2658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHorz%26rft.auinit1%253DW.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D2643%26rft.epage%253D2658%26rft.atitle%253DSequence%2Bspecific%2Bcleavage%2Bof%2BDNA%2Bby%2Bmicrococcal%2Bnuclease%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F9.12.2643%26rft_id%253Dinfo%253Apmid%252F7279658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjkvMTIvMjY0MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.22" data-doi="10.1101/gr.9.9.868"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">CAP3: a DNA sequence assembly program.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">868</span>–<span class="cit-lpage">877</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D868%26rft.epage%253D877%26rft.atitle%253DCAP3%253A%2BA%2BDNA%2BSequence%2BAssembly%2BProgram%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.9.9.868%26rft_id%253Dinfo%253Apmid%252F10508846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjc6IjkvOS84NjgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.23" data-doi="10.1105/tpc.018937"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melo</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagaki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbert</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Maize centromeres: organization and functional adaptation in the genetic background of oat.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.018937%26rft_id%253Dinfo%253Apmid%252F14973167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE2LzMvNTcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.24" data-doi="10.1007/s10577-008-1269-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolano</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Histone modifications associated with both A and B chromosomes of maize.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1203</span>–<span class="cit-lpage">1214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DJin%26rft.auinit1%253DW.%26rft.volume%253D16%26rft.issue%253D8%26rft.spage%253D1203%26rft.epage%253D1214%26rft.atitle%253DHistone%2Bmodifications%2Bassociated%2Bwith%2Bboth%2BA%2Band%2BB%2Bchromosomes%2Bof%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-008-1269-8%26rft_id%253Dinfo%253Apmid%252F18987983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-008-1269-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18987983&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261966800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.25" data-doi="10.1186/1471-2105-7-474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohany</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gentles</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hankus</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Annotation, submission and screening of repetitive elements in Repbase: RepbaseSubmitter and Censor.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">474</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKohany%26rft.auinit1%253DO.%26rft.volume%253D7%26rft.spage%253D474%26rft.epage%253D474%26rft.atitle%253DAnnotation%252C%2Bsubmission%2Band%2Bscreening%2Bof%2Brepetitive%2Belements%2Bin%2BRepbase%253A%2BRepbaseSubmitter%2Band%2BCensor.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-7-474%26rft_id%253Dinfo%253Apmid%252F17064419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-7-474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17064419&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.26" data-doi="10.1073/pnas.1118898109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krassovsky</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Tripartite organization of centromeric chromatin in budding yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1118898109%26rft_id%253Dinfo%253Apmid%252F22184235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDkvMS8yNDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-web" id="cit-6.1.183.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. <a href="http://arxiv.org/abs/1303.3997">http://arxiv.org/abs/1303.3997</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.28" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.29" data-doi="10.1093/bioinformatics/btr507"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magoc</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">FLASH: fast length adjustment of short reads to improve genome assemblies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2957</span>–<span class="cit-lpage">2963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr507%26rft_id%253Dinfo%253Apmid%252F21903629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMjEvMjk1NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> Cutadapt removes adapter sequences from high-throughput sequencing reads. <em>EMBnet.journal</em> 17:10–12.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.31" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi82Lzg0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.32" data-doi="10.1073/pnas.1002325107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ribeiro</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vagnarelli</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hori</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEwen</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A super-resolution map of the vertebrate kinetochore.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">10484</span>–<span class="cit-lpage">10489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1002325107%26rft_id%253Dinfo%253Apmid%252F20483991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzIzLzEwNDg0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.33" data-doi="10.1126/science.1178534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The B73 maize genome: complexity, diversity, and dynamics.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchnable%26rft.auinit1%253DP.%2BS.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1112%26rft.epage%253D1115%26rft.atitle%253DThe%2BB73%2BMaize%2BGenome%253A%2BComplexity%252C%2BDiversity%252C%2Band%2BDynamics%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1178534%26rft_id%253Dinfo%253Apmid%252F19965430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.34" data-doi="10.1534/genetics.106.056853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawe</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Partitioning of the maize epigenome by the number of methyl groups on histone H3 lysines 9 and 27.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1571</span>–<span class="cit-lpage">1583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.056853%26rft_id%253Dinfo%253Apmid%252F16624902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE1NzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.35" data-doi="10.1016/j.cell.2012.05.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shivaraju</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Unruh</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slaughter</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mattingly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cell-cycle-coupled structural oscillation of centromeric nucleosomes in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">304</span>–<span class="cit-lpage">316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DShivaraju%26rft.auinit1%253DM.%26rft.volume%253D150%26rft.issue%253D2%26rft.spage%253D304%26rft.epage%253D316%26rft.atitle%253DCell-cycle-coupled%2Bstructural%2Boscillation%2Bof%2Bcentromeric%2Bnucleosomes%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.05.034%26rft_id%253Dinfo%253Apmid%252F22817893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.05.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22817893&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306595700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.36" data-doi="10.1007/s10577-012-9290-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galazka</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phatale</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connolly</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freitag</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Centromeres of filamentous fungi.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DSmith%26rft.auinit1%253DK.%2BM.%26rft.volume%253D20%26rft.issue%253D5%26rft.spage%253D635%26rft.epage%253D656%26rft.atitle%253DCentromeres%2Bof%2Bfilamentous%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-012-9290-3%26rft_id%253Dinfo%253Apmid%252F22752455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-012-9290-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22752455&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.37" data-doi="10.1038/nsmb845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karpen</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Centromeric chromatin exhibits a histone modification pattern that is distinct from both euchromatin and heterochromatin.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1076</span>–<span class="cit-lpage">1083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DSullivan%26rft.auinit1%253DB.%2BA.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253D1076%26rft.epage%253D1083%26rft.atitle%253DCentromeric%2Bchromatin%2Bexhibits%2Ba%2Bhistone%2Bmodification%2Bpattern%2Bthat%2Bis%2Bdistinct%2Bfrom%2Bboth%2Beuchromatin%2Band%2Bheterochromatin.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb845%26rft_id%253Dinfo%253Apmid%252F15475964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15475964&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224847600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.38" data-doi="10.1016/0020-711X(89)90100-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Telford</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Micrococcal nuclease: its specificity and use for chromatin analysis.</span> <abbr class="cit-jnl-abbrev">Int. J. Biochem.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BInternational%2Bjournal%2Bof%2Bbiochemistry%26rft.stitle%253DInt%2BJ%2BBiochem%26rft.aulast%253DTelford%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D127%26rft.epage%253D137%26rft.atitle%253DMicrococcal%2Bnuclease%253A%2Bits%2Bspecificity%2Band%2Buse%2Bfor%2Bchromatin%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0020-711X%252889%252990100-6%26rft_id%253Dinfo%253Apmid%252F2663558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0020-711X(89)90100-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2663558&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T106500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.39" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNS85LzExMDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTgzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.40" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.41" data-doi="10.1038/nrm3107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verdaasdonk</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Centromeres: unique chromatin structures that drive chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DVerdaasdonk%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D320%26rft.epage%253D332%26rft.atitle%253DCentromeres%253A%2Bunique%2Bchromatin%2Bstructures%2Bthat%2Bdrive%2Bchromosome%2Bsegregation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm3107%26rft_id%253Dinfo%253Apmid%252F21508988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm3107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21508988&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.42" data-doi="10.1101/gr.160887.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawe</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Maize centromeres expand and adopt a uniform size in the genetic background of oat.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.160887.113%26rft_id%253Dinfo%253Apmid%252F24100079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI0LzEvMTA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.43" data-doi="10.1105/tpc.109.065714"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elling</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide and organ-specific landscapes of epigenetic modifications and their relationships to mRNA and small RNA transcriptomes in maize.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1053</span>–<span class="cit-lpage">1069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.065714%26rft_id%253Dinfo%253Apmid%252F19376930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIxLzQvMTA1MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.44" data-doi="10.1371/journal.pgen.1000743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfgruber</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koo</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Maize centromere structure and evolution: Sequence analysis of centromeres 2 and 5 reveals dynamic loci shaped primarily by retrotransposons.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWolfgruber%26rft.auinit1%253DT.%2BK.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De1000743%26rft.epage%253De1000743%26rft.atitle%253DMaize%2Bcentromere%2Bstructure%2Band%2Bevolution%253A%2Bsequence%2Banalysis%2Bof%2Bcentromeres%2B2%2Band%2B5%2Breveals%2Bdynamic%2BLoci%2Bshaped%2Bprimarily%2Bby%2Bretrotransposons.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000743%26rft_id%253Dinfo%253Apmid%252F19956743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19956743&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.45" data-doi="10.1105/tpc.111.090043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kikuchi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenbaum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Euchromatic subdomains in rice centromeres are associated with genes and transcription.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4054</span>–<span class="cit-lpage">4064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.111.090043%26rft_id%253Dinfo%253Apmid%252F22080597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xMS80MDU0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.46" data-doi="10.1105/tpc.105.037945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagaki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouyang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Transcription and histone modifications in the recombination-free region spanning a rice centromere.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">3227</span>–<span class="cit-lpage">3238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.105.037945%26rft_id%253Dinfo%253Apmid%252F16272428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNy8xMi8zMjI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.47" data-doi="10.1105/tpc.106.043794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nobuta</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouyang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic and genetic characterization of rice Cen3 reveals extensive transcription and evolutionary implications of a complex centromere.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2123</span>–<span class="cit-lpage">2133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.106.043794%26rft_id%253Dinfo%253Apmid%252F16877494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE4LzkvMjEyMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbert</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jett</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Intergenic locations of rice centromeric chromatin.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2563</span>–<span class="cit-lpage">2575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D6%26rft.spage%253D2563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000261187900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.49" data-doi="10.1093/bioinformatics/btp340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schones</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A clustering approach for identification of enriched domains from histone modification ChIP-Seq data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1952</span>–<span class="cit-lpage">1958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp340%26rft_id%253Dinfo%253Apmid%252F19505939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTUvMTk1MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xODMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.50" data-doi="10.1007/s10577-012-9284-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawe</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Total centromere size and genome size are strongly correlated in ten grass species.</span> <abbr class="cit-jnl-abbrev">Chromosome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosome%2Bresearch%2B%253A%2B%2Ban%2Binternational%2Bjournal%2Bon%2Bthe%2Bmolecular%252C%2Bsupramolecular%2Band%2Bevolutionary%2Baspects%2Bof%2Bchromosome%2Bbiology%26rft.stitle%253DChromosome%2BRes%26rft.aulast%253DZhang%26rft.auinit1%253DH.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253D403%26rft.epage%253D412%26rft.atitle%253DTotal%2Bcentromere%2Bsize%2Band%2Bgenome%2Bsize%2Bare%2Bstrongly%2Bcorrelated%2Bin%2Bten%2Bgrass%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10577-012-9284-1%26rft_id%253Dinfo%253Apmid%252F22552915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10577-012-9284-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22552915&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306127600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.51" data-doi="10.1073/pnas.1319548110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbert</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The CentO satellite confers translational and rotational phasing on cenH3 nucleosomes in rice centromeres.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E4875</span>–<span class="cit-lpage">E4883</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1319548110%26rft_id%253Dinfo%253Apmid%252F24191062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzUwL0U0ODc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.52" data-doi="10.1101/gr.131342.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution mapping of open chromatin in the rice genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.131342.111%26rft_id%253Dinfo%253Apmid%252F22110044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzEvMTUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.53" data-doi="10.1186/gb-2008-9-9-r137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eeckhoute</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Model-based analysis of ChIP-Seq (MACS).</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">R137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253DR137%26rft.epage%253DR137%26rft.atitle%253DModel-based%2Banalysis%2Bof%2BChIP-Seq%2B%2528MACS%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-9-r137%26rft_id%253Dinfo%253Apmid%252F18798982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-9-r137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18798982&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F183.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.183.54" data-doi="10.1105/tpc.006106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Topp</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mroczek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Centromeric retroelements and satellites interact with maize kinetochore protein CENH3.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2825</span>–<span class="cit-lpage">2836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.006106%26rft_id%253Dinfo%253Apmid%252F12417704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMS8yODI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE4My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/1/183.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/171" title="On the Consequences of Purging and Linkage on Fitness and Genetic Diversity" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/193" title="Identification of a Classical Mutant in the Industrial Host Aspergillus niger by Systems Genetics: LaeA Is Required for Citric Acid Production and Regulates the Formation of Some Secondary Metabolites" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/1"><span>Volume 6 Issue 1, January 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/1.cover-source.jpg?itok=9zFiZvCK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/183.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dbNAnfs3UM5yPvFtypPYiaVaVwJ0gczvsfjs5hbLgY4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/1/183" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/1/183" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/1/183" class="active" data-icon-position="" data-hide-link-title="0">Gene Expression and Chromatin Modifications Associated with Maize Centromeres</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460246" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-5mQTK8bUhhYY32YwpplUFeYeEq1Mo6gv-fsFh2kuxbg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/183.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/1/183" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-jD2q8gyCwaivGfhToNql1udlJevL-Mn55yUZ1Vwn2b8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/1/183" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/183.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460246" id="node460246--2" data-pisa="ggg;6/1/183" data-pisa-master="ggg;g3.115.022764" data-apath="/ggg/6/1/183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gene Expression and Chromatin Modifications Associated with Maize Centromeres</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hainan Zhao</span>, <span class="highwire-citation-author" data-delta="1">Xiaobiao Zhu</span>, <span class="highwire-citation-author" data-delta="2">Kai Wang</span>, <span class="highwire-citation-author" data-delta="3">Jonathan I. Gent</span>, <span class="highwire-citation-author" data-delta="4">Wenli Zhang</span>, <span class="highwire-citation-author" data-delta="5">R. Kelly Dawe</span> and <span class="highwire-citation-author" data-delta="6">Jiming Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">183-192;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022764 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460246--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hainan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hainan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHainan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaobiao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaobiao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaobiao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kai Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKai%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jonathan I. Gent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BI.%2BGent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gent%20JI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BI.%2BGent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wenli Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenli%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWenli%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">R. Kelly Dawe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BKelly%2BDawe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dawe%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BKelly%2BDawe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jiming Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiming%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiming%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jjiang1@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">jjiang1@wisc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/1/183" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/1/183&amp;title=Gene%20Expression%20and%20Chromatin%20Modifications%20Associated%20with%20Maize%20Centromeres" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/1/183&amp;title=Gene%20Expression%20and%20Chromatin%20Modifications%20Associated%20with%20Maize%20Centromeres" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/1/183&amp;title=Gene%20Expression%20and%20Chromatin%20Modifications%20Associated%20with%20Maize%20Centromeres" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/1/183&amp;text=Gene%20Expression%20and%20Chromatin%20Modifications%20Associated%20with%20Maize%20Centromeres" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/1/183&amp;title=Gene%20Expression%20and%20Chromatin%20Modifications%20Associated%20with%20Maize%20Centromeres" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/1/183&amp;t=Gene%20Expression%20and%20Chromatin%20Modifications%20Associated%20with%20Maize%20Centromeres" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/1/183&amp;title=Gene%20Expression%20and%20Chromatin%20Modifications%20Associated%20with%20Maize%20Centromeres" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/1/183&amp;title=Gene%20Expression%20and%20Chromatin%20Modifications%20Associated%20with%20Maize%20Centromeres" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460246/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460246" id="node460246--3" data-pisa="ggg;6/1/183" data-pisa-master="ggg;g3.115.022764" data-apath="/ggg/6/1/183.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Gene Expression and Chromatin Modifications Associated with Maize Centromeres</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hainan Zhao</span>, <span class="highwire-citation-author" data-delta="1">Xiaobiao Zhu</span>, <span class="highwire-citation-author" data-delta="2">Kai Wang</span>, <span class="highwire-citation-author" data-delta="3">Jonathan I. Gent</span>, <span class="highwire-citation-author" data-delta="4">Wenli Zhang</span>, <span class="highwire-citation-author" data-delta="5">R. Kelly Dawe</span> and <span class="highwire-citation-author" data-delta="6">Jiming Jiang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">183-192;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022764 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460246--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hainan Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hainan%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHainan%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xiaobiao Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xiaobiao%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXiaobiao%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kai Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kai%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKai%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jonathan I. Gent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BI.%2BGent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gent%20JI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BI.%2BGent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Wenli Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wenli%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWenli%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">R. Kelly Dawe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R.%2BKelly%2BDawe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dawe%20RK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AR.%2BKelly%2BDawe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jiming Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Horticulture, University of Wisconsin, Madison, Wisconsin 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiming%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiming%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jjiang1@wisc.edu" class="" data-icon-position="" data-hide-link-title="0">jjiang1@wisc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=183&amp;rft.epage=192&amp;rft.atitle=Gene%20Expression%20and%20Chromatin%20Modifications%20Associated%20with%20Maize%20Centromeres&amp;rft.volume=6&amp;rft.issue=1&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhao%2C+Hainan&amp;rft.au=Zhu%2C+Xiaobiao&amp;rft.au=Wang%2C+Kai&amp;rft.au=Gent%2C+Jonathan+I.&amp;rft.au=Zhang%2C+Wenli&amp;rft.au=Dawe%2C+R.+Kelly&amp;rft.au=Jiang%2C+Jiming"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460246/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460246/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460246/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460246/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460246/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460246/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460246/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460246/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460246/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460246/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460246/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460246/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/1/183" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/1/183.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/1/183.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/1/183.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/1/183.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
