<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/1/149" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/1/149.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/1/149.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/1/149.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/1/149.atom" />
<meta name="HW.pisa" content="ggg;6/1/149" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Estimates of Mutation Rates and Spectrum in Schizosaccharomyces pombe Indicate CpG Sites are Highly Mutagenic Despite the Absence of DNA Methylation" />
<meta name="DC.Identifier" content="10.1534/g3.115.022129" />
<meta name="DC.Date" content="2016-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Behringer and Hall. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="We accumulated mutations for 1952 generations in 79 initially identical, haploid lines of the fission yeast Schizosaccharomyces pombe , and then performed whole-genome sequencing to determine the mutation rates and spectrum. We captured 696 spontaneous mutations across the 79 mutation accumulation (MA) lines. We compared the mutation spectrum and rate to a recently published equivalent experiment on the same species, and to another model ascomycetous yeast, the budding yeast Saccharomyces cerevisiae . While the two species are approximately 600 million years diverged from each other, they share similar life histories, genome size and genomic G/C content. We found that Sc. pombe and S. cerevisiae have similar mutation rates, but Sc. pombe exhibits a stronger insertion bias. Intriguingly, we observed an increased mutation rate at cytosine nucleotides, specifically CpG nucleotides, which is also seen in S. cerevisiae . However, the absence of methylation in Sc. pombe and the pattern of mutation at these sites, primarily C → A as opposed to C → T, strongly suggest that the increased mutation rate is not caused by deamination of methylated cytosines. This result implies that the high mutability of CpG dinucleotides in other species may be caused in part by a methylation-independent mechanism. Many of our findings mirror those seen in the recent study, despite the use of different passaging conditions, indicating that MA is a reliable method for estimating mutation rates and spectra." />
<meta name="DC.Contributor" content="Megan G. Behringer" />
<meta name="DC.Contributor" content="David W. Hall" />
<meta name="article:published_time" content="2016-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Estimates of Mutation Rates and Spectrum in Schizosaccharomyces pombe Indicate CpG Sites are Highly Mutagenic Despite the Absence of DNA Methylation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;We accumulated mutations for 1952 generations in 79 initially identical, haploid lines of the fission yeast &lt;i&gt;Schizosaccharomyces pombe&lt;/i&gt;, and then performed whole-genome sequencing to determine the mutation rates and spectrum. We captured 696 spontaneous mutations across the 79 mutation accumulation (MA) lines. We compared the mutation spectrum and rate to a recently published equivalent experiment on the same species, and to another model ascomycetous yeast, the budding yeast &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;. While the two species are approximately 600&amp;nbsp;million&amp;nbsp;years diverged from each other, they share similar life histories, genome size and genomic G/C content. We found that &lt;i&gt;Sc. pombe&lt;/i&gt; and &lt;i&gt;S. cerevisiae&lt;/i&gt; have similar mutation rates, but &lt;i&gt;Sc. pombe&lt;/i&gt; exhibits a stronger insertion bias. Intriguingly, we observed an increased mutation rate at cytosine nucleotides, specifically CpG nucleotides, which is also seen in &lt;i&gt;S. cerevisiae&lt;/i&gt;. However, the absence of methylation in &lt;i&gt;Sc. pombe&lt;/i&gt; and the pattern of mutation at these sites, primarily C → A as opposed to C → T, strongly suggest that the increased mutation rate is not caused by deamination of methylated cytosines. This result implies that the high mutability of CpG dinucleotides in other species may be caused in part by a methylation-independent mechanism. Many of our findings mirror those seen in the recent study, despite the use of different passaging conditions, indicating that MA is a reliable method for estimating mutation rates and spectra.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="ggg;6/1/149" />
<meta name="citation_id" content="6/1/149" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/1/149" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/1/149.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/1/149.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/1/149.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.022129" />
<meta name="citation_pmid" content="26564949" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="149" />
<meta name="citation_lastpage" content="160" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Megan G. Behringer" />
<meta name="citation_author_institution" content="Department of Genetics, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_email" content="megbehri@indiana.edu" />
<meta name="citation_author" content="David W. Hall" />
<meta name="citation_author_institution" content="Department of Genetics, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Andolfatto;citation_title=Adaptive evolution of non-coding DNA in Drosophila.;citation_pages=1149-1152;citation_volume=437;citation_year=2005;citation_issue=7062;citation_pmid=16237443;citation_doi=10.1038/nature04107" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=F. Antequera;citation_author=M. Tamame;citation_author=J. Villanueva;citation_author=T. Santos;citation_title=DNA methylation in the fungi;citation_pages=8033-8036;citation_volume=259;citation_year=1984;citation_issue=13;citation_pmid=6330093" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=J. Bell;citation_title=A simple way to treat PCR products prior to sequencing using ExoSAP-IT.;citation_pages=834-834;citation_volume=44;citation_year=2008;citation_issue=6;citation_pmid=18476841;citation_doi=10.2144/000112890" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=T. H. Bestor;citation_author=G. L. Verdine;citation_title=DNA methyltransferases.;citation_pages=380-389;citation_volume=6;citation_year=1994;citation_issue=3;citation_pmid=7917329;citation_doi=10.1016/0955-0674(94)90030-2" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Blumenstiel;citation_author=A. C. Noll;citation_author=J. A. Griffiths;citation_author=A. G. Perera;citation_author=K. N. Walton;citation_title=Identification of EMS-induced mutations in Drosophila melanogaster by whole-genome sequencing.;citation_volume=182;citation_year=2009;citation_pmid=19307605;citation_doi=10.1534/genetics.109.101998" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=B. J. Brewer;citation_author=E. Chlebowicz-Sledziewska;citation_author=W. L. Fangman;citation_title=Cell cycle phases in the unequal mother/daughter cell cycles of Saccharomyces cerevisiae.;citation_pages=2529-2531;citation_volume=4;citation_year=1984;citation_issue=11;citation_pmid=6392855;citation_doi=10.1128/MCB.4.11.2529" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Cavero;citation_author=C. Chahwan;citation_author=P. Russell;citation_title=Xlf1 is required for DNA repair by nonhomologous end joining in Schizosaccharomyces pombe.;citation_volume=175;citation_year=2007;citation_pmid=17151234;citation_doi=10.1534/genetics.106.067850" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=K. C. Cheng;citation_author=D. S. Cahill;citation_author=H. Kasai;citation_author=S. Nishimura;citation_author=L. A. Loeb;citation_title=8-Hydroxyguanine, an abundant form of oxidative DNA damage, causes G---- T and A----C substitutions;citation_pages=166-172;citation_volume=267;citation_year=1992;citation_issue=1;citation_pmid=1730583" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=J. M. Daley;citation_author=P. L. Palmbos;citation_author=D. Wu;citation_author=T. E. Wilson;citation_title=Nonhomologous end joining in yeast.;citation_pages=431-451;citation_volume=39;citation_year=2005;citation_pmid=16285867;citation_doi=10.1146/annurev.genet.39.073003.113340" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools.;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. R. Denver;citation_author=K. Morris;citation_author=M. Lynch;citation_author=W. K. Thomas;citation_title=High mutation rate and predominance of insertions in the Caenorhabditis elegans nuclear genome.;citation_pages=679-682;citation_volume=430;citation_year=2004;citation_issue=7000;citation_pmid=15295601;citation_doi=10.1038/nature02697" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. R. Denver;citation_author=S. Feinberg;citation_author=C. Steding;citation_author=M. Durbin;citation_author=M. Lynch;citation_title=The relative roles of three DNA repair pathways in preventing Caenorhabditis elegans mutation accumulation.;citation_volume=174;citation_year=2006;citation_pmid=16783005;citation_doi=10.1534/genetics.106.059840" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_title=Variation in base-substitution mutation in experimental and natural lineages of Caenorhabditis nematodes.;citation_author=D. R. Denver;citation_author=L. J. Wilhelm;citation_author=D. K. Howe;citation_author=K. Gafner;citation_author=P. C. Dolan;citation_volume=4;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. J. Douzery;citation_author=E. A. Snell;citation_author=E. Bapteste;citation_author=F. Delsuc;citation_author=H. Philippe;citation_title=The timing of eukaryotic evolution: does a relaxed molecular clock reconcile proteins and fossils?;citation_volume=101;citation_year=2004;citation_pmid=15494441;citation_doi=10.1073/pnas.0403984101;citation_georef=2010005309" />
<meta name="citation_reference" content="citation_author=A. Farlow;citation_author=H. Long;citation_author=S. Arnoux;citation_author=W. Sung;citation_author=T. G. Doak;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Goffeau;citation_author=B. G. Barrell;citation_author=H. Bussey;citation_author=R. W. Davis;citation_author=B. Dujon;citation_title=Life with 6000 Genes;citation_pages=546-567;citation_volume=274;citation_year=1996;citation_issue=5287;citation_pmid=8849441;citation_doi=10.1126/science.274.5287.546" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. A. Greene;citation_author=C. A. Codomo;citation_author=N. E. Taylor;citation_author=J. G. Henikoff;citation_author=B. J. Till;citation_title=Spectrum of Chemically Induced Mutations From a Large-Scale Reverse-Genetic Screen in Arabidopsis;citation_pages=731-740;citation_volume=164;citation_year=2003;citation_issue=2;citation_pmid=12807792" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=D. W. Hall;citation_author=R. Mahmoudizad;citation_author=A. W. Hurd;citation_author=S. B. Joseph;citation_title=Spontaneous mutations in diploid Saccharomyces cerevisiae: another thousand cell generations.;citation_pages=229-241;citation_volume=90;citation_year=2008;citation_issue=3;citation_pmid=18593510;citation_doi=10.1017/S0016672308009324" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=D. L. Halligan;citation_author=P. D. Keightley;citation_title=Spontaneous mutation accumulation studies in evolutionary genetics.;citation_pages=151-172;citation_volume=40;citation_year=2009;citation_doi=10.1146/annurev.ecolsys.39.110707.173437" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. S. Heckman;citation_author=D. M. Geiser;citation_author=B. R. Eidell;citation_author=R. L. Stauffer;citation_author=N. L. Kardos;citation_title=Molecular Evidence for the Early Colonization of Land by Fungi and Plants;citation_pages=1129-1133;citation_volume=293;citation_year=2001;citation_issue=5532;citation_pmid=11498589;citation_doi=10.1126/science.1061457;citation_georef=2001065752" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=C. Heichinger;citation_author=C. J. Penkett;citation_author=J. Bähler;citation_author=P. Nurse;citation_title=Genome-wide characterization of fission yeast DNA replication origins;citation_pages=5171-5179;citation_volume=25;citation_year=2006;citation_issue=21;citation_pmid=17053780;citation_doi=10.1038/sj.emboj.7601390" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=R. Hershberg;citation_author=D. A. Petrov;citation_title=Selection on codon bias.;citation_pages=287-299;citation_volume=42;citation_year=2008;citation_pmid=18983258;citation_doi=10.1146/annurev.genet.42.110807.091442" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Y. Ho;citation_author=M. J. Phillips;citation_author=A. Cooper;citation_author=A. J. Drummond;citation_title=Time dependency of molecular rate estimates and systematic overestimation of recent divergence times.;citation_volume=22;citation_year=2005;citation_pmid=15814826;citation_doi=10.1093/molbev/msi145" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=P. D. Hoffman;citation_author=J. M. Leonard;citation_author=G. E. Lindberg;citation_author=S. R. Bollmann;citation_author=J. B. Hays;citation_title=Rapid accumulation of mutations during seed-to-seed propagation of mismatch-repair-defective Arabidopsis;citation_pages=2676-2685;citation_volume=18;citation_year=2004;citation_issue=21;citation_pmid=15520284;citation_doi=10.1101/gad.1217204" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. L. Holbeck;citation_author=J. N. Strathern;citation_title=A Role for REV3 in Mutagenesis During Double-Strand Break Repair in Saccharomyces cerevisiae;citation_pages=1017-1024;citation_volume=147;citation_year=1997;citation_issue=3;citation_pmid=9383049" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res. Fundam. Mol. Mech. Mutagen.;citation_author=T. Humphrey;citation_title=DNA damage and cell cycle control in Schizosaccharomyces pombe.;citation_pages=211-226;citation_volume=451;citation_year=2000;citation_doi=10.1016/S0027-5107(00)00051-8" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=L. D. Hurst;citation_author=E. J. B. Williams;citation_title=Covariation of GC content and the silent site substitution rate in rodents: implications for methodology and for the evolution of isochores.;citation_pages=107-114;citation_volume=261;citation_year=2000;citation_issue=1;citation_pmid=11164042;citation_doi=10.1016/S0378-1119(00)00489-3" />
<meta name="citation_reference" content="citation_journal_title=Nature Genetics;citation_author=D. C. Jeffares;citation_author=C. Rallis;citation_author=A. Rieux;citation_author=D. Speed;citation_author=M. Převorovský;citation_title=The genomic and phenotypic diversity of Schizosaccharomyces pombe.;citation_pages=235-241;citation_volume=47;citation_year=2015;citation_pmid=25665008;citation_doi=10.1038/ng.3215" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. B. Joseph;citation_author=D. W. Hall;citation_title=Spontaneous Mutations in Diploid Saccharomyces cerevisiae: More Beneficial Than Expected;citation_pages=1817-1825;citation_volume=168;citation_year=2004;citation_issue=4;citation_pmid=15611159;citation_doi=10.1534/genetics.104.033761" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=H. Karaoglu;citation_author=C. M. Y. Lee;citation_author=W. Meyer;citation_title=Survey of simple sequence repeats in completed fungal genomes.;citation_volume=22;citation_year=2005;citation_pmid=15563717;citation_doi=10.1093/molbev/msi057" />
<meta name="citation_reference" content="citation_author=P. D. Keightley;citation_author=U. Trivedi;citation_author=M. Thomson;citation_author=F. Oliver;citation_author=S. Kumar;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. H. Knutsen;citation_author=I. D. Rein;citation_author=C. Rothe;citation_author=T. Stokke;citation_author=B. Grallert;citation_title=Cell-cycle analysis of fission yeast cells by flow cytometry.;citation_pages=e17175-e17175;citation_volume=6;citation_year=2011;citation_issue=2;citation_pmid=21386895;citation_doi=10.1371/journal.pone.0017175" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res. Fundam. Mol. Mech. Mutagen.;citation_author=M. Koornneeff;citation_author=L. Dellaert;citation_author=J. Van der Veen;citation_title=EMS-and relation-induced mutation frequencies at individual loci in Arabidopsis thaliana (L.) Heynh.;citation_pages=109-123;citation_volume=93;citation_year=1982;citation_doi=10.1016/0027-5107(82)90129-4" />
<meta name="citation_reference" content="citation_journal_title=C. R. Trav. Lab. Carlsberg., Ser. Physiol.;citation_author=U. Leupold;citation_title=Die vererbung von homothallie und heterothallie bei Schizosaccharomyces pombe.;citation_pages=381-480;citation_volume=24;citation_year=1950" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=G. Levinson;citation_author=G. A. Gutman;citation_title=Slipped-strand mispairing: a major mechanism for DNA sequence evolution;citation_pages=203-221;citation_volume=4;citation_year=1987;citation_issue=3;citation_pmid=3328815" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. Lynch;citation_title=Evolution of the mutation rate.;citation_pages=345-352;citation_volume=26;citation_year=2010;citation_issue=8;citation_pmid=20594608;citation_doi=10.1016/j.tig.2010.05.003" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_author=W. Sung;citation_author=K. Morris;citation_author=N. Coffey;citation_author=C. R. Landry;citation_title=A genome-wide view of the spectrum of spontaneous mutations in yeast.;citation_volume=105;citation_year=2008;citation_pmid=18583475;citation_doi=10.1073/pnas.0803466105" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. K. Mable;citation_author=S. P. Otto;citation_title=Masking and purging mutations following EMS treatment in haploid, diploid and tetraploid yeast (Saccharomyces cerevisiae).;citation_pages=9-26;citation_volume=77;citation_year=2001;citation_issue=1;citation_pmid=11279834;citation_doi=10.1017/S0016672300004821" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. A. Mansour;citation_author=C. Tornier;citation_author=E. Lehmann;citation_author=M. Darmon;citation_author=O. Fleck;citation_title=Control of GT Repeat Stability in Schizosaccharomyces pombe by Mismatch Repair Factors;citation_pages=77-85;citation_volume=158;citation_year=2001;citation_issue=1;citation_pmid=11333219" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. McLaren;citation_author=B. Pritchard;citation_author=D. Rios;citation_author=Y. Chen;citation_author=P. Flicek;citation_title=Deriving the consequences of genomic variants with the Ensembl API and SNP Effect Predictor.;citation_volume=26;citation_year=2010;citation_pmid=20562413;citation_doi=10.1093/bioinformatics/btq330" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=H. J. Muller;citation_title=Types of visible variations induced by X-rays in Drosophila.;citation_pages=299-334;citation_volume=22;citation_year=1930;citation_doi=10.1007/BF02984195" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. W. Nachman;citation_author=S. L. Crowell;citation_title=Estimate of the Mutation Rate per Nucleotide in Humans;citation_pages=297-304;citation_volume=156;citation_year=2000;citation_issue=1;citation_pmid=10978293" />
<meta name="citation_reference" content="citation_journal_title=Journal of Cell Science;citation_journal_abbrev=J. Cell Sci.;citation_author=K. A. Nasmyth;citation_title=A control acting over the initiation of DNA replication in the yeast Schizosaccharomyces pombe;citation_pages=155-168;citation_volume=36;citation_year=1979;citation_issue=1;citation_pmid=457803" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. W. Ness;citation_author=A. D. Morgan;citation_author=N. Colegrave;citation_author=P. D. Keightley;citation_title=Estimate of the spontaneous mutation rate in Chlamydomonas reinhardtii.;citation_volume=192;citation_year=2012;citation_pmid=23051642;citation_doi=10.1534/genetics.112.145078" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. Nishant;citation_author=W. Wei;citation_author=E. Mancera;citation_author=J. L. Argueso;citation_author=A. Schlattl;citation_title=The baker’s yeast diploid genome is remarkably stable in vegetative growth and meiosis.;citation_pages=e1001109;citation_volume=6;citation_year=2010;citation_pmid=20838597;citation_doi=10.1371/journal.pgen.1001109" />
<meta name="citation_reference" content="citation_journal_title=Curr. Genet.;citation_author=O. Niwa;citation_author=M. Yanagida;citation_title=Triploid meiosis and aneuploidy in Schizosaccharomyces pombe: an unstable aneuploid disomic for chromosome III.;citation_pages=463-470;citation_volume=9;citation_year=1985;citation_doi=10.1007/BF00434051" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=O. Niwa;citation_author=Y. Tange;citation_author=A. Kurabayashi;citation_title=Growth arrest and chromosome instability in aneuploid yeast.;citation_pages=937-950;citation_volume=23;citation_year=2006;citation_issue=13;citation_pmid=17072887;citation_doi=10.1002/yea.1411" />
<meta name="citation_reference" content="citation_journal_title=EMBO Rep.;citation_author=C. Park;citation_author=W. Qian;citation_author=J. Zhang;citation_title=Genomic evidence for elevated mutation rates in highly expressed genes.;citation_volume=13;citation_year=2012;citation_pmid=23146897;citation_doi=10.1038/embor.2012.165" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. H. Proffitt;citation_author=J. R. Davie;citation_author=D. Swinton;citation_author=S. Hattman;citation_title=5-Methylcytosine is not detectable in Saccharomyces cerevisiae DNA.;citation_pages=985-988;citation_volume=4;citation_year=1984;citation_issue=5;citation_pmid=6374428;citation_doi=10.1128/MCB.4.5.985" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=H. Raji;citation_author=E. Hartsuiker;citation_title=Double-strand break repair and homologous recombination in Schizosaccharomyces pombe.;citation_pages=963-976;citation_volume=23;citation_year=2006;citation_issue=13;citation_pmid=17072889;citation_doi=10.1002/yea.1414" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Rausch;citation_author=T. Zichner;citation_author=A. Schlattl;citation_author=A. M. Stütz;citation_author=V. Benes;citation_title=DELLY: structural variant discovery by integrated paired-end and split-read analysis.;citation_volume=28;citation_year=2012;citation_pmid=22962449;citation_doi=10.1093/bioinformatics/bts378" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdóttir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. Rudolph;citation_author=C. Kunz;citation_author=S. Parisi;citation_author=E. Lehmann;citation_author=E. Hartsuiker;citation_title=The msh2 Gene of Schizosaccharomyces pombe Is Involved in Mismatch Repair, Mating-Type Switching, and Meiotic Chromosome Organization;citation_pages=241-250;citation_volume=19;citation_year=1999;citation_issue=1;citation_pmid=9858548;citation_doi=10.1128/MCB.19.1.241" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=M. T. Rutter;citation_author=A. Roles;citation_author=J. K. Conner;citation_author=R. G. Shaw;citation_author=F. H. Shaw;citation_title=Fitness of Arabidopsis thaliana mutation accumulation lines whose spontaneous mutations are known.;citation_pages=2335-2339;citation_volume=66;citation_year=2012;citation_issue=7;citation_pmid=22759306;citation_doi=10.1111/j.1558-5646.2012.01583.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. Saxer;citation_author=P. Havlak;citation_author=S. A. Fox;citation_author=M. A. Quance;citation_author=S. Gupta;citation_title=Whole Genome Sequencing of Mutation Accumulation Lines Reveals a Low Mutation Rate in the Social Amoeba Dictyostelium discoideum.;citation_pages=e46759-e46759;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23056439;citation_doi=10.1371/journal.pone.0046759" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. A. Schneider;citation_author=W. S. Rasband;citation_author=K. W. Eliceiri;citation_title=NIH Image to ImageJ: 25 years of image analysis.;citation_pages=671-675;citation_volume=9;citation_year=2012;citation_issue=7;citation_pmid=22930834;citation_doi=10.1038/nmeth.2089" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=D. R. Schrider;citation_author=J. N. Hourmozdi;citation_author=M. W. Hahn;citation_title=Pervasive multinucleotide mutational events in eukaryotes.;citation_pages=1051-1054;citation_volume=21;citation_year=2011;citation_issue=12;citation_pmid=21636278;citation_doi=10.1016/j.cub.2011.05.013" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J.-C. Shen;citation_author=W. M. Rideout;citation_author=P. A. Jones;citation_title=The rate of hydrolytic deamination of 5-methylcytosine in double-stranded DNA;citation_pages=972-976;citation_volume=22;citation_year=1994;citation_issue=6;citation_pmid=8152929;citation_doi=10.1093/nar/22.6.972" />
<meta name="citation_reference" content="citation_journal_title=Cell research;citation_journal_abbrev=Cell Res;citation_author=M. Shrivastav;citation_author=L. P. De Haro;citation_author=J. A. Nickoloff;citation_title=Regulation of DNA double-strand break repair pathway choice.;citation_pages=134-147;citation_volume=18;citation_year=2008;citation_issue=1;citation_pmid=18157161;citation_doi=10.1038/cr.2007.111" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. L. Slater;citation_author=S. O. Sharrow;citation_author=J. J. Gart;citation_title=Cell cycle of Saccharomycescerevisiae in populations growing at different rates.;citation_pages=3850-3854;citation_volume=74;citation_year=1977;citation_issue=9;citation_pmid=333447;citation_doi=10.1073/pnas.74.9.3850" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. A. Stamatoyannopoulos;citation_author=I. Adzhubei;citation_author=R. E. Thurman;citation_author=G. V. Kryukov;citation_author=S. M. Mirkin;citation_title=Human mutation rate associated with DNA replication timing.;citation_pages=393-395;citation_volume=41;citation_year=2009;citation_issue=4;citation_pmid=19287383;citation_doi=10.1038/ng.363" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. N. Strathern;citation_author=B. K. Shafer;citation_author=C. B. McGill;citation_title=DNA synthesis errors associated with double-strand-break repair.;citation_pages=965-972;citation_volume=140;citation_year=1995;citation_issue=3;citation_pmid=7672595" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Sung;citation_author=M. S. Ackerman;citation_author=S. F. Miller;citation_author=T. G. Doak;citation_author=M. Lynch;citation_title=Drift-barrier hypothesis and mutation-rate evolution.;citation_volume=109;citation_year=2012;citation_pmid=23077252;citation_doi=10.1073/pnas.1216223109" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Sung;citation_author=A. E. Tucker;citation_author=T. G. Doak;citation_author=E. Choi;citation_author=W. K. Thomas;citation_title=Extraordinary genome stability in the ciliate Paramecium tetraurelia.;citation_volume=109;citation_year=2012;citation_pmid=23129619;citation_doi=10.1073/pnas.1210663109" />
<meta name="citation_reference" content="citation_journal_title=Analytical Chemistry (Washington, DC);citation_journal_abbrev=Analytical Chemistry (Washington, DC);citation_author=Y. Tang;citation_author=X.-D. Gao;citation_author=Y. Wang;citation_author=B.-F. Yuan;citation_author=Y.-Q. Feng;citation_title=Widespread existence of cytosine methylation in yeast DNA measured by gas chromatography/mass spectrometry.;citation_pages=7249-7255;citation_volume=84;citation_year=2012;citation_issue=16;citation_pmid=22852529;citation_doi=10.1021/ac301727c" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Trapnell;citation_author=L. Pachter;citation_author=S. L. Salzberg;citation_title=TopHat: discovering splice junctions with RNA-Seq.;citation_volume=25;citation_year=2009;citation_pmid=19289445;citation_doi=10.1093/bioinformatics/btp120" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=K. J. Verstrepen;citation_author=A. Jansen;citation_author=F. Lewitter;citation_author=G. R. Fink;citation_title=Intragenic tandem repeats generate functional variability.;citation_pages=986-990;citation_volume=37;citation_year=2005;citation_issue=9;citation_pmid=16086015;citation_doi=10.1038/ng1618" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=C. R. M. Wilkinson;citation_author=R. Bartlett;citation_author=P. Nurse;citation_author=A. P. Bird;citation_title=The fission yeast gene pmt1+ encodes a DNA methyltransferase homologue;citation_pages=203-210;citation_volume=23;citation_year=1995;citation_issue=2;citation_pmid=7862522;citation_doi=10.1093/nar/23.2.203" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=V. Wood;citation_author=R. Gwilliam;citation_author=M.-A. Rajandream;citation_author=M. Lyne;citation_author=R. Lyne;citation_title=The genome sequence of Schizosaccharomyces pombe.;citation_pages=871-880;citation_volume=415;citation_year=2002;citation_issue=6874;citation_pmid=11859360;citation_doi=10.1038/nature724" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=PomBase: a comprehensive online resource for fission yeast.;citation_author=V. Wood;citation_author=M. A. Harris;citation_author=M. D. McDowall;citation_author=K. Rutherford;citation_author=B. W. Vaughan;citation_volume=40;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. O. Zhu;citation_author=M. L. Siegal;citation_author=D. W. Hall;citation_author=D. A. Petrov;citation_title=Precise estimates of mutation rate and spectrum in yeast.;citation_volume=111;citation_year=2014;citation_pmid=24847077;citation_doi=10.1073/pnas.1323011111" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Estimates of Mutation Rates and Spectrum in Schizosaccharomyces pombe Indicate CpG Sites are Highly Mutagenic Despite the Absence of DNA Methylation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="twitter:description" content="We accumulated mutations for 1952 generations in 79 initially identical, haploid lines of the fission yeast Schizosaccharomyces pombe , and then performed whole-genome sequencing to determine the mutation rates and spectrum. We captured 696 spontaneous mutations across the 79 mutation accumulation (MA) lines. We compared the mutation spectrum and rate to a recently published equivalent experiment on the same species, and to another model ascomycetous yeast, the budding yeast Saccharomyces cerevisiae . While the two species are approximately 600 million years diverged from each other, they share similar life histories, genome size and genomic G/C content. We found that Sc. pombe and S. cerevisiae have similar mutation rates, but Sc. pombe exhibits a stronger insertion bias. Intriguingly, we observed an increased mutation rate at cytosine nucleotides, specifically CpG nucleotides, which is also seen in S. cerevisiae . However, the absence of methylation in Sc. pombe and the pattern of mutation at these sites, primarily C → A as opposed to C → T, strongly suggest that the increased mutation rate is not caused by deamination of methylated cytosines. This result implies that the high mutability of CpG dinucleotides in other species may be caused in part by a methylation-independent mechanism. Many of our findings mirror those seen in the recent study, despite the use of different passaging conditions, indicating that MA is a reliable method for estimating mutation rates and spectra." />
<meta name="og-title" property="og:title" content="Genome-Wide Estimates of Mutation Rates and Spectrum in Schizosaccharomyces pombe Indicate CpG Sites are Highly Mutagenic Despite the Absence of DNA Methylation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/1/149" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="We accumulated mutations for 1952 generations in 79 initially identical, haploid lines of the fission yeast Schizosaccharomyces pombe , and then performed whole-genome sequencing to determine the mutation rates and spectrum. We captured 696 spontaneous mutations across the 79 mutation accumulation (MA) lines. We compared the mutation spectrum and rate to a recently published equivalent experiment on the same species, and to another model ascomycetous yeast, the budding yeast Saccharomyces cerevisiae . While the two species are approximately 600 million years diverged from each other, they share similar life histories, genome size and genomic G/C content. We found that Sc. pombe and S. cerevisiae have similar mutation rates, but Sc. pombe exhibits a stronger insertion bias. Intriguingly, we observed an increased mutation rate at cytosine nucleotides, specifically CpG nucleotides, which is also seen in S. cerevisiae . However, the absence of methylation in Sc. pombe and the pattern of mutation at these sites, primarily C → A as opposed to C → T, strongly suggest that the increased mutation rate is not caused by deamination of methylated cytosines. This result implies that the high mutability of CpG dinucleotides in other species may be caused in part by a methylation-independent mechanism. Many of our findings mirror those seen in the recent study, despite the use of different passaging conditions, indicating that MA is a reliable method for estimating mutation rates and spectra." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
    <title>Genome-Wide Estimates of Mutation Rates and Spectrum in Schizosaccharomyces pombe Indicate CpG Sites are Highly Mutagenic Despite the Absence of DNA Methylation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"AGeR7p_U9-Oqn10f44t3YxpjC8yv5w2lzFWHM8gsKuc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460247","apath":"\/ggg\/6\/1\/149.atom","pisa":"ggg;6\/1\/149","ac":{"\/ggg\/6\/1\/149.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/1\/149.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/1\/149"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/1\/149"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/1\/149":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022imnOAbbKumnQHK2g3tn\\\\\\\/rP8bh61bqVcj179H+mlFhu0Mfq0gKM63Y6e+estXBeNuDjwRt8DSHJXRlOQDZWYQWIxla8BFpUSlbHi3YHRQALrgwSI\\\\\\\/H2gRq6Nttrsi71RugdP4l6O0rF3x5NufxtBX8wPLAyJVaNCKWRTDls9+wcA1LOv7NfVnA\\\\\\\/4LVjyJu0+Kxx6WVvdhykgw6W84Q2NKlsCLcwr0xTESAVJxR8aA5cf51bhC77q3KIW\\\\\\\/Udq0eLZAKeGUgsYKyHVivRf\\\\\\\/7uSRXwMe0sRMEe3GrUB3ZTJVmEhW04Qc8I7mKo6fHdHlHImt9S\\\\\\\/fwxZF8sgUk4QJufjoeLSuhgHqRr7MdxlbgyfZdTGmTs8ttMK1eCKYWPBxMOo55Ggot6lqgnWt\\\\\\\/sWDtKW7GwVWEKbHnVZg8Ii1LpI9uMP98+Mp\\\\\\\/S8cNkQmcrh44mgN8X1c2TTIm\\\\\\\/ZgkuzO99JHMK4M9mHGHuYHbfnbpkKwoKPiHRr4kqjiqijfyr1OWHT\\\\\\\/gqG4o9WjBq9WaxmFu5aCo5Y1TCZvak\\\\\\\/v3C+h3h6x4+XTaAWmo3Wf8IzXtagA8XNatOaa7zH8Lkh4tDQV\\\\\\\/2ywyt4gTdU7a6YYw5V73aEJnsks3wShxNjzfiYsXyAq\\\\\\\/4mHxiohOSFbW95v6gYj49XAfRYsqPcV9O3gZC+UYZ24wzBtpppkeDV7x0+gy9VuBnblBzSWp3LiuFFLeEVjBanc1qKDPf0I1MujH7d0tZQChQOWpKtB7GWT8kbPHPJw8g0LzLhcOwXDaZ+g8oYzfljqWyhzCwN+3UQ0zFlaVCq4+lXx7FnsLd\\\\\\\/sEANfH6Lnqg194W86gXm0L1PquheRsP5OzDd9nCBh3ljBhfhF\\\\\\\/vJSWfeeFU4OulwPCFAXJl6bGTRLCowQ8fVD1GYw6kvlhmnsfu646RUf610WDasRpusTLqk9CeggjwusD\\\\\\\/HY61lV2NMBfOVnP46tcYd4oHDC9wPlp89TIzX5hoM0PFz+qUe++YIUfcjQer88hXYXiBImdrGaAmXrzBTEG8c+v2fPkt6M53DPlnUSIf9At0kJna9PRXRrvG2so4uCC0TFZwVuGHqMGBwrxvSPEavR9rWcX33dPvcgrZp7MaskqEIziWEcSMcwKFMqXK\\\\\\\/IJ1dQRa8V55rTYCmBdqkoYoL2hehxTZ+OAJ8sr6l9NRqQrIZP4ioTO+8O9F4LcE8BGm1lwkAwxz2o1C8nURGLLJadh1smli2BjwMf4pQYRvxXi0PhaiH1vji3U2JP730YuQJ4RI4A61++\\\u0022,\\\u0022iv\\\u0022:\\\u0022WKJkB62QwgDtV1+H3Qge2w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002291b0373f62d0f02393cd66f0bd283cef2c1409073e992bd5922765b378b459fc\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226nrJTT3VDzpMamghPV4ZzYsKCjl3cOCUaNliUdAD9vA3JhvtQFrQ+C32RDd\\\\\\\/QNyC1qE\\\\\\\/Wu+UbyrdGzsBPdq8xVUM503flZXJ4dWhnIKZEyJbpymP4h9yMQN4osgNlE++pmFfyrAcRF8IkU0ZMmRHPuqKMfSNvyCbUtWt+Mpj\\\\\\\/59zi\\\\\\\/SprPrXk7dYURcm9AhaceZ+5cQJTYkjNySNg98G5v+hV+At4vdumVuciWukiP9005cdTTq\\\\\\\/WhUaDR3lCwljryCiOtLRhElptHXYNTYb0XvI2\\\\\\\/u5xdtGVHrQfH19hbFqHEk9iIdK4rHMkZsUqyiMl79izns+EUiL64jgUd0C3S07AWpHZhe+xcOTd3LAKFzl5ZwT9DvNl8HOOEZMy9G+rVWn6cOofCtkCfBRU1gzK4m4zwTCrvE3P3v3KIuHRLxjkEeoftZhuG14HputTaAPT5dhZBJasKQ6sIhHybf2Yoir8i5p0AKAZqrIsSiy8itxsschF\\\\\\\/2MtMYm2Nvmzwgx2DwoVi4TiP+uH7wkC1BK+13AdrQ30gHrNbuBBP8R1nokZfV\\\\\\\/jYS9CxTfd\\\\\\\/9bVZMZiXd\\\\\\\/gEIpx84WgHurnO\\\\\\\/41\\\\\\\/lbwjGPsrgVmlV0DHHDGcCgFq79oDIdoSBNogE2beN2sM9nNqmzyvvwYsS0hDaVCXg4U\\\\\\\/q0BZCGqR85KBsKsfVbDnofC1R\\\\\\\/HvikWJf77Uf2RHrc4zFKi7eRwEka9yzWVUCeqJgOe1m70cIlELwet7xSLlDTd8BvLBPMckgHmU++doVEp\\\\\\\/UpA6nLBA4TtJMSFJQGCXpgtjwfBg11gvtBsBYgF3rULfRv3u988FoJ77OENg2TT\\\\\\\/BhfeMkjP8ml9C1Rui9Bh6H0Ek8PjW169ULlSNpZ4KD0vKHhJIe20dIahOz\\\\\\\/wDljBMXruw9BrL9a5s8PfKo82NRuM5P+W53hc7ujTRfKv2+13DIEBy6rRnhF4e1P1VBEN\\\\\\\/o0hddcxqxlBOo1UWZDSjwKCt0y4OsxTshBoYGe6un4uE98+zAcphFkRBj018zmhTFNdvQO2+3oY8ph0hbvUOKQhaT1Rmw0CyGM4y0gjbB858loAVAOEvhPPPhkog2txWHRdNxQsF8KcjCVXtmsdQdSgXAzEfA2Dz5PID677GHvO0YI2KwDCIK4qqeZwMszq9LxvRQEKa58wqcetu+OdLQb5d1uEN7EV4YUx6wiNhByylKdhzBYRiVhsGbskUp650QZkhMjFYNL8PnVynvFarEfXIX7Afg4XwpWCfBq2tva+rFRMRvj7Tbq8JgrM3xzusNG3qkptvY\\\\\\\/vg9XsyjhQjE50taJQRgfEsQiTdUUCWqKOmRnEniOHIQt7rZtVeNO88l2ad6iAjn578IhGouzOdNbG4eOxZuKlvpyECbVhycQ5wTVT4PclhXlM0+opniVegoPkFGs7vIgdcgViKJ8de\\\\\\\/74oe71a60mrp\\\\\\\/WwXtAlRtNC7yz5NCrhT0036Mezljm8AT4xDcwUIO+W0MkGEm+zNgj5PXYv+yBzVnsFyJ+EJ0\\\\\\\/NoeVZQnBQ59sV1J7Tys+0ZRBuqN3OT\\\\\\\/hvZNrDSS+Er9LlhhgvKkqTOf6o3yqJoDlo2DBC5jc7hS\\\\\\\/Qd\\\\\\\/EERhWMgAEVYchkU\\\\\\\/Ht2Waq78pVk6Fl07Ey6M5s\\\\\\\/PFngxGjXkODPvIcitLPKofnnYOpucbb6QeRYwdvKGM+ol15XijzmGJt0sI8FlReXr4j47i49fVP14CmzyHDK\\\\\\\/uhG6MGxjY76bwhVGKB59Lk6XNXnNMZ+ns\\\\\\\/Fep5oU+Pe3j8YdDolA\\\\\\\/nNzy6cYhTzUQtW8fLQuncz5vvqDDvKwj1i3QYshMy3I\\\\\\\/vqQZnPADpHIckCMYGUeuJNwILG2jQ35Hnl+6W1bwfPhW2dq4sEdtOaukKdHiGZSp80yyRNCPLC2TZqFFIyzUpo5CIMmrSae8sDTBwVUWzg5TSXYqQLEh+3S+7\\\\\\\/0vILF+OX5WoC9QuCpE5xIQ\\\\\\\/TmjMCZaCvZc2hT70ReZmYyJ9uMALmKl3ePHRbA8vwrQJkjcJbcuS\\\\\\\/Eft+y6rBTLVPXCcZV0BFOqT6biFDMnc9TXl0fM25XFH9TBkbutob\\\\\\\/pZOn5Ty1LaHpPyJmjl6cxFvsESDioIU+q7fcxaYdlDHb70D9L0okCTp91EB35A2Y9QCrsn9nford4HOo4Aget2i0jdMFYTehMOkCcBatIg7HCCRokAyUEaGUtW4y5pPIa6DE6G3RqA5TO70Y6U5eZC4475GWWb+R6C0ME7yPFUUmIonTJAPcSNzlNpaLJeduf77OQeFmVNDBSb5z5R3xPMZUVExfsaRlvP6RhRIWmixROiZs1QTcl12uKEiyKPwzj4Qqv2TimkaG+l3zfUfxEM5hJPjSSrGBXiPZa\\\\\\\/WGQH1wVkD6gAqiWoqsQwsZEFdG0LHouszxEQ2DXSN85NjTMlB8tKByd1hL5zqvW7a+zPRLfd+3j5ZIky04Be0KIk7rLVV8yCg7yNS0aE6936EjOQq4g7nrTl2Ou\\\\\\\/xpKEri7hHI26w6io4ayVkUkNnOCTfhDfF4qflpDGjnUQOe3SiozPxpGvcwmdmjF2+g3GnAMFA1oX2LuCdKmyKGjIbq2RR1x9EeN5UTQtnIU9UdACAvOnSVzDSTv1xQJEWZlSwEV8YLKPjvT3ceTl74BkeR6ScM\\\\\\\/VTvjY9g7ipALtsp22zDFnqYU1Orx3F7JSfg5keGsxmKhrsr4KX2CmrwwXXIrUGNDeiNA\\\\\\\/tPCwAYgEIeROHvy\\\\\\\/HVl\\\\\\\/QDthYyqUpkTExEDuk01H2NTIoMIW7rbhzdH3SgeaK+s5G5iH7phyZK\\\\\\\/Th1vQc9A6RNlnZxrhA3lvPpS9+FuSn9lSJu4aAD2YJN0YWH8pq9KnXRBL2wvfzqm3zrRBVKIwnz5uETQncnt2p89A\\\\\\\/veFCVcJbalZ254KvwsseJSaox4cMv1PW9\\\\\\\/SSCyjJM8OLcpmquiwhQu2RG1wqAV7YzGalDIH+txv3bxlisN8y4EazPLlnXLMIPdWwVH1UDjj013oVCenkeTa\\\\\\\/J5oQH6yObIUxJNmnGh\\\\\\\/EuTxcQ5BeiGzA3cgCerrurhSp7L4wzYu2OZoUboE\\\\\\\/dIILKHeKGrKeEc8F902gmjr13vGTkCYxUPy407Cf6N4Em6GIABS8\\\\\\\/UICzap5KWuZF61g8Ltd+GXgAsI4CGNo8T\\\\\\\/36\\\\\\\/ZhhxFux+NLz56\\\\\\\/d\\\\\\\/EIRN60Oh9zQUoYIRttYJtHGAKqpQJgcMXsRkaUURK1C4pBPklf38esA6mDsXp5rZ+P4PNuEoT3HE0wlS95g7p\\\\\\\/lIYM2INo6N5YVWnc4X4uvLV5c5Dw10iLXHRCPUvS\\\\\\\/wZBERJjnOUZ\\\\\\\/8x+lXvVK7ha8e5nz9Z5btdAbwMLpy4\\\\\\\/k4DMwGvvURYUBG5gMv2PtCO0v+Cm2vG1WitolZksDsPOJuzeilS6sBw+nCMWHWqSD6PgorFK1DhUMQxF+vH55z5pgnOllVDzo60FMUqP4LtswOE+N5gZ9oSSQjVlbAB4RnE3+JLR19f\\\\\\\/FLd207HAGd2rpM975c0uXasHaIuPSNzfw6MyrmkuuOVe3+U+dE9uJowhLoG8+ZdsG2cRDG+a9\\\\\\\/FBa37FVqPVsRAHlvRyIErN4\\\\\\\/3coQkINw2lOcpsDFgs6AxsMGI8JyGLYqtk58zwSZ4+Jj7uWqt7tyULkvqxSdSf2ql+YavhY+KPE7mlqVJICCbAAazdzbP7\\\\\\\/RsmbCY49Mumi5xvWsfiwGIcJzDWrbD2M+AX8N4IVx7yGK2BZM4dBCCtpBmY6jIEnZApTWlWq8kR0HQfvs48Un8Lk\\\\\\\/RLQNV1UnjWqdCSbpGDslO8WMHZ8lNzIJnOdZlZw8V+d7c\\\\\\\/xdCbSRx+gm2pDdZLE7HaFmj4p4Q0w2Bi2bQSq4ewrWtyWNAnfvJE8fUe9XCBkB6emMhAG3ZFAEsq9EpwNlFJMZ1pTRGzS528A94y6ZO+VJhcWPpLmFYu4XRVDUoUS1QSjC+HpayP2acSti0SJebB8FM3SORCr5KUHF78FD\\\\\\\/OB\\\\\\\/imJI4uxRLT+uLviiK+DPwQ+0VdLdlossq3vRv1D5ZYJRwaOVuNNtRlHdfaIOF2zuzRYcLQS3Eiz9\\\\\\\/1JhoWyBgtn1LD+GVfF+X3mlzCBNPLgrzI5+iTZMm2QZz3dq4Cj8ez5YPs5+WDOhbTUk6jRF0eqNwm2tvCK8mNGMP13G7jAZ0x73lNyp6RZPS+OqV6r8S9FE9VWvAyyjwLBqCEEEB7taYg1+BalSdeWcoMJmVIYrsNVKTnmyFChrVvTGVYjjkM3Su\\\\\\\/AeGjKF3tvqIq9CvZH7kWo90H0A0eILiOiqg\\\\\\\/b89T7elsZAFHK+ajJ3TtxTyKut0DPhwlTtVKa4hwUBrje8zgRU5fKNCIWjrgdRS+nezIRm5su5sqxJUYyLHAnlaeoVPL1l7UjZxkrcSIcLcWqqGpcG++UtjwejGuTphYISX+PtP4ZVJx7VVzGbLJOCe1vt2000IcEATdlNWl59UE7Uw6oQ5lambW05Dys7nPQV2x0FCgFPmDxDj5SpEeqNS+3Am\\\\\\\/0yqpr294Ql1KG+cMy9dMzfG2N1RSFFVD96S4bBRlojtvoZ0GiFuSj+Srjx6lgqgbd7fgVL7UJWTLKDbjdgYB1EvrWDN6ZxztxhnRh4g3zpekwMOXO8V78aU6pv+h+BqCB1yGbjx81U83zHfsPwf\\\\\\\/5vFoIyFrTb9eNuetaHJ1J8UTZ2kPY7sVkKylbL9+Ku8tsf8Q\\\\\\\/mTxlqbjiTOlVQTkufD+E6v6qng62Kz83vizGjZmQoOyTJ6kSG+kDTLqYVTOOH\\\\\\\/yHPGAVehciM0XDILunseGgEIfHrJDwCFwALwhMkOYcp3XzYsSIDaLQPbMqZYdfqm2QrYbV7xwmvhRDlgQF08vBHAyilI4ydeDygSQNUcZz7T2br4yc8kDpkouXTjtI0dgjIHZo1cjK3PWDUQKMsIrVtB1Bs6P6OKdY8j5ImQFUvOUDjmWWth4re22+Kw1QzEYtH8YuJzF0fMX+muaWv8qSK3xnGh\\\\\\\/ZYO3\\\\\\\/H1ab0QaFeKl6rAFQQHJgeIS0b9T3a0MtbiHc5JXFOzH4OFTvY\\\\\\\/zZdLRn7qKQMPaeB\\\\\\\/2+EZ8qGJGhfxAVoxwtgCA3i89i2KBUfc0MjwtD7vDgLXd2mprz+7N2Sj3ioxCB2YSTOwPP0t\\\\\\\/bCaNswu8zeMGo5ZAeVs0aW+FrmKxsVRY66hy3prffyClFna3e1AtSTly3N9gYGh7oReuYvG6Zu0iGDDPbM3pIB55N7HLlmHkF9DHbIvZlUQwz71aFVvaglThedSJpjrVjrvsXY9tMkj+IPnpfazuRpGu32jGleSYPO4zDGG\\\\\\\/7gW8SCJwzLxCr6G4NKdKGQIC1Yk5UW6f7fD6AA72OPHQLhMk2kMVJirhn1Tuw3zN4TxxfP5rSNJj3uAGFId3VTfKa\\\\\\\/HECThT821tp6X6PPZ17VCDFaH\\\\\\\/cAjtX8x4h8DNIyi3gjlKrEsTFwD7tR2ReAaYNkDrREhwVanHuB2oEgo0rMS381rAeUsjqwJU1xWWSdtI4dxFEyVEI3Cb+mueVzjpzHZonfXtPQnBYmmSSwdll+K3NUgTKi7mnHduSZ9d19LngDszpnJuQf553sfaAtcwiFoCFA+2Z66LAXs7tdk67L5HS+vI6oCP0weZ2tqc\\\\\\\/H0jiNNhgVuRsx4Ik5bM7s0\\\\\\\/Ejj4Mg7fFyj74ZbSfxUK39QQ1omzeZkfXODhe94zeV9wQreQiR4j8ud4vloz+VRrAWf+3QQdi7dcLp6tRIjAR09M5jsN7X2aRT1qajpbaUiQ60eaxYkA76I6ZTeOnQNUBHhJnVHCMA5dHMTLwdK+ZpLrG7gwWdPI5ZVS37nPHqSXvCJKaRSLAwgEW9N3+fJQC6L5liOImU5RYTjSTTnaV6KTF4MyVkHXavNJ6WK1DZYkwZJlItJpv0nCkYDl6C5ReUY6kUaC0pvMmoakoMhSi+lw3mOG3vJItmCrwJblVsRl+2OjzFwRECLZ\\\\\\\/0gObB6M+wvAJ1JvMm\\\\\\\/1if8FlgKUAP8j+Ih9GHHq+br5BqBRqzAtRVjSYK+jDSznGwqnYUy0mnWoFRxdZq4pk9CE1kmVly1\\\\\\\/4mf51ZtStHS2umHYNS657xfpLmGxe5tfyPfV3W+UnkG56yKRdIQze3TT2Ckqjp+O9VpXcfdw\\\\\\\/Q2Wud77r\\\\\\\/dUueC5rlN7WAPmk7jGAxRpnhYKx60WaOx8WKEaZdj7Q+pEyJi3mCTUFdLig+s7B75ZNgzh2jEex1R67meY0nMr4bUFjiPyPfPm9gIhR3uqskkdWktiUmnwWmzAPbMA0h0fe9qZ+N03IgftbPCxsnH+nwIznSENIrxUGdDmGxHhHQtB+97apb7vkLqx\\\\\\\/eOpXw4+WiA8N6f+A4f5gkKHkuGYo2YHDwxsStR9HbPgDpRBV9d4E7dNyMASk\\\\\\\/gCgKNK8OtGLGajO8wwccY2XTfTMpWshuBh7U\\\\\\\/sKrwaGZ3Ho0wWQxWxctoyEjbOxdmp5DQkLc2hO19zMuoSVnaiGxdZE+JJ57a2VMuYhAglttK2G4kT6qh7QXpq4YIjpCkFVtVAvyM+lkuwpfgc2pfSda4TiFhdHW3p\\\\\\\/Yu87qRN3Qg\\\\\\\/rmyZant4IkzZ2RxBqDNtbp13pFx8a+CxUvDG47KQSiPYfS+sUoNuM\\\\\\\/Xi6n2je3Rp1GPXd3bKvCCJoUgIvYLQPLwv9ujfWyv6LO12kEx0ylcjh3FMLVaYvDb+DBieKJOPFHCT6bnrNciP8TM1csBBTK37rIsG2b\\\\\\\/kQkNGxCIEVO41b6moRAWBwpKkqBTPlvMp5El3iFg25LfwAvl52RCvwRaSUYxg+fRGzme5S155ZOQ46+NEVcO6mCQnSQWjB\\\\\\\/ZQf1HP0kdmnFCkjxNwduOxeRNUFKt\\\\\\\/Fyr6ukb2qn3byd8ORYab27+wzy90F1LnZ398\\\\\\\/mfRc6jv65vrc2S438hGm9zj\\\\\\\/rxbNUPc0jHLrL5PJR5D376G9iKUW8y0x+BX934SmXnXEQtrmNNnLCETxSCptiHXmwhmqtoiH\\\\\\\/02QuUtYqbww845M4ngo8G\\\\\\\/BO1+dJoRcSaWorSUMgUa1PaTfgaE0XCZ9Sb0MUsZLjoX1ApArg4LLkB+1IIevzkUqEAcFtr0RazOhtnxe+zmWgJrotiJrOfCIz9T+LO5wliazifjurEEAqBB8uEuKJzJfE4dD6kGxPpHoyMMepgyViNgjiVz7aTcotpwmprQf7P\\\\\\\/INlqdkJUNm1vmVGhdkzIDzHpeHzn\\\\\\\/Y2N7w+K\\\\\\\/nbzPhUHZcu0UD8yvwiPN5l3JP78dU\\\\\\\/f4z0DSqy5Q3fO9QoGI+tTUwTLFStzmP4OK2oeOlGUts6B85E37H1fWAR8QAm\\\\\\\/kgXj02VCjnAAnsm4nQ7Hr90\\\\\\\/aMWbGWqMO1ihl5yFbgDPMSy56DM8hOEVedAS8KIrk6Jp4K5CvU8Acr4sW8Xj7gXyFclA8MQMRD8OD54NBPfevxyA+oua05KgEC4WO5JU5Zo5lJ2pTgshdGT\\\\\\\/NuSvd1MC1j965ZcC1EiHZgfXjHUn01elgjoacMemA4rUTiS7Or1Q0\\\\\\\/jUtiIrF5HuvphiW1KdtyS8iEnN7L\\\\\\\/2durPhrAog+He4mGFe3c1d4mgk+V5+0VC\\\\\\\/cECtziDkwzHajIyJKEWlkJt37CoG9JodC9AYy+cIVpn5nU\\\\\\\/2jsRG83ubHGtfxtjcwFSrztwgaMnDz7yaIB+LSbE9kF0gyWzA0qUyS0Z0WkHWK6dHmVT2oODRQjs5vnWs4g+pVntnzBhHU0O0US9aKOff2fONYU8x+8ddA1bMN60aGjZJ4wF5QOGKr7ULCDFaHQbyR1n7wdCCSPa6+J6cxUugcdUNHMH3BL9yyMMtJMH1M0f9OtZTfhNd0KhwsjQrMC9Vzvq805aHG6yUrzIWomEtRSDAqzrSNzkjlSEcOwdPa7FFX1351RXp4mq6E4Rh09EaomduXASf9sddx1SndVq4k4dt8840OFp9Ejmp9XRsOzN7HNnXkz9nCWrDU4Wv5OJtrmeZK1t\\\\\\\/M9nTOJubt\\\\\\\/jUlHSCRXqMdgMXIwS8MdoqJQxcW991ZBMxk2Nf1bMhgJ9+Jaq97A+AR4c\\\\\\\/pCfbsauc0xlI7bn3MKWrTcTnHQQuwAram2GWoYMG0tx1phec6jAVzKjEYPnJ4UTJz6p6TsrE\\\\\\\/lflmA1egLzYljZib4PgxkVAQFGevKxxTuP3Io+IW8oLW54znY5IpXS30z5kXMikFAQzE0m7ljsq0kL5UdrCkTWEYhHovEIKZ\\\\\\\/Qi7WODw\\\\\\\/z+wInLetDNrsay7Va8M2mKUtHz743KQKE+kL4+dTNVW4096bHjagthu9uVRt5t1D4HJ3\\\\\\\/1agCJ9\\\\\\\/OktSjwlS5bb4TTLcZnYOoDd1ZWhBbvc\\\\\\\/Kijzb13T6D4YjrOpEJNEXuc0Sk6Fp6TnrbASgZT+v1lyW+4cJpoHT85YAyMROGTJwgcTlra03zQhxaktoi+ARFzuso2CT2+0QbG9Zetjp318NjPZcE58RxlFLgIvPVkRLivu5K5pAFelZuTwqRb5BD3pAL2y1m8AW7Tnbx8Z3Tdax6KJdZBsobpRWVXE8M1pujK7CjALmdzK6wOJLDxB9lY++5JB3pviWZr8RHy1wkkuM+OWrPpk60cFgN\\\\\\\/NMQCOBgV4Ie2QpBT10haZgGznnszPucPzRy+PafIXctxpnZa\\\\\\\/+YYCUF1fDvZPA8MRUZgJQ\\\\\\\/kOAoAnyP7cKr0Z+5VkDCCufdZ6QCrUPdeKurU7l6LP9dO7RclOUwFKi87TZ5+vRk9bUiZXn38zinhSv7eOPLR0WQ9q6lsQWR\\\\\\\/rMsuHxK6W4CC+R3SylqaUD8gpldqXaBql822HI0nUlRdzJ+dOWcF4lEGQCJwJ8bbCTT89ClvqYkDBocbKnms0BS0r5b9TR8mxpTFD3fN\\\\\\\/Bq0U7G8CnDkMIAgd2i8AgOJpOO9GO\\\\\\\/9V28zBt6IMR8IyogH3pcthvhGUxauf9XX+zaphzbjmb0B6xQFR999ruxJOVHuzslWXxlV8oZFF7WXs4gJrBgeATO2L2o4vR3GLvelX+AHKMZa256oeu5YUHmdehBe09mjAyyyzclnPLi2N5fMyUYoQawZjwHuTtn5fjn8g+bwLPpgfMzeH3lO0+4l1\\\\\\\/f6Ij1dsic2W\\\\\\\/WbAcUh82Ccz2EXFd5+gdmKhwRsB6Z8LcI+qAeWYMj+0XfKwI84rDQh7jUsA6Ayit10vv5pefeTVclot51sB6NkGWG65PTiZNNdwScYIksvD+FID7iRnLUIpEfg0HKBLqYmBntEGRK4xD\\\\\\\/ajhv6R1riABFajn3oq0P5SIJ868ha9MvKcdt69fFML5DG8NqOudeJfJwT8FpOV1yfAftgT627wfh3WHsjSfcuwn7hwNz\\\\\\\/mWM7+vi3+bYf+oamj5xZXGgTZN+hVxLzvzSlqAUbKlbux5xkB49c0KCXZxjGEQ5ZEGeggztPBjGYyNvtZeY5oY5SEYqc7ivXkbH\\\\\\\/X0u3Gy\\\\\\\/xb+QgTEYK77gpFkDD60ox6WeYqLDlFLGRqrEC2\\\\\\\/Y+y7msiK3YF2RElvyi90MNdpKWsiNmvfG13Y8ugjzmjY27Uau8PD\\\\\\\/D7uDphZFuUQ9fJAp5YVtZnnLKWZSWPBr5iZzohcfFupYxmCuCN\\\\\\\/BUDVa9o1mF+L+HbIBKnvZMeH+giVCkoM37pPB1EdwAEt7+jZL1KgUUNQ3laoTSlDHcP83a5ZYB1\\\\\\\/BMVvPUL0HkM0Zq92vporOwVnerHogAU06i3W+HCa+qGAWnNTDhCtpzk41Jxq1hk8OimmJo3fDTEzSVTFNhH9V0QF8Ipf\\\\\\\/kAf12l3A8U5p1tHo8i9eVDI1TbHtBmAnLKzP+gu+d4rzWZtfbfZ4MCJwY9kLLYNNSM8agb0bgE8A0ItjUe7bYbSRQAFS36lfi2k4lK6dt7gcQEGbFDMVxtDBr04aOXClXd8DzpT1cmuh4E\\\\\\\/p5gXZ\\\\\\\/suJpChESg8qDhhykJ9BTWTt8AR4hgImq1ACI7hUpjsgRbHQXz48SNMmbf2+OFI3A6ja\\\\\\\/rQTINNU5qAP8GjycTR6hLmjzNGroS88eeEJPwc5QBVuT+6jT+lZN1mJxKS4GZtDRi8dSfCKEAVWSAiNSCJj1b+kUXJobpKBWAtefiaX6XeSeUbqdoww88Dk8jwDkTMKWY5tW1AIsC0pW17TvGzQFXiU8PtitWnm8GOBGaBpDk9Saa6bQ\\\\\\\/AGtoQtG0GBmGVsEkWBkUdm6LTwo4\\\\\\\/7SbMpmXVk1dQ1+\\\\\\\/20FgXZT9DZVg4iiJeS1O2DkxQyxtOpS6iCqqFwX5F+swg6FavcBvwTivkhYerXvY28cMC+895CA8fqHRO0cMh01ZDsO25siGfpUt09R28XQZ8lprOhHSQRUB+U8j+GW\\\\\\\/4PgMIsSxMitHpjM3EaA6TKyAl4T8N40TKsON5x1yGNXMXZePjRxDNSkApBxRLSrfUiemqvLu2bRyvfaMe8lW+A2EICK6ALfuoYbv3B205Bgh1daxhz6W72b+\\\\\\\/3q4zoJECfElLwo5D6Dr9Hb\\\\\\\/J9UzEbviCzqaRizp56xAzigKAOc9aO7WPsS2mbHfJQUJ2lDABy3r+PDWBvCYEz\\\\\\\/gGqMhqTTqUcXWZ2emfK4rAUi\\\\\\\/NbdaxlaTJq61jFVIMjwRzgk7h47RzPDv7HqI\\\\\\\/fSrlW1bJl9lKPvv2gX4xcjFyHl+NgRo40sagHvCeCcfztAfRKnyZ2DCz5js65QJQdt5ENNmR0R8HGgh9DtoG5NesiiWtVAQN0zP6b4QUTYs+Vy26hzckdsuQe0Y+NVN4YH+0Y5F8RxzJI0B2atwUzbPOooncYznVTv1h3PM1ra7hv7Bptso\\\\\\\/txzv4M6g3HSktsJLClL32A41GU5WtyJqCyCidCZmsiaisVRK6xObFjXTNzxBiG31VJYXTx6ln2NdZ5+gYbjKZEjWyNyMWoBw2tJG9zrftRyndLbRtPLPAnVBzIu2rK7C1MD8geXo5W19RdPNY3zgwdoCAhxj+J7GfLNHjcddj0Ux0DS\\\\\\\/PdiIy8T7hTvFYo8i3XdS1j2gEcpwRxK5CBX98hVZtU\\\\\\\/psoHlY8izNE\\\\\\\/DfkH7x4h8WNGQi4gkNKPnJgTtloVW+9gfv9t2zliPYlfrcohvP5JdtwV286kt0QNeBfklWQ6KohxbuAjAHKc0ld35bRfk85mxRtqrys1K0t9ByMXnmgkqc6cjG+cnwG97leZNyHMMorr01dzfdY6OLnHUa\\\\\\\/NCDuY0OZrBwlQzqCBFvsthEwHo76rvAxAv08ReKE+1iE1l3wQTmKqpiY7cDvHShzEZd5Sbnz+971hBi9\\\\\\\/wSMpYWZ0GjDQIJXUi+tD+4X8HeOsvrg2lI6zIDhxdct43I6eC8gzRHD7HWkIMtpMfZECjCOOgfIBuZ0QRDEtjh5wucnLQJNjQVAAoktXJ\\\\\\\/aLLZWXJTmn3IuBxzVnctv8o1IArVc52OVmRIz+F5IJ7i1TkJwjMLujADepZ6r6YWP6t3MJMZFkFeGTa60Igff4VBkvClpAppdMJyoHSI54tJKm5SC+Hy\\\\\\\/4iFCJFLzW7izJqrdSinB33FMsOcYNJO+s3C1oFBO406UNdUaFKgjub5D+Fj+7BoIpR6mGFdP6DKRsDTwiIjfezv4FsG+jTRvSunyetAacSwXPoiiiHw1itgxZhnEKYW7HygzFv71wAGaX6xPIi9HKEFiBPOYKOviSaPvBC3B6mJSRHBLc0BRjiWYtE9SQa7EUvpqb\\\\\\\/sLXehlhIFhRwFumxyc0KuGpoAlRHVoW\\\\\\\/P2wdD0ntdnHnWF6ufE9qGrLSg9uCMUqWhnvHEBDgpL2ROYTeJcD1FOiiEyqwyQvccs+Z9w4T0VSNxaUVbQlBDUbbPq9pVYOMKTa1krnD49FHJ1ZRvR3T6n4U5cTo7bznY92c5e1NI9rpWj8L6WGJQH9+MZTXVoJYh5hjS3F8ATc19nJQYVCXej5FILvS8OpYh4rGP1O8gjbcZ0PmLIOUqS09si++LPNLJO3gFugk9eZ1Aoa5WiBY37ysbE9pFVR7oOItNI05wnpPXBs+p1iE3heyKkhOkmApKoSfH9g5b5NhMaBpiuQhXtTKgJJoUFM0+CjUgg6O1FlzJ95\\\\\\\/oVT+ov7boWnKZ++PMf0NjAhDpKRKGO9mAMZSRoLi\\\\\\\/cUHSN++BzQO3nkWB023hPWDbsGlgveHzsq\\\\\\\/KIgcJWn9plZsBUlU+g7fbV+k7pHRF9ucQCifUg3QmQAKrnOkXTl3TAAuvzWveI+OtNIH1yMsOWv\\\\\\\/Gzsy6+fbNIjGM5G2RuEzNIkXHvUsqK439s4TNf3Bzh53V7n0TzlnPdVTHALw0S2pptE5WbKgh\\\\\\\/HJQTyusM09y0Vj1DZt6Av4BH8S3vgR+KjP0s+aeqaRjmQ6fCGF0UI\\\\\\\/ToeijaHCw704ZI9kZ7\\\\\\\/9KeRm1IN\\\\\\\/uJDJ366aUm\\\\\\\/0J99yfPaFuR6WcaDg021t4CdyrIK4b7db9aDXYybxPKzueLhX7DSvXqv1YYaOwAznz68RCL2NTZLcIcwB3wdCqiIplYrOEQeFnrFunuaCYLHGUx1Z3AkFgiBXMZuiCZWVYK4iYOK6f1qunaW\\\\\\\/xCtRlx4tRmlM61QywMgHCd0VIsX48l9kk8j3HJmQga8irEg0It2WeDpA36FmrshIt49CsbQ5Z8O23W6QQKAbYRO7AMkgSmVo+UMmIZntJYxlUViYOO4MZCt9ao\\\\\\\/hwyevRueyPo68UO8QZB6ZYML+Oi65kT8J8xQ9KYF0aDnAc3UvY6oRa14JspzRbCfxWaxtq4zQ4oj+InaoEoF0O05a2FtmZleJJtY\\\\\\\/o5JKaZUUrgd73id8hzj\\\\\\\/X+sNWuQLKDFdcO6TyfINfR7HRw1iXu7Kem3Tsi5kNwxOLthhth6b5ltRXfyg\\\\\\\/8PV+h2CIFS1QASSKZUJkTx5WZK\\\\\\\/j7PfzQ+LmcvErUumZYoTb5ElJYPZBUuLjwHBH5QFqQSSUFSzINWHc8PgK\\\\\\\/eCQ4KBGM28f9zBX+OPHbhb0spnWzJ9SLFwrJfyfwJ+qUXsyBWTgfjQtQt\\\\\\\/pcamCxpW093dQtHNvFDnMV29blVVEB49UDC2rdKvy1C73nrT6bpK+3Xn1\\\\\\\/go1jNunG9Yk\\\\\\\/kmLMJO2dPUOJ+5utUqCvxxnQKR2hYrRVZ7LXLNP3RXDN8P66pedGvAmWBLDQTMUqsLg3p2pAVnoI22p20WWF2ZJWj2wIzHGGx7YFMhKS9\\\\\\\/JCJcwXuv5vcZnPJG4oQSOhjew7+HYE9\\\\\\\/XtptEG5S4PIn6wwm0ePhCWnhqdXPnYo\\\\\\\/6XfGnISqoKTfIQx5ZfbdSxfXWbq2uIoBo\\\\\\\/343HVLJHLWMcj9ql3M7ZpsDM7AV2UJnR7UGIvp\\\\\\\/pr018YFOnlE6qaWFzWlcYAvSLrlZfJRaYxbwhv6TN2DxbdYwQ6XbCyx209whdlW53v9gg6ByQc9se83rGZGm10lxlETs7zsFb1EYlr1TkH3Bow5p+aRjfOYTMTBeZBIL+b7jOvkEKz86rNOYTZwBgZfYh4QohRbBU0Bwf5jh0fcWEtZ\\\\\\\/pwKKbt1V1EVTPsKEkjn3EALyZG8QD9upwvq+naeHkCqVW2b7hjEWKmgkcgntsDaTYmIcP+j1pEPIA8cqpqAzSURPAzdsGBw8i4+pYPXlWFz+6NW1h4AAfOJ081ZdMAgvO0pluhLuwGMw4+RX\\\\\\\/njyPSXGQy37kvfoeAF6WHgC1Y5pkbAVU0vRZkWIGdKlWlh\\\\\\\/hsD+DihQBcsu5krGimKlTbZAP+8M294ElcjAO4XH+VExbk2J2gOLZleVe1nkZFa9lGnqJNUNb9xuqQp9J0e55175ohnJ\\\\\\\/vtfUTUlSySf2PJVanIOc8zKHJZSMYZarIub4rNOKmJpC1iHYsaIJ9h5h5EwYVgl1pFREhPa0EsGTV9GdlDndK1dDGlS8VeH5YPUZfXJF6phbNJqhvDxmfx7pnJ+qZPcUwPbcjO\\\\\\\/\\\\\\\/bUZzsq5Au3gRs5GjIcYvkRUCC3lRbFEQ\\\\\\\/bLMXiE7Sy1gMdgH1+eHNrjghASHk9SMSR40qwvVMRZ2YbIWCOyhCgfblBd3XBegg7pd8GdZbWwp48OU4JfcnHj1uddBpxzeDcx9dXZtXVz6WaTRx0qauLGR1Q17lUEW6gE0VRvyw37PwEwG6r1ZU9LVduglkwha1DbKiBhFHkmO\\\\\\\/E1Id9Hla8Z6303YXObdDY5FvdwpNyWM87gwOBbDY4lP6679irGC7DMmvvHmdtoFQmkBpT29DK3cdb3FDcg22UruR8r1Z13dv26BELfDmXenmeJ6BFHHiYoeDfyxrMBNr88eSic4uTrkw53SxPgZjVgqLBgyAlz1SYHu4L1+zqKXH6tjCQREASdefJFQb30Dw8KgrmSYRiMw8vqHWVTqK57FVN4IDtOg+uru8+gWnjZykqBzIekuACqfMHQZGPlGTFVw7HCP17umGvCnRrgtvobcKXouCM\\\\\\\/AcMJpn1Dlwf+pwcM9mh7CuSb+2\\\\\\\/rp\\\\\\\/bfjHNjj2lHKjrzYFCcmcbQXYL3\\\\\\\/3kBMeoB1SLI2yOkNQLZn3P2lHnhaVh\\\\\\\/3YVY3R0S3AM9XjJ0+23NvKOIMU3aV7LjAzVLLObezYH2KeyoTP1sQcHI0QtRxpTboj0AtYNoFLhBT9NOKx5YGW6Q\\\\\\\/k\\\\\\\/CJ9msqMZDyzRiiWTGEWwYnRQ+mA0BscJtxmOGnuZg68eqFx9Zfvr+kv7za+AKpbZrqmWNa+3WvHgGewv5rSjRK2U8tO6nvOoQue0aMIKCGak1pWk23zFInivQkLj2xSOjINX983gqNunUsJeTBaTpz0BUmkbyxQw\\\\\\\/zqeEPX9Qxdsp+Fsibv44IzpZY7pRMJA7YBgZiwHvH1TjEuETF8EZ0REleoNfHWnEccToMQQLnhfxU3K19K08cRDsR0KvIS7YHzZLZ2rGpQqTmbpNJMHstOyJkgAxmCcK+BqsLfyOhewEnSYZhLW6m6582Z1YiauJeMc1aH0\\\\\\\/gLgN5tJxl6H5glss3GpJV5iVH+LxQorDOV\\\\\\\/EqKfpp0O9JO8D6o\\\\\\\/ZceMXWonNGRjRydylc6lCvNsl3yShNn2OcZxrir2EMDhqAExqks\\\\\\\/28YcedjR1xcF2B7HY7xIpTOQozG15NyhGrsjx0oDYK\\\\\\\/C0QAvGn9CFL0MRV7jIQMNDfcYbXlEq7Cz3MdG8fbJLd1dafvYisl+2iV1Y9MNkzRvuvTQxl9M6qO2AgHQosTBJ6XQHc3ZqkOXikY1Tyt7Cd903pUvXBdqRJm87nddyDKfFPLNhuF4q9Zcc4nalUo0Twfo9cnTC4UgtMJW\\\\\\\/USkjfGpmlyFQF0GWh9aKZwsRtcUHjHDyzp8s4JRKj0uoDrxPl4BwKiXRGo+ju4qHsdHUfB14nNnVKHVkXRYc+BXVvfoH3aBosECwWxGQ2UBFxt7uBLWUq+8hrYr+FS+sICE3BUi3XHG4h+I0PS0XtlOaQS8ITP\\\\\\\/+aBqj+JO8e377I+SD3A8rum2XHmtEag9NoDWM2XvSiUpqjaYAavVsEo0JtblI9420u8HHmAw5kuwQQZIHw\\\\\\\/+iy3yRbOWLUTFsC1LQQH42\\\\\\\/YkUpe0gmNn0Jj49IwzTx4MonrCYVQxiI5KzXU\\\\\\\/6F\\\\\\\/5YSHRuCVgUrM2LCbemsd7ZF1pUncEpLgP1i5exH2XHTBrNH7r3+LCtaK3TPQWMrqZ+kdxBzbWGDvC8nuMw\\\\\\\/OiZQjKwyjicTw3gpXjrDeRWBSgdvFIYR6ec244jbkncLFjc1WgzQFW3RNAN7SfdEafbFp6wCIEtGDuYmkZC3TMMi1IDEN+WlhPuQrIehnRzO4RoIyvvJwIjvDsVcxl+XV9jma3JNxomGLu3kTQSe\\\\\\\/kyBN1Qonm6osexi71O2ILuu3KvCYYbnXRvdhNGuo5x9KVd\\\\\\\/WKhwt5yIZYbP2Cq3A\\\\\\\/Hioh+7c8reWcBZDMYd3momVzugjnVTaQ1kB2N3DSdKzZNCabOX4gJj3EadeLmHGoN20tmsy76ZyVd62lZlrrJ9PIWjOlwkvbeBsAqaCMMYF0XRgtK77S6KTLOJJrjaWE+aTR7UymT7UiwnU1M4JBF9+cAQbOiUhnZqkyIXQTvBzttijLwhkElwRObY5LW+9G9gLd6DYeBHqYoAw9Zog\\\\\\\/wH0CIUccextviHPcbfrdj\\\\\\\/FuQs16tyurxSevTIONyUJizQt7OAoyg5Vy5SZDjPkG35f7N3kEtx8i3APSTr+eJiqeA03sZVLlnyrlXZ5zHIZ2Sj1OJ1a\\\\\\\/wdIJYDLlCBijRprm1jAaKFtmqHi+SnAtjAdRErGlwUmCi5z5d7rSvNwQeJsRD47MdkfudlqSTaUvXpwQ2q2sIchky2+23iWrHibbJeLahlBZdY5UudKYDYGBDPeFPp+XpT8vRl8hCezcnefDJ+ccvVeF+f7TlVSc11PSnznp+DuzDeZOYHLMr9ZvqxJPR8eVh0y5TwftqBl4M3PKMevdFR7\\\\\\\/536O4ca3A+HUW5EYPMRhCkuxc9lMm30guLR2g\\\\\\\/U1R+Xg2gq5f8ASxYYBUK0GdKeX5oAXtw3j9XxhgikmoRg7GGLYbNDQBanyJchFw1vlrTHENr0mxqggPbXnz0ycAB0WNqifNuSpU6in7XmV+d4ND9e7HyoN6TSrfKxyTAxtWs7k3L+D4L9qx47qyYSRguzv+z840siqjBvvz8OUUfLh0r9ifrhw+fjvjqKGqekkTLTk9Ow9tYWi36VhT7KzhfA0mJvwU0dXnC2Yqxcx4EA6\\\\\\\/w1jYUZmn8vqK4D0O5X2O1WJQORTV7kztwuoDZomhYVL0lywnNJcya44hKs\\\\\\\/zjeLrLId29Aa3e7C3ARXsKXJyRQyWcJ+gALEQcqKOw+4tfJOHtbyojeT+Pu7PGfc3mM778wdpM2V6mppqZnNFPJFmeN15DZWevePVf3Hoo6cecivC4hRs8e6fdxwyDnlY8M3kNz\\\\\\\/7hAIwKOTajcJFryJkKqryly1fuWCSzGQz+xTEx2Aa1ump0nUrfNkRmrAjJKV\\\\\\\/SqNGt9EIvDGMl3b7zVWsFC60MBqXj7Q6t+E8VTvZtbdmb6tqF2dicLmzBjvumZQgvp29+Y\\\\\\\/W5o1N+xEfKV93bNOsRmm6jvriso7iJsimqp+qKkTUBDgSJKDl\\\\\\\/UKxSg7G0+RmnnvUthGG+K\\\\\\\/dSy4ga8MBkAdiEXyGcgqpkxJyHoMluRr\\\\\\\/mFHgvXZHZICC6j7GghQ5\\\\\\\/QUQIRB4cLFfy75NzTLn9ovQ57KWPxuZwPcNjQ2C1PbROjvSCI6SeNubgfn05hbmOslWXQjchV42DYPG8Blyv9eRE++Ewli4yvNFGSRkEPGoNpp3mwgZ8Mh7K7zgzYS\\\\\\\/nmCDua1MFP\\\\\\\/u07iiivXYD4ZyVTW8IAgvTaF632cUkPJzSEYEVCi8+krVolg2nWH7ZC5nPoAb0HcRz+iNEcA4WP1s\\\\\\\/m\\\\\\\/+eVoaaMMkHsSzeP5I48K9gAPu8YA0+Hu7WH6s9OXNh2DsyYFaHgZF3ml4eUGj\\\\\\\/Tesh1VQ+kD6CKzstICPgWcqnFhpMLXoZuMSTcJQOoh\\\\\\\/QRbgMbdjpZeZrBDxBIaFywhI70HelHTcgICnKOfesz8Uprvj+rqSEnINW9KIOcoZBptcoiVGKkZcy+I+5zug0EgbmfblqtnHnRYWVyfPhCU4wzw09KDPPviKFr+wFY526mDtTco+TNIDy06bpeTrDmS1U0nXeHp+L7stEEI7TkenViJ1Pvsv5roSPqH6QZgr4jCuebAi1DYX4lIMCtYsIRDQqx5OiBBuqjxa2OrPuUm\\\\\\\/M8iFgwrIQD1ctP+vNY\\\\\\\/w\\\\\\\/HxWUIWIuKh+N+rdN1UjvG+Q1Hez0\\\\\\\/5y777NhvDuTqscWshQEQJUVyWz3EieY4SzUMq44PM84gGOwmx7d6nDgxs0fnj037UacE5ExM++Mlc\\\\\\\/ws4+rBk+1jMXlWiG8IaJE8+tHGog4vgmW20kqiXEzGmshuRVOiLjxQK\\\\\\\/o907piG2InEp97DuiLUFLNl53wrwT+HbIMGlLpEAG9J1I99mcDwufxmO1ykJPZG08V0k9C+sBe7+DdI9RaCa6hj7n6NRKmjoZ\\\\\\\/1PWYQwcddQzXL94i\\\\\\\/KQxbXBP3Rl0UU9dH6QQOaHf4j+NdR+C7G+u92o8eg2sLJ1qIz2dtIb0Klr2P64ctlc1qXlH2jBEOw1fTZrbbQ99iTUx84ddaP9mLNKTXW5JNz8ueUPVTFn743mJXEVAZJ6\\\\\\\/HPnS9OCdydXG1rqA9rK3FjPV+yUOMP2Fh\\\\\\\/prf40IKz6USz7kdJ9dlPlS35xeiD55g76jQR+YvN4VRffagJgu0Hio3+3\\\\\\\/dJ6iR8w9+YoDi+ETGckU9nMGMPMoGP32DEZywxwV1+tJz5FUo0wY5hPycnJrRJ7qEYoQV7NvgekRJhaGwskb5UPtdjH1oUkUMygfQ2CWML8Kc\\\\\\\/w13Fg1OZiq\\\\\\\/gqQRaDyKmlDMs6tLeSNC+eWEHUGAVYeqLjNtAe2K91rBWvE03hS1Up\\\\\\\/fv5BTnnndPkzIVjQ84KSE9rU4MeKX0Gc\\\\\\\/LzeKpgzRk\\\\\\\/+Pn5vUZcUtL0Yhp0J4PRC8ssAv6KotZMgloIHwPSrNTPMaT7RWZJuxYmFON+rHZONm+2YazEnna09CcRGj0hZznqv3o4xqVf9FqPS8to0jQTbCI0+MOK1mNqdZXNO7dZOi3YZ8JpHdD15oCY6+o\\\\\\\/9lJao3hxq7nHmN7vwKgOPbXGzNkJzuTLtBZ7hILnGxzmG7Cg1F2brRCEAHxFHREZWmzzPqkr46laI\\\\\\\/x+wviBEBOD1xqQIynt525+ZhxN40mTNErmjksHvo4+cC6Mou5o2zLnrIIBR2DVxRM2wbjRjM7z7X5GgBfu0DjPQljfWAuBiBHiY7seKYLADfkrZ+RKg\\\\\\\/R09qwg247SxhXCx65BItczrnv96jmPL9T2UaYSwF5GS6MniJvsWrwnCIEB9xOhdUrwbkLtO+jQHljhlaNkRx+h\\\\\\\/D46b9lWM+aiLK1YPTpkUyUP76pBdjFIRies+atOj5b6iwtBDFHpIeFWFUhwmSW690q7xEjGXQhUlfuedrSboZPoaPYA1l6E80MHIZ2ciDQnBUioYZaI1GU4ahDBmvyA4oyurshg575ruMKUawfAC2J7ENwW87SQBsa5Ee9KyAcDS0809ULJc+X1vBOCSlsc16xYnCK3+C651fvSZ80Ba+R+CstxfKaXaNXeXcJvr8eOiBKCJotxRSAMTOCDyxesghZ9KTVMO7yNegwH6aIiC0\\\\\\\/7CidpIIBC1u+\\\\\\\/ztIrWGGt6XUSVzKvKUcug\\\\\\\/p5ji0+Y9uglLHYJr69b9qVhgDFaKLh99ncvIEoYSgsqQn6pdAA+W2gQpw0OA7elHDVf399V6Lnk87fpxzu56MZpU1ZXWleGRAAKfFOTWo7P4W8oPYRhcdqm2mMHMjA+qRmWZW8lKrAbcwvbf6pl1Yipgge7HlX2HZ8rq4FUDTzMk1vhqnoSfDQEMN3gcDGcL1nDzmmNbW1grrwdfr1xUdnynVmBFHeqRBIVnh3gM1XD9FpDKNGIpx\\\\\\\/QOJixOnJAAX6IxI1sZVIApad8+mtPwJUzlyCC977yFXCPfWlUsPBi4+ZMG+9H+zCYb0p1MFOkZHjmSiqPWrOyS2+hAKli5rgRXhr6aPXwFhH5K0ICw9Eq47F7sh43vIV8IpTxYw6PR8JiUfeDYiWPoWv\\\\\\\/UjTrIGjCnHG\\\\\\\/cddw8WzpRk4gxHIjBEyg\\\\\\\/Hw3ogUsyrahLKuosHMNyBY40aati\\\\\\\/K5JOqVSe1ZefXBtRs23nhnNrFvzzuBKFmUzZBJ3qw40ky\\\\\\\/zwSUB+XRzcplmDHpFGAp8ZVkT6niwStr7WyeHRZmzrnTtL+SSjJUKcshZPukoJjWySPK9kj65c27cefNDtI59sTuGveaWLNND4uOm7K8kr+UkMJw8dMxBpL2pKzdY1klyF6kD7It1tILnHJGnawhNvbH3lEjlwkngXPZWNlme\\\\\\\/yEffxcaLKRy8zbDbTlCzZnC3jWfKRMKaxhkZvtXpl0xlHOVrtPS\\\\\\\/KwqwPva2jGCZVHjXd7yuCoD1UrEZyzQQwHcpVf8UFoCSQc9V1x+inCpLKiAUgVNflxoZb2G+fEpEWJCfJJHcil1mk9Jd9F0L2fyybY\\\\\\\/2Gzi\\\\\\\/oATnFhzxp3Qcq7NlS8pSnuOHn\\\\\\\/OZYZc78xLZ9CBF43yaYCYHP8q5qYW9iAn3tbcAolkjihAnTwMeqPei8jI+nGGLAwQuvs0FTrICL6dU58OOO2fIwr6bSRDDNKvadS8zyjo+X8wjjErr7Wh40hwv7TWQyRn8wAFVA14MWu2DIgOCCUksamN7xniBWOAVOi920Np+53irHiZs2HoYlZbLFn+aMxA1PwvRpCjCE8Tl0TbSHts6lAe0Sn5a7191v1LKt4hjXocadv3oVSsn0EFhuaeULG1H6nRZGPiU04E6cqtjW2HCEFelDPcsJtEyDG7P\\\\\\\/S8Hl1Ogtil59jNbPNPAK1ZijIap83AUiy5EJGd2JQNLoernc6ypQcTUMA7dVo9tS4iswD4lrm+uwk9iU19I3fL5MHIgP6D7U33eMNGUkdyHc7w7hqGTCl4HZTidifvalqgjwzB9jIAJKUVu9+6PA+RwIamdtX7FACDhcDK1qi7O9bm9oVmYDdj2ZL2eBkmM50GJuc80bRYZ19FrdRmm1D3wppqAzVKHiVw\\\\\\\/LI\\\\\\\/Y30RaCL9wI2AMGIBTOI\\\\\\\/Rxh6y1iMi89f0W1M8J69RmoFTz3oiVA\\\\\\\/XAw23A+e4WUNSTv7Ckwr9D\\\\\\\/1qfPX9p5yMruDbVEaXRelMp35ag+VxlCZfYWubZpj7CVnNECSC\\\\\\\/MvTlVtzo4uxLt7g6FelGcwY1rx+bbY+nGCTJY89JIsuiRvApSLLCeiwYj9CXADYVmtptpPVmBmFMeqTt\\\\\\\/4Krn0xwvGYyTOVhSzsWqT\\\\\\\/hXs95wIyFl5LfWby5Ns7U9GCMxHCw4ICxPFtqzl8L45irkKP9cZN4T47\\\\\\\/avyyek+o6wyl1lobh6A\\\\\\\/F6q4AYZSvjYp5Dyx+XJFvfVSNSx4JJS19rx0Um7EzRrsFf9Ypz\\\\\\\/iJsWsFwYCZlJPKJEd0vNr\\\\\\\/9ZsLrTIURxwFnZUSTTm7yGhJlppfm3UPanVfWL0srQh0CA95W7Nhl+navaYNQTYnwRh8eQWuywq8PxVl9TDQlsH6sK5EpbQHI2pPsi3ROg1X4npc1oIX4iXLeDfexl\\\\\\\/GbRljGkKpcoeZjbwXl4azqLvqh5qfB+CWoqbGtw6hTAfAH6wjtKznmV4KIyCgulGLK84BSZuUxQ95zs+OoQHC1g9e9+CbVBuP9dS31kKZhwBUZ7l6thboc8n5pKxHk2mIAe0gROKqH0BigSi1tCcgYKaxIP2SfpaDZ08bIiuqPkLL63q8H5EhMVp7g5XOBOEmUJ\\\\\\\/iYVDyQz8bvoT\\\\\\\/u\\\\\\\/Cwdjq27PgX2uCk8sVcCrONKEmEAa\\\\\\\/pYH+tkmB\\\\\\\/tegbP\\\\\\\/lrTvbIdUi3cI0gwsUmzBkzrSTaPfqA0XPbQfsbkPKUs2tpMTMYq4Gw\\\\\\\/tqgFbhu8Z4u4yd1EzJFnWY9qcf+j5S+IvgTeMnyEi4lzZ2xkYwKUnUSOErXac\\\\\\\/2TAdtoxEwKxlpjRKrnEeHM5tnqMBntvim5yK0pOVhwemVjn\\\\\\\/\\\\\\\/X9gVEewX7HBCJOQBkF04ZxZt0MAr5fKSIaCM4PK8HgNZIsaprdj+MFh3+mHhldLCTKNd3TatfhndtwbNV10VKgTaU\\\\\\\/1kFnHnAa0bJunKfc2IBL93mhXSze9FRzokuB3S8IxFCXEDpO8veRzK+QCp1pxOjc1juganCqYWlhm1k62+EItO4WA9hLRuw15+aZCzD5FQEUzHDrYCIZv\\\\\\\/+\\\\\\\/yP6z6y+ctZNO92C5ixbkmiRzkfWCo5JjDGuNR0KwKRRwFW9eR8r+eKuz9bWGCfYt7v\\\\\\\/B8hic\\\\\\\/7byAAk4EJSlUjS4SImGuhVJvrvMp0Z6E4rh8Jv7TDlNfkrM9A7ouXUhfIOEnllPuo3iiwCQ+8Nw2G8+EMbnZ1lE6SA+tRtqlLPalxn+\\\\\\\/xZun1udtct4vRXh9oFkcUMnVZ4AB46dre0Et8j4\\\\\\\/p7A21TBhJ42Mo4CI2NJ2Tr8cqzMWSDijRq\\\\\\\/+zIJ2xByQC+IG9nalKkDy4gDnlvoFj8V+rolv+wuJ8oBVjjnJqvlV\\\\\\\/KiObN\\\\\\\/zz8m2ysV44aCe\\\\\\\/UTAP6ff7HL8D3kFmEIIKG6iMvj5tFr7aL2IhWXjHIu2RmSlSA7s3FsBqJdsP0+u2Yljf\\\\\\\/eVzfdOy\\\\\\\/xAWNZZ1D3ZinT5fIjKOkisMseml2vvjtKwxERKty5acYOoK9f2iIQBvcZE5ttxdrRpTpWs0RpFYJbh+SRSDv7m9hEdjsY6mJCV7JOU5\\\\\\\/AzqOLGdqR6bxvA89ECjEBxZxO0ha0g7SXEi5Y\\\\\\\/NNaWY8yWCPeSJAAWY6QiJdB\\\\\\\/KVDxFGdOSCaLE46m\\\\\\\/H4+CXk\\\\\\\/Yl5pljM4YKf58BdSQ3HMtHBkt8XfymRQTFC+InhW4ndnHPikxyKIJKT8TZDCp2MbNk4\\\\\\\/wWkdLnFD\\\\\\\/HvSlIcKgR1+dHZ\\\\\\\/SYfxHp9alerC2mwQ0E7o9vQ48pKw6HDhzweoU7oUqR6dEGelt43L9HH0t03TWnde3GoVGtuS0vsW3Y2B39rGZs38cgskEd2Szxj1blv78sKOMFHBiYpD4FuN3jfXHc1IvQKDtvY4ersVAMo5ThZqjaUAhOMcApuZy8SoJrbc9dgiA0IqtM8vtoYiF5UihtOCy86Ru2VkpJ2Cwgtq9lLxKTy5a1RJBcvxSH7rHXOZqoLAnpUNIics7WIc89TQo3YTRFd+gBD6Q098S+xazgA\\\\\\\/7nvcsuiwXkYYTrQZ+QC5u\\\\\\\/EWAMPBfS9TK5Ugs2QrPq1Avli2V43mdI3N5IzoGi+nGWYAOjYNxnuux3VL+5YbplDPN12o2QBqPY\\\\\\\/2TuB\\\\\\\/ugCAQUkJaPa2SMDw9eS2b64dRLIYxAiLPPsKKwwYtTn6C1IgpMLpzGQ5QajNAAXTgG8Yu0ctpTqU1CHMGxk7zGYYJ2HIId9c5rqsYrjf805Z8VVAZk4a7lfUWzYItrGk0JEyHrR7wLwCBdUvtdnYcXdvWxEK6fFZUsB9xR7+9fSPBQYXCeGU1HE\\\\\\\/CY54rQrmdQQ+VhBCH2o0t+beAe0DybTU27ZHMxOent0tyKuYqZOQ5RS+6OcG+TrO+0e4KB6nObls3E5EhXO\\\\\\\/P3651gJ\\\\\\\/\\\\\\\/ySVEUj7+2ulBgnjn4GN5gTc4jGBKKGq+Q14WJSJDQ\\\\\\\/gZPZIGmCBRF7XEtIkEgAHzPJaj0Fdkv+1eF1ldGATdWFuM0dkzmdFScUOgMXFOZPN9eSYobXrmqP+0U6zxnI4eMYGiVXgjrYmk+x9gpRzep1RISdSS+OMb\\\\\\\/m4bLVXeqJDXAlDCpJHg7XKXCgxmlcaUlTbaZk\\\\\\\/7owBWGuY5cHssg5Lexuy\\\\\\\/i2jMU+b26MksXkWkRpLH+KqtY7QsWpdhVBMClGdK8Aa4Ivz08Rs2VHatKSaj8wwQWhaXPe38ud6TjvsiUdoLYRCs4WpXxjCe43QsSq37wbXeorwPLOwW\\\\\\\/FYVANAHwixTOXTciBkAfhA7GfUjcihunkLXT5U4+B56Y6K7xPoGnX+skwClEH7AkOYjs8LaYSDF5avgfnbyglvAXOOTI+QeGq1R27qg\\\\\\\/49SkIqLaa3\\\\\\\/nF1lOX1RSpOvYmmGw5HyNPffeH7LpuASg5+BYFXV9I5DYasMIOHXnoV8bzVKfUrHYFJg\\\\\\\/C5O8zjl5YrhTOGGnmA6hwAcnRoUn5hWzem\\\\\\\/K24YbVHZVA5B6FuJy5fkKL1a0zIrvX+Xhdm6xl+xmZhCL9vLBv4fFoH2O5ycktGdePCdHIW+HcXw11gmrVBiiWsweHFw3aoJNaaacu8XLsgzpO+tYqTZKkHGJQft8ET07D9AB1z8bkY+mE2PLNYmyLK5XaIPnrqInT+CTepF2wv+za7Dw5PuMbRxRayrUpel\\\\\\\/sLpzPBxQuamMfBAu5HV8MTCXOnVyxno339h7wEb7XvN7iPrncsBVm24McRsXk6F68bIOBkjmtYA1eaepVtuiGPNz+GLFg6xEgzRdlUe5x0Y4P4UE1fesrDGYVlYqzWvfwtQV3mmX1cMWq\\\\\\\/\\\\\\\/cKr63qIO1md1BIWajc+r1x3lz8BGpnBz\\\\\\\/i1SI6v3ttSnLUOqK6iuoOvrP24pcB8KDaposAhZefSDYMbcm6wcy9TxgEHX3pRiEHiz9p8+\\\\\\\/g7COvjSVYgyaFYbTFpe391neYmYjpqcycFFXDue2mI2HYYQzhSAxFeesQUaq\\\\\\\/24AF79C5P62gqRPhpaNXi3Btwg8K9\\\\\\\/csqXE9y1IkAhj03NKpQnCXmTfL51+NJiHelI1xMobZt\\\\\\\/6Kqc5H3A5b33nECVsYm9ZwbBkjtm+WIqm1hbCWjZ+qDeL3B\\\\\\\/EvmTryrYSNi7lBzDdfam\\\\\\\/Brib1hGEoi+mjy2RU0LVTA9OkQ0TYP6aPjBE66tSFp\\\\\\\/ARSmsjB3dPfHlfPCHCeWLVkLgObiHgPjx6PyDaSdHfFwMnfD3qnIMpBAhoEBMIuGSxCm8U2QYalERIrL3NReYqvbTajwaEvvNH1HMz8qzrr7xZURoOTc9cqS4EXsHaQXDPPqwcsiqO0t8OOxesmRqoMG5FakD10jV2ZYU6tA05ZOb8GqXvda8pTCD\\\\\\\/hhyoNuzwVT4skbg7hMrcbi6N4C1\\\\\\\/B8vCVfLF9bpY+y9nt+RYTAsT4pecEYqnOB70cTtnh+m5kQRe49BmBVEpby8soSWurEiSjtorYwHyk9H9U9DZcTKyk\\\\\\\/pjCdg\\\\\\\/nruRr7f0bka3dKaJ7fXG2p\\\\\\\/U1EipR6okSHkxldjMDWc0jjQyb0xA\\\\\\\/aCwWzY32nGYBOi18MnYakh9SiCca48EBXm8+kf5vIqufKQogWsBADrBQHXbIuCafNhUJnwn4MY00iINw\\\\\\\/NaASQKQHGtajx9mQQIglm9C2\\\\\\\/M7+DOEIHnvASPyVxLfPQSxsSwTdGE+G5Ma28OoTeCShRrodjJQfjvJyhfkDjrnrPqMujR6thi8sUTdKPZNCVUsGYkWJYmX8okIppedoUFL\\\\\\\/a7pg9D7SP\\\\\\\/\\\\\\\/Z\\\\\\\/y8W1fayMR8Bmj\\\\\\\/TFI35b1g7QZrkpwrNecU\\\\\\\/uOFh27N\\\\\\\/7AVCE\\\\\\\/pszcuGFAytJ9BQEhAkd2NPcQwMgegnzMqIxpPyWqkD2rI+hHeZE6jGFuTnWEwBe9mf1CCHzn0bsf1WF2vcy1fK+gCv5mwRl3cEmGStk+RFqS2IF9yHHaWAycSQRn+CVQ9mtEAifJsyxL5Hh975Jja2PBnlxOYh5KvtCsur1+GoD3gyX1q3bNfYHhCXmbTRpYca+XrY2LU4YmegUvtphfd9UkCXR7X7H09QFT1SEFnnP5MuxgkDKkmJeVgxfUW325Vkes3RNJA5Eax0GGbsr5Xjc74A2XsOqmSSrhkVgTwPwgXiSkxhrp+sbrnnRtYEv3m\\\\\\\/jL\\\\\\\/6XVa1Q3T7gOvxZHsJSXkd5tk3wFhjJlm4zNdRHWSm8PFn6PpWfJOiL40xr3hglMOKQG82ridjQ6Y87WdF5BhuQtnaCRnSqhSfakR19aNhYobvLm+PKCOYRHijdEH53kvXGHj3pFTUHyfLxEhffiQk8QtK\\\\\\\/dfPnOdBACTuYWgz2KvSnB4lC9PhKmJG5WGXi4Bt2mGaoXeTnaIXFg8X0k5KlEKfpIRJ5XFSctcWebCJYCs5jLmiHnM9p3uZq0jiya4V3WIIOBylNiS7Eszxt14OVXzPOcxONOJsk4rbkAgdps\\\\\\\/RPdTzFFhuOkq2RMRg520+ZLZVId3RLX171L8hT1BghTig7HAn2rYmuKAihMruMs7x1vR6\\\\\\\/o9cgIM4\\\\\\\/g6dalDSi4tXswQGly4G6J+HnClBrvN\\\\\\\/bYjQG3TMeaY5ZSYXDUNiG0jywIyktNKo9kpOiVai7AUxMIo1vtJIoM6S+mUdvojXbUVMO5xJmn2FHluVk\\\\\\\/7cUmdZDnmmxTAc4R\\\\\\\/SmzTPxzjhfTbOxVbxKSPurzhLOW+VRbscxoNwnRx23w6+3Yxzm\\\\\\\/sASedfwfRYWn2cVWWGTNT6clfH0W7QrG2knVc1\\\\\\\/AzBp35GNffOcLPQSRwp+mi01+fz9riDHYQltXSucWV0NzY20jpRfJZ4TZgKEJBhFk6oq6AcbHw0aWnB\\\\\\\/VtNAqt13KBuU+umZPaqFa2sfvYxryw7d9wAaWMKqlpiKVWzNh7tNkd1BUE9VUdCnrfX2ifYVm4h3RI8rivgDc3e1NVfKK9CklVi4ovZMCWC+njLsWokhSQ7hcHTxEswgBzJDvCLzoVV8ldGtVdgU4bt30mdwhOZd86pW4Ctq5bOcn+c5Gc3IRfEtRow\\\\\\\/GC3tl4Y53Y7XcdBv8QT\\\\\\\/H4NpjapQ1GDMxhhYz6d8XjPlCpT12TkLpyTxyDPaMt25ZWFSIJcJYFdBumVCHa07mT8tvCKyRtOF4kTHemGzXH9zTe6lQKEyG1e70AwXIDAag6udJsdgl7Fc1vBLVTIqcAy7sxGBD\\\\\\\/z9haHDg5dKn6Tspo0XmmUHGXbg58ozspECCCnbE9hBgKx9Xk7sIkO3pnXOMF+NNO00+Q0YGmPysvzPpfobG\\\\\\\/rphnRs4zUOa4o\\\\\\\/cmYALQTB7mSgT+6B3\\\\\\\/LNfY9uir0Aq0GUcA6m\\\\\\\/szVZdKClnNyRCLZobNieIdDidhynYhB8o1qZHZzx60rNP+hIjr\\\\\\\/xHO\\\\\\\/pbbbJ5Yoprjy5tv5UQibxn9fwHQvA+tjQrz2qEEenY2z3\\\\\\\/tmnA8R14+vjgt5zhlDY1\\\\\\\/WiCl4btKai47pxNH+fHZ3DZ9hhwpziRLXA62P+lyOg1NPuZxi0yM+zf4sS4H+rQi\\\\\\\/0qj9I0Uj3Ih7MSTuRzBc+j4wZNlfdNVZD0vVmAKOciKzVUYtmEWb8IsHJ1Mw40KfObBBZ59TZJI36yDhIlWD6w3owBov8+pv6ggyXuAbI4mIhH4d+4vWfYPQ19PoA1gG513SVdo7vTuDWiVHXNPKgbewofRN8lLIS3ArpDsShNB1J26eL+GC9QnVtovOk9\\\\\\\/uGKny98Np3ue0kF+20\\\\\\\/YB96WQip6bJxmupera6HEP2cV5eZAjhr1brdfIEdq+IqgySjSTkc6xJxw7rQxboBQXPaoMwzb\\\\\\\/GbwajaozDq9sU3WbWTrvQxXuYzArw3ulqmfJ6HEu6qzsDI0jdQgErFAPgCwaN5RxO73LBAbRbfbt84X9PbQn8ppekUYkcgpkkpOCW3lqVZfeIcsV+kkjj2XFYMBJe70WHkcT\\\\\\\/uBmSKqR5r\\\\\\\/KMm6W5DOJ6MlmsSjtm1VY6iPW+PeI2\\\\\\\/Bl3B28akArivPdYyCqb8XPkO3Zh\\\\\\\/8piSis2B\\\\\\\/aqq1Y6cmaPhWRtO3Avcg6aLGSdQJFIlDrZvHQEstaZBPOHild0qu4f4CStvrfXEpZFx3Aj61ZYE3falk4664F+4XOCDnyDB\\\\\\\/DpS17Qa0UaFiOYObtX1rXP+oRkrXd67TNCMyQcOdSH9BGbI+ZHxr+sQ+cpgWQOKvKbBxnWRwPtIoa6Zm8ScI11rutuDnvd3v\\\\\\\/Hg2SXtPFkac4SNMmmrDmk7\\\\\\\/4Ox7u\\\\\\\/X21uXlPPoOF01Ufp9\\\\\\\/JD2cz741zWn7sIYe8fLUuQEWSFgEB0LexLJWJDz82X7z\\\\\\\/QtU7z8Lo7xhnZJTsbqWKF4nbBRrbbvSgd+\\\\\\\/9+eTkX5DNXHigGijSkLpZz2VOosH3P5OrxpT8rZuVNS0eLdPgi79ngRGs8lEc3LLpcbTh4hAUP0RnfTmHfYPHB0RsqwGaVfTon8DFJ7wOYq\\\\\\\/GXo2m9Fmb7pjPCAzH\\\\\\\/fdVKiE3kbyBccF7gbBW1oU7S1uT+e5a6CY3LKZyxX+5WFEdOIdZZCD1B4CjFbPOctyx+HhpOy+jVWyEUETa\\\\\\\/4Ztjurldra0bpJzKWec6RjX\\\\\\\/DSjPezt90MBLn+vUcZ0dm7e5v2CkMDb\\\\\\\/oVDgOg80zPC6UKgMqnQg03EexwMIJfvMqGboQ2KKPKxMdePq\\\\\\\/1ZK1ztGqpgOVG9aKF1mS9tcHBh53\\\\\\\/liiwFsFrMsJWe0wC1GzxmCU+35ywDgAd6bZuN5B8k+xvAnEVfKjHdgc\\\\\\\/D7p3Ee+DeAyuN8mZ9JzxfEb6lnktkrv7m09QrNpjjq2r7oDQwzktAqpuSb5LrNoeyJ+oUmD3lVk\\\\\\\/AO9FGqWcd0TOPA6mA1nUQbCmufSexXYO6ll8ZCXivkFJoEfDfXcr7+D0UXrDEy87nERPreIKwHQpbDZH570E6rR13dOV+dOsDgSHpNYM54\\\\\\\/Vg\\\\\\\/I4afTGMqSESHxJMqYeDoaK3DQtRfl\\\\\\\/qWIk80Gcb2HqVHZqmsybgUBr34c9t\\\u0022,\\\u0022iv\\\u0022:\\\u0022V0GdBeDxjfJCTv6KqYm57w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002286d7c4c8762c41e2a86d8e25791f11c03dde138a445f951caddadf71cb4eaeec\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226TegdrJp21aG4fk33CMwkDrKADUcKq0mdJZv11ki2nCxOWUGmFpoZ2ChguZnlTqKiCFyDtRIO2RrNIDakLilkaAxrTDPxiyf6DXkLMYDodSLBRUGgjFJevlmOl0OsDKcv4UV8iUmCcy3VpUrMc87Y6L4zGw1cM8v9x8QNcljH4SgLUhECcWfW8Qx+lV0VZk1YOscPvawJ0z3U9AcVTf2GWoqn9xZv3nHB\\\\\\\/Im94ESIQBzpHOHlTEIhVuAVyGQwnQavJAl0xV02+wrQfxqIcYklbc24GDlUzAhMOHevMunSyNh\\\\\\\/5X+KSsqjzL0AzUWmcvpJGq4i2MBSr4iDkFEzdRp4ZV9N5pyMuWVfB+dbdPXLG6O2rpJZzOAppKNKuon1gmO0HMGQdlQuqKSLa+68QR+hvim8jgW2Zj\\\\\\\/EgmKWzIs1GZzXZkP72TqeKpYjpEnsGyH7bjqw0WdosdgnVJgx5usWpNHl\\\\\\\/qi8vEo1n0CGj9cbg+p61jHgm4BVRJoGd0XeLTw2P8K7qI5pRryFtOBb2VBika\\\\\\\/gOBLwQ+pYkSXILL0NKV3qr7n5it+5fPQxffbzhspX70FzZvjXUfzB6Cmiom8EtKrsviJ\\\\\\\/OclndZMF0vr1XnDapdHM+T0ejvUXM8Yvp9lh6HJuy9r4XU6eO1cexK9EdLIH\\\\\\\/KVNdiYYBG4XMlCY1YzMyEW8kqig2a5ihuPh8N4eX3XGgAZF6Ok1zUfpUDmN\\\\\\\/6o3JDZJezgcuIlx3OtSVsVmXAgnmCEPcT8v2\\\\\\\/RAM2ZXawAtJa7dkrxHFFFJgoFYWmmB72VZd5ZWYew9uaF3TJ4muofjQfrFRyXeIDsO1znG57bvvhWcC8RbEbmxj03yENmvGIfHgQ1Il0huw\\\\\\\/5A9lScGbk0LETdfVcw0HXHRngKDe0c6KC+HYcgpoLmIodqpoyFM77PhAIMWq13zLlEXygRNLOwaM4rHgYXdU1xNIHr5slWL5BxHWCUthSC6sPlf+T1JW1J67GoaS2lSvVXsVgEBHMBcCTJ98BSP+Q0qlKzEbTU2Xtr7P6c3i\\\\\\\/\\\\\\\/aIotDAcojejGd2chOD2f0kwWt2E2pmMSnT0s0VSdBSj8n7w+7j5m+p3QHP30IjmHPE\\\\\\\/RBhqvBAlIMTJ8NC5MgnklOkzil4irmJ+upKaQGbEuuJ3Sxhm29LMvW1zKg9OIl3KiwyYD7JIvLFDeZGBXSHlrZFYs3eh+kXc19tnRFX5SPdGGpoMx9pO3o6ONUF+LoVGV52lOQNzwK7AtTXaaw91Py8eXuVnjfRJne\\\\\\\/NBGQ9SmlwR0Vgmq5w+Zc3WI4t75d4lwFNuPLKNOum7L57aGdDIsUizchBLalvIPccj4kznq3koCRJrO3VdqFHkRR7IK9KZbiZtlj6AEDZ4wO3kEMsh1\\\\\\\/VTr1X5G0dUpCn9JZJ9XVZwtKN9hutjyqE0rJN9ximJ5PpqFk8gSK+KJyRRv+XZn5Szf8opFvKY8MiW4QdlY6Zr3i8BD3ytH\\\\\\\/xcpiX+gcYrm2BNrHJwWp3NwA9+DyP\\\\\\\/SkaSYpxS4pA5uB\\\\\\\/2tIT9vYIcBYssUHVngy2yb\\\\\\\/7TiRTJA5O+CK0zf0IPVcKs74NFkABLwYMQw9MNMVmP1S+gSJ0NGiCeSFqfVKmSt5NJ\\\\\\\/3D9NUmc9936Bx0P7e08X3\\\\\\\/GFOgTtOzkyINSzck1kSCmbBt71JuKMAjdwqQTmCkqdXaDGqkRVPfZMat9NvxPj7xaMbNfJV1QbmGKbNB\\\\\\\/X4q91vg2fuqKS\\\\\\\/rC2ZppRDkVoXRzxuJlmjVN+I0dO0D\\\\\\\/dIs9s\\\\\\\/K7WTzpRojzTrsEIbrqlLT9MAfEttpqhx5Q1TDCK\\\\\\\/EHKbLcc+HMXJyDpM6qBGftbcQnhWid1oBA5ymgUFgoRFS+GdTdXv4az7SrwcND\\\\\\\/QKNLG89thrWKRaP+0qhYOeNN2sRiBn+IN+ODz+f6gbfrJvAEN3I3KgorjbK6dCWZIDSBDCtSlOAP5lI54kJzUSCvQLYkWladnLdV23wj+xsItQHzXYa\\\\\\\/5Jq8uXY+dKIU6EIl4iyPtWthsNHt+gaDCfyHo=\\\u0022,\\\u0022iv\\\u0022:\\\u0022NGeEaVtlzZbOnxO+TQLRUQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f97ac408231461c246cc2f72be33d671642d4901dac4991ade459f03582a0478\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460247 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/149" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bwaYppPFpdsQALkjLS1aHKNm-j7JyZt1GwLxKzIjG4o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/149" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-cW1kFreKWnfwaAYerZkLSv1DZHUM2r8zf88KBXDIMC4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/141" title="Comprehensive Transcriptome Analysis of Six Catfish Species from an Altitude Gradient Reveals Adaptive Evolution in Tibetan Fishes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/161" title="Genomic Analysis of ATP Efflux in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460247" id="node460247" data-pisa="ggg;6/1/149" data-pisa-master="ggg;g3.115.022129" data-apath="/ggg/6/1/149.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Estimates of Mutation Rates and Spectrum in <em>Schizosaccharomyces pombe</em> Indicate CpG Sites are Highly Mutagenic Despite the Absence of DNA Methylation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Megan G. Behringer</span> and <span class="highwire-citation-author" data-delta="1">David W. Hall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">149-160;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022129 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460247" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Megan G. Behringer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Megan%2BG.%2BBehringer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Behringer%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMegan%2BG.%2BBehringer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:megbehri@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">megbehri@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David W. Hall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BW.%2BHall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hall%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BW.%2BHall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/1/149" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460247" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/149.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460247" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/149.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460247" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/149.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460247" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/1/149.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460247" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460247/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">We accumulated mutations for 1952 generations in 79 initially identical, haploid lines of the fission yeast <em>Schizosaccharomyces pombe</em>, and then performed whole-genome sequencing to determine the mutation rates and spectrum. We captured 696 spontaneous mutations across the 79 mutation accumulation (MA) lines. We compared the mutation spectrum and rate to a recently published equivalent experiment on the same species, and to another model ascomycetous yeast, the budding yeast <em>Saccharomyces cerevisiae</em>. While the two species are approximately 600 million years diverged from each other, they share similar life histories, genome size and genomic G/C content. We found that <em>Sc. pombe</em> and <em>S. cerevisiae</em> have similar mutation rates, but <em>Sc. pombe</em> exhibits a stronger insertion bias. Intriguingly, we observed an increased mutation rate at cytosine nucleotides, specifically CpG nucleotides, which is also seen in <em>S. cerevisiae</em>. However, the absence of methylation in <em>Sc. pombe</em> and the pattern of mutation at these sites, primarily C → A as opposed to C → T, strongly suggest that the increased mutation rate is not caused by deamination of methylated cytosines. This result implies that the high mutability of CpG dinucleotides in other species may be caused in part by a methylation-independent mechanism. Many of our findings mirror those seen in the recent study, despite the use of different passaging conditions, indicating that MA is a reliable method for estimating mutation rates and spectra.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mutation-accumulation" class="hw-term hw-article-keyword hw-article-keyword-mutation-accumulation" rel="nofollow">mutation accumulation</a></li><li class="kwd"><a href="/keyword/insertion-bias" class="hw-term hw-article-keyword hw-article-keyword-insertion-bias" rel="nofollow">insertion bias</a></li><li class="kwd"><a href="/keyword/cytosine-mutation" class="hw-term hw-article-keyword hw-article-keyword-cytosine-mutation" rel="nofollow">cytosine mutation</a></li></ul><p id="p-3">Spontaneous mutation is the fuel for evolution and the ultimate source of all genetic differences within and between species. A complete understanding of the mutational process, both in terms of the rate at which different mutations arise, including kind and location, and the determination of their fitness effects, is thus critical to understanding genetic variation at all levels. However, elucidating the important parameters of mutation is difficult for two reasons. First, spontaneous mutations occur infrequently, making it difficult to obtain large samples of spontaneous mutations to robustly detect patterns. One workaround is to artificially increase the mutation rate using chemical mutagens and X-rays (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Muller 1930</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Greene <em>et al.</em> 2003</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Blumenstiel <em>et al.</em> 2009</a>), or genetic methods such as repair pathway knock-outs (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hoffman <em>et al.</em> 2004</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Denver <em>et al.</em> 2006</a>). However, it is clear that such manipulations bias the mutational spectrum in various ways (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Koornneeff <em>et al.</em> 1982</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Greene <em>et al.</em> 2003</a>). Another way to deal with the rarity of spontaneous mutations is to examine genetic differences between individuals, populations or species that have arisen via mutation. Unfortunately, because many mutations are acted upon by natural selection, this raises the second main difficulty with studying spontaneous mutations: the observed genetic variation has been acted upon by selection and is thus a biased sample of spontaneous mutations (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Nachman and Crowell 2000</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ho <em>et al.</em> 2005</a>). Exacerbating this problem is the finding that sites in the genome previously thought to be essentially free of selection, such as those in intronic regions, intergenic regions and fourfold redundant codon positions are, in fact, often constrained by selection, making the study of mutation at these sites biased by selection (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andolfatto 2005</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hershberg and Petrov 2008</a>).</p><p id="p-4">One approach that has been employed to overcome the problems of rarity and selection in studying spontaneous mutations is the mutation accumulation (MA) experiment. MA experiments maintain multiple, initially identical lines at very low effective population sizes for many generations (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Halligan and Keightley 2009</a>). Lines accumulate spontaneous mutations at rates proportional to their occurrence since selection is ineffective at enriching for beneficial or decreasing deleterious mutations, unless their fitness effects are large (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sung <em>et al.</em> 2012a</a>). Each line accumulates only a few mutations but, by having numerous lines, several hundred mutations can be captured across the lines. Whole-genome sequencing (WGS) of the MA lines allows mutations to be identified and used to estimate the frequency and spectrum of spontaneous mutation. This approach has been used to examine spontaneous mutations in MA lines in a number of eukaryotic species including <em>Arabidopsis thaliana</em>, <em>Caenorhabditis elegans</em>, <em>Chlamydomonas reinhardtii</em>, <em>Dictyostelium discoideum</em>, <em>Drosophila melanogaster</em>, <em>Paramecium tetraurelia</em>, <em>Saccharomyces cerevisiae</em> and <em>Schizosaccharomyces pombe</em> (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Keightley <em>et al.</em> 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Denver <em>et al.</em> 2012</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ness <em>et al.</em> 2012</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Rutter <em>et al.</em> 2012</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Saxer <em>et al.</em> 2012</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Sung <em>et al.</em> 2012b</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> 2015</a>).</p><p id="p-5">The haploid, fission yeast <em>S. pombe</em> was originally isolated from millet beer, and is an important model organism in molecular biology (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Leupold 1950</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Humphrey 2000</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wood <em>et al.</em> 2002</a>). While both are ascomycete yeasts, <em>Sc. pombe</em> is a distant relative of <em>S. cerevisiae</em>, with a divergence time of 600–1200 million years (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Heckman <em>et al.</em> 2001</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Douzery <em>et al.</em> 2004</a>). Even so, they exhibit similar life histories as single-celled, sexual yeasts, and are cultured in the lab using essentially identical methods. In addition, they have similar genomic G/C content (36.06% in <em>Sc. pombe</em> <em>vs.</em> 38.29% in <em>S. cerevisiae</em>) and genome size [13.8 Mb in <em>Sc. pombe</em> (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Wood <em>et al.</em> 2002</a>) <em>vs.</em> 12.1 Mb in <em>S. cerevisiae</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Goffeau <em>et al.</em> 1996</a>)]. While similar, they differ in one feature that is expected to have implications for their mutation rate and spectrum: <em>Sc. pombe</em> is a haploid as opposed to a diploid species in nature. The similarities and difference between these two yeast species allowed us to make several <em>a priori</em> predictions concerning mutation rate and spectrum in <em>Sc. pombe</em>. Before listing these predictions, a brief note on our terminology is useful: G/C and A/T indicate G or C and A or T, and are used when discussing nucleotide composition of regions of the genome; adjacent nucleotides in a single strand of DNA are indicated by the juxtaposition of the base designations with the 5′ nucleotide on the left, <em>e.g.</em>, CGA indicates a C nucleotide adjacent to a G nucleotide adjacent to an A nucleotide, with the A in the 3′ position; dinucleotides in the same strand that are Watson-Crick partners when in different strands, are indicated as CpG, GpC, ApT, and TpA to clarify that the nucleotides are in same strand; and base pairs in a DNA double helix are indicated with a colon, <em>e.g.</em>, C:G.</p><p id="p-6">Our first prediction was that the genome-wide single-nucleotide mutation (SNM) rate would be higher in <em>Sc. pombe</em> than in <em>S. cerevisiae</em> based on estimates from reporter genes that average 8.2 × 10<sup>−10</sup> single nucleotide substitutions per base per generation (summarized in <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lynch 2010</a>), which is 4.8-fold higher than the most recent estimate in <em>S. cerevisiae</em> (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>).</p><p id="p-7">Our second prediction was that most of the SNM biases, which are the relative rates of mutations among the different base pairs, in <em>Sc. pombe</em> would generally be the same as in <em>S. cerevisiae</em>. This prediction comes from the fact that the genomic G/C content of <em>Sc. pombe</em> (36.06%) is similar to that of <em>S. cerevisiae</em> (38.29%). Interestingly, the SNM biases observed in <em>S. cerevisiae</em> predict a lower G/C content (32%) than observed (<a id="xref-ref-79-3" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>), suggesting another force acts on G/C content. One possibility is selection. If weak selection drives G/C content to a higher equilibrium than predicted based on SNM bias in both species, then we expect <em>Sc. pombe</em> to be closer to the mutation-bias equilibrium than <em>S. cerevisiae</em>, <em>i.e.</em>, lower, as observed, because of its smaller effective population size (calculated using previous estimates of diversity and mutation rate), which reduces the efficacy of selection. The SNM bias in <em>S. cerevisiae</em> shows an elevated mutation rate of C:G base pairs (<a id="xref-ref-79-4" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>). A major surprise concerning the elevated mutation rate at C:G base pairs was the finding that their mutagenicity is affected by trinucleotide context. Specifically, when C:G is the middle base pair in CCG (equivalent to CGG) and TCG (equivalent to CGA) trinucleotides, the mutation rate is elevated even more than for other C:G base pairs. Both of these trinucleotides include a CpG dinucleotide, which is a well-characterized target for methyl-transferases (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bestor and Verdine 1994</a>). The bias at CpG dinucleotides in <em>S. cerevisiae</em> was in the C → T direction. For this reason, the finding of elevated C:G mutation at CCG and TCG trinucleotides was interpreted as indicating a very low occurrence of DNA methylation in <em>S. cerevisiae</em> (<a id="xref-ref-79-5" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>), in agreement with a previous study (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Tang <em>et al.</em> 2012</a>). <em>Sc. pombe</em> is believed to lack DNA methylation so we did not expect to see an elevation of the G:C mutation rate at CpG sites (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Antequera <em>et al.</em> 1984</a>).</p><p id="p-8">Our third prediction was that among small insertions and deletions (indels) there would be an insertion bias. Small indels occur primarily at microsatellites, presumably primarily due to slippage of the DNA polymerase during replication (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Levinson and Gutman 1987</a>). In <em>S. cerevisiae</em>, indels of less than 50 bp were moderately biased toward deletions (18 deletions <em>vs.</em> eight insertions) across the diploid MA lines (<a id="xref-ref-79-6" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>). However, analysis of indels in haploid MA lines of <em>S. cerevisiae</em> showed a bias toward insertions (34 insertions and eight deletions at microsatellite loci) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lynch <em>et al.</em> 2008</a>). Attributing this insertion bias to haploidy is complicated by the fact that many of the haploid MA lines reverted to diploidy over the course of the MA experiment (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Lynch <em>et al.</em> 2008</a>). Nevertheless, <em>Sc. pombe</em> is a natural haploid and was passaged as such in our MA experiment, and so we hypothesized that we would see a bias toward insertions for small indels in this species.</p><p id="p-9">Our fourth prediction was that mutations resulting from nonrecombinational repair of double-strand breaks would be more common in haploid <em>Sc. pombe</em> than in diploid <em>S. cerevisiae</em>. Nonrecombinational repair is substantially more mutagenic than recombinational repair (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Daley <em>et al.</em> 2005</a>). In a diploid cell, there is always a nonsister homolog present that can be used for recombinational repair. However, in a haploid cell, recombinational repair is only possible during the S or G2 phase of the cell cycle, when a sister chromatid is present. Repair of double-strand breaks is thus expected to be more error-prone in a haploid because of the higher likelihood of using nonrecombinational repair. While double-strand breaks cannot be directly observed or always inferred with certainty in the MA framework, their occurrence can be indicated, if they are inaccurately repaired, by the presence of multiple mutations in close proximity (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Strathern <em>et al.</em> 1995</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Holbeck and Strathern 1997</a>). In diploid <em>S. cerevisiae</em>, three double mutations, which consist of two SNMs adjacent to one another, were observed across the MA lines (<a id="xref-ref-79-7" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>). The relative occurrence of double SNMs was thus ∼0.35% compared to SNMs. In addition, there were five “complex mutations”, in which multiple SNMs, and often small indels, were in close proximity, giving a relative occurrence of ∼0.58% compared to SNMs. If both double and complex mutations were due to error-prone double-strand break repair, then the observed number of these mutations accounted for ∼1% of SNMs. The haploid MA experiment in <em>S. cerevisiae</em> was too small to detect such rare events (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Lynch <em>et al.</em> 2008</a>). However, our MA experiment is large enough that we predicted we would see an increase in the rate of double mutations and complex mutations in <em>Sc. pombe</em> compared to <em>S. cerevisiae</em>, assuming these events are indeed caused by repair of double-stranded breaks. Our prediction was somewhat tempered by the fact that <em>Sc. pombe</em> spends only ∼10% of its cell cycle in the G1 phase when growing exponentially (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Knutsen <em>et al.</em> 2011</a>), and is thus haploid for a minority of the time, though this proportion can increase substantially when growth slows (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nasmyth 1979</a>). An average, exponentially growing, <em>Sc. pombe</em> strain, is thus expected to have two copies of its genome in each cell 90% of the time, and a single copy 10% of the time, implying an average ploidy of 1.9 genomes. <em>S. cerevisiae</em>, in contrast, is in G1 (with two copies of each DNA region) about 25–35% of the time when growing exponentially (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Slater <em>et al.</em> 1977</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brewer <em>et al.</em> 1984</a>), implying an average ploidy of 3.4 genomes. These data imply that a double-strand break in <em>Sc. pombe</em> will be in a cell containing one homologous copy for possible recombinational repair about 90% of the time, and will have no template for repair about 10% of the time. In contrast, a double-strand break in <em>S. cerevisiae</em> will always have at least one copy, and will have three copies ∼70% of the time. So, while <em>Sc. pombe</em> may have a copy available for repair up to 90% of the time, the average number of copies available for repair is about 2.5 times less (0.9 <em>vs.</em> 2.4 extra copies) than in <em>S. cerevisiae</em>.</p><p id="p-10">Our final <em>a priori</em> prediction was that we would not observe aneuploidy in <em>Sc. pombe</em>. Since <em>Sc. pombe</em> is haploid, loss of a single chromosome (nullisomy) results in loss of all copies of the genes on that chromosome, which would likely be lethal and thus unobservable. In addition, gain of a single chromosome (disomy) would result in a doubling of gene dose for all genes on the chromosome, which is a larger increase than the 1.5-fold increase that occurs with trisomy in diploid <em>S. cerevisiae</em>. If deleterious effects due to gene dosage were increased with larger differences in dosage across genes, this would further reduce the likelihood of observing aneuploidy. In addition, <em>Sc. pombe</em> has only three chromosomes, implying that many more genes would be affected by an aneuploidy event than in <em>S. cerevisiae</em>, which has a similar genome size but 16 chromosomes. The only instance of aneuploidy reported in previous work in <em>Sc. pombe</em> is disomy of chromosome III, which was highly deleterious (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Niwa <em>et al.</em> 2006</a>). For these reasons, we predicted we would see no nullisomy, and likely no disomy in any of our MA lines.</p><p id="p-11">One month prior to submission of our results, Farlow and colleagues (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> 2015</a>) published the results of an MA experiment on <em>Sc. pombe</em> of essentially identical scope. While the inadvertent duplication of effort is unfortunate, their results allow us to investigate the repeatability of the MA framework for determining estimates of parameters of mutation. Their study utilized a different starting strain, different media, different growth temperatures, and longer time between transfers, allowing us to determine whether these factors together alter parameter estimates.</p><p id="p-12">This study presents genome-wide estimates of mutational parameters for <em>Sc. pombe</em>. We examined 79 MA lines, cultivated as haploids for an average of 1952 generations. We were able to identify a total of 696 mutations. These mutations allowed us to calculate precise estimates of the mutation rate and spectrum for <em>Sc. pombe</em>, test our <em>a priori</em> predictions and determine the repeatability of MA for parameter estimation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Mutation accumulation lines</h3><p id="p-13"><em>Sc. pombe</em> MA lines were passaged in the same manner as described for <em>S. cerevisiae</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Joseph and Hall 2004</a>). Briefly, the haploid ancestral line, 972 h– (ATCC 26189), was streaked onto rich, solid YPD medium (1% yeast extract, 2% peptone, 2% dextrose, 2% agar) and incubated at 30°. From the streaked ancestor, 96 random isolated colonies were selected after 48 hr and used to found 96 MA lines. Lines were cultured six to a YPD plate, and bottlenecked by randomly selecting one isolated colony per line every 48 hr and transferring to a new plate. Lines were passaged for a total of 100 transfers (200 days). Every 10 transfers, a random colony from each line was frozen and stored in 15% glycerol at –80°.</p><p id="p-14">Every 10 transfers, photographs were taken of all 96 MA lines. From these photographs, colony size was measured for five random colonies per line using ImageJ (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Schneider <em>et al.</em> 2012</a>). In addition, the numbers of cells per colony for colonies of various sizes were recorded at transfer 10 (T10) and 100 (T100) by suspending a single colony in 1 ml of water, and counting individual cells using a hemocytometer. These measurements were used to determine standard curves of colony size <em>vs.</em> cell number at these two transfers. Measurements of average colony sizes at every 10th transfer were then used to calculate the average number of cells at those transfers. From the average number of cells at every 10th transfer, the number of cell generations per transfer was estimated, using the fact that, with no mortality, the number of cells in a colony is equal to 2<em><sup>g</sup></em>, where <em>g</em> is the number of generations. The number of generations per transfer was then multiplied by the number of transfers to estimate the number of generations across the entire experiment. The effective population size across the experiment was determined by calculating the harmonic mean of the number of cells present each generation, assuming cell number doubles each generation between transfers, at which point it bottlenecks to one cell.</p></div><div id="sec-3" class="subsection"><h3>Sequencing</h3><p id="p-15">MA lines were cultured from frozen T100 stock on solid YPD medium at 30° for 48 hr. A single colony from each line was selected, inoculated into 3 ml liquid YPD, and incubated on a rotator at 30° for 48 hr. Cells were then pelleted and DNA was extracted using the YeaSTAR kit (Zymo Research) protocol with chloroform, and an extended digestion time with zymolase of 2.5 hr at 37°. Whole genome shotgun libraries were prepared by the Georgia Genomics Facility using the Kapa Library Low-Throughput Library Preparation Kit with Standard PCR Amp Module KK8232 with dual SPRI size selection cleanup to generate 100 bp paired-end fragments with ∼300 bp inserts. After seven cycles of PCR, the libraries (96 haploid MA lines and two haploid ancestor samples) were pooled across two lanes of Illumina HiSequation 2000 machines (raw sequences are deposited at NCBI, BioProject SRP065886).</p></div><div id="sec-4" class="subsection"><h3>Quality control (QC), mapping, and identification of mutations</h3><p id="p-16">Sequence reads from each library were quality controlled with the ea-utils and fastx toolkit in order to remove low quality reads and residual adaptor sequence (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gordon and Hannon 2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aronesty 2011</a>)(Workflow deposited at <a href="https://github.com/behrimg/Scripts/blob/master/Hall_Projects/Pombe/MA_Pipeline.txt">https://github.com/behrimg/Scripts/blob/master/Hall_Projects/Pombe/MA_Pipeline.txt</a>). Based on the workflow outlined in <a id="xref-ref-79-8" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> (2014)</a> and adjusted for a haploid dataset, quality controlled (QCed) reads were then mapped to the <em>Sc</em><em>. pombe</em> reference genome ASM294v2.24 with BWA v1.1.2, sorted and indexed with SAMtools v1.0, and assigned line identification numbers with Picard Tools v1.87 (<a id="xref-ref-77-3" class="xref-bibr" href="#ref-77">Wood <em>et al.</em> 2002</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Li and Durbin 2009</a>). Duplicated reads were marked with Picard Tools and removed, and then the remaining sequence reads were locally realigned with GATK v3.2.2 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">McKenna <em>et al.</em> 2010</a>). SNM and indel variants for each line and the ancestor were identified simultaneously using GATK’s Unified Genotyper tool with parameter settings for haploid organisms. The resulting VCF files were converted to tab delimited text using VCFtools v0.1.12a vcf-to-tab function (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Danecek <em>et al.</em> 2011</a>). All sequence differences between the MA ancestor, which was sequenced twice, and the reference were identified to determine the sequence of the ancestor. The differences between each MA line and the reference were determined, and those that were present in the ancestor were ignored. In order to call a variant, a minimum of four reads with ≥75% of the reads favoring the variant allele was needed. Regions of the genome that corresponded to centromeres, telomeres, and mating type loci (approximately 472 kbp) were excluded from the analysis to avoid inaccurate mapping. This was in addition to the two tandem rDNA repeat arrays on chromosome III accounting for 1,465 kbp, which are excluded from the reference genome. Identified SNMs and small indels were annotated using Ensembl’s variant effect predictor (VEP) while flanking regions were determined using the fill-fs program from the VCFtools package (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Li and Durbin 2009</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">McLaren <em>et al.</em> 2010</a>).</p><p id="p-17">Presence of medium and large structural variants were investigated using the Delly software package (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rausch <em>et al.</em> 2012</a>), and variants that passed Delly’s QC were investigated further using the integrated genome viewer (IGV) v2.1.23 (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Robinson <em>et al.</em> 2011</a>). When IGV supported a structural variant call, the variant was tested with PCR.</p><p id="p-18">Sequencing also allowed the detection of across-line and other microbial contamination. Across-line contamination was deemed to have occurred if any two lines shared an identical new mutation. When this happened, one of the lines (chosen by coin flip) was discarded from the remainder of the analysis.</p></div><div id="sec-5" class="subsection"><h3>Gene expression</h3><p id="p-19">To estimate mRNA concentrations, as a surrogate for gene expression levels for our ancestor strain, we sequenced mRNA from 10 biological replicates. We selected 10 colonies, inoculated each into 3 ml liquid YPD medium, and incubated on a rotator at 30° for 48 hr. After 48 hr, mRNA was extracted using the MasterPure Yeast RNA Purification kit (Epicentre). mRNA libraries were constructed using the Illumina Truseq mRNA Stranded Kit, amplified using 13 cycles of PCR and sequenced on an Illumina HiSequation 2500. Libraries were sequenced as 100 bp single-end reads (NCBI SRA BioProject: SRP065886). Sequenced reads were QCed in the same manner as genomic sequencing reads, reference-mapped with TopHat v.2.0.13 (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Trapnell <em>et al.</em> 2009</a>), and assembled with Cufflinks (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Trapnell <em>et al.</em> 2010</a>). The log-median fragments per kilobase of exon per million fragments mapped (FPKM) for each site across ancestor replicates was chosen to represent the level of expression at that site.</p></div><div id="sec-6" class="subsection"><h3>Verification of identified mutations to determine false positive rate</h3><p id="p-20">Five lines were randomly selected to verify the mutations that were identified bioinformatically with Sanger sequencing (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/TableS1.pdf">Table S1</a>). Primers were designed using Primer3 (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rozen and Skaletsky 1999</a>) and PCR products destined for sequencing were cleaned using a standard Exo-SAP protocol (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bell 2008</a>), and sequenced with an ABI BigDye Terminator Cycle Sequencing Kit (Applied Biosystems, Foster City, CA). Completed sequencing reactions were submitted to the Georgia Genomics Facility, and analyzed using an Applied Biosystems 3730xl 96-capillary DNA Analyzer.</p></div><div id="sec-7" class="subsection"><h3>Identification of ancestor mutations across lines to determine false negative rate</h3><p id="p-21">Mutations present in the ancestor relative to the reference should be present in every MA line. This implies that an ancestral mutation that is not found bioinformatically in an MA line is a false negative. To estimate the false negative error rate, the total number of ancestral mutations that should have been scored (equal to the product of the number of mutations in the ancestor and the number of MA lines) was compared to the number actually scored, and the probability of missing a mutation was calculated.</p></div><div id="sec-8" class="subsection"><h3>Relative mutation rate analysis</h3><p id="p-22">To investigate the effects of G/C content, replication time, and transcription rate, relative mutation rates were calculated across groups differing in these variables. G/C content was determined in nonoverlapping 10 kb windows across the entire genome. The number of SNMs was determined for each window and separated into 14 bins representing G/C content rounded down to the nearest 1%. Relative mutation rates were determined for each bin by determining the per base pair mutation rate for each bin and dividing by the observed genome-wide per base pair mutation rate. Bins were then separated into groups so that each group contained roughly equal numbers of SNMs, but not necessarily equal numbers of bins. Weighted averages and standard errors for each group were determined by weighting each bin by the amount of the genome covered by the bin. The same bins were used to determine relative mutation rates at A/T and G/C bases by calculating the per base pair mutation rate for just A/T and G/C bases.</p><p id="p-23">Replication time was determined genome wide by assigning each SNM a replication time based on the closest origin of replication (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Heichinger <em>et al.</em> 2006</a>). SNMs were separated into 17 bins, each representing 1 min of S phase of the cell cycle. Relative mutation rates were then determined in an analogous manner as for G/C content.</p><p id="p-24">Transcript abundance was determined genome wide and transcripts were separated into 15 bins based on abundance increasing from 0 to 3.5 in intervals of 0.25 log<sub>10</sub>FPKM. SNMs within genes were assigned a bin based on the transcript in which they reside; SNMs not mapping to a transcript were assigned a gene expression of 0. Relative mutation rate was then determined in an analogous manner as G/C content and replication time.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-25">Scripts written to process and analyze the data are available on GitHub (<a href="https://github.com/behrimg/Scripts/blob/master/Hall_Projects/Pombe/MA_Pipeline.txt">https://github.com/behrimg/Scripts/blob/master/Hall_Projects/Pombe/MA_Pipeline.txt</a>) and raw sequences are available on the Sequence Read Archive at NCBI (BioProject SRP065886). All other data is available upon request.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>MA lines</h3><p id="p-26">The relationship between colony area and the number of cells in the colony did not differ between transfers 10 and 100 (data not shown). The regression between colony size and transfer number indicated that average colony size declined over the course of the experiment such that the number of cells at T100 was approximately half the number at T10, indicating an average reduction in per-generation growth rate of approximately 5%. A reduction in growth rate is expected as haploid MA lines accumulate deleterious mutations. In diploid <em>S. cerevisiae</em>, a trend toward reduced colony size was not observed (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Joseph and Hall 2004</a>), presumably because new mutations accumulated in the heterozygous state and thus had a smaller effect on fitness due to masking. The average number of cells in a colony was used to estimate the average effective population size in a MA line as 10.26 cells. This small effective population size implies that deleterious (beneficial) mutations with a fitness effect greater than ∼0.097 would be underrepresented (overrepresented) due to the action of selection (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hall <em>et al.</em> 2008</a>; <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Lynch <em>et al.</em> 2008</a>). Mutations with smaller fitness effects are expected to accumulate approximately at random in the MA lines. The average number of cells in a colony was also used to estimate that the MA experiment lasted 1952 cell generations.</p><p id="p-27">Mapping of sequencing reads to the <em>Sc. pombe</em> reference genome revealed that coverage was uniform across all three chromosomes except for telomeric regions in those lines with average sequencing depth below 43x, where telomeric sequences appeared to be affected by amplification bias (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/FigureS1.pdf">Figure S1</a>). After sequencing and identification of nucleotide variants, we discarded 17 of the 96 MA lines due either to contamination by a different microbial species (six lines) or to across MA line contamination (11 lines), which left 79 lines for further analysis. We note that for most of the contaminated lines (10 of 11), all nucleotide variants were shared, indicating that contamination likely occurred either at one of the last transfers, or when the lines were removed from the freezer for DNA sequencing.</p><p id="p-28">Approximately 472 kbp, which is ∼3.8% of the sequenced genome, was excluded from the analysis to ensure precise identification of new mutations. This included the low-complexity sequence, comprising centromeres, telomeres, the mating type region, and the representative rDNA repeat belonging to the two tandem rDNA repeat arrays on chromosome III, which had been previously excluded from the 12.47 Mb reference genome (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wood <em>et al.</em> 2011</a>). The exact coordinates of the regions removed from the analysis are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/TableS2.pdf">Table S2</a>.</p></div><div id="sec-12" class="subsection"><h3>Differences between the MA ancestor and <em>Sc. pombe</em> reference genome</h3><p id="p-29">Sequencing of the ancestor revealed a total of 262 differences between it and the reference genome. These differences included 80 SNMs, 120 small insertions (&lt; 50 bp), 42 small deletions (&lt; 50 bp), six double SNMs, and 14 complex mutations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/FigureS2.pdf">Figure S2</a>). We define double SNMs as two SNMs that occur within 50 nucleotides of one another. Complex mutations are three or more SNMs and/or indels occurring within 50 nucleotides of one another. Both complex mutations and double SNMs were deemed to be nonindependent events and were thus analyzed separately (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Schrider <em>et al.</em> 2011</a>). Of the 80 SNMs, 39 were transitions and 41 were transversions, giving a transition to transversion ratio of 0.95. Within transitions, after correcting for genomic G/C content, and assuming that all changes were mutations from the reference strain sequence to the ancestral strain sequence, G:C → A:T or T:A mutations were 1.38 times more frequent than A:T → G:C or C:G mutations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/149/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of mutations for each of six possible nucleotide changes for the ancestor (ATCC 26189) vs. the reference, the 79 mutation&#xA0;accumulation (MA) lines in this study, and the 96 Farlow et al. (2015) MA lines. Also shown are the mutation frequencies observed in the 145 diploid Saccharomyces cerevisiae MA lines (Zhu et al. 2014)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-470807299" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of mutations for each of six possible nucleotide changes for the ancestor (ATCC 26189) &lt;em&gt;vs.&lt;/em&gt; the reference, the 79 mutation&#xA0;accumulation (MA) lines in this study, and the 96 Farlow &lt;em&gt;et al.&lt;/em&gt; (2015) MA lines. Also shown are the mutation frequencies observed in the 145 diploid &lt;em&gt;Saccharomyces cerevisiae&lt;/em&gt; MA lines (Zhu &lt;em&gt;et al.&lt;/em&gt; 2014).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/149/F1.medium.gif" width="440" height="325"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/1/149/F1.medium.gif" width="440" height="325"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/149/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/149/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467384" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">Summary of mutations for each of six possible nucleotide changes for the ancestor (ATCC 26189) <em>vs.</em> the reference, the 79 mutation accumulation (MA) lines in this study, and the 96 <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a> MA lines. Also shown are the mutation frequencies observed in the 145 diploid <em>Saccharomyces cerevisiae</em> MA lines (<a id="xref-ref-79-9" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The distribution of SNMs across chromosomes was not significantly different from the expectation based on chromosome length (X<sup>2</sup> = 5.45, <em>P</em> = 0.06). This result held whether we tested all three chromosomes, or just chromosomes I and II (X<sup>2</sup> = 3.05, <em>P</em> = 0.06), which together represent 81% of the genome. The distribution of small insertions was nonrandom, with chromosome I (5.58 Mb) containing 68, which is greater than the 53 expected based on chromosome length (X<sup>2</sup> = 7.603, <em>P</em> = 0.006). Other mutations, including small deletions, complex mutations, and double SNMs, did not show bias across chromosomes, after accounting for length (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/FigureS3.pdf">Figure S3</a>).</p></div><div id="sec-13" class="subsection"><h3>Differences between the MA lines and the MA ancestor</h3><div id="sec-14" class="subsection"><h4>Single nucleotide mutations:</h4><p id="p-32">Across the 79 MA lines, 326 SNMs arose during MA, which gives the SNM rate (± 1 SE) for <em>Sc. pombe</em> as 1.70 ± 0.13 × 10<sup>−10</sup> per base per generation (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/TableS3.pdf">Table S3</a>). The SE of the estimate is based on the variance across MA lines. This mutation rate estimate does not include the 123 SNMs that were in double or complex mutations. Including SNMs in double and complex mutations increases the genome-wide mutation rate estimate to 2.34 ± 0.23 × 10<sup>−10</sup> per base per generation.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/149/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of types of mutations identified across 79 MA lines. SNM, Single nucleotide mutation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-470807299" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary of types of mutations identified across 79 MA lines. SNM, Single nucleotide mutation.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/149/F2.medium.gif" width="440" height="254"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/1/149/F2.medium.gif" width="440" height="254"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/149/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/149/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467405" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">Summary of types of mutations identified across 79 MA lines. SNM, Single nucleotide mutation.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The number of SNMs varied across the 79 MA lines from zero (eight lines) to 13 (one line), with an average of 4.13 SNMs per line (ignoring doubles and complex mutations). Surprisingly, the distribution of mutations across the MA lines was not consistent with a Poisson distribution (χ<sup>2</sup>; <em>P</em> &lt; 0.001). A negative binomial distribution (Gamma-Poisson), with mean = 4.13 and overdispersion parameter = 2.06, could not be rejected (χ<sup>2</sup>; <em>P</em> = 0.954, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Using Akaike’s information criterion (AIC), the negative binomial was a substantially better fit to our data than the Poisson (Poisson AIC = 15.58, negative binomial AIC = 4.45; Poisson is 0.0034 as likely to explain the data). SNMs occurred at random with respect to protein-encoding genes: 52.4% of SNMs were in the 57% of the genome that is protein coding (Fisher’s exact, <em>P</em> = 0.27), and 3.6% of SNMs were in the 3% of the genome that is intronic sequence (Fisher’s exact, <em>P</em> = 0.83), suggesting that selection was inefficient during MA.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/149/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histogram of SNM numbers per line across the 79 MA lines. The distribution is consistent with a negative binomial (P&#xA0;=&#xA0;0.95, &#x3C7;2 test). Dashes represent the expected numbers for the best-fit negative binomial distribution (mean&#xA0;=&#xA0;4.13, overdispersion&#xA0;=&#xA0;2.06). The total number of SNMs across all 79 MA lines is 326." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-470807299" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Histogram of SNM numbers per line across the 79 MA lines. The distribution is consistent with a negative binomial (&lt;em&gt;P&lt;/em&gt;&#xA0;=&#xA0;0.95, &#x3C7;&lt;sup&gt;2&lt;/sup&gt; test). Dashes represent the expected numbers for the best-fit negative binomial distribution (mean&#xA0;=&#xA0;4.13, overdispersion&#xA0;=&#xA0;2.06). The total number of SNMs across all 79 MA lines is 326.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/149/F3.medium.gif" width="423" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/1/149/F3.medium.gif" width="423" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/149/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/149/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467499" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Histogram of SNM numbers per line across the 79 MA lines. The distribution is consistent with a negative binomial (<em>P</em> = 0.95, χ<sup>2</sup> test). Dashes represent the expected numbers for the best-fit negative binomial distribution (mean = 4.13, overdispersion = 2.06). The total number of SNMs across all 79 MA lines is 326.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The distribution of SNMs and all other mutation classes (small insertions, deletions, complex, double SNMs) across the three chromosomes were not significantly different from the expectation based on chromosome length.</p></div><div id="sec-15" class="subsection"><h4>Single nucleotide mutation biases:</h4><p id="p-37">Ignoring SNMs in complex and double mutations, we observed a transition to transversion (Ts/Tv) bias of 0.72 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Within transitions, G:C → A:T mutations were 2.02 times more frequent than A:T → G:C. Within transversions, G:C → T:A mutations were 4.55 times more frequent than A:T → C:G. Across transitions and transversions, G/C → A/T mutations were 2.97 times more frequent than A/T → G/C mutations. If the mutational process were the sole determinant of the G/C content in <em>Sc. pombe</em>, the equilibrium genome G/C content would be 25.14%, far less than the 36.06% observed in the reference genome.</p><p id="p-38">G/C content has been shown to affect local mutation rate in rodents (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hurst and Williams 2000</a>). However, in this study, we found no effect of G/C content on the local mutation rate. We determined the G/C content in 10-kb windows across the genome, and then separated the data into groups such that each contained a similar number of SNMs. Regardless of whether we split the data into two or three groups, we found no significant difference in mutation rate across them, suggesting that local G/C content has no effect on local mutation rate. Data for the two-group case where the division is for a GC content of 36% is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/FigureS4.pdf">Figure S4</a>A.</p><p id="p-39">In humans, late replicating regions have higher mutation rates (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Stamatoyannopoulos <em>et al.</em> 2009</a>). We tested for a similar relationship and found that replication time did not affect mutation rate. We assigned each nucleotide in the genome a replication time based on the time at which the closest origin of replication (ori) initiates replication during mitosis (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Heichinger <em>et al.</em> 2006</a>). There is little variation in the replication initiation times of different oris in <em>Sc. pombe</em>, with all oris firing between 68 and 85 min after release from G2 arrest (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Heichinger <em>et al.</em> 2006</a>). Further, published initiation times are reported in 1-min intervals, and so we could not split times into smaller intervals. We separated replication times into three groups and found no significant differences in mutation rate for different replication times (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/FigureS4.pdf">Figure S4</a>B).</p><p id="p-40">Mutation rate has been shown to correlate with gene expression in <em>S. cerevisiae</em> and humans (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Park <em>et al.</em> 2012</a>), but see <a id="xref-ref-79-10" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> (2014)</a>. We found that mRNA levels were not associated with mutation rate. Using RNA-seq data collected from our ancestor line, we examined whether gene expression, measured as transcript abundance, influenced local mutation rate. Transcription rates were split into 15 bins that were then separated into two groups. These two groups did not differ in mutation rate. Adding rRNA and tRNA genes, and assuming they are expressed in the highest bin, did not alter this conclusion (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/FigureS5.pdf">Figure S5</a>).</p><p id="p-41">We did find an effect of trinucleotide context, which is the identity of the nucleotides immediately before and after the mutated nucleotide, on mutation rate (<a id="xref-ref-79-11" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>). Each nucleotide position within the genome, along with its neighboring bases, was assigned to one of the 64 trinucleotide possibilities. Ignoring strand orientation allowed us to reduce the 64 possibilities to 32 groups, defined such that complementary trinucleotides belong to the same group (<em>e.g.</em>, GCA and TGC are in the same group). Mutation rates at the center position across the 32 groups were quite variable (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We analyzed the 16 groups with A (or T) at the center position separately from those with G (or C) because C:G base pairs have a higher mutation rate than A:T base pairs, as is clearly apparent in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>. After correcting for multiple comparisons, we found context did not affect mutation rate at A:T base pairs. In contrast, at C:G base pairs, mutation rates were increased in the CCG (equivalent to CGG), GCG (equivalent to CGC), and GCA (equivalent to TGC) trinucleotide groups (<em>t</em>-test; <em>P</em> = 0.0003, <em>P</em> = 0.0001, and <em>P</em> = 0.0009 respectively; Bonferroni corrected critical <em>P</em>-value = 0.0031). Interestingly, two of these trinucleotides are two of the four trinucleotide groups that contain a CpG dinucleotide, where the C is the at the center position. If increased CpG mutation rate was due to deamination of methylated cytosine (giving thymine), then we would expect CpGs to mutate to TpGs. However, of the 44 mutations occurring at CpG sites, 23 were C → A mutations (52.3%), 18 were C → T (40.1%) and three were C → G (6.6%); which is no different than expected based on the mutational spectrum of <em>Sc. pombe</em> (X<sup>2</sup> = 1.76, <em>P</em> = 0.41). For the GCA trinucleotide group, C → A mutations were overrepresented relative to the mutational spectrum (18 of 22 mutations were C → A, 11 were expected; X<sup>2</sup> = 8.91, <em>P</em> = 0.011).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/149/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mutation rate is affected by context. Relative SNM rate adjusted by its trinucleotide context. Trinucleotide classes represent mutation rates of both strand orientations. For example, aCa trinucleotide class includes overall mutation rate at aCa and tGt sites. Mutation rate is shown relative to the average SNM rate across all sites (=&#xA0;1.7&#xA0;&#xD7;&#xA0;10&#x2212;10 per base, per generation). The average, relative mutation rate of 1.8 at G:C bases shows clear overall elevation over the corresponding rate of 0.66 at A:T bases. The four black-filled points are those trinucleotides with a C in the central position with a G as the 3&#x2032; neighbor. * P&#xA0;&#x2264;&#xA0;0.003 Bonferroni corrected t-test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-470807299" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mutation rate is affected by context. Relative SNM rate adjusted by its trinucleotide context. Trinucleotide classes represent mutation rates of both strand orientations. For example, aCa trinucleotide class includes overall mutation rate at aCa and tGt sites. Mutation rate is shown relative to the average SNM rate across all sites (=&#xA0;1.7&#xA0;&#xD7;&#xA0;10&lt;sup&gt;&#x2212;10&lt;/sup&gt; per base, per generation). The average, relative mutation rate of 1.8 at G:C bases shows clear overall elevation over the corresponding rate of 0.66 at A:T bases. The four black-filled points are those trinucleotides with a C in the central position with a G as the 3&#x2032; neighbor. * &lt;em&gt;P&lt;/em&gt;&#xA0;&#x2264;&#xA0;0.003 Bonferroni corrected &lt;em&gt;t&lt;/em&gt;-test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/149/F4.medium.gif" width="440" height="216"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/1/149/F4.medium.gif" width="440" height="216"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/149/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/149/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467363" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-42" class="first-child">Mutation rate is affected by context. Relative SNM rate adjusted by its trinucleotide context. Trinucleotide classes represent mutation rates of both strand orientations. For example, aCa trinucleotide class includes overall mutation rate at aCa and tGt sites. Mutation rate is shown relative to the average SNM rate across all sites (= 1.7 × 10<sup>−10</sup> per base, per generation). The average, relative mutation rate of 1.8 at G:C bases shows clear overall elevation over the corresponding rate of 0.66 at A:T bases. The four black-filled points are those trinucleotides with a C in the central position with a G as the 3′ neighbor. * <em>P</em> ≤ 0.003 Bonferroni corrected <em>t</em>-test.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Small insertion and deletion mutations:</h4><p id="p-43">Across the MA lines, we identified 335 small indels of less than 50 bp, including 288 insertions and 47 deletions. A complete list of indels is in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/TableS4.pdf">Table S4</a>. Average insertion size was 1.6 bp, while average deletion size was 3.1 bp, with insertions occurring six times more frequently than deletions, resulting in a net gain in DNA sequence across all lines of 340 bp. Small indels occurred as frequently as SNMs across the MA lines, and the resulting spontaneous indel rate, 0.174 × 10<sup>−9</sup> indels/base/generation, is essentially identical to the mutation rate calculated for SNMs, ignoring double SNMs and complex mutations. Indels, however, were not randomly distributed with respect to genomic features. They were substantially underrepresented in protein coding sequence (observed: 33, expected: 191; Fisher’s exact test: <em>P</em> &lt; 0.001), occurred as frequently as expected in introns (observed 21, expected 10; <em>P</em> = 0.064), and were over represented in noncoding regions (observed 288, expected 134; <em>P</em> &lt; 0.001).</p></div><div id="sec-17" class="subsection"><h4>Effects of SNMs and indels:</h4><p id="p-44">We annotated the expected functional effects of our SNM and small indel mutations using Ensembl’s VEP software (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">McLaren <em>et al.</em> 2010</a>). For the 183 SNMs that occurred within coding regions, 53 were synonymous, 113 were missense, 12 were within introns, one was within a splice donor site, one was within a splice acceptor site, two were nonsense, and one changed a termination codon (UAA) into another termination codon (UAG) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). The ratio of synonymous to missense changes (0.398) was higher but not statistically different from the expected ratio (0.323), computed using randomly generated protein coding sequences (Z-test for proportions: 1.104, <em>P</em> = 0.271) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/FigureS6.pdf">Figure S6</a>) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Graur 2003</a>). This suggests selection did not substantially reduce the number of missense mutations captured during MA.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/149/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of the locations and predicted functional consequences of the 326 SNMs (excluding SNMs in double and complex mutations), 288 small insertions and 47 small deletions accumulated in the MA lines. To facilitate comparison across the three mutational classes, frequencies are shown. NC, noncoding; UTR transcribed, untranslated region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-470807299" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Summary of the locations and predicted functional consequences of the 326 SNMs (excluding SNMs in double and complex mutations), 288 small insertions and 47 small deletions accumulated in the MA lines. To facilitate comparison across the three mutational classes, frequencies are shown. NC, noncoding; UTR transcribed, untranslated region.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/149/F5.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/1/149/F5.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/149/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/149/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467497" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-45" class="first-child">Summary of the locations and predicted functional consequences of the 326 SNMs (excluding SNMs in double and complex mutations), 288 small insertions and 47 small deletions accumulated in the MA lines. To facilitate comparison across the three mutational classes, frequencies are shown. NC, noncoding; UTR transcribed, untranslated region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Among the 54 small indels that occurred within protein and intronic coding sequence, seven were deletions, of which five were frameshift deletions and two were within introns. For the remaining 47 insertions, 23 were frameshift insertions, five were in-frame insertions ranging from 6 to 12 bp in length, and 19 were within introns (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The proportion of indels in protein coding sequence that were in-frame was not significantly different from 1/3 (Fisher’s exact test <em>P</em> &gt; 0.2). Again, these results suggest that selection did not substantially reduce the number of frameshift mutations captured during MA.</p></div><div id="sec-18" class="subsection"><h4>Double mutations, complex mutations, and structural variants:</h4><p id="p-47">As expected, no aneuploids were detected among the haploid MA lines. We specifically searched for other structural variants including medium-sized deletions (50–1000 bp), inversions, translocations and duplications using the DELLY software that predicts structural variants from short insert paired-end sequences (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Rausch <em>et al.</em> 2012</a>). We detected three medium-sized deletions, two of which were flanked by a number of SNMs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/TableS5.pdf">Table S5</a>). In addition to multiple SNMs in the region of the three medium deletions, 16 other complex mutations and 17 separate double SNMs were also detected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/TableS6.pdf">Table S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/TableS7.pdf">Table S7</a>). Complex mutations thus occurred at a rate of 8.32 × 10<sup>−12</sup> per base per generation and double SNMs occurred at a rate of 8.84 × 10<sup>−12</sup> per base per generation.</p></div></div><div id="sec-19" class="subsection"><h3>Error in identification of mutations</h3><p id="p-48">Our <em>Sc. pombe</em> lines were haploids and our sequencing coverage exceeded 25x in every line (average = 44x). Given previous work in diploid <em>S. cerevisiae</em>, we predicted we would have a very low likelihood of incorrectly calling a SNM or indel. To confirm a low false positive error rate, we randomly chose five lines and checked all predicted mutations using Sanger sequencing. Together, these lines contained 15 SNMs, 20 small insertions, three small deletions, one double SNM and one complex mutation. Sanger sequencing confirmed all of the mutations identified from next-generation sequencing, suggesting that the false positive error rate is likely is no larger than 0.025, and could be close to zero. We did find that, in the region of the complex mutation (MA Line 58, Chromosome II, positions 4,273,519–4,274,319), additional mutations not identified in next-generation sequencing data were present (<em>i.e.</em>, false negatives). These mutations were probably not detected due to the difficulty in reference mapping sequencing reads that are too divergent from the reference genome. This conjecture is supported by the extreme loss of coverage that was observed in this 800-bp region. Just 80 bp on either side of the region coverage was 48x, while across the complex mutation coverage averaged 9x, with three nucleotides in the center only being represented by a single read. Our results for this complex mutation suggest that the numbers of detected changes present in the five complex mutations we identified between the ancestor and reference strain and the 18 in the MA lines, are underestimates of the number actually present.</p><p id="p-49">We also examined the probability of false negatives for SNMs. This was done by asking how many of the 272 mutations that differ between the reference and the ancestor were not found in each of the 79 MA lines. Any mutation present in the ancestor that is not identified in an MA line is a false negative, assuming the probability that a mutation converting the ancestor base back to the reference base at any of these sites is negligible. Given our estimate of the mutation rate, the probability of a mutation occurring at any of these sites in any line during the experiment is ∼0.007, indicating that failure to detect a site that differed in the ancestor is almost certainly a false negative. For SNMs, the false negative rate was 0.0001, while for small insertions and small deletions it was 0.011 and 0.017, respectively. Thus, we may have underestimated the base substitution, small insertion and small deletion mutation rates by 0.01, 1.1 and 1.7%, respectively. No mutations were found in three or more MA lines that were not also found in the ancestor, indicating that we likely detected all of the mutations in the ancestor.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><div id="sec-21" class="subsection"><h3>Rate of mutation at single nucleotides</h3><p id="p-50">We expected to observe a higher SNM rate in <em>Sc. pombe</em> than in <em>S. cerevisiae</em>, based on data using reporter genes (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Lynch 2010</a>). Our estimate of the base pair mutation rate in <em>Sc. pombe</em>, from the 326 accumulated SNMs, is 1.70 ± 0.13 × 10<sup>−10</sup> per base per generation, which is almost identical to the mutation rate of 1.67 × 10<sup>−10</sup> per base per generation, based on 867 accumulated SNM mutations, in <em>S. cerevisiae</em> (<a id="xref-ref-79-12" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>). Our estimate is slightly, but significantly, lower than the 2.00 ± 0.10 × 10<sup>−10</sup> mutation rate per base per generation estimate reported by <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a>, but that estimate did not remove SNMs in double and complex mutations, except for in flocculation-related genes. Including the 123 SNMS that we observed in double and complex mutations raises the genome-wide mutation rate estimate to 2.34 ± 0.23 × 10<sup>−10</sup> per base per generation, which is not significantly different from the <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a> estimate. In both studies, there was no effect of either local G/C content, transcript abundance, used as a proxy for gene expression, or timing of replication on the local mutation rate.</p><p id="p-51">With our estimates of the mutation rate and the genome-wide estimate of π from <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Jeffares <em>et al</em>. (2015)</a>, we can calculate the effective population size for <em>Sc. pombe</em> as either 6.52 × 10<sup>6</sup> (including SNMs in double and complex mutations) or 8.80 × 10<sup>6</sup> (ignoring SNMs in double and complex mutations), which span the estimate of effective population size for <em>S. cerevisiae</em>, 8.53 × 10<sup>6</sup> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>). Thus, <em>Sc. pombe</em> and <em>S. cerevisiae</em> do not seem to differ very much in their effective population size, suggesting that selection will encounter the drift barrier (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Sung <em>et al.</em> 2012a</a>), the point at which the fitness advantage of new mutation is the same magnitude as the strength of genetic drift, at similar selection coefficients in both species. We note that our population size estimates are smaller than the estimate of 12 million reported in <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a>, the reason being that they utilize π calculated from intergenic regions, which is larger (0.0048) than the genome-wide π (0.0030) that we use here.</p></div><div id="sec-22" class="subsection"><h3>Bias in single nucleotide mutations</h3><p id="p-52">There are both similarities and differences between the SNM mutation biases in <em>Sc. pombe</em> and <em>S. cerevisiae</em>. The most obvious difference is that the most common SNM in <em>S. cerevisiae</em> is a C:G → T:A transition, while in <em>Sc. pombe</em> it is a C:G → A:T transversion (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). In addition, C:G → G:C transversions are more common, and A:T → G:C transitions less common in <em>S. cerevisiae</em> than in <em>Sc. pombe</em>. In contrast, both A:T → C:G and A:T → T:A transversions are similarly frequent in <em>S. cerevisiae</em> and <em>Sc. pombe</em> (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Both species show an elevated rate of G/C → A/T, which points to oxidative damage as a major cause of mutagenesis (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cheng <em>et al.</em> 1992</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Shen <em>et al.</em> 1994</a>). Within our MA lines, G/C → A/T mutations occurred at a rate that was ∼three times greater than A/T → G/C mutations, which is the same as the bias observed by <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a>, and higher than the ∼two-fold bias seen in <em>S. cerevisiae</em> (<a id="xref-ref-79-13" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>). The equilibrium G/C content calculated from the mutation spectra is 25% and 32% in <em>Sc. pombe</em> and <em>S. cerevisiae</em>, respectively.</p><p id="p-53">In both species the observed G/C content (35% in <em>Sc. pombe</em> and 38% in <em>S. cerevisiae</em>), is higher than predicted from mutation biases, suggesting that there is either selection for lower G/C content or some other mechanism, perhaps biased gene conversion, or selective constraint on protein sequence, which is causing the increase in genomic G/C content. Biased gene conversion is unlikely given the rarity of heterozygote formation in <em>Sc. pombe</em> (<a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> 2015</a>). Suggestive evidence for an additional force is present in the ancestor, whose G/C → A/T bias relative to the reference strain is 24% less than observed in the MA lines, indicating that G/C → A/T mutations are fixing at a lower rate than expected.</p></div><div id="sec-23" class="subsection"><h3>Rate and spectrum of indels</h3><p id="p-54">In our MA lines, we observed a 6.13-fold insertion bias. Insertions accounted for 86% of small indel events, consistent with previous observations of the GT repeat region in the <em>ade6</em> gene of <em>Sc. pombe</em>, where 83% of indel events in wild-type were insertions (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mansour <em>et al.</em> 2001</a>). This bias is identical to that observed by <a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a>. A similar bias toward insertions has also been observed in <em>C. elegans</em> (3.75 insertions to deletions) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Denver <em>et al.</em> 2004</a>) and haploid, but not diploid, <em>S. cerevisiae</em> (3.55 insertions to deletions) (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Lynch <em>et al.</em> 2008</a>). We do not know of any mechanism that would predict more insertion mutations based on ploidy, but if small deletions are more often strongly deleterious than small insertions, they would be underrepresented in MA experiments in haploid species, and in diploid species passaged with strong inbreeding (<em>e.g.</em>, <em>C. elegans</em>). In haploid <em>S. cerevisiae</em>, despite their rarity, the size of small deletions was able to negate the effect of small insertions in terms of changes in genome size. This is not the case in <em>Sc. pombe</em>. Even though small deletions were twice as large as small insertions, small insertions occurred six times more frequently resulting in a net gain of 340 bp across the genome across all MA lines. This suggests that <em>Sc. pombe</em>’s current genome size is not at equilibrium with respect to an insertion/deletion mutational balance. Additionally, in the ancestor, small deletions may have been favored by selection: 17.3% of the differences between the ancestor and reference are small deletions compared to only 6.7% in the MA lines, which is a significant difference.</p><p id="p-55">Similar to observations in <em>C. elegans</em>, <em>Sc. pombe</em> experiences small indels as often as SNMs (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Denver <em>et al.</em> 2004</a>), resulting in an indel rate that is 10-fold higher than haploid <em>S. cerevisiae</em> and almost 35-fold higher than diploid <em>S. cerevisiae</em> (<em>Sc. pombe</em>: 1.74 × 10<sup>−10</sup> <em>vs.</em> haploid <em>S. cerevisiae</em>: 0.2 × 10<sup>−10</sup>, and diploid <em>S. cerevisiae</em>: 0.05 × 10<sup>−10</sup> per base pair per generation). One hypothesis for the differences in indel rate may be due to differences in genome complexity between the two species. <em>Sc. pombe</em>’s genome is 60.2% protein coding (57% excluding introns) while <em>S. cerevisiae</em>’s is 71% protein coding (70.5% excluding introns) indicating a higher prevalence of noncoding DNA in <em>Sc. pombe</em>. Most indels identified in <em>Sc. pombe</em> and <em>S. cerevisiae</em> are within low complexity, intergenic regions such as microsatellites and mononucleotide runs. However, in an analysis of short simple repetitive sequences, there is little observed difference in the amount of repetitive sequence in <em>S. cerevisiae</em> and <em>Sc. pombe</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Karaoglu <em>et al.</em> 2005</a>). An alternate hypothesis is that there are differences in mismatch repair pathways between these two species. However, both species contain <a href="http://www.yeastgenome.org/locus/S000005450/overview">MSH2</a> homologs, which are responsible for high fidelity repair of small indels (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Rudolph <em>et al.</em> 1999</a>), suggesting that this hypothesis would require differences in fidelity of the mismatch repair pathway, rather than presence <em>vs.</em> absence. A third hypothesis is that there may be differences in polymerase fidelity between <em>Sc. pombe</em> and <em>S. cerevisiae</em> causing differing rates of strand slippage resulting in <em>Sc. pombe</em>’s increased insertion rate. However, given similar effective population sizes, we do not expect fidelity to differ in these two species. Finally, the difference may be due to detection issues, especially the problem of false negatives. Our study yielded an indel rate that is three times as large as that reported by <a id="xref-ref-16-10" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a>, which might be explained by false negatives. The methods of sequence mapping and variant calling differ between the two studies. Assuming there is no substantial difference in indel mutation rate between the two strains, the <a id="xref-ref-16-11" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a> study should have accumulated approximately 354 indels. Instead, only 119 were discovered. This difference suggests that our methods reliably identify a greater percentage of the mutations (<em>i.e.</em>, we have fewer false negatives). We directly estimated our false negative error rate by determining whether an ancestral mutation, of which there were 272, was detected in all MA lines, which represents a total of 21,488 mutations tested across the 79 MA lines. For small insertions and deletions, the false negative error rates were 1.1% and 1.7%, respectively. A direct estimate of the false negative error rate was not performed in the study of <a id="xref-ref-16-12" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a>, though they did introduce simulated SNMs into their reference assemblies to indirectly estimate a false negative error rate for SNMs. If the difference in indel rate estimates between the two studies is due to false negatives, then the <a id="xref-ref-16-13" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a> study missed about two thirds of all indels.</p></div><div id="sec-24" class="subsection"><h3>Complex mutations, double mutations, and aneuploidy</h3><p id="p-56">We also predicted an increase of mutations associated with double-stranded breaks; particularly double SNMs and complex mutations. Double-stranded breaks are lethal to a cell unless repaired. Repair can involve homologous recombination, which tends to be the preferred mechanism (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Raji and Hartsuiker 2006</a>), but can also utilize nonhomologous end-joining (NHEJ). Recombinational repair requires homologous copies of DNA. Compared to <em>S. cerevisiae</em>, <em>Sc. pombe</em> has 2.5 times fewer homologous copies available on average for recombinational repair of a double-strand break, and sometimes has no copies present (see <em>Introduction</em>). In the absence of homologous DNA, double-stranded breaks are repaired through NHEJ. Homologous recombination is considered an error-free method of repair, while nonhomologous end joining is considered to be error-prone, introducing small insertions or deletions when partially degraded ends inhibit precise repair (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cavero <em>et al.</em> 2007</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Shrivastav <em>et al.</em> 2008</a>).</p><p id="p-57">The locations of complex mutations were not random. Six of the 18 complex mutations occurred within three of the nine flocculin genes. All of these six mutations occurred within the characteristic tandem repeats found in these genes (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Verstrepen <em>et al.</em> 2005</a>). The flocculin tandem repeats are known to cause replication errors and are highly prone to double-stranded breaks and, as a result, recombination (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Verstrepen <em>et al.</em> 2005</a>). Their propensity for double-stranded breaks could explain the repeated observation of complex mutations within them if complex mutations are, in fact, caused by mutagenic NHEJ repair of double-stranded breaks. To make sure that inferred mutations in the flocculin genes were not caused by gene conversion/recombination, we checked for and found no evidence of recombination between paralogs. An excess of mutations in flocculin genes was also observed by <a id="xref-ref-16-14" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a>. They interpreted this finding as indicating positive selection for some unknown flocculation phenotype on their plates. They noted that only one mutation mapped to flocculation-related genes in <em>S. cerevisiae</em> (<a id="xref-ref-79-14" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>), suggesting that an increased mutation rate was not the underlying cause of the observed overrepresentation of mutations in these genes. However, mapping to flocculation genes in <em>S. cerevisiae</em> is problematic (M. G. Behringer, unpublished data) due to their repetitive nature, so mutations in these genes may have been missed in that study. In addition, given our estimate of the small effective population size within MA lines (10.26 cells), and the location of mutations in the MA lines indicating ineffectual selection, we favor increased mutation rate, caused by a high propensity for double-stranded breaks, as the explanation for more mutations occurring in these genes.</p><p id="p-58">As we predicted given its haploid state, possession of only three chromosomes, and previous work (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Niwa <em>et al.</em> 2006</a>), there were no instances of aneuploidy. In <em>Sc. pombe</em>, aneuploidy has only been observed as a disomic haploid of chromosome III. Even if a disomic haploid had been fixed in a line at an intermediate transfer, it would likely have been unstable (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Niwa and Yanagida 1985</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Niwa <em>et al.</em> 2006</a>) and thus rapidly lost. Interestingly, when <em>S. cerevisiae</em> is passaged as a mitotic haploid it tends to be very unstable (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Lynch <em>et al.</em> 2008</a>), and at large population size, where selection is effective, it reverts to a diploid state (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Mable and Otto 2001</a>). This instability is in contrast to the relative stability of diploid strains (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nishant <em>et al.</em> 2010</a>; <a id="xref-ref-79-15" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>). This implies that aneuploidy is generally less likely to occur in a natural haploid than in a natural diploid. However, when a natural diploid is kept as a haploid strain artificially, disomies, or other steps toward diploidy, may be tolerated and perhaps even favored. It would be interesting to see if the instability of haploidy in <em>S. cerevisiae</em>, which is a natural diploid, would also be reflected in instability of a diploid <em>Sc. pombe</em>, which is a natural haploid.</p></div><div id="sec-25" class="subsection"><h3>Cytosine mutation in absence of methylation</h3><p id="p-59">One of the major surprises of the <em>S. cerevisiae</em> mutation spectrum is the high mutation rate observed at some C:G base pairs, particularly in CpG dinucleotides found in CCG and TCG trinucleotides. We observed a similar, unexpectedly high mutation rate at some C:G base pairs, especially CCG and GCG. In many eukaryotes a major cause of mutation at cytosine nucleotides is spontaneous deamination of methylated cytosines (5-mC) to thymine, which results in a T:G mismatch that can then be repaired, or replicated, to give a C:G → T:A substitution. Deamination of methylated cytosines was hypothesized to in part explain the elevated mutation rate of C:G base pairs in <em>S. cerevisiae</em> (<a id="xref-ref-79-16" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>), in spite of the fact that 5-mC is generally thought to be absent in this species (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Antequera <em>et al.</em> 1984</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Proffitt <em>et al.</em> 1984</a>). The reason for considering the possibility that CpG mutation is caused by methylation is the low level of DNA methylation reported in <em>S. cerevisiae</em> (<a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Tang <em>et al.</em> 2012</a>). <em>Sc. pombe</em> is thought to lack 5-mC (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Antequera <em>et al.</em> 1984</a>), though, unlike <em>S. cerevisiae</em>, it does have a DNA methyltransferase homolog, <em>pmt1</em> (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wilkinson <em>et al.</em> 1995</a>), so methylation might occur. However, even if cytosines are methylated at a very low rate, our data suggest that methylation cannot explain the mutational bias at these sites. Methylated cytosines cause C:G → T:A transition mutations because of spontaneous deamination, not the observed C:G → A:T transversions. Additionally, the mutation biases at the C:G base pair in a CpG dinucleotide are no different from the biases at a C:G base pair that is not in a CpG context. Together, these observations suggest a mechanism other than deamination of methylated cytosines is driving the increased relative mutation rate at CpG dinucleotides. This suggests that methylation may not be the only cause of high mutation rates at CpG dinucleotides in other species. Surprisingly, the <a id="xref-ref-16-15" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a> study reported that C:G → T:A transitions were significantly more common at CpG sites than in the non-CpG context, which is in sharp contrast to our finding, where both contexts show the same spectrum of mutations.</p></div><div id="sec-26" class="subsection"><h3>Spectrum of single nucleotide mutations</h3><p id="p-60">If differences between the ancestor and the reference represent both neutral and selected mutations, then the mutation spectrum might differ from that observed among the MA lines. In the MA lines SNMs represent 46.9% of the observed mutations, while in the ancestor they are significantly less common. The lower proportion of SNMs among differences between the ancestor and reference suggests that many of the SNMs that arose in the lineages linking the two strains were deleterious and were thus prevented from fixing. The location of the SNMs in the genome with respect to exon <em>vs.</em> intron, and noncoding regions is similar in both the ancestor and MA lines, suggesting that many of SNMs present in the ancestor relative to the reference are likely neutral (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/FigureS6.pdf">Figure S6</a>).</p></div><div id="sec-27" class="subsection"><h3>Divergence of the ancestor and reference strains</h3><p id="p-61">Both the ancestor and the sequenced reference strains have the same strain designation, 972 h–. However, given that they differ in sequence, they must have been separately passaged for some period of time. If all 80 of the observed SNM differences between the ancestor and the reference strains are neutral, that would suggest the two strains are between 26,800 and 36,900 generations diverged, depending on the mutation rate used to estimate the number of generations. Assuming that a strain might be passaged at most 50 times per year (∼1000 cell generations), this would indicate that the strains are almost 32 years separated (16 years of independent culturing in the lab if both were being passaged). The 972 h– strain was brought into the lab in the 1940s (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Leupold 1950</a>), so this number of generations is certainly possible. Alternatively, these mutations may have accumulated during refrigerative storage, though we know of no data to test this hypothesis. Another possibility is that the ancestor and reference strains are fewer generations diverged, with many of the sequence differences having been fixed by positive selection. Selection seems to be a less likely explanation because the distribution of effects of mutational differences between the ancestor and reference is strikingly similar to that for the MA lines, where selection is known to be minimal (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1/FigureS6.pdf">Figure S6</a>).</p></div><div id="sec-28" class="subsection"><h3>Reproducibility of MA experiments</h3><p id="p-62">Throughout, we have compared our results to those of a recently published mutation accumulation study in <em>Sc. pombe</em> (<a id="xref-ref-16-16" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> 2015</a>). It is striking how the results from the two studies are so similar, especially given that we used different strains (972 h– <em>vs.</em> ED668, BG_0000H8 h+), and passaged them at different temperatures (30 <em>vs.</em> 32°) on different media (YPD <em>vs.</em> YES) at different intervals (every 2 <em>vs.</em> every 3–4 days). In our study, the specifics of the protocol were chosen in order to be identical to the conditions used in the <em>S. cerevisiae</em> MA study (<a id="xref-ref-79-17" class="xref-bibr" href="#ref-79">Zhu <em>et al.</em> 2014</a>) to facilitate a comparison of the results between these two species. Fortuitously, 30° is the optimal growth temperature for our <em>Sc. pombe</em> isolate (<a href="http://www.atcc.org/Products/All/26189.aspx">http://www.atcc.org/Products/All/26189.aspx</a>).</p><p id="p-63">Comparing the two studies, the SNM mutation rate estimates are essentially identical (when we include SNMs from double and complex mutations), the SNM spectrum (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>) and the indel bias are essentially identical, and mutations in flocculin genes are overrepresented. Further, neither study found an effect of G/C content, transcription levels, scored as transcript abundance, or replication timing on local SNM rate. One discrepancy between the studies, the three-fold difference in indel rate, may be due to a high rate of missed mutations in their study and may thus not be a real difference; these indels could perhaps be discovered upon reanalysis. The long list of similar findings is reassuring given that MA experiments are time-consuming to repeat. However, there is one striking disparity between the two studies. CpG mutations in the <a id="xref-ref-16-17" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a> study were biased in favor of C:G → T:A, even though C:G → A:T mutations were more common when averaged across all C:G sites. In contrast, we found C:G → A:T mutations were more common in CpG dinucleotides, and the spectrum of mutations at CpG sites did not vary from the genome-wide mutation spectrum. This is an important difference because our study indicates that it is not methylation of cytosine that is driving the higher mutation rate at CpG sites, while the results of the <a id="xref-ref-16-18" class="xref-bibr" href="#ref-16">Farlow <em>et al.</em> (2015)</a> study suggest that methylation could be the cause. In summary, we find that <em>Sc. pombe</em> and <em>S. cerevisiae</em> have essentially identical base pair mutation rates, which, coupled with genome-wide estimates of within-species polymorphisms, suggests their effective population sizes are similar. We observe considerable differences in the mutation spectrum and indel rate, suggesting that there are species-specific differences in factors affecting mutation bias and indel rate between these two species. Additionally, we note that the rates of small insertions <em>vs.</em> deletions predict a growing genome, suggesting either that the genome size in <em>Sc. pombe</em> is not at equilibrium, or that rare large deletions offset increases caused by small indel bias. The sample size of captured large insertions and deletions was insufficient to test this hypothesis. We also found that CpG sites are highly mutagenic, but the mutation bias at these sites is not caused by deamination of methylated cytosine, suggesting another factor makes these sites prone to mutation. The results of this MA experiment are reassuringly similar to the results reported from a recently published MA experiment using <em>Sc. pombe</em>, suggesting that these experiments can give repeatable estimates despite differences in starting strains and growth conditions.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">We would like to thank the Associate Editor and two anonymous reviewers for very helpful comments on a previous version of this manuscript. This work was supported by the National Institutes of Health grants R01GM097415 and T32 GM007103.</p><p id="p-65">Author contributions: D.W.H. designed the research. M.G.B. performed the research and did the bioinformatics. D.W.H. and M.G.B. performed the data analysis and wrote the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-66">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.022129/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-67"><em>Communicating editor: J. C. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-09-03"><span class="received-label">Received </span>September 3, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-11-06"><span class="accepted-label">Accepted </span>November 6, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Behringer and Hall</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.1" data-doi="10.1038/nature04107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Adaptive evolution of non-coding DNA in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1149</span>–<span class="cit-lpage">1152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAndolfatto%26rft.auinit1%253DP.%26rft.volume%253D437%26rft.issue%253D7062%26rft.spage%253D1149%26rft.epage%253D1152%26rft.atitle%253DAdaptive%2Bevolution%2Bof%2Bnon-coding%2BDNA%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04107%26rft_id%253Dinfo%253Apmid%252F16237443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16237443&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232660500043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Antequera</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamame</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">DNA methylation in the fungi.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">259</span>: <span class="cit-fpage">8033</span>–<span class="cit-lpage">8036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DAntequera%26rft.auinit1%253DF.%26rft.volume%253D259%26rft.issue%253D13%26rft.spage%253D8033%26rft.epage%253D8036%26rft.atitle%253DDNA%2Bmethylation%2Bin%2Bthe%2Bfungi%26rft_id%253Dinfo%253Apmid%252F6330093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNTkvMTMvODAzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-web" id="cit-6.1.149.3"><div class="cit-metadata unstructured">Aronesty, E., 2011 ea-utils: Command-line tools for processing biological sequencing data. Available at: <a href="http://code.google.com/p/ea-utils">http://code.google.com/p/ea-utils</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.4" data-doi="10.2144/000112890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A simple way to treat PCR products prior to sequencing using ExoSAP-IT.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DBell%26rft.auinit1%253DJ.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D834%26rft.epage%253D834%26rft.atitle%253DA%2Bsimple%2Bway%2Bto%2Btreat%2BPCR%2Bproducts%2Bprior%2Bto%2Bsequencing%2Busing%2BExoSAP-IT.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000112890%26rft_id%253Dinfo%253Apmid%252F18476841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000112890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18476841&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.5" data-doi="10.1016/0955-0674(94)90030-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bestor</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdine</span>  <span class="cit-name-given-names">G. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">DNA methyltransferases.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">380</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DBestor%26rft.auinit1%253DT.%2BH.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D380%26rft.epage%253D389%26rft.atitle%253DDNA%2Bmethyltransferases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0955-0674%252894%252990030-2%26rft_id%253Dinfo%253Apmid%252F7917329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0955-0674(94)90030-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7917329&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NQ47800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.6" data-doi="10.1534/genetics.109.101998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blumenstiel</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noll</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perera</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walton</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of EMS-induced mutations in <em>Drosophila melanogaster</em> by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101998%26rft_id%253Dinfo%253Apmid%252F19307605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTgyLzEvMjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.7" data-doi="10.1128/MCB.4.11.2529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chlebowicz-Sledziewska</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Cell cycle phases in the unequal mother/daughter cell cycles of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2529</span>–<span class="cit-lpage">2531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBrewer%26rft.auinit1%253DB%2BJ%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D2529%26rft.epage%253D2531%26rft.atitle%253DCell%2Bcycle%2Bphases%2Bin%2Bthe%2Bunequal%2Bmother%252Fdaughter%2Bcell%2Bcycles%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.4.11.2529%26rft_id%253Dinfo%253Apmid%252F6392855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjQvMTEvMjUyOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.8" data-doi="10.1534/genetics.106.067850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavero</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chahwan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Xlf1 is required for DNA repair by nonhomologous end joining in <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067850%26rft_id%253Dinfo%253Apmid%252F17151234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzIvOTYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cahill</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loeb</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">8-Hydroxyguanine, an abundant form of oxidative DNA damage, causes G—-T and A—-C substitutions.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">267</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DCheng%26rft.auinit1%253DK.%2BC.%26rft.volume%253D267%26rft.issue%253D1%26rft.spage%253D166%26rft.epage%253D172%26rft.atitle%253D8-Hydroxyguanine%252C%2Ban%2Babundant%2Bform%2Bof%2Boxidative%2BDNA%2Bdamage%252C%2Bcauses%2BG----%2BT%2Band%2BA----C%2Bsubstitutions%26rft_id%253Dinfo%253Apmid%252F1730583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjk6IjI2Ny8xLzE2NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.10" data-doi="10.1146/annurev.genet.39.073003.113340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daley</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmbos</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">T. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Nonhomologous end joining in yeast.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DDaley%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D39%26rft.spage%253D431%26rft.epage%253D451%26rft.atitle%253DNonhomologous%2Bend%2Bjoining%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.39.073003.113340%26rft_id%253Dinfo%253Apmid%252F16285867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.39.073003.113340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16285867&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234685200019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.11" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albers</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTUvMjE1NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.12" data-doi="10.1038/nature02697"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">High mutation rate and predominance of insertions in the <em>Caenorhabditis elegans</em> nuclear genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">430</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">682</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDenver%26rft.auinit1%253DD.%2BR.%26rft.volume%253D430%26rft.issue%253D7000%26rft.spage%253D679%26rft.epage%253D682%26rft.atitle%253DHigh%2Bmutation%2Brate%2Band%2Bpredominance%2Bof%2Binsertions%2Bin%2Bthe%2BCaenorhabditis%2Belegans%2Bnuclear%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02697%26rft_id%253Dinfo%253Apmid%252F15295601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02697&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15295601&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223085400046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.13" data-doi="10.1534/genetics.106.059840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feinberg</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steding</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The relative roles of three DNA repair pathways in preventing <em>Caenorhabditis elegans</em> mutation accumulation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.059840%26rft_id%253Dinfo%253Apmid%252F16783005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTc0LzEvNTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gafner</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dolan</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Variation in base-substitution mutation in experimental and natural lineages of <em>Caenorhabditis</em> nematodes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">4</span>: 531–522.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.15" data-doi="10.1073/pnas.0403984101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Douzery</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snell</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bapteste</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delsuc</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippe</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The timing of eukaryotic evolution: does a relaxed molecular clock reconcile proteins and fossils?</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">15386</span>–<span class="cit-lpage">15391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0403984101%26rft_id%253Dinfo%253Apmid%252F15494441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQzLzE1Mzg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farlow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnoux</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doak</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> The spontaneous mutation rate in the fission yeast <em>Schizosaccharomyces pombe</em>. Genetics 201: 737–744.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.17" data-doi="10.1126/science.274.5287.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffeau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Life with 6000 genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">546</span>–<span class="cit-lpage">567</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoffeau%26rft.auinit1%253DA.%26rft.volume%253D274%26rft.issue%253D5287%26rft.spage%253D546%26rft.epage%253D567%26rft.atitle%253DLife%2Bwith%2B6000%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5287.546%26rft_id%253Dinfo%253Apmid%252F8849441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTI4Ny81NDYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-web" id="cit-6.1.149.18"><div class="cit-metadata unstructured">Gordon, A., and G. Hannon, 2010 Fastx-toolkit. FASTQ/A short-reads preprocessing tools (unpublished). Available at: <a href="http://hannonlab.cshl.edu/fastx_toolkit">http://hannonlab. cshl. edu/fastx_toolkit</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.149.19"><div class="cit-metadata unstructured">Graur, D., 2003 Single-base mutation, pp. 287–290 in <em>Nature encyclopedia of the human genome</em>, edited by D. Cooper, Macmillan Publishers, Nature Publishing Group, London.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Codomo</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Spectrum of chemically induced mutations from a large-scale reverse-genetic screen in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGreene%26rft.auinit1%253DE.%2BA.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D731%26rft.epage%253D740%26rft.atitle%253DSpectrum%2Bof%2BChemically%2BInduced%2BMutations%2BFrom%2Ba%2BLarge-Scale%2BReverse-Genetic%2BScreen%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Apmid%252F12807792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvNzMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.21" data-doi="10.1017/S0016672308009324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahmoudizad</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurd</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Spontaneous mutations in diploid <em>Saccharomyces cerevisiae</em>: another thousand cell generations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHall%26rft.auinit1%253DD.%2BW.%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D229%26rft.epage%253D241%26rft.atitle%253DSpontaneous%2Bmutations%2Bin%2Bdiploid%2BSaccharomyces%2Bcerevisiae%253A%2Banother%2Bthousand%2Bcell%2Bgenerations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009324%26rft_id%253Dinfo%253Apmid%252F18593510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18593510&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257787100002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.22" data-doi="10.1146/annurev.ecolsys.39.110707.173437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halligan</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Spontaneous mutation accumulation studies in evolutionary genetics.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D40%26rft.spage%253D151%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.39.110707.173437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.39.110707.173437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.23" data-doi="10.1126/science.1061457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heckman</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geiser</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eidell</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stauffer</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kardos</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular evidence for the early colonization of land by fungi and plants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">293</span>: <span class="cit-fpage">1129</span>–<span class="cit-lpage">1133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHeckman%26rft.auinit1%253DD.%2BS.%26rft.volume%253D293%26rft.issue%253D5532%26rft.spage%253D1129%26rft.epage%253D1133%26rft.atitle%253DMolecular%2BEvidence%2Bfor%2Bthe%2BEarly%2BColonization%2Bof%2BLand%2Bby%2BFungi%2Band%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1061457%26rft_id%253Dinfo%253Apmid%252F11498589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTMvNTUzMi8xMTI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.24" data-doi="10.1038/sj.emboj.7601390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heichinger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Penkett</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bähler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurse</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genome‐wide characterization of fission yeast DNA replication origins.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">5171</span>–<span class="cit-lpage">5179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DHeichinger%26rft.auinit1%253DC.%26rft.volume%253D25%26rft.issue%253D21%26rft.spage%253D5171%26rft.epage%253D5179%26rft.atitle%253DGenome-wide%2Bcharacterization%2Bof%2Bfission%2Byeast%2BDNA%2Breplication%2Borigins%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7601390%26rft_id%253Dinfo%253Apmid%252F17053780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjEvNTE3MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.25" data-doi="10.1146/annurev.genet.42.110807.091442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hershberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection on codon bias.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DHershberg%26rft.auinit1%253DR.%26rft.volume%253D42%26rft.spage%253D287%26rft.epage%253D299%26rft.atitle%253DSelection%2Bon%2Bcodon%2Bbias.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091442%26rft_id%253Dinfo%253Apmid%252F18983258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091442&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18983258&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.26" data-doi="10.1093/molbev/msi145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Time dependency of molecular rate estimates and systematic overestimation of recent divergence times.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1561</span>–<span class="cit-lpage">1568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi145%26rft_id%253Dinfo%253Apmid%252F15814826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIyLzcvMTU2MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.27" data-doi="10.1101/gad.1217204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leonard</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindberg</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bollmann</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hays</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Rapid accumulation of mutations during seed-to-seed propagation of mismatch-repair-defective <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2676</span>–<span class="cit-lpage">2685</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHoffman%26rft.auinit1%253DP.%2BD.%26rft.volume%253D18%26rft.issue%253D21%26rft.spage%253D2676%26rft.epage%253D2685%26rft.atitle%253DRapid%2Baccumulation%2Bof%2Bmutations%2Bduring%2Bseed-to-seed%2Bpropagation%2Bof%2Bmismatch-repair-defective%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1217204%26rft_id%253Dinfo%253Apmid%252F15520284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE4LzIxLzI2NzYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holbeck</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strathern</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A role for REV3 in mutagenesis during double-strand break repair in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">1017</span>–<span class="cit-lpage">1024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHolbeck%26rft.auinit1%253DS.%2BL.%26rft.volume%253D147%26rft.issue%253D3%26rft.spage%253D1017%26rft.epage%253D1024%26rft.atitle%253DA%2BRole%2Bfor%2BREV3%2Bin%2BMutagenesis%2BDuring%2BDouble-Strand%2BBreak%2BRepair%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F9383049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0Ny8zLzEwMTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.29" data-doi="10.1016/S0027-5107(00)00051-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Humphrey</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">DNA damage and cell cycle control in <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Mutat. Res. Fundam. Mol. Mech. Mutagen.</abbr> <span class="cit-vol">451</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%2BFundam.%2BMol.%2BMech.%2BMutagen.%26rft.volume%253D451%26rft.spage%253D211%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0027-5107%252800%252900051-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0027-5107(00)00051-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.30" data-doi="10.1016/S0378-1119(00)00489-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hurst</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">E. J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Covariation of GC content and the silent site substitution rate in rodents: implications for methodology and for the evolution of isochores.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">261</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DHurst%26rft.auinit1%253DL.%2BD.%26rft.volume%253D261%26rft.issue%253D1%26rft.spage%253D107%26rft.epage%253D114%26rft.atitle%253DCovariation%2Bof%2BGC%2Bcontent%2Band%2Bthe%2Bsilent%2Bsite%2Bsubstitution%2Brate%2Bin%2Brodents%253A%2Bimplications%2Bfor%2Bmethodology%2Band%2Bfor%2Bthe%2Bevolution%2Bof%2Bisochores.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252800%252900489-3%26rft_id%253Dinfo%253Apmid%252F11164042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(00)00489-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11164042&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166759700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.31" data-doi="10.1038/ng.3215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeffares</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rallis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieux</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Převorovský</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genomic and phenotypic diversity of <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Nature Genetics</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BGenetics%26rft.volume%253D47%26rft.spage%253D235%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3215%26rft_id%253Dinfo%253Apmid%252F25665008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25665008&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.32" data-doi="10.1534/genetics.104.033761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Spontaneous mutations in diploid <em>Saccharomyces cerevisiae</em> more beneficial than expected.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">1817</span>–<span class="cit-lpage">1825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DJoseph%26rft.auinit1%253DS.%2BB.%26rft.volume%253D168%26rft.issue%253D4%26rft.spage%253D1817%26rft.epage%253D1825%26rft.atitle%253DSpontaneous%2BMutations%2Bin%2BDiploid%2BSaccharomyces%2Bcerevisiae%253A%2BMore%2BBeneficial%2BThan%2BExpected%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.033761%26rft_id%253Dinfo%253Apmid%252F15611159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OC80LzE4MTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.33" data-doi="10.1093/molbev/msi057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karaoglu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C. M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Survey of simple sequence repeats in completed fungal genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi057%26rft_id%253Dinfo%253Apmid%252F15563717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNjM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trivedi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>  Analysis of the genome sequences of three <em>Drosophila melanogaster</em> spontaneous mutation accumulation lines. Genome Res. 19: 1195–1201.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.35" data-doi="10.1371/journal.pone.0017175"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knutsen</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rein</span>  <span class="cit-name-given-names">I. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stokke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grallert</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cell-cycle analysis of fission yeast cells by flow cytometry.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e17175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKnutsen%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De17175%26rft.epage%253De17175%26rft.atitle%253DCell-cycle%2Banalysis%2Bof%2Bfission%2Byeast%2Bcells%2Bby%2Bflow%2Bcytometry.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0017175%26rft_id%253Dinfo%253Apmid%252F21386895%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0017175&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21386895&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.36" data-doi="10.1016/0027-5107(82)90129-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koornneeff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dellaert</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van der Veen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">EMS-and relation-induced mutation frequencies at individual loci in <em>Arabidopsis thaliana (L.) Heynh</em>.</span> <abbr class="cit-jnl-abbrev">Mutat. Res. Fundam. Mol. Mech. Mutagen.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%2BFundam.%2BMol.%2BMech.%2BMutagen.%26rft.volume%253D93%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0027-5107%252882%252990129-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0027-5107(82)90129-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leupold</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">Die vererbung von homothallie und heterothallie bei Schizosaccharomyces pombe.</span> <abbr class="cit-jnl-abbrev">C. R. Trav. Lab. Carlsberg., Ser. Physiol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DC.%2BR.%2BTrav.%2BLab.%2BCarlsberg.%252C%2BSer.%2BPhysiol.%26rft.volume%253D24%26rft.spage%253D381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levinson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutman</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Slipped-strand mispairing: a major mechanism for DNA sequence evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DLevinson%26rft.auinit1%253DG%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D203%26rft.epage%253D221%26rft.atitle%253DSlipped-strand%2Bmispairing%253A%2Ba%2Bmajor%2Bmechanism%2Bfor%2BDNA%2Bsequence%2Bevolution%26rft_id%253Dinfo%253Apmid%252F3328815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjQvMy8yMDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.39" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.40" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.41" data-doi="10.1016/j.tig.2010.05.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Evolution of the mutation rate.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D26%26rft.issue%253D8%26rft.spage%253D345%26rft.epage%253D352%26rft.atitle%253DEvolution%2Bof%2Bthe%2Bmutation%2Brate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2010.05.003%26rft_id%253Dinfo%253Apmid%252F20594608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2010.05.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20594608&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280903800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.42" data-doi="10.1073/pnas.0803466105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-wide view of the spectrum of spontaneous mutations in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">9272</span>–<span class="cit-lpage">9277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803466105%26rft_id%253Dinfo%253Apmid%252F18583475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI3LzkyNzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.43" data-doi="10.1017/S0016672300004821"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Masking and purging mutations following EMS treatment in haploid, diploid and tetraploid yeast (<em>Saccharomyces cerevisiae</em>).</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DMable%26rft.auinit1%253DB.%2BK.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D26%26rft.atitle%253DMasking%2Band%2Bpurging%2Bmutations%2Bfollowing%2BEMS%2Btreatment%2Bin%2Bhaploid%252C%2Bdiploid%2Band%2Btetraploid%2Byeast%2B%2528Saccharomyces%2Bcerevisiae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300004821%26rft_id%253Dinfo%253Apmid%252F11279834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300004821&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11279834&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168340200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mansour</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tornier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darmon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fleck</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Control of GT repeat stability in <em>Schizosaccharomyces pombe</em> by mismatch repair factors.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMansour%26rft.auinit1%253DA.%2BA.%26rft.volume%253D158%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D85%26rft.atitle%253DControl%2Bof%2BGT%2BRepeat%2BStability%2Bin%2BSchizosaccharomyces%2Bpombe%2Bby%2BMismatch%2BRepair%2BFactors%26rft_id%253Dinfo%253Apmid%252F11333219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTU4LzEvNzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.45" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.46" data-doi="10.1093/bioinformatics/btq330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rios</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flicek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Deriving the consequences of genomic variants with the Ensembl API and SNP Effect Predictor.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2069</span>–<span class="cit-lpage">2070</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq330%26rft_id%253Dinfo%253Apmid%252F20562413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTYvMjA2OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.47" data-doi="10.1007/BF02984195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-article-title">Types of visible variations induced by X-rays in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D22%26rft.spage%253D299%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02984195%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02984195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000200610100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nachman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crowell</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Estimate of the mutation rate per nucleotide in humans.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">297</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNachman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D156%26rft.issue%253D1%26rft.spage%253D297%26rft.epage%253D304%26rft.atitle%253DEstimate%2Bof%2Bthe%2BMutation%2BRate%2Bper%2BNucleotide%2Bin%2BHumans%26rft_id%253Dinfo%253Apmid%252F10978293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU2LzEvMjk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">A control acting over the initiation of DNA replication in the yeast <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DNasmyth%26rft.auinit1%253DK.%2BA.%26rft.volume%253D36%26rft.issue%253D1%26rft.spage%253D155%26rft.epage%253D168%26rft.atitle%253DA%2Bcontrol%2Bacting%2Bover%2Bthe%2Binitiation%2Bof%2BDNA%2Breplication%2Bin%2Bthe%2Byeast%2BSchizosaccharomyces%2Bpombe%26rft_id%253Dinfo%253Apmid%252F457803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6ODoiMzYvMS8xNTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.50" data-doi="10.1534/genetics.112.145078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ness</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colegrave</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keightley</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimate of the spontaneous mutation rate in <em>Chlamydomonas reinhardtii</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1447</span>–<span class="cit-lpage">1454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145078%26rft_id%253Dinfo%253Apmid%252F23051642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi80LzE0NDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.51" data-doi="10.1371/journal.pgen.1001109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishant</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Argueso</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlattl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The baker’s yeast diploid genome is remarkably stable in vegetative growth and meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D6%26rft.spage%253De1001109%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001109%26rft_id%253Dinfo%253Apmid%252F20838597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20838597&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.52" data-doi="10.1007/BF00434051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niwa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanagida</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Triploid meiosis and aneuploidy in <em>Schizosaccharomyces pombe</em>: an unstable aneuploid disomic for chromosome III.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BGenet.%26rft.volume%253D9%26rft.spage%253D463%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00434051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00434051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1985AKX5900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.53" data-doi="10.1002/yea.1411"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niwa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tange</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurabayashi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Growth arrest and chromosome instability in aneuploid yeast.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">937</span>–<span class="cit-lpage">950</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DNiwa%26rft.auinit1%253DO.%26rft.volume%253D23%26rft.issue%253D13%26rft.spage%253D937%26rft.epage%253D950%26rft.atitle%253DGrowth%2Barrest%2Band%2Bchromosome%2Binstability%2Bin%2Baneuploid%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1411%26rft_id%253Dinfo%253Apmid%252F17072887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1411&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17072887&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242305500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.54" data-doi="10.1038/embor.2012.165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic evidence for elevated mutation rates in highly expressed genes.</span> <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1123</span>–<span class="cit-lpage">1129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BRep.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fembor.2012.165%26rft_id%253Dinfo%253Apmid%252F23146897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZW1ib3IiO3M6NToicmVzaWQiO3M6MTA6IjEzLzEyLzExMjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.55" data-doi="10.1128/MCB.4.5.985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Proffitt</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davie</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swinton</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattman</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">5-Methylcytosine is not detectable in <em>Saccharomyces cerevisiae</em> DNA.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">985</span>–<span class="cit-lpage">988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DProffitt%26rft.auinit1%253DJ%2BH%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253D985%26rft.epage%253D988%26rft.atitle%253D5-Methylcytosine%2Bis%2Bnot%2Bdetectable%2Bin%2BSaccharomyces%2Bcerevisiae%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.4.5.985%26rft_id%253Dinfo%253Apmid%252F6374428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjc6IjQvNS85ODUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.56" data-doi="10.1002/yea.1414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raji</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartsuiker</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Double and S. Hattman, 1984 5-Methylogous recombination in <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DRaji%26rft.auinit1%253DH.%26rft.volume%253D23%26rft.issue%253D13%26rft.spage%253D963%26rft.epage%253D976%26rft.atitle%253DDouble-strand%2Bbreak%2Brepair%2Band%2Bhomologous%2Brecombination%2Bin%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1414%26rft_id%253Dinfo%253Apmid%252F17072889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17072889&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242305500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.57" data-doi="10.1093/bioinformatics/bts378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rausch</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zichner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlattl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stütz</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benes</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">DELLY: structural variant discovery by integrated paired-end and split-read analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">i333</span>–<span class="cit-lpage">i339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts378%26rft_id%253Dinfo%253Apmid%252F22962449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTgvaTMzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.58" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdóttir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.149.59"><div class="cit-metadata unstructured">Rozen, S., and H. Skaletsky, 1999 Primer3 on the WWW for general users and for biologist programmers, pp. 365–386 in <em>Methods in Molecular Biology</em>, <em>Vol. 132: Bioinformatics methods and protocols</em>, edited by S. Misener, and S. A. Krawetz. Humana Press, Totowa, NJ.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.60" data-doi="10.1128/MCB.19.1.241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rudolph</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parisi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartsuiker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The <em>msh2</em> Gene of <em>Schizosaccharomyces pombe</em> Is Involved in Mismatch Repair, Mating-Type Switching, and Meiotic Chromosome Organization.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DRudolph%26rft.auinit1%253DC.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D241%26rft.epage%253D250%26rft.atitle%253DThe%2Bmsh2%2BGene%2Bof%2BSchizosaccharomyces%2Bpombe%2BIs%2BInvolved%2Bin%2BMismatch%2BRepair%252C%2BMating-Type%2BSwitching%252C%2Band%2BMeiotic%2BChromosome%2BOrganization%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.1.241%26rft_id%253Dinfo%253Apmid%252F9858548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMjQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.61" data-doi="10.1111/j.1558-5646.2012.01583.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutter</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roles</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conner</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fitness of <em>Arabidopsis thaliana</em> mutation accumulation lines whose spontaneous mutations are known.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">2335</span>–<span class="cit-lpage">2339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DRutter%26rft.auinit1%253DM.%2BT.%26rft.volume%253D66%26rft.issue%253D7%26rft.spage%253D2335%26rft.epage%253D2339%26rft.atitle%253DFitness%2Bof%2BArabidopsis%2Bthaliana%2Bmutation%2Baccumulation%2Blines%2Bwhose%2Bspontaneous%2Bmutations%2Bare%2Bknown.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2012.01583.x%26rft_id%253Dinfo%253Apmid%252F22759306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2012.01583.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22759306&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305945400027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.62" data-doi="10.1371/journal.pone.0046759"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saxer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Havlak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quance</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole genome sequencing of mutation accumulation lines reveals a low mutation rate in the social amoeba <em>Dictyostelium discoideum</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSaxer%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46759%26rft.epage%253De46759%26rft.atitle%253DWhole%2BGenome%2BSequencing%2Bof%2BMutation%2BAccumulation%2BLines%2BReveals%2Ba%2BLow%2BMutation%2BRate%2Bin%2Bthe%2BSocial%2BAmoeba%2BDictyostelium%2Bdiscoideum.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046759%26rft_id%253Dinfo%253Apmid%252F23056439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046759&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23056439&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.63" data-doi="10.1038/nmeth.2089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasband</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eliceiri</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">NIH Image to ImageJ: 25 years of image analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchneider%26rft.auinit1%253DC.%2BA.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D671%26rft.epage%253D675%26rft.atitle%253DNIH%2BImage%2Bto%2BImageJ%253A%2B25%2Byears%2Bof%2Bimage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2089%26rft_id%253Dinfo%253Apmid%252F22930834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22930834&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305942200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.64" data-doi="10.1016/j.cub.2011.05.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schrider</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hourmozdi</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Pervasive multinucleotide mutational events in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1051</span>–<span class="cit-lpage">1054</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSchrider%26rft.auinit1%253DD.%2BR.%26rft.volume%253D21%26rft.issue%253D12%26rft.spage%253D1051%26rft.epage%253D1054%26rft.atitle%253DPervasive%2Bmultinucleotide%2Bmutational%2Bevents%2Bin%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.05.013%26rft_id%253Dinfo%253Apmid%252F21636278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.05.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21636278&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.65" data-doi="10.1093/nar/22.6.972"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rideout</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The rate of hydrolytic deamination of 5-methylcytosine in double-stranded DNA.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">972</span>–<span class="cit-lpage">976</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DShen%26rft.auinit1%253DJ.-C.%26rft.volume%253D22%26rft.issue%253D6%26rft.spage%253D972%26rft.epage%253D976%26rft.atitle%253DThe%2Brate%2Bof%2Bhydrolytic%2Bdeamination%2Bof%2B5-methylcytosine%2Bin%2Bdouble-stranded%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F22.6.972%26rft_id%253Dinfo%253Apmid%252F8152929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjIyLzYvOTcyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.66" data-doi="10.1038/cr.2007.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shrivastav</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Haro</span>  <span class="cit-name-given-names">L. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickoloff</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Regulation of DNA double-strand break repair pathway choice.</span> <abbr class="cit-jnl-abbrev">Cell Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2Bresearch%26rft.stitle%253DCell%2BRes%26rft.aulast%253DZhang%26rft.auinit1%253DJ.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D134%26rft.epage%253D147%26rft.atitle%253DRegulation%2Bof%2BDNA%2Bdouble-strand%2Bbreak%2Brepair%2Bpathway%2Bchoice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcr.2007.111%26rft_id%253Dinfo%253Apmid%252F18157161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cr.2007.111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18157161&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252920000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.67" data-doi="10.1073/pnas.74.9.3850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharrow</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gart</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Cell cycle of <em>Saccharomyces cerevisiae</em> in populations growing at different rates.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">3850</span>–<span class="cit-lpage">3854</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSlater%26rft.auinit1%253DM.%2BL.%26rft.volume%253D74%26rft.issue%253D9%26rft.spage%253D3850%26rft.epage%253D3854%26rft.atitle%253DCell%2Bcycle%2Bof%2BSaccharomycescerevisiae%2Bin%2Bpopulations%2Bgrowing%2Bat%2Bdifferent%2Brates.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.74.9.3850%26rft_id%253Dinfo%253Apmid%252F333447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3NC85LzM4NTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.68" data-doi="10.1038/ng.363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatoyannopoulos</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adzhubei</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kryukov</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mirkin</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Human mutation rate associated with DNA replication timing.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">393</span>–<span class="cit-lpage">395</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DStamatoyannopoulos%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D393%26rft.epage%253D395%26rft.atitle%253DHuman%2Bmutation%2Brate%2Bassociated%2Bwith%2BDNA%2Breplication%2Btiming.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.363%26rft_id%253Dinfo%253Apmid%252F19287383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19287383&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264659000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strathern</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shafer</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGill</span>  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">DNA synthesis errors associated with double-strand-break repair.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStrathern%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D140%26rft.issue%253D3%26rft.spage%253D965%26rft.epage%253D972%26rft.atitle%253DDNA%2Bsynthesis%2Berrors%2Bassociated%2Bwith%2Bdouble-strand-break%2Brepair.%26rft_id%253Dinfo%253Apmid%252F7672595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQwLzMvOTY1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.70" data-doi="10.1073/pnas.1216223109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackerman</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doak</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Drift-barrier hypothesis and mutation-rate evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">18488</span>–<span class="cit-lpage">18492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1216223109%26rft_id%253Dinfo%253Apmid%252F23077252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ1LzE4NDg4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.71" data-doi="10.1073/pnas.1210663109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doak</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Extraordinary genome stability in the ciliate <em>Paramecium tetraurelia</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">19339</span>–<span class="cit-lpage">19344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1210663109%26rft_id%253Dinfo%253Apmid%252F23129619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQ3LzE5MzM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.72" data-doi="10.1021/ac301727c"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">X.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">B.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.-Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Widespread existence of cytosine methylation in yeast DNA measured by gas chromatography/mass spectrometry.</span> <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">7249</span>–<span class="cit-lpage">7255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DTang%26rft.auinit1%253DY.%26rft.volume%253D84%26rft.issue%253D16%26rft.spage%253D7249%26rft.epage%253D7255%26rft.atitle%253DWidespread%2Bexistence%2Bof%2Bcytosine%2Bmethylation%2Bin%2Byeast%2BDNA%2Bmeasured%2Bby%2Bgas%2Bchromatography%252Fmass%2Bspectrometry.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac301727c%26rft_id%253Dinfo%253Apmid%252F22852529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac301727c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22852529&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.73" data-doi="10.1093/bioinformatics/btp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TopHat: discovering splice junctions with RNA-Seq.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp120%26rft_id%253Dinfo%253Apmid%252F19289445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNS85LzExMDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.74" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.75" data-doi="10.1038/ng1618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verstrepen</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewitter</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Intragenic tandem repeats generate functional variability.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">986</span>–<span class="cit-lpage">990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DVerstrepen%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D37%26rft.issue%253D9%26rft.spage%253D986%26rft.epage%253D990%26rft.atitle%253DIntragenic%2Btandem%2Brepeats%2Bgenerate%2Bfunctional%2Bvariability.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1618%26rft_id%253Dinfo%253Apmid%252F16086015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1618&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16086015&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231591900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.76" data-doi="10.1093/nar/23.2.203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>  <span class="cit-name-given-names">C. R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartlett</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurse</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The fission yeast gene pmt1+ encodes a DNA methyltransferase homologue.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWilkinson%26rft.auinit1%253DC.%2BR.%2BM.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D210%26rft.atitle%253DThe%2Bfission%2Byeast%2Bgene%2Bpmt1%252B%2Bencodes%2Ba%2BDNA%2Bmethyltransferase%2Bhomologue%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.2.203%26rft_id%253Dinfo%253Apmid%252F7862522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvMjAzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.77" data-doi="10.1038/nature724"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gwilliam</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajandream</span>  <span class="cit-name-given-names">M.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyne</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyne</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The genome sequence of <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">415</span>: <span class="cit-fpage">871</span>–<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWood%26rft.auinit1%253DV.%26rft.volume%253D415%26rft.issue%253D6874%26rft.spage%253D871%26rft.epage%253D880%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature724%26rft_id%253Dinfo%253Apmid%252F11859360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature724&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11859360&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F149.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173941000036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDowall</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutherford</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PomBase: a comprehensive online resource for fission yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: D695–D659.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.149.79" data-doi="10.1073/pnas.1323011111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegal</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Precise estimates of mutation rate and spectrum in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E2310</span>–<span class="cit-lpage">E2318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1323011111%26rft_id%253Dinfo%253Apmid%252F24847077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzIyL0UyMzEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzE0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/1/149.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/141" title="Comprehensive Transcriptome Analysis of Six Catfish Species from an Altitude Gradient Reveals Adaptive Evolution in Tibetan Fishes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/161" title="Genomic Analysis of ATP Efflux in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/1"><span>Volume 6 Issue 1, January 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/1.cover-source.jpg?itok=9zFiZvCK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/149" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-f2kebULp3eLuVSwktfF6c5BdMZ-6VfQ3ZLYWfUWuARY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/1/149" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/1/149" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/1/149" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Estimates of Mutation Rates and Spectrum in Schizosaccharomyces pombe Indicate CpG Sites are Highly Mutagenic Despite the Absence of DNA Methylation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460247" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LY4wkI8tkU4kimMYxwedw7EtDXxM5j7qNgxa6DgBgbg" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/149.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/1/149" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-TiFod_Q-TNvf8yOM-4EZpsDy80wr3uJv-5tyg5IsdSU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/1/149" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/149.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460247" id="node460247--2" data-pisa="ggg;6/1/149" data-pisa-master="ggg;g3.115.022129" data-apath="/ggg/6/1/149.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Estimates of Mutation Rates and Spectrum in <em>Schizosaccharomyces pombe</em> Indicate CpG Sites are Highly Mutagenic Despite the Absence of DNA Methylation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Megan G. Behringer</span> and <span class="highwire-citation-author" data-delta="1">David W. Hall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">149-160;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022129 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460247--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Megan G. Behringer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Megan%2BG.%2BBehringer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Behringer%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMegan%2BG.%2BBehringer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:megbehri@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">megbehri@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David W. Hall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BW.%2BHall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hall%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BW.%2BHall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/1/149" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/1/149&amp;title=Genome-Wide%20Estimates%20of%20Mutation%20Rates%20and%20Spectrum%20in%20Schizosaccharomyces%20pombe%20Indicate%20CpG%20Sites%20are%20Highly%20Mutagenic%20Despite%20the%20Absence%20of%20DNA%20Methylation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/1/149&amp;title=Genome-Wide%20Estimates%20of%20Mutation%20Rates%20and%20Spectrum%20in%20Schizosaccharomyces%20pombe%20Indicate%20CpG%20Sites%20are%20Highly%20Mutagenic%20Despite%20the%20Absence%20of%20DNA%20Methylation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/1/149&amp;title=Genome-Wide%20Estimates%20of%20Mutation%20Rates%20and%20Spectrum%20in%20Schizosaccharomyces%20pombe%20Indicate%20CpG%20Sites%20are%20Highly%20Mutagenic%20Despite%20the%20Absence%20of%20DNA%20Methylation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/1/149&amp;text=Genome-Wide%20Estimates%20of%20Mutation%20Rates%20and%20Spectrum%20in%20Schizosaccharomyces%20pombe%20Indicate%20CpG%20Sites%20are%20Highly%20Mutagenic%20Despite%20the%20Absence%20of%20DNA%20Methylation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/1/149&amp;title=Genome-Wide%20Estimates%20of%20Mutation%20Rates%20and%20Spectrum%20in%20Schizosaccharomyces%20pombe%20Indicate%20CpG%20Sites%20are%20Highly%20Mutagenic%20Despite%20the%20Absence%20of%20DNA%20Methylation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/1/149&amp;t=Genome-Wide%20Estimates%20of%20Mutation%20Rates%20and%20Spectrum%20in%20Schizosaccharomyces%20pombe%20Indicate%20CpG%20Sites%20are%20Highly%20Mutagenic%20Despite%20the%20Absence%20of%20DNA%20Methylation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/1/149&amp;title=Genome-Wide%20Estimates%20of%20Mutation%20Rates%20and%20Spectrum%20in%20Schizosaccharomyces%20pombe%20Indicate%20CpG%20Sites%20are%20Highly%20Mutagenic%20Despite%20the%20Absence%20of%20DNA%20Methylation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/1/149&amp;title=Genome-Wide%20Estimates%20of%20Mutation%20Rates%20and%20Spectrum%20in%20Schizosaccharomyces%20pombe%20Indicate%20CpG%20Sites%20are%20Highly%20Mutagenic%20Despite%20the%20Absence%20of%20DNA%20Methylation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460247/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460247" id="node460247--3" data-pisa="ggg;6/1/149" data-pisa-master="ggg;g3.115.022129" data-apath="/ggg/6/1/149.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Estimates of Mutation Rates and Spectrum in <em>Schizosaccharomyces pombe</em> Indicate CpG Sites are Highly Mutagenic Despite the Absence of DNA Methylation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Megan G. Behringer</span> and <span class="highwire-citation-author" data-delta="1">David W. Hall</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">149-160;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.022129 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460247--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Megan G. Behringer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Megan%2BG.%2BBehringer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Behringer%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMegan%2BG.%2BBehringer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:megbehri@indiana.edu" class="" data-icon-position="" data-hide-link-title="0">megbehri@indiana.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">David W. Hall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BW.%2BHall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hall%20DW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BW.%2BHall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=149&amp;rft.epage=160&amp;rft.atitle=Genome-Wide%20Estimates%20of%20Mutation%20Rates%20and%20Spectrum%20in%20Schizosaccharomyces%20pombe%20Indicate%20CpG%20Sites%20are%20Highly%20Mutagenic%20Despite%20the%20Absence%20of%20DNA%20Methylation&amp;rft.volume=6&amp;rft.issue=1&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Behringer%2C+Megan+G.&amp;rft.au=Hall%2C+David+W."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460247/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460247/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460247/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460247/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460247/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460247/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460247/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460247/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460247/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460247/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460247/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460247/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/1/149" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/1/149.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/1/149.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/1/149.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/1/149.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
