<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/1/67" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/1/67.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/1/67.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/1/67.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/1/67/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/1/67.atom" />
<meta name="HW.pisa" content="ggg;6/1/67" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.115.023671" />
<meta name="DC.Date" content="2016-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Guo et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Complex traits such as crop performance and human diseases are controlled by multiple genetic loci, many of which have small effects and often go undetected by traditional quantitative trait locus (QTL) mapping. Recently, bulked segregant analysis with large F2 pools and genome-level markers (named extreme-QTL or X-QTL mapping) has been used to identify many QTL. To estimate parameters impacting QTL detection for X-QTL mapping, we simulated the effects of population size, marker density, and sequencing depth of markers on QTL detectability for traits with differing heritabilities. These simulations indicate that a high (&gt;90%) chance of detecting QTL with at least 5% effect requires 5000× sequencing depth for a trait with heritability of 0.4−0.7. For most eukaryotic organisms, whole-genome sequencing at this depth is not economically feasible. Therefore, we tested and confirmed the feasibility of applying deep sequencing of target-enriched markers for X-QTL mapping. We used two traits in Arabidopsis thaliana with different heritabilities: seed size (H2 = 0.61) and seedling greening in response to salt (H2 = 0.94). We used a modified G test to identify QTL regions and developed a model-based statistical framework to resolve individual peaks by incorporating recombination rates. Multiple QTL were identified for both traits, including previously undiscovered QTL. We call our method target-enriched X-QTL (TEX-QTL) mapping; this mapping approach is not limited by the genome size or the availability of recombinant inbred populations and should be applicable to many organisms and traits." />
<meta name="DC.Contributor" content="Jianjun Guo" />
<meta name="DC.Contributor" content="Jue Fan" />
<meta name="DC.Contributor" content="Bernard A. Hauser" />
<meta name="DC.Contributor" content="Seung Y. Rhee" />
<meta name="article:published_time" content="2016-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Complex traits such as crop performance and human diseases are controlled by multiple genetic loci, many of which have small effects and often go undetected by traditional quantitative trait locus (QTL) mapping. Recently, bulked segregant analysis with large F2 pools and genome-level markers (named extreme-QTL or X-QTL mapping) has been used to identify many QTL. To estimate parameters impacting QTL detection for X-QTL mapping, we simulated the effects of population size, marker density, and sequencing depth of markers on QTL detectability for traits with differing heritabilities. These simulations indicate that a high (&amp;gt;90%) chance of detecting QTL with at least 5% effect requires 5000× sequencing depth for a trait with heritability of 0.4−0.7. For most eukaryotic organisms, whole-genome sequencing at this depth is not economically feasible. Therefore, we tested and confirmed the feasibility of applying deep sequencing of target-enriched markers for X-QTL mapping. We used two traits in &lt;i&gt;Arabidopsis thaliana&lt;/i&gt; with different heritabilities: seed size (H&lt;sup&gt;2&lt;/sup&gt; = 0.61) and seedling greening in response to salt (H&lt;sup&gt;2&lt;/sup&gt; = 0.94). We used a modified G test to identify QTL regions and developed a model-based statistical framework to resolve individual peaks by incorporating recombination rates. Multiple QTL were identified for both traits, including previously undiscovered QTL. We call our method target-enriched X-QTL (TEX-QTL) mapping; this mapping approach is not limited by the genome size or the availability of recombinant inbred populations and should be applicable to many organisms and traits.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="ggg;6/1/67" />
<meta name="citation_id" content="6/1/67" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/1/67" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/1/67.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/1/67.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/1/67.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.023671" />
<meta name="citation_pmid" content="26530422" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="67" />
<meta name="citation_lastpage" content="77" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jianjun Guo" />
<meta name="citation_author_institution" content="Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305" />
<meta name="citation_author" content="Jue Fan" />
<meta name="citation_author_institution" content="Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305" />
<meta name="citation_author" content="Bernard A. Hauser" />
<meta name="citation_author_institution" content="Department of Biology, Plant Molecular and Cellular Biology Program, University of Florida, Gainesville, Florida 32611" />
<meta name="citation_author" content="Seung Y. Rhee" />
<meta name="citation_author_institution" content="Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305" />
<meta name="citation_author_email" content="srhee@carnegiescience.edu" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. Alonso-Blanco;citation_author=H. Blankestijn-de Vries;citation_author=C. J. Hanhart;citation_author=M. Koornneef;citation_title=Natural allelic variation at seed size loci in relation to other life history traits of Arabidopsis thaliana;citation_pages=4710-4717;citation_volume=96;citation_year=1999;citation_issue=8;citation_pmid=10200327;citation_doi=10.1073/pnas.96.8.4710" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=Y. Asan;citation_author=H. Xu;citation_author=C. Jiang;citation_author=Y. Tyler-Smith;citation_author=. Xue;citation_title=Comprehensive comparison of three commercial human whole-exome capture platforms.;citation_pages=R95-R95;citation_volume=12;citation_year=2011;citation_issue=9;citation_pmid=21955857;citation_doi=10.1186/gb-2011-12-9-r95" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B.;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_title=A Guide to QTL Mapping With R/qtl;citation_author=K. W. Broman;citation_author=S. Sen;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. S. Buckler;citation_author=J. B. Holland;citation_author=P. J. Bradbury;citation_author=C. B. Acharya;citation_author=P. J. Brown;citation_title=The Genetic Architecture of Maize Flowering Time;citation_pages=714-718;citation_volume=325;citation_year=2009;citation_issue=5941;citation_pmid=19661422;citation_doi=10.1126/science.1174276" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=F. D. Chevalier;citation_author=C. Valentim;citation_author=P. T. LoVerde;citation_title=Efficient linkage mapping using exome capture and extreme QTL in schistosome parasites.;citation_pages=617;citation_volume=15;citation_year=2014;citation_pmid=25048426;citation_doi=10.1186/1471-2164-15-617" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. M. Clark;citation_author=G. Schweikert;citation_author=C. Toomajian;citation_author=S. Ossowski;citation_author=G. Zeller;citation_title=Common Sequence Polymorphisms Shaping Genetic Diversity in Arabidopsis thaliana;citation_pages=338-342;citation_volume=317;citation_year=2007;citation_issue=5836;citation_pmid=17641193;citation_doi=10.1126/science.1138632" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=E. J. Clerkx;citation_author=M. E. El-Lithy;citation_author=E. Vierling;citation_author=G. J. Ruys;citation_author=H. Blankestijn-De Vries;citation_title=Analysis of natural allelic variation of Arabidopsis seed germination and seed longevity traits between the accessions Landsberg erecta and Shakdara, using a new recombinant inbred line population.;citation_volume=135;citation_year=2004;citation_pmid=15122038;citation_doi=10.1104/pp.103.036814" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=W. S. Cleveland;citation_title=Robust locally weighted regression and smoothing scatterplots.;citation_pages=829-836;citation_volume=74;citation_year=1979;citation_doi=10.2307/2286407" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=L. DeRose-Wilson;citation_author=B. S. Gaut;citation_title=Mapping salinity tolerance during Arabidopsis thaliana germination and seedling growth.;citation_pages=e22832-e22832;citation_volume=6;citation_year=2011;citation_issue=8;citation_pmid=21857956;citation_doi=10.1371/journal.pone.0022832" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Duitama;citation_author=A. Sanchez-Rodriguez;citation_author=A. Goovaerts;citation_author=S. Pulido-Tamayo;citation_author=G. Hubmann;citation_title=Improved linkage analysis of quantitative trait loci using bulk segregants unveils a novel determinant of high ethanol tolerance in yeast.;citation_pages=207;citation_volume=15;citation_year=2014;citation_pmid=24640961;citation_doi=10.1186/1471-2164-15-207" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=M. D. Edwards;citation_author=D. K. Gifford;citation_title=High-resolution genetic mapping with pooled sequencing.;citation_pages=S8;citation_volume=13;citation_year=2012;citation_pmid=23282012;citation_doi=10.1186/1471-2105-13-S14-S8" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=I. Ehrenreich;citation_author=N. Torabi;citation_author=Y. Jia;citation_author=J. Kent;citation_author=S. Martis;citation_title=Dissection of genetically complex traits with extremely large pools of yeast segregants.;citation_pages=1039-1042;citation_volume=464;citation_year=2010;citation_issue=7291;citation_pmid=20393561;citation_doi=10.1038/nature08923" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. Galpaz;citation_author=M. Reymond;citation_title=Natural variation in Arabidopsis thaliana revealed a genetic network controlling germination under salt stress.;citation_pages=e15198-e15198;citation_volume=5;citation_year=2010;citation_issue=12;citation_pmid=21188145;citation_doi=10.1371/journal.pone.0015198" />
<meta name="citation_reference" content="citation_title=An Introduction To Genetic Analysis;citation_author=A. J. F. Griffiths;citation_author=J. H. Miller;citation_author=D. T. Suzuki;citation_author=R. C. Lewontin;citation_author=W. M. Gelbart;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. C. Koboldt;citation_author=Q. Zhang;citation_author=D. E. Larson;citation_author=D. Shen;citation_author=M. D. McLellan;citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.;citation_volume=22;citation_year=2012;citation_pmid=22300766;citation_doi=10.1101/gr.129684.111" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=A. Korte;citation_author=A. Farlow;citation_title=The advantages and limitations of trait analysis with GWAS: a review.;citation_pages=29;citation_volume=9;citation_year=2013;citation_pmid=23876160;citation_doi=10.1186/1746-4811-9-29" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Kvilekval;citation_author=D. Fedorov;citation_author=B. Obara;citation_author=A. Singh;citation_author=B. S. Manjunath;citation_title=Bisque: a platform for bioimage analysis and management.;citation_volume=26;citation_year=2010;citation_pmid=20031971;citation_doi=10.1093/bioinformatics/btp699" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C.-Q. Lai;citation_author=J. Leips;citation_author=W. Zou;citation_author=J. Roberts;citation_author=K. Wollenberg;citation_title=Speed-mapping quantitative trait loci using microarrays.;citation_pages=839-841;citation_volume=4;citation_year=2007;citation_issue=10;citation_pmid=17873888;citation_doi=10.1038/nmeth1084" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. Lamesch;citation_author=T. Z. Berardini;citation_author=D. Li;citation_author=D. Swarbreck;citation_author=C. Wilks;citation_title=The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools.;citation_volume=40;citation_year=2012;citation_pmid=22140109;citation_doi=10.1093/nar/gkr1090" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=E. S. Lander;citation_author=M. S. Waterman;citation_title=Genomic mapping by fingerprinting random clones: a mathematical analysis.;citation_pages=231-239;citation_volume=2;citation_year=1988;citation_issue=3;citation_pmid=3294162;citation_doi=10.1016/0888-7543(88)90007-9" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The Sequence Alignment/Map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_title=Statistical Genomics: Linkage, Mapping, and QTL Analysis;citation_author=B. H. Liu;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. F. Mackay;citation_author=E. A. Stone;citation_author=J. F. Ayroles;citation_title=The genetics of quantitative traits: challenges and prospects.;citation_pages=565-577;citation_volume=10;citation_year=2009;citation_issue=8;citation_pmid=19584810;citation_doi=10.1038/nrg2612" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=E. Magnani;citation_author=N. de Klein;citation_author=H. I. Nam;citation_author=J. G. Kim;citation_author=K. Pham;citation_title=A comprehensive analysis of microProteins reveals their potentially widespread mechanism of transcriptional regulation.;citation_volume=165;citation_year=2014;citation_pmid=24616380;citation_doi=10.1104/pp.114.235903" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=P. Magwene;citation_author=J. Willis;citation_author=J. Kelly;citation_title=The statistics of bulk segregant analysis using next generation sequencing.;citation_pages=e1002255-e1002255;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22072954;citation_doi=10.1371/journal.pcbi.1002255" />
<meta name="citation_reference" content="citation_title=Handbook of Biological Statistics;citation_author=J. H. McDonald;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=H. I. McKhann;citation_author=C. Camilleri;citation_author=A. Berard;citation_author=T. Bataillon;citation_author=J. L. David;citation_title=Nested core collections maximizing genetic diversity in Arabidopsis thaliana.;citation_pages=193-202;citation_volume=38;citation_year=2004;citation_issue=1;citation_pmid=15053772;citation_doi=10.1111/j.1365-313X.2004.02034.x" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. E. Melchinger;citation_author=H. F. Utz;citation_author=C. C. Schon;citation_title=QTL analyses of complex traits with cross validation, bootstrapping and other biometric methods.;citation_pages=1-11;citation_volume=137;citation_year=2004;citation_doi=10.1023/B:EUPH.0000040498.48379.68" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. Michelmore;citation_author=I. Paran;citation_author=R. Kesseli;citation_title=Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.;citation_pages=9828-9832;citation_volume=88;citation_year=1991;citation_issue=21;citation_pmid=1682921;citation_doi=10.1073/pnas.88.21.9828" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=C. R. Moore;citation_author=D. S. Gronwall;citation_author=N. D. Miller;citation_author=E. P. Spalding;citation_title=Mapping Quantitative Trait Loci Affecting Arabidopsis thaliana Seed Morphology Features Extracted Computationally From Images;citation_pages=109-118;citation_volume=3;citation_year=2013;citation_issue=1;citation_pmid=23316443;citation_doi=10.1534/g3.112.003806" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Ossowski;citation_author=K. Schneeberger;citation_author=J. I. Lucas-Lledo;citation_author=N. Warthmann;citation_author=R. M. Clark;citation_title=The Rate and Molecular Spectrum of Spontaneous Mutations in Arabidopsis thaliana;citation_pages=92-94;citation_volume=327;citation_year=2010;citation_issue=5961;citation_pmid=20044577;citation_doi=10.1126/science.1180677" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=T. M. Pais;citation_author=M. R. Foulquie-Moreno;citation_author=J. M. Thevelein;citation_title=QTL mapping by pooled-segregant whole-genome sequencing in yeast.;citation_pages=251-266;citation_volume=1152;citation_year=2014;citation_pmid=24744038;citation_doi=10.1007/978-1-4939-0563-8_15" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Peiffer;citation_author=M. C. Romay;citation_author=M. A. Gore;citation_author=S. A. Flint-Garcia;citation_author=Z. Zhang;citation_title=The genetic architecture of maize height.;citation_volume=196;citation_year=2014;citation_pmid=24514905;citation_doi=10.1534/genetics.113.159152" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Plomin;citation_author=C. M. Haworth;citation_author=O. S. Davis;citation_title=Common disorders are quantitative traits.;citation_pages=872-878;citation_volume=10;citation_year=2009;citation_issue=12;citation_pmid=19859063;citation_doi=10.1038/ni.1747" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=V. Quesada;citation_author=S. Garcia-Martinez;citation_author=P. Piqueras;citation_author=M. R. Ponce;citation_author=J. L. Micol;citation_title=Genetic architecture of NaCl tolerance in Arabidopsis.;citation_volume=130;citation_year=2002;citation_pmid=12376659;citation_doi=10.1104/pp.006536" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Ren;citation_author=Z. Zheng;citation_author=V. Chinnusamy;citation_author=J. Zhu;citation_author=X. Cui;citation_title=RAS1, a quantitative trait locus for salt tolerance and ABA sensitivity in Arabidopsis.;citation_volume=107;citation_year=2010;citation_pmid=20212128;citation_doi=10.1073/pnas.0910798107" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Environ.;citation_author=S. J. Roy;citation_author=W. Huang;citation_author=X. J. Wang;citation_author=A. Evrard;citation_author=S. M. Schmockel;citation_title=A novel protein kinase involved in Na(+) exclusion revealed from positional cloning.;citation_pages=553-568;citation_volume=36;citation_year=2013;citation_doi=10.1111/j.1365-3040.2012.02595.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. Rus;citation_author=I. Baxter;citation_author=B. Muthukumar;citation_author=J. Gustin;citation_author=B. Lahner;citation_title=Natural variants of AtHKT1 enhance Na+ accumulation in two wild populations of Arabidopsis.;citation_pages=e210-e210;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17140289;citation_doi=10.1371/journal.pgen.0020210" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=P. A. Salomé;citation_author=K. Bomblies;citation_author=J. Fitz;citation_author=R. A. Laitinen;citation_author=N. Warthmann;citation_title=The recombination landscape in Arabidopsis thaliana F2 populations.;citation_pages=447-455;citation_volume=108;citation_year=2012;citation_pmid=22072068;citation_doi=10.1038/hdy.2011.95" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=G. E. Schwarz;citation_title=Estimating the dimension of a model.;citation_pages=461-464;citation_volume=6;citation_year=1978;citation_doi=10.1214/aos/1176344136" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=D. Summerer;citation_title=Enabling technologies of genomic-scale sequence enrichment for targeted high-throughput sequencing.;citation_pages=363-368;citation_volume=94;citation_year=2009;citation_issue=6;citation_pmid=19720138;citation_doi=10.1016/j.ygeno.2009.08.012" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=A. Vallejo;citation_author=M. Yanovsky;citation_author=J. Botto;citation_title=Germination variation in Arabidopsis thaliana accessions under moderate osmotic and salt stresses.;citation_volume=106;citation_year=2010;citation_pmid=20861274;citation_doi=10.1093/aob/mcq179" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Z. Yang;citation_author=D. Huang;citation_author=W. Tang;citation_author=Y. Zheng;citation_author=K. Liang;citation_title=Mapping of quantitative trait loci underlying cold tolerance in rice seedlings via high-throughput sequencing of pooled extremes.;citation_pages=e68433;citation_volume=8;citation_year=2013;citation_pmid=23935868;citation_doi=10.1371/journal.pone.0068433" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="twitter:description" content="Complex traits such as crop performance and human diseases are controlled by multiple genetic loci, many of which have small effects and often go undetected by traditional quantitative trait locus (QTL) mapping. Recently, bulked segregant analysis with large F2 pools and genome-level markers (named extreme-QTL or X-QTL mapping) has been used to identify many QTL. To estimate parameters impacting QTL detection for X-QTL mapping, we simulated the effects of population size, marker density, and sequencing depth of markers on QTL detectability for traits with differing heritabilities. These simulations indicate that a high (&gt;90%) chance of detecting QTL with at least 5% effect requires 5000× sequencing depth for a trait with heritability of 0.4−0.7. For most eukaryotic organisms, whole-genome sequencing at this depth is not economically feasible. Therefore, we tested and confirmed the feasibility of applying deep sequencing of target-enriched markers for X-QTL mapping. We used two traits in Arabidopsis thaliana with different heritabilities: seed size (H2 = 0.61) and seedling greening in response to salt (H2 = 0.94). We used a modified G test to identify QTL regions and developed a model-based statistical framework to resolve individual peaks by incorporating recombination rates. Multiple QTL were identified for both traits, including previously undiscovered QTL. We call our method target-enriched X-QTL (TEX-QTL) mapping; this mapping approach is not limited by the genome size or the availability of recombinant inbred populations and should be applicable to many organisms and traits." />
<meta name="og-title" property="og:title" content="Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/1/67" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Complex traits such as crop performance and human diseases are controlled by multiple genetic loci, many of which have small effects and often go undetected by traditional quantitative trait locus (QTL) mapping. Recently, bulked segregant analysis with large F2 pools and genome-level markers (named extreme-QTL or X-QTL mapping) has been used to identify many QTL. To estimate parameters impacting QTL detection for X-QTL mapping, we simulated the effects of population size, marker density, and sequencing depth of markers on QTL detectability for traits with differing heritabilities. These simulations indicate that a high (&gt;90%) chance of detecting QTL with at least 5% effect requires 5000× sequencing depth for a trait with heritability of 0.4−0.7. For most eukaryotic organisms, whole-genome sequencing at this depth is not economically feasible. Therefore, we tested and confirmed the feasibility of applying deep sequencing of target-enriched markers for X-QTL mapping. We used two traits in Arabidopsis thaliana with different heritabilities: seed size (H2 = 0.61) and seedling greening in response to salt (H2 = 0.94). We used a modified G test to identify QTL regions and developed a model-based statistical framework to resolve individual peaks by incorporating recombination rates. Multiple QTL were identified for both traits, including previously undiscovered QTL. We call our method target-enriched X-QTL (TEX-QTL) mapping; this mapping approach is not limited by the genome size or the availability of recombinant inbred populations and should be applicable to many organisms and traits." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
    <title>Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"nk8jO-zEJcdjl4YZ5t1mPtoS0zGQ2limkbx_HqEgFIw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460248","apath":"\/ggg\/6\/1\/67.atom","pisa":"ggg;6\/1\/67","ac":{"\/ggg\/6\/1\/67.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/1\/67.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/1\/67"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/1\/67"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/1\/67":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HWAcHtVfXtOzhW1WO663N2HaXjIXAM8mvNT3Esc2pChxf9\\\\\\\/qgZ9NfBaFkRs\\\\\\\/71sF93Yfw6tBWLzY21Me69v0oN0179fDBMAMTa0\\\\\\\/40ms8oJu3EetBCW3woWyZdogrvwaLfjoSQvAtqoELcm7JL6ja8nkHTLVLsfLZ1jAz7GOMstaCjcpYHdD7KUt\\\\\\\/8TIkQjxguJlg0ww\\\\\\\/FuFtLvhGUVnK\\\\\\\/+q47i+9JGGviGMf1cBiixIdD+8ex2UeP1NKE9cAG7Dgxsyp2Zod4KcT1MDfNK7JrnDqd\\\\\\\/IGyplzo88xxgWV5M2KbK9v0akcZlGigGTb\\\\\\\/iQCd5PFJx4CyyFi8yzGLB8C37naIfG3GK7pFHFRMELI6IzAM6GgRX2\\\\\\\/k3bPBMNbr+lv7OfkogCNY9YhubnF1BG6EtjxhksRq1rib7rMKOl19LuobrPFhHxb14T8KLXkBdLCpHe7+yKyUT245obYaL6x16wotE7iUvwIUBZGfBVG8xqv8+ZMIgLMLn+NPuAy7J64nnWgiuhhP5\\\\\\\/musWCo2yqFU\\\\\\\/wb4e6bg30Uu\\\\\\\/NZCU3jk+zGAI\\\\\\\/MNEYjMJy4OwxY4jkfNAUSuRIlNjshtBy7M14NVFUh3zHMQxAiZoS93G4ZGvfNvSuVGorTUJt\\\\\\\/iOXZfKVSTml3LKn\\\\\\\/4CAXHwpC41THWKTeV5wtM7USDRc7OsHGz2JOjPKN5RJTlo+1ZLDnjPNKldldls9RPWwtJD5OFY4XVq+3Q4V+fLcvb0rZKG57EehMxxJLBxHZlXvdFQK397SwK0yIyDjykMXnSN\\\\\\\/Dh0kivSEh6ECFb7wqniz3Xpzr+we2N6urRVpF0mfBt+yO8vKSz7U1cKFO2thDIvAy9dgicG\\\\\\\/S6edrXXnKmCwM+0s8TCZobZ85ZyyNY84ZNj1qiE4QYO1YIdtDdtZ2yAY6Y9kUA68Ehen2C3UwFs3hHjV7xXOKN63NG+p3I06uTvroFQA2QxULln1KDAXm2Ctvu3SXMXNPyOu+xihXl9VLOIfSsMTBi+4V5upCo4a5Y9wpNomXstHy5HOL+spJ2hpdJDvtDNrq52rlXfgH0enoNCc2IN7PFA1mO8XbnrWEu9kiDwic7G0IuxY1LBq3XfzcniwEVgvNwtsc\\\\\\\/WQfpiaVrYN4rP1PYhEwb9sWQqfF05UC5oQz3SiBvWiD7LWyvkFwPhMidynJTKUORbH9uiK\\\\\\\/pjc8hs\\\\\\\/CKHXk\\\\\\\/FyiKwkh8D\\\\\\\/OLGhwgZH2Ufk8DIyJH+1wcwEXdxSJsPqK2coXZRO3tyC55eOPEYZe66kVYpG2EU6Vwp\\\u0022,\\\u0022iv\\\u0022:\\\u0022TY94Fxib6z4z7G9hczMhxw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ba600b187f083e258b2dc7a280adda24a16ff00533c1d9edcd86c78e50fb0413\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hYCBqZbIC25xSe3Cr9txZ1ck2g9ySSzVnJEZa6fJZ6cvRWIxDJjbzKL0cFG9B0zc03cN+Rs3NVaySljPXoB90ZyMJhbiuVnVEQSyVXXmoiOc9icGUgicC4\\\\\\\/j3aaTU6Segf4RHn82UZXHRXL19kX0X3Wi+47Y4+ku9UpYMXXN3tU4ujd2KtR7B974bqCg+eh6BnYLW43viTvlfvRRKpHHnokepz6EjAPTBXOxonpfZoAPX6B2cKcKFOW3X\\\\\\\/s9vrAaP+lPSn1pj+pL0lzdD5s3d5D5b+weJRuMZZQc+vdx+FGRDJMgDhIBrdfwmtbjo2Uj8AGDveOb5HQCWikvxNBV+TOOAF2g\\\\\\\/l+0PxKskoQJGWeM4iDo1nFiFN7w3P8+9Ty\\\\\\\/XwFSGBVllViy9MDZgK5d81pd26Myq+KzaBjWbwqmsZqZiik3QVDVAxbJOBdf1x5KnYHiytAEBxjzAu5FS5g\\\\\\\/fVqQpI5Y2zPSownbcO5agF6A6L0tFipu3II7HkRPjAbvmk9oNGt\\\\\\\/JlDZnw0baMzhwOXCteqTxQWdnepsPqBoNFil4WzWDXGGweZn4qI6nkIzTLOSVCw9jyKrMNT5N0Tlan8CKJIQ2xw76uV3gedYfRp9uXv5fQzZTAI1WaoeJq9h0jhOGxZaBX8l7lpGP2Je6fxq4kz058dO525U73dK0Y0TCSgVUH6bVPPer3ZTt4+F82OzItBKSvbm6SYks+2Hpbb7WNo8KU+bq2M9MBqKAqi4nT2l\\\\\\\/Obn7KmZULQCPF11fYrSR679BUyO3tk282hlnh1sioGR8Q5qailkBZYcwLF6XEtUJb\\\\\\\/4XeU9Wh347o9W9JwW4rN\\\\\\\/tBRxsmfsmhmYO0FER4B\\\\\\\/L8+GCKiUiRMWloygCvfeoK5hR2r5D\\\\\\\/Wsg1mZxIm2UgIkKJj5WERh8q3sFuW3olPvTm9+IDE+12Eux1ww9NFXNXpg4K7sp8BEE6J6IMOIw6XAVkNBU7szaxgfsxlznYi2LY1kBPK9kWqZtibU2yMuIHvC1WCQRQab3sQF0OCfOfx\\\\\\\/vZUqE+3JNZhiR5DveDPaAsfhTm7qqvL9h8OZBCKczTHJpmZIhLw2l7PZ9dN0Q4mb0b+KkiTf9y3YU+Gw18p\\\\\\\/CzKKCZqYjLcxV8WUAGUMVnKpKzk0FKUnwT6Hh+xQFzBjwhOVBAlzmu8AchPTpIayVLZSO+HLPNu6FC3MLcLi7D\\\\\\\/Q8vm\\\\\\\/9\\\\\\\/nIgV+nAn2i4\\\\\\\/fSae1eaZnP+uwe5hQwQRdESeC1DBGctmg35\\\\\\\/C8q0QF4ZUTr1iM2fE+R+tE\\\\\\\/6k\\\\\\\/tAd3xv1vN0YeqbImMl1GYGuFK0hVWpvT8xsyb7SBCu+moW5a+nQXj5eD7gknPJKyg10tPGvVwZjX2yktK9xV9mJvRINrkFau937KMmK37eNE60TNozwLDZNyJ3weR+TDTcxZbfWbx2K6UQivFeA3q3p3zvkBVR7KLaF\\\\\\\/Z5Q2onbT5cTKAgnQZXZrftgzmEKwRrTrPiVo9ptaDIebBWcZDOY5y6VjfVhZvt\\\\\\\/QISK3FwhoaCBkaRxduU0VQJfuDed\\\\\\\/GoiqKwByC1jxzoLkh+t4HRk5EGjJBgeHszvk2AHwROzRg3yX9jh0uUqYduIaXgbAVumPWE5CSQQfpwY6+QcY4R4T9cvJTey85PyI1Vg42IJd+o6EQHgk84zm3355F3vCSPoPXte83KqaTqWkvBSUqOPXBBN9QiWEPtlV4GWurzPadDwaHHB1EUF+EpQpeAFZ2\\\\\\\/KJlOiZf9XGlaglEupCTiMzTcriJeNRW8Ym6r2eVzEalS19ChVGyu9mYmdwLvV8f0xq2i8eQc2D2WyiTnrTh0fu7pMKRYDA66\\\\\\\/eb0+Yj+PnsqyY16h67gELvmhpwfBF1AAvOsymjzNS\\\\\\\/xsPbiiJ28TLhmdk2LLv\\\\\\\/fsl7iBNXNqZDYXUeOfPwDuAKA5nFcET2KRC91AzebMkOsvhIB7hnX6\\\\\\\/NBX1uAQwCdLwId3\\\\\\\/pOMa2ilhGuGJzOY3Ag6I3MawFzT8FPVoGr81727yePjuEQap5z5zDqAcMnbKM8djzP5yuqHPs2tJ7o4pT\\\\\\\/zgjn5uzuXx0Ab7nbUlKm3mRYzIDH6kKmACNujp6NDp3OoeBR7p0nqZ9J9mNG5dozuYw+a1pkfFUIl5xCI7SJkp8HGaM6Xr1mv\\\\\\\/XulDtsczF2bdo0NxmX2y7h3pg9jY0BcYVU4g3jrLDX2o0VZY\\\\\\\/4HmdvUaANZMkiaJ7pnnguC\\\\\\\/CA3OR6yvnjsE2bonI5t6PhY0yEF6tu4L27JL+lWDvH7kwr1QD8YajYvbbNdUwRGSM0RaImMC1iu6HSTD4pPwS98OnDR\\\\\\\/BgNm4X8ksbs8A5X2NPKrajgHE\\\\\\\/3jGmzJBns52FBT5OM69Y1TaCDNMBY2+zvfW5qZY1UxOgDM34dAhAhc\\\\\\\/Cg6XG3G\\\\\\\/JB0ZVtv8WLN\\\\\\\/JjTJl1mshT\\\\\\\/aIYEHteaTEsHvEXIGusyCak12qJxEg1bXWqbkI52EAGBNDlaCpezG49\\\\\\\/aP3\\\\\\\/YJ\\\\\\\/30yJGQ3m9Ww43MX3P6lRqOhMa2v9UD086AhVAaToFWsVrPYJy1Ep2aK\\\\\\\/pAXIgStZgh4f3VUZq3qcF2ZVVn79DHMcb5Yepk8McPL\\\\\\\/gqDaSRWDcUSszJk2r\\\\\\\/aNrOLSuP5YsTb8Qrt7NUKxbdGDp5r49A8vpXgmt7ia2XHZNsxbxRC8RN+Hk11T6+DrJPVYVsg\\\\\\\/jjErEM9ac1bBoaDTbkTUQPOiACGmoo55aqqIj6KWLG4YvsCZwEVoY1XemBIzNPFzo2P4lA2kvTL8gOzMfzfbH2PMr9SzkWBP3xOkK1TX0v23js4B9xkKyTmG7ddyrgLRUmnPyewxGdNsnq\\\\\\\/yqslxOFnVdLCDS05zecib0vQuL\\\\\\\/n43hFytOekBHKO8\\\\\\\/q5\\\\\\\/6yLGALc3G9A\\\\\\\/dA4Y285VTwt5s74C28h7P0mAmewroEwWECNzxEza5R1qfRwz9aBntUwdGp+46Q8p6Zhp8hoSORCN616XFfKGMPXdAVfCNlgkUYDeR94SSghdXasGPIwP4zz7k35Se3RZx0796bWBrflI558UEjqTrMbOq+fkAinvipCmpLIlBE0leDr\\\\\\\/ZwFKeaDaneLSpSSH5sAR\\\\\\\/8gKFR6HTqvD2HX8jdTPl6mHZaP9z8nU9r+RveKJED3NpZNiLfF9mCGBFT2qSvyybOGt2f5RhsBzZ+a1Q+t7S1ezj1EpLeJ87KAyYiLrnhXzWnl0v4WIZpK7CyPJ\\\\\\\/EhnCjd2mvVQsCKbAj8GdzssuV9X3iKql4Be6qSilcEYUpiS+0Qi+FV50DfgJG+OhicA6+f4RocepBLeA5P06ZHqbyfSP2fL16Tv9VIsDvgWfQTBHC3VWmd3WmWnYp1HJvAUiRBCp4uQKVrNfjjC4RQXpOkg\\\\\\\/yrG\\\\\\\/laZ906nv6DRtiLSfVNRHVgjdzEvOXgfL9JvEiTG9pIyo9s1tzvfGpeR1J7c1qQLjOES9S0LiXLqqPi\\\\\\\/guDTDpVDzw9dtCXonmmOWeQtF8ONSwh7S36di7rku+yw7oE+Z0hpag5bE1mxT8bNCgSdsC1mSrCkBa+cF0AuJvBFRtO6yFhNU3Bws\\\\\\\/Kk5JF0Xpghl2pL63+zdm5ph56We4D15ohT0UdbjaFLXXpFGgDZP3NHtKxuIEvNF3mEPz3o1pnO6S9HdM4EYsXpfKizkWhEIq5Wq7L7Y9uN\\\\\\\/sILlmHqCDTEWoyLlAbWmUVvKps6UZM+t\\\\\\\/SX3R57KVCaaw9PhcNXizrxatYzaXxvxQhUhezh7+MgqwGWhkr1NWZ0sqfA7sj4EIueR3o2sww9lECnJzRTnumdF3Vc+qZJJjguTlKDBLiWYKV8BEy5gi8WFlkreCNbB0zLCoZ7GpoNH4To0XyPmEJWdoiGVdEvFrBplH7OFAk8PwGyGaNRHvwka0nR3YM3vK9\\\\\\\/kp7LOYpNiNI+sUgnRXcsSP3lj8UcHV\\\\\\\/kjOBeOGScEovDx1JolT015HzpzdvXB69VyfqoPtogHpaEQ9N99MAHUfRWBuUzC\\\\\\\/1B9882jaASzZKvwKq0cNgEtRbkiRh8LQdoYkttw\\\\\\\/TzNxR0oOkQ8A5MnSUn90fpKuAlJwPH1MTdmikRNoGdWvrz84P68TjT\\\\\\\/IhgZzu0nra42jCTxKZSJz3ddsgxZHBBzIcEVf6LR\\\\\\\/C\\\\\\\/9oai+GBqMHAoqE+lsxPK+g3MXaSasEug2YL1nviWHmCmUInWGdwgj6C3olmCedkcGKvqNBlDw2XOZ7fMDSREc7pHv0hbWBY5jHuMo6AaY1NdLi4Z\\\\\\\/6R7RJ7i9FyA1TSwbHs7hxcCjw8YTL13eHT8bqxhx3\\\\\\\/548lsYfEdCwEsejVKMQFhDKfMRYiVJYLK1yjkXfkSrBWBzFtPqgd7tq4IM+fATGhcAsl\\\\\\\/Kg1MxW7nwpXCDsl56pnoxdADlnzZl94aB8ozZDOObVyNfclZEHugHbnLiy4KmhYbrx4OpIFM8NXFYd5ET2Xnd2GPAtblZ1N7IdWX3tZ9me1mzrUK\\\\\\\/Ri2xqUixcORVHrzdF7oKVMnSz5aJKJwvSnSnhNUt2jQhFlNhQU39egZ\\\\\\\/vI51hWa1uRpxf\\\\\\\/QxgUfHQi53ssgDTagPBS3NbIC3NNuWR0WoPxpZFS6kwakYygR8kZak6DkF16lq8TSuRRuHDlM\\\\\\\/Ak6xfguGKIzGAQwrfsrd0E+inSJIoAIbfKsbkpRO3vxM9nz1J7g\\\\\\\/aLj9nAkT22SDoFQX5+I0ZFp7g\\\\\\\/wgDuB8cWxEtXb89oYn3ypyKPjSk5QMFUpPTOXkA50TEmmQf+OyIQDTONvqEWU\\\\\\\/Y+xlD2PuiuEopXmwusGud0bxNkJHwbX5Q1JUDX9iWSvhSIQnbQk+ZkDyxRc8NaC3I2BxYyRkMTxUdsfFCYhCoEVZAjO+6bWB2jMuygPVVVjgBLGVGzOFAXd1qXPGasOjF6rKShEXDlFQI7cB6bHTX2HgW0X1vPuVeP35gIE\\\\\\\/3\\\\\\\/eofV2UviarMmt8uZSDTZNpWdmJze5QZim6knpGe+IEqg12U5wqtRNqWcQFCjakms4h\\\\\\\/25ZKqiAaliJR5JcFVe4dYVxt74Y7aeVQNxe3gpWnUhDW1zbd2A2L\\\\\\\/K2TKS6\\\\\\\/1ayA7KJt7uvA3cxlbTazMCBprLRwN6M\\\\\\\/rOV+IC0qgyrZGWpZSdOeU5XHnBnC8qPbBtDVssW80cIIWgYAoQy4MKX67dQ6TomMBBMEePrU9Za2uvzq1+OJS+PF8ZuHIdkGz468TxiCxiDU24Jk1bJH3VOaNWxoAucbjo4Rr0O3y628dfZzCI2ZFruk\\\\\\\/oOTXST4b9x9BRmGhJCcqkyjJP1HK8NUh7qQr8IP7rH\\\\\\\/dD6I4k5gmPoKtuh0n9f\\\\\\\/MLJ00GgrusQMc627D7yF0HLXbmJ3RbO9xkz9m0jk426RpA81Mv9CnQlRLkELU+RkTRrKN8iitae6HvqMX6sEPm2IsV\\\\\\\/ArXkRxAn7INwgxH1UpueeuZYFrRxyXv8ydaA6dyo+Lff7qrKRKCkY674BeNWAKmRDhv9UypTUdDdAD+fIODgPsPprxBu+t+Rta\\\\\\\/7lcIf3Cfi5nF2P0yPbiC1Iyy5WD+kroa+OSpcAuNoa7qs80tacQECypCUsCh3zVKEYL\\\\\\\/y5h\\\\\\\/JRNKzcn3u9UeT8ySBCwL7rcbpiHTEynE2zNgOxcYD9R9rXHJMEM5Ba2Ayl1YzowfVtGWHB7MQ0Ld63XX+NDvPvzFVxATCLPa57n0BVm0XAmpBxkUbCOU0sAd6CMUSwBnB5wqs6hwIQBhoFYHNSmgv48kJ9xRSnU1Idw7lCBdDOkY9IUYMCYL8MkSr1Z9bNSXy2\\\\\\\/\\\\\\\/S1OharjbHlWb13P\\\\\\\/2lA2o7kRpfUeJtjlfIe0aVZUfQkOxcaItFKOx8XPKe\\\\\\\/qrPL0hgp1o0NeFSmzyndCgSEOhEH71F\\\\\\\/X\\\\\\\/6xFtz1+Mz8jQTdqQuJlD8WGfyBbnF1rFtC50y2J\\\\\\\/qjF7aB8OUCOhTGKA6Xy2ZJPPd27YvTTq6rLoh7eJTW28lW0j+oVb3fpbXy1q3C1VV0WUoY8vJ8YV0QWQFVC4sTUhwEKLjDmL8SGF38Fo4SwInXEYtmIzKgac8iALLy+ZFxX3qp8fujwHAZOpTfbXuDxbsLyZN4ZdfKAfTXWR90f4NnopzrcWxisaRylEZ+1FCJUlW\\\\\\\/+5qft0Q22+JxPcVeGRDypJhhtnvTuCQu8R9Lnc6heaXCm88F9r0LSKrXUSaT83A3MiroUpF8RVmx8MwfPSshcvV3j3Fyoy2hTVlIwfz8KX+cy7pcz2acLUtLlYCf6yGggZXdP2bqj7PdZi2W6PcKjOxurURz+ErutBQXiAqJIMfG1VuaIl8E3KZ9pcjdl2fvjjCqkX+GqLBuVJ7c3yRyO75ymEo\\\\\\\/CNl\\\\\\\/AUZ7SeBfCYTXPAZfyLTOwLhr7FxXgFvU9J4T\\\\\\\/6V0z2Nl6BUGogn3JKUGPMYqy69PDa8Z7GHOD2fmik\\\\\\\/Wp5D4dy7M8tSSSy2FxU4D5oGsSorLd5ReR7otZBiO5JTKQO95ei+f0J83f7SgLXwik9IztqZOrc8JxkxvWxgkHwoaolxw0STKLwTVw7lJxd43P9qQMdIaLhA+yoIcHDM+3w1ZpN3IDFTv2QLVPNU5kacWV\\\\\\\/JWS4BKXL\\\\\\\/Zm75qbbCDZJ2pl4gFsEd6y0if7\\\\\\\/ZwNChB7kdcr1+hj4qhytlid3xrQo+Gk0JUTCqvA4P2hWCxJryYUSRayqNyrthwYFZVZgO8kRhOL4KgUoHHiblKuEXZ6DLXPjcwZpB9OqNyVENzC7qWHt0oOKTTlRSudnC8SR\\\\\\\/+gsYg6nf01w+xaXuWDZkiHzL0SX3QgvlyEuNrCt2f47bZQ\\\\\\\/PNHC+dk2G\\\\\\\/suWcJugNam42RJOosEa4cqiyYzeAmfVqCwyBDoxPBAf+XHwZIyHVL89nqGK1k8mtggKH6GfnKVCDK7uIykuYJk6oedOOTwdq++yPpOyCZ2NuL+Y2ejJnU2Yx3Utwu3zhXWvvtRyYA+1ZPsObNCzcK95\\\\\\\/dHPMe+Be+CrEDTJ2uF2fQbNzeYZGtGM5Ub5kUTdzMXGoWuyk1lNQKvZMPHMGb8PjuSkZKnmyx69K6jxdNdy1xGitU\\\\\\\/Ty+UAKN1SxkPuD1xxbvJGX0IlZEPo\\\\\\\/7zzK8FN7onGA\\\\\\\/wcdPz2+M2vmTB7Bg5Al4HgGUw\\\\\\\/1qgGIxmkuZ8OojDqTCflvW\\\\\\\/Xtyg3r6gBg+fdZhGobCkwkasZhSCFv9004r9bp3wSMIPTezzrPdG8NG2A+d32rxs5v1SqGJExFCmOxdXgohmoRmBj8f6hACg8wuaaqPFZL8cdzNrRpAzmMw9vyNZ40BEPMeEaIEDIbS\\\\\\\/7LHg2EXIM+q\\\\\\\/4FoHCJXSATu0y6oFxf\\\\\\\/myly78qqnQGmphU8\\\\\\\/F7Mvs2E4KVq0EPOVNOZWmHr8bo4vZ\\\\\\\/arqSNgwWFL\\\\\\\/MKz7Rr37BfBpjT08cN0HleLkI\\\\\\\/K95WeRumtleu4nXpyGTotnoyZ3WtsBxihA8TvULFnH\\\\\\\/UTnzKh6GoO60RZHz+s7qpe8ROIwH14Fk9imHwR1VKPeu69edvYDOfd8EpfqCNaso9ondM9ps9VgxVEKzMrONtiztxQ3x4lun\\\\\\\/nBjNigzLP6Ehpcr5VI5uG51bzSwQyh\\\\\\\/aqb7hzFB3HKAaNlpbdKwvIm6Hn+JILjYwx6Rjzm4E43q3Lc\\\\\\\/XRjHvFnW4DpuXMfp6dnjKYnAbuEMBEM6JnHwr+mnqsdAtTrz0zOrp7TOl5g7k3wLzkXE7s0G4W0QukPjdFu8N3Bq14Ub8RHsT4yf+eUQ8l9wLOLm5Qx6\\\\\\\/TR8+YOTP24sUSuWummiYOZP9CaE3aems+o3V+i30rsQ5FCPec30V04KyMqlt8PelymVDwD+BGgG2eawBI7pkPu0OBxnLiI+lt4OB9zD0OGY0XHL6E1t8mof3ZIqMXwMDf10IcSD9vDtimt+9Bl+yfm+rMv1xHgDFscXG6woMTokd2mKbLBFULHcyPMTB5ds2QbdBKGTY1NtccYw5ZXdcI32GdpGKUYJIJkQINd4kNXeXmOCUXmhCnIwBThinzbwGiA13\\\\\\\/F8IL8367Nfndtbh8fWaguVzFaz81MsMZ0lUiBWDl9yN4cRaoF9otSfY\\\\\\\/frjV4UOlO5IHCGjPrtCE7XzbmXXdEwnqWwUjr9WwOq3HK95PEmzpQaZfO8f4Eq24edRHVQCyRACjDROJ5XchMydxt9Jg\\\\\\\/z3PYSscLb8VzcgLStZ2oMrr7LSvuSLequUfdpwl5hUHCBEqDHASw001c5pInsiaKBo8F+j+tvLLR\\\\\\\/qvXVm1tJmLIJ2Q4GncfK+ht3J4vYhitb+bg7gSLI8271x7kz14QoRcWxh1NMLBvJ2GXmMiZiCR5CPjZFnZkSDQAE3MgkxvWFPGmu4R4RL3Q+3TaLTrY6rw1MlJs5Vo1u7lbluXO2lbSGwKjhMcHC6rj3teUBM3pgXKQCMUKGEX5H22Nx\\\\\\\/sDyqcMOlcKs1NSbkjOCrSrE2F7rs59AkgGuUPVBYj8B+pQOX9eoaNNqKyv6B+\\\\\\\/a6kq+SSoG+G4rdg7X7o44tEI9KFegaiai20IWsHG34XikcYp4bJ29CF6BW+vGGFHpu0ePuzsHXejS8KaDeHbo4mrZr8sl3h8WolZBatoqg0dgJnnpZeXSgY+GwZxc1AYHfUmhhSwMEXAyDt2NpQ9fbFNh2oFR+3hmDS+UCvox\\\\\\\/gQAyCXc44ITQ301ZlrNSV33w3n0H0kZjmVxgXLXiF81x52THbMdfIn9eFMMztcpsT+wSzCVr3sKb4YMyKydC3YCtYpByhA8oZ8iwVpdK\\\\\\\/\\\\\\\/vMYmrYSYVvg4\\\\\\\/z2hB4W7YJvnfjMFizT6ZlMAejTx\\\\\\\/I7R+pF6Qe5Z9Gc9f1kXY8HxZPM\\\\\\\/iusS8oTF6ps5mgDrVLZC7xHK7ABf8Pce6i2GPyJAuGe5eQDpEuuO9zKozXXoj92DxV0jRt7s08EJUOOztLnLIlpCW+UrwD\\\\\\\/PcXljb1lGVwbqNetNXl3+9pHSgYJA4h3EykH2iFVj8khvwmMBbUuaY\\\\\\\/9phU0WfmOS2PHEMLiYOJwBEAy\\\\\\\/nGXKrvQxG2zzo7jjLSrsnCGRTJ4jOcDyrdj4NreFX2HrWruCl9ivlargcYCqvVEvcGp1Y+x1myu51stnmZI8fSFZss+yFHuvQsuVAPOiaptFe5TaN5DafQ13+Avq9ypG8Wdx7sae4IpsvFEU0\\\\\\\/zIcJVxNRmsmQlW4C2ReVT1Ngrb4JDHfBqdns7n8zFyA3CqYaRxsZJ5gFoab8Js36bCCt3hywe1t3ej1qEJI4OidF5waV56t\\\\\\\/UYv0ximcmwyVJgwuLdhwxR\\\\\\\/PWzlA8ENbN575v2Nv\\\\\\\/6JaXlsJcYvpTKQS5TmZ5oniu6qc0rNnMrfAeCT2WUwAJN7CrTDX0l3z4zVrpIPI2LRGrEsB\\\\\\\/VadfCkHZ3S1hNHpFgyQMAx+HEy6u0LLIeq1l8UDDKj+D2oMgFHD9zGglFwHHLqeGwedtDm6uvXYho5x2HdVZDvakSmIc8Frz4N2uPTrZICjEQsE93AWesb+eDgALxi58Zc0cMNNUGRxLzgQAlifnI\\\\\\\/M3Yxkw3eyYVqhTYJnLD3oypldeldrPlLQU0yXY1vHM4DZIJzL0zmrUV9la6fDDqSMNH6WD41sfnUeCtqivdn8FPBIpnQCmmj1py0qBklTVqTstjNz1cmX9ULcZ1l4r92empkfhrwbRdlHkeMM154Oe9l8q3VftlMxC90Ekhqcbj8SDxdIza9VnBPPZ3BZZczs3WUofOU4cjnA14Wv+lWHlEslyy7FZ+RNjkBXX9q7OGzzMriBtRdkhHy5c\\\\\\\/HVr9iVebtcoXk4A3730lGZEr03fNTRlZkycsI2vXdErEB215QULe9XOoPY8UNwQuFb816a1qlAbgK1aEo3eJI6JJUlNOqH0k3PxFmFhJJGwhJdiXp8+InywfxczuJm0\\\\\\\/su79bSeiD97s6WaT\\\\\\\/9Z9h8CzYB\\\\\\\/zKx1H2FuYQBjtOI2SKRvsvMojuH8yYDIXmdBm2cjzbRo022KamfIA5hDepa9hqoYnmJMQ4uvad2aTaUgP6qq32Kkzifxv6jVcxZz6JgoGQGObkFEhUykjSnCW\\\\\\\/85XRIlatb+5JC1H3K7YxrjixM79Y21HkaNBaLjWXPkQ6DVzLoaxEoJt7\\\\\\\/UKp2NZcCDPp0GhwL+fqVqhhqXd2cPpcVOwQOlggSNletUc3+8+6Fk9Z\\\\\\\/vTrjM1u3ytcMUllgzXjOsqRR8C60L8EFCKB8nawf22KNspil6n8UPemHRRnYwlat3u9pWNwFHRp2t+Nspn7ye3JY3vy6aTiaV0irXC0yWmURximMEfBHWr7cfM9+sliRUjvSLomkNg2QVWbSr0xGZDGeZswrrTS4IfNamHQv\\\\\\\/WeMmEyEe\\\\\\\/wgibX1\\\\\\\/ShTzJgjwVu8vbxbTKFuZkbUl4k0penA0rFFv8rkzLdbA+t9aesqsw4BStlaIbwtDUsRi7Kffkc3cbEhb+AB+bbiKA5l4HIPq9w+ibdYjz23TOZnANDZhETIVXCWmweBGLkPMFvzOpeC3auycm6Me7ILYISgccMSIdya9e2fQo\\\\\\\/zCC8q2Gz0gtFtAx1qUR6aegrRoBDFWywUyu1cIYfwTPBs15+3gkLj53RN4C\\\\\\\/dNLI3jLGeGWIMgVlwvovKbWAvBBa6xA2HFcM6xRburknUT\\\\\\\/P9aeHQ7c+curfScp6GR4Qg5bjBekRuyq\\\\\\\/81SWbVy+6IhF3hycRJo+7S+b8oTVmP2w7mopPLj7TridgfPLHPy3GEwYHzRJ9gXHuUWOZnVrG\\\\\\\/ZE7f+4JKpj1\\\\\\\/piBEblBrPcTc84qXekZOUhce5LBX3R1KRuXTjVEtSQ0ayXtGJEiRfKjCtYXUynD7ZasXW1DGWbkN0W3z05VerxBre3ma4HQJ9ZLJA5lNHg7bNRt3sn1JE5I0B1dA6ybkT03pLO1rjCQvHg088x1tqnvGiylHUzrRLpLYSk5mUqAQyOw\\\\\\\/dBy\\\\\\\/7LDnLYfLUG9VoDkKfkSfyPYvq25B0vWmdQe\\\\\\\/0z8MGeoASgOnuumZKftUHtM\\\\\\\/fUK+u8VeQ1LroU1KdISY6RzlBdRqHlJEs\\\\\\\/iihAs1qlxCrTN57EdU69L0jM6rw9UrQZscxLsRu6iZgzUzZrtUdjETZFDPW\\\\\\\/8AmwcyPJlDjGRFPnIPYK6i4pmm9pdit5vimjsSEXJ4Pd7GkSXCY1LO9ppEDCKVMLY0gdksLrNj8KEVnQGJ6qoEerTdyosjshQ8YklX16jlddWbN90+8jkFID\\\\\\\/sSE2GUzBgwA\\\\\\\/olcN4306D0ZpTL5pymlrvcMJFl5H0m7kZed8idMqYR6Mg29Qfh\\\\\\\/2KaRRIwMJUqmMm\\\\\\\/GCRzbo8pBuUiVQqnDg\\\\\\\/Vf1n8JpcZwJ\\\\\\\/Szxz27A6SZMQur+PDj2Ba5NALusanJJhSqe0iyl+2MZoRp2GC0AWQ37\\\\\\\/7SJ\\\\\\\/2f3Iq\\\\\\\/DDZdY2\\\\\\\/5wGbUMl4cT5JBJH2tvJXXE+v4\\\\\\\/5mhPN\\\\\\\/jtgsGwoRc9LugB6upkxWwT7UJtCOa8ioi6feNrlwfY3DhT7ZHdxcPQqZQtOug\\\\\\\/4lDl+I1MefR741VoOREPlE4NhEnggsXaUZ9DlEY6T845h+ME10xKehZEmw6LRrNSnoMTd5ibVx5hGqw3L9xbGa74NVHqtZ6f7DOHJcVLR4DQHeHu7PzR\\\\\\\/P0ZZysbh6NEn19MB1tGVjR5WQSclIEHStwXVcd68eoYM0XXMbMvc0ysV0CdXgrbBk1eKbyywMRy7jSh0XZeFuDr6D1RBn49NL24ozm\\\\\\\/70HzUZPeDJkqKf26DSkW3\\\\\\\/gXbsoKmEpF4b+E4Jdo1lDxGhuC6ptosyerAKbPacfsjxyj3XtW+W6l80n\\\\\\\/dS3Mam5b7CAgC4YwWtsQu79VKnzFSy0V4F1c86HE7UMx5hxm2gDdK79h4KcsPEEsPuG1td\\\\\\\/Iox1DpRpeuPgQYNFGXertyUWdUNNEhoefhRylQswMsoB2Wb3l0pQPgoRyuBcCE1GZWXGngsgXyrYt9EjXeT1\\\\\\\/6oSsLYG2CyJcS35uTqexpo+NCfykoE4AOGjj6ddhpuUdlyV8Loda7vegP3Tew4K\\\\\\\/uKMBWJjI7Ilcn8APOhQVOlKuxUc5s1A1l5zUEwd+zw5\\\\\\\/9RdYcONwJxLXuh3g6vp63iuJKIjXi4JA+80T9nzey\\\\\\\/\\\\\\\/xsKZiDNqDcWxnb6VosNSJa47sD6hbNN6EyJKc2TDFxesAMDBkitkK8O6LZI3QqntwUeaHjCYiwKZ8i60IvWwyPwKNiK1kYhEVL72SKfxtTAXyMXLCKRCGAW4uRVIF4OQM9rtSydeG\\\\\\\/UEzicSpN\\\\\\\/S7sufS2R+u0LoVrizpazyFcbNMSos8nYmulV1BDWbpJR25HKmf2ItocuV6xwBCrK5M3FLtrVjLg7i5h56pY\\\\\\\/obV\\\\\\\/GZtDWMr\\\\\\\/m2FMOItDo3ZYsapJVLbhIFSRQSfIheAfHjRmltzLrg4HwSXMOKuqhfnW0Px+TvGNsCJVhEyDWO4Hs80xH9nW8yaBCF6AaRL6kyaqiZaVaTRIiXOvPDbezXjqWSYpszNKFTs\\\\\\\/rAdMxd0ZjvXMBFuomm4k+atuHlF4avHOc\\\\\\\/9n00n1Yocdic6d3S7Usf8bLAasNlQSquehpUVVxnZwj9LUMxd\\\\\\\/vH8BedKNWs6wDOtXaawWZlTpf13wxj8HZNXwYlH68wLzCDEzGnAw76t9Mk89ZML1IGj0Hn5k67njaWI7dazBmodWwocVsGiM+B1U4MWhEnKiYdH+bFjCtIBOCQA\\\\\\\/SnAkYmybnsRzXsZ66vkeDOJAkQyJeAyTFXOMVow6kMn3OVnpJ\\\\\\\/I7YGGdRxiQCXB09774mqRWGe95wkT7jQmDO0Fzl0A8ijer9stTdzL52GHckrlPuQ1Fx12Qm9wl381k0rQ+VqvpD2wJMTmytP4XWAUbelMi0EePegJgmEDZurC7EGNEGhccDVj6hySmq27QTvpMWpxmMGKmvEcr1WbMwqpcKwAwhfVCU7AsShLPzusMgAaWRYq+8cJqn6cQkkTmy2I5jj3KY4OzUZtcOI9Wt1ueancRHRKlLtzHKgDZrs5OfRSAIaoMQmHpeDA1sIlL5+xiGH0wa4aehPf8W4XgEvErhDJc4TOAHNHu27QflLiVNmu\\\\\\\/LidxdPYtHrciGVfI5KRZBce41arghfjxijiHrzhnktYHJ1i+ROL3BGknMrLLCu0GtlFdYp6Dx2ttPHXImKjeVwjsgl4wnh6HhIL2iOY5eIgWCzEKZb1YYbiiSXhPCdtlpCsLQtxgznm7uae+QOQEX2HgKOh6wW4OBH2BC4WNpRgPcqexOltEY09P1bFS3ScALpMdrrSZ0HdKMMAnDC8BB535+Oozqjc5YXq3uT\\\\\\\/mdWZo6JfYj92JsFAhAM05OFxIO4bN\\\\\\\/jASo4VqUxb5CrdrbJWLuAxjmKam0q8n6TwyzOlW993ytI4bRRk6qr9YWRVA1cSW70Pam\\\\\\\/IWtE9i+kWJ7EaG6Lsnduq8DqntwdOCG+odE1lzK0onWhwN9mV9vOxko2W3eGCNR7r1zHR9zrwQojd2d\\\\\\\/CpD2xQ2lyoB01QMiCjTagsk82OtNc0vJYQ9IYoBAGKc+FWYvXPJPpV7eLgx94C7WoaqAikbuuL4PyNVaQ9X6uXkxGT11FPPU0crPmzrwr8zQtZC\\\\\\\/3xGC0am5yXcYTUay93FCy2Y0Y3aEKY5J5SfZa+cGYmNMk\\\\\\\/Lc\\\\\\\/E7opJH81m\\\\\\\/i+K9ToxnxJSeRBqvnyqlnRss7BiwjcB33711iqMnXHc1Q9s488yLqdjnQNZgUTFEc+NKHK+W6WCEISP8brN6xp6Sfoxc7UN0xmDe1gQXjpm3yyRlgKuyuh86tC4nqDW3qQCmOv9M3Ml+\\\\\\\/hf9YcVWcVuadXHf6hkWxOEMt3GTEprmKhuh4Q1Qm9UvBoOTCQsK1Qo7Tgyz4sLYs6pbHEBPLv9RG90PwMabVuu2RIMB5QPaDsyYK4inlFibh2cDCDnxW+CgGoCNDHVDeWhzqBvP83Jc8y9nM3QtIzMpSiLiDkPNtgtLdxwS+075Kb5nZwBHiPPrTY6wIcMovxcz9O\\\\\\\/x1Nq5h7mStPQZWF+8eztfa9\\\\\\\/8pUm6Pvn+sYE+391RplPvwaFvHR7DCb\\\\\\\/lES8Ai+ieuu\\\\\\\/6d3PKalwIKK2jxNZViidQ6Qdtg3DH8UGnbEzl4wA4hi\\\\\\\/80tt6uN26cBeR3ysjj7FlH06GtVW8mZdaZ6\\\\\\\/lwSX0aOFIJ1WWeYNtlVF8B2Q1kkRHZKp0l2PQxKiDgyyAGxHUsG1uxMtpjcK9yr3a636oZQCAod1AUXj0ymee71i3c1d0FhDDGU8iBoyZQxD5QuttbAy\\\\\\\/\\\\\\\/MC1\\\\\\\/MD3+EwW1Y5GAVMc9\\\\\\\/7DthzGhYrOzrD3Vd1FZJae3HRbvHQtIeuyfFxaCBEU9Wrxr8eWLs5bPwUARvUR0UbqI0DBs+6L9v1MDo+9vOnwRyRgJAt0C\\\\\\\/YfBoIfvy7ZuazMbY83icYe3j67e5DoDf5fgyr56rsBjyDhRnwUpLUBQvLAXeDAegjQeWDDE+DaO9Le1\\\\\\\/0iAYGsZoH8QgLrxzZ+iVH2Ci7XSzy8AmTFKgh0cTjQjMAIw5k2sX7wZIWiywV4mj22QiOHPxSaZA\\\\\\\/oCeP9\\\\\\\/6NVAiLCfY+tB19FZEoLOK3EhXNVchPX2VsZXF66FZI15zyyJvtOsiA10ojqUGZX7PR4czs0LLopajTRtk7AgShtE9d2V\\\\\\\/pumXjFDqy0OE1mJCMWK4oi1kCRHkPUBhd29YxyFZTnegjXYW4kDSAtvS1q\\\\\\\/nBr5f6HbWB88DD3\\\\\\\/Ce8B4pAv\\\\\\\/JPGh5udm0KdateuPhSAU75cFpRSAItvJFOOwJPrKfhaJQGZ6v5CkOWb1AP9FGPXqXt+dhpaHPtDcjM5cswAPCMxRkuOGopuyfd3cIuFUbt+GBxnIO0W6TO0FrOh88+ONer09qYernwlLnwkv7qqSe0XEj\\\\\\\/n\\\\\\\/DDnS0IU2OeK6ExtzPQyWB8OHvyENgnjyCDfREusIC6V0aD1kZcT4\\\\\\\/7IHWXOQ5egKaMHZ9eubpdiwmovkrzD9fIZz8ld9uoE+F+wBMQ1umAi\\\\\\\/fmIMJQvgWL\\\\\\\/\\\\\\\/K4C2vhFBeam2ngblf9SwjmLXzzeE1zMNNuKh2zzsLgWC22NzZMRLcAIkLdZ+RybJ9MaSZSTpanQHntDoPmaCBWWDi09U7qn+uazysUYh4fj2TVV2qECVhiXRL5fzR7id6GNFzFm4nfR1k9yYNMMHVWrnMZO4nw\\\\\\\/WMHlZCiivWM0Ese8d8JEJbFluTywDW8QdEIvpKAT\\\\\\\/I0LXLasK81qldzHA\\\\\\\/QsZb9ucG0\\\\\\\/pKKolFLn\\\\\\\/bjJ886OU0v\\\\\\\/2fuJgAzWM8EFDvAcWsnW74j6n+nq79nxIsURbtHvJKmkuijsRQvN28pcfi\\\\\\\/u5H1HDinc0KGHO0oYd2yOycljEtiQR59mSewPGcXWJI3p3st84\\\\\\\/VaHvBcMgn9xtLpfIIjSIoCc03vWglID1i3oZPT7f3PHho8dzffhzi9+BtBZJdjpd4uVaPoceI6frx47TalvgwJgfPvSWrmyll14ZQhR+1\\\\\\\/l1+O5K7XUdN1G\\\\\\\/OksI+T8xVvBJO4+at50rq3T1CD9GjSrMCp8p\\\\\\\/kGtuSwqVCh6mXlx4v+Q0LgDHYVfoluSgxJAHEWPLKa6f1i98fSgcv2H3KU7raM6dhE\\\\\\\/GAabYLr1mNBF7Vxt6Ynw0+8e1ohpRroXzDgqUCitueuwZgo60mEoPhdlCzANiv9lIXNqE5n4wkQH\\\\\\\/tV47GtIV7w1FaWMmkn8OkLuC26Zrdey\\\\\\\/LJ4SQWUJlwOXMispELZtYsXm740lKGRNfpOZayMyUTN4VHAG1O2BFErvx1t+wTe1xprrlE8JYI31xQr7qFYSaJ7wh\\\\\\\/ZDsAnx\\\\\\\/ZZUwiSAK6f3s5czuq+fWX4L6wG5k8v378A18rav5OtOZrXMB54VtRNa9pwU+Gv\\\\\\\/zeznWF\\\\\\\/VLV7n\\\\\\\/lsUGdvN0i1gRdv+GF6WUk+HtUlRiyTmIdYx6nP\\\\\\\/+\\\\\\\/YtwL9h6kEFxq4ZOCvWzcWCIVgSeprpSOPw9BAny5gcYz7gKrFP9tLpFaJCtC1aly9liqLk4RzOshdaN7uYiL4SyUbTTq5zCorYQ3JoXsd45n1MbEkJPJdS0dxQGeUxfoaLqwjcu3Dv+tDGjgX1SFT1r\\\\\\\/YIPWESbV6b0AhEzC44Ebh4aYKEebCjYSs2lChywfi6t7nC\\\\\\\/nM59kTGNJ+HkItZOI4mcjFz0IX7FMj0vglO\\\\\\\/P+UHEKXbFXY1TjVHXScXObvOIeMCFuYT98ZPacMYwID4uUuXN4XOmYnujJQZNLdPFE9wqBc1DIYok+WLBIeTGKCNKeflZDCNI+m0w4WMSeRwP1vxd19YZmOGRhpQVqMOJVB8NwzSRjTRhj\\\\\\\/Fh2JnpDlQ7s6+xnT68JpWZZD061PfgOAjyiLZedPl6g6olpMUzhYK38bfR9a5FnCPkk7a4Pr7PhthYU\\\\\\\/needIeZHrvBt6OHvw4bOvIAq1BDsrS8RY+KSwaJSmnp6gVJAPKasWbpv0NCvWM7a5rtd+a4Suoe59MdbTgKVk5QvIQFwRomGLBGQuU40CabULNrtddRj\\\\\\\/6JQwAZ4gppz6iLMTVk3a34liYUOUaIzXK\\\\\\\/2b5D6phcMWAea6waBVsMq51BbYfXIr7JbDYofM4vvw3pq0TOdlmdQnCW3Lg4SlPWIqk1mNVgY8seA89SKo6FeQcbmN8GjR55UsBsZBB\\\\\\\/\\\\\\\/ZOMEe6hxzl4ru0qBywdCEt8Zo242tQScmosEWIVCtESNss\\\\\\\/Ps3zuyghGQHaYUpPRNIOyQpDbu1YK1DdbPe4SBf1vNOb9Rn3oeOgT0XlXJT2vv5wTRc5JEIcqhmxPr\\\\\\\/zL2875r8NKYx6tjWPn59J1vcSFJV8+XIWUAdgFzmXrr34vX2Uvwz00nGkIm05zUJsz+XVa8N+\\\\\\\/QHta5RK3oR0zQXm5adl2K1EQWMsVhXklQZPYCPPkhJh0nK8vk60p3DrXBqY2PbwFaNwfwLgdVT+SFqXHddszGNA4z4qIcEgHK4iErO7k6I2X13h0FHyoFFMXSknOgnGnsHEgM2\\\\\\\/g6oVb717hebsu5cYMuAqyfiynRdfEU8QAmE2HjMMESVcOB8il5i3Oru6rntQ9wTMKC2t6+06SVJ8ybCF6MTAhmhwzmot8uxm1wdROklnMMipmpEAIH1D0n+fTc5134+oazpJ03rnAGz0BdKmS8MahCmUXzTfiTryfB34ozF4FqSxF+WVDTVNUaHX4bBVsEH0nefPAVpV2PjRXkQBTrzeL5MP3K7DZamEbGhaV0oDrtHuLersQc5mxhmrZ5J\\\\\\\/jKVnCyZrRHbdYTrvbz0qkM9mRRD00RQVgUhAeAu1ZCciFeYrWvXs2ki7dDWwN1UYDK4Ny1ff1LUKDjLeteM7QdRl3JMxsc1kFtciQXrDIeOhV8PXj71+RcTR3rv+rxsZ\\\\\\\/rGGLDOTftqE16UmRbQTSTO51eyPIjmqHP5DaYfCCRuoButOUkM\\\\\\\/L3k4aZVa3UxXj1hvUuMv2fvHpCscrKm0YcegVKpwUtj6NVA\\\\\\\/wXhMMmokKnrbpzgET8BHASqOAkZxKffwRgCktLrTyRaz5KHTEd6RD+LxVU+Z5fVFIGXqgO92LqEkQwDMFS7mcvR8GOtYBsl96t4xrE9bpEJFIGC+oXrKjneWOQFPI6b1cZGi2aN+wmXTL5P0YGmVN5wQLEQxak1jV\\\\\\\/ssrZF99MDOQZYYrcKmM3ugv0fegw\\\\\\\/9PSb8aD9TQ87bpvDHrvnFtes3A92yqcyyP+VfJx+9mCmtaBU9e7ynuIsYcmiZcvh2O+tGWtnal0b3XFJspvyRQEMdLBD6EnsU0AKAAUpU1v1hJwEGs9Dgeb0qtL3XZAlcDLjj1KlOA1TndQ1e5nwd7FGBwdv3ZIKJjGZ31cn5NwUoGiM4vxN1gDXR7YEhE+FJoiE3yWOwRU31PzErOrhk\\\\\\\/2LQEzcZ4CAszkBoxmkZuyOGa71UK88KJzmyB7Rtb2JBWCxluoSwJFyq1EQHm8MCQ53rnsiiZI5KT\\\\\\\/1DVUfGUVMg\\\\\\\/fwgr9+8DnzBRIrzXKUcwJKxfdPCxU82UmhaFgqqEO2DZ03\\\\\\\/6w\\\\\\\/JDHvPmNfgjYnmFfAHBSeqUVjykSVK4uKr9KZbx9VFnLxfv4I6z0T4QikZF0Zy3Fz6RBTlfc1UNrixI1qdyZCORA2tnqgDttcYfxh\\\\\\\/wyh0bKm3+cExrNk3j027ffbQO\\\\\\\/2IOILG6OUdjEgLuBExTHjGBQHIw1t5HpQ+Eo745p3d4nqIu0Ep7jEBpURPMbCqcAMXkl1nyfMgQ+IfBTNntQUPnZXihOgD5BIS9mavnq6VvDeiAkc85+U8HOtgsDJzy05J+Y+kBA9j2Jj9Yg2AXAnfnWcUNiiS7jFdNb7oTURoskBi5Nn2RpxWjfyOzToGC0PkNFyjXoiIUxyOe0eQDOWoFpNjIS86nxDvQbPI3IpUn+GKLxvE3oBHah9xmGisgVIMjVgSkvnHUj8zTbqLbK7Z0e7l9unD6MCQ2GFzBwKeKwCCw6Bc0CBOvZWmRkGmsBr4O7WqBCkNNGiGxG9pGvHOAGmh5QvPPGKqE3ooXUegH79n0BCGkB95H+Wsqk6qlz6qYWp5aIZh2Nwy7HqhP6sl4UMG7ehTT2o7V+9r7Yu78Lv2znYDOL0fTlJMxMH2mq6S8Cfbll6lUPtoSjyuc1E\\\\\\\/zUOvOmNZNH9I7+ZxKbSpf+CCQ0dGd6DHuFpyD4DlqWfH9aqfsu0ky8sSQJn\\\\\\\/EccMlzOGeUZRw9eE4h1i3a7pV4PxSxCu8b8Bxpt8XfuwTi2UHenDVA0DKBONNo8tMomxPaVPV3TqkVL3bVCWKyoacyHnHmXpOZZEETXNPJf0a3akU2eNe4Ht4Hwcngs9jn0WnzgQOg3XoFLGjFTSOhTpL6QxRHg9qphjrAyk5e3\\\\\\\/46oITsyLKpN+BhPMm3UuiLlCMe1uKarETnnWb4fAVxCn2h8CWiiKZAvjDNN5LJLg1w+BXnxBwh7ykqkBnQkrXL+TlsO5oalvUa6JOIM6OcWrVtrWTXPjXacLMV0ZWRARigRMBPKG5vk\\\\\\\/4zwUhVHRyI3thzBo1ZZvb81kSSseTjox1yLLfmUzqH2DrDWPWTzFujEsCBxH6CdsiGVba5QRCfrmMhsfWTkoKlDxkmDp1zGZvj8bs1Qa1Aofw3Md4WD+o07iLSmmLBfJoI+t\\\\\\\/Xw2pHu2eYFDRfR4CjNIRptXOSttT6UeCQoGG4JCZCCJHHBk6Rb+OPol82W5mgyjcFXydH0IvYTlSq89BGp+4va\\\\\\\/mJ5NSU8OYo1fBqbc805T8X6GHBm9XBAabI34ALChjyHsQ2trF1AI+f2hKFAr0NtNFvLMUWSwbDav9DSpGLt9Oj4cNajoPm+GBuznXwF1ySlgi0EqqgKcwVEbIBJzvQLRE9tXvdhiIQyc\\\\\\\/1VS2OC+I0nDgw8i7DC9mryQmnEvkLmtwaZP9SVTUM8WW6FLffn8mj80D6HQa49hXFeJ6C86ct9lH2VuBuaCumvd4RD+YeZSIFTq8LUFeRzZUV1uqX9G0vNUcwXScv9nrNc0xvdyF9umAUWHbD+i0dTXb9LEswrySmQ\\\\\\\/U2TXJi1o9Bqpyck22YPY32ibzoouz5xmx1\\\\\\\/VUfi+7cn0Q6mpJ1oj\\\\\\\/zRj3cru8OwxzgeBi2AY+cng0cJWtTBYkaxNqAYG\\\\\\\/jdIAhqE8YdrPYlAgmbHONwsHnQ4b2vfElvTTLC5\\\\\\\/0qYVtq2E0PscoPt7aLcfO9R9lGXsjpTrPI0KNdfYVWl3QlZdexW4\\\\\\\/JS9mu6Ff\\\\\\\/i\\\\\\\/pkdhplKIPX5j9ADjnwTgFYc0UoTVa7GeAQWwX8G3nISpoKfWjKRQpwA4sBQtRQs9OZlBMo0SVVRPwnabI\\\\\\\/rArYLbySrOBl8ybK0B5s5XxnXkOMIWsgNMvekkYoiPZgz9ZcYcQbrOoDK21EsDVGDauxDYb\\\\\\\/Yxxwseprd94s1kPiAhfIAiBdgbg6U9lQUDhk0sujrKaNqqGqtjhbYr\\\\\\\/cAl+HeymJR1uAMuv03TLhBkGdvXYJMqE8Yxm3mmNDFsE8Jai2x3L\\\\\\\/ilOQ3IWADJyvBT\\\\\\\/wGZk4NcSruoI5UBgqQDym0SiYwI9ZBRU+qOY947qlDGgXVS4HYv4L3f5BpwPycXuogjUEQ5T+8tQA0ER55CQ6Ggr6Tj+Dt8VyatoivXQgRj9moWagzlS5Vikd1YTG+xjlanNQN5CC0jx7aBWt\\\\\\\/Ve3d2ubvNMiYIwRaD0zyuD0jiwoTWzjVUzzN0OVgsvhaMBZGiwRll8\\\\\\\/7kEsxO7agotVaNxtayOglCVEH\\\\\\\/IIBF0HoUML3WRXHIbJ0o7cZUc7ambBiVMvOWf6Mlv4jpxEVJbe0X\\\\\\\/u51S4WINtqRi3RwmwIpXtwArxcqaoxdOAz\\\\\\\/Gb5SIHOSb8+1gw8ocJ+TdH2jb2wZiO\\\\\\\/VKucSN2lx9fqJ2L5Kz+gKRFwFbZPG1n+S0knEkRLlykbPoLj2oWC1KePWB02FaH9DTgplfQbNZwuocNGo4zJTOlwq8ouxud0wNpAAALAiU2BYtHisOPtW28gFsHjwHu3gvJDWFcBiwWgxN0X7UCCcNMGqtHhpsOBaO5peBQ1sllND7c\\\\\\\/w5kbfMwWArIWZ3v0hB5G6QDVjl2BxnPE06jd4takZJNTdqIlkDLaTkW4TsPnlHyWheW7EW5lfX7O1xY\\\\\\\/+qGTG2LqPkkGqLmsHCr7eRny9Vrr6+uOXNe5UwTOFjytyP4CTstf+YXpTc5IOhqOhzyXJITduOz1eEPyfjXi50ca+EltXWIFEq9tAWV22neTpNm9u3lo4V9nO88RrZnueozt9OvqMyIKWnI2S08QWg0kVZf7T\\\\\\\/nJ4XIouqMb3CK0t+QtynfuoE6g9l114BTyZyjCxQKPskJ0hhciROROGtfdHk6h\\\\\\\/xLvS\\\\\\\/8\\\\\\\/+QVRIVbmTWB0BXlIBCSXNnw1x+iGT4yP+3+j8j3Az5ldqS0KLE+jfOBCZwZqN0jm8Mo6XHevC2RRW6UBFBttnNFDIY12EykRnKKlezW+iygN60C8qksvabHH4qsB\\\\\\\/rVkZufGPpUG2z8MYMs5f2pc6HvADi33+4la+iK2IePKdzA2PMfy6jYFZAIERZgwJOcNZbB4nqNZ3jNdKFwOmQbcZ8XvXqyrPHE6uh11dLxqArGlQaH3dE8FfNltM5wmIGh0ZuLPyLfjlA4MMEbJyrQvYQRMTF+PEJuRPZfX5DwqRjDQ9PwxqghoxeUaIKZ0sVozBR\\\\\\\/DIuTrOsOBnkdA0wpEP22soVRqRV6UWv\\\\\\\/FpP8AUQpHezU4BvAbvXeb0ojqYThMmMgkuejps9uPgJx1l9TZX288pZK8UKTx8Ub\\\\\\\/vt00OehUW3S0833eFFylD2AB5F+d3hZ1jVr+SM1zXS0nT1DcOhntugmqeVmYxLiofK\\\\\\\/CcJauc7PCEG+C1Y8N66NgItu+oFCDnIFzmJzIEghFGjGlmJINemnWg4pM5lk9LhKR6kDXwI9SeK1c9afCTxWBsiLJvVLGPpdASZ\\\\\\\/jwM8J+fZe+be+CDb1wqj\\\\\\\/CpR40tZFSUpl1QTAu5YbIvyXhYV8KESGbugp3iAndpO3lHO5p+lAT+4rkkP\\\\\\\/XM5ht8FqE+0abCaAp40Qnr3VaWJ1ZHgBo6H5bbIud7PyjBlAr7MJuH6MZdMeRjtoE7Zl6JnGWvUoklZh5VGy8SfpyG4gmk83VoiySPLhZoTi8D5l7U6Yn7t53jLheN12dLxVTpMouvxHEnl8H65+BAqDexfZTeRoKiuB1+Mwzh9SCuYBJitMydGpuGArkZ7E9\\\\\\\/n\\\\\\\/Bhb7KKJ4YgaOJ3thquFRwBCQFwIvFtRFYcgURfFKyNS8nP3O07tMLVxMBj4cIokKaAplvItyeppzRhN+QFI1JyLYzQhldXa+B4lAgugNgvJcLksdynibSthvmOR97SPyXYSSFt\\\\\\\/uMTBI359k2B5nG7xMGAZl4gHSm7IVccJO8N19nHB8JJ1dDd8dUNctCP5Z\\\\\\\/k6j237ka5Mxj2vlOP6M7dIW7FfgccTbGRrEhm4POzypBjuKhojSCGI99kjRCF+Onto\\\\\\\/JxaWGFCvBaTcce+z9oJKm7UdfHQlU0y6WDtYvJM1zxF1fXHzZMtC+w8vbXiQMwJ+TT3Ln6Bt8x41IAhp\\\\\\\/+cY1jCuzY4CyrYLeAcc4Ob2EimBG4wHwKjMsVOdrRiktsIqD\\\\\\\/Dp2UALfqsEqekZrgqaVGcFDk986sSZxT0U+n7KSwovJ7QgCv9JrpeAQXjJ7P1ZNiF9dbjFpzGRH5SOfQ2+9MLkvieSjjEizmTVnZNbL4Suwsma66x3MXoyZhZjvQC+mFPQDxF9H3ngrg+NXmJlNC30ekwB1H1vv7prDUaz4vAD\\\\\\\/GP\\\\\\\/2zdL81aLnAQRTRzuM0NwJtqi4FQJ0gVySsyXusbjh0YRlPO1u10bsqK6bsNR7YkxHApMpg6Aqv4ILn07yOBpfcMrs8kTuijxjTujGcxq7CNED81lvd6wpqoZg7+J8zoFVs4y5oetMf2sau7XOqfA\\\\\\\/O+pim8KwpxrHo23EgLsKBJ7mPcsOI1JkyzKgAPs5nB+TH79y8dgFtnemeeT\\\\\\\/ywR1HMA9pvzhGwgwN+Hw2EHp3PdnhGwdYuE+1oVrrwM8rXoPnon2DbeCtbITt8WppIOxMYLHrordKuo4Uc7anZlg6Vm352\\\\\\\/J03agabFxhnDxlh9Bublx+Og9xLwlwcftZE44Q077ypbXeKOqrblov5QhRqaUtdz81e0rAtX9DDmD8KMR7PtkI2g5ZUvcji2wft5lwkmpAtp7qrOyhc\\\\\\\/bMWC7VueieJZcUb32Lqodq9m8qJfYCpYI0VSb03cKiL72X3y+IdXvL2y7XI4BSzDd3LnUgQtNMhsd+9q0NO+pkNoMEocDwS44R6vRkiKX2\\\\\\\/vJs6EYaAVU5Y9dPvZNZAqoIxVzGjOLRFC3jRgUENlJ0c\\\\\\\/dO0GPxtatPIaZv6DRobRNZEFoVrrcDwbJOdT+e2dLNTvOSX1\\\\\\\/5XaByBwTTwlmIDfo7bLa7aTMmYTWE0kQbsezUZdxqteb3tjiz71dWDS2qA6846pz7DGMzUc1Rxml40lRqcVeDAJBaLzdSQdgLpmTJtnaiV+VcGDGsctH1YLSf2oabG3iwVxLEuMAjHLm5woG1sEOL1mVhT2JbQUYcMwSK82dXkfeVXpEUX1uUNA63EX4VK3lI8qc\\\\\\\/3jf0l\\\\\\\/VaZODtKQOEeEIR0KhCFRyxgHPuFQjlj6fBxn+h33xGw4Aq7cDU1fZGzUVhSHzk4b3Da3KHTYQBz60RjctHmZHVvgV5ToMS5sn\\\\\\\/PUMxjA9puIeYJwihkEAQORctucvgDktN7bnVorknB+SM5z8L+XCQpT0hmdpmeFtVt+vGry+McNjnbrT4cD9aeMZrXIlkf3wrN+KSWNUalonUeBsPyZJkhHpTC6877L+3TKzGzVooMH79WVY91vkqN88Q+13MFuBAvTcxWSAsERLBNGmv9Cru1cZTGzuEBcWeQSVkZDqjUWQr5pD3IykZiZzr1KulIVng+UcsyQC6R0wFg34XTv7NCy1HwSDtDKJyGv+omz9a9if226NInT0ROxOFO5JgBngdF98JoBQwirdAegPny5q5e9UvJp4PdSxaY4ZUrXwprDAk9IOlrIS661Cbx6kszU7v1b67zS71HA7M6WvopFvEET+89BM1t9S1ysDHKkhnt7OoejC+mQ9wscbSE94PdIVYRgut6KoZZX2YU0\\\\\\\/9HIzLI0OILO\\\\\\\/ujZYD6XKZaN5fYn+L6aNlUuL7+gvXCDTKpVRlhY34vfqRyalGA56VKiwZBBI\\\\\\\/uhNsSriyXgjl1ZqJKk5kE3xbAUEIgr+jyL\\\\\\\/cvtLXA\\\\\\\/ItNSbwbCej8nehU6LXDn7qeD+p\\\\\\\/2vfVQj25P07NXg3I+dDWoPiztByR9HjQl0mJn1ks3rbefOTm\\\\\\\/33t5wPFI+Unx3EaFu4OHMiC8kgeAuxQb2gz+XbutwDMfHMopfTFkQDAg4oQJrWCYgwLa6r8Jh6677HWCves2ZsKjPfTF\\\\\\\/dB6NsX7fW8a939GtFGgfH0fyfs4YdUBZ+aUdwN+fv7LYlvojoAfUB\\\\\\\/2pXaE\\\\\\\/kSLjZwqGFAW2f+gezrikQo7xsx6o1Wxanm4mHBU7wbpSv+k4R1UzV5eQKGu93GSmUEwSvDDzAkHzlGWA4mDTH+FVpCiewS1K39SC1jrvAJOIalgy65JKpOzWgQQJ6a\\\\\\\/cvFHGcFMjblYEtNRY5pYxQZ+b8h6fw8VaX6sbf7MQ+xWo043oceXqQ9wgNsrI\\\\\\\/9AJrPRbmEgLfuzs9bQ1AVgUVRAFFcpgIbo5S3o0IQNNu9mFuyqC5J7Q2TznncRv1mAGDfSyuyX3ThX1DdkOXUOE8p5JqPxmm818GiIJzdXXEjJRiYO+tjKMKsLHFbKh3gDD1RFd60Y+R\\\\\\\/U4M\\\\\\\/cI1oKjqWJn\\\\\\\/vAmDbVTCC3zPccYvLkwV+i61AlBuo2B9hQQoBQ\\\\\\\/v5d0DWCIjblb2ajkDEA4Pq5A5dzL6jleG\\\\\\\/b5jyuDVkrFF+UMAbjJ0znNtlEk7AXyjSuOy5I6\\\\\\\/qHYG3MGaC\\\\\\\/OaryvnM+TNkqrchpgRo6n4WS61UImrIT5klxHTnjDk2\\\\\\\/YKVdeQI6d0hCThi69OnR3dfVoIinfNkzu3N3slXtAgqQEr96C0TmTFzw4Wif91AiGr3g3\\\\\\\/pMa4MvAA1I8tTkaDO\\\\\\\/Ok86mRMOkhHRVF0TbCny9uFNBiug7KfJWr+nD\\\\\\\/6dhUMb08AYYQ0whJ8Di+SvpfPkfG1CoxOKiOaqgKOF8zwEnhYhopeUk2GNuEvsoc2Tk0kKEvI5pFXX3Wf6mf3EclpuuZrQ9ui\\\\\\\/N2Pj2Ua9+si6j6AI4+p2TPX3PBn0rXr6lEe4BA7BqmVeuf0vmps59WEnbZdQ\\\\\\\/Gr3WSH0g5t0l3YyuKcc+niKTLRu+cUoZUQwY++urfHs3\\\\\\\/QNk+KTZ9saGqoxW8RBJPQPWUvKSNqCfrIjAPImXebQqR5uCFO9uqnfSo3yGUxzJqZe7xDgZfobRYZ3KVekA9OsCtIndN+vIJnMJP37uvw4tKY9AH0YXlSGqW3xwCJXvFLQH5HoYcjdwP4w+MtMRG+7BclhlEr6VCrrcJwL98wZ1r8TaAnw7wu0SshFJKzNq9NZQywBHU4gxM5M7xvShDldepMLeMIjqWkGVzuuH9ALDmFw50\\\\\\\/k9kaCtGkOS6cSbPWLzgjPXw1qR2R8buOKGZZXwlByQfAVaw02HtTxTgaBFfcB9VD+kqBQ0Io4mwNJ4ptFnZb7r3J7g1MIPJ0OZX0BFF2keTc9UKiHa2+WNyVQXrC\\\\\\\/zrlPeP6FufWKFPQeTZukXc4njwqKZprf\\\\\\\/b1fT2btsnITp2gryumaPHOs9+6dHiXjNsmw6LjewYIQ0pAXpJuBBdnbYAeSx1KSpGHLNNiSnZ3zvNEbkGfK+VsjuwJ0Cr3pdNgctfPUOBr8EMOS\\\\\\\/A4Y0Y+tSKYmCwKJuR5qcdjif0T\\\\\\\/nYHhhQRiRZnMs3+\\\\\\\/M9ZTAa7uwjsQgwfs+Ga8FB8dGibzydJ\\\\\\\/I8\\\\\\\/HYAujEKubapNu8lL3ht2c2o6zzQ3XHDcIZ+DdQ9l3ueqMp9w7gps98AD3c7T1YbWNJ\\\\\\\/W5ptYDanRU7SlwWi4pVRkmiKfCns6PDiumHNiVaGc0\\\\\\\/Dghg6DK97O8j746tCP5aBdi2tXRi0iKNZN3DBjWE6D13n7E\\\\\\\/BAmLNRISMeXwOjI4CHcGqeeqBXsQIOR3k+UFWTAfPJuf3uQKvtlFmt3KlIfr\\\\\\\/emjgPe09inyO5PfQmPn8TCskOzzXyxWVzIAOSc+qReEHAMUwhNiubizTly0NNAJnMJhhL9S23NTZbMOzLAW+Q9a4lMqTWYylXgAjMIGyt0n0rtmUwsPj5JXHtlw7hY\\\\\\\/l\\\\\\\/xcBnbL7KLvTqY4M\\\\\\\/KCXkYip3JvhA+k6eLtpMlWIwt7avwL65JMC1Nnho98uKx5RTpjHzao0+ajsInQgmagXbA3NkCdx0uyqORENRw3hUFdjVP6t+IGubhyeP\\\\\\\/qsU+wsDzIQm8UOuPAMqyyCRx7BWGug143O6cQ42T\\\\\\\/ocgqQ42JIFXXIDjVPodsyq8OophLfSDSpGI9pqt0mg4FwkIxEHu5fhC6iy+mQ2jQt5rqYGTNG65d\\\\\\\/HrfqOF43A5\\\\\\\/7qB1HEimOnLhjwpLCP\\\\\\\/VeLl36bemMD9zb9RkdY+mUWteTI0SR\\\\\\\/GF8SDMbGOlyYKwyppiL20RqBlfcppHV6nf6jbztYKDMh5NNT3ndlfBYvVolGLVPaV9ZNJDV1DGzAPXoD\\\\\\\/XwbQDDwrUTYyo4ah5qrptNX7IvS8yqwGBkCFhnW\\\\\\\/MZEihL11fNduV89aAOwmVAwTJWYjSAcNUOBiVDJyb1n52IpH08t4oNxTg9bRDgSCbgHd\\\\\\\/dicYp+GTUDtb8pxPFSarbunAbqP8bYCShGl3evJYP2YjznNTX8QNcFgiJwo9gmUOOZcm0SGKbq+fj2r0f9U5ojD7h3cnflKij2fpf8TlBsI+VxlHQnsRB6FUGv4+RYg58HfcgWjulr+xl40FNQpFFZmXoSx9F45NYhya85uJ4AiBCV7djU8lf20GMvVp9fZapKy21N+225TWC7e8ob1NsIagMeIgSjEIHhLnhpS43fVMCg0yn043corMbAHeAWR87MrlcqdPe0tcXjbApvCKgMrNMvzvMbam5tL\\\\\\\/UkqghPnPYFMmaRbpZ\\\\\\\/5JkBXJc8FZannQ0P5+UtKYsteQfeCo4P+Jv7UgcFIBoV5fMlRFw1PWwlG\\\\\\\/sM+Bw\\\\\\\/SPLnCcxMFVGmt1ZfgTeVX47uO\\\\\\\/X8DXN6ySjnIlx3TN0L4KF78IRZiuz0C4kuqx\\\\\\\/m1HTdAgeiXP9jkv8WnXRQCYSs0FyitYWraPUfjRQaky8lYX6UJxE7BiRAazP2dq+uuVwi7mpyQKd2jjuQEKfGdscwNUEtOIi6Px3jwsc0YkLW7sbwqFQRQl7h5PyrZV5Tz4Y5rXbhELBy8ADFR7dFJRhqO+rvq\\\\\\\/tu8jZYo5OTfCT9z6D6K58wzt42+7meBbCfxrGCSrnDqGWH3aydoWuZlL9lAKUfxu+gXeQgAr2BlWNBnJok9Vgkb\\\\\\\/XNjSbo+T8fZgq8ozgCH3CL99WdjAoUwub6CtjMmZCZzOy02tlvMEMD03dOpHIHepZm+VEobDRbyZ\\\\\\\/JHBV3PNu1ypVz9PfsbaWs\\\\\\\/6p9linE\\\\\\\/1QTgxyNKY72d4Nh4hpAwLa3CMjiFqvbfc7HjGrLSIUvqdD0zWWSIZigXwKb2ahMPIRabkWr6OaEw7lkjThrZXBdsoci1vWm3mp0n5fkR97lGgh2gilSSKa2Xfu3nFCEk66zSzPv+uGWV203JNvHAV9YAIQSdWiX5m3kIzxdvaamN3YrAW3Hnmn+BN2g56mb8LUjOg4kFxkLZ1KpH3LSFz+NJ\\\\\\\/qNg8k9Kwzj8oUXCQOvj9HKLF5NP9sTmxYDKh7J3\\\\\\\/o7BYUwZXxlsbmuMvgfXUaxP0tsTJWDZGuycnBl5K+rh6dmvYfX48+QUeMJQJnW+hlQF5Ko8O72Fjn5q6DmHfVrlxRqVoLC2er7C1aFWEWxtmXsRQWTlYOwV\\\\\\\/Af8x3gLBKoy\\\\\\\/opZOObREqmIRoddWKL3mVRdTLbK7JHyOSLKnBQGu+Z2gISJr3ACFnDL+HVrmlNyRdyzPZZbcxTKzGC30LVcqvMWm1kTWh5U8U0RR7DOyOiVDqDuLp+ZsqKH+nyDQeDn5+D0gYWBORPlu9oMZ+vHvuKHiHmcptA8URxeX\\\\\\\/\\\\\\\/YWKcfi1ZFW9tsDpJD9p7\\\\\\\/QTFOFAcqiJTgMESPvdXNRP\\\\\\\/X0xCGhEWs79cWooUlCJeqYegHkWBzo8WBcd7ZgYY\\\\\\\/p5fceB1Dj5fAHPLrrsr6b+5YStpkH0Osyu0HZ0+UaXc5FCcdyujyh\\\\\\\/mBu\\\\\\\/jkYJTZDL5UKy1X16eDHuS4KBllOwQm1ad46Oc2XcHFyIlaVphcriRksuqt70Ck711rKw9Ui6ciKKweq7toc2RouHWtECs2b7j2LDlBe0WdymLbpiqK\\\\\\\/KWjBiekvMn9ZcaCvJq6sn1qVepgywtsX6kSOesF2IjVEDSsbUhX0X\\\\\\\/WNVBj21k6AsAHmbDQoNAFLrgNJ7I7R8lo7pTY6C0aIpGDqjVnUMt8\\\\\\\/SUdq76Sf4ejZJWXFRpcjupCI9RXxjPNzvomy5J4+uJqToCk6yt1K7wXMtl2KUDod+RIIDx9PtdLt\\\\\\\/PMseaSQccdRvxwwElYCT1+LwkxX2iGU+C1cV4bYSjODrKwchyOPa2UL8YQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OSLFS+tEWXCXHtbG4RxSig==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222510b4d939c26af08d5bf7342bb9f8df6729f601aeb186e03c5a16ade0a2e19e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022byf3C3hQ8FIFlVsHM3Z1tpk5AUDEpfUoKtmW1Zq5+8I0uwjABYbgJ0g4I3TOiUFJxgp1juqbnPZm78jXjtehNZQ3WKUTK97IkFNAlMNR80sOdsV4HUh6l5d1J207G68Iyq5eIqu5CGC67iiQM7+Lzzcovagzk5nL2T5qgj8HLGpnkzJWS0tyyjCBgtxwfYKQYCFxm0m5OZuHfFqy8lRPJhZLGjdAdL8lIUMj4vusvXLcQxtE+pnKOKBwJ+MDribqacwLwsPMFPl2zpjs9z83m+rONVvYdlMFhLYMUERflr0er693UWQrIMjcDha3Zl\\\\\\\/Ik8pxXYQEOAS4cyVmWcUGx8sGMi9ir5vP4tGL3WeGbFXYlzgbqo7q1QlPaO\\\\\\\/aJE3DH30l2GhEzXwOKsPpI8i4VKZWOtoFzf76OhrPg+MXNyrM0YFYhtci\\\\\\\/Ifrj4W0x1YnwaYnqhdXOGmE5g\\\\\\\/hDNfTyFcC7r9yhLOwg7V2QJ+styKAy4ib8FdGxTaRUY+vMMLtGcVlCQL8WYjjG+eHipJyiDovBXcksclv6EK1Z0twiDfH4mJucAXz93yqc7Hprl938xkHn4z\\\\\\\/t7YZ9Vp70Y6MzGLZkmCIqTiBzEmaJ4UqCQbKhp1JTTJAJFJhs0a2H36s4rTjpPYLW+OokRktydF0hwSgRtbRHHOqn9DreM+tjfRysJOOGwimPreiEsYGANtONKpP4FBtLmURF3VBsWXT7FTRj+IkYvFw\\\\\\\/5NOHPuic4s4gH4r0pz7Dw2hBsuzYaoWF8azS7IOsHfEzhRCxeSU75oGLYVrYvdfe9fPOtQ+x3bC2Y0uhzXnzKwUgzXQcUenBYWFa2wxjMqegiHkBT5JxNFakjDFb8E1N263xVR1ndKy5UCsQlQ1d8HkDeiI1q+rJLaWzysQJDMjTK4ty73eJeuZsbA7+NTqRHLWMTyPuwHY+kVANU0TKhswR1OfX0Wmp9C146AW8NktD+MXxD\\\\\\\/8z0zPM+fsJXbEejvGkViGLg9su1\\\\\\\/FxbMG6C2kOzbce6YoQgSLqFTdQXPwbyXu2MtGnk5YiAgQ5mRSaQaQbPK88VbWaMW4OZOpFALoshZcudnCrz\\\\\\\/Z9aYpBq2JrkkegvEMMKKWML2\\\\\\\/Dko\\\\\\\/fUP5tJoqXi0tyqu7Zco1NCt90jN2XuFscTLwF9YVAHzBUttw5olA3mwdJDVxNfYtUTdpvbe9oSYvm3TX3aCQK5NTcCx+dnwxsdCEPTMuQDpRS4YLgYY8spuI1a4uQn7yqXE8bkFaUPhDqdVyFi4b3B97pvtwrryCtcfETa4TrxurZVG87fZbsBuUCT01ROVcGn5L036zklRQfF7\\\\\\\/6IZSrxwqcGfRC\\\\\\\/pqAvkMW3bZoOo5GUYQuKcIHZbCqrLbkfCgzkKYQClUot2NnsScRt2t3r+hTYrgoCUjw5b0Jtl6H\\\\\\\/EEiyTU8CJq5ewSvJfmvhA3BFZOPzDCtHZdgE4+SF7ZNrsFSMJCymNM2ZYg8WjBDWDLEGvdUv\\\\\\\/eWK+mpQs1fSiAvvt9hZmRMofmbcc2ym39PpbrxNDBgtmkAa9KgLhXSXK+yh2hAUSc\\\\\\\/FlwnRAU\\\\\\\/WnahRoP\\\\\\\/LSUBli+CxZlaPUGDi75SUpLfnggB75LdJ3BX2ZZBKFhLN8BD4FIoiUeBDf5Vfx8XYHQvhX6qlKCUNt1oUcFRrE6oN+6riKYnlBEUfUCtgzvYRQEVdFWYy8QpYVNHe+GTK6gRwe21YwS3MWpQ+BSnTLdu2hXs+4qxc47btppeEXrNCujzu0Xi+c4a634tS1f1ypQmTzcptIPdOO6uzMKCShMm8waPw1BUGq6U1VnYFbHQnvRBpTIZeY8UTZsUG2RmKCpkLvxJ0w6L80fEykqsLJ3IGW\\\\\\\/RQB3DD2sk4OM+NRnoU6ja8g5xvQ0juhX63\\\\\\\/EvgAiotVYLvKfsxo+Ue6Qi2Epq42JcKHxzTFmlt8kp5XeTFLXrICXwvy1s+Czj6PqAOIs6vLbO0aq4\\\\\\\/r1Pd5DQHZFaiu4kiXTMvpq+wfaoFr1\\\\\\\/H7EJw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022LCrAyk44qAWmKNIX\\\\\\\/BWp0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002268a511ac1cc56123226c11dbab8d3e62a1545898f36549cd23a177fa9c8d9264\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460248 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/67" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OwP48OrBZjf3szGvbFAcmXIlnCzM0SFj7dwqtasIIsw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/67" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5__-j4vofcRZuPBZ9SOdXm_5dyFG4bMuRP-HLVp0yLY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/59" title="Chemically Induced Oncogenesis in the Peripheral Nervous System Is Suppressed in Congenic BDIX.BDIV-Mss1 and -Mss7 Rats" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/79" title="Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460248" id="node460248" data-pisa="ggg;6/1/67" data-pisa-master="ggg;g3.115.023671" data-apath="/ggg/6/1/67.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jianjun Guo</span>, <span class="highwire-citation-author" data-delta="1">Jue Fan</span>, <span class="highwire-citation-author" data-delta="2">Bernard A. Hauser</span> and <span class="highwire-citation-author" data-delta="3">Seung Y. Rhee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">67-77;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023671 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460248" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jianjun Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianjun%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianjun%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jue Fan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jue%2BFan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJue%2BFan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bernard A. Hauser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Plant Molecular and Cellular Biology Program, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernard%2BA.%2BHauser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hauser%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernard%2BA.%2BHauser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Seung Y. Rhee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seung%2BY.%2BRhee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rhee%20SY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASeung%2BY.%2BRhee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srhee@carnegiescience.edu" class="" data-icon-position="" data-hide-link-title="0">srhee@carnegiescience.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/1/67" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460248" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460248/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/67.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460248" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460248/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/67.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460248" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460248/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/67.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460248" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460248/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/1/67.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460248" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460248/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Complex traits such as crop performance and human diseases are controlled by multiple genetic loci, many of which have small effects and often go undetected by traditional quantitative trait locus (QTL) mapping. Recently, bulked segregant analysis with large F2 pools and genome-level markers (named extreme-QTL or X-QTL mapping) has been used to identify many QTL. To estimate parameters impacting QTL detection for X-QTL mapping, we simulated the effects of population size, marker density, and sequencing depth of markers on QTL detectability for traits with differing heritabilities. These simulations indicate that a high (&gt;90%) chance of detecting QTL with at least 5% effect requires 5000× sequencing depth for a trait with heritability of 0.4−0.7. For most eukaryotic organisms, whole-genome sequencing at this depth is not economically feasible. Therefore, we tested and confirmed the feasibility of applying deep sequencing of target-enriched markers for X-QTL mapping. We used two traits in <em>Arabidopsis thaliana</em> with different heritabilities: seed size (H<sup>2</sup> = 0.61) and seedling greening in response to salt (H<sup>2</sup> = 0.94). We used a modified G test to identify QTL regions and developed a model-based statistical framework to resolve individual peaks by incorporating recombination rates. Multiple QTL were identified for both traits, including previously undiscovered QTL. We call our method target-enriched X-QTL (TEX-QTL) mapping; this mapping approach is not limited by the genome size or the availability of recombinant inbred populations and should be applicable to many organisms and traits.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/target-enriched-extreme-quantitative-trait-locus-mapping" class="hw-term hw-article-keyword hw-article-keyword-target-enriched-extreme-quantitative-trait-locus-mapping" rel="nofollow">target-enriched extreme quantitative trait locus mapping</a></li><li class="kwd"><a href="/keyword/bulked-segregant-analysis" class="hw-term hw-article-keyword hw-article-keyword-bulked-segregant-analysis" rel="nofollow">bulked segregant analysis</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/arabidopsis-thaliana" class="hw-term hw-article-keyword hw-article-keyword-arabidopsis-thaliana" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Arabidopsis thaliana</em></a></li><li class="kwd"><a href="/keyword/seed-size" class="hw-term hw-article-keyword hw-article-keyword-seed-size" rel="nofollow">seed size</a></li><li class="kwd"><a href="/keyword/salt-tolerance" class="hw-term hw-article-keyword hw-article-keyword-salt-tolerance" rel="nofollow">salt tolerance</a></li></ul><p id="p-5">Phenotypic variation of complex traits such as crop performance and human diseases are controlled by multiple genetic loci, many with small effect size (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Buckler <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Plomin <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Peiffer <em>et al.</em> 2014</a>). Identification of all the genetic loci contributing to the phenotypic variation of a trait is an important step toward understanding the underlying molecular mechanisms of trait evolution. Quantitative trait locus (QTL) mapping and genome-wide association studies (GWAS) are two common approaches for dissecting the genetic landscape of complex traits. Compared with QTL mapping, GWAS usually have greater mapping resolution, but the underlying population structure can cause many false-positives (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Korte and Farlow 2013</a>). QTL mapping using populations generated from controlled crosses typically uses small populations (a few hundred), which can lead to not only an underestimation of the number of QTL but also an overestimation of the contributions of the detected QTL to the observed phenotypic variation (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Melchinger <em>et al.</em> 2004</a>). To increase the detection capacity of the genetic loci underlying complex traits with QTL mapping, a large population is required to increase both statistical power and recombination events (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Ehrenreich <em>et al.</em> 2010</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mackay <em>et al.</em> 2009</a>). This is rarely done because genotyping and phenotyping a large number of individuals is costly and labor intensive.</p><p id="p-6">Bulked segregant analysis was developed as a fast approach to map major QTL, where individuals with phenotypes at two extremes are pooled separately and genetic markers from each pool are compared to determine the linkage between the markers and the phenotype (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Michelmore <em>et al.</em> 1991</a>). Recently, bulked segregant analysis with large F2 pools and genome-level single-nucleotide polymorphism (SNP) markers (also named extreme-QTL or X-QTL) has been applied in yeast and <em>Drosophila melanogaster</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lai <em>et al.</em> 2007</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Ehrenreich <em>et al.</em> 2010</a>). This approach demonstrated the power of X-QTL mapping to dissect multiple loci contributing to complex traits. A large population size (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Ehrenreich <em>et al.</em> 2010</a>) and 10–20% bulk size (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>) are necessary for increasing the detection power of X-QTL mapping. In addition, high sequencing depth was predicted to be necessary for increased QTL detectability (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>). However, a comprehensive power analysis for X-QTL mapping that considers all the major contributing factors for increasing QTL detection power has not been conducted.</p><p id="p-7">Here, we performed an in-depth power analysis to guide the choices of experimental conditions for increased QTL detection using X-QTL mapping. The simulations suggested that both a large population size and high sequencing depth of markers are needed to increase the detection power of X-QTL mapping. To achieve the required sequencing depth at a reasonable cost, we used target enrichment to sequence SNP markers and developed a statistical framework based on model selection to estimate intervals of mapped QTL peak locations. We call this mapping approach <span class="underline">t</span>arget-<span class="underline">e</span>nriched X-QTL, or TEX-QTL mapping. In <em>Arabidopsis thaliana</em>, we applied TEX-QTL mapping to identify QTL for two traits: 1) seedling greening in response to salt and 2) seed size. Using both theoretical and empirical approaches, we demonstrated increased QTL detection power of TEX-QTL mapping.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>TEX-QTL detectability simulations</h3><p id="p-8">To identify parameters that could increase QTL detection power using TEX-QTL mapping, we wrote an R script (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS1.R">File S1</a>) to perform simulations using the R (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">R Core Team 2013</a>) package hypred (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Technow 2013</a>). The genome was set up as five chromosomes of 100 cM each. We chose additive QTL with no interactions and each chromosome contained one QTL. On the basis of experimental data in the literature, the spontaneous mutation rate was set to 7.0E-9 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ossowski <em>et al.</em> 2010</a>). Phenotypic values were assigned as the sum of genotypic and random environmental effects (normally distributed). The maximal size of the environmental effect was determined based on the heritability.</p><p id="p-9">Two experimental designs were simulated: high <em>vs.</em> control bulks (one-tailed bulk design) and high <em>vs.</em> low bulks (two-tailed bulk design). We randomly selected 10% of the individuals as the control bulk, and top/bottom 10% of the individuals with the highest/lowest phenotypic values as the high/low bulks. G-test was performed to detect markers with a significant difference in allele frequency between the two bulks at the significance level of 0.01. The p-value was corrected for multiple testing by the Benjamini-Hochberg method (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>). Under a fixed set of parameters (QTL effect size, population size, marker density, and sequencing depth), the probability of QTL detection was determined based on 100 simulations for each of 10 heritability values ranging from 0 to 1. To simulate a QTL with 20% effect, five equally contributing QTL for the five chromosomes were used. To simulate QTL with 10% and 5% effects, five QTL with effect sizes of 70%, 10%, 10%, 5%, and 5% were used.</p><p id="p-10">To test the effect of population size, marker density was set to 5 per cM. Population sizes of 200, 1000, 4000 and 10,000 were simulated by randomly selecting individuals from the two bulks. To test the effect of sequencing depth, population size was fixed to 10,000 and marker density was set to five markers per cM. Sequencing depths of 100×, 500×, 1000×, and 5000× were selected for simulations. Finally, to test the effect of marker density, population size was fixed to 10,000 and sequencing depth was fixed to 1000×. Marker densities of 0.2, 1, 5, and 20 per cM were compared in the simulations.</p></div><div id="sec-3" class="subsection"><h3>Bulked segregant analysis</h3><p id="p-11">For the seed size trait analysis, plants were grown in a greenhouse with 16-hr/8-hr light/dark cycle at 22°. Reciprocal crosses between Sha (ABRCstock number: CS22652) and Tsu-1 (ABRC stock number: CS22641) parents were made. F2 seeds were separated by size with sieves from Industrial Netting (Minneapolis, MN). Sieves with pore sizes of 200, 250, 300, and 400 µm separated 140,000 F2 seeds into four discrete bins: 1) &lt;200 μm, 2) 200–250 μm, 3) 250–300 μm, and 4) 300–400 μm. Most seeds sorted into the 250–300 μm bin. The high bulk had 10,000 seeds from the 300- to 400-µm bin, which made up 7.1% of the population. While ~1% of the seeds were smaller than 200 μm, a lot of chaff, debris, and broken seeds were in this bin. Therefore, this bin was not used for further analysis. The 200- to 250-µm bin contained the bottom 14% of the population. From this bin, 10,000 seeds were selected randomly to form the low bulk. A Mettler MT5 microbalance was used to measure seed mass. To measure seed dimensions, captured stereomicroscope images were analyzed using the Bisque Seed Size tool (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kvilekval <em>et al.</em> 2010</a>). Assuming <em>Arabidopsis</em> seeds form an ellipsoid, their volume was estimated using the formula: 4/3π(w<em>/</em>2)<sup>2</sup>(l<em>/</em>2) (where w = seed width, l = seed length). Seeds from each bulk were grown on media (1/2 × Murashige and Skoog (MS), 0.05% 2-(<em>N</em>-morpholino)ethanesulfonic acid, 3% sucrose, 0.7% agar, pH 5.7) for 5 d, and seedlings were pooled for genomic DNA extraction and target enrichment.</p><p id="p-12">For salt tolerance bulked segregant analysis, 16,000 F2 individuals descended from crossing Sha female (ABRC stock number: CS22652) to Tsu-1 male (ABRC stock number: CS22641) were planted on growth media (1/2 × MS, 0.05% 2-(<em>N</em>-morpholino)ethanesulfonic acid, 3% sucrose, 0.7% agar, pH 5.7) containing 0 or 150 mM NaCl. Approximately 2000 seeds were planted on each 150 × 15-mm Petri dish. Plants were grown in an environmentally controlled chamber with 16-hr/8-hr light/dark cycle at 22°. A sample plate (60 × 15 mm) with approximately 200 seeds was used to estimate the proportion of green seedlings. The difference in seed density between the sample plate (7 seeds per cm<sup>2</sup>) and the bulk-selection plates (11.3 seeds per cm<sup>2</sup>) did not significantly affect the timing of seedling greening on salt plates under these growth conditions (data not shown). Sixteen-thousand F2 individuals were germinated on agar plates containing 150 mM NaCl, and the seedlings with green cotyledons were harvested between 96- to 120-hr after germination; these seedlings accounted for 10% of the population and formed the high bulk (“salt-tolerant bulk”). For the control bulk, we randomly selected 10% of seedlings from 16,000 F2 plants grown on agar plates without NaCl 72 hr after germination. We used the control bulk rather than the 10% most sensitive seedlings for comparison because poor germination on high salt could arise from many reasons not directly related to salt stress response. In total, three independent experiments were conducted.</p></div><div id="sec-4" class="subsection"><h3>Determining heritability</h3><p id="p-13">To determine the broad-sense heritability, the environmental and total phenotypic variances were estimated by the mean within-accession and between-accession variances of Sha and Tsu-1 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Griffiths <em>et al.</em> 2000</a>). The heritability of seed size was estimated from 20 seeds per accession (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS2.xlsx">File S2</a>) and that of seedling greening in response to salt was estimated from data collected from four independent experiments with 25−59 seedlings per accession per experiment (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS3.xlsx">File S3</a>). For seed size, the parental and F1 variances were used to measure environmental variance.</p></div><div id="sec-5" class="subsection"><h3>Probe design</h3><p id="p-14">We used the “SNPs between accessions” tool provided on the Polymorph Web site (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Clark <em>et al.</em> 2007</a>) to search for SNPs between Sha and Tsu-1 genome using the TAIR7 <em>Arabidopsis</em> genome (<a href="http://www.arabidopsis.org">www.arabidopsis.org</a>) as the reference genome. Because the sequence from only one accession can be used to design each probe, we positioned probes at regions that are one or two nucleotide(s) away from the targeted SNP positions to minimize capture bias. These sequences (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS4.xls">File S4</a>) were sent to Agilent for the synthesis of biotinylated-RNA probes for target enrichment.</p></div><div id="sec-6" class="subsection"><h3>DNA sample preparation</h3><p id="p-15">For each pooled tissue sample, genomic DNA was isolated with the QIAGEN plant genomic DNA maxi prep kit. To remove polysaccharide contamination, the isolated genomic DNA was re-extracted with a cetyltrimethyl ammonium bromide (CTAB) concentration. For each sample, 3 µg of genomic DNA was fragmented with a Covaris S220 Focused-ultrasonicator. The library preparation and probe hybridization were conducted following the protocol of Agilent SureSelect target enrichment system (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agilent 2013</a>). To summarize briefly, fragmented genomic DNA was end-repaired with a single adenine nucleotide overhang and ligated to sequencing adaptors with a single thymine nucleotide overhang. The successfully ligated DNA fragments were amplified by polymerase chain reaction and hybridized with biotin-labeled RNA probes. The RNA/DNA hybrids were isolated by incubation with streptavidin-coated magnetic beads, which bound the biotin. The enriched genomic DNA fragments were released, bar-coded, and sequenced using Illumina’s HiSeq with 100 base, paired-end sequencing (National Center for Biotechnology Information accession number: SRX659684).</p></div><div id="sec-7" class="subsection"><h3>TEX-QTL mapping</h3><p id="p-16">Paired-end reads were aligned to the TAIR10 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Lamesch <em>et al.</em> 2012</a>) reference genome with the use of Bowtie2 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Langmead and Salzberg 2012</a>). Mapped reads were reformatted with the mpileup function of SAMtools (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2009</a>), and the allele frequencies at polymorphic sites with Phred quality score of &gt;20 and coverage &gt;500 were counted by varScan2 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Koboldt <em>et al.</em> 2012</a>).</p><p id="p-17">The G-test was used to detect the SNP markers whose Sha/Tsu-1 ratio between two bulk populations significantly differed. To reduce the random binomial sampling noise of two alleles introduced by sequencing, the smoothed G test (G′ test) (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>) was performed at each SNP marker position with a sliding window of 9 Mb (∼36 cM as determined in <em>Results</em> section) and the tri-cube weight function (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cleveland 1979</a>).</p></div><div id="sec-8" class="subsection"><h3>Simulations to determine the smoothing window size of TEX-QTL</h3><p id="p-18">To investigate the effect of G′-test’s smoothing window sizes on the number of false-positive and false-negative peaks, we simulated TEX-QTL on a chromosome with one QTL by using a range of smoothing window sizes. One hundred simulations were carried out for each window size. The length of the chromosome was set to 100 cM. The single QTL was positioned in the middle and flanked by 50 cM on both sides. We used the high <em>vs.</em> control bulk design and the beneficial allele in the high bulk was set to 0.6. The sequencing coverage was set to 5000×. The marker density was set to 5 per cM, which was used in our experimental design. The tested smoothing window sizes were 16, 20, 24, 28, 32, and 36 cM.</p><p id="p-19">We then applied TEX-QTL on a chromosome with two QTL using 36 cM as the smoothing window size, chosen based on the results of the 1-QTL simulations mentioned previously. Two QTL were set to have the same allele frequency of 0.6. Various distances between the two QTL were simulated to determine the probability of two QTL merging into one when their distance was smaller than the smoothing window size. The distances of 8, 12, 16, 20, 24, 28, and 32 cM were compared. The other parameters were the same as the single QTL simulations.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-20">Code used to generate the simulated data is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS1.R">File S1</a>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS2.R">File S2</a> contains seed dimension data. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS3.R">File S3</a> contains green seedling response to salt heritability data. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS4.R">File S4</a> contains all probe sequences and genomic locations. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS5.R">File S5</a> contains seed weight data. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS6.py">File S6</a> contains the TEX-QTL mapping program in Python. <a href="http://www.g3journal.org/content/suppl/2015/10/30/g3.115.023671.DC1/FileS7.pdf">File S7</a> contains detailed descriptions for generating the expected shape of G and G′ for linked causal SNPs. Sequence data are available at GenBank with the accession number: SRX659684.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Determining TEX-QTL mapping parameters using power analysis</h3><p id="p-21">To increase the detection power of TEX-QTL mapping, we tested parameters that can influence QTL detectability such as population size, sequencing depth, marker density, heritability, and QTL effect size using simulations. The bulk size was not simulated because a previous simulation study showed that the bulk size of 10–20% is needed for increased QTL detection power (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>). We simulated two commonly used bulk-selection schemes, two-tailed (subpopulations with phenotypes at the two extreme ends of the phenotypic distribution) and one-tailed (one subpopulation at one end of the phenotypic distribution and a random selection of equal-size bulk from the population). The one-tailed design is used where individuals from one end of the phenotypic distribution cannot be effectively collected (<em>e.g.</em>, cell survival experiments) (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Ehrenreich <em>et al.</em> 2010</a>).</p><p id="p-22">Our simulation results indicate that the detection power was positively correlated with heritability, population size, and sequencing depth (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In addition, the one-tailed bulk design was less powerful than the two-tailed bulk design at comparable population size, sequencing depth, and heritability (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). For traits with greater heritability (&gt;0.7), an F2 population size of 4000 is sufficient to detect all additive QTL of greater than 5% effect size when the two-tailed bulk design was used (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>), whereas a population size of 10,000 is needed to achieve the same detection power with one-tailed bulk design (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>). If we consider the difference in the effect of the sequencing depth, the two-tailed bulk design needs 1000× per marker for &gt;99% chance of detecting a 5%-effect QTL, whereas the one-tailed bulk design needs a sequencing depth of 5000× (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, B and E</a>) to reach equivalent detection power. Similar trends were observed for traits with moderate heritability (0.4−0.7). For example, 5000× sequencing depth is sufficient for 99% chance of detecting a 5%-effect QTL with a two-tailed bulk design, but has the detection power of just 90% with one-tailed bulk design (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1, B and E</a>). In contrast, marker density (0.2−20 markers per cM) had little effect on the detectability of QTL under both designs (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1, C and F</a>). Our simulation of the effect of sequencing depth was conducted with the assumption of a large population size (&gt;4000 F2 individuals) since a smaller population size would limit the QTL detection power as shown in <a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1, A and D</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/67/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulations revealing the probability of finding a quantitative trait locus (QTL) with 5% effect for two target-enriched extreme QTL mapping designs: two-tailed (A&#x2212;C) and one-tailed (D&#x2212;F). (A) and (D) Effect of population size with individual genotyping. Marker density was set to 5 per cM. (B) and (E) Effect of sequencing depth. The population size was fixed to 10,000, and the marker density was set to five markers per cM. (C) and (F) Effect of marker density. The population size and sequencing depth are fixed to 10,000 and 1000, respectively. For each combination of parameters, 100 simulations were conducted." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-128051212" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulations revealing the probability of finding a quantitative trait locus (QTL) with 5% effect for two target-enriched extreme QTL mapping designs: two-tailed (A&#x2212;C) and one-tailed (D&#x2212;F). (A) and (D) Effect of population size with individual genotyping. Marker density was set to 5 per cM. (B) and (E) Effect of sequencing depth. The population size was fixed to 10,000, and the marker density was set to five markers per cM. (C) and (F) Effect of marker density. The population size and sequencing depth are fixed to 10,000 and 1000, respectively. For each combination of parameters, 100 simulations were conducted.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/67/F1.medium.gif" width="285" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/1/67/F1.medium.gif" width="285" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/67/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/67/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467477" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Simulations revealing the probability of finding a quantitative trait locus (QTL) with 5% effect for two target-enriched extreme QTL mapping designs: two-tailed (A−C) and one-tailed (D−F). (A) and (D) Effect of population size with individual genotyping. Marker density was set to 5 per cM. (B) and (E) Effect of sequencing depth. The population size was fixed to 10,000, and the marker density was set to five markers per cM. (C) and (F) Effect of marker density. The population size and sequencing depth are fixed to 10,000 and 1000, respectively. For each combination of parameters, 100 simulations were conducted.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Similarly, for QTL of 10%- or 20%-effect sizes, greater detection power was observed in the two-tailed bulk design compared with that in the one-tailed bulk design when population size, sequencing depth and heritability were held equal (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS1.pdf">Figure S1</a>). Moreover, larger population size, greater sequencing depth, and increased heritability all contributed to greater QTL detection power (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS1.pdf">Figure S1</a>). In contrast, marker density did not have a significant effect on detectability of QTL at the range of 0.2−20 markers per cM (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS1.pdf">Figure S1</a>C, F, I, and L). In addition, for QTL with larger effect sizes, a similar detection power could be achieved with smaller population size, lower marker sequencing depth, and lower heritability (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-12" class="subsection"><h3>Developing TEX-QTL mapping strategy using two complex traits in <em>Arabidopsis</em> and different bulk-selection schemes</h3><p id="p-25">Guided by the parameters determined from the simulations, we developed the TEX-QTL mapping method by using two complex traits in <em>Arabidopsis thaliana</em>: seed size with moderate (0.61) heritability and seedling greening in response to salt with high (0.94) heritability (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS2.xlsx">File S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS3.xlsx">File S3</a>). The high <em>vs.</em> low bulk design was used to map seed size QTL and high <em>vs.</em> control design was used to map salt response QTL.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/67/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of seedling greening in response to salt quantitative trait locus (QTL) by the use of target-enriched extreme QTL (TEX-QTL) mapping with F2 populations of Sha &#xD7; Tsu-1 Arabidopsis thaliana ecotypes. (A) A schematic of the TEX-QTL mapping procedure. (B) Differential seedling greening of Sha and Tsu-1 parental lines grown on control or salt medium. Pictures were taken 4 d after germination." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-128051212" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of seedling greening in response to salt quantitative trait locus (QTL) by the use of target-enriched extreme QTL (TEX-QTL) mapping with F2 populations of Sha &#xD7; Tsu-1 &lt;em&gt;Arabidopsis thaliana&lt;/em&gt; ecotypes. (A) A schematic of the TEX-QTL mapping procedure. (B) Differential seedling greening of Sha and Tsu-1 parental lines grown on control or salt medium. Pictures were taken 4 d after germination.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/67/F2.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/1/67/F2.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/67/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/67/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467442" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Identification of seedling greening in response to salt quantitative trait locus (QTL) by the use of target-enriched extreme QTL (TEX-QTL) mapping with F2 populations of Sha × Tsu-1 <em>Arabidopsis thaliana</em> ecotypes. (A) A schematic of the TEX-QTL mapping procedure. (B) Differential seedling greening of Sha and Tsu-1 parental lines grown on control or salt medium. Pictures were taken 4 d after germination.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">For the seed size trait, Tsu-1 seeds are significantly larger than Sha seeds (<em>P</em> &lt; 0.001, one-way analysis of variance test) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS2.xlsx">File S2</a>). We screened 140,000 F2 seeds from Sha and Tsu-1 crosses and used the two-tailed bulk design to select top and bottom 7% as the high and low bulks. Interestingly, a significant maternal effect on seed mass was observed in F1 but not in F2 seeds (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS5.xlsx">File S5</a>). When Tsu-1 was used as the female in crosses, the resulting F1 and F2 seeds were heavier than those seeds descended from crosses where Sha was the female. We therefore mapped QTL with F2 populations generated from reciprocal crosses.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467377/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467377/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Mass and size of seeds and progenitors used for QTL analysis</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">For seedling greening in response to salt, a one-tailed bulk design was employed because of the difficulty in obtaining the salt sensitive bulk. We first screened 40 <em>Arabidopsis thaliana</em> accessions for their ability to germinate and establish green seedlings when grown with 150 mM NaCl in the media. The 40 accessions belong to the 64 core accessions collected world-wide representing a broad genetic diversity (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">McKhann <em>et al.</em> 2004</a>). Among the 40 accessions tested, Sha and Tsu-1 displayed contrasting seedling greening phenotype in the presence of 150 mM NaCl (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). We performed bulked segregant analysis by screening 16,000 F2 seeds derived from crossing Sha (female) and Tsu-1 (male). The high bulk consisted top 10% (1600) of seedlings with green cotyledons grown in 150 mM NaCl. The same number (1600) of seedlings from the control plate was selected as the control bulk.</p></div><div id="sec-13" class="subsection"><h3>Marker selection and quality assessment</h3><p id="p-30">The power analysis showed that sequencing depth is more important than marker density for bulked segregant analysis with a large F2 population. We therefore incorporated the target enrichment technique, which initially was developed to reduce genome-sequencing costs by sequencing only exons (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Summerer 2009</a>), to enrich and sequence only selected SNPs to achieve high sequencing depth desired for increased QTL detection power.</p><p id="p-31">We designed 110 bp probes against 2478 SNPs approximately evenly distributed across the genome. The probes were enriched and sequenced with Illumina HiSeq with an average sequencing depth of 2600× per probe for seedling greening in response to salt and 1610× per probe for seed size. Approximately 200 bp surrounding each probe was sequenced by paired-end sequencing, which resulted in the detection of 4587 SNP markers.</p><p id="p-32">Since the allelic status of the probe can influence the frequency of captured allelic sequence at heterozygous loci (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Asan <em>et al.</em> 2011</a>), we assessed the allelic bias of probe capture and the potential impact of the capture bias on TEX-QTL mapping. To test this, we extracted genomic DNA from Sha × Tsu-1 F1 plants from three biological replicates and subjected it to target enrichment and sequencing at an average depth of 2600× from each replicate. If the capture is unbiased, the expected allele frequency is 0.5 for both Sha and Tsu-1 alleles. We observed a range of distribution for allelic frequency (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS2.pdf">Figure S2</a>), indicating the presence of allelic bias during the capture process. However, more than 97% of the probes showed consistent allelic frequencies among the three independent F1 capture events (standard deviation &lt; 0.05). We discarded the probes that showed allelic frequency and capture bias outside of the 95% confidence interval of the expected mean of 0.5 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS2.pdf">Figure S2</a>). A total of 2425 probes and 4278 SNP markers were retained for further analysis.</p><p id="p-33">We next assessed the reproducibility of TEX-QTL mapping. Using three independent bulk selections of the salt response experiments, we calculated pair-wise correlations of the G-values of all the 4278 SNP markers between replicates. We observed good reproducibility between each pair of the replicates with R<sup>2</sup> = 0.63-0.66 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS3.pdf">Figure S3</a>). Recently, a smoothed version of G test (G′ test) was developed to account for the variation introduced by bulk selection and sequencing of the bulks (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>). After we applied the G′ test, the correlation between each pair of the replicates improved to R<sup>2</sup> = 0.90−0.94 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS3.pdf">Figure S3</a>).</p><p id="p-34">Because of the high overall reproducibility between the replicates, we tested whether the data from the three biological replicates could be combined to increase sequencing depth of the markers using repeated G-tests (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McDonald 2009</a>). For most of the SNP markers (3903/4278 for salt response and 3158/4278 for seed size), we observed no significant heterogeneity among the replicates. After removing the markers showing significant deviations among the replicates, the data from the biological replicates were pooled to generate the final QTL maps, which had an average sequencing depth of 7800× for the salt response trait and 6440× for the seed size trait.</p></div><div id="sec-14" class="subsection"><h3>Identification of statistically significant markers</h3><p id="p-35">The observed allele frequency of the markers deviates from the true allele frequency by two sources of variations: the sequencing coverage of the SNP markers and binominal sampling of two alleles in the bulk (<a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>), the latter being stochastic. Under the whole-genome sequencing scheme, the former is also stochastic, since the distribution of sequencing coverage is ideally Poisson (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lander and Waterman 1988</a>). Therefore, it is possible to simultaneously remove both types of the random variations by kernel smoothing (<a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>). However, sequencing coverage of the SNP markers captured by target enrichment did not follow a Poisson distribution (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS4.pdf">Figure S4</a>). The interquartile ranges (IQR) of the control bulk and high bulk of green seedling in response to salt are 5792 and 7105, respectively, indicating a highly nonuniform distribution compared to IQR = 119 for a Poisson distribution when the mean coverage is 7800×. Similarly, IQR of low and high bulk of seed size are 4321 and 4318, compared with IQR = 108 for a Poisson distribution when the mean coverage is 6440×.</p><p id="p-36">To address the sequencing depth variation of the markers, we normalized them using a minimum threshold (set as 5000× for the salt response trait and 3000× for the seed size trait). The SNP markers with sequencing coverage higher than the threshold were downscaled proportionally to the threshold and those with lower coverage were discarded. After the normalization, 2653 and 2850 markers were retained for the salt response and seed size traits, respectively.</p><p id="p-37">After normalizing for the sequencing depth, the binomial sampling variation was corrected by G′-test (<a id="xref-ref-28-8" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>). The G′ distributions of five chromosomes were plotted (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) and the false-discovery rate (FDR) was determined empirically. For each trait, the null distribution was calculated by performing G′ test among biological replicates. The distribution was then fitted to a log-normal distribution (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS5.pdf">Figure S5</a>). Based on the fitted null distribution, SNP markers with FDR &lt; 0.01 after we adjusted for multiple hypothesis testing were deemed significant (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Benjamini and Hochberg 1995</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/67/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mapping results using target-enriched extreme QTL. (A) Quantitative trait locus (QTL) map of Arabidopsis thaliana seedling greening in response to salt. (B) QTL map of Arabidopsis thaliana seed size. Black dots are observed G&#x2032; values of markers from three replicated experiments combined. The smoothing window of 36cM was used. Red dots are fitted G&#x2032; values based on the model-selection method. Blue bars show the support intervals of QTL peaks with 95% confidence. Dashed horizontal line shows the 1% false-discovery rate threshold of G&#x2032; values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-128051212" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mapping results using target-enriched extreme QTL. (A) Quantitative trait locus (QTL) map of &lt;em&gt;Arabidopsis thaliana&lt;/em&gt; seedling greening in response to salt. (B) QTL map of &lt;em&gt;Arabidopsis thaliana&lt;/em&gt; seed size. Black dots are observed G&#x2032; values of markers from three replicated experiments combined. The smoothing window of 36cM was used. Red dots are fitted G&#x2032; values based on the model-selection method. Blue bars show the support intervals of QTL peaks with 95% confidence. Dashed horizontal line shows the 1% false-discovery rate threshold of G&#x2032; values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/67/F3.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/1/67/F3.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/67/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/67/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467331" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Mapping results using target-enriched extreme QTL. (A) Quantitative trait locus (QTL) map of <em>Arabidopsis thaliana</em> seedling greening in response to salt. (B) QTL map of <em>Arabidopsis thaliana</em> seed size. Black dots are observed G′ values of markers from three replicated experiments combined. The smoothing window of 36cM was used. Red dots are fitted G′ values based on the model-selection method. Blue bars show the support intervals of QTL peaks with 95% confidence. Dashed horizontal line shows the 1% false-discovery rate threshold of G′ values.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Resolving QTL regions</h3><p id="p-39">Similar to a previous observation (<a id="xref-ref-28-9" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>), a large portion of the SNP markers have significant G′ values, due to the closely linked peaks on the same chromosome (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). To resolve linked QTL and estimate their location intervals, we developed a model selection approach to fit the observed G′ values based on a multiple-QTL model calculated from empirical recombination rates of <em>Arabidopsis</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Salomé <em>et al.</em> 2012</a>).</p><div id="sec-16" class="subsection"><h4>Model selection:</h4><p id="p-40">To locate the regions with a high likelihood of containing the QTL, we developed a method to find the multiple-QTL model that best fits the observed G′ curve. Here we call the SNP marker that is closest to the causal site as the QTL marker. When only a single QTL is present in a chromosome, the G′ curve is expected to be unimodal (<a id="xref-ref-28-10" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>) and we can locate the SNP marker that has the maximal G′score as the QTL marker:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-1.gif"/></noscript></span></span>However, when multiple QTL are present in a chromosome, the allele frequency of each marker is influenced by genetic linkage to the flanking QTL, making the intuitive detection of the number and positions of the QTL difficult (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, black lines). Here we used model selection to fit a multiple-QTL model to the observed G′ values. Backward elimination strategy (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman and Sen 2009</a>) was applied to search the set of QTL positions that can best explain the observed G’ values of all the SNP markers.</p><p>A QTL model for each chromosome consists of QTL markers <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-2.gif"/></noscript></span></span> with their locations and allele frequencies <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-3.gif"/></noscript></span></span>. The best model was selected using the following steps:</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-42">Step 1. The initial model included all the local maxima of the observed G′ curve as putative QTL markers. Local maximum was determined as a SNP marker <em>x</em> that has G′ value strictly higher than both of its flanking neighbors: <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-4.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-5.gif"/></noscript></span></span>. In addition, local minima of the absolute value of the first derivatives of G′ also were included as the putative QTL markers. The first derivative of a certain SNP marker was estimated by linear regression using SNP markers surrounding the site. The window size was the same as the one used in the G′-test. The one-to-one mappings of allele frequencies of the QTL markers to their G′ values were calculated based on the coverage, smoothing window of G′-test, and the bulk design. They were then used to map from G′ values to allele frequencies.</p></li><li id="list-item-2"><p id="p-43">Step 2. The allele frequencies of all remaining markers were calculated based on the allele frequencies of the flanking putative QTL markers and recombination rates using the three-locus model assuming no interference from cross-overs (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Liu 1997</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS7.pdf">File S7</a>). We used empirical recombination rates that were calculated based on crossover events averaged over a diverse set of F2 populations (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Salomé <em>et al.</em> 2012</a>). For markers between the ends of the chromosome and the QTL marker, two-locus model (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Liu 1997</a>) was applied conditioned on the allele frequency of the single flanking QTL marker. The computed allele frequencies of the non-QTL markers were used to derive estimated G′ scores of non-QTL markers.</p></li><li id="list-item-3"><p id="p-44">Step 3. The estimated G′ curve was evaluated against the observed G′ curve using goodness of fit by root mean squared error (RMSE):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-6.gif"/></noscript></span></span></p></li></ul><p>where <em>n</em> is the total number of SNP markers, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-7.gif"/></noscript></span></span> is the observed G’ value of the <em>i</em>th SNP marker, and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-8.gif"/></noscript></span></span> is the expected G’ value of the <em>i</em>th SNP marker.</p><ul class="list-simple " id="list-2"><li id="list-item-4"><p id="p-45">Step 4. Step-wise backward elimination was used to determine whether any QTL marker in the current model should be omitted to increase the goodness of fit. The number of parameters in the model was penalized using Bayesian Information Criterion (BIC) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Schwarz 1978</a>). Assuming that the deviations between the fitted model and observed G′ curve follow independent and identical normal distribution with a mean of zero, BIC could be written as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-9.gif"/></noscript></span></span></p></li></ul><p>where <em>n</em> is the total number of SNP markers and m is the number of QTL. If the BIC score decreased by removing a QTL marker, we updated the model by removing that marker. The backward elimination continued until BIC could not be improved any further.</p><ul class="list-simple " id="list-3"><li id="list-item-5"><p id="p-46">Step 5. The positions and allele frequencies of the QTL markers were refined to improve the goodness of fit. Each QTL was chosen and adjusted for both its allele frequency and position while keeping all the other QTL markers fixed. The adjustment was iterated for all QTL markers until no changes in any of the parameters resulted in a better fit.</p></li><li id="list-item-6"><p id="p-47">Step 6. We repeated steps 4 and 5 until no better model was found.</p></li></ul></div><div id="sec-17" class="subsection"><h4>Estimating the interval of QTL location:</h4><p id="p-48">To identify regions with a high likelihood to contain QTL, we assumed the deviations between the fitted model and observed G′ curve are independently and identically distributed:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-10.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-11.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-12.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-13.gif"/></noscript></span></span> are the observed and fitted G′ value of the SNP marker <em>i</em>, and <em>σ</em> is the standard deviation of <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-14.gif"/></noscript></span></span> For random variable <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-15.gif"/></noscript></span></span>,<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-16.gif"/></noscript></span></span>follows a <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-17.gif"/></noscript></span></span> distribution with n degrees of freedom, where n is the number of SNP markers. To rewrite the equation into the form of root mean square error (RMSE),<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-18.gif"/></noscript></span></span>follows a <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-19.gif"/></noscript></span></span> distribution with n degrees of freedom. Using <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-20.gif"/></noscript></span></span>probability distribution, we can define the 95% confidence interval of RMSE as:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-21.gif"/></noscript></span></span>where <em>α</em> is 95% here (and could be changed to other values as well). Rearranging the formula, we will get the interval of <em>σ</em> as:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/67/embed/mml-math-22.gif"/></noscript></span></span>Therefore, we estimated the confidence interval of each QTL peak to include all the nearby markers with smaller RMSE than the upper bound of the best model's 95% confidence interval when all the other parameters were fixed.</p></div><div id="sec-18" class="subsection"><h4>Window size determination:</h4><p id="p-49">The model selection method is sensitive to the smoothing window size of G′-test in two ways. First, smaller smoothing windows would leave more noise and potentially lead to false-positive peaks. On the other hand, larger smoothing windows would cause a decrease of resolution in peak identification because multiple peaks within the same smoothing window are likely to merge and become indistinguishable. Although a window size of 24–37 cM was suggested previously (<a id="xref-ref-28-11" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>), it is not clear what the best choice would be in terms of the model selection process. Here we used simulations to find the most appropriate window size to balance the number of false peaks with the possibility of merging closely lined peaks.</p><p id="p-50">To estimate the probability of identifying false peaks using different smoothing window sizes, we simulated a single QTL flanked by 50 cM on both sides. The probability of identifying false peaks was defined as the total number of false peaks found by the model selection process divided by the number of simulations, 100. As the window size increases, FDR decreased linearly (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/67/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessing false-positive and false-negative rates of quantitative trait locus (QTL) detection by using simulations. (A) The probability of identifying false peaks for a 1-QTL chromosome using different smoothing window sizes. One hundred simulations were carried out for each data point. (B) The probability of missing true peaks and identifying false peaks for a two-QTL chromosome with various distances between the QTL. The two QTL have equal contribution to the phenotypic variation. The smoothing window is fixed at 36 cM. One hundred simulations were carried out for each data point." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-128051212" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Assessing false-positive and false-negative rates of quantitative trait locus (QTL) detection by using simulations. (A) The probability of identifying false peaks for a 1-QTL chromosome using different smoothing window sizes. One hundred simulations were carried out for each data point. (B) The probability of missing true peaks and identifying false peaks for a two-QTL chromosome with various distances between the QTL. The two QTL have equal contribution to the phenotypic variation. The smoothing window is fixed at 36 cM. One hundred simulations were carried out for each data point.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/67/F4.medium.gif" width="440" height="207"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/1/67/F4.medium.gif" width="440" height="207"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/67/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/67/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467475" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-51" class="first-child">Assessing false-positive and false-negative rates of quantitative trait locus (QTL) detection by using simulations. (A) The probability of identifying false peaks for a 1-QTL chromosome using different smoothing window sizes. One hundred simulations were carried out for each data point. (B) The probability of missing true peaks and identifying false peaks for a two-QTL chromosome with various distances between the QTL. The two QTL have equal contribution to the phenotypic variation. The smoothing window is fixed at 36 cM. One hundred simulations were carried out for each data point.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">To estimate the detection limit of closely linked QTL, we simulated two equally contributing QTL linked on a 100 cM chromosome. We simulated the detectability of both QTL with various distances set between the two QTL. From the one-QTL simulations, we chose 36 cM as the smoothing window to minimize the false peaks. When the distance was less than 16 cM, the two QTL were not distinguishable and only one peak could be detected by TEX-QTL mapping (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). When the distance was between 16–24 cM, the chance of merging decreased rapidly from 100 to less than 10%. When the distance was more than 24 cM, two peaks were most likely to be separable with the probability of less than 10%. We also tested the probability of detecting false peaks with the two-QTL simulations. They were 0 for all configurations, proving the window size of 36 cM is sufficient to minimize the noise of the G′ curve.</p><p id="p-53">In summary, our simulations suggested that a larger window size (16−36 cM) decreases the chance of detecting false peaks. Our choice of 36 cM smoothing window achieves the lowest false detection rate within the range of our simulations. However, the limitation of our approach is the high chance of detecting two close-linked peaks (<em>e.g.</em>, two peaks with 20 cM or smaller distance) as a single merged peak.</p></div></div><div id="sec-19" class="subsection"><h3>TEX-QTL mapping on two complex traits</h3><p id="p-54">Using TEX-QTL mapping, we identified several previously unidentified QTL as well as peaks that were previously reported (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). For the greening in response to salt trait, the Sha allele increased salt tolerance for eight of nine QTL (GRS1−7, GRS9). The Tsu-1 allele contributed to salt tolerance for one QTL on chromosome 4 (GRS8). Comparing our QTL with the published salt-tolerance QTL revealed that only one QTL overlapped with the previously reported QTL. GRS1 contains <em>RAS1</em> (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Ren <em>et al.</em> 2010</a>), previously identified as a causal gene for a QTL controlling variation in salt tolerance during early seedling development between Sha and Landsberg <em>erecta</em> (L<em>er</em>) (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Ren <em>et al.</em> 2010</a>). Recently, it was shown to be a microProtein that regulates the TGA1 transcription factor (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Magnani <em>et al.</em> 2014</a>) (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467402/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467402/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparison of GRS QTL for Tsu-1 × Sha mapped in this study and salt tolerance QTL reported in the literature</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467454/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467454/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Comparison of SSQ QTL mapped for Tsu-1 × Sha in this study and the L<em>er</em> × Cvi QTL described by Alonso Blanco <em>et al.</em> (1999)</span><div class="sb-div caption-clear"></div></div></div><p id="p-57">For the seed size trait, even though maternal effects were observed (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), no QTL showed parent-of-origin effects. Although seeds size QTL were reported previously (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Alonso-Blanco <em>et al.</em> 1999</a>), this study used different accessions. Nonetheless five SSQs overlapped with previously identified QTL (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Recently the QTL intervals reported by <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Alonso-Blanco <em>et al.</em> (1999)</a> were refined (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Moore <em>et al.</em> 2013</a>), but because the genetic markers used for this study have not been placed on the combined genetic and physical maps, we cannot determine whether the recently refined peaks overlap with the SSQ intervals. <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Moore <em>et al.</em> (2013)</a> analyzed three L<em>er</em> × Cvi mapping populations and reported that some of these QTL overlapped with those found by Alonso-Blanco (bold-face intervals in <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). The differences between these four existing L<em>er</em> × Cvi mapping populations were attributed to genotype × environment interactions that were revealed when plants were grown under different environmental conditions (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Moore <em>et al.</em> 2013</a>). The interval for SSQ1 corresponds to a previously reported QTL that accounts for 10–15% of the variation in seed size (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). SSQ5, SSQ9, SSQ10, and SSQ11 overlap with previously reported QTL, each of which accounts for less than 5% or 5–10% of the variability (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). SSQ2, SSQ3, SSQ4, SSQ6, SSQ7, SSQ8, and SSQ12 are novel QTL. <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Alonso-Blanco <em>et al.</em> (1999)</a> found two seed size QTL on chromosome 3, but these were not identified in this study. This could be a problem of detection power, but more likely it is a difference in alleles among the different accessions or epistatic interactions that permitted loci to be found in RIL lines, but not in segregating F2 populations.</p></div><div id="sec-20" class="subsection"><h3>Assessing the effect of sequencing depth on QTL detection power</h3><p id="p-58">Using the empirically derived sequence data, we evaluated the effect of sequencing depth on QTL detectability for the greening in response to salt trait. We iteratively sampled 10- and 100-fold fewer reads at random (1000 iterations for each sequencing depth). Reducing the sequencing depth by 10-fold (500× sequencing depth) resulted in a QTL map similar to the one with 5000× sequencing depth, except for GRS6. For a trait with a heritability of 0.94, we expect a sequencing depth of 500× to detect a QTL of at least 5% effect ~70% of the time (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1E</a>). Therefore, the observed detection power (8/9 QTL) is consistent with the predictions. Reducing the sequencing depth by 100-fold (50X sequencing depth) yielded no significant peaks (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5B</a>). The <em>in silico</em> sampling experiments demonstrate the necessity of high sequencing depth for increasing QTL detection in bulked segregant analysis.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/67/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulated mapping derived from subsampling the empirical target-enriched extreme QTL mapping results of Arabidopsis thaliana seedling greening in response to salt. (A) Simulated mapping result for sequencing depth of 500&#xD7;. (B) Simulated mapping result for sequencing depth 50&#xD7;. Dashed horizontal lines are 1% false-discovery rate threshold for G&#x2032; values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-128051212" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Simulated mapping derived from subsampling the empirical target-enriched extreme QTL mapping results of &lt;em&gt;Arabidopsis thaliana&lt;/em&gt; seedling greening in response to salt. (A) Simulated mapping result for sequencing depth of 500&#xD7;. (B) Simulated mapping result for sequencing depth 50&#xD7;. Dashed horizontal lines are 1% false-discovery rate threshold for G&#x2032; values.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/67/F5.medium.gif" width="440" height="331"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/1/67/F5.medium.gif" width="440" height="331"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/67/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/67/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467369" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-59" class="first-child">Simulated mapping derived from subsampling the empirical target-enriched extreme QTL mapping results of <em>Arabidopsis thaliana</em> seedling greening in response to salt. (A) Simulated mapping result for sequencing depth of 500×. (B) Simulated mapping result for sequencing depth 50×. Dashed horizontal lines are 1% false-discovery rate threshold for G′ values.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-60">Here we developed a QTL mapping approach called TEX-QTL analysis, which increases the QTL detection power by incorporating deep sequencing of target-enriched SNP markers with bulked segregant analysis. Although bulked segregant analysis with large population size (X-QTL mapping) can identify many QTL with small effect sizes and additive effects (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Lai <em>et al.</em> 2007</a>; <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Ehrenreich <em>et al.</em> 2010</a>), approaches that can increase the detection power of X-QTL have not been explored fully. To date, X-QTL mapping mainly has used whole-genome sequencing or SNP arrays to capture allelic frequency differences of SNP markers (<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Ehrenreich <em>et al.</em> 2010</a>; <a id="xref-ref-28-12" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Yang <em>et al.</em> 2013</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pais <em>et al.</em> 2014</a>). SNP arrays are limited by the number of SNP markers that could be spotted on the array. In contrast, whole-genome sequencing detects a majority of the SNPs in the genome but usually has relatively low sequencing depth per SNP marker to be cost-effective. More recently, exome capture was used to reduce the representation of genome sequencing for X-QTL mapping with relatively low sequencing depth (95−366×) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chevalier <em>et al.</em> 2014</a>). In theory, target enrichment and deep sequencing of SNP markers should permit the high sequencing depth needed for increased detection power. Our simulations suggested that by combining a large F2 population size, deeply sequenced markers, and 10–20% bulk size, we could in theory identify most QTL within two generations (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS1.R">File S1</a>). We empirically validated these theoretical predictions on two traits in <em>Arabidopsis thaliana</em>: 1) seed size and 2) seedling greening in response to salt.</p><p id="p-61">Both our empirical and simulation results indicated increased detection power with TEX-QTL mapping. By sampling subpopulations of the sequence reads, we demonstrated that the higher detection power can be achieved with increased sequencing depth (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Despite the reported multiple QTL mapping studies with different natural accessions on salt tolerance (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Quesada <em>et al.</em> 2002</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Clerkx <em>et al.</em> 2004</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rus <em>et al.</em> 2006</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Galpaz and Reymond 2010</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Ren <em>et al.</em> 2010</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Vallejo <em>et al.</em> 2010</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Derose-Wilson and Gaut 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Roy <em>et al.</em> 2013</a>), this study detected several new QTL peaks. These newly identified QTL might be due to an increased QTL mapping power using this TEX-QTL approach. It is also possible that the new QTL might reflect the unique genetic components contributing to the phenotypic variation of the mapping population derived from Sha × Tsu-1. Comparing TEX-QTL and traditional RIL-based QTL mapping by using populations derived from same parental lines should help address this issue in the future.</p><p id="p-62">We developed a statistical pipeline to identify TEX-QTL peaks along with interval estimates of their locations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1/FileS6.py">File S6</a>). In the framework of whole-genome sequencing-based bulked segregant analysis, several statistical methods have been developed recently, including the smoothed G test (<a id="xref-ref-28-13" class="xref-bibr" href="#ref-28">Magwene <em>et al.</em> 2011</a>), a dynamic Bayesian network (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Edwards and Gifford 2012</a>), and a Hidden Markov Model (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Duitama <em>et al.</em> 2014</a>). Application of the latter two methods to TEX-QTL mapping data yielded poorly resolved QTL regions (data not shown), indicating that these approaches might not be suitable for TEX-QTL mapping. The smoothed G test uses neighboring SNPs to reduce the random noises associated with the sequencing reads, but doesn’t identify putative QTL peaks and their support intervals. Therefore, we developed and applied a model-based method, which resolved linked QTL regions into more discrete peaks (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-63">One limitation of the current method is the detection of closely linked QTL. As suggested in the simulation, two very closely linked QTL may be identified as one QTL. Because the support intervals are calculated under the assumption of one causal gene, and the center of the interval would be positioned between two closely linked QTL, the interval may not include both causal genes. It may be helpful to examine the loci flanking outside of the support interval if no causal gene can be found within the interval.</p><p id="p-64">Another limitation of TEX-QTL mapping, similar to that of bulked segregant analysis in general, is that the allele frequency of the bulks does not permit the detection of epistasis between QTL or calculation of the contribution of each QTL to the phenotypic variation. Nevertheless, TEX-QTL mapping is advantageous in that it offers a quick and cost-effective way to identify a theoretically maximal number of QTL (QTL with both small and large effects) contributing to the variation of a trait.</p><p id="p-65">Here we demonstrated that TEX-QTL mapping increases the resolution of complex traits rapidly and reveals genetic architectures in a cost-effective manner. TEX-QTL mapping is applicable to many species such as insects, fish, frogs, or any other organisms where genetic crosses can be made and large F2 populations can be generated. It is particularly well suited to species with large genomes, for which obtaining the high sequencing depth needed to achieve increased QTL detection power is not practical with whole genome sequencing. This approach could accelerate the discovery of genetic architectures underlying complex traits and provides a powerful way to harness the genetic potential of germplasm to increase crop or livestock performance through marker-assisted breeding or genetic engineering.</p></div><div class="section" id="sec-22"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-66">We thank K. Dreher, C. Trontin, and I. Baxter for discussions and L. Peterson for initiating the simulation work. This work was supported in part by grants from the National Science Foundation (MCB-1052348 and IOS-1026003) and Department of Energy (BER65472). llumina sequencing services were performed by the Stanford Center for Genomics and Personalized Medicine.</p><p id="p-67">J.G. and S.Y.R. conceived the TEX-QTL mapping strategy and salt tolerance QTL mapping. B.H. and S.Y.R. conceived the seed size QTL mapping. J.G. designed probes. J.F. developed statistical approaches to resolve QTL peaks and estimate their support intervals. S.Y.R. generated plant material. B.H. generated segregating bulks and DNA for seed size TEX-QTL mapping and measured seed size and mass. J.G. performed all other experiments including surveying accessions for salt tolerance, salt tolerance bulked segregant analysis, DNA library preparation, and target enrichment. J.F. conducted all computational work including simulations, statistical analysis, and sequence data analysis. J.G., J.F., B.H., and S.Y.R. wrote the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-6"><p id="p-68">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023671/-/DC1</a></p></li><li class="fn" id="fn-7"><p id="p-69"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-08-17"><span class="received-label">Received </span>August 17, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-10-28"><span class="accepted-label">Accepted </span>October 28, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Guo <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-6.1.67.1"><div class="cit-metadata unstructured">Agilent, 2013 SureSelectXT target enrichment system for Illumina paired-end sequencing library. Available at: <a href="http://www.chem.agilent.com/library/usermanuals/Public/G7530&amp;#x2013;90000_SureSelect_IlluminaXTMultiplexed_1.6.pdf">http://www.chem.agilent.com/library/usermanuals/Public/G7530–90000_SureSelect_IlluminaXTMultiplexed_1.6.pdf</a>. Accessed November 10, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-2" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.2" data-doi="10.1073/pnas.96.8.4710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Blanco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankestijn-de Vries</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanhart</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koornneef</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Natural allelic variation at seed size loci in relation to other life history traits of <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">4710</span>–<span class="cit-lpage">4717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DAlonso-Blanco%26rft.auinit1%253DC.%26rft.volume%253D96%26rft.issue%253D8%26rft.spage%253D4710%26rft.epage%253D4717%26rft.atitle%253DNatural%2Ballelic%2Bvariation%2Bat%2Bseed%2Bsize%2Bloci%2Bin%2Brelation%2Bto%2Bother%2Blife%2Bhistory%2Btraits%2Bof%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.8.4710%26rft_id%253Dinfo%253Apmid%252F10200327%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ni84LzQ3MTAiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.3" data-doi="10.1186/gb-2011-12-9-r95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyler-Smith</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xue</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comprehensive comparison of three commercial human whole-exome capture platforms.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAsan%26rft.auinit1%253DN.%2BF.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253DR95%26rft.epage%253DR95%26rft.atitle%253DComprehensive%2Bcomparison%2Bof%2Bthree%2Bcommercial%2Bhuman%2Bwhole-exome%2Bcapture%2Bplatforms.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-9-r95%26rft_id%253Dinfo%253Apmid%252F21955857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-9-r95&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21955857&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB.%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.67.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">A Guide to QTL Mapping With R/qtl</span>. <span class="cit-publ-name">Springer Dordrecht Heidelberg</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.6" data-doi="10.1126/science.1174276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetic architecture of maize flowering time.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">714</span>–<span class="cit-lpage">718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBuckler%26rft.auinit1%253DE.%2BS.%26rft.volume%253D325%26rft.issue%253D5941%26rft.spage%253D714%26rft.epage%253D718%26rft.atitle%253DThe%2BGenetic%2BArchitecture%2Bof%2BMaize%2BFlowering%2BTime%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1174276%26rft_id%253Dinfo%253Apmid%252F19661422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTk0MS83MTQiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.7" data-doi="10.1186/1471-2164-15-617"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chevalier</span>  <span class="cit-name-given-names">F. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valentim</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LoVerde</span>  <span class="cit-name-given-names">P. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient linkage mapping using exome capture and extreme QTL in schistosome parasites.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D617%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-617%26rft_id%253Dinfo%253Apmid%252F25048426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-617&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25048426&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.8" data-doi="10.1126/science.1138632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweikert</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeller</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Common sequence polymorphisms shaping genetic diversity in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">338</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DClark%26rft.auinit1%253DR.%2BM.%26rft.volume%253D317%26rft.issue%253D5836%26rft.spage%253D338%26rft.epage%253D342%26rft.atitle%253DCommon%2BSequence%2BPolymorphisms%2BShaping%2BGenetic%2BDiversity%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1138632%26rft_id%253Dinfo%253Apmid%252F17641193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTgzNi8zMzgiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-2" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.9" data-doi="10.1104/pp.103.036814"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clerkx</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Lithy</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vierling</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruys</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blankestijn-De Vries</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Analysis of natural allelic variation of <em>Arabidopsis</em> seed germination and seed longevity traits between the accessions Landsberg <em>erecta</em> and Shakdara, using a new recombinant inbred line population.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">432</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.103.036814%26rft_id%253Dinfo%253Apmid%252F15122038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzUvMS80MzIiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.10" data-doi="10.2307/2286407"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cleveland</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Robust locally weighted regression and smoothing scatterplots.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D74%26rft.spage%253D829%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2286407%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2286407&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1979JD82100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.11" data-doi="10.1371/journal.pone.0022832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeRose-Wilson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping salinity tolerance during <em>Arabidopsis thaliana</em> germination and seedling growth.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e22832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDerose-Wilson%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253De22832%26rft.epage%253De22832%26rft.atitle%253DMapping%2Bsalinity%2Btolerance%2Bduring%2BArabidopsis%2Bthaliana%2Bgermination%2Band%2Bseedling%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0022832%26rft_id%253Dinfo%253Apmid%252F21857956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0022832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21857956&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.12" data-doi="10.1186/1471-2164-15-207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duitama</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Rodriguez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goovaerts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pulido-Tamayo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubmann</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Improved linkage analysis of quantitative trait loci using bulk segregants unveils a novel determinant of high ethanol tolerance in yeast.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D207%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-207%26rft_id%253Dinfo%253Apmid%252F24640961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-207&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24640961&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.13" data-doi="10.1186/1471-2105-13-S14-S8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping with pooled sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>(<span class="cit-supplement">Suppl 6</span>)<strong>:</strong> <span class="cit-fpage">S8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D13%26rft.spage%253DS8%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-S14-S8%26rft_id%253Dinfo%253Apmid%252F23282012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-S14-S8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23282012&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.14" data-doi="10.1038/nature08923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torabi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dissection of genetically complex traits with extremely large pools of yeast segregants.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">1039</span>–<span class="cit-lpage">1042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DEhrenreich%26rft.auinit1%253DI.%2BM.%26rft.volume%253D464%26rft.issue%253D7291%26rft.spage%253D1039%26rft.epage%253D1042%26rft.atitle%253DDissection%2Bof%2Bgenetically%2Bcomplex%2Btraits%2Bwith%2Bextremely%2Blarge%2Bpools%2Bof%2Byeast%2Bsegregants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08923%26rft_id%253Dinfo%253Apmid%252F20393561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20393561&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276635000038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.15" data-doi="10.1371/journal.pone.0015198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galpaz</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reymond</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Natural variation in <em>Arabidopsis thaliana</em> revealed a genetic network controlling germination under salt stress.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e15198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGalpaz%26rft.auinit1%253DN.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De15198%26rft.epage%253De15198%26rft.atitle%253DNatural%2Bvariation%2Bin%2BArabidopsis%2Bthaliana%2Brevealed%2Ba%2Bgenetic%2Bnetwork%2Bcontrolling%2Bgermination%2Bunder%2Bsalt%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015198%26rft_id%253Dinfo%253Apmid%252F21188145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21188145&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.67.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">A. J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gelbart</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">An Introduction To Genetic Analysis</span>. <span class="cit-publ-name">W. H. Freeman</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.17" data-doi="10.1101/gr.129684.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koboldt</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLellan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">568</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.129684.111%26rft_id%253Dinfo%253Apmid%252F22300766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNTY4IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.18" data-doi="10.1186/1746-4811-9-29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farlow</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The advantages and limitations of trait analysis with GWAS: a review.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D9%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1746-4811-9-29%26rft_id%253Dinfo%253Apmid%252F23876160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1746-4811-9-29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23876160&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.19" data-doi="10.1093/bioinformatics/btp699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kvilekval</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedorov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obara</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manjunath</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bisque: a platform for bioimage analysis and management.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">544</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp699%26rft_id%253Dinfo%253Apmid%252F20031971%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi80LzU0NCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS82Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.20" data-doi="10.1038/nmeth1084"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leips</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wollenberg</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Speed-mapping quantitative trait loci using microarrays.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLai%26rft.auinit1%253DC.%2BQ.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253D839%26rft.epage%253D841%26rft.atitle%253DSpeed-mapping%2Bquantitative%2Btrait%2Bloci%2Busing%2Bmicroarrays.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth1084%26rft_id%253Dinfo%253Apmid%252F17873888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth1084&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17873888&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249778200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.21" data-doi="10.1093/nar/gkr1090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamesch</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berardini</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarbreck</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1202</span>–<span class="cit-lpage">D1210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1090%26rft_id%253Dinfo%253Apmid%252F22140109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0MC9EMS9EMTIwMiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS82Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.22" data-doi="10.1016/0888-7543(88)90007-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterman</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Genomic mapping by fingerprinting random</span> <span class="cit-article-title">clones: a mathematical analysis.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">239</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253D231%26rft.epage%253D239%26rft.atitle%253DGenomic%2Bmapping%2Bby%2Bfingerprinting%2Brandom%2Bclones%253A%2Ba%2Bmathematical%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252888%252990007-9%26rft_id%253Dinfo%253Apmid%252F3294162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(88)90007-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3294162&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.23" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.24" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sequence Alignment/Map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS82Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.67.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-source"><em>Statistical Genomics: Linkage</em>, <em>Mapping</em>, <em>and QTL Analysis</em></span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Florida</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.26" data-doi="10.1038/nrg2612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetics of quantitative traits: challenges and prospects.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">577</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253D565%26rft.epage%253D577%26rft.atitle%253DThe%2Bgenetics%2Bof%2Bquantitative%2Btraits%253A%2Bchallenges%2Band%2Bprospects.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2612%26rft_id%253Dinfo%253Apmid%252F19584810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19584810&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268038200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.27" data-doi="10.1104/pp.114.235903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magnani</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Klein</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nam</span>  <span class="cit-name-given-names">H. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A comprehensive analysis of microProteins reveals their potentially widespread mechanism of transcriptional regulation.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.114.235903%26rft_id%253Dinfo%253Apmid%252F24616380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjUvMS8xNDkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.28" data-doi="10.1371/journal.pcbi.1002255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Magwene</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The statistics of bulk segregant analysis using next generation sequencing.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DMagwene%26rft.auinit1%253DP.%2BM.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002255%26rft.epage%253De1002255%26rft.atitle%253DThe%2Bstatistics%2Bof%2Bbulk%2Bsegregant%2Banalysis%2Busing%2Bnext%2Bgeneration%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002255%26rft_id%253Dinfo%253Apmid%252F22072954%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072954&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.67.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McDonald</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">Handbook of Biological Statistics</span>. <span class="cit-publ-name">Sparky House Publishing</span>, <span class="cit-publ-loc">Baltimore, Maryland</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.30" data-doi="10.1111/j.1365-313X.2004.02034.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKhann</span>  <span class="cit-name-given-names">H. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camilleri</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bataillon</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Nested core collections maximizing genetic diversity in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DMcKhann%26rft.auinit1%253DH.%2BI.%26rft.volume%253D38%26rft.issue%253D1%26rft.spage%253D193%26rft.epage%253D202%26rft.atitle%253DNested%2Bcore%2Bcollections%2Bmaximizing%2Bgenetic%2Bdiversity%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2004.02034.x%26rft_id%253Dinfo%253Apmid%252F15053772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2004.02034.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15053772&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220521800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.31" data-doi="10.1023/B:EUPH.0000040498.48379.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Utz</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schon</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">QTL analyses of complex traits with cross validation, bootstrapping and other biometric methods.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D137%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253AEUPH.0000040498.48379.68%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:EUPH.0000040498.48379.68&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000223738600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.32" data-doi="10.1073/pnas.88.21.9828"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michelmore</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paran</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesseli</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Identification of markers linked to disease-resistance genes by bulked segregant analysis: a rapid method to detect markers in specific genomic regions by using segregating populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">9828</span>–<span class="cit-lpage">9832</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMichelmore%26rft.auinit1%253DR.%2BW.%26rft.volume%253D88%26rft.issue%253D21%26rft.spage%253D9828%26rft.epage%253D9832%26rft.atitle%253DIdentification%2Bof%2Bmarkers%2Blinked%2Bto%2Bdisease-resistance%2Bgenes%2Bby%2Bbulked%2Bsegregant%2Banalysis%253A%2Ba%2Brapid%2Bmethod%2Bto%2Bdetect%2Bmarkers%2Bin%2Bspecific%2Bgenomic%2Bregions%2Bby%2Busing%2Bsegregating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.88.21.9828%26rft_id%253Dinfo%253Apmid%252F1682921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODgvMjEvOTgyOCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS82Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-2" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.33" data-doi="10.1534/g3.112.003806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gronwall</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spalding</span>  <span class="cit-name-given-names">E. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping quantitative trait loci affecting <em>Arabidopsis thaliana</em> seed morphology features extracted computationally from images.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DMoore%26rft.auinit1%253DC.%2BR.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D118%26rft.atitle%253DMapping%2BQuantitative%2BTrait%2BLoci%2BAffecting%2BArabidopsis%2Bthaliana%2BSeed%2BMorphology%2BFeatures%2BExtracted%2BComputationally%2BFrom%2BImages%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003806%26rft_id%253Dinfo%253Apmid%252F23316443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMS8xMDkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.34" data-doi="10.1126/science.1180677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas-Lledo</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warthmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The rate and molecular spectrum of spontaneous mutations in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DOssowski%26rft.auinit1%253DS.%26rft.volume%253D327%26rft.issue%253D5961%26rft.spage%253D92%26rft.epage%253D94%26rft.atitle%253DThe%2BRate%2Band%2BMolecular%2BSpectrum%2Bof%2BSpontaneous%2BMutations%2Bin%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180677%26rft_id%253Dinfo%253Apmid%252F20044577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMjcvNTk2MS85MiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS82Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.35" data-doi="10.1007/978-1-4939-0563-8_15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pais</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulquie-Moreno</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thevelein</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL mapping by pooled-segregant whole-genome sequencing in yeast.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1152</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1152%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-0563-8_15%26rft_id%253Dinfo%253Apmid%252F24744038%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-0563-8_15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24744038&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.36" data-doi="10.1534/genetics.113.159152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genetic architecture of maize height.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1337</span>–<span class="cit-lpage">1356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159152%26rft_id%253Dinfo%253Apmid%252F24514905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzEzMzciO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.37" data-doi="10.1038/ni.1747"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plomin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haworth</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">O. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Common disorders are quantitative traits.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">872</span>–<span class="cit-lpage">878</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPlomin%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.issue%253D12%26rft.spage%253D872%26rft.epage%253D878%26rft.atitle%253DCommon%2Bdisorders%2Bare%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fni.1747%26rft_id%253Dinfo%253Apmid%252F19859063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ni.1747&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19859063&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271919200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.38" data-doi="10.1104/pp.006536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quesada</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Martinez</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piqueras</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponce</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Micol</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic architecture of NaCl tolerance in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">951</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.006536%26rft_id%253Dinfo%253Apmid%252F12376659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzAvMi85NTEiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-web" id="cit-6.1.67.39"><div class="cit-metadata unstructured">R Core Team, 2013 R: A language and environment for statistical computing. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>. Accessed November 10, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.40" data-doi="10.1073/pnas.0910798107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chinnusamy</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">RAS1, a quantitative trait locus for salt tolerance and ABA sensitivity in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">5669</span>–<span class="cit-lpage">5674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0910798107%26rft_id%253Dinfo%253Apmid%252F20212128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA3LzEyLzU2NjkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.41" data-doi="10.1111/j.1365-3040.2012.02595.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roy</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evrard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmockel</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A novel protein kinase involved in Na(+) exclusion revealed from positional cloning.</span> <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BEnviron.%26rft.volume%253D36%26rft.spage%253D553%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3040.2012.02595.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3040.2012.02595.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000314187300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.42" data-doi="10.1371/journal.pgen.0020210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muthukumar</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gustin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Natural variants of AtHKT1 enhance Na(+) accumulation in two wild populations of Arabidopsis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DRus%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De210%26rft.epage%253De210%26rft.atitle%253DNatural%2Bvariants%2Bof%2BAtHKT1%2Benhance%2BNa%252B%2Baccumulation%2Bin%2Btwo%2Bwild%2Bpopulations%2Bof%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020210%26rft_id%253Dinfo%253Apmid%252F17140289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020210&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17140289&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.43" data-doi="10.1038/hdy.2011.95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salomé</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bomblies</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laitinen</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warthmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The recombination landscape in <em>Arabidopsis thaliana</em> F2 populations.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D108%26rft.spage%253D447%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.95%26rft_id%253Dinfo%253Apmid%252F22072068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.95&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072068&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.44" data-doi="10.1214/aos/1176344136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Estimating the dimension of a model.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D6%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978EQ63300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.45" data-doi="10.1016/j.ygeno.2009.08.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Summerer</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Enabling technologies of genomic-scale sequence enrichment for targeted high-throughput sequencing.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DSummerer%26rft.auinit1%253DD.%26rft.volume%253D94%26rft.issue%253D6%26rft.spage%253D363%26rft.epage%253D368%26rft.atitle%253DEnabling%2Btechnologies%2Bof%2Bgenomic-scale%2Bsequence%2Benrichment%2Bfor%2Btargeted%2Bhigh-throughput%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2009.08.012%26rft_id%253Dinfo%253Apmid%252F19720138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2009.08.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19720138&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272103600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-web" id="cit-6.1.67.46"><div class="cit-metadata unstructured">Technow, F., 2013 Hypred: Simulation of genomic data in applied genetics. <a href="http://cran.r-project.org/web/packages/hypred/vignettes/hypred.pdf">http://cran.r-project.org/web/packages/hypred/vignettes/hypred.pdf</a>. Accessed November 10, 2015.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-2" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.47" data-doi="10.1093/aob/mcq179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanovsky</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botto</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Germination variation in <em>Arabidopsis thaliana</em> accessions under moderate osmotic and salt stresses.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcq179%26rft_id%253Dinfo%253Apmid%252F20861274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjk6IjEwNi81LzgzMyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS82Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.67.48" data-doi="10.1371/journal.pone.0068433"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping of quantitative trait loci underlying cold tolerance in rice seedlings via high-throughput sequencing of pooled extremes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e68433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De68433%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0068433%26rft_id%253Dinfo%253Apmid%252F23935868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0068433&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23935868&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F67.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/1/67.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/59" title="Chemically Induced Oncogenesis in the Peripheral Nervous System Is Suppressed in Congenic BDIX.BDIV-Mss1 and -Mss7 Rats" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/79" title="Mapping Quantitative Trait Loci Underlying Function-Valued Traits Using Functional Principal Component Analysis and Multi-Trait Mapping" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/1"><span>Volume 6 Issue 1, January 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/1.cover-source.jpg?itok=9zFiZvCK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/67" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--2dei3iwE-7qznuBwm1kO5wx1bwXBQC3OjEsiXzVfNM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/1/67" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/1/67" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/1/67" class="active" data-icon-position="" data-hide-link-title="0">Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460248" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Bvg2GK-MK8vl5fNbfnBUj4PyQvZ2C87r2rn30m-7W64" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/67.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/1/67" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ODbBFinvYU19ACGW7ELCkLcl_SE0S9MNnFetCRHruwk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/1/67" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/67.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460248" id="node460248--2" data-pisa="ggg;6/1/67" data-pisa-master="ggg;g3.115.023671" data-apath="/ggg/6/1/67.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jianjun Guo</span>, <span class="highwire-citation-author" data-delta="1">Jue Fan</span>, <span class="highwire-citation-author" data-delta="2">Bernard A. Hauser</span> and <span class="highwire-citation-author" data-delta="3">Seung Y. Rhee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">67-77;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023671 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460248--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jianjun Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianjun%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianjun%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jue Fan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jue%2BFan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJue%2BFan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bernard A. Hauser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Plant Molecular and Cellular Biology Program, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernard%2BA.%2BHauser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hauser%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernard%2BA.%2BHauser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Seung Y. Rhee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seung%2BY.%2BRhee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rhee%20SY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASeung%2BY.%2BRhee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srhee@carnegiescience.edu" class="" data-icon-position="" data-hide-link-title="0">srhee@carnegiescience.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/1/67" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/1/67&amp;title=Target%20Enrichment%20Improves%20Mapping%20of%20Complex%20Traits%20by%20Deep%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/1/67&amp;title=Target%20Enrichment%20Improves%20Mapping%20of%20Complex%20Traits%20by%20Deep%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/1/67&amp;title=Target%20Enrichment%20Improves%20Mapping%20of%20Complex%20Traits%20by%20Deep%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/1/67&amp;text=Target%20Enrichment%20Improves%20Mapping%20of%20Complex%20Traits%20by%20Deep%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/1/67&amp;title=Target%20Enrichment%20Improves%20Mapping%20of%20Complex%20Traits%20by%20Deep%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/1/67&amp;t=Target%20Enrichment%20Improves%20Mapping%20of%20Complex%20Traits%20by%20Deep%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/1/67&amp;title=Target%20Enrichment%20Improves%20Mapping%20of%20Complex%20Traits%20by%20Deep%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/1/67&amp;title=Target%20Enrichment%20Improves%20Mapping%20of%20Complex%20Traits%20by%20Deep%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460248/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460248" id="node460248--3" data-pisa="ggg;6/1/67" data-pisa-master="ggg;g3.115.023671" data-apath="/ggg/6/1/67.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Target Enrichment Improves Mapping of Complex Traits by Deep Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jianjun Guo</span>, <span class="highwire-citation-author" data-delta="1">Jue Fan</span>, <span class="highwire-citation-author" data-delta="2">Bernard A. Hauser</span> and <span class="highwire-citation-author" data-delta="3">Seung Y. Rhee</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">67-77;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023671 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460248--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jianjun Guo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianjun%2BGuo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guo%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianjun%2BGuo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jue Fan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jue%2BFan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fan%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJue%2BFan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Bernard A. Hauser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Plant Molecular and Cellular Biology Program, University of Florida, Gainesville, Florida 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernard%2BA.%2BHauser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hauser%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernard%2BA.%2BHauser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Seung Y. Rhee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Seung%2BY.%2BRhee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rhee%20SY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASeung%2BY.%2BRhee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:srhee@carnegiescience.edu" class="" data-icon-position="" data-hide-link-title="0">srhee@carnegiescience.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=67&amp;rft.epage=77&amp;rft.atitle=Target%20Enrichment%20Improves%20Mapping%20of%20Complex%20Traits%20by%20Deep%20Sequencing&amp;rft.volume=6&amp;rft.issue=1&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Guo%2C+Jianjun&amp;rft.au=Fan%2C+Jue&amp;rft.au=Hauser%2C+Bernard+A.&amp;rft.au=Rhee%2C+Seung+Y."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460248/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460248/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460248/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460248/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460248/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460248/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460248/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460248/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460248/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460248/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460248/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460248/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/1/67" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/1/67.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/1/67.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/1/67.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/1/67.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
