<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/1/29" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/1/29.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/1/29.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/1/29.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/1/29.atom" />
<meta name="HW.pisa" content="ggg;6/1/29" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Haplotype Detection from Next-Generation Sequencing in High-Ploidy-Level Species: 45S rDNA Gene Copies in the Hexaploid Spartina maritima" />
<meta name="DC.Identifier" content="10.1534/g3.115.023242" />
<meta name="DC.Date" content="2016-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Boutte et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Gene and whole-genome duplications are widespread in plant nuclear genomes, resulting in sequence heterogeneity. Identification of duplicated genes may be particularly challenging in highly redundant genomes, especially when there are no diploid parents as a reference. Here, we developed a pipeline to detect the different copies in the ribosomal RNA gene family in the hexaploid grass Spartina maritima from next-generation sequencing (Roche-454) reads. The heterogeneity of the different domains of the highly repeated 45S unit was explored by identifying single nucleotide polymorphisms (SNPs) and assembling reads based on shared polymorphisms. SNPs were validated using comparisons with Illumina sequence data sets and by cloning and Sanger (re)sequencing. Using this approach, 29 validated polymorphisms and 11 validated haplotypes were reported (out of 34 and 20, respectively, that were initially predicted by our program). The rDNA domains of S. maritima have similar lengths as those found in other Poaceae, apart from the 5′-ETS, which is approximately two-times longer in S. maritima . Sequence homogeneity was encountered in coding regions and both internal transcribed spacers (ITS), whereas high intragenomic variability was detected in the intergenic spacer (IGS) and the external transcribed spacer (ETS). Molecular cytogenetic analysis by fluorescent in situ hybridization (FISH) revealed the presence of one pair of 45S rDNA signals on the chromosomes of S. maritima instead of three expected pairs for a hexaploid genome, indicating loss of duplicated homeologous loci through the diploidization process. The procedure developed here may be used at any ploidy level and using different sequencing technologies." />
<meta name="DC.Contributor" content="Julien Boutte" />
<meta name="DC.Contributor" content="Benoît Aliaga" />
<meta name="DC.Contributor" content="Oscar Lima" />
<meta name="DC.Contributor" content="Julie Ferreira de Carvalho" />
<meta name="DC.Contributor" content="Abdelkader Ainouche" />
<meta name="DC.Contributor" content="Jiri Macas" />
<meta name="DC.Contributor" content="Mathieu Rousseau-Gueutin" />
<meta name="DC.Contributor" content="Olivier Coriton" />
<meta name="DC.Contributor" content="Malika Ainouche" />
<meta name="DC.Contributor" content="Armel Salmon" />
<meta name="article:published_time" content="2016-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Haplotype Detection from Next-Generation Sequencing in High-Ploidy-Level Species: 45S rDNA Gene Copies in the Hexaploid Spartina maritima" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Gene and whole-genome duplications are widespread in plant nuclear genomes, resulting in sequence heterogeneity. Identification of duplicated genes may be particularly challenging in highly redundant genomes, especially when there are no diploid parents as a reference. Here, we developed a pipeline to detect the different copies in the ribosomal RNA gene family in the hexaploid grass &lt;i&gt;Spartina maritima&lt;/i&gt; from next-generation sequencing (Roche-454) reads. The heterogeneity of the different domains of the highly repeated 45S unit was explored by identifying single nucleotide polymorphisms (SNPs) and assembling reads based on shared polymorphisms. SNPs were validated using comparisons with Illumina sequence data sets and by cloning and Sanger (re)sequencing. Using this approach, 29 validated polymorphisms and 11 validated haplotypes were reported (out of 34 and 20, respectively, that were initially predicted by our program). The rDNA domains of &lt;i&gt;S. maritima&lt;/i&gt; have similar lengths as those found in other Poaceae, apart from the 5′-ETS, which is approximately two-times longer in &lt;i&gt;S. maritima&lt;/i&gt;. Sequence homogeneity was encountered in coding regions and both internal transcribed spacers (ITS), whereas high intragenomic variability was detected in the intergenic spacer (IGS) and the external transcribed spacer (ETS). Molecular cytogenetic analysis by fluorescent &lt;i&gt;in situ&lt;/i&gt; hybridization (FISH) revealed the presence of one pair of 45S rDNA signals on the chromosomes of &lt;i&gt;S. maritima&lt;/i&gt; instead of three expected pairs for a hexaploid genome, indicating loss of duplicated homeologous loci through the diploidization process. The procedure developed here may be used at any ploidy level and using different sequencing technologies.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="ggg;6/1/29" />
<meta name="citation_id" content="6/1/29" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/1/29" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/1/29.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/1/29.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/1/29.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.023242" />
<meta name="citation_pmid" content="26530424" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="29" />
<meta name="citation_lastpage" content="40" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Julien Boutte" />
<meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France" />
<meta name="citation_author" content="Benoît Aliaga" />
<meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France" />
<meta name="citation_author" content="Oscar Lima" />
<meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France" />
<meta name="citation_author" content="Julie Ferreira de Carvalho" />
<meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France" />
<meta name="citation_author" content="Abdelkader Ainouche" />
<meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France" />
<meta name="citation_author" content="Jiri Macas" />
<meta name="citation_author_institution" content="Biology Centre ASCR, Institute of Plant Molecular Biology, Branišovská 31, České Budějovice, CZ-37005, Czech Republic" />
<meta name="citation_author" content="Mathieu Rousseau-Gueutin" />
<meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France" />
<meta name="citation_author_institution" content="UMR Institut de Génétique, Environnement et Protection des Plantes, Institut National de la Recherche Agronomique, BP35327, 35653 Le Rheu Cedex, France" />
<meta name="citation_author" content="Olivier Coriton" />
<meta name="citation_author_institution" content="UMR Institut de Génétique, Environnement et Protection des Plantes, Institut National de la Recherche Agronomique, BP35327, 35653 Le Rheu Cedex, France" />
<meta name="citation_author" content="Malika Ainouche" />
<meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France" />
<meta name="citation_author" content="Armel Salmon" />
<meta name="citation_author_institution" content="UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France" />
<meta name="citation_author_email" content="armel.salmon@univ-rennes1.fr" />
<meta name="citation_reference" content="citation_title=Polyploidy and Genome Evolution;citation_title=Polyploid evolution in Spartina: Dealing with highly redundant hybrid genomes;citation_author=M. L. Ainouche;citation_author=H. Chelaifa;citation_author=J. Ferreira de Carvalho;citation_author=S. Bellot;citation_author=A. K. Ainouche;citation_pages=225-243;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Biol. Invasions;citation_author=M. L. Ainouche;citation_author=P. M. Fortune;citation_author=A. Salmon;citation_author=C. Parisod;citation_author=M.-A. Grandbastien;citation_title=Hybridization, polyploidy and invasion: lessons from Spartina (Poaceae).;citation_pages=1159-1173;citation_volume=11;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. R. Akhunova;citation_author=R. T. Matniyazov;citation_author=H. Liang;citation_author=E. D. Akhunov;citation_title=Homoeolog-specific transcriptional bias in allopolyploid wheat.;citation_pages=505-505;citation_volume=11;citation_year=2010;citation_pmid=20849627;citation_doi=10.1186/1471-2164-11-505" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schäffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=I. Álvarez;citation_author=J. F. Wendel;citation_title=Ribosomal ITS sequences and plant phylogenetic inference.;citation_pages=417-434;citation_volume=29;citation_year=2003;citation_issue=3;citation_pmid=14615184;citation_doi=10.1016/S1055-7903(03)00208-2" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=E. D. Badaeva;citation_author=B. Friebe;citation_author=B. S. Gill;citation_title=Genome differentiation in Aegilops. 2. Physical mapping of 5S and 18S-26S ribosomal RNA gene families in diploid species.;citation_pages=1150-1158;citation_volume=39;citation_year=1996;citation_issue=6;citation_pmid=18469963" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=R. F. Barker;citation_author=N. P. Harberd;citation_author=M. G. Jarvis;citation_author=R. B. Flavell;citation_title=Structure and evolution of the intergenic region in a ribosomal DNA repeat unit of wheat.;citation_pages=1-17;citation_volume=201;citation_year=1988;citation_issue=1;citation_pmid=3418693;citation_doi=10.1016/0022-2836(88)90434-2" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=A. Baumel;citation_author=M. L. Ainouche;citation_author=J. E. Levasseur;citation_title=Molecular investigations in populations of Spartina anglica C.E. Hubbard (Poaceae) invading coastal Brittany (France).;citation_pages=1689-1701;citation_volume=10;citation_year=2001;citation_issue=7;citation_pmid=11472537;citation_doi=10.1046/j.1365-294X.2001.01299.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=A. Baumel;citation_author=M. L. Ainouche;citation_author=R. J. Bayer;citation_author=A. K. Ainouche;citation_author=M. T. Misset;citation_title=Molecular phylogeny of hybridizing species from the genus Spartina Schreb. (Poaceae).;citation_pages=303-314;citation_volume=22;citation_year=2002;citation_issue=2;citation_pmid=11820850;citation_doi=10.1006/mpev.2001.1064" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=G. Bena;citation_author=M.-F. Jubier;citation_author=I. Olivieri;citation_author=B. Lejeune;citation_title=Ribosomal External and Internal Transcribed Spacers: Combined Use in the Phylogenetic Analysis of Medicago (Leguminosae);citation_pages=299-306;citation_volume=46;citation_year=1998;citation_issue=3;citation_pmid=9502673;citation_doi=10.1007/PL00006306" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. Chalhoub;citation_author=F. Denoeud;citation_author=S. Liu;citation_author=I. A. P. Parkin;citation_author=H. Tang;citation_title=Early allopolyploid evolution in the post-Neolithic Brassica napus oilseed genome;citation_pages=950-953;citation_volume=345;citation_year=2014;citation_issue=6199;citation_pmid=25146293;citation_doi=10.1126/science.1253435" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=K.-D. Chang;citation_author=S.-A. Fang;citation_author=F.-C. Chang;citation_author=M.-C. Chung;citation_title=Chromosomal conservation and sequence diversity of ribosomal RNA genes of two distant Oryza species.;citation_pages=181-190;citation_volume=96;citation_year=2010;citation_issue=3;citation_pmid=20580815;citation_doi=10.1016/j.ygeno.2010.05.005" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=M.-C. Combes;citation_author=A. Cenci;citation_author=H. Baraille;citation_author=B. Bertrand;citation_author=P. Lashermes;citation_title=Homeologous gene expression in response to growing temperature in a recent allopolyploid (Coffea arabica L.).;citation_pages=36-46;citation_volume=103;citation_year=2011;citation_pmid=22039298" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=W. Duchemin;citation_author=P.-Y. Dupont;citation_author=M. A. Campbell;citation_author=A. R. Ganley;citation_author=M. P. Cox;citation_title=HyLiTE: accurate and flexible analysis of gene expression in hybrid and allopolyploid species.;citation_pages=8;citation_volume=16;citation_year=2014" />
<meta name="citation_reference" content="citation_title=Next generation sequencing technologies and challenges in sequence assembly;citation_author=S. El-Metwally;citation_author=O. M. Ouda;citation_author=M. Helmy;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Arab J. Biotechnol.;citation_author=M. H. A. El-Twab;citation_title=Physical mapping of the 45S rDNA on the chromosomes of Triticum turgidum and T. aestivum using fluorescence in situ hybridization for chromosome ancestors.;citation_pages=69-80;citation_volume=10;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=H. Fan;citation_author=K. Yakura;citation_author=M. Miyanishi;citation_author=M. Sugita;citation_author=M. Sugiura;citation_title=In vitro transcription of plant RNA polymerase I-dependent rRNA genes is species-specific.;citation_pages=295-298;citation_volume=8;citation_year=1995;citation_issue=2;citation_pmid=7670508;citation_doi=10.1046/j.1365-313X.1995.08020295.x" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Biol;citation_author=L. Flagel;citation_author=J. Udall;citation_author=D. Nettleton;citation_author=J. Wendel;citation_title=Duplicate gene expression in allopolyploid Gossypium reveals two temporally distinct phases of expression evolution.;citation_pages=16-16;citation_volume=6;citation_year=2008;citation_pmid=18416842;citation_doi=10.1186/1741-7007-6-16" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=P. M. Fortune;citation_author=K. A. Schierenbeck;citation_author=A. K. Ainouche;citation_author=J. Jacquemin;citation_author=J. F. Wendel;citation_title=Evolutionary dynamics of Waxy and the origin of hexaploid Spartina species (Poaceae).;citation_pages=1040-1055;citation_volume=43;citation_year=2007;citation_issue=3;citation_pmid=17208463;citation_doi=10.1016/j.ympev.2006.11.018" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. M. Fortune;citation_author=K. Schierenbeck;citation_author=D. Ayres;citation_author=A. Bortolus;citation_author=O. Catrice;citation_title=The enigmatic invasive Spartina densiflora: a history of hybridizations in a polyploidy context.;citation_pages=4304-4316;citation_volume=17;citation_year=2008;citation_issue=19;citation_pmid=18986503;citation_doi=10.1111/j.1365-294X.2008.03916.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. Frankowski;citation_author=R. Bastrop;citation_title=Identification of Anguilla anguilla (L.) and Anguilla rostrata (Le Sueur) and their hybrids based on a diagnostic single nucleotide polymorphism in nuclear 18S rDNA.;citation_pages=173-176;citation_volume=10;citation_year=2010;citation_pmid=21565005;citation_doi=10.1111/j.1755-0998.2009.02698.x" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=W. L. Gerlach;citation_author=J. R. Bedbrook;citation_title=Cloning and characterization of ribosomal RNA genes from wheat and barley;citation_pages=1869-1885;citation_volume=7;citation_year=1979;citation_issue=7;citation_pmid=537913;citation_doi=10.1093/nar/7.7.1869" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. M. Hancock;citation_author=G. A. Dover;citation_title=Molecular coevolution among cryptically simple expansion segments of eukaryotic 26S/28S rRNAs;citation_pages=377-391;citation_volume=5;citation_year=1988;citation_issue=4;citation_pmid=3405077" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=D. C. Ilut;citation_author=J. E. Coate;citation_author=A. K. Luciano;citation_author=T. G. Owens;citation_author=G. D. May;citation_title=A comparative transcriptomic study of an allotetraploid and its diploid progenitors illustrates the unique advantages and challenges of RNA-seq in plant species.;citation_volume=99;citation_year=2012;citation_pmid=22301896;citation_doi=10.3732/ajb.1100312" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Jiao;citation_author=N. J. Wickett;citation_author=S. Ayyampalayam;citation_author=A. S. Chanderbali;citation_author=L. Landherr;citation_title=Ancestral polyploidy in seed plants and angiosperms.;citation_pages=97-100;citation_volume=473;citation_year=2011;citation_issue=7345;citation_pmid=21478875;citation_doi=10.1038/nature09916" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Katoh;citation_author=H. Toh;citation_title=Parallelization of the MAFFT multiple sequence alignment program.;citation_volume=26;citation_year=2010;citation_pmid=20427515;citation_doi=10.1093/bioinformatics/btq224" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Koh;citation_author=P. S. Soltis;citation_author=D. E. Soltis;citation_title=Homeolog loss and expression changes in natural populations of the recently and repeatedly formed allotetraploid Tragopogon mirus (Asteraceae).;citation_pages=97-97;citation_volume=11;citation_year=2010;citation_pmid=20141639;citation_doi=10.1186/1471-2164-11-97" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Kovarik;citation_title=Rapid concerted evolution of nuclear ribosomal DNA in two Tragopogon allopolyploids of recent and recurrent origin.;citation_volume=169;citation_year=2005;citation_pmid=15654116;citation_doi=10.1534/genetics.104.032839" />
<meta name="citation_reference" content="citation_journal_title=Bot-London;citation_author=A. Kovarik;citation_author=M. Dadejova;citation_author=Y. K. Lim;citation_author=M. W. Chase;citation_author=J. J. Clarkson;citation_title=Evolution of rDNA in Nicotiana allopolyploids: a potential link between rDNA homogenization and epigenetics. Ann.;citation_pages=815-823;citation_volume=101;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=R. K. Kuzoff;citation_author=A. J. Sweere;citation_author=D. E. Soltis;citation_author=P. S. Soltis;citation_author=E. A. Zimmer;citation_title=The phylogenetic potential of entire 26S rDNA sequences in plants;citation_pages=251-263;citation_volume=15;citation_year=1998;citation_issue=3;citation_pmid=9501492;citation_doi=10.1093/oxfordjournals.molbev.a025922" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=I. J. Leitch;citation_author=M. D. Bennett;citation_title=Genome downsizing in polyploid plants.;citation_pages=651-663;citation_volume=82;citation_year=2004;citation_doi=10.1111/j.1095-8312.2004.00349.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=A. A. Levy;citation_author=M. Feldman;citation_title=The Impact of Polyploidy on Grass Genome Evolution;citation_pages=1587-1593;citation_volume=130;citation_year=2002;citation_issue=4;citation_pmid=12481041;citation_doi=10.1104/pp.015727" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=B. Liu;citation_author=T. M. Davis;citation_title=Conservation and loss of ribosomal RNA gene sites in diploid and polyploid Fragaria (Rosaceae).;citation_pages=157-157;citation_volume=11;citation_year=2011;citation_pmid=22074487;citation_doi=10.1186/1471-2229-11-157" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=B. K. Mable;citation_title=“Why polyploidy is rarer in animals than in plants”: myths and mechanisms.;citation_pages=453-466;citation_volume=82;citation_year=2004;citation_doi=10.1111/j.1095-8312.2004.00332.x" />
<meta name="citation_reference" content="citation_journal_title=Pascual. Genet. Resour. Crop Ev.;citation_author=F. Mahé;citation_author=H. Pascual;citation_author=O. Coriton;citation_author=V. Huteau;citation_author=A. Navarro Perris;citation_title=New data and phylogenetic placement of the enigmatic Old World lupin: Lupinus mariae-josephi H.;citation_pages=101-114;citation_volume=58;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. Maluszynska;citation_author=J. S. Heslop-Harrison;citation_title=Physical mapping of rDNA loci in Brassica species.;citation_pages=774-781;citation_volume=36;citation_year=1993;citation_issue=4;citation_pmid=18470023" />
<meta name="citation_reference" content="citation_journal_title=Biol. J. Linn. Soc. Lond.;citation_author=C. Marchant;citation_title=Evolution in Spartina (Gramineae). II. Chromosomes, basic relationships and the problem of Spartina x townsendii agg.;citation_pages=381-409;citation_volume=60;citation_year=1968" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Margulies;citation_author=M. Egholm;citation_author=W. E. Altman;citation_author=S. Attiya;citation_author=J. S. Bader;citation_title=Genome sequencing in microfabricated high-density picolitre reactors.;citation_pages=376-380;citation_volume=437;citation_year=2005;citation_issue=7057;citation_pmid=16056220;citation_doi=10.1038/nature03959" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Matyášek;citation_author=S. Renny-Byfield;citation_author=J. Fulneček;citation_author=J. Macas;citation_author=M.-A. Grandbastien;citation_title=Next generation sequencing analysis reveals a relationship between rDNA unit diversity and locus number in Nicotiana diploids.;citation_pages=722;citation_volume=13;citation_year=2012;citation_pmid=23259460;citation_doi=10.1186/1471-2164-13-722" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Res Notes;citation_author=A. B. Mentewab;citation_author=M. J. Jacobsen;citation_author=R. A. Flowers;citation_title=Incomplete homogenization of 18 S ribosomal DNA coding regions in Arabidopsis thaliana.;citation_pages=93-93;citation_volume=4;citation_year=2011;citation_pmid=21453453;citation_doi=10.1186/1756-0500-4-93" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=M. L. Metzker;citation_title=Sequencing technologies—the next generation.;citation_pages=31-46;citation_volume=11;citation_year=2009;citation_pmid=19997069;citation_doi=10.1038/nrg2626" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Milne;citation_author=M. Bayer;citation_author=L. Cardle;citation_author=P. Shaw;citation_author=G. Stephen;citation_title=Tablet–next generation sequence assembly visualization.;citation_pages=401-402;citation_volume=26;citation_year=2009;citation_pmid=19965881" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=M. Nei;citation_author=A. P. Rooney;citation_title=Concerted and birth-and-death evolution of multigene families.;citation_pages=121-152;citation_volume=39;citation_year=2005;citation_pmid=16285855;citation_doi=10.1146/annurev.genet.39.073003.112240" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=P. Novák;citation_author=P. Neumann;citation_author=J. Macas;citation_title=Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data.;citation_pages=378-378;citation_volume=11;citation_year=2010;citation_pmid=20633259;citation_doi=10.1186/1471-2105-11-378" />
<meta name="citation_reference" content="citation_title=Evolution by Gene Duplication.;citation_author=S. Ohno;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. T. Page;citation_author=A. R. Gingle;citation_author=J. A. Udall;citation_title=PolyCat: A Resource for Genome Categorization of Sequencing Reads From Allopolyploid Organisms;citation_pages=517-525;citation_volume=3;citation_year=2013;citation_issue=3;citation_pmid=23450226;citation_doi=10.1534/g3.112.005298" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. T. Page;citation_author=M. D. Huynh;citation_author=Z. S. Liechty;citation_author=K. Grupp;citation_author=D. Stelly;citation_title=Insights into the evolution of cotton diploids and polyploids from whole-genome re-sequencing.;citation_volume=3;citation_year=2013;citation_pmid=23979935;citation_doi=10.1534/g3.113.007229" />
<meta name="citation_reference" content="citation_journal_title=BMC Res. Notes;citation_author=J. T. Page;citation_author=Z. S. Liechty;citation_author=M. D. Huynh;citation_author=J. A. Udall;citation_title=BamBam: genome sequence analysis tools for biologists.;citation_pages=829;citation_volume=7;citation_year=2014;citation_pmid=25421351;citation_doi=10.1186/1756-0500-7-829" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Plant Genomics;citation_author=M. Peralta;citation_author=M.-C. Combes;citation_author=A. Cenci;citation_author=P. Lashermes;citation_author=A. Dereeper;citation_title=SNiPloid: A utility to exploit high-throughput SNP data derived from RNA-Seq in allopolyploid species.;citation_pages=1-6;citation_volume=2013;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Taxon;citation_author=P. M. Peterson;citation_author=K. Romaschenko;citation_author=Y. H. Arrieta;citation_author=J. M. Saarela;citation_title=A molecular phylogeny and new subgeneric classification of Sporobolus (Poaceae: Chloridoideae: Sporobolinae).;citation_pages=1212-1243;citation_volume=63;citation_year=2014;citation_doi=10.12705/636.19" />
<meta name="citation_reference" content="citation_journal_title=Molecular biology reports;citation_journal_abbrev=Mol Biol Rep;citation_author=P. Poczai;citation_author=J. Hyvönen;citation_title=Nuclear ribosomal spacer regions in plant phylogenetics: problems and prospects.;citation_pages=1897-1912;citation_volume=37;citation_year=2010;citation_issue=4;citation_pmid=19626457;citation_doi=10.1007/s11033-009-9630-3" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=C. D. Prokopowich;citation_author=T. R. Gregory;citation_author=T. J. Crease;citation_title=The correlation between rDNA copy number and genome size in eukaryotes.;citation_pages=48-50;citation_volume=46;citation_year=2003;citation_issue=1;citation_pmid=12669795;citation_doi=10.1139/g02-103" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Ecology and Systematics;citation_journal_abbrev=Annu. Rev. Ecol. Syst.;citation_author=J. Ramsey;citation_author=D. W. Schemske;citation_title=PATHWAYS, MECHANISMS, AND RATES OF POLYPLOID FORMATION IN FLOWERING PLANTS;citation_pages=467-501;citation_volume=29;citation_year=1998;citation_issue=1;citation_doi=10.1146/annurev.ecolsys.29.1.467" />
<meta name="citation_reference" content="citation_journal_title=Bot-London;citation_author=M. Rosato;citation_author=M. Castro;citation_author=J. A. Rossello;citation_title=Relationships of the woody Medicago species (Section Dendrotelis) assessed by molecular cytogenetic analyses. Ann.;citation_pages=15-22;citation_volume=102;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=M. Rousseau-Gueutin;citation_author=S. Bellot;citation_author=G. E. Martin;citation_author=J. Boutte;citation_author=H. Chelaifa;citation_title=The chloroplast genome of the hexaploid Spartina maritima (Poaceae, Chloridoideae): Comparative analyses and molecular dating.;citation_pages=5-16;citation_volume=93;citation_year=2015;citation_pmid=26182838;citation_doi=10.1016/j.ympev.2015.06.013" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=S. Rozen;citation_author=H. Skaletsky;citation_title=Primer3 on the WWW for general users and for biologist programmers.;citation_pages=365-386;citation_volume=132;citation_year=2000;citation_pmid=10547847;citation_doi=10.1385/1-59259-192-2:365" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=A. Salmon;citation_author=L. Flagel;citation_author=B. Ying;citation_author=J. A. Udall;citation_author=J. F. Wendel;citation_title=Homoeologous nonreciprocal recombination in polyploid cotton.;citation_pages=123-134;citation_volume=186;citation_year=2010;citation_issue=1;citation_pmid=19925554;citation_doi=10.1111/j.1469-8137.2009.03093.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. Mo. Bot. Gard.;citation_author=B. A. Schaal;citation_author=G. H. Learn;citation_title=Ribosomal DNA variation within and among plant populations.;citation_pages=1207-1216;citation_volume=75;citation_year=1988;citation_doi=10.2307/2399280" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomics;citation_author=J. H. Seo;citation_author=H. G. Bae;citation_author=D. H. Park;citation_author=B. S. Kim;citation_author=J. W. Lee;citation_title=Sequence polymorphisms in ribosomal RNA genes and variations in chromosomal loci of Oenothera odorata and O. laciniata.;citation_pages=117-124;citation_volume=35;citation_year=2013;citation_doi=10.1007/s13258-013-0084-6" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=R. J. Snowdon;citation_author=W. Köhler;citation_author=A. Köhler;citation_title=Chromosomal localization and characterization of rDNA loci in the Brassica A and C genomes.;citation_pages=582-587;citation_volume=40;citation_year=1997;citation_issue=4;citation_pmid=18464849" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=D. E. Soltis;citation_author=V. A. Albert;citation_author=J. Leebens-Mack;citation_author=C. D. Bell;citation_author=A. H. Paterson;citation_title=Polyploidy and angiosperm diversification;citation_pages=336-348;citation_volume=96;citation_year=2009;citation_issue=1;citation_pmid=21628192;citation_doi=10.3732/ajb.0800079" />
<meta name="citation_reference" content="citation_journal_title=J. Syst. Evol.;citation_author=R. J. Soreng;citation_author=P. M. Peterson;citation_author=K. Romaschenko;citation_author=G. Davidse;citation_author=F. O. Zuloaga;citation_title=A worldwide phylogenetic classification of the Poaceae (Gramineae): Phylogenetic classification of the grasses.;citation_pages=117-137;citation_volume=53;citation_year=2015;citation_doi=10.1111/jse.12150" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=D. R. Strong;citation_author=D. R. Ayres;citation_title=Ecological and evolutionary misadventures of Spartina.;citation_pages=389-410;citation_volume=44;citation_year=2013;citation_doi=10.1146/annurev-ecolsys-110512-135803" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=J. A. Tennessen;citation_author=R. Govindarajulu;citation_author=T.-L. Ashman;citation_author=A. Liston;citation_title=Evolutionary origins and dynamics of octoploid strawberry subgenomes revealed by dense targeted capture linkage maps.;citation_volume=6;citation_year=2014;citation_pmid=25477420;citation_doi=10.1093/gbe/evu261" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=C. Toloczyki;citation_author=G. Feix;citation_title=Occurrence of 9 homologous repeat units in the external spacer region of a nuclear maize rRNA gene unit;citation_pages=4969-4986;citation_volume=14;citation_year=1986;citation_issue=12;citation_pmid=3014440;citation_doi=10.1093/nar/14.12.4969" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. A. Udall;citation_author=J. M. Swanson;citation_author=K. Haller;citation_author=R. A. Rapp;citation_author=M. E. Sparks;citation_title=A global assembly of cotton ESTs.;citation_volume=16;citation_year=2006;citation_pmid=16478941;citation_doi=10.1101/gr.4602906" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=Y. Van de Peer;citation_author=S. Maere;citation_author=A. Meyer;citation_title=The evolutionary significance of ancient genome duplications.;citation_pages=725-732;citation_volume=10;citation_year=2009;citation_issue=10;citation_pmid=19652647;citation_doi=10.1038/nrg2600" />
<meta name="citation_reference" content="citation_journal_title=Plant Syst. Evol.;citation_author=R. Volkov;citation_author=S. Kostishin;citation_author=F. Ehrendorfer;citation_author=D. Schweizer;citation_title=Molecular organization and evolution of the external transcribed rDNA spacer region in two diploid relatives of Nicotiana tabacum (Solanaceae).;citation_pages=117-129;citation_volume=201;citation_year=1996;citation_doi=10.1007/BF00989055" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=R. A. Volkov;citation_author=N. V. Borisjuk;citation_author=I. I. Panchuk;citation_author=D. Schweizer;citation_author=V. Hemleben;citation_title=Elimination and rearrangement of parental rDNA in the allotetraploid Nicotiana tabacum;citation_pages=311-320;citation_volume=16;citation_year=1999;citation_issue=3;citation_pmid=10331258;citation_doi=10.1093/oxfordjournals.molbev.a026112" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Waterhouse;citation_author=J. B. Procter;citation_author=D. M. A. Martin;citation_author=M. Clamp;citation_author=G. J. Barton;citation_title=Jalview Version 2–a multiple sequence alignment editor and analysis workbench.;citation_volume=25;citation_year=2009;citation_pmid=19151095;citation_doi=10.1093/bioinformatics/btp033" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Niol.;citation_author=J. F. Wendel;citation_title=Genome evolution in polyploids.;citation_pages=225-249;citation_volume=42;citation_year=2000;citation_doi=10.1023/A:1006392424384" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. F. Wendel;citation_author=A. Schnabel;citation_author=T. Seelanan;citation_title=Bidirectional interlocus concerted evolution following allopolyploid speciation in cotton (Gossypium).;citation_pages=280-284;citation_volume=92;citation_year=1995;citation_issue=1;citation_pmid=7816833;citation_doi=10.1073/pnas.92.1.280" />
<meta name="citation_reference" content="citation_title=Progress in Botany;citation_title=Different Transcriptional Activities in the Nucleus;citation_author=U. Zentgraf;citation_author=R. Velasco;citation_author=V. Hemleben;citation_pages=131-168;citation_year=1998" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Haplotype Detection from Next-Generation Sequencing in High-Ploidy-Level Species: 45S rDNA Gene Copies in the Hexaploid Spartina maritima" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="twitter:description" content="Gene and whole-genome duplications are widespread in plant nuclear genomes, resulting in sequence heterogeneity. Identification of duplicated genes may be particularly challenging in highly redundant genomes, especially when there are no diploid parents as a reference. Here, we developed a pipeline to detect the different copies in the ribosomal RNA gene family in the hexaploid grass Spartina maritima from next-generation sequencing (Roche-454) reads. The heterogeneity of the different domains of the highly repeated 45S unit was explored by identifying single nucleotide polymorphisms (SNPs) and assembling reads based on shared polymorphisms. SNPs were validated using comparisons with Illumina sequence data sets and by cloning and Sanger (re)sequencing. Using this approach, 29 validated polymorphisms and 11 validated haplotypes were reported (out of 34 and 20, respectively, that were initially predicted by our program). The rDNA domains of S. maritima have similar lengths as those found in other Poaceae, apart from the 5′-ETS, which is approximately two-times longer in S. maritima . Sequence homogeneity was encountered in coding regions and both internal transcribed spacers (ITS), whereas high intragenomic variability was detected in the intergenic spacer (IGS) and the external transcribed spacer (ETS). Molecular cytogenetic analysis by fluorescent in situ hybridization (FISH) revealed the presence of one pair of 45S rDNA signals on the chromosomes of S. maritima instead of three expected pairs for a hexaploid genome, indicating loss of duplicated homeologous loci through the diploidization process. The procedure developed here may be used at any ploidy level and using different sequencing technologies." />
<meta name="og-title" property="og:title" content="Haplotype Detection from Next-Generation Sequencing in High-Ploidy-Level Species: 45S rDNA Gene Copies in the Hexaploid Spartina maritima" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/1/29" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Gene and whole-genome duplications are widespread in plant nuclear genomes, resulting in sequence heterogeneity. Identification of duplicated genes may be particularly challenging in highly redundant genomes, especially when there are no diploid parents as a reference. Here, we developed a pipeline to detect the different copies in the ribosomal RNA gene family in the hexaploid grass Spartina maritima from next-generation sequencing (Roche-454) reads. The heterogeneity of the different domains of the highly repeated 45S unit was explored by identifying single nucleotide polymorphisms (SNPs) and assembling reads based on shared polymorphisms. SNPs were validated using comparisons with Illumina sequence data sets and by cloning and Sanger (re)sequencing. Using this approach, 29 validated polymorphisms and 11 validated haplotypes were reported (out of 34 and 20, respectively, that were initially predicted by our program). The rDNA domains of S. maritima have similar lengths as those found in other Poaceae, apart from the 5′-ETS, which is approximately two-times longer in S. maritima . Sequence homogeneity was encountered in coding regions and both internal transcribed spacers (ITS), whereas high intragenomic variability was detected in the intergenic spacer (IGS) and the external transcribed spacer (ETS). Molecular cytogenetic analysis by fluorescent in situ hybridization (FISH) revealed the presence of one pair of 45S rDNA signals on the chromosomes of S. maritima instead of three expected pairs for a hexaploid genome, indicating loss of duplicated homeologous loci through the diploidization process. The procedure developed here may be used at any ploidy level and using different sequencing technologies." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
    <title>Haplotype Detection from Next-Generation Sequencing in High-Ploidy-Level Species: 45S rDNA Gene Copies in the Hexaploid Spartina maritima | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ynFdoBtYWkPcrBNjNrTClGgIOtBz0lfmm998Be6LEf8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460240","apath":"\/ggg\/6\/1\/29.atom","pisa":"ggg;6\/1\/29","ac":{"\/ggg\/6\/1\/29.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/1\/29.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/1\/29"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/1\/29"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/1\/29?cited-by=yes\u0026legid=ggg;6\/1\/29":true,"\/content\/6\/1\/29":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ASjpJ5fhjUvkAFOIN81UymMBc6zYJe1kHnINaSh9LFVWDnE1Md5rA7Hz5J+QZarigCDqCGREwl0dw4PMr9q6ATsK7b7UqbNUcgZGhSk2Uc6Y3FuiuFNUJhARyxyCsUrxBFUP2+VmVNSDpp5Kz6f0vLYJipArsDotlIqX7AnduAJ93qQLw835lWPG00ExkTmHIygJ6f17R5QIbhHE0TbsxQ8so2lTXc7P0vzLR0eTQz1d950Js32TBcL5fPUgPVQRA+7M+iovbspBM06v3khA0+\\\\\\\/aYWBuBI6URuDZWdLlAhrZv2vyouCR1pD4CRY3BlMoUagAfwM2DmooLbyCGWg8cm6oeTGZ4s0dqdwcDR2ckxw6dBxKlyDo083yHdSjGR78S7IYYd28R9l+jRjjNCh6+9qy4WScPqo9HM5M1lCPR1mLgqXKEG7+A3J8GxcqG4oRQLh4SMS0SevE16cHEBu4a57ntSJLScWQ6j2zXGgLUKioGR5SD5QqFI+ozo+0UACuH9hy\\\\\\\/T13GmRUNarSwm\\\\\\\/qrAaalbHbZIikdRatY7yPeLJgTU35tx8eTTpHy37yoBFFVlnV2MUWKk7pACnHxbMYP0hOfOEbPp6\\\\\\\/NBsAnLyXhF6eYwLiFCJtVUAUXNMvMFz+QthnHZMxjRSE2+wF6Jy+E6IMO6HeOoj+nljI5ttccZXs8JD+jihhtZW4O2bLB0+lwibXBJOJaRAAGhT82PIuRLLKrgyFOWT\\\\\\\/CBFxU40yVPC+PC235hooW6BnjpF2wmyebtoBmBW1t\\\\\\\/q\\\\\\\/Yif+BFIsAkFePcaoBzbJo0mWwbBxN8yCaO2rI80yoJ0jVkoN8PdiDtODU4Qd7lNcoL8afmmedgU5j3zIyrCuAjgzgyEy1w5H4nFeKOQXl8dVg4hGIhMUlZ8Efe2lQDNjtYOVBZUBYDplBep+eleyGYrjTIry4VlF79+IpplF3jQunmY5fnIrDLHbQdNx7zHUT4EPRBo7mQzSQGgl+oBONCIS8FABoRANJWZrCLIDilziR1J3jhp8ZJgSsZ9H\\\\\\\/QoGazpnjQUae\\\\\\\/teBD7k8JRf7yOszUP+\\\\\\\/sXpdwOwFAJWbSv+jP0yX459lPPd\\\\\\\/w\\\\\\\/iE+L0N\\\\\\\/oNbJdtJ9dmwZVP5pJ4lT\\\\\\\/9pLYHCyJpxl4b\\\\\\\/bC8sZpdsbK31POVRhMweeACaiJI90YYCEWPBCxAOgqdNzmrLZqNyXKoLIG6Jz3Ceyk41zLSpo3x1EGdWVYbau8BWjeucuvovgzimIl5pPilHgPvWPuv8KtXcBYkwp2RqtCkkDIX9tqPSf+A\\\u0022,\\\u0022iv\\\u0022:\\\u0022+Id6v7BZ32S1WX92QATLPA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a0b4ee968ba24be38f629348ae5a6dafdaf71130d2f168a94f8ffc673e6ffee5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ksBBdbxU8jRbyaRlBEjT8spQC97Z5Y\\\\\\\/g\\\\\\\/899igKAGI8g2oI2Cz7K6kCBruhAoyAhMkBtvD1VVLgjz4R0Im7sjLknyILGrMiKyhkHouaKsERa1HpMxDH9VVQBcGVdADnDcxUxJCzDQuwNKZe8FIx9OxQKZSLtr\\\\\\\/15+Deozh3cbebIwuEumv4NSBUgR1NcHmTZ2G7646S0FsM6e3grWqvuPgrpMElpbvt6aHfz7qle7SWVzVzB40vWONLNBPvsrJkJe2\\\\\\\/qpffHLtSSM3bFp+QY7Rb8wwt4TKah0mNX7ki1CRYg8lPCi9CVgRBXGaam9zFoZRDqZy7bSAxw1HL+Gs2FUwKhcIGhQZn42HOdfv6wrQQJwRdRoh3ps3f8NuehsxhCFl76d3QyMeG4qNuQs\\\\\\\/hzCEYVJhVKXEtWTNgAhXFOCzSCfYl8+tgH5P3sCAd+6t7mjR\\\\\\\/+m6YAwcirncDm9yWarDfVE5vB+kz\\\\\\\/XVAft4jdW2LInrRdTPncDLWyg7ZbTn8EYW4ZnjzSMwaEOYtb2OKXeayaKrSQ5u\\\\\\\/0fdFXI50z3Iq8mumlX0\\\\\\\/rRzJ6GAwUebNdaJaaSgXPkt1Ji2YVDAvbd5ySTwTIcJyZ4IK0Vu+tTV5RmQUc1in5k1tx7\\\\\\\/8pY0o\\\\\\\/PuoY8vdXA3eB6G+KUIf\\\\\\\/S9ssspbnDfclsMStsUd3AsEia7w+6UjagC89eyCl3D90Y4Gn0vnQF\\\\\\\/Yuy\\\\\\\/K\\\\\\\/i0K9ZJgjmhrp4dB+wu3S1RRe0WpEjWQQ\\\\\\\/q2fRwYO9uVP5+bY+HQAzq3nJmK86Pblbi\\\\\\\/FZtF4ttgv8cnSkY8gCvYhN1XHgYUSIPz+301toUAEteV+XcZU+RFoCNpeYUfn9maO5O3Op7tVNFOMt\\\\\\\/dbXn3RLOHiTe3f7ly\\\\\\\/CY8G4aLaZr4+BYDtJ8m\\\\\\\/TZkrTcIBoCOQ25tyqvwhbOo8pjsyb7\\\\\\\/MT\\\\\\\/w8apPscczx7I2z7LQCvwQ5AvbsskHKEx2inKne08IfYjUxfH4rpI+Hp8jwgkAGGvB54PJ5ZMKp1+skzZkY1YzVHwM9LuFnOF1sW0X6jAWA+pebCmC0wHQTdYvB0sTGEgFGizlUox9X0hcvolr0MoPEj+v4BMUt2Qdl+bFxQhpnedvAXFupAit\\\\\\\/QVF7YhE2Sm887lBwcwyjWiajdaQLTGNxDpkr3V4fg\\\\\\\/l6PuRGFGjT+3wojkS4kg2ZKUv3AYtx5Vv3F5TgoBnfavMOVtAnPzRRBo7S+vV88\\\\\\\/HIF89qJiTwxOq2HU36r7zkY1xsvTHKr48hkOSwsrJhHuOpAg6SkjtnqtwCuy3FHNFmUGOyiAT9+tF1lFhwhapMmGMgnZRKN8Gb7ND+AD7Ah7U\\\\\\\/ZFAdCluFmCJgUbS0D11WCtoLnXVi1jKolA7Btanv1XOlC9bmj7VWn2C2O19seQxU1s1CPm7IkHEm3FAAwJCXgdEqCoEeuwVKxPAvlEtGrh7UMe1fGGgfYJO3r0Oyh9wE9fBElzGVL8tEvgoUTWFmtBIqSD3ZUetdUwx22buksZdKI21B16N7mkvUs96atZx97rr+jkCtnNxk0pcGBUy7Dtt1DM83hwFIeoFp+3zSIRKKyllqFWpIdXddKJCnJFpi42YlNzYusbY\\\\\\\/BjTDlNJuegAqeTU5br0aCkrCS293ocr1IGVu5uioDhbEFPU50H9fHGnnpDREVuogbaHg2Ffll3xnYqQCuFkqaFv5CnikMd4TjmcsJ0r0OTVJcTAYjUlvBvfaLouKH4N8JA5EreegyZNeiEAOCWO+mPSZZHHpK9S2QU3GXepNux8PKNd8neWo3cqsbwIfudlI8mlEXoa0MO2h8wI1+Hb119\\\\\\\/PwuMIuA0UbC06wRGnM30U2sxpJbgVz\\\\\\\/4lCUBFCApRMgHIwxHwjgzV4V\\\\\\\/8Fd4dpqYEnND3a29I3Jv4\\\\\\\/BwXc7khjtycHB3PTJbxupuEhm6UsIFdCjZM8HCITGoASuYnqDhr9uaczQCMDcPmQTr2RcgZqZHbHIAACLLr3EOwP47qwPXClV7NIFDWeFFa0AeffaFrG+giPBqIwyIM35Ju8vAW7tHK27doWkYoqkgcXelrECAEOK5FC7mlkk96NB+BmkspnhZvDvtMfiP8xnf+YWN9cO1+cSCFzGI\\\\\\\/WhlH3d22jLbHSVKtUM6Basl3vNpOVxCYnJccdxEvrIJ\\\\\\\/0gcQql0qVmHCmrqZ25TD\\\\\\\/5\\\\\\\/\\\\\\\/CtjRy\\\\\\\/pZoLf7ZKOEtK32OnVAoUw0ORNNRWS3TpffTKWorN2VlyMHLc8ppUiYTuz8yU3nP\\\\\\\/MP45r7FUXqusXksr9hfVUlb02ZJ05KpgWMONU1jEqoTPqMn3TfWXY4ysxG8\\\\\\\/ECQZ1RRH8MBk4GOIzaVD0cTBML1+UsKjDOM3Yjf2amHuJqjWJpdCP9SBNNBF5DzOEEI+N7hcFwM\\\\\\\/HgUtR1k2oTdosKMhOFhD7zG6dqL39a0OhHs6HXxE2AzjVe3YHNgM\\\\\\\/m+Fk0BQnyU6aGmJJMitYudu9avvRivoetmfUqyRGMvYcSpyRZSSw55ejSxNs0AHienDvSBI4uzQLXnLyZPNR2I10Vb2KPWzewpAT39WfS5JEBmlyjIL4t25H+HELlRF1ZdvRUFaZXDmWvzB5w2GLBe8lPuy7rs1gFnnXeds8Uw3CnuzxAnRWHZqPTwNXFlng0lFN5kMEjGgdy62FnRGDhmsTzB\\\\\\\/OEKdWVi0wvRDDMnHRok1hyw6xsuuIfye1FMIJVjkztA4bPB7cZ6t+PHLmxhT9rAA8wUwIMmsumMlH2MtnxrLPIbcyz0FcZEpus+98UB\\\\\\\/87asvfm2tw36ICPhSLXzIRKrHyAUXt\\\\\\\/mq7douv4v+qHl7DkAWTPA8sEp\\\\\\\/vminXSm3BmijKrCpziLZtcZttwHn7yNgdoIotYkVU3UPO7wG3m19XqU3LpdmgJOK2yuY7zAv0lkAqa5XZ9pYjPnvymIhazsLcfzYi4U71zabxy9OOYG8n3t5SvsP+udAASfYf+QBqeyOsAHT+cs+yM\\\\\\\/ilemcz0GWIufWqAFCImhjfhRFpvrq8WpQSSdNEx9g3\\\\\\\/scUY8OhmfBEOWG5z3XSykbt0Zt92nxLhu3Bd3XOA\\\\\\\/zaTd87izXfPHAsZb3whb4AvgfNsKNOZ3eufKlJugdv03vxeTxVqan7IDZ3v63eiTBTDNB+0MmAV4nl8B5RwkU38LSmwEr7Dg0TCH\\\\\\\/0UxBdt71iXxiXeqLW\\\\\\\/c2TcTlVmqGUaC095G168XmsfZeES\\\\\\\/cTN4h6IOsv1o0djQq34nCzmKBtyzXkol3lFpRjlcnyUaHwhZKYS6QG5QYTV59hea6+qIx2uF0Eb5YS1Fetc8hdFoMF9JkbcnOoxkiH7LtpkWwsbnIEfShoMsSDi+zIuJ1VaJoAIwwDqYDcAcaJm+bZdtyxEQCXCJbRDwuRwKCHfLlFxxKHU6SN7FxZ2Is5gsvx5pTOYHiah4FwAw9yo9bFce8eMf3D4GBoxPpZTKfWrtwLgViqEfBLSA4gHo9xxKRYCZNaI7UsHgf11upR7VFtcJsevAfheDId0JESRAEPjS5+LGw0aX0wpeAY9+cGDkpNQgkJQv+LoZUug7EzeC+K7vj2UjNJe\\\\\\\/W3orr9OKbwwvXPf6dIsX\\\\\\\/TdYSa33MfMX+ZzbZwA4jqZr+s4k+ierjKdJ+wftGHoT0gKPbAjAPVg6zzDGI0QhnJ4M1yjy0HvQdD\\\\\\\/AQ8MJc9u8\\\\\\\/S7t4XDayHxtHPR589OaY04+rQhy2PO8U3TC9pl5DX\\\\\\\/USwUFM1cf05IxNjOso8490oxjo8o56MJPPbUNbdQyUP0q\\\\\\\/urvGC2vdpfQrTPC3di3sstrnko3t3cWCU86Gfa56+8iol+\\\\\\\/CvsIzDoPly2YyaK+grueVoE0tIMgulYKywMqREtiNRdCjjCgCRmXQI\\\\\\\/O0qqc53VBI1o0MSWsVS0ADNIOO7D805ir7LQKjXCV22\\\\\\\/aLFgmVAFZ1zD40Tz58pUy3ytXWzCdhSZtX7UgolEMfLJlympJ8fti3bWJ4SX8\\\\\\\/aPB5qhmAY93G3F2fpvwCMhW5ILFoV\\\\\\\/AGh+10Qnx7uVDiLnSTSX0DhklsZSkqMTCb7SRuZsj1Usu3aVh1+5vWX12gXuqo3SXCzF2SoIeGP\\\\\\\/tfx4ZcQj82mOMy3k0RfVIsTGPK60gFBArTXOu+RkzPpPp8EQ5SXSrpTm9J5DCE+o7W+\\\\\\\/48An0GEL+jidWHy2k6iEzWrlpVP3kc3cRV6po6R6parn1h5RW\\\\\\\/um3fJsgb76NKOJAakbYAaiqU9XTxIUQu0OE7lzC6VNtFglC06sYDGFHqDhGQWlDs\\\\\\\/B\\\\\\\/6rJVAisYAIDOo3TN\\\\\\\/2xxkE1ihyHsvSEjmMUscpd6gcD\\\\\\\/DjiPwFCAFkvb4g0mfF0QnnxrUfAxbSnjYYQkCvukd+Oo1GhePZ2i9nTHMd75NaAf0C3iVMH\\\\\\\/JCPWx8qSovbRx0J3MrLkOzQhwCLRjZlWLQXVOKWxjloijnrEAj1phw1seTko0miqvZ2EeKXYo8aGA6ieFRLq43+rL0xur1fp\\\\\\\/Fz0TT+KoTOi+fXXyEvl7oKtzloNrhIZTMbkK2uXfLqA21s+qYaw7Lui8gOgXH9q5GkgyKNeBnB4Hvrfgz4MOb+fD5jHeqJZWdUOFHy4GhrG9QIihg0Xzt31g5Kh8pZtmHxYPlufVDaZoXGJp2FxYFAi9BWa6VpnTGXvX0GYsn\\\\\\\/5EXI3gWsRHQYhpfFRKjjrelkzytdxzbm1qethhg6HxcRMZOe6S0aGFVjot3+4pAsG2r0y+4BjjZsE8+AuQKAt6QpV83g6Kx952yIsxybluHF0xNqqcNNWx\\\\\\\/YdaeVCLwOH4cn7hF+K6inDAvgRQdJTK5rbzCO6qxNsFukPGv1HKIkvzIFGmTPCToLIF6uY3GA7Ci5w3GmIwVxiSMOmczfk9eFiClXLxoPLS1YpZRTNyFORhYLTQ\\\\\\\/PkTGs6VjLAZoB0xonkqyj+PjNoYyI8PAgk9gnlJFYgO+QNduFatTcoX2Fwu6N9tTO3teTmWWdjp3pudH8zXICCArD56t+1+x4Oy3e3htMnyyT2S9P1cxY9KoUh6P77DLdskPYuPbnJ\\\\\\\/iX4VG9IuFvNS020s8xiWNuQl2CDEl8EAoaLby87W+ukxxXKCbrcLn\\\\\\\/fmGIYdqnil2aMb7DW6rDgaw2HatFrh8bjBqtaesrNNwYRu\\\\\\\/Q0mOXAjnZUPTxaQLuCFUH708AB8UMcDseW1UJ3ilA8n6Cj\\\\\\\/TixlWM\\\\\\\/nRtu1KfwNHgKH06wW2CJxd9HUQiUiYkAwJulcexGkxvoYQH135D3I5AmSNU2j7JaKT3EH4ZZuyBtSRkHD6NnzyIrE92GdQdnMomJx5tUONFeXH32f0MqNBq3QJA3ReVZrIGbkxjHr2K4FTw+vsmJim04bvHxjyKs8X0gcbAEBpvdaVLUfXBTlyXiNF1M+YidQPBJtaCEULWq+tWSdcQGWYUpIRviIPipegxepcyFXys8XBAvG0hQbxnkZlS+aUEtuXclGlevkWR+7vVyv3K9L692RehNnOHYka83aco6L2U\\\\\\\/flsx8RuiOuME2OECIy3eeQCrvty6jQ6G+IrVwdRV0r0FACCNdPjrCUdyuoYHJLWUsSCNiS\\\\\\\/25rEKAtNT6Nm+K55Vrz1xLBMRqBflDuVTGWwH1w0\\\\\\\/QM1kHvDiBMZPVJeAWAySYcy5VwdSW7uSho1nBLEWaDQqhLjNyFvmeUJqsqjdnP1TwNhqPkOM2ubfeB9G+B7mda0i3VmzCC3P+fNDG1k1Isg25nw9SgHkjBcA4Sfjmd2Q\\\\\\\/mMncpHv0jyYo3qFGIeEohnDjEzfxXUGN9JY3G3ngCCCshBTUR8xNUOqU0cwblbVjb6TDfBhW2G4XT10zG2t31jd7Aag+yYL3u3E59Z5hvYulCaVAVpMKLS01CmX8TRQ4scX1WazD7eEbD\\\\\\\/t+QBbnxRROYok5oq2GJbCHOitDI8qBBGza3gwDH9AOZtw75sjn9HOvDjgYleUYjluDmk6wecAbYB90G0vzoRaozadHGcioH62\\\\\\\/SL9P20PTXbfkqWIkwxY+OLfcGqKQLbvp2Dx0fJMH5GIIsNjeT\\\\\\\/TFBtaJg1ZgBteMWtgxAUrfhZTgWZuDk8dMIj+wAV\\\\\\\/PnSkHlMaLDazrT4BFEfx2LOjr8HY+W57sXBtEkhAYbNjGhjz21hdgX\\\\\\\/clJ7IoqdFmtyAzcjd42KRFYFre8d3o6eTWER46a4JfM1TDmvApf6JBnpG\\\\\\\/O3xWmFOJeB6K8iXUOxWQoisOsesRasOA0bo8bKkNb\\\\\\\/sSrpsAZN3B8XZ1R+lrQNwTRhzGftDzzOzhysplc\\\\\\\/xESPfRa\\\\\\\/Xb+YQ7IUwLvYIMT3m1hzlBWlr+EeTb6XFYXu9tGKRtVxlt1NpdEmhsJq+rEtLmR7tSn2Xg4ffy1okQycvEDRngR21WpejrH9\\\\\\\/H1MB6ODotlOqC3nIklfIIK4BBGXASS8gqnZdmZqcW1XwXDKX177d9n0Z+I6zNo8ScufHtFMW9Kj3yKL3KRvkcNJdShxPIwfIGhbmHhoEeXwFHnKBRFmBtf46ZCuVtyCETTBgEwCAO\\\\\\\/S0XJmmQ5gYEXyV2+kbI3f4DPdxqAneDX94ZvupeoglQsg4mi0YZ18HcVXJHKfNjdj1oPSGNxuJ7mxrTS2Y5XLxACwICzKgUzwldUDuzNi3GbNvejcXrh8WbEBE8C51HREt\\\\\\\/oxBEA9mvPgPG50sF55Ghql35ef\\\\\\\/vSiSy7epIQxo5v5alL2VjPSRo6o9+CcXWwlL3wQabFjjiqqaLc0qNHnryCfBO0lrKZsXZ7FJiyI5+auqbCiosj5ky0AUx+62m+BDBlDXDGJWr19zTRjOddflkdFIuVrVE1Sdri3W72kiiPhQ14J9F7d\\\\\\\/EMJ9qQ2BZJJDrSkvo0svIYsmt6LdS0bYruXfstXo9YdBBxdkER1\\\\\\\/C8EZDDt8fHhuc1aYDck8DpNmyFIHZDszYOdJQbMEVYPgd7vyhPuxr9WmxuPdQKkUlfWbAyt+uQ5E82GVWpGvQz0SVyfk1wX7p0EEwORiFu2YVu7P1DE8jSLOG12PO5fIYssdUJwBD4KuCXYfuVFhpFjedbz1C9\\\\\\\/3ePWPHTFEdIXJARYYdiyBqkXBJ\\\\\\\/7kQLkqfX4KK++aEUw+yfFf81kfgVi80xxU5XnXAn6hhsR83GabNZMCzeo54sn8LXVREcNgHWkLvhwUZ+NhLvPvEhoB0ZuGkhHI0v79sy1nGMNaHXosZJXNltHv7q\\\\\\\/pGQ40t7YOp7bPzjwRgE92\\\\\\\/QsplLbXtlFBuwX\\\\\\\/KuutooV+6Dc4INn\\\\\\\/LVS9ltIcIHmdOttl3sTFEvSpJCsABpmFkEoUURvycbKL63F+QxMYAFcOpgHstryDzMC0lQOv\\\\\\\/LdvEkf9m67KA4jMuPM5fvXf1CojF59D840f7oeTd6DXL7TBgCLwUfvKkh7YzsZT3W1hK43Q1E5aCwD6MlLHm2GQksXshP4t20GupoREjo94kbC+GA76KEk4lqG+Mjfu0ZiTyeysZ9xszMOAc1MjGd1m\\\\\\\/+KRON73GIzAMnIRsPr+DUbnh3FGs9wOA033veSmy42VJopDUiy5wj++xLhF+x0EJv5GLeCMx2MAOqL5kS\\\\\\\/s6+bv2vR2NtzdljwEgYtp3Cvg6j63M22LNnDWwvtyGIRv4SxSX2pNNDbeQMrZG41HjmZ3cPW3Eh4wfvDQ84JHWJLqr4LAPBiJrVJPP4l5seftvSTFE91MDv2YKFjHWO9+FUuZRsOda3+g36d0fh7mu53I6cU7v2NSQ5rRqmte6U8i7\\\\\\\/i3Qm8qEpVzGQsZeNHHw3+5YzrY\\\\\\\/VgxXF4bKcoM\\\\\\\/oAckS1U4tDuDOzEric+Lgjar0n2bBYZhZQoxXszYtCWq51o0ffHPWg+2azkbwX8IiijKXjgv9852oO8gfLFEBvDsq+siTncvOedNtLNw47XfLWkgnjXF2PEfsHB99BfNwGS0EUVG6+zQdjMugoB13Ig9ct8\\\\\\\/X+Lv7ofON8tFqtv6w9+88MwMSDDLVad5VKqgLIBGYEUEg6iFL0HoeWXVYcTyq66GynzvMmagiCq4dHuCNnG4FH90HnNjjySJCmPE2LOXMDNRl9rHlFiX+n+kY1Qv2WOr5RtLwu78CxdHYssI5UUfpQ7\\\\\\\/jpvi1TXs7XjSJkXu7Jap2QADS\\\\\\\/6CLnvm0s3KHQx0jQDHY7siaEO4lYhQlT36OWmkYW10Z1ZYyGCxQ7\\\\\\\/oZsCVvp9bU4i5sT8ROmY8EuYUGSQI52kSuYrKJIPRwh5DtFgv\\\\\\\/PZQK5bFgZ0FMLyL7dfv01Ln0zH6el3G5xXVDN3ZirNWd5Dr7I\\\\\\\/8rfapUgZsXxOYqYFoefdAnd13\\\\\\\/K1VKAJZUvdqRiCJ21b1+YIb2Ej9HByjTvN9zOuVkxXkpqP34yUc7uxtrl4Vo0fx5zRMtl5Zd\\\\\\\/fll4qTfRvszOyc10HmkfQEtjdAxJW1B8GC2x6GO39vk0A1agJBxL4BDZQ3OpY0h6IhcXo0A9l73GxqiINgB0YZEkGrxNaLXKwztyO\\\\\\\/0Vr9zamoSvBWcBrwGFS\\\\\\\/\\\\\\\/+C2djagbKMM8t+lu6KwyWe77BZwWZZTPaBqGUvRQOkazvQPZZeuS0ROYrwEoJLj9\\\\\\\/E5tKuRLOLMcOfnfICGpB4AMhcLp8kbzLcBFoVfXAb4FZabQfzXlDXNK3ztUSlBShaLUyQKiGGW\\\\\\\/kL5QvsLyMd5yz+J4Zr5o+JJm9hRgnr1ksTcSq3YHRYxM2Fccf0CWX4lY\\\\\\\/yv\\\\\\\/qE0uV74n8uWiG8rf493rxojg42UfglJTjh6dz+V9AB2sLpo2fkPnM4WiqPIfDFmmL7fjvtb2VmDo2hIQMRNS1Fimtt9\\\\\\\/1LdaNC8MghAuwfP1Al28\\\\\\\/LRKQhqjEa1ZQ7QuXLqy8y1OxqaiNncc\\\\\\\/HasAANhDWd2nU6vd+xEgy+BPEylRhaUHSGQlJGajkb9pWKyKWG1HYiwOyBMdUQdamcvIiJAyY4p1rwNaLpq0uhHJeZbt7P2mt6JV192vkoo+kas89NKqEY7Rh8QpwW7OkQZT+qBgameBVYhYm\\\\\\\/SEiR4Ph5Ty6cKSBhRlFLuC5p68+vn7n0GArnMtl7Er7v5AGc4tsvZaN58Gc8XwmEAGgcHtEqoCFPb0byb1SoNRP6Ga0cRDJn0oWwDJgIS\\\\\\\/dI8pftfY0nS2nv65sEOEKOYv4jbBpuqTb\\\\\\\/e9cpNzmM0QTwZPNY+VOcepBr7CXutEx0+JmmlvEPkB0RmhIsN4\\\\\\\/BNCNhuqxzNowTtS8Lh7U5FfEJ7P5VK4fZFVDDeboAxDn\\\\\\\/UsHEMMTLTnV8TSSg+sPL41qg+8t\\\\\\\/2u4R8iKVR77seAxWDLxu3GMwnuufydoY\\\\\\\/l7HVIfJzTlsryBV5c+NTR4UXJbv0ANN4VaTFpNTDDNy3\\\\\\\/lgkDm5E4C7ALTl4aq+xn7fkv66DXRWOwTdXqUvvNdLLq4Ak2bw\\\\\\\/fvrp\\\\\\\/XEBhNidIMdOgktxZ6Ul0n2gWZ3IsYJXcreCjJ0k\\\\\\\/phYfnP\\\\\\\/9T9PDu7pYVqvCj\\\\\\\/GvC3wWUfkHukNVHgSeYw+j6JzIkeJJu4u5WfS+xvbRXk3GteuxSnIbG8M+pmIBReqWYmB6NDmGosmMBG059IFDP1pR4txrrPsazOUCu\\\\\\\/htKGADzZRwWSi5qQFwrx6GVZRvUbbPlhfwuG2RPThbtqQhGONTwRReuHarHor5MCrnOQXvxR5UEDb+WyMGiiD+dite4LvbqDX7UXI5bl\\\\\\\/kcMybQ87jfcrEBFuKvpk7mxTZjoagGe+huWoKBFSWgsZdlJppUECSpcPVpv6rO9vNqzv2rEnX1OQfd5q1\\\\\\\/lV1T16wPhFWuMo4lpmQSkXl69hNPSgoiataqmtbIdjOu+egulluYxUndJofc9fXmo5266FI8hdrTYbsTQ+5UCu2HkeyfJLOb78i\\\\\\\/W1QJvwz8Tq4u4Bf9+XNTTrMKiJicZjmcngzKYkJ0BlYS17H2cS92jEDit978xB1T8MkKiMMSTJUgA8NnWDKdFrBtkRclGxpoi1mkv\\\\\\\/X4\\\\\\\/G4qyzZZY+VPXkFcq\\\\\\\/Gvh19tw4SL\\\\\\\/rcdKQG07QQtiAN0xNmSP+Jmgn38OKaKW3uz0sy15HvHMPCO8uy7FUl8Do1wlTbZ8Lkyb2x3pcUyZ3I7fUBGWzCa54yYSKMDOB\\\\\\\/NDEcAz3HeJUoLvlRA6Ny0XjmIL58+uKfs6lo7jBSbSuW0RKGmK8ZBqHgI5dSYGT56\\\\\\\/4qSzDRvgmx+fhlkkQMu4U9OgDjJOD+\\\\\\\/ieIAuLb++UWls3yzgfgv7+RXMH9raK+m5HuLxGNI0PH1933VavTiGZkXfVCNlnO+KnQ0rqwC7dWEOrc0SC3zXOKmprnJTMnSnpG19Ao0u\\\\\\\/cnKl9rWmI+sjz086IIWaNqyAQJ2HbTgAVbhGsMi50H5oK6c1yI223UvlrbJFVHD5tZyjY\\\\\\\/9ocOv9PC0Jwvg2V6Z4aIZVqfqiOJUTqGK7U37WAgF\\\\\\\/2dO0LoQmYhmC8QamshUgCMkShK6SrT9vVsBaoBBae1Omm8Hg4u4NEPokIA+5qkWFfU3HAjz4r9tgTkVPD7LVoeqZ9\\\\\\\/JMq184wBgDquQonbLAQMfW3ZABxDxk2JlkpmhW\\\\\\\/6eUO83Nd9QADJeo2merFJwMcuqaalIhZqMQjTIHmHekuTpOQWNxeBojr9DWOV6oKpBMgL1CBN8chJ32KNa3yMeP7Deg8tVfhC2g9\\\\\\\/koqKT1p8G8\\\\\\\/CBwaKl5Cbs\\\\\\\/PVT5f4jZykDeA2Up2KNtIZL1NnWh8JgFxfkx4m5sb9UMrFZlg4yR6lSNsogqaL0uE7TNNd0WIZ4Dp0Qcv6rUhILqP7AshcSbzws+jBGmRk4bLuUXEEeTj0AhvluObF1zvxtY3e6LGtADnQTRTQQBUqS\\\\\\\/u42oqdVZ9uuA8fzOWxHfOC+rddVUMvsrfm5Z9CTSatnnBPmSIT6POO3CUBlxGgfRAC2EBEdMqR6RDv0z5eZ6WaZbd8I30SZMz7XtOZO4PWZ9M2rVe\\\\\\\/Srh4Pt4etVZ153Z8YnhLs6mQYMZKq0+brY6jJdjWT5rqjCMVmIpzQ3Qw0gz89nFxiRuQS0rQ4oChtvPVQlYUq5Tf8UREevaBKc5B3ZCSZlldEZoQkYB6QIG12FZ8z7M+xpECWcAgtnKcQILadwtg1kQmhrWbwCzAeHYMTZY7zHAkMqh+1s5LmKRlHy8\\\\\\\/RNWuvw2GbC8SXK6Ul1jOLt+tUxR5Tgp\\\\\\\/HCHw3Wn5LdefNJ+IGP434OkWfsUbhwlc4Xkb2rVgoLJvZiENWQBRj8VgS9WdfV\\\\\\\/NH\\\\\\\/Coll9NYNz61dQXrfXfTNXZAkCv2zjpHbRr7B36cU\\\\\\\/SHSfWM7CT36in0p1H76BFarprEsDnE6FXCDxME2tSrX\\\\\\\/26iYyl+67M73hb8scmt4vzMHMCK5LbhQgJKvWCNDhoRfB1C7M69xEvhvXdumjqJOpki3Sq35Tn9bJ5vD9oJLu0Kz64rjJXsNkcIKcbYM8+qG92lHUXVRvm\\\\\\\/I0B\\\\\\\/+Sa9Xw+00MBbpwz3I6TMHl6eynQ1qMwvZz1XGF6+ktd7onJRqLxXXYzookv0K7tS0NVdIv+YODtlw74WQgMiaBJ\\\\\\\/pCJekklxB2\\\\\\\/+d0Iiq1mNmCpBpi8d8arwbjUvCG9gKj3N59mJnSbTGzhX7RpJwVUDlaJU7i1vc2zOIk9TsmCor0VcZztfumUV4jKWgufk8C1xsOH1XXPjDet7aZErp9ctrS3N\\\\\\\/dNPXm\\\\\\\/kcjUqntF9+Fd2Ea2\\\\\\\/Y1EV1n4pIgCMhlsqVKRfut9oEhJYgm20GtfOFgjRP6K3NIpQxe8d5o3o5MlZAzBanR+5CROD0bQHA02yN5e0coTBBGE2R3Wpzn2Zr4d2FQI4ZbEVacz7z97fP1Agwcr6O1R5l2\\\\\\\/qn0WHt1ZHkHSZ2WRJ3DGY4hqg4kqteG9LDPo60K0VbpOBd81gzERQP3WzRr1SS9C2xTpED6rIj1reaoEK4OYEPpFstfUsjNQGqexri871mnVK6aznlNnTbn3nc3buZzuVTmnFx6GUe+u87JguRefJQE2ZENEncS5gIVUXo88UZidJSsQXXCvVZOnBBzHbrN27aIstFcDg06UCXZpmYF3C43S8CpYRSnkd1Q8bAI0dFvwMJlIElwaL0jsldEf4o3usDgcrLcK1GGS3Ah+SVQv7kSdWalHz7mJzRAQOpGOkaER78dllyYuPnKt9EoUrybWiMNkWbF\\\\\\\/1b91WSiVlHu8gTNdlPYtFGDFW1eDhixWsRrBAsZRs6Ljp\\\\\\\/IMJzlU1ekgCYBoBTQ17UuzmffNcFwkYuThia5h2H5lBRsBBBiVThJHi+fwVtk9x88fjp+LdZg4rgj7ykFQD+aSqbLs0UGcRk9EmdZHQ71Yq3ueY03KE0hOzzP\\\\\\\/DyXhtnLWnxsTBY4sahomr9bsNX0Y3\\\\\\\/pL+uTa69K4uuMOn7frL5LbXtexz9hwJJkV4c+bJRS9rcyb7\\\\\\\/LoOSgFD4ze5qfpE48xmSfk3q8pLfdm\\\\\\\/o+2tyOnO5gV+NUe9pPj79fujUA5fHFtNvoanLG0zwi3yR5JK6R9S33AykXVASZs0+\\\\\\\/WwHEa+WX8HZtmIOVqTDp4wy5lrxyMYECE3xjaTzreTaWcU7QMsCzzonjjOq6m+hmKubANQ0xPwAvbXlvbsEBzY887k7S9\\\\\\\/BKvuYdnlBz8BEwuJJ1PrVaA7KdzE9dIq1hT6mlu\\\\\\\/nIlzZNFtveuGXZq2h30AcEOEgwMGfaE7w6Txz1IT+x9CMYl5NqUoFBZTzjkGU8v0vIXedisVpuMEwkFUyAghHm3qBe\\\\\\\/jcX8mLbQX6JtPo4EK1GnijUvc1lvpbXqkZi+D4sxnb7F+mVnOzPQPvQxHIC\\\\\\\/RlvbtuQTtJP13CdgT2iBQVrAp4vFmr4Suuf2wv232jMjwPSQBouvzeIRzCHQP2SnyjrZEvaLaLHHPoNLdttLPuhGJSH3\\\\\\\/QU3ORNiDn98VUuIjS88VEqE2pfnY1VH2i2LR7TbY6U1Je06f15GHlrT2DbnFv9N+CMr39N0Vd5Rsk8d5aD4BVK5hM\\\\\\\/HtQmtGKTYrq77M6tUb\\\\\\\/VNAWS7S\\\\\\\/\\\\\\\/Zn+ChHenhMknT5qvFU4EunkwofnrnPrNeUD9876C1c+7ckCilzP+ZPDcacqcGkFeXwxc29cgnAzNFERkSuRF4QH6K53iSeHwjck2ev+taha3S3bKtn\\\\\\\/DIlNq+sHyx7fz9aHuw8ZvRhKOfDWe\\\\\\\/4HexN7rqoK54nhTP58BzaZ8foUCcNblePMD+VTI\\\\\\\/KtOIxZw0Ts3yS6roqxGIdTohWaJkOVzAfC\\\\\\\/Ng0jxK3qWAZPKeJuTB44jedS4hKqhnniHM1JYZIucWX9b0GSZCCrsObpxj2pxYRP7clsjVW6o78jPPl+NlYfy\\\\\\\/JAfi5lh385hd0NmyaXQxePGRqQWF3rZcyhz9elPVDRgtqjDIfJerDHGKzzpENDb0kydd\\\\\\\/0l89\\\\\\\/kEFL3InA76i527HtDTr6N4+4SBNt7Uj6dTy80ykoFU7CD0xwa6SfNjfuiuFjzu\\\\\\\/euOHRFjG1ES9kRoGy\\\\\\\/lN876pheCEHaYLGMUSefqO+vHh3zN13BQqn7SgfThi+kiu5gZgavQfcPWWrxl0DPg2KwHfas6nLAgMKYDXcLH2B+a8A0GjDfDHfCDC67ICkwdViRMFCDpKhVcKTBVMHOHdCXXQeCPr\\\\\\\/TzlMY1HAximLyv6q1S2zFfcOM\\\\\\\/2XmUxaL+dJ+PA6fLGWfksybb7CQeEvK4E3bRXscfFZ5AonlQCsWqpnnZt0aFdHP7t5Egm3sISp8HQ+WvY0YQNa4THOrwken4\\\\\\\/aOKMtE5CL3C3fM\\\\\\\/0MyBEKFPSTOfet7UEGdhd9j9JDIC\\\\\\\/Ptkh7jlP8pQMWcQY\\\\\\\/sg9gmU54O5R+7CnlH5GBX1FFe9\\\\\\\/T9UzYzFOLaH7HtlFGFMqTIKROs2cuOup3+n5V0JfWgX3pY+cLh2YhGaENmRpCJ+JKpCF8LblPFH8tmoXykrR3KvUT36gc261PjvRBYmE62YcobUZqWfIA4iqbmbgc5+d36ZOf8Rs4dQbJPZUj87aK83TNvi5lEcBCAw1J67isrRuObXC44evoL42hXNVovbckddJE5jCXg9PriNw3r7HMyN1JSO1jLEZPJVbpEPT4z+gi\\\\\\\/p0RQ9XGDC0xFiNKD5VtnD0Hnfi9FgLSMtMClf2duUzpGDdC2z0LLgQKbqcR0ZRrpPlUldJd6zfa9QM1FLZHM+lheRya1LIf1r4SzCgJgBVYKUrqd+AvRW0B\\\\\\\/b6FJW40WBt\\\\\\\/tOta+kHcrDwnpIPoiCY2\\\\\\\/ov\\\\\\\/nk7vAMZ\\\\\\\/sf+L\\\\\\\/wb+Vu9gu8PgLCFyEAMgDeizmjOTQQM82J5ak\\\\\\\/8GcELbfM39GDQ9SWa86MDEaCGT55uHBvSSlLb0XUersaIp\\\\\\\/0pidH58p1Wi4Q+nv0+U\\\\\\\/RZmWpMgy8k7cpxno2hhK28syuc0pW+uaNNtJETA44GiK\\\\\\\/YRIaPrqGYBsEMU2uzb4aqsx5dpNsLK3mSCEtN7VyQHwpatQYFw\\\\\\\/GxOIo6sj\\\\\\\/HK++5o7D2+RSrGg4nqxC+4wif47xHKbcH4YmDrbVALJyvlC904sX7MvmTIG4eeQoM+RXDLjdpEHxeG6pVHJF6mWndv3PYBRv3n6LVmCwkUb0pbtqoWEJ2Pczsv1904KQfG8UzX\\\\\\\/1als9UphGSzGHIIuxTrCxdeJCzn\\\\\\\/dAizXmuwp9wQwA32X0OM+i\\\\\\\/mfn6lg\\\\\\\/7IHBXPyu1dLsbbxq554Q5l6nJwjWb5rOF3jlTqO+8OAaAb1H+fVTD+jrgCEGAr9P1erNYrJv9Omd4ot8KfCblKfl4BSYKXLOOighpPmPm8LFblT\\\\\\\/6oiZ+g8u\\\\\\\/9mLPJszlYxzJhm01fpkfH6FTo9W+2FxOQU9GMoE20qO3nZKhVnMMet1PVr3c0R4KgHo26q5xVcyeKkhaYBi1k21yZdHeZf1X28W+E1+p2uKMQA9tgT+Dpt28zQDUuweHgsirZzQCyRnDy\\\\\\\/lWLGm9ezOyIgKlP\\\\\\\/KvnjaxWxd2pRYMHaly1Aub2\\\\\\\/h8fG7bOe5h0Wym68enMtlb03yMURCe\\\\\\\/JIJxMw4BPVeAL\\\\\\\/AybwNnC4MG9VNcmiQFhJAmKJ71cDyILA6y7CIGD+Jgcf9oiMSEQ2LL61kWDpxz4WTpy7HUs6fzb99j77Yp\\\\\\\/cTfyoL4Sf7gsHM4eyDyYCHN9ACZzFi\\\\\\\/MRGQgNktXm\\\\\\\/m\\\\\\\/D5SLZF9FoBH\\\\\\\/IOrZbzXwXYIw6EbfmLG+KpG0ZUSboRW9Bg75aHkZ+RUh8EsO7vIND0pQbCd1hM5iPi+HzovsSqyEXuHsKqnCiOdX9QgHQkdFz3L0TTE1+J8Ku3YY7ENIB32PGtH5io9CctpxdQVCgoirF+kfkGMuqMWzIvwWu0lsAlxpWKnDuWP\\\\\\\/CBEaM\\\\\\\/MvtFeij8RoDxy31MsIiNm9D16MkGQs+3G2P5TtLmtZvY1cGnFWyYNtaPB6eukliWr3eSPqcAL+FglHtlxojpLOp1vNo7+sglEtBH0QIK7EQVoyVvNzCnros\\\\\\\/mVlYMY31dzkYU1g3pRLuBtc69nJj\\\\\\\/oG+5TFt60KTe5U\\\\\\\/mCbYmzx4xz87mBz42tSDmN45UmLDqyIc9kucUvLgpef5Sf4dpYS2eY2GgIP0BePccpUw4HvJI6gSi\\\\\\\/ckvYbqFhOGYy0cAsQBH095XVe9GTsIt3icrS0qrX1arKwszo0Fpw96icsSU8UMJZBKLyx01PhxuTa57agYYe1K0MF2y9DGz3fh6iMpudKPvaZqrauTcoCOm80iho6+M527vPHvkmQOcgMruiy2dJZz1Ac\\\\\\\/1JfPUkZnypHRONGGsfQYdjFDPHROBRq4tn3XJh1IIfREIwncBl+el0gFcPN7qv4DQ82NtMSxZrjf6fCbtmiRt+nzM64RWRqiQzol\\\\\\\/GfUz3HQ8axBm6WKd4GwyyjPynYDOwW8dEhPDtO6u1Qc118BNR30VI+dOi5rRUcPgUbyK1Sh+3ilv88qcZidC4pvHOqiu8Fo0E+VEwGo995VYDtt3bXrLLnP1eCSVa+CkDrd9KnN+zHni0zNQW\\\\\\\/DvduPKKPhTvQqikmbbnAmX5OZXlEn6\\\\\\\/3H\\\\\\\/uBRqy8YtFzd9i6vAWZt16DSGQNg+q82Pk3GNHeED+JYJX4Jxra94\\\\\\\/8j9AATTiiM2ZPPF0YvRXfjGS8C9yilZMfsE16g0olyII6Te7gkpseHRMER6ujy1TZRSHIT9kTKCo\\\\\\\/PihItDyoOa8\\\\\\\/Vx4YO3dotubZacBLPSQSStYlCN2wi\\\\\\\/ImimaBwAHkM6yHVbV9kc\\\\\\\/xhjX6CZ+00Mq+49cSDDJ5NoaAnuhjXi5i2V43M8+3B+WLANA7bOFvJEzPyU8anYhB1lIJBiEmgAQhJBcVo1nPDdliAfxrFgEIaabJI7AKy7ZieLrgixS+S2dwKD2N34zXy65OJh9jUiacdwAIz47lpB8bT1yRrb0Ed5VRjb1ZsUESV6Um30rzkBogw6YsconfGgyoMgiwaAhSGE1\\\\\\\/wT6vCh\\\\\\\/FfxZd2HojqumHyM8kn4\\\\\\\/FTDxEggnGoIzEG5U6Gz8dfrJHhOkuoHhMo8f+Jqirnq8Cy2nBM\\\\\\\/uerGOaE2I+NQRz6b7x6KUkeQ8R8xZkgPiEMgFL3tptYYYBxp3mZT2pgajWlFmnpY3CJ+XUiavi2eAl2KUmoLhrSJvDtjKHiTyqAA9LLfiJx3wLUShx7LGlpojCVsguIqLFUIk3+e27ikZMZXyBIXmfW+iIQvFo8R7ayqkCBECIGy0kQSsuPD4vDgWolIedu1+Z+wQTkrnWvCdWt6aiMmN9pqAiw0BIrzmCsxZbTnZNAH6\\\\\\\/vvTJVyElCDnWjX8tRGjsF3LMhClpzL+X4ESAzRN5uD5ptBPOQSx\\\\\\\/1h4ERDkXoXUPuNQctNUkjVuFYbv9ZCdGMai0DXXlTwTVk9eQn1HEg7fB2aXxDADfjQU7u0oo42tN4uHeokyVadhR4eVv2ZdbuoUDA\\\\\\\/dTXQmgd3Ib\\\\\\\/j4Zt5+T\\\\\\\/9WsNyxkf5vc2Y+Z5cTI0zPm+FswgALz06rpprji5\\\\\\\/RCM2h\\\\\\\/DXZUxKRFTtl+HzssVN1r9OpshAICUNAI7+A3GF9qZ0Db3WQQ4ccdefG5ea47b6EcFitkjMMnyzyE6I6b\\\\\\\/\\\\\\\/RyIPdDmPyNr9UwfWlct+IKZjIzZZ1NJeCOei6WHVbbTDgT5JqFCB5ZBhFxbNSZe5yyK6PyrDxL+ohRwRV4h28FK\\\\\\\/xLVsEtHLHTb+hIXPZHTpcUuQ5EhKSh7PS6A4e5qcBONlL4sGbT12qyeyYUWEouaByIsfDgW\\\\\\\/GsjSVEW2aamDsQwT7isECeSUAxLm7T00F7+VznSB53Lusk3hXXA0aAWWi0t7WslZM79vQuC+HpWX9CXUzLXJ\\\\\\\/aKJWW45FTupZd31Qb56MAxUhWVmXX6NpJz2n34w39aB3WKDT9X2HHooBjUJ0Jpcapioty17P6jCMPj9ixmkBs5O\\\\\\\/2a1AhXRssGLctXP6qBOfpLmFgrNQ2McTlLWQDI8HwQuzMXm4IcH85dR08WvDeGnnXeW4+jOZgXEsUx4u\\\\\\\/g76iy9mcAo9mk5JxuxMkGb2+qQIkJjsI3X7ldUgowVASp6OtCxd4jxR\\\\\\\/oYc1UtfKaMWWsVZJgNyhlBJTUx2x7pDTSVDspYFPcbDtdKHWOWkAgK97THhe8vvwCdkYnaPGOJr+zwu+\\\\\\\/qR4ULVRchAmy+XLekQdD8qxCC9+\\\\\\\/6CSqqC0o2LjkwK+6HXYNQWL4qlR+2wFXgX33iiAXJCm85nnvtf0nDmZxvbJbQQASEOY03lOWr+Q7jh8dVmMUIHRElzVoMtMCt8U\\\\\\\/WxWkMLACYiMUXTd+xTGVFRKrirm5dzOfgdSSoA7lUyIC34OypO1NbxOoWY3i\\\\\\\/5gliiwSiC8uYhB3dbzZnZiTu8MY3WmDl7DMFH4UZBFuOTkyk\\\\\\\/qkqZUnS\\\\\\\/a8UlFNHae70S\\\\\\\/2\\\\\\\/EOAInBnJ439ZNjbpMTayA4xjT3oAZiZVoTo1xFlZpFzasYjZRjpLqgtna6MLX1FHk6kGrDFrpkk53xED\\\\\\\/QiESWxpbhgmdKb\\\\\\\/C1Rsw2+VPb22y8fokFVEnBcR2Y0B+t9sxupFMd30H285vaGta11RvkTC8WC71hmQV2bau4X4kRnYxvNZXCmNO4PoGh2WjOXYOMrSMLjkBdAtTbAKuELRRm5Rz6H23u7in5H\\\\\\\/axz0IfPiuEgLUil7OfhM3PMy3pmxVYDbhmJ6RJCC1InJzv2HFUzyUvGKflNSHDPf5TeXjVzTcaxpmxPI1aRNHakZ4xvzCho4zqG9gJ7BgSKP6Xg78o865NMk7oFBOPvHChU4tDkY8gPrqXjWJhOvjXJEmsUGIc6Zv3Q3M9rn19Af9BSlkmjD8p7fO7MQ6UVk\\\\\\\/Q01kRUKMqftr77oBb6gZ2ywMVYyLO6CHBmm0YzEFEaJ4kgSEGgkM2AbNjQBmYk0bikgB4EhednKnhHJMOhohmSzLBFWHECcdMq5HUsXKDyBJC3POIu8RlwQGJLCA9FhpHeyzcE2rZQ6hthFFvCDm3gFqnlERcL2H5kkqBEEyAbqDGHzzg1r6mMQwZW4AlszZ5uv\\\\\\\/bBtZG4HQKR8qhZAbuuL5iIyMNRvhrlJG8VU5F6Kh1f7H2zTNsfuH9eT4AsJy+56M\\\\\\\/486kvZj29ENItdD+2d0ulMgrstI1mGnFR2gVuRZk5CvCqUHAt77j0yz623DKV2kUaIRut+I8Ufgjeb99ltQUAUxSxdYGoujDXR+V0Lw7e5s9IHTab3VGjXjG54Ys7XA9Ufb+NxRqIIf6145AYBuBpvxlIFfwxRgb0VUk\\\\\\\/v46op3bxIY7vqW8It803nG0GME5KNYypG8Ovy3xOYNMi3la\\\\\\\/3lfp5dMGYxiNWpBvgGIla4YawnVDpIDCSAApBjzIf7IJ9hJqjft5nRfdOkBgszTJyCzxHoZRXEErQYmk3qbWtfjVM++xlGGkWSXiso4p21R\\\\\\\/yZT\\\\\\\/nncaEFSBDln5MCapqHE2iyGIGyAN1\\\\\\\/B6\\\\\\\/Qxqx6\\\\\\\/sd2tGNa3xEQh5nNTSW1ET0zyYwSQwyBf\\\\\\\/7PQSkZmqGKqq1Y6SnYvdzr9IrVfocZNXVX\\\\\\\/fRvxh8D00273mjtLxgn0l2JDCsJxuAc1g6080jxXBvtyK05TgqIKJQ+kP5wRSCROoX3vt2pI4zl9wBcny5VemJdDKAWfR03c86u\\\\\\\/aKWVAlhcuceNtrUtVHJgpFr1PNkiViFoBfv3ilNaMqVcd8ocMF199M4az1Ah3QhvyP\\\\\\\/l5M2lkqcm+83P8fG64qd8a5nJXTMmAXfPHqJOOS7YohzPR5JfZK0cHUa5mU64KC2TmuMJG9L6eHS1brPu8R41g6g9rE2ZFVg5YYDXNq8ai98xpIVXoddKpujHzuBHNrc7O1aladlvF1S22FIGJS5dPLRZHzwebd+l32WnKgIcOehzM6iSTriPsgpgXj+WvA+8UTEUwJ+j8ZCIFLklQedXgocC4T5o+f\\\\\\\/Nk1LOx4NKBwtkGad6enbAx5L6YAC31gbVgHyG30jPsshAjWHu8pNIg8sH75UrCcnDL7gY0yEiwC085VJENZhA57+vmliSpS6yRpZj3uoKyIX03Yl7Bd9Z5Fz8tgc92YunjImgdUutAgc4n1E2I8d\\\\\\\/\\\\\\\/iPk4uQw3MvRwbniZGvaQgRD81KjN6ZUGYej6mX5JtZXCK6X\\\\\\\/IZ3BAH9Bq4vX16VSXFlzUF5Zo0gIIbfRBIcKsVVcaNH7sAwEHPS0+46+6kNTH1+2Bn1YJuS3Jwo9FCwmrp661UDsVYYOEKY7LgzdrZpOsSGtLKT1F6GTGtzPY6i\\\\\\\/eWdAI9iC34SPgRqol2zvWefOIkikfaNbNTbSBXYapT1xVvVX6ukyWfomIBz\\\\\\\/dr+kBSjZjdC8GSMBR1tk1CjqYXIpIhCi6ybscaTde0EvWU4gK\\\\\\\/3glVTxAHhG\\\\\\\/iGU9XNXc87Vmkhqyceb\\\\\\\/IVzj20YWzFOR8Wqabpbs3DjPp1IVSMDociSzYyBHQWnSXUenKYKa\\\\\\\/r1nsfwQ0eeYo5pq+9+njL6oQKrzIzPoJxG5c+G6Tt5nVhJTGqyPMcmlW1OrjcwtwkcyXPb5S1Nga27+6NqK3KzDb4JX7czRbBZ9XeMX3Z4dOFBMgpX\\\\\\\/gbmie7Rksxf7r7KRlHVQwf69ZdTOHaskA6fNYtM8tF9l0ye2vY6VTm7hQUcNq9oTpQ3SPKt\\\\\\\/mj6Bsi9lisTqo+edm9H1GpwiyNQHObj7yPS8JLPmU68z5unXPgp3SBE9RIy+in1vPAWHeZukcMYm28NnmgOAcqAJcusXrYiIQqkxMKPuWIiHbJTMp2MxONtuwaz8Llt4SRO2xV0Tv32vX0qwKT3oSGBfhKaACZq4O8fkVpLMEm\\\\\\\/E3uUIPTnmmU+JNriGbT4IpYpq\\\\\\\/ceO1XpEHSgXwvhHffsTFYlOifRHSXJyY\\\\\\\/B3bT5G6jeqxjUuJTerq3mLRiQ9OY9Fvwg5g6EDDmeW63JHizOJK1ZnapXZqACZCvwpGBozWF7Q2C7y0dP1AiOu17ztv7RHImemnZLer\\\\\\\/q1OAB5hfOVm16qUe3yZngvSmtiGOuhoOHYDZJ3x472AzcRuaQlHcf4r05+rLwoi1qW1v3zyRYxeowsh1NSDZ1qmGZVa55FA1TcNE+g+qxma8NINOT9a\\\\\\\/shtcQNVwu\\\\\\\/lDyUZyeQ89deuDtOLXnES0dA8yYvKq\\\\\\\/YhrucAv7pYcgLC2HRzuANhW31uVMNwQX6RNqgRZeXyfaXchKjG0mnv9MoTbQ+goOv5663be7PkWqbnDP2QPxXnTI\\\\\\\/jUgoVc4lKjnWBXVZlyvuoiJ3Ytr27ct5G3va7bXLVFZt6ok9ko8I6iuwwZHM1KeQYPGiqKRj1c9brTu3yXoSkYVk+7ZmD9Z7bEcgo+xTe818ZSOwzjB+gxchQXah65f1ePRMTWSq7HuSuGTDdtVtEBWvyrMV7fwZwOd4\\\\\\\/5w\\\\\\\/Dto\\\\\\\/p4i3EQhrZCGcgmtVyiLJhP60kAYHTTCPAy7ltAK3XCdQtqu\\\\\\\/bP0gLoAxtY\\\\\\\/+RDEJClua8CzkHc4y9CPK90zHnrUdz+TNKufSJ0hUpfZ1PKHq9QXIAIjiubjqIKw0UgRL2wVG72iTqHeXEfpKdvkpDTQIE4pjufhOtK4WudXD9kaZ5jmo2QB38DVTOtmRPMIbjJlgvYx5kE\\\\\\\/r9njSV\\\\\\\/j9gsx9gy6hA0fpdEyeRdG5DhRkAHUGPQyceNwQEl4yUB55+sO8RwSZusdh4lwti6Mbphgkdbl+ZXVhNcLysIe1LnyBZhR\\\\\\\/oznx208nMcp\\\\\\\/vYHFgbIBfNPioQRZBnRAwMcJ72plDu6Ww17YwkYQ4kPE5tLIXFXMUMvtX07ri5hunT6iO6\\\\\\\/HQEKCFSlpj0wcztxZZpZoXf3OrYwuA6HbwOa9yz3jWYK1kcje7V0cikvTsBhkgVOHRaca\\\\\\\/7SquQz3LC6rYU3Lqnk12Kgw640vbV8gUbZ1bbFz8HbTunVoT0zgKJ0w6kApfzNARry36EN21uJSix2hLeXT+X0DJQbgWfcuUSDRVLTxaRelDOaxp1JpJUADfWmjtdga\\\\\\\/qU64sUrlPE05oN21zzhmIxo8GSElUrB1gQQkI+HqrnsUILjtrP\\\\\\\/HCbYA3r+SULIv4MKO6uPTwZS58a1HgvRrgM2Pkkfeuztz5plOtrM0kdjdSnLyOjxHyUm+JzwZMb4en2DYNbPc+ivDIqcQdbXQG1rVLkeWS4IzHcod00DAXQEju2gTSTGQqt3c5WvxwtjqsCkmmdHAQaahq5Wa8LAPxSH+R1cF26WjvtfnJaoYs67pjGVJQOGg56Wh17J2zUBCijm\\\\\\\/SpTBFpE775XRTQ3s+nmXJJb7cGhzS2W+llbxsFlk6MhtU9YmtHnTOXyiljLHxtdHssdbHIjW79CdT1O9j2Pk9dykuBVA9IWktXZ8W+cA\\\\\\\/Fc6\\\\\\\/Z\\\\\\\/WCJ1MEiefgwLzD2QUHi+4MZp6RDRZ\\\\\\\/oD3nEtJsKEV1sxBXjKKcTX+uuTcczYka4XY+dlGbA3BVr8OpgcMlLiReeXY4VNZZ1KccbrRxr9B0vI7p2mEAhCC+ygx+KRHwu7F0H0ouNApxgBqSfJQFLh6rwwLheAyncwKvzggyEzicpCcT6afqXmdhicQQGm9qBkDBtdhB4oCE\\\\\\\/l0s5ASByV61uGmGHuWYagDhR4fKF5nKXRXR3ZSlJmMRWCaNIw8gceWlSCZeQJrJPpHr6vHOe3oeTjC1LG3skmcDMWxpfh58nZrkLy9\\\\\\\/BPp0rMlfWs70FEP98o4FmH7n2EPXYcqIlPhZGJepB8D483DfDODMV9TeQmZPBTut0p8u66y+lp55eGFh99JNKKalgyXqQZZWzTUzbV86BTkF3GwsCIIYJoUFjBnwsaFRhTlnl5ZQqlgRTQ7qtHo6OU37fRe9UEVUWYBx6rPtlBsTgY2usmAJdc3jOY13m8ukTSb8kyENWYoxaeijcSUaN3Jt8ZILElsnolARxcdz9+63t9BYoORfODIFPEwZ0heM\\\\\\\/O3yHrWbegWBva+KxqwKxPlog93AE02gCkmsEMI8w6bfkFxhMVZPCr7Ek09Y8iQ1jb2W3vAERA5a7l4GT8eO91xuJslen+oJ1TvsIRz3JsX3ywTcUB33xjAzhzqtBo6VOM8Dywgt0+mqJ\\\\\\\/lssSOyDvsWeAXIKqFipMlD2RGCwC0y80r3EO56lH9ynxyMqVmyE3yTnJWhkfLcm37sAXGl109Rariv59XpwohTWkocEGl5HgEMO4sR7BeoNnFuiFFT425C\\\\\\\/gWU6EltoWu6wcNESYNkgQs6b2rVweYD9ioQlgqFRQly0jPC6k11S+iDEiWwNltgTSmmb8Y02j1A8+RIkZD3MA9zH71RB35hM73Opub859s7h37yeIClsK4wbmhlmBcwgNhiDmHHwvXjsAVHwO5k7lIibgD3uUllt469YkJaIuBVE29PjoNKAc1hEyvWXDUrrHyuDuFdUHCZGkuCrWmnVRTWq+I8xg9zwwVJaD6TNqTuSU2LWl8zodYrLPumuA4QbPUYlJ0ssJ+Vai6L1Q5EOdgrGzPcWFQfkitb0TKROd4BtTGMNzfH6FL4u0GC5Xq9SH86SsuuSN0dTWR4N3jv9EfEmVunoqKtseAGBBr7DQGDVMDRvQrQh6g28x0+SDnPgbdCeiS5flhDV2xnqkffjzDyLWRFXB4qJ12qnVqWnKM6tEAOqSzqJG8uWcMsUsUMK19YDGdOGV555p4cUAHOhHqSUNv3rcj8BG7RN8i2ldHWVeJY2i3QFFBnwFueg8he1fxoQ8r98YJzbsfAslT9lO2cIp9p0lRzfSHFJjKFGlVyQXPamTirVD9fSDklX9ojQctu5OY7LOwrrYrqhQIYcB4jQrYJiCb7ZUSOC+MBe1GGxU4niWn2Sci\\\\\\\/RGRiDHrV79p7vVOsxPZvQWQvt6n5re984HlGCGLF6RqJLWzQYLdqbJBCgypfpfMTRu0l6Y9xuaWO1CdGMzfwXssirqKRd\\\\\\\/nZ57\\\\\\\/\\\\\\\/qN0zhNr\\\\\\\/tFTXMsyri7QZBAk6mkTfASqDf9W69h+nLw3Xw3SE8HMRuWymtCed87kJHPE+9aHBdMzb3gC\\\\\\\/w0ZInCGGVDML6HMp5zb44YW1VjjEm1xzIqaC43xJYdnuS+GElCkIl\\\\\\\/8EOHRDKfhs6VOvfD82Azw\\\\\\\/PVrvV2YZ5Ym4sGLkH2enGAkLXek7nuzS4C3OS1y+C0qCNp7NU34VzxnCNSFATt0LId4QTIx+NjE+fe1m8oUcb8QzFtR88KUJZVSBeTOMNDx2dAJYe2QrerbASk7Kv3qOxmQhGaxA\\\\\\\/78v4XqYwzy2pK\\\\\\\/B0USsL7xlOLWD8c11JHnsOefKjwJG\\\\\\\/pW0tB7+dkm5oX7Lb\\\\\\\/Pfpwb9ovEn8BBr5cK+\\\\\\\/MUZphvOgK26t3Bqj5enWLEvk86W\\\\\\\/+Kv0e3Gd3sXuY5\\\\\\\/wejC\\\\\\\/nDhbtA8HcI9TL2WKJVwCdTIyThDmoWaw14XBdI7NQCmfzaOcgKAzyIqlN3JcbiEckkxrzb\\\\\\\/xLXPFfi5ResQl+9u4mVrPIfAtIACu97fWJAaaE+aX5CKp+FGWlgwuewoCNR+bEgCJ+FjPvWFSWR6ySfgDFS1uIrN5qz86CcDhh7RWWDKWQMGcGyp1LB9\\\\\\\/9jVEjK410MeFPj2BV+ylAlEjo+gXlEtbjSFZGvc+1ey99Daci9hLGPV5Xh1qIJGX96\\\\\\\/rdjEOykxTqglZuRCbRS7DI7pVXvT6QCLwSiT8Wtm289Vitwk3HYDc5Db1N\\\\\\\/wVhLfDnJNTZ+qF8euYue5TrloWVaehKrDS07U1C7BiDsdwsh7JuH00XO3DOgo4Ru\\\\\\\/ZQCvK2sFmZVHN1dHzJCx4LW1FV0zVSHeKUcNlpb5giGLEb7DR5SL6VS3NIP468T6Cx914IyrtmQQMaWV2EiS3Gkt4tRWZjZyJ9EDivkNrZxEiK7RW\\\\\\\/Oe7vU8V2fnlaMepk+VTszlknCaPm0sfd13W80x9e3cJpsZxh0BmGAfC7AI5HZALWYhr2APPvLuQdUH7vx2T7wNOLUKgyvshVHvBfSoFa+6nc78YSCPZWtztoAlPyRA3jgBrZFOx6uU9ZHo3Ik86+Ag3zJxJ6x\\\\\\\/1ePk6AmNOaOmjVhA1TuQqXK2uoo8xbSsLhexIhbNqD2pvk4G\\\\\\\/vTodI2qo1bsBWLOT44Fr9EO5f7AB37uaLD+NYnRMUnI5n3P6+5FTyY9LlMbP2Jzw6BZ1dIu8\\\\\\\/9obqLpWQjR9rqyJ\\\\\\\/TlP2\\\\\\\/AxGUGFZly8LXq3LbksOrHYrr3hFHRSRgyBGavq81TJOP9urbHfiVdUQOy\\\\\\\/uSj1ehM\\\\\\\/SDnEaa1lCqrNiIxCfCUON9RueexVIQNTJnZDWF1YQuC+JxGyikK9wFilHep0FYrLQ3BglCJ0yOFYHFft6tOQHcSy8HhITCMI6d\\\\\\\/yCb45zqhjONr0AuXN3Vfqra7UjAlto9e1jRQqGyEVtqmCH2tj3J\\\\\\\/liH11SIXLIJeI5Bnrqo+FBqWzCBcFanLwFpi4e1XnZJeI2rC2CFAIUejAZL7+ClSgtbge7O\\\\\\\/RQ6WGytUTTOJYXank43ZAul8U4xNLsiQNlZWFLscerIPdzMSa8RNRawu+y12cszi22CKOuY4\\\\\\\/LuwLrl1kuf8FyOGzYCHIR6oGCVyxc7CWgwSH3rkqiZGGRRubs8lIyQBomXQjIpggNLmZBekdK3Y6lrunRZGY3Dyi1P0Xas3K7LmAsJrmvyAuBijnG9dNcmYFnv1zfpJpolQR8TujT60RqubPdux12v0HE706JZ5bhS7krC6rl\\\\\\\/kiDEIo0kr8H2FRtjHJdjpCqlMIXxYePEUAM9Ax7qNFJW42pNEXxI45PYFiuEw0WttxEbE1n9xQWtZEwJzMjWBEA86aIk0qBVnv3xt0JLPV\\\\\\\/2mWdwvLDN1NAN\\\\\\\/kt5NhV3O4xpZAIhcNEbRwi\\\\\\\/uqVjIX\\\\\\\/7ppGhott621CKqwA+3M\\\\\\\/1p6LpCVg6PsLjTY2JdZjrurppMtme4YgXQRhOWhpOXo7tEw9FI5k22t\\\\\\\/1WlnfvW8TCRplRA3LcAtGH7fkcifs2aYoAx8ftdMNVPKsfCHPw2doVqy1eEzYNjEa3CxaS\\\\\\\/iWFKsi\\\\\\\/Ek3EiWwzVUoeFZUFjsWuAuVdGwGnmtX5pozNMtefgRB01dv37RnA5jv9Ozg5CQR4zXeJoid7TVaFvgYqOF+L0zHIkUK0Yhgi23cK8j\\\\\\\/0SfEh2IElHF1t9tNUsFnUZHPfosc8BPYH+ZG0L9R59iybmVXxVRhSXsa+WeNVv1FtkxTrRZmcI77dI7CtodlNF93\\\\\\\/FI49MEQCIkBGaVnqfqGG43gHCy36HQEWOPBcqb\\\\\\\/NwRYfXXmvfUD6IPPRFptr+TqGUiHnSxlUgvHIXw+Mg3\\\\\\\/eL7ggJu943jmcxkYbVQA7PhExCSsRvTjWTFWcTbCNbtpXWiqMcTtgEBmHlCLfmTKVD7m4ODBDpsWO4ilrVJ6mWCH1h3hZcrlXGTUVL0RUquJIlQ9UxVJJQxLO2gHSU4\\\\\\\/sdg3ykQ0bKE3GtCsZDeuj2yT52097vTmEymka2KV1Fc\\\\\\\/BXjn7ap7\\\\\\\/Vc3m+iJwTfOfEmkyeoRaEA5HRgxamCzsVnlQvrMUftgPS49VtxHjjXrhl4XuWhLR1CxVH0+wihXzLSGF2R7tb8i+WkVLgMloprKjXwAlqvgB0j80YB0aQSp62K\\\\\\\/lwc8MMZ0DqpsMw5lMfVOxaawX3hcaqBwS5I4GzAgxMspBWSexfdV9IyMudk540dKfF2VttnKk3j\\\\\\\/Z4qoaehgUdruytMkJcRcADOeismkLsDIIUGm5\\\\\\\/bagLU3ojD8F9brp1Ud9S\\\\\\\/sUVSHEqcZ2nE\\\\\\\/s++M\\\\\\\/\\\\\\\/qX5A3ZQOsEkCg0J2a6rR0bA\\\\\\\/BPzh9hz2KrZM39CEIWM1fXjis7kAB66kYr2EsKHMnWab9oLfGQCXiJaMRjgeBymbLC4n1WEa\\\\\\\/AqcDfR1MWpnahrPniOg6gyi8gRk7xmyUOipBO5gdgZoGYEvBTfqMz6EiiZxP0HFmvT1wxXm23Ujhv0f6TFsiYHl7+QNA3WuC1T6g\\\\\\\/kHxLWETrmknsN\\\\\\\/dKi3kpx03m4gXwoupFbnNQ+i4XiSByFWtpefRD+Wm54rk5Ojizjt+krZmMK1xfjSQGRrgqjByHaV3DLf0VxCAq\\\\\\\/\\\\\\\/HadBCCn5l6d\\\\\\\/6P01r9hPP6t0+Jt2S+VJ+HJ5zm++9xpeiNccfT8Bk5oWsuRmgshXFZxLTh+oNZgP6+EGITUNDeOJJdKzGAZ7hjh22NWquB0h\\\\\\\/enrbx2kYNw9mdxEsP1rHGLvYdqFMA9\\\\\\\/jhNOBxn0eYVAcAyfZx1TxBdkN4IrAbu6Rxvs9mReVll45iFuUKpKg6QHJjf4+jjkZDJVnm6HNiz+k9EGylsBu7ltdydK9hCCPShss3vCI6iNH0huwQd7oIiC\\\\\\\/Hw52FBDon0FQf6d7Ojp1JTIBeJHHlmtg6ITTaGXfrXNnrcHT1xTAr5F3B4CegwYma+x9iGzSnpILtgIfFrEDLKvEtPtDX3C0Ppf+99mb7bIxSQiXgwC3iRuLFZIwelulyuaqKCZ32eCayJxs1IVggje9q588YXc\\\\\\\/fETjGHjMDtSmOV+TJIZ3DxCzFy5dU+av5aQq1E+kMAo2z6pRxIT7hkZrZJu+c\\\\\\\/tMqMKU\\\\\\\/Z0W64N4b9m\\\\\\\/U+kdtKpFT1lnPX5MoJUA2w2JrhcXSZw4+tT7vqELgckqsiwSvp97TcSGp5Dnu4O8d+Od62o+L4nR646atBx0wN7M+mCNhPqMiOKfhCcR2\\\\\\\/BQJrOp9QkoUthUazAreirV6a3Gql68dqgGUWW5EPEzvabzMVEFNTWtlYrfJHrc\\\\\\\/Uaid0Y+EO+Vgo\\\\\\\/WOnLrN9nxOnbhUoAwNC77QM88qcUt\\\\\\\/1Pl4aO2IH7ZI\\\\\\\/ek5U5fiwMIoMWZeQbl2EZK0Iy1Mzdraufdbsrzcxd2XKYZpjbyBikpoQTzKSK2tu7pUh4ox3AOGLLbVy7Z1a0lPDRtmFuvm4bHVnECy6sSvAUz5Hx9VOyB6N1Hv4iKibezVFTixNWn314uUpEQPSGZQyqForaDSFbxfk8PfZfH\\\\\\\/ZziJ4dx\\\\\\\/fc2DNFaQ14OrmbxTd\\\\\\\/h1vCAd3LTLhzGjMKPwBPML2ylChGcpvOr5Sc70iHhvE7R2xYii7MLEj2Um0II+yeFkzlB8Eq9dROvkUnDn09OyMLb3tVZpP8EcsWFEul3USj9fcQINXH8QMtf7zdgQCJMNrutw6t4YP9HAfB4nUm3kcvH5\\\\\\\/VE51vC4jXVKkgr6fuvC1OIGExdmOovCBbyg4JkY7gwbFjv61P7s4CfVElW5v4snfAvpDBxEHkZ3EGENOCRdsKwF90kAQIZKXrfIm2ah7oOrhHAZya60Nnsj4\\\\\\\/6XqAR7nq7bF+Uung+fNZfakSj3at0dDgdL9Lm9I0EensHUnx4OpB2QCRuaLz4qlIyc+VUcD7tI3B4p9vnTieJdKol13FVH14uXva22wuT8PxO70fTFdMrBpxXL\\\\\\\/DSiTSZ7iQd86AHjNxe1PfYuK7zPOZg4RMbkIGvRnE56ZUjPKKDEIY1IIg1PK8idLvipa0T4c7mXXrqFr3mCY+1ih58yjuDa+029N0\\\\\\\/JwM2aY6nVfxmAkO0xAJTVUU7RLap5JSChi2TsT7tCcPUQjwnwXHz5cUopI2pOssBFgrX2hvqJDqePjLAyAwApvE5aRBpCCD8dW9lj4vCcBiqP2QXmU1cXNqLmTBsEsjHrJGt7Yi0T6\\\\\\\/D53ophOHj6SbZQ6iQr0l03TiJLVekEC1d\\\\\\\/piKwHrgM139ceItq7L1FdAYW1GPlJ3TiX8Tpdg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022dIxlM2THIokacSU+sU6YVg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00224bd7667e3f1e50125c0b761c43a8a68f1ab5de39ec952049b5f270bfc5cdda25\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CEYRqYugryEAzI\\\\\\\/3Sq56wGIuAKaLq+YMESEZFIwjDg0MXsSpFVcYl5IiZocU5rQfB8gCq9lpObNLlWpIMZHXW1S6f6xDrCCRljC5\\\\\\\/zbg0e59lnH4CoDta6BJBGyo02SukO3FLdQU9OSzyjunNsGI5LLCBRABSo2ahwwLDcvRRspWqrYFQoPJFPRUgtUT4ycKHSlpXDmgiXYCrxn7QMm2DouMash2tpzdqSliW8mWGp4NEMxuD\\\\\\\/+le\\\\\\\/SMWdmXH6a4R62DQm0A3K5oNKkW5DPz9wLvgNgT5O+MoNz8EReRt1fx9taGgQqij60vEqbRG3Ii+Gs6ikFo+ZC4Md9\\\\\\\/MQoPhEHooZng9LizZUf22tsmG1uEO3bs62NlChs6pq4CJjhwob2pPKqOvb3RUufAachEddHnAvkJUe8Ak+Ze\\\\\\\/4WO\\\\\\\/ZecmYt5PikE3q34Ob4sAFOp+bJmucDurZZKe4jolTZnd3rPnYWWLA2B7GNu3jE\\\\\\\/H97uiB4zy1eI8ijFbD1o1HzgU3DipsyhyLoRRMw+cipbhPaAOqMEqM\\\\\\\/cjNwlkmw\\\\\\\/JPF705PBvXwZx8HagbNaUTj2I5rTm0Xg\\\\\\\/K1wLQsGDvSQjTNzxPs6sImQqC8g2QmbmGCV9pZowH3YCSOVm4ljFg0k7iyMjjyfr9HvAsWrs0FFIx329Xx4zDnQmTXr5JNMA7PNSFhjs\\\\\\\/C6sPfXGceZjMbq8ao04hCuKYjGCjKRpcRdxyBc3nmIbCib8v5xw\\\\\\\/A5OGqyHyb9KpVURVZV2SVmeLAB3xnfyo\\\\\\\/mQ9Ulerw97QaEgddmNYum7rngp9bJbg4DujxCF0\\\\\\\/FmSBNsI2LHjL1BS+cOASo\\\\\\\/\\\\\\\/KyyQZLco2CSRuuEENQwWKqIAOQNSJer0RgsF7OlygZufpl6pGJ4WCfEMjM28TTMM7SeYYWXWb2OXRZpPCWeqjCE4UyOkWjtDQBpOcQfKUR+xPbq\\\\\\\/t8tMsF3VyRJceyfl5N\\\\\\\/FG0IDCClky0JkohVisM7o8Yb9rP3k4NfZ5PpnXVyP1LBrCmQ7N3VPLZbnftzHggh3+UQTPsxBFvs2miTiII8SklKaPSSWeoRvJDz2FxRpPzu70fuE1tgy1P8FgU3v03a8+XlDZleC2HmxllPjxcKgG9NqR\\\\\\\/h9FT5d+qOmdnfPXCTUtTkSLduClnRchS7xhhggasQpwyyP5oH8dvYuLJb8+wzXRY1O8ZLNJW1cVMEywBqvmdriiWEzAyfplzCo8gZpyHUy24+y\\\\\\\/U+XEFkRTMS5rphEdDnQzWEIBSOPtxZA2Ed19df11fwK6zEkU5b6vBZ7fGFfNuC+ccpeR62YSJrOx4nt1JJXe092rPzmOShvAU5rcUXWiMc7NWBdsFi7LBcpXYVJOq7sE4Lc6vtwmEz1OTP\\\\\\\/DHE7naO7n6q\\\\\\\/\\\\\\\/qn7Kgotsp+YPjb0aeDFidVtwcC2LylOTxqIMn+c6uRQQEWb454hY2ZgK1SOgYwZYiWq8UvsKWSr2m\\\\\\\/LxvSCeQig0W7gvNfe+rX2YRwSOT3Iv1YHzY5vkE\\\\\\\/OgDrBbLNSQCjMcn61D\\\\\\\/DuR2TABMeP4tDFA2ybvWRvAFGA0u352C6j0dM3wwSzpqtGCf99nlo784V1B4\\\\\\\/kcpALwKnam8sK1YTO7Xjiff5O6k\\\\\\\/CqdC9FfUtzW+9H2UlEPNuAWj0niuSP93jAVpWB+IMP0QmD6rVRvAPUkEVdognN6PpniZkvVJNBT6FdXgz6MZ1e14\\\\\\\/OUZeONzVgLPiRlpBCLEreDU3Oe\\\\\\\/6qwfixKpmNfR5eN9kCg6nbWPDVm9dNKSASuHKmJd8CErD2mjgXWYp9FVjD+28xnjapM3BNxgMuTmdLqHyTBcfP6thaYnL9Z915s5jhtkFvV+gXvzfYmB2oqkYwhtmtDAsDQHa1UF3Wr6exI50Mk0bB8pdK9bVrMTS3Zg25IFyZCEfrcUreiMsNAhY9QKMbMCKhNLicuhpLDCR4wbUntTakokpewNvVhze7xdP2EBAnTM3sQeD7kFCyB5PteFytYsOXyVll8W8iXHschpA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022wwhNjrljbE\\\\\\\/hdYmZQNbWTA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f3acd0f145dfca0a7f060907df0125133def649e4fd98c8dfefa02fbb518156c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460240 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/29" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UNxbkEDGwUJRozWiD4uvvj5mwqdI9n7Dxm8rhS3jewU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/29" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KV5aCfWzypm6EKCtLADz_QyD9R6ZT-d42TpnxBxlA6M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/15" title="Comparative Genomics of an Emerging Amphibian Virus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/41" title="Disease Variant Landscape of a Large Multiethnic Population of Moyamoya Patients by Exome Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460240" id="node460240" data-pisa="ggg;6/1/29" data-pisa-master="ggg;g3.115.023242" data-apath="/ggg/6/1/29.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Haplotype Detection from Next-Generation Sequencing in High-Ploidy-Level Species: 45S rDNA Gene Copies in the Hexaploid <em>Spartina maritima</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Julien Boutte</span>, <span class="highwire-citation-author" data-delta="1">Benoît Aliaga</span>, <span class="highwire-citation-author" data-delta="2">Oscar Lima</span>, <span class="highwire-citation-author" data-delta="3">Julie Ferreira de Carvalho</span>, <span class="highwire-citation-author" data-delta="4">Abdelkader Ainouche</span>, <span class="highwire-citation-author" data-delta="5">Jiri Macas</span>, <span class="highwire-citation-author" data-delta="6">Mathieu Rousseau-Gueutin</span>, <span class="highwire-citation-author" data-delta="7">Olivier Coriton</span>, <span class="highwire-citation-author" data-delta="8">Malika Ainouche</span> and <span class="highwire-citation-author" data-delta="9">Armel Salmon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">29-40;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023242 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460240" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Julien Boutte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julien%2BBoutte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boutte%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulien%2BBoutte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benoît Aliaga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beno%C3%AEt%2BAliaga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aliaga%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeno%25C3%25AEt%2BAliaga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Oscar Lima </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oscar%2BLima%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lima%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOscar%2BLima%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Julie Ferreira de Carvalho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julie%2BFerreira%2Bde%2BCarvalho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferreira%20de%20Carvalho%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulie%2BFerreira%2Bde%2BCarvalho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Abdelkader Ainouche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abdelkader%2BAinouche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ainouche%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbdelkader%2BAinouche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jiri Macas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biology Centre ASCR, Institute of Plant Molecular Biology, Branišovská 31, České Budějovice, CZ-37005, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiri%2BMacas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiri%2BMacas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mathieu Rousseau-Gueutin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>UMR Institut de Génétique, Environnement et Protection des Plantes, Institut National de la Recherche Agronomique, BP35327, 35653 Le Rheu Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BRousseau-Gueutin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rousseau-Gueutin%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMathieu%2BRousseau-Gueutin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Olivier Coriton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>UMR Institut de Génétique, Environnement et Protection des Plantes, Institut National de la Recherche Agronomique, BP35327, 35653 Le Rheu Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BCoriton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coriton%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlivier%2BCoriton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Malika Ainouche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malika%2BAinouche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ainouche%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMalika%2BAinouche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Armel Salmon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Armel%2BSalmon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salmon%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArmel%2BSalmon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:armel.salmon@univ-rennes1.fr" class="" data-icon-position="" data-hide-link-title="0">armel.salmon@univ-rennes1.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/1/29" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460240" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460240/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/29.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460240" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460240/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/29.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460240" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460240/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/29.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460240" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460240/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/1/29.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460240" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460240/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Gene and whole-genome duplications are widespread in plant nuclear genomes, resulting in sequence heterogeneity. Identification of duplicated genes may be particularly challenging in highly redundant genomes, especially when there are no diploid parents as a reference. Here, we developed a pipeline to detect the different copies in the ribosomal RNA gene family in the hexaploid grass <em>Spartina maritima</em> from next-generation sequencing (Roche-454) reads. The heterogeneity of the different domains of the highly repeated 45S unit was explored by identifying single nucleotide polymorphisms (SNPs) and assembling reads based on shared polymorphisms. SNPs were validated using comparisons with Illumina sequence data sets and by cloning and Sanger (re)sequencing. Using this approach, 29 validated polymorphisms and 11 validated haplotypes were reported (out of 34 and 20, respectively, that were initially predicted by our program). The rDNA domains of <em>S. maritima</em> have similar lengths as those found in other Poaceae, apart from the 5′-ETS, which is approximately two-times longer in <em>S. maritima</em>. Sequence homogeneity was encountered in coding regions and both internal transcribed spacers (ITS), whereas high intragenomic variability was detected in the intergenic spacer (IGS) and the external transcribed spacer (ETS). Molecular cytogenetic analysis by fluorescent <em>in situ</em> hybridization (FISH) revealed the presence of one pair of 45S rDNA signals on the chromosomes of <em>S. maritima</em> instead of three expected pairs for a hexaploid genome, indicating loss of duplicated homeologous loci through the diploidization process. The procedure developed here may be used at any ploidy level and using different sequencing technologies.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/poaceae" class="hw-term hw-article-keyword hw-article-keyword-poaceae" rel="nofollow">poaceae</a></li><li class="kwd"><a href="/keyword/duplication" class="hw-term hw-article-keyword hw-article-keyword-duplication" rel="nofollow">duplication</a></li><li class="kwd"><a href="/keyword/paralogy" class="hw-term hw-article-keyword hw-article-keyword-paralogy" rel="nofollow">paralogy</a></li><li class="kwd"><a href="/keyword/polyploidy" class="hw-term hw-article-keyword hw-article-keyword-polyploidy" rel="nofollow">polyploidy</a></li><li class="kwd"><a href="/keyword/bioinformatics" class="hw-term hw-article-keyword hw-article-keyword-bioinformatics" rel="nofollow">bioinformatics</a></li></ul><p id="p-3">Gene and genome duplications play an important role in the diversification of eukaryotic functions and the formation of new species (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ohno 1970</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wendel 2000</a>). These common phenomena contribute to sequence heterogeneity at homologous loci in all plant (eukaryotic) genomes. Detection of the duplicated copies is essential to investigate species history and the evolutionary dynamics of genes. In this work, we focus on a multigene family widely used in evolutionary genetics, coding for ribosomal RNA (45S rDNA) in the context of a highly duplicated genome, the hexaploid <em>Spartina maritima</em> (Curtis) Fern. (Poaceae, Chloridoideae).</p><p id="p-4">Polyploidy or whole genome duplication is a process that has played a major role in the evolution and the adaptation of many eukaryotes in both animals (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mable 2004</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Van de Peer <em>et al.</em> 2009</a>) and plants (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Jiao <em>et al.</em> 2011</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Levy and Feldman 2002</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Soltis <em>et al.</em> 2009</a>). Whole genome duplication is most often due to the formation of unreduced gametes during meiosis in the same species (autopolyploidy) or in an interspecific hybrid (allopolyploidy) (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ramsey and Schemske 1998</a>). Autopolyploid species contain several homologous genomes, whereas allopolyploid species contain more or less divergent duplicated homeologous genomes.</p><p id="p-5">The grass genus <em>Spartina</em> Schreb. belongs to the subfamily Chloridoideae (a poorly investigated group in genomics), where it forms a monophyletic lineage embedded in the paraphyletic <em>Sporobolus</em> genus, which recently led some authors to consider taxonomical inclusion of <em>Spartina</em> in genus <em>Sporobolus</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Peterson <em>et al.</em> 2014</a>). The <em>Spartina</em> clade is particularly affected by recurrent polyploidization and/or interspecific hybridization events (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ainouche <em>et al.</em> 2008</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Strong and Ayres 2013</a>). It contains 13–15 species ranging from tetraploid (2n = 4x = 40) to dodecaploid (2n = 12x = 120, 122, 124) levels, with a basic chromosomes number x = 10. So far, no diploid <em>Spartina</em> species has been identified (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ainouche <em>et al.</em> 2012</a>). The different <em>Spartina</em> species have evolved into two major lineages (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Baumel <em>et al.</em> 2002</a>): a tetraploid clade mainly consisting of American native species and a hexaploid clade, including New World and Old World species. Within this hexaploid clade, <em>Spartina maritima</em>, native to European Atlantic coasts and <em>Spartina alterniflora</em> Loisel., native to American east coasts, have naturally hybridized twice in the early 19<sup>th</sup> century following introductions of the American species in Europe. Hybridization between these species led to the formation of two sterile F1 hybrids, <em>Spartina x townsendii</em> in the Bay of Southampton (England) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Groves and Groves 1880</a>) and <em>Spartina x neyrautii</em> in the Basque region (France) (Foucaud 1897). Genome duplication of <em>S. x townsendii</em> gave rise to the highly fertile and invasive allopolyploid <em>Spartina anglica</em> C. E. Hubbard (2n = 12x = 120,122,124) (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Marchant 1968</a>). <em>S. anglica</em> colonized Western Europe as well as various regions (<em>e.g.</em>, Australia and China). This expansion has many ecological effects, with this species playing an important role in the salt marsh sediment dynamics. <em>S. anglica</em> is able to colonize mudflats and to accelerate sediment accretion, thus altering the characteristics of the colonized habitats (referred to as an “ecosystem engineer” species) (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Ainouche <em>et al.</em> 2008</a> and references therein). Genome evolution analysis of <em>S. anglica</em> compared to its parents (<em>S. maritima</em> and <em>S. alterniflora</em>) offers a special opportunity to understand the genomic and adaptive mechanisms associated with the formation of a new species in the wild (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ainouche <em>et al.</em> 2012</a>). With this perspective, knowledge of the parental genomes is essential.</p><p id="p-6">Our study focused on the European parental species <em>S. maritima</em> (2n = 60), which has a genome size that is estimated to be 2C = 3.8 pg or ∼3700 MB (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fortune <em>et al.</em> 2008</a>). This hexaploid species, probably of hybrid origin, is expected to contain three duplicated homeologous genomes (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fortune <em>et al.</em> 2007</a>) that may have diverged in the past 10 mya (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Rousseau-Gueutin <em>et al.</em> 2015</a>). Sequence heterogeneity is then expected as a result of successive whole-genome duplications. This may be more complicated in gene families where both paralogs and homeologs may be encountered, such as the 45S ribosomal gene family, which is known for its highly dynamic evolution, most particularly in polyploids (see below). Because these genes are commonly used in phylogenetic studies, distinguishing paralogs and homeologs are critical.</p><p id="p-7">The 45S rDNA unit is composed of a high number of transcription units (TU) per genome (&gt;500 in plants) arranged in tandem repeats on one or several loci (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rogers and Bendich 1987</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Prokopowich <em>et al.</em> 2003</a>). Each unit contains three coding regions (18S, 5.8S, and 25S/26S) separated by two internal transcribed spacers (ITS-1 and ITS-2). The 18S and 25S coding regions are flanked by two external transcribed spacers or ETS (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Schaal and Learn 1988</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Poczai and Hyvönen 2010</a>). The 5′-ETS is subdivided into three regions: the ETS region I, which contains the TATA box corresponding to the transcription initial site (TIS); the ETS region II, which includes several and highly variable subrepeats; and the ETS region III, which is adjacent to the 18S coding region (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Volkov <em>et al.</em> 1996</a>). Each TU is separated from the other by an intergenic spacer (IGS) that contains the two ETS bordering the nontranscribed spacer region (NTS) composed of several repeats of 80 bp. The number of repeats and the length of each vary considerably among species (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Poczai and Hyvönen 2010</a>). The different repeats are known to undergo a process of homogenization by gene conversion leading to the observation of “concerted evolution” of these ribosomal genes (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nei and Rooney 2005</a>). This homogenization limits the number and divergence of paralogous copies in this gene family, which has promoted its use in molecular phylogenies. The differential rates of variation along the transcriptional unit (the coding regions are more conserved than the spacers) allow phylogenetic inferences at different levels of the taxonomic hierarchy (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Alvarez and Wendel 2003</a>). In allopolyploid species, it was shown that the process of concerted evolution also affects homeologs, resulting in a preferential retention of one of the parental repeat copies, as demonstrated in polyploid cottons (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wendel <em>et al.</em> 1995</a>) or tobacco (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kovarik <em>et al.</em> 2008</a>). This process can occur rapidly after allopolyploid speciation, as evidenced in natural populations of allotetraploid <em>Tragopogon</em> formed in the past 80 years (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Koh <em>et al.</em> 2010</a>). In <em>Spartina</em>, the ITS region (including the 5.8S gene flanked by the ITS-1 and ITS-2) has been used to elucidate phylogenetic relationships among the different species (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Baumel <em>et al.</em> 2002</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Fortune <em>et al.</em> 2007</a>), but studies on the evolutionary dynamics of the paralogous and homeologous copies are lacking in this system. The recently formed <em>Spartina</em> hybrid (<em>S. x townsendii</em>) and its allododecaploid derivative (<em>S. anglica</em>) exhibit parental additivity of the ITS regions inherited from their hexaploid parents (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Baumel <em>et al.</em> 2001</a>), and recent investigations in natural populations suggest that interlocus homogenization and/or homeolog loss may also occur in this region (D. Huska, I. J. Leitch, J. Ferreira de Carvalho, A. R. Leitch, A. Salmon, M. Ainouche, and A. Kovarik, unpublished data).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/29/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of the 45S ribosomal DNA in Z. mays, S. maritima, and O. sativa. The lines correspond to the intergenic spacers, the gray boxes correspond to the coding regions (18S, 5.8S, and 25S), and the white boxes correspond to the external and internal transcribed spacers (ETS, ITS-1, and ITS-2). Black boxes represent the regions presenting less than 90% identity between either Z. mays and S. maritima or O. sativa and S. maritima. The rice expansion segments (D1 to D12, represented in black) detected by Hancock and Dover (1988) and Kuzoff et al. (1998) are indicated above the rice 25S. The figure is drawn to scale." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2039144746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic representation of the 45S ribosomal DNA in &lt;em&gt;Z. mays&lt;/em&gt;, &lt;em&gt;S. maritima&lt;/em&gt;, and &lt;em&gt;O. sativa&lt;/em&gt;. The lines correspond to the intergenic spacers, the gray boxes correspond to the coding regions (18S, 5.8S, and 25S), and the white boxes correspond to the external and internal transcribed spacers (ETS, ITS-1, and ITS-2). Black boxes represent the regions presenting less than 90% identity between either &lt;em&gt;Z. mays&lt;/em&gt; and &lt;em&gt;S. maritima&lt;/em&gt; or &lt;em&gt;O. sativa&lt;/em&gt; and &lt;em&gt;S. maritima&lt;/em&gt;. The rice expansion segments (D1 to D12, represented in black) detected by Hancock and Dover (1988) and Kuzoff &lt;em&gt;et al.&lt;/em&gt; (1998) are indicated above the rice 25S. The figure is drawn to scale.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/29/F1.medium.gif" width="440" height="275"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/1/29/F1.medium.gif" width="440" height="275"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/29/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/29/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467323" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Schematic representation of the 45S ribosomal DNA in <em>Z. mays</em>, <em>S. maritima</em>, and <em>O. sativa</em>. The lines correspond to the intergenic spacers, the gray boxes correspond to the coding regions (18S, 5.8S, and 25S), and the white boxes correspond to the external and internal transcribed spacers (ETS, ITS-1, and ITS-2). Black boxes represent the regions presenting less than 90% identity between either <em>Z. mays</em> and <em>S. maritima</em> or <em>O. sativa</em> and <em>S. maritima</em>. The rice expansion segments (D1 to D12, represented in black) detected by Hancock and Dover (1988) and <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kuzoff <em>et al.</em> (1998)</a> are indicated above the rice 25S. The figure is drawn to scale.</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">Next-generation sequencing technologies (<em>e.g.</em>, pyrosequencing or sequencing by synthesis) offer powerful tools to generate suitable sequence read depth especially in nonmodel species where genomic resources were previously lacking (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">El-Metwally <em>et al.</em> 2014</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Metzker 2009</a>). Detection of different copies expected in these genomes may be challenging, because individual gene duplications (resulting in paralogs in both diploid and polyploid genomes) usually add an additional layer of complexity (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Ainouche <em>et al.</em> 2012</a>). Detection and analysis of homeologs is central to polyploidy research, and several studies have focused on the detection of different copies in polyploid genomes such as in <em>Glycine</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ilut <em>et al.</em> 2012</a>), <em>Gossypium</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Flagel <em>et al.</em> 2008</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Salmon <em>et al.</em> 2010</a>), <em>Coffea</em> (Combes <em>et al.</em> 2011), or <em>Triticum</em> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akhunova <em>et al.</em> 2010</a>) species. In these systems, the diploid parental (or related) representatives are known and can be used to identify duplicated homeologs in the allotetraploids. The strategy developed in these studies is to assemble NGS datasets using parameters adapted to optimize recovery of paralogous and homeologous copies. Contigs obtained are then compared with diploid parental genomes using species-specific polymorphic sites.</p><p id="p-10">The detection of different copies in polyploid species where the parents are not identified or extinct still requires the development of adapted tools. The goal of this study is to develop a bioinformatic pipeline to detect the different copies within a set of NGS reads from a highly polyploid species without any reference parental diploid genome. We developed this pipeline to explore the heterogeneity of the coding and noncoding regions of the highly repeated 45S unit of the hexaploid <em>Spartina maritima</em>.</p><p id="p-11">The strategy was to: (i) identify single nucleotide polymorphisms (SNPs) and indels among reads from the hexaploid genome and (ii) assemble reads based on shared polymorphisms (after removing putative sequencing errors) to distinguish the different copies from reads. Polymorphisms were validated using comparisons with Illumina sequence data sets and by cloning and Sanger (re)-sequencing. The method presented here can be used to identify duplicated copies from any sequenced regions of polyploid or diploid genomes.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant material sequencing</h3><p id="p-12">Samples from <em>Spartina maritima</em> were collected at the Etel river estuary (Morbihan, France). Plants were transplanted and maintained in controlled conditions in the greenhouse (University of Rennes 1, France). Total genomic DNA was isolated from fresh leaf tissue with the Nucleospin Plant II (Macherey-Nagel) extraction kit following the manufacturer’s instructions. One run of <em>S. maritima</em> DNA was sequenced at the Functional and Environmental Genomics platform (Biogenouest, OSUR Rennes, France) using a 454 GS FLX Titanium pyrosequencer (Life Sciences, Roche) that generated 999,229 reads with an average length of 377.0 bp. Illumina genomic reads (172,528,550 reads of 100 bp length; 500-bp paired ends) from <em>S. maritima</em> (from the same population) were also used for mapping and validation or correction of the detected SNPs. Genome coverage and 45S rDNA copy number estimates are presented in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023242/-/DC1/TableS1.docx">Table S1</a> for both Roche-454 and Illumina Whole Genome Shotgun sequencing.</p></div><div id="sec-3" class="subsection"><h3>Assembly and annotation of the 45S rDNA region</h3><p id="p-13">We first analyzed Roche-454 genomic reads of <em>S. maritima</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>) using the computational pipeline developed by <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Novák <em>et al.</em> (2010)</a>. This pipeline uses graph-based clustering of sequence reads sharing mutual sequence similarities (minimum percent identity: 90%; minimum overlap: 55% of the shorter sequence length) to detect groups of frequently overlapping reads representing genomic repeats. In addition, it performs contig assembly within identified clusters and provides information aiding in repeat annotation. The cluster representing <em>S. maritima</em> 45S rDNA was identified by similarity searches against various available 45S rDNAs of Angiosperms, and an 8456 bp-long contig was retrieved from the assembled reads. Roche-454 genomic reads were then mapped on this contig to increase the number of reads. This first mapping procedure (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>) was performed using the GSMapper tools from Newbler (ml = 100 bp; mi = 95%). Annotation of the different 45S domains was performed by aligning the <em>S. maritima</em> contig with <em>Oryza sativa</em> and <em>Zea mays</em> 45S genes available on the NCBI database (GenBank IDs: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=X00755.1&amp;atom=%2Fggg%2F6%2F1%2F29.atom">X00755.1</a>, AF169230, M11585.1 and X03989, NR_036655.1, AF019817, NR_028022.2, respectively) using BLASTn with default parameters for the alignment algorithm Megablast (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Altschul <em>et al.</em> 1997</a>). The <em>Oryza sativa</em> genome was downloaded from <a href="http://www.phytozome.net">www.phytozome.net</a> (v. 204) to locate the 5′-ETS region. To identify this region, we followed the approach of <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Barker <em>et al.</em> (1988)</a>, who identified the repeat unit of the rDNA intergenic region and the transcription initial site.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/29/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overall workflow for haplotype construction using Roche-454 data." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2039144746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Overall workflow for haplotype construction using Roche-454 data.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/29/F2.medium.gif" width="440" height="364"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/1/29/F2.medium.gif" width="440" height="364"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/29/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/29/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467338" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-14" class="first-child">Overall workflow for haplotype construction using Roche-454 data.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467379/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467379/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Identification of the different rDNA regions in <em>S. maritima</em> after comparisons with <em>Oryza sativa</em> and <em>Zea mays</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>SNPs detection and sequencing error removal</h3><p id="p-17">To detect SNPs, we developed a pipeline allowing the parsing of “.ACE” alignment files (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>) using the Ace.py program from biopython (<a href="http://biopython.org/">http://biopython.org/</a>) and custom python script for editing homopolymer-driven false-positive SNPs. This step produced three files containing original reads, the original contig, and the list of the original reads (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>). This pipeline uses a mapping step with GSMapper (v.2.6, Roche; default parameters: ml = 40 bp; mi = 90%) to enhance alignments and ease SNP detection. The newly generated “.ACE” file is parsed to extract the aligned reads (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2E</a>). To remove false-positive SNPs caused by sequencing errors (within reads, and at the first or last position), the first five and the last five nucleotides of each read, as well as nucleotides with frequency lower than 20% were not considered. SNPs present at least in one-fifth of aligned reads were then considered true positives. SNPs present in homopolymeric region (more than four identical nucleotides) were not considered. The nature of the detected polymorphisms and their frequency are stored as output results (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2F</a>). Visual checking of the detected and not considered false positives was done using the Tablet software (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Milne <em>et al.</em> 2009</a>) for “.ACE” alignment files and with the Jalview software (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Waterhouse <em>et al.</em> 2009</a>) for “.FASTA” files.</p></div><div id="sec-5" class="subsection"><h3>Haplotype detection</h3><p id="p-18">Haplotypes were detected after the SNP detection by identifying and assembling reads sharing the same SNPs. This assembly was performed by comparing each read to all the others and by screening all the SNPs present in overlapping regions. If all the SNPs are identical in this region, the considered reads are assembled to create a haplotype with a maximum size. A unique haplotype is then a sequence characterized by multiple polymorphic sites corresponding to the same and unique allele (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). However, for each alignment, it was necessary to define different windows corresponding to local alignments for counting haplotypes. The windows exhibiting at least one SNP were selected to count the number of locally aligned haplotypes. Two output files allow the dataset visualization: (1) a “.FASTA” file that contains the consensus sequence with the different aligned haplotypes and (2) a tabulated file describing SNPs positions and their nature. A third output file is created during these processes and contains information about the assembly, the name of the contig, the initial and final lengths of the contig obtained after the correction of sequencing errors, the number of reads used to assemble the contig, and the different haplotypes. This file contains the number of SNPs and their position, the number of haplotypes, and their coverage (mean, SD, and C.I. of 95%). The last information contained in this file is the number of haplotypes for each window.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/29/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representation of a read assembly. After SNP detection and false-positive correction, all reads displaying the same SNP (represented with the same line) are assembled to create a haplotype that corresponds to a consensus sequence of maximal size." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2039144746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Representation of a read assembly. After SNP detection and false-positive correction, all reads displaying the same SNP (represented with the same line) are assembled to create a haplotype that corresponds to a consensus sequence of maximal size.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/29/F3.medium.gif" width="440" height="359"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/1/29/F3.medium.gif" width="440" height="359"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/29/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/29/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467386" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-19" class="first-child">Representation of a read assembly. After SNP detection and false-positive correction, all reads displaying the same SNP (represented with the same line) are assembled to create a haplotype that corresponds to a consensus sequence of maximal size.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">The SNP and haplotype detection programs described above were incorporated into a single pipeline that includes four adjustable parameters: (i) the read trimming length corresponding to the number of nucleotides at the beginning and the end of each read not to consider in the SNP detection step; (ii) the minimum read depth corresponding to the number of reads required to detect SNP from; (iii) the SNP detection threshold corresponding to the minimal proportion of reads displaying a putative polymorphism; and (iv) the minimal number of shared SNPs to assemble two reads into a haplotype. The minimum depth of the analyzed rDNA contig was 61. Consequently, the minimum read depth parameter was fixed to 60. The other default parameters used in our pipeline were as follows: read trimming length = 5; SNP detection threshold = 20; number of shared SNPs = 1.</p></div><div id="sec-6" class="subsection"><h3>Cloning and sequencing of polymorphic 45S rDNA regions</h3><p id="p-21">For validating the SNP and haplotype detection pipeline we developed, a subset of the detected SNPs and haplotypes was targeted for molecular cloning and (re)-sequencing (Sanger method). We have designed four pairs of primers using Primer 3, v. 1.1.4 (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Rozen and Skaletsky 2000</a>) (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) flanking variable regions detected with pyrosequencing (5′- IGS/ETS, ITS-1, 25S and 3′- ETS/IGS). All amplifications were carried out in a 50 μL reaction mixture containing 45 µl of Platinum PCR SuperMix High Fidelity (Invitrogen), 2 μl of each 5 μM primer, and 1 μl of DNA (50 ng). The PCR amplification conditions were as follows: 2 min of DNA denaturation at 94° followed by 30 cycles of 30 sec at 94°, 20 sec at 58°, and 30 sec at 68° for each cycle, followed by 8 min final extension at 68°. PCR products were purified with the NucleoSpin Gel and PCR Clean-up (Macherey-Nagel). Cloning reaction was performed with TOPO TA Cloning Kits for sequencing (Invitrogen) using 2 μl of PCR purified products, 1 µl of salt solution (1.2 M of NaCl, 0.06 M of MgCl<sub>2</sub>), 2 µl of water, and 1 µl of plasmid vector pCR4Blunt-TOPO (Invitrogen). The reactions were incubated for 5 min at room temperature. Transformations were realized using electro-competent <em>Escherichia coli</em> DH5α and Gene Pulser Xcell Electroporation System (BioRad); 18 µl of water was added to 6 µl of TOPO cloning reaction. Transformations were done in 0.2 cm cuvettes with 2 µl of DNA added to 40 µl of ElectroMax DH5α Competent Cells (Invitrogen) incubated 1 min on ice and pulsed at 2.5 kV, 25 µF, and 200 Ω. After the addition of SOC medium (1 ml), the cell suspension were incubated at 37° for 1 hr; 25–100 µL of each transformation culture was plated onto LB Agar with ampicillin (100 mg/ml) and incubated overnight in liquid medium at 37° (LB Broth and 100 mg/ml of ampicillin). Plasmids were purified with the Pure Yield Plasmid Miniprep System (Promega) and sequenced using T7 and T3 primers to sequence the samples (16–24 clones for each region) with 3730XL DNA sequencer technology (Sanger method) from both ends. For each region, the different sequences were aligned with MAFFT (v 6.864b) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Katoh and Toh 2010</a>). All sequences were cleaned (plasmid deletion) and the sequencing errors were visually corrected using chromatographs.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467361/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467361/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Primers designed to amplify and sequence the 45S ribosomal DNA in <em>S</em>. <em>maritima</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>SNP validation and haplotype estimation with Illumina data</h3><p id="p-22">To compare pyrosequencing and cloning SNPs to Illumina SNPs, we mapped 1,263,153 Illumina paired reads with an average read depth of 7886 (SE = 273.84) on the rDNA sequence consensus of <em>S. maritima</em> using Bowtie2 (score-min: G, 52, 8) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Langmead and Salzberg 2012</a>). These “score-min” parameters were included in the minimum score function <em>f(x)= 52 + 8 * ln(x)</em>, where <em>x</em> corresponds to the read length. This function corresponds to a minimum of 87.06–90.30% of identity between the mapped reads and the reference sequence for reads with a length of 80–120 bp. The “.SAM” file created by Bowtie2 was converted in a “.PILEUP” format using the Samtools software suite (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2009</a>). We have detected the different SNPs within the Illumina data using custom python scripts (minimum read depth = 30; SNP detection threshold = 2, corresponding to nucleotides that are present more than two out of 100 times per position). To estimate the relative presence of each haplotype, Illumina reads were mapped on previously detected haplotypes (∼100 bp for each region including 5′-ETS/IGS, 18S, ITS-1 and 25S) using Bowtie2 (score-min: L, 100, 0; <em>f(x)= 100 + 0 * x</em> where x = length of read). Results obtained were filtered using custom python scripts to identify the number of reads mapped with 100% of identity.</p></div><div id="sec-8" class="subsection"><h3>Chromosome preparation and fluorescence <em>in situ</em> hybridization</h3><p id="p-23"><em>In situ</em> hybridization was performed on mitotic chromosomes from <em>S. maritima</em> roots. Root tips with a length of 0.5–1.5 cm were treated in the dark with 0.04% 8-hydroxiquinoline for 2 hr at 4°, followed by 2 hr at room temperature to accumulate metaphases, then fixed in ethanol–acetic acid (3:1, <em>v/v</em>) for 48 hr at 4°, and stored in ethanol 70% at −20° until required. After washing in 0.01 M enzyme buffer (citric acid–sodium citrate, pH 4.5) for 15 min, the prepared roots were digested in a solution of 5% Onozuka R-10 cellulase (Sigma) and 1% Y23 pectolyase (Sigma) at 37° for 30 min. The root tips were then washed with distilled water for 30 min. A root tip was transferred to a slide and macerated with a drop of 3:1 fixation solution. After air-drying, slides with good metaphase chromosome spreads were stored at −20°. Fluorescence <em>in situ</em> hybridization was carried out using the ribosomal probe pTa 71 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Gerlach and Bedbrook 1979</a>), which contained a 9-kb <em>Eco</em>RI fragment of rDNA repeat unit (18S-5.8S-25S genes and spacers) isolated from <em>Triticum aestivum</em>. The pTa 71 probe was labeled by random priming with biotin-14-dUTP (Invitrogen, Life Technologies). Chromosome preparations were incubated in RNase A (100 ng/µl) and pepsin (100 mg/ml) in 0.01 M HCl, and fixed with paraformaldehyde (4%). Chromosomes were denatured in a solution of 70% formamide in 2X SSC at 70° for 2 min, dehydrated in an ethanol series (70%, 90%, and 100%), and air-dried. The hybridization mixture, consisting of 50% deionized formamide, 10% dextran sulfate, 2X SSC, 1% SDS, and labeled probe (200 ng per slide), was denatured at 92° for 6 min and transferred to ice. The denatured probe was placed on the slide and <em>in situ</em> hybridization was carried out overnight in a moist chamber at 37°. After hybridization, slides were washed for 5 min in 50% formamide in 2X SSC at 42°, followed by several washes in 4X SSC-Tween. The biotinylated probe was immunodetected by Texas Red avidin DCS (Vector Laboratories). The chromosomes were mounted and conterstained in Vectashield (Vector Laboratories) containing 2.5 µg/ml 4’,6-diamidino-2-phenylindole (DAPI). Fluorescence images were captured using a CoolSnap HQ camera (Photometrics, Tucson, AZ) on an Axioplan 2 microscope (Zeiss, Oberkochen, Germany) and analyzed using MetaVue (Universal Imaging Corporation, Downingtown, PA).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-24">The S. <em>maritima</em> rDNA consensus sequence and haplotype sequences are available at NCBI under the KT874468 to KT874488 accession numbers.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Analysis and annotation of the 45S rDNA region</h3><p id="p-25">The contig obtained with the clustering approach was 8456 nucleotides long; it was assembled from 3219 reads with an average length of 377.0 bp. Genomic DNA reads of <em>S. maritima</em> were then mapped to the contig with GSMapper (with the following parameters: ml = 100 bp; mi = 95%) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Margulies <em>et al.</em> 2005</a>), resulting in a total of 4014 aligned reads with an average length of 506.8 nucleotides. This step allowed us to build a 45S rDNA reference sequence totaling 8464 bp. The different regions of the 45S rDNA of <em>S. maritima</em> were annotated by homology searches using rDNA sequences from Poaceae species (<em>Zea mays</em>, <em>Oryza sativa</em>) available in GenBank (NCBI) and by motif searches for the 5′-ETS region (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The beginning of the 5′-ETS region was identified using the TATA-box (5′-TATATTAGGGGG-3′) motif at position 395, corresponding to the expected plant TATA-box motif (5′-TATA(G)TA(N)GGGGG-3′), as found in several species such as <em>Zea mays</em>, <em>Oryza sativa</em>, or <em>Arabidopsis thaliana</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fan <em>et al.</em> 1995</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Zentgraf <em>et al.</em> 1998</a>). To confirm this position, a dot plot of the 5′-IGS/ETS region against itself was performed. Regions 2 and 3 of the 5′-ETS (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Volkov <em>et al.</em> 1996</a>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">1999</a>) were also detected. We found that regions 2 and 3 of the 5′-ETS totaled 1600 bp. This confirms the total length of the 5′-ETS of <em>S. maritima</em> (1754 bp), which is approximately two-times larger than the 5′-ETS of <em>Z. mays</em> (825 bp; X03989) and <em>O. sativa</em> (966 bp). The <em>S. maritima</em> rDNA contig covers the whole coding region, spanning the 18S, 5.8S, and 25S, the ITS-1, ITS-2, 5′-ETS (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), and part of the IGS (5′-IGS: 391 bp; 52.9 GC% and 3′ETS/3′-IGS: 517 bp and 42.6 GC%) (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Interspecific comparisons of the three coding rDNA regions between maize and <em>Spartina</em> and between rice and <em>Spartina</em> indicate that nucleotide variation is distributed all along these regions (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>).</p></div><div id="sec-12" class="subsection"><h3>SNP validation in coding and noncoding regions</h3><p id="p-26">Using the mapped Roche-454 data, 34 SNPs or indels (insertion/deletion) were identified. These polymorphisms were localized both in coding (18S, 25S) and noncoding regions (5′-IGS, 5′-ETS, ITS-1 and 3′-ETS/IGS). Cloning and resequencing were performed and confirmed 10 of the 34 polymorphic sites (two substitutions, four variants including substitution and indel at the same position, and four indels) that had been detected using the mapping procedure. Twenty-nine SNPs and indels (including nine polymorphic sites found by cloning and (re)-sequencing) were subsequently validated using Illumina data (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Within the coding region, one indel and four SNPs were detected and validated using the Roche-454 and lllumina data in the 18S and 25S domains, respectively. In the noncoding regions, a substitution, the variant presenting substitution and indel at the same position, and two indels were validated using Illumina method in the 5′-IGS. In the 5′-IGS/ETS noncoding region, a total of 21 polymorphisms were detected using the mapped Roche-454 data, including nine substitutions and 12 indels (two of 4 bp) (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). In this region, all the identified substitutions (except one G/T, position 1235) were validated using Illumina data. Within the ITS-1 region, the Roche-454 and Illumina data detected the same SNP (A/T/-) that was validated by (re)-sequencing. However, the indel was only found in the pyrosequencing data. In the 3′-ETS/IGS noncoding region, one indel using the Roche-454 data was not validated by Illumina data. Thus, the cloning-sequencing data as well as the Illumina reads substantiated 29 of the 34 polymorphisms previously identified in the Roche-454 reads. Four of the five invalidated variants were included in homopolymeric regions. The last indel was not found in the Illumina data, but cloning and sequencing data revealed variation in the chromatograph, suggesting either a sequencing error or a substitution.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467511/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467511/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Nucleotide polymorphisms detected within coding and noncoding regions of <em>Spartina maritima</em> rDNA using the three methods (mapping method, cloning-sequencing Sanger method, and Illumina method)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Haplotype detection</h3><p id="p-28">After rDNA domain annotations and SNP detection and validation, the number of haplotypes for each region of the 45S unit was determined. In this study we retained only the Roche-454 haplotypes constructed with a minimum of 10× read depth. The five indels not validated by cloning and Illumina data were not considered. From the 29 polymorphic sites retained, 20 haplotypes with a mean length of 1197.70 bp (σ = 938.32; C.I.<sub>95%</sub> = 786.47–1608.93) were constructed using our program. These haplotypes were constructed from 69.35 reads on average (σ = 66.24; C.I.<sub>95%</sub> = 40.32–98.38). The 20 different haplotypes were localized on the different rDNA domains as shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. We did not identify any polymorphism in the 5.8S coding region, the internal transcribed spacer 2 (ITS-2), and the 3′ intergenic spacer (IGS). The 5′-IGS/ETS region was the most variable; we detected four variants in the 5′-IGS and 21 variants in the 5′-ETS. These SNPs allowed the construction of 13 haplotypes in this region (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, Block I). One indel was detected in the 18S coding region indicating the presence of two haplotypes (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, Block II). Within the ITS-1 we detected only one SNP indicating the presence of three haplotypes (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>, Block III). Within the 25S, four haplotypes were constructed from two SNPs (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>, Block IV). Finally, 20 haplotypes were obtained in total.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/29/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic representation of haplotypes detected in S. maritima rDNA using our developed program (mapping data) and cloning data. The blue boxes represent the haplotypes detected only by the developed program. The red boxes represent the haplotypes detected by the developed program and validated by cloning and (re)-sequencing or Illumina data (i.e., haplotypes in the block II). The different blocks (I to IV) correspond to regions of rDNA presenting at least two haplotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2039144746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic representation of haplotypes detected in &lt;em&gt;S. maritima&lt;/em&gt; rDNA using our developed program (mapping data) and cloning data. The blue boxes represent the haplotypes detected only by the developed program. The red boxes represent the haplotypes detected by the developed program and validated by cloning and (re)-sequencing or Illumina data (&lt;em&gt;i.e.&lt;/em&gt;, haplotypes in the block II). The different blocks (I to IV) correspond to regions of rDNA presenting at least two haplotypes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/29/F4.medium.gif" width="440" height="265"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/1/29/F4.medium.gif" width="440" height="265"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/29/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/29/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467326" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Schematic representation of haplotypes detected in <em>S. maritima</em> rDNA using our developed program (mapping data) and cloning data. The blue boxes represent the haplotypes detected only by the developed program. The red boxes represent the haplotypes detected by the developed program and validated by cloning and (re)-sequencing or Illumina data (<em>i.e.</em>, haplotypes in the block II). The different blocks (I to IV) correspond to regions of rDNA presenting at least two haplotypes.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Haplotype validation</h3><p id="p-30">To validate the developed program, the constructed Roche-454 haplotypes were compared with haplotypes obtained using the cloned sequences (16X to 24X both ends). Four regions were selected for these comparisons (5′-IGS/ETS, 5′-ETS, ITS-1/5.8S/ITS-2, and 25S). In the 5′-IGS/ETS noncoding region, seven polymorphisms were validated by cloning, including the indel block of 2 bp, one substitution, two deletions, and two substitutions/insertions/deletions. Using these polymorphisms, it was possible to validate three of nine Roche-454 constructed haplotypes. Cloning depth could explain the low number of haplotypes validated. In the 5′-ETS noncoding region, only one polymorphism was validated by the cloning approach, which confirms one haplotype (two haplotypes were detected with this variant using Roche-454 data) (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>, Block I). In the ITS-1/5.8S/ITS-2, cloning-sequencing data validated the variant detected with the program, and two of three detected haplotypes were validated (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>, Block III). In the 18S coding region, the two haplotypes detected with the program were validated using Illumina data (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a>, Block II). In the 25S coding region, the two SNPs detected using our program (developed on Roche-454 data) were found with cloning/(re)-sequencing data. Four haplotypes were found with the program and three were validated (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4</a>, Block IV).</p></div><div id="sec-15" class="subsection"><h3>Haplotype proportion estimation</h3><p id="p-31">Using the number of Illumina reads mapped on the different haplotypes, it was possible to estimate the relative proportion of each haplotype. For each rDNA region, the number of reads per haplotype was estimated (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). In the 5′-IGS/ETS, 12 haplotypes were compared, including three haplotypes presenting a high proportion (more than 17.8% of the rDNA copies). Three haplotypes presented a very small number of estimated copies (between 0.01% and 2.50%). The estimated proportions for the six other haplotypes detected in this region were 6.92% on average. In the 18S coding region, two haplotypes were present in very different frequencies. The first one represented only 1.28% of the rDNA copies and the second one was overrepresented (98.72% of rDNA copies). In the ITS-1, two haplotypes were highly frequent (46.45% and 51.49% respectively). The third haplotype detected in the ITS-1 was represented in a low copy number (2.06%). In the 25S coding region, two haplotypes were approximately equally frequent (39.84% and 40.66%), and two others were less represented (4.86% and 14.65%). Finally, in the different regions studied, we found between one and three copies highly represented, which could indicate that these copies present a high number of repetitions in the <em>S. maritima</em> genome.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/29/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of reads mapped with 100% of identity on the different haplotypes for each region (5&#x2032;-IGS/ETS, 18S, ITS-1, and 25S). The relative proportion of haplotype is presented above each histogram. White bars represent haplotypes validated by either Sanger sequences or Illumina data (18S coding region only). Black lines represent the length of the haplotypes constructed by our program. Some haplotypes are spanning two or more subregions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2039144746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Number of reads mapped with 100% of identity on the different haplotypes for each region (5&#x2032;-IGS/ETS, 18S, ITS-1, and 25S). The relative proportion of haplotype is presented above each histogram. White bars represent haplotypes validated by either Sanger sequences or Illumina data (18S coding region only). Black lines represent the length of the haplotypes constructed by our program. Some haplotypes are spanning two or more subregions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/29/F5.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/1/29/F5.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/29/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/29/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467341" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Number of reads mapped with 100% of identity on the different haplotypes for each region (5′-IGS/ETS, 18S, ITS-1, and 25S). The relative proportion of haplotype is presented above each histogram. White bars represent haplotypes validated by either Sanger sequences or Illumina data (18S coding region only). Black lines represent the length of the haplotypes constructed by our program. Some haplotypes are spanning two or more subregions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>rDNA location on <em>S. maritima</em> chromosomes</h3><p id="p-33">Fluorescence <em>in situ</em> hybridization (FISH) was performed from <em>S. maritima</em> somatic metaphase chromosomes (counterstained with DAPI) to identify the physical position of 45S rDNA arrays. The chromosomal locations of 45S rDNA arrays in <em>S. maritima</em> are shown in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>. Hybridization signals were consistently observed on two chromosomes indicating the presence of one pair of 45S rDNA loci.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/29/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fluorescence in situ hybridization (FISH) of the 45S rDNA in S. maritima on somatic metaphase chromosomes counterstained with DAPI (blue), in the absence (left) or presence of the red rDNA probe (right). Bar, 5 &#x3BC;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2039144746" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fluorescence &lt;em&gt;in situ&lt;/em&gt; hybridization (FISH) of the 45S rDNA in &lt;em&gt;S. maritima&lt;/em&gt; on somatic metaphase chromosomes counterstained with DAPI (blue), in the absence (left) or presence of the red rDNA probe (right). Bar, 5 &#x3BC;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/29/F6.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/1/29/F6.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/29/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/29/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467423" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-34" class="first-child">Fluorescence <em>in situ</em> hybridization (FISH) of the 45S rDNA in <em>S. maritima</em> on somatic metaphase chromosomes counterstained with DAPI (blue), in the absence (left) or presence of the red rDNA probe (right). Bar, 5 μm.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-35">In this study, we developed a program for SNP and haplotype detection. This program was applied on Roche-454 pyrosequencing data corresponding to the 45S rDNA genes of <em>S. maritima</em>. The objective was to detect the different rDNA sequences (designated as haplotypes) in this hexaploid species where both paralogous and homeologous copies are expected for this gene family. Results were validated by cloning and (re)-sequencing and also from Illumina sequencing datasets; the results allowed the estimation of the relative proportion of haplotype variation. To date, very few programs or pipelines have been developed to distinguish duplicated copies from next-generation sequencing data without diploid parents as a reference. SNiPloid (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Peralta <em>et al.</em> 2013</a>), BamBam (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Page <em>et al.</em> 2014</a>), or HyLiTE (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Duchemin <em>et al.</em> 2014</a>) were developed for haplotype detection in polyploid species using comparisons between the polyploids and their diploid progenitors. The POLiMAPS program developed by <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Tennessen <em>et al.</em> (2014)</a> used a related sequenced diploid genome combined with dense genetic map to infer homeologs in octoploid <em>Fragaria</em>. Our program will be particularly useful in nonmodel systems having experienced repeated duplication events and for which diploid reference genomes are not available.</p><div id="sec-18" class="subsection"><h3>Number of rDNA loci in <em>S. maritima</em></h3><p id="p-36">Molecular cytogenetic analysis by FISH revealed the presence of one pair of 45S rDNA signals on the chromosomes of <em>S. maritima</em> instead of the three expected pairs for a hexaploid genome. The number of rDNA loci varies greatly among flowering plants of the same ploidy level. For instance, only one locus was identified in diploid <em>Medicago</em> species (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rosato <em>et al.</em> 2008</a>), whereas five loci were detected in diploid <em>Brassica rapa</em> species (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Maluszynska and Heslop-Harrison 1993</a>). In polyploids, the number of rDNA loci can be either retained (in recent or old polyploids) or deeply decreased, most probably as part of the diploidization process (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Leitch and Bennett 2004</a>). For example, three rDNA loci were identified in the hexaploid bread wheat (AABBDD), corresponding to the sum of the loci present in the parental genomes [two in the allotetraploid <em>Triticum turgidum</em> (AABB) and one in the diploid <em>Aegilops tauschii</em> (DD)] (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Badaeva <em>et al.</em> 1996</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">El-Twab 2007</a>). Similarly, the number of loci detected in the recent <em>Tragopogon</em> allotetraploids is additive with respect to their diploid parental species (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kovarik 2005</a>). In contrast, in other polyploid genera, such as <em>Fragaria</em>, a decrease in the number of rDNA loci during the diploidization process is observed, with only six loci in the decaploid <em>F. iturupensis</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Liu and Davis 2011</a>), whereas related diploid species exhibit three loci. In the hexaploid <em>S. maritima</em>, the presence of one locus indicates rDNA loci have been lost since its formation, following evolution of the tetraploid and hexaploid <em>Spartina</em> lineages that occurred in the past 6–10 mya (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Rousseau-Gueutin <em>et al.</em> 2015</a>). To our knowledge, this is the first 45S locus number reported in <em>Spartina</em>, and it would be interesting to examine this number in other polyploid <em>Spartina</em> species. Such rDNA locus loss can occur rapidly, as in the recent allotetraploid <em>Brassica napus</em>, which formed approximately 8000 years ago (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chalhoub <em>et al.</em> 2014</a>), where six loci are retained instead of the seven expected from the progenitor species (five in <em>B. rapa</em> and two in <em>B. oleracea</em>) (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Snowdon <em>et al.</em> 1997</a>).</p></div><div id="sec-19" class="subsection"><h3><em>In silico</em> detection of rDNA domains</h3><p id="p-37">The intraindividual homogeneity of ribosomal genes was so far mainly examined across plant species by cloning and sequencing, a relatively time-consuming procedure that limits the number of sequences that can be sampled in the genome. Massively parallel sequencing now provides the opportunity to assess more accurately the intraindividual sequence polymorphism (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Matyášek <em>et al.</em> 2012</a>). Furthermore, in rDNA genes, most studies have focused on the ITS region (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Alvarez and Wendel 2003</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Poczai and Hyvönen 2010</a>) or the 18S and 25S coding regions. For the first time, all the repeat unit and spacers (IGS and ETS) in <em>Spartina</em> are reported here. Analysis of the contig corresponding to the 45S ribosomal DNA unit of <em>S. maritima</em> enabled us to annotate the different regions of the transcriptional unit and to detect the different copies for each region (IGS/ETS; 18S; ITS-1; 5.8S; ITS-2; and 25S) using an approach relying on a low-stringency mapping of NGS reads.</p><p id="p-38">Detection of the different domains was performed by sequence alignments of 45S rDNA available in the databases of two Poaceae species (maize and rice). Lengths of the rDNA coding regions and ITS of <em>S. maritima</em> are similar to those found in two Poaceae. However, the percentage of GC of the two ITS and 5.8S coding regions is very low in comparison with other Poaceae (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). When examining the GC content of the ITS region in various grass species, we found that some Chloridoideae lineages related to <em>Spartina</em> in the Zoysieae, Cynodonteae, and Eragrostideae tribes similarly exhibit less than 60% GC content (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023242/-/DC1/TableS2.docx">Table S2</a>). In more distantly related Chloridoideae tribes such as Triraphideae and Centropodieae (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Soreng <em>et al.</em> 2015</a>), as well as other grass subfamilies, the GC content is greater (up to 79.1% in rice ITS-2) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023242/-/DC1/TableS2.docx">Table S2</a>). Other monocot lineages (<em>e.g.</em>, <em>Prospero autumnale</em>, Hyacinthaceae) also exhibit higher GC content. Although information about more monocot representatives is needed, these results examined in the light of recent grass phylogenies (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Soreng <em>et al.</em> 2015</a>) suggest that GC content decrease occurred in the Chloridoideae lineage following the split between the basal Centropodieae–Triraphideae tribes and the other more derived tribes including <em>Spartina</em>.</p><p id="p-39">Within 25S, variable regions were identified when comparing <em>S. maritima</em> with <em>Zea mays</em> and <em>Oryza sativa</em> (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). These regions mostly correspond to previously described expansion segments (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hancock and Dover 1988</a>) representing variable 26S subdomains in plants (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kuzoff <em>et al.</em> 1998</a>). The 5′-ETS detected in <em>Oryza sativa</em> (966 bp) has a similar length compared to the 5′-ETS of <em>Zea mays</em> (825 bp) identified by <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Toloczyki and Feix (1986)</a>. However, this spacer is much larger in <em>S. maritima</em> (1754 bp), with lower GC content (50.9%) than in other Poaceae (70.0% and 73.0% for maize and rice, respectively). These results are consistent with the literature showing that the size of 5′-ETS is variable across species, even within genera. <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Volkov <em>et al.</em> (1996)</a> found different lengths of the 5′-ETS between two tobacco species: <em>Nicotiana sylvestris</em> and <em>Nicotiana tomentosiformis</em>, which have, respectively, an ETS size of 1444 bp and 2172 bp. The size difference between these two ETS is explained by the number of subrepeats present in region 2 of the ETS, where five and 10 subrepeats are present in <em>N. sylvestris</em> and <em>N. tomentosiformis</em>, respectively (<a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">Volkov <em>et al.</em> 1996</a>). In the <em>Fabaceae</em> family, <em>Lupinus luteus</em> has a small 5′-ETS with a size of 487 bp (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mahé <em>et al.</em> 2010</a>). It would be interesting to identify and compare the 5′-ETS in other hexaploid and tetraploid <em>Spartina</em> species as well as in related Chloridoids to explore the amplitude and origin of this size variation. These studies would verify whether this heterogeneity is widespread in the polyploid <em>Spartina</em> genus, or whether it is a characteristic of the hexaploid <em>Spartina maritima</em>.</p></div><div id="sec-20" class="subsection"><h3>Polymorphism of coding and noncoding regions within the <em>S. maritima</em> rDNA locus</h3><p id="p-40">Thirty-nine polymorphic variants and 20 haplotypes were detected in <em>S. maritima</em> rDNA using our developed program. Thirty-four SNPs and 11 haplotypes were validated by cloning and Sanger sequencing or with Illumina data. The 5.8S coding region exhibited less heterogeneity than 18S and 25S coding regions. Indeed, no polymorphism was encountered in the 5.8S and only one polymorphic site (one deletion) was encountered in the 18S coding region, highlighting their high sequence homogeneity. The 25S unit also presents little variation, with two SNPs (variants presenting substitution and indel at the same position) detected using pyrosequencing data and validated by the other methods. Although most coding regions are highly conserved, some variation may be observed in the 18S region. Such results are in accordance with those of <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mentewab <em>et al.</em> (2011)</a>, who detected a deletion of 270 bp within <em>Arabidopsis thaliana</em> 18S, or those obtained in animals (<em>e.g.</em>, <em>Anguilla</em>, for which a specific SNP was found) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Frankowski and Bastrop 2010</a>). Similarly, the intraspecific variation of the 18S and 25S detected in <em>S. maritima</em> accords with the results obtained in two <em>Oenothera</em> species using cloning and sequencing methods (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Seo <em>et al.</em> 2013</a>), for which 17 and 10 SNPs were identified in the 18S of <em>O. odorata</em> and <em>O. laciniata</em> and 11 and 13 SNPs in the 25S regions, respectively.</p><p id="p-41">In the noncoding regions, the internal transcribed spacers of <em>S. maritima</em> exhibit less heterogeneity than the IGS/ETS regions, without any polymorphism in ITS-2 and only one substitution/deletion in ITS-1. The high polymorphism encountered in the IGS/ETS region (25 variants in the 5′-IGS/ETS) is consistent with the literature, which indicates that intergenic regions evolve more rapidly than the ITS region, which is most particularly prone to homogenization (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Alvarez and Wendel 2003</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kovarik <em>et al.</em> 2008</a>). The high IGS/ETS variation we detected at the intraindividual level is in accordance with other studies at the interindividual or intraspecific levels (<a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Poczai and Hyvönen 2010</a>). Indeed, ETS regions evolve very quickly, 1.5-times faster than ITS regions (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bena <em>et al.</em> 1998</a>); IGS regions also evolve rapidly (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chang <em>et al.</em> 2010</a>). The number of detected haplotypes in the IGS/ETS region (13) exceeds the number of expected homeologous copies in a hexaploid species, which suggests incomplete homogeneization within the rDNA locus as also evidenced in diploid <em>Nicotiana</em> species (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Matyášek <em>et al.</em> 2012</a>). Our finding of a single rDNA locus in <em>Spartina maritima</em> mentioned above supports this hypothesis.</p><p id="p-42">Like coding regions, the ITS region in <em>S. maritima</em> contains two haplotypes that are highly represented. The ITS region might be as strongly selected as the flanking coding regions, but the rapid evolution of the ITS region at the interspecific level suggests that this region is rather subject to more rapid homogenization (within individual genomes) than the other regions. Interspecies variation is then most likely more important than intraindividual (intragenomic variation), which explains why this region is useful in phylogenetic studies aiming at reconstructing organismal (<em>e.g.</em>, species) history (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Baumel <em>et al.</em> 2002</a> for <em>Spartina</em>). In the 5′-IGS/ETS, three haplotypes are present in high proportion and two haplotypes are less represented, which could indicate that the homogenization process is underway in these regions.</p></div><div id="sec-21" class="subsection"><h3>Impact of the parameters on polymorphism and haplotype detection</h3><p id="p-43">The number of SNPs and haplotypes that may be detected in our study is indeed limited by the employed technology (<em>i.e.</em>, 454 read depth) and the parameters used to avoid false positives resulting from sequencing errors. Thus, the number of actual rDNA variants and their relative proportion (or copy number) in the <em>S. maritima</em> genome is most likely underestimated. In our developed program, several parameters can be tuned for relaxing or constraining read mapping, SNP detection, and haplotype construction. For example, the choice to not consider nucleotides present in less with a frequency lower than 20% at a specific position is in line with the procedure usually used when analyzing polyploid genomes. In their analysis of allotetraploid cotton EST (Expressed Sequence Tag) assemblies, <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Udall <em>et al.</em> (2006)</a> have chosen to set this parameter to 25%. <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Tennessen <em>et al.</em> (2014)</a> have chosen to adjust this parameter to 12.5% (one-eighth frequency) to detect variants in octoploid <em>Fragaria</em> species. Our parameter is less stringent than the parameter used for homeo-SNP detection (fixed to 40%) in allotetraploid cotton (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Page <em>et al.</em> 2013a</a>,<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">b</a>). This lower value allowed us to detect putative homeologous and paralogous copies, as expected for highly repeated rDNA arrays. Using this parameter, 34 variants were detected and 29 were validated. Four of these variants are localized in homopolymeric regions of three or four repeats. The developed program does not consider the homopolymeric regions of at least five repeats. These results could indicate that variants in homopolymeric regions of three or four repeats correspond to false positives and should also not be considered. In the internal transcribed spacer and the coding region, it was possible to validate most of the haplotypes constructed by the developed program (seven haplotypes validated on nine). In the noncoding region, the number of haplotypes, their relative estimated proportion, and the cloning depth explain the number of haplotypes not validated by the cloning method. In fact, several haplotypes localized in the 5′-IGS/ETS built with the developed program were also detected and validated using cloning/(re)-sequencing in the <em>Spartina</em> hybrids between <em>S. maritima</em> and <em>S. alterniflora</em> (<em>S. x townsendii</em> and <em>S. x neyrautii</em>; unpublished data). New and emerging single molecule sequencing methods that are producing longer reads should provide precious information regarding the assembly of longer (or even whole-length) 45S rDNA haplotypes.</p><p id="p-44">In summary, our program enables detection of SNPs and haplotypes within NGS read datasets. Haplotype construction and validation are based on three methods/technologies [Roche-454, cloning, and (re)-sequencing Sanger method and Illumina approaches] in the context of polyploidy in a nonmodel species where diploid reference parental genomes are not available. The program developed in this study could then be used for any polyploid species (including high ploidy levels) with or without reference genome sequences but also on diploid species (<em>e.g.</em>, for allele detection from amplicon data sets). In the polyploid <em>Spartina</em> genus, which contains various ploidy levels (4×, 6×, 7×, 9×, and 12×), such a program will be useful to detect duplicated homeologous genes and to increase our understanding of their origin and evolutionary fate.</p></div></div><div class="section" id="sec-22"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">We thank two anonymous reviewers for helpful comments on the manuscript. This work was supported by the International Associated Laboratory "Ecological Genomics of Polyploidy" supported by CNRS (INEE, UMR CNRS 6553 Ecobio), University of Rennes 1, Iowa State University (Ames, USA), and the Partner University Funds (to M. A., A. S., J. B.). The analyses benefited from the Molecular Ecology Platform (UMR CNRS 6553 Ecobio) and Genouest (Bioinformatics) facilities. Author J. Boutte benefited from a PhD scholarship from the University of Rennes 1. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-4"><p id="p-46">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023242/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023242/-/DC1</a></p></li><li class="fn" id="fn-5"><p id="p-47"><em>Communicating editor: A. H. Paterson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-09-30"><span class="received-label">Received </span>September 30, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-10-23"><span class="accepted-label">Accepted </span>October 23, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Boutte <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.29.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ainouche</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chelaifa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira de Carvalho</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellot</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ainouche</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Polyploid evolution in <em>Spartina</em>: Dealing with highly redundant hybrid genomes</span>, pp. <span class="cit-fpage">225</span>–<span class="cit-lpage">243</span> in <span class="cit-source"><em>Polyploidy and Genome Evolution</em></span> edited by E. Douglas, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ainouche</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fortune</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parisod</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandbastien</span>  <span class="cit-name-given-names">M.-A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Hybridization, polyploidy and invasion: lessons from <em>Spartina</em> (Poaceae).</span> <abbr class="cit-jnl-abbrev">Biol. Invasions</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1159</span>–<span class="cit-lpage">1173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BInvasions%26rft.volume%253D11%26rft.spage%253D1159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.3" data-doi="10.1186/1471-2164-11-505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akhunova</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matniyazov</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Homoeolog-specific transcriptional bias in allopolyploid wheat.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAkhunova%26rft.auinit1%253DA.%2BR.%26rft.volume%253D11%26rft.spage%253D505%26rft.epage%253D505%26rft.atitle%253DHomoeolog-specific%2Btranscriptional%2Bbias%2Bin%2Ballopolyploid%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-505%26rft_id%253Dinfo%253Apmid%252F20849627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20849627&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.4" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schäffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.5" data-doi="10.1016/S1055-7903(03)00208-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Álvarez</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Ribosomal ITS sequences and plant phylogenetic inference.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">417</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DAlvarez%26rft.auinit1%253DI.%26rft.volume%253D29%26rft.issue%253D3%26rft.spage%253D417%26rft.epage%253D434%26rft.atitle%253DRibosomal%2BITS%2Bsequences%2Band%2Bplant%2Bphylogenetic%2Binference.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1055-7903%252803%252900208-2%26rft_id%253Dinfo%253Apmid%252F14615184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1055-7903(03)00208-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14615184&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186738000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badaeva</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friebe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Genome differentiation in <em>Aegilops</em>. 2. Physical mapping of 5S and 18S–26S ribosomal RNA gene families in diploid species.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1150</span>–<span class="cit-lpage">1158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DBadaeva%26rft.auinit1%253DE.%2BD.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D1150%26rft.epage%253D1158%26rft.atitle%253DGenome%2Bdifferentiation%2Bin%2BAegilops.%2B2.%2BPhysical%2Bmapping%2Bof%2B5S%2Band%2B18S-26S%2Bribosomal%2BRNA%2Bgene%2Bfamilies%2Bin%2Bdiploid%2Bspecies.%26rft_id%253Dinfo%253Apmid%252F18469963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18469963&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.7" data-doi="10.1016/0022-2836(88)90434-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harberd</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flavell</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Structure and evolution of the intergenic region in a ribosomal DNA repeat unit of wheat.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBarker%26rft.auinit1%253DR.%2BF.%26rft.volume%253D201%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D17%26rft.atitle%253DStructure%2Band%2Bevolution%2Bof%2Bthe%2Bintergenic%2Bregion%2Bin%2Ba%2Bribosomal%2BDNA%2Brepeat%2Bunit%2Bof%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252888%252990434-2%26rft_id%253Dinfo%253Apmid%252F3418693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(88)90434-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3418693&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988N259100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.8" data-doi="10.1046/j.1365-294X.2001.01299.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baumel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ainouche</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levasseur</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Molecular investigations in populations of <em>Spartina anglica</em> CE Hubbard (Poaceae) invading coastal Brittany (France).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1689</span>–<span class="cit-lpage">1701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBaumel%26rft.auinit1%253DA.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D1689%26rft.epage%253D1701%26rft.atitle%253DMolecular%2Binvestigations%2Bin%2Bpopulations%2Bof%2BSpartina%2Banglica%2BC.E.%2BHubbard%2B%2528Poaceae%2529%2Binvading%2Bcoastal%2BBrittany%2B%2528France%2529.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2001.01299.x%26rft_id%253Dinfo%253Apmid%252F11472537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2001.01299.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11472537&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170006100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.9" data-doi="10.1006/mpev.2001.1064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baumel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ainouche</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ainouche</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misset</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">molecular phylogeny of hybridizing species from the genus <em>Spartina</em> Schreb. (Poaceae).</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DBaumel%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D303%26rft.epage%253D314%26rft.atitle%253DMolecular%2Bphylogeny%2Bof%2Bhybridizing%2Bspecies%2Bfrom%2Bthe%2Bgenus%2BSpartina%2BSchreb.%2B%2528Poaceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fmpev.2001.1064%26rft_id%253Dinfo%253Apmid%252F11820850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/mpev.2001.1064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11820850&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173689000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.10" data-doi="10.1007/PL00006306"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bena</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jubier</span>  <span class="cit-name-given-names">M.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivieri</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lejeune</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Ribosomal external and internal transcribed spacers: combined use in the phylogenetic analysis of <em>Medicago</em> (<em>Leguminosae</em>).</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DBena%26rft.auinit1%253DG.%26rft.volume%253D46%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D306%26rft.atitle%253DRibosomal%2BExternal%2Band%2BInternal%2BTranscribed%2BSpacers%253A%2BCombined%2BUse%2Bin%2Bthe%2BPhylogenetic%2BAnalysis%2Bof%2BMedicago%2B%2528Leguminosae%2529%26rft_id%253Dinfo%253Adoi%252F10.1007%252FPL00006306%26rft_id%253Dinfo%253Apmid%252F9502673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/PL00006306&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9502673&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000071921900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.11" data-doi="10.1126/science.1253435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chalhoub</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denoeud</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkin</span>  <span class="cit-name-given-names">I. A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Early allopolyploid evolution in the post-Neolithic <em>Brassica napus</em> oilseed genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">345</span>: <span class="cit-fpage">950</span>–<span class="cit-lpage">953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChalhoub%26rft.auinit1%253DB.%26rft.volume%253D345%26rft.issue%253D6199%26rft.spage%253D950%26rft.epage%253D953%26rft.atitle%253DEarly%2Ballopolyploid%2Bevolution%2Bin%2Bthe%2Bpost-Neolithic%2BBrassica%2Bnapus%2Boilseed%2Bgenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1253435%26rft_id%253Dinfo%253Apmid%252F25146293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDUvNjE5OS85NTAiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.12" data-doi="10.1016/j.ygeno.2010.05.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">K.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">S.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">F.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chung</span>  <span class="cit-name-given-names">M.-C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Chromosomal conservation and sequence diversity of ribosomal RNA genes of two distant <em>Oryza</em> species.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DChang%26rft.auinit1%253DK.%2BD.%26rft.volume%253D96%26rft.issue%253D3%26rft.spage%253D181%26rft.epage%253D190%26rft.atitle%253DChromosomal%2Bconservation%2Band%2Bsequence%2Bdiversity%2Bof%2Bribosomal%2BRNA%2Bgenes%2Bof%2Btwo%2Bdistant%2BOryza%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2010.05.005%26rft_id%253Dinfo%253Apmid%252F20580815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2010.05.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20580815&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.1.29.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Combes</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cenci</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baraille</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertrand</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lashermes</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Homeologous gene expression in response to growing temperature in a recent allopolyploid (<em>Coffea arabica L</em>.).</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D103%26rft.spage%253D36%26rft_id%253Dinfo%253Apmid%252F22039298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22039298&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duchemin</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupont</span>  <span class="cit-name-given-names">P.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">HyLiTE: accurate and flexible analysis of gene expression in hybrid and allopolyploid species.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.29.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Metwally</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ouda</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helmy</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Next generation sequencing technologies and challenges in sequence assembly</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Twab</span>  <span class="cit-name-given-names">M. H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Physical mapping of the 45S rDNA on the chromosomes of <em>Triticum turgidum</em> and <em>T. aestivum</em> using fluorescence in situ hybridization for chromosome ancestors.</span> <abbr class="cit-jnl-abbrev">Arab J. Biotechnol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DArab%2BJ.%2BBiotechnol.%26rft.volume%253D10%26rft.spage%253D69%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.17" data-doi="10.1046/j.1365-313X.1995.08020295.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyanishi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugita</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugiura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">In vitro transcription of plant RNA polymerase I-dependent rRNA genes is species-specific.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFan%26rft.auinit1%253DH.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D295%26rft.epage%253D298%26rft.atitle%253DIn%2Bvitro%2Btranscription%2Bof%2Bplant%2BRNA%2Bpolymerase%2BI-dependent%2BrRNA%2Bgenes%2Bis%2Bspecies-specific.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.1995.08020295.x%26rft_id%253Dinfo%253Apmid%252F7670508%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.1995.08020295.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7670508&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RP94800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.18" data-doi="10.1186/1741-7007-6-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nettleton</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Duplicate gene expression in allopolyploid <em>Gossypium</em> reveals two temporally distinct phases of expression evolution.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BBiol%26rft.aulast%253DFlagel%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.spage%253D16%26rft.epage%253D16%26rft.atitle%253DDuplicate%2Bgene%2Bexpression%2Bin%2Ballopolyploid%2BGossypium%2Breveals%2Btwo%2Btemporally%2Bdistinct%2Bphases%2Bof%2Bexpression%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-6-16%26rft_id%253Dinfo%253Apmid%252F18416842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-6-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18416842&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.19" data-doi="10.1016/j.ympev.2006.11.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fortune</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierenbeck</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ainouche</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquemin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Evolutionary dynamics of Waxy and the origin of hexaploid <em>Spartina</em> species (Poaceae).</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1040</span>–<span class="cit-lpage">1055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DFortune%26rft.auinit1%253DP.%2BM.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D1040%26rft.epage%253D1055%26rft.atitle%253DEvolutionary%2Bdynamics%2Bof%2BWaxy%2Band%2Bthe%2Borigin%2Bof%2Bhexaploid%2BSpartina%2Bspecies%2B%2528Poaceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2006.11.018%26rft_id%253Dinfo%253Apmid%252F17208463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2006.11.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17208463&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247539000024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.20" data-doi="10.1111/j.1365-294X.2008.03916.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fortune</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schierenbeck</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayres</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bortolus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catrice</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The enigmatic invasive <em>Spartina densiflora</em> : A history of hybridizations in a polyploidy context.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">4304</span>–<span class="cit-lpage">4316</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DFortune%26rft.auinit1%253DP.%2BM.%26rft.volume%253D17%26rft.issue%253D19%26rft.spage%253D4304%26rft.epage%253D4316%26rft.atitle%253DThe%2Benigmatic%2Binvasive%2BSpartina%2Bdensiflora%253A%2Ba%2Bhistory%2Bof%2Bhybridizations%2Bin%2Ba%2Bpolyploidy%2Bcontext.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03916.x%26rft_id%253Dinfo%253Apmid%252F18986503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03916.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18986503&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259525900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-6.1.29.21"><div class="cit-metadata unstructured">Foucaud, 1897 Un <em>Spartina</em> inédit. Ann. Soc. Sci. Nat. Char. Inf. 32: 220–222.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.22" data-doi="10.1111/j.1755-0998.2009.02698.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frankowski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastrop</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identification of <em>Anguilla anguilla</em> (<em>L</em>.) and <em>Anguilla rostrata</em> (Le Sueur) and their hybrids based on a diagnostic single nucleotide polymorphism in nuclear 18S rDNA.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D10%26rft.spage%253D173%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2009.02698.x%26rft_id%253Dinfo%253Apmid%252F21565005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2009.02698.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21565005&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.23" data-doi="10.1093/nar/7.7.1869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerlach</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bedbrook</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Cloning and characterization of ribosomal RNA genes from wheat and barley.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1869</span>–<span class="cit-lpage">1885</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGerlach%26rft.auinit1%253DW.L.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D1869%26rft.epage%253D1885%26rft.atitle%253DCloning%2Band%2Bcharacterization%2Bof%2Bribosomal%2BRNA%2Bgenes%2Bfrom%2Bwheat%2Band%2Bbarley%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F7.7.1869%26rft_id%253Dinfo%253Apmid%252F537913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjcvNy8xODY5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-6.1.29.24"><div class="cit-metadata unstructured">Groves, H., and J. Groves, 1880 <em>Spartina x townsendii</em> Nobis. Rep. Bot. Soc. Exch. club Br. Id. 1–37.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dover</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Molecular coevolution among cryptically simple expansion segments of eukaryotic 26S/28S rRNAs.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DHancock%26rft.auinit1%253DJ.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D377%26rft.epage%253D391%26rft.atitle%253DMolecular%2Bcoevolution%2Bamong%2Bcryptically%2Bsimple%2Bexpansion%2Bsegments%2Bof%2Beukaryotic%2B26S%252F28S%2BrRNAs%26rft_id%253Dinfo%253Apmid%252F3405077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjUvNC8zNzciO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.26" data-doi="10.3732/ajb.1100312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ilut</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coate</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luciano</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owens</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comparative transcriptomic study of an allotetraploid and its diploid progenitors illustrates the unique advantages and challenges of RNA-seq in plant species.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">396</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1100312%26rft_id%253Dinfo%253Apmid%252F22301896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk5LzIvMzgzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.27" data-doi="10.1038/nature09916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickett</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayyampalayam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chanderbali</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landherr</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ancestral polyploidy in seed plants and angiosperms.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">473</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJiao%26rft.auinit1%253DY.%26rft.volume%253D473%26rft.issue%253D7345%26rft.spage%253D97%26rft.epage%253D100%26rft.atitle%253DAncestral%2Bpolyploidy%2Bin%2Bseed%2Bplants%2Band%2Bangiosperms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09916%26rft_id%253Dinfo%253Apmid%252F21478875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478875&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290218300038&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.28" data-doi="10.1093/bioinformatics/btq224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toh</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Parallelization of the MAFFT multiple sequence alignment program.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1899</span>–<span class="cit-lpage">1900</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq224%26rft_id%253Dinfo%253Apmid%252F20427515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjYvMTUvMTg5OSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS8yOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.29" data-doi="10.1186/1471-2164-11-97"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Homeolog loss and expression changes in natural populations of the recently and repeatedly formed allotetraploid <em>Tragopogon mirus</em> (Asteraceae).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKoh%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.spage%253D97%26rft.epage%253D97%26rft.atitle%253DHomeolog%2Bloss%2Band%2Bexpression%2Bchanges%2Bin%2Bnatural%2Bpopulations%2Bof%2Bthe%2Brecently%2Band%2Brepeatedly%2Bformed%2Ballotetraploid%2BTragopogon%2Bmirus%2B%2528Asteraceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-97%26rft_id%253Dinfo%253Apmid%252F20141639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-97&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20141639&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.30" data-doi="10.1534/genetics.104.032839"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kovarik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rapid concerted evolution of nuclear ribosomal DNA in two <em>Tragopogon</em> allopolyploids of recent and recurrent origin.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">944</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.032839%26rft_id%253Dinfo%253Apmid%252F15654116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY5LzIvOTMxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kovarik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dadejova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">Y. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chase</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarkson</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evolution of rDNA in <em>Nicotiana</em> allopolyploids: a potential link between rDNA homogenization and epigenetics. Ann.</span> <abbr class="cit-jnl-abbrev">Bot-London</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">815</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot-London%26rft.volume%253D101%26rft.spage%253D815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.32" data-doi="10.1093/oxfordjournals.molbev.a025922"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuzoff</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweere</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmer</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The phylogenetic potential of entire 26S rDNA sequences in plants.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DKuzoff%26rft.auinit1%253DR.%26rft.volume%253D15%26rft.issue%253D3%26rft.spage%253D251%26rft.epage%253D263%26rft.atitle%253DThe%2Bphylogenetic%2Bpotential%2Bof%2Bentire%2B26S%2BrDNA%2Bsequences%2Bin%2Bplants%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025922%26rft_id%253Dinfo%253Apmid%252F9501492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE1LzMvMjUxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.33" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.34" data-doi="10.1111/j.1095-8312.2004.00349.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leitch</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome downsizing in polyploid plants.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">651</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D82%26rft.spage%253D651%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.2004.00349.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.2004.00349.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000223808600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.35" data-doi="10.1104/pp.015727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The impact of polyploidy on grass genome evolution.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">1587</span>–<span class="cit-lpage">1593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DLevy%26rft.auinit1%253DA.%2BA.%26rft.volume%253D130%26rft.issue%253D4%26rft.spage%253D1587%26rft.epage%253D1593%26rft.atitle%253DThe%2BImpact%2Bof%2BPolyploidy%2Bon%2BGrass%2BGenome%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.015727%26rft_id%253Dinfo%253Apmid%252F12481041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTMwLzQvMTU4NyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS8yOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.36" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS8yOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.37" data-doi="10.1186/1471-2229-11-157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Conservation and loss of ribosomal RNA gene sites in diploid and polyploid <em>Fragaria</em> (Rosaceae).</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DLiu%26rft.auinit1%253DB.%26rft.volume%253D11%26rft.spage%253D157%26rft.epage%253D157%26rft.atitle%253DConservation%2Band%2Bloss%2Bof%2Bribosomal%2BRNA%2Bgene%2Bsites%2Bin%2Bdiploid%2Band%2Bpolyploid%2BFragaria%2B%2528Rosaceae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-11-157%26rft_id%253Dinfo%253Apmid%252F22074487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-11-157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22074487&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.38" data-doi="10.1111/j.1095-8312.2004.00332.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">B. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">“Why polyploidy is rarer in animals than in plants”: myths and mechanisms.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D82%26rft.spage%253D453%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1095-8312.2004.00332.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1095-8312.2004.00332.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000223808600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mahé</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pascual</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coriton</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huteau</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro Perris</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New data and phylogenetic placement of the enigmatic Old World lupin: <em>Lupinus mariae</em>-<em>josephi</em> H.</span> <abbr class="cit-jnl-abbrev">Pascual. Genet. Resour. Crop Ev.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPascual.%2BGenet.%2BResour.%2BCrop%2BEv.%26rft.volume%253D58%26rft.spage%253D101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maluszynska</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslop-Harrison</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Physical mapping of rDNA loci in <em>Brassica</em> species.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">774</span>–<span class="cit-lpage">781</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DMaluszynska%26rft.auinit1%253DJ.%26rft.volume%253D36%26rft.issue%253D4%26rft.spage%253D774%26rft.epage%253D781%26rft.atitle%253DPhysical%2Bmapping%2Bof%2BrDNA%2Bloci%2Bin%2BBrassica%2Bspecies.%26rft_id%253Dinfo%253Apmid%252F18470023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18470023&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchant</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Evolution in <em>Spartina</em> (Gramineae). II. Chromosomes, basic relationships and the problem of <em>Spartina x townsendii</em> agg.</span> <abbr class="cit-jnl-abbrev">Biol. J. Linn. Soc. Lond.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BJ.%2BLinn.%2BSoc.%2BLond.%26rft.volume%253D60%26rft.spage%253D381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.42" data-doi="10.1038/nature03959"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margulies</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Egholm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altman</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attiya</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genome sequencing in microfabricated high-density picolitre reactors.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">380</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMargulies%26rft.auinit1%253DM.%26rft.volume%253D437%26rft.issue%253D7057%26rft.spage%253D376%26rft.epage%253D380%26rft.atitle%253DGenome%2Bsequencing%2Bin%2Bmicrofabricated%2Bhigh-density%2Bpicolitre%2Breactors.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03959%26rft_id%253Dinfo%253Apmid%252F16056220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03959&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16056220&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231849100045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.43" data-doi="10.1186/1471-2164-13-722"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matyášek</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renny-Byfield</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulneček</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandbastien</span>  <span class="cit-name-given-names">M.-A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Next generation sequencing analysis reveals a relationship between rDNA unit diversity and locus number in <em>Nicotiana</em> diploids.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D13%26rft.spage%253D722%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-722%26rft_id%253Dinfo%253Apmid%252F23259460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-722&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23259460&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.44" data-doi="10.1186/1756-0500-4-93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mentewab</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsen</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flowers</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Incomplete homogenization of 18S ribosomal DNA coding regions in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DMentewab%26rft.auinit1%253DA.%2BB.%26rft.volume%253D4%26rft.spage%253D93%26rft.epage%253D93%26rft.atitle%253DIncomplete%2Bhomogenization%2Bof%2B18%2BS%2Bribosomal%2BDNA%2Bcoding%2Bregions%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-4-93%26rft_id%253Dinfo%253Apmid%252F21453453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-4-93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21453453&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.45" data-doi="10.1038/nrg2626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Metzker</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sequencing technologies—the next generation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D11%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2626%26rft_id%253Dinfo%253Apmid%252F19997069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2626&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19997069&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272867700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardle</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Tablet–next generation sequence assembly visualization.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D26%26rft.spage%253D401%26rft_id%253Dinfo%253Apmid%252F19965881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19965881&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274342800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.47" data-doi="10.1146/annurev.genet.39.073003.112240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Concerted and birth-and-death evolution of multigene families.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D39%26rft.spage%253D121%26rft.epage%253D152%26rft.atitle%253DConcerted%2Band%2Bbirth-and-death%2Bevolution%2Bof%2Bmultigene%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.39.073003.112240%26rft_id%253Dinfo%253Apmid%252F16285855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.39.073003.112240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16285855&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234685200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.48" data-doi="10.1186/1471-2105-11-378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novák</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neumann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macas</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Graph-based clustering and characterization of repetitive sequences in next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DNovak%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.spage%253D378%26rft.epage%253D378%26rft.atitle%253DGraph-based%2Bclustering%2Band%2Bcharacterization%2Bof%2Brepetitive%2Bsequences%2Bin%2Bnext-generation%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-378%26rft_id%253Dinfo%253Apmid%252F20633259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20633259&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.29.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source"><em>Evolution by Gene Duplication</em>.</span> <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.50" data-doi="10.1534/g3.112.005298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gingle</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">PolyCat: A resource for genome categorization of sequencing reads from allopolyploid organisms.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">525</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DPage%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D517%26rft.epage%253D525%26rft.atitle%253DPolyCat%253A%2BA%2BResource%2Bfor%2BGenome%2BCategorization%2Bof%2BSequencing%2BReads%2BFrom%2BAllopolyploid%2BOrganisms%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.005298%26rft_id%253Dinfo%253Apmid%252F23450226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjMvMy81MTciO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.51" data-doi="10.1534/g3.113.007229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liechty</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grupp</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stelly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Insights into the evolution of cotton diploids and polyploids from whole-genome re-sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1809</span>–<span class="cit-lpage">1818</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007229%26rft_id%253Dinfo%253Apmid%252F23979935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTAvMTgwOSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS8yOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.52" data-doi="10.1186/1756-0500-7-829"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liechty</span>  <span class="cit-name-given-names">Z. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">BamBam: genome sequence analysis tools for biologists.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BRes.%2BNotes%26rft.volume%253D7%26rft.spage%253D829%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-7-829%26rft_id%253Dinfo%253Apmid%252F25421351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-7-829&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25421351&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peralta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Combes</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cenci</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lashermes</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dereeper</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SNiPloid: A utility to exploit high-throughput SNP data derived from RNA-Seq in allopolyploid species.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> <span class="cit-vol">2013</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPlant%2BGenomics%26rft.volume%253D2013%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.54" data-doi="10.12705/636.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romaschenko</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arrieta</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saarela</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A molecular phylogeny and new subgeneric classification of <em>Sporobolus</em> (Poaceae: Chloridoideae: Sporobolinae).</span> <abbr class="cit-jnl-abbrev">Taxon</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">1212</span>–<span class="cit-lpage">1243</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTaxon%26rft.volume%253D63%26rft.spage%253D1212%26rft_id%253Dinfo%253Adoi%252F10.12705%252F636.19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12705/636.19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.55" data-doi="10.1007/s11033-009-9630-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poczai</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyvönen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nuclear ribosomal spacer regions in plant phylogenetics: problems and prospects.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Rep.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1897</span>–<span class="cit-lpage">1912</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bbiology%2Breports%26rft.stitle%253DMol%2BBiol%2BRep%26rft.aulast%253DPoczai%26rft.auinit1%253DP.%26rft.volume%253D37%26rft.issue%253D4%26rft.spage%253D1897%26rft.epage%253D1912%26rft.atitle%253DNuclear%2Bribosomal%2Bspacer%2Bregions%2Bin%2Bplant%2Bphylogenetics%253A%2Bproblems%2Band%2Bprospects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11033-009-9630-3%26rft_id%253Dinfo%253Apmid%252F19626457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11033-009-9630-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19626457&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.56" data-doi="10.1139/g02-103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prokopowich</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crease</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The correlation between rDNA copy number and genome size in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DProkopowich%26rft.auinit1%253DC.%2BD.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D48%26rft.epage%253D50%26rft.atitle%253DThe%2Bcorrelation%2Bbetween%2BrDNA%2Bcopy%2Bnumber%2Band%2Bgenome%2Bsize%2Bin%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg02-103%26rft_id%253Dinfo%253Apmid%252F12669795%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g02-103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12669795&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.57" data-doi="10.1146/annurev.ecolsys.29.1.467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramsey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schemske</span>  <span class="cit-name-given-names">D. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Pathways, mechanisms, and rates of polyploid formation in flowering plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">467</span>–<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DRamsey%26rft.auinit1%253DJ.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D467%26rft.epage%253D501%26rft.atitle%253DPATHWAYS%252C%2BMECHANISMS%252C%2BAND%2BRATES%2BOF%2BPOLYPLOID%2BFORMATION%2BIN%2BFLOWERING%2BPLANTS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.29.1.467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.29.1.467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000077648200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-other" id="cit-6.1.29.58"><div class="cit-metadata unstructured">Rogers, S. O., and A. J. Bendich. 1987 Ribosomal RNA genes in plants: variability in copy number and in the intergenic spacer. Plant Mol. Biol. 9:509:520.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castro</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rossello</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Relationships of the woody <em>Medicago</em> species (Section Dendrotelis) assessed by molecular cytogenetic analyses. Ann.</span> <abbr class="cit-jnl-abbrev">Bot-London</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBot-London%26rft.volume%253D102%26rft.spage%253D15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.60" data-doi="10.1016/j.ympev.2015.06.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rousseau-Gueutin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellot</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boutte</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chelaifa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The chloroplast genome of the hexaploid <em>Spartina maritima</em> (Poaceae, Chloridoideae): Comparative analyses and molecular dating.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D93%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2015.06.013%26rft_id%253Dinfo%253Apmid%252F26182838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2015.06.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26182838&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.61" data-doi="10.1385/1-59259-192-2:365"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skaletsky</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Primer3 on the WWW for general users and for biologist programmers.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">365</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DRozen%26rft.auinit1%253DS.%26rft.volume%253D132%26rft.spage%253D365%26rft.epage%253D386%26rft.atitle%253DPrimer3%2Bon%2Bthe%2BWWW%2Bfor%2Bgeneral%2Busers%2Band%2Bfor%2Bbiologist%2Bprogrammers.%26rft_id%253Dinfo%253Adoi%252F10.1385%252F1-59259-192-2%253A365%26rft_id%253Dinfo%253Apmid%252F10547847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1385/1-59259-192-2:365&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10547847&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.62" data-doi="10.1111/j.1469-8137.2009.03093.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Homoeologous nonreciprocal recombination in polyploid cotton.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSalmon%26rft.auinit1%253DA.%26rft.volume%253D186%26rft.issue%253D1%26rft.spage%253D123%26rft.epage%253D134%26rft.atitle%253DHomoeologous%2Bnonreciprocal%2Brecombination%2Bin%2Bpolyploid%2Bcotton.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03093.x%26rft_id%253Dinfo%253Apmid%252F19925554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03093.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19925554&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275184700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.63" data-doi="10.2307/2399280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaal</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Learn</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Ribosomal DNA variation within and among plant populations.</span> <abbr class="cit-jnl-abbrev">Ann. Mo. Bot. Gard.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">1207</span>–<span class="cit-lpage">1216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BMo.%2BBot.%2BGard.%26rft.volume%253D75%26rft.spage%253D1207%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2399280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2399280&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.64" data-doi="10.1007/s13258-013-0084-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seo</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bae</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sequence polymorphisms in ribosomal RNA genes and variations in chromosomal loci of <em>Oenothera odorata</em> and <em>O. laciniata</em>.</span> <abbr class="cit-jnl-abbrev">Genes Genomics</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomics%26rft.volume%253D35%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13258-013-0084-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13258-013-0084-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snowdon</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Köhler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Köhler</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Chromosomal localization and characterization of rDNA loci in the <em>Brassica</em> A and C genomes.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">582</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DSnowdon%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D40%26rft.issue%253D4%26rft.spage%253D582%26rft.epage%253D587%26rft.atitle%253DChromosomal%2Blocalization%2Band%2Bcharacterization%2Bof%2BrDNA%2Bloci%2Bin%2Bthe%2BBrassica%2BA%2Band%2BC%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F18464849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18464849&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.66" data-doi="10.3732/ajb.0800079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leebens-Mack</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Polyploidy and angiosperm diversification.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">336</span>–<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DSoltis%26rft.auinit1%253DD.%2BE.%26rft.volume%253D96%26rft.issue%253D1%26rft.spage%253D336%26rft.epage%253D348%26rft.atitle%253DPolyploidy%2Band%2Bangiosperm%2Bdiversification%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.0800079%26rft_id%253Dinfo%253Apmid%252F21628192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk2LzEvMzM2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.67" data-doi="10.1111/jse.12150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soreng</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romaschenko</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidse</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuloaga</span>  <span class="cit-name-given-names">F. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A worldwide phylogenetic classification of the Poaceae (Gramineae): Phylogenetic classification of the grasses.</span> <abbr class="cit-jnl-abbrev">J. Syst. Evol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BSyst.%2BEvol.%26rft.volume%253D53%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjse.12150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jse.12150&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.68" data-doi="10.1146/annurev-ecolsys-110512-135803"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strong</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayres</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Ecological and evolutionary misadventures of <em>Spartina</em>.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D44%26rft.spage%253D389%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-ecolsys-110512-135803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-ecolsys-110512-135803&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.69" data-doi="10.1093/gbe/evu261"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tennessen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govindarajulu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashman</span>  <span class="cit-name-given-names">T.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liston</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolutionary origins and dynamics of octoploid strawberry subgenomes revealed by dense targeted capture linkage maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">3295</span>–<span class="cit-lpage">3313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevu261%26rft_id%253Dinfo%253Apmid%252F25477420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjk6IjYvMTIvMzI5NSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS8yOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.70" data-doi="10.1093/nar/14.12.4969"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toloczyki</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feix</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Occurrence of 9 homologous repeat units in the external spacer region of a nuclear maize rRNA gene unit.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">4969</span>–<span class="cit-lpage">4986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DToloczyki%26rft.auinit1%253DC.%26rft.volume%253D14%26rft.issue%253D12%26rft.spage%253D4969%26rft.epage%253D4986%26rft.atitle%253DOccurrence%2Bof%2B9%2Bhomologous%2Brepeat%2Bunits%2Bin%2Bthe%2Bexternal%2Bspacer%2Bregion%2Bof%2Ba%2Bnuclear%2Bmaize%2BrRNA%2Bgene%2Bunit%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F14.12.4969%26rft_id%253Dinfo%253Apmid%252F3014440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8xMi80OTY5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.71" data-doi="10.1101/gr.4602906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Udall</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rapp</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sparks</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A global assembly of cotton ESTs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">450</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4602906%26rft_id%253Dinfo%253Apmid%252F16478941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzMvNDQxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.72" data-doi="10.1038/nrg2600"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van de Peer</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maere</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The evolutionary significance of ancient genome duplications.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">725</span>–<span class="cit-lpage">732</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DVan%2Bde%2BPeer%26rft.auinit1%253DY.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D725%26rft.epage%253D732%26rft.atitle%253DThe%2Bevolutionary%2Bsignificance%2Bof%2Bancient%2Bgenome%2Bduplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2600%26rft_id%253Dinfo%253Apmid%252F19652647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2600&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19652647&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F29.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269965100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.73" data-doi="10.1007/BF00989055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volkov</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kostishin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrendorfer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweizer</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Molecular organization and evolution of the external transcribed rDNA spacer region in two diploid relatives of <em>Nicotiana tabacum</em> (<em>Solanaceae</em>).</span> <abbr class="cit-jnl-abbrev">Plant Syst. Evol.</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSyst.%2BEvol.%26rft.volume%253D201%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00989055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00989055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.74" data-doi="10.1093/oxfordjournals.molbev.a026112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volkov</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borisjuk</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panchuk</span>  <span class="cit-name-given-names">I. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweizer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemleben</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Elimination and rearrangement of parental rDNA in the allotetraploid <em>Nicotiana tabacum</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DVolkov%26rft.auinit1%253DR.%26rft.volume%253D16%26rft.issue%253D3%26rft.spage%253D311%26rft.epage%253D320%26rft.atitle%253DElimination%2Band%2Brearrangement%2Bof%2Bparental%2BrDNA%2Bin%2Bthe%2Ballotetraploid%2BNicotiana%2Btabacum%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026112%26rft_id%253Dinfo%253Apmid%252F10331258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE2LzMvMzExIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvNi8xLzI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.75" data-doi="10.1093/bioinformatics/btp033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Procter</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">D. M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clamp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Jalview Version 2–a multiple sequence alignment editor and analysis workbench.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1189</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp033%26rft_id%253Dinfo%253Apmid%252F19151095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyNS85LzExODkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy82LzEvMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.76" data-doi="10.1023/A:1006392424384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genome evolution in polyploids.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Niol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BNiol.%26rft.volume%253D42%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1006392424384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1006392424384&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.29.77" data-doi="10.1073/pnas.92.1.280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seelanan</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bidirectional interlocus concerted evolution following allopolyploid speciation in cotton (<em>Gossypium</em>).</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWendel%26rft.auinit1%253DJ.%2BF.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D280%26rft.epage%253D284%26rft.atitle%253DBidirectional%2Binterlocus%2Bconcerted%2Bevolution%2Bfollowing%2Ballopolyploid%2Bspeciation%2Bin%2Bcotton%2B%2528Gossypium%2529.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.1.280%26rft_id%253Dinfo%253Apmid%252F7816833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5Mi8xLzI4MCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzYvMS8yOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.29.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zentgraf</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velasco</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hemleben</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Different Transcriptional Activities in the Nucleus</span>, pp. <span class="cit-fpage">131</span>–<span class="cit-lpage">168</span> in <span class="cit-source">Progress in Botany</span>, edited by H.-D. <span class="cit-ed"><span class="cit-name-surname">Behnke</span>  <span class="cit-name-given-names">K.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Esser</span>  <span class="cit-name-given-names">J. W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Kadereit</span>  <span class="cit-name-given-names">U.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Lüttge</span></span>, <span class="cit-ed"><span class="cit-name-surname">Runge</span>  <span class="cit-name-given-names">M.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/1/29.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/15" title="Comparative Genomics of an Emerging Amphibian Virus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/41" title="Disease Variant Landscape of a Large Multiethnic Population of Moyamoya Patients by Exome Sequencing" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/1"><span>Volume 6 Issue 1, January 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/1.cover-source.jpg?itok=9zFiZvCK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/29?cited-by=yes&amp;legid=ggg;6/1/29" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-foN5m5CL9LSAt7gf08n38tTBB9YrGwczhMJnRhBiUwE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/1/29" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/1/29" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/1/29" class="active" data-icon-position="" data-hide-link-title="0">Haplotype Detection from Next-Generation Sequencing in High-Ploidy-Level Species: 45S rDNA Gene Copies in the Hexaploid Spartina maritima</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460240" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-CzVWAMB-_DiagriKDX-L6GlwG75NLXlKIx5PDbETBHU" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/29.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/1/29" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-FRaDfI_DrmUyAJtLabtfcwJKtPsCUNuk8lVA3-EHFCI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/1/29" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/29.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460240" id="node460240--2" data-pisa="ggg;6/1/29" data-pisa-master="ggg;g3.115.023242" data-apath="/ggg/6/1/29.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Haplotype Detection from Next-Generation Sequencing in High-Ploidy-Level Species: 45S rDNA Gene Copies in the Hexaploid <em>Spartina maritima</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Julien Boutte</span>, <span class="highwire-citation-author" data-delta="1">Benoît Aliaga</span>, <span class="highwire-citation-author" data-delta="2">Oscar Lima</span>, <span class="highwire-citation-author" data-delta="3">Julie Ferreira de Carvalho</span>, <span class="highwire-citation-author" data-delta="4">Abdelkader Ainouche</span>, <span class="highwire-citation-author" data-delta="5">Jiri Macas</span>, <span class="highwire-citation-author" data-delta="6">Mathieu Rousseau-Gueutin</span>, <span class="highwire-citation-author" data-delta="7">Olivier Coriton</span>, <span class="highwire-citation-author" data-delta="8">Malika Ainouche</span> and <span class="highwire-citation-author" data-delta="9">Armel Salmon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">29-40;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023242 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460240--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Julien Boutte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julien%2BBoutte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boutte%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulien%2BBoutte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benoît Aliaga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beno%C3%AEt%2BAliaga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aliaga%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeno%25C3%25AEt%2BAliaga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Oscar Lima </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oscar%2BLima%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lima%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOscar%2BLima%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Julie Ferreira de Carvalho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julie%2BFerreira%2Bde%2BCarvalho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferreira%20de%20Carvalho%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulie%2BFerreira%2Bde%2BCarvalho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Abdelkader Ainouche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abdelkader%2BAinouche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ainouche%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbdelkader%2BAinouche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jiri Macas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biology Centre ASCR, Institute of Plant Molecular Biology, Branišovská 31, České Budějovice, CZ-37005, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiri%2BMacas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiri%2BMacas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mathieu Rousseau-Gueutin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>UMR Institut de Génétique, Environnement et Protection des Plantes, Institut National de la Recherche Agronomique, BP35327, 35653 Le Rheu Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BRousseau-Gueutin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rousseau-Gueutin%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMathieu%2BRousseau-Gueutin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Olivier Coriton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>UMR Institut de Génétique, Environnement et Protection des Plantes, Institut National de la Recherche Agronomique, BP35327, 35653 Le Rheu Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BCoriton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coriton%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlivier%2BCoriton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Malika Ainouche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malika%2BAinouche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ainouche%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMalika%2BAinouche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Armel Salmon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Armel%2BSalmon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salmon%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArmel%2BSalmon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:armel.salmon@univ-rennes1.fr" class="" data-icon-position="" data-hide-link-title="0">armel.salmon@univ-rennes1.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/1/29" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/1/29&amp;title=Haplotype%20Detection%20from%20Next-Generation%20Sequencing%20in%20High-Ploidy-Level%20Species%3A%2045S%20rDNA%20Gene%20Copies%20in%20the%20Hexaploid%20Spartina%20maritima" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/1/29&amp;title=Haplotype%20Detection%20from%20Next-Generation%20Sequencing%20in%20High-Ploidy-Level%20Species%3A%2045S%20rDNA%20Gene%20Copies%20in%20the%20Hexaploid%20Spartina%20maritima" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/1/29&amp;title=Haplotype%20Detection%20from%20Next-Generation%20Sequencing%20in%20High-Ploidy-Level%20Species%3A%2045S%20rDNA%20Gene%20Copies%20in%20the%20Hexaploid%20Spartina%20maritima" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/1/29&amp;text=Haplotype%20Detection%20from%20Next-Generation%20Sequencing%20in%20High-Ploidy-Level%20Species%3A%2045S%20rDNA%20Gene%20Copies%20in%20the%20Hexaploid%20Spartina%20maritima" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/1/29&amp;title=Haplotype%20Detection%20from%20Next-Generation%20Sequencing%20in%20High-Ploidy-Level%20Species%3A%2045S%20rDNA%20Gene%20Copies%20in%20the%20Hexaploid%20Spartina%20maritima" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/1/29&amp;t=Haplotype%20Detection%20from%20Next-Generation%20Sequencing%20in%20High-Ploidy-Level%20Species%3A%2045S%20rDNA%20Gene%20Copies%20in%20the%20Hexaploid%20Spartina%20maritima" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/1/29&amp;title=Haplotype%20Detection%20from%20Next-Generation%20Sequencing%20in%20High-Ploidy-Level%20Species%3A%2045S%20rDNA%20Gene%20Copies%20in%20the%20Hexaploid%20Spartina%20maritima" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/1/29&amp;title=Haplotype%20Detection%20from%20Next-Generation%20Sequencing%20in%20High-Ploidy-Level%20Species%3A%2045S%20rDNA%20Gene%20Copies%20in%20the%20Hexaploid%20Spartina%20maritima" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460240/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460240" id="node460240--3" data-pisa="ggg;6/1/29" data-pisa-master="ggg;g3.115.023242" data-apath="/ggg/6/1/29.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Haplotype Detection from Next-Generation Sequencing in High-Ploidy-Level Species: 45S rDNA Gene Copies in the Hexaploid <em>Spartina maritima</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Julien Boutte</span>, <span class="highwire-citation-author" data-delta="1">Benoît Aliaga</span>, <span class="highwire-citation-author" data-delta="2">Oscar Lima</span>, <span class="highwire-citation-author" data-delta="3">Julie Ferreira de Carvalho</span>, <span class="highwire-citation-author" data-delta="4">Abdelkader Ainouche</span>, <span class="highwire-citation-author" data-delta="5">Jiri Macas</span>, <span class="highwire-citation-author" data-delta="6">Mathieu Rousseau-Gueutin</span>, <span class="highwire-citation-author" data-delta="7">Olivier Coriton</span>, <span class="highwire-citation-author" data-delta="8">Malika Ainouche</span> and <span class="highwire-citation-author" data-delta="9">Armel Salmon</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">29-40;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023242 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460240--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Julien Boutte </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julien%2BBoutte%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boutte%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulien%2BBoutte%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benoît Aliaga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beno%C3%AEt%2BAliaga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aliaga%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABeno%25C3%25AEt%2BAliaga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Oscar Lima </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oscar%2BLima%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lima%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOscar%2BLima%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Julie Ferreira de Carvalho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julie%2BFerreira%2Bde%2BCarvalho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferreira%20de%20Carvalho%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJulie%2BFerreira%2Bde%2BCarvalho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Abdelkader Ainouche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abdelkader%2BAinouche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ainouche%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbdelkader%2BAinouche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jiri Macas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biology Centre ASCR, Institute of Plant Molecular Biology, Branišovská 31, České Budějovice, CZ-37005, Czech Republic</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiri%2BMacas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Macas%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiri%2BMacas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Mathieu Rousseau-Gueutin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>UMR Institut de Génétique, Environnement et Protection des Plantes, Institut National de la Recherche Agronomique, BP35327, 35653 Le Rheu Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mathieu%2BRousseau-Gueutin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rousseau-Gueutin%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMathieu%2BRousseau-Gueutin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Olivier Coriton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>UMR Institut de Génétique, Environnement et Protection des Plantes, Institut National de la Recherche Agronomique, BP35327, 35653 Le Rheu Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BCoriton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coriton%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOlivier%2BCoriton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Malika Ainouche </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malika%2BAinouche%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ainouche%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMalika%2BAinouche%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Armel Salmon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR CNRS 6553 Ecobio, OSUR (Observatoire des Sciences de l’Univers de Rennes), University of Rennes 1, Bât 14A Campus Scientifique de Beaulieu, 35 042 Rennes Cedex, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Armel%2BSalmon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Salmon%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArmel%2BSalmon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:armel.salmon@univ-rennes1.fr" class="" data-icon-position="" data-hide-link-title="0">armel.salmon@univ-rennes1.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=29&amp;rft.epage=40&amp;rft.atitle=Haplotype%20Detection%20from%20Next-Generation%20Sequencing%20in%20High-Ploidy-Level%20Species%3A%2045S%20rDNA%20Gene%20Copies%20in%20the%20Hexaploid%20Spartina%20maritima&amp;rft.volume=6&amp;rft.issue=1&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Boutte%2C+Julien&amp;rft.au=Aliaga%2C+Beno%C3%AEt&amp;rft.au=Lima%2C+Oscar&amp;rft.au=Ferreira+de+Carvalho%2C+Julie&amp;rft.au=Ainouche%2C+Abdelkader&amp;rft.au=Macas%2C+Jiri&amp;rft.au=Rousseau-Gueutin%2C+Mathieu&amp;rft.au=Coriton%2C+Olivier&amp;rft.au=Ainouche%2C+Malika&amp;rft.au=Salmon%2C+Armel"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460240/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460240/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460240/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460240/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460240/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460240/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460240/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460240/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460240/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460240/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460240/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460240/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/1/29" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/1/29.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/1/29.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/1/29.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/1/29.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
