<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/1/107" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/1/107.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/1/107.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/1/107.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/1/107/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/1/107.atom" />
<meta name="HW.pisa" content="ggg;6/1/107" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV" />
<meta name="DC.Identifier" content="10.1534/g3.115.024208" />
<meta name="DC.Date" content="2016-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Alcala et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Ease of worldwide travel provides increased opportunities for organisms not only to colonize new environments but also to encounter related but diverged populations. Such events of reconnection and secondary contact of previously isolated populations are widely observed at different time scales. For example, during the quaternary glaciation, sea water level fluctuations caused temporal isolation of populations, often to be followed by secondary contact. At shorter time scales, population isolation and reconnection of viruses are commonly observed, and such events are often associated with epidemics and pandemics. Here, using coalescent theory and simulations, we describe the temporal impact of population reconnection after isolation on nucleotide differences and the site frequency spectrum, as well as common summary statistics of DNA variation. We identify robust genomic signatures of population reconnection after isolation. We utilize our development to infer the recent evolutionary history of human immunodeficiency virus 1 (HIV-1) in Asia and South America, successfully retrieving the successive HIV subtype colonization events in these regions. Our analysis reveals that divergent HIV-1 subtype populations are currently admixing in these regions, suggesting that HIV-1 may be undergoing a process of homogenization, contrary to popular belief." />
<meta name="DC.Contributor" content="Nicolas Alcala" />
<meta name="DC.Contributor" content="Jeffrey D. Jensen" />
<meta name="DC.Contributor" content="Amalio Telenti" />
<meta name="DC.Contributor" content="Séverine Vuilleumier" />
<meta name="article:published_time" content="2016-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Ease of worldwide travel provides increased opportunities for organisms not only to colonize new environments but also to encounter related but diverged populations. Such events of reconnection and secondary contact of previously isolated populations are widely observed at different time scales. For example, during the quaternary glaciation, sea water level fluctuations caused temporal isolation of populations, often to be followed by secondary contact. At shorter time scales, population isolation and reconnection of viruses are commonly observed, and such events are often associated with epidemics and pandemics. Here, using coalescent theory and simulations, we describe the temporal impact of population reconnection after isolation on nucleotide differences and the site frequency spectrum, as well as common summary statistics of DNA variation. We identify robust genomic signatures of population reconnection after isolation. We utilize our development to infer the recent evolutionary history of human immunodeficiency virus 1 (HIV-1) in Asia and South America, successfully retrieving the successive HIV subtype colonization events in these regions. Our analysis reveals that divergent HIV-1 subtype populations are currently admixing in these regions, suggesting that HIV-1 may be undergoing a process of homogenization, contrary to popular belief.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/01/01" />
<meta name="citation_mjid" content="ggg;6/1/107" />
<meta name="citation_id" content="6/1/107" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/1/107" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/1/107.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/1/107.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/1/107.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.024208" />
<meta name="citation_pmid" content="26546308" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="107" />
<meta name="citation_lastpage" content="120" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nicolas Alcala" />
<meta name="citation_author_institution" content="Department of Ecology and Evolution, University of Lausanne, Lausanne CH-1015, Switzerland" />
<meta name="citation_author_institution" content="Department of Biology, Stanford University, Stanford, California 94305-5020" />
<meta name="citation_author_email" content="nalcala@stanford.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5961-5064" />
<meta name="citation_author" content="Jeffrey D. Jensen" />
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne CH-1015, Switzerland" />
<meta name="citation_author" content="Amalio Telenti" />
<meta name="citation_author_institution" content="Genomic Medicine, The J. Craig Venter Institute, 4120 Capricorn Lane, La Jolla, California 92037" />
<meta name="citation_author" content="Séverine Vuilleumier" />
<meta name="citation_author_institution" content="Department of Ecology and Evolution, University of Lausanne, Lausanne CH-1015, Switzerland" />
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne CH-1015, Switzerland" />
<meta name="citation_author_institution" content="Institute of Microbiology, University Hospital and University of Lausanne, Lausanne CH-1011, Switzerland" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdiscip. Rev. Comput. Stat.;citation_author=H. Abdi;citation_author=L. J. Williams;citation_title=Principal component analysis.;citation_pages=433-459;citation_volume=2;citation_year=2010;citation_doi=10.1002/wics.101" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Achaz;citation_title=Frequency spectrum neutrality tests: one for all and all for one.;citation_volume=183;citation_year=2009;citation_pmid=19546320;citation_doi=10.1534/genetics.109.104042" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=R. Aguilée;citation_author=D. Claessen;citation_author=A. Lambert;citation_title=Adaptive radiation driven by the interplay of eco-evolutionary and landscape dynamics.;citation_pages=1291-1306;citation_volume=67;citation_year=2013;citation_pmid=23617909" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=N. Alcala;citation_author=S. Vuilleumier;citation_title=Turnover and accumulation of genetic diversity across large time-scale cycles of isolation and connection of populations.;citation_volume=281;citation_year=2014;citation_pmid=25253456;citation_doi=10.1098/rspb.2014.1369" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Alcala;citation_author=D. Streit;citation_author=J. Goudet;citation_author=S. Vuilleumier;citation_title=Peak and persistent excess of genetic diversity following an abrupt migration increase.;citation_volume=193;citation_year=2013;citation_pmid=23307901;citation_doi=10.1534/genetics.112.147785" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=M. An;citation_author=X. Han;citation_author=J. Xu;citation_author=Z. Chu;citation_author=M. Jia;citation_title=Reconstituting the epidemic history of HIV strain crf01_ae among men who have sex with men (msm) in liaoning, northeastern China: Implications for the expanding epidemic among msm in China.;citation_volume=86;citation_year=2012;citation_pmid=22933290;citation_doi=10.1128/JVI.00262-12" />
<meta name="citation_reference" content="citation_journal_title=AIDS research and human retroviruses;citation_journal_abbrev=AIDS Res Hum Retroviruses;citation_author=P. C. Aulicino;citation_author=G. Bello;citation_author=C. Rocco;citation_author=H. Romero;citation_author=A. Mangano;citation_title=Description of the first full-length HIV type 1 subtype F1 strain in Argentina: implications for the origin and dispersion of this subtype in South America.;citation_pages=1176-1182;citation_volume=23;citation_year=2007;citation_issue=10;citation_pmid=17961101;citation_doi=10.1089/aid.2007.0038" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Bailes;citation_author=F. Gao;citation_author=F. Bibollet-Ruche;citation_author=V. Courgnaud;citation_author=M. Peeters;citation_title=Hybrid Origin of SIV in Chimpanzees;citation_pages=1713-1713;citation_volume=300;citation_year=2003;citation_issue=5626;citation_pmid=12805540;citation_doi=10.1126/science.1080657" />
<meta name="citation_reference" content="citation_journal_title=Proc. R. Soc. Lond. A. Math. Phys. Sci.;citation_author=M. S. Bartlett;citation_title=Properties of sufficiency and statistical tests.;citation_pages=268-282;citation_volume=160;citation_year=1937;citation_doi=10.1098/rspa.1937.0109" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=N. H. Barton;citation_title=The effect of hitch-hiking on neutral genealogies.;citation_pages=123-133;citation_volume=72;citation_year=1998;citation_doi=10.1017/S0016672398003462" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=N. H. Barton;citation_author=A. M. Etheridge;citation_title=The Effect of Selection on Genealogies;citation_pages=1115-1131;citation_volume=166;citation_year=2004;citation_issue=2;citation_pmid=15020491;citation_doi=10.1534/genetics.166.2.1115" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. A. Beaumont;citation_author=W. Zhang;citation_author=D. J. Balding;citation_title=Approximate Bayesian Computation in Population Genetics;citation_pages=2025-2035;citation_volume=162;citation_year=2002;citation_issue=4;citation_pmid=12524368" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=C. Becquet;citation_author=M. Przeworski;citation_title=Learning about modes of speciation by computational approaches.;citation_pages=2547-2562;citation_volume=63;citation_year=2009;citation_issue=10;citation_pmid=19228187;citation_doi=10.1111/j.1558-5646.2009.00662.x" />
<meta name="citation_reference" content="citation_journal_abbrev=AIDS Res Treat;citation_author=G. Bello;citation_author=M. A. Soares;citation_author=C. G. Schrago;citation_title=The Use of Bioinformatics for Studying HIV Evolutionary and Epidemiological History in South America.;citation_pages=154945-154945;citation_volume=2011;citation_year=2011;citation_pmid=22162803" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=T. Benaglia;citation_author=D. Chauveau;citation_author=D. R. Hunter;citation_author=D. S. Young;citation_title=mixtools: An r package for analyzing finite mixture models.;citation_pages=1-29;citation_volume=32;citation_year=2009;citation_doi=10.18637/jss.v032.i06" />
<meta name="citation_reference" content="citation_journal_title=Journal of mathematical biology;citation_journal_abbrev=J Math Biol;citation_author=M. Birkner;citation_author=J. Blath;citation_title=Computing likelihoods for coalescents with multiple collisions in the infinitely many sites model.;citation_pages=435-465;citation_volume=57;citation_year=2008;citation_issue=3;citation_pmid=18347796;citation_doi=10.1007/s00285-008-0170-6" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=M. J. Buzón;citation_author=F. M. Codoñer;citation_author=S. D. W. Frost;citation_author=C. Pou;citation_author=M. C. Puertas;citation_title=Deep molecular characterization of HIV-1 dynamics under suppressive HAART.;citation_pages=e1002314-e1002314;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22046128;citation_doi=10.1371/journal.ppat.1002314" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. L. Caicedo;citation_author=S. H. Williamson;citation_author=R. D. Hernandez;citation_author=A. Boyko;citation_author=A. Fledel-Alon;citation_title=Genome-wide patterns of nucleotide polymorphism in domesticated rice.;citation_pages=1745-1756;citation_volume=3;citation_year=2007;citation_issue=9;citation_pmid=17907810" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=E. Castro-Nallar;citation_author=M. Pérez-Losada;citation_author=G. F. Burton;citation_author=K. A. Crandall;citation_title=The evolution of HIV: inferences using phylogenetics.;citation_pages=777-792;citation_volume=62;citation_year=2012;citation_pmid=22138161;citation_doi=10.1016/j.ympev.2011.11.019" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=T. Chikata;citation_author=J. M. Carlson;citation_author=Y. Tamura;citation_author=M. A. Borghan;citation_author=T. Naruto;citation_title=Host-specific adaptation of HIV-1 subtype b in the japanese population.;citation_volume=88;citation_year=2014;citation_pmid=24522911;citation_doi=10.1128/JVI.00147-14" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. J. Drummond;citation_author=A. Rambaut;citation_author=B. Shapiro;citation_author=O. G. Pybus;citation_title=Bayesian coalescent inference of past population dynamics from molecular sequences.;citation_volume=22;citation_year=2005;citation_pmid=15703244;citation_doi=10.1093/molbev/msi103" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Ecology and Systematics;citation_journal_abbrev=Annu. Rev. Ecol. Syst.;citation_author=N. C. Ellstrand;citation_author=H. C. Prentice;citation_author=J. F. Hancock;citation_title=GENE FLOW AND INTROGRESSION FROM DOMESTICATED PLANTS INTO THEIR WILD RELATIVES;citation_pages=539-563;citation_volume=30;citation_year=1999;citation_issue=1;citation_doi=10.1146/annurev.ecolsys.30.1.539" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=L. Excoffier;citation_title=Patterns of DNA sequence diversity and genetic structure after a range expansion: lessons from the infinite-island model.;citation_pages=853-864;citation_volume=13;citation_year=2004;citation_issue=4;citation_pmid=15012760;citation_doi=10.1046/j.1365-294X.2003.02004.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. Excoffier;citation_author=I. Dupanloup;citation_author=E. Huerta-Sánchez;citation_author=V. C. Sousa;citation_author=M. Foll;citation_title=Robust demographic inference from genomic and SNP data.;citation_pages=e1003905;citation_volume=9;citation_year=2013;citation_pmid=24204310;citation_doi=10.1371/journal.pgen.1003905" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Fay;citation_author=C. I. Wu;citation_title=Hitchhiking Under Positive Darwinian Selection;citation_pages=1405-1413;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880498" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. Fellay;citation_author=K. V. Shianna;citation_author=D. Ge;citation_author=S. Colombo;citation_author=B. Ledergerber;citation_title=A whole-genome association study of major determinants for host control of HIV-1.;citation_volume=317;citation_year=2007;citation_pmid=17641165;citation_doi=10.1126/science.1143767" />
<meta name="citation_reference" content="citation_journal_title=AIDS;citation_author=Y. Feng;citation_author=X. He;citation_author=J. H. Hsi;citation_author=F. Li;citation_author=X. Li;citation_title=The rapidly expanding crf01_ae epidemic in China is driven by multiple lineages of HIV-1 viruses introduced in the 1990s.;citation_pages=1793-1802;citation_volume=27;citation_year=2013;citation_pmid=23807275;citation_doi=10.1097/QAD.0b013e328360db2d" />
<meta name="citation_reference" content="citation_journal_title=AIDS Res. Hum. Retroviruses;citation_author=Y. Feng;citation_author=H. Wei;citation_author=J. Hsi;citation_author=H. Xing;citation_author=X. He;citation_title=Identification of a novel HIV type 1 circulating recombinant form (crf65_cpx) composed of crf01_ae and subtypes b and c in western Yunnan, China.;citation_pages=598-602;citation_volume=30;citation_year=2014;citation_pmid=24279591;citation_doi=10.1089/aid.2013.0233" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Ferretti;citation_author=M. Perez-Enciso;citation_author=S. Ramos-Onsins;citation_title=Optimal neutrality tests based on the frequency spectrum.;citation_volume=186;citation_year=2010;citation_pmid=20610408;citation_doi=10.1534/genetics.110.118570" />
<meta name="citation_reference" content="citation_title=The genetical theory of natural selection;citation_author=R. A. Fisher;citation_year=1930" />
<meta name="citation_reference" content="citation_journal_title=Public Health Rev.;citation_author=A. Flahault;citation_author=P. Zylberman;citation_title=Influenza pandemics: past, present and future challenges.;citation_pages=319-340;citation_volume=32;citation_year=2010" />
<meta name="citation_reference" content="citation_title=Mol. Ecol. Resour.;citation_author=M. Foll;citation_author=H. Shim;citation_author=J. D. Jensen;citation_title=Wfabc: a Wright–Fisher abc-based approach for inferring effective population sizes and selection coefficients from time-sampled data.;citation_pages=87-98;citation_volume=15;citation_year=2014;citation_pmid=24834845" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. X. Fu;citation_author=W. H. Li;citation_title=Statistical tests of neutrality of mutations.;citation_pages=693-709;citation_volume=133;citation_year=1993;citation_issue=3;citation_pmid=8454210" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=R. J. Garten;citation_author=C. T. Davis;citation_author=C. A. Russell;citation_author=B. Shu;citation_author=S. Lindstrom;citation_title=Antigenic and genetic characteristics of swine-origin 2009 a(h1n1) influenza viruses circulating in humans.;citation_volume=325;citation_year=2009;citation_pmid=19465683;citation_doi=10.1126/science.1176225" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. Gaschen;citation_author=J. Taylor;citation_author=K. Yusim;citation_author=B. Foley;citation_author=F. Gao;citation_title=Diversity Considerations in HIV-1 Vaccine Selection;citation_pages=2354-2360;citation_volume=296;citation_year=2002;citation_issue=5577;citation_pmid=12089434;citation_doi=10.1126/science.1070441" />
<meta name="citation_reference" content="citation_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gravel;citation_author=B. M. Henn;citation_author=R. N. Gutenkunst;citation_author=A. R. Indap;citation_author=G. T. Marth;citation_title=Demographic history and rare allele sharing among human populations.;citation_volume=108;citation_year=2011;citation_pmid=21730125;citation_doi=10.1073/pnas.1019276108" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. E. Green;citation_author=J. Krause;citation_author=A. W. Briggs;citation_author=T. Maricic;citation_author=U. Stenzel;citation_title=A Draft Sequence of the Neandertal Genome;citation_pages=710-722;citation_volume=328;citation_year=2010;citation_issue=5979;citation_pmid=20448178;citation_doi=10.1126/science.1188021;citation_georef=2010072445" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. N. Gutenkunst;citation_author=R. D. Hernandez;citation_author=S. H. Williamson;citation_author=C. D. Bustamante;citation_title=Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.;citation_pages=e1000695-e1000695;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19851460;citation_doi=10.1371/journal.pgen.1000695" />
<meta name="citation_reference" content="citation_journal_title=AIDS (London, England);citation_journal_abbrev=AIDS;citation_author=J. Hamelaar;citation_author=E. Gouws;citation_author=P. D. Ghys;citation_author=S. Osmanov;citation_title=Global trends in molecular epidemiology of HIV-1 during 2000-2007.;citation_pages=679-689;citation_volume=25;citation_year=2011;citation_issue=5;citation_pmid=21297424;citation_doi=10.1097/QAD.0b013e328342ff93" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. Han;citation_author=M. An;citation_author=B. Zhao;citation_author=S. Duan;citation_author=S. Yang;citation_title=High prevalence of HIV-1 intersubtype b /c recombinants among injecting drug users in Dehong, China.;citation_pages=e65337;citation_volume=8;citation_year=2013;citation_pmid=23741489;citation_doi=10.1371/journal.pone.0065337" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. C. Harpending;citation_author=M. A. Batzer;citation_author=M. Gurven;citation_author=L. B. Jorde;citation_author=A. R. Rogers;citation_title=Genetic traces of ancient demography;citation_pages=1961-1967;citation_volume=95;citation_year=1998;citation_issue=4;citation_pmid=9465125;citation_doi=10.1073/pnas.95.4.1961" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. Hey;citation_title=Isolation with migration models for more than two populations.;citation_volume=27;citation_year=2010;citation_pmid=19955477;citation_doi=10.1093/molbev/msp296" />
<meta name="citation_reference" content="citation_journal_title=AIDS Res. Hum. Retroviruses;citation_author=J. Hsi;citation_author=H. Wei;citation_author=H. Xing;citation_author=Y. Feng;citation_author=X. He;citation_title=Genome sequence of a novel HIV-1 circulating recombinant form (crf64_bc) identified from Yunnan, China.;citation_pages=389-393;citation_volume=30;citation_year=2014;citation_pmid=24205972;citation_doi=10.1089/aid.2013.0234" />
<meta name="citation_reference" content="citation_journal_title=Journal of the Formosan Medical Association = Taiwan yi zhi;citation_journal_abbrev=J Formos Med Assoc;citation_author=Y.-C. Hsieh;citation_author=T.-Z. Wu;citation_author=D.-P. Liu;citation_author=P.-L. Shao;citation_author=L.-Y. Chang;citation_title=Influenza pandemics: past, present and future.;citation_pages=1-6;citation_volume=105;citation_year=2006;citation_issue=1;citation_pmid=16440064;citation_doi=10.1016/S0929-6646(09)60102-9" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=R. R. Hudson;citation_title=Generating samples under a Wright-Fisher neutral model of  genetic variation;citation_pages=337-338;citation_volume=18;citation_year=2002;citation_issue=2;citation_pmid=11847089;citation_doi=10.1093/bioinformatics/18.2.337" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=T. Jombart;citation_author=S. Devillard;citation_author=F. Balloux;citation_title=Discriminant analysis of principal components: a new method for the analysis of genetically structured populations.;citation_pages=94-94;citation_volume=11;citation_year=2010;citation_pmid=20950446;citation_doi=10.1186/1471-2156-11-94" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=E. K. Karlsson;citation_author=D. P. Kwiatkowski;citation_author=P. C. Sabeti;citation_title=Natural selection and infectious disease in human populations.;citation_pages=379-393;citation_volume=15;citation_year=2014;citation_pmid=24776769;citation_doi=10.1038/nrg3734" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. F. Keele;citation_author=E. E. Giorgi;citation_author=J. F. Salazar-Gonzalez;citation_author=J. M. Decker;citation_author=K. T. Pham;citation_title=Identification and characterization of transmitted and early founder virus envelopes in primary HIV-1 infection.;citation_volume=105;citation_year=2008;citation_pmid=18490657;citation_doi=10.1073/pnas.0802203105" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Kimura;citation_title=THE NUMBER OF HETEROZYGOUS NUCLEOTIDE SITES MAINTAINED IN A FINITE POPULATION DUE TO STEADY FLUX OF MUTATIONS;citation_pages=893-903;citation_volume=61;citation_year=1969;citation_issue=4;citation_pmid=5364968" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. Korber;citation_author=M. Muldoon;citation_author=J. Theiler;citation_author=F. Gao;citation_author=R. Gupta;citation_title=Timing the Ancestor of the HIV-1 Pandemic Strains;citation_pages=1789-1796;citation_volume=288;citation_year=2000;citation_issue=5472;citation_pmid=10846155;citation_doi=10.1126/science.288.5472.1789" />
<meta name="citation_reference" content="citation_journal_title=British Medical Bulletin;citation_journal_abbrev=Br Med Bull;citation_author=B. Korber;citation_author=B. Gaschen;citation_author=K. Yusim;citation_author=R. Thakallapally;citation_author=C. Kesmir;citation_title=Evolutionary and immunological implications of contemporary HIV-1 variation;citation_pages=19-42;citation_volume=58;citation_year=2001;citation_issue=1;citation_pmid=11714622;citation_doi=10.1093/bmb/58.1.19" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=R. Lecis;citation_author=M. Pierpaoli;citation_author=Z. Biro;citation_author=L. Szemethy;citation_author=B. Ragni;citation_title=Bayesian analyses of admixture in wild and domestic cats (Felis silvestris) using linked microsatellite loci.;citation_pages=119-131;citation_volume=15;citation_year=2006;citation_issue=1;citation_pmid=16367835;citation_doi=10.1111/j.1365-294X.2005.02812.x" />
<meta name="citation_reference" content="citation_title=AIDS Res. Hum. Retroviruses;citation_author=X. Li;citation_author=C. Ning;citation_author=X. He;citation_author=Y. Yang;citation_author=H. Xing;citation_title=Near full-length genome sequence of a novel HIV type 1 second-generation recombinant form (crf01_ae/crf07_bc) identified among men who have sex with men in jilin, china.;citation_pages=1604-1608;citation_volume=29;citation_year=2013;citation_pmid=23809010;citation_doi=10.1089/aid.2013.0116" />
<meta name="citation_reference" content="citation_journal_title=Viruses;citation_author=F. Liégeois;citation_author=F. Schmidt;citation_author=V. Boué;citation_author=C. Butel;citation_author=F. Mouacha;citation_title=Full-length genome analyses of two new simian immunodeficiency virus (SIV) strains from mustached monkeys (c. cephus) in gabon illustrate a complex evolutionary history among the sivmus/mon/gsn lineage.;citation_pages=2880-2898;citation_volume=6;citation_year=2014;citation_pmid=25054885;citation_doi=10.3390/v6072880" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Lohse;citation_author=R. Harrison;citation_author=N. H. Barton;citation_title=A general method for calculating likelihoods under the coalescent process.;citation_volume=189;citation_year=2011;citation_pmid=21900266;citation_doi=10.1534/genetics.111.129569" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. Lu;citation_author=M. Jia;citation_author=Y. Ma;citation_author=L. Yang;citation_author=Z. Chen;citation_title=The changing face of HIV in China.;citation_pages=609-611;citation_volume=455;citation_year=2008;citation_issue=7213;citation_pmid=18833270;citation_doi=10.1038/455609a" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Ma;citation_author=C. I. Amos;citation_title=Principal components analysis of population admixture.;citation_pages=e40115-e40115;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22808102;citation_doi=10.1371/journal.pone.0040115" />
<meta name="citation_reference" content="citation_journal_title=Biometrika;citation_journal_abbrev=Biometrika;citation_author=X.-L. Meng;citation_author=D. B. Rubin;citation_title=Maximum likelihood estimation via the ECM algorithm: A general framework;citation_pages=267-278;citation_volume=80;citation_year=1993;citation_issue=2;citation_doi=10.1093/biomet/80.2.267" />
<meta name="citation_reference" content="citation_title=PLoS pathogens;citation_author=P. Mohammadi;citation_author=S. Desfarges;citation_author=I. Bartha;citation_author=B. Joos;citation_author=N. Zangger;citation_title=24 hours in the life of HIV-1 in a t cell line.;citation_pages=e1003161;citation_volume=9;citation_year=2013;citation_pmid=23382686;citation_doi=10.1371/journal.ppat.1003161" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=L. Naduvilezhath;citation_author=L. E. Rose;citation_author=D. Metzler;citation_title=Jaatha: a fast composite-likelihood approach to estimate demographic parameters.;citation_pages=2709-2723;citation_volume=20;citation_year=2011;citation_issue=13;citation_pmid=21645157;citation_doi=10.1111/j.1365-294X.2011.05131.x" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; genetic systems;citation_journal_abbrev=Genes Genet Syst;citation_author=N. Nawa;citation_author=F. Tajima;citation_title=Simple method for analyzing the pattern of DNA polymorphism and its application to SNP data of human.;citation_pages=353-360;citation_volume=83;citation_year=2008;citation_issue=4;citation_pmid=18931461;citation_doi=10.1266/ggs.83.353" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. A. Neher;citation_author=O. Hallatschek;citation_title=Genealogies of rapidly adapting populations.;citation_volume=110;citation_year=2013;citation_pmid=23269838;citation_doi=10.1073/pnas.1213113110" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Nielsen;citation_author=J. Wakeley;citation_title=Distinguishing Migration From Isolation: A Markov Chain Monte Carlo Approach;citation_pages=885-896;citation_volume=158;citation_year=2001;citation_issue=2;citation_pmid=11404349" />
<meta name="citation_reference" content="citation_journal_title=Journal of mathematical biology;citation_journal_abbrev=J Math Biol;citation_author=M. Notohara;citation_title=The coalescent and the genealogical process in geographically structured population.;citation_pages=59-75;citation_volume=29;citation_year=1990;citation_issue=1;citation_pmid=2277236" />
<meta name="citation_reference" content="citation_journal_title=AIDS (London, England);citation_journal_abbrev=AIDS;citation_author=W. Pang;citation_author=C. Zhang;citation_author=L. Duo;citation_author=Y.-H. Zhou;citation_author=Z.-H. Yao;citation_title=Extensive and complex HIV-1 recombination between B', C and CRF01_AE among IDUs in south-east Asia.;citation_pages=1121-1129;citation_volume=26;citation_year=2012;citation_issue=9;citation_pmid=22333750;citation_doi=10.1097/QAD.0b013e3283522c97" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Patterson;citation_author=P. Moorjani;citation_author=Y. Luo;citation_author=S. Mallick;citation_author=N. Rohland;citation_title=Ancient admixture in human history.;citation_volume=192;citation_year=2012;citation_pmid=22960212;citation_doi=10.1534/genetics.112.145037" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=K. Pelak;citation_author=A. C. Need;citation_author=J. Fellay;citation_author=K. V. Shianna;citation_author=S. Feng;citation_title=Copy number variation of KIR genes influences HIV-1 control.;citation_pages=e1001208-e1001208;citation_volume=9;citation_year=2011;citation_issue=11;citation_pmid=22140359;citation_doi=10.1371/journal.pbio.1001208" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. E. Pool;citation_author=R. B. Corbett-Detig;citation_author=R. P. Sugino;citation_author=K. A. Stevens;citation_author=C. M. Cardeno;citation_title=Population genomics of sub-saharan Drosophila melanogaster: African diversity and non-african admixture.;citation_pages=e1003080;citation_volume=8;citation_year=2012;citation_pmid=23284287;citation_doi=10.1371/journal.pgen.1003080" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Reich;citation_author=N. Patterson;citation_author=D. Campbell;citation_author=A. Tandon;citation_author=S. Mazieres;citation_title=Reconstructing Native American population history.;citation_pages=370-374;citation_volume=488;citation_year=2012;citation_issue=7411;citation_pmid=22801491;citation_doi=10.1038/nature11258" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=N. Renzette;citation_author=B. Bhattacharjee;citation_author=J. D. Jensen;citation_author=L. Gibson;citation_author=T. F. Kowalik;citation_title=Extensive genome-wide variability of human cytomegalovirus in congenitally infected infants.;citation_pages=e1001344-e1001344;citation_volume=7;citation_year=2011;citation_issue=5;citation_pmid=21625576;citation_doi=10.1371/journal.ppat.1001344" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. Renzette;citation_author=L. Gibson;citation_author=B. Bhattacharjee;citation_author=D. Fisher;citation_author=M. R. Schleiss;citation_title=Rapid intrahost evolution of human cytomegalovirus is shaped by demography and positive selection.;citation_pages=e1003735;citation_volume=9;citation_year=2013;citation_pmid=24086142;citation_doi=10.1371/journal.pgen.1003735" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Virol.;citation_author=N. Renzette;citation_author=L. Gibson;citation_author=J. D. Jensen;citation_author=T. F. Kowalik;citation_title=Human cytomegalovirus intrahost evolution–a new avenue for understanding and controlling herpesvirus infections.;citation_pages=109-115;citation_volume=8;citation_year=2014;citation_pmid=25154343;citation_doi=10.1016/j.coviro.2014.08.001" />
<meta name="citation_reference" content="citation_title=Proc. Natl. Acad. Sci. USA;citation_author=N. Renzette;citation_author=C. Pokalyuk;citation_author=L. Gibson;citation_author=B. Bhattacharjee;citation_author=M. R. Schleiss;citation_title=Limits and patterns of cytomegalovirus genomic diversity in humans.;citation_volume=112;citation_year=2015;citation_pmid=26150505;citation_doi=10.1073/pnas.1501880112" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. Robertson;citation_author=J. Anderson;citation_author=J. Bradac;citation_author=J. Carr;citation_author=B. Foley;citation_title=HIV-1 nomenclature proposal.;citation_pages=55-56;citation_volume=288;citation_year=2000;citation_pmid=10766634;citation_doi=10.1126/science.288.5463.55c" />
<meta name="citation_reference" content="citation_title=Modern Developments in Theoretical Population Genetics: The legacy of Gustave Malécot;citation_title=The relationship between coalescence times and population divergence times;citation_author=N. A. Rosenberg;citation_author=M. W. Feldman;citation_pages=130-164;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Sankararaman;citation_author=N. Patterson;citation_author=H. Li;citation_author=S. Pääbo;citation_author=D. Reich;citation_title=The Date of Interbreeding between Neandertals and Modern Humans.;citation_pages=e1002947-e1002947;citation_volume=8;citation_year=2012;citation_issue=10;citation_pmid=23055938;citation_doi=10.1371/journal.pgen.1002947" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=T. E. Schlub;citation_author=A. J. Grimm;citation_author=R. P. Smyth;citation_author=D. Cromer;citation_author=A. Chopra;citation_title=Fifteen to twenty percent of HIV substitution mutations are associated with recombination.;citation_volume=88;citation_year=2014;citation_pmid=24453357;citation_doi=10.1128/JVI.03136-13" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Schneider;citation_author=L. Excoffier;citation_title=Estimation of Past Demographic Parameters From the Distribution of Pairwise Differences When the Mutation Rates Vary Among Sites: Application to Human Mitochondrial DNA;citation_pages=1079-1089;citation_volume=152;citation_year=1999;citation_issue=3;citation_pmid=10388826" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Pathog;citation_author=G. Schnell;citation_author=S. Joseph;citation_author=S. Spudich;citation_author=R. W. Price;citation_author=R. Swanstrom;citation_title=HIV-1 replication in the central nervous system occurs in two distinct cell types.;citation_pages=e1002286-e1002286;citation_volume=7;citation_year=2011;citation_issue=10;citation_pmid=22007152;citation_doi=10.1371/journal.ppat.1002286" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A.-K. Schultz;citation_author=M. Zhang;citation_author=I. Bulla;citation_author=T. Leitner;citation_author=B. Korber;citation_title=jphmm: improving the reliability of recombination prediction in HIV-1.;citation_volume=37;citation_year=2009;citation_pmid=19443440;citation_doi=10.1093/nar/gkp371" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=O. Seehausen;citation_title=Patterns in fish radiation are compatible with Pleistocene desiccation of Lake Victoria and 14 600 year history for its cichlid species flock;citation_pages=491-497;citation_volume=269;citation_year=2002;citation_issue=1490;citation_pmid=11886641;citation_doi=10.1098/rspb.2001.1906;citation_georef=2013069896" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=O. Seehausen;citation_title=Hybridization and adaptive radiation.;citation_pages=198-207;citation_volume=19;citation_year=2004;citation_issue=4;citation_pmid=16701254;citation_doi=10.1016/j.tree.2004.01.003" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=P. M. Sharp;citation_author=B. H. Hahn;citation_title=The evolution of HIV-1 and the origin of AIDS;citation_pages=2487-2494;citation_volume=365;citation_year=2010;citation_issue=1552;citation_pmid=20643738;citation_doi=10.1098/rstb.2010.0031" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. L. Simonsen;citation_author=G. A. Churchill;citation_author=C. F. Aquadro;citation_title=Properties of statistical tests of neutrality for DNA polymorphism data.;citation_pages=413-429;citation_volume=141;citation_year=1995;citation_issue=1;citation_pmid=8536987" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Slatkin;citation_title=Gene Genealogies Within Mutant Allelic Classes;citation_pages=579-587;citation_volume=143;citation_year=1996;citation_issue=1;citation_pmid=8722806" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Slatkin;citation_author=R. R. Hudson;citation_title=Pairwise comparisons of mitochondrial DNA sequences in stable and exponentially growing populations.;citation_pages=555-562;citation_volume=129;citation_year=1991;citation_issue=2;citation_pmid=1743491" />
<meta name="citation_reference" content="citation_journal_abbrev=Retrovirology;citation_author=J. Snoeck;citation_author=J. Fellay;citation_author=I. Bartha;citation_author=D. C. Douek;citation_author=A. Telenti;citation_title=Mapping of positive selection sites in the HIV-1 genome in the context of RNA and protein structural constraints.;citation_pages=87-87;citation_volume=8;citation_year=2011;citation_pmid=22044801;citation_doi=10.1186/1742-4690-8-87" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=V. Sousa;citation_author=J. Hey;citation_title=Understanding the origin of species with genome-scale data: modelling gene flow.;citation_pages=404-414;citation_volume=14;citation_year=2013;citation_pmid=23657479;citation_doi=10.1038/nrg3446" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=V. C. Sousa;citation_author=A. Grelaud;citation_author=J. Hey;citation_title=On the nonidentifiability of migration time estimates in isolation with migration models.;citation_pages=3956-3962;citation_volume=20;citation_year=2011;citation_issue=19;citation_pmid=22039614;citation_doi=10.1111/j.1365-294X.2011.05247.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=J. L. Strasburg;citation_author=L. H. Rieseberg;citation_title=Interpreting the estimated timing of migration events between hybridizing species.;citation_pages=2353-2366;citation_volume=20;citation_year=2011;citation_issue=11;citation_pmid=21375636;citation_doi=10.1111/j.1365-294X.2011.05048.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=K. Strimmer;citation_author=O. G. Pybus;citation_title=Exploring the Demographic History of DNA Sequences Using the Generalized Skyline Plot;citation_pages=2298-2305;citation_volume=18;citation_year=2001;citation_issue=12;citation_pmid=11719579;citation_doi=10.1093/oxfordjournals.molbev.a003776" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=L. Su;citation_author=M. Graf;citation_author=Y. Zhang;citation_author=H. von Briesen;citation_author=H. Xing;citation_title=Characterization of a Virtually Full-Length Human Immunodeficiency Virus Type 1 Genome of a Prevalent Intersubtype (C/B') Recombinant Strain in China;citation_pages=11367-11376;citation_volume=74;citation_year=2000;citation_issue=23;citation_pmid=11070037;citation_doi=10.1128/JVI.74.23.11367-11376.2000" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Ecology and Systematics;citation_journal_abbrev=Annu. Rev. Ecol. Syst.;citation_author=N. Takahata;citation_title=A Genetic Perspective on the Origin and History of Humans;citation_pages=343-372;citation_volume=26;citation_year=1995;citation_issue=1;citation_doi=10.1146/annurev.es.26.110195.002015" />
<meta name="citation_reference" content="citation_journal_title=Vaccine;citation_author=Y. Takebe;citation_author=H. Liao;citation_author=S. Hase;citation_author=R. Uenishi;citation_author=Y. Li;citation_title=Reconstructing the epidemic history of HIV-1 circulating recombinant forms crf07_bc and crf08_bc in east asia: the relevance of genetic diversity and phylodynamics for vaccine strategies.;citation_pages=B39-B44;citation_volume=28;citation_year=2010;citation_pmid=20510742;citation_doi=10.1016/j.vaccine.2009.07.101" />
<meta name="citation_reference" content="citation_journal_title=The Lancet infectious diseases;citation_journal_abbrev=Lancet Infect Dis;citation_author=D. M. Tebit;citation_author=E. J. Arts;citation_title=Tracking a century of global expansion and evolution of HIV to drive understanding and to combat disease.;citation_pages=45-56;citation_volume=11;citation_year=2011;citation_issue=1;citation_pmid=21126914;citation_doi=10.1016/S1473-3099(10)70186-9" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=A. Tellier;citation_author=C. Lemaire;citation_title=Coalescence 2.0: a multiple branching of recent theoretical developments and their applications.;citation_pages=2637-2652;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12755" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. HIV AIDS;citation_author=S. Vuilleumier;citation_author=S. Bonhoeffer;citation_title=Contribution of recombination to the evolutionary history of HIV.;citation_pages=77-127;citation_volume=10;citation_year=2015;citation_pmid=25612323;citation_doi=10.1097/COH.0000000000000140" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=J. Wakeley;citation_title=Pairwise differences under a general model of population subdivision.;citation_pages=81-89;citation_volume=75;citation_year=1996;citation_doi=10.1007/BF02931753" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Wakeley;citation_title=Nonequilibrium Migration in Human History;citation_pages=1863-1871;citation_volume=153;citation_year=1999;citation_issue=4;citation_pmid=10581291" />
<meta name="citation_reference" content="citation_journal_title=AIDS Res. Hum. Retroviruses;citation_author=N. Wang;citation_author=H. Wei;citation_author=R. Xiong;citation_author=H. Zhang;citation_author=J. H. Hsi;citation_title=Near full-length genome characterization of a new crf01_ae/crf08_bc recombinant transmitted between a heterosexual couple in Guangxi, China.;citation_pages=484-488;citation_volume=30;citation_year=2014;citation_pmid=24164454;citation_doi=10.1089/aid.2013.0230" />
<meta name="citation_reference" content="citation_journal_title=AIDS Res. Hum. Retroviruses;citation_author=H. Wei;citation_author=J. His;citation_author=Y. Feng;citation_author=H. Xing;citation_author=X. He;citation_title=Identification of a novel HIV-1 circulating recombinant form (crf62_bc) in western Yunnan of China.;citation_pages=380-383;citation_volume=30;citation_year=2014;citation_pmid=24164474;citation_doi=10.1089/aid.2013.0235" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=H. M. Wilkinson-Herbots;citation_title=The distribution of the coalescence time and the number of pairwise nucleotide differences in a model of population divergence or speciation with an initial period of gene flow.;citation_pages=92-108;citation_volume=82;citation_year=2012;citation_issue=2;citation_pmid=22687581;citation_doi=10.1016/j.tpb.2012.05.003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Wright;citation_title=EVOLUTION IN MENDELIAN POPULATIONS;citation_pages=97-159;citation_volume=16;citation_year=1931;citation_issue=2;citation_pmid=17246615" />
<meta name="citation_reference" content="citation_journal_title=AIDS (London, England);citation_journal_abbrev=AIDS;citation_author=R. Yang;citation_author=X. Xia;citation_author=S. Kusagawa;citation_author=C. Zhang;citation_author=K. Ben;citation_title=On-going generation of multiple forms of HIV-1 intersubtype recombinants in the Yunnan Province of China.;citation_pages=1401-1407;citation_volume=16;citation_year=2002;citation_issue=10;citation_pmid=12131217;citation_doi=10.1097/00002030-200207050-00012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Zeng;citation_author=Y.-X. Fu;citation_author=S. Shi;citation_author=C.-I. Wu;citation_title=Statistical tests for detecting positive selection by utilizing high-frequency variants.;citation_volume=174;citation_year=2006;citation_pmid=16951063;citation_doi=10.1534/genetics.106.061432" />
<meta name="citation_reference" content="citation_journal_title=Lancet Infect. Dis.;citation_author=L. Zhang;citation_author=E. P. F. Chow;citation_author=J. Jing;citation_author=X. Zhuang;citation_author=X. Li;citation_title=HIV prevalence in China: integration of surveillance data and a systematic review.;citation_pages=955-963;citation_volume=13;citation_year=2013;citation_pmid=24107261;citation_doi=10.1016/S1473-3099(13)70245-7" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
<meta name="twitter:description" content="Ease of worldwide travel provides increased opportunities for organisms not only to colonize new environments but also to encounter related but diverged populations. Such events of reconnection and secondary contact of previously isolated populations are widely observed at different time scales. For example, during the quaternary glaciation, sea water level fluctuations caused temporal isolation of populations, often to be followed by secondary contact. At shorter time scales, population isolation and reconnection of viruses are commonly observed, and such events are often associated with epidemics and pandemics. Here, using coalescent theory and simulations, we describe the temporal impact of population reconnection after isolation on nucleotide differences and the site frequency spectrum, as well as common summary statistics of DNA variation. We identify robust genomic signatures of population reconnection after isolation. We utilize our development to infer the recent evolutionary history of human immunodeficiency virus 1 (HIV-1) in Asia and South America, successfully retrieving the successive HIV subtype colonization events in these regions. Our analysis reveals that divergent HIV-1 subtype populations are currently admixing in these regions, suggesting that HIV-1 may be undergoing a process of homogenization, contrary to popular belief." />
<meta name="og-title" property="og:title" content="The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/1/107" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Ease of worldwide travel provides increased opportunities for organisms not only to colonize new environments but also to encounter related but diverged populations. Such events of reconnection and secondary contact of previously isolated populations are widely observed at different time scales. For example, during the quaternary glaciation, sea water level fluctuations caused temporal isolation of populations, often to be followed by secondary contact. At shorter time scales, population isolation and reconnection of viruses are commonly observed, and such events are often associated with epidemics and pandemics. Here, using coalescent theory and simulations, we describe the temporal impact of population reconnection after isolation on nucleotide differences and the site frequency spectrum, as well as common summary statistics of DNA variation. We identify robust genomic signatures of population reconnection after isolation. We utilize our development to infer the recent evolutionary history of human immunodeficiency virus 1 (HIV-1) in Asia and South America, successfully retrieving the successive HIV subtype colonization events in these regions. Our analysis reveals that divergent HIV-1 subtype populations are currently admixing in these regions, suggesting that HIV-1 may be undergoing a process of homogenization, contrary to popular belief." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1.cover-source.jpg" />
    <title>The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__-8wSh-loC8aZQjVpkiWXeUnH4axtmjVhZGuSdz8kVhA__4doWch-QYHBi-iQ7qgOafMkMn_zUNZJNYpe8oI0O6J8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Jn4CAauWJZ0iVEwHBYQKZzRD6vPcxJtkMz6xCnqrirQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460228","apath":"\/ggg\/6\/1\/107.atom","pisa":"ggg;6\/1\/107","ac":{"\/ggg\/6\/1\/107.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/1\/107.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/1\/107"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/1\/107"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/1\/107.figures-only":true,"\/content\/6\/1\/107":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yuuwmCUTV8Q91qwvhU0zAfs0CyYNYuHNqnzl+lTbwyJObCeHXEN8\\\\\\\/pzH0FGCMzXk688cTBdDXmj9\\\\\\\/POfyeZD4EL7L0APcIrSWAGdHjDRFFF2O5ZNGhSpB3+xRv24CKBUgZFXYAtJhLZ4mDhJv\\\\\\\/FmDa+2MEfo6PbfBeD0iQscqmrc2kbOnfif40IaRAaErZ6Epl4omOBMd\\\\\\\/sdQLbO8Ggn8mjHxP6LGzGEJ1A3aJU4B7WXHSiD1UZ5YICZ3NRtqWzYI5gLn2n1mAfS+QQJ0BbM43vZGeXL97Ty\\\\\\\/s8Hl1COATNrxpYhg9gBlcTDN4RAAGfbn3ndIQUOSYnoIR9yVH49SeC87uWG+trmp+LJbxfUOw36w5940WHiBW8hquPon+1o66MoWK3d9oZRrnXjAeCMFarqh64llWY5z5OovP0sqo5yvGkUakbFw77daWyiizl9vQeVKnKdtI6edDM0irgr91SlspZfvHMHj++G+JDLhRvx66OhGW\\\\\\\/XuVRr3Keqkv55wMo\\\\\\\/5PA\\\\\\\/tbmC\\\\\\\/sQygXh+kO9eetUe7dHSnR4syNpAJ2UXFaSDwwgM1k6koe3X7wGJy6Ak594ChWjciz\\\\\\\/vy+0Y3VwXDwhA5qouDCeZ3AL7iKOjc6TSakZJtdwMl\\\\\\\/FfbU9DEppcy3BHVC+okZ0Bd3FKSsyAEd9xANW+JHszEF32ywRywCWVc2xU8EVb3F\\\\\\\/x9xLWlh3BiFIOYBnrE0xoVDHvIds8IrbzaZYpHfiWG71bHsSo5E7k2AfNRUVjw9eGVbwGM0E1d91EBcAoEbdamwHSvRt4QO2XMcA5hxnS2YaSyTkq9nQAhngS4MmRVmylK47IHaLqP0N4qrx74Jm0E65A6JjLVD1LolYt1Yc7HUFG38hH4CxjWLtQEW+ZPb4aRYybb2M0UZViRDS8HjoXTmqJvC7LPZE75Xwx248zbuQNQlhbGVcqxVRXrj31Z7UnGUOucFN7R4Pq7MfLInBx4mBvy4krAJ3U444bU4rPgxs35Fu0c0oaGmqgPXZ3iZqtRhNJXCGixI9SqBQX75A+SgjqgU3QBnthmEg5laK8C\\\\\\\/Y3wIm4XCVYh9QVuo+4FQSWoLypwL28IgV7xO9GbeMDlUBaoOnh08YpD3M7evTErpqyMYDsoBgXR0ELArZ8Gc4bec\\\\\\\/+HANQgX0twADbEdiV0BgtUjDBjJt5r8pWbFWyzkoz4TQ9cpVgaCTQ8sfHaj4IkJ4\\\\\\\/zssXohXEehrOYtyaIiM6UhrI8StkfgrfW\\\\\\\/jQ9ccStMvC5SGp0uRK3rXG8\\\\\\\/QZAFS8\\\u0022,\\\u0022iv\\\u0022:\\\u0022ud\\\\\\\/s4ISnjk\\\\\\\/\\\\\\\/gxwnoONX2A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002228b790e39d743626ac5940416f8c0840581b2cbf99699991a065dbc0cd4b0bbb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rMMgfvfVjKxszYqzC\\\\\\\/s3m0W58j0Ac\\\\\\\/nvUAyYh3TcPDw5FHW2\\\\\\\/JEKOkNFOLQ+BRSV4c4\\\\\\\/CIYGIaoUIFEJIbBmEme7YAfDgZKI1sfgD05r5rpSOtFHNmk6HKH2rorwYJ0c2aLQDxbxxBdk\\\\\\\/HwMA0PTcFN2tdjvxqCxVBGeKTZeRVW7aDWYgTF0BrnnFaXRvhqujzes2MN5fJvP9+W9Ls+zTB42K63IRxH3F73ZPYhBp6MA+mYakJ8tfQc6Sllm\\\\\\\/W87xnDeuKyX0HbULk9lKbVDTJTI24T57XaUd3Tx5uxB+2Dok9oMW8WiuR5Cv3fukSIaBybU3oUh3n9gWhv54Ehe0kgUVnrqB3VRd8t4OiWe\\\\\\\/hZzbXJxS7uUAXx58EJyoeWTLC10nkHgtCmHhwNRy2wRpP3yDd5iaGsDgddscKb2f\\\\\\\/aLQOWe2nOk+OdTw3hU6Rvtru2wAkNWTF5ZyoMqwn3iy4PCYcW8bfsx5FSkFFNYXfFetclNsi58EvvSSyr+iTSbBA314v9hpSyuyTxVlt4RDSf0nd6MrV8KUgmi2oDFtf+zb8IWEE3ONoBlxqf1rBaVqgdudVpZxik4Et1pihPm\\\\\\\/hbOcO6ZPRWx2FyJrm\\\\\\\/ya5cCZOULrgQh277O+FJSv2bIM3pBDnwFMTcfL7HMp3rtJ8IFRa3jdXq2r6f5elFhcBW43MUvizTl1YQtjzK5qUBgP6jRKdFc0ppdz6\\\\\\\/nxOtp96frYkgnfxNy3yuEFFHVbJ8iyuXIFQKKzReYCcUMSLrGQgHCoCQUKaewOMIjO7UmwxigiAsoSeXj31gK4dghHHyZI4F8xrnmH142ZYLFy2Iuw9jO64vZ7iwmRHS\\\\\\\/MgtLZyCNGEHGvQZE8eORty4LQfg8dd4Rs0x8j6IF3TSwD9yVPsrKbdISKo7P98fabjUa8QCvuoVrfPfhc8AHYogdXeRtz3\\\\\\\/zNwQ74Ey5NdKzcyfsUlu+7JNOCOygT3bEF5eGp\\\\\\\/va1F+VegEVahQT\\\\\\\/BatscY308k2CM2TviRsPeDobY\\\\\\\/znalQ3gl9x3Xn9BWFJiQ9wVEJUd2BDGJV0nLpwsfxXoOqFJc7QoM4qRUSiSMC6KVrMcIEFujhxt5++FeYp4m1LuFaPPrqAbUY0vWEYWFjEzmNHAJxQDqk5ownWpBPPTnR0AJu2sk338rsEMl\\\\\\\/XuG3yeWpy7OVhxAvWYcjySanep846R\\\\\\\/a5M\\\\\\\/0qwFQ4n9tP3aV1rNgm43J9uGHcaNQ8AShnM9okuTQjWpkG1TOWbXprWb8KbmzLmZny4RxmiIMReRi69n5rmJzoVRcP1OROAibxn2mDknG6+cDxP1zoW6FGviu+wAmTUNUwndp1YcKAcj33qe3Rhty0axHw6vlBP9dAzRdVB9DCOu9Td\\\\\\\/ldU4vDze0sWVmAMAMo08Zee4pu0Rrk6fedqZ3QQvPPu+MKKmszeHsPpkFkaiAv+xq5My72YpTSQetp50hUQcrDt\\\\\\\/0SPjglG57iM+iD3JcNfndQ2mXbSj0bvb6n6OCPe0G8YQ5DEHKWyfFXGY3VDdIWJNQCYRkqSjDjRpZSoanDdTeQHP7gsGIlnYuQKw1IrlAkwuFvsxandD78ixCzYcQz6cvzRmadeCiJV5eIrXJgxWJe6GMUMzTKVQdVyhG3etkU41xBNyX4y4\\\\\\\/\\\\\\\/I3mTZn0rTo6hSNjyNMNgyOgd3+MqblgvHFuA2T6g1htWoLTPFdrIFIph6N2AIa55BnhwzKciKmMvet52ZMZUCSSOIAJn30MEMOfiGIjOwSSWJgLyudjOU2cotNQcrJXhpRLQ4hSqtDf1z2QQGghi12zJ7N4LlzDi619CfFExIzJfFcfZXZpMvGODvSMtGGXBWNs6r4lbRtu6SOPN5DjmUQyAxlGg7xu0HYkgfdZPE\\\\\\\/xfCkueHJ\\\\\\\/uvc00IVv+zxSi9DR5hBNMxhUGWwx39iHtxSSIEI8m\\\\\\\/\\\\\\\/U5M285g0KxS4tAzqD1EtxVZ01ILpC1326HwzYrgX+E5zB3nulj6dzgyTHLX2s7hpwmDIZjiIjHkFpYz\\\\\\\/jcQ+d1k2dfWKQ7ZJx9mtzhlE6lE5Ny3h93wU9CayG\\\\\\\/0tvA2uWdB8LNADjusSpJpDmdq9N2U3Y74+outV+P\\\\\\\/Ys+IdfGK4ZuRCvVLRiU9pM7dK3ev+tVHiKL8s+bykhXacSynoT7C2vcEWXsEstB1bztarPjclkCAvBBCfuOxPFEuvX71FrtgRz7pEwM6gnhhGZCPF2i4N33xNcZQhFIBEYMkrlZ+PMSKR+Mv59RZf7L9B+\\\\\\\/CZr7QinFNpmpnoVt8njHaSCp79FL40QmaD08pOfERQHsfyJOxbIkfMeLm6QarniQOC+3OH3TlqQPYLI1Zh6cwZy6mKvlhz99EYILD0cFPB5a4dQxAivK1pIooh7XrenHucIJfi\\\\\\\/PVeb5qcZgdZ0rB0SAYpzObVPGTl1LI5XV5hMbT4QKhftYJ0UrMb+uKYbXEbZHuJKmblqzCRF+VbMM9+S9Mk89TA5+\\\\\\\/MuvoMtg2AsZiDfv9XRpaxOcalOfUUMhApWO2FLCod+mz9gc55SzX4syGasSBHRldPmsirb9Jpi6WDOzDUER7JqJ+waWLwlC5rJ9qoFmKwWX5sG3Nckq6CmQcR79jQL4VmdVXWGjiCMt8PGFBCX\\\\\\\/n9yn2Kil11tHxdlKQ8if2NztGj6gLpMsCN+\\\\\\\/4CfGb9Oa\\\\\\\/80eLXv3GqJgJlFPM3OEw7h9n3oWL39zL7eGb6oXNUBVIs1uBiNHe78Yf2Wq5jwkpE3I0AnWquEZKXOIMppRl8LDQzj+c5lP+\\\\\\\/lh38y49BsXXmd\\\\\\\/+FI4o8chWYJxHDHUnk3q3oqPx8rSUEPpOY1dmDF7AAYqNp6nDq\\\\\\\/4puAOX8Fa8SA07w9FJeBDs3twYE\\\\\\\/p1I0up\\\\\\\/K5VOu+uIN1B6l7ACLHUzTC2BifRgURWc5uXGWcCQNNjfZsp1nTTnVHDTAfOlhglYczcPhWoNBN3Eq792BAQgimvMdFHGQwHxTnKs7A27JPYdY4aIeq7hLKZ3eWFD0Yc2j0LbcZHd9Mwy2TP08ddEkx4dvcBxPNueC03u\\\\\\\/YtS3Fp23TFrjSBEoYEMG48VBfi5zF2S6rzQD5d9Q+NfXyql1wCMBXco7fB3zbTuj4ArRHYtMyWYcAdaATpxVdc1I\\\\\\\/8ZJUP8yZ3N3bHysR2b3AfdpVFSU+A1sxpewmH1upllqGThkAwFhdDu50KxB79WNaBcRjEEs+O9gxmVLRSORspqMSwhmuAG44kZwSJfFJBCMZ\\\\\\\/SlJZ6+k6dAWwJOcq1Jorr0WMO7LR6YJhTSUKyCa6W2Ch6oeKHJd5xjokLY9TKtmF8Zug0lmmRrrhJPJaFae8f3o6Pbz2oOb+foiCurBiPPakVSdHJb9M4yjjEW\\\\\\\/uY+thu9XMUYQRVELasKqiysslKP+K+MtO+TtfVhz\\\\\\\/o6RR0Kkfg0m0wR+lkNF8gmJRDLUuvAXyP\\\\\\\/6Y1ljDxcVujIkhwjjHDcNyFWHjjL1yitK3mpL82M7j9NAiMmqivKoOIWgXkCWjXkSMv02wEqgZWwX9HNe3ZGwaikHd4wg5tri7u1Trcj6NVMFiiznHBjlAAQW\\\\\\\/0WvPt3utFGn\\\\\\\/YagnOJlPkH43dOHhmsDniiZGMzGKeR+XRuKPmXhVvDsfyFPrsA\\\\\\\/j5H2ZrRyVf5DDt\\\\\\\/DJhqyOxrZejw1w8UhRYNVDEqR1FO8\\\\\\\/1psVokXJnfu69YMRnTKAo0AINUrB8onnOO2OAdSZhcufmVln7IP1WfDv3\\\\\\\/Yto6EyncLlYprUu9pIcet3wQOAoXc2UnGOns3ai59aHrZ9g0IWePTlhn5IQS118On6XYFcVUaV88ds7hc7yD0qmbApcIsGTvo\\\\\\\/PGf4k2ugAENIwEF4iE\\\\\\\/74mHIkyooJG5vGswIjmkizgBQ\\\\\\\/\\\\\\\/czSSN\\\\\\\/C4HFYzD2lAhFtvshjtNzu8onInd2mbxQ7zYEhohQKQD7IsWNpX9mCZJ0D7mzcXyURP+Q79L12S1jZ0NV5NoRE5BRkogNp4gHsVjFfD6ACbVJq6zz7wKlCLkNvSUIFtGBp\\\\\\\/Q1dNQsB2BcriHyGFKOsamb09x6mPdgnhxmN3s9kvFN6Hz7FN+\\\\\\\/4fAv6twnLS9JEyWHUYmOx9WlSh1tgkf8BURSQpx\\\\\\\/Dpwf0C8Q9urY1SDPDrrD6pRGY\\\\\\\/qKQkzOHsU8fbSV8XgVA6bw7uciekJtVns3wvIszYBKLSeQ+L\\\\\\\/LIRpMhEmIl9Mu1ARGOc9umnqpPqEQzIsMCJReNH2AcuOnOtKv1bn42i\\\\\\\/J82RALxqh+HJ4gp11XojwVv4C73m4EhW\\\\\\\/kyDGTr6dxYae8oRHXooMO8DF6OP7lvoiG6ZMmkIH66EYibqao9c2ida9ZnbcTxk2fjKx9um\\\\\\\/\\\\\\\/4Ive1R48nc5hAblLO17Dtf1zUDXE6AtzWhw4Y2k1RwyRhB4nlhOsjLrnazousJajUkerOWR5ShGT0HxIL3cz2jI0p0pYO2u92F91T3KckWBKjsuBi5Kp733ANVGQuRBLk3j1vofsqfBUNqmzzzhrtVgEZavrOSAyj27Fwg6UZE\\\\\\\/3PELc6oEVFApZdY\\\\\\\/py1Mom92bXC+4Dc5Rcqc98OgQU2fuWljIZlD5VRbjwGBnB5z2gIO39Tan1vqMHs\\\\\\\/F2uk4gS\\\\\\\/gbkClD0xsbKJDM7tEk5+0B++UhOSN8N9mCkC11\\\\\\\/fJqWdKBPlPlK6F4K32svSYhMN617OEv5btpkqR7R90Em1yoPUVBU4v7kfvdYwYmw64wCYztsEmLPv+SvmYwYpbaFhzRmA8IX9yQYmLUZ8iKYWi1Lfj1WEGD2L0Pp4qTZWPJ\\\\\\\/fomhiZczYQGvJsjXgt+ibgSLK8SMlRVi0+X8ipxnfFgyjS8CfGAuU4GamcDgds7EQuZdaaOy+BkY\\\\\\\/QnyZ2l4IoNQA3V8eFRlbRUc6qa+SOxin8KOWck9sRdUOrfTOfnYvSJIXMmJYBEBheDOyzUTmwAisC30XPP6wrVLhVtzg9HBv76P6g0Ucxhab7oK5sEjTDVx6v42uEPEdvTnLJek6o9KMpcSeiyUuRC6D8U\\\\\\\/t9y5MR42kRXAQSSJee9Gs9+lExW6bCmiIjOoq06X1SywCDLXHxO3iyFc9nQe55Yx8nLrTkGeggpMAKFxUq8r8L1f4YuybfeNrgEutm34gYb+dqPyZRCR1cDfCeOLyu6kjsBIFfRaeFhWiWhdvBJ+zWywo9GJc67HIyuHfRajH1IAEW1kkx8mjnt5TnWPgrxf9znTmyXMqBm9IWbJXSAPAO9gooR9qQBOx2YT3KDVc+FfxPL594LnDo6z3ldYB5W1\\\\\\\/5d8OMYoT4C2JrY5SX5kxu8BoBwzL+x8qNM9vGMNPQRmhn7S9C5foU0uMdkqSZqbYm2nLUfmWWUVeLcgwIH6CdkHlTtViFJUBjJ+w5yv3NQheLqD9rKI0cYaro6zulm4ibDEopz1vLAs7Kx3uCsh9AIJaK2lNSCVlCp5Rq6dJnkGrxy\\\\\\\/fVyPLKh+NNJexLdFy9UDur4WKe7oBbIaOEvl1E2y2HpZXqAPDeqjGi2DIsggcl5Y8iQ0MihTUnXJEB3sqJI+93CdDdk9vQNhrf85IOWEYNfppIMJISJiO51w73j9RhaaTa67QpMjoZGJ8pav6HxbT52wgnf8G0mCd6cP7DkDeRimMu0+Wok5YfLwVGLehOm6luengq+6KJnhpGyMv4kbkAk8zW0wJvldriXeetmrsQaH7UjD8oMzs+TsmVYlIs18zX1TX0jqoOOI6x6hdaAh8kiNAywqTrPgixu3++ro8LI9Ot1RFNpDoQp5J0ugG\\\\\\\/4CxbZTMcohoG2x0+alTIRrhaN2SRM94uI1IGWixoMAd1NG\\\\\\\/1EmbULChs6AbTy2BcHBu8N26nRB+\\\\\\\/snVT\\\\\\\/3PCN4qarkoprNt+3HcwRrKzUexPusV08ibDssrKdGZljg9lb3CdX1jL1HJweM9pTxojkVIEQzyBnTdO4a7F9v5Mq1IZOHO48\\\\\\\/CNUcZMZnuLXMENXUsCytRK+wjnUfRCApD+oFmc2ppw\\\\\\\/lkdbhEeWCbBqPFByrlF58ZWkzDb4k13151dlpCFZeHMUTZ1Fxy7IbHdBUoWnNsFWgtYYcpFzdh+EBddfRDMdx1Z5S7BEBzy7+cHcKsN1HAokKPHhA9w161OFTI\\\\\\\/ofJghW9mLr7fVWPHxyCF5ETTHmueE9Bb1yBcHhIeWHiTN5h2y53yz6GGandNyaSipWLNtIhOhD613IS3aCQLjKIoTlx7EiB+NnmrRJonIr46I4a0+v50sKkpLy+58adzdoGx7Y61NjmG7Tr54LT3BLJaVc0ltwG\\\\\\\/iQIVSlf0868B3JKQZ7krLc5iLrXTU0UQn4YA0Vdj9AqIpYeUxDESU08BpN9DgpN5yHmxirFCvA9dU5iRdxwIzEkVlb2Wr3ibkSWYs1sZr72m\\\\\\\/MPPjwuOqXVoRPTR1E5zlnIbmiQLx13p7HWL2E5Hwop7OM2OKIVTaKXuPqNz2hzyYShH7X3Z4zbCmt075btRtjTljw8rvw+LNG4GdAfRrFXBYGY3tlLzFi93F2+4+9di1Y5RVx9k64nAgXpfc7AVRQv3GVB4o0DyycA4sAuxL7nOQOlarc9eI\\\\\\\/grZlqwPH8xgGfFmvpQnKHGfNjqK99qrwpT31iJpmHKhxZQ598DuaJ\\\\\\\/gZxdldc2fkIIfp1G5jhDF9GZM0krFAYj0+CmhPnuo3w5I4UrZymgaZxm7tW5waLFXEjhm\\\\\\\/f55M53RzQoqkePyM00QbhD6Cu8A6t03ykOKvnVZ6kdVFDD+dA7zV76eLxQjT3PmYu23pyquKsGzftlqkf1kngirb7tQuT9FnKSlRPVNcUnj2s9kTqCrM8ZQpSuOm0EhvNLVcc9hEcWRWCvNCPEYNfCrCkhBTUiyZ9zn4f9Q3Zu7etXzwwwA09dC7unIFOLT1sVdhEG9Zrfly2TyA1i16bVj7Cstq3U+4FJUuYrMmua7csFt0DUL7JjlkFNrJSPTMdRV8p4CyBTFHK02Vv\\\\\\\/yZ+gUDwlS\\\\\\\/2x1fbdBnA7xPGuZzbx7NHZHiSuT0EWBkf\\\\\\\/oVP9oEB21XZDmLjPrCGWmEPFCEC6Pu5exjnEE47jdpb+j+2pd+bOw5ETvq52UfxwEBUohn1Y5U1gmvtuCxNXZNLZ4XOyG+I4aZ4hlSPa6mr1+nscpNtEYw+yDbU5Dtvm5F2sa8lfkueFnuQ3nJS\\\\\\\/+MYbyDw7RElxQFB2ShZ04Ht4Tvrjswe5KrLTCj1q6pHSerrGlmyYfHnsHGh+ydZXj3Rh8oX3Nz3X08MuO62TIFdA14FDsOtQ4CFkKzANjb0KgMkcLNE1NwIkn0tC+PrSVcj+5wojyAl2LohsEb4jbMsuH77fwSo\\\\\\\/VUXhVnmeqJ6jwkEBXGXC+PGXVF6UrmuQTRuUET9xfrFU2ie++uQsO6ccbm1rIjShKTzMm\\\\\\\/7L9+AGab07eqSwuhKZCAp\\\\\\\/q8i5GGgWPQtILYlTdYvLm99t55\\\\\\\/5WZYrpfNEs6wdu6JOLO85GguC\\\\\\\/vte5Sprnzq0zwg8bO8OW0\\\\\\\/ClpWjT4snwQ5UabwdJY4ryUbZM34vrOpj1wM621qQaz7GIv3lS56URe6nNbVhnWYWEI+oXNhj1ZZS0PqtMWzETUMuASbnloY5yas8RJfS97BJd1MrDzICRi67looERTBKor4QnN4prPmJRo3P1Qhy\\\\\\\/Kd15l6U9eEzt3Kd+uFLADZu3xO4Bu2CKWtB5gRI1VIvg6urcxYWt71ccxb5XDLbWRCVIfiAVOEhGs4XhCuUV1GQJJjPt4Pmo2GjnUFDZ5WeDVERsgugTwnNK07ywRvlNXo3BawD42rpWsbdnGsZYJPdLZncAzBBAmhR02VHOJo8AFJGWW4wFaDVqq9gIpM2S1AzClkLLTfuBMcnFyBhunSf42fBBFQ2mO8MJt8NnwA1WgoVc2pHVRz1w9I+XDMr9wBEIhSdycRwDicoG\\\\\\\/2qClQQPkX8AxgWX\\\\\\\/rR6AG57I6attNbCSSQl5R+QO1kOHxBp6SFBQT+PRoQLg69zgy8zp\\\\\\\/CR7MGNX1wKp53K5pzbhXgb1bE5S1tKBSluOyU0DXpEgnRQnU1LkpGE5E+t2inb1dNMMkGCx9JqOFET6tFqp5gAohlNd5D2NdL5B2xbqHEa8PRPTGrYtXi8gpqD6g9M\\\\\\\/Jzoi\\\\\\\/R5rSF5jzHYyKFIbvJrZHOc8TdS0O5FuwJyzT0PEJEm0Y99KDAadouJkP\\\\\\\/Rig34\\\\\\\/wcYz3LqtxPJGgHn7kj\\\\\\\/DafOMNblNvZ0+Xr1B7FLQUlQN5gTFuEXw+9NAFEiXVgXda3Bg095jtSkElf4NEC4qtCZ0Q\\\\\\\/W8SNW0hHHnXmLY240PbCmUJD3ZPBNR\\\\\\\/tIQ9Ui7CqJe9A9eCQkqB7LW076voClogRAvDreIk0PSVOx7LfOOXRRFzaaDzBLT0z9XS9cAzbDKs66jttgKYHPQVsWmvQbkSkKA3o53a4ua\\\\\\\/\\\\\\\/RQLu7unZRxgdhpx3EFFm7Y7avjiQrX5tO\\\\\\\/Eaccx05ivjpBHk1iGUNTZS8aF46mWRzogs4mZbfPTOwqAH9EqpVZPJGBier55Gdcd6bwbaT2dssAtq+FLfVC9aVCf5ANGAgDbgVNgnqlhJV2W0LjGJ\\\\\\\/KIci\\\\\\\/s36hvYHSmvbsIu9EKinqA5a2yKZSBcFBktyR9CmXMi89YkFmj5WSGJy0R9I9pYaXgTSmmGcT\\\\\\\/1QgYK1KYHe5r6F2coaGnhLaLxW0bU1ikDJRBxrxD7WnWWmJaswPDCeMLDwYj1pZLybAX3j3a8SRzwTfmoDUqlGMRc0YkBZFSqUJWF3+s\\\\\\\/H\\\\\\\/Kp6QUuhAli98zV3YHkPGkcaP94y1g\\\\\\\/zDm7UzzmipX094IP2EmRvrLQhgtm4eZsFnyL7PuYrKkB4ME9cLBbHHHUHPXmIhWJsI4boreEukM6DbURGbLyn8GaGj4ZbvE5vLQPEBtl6TbEAYz8NmwnJPa1oGAFCqsZBoJzSyB6BSfHNFtFiBD8FTRYIvP6p6qN2KLFgzmzhGWAfdicqdemA52AkiZcocxx06U\\\\\\\/rF7BbyYOyqtGgbiRzGjv0ZGqHp9OPObjCEHBK8G6C9oVTFEYudIX6N32pCeLKOumJVb\\\\\\\/08VHgJk+tbOJRmNPxC3\\\\\\\/HuNn01N+UJIh\\\\\\\/dZftl+y0WNHPtvtqonDawB0fCU9D+2fhWwNJA4dpBagPiYtTTMHHR89t0buW81oqMEN\\\\\\\/Fn4pHGw4vEHZLTrJhMgM7cDe4y8Sh9w6sr4iF9PIMmPReOsT9e63xwDvlFn30sunG+jfdVeAyXbaBUmIatgGxbLXmsP7NMn0zun5hsR4v8bqXY82RbP9oSy8hYngs+a6JUMz5Gp4WY7Tapjq1xsr3SZ1uNv4EYlxH5wbMAP3mllkqvpluM6DXK\\\\\\\/w\\\\\\\/DfCwYSUSZAcNBqQ4r+AqVoq0QHmnWKGCLkClEDe4B4iDyaZ3OpmspcUI8sc1lj0VurGqFY05YI+Obzijj6jN\\\\\\\/DGWkNHbcs6+yFOyC8YucZMxlwOZp4CLS7rDiL\\\\\\\/qarGx6DUcdHW34zTlidOrhbJp3cbL3C\\\\\\\/fMjifBZ45HoKLRSSqMzTgcxB+Xu2ggFT444DYBg8tMAPPcUhKqjD9yqM1FbmMwR0RFWzRHlBls+j4k43dYmSudwTH8eLD2K3a88r7RjsK7uXbQJm5evxHiVnSkVzmr4iHDs8BVLbcIrdguQUv\\\\\\\/\\\\\\\/Z3rZ98Ayh7ABnjKC0AJB8N5hWUzYJ03IjO7HypJSAXXkODkX8oRduxd1PYGPObgAKDkYm1LyytqUILYECUmO9ZNLLsjjsBFEK9\\\\\\\/vL1GPcMST0CshK4JHjonK9RTWjqXj+KJP4IAgXZqCS9o+klpN+NrViKC5yICvxR0WLfz2QrJLcWevdc9HoPLdyguuXgPmDVn3ZVXBvEdGTDVYlDvwZ6KNoQXalHiX9EzRDsvO4eq47en3oksxl9Xu4V6F4O0KvF4RPn+mv30ADt\\\\\\\/3lC6J7y3c7lTwToFchbG+qUu1iAqSWFGXvMlikwVgAF4UKImfRUcuEHKJh2fwdCVOzmeHB6yqGN3qYrLG5suYeOj3cbcuj+2aXNq6YXVIBUthyMek1PNVL7qAWzYcNMyp3a24kc\\\\\\\/PDncGasjsrEqjYfTwDghUWeR\\\\\\\/XhYVPDod72UDtRJTD1kHbrYysAUG4uuAHKf0dwerY6kjQXWaMp+qgILqT8zO9K8cTUtuCbmQbHh\\\\\\\/PqsK+AwohgrX+9LFbmYmcuLRsxHce4KO3XHlFb35xMpN2E0gyhIJAeUeQCxsy52XydHLAxX1\\\\\\\/XSb+eAlCblls44r6oDvlviu6UzWUG46Bd\\\\\\\/zlwhSFNrpfeH+15wzB2cbRUPIhQDly6Z+QO6KXsuVoZxG0VlwSPucV230bMmD6tHj+08SiIa6N2G25+Un7GrWC+qwoWH1GNzkHJ0XAovuhilu6bsKv1xpLvuymQ9R7fo6eXXVDJ\\\\\\\/xA8Y+pb1e2fFUJ7+7GP75R0fl2cSeloBrF135er29o8sX0lik9bgro+j4bOVOocg25PjEMvvocFDGmlZgwBhdj+DMkbuO3ia5upE+cpYvikQm9lloa0vFWN6sQla9NQIWeqgnvcYFuZcLfKyfuE65JRyn80rmExc9F+DSx7E3CFRoYPdqHMcWriB3EwQbY+6PMeFkBM+oOb43M1gFZWyl3WnZDXpzKXBgtPYowGTKXwnqALiE\\\\\\\/7Ov8JBuEFo5JhCAuhMkMN2kEJSkk9iUgnaOi4XGIL7snhOuoFEY6IvWRPEOe+I2ralE9AteS1\\\\\\\/xyjWXlBdY3BvDF99EwARJ+77TMGnGBjbzoSnKCvqE7KPByvRwvekiTgj4ibjuaLtXxPFgU4EWX2NLQ5CC8MqI1xtXIMAYYR9il3arFOosky3Cmgc5jjsmUZ3FoW01FaIsV8D3SCN4m0Wh8VM\\\\\\\/TJ+Fl\\\\\\\/SHiCo97yftaWraDyUIsaHfFjSsnh7HbW7sb7OqWo1k7p7veutOsfDp2R8o7UNsjDqwMdBSPilNfibhbESUJ4AfXo0WAyeI5NmVr\\\\\\\/3kxal6PDLeRoltfMbPfvQiV4acHYYgga64CUy9YAtV6WjFrLotBCUlChZ+K\\\\\\\/x7RCV1US4N\\\\\\\/Vt5iDeRKKR0s9dO+umy9s\\\\\\\/Texlu9zMYza+gxXHmgGmYAzVpfZGYk4kQa7tv3LByDRDjaAGOCBFNY05+71T9NMr6CoFgt9XW32\\\\\\\/y14ZrWBVvKAwnQ38QGj1WHf6EpXUT0G2eVNf3q7uyyiHzBn7rW3wdZPBPBUk67H3VA5juCETtN4D8NKaMroPXoSMzKrLnP4+TGc+mABVLJxUr1TB+lgOZdD7mmSObplIa6KG3bSjcCYnT+CQHnq8UNRZLhP2pLKoDk6+41CAEvf1JEJpocT8vhF2QHTVtkYfntZ8p4qo3qKfmeykhMMFUxUs5\\\\\\\/XObnCJYgFhr1qFZSCeSOtc\\\\\\\/0fcvEfs18nrofwXYC7AL173YXIEtkOlMkB3lmUgkaf23bvJgU353NAFAssWabXv4MbAbIPupdIQChLWPcJrsgc38nJZd+I1tzKm3N6j4qK3zutcLrzD7+Q8B2Vzrxka+hMQwgzL+LS+5snN7vBOrfplALHD7GtQ8Whgi+MbDgxWx4uxEnWC3SIx6zYzVohjKFr4H6T9n8mcVVhu\\\\\\\/Vtj23lALsLl34g3uGq6k6gL32r1MibMFHpKmKiuYopndaE\\\\\\\/jDmrDd9gaR2hfBsk14jDKOAHSFq6bo8c79hG\\\\\\\/dfypceb6fK1SisIrM4lYcSooFBASqFGom6fgj4qca4G4LmEiQZCtQi5XcJwjWJTaIfH7T1vSnr1Col2xubTo8EyGYTzz2Nfhj\\\\\\\/vZQIVBONQAtO\\\\\\\/xeUG99W6\\\\\\\/AjPmr1tN\\\\\\\/6CBppZCN+NQJMZBcG73NAl3Q\\\\\\\/XlbiOrnzqP0+QknbZEQJZRdstjb0EkXntaQ6ZCDARp2T3SIQ\\\\\\\/C5+WWpxEREEJEo7zpZE5pFMKDoPuFa72AtyTL+o5UEnFdNN6Blb2WLSJHilkGKkNmXLkL7WQKWwNlUYSk5xgUaliESoDM+AP8N6ISG0iH79SroOnJTf\\\\\\\/uor\\\\\\\/679LS4wx13C4hRMUQrVlw1MSn49xQ6XmXswh7+FEw7cWhae21DDEo2+zM72tmXR+JZ9SSKXaiJoNlQ01WeZzfVlHz7ytZLEsTG4gzU0wRhqdo8eMcPHTlp6oZW6qkxALJc6YHYvrS6z0kzl2msOUt93o51TAFGw3N7KskGnUigJQ5jNoHF5aRzlaStwIytpwuuh\\\\\\\/ZCNqiQvDf5tb6PasMkl1JrN8hotxf4ars41e+bJaadpjlb0xohxHEeQPWoWnSqItHevlNNQDY3LelMB7C+jo1rAtqYLF221lCfi9CYGphm63tLkVLPVpaIVJVaMekKYZZR5GwgSMBHk6F6MzgW+CvhUkUJ120XFwxF0eHICzjKVbGGC6O2RcIa6frC9EQq+UYJMSYpkOjC\\\\\\\/C5rMCoDxBYXdsuDdL+JxTxZ+GSSrJp6zJUJhr51x69whdVSa+Coasouj+BmdMec1xMvygiu7qXNDozQBdhphDisu+mC47hkXbBRB0vRIrZN7WoKDBxelL1hS3bPR5Vc3Ej3sWxryCOKhIp7kiijDZeXPAalqRxeHDtuo+gEMmBqJYyGTkuoZWLs3SNku193En5EEY3cRP0gO6\\\\\\\/xIUzfSw377l+\\\\\\\/mO3JmaV7itndU3sJpJoYOXs2Opcj11xpter3oXaofDaxUmM8BiuXNrQJEqZk4Zr0xYlZwVW+1XQSfYlXuGOaCRkSdsCmmjw8D7LSleEIk0ONtGBYxM2+rzC\\\\\\\/i+QEwGZ2whU9VTNz8S4OQheBbPUd5z54rm7bT8m9nFYGoZvuv89FzE3inOtLD8N82z5wzm4NLulIUotNPVySXDxvAbcZIf4XxXRt3Wwtd9zNr\\\\\\\/\\\\\\\/FhFdtXcQzMc1BAk84c7w5qgbOzQXMHcVBoC+mL6r1QJP4k5qh3rB3+8duHAk\\\\\\\/FXjLxzA65YVZBZhYhIBtUthWqktg4aexB1hOWNjhqyDcvuSBPGfp489+bGyXcEyTIxz4KXJ8ub1cxFajmrumRzC9fIn2GePHOUDZGJDKejDCRJ+zd5+MqAVqS47qXCZEYqPncYSwQQ+UWKpzwBPn5rj13JaxqMNknTZxrwbWAfPwE2UZ1hX4GH4U7X6rNYeR1kChFPklOSX5K0DLz8DdvEkRe4FNGl+b8EbGI62euZKOrOB796zMHJzd8+p+z5dw79Wa5cT3n\\\\\\\/KjauraVYZksEWhT0o\\\\\\\/jlJFOiStIk8hQJtcLSZTKS6eO8qlPUDxV3KCPbHvMaXIg0oDEA+Bfko08fbK+S3o4cdifZtTvhZcHcfjb15m2UiE2VMC+B\\\\\\\/9qEdbVM3WeOCCb30hjNY9SkwczRG70pDNy70\\\\\\\/yuYArwWj5tIEZqoN+CSa9u4RiiQt5+tiTwB+clEDwtASCxTxHhj2QHl4XcsEE5R56erLtB698uC4UlNLhHhRVwoNXwnHHIXApikfK7onXU7OJ2cpaKulLqeZybmHOkPa1JHQyPLIqShIECFSxyvUWTGu0OjhqMZUH1JmkzZ8WNzoZPhdZoKP5g2\\\\\\\/7JvBoABHmukK\\\\\\\/yCNNl8LH3ZvxaVy410fthzrbjNS6JkZA\\\\\\\/OHrqkCz0eaLZySyefxbDfFXqp4RfGIugDRCa\\\\\\\/73Xmm1DUFClKfzbOZLN4o99UDzkwmxkg+ODuzclV6T3mNNpBgvxmvVTEbUSZlOA9DCiWR28zcUuwL5t9HyORxaRPS7sNG5zaZOyMxnjsjk1yAhQOuAxKdpZnzIqAVAyIUip6iIjl0X2Yeay6xIPQpGDqvomJxe1gP2kC6N\\\\\\\/hsbsyK72Ewl3pAOHLHdZWQAIxr9u\\\\\\\/hwIZqTQIKWYoVhRkEafRxyBAkoqJ2NbUNTY8fqTmDYOWChKorQ5nnWqsapfF3tMHb6PTZ8TPBuUvWdj4y9KpP30CHj8gw55+\\\\\\\/yn1EoE7rZZw1Kr2FzV2\\\\\\\/T0XzGTYQ1dyxz1BfLiKynwW0pwVlLpjeDB5mJ6PBl7npu1mVJowINNbZ1NwNo3OUS5bGJNfX7M3AqfQr0c3sCQoPhvhBWhFe616hKIRMIvdHz93\\\\\\\/Cdt9HJdWndi6dsvGovOo\\\\\\\/ydnyfaje3hXlvaOCbPpJj9sV\\\\\\\/WKrMnQEjhRAh0bVo\\\\\\\/eTKUbJgrGiLA7DTi77P4b1fb1RGMUFkq\\\\\\\/SDYn0CzzUP7ByjoMxkt1jF\\\\\\\/AL1g9Gb+vzMCc2vAwWB04OgUM7IuAhg9z9uIoWq1A8Uhdi\\\\\\\/ekhaT7GRDLCdj2QNWJCGNVuUZcTjfvDqTJmKWVk+MqjPgxhxjvdojbSKJSE50siecQuZa4N+AlAsU\\\\\\\/a4+d0KpvTJRG1LeUkeUdBDPaX4u5NpXe9AvkWPwDAOk7NQ\\\\\\\/QEV2rUDJClJEV2x2MfkRhiagyBUG4Isfg1gT9B+LfWa61Qi6Gy8rWiEiBOItw+saD1tslsS4KGBugsClTXHBvsfSXuO5A7fBcXWGQ\\\\\\\/M35ezg8mbAE8Dj\\\\\\\/oa6QU8ZXBjk1XYnbkR64nzZhIS3F8j8yaOFGOv1OOPxPifcmdi1whKQSB\\\\\\\/ZqdVVSz7BzFzs35x8BAwkklpawJwuYTqHnP6KgGBkTcFkTTVx76DSertTdeL\\\\\\\/TnrhOJSqpwu7Zx1wmvDAN5AG9woCfXF5UuoyO9OL9eO9H0GCn+Zc8AyHy6n2KpwnVCmV1rLYMtde\\\\\\\/wBa8VzhhuchS3rhoJPm4tY9uHVOiz8GtxVE6IPHcpJTmRChJfh1ubsg0qshs87+OFy3U+k9nZc+d5kP62\\\\\\\/ihfZJIRsRoeUBItEoWnbKMjGnPobSH7PjF+2JYoP+ZWmAMmtxUKD8f3ouB39CWbZ60exysGqHtaKYpt9Q+sMUpeC5DL0wtG3a08he9DC6oJW5dT7Mnto\\\\\\\/icIrRnwqwDrn+vKLiy60WZ8K7xLbU4DmodWNySdkc5g2qH38y3LfH12nB8A6wyMsTwJaU+ugSkcF1sLvHcSGPUwFqCiJy7gEx7lQ6bflXKplM5Q0cx53pm1eLvko307Qxv0XXRvLgIRs0EBsc4ZZoCDO0muFGYksPqH1uCNwlPqhv708sDSuZ5\\\\\\\/xgIasTNSqOyfxusBALRGrCX9ITbUwQRYO46aJBMKM0GzTEySs8RdJIRLlueGkk0xrgQItKbuS2h0zAoOV4eO\\\\\\\/xLbhHnHvryaT4QZDsPVTdeXMjlaUiK64IDDagp\\\\\\\/W\\\\\\\/7J5qjhr+n+ZRVFKXKrG6B4gfNotmKKad1wJA4yDNpWtDsph+GzAQEHXjnwN51vK3qOrF+s8l6kj+qb4mrBAGnrwbAFhHknhvitHmdrpBqn2hm3al6UcO37FhI5vkUJwcfXOmtVP82Zo\\\\\\\/N5kCTzvmNtFjjqMVeZ3rLgvaw3lMn+wvGTQ\\\\\\\/qXpKHcl+RJaLjt5AMPIhlpUPgg2wVpR3fSTBcmhev8i7NeKEmnyl6oPdjYo3AN6qYEHvc99zVTWPH5d8ZPXw2E7VUbN6qFaXMe6At9H1XHdg7vaOFyBLmj0jgBiPrXfEwQ5pfVEbM2MXby2O6jnnj68WpFKYiNME+BFEjEeDOxXTisd4Lw471\\\\\\\/Ku2sHGYN+TfotfG4g1JWHyBGHTktNGxYcnAp95MoAN37WTVeyT+av4KHrj\\\\\\\/b+vbTL1OewtVz7qaF6Z26swnjBMzuDeGH4pOsKxxwuvUcQQTLQYlSeNZSCUsmCJftr3xLQQdbyiZqhIA3ztdtTyNGA5VOiAM1UXt\\\\\\\/8P8rJmhNi8CiicX6Fkd61A6kbf5VuGMt2vJayyVRBtNmcVvyI24JD7nI4OqCXpHyO3a9LKBVU\\\\\\\/PYJEN1jEqx02wBfIMyUAKu2oPK\\\\\\\/xll5EjYAbB+lzkDz9WdltjnUdKpSSmqCkRru8wF655yfOAbajiZS9koBjWk3pIWshEexQDaPFul96HiqUVXHfCE5y8p8Cst5PEXjqD4AEbhjkCZQiu1WkzGxrt+tGdbMyDWOLRRf+OOSQvecibA39PUYrScjc3+JbEPFeVlycZNiCI\\\\\\\/W4XDXCpshkBjmtaqoAog3bt3IyEhN9LfTpG3bV279eTNLuoz8NoBJVnMYAfmGQBK529IMVlC2axJe1EqliWW001PWNHgj9TtLYjKsjnNEV+5f5Ce\\\\\\\/46JtK35S7y6CcGcaBlW2+yDejFWZ6XcM+Z0KVAsC51ODf0P6a9ZTfm8ah6ZmZs8SPEyeyZgIDifFyjGwUkhldK5Rxzk2sfoC6usdnamDV0LYNMJKaH7BbN9F05H1K5YNE+LFBuvfBArQmkpBWP\\\\\\\/umacA0P0mhH+ah5K7lT1BNGZ8FUnPkx1NBZyZtZwfjUE2\\\\\\\/2AC7+DFvnDzgKvmbE7f8X\\\\\\\/jbFNZ94qK8TjLvPrpxAGDnLFCGTkj2VnZjTPBAZTs6mbybv344YHjY4KWroAicDznB9pdxmUFXGx6yEOx5rii3YSaHCdA0Elf50oreYPL+eUuje1YxVTTP9GE4lfsJLsINM9fo8y5AAoELlXn8xyQz4A5k5+ieZNK27ofwPw+JdUMc8inwL5cgheytZL\\\\\\\/3Y6TNti2L5quhYTzehYspbeJktPnwM61Mjz63xH5vWoDwQsIg42wYGne5XJ1qzW+HrLnJ00ROPIYeBUrdkqkufxSplD+LyAmAOwRh5KvXgBKRvC8apouHEXbHorRCJqg\\\\\\\/lENcFxL7VA3Y6fQIOW2BXfCSjbkOvR09kFBhd8AndymvJxcdpzTgeAikMBGIs\\\\\\\/Unl1FAv+vGJ8xPGL\\\\\\\/NTn2ZaGbdRNq+psFXaDlRa5HrmrTh19TA4XV0UNYc8ZB80l3U\\\\\\\/vV6nTkyqHOKYK8vF09+NZMiuj8z\\\\\\\/EbGYwaxcN1h3Mf+IL\\\\\\\/bXah+H+5AuxDKlTvki0il\\\\\\\/0IC3sgRSYxnGTRSwREReGdFp62yKXxdy68dOE8jyfwVhpAnfKFD3dlNSdAg8xwv6cTkWhXJ7+Ib5eWVuDhpuRZ\\\\\\\/qPgcOq9KTaKlFDDU+z8RjexLPDSddmaFoRSKpMOkz6B5nWr0NlXXlR7cXYVtXFl5O\\\\\\\/3y6ZoRwtYaz7vm++LGnAzTXZZYX3uCIKZ1xNTGVkt4Ws9k7mmAw5Ezedg2VpbxmMqIatoyWkWcZKrDZD\\\\\\\/k7NIlsCOQ5Dvm0VU7QHvDpyyxw58Gno98I5nK3Y\\\\\\\/Rh18reT3BTGJGo8B36ikjJOOqUzhMM+TGyrNMUzaTmsPUR8AelslfQE0RHjLCY673\\\\\\\/TDaHgMB8TbQ4aDZvRtl2AtzYx5EBfYSOAgDzge30Wh3Ntav9RmBmfMF5HaDLtaHJ6hKHmwikZ36l2SFyoSN4Eg2D9hYKhCEdpL\\\\\\\/LDY+f9FR1SFRMZ1I10jeBOhlKnab\\\\\\\/W3UqJD9Bj4JseXiwo6qGrgT6aiDUMgRtiTeLvAqEVcv5kJkI8wsqnwQo5P2PP2B8uZBQfojGp7dmM3VhT1Blau4n\\\\\\\/WsVSooHebbb+aNABODkrJ6skxa4ync3DwSMXTS3p2i0Qxv9XK5621sefE0oAwCNJ1qY+KqluD1RP4px4AIekSrUVb13ieFhTuw8iKCoYSdfhvED08fPqBxo8USvChH8i9u7FjK\\\\\\\/ZDpW2zkC+wSBkZ5dQIHrmcvxFOoRBpVUnXC4TJNtT+q0+wanPu2hC+MhI0FHPwC4cuHc0VZiLIl2ZLRuTUBAOrR3Pae65Nk3ty+7Q96wbMBDUenlVvisRwoKclCrT6MQugTCA3z8jmv1oRcgyS3jT+ltK492H0ZT4cktZXfaH+\\\\\\\/3YTP9yXhCoDTVFJ+xFoBHaT1rWd84SJ4DnA11nwC0RP6U4sCNkbmKCTC9a3qGVegw2P5O6g+8sLlubsj4EjFBxSYUAJdAkz0huii8V68\\\\\\\/5RSF4I\\\\\\\/ca0FzTSB+Rd0DnRBKgDH+c8+alU76lVxS45\\\\\\\/l9nukH5uDbFYMUGCvoxsxa6ZMp4ZozjrbPe3TkmrLaAAJDl5D\\\\\\\/x35gpq+UUT83cYw9bNP6+V02dE8NSVcq5kqlccEBwFkYySz4FuYwdXjmMzaYuhNoRm4WeOSnzXwRdOBCHC5y5LhmGsY9cxo8ljFXVvKXaXcMK\\\\\\\/icoiixYDUR3xfWk6edi75pOl35fjV4Em+4vUl1vrb7rJqq\\\\\\\/ICZDXaCM0KGxEkfJutnxHmqp\\\\\\\/MH7d4xG8lh5fnfyPrH9ys62jg5C50YHVzVQrmQVC0z+uHgQX\\\\\\\/gyo1ZQiAjfY57\\\\\\\/27SpkKOR16YzX3DtRJccXh6WeSu4zYBSQk8ptGQxFvfot57Tl\\\\\\\/RdZAByDbltsUVYZQvWN\\\\\\\/Hff8+iMKOGrDZ0MRFw6mQBPLBw0veqnbaOp5TlgNSqY0mKbZahoX+kXAOXPkcky4AJxDH5gKiPQ7o5SHUICtWgMbsPabv6ZVYqsuhFCJfa1cpQCoJ40qWLFZDyZhicr4g+LgwTzfnhpTQ2eKNoFsItVAtvNrE9qojl1zsJFJAJ1ln9UrMxyPD5Lr6vZY1qadRbqVUSMOMWXg\\\\\\\/w+4pDQsFRTXs6kH2V+001+EEsphPnwvUAPGdtLsVP9VBHEuaaus+rKbFwb2Qb1nW5ygTXIii\\\\\\\/HrH\\\\\\\/XI+byAfG1sFr0zqSXrG3hB+Sm9bVKZpDL5HrLIKxpIjCbGrPp+GWJlY8kTSetVa6izv6Pvzb6QYcyFkejMf\\\\\\\/KOmqcIzUgPVGB0cTRpEEtlB5R9sK6TyBXklWM3ufJsG3BkH33BRrMnfuwc2pcym+yxsFFvnyNirl7oKHxA1Wk4G1ET\\\\\\\/FX5qqcVs89a68ALYRNc3bfwNEsxU\\\\\\\/wDtNbdv2agc4iSimrzBICzmOX4Slb62YXq1f6jvvjFL9yBuCs30tdorhCHBbjE766lObn9RQszEhJ\\\\\\\/CooGSalK1NhgFxocLgXATXMwyHk4doxujhRY+BnIV4M8T7b3aGRQje5ZeqqvjBkivM4TR3UZUuNpUI9FYHSEflMRsdOdp6IFCWV30PIUUnD\\\\\\\/0nnGUUW1ctRo7GkzcAzwUKCZB74sieCVP7yTRDX1vq8nJVbiKft+6lI02UKzAaYDr9AxPFEdFOrfN4dlNWqYCKnGzCOUHjmIJbqo+BMc51fffAFMsHBRgg1KRLullkOaaCaLXm9tUydo1ENBd7rzXjf3ukg8\\\\\\\/lJMFXba9OXbu4GyROQRVLlWr+nDkAklpyxZO5e5UC3LaiZVjmYm\\\\\\\/RutKzN+tshNAjEUlGEN1ExRErKUnttqYhcoMVBIboakS+tHor0MM2BI0exIjdPkyKOEL+ZIT52p5RFnM0KGbO\\\\\\\/64Ws2MwgD9QdYsWQjb\\\\\\\/+\\\\\\\/iK66o5IIwLHitYvob2Y7iPGhfDPI9CljJFZCS4F1lshipPWfibk33UbfsY2T+wk2aUJgR2V0N4NkEJbUObYAnjeJ7QxUYnsmRi371Rapl+RNICxSgvofI+T40xpSKV34lzd2\\\\\\\/ma\\\\\\\/2cFfojTOHKUt+Yk4Up26i3EYFNJVH2a+1NlbI9FCF5gprpOSj\\\\\\\/Qg8aUKKizeeGq\\\\\\\/+HgZsOM2wZNcBuklOfHI9BrCDKdSkCJWDvp1f7c6c48Kx5Y2udsVAH0cMz1oEq88PCdAEKFmVAbXn+t4v01Lr57t3oMiplBzktMnOpriL0pbl0SvBfrH5g7xoLb5CQOPaAQO1PzQZqdkrbZiknCLxtV4osDv9SyDIMgfKlaIeQwrlN6I2QLRcH2eftXniq55Rn1upaHLvM20Vk8Rzs6AexvZ4wYR3Vbm4v2lHBNWuCI+nH1LJHY1JHaqBKKH34UQfCHzgTz9k\\\\\\\/8d4hqJd\\\\\\\/mtpELD+5JKnDAxJ0zXPn9Em3HY\\\\\\\/2ga1Q9TAszj8wiYwkB\\\\\\\/Xe0fNIA0bef1EV0tkgUxO+UYK6KxD0dlATPpZ9Y0munlKYItgoe34QBuWwRw9F4\\\\\\\/dDDJLhB0vLRdB7pyYePe5iO6mC6RVSxhxtmCKfumgb4AukDLTQ+nYKNmrcY2IXjdLzFJzI+4BK0IPWD\\\\\\\/C5L4ackxGkynf7Qtbc2vpyO9kc5wlviEcjOFMZjJImluM8hchdqg1bZWFwaV1\\\\\\\/\\\\\\\/fLSBL\\\\\\\/dC8+In3bEk41\\\\\\\/D6ffuijSGVSxpGJ5n9e054uIogDI2SlWl2Alt1H+E4QF9xd9tkNsatMPU5YGchLpsRojzLKEP3p\\\\\\\/V0uUa2DIlMv8CIVLv7H3UiZZEjQqb576DNA3FORfC8I7dtLO7z83GF6SdLM18FcUq3\\\\\\\/X1b2\\\\\\\/nao6eOXJFi7Xz\\\\\\\/\\\\\\\/QKt3AHxLcXN9VJXyRT6F4GzjuL3rwWb44W1kxnThdT5yK\\\\\\\/RQIW1er4EFO4tgR9vwsLQnPRmiVkNKT6w1Hawh7O8iOU7+5uFzAFzsg9A4ypyAEXr5PaWmLDDu\\\\\\\/olxZaXgFerJHiNTBR+X7TkocU3mGJgVAbpezK+C4htzO\\\\\\\/eHyCRSTa\\\\\\\/4PWKbF6zuryOGXUqCdIi1wnZsIbQhng1wLKn\\\\\\\/4yfRD+TIDNMdKId5MY2+XQTUFnPawOXEY4NUhZACQofBMbgd1+dNOalA6iEYihQ7LzDgqafUgwcbBrQODmMcu9q\\\\\\\/Nk4BFCEbUYDbVdXVX4vpV3gbXkWjmoNNY9DFmTfJyvTN1fH0\\\\\\\/c25f+yT2\\\\\\\/3A7WnmMMubm\\\\\\\/FAeT59c09U1cFPFMvBgve2551wTV95wizZfoHeK16vNGRNgCflXvqlUtTCpgBvb0aDqMLl6K2LtBiffEQtjaOz4e4MmTwfl3Wr1rKNxdu63OXz9+FsOSsYkK5e9ihMMVS5A\\\\\\\/YDernrf9oTClTgftNX5iE9lF+0aaoMl1qdGjRqTFhzXlJruCYcs2wPqP9dsvFske3AMNKbVyLDQarmxolS\\\\\\\/DvsZvvZJgQPjkgcMMj1xAEl8HzOk3md\\\\\\\/t5KaKuj\\\\\\\/WEOE2XpmPap24SOvkB8V7X2tQecoaboTk5g1IsUajDXUP3zvmYoGO7Z3cpAxvlRYeVuMC2zGwPA8gwnMf9xzkfxvvWFAPqMhzbMg0kYdSbeW6F34Xr1y7YY1gKazQeWcXLt6J7zkkHijLEYNwI8\\\\\\\/\\\\\\\/jeVa2LsPFv1g7QD7LRWiahXFxWPxd9Hs0uATgCCWIARrwGcey5A6+Cre7JTmkVmMbb2W2AXtqtWkQVqI3shH+OxMi+ZsjAC1mFiwAYn7fgb7TF0ctsTg509+PjoC\\\\\\\/x+8hO54CA3Ux\\\\\\\/eIIMwhqYhPJq+7MYazZHRzNju+xDcqNBjH7A4bOyjXRBd4h8vSayqpFbUrSvue0YyOiL3PzZqloqvW8hvOoE1gkAQ4UwUvdU+eoG7dDwKq6pOKtu9MhpDp6tgLxtFCcGuyfxSdfDOMdOI\\\\\\\/MkznKOt5WyidyCZI2asPi8+s8j7NbJ+KGEujfXry2rd43N6s0+HnW70ss19bRBTRmw9nIhZdMIOEuYqjah0WeNQF0bRpjkBvymmOrgQyab5NlM\\\\\\\/Hw6mJ43TNUYHq1MmKzskWOhNtwuC70385pDXNE+5VYgkc8wXHlvA3hmO4HMyN8r9+why90sYxLi0l\\\\\\\/UYxe2llvP+HN9eS+zS1gGlqxbULnWcywrS5EuoeHezADoxBIiAOZdM\\\\\\\/pzCAMeewLqfk5tVEhoYljzZAeOZTLcDyJSPvY4zIYrlW9acd8nKJTZBylDD1dffRwfUIHZeR9kf8IzYAx0gzmgY2r5Jj\\\\\\\/1DDTLlpe\\\\\\\/tud+bcrfkgIfT42cY0qebCHC+pxk7pMzwfUbe7F8mMbzd87u7I9t5J6cy9dZuv4N3hFopZL36i56JiXW6dYiOMD\\\\\\\/B1gZF\\\\\\\/Z+ftLa4Mc7GxQkW1QZh7nHWKEvAcrktDqS9Rg6IyyDPF2Bqt8TCNT0x\\\\\\\/1uxWRX4I4FcOubH4ZCw9TgxOdo0BBMQg6T6Bmhc5i89EdcTjJiaa7SN1Bve5HJI5G0Hw6U0KKPtaovc2r7IwOQzI6eyReC8pBNk++uc0ifwLIw15Qh\\\\\\\/Df8RoSRJIpziZz23cklBWuhTvlGHsNqYeUt4gFUOaorjfWmdcSUfu1CSudUXBwBEE+\\\\\\\/q1yeEGDqB+1p+a91vbeaAF9pEk7BgYoY\\\\\\\/0aThNCXzR2ACsqOcnX+RtlreLTBOSOcZBxsMpLEmXzLlgzaorWjvuhjSRgYeXYgDfo3yCpG5seqgAnfIRiVls7wj0GLoUdxBuecHCWL9opfuEhxPz+do9nE3coSB4ndtzr283H9xdn44WotL5uRxG9K4QAlgW47hGlP0seo+LH1AFeMAwSPkz1gJ\\\\\\\/kgVtFU9uFI2Xrrnwh0l7343IKQegEcvhpCdT+sgsPrangk93qXKuFmax7nGy4esiz5pNaRCx3SfET2TR9hovaDzCECU8eePBxk2PVuFpsUVtx9mRDoH9njB1OWzE1ziY7a\\\\\\\/WHI0W+FRiZoMvgt7jISIb6RcM0E0z2i6hHSX5wY68m3MOWGCUt7DOhAtdWbEJHlZAxZmE3pRmhcjIukSE9pLt1h6P1lVk83CRK6oSlIsqkVakO6WQDpDdC\\\\\\\/prk3ajPYOSsYnr3Nxx9ixFpiioGmT1kKfzuDC3WyFfL\\\\\\\/un0j8CCHEtmXSk5Ky8++gJWYL\\\\\\\/T5DaIiKvi+UeoqVDbaT8wXvbLMxTqdq6e6Ff63TxQVuyCGi3bREYBVQ+1jghtxYb93VJLDiqRHAVPczvDdG2bVT5aNU1BIGYChw\\\\\\\/PzWaQoBvs9WgL9tc36M9z9NXrNHi+yH6cvd96ojVWDcmcvL+hOZkbQPGUhRJy07Wr2b9U+ISo\\\\\\\/biqXVRzG7tI4njKVGqWcVSxJoAIBaevCYZQ6aEYTxGliPo0s3wjvX74zn6Q1hfxn7kOW5PGnlBBbzkJDqfreCGG87vP8oPVWWmimRsk4SGE8HMtUIiuMExRBzzbNvohyhJyGhaQapa3woQaivn93VmFupo1esF5P7DIGwXPFez\\\\\\\/+zG0FNtdKrDc2p2JJ5g9DcRZM+VICh3THKZURIZBgSTUrjy7LtRdHLuX7R1t+uKbqAwmvyrF6r\\\\\\\/hjANqSQYrK923U\\\\\\\/ylTQyYHDloUPQwfvbCm\\\\\\\/pa2gfvi\\\\\\\/HkOySqN65iN66ICY3+kyU80tbGbXzH5f6JQuu5ZVlV+iQpkyQoFRht89uaaxTiSe7H5DfABHEyI93mR77GFPXzRfpycLjnLrdxhbSaBk6JS4M9DLOApHer+KMgcowEQEBAJKGklthV5aa4IWe+DO+nHtTcw3qep5WW3Vi1z90V\\\\\\\/X0QhMaL7CYcvVAqkveaSAGsWisuDr+y5uTk2STMMEEZYb4ToTUc3OPRtZJWd\\\\\\\/bSrq17C6XKQQGTtuYxz85uNpiZ14zV30jTdNGx\\\\\\\/0xQKE2WdgeiiErObSeevNuOLft2+NkCZI6z69F0EP3kJ5smfaCFNOVMfeMgeOUQtuDvEFHuNmn3xwzjtKER3rjjomYx\\\\\\\/JhBYSwCbMo6zxGQ3wgnSaIB1oo1A6UUUbS0AakhZQ+N8P4P7HyXDcGIa+LNaMOjIzfGfo\\\\\\\/4dhznF6V1yY95bxqBWadKRCNttlXX+j2N4CO2UIoj7oTFt23ARWt8I5RWGUd8rN+Iw0t9w2TOUzm\\\\\\\/wA9NjF2HAAXrppxqcvlBG\\\\\\\/u9me4Sdg9b1p7fZ+cvuxktEdaRa\\\\\\\/1MuHt5BGXNZAM2CJII4Bd+rXM8x42L5MtDbfM21DvS5ASZXirTd\\\\\\\/Lh7Ld5YModGUOtkxiFki5YMRtc71MpmwTm+QOr85ZPvQgVcMxyN9jxb8tYCmmWSrplt5f0HHiqz720Te\\\\\\\/7QNgdNOhnrFx9bMosY4cHD6TmkW4gyKW\\\\\\\/zKvjo\\\\\\\/CHkafUTyaKIYgpyD1VNnFFExo5B4QQRv7MWE1\\\\\\\/lfA3+wf9STTrzhyomKPRq4RY0pK6hVTG46z4k6ZEteWwsvdYqcng2vAFqFmrEhHa7Gyz2\\\\\\\/w7L4cdSORTLFObLgy023ItR+tleXzpTeIOPqrichfEYvz\\\\\\\/JCeNpffPDduC0MNjOburPAWicb4KmvkO0dmrATtiaH7Oxy21ETp7pdFSpqKc5+zK+yYYNOnyU\\\\\\\/JNlecuTzr1OmydJRjY86fdXd0XWor69AwehTUA8uDOXlBdGUKrnw5DeRvB5hjYYWLJ+75r7adkqRIAiaFoHpuQiJKnS6fbjD445dbYTRe0OtnYewfC\\\\\\\/qK1Y5T2u8mcuVaUP6IYDUsa50wTy1R9k5ImMUKhnF3ICZrU9hhiZY72JtUsDj+FZOfs5jqh3ZR2GCLNkyWo2ngJv85GuQ2XveX7BdFLVgR5WYmXD+yH2UY+Od\\\\\\\/9qyFJHGNe\\\\\\\/vUlVV9c6AGwXZ0b29rEmvG81y9pxNVuQ4kWRc3xOHjfpmi3GuT9r3iao0aX\\\\\\\/TAEpWV+XdGfzy2EdNanuy20GnTHkxVVf\\\\\\\/utwrgh8JJuQat3HU99pIJoUW3JjU0ZKz6hFw\\\\\\\/o8HbGflxkPxqKGC7qWt\\\\\\\/xhZZA25E+RqLdgY6fj98ooe9LV3vWkeeBX7t1t2WrROAwC39TQ7HkLqpyNMMOV9TXArmP3XpsoX2HBykcjq4mlKzVEeQoTR0j0rQ2gRIcQ2wRD77CPteiwc4Lulakaa7tJAEB3shpnYnYgGclDJhHDhfZHBq6S2joNzNSxbZUIvDGbdFU3jlkACdubeXvVEb80uDwRa0nkSsrhuHS8ZmLXTNXujNB3f54T1D70pff3hg7s34cBoYw2yVAjulw1GzxvEemJhXcD2x\\\\\\\/3COO\\\\\\\/4kvsi1NftCaLIyYr4tKHsM8oYxq3yZZpt\\\\\\\/Q9cfhK4eHv1phMxBGOByqjiJ9gBY20k0gEE+b02kUGuBrn7YL1FNt0oj5QxBNnqLqRGE\\\\\\\/4KKr6pvVX+9YOIy7pAE\\\\\\\/KVs0TUOHmBjbfFY65OCw4UVeY1iq5rBBlQzyTbqsiJP1RM4DmKRncOYFh0tVYPr51GEMNgW957Ku8pjfa84CiHUsoValgK7vMqfGyxAb1F+jjrakZqX2CgXFdywgIlz+dEmFIhMQZ2rH3c0SAxYmCV1R7OjuNZG1OCXqMsL\\\\\\\/Nd836Mpz7uqImUR42sS6aES6AcDaSjk6uNPEXb0WYg9y5bZQpEdutyKdcnmePCZvMPWNewannRHNBjLyCE7m504XqiuG6APtyp5f0aTwfrRvkgvcUXSlXw+aRyfEoI\\\\\\\/LhTRu4utGgcanGL4svBDTaRwomyGWNr02AZD+NPSaeLkvFeeRAogHB5OdDFRpfEGLyHY6CXJTaSE\\\\\\\/JMhQm5UeNRYk2\\\\\\\/795YkBSR80nMNA7EVWNqN61YufH4vesQd20e6V0eKxbD5hrvs5iil2gEmP1qK3jYd\\\\\\\/YLuV3IuPeQpAoJAhnxad8Sqbx6PUCP3WXkBwnjvbh5VD107BuD+8vfWwcd9RFaFBstLrVHE\\\\\\\/qdqeEji9cfn\\\\\\\/miVbSnvkxeyg6x1eq9yNv64mMLVZNW+nxnTWB5KTM0QtdSzzpzqYICIOpYU3rurFCVTnwlemoRoDCLbadDWeLNFR9VG2cOJzKYhsnHuLM8JkeHODXNlDKd50yB8L2OI0WKHEkrs9G01M6ga39lYZo4uiIWNKGrlIk0YjgEKmkZVlp8iTW3dv7QoT3zA0rzyPkChUFEs+aN4wnqwu9f2ZG2GjUBOUziLQBockLfbjmizPm9E6Pfu7CT+qEDnHCH+AqYNVmawaBjvNiv3C68mCePQGMN\\\\\\\/3Ym\\\\\\\/sP5WVk6gVSMdFLB\\\\\\\/\\\\\\\/UNM4GiC0xgUTkUSlDWVR\\\\\\\/uE96EfWPkDTII0RWbX4xmPj96BX+mUTOxZDoX5KMCkpq3iMU2L+yDKcSjUjCaFY5b4rse\\\\\\\/3N\\\\\\\/5xH1YFGQXV2AFVO2PrwuABCWp\\\\\\\/mAImj6+PauenCfxkJng2KuwCsqrQgqCPRXA5+GOHY5Vo7YTBru7LawFHKvUUaSwKQCbfBv6rvM9L9L0rny904KFAqohLOzcVAB5PGKNb6El31rTI+p7wN0DkosglRXnOvUa7H93rcs0fVUv2r3+rkGjcdZBk6lPfor6e0zF4Mn7eHTMQy5MxH+4dbYGwGw+X7NBr4Y6ysXg9d9XwBPqzQYWX6EWBzn1ZkyEUsz3ivg7\\\\\\\/vsmE8gUeJZsVERm2\\\\\\\/pfYmp1fF5jzJACHVz8M\\\\\\\/a8WNOTQPvCm5V4KIXZXevG+Apwhc\\\\\\\/bM8Zdh1zAO2OGKUuRUa98BfuYfj9Uut1Z3tJG7Buq7D2\\\\\\\/up9FADuPK23qre9ioAgXDbLEhv+tA+bq5XPSe8giqmKzbsqIFhJ9Ju3Plq3gGwhq0Q6TZ33AfANMx5k4f+DMSD9SaQmGa727kcCDak3xSRHm97iRGawhyJQeqbboI+SA0oCV88\\\\\\\/Z432FpOjlu7VLGVo4IxuIEti+QAuWNQFCoCvcgo6T3hbiRww5m+a6aMm+Hspa7RTituEqVQNOmBhBH8bTGj1Q1jCB4xrv1+GpId0lj8pbaSQXsLyVhzHQgNqcDqIt2ZqZetxYiyT167u\\\\\\\/GlQ8GJc95Nk0mEvrEZZ3VmkoMoYrkYW6fy6uNfxKBtNPqM9RPt+F9i4X8\\\\\\\/pQwaeISFpfToFr78w6Ac2f+2XBHhgbXI4KU+uh43DLaIAEhAVaMkcN7zYDWOOW7yF4wng5p1ORbN4xiiEobzHv5C+9A7No\\\\\\\/sKgNpxEKF9A+7jtqfd+qrtm77s4DcOFdba1JDnLpoqZ61NRk5gT44aDgSO\\\\\\\/dalQcIOb8IH2briqcWE2I3GMjlR1ZrG+pFGUlbwzD22kx3PMVyZKGTyLcb0ELpE4R65dbLUF\\\\\\\/S9wODOIiIdNKVQ2itZ6cMSwPsBoSUv95VL3YBIwsdGDkDtoaBpBdZd5IuRO8xcX1EKzpQOTScd0SceSuj3519W5873vSz7CZVZyTKJ0S3cxV9vk4Hr0lLnhHhoCwGOLo31t8jtecTC91ICgUC20id1XrwCqd8A+mKEYOraVWZZN8zlaf33HSyAbQdwuJNv87trqBfhq\\\\\\\/nnckFyjnwJ\\\\\\\/0Vvg9N20LBtbGVPgCENDNJ8yF3nx81zDBymwaTQyGO9C2tDZgzL1NL12ItU\\\\\\\/OMpdJNvB\\\\\\\/OQar6L\\\\\\\/mgBrMs6tUGUrwJlS8n4rtrgglzsTOzTpddsShZH4uu+YDLll83rvGD1PIhD9hQpDTJAMpoTV0Tl2YhlMZv3ropCmRnhg08BsAGUeynN0bicxycc+E+YCHjD8hwt2v0y3FFh0KKpkLCHQK0Fy8IHMrUqqh7HJZfYp64lrK+Z7Jck0bg1UGHPDAJS2F2Yg\\\\\\\/2WVNDPtrzoPV0h3PhB+GLBq86Xc2ZxqW1X72EKzKK+lufxf8B4cpEky+I2jtd+Nfp29lYyVWp9\\\\\\\/2AupFApq3SGvNNnYCmKoK9EWkzG7XnlPA7eywT8rmFNgHhPb49u2H\\\\\\\/AboCW4YCPYz1+imtwubM\\\\\\\/WWRF6CYuBAPIKrn2+3K+8bOpf3JUK8tFc1WrtmOy8Hwc70VZiwMS9J+ojIUz2iVAqEt3p6\\\\\\\/TShAURnuRBTbx3ErQlpcEoAQel48uDkEKtxp\\\\\\\/YjYt6MUvRMnMKOEzl8gwsE3WOipa3bHHFeJsVBzTtc8mw9QeASAlsSAqJbrg9EFOl33k89q6rS1pfsV0SWcvR7kS\\\\\\\/QiEM0F13J4Ly3ARauQGJ6hruSNc+4Zav5D4oWzJZIhRlohLYrFkBn75ABFNEfU1lQDG10tmeLcM6dWvwpx36+JLuz35sSp9\\\\\\\/MZnP8McJ+3dxZtglpkzDb\\\\\\\/nYW9gC\\\\\\\/XtysJHUnpDMa5kY8a8McO1jzyre1f+oMzbd6pTt1kZ3WikHmG+S0c\\\\\\\/QNwT9KY48z\\\\\\\/Y\\\\\\\/6WsLjerLa5wP\\\\\\\/6VbD4p89yz3AGIG+oNqVKqwq2R\\\\\\\/cSwsn4oqXW26s2X9Zw+9\\\\\\\/KIgN3hzXSsOU9BhD7jsA+AX5lS47sq9Z7pIwEs3d\\\\\\\/M4oAchtDBCUm+jIzK6Tov9J7ME2NP1XBwf2ggiich4UQ9gvHWmEVxYhj2EB38jnk2x\\\\\\\/9y7GV4N9ULm+3m3vh4BicSxoWSi5QQAxB0mDa\\\\\\\/+ORL7eRFOPcEdlCr5qVu0a5Dqaoy9oogs\\\\\\\/q2+K5n9xp7eDZox1hZOUrdEkKC0Wdyka2AweimzSvDodChR5F\\\\\\\/Gtf5wJewOrL16m9+NNXeDcAHQrQAyIbS121QRzPKiZcX3G65+i1VQL5zl8FUSKwgGpaiAkEvwk2dk2VMlHNmK63HbcNNiLSW3rItNfQlvSNszkW\\\\\\\/377IOcR+StPwidlHs1PgaHpXZHdNxr+0euLq\\\\\\\/bt2lNfyIUN1IEQKIWhlSdC9Q5ULrL\\\\\\\/jzSxsuvO9OKWBkKIVWCenUTDUxoOVdQr+oC9EuoO9TKCjmNV9R4NTQIEer6Q==\\\u0022,\\\u0022iv\\\u0022:\\\u0022i3IyRMrR7hFPnMC1x4o4Qw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022416fcf0de97dff0fabba4d9e79ed60eedbc7845c85a3ae19ece7639c4de9c80d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022B4lOEyO6+Qm1qzcpkXqx+8i2J+2Vb6Smji6qtqRJgj9QQeIEglnWKdPm65BrBX\\\\\\\/sfsLV+XbtSUWZTvLlb4J5y6BxRSMdTG6YniGQNeIYYL1gI17n6KzOXEr0VX66V\\\\\\\/0Woll8AfO9+nttZTU4BRXNPuwmVsvkv4GRi4BvCpadItUvkglnABhEKo1rh8wnxmNI8+Bui08kDuGrbd5ObkIhyCb+eQaNxw087jFBiU5ylkWXAWQwcsT2VxlMgQToAqldWjdj03nkXwh0LMgJ4gkzaGF9CKPVR+iwmuIJXVQ0Qyi5U98pm1IceA0pBfq5fvKR7MbMl4xjZ0OEYT4iiJW0rYBt3pofbabpWMUiU5wPqk4xOuqNNiQIj0YtCSU56UhVIQn6QyOq4jlSInxaf2wKJxA2JHLCE6+RzgF4eK34e0eM8PtC1yc3XAeAei4FKrfZXu+LIihbwzQcGmECF+Ma5KijvE5kKx6tr1Vs\\\\\\\/86tWgX\\\\\\\/GgmfKCcBUqEir31I6N8E3Dz+UYHWby8\\\\\\\/kFq4p5Vf+EYAHusnffeZDKKno+mXGHlyYOSFPsmx9dPjP7sPkRFgAgj8LFJpjfjssLyXtK9tCUYML7c8b2nz3BKYdhDNTdiMsrMWx2YMD85\\\\\\\/nxAY7a8EaLtpy9Mf2tBrDTaRZ+GfCNhrsjGcA4\\\\\\\/e9rnoVudgCjMdZUNolFHCDBVzNC1fq5YoOom53b5ZDksgLIkxG9eboV1YYK7p7QNvjzXoGG\\\\\\\/P6GuQNfDekY+zIiFEm29FrysmYcSmYQgylXr9XQiByHYq\\\\\\\/NVs0ZXLpkUcr+gURrRWqPtwzTFjlxXRi+bB51EJTU+fe5T7ZfJtMRII8RDLMhlAnc97FKB\\\\\\\/nq2+n+B1FZfKj+PUYBYQSUyM\\\\\\\/gYj0F5pY7RheO\\\\\\\/jpu0bP+r9KNYyyz0AuKIjOZtlbSJHfLNJL7inh9u3Ga6IiKKVcI9v0GzzGlrHAGL0wt912jxSLDXTOf815QLl7cYXUzRhLc8zRDdB99xhlCYXi8iEqFuYjXKpWrUL5GNYV1SfLkglSKqV73ASolwwexgYAWj8j1onO5CgvXxiOg5OVQxE5iDGSD6MDPC8taZRz4rAtQ7Hz+j4lhfcf4kPNWBB8dKUFx1GviEDTOKT8Mwxk0dGxDY9Tj0udzsvUmhngdT1wjoQkkmMWTS3b05pssPOG3575QKliLgKRJmXoZ0G4xIys9dFDhFg2TzccVZEeRxXpX42b3o73fcgThM03r5izc0Qm5Lsi1c3te+uzmbXleZlEpDBOBpM7eagVEzVvCOwVhmDtsJ+bEuDKQ1+0kDBq4IgvbcWSdKMe0msMqrW04+zCbRxdn0p8xQYgrUYe484DGm2IMp6xkBWg82Z0zac3hHhw95dQ5eotAah2JSiUj\\\\\\\/jqjbX1NoFtegYskKMxobX7mkr2ldt71UZn0gGiScuJyVkcwPCFBROqDU6wrx4\\\\\\\/48CPzKNq5ThisLfxTuWh\\\\\\\/UA\\\\\\\/OLGNRt8sqxKM4bqPcWqC3QEIX9jv7gRjkJqsnbo+ASQXyTBR+KGyfKNAR3ERmNxn43WMGXL0B2\\\\\\\/tZ3HjJaF6sHWpnZ4zi8leaUVo\\\\\\\/h0I+rTlca6WWXSzdzRNf577Mw79mJZYmP4ceuUfG7\\\\\\\/91Y0tcT5x1L1x+gcPwi8dCsUdkOSkNcJql08b8vP0xYY72Q6drzniIA+W0QIZ2lzoYjAopSsE6LdndXmWhVS3mmqXNvrCFw2YzYrdRvYf0pza9+Rbcjzb7+q7MvycfQxtHYMcYqZ6PtUybSVwd1XU2MpNYxhjJFrssDILRNmggdIeP6yJ7sRClS7nG7Q\\\\\\\/s68xfiPfX2lMfi0Ts3yDY4ukCzJN+sL44\\\\\\\/W5auv\\\\\\\/J4BKy38IwiSo9XzOCanHCPz\\\\\\\/rRYxNZ1r2MrFPqB4PxqMKaIOg+4Q+8NkjQoiKD2BW2BQuHsqqkVAmVjqmDT7PI3LUjbKw0kvK\\\\\\\/V7kM9UoYZTnR9iQEbB2D7BkPFCkbM+Oe5S1x46o6lBEvlcHaMB\\\\\\\/WbqA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/LfSufcd2FmOLfxYD\\\\\\\/82Fg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d637cb428f214cd751c214038490dc520319b9586e8b63630cc72c0d667c7d98\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460228 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/107" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5cpyfcg6w78YLlbpmEO5gJdE-dMCNkwB0ZDTB90R_5E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/107" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Nc98JzEB4O4Xhn5CxJSxm4tQhU4HxtYDa-WhD5ejq9M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/99" title="Using Ancient Samples in Projection Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/121" title="Disruption of Methionine Metabolism in Drosophila melanogaster Impacts Histone Methylation and Results in Loss of Viability" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460228" id="node460228" data-pisa="ggg;6/1/107" data-pisa-master="ggg;g3.115.024208" data-apath="/ggg/6/1/107.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5961-5064" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicolas Alcala</span>, <span class="highwire-citation-author" data-delta="1">Jeffrey D. Jensen</span>, <span class="highwire-citation-author" data-delta="2">Amalio Telenti</span> and <span class="highwire-citation-author" data-delta="3">Séverine Vuilleumier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">107-120;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024208 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460228" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas Alcala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Lausanne, Lausanne CH-1015, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BAlcala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alcala%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BAlcala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5961-5064" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicolas Alcala</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nalcala@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">nalcala@stanford.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey D. Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne CH-1015, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BD.%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BD.%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Amalio Telenti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genomic Medicine, The J. Craig Venter Institute, 4120 Capricorn Lane, La Jolla, California 92037</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amalio%2BTelenti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Telenti%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmalio%2BTelenti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Séverine Vuilleumier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Lausanne, Lausanne CH-1015, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne CH-1015, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Microbiology, University Hospital and University of Lausanne, Lausanne CH-1011, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S%C3%A9verine%2BVuilleumier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vuilleumier%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS%25C3%25A9verine%2BVuilleumier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/1/107" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460228" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460228/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/107.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460228" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460228/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/107.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460228" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460228/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/1/107.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460228" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460228/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/1/107.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460228" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460228/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Ease of worldwide travel provides increased opportunities for organisms not only to colonize new environments but also to encounter related but diverged populations. Such events of reconnection and secondary contact of previously isolated populations are widely observed at different time scales. For example, during the quaternary glaciation, sea water level fluctuations caused temporal isolation of populations, often to be followed by secondary contact. At shorter time scales, population isolation and reconnection of viruses are commonly observed, and such events are often associated with epidemics and pandemics. Here, using coalescent theory and simulations, we describe the temporal impact of population reconnection after isolation on nucleotide differences and the site frequency spectrum, as well as common summary statistics of DNA variation. We identify robust genomic signatures of population reconnection after isolation. We utilize our development to infer the recent evolutionary history of human immunodeficiency virus 1 (HIV-1) in Asia and South America, successfully retrieving the successive HIV subtype colonization events in these regions. Our analysis reveals that divergent HIV-1 subtype populations are currently admixing in these regions, suggesting that HIV-1 may be undergoing a process of homogenization, contrary to popular belief.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/admixture" class="hw-term hw-article-keyword hw-article-keyword-admixture" rel="nofollow">admixture</a></li><li class="kwd"><a href="/keyword/migration" class="hw-term hw-article-keyword hw-article-keyword-migration" rel="nofollow">migration</a></li><li class="kwd"><a href="/keyword/coalescent" class="hw-term hw-article-keyword hw-article-keyword-coalescent" rel="nofollow">coalescent</a></li><li class="kwd"><a href="/keyword/site-frequency-spectrum" class="hw-term hw-article-keyword hw-article-keyword-site-frequency-spectrum" rel="nofollow">site frequency spectrum</a></li><li class="kwd"><a href="/keyword/hiv" class="hw-term hw-article-keyword hw-article-keyword-hiv" rel="nofollow">HIV</a></li></ul><p id="p-3">Past population demographic events leave distinctive signatures in DNA sequence polymorphisms and gene genealogies (<a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Tajima 1989</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Simonsen <em>et al.</em> 1995</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Schneider and Excoffier 1999</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Excoffier 2004</a>; <a id="xref-ref-106-1" class="xref-bibr" href="#ref-106">Zeng <em>et al.</em> 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Gutenkunst <em>et al.</em> 2009</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Naduvilezhath <em>et al.</em> 2011</a>). For example, a population bottleneck increases the variance of coalescence times and reduces the number of variants at low frequency in the site frequency spectrum (SFS) (<a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">Tajima 1989</a>). Population expansion creates star-shaped genealogies and increases the mean number of variants at low frequency in the SFS (<a id="xref-ref-93-3" class="xref-bibr" href="#ref-93">Tajima 1989</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Slatkin 1996</a>). Population sub-division leads to long internal branches in the genealogy and increases the number of fixed variants (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Harpending <em>et al.</em> 1998</a>). It also results in “structured” genealogies in which the means and variances of coalescence times (<em>e.g.</em>, time to the most recent common ancestor) are large, resulting in an excess of intermediate frequency variants in the SFS (<a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Wakeley 1999</a>). Balancing selection leads to a similar structure, and to an excess of variants at intermediate frequency (<a id="xref-ref-93-4" class="xref-bibr" href="#ref-93">Tajima 1989</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fay and Wu 2000</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Barton and Etheridge 2004</a>; <a id="xref-ref-106-2" class="xref-bibr" href="#ref-106">Zeng <em>et al.</em> 2006</a>). Directional selection leads to genealogies with both star shapes (leading to the common ancestor of the selected lineages) and long branches (due to remaining ancestral lineages and/or recombination with the selected haplotype), and generates an excess of variants at both low and high frequency (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Barton 1998</a>).</p><p id="p-4">Consequently, DNA sequence polymorphisms are increasingly used to infer past demographic events and have successfully reconstructed past histories of many organisms, including humans (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Gravel <em>et al.</em> 2011</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Reich <em>et al.</em> 2012</a>). Some of the first methods proposed were developed to infer population growth rates (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Slatkin and Hudson 1991</a>) or for specific demographic scenarios (<em>e.g.</em>, <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Nielsen and Wakeley 2001</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Hey 2010</a>). Recently developed approaches allow for the inference of a wide range of demographic parameters (<em>e.g.</em>, <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Strimmer and Pybus 2001</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Beaumont <em>et al.</em> 2002</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Drummond <em>et al.</em> 2005</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Gutenkunst <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Excoffier <em>et al.</em> 2013</a>). However, inferring reconnection of diverged populations after isolation remains difficult, as models of past isolation with recent contact are challenging to distinguish from models of ancient population separation with a continuous exchange of migrants. Also, parameter inference requires <em>a priori</em> knowledge of a population split (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Hey 2010</a>), and the time at which migration event(s) occurred has thus far been difficult to estimate (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Sousa <em>et al.</em> 2011</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Strasburg and Rieseberg 2011</a>). <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Nielsen and Wakeley (2001)</a> first proposed a method to infer time of divergence (IM model), the migration rate between the two populations, as well as the relative sizes of the populations. This approach was extended to account for the divergence of multiple populations (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Hey 2010</a>) and temporal reduction of gene flow following an initial population split (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Wilkinson-Herbots 2012</a>). Though others have discussed the expected signature of a reconnection event following a past isolation period (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Becquet and Przeworski 2009</a>; <a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Sousa and Hey 2013</a>), detailed theoretical formalization is still lacking. Besides this, the use of the aforementioned methods for general inference is also challenging as they are difficult to apply when the number of sampled populations is large (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Beaumont <em>et al.</em> 2002</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Drummond <em>et al.</em> 2005</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Gutenkunst <em>et al.</em> 2009</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Lohse <em>et al.</em> 2011</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Excoffier <em>et al.</em> 2013</a>), and when multiple sampled time-points are available (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Excoffier <em>et al.</em> 2013</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Foll <em>et al.</em> 2014</a>). Further, they often rely on a specific evolutionary model, such as the Wright-Fisher model (<em>e.g.</em>, <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Gutenkunst <em>et al.</em> 2009</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Lohse <em>et al.</em> 2011</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Excoffier <em>et al.</em> 2013</a>) which might not be representative of, for example, the skewed offspring distributions found for marine species or viruses (<a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Tellier and Lemaire 2014</a>)</p><p id="p-5">Past population isolation and subsequent reconnection or secondary contact are common in natural populations and can have a drastic impact on species’ evolutionary histories. Anatomically, modern humans are thought to have admixed with Neanderthal populations after a period of isolation between the African and European continents (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Green <em>et al.</em> 2010</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Patterson <em>et al.</em> 2012</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Sankararaman <em>et al.</em> 2012</a>). Similarly, African and non-African populations of <em>Drosophila melanogaster</em> also experienced a period of isolation, followed by subsequent admixture (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Pool <em>et al.</em> 2012</a>). Domesticated species of both plants and animals are isolated from their wild relatives, and often experience subsequent genetic exchange (<em>e.g.</em>, in crops; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ellstrand <em>et al.</em> 1999</a>). Importantly, isolation and subsequent reconnection events are also common features in viral histories, including the Influenza A virus, Human Immunodeficiency Virus (HIV) and Human Cytomegalovirus (HCMV; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Renzette <em>et al.</em> 2013</a>), where dependency on the host cell and other features of the life cycle isolate viral populations within hosts, groups of hosts, or host species in-between infections. For example, reassortment between avian and human influenza viruses caused the pandemic outbreaks in 1957 and 1968, and a reassortment of swine, avian and human influenza viruses was responsible for the 2009 pandemic (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Hsieh <em>et al.</em> 2006</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Garten <em>et al.</em> 2009</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Flahault and Zylberman 2010</a>).</p><p id="p-6">As in influenza, events of HIV isolation and reconnection are common and can be observed at different spatial and temporal scales. Indeed, tracing the origin of HIV has confirmed that cross-species transmission of SIVs from other primates to humans, acting as sources of HIV, occurred several times over the past 100 years (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Liégeois <em>et al.</em> 2014</a>). The virus remained isolated for a very long period and adapted rapidly in humans, which ultimately rose to epidemic levels (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Korber <em>et al.</em> 2001</a>; <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Tebit and Arts 2011</a>). Circulating HIV populations in humans are strongly divergent. Two major types of HIV exist (HIV-1 and HIV-2) and both have diversified into several groups or subtypes (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Robertson <em>et al.</em> 2000</a>). The HIV pandemic is mainly caused by HIV-1 Group M, which is composed by divergent (up to 35% sequence divergence) subtypes (named A–D, F–H, J, and K, <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gaschen <em>et al.</em> 2002</a>). The HIV epidemic is associated with the worldwide movements of people that allowed HIV-1 subtype colonization events (<a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">Tebit and Arts 2011</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hamelaar <em>et al.</em> 2011</a>). In the course of their evolution in human populations, HIV-1 subtypes experienced several large-scale isolation events within both transmission risk groups (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Su <em>et al.</em> 2000</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Pang <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Han <em>et al.</em> 2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Li <em>et al.</em> 2013</a>) and countries (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Bello <em>et al.</em> 2011</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Castro-Nallar <em>et al.</em> 2012</a>). Subsequent reconnection then occurred through coinfection, superinfection, and recombination (<a id="xref-ref-96-3" class="xref-bibr" href="#ref-96">Tebit and Arts 2011</a>; <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">Vuilleumier and Bonhoeffer 2015</a>). Those reconnection events are the source of emergence of many new recombinant forms which constitute a major challenge for vaccine development and are at the origin of new epidemics (<a id="xref-ref-96-4" class="xref-bibr" href="#ref-96">Tebit and Arts 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Feng <em>et al.</em> 2013</a>). The multiple colonization events of HIV-1 that occurred in Asia and in South America are representative of this trend (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Hamelaar <em>et al.</em> 2011</a>). Indeed, with an estimated 4.8 million people living with HIV as of 2011, Asia is the second most HIV-affected region worldwide (<a id="xref-ref-107-1" class="xref-bibr" href="#ref-107">Zhang <em>et al.</em> 2013</a>). Historical epidemics in Asia are associated with transmission events to various risk groups; HIV-1 subtypes B and C were first detected in the mid-1980s and early 1990s, respectively, in the Injecting Drug Users (IDUs) risk group (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Lu <em>et al.</em> 2008</a>), and then spread to other populations. Subtype CRF01_AE was first detected in the Men who have Sex with Men (MSM) risk group in the 1990s, then spread to other groups and became the most prevalent HIV-1 subtype in many parts of China. Similarly, the epidemic clade circulating in South America is derived from subtype B viruses that migrated out of Haiti around 1969 (1966–1972) and spread through the world (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Bello <em>et al.</em> 2011</a>). Later, a single-founder subtype F1 strain, introduced in Brazil, spread and recombined with local subtype B viruses to form the HIV-1 BF1 and F1 epidemics (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Aulicino <em>et al.</em> 2007</a>).</p><p id="p-7">The present study identifies specific temporal signatures of reconnection of diverged populations in two broadly used summaries of DNA polymorphism: the distribution of pairwise nucleotide differences (via coalescent theory), and the Site Frequency Spectrum (SFS, via simulation). We also describe how past isolation can influence commonly used SFS-based test statistics (Tajima’s <em>D</em>, Fu and Li’s <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-1.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-2.gif"/></noscript></span></span>, Fay and Wu’s <em>H</em>, Zeng <em>et al.</em>’s <em>E</em>), and discuss the specificity and the robustness of the signature relative to other demographic and selective processes. Then, using theoretical results and full genome polymorphism data, we reconstruct the successive invasions of HIV-1 subtypes in China and in South America and analyze the subsequent dynamics of genome composition. We find extensive admixture between HIV-1 subtypes, potentially leading in the long term to homogenization of HIV-1 genomes in these regions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-8">Using coalescent theory, we first present methods for detecting the signature of population reconnection after isolation on the distribution of pairwise nucleotide differences. Then, in the second section, we use coalescent simulations to derive the signature of population reconnection after isolation on the SFS. The third section highlights the robustness of these signatures to alternative demographic scenarios. Finally, the fourth section presents a data application for these methods: inference of recent HIV-1 evolution in China and in South America.</p><div id="sec-2" class="subsection"><h3>Theoretical signature</h3><p id="p-9">In our model, we consider <em>d</em> previously connected populations that became isolated at time <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-3.gif"/></noscript></span></span> and then reconnected at time <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-4.gif"/></noscript></span></span>. The scaled migration rate <em>M</em> between populations (<em>i.e.</em>, twice the number of migrant genes per population per generation) is the same before and after the isolation period and mutations follow a Poisson process of rate <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-5.gif"/></noscript></span></span> (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Kimura 1969</a>). We used an infinite sites model for the analytical investigations, and for the simulations we considered a finite sites model with a number of sites corresponding to HIV-1 genome size (9719bp, from the reference HXB1 sequence). Our developments are based on the Wright-Fisher model (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Fisher 1930</a>; <a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Wright 1931</a>), where each population has a constant size <em>N</em>. For analytical simplicity, we considered that times <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-7.gif"/></noscript></span></span> are scaled in units of <em>N</em> generations, so <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-8.gif"/></noscript></span></span> corresponds to an isolation event that occurred <em>N</em> generations ago. We derived an exact theoretical formula for the distribution of pairwise nucleotide differences in nonrecombining genomic regions (<em>Appendix A</em>), and simulated this distribution using coalescent simulations (software <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-9.gif"/></noscript></span></span>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Hudson 2002</a>) in the case of recombining regions. We computed the power to sample sequences from each mode of the distribution using equations A.6a–A.6b (Supporting Information, Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-10">We generated the SFS through time following isolation and reconnection events using coalescent simulations. We studied the <em>total SFS</em> (considering pooled samples from all populations); in the main text, we present results for equal sample sizes in each populations, and we present results for alternative sampling schemes (Figures B and C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). We also studied the joint SFS between pairs of populations. We assessed the temporal changes of the total SFS using the visual test of <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Nawa and Tajima (2008</a>), which is based on the difference between the expected SFS under neutrality in a population with constant size, and the observed SFS. For each frequency <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-10.gif"/></noscript></span></span>, where <em>n</em> is the sample size, the transformation represents <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-11.gif"/></noscript></span></span>, instead of the corresponding number of variants <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-12.gif"/></noscript></span></span>. This simple test enables one to easily detect which frequencies present an excess or a deficit of variants. We also derived an optimal test statistic to detect the signature of population reconnection from the SFS (Text A and Figure D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>) and compute the power of common neutrality tests to detect such events (Figure E in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-3" class="subsection"><h3>Robustness analysis of the theoretical signature</h3><p id="p-11">We then analyzed the sensitivity of our results to departures from the infinite sites model (Figure F in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>), from the constant population size model (Figure G in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>) and from the Wright-Fisher model (Figure H in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). To account for repeated bottlenecks (<em>e.g.</em>, during transmission events), extinction-recolonization dynamics, and rapid adaptation, we simulated the distribution of pairwise nucleotide differences under a scenario of reconnection of isolated populations under a beta-coalescent model. Beta-coalescent models have recently been proposed to model the genealogies of many organisms with a large variance in the number of offspring per individual (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Birkner and Blath 2008</a>; <a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Tellier and Lemaire 2014</a>), in particular viruses (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Neher and Hallatschek 2013</a>). We extended the algorithm from Birkner and Blath (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Birkner and Blath 2008</a>) – which generates samples under a beta-coalescent at equilibrium – to include an island model of migration (each lineage migrates to another population at rate <em>M</em>) and an isolation period. We assumed that coalescence processes follow the beta-coalescent at all times: before isolation, during isolation and after reconnection. In addition to mutation and migration rates, beta-coalescents rely on a parameter α (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-13.gif"/></noscript></span></span>) that gives the probability of multiple lineages coalescing at the same time. We present the results for four different parameter values: <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-14.gif"/></noscript></span></span> (equivalent to the classic Kingman coalescent), <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-15.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-16.gif"/></noscript></span></span> [corresponding to the Bolthausen-Sznitman coalescent, used to model strong positive selection (<em>e.g.</em>, <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Neher and Hallatschek 2013</a>)], and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-17.gif"/></noscript></span></span>.</p><p id="p-12">We also compared the signature of population reconnection after isolation with that of a closely related model: the Isolation with Migration model [IM (<em>e.g.</em>, <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Takahata 1995</a>; <a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Wakeley 1996</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Rosenberg and Feldman 2002</a>)], using coalescent simulations (Figure I in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-4" class="subsection"><h3>HIV data analysis</h3><p id="p-13">We used 1646 whole genome HIV-1 samples from subtypes B, C, F1 and CRF01_AE and recombinant forms between these subtypes (Table A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). We used the subset of genomes from China sequenced between 2005 and 2009 (297 sequences; Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). For the study in South America (Brazil and Argentina), we used data available between 1999 and 2005 (128 sequences; Table C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Finally, for the worldwide PCA analysis, we used all 1646 genomes available.</p><p id="p-14">For the Chinese and South American data, we identified HIV-1 populations (subtype clusters) by Discriminant Analysis of Principal Components (DAPC) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Jombart <em>et al.</em> 2010</a>). Our investigations considered dominant subtypes B, C, CRF01_AE and F1 and their associated recombinants. Each cluster encompassed a dominant subtype and some recombinant forms between subtypes. We found strong support for four groups using the Bayesian Information Criterion (ranging from 2 to 100 PCA axes) that are independent of sampling year (See Figure J in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a> for a neighbor joining tree of the sequences where the different clusters and sampling years are represented). We then computed the distribution of pairwise nucleotide differences and the total and joint SFS for the major circulating HIV-1 subtypes in China and South America.</p></div><div id="sec-5" class="subsection"><h3>Computing the temporal patterns of genetic variation of HIV subtypes</h3><p id="p-15">We computed the distribution of pairwise nucleotide differences, and the total and joint SFS for the major circulating HIV-1 subtypes in China and South America. The distribution of pairwise nucleotide differences within (within-population pairwise differences) and between clusters (between-population pairwise differences) were generated for each nonrecombinant block between subtypes that we identified with <em>jpHMM</em> (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Schultz <em>et al.</em> 2009</a>). In total, 19 and 24 blocks were identified in sequences sampled in China and South America, respectively; the 5%, median and 95% quantiles of block length were 111bp, 220bp and 1333bp, respectively. Detection of bimodality in the distribution, the signature of a past isolation event, was performed by fitting a Gaussian Mixture Model using the Expectation-Maximization algorithm from <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Meng and Rubin (1993</a>) as implemented in the R package <em>mixtools</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Benaglia <em>et al.</em> 2009</a>). Gaussian distributions are used to estimate positions of the mode as distributions are symmetric (the 5% and 95% quantiles of the skewness of all components are -1.05 and 1.05, respectively). The algorithm also provides posterior probabilities of membership into modes, which is used to identify genomes that have bimodal distribution (<em>i.e.</em>, indicating admixture). We analyzed the signatures in the pairwise nucleotide differences by comparing HIV genome signatures with the expected results, assuming either small linked sequences or large recombining sequences.</p><p id="p-16">The total and joint SFS were computed using subtype J as an outgroup. Results were robust to the chosen outgroup (Figure K in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Note that the distinction between recombining and nonrecombining sequences does not apply for the SFS, as the expectation (and the maximum likelihood estimate) of the SFS is not affected by recombination: only its variance is affected (<a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Gutenkunst <em>et al.</em> 2009</a>). We tested the significance of the temporal changes of the SFS using a bootstrap test (Figure L in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-17">All data used are from the Los Alamos HIV database (<a href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a>) (Table A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Bimodality in pairwise nucleotide differences as a signal of population reconnection after isolation</h3><p id="p-18">Two distinct signatures of population reconnection after isolation were found in the temporal distribution of pairwise nucleotide differences: bimodality and increased variance of the first mode (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Bimodality was observed in these distributions (both within and between populations, <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-18.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-19.gif"/></noscript></span></span>) only when a population reconnection occurred (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). This bimodality reflects the different possible origins of the two sequences sampled: the two sequences either have a recent common ancestor (<em>i.e.</em>, after the reconnection event; first mode in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>) or an ancient common ancestor (before the isolation event; second mode in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>). Bimodality was also observed in the distributions of total pairwise nucleotide difference (considering all populations, π) after a reconnection event but, in this case, bimodality was also observed when populations were completely isolated (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-20.gif"/></noscript></span></span>, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, B and C</a>). To differentiate a connection from an isolation event, bimodality needs to be associated with a temporal increase of the variance of the modes in the distribution (which depends on the recombination rate; see <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). Although testing for increase in variance requires data from different time-points, it has the advantage of not requiring <em>a priori</em> knowledge of the origin of the samples (<em>i.e.</em>, being within or between populations).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/107/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Signature of reconnection of previously isolated populations on the distribution of pairwise nucleotide differences as a function of time since the reconnection event  (columns), (A) within- and between-population , (B&#x2013;C) total &#x3C0; pairwise nucleotide differences. In panel A, we consider a sequence of size G, with G larger than the number of segregating sites without recombination; plots in panel A are computed using equation A.5. In panels B and C, we consider a sequence of 9719 bp (length of the reference sequence for HIV-1, HXB1) with various mean numbers of recombination events per generation,  and . Nucleotide differences are considered from a sample of 16 sequences and each plot represents the mean distribution over 2000 replicate simulations. Parameters are  populations, duration of the isolation period , scaled migration rate , scaled mutation rate ." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039178009" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Signature of reconnection of previously isolated populations on the distribution of pairwise nucleotide differences as a function of time since the reconnection event &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-21&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (columns), (A) within-&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-22&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and between-population &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-23&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, (B&#x2013;C) total &#x3C0; pairwise nucleotide differences. In panel A, we consider a sequence of size &lt;em&gt;G&lt;/em&gt;, with &lt;em&gt;G&lt;/em&gt; larger than the number of segregating sites without recombination; plots in panel A are computed using equation A.5. In panels B and C, we consider a sequence of 9719 bp (length of the reference sequence for HIV-1, HXB1) with various mean numbers of recombination events per generation, &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-24&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-25&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Nucleotide differences are considered from a sample of 16 sequences and each plot represents the mean distribution over 2000 replicate simulations. Parameters are &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-26&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; populations, duration of the isolation period &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-27&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, scaled migration rate &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-28&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, scaled mutation rate &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-29&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/107/F1.medium.gif" width="386" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/1/107/F1.medium.gif" width="386" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/107/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/107/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467337" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Signature of reconnection of previously isolated populations on the distribution of pairwise nucleotide differences as a function of time since the reconnection event <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-21.gif"/></noscript></span></span> (columns), (A) within-<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-22.gif"/></noscript></span></span> and between-population <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-23.gif"/></noscript></span></span>, (B–C) total π pairwise nucleotide differences. In panel A, we consider a sequence of size <em>G</em>, with <em>G</em> larger than the number of segregating sites without recombination; plots in panel A are computed using equation A.5. In panels B and C, we consider a sequence of 9719 bp (length of the reference sequence for HIV-1, HXB1) with various mean numbers of recombination events per generation, <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-24.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-25.gif"/></noscript></span></span>. Nucleotide differences are considered from a sample of 16 sequences and each plot represents the mean distribution over 2000 replicate simulations. Parameters are <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-26.gif"/></noscript></span></span> populations, duration of the isolation period <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-27.gif"/></noscript></span></span>, scaled migration rate <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-28.gif"/></noscript></span></span>, scaled mutation rate <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F1/embed/mml-math-29.gif"/></noscript></span></span>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Population reconnection after isolation can thus be detected by testing for bimodality and a temporal increase of the variance of the modes. Power to detect bimodality, with high probability (<span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-30.gif"/></noscript></span></span>) (Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>), depends on the sample size, the duration of the reconnection period, and the amount of gene flow among populations (scaled migration rate). As shown in Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>, it is easier to detect a reconnection event from the distribution of pairwise nucleotide differences between populations than within populations. Indeed, the second mode in the distribution of pairwise nucleotide differences was larger between population (<span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-31.gif"/></noscript></span></span>) than that of within population (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-32.gif"/></noscript></span></span>, <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a>). This difference in mode size increases when migration is weak or intermediate (<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-33.gif"/></noscript></span></span>) and the reconnection event is not recent (Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Ancient reconnection events can be detected even with small sample sizes (<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-34.gif"/></noscript></span></span>) when migration is moderate or strong (<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-35.gif"/></noscript></span></span>; Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). To evaluate the temporal increases in variance of the mode, no further development is required as the Bartlett test (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bartlett 1937</a>) can be used.</p><p id="p-21">Thus, we show that distinctive signatures of past isolation and reconnection events can be identified on the distribution of the number of nucleotide differences between pairs of genomes sampled (within, between, or in the total population). We also determine the parameter ranges for which this signature can be detected (sample size, migration rate, and time since reconnection).</p></div><div id="sec-9" class="subsection"><h3>The signature of population reconnection after isolation on the Site Frequency Spectrum (SFS)</h3><p id="p-22">When a population is at mutation-drift equilibrium, the distribution of its alleles, or SFS, has a characteristic shape (dotted line <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). When populations experience demographic changes, departures from this characteristic shape are expected and the size and the position of those changes are informative as to the demographic change. Consistent with the expectation, we found a large excess of variants at intermediate frequencies on the SFS following reconnection events between previously isolated populations (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). This excess was due to exchanges of variants among populations that were once fixed in one or several populations during isolation. The size and the position of this excess is indicative of the number of populations that have reconnected and of the timing of the reconnection event (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>), and can be identified by the visual test of equilibrium neutrality proposed by <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Nawa and Tajima (2008</a>). We predict that in the total SFS (SFS obtained with merged samples from all populations), shortly after a reconnection event one would expect narrow, high peaks of specific variants. These peaks are found at frequencies <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-36.gif"/></noscript></span></span>, where <em>d</em> is the number of sampled populations and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-37.gif"/></noscript></span></span>, when populations have equal sample sizes. In <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2A</a>, peaks can be seen at frequencies <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-38.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-39.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-40.gif"/></noscript></span></span>). With unbalanced sample sizes, the number of observed peaks can be higher (up to <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-41.gif"/></noscript></span></span> peaks; see Figure B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/107/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Signature of a reconnection event between previously isolated populations on (A) the total and (B) joint site frequency spectrum (SFS), as a function of the number of generations since the reconnection event, . (A) The SFS representation of Nawa and Tajima (2008) is used; the expected value of a single panmictic population at equilibrium is given by a straight line (dashed line), and deviations from a straight line indicate an excess (in blue) or deficit (in red) of variants at a given frequency. Parameters are  populations,  sampled genes per population (see Figures B and C in File S1 for alternative sampling schemes), with . Means are over 2000 replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039178009" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Signature of a reconnection event between previously isolated populations on (A) the total and (B) joint site frequency spectrum (SFS), as a function of the number of generations since the reconnection event, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-42&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. (A) The SFS representation of Nawa and Tajima (2008) is used; the expected value of a single panmictic population at equilibrium is given by a straight line (dashed line), and deviations from a straight line indicate an excess (in blue) or deficit (in red) of variants at a given frequency. Parameters are &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-43&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; populations, &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-44&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; sampled genes per population (see Figures B and C in File S1 for alternative sampling schemes), with &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-45&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. Means are over 2000 replicates.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/107/F2.medium.gif" width="440" height="245"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/1/107/F2.medium.gif" width="440" height="245"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/107/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/107/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467349" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Signature of a reconnection event between previously isolated populations on (A) the total and (B) joint site frequency spectrum (SFS), as a function of the number of generations since the reconnection event, <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F2/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F2/embed/mml-math-42.gif"/></noscript></span></span>. (A) The SFS representation of <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Nawa and Tajima (2008</a>) is used; the expected value of a single panmictic population at equilibrium is given by a straight line (dashed line), and deviations from a straight line indicate an excess (in blue) or deficit (in red) of variants at a given frequency. Parameters are <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F2/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F2/embed/mml-math-43.gif"/></noscript></span></span> populations, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F2/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F2/embed/mml-math-44.gif"/></noscript></span></span> sampled genes per population (see Figures B and C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a> for alternative sampling schemes), with <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F2/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F2/embed/mml-math-45.gif"/></noscript></span></span>. Means are over 2000 replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The temporal patterns in the joint SFS (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a>) are also strongly informative as to the occurrence of reconnection events and as to the amount of gene flow among populations since the reconnection event. When populations are isolated, each has its own segregating variants. This translates, in the joint SFS, into variants being restricted to the axes of the joint SFS [classes (20,<em>i</em>), (<em>i</em>,20), with <em>i</em> ranging from 1 to 20 in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B</a>]. Then, following the reconnection event, gene flow distributes the variants among populations to the point that they occur at similar frequencies in each population and the genetic compositions of populations are fully homogenized. Consequently, in the joint SFS, several variants move progressively away from the axes toward the diagonal of the joint SFS, as expected in a structured population at equilibrium (last column <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a>).</p><p id="p-25">To robustly detect the excess of variants on the SFS generated by a reconnection event, we derived an optimal test statistic, denoted <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-46.gif"/></noscript></span></span>, using developments provided by <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ferretti <em>et al.</em> (2010)</a> and <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Achaz (2009)</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Although use of this test is restricted to recent reconnection events that follow long periods of isolation, it performs better than commonly used statistics such as Tajima’s <em>D</em> (<a id="xref-ref-93-5" class="xref-bibr" href="#ref-93">Tajima 1989</a>), Fu and Li’s <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-47.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-48.gif"/></noscript></span></span> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Fu and Li 1993</a>), Fay and Wu’s <em>H</em> (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Fay and Wu 2000</a>), Zeng <em>et al.</em>’s <em>E</em> (<a id="xref-ref-106-3" class="xref-bibr" href="#ref-106">Zeng <em>et al.</em> 2006</a>). Indeed, even with long isolation periods, those statistics have less than 60% power to detect reconnection events (Figure E in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). The results presented in this section apply to both recombining and nonrecombining sequences, as the expected SFS is the same in both cases (<a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Gutenkunst <em>et al.</em> 2009</a>). Also, the power estimates from the tests based on nonrecombining sequences are conservative as they correspond to the case with the highest variance (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Achaz 2009</a>) and thus the lowest power.</p><p id="p-26">Thus, we show that distinctive signatures of past isolation and reconnection events can be identified in the SFS as well as in the joint SFS. Namely, the SFS has an excess of variants, the size and position of which are indicative of the number of previously isolated populations and of the timing since the reconnection event. The temporal dynamic of the joint SFS is informative as to the level of gene flow between populations.</p></div><div id="sec-10" class="subsection"><h3>Robustness of identified signatures to alternative demographic scenarios</h3><p id="p-27">The signatures of population reconnection identified here, <em>i.e.</em>, bimodality and temporal increase of variance of peaks in the SFS, are robust to population expansion (Figure G in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>), alternative evolutionary and population dynamic models such as repeated bottlenecks, rapid extinction-recolonization dynamics, recurrent positive selection or highly skewed offspring distributions (Figure H in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Moreover, the signature is distinct from the signature of a closely related model: the Isolation with Migration model (IM) (Figure I in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-28">The observed signatures are robust even when a massive population expansion occurs concomitantly with the reconnection event (Figure G in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Such events modify the expected signature only by presenting an excess of low frequency variants compared to what is expected with populations of constant size (as expected from the results of Tajima (<a id="xref-ref-93-6" class="xref-bibr" href="#ref-93">Tajima 1989</a>) in a single panmictic population). Replacing the classic Kingman coalescent model (Wright-Fisher model) with a beta-coalescent model does not alter our results for values of the model parameter α (Figure H in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). Beta-coalescent models account for large variance in the number of offspring per individual, as expected under repeated bottlenecks, fast extinction-recolonization dynamics, or recurrent beneficial fixations (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Neher and Hallatschek 2013</a>; <a id="xref-ref-97-3" class="xref-bibr" href="#ref-97">Tellier and Lemaire 2014</a>). Such variance leads to an excess of low frequency variants and smaller peaks in the total SFS compared to Kingman’s coalescent model; this effect is larger for small α values (light-gray lines Figure H in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). However, having an excess of variants at low frequency does not alter the identified signature of reconnection on the total SFS (dynamics of the peaks). Also the temporal dynamics of the signature distinguish this pattern from that of an IM model. Indeed under the IM model, the size of the peaks in the SFS increases with time (Figure I.B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>), whereas the signature of a reconnection of isolated populations is characterized by a decrease in peak size with time (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2A</a>).</p><p id="p-29">Thus, we show that, even though the signature at specific time-points may be confounded with that of other selective or demographic processes, the temporal signature of past isolation and reconnection identified is unlikely to be confounded, and provides a robust signal.</p></div><div id="sec-11" class="subsection"><h3>Detection of past isolation and current reconnection of HIV-1 subtypes</h3><p id="p-30">We applied the conceptual and theoretical approaches via an analysis of the colonization history of different HIV-1 subtypes in Asia and in South America. Using data from the Los Alamos HIV database, we detected a strong signal of reconnection events between previously isolated HIV-1 subtypes in these populations.</p><p id="p-31">Past isolation between HIV-1 subtypes in China and South America is signaled by the excess of variants, with peaks at intermediate frequencies in the total SFS (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and B and G and H</a>). These peaks reflect the presence of variants from the sampled population that diverged during isolation and were present at high frequencies in each previously isolated subtype population identified with cluster analysis (see <em>Results</em>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/107/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Signatures of HIV-1 subtype reconnection events on the Site Frequency Spectrum (SFS). Results are presented for China (left major panel) and for South America (right major panel). Two time points of HIV-1 sequences sampled are considered in China (Table B in File S1) and in South America (Table C in File S1) (see method section &#x2018;HIV-1 genome analysis&#x2019;). The total (pooled) SFS is presented in the first row following representation of Nawa and Tajima (2008), the joint SFSs of each pairwise group of HIV subtypes are presented in the second, third, and fourth row and, in the fifth row, the Principal Component Analyses (PCAs) are presented. Subtype J is used as an outgroup (see Figure K in File S1, results for alternative outgroups). Original subtypes (subtypes found early in the epidemics in Africa; at the corners) are used to define a triangle in the PCA space. The position of the HIV-1 genomes on the straight edges of the triangle suggests strong admixture between subtypes. For the total and joint SFS, we consider the mean value of 500 random samples of sequences in each subtypes cluster (see Figure J in File S1 for the different clusters). A sample of 16 sequences per subtype cluster is considered in China, and of eight sequences per subtype cluster in South America. The significance of the trends in the dynamics is tested using the optimal test statistic  derived in the SI (Figure L in File S1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039178009" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Signatures of HIV-1 subtype reconnection events on the Site Frequency Spectrum (SFS). Results are presented for China (left major panel) and for South America (right major panel). Two time points of HIV-1 sequences sampled are considered in China (Table B in File S1) and in South America (Table C in File S1) (see method section &#x2018;HIV-1 genome analysis&#x2019;). The total (pooled) SFS is presented in the first row following representation of Nawa and Tajima (2008), the joint SFSs of each pairwise group of HIV subtypes are presented in the second, third, and fourth row and, in the fifth row, the Principal Component Analyses (PCAs) are presented. Subtype J is used as an outgroup (see Figure K in File S1, results for alternative outgroups). Original subtypes (subtypes found early in the epidemics in Africa; at the corners) are used to define a triangle in the PCA space. The position of the HIV-1 genomes on the straight edges of the triangle suggests strong admixture between subtypes. For the total and joint SFS, we consider the mean value of 500 random samples of sequences in each subtypes cluster (see Figure J in File S1 for the different clusters). A sample of 16 sequences per subtype cluster is considered in China, and of eight sequences per subtype cluster in South America. The significance of the trends in the dynamics is tested using the optimal test statistic &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-49&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; derived in the SI (Figure L in File S1).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/107/F3.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/1/107/F3.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/107/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/107/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467366" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Signatures of HIV-1 subtype reconnection events on the Site Frequency Spectrum (SFS). Results are presented for China (left major panel) and for South America (right major panel). Two time points of HIV-1 sequences sampled are considered in China (Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>) and in South America (Table C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>) (see method section ‘HIV-1 genome analysis’). The total (pooled) SFS is presented in the first row following representation of <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Nawa and Tajima (2008</a>), the joint SFSs of each pairwise group of HIV subtypes are presented in the second, third, and fourth row and, in the fifth row, the Principal Component Analyses (PCAs) are presented. Subtype J is used as an outgroup (see Figure K in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>, results for alternative outgroups). Original subtypes (subtypes found early in the epidemics in Africa; at the corners) are used to define a triangle in the PCA space. The position of the HIV-1 genomes on the straight edges of the triangle suggests strong admixture between subtypes. For the total and joint SFS, we consider the mean value of 500 random samples of sequences in each subtypes cluster (see Figure J in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a> for the different clusters). A sample of 16 sequences per subtype cluster is considered in China, and of eight sequences per subtype cluster in South America. The significance of the trends in the dynamics is tested using the optimal test statistic <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F3/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F3/embed/mml-math-49.gif"/></noscript></span></span> derived in the SI (Figure L in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">The reconnection of HIV-1 subtype populations in China and South America is indicated by the bimodality in the distribution of pairwise nucleotide differences between-populations (considering sequences of small length <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We found that a large proportion of the genome of HIV-1 displays bimodality (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). In China, this proportion was initially low but rapidly increased through time. Between 2006 and 2007, the proportion of bimodality increased fourfold (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>), and increased a further 40% between 2008 and 2009. This suggests that the time series analyzed here captures a burst of HIV-1 subtype admixture. The dynamics of genomic admixture between HIV-1 subtypes in China is also highlighted by the temporal changes in the distribution of pairwise nucleotide differences and the SFS. The peaks formed by the excess variants (resulting from population admixture in the total SFS) decreased in size, and the values of optimal test statistics <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-50.gif"/></noscript></span></span> significantly decreased (Figure L in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). We find a significant increase in the variance of the first mode in the distribution of pairwise nucleotide differences between 2005 and 2009 (<span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-51.gif"/></noscript></span></span> two-sided Bartlett test). The joint SFSs for the two time points also indicate that the reconnection event and genome admixture is either recent or that genome admixture is limited but ongoing. Indeed, we can observe the excess of variants along the axes of the joint SFS shifting to the diagonal (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, C and D</a>). In South America, the proportion of bimodality in HIV-1 samples from 1999 is large (60%) and remains approximately constant from 1999 to 2004, signaling a prior reconnection event. Patterns observed in the joint SFS suggest that HIV-1 subtype genomes are well homogenized overall; further, this process of homogenization seems to have either slowed or ceased. Indeed, no increase of variance in the modes of the distribution of pairwise nucleotide differences was detected (two-sided Bartlett test), and the total SFS shows only small changes between 2001 and 2005, with nonsignificant changes in values of optimal test statistics <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-52.gif"/></noscript></span></span> (Figure L in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/107/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A&#x2013;B) Proportion of bimodality in pairwise nucleotide differences (between populations) detected in the genomes and (C&#x2013;D) distribution of total pairwise nucleotide differences (all populations) of HIV-1 sequences for different time points in China (see Table B in File S1) and South America (see Table B in File S1). As expected under genomic admixture (Figure 1), the proportion of bimodality detected in the pairwise nucleotide differences between populations in HIV-1 genomes increased, and the variance of modes significantly increased with time in China (, two-sided Bartlett test). We also see a (nonsignificant) trend of temporal changes in South America." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039178009" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A&#x2013;B) Proportion of bimodality in pairwise nucleotide differences (between populations) detected in the genomes and (C&#x2013;D) distribution of total pairwise nucleotide differences (all populations) of HIV-1 sequences for different time points in China (see Table B in File S1) and South America (see Table B in File S1). As expected under genomic admixture (Figure 1), the proportion of bimodality detected in the pairwise nucleotide differences between populations in HIV-1 genomes increased, and the variance of modes significantly increased with time in China (&lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-53&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, two-sided Bartlett test). We also see a (nonsignificant) trend of temporal changes in South America.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/107/F4.medium.gif" width="440" height="166"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/1/107/F4.medium.gif" width="440" height="166"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/107/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/107/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467336" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-34" class="first-child">(A–B) Proportion of bimodality in pairwise nucleotide differences (between populations) detected in the genomes and (C–D) distribution of total pairwise nucleotide differences (all populations) of HIV-1 sequences for different time points in China (see Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>) and South America (see Table B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). As expected under genomic admixture (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1</a>), the proportion of bimodality detected in the pairwise nucleotide differences between populations in HIV-1 genomes increased, and the variance of modes significantly increased with time in China (<span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F4/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F4/embed/mml-math-53.gif"/></noscript></span></span>, two-sided Bartlett test). We also see a (nonsignificant) trend of temporal changes in South America.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">Genome admixture in China and in South America is also visible when the sampled genomes are projected together in a PCA space defined by original subtypes <em>i.e.</em>, which originate from the beginning of the HIV epidemic in Africa (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, E and F and K and L</a>). Several HIV-1 genomes are present on the straight edges of the triangles defined by the ones at the origin. Temporal PCA patterns also suggest homogenization between subtypes B and C in China and between B and F1 in South America, as they move away from the pure subtypes in PCA space.</p><p id="p-36">To provide insights on the generality and global importance of these results, we performed a PCA analysis of worldwide genomes from HIV-1 subtypes B, C, F1, and CRF01_AE (<a href="http://www.hiv.lanl.gov">http://www.hiv.lanl.gov</a>). Again we projected them in the PCA with original subtypes B, C, F1, and CRF01_AE as references (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Consistent with the observations for Chinese and South American samples, we observed that HIV-1 genomes are positioned on the straight edges of the triangle formed by original subtypes, positioned at the corners, forming a continuum along the axes rather than discrete clusters within. Very few genomes are localized outside the edges of the triangle defined by the 3 original subtypes. This pattern not only signals that most of the circulating HIV-1 genomes experience constant gene flow from other subtypes but also represents the current spatial distribution of HIV-1 populations. It reflects, for example, the extensive admixture between subtypes B and CRF01_AE in Brazil and China, between B and CRF01_AE in Thailand, and B and F1 in Brazil, Argentina, and Spain.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/1/107/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Projection of worldwide distributed HIV-1 sequences from subtypes B, C, CRF01_AE and F1, and URF and CRF recombinant forms between these subtypes. 1646 genome sequences (see Table A in File S1). (A) The first two axes of the principal component analysis (PC1 and PC2), and (B) the second and third axes of the principal component analysis (PC2 and PC3) defined by pure subtypes B, C, CRF01_AE and F1 (see method section &#x2018;HIV-1 genome analysis&#x2019;). The percentage of variance explained by each PC is indicated between parentheses. The position of a HIV-1 genome sequence along the axes of the triangle formed by three pure subtypes reflect admixture (proportion) between the subtypes (Ma and Amos 2012). Standard country codes are used: Argentina (AR), Brazil (BR), Botswana (BW), China (CN), Cyprus (CY), Spain (ES), Japan (JP), Thailand (TH), USA (US), Vietnam (VN), South Africa (ZA). Misrepresented sequences [squared cosine of the PC plane with the observation (Abdi and Williams 2010)] were excluded." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1039178009" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Projection of worldwide distributed HIV-1 sequences from subtypes B, C, CRF01_AE and F1, and URF and CRF recombinant forms between these subtypes. 1646 genome sequences (see Table A in File S1). (A) The first two axes of the principal component analysis (PC1 and PC2), and (B) the second and third axes of the principal component analysis (PC2 and PC3) defined by pure subtypes B, C, CRF01_AE and F1 (see method section &#x2018;HIV-1 genome analysis&#x2019;). The percentage of variance explained by each PC is indicated between parentheses. The position of a HIV-1 genome sequence along the axes of the triangle formed by three pure subtypes reflect admixture (proportion) between the subtypes (Ma and Amos 2012). Standard country codes are used: Argentina (AR), Brazil (BR), Botswana (BW), China (CN), Cyprus (CY), Spain (ES), Japan (JP), Thailand (TH), USA (US), Vietnam (VN), South Africa (ZA). Misrepresented sequences [squared cosine of the PC plane with the observation &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-54&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;(Abdi and Williams 2010)] were excluded.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/1/107/F5.medium.gif" width="440" height="208"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/1/107/F5.medium.gif" width="440" height="208"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/1/107/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/1/107/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467432" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-37" class="first-child">Projection of worldwide distributed HIV-1 sequences from subtypes B, C, CRF01_AE and F1, and URF and CRF recombinant forms between these subtypes. 1646 genome sequences (see Table A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>). (A) The first two axes of the principal component analysis (PC1 and PC2), and (B) the second and third axes of the principal component analysis (PC2 and PC3) defined by pure subtypes B, C, CRF01_AE and F1 (see method section ‘HIV-1 genome analysis’). The percentage of variance explained by each PC is indicated between parentheses. The position of a HIV-1 genome sequence along the axes of the triangle formed by three pure subtypes reflect admixture (proportion) between the subtypes (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ma and Amos 2012</a>). Standard country codes are used: Argentina (AR), Brazil (BR), Botswana (BW), China (CN), Cyprus (CY), Spain (ES), Japan (JP), Thailand (TH), USA (US), Vietnam (VN), South Africa (ZA). Misrepresented sequences [squared cosine of the PC plane with the observation <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F5/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/F5/embed/mml-math-54.gif"/></noscript></span></span>(<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abdi and Williams 2010</a>)] were excluded.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-38">We present an analytical framework to understand the evolutionary histories of organisms that are characterized by periods of isolation. Our theoretical investigations identify specific genomic signatures of past population isolation. We show, for example, that the number, size, and dynamics of modes in the distributions of the pairwise nucleotide differences in populations are informative on the existence of past isolation. Our results extend previous work on the distribution of pairwise coalescence times and pairwise nucleotide differences under a model of population split with or without migration (IM model). We show, for example, that under the reconnection model, the distribution of pairwise nucleotide differences has a temporal evolution which is distinct from the one expected under an IM model (<a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Takahata 1995</a>; <a id="xref-ref-99-2" class="xref-bibr" href="#ref-99">Wakeley 1996</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Rosenberg and Feldman 2002</a>): the distribution is first bimodal and then tends to become unimodal with time. We also found that in the Site Frequency Spectrum (SFS), the presence, the position and the value variants in excess are informative as to the numbers of previously isolated populations, and the joint SFS characterizes the amount of genetic overlap between populations. Signatures identified here are robust to the presence of recombination, extinction-colonization processes, population bottlenecks, and expansion as well as skewed offspring distributions. Our developments add to the current literature in the field as we demonstrate that population reconnection after isolation is not fully captured by commonly used test statistics based on SFS classes of variants at high and/or low frequencies such as <em>D</em>, <em>H</em>, and <em>E</em> (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Ferretti <em>et al.</em> 2010</a>). Taken together, this provides critical information on the evolution of the genomic composition of populations.</p><p id="p-39">We apply our theory in order to analyze the genome signature of successive colonization of HIV-1 subtypes in Asia and South America. In both regions, we reconstruct the successive colonization of HIV-1 subtypes, and find that they are followed by strong HIV-1 subtype admixture. We find, in these regions, that HIV diversification is followed by a temporal homogenization of HIV-1 genome composition. Our analysis highlights a large degree of admixture between circulating HIV-1 subtype genomes in China and South America, and most probably worldwide. These results suggest that genomic exchanges may indeed be relatively strong, and that HIV-1 is currently in a process of homogenization subsequent to isolation and diversification. HIV genome homogenization may seem to contradict the idea that HIV is undergoing a diversification process (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Hamelaar <em>et al.</em> 2011</a>; <a id="xref-ref-96-5" class="xref-bibr" href="#ref-96">Tebit and Arts 2011</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Feng <em>et al.</em> 2013</a>). However, this homogenization is consistent with observed diversity in the HIV-1 population and the increased number of new HIV recombinant forms described worldwide (<a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Hamelaar <em>et al.</em> 2011</a>; <a id="xref-ref-96-6" class="xref-bibr" href="#ref-96">Tebit and Arts 2011</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Feng <em>et al.</em> 2013</a>). Indeed, when isolated and diverged populations reconnect, a transient excess of genetic diversity is predicted in the resulting population (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Alcala <em>et al.</em> 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Alcala and Vuilleumier 2014</a>) as all the genetic material that accumulated during the isolation phase is shared among populations. However, this excess of genetic diversity is transient and with time, the diversity decreases (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Alcala <em>et al.</em> 2013</a>). In HIV-1, such processes occur through multiple recombination events among subtypes through coinfection or superinfection and are the source of novel recombinant forms. In the long term, successive recombination events will ultimately lead to the homogenization of the genomic composition of HIV-1. Homogenization might be further enhanced by selective forces imposed by the immune response or treatment acting on the HIV genome (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fellay <em>et al.</em> 2007</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Pelak <em>et al.</em> 2011</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Chikata <em>et al.</em> 2014</a>). As many world regions are currently being newly colonized by different HIV-1 subtypes, it is expected that new recombinants may still emerge and HIV diversity will further increase. However, as demonstrated here, this is expected to be a transient state toward global homogenization.</p><p id="p-40">We find signatures of reconnection events among populations of HIV-1 subtypes that were previously isolated both in China and South America. The signatures on HIV-1 DNA polymorphism suggest reconnection events of three previously isolated HIV-1 populations in both regions. These results align with the known epidemiologic history of HIV-1 in the two regions (<a id="xref-ref-92-2" class="xref-bibr" href="#ref-92">Su <em>et al.</em> 2000</a>; <a id="xref-ref-105-1" class="xref-bibr" href="#ref-105">Yang <em>et al.</em> 2002</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Aulicino <em>et al.</em> 2007</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Lu <em>et al.</em> 2008</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Takebe <em>et al.</em> 2010</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Bello <em>et al.</em> 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">An <em>et al.</em> 2012</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Pang <em>et al.</em> 2012</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Han <em>et al.</em> 2013</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Feng <em>et al.</em> 2013</a>) and are supported by recently identified recombinants in China (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Feng <em>et al.</em> 2014</a>; <a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">Wei <em>et al.</em> 2014</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Hsi <em>et al.</em> 2014</a>). To confirm this, we identified the genome sequences that signal the reconnection event. As expected, they reflect first the HIV outbreak in China in 1989: colonization of B and C subtypes (variance of the first peak in the SFS is generated by B and C subtypes). The subsequent colonization of CRF01_AE at the origin of the HIV outbreak in the late 1990s is also signaled in the SFS (a high proportion of CRF01_AE in the highest mode). We obtained similar concordance for the origin of the sequences forming the peaks and the successive subtypes’ colonization events in South America.</p><p id="p-41">Our methods also appear robust to high mutation rates and the presence of selection. Indeed, we detect past HIV-1 history despite its extraordinary defining features: HIV has a small genome (about 9.8 kb in length), a short generation time (24 hr; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Mohammadi <em>et al.</em> 2013</a>), and a high mutation rate (about 0.002 substitutions/site/year; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Korber <em>et al.</em> 2000</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Schlub <em>et al.</em> 2014</a>). It is also under strong selective pressure from the immune system (<em>e.g.</em>, <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Snoeck <em>et al.</em> 2011</a>) and undergoes frequent bottlenecks followed by population expansion [during transmission (<em>e.g.</em>, <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Keele <em>et al.</em> 2008</a>)]. Nevertheless, the signature observed here in the distributions of pairwise nucleotide difference and on SFS only slightly deviates from theoretical expectations. The robustness of these results may have a simple explanation: a separation of time-scales of the processes involved. Selection and population demographic changes drive divergence of populations during isolation. Migration and recombination during population reconnection allows a rapid – relative to divergence – genome admixture. Both impact genome polymorphism, but the latter does so more drastically and at much shorter time-scales.</p><p id="p-42">The evolutionary history of an organism is a powerful resource for understanding its current evolutionary trajectory. The analytical framework developed can be applied to understanding the potential evolutionary pathway of many pathogens and other species that are characterized by periods of isolation. These events are common features of pathogens (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Karlsson <em>et al.</em> 2014</a>), but also occur in numerous other species, including humans and domesticated plants and animals (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Ellstrand <em>et al.</em> 1999</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Lecis <em>et al.</em> 2006</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Caicedo <em>et al.</em> 2007</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Green <em>et al.</em> 2010</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Patterson <em>et al.</em> 2012</a>). Reconnection following a period of isolation is increasingly recognized as an effective mechanism to trigger fast and complex adaptation (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Seehausen 2002</a>, <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">2004</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aguilée <em>et al.</em> 2013</a>). Following reconnection, accumulated diversity (from previously isolated populations) is shared, and the resulting high level of genetic diversity may increase the potential for adaptation (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Alcala and Vuilleumier 2014</a>). Admixture among divergent populations may also promote complex adaptation by bringing together new functions acquired in different environments that have evolved independently but are compatible with the genome. Such processes have been observed in influenza and in HIV, for example. Indeed, recombination between strains previously isolated in their nonhuman hosts preceded the HIV epidemic in humans (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bailes <em>et al.</em> 2003</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Sharp and Hahn 2010</a>; <a id="xref-ref-98-2" class="xref-bibr" href="#ref-98">Vuilleumier and Bonhoeffer 2015</a>). Such events are expected to be more frequent in the future due to the increase in human mobility (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Karlsson <em>et al.</em> 2014</a>). Finally, our analytical framework could also be applied to evolutionary trajectories of other zoonotic viruses, such as Ebola virus, coronaviruses, hantaviruses, or henipaviruses.</p><p id="p-43">It is important to keep in mind that viral population isolation and reconnection events can occur at a much smaller time scale within hosts. There is increasing evidence suggesting that genetic compartmentalization of viral populations occurs in different organs, tissues, or cells, and might play a critical role for virus evolution and more importantly for its control. This has been demonstrated for HIV (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Schnell <em>et al.</em> 2011</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Buzón <em>et al.</em> 2011</a>) as well as human cytomegalovirus (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Renzette <em>et al.</em> 2011</a>, <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">2013</a>, <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">2014</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">2015</a>). Appling the analytical framework detailed here could provide a powerful tool to dissect the dynamics and genetic composition of virus across different genetic compartments, in addition to those dynamics between hosts.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-49">We thank the editor Stephen Wright, John Wakeley, and three anonymous reviewers, whose comments and suggestions greatly improved the manuscript. We would also like to thank Kristen Irwin and Nicholas Renzette for careful reading of the manuscript. This project was funded by the Swiss National Science Foundation (SNSF) grants PZ00P3_139421 and 31003A-130065 and PMPDP3_158381 (S.V.), 31003A_149724/1 (A.T.) and an interdisciplinary grant from the Faculty of Biology and Medicine (FBM) of the University of Lausanne (S.V. and A.T.). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-13"><h3>Derivation of the Theoretical Signature of Population Reconnection on Pairwise Nucleotide Differences</h3><div id="sec-14" class="subsection"><h4>Derivation of the distribution of pairwise coalescence times:</h4><p id="p-44">We here derive the distribution of within- and between-population pairwise coalescence times, <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-55.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-56.gif"/></noscript></span></span>, assuming a long isolation period (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-57.gif"/></noscript></span></span>) and small sequences, <em>i.e.</em>, without recombination. These coalescence times <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-58.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-59.gif"/></noscript></span></span> correspond to the time to the most recent common ancestor of two lineages sampled in the same and in different populations, respectively, scaled by the number of genes per population <em>N</em>.</p><p id="p-45">The coalescence of two genes under the finite island model can be described by a three states continuous time Markov chain (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Notohara 1990</a>): the two genes can be present (1) in the same population, (2) in different populations or (3) be coalesced. The transition probabilities from one state to another depend on the scaled migration rate <em>M</em> between populations and on the number of populations <em>d</em>. Time <em>T</em> is counted in units of <em>N</em> generations. The corresponding Markov chain can be summarized by the following transition rate matrix (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Notohara 1990</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-60.gif"/></noscript></span> <span class="disp-formula-label">(A.1)</span></span>The probability that two genes were in state (1), (2) or (3) <em>T</em> generations ago (row vector <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-61.gif"/></noscript></span></span>), given that they are in state (1), (2) or (3) at the current generation is given by <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-62.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-63.gif"/></noscript></span></span> at generation 0), where <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-64.gif"/></noscript></span></span> is a <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-65.gif"/></noscript></span></span> matrix. The elements of matrix <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-66.gif"/></noscript></span></span> are denoted <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-67.gif"/></noscript></span></span> and represent the probability that a gene in state <em>i</em> at the current generation was in state <em>j T</em> generations ago (<em>cumulative</em> distribution function). Thus, the probability distribution (or probability <em>density</em> function) of within-population and between-population pairwise coalescence times are given by the derivatives of corresponding <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-68.gif"/></noscript></span></span> functions: <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-69.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-70.gif"/></noscript></span></span>. Populations are considered isolated when <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-71.gif"/></noscript></span></span> and no migration occurs, with <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-72.gif"/></noscript></span></span> in the transition matrix <strong>Q</strong>. Populations exchange migrants when <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-73.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-74.gif"/></noscript></span></span>, thus <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-75.gif"/></noscript></span></span> in the transition matrix <strong>Q</strong>. Consequently, the distributions of within- and between-population pairwise coalescence times, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-76.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-77.gif"/></noscript></span></span>, following a past isolation event, are:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-78.gif"/></noscript></span> <span class="disp-formula-label">(A.2a)</span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-79.gif"/></noscript></span> <span class="disp-formula-label">(A.2b)</span></span>These distributions have a simple interpretation. In the most recent period, the reconnection period, when <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-80.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-81.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-82.gif"/></noscript></span></span> correspond to the probability of coalescence of 2 lineages under the finite island model, <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-83.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-84.gif"/></noscript></span></span>. During the past isolation period, <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-85.gif"/></noscript></span></span>, two lineages can only coalesce if they are in the same population after <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-86.gif"/></noscript></span></span> generations of connection (probabilities <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-87.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-88.gif"/></noscript></span></span> within- and between-populations, respectively). Their probability of coalescence at generation <em>T</em> in an isolated population is <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-89.gif"/></noscript></span></span>. During the older connection period, <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-90.gif"/></noscript></span></span>, two genes can coalesce if they were in the same population during isolation (probabilities <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-91.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-92.gif"/></noscript></span></span>, within- and between-populations, respectively) and if they did not coalesce during isolation (probability <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-93.gif"/></noscript></span></span>). Their probability of coalescence at generation <em>T</em> is <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-94.gif"/></noscript></span></span>. Alternatively, they could have coalesced if they were in different populations during isolation (probabilities <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-95.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-96.gif"/></noscript></span></span>). Their probability of coalescence at generation <em>T</em> is <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-97.gif"/></noscript></span></span>.</p><p id="p-46">Functions <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-98.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-99.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-100.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-101.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-102.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-103.gif"/></noscript></span></span> all depend on the eigenvalues of matrix <strong>Q</strong>, <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-104.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-105.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-106.gif"/></noscript></span></span>. We obtain:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-107.gif"/></noscript></span> <span class="disp-formula-label">(A.3a)</span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-108.gif"/></noscript></span> <span class="disp-formula-label">(A.3b)</span></span>Where <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-109.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-110.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-111.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-112.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-113.gif"/></noscript></span></span>.</p></div><div id="sec-15" class="subsection"><h4>Derivation of the distribution of pairwise nucleotide differences:</h4><p id="p-47">From equations A.2a and A.2b, we can derive the distributions of within- (<span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-114.gif"/></noscript></span></span>) and between-population pairwise nucleotide differences (<span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-115.gif"/></noscript></span></span>). Given that mutations follow a Poisson process of mean θ, we have <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-116.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-117.gif"/></noscript></span></span>, where <em>k</em> is a positive integer and <em>G</em> is the length of the sequence (under the infinite size model, we assume that <em>G</em> is much larger than the number of segregating site). We can decompose <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-118.gif"/></noscript></span></span> into a sum of smaller integrals (respectively for <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-119.gif"/></noscript></span></span>):<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-120.gif"/></noscript></span> <span class="disp-formula-label">(A.4)</span></span>Substituting the value of <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-121.gif"/></noscript></span></span> in each term of equation A.4 by the corresponding value in equation A.2a and using the result <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-122.gif"/></noscript></span></span>, we obtain:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-123.gif"/></noscript></span> <span class="disp-formula-label">(A.5)</span></span>The resulting distributions of pairwise nucleotide differences (from equation A.5) are presented in <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1A</a>. Further assuming that <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-124.gif"/></noscript></span></span> (thus <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-125.gif"/></noscript></span></span>) leads to:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-126.gif"/></noscript></span> <span class="disp-formula-label">(A.6a)</span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-127.gif"/></noscript></span> <span class="disp-formula-label">(A.6b)</span></span>Terms in equation A.6a (resp. A.6b) have important interpretations in terms of coalescent events. The first term is the contribution of sequences which coalesce during the recent connection period; indeed, <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-128.gif"/></noscript></span></span> (resp. <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-129.gif"/></noscript></span></span>) corresponds to the distribution of <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-130.gif"/></noscript></span></span> (resp. <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-131.gif"/></noscript></span></span>) under the finite island model at equilibrium, while terms <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-132.gif"/></noscript></span></span> are due to the truncation of the distribution between <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-133.gif"/></noscript></span></span> and infinity. The second term corresponds to sequences that coalesce during the isolation period. Indeed, with probability <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-134.gif"/></noscript></span></span> (resp. <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-135.gif"/></noscript></span></span>), sequences did not coalesce during the reconnection period and were in the same population during isolation; <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-136.gif"/></noscript></span></span> correspond to the distribution of <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-137.gif"/></noscript></span></span> in an isolated population, and the term <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-138.gif"/></noscript></span></span> is due to the truncation between 0 and <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-139.gif"/></noscript></span></span>. Finally, the third term corresponds to sequences that coalesce during the prior connection period. With probability <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-140.gif"/></noscript></span></span> (resp. <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-141.gif"/></noscript></span></span>), sequences coalesce during the prior connection period (as they did not coalesce during the reconnection period and were in different populations during isolation). Then, <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-142.gif"/></noscript></span></span> corresponds to the distribution of <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-143.gif"/></noscript></span></span> under the finite island model at equilibrium, while terms <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-144.gif"/></noscript></span></span> are due to the truncation of the distribution between 0 and <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-145.gif"/></noscript></span></span>.</p></div><div id="sec-16" class="subsection"><h4>Derivation of the power to detect bimodality:</h4><p id="p-48">We can derive the power to detect bimodality from the distributions of <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-146.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-147.gif"/></noscript></span></span> by computing the probability that, in a sample of <em>n</em> independent pairwise differences sequences, at least one is from the first mode and at least one is from the second. Given that the probability for a difference <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-148.gif"/></noscript></span></span> to be drawn from the second mode of the distribution is <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-149.gif"/></noscript></span></span>, the probability that among <em>n</em> differences, at least one is from the first mode and one from the second mode is <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-150.gif"/></noscript></span></span>. Similarly, the probability to observe a bimodal distribution of <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-151.gif"/></noscript></span></span> from a sample of size <em>n</em> is <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-152.gif"/></noscript></span></span>. These results are used to draw Figure A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1/FileS1.pdf">File S1</a>. Note that with <em>n</em> sequences, we can only compute <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-153.gif"/></noscript></span></span> independent pairwise differences, although the total number of pairwise differences we can compute is <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/1/107/embed/mml-math-154.gif"/></noscript></span></span>.</p></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-50">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024208/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-51"><em>Communicating editor: S. I. Wright</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-08-01"><span class="received-label">Received </span>August 1, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-11-02"><span class="accepted-label">Accepted </span>November 2, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Alcala <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.1" data-doi="10.1002/wics.101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abdi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Principal component analysis.</span> <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. Comput. Stat.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.%2BRev.%2BComput.%2BStat.%26rft.volume%253D2%26rft.spage%253D433%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwics.101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wics.101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.2" data-doi="10.1534/genetics.109.104042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Achaz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Frequency spectrum neutrality tests: one for all and all for one.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.104042%26rft_id%253Dinfo%253Apmid%252F19546320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMjQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilée</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Claessen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambert</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Adaptive radiation driven by the interplay of eco-evolutionary and landscape dynamics.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1291</span>–<span class="cit-lpage">1306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D67%26rft.spage%253D1291%26rft_id%253Dinfo%253Apmid%252F23617909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23617909&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318234900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.4" data-doi="10.1098/rspb.2014.1369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alcala</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vuilleumier</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Turnover and accumulation of genetic diversity across large time-scale cycles of isolation and connection of populations.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">20141369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2014.1369%26rft_id%253Dinfo%253Apmid%252F25253456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjgxLzE3OTQvMjAxNDEzNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.5" data-doi="10.1534/genetics.112.147785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alcala</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Streit</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vuilleumier</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Peak and persistent excess of genetic diversity following an abrupt migration increase.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147785%26rft_id%253Dinfo%253Apmid%252F23307901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzMvOTUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.6" data-doi="10.1128/JVI.00262-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reconstituting the epidemic history of HIV strain crf01_ae among men who have sex with men (msm) in liaoning, northeastern China: Implications for the expanding epidemic among msm in China.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">12402</span>–<span class="cit-lpage">12406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00262-12%26rft_id%253Dinfo%253Apmid%252F22933290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4Ni8yMi8xMjQwMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.7" data-doi="10.1089/aid.2007.0038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aulicino</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bello</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rocco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mangano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Description of the first full-length HIV type 1 subtype f1 strain in argentina: implications for the origin and dispersion of this subtype in South America.</span> <abbr class="cit-jnl-abbrev">AIDS Res. Hum. Retroviruses</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1176</span>–<span class="cit-lpage">1182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2Bresearch%2Band%2Bhuman%2Bretroviruses%26rft.stitle%253DAIDS%2BRes%2BHum%2BRetroviruses%26rft.aulast%253DAulicino%26rft.auinit1%253DP.%2BC.%26rft.volume%253D23%26rft.issue%253D10%26rft.spage%253D1176%26rft.epage%253D1182%26rft.atitle%253DDescription%2Bof%2Bthe%2Bfirst%2Bfull-length%2BHIV%2Btype%2B1%2Bsubtype%2BF1%2Bstrain%2Bin%2BArgentina%253A%2Bimplications%2Bfor%2Bthe%2Borigin%2Band%2Bdispersion%2Bof%2Bthis%2Bsubtype%2Bin%2BSouth%2BAmerica.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Faid.2007.0038%26rft_id%253Dinfo%253Apmid%252F17961101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/aid.2007.0038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17961101&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.8" data-doi="10.1126/science.1080657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bibollet-Ruche</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Courgnaud</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peeters</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Hybrid origin of SIV in chimpanzees.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">300</span>: <span class="cit-fpage">1713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBailes%26rft.auinit1%253DE.%26rft.volume%253D300%26rft.issue%253D5626%26rft.spage%253D1713%26rft.epage%253D1713%26rft.atitle%253DHybrid%2BOrigin%2Bof%2BSIV%2Bin%2BChimpanzees%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1080657%26rft_id%253Dinfo%253Apmid%252F12805540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDAvNTYyNi8xNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.9" data-doi="10.1098/rspa.1937.0109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartlett</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1937</span> <span class="cit-article-title">Properties of sufficiency and statistical tests.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. A. Math. Phys. Sci.</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">268</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BLond.%2BA.%2BMath.%2BPhys.%2BSci.%26rft.volume%253D160%26rft.spage%253D268%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspa.1937.0109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspa.1937.0109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.10" data-doi="10.1017/S0016672398003462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The effect of hitch-hiking on neutral genealogies.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D72%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672398003462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672398003462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000077703600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.11" data-doi="10.1534/genetics.166.2.1115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etheridge</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The effect of selection on genealogies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBarton%26rft.auinit1%253DN.%2BH.%26rft.volume%253D166%26rft.issue%253D2%26rft.spage%253D1115%26rft.epage%253D1131%26rft.atitle%253DThe%2BEffect%2Bof%2BSelection%2Bon%2BGenealogies%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.2.1115%26rft_id%253Dinfo%253Apmid%252F15020491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni8yLzExMTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Approximate bayesian computation in population genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">2025</span>–<span class="cit-lpage">2035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBeaumont%26rft.auinit1%253DM.%2BA.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D2025%26rft.epage%253D2035%26rft.atitle%253DApproximate%2BBayesian%2BComputation%2Bin%2BPopulation%2BGenetics%26rft_id%253Dinfo%253Apmid%252F12524368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzIwMjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.13" data-doi="10.1111/j.1558-5646.2009.00662.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Becquet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Przeworski</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Learning about modes of speciation by computational approaches.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">2547</span>–<span class="cit-lpage">2562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBerner%26rft.auinit1%253DD.%26rft.volume%253D63%26rft.issue%253D10%26rft.spage%253D2547%26rft.epage%253D2562%26rft.atitle%253DLearning%2Babout%2Bmodes%2Bof%2Bspeciation%2Bby%2Bcomputational%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2009.00662.x%26rft_id%253Dinfo%253Apmid%252F19228187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2009.00662.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19228187&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270651800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bello</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soares</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrago</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The use of bioinformatics for studying HIV evolutionary and epidemiological history in south america.</span> <abbr class="cit-jnl-abbrev">Aids Res. Treat.</abbr> <span class="cit-vol">2011</span>: <span class="cit-fpage">154945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAIDS%2BRes%2BTreat%26rft.aulast%253DBello%26rft.auinit1%253DG.%26rft.volume%253D2011%26rft.spage%253D154945%26rft.epage%253D154945%26rft.atitle%253DThe%2BUse%2Bof%2BBioinformatics%2Bfor%2BStudying%2BHIV%2BEvolutionary%2Band%2BEpidemiological%2BHistory%2Bin%2BSouth%2BAmerica.%26rft_id%253Dinfo%253Apmid%252F22162803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22162803&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.15" data-doi="10.18637/jss.v032.i06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benaglia</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauveau</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">mixtools: An r package for analyzing finite mixture models.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D32%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v032.i06%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v032.i06&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.16" data-doi="10.1007/s00285-008-0170-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birkner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blath</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Computing likelihoods for coalescents with multiple collisions in the infinitely many sites model.</span> <abbr class="cit-jnl-abbrev">J. Math. Biol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DJ%2BMath%2BBiol%26rft.aulast%253DBirkner%26rft.auinit1%253DM.%26rft.volume%253D57%26rft.issue%253D3%26rft.spage%253D435%26rft.epage%253D465%26rft.atitle%253DComputing%2Blikelihoods%2Bfor%2Bcoalescents%2Bwith%2Bmultiple%2Bcollisions%2Bin%2Bthe%2Binfinitely%2Bmany%2Bsites%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00285-008-0170-6%26rft_id%253Dinfo%253Apmid%252F18347796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00285-008-0170-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18347796&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256659700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.17" data-doi="10.1371/journal.ppat.1002314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buzón</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Codoñer</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">S. D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puertas</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Deep molecular characterization of HIV-1 dynamics under suppressive haart.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DBuzon%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002314%26rft.epage%253De1002314%26rft.atitle%253DDeep%2Bmolecular%2Bcharacterization%2Bof%2BHIV-1%2Bdynamics%2Bunder%2Bsuppressive%2BHAART.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002314%26rft_id%253Dinfo%253Apmid%252F22046128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22046128&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caicedo</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide patterns of nucleotide polymorphism in domesticated rice.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1745</span>–<span class="cit-lpage">1756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCaicedo%26rft.auinit1%253DA.%2BL.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D1745%26rft.epage%253D1756%26rft.atitle%253DGenome-wide%2Bpatterns%2Bof%2Bnucleotide%2Bpolymorphism%2Bin%2Bdomesticated%2Brice.%26rft_id%253Dinfo%253Apmid%252F17907810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17907810&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249767800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.19" data-doi="10.1016/j.ympev.2011.11.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castro-Nallar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Losada</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The evolution of HIV: inferences using phylogenetics.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D62%26rft.spage%253D777%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2011.11.019%26rft_id%253Dinfo%253Apmid%252F22138161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2011.11.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22138161&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.20" data-doi="10.1128/JVI.00147-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chikata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borghan</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naruto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Host-specific adaptation of HIV-1 subtype b in the japanese population.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">4764</span>–<span class="cit-lpage">4775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00147-14%26rft_id%253Dinfo%253Apmid%252F24522911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzkvNDc2NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.21" data-doi="10.1093/molbev/msi103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drummond</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pybus</span>  <span class="cit-name-given-names">O. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Bayesian coalescent inference of past population dynamics from molecular sequences.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi103%26rft_id%253Dinfo%253Apmid%252F15703244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIyLzUvMTE4NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.22" data-doi="10.1146/annurev.ecolsys.30.1.539"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellstrand</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prentice</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Gene flow and introgression from domesticated plants into their wild relatives.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">539</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DEllstrand%26rft.auinit1%253DN.%2BC.%26rft.volume%253D30%26rft.issue%253D1%26rft.spage%253D539%26rft.epage%253D563%26rft.atitle%253DGENE%2BFLOW%2BAND%2BINTROGRESSION%2BFROM%2BDOMESTICATED%2BPLANTS%2BINTO%2BTHEIR%2BWILD%2BRELATIVES%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.30.1.539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.30.1.539&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000084882700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.23" data-doi="10.1046/j.1365-294X.2003.02004.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Patterns of DNA sequence diversity and genetic structure after a range expansion: lessons from the infinite-island model.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DExcoffier%26rft.auinit1%253DL.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D853%26rft.epage%253D864%26rft.atitle%253DPatterns%2Bof%2BDNA%2Bsequence%2Bdiversity%2Band%2Bgenetic%2Bstructure%2Bafter%2Ba%2Brange%2Bexpansion%253A%2Blessons%2Bfrom%2Bthe%2Binfinite-island%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2003.02004.x%26rft_id%253Dinfo%253Apmid%252F15012760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-294X.2003.02004.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15012760&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220153000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.24" data-doi="10.1371/journal.pgen.1003905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupanloup</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Sánchez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Robust demographic inference from genomic and SNP data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003905%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003905%26rft_id%253Dinfo%253Apmid%252F24204310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003905&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24204310&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Hitchhiking under positive darwinian selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1405%26rft.epage%253D1413%26rft.atitle%253DHitchhiking%2BUnder%2BPositive%2BDarwinian%2BSelection%26rft_id%253Dinfo%253Apmid%252F10880498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzE0MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.26" data-doi="10.1126/science.1143767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shianna</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colombo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ledergerber</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A whole-genome association study of major determinants for host control of HIV-1.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">944</span>–<span class="cit-lpage">947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1143767%26rft_id%253Dinfo%253Apmid%252F17641165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTg0MC85NDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.27" data-doi="10.1097/QAD.0b013e328360db2d"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsi</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The rapidly expanding crf01_ae epidemic in China is driven by multiple lineages of HIV-1 viruses introduced in the 1990s.</span> <abbr class="cit-jnl-abbrev">AIDS</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1793</span>–<span class="cit-lpage">1802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%26rft.volume%253D27%26rft.spage%253D1793%26rft_id%253Dinfo%253Adoi%252F10.1097%252FQAD.0b013e328360db2d%26rft_id%253Dinfo%253Apmid%252F23807275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/QAD.0b013e328360db2d&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23807275&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.28" data-doi="10.1089/aid.2013.0233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification of a novel HIV type 1 circulating recombinant form (crf65_cpx) composed of crf01_ae and subtypes b and c in western Yunnan, China.</span> <abbr class="cit-jnl-abbrev">AIDS Res. Hum. Retroviruses</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">598</span>–<span class="cit-lpage">602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2BRes.%2BHum.%2BRetroviruses%26rft.volume%253D30%26rft.spage%253D598%26rft_id%253Dinfo%253Adoi%252F10.1089%252Faid.2013.0233%26rft_id%253Dinfo%253Apmid%252F24279591%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/aid.2013.0233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24279591&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.29" data-doi="10.1534/genetics.110.118570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferretti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez-Enciso</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Optimal neutrality tests based on the frequency spectrum.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118570%26rft_id%253Dinfo%253Apmid%252F20610408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzEvMzUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.107.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-source">The genetical theory of natural selection</span>, <span class="cit-publ-name">Clarendon Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flahault</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zylberman</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Influenza pandemics: past, present and future challenges.</span> <abbr class="cit-jnl-abbrev">Public Health Rev.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPublic%2BHealth%2BRev.%26rft.volume%253D32%26rft.spage%253D319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.107.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Wfabc: a Wright–Fisher abc-based approach for inferring effective population sizes and selection coefficients from time-sampled data.</span> <span class="cit-source">Mol. Ecol. Resour.</span> <span class="cit-vol">15</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D87%26rft_id%253Dinfo%253Apmid%252F24834845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24834845&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Statistical tests of neutrality of mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFu%26rft.auinit1%253DY.%2BX.%26rft.volume%253D133%26rft.issue%253D3%26rft.spage%253D693%26rft.epage%253D709%26rft.atitle%253DStatistical%2Btests%2Bof%2Bneutrality%2Bof%2Bmutations.%26rft_id%253Dinfo%253Apmid%252F8454210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMzLzMvNjkzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.34" data-doi="10.1126/science.1176225"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garten</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindstrom</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Antigenic and genetic characteristics of swine-origin 2009 a(h1n1) influenza viruses circulating in humans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1176225%26rft_id%253Dinfo%253Apmid%252F19465683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTkzNy8xOTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.35" data-doi="10.1126/science.1070441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaschen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yusim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Diversity considerations in HIV-1 vaccine selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">2354</span>–<span class="cit-lpage">2360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGaschen%26rft.auinit1%253DB.%26rft.volume%253D296%26rft.issue%253D5577%26rft.spage%253D2354%26rft.epage%253D2360%26rft.atitle%253DDiversity%2BConsiderations%2Bin%2BHIV-1%2BVaccine%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1070441%26rft_id%253Dinfo%253Apmid%252F12089434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTYvNTU3Ny8yMzU0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.1.107.36" data-doi="10.1073/pnas.1019276108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marth</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Demographic history and rare allele sharing among human populations.</span> <span class="cit-source">Proc. Natl. Acad. Sci. USA</span> <span class="cit-vol">108</span>: <span class="cit-fpage">11983</span>–<span class="cit-lpage">11988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1019276108%26rft_id%253Dinfo%253Apmid%252F21730125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI5LzExOTgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.37" data-doi="10.1126/science.1188021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maricic</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stenzel</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A draft sequence of the neandertal genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">328</span>: <span class="cit-fpage">710</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGreen%26rft.auinit1%253DR.%2BE.%26rft.volume%253D328%26rft.issue%253D5979%26rft.spage%253D710%26rft.epage%253D722%26rft.atitle%253DA%2BDraft%2BSequence%2Bof%2Bthe%2BNeandertal%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1188021%26rft_id%253Dinfo%253Apmid%252F20448178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjgvNTk3OS83MTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.38" data-doi="10.1371/journal.pgen.1000695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGutenkunst%26rft.auinit1%253DR.%2BN.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000695%26rft.epage%253De1000695%26rft.atitle%253DInferring%2Bthe%2Bjoint%2Bdemographic%2Bhistory%2Bof%2Bmultiple%2Bpopulations%2Bfrom%2Bmultidimensional%2BSNP%2Bfrequency%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000695%26rft_id%253Dinfo%253Apmid%252F19851460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19851460&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.39" data-doi="10.1097/QAD.0b013e328342ff93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamelaar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gouws</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghys</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osmanov</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, and <em>WHO-UN AIDS Network for HIV Isolation and Characterisation</em>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Global trends in molecular epidemiology of HIV-1 during 2000–2007.</span> <abbr class="cit-jnl-abbrev">AIDS</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">689</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2B%2528London%252C%2BEngland%2529%26rft.stitle%253DAIDS%26rft.aulast%253DHemelaar%26rft.auinit1%253DJ.%26rft.volume%253D25%26rft.issue%253D5%26rft.spage%253D679%26rft.epage%253D689%26rft.atitle%253DGlobal%2Btrends%2Bin%2Bmolecular%2Bepidemiology%2Bof%2BHIV-1%2Bduring%2B2000-2007.%26rft_id%253Dinfo%253Adoi%252F10.1097%252FQAD.0b013e328342ff93%26rft_id%253Dinfo%253Apmid%252F21297424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/QAD.0b013e328342ff93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21297424&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287963600018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.40" data-doi="10.1371/journal.pone.0065337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High prevalence of HIV-1 intersubtype b /c recombinants among injecting drug users in Dehong, China.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e65337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De65337%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0065337%26rft_id%253Dinfo%253Apmid%252F23741489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0065337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23741489&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.41" data-doi="10.1073/pnas.95.4.1961"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harpending</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batzer</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurven</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Genetic traces of ancient demography.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">1961</span>–<span class="cit-lpage">1967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHarpending%26rft.auinit1%253DH.%2BC.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D1961%26rft.epage%253D1967%26rft.atitle%253DGenetic%2Btraces%2Bof%2Bancient%2Bdemography%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.4.1961%26rft_id%253Dinfo%253Apmid%252F9465125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5NS80LzE5NjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.42" data-doi="10.1093/molbev/msp296"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Isolation with migration models for more than two populations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">920</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp296%26rft_id%253Dinfo%253Apmid%252F19955477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI3LzQvOTA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.43" data-doi="10.1089/aid.2013.0234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome sequence of a novel HIV-1 circulating recombinant form (crf64_bc) identified from Yunnan, China.</span> <abbr class="cit-jnl-abbrev">AIDS Res. Hum. Retroviruses</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2BRes.%2BHum.%2BRetroviruses%26rft.volume%253D30%26rft.spage%253D389%26rft_id%253Dinfo%253Adoi%252F10.1089%252Faid.2013.0234%26rft_id%253Dinfo%253Apmid%252F24205972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/aid.2013.0234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24205972&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.44" data-doi="10.1016/S0929-6646(09)60102-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">Y.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">P.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">L.-Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Influenza pandemics: past, present and future.</span> <abbr class="cit-jnl-abbrev">J. Formos. Med. Assoc.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">6</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bthe%2BFormosan%2BMedical%2BAssociation%2B%253D%2BTaiwan%2Byi%2Bzhi%26rft.stitle%253DJ%2BFormos%2BMed%2BAssoc%26rft.aulast%253DHsieh%26rft.auinit1%253DY.%2BC.%26rft.volume%253D105%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D6%26rft.atitle%253DInfluenza%2Bpandemics%253A%2Bpast%252C%2Bpresent%2Band%2Bfuture.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0929-6646%252809%252960102-9%26rft_id%253Dinfo%253Apmid%252F16440064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0929-6646(09)60102-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16440064&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236017700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.45" data-doi="10.1093/bioinformatics/18.2.337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Generating samples under a Wright–Fisher neutral model of genetic variation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D338%26rft.atitle%253DGenerating%2Bsamples%2Bunder%2Ba%2BWright-Fisher%2Bneutral%2Bmodel%2Bof%2B%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.2.337%26rft_id%253Dinfo%253Apmid%252F11847089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOC8yLzMzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.46" data-doi="10.1186/1471-2156-11-94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devillard</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balloux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Discriminant analysis of principal components: a new method for the analysis of genetically structured populations.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DJombart%26rft.auinit1%253DT.%26rft.volume%253D11%26rft.spage%253D94%26rft.epage%253D94%26rft.atitle%253DDiscriminant%2Banalysis%2Bof%2Bprincipal%2Bcomponents%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Bthe%2Banalysis%2Bof%2Bgenetically%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-11-94%26rft_id%253Dinfo%253Apmid%252F20950446%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-11-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20950446&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.47" data-doi="10.1038/nrg3734"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlsson</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwiatkowski</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural selection and infectious disease in human populations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3734%26rft_id%253Dinfo%253Apmid%252F24776769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3734&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24776769&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.48" data-doi="10.1073/pnas.0802203105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keele</span>  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giorgi</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salazar-Gonzalez</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Decker</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">K. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification and characterization of transmitted and early founder virus envelopes in primary HIV-1 infection.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">7552</span>–<span class="cit-lpage">7557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0802203105%26rft_id%253Dinfo%253Apmid%252F18490657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzIxLzc1NTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">The number of heterozygous nucleotide sites maintained in a finite population due to steady flux of mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">893</span>–<span class="cit-lpage">903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D61%26rft.issue%253D4%26rft.spage%253D893%26rft.epage%253D903%26rft.atitle%253DTHE%2BNUMBER%2BOF%2BHETEROZYGOUS%2BNUCLEOTIDE%2BSITES%2BMAINTAINED%2BIN%2BA%2BFINITE%2BPOPULATION%2BDUE%2BTO%2BSTEADY%2BFLUX%2BOF%2BMUTATIONS%26rft_id%253Dinfo%253Apmid%252F5364968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI2MS80Lzg5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.50" data-doi="10.1126/science.288.5472.1789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korber</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muldoon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theiler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Timing the ancestor of the HIV-1 pandemic strains.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">1789</span>–<span class="cit-lpage">1796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKorber%26rft.auinit1%253DB.%26rft.volume%253D288%26rft.issue%253D5472%26rft.spage%253D1789%26rft.epage%253D1796%26rft.atitle%253DTiming%2Bthe%2BAncestor%2Bof%2Bthe%2BHIV-1%2BPandemic%2BStrains%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5472.1789%26rft_id%253Dinfo%253Apmid%252F10846155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODgvNTQ3Mi8xNzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.51" data-doi="10.1093/bmb/58.1.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korber</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaschen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yusim</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thakallapally</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kesmir</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Evolutionary and immunological implications of contemporary HIV-1 variation.</span> <abbr class="cit-jnl-abbrev">Br. Med. Bull.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBritish%2BMedical%2BBulletin%26rft.stitle%253DBr%2BMed%2BBull%26rft.aulast%253DKorber%26rft.auinit1%253DB.%26rft.volume%253D58%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D42%26rft.atitle%253DEvolutionary%2Band%2Bimmunological%2Bimplications%2Bof%2Bcontemporary%2BHIV-1%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbmb%252F58.1.19%26rft_id%253Dinfo%253Apmid%252F11714622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYm1idWxsIjtzOjU6InJlc2lkIjtzOjc6IjU4LzEvMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.52" data-doi="10.1111/j.1365-294X.2005.02812.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lecis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierpaoli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biro</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szemethy</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ragni</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Bayesian analyses of admixture in wild and domestic cats (<em>Felis silvestris</em>) using linked microsatellite loci.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLecis%26rft.auinit1%253DR.%26rft.volume%253D15%26rft.issue%253D1%26rft.spage%253D119%26rft.epage%253D131%26rft.atitle%253DBayesian%2Banalyses%2Bof%2Badmixture%2Bin%2Bwild%2Band%2Bdomestic%2Bcats%2B%2528Felis%2Bsilvestris%2529%2Busing%2Blinked%2Bmicrosatellite%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02812.x%26rft_id%253Dinfo%253Apmid%252F16367835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02812.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16367835&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234029800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-other" id="cit-6.1.107.53" data-doi="10.1089/aid.2013.0116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Near full-length genome sequence of a novel HIV type 1 second-generation recombinant form (crf01_ae/crf07_bc) identified among men who have sex with men in jilin, china.</span> <span class="cit-source">AIDS Res. Hum. Retroviruses</span> <span class="cit-vol">29</span>: <span class="cit-fpage">1604</span>–<span class="cit-lpage">1608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2BRes.%2BHum.%2BRetroviruses%26rft.volume%253D29%26rft.spage%253D1604%26rft_id%253Dinfo%253Adoi%252F10.1089%252Faid.2013.0116%26rft_id%253Dinfo%253Apmid%252F23809010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/aid.2013.0116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23809010&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.54" data-doi="10.3390/v6072880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liégeois</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boué</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mouacha</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Full-length genome analyses of two new simian immunodeficiency virus (SIV) strains from mustached monkeys (<em>c. cephus</em>) in gabon illustrate a complex evolutionary history among the sivmus/mon/gsn lineage.</span> <abbr class="cit-jnl-abbrev">Viruses</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2880</span>–<span class="cit-lpage">2898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DViruses%26rft.volume%253D6%26rft.spage%253D2880%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fv6072880%26rft_id%253Dinfo%253Apmid%252F25054885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/v6072880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25054885&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.55" data-doi="10.1534/genetics.111.129569"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A general method for calculating likelihoods under the coalescent process.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">977</span>–<span class="cit-lpage">987</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.129569%26rft_id%253Dinfo%253Apmid%252F21900266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzMvOTc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.56" data-doi="10.1038/455609a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The changing face of HIV in china.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">455</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">611</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLu%26rft.auinit1%253DL.%26rft.volume%253D455%26rft.issue%253D7213%26rft.spage%253D609%26rft.epage%253D611%26rft.atitle%253DThe%2Bchanging%2Bface%2Bof%2BHIV%2Bin%2BChina.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F455609a%26rft_id%253Dinfo%253Apmid%252F18833270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/455609a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18833270&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259639700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.57" data-doi="10.1371/journal.pone.0040115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amos</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Principal components analysis of population admixture.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e40115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DMa%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De40115%26rft.epage%253De40115%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bof%2Bpopulation%2Badmixture.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0040115%26rft_id%253Dinfo%253Apmid%252F22808102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0040115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22808102&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.58" data-doi="10.1093/biomet/80.2.267"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meng</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Maximum likelihood estimation via the ecm algorithm: A general framework.</span> <abbr class="cit-jnl-abbrev">Biometrika</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrika%26rft.stitle%253DBiometrika%26rft.aulast%253DMENG%26rft.auinit1%253DX.-L.%26rft.volume%253D80%26rft.issue%253D2%26rft.spage%253D267%26rft.epage%253D278%26rft.atitle%253DMaximum%2Blikelihood%2Bestimation%2Bvia%2Bthe%2BECM%2Balgorithm%253A%2BA%2Bgeneral%2Bframework%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbiomet%252F80.2.267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYmlvbWV0IjtzOjU6InJlc2lkIjtzOjg6IjgwLzIvMjY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-other" id="cit-6.1.107.59" data-doi="10.1371/journal.ppat.1003161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammadi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desfarges</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartha</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joos</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zangger</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">24 hours in the life of HIV-1 in a t cell line.</span> <span class="cit-source">PLoS pathogens</span> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bpathogens%26rft.volume%253D9%26rft.spage%253De1003161%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003161%26rft_id%253Dinfo%253Apmid%252F23382686%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003161&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23382686&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.60" data-doi="10.1111/j.1365-294X.2011.05131.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naduvilezhath</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzler</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Jaatha: a fast composite-likelihood approach to estimate demographic parameters.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2709</span>–<span class="cit-lpage">2723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DNaduvilezhath%26rft.auinit1%253DL.%26rft.volume%253D20%26rft.issue%253D13%26rft.spage%253D2709%26rft.epage%253D2723%26rft.atitle%253DJaatha%253A%2Ba%2Bfast%2Bcomposite-likelihood%2Bapproach%2Bto%2Bestimate%2Bdemographic%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05131.x%26rft_id%253Dinfo%253Apmid%252F21645157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05131.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21645157&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.61" data-doi="10.1266/ggs.83.353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nawa</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Simple method for analyzing the pattern of DNA polymorphism and its application to SNP data of human.</span> <abbr class="cit-jnl-abbrev">Genes Genet. Syst.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2Bgenetic%2Bsystems%26rft.stitle%253DGenes%2BGenet%2BSyst%26rft.aulast%253DNawa%26rft.auinit1%253DN.%26rft.volume%253D83%26rft.issue%253D4%26rft.spage%253D353%26rft.epage%253D360%26rft.atitle%253DSimple%2Bmethod%2Bfor%2Banalyzing%2Bthe%2Bpattern%2Bof%2BDNA%2Bpolymorphism%2Band%2Bits%2Bapplication%2Bto%2BSNP%2Bdata%2Bof%2Bhuman.%26rft_id%253Dinfo%253Adoi%252F10.1266%252Fggs.83.353%26rft_id%253Dinfo%253Apmid%252F18931461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1266/ggs.83.353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18931461&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.62" data-doi="10.1073/pnas.1213113110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neher</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallatschek</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genealogies of rapidly adapting populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1213113110%26rft_id%253Dinfo%253Apmid%252F23269838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMi80MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Distinguishing migration from isolation: a markov chain monte carlo approach.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">885</span>–<span class="cit-lpage">896</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D158%26rft.issue%253D2%26rft.spage%253D885%26rft.epage%253D896%26rft.atitle%253DDistinguishing%2BMigration%2BFrom%2BIsolation%253A%2BA%2BMarkov%2BChain%2BMonte%2BCarlo%2BApproach%26rft_id%253Dinfo%253Apmid%252F11404349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU4LzIvODg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Notohara</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">The coalescent and the genealogical process in geographically structured population.</span> <abbr class="cit-jnl-abbrev">J. Math. Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DJ%2BMath%2BBiol%26rft.aulast%253DNotohara%26rft.auinit1%253DM.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D75%26rft.atitle%253DThe%2Bcoalescent%2Band%2Bthe%2Bgenealogical%2Bprocess%2Bin%2Bgeographically%2Bstructured%2Bpopulation.%26rft_id%253Dinfo%253Apmid%252F2277236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=2277236&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EE60800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.65" data-doi="10.1097/QAD.0b013e3283522c97"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">Z.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extensive and complex HIV-1 recombination between b′, c and crf01_ae among Idus in south-east Asia.</span> <abbr class="cit-jnl-abbrev">AIDS</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1121</span>–<span class="cit-lpage">1129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2B%2528London%252C%2BEngland%2529%26rft.stitle%253DAIDS%26rft.aulast%253DPang%26rft.auinit1%253DW.%26rft.volume%253D26%26rft.issue%253D9%26rft.spage%253D1121%26rft.epage%253D1129%26rft.atitle%253DExtensive%2Band%2Bcomplex%2BHIV-1%2Brecombination%2Bbetween%2BB%2527%252C%2BC%2Band%2BCRF01_AE%2Bamong%2BIDUs%2Bin%2Bsouth-east%2BAsia.%26rft_id%253Dinfo%253Adoi%252F10.1097%252FQAD.0b013e3283522c97%26rft_id%253Dinfo%253Apmid%252F22333750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/QAD.0b013e3283522c97&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22333750&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.66" data-doi="10.1534/genetics.112.145037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moorjani</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohland</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ancient admixture in human history.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">1065</span>–<span class="cit-lpage">1093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145037%26rft_id%253Dinfo%253Apmid%252F22960212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Mi8zLzEwNjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.67" data-doi="10.1371/journal.pbio.1001208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pelak</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Need</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shianna</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Copy number variation of kir genes influences HIV-1 control.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1001208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DPelak%26rft.auinit1%253DK.%26rft.volume%253D9%26rft.issue%253D11%26rft.spage%253De1001208%26rft.epage%253De1001208%26rft.atitle%253DCopy%2Bnumber%2Bvariation%2Bof%2BKIR%2Bgenes%2Binfluences%2BHIV-1%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001208%26rft_id%253Dinfo%253Apmid%252F22140359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001208&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22140359&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.68" data-doi="10.1371/journal.pgen.1003080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of sub-saharan <em>Drosophila melanogaster</em>: African diversity and non-african admixture.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003080%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003080%26rft_id%253Dinfo%253Apmid%252F23284287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284287&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.69" data-doi="10.1038/nature11258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tandon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mazieres</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reconstructing native american population history.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">488</span>: <span class="cit-fpage">370</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DReich%26rft.auinit1%253DD.%26rft.volume%253D488%26rft.issue%253D7411%26rft.spage%253D370%26rft.epage%253D374%26rft.atitle%253DReconstructing%2BNative%2BAmerican%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11258%26rft_id%253Dinfo%253Apmid%252F22801491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22801491&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307501000040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.70" data-doi="10.1371/journal.ppat.1001344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renzette</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharjee</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalik</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extensive genome-wide variability of human cytomegalovirus in congenitally infected infants.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DRenzette%26rft.auinit1%253DN.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De1001344%26rft.epage%253De1001344%26rft.atitle%253DExtensive%2Bgenome-wide%2Bvariability%2Bof%2Bhuman%2Bcytomegalovirus%2Bin%2Bcongenitally%2Binfected%2Binfants.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1001344%26rft_id%253Dinfo%253Apmid%252F21625576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1001344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21625576&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.71" data-doi="10.1371/journal.pgen.1003735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renzette</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharjee</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schleiss</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rapid intrahost evolution of human cytomegalovirus is shaped by demography and positive selection.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003735</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003735%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003735%26rft_id%253Dinfo%253Apmid%252F24086142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003735&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24086142&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.72" data-doi="10.1016/j.coviro.2014.08.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renzette</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalik</span>  <span class="cit-name-given-names">T. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Human cytomegalovirus intrahost evolution–a new avenue for understanding and controlling herpesvirus infections.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Virol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BVirol.%26rft.volume%253D8%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.coviro.2014.08.001%26rft_id%253Dinfo%253Apmid%252F25154343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.coviro.2014.08.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25154343&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-confproc" id="cit-6.1.107.73" data-doi="10.1073/pnas.1501880112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renzette</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pokalyuk</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharjee</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schleiss</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Limits and patterns of cytomegalovirus genomic diversity in humans.</span> <span class="cit-source">Proc. Natl. Acad. Sci. USA</span> <span class="cit-vol">112</span>: <span class="cit-fpage">E4120</span>–<span class="cit-lpage">E4128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1501880112%26rft_id%253Dinfo%253Apmid%252F26150505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzMwL0U0MTIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.74" data-doi="10.1126/science.288.5463.55c"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradac</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foley</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">HIV-1 nomenclature proposal.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">288</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.volume%253D288%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.288.5463.55c%26rft_id%253Dinfo%253Apmid%252F10766634%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.288.5463.55c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10766634&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-book" id="cit-6.1.107.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The relationship between coalescence times and population divergence times</span>, pp. <span class="cit-fpage">130</span>–<span class="cit-lpage">164</span> in <span class="cit-source">Modern Developments in Theoretical Population Genetics: The legacy of Gustave Malécot</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M. W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Veuille</span>  <span class="cit-name-given-names">M.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.76" data-doi="10.1371/journal.pgen.1002947"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pääbo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The date of interbreeding between neandertals and modern humans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSankararaman%26rft.auinit1%253DS.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1002947%26rft.epage%253De1002947%26rft.atitle%253DThe%2BDate%2Bof%2BInterbreeding%2Bbetween%2BNeandertals%2Band%2BModern%2BHumans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002947%26rft_id%253Dinfo%253Apmid%252F23055938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002947&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23055938&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.77" data-doi="10.1128/JVI.03136-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schlub</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimm</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chopra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Fifteen to twenty percent of HIV substitution mutations are associated with recombination.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">3837</span>–<span class="cit-lpage">3849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.03136-13%26rft_id%253Dinfo%253Apmid%252F24453357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6Ijg4LzcvMzgzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Estimation of past demographic parameters from the distribution of pairwise differences when the mutation rates vary among sites: application to human mitochondrial DNA.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1079</span>–<span class="cit-lpage">1089</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSchneider%26rft.auinit1%253DS.%26rft.volume%253D152%26rft.issue%253D3%26rft.spage%253D1079%26rft.epage%253D1089%26rft.atitle%253DEstimation%2Bof%2BPast%2BDemographic%2BParameters%2BFrom%2Bthe%2BDistribution%2Bof%2BPairwise%2BDifferences%2BWhen%2Bthe%2BMutation%2BRates%2BVary%2BAmong%2BSites%253A%2BApplication%2Bto%2BHuman%2BMitochondrial%2BDNA%26rft_id%253Dinfo%253Apmid%252F10388826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi8zLzEwNzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.79" data-doi="10.1371/journal.ppat.1002286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnell</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joseph</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spudich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanstrom</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">HIV-1 replication in the central nervous system occurs in two distinct cell types.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BPathog%26rft.aulast%253DSchnell%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De1002286%26rft.epage%253De1002286%26rft.atitle%253DHIV-1%2Breplication%2Bin%2Bthe%2Bcentral%2Bnervous%2Bsystem%2Boccurs%2Bin%2Btwo%2Bdistinct%2Bcell%2Btypes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1002286%26rft_id%253Dinfo%253Apmid%252F22007152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1002286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22007152&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.80" data-doi="10.1093/nar/gkp371"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">A.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bulla</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leitner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korber</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">jphmm: improving the reliability of recombination prediction in HIV-1.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">W647</span>–<span class="cit-lpage">W651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp371%26rft_id%253Dinfo%253Apmid%252F19443440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNy9zdXBwbF8yL1c2NDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.81" data-doi="10.1098/rspb.2001.1906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seehausen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Patterns in fish radiation are compatible with pleistocene desiccation of lake victoria and 14,600 year history for its cichlid species flock.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DSeehausen%26rft.auinit1%253DO.%26rft.volume%253D269%26rft.issue%253D1490%26rft.spage%253D491%26rft.epage%253D497%26rft.atitle%253DPatterns%2Bin%2Bfish%2Bradiation%2Bare%2Bcompatible%2Bwith%2BPleistocene%2Bdesiccation%2Bof%2BLake%2BVictoria%2Band%2B14%2B600%2Byear%2Bhistory%2Bfor%2Bits%2Bcichlid%2Bspecies%2Bflock%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2001.1906%26rft_id%253Dinfo%253Apmid%252F11886641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMjoiMjY5LzE0OTAvNDkxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.82" data-doi="10.1016/j.tree.2004.01.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seehausen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Hybridization and adaptive radiation.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">198</span>–<span class="cit-lpage">207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DSeehausen%26rft.auinit1%253DO.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D198%26rft.epage%253D207%26rft.atitle%253DHybridization%2Band%2Badaptive%2Bradiation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2004.01.003%26rft_id%253Dinfo%253Apmid%252F16701254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2004.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16701254&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220842400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.83" data-doi="10.1098/rstb.2010.0031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">B. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The evolution of HIV-1 and the origin of AIDS.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">365</span>: <span class="cit-fpage">2487</span>–<span class="cit-lpage">2494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DSharp%26rft.auinit1%253DP.%2BM.%26rft.volume%253D365%26rft.issue%253D1552%26rft.spage%253D2487%26rft.epage%253D2494%26rft.atitle%253DThe%2Bevolution%2Bof%2BHIV-1%2Band%2Bthe%2Borigin%2Bof%2BAIDS%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2010.0031%26rft_id%253Dinfo%253Apmid%252F20643738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjUvMTU1Mi8yNDg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.84"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simonsen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aquadro</span>  <span class="cit-name-given-names">C. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Properties of statistical tests of neutrality for DNA polymorphism data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSimonsen%26rft.auinit1%253DK.%2BL.%26rft.volume%253D141%26rft.issue%253D1%26rft.spage%253D413%26rft.epage%253D429%26rft.atitle%253DProperties%2Bof%2Bstatistical%2Btests%2Bof%2Bneutrality%2Bfor%2BDNA%2Bpolymorphism%2Bdata.%26rft_id%253Dinfo%253Apmid%252F8536987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQxLzEvNDEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Gene genealogies within mutant allelic classes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">579</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D143%26rft.issue%253D1%26rft.spage%253D579%26rft.epage%253D587%26rft.atitle%253DGene%2BGenealogies%2BWithin%2BMutant%2BAllelic%2BClasses%26rft_id%253Dinfo%253Apmid%252F8722806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQzLzEvNTc5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Pairwise comparisons of mitochondrial DNA sequences in stable and exponentially growing populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSlatkin%26rft.auinit1%253DM.%26rft.volume%253D129%26rft.issue%253D2%26rft.spage%253D555%26rft.epage%253D562%26rft.atitle%253DPairwise%2Bcomparisons%2Bof%2Bmitochondrial%2BDNA%2Bsequences%2Bin%2Bstable%2Band%2Bexponentially%2Bgrowing%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F1743491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI5LzIvNTU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.87" data-doi="10.1186/1742-4690-8-87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snoeck</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartha</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douek</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Telenti</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping of positive selection sites in the HIV-1 genome in the context of RNA and protein structural constraints.</span> <abbr class="cit-jnl-abbrev">Retrovirology</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DRetrovirology%26rft.aulast%253DSnoeck%26rft.auinit1%253DJ.%26rft.volume%253D8%26rft.spage%253D87%26rft.epage%253D87%26rft.atitle%253DMapping%2Bof%2Bpositive%2Bselection%2Bsites%2Bin%2Bthe%2BHIV-1%2Bgenome%2Bin%2Bthe%2Bcontext%2Bof%2BRNA%2Band%2Bprotein%2Bstructural%2Bconstraints.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1742-4690-8-87%26rft_id%253Dinfo%253Apmid%252F22044801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1742-4690-8-87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22044801&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.88" data-doi="10.1038/nrg3446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Understanding the origin of species with genome-scale data: modelling gene flow.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">404</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D404%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3446%26rft_id%253Dinfo%253Apmid%252F23657479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3446&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23657479&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.89" data-doi="10.1111/j.1365-294X.2011.05247.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sousa</span>  <span class="cit-name-given-names">V. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grelaud</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hey</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">On the nonidentifiability of migration time estimates in isolation with migration models.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3956</span>–<span class="cit-lpage">3962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DSousa%26rft.auinit1%253DV.%2BC.%26rft.volume%253D20%26rft.issue%253D19%26rft.spage%253D3956%26rft.epage%253D3962%26rft.atitle%253DOn%2Bthe%2Bnonidentifiability%2Bof%2Bmigration%2Btime%2Bestimates%2Bin%2Bisolation%2Bwith%2Bmigration%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05247.x%26rft_id%253Dinfo%253Apmid%252F22039614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05247.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22039614&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.90" data-doi="10.1111/j.1365-294X.2011.05048.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strasburg</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Interpreting the estimated timing of migration events between hybridizing species.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2353</span>–<span class="cit-lpage">2366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DStrasburg%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D20%26rft.issue%253D11%26rft.spage%253D2353%26rft.epage%253D2366%26rft.atitle%253DInterpreting%2Bthe%2Bestimated%2Btiming%2Bof%2Bmigration%2Bevents%2Bbetween%2Bhybridizing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05048.x%26rft_id%253Dinfo%253Apmid%252F21375636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05048.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21375636&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290725600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.91" data-doi="10.1093/oxfordjournals.molbev.a003776"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strimmer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pybus</span>  <span class="cit-name-given-names">O. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Exploring the demographic history of DNA sequences using the generalized skyline plot.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2298</span>–<span class="cit-lpage">2305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DStrimmer%26rft.auinit1%253DK.%26rft.volume%253D18%26rft.issue%253D12%26rft.spage%253D2298%26rft.epage%253D2305%26rft.atitle%253DExploring%2Bthe%2BDemographic%2BHistory%2Bof%2BDNA%2BSequences%2BUsing%2Bthe%2BGeneralized%2BSkyline%2BPlot%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a003776%26rft_id%253Dinfo%253Apmid%252F11719579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8yMjk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.92" data-doi="10.1128/JVI.74.23.11367-11376.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graf</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Briesen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Characterization of a virtually full-length human immunodeficiency virus type 1 genome of a prevalent intersubtype (c/b) recombinant strain in China.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">11367</span>–<span class="cit-lpage">11376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DSu%26rft.auinit1%253DL.%26rft.volume%253D74%26rft.issue%253D23%26rft.spage%253D11367%26rft.epage%253D11376%26rft.atitle%253DCharacterization%2Bof%2Ba%2BVirtually%2BFull-Length%2BHuman%2BImmunodeficiency%2BVirus%2BType%2B1%2BGenome%2Bof%2Ba%2BPrevalent%2BIntersubtype%2B%2528C%252FB%2527%2529%2BRecombinant%2BStrain%2Bin%2BChina%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.74.23.11367-11376.2000%26rft_id%253Dinfo%253Apmid%252F11070037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI3NC8yMy8xMTM2NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMS8xMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.94" data-doi="10.1146/annurev.es.26.110195.002015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takahata</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A genetic perspective on the origin and history of humans.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DTakahata%26rft.auinit1%253DN.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D343%26rft.epage%253D372%26rft.atitle%253DA%2BGenetic%2BPerspective%2Bon%2Bthe%2BOrigin%2Band%2BHistory%2Bof%2BHumans%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.es.26.110195.002015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.es.26.110195.002015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TH33000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.95" data-doi="10.1016/j.vaccine.2009.07.101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takebe</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hase</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uenishi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reconstructing the epidemic history of HIV-1 circulating recombinant forms crf07_bc and crf08_bc in east asia: the relevance of genetic diversity and phylodynamics for vaccine strategies.</span> <abbr class="cit-jnl-abbrev">Vaccine</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">B39</span>–<span class="cit-lpage">B44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVaccine%26rft.volume%253D28%26rft.spage%253DB39%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vaccine.2009.07.101%26rft_id%253Dinfo%253Apmid%252F20510742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vaccine.2009.07.101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20510742&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.96" data-doi="10.1016/S1473-3099(10)70186-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tebit</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arts</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Tracking a century of global expansion and evolution of HIV to drive understanding and to combat disease.</span> <abbr class="cit-jnl-abbrev">Lancet Infect. Dis.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BLancet%2Binfectious%2Bdiseases%26rft.stitle%253DLancet%2BInfect%2BDis%26rft.aulast%253DTebit%26rft.auinit1%253DD.%2BM.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D45%26rft.epage%253D56%26rft.atitle%253DTracking%2Ba%2Bcentury%2Bof%2Bglobal%2Bexpansion%2Band%2Bevolution%2Bof%2BHIV%2Bto%2Bdrive%2Bunderstanding%2Band%2Bto%2Bcombat%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252810%252970186-9%26rft_id%253Dinfo%253Apmid%252F21126914%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(10)70186-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21126914&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287177700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.97" data-doi="10.1111/mec.12755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tellier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemaire</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Coalescence 2.0: a multiple branching of recent theoretical developments and their applications.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2637</span>–<span class="cit-lpage">2652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D2637%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.98" data-doi="10.1097/COH.0000000000000140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vuilleumier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonhoeffer</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Contribution of recombination to the evolutionary history of HIV.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. HIV AIDS</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BHIV%2BAIDS%26rft.volume%253D10%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1097%252FCOH.0000000000000140%26rft_id%253Dinfo%253Apmid%252F25612323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/COH.0000000000000140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25612323&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.99" data-doi="10.1007/BF02931753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Pairwise differences under a general model of population subdivision.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D75%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02931753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02931753&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wakeley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Nonequilibrium migration in human history.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1863</span>–<span class="cit-lpage">1871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWakeley%26rft.auinit1%253DJ.%26rft.volume%253D153%26rft.issue%253D4%26rft.spage%253D1863%26rft.epage%253D1871%26rft.atitle%253DNonequilibrium%2BMigration%2Bin%2BHuman%2BHistory%26rft_id%253Dinfo%253Apmid%252F10581291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1My80LzE4NjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.101" data-doi="10.1089/aid.2013.0230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsi</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Near full-length genome characterization of a new crf01_ae/crf08_bc recombinant transmitted between a heterosexual couple in Guangxi, China.</span> <abbr class="cit-jnl-abbrev">AIDS Res. Hum. Retroviruses</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">484</span>–<span class="cit-lpage">488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2BRes.%2BHum.%2BRetroviruses%26rft.volume%253D30%26rft.spage%253D484%26rft_id%253Dinfo%253Adoi%252F10.1089%252Faid.2013.0230%26rft_id%253Dinfo%253Apmid%252F24164454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/aid.2013.0230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24164454&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.102" data-doi="10.1089/aid.2013.0235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">His</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification of a novel HIV-1 circulating recombinant form (crf62_bc) in western Yunnan of China.</span> <abbr class="cit-jnl-abbrev">AIDS Res. Hum. Retroviruses</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">380</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2BRes.%2BHum.%2BRetroviruses%26rft.volume%253D30%26rft.spage%253D380%26rft_id%253Dinfo%253Adoi%252F10.1089%252Faid.2013.0235%26rft_id%253Dinfo%253Apmid%252F24164474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/aid.2013.0235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24164474&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.103" data-doi="10.1016/j.tpb.2012.05.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson-Herbots</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The distribution of the coalescence time and the number of pairwise nucleotide differences in a model of population divergence or speciation with an initial period of gene flow.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">92</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWilkinson-Herbots%26rft.auinit1%253DH.%2BM.%26rft.volume%253D82%26rft.issue%253D2%26rft.spage%253D92%26rft.epage%253D108%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bthe%2Bcoalescence%2Btime%2Band%2Bthe%2Bnumber%2Bof%2Bpairwise%2Bnucleotide%2Bdifferences%2Bin%2Ba%2Bmodel%2Bof%2Bpopulation%2Bdivergence%2Bor%2Bspeciation%2Bwith%2Ban%2Binitial%2Bperiod%2Bof%2Bgene%2Bflow.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2012.05.003%26rft_id%253Dinfo%253Apmid%252F22687581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2012.05.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22687581&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1931</span> <span class="cit-article-title">Evolution in mendelian populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D159%26rft.atitle%253DEVOLUTION%2BIN%2BMENDELIAN%2BPOPULATIONS%26rft_id%253Dinfo%253Apmid%252F17246615%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiIxNi8yLzk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8xLzEwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-105-1" title="View reference  in text" id="ref-105">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.105" data-doi="10.1097/00002030-200207050-00012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xia</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kusagawa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ben</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">On-going generation of multiple forms of HIV-1 intersubtype recombinants in the Yunnan province of China.</span> <abbr class="cit-jnl-abbrev">AIDS</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1401</span>–<span class="cit-lpage">1407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAIDS%2B%2528London%252C%2BEngland%2529%26rft.stitle%253DAIDS%26rft.aulast%253DYang%26rft.auinit1%253DR.%26rft.volume%253D16%26rft.issue%253D10%26rft.spage%253D1401%26rft.epage%253D1407%26rft.atitle%253DOn-going%2Bgeneration%2Bof%2Bmultiple%2Bforms%2Bof%2BHIV-1%2Bintersubtype%2Brecombinants%2Bin%2Bthe%2BYunnan%2BProvince%2Bof%2BChina.%26rft_id%253Dinfo%253Adoi%252F10.1097%252F00002030-200207050-00012%26rft_id%253Dinfo%253Apmid%252F12131217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1097/00002030-200207050-00012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12131217&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-106-1" title="View reference  in text" id="ref-106">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.106" data-doi="10.1534/genetics.106.061432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.-I.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Statistical tests for detecting positive selection by utilizing high-frequency variants.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1431</span>–<span class="cit-lpage">1439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061432%26rft_id%253Dinfo%253Apmid%252F16951063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC8zLzE0MzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzEvMTA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-107-1" title="View reference  in text" id="ref-107">↵</a><div class="cit ref-cit ref-journal" id="cit-6.1.107.107" data-doi="10.1016/S1473-3099(13)70245-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">E. P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jing</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">HIV prevalence in China: integration of surveillance data and a systematic review.</span> <abbr class="cit-jnl-abbrev">Lancet Infect. Dis.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">955</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%2BInfect.%2BDis.%26rft.volume%253D13%26rft.spage%253D955%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1473-3099%252813%252970245-7%26rft_id%253Dinfo%253Apmid%252F24107261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1473-3099(13)70245-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24107261&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F1%2F107.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325956000033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/1/107.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/1/99" title="Using Ancient Samples in Projection Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/1/121" title="Disruption of Methionine Metabolism in Drosophila melanogaster Impacts Histone Methylation and Results in Loss of Viability" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/1"><span>Volume 6 Issue 1, January 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/1.cover-source.jpg?itok=9zFiZvCK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/1/107.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gvlC4hZX1qVbk5iAPqmMWz9403_dc-b_2um33DoFFq8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/1/107" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/1/107" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/1/107" class="active" data-icon-position="" data-hide-link-title="0">The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460228" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-l1gAmGWJn4RWiF6LE0ux97QtHOTwvDnEDbQpbvIOjMM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/107.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/1/107" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-jCfHI-FQupTclGoHRwyfB0lbEkIL52AGRaPhExcUB1A" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/1/107" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/1/107.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460228" id="node460228--2" data-pisa="ggg;6/1/107" data-pisa-master="ggg;g3.115.024208" data-apath="/ggg/6/1/107.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5961-5064" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicolas Alcala</span>, <span class="highwire-citation-author" data-delta="1">Jeffrey D. Jensen</span>, <span class="highwire-citation-author" data-delta="2">Amalio Telenti</span> and <span class="highwire-citation-author" data-delta="3">Séverine Vuilleumier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">107-120;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024208 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460228--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas Alcala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Lausanne, Lausanne CH-1015, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BAlcala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alcala%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BAlcala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5961-5064" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicolas Alcala</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nalcala@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">nalcala@stanford.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey D. Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne CH-1015, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BD.%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BD.%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Amalio Telenti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genomic Medicine, The J. Craig Venter Institute, 4120 Capricorn Lane, La Jolla, California 92037</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amalio%2BTelenti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Telenti%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmalio%2BTelenti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Séverine Vuilleumier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Lausanne, Lausanne CH-1015, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne CH-1015, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Microbiology, University Hospital and University of Lausanne, Lausanne CH-1011, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S%C3%A9verine%2BVuilleumier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vuilleumier%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS%25C3%25A9verine%2BVuilleumier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/1/107" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/1/107&amp;title=The%20Genomic%20Signature%20of%20Population%20Reconnection%20Following%20Isolation%3A%20From%20Theory%20to%20HIV" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/1/107&amp;title=The%20Genomic%20Signature%20of%20Population%20Reconnection%20Following%20Isolation%3A%20From%20Theory%20to%20HIV" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/1/107&amp;title=The%20Genomic%20Signature%20of%20Population%20Reconnection%20Following%20Isolation%3A%20From%20Theory%20to%20HIV" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/1/107&amp;text=The%20Genomic%20Signature%20of%20Population%20Reconnection%20Following%20Isolation%3A%20From%20Theory%20to%20HIV" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/1/107&amp;title=The%20Genomic%20Signature%20of%20Population%20Reconnection%20Following%20Isolation%3A%20From%20Theory%20to%20HIV" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/1/107&amp;t=The%20Genomic%20Signature%20of%20Population%20Reconnection%20Following%20Isolation%3A%20From%20Theory%20to%20HIV" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/1/107&amp;title=The%20Genomic%20Signature%20of%20Population%20Reconnection%20Following%20Isolation%3A%20From%20Theory%20to%20HIV" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/1/107&amp;title=The%20Genomic%20Signature%20of%20Population%20Reconnection%20Following%20Isolation%3A%20From%20Theory%20to%20HIV" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460228/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460228" id="node460228--3" data-pisa="ggg;6/1/107" data-pisa-master="ggg;g3.115.024208" data-apath="/ggg/6/1/107.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Genomic Signature of Population Reconnection Following Isolation: From Theory to HIV</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5961-5064" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Nicolas Alcala</span>, <span class="highwire-citation-author" data-delta="1">Jeffrey D. Jensen</span>, <span class="highwire-citation-author" data-delta="2">Amalio Telenti</span> and <span class="highwire-citation-author" data-delta="3">Séverine Vuilleumier</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">107-120;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024208 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460228--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nicolas Alcala </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Lausanne, Lausanne CH-1015, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, Stanford University, Stanford, California 94305-5020</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BAlcala%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Alcala%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BAlcala%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5961-5064" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Nicolas Alcala</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nalcala@stanford.edu" class="" data-icon-position="" data-hide-link-title="0">nalcala@stanford.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jeffrey D. Jensen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne CH-1015, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BD.%2BJensen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jensen%20JD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJeffrey%2BD.%2BJensen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Amalio Telenti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Genomic Medicine, The J. Craig Venter Institute, 4120 Capricorn Lane, La Jolla, California 92037</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amalio%2BTelenti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Telenti%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAmalio%2BTelenti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Séverine Vuilleumier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolution, University of Lausanne, Lausanne CH-1015, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne, Lausanne CH-1015, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Institute of Microbiology, University Hospital and University of Lausanne, Lausanne CH-1011, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=S%C3%A9verine%2BVuilleumier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vuilleumier%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AS%25C3%25A9verine%2BVuilleumier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=107&amp;rft.epage=120&amp;rft.atitle=The%20Genomic%20Signature%20of%20Population%20Reconnection%20Following%20Isolation%3A%20From%20Theory%20to%20HIV&amp;rft.volume=6&amp;rft.issue=1&amp;rft.date=2016-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Alcala%2C+Nicolas&amp;rft.au=Jensen%2C+Jeffrey+D.&amp;rft.au=Telenti%2C+Amalio&amp;rft.au=Vuilleumier%2C+S%C3%A9verine"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460228/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460228/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460228/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460228/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460228/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460228/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460228/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460228/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460228/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460228/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460228/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460228/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/1/107" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/1/107.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/1/107.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/1/107.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/1/107.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
